{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "8ed1778b",
   "metadata": {},
   "source": [
    "# Assignment 3\n",
    "\n",
    "### Josh Gilheany 22221584\n",
    "\n",
    "***\n",
    "\n",
    "#### Question 1:\n",
    "\n",
    "\n",
    "For this question I decided to use a function defined below to track the sequence provided in the question. Each time a match was found a counter ticked up and printed the presence of a binding site, as well as the matching sequence and the start/stop location of said sequences."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "8af3bd35",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CTCF Motif from 89 to 100\n",
      "Seq is: CCCGGAGTTGG\n",
      "\n",
      "1 CTCF motifs detected in file\n"
     ]
    }
   ],
   "source": [
    "import re\n",
    "def find_CTCF_motif(sequence):\n",
    "    motif_regex = re.compile(r'CC...AG..GG') #The sequence we're searching for\n",
    "    runs = re.finditer(motif_regex,sequence) #finditer used because we want to scan entire code\n",
    "    match_count=0 #used to keep track of the number of hits\n",
    "    \n",
    "    for match in runs:\n",
    "        match_count += 1\n",
    "        run_start = match.start()\n",
    "        run_end = match.end()\n",
    "        match_seq = sequence[run_start:run_end] #isolates the part of the string corresponding to the ctcf motif for later printing\n",
    "        print(\"CTCF Motif from \" + str(run_start) + \" to \" +str(run_end))\n",
    "        print(\"Seq is: \"+match_seq)\n",
    "        print(\"\")\n",
    "    print(f\"{match_count} CTCF motifs detected in file\") \n",
    "\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "with open('/home/josh/Documents/MA5114_ProgrammingforBio/intron.txt') as in_file:\n",
    "    for line in in_file:\n",
    "        find_CTCF_motif(line) #reads in the file and iterates it through the above function\n",
    "        \n",
    "    \n",
    "    \n",
    "    \n",
    "    "
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d35fdbd8",
   "metadata": {},
   "source": [
    "***\n",
    "#### Question 2.\n",
    "\n",
    "Here I created a small extract_exon function which takes two variables, a list and a string, and uses them to return a list of sequences which can then be cocatenated later. I believe it would have been easier to create one large function that accepted files as inputs, but I don't think the question was asking for this."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "59963550",
   "metadata": {},
   "outputs": [],
   "source": [
    "def extract_exon(this_exon,sequence):\n",
    "    \n",
    "    start = int(this_exon[0])\n",
    "    end = int(this_exon[1])\n",
    "    match_seq = sequence[start:end]\n",
    "    return match_seq\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "exon_list = []\n",
    "\n",
    "with open('/home/josh/Documents/MA5114_ProgrammingforBio/exons.txt') as exon_file:\n",
    "    for line in exon_file:\n",
    "        line = line.strip()\n",
    "        line = line.split(',')\n",
    "        exon_list.append(line) #creates a list of list corresponding to the read-in exons\n",
    "\n",
    "genomicRNA = \"\"\n",
    "with open('/home/josh/Documents/MA5114_ProgrammingforBio/genomic_RNA.fa') as rna_file:\n",
    "    for line in rna_file:\n",
    "        if line != \"\" and line[0].isalpha():\n",
    "            genomicRNA = line + genomicRNA #appends every line in the genomic_RNA.fa file that starts as a letter to a string\n",
    "        \n",
    "\n",
    "splicedRNA=\"\"            \n",
    "for exon in exon_list: #runs through the list of exons, chopping out matching sequence from the genomicRNA\n",
    "    rna = extract_exon(exon,genomicRNA)\n",
    "    splicedRNA = rna + splicedRNA #appends each match to a new string\n",
    "\n",
    "\n",
    "with open('genomic_mRNA.fa','w') as out_file:\n",
    "    print(splicedRNA, file = out_file) #prints out the resulting spliced sequence to a .fa file"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "dbdc64d0",
   "metadata": {},
   "source": [
    "***\n",
    "\n",
    "#### Question 3.\n",
    "\n",
    "Here I attempted to read in a file, strip it, convert it to uppercase and verify that it started with a letter. This line would then be appended to a new table, which I then joined into one large string.\n",
    "\n",
    "From that, I generated the reverse complement of the DNA by taking the normal DNA seq and running through it in reverse (i.e. [::-1])\n",
    "\n",
    "Then I created a window length using a list in the range of 6 and 21, generating integers from 6 to 20.\n",
    "\n",
    "I initialised a list for a future table, but sadly I did not have time to finish the table.\n",
    "\n",
    "I iterated through the length of the window, creating chunks of the complement strand equal to the window length. This was then compared to the standard chromosome using a modified version of the code from question 1, but due to time constraints it only printed to the console rather than the as planned table :["
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "72eb1534",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Complement found from 1954 to 1960\n",
      "Complement found from 2592 to 2598\n",
      "Complement found from 12609 to 12615\n",
      "Complement found from 13095 to 13101\n",
      "Complement found from 15802 to 15808\n",
      "Seq is: GTAGCA\n",
      "5 GTAGCA motifs detected in file\n",
      "\n",
      "Seq is: CTACAG\n",
      "0 CTACAG motifs detected in file\n",
      "\n",
      "Complement found from 835 to 841\n",
      "Complement found from 965 to 971\n",
      "Complement found from 1735 to 1741\n",
      "Complement found from 4004 to 4010\n",
      "Complement found from 4570 to 4576\n",
      "Complement found from 4609 to 4615\n",
      "Complement found from 4777 to 4783\n",
      "Complement found from 5404 to 5410\n",
      "Complement found from 8493 to 8499\n",
      "Complement found from 10057 to 10063\n",
      "Complement found from 12326 to 12332\n",
      "Complement found from 13871 to 13877\n",
      "Complement found from 14495 to 14501\n",
      "Complement found from 14775 to 14781\n",
      "Seq is: AATAAA\n",
      "14 AATAAA motifs detected in file\n",
      "\n",
      "Complement found from 453 to 459\n",
      "Complement found from 3165 to 3171\n",
      "Complement found from 4246 to 4252\n",
      "Complement found from 6150 to 6156\n",
      "Complement found from 6183 to 6189\n",
      "Complement found from 7092 to 7098\n",
      "Complement found from 7623 to 7629\n",
      "Complement found from 7952 to 7958\n",
      "Complement found from 8225 to 8231\n",
      "Complement found from 8602 to 8608\n",
      "Complement found from 10914 to 10920\n",
      "Complement found from 11231 to 11237\n",
      "Complement found from 12382 to 12388\n",
      "Complement found from 12480 to 12486\n",
      "Complement found from 13644 to 13650\n",
      "Complement found from 13751 to 13757\n",
      "Complement found from 14152 to 14158\n",
      "Complement found from 14486 to 14492\n",
      "Complement found from 16543 to 16549\n",
      "Seq is: TTCCCC\n",
      "19 TTCCCC motifs detected in file\n",
      "\n",
      "Complement found from 8062 to 8068\n",
      "Complement found from 15343 to 15349\n",
      "Seq is: TTGCAC\n",
      "2 TTGCAC motifs detected in file\n",
      "\n",
      "Complement found from 11158 to 11164\n",
      "Complement found from 14555 to 14561\n",
      "Seq is: ACCCGA\n",
      "2 ACCCGA motifs detected in file\n",
      "\n",
      "Complement found from 728 to 734\n",
      "Complement found from 1114 to 1120\n",
      "Complement found from 2070 to 2076\n",
      "Complement found from 7164 to 7170\n",
      "Complement found from 7567 to 7573\n",
      "Complement found from 9576 to 9582\n",
      "Seq is: TAAATC\n",
      "6 TAAATC motifs detected in file\n",
      "\n",
      "Seq is: CGAAAT\n",
      "0 CGAAAT motifs detected in file\n",
      "\n",
      "Complement found from 1068 to 1074\n",
      "Complement found from 9709 to 9715\n",
      "Complement found from 11883 to 11889\n",
      "Seq is: ACTGGG\n",
      "3 ACTGGG motifs detected in file\n",
      "\n",
      "Complement found from 1506 to 1512\n",
      "Complement found from 2860 to 2866\n",
      "Complement found from 3150 to 3156\n",
      "Complement found from 9820 to 9826\n",
      "Complement found from 11799 to 11805\n",
      "Complement found from 13251 to 13257\n",
      "Complement found from 13440 to 13446\n",
      "Complement found from 15290 to 15296\n",
      "Complement found from 15818 to 15824\n",
      "Complement found from 16509 to 16515\n",
      "Seq is: ACTTCA\n",
      "10 ACTTCA motifs detected in file\n",
      "\n",
      "Complement found from 7988 to 7994\n",
      "Complement found from 11540 to 11546\n",
      "Complement found from 14852 to 14858\n",
      "Complement found from 15617 to 15623\n",
      "Complement found from 15888 to 15894\n",
      "Seq is: TCCTTG\n",
      "5 TCCTTG motifs detected in file\n",
      "\n",
      "Complement found from 6284 to 6290\n",
      "Complement found from 7607 to 7613\n",
      "Seq is: GTCTAC\n",
      "2 GTCTAC motifs detected in file\n",
      "\n",
      "Complement found from 378 to 384\n",
      "Complement found from 789 to 795\n",
      "Complement found from 1250 to 1256\n",
      "Complement found from 3633 to 3639\n",
      "Complement found from 8967 to 8973\n",
      "Complement found from 9155 to 9161\n",
      "Complement found from 9518 to 9524\n",
      "Complement found from 10849 to 10855\n",
      "Complement found from 11529 to 11535\n",
      "Complement found from 13464 to 13470\n",
      "Complement found from 13716 to 13722\n",
      "Complement found from 16524 to 16530\n",
      "Seq is: AGCCTA\n",
      "12 AGCCTA motifs detected in file\n",
      "\n",
      "Complement found from 2419 to 2425\n",
      "Complement found from 7546 to 7552\n",
      "Seq is: TGTCAA\n",
      "2 TGTCAA motifs detected in file\n",
      "\n",
      "Complement found from 2672 to 2678\n",
      "Complement found from 8360 to 8366\n",
      "Complement found from 16411 to 16417\n",
      "Seq is: GTGAAA\n",
      "3 GTGAAA motifs detected in file\n",
      "\n",
      "Complement found from 2986 to 2992\n",
      "Seq is: TCGATG\n",
      "1 TCGATG motifs detected in file\n",
      "\n",
      "Seq is: GGGGTT\n",
      "0 GGGGTT motifs detected in file\n",
      "\n",
      "Complement found from 1049 to 1055\n",
      "Complement found from 8561 to 8567\n",
      "Complement found from 9104 to 9110\n",
      "Complement found from 10983 to 10989\n",
      "Complement found from 11355 to 11361\n",
      "Complement found from 11999 to 12005\n",
      "Complement found from 13940 to 13946\n",
      "Complement found from 15417 to 15423\n",
      "Complement found from 15579 to 15585\n",
      "Seq is: CACAAT\n",
      "9 CACAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCCGG\n",
      "0 ACCCGG motifs detected in file\n",
      "\n",
      "Complement found from 11846 to 11852\n",
      "Seq is: GCCTCG\n",
      "1 GCCTCG motifs detected in file\n",
      "\n",
      "Complement found from 591 to 597\n",
      "Complement found from 1494 to 1500\n",
      "Complement found from 2251 to 2257\n",
      "Complement found from 2325 to 2331\n",
      "Complement found from 3327 to 3333\n",
      "Complement found from 4928 to 4934\n",
      "Complement found from 5011 to 5017\n",
      "Complement found from 5210 to 5216\n",
      "Complement found from 6367 to 6373\n",
      "Complement found from 8775 to 8781\n",
      "Complement found from 9329 to 9335\n",
      "Complement found from 12091 to 12097\n",
      "Complement found from 12981 to 12987\n",
      "Complement found from 13986 to 13992\n",
      "Complement found from 14442 to 14448\n",
      "Complement found from 15739 to 15745\n",
      "Complement found from 16443 to 16449\n",
      "Seq is: CTCCTC\n",
      "17 CTCCTC motifs detected in file\n",
      "\n",
      "Complement found from 5454 to 5460\n",
      "Complement found from 6834 to 6840\n",
      "Complement found from 7155 to 7161\n",
      "Complement found from 7808 to 7814\n",
      "Complement found from 11240 to 11246\n",
      "Complement found from 12757 to 12763\n",
      "Complement found from 13519 to 13525\n",
      "Complement found from 13573 to 13579\n",
      "Complement found from 14798 to 14804\n",
      "Seq is: TCATCG\n",
      "9 TCATCG motifs detected in file\n",
      "\n",
      "Complement found from 4337 to 4343\n",
      "Complement found from 4356 to 4362\n",
      "Complement found from 7319 to 7325\n",
      "Complement found from 8515 to 8521\n",
      "Seq is: TGAGAA\n",
      "4 TGAGAA motifs detected in file\n",
      "\n",
      "Complement found from 7660 to 7666\n",
      "Complement found from 13329 to 13335\n",
      "Complement found from 14992 to 14998\n",
      "Seq is: CACGCC\n",
      "3 CACGCC motifs detected in file\n",
      "\n",
      "Complement found from 8821 to 8827\n",
      "Complement found from 12352 to 12358\n",
      "Seq is: CTATAA\n",
      "2 CTATAA motifs detected in file\n",
      "\n",
      "Complement found from 197 to 203\n",
      "Complement found from 4397 to 4403\n",
      "Complement found from 8713 to 8719\n",
      "Complement found from 11408 to 11414\n",
      "Complement found from 16477 to 16483\n",
      "Seq is: CTAAAG\n",
      "5 CTAAAG motifs detected in file\n",
      "\n",
      "Complement found from 141 to 147\n",
      "Complement found from 8788 to 8794\n",
      "Complement found from 14430 to 14436\n",
      "Complement found from 15022 to 15028\n",
      "Seq is: TGCCTC\n",
      "4 TGCCTC motifs detected in file\n",
      "\n",
      "Complement found from 1702 to 1708\n",
      "Complement found from 2784 to 2790\n",
      "Complement found from 3502 to 3508\n",
      "Complement found from 4185 to 4191\n",
      "Complement found from 4500 to 4506\n",
      "Complement found from 4734 to 4740\n",
      "Complement found from 6644 to 6650\n",
      "Complement found from 6825 to 6831\n",
      "Complement found from 7405 to 7411\n",
      "Complement found from 8460 to 8466\n",
      "Complement found from 10127 to 10133\n",
      "Complement found from 10280 to 10286\n",
      "Complement found from 10792 to 10798\n",
      "Seq is: CTACCA\n",
      "13 CTACCA motifs detected in file\n",
      "\n",
      "Complement found from 1000 to 1006\n",
      "Complement found from 4592 to 4598\n",
      "Seq is: CCAGTT\n",
      "2 CCAGTT motifs detected in file\n",
      "\n",
      "Complement found from 3586 to 3592\n",
      "Complement found from 4476 to 4482\n",
      "Complement found from 6316 to 6322\n",
      "Complement found from 8989 to 8995\n",
      "Seq is: CCCTGG\n",
      "4 CCCTGG motifs detected in file\n",
      "\n",
      "Seq is: GGATAG\n",
      "0 GGATAG motifs detected in file\n",
      "\n",
      "Complement found from 463 to 469\n",
      "Complement found from 5380 to 5386\n",
      "Complement found from 6309 to 6315\n",
      "Complement found from 8013 to 8019\n",
      "Complement found from 10623 to 10629\n",
      "Complement found from 11403 to 11409\n",
      "Complement found from 11811 to 11817\n",
      "Complement found from 11975 to 11981\n",
      "Complement found from 13031 to 13037\n",
      "Seq is: ACTCCC\n",
      "9 ACTCCC motifs detected in file\n",
      "\n",
      "Complement found from 2546 to 2552\n",
      "Complement found from 6499 to 6505\n",
      "Complement found from 13058 to 13064\n",
      "Seq is: CCCAGT\n",
      "3 CCCAGT motifs detected in file\n",
      "\n",
      "Seq is: AGGTAC\n",
      "0 AGGTAC motifs detected in file\n",
      "\n",
      "Complement found from 6421 to 6427\n",
      "Seq is: CCCTGC\n",
      "1 CCCTGC motifs detected in file\n",
      "\n",
      "Complement found from 4027 to 4033\n",
      "Complement found from 7176 to 7182\n",
      "Complement found from 12478 to 12484\n",
      "Complement found from 12703 to 12709\n",
      "Complement found from 14110 to 14116\n",
      "Complement found from 15026 to 15032\n",
      "Complement found from 15446 to 15452\n",
      "Seq is: TCTTCC\n",
      "7 TCTTCC motifs detected in file\n",
      "\n",
      "Complement found from 1099 to 1105\n",
      "Complement found from 1671 to 1677\n",
      "Complement found from 2779 to 2785\n",
      "Complement found from 4913 to 4919\n",
      "Complement found from 8119 to 8125\n",
      "Complement found from 9599 to 9605\n",
      "Complement found from 10745 to 10751\n",
      "Complement found from 11278 to 11284\n",
      "Complement found from 12945 to 12951\n",
      "Complement found from 13686 to 13692\n",
      "Complement found from 14582 to 14588\n",
      "Complement found from 14636 to 14642\n",
      "Seq is: CTAAAC\n",
      "12 CTAAAC motifs detected in file\n",
      "\n",
      "Complement found from 4839 to 4845\n",
      "Complement found from 10799 to 10805\n",
      "Complement found from 13223 to 13229\n",
      "Complement found from 15464 to 15470\n",
      "Seq is: TGACAT\n",
      "4 TGACAT motifs detected in file\n",
      "\n",
      "Complement found from 7002 to 7008\n",
      "Complement found from 8142 to 8148\n",
      "Seq is: TACACG\n",
      "2 TACACG motifs detected in file\n",
      "\n",
      "Complement found from 6709 to 6715\n",
      "Complement found from 7429 to 7435\n",
      "Complement found from 14664 to 14670\n",
      "Complement found from 15207 to 15213\n",
      "Seq is: ATACAT\n",
      "4 ATACAT motifs detected in file\n",
      "\n",
      "Complement found from 5261 to 5267\n",
      "Complement found from 5880 to 5886\n",
      "Complement found from 11656 to 11662\n",
      "Complement found from 12837 to 12843\n",
      "Complement found from 16318 to 16324\n",
      "Seq is: GCCATT\n",
      "5 GCCATT motifs detected in file\n",
      "\n",
      "Complement found from 1997 to 2003\n",
      "Complement found from 3371 to 3377\n",
      "Complement found from 12202 to 12208\n",
      "Seq is: TACCGA\n",
      "3 TACCGA motifs detected in file\n",
      "\n",
      "Complement found from 486 to 492\n",
      "Complement found from 602 to 608\n",
      "Complement found from 3985 to 3991\n",
      "Complement found from 5525 to 5531\n",
      "Complement found from 15905 to 15911\n",
      "Seq is: AATACA\n",
      "5 AATACA motifs detected in file\n",
      "\n",
      "Complement found from 9923 to 9929\n",
      "Complement found from 12807 to 12813\n",
      "Seq is: TGATAC\n",
      "2 TGATAC motifs detected in file\n",
      "\n",
      "Complement found from 4864 to 4870\n",
      "Complement found from 5410 to 5416\n",
      "Complement found from 13222 to 13228\n",
      "Complement found from 15463 to 15469\n",
      "Seq is: ATGACA\n",
      "4 ATGACA motifs detected in file\n",
      "\n",
      "Complement found from 4245 to 4251\n",
      "Complement found from 8107 to 8113\n",
      "Complement found from 8224 to 8230\n",
      "Complement found from 12381 to 12387\n",
      "Complement found from 14151 to 14157\n",
      "Complement found from 14485 to 14491\n",
      "Seq is: ATTCCC\n",
      "6 ATTCCC motifs detected in file\n",
      "\n",
      "Complement found from 2162 to 2168\n",
      "Complement found from 3308 to 3314\n",
      "Complement found from 3338 to 3344\n",
      "Complement found from 4226 to 4232\n",
      "Complement found from 4347 to 4353\n",
      "Complement found from 6115 to 6121\n",
      "Complement found from 14517 to 14523\n",
      "Complement found from 16069 to 16075\n",
      "Seq is: ACCCAT\n",
      "8 ACCCAT motifs detected in file\n",
      "\n",
      "Complement found from 297 to 303\n",
      "Complement found from 347 to 353\n",
      "Complement found from 551 to 557\n",
      "Complement found from 1064 to 1070\n",
      "Complement found from 1683 to 1689\n",
      "Complement found from 1719 to 1725\n",
      "Complement found from 2234 to 2240\n",
      "Complement found from 2787 to 2793\n",
      "Complement found from 3716 to 3722\n",
      "Complement found from 4981 to 4987\n",
      "Complement found from 6439 to 6445\n",
      "Complement found from 7120 to 7126\n",
      "Complement found from 8124 to 8130\n",
      "Complement found from 9891 to 9897\n",
      "Complement found from 11667 to 11673\n",
      "Complement found from 12548 to 12554\n",
      "Complement found from 12669 to 12675\n",
      "Complement found from 13154 to 13160\n",
      "Complement found from 13769 to 13775\n",
      "Complement found from 15674 to 15680\n",
      "Seq is: CCAAAC\n",
      "20 CCAAAC motifs detected in file\n",
      "\n",
      "Complement found from 1722 to 1728\n",
      "Complement found from 4720 to 4726\n",
      "Complement found from 5068 to 5074\n",
      "Complement found from 6337 to 6343\n",
      "Complement found from 6699 to 6705\n",
      "Complement found from 7530 to 7536\n",
      "Complement found from 8699 to 8705\n",
      "Complement found from 8960 to 8966\n",
      "Complement found from 9071 to 9077\n",
      "Complement found from 9361 to 9367\n",
      "Complement found from 12338 to 12344\n",
      "Complement found from 13427 to 13433\n",
      "Complement found from 14478 to 14484\n",
      "Complement found from 14713 to 14719\n",
      "Seq is: AACCAT\n",
      "14 AACCAT motifs detected in file\n",
      "\n",
      "Complement found from 14256 to 14262\n",
      "Seq is: AGGATC\n",
      "1 AGGATC motifs detected in file\n",
      "\n",
      "Complement found from 1308 to 1314\n",
      "Complement found from 1687 to 1693\n",
      "Complement found from 2215 to 2221\n",
      "Complement found from 2768 to 2774\n",
      "Complement found from 3202 to 3208\n",
      "Complement found from 3208 to 3214\n",
      "Complement found from 5021 to 5027\n",
      "Complement found from 5431 to 5437\n",
      "Complement found from 5675 to 5681\n",
      "Complement found from 9209 to 9215\n",
      "Complement found from 10619 to 10625\n",
      "Complement found from 10843 to 10849\n",
      "Complement found from 12016 to 12022\n",
      "Complement found from 12925 to 12931\n",
      "Complement found from 13326 to 13332\n",
      "Complement found from 14356 to 14362\n",
      "Complement found from 14640 to 14646\n",
      "Complement found from 16264 to 16270\n",
      "Complement found from 16288 to 16294\n",
      "Seq is: ACCCAC\n",
      "19 ACCCAC motifs detected in file\n",
      "\n",
      "Complement found from 960 to 966\n",
      "Complement found from 3571 to 3577\n",
      "Complement found from 5301 to 5307\n",
      "Complement found from 5382 to 5388\n",
      "Complement found from 5445 to 5451\n",
      "Complement found from 6843 to 6849\n",
      "Complement found from 8078 to 8084\n",
      "Complement found from 8617 to 8623\n",
      "Complement found from 8938 to 8944\n",
      "Complement found from 10915 to 10921\n",
      "Complement found from 11138 to 11144\n",
      "Complement found from 12481 to 12487\n",
      "Complement found from 13645 to 13651\n",
      "Complement found from 13878 to 13884\n",
      "Complement found from 14807 to 14813\n",
      "Complement found from 15540 to 15546\n",
      "Complement found from 15656 to 15662\n",
      "Complement found from 16188 to 16194\n",
      "Complement found from 16361 to 16367\n",
      "Seq is: TCCCCA\n",
      "19 TCCCCA motifs detected in file\n",
      "\n",
      "Complement found from 1903 to 1909\n",
      "Complement found from 4437 to 4443\n",
      "Complement found from 9670 to 9676\n",
      "Seq is: CCGAAA\n",
      "3 CCGAAA motifs detected in file\n",
      "\n",
      "Complement found from 593 to 599\n",
      "Complement found from 1104 to 1110\n",
      "Complement found from 1496 to 1502\n",
      "Complement found from 3596 to 3602\n",
      "Complement found from 4251 to 4257\n",
      "Complement found from 5013 to 5019\n",
      "Complement found from 5367 to 5373\n",
      "Complement found from 6544 to 6550\n",
      "Complement found from 8683 to 8689\n",
      "Complement found from 10612 to 10618\n",
      "Complement found from 12102 to 12108\n",
      "Complement found from 13844 to 13850\n",
      "Complement found from 14065 to 14071\n",
      "Complement found from 14411 to 14417\n",
      "Complement found from 14444 to 14450\n",
      "Complement found from 14921 to 14927\n",
      "Complement found from 15005 to 15011\n",
      "Complement found from 16221 to 16227\n",
      "Seq is: CCTCAA\n",
      "18 CCTCAA motifs detected in file\n",
      "\n",
      "Complement found from 6853 to 6859\n",
      "Seq is: CGTCAA\n",
      "1 CGTCAA motifs detected in file\n",
      "\n",
      "Complement found from 4067 to 4073\n",
      "Complement found from 7001 to 7007\n",
      "Complement found from 7108 to 7114\n",
      "Complement found from 8141 to 8147\n",
      "Complement found from 8926 to 8932\n",
      "Complement found from 9088 to 9094\n",
      "Complement found from 11211 to 11217\n",
      "Complement found from 12906 to 12912\n",
      "Complement found from 14309 to 14315\n",
      "Complement found from 15031 to 15037\n",
      "Complement found from 15414 to 15420\n",
      "Complement found from 15576 to 15582\n",
      "Seq is: CTACAC\n",
      "12 CTACAC motifs detected in file\n",
      "\n",
      "Complement found from 3755 to 3761\n",
      "Complement found from 5155 to 5161\n",
      "Complement found from 7145 to 7151\n",
      "Complement found from 7780 to 7786\n",
      "Complement found from 8816 to 8822\n",
      "Complement found from 9123 to 9129\n",
      "Complement found from 9851 to 9857\n",
      "Complement found from 10579 to 10585\n",
      "Complement found from 11022 to 11028\n",
      "Complement found from 11313 to 11319\n",
      "Complement found from 11547 to 11553\n",
      "Complement found from 15192 to 15198\n",
      "Complement found from 15630 to 15636\n",
      "Complement found from 15847 to 15853\n",
      "Complement found from 16226 to 16232\n",
      "Seq is: ACTATC\n",
      "15 ACTATC motifs detected in file\n",
      "\n",
      "Complement found from 996 to 1002\n",
      "Complement found from 2249 to 2255\n",
      "Complement found from 5132 to 5138\n",
      "Complement found from 7939 to 7945\n",
      "Complement found from 11320 to 11326\n",
      "Complement found from 12319 to 12325\n",
      "Complement found from 15972 to 15978\n",
      "Complement found from 16245 to 16251\n",
      "Seq is: AACTCC\n",
      "8 AACTCC motifs detected in file\n",
      "\n",
      "Complement found from 437 to 443\n",
      "Complement found from 3399 to 3405\n",
      "Complement found from 8371 to 8377\n",
      "Complement found from 8437 to 8443\n",
      "Complement found from 8767 to 8773\n",
      "Complement found from 8814 to 8820\n",
      "Complement found from 9869 to 9875\n",
      "Complement found from 10299 to 10305\n",
      "Complement found from 13260 to 13266\n",
      "Complement found from 13847 to 13853\n",
      "Complement found from 14729 to 14735\n",
      "Complement found from 15110 to 15116\n",
      "Complement found from 15845 to 15851\n",
      "Complement found from 16224 to 16230\n",
      "Seq is: CAACTA\n",
      "14 CAACTA motifs detected in file\n",
      "\n",
      "Complement found from 7005 to 7011\n",
      "Complement found from 9193 to 9199\n",
      "Complement found from 11589 to 11595\n",
      "Seq is: ACGACA\n",
      "3 ACGACA motifs detected in file\n",
      "\n",
      "Complement found from 5617 to 5623\n",
      "Complement found from 8527 to 8533\n",
      "Complement found from 11186 to 11192\n",
      "Seq is: TGAACG\n",
      "3 TGAACG motifs detected in file\n",
      "\n",
      "Complement found from 2319 to 2325\n",
      "Complement found from 3761 to 3767\n",
      "Complement found from 3993 to 3999\n",
      "Complement found from 9012 to 9018\n",
      "Complement found from 9790 to 9796\n",
      "Complement found from 10040 to 10046\n",
      "Complement found from 11082 to 11088\n",
      "Complement found from 11281 to 11287\n",
      "Complement found from 12672 to 12678\n",
      "Complement found from 13659 to 13665\n",
      "Complement found from 13746 to 13752\n",
      "Seq is: AACATT\n",
      "11 AACATT motifs detected in file\n",
      "\n",
      "Seq is: CGTACC\n",
      "0 CGTACC motifs detected in file\n",
      "\n",
      "Complement found from 307 to 313\n",
      "Complement found from 457 to 463\n",
      "Complement found from 958 to 964\n",
      "Complement found from 3115 to 3121\n",
      "Complement found from 3569 to 3575\n",
      "Complement found from 4105 to 4111\n",
      "Complement found from 5443 to 5449\n",
      "Complement found from 6218 to 6224\n",
      "Complement found from 6290 to 6296\n",
      "Complement found from 7815 to 7821\n",
      "Complement found from 7862 to 7868\n",
      "Complement found from 8470 to 8476\n",
      "Complement found from 10551 to 10557\n",
      "Complement found from 13447 to 13453\n",
      "Complement found from 13582 to 13588\n",
      "Complement found from 14261 to 14267\n",
      "Complement found from 14527 to 14533\n",
      "Complement found from 14805 to 14811\n",
      "Complement found from 15141 to 15147\n",
      "Complement found from 15380 to 15386\n",
      "Complement found from 15538 to 15544\n",
      "Complement found from 16186 to 16192\n",
      "Seq is: CCTCCC\n",
      "22 CCTCCC motifs detected in file\n",
      "\n",
      "Complement found from 354 to 360\n",
      "Complement found from 2500 to 2506\n",
      "Complement found from 4363 to 4369\n",
      "Complement found from 4602 to 4608\n",
      "Complement found from 7131 to 7137\n",
      "Complement found from 8521 to 8527\n",
      "Complement found from 9430 to 9436\n",
      "Complement found from 10811 to 10817\n",
      "Complement found from 12152 to 12158\n",
      "Complement found from 12303 to 12309\n",
      "Complement found from 13970 to 13976\n",
      "Complement found from 14072 to 14078\n",
      "Complement found from 14533 to 14539\n",
      "Seq is: CCAAAA\n",
      "13 CCAAAA motifs detected in file\n",
      "\n",
      "Complement found from 4067 to 4073\n",
      "Complement found from 7001 to 7007\n",
      "Complement found from 7108 to 7114\n",
      "Complement found from 8141 to 8147\n",
      "Complement found from 8926 to 8932\n",
      "Complement found from 9088 to 9094\n",
      "Complement found from 11211 to 11217\n",
      "Complement found from 12906 to 12912\n",
      "Complement found from 14309 to 14315\n",
      "Complement found from 15031 to 15037\n",
      "Complement found from 15414 to 15420\n",
      "Complement found from 15576 to 15582\n",
      "Seq is: CTACAC\n",
      "12 CTACAC motifs detected in file\n",
      "\n",
      "Complement found from 381 to 387\n",
      "Complement found from 529 to 535\n",
      "Complement found from 862 to 868\n",
      "Complement found from 1832 to 1838\n",
      "Complement found from 2578 to 2584\n",
      "Complement found from 4101 to 4107\n",
      "Complement found from 4598 to 4604\n",
      "Complement found from 5237 to 5243\n",
      "Complement found from 5839 to 5845\n",
      "Complement found from 6335 to 6341\n",
      "Complement found from 8679 to 8685\n",
      "Complement found from 8770 to 8776\n",
      "Complement found from 9003 to 9009\n",
      "Complement found from 9347 to 9353\n",
      "Complement found from 9359 to 9365\n",
      "Complement found from 10302 to 10308\n",
      "Complement found from 11852 to 11858\n",
      "Complement found from 12363 to 12369\n",
      "Complement found from 13854 to 13860\n",
      "Complement found from 13998 to 14004\n",
      "Complement found from 14124 to 14130\n",
      "Complement found from 14384 to 14390\n",
      "Complement found from 14765 to 14771\n",
      "Complement found from 15748 to 15754\n",
      "Seq is: CTAACC\n",
      "24 CTAACC motifs detected in file\n",
      "\n",
      "Complement found from 284 to 290\n",
      "Complement found from 355 to 361\n",
      "Complement found from 2501 to 2507\n",
      "Complement found from 4603 to 4609\n",
      "Complement found from 4868 to 4874\n",
      "Complement found from 5280 to 5286\n",
      "Complement found from 7445 to 7451\n",
      "Complement found from 7513 to 7519\n",
      "Complement found from 9431 to 9437\n",
      "Complement found from 10046 to 10052\n",
      "Complement found from 10379 to 10385\n",
      "Complement found from 10812 to 10818\n",
      "Complement found from 12304 to 12310\n",
      "Complement found from 12416 to 12422\n",
      "Complement found from 13229 to 13235\n",
      "Complement found from 14073 to 14079\n",
      "Seq is: CAAAAA\n",
      "16 CAAAAA motifs detected in file\n",
      "\n",
      "Complement found from 2849 to 2855\n",
      "Complement found from 12623 to 12629\n",
      "Seq is: TACATG\n",
      "2 TACATG motifs detected in file\n",
      "\n",
      "Complement found from 7038 to 7044\n",
      "Complement found from 15137 to 15143\n",
      "Seq is: ATGTCC\n",
      "2 ATGTCC motifs detected in file\n",
      "\n",
      "Complement found from 2523 to 2529\n",
      "Complement found from 2668 to 2674\n",
      "Complement found from 2865 to 2871\n",
      "Complement found from 14202 to 14208\n",
      "Complement found from 15768 to 15774\n",
      "Complement found from 15910 to 15916\n",
      "Seq is: ACCAGT\n",
      "6 ACCAGT motifs detected in file\n",
      "\n",
      "Complement found from 278 to 284\n",
      "Complement found from 2441 to 2447\n",
      "Complement found from 4761 to 4767\n",
      "Complement found from 6120 to 6126\n",
      "Complement found from 7537 to 7543\n",
      "Complement found from 7667 to 7673\n",
      "Complement found from 10605 to 10611\n",
      "Complement found from 11702 to 11708\n",
      "Complement found from 11780 to 11786\n",
      "Complement found from 13270 to 13276\n",
      "Complement found from 14290 to 14296\n",
      "Complement found from 16518 to 16524\n",
      "Seq is: TCATAA\n",
      "12 TCATAA motifs detected in file\n",
      "\n",
      "Complement found from 4894 to 4900\n",
      "Complement found from 6436 to 6442\n",
      "Complement found from 6592 to 6598\n",
      "Complement found from 9367 to 9373\n",
      "Complement found from 9403 to 9409\n",
      "Complement found from 15841 to 15847\n",
      "Complement found from 16274 to 16280\n",
      "Seq is: ATACCA\n",
      "7 ATACCA motifs detected in file\n",
      "\n",
      "Complement found from 3355 to 3361\n",
      "Complement found from 6514 to 6520\n",
      "Complement found from 6964 to 6970\n",
      "Complement found from 9929 to 9935\n",
      "Seq is: TGGCAT\n",
      "4 TGGCAT motifs detected in file\n",
      "\n",
      "Complement found from 4449 to 4455\n",
      "Seq is: GTTATA\n",
      "1 GTTATA motifs detected in file\n",
      "\n",
      "Complement found from 1305 to 1311\n",
      "Complement found from 1780 to 1786\n",
      "Complement found from 2756 to 2762\n",
      "Complement found from 10007 to 10013\n",
      "Seq is: AGTACC\n",
      "4 AGTACC motifs detected in file\n",
      "\n",
      "Complement found from 7915 to 7921\n",
      "Complement found from 8644 to 8650\n",
      "Seq is: ACCGAC\n",
      "2 ACCGAC motifs detected in file\n",
      "\n",
      "Complement found from 4489 to 4495\n",
      "Complement found from 9826 to 9832\n",
      "Seq is: CGTCAT\n",
      "2 CGTCAT motifs detected in file\n",
      "\n",
      "Complement found from 2849 to 2855\n",
      "Complement found from 12623 to 12629\n",
      "Seq is: TACATG\n",
      "2 TACATG motifs detected in file\n",
      "\n",
      "Complement found from 12464 to 12470\n",
      "Complement found from 15015 to 15021\n",
      "Complement found from 15715 to 15721\n",
      "Seq is: CTTTAT\n",
      "3 CTTTAT motifs detected in file\n",
      "\n",
      "Complement found from 12862 to 12868\n",
      "Seq is: GTATCG\n",
      "1 GTATCG motifs detected in file\n",
      "\n",
      "Complement found from 2395 to 2401\n",
      "Complement found from 2426 to 2432\n",
      "Complement found from 6595 to 6601\n",
      "Complement found from 8344 to 8350\n",
      "Complement found from 8660 to 8666\n",
      "Complement found from 9351 to 9357\n",
      "Complement found from 9557 to 9563\n",
      "Complement found from 10713 to 10719\n",
      "Complement found from 10777 to 10783\n",
      "Complement found from 11330 to 11336\n",
      "Complement found from 12826 to 12832\n",
      "Complement found from 13858 to 13864\n",
      "Complement found from 13903 to 13909\n",
      "Complement found from 14185 to 14191\n",
      "Complement found from 14819 to 14825\n",
      "Complement found from 16277 to 16283\n",
      "Seq is: CCAACA\n",
      "16 CCAACA motifs detected in file\n",
      "\n",
      "Complement found from 1701 to 1707\n",
      "Complement found from 2219 to 2225\n",
      "Complement found from 2783 to 2789\n",
      "Complement found from 4733 to 4739\n",
      "Complement found from 5105 to 5111\n",
      "Complement found from 7227 to 7233\n",
      "Complement found from 8380 to 8386\n",
      "Complement found from 8459 to 8465\n",
      "Complement found from 10126 to 10132\n",
      "Complement found from 10791 to 10797\n",
      "Complement found from 10963 to 10969\n",
      "Complement found from 13849 to 13855\n",
      "Seq is: ACTACC\n",
      "12 ACTACC motifs detected in file\n",
      "\n",
      "Complement found from 26 to 32\n",
      "Complement found from 4189 to 4195\n",
      "Complement found from 4518 to 4524\n",
      "Complement found from 5451 to 5457\n",
      "Complement found from 5874 to 5880\n",
      "Complement found from 7718 to 7724\n",
      "Complement found from 8065 to 8071\n",
      "Complement found from 8793 to 8799\n",
      "Complement found from 11256 to 11262\n",
      "Complement found from 11498 to 11504\n",
      "Complement found from 11766 to 11772\n",
      "Complement found from 12011 to 12017\n",
      "Complement found from 13077 to 13083\n",
      "Complement found from 14116 to 14122\n",
      "Complement found from 14399 to 14405\n",
      "Complement found from 14645 to 14651\n",
      "Complement found from 14791 to 14797\n",
      "Seq is: CACTCA\n",
      "17 CACTCA motifs detected in file\n",
      "\n",
      "Complement found from 10319 to 10325\n",
      "Seq is: GTTATG\n",
      "1 GTTATG motifs detected in file\n",
      "\n",
      "Complement found from 912 to 918\n",
      "Complement found from 1686 to 1692\n",
      "Complement found from 2423 to 2429\n",
      "Complement found from 2767 to 2773\n",
      "Complement found from 2831 to 2837\n",
      "Complement found from 4346 to 4352\n",
      "Complement found from 4984 to 4990\n",
      "Complement found from 5430 to 5436\n",
      "Complement found from 5674 to 5680\n",
      "Complement found from 6430 to 6436\n",
      "Complement found from 9240 to 9246\n",
      "Complement found from 12545 to 12551\n",
      "Complement found from 12554 to 12560\n",
      "Complement found from 14069 to 14075\n",
      "Complement found from 14516 to 14522\n",
      "Complement found from 14639 to 14645\n",
      "Complement found from 16164 to 16170\n",
      "Seq is: AACCCA\n",
      "17 AACCCA motifs detected in file\n",
      "\n",
      "Complement found from 3310 to 3316\n",
      "Complement found from 7492 to 7498\n",
      "Complement found from 8833 to 8839\n",
      "Complement found from 16364 to 16370\n",
      "Seq is: CCATGG\n",
      "4 CCATGG motifs detected in file\n",
      "\n",
      "Complement found from 616 to 622\n",
      "Seq is: GTTTAG\n",
      "1 GTTTAG motifs detected in file\n",
      "\n",
      "Seq is: ACGAAG\n",
      "0 ACGAAG motifs detected in file\n",
      "\n",
      "Complement found from 16046 to 16052\n",
      "Seq is: GGGTAC\n",
      "1 GGGTAC motifs detected in file\n",
      "\n",
      "Complement found from 6938 to 6944\n",
      "Complement found from 7174 to 7180\n",
      "Complement found from 9481 to 9487\n",
      "Complement found from 14105 to 14111\n",
      "Seq is: TTTCTT\n",
      "4 TTTCTT motifs detected in file\n",
      "\n",
      "Complement found from 2654 to 2660\n",
      "Complement found from 9714 to 9720\n",
      "Complement found from 12475 to 12481\n",
      "Seq is: GTCTCT\n",
      "3 GTCTCT motifs detected in file\n",
      "\n",
      "Complement found from 2963 to 2969\n",
      "Complement found from 3757 to 3763\n",
      "Complement found from 6406 to 6412\n",
      "Complement found from 6735 to 6741\n",
      "Complement found from 7044 to 7050\n",
      "Complement found from 9067 to 9073\n",
      "Complement found from 9630 to 9636\n",
      "Complement found from 11315 to 11321\n",
      "Seq is: TATCAA\n",
      "8 TATCAA motifs detected in file\n",
      "\n",
      "Complement found from 1518 to 1524\n",
      "Complement found from 2085 to 2091\n",
      "Complement found from 2155 to 2161\n",
      "Complement found from 5079 to 5085\n",
      "Complement found from 16006 to 16012\n",
      "Seq is: ATTTAA\n",
      "5 ATTTAA motifs detected in file\n",
      "\n",
      "Complement found from 677 to 683\n",
      "Complement found from 2034 to 2040\n",
      "Complement found from 5001 to 5007\n",
      "Complement found from 6375 to 6381\n",
      "Complement found from 10224 to 10230\n",
      "Complement found from 10629 to 10635\n",
      "Complement found from 12294 to 12300\n",
      "Complement found from 12724 to 12730\n",
      "Seq is: TCTTAG\n",
      "8 TCTTAG motifs detected in file\n",
      "\n",
      "Complement found from 1212 to 1218\n",
      "Complement found from 4341 to 4347\n",
      "Complement found from 7458 to 7464\n",
      "Complement found from 8002 to 8008\n",
      "Seq is: AATCGA\n",
      "4 AATCGA motifs detected in file\n",
      "\n",
      "Complement found from 912 to 918\n",
      "Complement found from 1686 to 1692\n",
      "Complement found from 2423 to 2429\n",
      "Complement found from 2767 to 2773\n",
      "Complement found from 2831 to 2837\n",
      "Complement found from 4346 to 4352\n",
      "Complement found from 4984 to 4990\n",
      "Complement found from 5430 to 5436\n",
      "Complement found from 5674 to 5680\n",
      "Complement found from 6430 to 6436\n",
      "Complement found from 9240 to 9246\n",
      "Complement found from 12545 to 12551\n",
      "Complement found from 12554 to 12560\n",
      "Complement found from 14069 to 14075\n",
      "Complement found from 14516 to 14522\n",
      "Complement found from 14639 to 14645\n",
      "Complement found from 16164 to 16170\n",
      "Seq is: AACCCA\n",
      "17 AACCCA motifs detected in file\n",
      "\n",
      "Complement found from 907 to 913\n",
      "Complement found from 3045 to 3051\n",
      "Seq is: CGATTA\n",
      "2 CGATTA motifs detected in file\n",
      "\n",
      "Complement found from 1237 to 1243\n",
      "Complement found from 1694 to 1700\n",
      "Complement found from 3625 to 3631\n",
      "Complement found from 4378 to 4384\n",
      "Complement found from 5364 to 5370\n",
      "Complement found from 6552 to 6558\n",
      "Complement found from 8463 to 8469\n",
      "Complement found from 8620 to 8626\n",
      "Complement found from 9026 to 9032\n",
      "Complement found from 9422 to 9428\n",
      "Complement found from 11141 to 11147\n",
      "Complement found from 12460 to 12466\n",
      "Complement found from 14050 to 14056\n",
      "Complement found from 15330 to 15336\n",
      "Complement found from 16146 to 16152\n",
      "Seq is: CCACCT\n",
      "15 CCACCT motifs detected in file\n",
      "\n",
      "Complement found from 879 to 885\n",
      "Complement found from 6388 to 6394\n",
      "Complement found from 9702 to 9708\n",
      "Complement found from 14029 to 14035\n",
      "Seq is: CAATTT\n",
      "4 CAATTT motifs detected in file\n",
      "\n",
      "Complement found from 608 to 614\n",
      "Complement found from 5167 to 5173\n",
      "Complement found from 8174 to 8180\n",
      "Complement found from 15081 to 15087\n",
      "Seq is: CTGAAA\n",
      "4 CTGAAA motifs detected in file\n",
      "\n",
      "Complement found from 3134 to 3140\n",
      "Complement found from 8337 to 8343\n",
      "Seq is: AAGAGA\n",
      "2 AAGAGA motifs detected in file\n",
      "\n",
      "Complement found from 1099 to 1105\n",
      "Complement found from 1671 to 1677\n",
      "Complement found from 2779 to 2785\n",
      "Complement found from 4913 to 4919\n",
      "Complement found from 8119 to 8125\n",
      "Complement found from 9599 to 9605\n",
      "Complement found from 10745 to 10751\n",
      "Complement found from 11278 to 11284\n",
      "Complement found from 12945 to 12951\n",
      "Complement found from 13686 to 13692\n",
      "Complement found from 14582 to 14588\n",
      "Complement found from 14636 to 14642\n",
      "Seq is: CTAAAC\n",
      "12 CTAAAC motifs detected in file\n",
      "\n",
      "Complement found from 746 to 752\n",
      "Complement found from 2105 to 2111\n",
      "Complement found from 2468 to 2474\n",
      "Complement found from 4034 to 4040\n",
      "Complement found from 6268 to 6274\n",
      "Seq is: AGGAAC\n",
      "5 AGGAAC motifs detected in file\n",
      "\n",
      "Complement found from 5245 to 5251\n",
      "Complement found from 7507 to 7513\n",
      "Complement found from 11825 to 11831\n",
      "Complement found from 15938 to 15944\n",
      "Seq is: CTTTTT\n",
      "4 CTTTTT motifs detected in file\n",
      "\n",
      "Complement found from 354 to 360\n",
      "Complement found from 2500 to 2506\n",
      "Complement found from 4363 to 4369\n",
      "Complement found from 4602 to 4608\n",
      "Complement found from 7131 to 7137\n",
      "Complement found from 8521 to 8527\n",
      "Complement found from 9430 to 9436\n",
      "Complement found from 10811 to 10817\n",
      "Complement found from 12152 to 12158\n",
      "Complement found from 12303 to 12309\n",
      "Complement found from 13970 to 13976\n",
      "Complement found from 14072 to 14078\n",
      "Complement found from 14533 to 14539\n",
      "Seq is: CCAAAA\n",
      "13 CCAAAA motifs detected in file\n",
      "\n",
      "Complement found from 1441 to 1447\n",
      "Complement found from 1981 to 1987\n",
      "Complement found from 2139 to 2145\n",
      "Seq is: GTAGAG\n",
      "3 GTAGAG motifs detected in file\n",
      "\n",
      "Complement found from 346 to 352\n",
      "Complement found from 1718 to 1724\n",
      "Complement found from 1887 to 1893\n",
      "Complement found from 7130 to 7136\n",
      "Complement found from 13969 to 13975\n",
      "Complement found from 15152 to 15158\n",
      "Seq is: GCCAAA\n",
      "6 GCCAAA motifs detected in file\n",
      "\n",
      "Complement found from 1204 to 1210\n",
      "Complement found from 4111 to 4117\n",
      "Complement found from 16021 to 16027\n",
      "Seq is: TGTTCT\n",
      "3 TGTTCT motifs detected in file\n",
      "\n",
      "Complement found from 6053 to 6059\n",
      "Complement found from 13512 to 13518\n",
      "Complement found from 14559 to 14565\n",
      "Complement found from 16144 to 16150\n",
      "Complement found from 16397 to 16403\n",
      "Seq is: GACCAC\n",
      "5 GACCAC motifs detected in file\n",
      "\n",
      "Complement found from 2597 to 2603\n",
      "Complement found from 4670 to 4676\n",
      "Complement found from 4757 to 4763\n",
      "Complement found from 5501 to 5507\n",
      "Complement found from 6095 to 6101\n",
      "Complement found from 6122 to 6128\n",
      "Complement found from 6158 to 6164\n",
      "Complement found from 6659 to 6665\n",
      "Complement found from 6830 to 6836\n",
      "Complement found from 7669 to 7675\n",
      "Complement found from 9452 to 9458\n",
      "Complement found from 10079 to 10085\n",
      "Complement found from 10452 to 10458\n",
      "Complement found from 11704 to 11710\n",
      "Complement found from 11782 to 11788\n",
      "Complement found from 14229 to 14235\n",
      "Complement found from 15652 to 15658\n",
      "Seq is: ATAATC\n",
      "17 ATAATC motifs detected in file\n",
      "\n",
      "Complement found from 4264 to 4270\n",
      "Complement found from 8444 to 8450\n",
      "Complement found from 8628 to 8634\n",
      "Complement found from 10486 to 10492\n",
      "Complement found from 11925 to 11931\n",
      "Complement found from 12139 to 12145\n",
      "Complement found from 12690 to 12696\n",
      "Complement found from 15155 to 15161\n",
      "Seq is: AAATAT\n",
      "8 AAATAT motifs detected in file\n",
      "\n",
      "Complement found from 2988 to 2994\n",
      "Complement found from 5902 to 5908\n",
      "Seq is: GATGTT\n",
      "2 GATGTT motifs detected in file\n",
      "\n",
      "Complement found from 5845 to 5851\n",
      "Complement found from 9425 to 9431\n",
      "Complement found from 15884 to 15890\n",
      "Seq is: CCTGTC\n",
      "3 CCTGTC motifs detected in file\n",
      "\n",
      "Seq is: CGGGTA\n",
      "0 CGGGTA motifs detected in file\n",
      "\n",
      "Complement found from 976 to 982\n",
      "Complement found from 1160 to 1166\n",
      "Complement found from 3675 to 3681\n",
      "Complement found from 4156 to 4162\n",
      "Complement found from 6980 to 6986\n",
      "Complement found from 10134 to 10140\n",
      "Complement found from 11749 to 11755\n",
      "Complement found from 12228 to 12234\n",
      "Complement found from 12423 to 12429\n",
      "Complement found from 12660 to 12666\n",
      "Complement found from 12915 to 12921\n",
      "Complement found from 13795 to 13801\n",
      "Seq is: AACTCA\n",
      "12 AACTCA motifs detected in file\n",
      "\n",
      "Complement found from 973 to 979\n",
      "Complement found from 1157 to 1163\n",
      "Complement found from 1525 to 1531\n",
      "Complement found from 2351 to 2357\n",
      "Complement found from 3257 to 3263\n",
      "Complement found from 3264 to 3270\n",
      "Complement found from 3463 to 3469\n",
      "Complement found from 3487 to 3493\n",
      "Complement found from 4779 to 4785\n",
      "Complement found from 6414 to 6420\n",
      "Complement found from 9237 to 9243\n",
      "Complement found from 10763 to 10769\n",
      "Complement found from 11465 to 11471\n",
      "Complement found from 12035 to 12041\n",
      "Complement found from 13792 to 13798\n",
      "Complement found from 14024 to 14030\n",
      "Complement found from 14394 to 14400\n",
      "Seq is: TAAAAC\n",
      "17 TAAAAC motifs detected in file\n",
      "\n",
      "Complement found from 2459 to 2465\n",
      "Complement found from 7342 to 7348\n",
      "Complement found from 12329 to 12335\n",
      "Complement found from 15962 to 15968\n",
      "Seq is: AAAAGT\n",
      "4 AAAAGT motifs detected in file\n",
      "\n",
      "Complement found from 3082 to 3088\n",
      "Complement found from 4471 to 4477\n",
      "Complement found from 4671 to 4677\n",
      "Complement found from 9148 to 9154\n",
      "Complement found from 9453 to 9459\n",
      "Complement found from 11783 to 11789\n",
      "Complement found from 12953 to 12959\n",
      "Complement found from 13150 to 13156\n",
      "Complement found from 15653 to 15659\n",
      "Complement found from 15833 to 15839\n",
      "Seq is: TAATCC\n",
      "10 TAATCC motifs detected in file\n",
      "\n",
      "Complement found from 6370 to 6376\n",
      "Complement found from 8608 to 8614\n",
      "Complement found from 9716 to 9722\n",
      "Complement found from 11046 to 11052\n",
      "Seq is: CTCTAT\n",
      "4 CTCTAT motifs detected in file\n",
      "\n",
      "Complement found from 548 to 554\n",
      "Complement found from 2392 to 2398\n",
      "Complement found from 8806 to 8812\n",
      "Complement found from 8979 to 8985\n",
      "Complement found from 9070 to 9076\n",
      "Complement found from 10838 to 10844\n",
      "Complement found from 11169 to 11175\n",
      "Complement found from 13292 to 13298\n",
      "Complement found from 14200 to 14206\n",
      "Complement found from 14330 to 14336\n",
      "Complement found from 14477 to 14483\n",
      "Complement found from 14690 to 14696\n",
      "Complement found from 15766 to 15772\n",
      "Seq is: CAACCA\n",
      "13 CAACCA motifs detected in file\n",
      "\n",
      "Complement found from 3082 to 3088\n",
      "Complement found from 4471 to 4477\n",
      "Complement found from 4671 to 4677\n",
      "Complement found from 9148 to 9154\n",
      "Complement found from 9453 to 9459\n",
      "Complement found from 11783 to 11789\n",
      "Complement found from 12953 to 12959\n",
      "Complement found from 13150 to 13156\n",
      "Complement found from 15653 to 15659\n",
      "Complement found from 15833 to 15839\n",
      "Seq is: TAATCC\n",
      "10 TAATCC motifs detected in file\n",
      "\n",
      "Complement found from 3082 to 3088\n",
      "Complement found from 4471 to 4477\n",
      "Complement found from 4671 to 4677\n",
      "Complement found from 9148 to 9154\n",
      "Complement found from 9453 to 9459\n",
      "Complement found from 11783 to 11789\n",
      "Complement found from 12953 to 12959\n",
      "Complement found from 13150 to 13156\n",
      "Complement found from 15653 to 15659\n",
      "Complement found from 15833 to 15839\n",
      "Seq is: TAATCC\n",
      "10 TAATCC motifs detected in file\n",
      "\n",
      "Complement found from 238 to 244\n",
      "Complement found from 281 to 287\n",
      "Complement found from 2762 to 2768\n",
      "Complement found from 3299 to 3305\n",
      "Complement found from 5392 to 5398\n",
      "Complement found from 8505 to 8511\n",
      "Complement found from 10940 to 10946\n",
      "Complement found from 11570 to 11576\n",
      "Complement found from 12027 to 12033\n",
      "Complement found from 12246 to 12252\n",
      "Complement found from 12413 to 12419\n",
      "Complement found from 12737 to 12743\n",
      "Complement found from 13385 to 13391\n",
      "Complement found from 13742 to 13748\n",
      "Complement found from 14570 to 14576\n",
      "Complement found from 15600 to 15606\n",
      "Seq is: TAACAA\n",
      "16 TAACAA motifs detected in file\n",
      "\n",
      "Complement found from 4802 to 4808\n",
      "Complement found from 7028 to 7034\n",
      "Complement found from 8904 to 8910\n",
      "Complement found from 9169 to 9175\n",
      "Complement found from 9311 to 9317\n",
      "Complement found from 10512 to 10518\n",
      "Complement found from 13250 to 13256\n",
      "Complement found from 13439 to 13445\n",
      "Complement found from 15289 to 15295\n",
      "Seq is: CACTTC\n",
      "9 CACTTC motifs detected in file\n",
      "\n",
      "Complement found from 2962 to 2968\n",
      "Complement found from 3179 to 3185\n",
      "Complement found from 6734 to 6740\n",
      "Complement found from 9066 to 9072\n",
      "Complement found from 11927 to 11933\n",
      "Complement found from 13536 to 13542\n",
      "Complement found from 15157 to 15163\n",
      "Seq is: ATATCA\n",
      "7 ATATCA motifs detected in file\n",
      "\n",
      "Complement found from 9223 to 9229\n",
      "Complement found from 10562 to 10568\n",
      "Complement found from 10644 to 10650\n",
      "Complement found from 11584 to 11590\n",
      "Complement found from 11740 to 11746\n",
      "Complement found from 12501 to 12507\n",
      "Seq is: TGCCTA\n",
      "6 TGCCTA motifs detected in file\n",
      "\n",
      "Complement found from 11161 to 11167\n",
      "Seq is: CGATGA\n",
      "1 CGATGA motifs detected in file\n",
      "\n",
      "Complement found from 4 to 10\n",
      "Complement found from 2772 to 2778\n",
      "Complement found from 6272 to 6278\n",
      "Complement found from 13494 to 13500\n",
      "Complement found from 13629 to 13635\n",
      "Seq is: ACAGGT\n",
      "5 ACAGGT motifs detected in file\n",
      "\n",
      "Complement found from 470 to 476\n",
      "Complement found from 1700 to 1706\n",
      "Complement found from 3439 to 3445\n",
      "Complement found from 4732 to 4738\n",
      "Complement found from 5104 to 5110\n",
      "Complement found from 5118 to 5124\n",
      "Complement found from 5151 to 5157\n",
      "Complement found from 6525 to 6531\n",
      "Complement found from 6680 to 6686\n",
      "Complement found from 6999 to 7005\n",
      "Complement found from 7013 to 7019\n",
      "Complement found from 8158 to 8164\n",
      "Complement found from 8379 to 8385\n",
      "Complement found from 10101 to 10107\n",
      "Complement found from 10790 to 10796\n",
      "Complement found from 11288 to 11294\n",
      "Complement found from 14211 to 14217\n",
      "Complement found from 15412 to 15418\n",
      "Seq is: TACTAC\n",
      "18 TACTAC motifs detected in file\n",
      "\n",
      "Complement found from 1365 to 1371\n",
      "Complement found from 5882 to 5888\n",
      "Complement found from 7674 to 7680\n",
      "Complement found from 8753 to 8759\n",
      "Complement found from 9792 to 9798\n",
      "Complement found from 9932 to 9938\n",
      "Complement found from 10119 to 10125\n",
      "Complement found from 13891 to 13897\n",
      "Seq is: CATTTT\n",
      "8 CATTTT motifs detected in file\n",
      "\n",
      "Complement found from 499 to 505\n",
      "Complement found from 4348 to 4354\n",
      "Complement found from 4391 to 4397\n",
      "Complement found from 4880 to 4886\n",
      "Complement found from 6116 to 6122\n",
      "Complement found from 7818 to 7824\n",
      "Complement found from 8206 to 8212\n",
      "Complement found from 11428 to 11434\n",
      "Complement found from 12387 to 12393\n",
      "Complement found from 14340 to 14346\n",
      "Complement found from 14814 to 14820\n",
      "Complement found from 15658 to 15664\n",
      "Complement found from 16070 to 16076\n",
      "Seq is: CCCATC\n",
      "13 CCCATC motifs detected in file\n",
      "\n",
      "Complement found from 10426 to 10432\n",
      "Complement found from 15556 to 15562\n",
      "Seq is: GAATGA\n",
      "2 GAATGA motifs detected in file\n",
      "\n",
      "Complement found from 2395 to 2401\n",
      "Complement found from 2426 to 2432\n",
      "Complement found from 6595 to 6601\n",
      "Complement found from 8344 to 8350\n",
      "Complement found from 8660 to 8666\n",
      "Complement found from 9351 to 9357\n",
      "Complement found from 9557 to 9563\n",
      "Complement found from 10713 to 10719\n",
      "Complement found from 10777 to 10783\n",
      "Complement found from 11330 to 11336\n",
      "Complement found from 12826 to 12832\n",
      "Complement found from 13858 to 13864\n",
      "Complement found from 13903 to 13909\n",
      "Complement found from 14185 to 14191\n",
      "Complement found from 14819 to 14825\n",
      "Complement found from 16277 to 16283\n",
      "Seq is: CCAACA\n",
      "16 CCAACA motifs detected in file\n",
      "\n",
      "Complement found from 5222 to 5228\n",
      "Complement found from 6128 to 6134\n",
      "Complement found from 6256 to 6262\n",
      "Complement found from 7073 to 7079\n",
      "Complement found from 15241 to 15247\n",
      "Complement found from 15610 to 15616\n",
      "Seq is: GGAGGC\n",
      "6 GGAGGC motifs detected in file\n",
      "\n",
      "Complement found from 10356 to 10362\n",
      "Seq is: TAAGTC\n",
      "1 TAAGTC motifs detected in file\n",
      "\n",
      "Complement found from 2271 to 2277\n",
      "Complement found from 3721 to 3727\n",
      "Complement found from 4024 to 4030\n",
      "Complement found from 4234 to 4240\n",
      "Complement found from 4941 to 4947\n",
      "Complement found from 5722 to 5728\n",
      "Complement found from 10707 to 10713\n",
      "Complement found from 14162 to 14168\n",
      "Seq is: CAATCT\n",
      "8 CAATCT motifs detected in file\n",
      "\n",
      "Complement found from 3325 to 3331\n",
      "Complement found from 3826 to 3832\n",
      "Complement found from 5009 to 5015\n",
      "Complement found from 5360 to 5366\n",
      "Complement found from 5379 to 5385\n",
      "Complement found from 5472 to 5478\n",
      "Complement found from 6231 to 6237\n",
      "Complement found from 6308 to 6314\n",
      "Complement found from 6683 to 6689\n",
      "Complement found from 8012 to 8018\n",
      "Complement found from 8411 to 8417\n",
      "Complement found from 10752 to 10758\n",
      "Complement found from 11810 to 11816\n",
      "Complement found from 11974 to 11980\n",
      "Complement found from 13072 to 13078\n",
      "Complement found from 13503 to 13509\n",
      "Complement found from 13984 to 13990\n",
      "Complement found from 14131 to 14137\n",
      "Complement found from 14440 to 14446\n",
      "Seq is: TACTCC\n",
      "19 TACTCC motifs detected in file\n",
      "\n",
      "Seq is: TCCAGA\n",
      "0 TCCAGA motifs detected in file\n",
      "\n",
      "Complement found from 5908 to 5914\n",
      "Seq is: CGCCGA\n",
      "1 CGCCGA motifs detected in file\n",
      "\n",
      "Complement found from 592 to 598\n",
      "Complement found from 1495 to 1501\n",
      "Complement found from 2252 to 2258\n",
      "Complement found from 3328 to 3334\n",
      "Complement found from 4647 to 4653\n",
      "Complement found from 4929 to 4935\n",
      "Complement found from 5012 to 5018\n",
      "Complement found from 7805 to 7811\n",
      "Complement found from 8426 to 8432\n",
      "Complement found from 9330 to 9336\n",
      "Complement found from 9846 to 9852\n",
      "Complement found from 11725 to 11731\n",
      "Complement found from 13489 to 13495\n",
      "Complement found from 14443 to 14449\n",
      "Complement found from 15638 to 15644\n",
      "Complement found from 15740 to 15746\n",
      "Seq is: TCCTCA\n",
      "16 TCCTCA motifs detected in file\n",
      "\n",
      "Complement found from 8949 to 8955\n",
      "Complement found from 12516 to 12522\n",
      "Seq is: GTTATT\n",
      "2 GTTATT motifs detected in file\n",
      "\n",
      "Complement found from 4910 to 4916\n",
      "Complement found from 6519 to 6525\n",
      "Complement found from 6986 to 6992\n",
      "Complement found from 7143 to 7149\n",
      "Complement found from 9849 to 9855\n",
      "Complement found from 11275 to 11281\n",
      "Seq is: TCACTA\n",
      "6 TCACTA motifs detected in file\n",
      "\n",
      "Complement found from 3372 to 3378\n",
      "Complement found from 3886 to 3892\n",
      "Complement found from 9669 to 9675\n",
      "Complement found from 10397 to 10403\n",
      "Seq is: ACCGAA\n",
      "4 ACCGAA motifs detected in file\n",
      "\n",
      "Complement found from 13 to 19\n",
      "Complement found from 429 to 435\n",
      "Complement found from 633 to 639\n",
      "Complement found from 720 to 726\n",
      "Complement found from 952 to 958\n",
      "Complement found from 1489 to 1495\n",
      "Complement found from 2278 to 2284\n",
      "Complement found from 3508 to 3514\n",
      "Complement found from 3736 to 3742\n",
      "Complement found from 4192 to 4198\n",
      "Complement found from 5322 to 5328\n",
      "Complement found from 5891 to 5897\n",
      "Complement found from 6619 to 6625\n",
      "Complement found from 8432 to 8438\n",
      "Complement found from 9567 to 9573\n",
      "Complement found from 11156 to 11162\n",
      "Complement found from 12014 to 12020\n",
      "Complement found from 12964 to 12970\n",
      "Complement found from 13621 to 13627\n",
      "Complement found from 14354 to 14360\n",
      "Complement found from 16067 to 16073\n",
      "Complement found from 16262 to 16268\n",
      "Seq is: TCACCC\n",
      "22 TCACCC motifs detected in file\n",
      "\n",
      "Complement found from 1030 to 1036\n",
      "Complement found from 2734 to 2740\n",
      "Seq is: GCTTTA\n",
      "2 GCTTTA motifs detected in file\n",
      "\n",
      "Complement found from 6111 to 6117\n",
      "Complement found from 7737 to 7743\n",
      "Complement found from 10575 to 10581\n",
      "Complement found from 10955 to 10961\n",
      "Complement found from 11486 to 11492\n",
      "Complement found from 15839 to 15845\n",
      "Complement found from 15904 to 15910\n",
      "Seq is: TAATAC\n",
      "7 TAATAC motifs detected in file\n",
      "\n",
      "Complement found from 810 to 816\n",
      "Complement found from 5169 to 5175\n",
      "Complement found from 7251 to 7257\n",
      "Complement found from 14654 to 14660\n",
      "Complement found from 15083 to 15089\n",
      "Seq is: GAAACA\n",
      "5 GAAACA motifs detected in file\n",
      "\n",
      "Complement found from 1990 to 1996\n",
      "Complement found from 2764 to 2770\n",
      "Complement found from 3990 to 3996\n",
      "Complement found from 5679 to 5685\n",
      "Complement found from 5930 to 5936\n",
      "Complement found from 8455 to 8461\n",
      "Complement found from 8507 to 8513\n",
      "Complement found from 10294 to 10300\n",
      "Complement found from 11592 to 11598\n",
      "Complement found from 13545 to 13551\n",
      "Complement found from 13861 to 13867\n",
      "Complement found from 14188 to 14194\n",
      "Complement found from 14623 to 14629\n",
      "Complement found from 15184 to 15190\n",
      "Complement found from 15602 to 15608\n",
      "Complement found from 16280 to 16286\n",
      "Seq is: ACAAAC\n",
      "16 ACAAAC motifs detected in file\n",
      "\n",
      "Complement found from 1253 to 1259\n",
      "Complement found from 3392 to 3398\n",
      "Complement found from 7385 to 7391\n",
      "Complement found from 7573 to 7579\n",
      "Complement found from 10185 to 10191\n",
      "Complement found from 15049 to 15055\n",
      "Seq is: CTATAT\n",
      "6 CTATAT motifs detected in file\n",
      "\n",
      "Complement found from 589 to 595\n",
      "Complement found from 2837 to 2843\n",
      "Complement found from 3319 to 3325\n",
      "Complement found from 4104 to 4110\n",
      "Complement found from 4165 to 4171\n",
      "Complement found from 6217 to 6223\n",
      "Complement found from 6818 to 6824\n",
      "Complement found from 8469 to 8475\n",
      "Complement found from 8622 to 8628\n",
      "Complement found from 8773 to 8779\n",
      "Complement found from 9287 to 9293\n",
      "Complement found from 13446 to 13452\n",
      "Complement found from 13581 to 13587\n",
      "Complement found from 14052 to 14058\n",
      "Complement found from 14374 to 14380\n",
      "Complement found from 14526 to 14532\n",
      "Complement found from 14804 to 14810\n",
      "Complement found from 15332 to 15338\n",
      "Complement found from 15379 to 15385\n",
      "Complement found from 15488 to 15494\n",
      "Complement found from 15737 to 15743\n",
      "Seq is: ACCTCC\n",
      "21 ACCTCC motifs detected in file\n",
      "\n",
      "Complement found from 538 to 544\n",
      "Complement found from 1080 to 1086\n",
      "Complement found from 1372 to 1378\n",
      "Complement found from 2485 to 2491\n",
      "Complement found from 4134 to 4140\n",
      "Complement found from 4433 to 4439\n",
      "Complement found from 7229 to 7235\n",
      "Complement found from 8278 to 8284\n",
      "Complement found from 8403 to 8409\n",
      "Complement found from 9529 to 9535\n",
      "Complement found from 10275 to 10281\n",
      "Complement found from 10976 to 10982\n",
      "Complement found from 11533 to 11539\n",
      "Complement found from 11864 to 11870\n",
      "Complement found from 12429 to 12435\n",
      "Seq is: TACCCC\n",
      "15 TACCCC motifs detected in file\n",
      "\n",
      "Complement found from 232 to 238\n",
      "Complement found from 3769 to 3775\n",
      "Complement found from 4003 to 4009\n",
      "Complement found from 4755 to 4761\n",
      "Complement found from 5499 to 5505\n",
      "Complement found from 6093 to 6099\n",
      "Complement found from 6156 to 6162\n",
      "Complement found from 7302 to 7308\n",
      "Complement found from 8039 to 8045\n",
      "Complement found from 10056 to 10062\n",
      "Complement found from 10077 to 10083\n",
      "Complement found from 10107 to 10113\n",
      "Complement found from 12334 to 12340\n",
      "Complement found from 14546 to 14552\n",
      "Complement found from 14774 to 14780\n",
      "Seq is: TAATAA\n",
      "15 TAATAA motifs detected in file\n",
      "\n",
      "Complement found from 7857 to 7863\n",
      "Complement found from 15589 to 15595\n",
      "Seq is: CGATCC\n",
      "2 CGATCC motifs detected in file\n",
      "\n",
      "Complement found from 3325 to 3331\n",
      "Complement found from 3826 to 3832\n",
      "Complement found from 5009 to 5015\n",
      "Complement found from 5360 to 5366\n",
      "Complement found from 5379 to 5385\n",
      "Complement found from 5472 to 5478\n",
      "Complement found from 6231 to 6237\n",
      "Complement found from 6308 to 6314\n",
      "Complement found from 6683 to 6689\n",
      "Complement found from 8012 to 8018\n",
      "Complement found from 8411 to 8417\n",
      "Complement found from 10752 to 10758\n",
      "Complement found from 11810 to 11816\n",
      "Complement found from 11974 to 11980\n",
      "Complement found from 13072 to 13078\n",
      "Complement found from 13503 to 13509\n",
      "Complement found from 13984 to 13990\n",
      "Complement found from 14131 to 14137\n",
      "Complement found from 14440 to 14446\n",
      "Seq is: TACTCC\n",
      "19 TACTCC motifs detected in file\n",
      "\n",
      "Complement found from 1919 to 1925\n",
      "Complement found from 2221 to 2227\n",
      "Complement found from 2758 to 2764\n",
      "Complement found from 5345 to 5351\n",
      "Complement found from 5478 to 5484\n",
      "Complement found from 5957 to 5963\n",
      "Complement found from 13788 to 13794\n",
      "Complement found from 13851 to 13857\n",
      "Complement found from 14020 to 14026\n",
      "Seq is: TACCTA\n",
      "9 TACCTA motifs detected in file\n",
      "\n",
      "Complement found from 2403 to 2409\n",
      "Complement found from 4751 to 4757\n",
      "Complement found from 9828 to 9834\n",
      "Complement found from 10439 to 10445\n",
      "Complement found from 10590 to 10596\n",
      "Complement found from 11728 to 11734\n",
      "Complement found from 12116 to 12122\n",
      "Complement found from 13735 to 13741\n",
      "Complement found from 14669 to 14675\n",
      "Complement found from 15305 to 15311\n",
      "Seq is: TCATTA\n",
      "10 TCATTA motifs detected in file\n",
      "\n",
      "Complement found from 4459 to 4465\n",
      "Complement found from 15143 to 15149\n",
      "Seq is: TCCCGT\n",
      "2 TCCCGT motifs detected in file\n",
      "\n",
      "Complement found from 138 to 144\n",
      "Complement found from 3829 to 3835\n",
      "Complement found from 6234 to 6240\n",
      "Complement found from 7784 to 7790\n",
      "Complement found from 8785 to 8791\n",
      "Complement found from 13312 to 13318\n",
      "Complement found from 15101 to 15107\n",
      "Seq is: TCCTGC\n",
      "7 TCCTGC motifs detected in file\n",
      "\n",
      "Complement found from 6572 to 6578\n",
      "Complement found from 13413 to 13419\n",
      "Complement found from 15760 to 15766\n",
      "Seq is: GGAGGA\n",
      "3 GGAGGA motifs detected in file\n",
      "\n",
      "Complement found from 3672 to 3678\n",
      "Complement found from 3678 to 3684\n",
      "Complement found from 4253 to 4259\n",
      "Complement found from 8674 to 8680\n",
      "Complement found from 11317 to 11323\n",
      "Complement found from 11752 to 11758\n",
      "Complement found from 11802 to 11808\n",
      "Complement found from 12574 to 12580\n",
      "Complement found from 15358 to 15364\n",
      "Seq is: TCAAAC\n",
      "9 TCAAAC motifs detected in file\n",
      "\n",
      "Complement found from 2072 to 2078\n",
      "Complement found from 2230 to 2236\n",
      "Complement found from 4472 to 4478\n",
      "Complement found from 9578 to 9584\n",
      "Complement found from 13778 to 13784\n",
      "Complement found from 13876 to 13882\n",
      "Complement found from 13943 to 13949\n",
      "Complement found from 15654 to 15660\n",
      "Complement found from 16349 to 16355\n",
      "Seq is: AATCCC\n",
      "9 AATCCC motifs detected in file\n",
      "\n",
      "Complement found from 9223 to 9229\n",
      "Complement found from 10562 to 10568\n",
      "Complement found from 10644 to 10650\n",
      "Complement found from 11584 to 11590\n",
      "Complement found from 11740 to 11746\n",
      "Complement found from 12501 to 12507\n",
      "Seq is: TGCCTA\n",
      "6 TGCCTA motifs detected in file\n",
      "\n",
      "Complement found from 9181 to 9187\n",
      "Complement found from 15023 to 15029\n",
      "Seq is: GCCTCT\n",
      "2 GCCTCT motifs detected in file\n",
      "\n",
      "Complement found from 371 to 377\n",
      "Complement found from 441 to 447\n",
      "Complement found from 1613 to 1619\n",
      "Complement found from 2158 to 2164\n",
      "Complement found from 2704 to 2710\n",
      "Complement found from 5187 to 5193\n",
      "Complement found from 7715 to 7721\n",
      "Complement found from 9387 to 9393\n",
      "Complement found from 13162 to 13168\n",
      "Complement found from 14549 to 14555\n",
      "Complement found from 15470 to 15476\n",
      "Complement found from 16461 to 16467\n",
      "Seq is: TAACAC\n",
      "12 TAACAC motifs detected in file\n",
      "\n",
      "Complement found from 9863 to 9869\n",
      "Complement found from 13116 to 13122\n",
      "Complement found from 14980 to 14986\n",
      "Complement found from 15195 to 15201\n",
      "Seq is: ATCCGC\n",
      "4 ATCCGC motifs detected in file\n",
      "\n",
      "Complement found from 4946 to 4952\n",
      "Complement found from 5094 to 5100\n",
      "Complement found from 6639 to 6645\n",
      "Complement found from 8935 to 8941\n",
      "Complement found from 11008 to 11014\n",
      "Complement found from 11135 to 11141\n",
      "Complement found from 12901 to 12907\n",
      "Complement found from 15095 to 15101\n",
      "Seq is: TTATCC\n",
      "8 TTATCC motifs detected in file\n",
      "\n",
      "Complement found from 1540 to 1546\n",
      "Complement found from 1974 to 1980\n",
      "Complement found from 5088 to 5094\n",
      "Complement found from 5492 to 5498\n",
      "Complement found from 6630 to 6636\n",
      "Complement found from 11111 to 11117\n",
      "Complement found from 11365 to 11371\n",
      "Seq is: TTTATA\n",
      "7 TTTATA motifs detected in file\n",
      "\n",
      "Complement found from 1296 to 1302\n",
      "Complement found from 4919 to 4925\n",
      "Complement found from 9177 to 9183\n",
      "Complement found from 15772 to 15778\n",
      "Seq is: GTAAGC\n",
      "4 GTAAGC motifs detected in file\n",
      "\n",
      "Complement found from 542 to 548\n",
      "Complement found from 3373 to 3379\n",
      "Complement found from 3887 to 3893\n",
      "Complement found from 3919 to 3925\n",
      "Seq is: CCGAAC\n",
      "4 CCGAAC motifs detected in file\n",
      "\n",
      "Complement found from 4162 to 4168\n",
      "Complement found from 6351 to 6357\n",
      "Complement found from 7109 to 7115\n",
      "Complement found from 7241 to 7247\n",
      "Complement found from 7912 to 7918\n",
      "Complement found from 8801 to 8807\n",
      "Complement found from 8927 to 8933\n",
      "Complement found from 11212 to 11218\n",
      "Complement found from 12074 to 12080\n",
      "Complement found from 14181 to 14187\n",
      "Complement found from 15907 to 15913\n",
      "Seq is: TACACC\n",
      "11 TACACC motifs detected in file\n",
      "\n",
      "Complement found from 307 to 313\n",
      "Complement found from 457 to 463\n",
      "Complement found from 958 to 964\n",
      "Complement found from 3115 to 3121\n",
      "Complement found from 3569 to 3575\n",
      "Complement found from 4105 to 4111\n",
      "Complement found from 5443 to 5449\n",
      "Complement found from 6218 to 6224\n",
      "Complement found from 6290 to 6296\n",
      "Complement found from 7815 to 7821\n",
      "Complement found from 7862 to 7868\n",
      "Complement found from 8470 to 8476\n",
      "Complement found from 10551 to 10557\n",
      "Complement found from 13447 to 13453\n",
      "Complement found from 13582 to 13588\n",
      "Complement found from 14261 to 14267\n",
      "Complement found from 14527 to 14533\n",
      "Complement found from 14805 to 14811\n",
      "Complement found from 15141 to 15147\n",
      "Complement found from 15380 to 15386\n",
      "Complement found from 15538 to 15544\n",
      "Complement found from 16186 to 16192\n",
      "Seq is: CCTCCC\n",
      "22 CCTCCC motifs detected in file\n",
      "\n",
      "Complement found from 1008 to 1014\n",
      "Complement found from 1616 to 1622\n",
      "Complement found from 3154 to 3160\n",
      "Complement found from 3989 to 3995\n",
      "Complement found from 5278 to 5284\n",
      "Complement found from 5678 to 5684\n",
      "Complement found from 5936 to 5942\n",
      "Complement found from 8454 to 8460\n",
      "Complement found from 13216 to 13222\n",
      "Complement found from 13544 to 13550\n",
      "Complement found from 14622 to 14628\n",
      "Seq is: CACAAA\n",
      "11 CACAAA motifs detected in file\n",
      "\n",
      "Complement found from 453 to 459\n",
      "Complement found from 3165 to 3171\n",
      "Complement found from 4246 to 4252\n",
      "Complement found from 6150 to 6156\n",
      "Complement found from 6183 to 6189\n",
      "Complement found from 7092 to 7098\n",
      "Complement found from 7623 to 7629\n",
      "Complement found from 7952 to 7958\n",
      "Complement found from 8225 to 8231\n",
      "Complement found from 8602 to 8608\n",
      "Complement found from 10914 to 10920\n",
      "Complement found from 11231 to 11237\n",
      "Complement found from 12382 to 12388\n",
      "Complement found from 12480 to 12486\n",
      "Complement found from 13644 to 13650\n",
      "Complement found from 13751 to 13757\n",
      "Complement found from 14152 to 14158\n",
      "Complement found from 14486 to 14492\n",
      "Complement found from 16543 to 16549\n",
      "Seq is: TTCCCC\n",
      "19 TTCCCC motifs detected in file\n",
      "\n",
      "Complement found from 3885 to 3891\n",
      "Complement found from 8643 to 8649\n",
      "Complement found from 9668 to 9674\n",
      "Complement found from 10396 to 10402\n",
      "Seq is: AACCGA\n",
      "4 AACCGA motifs detected in file\n",
      "\n",
      "Complement found from 465 to 471\n",
      "Complement found from 6665 to 6671\n",
      "Complement found from 7817 to 7823\n",
      "Complement found from 15203 to 15209\n",
      "Complement found from 15382 to 15388\n",
      "Seq is: TCCCAT\n",
      "5 TCCCAT motifs detected in file\n",
      "\n",
      "Complement found from 20 to 26\n",
      "Complement found from 822 to 828\n",
      "Complement found from 909 to 915\n",
      "Complement found from 1859 to 1865\n",
      "Complement found from 2369 to 2375\n",
      "Complement found from 8314 to 8320\n",
      "Complement found from 9075 to 9081\n",
      "Complement found from 10024 to 10030\n",
      "Complement found from 11875 to 11881\n",
      "Complement found from 12025 to 12031\n",
      "Complement found from 13662 to 13668\n",
      "Complement found from 14785 to 14791\n",
      "Complement found from 15468 to 15474\n",
      "Seq is: ATTAAC\n",
      "13 ATTAAC motifs detected in file\n",
      "\n",
      "Complement found from 1060 to 1066\n",
      "Complement found from 1897 to 1903\n",
      "Complement found from 2722 to 2728\n",
      "Complement found from 4090 to 4096\n",
      "Complement found from 6580 to 6586\n",
      "Complement found from 12665 to 12671\n",
      "Complement found from 12923 to 12929\n",
      "Seq is: AGACCC\n",
      "7 AGACCC motifs detected in file\n",
      "\n",
      "Seq is: AGCGGA\n",
      "0 AGCGGA motifs detected in file\n",
      "\n",
      "Complement found from 2326 to 2332\n",
      "Complement found from 3114 to 3120\n",
      "Complement found from 5442 to 5448\n",
      "Complement found from 10550 to 10556\n",
      "Complement found from 10927 to 10933\n",
      "Complement found from 12092 to 12098\n",
      "Complement found from 12982 to 12988\n",
      "Complement found from 13987 to 13993\n",
      "Complement found from 14260 to 14266\n",
      "Complement found from 14870 to 14876\n",
      "Complement found from 15098 to 15104\n",
      "Complement found from 15140 to 15146\n",
      "Complement found from 15662 to 15668\n",
      "Complement found from 16405 to 16411\n",
      "Seq is: TCCTCC\n",
      "14 TCCTCC motifs detected in file\n",
      "\n",
      "Complement found from 3880 to 3886\n",
      "Complement found from 5530 to 5536\n",
      "Complement found from 5658 to 5664\n",
      "Complement found from 7117 to 7123\n",
      "Complement found from 12506 to 12512\n",
      "Complement found from 13511 to 13517\n",
      "Seq is: AGACCA\n",
      "6 AGACCA motifs detected in file\n",
      "\n",
      "Complement found from 676 to 682\n",
      "Complement found from 2656 to 2662\n",
      "Complement found from 6212 to 6218\n",
      "Complement found from 8731 to 8737\n",
      "Complement found from 10331 to 10337\n",
      "Complement found from 10628 to 10634\n",
      "Complement found from 11461 to 11467\n",
      "Seq is: CTCTTA\n",
      "7 CTCTTA motifs detected in file\n",
      "\n",
      "Complement found from 3153 to 3159\n",
      "Complement found from 3802 to 3808\n",
      "Complement found from 5277 to 5283\n",
      "Complement found from 6396 to 6402\n",
      "Complement found from 7721 to 7727\n",
      "Complement found from 8050 to 8056\n",
      "Complement found from 9103 to 9109\n",
      "Complement found from 10982 to 10988\n",
      "Complement found from 11259 to 11265\n",
      "Complement found from 12214 to 12220\n",
      "Complement found from 15821 to 15827\n",
      "Seq is: TCACAA\n",
      "11 TCACAA motifs detected in file\n",
      "\n",
      "Complement found from 178 to 184\n",
      "Complement found from 3271 to 3277\n",
      "Complement found from 5858 to 5864\n",
      "Complement found from 6780 to 6786\n",
      "Complement found from 8355 to 8361\n",
      "Complement found from 11942 to 11948\n",
      "Complement found from 16340 to 16346\n",
      "Seq is: TTACAG\n",
      "7 TTACAG motifs detected in file\n",
      "\n",
      "Complement found from 5196 to 5202\n",
      "Complement found from 8222 to 8228\n",
      "Complement found from 9112 to 9118\n",
      "Complement found from 9681 to 9687\n",
      "Complement found from 12379 to 12385\n",
      "Complement found from 13612 to 13618\n",
      "Seq is: TAATTC\n",
      "6 TAATTC motifs detected in file\n",
      "\n",
      "Complement found from 4935 to 4941\n",
      "Complement found from 6223 to 6229\n",
      "Complement found from 11044 to 11050\n",
      "Complement found from 11788 to 11794\n",
      "Complement found from 11893 to 11899\n",
      "Complement found from 14100 to 14106\n",
      "Seq is: CTCTCT\n",
      "6 CTCTCT motifs detected in file\n",
      "\n",
      "Complement found from 4674 to 4680\n",
      "Complement found from 12786 to 12792\n",
      "Complement found from 14286 to 14292\n",
      "Complement found from 15449 to 15455\n",
      "Seq is: TCCTTC\n",
      "4 TCCTTC motifs detected in file\n",
      "\n",
      "Complement found from 1266 to 1272\n",
      "Complement found from 3469 to 3475\n",
      "Complement found from 3973 to 3979\n",
      "Complement found from 4692 to 4698\n",
      "Complement found from 6102 to 6108\n",
      "Complement found from 7934 to 7940\n",
      "Complement found from 9100 to 9106\n",
      "Complement found from 11623 to 11629\n",
      "Complement found from 12685 to 12691\n",
      "Complement found from 13336 to 13342\n",
      "Seq is: TCTTCA\n",
      "10 TCTTCA motifs detected in file\n",
      "\n",
      "Complement found from 5966 to 5972\n",
      "Complement found from 14842 to 14848\n",
      "Seq is: TTCGGC\n",
      "2 TTCGGC motifs detected in file\n",
      "\n",
      "Complement found from 5733 to 5739\n",
      "Complement found from 16423 to 16429\n",
      "Seq is: TCCCGC\n",
      "2 TCCCGC motifs detected in file\n",
      "\n",
      "Complement found from 7767 to 7773\n",
      "Complement found from 14653 to 14659\n",
      "Complement found from 15075 to 15081\n",
      "Seq is: AGAAAC\n",
      "3 AGAAAC motifs detected in file\n",
      "\n",
      "Complement found from 371 to 377\n",
      "Complement found from 441 to 447\n",
      "Complement found from 1613 to 1619\n",
      "Complement found from 2158 to 2164\n",
      "Complement found from 2704 to 2710\n",
      "Complement found from 5187 to 5193\n",
      "Complement found from 7715 to 7721\n",
      "Complement found from 9387 to 9393\n",
      "Complement found from 13162 to 13168\n",
      "Complement found from 14549 to 14555\n",
      "Complement found from 15470 to 15476\n",
      "Complement found from 16461 to 16467\n",
      "Seq is: TAACAC\n",
      "12 TAACAC motifs detected in file\n",
      "\n",
      "Complement found from 3747 to 3753\n",
      "Complement found from 4750 to 4756\n",
      "Complement found from 7672 to 7678\n",
      "Complement found from 8592 to 8598\n",
      "Complement found from 8751 to 8757\n",
      "Complement found from 12115 to 12121\n",
      "Complement found from 14482 to 14488\n",
      "Complement found from 14668 to 14674\n",
      "Complement found from 15060 to 15066\n",
      "Complement found from 15159 to 15165\n",
      "Complement found from 15790 to 15796\n",
      "Seq is: ATCATT\n",
      "11 ATCATT motifs detected in file\n",
      "\n",
      "Complement found from 3209 to 3215\n",
      "Complement found from 5303 to 5309\n",
      "Complement found from 5432 to 5438\n",
      "Complement found from 6312 to 6318\n",
      "Complement found from 6845 to 6851\n",
      "Complement found from 7399 to 7405\n",
      "Complement found from 8392 to 8398\n",
      "Complement found from 8619 to 8625\n",
      "Complement found from 9210 to 9216\n",
      "Complement found from 11140 to 11146\n",
      "Complement found from 12017 to 12023\n",
      "Complement found from 13053 to 13059\n",
      "Complement found from 13647 to 13653\n",
      "Complement found from 13677 to 13683\n",
      "Complement found from 14809 to 14815\n",
      "Complement found from 15262 to 15268\n",
      "Complement found from 16289 to 16295\n",
      "Seq is: CCCACC\n",
      "17 CCCACC motifs detected in file\n",
      "\n",
      "Complement found from 265 to 271\n",
      "Complement found from 292 to 298\n",
      "Complement found from 4621 to 4627\n",
      "Complement found from 7031 to 7037\n",
      "Complement found from 9314 to 9320\n",
      "Complement found from 9812 to 9818\n",
      "Complement found from 13122 to 13128\n",
      "Complement found from 15403 to 15409\n",
      "Seq is: TTCCAC\n",
      "8 TTCCAC motifs detected in file\n",
      "\n",
      "Complement found from 5406 to 5412\n",
      "Complement found from 7434 to 7440\n",
      "Complement found from 10215 to 10221\n",
      "Complement found from 11603 to 11609\n",
      "Complement found from 12442 to 12448\n",
      "Complement found from 13868 to 13874\n",
      "Complement found from 14777 to 14783\n",
      "Complement found from 15393 to 15399\n",
      "Seq is: TAAAAT\n",
      "8 TAAAAT motifs detected in file\n",
      "\n",
      "Complement found from 662 to 668\n",
      "Complement found from 4922 to 4928\n",
      "Complement found from 5538 to 5544\n",
      "Complement found from 7324 to 7330\n",
      "Complement found from 9502 to 9508\n",
      "Complement found from 10096 to 10102\n",
      "Complement found from 13241 to 13247\n",
      "Complement found from 15123 to 15129\n",
      "Seq is: AGCCTT\n",
      "8 AGCCTT motifs detected in file\n",
      "\n",
      "Complement found from 722 to 728\n",
      "Complement found from 1491 to 1497\n",
      "Complement found from 2064 to 2070\n",
      "Complement found from 2411 to 2417\n",
      "Complement found from 3510 to 3516\n",
      "Complement found from 4011 to 4017\n",
      "Complement found from 4614 to 4620\n",
      "Complement found from 5207 to 5213\n",
      "Complement found from 5324 to 5330\n",
      "Complement found from 6288 to 6294\n",
      "Complement found from 7363 to 7369\n",
      "Complement found from 9725 to 9731\n",
      "Complement found from 10088 to 10094\n",
      "Complement found from 10610 to 10616\n",
      "Complement found from 12039 to 12045\n",
      "Complement found from 12061 to 12067\n",
      "Complement found from 12399 to 12405\n",
      "Complement found from 15265 to 15271\n",
      "Complement found from 16219 to 16225\n",
      "Seq is: ACCCTC\n",
      "19 ACCCTC motifs detected in file\n",
      "\n",
      "Complement found from 4911 to 4917\n",
      "Complement found from 8711 to 8717\n",
      "Complement found from 9357 to 9363\n",
      "Complement found from 10311 to 10317\n",
      "Complement found from 11246 to 11252\n",
      "Complement found from 11276 to 11282\n",
      "Complement found from 11816 to 11822\n",
      "Complement found from 13147 to 13153\n",
      "Complement found from 14391 to 14397\n",
      "Complement found from 15701 to 15707\n",
      "Seq is: CACTAA\n",
      "10 CACTAA motifs detected in file\n",
      "\n",
      "Complement found from 14257 to 14263\n",
      "Seq is: GGATCC\n",
      "1 GGATCC motifs detected in file\n",
      "\n",
      "Complement found from 435 to 441\n",
      "Complement found from 1625 to 1631\n",
      "Complement found from 2425 to 2431\n",
      "Complement found from 2833 to 2839\n",
      "Complement found from 3414 to 3420\n",
      "Complement found from 3579 to 3585\n",
      "Complement found from 4482 to 4488\n",
      "Complement found from 8369 to 8375\n",
      "Complement found from 8435 to 8441\n",
      "Complement found from 8659 to 8665\n",
      "Complement found from 8812 to 8818\n",
      "Complement found from 9556 to 9562\n",
      "Complement found from 10776 to 10782\n",
      "Complement found from 10917 to 10923\n",
      "Seq is: CCCAAC\n",
      "14 CCCAAC motifs detected in file\n",
      "\n",
      "Complement found from 1381 to 1387\n",
      "Complement found from 1432 to 1438\n",
      "Complement found from 2781 to 2787\n",
      "Complement found from 3680 to 3686\n",
      "Complement found from 4781 to 4787\n",
      "Complement found from 4871 to 4877\n",
      "Complement found from 7729 to 7735\n",
      "Complement found from 8457 to 8463\n",
      "Complement found from 8676 to 8682\n",
      "Complement found from 10765 to 10771\n",
      "Complement found from 11467 to 11473\n",
      "Complement found from 11754 to 11760\n",
      "Complement found from 12576 to 12582\n",
      "Complement found from 14507 to 14513\n",
      "Complement found from 14762 to 14768\n",
      "Complement found from 15604 to 15610\n",
      "Complement found from 15900 to 15906\n",
      "Complement found from 16010 to 16016\n",
      "Seq is: AAACTA\n",
      "18 AAACTA motifs detected in file\n",
      "\n",
      "Seq is: GGGCAA\n",
      "0 GGGCAA motifs detected in file\n",
      "\n",
      "Complement found from 760 to 766\n",
      "Seq is: AGCACG\n",
      "1 AGCACG motifs detected in file\n",
      "\n",
      "Complement found from 3295 to 3301\n",
      "Complement found from 4113 to 4119\n",
      "Complement found from 5073 to 5079\n",
      "Complement found from 6636 to 6642\n",
      "Complement found from 8907 to 8913\n",
      "Complement found from 10223 to 10229\n",
      "Complement found from 10241 to 10247\n",
      "Complement found from 13107 to 13113\n",
      "Complement found from 13960 to 13966\n",
      "Seq is: TTCTTA\n",
      "9 TTCTTA motifs detected in file\n",
      "\n",
      "Complement found from 2326 to 2332\n",
      "Complement found from 3114 to 3120\n",
      "Complement found from 5442 to 5448\n",
      "Complement found from 10550 to 10556\n",
      "Complement found from 10927 to 10933\n",
      "Complement found from 12092 to 12098\n",
      "Complement found from 12982 to 12988\n",
      "Complement found from 13987 to 13993\n",
      "Complement found from 14260 to 14266\n",
      "Complement found from 14870 to 14876\n",
      "Complement found from 15098 to 15104\n",
      "Complement found from 15140 to 15146\n",
      "Complement found from 15662 to 15668\n",
      "Complement found from 16405 to 16411\n",
      "Seq is: TCCTCC\n",
      "14 TCCTCC motifs detected in file\n",
      "\n",
      "Complement found from 1103 to 1109\n",
      "Complement found from 1231 to 1237\n",
      "Complement found from 3595 to 3601\n",
      "Complement found from 5366 to 5372\n",
      "Complement found from 5888 to 5894\n",
      "Complement found from 6543 to 6549\n",
      "Complement found from 8682 to 8688\n",
      "Complement found from 9467 to 9473\n",
      "Complement found from 10542 to 10548\n",
      "Complement found from 13843 to 13849\n",
      "Complement found from 14064 to 14070\n",
      "Complement found from 14410 to 14416\n",
      "Seq is: ACCTCA\n",
      "12 ACCTCA motifs detected in file\n",
      "\n",
      "Complement found from 3042 to 3048\n",
      "Complement found from 7854 to 7860\n",
      "Seq is: CAACGA\n",
      "2 CAACGA motifs detected in file\n",
      "\n",
      "Complement found from 3005 to 3011\n",
      "Complement found from 8015 to 8021\n",
      "Complement found from 14263 to 14269\n",
      "Seq is: TCCCGA\n",
      "3 TCCCGA motifs detected in file\n",
      "\n",
      "Complement found from 6067 to 6073\n",
      "Seq is: CGTTAT\n",
      "1 CGTTAT motifs detected in file\n",
      "\n",
      "Complement found from 1505 to 1511\n",
      "Complement found from 3149 to 3155\n",
      "Complement found from 4096 to 4102\n",
      "Complement found from 5339 to 5345\n",
      "Complement found from 5727 to 5733\n",
      "Complement found from 6483 to 6489\n",
      "Complement found from 7620 to 7626\n",
      "Complement found from 7949 to 7955\n",
      "Complement found from 9746 to 9752\n",
      "Complement found from 11200 to 11206\n",
      "Complement found from 12585 to 12591\n",
      "Complement found from 14094 to 14100\n",
      "Complement found from 15440 to 15446\n",
      "Complement found from 15817 to 15823\n",
      "Complement found from 16508 to 16514\n",
      "Seq is: TACTTC\n",
      "15 TACTTC motifs detected in file\n",
      "\n",
      "Complement found from 706 to 712\n",
      "Seq is: CCGTTC\n",
      "1 CCGTTC motifs detected in file\n",
      "\n",
      "Complement found from 1505 to 1511\n",
      "Complement found from 3149 to 3155\n",
      "Complement found from 4096 to 4102\n",
      "Complement found from 5339 to 5345\n",
      "Complement found from 5727 to 5733\n",
      "Complement found from 6483 to 6489\n",
      "Complement found from 7620 to 7626\n",
      "Complement found from 7949 to 7955\n",
      "Complement found from 9746 to 9752\n",
      "Complement found from 11200 to 11206\n",
      "Complement found from 12585 to 12591\n",
      "Complement found from 14094 to 14100\n",
      "Complement found from 15440 to 15446\n",
      "Complement found from 15817 to 15823\n",
      "Complement found from 16508 to 16514\n",
      "Seq is: TACTTC\n",
      "15 TACTTC motifs detected in file\n",
      "\n",
      "Complement found from 262 to 268\n",
      "Complement found from 7172 to 7178\n",
      "Complement found from 7188 to 7194\n",
      "Complement found from 8131 to 8137\n",
      "Complement found from 9842 to 9848\n",
      "Complement found from 10806 to 10812\n",
      "Complement found from 13815 to 13821\n",
      "Seq is: ACTTTC\n",
      "7 ACTTTC motifs detected in file\n",
      "\n",
      "Complement found from 7139 to 7145\n",
      "Complement found from 7279 to 7285\n",
      "Complement found from 7533 to 7539\n",
      "Complement found from 9766 to 9772\n",
      "Complement found from 13748 to 13754\n",
      "Complement found from 15062 to 15068\n",
      "Seq is: CATTTC\n",
      "6 CATTTC motifs detected in file\n",
      "\n",
      "Complement found from 830 to 836\n",
      "Complement found from 1424 to 1430\n",
      "Complement found from 5003 to 5009\n",
      "Complement found from 6296 to 6302\n",
      "Complement found from 6974 to 6980\n",
      "Complement found from 8309 to 8315\n",
      "Complement found from 10859 to 10865\n",
      "Complement found from 12891 to 12897\n",
      "Complement found from 13473 to 13479\n",
      "Complement found from 15982 to 15988\n",
      "Seq is: TTAGCA\n",
      "10 TTAGCA motifs detected in file\n",
      "\n",
      "Complement found from 462 to 468\n",
      "Complement found from 1690 to 1696\n",
      "Complement found from 6879 to 6885\n",
      "Complement found from 9317 to 9323\n",
      "Complement found from 9512 to 9518\n",
      "Complement found from 9594 to 9600\n",
      "Complement found from 10622 to 10628\n",
      "Complement found from 12909 to 12915\n",
      "Complement found from 14849 to 14855\n",
      "Complement found from 15326 to 15332\n",
      "Seq is: CACTCC\n",
      "10 CACTCC motifs detected in file\n",
      "\n",
      "Complement found from 14 to 20\n",
      "Complement found from 109 to 115\n",
      "Complement found from 721 to 727\n",
      "Complement found from 1490 to 1496\n",
      "Complement found from 2279 to 2285\n",
      "Complement found from 3509 to 3515\n",
      "Complement found from 3737 to 3743\n",
      "Complement found from 3794 to 3800\n",
      "Complement found from 4010 to 4016\n",
      "Complement found from 4193 to 4199\n",
      "Complement found from 5190 to 5196\n",
      "Complement found from 5206 to 5212\n",
      "Complement found from 5323 to 5329\n",
      "Complement found from 5703 to 5709\n",
      "Complement found from 6314 to 6320\n",
      "Complement found from 6620 to 6626\n",
      "Complement found from 7111 to 7117\n",
      "Complement found from 7401 to 7407\n",
      "Complement found from 9056 to 9062\n",
      "Complement found from 10087 to 10093\n",
      "Complement found from 11214 to 11220\n",
      "Complement found from 11265 to 11271\n",
      "Complement found from 12060 to 12066\n",
      "Complement found from 12359 to 12365\n",
      "Complement found from 12398 to 12404\n",
      "Complement found from 13622 to 13628\n",
      "Complement found from 13649 to 13655\n",
      "Complement found from 13679 to 13685\n",
      "Complement found from 15264 to 15270\n",
      "Complement found from 15406 to 15412\n",
      "Complement found from 16291 to 16297\n",
      "Seq is: CACCCT\n",
      "31 CACCCT motifs detected in file\n",
      "\n",
      "Complement found from 15216 to 15222\n",
      "Seq is: GACAGA\n",
      "1 GACAGA motifs detected in file\n",
      "\n",
      "Complement found from 6209 to 6215\n",
      "Complement found from 6869 to 6875\n",
      "Complement found from 10969 to 10975\n",
      "Complement found from 11401 to 11407\n",
      "Complement found from 13029 to 13035\n",
      "Complement found from 15721 to 15727\n",
      "Complement found from 16063 to 16069\n",
      "Seq is: TGACTC\n",
      "7 TGACTC motifs detected in file\n",
      "\n",
      "Complement found from 6125 to 6131\n",
      "Complement found from 15757 to 15763\n",
      "Seq is: ATCGGA\n",
      "2 ATCGGA motifs detected in file\n",
      "\n",
      "Complement found from 3914 to 3920\n",
      "Complement found from 5981 to 5987\n",
      "Seq is: GGAGTC\n",
      "2 GGAGTC motifs detected in file\n",
      "\n",
      "Complement found from 2306 to 2312\n",
      "Seq is: TAAGTA\n",
      "1 TAAGTA motifs detected in file\n",
      "\n",
      "Complement found from 1655 to 1661\n",
      "Complement found from 2904 to 2910\n",
      "Complement found from 16140 to 16146\n",
      "Seq is: ACTTGA\n",
      "3 ACTTGA motifs detected in file\n",
      "\n",
      "Seq is: TCCAGA\n",
      "0 TCCAGA motifs detected in file\n",
      "\n",
      "Complement found from 869 to 875\n",
      "Complement found from 3221 to 3227\n",
      "Complement found from 3661 to 3667\n",
      "Complement found from 16513 to 16519\n",
      "Seq is: CAGGGT\n",
      "4 CAGGGT motifs detected in file\n",
      "\n",
      "Complement found from 6710 to 6716\n",
      "Complement found from 7430 to 7436\n",
      "Complement found from 7837 to 7843\n",
      "Complement found from 7945 to 7951\n",
      "Complement found from 10145 to 10151\n",
      "Complement found from 10481 to 10487\n",
      "Complement found from 10737 to 10743\n",
      "Complement found from 11983 to 11989\n",
      "Complement found from 16156 to 16162\n",
      "Complement found from 16303 to 16309\n",
      "Complement found from 16310 to 16316\n",
      "Complement found from 16329 to 16335\n",
      "Seq is: TACATA\n",
      "12 TACATA motifs detected in file\n",
      "\n",
      "Complement found from 1531 to 1537\n",
      "Complement found from 3429 to 3435\n",
      "Complement found from 4093 to 4099\n",
      "Complement found from 5148 to 5154\n",
      "Complement found from 7403 to 7409\n",
      "Complement found from 7823 to 7829\n",
      "Complement found from 9526 to 9532\n",
      "Complement found from 10278 to 10284\n",
      "Complement found from 10290 to 10296\n",
      "Complement found from 10554 to 10560\n",
      "Complement found from 10695 to 10701\n",
      "Complement found from 11060 to 11066\n",
      "Complement found from 11234 to 11240\n",
      "Complement found from 13069 to 13075\n",
      "Complement found from 13681 to 13687\n",
      "Complement found from 13981 to 13987\n",
      "Complement found from 14128 to 14134\n",
      "Seq is: CCCTAC\n",
      "17 CCCTAC motifs detected in file\n",
      "\n",
      "Complement found from 5351 to 5357\n",
      "Complement found from 9000 to 9006\n",
      "Complement found from 13595 to 13601\n",
      "Complement found from 15573 to 15579\n",
      "Seq is: CGCCTA\n",
      "4 CGCCTA motifs detected in file\n",
      "\n",
      "Complement found from 3748 to 3754\n",
      "Complement found from 7080 to 7086\n",
      "Complement found from 7274 to 7280\n",
      "Complement found from 8547 to 8553\n",
      "Complement found from 8593 to 8599\n",
      "Complement found from 8974 to 8980\n",
      "Complement found from 11501 to 11507\n",
      "Complement found from 11696 to 11702\n",
      "Complement found from 12043 to 12049\n",
      "Complement found from 14483 to 14489\n",
      "Complement found from 14794 to 14800\n",
      "Complement found from 15160 to 15166\n",
      "Complement found from 15743 to 15749\n",
      "Seq is: TCATTC\n",
      "13 TCATTC motifs detected in file\n",
      "\n",
      "Complement found from 2236 to 2242\n",
      "Complement found from 2318 to 2324\n",
      "Complement found from 2504 to 2510\n",
      "Complement found from 3992 to 3998\n",
      "Complement found from 4573 to 4579\n",
      "Complement found from 7252 to 7258\n",
      "Complement found from 9893 to 9899\n",
      "Complement found from 11280 to 11286\n",
      "Complement found from 12155 to 12161\n",
      "Complement found from 12671 to 12677\n",
      "Complement found from 13532 to 13538\n",
      "Complement found from 15084 to 15090\n",
      "Seq is: AAACAT\n",
      "12 AAACAT motifs detected in file\n",
      "\n",
      "Complement found from 1853 to 1859\n",
      "Complement found from 9204 to 9210\n",
      "Complement found from 9282 to 9288\n",
      "Complement found from 15461 to 15467\n",
      "Seq is: TAATGA\n",
      "4 TAATGA motifs detected in file\n",
      "\n",
      "Complement found from 6847 to 6853\n",
      "Complement found from 11685 to 11691\n",
      "Seq is: CACCGG\n",
      "2 CACCGG motifs detected in file\n",
      "\n",
      "Complement found from 3914 to 3920\n",
      "Complement found from 5981 to 5987\n",
      "Seq is: GGAGTC\n",
      "2 GGAGTC motifs detected in file\n",
      "\n",
      "Complement found from 194 to 200\n",
      "Complement found from 1699 to 1705\n",
      "Complement found from 3765 to 3771\n",
      "Complement found from 5653 to 5659\n",
      "Complement found from 6679 to 6685\n",
      "Complement found from 10100 to 10106\n",
      "Complement found from 10789 to 10795\n",
      "Complement found from 11731 to 11737\n",
      "Complement found from 13654 to 13660\n",
      "Complement found from 13738 to 13744\n",
      "Complement found from 14633 to 14639\n",
      "Complement found from 15190 to 15196\n",
      "Complement found from 15411 to 15417\n",
      "Complement found from 15628 to 15634\n",
      "Seq is: TTACTA\n",
      "14 TTACTA motifs detected in file\n",
      "\n",
      "Complement found from 1100 to 1106\n",
      "Complement found from 1218 to 1224\n",
      "Complement found from 1672 to 1678\n",
      "Complement found from 4313 to 4319\n",
      "Complement found from 4611 to 4617\n",
      "Complement found from 4977 to 4983\n",
      "Complement found from 5672 to 5678\n",
      "Complement found from 7371 to 7377\n",
      "Complement found from 8120 to 8126\n",
      "Complement found from 8824 to 8830\n",
      "Complement found from 10746 to 10752\n",
      "Complement found from 13687 to 13693\n",
      "Complement found from 14514 to 14520\n",
      "Complement found from 14583 to 14589\n",
      "Complement found from 14637 to 14643\n",
      "Complement found from 15920 to 15926\n",
      "Seq is: TAAACC\n",
      "16 TAAACC motifs detected in file\n",
      "\n",
      "Complement found from 7378 to 7384\n",
      "Seq is: GGAGTG\n",
      "1 GGAGTG motifs detected in file\n",
      "\n",
      "Complement found from 306 to 312\n",
      "Complement found from 456 to 462\n",
      "Complement found from 957 to 963\n",
      "Complement found from 1492 to 1498\n",
      "Complement found from 3568 to 3574\n",
      "Complement found from 5208 to 5214\n",
      "Complement found from 5325 to 5331\n",
      "Complement found from 6289 to 6295\n",
      "Complement found from 7364 to 7370\n",
      "Complement found from 7814 to 7820\n",
      "Complement found from 7861 to 7867\n",
      "Complement found from 9276 to 9282\n",
      "Complement found from 9726 to 9732\n",
      "Complement found from 10089 to 10095\n",
      "Complement found from 10266 to 10272\n",
      "Complement found from 10949 to 10955\n",
      "Complement found from 15537 to 15543\n",
      "Complement found from 16185 to 16191\n",
      "Seq is: CCCTCC\n",
      "18 CCCTCC motifs detected in file\n",
      "\n",
      "Complement found from 16488 to 16494\n",
      "Seq is: TGTATC\n",
      "1 TGTATC motifs detected in file\n",
      "\n",
      "Complement found from 6707 to 6713\n",
      "Complement found from 14437 to 14443\n",
      "Complement found from 16272 to 16278\n",
      "Seq is: GGATAC\n",
      "3 GGATAC motifs detected in file\n",
      "\n",
      "Complement found from 9769 to 9775\n",
      "Complement found from 15387 to 15393\n",
      "Seq is: TTCCGA\n",
      "2 TTCCGA motifs detected in file\n",
      "\n",
      "Complement found from 3042 to 3048\n",
      "Complement found from 7854 to 7860\n",
      "Seq is: CAACGA\n",
      "2 CAACGA motifs detected in file\n",
      "\n",
      "Complement found from 2963 to 2969\n",
      "Complement found from 3757 to 3763\n",
      "Complement found from 6406 to 6412\n",
      "Complement found from 6735 to 6741\n",
      "Complement found from 7044 to 7050\n",
      "Complement found from 9067 to 9073\n",
      "Complement found from 9630 to 9636\n",
      "Complement found from 11315 to 11321\n",
      "Seq is: TATCAA\n",
      "8 TATCAA motifs detected in file\n",
      "\n",
      "Seq is: CGTTCG\n",
      "0 CGTTCG motifs detected in file\n",
      "\n",
      "Complement found from 2326 to 2332\n",
      "Complement found from 3114 to 3120\n",
      "Complement found from 5442 to 5448\n",
      "Complement found from 10550 to 10556\n",
      "Complement found from 10927 to 10933\n",
      "Complement found from 12092 to 12098\n",
      "Complement found from 12982 to 12988\n",
      "Complement found from 13987 to 13993\n",
      "Complement found from 14260 to 14266\n",
      "Complement found from 14870 to 14876\n",
      "Complement found from 15098 to 15104\n",
      "Complement found from 15140 to 15146\n",
      "Complement found from 15662 to 15668\n",
      "Complement found from 16405 to 16411\n",
      "Seq is: TCCTCC\n",
      "14 TCCTCC motifs detected in file\n",
      "\n",
      "Complement found from 176 to 182\n",
      "Complement found from 2407 to 2413\n",
      "Complement found from 9463 to 9469\n",
      "Complement found from 9612 to 9618\n",
      "Complement found from 9697 to 9703\n",
      "Complement found from 10114 to 10120\n",
      "Complement found from 10234 to 10240\n",
      "Complement found from 10787 to 10793\n",
      "Complement found from 13565 to 13571\n",
      "Complement found from 14017 to 14023\n",
      "Complement found from 15052 to 15058\n",
      "Complement found from 15626 to 15632\n",
      "Seq is: TATTAC\n",
      "12 TATTAC motifs detected in file\n",
      "\n",
      "Complement found from 1287 to 1293\n",
      "Complement found from 1360 to 1366\n",
      "Complement found from 3436 to 3442\n",
      "Complement found from 7106 to 7112\n",
      "Complement found from 10142 to 10148\n",
      "Complement found from 15244 to 15250\n",
      "Seq is: GGCTAC\n",
      "6 GGCTAC motifs detected in file\n",
      "\n",
      "Complement found from 3050 to 3056\n",
      "Complement found from 7343 to 7349\n",
      "Complement found from 15963 to 15969\n",
      "Seq is: AAAGTC\n",
      "3 AAAGTC motifs detected in file\n",
      "\n",
      "Complement found from 361 to 367\n",
      "Complement found from 559 to 565\n",
      "Complement found from 2100 to 2106\n",
      "Complement found from 3476 to 3482\n",
      "Complement found from 5938 to 5944\n",
      "Complement found from 13508 to 13514\n",
      "Complement found from 14471 to 14477\n",
      "Complement found from 15423 to 15429\n",
      "Seq is: CAAAGA\n",
      "8 CAAAGA motifs detected in file\n",
      "\n",
      "Complement found from 144 to 150\n",
      "Complement found from 480 to 486\n",
      "Complement found from 4520 to 4526\n",
      "Complement found from 4747 to 4753\n",
      "Complement found from 5294 to 5300\n",
      "Complement found from 5453 to 5459\n",
      "Complement found from 6982 to 6988\n",
      "Complement found from 7807 to 7813\n",
      "Complement found from 8428 to 8434\n",
      "Complement found from 8634 to 8640\n",
      "Complement found from 11239 to 11245\n",
      "Complement found from 11662 to 11668\n",
      "Complement found from 12699 to 12705\n",
      "Complement found from 12795 to 12801\n",
      "Complement found from 13113 to 13119\n",
      "Complement found from 13572 to 13578\n",
      "Complement found from 14118 to 14124\n",
      "Complement found from 15640 to 15646\n",
      "Seq is: CTCATC\n",
      "18 CTCATC motifs detected in file\n",
      "\n",
      "Complement found from 5849 to 5855\n",
      "Complement found from 8352 to 8358\n",
      "Complement found from 11382 to 11388\n",
      "Complement found from 15014 to 15020\n",
      "Complement found from 15278 to 15284\n",
      "Complement found from 15967 to 15973\n",
      "Seq is: TCTTTA\n",
      "6 TCTTTA motifs detected in file\n",
      "\n",
      "Complement found from 3387 to 3393\n",
      "Complement found from 3603 to 3609\n",
      "Complement found from 5987 to 5993\n",
      "Complement found from 8568 to 8574\n",
      "Complement found from 9338 to 9344\n",
      "Complement found from 11269 to 11275\n",
      "Complement found from 11470 to 11476\n",
      "Complement found from 12975 to 12981\n",
      "Complement found from 13953 to 13959\n",
      "Complement found from 15493 to 15499\n",
      "Seq is: CTAGGC\n",
      "10 CTAGGC motifs detected in file\n",
      "\n",
      "Complement found from 1804 to 1810\n",
      "Complement found from 3197 to 3203\n",
      "Complement found from 3999 to 4005\n",
      "Complement found from 7558 to 7564\n",
      "Complement found from 8501 to 8507\n",
      "Complement found from 8865 to 8871\n",
      "Complement found from 10490 to 10496\n",
      "Complement found from 10592 to 10598\n",
      "Complement found from 10783 to 10789\n",
      "Complement found from 11077 to 11083\n",
      "Complement found from 11566 to 11572\n",
      "Complement found from 12780 to 12786\n",
      "Complement found from 15509 to 15515\n",
      "Seq is: ATTATA\n",
      "13 ATTATA motifs detected in file\n",
      "\n",
      "Complement found from 4709 to 4715\n",
      "Complement found from 6686 to 6692\n",
      "Complement found from 7202 to 7208\n",
      "Seq is: TCCGGA\n",
      "3 TCCGGA motifs detected in file\n",
      "\n",
      "Complement found from 2697 to 2703\n",
      "Complement found from 8853 to 8859\n",
      "Seq is: GCGGGC\n",
      "2 GCGGGC motifs detected in file\n",
      "\n",
      "Complement found from 689 to 695\n",
      "Complement found from 1476 to 1482\n",
      "Complement found from 3987 to 3993\n",
      "Complement found from 4068 to 4074\n",
      "Complement found from 8704 to 8710\n",
      "Complement found from 11353 to 11359\n",
      "Complement found from 13214 to 13220\n",
      "Complement found from 13542 to 13548\n",
      "Complement found from 15032 to 15038\n",
      "Complement found from 15415 to 15421\n",
      "Complement found from 15577 to 15583\n",
      "Seq is: TACACA\n",
      "11 TACACA motifs detected in file\n",
      "\n",
      "Complement found from 4674 to 4680\n",
      "Complement found from 12786 to 12792\n",
      "Complement found from 14286 to 14292\n",
      "Complement found from 15449 to 15455\n",
      "Seq is: TCCTTC\n",
      "4 TCCTTC motifs detected in file\n",
      "\n",
      "Complement found from 6461 to 6467\n",
      "Complement found from 15592 to 15598\n",
      "Seq is: TCCGTC\n",
      "2 TCCGTC motifs detected in file\n",
      "\n",
      "Complement found from 4323 to 4329\n",
      "Complement found from 4643 to 4649\n",
      "Complement found from 6812 to 6818\n",
      "Complement found from 8599 to 8605\n",
      "Complement found from 9876 to 9882\n",
      "Complement found from 9954 to 9960\n",
      "Complement found from 13896 to 13902\n",
      "Complement found from 14724 to 14730\n",
      "Complement found from 15562 to 15568\n",
      "Complement found from 15692 to 15698\n",
      "Complement found from 16089 to 16095\n",
      "Seq is: TATTTC\n",
      "11 TATTTC motifs detected in file\n",
      "\n",
      "Complement found from 1805 to 1811\n",
      "Complement found from 4000 to 4006\n",
      "Complement found from 8502 to 8508\n",
      "Complement found from 11078 to 11084\n",
      "Complement found from 11567 to 11573\n",
      "Seq is: TTATAA\n",
      "5 TTATAA motifs detected in file\n",
      "\n",
      "Complement found from 2328 to 2334\n",
      "Complement found from 6820 to 6826\n",
      "Complement found from 14826 to 14832\n",
      "Seq is: CTCCGC\n",
      "3 CTCCGC motifs detected in file\n",
      "\n",
      "Complement found from 6047 to 6053\n",
      "Seq is: GGTAAC\n",
      "1 GGTAAC motifs detected in file\n",
      "\n",
      "Complement found from 3698 to 3704\n",
      "Complement found from 4049 to 4055\n",
      "Complement found from 4539 to 4545\n",
      "Complement found from 11244 to 11250\n",
      "Complement found from 11764 to 11770\n",
      "Seq is: CGCACT\n",
      "5 CGCACT motifs detected in file\n",
      "\n",
      "Complement found from 4949 to 4955\n",
      "Complement found from 5200 to 5206\n",
      "Complement found from 9968 to 9974\n",
      "Complement found from 11578 to 11584\n",
      "Complement found from 12630 to 12636\n",
      "Complement found from 13368 to 13374\n",
      "Complement found from 14055 to 14061\n",
      "Complement found from 14377 to 14383\n",
      "Complement found from 15634 to 15640\n",
      "Seq is: TCCATC\n",
      "9 TCCATC motifs detected in file\n",
      "\n",
      "Complement found from 3146 to 3152\n",
      "Complement found from 8572 to 8578\n",
      "Complement found from 8968 to 8974\n",
      "Complement found from 9156 to 9162\n",
      "Complement found from 9342 to 9348\n",
      "Complement found from 11530 to 11536\n",
      "Complement found from 11585 to 11591\n",
      "Complement found from 15574 to 15580\n",
      "Seq is: GCCTAC\n",
      "8 GCCTAC motifs detected in file\n",
      "\n",
      "Complement found from 10356 to 10362\n",
      "Seq is: TAAGTC\n",
      "1 TAAGTC motifs detected in file\n",
      "\n",
      "Complement found from 52 to 58\n",
      "Complement found from 7519 to 7525\n",
      "Seq is: GGTATT\n",
      "2 GGTATT motifs detected in file\n",
      "\n",
      "Complement found from 8363 to 8369\n",
      "Complement found from 8891 to 8897\n",
      "Complement found from 10467 to 10473\n",
      "Seq is: AAATGC\n",
      "3 AAATGC motifs detected in file\n",
      "\n",
      "Complement found from 7638 to 7644\n",
      "Seq is: AGAGCT\n",
      "1 AGAGCT motifs detected in file\n",
      "\n",
      "Complement found from 1140 to 1146\n",
      "Complement found from 2218 to 2224\n",
      "Complement found from 4019 to 4025\n",
      "Complement found from 5376 to 5382\n",
      "Complement found from 12347 to 12353\n",
      "Complement found from 12970 to 12976\n",
      "Complement found from 14905 to 14911\n",
      "Seq is: CACTAC\n",
      "7 CACTAC motifs detected in file\n",
      "\n",
      "Complement found from 3491 to 3497\n",
      "Complement found from 8576 to 8582\n",
      "Seq is: ACCCGC\n",
      "2 ACCCGC motifs detected in file\n",
      "\n",
      "Complement found from 849 to 855\n",
      "Complement found from 1521 to 1527\n",
      "Complement found from 1861 to 1867\n",
      "Complement found from 5082 to 5088\n",
      "Complement found from 5100 to 5106\n",
      "Complement found from 7733 to 7739\n",
      "Complement found from 10026 to 10032\n",
      "Complement found from 10961 to 10967\n",
      "Complement found from 14207 to 14213\n",
      "Seq is: TAACTA\n",
      "9 TAACTA motifs detected in file\n",
      "\n",
      "Complement found from 6941 to 6947\n",
      "Complement found from 7708 to 7714\n",
      "Complement found from 10923 to 10929\n",
      "Complement found from 14931 to 14937\n",
      "Seq is: CTTTTC\n",
      "4 CTTTTC motifs detected in file\n",
      "\n",
      "Complement found from 7129 to 7135\n",
      "Complement found from 9866 to 9872\n",
      "Complement found from 14994 to 15000\n",
      "Seq is: CGCCAA\n",
      "3 CGCCAA motifs detected in file\n",
      "\n",
      "Complement found from 2328 to 2334\n",
      "Complement found from 6820 to 6826\n",
      "Complement found from 14826 to 14832\n",
      "Seq is: CTCCGC\n",
      "3 CTCCGC motifs detected in file\n",
      "\n",
      "Complement found from 3880 to 3886\n",
      "Complement found from 5530 to 5536\n",
      "Complement found from 5658 to 5664\n",
      "Complement found from 7117 to 7123\n",
      "Complement found from 12506 to 12512\n",
      "Complement found from 13511 to 13517\n",
      "Seq is: AGACCA\n",
      "6 AGACCA motifs detected in file\n",
      "\n",
      "Complement found from 144 to 150\n",
      "Complement found from 480 to 486\n",
      "Complement found from 4520 to 4526\n",
      "Complement found from 4747 to 4753\n",
      "Complement found from 5294 to 5300\n",
      "Complement found from 5453 to 5459\n",
      "Complement found from 6982 to 6988\n",
      "Complement found from 7807 to 7813\n",
      "Complement found from 8428 to 8434\n",
      "Complement found from 8634 to 8640\n",
      "Complement found from 11239 to 11245\n",
      "Complement found from 11662 to 11668\n",
      "Complement found from 12699 to 12705\n",
      "Complement found from 12795 to 12801\n",
      "Complement found from 13113 to 13119\n",
      "Complement found from 13572 to 13578\n",
      "Complement found from 14118 to 14124\n",
      "Complement found from 15640 to 15646\n",
      "Seq is: CTCATC\n",
      "18 CTCATC motifs detected in file\n",
      "\n",
      "Complement found from 7010 to 7016\n",
      "Complement found from 10734 to 10740\n",
      "Seq is: ACGTAC\n",
      "2 ACGTAC motifs detected in file\n",
      "\n",
      "Complement found from 7857 to 7863\n",
      "Complement found from 15589 to 15595\n",
      "Seq is: CGATCC\n",
      "2 CGATCC motifs detected in file\n",
      "\n",
      "Complement found from 3799 to 3805\n",
      "Complement found from 6405 to 6411\n",
      "Complement found from 7643 to 7649\n",
      "Complement found from 9094 to 9100\n",
      "Complement found from 12469 to 12475\n",
      "Seq is: TTATCA\n",
      "5 TTATCA motifs detected in file\n",
      "\n",
      "Complement found from 2119 to 2125\n",
      "Seq is: GGACAC\n",
      "1 GGACAC motifs detected in file\n",
      "\n",
      "Complement found from 4911 to 4917\n",
      "Complement found from 8711 to 8717\n",
      "Complement found from 9357 to 9363\n",
      "Complement found from 10311 to 10317\n",
      "Complement found from 11246 to 11252\n",
      "Complement found from 11276 to 11282\n",
      "Complement found from 11816 to 11822\n",
      "Complement found from 13147 to 13153\n",
      "Complement found from 14391 to 14397\n",
      "Complement found from 15701 to 15707\n",
      "Seq is: CACTAA\n",
      "10 CACTAA motifs detected in file\n",
      "\n",
      "Complement found from 589 to 595\n",
      "Complement found from 2837 to 2843\n",
      "Complement found from 3319 to 3325\n",
      "Complement found from 4104 to 4110\n",
      "Complement found from 4165 to 4171\n",
      "Complement found from 6217 to 6223\n",
      "Complement found from 6818 to 6824\n",
      "Complement found from 8469 to 8475\n",
      "Complement found from 8622 to 8628\n",
      "Complement found from 8773 to 8779\n",
      "Complement found from 9287 to 9293\n",
      "Complement found from 13446 to 13452\n",
      "Complement found from 13581 to 13587\n",
      "Complement found from 14052 to 14058\n",
      "Complement found from 14374 to 14380\n",
      "Complement found from 14526 to 14532\n",
      "Complement found from 14804 to 14810\n",
      "Complement found from 15332 to 15338\n",
      "Complement found from 15379 to 15385\n",
      "Complement found from 15488 to 15494\n",
      "Complement found from 15737 to 15743\n",
      "Seq is: ACCTCC\n",
      "21 ACCTCC motifs detected in file\n",
      "\n",
      "Complement found from 2095 to 2101\n",
      "Complement found from 7694 to 7700\n",
      "Complement found from 7802 to 7808\n",
      "Seq is: TAGTCC\n",
      "3 TAGTCC motifs detected in file\n",
      "\n",
      "Seq is: GTCCGC\n",
      "0 GTCCGC motifs detected in file\n",
      "\n",
      "Complement found from 16502 to 16508\n",
      "Seq is: GGTTCC\n",
      "1 GGTTCC motifs detected in file\n",
      "\n",
      "Complement found from 4932 to 4938\n",
      "Complement found from 5873 to 5879\n",
      "Complement found from 8792 to 8798\n",
      "Complement found from 11294 to 11300\n",
      "Complement found from 11930 to 11936\n",
      "Complement found from 12010 to 12016\n",
      "Complement found from 14848 to 14854\n",
      "Complement found from 14951 to 14957\n",
      "Seq is: TCACTC\n",
      "8 TCACTC motifs detected in file\n",
      "\n",
      "Complement found from 3933 to 3939\n",
      "Complement found from 5717 to 5723\n",
      "Complement found from 6650 to 6656\n",
      "Complement found from 6743 to 6749\n",
      "Complement found from 7076 to 7082\n",
      "Complement found from 9809 to 9815\n",
      "Complement found from 9905 to 9911\n",
      "Complement found from 10175 to 10181\n",
      "Seq is: GGCTTC\n",
      "8 GGCTTC motifs detected in file\n",
      "\n",
      "Complement found from 1293 to 1299\n",
      "Complement found from 1738 to 1744\n",
      "Complement found from 2460 to 2466\n",
      "Complement found from 4399 to 4405\n",
      "Complement found from 6857 to 6863\n",
      "Complement found from 12330 to 12336\n",
      "Seq is: AAAGTA\n",
      "6 AAAGTA motifs detected in file\n",
      "\n",
      "Complement found from 8997 to 9003\n",
      "Seq is: GTACGC\n",
      "1 GTACGC motifs detected in file\n",
      "\n",
      "Complement found from 3513 to 3519\n",
      "Complement found from 4065 to 4071\n",
      "Complement found from 4498 to 4504\n",
      "Complement found from 5336 to 5342\n",
      "Complement found from 5926 to 5932\n",
      "Complement found from 8275 to 8281\n",
      "Complement found from 9086 to 9092\n",
      "Complement found from 9183 to 9189\n",
      "Complement found from 10871 to 10877\n",
      "Complement found from 11038 to 11044\n",
      "Complement found from 11807 to 11813\n",
      "Complement found from 11980 to 11986\n",
      "Complement found from 13785 to 13791\n",
      "Complement found from 15067 to 15073\n",
      "Seq is: CTCTAC\n",
      "14 CTCTAC motifs detected in file\n",
      "\n",
      "Complement found from 1674 to 1680\n",
      "Complement found from 1993 to 1999\n",
      "Complement found from 3600 to 3606\n",
      "Complement found from 4256 to 4262\n",
      "Complement found from 7123 to 7129\n",
      "Complement found from 7902 to 7908\n",
      "Complement found from 8826 to 8832\n",
      "Complement found from 10742 to 10748\n",
      "Complement found from 10748 to 10754\n",
      "Complement found from 10900 to 10906\n",
      "Complement found from 11878 to 11884\n",
      "Complement found from 12741 to 12747\n",
      "Complement found from 14146 to 14152\n",
      "Complement found from 16283 to 16289\n",
      "Seq is: AACCTA\n",
      "14 AACCTA motifs detected in file\n",
      "\n",
      "Complement found from 569 to 575\n",
      "Complement found from 802 to 808\n",
      "Complement found from 1082 to 1088\n",
      "Complement found from 5302 to 5308\n",
      "Complement found from 5446 to 5452\n",
      "Complement found from 5600 to 5606\n",
      "Complement found from 5895 to 5901\n",
      "Complement found from 6844 to 6850\n",
      "Complement found from 7398 to 7404\n",
      "Complement found from 8079 to 8085\n",
      "Complement found from 8558 to 8564\n",
      "Complement found from 8618 to 8624\n",
      "Complement found from 11139 to 11145\n",
      "Complement found from 11868 to 11874\n",
      "Complement found from 12482 to 12488\n",
      "Complement found from 12967 to 12973\n",
      "Complement found from 13052 to 13058\n",
      "Complement found from 13646 to 13652\n",
      "Complement found from 13676 to 13682\n",
      "Complement found from 13879 to 13885\n",
      "Complement found from 14388 to 14394\n",
      "Complement found from 14619 to 14625\n",
      "Complement found from 14808 to 14814\n",
      "Complement found from 15541 to 15547\n",
      "Seq is: CCCCAC\n",
      "24 CCCCAC motifs detected in file\n",
      "\n",
      "Complement found from 306 to 312\n",
      "Complement found from 456 to 462\n",
      "Complement found from 957 to 963\n",
      "Complement found from 1492 to 1498\n",
      "Complement found from 3568 to 3574\n",
      "Complement found from 5208 to 5214\n",
      "Complement found from 5325 to 5331\n",
      "Complement found from 6289 to 6295\n",
      "Complement found from 7364 to 7370\n",
      "Complement found from 7814 to 7820\n",
      "Complement found from 7861 to 7867\n",
      "Complement found from 9276 to 9282\n",
      "Complement found from 9726 to 9732\n",
      "Complement found from 10089 to 10095\n",
      "Complement found from 10266 to 10272\n",
      "Complement found from 10949 to 10955\n",
      "Complement found from 15537 to 15543\n",
      "Complement found from 16185 to 16191\n",
      "Seq is: CCCTCC\n",
      "18 CCCTCC motifs detected in file\n",
      "\n",
      "Complement found from 1916 to 1922\n",
      "Complement found from 4989 to 4995\n",
      "Complement found from 10597 to 10603\n",
      "Complement found from 15775 to 15781\n",
      "Seq is: AGCTAC\n",
      "4 AGCTAC motifs detected in file\n",
      "\n",
      "Complement found from 3644 to 3650\n",
      "Complement found from 3825 to 3831\n",
      "Complement found from 7219 to 7225\n",
      "Complement found from 9614 to 9620\n",
      "Complement found from 9988 to 9994\n",
      "Complement found from 13110 to 13116\n",
      "Complement found from 13567 to 13573\n",
      "Seq is: TTACTC\n",
      "7 TTACTC motifs detected in file\n",
      "\n",
      "Complement found from 2187 to 2193\n",
      "Complement found from 2269 to 2275\n",
      "Complement found from 4727 to 4733\n",
      "Complement found from 4736 to 4742\n",
      "Complement found from 5660 to 5666\n",
      "Complement found from 7889 to 7895\n",
      "Complement found from 8981 to 8987\n",
      "Complement found from 9213 to 9219\n",
      "Complement found from 9369 to 9375\n",
      "Complement found from 14250 to 14256\n",
      "Complement found from 14365 to 14371\n",
      "Complement found from 14698 to 14704\n",
      "Complement found from 14742 to 14748\n",
      "Seq is: ACCAAT\n",
      "13 ACCAAT motifs detected in file\n",
      "\n",
      "Complement found from 208 to 214\n",
      "Complement found from 4467 to 4473\n",
      "Complement found from 5637 to 5643\n",
      "Complement found from 8042 to 8048\n",
      "Complement found from 8399 to 8405\n",
      "Complement found from 10110 to 10116\n",
      "Complement found from 10853 to 10859\n",
      "Complement found from 11075 to 11081\n",
      "Complement found from 11564 to 11570\n",
      "Complement found from 12709 to 12715\n",
      "Complement found from 14083 to 14089\n",
      "Complement found from 14783 to 14789\n",
      "Complement found from 15179 to 15185\n",
      "Seq is: TAATTA\n",
      "13 TAATTA motifs detected in file\n",
      "\n",
      "Complement found from 1734 to 1740\n",
      "Complement found from 1868 to 1874\n",
      "Complement found from 3139 to 3145\n",
      "Complement found from 4299 to 4305\n",
      "Complement found from 4414 to 4420\n",
      "Complement found from 4569 to 4575\n",
      "Complement found from 4608 to 4614\n",
      "Complement found from 5403 to 5409\n",
      "Complement found from 8492 to 8498\n",
      "Complement found from 9678 to 9684\n",
      "Complement found from 12325 to 12331\n",
      "Complement found from 13670 to 13676\n",
      "Complement found from 13870 to 13876\n",
      "Complement found from 14494 to 14500\n",
      "Complement found from 16551 to 16557\n",
      "Seq is: AAATAA\n",
      "15 AAATAA motifs detected in file\n",
      "\n",
      "Complement found from 309 to 315\n",
      "Complement found from 3166 to 3172\n",
      "Complement found from 4247 to 4253\n",
      "Complement found from 7953 to 7959\n",
      "Complement found from 8603 to 8609\n",
      "Complement found from 10190 to 10196\n",
      "Complement found from 12082 to 12088\n",
      "Complement found from 12383 to 12389\n",
      "Complement found from 13752 to 13758\n",
      "Complement found from 13761 to 13767\n",
      "Complement found from 13780 to 13786\n",
      "Complement found from 14153 to 14159\n",
      "Complement found from 14487 to 14493\n",
      "Complement found from 14529 to 14535\n",
      "Seq is: TCCCCC\n",
      "14 TCCCCC motifs detected in file\n",
      "\n",
      "Complement found from 1116 to 1122\n",
      "Complement found from 2389 to 2395\n",
      "Complement found from 4739 to 4745\n",
      "Complement found from 5709 to 5715\n",
      "Complement found from 7877 to 7883\n",
      "Complement found from 8672 to 8678\n",
      "Complement found from 10081 to 10087\n",
      "Complement found from 10830 to 10836\n",
      "Complement found from 10890 to 10896\n",
      "Complement found from 14218 to 14224\n",
      "Complement found from 14268 to 14274\n",
      "Complement found from 14574 to 14580\n",
      "Complement found from 15420 to 15426\n",
      "Complement found from 16214 to 16220\n",
      "Complement found from 16415 to 16421\n",
      "Seq is: AATCAA\n",
      "15 AATCAA motifs detected in file\n",
      "\n",
      "Complement found from 5619 to 5625\n",
      "Complement found from 11188 to 11194\n",
      "Complement found from 11762 to 11768\n",
      "Seq is: AACGCA\n",
      "3 AACGCA motifs detected in file\n",
      "\n",
      "Complement found from 530 to 536\n",
      "Complement found from 863 to 869\n",
      "Complement found from 911 to 917\n",
      "Complement found from 1833 to 1839\n",
      "Complement found from 5840 to 5846\n",
      "Complement found from 6429 to 6435\n",
      "Complement found from 10608 to 10614\n",
      "Complement found from 12364 to 12370\n",
      "Complement found from 12407 to 12413\n",
      "Complement found from 13673 to 13679\n",
      "Complement found from 14125 to 14131\n",
      "Complement found from 14385 to 14391\n",
      "Complement found from 14766 to 14772\n",
      "Seq is: TAACCC\n",
      "13 TAACCC motifs detected in file\n",
      "\n",
      "Complement found from 1428 to 1434\n",
      "Complement found from 5553 to 5559\n",
      "Complement found from 7293 to 7299\n",
      "Complement found from 14204 to 14210\n",
      "Complement found from 15176 to 15182\n",
      "Complement found from 15770 to 15776\n",
      "Seq is: CAGTAA\n",
      "6 CAGTAA motifs detected in file\n",
      "\n",
      "Complement found from 5677 to 5683\n",
      "Complement found from 5935 to 5941\n",
      "Complement found from 7181 to 7187\n",
      "Complement found from 8560 to 8566\n",
      "Complement found from 8764 to 8770\n",
      "Complement found from 8913 to 8919\n",
      "Complement found from 10130 to 10136\n",
      "Complement found from 11993 to 11999\n",
      "Complement found from 12484 to 12490\n",
      "Complement found from 12541 to 12547\n",
      "Complement found from 12927 to 12933\n",
      "Complement found from 13376 to 13382\n",
      "Complement found from 14327 to 14333\n",
      "Complement found from 14621 to 14627\n",
      "Seq is: CCACAA\n",
      "14 CCACAA motifs detected in file\n",
      "\n",
      "Complement found from 186 to 192\n",
      "Complement found from 5420 to 5426\n",
      "Seq is: GAACAT\n",
      "2 GAACAT motifs detected in file\n",
      "\n",
      "Complement found from 1627 to 1633\n",
      "Complement found from 1648 to 1654\n",
      "Complement found from 2042 to 2048\n",
      "Complement found from 3188 to 3194\n",
      "Complement found from 5125 to 5131\n",
      "Complement found from 9840 to 9846\n",
      "Complement found from 11334 to 11340\n",
      "Complement found from 12263 to 12269\n",
      "Seq is: CAACTT\n",
      "8 CAACTT motifs detected in file\n",
      "\n",
      "Seq is: TATGTT\n",
      "0 TATGTT motifs detected in file\n",
      "\n",
      "Complement found from 1917 to 1923\n",
      "Complement found from 6824 to 6830\n",
      "Complement found from 13578 to 13584\n",
      "Complement found from 14984 to 14990\n",
      "Complement found from 15776 to 15782\n",
      "Seq is: GCTACC\n",
      "5 GCTACC motifs detected in file\n",
      "\n",
      "Complement found from 2458 to 2464\n",
      "Complement found from 5747 to 5753\n",
      "Complement found from 5829 to 5835\n",
      "Complement found from 6693 to 6699\n",
      "Complement found from 7447 to 7453\n",
      "Complement found from 7514 to 7520\n",
      "Complement found from 9432 to 9438\n",
      "Complement found from 10048 to 10054\n",
      "Complement found from 10380 to 10386\n",
      "Complement found from 14074 to 14080\n",
      "Complement found from 15961 to 15967\n",
      "Seq is: AAAAAG\n",
      "11 AAAAAG motifs detected in file\n",
      "\n",
      "Complement found from 1777 to 1783\n",
      "Complement found from 6250 to 6256\n",
      "Complement found from 9232 to 9238\n",
      "Complement found from 10409 to 10415\n",
      "Complement found from 11367 to 11373\n",
      "Complement found from 12142 to 12148\n",
      "Complement found from 13088 to 13094\n",
      "Seq is: TATAGT\n",
      "7 TATAGT motifs detected in file\n",
      "\n",
      "Complement found from 383 to 389\n",
      "Complement found from 1598 to 1604\n",
      "Complement found from 1907 to 1913\n",
      "Complement found from 2667 to 2673\n",
      "Complement found from 7970 to 7976\n",
      "Complement found from 11170 to 11176\n",
      "Complement found from 14201 to 14207\n",
      "Complement found from 15767 to 15773\n",
      "Seq is: AACCAG\n",
      "8 AACCAG motifs detected in file\n",
      "\n",
      "Complement found from 295 to 301\n",
      "Complement found from 2186 to 2192\n",
      "Complement found from 3473 to 3479\n",
      "Complement found from 4085 to 4091\n",
      "Complement found from 7888 to 7894\n",
      "Complement found from 8477 to 8483\n",
      "Complement found from 8803 to 8809\n",
      "Complement found from 9212 to 9218\n",
      "Complement found from 14040 to 14046\n",
      "Complement found from 14183 to 14189\n",
      "Complement found from 14249 to 14255\n",
      "Complement found from 14364 to 14370\n",
      "Complement found from 14403 to 14409\n",
      "Complement found from 14741 to 14747\n",
      "Seq is: CACCAA\n",
      "14 CACCAA motifs detected in file\n",
      "\n",
      "Complement found from 8963 to 8969\n",
      "Complement found from 9621 to 9627\n",
      "Complement found from 12158 to 12164\n",
      "Complement found from 12797 to 12803\n",
      "Seq is: CATCAG\n",
      "4 CATCAG motifs detected in file\n",
      "\n",
      "Complement found from 74 to 80\n",
      "Complement found from 761 to 767\n",
      "Seq is: GCACGC\n",
      "2 GCACGC motifs detected in file\n",
      "\n",
      "Complement found from 4114 to 4120\n",
      "Complement found from 4944 to 4950\n",
      "Complement found from 5505 to 5511\n",
      "Complement found from 6637 to 6643\n",
      "Complement found from 8732 to 8738\n",
      "Complement found from 10242 to 10248\n",
      "Complement found from 10332 to 10338\n",
      "Seq is: TCTTAT\n",
      "7 TCTTAT motifs detected in file\n",
      "\n",
      "Complement found from 470 to 476\n",
      "Complement found from 1700 to 1706\n",
      "Complement found from 3439 to 3445\n",
      "Complement found from 4732 to 4738\n",
      "Complement found from 5104 to 5110\n",
      "Complement found from 5118 to 5124\n",
      "Complement found from 5151 to 5157\n",
      "Complement found from 6525 to 6531\n",
      "Complement found from 6680 to 6686\n",
      "Complement found from 6999 to 7005\n",
      "Complement found from 7013 to 7019\n",
      "Complement found from 8158 to 8164\n",
      "Complement found from 8379 to 8385\n",
      "Complement found from 10101 to 10107\n",
      "Complement found from 10790 to 10796\n",
      "Complement found from 11288 to 11294\n",
      "Complement found from 14211 to 14217\n",
      "Complement found from 15412 to 15418\n",
      "Seq is: TACTAC\n",
      "18 TACTAC motifs detected in file\n",
      "\n",
      "Seq is: ATACGA\n",
      "0 ATACGA motifs detected in file\n",
      "\n",
      "Complement found from 282 to 288\n",
      "Complement found from 359 to 365\n",
      "Complement found from 643 to 649\n",
      "Complement found from 1120 to 1126\n",
      "Complement found from 2763 to 2769\n",
      "Complement found from 7725 to 7731\n",
      "Complement found from 8506 to 8512\n",
      "Complement found from 8689 to 8695\n",
      "Complement found from 10418 to 10424\n",
      "Complement found from 12414 to 12420\n",
      "Complement found from 12931 to 12937\n",
      "Complement found from 13860 to 13866\n",
      "Complement found from 14187 to 14193\n",
      "Complement found from 14656 to 14662\n",
      "Complement found from 15601 to 15607\n",
      "Complement found from 15680 to 15686\n",
      "Complement found from 15865 to 15871\n",
      "Complement found from 16279 to 16285\n",
      "Seq is: AACAAA\n",
      "18 AACAAA motifs detected in file\n",
      "\n",
      "Complement found from 1708 to 1714\n",
      "Complement found from 9195 to 9201\n",
      "Complement found from 10037 to 10043\n",
      "Complement found from 12175 to 12181\n",
      "Complement found from 14475 to 14481\n",
      "Complement found from 15764 to 15770\n",
      "Seq is: GACAAC\n",
      "6 GACAAC motifs detected in file\n",
      "\n",
      "Complement found from 902 to 908\n",
      "Complement found from 2255 to 2261\n",
      "Complement found from 4385 to 4391\n",
      "Complement found from 5373 to 5379\n",
      "Complement found from 9166 to 9172\n",
      "Complement found from 10538 to 10544\n",
      "Complement found from 11495 to 11501\n",
      "Complement found from 12047 to 12053\n",
      "Complement found from 13930 to 13936\n",
      "Complement found from 15269 to 15275\n",
      "Complement found from 16230 to 16236\n",
      "Seq is: TCACAC\n",
      "11 TCACAC motifs detected in file\n",
      "\n",
      "Complement found from 1732 to 1738\n",
      "Complement found from 4897 to 4903\n",
      "Complement found from 5253 to 5259\n",
      "Complement found from 8626 to 8632\n",
      "Complement found from 9676 to 9682\n",
      "Complement found from 10465 to 10471\n",
      "Complement found from 10887 to 10893\n",
      "Complement found from 12323 to 12329\n",
      "Complement found from 14042 to 14048\n",
      "Complement found from 14874 to 14880\n",
      "Complement found from 15153 to 15159\n",
      "Seq is: CCAAAT\n",
      "11 CCAAAT motifs detected in file\n",
      "\n",
      "Complement found from 3763 to 3769\n",
      "Complement found from 4201 to 4207\n",
      "Complement found from 4229 to 4235\n",
      "Complement found from 9014 to 9020\n",
      "Complement found from 11729 to 11735\n",
      "Complement found from 12117 to 12123\n",
      "Complement found from 13736 to 13742\n",
      "Complement found from 14631 to 14637\n",
      "Complement found from 16098 to 16104\n",
      "Complement found from 16338 to 16344\n",
      "Seq is: CATTAC\n",
      "10 CATTAC motifs detected in file\n",
      "\n",
      "Complement found from 353 to 359\n",
      "Complement found from 557 to 563\n",
      "Complement found from 1063 to 1069\n",
      "Complement found from 1682 to 1688\n",
      "Complement found from 1731 to 1737\n",
      "Complement found from 2233 to 2239\n",
      "Complement found from 3715 to 3721\n",
      "Complement found from 5252 to 5258\n",
      "Complement found from 7468 to 7474\n",
      "Complement found from 12302 to 12308\n",
      "Complement found from 12547 to 12553\n",
      "Complement found from 12668 to 12674\n",
      "Complement found from 14071 to 14077\n",
      "Complement found from 14532 to 14538\n",
      "Complement found from 15988 to 15994\n",
      "Seq is: CCCAAA\n",
      "15 CCCAAA motifs detected in file\n",
      "\n",
      "Complement found from 430 to 436\n",
      "Complement found from 565 to 571\n",
      "Complement found from 634 to 640\n",
      "Complement found from 799 to 805\n",
      "Complement found from 953 to 959\n",
      "Complement found from 4388 to 4394\n",
      "Complement found from 4831 to 4837\n",
      "Complement found from 5434 to 5440\n",
      "Complement found from 5892 to 5898\n",
      "Complement found from 8269 to 8275\n",
      "Complement found from 8929 to 8935\n",
      "Complement found from 9568 to 9574\n",
      "Complement found from 10159 to 10165\n",
      "Complement found from 12965 to 12971\n",
      "Complement found from 13023 to 13029\n",
      "Complement found from 13055 to 13061\n",
      "Complement found from 13125 to 13131\n",
      "Complement found from 14337 to 14343\n",
      "Complement found from 14811 to 14817\n",
      "Complement found from 15534 to 15540\n",
      "Complement found from 16256 to 16262\n",
      "Seq is: CACCCC\n",
      "21 CACCCC motifs detected in file\n",
      "\n",
      "Complement found from 1934 to 1940\n",
      "Complement found from 4277 to 4283\n",
      "Complement found from 5830 to 5836\n",
      "Complement found from 6694 to 6700\n",
      "Complement found from 10049 to 10055\n",
      "Seq is: AAAAGA\n",
      "5 AAAAGA motifs detected in file\n",
      "\n",
      "Complement found from 15998 to 16004\n",
      "Seq is: AGATTC\n",
      "1 AGATTC motifs detected in file\n",
      "\n",
      "Complement found from 1968 to 1974\n",
      "Seq is: GGAAGA\n",
      "1 GGAAGA motifs detected in file\n",
      "\n",
      "Complement found from 2932 to 2938\n",
      "Complement found from 9450 to 9456\n",
      "Complement found from 12274 to 12280\n",
      "Seq is: GGATAA\n",
      "3 GGATAA motifs detected in file\n",
      "\n",
      "Complement found from 537 to 543\n",
      "Complement found from 1079 to 1085\n",
      "Complement found from 3200 to 3206\n",
      "Complement found from 3306 to 3312\n",
      "Complement found from 3576 to 3582\n",
      "Complement found from 4133 to 4139\n",
      "Complement found from 4224 to 4230\n",
      "Complement found from 4432 to 4438\n",
      "Complement found from 4452 to 4458\n",
      "Complement found from 6113 to 6119\n",
      "Complement found from 12428 to 12434\n",
      "Complement found from 15512 to 15518\n",
      "Seq is: ATACCC\n",
      "12 ATACCC motifs detected in file\n",
      "\n",
      "Complement found from 1732 to 1738\n",
      "Complement found from 4897 to 4903\n",
      "Complement found from 5253 to 5259\n",
      "Complement found from 8626 to 8632\n",
      "Complement found from 9676 to 9682\n",
      "Complement found from 10465 to 10471\n",
      "Complement found from 10887 to 10893\n",
      "Complement found from 12323 to 12329\n",
      "Complement found from 14042 to 14048\n",
      "Complement found from 14874 to 14880\n",
      "Complement found from 15153 to 15159\n",
      "Seq is: CCAAAT\n",
      "11 CCAAAT motifs detected in file\n",
      "\n",
      "Complement found from 279 to 285\n",
      "Complement found from 2702 to 2708\n",
      "Complement found from 4723 to 4729\n",
      "Complement found from 5065 to 5071\n",
      "Complement found from 6427 to 6433\n",
      "Complement found from 7538 to 7544\n",
      "Complement found from 7839 to 7845\n",
      "Complement found from 9322 to 9328\n",
      "Complement found from 10606 to 10612\n",
      "Complement found from 10739 to 10745\n",
      "Complement found from 14143 to 14149\n",
      "Complement found from 16459 to 16465\n",
      "Seq is: CATAAC\n",
      "12 CATAAC motifs detected in file\n",
      "\n",
      "Complement found from 238 to 244\n",
      "Complement found from 281 to 287\n",
      "Complement found from 2762 to 2768\n",
      "Complement found from 3299 to 3305\n",
      "Complement found from 5392 to 5398\n",
      "Complement found from 8505 to 8511\n",
      "Complement found from 10940 to 10946\n",
      "Complement found from 11570 to 11576\n",
      "Complement found from 12027 to 12033\n",
      "Complement found from 12246 to 12252\n",
      "Complement found from 12413 to 12419\n",
      "Complement found from 12737 to 12743\n",
      "Complement found from 13385 to 13391\n",
      "Complement found from 13742 to 13748\n",
      "Complement found from 14570 to 14576\n",
      "Complement found from 15600 to 15606\n",
      "Seq is: TAACAA\n",
      "16 TAACAA motifs detected in file\n",
      "\n",
      "Complement found from 1131 to 1137\n",
      "Complement found from 6873 to 6879\n",
      "Seq is: TCGCCA\n",
      "2 TCGCCA motifs detected in file\n",
      "\n",
      "Complement found from 374 to 380\n",
      "Complement found from 2522 to 2528\n",
      "Complement found from 2864 to 2870\n",
      "Complement found from 14912 to 14918\n",
      "Complement found from 15482 to 15488\n",
      "Complement found from 15909 to 15915\n",
      "Seq is: CACCAG\n",
      "6 CACCAG motifs detected in file\n",
      "\n",
      "Complement found from 570 to 576\n",
      "Complement found from 2056 to 2062\n",
      "Complement found from 2769 to 2775\n",
      "Complement found from 3203 to 3209\n",
      "Complement found from 5022 to 5028\n",
      "Complement found from 5447 to 5453\n",
      "Complement found from 5676 to 5682\n",
      "Complement found from 7180 to 7186\n",
      "Complement found from 8080 to 8086\n",
      "Complement found from 8559 to 8565\n",
      "Complement found from 10844 to 10850\n",
      "Complement found from 12483 to 12489\n",
      "Complement found from 12926 to 12932\n",
      "Complement found from 14357 to 14363\n",
      "Complement found from 14620 to 14626\n",
      "Complement found from 14641 to 14647\n",
      "Complement found from 14945 to 14951\n",
      "Complement found from 15542 to 15548\n",
      "Complement found from 16535 to 16541\n",
      "Seq is: CCCACA\n",
      "19 CCCACA motifs detected in file\n",
      "\n",
      "Complement found from 834 to 840\n",
      "Complement found from 964 to 970\n",
      "Complement found from 2899 to 2905\n",
      "Complement found from 4776 to 4782\n",
      "Complement found from 15650 to 15656\n",
      "Seq is: CAATAA\n",
      "5 CAATAA motifs detected in file\n",
      "\n",
      "Complement found from 1058 to 1064\n",
      "Complement found from 1895 to 1901\n",
      "Complement found from 2856 to 2862\n",
      "Complement found from 16554 to 16560\n",
      "Seq is: TAAGAC\n",
      "4 TAAGAC motifs detected in file\n",
      "\n",
      "Complement found from 1156 to 1162\n",
      "Complement found from 2350 to 2356\n",
      "Complement found from 2454 to 2460\n",
      "Complement found from 2798 to 2804\n",
      "Complement found from 3263 to 3269\n",
      "Complement found from 8092 to 8098\n",
      "Complement found from 8887 to 8893\n",
      "Complement found from 11464 to 11470\n",
      "Complement found from 13867 to 13873\n",
      "Complement found from 14501 to 14507\n",
      "Seq is: TTAAAA\n",
      "10 TTAAAA motifs detected in file\n",
      "\n",
      "Complement found from 304 to 310\n",
      "Complement found from 955 to 961\n",
      "Complement found from 3566 to 3572\n",
      "Complement found from 3584 to 3590\n",
      "Complement found from 3892 to 3898\n",
      "Complement found from 4248 to 4254\n",
      "Complement found from 4317 to 4323\n",
      "Complement found from 4794 to 4800\n",
      "Complement found from 6419 to 6425\n",
      "Complement found from 8271 to 8277\n",
      "Complement found from 8280 to 8286\n",
      "Complement found from 8604 to 8610\n",
      "Complement found from 10935 to 10941\n",
      "Complement found from 10947 to 10953\n",
      "Complement found from 11511 to 11517\n",
      "Complement found from 11672 to 11678\n",
      "Complement found from 13127 to 13133\n",
      "Complement found from 13762 to 13768\n",
      "Complement found from 13781 to 13787\n",
      "Complement found from 14488 to 14494\n",
      "Complement found from 14769 to 14775\n",
      "Complement found from 15366 to 15372\n",
      "Complement found from 16183 to 16189\n",
      "Complement found from 16375 to 16381\n",
      "Seq is: CCCCCT\n",
      "24 CCCCCT motifs detected in file\n",
      "\n",
      "Complement found from 963 to 969\n",
      "Complement found from 2378 to 2384\n",
      "Complement found from 2898 to 2904\n",
      "Complement found from 4728 to 4734\n",
      "Complement found from 6433 to 6439\n",
      "Complement found from 8982 to 8988\n",
      "Complement found from 10635 to 10641\n",
      "Complement found from 14251 to 14257\n",
      "Complement found from 14752 to 14758\n",
      "Seq is: CCAATA\n",
      "9 CCAATA motifs detected in file\n",
      "\n",
      "Complement found from 506 to 512\n",
      "Complement found from 1307 to 1313\n",
      "Complement found from 1729 to 1735\n",
      "Complement found from 3201 to 3207\n",
      "Complement found from 3307 to 3313\n",
      "Complement found from 3337 to 3343\n",
      "Complement found from 3577 to 3583\n",
      "Complement found from 4225 to 4231\n",
      "Complement found from 4822 to 4828\n",
      "Complement found from 5020 to 5026\n",
      "Complement found from 6114 to 6120\n",
      "Complement found from 13325 to 13331\n",
      "Complement found from 16287 to 16293\n",
      "Seq is: TACCCA\n",
      "13 TACCCA motifs detected in file\n",
      "\n",
      "Complement found from 1803 to 1809\n",
      "Complement found from 6403 to 6409\n",
      "Complement found from 7557 to 7563\n",
      "Complement found from 8500 to 8506\n",
      "Complement found from 10111 to 10117\n",
      "Complement found from 10445 to 10451\n",
      "Complement found from 10782 to 10788\n",
      "Complement found from 10854 to 10860\n",
      "Complement found from 11076 to 11082\n",
      "Complement found from 11565 to 11571\n",
      "Complement found from 12779 to 12785\n",
      "Complement found from 14295 to 14301\n",
      "Complement found from 14965 to 14971\n",
      "Complement found from 15508 to 15514\n",
      "Seq is: AATTAT\n",
      "14 AATTAT motifs detected in file\n",
      "\n",
      "Complement found from 284 to 290\n",
      "Complement found from 355 to 361\n",
      "Complement found from 2501 to 2507\n",
      "Complement found from 4603 to 4609\n",
      "Complement found from 4868 to 4874\n",
      "Complement found from 5280 to 5286\n",
      "Complement found from 7445 to 7451\n",
      "Complement found from 7513 to 7519\n",
      "Complement found from 9431 to 9437\n",
      "Complement found from 10046 to 10052\n",
      "Complement found from 10379 to 10385\n",
      "Complement found from 10812 to 10818\n",
      "Complement found from 12304 to 12310\n",
      "Complement found from 12416 to 12422\n",
      "Complement found from 13229 to 13235\n",
      "Complement found from 14073 to 14079\n",
      "Seq is: CAAAAA\n",
      "16 CAAAAA motifs detected in file\n",
      "\n",
      "Complement found from 209 to 215\n",
      "Complement found from 1858 to 1864\n",
      "Complement found from 2190 to 2196\n",
      "Complement found from 2368 to 2374\n",
      "Complement found from 4468 to 4474\n",
      "Complement found from 5638 to 5644\n",
      "Complement found from 8219 to 8225\n",
      "Complement found from 10023 to 10029\n",
      "Complement found from 14084 to 14090\n",
      "Complement found from 14499 to 14505\n",
      "Complement found from 14780 to 14786\n",
      "Seq is: AATTAA\n",
      "11 AATTAA motifs detected in file\n",
      "\n",
      "Complement found from 10001 to 10007\n",
      "Complement found from 10484 to 10490\n",
      "Complement found from 14292 to 14298\n",
      "Complement found from 14496 to 14502\n",
      "Complement found from 14592 to 14598\n",
      "Complement found from 16134 to 16140\n",
      "Seq is: ATAAAT\n",
      "6 ATAAAT motifs detected in file\n",
      "\n",
      "Complement found from 304 to 310\n",
      "Complement found from 955 to 961\n",
      "Complement found from 3566 to 3572\n",
      "Complement found from 3584 to 3590\n",
      "Complement found from 3892 to 3898\n",
      "Complement found from 4248 to 4254\n",
      "Complement found from 4317 to 4323\n",
      "Complement found from 4794 to 4800\n",
      "Complement found from 6419 to 6425\n",
      "Complement found from 8271 to 8277\n",
      "Complement found from 8280 to 8286\n",
      "Complement found from 8604 to 8610\n",
      "Complement found from 10935 to 10941\n",
      "Complement found from 10947 to 10953\n",
      "Complement found from 11511 to 11517\n",
      "Complement found from 11672 to 11678\n",
      "Complement found from 13127 to 13133\n",
      "Complement found from 13762 to 13768\n",
      "Complement found from 13781 to 13787\n",
      "Complement found from 14488 to 14494\n",
      "Complement found from 14769 to 14775\n",
      "Complement found from 15366 to 15372\n",
      "Complement found from 16183 to 16189\n",
      "Complement found from 16375 to 16381\n",
      "Seq is: CCCCCT\n",
      "24 CCCCCT motifs detected in file\n",
      "\n",
      "Complement found from 470 to 476\n",
      "Complement found from 1700 to 1706\n",
      "Complement found from 3439 to 3445\n",
      "Complement found from 4732 to 4738\n",
      "Complement found from 5104 to 5110\n",
      "Complement found from 5118 to 5124\n",
      "Complement found from 5151 to 5157\n",
      "Complement found from 6525 to 6531\n",
      "Complement found from 6680 to 6686\n",
      "Complement found from 6999 to 7005\n",
      "Complement found from 7013 to 7019\n",
      "Complement found from 8158 to 8164\n",
      "Complement found from 8379 to 8385\n",
      "Complement found from 10101 to 10107\n",
      "Complement found from 10790 to 10796\n",
      "Complement found from 11288 to 11294\n",
      "Complement found from 14211 to 14217\n",
      "Complement found from 15412 to 15418\n",
      "Seq is: TACTAC\n",
      "18 TACTAC motifs detected in file\n",
      "\n",
      "Complement found from 1107 to 1113\n",
      "Complement found from 9558 to 9564\n",
      "Complement found from 12177 to 12183\n",
      "Complement found from 14649 to 14655\n",
      "Complement found from 15119 to 15125\n",
      "Seq is: CAACAG\n",
      "5 CAACAG motifs detected in file\n",
      "\n",
      "Complement found from 1101 to 1107\n",
      "Complement found from 1673 to 1679\n",
      "Complement found from 1760 to 1766\n",
      "Complement found from 1992 to 1998\n",
      "Complement found from 2132 to 2138\n",
      "Complement found from 2789 to 2795\n",
      "Complement found from 4255 to 4261\n",
      "Complement found from 7122 to 7128\n",
      "Complement found from 7372 to 7378\n",
      "Complement found from 8825 to 8831\n",
      "Complement found from 10747 to 10753\n",
      "Complement found from 13973 to 13979\n",
      "Complement found from 15077 to 15083\n",
      "Complement found from 15934 to 15940\n",
      "Complement found from 16282 to 16288\n",
      "Seq is: AAACCT\n",
      "15 AAACCT motifs detected in file\n",
      "\n",
      "Complement found from 3728 to 3734\n",
      "Complement found from 3855 to 3861\n",
      "Complement found from 4043 to 4049\n",
      "Complement found from 4209 to 4215\n",
      "Complement found from 5804 to 5810\n",
      "Complement found from 6894 to 6900\n",
      "Complement found from 11341 to 11347\n",
      "Complement found from 14705 to 14711\n",
      "Seq is: ATATGA\n",
      "8 ATATGA motifs detected in file\n",
      "\n",
      "Complement found from 116 to 122\n",
      "Complement found from 9140 to 9146\n",
      "Complement found from 12452 to 12458\n",
      "Seq is: TGTCGC\n",
      "3 TGTCGC motifs detected in file\n",
      "\n",
      "Complement found from 1997 to 2003\n",
      "Complement found from 3371 to 3377\n",
      "Complement found from 12202 to 12208\n",
      "Seq is: TACCGA\n",
      "3 TACCGA motifs detected in file\n",
      "\n",
      "Complement found from 12227 to 12233\n",
      "Complement found from 15971 to 15977\n",
      "Seq is: TAACTC\n",
      "2 TAACTC motifs detected in file\n",
      "\n",
      "Complement found from 2440 to 2446\n",
      "Complement found from 3725 to 3731\n",
      "Complement found from 4158 to 4164\n",
      "Complement found from 7666 to 7672\n",
      "Complement found from 9332 to 9338\n",
      "Complement found from 10544 to 10550\n",
      "Complement found from 10604 to 10610\n",
      "Complement found from 11701 to 11707\n",
      "Complement found from 12425 to 12431\n",
      "Complement found from 13269 to 13275\n",
      "Seq is: CTCATA\n",
      "10 CTCATA motifs detected in file\n",
      "\n",
      "Complement found from 8781 to 8787\n",
      "Complement found from 11389 to 11395\n",
      "Complement found from 11947 to 11953\n",
      "Complement found from 13266 to 13272\n",
      "Seq is: GGACTC\n",
      "4 GGACTC motifs detected in file\n",
      "\n",
      "Seq is: CGTACC\n",
      "0 CGTACC motifs detected in file\n",
      "\n",
      "Complement found from 2546 to 2552\n",
      "Complement found from 6499 to 6505\n",
      "Complement found from 13058 to 13064\n",
      "Seq is: CCCAGT\n",
      "3 CCCAGT motifs detected in file\n",
      "\n",
      "Complement found from 352 to 358\n",
      "Complement found from 434 to 440\n",
      "Complement found from 556 to 562\n",
      "Complement found from 961 to 967\n",
      "Complement found from 1681 to 1687\n",
      "Complement found from 3413 to 3419\n",
      "Complement found from 7467 to 7473\n",
      "Complement found from 8368 to 8374\n",
      "Complement found from 9533 to 9539\n",
      "Complement found from 9555 to 9561\n",
      "Complement found from 10916 to 10922\n",
      "Complement found from 12301 to 12307\n",
      "Complement found from 14531 to 14537\n",
      "Complement found from 14750 to 14756\n",
      "Seq is: CCCCAA\n",
      "14 CCCCAA motifs detected in file\n",
      "\n",
      "Complement found from 998 to 1004\n",
      "Complement found from 4238 to 4244\n",
      "Complement found from 5134 to 5140\n",
      "Complement found from 9514 to 9520\n",
      "Seq is: CTCCAG\n",
      "4 CTCCAG motifs detected in file\n",
      "\n",
      "Complement found from 23 to 29\n",
      "Complement found from 5933 to 5939\n",
      "Complement found from 8127 to 8133\n",
      "Complement found from 8191 to 8197\n",
      "Complement found from 8807 to 8813\n",
      "Complement found from 10839 to 10845\n",
      "Complement found from 11018 to 11024\n",
      "Complement found from 11127 to 11133\n",
      "Complement found from 11907 to 11913\n",
      "Complement found from 12356 to 12362\n",
      "Complement found from 13297 to 13303\n",
      "Complement found from 14331 to 14337\n",
      "Complement found from 14691 to 14697\n",
      "Complement found from 14788 to 14794\n",
      "Seq is: AACCAC\n",
      "14 AACCAC motifs detected in file\n",
      "\n",
      "Complement found from 3153 to 3159\n",
      "Complement found from 3802 to 3808\n",
      "Complement found from 5277 to 5283\n",
      "Complement found from 6396 to 6402\n",
      "Complement found from 7721 to 7727\n",
      "Complement found from 8050 to 8056\n",
      "Complement found from 9103 to 9109\n",
      "Complement found from 10982 to 10988\n",
      "Complement found from 11259 to 11265\n",
      "Complement found from 12214 to 12220\n",
      "Complement found from 15821 to 15827\n",
      "Seq is: TCACAA\n",
      "11 TCACAA motifs detected in file\n",
      "\n",
      "Complement found from 730 to 736\n",
      "Complement found from 2599 to 2605\n",
      "Complement found from 5371 to 5377\n",
      "Complement found from 5811 to 5817\n",
      "Complement found from 6469 to 6475\n",
      "Complement found from 8651 to 8657\n",
      "Complement found from 9216 to 9222\n",
      "Complement found from 9634 to 9640\n",
      "Complement found from 14138 to 14144\n",
      "Complement found from 14877 to 14883\n",
      "Complement found from 15375 to 15381\n",
      "Complement found from 15396 to 15402\n",
      "Complement found from 15710 to 15716\n",
      "Seq is: AATCAC\n",
      "13 AATCAC motifs detected in file\n",
      "\n",
      "Complement found from 962 to 968\n",
      "Complement found from 2260 to 2266\n",
      "Complement found from 2377 to 2383\n",
      "Complement found from 6432 to 6438\n",
      "Complement found from 9534 to 9540\n",
      "Complement found from 13009 to 13015\n",
      "Complement found from 14751 to 14757\n",
      "Complement found from 16166 to 16172\n",
      "Seq is: CCCAAT\n",
      "8 CCCAAT motifs detected in file\n",
      "\n",
      "Complement found from 4146 to 4152\n",
      "Complement found from 5469 to 5475\n",
      "Complement found from 6823 to 6829\n",
      "Complement found from 7617 to 7623\n",
      "Complement found from 8139 to 8145\n",
      "Complement found from 13577 to 13583\n",
      "Complement found from 14983 to 14989\n",
      "Seq is: CGCTAC\n",
      "7 CGCTAC motifs detected in file\n",
      "\n",
      "Complement found from 40 to 46\n",
      "Complement found from 4220 to 4226\n",
      "Complement found from 7366 to 7372\n",
      "Complement found from 7499 to 7505\n",
      "Complement found from 9319 to 9325\n",
      "Complement found from 9967 to 9973\n",
      "Complement found from 10210 to 10216\n",
      "Complement found from 11577 to 11583\n",
      "Complement found from 12590 to 12596\n",
      "Complement found from 14054 to 14060\n",
      "Complement found from 14376 to 14382\n",
      "Complement found from 15664 to 15670\n",
      "Seq is: CTCCAT\n",
      "12 CTCCAT motifs detected in file\n",
      "\n",
      "Complement found from 369 to 375\n",
      "Complement found from 380 to 386\n",
      "Complement found from 2577 to 2583\n",
      "Complement found from 2760 to 2766\n",
      "Complement found from 5060 to 5066\n",
      "Complement found from 5098 to 5104\n",
      "Complement found from 5838 to 5844\n",
      "Complement found from 6334 to 6340\n",
      "Complement found from 7713 to 7719\n",
      "Complement found from 9002 to 9008\n",
      "Complement found from 9256 to 9262\n",
      "Complement found from 10938 to 10944\n",
      "Complement found from 12362 to 12368\n",
      "Complement found from 12411 to 12417\n",
      "Complement found from 13625 to 13631\n",
      "Complement found from 13853 to 13859\n",
      "Complement found from 13997 to 14003\n",
      "Complement found from 14123 to 14129\n",
      "Complement found from 15598 to 15604\n",
      "Seq is: CCTAAC\n",
      "19 CCTAAC motifs detected in file\n",
      "\n",
      "Complement found from 5143 to 5149\n",
      "Complement found from 7004 to 7010\n",
      "Complement found from 8144 to 8150\n",
      "Complement found from 9192 to 9198\n",
      "Complement found from 14694 to 14700\n",
      "Seq is: CACGAC\n",
      "5 CACGAC motifs detected in file\n",
      "\n",
      "Complement found from 1308 to 1314\n",
      "Complement found from 1687 to 1693\n",
      "Complement found from 2215 to 2221\n",
      "Complement found from 2768 to 2774\n",
      "Complement found from 3202 to 3208\n",
      "Complement found from 3208 to 3214\n",
      "Complement found from 5021 to 5027\n",
      "Complement found from 5431 to 5437\n",
      "Complement found from 5675 to 5681\n",
      "Complement found from 9209 to 9215\n",
      "Complement found from 10619 to 10625\n",
      "Complement found from 10843 to 10849\n",
      "Complement found from 12016 to 12022\n",
      "Complement found from 12925 to 12931\n",
      "Complement found from 13326 to 13332\n",
      "Complement found from 14356 to 14362\n",
      "Complement found from 14640 to 14646\n",
      "Complement found from 16264 to 16270\n",
      "Complement found from 16288 to 16294\n",
      "Seq is: ACCCAC\n",
      "19 ACCCAC motifs detected in file\n",
      "\n",
      "Complement found from 7190 to 7196\n",
      "Complement found from 7281 to 7287\n",
      "Complement found from 10207 to 10213\n",
      "Complement found from 12258 to 12264\n",
      "Complement found from 13731 to 13737\n",
      "Complement found from 13898 to 13904\n",
      "Complement found from 15064 to 15070\n",
      "Seq is: TTTCTC\n",
      "7 TTTCTC motifs detected in file\n",
      "\n",
      "Complement found from 469 to 475\n",
      "Complement found from 859 to 865\n",
      "Complement found from 4731 to 4737\n",
      "Complement found from 5495 to 5501\n",
      "Complement found from 6524 to 6530\n",
      "Complement found from 7739 to 7745\n",
      "Complement found from 8157 to 8163\n",
      "Complement found from 8378 to 8384\n",
      "Complement found from 8736 to 8742\n",
      "Complement found from 8943 to 8949\n",
      "Complement found from 9335 to 9341\n",
      "Complement found from 10493 to 10499\n",
      "Complement found from 10523 to 10529\n",
      "Complement found from 10577 to 10583\n",
      "Complement found from 10655 to 10661\n",
      "Complement found from 10957 to 10963\n",
      "Complement found from 11050 to 11056\n",
      "Complement found from 11956 to 11962\n",
      "Complement found from 12717 to 12723\n",
      "Complement found from 13347 to 13353\n",
      "Complement found from 14579 to 14585\n",
      "Seq is: ATACTA\n",
      "21 ATACTA motifs detected in file\n",
      "\n",
      "Complement found from 569 to 575\n",
      "Complement found from 802 to 808\n",
      "Complement found from 1082 to 1088\n",
      "Complement found from 5302 to 5308\n",
      "Complement found from 5446 to 5452\n",
      "Complement found from 5600 to 5606\n",
      "Complement found from 5895 to 5901\n",
      "Complement found from 6844 to 6850\n",
      "Complement found from 7398 to 7404\n",
      "Complement found from 8079 to 8085\n",
      "Complement found from 8558 to 8564\n",
      "Complement found from 8618 to 8624\n",
      "Complement found from 11139 to 11145\n",
      "Complement found from 11868 to 11874\n",
      "Complement found from 12482 to 12488\n",
      "Complement found from 12967 to 12973\n",
      "Complement found from 13052 to 13058\n",
      "Complement found from 13646 to 13652\n",
      "Complement found from 13676 to 13682\n",
      "Complement found from 13879 to 13885\n",
      "Complement found from 14388 to 14394\n",
      "Complement found from 14619 to 14625\n",
      "Complement found from 14808 to 14814\n",
      "Complement found from 15541 to 15547\n",
      "Seq is: CCCCAC\n",
      "24 CCCCAC motifs detected in file\n",
      "\n",
      "Complement found from 295 to 301\n",
      "Complement found from 2186 to 2192\n",
      "Complement found from 3473 to 3479\n",
      "Complement found from 4085 to 4091\n",
      "Complement found from 7888 to 7894\n",
      "Complement found from 8477 to 8483\n",
      "Complement found from 8803 to 8809\n",
      "Complement found from 9212 to 9218\n",
      "Complement found from 14040 to 14046\n",
      "Complement found from 14183 to 14189\n",
      "Complement found from 14249 to 14255\n",
      "Complement found from 14364 to 14370\n",
      "Complement found from 14403 to 14409\n",
      "Complement found from 14741 to 14747\n",
      "Seq is: CACCAA\n",
      "14 CACCAA motifs detected in file\n",
      "\n",
      "Complement found from 3542 to 3548\n",
      "Complement found from 5305 to 5311\n",
      "Complement found from 5317 to 5323\n",
      "Complement found from 6770 to 6776\n",
      "Complement found from 8394 to 8400\n",
      "Complement found from 9647 to 9653\n",
      "Complement found from 9763 to 9769\n",
      "Complement found from 13454 to 13460\n",
      "Complement found from 15977 to 15983\n",
      "Complement found from 16111 to 16117\n",
      "Complement found from 16400 to 16406\n",
      "Seq is: CACCAT\n",
      "11 CACCAT motifs detected in file\n",
      "\n",
      "Complement found from 1757 to 1763\n",
      "Complement found from 8240 to 8246\n",
      "Complement found from 15860 to 15866\n",
      "Seq is: TTGAAA\n",
      "3 TTGAAA motifs detected in file\n",
      "\n",
      "Complement found from 3799 to 3805\n",
      "Complement found from 6405 to 6411\n",
      "Complement found from 7643 to 7649\n",
      "Complement found from 9094 to 9100\n",
      "Complement found from 12469 to 12475\n",
      "Seq is: TTATCA\n",
      "5 TTATCA motifs detected in file\n",
      "\n",
      "Complement found from 146 to 152\n",
      "Complement found from 501 to 507\n",
      "Complement found from 4393 to 4399\n",
      "Complement found from 7255 to 7261\n",
      "Complement found from 7797 to 7803\n",
      "Complement found from 7830 to 7836\n",
      "Complement found from 10345 to 10351\n",
      "Complement found from 11723 to 11729\n",
      "Complement found from 12389 to 12395\n",
      "Complement found from 12881 to 12887\n",
      "Complement found from 14120 to 14126\n",
      "Complement found from 15636 to 15642\n",
      "Complement found from 15642 to 15648\n",
      "Complement found from 15660 to 15666\n",
      "Complement found from 16403 to 16409\n",
      "Seq is: CATCCT\n",
      "15 CATCCT motifs detected in file\n",
      "\n",
      "Complement found from 10434 to 10440\n",
      "Seq is: TCGACT\n",
      "1 TCGACT motifs detected in file\n",
      "\n",
      "Complement found from 19 to 25\n",
      "Complement found from 2743 to 2749\n",
      "Complement found from 3023 to 3029\n",
      "Complement found from 7299 to 7305\n",
      "Complement found from 8447 to 8453\n",
      "Complement found from 10335 to 10341\n",
      "Complement found from 11874 to 11880\n",
      "Complement found from 14315 to 14321\n",
      "Complement found from 14511 to 14517\n",
      "Seq is: TATTAA\n",
      "9 TATTAA motifs detected in file\n",
      "\n",
      "Complement found from 190 to 196\n",
      "Complement found from 1504 to 1510\n",
      "Complement found from 5565 to 5571\n",
      "Complement found from 7948 to 7954\n",
      "Complement found from 11199 to 11205\n",
      "Complement found from 15816 to 15822\n",
      "Complement found from 16138 to 16144\n",
      "Seq is: ATACTT\n",
      "7 ATACTT motifs detected in file\n",
      "\n",
      "Complement found from 3788 to 3794\n",
      "Complement found from 5212 to 5218\n",
      "Complement found from 6222 to 6228\n",
      "Complement found from 9268 to 9274\n",
      "Complement found from 11043 to 11049\n",
      "Complement found from 11787 to 11793\n",
      "Complement found from 13434 to 13440\n",
      "Complement found from 14099 to 14105\n",
      "Complement found from 14282 to 14288\n",
      "Seq is: CCTCTC\n",
      "9 CCTCTC motifs detected in file\n",
      "\n",
      "Complement found from 2546 to 2552\n",
      "Complement found from 6499 to 6505\n",
      "Complement found from 13058 to 13064\n",
      "Seq is: CCCAGT\n",
      "3 CCCAGT motifs detected in file\n",
      "\n",
      "Complement found from 435 to 441\n",
      "Complement found from 1625 to 1631\n",
      "Complement found from 2425 to 2431\n",
      "Complement found from 2833 to 2839\n",
      "Complement found from 3414 to 3420\n",
      "Complement found from 3579 to 3585\n",
      "Complement found from 4482 to 4488\n",
      "Complement found from 8369 to 8375\n",
      "Complement found from 8435 to 8441\n",
      "Complement found from 8659 to 8665\n",
      "Complement found from 8812 to 8818\n",
      "Complement found from 9556 to 9562\n",
      "Complement found from 10776 to 10782\n",
      "Complement found from 10917 to 10923\n",
      "Seq is: CCCAAC\n",
      "14 CCCAAC motifs detected in file\n",
      "\n",
      "Complement found from 2335 to 2341\n",
      "Complement found from 4920 to 4926\n",
      "Complement found from 5646 to 5652\n",
      "Complement found from 6000 to 6006\n",
      "Complement found from 9178 to 9184\n",
      "Complement found from 15704 to 15710\n",
      "Seq is: TAAGCC\n",
      "6 TAAGCC motifs detected in file\n",
      "\n",
      "Complement found from 3321 to 3327\n",
      "Complement found from 3609 to 3615\n",
      "Complement found from 4167 to 4173\n",
      "Complement found from 5474 to 5480\n",
      "Complement found from 6227 to 6233\n",
      "Complement found from 6488 to 6494\n",
      "Complement found from 7941 to 7947\n",
      "Complement found from 9278 to 9284\n",
      "Complement found from 9596 to 9602\n",
      "Complement found from 9728 to 9734\n",
      "Complement found from 10091 to 10097\n",
      "Complement found from 10951 to 10957\n",
      "Complement found from 10972 to 10978\n",
      "Complement found from 11935 to 11941\n",
      "Complement found from 12094 to 12100\n",
      "Complement found from 12984 to 12990\n",
      "Complement found from 13989 to 13995\n",
      "Complement found from 14133 to 14139\n",
      "Complement found from 15334 to 15340\n",
      "Complement found from 15490 to 15496\n",
      "Complement found from 15724 to 15730\n",
      "Seq is: CTCCTA\n",
      "21 CTCCTA motifs detected in file\n",
      "\n",
      "Complement found from 2932 to 2938\n",
      "Complement found from 9450 to 9456\n",
      "Complement found from 12274 to 12280\n",
      "Seq is: GGATAA\n",
      "3 GGATAA motifs detected in file\n",
      "\n",
      "Complement found from 5466 to 5472\n",
      "Complement found from 13328 to 13334\n",
      "Seq is: CCACGC\n",
      "2 CCACGC motifs detected in file\n",
      "\n",
      "Complement found from 540 to 546\n",
      "Complement found from 1222 to 1228\n",
      "Complement found from 1901 to 1907\n",
      "Complement found from 3526 to 3532\n",
      "Complement found from 4137 to 4143\n",
      "Complement found from 4435 to 4441\n",
      "Complement found from 6169 to 6175\n",
      "Complement found from 7211 to 7217\n",
      "Complement found from 7231 to 7237\n",
      "Complement found from 12108 to 12114\n",
      "Complement found from 14155 to 14161\n",
      "Seq is: CCCCGA\n",
      "11 CCCCGA motifs detected in file\n",
      "\n",
      "Complement found from 187 to 193\n",
      "Complement found from 1035 to 1041\n",
      "Complement found from 2237 to 2243\n",
      "Complement found from 3303 to 3309\n",
      "Complement found from 4040 to 4046\n",
      "Complement found from 4073 to 4079\n",
      "Complement found from 5063 to 5069\n",
      "Complement found from 5421 to 5427\n",
      "Complement found from 6197 to 6203\n",
      "Complement found from 11953 to 11959\n",
      "Complement found from 12031 to 12037\n",
      "Complement found from 13533 to 13539\n",
      "Complement found from 13905 to 13911\n",
      "Seq is: AACATA\n",
      "13 AACATA motifs detected in file\n",
      "\n",
      "Complement found from 3768 to 3774\n",
      "Complement found from 4679 to 4685\n",
      "Complement found from 5498 to 5504\n",
      "Complement found from 6155 to 6161\n",
      "Complement found from 7349 to 7355\n",
      "Complement found from 7736 to 7742\n",
      "Complement found from 9656 to 9662\n",
      "Complement found from 9872 to 9878\n",
      "Complement found from 10106 to 10112\n",
      "Complement found from 10313 to 10319\n",
      "Complement found from 10954 to 10960\n",
      "Complement found from 11818 to 11824\n",
      "Complement found from 14773 to 14779\n",
      "Complement found from 15838 to 15844\n",
      "Complement found from 15903 to 15909\n",
      "Seq is: CTAATA\n",
      "15 CTAATA motifs detected in file\n",
      "\n",
      "Complement found from 6186 to 6192\n",
      "Complement found from 13755 to 13761\n",
      "Complement found from 14245 to 14251\n",
      "Complement found from 16424 to 16430\n",
      "Seq is: CCCGCA\n",
      "4 CCCGCA motifs detected in file\n",
      "\n",
      "Complement found from 474 to 480\n",
      "Complement found from 2293 to 2299\n",
      "Complement found from 3767 to 3773\n",
      "Complement found from 4465 to 4471\n",
      "Complement found from 5497 to 5503\n",
      "Complement found from 7735 to 7741\n",
      "Complement found from 7929 to 7935\n",
      "Complement found from 8648 to 8654\n",
      "Complement found from 8669 to 8675\n",
      "Complement found from 9871 to 9877\n",
      "Complement found from 10105 to 10111\n",
      "Complement found from 10312 to 10318\n",
      "Complement found from 10767 to 10773\n",
      "Complement found from 11052 to 11058\n",
      "Complement found from 11100 to 11106\n",
      "Complement found from 11247 to 11253\n",
      "Complement found from 11817 to 11823\n",
      "Complement found from 12719 to 12725\n",
      "Complement found from 13148 to 13154\n",
      "Complement found from 14215 to 14221\n",
      "Complement found from 15902 to 15908\n",
      "Seq is: ACTAAT\n",
      "21 ACTAAT motifs detected in file\n",
      "\n",
      "Complement found from 276 to 282\n",
      "Complement found from 3746 to 3752\n",
      "Complement found from 3835 to 3841\n",
      "Complement found from 4749 to 4755\n",
      "Complement found from 4951 to 4957\n",
      "Complement found from 5296 to 5302\n",
      "Complement found from 5308 to 5314\n",
      "Complement found from 6118 to 6124\n",
      "Complement found from 7066 to 7072\n",
      "Complement found from 7794 to 7800\n",
      "Complement found from 10342 to 10348\n",
      "Complement found from 10863 to 10869\n",
      "Complement found from 11778 to 11784\n",
      "Complement found from 12114 to 12120\n",
      "Complement found from 12632 to 12638\n",
      "Complement found from 13370 to 13376\n",
      "Complement found from 13517 to 13523\n",
      "Complement found from 14057 to 14063\n",
      "Complement found from 14342 to 14348\n",
      "Complement found from 14481 to 14487\n",
      "Complement found from 14667 to 14673\n",
      "Complement found from 15789 to 15795\n",
      "Seq is: CATCAT\n",
      "22 CATCAT motifs detected in file\n",
      "\n",
      "Complement found from 4715 to 4721\n",
      "Complement found from 8664 to 8670\n",
      "Complement found from 9371 to 9377\n",
      "Complement found from 13388 to 13394\n",
      "Complement found from 14700 to 14706\n",
      "Complement found from 14744 to 14750\n",
      "Complement found from 15229 to 15235\n",
      "Seq is: CAATGA\n",
      "7 CAATGA motifs detected in file\n",
      "\n",
      "Complement found from 436 to 442\n",
      "Complement found from 1626 to 1632\n",
      "Complement found from 4154 to 4160\n",
      "Complement found from 8370 to 8376\n",
      "Complement found from 8436 to 8442\n",
      "Complement found from 8813 to 8819\n",
      "Complement found from 9868 to 9874\n",
      "Complement found from 12749 to 12755\n",
      "Complement found from 12913 to 12919\n",
      "Complement found from 15844 to 15850\n",
      "Seq is: CCAACT\n",
      "10 CCAACT motifs detected in file\n",
      "\n",
      "Complement found from 5577 to 5583\n",
      "Complement found from 6673 to 6679\n",
      "Complement found from 9385 to 9391\n",
      "Seq is: TGTAAC\n",
      "3 TGTAAC motifs detected in file\n",
      "\n",
      "Complement found from 358 to 364\n",
      "Complement found from 642 to 648\n",
      "Complement found from 3718 to 3724\n",
      "Complement found from 5170 to 5176\n",
      "Complement found from 5284 to 5290\n",
      "Complement found from 6193 to 6199\n",
      "Complement found from 8688 to 8694\n",
      "Complement found from 9664 to 9670\n",
      "Complement found from 10296 to 10302\n",
      "Complement found from 10417 to 10423\n",
      "Complement found from 12550 to 12556\n",
      "Complement found from 13771 to 13777\n",
      "Complement found from 14026 to 14032\n",
      "Complement found from 14190 to 14196\n",
      "Complement found from 14655 to 14661\n",
      "Complement found from 15360 to 15366\n",
      "Complement found from 15676 to 15682\n",
      "Complement found from 15864 to 15870\n",
      "Seq is: AAACAA\n",
      "18 AAACAA motifs detected in file\n",
      "\n",
      "Complement found from 3496 to 3502\n",
      "Complement found from 5023 to 5029\n",
      "Complement found from 6055 to 6061\n",
      "Complement found from 7245 to 7251\n",
      "Complement found from 8081 to 8087\n",
      "Complement found from 11634 to 11640\n",
      "Complement found from 12021 to 12027\n",
      "Complement found from 13514 to 13520\n",
      "Complement found from 14946 to 14952\n",
      "Complement found from 15543 to 15549\n",
      "Complement found from 16172 to 16178\n",
      "Seq is: CCACAT\n",
      "11 CCACAT motifs detected in file\n",
      "\n",
      "Complement found from 2240 to 2246\n",
      "Complement found from 6411 to 6417\n",
      "Complement found from 12655 to 12661\n",
      "Complement found from 14521 to 14527\n",
      "Seq is: ATATAA\n",
      "4 ATATAA motifs detected in file\n",
      "\n",
      "Complement found from 2796 to 2802\n",
      "Complement found from 4752 to 4758\n",
      "Complement found from 8313 to 8319\n",
      "Complement found from 9074 to 9080\n",
      "Complement found from 10440 to 10446\n",
      "Complement found from 12024 to 12030\n",
      "Complement found from 12674 to 12680\n",
      "Complement found from 13661 to 13667\n",
      "Complement found from 13694 to 13700\n",
      "Complement found from 15467 to 15473\n",
      "Seq is: CATTAA\n",
      "10 CATTAA motifs detected in file\n",
      "\n",
      "Complement found from 725 to 731\n",
      "Complement found from 2067 to 2073\n",
      "Complement found from 5997 to 6003\n",
      "Complement found from 7284 to 7290\n",
      "Complement found from 8879 to 8885\n",
      "Complement found from 13159 to 13165\n",
      "Seq is: CTCTAA\n",
      "6 CTCTAA motifs detected in file\n",
      "\n",
      "Complement found from 1145 to 1151\n",
      "Complement found from 2000 to 2006\n",
      "Complement found from 6014 to 6020\n",
      "Complement found from 13966 to 13972\n",
      "Seq is: CGAGCC\n",
      "4 CGAGCC motifs detected in file\n",
      "\n",
      "Complement found from 1393 to 1399\n",
      "Complement found from 3796 to 3802\n",
      "Complement found from 4319 to 4325\n",
      "Complement found from 5192 to 5198\n",
      "Complement found from 5460 to 5466\n",
      "Complement found from 5650 to 5656\n",
      "Complement found from 6293 to 6299\n",
      "Complement found from 7865 to 7871\n",
      "Complement found from 8844 to 8850\n",
      "Complement found from 8932 to 8938\n",
      "Complement found from 10162 to 10168\n",
      "Complement found from 12194 to 12200\n",
      "Complement found from 13210 to 13216\n",
      "Complement found from 13651 to 13657\n",
      "Complement found from 15408 to 15414\n",
      "Complement found from 15526 to 15532\n",
      "Complement found from 16293 to 16299\n",
      "Complement found from 16546 to 16552\n",
      "Seq is: CCCTTA\n",
      "18 CCCTTA motifs detected in file\n",
      "\n",
      "Complement found from 2897 to 2903\n",
      "Complement found from 5864 to 5870\n",
      "Complement found from 10020 to 10026\n",
      "Complement found from 10755 to 10761\n",
      "Seq is: TCCAAT\n",
      "4 TCCAAT motifs detected in file\n",
      "\n",
      "Complement found from 1139 to 1145\n",
      "Complement found from 2121 to 2127\n",
      "Complement found from 3870 to 3876\n",
      "Complement found from 5375 to 5381\n",
      "Complement found from 5951 to 5957\n",
      "Complement found from 8419 to 8425\n",
      "Complement found from 8710 to 8716\n",
      "Complement found from 9356 to 9362\n",
      "Complement found from 12346 to 12352\n",
      "Complement found from 13164 to 13170\n",
      "Complement found from 14311 to 14317\n",
      "Complement found from 15472 to 15478\n",
      "Seq is: ACACTA\n",
      "12 ACACTA motifs detected in file\n",
      "\n",
      "Complement found from 3651 to 3657\n",
      "Complement found from 4688 to 4694\n",
      "Complement found from 10549 to 10555\n",
      "Complement found from 11724 to 11730\n",
      "Complement found from 11785 to 11791\n",
      "Complement found from 12882 to 12888\n",
      "Complement found from 14259 to 14265\n",
      "Complement found from 14869 to 14875\n",
      "Complement found from 15097 to 15103\n",
      "Complement found from 15637 to 15643\n",
      "Complement found from 15661 to 15667\n",
      "Complement found from 16404 to 16410\n",
      "Seq is: ATCCTC\n",
      "12 ATCCTC motifs detected in file\n",
      "\n",
      "Complement found from 2231 to 2237\n",
      "Complement found from 15202 to 15208\n",
      "Seq is: ATCCCA\n",
      "2 ATCCCA motifs detected in file\n",
      "\n",
      "Complement found from 147 to 153\n",
      "Complement found from 502 to 508\n",
      "Complement found from 2945 to 2951\n",
      "Complement found from 4394 to 4400\n",
      "Complement found from 5096 to 5102\n",
      "Complement found from 6641 to 6647\n",
      "Complement found from 7256 to 7262\n",
      "Complement found from 7570 to 7576\n",
      "Complement found from 7798 to 7804\n",
      "Complement found from 8565 to 8571\n",
      "Complement found from 9126 to 9132\n",
      "Complement found from 9455 to 9461\n",
      "Complement found from 10346 to 10352\n",
      "Complement found from 12849 to 12855\n",
      "Complement found from 12903 to 12909\n",
      "Complement found from 14121 to 14127\n",
      "Complement found from 14369 to 14375\n",
      "Complement found from 15643 to 15649\n",
      "Complement found from 15829 to 15835\n",
      "Complement found from 15835 to 15841\n",
      "Seq is: ATCCTA\n",
      "20 ATCCTA motifs detected in file\n",
      "\n",
      "Complement found from 978 to 984\n",
      "Complement found from 1233 to 1239\n",
      "Complement found from 3540 to 3546\n",
      "Complement found from 4014 to 4020\n",
      "Complement found from 4191 to 4197\n",
      "Complement found from 5890 to 5896\n",
      "Complement found from 8475 to 8481\n",
      "Complement found from 9645 to 9651\n",
      "Complement found from 12013 to 12019\n",
      "Complement found from 12963 to 12969\n",
      "Complement found from 13452 to 13458\n",
      "Complement found from 13620 to 13626\n",
      "Complement found from 14401 to 14407\n",
      "Complement found from 14910 to 14916\n",
      "Complement found from 15480 to 15486\n",
      "Complement found from 16066 to 16072\n",
      "Complement found from 16261 to 16267\n",
      "Seq is: CTCACC\n",
      "17 CTCACC motifs detected in file\n",
      "\n",
      "Complement found from 3294 to 3300\n",
      "Complement found from 4853 to 4859\n",
      "Complement found from 5783 to 5789\n",
      "Complement found from 6100 to 6106\n",
      "Complement found from 6556 to 6562\n",
      "Complement found from 8906 to 8912\n",
      "Complement found from 10222 to 10228\n",
      "Complement found from 10240 to 10246\n",
      "Complement found from 11118 to 11124\n",
      "Complement found from 12788 to 12794\n",
      "Complement found from 13106 to 13112\n",
      "Complement found from 13334 to 13340\n",
      "Complement found from 13959 to 13965\n",
      "Complement found from 14108 to 14114\n",
      "Seq is: CTTCTT\n",
      "14 CTTCTT motifs detected in file\n",
      "\n",
      "Complement found from 665 to 671\n",
      "Complement found from 6937 to 6943\n",
      "Complement found from 7173 to 7179\n",
      "Complement found from 7189 to 7195\n",
      "Complement found from 10206 to 10212\n",
      "Complement found from 12257 to 12263\n",
      "Complement found from 14104 to 14110\n",
      "Seq is: CTTTCT\n",
      "7 CTTTCT motifs detected in file\n",
      "\n",
      "Complement found from 3779 to 3785\n",
      "Complement found from 5327 to 5333\n",
      "Complement found from 6005 to 6011\n",
      "Complement found from 6346 to 6352\n",
      "Complement found from 7987 to 7993\n",
      "Complement found from 8413 to 8419\n",
      "Complement found from 10268 to 10274\n",
      "Complement found from 11070 to 11076\n",
      "Complement found from 14285 to 14291\n",
      "Complement found from 14851 to 14857\n",
      "Complement found from 15456 to 15462\n",
      "Seq is: CTCCTT\n",
      "11 CTCCTT motifs detected in file\n",
      "\n",
      "Complement found from 7139 to 7145\n",
      "Complement found from 7279 to 7285\n",
      "Complement found from 7533 to 7539\n",
      "Complement found from 9766 to 9772\n",
      "Complement found from 13748 to 13754\n",
      "Complement found from 15062 to 15068\n",
      "Seq is: CATTTC\n",
      "6 CATTTC motifs detected in file\n",
      "\n",
      "Complement found from 1802 to 1808\n",
      "Complement found from 7556 to 7562\n",
      "Complement found from 8499 to 8505\n",
      "Complement found from 10444 to 10450\n",
      "Complement found from 14294 to 14300\n",
      "Complement found from 14498 to 14504\n",
      "Complement found from 14779 to 14785\n",
      "Complement found from 14964 to 14970\n",
      "Seq is: AAATTA\n",
      "8 AAATTA motifs detected in file\n",
      "\n",
      "Complement found from 9445 to 9451\n",
      "Seq is: ATACGG\n",
      "1 ATACGG motifs detected in file\n",
      "\n",
      "Complement found from 356 to 362\n",
      "Complement found from 993 to 999\n",
      "Complement found from 2130 to 2136\n",
      "Complement found from 2502 to 2508\n",
      "Complement found from 4176 to 4182\n",
      "Complement found from 4869 to 4875\n",
      "Complement found from 5282 to 5288\n",
      "Complement found from 7527 to 7533\n",
      "Complement found from 8094 to 8100\n",
      "Complement found from 10814 to 10820\n",
      "Complement found from 11033 to 11039\n",
      "Complement found from 12420 to 12426\n",
      "Complement found from 14505 to 14511\n",
      "Complement found from 14710 to 14716\n",
      "Complement found from 16161 to 16167\n",
      "Seq is: AAAAAC\n",
      "15 AAAAAC motifs detected in file\n",
      "\n",
      "Complement found from 436 to 442\n",
      "Complement found from 1626 to 1632\n",
      "Complement found from 4154 to 4160\n",
      "Complement found from 8370 to 8376\n",
      "Complement found from 8436 to 8442\n",
      "Complement found from 8813 to 8819\n",
      "Complement found from 9868 to 9874\n",
      "Complement found from 12749 to 12755\n",
      "Complement found from 12913 to 12919\n",
      "Complement found from 15844 to 15850\n",
      "Seq is: CCAACT\n",
      "10 CCAACT motifs detected in file\n",
      "\n",
      "Complement found from 1084 to 1090\n",
      "Complement found from 2217 to 2223\n",
      "Complement found from 4018 to 4024\n",
      "Complement found from 7033 to 7039\n",
      "Complement found from 10310 to 10316\n",
      "Complement found from 11020 to 11026\n",
      "Complement found from 11815 to 11821\n",
      "Complement found from 11870 to 11876\n",
      "Complement found from 12969 to 12975\n",
      "Complement found from 13146 to 13152\n",
      "Complement found from 13881 to 13887\n",
      "Complement found from 14390 to 14396\n",
      "Complement found from 15700 to 15706\n",
      "Complement found from 16266 to 16272\n",
      "Seq is: CCACTA\n",
      "14 CCACTA motifs detected in file\n",
      "\n",
      "Complement found from 1918 to 1924\n",
      "Complement found from 2220 to 2226\n",
      "Complement found from 5344 to 5350\n",
      "Complement found from 5477 to 5483\n",
      "Complement found from 8467 to 8473\n",
      "Complement found from 9185 to 9191\n",
      "Complement found from 10964 to 10970\n",
      "Complement found from 11040 to 11046\n",
      "Complement found from 13579 to 13585\n",
      "Complement found from 13787 to 13793\n",
      "Complement found from 13850 to 13856\n",
      "Complement found from 14372 to 14378\n",
      "Complement found from 14985 to 14991\n",
      "Seq is: CTACCT\n",
      "13 CTACCT motifs detected in file\n",
      "\n",
      "Complement found from 1237 to 1243\n",
      "Complement found from 1694 to 1700\n",
      "Complement found from 3625 to 3631\n",
      "Complement found from 4378 to 4384\n",
      "Complement found from 5364 to 5370\n",
      "Complement found from 6552 to 6558\n",
      "Complement found from 8463 to 8469\n",
      "Complement found from 8620 to 8626\n",
      "Complement found from 9026 to 9032\n",
      "Complement found from 9422 to 9428\n",
      "Complement found from 11141 to 11147\n",
      "Complement found from 12460 to 12466\n",
      "Complement found from 14050 to 14056\n",
      "Complement found from 15330 to 15336\n",
      "Complement found from 16146 to 16152\n",
      "Seq is: CCACCT\n",
      "15 CCACCT motifs detected in file\n",
      "\n",
      "Complement found from 1099 to 1105\n",
      "Complement found from 1671 to 1677\n",
      "Complement found from 2779 to 2785\n",
      "Complement found from 4913 to 4919\n",
      "Complement found from 8119 to 8125\n",
      "Complement found from 9599 to 9605\n",
      "Complement found from 10745 to 10751\n",
      "Complement found from 11278 to 11284\n",
      "Complement found from 12945 to 12951\n",
      "Complement found from 13686 to 13692\n",
      "Complement found from 14582 to 14588\n",
      "Complement found from 14636 to 14642\n",
      "Seq is: CTAAAC\n",
      "12 CTAAAC motifs detected in file\n",
      "\n",
      "Complement found from 5143 to 5149\n",
      "Complement found from 7004 to 7010\n",
      "Complement found from 8144 to 8150\n",
      "Complement found from 9192 to 9198\n",
      "Complement found from 14694 to 14700\n",
      "Seq is: CACGAC\n",
      "5 CACGAC motifs detected in file\n",
      "\n",
      "Complement found from 2044 to 2050\n",
      "Complement found from 3268 to 3274\n",
      "Complement found from 5633 to 5639\n",
      "Complement found from 9882 to 9888\n",
      "Complement found from 14090 to 14096\n",
      "Complement found from 15714 to 15720\n",
      "Seq is: ACTTTA\n",
      "6 ACTTTA motifs detected in file\n",
      "\n",
      "Complement found from 283 to 289\n",
      "Complement found from 1009 to 1015\n",
      "Complement found from 1121 to 1127\n",
      "Complement found from 4867 to 4873\n",
      "Complement found from 5279 to 5285\n",
      "Complement found from 5426 to 5432\n",
      "Complement found from 7444 to 7450\n",
      "Complement found from 7726 to 7732\n",
      "Complement found from 10378 to 10384\n",
      "Complement found from 10419 to 10425\n",
      "Complement found from 11518 to 11524\n",
      "Complement found from 12415 to 12421\n",
      "Complement found from 13217 to 13223\n",
      "Complement found from 15866 to 15872\n",
      "Seq is: ACAAAA\n",
      "14 ACAAAA motifs detected in file\n",
      "\n",
      "Complement found from 7137 to 7143\n",
      "Complement found from 12286 to 12292\n",
      "Complement found from 12447 to 12453\n",
      "Seq is: TCCATT\n",
      "3 TCCATT motifs detected in file\n",
      "\n",
      "Complement found from 3942 to 3948\n",
      "Complement found from 4342 to 4348\n",
      "Complement found from 5267 to 5273\n",
      "Complement found from 7459 to 7465\n",
      "Complement found from 8955 to 8961\n",
      "Complement found from 13521 to 13527\n",
      "Seq is: ATCGAA\n",
      "6 ATCGAA motifs detected in file\n",
      "\n",
      "Seq is: AAGATC\n",
      "0 AAGATC motifs detected in file\n",
      "\n",
      "Complement found from 2096 to 2102\n",
      "Complement found from 2957 to 2963\n",
      "Complement found from 5862 to 5868\n",
      "Seq is: AGTCCA\n",
      "3 AGTCCA motifs detected in file\n",
      "\n",
      "Complement found from 3084 to 3090\n",
      "Complement found from 11010 to 11016\n",
      "Seq is: ATCCAG\n",
      "2 ATCCAG motifs detected in file\n",
      "\n",
      "Complement found from 3651 to 3657\n",
      "Complement found from 4688 to 4694\n",
      "Complement found from 10549 to 10555\n",
      "Complement found from 11724 to 11730\n",
      "Complement found from 11785 to 11791\n",
      "Complement found from 12882 to 12888\n",
      "Complement found from 14259 to 14265\n",
      "Complement found from 14869 to 14875\n",
      "Complement found from 15097 to 15103\n",
      "Complement found from 15637 to 15643\n",
      "Complement found from 15661 to 15667\n",
      "Complement found from 16404 to 16410\n",
      "Seq is: ATCCTC\n",
      "12 ATCCTC motifs detected in file\n",
      "\n",
      "Complement found from 144 to 150\n",
      "Complement found from 480 to 486\n",
      "Complement found from 4520 to 4526\n",
      "Complement found from 4747 to 4753\n",
      "Complement found from 5294 to 5300\n",
      "Complement found from 5453 to 5459\n",
      "Complement found from 6982 to 6988\n",
      "Complement found from 7807 to 7813\n",
      "Complement found from 8428 to 8434\n",
      "Complement found from 8634 to 8640\n",
      "Complement found from 11239 to 11245\n",
      "Complement found from 11662 to 11668\n",
      "Complement found from 12699 to 12705\n",
      "Complement found from 12795 to 12801\n",
      "Complement found from 13113 to 13119\n",
      "Complement found from 13572 to 13578\n",
      "Complement found from 14118 to 14124\n",
      "Complement found from 15640 to 15646\n",
      "Seq is: CTCATC\n",
      "18 CTCATC motifs detected in file\n",
      "\n",
      "Complement found from 1485 to 1491\n",
      "Complement found from 1945 to 1951\n",
      "Complement found from 4487 to 4493\n",
      "Seq is: CCCGTC\n",
      "3 CCCGTC motifs detected in file\n",
      "\n",
      "Complement found from 779 to 785\n",
      "Complement found from 1122 to 1128\n",
      "Complement found from 5427 to 5433\n",
      "Complement found from 5595 to 5601\n",
      "Complement found from 7727 to 7733\n",
      "Complement found from 8686 to 8692\n",
      "Complement found from 10420 to 10426\n",
      "Complement found from 11519 to 11525\n",
      "Complement found from 12153 to 12159\n",
      "Complement found from 13424 to 13430\n",
      "Complement found from 13971 to 13977\n",
      "Complement found from 14760 to 14766\n",
      "Complement found from 16178 to 16184\n",
      "Seq is: CAAAAC\n",
      "13 CAAAAC motifs detected in file\n",
      "\n",
      "Complement found from 2842 to 2848\n",
      "Complement found from 3705 to 3711\n",
      "Complement found from 6806 to 6812\n",
      "Complement found from 12816 to 12822\n",
      "Complement found from 14158 to 14164\n",
      "Seq is: CGAGCA\n",
      "5 CGAGCA motifs detected in file\n",
      "\n",
      "Complement found from 3972 to 3978\n",
      "Complement found from 4854 to 4860\n",
      "Complement found from 5784 to 5790\n",
      "Complement found from 6101 to 6107\n",
      "Complement found from 6557 to 6563\n",
      "Complement found from 7175 to 7181\n",
      "Complement found from 9482 to 9488\n",
      "Complement found from 10220 to 10226\n",
      "Complement found from 11119 to 11125\n",
      "Complement found from 12684 to 12690\n",
      "Complement found from 13335 to 13341\n",
      "Complement found from 13615 to 13621\n",
      "Complement found from 14106 to 14112\n",
      "Seq is: TTCTTC\n",
      "13 TTCTTC motifs detected in file\n",
      "\n",
      "Complement found from 15057 to 15063\n",
      "Seq is: CGGATC\n",
      "1 CGGATC motifs detected in file\n",
      "\n",
      "Complement found from 1186 to 1192\n",
      "Complement found from 6841 to 6847\n",
      "Complement found from 8936 to 8942\n",
      "Complement found from 10188 to 10194\n",
      "Complement found from 11136 to 11142\n",
      "Complement found from 11549 to 11555\n",
      "Complement found from 12080 to 12086\n",
      "Complement found from 12098 to 12104\n",
      "Complement found from 16421 to 16427\n",
      "Seq is: TATCCC\n",
      "9 TATCCC motifs detected in file\n",
      "\n",
      "Complement found from 370 to 376\n",
      "Complement found from 440 to 446\n",
      "Complement found from 2761 to 2767\n",
      "Complement found from 5061 to 5067\n",
      "Complement found from 5177 to 5183\n",
      "Complement found from 5186 to 5192\n",
      "Complement found from 5391 to 5397\n",
      "Complement found from 6530 to 6536\n",
      "Complement found from 7286 to 7292\n",
      "Complement found from 7714 to 7720\n",
      "Complement found from 7742 to 7748\n",
      "Complement found from 9010 to 9016\n",
      "Complement found from 9257 to 9263\n",
      "Complement found from 10939 to 10945\n",
      "Complement found from 12245 to 12251\n",
      "Complement found from 12412 to 12418\n",
      "Complement found from 12736 to 12742\n",
      "Complement found from 13161 to 13167\n",
      "Complement found from 13626 to 13632\n",
      "Complement found from 13657 to 13663\n",
      "Complement found from 13741 to 13747\n",
      "Complement found from 13830 to 13836\n",
      "Complement found from 14569 to 14575\n",
      "Complement found from 15599 to 15605\n",
      "Seq is: CTAACA\n",
      "24 CTAACA motifs detected in file\n",
      "\n",
      "Complement found from 3494 to 3500\n",
      "Complement found from 6874 to 6880\n",
      "Complement found from 9053 to 9059\n",
      "Complement found from 11002 to 11008\n",
      "Seq is: CGCCAC\n",
      "4 CGCCAC motifs detected in file\n",
      "\n",
      "Complement found from 1139 to 1145\n",
      "Complement found from 2121 to 2127\n",
      "Complement found from 3870 to 3876\n",
      "Complement found from 5375 to 5381\n",
      "Complement found from 5951 to 5957\n",
      "Complement found from 8419 to 8425\n",
      "Complement found from 8710 to 8716\n",
      "Complement found from 9356 to 9362\n",
      "Complement found from 12346 to 12352\n",
      "Complement found from 13164 to 13170\n",
      "Complement found from 14311 to 14317\n",
      "Complement found from 15472 to 15478\n",
      "Seq is: ACACTA\n",
      "12 ACACTA motifs detected in file\n",
      "\n",
      "Complement found from 7857 to 7863\n",
      "Complement found from 15589 to 15595\n",
      "Seq is: CGATCC\n",
      "2 CGATCC motifs detected in file\n",
      "\n",
      "Complement found from 1264 to 1270\n",
      "Complement found from 4504 to 4510\n",
      "Complement found from 6340 to 6346\n",
      "Complement found from 6934 to 6940\n",
      "Complement found from 9098 to 9104\n",
      "Complement found from 12701 to 12707\n",
      "Complement found from 15294 to 15300\n",
      "Seq is: CATCTT\n",
      "7 CATCTT motifs detected in file\n",
      "\n",
      "Complement found from 7270 to 7276\n",
      "Complement found from 11223 to 11229\n",
      "Complement found from 11271 to 11277\n",
      "Seq is: AGGCTC\n",
      "3 AGGCTC motifs detected in file\n",
      "\n",
      "Complement found from 487 to 493\n",
      "Complement found from 3396 to 3402\n",
      "Complement found from 5424 to 5430\n",
      "Complement found from 12854 to 12860\n",
      "Complement found from 14235 to 14241\n",
      "Seq is: ATACAA\n",
      "5 ATACAA motifs detected in file\n",
      "\n",
      "Complement found from 1240 to 1246\n",
      "Complement found from 3291 to 3297\n",
      "Complement found from 4690 to 4696\n",
      "Complement found from 6448 to 6454\n",
      "Complement found from 8350 to 8356\n",
      "Complement found from 10330 to 10336\n",
      "Complement found from 10627 to 10633\n",
      "Complement found from 11380 to 11386\n",
      "Complement found from 12131 to 12137\n",
      "Complement found from 15024 to 15030\n",
      "Seq is: CCTCTT\n",
      "10 CCTCTT motifs detected in file\n",
      "\n",
      "Complement found from 54 to 60\n",
      "Complement found from 449 to 455\n",
      "Complement found from 4077 to 4083\n",
      "Complement found from 9719 to 9725\n",
      "Complement found from 10877 to 10883\n",
      "Complement found from 11108 to 11114\n",
      "Seq is: TATTTT\n",
      "6 TATTTT motifs detected in file\n",
      "\n",
      "Complement found from 7008 to 7014\n",
      "Complement found from 16538 to 16544\n",
      "Seq is: ACACGT\n",
      "2 ACACGT motifs detected in file\n",
      "\n",
      "Complement found from 12 to 18\n",
      "Complement found from 632 to 638\n",
      "Complement found from 731 to 737\n",
      "Complement found from 951 to 957\n",
      "Complement found from 2277 to 2283\n",
      "Complement found from 2508 to 2514\n",
      "Complement found from 2520 to 2526\n",
      "Complement found from 3507 to 3513\n",
      "Complement found from 3519 to 3525\n",
      "Complement found from 5321 to 5327\n",
      "Complement found from 5812 to 5818\n",
      "Complement found from 7645 to 7651\n",
      "Complement found from 8431 to 8437\n",
      "Complement found from 8652 to 8658\n",
      "Complement found from 9566 to 9572\n",
      "Complement found from 9635 to 9641\n",
      "Complement found from 11155 to 11161\n",
      "Complement found from 13182 to 13188\n",
      "Complement found from 14061 to 14067\n",
      "Complement found from 14878 to 14884\n",
      "Complement found from 15376 to 15382\n",
      "Complement found from 15397 to 15403\n",
      "Seq is: ATCACC\n",
      "22 ATCACC motifs detected in file\n",
      "\n",
      "Complement found from 1098 to 1104\n",
      "Complement found from 2174 to 2180\n",
      "Complement found from 2223 to 2229\n",
      "Complement found from 2778 to 2784\n",
      "Complement found from 3485 to 3491\n",
      "Complement found from 4396 to 4402\n",
      "Complement found from 8229 to 8235\n",
      "Complement found from 9598 to 9604\n",
      "Complement found from 10744 to 10750\n",
      "Complement found from 11407 to 11413\n",
      "Complement found from 11601 to 11607\n",
      "Complement found from 13790 to 13796\n",
      "Complement found from 14022 to 14028\n",
      "Complement found from 14491 to 14497\n",
      "Complement found from 16526 to 16532\n",
      "Seq is: CCTAAA\n",
      "15 CCTAAA motifs detected in file\n",
      "\n",
      "Complement found from 3256 to 3262\n",
      "Complement found from 3462 to 3468\n",
      "Complement found from 4275 to 4281\n",
      "Complement found from 4778 to 4784\n",
      "Complement found from 5405 to 5411\n",
      "Complement found from 6413 to 6419\n",
      "Complement found from 7433 to 7439\n",
      "Complement found from 8488 to 8494\n",
      "Complement found from 8494 to 8500\n",
      "Complement found from 10214 to 10220\n",
      "Complement found from 12034 to 12040\n",
      "Complement found from 12327 to 12333\n",
      "Complement found from 13872 to 13878\n",
      "Complement found from 14776 to 14782\n",
      "Complement found from 15392 to 15398\n",
      "Complement found from 16159 to 16165\n",
      "Seq is: ATAAAA\n",
      "16 ATAAAA motifs detected in file\n",
      "\n",
      "Complement found from 391 to 397\n",
      "Complement found from 1508 to 1514\n",
      "Complement found from 3107 to 3113\n",
      "Complement found from 5542 to 5548\n",
      "Complement found from 7511 to 7517\n",
      "Complement found from 10044 to 10050\n",
      "Complement found from 11801 to 11807\n",
      "Complement found from 12573 to 12579\n",
      "Complement found from 12687 to 12693\n",
      "Complement found from 13338 to 13344\n",
      "Seq is: TTCAAA\n",
      "10 TTCAAA motifs detected in file\n",
      "\n",
      "Complement found from 548 to 554\n",
      "Complement found from 2392 to 2398\n",
      "Complement found from 8806 to 8812\n",
      "Complement found from 8979 to 8985\n",
      "Complement found from 9070 to 9076\n",
      "Complement found from 10838 to 10844\n",
      "Complement found from 11169 to 11175\n",
      "Complement found from 13292 to 13298\n",
      "Complement found from 14200 to 14206\n",
      "Complement found from 14330 to 14336\n",
      "Complement found from 14477 to 14483\n",
      "Complement found from 14690 to 14696\n",
      "Complement found from 15766 to 15772\n",
      "Seq is: CAACCA\n",
      "13 CAACCA motifs detected in file\n",
      "\n",
      "Complement found from 4666 to 4672\n",
      "Complement found from 4948 to 4954\n",
      "Complement found from 7136 to 7142\n",
      "Complement found from 12285 to 12291\n",
      "Complement found from 12446 to 12452\n",
      "Complement found from 15633 to 15639\n",
      "Seq is: ATCCAT\n",
      "6 ATCCAT motifs detected in file\n",
      "\n",
      "Complement found from 4155 to 4161\n",
      "Complement found from 7938 to 7944\n",
      "Complement found from 10133 to 10139\n",
      "Complement found from 12318 to 12324\n",
      "Complement found from 12914 to 12920\n",
      "Complement found from 16244 to 16250\n",
      "Seq is: CAACTC\n",
      "6 CAACTC motifs detected in file\n",
      "\n",
      "Seq is: CAGATC\n",
      "0 CAGATC motifs detected in file\n",
      "\n",
      "Complement found from 12110 to 12116\n",
      "Complement found from 16493 to 16499\n",
      "Seq is: CCGACA\n",
      "2 CCGACA motifs detected in file\n",
      "\n",
      "Complement found from 1265 to 1271\n",
      "Complement found from 4026 to 4032\n",
      "Complement found from 6098 to 6104\n",
      "Complement found from 6341 to 6347\n",
      "Complement found from 7933 to 7939\n",
      "Complement found from 9099 to 9105\n",
      "Complement found from 11116 to 11122\n",
      "Complement found from 12702 to 12708\n",
      "Complement found from 13104 to 13110\n",
      "Seq is: ATCTTC\n",
      "9 ATCTTC motifs detected in file\n",
      "\n",
      "Complement found from 14256 to 14262\n",
      "Seq is: AGGATC\n",
      "1 AGGATC motifs detected in file\n",
      "\n",
      "Complement found from 4798 to 4804\n",
      "Complement found from 7650 to 7656\n",
      "Complement found from 8132 to 8138\n",
      "Complement found from 14351 to 14357\n",
      "Complement found from 15285 to 15291\n",
      "Complement found from 16025 to 16031\n",
      "Seq is: CTTTCA\n",
      "6 CTTTCA motifs detected in file\n",
      "\n",
      "Complement found from 9139 to 9145\n",
      "Seq is: CTGTCG\n",
      "1 CTGTCG motifs detected in file\n",
      "\n",
      "Complement found from 5732 to 5738\n",
      "Complement found from 8014 to 8020\n",
      "Complement found from 14262 to 14268\n",
      "Complement found from 15142 to 15148\n",
      "Seq is: CTCCCG\n",
      "4 CTCCCG motifs detected in file\n",
      "\n",
      "Complement found from 4517 to 4523\n",
      "Complement found from 5450 to 5456\n",
      "Complement found from 6878 to 6884\n",
      "Complement found from 7717 to 7723\n",
      "Complement found from 11255 to 11261\n",
      "Complement found from 11497 to 11503\n",
      "Complement found from 12908 to 12914\n",
      "Complement found from 14398 to 14404\n",
      "Complement found from 14644 to 14650\n",
      "Complement found from 15325 to 15331\n",
      "Seq is: ACACTC\n",
      "10 ACACTC motifs detected in file\n",
      "\n",
      "Complement found from 2228 to 2234\n",
      "Complement found from 4997 to 5003\n",
      "Complement found from 5809 to 5815\n",
      "Complement found from 7133 to 7139\n",
      "Complement found from 7435 to 7441\n",
      "Complement found from 8233 to 8239\n",
      "Complement found from 8533 to 8539\n",
      "Complement found from 10153 to 10159\n",
      "Complement found from 11604 to 11610\n",
      "Complement found from 12443 to 12449\n",
      "Complement found from 13233 to 13239\n",
      "Complement found from 13874 to 13880\n",
      "Complement found from 15394 to 15400\n",
      "Seq is: AAAATC\n",
      "13 AAAATC motifs detected in file\n",
      "\n",
      "Complement found from 339 to 345\n",
      "Complement found from 3183 to 3189\n",
      "Complement found from 4882 to 4888\n",
      "Complement found from 7746 to 7752\n",
      "Complement found from 10507 to 10513\n",
      "Complement found from 14823 to 14829\n",
      "Seq is: CATCTC\n",
      "6 CATCTC motifs detected in file\n",
      "\n",
      "Complement found from 1530 to 1536\n",
      "Complement found from 1836 to 1842\n",
      "Complement found from 3428 to 3434\n",
      "Complement found from 3483 to 3489\n",
      "Complement found from 6152 to 6158\n",
      "Complement found from 7094 to 7100\n",
      "Complement found from 7625 to 7631\n",
      "Complement found from 8227 to 8233\n",
      "Complement found from 9254 to 9260\n",
      "Complement found from 9525 to 9531\n",
      "Complement found from 9581 to 9587\n",
      "Complement found from 10277 to 10283\n",
      "Complement found from 10936 to 10942\n",
      "Complement found from 11233 to 11239\n",
      "Complement found from 13128 to 13134\n",
      "Complement found from 13946 to 13952\n",
      "Complement found from 13980 to 13986\n",
      "Complement found from 14489 to 14495\n",
      "Complement found from 14770 to 14776\n",
      "Complement found from 15367 to 15373\n",
      "Seq is: CCCCTA\n",
      "20 CCCCTA motifs detected in file\n",
      "\n",
      "Complement found from 3301 to 3307\n",
      "Complement found from 3804 to 3810\n",
      "Complement found from 4038 to 4044\n",
      "Complement found from 4071 to 4077\n",
      "Complement found from 6195 to 6201\n",
      "Complement found from 6398 to 6404\n",
      "Complement found from 7183 to 7189\n",
      "Complement found from 7723 to 7729\n",
      "Complement found from 8707 to 8713\n",
      "Complement found from 9196 to 9202\n",
      "Complement found from 10038 to 10044\n",
      "Complement found from 10895 to 10901\n",
      "Complement found from 11261 to 11267\n",
      "Complement found from 11995 to 12001\n",
      "Complement found from 12029 to 12035\n",
      "Complement found from 12176 to 12182\n",
      "Complement found from 12248 to 12254\n",
      "Complement found from 12486 to 12492\n",
      "Complement found from 12929 to 12935\n",
      "Complement found from 13744 to 13750\n",
      "Complement found from 13773 to 13779\n",
      "Complement found from 15678 to 15684\n",
      "Complement found from 15823 to 15829\n",
      "Seq is: ACAACA\n",
      "23 ACAACA motifs detected in file\n",
      "\n",
      "Complement found from 825 to 831\n",
      "Complement found from 1711 to 1717\n",
      "Complement found from 2133 to 2139\n",
      "Complement found from 6038 to 6044\n",
      "Complement found from 8317 to 8323\n",
      "Complement found from 9078 to 9084\n",
      "Complement found from 10920 to 10926\n",
      "Complement found from 13380 to 13386\n",
      "Complement found from 15935 to 15941\n",
      "Seq is: AACCTT\n",
      "9 AACCTT motifs detected in file\n",
      "\n",
      "Complement found from 304 to 310\n",
      "Complement found from 955 to 961\n",
      "Complement found from 3566 to 3572\n",
      "Complement found from 3584 to 3590\n",
      "Complement found from 3892 to 3898\n",
      "Complement found from 4248 to 4254\n",
      "Complement found from 4317 to 4323\n",
      "Complement found from 4794 to 4800\n",
      "Complement found from 6419 to 6425\n",
      "Complement found from 8271 to 8277\n",
      "Complement found from 8280 to 8286\n",
      "Complement found from 8604 to 8610\n",
      "Complement found from 10935 to 10941\n",
      "Complement found from 10947 to 10953\n",
      "Complement found from 11511 to 11517\n",
      "Complement found from 11672 to 11678\n",
      "Complement found from 13127 to 13133\n",
      "Complement found from 13762 to 13768\n",
      "Complement found from 13781 to 13787\n",
      "Complement found from 14488 to 14494\n",
      "Complement found from 14769 to 14775\n",
      "Complement found from 15366 to 15372\n",
      "Complement found from 16183 to 16189\n",
      "Complement found from 16375 to 16381\n",
      "Seq is: CCCCCT\n",
      "24 CCCCCT motifs detected in file\n",
      "\n",
      "Complement found from 3685 to 3691\n",
      "Complement found from 6443 to 6449\n",
      "Complement found from 7661 to 7667\n",
      "Complement found from 12811 to 12817\n",
      "Complement found from 14223 to 14229\n",
      "Complement found from 15428 to 15434\n",
      "Seq is: ACGCCC\n",
      "6 ACGCCC motifs detected in file\n",
      "\n",
      "Complement found from 827 to 833\n",
      "Complement found from 3781 to 3787\n",
      "Complement found from 7833 to 7839\n",
      "Complement found from 12463 to 12469\n",
      "Seq is: CCTTTA\n",
      "4 CCTTTA motifs detected in file\n",
      "\n",
      "Complement found from 1119 to 1125\n",
      "Complement found from 2396 to 2402\n",
      "Complement found from 2966 to 2972\n",
      "Complement found from 4696 to 4702\n",
      "Complement found from 6399 to 6405\n",
      "Complement found from 7724 to 7730\n",
      "Complement found from 8639 to 8645\n",
      "Complement found from 8661 to 8667\n",
      "Complement found from 10778 to 10784\n",
      "Complement found from 10893 to 10899\n",
      "Complement found from 11331 to 11337\n",
      "Complement found from 12487 to 12493\n",
      "Complement found from 12930 to 12936\n",
      "Complement found from 13774 to 13780\n",
      "Complement found from 13859 to 13865\n",
      "Complement found from 14186 to 14192\n",
      "Complement found from 15679 to 15685\n",
      "Complement found from 15824 to 15830\n",
      "Complement found from 16075 to 16081\n",
      "Complement found from 16278 to 16284\n",
      "Seq is: CAACAA\n",
      "20 CAACAA motifs detected in file\n",
      "\n",
      "Complement found from 2403 to 2409\n",
      "Complement found from 4751 to 4757\n",
      "Complement found from 9828 to 9834\n",
      "Complement found from 10439 to 10445\n",
      "Complement found from 10590 to 10596\n",
      "Complement found from 11728 to 11734\n",
      "Complement found from 12116 to 12122\n",
      "Complement found from 13735 to 13741\n",
      "Complement found from 14669 to 14675\n",
      "Complement found from 15305 to 15311\n",
      "Seq is: TCATTA\n",
      "10 TCATTA motifs detected in file\n",
      "\n",
      "Complement found from 2113 to 2119\n",
      "Complement found from 8351 to 8357\n",
      "Complement found from 9991 to 9997\n",
      "Complement found from 11381 to 11387\n",
      "Complement found from 14102 to 14108\n",
      "Complement found from 14349 to 14355\n",
      "Seq is: CTCTTT\n",
      "6 CTCTTT motifs detected in file\n",
      "\n",
      "Complement found from 8717 to 8723\n",
      "Seq is: AGGACG\n",
      "1 AGGACG motifs detected in file\n",
      "\n",
      "Complement found from 3798 to 3804\n",
      "Complement found from 4945 to 4951\n",
      "Complement found from 6638 to 6644\n",
      "Complement found from 7642 to 7648\n",
      "Complement found from 7681 to 7687\n",
      "Complement found from 8934 to 8940\n",
      "Complement found from 9093 to 9099\n",
      "Complement found from 11007 to 11013\n",
      "Complement found from 11134 to 11140\n",
      "Seq is: CTTATC\n",
      "9 CTTATC motifs detected in file\n",
      "\n",
      "Complement found from 1413 to 1419\n",
      "Complement found from 3908 to 3914\n",
      "Seq is: CGAAGG\n",
      "2 CGAAGG motifs detected in file\n",
      "\n",
      "Complement found from 7213 to 7219\n",
      "Complement found from 9771 to 9777\n",
      "Seq is: CCGACG\n",
      "2 CCGACG motifs detected in file\n",
      "\n",
      "Seq is: GTCCGC\n",
      "0 GTCCGC motifs detected in file\n",
      "\n",
      "Complement found from 1802 to 1808\n",
      "Complement found from 7556 to 7562\n",
      "Complement found from 8499 to 8505\n",
      "Complement found from 10444 to 10450\n",
      "Complement found from 14294 to 14300\n",
      "Complement found from 14498 to 14504\n",
      "Complement found from 14779 to 14785\n",
      "Complement found from 14964 to 14970\n",
      "Seq is: AAATTA\n",
      "8 AAATTA motifs detected in file\n",
      "\n",
      "Complement found from 352 to 358\n",
      "Complement found from 434 to 440\n",
      "Complement found from 556 to 562\n",
      "Complement found from 961 to 967\n",
      "Complement found from 1681 to 1687\n",
      "Complement found from 3413 to 3419\n",
      "Complement found from 7467 to 7473\n",
      "Complement found from 8368 to 8374\n",
      "Complement found from 9533 to 9539\n",
      "Complement found from 9555 to 9561\n",
      "Complement found from 10916 to 10922\n",
      "Complement found from 12301 to 12307\n",
      "Complement found from 14531 to 14537\n",
      "Complement found from 14750 to 14756\n",
      "Seq is: CCCCAA\n",
      "14 CCCCAA motifs detected in file\n",
      "\n",
      "Complement found from 3181 to 3187\n",
      "Complement found from 5297 to 5303\n",
      "Complement found from 6833 to 6839\n",
      "Complement found from 7261 to 7267\n",
      "Complement found from 7795 to 7801\n",
      "Complement found from 9096 to 9102\n",
      "Complement found from 10340 to 10346\n",
      "Complement found from 10864 to 10870\n",
      "Complement found from 11152 to 11158\n",
      "Complement found from 13371 to 13377\n",
      "Complement found from 13518 to 13524\n",
      "Complement found from 14058 to 14064\n",
      "Complement found from 14977 to 14983\n",
      "Seq is: ATCATC\n",
      "13 ATCATC motifs detected in file\n",
      "\n",
      "Complement found from 3210 to 3216\n",
      "Complement found from 3793 to 3799\n",
      "Complement found from 5205 to 5211\n",
      "Complement found from 5433 to 5439\n",
      "Complement found from 6313 to 6319\n",
      "Complement found from 7400 to 7406\n",
      "Complement found from 8656 to 8662\n",
      "Complement found from 8809 to 8815\n",
      "Complement found from 9055 to 9061\n",
      "Complement found from 10158 to 10164\n",
      "Complement found from 10841 to 10847\n",
      "Complement found from 12358 to 12364\n",
      "Complement found from 12397 to 12403\n",
      "Complement found from 13022 to 13028\n",
      "Complement found from 13054 to 13060\n",
      "Complement found from 13124 to 13130\n",
      "Complement found from 13648 to 13654\n",
      "Complement found from 13678 to 13684\n",
      "Complement found from 14336 to 14342\n",
      "Complement found from 14810 to 14816\n",
      "Complement found from 15263 to 15269\n",
      "Complement found from 15405 to 15411\n",
      "Complement found from 16051 to 16057\n",
      "Complement found from 16255 to 16261\n",
      "Complement found from 16290 to 16296\n",
      "Seq is: CCACCC\n",
      "25 CCACCC motifs detected in file\n",
      "\n",
      "Complement found from 834 to 840\n",
      "Complement found from 964 to 970\n",
      "Complement found from 2899 to 2905\n",
      "Complement found from 4776 to 4782\n",
      "Complement found from 15650 to 15656\n",
      "Seq is: CAATAA\n",
      "5 CAATAA motifs detected in file\n",
      "\n",
      "Complement found from 598 to 604\n",
      "Complement found from 4656 to 4662\n",
      "Complement found from 6889 to 6895\n",
      "Complement found from 12844 to 12850\n",
      "Complement found from 16201 to 16207\n",
      "Seq is: AAGCAA\n",
      "5 AAGCAA motifs detected in file\n",
      "\n",
      "Complement found from 4231 to 4237\n",
      "Complement found from 9699 to 9705\n",
      "Complement found from 13279 to 13285\n",
      "Complement found from 14171 to 14177\n",
      "Complement found from 15182 to 15188\n",
      "Complement found from 16197 to 16203\n",
      "Seq is: TTACAA\n",
      "6 TTACAA motifs detected in file\n",
      "\n",
      "Complement found from 3748 to 3754\n",
      "Complement found from 7080 to 7086\n",
      "Complement found from 7274 to 7280\n",
      "Complement found from 8547 to 8553\n",
      "Complement found from 8593 to 8599\n",
      "Complement found from 8974 to 8980\n",
      "Complement found from 11501 to 11507\n",
      "Complement found from 11696 to 11702\n",
      "Complement found from 12043 to 12049\n",
      "Complement found from 14483 to 14489\n",
      "Complement found from 14794 to 14800\n",
      "Complement found from 15160 to 15166\n",
      "Complement found from 15743 to 15749\n",
      "Seq is: TCATTC\n",
      "13 TCATTC motifs detected in file\n",
      "\n",
      "Complement found from 3210 to 3216\n",
      "Complement found from 3793 to 3799\n",
      "Complement found from 5205 to 5211\n",
      "Complement found from 5433 to 5439\n",
      "Complement found from 6313 to 6319\n",
      "Complement found from 7400 to 7406\n",
      "Complement found from 8656 to 8662\n",
      "Complement found from 8809 to 8815\n",
      "Complement found from 9055 to 9061\n",
      "Complement found from 10158 to 10164\n",
      "Complement found from 10841 to 10847\n",
      "Complement found from 12358 to 12364\n",
      "Complement found from 12397 to 12403\n",
      "Complement found from 13022 to 13028\n",
      "Complement found from 13054 to 13060\n",
      "Complement found from 13124 to 13130\n",
      "Complement found from 13648 to 13654\n",
      "Complement found from 13678 to 13684\n",
      "Complement found from 14336 to 14342\n",
      "Complement found from 14810 to 14816\n",
      "Complement found from 15263 to 15269\n",
      "Complement found from 15405 to 15411\n",
      "Complement found from 16051 to 16057\n",
      "Complement found from 16255 to 16261\n",
      "Complement found from 16290 to 16296\n",
      "Seq is: CCACCC\n",
      "25 CCACCC motifs detected in file\n",
      "\n",
      "Complement found from 3449 to 3455\n",
      "Complement found from 3962 to 3968\n",
      "Complement found from 7327 to 7333\n",
      "Complement found from 9484 to 9490\n",
      "Complement found from 10063 to 10069\n",
      "Seq is: CTTCGC\n",
      "5 CTTCGC motifs detected in file\n",
      "\n",
      "Complement found from 10712 to 10718\n",
      "Complement found from 12748 to 12754\n",
      "Complement found from 12912 to 12918\n",
      "Complement found from 13902 to 13908\n",
      "Complement found from 14818 to 14824\n",
      "Seq is: TCCAAC\n",
      "5 TCCAAC motifs detected in file\n",
      "\n",
      "Complement found from 2118 to 2124\n",
      "Complement found from 15795 to 15801\n",
      "Seq is: TGGACA\n",
      "2 TGGACA motifs detected in file\n",
      "\n",
      "Complement found from 2231 to 2237\n",
      "Complement found from 15202 to 15208\n",
      "Seq is: ATCCCA\n",
      "2 ATCCCA motifs detected in file\n",
      "\n",
      "Complement found from 3292 to 3298\n",
      "Complement found from 3468 to 3474\n",
      "Complement found from 3550 to 3556\n",
      "Complement found from 4691 to 4697\n",
      "Complement found from 6449 to 6455\n",
      "Complement found from 11622 to 11628\n",
      "Complement found from 12477 to 12483\n",
      "Complement found from 15025 to 15031\n",
      "Complement found from 15445 to 15451\n",
      "Seq is: CTCTTC\n",
      "9 CTCTTC motifs detected in file\n",
      "\n",
      "Complement found from 4754 to 4760\n",
      "Complement found from 7301 to 7307\n",
      "Complement found from 10076 to 10082\n",
      "Complement found from 11338 to 11344\n",
      "Seq is: TTAATA\n",
      "4 TTAATA motifs detected in file\n",
      "\n",
      "Complement found from 775 to 781\n",
      "Complement found from 2207 to 2213\n",
      "Complement found from 9643 to 9649\n",
      "Complement found from 12211 to 12217\n",
      "Seq is: AGCTCA\n",
      "4 AGCTCA motifs detected in file\n",
      "\n",
      "Complement found from 6172 to 6178\n",
      "Complement found from 12869 to 12875\n",
      "Seq is: CGATAT\n",
      "2 CGATAT motifs detected in file\n",
      "\n",
      "Seq is: CCGCGA\n",
      "0 CCGCGA motifs detected in file\n",
      "\n",
      "Complement found from 425 to 431\n",
      "Complement found from 3273 to 3279\n",
      "Complement found from 5860 to 5866\n",
      "Complement found from 6281 to 6287\n",
      "Complement found from 11771 to 11777\n",
      "Complement found from 15257 to 15263\n",
      "Complement found from 16342 to 16348\n",
      "Seq is: ACAGTC\n",
      "7 ACAGTC motifs detected in file\n",
      "\n",
      "Complement found from 7365 to 7371\n",
      "Complement found from 7498 to 7504\n",
      "Complement found from 8623 to 8629\n",
      "Complement found from 14053 to 14059\n",
      "Complement found from 14375 to 14381\n",
      "Complement found from 14871 to 14877\n",
      "Complement found from 15663 to 15669\n",
      "Seq is: CCTCCA\n",
      "7 CCTCCA motifs detected in file\n",
      "\n",
      "Complement found from 6837 to 6843\n",
      "Complement found from 11849 to 11855\n",
      "Complement found from 13576 to 13582\n",
      "Complement found from 14381 to 14387\n",
      "Seq is: TCGCTA\n",
      "4 TCGCTA motifs detected in file\n",
      "\n",
      "Complement found from 3538 to 3544\n",
      "Complement found from 4937 to 4943\n",
      "Complement found from 9269 to 9275\n",
      "Complement found from 10602 to 10608\n",
      "Complement found from 11297 to 11303\n",
      "Complement found from 11790 to 11796\n",
      "Complement found from 13435 to 13441\n",
      "Complement found from 13570 to 13576\n",
      "Seq is: CTCTCA\n",
      "8 CTCTCA motifs detected in file\n",
      "\n",
      "Complement found from 399 to 405\n",
      "Complement found from 3862 to 3868\n",
      "Complement found from 7682 to 7688\n",
      "Complement found from 12520 to 12526\n",
      "Complement found from 15017 to 15023\n",
      "Seq is: TTATCT\n",
      "5 TTATCT motifs detected in file\n",
      "\n",
      "Complement found from 3004 to 3010\n",
      "Complement found from 16422 to 16428\n",
      "Seq is: ATCCCG\n",
      "2 ATCCCG motifs detected in file\n",
      "\n",
      "Complement found from 3916 to 3922\n",
      "Seq is: AGTCCG\n",
      "1 AGTCCG motifs detected in file\n",
      "\n",
      "Complement found from 2235 to 2241\n",
      "Complement found from 2751 to 2757\n",
      "Complement found from 3717 to 3723\n",
      "Complement found from 3991 to 3997\n",
      "Complement found from 5680 to 5686\n",
      "Complement found from 9892 to 9898\n",
      "Complement found from 10295 to 10301\n",
      "Complement found from 11593 to 11599\n",
      "Complement found from 12549 to 12555\n",
      "Complement found from 12670 to 12676\n",
      "Complement found from 13531 to 13537\n",
      "Complement found from 13770 to 13776\n",
      "Complement found from 14189 to 14195\n",
      "Complement found from 15359 to 15365\n",
      "Complement found from 15675 to 15681\n",
      "Seq is: CAAACA\n",
      "15 CAAACA motifs detected in file\n",
      "\n",
      "Complement found from 189 to 195\n",
      "Complement found from 468 to 474\n",
      "Complement found from 5007 to 5013\n",
      "Complement found from 7947 to 7953\n",
      "Complement found from 8409 to 8415\n",
      "Complement found from 8942 to 8948\n",
      "Complement found from 9334 to 9340\n",
      "Complement found from 10654 to 10660\n",
      "Complement found from 11198 to 11204\n",
      "Complement found from 11618 to 11624\n",
      "Complement found from 11955 to 11961\n",
      "Complement found from 12716 to 12722\n",
      "Complement found from 13346 to 13352\n",
      "Complement found from 13907 to 13913\n",
      "Complement found from 14345 to 14351\n",
      "Seq is: CATACT\n",
      "15 CATACT motifs detected in file\n",
      "\n",
      "Complement found from 487 to 493\n",
      "Complement found from 3396 to 3402\n",
      "Complement found from 5424 to 5430\n",
      "Complement found from 12854 to 12860\n",
      "Complement found from 14235 to 14241\n",
      "Seq is: ATACAA\n",
      "5 ATACAA motifs detected in file\n",
      "\n",
      "Complement found from 3457 to 3463\n",
      "Complement found from 7128 to 7134\n",
      "Complement found from 11001 to 11007\n",
      "Complement found from 14993 to 14999\n",
      "Seq is: ACGCCA\n",
      "4 ACGCCA motifs detected in file\n",
      "\n",
      "Complement found from 854 to 860\n",
      "Complement found from 969 to 975\n",
      "Complement found from 1891 to 1897\n",
      "Complement found from 4418 to 4424\n",
      "Complement found from 5174 to 5180\n",
      "Complement found from 5642 to 5648\n",
      "Complement found from 8301 to 8307\n",
      "Complement found from 14013 to 14019\n",
      "Complement found from 15774 to 15780\n",
      "Complement found from 15992 to 15998\n",
      "Seq is: AAGCTA\n",
      "10 AAGCTA motifs detected in file\n",
      "\n",
      "Complement found from 4067 to 4073\n",
      "Complement found from 7001 to 7007\n",
      "Complement found from 7108 to 7114\n",
      "Complement found from 8141 to 8147\n",
      "Complement found from 8926 to 8932\n",
      "Complement found from 9088 to 9094\n",
      "Complement found from 11211 to 11217\n",
      "Complement found from 12906 to 12912\n",
      "Complement found from 14309 to 14315\n",
      "Complement found from 15031 to 15037\n",
      "Complement found from 15414 to 15420\n",
      "Complement found from 15576 to 15582\n",
      "Seq is: CTACAC\n",
      "12 CTACAC motifs detected in file\n",
      "\n",
      "Complement found from 1377 to 1383\n",
      "Complement found from 13135 to 13141\n",
      "Complement found from 14652 to 14658\n",
      "Complement found from 15074 to 15080\n",
      "Seq is: CAGAAA\n",
      "4 CAGAAA motifs detected in file\n",
      "\n",
      "Complement found from 143 to 149\n",
      "Complement found from 3329 to 3335\n",
      "Complement found from 5293 to 5299\n",
      "Complement found from 7665 to 7671\n",
      "Complement found from 7806 to 7812\n",
      "Complement found from 8427 to 8433\n",
      "Complement found from 9331 to 9337\n",
      "Complement found from 10474 to 10480\n",
      "Complement found from 10543 to 10549\n",
      "Complement found from 11726 to 11732\n",
      "Complement found from 12041 to 12047\n",
      "Complement found from 12063 to 12069\n",
      "Complement found from 15639 to 15645\n",
      "Complement found from 15741 to 15747\n",
      "Seq is: CCTCAT\n",
      "14 CCTCAT motifs detected in file\n",
      "\n",
      "Complement found from 2115 to 2121\n",
      "Complement found from 6133 to 6139\n",
      "Complement found from 9901 to 9907\n",
      "Seq is: CTTTGG\n",
      "3 CTTTGG motifs detected in file\n",
      "\n",
      "Complement found from 4517 to 4523\n",
      "Complement found from 5450 to 5456\n",
      "Complement found from 6878 to 6884\n",
      "Complement found from 7717 to 7723\n",
      "Complement found from 11255 to 11261\n",
      "Complement found from 11497 to 11503\n",
      "Complement found from 12908 to 12914\n",
      "Complement found from 14398 to 14404\n",
      "Complement found from 14644 to 14650\n",
      "Complement found from 15325 to 15331\n",
      "Seq is: ACACTC\n",
      "10 ACACTC motifs detected in file\n",
      "\n",
      "Complement found from 263 to 269\n",
      "Complement found from 9843 to 9849\n",
      "Complement found from 10807 to 10813\n",
      "Complement found from 13486 to 13492\n",
      "Complement found from 13816 to 13822\n",
      "Seq is: CTTTCC\n",
      "5 CTTTCC motifs detected in file\n",
      "\n",
      "Complement found from 2443 to 2449\n",
      "Complement found from 3141 to 3147\n",
      "Seq is: ATAAGG\n",
      "2 ATAAGG motifs detected in file\n",
      "\n",
      "Complement found from 906 to 912\n",
      "Complement found from 3044 to 3050\n",
      "Complement found from 15273 to 15279\n",
      "Seq is: ACGATT\n",
      "3 ACGATT motifs detected in file\n",
      "\n",
      "Complement found from 737 to 743\n",
      "Complement found from 7856 to 7862\n",
      "Seq is: ACGATC\n",
      "2 ACGATC motifs detected in file\n",
      "\n",
      "Complement found from 9772 to 9778\n",
      "Seq is: CGACGG\n",
      "1 CGACGG motifs detected in file\n",
      "\n",
      "Complement found from 2497 to 2503\n",
      "Complement found from 5463 to 5469\n",
      "Complement found from 7868 to 7874\n",
      "Complement found from 8910 to 8916\n",
      "Complement found from 9508 to 9514\n",
      "Complement found from 10462 to 10468\n",
      "Complement found from 10503 to 10509\n",
      "Complement found from 11990 to 11996\n",
      "Complement found from 12394 to 12400\n",
      "Complement found from 12712 to 12718\n",
      "Complement found from 14324 to 14330\n",
      "Complement found from 15785 to 15791\n",
      "Seq is: TTACCA\n",
      "12 TTACCA motifs detected in file\n",
      "\n",
      "Complement found from 3116 to 3122\n",
      "Complement found from 4106 to 4112\n",
      "Complement found from 4905 to 4911\n",
      "Complement found from 5215 to 5221\n",
      "Complement found from 6219 to 6225\n",
      "Complement found from 6291 to 6297\n",
      "Complement found from 7863 to 7869\n",
      "Complement found from 8471 to 8477\n",
      "Complement found from 9756 to 9762\n",
      "Complement found from 10552 to 10558\n",
      "Complement found from 10624 to 10630\n",
      "Complement found from 11058 to 11064\n",
      "Complement found from 11226 to 11232\n",
      "Complement found from 11404 to 11410\n",
      "Complement found from 11976 to 11982\n",
      "Complement found from 12563 to 12569\n",
      "Complement found from 13448 to 13454\n",
      "Complement found from 13583 to 13589\n",
      "Complement found from 15852 to 15858\n",
      "Seq is: CTCCCT\n",
      "19 CTCCCT motifs detected in file\n",
      "\n",
      "Complement found from 436 to 442\n",
      "Complement found from 1626 to 1632\n",
      "Complement found from 4154 to 4160\n",
      "Complement found from 8370 to 8376\n",
      "Complement found from 8436 to 8442\n",
      "Complement found from 8813 to 8819\n",
      "Complement found from 9868 to 9874\n",
      "Complement found from 12749 to 12755\n",
      "Complement found from 12913 to 12919\n",
      "Complement found from 15844 to 15850\n",
      "Seq is: CCAACT\n",
      "10 CCAACT motifs detected in file\n",
      "\n",
      "Complement found from 4932 to 4938\n",
      "Complement found from 5873 to 5879\n",
      "Complement found from 8792 to 8798\n",
      "Complement found from 11294 to 11300\n",
      "Complement found from 11930 to 11936\n",
      "Complement found from 12010 to 12016\n",
      "Complement found from 14848 to 14854\n",
      "Complement found from 14951 to 14957\n",
      "Seq is: TCACTC\n",
      "8 TCACTC motifs detected in file\n",
      "\n",
      "Complement found from 2959 to 2965\n",
      "Complement found from 4221 to 4227\n",
      "Complement found from 4667 to 4673\n",
      "Complement found from 7367 to 7373\n",
      "Complement found from 9320 to 9326\n",
      "Complement found from 10211 to 10217\n",
      "Complement found from 12591 to 12597\n",
      "Complement found from 15665 to 15671\n",
      "Seq is: TCCATA\n",
      "8 TCCATA motifs detected in file\n",
      "\n",
      "Complement found from 354 to 360\n",
      "Complement found from 2500 to 2506\n",
      "Complement found from 4363 to 4369\n",
      "Complement found from 4602 to 4608\n",
      "Complement found from 7131 to 7137\n",
      "Complement found from 8521 to 8527\n",
      "Complement found from 9430 to 9436\n",
      "Complement found from 10811 to 10817\n",
      "Complement found from 12152 to 12158\n",
      "Complement found from 12303 to 12309\n",
      "Complement found from 13970 to 13976\n",
      "Complement found from 14072 to 14078\n",
      "Complement found from 14533 to 14539\n",
      "Seq is: CCAAAA\n",
      "13 CCAAAA motifs detected in file\n",
      "\n",
      "Complement found from 144 to 150\n",
      "Complement found from 480 to 486\n",
      "Complement found from 4520 to 4526\n",
      "Complement found from 4747 to 4753\n",
      "Complement found from 5294 to 5300\n",
      "Complement found from 5453 to 5459\n",
      "Complement found from 6982 to 6988\n",
      "Complement found from 7807 to 7813\n",
      "Complement found from 8428 to 8434\n",
      "Complement found from 8634 to 8640\n",
      "Complement found from 11239 to 11245\n",
      "Complement found from 11662 to 11668\n",
      "Complement found from 12699 to 12705\n",
      "Complement found from 12795 to 12801\n",
      "Complement found from 13113 to 13119\n",
      "Complement found from 13572 to 13578\n",
      "Complement found from 14118 to 14124\n",
      "Complement found from 15640 to 15646\n",
      "Seq is: CTCATC\n",
      "18 CTCATC motifs detected in file\n",
      "\n",
      "Complement found from 5221 to 5227\n",
      "Complement found from 6574 to 6580\n",
      "Complement found from 7072 to 7078\n",
      "Complement found from 9541 to 9547\n",
      "Complement found from 13412 to 13418\n",
      "Complement found from 15240 to 15246\n",
      "Complement found from 15609 to 15615\n",
      "Seq is: AGGAGG\n",
      "7 AGGAGG motifs detected in file\n",
      "\n",
      "Complement found from 3256 to 3262\n",
      "Complement found from 3462 to 3468\n",
      "Complement found from 4275 to 4281\n",
      "Complement found from 4778 to 4784\n",
      "Complement found from 5405 to 5411\n",
      "Complement found from 6413 to 6419\n",
      "Complement found from 7433 to 7439\n",
      "Complement found from 8488 to 8494\n",
      "Complement found from 8494 to 8500\n",
      "Complement found from 10214 to 10220\n",
      "Complement found from 12034 to 12040\n",
      "Complement found from 12327 to 12333\n",
      "Complement found from 13872 to 13878\n",
      "Complement found from 14776 to 14782\n",
      "Complement found from 15392 to 15398\n",
      "Complement found from 16159 to 16165\n",
      "Seq is: ATAAAA\n",
      "16 ATAAAA motifs detected in file\n",
      "\n",
      "Complement found from 584 to 590\n",
      "Complement found from 1153 to 1159\n",
      "Complement found from 1609 to 1615\n",
      "Complement found from 4309 to 4315\n",
      "Complement found from 7640 to 7646\n",
      "Complement found from 11349 to 11355\n",
      "Seq is: AGCTTA\n",
      "6 AGCTTA motifs detected in file\n",
      "\n",
      "Complement found from 7967 to 7973\n",
      "Seq is: TAGAAC\n",
      "1 TAGAAC motifs detected in file\n",
      "\n",
      "Complement found from 1294 to 1300\n",
      "Complement found from 2307 to 2313\n",
      "Complement found from 2461 to 2467\n",
      "Complement found from 4400 to 4406\n",
      "Complement found from 12331 to 12337\n",
      "Seq is: AAGTAA\n",
      "5 AAGTAA motifs detected in file\n",
      "\n",
      "Complement found from 3286 to 3292\n",
      "Complement found from 5797 to 5803\n",
      "Complement found from 8105 to 8111\n",
      "Complement found from 9106 to 9112\n",
      "Complement found from 15581 to 15587\n",
      "Seq is: CAATTC\n",
      "5 CAATTC motifs detected in file\n",
      "\n",
      "Complement found from 2211 to 2217\n",
      "Complement found from 2427 to 2433\n",
      "Complement found from 3805 to 3811\n",
      "Complement found from 6547 to 6553\n",
      "Complement found from 6596 to 6602\n",
      "Complement found from 7184 to 7190\n",
      "Complement found from 8345 to 8351\n",
      "Complement found from 8708 to 8714\n",
      "Complement found from 9197 to 9203\n",
      "Complement found from 9352 to 9358\n",
      "Complement found from 10084 to 10090\n",
      "Complement found from 10615 to 10621\n",
      "Complement found from 10714 to 10720\n",
      "Complement found from 10833 to 10839\n",
      "Complement found from 11262 to 11268\n",
      "Complement found from 11996 to 12002\n",
      "Complement found from 12827 to 12833\n",
      "Complement found from 15323 to 15329\n",
      "Seq is: CAACAC\n",
      "18 CAACAC motifs detected in file\n",
      "\n",
      "Complement found from 472 to 478\n",
      "Complement found from 2880 to 2886\n",
      "Complement found from 3438 to 3444\n",
      "Complement found from 3555 to 3561\n",
      "Complement found from 3753 to 3759\n",
      "Complement found from 5103 to 5109\n",
      "Complement found from 5117 to 5123\n",
      "Complement found from 5150 to 5156\n",
      "Complement found from 6527 to 6533\n",
      "Complement found from 9344 to 9350\n",
      "Complement found from 10103 to 10109\n",
      "Complement found from 10556 to 10562\n",
      "Complement found from 10697 to 10703\n",
      "Complement found from 10873 to 10879\n",
      "Complement found from 11287 to 11293\n",
      "Complement found from 12349 to 12355\n",
      "Complement found from 12972 to 12978\n",
      "Complement found from 13683 to 13689\n",
      "Complement found from 14210 to 14216\n",
      "Seq is: CTACTA\n",
      "19 CTACTA motifs detected in file\n",
      "\n",
      "Seq is: CCTGGG\n",
      "0 CCTGGG motifs detected in file\n",
      "\n",
      "Complement found from 4616 to 4622\n",
      "Complement found from 11643 to 11649\n",
      "Complement found from 12401 to 12407\n",
      "Seq is: CCTCGT\n",
      "3 CCTCGT motifs detected in file\n",
      "\n",
      "Complement found from 53 to 59\n",
      "Complement found from 4642 to 4648\n",
      "Complement found from 6860 to 6866\n",
      "Complement found from 7058 to 7064\n",
      "Complement found from 8254 to 8260\n",
      "Complement found from 14723 to 14729\n",
      "Complement found from 16088 to 16094\n",
      "Seq is: GTATTT\n",
      "7 GTATTT motifs detected in file\n",
      "\n",
      "Complement found from 277 to 283\n",
      "Complement found from 4760 to 4766\n",
      "Complement found from 4889 to 4895\n",
      "Complement found from 4952 to 4958\n",
      "Complement found from 5309 to 5315\n",
      "Complement found from 6119 to 6125\n",
      "Complement found from 7067 to 7073\n",
      "Complement found from 7148 to 7154\n",
      "Complement found from 7630 to 7636\n",
      "Complement found from 9228 to 9234\n",
      "Complement found from 10455 to 10461\n",
      "Complement found from 11104 to 11110\n",
      "Complement found from 11779 to 11785\n",
      "Complement found from 12633 to 12639\n",
      "Complement found from 13538 to 13544\n",
      "Complement found from 14232 to 14238\n",
      "Complement found from 14343 to 14349\n",
      "Seq is: ATCATA\n",
      "17 ATCATA motifs detected in file\n",
      "\n",
      "Complement found from 1903 to 1909\n",
      "Complement found from 4437 to 4443\n",
      "Complement found from 9670 to 9676\n",
      "Seq is: CCGAAA\n",
      "3 CCGAAA motifs detected in file\n",
      "\n",
      "Complement found from 3294 to 3300\n",
      "Complement found from 4853 to 4859\n",
      "Complement found from 5783 to 5789\n",
      "Complement found from 6100 to 6106\n",
      "Complement found from 6556 to 6562\n",
      "Complement found from 8906 to 8912\n",
      "Complement found from 10222 to 10228\n",
      "Complement found from 10240 to 10246\n",
      "Complement found from 11118 to 11124\n",
      "Complement found from 12788 to 12794\n",
      "Complement found from 13106 to 13112\n",
      "Complement found from 13334 to 13340\n",
      "Complement found from 13959 to 13965\n",
      "Complement found from 14108 to 14114\n",
      "Seq is: CTTCTT\n",
      "14 CTTCTT motifs detected in file\n",
      "\n",
      "Complement found from 13937 to 13943\n",
      "Complement found from 14246 to 14252\n",
      "Complement found from 16425 to 16431\n",
      "Seq is: CCGCAC\n",
      "3 CCGCAC motifs detected in file\n",
      "\n",
      "Complement found from 9300 to 9306\n",
      "Complement found from 11415 to 11421\n",
      "Complement found from 12239 to 12245\n",
      "Seq is: CCATGT\n",
      "3 CCATGT motifs detected in file\n",
      "\n",
      "Complement found from 4067 to 4073\n",
      "Complement found from 7001 to 7007\n",
      "Complement found from 7108 to 7114\n",
      "Complement found from 8141 to 8147\n",
      "Complement found from 8926 to 8932\n",
      "Complement found from 9088 to 9094\n",
      "Complement found from 11211 to 11217\n",
      "Complement found from 12906 to 12912\n",
      "Complement found from 14309 to 14315\n",
      "Complement found from 15031 to 15037\n",
      "Complement found from 15414 to 15420\n",
      "Complement found from 15576 to 15582\n",
      "Seq is: CTACAC\n",
      "12 CTACAC motifs detected in file\n",
      "\n",
      "Complement found from 15616 to 15622\n",
      "Complement found from 15887 to 15893\n",
      "Seq is: GTCCTT\n",
      "2 GTCCTT motifs detected in file\n",
      "\n",
      "Complement found from 737 to 743\n",
      "Complement found from 7856 to 7862\n",
      "Seq is: ACGATC\n",
      "2 ACGATC motifs detected in file\n",
      "\n",
      "Complement found from 519 to 525\n",
      "Complement found from 797 to 803\n",
      "Complement found from 1478 to 1484\n",
      "Complement found from 1941 to 1947\n",
      "Complement found from 2256 to 2262\n",
      "Complement found from 3205 to 3211\n",
      "Complement found from 4386 to 4392\n",
      "Complement found from 6768 to 6774\n",
      "Complement found from 8921 to 8927\n",
      "Complement found from 9418 to 9424\n",
      "Complement found from 10539 to 10545\n",
      "Complement found from 13300 to 13306\n",
      "Complement found from 13933 to 13939\n",
      "Complement found from 14552 to 14558\n",
      "Complement found from 14562 to 14568\n",
      "Seq is: CACACC\n",
      "15 CACACC motifs detected in file\n",
      "\n",
      "Complement found from 545 to 551\n",
      "Complement found from 1809 to 1815\n",
      "Complement found from 2393 to 2399\n",
      "Complement found from 4600 to 4606\n",
      "Complement found from 4726 to 4732\n",
      "Complement found from 4979 to 4985\n",
      "Complement found from 8122 to 8128\n",
      "Complement found from 8342 to 8348\n",
      "Complement found from 8519 to 8525\n",
      "Complement found from 8980 to 8986\n",
      "Complement found from 9349 to 9355\n",
      "Complement found from 9674 to 9680\n",
      "Complement found from 10885 to 10891\n",
      "Complement found from 12150 to 12156\n",
      "Complement found from 13293 to 13299\n",
      "Complement found from 13856 to 13862\n",
      "Seq is: AACCAA\n",
      "16 AACCAA motifs detected in file\n",
      "\n",
      "Complement found from 3431 to 3437\n",
      "Complement found from 7920 to 7926\n",
      "Complement found from 8160 to 8166\n",
      "Complement found from 9781 to 9787\n",
      "Seq is: CTACGG\n",
      "4 CTACGG motifs detected in file\n",
      "\n",
      "Complement found from 370 to 376\n",
      "Complement found from 440 to 446\n",
      "Complement found from 2761 to 2767\n",
      "Complement found from 5061 to 5067\n",
      "Complement found from 5177 to 5183\n",
      "Complement found from 5186 to 5192\n",
      "Complement found from 5391 to 5397\n",
      "Complement found from 6530 to 6536\n",
      "Complement found from 7286 to 7292\n",
      "Complement found from 7714 to 7720\n",
      "Complement found from 7742 to 7748\n",
      "Complement found from 9010 to 9016\n",
      "Complement found from 9257 to 9263\n",
      "Complement found from 10939 to 10945\n",
      "Complement found from 12245 to 12251\n",
      "Complement found from 12412 to 12418\n",
      "Complement found from 12736 to 12742\n",
      "Complement found from 13161 to 13167\n",
      "Complement found from 13626 to 13632\n",
      "Complement found from 13657 to 13663\n",
      "Complement found from 13741 to 13747\n",
      "Complement found from 13830 to 13836\n",
      "Complement found from 14569 to 14575\n",
      "Complement found from 15599 to 15605\n",
      "Seq is: CTAACA\n",
      "24 CTAACA motifs detected in file\n",
      "\n",
      "Complement found from 4289 to 4295\n",
      "Seq is: TTGATA\n",
      "1 TTGATA motifs detected in file\n",
      "\n",
      "Complement found from 2885 to 2891\n",
      "Complement found from 4703 to 4709\n",
      "Complement found from 4744 to 4750\n",
      "Complement found from 5008 to 5014\n",
      "Complement found from 8410 to 8416\n",
      "Complement found from 11619 to 11625\n",
      "Complement found from 11973 to 11979\n",
      "Complement found from 13908 to 13914\n",
      "Complement found from 14346 to 14352\n",
      "Complement found from 14439 to 14445\n",
      "Complement found from 15871 to 15877\n",
      "Seq is: ATACTC\n",
      "11 ATACTC motifs detected in file\n",
      "\n",
      "Complement found from 14256 to 14262\n",
      "Seq is: AGGATC\n",
      "1 AGGATC motifs detected in file\n",
      "\n",
      "Complement found from 2244 to 2250\n",
      "Complement found from 2361 to 2367\n",
      "Complement found from 5614 to 5620\n",
      "Complement found from 6140 to 6146\n",
      "Complement found from 12528 to 12534\n",
      "Seq is: AACTGA\n",
      "5 AACTGA motifs detected in file\n",
      "\n",
      "Complement found from 1506 to 1512\n",
      "Complement found from 2860 to 2866\n",
      "Complement found from 3150 to 3156\n",
      "Complement found from 9820 to 9826\n",
      "Complement found from 11799 to 11805\n",
      "Complement found from 13251 to 13257\n",
      "Complement found from 13440 to 13446\n",
      "Complement found from 15290 to 15296\n",
      "Complement found from 15818 to 15824\n",
      "Complement found from 16509 to 16515\n",
      "Seq is: ACTTCA\n",
      "10 ACTTCA motifs detected in file\n",
      "\n",
      "Complement found from 1240 to 1246\n",
      "Complement found from 3291 to 3297\n",
      "Complement found from 4690 to 4696\n",
      "Complement found from 6448 to 6454\n",
      "Complement found from 8350 to 8356\n",
      "Complement found from 10330 to 10336\n",
      "Complement found from 10627 to 10633\n",
      "Complement found from 11380 to 11386\n",
      "Complement found from 12131 to 12137\n",
      "Complement found from 15024 to 15030\n",
      "Seq is: CCTCTT\n",
      "10 CCTCTT motifs detected in file\n",
      "\n",
      "Complement found from 3007 to 3013\n",
      "Complement found from 7233 to 7239\n",
      "Complement found from 11160 to 11166\n",
      "Seq is: CCGATG\n",
      "3 CCGATG motifs detected in file\n",
      "\n",
      "Complement found from 972 to 978\n",
      "Complement found from 1524 to 1530\n",
      "Complement found from 1932 to 1938\n",
      "Complement found from 2175 to 2181\n",
      "Complement found from 2224 to 2230\n",
      "Complement found from 3486 to 3492\n",
      "Complement found from 8230 to 8236\n",
      "Complement found from 8440 to 8446\n",
      "Complement found from 10762 to 10768\n",
      "Complement found from 11602 to 11608\n",
      "Complement found from 13791 to 13797\n",
      "Complement found from 14023 to 14029\n",
      "Complement found from 14393 to 14399\n",
      "Seq is: CTAAAA\n",
      "13 CTAAAA motifs detected in file\n",
      "\n",
      "Complement found from 1381 to 1387\n",
      "Complement found from 1432 to 1438\n",
      "Complement found from 2781 to 2787\n",
      "Complement found from 3680 to 3686\n",
      "Complement found from 4781 to 4787\n",
      "Complement found from 4871 to 4877\n",
      "Complement found from 7729 to 7735\n",
      "Complement found from 8457 to 8463\n",
      "Complement found from 8676 to 8682\n",
      "Complement found from 10765 to 10771\n",
      "Complement found from 11467 to 11473\n",
      "Complement found from 11754 to 11760\n",
      "Complement found from 12576 to 12582\n",
      "Complement found from 14507 to 14513\n",
      "Complement found from 14762 to 14768\n",
      "Complement found from 15604 to 15610\n",
      "Complement found from 15900 to 15906\n",
      "Complement found from 16010 to 16016\n",
      "Seq is: AAACTA\n",
      "18 AAACTA motifs detected in file\n",
      "\n",
      "Complement found from 1428 to 1434\n",
      "Complement found from 5553 to 5559\n",
      "Complement found from 7293 to 7299\n",
      "Complement found from 14204 to 14210\n",
      "Complement found from 15176 to 15182\n",
      "Complement found from 15770 to 15776\n",
      "Seq is: CAGTAA\n",
      "6 CAGTAA motifs detected in file\n",
      "\n",
      "Complement found from 335 to 341\n",
      "Complement found from 442 to 448\n",
      "Complement found from 1045 to 1051\n",
      "Complement found from 1614 to 1620\n",
      "Complement found from 2428 to 2434\n",
      "Complement found from 2705 to 2711\n",
      "Complement found from 3806 to 3812\n",
      "Complement found from 8452 to 8458\n",
      "Complement found from 9198 to 9204\n",
      "Complement found from 9353 to 9359\n",
      "Complement found from 9602 to 9608\n",
      "Complement found from 10715 to 10721\n",
      "Complement found from 10817 to 10823\n",
      "Complement found from 10834 to 10840\n",
      "Complement found from 11522 to 11528\n",
      "Complement found from 11997 to 12003\n",
      "Complement found from 12828 to 12834\n",
      "Complement found from 14550 to 14556\n",
      "Seq is: AACACA\n",
      "18 AACACA motifs detected in file\n",
      "\n",
      "Complement found from 4458 to 4464\n",
      "Complement found from 8108 to 8114\n",
      "Seq is: TTCCCG\n",
      "2 TTCCCG motifs detected in file\n",
      "\n",
      "Complement found from 60 to 66\n",
      "Complement found from 6454 to 6460\n",
      "Complement found from 7773 to 7779\n",
      "Seq is: CGTCTG\n",
      "3 CGTCTG motifs detected in file\n",
      "\n",
      "Seq is: ACGACG\n",
      "0 ACGACG motifs detected in file\n",
      "\n",
      "Seq is: CTTGTC\n",
      "0 CTTGTC motifs detected in file\n",
      "\n",
      "Complement found from 732 to 738\n",
      "Complement found from 1234 to 1240\n",
      "Complement found from 2521 to 2527\n",
      "Complement found from 2863 to 2869\n",
      "Complement found from 3472 to 3478\n",
      "Complement found from 3541 to 3547\n",
      "Complement found from 4015 to 4021\n",
      "Complement found from 4084 to 4090\n",
      "Complement found from 8476 to 8482\n",
      "Complement found from 8653 to 8659\n",
      "Complement found from 9646 to 9652\n",
      "Complement found from 9762 to 9768\n",
      "Complement found from 13183 to 13189\n",
      "Complement found from 13453 to 13459\n",
      "Complement found from 14402 to 14408\n",
      "Complement found from 14879 to 14885\n",
      "Complement found from 14911 to 14917\n",
      "Complement found from 15481 to 15487\n",
      "Seq is: TCACCA\n",
      "18 TCACCA motifs detected in file\n",
      "\n",
      "Complement found from 4421 to 4427\n",
      "Complement found from 10580 to 10586\n",
      "Seq is: CTATCG\n",
      "2 CTATCG motifs detected in file\n",
      "\n",
      "Complement found from 13913 to 13919\n",
      "Seq is: CGGATT\n",
      "1 CGGATT motifs detected in file\n",
      "\n",
      "Complement found from 12861 to 12867\n",
      "Seq is: CGTATC\n",
      "1 CGTATC motifs detected in file\n",
      "\n",
      "Complement found from 2387 to 2393\n",
      "Complement found from 3720 to 3726\n",
      "Complement found from 4023 to 4029\n",
      "Complement found from 4233 to 4239\n",
      "Complement found from 8000 to 8006\n",
      "Complement found from 8562 to 8568\n",
      "Complement found from 10984 to 10990\n",
      "Complement found from 13281 to 13287\n",
      "Complement found from 13776 to 13782\n",
      "Complement found from 13941 to 13947\n",
      "Complement found from 14572 to 14578\n",
      "Complement found from 15418 to 15424\n",
      "Complement found from 15826 to 15832\n",
      "Seq is: ACAATC\n",
      "13 ACAATC motifs detected in file\n",
      "\n",
      "Complement found from 3672 to 3678\n",
      "Complement found from 3678 to 3684\n",
      "Complement found from 4253 to 4259\n",
      "Complement found from 8674 to 8680\n",
      "Complement found from 11317 to 11323\n",
      "Complement found from 11752 to 11758\n",
      "Complement found from 11802 to 11808\n",
      "Complement found from 12574 to 12580\n",
      "Complement found from 15358 to 15364\n",
      "Seq is: TCAAAC\n",
      "9 TCAAAC motifs detected in file\n",
      "\n",
      "Complement found from 475 to 481\n",
      "Complement found from 3345 to 3351\n",
      "Complement found from 5354 to 5360\n",
      "Complement found from 5707 to 5713\n",
      "Complement found from 6467 to 6473\n",
      "Complement found from 7930 to 7936\n",
      "Complement found from 8649 to 8655\n",
      "Complement found from 8670 to 8676\n",
      "Complement found from 10768 to 10774\n",
      "Complement found from 11053 to 11059\n",
      "Complement found from 11101 to 11107\n",
      "Complement found from 12720 to 12726\n",
      "Complement found from 12952 to 12958\n",
      "Complement found from 13149 to 13155\n",
      "Complement found from 14136 to 14142\n",
      "Complement found from 14216 to 14222\n",
      "Complement found from 15832 to 15838\n",
      "Seq is: CTAATC\n",
      "17 CTAATC motifs detected in file\n",
      "\n",
      "Complement found from 11158 to 11164\n",
      "Complement found from 14555 to 14561\n",
      "Seq is: ACCCGA\n",
      "2 ACCCGA motifs detected in file\n",
      "\n",
      "Complement found from 1933 to 1939\n",
      "Complement found from 2176 to 2182\n",
      "Complement found from 2464 to 2470\n",
      "Complement found from 4276 to 4282\n",
      "Complement found from 12328 to 12334\n",
      "Seq is: TAAAAG\n",
      "5 TAAAAG motifs detected in file\n",
      "\n",
      "Complement found from 737 to 743\n",
      "Complement found from 7856 to 7862\n",
      "Seq is: ACGATC\n",
      "2 ACGATC motifs detected in file\n",
      "\n",
      "Complement found from 569 to 575\n",
      "Complement found from 802 to 808\n",
      "Complement found from 1082 to 1088\n",
      "Complement found from 5302 to 5308\n",
      "Complement found from 5446 to 5452\n",
      "Complement found from 5600 to 5606\n",
      "Complement found from 5895 to 5901\n",
      "Complement found from 6844 to 6850\n",
      "Complement found from 7398 to 7404\n",
      "Complement found from 8079 to 8085\n",
      "Complement found from 8558 to 8564\n",
      "Complement found from 8618 to 8624\n",
      "Complement found from 11139 to 11145\n",
      "Complement found from 11868 to 11874\n",
      "Complement found from 12482 to 12488\n",
      "Complement found from 12967 to 12973\n",
      "Complement found from 13052 to 13058\n",
      "Complement found from 13646 to 13652\n",
      "Complement found from 13676 to 13682\n",
      "Complement found from 13879 to 13885\n",
      "Complement found from 14388 to 14394\n",
      "Complement found from 14619 to 14625\n",
      "Complement found from 14808 to 14814\n",
      "Complement found from 15541 to 15547\n",
      "Seq is: CCCCAC\n",
      "24 CCCCAC motifs detected in file\n",
      "\n",
      "Complement found from 3449 to 3455\n",
      "Complement found from 3962 to 3968\n",
      "Complement found from 7327 to 7333\n",
      "Complement found from 9484 to 9490\n",
      "Complement found from 10063 to 10069\n",
      "Seq is: CTTCGC\n",
      "5 CTTCGC motifs detected in file\n",
      "\n",
      "Complement found from 3324 to 3330\n",
      "Complement found from 4495 to 4501\n",
      "Complement found from 5120 to 5126\n",
      "Complement found from 5359 to 5365\n",
      "Complement found from 5378 to 5384\n",
      "Complement found from 5471 to 5477\n",
      "Complement found from 6230 to 6236\n",
      "Complement found from 6307 to 6313\n",
      "Complement found from 6682 to 6688\n",
      "Complement found from 8970 to 8976\n",
      "Complement found from 9030 to 9036\n",
      "Complement found from 10599 to 10605\n",
      "Complement found from 10751 to 10757\n",
      "Complement found from 11236 to 11242\n",
      "Complement found from 11290 to 11296\n",
      "Complement found from 11809 to 11815\n",
      "Complement found from 12696 to 12702\n",
      "Complement found from 13071 to 13077\n",
      "Complement found from 13419 to 13425\n",
      "Complement found from 13502 to 13508\n",
      "Complement found from 13983 to 13989\n",
      "Complement found from 14130 to 14136\n",
      "Complement found from 14907 to 14913\n",
      "Complement found from 15069 to 15075\n",
      "Complement found from 15246 to 15252\n",
      "Complement found from 16438 to 16444\n",
      "Seq is: CTACTC\n",
      "26 CTACTC motifs detected in file\n",
      "\n",
      "Complement found from 3971 to 3977\n",
      "Complement found from 5072 to 5078\n",
      "Complement found from 6635 to 6641\n",
      "Complement found from 10219 to 10225\n",
      "Complement found from 13614 to 13620\n",
      "Complement found from 15012 to 15018\n",
      "Complement found from 15276 to 15282\n",
      "Complement found from 15339 to 15345\n",
      "Seq is: ATTCTT\n",
      "8 ATTCTT motifs detected in file\n",
      "\n",
      "Complement found from 5585 to 5591\n",
      "Seq is: CTAAGG\n",
      "1 CTAAGG motifs detected in file\n",
      "\n",
      "Complement found from 16563 to 16569\n",
      "Seq is: ACGATG\n",
      "1 ACGATG motifs detected in file\n",
      "\n",
      "Complement found from 4289 to 4295\n",
      "Seq is: TTGATA\n",
      "1 TTGATA motifs detected in file\n",
      "\n",
      "Complement found from 11026 to 11032\n",
      "Complement found from 16561 to 16567\n",
      "Seq is: TCACGA\n",
      "2 TCACGA motifs detected in file\n",
      "\n",
      "Complement found from 27 to 33\n",
      "Complement found from 977 to 983\n",
      "Complement found from 4190 to 4196\n",
      "Complement found from 7719 to 7725\n",
      "Complement found from 11257 to 11263\n",
      "Complement found from 11292 to 11298\n",
      "Complement found from 11767 to 11773\n",
      "Complement found from 12012 to 12018\n",
      "Complement found from 13796 to 13802\n",
      "Complement found from 14400 to 14406\n",
      "Complement found from 14909 to 14915\n",
      "Complement found from 16065 to 16071\n",
      "Seq is: ACTCAC\n",
      "12 ACTCAC motifs detected in file\n",
      "\n",
      "Complement found from 144 to 150\n",
      "Complement found from 480 to 486\n",
      "Complement found from 4520 to 4526\n",
      "Complement found from 4747 to 4753\n",
      "Complement found from 5294 to 5300\n",
      "Complement found from 5453 to 5459\n",
      "Complement found from 6982 to 6988\n",
      "Complement found from 7807 to 7813\n",
      "Complement found from 8428 to 8434\n",
      "Complement found from 8634 to 8640\n",
      "Complement found from 11239 to 11245\n",
      "Complement found from 11662 to 11668\n",
      "Complement found from 12699 to 12705\n",
      "Complement found from 12795 to 12801\n",
      "Complement found from 13113 to 13119\n",
      "Complement found from 13572 to 13578\n",
      "Complement found from 14118 to 14124\n",
      "Complement found from 15640 to 15646\n",
      "Seq is: CTCATC\n",
      "18 CTCATC motifs detected in file\n",
      "\n",
      "Complement found from 7916 to 7922\n",
      "Complement found from 8645 to 8651\n",
      "Seq is: CCGACT\n",
      "2 CCGACT motifs detected in file\n",
      "\n",
      "Complement found from 14276 to 14282\n",
      "Complement found from 14420 to 14426\n",
      "Seq is: CTGACC\n",
      "2 CTGACC motifs detected in file\n",
      "\n",
      "Complement found from 3210 to 3216\n",
      "Complement found from 3793 to 3799\n",
      "Complement found from 5205 to 5211\n",
      "Complement found from 5433 to 5439\n",
      "Complement found from 6313 to 6319\n",
      "Complement found from 7400 to 7406\n",
      "Complement found from 8656 to 8662\n",
      "Complement found from 8809 to 8815\n",
      "Complement found from 9055 to 9061\n",
      "Complement found from 10158 to 10164\n",
      "Complement found from 10841 to 10847\n",
      "Complement found from 12358 to 12364\n",
      "Complement found from 12397 to 12403\n",
      "Complement found from 13022 to 13028\n",
      "Complement found from 13054 to 13060\n",
      "Complement found from 13124 to 13130\n",
      "Complement found from 13648 to 13654\n",
      "Complement found from 13678 to 13684\n",
      "Complement found from 14336 to 14342\n",
      "Complement found from 14810 to 14816\n",
      "Complement found from 15263 to 15269\n",
      "Complement found from 15405 to 15411\n",
      "Complement found from 16051 to 16057\n",
      "Complement found from 16255 to 16261\n",
      "Complement found from 16290 to 16296\n",
      "Seq is: CCACCC\n",
      "25 CCACCC motifs detected in file\n",
      "\n",
      "Complement found from 6886 to 6892\n",
      "Complement found from 13712 to 13718\n",
      "Seq is: CGGAAG\n",
      "2 CGGAAG motifs detected in file\n",
      "\n",
      "Complement found from 1257 to 1263\n",
      "Seq is: ATACCG\n",
      "1 ATACCG motifs detected in file\n",
      "\n",
      "Complement found from 463 to 469\n",
      "Complement found from 5380 to 5386\n",
      "Complement found from 6309 to 6315\n",
      "Complement found from 8013 to 8019\n",
      "Complement found from 10623 to 10629\n",
      "Complement found from 11403 to 11409\n",
      "Complement found from 11811 to 11817\n",
      "Complement found from 11975 to 11981\n",
      "Complement found from 13031 to 13037\n",
      "Seq is: ACTCCC\n",
      "9 ACTCCC motifs detected in file\n",
      "\n",
      "Complement found from 5551 to 5557\n",
      "Complement found from 15249 to 15255\n",
      "Seq is: CTCAGT\n",
      "2 CTCAGT motifs detected in file\n",
      "\n",
      "Complement found from 569 to 575\n",
      "Complement found from 802 to 808\n",
      "Complement found from 1082 to 1088\n",
      "Complement found from 5302 to 5308\n",
      "Complement found from 5446 to 5452\n",
      "Complement found from 5600 to 5606\n",
      "Complement found from 5895 to 5901\n",
      "Complement found from 6844 to 6850\n",
      "Complement found from 7398 to 7404\n",
      "Complement found from 8079 to 8085\n",
      "Complement found from 8558 to 8564\n",
      "Complement found from 8618 to 8624\n",
      "Complement found from 11139 to 11145\n",
      "Complement found from 11868 to 11874\n",
      "Complement found from 12482 to 12488\n",
      "Complement found from 12967 to 12973\n",
      "Complement found from 13052 to 13058\n",
      "Complement found from 13646 to 13652\n",
      "Complement found from 13676 to 13682\n",
      "Complement found from 13879 to 13885\n",
      "Complement found from 14388 to 14394\n",
      "Complement found from 14619 to 14625\n",
      "Complement found from 14808 to 14814\n",
      "Complement found from 15541 to 15547\n",
      "Seq is: CCCCAC\n",
      "24 CCCCAC motifs detected in file\n",
      "\n",
      "Seq is: CTCTGG\n",
      "0 CTCTGG motifs detected in file\n",
      "\n",
      "Complement found from 20 to 26\n",
      "Complement found from 822 to 828\n",
      "Complement found from 909 to 915\n",
      "Complement found from 1859 to 1865\n",
      "Complement found from 2369 to 2375\n",
      "Complement found from 8314 to 8320\n",
      "Complement found from 9075 to 9081\n",
      "Complement found from 10024 to 10030\n",
      "Complement found from 11875 to 11881\n",
      "Complement found from 12025 to 12031\n",
      "Complement found from 13662 to 13668\n",
      "Complement found from 14785 to 14791\n",
      "Complement found from 15468 to 15474\n",
      "Seq is: ATTAAC\n",
      "13 ATTAAC motifs detected in file\n",
      "\n",
      "Complement found from 7916 to 7922\n",
      "Complement found from 8645 to 8651\n",
      "Seq is: CCGACT\n",
      "2 CCGACT motifs detected in file\n",
      "\n",
      "Complement found from 15350 to 15356\n",
      "Seq is: AAACGG\n",
      "1 AAACGG motifs detected in file\n",
      "\n",
      "Seq is: ACGACG\n",
      "0 ACGACG motifs detected in file\n",
      "\n",
      "Complement found from 737 to 743\n",
      "Complement found from 7856 to 7862\n",
      "Seq is: ACGATC\n",
      "2 ACGATC motifs detected in file\n",
      "\n",
      "Complement found from 591 to 597\n",
      "Complement found from 1494 to 1500\n",
      "Complement found from 2251 to 2257\n",
      "Complement found from 2325 to 2331\n",
      "Complement found from 3327 to 3333\n",
      "Complement found from 4928 to 4934\n",
      "Complement found from 5011 to 5017\n",
      "Complement found from 5210 to 5216\n",
      "Complement found from 6367 to 6373\n",
      "Complement found from 8775 to 8781\n",
      "Complement found from 9329 to 9335\n",
      "Complement found from 12091 to 12097\n",
      "Complement found from 12981 to 12987\n",
      "Complement found from 13986 to 13992\n",
      "Complement found from 14442 to 14448\n",
      "Complement found from 15739 to 15745\n",
      "Complement found from 16443 to 16449\n",
      "Seq is: CTCCTC\n",
      "17 CTCCTC motifs detected in file\n",
      "\n",
      "Complement found from 15057 to 15063\n",
      "Seq is: CGGATC\n",
      "1 CGGATC motifs detected in file\n",
      "\n",
      "Complement found from 12 to 18\n",
      "Complement found from 632 to 638\n",
      "Complement found from 731 to 737\n",
      "Complement found from 951 to 957\n",
      "Complement found from 2277 to 2283\n",
      "Complement found from 2508 to 2514\n",
      "Complement found from 2520 to 2526\n",
      "Complement found from 3507 to 3513\n",
      "Complement found from 3519 to 3525\n",
      "Complement found from 5321 to 5327\n",
      "Complement found from 5812 to 5818\n",
      "Complement found from 7645 to 7651\n",
      "Complement found from 8431 to 8437\n",
      "Complement found from 8652 to 8658\n",
      "Complement found from 9566 to 9572\n",
      "Complement found from 9635 to 9641\n",
      "Complement found from 11155 to 11161\n",
      "Complement found from 13182 to 13188\n",
      "Complement found from 14061 to 14067\n",
      "Complement found from 14878 to 14884\n",
      "Complement found from 15376 to 15382\n",
      "Complement found from 15397 to 15403\n",
      "Seq is: ATCACC\n",
      "22 ATCACC motifs detected in file\n",
      "\n",
      "Complement found from 25 to 31\n",
      "Complement found from 461 to 467\n",
      "Complement found from 1689 to 1695\n",
      "Complement found from 4188 to 4194\n",
      "Complement found from 5602 to 5608\n",
      "Complement found from 9316 to 9322\n",
      "Complement found from 9511 to 9517\n",
      "Complement found from 9593 to 9599\n",
      "Complement found from 10621 to 10627\n",
      "Complement found from 13076 to 13082\n",
      "Complement found from 13186 to 13192\n",
      "Complement found from 14115 to 14121\n",
      "Complement found from 14790 to 14796\n",
      "Seq is: CCACTC\n",
      "13 CCACTC motifs detected in file\n",
      "\n",
      "Complement found from 543 to 549\n",
      "Complement found from 1596 to 1602\n",
      "Complement found from 3888 to 3894\n",
      "Complement found from 4344 to 4350\n",
      "Complement found from 7461 to 7467\n",
      "Complement found from 8721 to 8727\n",
      "Seq is: CGAACC\n",
      "6 CGAACC motifs detected in file\n",
      "\n",
      "Complement found from 1211 to 1217\n",
      "Complement found from 3249 to 3255\n",
      "Complement found from 3346 to 3352\n",
      "Complement found from 6123 to 6129\n",
      "Complement found from 6159 to 6165\n",
      "Complement found from 10769 to 10775\n",
      "Complement found from 11705 to 11711\n",
      "Seq is: TAATCG\n",
      "7 TAATCG motifs detected in file\n",
      "\n",
      "Complement found from 2478 to 2484\n",
      "Complement found from 4898 to 4904\n",
      "Complement found from 5623 to 5629\n",
      "Complement found from 7875 to 7881\n",
      "Complement found from 10888 to 10894\n",
      "Complement found from 11065 to 11071\n",
      "Complement found from 13001 to 13007\n",
      "Complement found from 14043 to 14049\n",
      "Complement found from 14875 to 14881\n",
      "Complement found from 15951 to 15957\n",
      "Complement found from 16347 to 16353\n",
      "Seq is: CAAATC\n",
      "11 CAAATC motifs detected in file\n",
      "\n",
      "Complement found from 4458 to 4464\n",
      "Complement found from 8108 to 8114\n",
      "Seq is: TTCCCG\n",
      "2 TTCCCG motifs detected in file\n",
      "\n",
      "Complement found from 640 to 646\n",
      "Complement found from 836 to 842\n",
      "Complement found from 1217 to 1223\n",
      "Complement found from 4005 to 4011\n",
      "Complement found from 4571 to 4577\n",
      "Complement found from 4610 to 4616\n",
      "Complement found from 6191 to 6197\n",
      "Complement found from 7370 to 7376\n",
      "Complement found from 8823 to 8829\n",
      "Complement found from 10058 to 10064\n",
      "Complement found from 12657 to 12663\n",
      "Complement found from 15898 to 15904\n",
      "Seq is: ATAAAC\n",
      "12 ATAAAC motifs detected in file\n",
      "\n",
      "Complement found from 372 to 378\n",
      "Complement found from 2159 to 2165\n",
      "Complement found from 2212 to 2218\n",
      "Complement found from 3814 to 3820\n",
      "Complement found from 4008 to 4014\n",
      "Complement found from 5188 to 5194\n",
      "Complement found from 6548 to 6554\n",
      "Complement found from 6597 to 6603\n",
      "Complement found from 8346 to 8352\n",
      "Complement found from 10085 to 10091\n",
      "Complement found from 10616 to 10622\n",
      "Complement found from 11263 to 11269\n",
      "Complement found from 12058 to 12064\n",
      "Complement found from 14739 to 14745\n",
      "Complement found from 15532 to 15538\n",
      "Seq is: AACACC\n",
      "15 AACACC motifs detected in file\n",
      "\n",
      "Complement found from 1601 to 1607\n",
      "Complement found from 9741 to 9747\n",
      "Seq is: CAGAGT\n",
      "2 CAGAGT motifs detected in file\n",
      "\n",
      "Complement found from 1161 to 1167\n",
      "Complement found from 2887 to 2893\n",
      "Complement found from 3646 to 3652\n",
      "Complement found from 3676 to 3682\n",
      "Complement found from 5122 to 5128\n",
      "Complement found from 10135 to 10141\n",
      "Complement found from 11750 to 11756\n",
      "Complement found from 11949 to 11955\n",
      "Complement found from 13078 to 13084\n",
      "Complement found from 13421 to 13427\n",
      "Complement found from 14646 to 14652\n",
      "Complement found from 15873 to 15879\n",
      "Seq is: ACTCAA\n",
      "12 ACTCAA motifs detected in file\n",
      "\n",
      "Complement found from 1232 to 1238\n",
      "Complement found from 2253 to 2259\n",
      "Complement found from 2413 to 2419\n",
      "Complement found from 4013 to 4019\n",
      "Complement found from 4648 to 4654\n",
      "Complement found from 4908 to 4914\n",
      "Complement found from 4930 to 4936\n",
      "Complement found from 5889 to 5895\n",
      "Complement found from 8474 to 8480\n",
      "Complement found from 8790 to 8796\n",
      "Complement found from 9847 to 9853\n",
      "Complement found from 10980 to 10986\n",
      "Complement found from 11493 to 11499\n",
      "Complement found from 12962 to 12968\n",
      "Complement found from 13451 to 13457\n",
      "Complement found from 13490 to 13496\n",
      "Complement found from 15267 to 15273\n",
      "Complement found from 16260 to 16266\n",
      "Seq is: CCTCAC\n",
      "18 CCTCAC motifs detected in file\n",
      "\n",
      "Complement found from 147 to 153\n",
      "Complement found from 502 to 508\n",
      "Complement found from 2945 to 2951\n",
      "Complement found from 4394 to 4400\n",
      "Complement found from 5096 to 5102\n",
      "Complement found from 6641 to 6647\n",
      "Complement found from 7256 to 7262\n",
      "Complement found from 7570 to 7576\n",
      "Complement found from 7798 to 7804\n",
      "Complement found from 8565 to 8571\n",
      "Complement found from 9126 to 9132\n",
      "Complement found from 9455 to 9461\n",
      "Complement found from 10346 to 10352\n",
      "Complement found from 12849 to 12855\n",
      "Complement found from 12903 to 12909\n",
      "Complement found from 14121 to 14127\n",
      "Complement found from 14369 to 14375\n",
      "Complement found from 15643 to 15649\n",
      "Complement found from 15829 to 15835\n",
      "Complement found from 15835 to 15841\n",
      "Seq is: ATCCTA\n",
      "20 ATCCTA motifs detected in file\n",
      "\n",
      "Complement found from 9995 to 10001\n",
      "Seq is: TTTAGT\n",
      "1 TTTAGT motifs detected in file\n",
      "\n",
      "Complement found from 906 to 912\n",
      "Complement found from 3044 to 3050\n",
      "Complement found from 15273 to 15279\n",
      "Seq is: ACGATT\n",
      "3 ACGATT motifs detected in file\n",
      "\n",
      "Complement found from 1661 to 1667\n",
      "Seq is: CCGCTC\n",
      "1 CCGCTC motifs detected in file\n",
      "\n",
      "Complement found from 3148 to 3154\n",
      "Complement found from 4095 to 4101\n",
      "Complement found from 5338 to 5344\n",
      "Complement found from 5726 to 5732\n",
      "Complement found from 6482 to 6488\n",
      "Complement found from 7619 to 7625\n",
      "Complement found from 11938 to 11944\n",
      "Complement found from 12584 to 12590\n",
      "Complement found from 16507 to 16513\n",
      "Seq is: CTACTT\n",
      "9 CTACTT motifs detected in file\n",
      "\n",
      "Complement found from 11147 to 11153\n",
      "Seq is: TGGCTA\n",
      "1 TGGCTA motifs detected in file\n",
      "\n",
      "Seq is: TAGCGG\n",
      "0 TAGCGG motifs detected in file\n",
      "\n",
      "Complement found from 1087 to 1093\n",
      "Complement found from 10559 to 10565\n",
      "Complement found from 13167 to 13173\n",
      "Complement found from 13884 to 13890\n",
      "Seq is: CTATGC\n",
      "4 CTATGC motifs detected in file\n",
      "\n",
      "Complement found from 3302 to 3308\n",
      "Complement found from 3760 to 3766\n",
      "Complement found from 3938 to 3944\n",
      "Complement found from 4039 to 4045\n",
      "Complement found from 4072 to 4078\n",
      "Complement found from 6196 to 6202\n",
      "Complement found from 9789 to 9795\n",
      "Complement found from 10039 to 10045\n",
      "Complement found from 11952 to 11958\n",
      "Complement found from 12030 to 12036\n",
      "Complement found from 12249 to 12255\n",
      "Complement found from 13745 to 13751\n",
      "Complement found from 13904 to 13910\n",
      "Complement found from 14820 to 14826\n",
      "Seq is: CAACAT\n",
      "14 CAACAT motifs detected in file\n",
      "\n",
      "Complement found from 147 to 153\n",
      "Complement found from 502 to 508\n",
      "Complement found from 2945 to 2951\n",
      "Complement found from 4394 to 4400\n",
      "Complement found from 5096 to 5102\n",
      "Complement found from 6641 to 6647\n",
      "Complement found from 7256 to 7262\n",
      "Complement found from 7570 to 7576\n",
      "Complement found from 7798 to 7804\n",
      "Complement found from 8565 to 8571\n",
      "Complement found from 9126 to 9132\n",
      "Complement found from 9455 to 9461\n",
      "Complement found from 10346 to 10352\n",
      "Complement found from 12849 to 12855\n",
      "Complement found from 12903 to 12909\n",
      "Complement found from 14121 to 14127\n",
      "Complement found from 14369 to 14375\n",
      "Complement found from 15643 to 15649\n",
      "Complement found from 15829 to 15835\n",
      "Complement found from 15835 to 15841\n",
      "Seq is: ATCCTA\n",
      "20 ATCCTA motifs detected in file\n",
      "\n",
      "Complement found from 1595 to 1601\n",
      "Complement found from 8720 to 8726\n",
      "Complement found from 11759 to 11765\n",
      "Seq is: ACGAAC\n",
      "3 ACGAAC motifs detected in file\n",
      "\n",
      "Complement found from 3370 to 3376\n",
      "Complement found from 12119 to 12125\n",
      "Complement found from 12201 to 12207\n",
      "Complement found from 12730 to 12736\n",
      "Complement found from 16323 to 16329\n",
      "Seq is: TTACCG\n",
      "5 TTACCG motifs detected in file\n",
      "\n",
      "Complement found from 7005 to 7011\n",
      "Complement found from 9193 to 9199\n",
      "Complement found from 11589 to 11595\n",
      "Seq is: ACGACA\n",
      "3 ACGACA motifs detected in file\n",
      "\n",
      "Complement found from 3884 to 3890\n",
      "Complement found from 4659 to 4665\n",
      "Complement found from 8642 to 8648\n",
      "Complement found from 9667 to 9673\n",
      "Complement found from 12857 to 12863\n",
      "Complement found from 14924 to 14930\n",
      "Complement found from 16078 to 16084\n",
      "Seq is: CAACCG\n",
      "7 CAACCG motifs detected in file\n",
      "\n",
      "Complement found from 619 to 625\n",
      "Complement found from 6792 to 6798\n",
      "Seq is: TAGACG\n",
      "2 TAGACG motifs detected in file\n",
      "\n",
      "Complement found from 3242 to 3248\n",
      "Complement found from 15551 to 15557\n",
      "Seq is: AGCCCG\n",
      "2 AGCCCG motifs detected in file\n",
      "\n",
      "Complement found from 2165 to 2171\n",
      "Complement found from 6106 to 6112\n",
      "Complement found from 9650 to 9656\n",
      "Complement found from 16305 to 16311\n",
      "Seq is: CATAGT\n",
      "4 CATAGT motifs detected in file\n",
      "\n",
      "Complement found from 1002 to 1008\n",
      "Complement found from 1452 to 1458\n",
      "Complement found from 4964 to 4970\n",
      "Complement found from 12801 to 12807\n",
      "Seq is: AGTTGA\n",
      "4 AGTTGA motifs detected in file\n",
      "\n",
      "Complement found from 3324 to 3330\n",
      "Complement found from 4495 to 4501\n",
      "Complement found from 5120 to 5126\n",
      "Complement found from 5359 to 5365\n",
      "Complement found from 5378 to 5384\n",
      "Complement found from 5471 to 5477\n",
      "Complement found from 6230 to 6236\n",
      "Complement found from 6307 to 6313\n",
      "Complement found from 6682 to 6688\n",
      "Complement found from 8970 to 8976\n",
      "Complement found from 9030 to 9036\n",
      "Complement found from 10599 to 10605\n",
      "Complement found from 10751 to 10757\n",
      "Complement found from 11236 to 11242\n",
      "Complement found from 11290 to 11296\n",
      "Complement found from 11809 to 11815\n",
      "Complement found from 12696 to 12702\n",
      "Complement found from 13071 to 13077\n",
      "Complement found from 13419 to 13425\n",
      "Complement found from 13502 to 13508\n",
      "Complement found from 13983 to 13989\n",
      "Complement found from 14130 to 14136\n",
      "Complement found from 14907 to 14913\n",
      "Complement found from 15069 to 15075\n",
      "Complement found from 15246 to 15252\n",
      "Complement found from 16438 to 16444\n",
      "Seq is: CTACTC\n",
      "26 CTACTC motifs detected in file\n",
      "\n",
      "Complement found from 4112 to 4118\n",
      "Complement found from 12683 to 12689\n",
      "Complement found from 16022 to 16028\n",
      "Seq is: GTTCTT\n",
      "3 GTTCTT motifs detected in file\n",
      "\n",
      "Complement found from 1252 to 1258\n",
      "Complement found from 1838 to 1844\n",
      "Complement found from 2282 to 2288\n",
      "Complement found from 7572 to 7578\n",
      "Complement found from 8262 to 8268\n",
      "Complement found from 10184 to 10190\n",
      "Complement found from 11970 to 11976\n",
      "Complement found from 12851 to 12857\n",
      "Complement found from 13597 to 13603\n",
      "Complement found from 15048 to 15054\n",
      "Seq is: CCTATA\n",
      "10 CCTATA motifs detected in file\n",
      "\n",
      "Complement found from 1405 to 1411\n",
      "Seq is: TTAAGG\n",
      "1 TTAAGG motifs detected in file\n",
      "\n",
      "Seq is: ATGCGG\n",
      "0 ATGCGG motifs detected in file\n",
      "\n",
      "Complement found from 2144 to 2150\n",
      "Seq is: GAGAGT\n",
      "1 GAGAGT motifs detected in file\n",
      "\n",
      "Complement found from 10141 to 10147\n",
      "Complement found from 11475 to 11481\n",
      "Seq is: CGGCTA\n",
      "2 CGGCTA motifs detected in file\n",
      "\n",
      "Seq is: CTTGTC\n",
      "0 CTTGTC motifs detected in file\n",
      "\n",
      "Complement found from 825 to 831\n",
      "Complement found from 1711 to 1717\n",
      "Complement found from 2133 to 2139\n",
      "Complement found from 6038 to 6044\n",
      "Complement found from 8317 to 8323\n",
      "Complement found from 9078 to 9084\n",
      "Complement found from 10920 to 10926\n",
      "Complement found from 13380 to 13386\n",
      "Complement found from 15935 to 15941\n",
      "Seq is: AACCTT\n",
      "9 AACCTT motifs detected in file\n",
      "\n",
      "Complement found from 2896 to 2902\n",
      "Complement found from 4361 to 4367\n",
      "Complement found from 9150 to 9156\n",
      "Complement found from 9889 to 9895\n",
      "Complement found from 11665 to 11671\n",
      "Complement found from 12955 to 12961\n",
      "Complement found from 13152 to 13158\n",
      "Complement found from 14468 to 14474\n",
      "Complement found from 14817 to 14823\n",
      "Complement found from 15672 to 15678\n",
      "Seq is: ATCCAA\n",
      "10 ATCCAA motifs detected in file\n",
      "\n",
      "Complement found from 8001 to 8007\n",
      "Complement found from 13282 to 13288\n",
      "Complement found from 14939 to 14945\n",
      "Seq is: CAATCG\n",
      "3 CAATCG motifs detected in file\n",
      "\n",
      "Complement found from 12287 to 12293\n",
      "Complement found from 12448 to 12454\n",
      "Complement found from 13456 to 13462\n",
      "Seq is: CCATTG\n",
      "3 CCATTG motifs detected in file\n",
      "\n",
      "Complement found from 2950 to 2956\n",
      "Complement found from 3342 to 3348\n",
      "Complement found from 3384 to 3390\n",
      "Complement found from 3618 to 3624\n",
      "Complement found from 3750 to 3756\n",
      "Complement found from 6587 to 6593\n",
      "Complement found from 8595 to 8601\n",
      "Complement found from 9108 to 9114\n",
      "Complement found from 9114 to 9120\n",
      "Complement found from 11208 to 11214\n",
      "Complement found from 11284 to 11290\n",
      "Complement found from 13916 to 13922\n",
      "Complement found from 15745 to 15751\n",
      "Complement found from 16000 to 16006\n",
      "Seq is: ATTCTA\n",
      "14 ATTCTA motifs detected in file\n",
      "\n",
      "Complement found from 277 to 283\n",
      "Complement found from 4760 to 4766\n",
      "Complement found from 4889 to 4895\n",
      "Complement found from 4952 to 4958\n",
      "Complement found from 5309 to 5315\n",
      "Complement found from 6119 to 6125\n",
      "Complement found from 7067 to 7073\n",
      "Complement found from 7148 to 7154\n",
      "Complement found from 7630 to 7636\n",
      "Complement found from 9228 to 9234\n",
      "Complement found from 10455 to 10461\n",
      "Complement found from 11104 to 11110\n",
      "Complement found from 11779 to 11785\n",
      "Complement found from 12633 to 12639\n",
      "Complement found from 13538 to 13544\n",
      "Complement found from 14232 to 14238\n",
      "Complement found from 14343 to 14349\n",
      "Seq is: ATCATA\n",
      "17 ATCATA motifs detected in file\n",
      "\n",
      "Complement found from 4228 to 4234\n",
      "Complement found from 5262 to 5268\n",
      "Complement found from 7957 to 7963\n",
      "Complement found from 9073 to 9079\n",
      "Complement found from 12434 to 12440\n",
      "Complement found from 13693 to 13699\n",
      "Complement found from 14630 to 14636\n",
      "Complement found from 15979 to 15985\n",
      "Seq is: CCATTA\n",
      "8 CCATTA motifs detected in file\n",
      "\n",
      "Complement found from 4673 to 4679\n",
      "Complement found from 7831 to 7837\n",
      "Complement found from 8744 to 8750\n",
      "Complement found from 12390 to 12396\n",
      "Complement found from 12785 to 12791\n",
      "Seq is: ATCCTT\n",
      "5 ATCCTT motifs detected in file\n",
      "\n",
      "Complement found from 3324 to 3330\n",
      "Complement found from 4495 to 4501\n",
      "Complement found from 5120 to 5126\n",
      "Complement found from 5359 to 5365\n",
      "Complement found from 5378 to 5384\n",
      "Complement found from 5471 to 5477\n",
      "Complement found from 6230 to 6236\n",
      "Complement found from 6307 to 6313\n",
      "Complement found from 6682 to 6688\n",
      "Complement found from 8970 to 8976\n",
      "Complement found from 9030 to 9036\n",
      "Complement found from 10599 to 10605\n",
      "Complement found from 10751 to 10757\n",
      "Complement found from 11236 to 11242\n",
      "Complement found from 11290 to 11296\n",
      "Complement found from 11809 to 11815\n",
      "Complement found from 12696 to 12702\n",
      "Complement found from 13071 to 13077\n",
      "Complement found from 13419 to 13425\n",
      "Complement found from 13502 to 13508\n",
      "Complement found from 13983 to 13989\n",
      "Complement found from 14130 to 14136\n",
      "Complement found from 14907 to 14913\n",
      "Complement found from 15069 to 15075\n",
      "Complement found from 15246 to 15252\n",
      "Complement found from 16438 to 16444\n",
      "Seq is: CTACTC\n",
      "26 CTACTC motifs detected in file\n",
      "\n",
      "Complement found from 2273 to 2279\n",
      "Complement found from 8819 to 8825\n",
      "Complement found from 9971 to 9977\n",
      "Complement found from 13562 to 13568\n",
      "Seq is: ATCTAT\n",
      "4 ATCTAT motifs detected in file\n",
      "\n",
      "Complement found from 1401 to 1407\n",
      "Complement found from 3259 to 3265\n",
      "Complement found from 3266 to 3272\n",
      "Complement found from 4178 to 4184\n",
      "Complement found from 10060 to 10066\n",
      "Complement found from 13863 to 13869\n",
      "Complement found from 14088 to 14094\n",
      "Complement found from 14838 to 14844\n",
      "Complement found from 15186 to 15192\n",
      "Seq is: AAACTT\n",
      "9 AAACTT motifs detected in file\n",
      "\n",
      "Complement found from 1656 to 1662\n",
      "Complement found from 2905 to 2911\n",
      "Complement found from 7990 to 7996\n",
      "Complement found from 16141 to 16147\n",
      "Complement found from 16394 to 16400\n",
      "Seq is: CTTGAC\n",
      "5 CTTGAC motifs detected in file\n",
      "\n",
      "Complement found from 208 to 214\n",
      "Complement found from 4467 to 4473\n",
      "Complement found from 5637 to 5643\n",
      "Complement found from 8042 to 8048\n",
      "Complement found from 8399 to 8405\n",
      "Complement found from 10110 to 10116\n",
      "Complement found from 10853 to 10859\n",
      "Complement found from 11075 to 11081\n",
      "Complement found from 11564 to 11570\n",
      "Complement found from 12709 to 12715\n",
      "Complement found from 14083 to 14089\n",
      "Complement found from 14783 to 14789\n",
      "Complement found from 15179 to 15185\n",
      "Seq is: TAATTA\n",
      "13 TAATTA motifs detected in file\n",
      "\n",
      "Complement found from 298 to 304\n",
      "Complement found from 348 to 354\n",
      "Complement found from 552 to 558\n",
      "Complement found from 1273 to 1279\n",
      "Complement found from 1684 to 1690\n",
      "Complement found from 1720 to 1726\n",
      "Complement found from 1991 to 1997\n",
      "Complement found from 2765 to 2771\n",
      "Complement found from 2788 to 2794\n",
      "Complement found from 4254 to 4260\n",
      "Complement found from 4982 to 4988\n",
      "Complement found from 5931 to 5937\n",
      "Complement found from 7121 to 7127\n",
      "Complement found from 8125 to 8131\n",
      "Complement found from 8189 to 8195\n",
      "Complement found from 8508 to 8514\n",
      "Complement found from 11668 to 11674\n",
      "Complement found from 14624 to 14630\n",
      "Complement found from 16281 to 16287\n",
      "Seq is: CAAACC\n",
      "19 CAAACC motifs detected in file\n",
      "\n",
      "Seq is: CAGACT\n",
      "0 CAGACT motifs detected in file\n",
      "\n",
      "Complement found from 645 to 651\n",
      "Complement found from 1733 to 1739\n",
      "Complement found from 8627 to 8633\n",
      "Complement found from 9677 to 9683\n",
      "Complement found from 11924 to 11930\n",
      "Complement found from 12324 to 12330\n",
      "Complement found from 12689 to 12695\n",
      "Complement found from 12933 to 12939\n",
      "Complement found from 15154 to 15160\n",
      "Seq is: CAAATA\n",
      "9 CAAATA motifs detected in file\n",
      "\n",
      "Complement found from 1777 to 1783\n",
      "Complement found from 6250 to 6256\n",
      "Complement found from 9232 to 9238\n",
      "Complement found from 10409 to 10415\n",
      "Complement found from 11367 to 11373\n",
      "Complement found from 12142 to 12148\n",
      "Complement found from 13088 to 13094\n",
      "Seq is: TATAGT\n",
      "7 TATAGT motifs detected in file\n",
      "\n",
      "Complement found from 13812 to 13818\n",
      "Seq is: GTCACT\n",
      "1 GTCACT motifs detected in file\n",
      "\n",
      "Complement found from 1404 to 1410\n",
      "Seq is: CTTAAG\n",
      "1 CTTAAG motifs detected in file\n",
      "\n",
      "Complement found from 469 to 475\n",
      "Complement found from 859 to 865\n",
      "Complement found from 4731 to 4737\n",
      "Complement found from 5495 to 5501\n",
      "Complement found from 6524 to 6530\n",
      "Complement found from 7739 to 7745\n",
      "Complement found from 8157 to 8163\n",
      "Complement found from 8378 to 8384\n",
      "Complement found from 8736 to 8742\n",
      "Complement found from 8943 to 8949\n",
      "Complement found from 9335 to 9341\n",
      "Complement found from 10493 to 10499\n",
      "Complement found from 10523 to 10529\n",
      "Complement found from 10577 to 10583\n",
      "Complement found from 10655 to 10661\n",
      "Complement found from 10957 to 10963\n",
      "Complement found from 11050 to 11056\n",
      "Complement found from 11956 to 11962\n",
      "Complement found from 12717 to 12723\n",
      "Complement found from 13347 to 13353\n",
      "Complement found from 14579 to 14585\n",
      "Seq is: ATACTA\n",
      "21 ATACTA motifs detected in file\n",
      "\n",
      "Complement found from 2004 to 2010\n",
      "Complement found from 3587 to 3593\n",
      "Seq is: CCTGGT\n",
      "2 CCTGGT motifs detected in file\n",
      "\n",
      "Complement found from 5943 to 5949\n",
      "Complement found from 15209 to 15215\n",
      "Seq is: ACATTG\n",
      "2 ACATTG motifs detected in file\n",
      "\n",
      "Complement found from 2604 to 2610\n",
      "Seq is: CTTGTT\n",
      "1 CTTGTT motifs detected in file\n",
      "\n",
      "Complement found from 16563 to 16569\n",
      "Seq is: ACGATG\n",
      "1 ACGATG motifs detected in file\n",
      "\n",
      "Complement found from 5216 to 5222\n",
      "Complement found from 7822 to 7828\n",
      "Complement found from 10553 to 10559\n",
      "Complement found from 11059 to 11065\n",
      "Complement found from 11405 to 11411\n",
      "Complement found from 11551 to 11557\n",
      "Complement found from 12375 to 12381\n",
      "Complement found from 12564 to 12570\n",
      "Complement found from 15596 to 15602\n",
      "Complement found from 15853 to 15859\n",
      "Seq is: TCCCTA\n",
      "10 TCCCTA motifs detected in file\n",
      "\n",
      "Complement found from 4206 to 4212\n",
      "Complement found from 5506 to 5512\n",
      "Complement found from 8733 to 8739\n",
      "Seq is: CTTATA\n",
      "3 CTTATA motifs detected in file\n",
      "\n",
      "Complement found from 1841 to 1847\n",
      "Complement found from 5956 to 5962\n",
      "Complement found from 11377 to 11383\n",
      "Complement found from 13431 to 13437\n",
      "Complement found from 13482 to 13488\n",
      "Seq is: ATACCT\n",
      "5 ATACCT motifs detected in file\n",
      "\n",
      "Complement found from 1181 to 1187\n",
      "Complement found from 3974 to 3980\n",
      "Complement found from 6103 to 6109\n",
      "Complement found from 7078 to 7084\n",
      "Complement found from 8545 to 8551\n",
      "Complement found from 9859 to 9865\n",
      "Complement found from 14288 to 14294\n",
      "Complement found from 15126 to 15132\n",
      "Complement found from 15291 to 15297\n",
      "Complement found from 15303 to 15309\n",
      "Seq is: CTTCAT\n",
      "10 CTTCAT motifs detected in file\n",
      "\n",
      "Seq is: CAGATC\n",
      "0 CAGATC motifs detected in file\n",
      "\n",
      "Complement found from 1401 to 1407\n",
      "Complement found from 3259 to 3265\n",
      "Complement found from 3266 to 3272\n",
      "Complement found from 4178 to 4184\n",
      "Complement found from 10060 to 10066\n",
      "Complement found from 13863 to 13869\n",
      "Complement found from 14088 to 14094\n",
      "Complement found from 14838 to 14844\n",
      "Complement found from 15186 to 15192\n",
      "Seq is: AAACTT\n",
      "9 AAACTT motifs detected in file\n",
      "\n",
      "Complement found from 14266 to 14272\n",
      "Seq is: CGAATC\n",
      "1 CGAATC motifs detected in file\n",
      "\n",
      "Complement found from 3788 to 3794\n",
      "Complement found from 5212 to 5218\n",
      "Complement found from 6222 to 6228\n",
      "Complement found from 9268 to 9274\n",
      "Complement found from 11043 to 11049\n",
      "Complement found from 11787 to 11793\n",
      "Complement found from 13434 to 13440\n",
      "Complement found from 14099 to 14105\n",
      "Complement found from 14282 to 14288\n",
      "Seq is: CCTCTC\n",
      "9 CCTCTC motifs detected in file\n",
      "\n",
      "Complement found from 1061 to 1067\n",
      "Complement found from 9208 to 9214\n",
      "Complement found from 12666 to 12672\n",
      "Complement found from 12924 to 12930\n",
      "Complement found from 15499 to 15505\n",
      "Seq is: GACCCA\n",
      "5 GACCCA motifs detected in file\n",
      "\n",
      "Complement found from 1990 to 1996\n",
      "Complement found from 2764 to 2770\n",
      "Complement found from 3990 to 3996\n",
      "Complement found from 5679 to 5685\n",
      "Complement found from 5930 to 5936\n",
      "Complement found from 8455 to 8461\n",
      "Complement found from 8507 to 8513\n",
      "Complement found from 10294 to 10300\n",
      "Complement found from 11592 to 11598\n",
      "Complement found from 13545 to 13551\n",
      "Complement found from 13861 to 13867\n",
      "Complement found from 14188 to 14194\n",
      "Complement found from 14623 to 14629\n",
      "Complement found from 15184 to 15190\n",
      "Complement found from 15602 to 15608\n",
      "Complement found from 16280 to 16286\n",
      "Seq is: ACAAAC\n",
      "16 ACAAAC motifs detected in file\n",
      "\n",
      "Complement found from 2395 to 2401\n",
      "Complement found from 2426 to 2432\n",
      "Complement found from 6595 to 6601\n",
      "Complement found from 8344 to 8350\n",
      "Complement found from 8660 to 8666\n",
      "Complement found from 9351 to 9357\n",
      "Complement found from 9557 to 9563\n",
      "Complement found from 10713 to 10719\n",
      "Complement found from 10777 to 10783\n",
      "Complement found from 11330 to 11336\n",
      "Complement found from 12826 to 12832\n",
      "Complement found from 13858 to 13864\n",
      "Complement found from 13903 to 13909\n",
      "Complement found from 14185 to 14191\n",
      "Complement found from 14819 to 14825\n",
      "Complement found from 16277 to 16283\n",
      "Seq is: CCAACA\n",
      "16 CCAACA motifs detected in file\n",
      "\n",
      "Seq is: CCGAGT\n",
      "0 CCGAGT motifs detected in file\n",
      "\n",
      "Complement found from 572 to 578\n",
      "Complement found from 8194 to 8200\n",
      "Complement found from 8858 to 8864\n",
      "Complement found from 11770 to 11776\n",
      "Complement found from 15174 to 15180\n",
      "Seq is: CACAGT\n",
      "5 CACAGT motifs detected in file\n",
      "\n",
      "Complement found from 2205 to 2211\n",
      "Complement found from 5173 to 5179\n",
      "Complement found from 11573 to 11579\n",
      "Seq is: CAAGCT\n",
      "3 CAAGCT motifs detected in file\n",
      "\n",
      "Complement found from 18 to 24\n",
      "Complement found from 150 to 156\n",
      "Complement found from 669 to 675\n",
      "Complement found from 3022 to 3028\n",
      "Complement found from 5960 to 5966\n",
      "Complement found from 10233 to 10239\n",
      "Complement found from 11873 to 11879\n",
      "Complement found from 13564 to 13570\n",
      "Complement found from 14016 to 14022\n",
      "Complement found from 14314 to 14320\n",
      "Complement found from 14510 to 14516\n",
      "Complement found from 15625 to 15631\n",
      "Seq is: CTATTA\n",
      "12 CTATTA motifs detected in file\n",
      "\n",
      "Complement found from 5774 to 5780\n",
      "Complement found from 8021 to 8027\n",
      "Seq is: TTGAAG\n",
      "2 TTGAAG motifs detected in file\n",
      "\n",
      "Complement found from 383 to 389\n",
      "Complement found from 1598 to 1604\n",
      "Complement found from 1907 to 1913\n",
      "Complement found from 2667 to 2673\n",
      "Complement found from 7970 to 7976\n",
      "Complement found from 11170 to 11176\n",
      "Complement found from 14201 to 14207\n",
      "Complement found from 15767 to 15773\n",
      "Seq is: AACCAG\n",
      "8 AACCAG motifs detected in file\n",
      "\n",
      "Complement found from 6460 to 6466\n",
      "Complement found from 9607 to 9613\n",
      "Complement found from 15591 to 15597\n",
      "Complement found from 15807 to 15813\n",
      "Seq is: ATCCGT\n",
      "4 ATCCGT motifs detected in file\n",
      "\n",
      "Complement found from 9745 to 9751\n",
      "Complement found from 11446 to 11452\n",
      "Seq is: GTACTT\n",
      "2 GTACTT motifs detected in file\n",
      "\n",
      "Complement found from 237 to 243\n",
      "Complement found from 280 to 286\n",
      "Complement found from 2703 to 2709\n",
      "Complement found from 2934 to 2940\n",
      "Complement found from 7840 to 7846\n",
      "Complement found from 8504 to 8510\n",
      "Complement found from 11080 to 11086\n",
      "Complement found from 11569 to 11575\n",
      "Complement found from 12276 to 12282\n",
      "Complement found from 14548 to 14554\n",
      "Complement found from 16460 to 16466\n",
      "Seq is: ATAACA\n",
      "11 ATAACA motifs detected in file\n",
      "\n",
      "Complement found from 564 to 570\n",
      "Complement found from 798 to 804\n",
      "Complement found from 1942 to 1948\n",
      "Complement found from 2160 to 2166\n",
      "Complement found from 2213 to 2219\n",
      "Complement found from 2257 to 2263\n",
      "Complement found from 3206 to 3212\n",
      "Complement found from 4009 to 4015\n",
      "Complement found from 4387 to 4393\n",
      "Complement found from 5189 to 5195\n",
      "Complement found from 7110 to 7116\n",
      "Complement found from 8928 to 8934\n",
      "Complement found from 10086 to 10092\n",
      "Complement found from 10617 to 10623\n",
      "Complement found from 11213 to 11219\n",
      "Complement found from 11264 to 11270\n",
      "Complement found from 12059 to 12065\n",
      "Complement found from 14553 to 14559\n",
      "Complement found from 15533 to 15539\n",
      "Seq is: ACACCC\n",
      "19 ACACCC motifs detected in file\n",
      "\n",
      "Complement found from 4856 to 4862\n",
      "Complement found from 4925 to 4931\n",
      "Complement found from 5729 to 5735\n",
      "Complement found from 6343 to 6349\n",
      "Complement found from 6485 to 6491\n",
      "Complement found from 12587 to 12593\n",
      "Complement found from 13244 to 13250\n",
      "Complement found from 13617 to 13623\n",
      "Complement found from 15442 to 15448\n",
      "Complement found from 15451 to 15457\n",
      "Complement found from 16354 to 16360\n",
      "Seq is: CTTCTC\n",
      "11 CTTCTC motifs detected in file\n",
      "\n",
      "Complement found from 5183 to 5189\n",
      "Complement found from 5918 to 5924\n",
      "Complement found from 6143 to 6149\n",
      "Complement found from 7382 to 7388\n",
      "Complement found from 8667 to 8673\n",
      "Complement found from 9121 to 9127\n",
      "Complement found from 9950 to 9956\n",
      "Complement found from 10124 to 10130\n",
      "Complement found from 10373 to 10379\n",
      "Complement found from 11344 to 11350\n",
      "Complement found from 14004 to 14010\n",
      "Seq is: TGACTA\n",
      "11 TGACTA motifs detected in file\n",
      "\n",
      "Complement found from 153 to 159\n",
      "Complement found from 448 to 454\n",
      "Complement found from 4322 to 4328\n",
      "Complement found from 10247 to 10253\n",
      "Complement found from 12197 to 12203\n",
      "Complement found from 13895 to 13901\n",
      "Seq is: TTATTT\n",
      "6 TTATTT motifs detected in file\n",
      "\n",
      "Complement found from 1237 to 1243\n",
      "Complement found from 1694 to 1700\n",
      "Complement found from 3625 to 3631\n",
      "Complement found from 4378 to 4384\n",
      "Complement found from 5364 to 5370\n",
      "Complement found from 6552 to 6558\n",
      "Complement found from 8463 to 8469\n",
      "Complement found from 8620 to 8626\n",
      "Complement found from 9026 to 9032\n",
      "Complement found from 9422 to 9428\n",
      "Complement found from 11141 to 11147\n",
      "Complement found from 12460 to 12466\n",
      "Complement found from 14050 to 14056\n",
      "Complement found from 15330 to 15336\n",
      "Complement found from 16146 to 16152\n",
      "Seq is: CCACCT\n",
      "15 CCACCT motifs detected in file\n",
      "\n",
      "Complement found from 94 to 100\n",
      "Seq is: ACGCTG\n",
      "1 ACGCTG motifs detected in file\n",
      "\n",
      "Complement found from 587 to 593\n",
      "Complement found from 4551 to 4557\n",
      "Complement found from 5886 to 5892\n",
      "Complement found from 6215 to 6221\n",
      "Complement found from 9465 to 9471\n",
      "Complement found from 10236 to 10242\n",
      "Complement found from 14019 to 14025\n",
      "Complement found from 15281 to 15287\n",
      "Seq is: TTACCT\n",
      "8 TTACCT motifs detected in file\n",
      "\n",
      "Complement found from 611 to 617\n",
      "Complement found from 4440 to 4446\n",
      "Complement found from 5407 to 5413\n",
      "Complement found from 8523 to 8529\n",
      "Complement found from 8890 to 8896\n",
      "Complement found from 13219 to 13225\n",
      "Seq is: AAAATG\n",
      "6 AAAATG motifs detected in file\n",
      "\n",
      "Complement found from 176 to 182\n",
      "Complement found from 2407 to 2413\n",
      "Complement found from 9463 to 9469\n",
      "Complement found from 9612 to 9618\n",
      "Complement found from 9697 to 9703\n",
      "Complement found from 10114 to 10120\n",
      "Complement found from 10234 to 10240\n",
      "Complement found from 10787 to 10793\n",
      "Complement found from 13565 to 13571\n",
      "Complement found from 14017 to 14023\n",
      "Complement found from 15052 to 15058\n",
      "Complement found from 15626 to 15632\n",
      "Seq is: TATTAC\n",
      "12 TATTAC motifs detected in file\n",
      "\n",
      "Complement found from 533 to 539\n",
      "Complement found from 636 to 642\n",
      "Complement found from 3572 to 3578\n",
      "Complement found from 4390 to 4396\n",
      "Complement found from 4879 to 4885\n",
      "Complement found from 5383 to 5389\n",
      "Complement found from 5436 to 5442\n",
      "Complement found from 6583 to 6589\n",
      "Complement found from 7490 to 7496\n",
      "Complement found from 7955 to 7961\n",
      "Complement found from 8028 to 8034\n",
      "Complement found from 8406 to 8412\n",
      "Complement found from 8939 to 8945\n",
      "Complement found from 11427 to 11433\n",
      "Complement found from 12084 to 12090\n",
      "Complement found from 12237 to 12243\n",
      "Complement found from 12386 to 12392\n",
      "Complement found from 12432 to 12438\n",
      "Complement found from 13691 to 13697\n",
      "Complement found from 14339 to 14345\n",
      "Complement found from 14425 to 14431\n",
      "Complement found from 14588 to 14594\n",
      "Complement found from 14628 to 14634\n",
      "Complement found from 14813 to 14819\n",
      "Complement found from 15657 to 15663\n",
      "Complement found from 16189 to 16195\n",
      "Complement found from 16362 to 16368\n",
      "Seq is: CCCCAT\n",
      "27 CCCCAT motifs detected in file\n",
      "\n",
      "Complement found from 1161 to 1167\n",
      "Complement found from 2887 to 2893\n",
      "Complement found from 3646 to 3652\n",
      "Complement found from 3676 to 3682\n",
      "Complement found from 5122 to 5128\n",
      "Complement found from 10135 to 10141\n",
      "Complement found from 11750 to 11756\n",
      "Complement found from 11949 to 11955\n",
      "Complement found from 13078 to 13084\n",
      "Complement found from 13421 to 13427\n",
      "Complement found from 14646 to 14652\n",
      "Complement found from 15873 to 15879\n",
      "Seq is: ACTCAA\n",
      "12 ACTCAA motifs detected in file\n",
      "\n",
      "Complement found from 285 to 291\n",
      "Complement found from 992 to 998\n",
      "Complement found from 2129 to 2135\n",
      "Complement found from 2150 to 2156\n",
      "Complement found from 2226 to 2232\n",
      "Complement found from 2456 to 2462\n",
      "Complement found from 4175 to 4181\n",
      "Complement found from 4604 to 4610\n",
      "Complement found from 5281 to 5287\n",
      "Complement found from 5746 to 5752\n",
      "Complement found from 6691 to 6697\n",
      "Complement found from 7446 to 7452\n",
      "Complement found from 8496 to 8502\n",
      "Complement found from 10047 to 10053\n",
      "Complement found from 10813 to 10819\n",
      "Complement found from 11031 to 11037\n",
      "Complement found from 12417 to 12423\n",
      "Complement found from 13230 to 13236\n",
      "Complement found from 14503 to 14509\n",
      "Seq is: AAAAAA\n",
      "19 AAAAAA motifs detected in file\n",
      "\n",
      "Complement found from 2943 to 2949\n",
      "Complement found from 3649 to 3655\n",
      "Complement found from 8563 to 8569\n",
      "Complement found from 12847 to 12853\n",
      "Complement found from 13777 to 13783\n",
      "Complement found from 13942 to 13948\n",
      "Complement found from 14367 to 14373\n",
      "Complement found from 15827 to 15833\n",
      "Complement found from 16168 to 16174\n",
      "Seq is: CAATCC\n",
      "9 CAATCC motifs detected in file\n",
      "\n",
      "Complement found from 241 to 247\n",
      "Complement found from 2262 to 2268\n",
      "Complement found from 2890 to 2896\n",
      "Complement found from 6738 to 6744\n",
      "Complement found from 7880 to 7886\n",
      "Seq is: CAATTG\n",
      "5 CAATTG motifs detected in file\n",
      "\n",
      "Complement found from 458 to 464\n",
      "Complement found from 464 to 470\n",
      "Complement found from 6310 to 6316\n",
      "Complement found from 6497 to 6503\n",
      "Complement found from 6664 to 6670\n",
      "Complement found from 7816 to 7822\n",
      "Complement found from 11812 to 11818\n",
      "Complement found from 15381 to 15387\n",
      "Seq is: CTCCCA\n",
      "8 CTCCCA motifs detected in file\n",
      "\n",
      "Complement found from 3340 to 3346\n",
      "Complement found from 5070 to 5076\n",
      "Complement found from 5438 to 5444\n",
      "Complement found from 6585 to 6591\n",
      "Complement found from 8030 to 8036\n",
      "Complement found from 11657 to 11663\n",
      "Complement found from 12086 to 12092\n",
      "Complement found from 12838 to 12844\n",
      "Complement found from 15384 to 15390\n",
      "Seq is: CCATTC\n",
      "9 CCATTC motifs detected in file\n",
      "\n",
      "Complement found from 505 to 511\n",
      "Complement found from 1371 to 1377\n",
      "Complement found from 6286 to 6292\n",
      "Complement found from 7228 to 7234\n",
      "Complement found from 8277 to 8283\n",
      "Complement found from 8574 to 8580\n",
      "Complement found from 9528 to 9534\n",
      "Complement found from 10975 to 10981\n",
      "Complement found from 11532 to 11538\n",
      "Complement found from 13919 to 13925\n",
      "Complement found from 15777 to 15783\n",
      "Complement found from 16286 to 16292\n",
      "Seq is: CTACCC\n",
      "12 CTACCC motifs detected in file\n",
      "\n",
      "Complement found from 1393 to 1399\n",
      "Complement found from 3796 to 3802\n",
      "Complement found from 4319 to 4325\n",
      "Complement found from 5192 to 5198\n",
      "Complement found from 5460 to 5466\n",
      "Complement found from 5650 to 5656\n",
      "Complement found from 6293 to 6299\n",
      "Complement found from 7865 to 7871\n",
      "Complement found from 8844 to 8850\n",
      "Complement found from 8932 to 8938\n",
      "Complement found from 10162 to 10168\n",
      "Complement found from 12194 to 12200\n",
      "Complement found from 13210 to 13216\n",
      "Complement found from 13651 to 13657\n",
      "Complement found from 15408 to 15414\n",
      "Complement found from 15526 to 15532\n",
      "Complement found from 16293 to 16299\n",
      "Complement found from 16546 to 16552\n",
      "Seq is: CCCTTA\n",
      "18 CCCTTA motifs detected in file\n",
      "\n",
      "Complement found from 1187 to 1193\n",
      "Complement found from 4351 to 4357\n",
      "Complement found from 7821 to 7827\n",
      "Complement found from 7859 to 7865\n",
      "Complement found from 10327 to 10333\n",
      "Complement found from 10867 to 10873\n",
      "Complement found from 11550 to 11556\n",
      "Complement found from 12099 to 12105\n",
      "Complement found from 12603 to 12609\n",
      "Complement found from 16350 to 16356\n",
      "Seq is: ATCCCT\n",
      "10 ATCCCT motifs detected in file\n",
      "\n",
      "Complement found from 12472 to 12478\n",
      "Seq is: TCAGTC\n",
      "1 TCAGTC motifs detected in file\n",
      "\n",
      "Complement found from 2270 to 2276\n",
      "Complement found from 4737 to 4743\n",
      "Complement found from 9214 to 9220\n",
      "Complement found from 14366 to 14372\n",
      "Complement found from 15708 to 15714\n",
      "Complement found from 16167 to 16173\n",
      "Seq is: CCAATC\n",
      "6 CCAATC motifs detected in file\n",
      "\n",
      "Complement found from 294 to 300\n",
      "Complement found from 2185 to 2191\n",
      "Complement found from 5304 to 5310\n",
      "Complement found from 5316 to 5322\n",
      "Complement found from 7887 to 7893\n",
      "Complement found from 8393 to 8399\n",
      "Complement found from 9211 to 9217\n",
      "Complement found from 9412 to 9418\n",
      "Complement found from 12018 to 12024\n",
      "Complement found from 14333 to 14339\n",
      "Complement found from 15976 to 15982\n",
      "Complement found from 16110 to 16116\n",
      "Complement found from 16399 to 16405\n",
      "Seq is: CCACCA\n",
      "13 CCACCA motifs detected in file\n",
      "\n",
      "Complement found from 2962 to 2968\n",
      "Complement found from 3179 to 3185\n",
      "Complement found from 6734 to 6740\n",
      "Complement found from 9066 to 9072\n",
      "Complement found from 11927 to 11933\n",
      "Complement found from 13536 to 13542\n",
      "Complement found from 15157 to 15163\n",
      "Seq is: ATATCA\n",
      "7 ATATCA motifs detected in file\n",
      "\n",
      "Complement found from 902 to 908\n",
      "Complement found from 2255 to 2261\n",
      "Complement found from 4385 to 4391\n",
      "Complement found from 5373 to 5379\n",
      "Complement found from 9166 to 9172\n",
      "Complement found from 10538 to 10544\n",
      "Complement found from 11495 to 11501\n",
      "Complement found from 12047 to 12053\n",
      "Complement found from 13930 to 13936\n",
      "Complement found from 15269 to 15275\n",
      "Complement found from 16230 to 16236\n",
      "Seq is: TCACAC\n",
      "11 TCACAC motifs detected in file\n",
      "\n",
      "Complement found from 16048 to 16054\n",
      "Complement found from 16129 to 16135\n",
      "Seq is: GTACCA\n",
      "2 GTACCA motifs detected in file\n",
      "\n",
      "Complement found from 1852 to 1858\n",
      "Complement found from 9203 to 9209\n",
      "Seq is: ATAATG\n",
      "2 ATAATG motifs detected in file\n",
      "\n",
      "Complement found from 1011 to 1017\n",
      "Complement found from 1959 to 1965\n",
      "Complement found from 4607 to 4613\n",
      "Complement found from 5402 to 5408\n",
      "Complement found from 8443 to 8449\n",
      "Complement found from 8491 to 8497\n",
      "Complement found from 13138 to 13144\n",
      "Complement found from 13407 to 13413\n",
      "Complement found from 13669 to 13675\n",
      "Complement found from 13869 to 13875\n",
      "Complement found from 15868 to 15874\n",
      "Seq is: AAAATA\n",
      "11 AAAATA motifs detected in file\n",
      "\n",
      "Complement found from 1102 to 1108\n",
      "Complement found from 1230 to 1236\n",
      "Complement found from 2836 to 2842\n",
      "Complement found from 3318 to 3324\n",
      "Complement found from 3594 to 3600\n",
      "Complement found from 3786 to 3792\n",
      "Complement found from 4103 to 4109\n",
      "Complement found from 5333 to 5339\n",
      "Complement found from 6542 to 6548\n",
      "Complement found from 8681 to 8687\n",
      "Complement found from 8772 to 8778\n",
      "Complement found from 11854 to 11860\n",
      "Complement found from 13445 to 13451\n",
      "Complement found from 13636 to 13642\n",
      "Complement found from 14525 to 14531\n",
      "Seq is: AACCTC\n",
      "15 AACCTC motifs detected in file\n",
      "\n",
      "Seq is: AACGTG\n",
      "0 AACGTG motifs detected in file\n",
      "\n",
      "Complement found from 944 to 950\n",
      "Seq is: GTTTTA\n",
      "1 GTTTTA motifs detected in file\n",
      "\n",
      "Complement found from 1526 to 1532\n",
      "Complement found from 2131 to 2137\n",
      "Complement found from 3488 to 3494\n",
      "Complement found from 5428 to 5434\n",
      "Complement found from 5596 to 5602\n",
      "Complement found from 6415 to 6421\n",
      "Complement found from 7528 to 7534\n",
      "Complement found from 9238 to 9244\n",
      "Complement found from 12036 to 12042\n",
      "Complement found from 13425 to 13431\n",
      "Complement found from 13972 to 13978\n",
      "Complement found from 14615 to 14621\n",
      "Complement found from 14711 to 14717\n",
      "Complement found from 15933 to 15939\n",
      "Complement found from 16162 to 16168\n",
      "Complement found from 16179 to 16185\n",
      "Seq is: AAAACC\n",
      "16 AAAACC motifs detected in file\n",
      "\n",
      "Seq is: CCGGAT\n",
      "0 CCGGAT motifs detected in file\n",
      "\n",
      "Complement found from 16499 to 16505\n",
      "Seq is: TCTGGT\n",
      "1 TCTGGT motifs detected in file\n",
      "\n",
      "Complement found from 1919 to 1925\n",
      "Complement found from 2221 to 2227\n",
      "Complement found from 2758 to 2764\n",
      "Complement found from 5345 to 5351\n",
      "Complement found from 5478 to 5484\n",
      "Complement found from 5957 to 5963\n",
      "Complement found from 13788 to 13794\n",
      "Complement found from 13851 to 13857\n",
      "Complement found from 14020 to 14026\n",
      "Seq is: TACCTA\n",
      "9 TACCTA motifs detected in file\n",
      "\n",
      "Seq is: TCGACA\n",
      "0 TCGACA motifs detected in file\n",
      "\n",
      "Complement found from 4304 to 4310\n",
      "Complement found from 5027 to 5033\n",
      "Complement found from 7049 to 7055\n",
      "Complement found from 7070 to 7076\n",
      "Complement found from 13410 to 13416\n",
      "Complement found from 14254 to 14260\n",
      "Complement found from 14596 to 14602\n",
      "Seq is: ATAGGA\n",
      "7 ATAGGA motifs detected in file\n",
      "\n",
      "Complement found from 395 to 401\n",
      "Complement found from 7306 to 7312\n",
      "Complement found from 9703 to 9709\n",
      "Complement found from 10072 to 10078\n",
      "Complement found from 12308 to 12314\n",
      "Seq is: AATTTT\n",
      "5 AATTTT motifs detected in file\n",
      "\n",
      "Complement found from 4155 to 4161\n",
      "Complement found from 7938 to 7944\n",
      "Complement found from 10133 to 10139\n",
      "Complement found from 12318 to 12324\n",
      "Complement found from 12914 to 12920\n",
      "Complement found from 16244 to 16250\n",
      "Seq is: CAACTC\n",
      "6 CAACTC motifs detected in file\n",
      "\n",
      "Complement found from 2805 to 2811\n",
      "Complement found from 6613 to 6619\n",
      "Seq is: TTTCGG\n",
      "2 TTTCGG motifs detected in file\n",
      "\n",
      "Complement found from 488 to 494\n",
      "Complement found from 3397 to 3403\n",
      "Complement found from 3442 to 3448\n",
      "Complement found from 5054 to 5060\n",
      "Complement found from 6062 to 6068\n",
      "Complement found from 12855 to 12861\n",
      "Complement found from 14688 to 14694\n",
      "Seq is: TACAAC\n",
      "7 TACAAC motifs detected in file\n",
      "\n",
      "Complement found from 8178 to 8184\n",
      "Complement found from 8535 to 8541\n",
      "Complement found from 12170 to 12176\n",
      "Complement found from 15234 to 15240\n",
      "Seq is: AATCTG\n",
      "4 AATCTG motifs detected in file\n",
      "\n",
      "Complement found from 538 to 544\n",
      "Complement found from 1080 to 1086\n",
      "Complement found from 1372 to 1378\n",
      "Complement found from 2485 to 2491\n",
      "Complement found from 4134 to 4140\n",
      "Complement found from 4433 to 4439\n",
      "Complement found from 7229 to 7235\n",
      "Complement found from 8278 to 8284\n",
      "Complement found from 8403 to 8409\n",
      "Complement found from 9529 to 9535\n",
      "Complement found from 10275 to 10281\n",
      "Complement found from 10976 to 10982\n",
      "Complement found from 11533 to 11539\n",
      "Complement found from 11864 to 11870\n",
      "Complement found from 12429 to 12435\n",
      "Seq is: TACCCC\n",
      "15 TACCCC motifs detected in file\n",
      "\n",
      "Complement found from 4462 to 4468\n",
      "Complement found from 6997 to 7003\n",
      "Complement found from 7011 to 7017\n",
      "Complement found from 15810 to 15816\n",
      "Seq is: CGTACT\n",
      "4 CGTACT motifs detected in file\n",
      "\n",
      "Complement found from 8001 to 8007\n",
      "Complement found from 13282 to 13288\n",
      "Complement found from 14939 to 14945\n",
      "Seq is: CAATCG\n",
      "3 CAATCG motifs detected in file\n",
      "\n",
      "Seq is: TCAAGA\n",
      "0 TCAAGA motifs detected in file\n",
      "\n",
      "Complement found from 4517 to 4523\n",
      "Complement found from 5450 to 5456\n",
      "Complement found from 6878 to 6884\n",
      "Complement found from 7717 to 7723\n",
      "Complement found from 11255 to 11261\n",
      "Complement found from 11497 to 11503\n",
      "Complement found from 12908 to 12914\n",
      "Complement found from 14398 to 14404\n",
      "Complement found from 14644 to 14650\n",
      "Complement found from 15325 to 15331\n",
      "Seq is: ACACTC\n",
      "10 ACACTC motifs detected in file\n",
      "\n",
      "Complement found from 1792 to 1798\n",
      "Seq is: GAAAGA\n",
      "1 GAAAGA motifs detected in file\n",
      "\n",
      "Complement found from 5261 to 5267\n",
      "Complement found from 5880 to 5886\n",
      "Complement found from 11656 to 11662\n",
      "Complement found from 12837 to 12843\n",
      "Complement found from 16318 to 16324\n",
      "Seq is: GCCATT\n",
      "5 GCCATT motifs detected in file\n",
      "\n",
      "Complement found from 4588 to 4594\n",
      "Complement found from 7961 to 7967\n",
      "Complement found from 8423 to 8429\n",
      "Complement found from 12745 to 12751\n",
      "Complement found from 14150 to 14156\n",
      "Complement found from 14891 to 14897\n",
      "Seq is: TATTCC\n",
      "6 TATTCC motifs detected in file\n",
      "\n",
      "Complement found from 509 to 515\n",
      "Complement found from 4240 to 4246\n",
      "Complement found from 5136 to 5142\n",
      "Seq is: CCAGCA\n",
      "3 CCAGCA motifs detected in file\n",
      "\n",
      "Complement found from 6653 to 6659\n",
      "Seq is: TTCGGA\n",
      "1 TTCGGA motifs detected in file\n",
      "\n",
      "Complement found from 1708 to 1714\n",
      "Complement found from 9195 to 9201\n",
      "Complement found from 10037 to 10043\n",
      "Complement found from 12175 to 12181\n",
      "Complement found from 14475 to 14481\n",
      "Complement found from 15764 to 15770\n",
      "Seq is: GACAAC\n",
      "6 GACAAC motifs detected in file\n",
      "\n",
      "Complement found from 6283 to 6289\n",
      "Complement found from 9653 to 9659\n",
      "Seq is: AGTCTA\n",
      "2 AGTCTA motifs detected in file\n",
      "\n",
      "Complement found from 941 to 947\n",
      "Seq is: AGTGTT\n",
      "1 AGTGTT motifs detected in file\n",
      "\n",
      "Complement found from 1016 to 1022\n",
      "Complement found from 10729 to 10735\n",
      "Complement found from 12581 to 12587\n",
      "Seq is: AGACTA\n",
      "3 AGACTA motifs detected in file\n",
      "\n",
      "Complement found from 779 to 785\n",
      "Complement found from 1122 to 1128\n",
      "Complement found from 5427 to 5433\n",
      "Complement found from 5595 to 5601\n",
      "Complement found from 7727 to 7733\n",
      "Complement found from 8686 to 8692\n",
      "Complement found from 10420 to 10426\n",
      "Complement found from 11519 to 11525\n",
      "Complement found from 12153 to 12159\n",
      "Complement found from 13424 to 13430\n",
      "Complement found from 13971 to 13977\n",
      "Complement found from 14760 to 14766\n",
      "Complement found from 16178 to 16184\n",
      "Seq is: CAAAAC\n",
      "13 CAAAAC motifs detected in file\n",
      "\n",
      "Complement found from 879 to 885\n",
      "Complement found from 6388 to 6394\n",
      "Complement found from 9702 to 9708\n",
      "Complement found from 14029 to 14035\n",
      "Seq is: CAATTT\n",
      "4 CAATTT motifs detected in file\n",
      "\n",
      "Complement found from 1775 to 1781\n",
      "Complement found from 12652 to 12658\n",
      "Seq is: GATATA\n",
      "2 GATATA motifs detected in file\n",
      "\n",
      "Complement found from 613 to 619\n",
      "Complement found from 2296 to 2302\n",
      "Complement found from 4442 to 4448\n",
      "Complement found from 14194 to 14200\n",
      "Seq is: AATGTT\n",
      "4 AATGTT motifs detected in file\n",
      "\n",
      "Complement found from 3779 to 3785\n",
      "Complement found from 5327 to 5333\n",
      "Complement found from 6005 to 6011\n",
      "Complement found from 6346 to 6352\n",
      "Complement found from 7987 to 7993\n",
      "Complement found from 8413 to 8419\n",
      "Complement found from 10268 to 10274\n",
      "Complement found from 11070 to 11076\n",
      "Complement found from 14285 to 14291\n",
      "Complement found from 14851 to 14857\n",
      "Complement found from 15456 to 15462\n",
      "Seq is: CTCCTT\n",
      "11 CTCCTT motifs detected in file\n",
      "\n",
      "Seq is: TTGGGC\n",
      "0 TTGGGC motifs detected in file\n",
      "\n",
      "Complement found from 3763 to 3769\n",
      "Complement found from 4201 to 4207\n",
      "Complement found from 4229 to 4235\n",
      "Complement found from 9014 to 9020\n",
      "Complement found from 11729 to 11735\n",
      "Complement found from 12117 to 12123\n",
      "Complement found from 13736 to 13742\n",
      "Complement found from 14631 to 14637\n",
      "Complement found from 16098 to 16104\n",
      "Complement found from 16338 to 16344\n",
      "Seq is: CATTAC\n",
      "10 CATTAC motifs detected in file\n",
      "\n",
      "Complement found from 16208 to 16214\n",
      "Seq is: TACAGC\n",
      "1 TACAGC motifs detected in file\n",
      "\n",
      "Complement found from 435 to 441\n",
      "Complement found from 1625 to 1631\n",
      "Complement found from 2425 to 2431\n",
      "Complement found from 2833 to 2839\n",
      "Complement found from 3414 to 3420\n",
      "Complement found from 3579 to 3585\n",
      "Complement found from 4482 to 4488\n",
      "Complement found from 8369 to 8375\n",
      "Complement found from 8435 to 8441\n",
      "Complement found from 8659 to 8665\n",
      "Complement found from 8812 to 8818\n",
      "Complement found from 9556 to 9562\n",
      "Complement found from 10776 to 10782\n",
      "Complement found from 10917 to 10923\n",
      "Seq is: CCCAAC\n",
      "14 CCCAAC motifs detected in file\n",
      "\n",
      "Complement found from 5216 to 5222\n",
      "Complement found from 7822 to 7828\n",
      "Complement found from 10553 to 10559\n",
      "Complement found from 11059 to 11065\n",
      "Complement found from 11405 to 11411\n",
      "Complement found from 11551 to 11557\n",
      "Complement found from 12375 to 12381\n",
      "Complement found from 12564 to 12570\n",
      "Complement found from 15596 to 15602\n",
      "Complement found from 15853 to 15859\n",
      "Seq is: TCCCTA\n",
      "10 TCCCTA motifs detected in file\n",
      "\n",
      "Complement found from 2326 to 2332\n",
      "Complement found from 3114 to 3120\n",
      "Complement found from 5442 to 5448\n",
      "Complement found from 10550 to 10556\n",
      "Complement found from 10927 to 10933\n",
      "Complement found from 12092 to 12098\n",
      "Complement found from 12982 to 12988\n",
      "Complement found from 13987 to 13993\n",
      "Complement found from 14260 to 14266\n",
      "Complement found from 14870 to 14876\n",
      "Complement found from 15098 to 15104\n",
      "Complement found from 15140 to 15146\n",
      "Complement found from 15662 to 15668\n",
      "Complement found from 16405 to 16411\n",
      "Seq is: TCCTCC\n",
      "14 TCCTCC motifs detected in file\n",
      "\n",
      "Complement found from 2482 to 2488\n",
      "Complement found from 2657 to 2663\n",
      "Complement found from 6213 to 6219\n",
      "Complement found from 8908 to 8914\n",
      "Complement found from 9986 to 9992\n",
      "Complement found from 13108 to 13114\n",
      "Complement found from 13961 to 13967\n",
      "Seq is: TCTTAC\n",
      "7 TCTTAC motifs detected in file\n",
      "\n",
      "Complement found from 1530 to 1536\n",
      "Complement found from 1836 to 1842\n",
      "Complement found from 3428 to 3434\n",
      "Complement found from 3483 to 3489\n",
      "Complement found from 6152 to 6158\n",
      "Complement found from 7094 to 7100\n",
      "Complement found from 7625 to 7631\n",
      "Complement found from 8227 to 8233\n",
      "Complement found from 9254 to 9260\n",
      "Complement found from 9525 to 9531\n",
      "Complement found from 9581 to 9587\n",
      "Complement found from 10277 to 10283\n",
      "Complement found from 10936 to 10942\n",
      "Complement found from 11233 to 11239\n",
      "Complement found from 13128 to 13134\n",
      "Complement found from 13946 to 13952\n",
      "Complement found from 13980 to 13986\n",
      "Complement found from 14489 to 14495\n",
      "Complement found from 14770 to 14776\n",
      "Complement found from 15367 to 15373\n",
      "Seq is: CCCCTA\n",
      "20 CCCCTA motifs detected in file\n",
      "\n",
      "Complement found from 266 to 272\n",
      "Complement found from 3867 to 3873\n",
      "Complement found from 4622 to 4628\n",
      "Complement found from 13375 to 13381\n",
      "Complement found from 16171 to 16177\n",
      "Seq is: TCCACA\n",
      "5 TCCACA motifs detected in file\n",
      "\n",
      "Complement found from 11447 to 11453\n",
      "Complement found from 16139 to 16145\n",
      "Seq is: TACTTG\n",
      "2 TACTTG motifs detected in file\n",
      "\n",
      "Complement found from 3325 to 3331\n",
      "Complement found from 3826 to 3832\n",
      "Complement found from 5009 to 5015\n",
      "Complement found from 5360 to 5366\n",
      "Complement found from 5379 to 5385\n",
      "Complement found from 5472 to 5478\n",
      "Complement found from 6231 to 6237\n",
      "Complement found from 6308 to 6314\n",
      "Complement found from 6683 to 6689\n",
      "Complement found from 8012 to 8018\n",
      "Complement found from 8411 to 8417\n",
      "Complement found from 10752 to 10758\n",
      "Complement found from 11810 to 11816\n",
      "Complement found from 11974 to 11980\n",
      "Complement found from 13072 to 13078\n",
      "Complement found from 13503 to 13509\n",
      "Complement found from 13984 to 13990\n",
      "Complement found from 14131 to 14137\n",
      "Complement found from 14440 to 14446\n",
      "Seq is: TACTCC\n",
      "19 TACTCC motifs detected in file\n",
      "\n",
      "Complement found from 1008 to 1014\n",
      "Complement found from 1616 to 1622\n",
      "Complement found from 3154 to 3160\n",
      "Complement found from 3989 to 3995\n",
      "Complement found from 5278 to 5284\n",
      "Complement found from 5678 to 5684\n",
      "Complement found from 5936 to 5942\n",
      "Complement found from 8454 to 8460\n",
      "Complement found from 13216 to 13222\n",
      "Complement found from 13544 to 13550\n",
      "Complement found from 14622 to 14628\n",
      "Seq is: CACAAA\n",
      "11 CACAAA motifs detected in file\n",
      "\n",
      "Complement found from 1937 to 1943\n",
      "Seq is: AGAGCA\n",
      "1 AGAGCA motifs detected in file\n",
      "\n",
      "Complement found from 329 to 335\n",
      "Complement found from 1634 to 1640\n",
      "Complement found from 5684 to 5690\n",
      "Complement found from 9091 to 9097\n",
      "Complement found from 11005 to 11011\n",
      "Complement found from 11132 to 11138\n",
      "Complement found from 11395 to 11401\n",
      "Seq is: CACTTA\n",
      "7 CACTTA motifs detected in file\n",
      "\n",
      "Complement found from 458 to 464\n",
      "Complement found from 464 to 470\n",
      "Complement found from 6310 to 6316\n",
      "Complement found from 6497 to 6503\n",
      "Complement found from 6664 to 6670\n",
      "Complement found from 7816 to 7822\n",
      "Complement found from 11812 to 11818\n",
      "Complement found from 15381 to 15387\n",
      "Seq is: CTCCCA\n",
      "8 CTCCCA motifs detected in file\n",
      "\n",
      "Complement found from 5524 to 5530\n",
      "Complement found from 8376 to 8382\n",
      "Complement found from 15869 to 15875\n",
      "Complement found from 16136 to 16142\n",
      "Seq is: AAATAC\n",
      "4 AAATAC motifs detected in file\n",
      "\n",
      "Complement found from 239 to 245\n",
      "Complement found from 2967 to 2973\n",
      "Complement found from 3719 to 3725\n",
      "Complement found from 4697 to 4703\n",
      "Complement found from 5285 to 5291\n",
      "Complement found from 6400 to 6406\n",
      "Complement found from 8662 to 8668\n",
      "Complement found from 10779 to 10785\n",
      "Complement found from 12488 to 12494\n",
      "Complement found from 13386 to 13392\n",
      "Complement found from 13775 to 13781\n",
      "Complement found from 14027 to 14033\n",
      "Complement found from 14191 to 14197\n",
      "Complement found from 14571 to 14577\n",
      "Complement found from 15825 to 15831\n",
      "Seq is: AACAAT\n",
      "15 AACAAT motifs detected in file\n",
      "\n",
      "Complement found from 4162 to 4168\n",
      "Complement found from 6351 to 6357\n",
      "Complement found from 7109 to 7115\n",
      "Complement found from 7241 to 7247\n",
      "Complement found from 7912 to 7918\n",
      "Complement found from 8801 to 8807\n",
      "Complement found from 8927 to 8933\n",
      "Complement found from 11212 to 11218\n",
      "Complement found from 12074 to 12080\n",
      "Complement found from 14181 to 14187\n",
      "Complement found from 15907 to 15913\n",
      "Seq is: TACACC\n",
      "11 TACACC motifs detected in file\n",
      "\n",
      "Complement found from 1308 to 1314\n",
      "Complement found from 1687 to 1693\n",
      "Complement found from 2215 to 2221\n",
      "Complement found from 2768 to 2774\n",
      "Complement found from 3202 to 3208\n",
      "Complement found from 3208 to 3214\n",
      "Complement found from 5021 to 5027\n",
      "Complement found from 5431 to 5437\n",
      "Complement found from 5675 to 5681\n",
      "Complement found from 9209 to 9215\n",
      "Complement found from 10619 to 10625\n",
      "Complement found from 10843 to 10849\n",
      "Complement found from 12016 to 12022\n",
      "Complement found from 12925 to 12931\n",
      "Complement found from 13326 to 13332\n",
      "Complement found from 14356 to 14362\n",
      "Complement found from 14640 to 14646\n",
      "Complement found from 16264 to 16270\n",
      "Complement found from 16288 to 16294\n",
      "Seq is: ACCCAC\n",
      "19 ACCCAC motifs detected in file\n",
      "\n",
      "Complement found from 4932 to 4938\n",
      "Complement found from 5873 to 5879\n",
      "Complement found from 8792 to 8798\n",
      "Complement found from 11294 to 11300\n",
      "Complement found from 11930 to 11936\n",
      "Complement found from 12010 to 12016\n",
      "Complement found from 14848 to 14854\n",
      "Complement found from 14951 to 14957\n",
      "Seq is: TCACTC\n",
      "8 TCACTC motifs detected in file\n",
      "\n",
      "Complement found from 67 to 73\n",
      "Complement found from 8152 to 8158\n",
      "Complement found from 16470 to 16476\n",
      "Seq is: GGGGTA\n",
      "3 GGGGTA motifs detected in file\n",
      "\n",
      "Complement found from 1007 to 1013\n",
      "Complement found from 1048 to 1054\n",
      "Complement found from 1615 to 1621\n",
      "Complement found from 3807 to 3813\n",
      "Complement found from 3988 to 3994\n",
      "Complement found from 4069 to 4075\n",
      "Complement found from 8453 to 8459\n",
      "Complement found from 8705 to 8711\n",
      "Complement found from 10835 to 10841\n",
      "Complement found from 11354 to 11360\n",
      "Complement found from 11998 to 12004\n",
      "Complement found from 13215 to 13221\n",
      "Complement found from 13543 to 13549\n",
      "Complement found from 15416 to 15422\n",
      "Complement found from 15578 to 15584\n",
      "Seq is: ACACAA\n",
      "15 ACACAA motifs detected in file\n",
      "\n",
      "Complement found from 3542 to 3548\n",
      "Complement found from 5305 to 5311\n",
      "Complement found from 5317 to 5323\n",
      "Complement found from 6770 to 6776\n",
      "Complement found from 8394 to 8400\n",
      "Complement found from 9647 to 9653\n",
      "Complement found from 9763 to 9769\n",
      "Complement found from 13454 to 13460\n",
      "Complement found from 15977 to 15983\n",
      "Complement found from 16111 to 16117\n",
      "Complement found from 16400 to 16406\n",
      "Seq is: CACCAT\n",
      "11 CACCAT motifs detected in file\n",
      "\n",
      "Complement found from 3198 to 3204\n",
      "Complement found from 4450 to 4456\n",
      "Complement found from 5493 to 5499\n",
      "Complement found from 8734 to 8740\n",
      "Complement found from 10491 to 10497\n",
      "Complement found from 15510 to 15516\n",
      "Seq is: TTATAC\n",
      "6 TTATAC motifs detected in file\n",
      "\n",
      "Complement found from 3864 to 3870\n",
      "Complement found from 4236 to 4242\n",
      "Complement found from 5483 to 5489\n",
      "Complement found from 6662 to 6668\n",
      "Complement found from 10709 to 10715\n",
      "Complement found from 11056 to 11062\n",
      "Complement found from 11068 to 11074\n",
      "Complement found from 14046 to 14052\n",
      "Complement found from 14824 to 14830\n",
      "Complement found from 15850 to 15856\n",
      "Seq is: ATCTCC\n",
      "10 ATCTCC motifs detected in file\n",
      "\n",
      "Complement found from 2440 to 2446\n",
      "Complement found from 3725 to 3731\n",
      "Complement found from 4158 to 4164\n",
      "Complement found from 7666 to 7672\n",
      "Complement found from 9332 to 9338\n",
      "Complement found from 10544 to 10550\n",
      "Complement found from 10604 to 10610\n",
      "Complement found from 11701 to 11707\n",
      "Complement found from 12425 to 12431\n",
      "Complement found from 13269 to 13275\n",
      "Seq is: CTCATA\n",
      "10 CTCATA motifs detected in file\n",
      "\n",
      "Complement found from 3005 to 3011\n",
      "Complement found from 8015 to 8021\n",
      "Complement found from 14263 to 14269\n",
      "Seq is: TCCCGA\n",
      "3 TCCCGA motifs detected in file\n",
      "\n",
      "Complement found from 606 to 612\n",
      "Complement found from 2356 to 2362\n",
      "Complement found from 4541 to 4547\n",
      "Complement found from 5898 to 5904\n",
      "Complement found from 7085 to 7091\n",
      "Complement found from 10796 to 10802\n",
      "Complement found from 12534 to 12540\n",
      "Seq is: CACTGA\n",
      "7 CACTGA motifs detected in file\n",
      "\n",
      "Complement found from 4159 to 4165\n",
      "Complement found from 9333 to 9339\n",
      "Complement found from 12071 to 12077\n",
      "Complement found from 12426 to 12432\n",
      "Complement found from 13539 to 13545\n",
      "Complement found from 14233 to 14239\n",
      "Complement found from 14344 to 14350\n",
      "Seq is: TCATAC\n",
      "7 TCATAC motifs detected in file\n",
      "\n",
      "Complement found from 976 to 982\n",
      "Complement found from 1160 to 1166\n",
      "Complement found from 3675 to 3681\n",
      "Complement found from 4156 to 4162\n",
      "Complement found from 6980 to 6986\n",
      "Complement found from 10134 to 10140\n",
      "Complement found from 11749 to 11755\n",
      "Complement found from 12228 to 12234\n",
      "Complement found from 12423 to 12429\n",
      "Complement found from 12660 to 12666\n",
      "Complement found from 12915 to 12921\n",
      "Complement found from 13795 to 13801\n",
      "Seq is: AACTCA\n",
      "12 AACTCA motifs detected in file\n",
      "\n",
      "Complement found from 227 to 233\n",
      "Complement found from 1514 to 1520\n",
      "Complement found from 3000 to 3006\n",
      "Seq is: GGACAT\n",
      "3 GGACAT motifs detected in file\n",
      "\n",
      "Complement found from 145 to 151\n",
      "Complement found from 5298 to 5304\n",
      "Complement found from 7796 to 7802\n",
      "Complement found from 9861 to 9867\n",
      "Complement found from 10325 to 10331\n",
      "Complement found from 10344 to 10350\n",
      "Complement found from 10865 to 10871\n",
      "Complement found from 11663 to 11669\n",
      "Complement found from 12601 to 12607\n",
      "Complement found from 12880 to 12886\n",
      "Complement found from 13114 to 13120\n",
      "Complement found from 13372 to 13378\n",
      "Complement found from 14119 to 14125\n",
      "Complement found from 14978 to 14984\n",
      "Complement found from 15641 to 15647\n",
      "Seq is: TCATCC\n",
      "15 TCATCC motifs detected in file\n",
      "\n",
      "Complement found from 3539 to 3545\n",
      "Complement found from 4858 to 4864\n",
      "Complement found from 10509 to 10515\n",
      "Complement found from 11298 to 11304\n",
      "Complement found from 12643 to 12649\n",
      "Complement found from 13436 to 13442\n",
      "Complement found from 13619 to 13625\n",
      "Complement found from 15479 to 15485\n",
      "Seq is: TCTCAC\n",
      "8 TCTCAC motifs detected in file\n",
      "\n",
      "Complement found from 6412 to 6418\n",
      "Complement found from 8822 to 8828\n",
      "Complement found from 10000 to 10006\n",
      "Complement found from 12656 to 12662\n",
      "Complement found from 15897 to 15903\n",
      "Seq is: TATAAA\n",
      "5 TATAAA motifs detected in file\n",
      "\n",
      "Complement found from 3924 to 3930\n",
      "Complement found from 7693 to 7699\n",
      "Complement found from 7801 to 7807\n",
      "Complement found from 10658 to 10664\n",
      "Complement found from 10700 to 10706\n",
      "Complement found from 11959 to 11965\n",
      "Seq is: CTAGTC\n",
      "6 CTAGTC motifs detected in file\n",
      "\n",
      "Complement found from 1241 to 1247\n",
      "Complement found from 12132 to 12138\n",
      "Seq is: CTCTTG\n",
      "2 CTCTTG motifs detected in file\n",
      "\n",
      "Complement found from 295 to 301\n",
      "Complement found from 2186 to 2192\n",
      "Complement found from 3473 to 3479\n",
      "Complement found from 4085 to 4091\n",
      "Complement found from 7888 to 7894\n",
      "Complement found from 8477 to 8483\n",
      "Complement found from 8803 to 8809\n",
      "Complement found from 9212 to 9218\n",
      "Complement found from 14040 to 14046\n",
      "Complement found from 14183 to 14189\n",
      "Complement found from 14249 to 14255\n",
      "Complement found from 14364 to 14370\n",
      "Complement found from 14403 to 14409\n",
      "Complement found from 14741 to 14747\n",
      "Seq is: CACCAA\n",
      "14 CACCAA motifs detected in file\n",
      "\n",
      "Complement found from 3691 to 3697\n",
      "Seq is: TGATCG\n",
      "1 TGATCG motifs detected in file\n",
      "\n",
      "Complement found from 2653 to 2659\n",
      "Complement found from 4217 to 4223\n",
      "Complement found from 6364 to 6370\n",
      "Complement found from 9964 to 9970\n",
      "Seq is: TGTCTC\n",
      "4 TGTCTC motifs detected in file\n",
      "\n",
      "Seq is: TCAAGA\n",
      "0 TCAAGA motifs detected in file\n",
      "\n",
      "Complement found from 11437 to 11443\n",
      "Complement found from 16515 to 16521\n",
      "Seq is: GGGTCA\n",
      "2 GGGTCA motifs detected in file\n",
      "\n",
      "Complement found from 2897 to 2903\n",
      "Complement found from 5864 to 5870\n",
      "Complement found from 10020 to 10026\n",
      "Complement found from 10755 to 10761\n",
      "Seq is: TCCAAT\n",
      "4 TCCAAT motifs detected in file\n",
      "\n",
      "Complement found from 11 to 17\n",
      "Complement found from 2276 to 2282\n",
      "Complement found from 3800 to 3806\n",
      "Complement found from 4383 to 4389\n",
      "Complement found from 7644 to 7650\n",
      "Complement found from 11024 to 11030\n",
      "Complement found from 11928 to 11934\n",
      "Complement found from 13181 to 13187\n",
      "Complement found from 16228 to 16234\n",
      "Seq is: TATCAC\n",
      "9 TATCAC motifs detected in file\n",
      "\n",
      "Complement found from 433 to 439\n",
      "Complement found from 568 to 574\n",
      "Complement found from 801 to 807\n",
      "Complement found from 4878 to 4884\n",
      "Complement found from 5894 to 5900\n",
      "Complement found from 7397 to 7403\n",
      "Complement found from 7466 to 7472\n",
      "Complement found from 7954 to 7960\n",
      "Complement found from 8027 to 8033\n",
      "Complement found from 8405 to 8411\n",
      "Complement found from 8557 to 8563\n",
      "Complement found from 9532 to 9538\n",
      "Complement found from 9554 to 9560\n",
      "Complement found from 11426 to 11432\n",
      "Complement found from 11867 to 11873\n",
      "Complement found from 12083 to 12089\n",
      "Complement found from 12236 to 12242\n",
      "Complement found from 12385 to 12391\n",
      "Complement found from 12431 to 12437\n",
      "Complement found from 14424 to 14430\n",
      "Complement found from 14530 to 14536\n",
      "Complement found from 14587 to 14593\n",
      "Seq is: CCCCCA\n",
      "22 CCCCCA motifs detected in file\n",
      "\n",
      "Complement found from 4143 to 4149\n",
      "Seq is: TTCCGC\n",
      "1 TTCCGC motifs detected in file\n",
      "\n",
      "Complement found from 2897 to 2903\n",
      "Complement found from 5864 to 5870\n",
      "Complement found from 10020 to 10026\n",
      "Complement found from 10755 to 10761\n",
      "Seq is: TCCAAT\n",
      "4 TCCAAT motifs detected in file\n",
      "\n",
      "Complement found from 9327 to 9333\n",
      "Complement found from 16448 to 16454\n",
      "Seq is: CGCTCC\n",
      "2 CGCTCC motifs detected in file\n",
      "\n",
      "Complement found from 3375 to 3381\n",
      "Complement found from 8528 to 8534\n",
      "Seq is: GAACGA\n",
      "2 GAACGA motifs detected in file\n",
      "\n",
      "Complement found from 1266 to 1272\n",
      "Complement found from 3469 to 3475\n",
      "Complement found from 3973 to 3979\n",
      "Complement found from 4692 to 4698\n",
      "Complement found from 6102 to 6108\n",
      "Complement found from 7934 to 7940\n",
      "Complement found from 9100 to 9106\n",
      "Complement found from 11623 to 11629\n",
      "Complement found from 12685 to 12691\n",
      "Complement found from 13336 to 13342\n",
      "Seq is: TCTTCA\n",
      "10 TCTTCA motifs detected in file\n",
      "\n",
      "Seq is: GTAGTT\n",
      "0 GTAGTT motifs detected in file\n",
      "\n",
      "Complement found from 10432 to 10438\n",
      "Seq is: TTTCGA\n",
      "1 TTTCGA motifs detected in file\n",
      "\n",
      "Complement found from 2598 to 2604\n",
      "Complement found from 4758 to 4764\n",
      "Complement found from 5708 to 5714\n",
      "Complement found from 6468 to 6474\n",
      "Complement found from 6831 to 6837\n",
      "Complement found from 7670 to 7676\n",
      "Complement found from 8650 to 8656\n",
      "Complement found from 8671 to 8677\n",
      "Complement found from 8749 to 8755\n",
      "Complement found from 10080 to 10086\n",
      "Complement found from 10338 to 10344\n",
      "Complement found from 10453 to 10459\n",
      "Complement found from 11102 to 11108\n",
      "Complement found from 12677 to 12683\n",
      "Complement found from 14137 to 14143\n",
      "Complement found from 14217 to 14223\n",
      "Complement found from 14230 to 14236\n",
      "Seq is: TAATCA\n",
      "17 TAATCA motifs detected in file\n",
      "\n",
      "Complement found from 2412 to 2418\n",
      "Complement found from 4012 to 4018\n",
      "Complement found from 4250 to 4256\n",
      "Complement found from 4907 to 4913\n",
      "Complement found from 5549 to 5555\n",
      "Complement found from 7664 to 7670\n",
      "Complement found from 8473 to 8479\n",
      "Complement found from 10473 to 10479\n",
      "Complement found from 10611 to 10617\n",
      "Complement found from 10979 to 10985\n",
      "Complement found from 12040 to 12046\n",
      "Complement found from 12062 to 12068\n",
      "Complement found from 12101 to 12107\n",
      "Complement found from 13035 to 13041\n",
      "Complement found from 13450 to 13456\n",
      "Complement found from 15266 to 15272\n",
      "Complement found from 16220 to 16226\n",
      "Complement found from 16259 to 16265\n",
      "Complement found from 16377 to 16383\n",
      "Seq is: CCCTCA\n",
      "19 CCCTCA motifs detected in file\n",
      "\n",
      "Complement found from 3185 to 3191\n",
      "Complement found from 4884 to 4890\n",
      "Complement found from 4938 to 4944\n",
      "Complement found from 10704 to 10710\n",
      "Complement found from 11505 to 11511\n",
      "Complement found from 11791 to 11797\n",
      "Complement found from 12260 to 12266\n",
      "Complement found from 14165 to 14171\n",
      "Seq is: TCTCAA\n",
      "8 TCTCAA motifs detected in file\n",
      "\n",
      "Complement found from 1506 to 1512\n",
      "Complement found from 2860 to 2866\n",
      "Complement found from 3150 to 3156\n",
      "Complement found from 9820 to 9826\n",
      "Complement found from 11799 to 11805\n",
      "Complement found from 13251 to 13257\n",
      "Complement found from 13440 to 13446\n",
      "Complement found from 15290 to 15296\n",
      "Complement found from 15818 to 15824\n",
      "Complement found from 16509 to 16515\n",
      "Seq is: ACTTCA\n",
      "10 ACTTCA motifs detected in file\n",
      "\n",
      "Complement found from 2469 to 2475\n",
      "Complement found from 6303 to 6309\n",
      "Seq is: GGAACT\n",
      "2 GGAACT motifs detected in file\n",
      "\n",
      "Complement found from 38 to 44\n",
      "Complement found from 3789 to 3795\n",
      "Complement found from 4053 to 4059\n",
      "Complement found from 4706 to 4712\n",
      "Complement found from 4903 to 4909\n",
      "Complement found from 5213 to 5219\n",
      "Complement found from 6225 to 6231\n",
      "Complement found from 6495 to 6501\n",
      "Complement found from 11933 to 11939\n",
      "Complement found from 12561 to 12567\n",
      "Complement found from 14283 to 14289\n",
      "Complement found from 15454 to 15460\n",
      "Complement found from 16441 to 16447\n",
      "Seq is: CTCTCC\n",
      "13 CTCTCC motifs detected in file\n",
      "\n",
      "Complement found from 6111 to 6117\n",
      "Complement found from 7737 to 7743\n",
      "Complement found from 10575 to 10581\n",
      "Complement found from 10955 to 10961\n",
      "Complement found from 11486 to 11492\n",
      "Complement found from 15839 to 15845\n",
      "Complement found from 15904 to 15910\n",
      "Seq is: TAATAC\n",
      "7 TAATAC motifs detected in file\n",
      "\n",
      "Seq is: TACGCT\n",
      "0 TACGCT motifs detected in file\n",
      "\n",
      "Complement found from 2120 to 2126\n",
      "Complement found from 12532 to 12538\n",
      "Seq is: GACACT\n",
      "2 GACACT motifs detected in file\n",
      "\n",
      "Complement found from 762 to 768\n",
      "Complement found from 4651 to 4657\n",
      "Seq is: CACGCA\n",
      "2 CACGCA motifs detected in file\n",
      "\n",
      "Complement found from 699 to 705\n",
      "Complement found from 753 to 759\n",
      "Complement found from 2517 to 2523\n",
      "Complement found from 3668 to 3674\n",
      "Complement found from 10861 to 10867\n",
      "Complement found from 13926 to 13932\n",
      "Complement found from 15804 to 15810\n",
      "Seq is: AGCATC\n",
      "7 AGCATC motifs detected in file\n",
      "\n",
      "Complement found from 975 to 981\n",
      "Complement found from 995 to 1001\n",
      "Complement found from 1159 to 1165\n",
      "Complement found from 3465 to 3471\n",
      "Complement found from 3674 to 3680\n",
      "Complement found from 5131 to 5137\n",
      "Complement found from 6979 to 6985\n",
      "Complement found from 11035 to 11041\n",
      "Complement found from 11319 to 11325\n",
      "Complement found from 11748 to 11754\n",
      "Complement found from 11804 to 11810\n",
      "Complement found from 12422 to 12428\n",
      "Complement found from 12659 to 12665\n",
      "Complement found from 13156 to 13162\n",
      "Complement found from 13794 to 13800\n",
      "Seq is: AAACTC\n",
      "15 AAACTC motifs detected in file\n",
      "\n",
      "Complement found from 838 to 844\n",
      "Complement found from 10422 to 10428\n",
      "Seq is: AAACGA\n",
      "2 AAACGA motifs detected in file\n",
      "\n",
      "Complement found from 6461 to 6467\n",
      "Complement found from 15592 to 15598\n",
      "Seq is: TCCGTC\n",
      "2 TCCGTC motifs detected in file\n",
      "\n",
      "Complement found from 3799 to 3805\n",
      "Complement found from 6405 to 6411\n",
      "Complement found from 7643 to 7649\n",
      "Complement found from 9094 to 9100\n",
      "Complement found from 12469 to 12475\n",
      "Seq is: TTATCA\n",
      "5 TTATCA motifs detected in file\n",
      "\n",
      "Complement found from 3644 to 3650\n",
      "Complement found from 3825 to 3831\n",
      "Complement found from 7219 to 7225\n",
      "Complement found from 9614 to 9620\n",
      "Complement found from 9988 to 9994\n",
      "Complement found from 13110 to 13116\n",
      "Complement found from 13567 to 13573\n",
      "Seq is: TTACTC\n",
      "7 TTACTC motifs detected in file\n",
      "\n",
      "Complement found from 1362 to 1368\n",
      "Complement found from 3515 to 3521\n",
      "Complement found from 7944 to 7950\n",
      "Complement found from 10144 to 10150\n",
      "Complement found from 11982 to 11988\n",
      "Seq is: CTACAT\n",
      "5 CTACAT motifs detected in file\n",
      "\n",
      "Complement found from 4424 to 4430\n",
      "Complement found from 15038 to 15044\n",
      "Seq is: TCGGGC\n",
      "2 TCGGGC motifs detected in file\n",
      "\n",
      "Complement found from 3491 to 3497\n",
      "Complement found from 8576 to 8582\n",
      "Seq is: ACCCGC\n",
      "2 ACCCGC motifs detected in file\n",
      "\n",
      "Complement found from 6111 to 6117\n",
      "Complement found from 7737 to 7743\n",
      "Complement found from 10575 to 10581\n",
      "Complement found from 10955 to 10961\n",
      "Complement found from 11486 to 11492\n",
      "Complement found from 15839 to 15845\n",
      "Complement found from 15904 to 15910\n",
      "Seq is: TAATAC\n",
      "7 TAATAC motifs detected in file\n",
      "\n",
      "Complement found from 2482 to 2488\n",
      "Complement found from 2657 to 2663\n",
      "Complement found from 6213 to 6219\n",
      "Complement found from 8908 to 8914\n",
      "Complement found from 9986 to 9992\n",
      "Complement found from 13108 to 13114\n",
      "Complement found from 13961 to 13967\n",
      "Seq is: TCTTAC\n",
      "7 TCTTAC motifs detected in file\n",
      "\n",
      "Complement found from 3455 to 3461\n",
      "Complement found from 4046 to 4052\n",
      "Seq is: TGACGC\n",
      "2 TGACGC motifs detected in file\n",
      "\n",
      "Complement found from 321 to 327\n",
      "Complement found from 7885 to 7891\n",
      "Complement found from 9024 to 9030\n",
      "Complement found from 9410 to 9416\n",
      "Complement found from 15171 to 15177\n",
      "Seq is: GGCCAC\n",
      "5 GGCCAC motifs detected in file\n",
      "\n",
      "Complement found from 4123 to 4129\n",
      "Complement found from 5787 to 5793\n",
      "Complement found from 7333 to 7339\n",
      "Complement found from 7415 to 7421\n",
      "Complement found from 9908 to 9914\n",
      "Complement found from 11122 to 11128\n",
      "Complement found from 13402 to 13408\n",
      "Seq is: TTCGAA\n",
      "7 TTCGAA motifs detected in file\n",
      "\n",
      "Complement found from 8077 to 8083\n",
      "Complement found from 16360 to 16366\n",
      "Seq is: GTCCCC\n",
      "2 GTCCCC motifs detected in file\n",
      "\n",
      "Complement found from 298 to 304\n",
      "Complement found from 348 to 354\n",
      "Complement found from 552 to 558\n",
      "Complement found from 1273 to 1279\n",
      "Complement found from 1684 to 1690\n",
      "Complement found from 1720 to 1726\n",
      "Complement found from 1991 to 1997\n",
      "Complement found from 2765 to 2771\n",
      "Complement found from 2788 to 2794\n",
      "Complement found from 4254 to 4260\n",
      "Complement found from 4982 to 4988\n",
      "Complement found from 5931 to 5937\n",
      "Complement found from 7121 to 7127\n",
      "Complement found from 8125 to 8131\n",
      "Complement found from 8189 to 8195\n",
      "Complement found from 8508 to 8514\n",
      "Complement found from 11668 to 11674\n",
      "Complement found from 14624 to 14630\n",
      "Complement found from 16281 to 16287\n",
      "Seq is: CAAACC\n",
      "19 CAAACC motifs detected in file\n",
      "\n",
      "Complement found from 4496 to 4502\n",
      "Complement found from 4704 to 4710\n",
      "Complement found from 9989 to 9995\n",
      "Complement found from 10600 to 10606\n",
      "Complement found from 11459 to 11465\n",
      "Complement found from 11620 to 11626\n",
      "Complement found from 13568 to 13574\n",
      "Complement found from 14347 to 14353\n",
      "Complement found from 16439 to 16445\n",
      "Seq is: TACTCT\n",
      "9 TACTCT motifs detected in file\n",
      "\n",
      "Complement found from 1997 to 2003\n",
      "Complement found from 3371 to 3377\n",
      "Complement found from 12202 to 12208\n",
      "Seq is: TACCGA\n",
      "3 TACCGA motifs detected in file\n",
      "\n",
      "Complement found from 4715 to 4721\n",
      "Complement found from 8664 to 8670\n",
      "Complement found from 9371 to 9377\n",
      "Complement found from 13388 to 13394\n",
      "Complement found from 14700 to 14706\n",
      "Complement found from 14744 to 14750\n",
      "Complement found from 15229 to 15235\n",
      "Seq is: CAATGA\n",
      "7 CAATGA motifs detected in file\n",
      "\n",
      "Complement found from 13359 to 13365\n",
      "Seq is: TGCTCC\n",
      "1 TGCTCC motifs detected in file\n",
      "\n",
      "Complement found from 9443 to 9449\n",
      "Seq is: CGATAC\n",
      "1 CGATAC motifs detected in file\n",
      "\n",
      "Complement found from 7915 to 7921\n",
      "Complement found from 8644 to 8650\n",
      "Seq is: ACCGAC\n",
      "2 ACCGAC motifs detected in file\n",
      "\n",
      "Complement found from 1653 to 1659\n",
      "Complement found from 1871 to 1877\n",
      "Complement found from 2902 to 2908\n",
      "Complement found from 6675 to 6681\n",
      "Complement found from 7170 to 7176\n",
      "Complement found from 7540 to 7546\n",
      "Complement found from 8305 to 8311\n",
      "Complement found from 10015 to 10021\n",
      "Seq is: TAACTT\n",
      "8 TAACTT motifs detected in file\n",
      "\n",
      "Complement found from 2440 to 2446\n",
      "Complement found from 3725 to 3731\n",
      "Complement found from 4158 to 4164\n",
      "Complement found from 7666 to 7672\n",
      "Complement found from 9332 to 9338\n",
      "Complement found from 10544 to 10550\n",
      "Complement found from 10604 to 10610\n",
      "Complement found from 11701 to 11707\n",
      "Complement found from 12425 to 12431\n",
      "Complement found from 13269 to 13275\n",
      "Seq is: CTCATA\n",
      "10 CTCATA motifs detected in file\n",
      "\n",
      "Complement found from 7217 to 7223\n",
      "Complement found from 12620 to 12626\n",
      "Seq is: CGTTAC\n",
      "2 CGTTAC motifs detected in file\n",
      "\n",
      "Complement found from 6837 to 6843\n",
      "Complement found from 11849 to 11855\n",
      "Complement found from 13576 to 13582\n",
      "Complement found from 14381 to 14387\n",
      "Seq is: TCGCTA\n",
      "4 TCGCTA motifs detected in file\n",
      "\n",
      "Complement found from 2071 to 2077\n",
      "Complement found from 2229 to 2235\n",
      "Complement found from 7134 to 7140\n",
      "Complement found from 7568 to 7574\n",
      "Complement found from 9577 to 9583\n",
      "Complement found from 10154 to 10160\n",
      "Complement found from 12444 to 12450\n",
      "Complement found from 13875 to 13881\n",
      "Complement found from 16348 to 16354\n",
      "Seq is: AAATCC\n",
      "9 AAATCC motifs detected in file\n",
      "\n",
      "Complement found from 1552 to 1558\n",
      "Complement found from 1707 to 1713\n",
      "Complement found from 7442 to 7448\n",
      "Complement found from 14474 to 14480\n",
      "Complement found from 15504 to 15510\n",
      "Seq is: AGACAA\n",
      "5 AGACAA motifs detected in file\n",
      "\n",
      "Complement found from 325 to 331\n",
      "Complement found from 813 to 819\n",
      "Complement found from 2708 to 2714\n",
      "Complement found from 4128 to 4134\n",
      "Complement found from 6473 to 6479\n",
      "Complement found from 7289 to 7295\n",
      "Complement found from 12831 to 12837\n",
      "Complement found from 14036 to 14042\n",
      "Complement found from 14360 to 14366\n",
      "Complement found from 16209 to 16215\n",
      "Seq is: ACAGCA\n",
      "10 ACAGCA motifs detected in file\n",
      "\n",
      "Complement found from 6461 to 6467\n",
      "Complement found from 15592 to 15598\n",
      "Seq is: TCCGTC\n",
      "2 TCCGTC motifs detected in file\n",
      "\n",
      "Complement found from 588 to 594\n",
      "Complement found from 5887 to 5893\n",
      "Complement found from 6216 to 6222\n",
      "Complement found from 8468 to 8474\n",
      "Complement found from 9466 to 9472\n",
      "Complement found from 11041 to 11047\n",
      "Complement found from 11378 to 11384\n",
      "Complement found from 13432 to 13438\n",
      "Complement found from 13580 to 13586\n",
      "Complement found from 14373 to 14379\n",
      "Seq is: TACCTC\n",
      "10 TACCTC motifs detected in file\n",
      "\n",
      "Complement found from 748 to 754\n",
      "Complement found from 2916 to 2922\n",
      "Complement found from 4036 to 4042\n",
      "Complement found from 13392 to 13398\n",
      "Seq is: GAACAA\n",
      "4 GAACAA motifs detected in file\n",
      "\n",
      "Complement found from 19 to 25\n",
      "Complement found from 2743 to 2749\n",
      "Complement found from 3023 to 3029\n",
      "Complement found from 7299 to 7305\n",
      "Complement found from 8447 to 8453\n",
      "Complement found from 10335 to 10341\n",
      "Complement found from 11874 to 11880\n",
      "Complement found from 14315 to 14321\n",
      "Complement found from 14511 to 14517\n",
      "Seq is: TATTAA\n",
      "9 TATTAA motifs detected in file\n",
      "\n",
      "Complement found from 11386 to 11392\n",
      "Complement found from 15055 to 15061\n",
      "Seq is: TACGGA\n",
      "2 TACGGA motifs detected in file\n",
      "\n",
      "Complement found from 8742 to 8748\n",
      "Complement found from 16489 to 16495\n",
      "Seq is: GTATCC\n",
      "2 GTATCC motifs detected in file\n",
      "\n",
      "Complement found from 10 to 16\n",
      "Complement found from 2275 to 2281\n",
      "Complement found from 3756 to 3762\n",
      "Complement found from 4382 to 4388\n",
      "Complement found from 7043 to 7049\n",
      "Complement found from 7146 to 7152\n",
      "Complement found from 7259 to 7265\n",
      "Complement found from 7628 to 7634\n",
      "Complement found from 9226 to 9232\n",
      "Complement found from 11023 to 11029\n",
      "Complement found from 11150 to 11156\n",
      "Complement found from 11314 to 11320\n",
      "Complement found from 13180 to 13186\n",
      "Complement found from 16227 to 16233\n",
      "Seq is: CTATCA\n",
      "14 CTATCA motifs detected in file\n",
      "\n",
      "Complement found from 2606 to 2612\n",
      "Complement found from 10912 to 10918\n",
      "Seq is: TGTTCC\n",
      "2 TGTTCC motifs detected in file\n",
      "\n",
      "Complement found from 453 to 459\n",
      "Complement found from 3165 to 3171\n",
      "Complement found from 4246 to 4252\n",
      "Complement found from 6150 to 6156\n",
      "Complement found from 6183 to 6189\n",
      "Complement found from 7092 to 7098\n",
      "Complement found from 7623 to 7629\n",
      "Complement found from 7952 to 7958\n",
      "Complement found from 8225 to 8231\n",
      "Complement found from 8602 to 8608\n",
      "Complement found from 10914 to 10920\n",
      "Complement found from 11231 to 11237\n",
      "Complement found from 12382 to 12388\n",
      "Complement found from 12480 to 12486\n",
      "Complement found from 13644 to 13650\n",
      "Complement found from 13751 to 13757\n",
      "Complement found from 14152 to 14158\n",
      "Complement found from 14486 to 14492\n",
      "Complement found from 16543 to 16549\n",
      "Seq is: TTCCCC\n",
      "19 TTCCCC motifs detected in file\n",
      "\n",
      "Complement found from 16491 to 16497\n",
      "Seq is: ATCCGA\n",
      "1 ATCCGA motifs detected in file\n",
      "\n",
      "Complement found from 689 to 695\n",
      "Complement found from 1476 to 1482\n",
      "Complement found from 3987 to 3993\n",
      "Complement found from 4068 to 4074\n",
      "Complement found from 8704 to 8710\n",
      "Complement found from 11353 to 11359\n",
      "Complement found from 13214 to 13220\n",
      "Complement found from 13542 to 13548\n",
      "Complement found from 15032 to 15038\n",
      "Complement found from 15415 to 15421\n",
      "Complement found from 15577 to 15583\n",
      "Seq is: TACACA\n",
      "11 TACACA motifs detected in file\n",
      "\n",
      "Complement found from 811 to 817\n",
      "Complement found from 2752 to 2758\n",
      "Complement found from 8096 to 8102\n",
      "Complement found from 11594 to 11600\n",
      "Seq is: AAACAG\n",
      "4 AAACAG motifs detected in file\n",
      "\n",
      "Complement found from 309 to 315\n",
      "Complement found from 3166 to 3172\n",
      "Complement found from 4247 to 4253\n",
      "Complement found from 7953 to 7959\n",
      "Complement found from 8603 to 8609\n",
      "Complement found from 10190 to 10196\n",
      "Complement found from 12082 to 12088\n",
      "Complement found from 12383 to 12389\n",
      "Complement found from 13752 to 13758\n",
      "Complement found from 13761 to 13767\n",
      "Complement found from 13780 to 13786\n",
      "Complement found from 14153 to 14159\n",
      "Complement found from 14487 to 14493\n",
      "Complement found from 14529 to 14535\n",
      "Seq is: TCCCCC\n",
      "14 TCCCCC motifs detected in file\n",
      "\n",
      "Complement found from 3466 to 3472\n",
      "Complement found from 4063 to 4069\n",
      "Complement found from 11036 to 11042\n",
      "Complement found from 11805 to 11811\n",
      "Complement found from 11891 to 11897\n",
      "Complement found from 13157 to 13163\n",
      "Seq is: AACTCT\n",
      "6 AACTCT motifs detected in file\n",
      "\n",
      "Complement found from 2886 to 2892\n",
      "Complement found from 3645 to 3651\n",
      "Complement found from 4745 to 4751\n",
      "Complement found from 5121 to 5127\n",
      "Complement found from 8971 to 8977\n",
      "Complement found from 9031 to 9037\n",
      "Complement found from 11237 to 11243\n",
      "Complement found from 11291 to 11297\n",
      "Complement found from 12697 to 12703\n",
      "Complement found from 13111 to 13117\n",
      "Complement found from 13420 to 13426\n",
      "Complement found from 14908 to 14914\n",
      "Complement found from 15070 to 15076\n",
      "Complement found from 15247 to 15253\n",
      "Complement found from 15872 to 15878\n",
      "Seq is: TACTCA\n",
      "15 TACTCA motifs detected in file\n",
      "\n",
      "Complement found from 462 to 468\n",
      "Complement found from 1690 to 1696\n",
      "Complement found from 6879 to 6885\n",
      "Complement found from 9317 to 9323\n",
      "Complement found from 9512 to 9518\n",
      "Complement found from 9594 to 9600\n",
      "Complement found from 10622 to 10628\n",
      "Complement found from 12909 to 12915\n",
      "Complement found from 14849 to 14855\n",
      "Complement found from 15326 to 15332\n",
      "Seq is: CACTCC\n",
      "10 CACTCC motifs detected in file\n",
      "\n",
      "Complement found from 1815 to 1821\n",
      "Complement found from 1850 to 1856\n",
      "Complement found from 2332 to 2338\n",
      "Complement found from 2595 to 2601\n",
      "Complement found from 2701 to 2707\n",
      "Complement found from 3254 to 3260\n",
      "Complement found from 6189 to 6195\n",
      "Complement found from 11561 to 11567\n",
      "Complement found from 14080 to 14086\n",
      "Complement found from 15686 to 15692\n",
      "Seq is: GCATAA\n",
      "10 GCATAA motifs detected in file\n",
      "\n",
      "Complement found from 6718 to 6724\n",
      "Complement found from 11479 to 11485\n",
      "Seq is: TATGGT\n",
      "2 TATGGT motifs detected in file\n",
      "\n",
      "Complement found from 3693 to 3699\n",
      "Complement found from 7157 to 7163\n",
      "Complement found from 12759 to 12765\n",
      "Complement found from 12864 to 12870\n",
      "Complement found from 13284 to 13290\n",
      "Complement found from 15088 to 15094\n",
      "Seq is: ATCGGC\n",
      "6 ATCGGC motifs detected in file\n",
      "\n",
      "Complement found from 2994 to 3000\n",
      "Complement found from 15058 to 15064\n",
      "Complement found from 15355 to 15361\n",
      "Seq is: GGATCA\n",
      "3 GGATCA motifs detected in file\n",
      "\n",
      "Complement found from 3110 to 3116\n",
      "Complement found from 3382 to 3388\n",
      "Complement found from 4366 to 4372\n",
      "Complement found from 10217 to 10223\n",
      "Complement found from 14536 to 14542\n",
      "Seq is: AAATTC\n",
      "5 AAATTC motifs detected in file\n",
      "\n",
      "Complement found from 3837 to 3843\n",
      "Complement found from 7311 to 7317\n",
      "Complement found from 7653 to 7659\n",
      "Complement found from 8068 to 8074\n",
      "Complement found from 12918 to 12924\n",
      "Seq is: TCATGA\n",
      "5 TCATGA motifs detected in file\n",
      "\n",
      "Seq is: CGCCGT\n",
      "0 CGCCGT motifs detected in file\n",
      "\n",
      "Complement found from 3837 to 3843\n",
      "Complement found from 7311 to 7317\n",
      "Complement found from 7653 to 7659\n",
      "Complement found from 8068 to 8074\n",
      "Complement found from 12918 to 12924\n",
      "Seq is: TCATGA\n",
      "5 TCATGA motifs detected in file\n",
      "\n",
      "Complement found from 2088 to 2094\n",
      "Complement found from 2243 to 2249\n",
      "Seq is: TAACTG\n",
      "2 TAACTG motifs detected in file\n",
      "\n",
      "Seq is: GGTCGC\n",
      "0 GGTCGC motifs detected in file\n",
      "\n",
      "Complement found from 538 to 544\n",
      "Complement found from 1080 to 1086\n",
      "Complement found from 1372 to 1378\n",
      "Complement found from 2485 to 2491\n",
      "Complement found from 4134 to 4140\n",
      "Complement found from 4433 to 4439\n",
      "Complement found from 7229 to 7235\n",
      "Complement found from 8278 to 8284\n",
      "Complement found from 8403 to 8409\n",
      "Complement found from 9529 to 9535\n",
      "Complement found from 10275 to 10281\n",
      "Complement found from 10976 to 10982\n",
      "Complement found from 11533 to 11539\n",
      "Complement found from 11864 to 11870\n",
      "Complement found from 12429 to 12435\n",
      "Seq is: TACCCC\n",
      "15 TACCCC motifs detected in file\n",
      "\n",
      "Complement found from 7335 to 7341\n",
      "Complement found from 9910 to 9916\n",
      "Complement found from 10675 to 10681\n",
      "Complement found from 11421 to 11427\n",
      "Seq is: CGAAGC\n",
      "4 CGAAGC motifs detected in file\n",
      "\n",
      "Complement found from 3335 to 3341\n",
      "Complement found from 13323 to 13329\n",
      "Seq is: TGTACC\n",
      "2 TGTACC motifs detected in file\n",
      "\n",
      "Seq is: CGAAAT\n",
      "0 CGAAAT motifs detected in file\n",
      "\n",
      "Complement found from 2412 to 2418\n",
      "Complement found from 4012 to 4018\n",
      "Complement found from 4250 to 4256\n",
      "Complement found from 4907 to 4913\n",
      "Complement found from 5549 to 5555\n",
      "Complement found from 7664 to 7670\n",
      "Complement found from 8473 to 8479\n",
      "Complement found from 10473 to 10479\n",
      "Complement found from 10611 to 10617\n",
      "Complement found from 10979 to 10985\n",
      "Complement found from 12040 to 12046\n",
      "Complement found from 12062 to 12068\n",
      "Complement found from 12101 to 12107\n",
      "Complement found from 13035 to 13041\n",
      "Complement found from 13450 to 13456\n",
      "Complement found from 15266 to 15272\n",
      "Complement found from 16220 to 16226\n",
      "Complement found from 16259 to 16265\n",
      "Complement found from 16377 to 16383\n",
      "Seq is: CCCTCA\n",
      "19 CCCTCA motifs detected in file\n",
      "\n",
      "Seq is: GTATTC\n",
      "0 GTATTC motifs detected in file\n",
      "\n",
      "Complement found from 1103 to 1109\n",
      "Complement found from 1231 to 1237\n",
      "Complement found from 3595 to 3601\n",
      "Complement found from 5366 to 5372\n",
      "Complement found from 5888 to 5894\n",
      "Complement found from 6543 to 6549\n",
      "Complement found from 8682 to 8688\n",
      "Complement found from 9467 to 9473\n",
      "Complement found from 10542 to 10548\n",
      "Complement found from 13843 to 13849\n",
      "Complement found from 14064 to 14070\n",
      "Complement found from 14410 to 14416\n",
      "Seq is: ACCTCA\n",
      "12 ACCTCA motifs detected in file\n",
      "\n",
      "Complement found from 3356 to 3362\n",
      "Complement found from 6965 to 6971\n",
      "Complement found from 9930 to 9936\n",
      "Complement found from 15091 to 15097\n",
      "Seq is: GGCATT\n",
      "4 GGCATT motifs detected in file\n",
      "\n",
      "Complement found from 39 to 45\n",
      "Complement found from 3790 to 3796\n",
      "Complement found from 3865 to 3871\n",
      "Complement found from 4219 to 4225\n",
      "Complement found from 4237 to 4243\n",
      "Complement found from 9966 to 9972\n",
      "Complement found from 10209 to 10215\n",
      "Complement found from 10710 to 10716\n",
      "Complement found from 12589 to 12595\n",
      "Complement found from 13019 to 13025\n",
      "Complement found from 13246 to 13252\n",
      "Complement found from 13900 to 13906\n",
      "Complement found from 14047 to 14053\n",
      "Seq is: TCTCCA\n",
      "13 TCTCCA motifs detected in file\n",
      "\n",
      "Complement found from 1751 to 1757\n",
      "Complement found from 1865 to 1871\n",
      "Complement found from 4566 to 4572\n",
      "Complement found from 5510 to 5516\n",
      "Complement found from 7524 to 7530\n",
      "Complement found from 7766 to 7772\n",
      "Complement found from 9130 to 9136\n",
      "Complement found from 9660 to 9666\n",
      "Complement found from 10149 to 10155\n",
      "Complement found from 10257 to 10263\n",
      "Complement found from 14008 to 14014\n",
      "Seq is: TAGAAA\n",
      "11 TAGAAA motifs detected in file\n",
      "\n",
      "Complement found from 3177 to 3183\n",
      "Complement found from 4212 to 4218\n",
      "Complement found from 6732 to 6738\n",
      "Complement found from 12651 to 12657\n",
      "Complement found from 14703 to 14709\n",
      "Complement found from 15559 to 15565\n",
      "Seq is: TGATAT\n",
      "6 TGATAT motifs detected in file\n",
      "\n",
      "Complement found from 10432 to 10438\n",
      "Seq is: TTTCGA\n",
      "1 TTTCGA motifs detected in file\n",
      "\n",
      "Complement found from 371 to 377\n",
      "Complement found from 441 to 447\n",
      "Complement found from 1613 to 1619\n",
      "Complement found from 2158 to 2164\n",
      "Complement found from 2704 to 2710\n",
      "Complement found from 5187 to 5193\n",
      "Complement found from 7715 to 7721\n",
      "Complement found from 9387 to 9393\n",
      "Complement found from 13162 to 13168\n",
      "Complement found from 14549 to 14555\n",
      "Complement found from 15470 to 15476\n",
      "Complement found from 16461 to 16467\n",
      "Seq is: TAACAC\n",
      "12 TAACAC motifs detected in file\n",
      "\n",
      "Complement found from 4122 to 4128\n",
      "Complement found from 6011 to 6017\n",
      "Complement found from 7414 to 7420\n",
      "Complement found from 13401 to 13407\n",
      "Seq is: ATTCGA\n",
      "4 ATTCGA motifs detected in file\n",
      "\n",
      "Complement found from 5552 to 5558\n",
      "Complement found from 15250 to 15256\n",
      "Seq is: TCAGTA\n",
      "2 TCAGTA motifs detected in file\n",
      "\n",
      "Complement found from 5196 to 5202\n",
      "Complement found from 8222 to 8228\n",
      "Complement found from 9112 to 9118\n",
      "Complement found from 9681 to 9687\n",
      "Complement found from 12379 to 12385\n",
      "Complement found from 13612 to 13618\n",
      "Seq is: TAATTC\n",
      "6 TAATTC motifs detected in file\n",
      "\n",
      "Complement found from 3300 to 3306\n",
      "Complement found from 4037 to 4043\n",
      "Complement found from 5393 to 5399\n",
      "Complement found from 6194 to 6200\n",
      "Complement found from 8640 to 8646\n",
      "Complement found from 9665 to 9671\n",
      "Complement found from 10297 to 10303\n",
      "Complement found from 10894 to 10900\n",
      "Complement found from 10941 to 10947\n",
      "Complement found from 11332 to 11338\n",
      "Complement found from 12028 to 12034\n",
      "Complement found from 12247 to 12253\n",
      "Complement found from 12551 to 12557\n",
      "Complement found from 12738 to 12744\n",
      "Complement found from 13743 to 13749\n",
      "Complement found from 13772 to 13778\n",
      "Complement found from 15361 to 15367\n",
      "Complement found from 15677 to 15683\n",
      "Complement found from 16076 to 16082\n",
      "Seq is: AACAAC\n",
      "19 AACAAC motifs detected in file\n",
      "\n",
      "Complement found from 9751 to 9757\n",
      "Seq is: CGAGTC\n",
      "1 CGAGTC motifs detected in file\n",
      "\n",
      "Complement found from 594 to 600\n",
      "Complement found from 777 to 783\n",
      "Complement found from 1162 to 1168\n",
      "Complement found from 3677 to 3683\n",
      "Complement found from 4252 to 4258\n",
      "Complement found from 8684 to 8690\n",
      "Complement found from 11751 to 11757\n",
      "Complement found from 13422 to 13428\n",
      "Complement found from 15874 to 15880\n",
      "Seq is: CTCAAA\n",
      "9 CTCAAA motifs detected in file\n",
      "\n",
      "Complement found from 10 to 16\n",
      "Complement found from 2275 to 2281\n",
      "Complement found from 3756 to 3762\n",
      "Complement found from 4382 to 4388\n",
      "Complement found from 7043 to 7049\n",
      "Complement found from 7146 to 7152\n",
      "Complement found from 7259 to 7265\n",
      "Complement found from 7628 to 7634\n",
      "Complement found from 9226 to 9232\n",
      "Complement found from 11023 to 11029\n",
      "Complement found from 11150 to 11156\n",
      "Complement found from 11314 to 11320\n",
      "Complement found from 13180 to 13186\n",
      "Complement found from 16227 to 16233\n",
      "Seq is: CTATCA\n",
      "14 CTATCA motifs detected in file\n",
      "\n",
      "Complement found from 364 to 370\n",
      "Complement found from 2061 to 2067\n",
      "Complement found from 2829 to 2835\n",
      "Complement found from 6697 to 6703\n",
      "Complement found from 7360 to 7366\n",
      "Complement found from 7420 to 7426\n",
      "Complement found from 7968 to 7974\n",
      "Complement found from 8340 to 8346\n",
      "Complement found from 8517 to 8523\n",
      "Seq is: AGAACC\n",
      "9 AGAACC motifs detected in file\n",
      "\n",
      "Complement found from 1487 to 1493\n",
      "Complement found from 6073 to 6079\n",
      "Seq is: CGTCAC\n",
      "2 CGTCAC motifs detected in file\n",
      "\n",
      "Complement found from 3539 to 3545\n",
      "Complement found from 4858 to 4864\n",
      "Complement found from 10509 to 10515\n",
      "Complement found from 11298 to 11304\n",
      "Complement found from 12643 to 12649\n",
      "Complement found from 13436 to 13442\n",
      "Complement found from 13619 to 13625\n",
      "Complement found from 15479 to 15485\n",
      "Seq is: TCTCAC\n",
      "8 TCTCAC motifs detected in file\n",
      "\n",
      "Complement found from 481 to 487\n",
      "Complement found from 4521 to 4527\n",
      "Complement found from 4748 to 4754\n",
      "Complement found from 5295 to 5301\n",
      "Complement found from 6393 to 6399\n",
      "Complement found from 6983 to 6989\n",
      "Complement found from 8429 to 8435\n",
      "Complement found from 8635 to 8641\n",
      "Complement found from 10341 to 10347\n",
      "Complement found from 11153 to 11159\n",
      "Complement found from 12796 to 12802\n",
      "Complement found from 14059 to 14065\n",
      "Complement found from 14935 to 14941\n",
      "Seq is: TCATCA\n",
      "13 TCATCA motifs detected in file\n",
      "\n",
      "Complement found from 2482 to 2488\n",
      "Complement found from 2657 to 2663\n",
      "Complement found from 6213 to 6219\n",
      "Complement found from 8908 to 8914\n",
      "Complement found from 9986 to 9992\n",
      "Complement found from 13108 to 13114\n",
      "Complement found from 13961 to 13967\n",
      "Seq is: TCTTAC\n",
      "7 TCTTAC motifs detected in file\n",
      "\n",
      "Complement found from 729 to 735\n",
      "Complement found from 1115 to 1121\n",
      "Complement found from 5624 to 5630\n",
      "Complement found from 5810 to 5816\n",
      "Complement found from 7876 to 7882\n",
      "Complement found from 10889 to 10895\n",
      "Complement found from 13002 to 13008\n",
      "Complement found from 14876 to 14882\n",
      "Complement found from 15395 to 15401\n",
      "Complement found from 15952 to 15958\n",
      "Complement found from 16414 to 16420\n",
      "Seq is: AAATCA\n",
      "11 AAATCA motifs detected in file\n",
      "\n",
      "Complement found from 2821 to 2827\n",
      "Complement found from 5817 to 5823\n",
      "Complement found from 7222 to 7228\n",
      "Complement found from 8778 to 8784\n",
      "Complement found from 13911 to 13917\n",
      "Seq is: CTCGGA\n",
      "5 CTCGGA motifs detected in file\n",
      "\n",
      "Complement found from 459 to 465\n",
      "Complement found from 6311 to 6317\n",
      "Complement found from 7179 to 7185\n",
      "Complement found from 9591 to 9597\n",
      "Complement found from 11813 to 11819\n",
      "Complement found from 14113 to 14119\n",
      "Complement found from 15261 to 15267\n",
      "Seq is: TCCCAC\n",
      "7 TCCCAC motifs detected in file\n",
      "\n",
      "Complement found from 1138 to 1144\n",
      "Complement found from 2354 to 2360\n",
      "Complement found from 5682 to 5688\n",
      "Complement found from 5950 to 5956\n",
      "Complement found from 7185 to 7191\n",
      "Complement found from 7716 to 7722\n",
      "Complement found from 8709 to 8715\n",
      "Complement found from 13163 to 13169\n",
      "Complement found from 14397 to 14403\n",
      "Complement found from 15324 to 15330\n",
      "Complement found from 15471 to 15477\n",
      "Complement found from 16462 to 16468\n",
      "Seq is: AACACT\n",
      "12 AACACT motifs detected in file\n",
      "\n",
      "Complement found from 444 to 450\n",
      "Complement found from 7411 to 7417\n",
      "Complement found from 8082 to 8088\n",
      "Complement found from 12022 to 12028\n",
      "Complement found from 13889 to 13895\n",
      "Complement found from 16336 to 16342\n",
      "Seq is: CACATT\n",
      "6 CACATT motifs detected in file\n",
      "\n",
      "Complement found from 2598 to 2604\n",
      "Complement found from 4758 to 4764\n",
      "Complement found from 5708 to 5714\n",
      "Complement found from 6468 to 6474\n",
      "Complement found from 6831 to 6837\n",
      "Complement found from 7670 to 7676\n",
      "Complement found from 8650 to 8656\n",
      "Complement found from 8671 to 8677\n",
      "Complement found from 8749 to 8755\n",
      "Complement found from 10080 to 10086\n",
      "Complement found from 10338 to 10344\n",
      "Complement found from 10453 to 10459\n",
      "Complement found from 11102 to 11108\n",
      "Complement found from 12677 to 12683\n",
      "Complement found from 14137 to 14143\n",
      "Complement found from 14217 to 14223\n",
      "Complement found from 14230 to 14236\n",
      "Seq is: TAATCA\n",
      "17 TAATCA motifs detected in file\n",
      "\n",
      "Complement found from 4146 to 4152\n",
      "Complement found from 5469 to 5475\n",
      "Complement found from 6823 to 6829\n",
      "Complement found from 7617 to 7623\n",
      "Complement found from 8139 to 8145\n",
      "Complement found from 13577 to 13583\n",
      "Complement found from 14983 to 14989\n",
      "Seq is: CGCTAC\n",
      "7 CGCTAC motifs detected in file\n",
      "\n",
      "Complement found from 145 to 151\n",
      "Complement found from 5298 to 5304\n",
      "Complement found from 7796 to 7802\n",
      "Complement found from 9861 to 9867\n",
      "Complement found from 10325 to 10331\n",
      "Complement found from 10344 to 10350\n",
      "Complement found from 10865 to 10871\n",
      "Complement found from 11663 to 11669\n",
      "Complement found from 12601 to 12607\n",
      "Complement found from 12880 to 12886\n",
      "Complement found from 13114 to 13120\n",
      "Complement found from 13372 to 13378\n",
      "Complement found from 14119 to 14125\n",
      "Complement found from 14978 to 14984\n",
      "Complement found from 15641 to 15647\n",
      "Seq is: TCATCC\n",
      "15 TCATCC motifs detected in file\n",
      "\n",
      "Complement found from 3163 to 3169\n",
      "Complement found from 4456 to 4462\n",
      "Complement found from 9080 to 9086\n",
      "Complement found from 11229 to 11235\n",
      "Complement found from 11537 to 11543\n",
      "Complement found from 13765 to 13771\n",
      "Complement found from 15401 to 15407\n",
      "Seq is: CCTTCC\n",
      "7 CCTTCC motifs detected in file\n",
      "\n",
      "Complement found from 2821 to 2827\n",
      "Complement found from 5817 to 5823\n",
      "Complement found from 7222 to 7228\n",
      "Complement found from 8778 to 8784\n",
      "Complement found from 13911 to 13917\n",
      "Seq is: CTCGGA\n",
      "5 CTCGGA motifs detected in file\n",
      "\n",
      "Complement found from 2894 to 2900\n",
      "Complement found from 6458 to 6464\n",
      "Complement found from 8614 to 8620\n",
      "Complement found from 14867 to 14873\n",
      "Seq is: TGATCC\n",
      "4 TGATCC motifs detected in file\n",
      "\n",
      "Complement found from 337 to 343\n",
      "Complement found from 629 to 635\n",
      "Complement found from 3497 to 3503\n",
      "Complement found from 6056 to 6062\n",
      "Complement found from 9604 to 9610\n",
      "Complement found from 13317 to 13323\n",
      "Complement found from 13515 to 13521\n",
      "Complement found from 14947 to 14953\n",
      "Complement found from 15034 to 15040\n",
      "Complement found from 15544 to 15550\n",
      "Complement found from 16173 to 16179\n",
      "Complement found from 16233 to 16239\n",
      "Seq is: CACATC\n",
      "12 CACATC motifs detected in file\n",
      "\n",
      "Complement found from 4946 to 4952\n",
      "Complement found from 5094 to 5100\n",
      "Complement found from 6639 to 6645\n",
      "Complement found from 8935 to 8941\n",
      "Complement found from 11008 to 11014\n",
      "Complement found from 11135 to 11141\n",
      "Complement found from 12901 to 12907\n",
      "Complement found from 15095 to 15101\n",
      "Seq is: TTATCC\n",
      "8 TTATCC motifs detected in file\n",
      "\n",
      "Complement found from 1182 to 1188\n",
      "Complement found from 3975 to 3981\n",
      "Complement found from 6104 to 6110\n",
      "Complement found from 7536 to 7542\n",
      "Complement found from 12070 to 12076\n",
      "Complement found from 14289 to 14295\n",
      "Complement found from 15127 to 15133\n",
      "Seq is: TTCATA\n",
      "7 TTCATA motifs detected in file\n",
      "\n",
      "Complement found from 6884 to 6890\n",
      "Complement found from 9815 to 9821\n",
      "Complement found from 14681 to 14687\n",
      "Seq is: CACGGA\n",
      "3 CACGGA motifs detected in file\n",
      "\n",
      "Complement found from 1301 to 1307\n",
      "Complement found from 1785 to 1791\n",
      "Complement found from 4653 to 4659\n",
      "Complement found from 7598 to 7604\n",
      "Seq is: CGCAAG\n",
      "4 CGCAAG motifs detected in file\n",
      "\n",
      "Complement found from 2329 to 2335\n",
      "Complement found from 14827 to 14833\n",
      "Seq is: TCCGCA\n",
      "2 TCCGCA motifs detected in file\n",
      "\n",
      "Complement found from 3073 to 3079\n",
      "Complement found from 6535 to 6541\n",
      "Seq is: AGACCG\n",
      "2 AGACCG motifs detected in file\n",
      "\n",
      "Complement found from 546 to 552\n",
      "Complement found from 2394 to 2400\n",
      "Complement found from 2909 to 2915\n",
      "Complement found from 3882 to 3888\n",
      "Complement found from 4153 to 4159\n",
      "Complement found from 6594 to 6600\n",
      "Complement found from 8343 to 8349\n",
      "Complement found from 8804 to 8810\n",
      "Complement found from 9350 to 9356\n",
      "Complement found from 13294 to 13300\n",
      "Complement found from 13857 to 13863\n",
      "Complement found from 14184 to 14190\n",
      "Complement found from 15843 to 15849\n",
      "Complement found from 16276 to 16282\n",
      "Seq is: ACCAAC\n",
      "14 ACCAAC motifs detected in file\n",
      "\n",
      "Complement found from 3078 to 3084\n",
      "Complement found from 9626 to 9632\n",
      "Seq is: GGAGTA\n",
      "2 GGAGTA motifs detected in file\n",
      "\n",
      "Complement found from 2055 to 2061\n",
      "Complement found from 7025 to 7031\n",
      "Complement found from 8291 to 8297\n",
      "Complement found from 8391 to 8397\n",
      "Complement found from 8901 to 8907\n",
      "Complement found from 11710 to 11716\n",
      "Complement found from 13144 to 13150\n",
      "Complement found from 14944 to 14950\n",
      "Complement found from 15698 to 15704\n",
      "Complement found from 16534 to 16540\n",
      "Seq is: GCCCAC\n",
      "10 GCCCAC motifs detected in file\n",
      "\n",
      "Complement found from 2881 to 2887\n",
      "Complement found from 3556 to 3562\n",
      "Complement found from 3754 to 3760\n",
      "Complement found from 5154 to 5160\n",
      "Complement found from 10557 to 10563\n",
      "Complement found from 10578 to 10584\n",
      "Complement found from 10874 to 10880\n",
      "Complement found from 11546 to 11552\n",
      "Complement found from 11732 to 11738\n",
      "Complement found from 12350 to 12356\n",
      "Complement found from 13348 to 13354\n",
      "Complement found from 15191 to 15197\n",
      "Complement found from 15629 to 15635\n",
      "Complement found from 15812 to 15818\n",
      "Seq is: TACTAT\n",
      "14 TACTAT motifs detected in file\n",
      "\n",
      "Seq is: CGGTTC\n",
      "0 CGGTTC motifs detected in file\n",
      "\n",
      "Complement found from 430 to 436\n",
      "Complement found from 565 to 571\n",
      "Complement found from 634 to 640\n",
      "Complement found from 799 to 805\n",
      "Complement found from 953 to 959\n",
      "Complement found from 4388 to 4394\n",
      "Complement found from 4831 to 4837\n",
      "Complement found from 5434 to 5440\n",
      "Complement found from 5892 to 5898\n",
      "Complement found from 8269 to 8275\n",
      "Complement found from 8929 to 8935\n",
      "Complement found from 9568 to 9574\n",
      "Complement found from 10159 to 10165\n",
      "Complement found from 12965 to 12971\n",
      "Complement found from 13023 to 13029\n",
      "Complement found from 13055 to 13061\n",
      "Complement found from 13125 to 13131\n",
      "Complement found from 14337 to 14343\n",
      "Complement found from 14811 to 14817\n",
      "Complement found from 15534 to 15540\n",
      "Complement found from 16256 to 16262\n",
      "Seq is: CACCCC\n",
      "21 CACCCC motifs detected in file\n",
      "\n",
      "Complement found from 7152 to 7158\n",
      "Complement found from 12493 to 12499\n",
      "Complement found from 12598 to 12604\n",
      "Complement found from 14298 to 14304\n",
      "Seq is: TATTCA\n",
      "4 TATTCA motifs detected in file\n",
      "\n",
      "Complement found from 295 to 301\n",
      "Complement found from 2186 to 2192\n",
      "Complement found from 3473 to 3479\n",
      "Complement found from 4085 to 4091\n",
      "Complement found from 7888 to 7894\n",
      "Complement found from 8477 to 8483\n",
      "Complement found from 8803 to 8809\n",
      "Complement found from 9212 to 9218\n",
      "Complement found from 14040 to 14046\n",
      "Complement found from 14183 to 14189\n",
      "Complement found from 14249 to 14255\n",
      "Complement found from 14364 to 14370\n",
      "Complement found from 14403 to 14409\n",
      "Complement found from 14741 to 14747\n",
      "Seq is: CACCAA\n",
      "14 CACCAA motifs detected in file\n",
      "\n",
      "Complement found from 6203 to 6209\n",
      "Complement found from 11680 to 11686\n",
      "Complement found from 12570 to 12576\n",
      "Complement found from 14303 to 14309\n",
      "Seq is: AGCTTC\n",
      "4 AGCTTC motifs detected in file\n",
      "\n",
      "Complement found from 667 to 673\n",
      "Complement found from 3096 to 3102\n",
      "Complement found from 6588 to 6594\n",
      "Complement found from 8596 to 8602\n",
      "Seq is: TTCTAT\n",
      "4 TTCTAT motifs detected in file\n",
      "\n",
      "Complement found from 396 to 402\n",
      "Complement found from 5883 to 5889\n",
      "Complement found from 9704 to 9710\n",
      "Complement found from 9720 to 9726\n",
      "Complement found from 10073 to 10079\n",
      "Complement found from 11109 to 11115\n",
      "Complement found from 13892 to 13898\n",
      "Seq is: ATTTTA\n",
      "7 ATTTTA motifs detected in file\n",
      "\n",
      "Complement found from 195 to 201\n",
      "Complement found from 473 to 479\n",
      "Complement found from 860 to 866\n",
      "Complement found from 3766 to 3772\n",
      "Complement found from 4464 to 4470\n",
      "Complement found from 5496 to 5502\n",
      "Complement found from 6528 to 6534\n",
      "Complement found from 7740 to 7746\n",
      "Complement found from 9345 to 9351\n",
      "Complement found from 10104 to 10110\n",
      "Complement found from 10958 to 10964\n",
      "Complement found from 11051 to 11057\n",
      "Complement found from 12718 to 12724\n",
      "Complement found from 13655 to 13661\n",
      "Complement found from 13684 to 13690\n",
      "Complement found from 13739 to 13745\n",
      "Complement found from 14214 to 14220\n",
      "Complement found from 14580 to 14586\n",
      "Complement found from 14634 to 14640\n",
      "Seq is: TACTAA\n",
      "19 TACTAA motifs detected in file\n",
      "\n",
      "Seq is: TCAAGA\n",
      "0 TCAAGA motifs detected in file\n",
      "\n",
      "Complement found from 7914 to 7920\n",
      "Seq is: CACCGA\n",
      "1 CACCGA motifs detected in file\n",
      "\n",
      "Complement found from 329 to 335\n",
      "Complement found from 1634 to 1640\n",
      "Complement found from 5684 to 5690\n",
      "Complement found from 9091 to 9097\n",
      "Complement found from 11005 to 11011\n",
      "Complement found from 11132 to 11138\n",
      "Complement found from 11395 to 11401\n",
      "Seq is: CACTTA\n",
      "7 CACTTA motifs detected in file\n",
      "\n",
      "Complement found from 173 to 179\n",
      "Complement found from 2379 to 2385\n",
      "Complement found from 4699 to 4705\n",
      "Complement found from 5802 to 5808\n",
      "Complement found from 6409 to 6415\n",
      "Complement found from 6892 to 6898\n",
      "Complement found from 9064 to 9070\n",
      "Complement found from 10636 to 10642\n",
      "Complement found from 12490 to 12496\n",
      "Complement found from 14174 to 14180\n",
      "Complement found from 15008 to 15014\n",
      "Complement found from 16418 to 16424\n",
      "Seq is: CAATAT\n",
      "12 CAATAT motifs detected in file\n",
      "\n",
      "Complement found from 5196 to 5202\n",
      "Complement found from 8222 to 8228\n",
      "Complement found from 9112 to 9118\n",
      "Complement found from 9681 to 9687\n",
      "Complement found from 12379 to 12385\n",
      "Complement found from 13612 to 13618\n",
      "Seq is: TAATTC\n",
      "6 TAATTC motifs detected in file\n",
      "\n",
      "Complement found from 725 to 731\n",
      "Complement found from 2067 to 2073\n",
      "Complement found from 5997 to 6003\n",
      "Complement found from 7284 to 7290\n",
      "Complement found from 8879 to 8885\n",
      "Complement found from 13159 to 13165\n",
      "Seq is: CTCTAA\n",
      "6 CTCTAA motifs detected in file\n",
      "\n",
      "Complement found from 3002 to 3008\n",
      "Complement found from 4841 to 4847\n",
      "Complement found from 7254 to 7260\n",
      "Complement found from 9605 to 9611\n",
      "Complement found from 9887 to 9893\n",
      "Complement found from 11722 to 11728\n",
      "Seq is: ACATCC\n",
      "6 ACATCC motifs detected in file\n",
      "\n",
      "Complement found from 725 to 731\n",
      "Complement found from 2067 to 2073\n",
      "Complement found from 5997 to 6003\n",
      "Complement found from 7284 to 7290\n",
      "Complement found from 8879 to 8885\n",
      "Complement found from 13159 to 13165\n",
      "Seq is: CTCTAA\n",
      "6 CTCTAA motifs detected in file\n",
      "\n",
      "Complement found from 1183 to 1189\n",
      "Complement found from 3726 to 3732\n",
      "Complement found from 4890 to 4896\n",
      "Complement found from 7149 to 7155\n",
      "Complement found from 9229 to 9235\n",
      "Complement found from 10456 to 10462\n",
      "Complement found from 10545 to 10551\n",
      "Complement found from 11105 to 11111\n",
      "Seq is: TCATAT\n",
      "8 TCATAT motifs detected in file\n",
      "\n",
      "Complement found from 38 to 44\n",
      "Complement found from 3789 to 3795\n",
      "Complement found from 4053 to 4059\n",
      "Complement found from 4706 to 4712\n",
      "Complement found from 4903 to 4909\n",
      "Complement found from 5213 to 5219\n",
      "Complement found from 6225 to 6231\n",
      "Complement found from 6495 to 6501\n",
      "Complement found from 11933 to 11939\n",
      "Complement found from 12561 to 12567\n",
      "Complement found from 14283 to 14289\n",
      "Complement found from 15454 to 15460\n",
      "Complement found from 16441 to 16447\n",
      "Seq is: CTCTCC\n",
      "13 CTCTCC motifs detected in file\n",
      "\n",
      "Complement found from 478 to 484\n",
      "Complement found from 3184 to 3190\n",
      "Complement found from 3723 to 3729\n",
      "Complement found from 4883 to 4889\n",
      "Complement found from 7747 to 7753\n",
      "Complement found from 8632 to 8638\n",
      "Complement found from 10508 to 10514\n",
      "Complement found from 14164 to 14170\n",
      "Seq is: ATCTCA\n",
      "8 ATCTCA motifs detected in file\n",
      "\n",
      "Complement found from 285 to 291\n",
      "Complement found from 992 to 998\n",
      "Complement found from 2129 to 2135\n",
      "Complement found from 2150 to 2156\n",
      "Complement found from 2226 to 2232\n",
      "Complement found from 2456 to 2462\n",
      "Complement found from 4175 to 4181\n",
      "Complement found from 4604 to 4610\n",
      "Complement found from 5281 to 5287\n",
      "Complement found from 5746 to 5752\n",
      "Complement found from 6691 to 6697\n",
      "Complement found from 7446 to 7452\n",
      "Complement found from 8496 to 8502\n",
      "Complement found from 10047 to 10053\n",
      "Complement found from 10813 to 10819\n",
      "Complement found from 11031 to 11037\n",
      "Complement found from 12417 to 12423\n",
      "Complement found from 13230 to 13236\n",
      "Complement found from 14503 to 14509\n",
      "Seq is: AAAAAA\n",
      "19 AAAAAA motifs detected in file\n",
      "\n",
      "Complement found from 11245 to 11251\n",
      "Complement found from 13084 to 13090\n",
      "Complement found from 14904 to 14910\n",
      "Seq is: GCACTA\n",
      "3 GCACTA motifs detected in file\n",
      "\n",
      "Complement found from 732 to 738\n",
      "Complement found from 1234 to 1240\n",
      "Complement found from 2521 to 2527\n",
      "Complement found from 2863 to 2869\n",
      "Complement found from 3472 to 3478\n",
      "Complement found from 3541 to 3547\n",
      "Complement found from 4015 to 4021\n",
      "Complement found from 4084 to 4090\n",
      "Complement found from 8476 to 8482\n",
      "Complement found from 8653 to 8659\n",
      "Complement found from 9646 to 9652\n",
      "Complement found from 9762 to 9768\n",
      "Complement found from 13183 to 13189\n",
      "Complement found from 13453 to 13459\n",
      "Complement found from 14402 to 14408\n",
      "Complement found from 14879 to 14885\n",
      "Complement found from 14911 to 14917\n",
      "Complement found from 15481 to 15487\n",
      "Seq is: TCACCA\n",
      "18 TCACCA motifs detected in file\n",
      "\n",
      "Complement found from 2549 to 2555\n",
      "Complement found from 7380 to 7386\n",
      "Complement found from 10371 to 10377\n",
      "Seq is: AGTGAC\n",
      "3 AGTGAC motifs detected in file\n",
      "\n",
      "Complement found from 2948 to 2954\n",
      "Complement found from 3969 to 3975\n",
      "Complement found from 5921 to 5927\n",
      "Complement found from 6602 to 6608\n",
      "Complement found from 7097 to 7103\n",
      "Complement found from 8422 to 8428\n",
      "Complement found from 11206 to 11212\n",
      "Complement found from 11734 to 11740\n",
      "Complement found from 12744 to 12750\n",
      "Complement found from 13719 to 13725\n",
      "Complement found from 14149 to 14155\n",
      "Complement found from 14890 to 14896\n",
      "Complement found from 15337 to 15343\n",
      "Complement found from 15475 to 15481\n",
      "Complement found from 15568 to 15574\n",
      "Complement found from 16013 to 16019\n",
      "Seq is: CTATTC\n",
      "16 CTATTC motifs detected in file\n",
      "\n",
      "Complement found from 1783 to 1789\n",
      "Complement found from 4661 to 4667\n",
      "Complement found from 5108 to 5114\n",
      "Complement found from 6537 to 6543\n",
      "Complement found from 13527 to 13533\n",
      "Complement found from 13936 to 13942\n",
      "Seq is: ACCGCA\n",
      "6 ACCGCA motifs detected in file\n",
      "\n",
      "Complement found from 3372 to 3378\n",
      "Complement found from 3886 to 3892\n",
      "Complement found from 9669 to 9675\n",
      "Complement found from 10397 to 10403\n",
      "Seq is: ACCGAA\n",
      "4 ACCGAA motifs detected in file\n",
      "\n",
      "Complement found from 2571 to 2577\n",
      "Complement found from 16127 to 16133\n",
      "Seq is: CGGTAC\n",
      "2 CGGTAC motifs detected in file\n",
      "\n",
      "Complement found from 371 to 377\n",
      "Complement found from 441 to 447\n",
      "Complement found from 1613 to 1619\n",
      "Complement found from 2158 to 2164\n",
      "Complement found from 2704 to 2710\n",
      "Complement found from 5187 to 5193\n",
      "Complement found from 7715 to 7721\n",
      "Complement found from 9387 to 9393\n",
      "Complement found from 13162 to 13168\n",
      "Complement found from 14549 to 14555\n",
      "Complement found from 15470 to 15476\n",
      "Complement found from 16461 to 16467\n",
      "Seq is: TAACAC\n",
      "12 TAACAC motifs detected in file\n",
      "\n",
      "Complement found from 960 to 966\n",
      "Complement found from 3571 to 3577\n",
      "Complement found from 5301 to 5307\n",
      "Complement found from 5382 to 5388\n",
      "Complement found from 5445 to 5451\n",
      "Complement found from 6843 to 6849\n",
      "Complement found from 8078 to 8084\n",
      "Complement found from 8617 to 8623\n",
      "Complement found from 8938 to 8944\n",
      "Complement found from 10915 to 10921\n",
      "Complement found from 11138 to 11144\n",
      "Complement found from 12481 to 12487\n",
      "Complement found from 13645 to 13651\n",
      "Complement found from 13878 to 13884\n",
      "Complement found from 14807 to 14813\n",
      "Complement found from 15540 to 15546\n",
      "Complement found from 15656 to 15662\n",
      "Complement found from 16188 to 16194\n",
      "Complement found from 16361 to 16367\n",
      "Seq is: TCCCCA\n",
      "19 TCCCCA motifs detected in file\n",
      "\n",
      "Complement found from 592 to 598\n",
      "Complement found from 1495 to 1501\n",
      "Complement found from 2252 to 2258\n",
      "Complement found from 3328 to 3334\n",
      "Complement found from 4647 to 4653\n",
      "Complement found from 4929 to 4935\n",
      "Complement found from 5012 to 5018\n",
      "Complement found from 7805 to 7811\n",
      "Complement found from 8426 to 8432\n",
      "Complement found from 9330 to 9336\n",
      "Complement found from 9846 to 9852\n",
      "Complement found from 11725 to 11731\n",
      "Complement found from 13489 to 13495\n",
      "Complement found from 14443 to 14449\n",
      "Complement found from 15638 to 15644\n",
      "Complement found from 15740 to 15746\n",
      "Seq is: TCCTCA\n",
      "16 TCCTCA motifs detected in file\n",
      "\n",
      "Complement found from 2958 to 2964\n",
      "Complement found from 12629 to 12635\n",
      "Complement found from 13367 to 13373\n",
      "Seq is: GTCCAT\n",
      "3 GTCCAT motifs detected in file\n",
      "\n",
      "Complement found from 2388 to 2394\n",
      "Complement found from 4738 to 4744\n",
      "Complement found from 4887 to 4893\n",
      "Complement found from 5370 to 5376\n",
      "Complement found from 9215 to 9221\n",
      "Complement found from 9633 to 9639\n",
      "Complement found from 10985 to 10991\n",
      "Complement found from 11627 to 11633\n",
      "Complement found from 14573 to 14579\n",
      "Complement found from 15419 to 15425\n",
      "Complement found from 15709 to 15715\n",
      "Complement found from 16213 to 16219\n",
      "Seq is: CAATCA\n",
      "12 CAATCA motifs detected in file\n",
      "\n",
      "Complement found from 3082 to 3088\n",
      "Complement found from 4471 to 4477\n",
      "Complement found from 4671 to 4677\n",
      "Complement found from 9148 to 9154\n",
      "Complement found from 9453 to 9459\n",
      "Complement found from 11783 to 11789\n",
      "Complement found from 12953 to 12959\n",
      "Complement found from 13150 to 13156\n",
      "Complement found from 15653 to 15659\n",
      "Complement found from 15833 to 15839\n",
      "Seq is: TAATCC\n",
      "10 TAATCC motifs detected in file\n",
      "\n",
      "Complement found from 309 to 315\n",
      "Complement found from 3166 to 3172\n",
      "Complement found from 4247 to 4253\n",
      "Complement found from 7953 to 7959\n",
      "Complement found from 8603 to 8609\n",
      "Complement found from 10190 to 10196\n",
      "Complement found from 12082 to 12088\n",
      "Complement found from 12383 to 12389\n",
      "Complement found from 13752 to 13758\n",
      "Complement found from 13761 to 13767\n",
      "Complement found from 13780 to 13786\n",
      "Complement found from 14153 to 14159\n",
      "Complement found from 14487 to 14493\n",
      "Complement found from 14529 to 14535\n",
      "Seq is: TCCCCC\n",
      "14 TCCCCC motifs detected in file\n",
      "\n",
      "Complement found from 1119 to 1125\n",
      "Complement found from 2396 to 2402\n",
      "Complement found from 2966 to 2972\n",
      "Complement found from 4696 to 4702\n",
      "Complement found from 6399 to 6405\n",
      "Complement found from 7724 to 7730\n",
      "Complement found from 8639 to 8645\n",
      "Complement found from 8661 to 8667\n",
      "Complement found from 10778 to 10784\n",
      "Complement found from 10893 to 10899\n",
      "Complement found from 11331 to 11337\n",
      "Complement found from 12487 to 12493\n",
      "Complement found from 12930 to 12936\n",
      "Complement found from 13774 to 13780\n",
      "Complement found from 13859 to 13865\n",
      "Complement found from 14186 to 14192\n",
      "Complement found from 15679 to 15685\n",
      "Complement found from 15824 to 15830\n",
      "Complement found from 16075 to 16081\n",
      "Complement found from 16278 to 16284\n",
      "Seq is: CAACAA\n",
      "20 CAACAA motifs detected in file\n",
      "\n",
      "Complement found from 309 to 315\n",
      "Complement found from 3166 to 3172\n",
      "Complement found from 4247 to 4253\n",
      "Complement found from 7953 to 7959\n",
      "Complement found from 8603 to 8609\n",
      "Complement found from 10190 to 10196\n",
      "Complement found from 12082 to 12088\n",
      "Complement found from 12383 to 12389\n",
      "Complement found from 13752 to 13758\n",
      "Complement found from 13761 to 13767\n",
      "Complement found from 13780 to 13786\n",
      "Complement found from 14153 to 14159\n",
      "Complement found from 14487 to 14493\n",
      "Complement found from 14529 to 14535\n",
      "Seq is: TCCCCC\n",
      "14 TCCCCC motifs detected in file\n",
      "\n",
      "Complement found from 5291 to 5297\n",
      "Complement found from 6002 to 6008\n",
      "Complement found from 6322 to 6328\n",
      "Complement found from 9180 to 9186\n",
      "Complement found from 9736 to 9742\n",
      "Complement found from 11845 to 11851\n",
      "Complement found from 12960 to 12966\n",
      "Seq is: AGCCTC\n",
      "7 AGCCTC motifs detected in file\n",
      "\n",
      "Complement found from 6941 to 6947\n",
      "Complement found from 7708 to 7714\n",
      "Complement found from 10923 to 10929\n",
      "Complement found from 14931 to 14937\n",
      "Seq is: CTTTTC\n",
      "4 CTTTTC motifs detected in file\n",
      "\n",
      "Complement found from 490 to 496\n",
      "Complement found from 2422 to 2428\n",
      "Complement found from 3444 to 3450\n",
      "Complement found from 3581 to 3587\n",
      "Complement found from 4484 to 4490\n",
      "Complement found from 5056 to 5062\n",
      "Complement found from 7487 to 7493\n",
      "Complement found from 10943 to 10949\n",
      "Complement found from 11508 to 11514\n",
      "Complement found from 12105 to 12111\n",
      "Complement found from 12544 to 12550\n",
      "Complement found from 12553 to 12559\n",
      "Complement found from 14068 to 14074\n",
      "Complement found from 14271 to 14277\n",
      "Complement found from 14414 to 14420\n",
      "Complement found from 15363 to 15369\n",
      "Complement found from 15522 to 15528\n",
      "Complement found from 16217 to 16223\n",
      "Seq is: CAACCC\n",
      "18 CAACCC motifs detected in file\n",
      "\n",
      "Seq is: CTTGTC\n",
      "0 CTTGTC motifs detected in file\n",
      "\n",
      "Complement found from 1421 to 1427\n",
      "Complement found from 1972 to 1978\n",
      "Complement found from 3859 to 3865\n",
      "Complement found from 5855 to 5861\n",
      "Complement found from 12898 to 12904\n",
      "Seq is: GATTTA\n",
      "5 GATTTA motifs detected in file\n",
      "\n",
      "Complement found from 10712 to 10718\n",
      "Complement found from 12748 to 12754\n",
      "Complement found from 12912 to 12918\n",
      "Complement found from 13902 to 13908\n",
      "Complement found from 14818 to 14824\n",
      "Seq is: TCCAAC\n",
      "5 TCCAAC motifs detected in file\n",
      "\n",
      "Complement found from 3300 to 3306\n",
      "Complement found from 4037 to 4043\n",
      "Complement found from 5393 to 5399\n",
      "Complement found from 6194 to 6200\n",
      "Complement found from 8640 to 8646\n",
      "Complement found from 9665 to 9671\n",
      "Complement found from 10297 to 10303\n",
      "Complement found from 10894 to 10900\n",
      "Complement found from 10941 to 10947\n",
      "Complement found from 11332 to 11338\n",
      "Complement found from 12028 to 12034\n",
      "Complement found from 12247 to 12253\n",
      "Complement found from 12551 to 12557\n",
      "Complement found from 12738 to 12744\n",
      "Complement found from 13743 to 13749\n",
      "Complement found from 13772 to 13778\n",
      "Complement found from 15361 to 15367\n",
      "Complement found from 15677 to 15683\n",
      "Complement found from 16076 to 16082\n",
      "Seq is: AACAAC\n",
      "19 AACAAC motifs detected in file\n",
      "\n",
      "Complement found from 1100 to 1106\n",
      "Complement found from 1218 to 1224\n",
      "Complement found from 1672 to 1678\n",
      "Complement found from 4313 to 4319\n",
      "Complement found from 4611 to 4617\n",
      "Complement found from 4977 to 4983\n",
      "Complement found from 5672 to 5678\n",
      "Complement found from 7371 to 7377\n",
      "Complement found from 8120 to 8126\n",
      "Complement found from 8824 to 8830\n",
      "Complement found from 10746 to 10752\n",
      "Complement found from 13687 to 13693\n",
      "Complement found from 14514 to 14520\n",
      "Complement found from 14583 to 14589\n",
      "Complement found from 14637 to 14643\n",
      "Complement found from 15920 to 15926\n",
      "Seq is: TAAACC\n",
      "16 TAAACC motifs detected in file\n",
      "\n",
      "Complement found from 395 to 401\n",
      "Complement found from 7306 to 7312\n",
      "Complement found from 9703 to 9709\n",
      "Complement found from 10072 to 10078\n",
      "Complement found from 12308 to 12314\n",
      "Seq is: AATTTT\n",
      "5 AATTTT motifs detected in file\n",
      "\n",
      "Complement found from 3799 to 3805\n",
      "Complement found from 6405 to 6411\n",
      "Complement found from 7643 to 7649\n",
      "Complement found from 9094 to 9100\n",
      "Complement found from 12469 to 12475\n",
      "Seq is: TTATCA\n",
      "5 TTATCA motifs detected in file\n",
      "\n",
      "Complement found from 4054 to 4060\n",
      "Complement found from 4904 to 4910\n",
      "Complement found from 5214 to 5220\n",
      "Complement found from 5484 to 5490\n",
      "Complement found from 5731 to 5737\n",
      "Complement found from 6496 to 6502\n",
      "Complement found from 6663 to 6669\n",
      "Complement found from 9755 to 9761\n",
      "Complement found from 11057 to 11063\n",
      "Complement found from 12562 to 12568\n",
      "Complement found from 15851 to 15857\n",
      "Seq is: TCTCCC\n",
      "11 TCTCCC motifs detected in file\n",
      "\n",
      "Complement found from 470 to 476\n",
      "Complement found from 1700 to 1706\n",
      "Complement found from 3439 to 3445\n",
      "Complement found from 4732 to 4738\n",
      "Complement found from 5104 to 5110\n",
      "Complement found from 5118 to 5124\n",
      "Complement found from 5151 to 5157\n",
      "Complement found from 6525 to 6531\n",
      "Complement found from 6680 to 6686\n",
      "Complement found from 6999 to 7005\n",
      "Complement found from 7013 to 7019\n",
      "Complement found from 8158 to 8164\n",
      "Complement found from 8379 to 8385\n",
      "Complement found from 10101 to 10107\n",
      "Complement found from 10790 to 10796\n",
      "Complement found from 11288 to 11294\n",
      "Complement found from 14211 to 14217\n",
      "Complement found from 15412 to 15418\n",
      "Seq is: TACTAC\n",
      "18 TACTAC motifs detected in file\n",
      "\n",
      "Complement found from 8864 to 8870\n",
      "Seq is: GATTAT\n",
      "1 GATTAT motifs detected in file\n",
      "\n",
      "Complement found from 3082 to 3088\n",
      "Complement found from 4471 to 4477\n",
      "Complement found from 4671 to 4677\n",
      "Complement found from 9148 to 9154\n",
      "Complement found from 9453 to 9459\n",
      "Complement found from 11783 to 11789\n",
      "Complement found from 12953 to 12959\n",
      "Complement found from 13150 to 13156\n",
      "Complement found from 15653 to 15659\n",
      "Complement found from 15833 to 15839\n",
      "Seq is: TAATCC\n",
      "10 TAATCC motifs detected in file\n",
      "\n",
      "Complement found from 563 to 569\n",
      "Seq is: GACACC\n",
      "1 GACACC motifs detected in file\n",
      "\n",
      "Complement found from 295 to 301\n",
      "Complement found from 2186 to 2192\n",
      "Complement found from 3473 to 3479\n",
      "Complement found from 4085 to 4091\n",
      "Complement found from 7888 to 7894\n",
      "Complement found from 8477 to 8483\n",
      "Complement found from 8803 to 8809\n",
      "Complement found from 9212 to 9218\n",
      "Complement found from 14040 to 14046\n",
      "Complement found from 14183 to 14189\n",
      "Complement found from 14249 to 14255\n",
      "Complement found from 14364 to 14370\n",
      "Complement found from 14403 to 14409\n",
      "Complement found from 14741 to 14747\n",
      "Seq is: CACCAA\n",
      "14 CACCAA motifs detected in file\n",
      "\n",
      "Complement found from 3803 to 3809\n",
      "Complement found from 4070 to 4076\n",
      "Complement found from 6397 to 6403\n",
      "Complement found from 7182 to 7188\n",
      "Complement found from 7722 to 7728\n",
      "Complement found from 8706 to 8712\n",
      "Complement found from 8765 to 8771\n",
      "Complement found from 10131 to 10137\n",
      "Complement found from 10836 to 10842\n",
      "Complement found from 11260 to 11266\n",
      "Complement found from 11994 to 12000\n",
      "Complement found from 12485 to 12491\n",
      "Complement found from 12542 to 12548\n",
      "Complement found from 12928 to 12934\n",
      "Complement found from 13377 to 13383\n",
      "Complement found from 14328 to 14334\n",
      "Complement found from 15822 to 15828\n",
      "Seq is: CACAAC\n",
      "17 CACAAC motifs detected in file\n",
      "\n",
      "Complement found from 5556 to 5562\n",
      "Complement found from 8324 to 8330\n",
      "Seq is: TAAGTT\n",
      "2 TAAGTT motifs detected in file\n",
      "\n",
      "Complement found from 6111 to 6117\n",
      "Complement found from 7737 to 7743\n",
      "Complement found from 10575 to 10581\n",
      "Complement found from 10955 to 10961\n",
      "Complement found from 11486 to 11492\n",
      "Complement found from 15839 to 15845\n",
      "Complement found from 15904 to 15910\n",
      "Seq is: TAATAC\n",
      "7 TAATAC motifs detected in file\n",
      "\n",
      "Complement found from 283 to 289\n",
      "Complement found from 1009 to 1015\n",
      "Complement found from 1121 to 1127\n",
      "Complement found from 4867 to 4873\n",
      "Complement found from 5279 to 5285\n",
      "Complement found from 5426 to 5432\n",
      "Complement found from 7444 to 7450\n",
      "Complement found from 7726 to 7732\n",
      "Complement found from 10378 to 10384\n",
      "Complement found from 10419 to 10425\n",
      "Complement found from 11518 to 11524\n",
      "Complement found from 12415 to 12421\n",
      "Complement found from 13217 to 13223\n",
      "Complement found from 15866 to 15872\n",
      "Seq is: ACAAAA\n",
      "14 ACAAAA motifs detected in file\n",
      "\n",
      "Complement found from 825 to 831\n",
      "Complement found from 1711 to 1717\n",
      "Complement found from 2133 to 2139\n",
      "Complement found from 6038 to 6044\n",
      "Complement found from 8317 to 8323\n",
      "Complement found from 9078 to 9084\n",
      "Complement found from 10920 to 10926\n",
      "Complement found from 13380 to 13386\n",
      "Complement found from 15935 to 15941\n",
      "Seq is: AACCTT\n",
      "9 AACCTT motifs detected in file\n",
      "\n",
      "Complement found from 5552 to 5558\n",
      "Complement found from 15250 to 15256\n",
      "Seq is: TCAGTA\n",
      "2 TCAGTA motifs detected in file\n",
      "\n",
      "Complement found from 426 to 432\n",
      "Complement found from 2867 to 2873\n",
      "Complement found from 3274 to 3280\n",
      "Complement found from 11693 to 11699\n",
      "Complement found from 16343 to 16349\n",
      "Seq is: CAGTCA\n",
      "5 CAGTCA motifs detected in file\n",
      "\n",
      "Complement found from 3505 to 3511\n",
      "Complement found from 3745 to 3751\n",
      "Complement found from 3834 to 3840\n",
      "Complement found from 4635 to 4641\n",
      "Complement found from 4950 to 4956\n",
      "Complement found from 5307 to 5313\n",
      "Complement found from 5319 to 5325\n",
      "Complement found from 6117 to 6123\n",
      "Complement found from 6384 to 6390\n",
      "Complement found from 7065 to 7071\n",
      "Complement found from 7793 to 7799\n",
      "Complement found from 7871 to 7877\n",
      "Complement found from 8962 to 8968\n",
      "Complement found from 12631 to 12637\n",
      "Complement found from 13369 to 13375\n",
      "Complement found from 14056 to 14062\n",
      "Complement found from 14341 to 14347\n",
      "Complement found from 14480 to 14486\n",
      "Complement found from 15788 to 15794\n",
      "Complement found from 16071 to 16077\n",
      "Seq is: CCATCA\n",
      "20 CCATCA motifs detected in file\n",
      "\n",
      "Complement found from 1541 to 1547\n",
      "Complement found from 4207 to 4213\n",
      "Complement found from 5089 to 5095\n",
      "Complement found from 6631 to 6637\n",
      "Complement found from 10784 to 10790\n",
      "Complement found from 11112 to 11118\n",
      "Complement found from 12781 to 12787\n",
      "Seq is: TTATAT\n",
      "7 TTATAT motifs detected in file\n",
      "\n",
      "Complement found from 238 to 244\n",
      "Complement found from 281 to 287\n",
      "Complement found from 2762 to 2768\n",
      "Complement found from 3299 to 3305\n",
      "Complement found from 5392 to 5398\n",
      "Complement found from 8505 to 8511\n",
      "Complement found from 10940 to 10946\n",
      "Complement found from 11570 to 11576\n",
      "Complement found from 12027 to 12033\n",
      "Complement found from 12246 to 12252\n",
      "Complement found from 12413 to 12419\n",
      "Complement found from 12737 to 12743\n",
      "Complement found from 13385 to 13391\n",
      "Complement found from 13742 to 13748\n",
      "Complement found from 14570 to 14576\n",
      "Complement found from 15600 to 15606\n",
      "Seq is: TAACAA\n",
      "16 TAACAA motifs detected in file\n",
      "\n",
      "Complement found from 6421 to 6427\n",
      "Seq is: CCCTGC\n",
      "1 CCCTGC motifs detected in file\n",
      "\n",
      "Complement found from 2598 to 2604\n",
      "Complement found from 4758 to 4764\n",
      "Complement found from 5708 to 5714\n",
      "Complement found from 6468 to 6474\n",
      "Complement found from 6831 to 6837\n",
      "Complement found from 7670 to 7676\n",
      "Complement found from 8650 to 8656\n",
      "Complement found from 8671 to 8677\n",
      "Complement found from 8749 to 8755\n",
      "Complement found from 10080 to 10086\n",
      "Complement found from 10338 to 10344\n",
      "Complement found from 10453 to 10459\n",
      "Complement found from 11102 to 11108\n",
      "Complement found from 12677 to 12683\n",
      "Complement found from 14137 to 14143\n",
      "Complement found from 14217 to 14223\n",
      "Complement found from 14230 to 14236\n",
      "Seq is: TAATCA\n",
      "17 TAATCA motifs detected in file\n",
      "\n",
      "Complement found from 9133 to 9139\n",
      "Complement found from 11605 to 11611\n",
      "Complement found from 13234 to 13240\n",
      "Seq is: AAATCG\n",
      "3 AAATCG motifs detected in file\n",
      "\n",
      "Complement found from 824 to 830\n",
      "Complement found from 3785 to 3791\n",
      "Complement found from 4102 to 4108\n",
      "Complement found from 5332 to 5338\n",
      "Complement found from 8316 to 8322\n",
      "Complement found from 8680 to 8686\n",
      "Complement found from 8771 to 8777\n",
      "Complement found from 9077 to 9083\n",
      "Complement found from 10303 to 10309\n",
      "Complement found from 10741 to 10747\n",
      "Complement found from 11853 to 11859\n",
      "Complement found from 11877 to 11883\n",
      "Complement found from 13999 to 14005\n",
      "Complement found from 14145 to 14151\n",
      "Complement found from 14524 to 14530\n",
      "Complement found from 15749 to 15755\n",
      "Seq is: TAACCT\n",
      "16 TAACCT motifs detected in file\n",
      "\n",
      "Complement found from 4665 to 4671\n",
      "Complement found from 5202 to 5208\n",
      "Complement found from 9888 to 9894\n",
      "Complement found from 11664 to 11670\n",
      "Complement found from 12457 to 12463\n",
      "Complement found from 13373 to 13379\n",
      "Complement found from 14816 to 14822\n",
      "Seq is: CATCCA\n",
      "7 CATCCA motifs detected in file\n",
      "\n",
      "Complement found from 3083 to 3089\n",
      "Complement found from 4360 to 4366\n",
      "Complement found from 7135 to 7141\n",
      "Complement found from 9149 to 9155\n",
      "Complement found from 10155 to 10161\n",
      "Complement found from 12445 to 12451\n",
      "Complement found from 12954 to 12960\n",
      "Complement found from 13151 to 13157\n",
      "Complement found from 16169 to 16175\n",
      "Seq is: AATCCA\n",
      "9 AATCCA motifs detected in file\n",
      "\n",
      "Complement found from 6709 to 6715\n",
      "Complement found from 7429 to 7435\n",
      "Complement found from 14664 to 14670\n",
      "Complement found from 15207 to 15213\n",
      "Seq is: ATACAT\n",
      "4 ATACAT motifs detected in file\n",
      "\n",
      "Complement found from 754 to 760\n",
      "Complement found from 2518 to 2524\n",
      "Complement found from 3669 to 3675\n",
      "Complement found from 5609 to 5615\n",
      "Complement found from 6516 to 6522\n",
      "Complement found from 9564 to 9570\n",
      "Complement found from 9620 to 9626\n",
      "Complement found from 10862 to 10868\n",
      "Complement found from 11777 to 11783\n",
      "Complement found from 13288 to 13294\n",
      "Complement found from 13927 to 13933\n",
      "Seq is: GCATCA\n",
      "11 GCATCA motifs detected in file\n",
      "\n",
      "Complement found from 6459 to 6465\n",
      "Complement found from 15590 to 15596\n",
      "Seq is: GATCCG\n",
      "2 GATCCG motifs detected in file\n",
      "\n",
      "Complement found from 1502 to 1508\n",
      "Complement found from 7427 to 7433\n",
      "Complement found from 8155 to 8161\n",
      "Seq is: GTATAC\n",
      "3 GTATAC motifs detected in file\n",
      "\n",
      "Complement found from 489 to 495\n",
      "Complement found from 1709 to 1715\n",
      "Complement found from 3443 to 3449\n",
      "Complement found from 5055 to 5061\n",
      "Complement found from 8641 to 8647\n",
      "Complement found from 9666 to 9672\n",
      "Complement found from 10837 to 10843\n",
      "Complement found from 10898 to 10904\n",
      "Complement found from 10942 to 10948\n",
      "Complement found from 12543 to 12549\n",
      "Complement found from 12552 to 12558\n",
      "Complement found from 12739 to 12745\n",
      "Complement found from 12856 to 12862\n",
      "Complement found from 13378 to 13384\n",
      "Complement found from 14329 to 14335\n",
      "Complement found from 14476 to 14482\n",
      "Complement found from 14689 to 14695\n",
      "Complement found from 15362 to 15368\n",
      "Complement found from 15765 to 15771\n",
      "Complement found from 16077 to 16083\n",
      "Seq is: ACAACC\n",
      "20 ACAACC motifs detected in file\n",
      "\n",
      "Complement found from 4100 to 4106\n",
      "Complement found from 4597 to 4603\n",
      "Complement found from 5390 to 5396\n",
      "Complement found from 7168 to 7174\n",
      "Complement found from 7285 to 7291\n",
      "Complement found from 12244 to 12250\n",
      "Complement found from 13160 to 13166\n",
      "Complement found from 13829 to 13835\n",
      "Complement found from 15747 to 15753\n",
      "Seq is: TCTAAC\n",
      "9 TCTAAC motifs detected in file\n",
      "\n",
      "Complement found from 3655 to 3661\n",
      "Complement found from 3820 to 3826\n",
      "Complement found from 4271 to 4277\n",
      "Complement found from 6456 to 6462\n",
      "Complement found from 6606 to 6612\n",
      "Seq is: TCTGAT\n",
      "5 TCTGAT motifs detected in file\n",
      "\n",
      "Complement found from 701 to 707\n",
      "Complement found from 3003 to 3009\n",
      "Complement found from 4350 to 4356\n",
      "Complement found from 5299 to 5305\n",
      "Complement found from 7820 to 7826\n",
      "Complement found from 8840 to 8846\n",
      "Complement found from 10326 to 10332\n",
      "Complement found from 10866 to 10872\n",
      "Complement found from 12602 to 12608\n",
      "Complement found from 13759 to 13765\n",
      "Complement found from 15201 to 15207\n",
      "Seq is: CATCCC\n",
      "11 CATCCC motifs detected in file\n",
      "\n",
      "Complement found from 6459 to 6465\n",
      "Complement found from 15590 to 15596\n",
      "Seq is: GATCCG\n",
      "2 GATCCG motifs detected in file\n",
      "\n",
      "Complement found from 1346 to 1352\n",
      "Complement found from 6953 to 6959\n",
      "Seq is: GGTGGC\n",
      "2 GGTGGC motifs detected in file\n",
      "\n",
      "Complement found from 1594 to 1600\n",
      "Complement found from 8719 to 8725\n",
      "Seq is: GACGAA\n",
      "2 GACGAA motifs detected in file\n",
      "\n",
      "Complement found from 10200 to 10206\n",
      "Complement found from 15614 to 15620\n",
      "Seq is: GCGTCC\n",
      "2 GCGTCC motifs detected in file\n",
      "\n",
      "Complement found from 3639 to 3645\n",
      "Seq is: GCCGTT\n",
      "1 GCCGTT motifs detected in file\n",
      "\n",
      "Complement found from 3655 to 3661\n",
      "Complement found from 3820 to 3826\n",
      "Complement found from 4271 to 4277\n",
      "Complement found from 6456 to 6462\n",
      "Complement found from 6606 to 6612\n",
      "Seq is: TCTGAT\n",
      "5 TCTGAT motifs detected in file\n",
      "\n",
      "Complement found from 536 to 542\n",
      "Complement found from 3305 to 3311\n",
      "Complement found from 3575 to 3581\n",
      "Complement found from 4132 to 4138\n",
      "Complement found from 4223 to 4229\n",
      "Complement found from 4431 to 4437\n",
      "Complement found from 9402 to 9408\n",
      "Complement found from 12427 to 12433\n",
      "Complement found from 13430 to 13436\n",
      "Seq is: CATACC\n",
      "9 CATACC motifs detected in file\n",
      "\n",
      "Complement found from 6068 to 6074\n",
      "Seq is: GTTATC\n",
      "1 GTTATC motifs detected in file\n",
      "\n",
      "Seq is: CGTGTT\n",
      "0 CGTGTT motifs detected in file\n",
      "\n",
      "Complement found from 1807 to 1813\n",
      "Complement found from 4724 to 4730\n",
      "Complement found from 5066 to 5072\n",
      "Complement found from 6428 to 6434\n",
      "Complement found from 8697 to 8703\n",
      "Complement found from 10607 to 10613\n",
      "Complement found from 10740 to 10746\n",
      "Complement found from 12336 to 12342\n",
      "Complement found from 12354 to 12360\n",
      "Complement found from 13672 to 13678\n",
      "Complement found from 14144 to 14150\n",
      "Complement found from 14523 to 14529\n",
      "Seq is: ATAACC\n",
      "12 ATAACC motifs detected in file\n",
      "\n",
      "Complement found from 13915 to 13921\n",
      "Complement found from 15275 to 15281\n",
      "Complement found from 15999 to 16005\n",
      "Seq is: GATTCT\n",
      "3 GATTCT motifs detected in file\n",
      "\n",
      "Complement found from 2412 to 2418\n",
      "Complement found from 4012 to 4018\n",
      "Complement found from 4250 to 4256\n",
      "Complement found from 4907 to 4913\n",
      "Complement found from 5549 to 5555\n",
      "Complement found from 7664 to 7670\n",
      "Complement found from 8473 to 8479\n",
      "Complement found from 10473 to 10479\n",
      "Complement found from 10611 to 10617\n",
      "Complement found from 10979 to 10985\n",
      "Complement found from 12040 to 12046\n",
      "Complement found from 12062 to 12068\n",
      "Complement found from 12101 to 12107\n",
      "Complement found from 13035 to 13041\n",
      "Complement found from 13450 to 13456\n",
      "Complement found from 15266 to 15272\n",
      "Complement found from 16220 to 16226\n",
      "Complement found from 16259 to 16265\n",
      "Complement found from 16377 to 16383\n",
      "Seq is: CCCTCA\n",
      "19 CCCTCA motifs detected in file\n",
      "\n",
      "Complement found from 570 to 576\n",
      "Complement found from 2056 to 2062\n",
      "Complement found from 2769 to 2775\n",
      "Complement found from 3203 to 3209\n",
      "Complement found from 5022 to 5028\n",
      "Complement found from 5447 to 5453\n",
      "Complement found from 5676 to 5682\n",
      "Complement found from 7180 to 7186\n",
      "Complement found from 8080 to 8086\n",
      "Complement found from 8559 to 8565\n",
      "Complement found from 10844 to 10850\n",
      "Complement found from 12483 to 12489\n",
      "Complement found from 12926 to 12932\n",
      "Complement found from 14357 to 14363\n",
      "Complement found from 14620 to 14626\n",
      "Complement found from 14641 to 14647\n",
      "Complement found from 14945 to 14951\n",
      "Complement found from 15542 to 15548\n",
      "Complement found from 16535 to 16541\n",
      "Seq is: CCCACA\n",
      "19 CCCACA motifs detected in file\n",
      "\n",
      "Complement found from 463 to 469\n",
      "Complement found from 5380 to 5386\n",
      "Complement found from 6309 to 6315\n",
      "Complement found from 8013 to 8019\n",
      "Complement found from 10623 to 10629\n",
      "Complement found from 11403 to 11409\n",
      "Complement found from 11811 to 11817\n",
      "Complement found from 11975 to 11981\n",
      "Complement found from 13031 to 13037\n",
      "Seq is: ACTCCC\n",
      "9 ACTCCC motifs detected in file\n",
      "\n",
      "Complement found from 4730 to 4736\n",
      "Complement found from 4743 to 4749\n",
      "Complement found from 5564 to 5570\n",
      "Complement found from 7738 to 7744\n",
      "Complement found from 8377 to 8383\n",
      "Complement found from 10522 to 10528\n",
      "Complement found from 10576 to 10582\n",
      "Complement found from 10956 to 10962\n",
      "Complement found from 14578 to 14584\n",
      "Complement found from 15870 to 15876\n",
      "Complement found from 16137 to 16143\n",
      "Seq is: AATACT\n",
      "11 AATACT motifs detected in file\n",
      "\n",
      "Complement found from 144 to 150\n",
      "Complement found from 480 to 486\n",
      "Complement found from 4520 to 4526\n",
      "Complement found from 4747 to 4753\n",
      "Complement found from 5294 to 5300\n",
      "Complement found from 5453 to 5459\n",
      "Complement found from 6982 to 6988\n",
      "Complement found from 7807 to 7813\n",
      "Complement found from 8428 to 8434\n",
      "Complement found from 8634 to 8640\n",
      "Complement found from 11239 to 11245\n",
      "Complement found from 11662 to 11668\n",
      "Complement found from 12699 to 12705\n",
      "Complement found from 12795 to 12801\n",
      "Complement found from 13113 to 13119\n",
      "Complement found from 13572 to 13578\n",
      "Complement found from 14118 to 14124\n",
      "Complement found from 15640 to 15646\n",
      "Seq is: CTCATC\n",
      "18 CTCATC motifs detected in file\n",
      "\n",
      "Complement found from 13398 to 13404\n",
      "Complement found from 15560 to 15566\n",
      "Seq is: GATATT\n",
      "2 GATATT motifs detected in file\n",
      "\n",
      "Complement found from 2603 to 2609\n",
      "Seq is: ACTTGT\n",
      "1 ACTTGT motifs detected in file\n",
      "\n",
      "Complement found from 3020 to 3026\n",
      "Complement found from 6838 to 6844\n",
      "Complement found from 13178 to 13184\n",
      "Complement found from 16082 to 16088\n",
      "Seq is: CGCTAT\n",
      "4 CGCTAT motifs detected in file\n",
      "\n",
      "Complement found from 230 to 236\n",
      "Complement found from 1816 to 1822\n",
      "Complement found from 1851 to 1857\n",
      "Complement found from 2596 to 2602\n",
      "Complement found from 3851 to 3857\n",
      "Complement found from 4669 to 4675\n",
      "Complement found from 4762 to 4768\n",
      "Complement found from 6121 to 6127\n",
      "Complement found from 6829 to 6835\n",
      "Complement found from 7668 to 7674\n",
      "Complement found from 8397 to 8403\n",
      "Complement found from 9202 to 9208\n",
      "Complement found from 10821 to 10827\n",
      "Complement found from 11562 to 11568\n",
      "Complement found from 11703 to 11709\n",
      "Complement found from 11781 to 11787\n",
      "Complement found from 12593 to 12599\n",
      "Complement found from 13271 to 13277\n",
      "Complement found from 14081 to 14087\n",
      "Complement found from 14228 to 14234\n",
      "Complement found from 15687 to 15693\n",
      "Seq is: CATAAT\n",
      "21 CATAAT motifs detected in file\n",
      "\n",
      "Complement found from 745 to 751\n",
      "Complement found from 2445 to 2451\n",
      "Complement found from 2467 to 2473\n",
      "Complement found from 7450 to 7456\n",
      "Complement found from 10569 to 10575\n",
      "Seq is: AAGGAA\n",
      "5 AAGGAA motifs detected in file\n",
      "\n",
      "Complement found from 6459 to 6465\n",
      "Complement found from 15590 to 15596\n",
      "Seq is: GATCCG\n",
      "2 GATCCG motifs detected in file\n",
      "\n",
      "Complement found from 3180 to 3186\n",
      "Complement found from 7147 to 7153\n",
      "Complement found from 7260 to 7266\n",
      "Complement found from 7629 to 7635\n",
      "Complement found from 9095 to 9101\n",
      "Complement found from 9227 to 9233\n",
      "Complement found from 11151 to 11157\n",
      "Complement found from 13537 to 13543\n",
      "Complement found from 15158 to 15164\n",
      "Seq is: TATCAT\n",
      "9 TATCAT motifs detected in file\n",
      "\n",
      "Complement found from 306 to 312\n",
      "Complement found from 456 to 462\n",
      "Complement found from 957 to 963\n",
      "Complement found from 1492 to 1498\n",
      "Complement found from 3568 to 3574\n",
      "Complement found from 5208 to 5214\n",
      "Complement found from 5325 to 5331\n",
      "Complement found from 6289 to 6295\n",
      "Complement found from 7364 to 7370\n",
      "Complement found from 7814 to 7820\n",
      "Complement found from 7861 to 7867\n",
      "Complement found from 9276 to 9282\n",
      "Complement found from 9726 to 9732\n",
      "Complement found from 10089 to 10095\n",
      "Complement found from 10266 to 10272\n",
      "Complement found from 10949 to 10955\n",
      "Complement found from 15537 to 15543\n",
      "Complement found from 16185 to 16191\n",
      "Seq is: CCCTCC\n",
      "18 CCCTCC motifs detected in file\n",
      "\n",
      "Complement found from 858 to 864\n",
      "Complement found from 1503 to 1509\n",
      "Complement found from 2884 to 2890\n",
      "Complement found from 4702 to 4708\n",
      "Complement found from 5494 to 5500\n",
      "Complement found from 6523 to 6529\n",
      "Complement found from 8156 to 8162\n",
      "Complement found from 8735 to 8741\n",
      "Complement found from 10492 to 10498\n",
      "Complement found from 11049 to 11055\n",
      "Complement found from 11972 to 11978\n",
      "Complement found from 15815 to 15821\n",
      "Seq is: TATACT\n",
      "12 TATACT motifs detected in file\n",
      "\n",
      "Complement found from 267 to 273\n",
      "Complement found from 796 to 802\n",
      "Complement found from 3204 to 3210\n",
      "Complement found from 3868 to 3874\n",
      "Complement found from 5448 to 5454\n",
      "Complement found from 6876 to 6882\n",
      "Complement found from 7408 to 7414\n",
      "Complement found from 9415 to 9421\n",
      "Complement found from 11129 to 11135\n",
      "Complement found from 13299 to 13305\n",
      "Complement found from 14561 to 14567\n",
      "Complement found from 14642 to 14648\n",
      "Complement found from 16536 to 16542\n",
      "Seq is: CCACAC\n",
      "13 CCACAC motifs detected in file\n",
      "\n",
      "Complement found from 6837 to 6843\n",
      "Complement found from 11849 to 11855\n",
      "Complement found from 13576 to 13582\n",
      "Complement found from 14381 to 14387\n",
      "Seq is: TCGCTA\n",
      "4 TCGCTA motifs detected in file\n",
      "\n",
      "Complement found from 3856 to 3862\n",
      "Complement found from 4210 to 4216\n",
      "Complement found from 6730 to 6736\n",
      "Complement found from 10448 to 10454\n",
      "Seq is: TATGAT\n",
      "4 TATGAT motifs detected in file\n",
      "\n",
      "Complement found from 2333 to 2339\n",
      "Complement found from 2442 to 2448\n",
      "Complement found from 6199 to 6205\n",
      "Seq is: CATAAG\n",
      "3 CATAAG motifs detected in file\n",
      "\n",
      "Seq is: GATCTT\n",
      "0 GATCTT motifs detected in file\n",
      "\n",
      "Complement found from 4051 to 4057\n",
      "Complement found from 4933 to 4939\n",
      "Complement found from 5603 to 5609\n",
      "Complement found from 11295 to 11301\n",
      "Complement found from 11931 to 11937\n",
      "Complement found from 13187 to 13193\n",
      "Seq is: CACTCT\n",
      "6 CACTCT motifs detected in file\n",
      "\n",
      "Complement found from 1723 to 1729\n",
      "Complement found from 5069 to 5075\n",
      "Complement found from 6700 to 6706\n",
      "Complement found from 7531 to 7537\n",
      "Complement found from 9072 to 9078\n",
      "Complement found from 9764 to 9770\n",
      "Complement found from 13455 to 13461\n",
      "Complement found from 15978 to 15984\n",
      "Seq is: ACCATT\n",
      "8 ACCATT motifs detected in file\n",
      "\n",
      "Complement found from 1385 to 1391\n",
      "Seq is: TACGAT\n",
      "1 TACGAT motifs detected in file\n",
      "\n",
      "Complement found from 4075 to 4081\n",
      "Complement found from 6668 to 6674\n",
      "Complement found from 6810 to 6816\n",
      "Complement found from 7150 to 7156\n",
      "Complement found from 10457 to 10463\n",
      "Complement found from 11106 to 11112\n",
      "Complement found from 11985 to 11991\n",
      "Seq is: CATATT\n",
      "7 CATATT motifs detected in file\n",
      "\n",
      "Complement found from 10001 to 10007\n",
      "Complement found from 10484 to 10490\n",
      "Complement found from 14292 to 14298\n",
      "Complement found from 14496 to 14502\n",
      "Complement found from 14592 to 14598\n",
      "Complement found from 16134 to 16140\n",
      "Seq is: ATAAAT\n",
      "6 ATAAAT motifs detected in file\n",
      "\n",
      "Complement found from 1364 to 1370\n",
      "Complement found from 1516 to 1522\n",
      "Complement found from 9791 to 9797\n",
      "Complement found from 10118 to 10124\n",
      "Complement found from 13747 to 13753\n",
      "Complement found from 13890 to 13896\n",
      "Seq is: ACATTT\n",
      "6 ACATTT motifs detected in file\n",
      "\n",
      "Complement found from 463 to 469\n",
      "Complement found from 5380 to 5386\n",
      "Complement found from 6309 to 6315\n",
      "Complement found from 8013 to 8019\n",
      "Complement found from 10623 to 10629\n",
      "Complement found from 11403 to 11409\n",
      "Complement found from 11811 to 11817\n",
      "Complement found from 11975 to 11981\n",
      "Complement found from 13031 to 13037\n",
      "Seq is: ACTCCC\n",
      "9 ACTCCC motifs detected in file\n",
      "\n",
      "Complement found from 934 to 940\n",
      "Complement found from 1313 to 1319\n",
      "Complement found from 3171 to 3177\n",
      "Complement found from 5398 to 5404\n",
      "Complement found from 7162 to 7168\n",
      "Complement found from 14961 to 14967\n",
      "Seq is: CGTAAA\n",
      "6 CGTAAA motifs detected in file\n",
      "\n",
      "Complement found from 1724 to 1730\n",
      "Complement found from 5881 to 5887\n",
      "Complement found from 6701 to 6707\n",
      "Complement found from 7138 to 7144\n",
      "Complement found from 7532 to 7538\n",
      "Complement found from 9765 to 9771\n",
      "Complement found from 16319 to 16325\n",
      "Seq is: CCATTT\n",
      "7 CCATTT motifs detected in file\n",
      "\n",
      "Complement found from 469 to 475\n",
      "Complement found from 859 to 865\n",
      "Complement found from 4731 to 4737\n",
      "Complement found from 5495 to 5501\n",
      "Complement found from 6524 to 6530\n",
      "Complement found from 7739 to 7745\n",
      "Complement found from 8157 to 8163\n",
      "Complement found from 8378 to 8384\n",
      "Complement found from 8736 to 8742\n",
      "Complement found from 8943 to 8949\n",
      "Complement found from 9335 to 9341\n",
      "Complement found from 10493 to 10499\n",
      "Complement found from 10523 to 10529\n",
      "Complement found from 10577 to 10583\n",
      "Complement found from 10655 to 10661\n",
      "Complement found from 10957 to 10963\n",
      "Complement found from 11050 to 11056\n",
      "Complement found from 11956 to 11962\n",
      "Complement found from 12717 to 12723\n",
      "Complement found from 13347 to 13353\n",
      "Complement found from 14579 to 14585\n",
      "Seq is: ATACTA\n",
      "21 ATACTA motifs detected in file\n",
      "\n",
      "Complement found from 1778 to 1784\n",
      "Complement found from 2166 to 2172\n",
      "Complement found from 6107 to 6113\n",
      "Complement found from 7352 to 7358\n",
      "Complement found from 9233 to 9239\n",
      "Complement found from 10005 to 10011\n",
      "Complement found from 11368 to 11374\n",
      "Complement found from 11443 to 11449\n",
      "Complement found from 16306 to 16312\n",
      "Seq is: ATAGTA\n",
      "9 ATAGTA motifs detected in file\n",
      "\n",
      "Complement found from 1113 to 1119\n",
      "Complement found from 2047 to 2053\n",
      "Complement found from 2612 to 2618\n",
      "Complement found from 5522 to 5528\n",
      "Complement found from 7554 to 7560\n",
      "Complement found from 10442 to 10448\n",
      "Complement found from 16549 to 16555\n",
      "Seq is: TTAAAT\n",
      "7 TTAAAT motifs detected in file\n",
      "\n",
      "Complement found from 2886 to 2892\n",
      "Complement found from 3645 to 3651\n",
      "Complement found from 4745 to 4751\n",
      "Complement found from 5121 to 5127\n",
      "Complement found from 8971 to 8977\n",
      "Complement found from 9031 to 9037\n",
      "Complement found from 11237 to 11243\n",
      "Complement found from 11291 to 11297\n",
      "Complement found from 12697 to 12703\n",
      "Complement found from 13111 to 13117\n",
      "Complement found from 13420 to 13426\n",
      "Complement found from 14908 to 14914\n",
      "Complement found from 15070 to 15076\n",
      "Complement found from 15247 to 15253\n",
      "Complement found from 15872 to 15878\n",
      "Seq is: TACTCA\n",
      "15 TACTCA motifs detected in file\n",
      "\n",
      "Complement found from 1030 to 1036\n",
      "Complement found from 2734 to 2740\n",
      "Seq is: GCTTTA\n",
      "2 GCTTTA motifs detected in file\n",
      "\n",
      "Complement found from 1296 to 1302\n",
      "Complement found from 4919 to 4925\n",
      "Complement found from 9177 to 9183\n",
      "Complement found from 15772 to 15778\n",
      "Seq is: GTAAGC\n",
      "4 GTAAGC motifs detected in file\n",
      "\n",
      "Complement found from 357 to 363\n",
      "Complement found from 2317 to 2323\n",
      "Complement found from 2352 to 2358\n",
      "Complement found from 2503 to 2509\n",
      "Complement found from 5283 to 5289\n",
      "Complement found from 8095 to 8101\n",
      "Complement found from 8687 to 8693\n",
      "Complement found from 9663 to 9669\n",
      "Complement found from 10815 to 10821\n",
      "Complement found from 11520 to 11526\n",
      "Complement found from 12056 to 12062\n",
      "Complement found from 12154 to 12160\n",
      "Complement found from 14025 to 14031\n",
      "Complement found from 14395 to 14401\n",
      "Complement found from 15863 to 15869\n",
      "Seq is: AAAACA\n",
      "15 AAAACA motifs detected in file\n",
      "\n",
      "Complement found from 2050 to 2056\n",
      "Complement found from 5791 to 5797\n",
      "Complement found from 10824 to 10830\n",
      "Seq is: AATTTG\n",
      "3 AATTTG motifs detected in file\n",
      "\n",
      "Complement found from 6774 to 6780\n",
      "Complement found from 7575 to 7581\n",
      "Complement found from 12653 to 12659\n",
      "Complement found from 14176 to 14182\n",
      "Complement found from 15668 to 15674\n",
      "Seq is: ATATAT\n",
      "5 ATATAT motifs detected in file\n",
      "\n",
      "Complement found from 2452 to 2458\n",
      "Complement found from 5520 to 5526\n",
      "Seq is: GGTTAA\n",
      "2 GGTTAA motifs detected in file\n",
      "\n",
      "Seq is: GCCAAG\n",
      "0 GCCAAG motifs detected in file\n",
      "\n",
      "Complement found from 2345 to 2351\n",
      "Complement found from 12160 to 12166\n",
      "Complement found from 16380 to 16386\n",
      "Seq is: TCAGAT\n",
      "3 TCAGAT motifs detected in file\n",
      "\n",
      "Complement found from 2125 to 2131\n",
      "Complement found from 4033 to 4039\n",
      "Complement found from 4785 to 4791\n",
      "Complement found from 6786 to 6792\n",
      "Complement found from 10518 to 10524\n",
      "Complement found from 12775 to 12781\n",
      "Complement found from 15371 to 15377\n",
      "Seq is: TAGGAA\n",
      "7 TAGGAA motifs detected in file\n",
      "\n",
      "Complement found from 2236 to 2242\n",
      "Complement found from 2318 to 2324\n",
      "Complement found from 2504 to 2510\n",
      "Complement found from 3992 to 3998\n",
      "Complement found from 4573 to 4579\n",
      "Complement found from 7252 to 7258\n",
      "Complement found from 9893 to 9899\n",
      "Complement found from 11280 to 11286\n",
      "Complement found from 12155 to 12161\n",
      "Complement found from 12671 to 12677\n",
      "Complement found from 13532 to 13538\n",
      "Complement found from 15084 to 15090\n",
      "Seq is: AAACAT\n",
      "12 AAACAT motifs detected in file\n",
      "\n",
      "Complement found from 712 to 718\n",
      "Complement found from 817 to 823\n",
      "Complement found from 2548 to 2554\n",
      "Complement found from 2670 to 2676\n",
      "Complement found from 8358 to 8364\n",
      "Complement found from 8860 to 8866\n",
      "Complement found from 11013 to 11019\n",
      "Seq is: CAGTGA\n",
      "7 CAGTGA motifs detected in file\n",
      "\n",
      "Complement found from 8742 to 8748\n",
      "Complement found from 16489 to 16495\n",
      "Seq is: GTATCC\n",
      "2 GTATCC motifs detected in file\n",
      "\n",
      "Complement found from 6721 to 6727\n",
      "Seq is: GGTCTG\n",
      "1 GGTCTG motifs detected in file\n",
      "\n",
      "Complement found from 2072 to 2078\n",
      "Complement found from 2230 to 2236\n",
      "Complement found from 4472 to 4478\n",
      "Complement found from 9578 to 9584\n",
      "Complement found from 13778 to 13784\n",
      "Complement found from 13876 to 13882\n",
      "Complement found from 13943 to 13949\n",
      "Complement found from 15654 to 15660\n",
      "Complement found from 16349 to 16355\n",
      "Seq is: AATCCC\n",
      "9 AATCCC motifs detected in file\n",
      "\n",
      "Complement found from 14258 to 14264\n",
      "Complement found from 14868 to 14874\n",
      "Seq is: GATCCT\n",
      "2 GATCCT motifs detected in file\n",
      "\n",
      "Complement found from 471 to 477\n",
      "Complement found from 2879 to 2885\n",
      "Complement found from 5102 to 5108\n",
      "Complement found from 5119 to 5125\n",
      "Complement found from 5152 to 5158\n",
      "Complement found from 5377 to 5383\n",
      "Complement found from 6306 to 6312\n",
      "Complement found from 6526 to 6532\n",
      "Complement found from 6681 to 6687\n",
      "Complement found from 10102 to 10108\n",
      "Complement found from 11289 to 11295\n",
      "Complement found from 12348 to 12354\n",
      "Complement found from 12583 to 12589\n",
      "Complement found from 12971 to 12977\n",
      "Complement found from 13418 to 13424\n",
      "Complement found from 14209 to 14215\n",
      "Complement found from 14906 to 14912\n",
      "Seq is: ACTACT\n",
      "17 ACTACT motifs detected in file\n",
      "\n",
      "Complement found from 1803 to 1809\n",
      "Complement found from 6403 to 6409\n",
      "Complement found from 7557 to 7563\n",
      "Complement found from 8500 to 8506\n",
      "Complement found from 10111 to 10117\n",
      "Complement found from 10445 to 10451\n",
      "Complement found from 10782 to 10788\n",
      "Complement found from 10854 to 10860\n",
      "Complement found from 11076 to 11082\n",
      "Complement found from 11565 to 11571\n",
      "Complement found from 12779 to 12785\n",
      "Complement found from 14295 to 14301\n",
      "Complement found from 14965 to 14971\n",
      "Complement found from 15508 to 15514\n",
      "Seq is: AATTAT\n",
      "14 AATTAT motifs detected in file\n",
      "\n",
      "Complement found from 4054 to 4060\n",
      "Complement found from 4904 to 4910\n",
      "Complement found from 5214 to 5220\n",
      "Complement found from 5484 to 5490\n",
      "Complement found from 5731 to 5737\n",
      "Complement found from 6496 to 6502\n",
      "Complement found from 6663 to 6669\n",
      "Complement found from 9755 to 9761\n",
      "Complement found from 11057 to 11063\n",
      "Complement found from 12562 to 12568\n",
      "Complement found from 15851 to 15857\n",
      "Seq is: TCTCCC\n",
      "11 TCTCCC motifs detected in file\n",
      "\n",
      "Seq is: TACTGT\n",
      "0 TACTGT motifs detected in file\n",
      "\n",
      "Complement found from 2892 to 2898\n",
      "Complement found from 8612 to 8618\n",
      "Complement found from 9975 to 9981\n",
      "Seq is: ATTGAT\n",
      "3 ATTGAT motifs detected in file\n",
      "\n",
      "Complement found from 730 to 736\n",
      "Complement found from 2599 to 2605\n",
      "Complement found from 5371 to 5377\n",
      "Complement found from 5811 to 5817\n",
      "Complement found from 6469 to 6475\n",
      "Complement found from 8651 to 8657\n",
      "Complement found from 9216 to 9222\n",
      "Complement found from 9634 to 9640\n",
      "Complement found from 14138 to 14144\n",
      "Complement found from 14877 to 14883\n",
      "Complement found from 15375 to 15381\n",
      "Complement found from 15396 to 15402\n",
      "Complement found from 15710 to 15716\n",
      "Seq is: AATCAC\n",
      "13 AATCAC motifs detected in file\n",
      "\n",
      "Seq is: CGTCCA\n",
      "0 CGTCCA motifs detected in file\n",
      "\n",
      "Complement found from 1117 to 1123\n",
      "Complement found from 1227 to 1233\n",
      "Complement found from 2390 to 2396\n",
      "Complement found from 2964 to 2970\n",
      "Complement found from 3758 to 3764\n",
      "Complement found from 5611 to 5617\n",
      "Complement found from 5710 to 5716\n",
      "Complement found from 8637 to 8643\n",
      "Complement found from 9068 to 9074\n",
      "Complement found from 10082 to 10088\n",
      "Complement found from 10831 to 10837\n",
      "Complement found from 10891 to 10897\n",
      "Complement found from 13290 to 13296\n",
      "Complement found from 14219 to 14225\n",
      "Complement found from 14269 to 14275\n",
      "Complement found from 16073 to 16079\n",
      "Complement found from 16215 to 16221\n",
      "Complement found from 16236 to 16242\n",
      "Seq is: ATCAAC\n",
      "18 ATCAAC motifs detected in file\n",
      "\n",
      "Complement found from 2236 to 2242\n",
      "Complement found from 2318 to 2324\n",
      "Complement found from 2504 to 2510\n",
      "Complement found from 3992 to 3998\n",
      "Complement found from 4573 to 4579\n",
      "Complement found from 7252 to 7258\n",
      "Complement found from 9893 to 9899\n",
      "Complement found from 11280 to 11286\n",
      "Complement found from 12155 to 12161\n",
      "Complement found from 12671 to 12677\n",
      "Complement found from 13532 to 13538\n",
      "Complement found from 15084 to 15090\n",
      "Seq is: AAACAT\n",
      "12 AAACAT motifs detected in file\n",
      "\n",
      "Complement found from 12814 to 12820\n",
      "Complement found from 14156 to 14162\n",
      "Seq is: CCCGAG\n",
      "2 CCCGAG motifs detected in file\n",
      "\n",
      "Complement found from 3503 to 3509\n",
      "Complement found from 4501 to 4507\n",
      "Complement found from 6826 to 6832\n",
      "Complement found from 7869 to 7875\n",
      "Complement found from 10281 to 10287\n",
      "Complement found from 10504 to 10510\n",
      "Complement found from 12713 to 12719\n",
      "Complement found from 15786 to 15792\n",
      "Complement found from 16130 to 16136\n",
      "Seq is: TACCAT\n",
      "9 TACCAT motifs detected in file\n",
      "\n",
      "Complement found from 533 to 539\n",
      "Complement found from 636 to 642\n",
      "Complement found from 3572 to 3578\n",
      "Complement found from 4390 to 4396\n",
      "Complement found from 4879 to 4885\n",
      "Complement found from 5383 to 5389\n",
      "Complement found from 5436 to 5442\n",
      "Complement found from 6583 to 6589\n",
      "Complement found from 7490 to 7496\n",
      "Complement found from 7955 to 7961\n",
      "Complement found from 8028 to 8034\n",
      "Complement found from 8406 to 8412\n",
      "Complement found from 8939 to 8945\n",
      "Complement found from 11427 to 11433\n",
      "Complement found from 12084 to 12090\n",
      "Complement found from 12237 to 12243\n",
      "Complement found from 12386 to 12392\n",
      "Complement found from 12432 to 12438\n",
      "Complement found from 13691 to 13697\n",
      "Complement found from 14339 to 14345\n",
      "Complement found from 14425 to 14431\n",
      "Complement found from 14588 to 14594\n",
      "Complement found from 14628 to 14634\n",
      "Complement found from 14813 to 14819\n",
      "Complement found from 15657 to 15663\n",
      "Complement found from 16189 to 16195\n",
      "Complement found from 16362 to 16368\n",
      "Seq is: CCCCAT\n",
      "27 CCCCAT motifs detected in file\n",
      "\n",
      "Complement found from 4645 to 4651\n",
      "Complement found from 7677 to 7683\n",
      "Complement found from 7710 to 7716\n",
      "Complement found from 9844 to 9850\n",
      "Complement found from 10925 to 10931\n",
      "Complement found from 12128 to 12134\n",
      "Complement found from 13487 to 13493\n",
      "Complement found from 13817 to 13823\n",
      "Complement found from 15564 to 15570\n",
      "Seq is: TTTCCT\n",
      "9 TTTCCT motifs detected in file\n",
      "\n",
      "Complement found from 705 to 711\n",
      "Seq is: CCCGTT\n",
      "1 CCCGTT motifs detected in file\n",
      "\n",
      "Complement found from 1793 to 1799\n",
      "Complement found from 8330 to 8336\n",
      "Complement found from 11373 to 11379\n",
      "Seq is: AAAGAT\n",
      "3 AAAGAT motifs detected in file\n",
      "\n",
      "Complement found from 6146 to 6152\n",
      "Complement found from 8946 to 8952\n",
      "Complement found from 10029 to 10035\n",
      "Complement found from 15223 to 15229\n",
      "Seq is: CTAGTT\n",
      "4 CTAGTT motifs detected in file\n",
      "\n",
      "Complement found from 151 to 157\n",
      "Complement found from 3196 to 3202\n",
      "Complement found from 3998 to 4004\n",
      "Complement found from 5092 to 5098\n",
      "Complement found from 5961 to 5967\n",
      "Complement found from 8951 to 8957\n",
      "Complement found from 10245 to 10251\n",
      "Complement found from 10489 to 10495\n",
      "Complement found from 12467 to 12473\n",
      "Complement found from 12518 to 12524\n",
      "Seq is: TATTAT\n",
      "10 TATTAT motifs detected in file\n",
      "\n",
      "Complement found from 4027 to 4033\n",
      "Complement found from 7176 to 7182\n",
      "Complement found from 12478 to 12484\n",
      "Complement found from 12703 to 12709\n",
      "Complement found from 14110 to 14116\n",
      "Complement found from 15026 to 15032\n",
      "Complement found from 15446 to 15452\n",
      "Seq is: TCTTCC\n",
      "7 TCTTCC motifs detected in file\n",
      "\n",
      "Complement found from 5093 to 5099\n",
      "Complement found from 5264 to 5270\n",
      "Complement found from 6404 to 6410\n",
      "Complement found from 8952 to 8958\n",
      "Complement found from 12468 to 12474\n",
      "Complement found from 12519 to 12525\n",
      "Complement found from 15094 to 15100\n",
      "Seq is: ATTATC\n",
      "7 ATTATC motifs detected in file\n",
      "\n",
      "Complement found from 12805 to 12811\n",
      "Seq is: GATGAT\n",
      "1 GATGAT motifs detected in file\n",
      "\n",
      "Complement found from 3293 to 3299\n",
      "Complement found from 3551 to 3557\n",
      "Complement found from 4855 to 4861\n",
      "Complement found from 6099 to 6105\n",
      "Complement found from 6342 to 6348\n",
      "Complement found from 10221 to 10227\n",
      "Complement found from 11117 to 11123\n",
      "Complement found from 13105 to 13111\n",
      "Complement found from 13616 to 13622\n",
      "Complement found from 14107 to 14113\n",
      "Seq is: TCTTCT\n",
      "10 TCTTCT motifs detected in file\n",
      "\n",
      "Complement found from 5406 to 5412\n",
      "Complement found from 7434 to 7440\n",
      "Complement found from 10215 to 10221\n",
      "Complement found from 11603 to 11609\n",
      "Complement found from 12442 to 12448\n",
      "Complement found from 13868 to 13874\n",
      "Complement found from 14777 to 14783\n",
      "Complement found from 15393 to 15399\n",
      "Seq is: TAAAAT\n",
      "8 TAAAAT motifs detected in file\n",
      "\n",
      "Complement found from 1239 to 1245\n",
      "Complement found from 2511 to 2517\n",
      "Complement found from 3627 to 3633\n",
      "Complement found from 3787 to 3793\n",
      "Complement found from 3817 to 3823\n",
      "Complement found from 5334 to 5340\n",
      "Complement found from 8349 to 8355\n",
      "Complement found from 11042 to 11048\n",
      "Complement found from 11379 to 11385\n",
      "Complement found from 13433 to 13439\n",
      "Seq is: ACCTCT\n",
      "10 ACCTCT motifs detected in file\n",
      "\n",
      "Complement found from 9082 to 9088\n",
      "Complement found from 12374 to 12380\n",
      "Seq is: TTCCCT\n",
      "2 TTCCCT motifs detected in file\n",
      "\n",
      "Complement found from 13207 to 13213\n",
      "Seq is: GCGCCC\n",
      "1 GCGCCC motifs detected in file\n",
      "\n",
      "Complement found from 4793 to 4799\n",
      "Complement found from 4877 to 4883\n",
      "Complement found from 5230 to 5236\n",
      "Complement found from 6167 to 6173\n",
      "Complement found from 7395 to 7401\n",
      "Complement found from 8026 to 8032\n",
      "Complement found from 8556 to 8562\n",
      "Complement found from 9553 to 9559\n",
      "Complement found from 11425 to 11431\n",
      "Complement found from 12235 to 12241\n",
      "Complement found from 14242 to 14248\n",
      "Seq is: GCCCCC\n",
      "11 GCCCCC motifs detected in file\n",
      "\n",
      "Complement found from 7576 to 7582\n",
      "Complement found from 10186 to 10192\n",
      "Complement found from 10530 to 10536\n",
      "Complement found from 11113 to 11119\n",
      "Complement found from 12782 to 12788\n",
      "Complement found from 14465 to 14471\n",
      "Complement found from 15669 to 15675\n",
      "Seq is: TATATC\n",
      "7 TATATC motifs detected in file\n",
      "\n",
      "Complement found from 4987 to 4993\n",
      "Complement found from 12557 to 12563\n",
      "Seq is: CCAGCT\n",
      "2 CCAGCT motifs detected in file\n",
      "\n",
      "Complement found from 3694 to 3700\n",
      "Complement found from 5967 to 5973\n",
      "Complement found from 7158 to 7164\n",
      "Complement found from 12865 to 12871\n",
      "Seq is: TCGGCG\n",
      "4 TCGGCG motifs detected in file\n",
      "\n",
      "Complement found from 3666 to 3672\n",
      "Complement found from 6764 to 6770\n",
      "Seq is: TGAGCA\n",
      "2 TGAGCA motifs detected in file\n",
      "\n",
      "Complement found from 453 to 459\n",
      "Complement found from 3165 to 3171\n",
      "Complement found from 4246 to 4252\n",
      "Complement found from 6150 to 6156\n",
      "Complement found from 6183 to 6189\n",
      "Complement found from 7092 to 7098\n",
      "Complement found from 7623 to 7629\n",
      "Complement found from 7952 to 7958\n",
      "Complement found from 8225 to 8231\n",
      "Complement found from 8602 to 8608\n",
      "Complement found from 10914 to 10920\n",
      "Complement found from 11231 to 11237\n",
      "Complement found from 12382 to 12388\n",
      "Complement found from 12480 to 12486\n",
      "Complement found from 13644 to 13650\n",
      "Complement found from 13751 to 13757\n",
      "Complement found from 14152 to 14158\n",
      "Complement found from 14486 to 14492\n",
      "Complement found from 16543 to 16549\n",
      "Seq is: TTCCCC\n",
      "19 TTCCCC motifs detected in file\n",
      "\n",
      "Complement found from 1920 to 1926\n",
      "Complement found from 2222 to 2228\n",
      "Complement found from 2759 to 2765\n",
      "Complement found from 4257 to 4263\n",
      "Complement found from 6333 to 6339\n",
      "Complement found from 9040 to 9046\n",
      "Complement found from 10743 to 10749\n",
      "Complement found from 11600 to 11606\n",
      "Complement found from 13789 to 13795\n",
      "Complement found from 13852 to 13858\n",
      "Complement found from 13996 to 14002\n",
      "Complement found from 14021 to 14027\n",
      "Seq is: ACCTAA\n",
      "12 ACCTAA motifs detected in file\n",
      "\n",
      "Complement found from 1793 to 1799\n",
      "Complement found from 8330 to 8336\n",
      "Complement found from 11373 to 11379\n",
      "Seq is: AAAGAT\n",
      "3 AAAGAT motifs detected in file\n",
      "\n",
      "Complement found from 3940 to 3946\n",
      "Complement found from 6993 to 6999\n",
      "Complement found from 15035 to 15041\n",
      "Complement found from 15086 to 15092\n",
      "Seq is: ACATCG\n",
      "4 ACATCG motifs detected in file\n",
      "\n",
      "Complement found from 6138 to 6144\n",
      "Complement found from 12317 to 12323\n",
      "Complement found from 15109 to 15115\n",
      "Complement found from 16243 to 16249\n",
      "Seq is: GCAACT\n",
      "4 GCAACT motifs detected in file\n",
      "\n",
      "Complement found from 2211 to 2217\n",
      "Complement found from 2427 to 2433\n",
      "Complement found from 3805 to 3811\n",
      "Complement found from 6547 to 6553\n",
      "Complement found from 6596 to 6602\n",
      "Complement found from 7184 to 7190\n",
      "Complement found from 8345 to 8351\n",
      "Complement found from 8708 to 8714\n",
      "Complement found from 9197 to 9203\n",
      "Complement found from 9352 to 9358\n",
      "Complement found from 10084 to 10090\n",
      "Complement found from 10615 to 10621\n",
      "Complement found from 10714 to 10720\n",
      "Complement found from 10833 to 10839\n",
      "Complement found from 11262 to 11268\n",
      "Complement found from 11996 to 12002\n",
      "Complement found from 12827 to 12833\n",
      "Complement found from 15323 to 15329\n",
      "Seq is: CAACAC\n",
      "18 CAACAC motifs detected in file\n",
      "\n",
      "Complement found from 8963 to 8969\n",
      "Complement found from 9621 to 9627\n",
      "Complement found from 12158 to 12164\n",
      "Complement found from 12797 to 12803\n",
      "Seq is: CATCAG\n",
      "4 CATCAG motifs detected in file\n",
      "\n",
      "Complement found from 4549 to 4555\n",
      "Complement found from 5884 to 5890\n",
      "Complement found from 9506 to 9512\n",
      "Complement found from 9705 to 9711\n",
      "Complement found from 9721 to 9727\n",
      "Complement found from 10272 to 10278\n",
      "Complement found from 15783 to 15789\n",
      "Seq is: TTTTAC\n",
      "7 TTTTAC motifs detected in file\n",
      "\n",
      "Complement found from 152 to 158\n",
      "Complement found from 447 to 453\n",
      "Complement found from 2405 to 2411\n",
      "Complement found from 3996 to 4002\n",
      "Complement found from 5962 to 5968\n",
      "Complement found from 7959 to 7965\n",
      "Complement found from 9830 to 9836\n",
      "Complement found from 10112 to 10118\n",
      "Complement found from 10246 to 10252\n",
      "Complement found from 10855 to 10861\n",
      "Complement found from 14296 to 14302\n",
      "Complement found from 14671 to 14677\n",
      "Complement found from 15307 to 15313\n",
      "Seq is: ATTATT\n",
      "13 ATTATT motifs detected in file\n",
      "\n",
      "Complement found from 233 to 239\n",
      "Complement found from 4300 to 4306\n",
      "Complement found from 4756 to 4762\n",
      "Complement found from 5035 to 5041\n",
      "Complement found from 5500 to 5506\n",
      "Complement found from 6094 to 6100\n",
      "Complement found from 6157 to 6163\n",
      "Complement found from 6658 to 6664\n",
      "Complement found from 7303 to 7309\n",
      "Complement found from 8040 to 8046\n",
      "Complement found from 9679 to 9685\n",
      "Complement found from 10078 to 10084\n",
      "Complement found from 10108 to 10114\n",
      "Complement found from 10573 to 10579\n",
      "Complement found from 13610 to 13616\n",
      "Complement found from 14544 to 14550\n",
      "Complement found from 15651 to 15657\n",
      "Seq is: AATAAT\n",
      "17 AATAAT motifs detected in file\n",
      "\n",
      "Complement found from 276 to 282\n",
      "Complement found from 3746 to 3752\n",
      "Complement found from 3835 to 3841\n",
      "Complement found from 4749 to 4755\n",
      "Complement found from 4951 to 4957\n",
      "Complement found from 5296 to 5302\n",
      "Complement found from 5308 to 5314\n",
      "Complement found from 6118 to 6124\n",
      "Complement found from 7066 to 7072\n",
      "Complement found from 7794 to 7800\n",
      "Complement found from 10342 to 10348\n",
      "Complement found from 10863 to 10869\n",
      "Complement found from 11778 to 11784\n",
      "Complement found from 12114 to 12120\n",
      "Complement found from 12632 to 12638\n",
      "Complement found from 13370 to 13376\n",
      "Complement found from 13517 to 13523\n",
      "Complement found from 14057 to 14063\n",
      "Complement found from 14342 to 14348\n",
      "Complement found from 14481 to 14487\n",
      "Complement found from 14667 to 14673\n",
      "Complement found from 15789 to 15795\n",
      "Seq is: CATCAT\n",
      "22 CATCAT motifs detected in file\n",
      "\n",
      "Complement found from 15388 to 15394\n",
      "Complement found from 15587 to 15593\n",
      "Seq is: TCCGAT\n",
      "2 TCCGAT motifs detected in file\n",
      "\n",
      "Complement found from 307 to 313\n",
      "Complement found from 457 to 463\n",
      "Complement found from 958 to 964\n",
      "Complement found from 3115 to 3121\n",
      "Complement found from 3569 to 3575\n",
      "Complement found from 4105 to 4111\n",
      "Complement found from 5443 to 5449\n",
      "Complement found from 6218 to 6224\n",
      "Complement found from 6290 to 6296\n",
      "Complement found from 7815 to 7821\n",
      "Complement found from 7862 to 7868\n",
      "Complement found from 8470 to 8476\n",
      "Complement found from 10551 to 10557\n",
      "Complement found from 13447 to 13453\n",
      "Complement found from 13582 to 13588\n",
      "Complement found from 14261 to 14267\n",
      "Complement found from 14527 to 14533\n",
      "Complement found from 14805 to 14811\n",
      "Complement found from 15141 to 15147\n",
      "Complement found from 15380 to 15386\n",
      "Complement found from 15538 to 15544\n",
      "Complement found from 16186 to 16192\n",
      "Seq is: CCTCCC\n",
      "22 CCTCCC motifs detected in file\n",
      "\n",
      "Complement found from 3398 to 3404\n",
      "Complement found from 8766 to 8772\n",
      "Complement found from 10132 to 10138\n",
      "Complement found from 10298 to 10304\n",
      "Complement found from 11333 to 11339\n",
      "Seq is: ACAACT\n",
      "5 ACAACT motifs detected in file\n",
      "\n",
      "Complement found from 233 to 239\n",
      "Complement found from 4300 to 4306\n",
      "Complement found from 4756 to 4762\n",
      "Complement found from 5035 to 5041\n",
      "Complement found from 5500 to 5506\n",
      "Complement found from 6094 to 6100\n",
      "Complement found from 6157 to 6163\n",
      "Complement found from 6658 to 6664\n",
      "Complement found from 7303 to 7309\n",
      "Complement found from 8040 to 8046\n",
      "Complement found from 9679 to 9685\n",
      "Complement found from 10078 to 10084\n",
      "Complement found from 10108 to 10114\n",
      "Complement found from 10573 to 10579\n",
      "Complement found from 13610 to 13616\n",
      "Complement found from 14544 to 14550\n",
      "Complement found from 15651 to 15657\n",
      "Seq is: AATAAT\n",
      "17 AATAAT motifs detected in file\n",
      "\n",
      "Complement found from 2736 to 2742\n",
      "Complement found from 5635 to 5641\n",
      "Complement found from 10075 to 10081\n",
      "Seq is: TTTAAT\n",
      "3 TTTAAT motifs detected in file\n",
      "\n",
      "Complement found from 4144 to 4150\n",
      "Complement found from 6821 to 6827\n",
      "Complement found from 13117 to 13123\n",
      "Complement found from 14981 to 14987\n",
      "Seq is: TCCGCT\n",
      "4 TCCGCT motifs detected in file\n",
      "\n",
      "Complement found from 392 to 398\n",
      "Complement found from 3108 to 3114\n",
      "Complement found from 7874 to 7880\n",
      "Complement found from 11923 to 11929\n",
      "Complement found from 12688 to 12694\n",
      "Complement found from 15875 to 15881\n",
      "Complement found from 16346 to 16352\n",
      "Seq is: TCAAAT\n",
      "7 TCAAAT motifs detected in file\n",
      "\n",
      "Seq is: AATGAG\n",
      "0 AATGAG motifs detected in file\n",
      "\n",
      "Complement found from 285 to 291\n",
      "Complement found from 992 to 998\n",
      "Complement found from 2129 to 2135\n",
      "Complement found from 2150 to 2156\n",
      "Complement found from 2226 to 2232\n",
      "Complement found from 2456 to 2462\n",
      "Complement found from 4175 to 4181\n",
      "Complement found from 4604 to 4610\n",
      "Complement found from 5281 to 5287\n",
      "Complement found from 5746 to 5752\n",
      "Complement found from 6691 to 6697\n",
      "Complement found from 7446 to 7452\n",
      "Complement found from 8496 to 8502\n",
      "Complement found from 10047 to 10053\n",
      "Complement found from 10813 to 10819\n",
      "Complement found from 11031 to 11037\n",
      "Complement found from 12417 to 12423\n",
      "Complement found from 13230 to 13236\n",
      "Complement found from 14503 to 14509\n",
      "Seq is: AAAAAA\n",
      "19 AAAAAA motifs detected in file\n",
      "\n",
      "Complement found from 1630 to 1636\n",
      "Complement found from 6349 to 6355\n",
      "Complement found from 8416 to 8422\n",
      "Complement found from 11351 to 11357\n",
      "Complement found from 11719 to 11725\n",
      "Complement found from 11941 to 11947\n",
      "Complement found from 13212 to 13218\n",
      "Complement found from 16196 to 16202\n",
      "Seq is: CTTACA\n",
      "8 CTTACA motifs detected in file\n",
      "\n",
      "Complement found from 573 to 579\n",
      "Complement found from 1109 to 1115\n",
      "Complement found from 5413 to 5419\n",
      "Complement found from 8195 to 8201\n",
      "Seq is: ACAGTT\n",
      "4 ACAGTT motifs detected in file\n",
      "\n",
      "Complement found from 2893 to 2899\n",
      "Complement found from 8613 to 8619\n",
      "Complement found from 10251 to 10257\n",
      "Seq is: TTGATC\n",
      "3 TTGATC motifs detected in file\n",
      "\n",
      "Complement found from 209 to 215\n",
      "Complement found from 1858 to 1864\n",
      "Complement found from 2190 to 2196\n",
      "Complement found from 2368 to 2374\n",
      "Complement found from 4468 to 4474\n",
      "Complement found from 5638 to 5644\n",
      "Complement found from 8219 to 8225\n",
      "Complement found from 10023 to 10029\n",
      "Complement found from 14084 to 14090\n",
      "Complement found from 14499 to 14505\n",
      "Complement found from 14780 to 14786\n",
      "Seq is: AATTAA\n",
      "11 AATTAA motifs detected in file\n",
      "\n",
      "Complement found from 5540 to 5546\n",
      "Complement found from 9759 to 9765\n",
      "Complement found from 14287 to 14293\n",
      "Complement found from 14988 to 14994\n",
      "Complement found from 15125 to 15131\n",
      "Complement found from 15302 to 15308\n",
      "Seq is: CCTTCA\n",
      "6 CCTTCA motifs detected in file\n",
      "\n",
      "Complement found from 8553 to 8559\n",
      "Complement found from 8760 to 8766\n",
      "Complement found from 10262 to 10268\n",
      "Complement found from 10649 to 10655\n",
      "Seq is: ATTGCC\n",
      "4 ATTGCC motifs detected in file\n",
      "\n",
      "Complement found from 3176 to 3182\n",
      "Complement found from 4211 to 4217\n",
      "Complement found from 6731 to 6737\n",
      "Complement found from 8694 to 8700\n",
      "Complement found from 10449 to 10455\n",
      "Complement found from 12806 to 12812\n",
      "Complement found from 14702 to 14708\n",
      "Complement found from 15558 to 15564\n",
      "Seq is: ATGATA\n",
      "8 ATGATA motifs detected in file\n",
      "\n",
      "Complement found from 3854 to 3860\n",
      "Complement found from 4265 to 4271\n",
      "Complement found from 5803 to 5809\n",
      "Complement found from 6893 to 6899\n",
      "Complement found from 11340 to 11346\n",
      "Seq is: AATATG\n",
      "5 AATATG motifs detected in file\n",
      "\n",
      "Complement found from 55 to 61\n",
      "Complement found from 450 to 456\n",
      "Complement found from 1366 to 1372\n",
      "Complement found from 7307 to 7313\n",
      "Complement found from 7675 to 7681\n",
      "Seq is: ATTTTC\n",
      "5 ATTTTC motifs detected in file\n",
      "\n",
      "Complement found from 3748 to 3754\n",
      "Complement found from 7080 to 7086\n",
      "Complement found from 7274 to 7280\n",
      "Complement found from 8547 to 8553\n",
      "Complement found from 8593 to 8599\n",
      "Complement found from 8974 to 8980\n",
      "Complement found from 11501 to 11507\n",
      "Complement found from 11696 to 11702\n",
      "Complement found from 12043 to 12049\n",
      "Complement found from 14483 to 14489\n",
      "Complement found from 14794 to 14800\n",
      "Complement found from 15160 to 15166\n",
      "Complement found from 15743 to 15749\n",
      "Seq is: TCATTC\n",
      "13 TCATTC motifs detected in file\n",
      "\n",
      "Complement found from 11885 to 11891\n",
      "Seq is: TGGGAG\n",
      "1 TGGGAG motifs detected in file\n",
      "\n",
      "Complement found from 5688 to 5694\n",
      "Complement found from 8947 to 8953\n",
      "Complement found from 10317 to 10323\n",
      "Complement found from 12727 to 12733\n",
      "Complement found from 13276 to 13282\n",
      "Seq is: TAGTTA\n",
      "5 TAGTTA motifs detected in file\n",
      "\n",
      "Complement found from 1370 to 1376\n",
      "Complement found from 3501 to 3507\n",
      "Complement found from 4499 to 4505\n",
      "Complement found from 5047 to 5053\n",
      "Complement found from 5343 to 5349\n",
      "Complement found from 6285 to 6291\n",
      "Complement found from 8276 to 8282\n",
      "Complement found from 9184 to 9190\n",
      "Complement found from 11039 to 11045\n",
      "Complement found from 13786 to 13792\n",
      "Complement found from 13918 to 13924\n",
      "Seq is: TCTACC\n",
      "11 TCTACC motifs detected in file\n",
      "\n",
      "Complement found from 1207 to 1213\n",
      "Complement found from 5575 to 5581\n",
      "Complement found from 7265 to 7271\n",
      "Complement found from 9958 to 9964\n",
      "Complement found from 13321 to 13327\n",
      "Seq is: TCTGTA\n",
      "5 TCTGTA motifs detected in file\n",
      "\n",
      "Complement found from 128 to 134\n",
      "Complement found from 5847 to 5853\n",
      "Seq is: TGTCTT\n",
      "2 TGTCTT motifs detected in file\n",
      "\n",
      "Complement found from 12470 to 12476\n",
      "Seq is: TATCAG\n",
      "1 TATCAG motifs detected in file\n",
      "\n",
      "Complement found from 49 to 55\n",
      "Complement found from 653 to 659\n",
      "Complement found from 12311 to 12317\n",
      "Seq is: TTTGGT\n",
      "3 TTTGGT motifs detected in file\n",
      "\n",
      "Complement found from 9938 to 9944\n",
      "Seq is: GTAGAT\n",
      "1 GTAGAT motifs detected in file\n",
      "\n",
      "Complement found from 944 to 950\n",
      "Seq is: GTTTTA\n",
      "1 GTTTTA motifs detected in file\n",
      "\n",
      "Complement found from 899 to 905\n",
      "Complement found from 6616 to 6622\n",
      "Complement found from 8163 to 8169\n",
      "Seq is: CGGTCA\n",
      "3 CGGTCA motifs detected in file\n",
      "\n",
      "Complement found from 2095 to 2101\n",
      "Complement found from 7694 to 7700\n",
      "Complement found from 7802 to 7808\n",
      "Seq is: TAGTCC\n",
      "3 TAGTCC motifs detected in file\n",
      "\n",
      "Complement found from 5737 to 5743\n",
      "Complement found from 9914 to 9920\n",
      "Complement found from 10667 to 10673\n",
      "Seq is: GCCGCC\n",
      "3 GCCGCC motifs detected in file\n",
      "\n",
      "Complement found from 4628 to 4634\n",
      "Complement found from 5776 to 5782\n",
      "Complement found from 11678 to 11684\n",
      "Seq is: GAAGCT\n",
      "3 GAAGCT motifs detected in file\n",
      "\n",
      "Complement found from 2807 to 2813\n",
      "Complement found from 3091 to 3097\n",
      "Complement found from 12874 to 12880\n",
      "Seq is: TCGGTT\n",
      "3 TCGGTT motifs detected in file\n",
      "\n",
      "Complement found from 4910 to 4916\n",
      "Complement found from 6519 to 6525\n",
      "Complement found from 6986 to 6992\n",
      "Complement found from 7143 to 7149\n",
      "Complement found from 9849 to 9855\n",
      "Complement found from 11275 to 11281\n",
      "Seq is: TCACTA\n",
      "6 TCACTA motifs detected in file\n",
      "\n",
      "Complement found from 298 to 304\n",
      "Complement found from 348 to 354\n",
      "Complement found from 552 to 558\n",
      "Complement found from 1273 to 1279\n",
      "Complement found from 1684 to 1690\n",
      "Complement found from 1720 to 1726\n",
      "Complement found from 1991 to 1997\n",
      "Complement found from 2765 to 2771\n",
      "Complement found from 2788 to 2794\n",
      "Complement found from 4254 to 4260\n",
      "Complement found from 4982 to 4988\n",
      "Complement found from 5931 to 5937\n",
      "Complement found from 7121 to 7127\n",
      "Complement found from 8125 to 8131\n",
      "Complement found from 8189 to 8195\n",
      "Complement found from 8508 to 8514\n",
      "Complement found from 11668 to 11674\n",
      "Complement found from 14624 to 14630\n",
      "Complement found from 16281 to 16287\n",
      "Seq is: CAAACC\n",
      "19 CAAACC motifs detected in file\n",
      "\n",
      "Complement found from 1363 to 1369\n",
      "Complement found from 10117 to 10123\n",
      "Complement found from 15208 to 15214\n",
      "Complement found from 16096 to 16102\n",
      "Seq is: TACATT\n",
      "4 TACATT motifs detected in file\n",
      "\n",
      "Complement found from 2601 to 2607\n",
      "Complement found from 4801 to 4807\n",
      "Complement found from 9310 to 9316\n",
      "Complement found from 9880 to 9886\n",
      "Complement found from 9898 to 9904\n",
      "Complement found from 10511 to 10517\n",
      "Complement found from 13438 to 13444\n",
      "Complement found from 13813 to 13819\n",
      "Complement found from 15288 to 15294\n",
      "Complement found from 15712 to 15718\n",
      "Seq is: TCACTT\n",
      "10 TCACTT motifs detected in file\n",
      "\n",
      "Complement found from 4001 to 4007\n",
      "Complement found from 8037 to 8043\n",
      "Complement found from 11484 to 11490\n",
      "Seq is: TATAAT\n",
      "3 TATAAT motifs detected in file\n",
      "\n",
      "Complement found from 3884 to 3890\n",
      "Complement found from 4659 to 4665\n",
      "Complement found from 8642 to 8648\n",
      "Complement found from 9667 to 9673\n",
      "Complement found from 12857 to 12863\n",
      "Complement found from 14924 to 14930\n",
      "Complement found from 16078 to 16084\n",
      "Seq is: CAACCG\n",
      "7 CAACCG motifs detected in file\n",
      "\n",
      "Complement found from 3147 to 3153\n",
      "Complement found from 3323 to 3329\n",
      "Complement found from 4094 to 4100\n",
      "Complement found from 5116 to 5122\n",
      "Complement found from 5149 to 5155\n",
      "Complement found from 6229 to 6235\n",
      "Complement found from 6481 to 6487\n",
      "Complement found from 8969 to 8975\n",
      "Complement found from 9029 to 9035\n",
      "Complement found from 9343 to 9349\n",
      "Complement found from 10555 to 10561\n",
      "Complement found from 10696 to 10702\n",
      "Complement found from 10750 to 10756\n",
      "Complement found from 11235 to 11241\n",
      "Complement found from 11880 to 11886\n",
      "Complement found from 11937 to 11943\n",
      "Complement found from 13070 to 13076\n",
      "Complement found from 13682 to 13688\n",
      "Complement found from 13982 to 13988\n",
      "Complement found from 14129 to 14135\n",
      "Complement found from 16506 to 16512\n",
      "Seq is: CCTACT\n",
      "21 CCTACT motifs detected in file\n",
      "\n",
      "Complement found from 59 to 65\n",
      "Complement found from 6453 to 6459\n",
      "Seq is: TCGTCT\n",
      "2 TCGTCT motifs detected in file\n",
      "\n",
      "Complement found from 2600 to 2606\n",
      "Complement found from 6518 to 6524\n",
      "Complement found from 6985 to 6991\n",
      "Complement found from 9897 to 9903\n",
      "Complement found from 11929 to 11935\n",
      "Complement found from 14950 to 14956\n",
      "Complement found from 15711 to 15717\n",
      "Seq is: ATCACT\n",
      "7 ATCACT motifs detected in file\n",
      "\n",
      "Complement found from 664 to 670\n",
      "Complement found from 4797 to 4803\n",
      "Complement found from 7649 to 7655\n",
      "Complement found from 10205 to 10211\n",
      "Complement found from 13485 to 13491\n",
      "Complement found from 15284 to 15290\n",
      "Seq is: CCTTTC\n",
      "6 CCTTTC motifs detected in file\n",
      "\n",
      "Complement found from 2471 to 2477\n",
      "Seq is: AACTCG\n",
      "1 AACTCG motifs detected in file\n",
      "\n",
      "Complement found from 8949 to 8955\n",
      "Complement found from 12516 to 12522\n",
      "Seq is: GTTATT\n",
      "2 GTTATT motifs detected in file\n",
      "\n",
      "Complement found from 5591 to 5597\n",
      "Complement found from 9018 to 9024\n",
      "Complement found from 16240 to 16246\n",
      "Seq is: ACTGCA\n",
      "3 ACTGCA motifs detected in file\n",
      "\n",
      "Complement found from 1267 to 1273\n",
      "Complement found from 16510 to 16516\n",
      "Seq is: CTTCAG\n",
      "2 CTTCAG motifs detected in file\n",
      "\n",
      "Complement found from 162 to 168\n",
      "Complement found from 5163 to 5169\n",
      "Complement found from 9038 to 9044\n",
      "Seq is: GCACCT\n",
      "3 GCACCT motifs detected in file\n",
      "\n",
      "Complement found from 7223 to 7229\n",
      "Complement found from 8779 to 8785\n",
      "Seq is: TCGGAC\n",
      "2 TCGGAC motifs detected in file\n",
      "\n",
      "Seq is: ACCGAT\n",
      "0 ACCGAT motifs detected in file\n",
      "\n",
      "Complement found from 9476 to 9482\n",
      "Seq is: GTTTTT\n",
      "1 GTTTTT motifs detected in file\n",
      "\n",
      "Complement found from 488 to 494\n",
      "Complement found from 3397 to 3403\n",
      "Complement found from 3442 to 3448\n",
      "Complement found from 5054 to 5060\n",
      "Complement found from 6062 to 6068\n",
      "Complement found from 12855 to 12861\n",
      "Complement found from 14688 to 14694\n",
      "Seq is: TACAAC\n",
      "7 TACAAC motifs detected in file\n",
      "\n",
      "Complement found from 2474 to 2480\n",
      "Complement found from 13285 to 13291\n",
      "Complement found from 15089 to 15095\n",
      "Seq is: TCGGCA\n",
      "3 TCGGCA motifs detected in file\n",
      "\n",
      "Complement found from 9780 to 9786\n",
      "Seq is: TCTACG\n",
      "1 TCTACG motifs detected in file\n",
      "\n",
      "Complement found from 2181 to 2187\n",
      "Complement found from 3015 to 3021\n",
      "Complement found from 12834 to 12840\n",
      "Complement found from 13462 to 13468\n",
      "Complement found from 13707 to 13713\n",
      "Complement found from 15313 to 15319\n",
      "Seq is: GCAGCC\n",
      "6 GCAGCC motifs detected in file\n",
      "\n",
      "Complement found from 1727 to 1733\n",
      "Complement found from 2496 to 2502\n",
      "Complement found from 4550 to 4556\n",
      "Complement found from 5885 to 5891\n",
      "Complement found from 8257 to 8263\n",
      "Complement found from 9507 to 9513\n",
      "Complement found from 9722 to 9728\n",
      "Complement found from 10273 to 10279\n",
      "Complement found from 10461 to 10467\n",
      "Complement found from 10502 to 10508\n",
      "Complement found from 11989 to 11995\n",
      "Complement found from 12200 to 12206\n",
      "Complement found from 14323 to 14329\n",
      "Complement found from 15280 to 15286\n",
      "Complement found from 15784 to 15790\n",
      "Complement found from 16322 to 16328\n",
      "Seq is: TTTACC\n",
      "16 TTTACC motifs detected in file\n",
      "\n",
      "Complement found from 4180 to 4186\n",
      "Complement found from 7029 to 7035\n",
      "Complement found from 7621 to 7627\n",
      "Complement found from 7950 to 7956\n",
      "Complement found from 9312 to 9318\n",
      "Complement found from 10017 to 10023\n",
      "Complement found from 11201 to 11207\n",
      "Complement found from 12372 to 12378\n",
      "Complement found from 14095 to 14101\n",
      "Seq is: ACTTCC\n",
      "9 ACTTCC motifs detected in file\n",
      "\n",
      "Complement found from 1664 to 1670\n",
      "Complement found from 3654 to 3660\n",
      "Complement found from 3819 to 3825\n",
      "Complement found from 4836 to 4842\n",
      "Complement found from 6917 to 6923\n",
      "Complement found from 8172 to 8178\n",
      "Seq is: CTCTGA\n",
      "6 CTCTGA motifs detected in file\n",
      "\n",
      "Complement found from 1180 to 1186\n",
      "Complement found from 3934 to 3940\n",
      "Complement found from 5718 to 5724\n",
      "Complement found from 5870 to 5876\n",
      "Complement found from 7077 to 7083\n",
      "Complement found from 8544 to 8550\n",
      "Complement found from 9858 to 9864\n",
      "Complement found from 11681 to 11687\n",
      "Complement found from 12571 to 12577\n",
      "Seq is: GCTTCA\n",
      "9 GCTTCA motifs detected in file\n",
      "\n",
      "Complement found from 12921 to 12927\n",
      "Seq is: TGAGAC\n",
      "1 TGAGAC motifs detected in file\n",
      "\n",
      "Complement found from 3918 to 3924\n",
      "Seq is: TCCGAA\n",
      "1 TCCGAA motifs detected in file\n",
      "\n",
      "Complement found from 146 to 152\n",
      "Complement found from 501 to 507\n",
      "Complement found from 4393 to 4399\n",
      "Complement found from 7255 to 7261\n",
      "Complement found from 7797 to 7803\n",
      "Complement found from 7830 to 7836\n",
      "Complement found from 10345 to 10351\n",
      "Complement found from 11723 to 11729\n",
      "Complement found from 12389 to 12395\n",
      "Complement found from 12881 to 12887\n",
      "Complement found from 14120 to 14126\n",
      "Complement found from 15636 to 15642\n",
      "Complement found from 15642 to 15648\n",
      "Complement found from 15660 to 15666\n",
      "Complement found from 16403 to 16409\n",
      "Seq is: CATCCT\n",
      "15 CATCCT motifs detected in file\n",
      "\n",
      "Complement found from 3339 to 3345\n",
      "Complement found from 4227 to 4233\n",
      "Complement found from 5437 to 5443\n",
      "Complement found from 6584 to 6590\n",
      "Complement found from 7956 to 7962\n",
      "Complement found from 8029 to 8035\n",
      "Complement found from 12085 to 12091\n",
      "Complement found from 12433 to 12439\n",
      "Complement found from 13692 to 13698\n",
      "Complement found from 14629 to 14635\n",
      "Complement found from 15383 to 15389\n",
      "Seq is: CCCATT\n",
      "11 CCCATT motifs detected in file\n",
      "\n",
      "Complement found from 399 to 405\n",
      "Complement found from 3862 to 3868\n",
      "Complement found from 7682 to 7688\n",
      "Complement found from 12520 to 12526\n",
      "Complement found from 15017 to 15023\n",
      "Seq is: TTATCT\n",
      "5 TTATCT motifs detected in file\n",
      "\n",
      "Complement found from 9710 to 9716\n",
      "Complement found from 11435 to 11441\n",
      "Seq is: CTGGGT\n",
      "2 CTGGGT motifs detected in file\n",
      "\n",
      "Complement found from 1365 to 1371\n",
      "Complement found from 5882 to 5888\n",
      "Complement found from 7674 to 7680\n",
      "Complement found from 8753 to 8759\n",
      "Complement found from 9792 to 9798\n",
      "Complement found from 9932 to 9938\n",
      "Complement found from 10119 to 10125\n",
      "Complement found from 13891 to 13897\n",
      "Seq is: CATTTT\n",
      "8 CATTTT motifs detected in file\n",
      "\n",
      "Complement found from 2319 to 2325\n",
      "Complement found from 3761 to 3767\n",
      "Complement found from 3993 to 3999\n",
      "Complement found from 9012 to 9018\n",
      "Complement found from 9790 to 9796\n",
      "Complement found from 10040 to 10046\n",
      "Complement found from 11082 to 11088\n",
      "Complement found from 11281 to 11287\n",
      "Complement found from 12672 to 12678\n",
      "Complement found from 13659 to 13665\n",
      "Complement found from 13746 to 13752\n",
      "Seq is: AACATT\n",
      "11 AACATT motifs detected in file\n",
      "\n",
      "Complement found from 8032 to 8038\n",
      "Seq is: ATTCGT\n",
      "1 ATTCGT motifs detected in file\n",
      "\n",
      "Complement found from 11027 to 11033\n",
      "Complement found from 15346 to 15352\n",
      "Seq is: CACGAA\n",
      "2 CACGAA motifs detected in file\n",
      "\n",
      "Complement found from 5075 to 5081\n",
      "Complement found from 5194 to 5200\n",
      "Complement found from 5568 to 5574\n",
      "Complement found from 7581 to 7587\n",
      "Complement found from 8747 to 8753\n",
      "Complement found from 9146 to 9152\n",
      "Complement found from 10069 to 10075\n",
      "Complement found from 11073 to 11079\n",
      "Complement found from 11337 to 11343\n",
      "Complement found from 15459 to 15465\n",
      "Seq is: CTTAAT\n",
      "10 CTTAAT motifs detected in file\n",
      "\n",
      "Complement found from 1721 to 1727\n",
      "Complement found from 1906 to 1912\n",
      "Complement found from 4978 to 4984\n",
      "Complement found from 5932 to 5938\n",
      "Complement found from 7529 to 7535\n",
      "Complement found from 8121 to 8127\n",
      "Complement found from 8190 to 8196\n",
      "Complement found from 8959 to 8965\n",
      "Complement found from 9673 to 9679\n",
      "Complement found from 11126 to 11132\n",
      "Complement found from 13426 to 13432\n",
      "Complement found from 14712 to 14718\n",
      "Seq is: AAACCA\n",
      "12 AAACCA motifs detected in file\n",
      "\n",
      "Complement found from 7483 to 7489\n",
      "Complement found from 10995 to 11001\n",
      "Complement found from 13342 to 13348\n",
      "Complement found from 15705 to 15711\n",
      "Complement found from 16252 to 16258\n",
      "Complement found from 16316 to 16322\n",
      "Seq is: AAGCCA\n",
      "6 AAGCCA motifs detected in file\n",
      "\n",
      "Complement found from 283 to 289\n",
      "Complement found from 1009 to 1015\n",
      "Complement found from 1121 to 1127\n",
      "Complement found from 4867 to 4873\n",
      "Complement found from 5279 to 5285\n",
      "Complement found from 5426 to 5432\n",
      "Complement found from 7444 to 7450\n",
      "Complement found from 7726 to 7732\n",
      "Complement found from 10378 to 10384\n",
      "Complement found from 10419 to 10425\n",
      "Complement found from 11518 to 11524\n",
      "Complement found from 12415 to 12421\n",
      "Complement found from 13217 to 13223\n",
      "Complement found from 15866 to 15872\n",
      "Seq is: ACAAAA\n",
      "14 ACAAAA motifs detected in file\n",
      "\n",
      "Complement found from 9451 to 9457\n",
      "Complement found from 10451 to 10457\n",
      "Seq is: GATAAT\n",
      "2 GATAAT motifs detected in file\n",
      "\n",
      "Complement found from 4272 to 4278\n",
      "Complement found from 9922 to 9928\n",
      "Seq is: CTGATA\n",
      "2 CTGATA motifs detected in file\n",
      "\n",
      "Complement found from 25 to 31\n",
      "Complement found from 461 to 467\n",
      "Complement found from 1689 to 1695\n",
      "Complement found from 4188 to 4194\n",
      "Complement found from 5602 to 5608\n",
      "Complement found from 9316 to 9322\n",
      "Complement found from 9511 to 9517\n",
      "Complement found from 9593 to 9599\n",
      "Complement found from 10621 to 10627\n",
      "Complement found from 13076 to 13082\n",
      "Complement found from 13186 to 13192\n",
      "Complement found from 14115 to 14121\n",
      "Complement found from 14790 to 14796\n",
      "Seq is: CCACTC\n",
      "13 CCACTC motifs detected in file\n",
      "\n",
      "Complement found from 2956 to 2962\n",
      "Complement found from 3915 to 3921\n",
      "Complement found from 4809 to 4815\n",
      "Complement found from 5982 to 5988\n",
      "Seq is: GAGTCC\n",
      "4 GAGTCC motifs detected in file\n",
      "\n",
      "Complement found from 439 to 445\n",
      "Complement found from 861 to 867\n",
      "Complement found from 1831 to 1837\n",
      "Complement found from 5185 to 5191\n",
      "Complement found from 6529 to 6535\n",
      "Complement found from 7731 to 7737\n",
      "Complement found from 7741 to 7747\n",
      "Complement found from 8678 to 8684\n",
      "Complement found from 8769 to 8775\n",
      "Complement found from 9346 to 9352\n",
      "Complement found from 9358 to 9364\n",
      "Complement found from 10301 to 10307\n",
      "Complement found from 10959 to 10965\n",
      "Complement found from 13656 to 13662\n",
      "Complement found from 13740 to 13746\n",
      "Complement found from 14764 to 14770\n",
      "Seq is: ACTAAC\n",
      "16 ACTAAC motifs detected in file\n",
      "\n",
      "Complement found from 4335 to 4341\n",
      "Complement found from 8848 to 8854\n",
      "Complement found from 10367 to 10373\n",
      "Complement found from 11555 to 11561\n",
      "Seq is: TATGAG\n",
      "4 TATGAG motifs detected in file\n",
      "\n",
      "Complement found from 1017 to 1023\n",
      "Complement found from 7226 to 7232\n",
      "Complement found from 7918 to 7924\n",
      "Complement found from 10125 to 10131\n",
      "Complement found from 10374 to 10380\n",
      "Complement found from 10730 to 10736\n",
      "Complement found from 12582 to 12588\n",
      "Complement found from 13417 to 13423\n",
      "Complement found from 14685 to 14691\n",
      "Seq is: GACTAC\n",
      "9 GACTAC motifs detected in file\n",
      "\n",
      "Complement found from 2439 to 2445\n",
      "Complement found from 7272 to 7278\n",
      "Complement found from 11610 to 11616\n",
      "Complement found from 12794 to 12800\n",
      "Seq is: GCTCAT\n",
      "4 GCTCAT motifs detected in file\n",
      "\n",
      "Complement found from 7702 to 7708\n",
      "Complement found from 10560 to 10566\n",
      "Seq is: TATGCC\n",
      "2 TATGCC motifs detected in file\n",
      "\n",
      "Complement found from 689 to 695\n",
      "Complement found from 1476 to 1482\n",
      "Complement found from 3987 to 3993\n",
      "Complement found from 4068 to 4074\n",
      "Complement found from 8704 to 8710\n",
      "Complement found from 11353 to 11359\n",
      "Complement found from 13214 to 13220\n",
      "Complement found from 13542 to 13548\n",
      "Complement found from 15032 to 15038\n",
      "Complement found from 15415 to 15421\n",
      "Complement found from 15577 to 15583\n",
      "Seq is: TACACA\n",
      "11 TACACA motifs detected in file\n",
      "\n",
      "Complement found from 2944 to 2950\n",
      "Complement found from 3650 to 3656\n",
      "Complement found from 4672 to 4678\n",
      "Complement found from 7569 to 7575\n",
      "Complement found from 8564 to 8570\n",
      "Complement found from 9454 to 9460\n",
      "Complement found from 11784 to 11790\n",
      "Complement found from 12848 to 12854\n",
      "Complement found from 14368 to 14374\n",
      "Complement found from 15828 to 15834\n",
      "Complement found from 15834 to 15840\n",
      "Seq is: AATCCT\n",
      "11 AATCCT motifs detected in file\n",
      "\n",
      "Complement found from 14 to 20\n",
      "Complement found from 109 to 115\n",
      "Complement found from 721 to 727\n",
      "Complement found from 1490 to 1496\n",
      "Complement found from 2279 to 2285\n",
      "Complement found from 3509 to 3515\n",
      "Complement found from 3737 to 3743\n",
      "Complement found from 3794 to 3800\n",
      "Complement found from 4010 to 4016\n",
      "Complement found from 4193 to 4199\n",
      "Complement found from 5190 to 5196\n",
      "Complement found from 5206 to 5212\n",
      "Complement found from 5323 to 5329\n",
      "Complement found from 5703 to 5709\n",
      "Complement found from 6314 to 6320\n",
      "Complement found from 6620 to 6626\n",
      "Complement found from 7111 to 7117\n",
      "Complement found from 7401 to 7407\n",
      "Complement found from 9056 to 9062\n",
      "Complement found from 10087 to 10093\n",
      "Complement found from 11214 to 11220\n",
      "Complement found from 11265 to 11271\n",
      "Complement found from 12060 to 12066\n",
      "Complement found from 12359 to 12365\n",
      "Complement found from 12398 to 12404\n",
      "Complement found from 13622 to 13628\n",
      "Complement found from 13649 to 13655\n",
      "Complement found from 13679 to 13685\n",
      "Complement found from 15264 to 15270\n",
      "Complement found from 15406 to 15412\n",
      "Complement found from 16291 to 16297\n",
      "Seq is: CACCCT\n",
      "31 CACCCT motifs detected in file\n",
      "\n",
      "Complement found from 1969 to 1975\n",
      "Seq is: GAAGAT\n",
      "1 GAAGAT motifs detected in file\n",
      "\n",
      "Complement found from 1530 to 1536\n",
      "Complement found from 1836 to 1842\n",
      "Complement found from 3428 to 3434\n",
      "Complement found from 3483 to 3489\n",
      "Complement found from 6152 to 6158\n",
      "Complement found from 7094 to 7100\n",
      "Complement found from 7625 to 7631\n",
      "Complement found from 8227 to 8233\n",
      "Complement found from 9254 to 9260\n",
      "Complement found from 9525 to 9531\n",
      "Complement found from 9581 to 9587\n",
      "Complement found from 10277 to 10283\n",
      "Complement found from 10936 to 10942\n",
      "Complement found from 11233 to 11239\n",
      "Complement found from 13128 to 13134\n",
      "Complement found from 13946 to 13952\n",
      "Complement found from 13980 to 13986\n",
      "Complement found from 14489 to 14495\n",
      "Complement found from 14770 to 14776\n",
      "Complement found from 15367 to 15373\n",
      "Seq is: CCCCTA\n",
      "20 CCCCTA motifs detected in file\n",
      "\n",
      "Complement found from 3250 to 3256\n",
      "Complement found from 3347 to 3353\n",
      "Complement found from 9134 to 9140\n",
      "Complement found from 11606 to 11612\n",
      "Complement found from 11706 to 11712\n",
      "Complement found from 14940 to 14946\n",
      "Seq is: AATCGC\n",
      "6 AATCGC motifs detected in file\n",
      "\n",
      "Complement found from 460 to 466\n",
      "Complement found from 1083 to 1089\n",
      "Complement found from 1688 to 1694\n",
      "Complement found from 2216 to 2222\n",
      "Complement found from 5601 to 5607\n",
      "Complement found from 5896 to 5902\n",
      "Complement found from 7026 to 7032\n",
      "Complement found from 8292 to 8298\n",
      "Complement found from 8902 to 8908\n",
      "Complement found from 9592 to 9598\n",
      "Complement found from 10620 to 10626\n",
      "Complement found from 11814 to 11820\n",
      "Complement found from 11869 to 11875\n",
      "Complement found from 12968 to 12974\n",
      "Complement found from 13145 to 13151\n",
      "Complement found from 13880 to 13886\n",
      "Complement found from 14114 to 14120\n",
      "Complement found from 14389 to 14395\n",
      "Complement found from 15699 to 15705\n",
      "Complement found from 16265 to 16271\n",
      "Seq is: CCCACT\n",
      "20 CCCACT motifs detected in file\n",
      "\n",
      "Complement found from 9816 to 9822\n",
      "Complement found from 11387 to 11393\n",
      "Complement found from 14682 to 14688\n",
      "Seq is: ACGGAC\n",
      "3 ACGGAC motifs detected in file\n",
      "\n",
      "Complement found from 300 to 306\n",
      "Complement found from 350 to 356\n",
      "Complement found from 491 to 497\n",
      "Complement found from 531 to 537\n",
      "Complement found from 554 to 560\n",
      "Complement found from 864 to 870\n",
      "Complement found from 1220 to 1226\n",
      "Complement found from 1528 to 1534\n",
      "Complement found from 1834 to 1840\n",
      "Complement found from 3563 to 3569\n",
      "Complement found from 3582 to 3588\n",
      "Complement found from 3890 to 3896\n",
      "Complement found from 4315 to 4321\n",
      "Complement found from 5598 to 5604\n",
      "Complement found from 5841 to 5847\n",
      "Complement found from 6417 to 6423\n",
      "Complement found from 7463 to 7469\n",
      "Complement found from 7488 to 7494\n",
      "Complement found from 10944 to 10950\n",
      "Complement found from 11509 to 11515\n",
      "Complement found from 11670 to 11676\n",
      "Complement found from 12106 to 12112\n",
      "Complement found from 13674 to 13680\n",
      "Complement found from 13689 to 13695\n",
      "Complement found from 14386 to 14392\n",
      "Complement found from 14415 to 14421\n",
      "Complement found from 14585 to 14591\n",
      "Complement found from 14617 to 14623\n",
      "Complement found from 14626 to 14632\n",
      "Complement found from 14767 to 14773\n",
      "Complement found from 15364 to 15370\n",
      "Complement found from 15523 to 15529\n",
      "Complement found from 16181 to 16187\n",
      "Seq is: AACCCC\n",
      "33 AACCCC motifs detected in file\n",
      "\n",
      "Complement found from 899 to 905\n",
      "Complement found from 6616 to 6622\n",
      "Complement found from 8163 to 8169\n",
      "Seq is: CGGTCA\n",
      "3 CGGTCA motifs detected in file\n",
      "\n",
      "Complement found from 32 to 38\n",
      "Complement found from 5754 to 5760\n",
      "Seq is: CGGGAG\n",
      "2 CGGGAG motifs detected in file\n",
      "\n",
      "Complement found from 821 to 827\n",
      "Complement found from 908 to 914\n",
      "Complement found from 2348 to 2354\n",
      "Complement found from 3046 to 3052\n",
      "Complement found from 4974 to 4980\n",
      "Complement found from 8333 to 8339\n",
      "Complement found from 8885 to 8891\n",
      "Seq is: GATTAA\n",
      "7 GATTAA motifs detected in file\n",
      "\n",
      "Complement found from 302 to 308\n",
      "Complement found from 432 to 438\n",
      "Complement found from 567 to 573\n",
      "Complement found from 3565 to 3571\n",
      "Complement found from 7396 to 7402\n",
      "Complement found from 7465 to 7471\n",
      "Complement found from 9531 to 9537\n",
      "Complement found from 10946 to 10952\n",
      "Complement found from 11866 to 11872\n",
      "Complement found from 12384 to 12390\n",
      "Complement found from 16374 to 16380\n",
      "Seq is: CCCCCC\n",
      "11 CCCCCC motifs detected in file\n",
      "\n",
      "Complement found from 702 to 708\n",
      "Complement found from 2073 to 2079\n",
      "Complement found from 4473 to 4479\n",
      "Complement found from 5300 to 5306\n",
      "Complement found from 6842 to 6848\n",
      "Complement found from 8616 to 8622\n",
      "Complement found from 8841 to 8847\n",
      "Complement found from 8937 to 8943\n",
      "Complement found from 9579 to 9585\n",
      "Complement found from 10189 to 10195\n",
      "Complement found from 11137 to 11143\n",
      "Complement found from 12081 to 12087\n",
      "Complement found from 13760 to 13766\n",
      "Complement found from 13779 to 13785\n",
      "Complement found from 13877 to 13883\n",
      "Complement found from 13944 to 13950\n",
      "Complement found from 15655 to 15661\n",
      "Seq is: ATCCCC\n",
      "17 ATCCCC motifs detected in file\n",
      "\n",
      "Complement found from 6459 to 6465\n",
      "Complement found from 15590 to 15596\n",
      "Seq is: GATCCG\n",
      "2 GATCCG motifs detected in file\n",
      "\n",
      "Complement found from 1103 to 1109\n",
      "Complement found from 1231 to 1237\n",
      "Complement found from 3595 to 3601\n",
      "Complement found from 5366 to 5372\n",
      "Complement found from 5888 to 5894\n",
      "Complement found from 6543 to 6549\n",
      "Complement found from 8682 to 8688\n",
      "Complement found from 9467 to 9473\n",
      "Complement found from 10542 to 10548\n",
      "Complement found from 13843 to 13849\n",
      "Complement found from 14064 to 14070\n",
      "Complement found from 14410 to 14416\n",
      "Seq is: ACCTCA\n",
      "12 ACCTCA motifs detected in file\n",
      "\n",
      "Complement found from 1724 to 1730\n",
      "Complement found from 5881 to 5887\n",
      "Complement found from 6701 to 6707\n",
      "Complement found from 7138 to 7144\n",
      "Complement found from 7532 to 7538\n",
      "Complement found from 9765 to 9771\n",
      "Complement found from 16319 to 16325\n",
      "Seq is: CCATTT\n",
      "7 CCATTT motifs detected in file\n",
      "\n",
      "Complement found from 2840 to 2846\n",
      "Seq is: TCCGAG\n",
      "1 TCCGAG motifs detected in file\n",
      "\n",
      "Complement found from 402 to 408\n",
      "Complement found from 6940 to 6946\n",
      "Complement found from 9992 to 9998\n",
      "Seq is: TCTTTT\n",
      "3 TCTTTT motifs detected in file\n",
      "\n",
      "Complement found from 225 to 231\n",
      "Complement found from 4329 to 4335\n",
      "Complement found from 13264 to 13270\n",
      "Complement found from 13822 to 13828\n",
      "Seq is: TAGGAC\n",
      "4 TAGGAC motifs detected in file\n",
      "\n",
      "Complement found from 315 to 321\n",
      "Complement found from 4852 to 4858\n",
      "Complement found from 5782 to 5788\n",
      "Complement found from 6204 to 6210\n",
      "Seq is: GCTTCT\n",
      "4 GCTTCT motifs detected in file\n",
      "\n",
      "Complement found from 4547 to 4553\n",
      "Complement found from 9477 to 9483\n",
      "Complement found from 9794 to 9800\n",
      "Complement found from 10879 to 10885\n",
      "Seq is: TTTTTT\n",
      "4 TTTTTT motifs detected in file\n",
      "\n",
      "Complement found from 2720 to 2726\n",
      "Seq is: GAAGAC\n",
      "1 GAAGAC motifs detected in file\n",
      "\n",
      "Complement found from 7137 to 7143\n",
      "Complement found from 12286 to 12292\n",
      "Complement found from 12447 to 12453\n",
      "Seq is: TCCATT\n",
      "3 TCCATT motifs detected in file\n",
      "\n",
      "Complement found from 155 to 161\n",
      "Complement found from 1539 to 1545\n",
      "Complement found from 1973 to 1979\n",
      "Complement found from 2740 to 2746\n",
      "Complement found from 3614 to 3620\n",
      "Complement found from 3860 to 3866\n",
      "Complement found from 5087 to 5093\n",
      "Complement found from 9460 to 9466\n",
      "Complement found from 12899 to 12905\n",
      "Complement found from 13352 to 13358\n",
      "Seq is: ATTTAT\n",
      "10 ATTTAT motifs detected in file\n",
      "\n",
      "Complement found from 3362 to 3368\n",
      "Complement found from 5353 to 5359\n",
      "Complement found from 5706 to 5712\n",
      "Complement found from 6154 to 6160\n",
      "Complement found from 6466 to 6472\n",
      "Complement found from 7348 to 7354\n",
      "Complement found from 9041 to 9047\n",
      "Complement found from 9280 to 9286\n",
      "Complement found from 10851 to 10857\n",
      "Complement found from 10953 to 10959\n",
      "Complement found from 12377 to 12383\n",
      "Complement found from 12707 to 12713\n",
      "Complement found from 14135 to 14141\n",
      "Complement found from 14772 to 14778\n",
      "Complement found from 15831 to 15837\n",
      "Complement found from 15837 to 15843\n",
      "Complement found from 15855 to 15861\n",
      "Seq is: CCTAAT\n",
      "17 CCTAAT motifs detected in file\n",
      "\n",
      "Complement found from 9544 to 9550\n",
      "Seq is: AGGGCA\n",
      "1 AGGGCA motifs detected in file\n",
      "\n",
      "Complement found from 583 to 589\n",
      "Complement found from 1608 to 1614\n",
      "Complement found from 4581 to 4587\n",
      "Complement found from 11348 to 11354\n",
      "Complement found from 11360 to 11366\n",
      "Complement found from 11822 to 11828\n",
      "Seq is: TAGCTT\n",
      "6 TAGCTT motifs detected in file\n",
      "\n",
      "Complement found from 6687 to 6693\n",
      "Complement found from 7203 to 7209\n",
      "Complement found from 13711 to 13717\n",
      "Seq is: CCGGAA\n",
      "3 CCGGAA motifs detected in file\n",
      "\n",
      "Complement found from 1101 to 1107\n",
      "Complement found from 1673 to 1679\n",
      "Complement found from 1760 to 1766\n",
      "Complement found from 1992 to 1998\n",
      "Complement found from 2132 to 2138\n",
      "Complement found from 2789 to 2795\n",
      "Complement found from 4255 to 4261\n",
      "Complement found from 7122 to 7128\n",
      "Complement found from 7372 to 7378\n",
      "Complement found from 8825 to 8831\n",
      "Complement found from 10747 to 10753\n",
      "Complement found from 13973 to 13979\n",
      "Complement found from 15077 to 15083\n",
      "Complement found from 15934 to 15940\n",
      "Complement found from 16282 to 16288\n",
      "Seq is: AAACCT\n",
      "15 AAACCT motifs detected in file\n",
      "\n",
      "Seq is: GTCCAC\n",
      "0 GTCCAC motifs detected in file\n",
      "\n",
      "Complement found from 268 to 274\n",
      "Complement found from 513 to 519\n",
      "Complement found from 1047 to 1053\n",
      "Complement found from 7409 to 7415\n",
      "Complement found from 9416 to 9422\n",
      "Complement found from 13931 to 13937\n",
      "Complement found from 16231 to 16237\n",
      "Seq is: CACACA\n",
      "7 CACACA motifs detected in file\n",
      "\n",
      "Complement found from 893 to 899\n",
      "Complement found from 6846 to 6852\n",
      "Seq is: CCACCG\n",
      "2 CCACCG motifs detected in file\n",
      "\n",
      "Complement found from 544 to 550\n",
      "Complement found from 1597 to 1603\n",
      "Complement found from 4719 to 4725\n",
      "Complement found from 6698 to 6704\n",
      "Complement found from 7969 to 7975\n",
      "Complement found from 8341 to 8347\n",
      "Complement found from 8518 to 8524\n",
      "Complement found from 11017 to 11023\n",
      "Seq is: GAACCA\n",
      "8 GAACCA motifs detected in file\n",
      "\n",
      "Complement found from 7002 to 7008\n",
      "Complement found from 8142 to 8148\n",
      "Seq is: TACACG\n",
      "2 TACACG motifs detected in file\n",
      "\n",
      "Complement found from 937 to 943\n",
      "Complement found from 1935 to 1941\n",
      "Complement found from 2101 to 2107\n",
      "Complement found from 3477 to 3483\n",
      "Complement found from 4278 to 4284\n",
      "Complement found from 5831 to 5837\n",
      "Complement found from 10050 to 10056\n",
      "Seq is: AAAGAG\n",
      "7 AAAGAG motifs detected in file\n",
      "\n",
      "Complement found from 1049 to 1055\n",
      "Complement found from 8561 to 8567\n",
      "Complement found from 9104 to 9110\n",
      "Complement found from 10983 to 10989\n",
      "Complement found from 11355 to 11361\n",
      "Complement found from 11999 to 12005\n",
      "Complement found from 13940 to 13946\n",
      "Complement found from 15417 to 15423\n",
      "Complement found from 15579 to 15585\n",
      "Seq is: CACAAT\n",
      "9 CACAAT motifs detected in file\n",
      "\n",
      "Seq is: GTAGCG\n",
      "0 GTAGCG motifs detected in file\n",
      "\n",
      "Seq is: CGGTAG\n",
      "0 CGGTAG motifs detected in file\n",
      "\n",
      "Complement found from 22 to 28\n",
      "Complement found from 382 to 388\n",
      "Complement found from 1808 to 1814\n",
      "Complement found from 2666 to 2672\n",
      "Complement found from 4599 to 4605\n",
      "Complement found from 4725 to 4731\n",
      "Complement found from 5067 to 5073\n",
      "Complement found from 6336 to 6342\n",
      "Complement found from 8698 to 8704\n",
      "Complement found from 9348 to 9354\n",
      "Complement found from 9360 to 9366\n",
      "Complement found from 10884 to 10890\n",
      "Complement found from 11906 to 11912\n",
      "Complement found from 12149 to 12155\n",
      "Complement found from 12337 to 12343\n",
      "Complement found from 12355 to 12361\n",
      "Complement found from 13855 to 13861\n",
      "Complement found from 14787 to 14793\n",
      "Seq is: TAACCA\n",
      "18 TAACCA motifs detected in file\n",
      "\n",
      "Complement found from 1256 to 1262\n",
      "Complement found from 1840 to 1846\n",
      "Complement found from 3199 to 3205\n",
      "Complement found from 4451 to 4457\n",
      "Complement found from 4893 to 4899\n",
      "Complement found from 5955 to 5961\n",
      "Complement found from 6591 to 6597\n",
      "Complement found from 9366 to 9372\n",
      "Complement found from 15511 to 15517\n",
      "Seq is: TATACC\n",
      "9 TATACC motifs detected in file\n",
      "\n",
      "Complement found from 730 to 736\n",
      "Complement found from 2599 to 2605\n",
      "Complement found from 5371 to 5377\n",
      "Complement found from 5811 to 5817\n",
      "Complement found from 6469 to 6475\n",
      "Complement found from 8651 to 8657\n",
      "Complement found from 9216 to 9222\n",
      "Complement found from 9634 to 9640\n",
      "Complement found from 14138 to 14144\n",
      "Complement found from 14877 to 14883\n",
      "Complement found from 15375 to 15381\n",
      "Complement found from 15396 to 15402\n",
      "Complement found from 15710 to 15716\n",
      "Seq is: AATCAC\n",
      "13 AATCAC motifs detected in file\n",
      "\n",
      "Complement found from 489 to 495\n",
      "Complement found from 1709 to 1715\n",
      "Complement found from 3443 to 3449\n",
      "Complement found from 5055 to 5061\n",
      "Complement found from 8641 to 8647\n",
      "Complement found from 9666 to 9672\n",
      "Complement found from 10837 to 10843\n",
      "Complement found from 10898 to 10904\n",
      "Complement found from 10942 to 10948\n",
      "Complement found from 12543 to 12549\n",
      "Complement found from 12552 to 12558\n",
      "Complement found from 12739 to 12745\n",
      "Complement found from 12856 to 12862\n",
      "Complement found from 13378 to 13384\n",
      "Complement found from 14329 to 14335\n",
      "Complement found from 14476 to 14482\n",
      "Complement found from 14689 to 14695\n",
      "Complement found from 15362 to 15368\n",
      "Complement found from 15765 to 15771\n",
      "Complement found from 16077 to 16083\n",
      "Seq is: ACAACC\n",
      "20 ACAACC motifs detected in file\n",
      "\n",
      "Complement found from 4759 to 4765\n",
      "Complement found from 4888 to 4894\n",
      "Complement found from 6832 to 6838\n",
      "Complement found from 7671 to 7677\n",
      "Complement found from 8750 to 8756\n",
      "Complement found from 10339 to 10345\n",
      "Complement found from 10454 to 10460\n",
      "Complement found from 10986 to 10992\n",
      "Complement found from 11103 to 11109\n",
      "Complement found from 14231 to 14237\n",
      "Complement found from 14976 to 14982\n",
      "Seq is: AATCAT\n",
      "11 AATCAT motifs detected in file\n",
      "\n",
      "Seq is: CCGGAT\n",
      "0 CCGGAT motifs detected in file\n",
      "\n",
      "Complement found from 189 to 195\n",
      "Complement found from 468 to 474\n",
      "Complement found from 5007 to 5013\n",
      "Complement found from 7947 to 7953\n",
      "Complement found from 8409 to 8415\n",
      "Complement found from 8942 to 8948\n",
      "Complement found from 9334 to 9340\n",
      "Complement found from 10654 to 10660\n",
      "Complement found from 11198 to 11204\n",
      "Complement found from 11618 to 11624\n",
      "Complement found from 11955 to 11961\n",
      "Complement found from 12716 to 12722\n",
      "Complement found from 13346 to 13352\n",
      "Complement found from 13907 to 13913\n",
      "Complement found from 14345 to 14351\n",
      "Seq is: CATACT\n",
      "15 CATACT motifs detected in file\n",
      "\n",
      "Complement found from 11847 to 11853\n",
      "Complement found from 11856 to 11862\n",
      "Complement found from 12884 to 12890\n",
      "Complement found from 13638 to 13644\n",
      "Complement found from 13805 to 13811\n",
      "Complement found from 16445 to 16451\n",
      "Seq is: CCTCGC\n",
      "6 CCTCGC motifs detected in file\n",
      "\n",
      "Complement found from 6112 to 6118\n",
      "Complement found from 6435 to 6441\n",
      "Complement found from 13481 to 13487\n",
      "Complement found from 15840 to 15846\n",
      "Seq is: AATACC\n",
      "4 AATACC motifs detected in file\n",
      "\n",
      "Complement found from 979 to 985\n",
      "Complement found from 2509 to 2515\n",
      "Complement found from 5813 to 5819\n",
      "Complement found from 6816 to 6822\n",
      "Complement found from 7646 to 7652\n",
      "Complement found from 9636 to 9642\n",
      "Complement found from 14062 to 14068\n",
      "Complement found from 15377 to 15383\n",
      "Complement found from 15398 to 15404\n",
      "Seq is: TCACCT\n",
      "9 TCACCT motifs detected in file\n",
      "\n",
      "Complement found from 2601 to 2607\n",
      "Complement found from 4801 to 4807\n",
      "Complement found from 9310 to 9316\n",
      "Complement found from 9880 to 9886\n",
      "Complement found from 9898 to 9904\n",
      "Complement found from 10511 to 10517\n",
      "Complement found from 13438 to 13444\n",
      "Complement found from 13813 to 13819\n",
      "Complement found from 15288 to 15294\n",
      "Complement found from 15712 to 15718\n",
      "Seq is: TCACTT\n",
      "10 TCACTT motifs detected in file\n",
      "\n",
      "Seq is: TAGTGT\n",
      "0 TAGTGT motifs detected in file\n",
      "\n",
      "Seq is: ACCGAT\n",
      "0 ACCGAT motifs detected in file\n",
      "\n",
      "Complement found from 4845 to 4851\n",
      "Complement found from 9291 to 9297\n",
      "Seq is: CCGGCC\n",
      "2 CCGGCC motifs detected in file\n",
      "\n",
      "Complement found from 710 to 716\n",
      "Complement found from 999 to 1005\n",
      "Complement found from 4591 to 4597\n",
      "Complement found from 11011 to 11017\n",
      "Seq is: TCCAGT\n",
      "4 TCCAGT motifs detected in file\n",
      "\n",
      "Complement found from 2944 to 2950\n",
      "Complement found from 3650 to 3656\n",
      "Complement found from 4672 to 4678\n",
      "Complement found from 7569 to 7575\n",
      "Complement found from 8564 to 8570\n",
      "Complement found from 9454 to 9460\n",
      "Complement found from 11784 to 11790\n",
      "Complement found from 12848 to 12854\n",
      "Complement found from 14368 to 14374\n",
      "Complement found from 15828 to 15834\n",
      "Complement found from 15834 to 15840\n",
      "Seq is: AATCCT\n",
      "11 AATCCT motifs detected in file\n",
      "\n",
      "Complement found from 3527 to 3533\n",
      "Complement found from 7212 to 7218\n",
      "Complement found from 12109 to 12115\n",
      "Complement found from 14556 to 14562\n",
      "Seq is: CCCGAC\n",
      "4 CCCGAC motifs detected in file\n",
      "\n",
      "Complement found from 4054 to 4060\n",
      "Complement found from 4904 to 4910\n",
      "Complement found from 5214 to 5220\n",
      "Complement found from 5484 to 5490\n",
      "Complement found from 5731 to 5737\n",
      "Complement found from 6496 to 6502\n",
      "Complement found from 6663 to 6669\n",
      "Complement found from 9755 to 9761\n",
      "Complement found from 11057 to 11063\n",
      "Complement found from 12562 to 12568\n",
      "Complement found from 15851 to 15857\n",
      "Seq is: TCTCCC\n",
      "11 TCTCCC motifs detected in file\n",
      "\n",
      "Seq is: GGGGAC\n",
      "0 GGGGAC motifs detected in file\n",
      "\n",
      "Complement found from 2072 to 2078\n",
      "Complement found from 2230 to 2236\n",
      "Complement found from 4472 to 4478\n",
      "Complement found from 9578 to 9584\n",
      "Complement found from 13778 to 13784\n",
      "Complement found from 13876 to 13882\n",
      "Complement found from 13943 to 13949\n",
      "Complement found from 15654 to 15660\n",
      "Complement found from 16349 to 16355\n",
      "Seq is: AATCCC\n",
      "9 AATCCC motifs detected in file\n",
      "\n",
      "Complement found from 2755 to 2761\n",
      "Complement found from 2846 to 2852\n",
      "Complement found from 8584 to 8590\n",
      "Complement found from 11456 to 11462\n",
      "Complement found from 16300 to 16306\n",
      "Seq is: CAGTAC\n",
      "5 CAGTAC motifs detected in file\n",
      "\n",
      "Complement found from 3528 to 3534\n",
      "Complement found from 5910 to 5916\n",
      "Complement found from 10931 to 10937\n",
      "Complement found from 14557 to 14563\n",
      "Seq is: CCGACC\n",
      "4 CCGACC motifs detected in file\n",
      "\n",
      "Complement found from 1010 to 1016\n",
      "Complement found from 1958 to 1964\n",
      "Complement found from 4364 to 4370\n",
      "Complement found from 4996 to 5002\n",
      "Complement found from 7132 to 7138\n",
      "Complement found from 8522 to 8528\n",
      "Complement found from 13218 to 13224\n",
      "Complement found from 14534 to 14540\n",
      "Complement found from 15867 to 15873\n",
      "Seq is: CAAAAT\n",
      "9 CAAAAT motifs detected in file\n",
      "\n",
      "Complement found from 1775 to 1781\n",
      "Complement found from 12652 to 12658\n",
      "Seq is: GATATA\n",
      "2 GATATA motifs detected in file\n",
      "\n",
      "Complement found from 4686 to 4692\n",
      "Complement found from 6840 to 6846\n",
      "Complement found from 7199 to 7205\n",
      "Complement found from 7781 to 7787\n",
      "Complement found from 9124 to 9130\n",
      "Complement found from 11548 to 11554\n",
      "Complement found from 12079 to 12085\n",
      "Complement found from 12097 to 12103\n",
      "Complement found from 12283 to 12289\n",
      "Complement found from 15193 to 15199\n",
      "Complement found from 15631 to 15637\n",
      "Seq is: CTATCC\n",
      "11 CTATCC motifs detected in file\n",
      "\n",
      "Complement found from 1475 to 1481\n",
      "Complement found from 7911 to 7917\n",
      "Seq is: GTACAC\n",
      "2 GTACAC motifs detected in file\n",
      "\n",
      "Complement found from 22 to 28\n",
      "Complement found from 382 to 388\n",
      "Complement found from 1808 to 1814\n",
      "Complement found from 2666 to 2672\n",
      "Complement found from 4599 to 4605\n",
      "Complement found from 4725 to 4731\n",
      "Complement found from 5067 to 5073\n",
      "Complement found from 6336 to 6342\n",
      "Complement found from 8698 to 8704\n",
      "Complement found from 9348 to 9354\n",
      "Complement found from 9360 to 9366\n",
      "Complement found from 10884 to 10890\n",
      "Complement found from 11906 to 11912\n",
      "Complement found from 12149 to 12155\n",
      "Complement found from 12337 to 12343\n",
      "Complement found from 12355 to 12361\n",
      "Complement found from 13855 to 13861\n",
      "Complement found from 14787 to 14793\n",
      "Seq is: TAACCA\n",
      "18 TAACCA motifs detected in file\n",
      "\n",
      "Complement found from 2546 to 2552\n",
      "Complement found from 6499 to 6505\n",
      "Complement found from 13058 to 13064\n",
      "Seq is: CCCAGT\n",
      "3 CCCAGT motifs detected in file\n",
      "\n",
      "Complement found from 486 to 492\n",
      "Complement found from 602 to 608\n",
      "Complement found from 3985 to 3991\n",
      "Complement found from 5525 to 5531\n",
      "Complement found from 15905 to 15911\n",
      "Seq is: AATACA\n",
      "5 AATACA motifs detected in file\n",
      "\n",
      "Complement found from 1107 to 1113\n",
      "Complement found from 9558 to 9564\n",
      "Complement found from 12177 to 12183\n",
      "Complement found from 14649 to 14655\n",
      "Complement found from 15119 to 15125\n",
      "Seq is: CAACAG\n",
      "5 CAACAG motifs detected in file\n",
      "\n",
      "Complement found from 9824 to 9830\n",
      "Seq is: CACGTC\n",
      "1 CACGTC motifs detected in file\n",
      "\n",
      "Complement found from 339 to 345\n",
      "Complement found from 3183 to 3189\n",
      "Complement found from 4882 to 4888\n",
      "Complement found from 7746 to 7752\n",
      "Complement found from 10507 to 10513\n",
      "Complement found from 14823 to 14829\n",
      "Seq is: CATCTC\n",
      "6 CATCTC motifs detected in file\n",
      "\n",
      "Complement found from 10425 to 10431\n",
      "Complement found from 15555 to 15561\n",
      "Seq is: CGAATG\n",
      "2 CGAATG motifs detected in file\n",
      "\n",
      "Complement found from 1265 to 1271\n",
      "Complement found from 4026 to 4032\n",
      "Complement found from 6098 to 6104\n",
      "Complement found from 6341 to 6347\n",
      "Complement found from 7933 to 7939\n",
      "Complement found from 9099 to 9105\n",
      "Complement found from 11116 to 11122\n",
      "Complement found from 12702 to 12708\n",
      "Complement found from 13104 to 13110\n",
      "Seq is: ATCTTC\n",
      "9 ATCTTC motifs detected in file\n",
      "\n",
      "Complement found from 1633 to 1639\n",
      "Complement found from 5683 to 5689\n",
      "Complement found from 7186 to 7192\n",
      "Complement found from 9090 to 9096\n",
      "Complement found from 9168 to 9174\n",
      "Complement found from 11131 to 11137\n",
      "Complement found from 16463 to 16469\n",
      "Seq is: ACACTT\n",
      "7 ACACTT motifs detected in file\n",
      "\n",
      "Complement found from 11615 to 11621\n",
      "Seq is: TTGCAT\n",
      "1 TTGCAT motifs detected in file\n",
      "\n",
      "Complement found from 542 to 548\n",
      "Complement found from 3373 to 3379\n",
      "Complement found from 3887 to 3893\n",
      "Complement found from 3919 to 3925\n",
      "Seq is: CCGAAC\n",
      "4 CCGAAC motifs detected in file\n",
      "\n",
      "Complement found from 4466 to 4472\n",
      "Complement found from 9042 to 9048\n",
      "Complement found from 9111 to 9117\n",
      "Complement found from 10852 to 10858\n",
      "Complement found from 11248 to 11254\n",
      "Complement found from 12378 to 12384\n",
      "Complement found from 12708 to 12714\n",
      "Complement found from 15856 to 15862\n",
      "Complement found from 16003 to 16009\n",
      "Seq is: CTAATT\n",
      "9 CTAATT motifs detected in file\n",
      "\n",
      "Complement found from 523 to 529\n",
      "Seq is: CCGCTG\n",
      "1 CCGCTG motifs detected in file\n",
      "\n",
      "Complement found from 6837 to 6843\n",
      "Complement found from 11849 to 11855\n",
      "Complement found from 13576 to 13582\n",
      "Complement found from 14381 to 14387\n",
      "Seq is: TCGCTA\n",
      "4 TCGCTA motifs detected in file\n",
      "\n",
      "Seq is: AAGATC\n",
      "0 AAGATC motifs detected in file\n",
      "\n",
      "Complement found from 10 to 16\n",
      "Complement found from 2275 to 2281\n",
      "Complement found from 3756 to 3762\n",
      "Complement found from 4382 to 4388\n",
      "Complement found from 7043 to 7049\n",
      "Complement found from 7146 to 7152\n",
      "Complement found from 7259 to 7265\n",
      "Complement found from 7628 to 7634\n",
      "Complement found from 9226 to 9232\n",
      "Complement found from 11023 to 11029\n",
      "Complement found from 11150 to 11156\n",
      "Complement found from 11314 to 11320\n",
      "Complement found from 13180 to 13186\n",
      "Complement found from 16227 to 16233\n",
      "Seq is: CTATCA\n",
      "14 CTATCA motifs detected in file\n",
      "\n",
      "Complement found from 4490 to 4496\n",
      "Complement found from 10324 to 10330\n",
      "Seq is: GTCATC\n",
      "2 GTCATC motifs detected in file\n",
      "\n",
      "Complement found from 4470 to 4476\n",
      "Complement found from 8748 to 8754\n",
      "Complement found from 9147 to 9153\n",
      "Complement found from 10337 to 10343\n",
      "Complement found from 12676 to 12682\n",
      "Seq is: TTAATC\n",
      "5 TTAATC motifs detected in file\n",
      "\n",
      "Complement found from 422 to 428\n",
      "Complement found from 1033 to 1039\n",
      "Complement found from 1612 to 1618\n",
      "Complement found from 2157 to 2163\n",
      "Complement found from 2370 to 2376\n",
      "Complement found from 3298 to 3304\n",
      "Complement found from 5691 to 5697\n",
      "Complement found from 12026 to 12032\n",
      "Complement found from 13384 to 13390\n",
      "Complement found from 15469 to 15475\n",
      "Complement found from 16296 to 16302\n",
      "Seq is: TTAACA\n",
      "11 TTAACA motifs detected in file\n",
      "\n",
      "Complement found from 3552 to 3558\n",
      "Complement found from 4098 to 4104\n",
      "Complement found from 4676 to 4682\n",
      "Complement found from 5341 to 5347\n",
      "Complement found from 6041 to 6047\n",
      "Complement found from 9171 to 9177\n",
      "Complement found from 10514 to 10520\n",
      "Complement found from 11836 to 11842\n",
      "Complement found from 12942 to 12948\n",
      "Complement found from 13827 to 13833\n",
      "Seq is: CTTCTA\n",
      "10 CTTCTA motifs detected in file\n",
      "\n",
      "Complement found from 2948 to 2954\n",
      "Complement found from 3969 to 3975\n",
      "Complement found from 5921 to 5927\n",
      "Complement found from 6602 to 6608\n",
      "Complement found from 7097 to 7103\n",
      "Complement found from 8422 to 8428\n",
      "Complement found from 11206 to 11212\n",
      "Complement found from 11734 to 11740\n",
      "Complement found from 12744 to 12750\n",
      "Complement found from 13719 to 13725\n",
      "Complement found from 14149 to 14155\n",
      "Complement found from 14890 to 14896\n",
      "Complement found from 15337 to 15343\n",
      "Complement found from 15475 to 15481\n",
      "Complement found from 15568 to 15574\n",
      "Complement found from 16013 to 16019\n",
      "Seq is: CTATTC\n",
      "16 CTATTC motifs detected in file\n",
      "\n",
      "Complement found from 338 to 344\n",
      "Complement found from 3498 to 3504\n",
      "Complement found from 6057 to 6063\n",
      "Complement found from 7745 to 7751\n",
      "Complement found from 13318 to 13324\n",
      "Complement found from 14822 to 14828\n",
      "Complement found from 16496 to 16502\n",
      "Seq is: ACATCT\n",
      "7 ACATCT motifs detected in file\n",
      "\n",
      "Complement found from 3447 to 3453\n",
      "Complement found from 3894 to 3900\n",
      "Complement found from 3960 to 3966\n",
      "Complement found from 4455 to 4461\n",
      "Complement found from 9758 to 9764\n",
      "Complement found from 11228 to 11234\n",
      "Complement found from 11536 to 11542\n",
      "Complement found from 12940 to 12946\n",
      "Complement found from 13764 to 13770\n",
      "Complement found from 15301 to 15307\n",
      "Complement found from 16352 to 16358\n",
      "Seq is: CCCTTC\n",
      "11 CCCTTC motifs detected in file\n",
      "\n",
      "Complement found from 2189 to 2195\n",
      "Complement found from 2367 to 2373\n",
      "Complement found from 5016 to 5022\n",
      "Complement found from 6402 to 6408\n",
      "Complement found from 9536 to 9542\n",
      "Complement found from 10022 to 10028\n",
      "Complement found from 10781 to 10787\n",
      "Complement found from 13011 to 13017\n",
      "Complement found from 14168 to 14174\n",
      "Complement found from 15507 to 15513\n",
      "Seq is: CAATTA\n",
      "10 CAATTA motifs detected in file\n",
      "\n",
      "Complement found from 436 to 442\n",
      "Complement found from 1626 to 1632\n",
      "Complement found from 4154 to 4160\n",
      "Complement found from 8370 to 8376\n",
      "Complement found from 8436 to 8442\n",
      "Complement found from 8813 to 8819\n",
      "Complement found from 9868 to 9874\n",
      "Complement found from 12749 to 12755\n",
      "Complement found from 12913 to 12919\n",
      "Complement found from 15844 to 15850\n",
      "Seq is: CCAACT\n",
      "10 CCAACT motifs detected in file\n",
      "\n",
      "Complement found from 9323 to 9329\n",
      "Seq is: ATAACG\n",
      "1 ATAACG motifs detected in file\n",
      "\n",
      "Complement found from 2231 to 2237\n",
      "Complement found from 15202 to 15208\n",
      "Seq is: ATCCCA\n",
      "2 ATCCCA motifs detected in file\n",
      "\n",
      "Seq is: CCGCGA\n",
      "0 CCGCGA motifs detected in file\n",
      "\n",
      "Complement found from 2451 to 2457\n",
      "Complement found from 4818 to 4824\n",
      "Complement found from 5519 to 5525\n",
      "Seq is: AGGTTA\n",
      "3 AGGTTA motifs detected in file\n",
      "\n",
      "Complement found from 5203 to 5209\n",
      "Complement found from 10156 to 10162\n",
      "Complement found from 12458 to 12464\n",
      "Complement found from 13374 to 13380\n",
      "Complement found from 16170 to 16176\n",
      "Seq is: ATCCAC\n",
      "5 ATCCAC motifs detected in file\n",
      "\n",
      "Complement found from 8011 to 8017\n",
      "Complement found from 11458 to 11464\n",
      "Seq is: GTACTC\n",
      "2 GTACTC motifs detected in file\n",
      "\n",
      "Complement found from 5203 to 5209\n",
      "Complement found from 10156 to 10162\n",
      "Complement found from 12458 to 12464\n",
      "Complement found from 13374 to 13380\n",
      "Complement found from 16170 to 16176\n",
      "Seq is: ATCCAC\n",
      "5 ATCCAC motifs detected in file\n",
      "\n",
      "Complement found from 8112 to 8118\n",
      "Seq is: CGGACG\n",
      "1 CGGACG motifs detected in file\n",
      "\n",
      "Complement found from 2403 to 2409\n",
      "Complement found from 4751 to 4757\n",
      "Complement found from 9828 to 9834\n",
      "Complement found from 10439 to 10445\n",
      "Complement found from 10590 to 10596\n",
      "Complement found from 11728 to 11734\n",
      "Complement found from 12116 to 12122\n",
      "Complement found from 13735 to 13741\n",
      "Complement found from 14669 to 14675\n",
      "Complement found from 15305 to 15311\n",
      "Seq is: TCATTA\n",
      "10 TCATTA motifs detected in file\n",
      "\n",
      "Complement found from 8001 to 8007\n",
      "Complement found from 13282 to 13288\n",
      "Complement found from 14939 to 14945\n",
      "Seq is: CAATCG\n",
      "3 CAATCG motifs detected in file\n",
      "\n",
      "Complement found from 2270 to 2276\n",
      "Complement found from 4737 to 4743\n",
      "Complement found from 9214 to 9220\n",
      "Complement found from 14366 to 14372\n",
      "Complement found from 15708 to 15714\n",
      "Complement found from 16167 to 16173\n",
      "Seq is: CCAATC\n",
      "6 CCAATC motifs detected in file\n",
      "\n",
      "Complement found from 5971 to 5977\n",
      "Complement found from 14829 to 14835\n",
      "Seq is: CGCATG\n",
      "2 CGCATG motifs detected in file\n",
      "\n",
      "Seq is: CCGGTC\n",
      "0 CCGGTC motifs detected in file\n",
      "\n",
      "Complement found from 6171 to 6177\n",
      "Complement found from 15389 to 15395\n",
      "Seq is: CCGATA\n",
      "2 CCGATA motifs detected in file\n",
      "\n",
      "Complement found from 546 to 552\n",
      "Complement found from 2394 to 2400\n",
      "Complement found from 2909 to 2915\n",
      "Complement found from 3882 to 3888\n",
      "Complement found from 4153 to 4159\n",
      "Complement found from 6594 to 6600\n",
      "Complement found from 8343 to 8349\n",
      "Complement found from 8804 to 8810\n",
      "Complement found from 9350 to 9356\n",
      "Complement found from 13294 to 13300\n",
      "Complement found from 13857 to 13863\n",
      "Complement found from 14184 to 14190\n",
      "Complement found from 15843 to 15849\n",
      "Complement found from 16276 to 16282\n",
      "Seq is: ACCAAC\n",
      "14 ACCAAC motifs detected in file\n",
      "\n",
      "Complement found from 3644 to 3650\n",
      "Complement found from 3825 to 3831\n",
      "Complement found from 7219 to 7225\n",
      "Complement found from 9614 to 9620\n",
      "Complement found from 9988 to 9994\n",
      "Complement found from 13110 to 13116\n",
      "Complement found from 13567 to 13573\n",
      "Seq is: TTACTC\n",
      "7 TTACTC motifs detected in file\n",
      "\n",
      "Complement found from 16491 to 16497\n",
      "Seq is: ATCCGA\n",
      "1 ATCCGA motifs detected in file\n",
      "\n",
      "Complement found from 1702 to 1708\n",
      "Complement found from 2784 to 2790\n",
      "Complement found from 3502 to 3508\n",
      "Complement found from 4185 to 4191\n",
      "Complement found from 4500 to 4506\n",
      "Complement found from 4734 to 4740\n",
      "Complement found from 6644 to 6650\n",
      "Complement found from 6825 to 6831\n",
      "Complement found from 7405 to 7411\n",
      "Complement found from 8460 to 8466\n",
      "Complement found from 10127 to 10133\n",
      "Complement found from 10280 to 10286\n",
      "Complement found from 10792 to 10798\n",
      "Seq is: CTACCA\n",
      "13 CTACCA motifs detected in file\n",
      "\n",
      "Complement found from 854 to 860\n",
      "Complement found from 969 to 975\n",
      "Complement found from 1891 to 1897\n",
      "Complement found from 4418 to 4424\n",
      "Complement found from 5174 to 5180\n",
      "Complement found from 5642 to 5648\n",
      "Complement found from 8301 to 8307\n",
      "Complement found from 14013 to 14019\n",
      "Complement found from 15774 to 15780\n",
      "Complement found from 15992 to 15998\n",
      "Seq is: AAGCTA\n",
      "10 AAGCTA motifs detected in file\n",
      "\n",
      "Complement found from 8758 to 8764\n",
      "Complement found from 9831 to 9837\n",
      "Complement found from 15308 to 15314\n",
      "Complement found from 15717 to 15723\n",
      "Seq is: TTATTG\n",
      "4 TTATTG motifs detected in file\n",
      "\n",
      "Complement found from 277 to 283\n",
      "Complement found from 4760 to 4766\n",
      "Complement found from 4889 to 4895\n",
      "Complement found from 4952 to 4958\n",
      "Complement found from 5309 to 5315\n",
      "Complement found from 6119 to 6125\n",
      "Complement found from 7067 to 7073\n",
      "Complement found from 7148 to 7154\n",
      "Complement found from 7630 to 7636\n",
      "Complement found from 9228 to 9234\n",
      "Complement found from 10455 to 10461\n",
      "Complement found from 11104 to 11110\n",
      "Complement found from 11779 to 11785\n",
      "Complement found from 12633 to 12639\n",
      "Complement found from 13538 to 13544\n",
      "Complement found from 14232 to 14238\n",
      "Complement found from 14343 to 14349\n",
      "Seq is: ATCATA\n",
      "17 ATCATA motifs detected in file\n",
      "\n",
      "Complement found from 1530 to 1536\n",
      "Complement found from 1836 to 1842\n",
      "Complement found from 3428 to 3434\n",
      "Complement found from 3483 to 3489\n",
      "Complement found from 6152 to 6158\n",
      "Complement found from 7094 to 7100\n",
      "Complement found from 7625 to 7631\n",
      "Complement found from 8227 to 8233\n",
      "Complement found from 9254 to 9260\n",
      "Complement found from 9525 to 9531\n",
      "Complement found from 9581 to 9587\n",
      "Complement found from 10277 to 10283\n",
      "Complement found from 10936 to 10942\n",
      "Complement found from 11233 to 11239\n",
      "Complement found from 13128 to 13134\n",
      "Complement found from 13946 to 13952\n",
      "Complement found from 13980 to 13986\n",
      "Complement found from 14489 to 14495\n",
      "Complement found from 14770 to 14776\n",
      "Complement found from 15367 to 15373\n",
      "Seq is: CCCCTA\n",
      "20 CCCCTA motifs detected in file\n",
      "\n",
      "Complement found from 453 to 459\n",
      "Complement found from 3165 to 3171\n",
      "Complement found from 4246 to 4252\n",
      "Complement found from 6150 to 6156\n",
      "Complement found from 6183 to 6189\n",
      "Complement found from 7092 to 7098\n",
      "Complement found from 7623 to 7629\n",
      "Complement found from 7952 to 7958\n",
      "Complement found from 8225 to 8231\n",
      "Complement found from 8602 to 8608\n",
      "Complement found from 10914 to 10920\n",
      "Complement found from 11231 to 11237\n",
      "Complement found from 12382 to 12388\n",
      "Complement found from 12480 to 12486\n",
      "Complement found from 13644 to 13650\n",
      "Complement found from 13751 to 13757\n",
      "Complement found from 14152 to 14158\n",
      "Complement found from 14486 to 14492\n",
      "Complement found from 16543 to 16549\n",
      "Seq is: TTCCCC\n",
      "19 TTCCCC motifs detected in file\n",
      "\n",
      "Complement found from 3002 to 3008\n",
      "Complement found from 4841 to 4847\n",
      "Complement found from 7254 to 7260\n",
      "Complement found from 9605 to 9611\n",
      "Complement found from 9887 to 9893\n",
      "Complement found from 11722 to 11728\n",
      "Seq is: ACATCC\n",
      "6 ACATCC motifs detected in file\n",
      "\n",
      "Seq is: ACACGG\n",
      "0 ACACGG motifs detected in file\n",
      "\n",
      "Complement found from 372 to 378\n",
      "Complement found from 2159 to 2165\n",
      "Complement found from 2212 to 2218\n",
      "Complement found from 3814 to 3820\n",
      "Complement found from 4008 to 4014\n",
      "Complement found from 5188 to 5194\n",
      "Complement found from 6548 to 6554\n",
      "Complement found from 6597 to 6603\n",
      "Complement found from 8346 to 8352\n",
      "Complement found from 10085 to 10091\n",
      "Complement found from 10616 to 10622\n",
      "Complement found from 11263 to 11269\n",
      "Complement found from 12058 to 12064\n",
      "Complement found from 14739 to 14745\n",
      "Complement found from 15532 to 15538\n",
      "Seq is: AACACC\n",
      "15 AACACC motifs detected in file\n",
      "\n",
      "Complement found from 3971 to 3977\n",
      "Complement found from 5072 to 5078\n",
      "Complement found from 6635 to 6641\n",
      "Complement found from 10219 to 10225\n",
      "Complement found from 13614 to 13620\n",
      "Complement found from 15012 to 15018\n",
      "Complement found from 15276 to 15282\n",
      "Complement found from 15339 to 15345\n",
      "Seq is: ATTCTT\n",
      "8 ATTCTT motifs detected in file\n",
      "\n",
      "Complement found from 1943 to 1949\n",
      "Complement found from 11157 to 11163\n",
      "Complement found from 14554 to 14560\n",
      "Seq is: CACCCG\n",
      "3 CACCCG motifs detected in file\n",
      "\n",
      "Complement found from 3004 to 3010\n",
      "Complement found from 16422 to 16428\n",
      "Seq is: ATCCCG\n",
      "2 ATCCCG motifs detected in file\n",
      "\n",
      "Complement found from 991 to 997\n",
      "Complement found from 2149 to 2155\n",
      "Complement found from 2225 to 2231\n",
      "Complement found from 2455 to 2461\n",
      "Complement found from 2799 to 2805\n",
      "Complement found from 5400 to 5406\n",
      "Complement found from 5828 to 5834\n",
      "Complement found from 8093 to 8099\n",
      "Complement found from 8231 to 8237\n",
      "Complement found from 8441 to 8447\n",
      "Complement found from 8489 to 8495\n",
      "Complement found from 8495 to 8501\n",
      "Complement found from 8888 to 8894\n",
      "Complement found from 14502 to 14508\n",
      "Complement found from 16160 to 16166\n",
      "Seq is: TAAAAA\n",
      "15 TAAAAA motifs detected in file\n",
      "\n",
      "Complement found from 7523 to 7529\n",
      "Complement found from 14608 to 14614\n",
      "Seq is: TTAGAA\n",
      "2 TTAGAA motifs detected in file\n",
      "\n",
      "Complement found from 5159 to 5165\n",
      "Complement found from 14676 to 14682\n",
      "Seq is: TCTCGC\n",
      "2 TCTCGC motifs detected in file\n",
      "\n",
      "Complement found from 2805 to 2811\n",
      "Complement found from 6613 to 6619\n",
      "Seq is: TTTCGG\n",
      "2 TTTCGG motifs detected in file\n",
      "\n",
      "Complement found from 175 to 181\n",
      "Complement found from 5091 to 5097\n",
      "Complement found from 7298 to 7304\n",
      "Complement found from 8446 to 8452\n",
      "Complement found from 10488 to 10494\n",
      "Complement found from 10786 to 10792\n",
      "Complement found from 15051 to 15057\n",
      "Seq is: ATATTA\n",
      "7 ATATTA motifs detected in file\n",
      "\n",
      "Complement found from 2550 to 2556\n",
      "Seq is: GTGACA\n",
      "1 GTGACA motifs detected in file\n",
      "\n",
      "Complement found from 15039 to 15045\n",
      "Seq is: CGGGCG\n",
      "1 CGGGCG motifs detected in file\n",
      "\n",
      "Complement found from 2526 to 2532\n",
      "Complement found from 3194 to 3200\n",
      "Complement found from 4641 to 4647\n",
      "Complement found from 6859 to 6865\n",
      "Complement found from 16058 to 16064\n",
      "Seq is: AGTATT\n",
      "5 AGTATT motifs detected in file\n",
      "\n",
      "Complement found from 1530 to 1536\n",
      "Complement found from 1836 to 1842\n",
      "Complement found from 3428 to 3434\n",
      "Complement found from 3483 to 3489\n",
      "Complement found from 6152 to 6158\n",
      "Complement found from 7094 to 7100\n",
      "Complement found from 7625 to 7631\n",
      "Complement found from 8227 to 8233\n",
      "Complement found from 9254 to 9260\n",
      "Complement found from 9525 to 9531\n",
      "Complement found from 9581 to 9587\n",
      "Complement found from 10277 to 10283\n",
      "Complement found from 10936 to 10942\n",
      "Complement found from 11233 to 11239\n",
      "Complement found from 13128 to 13134\n",
      "Complement found from 13946 to 13952\n",
      "Complement found from 13980 to 13986\n",
      "Complement found from 14489 to 14495\n",
      "Complement found from 14770 to 14776\n",
      "Complement found from 15367 to 15373\n",
      "Seq is: CCCCTA\n",
      "20 CCCCTA motifs detected in file\n",
      "\n",
      "Complement found from 3245 to 3251\n",
      "Seq is: CCGGTA\n",
      "1 CCGGTA motifs detected in file\n",
      "\n",
      "Complement found from 1224 to 1230\n",
      "Complement found from 15588 to 15594\n",
      "Seq is: CCGATC\n",
      "2 CCGATC motifs detected in file\n",
      "\n",
      "Complement found from 645 to 651\n",
      "Complement found from 1733 to 1739\n",
      "Complement found from 8627 to 8633\n",
      "Complement found from 9677 to 9683\n",
      "Complement found from 11924 to 11930\n",
      "Complement found from 12324 to 12330\n",
      "Complement found from 12689 to 12695\n",
      "Complement found from 12933 to 12939\n",
      "Complement found from 15154 to 15160\n",
      "Seq is: CAAATA\n",
      "9 CAAATA motifs detected in file\n",
      "\n",
      "Complement found from 9 to 15\n",
      "Complement found from 2274 to 2280\n",
      "Complement found from 3097 to 3103\n",
      "Complement found from 6371 to 6377\n",
      "Seq is: TCTATC\n",
      "4 TCTATC motifs detected in file\n",
      "\n",
      "Complement found from 912 to 918\n",
      "Complement found from 1686 to 1692\n",
      "Complement found from 2423 to 2429\n",
      "Complement found from 2767 to 2773\n",
      "Complement found from 2831 to 2837\n",
      "Complement found from 4346 to 4352\n",
      "Complement found from 4984 to 4990\n",
      "Complement found from 5430 to 5436\n",
      "Complement found from 5674 to 5680\n",
      "Complement found from 6430 to 6436\n",
      "Complement found from 9240 to 9246\n",
      "Complement found from 12545 to 12551\n",
      "Complement found from 12554 to 12560\n",
      "Complement found from 14069 to 14075\n",
      "Complement found from 14516 to 14522\n",
      "Complement found from 14639 to 14645\n",
      "Complement found from 16164 to 16170\n",
      "Seq is: AACCCA\n",
      "17 AACCCA motifs detected in file\n",
      "\n",
      "Complement found from 547 to 553\n",
      "Complement found from 2834 to 2840\n",
      "Complement found from 3316 to 3322\n",
      "Complement found from 3580 to 3586\n",
      "Complement found from 3883 to 3889\n",
      "Complement found from 4483 to 4489\n",
      "Complement found from 7486 to 7492\n",
      "Complement found from 8805 to 8811\n",
      "Complement found from 10918 to 10924\n",
      "Complement found from 13295 to 13301\n",
      "Complement found from 15521 to 15527\n",
      "Seq is: CCAACC\n",
      "11 CCAACC motifs detected in file\n",
      "\n",
      "Complement found from 1364 to 1370\n",
      "Complement found from 1516 to 1522\n",
      "Complement found from 9791 to 9797\n",
      "Complement found from 10118 to 10124\n",
      "Complement found from 13747 to 13753\n",
      "Complement found from 13890 to 13896\n",
      "Seq is: ACATTT\n",
      "6 ACATTT motifs detected in file\n",
      "\n",
      "Complement found from 27 to 33\n",
      "Complement found from 977 to 983\n",
      "Complement found from 4190 to 4196\n",
      "Complement found from 7719 to 7725\n",
      "Complement found from 11257 to 11263\n",
      "Complement found from 11292 to 11298\n",
      "Complement found from 11767 to 11773\n",
      "Complement found from 12012 to 12018\n",
      "Complement found from 13796 to 13802\n",
      "Complement found from 14400 to 14406\n",
      "Complement found from 14909 to 14915\n",
      "Complement found from 16065 to 16071\n",
      "Seq is: ACTCAC\n",
      "12 ACTCAC motifs detected in file\n",
      "\n",
      "Complement found from 6461 to 6467\n",
      "Complement found from 15592 to 15598\n",
      "Seq is: TCCGTC\n",
      "2 TCCGTC motifs detected in file\n",
      "\n",
      "Complement found from 3929 to 3935\n",
      "Complement found from 7102 to 7108\n",
      "Complement found from 7757 to 7763\n",
      "Complement found from 14433 to 14439\n",
      "Seq is: CTCAGG\n",
      "4 CTCAGG motifs detected in file\n",
      "\n",
      "Complement found from 591 to 597\n",
      "Complement found from 1494 to 1500\n",
      "Complement found from 2251 to 2257\n",
      "Complement found from 2325 to 2331\n",
      "Complement found from 3327 to 3333\n",
      "Complement found from 4928 to 4934\n",
      "Complement found from 5011 to 5017\n",
      "Complement found from 5210 to 5216\n",
      "Complement found from 6367 to 6373\n",
      "Complement found from 8775 to 8781\n",
      "Complement found from 9329 to 9335\n",
      "Complement found from 12091 to 12097\n",
      "Complement found from 12981 to 12987\n",
      "Complement found from 13986 to 13992\n",
      "Complement found from 14442 to 14448\n",
      "Complement found from 15739 to 15745\n",
      "Complement found from 16443 to 16449\n",
      "Seq is: CTCCTC\n",
      "17 CTCCTC motifs detected in file\n",
      "\n",
      "Complement found from 2388 to 2394\n",
      "Complement found from 4738 to 4744\n",
      "Complement found from 4887 to 4893\n",
      "Complement found from 5370 to 5376\n",
      "Complement found from 9215 to 9221\n",
      "Complement found from 9633 to 9639\n",
      "Complement found from 10985 to 10991\n",
      "Complement found from 11627 to 11633\n",
      "Complement found from 14573 to 14579\n",
      "Complement found from 15419 to 15425\n",
      "Complement found from 15709 to 15715\n",
      "Complement found from 16213 to 16219\n",
      "Seq is: CAATCA\n",
      "12 CAATCA motifs detected in file\n",
      "\n",
      "Complement found from 520 to 526\n",
      "Complement found from 1479 to 1485\n",
      "Complement found from 7913 to 7919\n",
      "Complement found from 13934 to 13940\n",
      "Complement found from 14563 to 14569\n",
      "Seq is: ACACCG\n",
      "5 ACACCG motifs detected in file\n",
      "\n",
      "Complement found from 153 to 159\n",
      "Complement found from 448 to 454\n",
      "Complement found from 4322 to 4328\n",
      "Complement found from 10247 to 10253\n",
      "Complement found from 12197 to 12203\n",
      "Complement found from 13895 to 13901\n",
      "Seq is: TTATTT\n",
      "6 TTATTT motifs detected in file\n",
      "\n",
      "Complement found from 194 to 200\n",
      "Complement found from 1699 to 1705\n",
      "Complement found from 3765 to 3771\n",
      "Complement found from 5653 to 5659\n",
      "Complement found from 6679 to 6685\n",
      "Complement found from 10100 to 10106\n",
      "Complement found from 10789 to 10795\n",
      "Complement found from 11731 to 11737\n",
      "Complement found from 13654 to 13660\n",
      "Complement found from 13738 to 13744\n",
      "Complement found from 14633 to 14639\n",
      "Complement found from 15190 to 15196\n",
      "Complement found from 15411 to 15417\n",
      "Complement found from 15628 to 15634\n",
      "Seq is: TTACTA\n",
      "14 TTACTA motifs detected in file\n",
      "\n",
      "Complement found from 136 to 142\n",
      "Complement found from 3112 to 3118\n",
      "Complement found from 3288 to 3294\n",
      "Complement found from 3359 to 3365\n",
      "Complement found from 5113 to 5119\n",
      "Complement found from 5440 to 5446\n",
      "Complement found from 7962 to 7968\n",
      "Complement found from 8424 to 8430\n",
      "Complement found from 13310 to 13316\n",
      "Complement found from 14892 to 14898\n",
      "Seq is: ATTCCT\n",
      "10 ATTCCT motifs detected in file\n",
      "\n",
      "Complement found from 6901 to 6907\n",
      "Complement found from 7655 to 7661\n",
      "Seq is: ATGATC\n",
      "2 ATGATC motifs detected in file\n",
      "\n",
      "Complement found from 6633 to 6639\n",
      "Complement found from 7151 to 7157\n",
      "Complement found from 12492 to 12498\n",
      "Complement found from 12597 to 12603\n",
      "Complement found from 13399 to 13405\n",
      "Complement found from 15010 to 15016\n",
      "Seq is: ATATTC\n",
      "6 ATATTC motifs detected in file\n",
      "\n",
      "Complement found from 9173 to 9179\n",
      "Seq is: TCTAGT\n",
      "1 TCTAGT motifs detected in file\n",
      "\n",
      "Complement found from 1811 to 1817\n",
      "Complement found from 9152 to 9158\n",
      "Complement found from 12957 to 12963\n",
      "Seq is: CCAAGC\n",
      "3 CCAAGC motifs detected in file\n",
      "\n",
      "Complement found from 2126 to 2132\n",
      "Complement found from 2446 to 2452\n",
      "Complement found from 7760 to 7766\n",
      "Seq is: AGGAAA\n",
      "3 AGGAAA motifs detected in file\n",
      "\n",
      "Complement found from 3153 to 3159\n",
      "Complement found from 3802 to 3808\n",
      "Complement found from 5277 to 5283\n",
      "Complement found from 6396 to 6402\n",
      "Complement found from 7721 to 7727\n",
      "Complement found from 8050 to 8056\n",
      "Complement found from 9103 to 9109\n",
      "Complement found from 10982 to 10988\n",
      "Complement found from 11259 to 11265\n",
      "Complement found from 12214 to 12220\n",
      "Complement found from 15821 to 15827\n",
      "Seq is: TCACAA\n",
      "11 TCACAA motifs detected in file\n",
      "\n",
      "Complement found from 5024 to 5030\n",
      "Complement found from 9200 to 9206\n",
      "Complement found from 9400 to 9406\n",
      "Complement found from 10717 to 10723\n",
      "Complement found from 10819 to 10825\n",
      "Complement found from 11196 to 11202\n",
      "Complement found from 11524 to 11530\n",
      "Complement found from 11635 to 11641\n",
      "Complement found from 14141 to 14147\n",
      "Seq is: CACATA\n",
      "9 CACATA motifs detected in file\n",
      "\n",
      "Complement found from 963 to 969\n",
      "Complement found from 2378 to 2384\n",
      "Complement found from 2898 to 2904\n",
      "Complement found from 4728 to 4734\n",
      "Complement found from 6433 to 6439\n",
      "Complement found from 8982 to 8988\n",
      "Complement found from 10635 to 10641\n",
      "Complement found from 14251 to 14257\n",
      "Complement found from 14752 to 14758\n",
      "Seq is: CCAATA\n",
      "9 CCAATA motifs detected in file\n",
      "\n",
      "Complement found from 1430 to 1436\n",
      "Complement found from 15919 to 15925\n",
      "Seq is: GTAAAC\n",
      "2 GTAAAC motifs detected in file\n",
      "\n",
      "Complement found from 974 to 980\n",
      "Complement found from 994 to 1000\n",
      "Complement found from 1123 to 1129\n",
      "Complement found from 1158 to 1164\n",
      "Complement found from 1380 to 1386\n",
      "Complement found from 3258 to 3264\n",
      "Complement found from 3265 to 3271\n",
      "Complement found from 3464 to 3470\n",
      "Complement found from 4177 to 4183\n",
      "Complement found from 4780 to 4786\n",
      "Complement found from 4870 to 4876\n",
      "Complement found from 7728 to 7734\n",
      "Complement found from 10764 to 10770\n",
      "Complement found from 11034 to 11040\n",
      "Complement found from 11466 to 11472\n",
      "Complement found from 12421 to 12427\n",
      "Complement found from 13793 to 13799\n",
      "Complement found from 14506 to 14512\n",
      "Complement found from 14761 to 14767\n",
      "Seq is: AAAACT\n",
      "19 AAAACT motifs detected in file\n",
      "\n",
      "Complement found from 2270 to 2276\n",
      "Complement found from 4737 to 4743\n",
      "Complement found from 9214 to 9220\n",
      "Complement found from 14366 to 14372\n",
      "Complement found from 15708 to 15714\n",
      "Complement found from 16167 to 16173\n",
      "Seq is: CCAATC\n",
      "6 CCAATC motifs detected in file\n",
      "\n",
      "Complement found from 1381 to 1387\n",
      "Complement found from 1432 to 1438\n",
      "Complement found from 2781 to 2787\n",
      "Complement found from 3680 to 3686\n",
      "Complement found from 4781 to 4787\n",
      "Complement found from 4871 to 4877\n",
      "Complement found from 7729 to 7735\n",
      "Complement found from 8457 to 8463\n",
      "Complement found from 8676 to 8682\n",
      "Complement found from 10765 to 10771\n",
      "Complement found from 11467 to 11473\n",
      "Complement found from 11754 to 11760\n",
      "Complement found from 12576 to 12582\n",
      "Complement found from 14507 to 14513\n",
      "Complement found from 14762 to 14768\n",
      "Complement found from 15604 to 15610\n",
      "Complement found from 15900 to 15906\n",
      "Complement found from 16010 to 16016\n",
      "Seq is: AAACTA\n",
      "18 AAACTA motifs detected in file\n",
      "\n",
      "Complement found from 12471 to 12477\n",
      "Complement found from 12679 to 12685\n",
      "Complement found from 12798 to 12804\n",
      "Seq is: ATCAGT\n",
      "3 ATCAGT motifs detected in file\n",
      "\n",
      "Complement found from 3300 to 3306\n",
      "Complement found from 4037 to 4043\n",
      "Complement found from 5393 to 5399\n",
      "Complement found from 6194 to 6200\n",
      "Complement found from 8640 to 8646\n",
      "Complement found from 9665 to 9671\n",
      "Complement found from 10297 to 10303\n",
      "Complement found from 10894 to 10900\n",
      "Complement found from 10941 to 10947\n",
      "Complement found from 11332 to 11338\n",
      "Complement found from 12028 to 12034\n",
      "Complement found from 12247 to 12253\n",
      "Complement found from 12551 to 12557\n",
      "Complement found from 12738 to 12744\n",
      "Complement found from 13743 to 13749\n",
      "Complement found from 13772 to 13778\n",
      "Complement found from 15361 to 15367\n",
      "Complement found from 15677 to 15683\n",
      "Complement found from 16076 to 16082\n",
      "Seq is: AACAAC\n",
      "19 AACAAC motifs detected in file\n",
      "\n",
      "Complement found from 294 to 300\n",
      "Complement found from 2185 to 2191\n",
      "Complement found from 5304 to 5310\n",
      "Complement found from 5316 to 5322\n",
      "Complement found from 7887 to 7893\n",
      "Complement found from 8393 to 8399\n",
      "Complement found from 9211 to 9217\n",
      "Complement found from 9412 to 9418\n",
      "Complement found from 12018 to 12024\n",
      "Complement found from 14333 to 14339\n",
      "Complement found from 15976 to 15982\n",
      "Complement found from 16110 to 16116\n",
      "Complement found from 16399 to 16405\n",
      "Seq is: CCACCA\n",
      "13 CCACCA motifs detected in file\n",
      "\n",
      "Complement found from 475 to 481\n",
      "Complement found from 3345 to 3351\n",
      "Complement found from 5354 to 5360\n",
      "Complement found from 5707 to 5713\n",
      "Complement found from 6467 to 6473\n",
      "Complement found from 7930 to 7936\n",
      "Complement found from 8649 to 8655\n",
      "Complement found from 8670 to 8676\n",
      "Complement found from 10768 to 10774\n",
      "Complement found from 11053 to 11059\n",
      "Complement found from 11101 to 11107\n",
      "Complement found from 12720 to 12726\n",
      "Complement found from 12952 to 12958\n",
      "Complement found from 13149 to 13155\n",
      "Complement found from 14136 to 14142\n",
      "Complement found from 14216 to 14222\n",
      "Complement found from 15832 to 15838\n",
      "Seq is: CTAATC\n",
      "17 CTAATC motifs detected in file\n",
      "\n",
      "Complement found from 1717 to 1723\n",
      "Complement found from 1886 to 1892\n",
      "Complement found from 7484 to 7490\n",
      "Complement found from 10633 to 10639\n",
      "Complement found from 10996 to 11002\n",
      "Complement found from 11328 to 11334\n",
      "Complement found from 13968 to 13974\n",
      "Complement found from 15519 to 15525\n",
      "Complement found from 15706 to 15712\n",
      "Seq is: AGCCAA\n",
      "9 AGCCAA motifs detected in file\n",
      "\n",
      "Complement found from 437 to 443\n",
      "Complement found from 3399 to 3405\n",
      "Complement found from 8371 to 8377\n",
      "Complement found from 8437 to 8443\n",
      "Complement found from 8767 to 8773\n",
      "Complement found from 8814 to 8820\n",
      "Complement found from 9869 to 9875\n",
      "Complement found from 10299 to 10305\n",
      "Complement found from 13260 to 13266\n",
      "Complement found from 13847 to 13853\n",
      "Complement found from 14729 to 14735\n",
      "Complement found from 15110 to 15116\n",
      "Complement found from 15845 to 15851\n",
      "Complement found from 16224 to 16230\n",
      "Seq is: CAACTA\n",
      "14 CAACTA motifs detected in file\n",
      "\n",
      "Complement found from 6370 to 6376\n",
      "Complement found from 8608 to 8614\n",
      "Complement found from 9716 to 9722\n",
      "Complement found from 11046 to 11052\n",
      "Seq is: CTCTAT\n",
      "4 CTCTAT motifs detected in file\n",
      "\n",
      "Complement found from 1101 to 1107\n",
      "Complement found from 1673 to 1679\n",
      "Complement found from 1760 to 1766\n",
      "Complement found from 1992 to 1998\n",
      "Complement found from 2132 to 2138\n",
      "Complement found from 2789 to 2795\n",
      "Complement found from 4255 to 4261\n",
      "Complement found from 7122 to 7128\n",
      "Complement found from 7372 to 7378\n",
      "Complement found from 8825 to 8831\n",
      "Complement found from 10747 to 10753\n",
      "Complement found from 13973 to 13979\n",
      "Complement found from 15077 to 15083\n",
      "Complement found from 15934 to 15940\n",
      "Complement found from 16282 to 16288\n",
      "Seq is: AAACCT\n",
      "15 AAACCT motifs detected in file\n",
      "\n",
      "Complement found from 3210 to 3216\n",
      "Complement found from 3793 to 3799\n",
      "Complement found from 5205 to 5211\n",
      "Complement found from 5433 to 5439\n",
      "Complement found from 6313 to 6319\n",
      "Complement found from 7400 to 7406\n",
      "Complement found from 8656 to 8662\n",
      "Complement found from 8809 to 8815\n",
      "Complement found from 9055 to 9061\n",
      "Complement found from 10158 to 10164\n",
      "Complement found from 10841 to 10847\n",
      "Complement found from 12358 to 12364\n",
      "Complement found from 12397 to 12403\n",
      "Complement found from 13022 to 13028\n",
      "Complement found from 13054 to 13060\n",
      "Complement found from 13124 to 13130\n",
      "Complement found from 13648 to 13654\n",
      "Complement found from 13678 to 13684\n",
      "Complement found from 14336 to 14342\n",
      "Complement found from 14810 to 14816\n",
      "Complement found from 15263 to 15269\n",
      "Complement found from 15405 to 15411\n",
      "Complement found from 16051 to 16057\n",
      "Complement found from 16255 to 16261\n",
      "Complement found from 16290 to 16296\n",
      "Seq is: CCACCC\n",
      "25 CCACCC motifs detected in file\n",
      "\n",
      "Complement found from 6146 to 6152\n",
      "Complement found from 8946 to 8952\n",
      "Complement found from 10029 to 10035\n",
      "Complement found from 15223 to 15229\n",
      "Seq is: CTAGTT\n",
      "4 CTAGTT motifs detected in file\n",
      "\n",
      "Complement found from 3864 to 3870\n",
      "Complement found from 4236 to 4242\n",
      "Complement found from 5483 to 5489\n",
      "Complement found from 6662 to 6668\n",
      "Complement found from 10709 to 10715\n",
      "Complement found from 11056 to 11062\n",
      "Complement found from 11068 to 11074\n",
      "Complement found from 14046 to 14052\n",
      "Complement found from 14824 to 14830\n",
      "Complement found from 15850 to 15856\n",
      "Seq is: ATCTCC\n",
      "10 ATCTCC motifs detected in file\n",
      "\n",
      "Complement found from 4796 to 4802\n",
      "Complement found from 5488 to 5494\n",
      "Complement found from 7706 to 7712\n",
      "Complement found from 10204 to 10210\n",
      "Complement found from 15780 to 15786\n",
      "Seq is: CCCTTT\n",
      "5 CCCTTT motifs detected in file\n",
      "\n",
      "Complement found from 2033 to 2039\n",
      "Complement found from 2481 to 2487\n",
      "Complement found from 4943 to 4949\n",
      "Complement found from 5000 to 5006\n",
      "Complement found from 5504 to 5510\n",
      "Complement found from 6374 to 6380\n",
      "Complement found from 7579 to 7585\n",
      "Complement found from 12723 to 12729\n",
      "Seq is: ATCTTA\n",
      "8 ATCTTA motifs detected in file\n",
      "\n",
      "Complement found from 3924 to 3930\n",
      "Complement found from 7693 to 7699\n",
      "Complement found from 7801 to 7807\n",
      "Complement found from 10658 to 10664\n",
      "Complement found from 10700 to 10706\n",
      "Complement found from 11959 to 11965\n",
      "Seq is: CTAGTC\n",
      "6 CTAGTC motifs detected in file\n",
      "\n",
      "Complement found from 4045 to 4051\n",
      "Seq is: ATGACG\n",
      "1 ATGACG motifs detected in file\n",
      "\n",
      "Complement found from 496 to 502\n",
      "Complement found from 1482 to 1488\n",
      "Complement found from 3523 to 3529\n",
      "Complement found from 10194 to 10200\n",
      "Seq is: CCGCCC\n",
      "4 CCGCCC motifs detected in file\n",
      "\n",
      "Complement found from 4843 to 4849\n",
      "Complement found from 7201 to 7207\n",
      "Seq is: ATCCGG\n",
      "2 ATCCGG motifs detected in file\n",
      "\n",
      "Complement found from 147 to 153\n",
      "Complement found from 502 to 508\n",
      "Complement found from 2945 to 2951\n",
      "Complement found from 4394 to 4400\n",
      "Complement found from 5096 to 5102\n",
      "Complement found from 6641 to 6647\n",
      "Complement found from 7256 to 7262\n",
      "Complement found from 7570 to 7576\n",
      "Complement found from 7798 to 7804\n",
      "Complement found from 8565 to 8571\n",
      "Complement found from 9126 to 9132\n",
      "Complement found from 9455 to 9461\n",
      "Complement found from 10346 to 10352\n",
      "Complement found from 12849 to 12855\n",
      "Complement found from 12903 to 12909\n",
      "Complement found from 14121 to 14127\n",
      "Complement found from 14369 to 14375\n",
      "Complement found from 15643 to 15649\n",
      "Complement found from 15829 to 15835\n",
      "Complement found from 15835 to 15841\n",
      "Seq is: ATCCTA\n",
      "20 ATCCTA motifs detected in file\n",
      "\n",
      "Complement found from 564 to 570\n",
      "Complement found from 798 to 804\n",
      "Complement found from 1942 to 1948\n",
      "Complement found from 2160 to 2166\n",
      "Complement found from 2213 to 2219\n",
      "Complement found from 2257 to 2263\n",
      "Complement found from 3206 to 3212\n",
      "Complement found from 4009 to 4015\n",
      "Complement found from 4387 to 4393\n",
      "Complement found from 5189 to 5195\n",
      "Complement found from 7110 to 7116\n",
      "Complement found from 8928 to 8934\n",
      "Complement found from 10086 to 10092\n",
      "Complement found from 10617 to 10623\n",
      "Complement found from 11213 to 11219\n",
      "Complement found from 11264 to 11270\n",
      "Complement found from 12059 to 12065\n",
      "Complement found from 14553 to 14559\n",
      "Complement found from 15533 to 15539\n",
      "Seq is: ACACCC\n",
      "19 ACACCC motifs detected in file\n",
      "\n",
      "Complement found from 10011 to 10017\n",
      "Seq is: CCGTTA\n",
      "1 CCGTTA motifs detected in file\n",
      "\n",
      "Complement found from 193 to 199\n",
      "Complement found from 1698 to 1704\n",
      "Complement found from 2658 to 2664\n",
      "Complement found from 5652 to 5658\n",
      "Complement found from 6678 to 6684\n",
      "Complement found from 9987 to 9993\n",
      "Complement found from 10099 to 10105\n",
      "Complement found from 13109 to 13115\n",
      "Complement found from 13653 to 13659\n",
      "Complement found from 15189 to 15195\n",
      "Complement found from 15410 to 15416\n",
      "Complement found from 15438 to 15444\n",
      "Seq is: CTTACT\n",
      "12 CTTACT motifs detected in file\n",
      "\n",
      "Complement found from 7329 to 7335\n",
      "Complement found from 8542 to 8548\n",
      "Complement found from 13640 to 13646\n",
      "Seq is: TCGCTT\n",
      "3 TCGCTT motifs detected in file\n",
      "\n",
      "Complement found from 8117 to 8123\n",
      "Complement found from 9654 to 9660\n",
      "Complement found from 12243 to 12249\n",
      "Seq is: GTCTAA\n",
      "3 GTCTAA motifs detected in file\n",
      "\n",
      "Complement found from 598 to 604\n",
      "Complement found from 4656 to 4662\n",
      "Complement found from 6889 to 6895\n",
      "Complement found from 12844 to 12850\n",
      "Complement found from 16201 to 16207\n",
      "Seq is: AAGCAA\n",
      "5 AAGCAA motifs detected in file\n",
      "\n",
      "Complement found from 990 to 996\n",
      "Complement found from 2148 to 2154\n",
      "Complement found from 2463 to 2469\n",
      "Complement found from 5399 to 5405\n",
      "Complement found from 5827 to 5833\n",
      "Complement found from 9236 to 9242\n",
      "Complement found from 12441 to 12447\n",
      "Seq is: GTAAAA\n",
      "7 GTAAAA motifs detected in file\n",
      "\n",
      "Complement found from 2023 to 2029\n",
      "Complement found from 3214 to 3220\n",
      "Complement found from 4087 to 4093\n",
      "Complement found from 5533 to 5539\n",
      "Complement found from 11307 to 11313\n",
      "Complement found from 12509 to 12515\n",
      "Complement found from 14405 to 14411\n",
      "Seq is: CCAAGA\n",
      "7 CCAAGA motifs detected in file\n",
      "\n",
      "Complement found from 4811 to 4817\n",
      "Complement found from 9590 to 9596\n",
      "Complement found from 10774 to 10780\n",
      "Complement found from 15260 to 15266\n",
      "Seq is: GTCCCA\n",
      "4 GTCCCA motifs detected in file\n",
      "\n",
      "Complement found from 239 to 245\n",
      "Complement found from 2967 to 2973\n",
      "Complement found from 3719 to 3725\n",
      "Complement found from 4697 to 4703\n",
      "Complement found from 5285 to 5291\n",
      "Complement found from 6400 to 6406\n",
      "Complement found from 8662 to 8668\n",
      "Complement found from 10779 to 10785\n",
      "Complement found from 12488 to 12494\n",
      "Complement found from 13386 to 13392\n",
      "Complement found from 13775 to 13781\n",
      "Complement found from 14027 to 14033\n",
      "Complement found from 14191 to 14197\n",
      "Complement found from 14571 to 14577\n",
      "Complement found from 15825 to 15831\n",
      "Seq is: AACAAT\n",
      "15 AACAAT motifs detected in file\n",
      "\n",
      "Complement found from 2349 to 2355\n",
      "Complement found from 2797 to 2803\n",
      "Complement found from 3024 to 3030\n",
      "Complement found from 3047 to 3053\n",
      "Complement found from 4975 to 4981\n",
      "Complement found from 8448 to 8454\n",
      "Complement found from 8886 to 8892\n",
      "Complement found from 10441 to 10447\n",
      "Complement found from 13695 to 13701\n",
      "Complement found from 14085 to 14091\n",
      "Complement found from 14316 to 14322\n",
      "Complement found from 14500 to 14506\n",
      "Complement found from 14512 to 14518\n",
      "Seq is: ATTAAA\n",
      "13 ATTAAA motifs detected in file\n",
      "\n",
      "Complement found from 1734 to 1740\n",
      "Complement found from 1868 to 1874\n",
      "Complement found from 3139 to 3145\n",
      "Complement found from 4299 to 4305\n",
      "Complement found from 4414 to 4420\n",
      "Complement found from 4569 to 4575\n",
      "Complement found from 4608 to 4614\n",
      "Complement found from 5403 to 5409\n",
      "Complement found from 8492 to 8498\n",
      "Complement found from 9678 to 9684\n",
      "Complement found from 12325 to 12331\n",
      "Complement found from 13670 to 13676\n",
      "Complement found from 13870 to 13876\n",
      "Complement found from 14494 to 14500\n",
      "Complement found from 16551 to 16557\n",
      "Seq is: AAATAA\n",
      "15 AAATAA motifs detected in file\n",
      "\n",
      "Complement found from 5524 to 5530\n",
      "Complement found from 8376 to 8382\n",
      "Complement found from 15869 to 15875\n",
      "Complement found from 16136 to 16142\n",
      "Seq is: AAATAC\n",
      "4 AAATAC motifs detected in file\n",
      "\n",
      "Complement found from 1903 to 1909\n",
      "Complement found from 4437 to 4443\n",
      "Complement found from 9670 to 9676\n",
      "Seq is: CCGAAA\n",
      "3 CCGAAA motifs detected in file\n",
      "\n",
      "Complement found from 25 to 31\n",
      "Complement found from 461 to 467\n",
      "Complement found from 1689 to 1695\n",
      "Complement found from 4188 to 4194\n",
      "Complement found from 5602 to 5608\n",
      "Complement found from 9316 to 9322\n",
      "Complement found from 9511 to 9517\n",
      "Complement found from 9593 to 9599\n",
      "Complement found from 10621 to 10627\n",
      "Complement found from 13076 to 13082\n",
      "Complement found from 13186 to 13192\n",
      "Complement found from 14115 to 14121\n",
      "Complement found from 14790 to 14796\n",
      "Seq is: CCACTC\n",
      "13 CCACTC motifs detected in file\n",
      "\n",
      "Complement found from 7365 to 7371\n",
      "Complement found from 7498 to 7504\n",
      "Complement found from 8623 to 8629\n",
      "Complement found from 14053 to 14059\n",
      "Complement found from 14375 to 14381\n",
      "Complement found from 14871 to 14877\n",
      "Complement found from 15663 to 15669\n",
      "Seq is: CCTCCA\n",
      "7 CCTCCA motifs detected in file\n",
      "\n",
      "Complement found from 293 to 299\n",
      "Complement found from 1693 to 1699\n",
      "Complement found from 3792 to 3798\n",
      "Complement found from 5204 to 5210\n",
      "Complement found from 5363 to 5369\n",
      "Complement found from 10157 to 10163\n",
      "Complement found from 12459 to 12465\n",
      "Complement found from 13021 to 13027\n",
      "Complement found from 13123 to 13129\n",
      "Complement found from 14049 to 14055\n",
      "Complement found from 15329 to 15335\n",
      "Complement found from 15404 to 15410\n",
      "Complement found from 15975 to 15981\n",
      "Seq is: TCCACC\n",
      "13 TCCACC motifs detected in file\n",
      "\n",
      "Complement found from 740 to 746\n",
      "Complement found from 3671 to 3677\n",
      "Complement found from 7873 to 7879\n",
      "Complement found from 8673 to 8679\n",
      "Complement found from 11316 to 11322\n",
      "Complement found from 11922 to 11928\n",
      "Complement found from 13227 to 13233\n",
      "Complement found from 15357 to 15363\n",
      "Complement found from 15421 to 15427\n",
      "Complement found from 16176 to 16182\n",
      "Seq is: ATCAAA\n",
      "10 ATCAAA motifs detected in file\n",
      "\n",
      "Complement found from 1008 to 1014\n",
      "Complement found from 1616 to 1622\n",
      "Complement found from 3154 to 3160\n",
      "Complement found from 3989 to 3995\n",
      "Complement found from 5278 to 5284\n",
      "Complement found from 5678 to 5684\n",
      "Complement found from 5936 to 5942\n",
      "Complement found from 8454 to 8460\n",
      "Complement found from 13216 to 13222\n",
      "Complement found from 13544 to 13550\n",
      "Complement found from 14622 to 14628\n",
      "Seq is: CACAAA\n",
      "11 CACAAA motifs detected in file\n",
      "\n",
      "Complement found from 1805 to 1811\n",
      "Complement found from 4000 to 4006\n",
      "Complement found from 8502 to 8508\n",
      "Complement found from 11078 to 11084\n",
      "Complement found from 11567 to 11573\n",
      "Seq is: TTATAA\n",
      "5 TTATAA motifs detected in file\n",
      "\n",
      "Complement found from 729 to 735\n",
      "Complement found from 1115 to 1121\n",
      "Complement found from 5624 to 5630\n",
      "Complement found from 5810 to 5816\n",
      "Complement found from 7876 to 7882\n",
      "Complement found from 10889 to 10895\n",
      "Complement found from 13002 to 13008\n",
      "Complement found from 14876 to 14882\n",
      "Complement found from 15395 to 15401\n",
      "Complement found from 15952 to 15958\n",
      "Complement found from 16414 to 16420\n",
      "Seq is: AAATCA\n",
      "11 AAATCA motifs detected in file\n",
      "\n",
      "Complement found from 1308 to 1314\n",
      "Complement found from 1687 to 1693\n",
      "Complement found from 2215 to 2221\n",
      "Complement found from 2768 to 2774\n",
      "Complement found from 3202 to 3208\n",
      "Complement found from 3208 to 3214\n",
      "Complement found from 5021 to 5027\n",
      "Complement found from 5431 to 5437\n",
      "Complement found from 5675 to 5681\n",
      "Complement found from 9209 to 9215\n",
      "Complement found from 10619 to 10625\n",
      "Complement found from 10843 to 10849\n",
      "Complement found from 12016 to 12022\n",
      "Complement found from 12925 to 12931\n",
      "Complement found from 13326 to 13332\n",
      "Complement found from 14356 to 14362\n",
      "Complement found from 14640 to 14646\n",
      "Complement found from 16264 to 16270\n",
      "Complement found from 16288 to 16294\n",
      "Seq is: ACCCAC\n",
      "19 ACCCAC motifs detected in file\n",
      "\n",
      "Complement found from 3325 to 3331\n",
      "Complement found from 3826 to 3832\n",
      "Complement found from 5009 to 5015\n",
      "Complement found from 5360 to 5366\n",
      "Complement found from 5379 to 5385\n",
      "Complement found from 5472 to 5478\n",
      "Complement found from 6231 to 6237\n",
      "Complement found from 6308 to 6314\n",
      "Complement found from 6683 to 6689\n",
      "Complement found from 8012 to 8018\n",
      "Complement found from 8411 to 8417\n",
      "Complement found from 10752 to 10758\n",
      "Complement found from 11810 to 11816\n",
      "Complement found from 11974 to 11980\n",
      "Complement found from 13072 to 13078\n",
      "Complement found from 13503 to 13509\n",
      "Complement found from 13984 to 13990\n",
      "Complement found from 14131 to 14137\n",
      "Complement found from 14440 to 14446\n",
      "Seq is: TACTCC\n",
      "19 TACTCC motifs detected in file\n",
      "\n",
      "Complement found from 3799 to 3805\n",
      "Complement found from 6405 to 6411\n",
      "Complement found from 7643 to 7649\n",
      "Complement found from 9094 to 9100\n",
      "Complement found from 12469 to 12475\n",
      "Seq is: TTATCA\n",
      "5 TTATCA motifs detected in file\n",
      "\n",
      "Complement found from 3358 to 3364\n",
      "Complement found from 4244 to 4250\n",
      "Complement found from 5112 to 5118\n",
      "Complement found from 5439 to 5445\n",
      "Complement found from 13309 to 13315\n",
      "Complement found from 14484 to 14490\n",
      "Complement found from 15385 to 15391\n",
      "Seq is: CATTCC\n",
      "7 CATTCC motifs detected in file\n",
      "\n",
      "Complement found from 4159 to 4165\n",
      "Complement found from 9333 to 9339\n",
      "Complement found from 12071 to 12077\n",
      "Complement found from 12426 to 12432\n",
      "Complement found from 13539 to 13545\n",
      "Complement found from 14233 to 14239\n",
      "Complement found from 14344 to 14350\n",
      "Seq is: TCATAC\n",
      "7 TCATAC motifs detected in file\n",
      "\n",
      "Complement found from 533 to 539\n",
      "Complement found from 636 to 642\n",
      "Complement found from 3572 to 3578\n",
      "Complement found from 4390 to 4396\n",
      "Complement found from 4879 to 4885\n",
      "Complement found from 5383 to 5389\n",
      "Complement found from 5436 to 5442\n",
      "Complement found from 6583 to 6589\n",
      "Complement found from 7490 to 7496\n",
      "Complement found from 7955 to 7961\n",
      "Complement found from 8028 to 8034\n",
      "Complement found from 8406 to 8412\n",
      "Complement found from 8939 to 8945\n",
      "Complement found from 11427 to 11433\n",
      "Complement found from 12084 to 12090\n",
      "Complement found from 12237 to 12243\n",
      "Complement found from 12386 to 12392\n",
      "Complement found from 12432 to 12438\n",
      "Complement found from 13691 to 13697\n",
      "Complement found from 14339 to 14345\n",
      "Complement found from 14425 to 14431\n",
      "Complement found from 14588 to 14594\n",
      "Complement found from 14628 to 14634\n",
      "Complement found from 14813 to 14819\n",
      "Complement found from 15657 to 15663\n",
      "Complement found from 16189 to 16195\n",
      "Complement found from 16362 to 16368\n",
      "Seq is: CCCCAT\n",
      "27 CCCCAT motifs detected in file\n",
      "\n",
      "Complement found from 6111 to 6117\n",
      "Complement found from 7737 to 7743\n",
      "Complement found from 10575 to 10581\n",
      "Complement found from 10955 to 10961\n",
      "Complement found from 11486 to 11492\n",
      "Complement found from 15839 to 15845\n",
      "Complement found from 15904 to 15910\n",
      "Seq is: TAATAC\n",
      "7 TAATAC motifs detected in file\n",
      "\n",
      "Complement found from 1943 to 1949\n",
      "Complement found from 11157 to 11163\n",
      "Complement found from 14554 to 14560\n",
      "Seq is: CACCCG\n",
      "3 CACCCG motifs detected in file\n",
      "\n",
      "Complement found from 70 to 76\n",
      "Complement found from 412 to 418\n",
      "Complement found from 7701 to 7707\n",
      "Seq is: GTATGC\n",
      "3 GTATGC motifs detected in file\n",
      "\n",
      "Complement found from 276 to 282\n",
      "Complement found from 3746 to 3752\n",
      "Complement found from 3835 to 3841\n",
      "Complement found from 4749 to 4755\n",
      "Complement found from 4951 to 4957\n",
      "Complement found from 5296 to 5302\n",
      "Complement found from 5308 to 5314\n",
      "Complement found from 6118 to 6124\n",
      "Complement found from 7066 to 7072\n",
      "Complement found from 7794 to 7800\n",
      "Complement found from 10342 to 10348\n",
      "Complement found from 10863 to 10869\n",
      "Complement found from 11778 to 11784\n",
      "Complement found from 12114 to 12120\n",
      "Complement found from 12632 to 12638\n",
      "Complement found from 13370 to 13376\n",
      "Complement found from 13517 to 13523\n",
      "Complement found from 14057 to 14063\n",
      "Complement found from 14342 to 14348\n",
      "Complement found from 14481 to 14487\n",
      "Complement found from 14667 to 14673\n",
      "Complement found from 15789 to 15795\n",
      "Seq is: CATCAT\n",
      "22 CATCAT motifs detected in file\n",
      "\n",
      "Complement found from 729 to 735\n",
      "Complement found from 1115 to 1121\n",
      "Complement found from 5624 to 5630\n",
      "Complement found from 5810 to 5816\n",
      "Complement found from 7876 to 7882\n",
      "Complement found from 10889 to 10895\n",
      "Complement found from 13002 to 13008\n",
      "Complement found from 14876 to 14882\n",
      "Complement found from 15395 to 15401\n",
      "Complement found from 15952 to 15958\n",
      "Complement found from 16414 to 16420\n",
      "Seq is: AAATCA\n",
      "11 AAATCA motifs detected in file\n",
      "\n",
      "Complement found from 539 to 545\n",
      "Complement found from 1221 to 1227\n",
      "Complement found from 2486 to 2492\n",
      "Complement found from 4434 to 4440\n",
      "Complement found from 6564 to 6570\n",
      "Complement found from 7230 to 7236\n",
      "Complement found from 9569 to 9575\n",
      "Complement found from 12107 to 12113\n",
      "Seq is: ACCCCG\n",
      "8 ACCCCG motifs detected in file\n",
      "\n",
      "Complement found from 198 to 204\n",
      "Complement found from 1737 to 1743\n",
      "Complement found from 3049 to 3055\n",
      "Complement found from 4398 to 4404\n",
      "Complement found from 14318 to 14324\n",
      "Complement found from 16478 to 16484\n",
      "Seq is: TAAAGT\n",
      "6 TAAAGT motifs detected in file\n",
      "\n",
      "Complement found from 1515 to 1521\n",
      "Complement found from 5942 to 5948\n",
      "Complement found from 15465 to 15471\n",
      "Seq is: GACATT\n",
      "3 GACATT motifs detected in file\n",
      "\n",
      "Complement found from 39 to 45\n",
      "Complement found from 3790 to 3796\n",
      "Complement found from 3865 to 3871\n",
      "Complement found from 4219 to 4225\n",
      "Complement found from 4237 to 4243\n",
      "Complement found from 9966 to 9972\n",
      "Complement found from 10209 to 10215\n",
      "Complement found from 10710 to 10716\n",
      "Complement found from 12589 to 12595\n",
      "Complement found from 13019 to 13025\n",
      "Complement found from 13246 to 13252\n",
      "Complement found from 13900 to 13906\n",
      "Complement found from 14047 to 14053\n",
      "Seq is: TCTCCA\n",
      "13 TCTCCA motifs detected in file\n",
      "\n",
      "Complement found from 548 to 554\n",
      "Complement found from 2392 to 2398\n",
      "Complement found from 8806 to 8812\n",
      "Complement found from 8979 to 8985\n",
      "Complement found from 9070 to 9076\n",
      "Complement found from 10838 to 10844\n",
      "Complement found from 11169 to 11175\n",
      "Complement found from 13292 to 13298\n",
      "Complement found from 14200 to 14206\n",
      "Complement found from 14330 to 14336\n",
      "Complement found from 14477 to 14483\n",
      "Complement found from 14690 to 14696\n",
      "Complement found from 15766 to 15772\n",
      "Seq is: CAACCA\n",
      "13 CAACCA motifs detected in file\n",
      "\n",
      "Complement found from 3135 to 3141\n",
      "Complement found from 8338 to 8344\n",
      "Complement found from 15957 to 15963\n",
      "Seq is: AGAGAA\n",
      "3 AGAGAA motifs detected in file\n",
      "\n",
      "Complement found from 7523 to 7529\n",
      "Complement found from 14608 to 14614\n",
      "Seq is: TTAGAA\n",
      "2 TTAGAA motifs detected in file\n",
      "\n",
      "Complement found from 243 to 249\n",
      "Complement found from 1756 to 1762\n",
      "Complement found from 8020 to 8026\n",
      "Complement found from 15859 to 15865\n",
      "Seq is: ATTGAA\n",
      "4 ATTGAA motifs detected in file\n",
      "\n",
      "Complement found from 451 to 457\n",
      "Complement found from 7676 to 7682\n",
      "Complement found from 7709 to 7715\n",
      "Complement found from 10924 to 10930\n",
      "Complement found from 12127 to 12133\n",
      "Complement found from 15940 to 15946\n",
      "Seq is: TTTTCC\n",
      "6 TTTTCC motifs detected in file\n",
      "\n",
      "Complement found from 5017 to 5023\n",
      "Complement found from 8043 to 8049\n",
      "Complement found from 8400 to 8406\n",
      "Complement found from 12710 to 12716\n",
      "Complement found from 14169 to 14175\n",
      "Complement found from 15180 to 15186\n",
      "Seq is: AATTAC\n",
      "6 AATTAC motifs detected in file\n",
      "\n",
      "Complement found from 6930 to 6936\n",
      "Seq is: GATTCA\n",
      "1 GATTCA motifs detected in file\n",
      "\n",
      "Complement found from 3942 to 3948\n",
      "Complement found from 4342 to 4348\n",
      "Complement found from 5267 to 5273\n",
      "Complement found from 7459 to 7465\n",
      "Complement found from 8955 to 8961\n",
      "Complement found from 13521 to 13527\n",
      "Seq is: ATCGAA\n",
      "6 ATCGAA motifs detected in file\n",
      "\n",
      "Complement found from 10322 to 10328\n",
      "Seq is: ATGTCA\n",
      "1 ATGTCA motifs detected in file\n",
      "\n",
      "Complement found from 12814 to 12820\n",
      "Complement found from 14156 to 14162\n",
      "Seq is: CCCGAG\n",
      "2 CCCGAG motifs detected in file\n",
      "\n",
      "Complement found from 3864 to 3870\n",
      "Complement found from 4236 to 4242\n",
      "Complement found from 5483 to 5489\n",
      "Complement found from 6662 to 6668\n",
      "Complement found from 10709 to 10715\n",
      "Complement found from 11056 to 11062\n",
      "Complement found from 11068 to 11074\n",
      "Complement found from 14046 to 14052\n",
      "Complement found from 14824 to 14830\n",
      "Complement found from 15850 to 15856\n",
      "Seq is: ATCTCC\n",
      "10 ATCTCC motifs detected in file\n",
      "\n",
      "Complement found from 499 to 505\n",
      "Complement found from 4348 to 4354\n",
      "Complement found from 4391 to 4397\n",
      "Complement found from 4880 to 4886\n",
      "Complement found from 6116 to 6122\n",
      "Complement found from 7818 to 7824\n",
      "Complement found from 8206 to 8212\n",
      "Complement found from 11428 to 11434\n",
      "Complement found from 12387 to 12393\n",
      "Complement found from 14340 to 14346\n",
      "Complement found from 14814 to 14820\n",
      "Complement found from 15658 to 15664\n",
      "Complement found from 16070 to 16076\n",
      "Seq is: CCCATC\n",
      "13 CCCATC motifs detected in file\n",
      "\n",
      "Complement found from 309 to 315\n",
      "Complement found from 3166 to 3172\n",
      "Complement found from 4247 to 4253\n",
      "Complement found from 7953 to 7959\n",
      "Complement found from 8603 to 8609\n",
      "Complement found from 10190 to 10196\n",
      "Complement found from 12082 to 12088\n",
      "Complement found from 12383 to 12389\n",
      "Complement found from 13752 to 13758\n",
      "Complement found from 13761 to 13767\n",
      "Complement found from 13780 to 13786\n",
      "Complement found from 14153 to 14159\n",
      "Complement found from 14487 to 14493\n",
      "Complement found from 14529 to 14535\n",
      "Seq is: TCCCCC\n",
      "14 TCCCCC motifs detected in file\n",
      "\n",
      "Complement found from 1386 to 1392\n",
      "Seq is: ACGATA\n",
      "1 ACGATA motifs detected in file\n",
      "\n",
      "Complement found from 465 to 471\n",
      "Complement found from 6665 to 6671\n",
      "Complement found from 7817 to 7823\n",
      "Complement found from 15203 to 15209\n",
      "Complement found from 15382 to 15388\n",
      "Seq is: TCCCAT\n",
      "5 TCCCAT motifs detected in file\n",
      "\n",
      "Seq is: TTATGC\n",
      "0 TTATGC motifs detected in file\n",
      "\n",
      "Complement found from 5741 to 5747\n",
      "Seq is: CCGGGA\n",
      "1 CCGGGA motifs detected in file\n",
      "\n",
      "Complement found from 198 to 204\n",
      "Complement found from 1737 to 1743\n",
      "Complement found from 3049 to 3055\n",
      "Complement found from 4398 to 4404\n",
      "Complement found from 14318 to 14324\n",
      "Complement found from 16478 to 16484\n",
      "Seq is: TAAAGT\n",
      "6 TAAAGT motifs detected in file\n",
      "\n",
      "Complement found from 3343 to 3349\n",
      "Complement found from 4099 to 4105\n",
      "Complement found from 4596 to 4602\n",
      "Complement found from 4677 to 4683\n",
      "Complement found from 9109 to 9115\n",
      "Complement found from 12943 to 12949\n",
      "Complement found from 13828 to 13834\n",
      "Complement found from 15746 to 15752\n",
      "Complement found from 16001 to 16007\n",
      "Seq is: TTCTAA\n",
      "9 TTCTAA motifs detected in file\n",
      "\n",
      "Complement found from 2071 to 2077\n",
      "Complement found from 2229 to 2235\n",
      "Complement found from 7134 to 7140\n",
      "Complement found from 7568 to 7574\n",
      "Complement found from 9577 to 9583\n",
      "Complement found from 10154 to 10160\n",
      "Complement found from 12444 to 12450\n",
      "Complement found from 13875 to 13881\n",
      "Complement found from 16348 to 16354\n",
      "Seq is: AAATCC\n",
      "9 AAATCC motifs detected in file\n",
      "\n",
      "Complement found from 2610 to 2616\n",
      "Complement found from 5193 to 5199\n",
      "Complement found from 5329 to 5335\n",
      "Complement found from 8746 to 8752\n",
      "Complement found from 9145 to 9151\n",
      "Complement found from 10068 to 10074\n",
      "Complement found from 11072 to 11078\n",
      "Complement found from 13382 to 13388\n",
      "Complement found from 15458 to 15464\n",
      "Complement found from 15527 to 15533\n",
      "Complement found from 16294 to 16300\n",
      "Complement found from 16547 to 16553\n",
      "Seq is: CCTTAA\n",
      "12 CCTTAA motifs detected in file\n",
      "\n",
      "Complement found from 2192 to 2198\n",
      "Complement found from 3230 to 3236\n",
      "Complement found from 8335 to 8341\n",
      "Seq is: TTAAGA\n",
      "3 TTAAGA motifs detected in file\n",
      "\n",
      "Complement found from 138 to 144\n",
      "Complement found from 3829 to 3835\n",
      "Complement found from 6234 to 6240\n",
      "Complement found from 7784 to 7790\n",
      "Complement found from 8785 to 8791\n",
      "Complement found from 13312 to 13318\n",
      "Complement found from 15101 to 15107\n",
      "Seq is: TCCTGC\n",
      "7 TCCTGC motifs detected in file\n",
      "\n",
      "Complement found from 8575 to 8581\n",
      "Seq is: TACCCG\n",
      "1 TACCCG motifs detected in file\n",
      "\n",
      "Complement found from 2660 to 2666\n",
      "Complement found from 4285 to 4291\n",
      "Seq is: TACTTT\n",
      "2 TACTTT motifs detected in file\n",
      "\n",
      "Complement found from 563 to 569\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Seq is: GACACC\n",
      "1 GACACC motifs detected in file\n",
      "\n",
      "Complement found from 838 to 844\n",
      "Complement found from 10422 to 10428\n",
      "Seq is: AAACGA\n",
      "2 AAACGA motifs detected in file\n",
      "\n",
      "Complement found from 6362 to 6368\n",
      "Seq is: GGTGTC\n",
      "1 GGTGTC motifs detected in file\n",
      "\n",
      "Complement found from 198 to 204\n",
      "Complement found from 1737 to 1743\n",
      "Complement found from 3049 to 3055\n",
      "Complement found from 4398 to 4404\n",
      "Complement found from 14318 to 14324\n",
      "Complement found from 16478 to 16484\n",
      "Seq is: TAAAGT\n",
      "6 TAAAGT motifs detected in file\n",
      "\n",
      "Complement found from 11644 to 11650\n",
      "Seq is: CTCGTA\n",
      "1 CTCGTA motifs detected in file\n",
      "\n",
      "Complement found from 5714 to 5720\n",
      "Complement found from 6962 to 6968\n",
      "Complement found from 9549 to 9555\n",
      "Complement found from 9927 to 9933\n",
      "Seq is: ACTGGC\n",
      "4 ACTGGC motifs detected in file\n",
      "\n",
      "Complement found from 277 to 283\n",
      "Complement found from 4760 to 4766\n",
      "Complement found from 4889 to 4895\n",
      "Complement found from 4952 to 4958\n",
      "Complement found from 5309 to 5315\n",
      "Complement found from 6119 to 6125\n",
      "Complement found from 7067 to 7073\n",
      "Complement found from 7148 to 7154\n",
      "Complement found from 7630 to 7636\n",
      "Complement found from 9228 to 9234\n",
      "Complement found from 10455 to 10461\n",
      "Complement found from 11104 to 11110\n",
      "Complement found from 11779 to 11785\n",
      "Complement found from 12633 to 12639\n",
      "Complement found from 13538 to 13544\n",
      "Complement found from 14232 to 14238\n",
      "Complement found from 14343 to 14349\n",
      "Seq is: ATCATA\n",
      "17 ATCATA motifs detected in file\n",
      "\n",
      "Complement found from 64 to 70\n",
      "Complement found from 16468 to 16474\n",
      "Seq is: TGGGGG\n",
      "2 TGGGGG motifs detected in file\n",
      "\n",
      "Complement found from 509 to 515\n",
      "Complement found from 4240 to 4246\n",
      "Complement found from 5136 to 5142\n",
      "Seq is: CCAGCA\n",
      "3 CCAGCA motifs detected in file\n",
      "\n",
      "Complement found from 3545 to 3551\n",
      "Complement found from 5455 to 5461\n",
      "Complement found from 6835 to 6841\n",
      "Complement found from 7809 to 7815\n",
      "Complement found from 11241 to 11247\n",
      "Complement found from 11430 to 11436\n",
      "Complement found from 13574 to 13580\n",
      "Complement found from 14379 to 14385\n",
      "Complement found from 14454 to 14460\n",
      "Seq is: CATCGC\n",
      "9 CATCGC motifs detected in file\n",
      "\n",
      "Complement found from 261 to 267\n",
      "Complement found from 417 to 423\n",
      "Complement found from 5632 to 5638\n",
      "Complement found from 7187 to 7193\n",
      "Complement found from 8130 to 8136\n",
      "Complement found from 9881 to 9887\n",
      "Complement found from 9899 to 9905\n",
      "Complement found from 13814 to 13820\n",
      "Complement found from 15713 to 15719\n",
      "Seq is: CACTTT\n",
      "9 CACTTT motifs detected in file\n",
      "\n",
      "Complement found from 295 to 301\n",
      "Complement found from 2186 to 2192\n",
      "Complement found from 3473 to 3479\n",
      "Complement found from 4085 to 4091\n",
      "Complement found from 7888 to 7894\n",
      "Complement found from 8477 to 8483\n",
      "Complement found from 8803 to 8809\n",
      "Complement found from 9212 to 9218\n",
      "Complement found from 14040 to 14046\n",
      "Complement found from 14183 to 14189\n",
      "Complement found from 14249 to 14255\n",
      "Complement found from 14364 to 14370\n",
      "Complement found from 14403 to 14409\n",
      "Complement found from 14741 to 14747\n",
      "Seq is: CACCAA\n",
      "14 CACCAA motifs detected in file\n",
      "\n",
      "Complement found from 296 to 302\n",
      "Complement found from 550 to 556\n",
      "Complement found from 2499 to 2505\n",
      "Complement found from 2786 to 2792\n",
      "Complement found from 3474 to 3480\n",
      "Complement found from 4601 to 4607\n",
      "Complement found from 4896 to 4902\n",
      "Complement found from 4980 to 4986\n",
      "Complement found from 6438 to 6444\n",
      "Complement found from 7119 to 7125\n",
      "Complement found from 8123 to 8129\n",
      "Complement found from 8478 to 8484\n",
      "Complement found from 8520 to 8526\n",
      "Complement found from 9675 to 9681\n",
      "Complement found from 10464 to 10470\n",
      "Complement found from 10886 to 10892\n",
      "Complement found from 12151 to 12157\n",
      "Complement found from 14041 to 14047\n",
      "Seq is: ACCAAA\n",
      "18 ACCAAA motifs detected in file\n",
      "\n",
      "Complement found from 251 to 257\n",
      "Complement found from 1847 to 1853\n",
      "Complement found from 5606 to 5612\n",
      "Seq is: TCTGCA\n",
      "3 TCTGCA motifs detected in file\n",
      "\n",
      "Complement found from 1462 to 1468\n",
      "Complement found from 9265 to 9271\n",
      "Seq is: GGCCCT\n",
      "2 GGCCCT motifs detected in file\n",
      "\n",
      "Seq is: TAACGT\n",
      "0 TAACGT motifs detected in file\n",
      "\n",
      "Complement found from 1552 to 1558\n",
      "Complement found from 1707 to 1713\n",
      "Complement found from 7442 to 7448\n",
      "Complement found from 14474 to 14480\n",
      "Complement found from 15504 to 15510\n",
      "Seq is: AGACAA\n",
      "5 AGACAA motifs detected in file\n",
      "\n",
      "Complement found from 3110 to 3116\n",
      "Complement found from 3382 to 3388\n",
      "Complement found from 4366 to 4372\n",
      "Complement found from 10217 to 10223\n",
      "Complement found from 14536 to 14542\n",
      "Seq is: AAATTC\n",
      "5 AAATTC motifs detected in file\n",
      "\n",
      "Complement found from 1072 to 1078\n",
      "Complement found from 4973 to 4979\n",
      "Complement found from 10385 to 10391\n",
      "Seq is: GGATTA\n",
      "3 GGATTA motifs detected in file\n",
      "\n",
      "Complement found from 430 to 436\n",
      "Complement found from 565 to 571\n",
      "Complement found from 634 to 640\n",
      "Complement found from 799 to 805\n",
      "Complement found from 953 to 959\n",
      "Complement found from 4388 to 4394\n",
      "Complement found from 4831 to 4837\n",
      "Complement found from 5434 to 5440\n",
      "Complement found from 5892 to 5898\n",
      "Complement found from 8269 to 8275\n",
      "Complement found from 8929 to 8935\n",
      "Complement found from 9568 to 9574\n",
      "Complement found from 10159 to 10165\n",
      "Complement found from 12965 to 12971\n",
      "Complement found from 13023 to 13029\n",
      "Complement found from 13055 to 13061\n",
      "Complement found from 13125 to 13131\n",
      "Complement found from 14337 to 14343\n",
      "Complement found from 14811 to 14817\n",
      "Complement found from 15534 to 15540\n",
      "Complement found from 16256 to 16262\n",
      "Seq is: CACCCC\n",
      "21 CACCCC motifs detected in file\n",
      "\n",
      "Complement found from 11418 to 11424\n",
      "Seq is: TGTCGA\n",
      "1 TGTCGA motifs detected in file\n",
      "\n",
      "Complement found from 8011 to 8017\n",
      "Complement found from 11458 to 11464\n",
      "Seq is: GTACTC\n",
      "2 GTACTC motifs detected in file\n",
      "\n",
      "Complement found from 168 to 174\n",
      "Complement found from 11911 to 11917\n",
      "Complement found from 16540 to 16546\n",
      "Seq is: ACGTTC\n",
      "3 ACGTTC motifs detected in file\n",
      "\n",
      "Seq is: TGCAGA\n",
      "0 TGCAGA motifs detected in file\n",
      "\n",
      "Complement found from 1139 to 1145\n",
      "Complement found from 2121 to 2127\n",
      "Complement found from 3870 to 3876\n",
      "Complement found from 5375 to 5381\n",
      "Complement found from 5951 to 5957\n",
      "Complement found from 8419 to 8425\n",
      "Complement found from 8710 to 8716\n",
      "Complement found from 9356 to 9362\n",
      "Complement found from 12346 to 12352\n",
      "Complement found from 13164 to 13170\n",
      "Complement found from 14311 to 14317\n",
      "Complement found from 15472 to 15478\n",
      "Seq is: ACACTA\n",
      "12 ACACTA motifs detected in file\n",
      "\n",
      "Complement found from 2796 to 2802\n",
      "Complement found from 4752 to 4758\n",
      "Complement found from 8313 to 8319\n",
      "Complement found from 9074 to 9080\n",
      "Complement found from 10440 to 10446\n",
      "Complement found from 12024 to 12030\n",
      "Complement found from 12674 to 12680\n",
      "Complement found from 13661 to 13667\n",
      "Complement found from 13694 to 13700\n",
      "Complement found from 15467 to 15473\n",
      "Seq is: CATTAA\n",
      "10 CATTAA motifs detected in file\n",
      "\n",
      "Complement found from 1818 to 1824\n",
      "Complement found from 3853 to 3859\n",
      "Complement found from 7296 to 7302\n",
      "Complement found from 9873 to 9879\n",
      "Complement found from 11339 to 11345\n",
      "Complement found from 12595 to 12601\n",
      "Complement found from 15689 to 15695\n",
      "Seq is: TAATAT\n",
      "7 TAATAT motifs detected in file\n",
      "\n",
      "Complement found from 585 to 591\n",
      "Complement found from 3368 to 3374\n",
      "Complement found from 11350 to 11356\n",
      "Complement found from 11718 to 11724\n",
      "Complement found from 12185 to 12191\n",
      "Complement found from 15437 to 15443\n",
      "Complement found from 16195 to 16201\n",
      "Seq is: GCTTAC\n",
      "7 GCTTAC motifs detected in file\n",
      "\n",
      "Complement found from 540 to 546\n",
      "Complement found from 1222 to 1228\n",
      "Complement found from 1901 to 1907\n",
      "Complement found from 3526 to 3532\n",
      "Complement found from 4137 to 4143\n",
      "Complement found from 4435 to 4441\n",
      "Complement found from 6169 to 6175\n",
      "Complement found from 7211 to 7217\n",
      "Complement found from 7231 to 7237\n",
      "Complement found from 12108 to 12114\n",
      "Complement found from 14155 to 14161\n",
      "Seq is: CCCCGA\n",
      "11 CCCCGA motifs detected in file\n",
      "\n",
      "Seq is: AGTTAG\n",
      "0 AGTTAG motifs detected in file\n",
      "\n",
      "Complement found from 2412 to 2418\n",
      "Complement found from 4012 to 4018\n",
      "Complement found from 4250 to 4256\n",
      "Complement found from 4907 to 4913\n",
      "Complement found from 5549 to 5555\n",
      "Complement found from 7664 to 7670\n",
      "Complement found from 8473 to 8479\n",
      "Complement found from 10473 to 10479\n",
      "Complement found from 10611 to 10617\n",
      "Complement found from 10979 to 10985\n",
      "Complement found from 12040 to 12046\n",
      "Complement found from 12062 to 12068\n",
      "Complement found from 12101 to 12107\n",
      "Complement found from 13035 to 13041\n",
      "Complement found from 13450 to 13456\n",
      "Complement found from 15266 to 15272\n",
      "Complement found from 16220 to 16226\n",
      "Complement found from 16259 to 16265\n",
      "Complement found from 16377 to 16383\n",
      "Seq is: CCCTCA\n",
      "19 CCCTCA motifs detected in file\n",
      "\n",
      "Complement found from 1280 to 1286\n",
      "Complement found from 9977 to 9983\n",
      "Complement found from 11830 to 11836\n",
      "Complement found from 12804 to 12810\n",
      "Complement found from 14833 to 14839\n",
      "Seq is: TGATGA\n",
      "5 TGATGA motifs detected in file\n",
      "\n",
      "Complement found from 528 to 534\n",
      "Complement found from 5176 to 5182\n",
      "Complement found from 5236 to 5242\n",
      "Complement found from 8303 to 8309\n",
      "Complement found from 9009 to 9015\n",
      "Complement found from 11851 to 11857\n",
      "Complement found from 12225 to 12231\n",
      "Complement found from 12735 to 12741\n",
      "Complement found from 14383 to 14389\n",
      "Complement found from 14568 to 14574\n",
      "Seq is: GCTAAC\n",
      "10 GCTAAC motifs detected in file\n",
      "\n",
      "Complement found from 5558 to 5564\n",
      "Seq is: AGTTGC\n",
      "1 AGTTGC motifs detected in file\n",
      "\n",
      "Complement found from 6148 to 6154\n",
      "Seq is: AGTTCC\n",
      "1 AGTTCC motifs detected in file\n",
      "\n",
      "Seq is: TCAGCG\n",
      "0 TCAGCG motifs detected in file\n",
      "\n",
      "Complement found from 4239 to 4245\n",
      "Complement found from 5135 to 5141\n",
      "Complement found from 9515 to 9521\n",
      "Seq is: TCCAGC\n",
      "3 TCCAGC motifs detected in file\n",
      "\n",
      "Complement found from 2267 to 2273\n",
      "Seq is: GGACCA\n",
      "1 GGACCA motifs detected in file\n",
      "\n",
      "Seq is: AGATCC\n",
      "0 AGATCC motifs detected in file\n",
      "\n",
      "Complement found from 2406 to 2412\n",
      "Complement found from 2742 to 2748\n",
      "Complement found from 3997 to 4003\n",
      "Complement found from 8950 to 8956\n",
      "Complement found from 9462 to 9468\n",
      "Complement found from 9696 to 9702\n",
      "Complement found from 10113 to 10119\n",
      "Complement found from 10244 to 10250\n",
      "Complement found from 10334 to 10340\n",
      "Complement found from 10856 to 10862\n",
      "Complement found from 12466 to 12472\n",
      "Complement found from 12517 to 12523\n",
      "Seq is: TTATTA\n",
      "12 TTATTA motifs detected in file\n",
      "\n",
      "Complement found from 304 to 310\n",
      "Complement found from 955 to 961\n",
      "Complement found from 3566 to 3572\n",
      "Complement found from 3584 to 3590\n",
      "Complement found from 3892 to 3898\n",
      "Complement found from 4248 to 4254\n",
      "Complement found from 4317 to 4323\n",
      "Complement found from 4794 to 4800\n",
      "Complement found from 6419 to 6425\n",
      "Complement found from 8271 to 8277\n",
      "Complement found from 8280 to 8286\n",
      "Complement found from 8604 to 8610\n",
      "Complement found from 10935 to 10941\n",
      "Complement found from 10947 to 10953\n",
      "Complement found from 11511 to 11517\n",
      "Complement found from 11672 to 11678\n",
      "Complement found from 13127 to 13133\n",
      "Complement found from 13762 to 13768\n",
      "Complement found from 13781 to 13787\n",
      "Complement found from 14488 to 14494\n",
      "Complement found from 14769 to 14775\n",
      "Complement found from 15366 to 15372\n",
      "Complement found from 16183 to 16189\n",
      "Complement found from 16375 to 16381\n",
      "Seq is: CCCCCT\n",
      "24 CCCCCT motifs detected in file\n",
      "\n",
      "Complement found from 4159 to 4165\n",
      "Complement found from 9333 to 9339\n",
      "Complement found from 12071 to 12077\n",
      "Complement found from 12426 to 12432\n",
      "Complement found from 13539 to 13545\n",
      "Complement found from 14233 to 14239\n",
      "Complement found from 14344 to 14350\n",
      "Seq is: TCATAC\n",
      "7 TCATAC motifs detected in file\n",
      "\n",
      "Complement found from 3651 to 3657\n",
      "Complement found from 4688 to 4694\n",
      "Complement found from 10549 to 10555\n",
      "Complement found from 11724 to 11730\n",
      "Complement found from 11785 to 11791\n",
      "Complement found from 12882 to 12888\n",
      "Complement found from 14259 to 14265\n",
      "Complement found from 14869 to 14875\n",
      "Complement found from 15097 to 15103\n",
      "Complement found from 15637 to 15643\n",
      "Complement found from 15661 to 15667\n",
      "Complement found from 16404 to 16410\n",
      "Seq is: ATCCTC\n",
      "12 ATCCTC motifs detected in file\n",
      "\n",
      "Complement found from 4179 to 4185\n",
      "Complement found from 10016 to 10022\n",
      "Complement found from 10061 to 10067\n",
      "Complement found from 14839 to 14845\n",
      "Seq is: AACTTC\n",
      "4 AACTTC motifs detected in file\n",
      "\n",
      "Complement found from 2598 to 2604\n",
      "Complement found from 4758 to 4764\n",
      "Complement found from 5708 to 5714\n",
      "Complement found from 6468 to 6474\n",
      "Complement found from 6831 to 6837\n",
      "Complement found from 7670 to 7676\n",
      "Complement found from 8650 to 8656\n",
      "Complement found from 8671 to 8677\n",
      "Complement found from 8749 to 8755\n",
      "Complement found from 10080 to 10086\n",
      "Complement found from 10338 to 10344\n",
      "Complement found from 10453 to 10459\n",
      "Complement found from 11102 to 11108\n",
      "Complement found from 12677 to 12683\n",
      "Complement found from 14137 to 14143\n",
      "Complement found from 14217 to 14223\n",
      "Complement found from 14230 to 14236\n",
      "Seq is: TAATCA\n",
      "17 TAATCA motifs detected in file\n",
      "\n",
      "Complement found from 11473 to 11479\n",
      "Seq is: GGCGGC\n",
      "1 GGCGGC motifs detected in file\n",
      "\n",
      "Complement found from 5626 to 5632\n",
      "Complement found from 8964 to 8970\n",
      "Complement found from 11629 to 11635\n",
      "Complement found from 13004 to 13010\n",
      "Seq is: ATCAGC\n",
      "4 ATCAGC motifs detected in file\n",
      "\n",
      "Complement found from 691 to 697\n",
      "Complement found from 2554 to 2560\n",
      "Complement found from 4861 to 4867\n",
      "Complement found from 7246 to 7252\n",
      "Complement found from 7589 to 7595\n",
      "Complement found from 9219 to 9225\n",
      "Seq is: CACATG\n",
      "6 CACATG motifs detected in file\n",
      "\n",
      "Complement found from 699 to 705\n",
      "Complement found from 753 to 759\n",
      "Complement found from 2517 to 2523\n",
      "Complement found from 3668 to 3674\n",
      "Complement found from 10861 to 10867\n",
      "Complement found from 13926 to 13932\n",
      "Complement found from 15804 to 15810\n",
      "Seq is: AGCATC\n",
      "7 AGCATC motifs detected in file\n",
      "\n",
      "Complement found from 916 to 922\n",
      "Complement found from 1555 to 1561\n",
      "Complement found from 2399 to 2405\n",
      "Complement found from 13255 to 13261\n",
      "Seq is: CAAGTC\n",
      "4 CAAGTC motifs detected in file\n",
      "\n",
      "Complement found from 1566 to 1572\n",
      "Complement found from 7893 to 7899\n",
      "Complement found from 11480 to 11486\n",
      "Seq is: ATGGTA\n",
      "3 ATGGTA motifs detected in file\n",
      "\n",
      "Complement found from 1236 to 1242\n",
      "Complement found from 6551 to 6557\n",
      "Complement found from 8462 to 8468\n",
      "Complement found from 8655 to 8661\n",
      "Complement found from 8808 to 8814\n",
      "Complement found from 9421 to 9427\n",
      "Complement found from 10840 to 10846\n",
      "Complement found from 12357 to 12363\n",
      "Complement found from 12396 to 12402\n",
      "Complement found from 14332 to 14338\n",
      "Complement found from 16050 to 16056\n",
      "Complement found from 16145 to 16151\n",
      "Complement found from 16398 to 16404\n",
      "Seq is: ACCACC\n",
      "13 ACCACC motifs detected in file\n",
      "\n",
      "Complement found from 2452 to 2458\n",
      "Complement found from 5520 to 5526\n",
      "Seq is: GGTTAA\n",
      "2 GGTTAA motifs detected in file\n",
      "\n",
      "Complement found from 1099 to 1105\n",
      "Complement found from 1671 to 1677\n",
      "Complement found from 2779 to 2785\n",
      "Complement found from 4913 to 4919\n",
      "Complement found from 8119 to 8125\n",
      "Complement found from 9599 to 9605\n",
      "Complement found from 10745 to 10751\n",
      "Complement found from 11278 to 11284\n",
      "Complement found from 12945 to 12951\n",
      "Complement found from 13686 to 13692\n",
      "Complement found from 14582 to 14588\n",
      "Complement found from 14636 to 14642\n",
      "Seq is: CTAAAC\n",
      "12 CTAAAC motifs detected in file\n",
      "\n",
      "Complement found from 3503 to 3509\n",
      "Complement found from 4501 to 4507\n",
      "Complement found from 6826 to 6832\n",
      "Complement found from 7869 to 7875\n",
      "Complement found from 10281 to 10287\n",
      "Complement found from 10504 to 10510\n",
      "Complement found from 12713 to 12719\n",
      "Complement found from 15786 to 15792\n",
      "Complement found from 16130 to 16136\n",
      "Seq is: TACCAT\n",
      "9 TACCAT motifs detected in file\n",
      "\n",
      "Complement found from 1188 to 1194\n",
      "Complement found from 4906 to 4912\n",
      "Complement found from 6220 to 6226\n",
      "Complement found from 7860 to 7866\n",
      "Complement found from 8472 to 8478\n",
      "Complement found from 9083 to 9089\n",
      "Complement found from 10328 to 10334\n",
      "Complement found from 10625 to 10631\n",
      "Complement found from 10868 to 10874\n",
      "Complement found from 11977 to 11983\n",
      "Complement found from 12100 to 12106\n",
      "Complement found from 13449 to 13455\n",
      "Seq is: TCCCTC\n",
      "12 TCCCTC motifs detected in file\n",
      "\n",
      "Complement found from 659 to 665\n",
      "Complement found from 791 to 797\n",
      "Complement found from 1676 to 1682\n",
      "Complement found from 3635 to 3641\n",
      "Complement found from 3740 to 3746\n",
      "Complement found from 4196 to 4202\n",
      "Complement found from 6355 to 6361\n",
      "Complement found from 6505 to 6511\n",
      "Complement found from 8828 to 8834\n",
      "Complement found from 8897 to 8903\n",
      "Complement found from 9059 to 9065\n",
      "Complement found from 9295 to 9301\n",
      "Complement found from 9520 to 9526\n",
      "Complement found from 10093 to 10099\n",
      "Complement found from 10348 to 10354\n",
      "Complement found from 10690 to 10696\n",
      "Complement found from 11742 to 11748\n",
      "Complement found from 12986 to 12992\n",
      "Complement found from 13130 to 13136\n",
      "Complement found from 13304 to 13310\n",
      "Complement found from 13466 to 13472\n",
      "Complement found from 13923 to 13929\n",
      "Complement found from 14895 to 14901\n",
      "Complement found from 15318 to 15324\n",
      "Complement found from 15516 to 15522\n",
      "Complement found from 15645 to 15651\n",
      "Complement found from 15726 to 15732\n",
      "Seq is: CCTAGC\n",
      "27 CCTAGC motifs detected in file\n",
      "\n",
      "Complement found from 1067 to 1073\n",
      "Complement found from 5713 to 5719\n",
      "Seq is: AACTGG\n",
      "2 AACTGG motifs detected in file\n",
      "\n",
      "Complement found from 2826 to 2832\n",
      "Complement found from 3875 to 3881\n",
      "Complement found from 12818 to 12824\n",
      "Complement found from 13133 to 13139\n",
      "Complement found from 16037 to 16043\n",
      "Seq is: AGCAGA\n",
      "5 AGCAGA motifs detected in file\n",
      "\n",
      "Complement found from 485 to 491\n",
      "Complement found from 601 to 607\n",
      "Complement found from 4729 to 4735\n",
      "Complement found from 4742 to 4748\n",
      "Complement found from 5563 to 5569\n",
      "Complement found from 6434 to 6440\n",
      "Complement found from 14577 to 14583\n",
      "Complement found from 14753 to 14759\n",
      "Seq is: CAATAC\n",
      "8 CAATAC motifs detected in file\n",
      "\n",
      "Complement found from 290 to 296\n",
      "Complement found from 4644 to 4650\n",
      "Complement found from 7090 to 7096\n",
      "Complement found from 8600 to 8606\n",
      "Complement found from 9767 to 9773\n",
      "Complement found from 13749 to 13755\n",
      "Complement found from 15563 to 15569\n",
      "Seq is: ATTTCC\n",
      "7 ATTTCC motifs detected in file\n",
      "\n",
      "Complement found from 1534 to 1540\n",
      "Complement found from 3403 to 3409\n",
      "Complement found from 4992 to 4998\n",
      "Complement found from 7826 to 7832\n",
      "Complement found from 14756 to 14762\n",
      "Seq is: TACGCA\n",
      "5 TACGCA motifs detected in file\n",
      "\n",
      "Complement found from 5216 to 5222\n",
      "Complement found from 7822 to 7828\n",
      "Complement found from 10553 to 10559\n",
      "Complement found from 11059 to 11065\n",
      "Complement found from 11405 to 11411\n",
      "Complement found from 11551 to 11557\n",
      "Complement found from 12375 to 12381\n",
      "Complement found from 12564 to 12570\n",
      "Complement found from 15596 to 15602\n",
      "Complement found from 15853 to 15859\n",
      "Seq is: TCCCTA\n",
      "10 TCCCTA motifs detected in file\n",
      "\n",
      "Complement found from 306 to 312\n",
      "Complement found from 456 to 462\n",
      "Complement found from 957 to 963\n",
      "Complement found from 1492 to 1498\n",
      "Complement found from 3568 to 3574\n",
      "Complement found from 5208 to 5214\n",
      "Complement found from 5325 to 5331\n",
      "Complement found from 6289 to 6295\n",
      "Complement found from 7364 to 7370\n",
      "Complement found from 7814 to 7820\n",
      "Complement found from 7861 to 7867\n",
      "Complement found from 9276 to 9282\n",
      "Complement found from 9726 to 9732\n",
      "Complement found from 10089 to 10095\n",
      "Complement found from 10266 to 10272\n",
      "Complement found from 10949 to 10955\n",
      "Complement found from 15537 to 15543\n",
      "Complement found from 16185 to 16191\n",
      "Seq is: CCCTCC\n",
      "18 CCCTCC motifs detected in file\n",
      "\n",
      "Complement found from 4146 to 4152\n",
      "Complement found from 5469 to 5475\n",
      "Complement found from 6823 to 6829\n",
      "Complement found from 7617 to 7623\n",
      "Complement found from 8139 to 8145\n",
      "Complement found from 13577 to 13583\n",
      "Complement found from 14983 to 14989\n",
      "Seq is: CGCTAC\n",
      "7 CGCTAC motifs detected in file\n",
      "\n",
      "Complement found from 11323 to 11329\n",
      "Complement found from 11917 to 11923\n",
      "Seq is: TCCTGA\n",
      "2 TCCTGA motifs detected in file\n",
      "\n",
      "Complement found from 503 to 509\n",
      "Complement found from 3054 to 3060\n",
      "Complement found from 3322 to 3328\n",
      "Complement found from 4183 to 4189\n",
      "Complement found from 5115 to 5121\n",
      "Complement found from 5475 to 5481\n",
      "Complement found from 6228 to 6234\n",
      "Complement found from 6480 to 6486\n",
      "Complement found from 6642 to 6648\n",
      "Complement found from 7942 to 7948\n",
      "Complement found from 9729 to 9735\n",
      "Complement found from 10973 to 10979\n",
      "Complement found from 11936 to 11942\n",
      "Complement found from 12904 to 12910\n",
      "Complement found from 14307 to 14313\n",
      "Complement found from 14370 to 14376\n",
      "Complement found from 15029 to 15035\n",
      "Complement found from 16505 to 16511\n",
      "Seq is: TCCTAC\n",
      "18 TCCTAC motifs detected in file\n",
      "\n",
      "Complement found from 1258 to 1264\n",
      "Complement found from 1782 to 1788\n",
      "Complement found from 5107 to 5113\n",
      "Complement found from 12731 to 12737\n",
      "Seq is: TACCGC\n",
      "4 TACCGC motifs detected in file\n",
      "\n",
      "Complement found from 6462 to 6468\n",
      "Complement found from 15593 to 15599\n",
      "Seq is: CCGTCC\n",
      "2 CCGTCC motifs detected in file\n",
      "\n",
      "Complement found from 2963 to 2969\n",
      "Complement found from 3757 to 3763\n",
      "Complement found from 6406 to 6412\n",
      "Complement found from 6735 to 6741\n",
      "Complement found from 7044 to 7050\n",
      "Complement found from 9067 to 9073\n",
      "Complement found from 9630 to 9636\n",
      "Complement found from 11315 to 11321\n",
      "Seq is: TATCAA\n",
      "8 TATCAA motifs detected in file\n",
      "\n",
      "Complement found from 250 to 256\n",
      "Complement found from 13203 to 13209\n",
      "Seq is: GTCTGC\n",
      "2 GTCTGC motifs detected in file\n",
      "\n",
      "Complement found from 361 to 367\n",
      "Complement found from 559 to 565\n",
      "Complement found from 2100 to 2106\n",
      "Complement found from 3476 to 3482\n",
      "Complement found from 5938 to 5944\n",
      "Complement found from 13508 to 13514\n",
      "Complement found from 14471 to 14477\n",
      "Complement found from 15423 to 15429\n",
      "Seq is: CAAAGA\n",
      "8 CAAAGA motifs detected in file\n",
      "\n",
      "Complement found from 3026 to 3032\n",
      "Complement found from 8714 to 8720\n",
      "Complement found from 12270 to 12276\n",
      "Seq is: TAAAGG\n",
      "3 TAAAGG motifs detected in file\n",
      "\n",
      "Complement found from 6871 to 6877\n",
      "Complement found from 9616 to 9622\n",
      "Seq is: ACTCGC\n",
      "2 ACTCGC motifs detected in file\n",
      "\n",
      "Seq is: AGACTC\n",
      "0 AGACTC motifs detected in file\n",
      "\n",
      "Complement found from 2386 to 2392\n",
      "Complement found from 4022 to 4028\n",
      "Complement found from 4232 to 4238\n",
      "Complement found from 9700 to 9706\n",
      "Complement found from 13280 to 13286\n",
      "Complement found from 14172 to 14178\n",
      "Seq is: TACAAT\n",
      "6 TACAAT motifs detected in file\n",
      "\n",
      "Complement found from 230 to 236\n",
      "Complement found from 1816 to 1822\n",
      "Complement found from 1851 to 1857\n",
      "Complement found from 2596 to 2602\n",
      "Complement found from 3851 to 3857\n",
      "Complement found from 4669 to 4675\n",
      "Complement found from 4762 to 4768\n",
      "Complement found from 6121 to 6127\n",
      "Complement found from 6829 to 6835\n",
      "Complement found from 7668 to 7674\n",
      "Complement found from 8397 to 8403\n",
      "Complement found from 9202 to 9208\n",
      "Complement found from 10821 to 10827\n",
      "Complement found from 11562 to 11568\n",
      "Complement found from 11703 to 11709\n",
      "Complement found from 11781 to 11787\n",
      "Complement found from 12593 to 12599\n",
      "Complement found from 13271 to 13277\n",
      "Complement found from 14081 to 14087\n",
      "Complement found from 14228 to 14234\n",
      "Complement found from 15687 to 15693\n",
      "Seq is: CATAAT\n",
      "21 CATAAT motifs detected in file\n",
      "\n",
      "Complement found from 2388 to 2394\n",
      "Complement found from 4738 to 4744\n",
      "Complement found from 4887 to 4893\n",
      "Complement found from 5370 to 5376\n",
      "Complement found from 9215 to 9221\n",
      "Complement found from 9633 to 9639\n",
      "Complement found from 10985 to 10991\n",
      "Complement found from 11627 to 11633\n",
      "Complement found from 14573 to 14579\n",
      "Complement found from 15419 to 15425\n",
      "Complement found from 15709 to 15715\n",
      "Complement found from 16213 to 16219\n",
      "Seq is: CAATCA\n",
      "12 CAATCA motifs detected in file\n",
      "\n",
      "Complement found from 358 to 364\n",
      "Complement found from 642 to 648\n",
      "Complement found from 3718 to 3724\n",
      "Complement found from 5170 to 5176\n",
      "Complement found from 5284 to 5290\n",
      "Complement found from 6193 to 6199\n",
      "Complement found from 8688 to 8694\n",
      "Complement found from 9664 to 9670\n",
      "Complement found from 10296 to 10302\n",
      "Complement found from 10417 to 10423\n",
      "Complement found from 12550 to 12556\n",
      "Complement found from 13771 to 13777\n",
      "Complement found from 14026 to 14032\n",
      "Complement found from 14190 to 14196\n",
      "Complement found from 14655 to 14661\n",
      "Complement found from 15360 to 15366\n",
      "Complement found from 15676 to 15682\n",
      "Complement found from 15864 to 15870\n",
      "Seq is: AAACAA\n",
      "18 AAACAA motifs detected in file\n",
      "\n",
      "Complement found from 26 to 32\n",
      "Complement found from 4189 to 4195\n",
      "Complement found from 4518 to 4524\n",
      "Complement found from 5451 to 5457\n",
      "Complement found from 5874 to 5880\n",
      "Complement found from 7718 to 7724\n",
      "Complement found from 8065 to 8071\n",
      "Complement found from 8793 to 8799\n",
      "Complement found from 11256 to 11262\n",
      "Complement found from 11498 to 11504\n",
      "Complement found from 11766 to 11772\n",
      "Complement found from 12011 to 12017\n",
      "Complement found from 13077 to 13083\n",
      "Complement found from 14116 to 14122\n",
      "Complement found from 14399 to 14405\n",
      "Complement found from 14645 to 14651\n",
      "Complement found from 14791 to 14797\n",
      "Seq is: CACTCA\n",
      "17 CACTCA motifs detected in file\n",
      "\n",
      "Complement found from 2943 to 2949\n",
      "Complement found from 3649 to 3655\n",
      "Complement found from 8563 to 8569\n",
      "Complement found from 12847 to 12853\n",
      "Complement found from 13777 to 13783\n",
      "Complement found from 13942 to 13948\n",
      "Complement found from 14367 to 14373\n",
      "Complement found from 15827 to 15833\n",
      "Complement found from 16168 to 16174\n",
      "Seq is: CAATCC\n",
      "9 CAATCC motifs detected in file\n",
      "\n",
      "Complement found from 451 to 457\n",
      "Complement found from 7676 to 7682\n",
      "Complement found from 7709 to 7715\n",
      "Complement found from 10924 to 10930\n",
      "Complement found from 12127 to 12133\n",
      "Complement found from 15940 to 15946\n",
      "Seq is: TTTTCC\n",
      "6 TTTTCC motifs detected in file\n",
      "\n",
      "Complement found from 8385 to 8391\n",
      "Seq is: CGTATG\n",
      "1 CGTATG motifs detected in file\n",
      "\n",
      "Complement found from 11323 to 11329\n",
      "Complement found from 11917 to 11923\n",
      "Seq is: TCCTGA\n",
      "2 TCCTGA motifs detected in file\n",
      "\n",
      "Complement found from 4674 to 4680\n",
      "Complement found from 12786 to 12792\n",
      "Complement found from 14286 to 14292\n",
      "Complement found from 15449 to 15455\n",
      "Seq is: TCCTTC\n",
      "4 TCCTTC motifs detected in file\n",
      "\n",
      "Complement found from 8 to 14\n",
      "Complement found from 1948 to 1954\n",
      "Seq is: GTCTAT\n",
      "2 GTCTAT motifs detected in file\n",
      "\n",
      "Complement found from 3780 to 3786\n",
      "Complement found from 7832 to 7838\n",
      "Complement found from 10269 to 10275\n",
      "Seq is: TCCTTT\n",
      "3 TCCTTT motifs detected in file\n",
      "\n",
      "Complement found from 195 to 201\n",
      "Complement found from 473 to 479\n",
      "Complement found from 860 to 866\n",
      "Complement found from 3766 to 3772\n",
      "Complement found from 4464 to 4470\n",
      "Complement found from 5496 to 5502\n",
      "Complement found from 6528 to 6534\n",
      "Complement found from 7740 to 7746\n",
      "Complement found from 9345 to 9351\n",
      "Complement found from 10104 to 10110\n",
      "Complement found from 10958 to 10964\n",
      "Complement found from 11051 to 11057\n",
      "Complement found from 12718 to 12724\n",
      "Complement found from 13655 to 13661\n",
      "Complement found from 13684 to 13690\n",
      "Complement found from 13739 to 13745\n",
      "Complement found from 14214 to 14220\n",
      "Complement found from 14580 to 14586\n",
      "Complement found from 14634 to 14640\n",
      "Seq is: TACTAA\n",
      "19 TACTAA motifs detected in file\n",
      "\n",
      "Complement found from 3325 to 3331\n",
      "Complement found from 3826 to 3832\n",
      "Complement found from 5009 to 5015\n",
      "Complement found from 5360 to 5366\n",
      "Complement found from 5379 to 5385\n",
      "Complement found from 5472 to 5478\n",
      "Complement found from 6231 to 6237\n",
      "Complement found from 6308 to 6314\n",
      "Complement found from 6683 to 6689\n",
      "Complement found from 8012 to 8018\n",
      "Complement found from 8411 to 8417\n",
      "Complement found from 10752 to 10758\n",
      "Complement found from 11810 to 11816\n",
      "Complement found from 11974 to 11980\n",
      "Complement found from 13072 to 13078\n",
      "Complement found from 13503 to 13509\n",
      "Complement found from 13984 to 13990\n",
      "Complement found from 14131 to 14137\n",
      "Complement found from 14440 to 14446\n",
      "Seq is: TACTCC\n",
      "19 TACTCC motifs detected in file\n",
      "\n",
      "Complement found from 3698 to 3704\n",
      "Complement found from 4049 to 4055\n",
      "Complement found from 4539 to 4545\n",
      "Complement found from 11244 to 11250\n",
      "Complement found from 11764 to 11770\n",
      "Seq is: CGCACT\n",
      "5 CGCACT motifs detected in file\n",
      "\n",
      "Complement found from 8010 to 8016\n",
      "Complement found from 8585 to 8591\n",
      "Complement found from 9744 to 9750\n",
      "Complement found from 11445 to 11451\n",
      "Complement found from 11457 to 11463\n",
      "Seq is: AGTACT\n",
      "5 AGTACT motifs detected in file\n",
      "\n",
      "Complement found from 265 to 271\n",
      "Complement found from 292 to 298\n",
      "Complement found from 4621 to 4627\n",
      "Complement found from 7031 to 7037\n",
      "Complement found from 9314 to 9320\n",
      "Complement found from 9812 to 9818\n",
      "Complement found from 13122 to 13128\n",
      "Complement found from 15403 to 15409\n",
      "Seq is: TTCCAC\n",
      "8 TTCCAC motifs detected in file\n",
      "\n",
      "Complement found from 5964 to 5970\n",
      "Complement found from 6010 to 6016\n",
      "Complement found from 13400 to 13406\n",
      "Complement found from 13720 to 13726\n",
      "Complement found from 15569 to 15575\n",
      "Seq is: TATTCG\n",
      "5 TATTCG motifs detected in file\n",
      "\n",
      "Complement found from 2287 to 2293\n",
      "Complement found from 2719 to 2725\n",
      "Complement found from 7357 to 7363\n",
      "Complement found from 7635 to 7641\n",
      "Complement found from 14610 to 14616\n",
      "Seq is: AGAAGA\n",
      "5 AGAAGA motifs detected in file\n",
      "\n",
      "Complement found from 2881 to 2887\n",
      "Complement found from 3556 to 3562\n",
      "Complement found from 3754 to 3760\n",
      "Complement found from 5154 to 5160\n",
      "Complement found from 10557 to 10563\n",
      "Complement found from 10578 to 10584\n",
      "Complement found from 10874 to 10880\n",
      "Complement found from 11546 to 11552\n",
      "Complement found from 11732 to 11738\n",
      "Complement found from 12350 to 12356\n",
      "Complement found from 13348 to 13354\n",
      "Complement found from 15191 to 15197\n",
      "Complement found from 15629 to 15635\n",
      "Complement found from 15812 to 15818\n",
      "Seq is: TACTAT\n",
      "14 TACTAT motifs detected in file\n",
      "\n",
      "Complement found from 2075 to 2081\n",
      "Complement found from 3893 to 3899\n",
      "Complement found from 3959 to 3965\n",
      "Complement found from 4318 to 4324\n",
      "Complement found from 4795 to 4801\n",
      "Complement found from 5487 to 5493\n",
      "Complement found from 8843 to 8849\n",
      "Complement found from 8931 to 8937\n",
      "Complement found from 10161 to 10167\n",
      "Complement found from 11535 to 11541\n",
      "Complement found from 12193 to 12199\n",
      "Complement found from 13763 to 13769\n",
      "Complement found from 15525 to 15531\n",
      "Complement found from 16545 to 16551\n",
      "Seq is: CCCCTT\n",
      "14 CCCCTT motifs detected in file\n",
      "\n",
      "Complement found from 3545 to 3551\n",
      "Complement found from 5455 to 5461\n",
      "Complement found from 6835 to 6841\n",
      "Complement found from 7809 to 7815\n",
      "Complement found from 11241 to 11247\n",
      "Complement found from 11430 to 11436\n",
      "Complement found from 13574 to 13580\n",
      "Complement found from 14379 to 14385\n",
      "Complement found from 14454 to 14460\n",
      "Seq is: CATCGC\n",
      "9 CATCGC motifs detected in file\n",
      "\n",
      "Complement found from 1136 to 1142\n",
      "Complement found from 1925 to 1931\n",
      "Complement found from 3217 to 3223\n",
      "Complement found from 3812 to 3818\n",
      "Complement found from 14737 to 14743\n",
      "Seq is: AGAACA\n",
      "5 AGAACA motifs detected in file\n",
      "\n",
      "Seq is: TCTGGA\n",
      "0 TCTGGA motifs detected in file\n",
      "\n",
      "Complement found from 5617 to 5623\n",
      "Complement found from 8527 to 8533\n",
      "Complement found from 11186 to 11192\n",
      "Seq is: TGAACG\n",
      "3 TGAACG motifs detected in file\n",
      "\n",
      "Complement found from 7214 to 7220\n",
      "Seq is: CGACGT\n",
      "1 CGACGT motifs detected in file\n",
      "\n",
      "Seq is: ACACGG\n",
      "0 ACACGG motifs detected in file\n",
      "\n",
      "Complement found from 5196 to 5202\n",
      "Complement found from 8222 to 8228\n",
      "Complement found from 9112 to 9118\n",
      "Complement found from 9681 to 9687\n",
      "Complement found from 12379 to 12385\n",
      "Complement found from 13612 to 13618\n",
      "Seq is: TAATTC\n",
      "6 TAATTC motifs detected in file\n",
      "\n",
      "Complement found from 1254 to 1260\n",
      "Complement found from 1542 to 1548\n",
      "Complement found from 3393 to 3399\n",
      "Complement found from 7574 to 7580\n",
      "Complement found from 10407 to 10413\n",
      "Complement found from 12654 to 12660\n",
      "Complement found from 14177 to 14183\n",
      "Seq is: TATATA\n",
      "7 TATATA motifs detected in file\n",
      "\n",
      "Complement found from 2777 to 2783\n",
      "Complement found from 3361 to 3367\n",
      "Complement found from 4395 to 4401\n",
      "Complement found from 5097 to 5103\n",
      "Complement found from 6465 to 6471\n",
      "Complement found from 7347 to 7353\n",
      "Complement found from 7712 to 7718\n",
      "Complement found from 9279 to 9285\n",
      "Complement found from 9597 to 9603\n",
      "Complement found from 10952 to 10958\n",
      "Complement found from 12706 to 12712\n",
      "Complement found from 14122 to 14128\n",
      "Complement found from 14134 to 14140\n",
      "Complement found from 15830 to 15836\n",
      "Complement found from 15836 to 15842\n",
      "Seq is: TCCTAA\n",
      "15 TCCTAA motifs detected in file\n",
      "\n",
      "Complement found from 6125 to 6131\n",
      "Complement found from 15757 to 15763\n",
      "Seq is: ATCGGA\n",
      "2 ATCGGA motifs detected in file\n",
      "\n",
      "Complement found from 19 to 25\n",
      "Complement found from 2743 to 2749\n",
      "Complement found from 3023 to 3029\n",
      "Complement found from 7299 to 7305\n",
      "Complement found from 8447 to 8453\n",
      "Complement found from 10335 to 10341\n",
      "Complement found from 11874 to 11880\n",
      "Complement found from 14315 to 14321\n",
      "Complement found from 14511 to 14517\n",
      "Seq is: TATTAA\n",
      "9 TATTAA motifs detected in file\n",
      "\n",
      "Complement found from 243 to 249\n",
      "Complement found from 1756 to 1762\n",
      "Complement found from 8020 to 8026\n",
      "Complement found from 15859 to 15865\n",
      "Seq is: ATTGAA\n",
      "4 ATTGAA motifs detected in file\n",
      "\n",
      "Complement found from 2090 to 2096\n",
      "Complement found from 12752 to 12758\n",
      "Seq is: ACTGTT\n",
      "2 ACTGTT motifs detected in file\n",
      "\n",
      "Complement found from 172 to 178\n",
      "Complement found from 484 to 490\n",
      "Complement found from 920 to 926\n",
      "Complement found from 4741 to 4747\n",
      "Complement found from 5801 to 5807\n",
      "Complement found from 6408 to 6414\n",
      "Complement found from 7046 to 7052\n",
      "Complement found from 11440 to 11446\n",
      "Complement found from 14446 to 14452\n",
      "Complement found from 14576 to 14582\n",
      "Complement found from 15007 to 15013\n",
      "Complement found from 16417 to 16423\n",
      "Seq is: TCAATA\n",
      "12 TCAATA motifs detected in file\n",
      "\n",
      "Complement found from 3269 to 3275\n",
      "Complement found from 7834 to 7840\n",
      "Complement found from 8353 to 8359\n",
      "Complement found from 9883 to 9889\n",
      "Complement found from 11383 to 11389\n",
      "Complement found from 14091 to 14097\n",
      "Complement found from 15279 to 15285\n",
      "Seq is: CTTTAC\n",
      "7 CTTTAC motifs detected in file\n",
      "\n",
      "Complement found from 284 to 290\n",
      "Complement found from 355 to 361\n",
      "Complement found from 2501 to 2507\n",
      "Complement found from 4603 to 4609\n",
      "Complement found from 4868 to 4874\n",
      "Complement found from 5280 to 5286\n",
      "Complement found from 7445 to 7451\n",
      "Complement found from 7513 to 7519\n",
      "Complement found from 9431 to 9437\n",
      "Complement found from 10046 to 10052\n",
      "Complement found from 10379 to 10385\n",
      "Complement found from 10812 to 10818\n",
      "Complement found from 12304 to 12310\n",
      "Complement found from 12416 to 12422\n",
      "Complement found from 13229 to 13235\n",
      "Complement found from 14073 to 14079\n",
      "Seq is: CAAAAA\n",
      "16 CAAAAA motifs detected in file\n",
      "\n",
      "Complement found from 8864 to 8870\n",
      "Seq is: GATTAT\n",
      "1 GATTAT motifs detected in file\n",
      "\n",
      "Complement found from 2127 to 2133\n",
      "Complement found from 5744 to 5750\n",
      "Complement found from 6689 to 6695\n",
      "Seq is: GGAAAA\n",
      "3 GGAAAA motifs detected in file\n",
      "\n",
      "Complement found from 418 to 424\n",
      "Complement found from 2661 to 2667\n",
      "Complement found from 7506 to 7512\n",
      "Complement found from 12265 to 12271\n",
      "Seq is: ACTTTT\n",
      "4 ACTTTT motifs detected in file\n",
      "\n",
      "Complement found from 5552 to 5558\n",
      "Complement found from 15250 to 15256\n",
      "Seq is: TCAGTA\n",
      "2 TCAGTA motifs detected in file\n",
      "\n",
      "Complement found from 2327 to 2333\n",
      "Complement found from 2838 to 2844\n",
      "Complement found from 6324 to 6330\n",
      "Complement found from 6819 to 6825\n",
      "Complement found from 9288 to 9294\n",
      "Complement found from 10928 to 10934\n",
      "Complement found from 16406 to 16412\n",
      "Seq is: CCTCCG\n",
      "7 CCTCCG motifs detected in file\n",
      "\n",
      "Complement found from 1306 to 1312\n",
      "Complement found from 2573 to 2579\n",
      "Complement found from 3336 to 3342\n",
      "Complement found from 13324 to 13330\n",
      "Seq is: GTACCC\n",
      "4 GTACCC motifs detected in file\n",
      "\n",
      "Complement found from 3884 to 3890\n",
      "Complement found from 4659 to 4665\n",
      "Complement found from 8642 to 8648\n",
      "Complement found from 9667 to 9673\n",
      "Complement found from 12857 to 12863\n",
      "Complement found from 14924 to 14930\n",
      "Complement found from 16078 to 16084\n",
      "Seq is: CAACCG\n",
      "7 CAACCG motifs detected in file\n",
      "\n",
      "Complement found from 1872 to 1878\n",
      "Complement found from 2043 to 2049\n",
      "Complement found from 3267 to 3273\n",
      "Complement found from 7171 to 7177\n",
      "Complement found from 7541 to 7547\n",
      "Complement found from 9841 to 9847\n",
      "Complement found from 12264 to 12270\n",
      "Complement found from 14089 to 14095\n",
      "Seq is: AACTTT\n",
      "8 AACTTT motifs detected in file\n",
      "\n",
      "Complement found from 1410 to 1416\n",
      "Seq is: GGTCGA\n",
      "1 GGTCGA motifs detected in file\n",
      "\n",
      "Complement found from 300 to 306\n",
      "Complement found from 350 to 356\n",
      "Complement found from 491 to 497\n",
      "Complement found from 531 to 537\n",
      "Complement found from 554 to 560\n",
      "Complement found from 864 to 870\n",
      "Complement found from 1220 to 1226\n",
      "Complement found from 1528 to 1534\n",
      "Complement found from 1834 to 1840\n",
      "Complement found from 3563 to 3569\n",
      "Complement found from 3582 to 3588\n",
      "Complement found from 3890 to 3896\n",
      "Complement found from 4315 to 4321\n",
      "Complement found from 5598 to 5604\n",
      "Complement found from 5841 to 5847\n",
      "Complement found from 6417 to 6423\n",
      "Complement found from 7463 to 7469\n",
      "Complement found from 7488 to 7494\n",
      "Complement found from 10944 to 10950\n",
      "Complement found from 11509 to 11515\n",
      "Complement found from 11670 to 11676\n",
      "Complement found from 12106 to 12112\n",
      "Complement found from 13674 to 13680\n",
      "Complement found from 13689 to 13695\n",
      "Complement found from 14386 to 14392\n",
      "Complement found from 14415 to 14421\n",
      "Complement found from 14585 to 14591\n",
      "Complement found from 14617 to 14623\n",
      "Complement found from 14626 to 14632\n",
      "Complement found from 14767 to 14773\n",
      "Complement found from 15364 to 15370\n",
      "Complement found from 15523 to 15529\n",
      "Complement found from 16181 to 16187\n",
      "Seq is: AACCCC\n",
      "33 AACCCC motifs detected in file\n",
      "\n",
      "Complement found from 1811 to 1817\n",
      "Complement found from 9152 to 9158\n",
      "Complement found from 12957 to 12963\n",
      "Seq is: CCAAGC\n",
      "3 CCAAGC motifs detected in file\n",
      "\n",
      "Complement found from 2444 to 2450\n",
      "Complement found from 5586 to 5592\n",
      "Seq is: TAAGGA\n",
      "2 TAAGGA motifs detected in file\n",
      "\n",
      "Complement found from 2126 to 2132\n",
      "Complement found from 2446 to 2452\n",
      "Complement found from 7760 to 7766\n",
      "Seq is: AGGAAA\n",
      "3 AGGAAA motifs detected in file\n",
      "\n",
      "Complement found from 811 to 817\n",
      "Complement found from 2752 to 2758\n",
      "Complement found from 8096 to 8102\n",
      "Complement found from 11594 to 11600\n",
      "Seq is: AAACAG\n",
      "4 AAACAG motifs detected in file\n",
      "\n",
      "Complement found from 5389 to 5395\n",
      "Complement found from 7167 to 7173\n",
      "Seq is: ATCTAA\n",
      "2 ATCTAA motifs detected in file\n",
      "\n",
      "Complement found from 486 to 492\n",
      "Complement found from 602 to 608\n",
      "Complement found from 3985 to 3991\n",
      "Complement found from 5525 to 5531\n",
      "Complement found from 15905 to 15911\n",
      "Seq is: AATACA\n",
      "5 AATACA motifs detected in file\n",
      "\n",
      "Complement found from 7702 to 7708\n",
      "Complement found from 10560 to 10566\n",
      "Seq is: TATGCC\n",
      "2 TATGCC motifs detected in file\n",
      "\n",
      "Complement found from 1351 to 1357\n",
      "Complement found from 3215 to 3221\n",
      "Complement found from 3810 to 3816\n",
      "Complement found from 11308 to 11314\n",
      "Complement found from 12217 to 12223\n",
      "Complement found from 12510 to 12516\n",
      "Complement found from 14735 to 14741\n",
      "Seq is: CAAGAA\n",
      "7 CAAGAA motifs detected in file\n",
      "\n",
      "Complement found from 585 to 591\n",
      "Complement found from 3368 to 3374\n",
      "Complement found from 11350 to 11356\n",
      "Complement found from 11718 to 11724\n",
      "Complement found from 12185 to 12191\n",
      "Complement found from 15437 to 15443\n",
      "Complement found from 16195 to 16201\n",
      "Seq is: GCTTAC\n",
      "7 GCTTAC motifs detected in file\n",
      "\n",
      "Complement found from 373 to 379\n",
      "Complement found from 6549 to 6555\n",
      "Complement found from 6769 to 6775\n",
      "Complement found from 7242 to 7248\n",
      "Complement found from 8802 to 8808\n",
      "Complement found from 9419 to 9425\n",
      "Complement found from 14182 to 14188\n",
      "Complement found from 14740 to 14746\n",
      "Complement found from 15908 to 15914\n",
      "Seq is: ACACCA\n",
      "9 ACACCA motifs detected in file\n",
      "\n",
      "Complement found from 459 to 465\n",
      "Complement found from 6311 to 6317\n",
      "Complement found from 7179 to 7185\n",
      "Complement found from 9591 to 9597\n",
      "Complement found from 11813 to 11819\n",
      "Complement found from 14113 to 14119\n",
      "Complement found from 15261 to 15267\n",
      "Seq is: TCCCAC\n",
      "7 TCCCAC motifs detected in file\n",
      "\n",
      "Complement found from 310 to 316\n",
      "Complement found from 493 to 499\n",
      "Complement found from 1900 to 1906\n",
      "Complement found from 3167 to 3173\n",
      "Complement found from 4136 to 4142\n",
      "Complement found from 5231 to 5237\n",
      "Complement found from 6168 to 6174\n",
      "Complement found from 10191 to 10197\n",
      "Complement found from 13753 to 13759\n",
      "Complement found from 14154 to 14160\n",
      "Complement found from 14243 to 14249\n",
      "Seq is: CCCCCG\n",
      "11 CCCCCG motifs detected in file\n",
      "\n",
      "Complement found from 1978 to 1984\n",
      "Complement found from 6045 to 6051\n",
      "Complement found from 6714 to 6720\n",
      "Seq is: TAGGTA\n",
      "3 TAGGTA motifs detected in file\n",
      "\n",
      "Complement found from 12470 to 12476\n",
      "Seq is: TATCAG\n",
      "1 TATCAG motifs detected in file\n",
      "\n",
      "Complement found from 1343 to 1349\n",
      "Complement found from 4967 to 4973\n",
      "Seq is: TGAGGT\n",
      "2 TGAGGT motifs detected in file\n",
      "\n",
      "Complement found from 1732 to 1738\n",
      "Complement found from 4897 to 4903\n",
      "Complement found from 5253 to 5259\n",
      "Complement found from 8626 to 8632\n",
      "Complement found from 9676 to 9682\n",
      "Complement found from 10465 to 10471\n",
      "Complement found from 10887 to 10893\n",
      "Complement found from 12323 to 12329\n",
      "Complement found from 14042 to 14048\n",
      "Complement found from 14874 to 14880\n",
      "Complement found from 15153 to 15159\n",
      "Seq is: CCAAAT\n",
      "11 CCAAAT motifs detected in file\n",
      "\n",
      "Complement found from 589 to 595\n",
      "Complement found from 2837 to 2843\n",
      "Complement found from 3319 to 3325\n",
      "Complement found from 4104 to 4110\n",
      "Complement found from 4165 to 4171\n",
      "Complement found from 6217 to 6223\n",
      "Complement found from 6818 to 6824\n",
      "Complement found from 8469 to 8475\n",
      "Complement found from 8622 to 8628\n",
      "Complement found from 8773 to 8779\n",
      "Complement found from 9287 to 9293\n",
      "Complement found from 13446 to 13452\n",
      "Complement found from 13581 to 13587\n",
      "Complement found from 14052 to 14058\n",
      "Complement found from 14374 to 14380\n",
      "Complement found from 14526 to 14532\n",
      "Complement found from 14804 to 14810\n",
      "Complement found from 15332 to 15338\n",
      "Complement found from 15379 to 15385\n",
      "Complement found from 15488 to 15494\n",
      "Complement found from 15737 to 15743\n",
      "Seq is: ACCTCC\n",
      "21 ACCTCC motifs detected in file\n",
      "\n",
      "Complement found from 1351 to 1357\n",
      "Complement found from 3215 to 3221\n",
      "Complement found from 3810 to 3816\n",
      "Complement found from 11308 to 11314\n",
      "Complement found from 12217 to 12223\n",
      "Complement found from 12510 to 12516\n",
      "Complement found from 14735 to 14741\n",
      "Seq is: CAAGAA\n",
      "7 CAAGAA motifs detected in file\n",
      "\n",
      "Complement found from 12805 to 12811\n",
      "Seq is: GATGAT\n",
      "1 GATGAT motifs detected in file\n",
      "\n",
      "Complement found from 2944 to 2950\n",
      "Complement found from 3650 to 3656\n",
      "Complement found from 4672 to 4678\n",
      "Complement found from 7569 to 7575\n",
      "Complement found from 8564 to 8570\n",
      "Complement found from 9454 to 9460\n",
      "Complement found from 11784 to 11790\n",
      "Complement found from 12848 to 12854\n",
      "Complement found from 14368 to 14374\n",
      "Complement found from 15828 to 15834\n",
      "Complement found from 15834 to 15840\n",
      "Seq is: AATCCT\n",
      "11 AATCCT motifs detected in file\n",
      "\n",
      "Complement found from 7341 to 7347\n",
      "Complement found from 14010 to 14016\n",
      "Seq is: GAAAAG\n",
      "2 GAAAAG motifs detected in file\n",
      "\n",
      "Complement found from 7335 to 7341\n",
      "Complement found from 9910 to 9916\n",
      "Complement found from 10675 to 10681\n",
      "Complement found from 11421 to 11427\n",
      "Seq is: CGAAGC\n",
      "4 CGAAGC motifs detected in file\n",
      "\n",
      "Complement found from 3450 to 3456\n",
      "Complement found from 7328 to 7334\n",
      "Complement found from 8541 to 8547\n",
      "Complement found from 8875 to 8881\n",
      "Seq is: TTCGCT\n",
      "4 TTCGCT motifs detected in file\n",
      "\n",
      "Complement found from 3918 to 3924\n",
      "Seq is: TCCGAA\n",
      "1 TCCGAA motifs detected in file\n",
      "\n",
      "Seq is: GAGTTT\n",
      "0 GAGTTT motifs detected in file\n",
      "\n",
      "Complement found from 8010 to 8016\n",
      "Complement found from 8585 to 8591\n",
      "Complement found from 9744 to 9750\n",
      "Complement found from 11445 to 11451\n",
      "Complement found from 11457 to 11463\n",
      "Seq is: AGTACT\n",
      "5 AGTACT motifs detected in file\n",
      "\n",
      "Complement found from 2736 to 2742\n",
      "Complement found from 5635 to 5641\n",
      "Complement found from 10075 to 10081\n",
      "Seq is: TTTAAT\n",
      "3 TTTAAT motifs detected in file\n",
      "\n",
      "Complement found from 1803 to 1809\n",
      "Complement found from 6403 to 6409\n",
      "Complement found from 7557 to 7563\n",
      "Complement found from 8500 to 8506\n",
      "Complement found from 10111 to 10117\n",
      "Complement found from 10445 to 10451\n",
      "Complement found from 10782 to 10788\n",
      "Complement found from 10854 to 10860\n",
      "Complement found from 11076 to 11082\n",
      "Complement found from 11565 to 11571\n",
      "Complement found from 12779 to 12785\n",
      "Complement found from 14295 to 14301\n",
      "Complement found from 14965 to 14971\n",
      "Complement found from 15508 to 15514\n",
      "Seq is: AATTAT\n",
      "14 AATTAT motifs detected in file\n",
      "\n",
      "Complement found from 5409 to 5415\n",
      "Complement found from 8665 to 8671\n",
      "Complement found from 9205 to 9211\n",
      "Complement found from 9283 to 9289\n",
      "Complement found from 13221 to 13227\n",
      "Complement found from 14745 to 14751\n",
      "Complement found from 15462 to 15468\n",
      "Seq is: AATGAC\n",
      "7 AATGAC motifs detected in file\n",
      "\n",
      "Complement found from 2365 to 2371\n",
      "Complement found from 4713 to 4719\n",
      "Complement found from 7999 to 8005\n",
      "Complement found from 15505 to 15511\n",
      "Seq is: GACAAT\n",
      "4 GACAAT motifs detected in file\n",
      "\n",
      "Complement found from 2113 to 2119\n",
      "Complement found from 8351 to 8357\n",
      "Complement found from 9991 to 9997\n",
      "Complement found from 11381 to 11387\n",
      "Complement found from 14102 to 14108\n",
      "Complement found from 14349 to 14355\n",
      "Seq is: CTCTTT\n",
      "6 CTCTTT motifs detected in file\n",
      "\n",
      "Complement found from 192 to 198\n",
      "Complement found from 1629 to 1635\n",
      "Complement found from 6677 to 6683\n",
      "Complement found from 11940 to 11946\n",
      "Complement found from 15188 to 15194\n",
      "Seq is: ACTTAC\n",
      "5 ACTTAC motifs detected in file\n",
      "\n",
      "Complement found from 13912 to 13918\n",
      "Seq is: TCGGAT\n",
      "1 TCGGAT motifs detected in file\n",
      "\n",
      "Complement found from 6284 to 6290\n",
      "Complement found from 7607 to 7613\n",
      "Seq is: GTCTAC\n",
      "2 GTCTAC motifs detected in file\n",
      "\n",
      "Complement found from 4687 to 4693\n",
      "Complement found from 5095 to 5101\n",
      "Complement found from 6640 to 6646\n",
      "Complement found from 7782 to 7788\n",
      "Complement found from 8743 to 8749\n",
      "Complement found from 9125 to 9131\n",
      "Complement found from 10548 to 10554\n",
      "Complement found from 12784 to 12790\n",
      "Complement found from 12902 to 12908\n",
      "Complement found from 15096 to 15102\n",
      "Seq is: TATCCT\n",
      "10 TATCCT motifs detected in file\n",
      "\n",
      "Complement found from 360 to 366\n",
      "Complement found from 1291 to 1297\n",
      "Complement found from 1617 to 1623\n",
      "Complement found from 3155 to 3161\n",
      "Complement found from 5937 to 5943\n",
      "Complement found from 14237 to 14243\n",
      "Complement found from 14657 to 14663\n",
      "Complement found from 15681 to 15687\n",
      "Seq is: ACAAAG\n",
      "8 ACAAAG motifs detected in file\n",
      "\n",
      "Complement found from 4162 to 4168\n",
      "Complement found from 6351 to 6357\n",
      "Complement found from 7109 to 7115\n",
      "Complement found from 7241 to 7247\n",
      "Complement found from 7912 to 7918\n",
      "Complement found from 8801 to 8807\n",
      "Complement found from 8927 to 8933\n",
      "Complement found from 11212 to 11218\n",
      "Complement found from 12074 to 12080\n",
      "Complement found from 14181 to 14187\n",
      "Complement found from 15907 to 15913\n",
      "Seq is: TACACC\n",
      "11 TACACC motifs detected in file\n",
      "\n",
      "Complement found from 188 to 194\n",
      "Complement found from 3304 to 3310\n",
      "Complement found from 5422 to 5428\n",
      "Complement found from 7946 to 7952\n",
      "Complement found from 9401 to 9407\n",
      "Complement found from 11197 to 11203\n",
      "Complement found from 11954 to 11960\n",
      "Complement found from 13906 to 13912\n",
      "Seq is: ACATAC\n",
      "8 ACATAC motifs detected in file\n",
      "\n",
      "Complement found from 1335 to 1341\n",
      "Complement found from 3711 to 3717\n",
      "Complement found from 7022 to 7028\n",
      "Complement found from 9800 to 9806\n",
      "Complement found from 13239 to 13245\n",
      "Seq is: GTAGCC\n",
      "5 GTAGCC motifs detected in file\n",
      "\n",
      "Complement found from 3505 to 3511\n",
      "Complement found from 3745 to 3751\n",
      "Complement found from 3834 to 3840\n",
      "Complement found from 4635 to 4641\n",
      "Complement found from 4950 to 4956\n",
      "Complement found from 5307 to 5313\n",
      "Complement found from 5319 to 5325\n",
      "Complement found from 6117 to 6123\n",
      "Complement found from 6384 to 6390\n",
      "Complement found from 7065 to 7071\n",
      "Complement found from 7793 to 7799\n",
      "Complement found from 7871 to 7877\n",
      "Complement found from 8962 to 8968\n",
      "Complement found from 12631 to 12637\n",
      "Complement found from 13369 to 13375\n",
      "Complement found from 14056 to 14062\n",
      "Complement found from 14341 to 14347\n",
      "Complement found from 14480 to 14486\n",
      "Complement found from 15788 to 15794\n",
      "Complement found from 16071 to 16077\n",
      "Seq is: CCATCA\n",
      "20 CCATCA motifs detected in file\n",
      "\n",
      "Complement found from 625 to 631\n",
      "Complement found from 7271 to 7277\n",
      "Complement found from 9785 to 9791\n",
      "Complement found from 9836 to 9842\n",
      "Complement found from 11272 to 11278\n",
      "Complement found from 12007 to 12013\n",
      "Complement found from 14845 to 14851\n",
      "Seq is: GGCTCA\n",
      "7 GGCTCA motifs detected in file\n",
      "\n",
      "Complement found from 4509 to 4515\n",
      "Complement found from 15311 to 15317\n",
      "Seq is: TTGCAG\n",
      "2 TTGCAG motifs detected in file\n",
      "\n",
      "Complement found from 704 to 710\n",
      "Complement found from 3168 to 3174\n",
      "Seq is: CCCCGT\n",
      "2 CCCCGT motifs detected in file\n",
      "\n",
      "Complement found from 3143 to 3149\n",
      "Complement found from 3409 to 3415\n",
      "Complement found from 9408 to 9414\n",
      "Complement found from 9435 to 9441\n",
      "Complement found from 13048 to 13054\n",
      "Seq is: AAGGCC\n",
      "5 AAGGCC motifs detected in file\n",
      "\n",
      "Complement found from 7200 to 7206\n",
      "Complement found from 15194 to 15200\n",
      "Complement found from 16490 to 16496\n",
      "Seq is: TATCCG\n",
      "3 TATCCG motifs detected in file\n",
      "\n",
      "Complement found from 675 to 681\n",
      "Complement found from 2112 to 2118\n",
      "Complement found from 3549 to 3555\n",
      "Seq is: GCTCTT\n",
      "3 GCTCTT motifs detected in file\n",
      "\n",
      "Complement found from 3153 to 3159\n",
      "Complement found from 3802 to 3808\n",
      "Complement found from 5277 to 5283\n",
      "Complement found from 6396 to 6402\n",
      "Complement found from 7721 to 7727\n",
      "Complement found from 8050 to 8056\n",
      "Complement found from 9103 to 9109\n",
      "Complement found from 10982 to 10988\n",
      "Complement found from 11259 to 11265\n",
      "Complement found from 12214 to 12220\n",
      "Complement found from 15821 to 15827\n",
      "Seq is: TCACAA\n",
      "11 TCACAA motifs detected in file\n",
      "\n",
      "Complement found from 14 to 20\n",
      "Complement found from 109 to 115\n",
      "Complement found from 721 to 727\n",
      "Complement found from 1490 to 1496\n",
      "Complement found from 2279 to 2285\n",
      "Complement found from 3509 to 3515\n",
      "Complement found from 3737 to 3743\n",
      "Complement found from 3794 to 3800\n",
      "Complement found from 4010 to 4016\n",
      "Complement found from 4193 to 4199\n",
      "Complement found from 5190 to 5196\n",
      "Complement found from 5206 to 5212\n",
      "Complement found from 5323 to 5329\n",
      "Complement found from 5703 to 5709\n",
      "Complement found from 6314 to 6320\n",
      "Complement found from 6620 to 6626\n",
      "Complement found from 7111 to 7117\n",
      "Complement found from 7401 to 7407\n",
      "Complement found from 9056 to 9062\n",
      "Complement found from 10087 to 10093\n",
      "Complement found from 11214 to 11220\n",
      "Complement found from 11265 to 11271\n",
      "Complement found from 12060 to 12066\n",
      "Complement found from 12359 to 12365\n",
      "Complement found from 12398 to 12404\n",
      "Complement found from 13622 to 13628\n",
      "Complement found from 13649 to 13655\n",
      "Complement found from 13679 to 13685\n",
      "Complement found from 15264 to 15270\n",
      "Complement found from 15406 to 15412\n",
      "Complement found from 16291 to 16297\n",
      "Seq is: CACCCT\n",
      "31 CACCCT motifs detected in file\n",
      "\n",
      "Complement found from 6936 to 6942\n",
      "Complement found from 14103 to 14109\n",
      "Complement found from 14350 to 14356\n",
      "Complement found from 16024 to 16030\n",
      "Seq is: TCTTTC\n",
      "4 TCTTTC motifs detected in file\n",
      "\n",
      "Complement found from 2272 to 2278\n",
      "Complement found from 5356 to 5362\n",
      "Complement found from 5723 to 5729\n",
      "Complement found from 7166 to 7172\n",
      "Complement found from 7437 to 7443\n",
      "Seq is: AATCTA\n",
      "5 AATCTA motifs detected in file\n",
      "\n",
      "Seq is: AATGCG\n",
      "0 AATGCG motifs detected in file\n",
      "\n",
      "Complement found from 3437 to 3443\n",
      "Complement found from 5470 to 5476\n",
      "Complement found from 7618 to 7624\n",
      "Complement found from 10598 to 10604\n",
      "Complement found from 15245 to 15251\n",
      "Complement found from 16437 to 16443\n",
      "Seq is: GCTACT\n",
      "6 GCTACT motifs detected in file\n",
      "\n",
      "Complement found from 858 to 864\n",
      "Complement found from 1503 to 1509\n",
      "Complement found from 2884 to 2890\n",
      "Complement found from 4702 to 4708\n",
      "Complement found from 5494 to 5500\n",
      "Complement found from 6523 to 6529\n",
      "Complement found from 8156 to 8162\n",
      "Complement found from 8735 to 8741\n",
      "Complement found from 10492 to 10498\n",
      "Complement found from 11049 to 11055\n",
      "Complement found from 11972 to 11978\n",
      "Complement found from 15815 to 15821\n",
      "Seq is: TATACT\n",
      "12 TATACT motifs detected in file\n",
      "\n",
      "Complement found from 2600 to 2606\n",
      "Complement found from 6518 to 6524\n",
      "Complement found from 6985 to 6991\n",
      "Complement found from 9897 to 9903\n",
      "Complement found from 11929 to 11935\n",
      "Complement found from 14950 to 14956\n",
      "Complement found from 15711 to 15717\n",
      "Seq is: ATCACT\n",
      "7 ATCACT motifs detected in file\n",
      "\n",
      "Complement found from 587 to 593\n",
      "Complement found from 4551 to 4557\n",
      "Complement found from 5886 to 5892\n",
      "Complement found from 6215 to 6221\n",
      "Complement found from 9465 to 9471\n",
      "Complement found from 10236 to 10242\n",
      "Complement found from 14019 to 14025\n",
      "Complement found from 15281 to 15287\n",
      "Seq is: TTACCT\n",
      "8 TTACCT motifs detected in file\n",
      "\n",
      "Complement found from 1526 to 1532\n",
      "Complement found from 2131 to 2137\n",
      "Complement found from 3488 to 3494\n",
      "Complement found from 5428 to 5434\n",
      "Complement found from 5596 to 5602\n",
      "Complement found from 6415 to 6421\n",
      "Complement found from 7528 to 7534\n",
      "Complement found from 9238 to 9244\n",
      "Complement found from 12036 to 12042\n",
      "Complement found from 13425 to 13431\n",
      "Complement found from 13972 to 13978\n",
      "Complement found from 14615 to 14621\n",
      "Complement found from 14711 to 14717\n",
      "Complement found from 15933 to 15939\n",
      "Complement found from 16162 to 16168\n",
      "Complement found from 16179 to 16185\n",
      "Seq is: AAAACC\n",
      "16 AAAACC motifs detected in file\n",
      "\n",
      "Complement found from 700 to 706\n",
      "Complement found from 4664 to 4670\n",
      "Complement found from 7829 to 7835\n",
      "Complement found from 12456 to 12462\n",
      "Complement found from 13758 to 13764\n",
      "Complement found from 15805 to 15811\n",
      "Seq is: GCATCC\n",
      "6 GCATCC motifs detected in file\n",
      "\n",
      "Complement found from 1721 to 1727\n",
      "Complement found from 1906 to 1912\n",
      "Complement found from 4978 to 4984\n",
      "Complement found from 5932 to 5938\n",
      "Complement found from 7529 to 7535\n",
      "Complement found from 8121 to 8127\n",
      "Complement found from 8190 to 8196\n",
      "Complement found from 8959 to 8965\n",
      "Complement found from 9673 to 9679\n",
      "Complement found from 11126 to 11132\n",
      "Complement found from 13426 to 13432\n",
      "Complement found from 14712 to 14718\n",
      "Seq is: AAACCA\n",
      "12 AAACCA motifs detected in file\n",
      "\n",
      "Complement found from 7858 to 7864\n",
      "Complement found from 8615 to 8621\n",
      "Seq is: GATCCC\n",
      "2 GATCCC motifs detected in file\n",
      "\n",
      "Complement found from 3940 to 3946\n",
      "Complement found from 6993 to 6999\n",
      "Complement found from 15035 to 15041\n",
      "Complement found from 15086 to 15092\n",
      "Seq is: ACATCG\n",
      "4 ACATCG motifs detected in file\n",
      "\n",
      "Complement found from 6210 to 6216\n",
      "Seq is: GACTCT\n",
      "1 GACTCT motifs detected in file\n",
      "\n",
      "Complement found from 1186 to 1192\n",
      "Complement found from 6841 to 6847\n",
      "Complement found from 8936 to 8942\n",
      "Complement found from 10188 to 10194\n",
      "Complement found from 11136 to 11142\n",
      "Complement found from 11549 to 11555\n",
      "Complement found from 12080 to 12086\n",
      "Complement found from 12098 to 12104\n",
      "Complement found from 16421 to 16427\n",
      "Seq is: TATCCC\n",
      "9 TATCCC motifs detected in file\n",
      "\n",
      "Complement found from 828 to 834\n",
      "Complement found from 5850 to 5856\n",
      "Seq is: CTTTAG\n",
      "2 CTTTAG motifs detected in file\n",
      "\n",
      "Complement found from 2601 to 2607\n",
      "Complement found from 4801 to 4807\n",
      "Complement found from 9310 to 9316\n",
      "Complement found from 9880 to 9886\n",
      "Complement found from 9898 to 9904\n",
      "Complement found from 10511 to 10517\n",
      "Complement found from 13438 to 13444\n",
      "Complement found from 13813 to 13819\n",
      "Complement found from 15288 to 15294\n",
      "Complement found from 15712 to 15718\n",
      "Seq is: TCACTT\n",
      "10 TCACTT motifs detected in file\n",
      "\n",
      "Complement found from 9747 to 9753\n",
      "Complement found from 10062 to 10068\n",
      "Complement found from 14840 to 14846\n",
      "Seq is: ACTTCG\n",
      "3 ACTTCG motifs detected in file\n",
      "\n",
      "Seq is: GAGGAT\n",
      "0 GAGGAT motifs detected in file\n",
      "\n",
      "Complement found from 1701 to 1707\n",
      "Complement found from 2219 to 2225\n",
      "Complement found from 2783 to 2789\n",
      "Complement found from 4733 to 4739\n",
      "Complement found from 5105 to 5111\n",
      "Complement found from 7227 to 7233\n",
      "Complement found from 8380 to 8386\n",
      "Complement found from 8459 to 8465\n",
      "Complement found from 10126 to 10132\n",
      "Complement found from 10791 to 10797\n",
      "Complement found from 10963 to 10969\n",
      "Complement found from 13849 to 13855\n",
      "Seq is: ACTACC\n",
      "12 ACTACC motifs detected in file\n",
      "\n",
      "Complement found from 576 to 582\n",
      "Complement found from 6629 to 6635\n",
      "Complement found from 6754 to 6760\n",
      "Seq is: GTTTAT\n",
      "3 GTTTAT motifs detected in file\n",
      "\n",
      "Seq is: GTCGAG\n",
      "0 GTCGAG motifs detected in file\n",
      "\n",
      "Complement found from 2933 to 2939\n",
      "Complement found from 8696 to 8702\n",
      "Complement found from 12275 to 12281\n",
      "Seq is: GATAAC\n",
      "3 GATAAC motifs detected in file\n",
      "\n",
      "Complement found from 4687 to 4693\n",
      "Complement found from 5095 to 5101\n",
      "Complement found from 6640 to 6646\n",
      "Complement found from 7782 to 7788\n",
      "Complement found from 8743 to 8749\n",
      "Complement found from 9125 to 9131\n",
      "Complement found from 10548 to 10554\n",
      "Complement found from 12784 to 12790\n",
      "Complement found from 12902 to 12908\n",
      "Complement found from 15096 to 15102\n",
      "Seq is: TATCCT\n",
      "10 TATCCT motifs detected in file\n",
      "\n",
      "Complement found from 9629 to 9635\n",
      "Seq is: GTATCA\n",
      "1 GTATCA motifs detected in file\n",
      "\n",
      "Complement found from 5540 to 5546\n",
      "Complement found from 9759 to 9765\n",
      "Complement found from 14287 to 14293\n",
      "Complement found from 14988 to 14994\n",
      "Complement found from 15125 to 15131\n",
      "Complement found from 15302 to 15308\n",
      "Seq is: CCTTCA\n",
      "6 CCTTCA motifs detected in file\n",
      "\n",
      "Complement found from 1223 to 1229\n",
      "Complement found from 3006 to 3012\n",
      "Complement found from 4138 to 4144\n",
      "Complement found from 6170 to 6176\n",
      "Complement found from 7232 to 7238\n",
      "Complement found from 8016 to 8022\n",
      "Complement found from 11159 to 11165\n",
      "Seq is: CCCGAT\n",
      "7 CCCGAT motifs detected in file\n",
      "\n",
      "Complement found from 5559 to 5565\n",
      "Seq is: GTTGCA\n",
      "1 GTTGCA motifs detected in file\n",
      "\n",
      "Complement found from 8201 to 8207\n",
      "Complement found from 9034 to 9040\n",
      "Complement found from 12231 to 12237\n",
      "Seq is: TCATGC\n",
      "3 TCATGC motifs detected in file\n",
      "\n",
      "Complement found from 325 to 331\n",
      "Complement found from 813 to 819\n",
      "Complement found from 2708 to 2714\n",
      "Complement found from 4128 to 4134\n",
      "Complement found from 6473 to 6479\n",
      "Complement found from 7289 to 7295\n",
      "Complement found from 12831 to 12837\n",
      "Complement found from 14036 to 14042\n",
      "Complement found from 14360 to 14366\n",
      "Complement found from 16209 to 16215\n",
      "Seq is: ACAGCA\n",
      "10 ACAGCA motifs detected in file\n",
      "\n",
      "Complement found from 276 to 282\n",
      "Complement found from 3746 to 3752\n",
      "Complement found from 3835 to 3841\n",
      "Complement found from 4749 to 4755\n",
      "Complement found from 4951 to 4957\n",
      "Complement found from 5296 to 5302\n",
      "Complement found from 5308 to 5314\n",
      "Complement found from 6118 to 6124\n",
      "Complement found from 7066 to 7072\n",
      "Complement found from 7794 to 7800\n",
      "Complement found from 10342 to 10348\n",
      "Complement found from 10863 to 10869\n",
      "Complement found from 11778 to 11784\n",
      "Complement found from 12114 to 12120\n",
      "Complement found from 12632 to 12638\n",
      "Complement found from 13370 to 13376\n",
      "Complement found from 13517 to 13523\n",
      "Complement found from 14057 to 14063\n",
      "Complement found from 14342 to 14348\n",
      "Complement found from 14481 to 14487\n",
      "Complement found from 14667 to 14673\n",
      "Complement found from 15789 to 15795\n",
      "Seq is: CATCAT\n",
      "22 CATCAT motifs detected in file\n",
      "\n",
      "Complement found from 1288 to 1294\n",
      "Complement found from 1361 to 1367\n",
      "Complement found from 7107 to 7113\n",
      "Complement found from 8140 to 8146\n",
      "Complement found from 10143 to 10149\n",
      "Seq is: GCTACA\n",
      "5 GCTACA motifs detected in file\n",
      "\n",
      "Complement found from 0 to 6\n",
      "Complement found from 950 to 956\n",
      "Complement found from 7657 to 7663\n",
      "Seq is: GATCAC\n",
      "3 GATCAC motifs detected in file\n",
      "\n",
      "Complement found from 2886 to 2892\n",
      "Complement found from 3645 to 3651\n",
      "Complement found from 4745 to 4751\n",
      "Complement found from 5121 to 5127\n",
      "Complement found from 8971 to 8977\n",
      "Complement found from 9031 to 9037\n",
      "Complement found from 11237 to 11243\n",
      "Complement found from 11291 to 11297\n",
      "Complement found from 12697 to 12703\n",
      "Complement found from 13111 to 13117\n",
      "Complement found from 13420 to 13426\n",
      "Complement found from 14908 to 14914\n",
      "Complement found from 15070 to 15076\n",
      "Complement found from 15247 to 15253\n",
      "Complement found from 15872 to 15878\n",
      "Seq is: TACTCA\n",
      "15 TACTCA motifs detected in file\n",
      "\n",
      "Complement found from 3043 to 3049\n",
      "Complement found from 7855 to 7861\n",
      "Seq is: AACGAT\n",
      "2 AACGAT motifs detected in file\n",
      "\n",
      "Seq is: TATGTT\n",
      "0 TATGTT motifs detected in file\n",
      "\n",
      "Complement found from 8162 to 8168\n",
      "Seq is: ACGGTC\n",
      "1 ACGGTC motifs detected in file\n",
      "\n",
      "Complement found from 3916 to 3922\n",
      "Seq is: AGTCCG\n",
      "1 AGTCCG motifs detected in file\n",
      "\n",
      "Complement found from 1418 to 1424\n",
      "Complement found from 4971 to 4977\n",
      "Seq is: GTGGAT\n",
      "2 GTGGAT motifs detected in file\n",
      "\n",
      "Complement found from 259 to 265\n",
      "Complement found from 5630 to 5636\n",
      "Complement found from 10309 to 10315\n",
      "Complement found from 11003 to 11009\n",
      "Seq is: GCCACT\n",
      "4 GCCACT motifs detected in file\n",
      "\n",
      "Complement found from 6938 to 6944\n",
      "Complement found from 7174 to 7180\n",
      "Complement found from 9481 to 9487\n",
      "Complement found from 14105 to 14111\n",
      "Seq is: TTTCTT\n",
      "4 TTTCTT motifs detected in file\n",
      "\n",
      "Complement found from 3554 to 3560\n",
      "Complement found from 3752 to 3758\n",
      "Complement found from 4494 to 4500\n",
      "Complement found from 5337 to 5343\n",
      "Complement found from 5358 to 5364\n",
      "Complement found from 5725 to 5731\n",
      "Complement found from 9116 to 9122\n",
      "Complement found from 10872 to 10878\n",
      "Complement found from 11286 to 11292\n",
      "Complement found from 11808 to 11814\n",
      "Complement found from 12695 to 12701\n",
      "Complement found from 13501 to 13507\n",
      "Complement found from 15068 to 15074\n",
      "Seq is: TCTACT\n",
      "13 TCTACT motifs detected in file\n",
      "\n",
      "Complement found from 5028 to 5034\n",
      "Complement found from 6927 to 6933\n",
      "Complement found from 14255 to 14261\n",
      "Complement found from 16270 to 16276\n",
      "Seq is: TAGGAT\n",
      "4 TAGGAT motifs detected in file\n",
      "\n",
      "Complement found from 12814 to 12820\n",
      "Complement found from 14156 to 14162\n",
      "Seq is: CCCGAG\n",
      "2 CCCGAG motifs detected in file\n",
      "\n",
      "Complement found from 16356 to 16362\n",
      "Seq is: TCTCGT\n",
      "1 TCTCGT motifs detected in file\n",
      "\n",
      "Complement found from 8056 to 8062\n",
      "Complement found from 8114 to 8120\n",
      "Seq is: GACGTC\n",
      "2 GACGTC motifs detected in file\n",
      "\n",
      "Seq is: GTCTAG\n",
      "0 GTCTAG motifs detected in file\n",
      "\n",
      "Complement found from 198 to 204\n",
      "Complement found from 1737 to 1743\n",
      "Complement found from 3049 to 3055\n",
      "Complement found from 4398 to 4404\n",
      "Complement found from 14318 to 14324\n",
      "Complement found from 16478 to 16484\n",
      "Seq is: TAAAGT\n",
      "6 TAAAGT motifs detected in file\n",
      "\n",
      "Complement found from 9323 to 9329\n",
      "Seq is: ATAACG\n",
      "1 ATAACG motifs detected in file\n",
      "\n",
      "Complement found from 4827 to 4833\n",
      "Complement found from 8917 to 8923\n",
      "Complement found from 14077 to 14083\n",
      "Seq is: AAGGCA\n",
      "3 AAGGCA motifs detected in file\n",
      "\n",
      "Complement found from 1232 to 1238\n",
      "Complement found from 2253 to 2259\n",
      "Complement found from 2413 to 2419\n",
      "Complement found from 4013 to 4019\n",
      "Complement found from 4648 to 4654\n",
      "Complement found from 4908 to 4914\n",
      "Complement found from 4930 to 4936\n",
      "Complement found from 5889 to 5895\n",
      "Complement found from 8474 to 8480\n",
      "Complement found from 8790 to 8796\n",
      "Complement found from 9847 to 9853\n",
      "Complement found from 10980 to 10986\n",
      "Complement found from 11493 to 11499\n",
      "Complement found from 12962 to 12968\n",
      "Complement found from 13451 to 13457\n",
      "Complement found from 13490 to 13496\n",
      "Complement found from 15267 to 15273\n",
      "Complement found from 16260 to 16266\n",
      "Seq is: CCTCAC\n",
      "18 CCTCAC motifs detected in file\n",
      "\n",
      "Complement found from 522 to 528\n",
      "Complement found from 1660 to 1666\n",
      "Complement found from 8137 to 8143\n",
      "Complement found from 9006 to 9012\n",
      "Complement found from 12732 to 12738\n",
      "Complement found from 14565 to 14571\n",
      "Complement found from 16080 to 16086\n",
      "Seq is: ACCGCT\n",
      "7 ACCGCT motifs detected in file\n",
      "\n",
      "Complement found from 11772 to 11778\n",
      "Seq is: CAGTCG\n",
      "1 CAGTCG motifs detected in file\n",
      "\n",
      "Complement found from 155 to 161\n",
      "Complement found from 1539 to 1545\n",
      "Complement found from 1973 to 1979\n",
      "Complement found from 2740 to 2746\n",
      "Complement found from 3614 to 3620\n",
      "Complement found from 3860 to 3866\n",
      "Complement found from 5087 to 5093\n",
      "Complement found from 9460 to 9466\n",
      "Complement found from 12899 to 12905\n",
      "Complement found from 13352 to 13358\n",
      "Seq is: ATTTAT\n",
      "10 ATTTAT motifs detected in file\n",
      "\n",
      "Complement found from 1400 to 1406\n",
      "Complement found from 14837 to 14843\n",
      "Seq is: GAAACT\n",
      "2 GAAACT motifs detected in file\n",
      "\n",
      "Seq is: GCGGCC\n",
      "0 GCGGCC motifs detected in file\n",
      "\n",
      "Complement found from 1529 to 1535\n",
      "Complement found from 1835 to 1841\n",
      "Complement found from 4832 to 4838\n",
      "Complement found from 5842 to 5848\n",
      "Complement found from 8930 to 8936\n",
      "Complement found from 9253 to 9259\n",
      "Complement found from 10160 to 10166\n",
      "Complement found from 10276 to 10282\n",
      "Complement found from 10977 to 10983\n",
      "Complement found from 11534 to 11540\n",
      "Complement found from 12192 to 12198\n",
      "Complement found from 13024 to 13030\n",
      "Complement found from 14279 to 14285\n",
      "Complement found from 14416 to 14422\n",
      "Complement found from 15524 to 15530\n",
      "Complement found from 15535 to 15541\n",
      "Complement found from 16257 to 16263\n",
      "Seq is: ACCCCT\n",
      "17 ACCCCT motifs detected in file\n",
      "\n",
      "Complement found from 3546 to 3552\n",
      "Complement found from 6836 to 6842\n",
      "Complement found from 9135 to 9141\n",
      "Complement found from 10533 to 10539\n",
      "Complement found from 10582 to 10588\n",
      "Complement found from 11431 to 11437\n",
      "Complement found from 11607 to 11613\n",
      "Complement found from 13575 to 13581\n",
      "Complement found from 14380 to 14386\n",
      "Complement found from 14455 to 14461\n",
      "Seq is: ATCGCT\n",
      "10 ATCGCT motifs detected in file\n",
      "\n",
      "Complement found from 474 to 480\n",
      "Complement found from 2293 to 2299\n",
      "Complement found from 3767 to 3773\n",
      "Complement found from 4465 to 4471\n",
      "Complement found from 5497 to 5503\n",
      "Complement found from 7735 to 7741\n",
      "Complement found from 7929 to 7935\n",
      "Complement found from 8648 to 8654\n",
      "Complement found from 8669 to 8675\n",
      "Complement found from 9871 to 9877\n",
      "Complement found from 10105 to 10111\n",
      "Complement found from 10312 to 10318\n",
      "Complement found from 10767 to 10773\n",
      "Complement found from 11052 to 11058\n",
      "Complement found from 11100 to 11106\n",
      "Complement found from 11247 to 11253\n",
      "Complement found from 11817 to 11823\n",
      "Complement found from 12719 to 12725\n",
      "Complement found from 13148 to 13154\n",
      "Complement found from 14215 to 14221\n",
      "Complement found from 15902 to 15908\n",
      "Seq is: ACTAAT\n",
      "21 ACTAAT motifs detected in file\n",
      "\n",
      "Complement found from 3504 to 3510\n",
      "Complement found from 3543 to 3549\n",
      "Complement found from 4502 to 4508\n",
      "Complement found from 5306 to 5312\n",
      "Complement found from 5318 to 5324\n",
      "Complement found from 6338 to 6344\n",
      "Complement found from 7870 to 7876\n",
      "Complement found from 8961 to 8967\n",
      "Complement found from 10505 to 10511\n",
      "Complement found from 14479 to 14485\n",
      "Complement found from 14714 to 14720\n",
      "Complement found from 15787 to 15793\n",
      "Complement found from 16401 to 16407\n",
      "Seq is: ACCATC\n",
      "13 ACCATC motifs detected in file\n",
      "\n",
      "Complement found from 3607 to 3613\n",
      "Complement found from 6003 to 6009\n",
      "Complement found from 6323 to 6329\n",
      "Complement found from 7497 to 7503\n",
      "Complement found from 12979 to 12985\n",
      "Seq is: GCCTCC\n",
      "5 GCCTCC motifs detected in file\n",
      "\n",
      "Complement found from 2044 to 2050\n",
      "Complement found from 3268 to 3274\n",
      "Complement found from 5633 to 5639\n",
      "Complement found from 9882 to 9888\n",
      "Complement found from 14090 to 14096\n",
      "Complement found from 15714 to 15720\n",
      "Seq is: ACTTTA\n",
      "6 ACTTTA motifs detected in file\n",
      "\n",
      "Complement found from 1143 to 1149\n",
      "Complement found from 7906 to 7912\n",
      "Complement found from 10166 to 10172\n",
      "Complement found from 13964 to 13970\n",
      "Seq is: TACGAG\n",
      "4 TACGAG motifs detected in file\n",
      "\n",
      "Complement found from 268 to 274\n",
      "Complement found from 513 to 519\n",
      "Complement found from 1047 to 1053\n",
      "Complement found from 7409 to 7415\n",
      "Complement found from 9416 to 9422\n",
      "Complement found from 13931 to 13937\n",
      "Complement found from 16231 to 16237\n",
      "Seq is: CACACA\n",
      "7 CACACA motifs detected in file\n",
      "\n",
      "Complement found from 7235 to 7241\n",
      "Complement found from 8101 to 8107\n",
      "Seq is: GATGCA\n",
      "2 GATGCA motifs detected in file\n",
      "\n",
      "Seq is: GATAAG\n",
      "0 GATAAG motifs detected in file\n",
      "\n",
      "Complement found from 11831 to 11837\n",
      "Complement found from 16369 to 16375\n",
      "Seq is: GATGAC\n",
      "2 GATGAC motifs detected in file\n",
      "\n",
      "Complement found from 155 to 161\n",
      "Complement found from 1539 to 1545\n",
      "Complement found from 1973 to 1979\n",
      "Complement found from 2740 to 2746\n",
      "Complement found from 3614 to 3620\n",
      "Complement found from 3860 to 3866\n",
      "Complement found from 5087 to 5093\n",
      "Complement found from 9460 to 9466\n",
      "Complement found from 12899 to 12905\n",
      "Complement found from 13352 to 13358\n",
      "Seq is: ATTTAT\n",
      "10 ATTTAT motifs detected in file\n",
      "\n",
      "Complement found from 4894 to 4900\n",
      "Complement found from 6436 to 6442\n",
      "Complement found from 6592 to 6598\n",
      "Complement found from 9367 to 9373\n",
      "Complement found from 9403 to 9409\n",
      "Complement found from 15841 to 15847\n",
      "Complement found from 16274 to 16280\n",
      "Seq is: ATACCA\n",
      "7 ATACCA motifs detected in file\n",
      "\n",
      "Complement found from 2639 to 2645\n",
      "Complement found from 6804 to 6810\n",
      "Complement found from 9390 to 9396\n",
      "Complement found from 12050 to 12056\n",
      "Seq is: CACGAG\n",
      "4 CACGAG motifs detected in file\n",
      "\n",
      "Complement found from 11898 to 11904\n",
      "Complement found from 13357 to 13363\n",
      "Seq is: TGTGCT\n",
      "2 TGTGCT motifs detected in file\n",
      "\n",
      "Complement found from 48 to 54\n",
      "Complement found from 6703 to 6709\n",
      "Complement found from 16042 to 16048\n",
      "Seq is: ATTTGG\n",
      "3 ATTTGG motifs detected in file\n",
      "\n",
      "Complement found from 14258 to 14264\n",
      "Complement found from 14868 to 14874\n",
      "Seq is: GATCCT\n",
      "2 GATCCT motifs detected in file\n",
      "\n",
      "Complement found from 2807 to 2813\n",
      "Complement found from 3091 to 3097\n",
      "Complement found from 12874 to 12880\n",
      "Seq is: TCGGTT\n",
      "3 TCGGTT motifs detected in file\n",
      "\n",
      "Complement found from 5083 to 5089\n",
      "Complement found from 7779 to 7785\n",
      "Complement found from 8815 to 8821\n",
      "Complement found from 11312 to 11318\n",
      "Complement found from 14508 to 14514\n",
      "Complement found from 15111 to 15117\n",
      "Complement found from 15846 to 15852\n",
      "Complement found from 16011 to 16017\n",
      "Complement found from 16225 to 16231\n",
      "Seq is: AACTAT\n",
      "9 AACTAT motifs detected in file\n",
      "\n",
      "Complement found from 122 to 128\n",
      "Complement found from 8741 to 8747\n",
      "Complement found from 9628 to 9634\n",
      "Seq is: AGTATC\n",
      "3 AGTATC motifs detected in file\n",
      "\n",
      "Complement found from 9752 to 9758\n",
      "Seq is: GAGTCT\n",
      "1 GAGTCT motifs detected in file\n",
      "\n",
      "Complement found from 69 to 75\n",
      "Complement found from 411 to 417\n",
      "Complement found from 6716 to 6722\n",
      "Seq is: GGTATG\n",
      "3 GGTATG motifs detected in file\n",
      "\n",
      "Complement found from 6708 to 6714\n",
      "Seq is: GATACA\n",
      "1 GATACA motifs detected in file\n",
      "\n",
      "Complement found from 649 to 655\n",
      "Complement found from 5518 to 5524\n",
      "Seq is: TAGGTT\n",
      "2 TAGGTT motifs detected in file\n",
      "\n",
      "Complement found from 2498 to 2504\n",
      "Complement found from 2785 to 2791\n",
      "Complement found from 4735 to 4741\n",
      "Complement found from 4895 to 4901\n",
      "Complement found from 6437 to 6443\n",
      "Complement found from 6593 to 6599\n",
      "Complement found from 9368 to 9374\n",
      "Complement found from 9404 to 9410\n",
      "Complement found from 10463 to 10469\n",
      "Complement found from 15842 to 15848\n",
      "Complement found from 16275 to 16281\n",
      "Seq is: TACCAA\n",
      "11 TACCAA motifs detected in file\n",
      "\n",
      "Complement found from 1799 to 1805\n",
      "Complement found from 2128 to 2134\n",
      "Complement found from 3379 to 3385\n",
      "Complement found from 4174 to 4180\n",
      "Complement found from 5745 to 5751\n",
      "Complement found from 6690 to 6696\n",
      "Complement found from 7526 to 7532\n",
      "Complement found from 10151 to 10157\n",
      "Complement found from 11030 to 11036\n",
      "Complement found from 13405 to 13411\n",
      "Complement found from 14709 to 14715\n",
      "Complement found from 15960 to 15966\n",
      "Seq is: GAAAAA\n",
      "12 GAAAAA motifs detected in file\n",
      "\n",
      "Complement found from 3143 to 3149\n",
      "Complement found from 3409 to 3415\n",
      "Complement found from 9408 to 9414\n",
      "Complement found from 9435 to 9441\n",
      "Complement found from 13048 to 13054\n",
      "Seq is: AAGGCC\n",
      "5 AAGGCC motifs detected in file\n",
      "\n",
      "Complement found from 481 to 487\n",
      "Complement found from 4521 to 4527\n",
      "Complement found from 4748 to 4754\n",
      "Complement found from 5295 to 5301\n",
      "Complement found from 6393 to 6399\n",
      "Complement found from 6983 to 6989\n",
      "Complement found from 8429 to 8435\n",
      "Complement found from 8635 to 8641\n",
      "Complement found from 10341 to 10347\n",
      "Complement found from 11153 to 11159\n",
      "Complement found from 12796 to 12802\n",
      "Complement found from 14059 to 14065\n",
      "Complement found from 14935 to 14941\n",
      "Seq is: TCATCA\n",
      "13 TCATCA motifs detected in file\n",
      "\n",
      "Complement found from 171 to 177\n",
      "Complement found from 3284 to 3290\n",
      "Complement found from 5720 to 5726\n",
      "Complement found from 5800 to 5806\n",
      "Complement found from 11625 to 11631\n",
      "Complement found from 15227 to 15233\n",
      "Seq is: TTCAAT\n",
      "6 TTCAAT motifs detected in file\n",
      "\n",
      "Complement found from 4449 to 4455\n",
      "Seq is: GTTATA\n",
      "1 GTTATA motifs detected in file\n",
      "\n",
      "Complement found from 723 to 729\n",
      "Complement found from 1189 to 1195\n",
      "Complement found from 2065 to 2071\n",
      "Complement found from 3511 to 3517\n",
      "Complement found from 4834 to 4840\n",
      "Complement found from 6221 to 6227\n",
      "Complement found from 6447 to 6453\n",
      "Complement found from 8273 to 8279\n",
      "Complement found from 8282 to 8288\n",
      "Complement found from 8606 to 8612\n",
      "Complement found from 9084 to 9090\n",
      "Complement found from 9267 to 9273\n",
      "Complement found from 10329 to 10335\n",
      "Complement found from 10626 to 10632\n",
      "Complement found from 10869 to 10875\n",
      "Complement found from 11978 to 11984\n",
      "Complement found from 13783 to 13789\n",
      "Complement found from 14281 to 14287\n",
      "Seq is: CCCTCT\n",
      "18 CCCTCT motifs detected in file\n",
      "\n",
      "Complement found from 3140 to 3146\n",
      "Complement found from 3770 to 3776\n",
      "Complement found from 4415 to 4421\n",
      "Complement found from 16552 to 16558\n",
      "Seq is: AATAAG\n",
      "4 AATAAG motifs detected in file\n",
      "\n",
      "Complement found from 6651 to 6657\n",
      "Complement found from 7331 to 7337\n",
      "Complement found from 9906 to 9912\n",
      "Complement found from 10176 to 10182\n",
      "Seq is: GCTTCG\n",
      "4 GCTTCG motifs detected in file\n",
      "\n",
      "Seq is: GACCAT\n",
      "0 GACCAT motifs detected in file\n",
      "\n",
      "Complement found from 17 to 23\n",
      "Complement found from 149 to 155\n",
      "Complement found from 2947 to 2953\n",
      "Complement found from 3611 to 3617\n",
      "Complement found from 3968 to 3974\n",
      "Complement found from 5959 to 5965\n",
      "Complement found from 6601 to 6607\n",
      "Complement found from 7096 to 7102\n",
      "Complement found from 9457 to 9463\n",
      "Complement found from 10646 to 10652\n",
      "Complement found from 10902 to 10908\n",
      "Complement found from 11205 to 11211\n",
      "Complement found from 12743 to 12749\n",
      "Complement found from 13718 to 13724\n",
      "Complement found from 14148 to 14154\n",
      "Complement found from 15336 to 15342\n",
      "Complement found from 15567 to 15573\n",
      "Complement found from 15624 to 15630\n",
      "Seq is: CCTATT\n",
      "18 CCTATT motifs detected in file\n",
      "\n",
      "Complement found from 4206 to 4212\n",
      "Complement found from 5506 to 5512\n",
      "Complement found from 8733 to 8739\n",
      "Seq is: CTTATA\n",
      "3 CTTATA motifs detected in file\n",
      "\n",
      "Complement found from 5417 to 5423\n",
      "Complement found from 5773 to 5779\n",
      "Complement found from 8239 to 8245\n",
      "Complement found from 10826 to 10832\n",
      "Seq is: TTTGAA\n",
      "4 TTTGAA motifs detected in file\n",
      "\n",
      "Complement found from 4811 to 4817\n",
      "Complement found from 9590 to 9596\n",
      "Complement found from 10774 to 10780\n",
      "Complement found from 15260 to 15266\n",
      "Seq is: GTCCCA\n",
      "4 GTCCCA motifs detected in file\n",
      "\n",
      "Complement found from 5715 to 5721\n",
      "Seq is: CTGGCT\n",
      "1 CTGGCT motifs detected in file\n",
      "\n",
      "Complement found from 945 to 951\n",
      "Complement found from 9994 to 10000\n",
      "Seq is: TTTTAG\n",
      "2 TTTTAG motifs detected in file\n",
      "\n",
      "Complement found from 4114 to 4120\n",
      "Complement found from 4944 to 4950\n",
      "Complement found from 5505 to 5511\n",
      "Complement found from 6637 to 6643\n",
      "Complement found from 8732 to 8738\n",
      "Complement found from 10242 to 10248\n",
      "Complement found from 10332 to 10338\n",
      "Seq is: TCTTAT\n",
      "7 TCTTAT motifs detected in file\n",
      "\n",
      "Complement found from 5677 to 5683\n",
      "Complement found from 5935 to 5941\n",
      "Complement found from 7181 to 7187\n",
      "Complement found from 8560 to 8566\n",
      "Complement found from 8764 to 8770\n",
      "Complement found from 8913 to 8919\n",
      "Complement found from 10130 to 10136\n",
      "Complement found from 11993 to 11999\n",
      "Complement found from 12484 to 12490\n",
      "Complement found from 12541 to 12547\n",
      "Complement found from 12927 to 12933\n",
      "Complement found from 13376 to 13382\n",
      "Complement found from 14327 to 14333\n",
      "Complement found from 14621 to 14627\n",
      "Seq is: CCACAA\n",
      "14 CCACAA motifs detected in file\n",
      "\n",
      "Complement found from 2960 to 2966\n",
      "Complement found from 5385 to 5391\n",
      "Complement found from 6667 to 6673\n",
      "Complement found from 6772 to 6778\n",
      "Complement found from 9363 to 9369\n",
      "Complement found from 14519 to 14525\n",
      "Complement found from 15666 to 15672\n",
      "Seq is: CCATAT\n",
      "7 CCATAT motifs detected in file\n",
      "\n",
      "Complement found from 586 to 592\n",
      "Complement found from 2483 to 2489\n",
      "Complement found from 3369 to 3375\n",
      "Complement found from 5462 to 5468\n",
      "Complement found from 6214 to 6220\n",
      "Complement found from 7867 to 7873\n",
      "Complement found from 8909 to 8915\n",
      "Complement found from 11862 to 11868\n",
      "Complement found from 12393 to 12399\n",
      "Seq is: CTTACC\n",
      "9 CTTACC motifs detected in file\n",
      "\n",
      "Complement found from 1600 to 1606\n",
      "Complement found from 4814 to 4820\n",
      "Seq is: CCAGAG\n",
      "2 CCAGAG motifs detected in file\n",
      "\n",
      "Complement found from 1196 to 1202\n",
      "Complement found from 1548 to 1554\n",
      "Complement found from 6573 to 6579\n",
      "Complement found from 15239 to 15245\n",
      "Seq is: GAGGAG\n",
      "4 GAGGAG motifs detected in file\n",
      "\n",
      "Complement found from 5737 to 5743\n",
      "Complement found from 9914 to 9920\n",
      "Complement found from 10667 to 10673\n",
      "Seq is: GCCGCC\n",
      "3 GCCGCC motifs detected in file\n",
      "\n",
      "Complement found from 4987 to 4993\n",
      "Complement found from 12557 to 12563\n",
      "Seq is: CCAGCT\n",
      "2 CCAGCT motifs detected in file\n",
      "\n",
      "Complement found from 4027 to 4033\n",
      "Complement found from 7176 to 7182\n",
      "Complement found from 12478 to 12484\n",
      "Complement found from 12703 to 12709\n",
      "Complement found from 14110 to 14116\n",
      "Complement found from 15026 to 15032\n",
      "Complement found from 15446 to 15452\n",
      "Seq is: TCTTCC\n",
      "7 TCTTCC motifs detected in file\n",
      "\n",
      "Complement found from 5934 to 5940\n",
      "Complement found from 6054 to 6060\n",
      "Complement found from 7244 to 7250\n",
      "Complement found from 7407 to 7413\n",
      "Complement found from 8192 to 8198\n",
      "Complement found from 8912 to 8918\n",
      "Complement found from 9414 to 9420\n",
      "Complement found from 10129 to 10135\n",
      "Complement found from 11128 to 11134\n",
      "Complement found from 11992 to 11998\n",
      "Complement found from 12020 to 12026\n",
      "Complement found from 13298 to 13304\n",
      "Complement found from 13513 to 13519\n",
      "Complement found from 14326 to 14332\n",
      "Complement found from 14560 to 14566\n",
      "Complement found from 14881 to 14887\n",
      "Seq is: ACCACA\n",
      "16 ACCACA motifs detected in file\n",
      "\n",
      "Complement found from 997 to 1003\n",
      "Complement found from 1691 to 1697\n",
      "Complement found from 5133 to 5139\n",
      "Complement found from 5361 to 5367\n",
      "Complement found from 6880 to 6886\n",
      "Complement found from 9318 to 9324\n",
      "Complement found from 9513 to 9519\n",
      "Complement found from 10753 to 10759\n",
      "Complement found from 11391 to 11397\n",
      "Complement found from 12320 to 12326\n",
      "Complement found from 12910 to 12916\n",
      "Complement found from 13073 to 13079\n",
      "Complement found from 13504 to 13510\n",
      "Complement found from 15327 to 15333\n",
      "Complement found from 15973 to 15979\n",
      "Complement found from 16246 to 16252\n",
      "Seq is: ACTCCA\n",
      "16 ACTCCA motifs detected in file\n",
      "\n",
      "Complement found from 3457 to 3463\n",
      "Complement found from 7128 to 7134\n",
      "Complement found from 11001 to 11007\n",
      "Complement found from 14993 to 14999\n",
      "Seq is: ACGCCA\n",
      "4 ACGCCA motifs detected in file\n",
      "\n",
      "Complement found from 2365 to 2371\n",
      "Complement found from 4713 to 4719\n",
      "Complement found from 7999 to 8005\n",
      "Complement found from 15505 to 15511\n",
      "Seq is: GACAAT\n",
      "4 GACAAT motifs detected in file\n",
      "\n",
      "Complement found from 276 to 282\n",
      "Complement found from 3746 to 3752\n",
      "Complement found from 3835 to 3841\n",
      "Complement found from 4749 to 4755\n",
      "Complement found from 4951 to 4957\n",
      "Complement found from 5296 to 5302\n",
      "Complement found from 5308 to 5314\n",
      "Complement found from 6118 to 6124\n",
      "Complement found from 7066 to 7072\n",
      "Complement found from 7794 to 7800\n",
      "Complement found from 10342 to 10348\n",
      "Complement found from 10863 to 10869\n",
      "Complement found from 11778 to 11784\n",
      "Complement found from 12114 to 12120\n",
      "Complement found from 12632 to 12638\n",
      "Complement found from 13370 to 13376\n",
      "Complement found from 13517 to 13523\n",
      "Complement found from 14057 to 14063\n",
      "Complement found from 14342 to 14348\n",
      "Complement found from 14481 to 14487\n",
      "Complement found from 14667 to 14673\n",
      "Complement found from 15789 to 15795\n",
      "Seq is: CATCAT\n",
      "22 CATCAT motifs detected in file\n",
      "\n",
      "Complement found from 2600 to 2606\n",
      "Complement found from 6518 to 6524\n",
      "Complement found from 6985 to 6991\n",
      "Complement found from 9897 to 9903\n",
      "Complement found from 11929 to 11935\n",
      "Complement found from 14950 to 14956\n",
      "Complement found from 15711 to 15717\n",
      "Seq is: ATCACT\n",
      "7 ATCACT motifs detected in file\n",
      "\n",
      "Complement found from 8162 to 8168\n",
      "Seq is: ACGGTC\n",
      "1 ACGGTC motifs detected in file\n",
      "\n",
      "Seq is: GTCGAT\n",
      "0 GTCGAT motifs detected in file\n",
      "\n",
      "Complement found from 6958 to 6964\n",
      "Complement found from 10967 to 10973\n",
      "Complement found from 11514 to 11520\n",
      "Complement found from 12368 to 12374\n",
      "Complement found from 13027 to 13033\n",
      "Complement found from 13586 to 13592\n",
      "Complement found from 14002 to 14008\n",
      "Complement found from 14275 to 14281\n",
      "Complement found from 14419 to 14425\n",
      "Seq is: CCTGAC\n",
      "9 CCTGAC motifs detected in file\n",
      "\n",
      "Complement found from 3788 to 3794\n",
      "Complement found from 5212 to 5218\n",
      "Complement found from 6222 to 6228\n",
      "Complement found from 9268 to 9274\n",
      "Complement found from 11043 to 11049\n",
      "Complement found from 11787 to 11793\n",
      "Complement found from 13434 to 13440\n",
      "Complement found from 14099 to 14105\n",
      "Complement found from 14282 to 14288\n",
      "Seq is: CCTCTC\n",
      "9 CCTCTC motifs detected in file\n",
      "\n",
      "Complement found from 4687 to 4693\n",
      "Complement found from 5095 to 5101\n",
      "Complement found from 6640 to 6646\n",
      "Complement found from 7782 to 7788\n",
      "Complement found from 8743 to 8749\n",
      "Complement found from 9125 to 9131\n",
      "Complement found from 10548 to 10554\n",
      "Complement found from 12784 to 12790\n",
      "Complement found from 12902 to 12908\n",
      "Complement found from 15096 to 15102\n",
      "Seq is: TATCCT\n",
      "10 TATCCT motifs detected in file\n",
      "\n",
      "Complement found from 1181 to 1187\n",
      "Complement found from 3974 to 3980\n",
      "Complement found from 6103 to 6109\n",
      "Complement found from 7078 to 7084\n",
      "Complement found from 8545 to 8551\n",
      "Complement found from 9859 to 9865\n",
      "Complement found from 14288 to 14294\n",
      "Complement found from 15126 to 15132\n",
      "Complement found from 15291 to 15297\n",
      "Complement found from 15303 to 15309\n",
      "Seq is: CTTCAT\n",
      "10 CTTCAT motifs detected in file\n",
      "\n",
      "Complement found from 6958 to 6964\n",
      "Complement found from 10967 to 10973\n",
      "Complement found from 11514 to 11520\n",
      "Complement found from 12368 to 12374\n",
      "Complement found from 13027 to 13033\n",
      "Complement found from 13586 to 13592\n",
      "Complement found from 14002 to 14008\n",
      "Complement found from 14275 to 14281\n",
      "Complement found from 14419 to 14425\n",
      "Seq is: CCTGAC\n",
      "9 CCTGAC motifs detected in file\n",
      "\n",
      "Complement found from 2120 to 2126\n",
      "Complement found from 12532 to 12538\n",
      "Seq is: GACACT\n",
      "2 GACACT motifs detected in file\n",
      "\n",
      "Complement found from 2944 to 2950\n",
      "Complement found from 3650 to 3656\n",
      "Complement found from 4672 to 4678\n",
      "Complement found from 7569 to 7575\n",
      "Complement found from 8564 to 8570\n",
      "Complement found from 9454 to 9460\n",
      "Complement found from 11784 to 11790\n",
      "Complement found from 12848 to 12854\n",
      "Complement found from 14368 to 14374\n",
      "Complement found from 15828 to 15834\n",
      "Complement found from 15834 to 15840\n",
      "Seq is: AATCCT\n",
      "11 AATCCT motifs detected in file\n",
      "\n",
      "Complement found from 790 to 796\n",
      "Complement found from 3634 to 3640\n",
      "Complement found from 4563 to 4569\n",
      "Complement found from 9294 to 9300\n",
      "Complement found from 9519 to 9525\n",
      "Complement found from 10563 to 10569\n",
      "Complement found from 10689 to 10695\n",
      "Complement found from 10725 to 10731\n",
      "Complement found from 11741 to 11747\n",
      "Complement found from 12502 to 12508\n",
      "Complement found from 13465 to 13471\n",
      "Seq is: GCCTAG\n",
      "11 GCCTAG motifs detected in file\n",
      "\n",
      "Complement found from 6245 to 6251\n",
      "Complement found from 6905 to 6911\n",
      "Complement found from 7685 to 7691\n",
      "Complement found from 9855 to 9861\n",
      "Seq is: TCTGCT\n",
      "4 TCTGCT motifs detected in file\n",
      "\n",
      "Complement found from 4054 to 4060\n",
      "Complement found from 4904 to 4910\n",
      "Complement found from 5214 to 5220\n",
      "Complement found from 5484 to 5490\n",
      "Complement found from 5731 to 5737\n",
      "Complement found from 6496 to 6502\n",
      "Complement found from 6663 to 6669\n",
      "Complement found from 9755 to 9761\n",
      "Complement found from 11057 to 11063\n",
      "Complement found from 12562 to 12568\n",
      "Complement found from 15851 to 15857\n",
      "Seq is: TCTCCC\n",
      "11 TCTCCC motifs detected in file\n",
      "\n",
      "Complement found from 3405 to 3411\n",
      "Complement found from 4994 to 5000\n",
      "Complement found from 5621 to 5627\n",
      "Complement found from 13529 to 13535\n",
      "Complement found from 14758 to 14764\n",
      "Seq is: CGCAAA\n",
      "5 CGCAAA motifs detected in file\n",
      "\n",
      "Complement found from 638 to 644\n",
      "Complement found from 3460 to 3466\n",
      "Complement found from 3850 to 3856\n",
      "Complement found from 4668 to 4674\n",
      "Complement found from 4722 to 4728\n",
      "Complement found from 6426 to 6432\n",
      "Complement found from 6828 to 6834\n",
      "Complement found from 7368 to 7374\n",
      "Complement found from 8396 to 8402\n",
      "Complement found from 8486 to 8492\n",
      "Complement found from 9321 to 9327\n",
      "Complement found from 10212 to 10218\n",
      "Complement found from 12592 to 12598\n",
      "Complement found from 14227 to 14233\n",
      "Complement found from 14590 to 14596\n",
      "Complement found from 16132 to 16138\n",
      "Complement found from 16458 to 16464\n",
      "Seq is: CCATAA\n",
      "17 CCATAA motifs detected in file\n",
      "\n",
      "Complement found from 962 to 968\n",
      "Complement found from 2260 to 2266\n",
      "Complement found from 2377 to 2383\n",
      "Complement found from 6432 to 6438\n",
      "Complement found from 9534 to 9540\n",
      "Complement found from 13009 to 13015\n",
      "Complement found from 14751 to 14757\n",
      "Complement found from 16166 to 16172\n",
      "Seq is: CCCAAT\n",
      "8 CCCAAT motifs detected in file\n",
      "\n",
      "Complement found from 7771 to 7777\n",
      "Seq is: ACCGTC\n",
      "1 ACCGTC motifs detected in file\n",
      "\n",
      "Complement found from 352 to 358\n",
      "Complement found from 434 to 440\n",
      "Complement found from 556 to 562\n",
      "Complement found from 961 to 967\n",
      "Complement found from 1681 to 1687\n",
      "Complement found from 3413 to 3419\n",
      "Complement found from 7467 to 7473\n",
      "Complement found from 8368 to 8374\n",
      "Complement found from 9533 to 9539\n",
      "Complement found from 9555 to 9561\n",
      "Complement found from 10916 to 10922\n",
      "Complement found from 12301 to 12307\n",
      "Complement found from 14531 to 14537\n",
      "Complement found from 14750 to 14756\n",
      "Seq is: CCCCAA\n",
      "14 CCCCAA motifs detected in file\n",
      "\n",
      "Complement found from 1819 to 1825\n",
      "Complement found from 4700 to 4706\n",
      "Complement found from 6410 to 6416\n",
      "Complement found from 12140 to 12146\n",
      "Complement found from 14175 to 14181\n",
      "Seq is: AATATA\n",
      "5 AATATA motifs detected in file\n",
      "\n",
      "Complement found from 5084 to 5090\n",
      "Complement found from 5920 to 5926\n",
      "Complement found from 8421 to 8427\n",
      "Complement found from 9952 to 9958\n",
      "Complement found from 10875 to 10881\n",
      "Complement found from 11733 to 11739\n",
      "Complement found from 11872 to 11878\n",
      "Complement found from 13349 to 13355\n",
      "Complement found from 14313 to 14319\n",
      "Complement found from 14509 to 14515\n",
      "Complement found from 14889 to 14895\n",
      "Complement found from 15474 to 15480\n",
      "Complement found from 16012 to 16018\n",
      "Seq is: ACTATT\n",
      "13 ACTATT motifs detected in file\n",
      "\n",
      "Complement found from 3300 to 3306\n",
      "Complement found from 4037 to 4043\n",
      "Complement found from 5393 to 5399\n",
      "Complement found from 6194 to 6200\n",
      "Complement found from 8640 to 8646\n",
      "Complement found from 9665 to 9671\n",
      "Complement found from 10297 to 10303\n",
      "Complement found from 10894 to 10900\n",
      "Complement found from 10941 to 10947\n",
      "Complement found from 11332 to 11338\n",
      "Complement found from 12028 to 12034\n",
      "Complement found from 12247 to 12253\n",
      "Complement found from 12551 to 12557\n",
      "Complement found from 12738 to 12744\n",
      "Complement found from 13743 to 13749\n",
      "Complement found from 13772 to 13778\n",
      "Complement found from 15361 to 15367\n",
      "Complement found from 15677 to 15683\n",
      "Complement found from 16076 to 16082\n",
      "Seq is: AACAAC\n",
      "19 AACAAC motifs detected in file\n",
      "\n",
      "Complement found from 471 to 477\n",
      "Complement found from 2879 to 2885\n",
      "Complement found from 5102 to 5108\n",
      "Complement found from 5119 to 5125\n",
      "Complement found from 5152 to 5158\n",
      "Complement found from 5377 to 5383\n",
      "Complement found from 6306 to 6312\n",
      "Complement found from 6526 to 6532\n",
      "Complement found from 6681 to 6687\n",
      "Complement found from 10102 to 10108\n",
      "Complement found from 11289 to 11295\n",
      "Complement found from 12348 to 12354\n",
      "Complement found from 12583 to 12589\n",
      "Complement found from 12971 to 12977\n",
      "Complement found from 13418 to 13424\n",
      "Complement found from 14209 to 14215\n",
      "Complement found from 14906 to 14912\n",
      "Seq is: ACTACT\n",
      "17 ACTACT motifs detected in file\n",
      "\n",
      "Complement found from 848 to 854\n",
      "Complement found from 1520 to 1526\n",
      "Complement found from 1652 to 1658\n",
      "Complement found from 1860 to 1866\n",
      "Complement found from 2087 to 2093\n",
      "Complement found from 5081 to 5087\n",
      "Complement found from 10014 to 10020\n",
      "Complement found from 10025 to 10031\n",
      "Complement found from 15970 to 15976\n",
      "Seq is: TTAACT\n",
      "9 TTAACT motifs detected in file\n",
      "\n",
      "Complement found from 8148 to 8154\n",
      "Complement found from 12121 to 12127\n",
      "Seq is: ACCGGG\n",
      "2 ACCGGG motifs detected in file\n",
      "\n",
      "Complement found from 13915 to 13921\n",
      "Complement found from 15275 to 15281\n",
      "Complement found from 15999 to 16005\n",
      "Seq is: GATTCT\n",
      "3 GATTCT motifs detected in file\n",
      "\n",
      "Complement found from 3864 to 3870\n",
      "Complement found from 4236 to 4242\n",
      "Complement found from 5483 to 5489\n",
      "Complement found from 6662 to 6668\n",
      "Complement found from 10709 to 10715\n",
      "Complement found from 11056 to 11062\n",
      "Complement found from 11068 to 11074\n",
      "Complement found from 14046 to 14052\n",
      "Complement found from 14824 to 14830\n",
      "Complement found from 15850 to 15856\n",
      "Seq is: ATCTCC\n",
      "10 ATCTCC motifs detected in file\n",
      "\n",
      "Complement found from 8180 to 8186\n",
      "Complement found from 11896 to 11902\n",
      "Seq is: TCTGTG\n",
      "2 TCTGTG motifs detected in file\n",
      "\n",
      "Seq is: GACGAT\n",
      "0 GACGAT motifs detected in file\n",
      "\n",
      "Complement found from 3496 to 3502\n",
      "Complement found from 5023 to 5029\n",
      "Complement found from 6055 to 6061\n",
      "Complement found from 7245 to 7251\n",
      "Complement found from 8081 to 8087\n",
      "Complement found from 11634 to 11640\n",
      "Complement found from 12021 to 12027\n",
      "Complement found from 13514 to 13520\n",
      "Complement found from 14946 to 14952\n",
      "Complement found from 15543 to 15549\n",
      "Complement found from 16172 to 16178\n",
      "Seq is: CCACAT\n",
      "11 CCACAT motifs detected in file\n",
      "\n",
      "Complement found from 3290 to 3296\n",
      "Complement found from 3652 to 3658\n",
      "Complement found from 4689 to 4695\n",
      "Complement found from 5211 to 5217\n",
      "Complement found from 6368 to 6374\n",
      "Complement found from 11786 to 11792\n",
      "Complement found from 12130 to 12136\n",
      "Complement found from 14098 to 14104\n",
      "Seq is: TCCTCT\n",
      "8 TCCTCT motifs detected in file\n",
      "\n",
      "Complement found from 1370 to 1376\n",
      "Complement found from 3501 to 3507\n",
      "Complement found from 4499 to 4505\n",
      "Complement found from 5047 to 5053\n",
      "Complement found from 5343 to 5349\n",
      "Complement found from 6285 to 6291\n",
      "Complement found from 8276 to 8282\n",
      "Complement found from 9184 to 9190\n",
      "Complement found from 11039 to 11045\n",
      "Complement found from 13786 to 13792\n",
      "Complement found from 13918 to 13924\n",
      "Seq is: TCTACC\n",
      "11 TCTACC motifs detected in file\n",
      "\n",
      "Complement found from 3083 to 3089\n",
      "Complement found from 4360 to 4366\n",
      "Complement found from 7135 to 7141\n",
      "Complement found from 9149 to 9155\n",
      "Complement found from 10155 to 10161\n",
      "Complement found from 12445 to 12451\n",
      "Complement found from 12954 to 12960\n",
      "Complement found from 13151 to 13157\n",
      "Complement found from 16169 to 16175\n",
      "Seq is: AATCCA\n",
      "9 AATCCA motifs detected in file\n",
      "\n",
      "Complement found from 7392 to 7398\n",
      "Complement found from 12822 to 12828\n",
      "Seq is: GATGCC\n",
      "2 GATGCC motifs detected in file\n",
      "\n",
      "Complement found from 2840 to 2846\n",
      "Seq is: TCCGAG\n",
      "1 TCCGAG motifs detected in file\n",
      "\n",
      "Complement found from 4811 to 4817\n",
      "Complement found from 9590 to 9596\n",
      "Complement found from 10774 to 10780\n",
      "Complement found from 15260 to 15266\n",
      "Seq is: GTCCCA\n",
      "4 GTCCCA motifs detected in file\n",
      "\n",
      "Complement found from 2412 to 2418\n",
      "Complement found from 4012 to 4018\n",
      "Complement found from 4250 to 4256\n",
      "Complement found from 4907 to 4913\n",
      "Complement found from 5549 to 5555\n",
      "Complement found from 7664 to 7670\n",
      "Complement found from 8473 to 8479\n",
      "Complement found from 10473 to 10479\n",
      "Complement found from 10611 to 10617\n",
      "Complement found from 10979 to 10985\n",
      "Complement found from 12040 to 12046\n",
      "Complement found from 12062 to 12068\n",
      "Complement found from 12101 to 12107\n",
      "Complement found from 13035 to 13041\n",
      "Complement found from 13450 to 13456\n",
      "Complement found from 15266 to 15272\n",
      "Complement found from 16220 to 16226\n",
      "Complement found from 16259 to 16265\n",
      "Complement found from 16377 to 16383\n",
      "Seq is: CCCTCA\n",
      "19 CCCTCA motifs detected in file\n",
      "\n",
      "Complement found from 1328 to 1334\n",
      "Complement found from 11793 to 11799\n",
      "Seq is: TCAAGG\n",
      "2 TCAAGG motifs detected in file\n",
      "\n",
      "Seq is: GACGAT\n",
      "0 GACGAT motifs detected in file\n",
      "\n",
      "Complement found from 1188 to 1194\n",
      "Complement found from 4906 to 4912\n",
      "Complement found from 6220 to 6226\n",
      "Complement found from 7860 to 7866\n",
      "Complement found from 8472 to 8478\n",
      "Complement found from 9083 to 9089\n",
      "Complement found from 10328 to 10334\n",
      "Complement found from 10625 to 10631\n",
      "Complement found from 10868 to 10874\n",
      "Complement found from 11977 to 11983\n",
      "Complement found from 12100 to 12106\n",
      "Complement found from 13449 to 13455\n",
      "Seq is: TCCCTC\n",
      "12 TCCCTC motifs detected in file\n",
      "\n",
      "Complement found from 1263 to 1269\n",
      "Complement found from 4503 to 4509\n",
      "Complement found from 4881 to 4887\n",
      "Complement found from 6339 to 6345\n",
      "Complement found from 9969 to 9975\n",
      "Complement found from 10506 to 10512\n",
      "Complement found from 11579 to 11585\n",
      "Seq is: CCATCT\n",
      "7 CCATCT motifs detected in file\n",
      "\n",
      "Complement found from 1553 to 1559\n",
      "Complement found from 3131 to 3137\n",
      "Complement found from 13589 to 13595\n",
      "Complement found from 15797 to 15803\n",
      "Seq is: GACAAG\n",
      "4 GACAAG motifs detected in file\n",
      "\n",
      "Complement found from 1591 to 1597\n",
      "Complement found from 2117 to 2123\n",
      "Complement found from 2265 to 2271\n",
      "Complement found from 15794 to 15800\n",
      "Seq is: TTGGAC\n",
      "4 TTGGAC motifs detected in file\n",
      "\n",
      "Complement found from 1166 to 1172\n",
      "Complement found from 1512 to 1518\n",
      "Complement found from 1827 to 1833\n",
      "Complement found from 3128 to 3134\n",
      "Complement found from 5587 to 5593\n",
      "Complement found from 8716 to 8722\n",
      "Complement found from 11795 to 11801\n",
      "Complement found from 15946 to 15952\n",
      "Seq is: AAGGAC\n",
      "8 AAGGAC motifs detected in file\n",
      "\n",
      "Complement found from 5223 to 5229\n",
      "Complement found from 5834 to 5840\n",
      "Complement found from 6257 to 6263\n",
      "Complement found from 15044 to 15050\n",
      "Complement found from 15149 to 15155\n",
      "Seq is: GAGGCC\n",
      "5 GAGGCC motifs detected in file\n",
      "\n",
      "Seq is: GGAGGT\n",
      "0 GGAGGT motifs detected in file\n",
      "\n",
      "Complement found from 3178 to 3184\n",
      "Complement found from 6733 to 6739\n",
      "Complement found from 12870 to 12876\n",
      "Seq is: GATATC\n",
      "3 GATATC motifs detected in file\n",
      "\n",
      "Complement found from 6284 to 6290\n",
      "Complement found from 7607 to 7613\n",
      "Seq is: GTCTAC\n",
      "2 GTCTAC motifs detected in file\n",
      "\n",
      "Seq is: GCTCGT\n",
      "0 GCTCGT motifs detected in file\n",
      "\n",
      "Complement found from 143 to 149\n",
      "Complement found from 3329 to 3335\n",
      "Complement found from 5293 to 5299\n",
      "Complement found from 7665 to 7671\n",
      "Complement found from 7806 to 7812\n",
      "Complement found from 8427 to 8433\n",
      "Complement found from 9331 to 9337\n",
      "Complement found from 10474 to 10480\n",
      "Complement found from 10543 to 10549\n",
      "Complement found from 11726 to 11732\n",
      "Complement found from 12041 to 12047\n",
      "Complement found from 12063 to 12069\n",
      "Complement found from 15639 to 15645\n",
      "Complement found from 15741 to 15747\n",
      "Seq is: CCTCAT\n",
      "14 CCTCAT motifs detected in file\n",
      "\n",
      "Complement found from 3788 to 3794\n",
      "Complement found from 5212 to 5218\n",
      "Complement found from 6222 to 6228\n",
      "Complement found from 9268 to 9274\n",
      "Complement found from 11043 to 11049\n",
      "Complement found from 11787 to 11793\n",
      "Complement found from 13434 to 13440\n",
      "Complement found from 14099 to 14105\n",
      "Complement found from 14282 to 14288\n",
      "Seq is: CCTCTC\n",
      "9 CCTCTC motifs detected in file\n",
      "\n",
      "Complement found from 1188 to 1194\n",
      "Complement found from 4906 to 4912\n",
      "Complement found from 6220 to 6226\n",
      "Complement found from 7860 to 7866\n",
      "Complement found from 8472 to 8478\n",
      "Complement found from 9083 to 9089\n",
      "Complement found from 10328 to 10334\n",
      "Complement found from 10625 to 10631\n",
      "Complement found from 10868 to 10874\n",
      "Complement found from 11977 to 11983\n",
      "Complement found from 12100 to 12106\n",
      "Complement found from 13449 to 13455\n",
      "Seq is: TCCCTC\n",
      "12 TCCCTC motifs detected in file\n",
      "\n",
      "Complement found from 2321 to 2327\n",
      "Complement found from 3341 to 3347\n",
      "Complement found from 3749 to 3755\n",
      "Complement found from 5071 to 5077\n",
      "Complement found from 6586 to 6592\n",
      "Complement found from 8594 to 8600\n",
      "Complement found from 11283 to 11289\n",
      "Complement found from 11502 to 11508\n",
      "Complement found from 11658 to 11664\n",
      "Complement found from 11697 to 11703\n",
      "Complement found from 12087 to 12093\n",
      "Complement found from 15161 to 15167\n",
      "Complement found from 15744 to 15750\n",
      "Seq is: CATTCT\n",
      "13 CATTCT motifs detected in file\n",
      "\n",
      "Complement found from 6282 to 6288\n",
      "Complement found from 12473 to 12479\n",
      "Complement found from 13060 to 13066\n",
      "Complement found from 13201 to 13207\n",
      "Complement found from 15912 to 15918\n",
      "Seq is: CAGTCT\n",
      "5 CAGTCT motifs detected in file\n",
      "\n",
      "Complement found from 3943 to 3949\n",
      "Complement found from 5788 to 5794\n",
      "Complement found from 13607 to 13613\n",
      "Seq is: TCGAAT\n",
      "3 TCGAAT motifs detected in file\n",
      "\n",
      "Complement found from 3301 to 3307\n",
      "Complement found from 3804 to 3810\n",
      "Complement found from 4038 to 4044\n",
      "Complement found from 4071 to 4077\n",
      "Complement found from 6195 to 6201\n",
      "Complement found from 6398 to 6404\n",
      "Complement found from 7183 to 7189\n",
      "Complement found from 7723 to 7729\n",
      "Complement found from 8707 to 8713\n",
      "Complement found from 9196 to 9202\n",
      "Complement found from 10038 to 10044\n",
      "Complement found from 10895 to 10901\n",
      "Complement found from 11261 to 11267\n",
      "Complement found from 11995 to 12001\n",
      "Complement found from 12029 to 12035\n",
      "Complement found from 12176 to 12182\n",
      "Complement found from 12248 to 12254\n",
      "Complement found from 12486 to 12492\n",
      "Complement found from 12929 to 12935\n",
      "Complement found from 13744 to 13750\n",
      "Complement found from 13773 to 13779\n",
      "Complement found from 15678 to 15684\n",
      "Complement found from 15823 to 15829\n",
      "Seq is: ACAACA\n",
      "23 ACAACA motifs detected in file\n",
      "\n",
      "Complement found from 3946 to 3952\n",
      "Complement found from 11487 to 11493\n",
      "Complement found from 14754 to 14760\n",
      "Seq is: AATACG\n",
      "3 AATACG motifs detected in file\n",
      "\n",
      "Complement found from 2075 to 2081\n",
      "Complement found from 3893 to 3899\n",
      "Complement found from 3959 to 3965\n",
      "Complement found from 4318 to 4324\n",
      "Complement found from 4795 to 4801\n",
      "Complement found from 5487 to 5493\n",
      "Complement found from 8843 to 8849\n",
      "Complement found from 8931 to 8937\n",
      "Complement found from 10161 to 10167\n",
      "Complement found from 11535 to 11541\n",
      "Complement found from 12193 to 12199\n",
      "Complement found from 13763 to 13769\n",
      "Complement found from 15525 to 15531\n",
      "Complement found from 16545 to 16551\n",
      "Seq is: CCCCTT\n",
      "14 CCCCTT motifs detected in file\n",
      "\n",
      "Seq is: TGCGGT\n",
      "0 TGCGGT motifs detected in file\n",
      "\n",
      "Complement found from 1389 to 1395\n",
      "Complement found from 3978 to 3984\n",
      "Complement found from 4790 to 4796\n",
      "Complement found from 5289 to 5295\n",
      "Complement found from 5312 to 5318\n",
      "Complement found from 8985 to 8991\n",
      "Complement found from 11527 to 11533\n",
      "Complement found from 11638 to 11644\n",
      "Complement found from 12936 to 12942\n",
      "Complement found from 13141 to 13147\n",
      "Complement found from 14449 to 14455\n",
      "Complement found from 16531 to 16537\n",
      "Seq is: ATAGCC\n",
      "12 ATAGCC motifs detected in file\n",
      "\n",
      "Complement found from 2686 to 2692\n",
      "Complement found from 15144 to 15150\n",
      "Seq is: CCCGTG\n",
      "2 CCCGTG motifs detected in file\n",
      "\n",
      "Complement found from 12951 to 12957\n",
      "Seq is: GCTAAT\n",
      "1 GCTAAT motifs detected in file\n",
      "\n",
      "Complement found from 2072 to 2078\n",
      "Complement found from 2230 to 2236\n",
      "Complement found from 4472 to 4478\n",
      "Complement found from 9578 to 9584\n",
      "Complement found from 13778 to 13784\n",
      "Complement found from 13876 to 13882\n",
      "Complement found from 13943 to 13949\n",
      "Complement found from 15654 to 15660\n",
      "Complement found from 16349 to 16355\n",
      "Seq is: AATCCC\n",
      "9 AATCCC motifs detected in file\n",
      "\n",
      "Seq is: CTTGAT\n",
      "0 CTTGAT motifs detected in file\n",
      "\n",
      "Complement found from 426 to 432\n",
      "Complement found from 2867 to 2873\n",
      "Complement found from 3274 to 3280\n",
      "Complement found from 11693 to 11699\n",
      "Complement found from 16343 to 16349\n",
      "Seq is: CAGTCA\n",
      "5 CAGTCA motifs detected in file\n",
      "\n",
      "Seq is: ACGGTT\n",
      "0 ACGGTT motifs detected in file\n",
      "\n",
      "Complement found from 2822 to 2828\n",
      "Complement found from 5818 to 5824\n",
      "Complement found from 6126 to 6132\n",
      "Complement found from 15758 to 15764\n",
      "Seq is: TCGGAG\n",
      "4 TCGGAG motifs detected in file\n",
      "\n",
      "Complement found from 12951 to 12957\n",
      "Seq is: GCTAAT\n",
      "1 GCTAAT motifs detected in file\n",
      "\n",
      "Complement found from 1701 to 1707\n",
      "Complement found from 2219 to 2225\n",
      "Complement found from 2783 to 2789\n",
      "Complement found from 4733 to 4739\n",
      "Complement found from 5105 to 5111\n",
      "Complement found from 7227 to 7233\n",
      "Complement found from 8380 to 8386\n",
      "Complement found from 8459 to 8465\n",
      "Complement found from 10126 to 10132\n",
      "Complement found from 10791 to 10797\n",
      "Complement found from 10963 to 10969\n",
      "Complement found from 13849 to 13855\n",
      "Seq is: ACTACC\n",
      "12 ACTACC motifs detected in file\n",
      "\n",
      "Complement found from 230 to 236\n",
      "Complement found from 1816 to 1822\n",
      "Complement found from 1851 to 1857\n",
      "Complement found from 2596 to 2602\n",
      "Complement found from 3851 to 3857\n",
      "Complement found from 4669 to 4675\n",
      "Complement found from 4762 to 4768\n",
      "Complement found from 6121 to 6127\n",
      "Complement found from 6829 to 6835\n",
      "Complement found from 7668 to 7674\n",
      "Complement found from 8397 to 8403\n",
      "Complement found from 9202 to 9208\n",
      "Complement found from 10821 to 10827\n",
      "Complement found from 11562 to 11568\n",
      "Complement found from 11703 to 11709\n",
      "Complement found from 11781 to 11787\n",
      "Complement found from 12593 to 12599\n",
      "Complement found from 13271 to 13277\n",
      "Complement found from 14081 to 14087\n",
      "Complement found from 14228 to 14234\n",
      "Complement found from 15687 to 15693\n",
      "Seq is: CATAAT\n",
      "21 CATAAT motifs detected in file\n",
      "\n",
      "Complement found from 9924 to 9930\n",
      "Complement found from 14438 to 14444\n",
      "Seq is: GATACT\n",
      "2 GATACT motifs detected in file\n",
      "\n",
      "Complement found from 3293 to 3299\n",
      "Complement found from 3551 to 3557\n",
      "Complement found from 4855 to 4861\n",
      "Complement found from 6099 to 6105\n",
      "Complement found from 6342 to 6348\n",
      "Complement found from 10221 to 10227\n",
      "Complement found from 11117 to 11123\n",
      "Complement found from 13105 to 13111\n",
      "Complement found from 13616 to 13622\n",
      "Complement found from 14107 to 14113\n",
      "Seq is: TCTTCT\n",
      "10 TCTTCT motifs detected in file\n",
      "\n",
      "Complement found from 233 to 239\n",
      "Complement found from 4300 to 4306\n",
      "Complement found from 4756 to 4762\n",
      "Complement found from 5035 to 5041\n",
      "Complement found from 5500 to 5506\n",
      "Complement found from 6094 to 6100\n",
      "Complement found from 6157 to 6163\n",
      "Complement found from 6658 to 6664\n",
      "Complement found from 7303 to 7309\n",
      "Complement found from 8040 to 8046\n",
      "Complement found from 9679 to 9685\n",
      "Complement found from 10078 to 10084\n",
      "Complement found from 10108 to 10114\n",
      "Complement found from 10573 to 10579\n",
      "Complement found from 13610 to 13616\n",
      "Complement found from 14544 to 14550\n",
      "Complement found from 15651 to 15657\n",
      "Seq is: AATAAT\n",
      "17 AATAAT motifs detected in file\n",
      "\n",
      "Complement found from 2495 to 2501\n",
      "Complement found from 2976 to 2982\n",
      "Complement found from 3642 to 3648\n",
      "Complement found from 14322 to 14328\n",
      "Seq is: GTTTAC\n",
      "4 GTTTAC motifs detected in file\n",
      "\n",
      "Complement found from 1306 to 1312\n",
      "Complement found from 2573 to 2579\n",
      "Complement found from 3336 to 3342\n",
      "Complement found from 13324 to 13330\n",
      "Seq is: GTACCC\n",
      "4 GTACCC motifs detected in file\n",
      "\n",
      "Complement found from 2120 to 2126\n",
      "Complement found from 12532 to 12538\n",
      "Seq is: GACACT\n",
      "2 GACACT motifs detected in file\n",
      "\n",
      "Complement found from 3021 to 3027\n",
      "Complement found from 10232 to 10238\n",
      "Complement found from 14015 to 14021\n",
      "Seq is: GCTATT\n",
      "3 GCTATT motifs detected in file\n",
      "\n",
      "Complement found from 15118 to 15124\n",
      "Complement found from 15322 to 15328\n",
      "Seq is: GCAACA\n",
      "2 GCAACA motifs detected in file\n",
      "\n",
      "Complement found from 2384 to 2390\n",
      "Complement found from 3514 to 3520\n",
      "Complement found from 4066 to 4072\n",
      "Complement found from 5927 to 5933\n",
      "Complement found from 6060 to 6066\n",
      "Complement found from 7608 to 7614\n",
      "Complement found from 9087 to 9093\n",
      "Complement found from 11210 to 11216\n",
      "Complement found from 11981 to 11987\n",
      "Seq is: TCTACA\n",
      "9 TCTACA motifs detected in file\n",
      "\n",
      "Complement found from 509 to 515\n",
      "Complement found from 4240 to 4246\n",
      "Complement found from 5136 to 5142\n",
      "Seq is: CCAGCA\n",
      "3 CCAGCA motifs detected in file\n",
      "\n",
      "Complement found from 7389 to 7395\n",
      "Complement found from 16366 to 16372\n",
      "Seq is: ATGGAT\n",
      "2 ATGGAT motifs detected in file\n",
      "\n",
      "Complement found from 7030 to 7036\n",
      "Complement found from 9313 to 9319\n",
      "Complement found from 9811 to 9817\n",
      "Complement found from 10018 to 10024\n",
      "Complement found from 13121 to 13127\n",
      "Complement found from 13766 to 13772\n",
      "Complement found from 15402 to 15408\n",
      "Seq is: CTTCCA\n",
      "7 CTTCCA motifs detected in file\n",
      "\n",
      "Complement found from 9816 to 9822\n",
      "Complement found from 11387 to 11393\n",
      "Complement found from 14682 to 14688\n",
      "Seq is: ACGGAC\n",
      "3 ACGGAC motifs detected in file\n",
      "\n",
      "Complement found from 5911 to 5917\n",
      "Complement found from 8146 to 8152\n",
      "Seq is: CGACCG\n",
      "2 CGACCG motifs detected in file\n",
      "\n",
      "Complement found from 1410 to 1416\n",
      "Seq is: GGTCGA\n",
      "1 GGTCGA motifs detected in file\n",
      "\n",
      "Complement found from 6017 to 6023\n",
      "Seq is: GCCGAG\n",
      "1 GCCGAG motifs detected in file\n",
      "\n",
      "Complement found from 4321 to 4327\n",
      "Complement found from 6008 to 6014\n",
      "Complement found from 9695 to 9701\n",
      "Complement found from 10243 to 10249\n",
      "Complement found from 10333 to 10339\n",
      "Complement found from 12196 to 12202\n",
      "Seq is: CTTATT\n",
      "6 CTTATT motifs detected in file\n",
      "\n",
      "Complement found from 2327 to 2333\n",
      "Complement found from 2838 to 2844\n",
      "Complement found from 6324 to 6330\n",
      "Complement found from 6819 to 6825\n",
      "Complement found from 9288 to 9294\n",
      "Complement found from 10928 to 10934\n",
      "Complement found from 16406 to 16412\n",
      "Seq is: CCTCCG\n",
      "7 CCTCCG motifs detected in file\n",
      "\n",
      "Complement found from 477 to 483\n",
      "Complement found from 3722 to 3728\n",
      "Complement found from 4235 to 4241\n",
      "Complement found from 4900 to 4906\n",
      "Complement found from 6661 to 6667\n",
      "Complement found from 10708 to 10714\n",
      "Complement found from 11055 to 11061\n",
      "Complement found from 11067 to 11073\n",
      "Complement found from 14045 to 14051\n",
      "Complement found from 14163 to 14169\n",
      "Seq is: AATCTC\n",
      "10 AATCTC motifs detected in file\n",
      "\n",
      "Complement found from 7007 to 7013\n",
      "Seq is: GACACG\n",
      "1 GACACG motifs detected in file\n",
      "\n",
      "Complement found from 14258 to 14264\n",
      "Complement found from 14868 to 14874\n",
      "Seq is: GATCCT\n",
      "2 GATCCT motifs detected in file\n",
      "\n",
      "Complement found from 7849 to 7855\n",
      "Seq is: GAGGTC\n",
      "1 GAGGTC motifs detected in file\n",
      "\n",
      "Complement found from 7909 to 7915\n",
      "Complement found from 9743 to 9749\n",
      "Seq is: GAGTAC\n",
      "2 GAGTAC motifs detected in file\n",
      "\n",
      "Complement found from 10173 to 10179\n",
      "Complement found from 11474 to 11480\n",
      "Seq is: GCGGCT\n",
      "2 GCGGCT motifs detected in file\n",
      "\n",
      "Complement found from 151 to 157\n",
      "Complement found from 3196 to 3202\n",
      "Complement found from 3998 to 4004\n",
      "Complement found from 5092 to 5098\n",
      "Complement found from 5961 to 5967\n",
      "Complement found from 8951 to 8957\n",
      "Complement found from 10245 to 10251\n",
      "Complement found from 10489 to 10495\n",
      "Complement found from 12467 to 12473\n",
      "Complement found from 12518 to 12524\n",
      "Seq is: TATTAT\n",
      "10 TATTAT motifs detected in file\n",
      "\n",
      "Complement found from 2960 to 2966\n",
      "Complement found from 5385 to 5391\n",
      "Complement found from 6667 to 6673\n",
      "Complement found from 6772 to 6778\n",
      "Complement found from 9363 to 9369\n",
      "Complement found from 14519 to 14525\n",
      "Complement found from 15666 to 15672\n",
      "Seq is: CCATAT\n",
      "7 CCATAT motifs detected in file\n",
      "\n",
      "Complement found from 3808 to 3814\n",
      "Complement found from 8051 to 8057\n",
      "Complement found from 8914 to 8920\n",
      "Complement found from 12215 to 12221\n",
      "Complement found from 16428 to 16434\n",
      "Seq is: CACAAG\n",
      "5 CACAAG motifs detected in file\n",
      "\n",
      "Complement found from 12621 to 12627\n",
      "Complement found from 13278 to 13284\n",
      "Seq is: GTTACA\n",
      "2 GTTACA motifs detected in file\n",
      "\n",
      "Complement found from 810 to 816\n",
      "Complement found from 5169 to 5175\n",
      "Complement found from 7251 to 7257\n",
      "Complement found from 14654 to 14660\n",
      "Complement found from 15083 to 15089\n",
      "Seq is: GAAACA\n",
      "5 GAAACA motifs detected in file\n",
      "\n",
      "Complement found from 297 to 303\n",
      "Complement found from 347 to 353\n",
      "Complement found from 551 to 557\n",
      "Complement found from 1064 to 1070\n",
      "Complement found from 1683 to 1689\n",
      "Complement found from 1719 to 1725\n",
      "Complement found from 2234 to 2240\n",
      "Complement found from 2787 to 2793\n",
      "Complement found from 3716 to 3722\n",
      "Complement found from 4981 to 4987\n",
      "Complement found from 6439 to 6445\n",
      "Complement found from 7120 to 7126\n",
      "Complement found from 8124 to 8130\n",
      "Complement found from 9891 to 9897\n",
      "Complement found from 11667 to 11673\n",
      "Complement found from 12548 to 12554\n",
      "Complement found from 12669 to 12675\n",
      "Complement found from 13154 to 13160\n",
      "Complement found from 13769 to 13775\n",
      "Complement found from 15674 to 15680\n",
      "Seq is: CCAAAC\n",
      "20 CCAAAC motifs detected in file\n",
      "\n",
      "Complement found from 340 to 346\n",
      "Complement found from 4901 to 4907\n",
      "Complement found from 6493 to 6499\n",
      "Complement found from 8729 to 8735\n",
      "Seq is: ATCTCT\n",
      "4 ATCTCT motifs detected in file\n",
      "\n",
      "Complement found from 12470 to 12476\n",
      "Seq is: TATCAG\n",
      "1 TATCAG motifs detected in file\n",
      "\n",
      "Complement found from 7062 to 7068\n",
      "Complement found from 8761 to 8767\n",
      "Complement found from 10650 to 10656\n",
      "Seq is: TTGCCA\n",
      "3 TTGCCA motifs detected in file\n",
      "\n",
      "Complement found from 3018 to 3024\n",
      "Seq is: GCCGCT\n",
      "1 GCCGCT motifs detected in file\n",
      "\n",
      "Complement found from 14460 to 14466\n",
      "Complement found from 15892 to 15898\n",
      "Complement found from 16151 to 16157\n",
      "Seq is: TGTAGT\n",
      "3 TGTAGT motifs detected in file\n",
      "\n",
      "Complement found from 430 to 436\n",
      "Complement found from 565 to 571\n",
      "Complement found from 634 to 640\n",
      "Complement found from 799 to 805\n",
      "Complement found from 953 to 959\n",
      "Complement found from 4388 to 4394\n",
      "Complement found from 4831 to 4837\n",
      "Complement found from 5434 to 5440\n",
      "Complement found from 5892 to 5898\n",
      "Complement found from 8269 to 8275\n",
      "Complement found from 8929 to 8935\n",
      "Complement found from 9568 to 9574\n",
      "Complement found from 10159 to 10165\n",
      "Complement found from 12965 to 12971\n",
      "Complement found from 13023 to 13029\n",
      "Complement found from 13055 to 13061\n",
      "Complement found from 13125 to 13131\n",
      "Complement found from 14337 to 14343\n",
      "Complement found from 14811 to 14817\n",
      "Complement found from 15534 to 15540\n",
      "Complement found from 16256 to 16262\n",
      "Seq is: CACCCC\n",
      "21 CACCCC motifs detected in file\n",
      "\n",
      "Complement found from 462 to 468\n",
      "Complement found from 1690 to 1696\n",
      "Complement found from 6879 to 6885\n",
      "Complement found from 9317 to 9323\n",
      "Complement found from 9512 to 9518\n",
      "Complement found from 9594 to 9600\n",
      "Complement found from 10622 to 10628\n",
      "Complement found from 12909 to 12915\n",
      "Complement found from 14849 to 14855\n",
      "Complement found from 15326 to 15332\n",
      "Seq is: CACTCC\n",
      "10 CACTCC motifs detected in file\n",
      "\n",
      "Complement found from 396 to 402\n",
      "Complement found from 5883 to 5889\n",
      "Complement found from 9704 to 9710\n",
      "Complement found from 9720 to 9726\n",
      "Complement found from 10073 to 10079\n",
      "Complement found from 11109 to 11115\n",
      "Complement found from 13892 to 13898\n",
      "Seq is: ATTTTA\n",
      "7 ATTTTA motifs detected in file\n",
      "\n",
      "Complement found from 7916 to 7922\n",
      "Complement found from 8645 to 8651\n",
      "Seq is: CCGACT\n",
      "2 CCGACT motifs detected in file\n",
      "\n",
      "Complement found from 4802 to 4808\n",
      "Complement found from 7028 to 7034\n",
      "Complement found from 8904 to 8910\n",
      "Complement found from 9169 to 9175\n",
      "Complement found from 9311 to 9317\n",
      "Complement found from 10512 to 10518\n",
      "Complement found from 13250 to 13256\n",
      "Complement found from 13439 to 13445\n",
      "Complement found from 15289 to 15295\n",
      "Seq is: CACTTC\n",
      "9 CACTTC motifs detected in file\n",
      "\n",
      "Complement found from 11905 to 11911\n",
      "Seq is: GTAACC\n",
      "1 GTAACC motifs detected in file\n",
      "\n",
      "Complement found from 4839 to 4845\n",
      "Complement found from 10799 to 10805\n",
      "Complement found from 13223 to 13229\n",
      "Complement found from 15464 to 15470\n",
      "Seq is: TGACAT\n",
      "4 TGACAT motifs detected in file\n",
      "\n",
      "Complement found from 947 to 953\n",
      "Complement found from 1075 to 1081\n",
      "Complement found from 5852 to 5858\n",
      "Seq is: TTAGAT\n",
      "3 TTAGAT motifs detected in file\n",
      "\n",
      "Complement found from 1206 to 1212\n",
      "Complement found from 5574 to 5580\n",
      "Complement found from 9957 to 9963\n",
      "Seq is: TTCTGT\n",
      "3 TTCTGT motifs detected in file\n",
      "\n",
      "Complement found from 352 to 358\n",
      "Complement found from 434 to 440\n",
      "Complement found from 556 to 562\n",
      "Complement found from 961 to 967\n",
      "Complement found from 1681 to 1687\n",
      "Complement found from 3413 to 3419\n",
      "Complement found from 7467 to 7473\n",
      "Complement found from 8368 to 8374\n",
      "Complement found from 9533 to 9539\n",
      "Complement found from 9555 to 9561\n",
      "Complement found from 10916 to 10922\n",
      "Complement found from 12301 to 12307\n",
      "Complement found from 14531 to 14537\n",
      "Complement found from 14750 to 14756\n",
      "Seq is: CCCCAA\n",
      "14 CCCCAA motifs detected in file\n",
      "\n",
      "Complement found from 4709 to 4715\n",
      "Complement found from 6686 to 6692\n",
      "Complement found from 7202 to 7208\n",
      "Seq is: TCCGGA\n",
      "3 TCCGGA motifs detected in file\n",
      "\n",
      "Complement found from 1799 to 1805\n",
      "Complement found from 2128 to 2134\n",
      "Complement found from 3379 to 3385\n",
      "Complement found from 4174 to 4180\n",
      "Complement found from 5745 to 5751\n",
      "Complement found from 6690 to 6696\n",
      "Complement found from 7526 to 7532\n",
      "Complement found from 10151 to 10157\n",
      "Complement found from 11030 to 11036\n",
      "Complement found from 13405 to 13411\n",
      "Complement found from 14709 to 14715\n",
      "Complement found from 15960 to 15966\n",
      "Seq is: GAAAAA\n",
      "12 GAAAAA motifs detected in file\n",
      "\n",
      "Seq is: TGGTCG\n",
      "0 TGGTCG motifs detected in file\n",
      "\n",
      "Complement found from 7270 to 7276\n",
      "Complement found from 11223 to 11229\n",
      "Complement found from 11271 to 11277\n",
      "Seq is: AGGCTC\n",
      "3 AGGCTC motifs detected in file\n",
      "\n",
      "Complement found from 4911 to 4917\n",
      "Complement found from 8711 to 8717\n",
      "Complement found from 9357 to 9363\n",
      "Complement found from 10311 to 10317\n",
      "Complement found from 11246 to 11252\n",
      "Complement found from 11276 to 11282\n",
      "Complement found from 11816 to 11822\n",
      "Complement found from 13147 to 13153\n",
      "Complement found from 14391 to 14397\n",
      "Complement found from 15701 to 15707\n",
      "Seq is: CACTAA\n",
      "10 CACTAA motifs detected in file\n",
      "\n",
      "Complement found from 1500 to 1506\n",
      "Complement found from 1739 to 1745\n",
      "Complement found from 4640 to 4646\n",
      "Complement found from 6858 to 6864\n",
      "Complement found from 16057 to 16063\n",
      "Seq is: AAGTAT\n",
      "5 AAGTAT motifs detected in file\n",
      "\n",
      "Complement found from 1402 to 1408\n",
      "Complement found from 1628 to 1634\n",
      "Complement found from 1649 to 1655\n",
      "Complement found from 3189 to 3195\n",
      "Complement found from 3260 to 3266\n",
      "Complement found from 5126 to 5132\n",
      "Complement found from 6676 to 6682\n",
      "Complement found from 8306 to 8312\n",
      "Complement found from 11335 to 11341\n",
      "Complement found from 13864 to 13870\n",
      "Complement found from 15187 to 15193\n",
      "Seq is: AACTTA\n",
      "11 AACTTA motifs detected in file\n",
      "\n",
      "Complement found from 9160 to 9166\n",
      "Seq is: ACGTTT\n",
      "1 ACGTTT motifs detected in file\n",
      "\n",
      "Complement found from 6202 to 6208\n",
      "Complement found from 11679 to 11685\n",
      "Complement found from 12569 to 12575\n",
      "Seq is: AAGCTT\n",
      "3 AAGCTT motifs detected in file\n",
      "\n",
      "Complement found from 6451 to 6457\n",
      "Seq is: CTTCGT\n",
      "1 CTTCGT motifs detected in file\n",
      "\n",
      "Seq is: CGAAGT\n",
      "0 CGAAGT motifs detected in file\n",
      "\n",
      "Complement found from 1591 to 1597\n",
      "Complement found from 2117 to 2123\n",
      "Complement found from 2265 to 2271\n",
      "Complement found from 15794 to 15800\n",
      "Seq is: TTGGAC\n",
      "4 TTGGAC motifs detected in file\n",
      "\n",
      "Complement found from 3411 to 3417\n",
      "Complement found from 3426 to 3432\n",
      "Complement found from 3957 to 3963\n",
      "Complement found from 9552 to 9558\n",
      "Complement found from 12299 to 12305\n",
      "Complement found from 13050 to 13056\n",
      "Seq is: GGCCCC\n",
      "6 GGCCCC motifs detected in file\n",
      "\n",
      "Complement found from 2691 to 2697\n",
      "Complement found from 7636 to 7642\n",
      "Seq is: GAAGAG\n",
      "2 GAAGAG motifs detected in file\n",
      "\n",
      "Complement found from 7924 to 7930\n",
      "Seq is: GGCGGA\n",
      "1 GGCGGA motifs detected in file\n",
      "\n",
      "Complement found from 2458 to 2464\n",
      "Complement found from 5747 to 5753\n",
      "Complement found from 5829 to 5835\n",
      "Complement found from 6693 to 6699\n",
      "Complement found from 7447 to 7453\n",
      "Complement found from 7514 to 7520\n",
      "Complement found from 9432 to 9438\n",
      "Complement found from 10048 to 10054\n",
      "Complement found from 10380 to 10386\n",
      "Complement found from 14074 to 14080\n",
      "Complement found from 15961 to 15967\n",
      "Seq is: AAAAAG\n",
      "11 AAAAAG motifs detected in file\n",
      "\n",
      "Complement found from 2566 to 2572\n",
      "Seq is: GGCCGC\n",
      "1 GGCCGC motifs detected in file\n",
      "\n",
      "Complement found from 3686 to 3692\n",
      "Complement found from 3965 to 3971\n",
      "Complement found from 5458 to 5464\n",
      "Complement found from 7662 to 7668\n",
      "Complement found from 7812 to 7818\n",
      "Complement found from 13208 to 13214\n",
      "Complement found from 15429 to 15435\n",
      "Seq is: CGCCCT\n",
      "7 CGCCCT motifs detected in file\n",
      "\n",
      "Complement found from 1181 to 1187\n",
      "Complement found from 3974 to 3980\n",
      "Complement found from 6103 to 6109\n",
      "Complement found from 7078 to 7084\n",
      "Complement found from 8545 to 8551\n",
      "Complement found from 9859 to 9865\n",
      "Complement found from 14288 to 14294\n",
      "Complement found from 15126 to 15132\n",
      "Complement found from 15291 to 15297\n",
      "Complement found from 15303 to 15309\n",
      "Seq is: CTTCAT\n",
      "10 CTTCAT motifs detected in file\n",
      "\n",
      "Complement found from 5099 to 5105\n",
      "Complement found from 7169 to 7175\n",
      "Complement found from 7732 to 7738\n",
      "Complement found from 8304 to 8310\n",
      "Complement found from 10960 to 10966\n",
      "Complement found from 12226 to 12232\n",
      "Seq is: CTAACT\n",
      "6 CTAACT motifs detected in file\n",
      "\n",
      "Complement found from 6615 to 6621\n",
      "Seq is: TCGGTC\n",
      "1 TCGGTC motifs detected in file\n",
      "\n",
      "Complement found from 438 to 444\n",
      "Complement found from 850 to 856\n",
      "Complement found from 1433 to 1439\n",
      "Complement found from 1522 to 1528\n",
      "Complement found from 2292 to 2298\n",
      "Complement found from 7730 to 7736\n",
      "Complement found from 8372 to 8378\n",
      "Complement found from 8438 to 8444\n",
      "Complement found from 8677 to 8683\n",
      "Complement found from 8768 to 8774\n",
      "Complement found from 9870 to 9876\n",
      "Complement found from 10300 to 10306\n",
      "Complement found from 10766 to 10772\n",
      "Complement found from 11099 to 11105\n",
      "Complement found from 14763 to 14769\n",
      "Complement found from 15901 to 15907\n",
      "Seq is: AACTAA\n",
      "16 AACTAA motifs detected in file\n",
      "\n",
      "Complement found from 459 to 465\n",
      "Complement found from 6311 to 6317\n",
      "Complement found from 7179 to 7185\n",
      "Complement found from 9591 to 9597\n",
      "Complement found from 11813 to 11819\n",
      "Complement found from 14113 to 14119\n",
      "Complement found from 15261 to 15267\n",
      "Seq is: TCCCAC\n",
      "7 TCCCAC motifs detected in file\n",
      "\n",
      "Complement found from 4340 to 4346\n",
      "Complement found from 7457 to 7463\n",
      "Complement found from 15755 to 15761\n",
      "Seq is: GAATCG\n",
      "3 GAATCG motifs detected in file\n",
      "\n",
      "Complement found from 240 to 246\n",
      "Complement found from 2366 to 2372\n",
      "Complement found from 6401 to 6407\n",
      "Complement found from 9105 to 9111\n",
      "Complement found from 9701 to 9707\n",
      "Complement found from 10780 to 10786\n",
      "Complement found from 14028 to 14034\n",
      "Complement found from 15506 to 15512\n",
      "Complement found from 15580 to 15586\n",
      "Seq is: ACAATT\n",
      "9 ACAATT motifs detected in file\n",
      "\n",
      "Complement found from 6930 to 6936\n",
      "Seq is: GATTCA\n",
      "1 GATTCA motifs detected in file\n",
      "\n",
      "Complement found from 2235 to 2241\n",
      "Complement found from 2751 to 2757\n",
      "Complement found from 3717 to 3723\n",
      "Complement found from 3991 to 3997\n",
      "Complement found from 5680 to 5686\n",
      "Complement found from 9892 to 9898\n",
      "Complement found from 10295 to 10301\n",
      "Complement found from 11593 to 11599\n",
      "Complement found from 12549 to 12555\n",
      "Complement found from 12670 to 12676\n",
      "Complement found from 13531 to 13537\n",
      "Complement found from 13770 to 13776\n",
      "Complement found from 14189 to 14195\n",
      "Complement found from 15359 to 15365\n",
      "Complement found from 15675 to 15681\n",
      "Seq is: CAAACA\n",
      "15 CAAACA motifs detected in file\n",
      "\n",
      "Complement found from 353 to 359\n",
      "Complement found from 557 to 563\n",
      "Complement found from 1063 to 1069\n",
      "Complement found from 1682 to 1688\n",
      "Complement found from 1731 to 1737\n",
      "Complement found from 2233 to 2239\n",
      "Complement found from 3715 to 3721\n",
      "Complement found from 5252 to 5258\n",
      "Complement found from 7468 to 7474\n",
      "Complement found from 12302 to 12308\n",
      "Complement found from 12547 to 12553\n",
      "Complement found from 12668 to 12674\n",
      "Complement found from 14071 to 14077\n",
      "Complement found from 14532 to 14538\n",
      "Complement found from 15988 to 15994\n",
      "Seq is: CCCAAA\n",
      "15 CCCAAA motifs detected in file\n",
      "\n",
      "Complement found from 16511 to 16517\n",
      "Seq is: TTCAGG\n",
      "1 TTCAGG motifs detected in file\n",
      "\n",
      "Complement found from 11905 to 11911\n",
      "Seq is: GTAACC\n",
      "1 GTAACC motifs detected in file\n",
      "\n",
      "Complement found from 949 to 955\n",
      "Seq is: AGATCA\n",
      "1 AGATCA motifs detected in file\n",
      "\n",
      "Complement found from 4458 to 4464\n",
      "Complement found from 8108 to 8114\n",
      "Seq is: TTCCCG\n",
      "2 TTCCCG motifs detected in file\n",
      "\n",
      "Complement found from 1212 to 1218\n",
      "Complement found from 4341 to 4347\n",
      "Complement found from 7458 to 7464\n",
      "Complement found from 8002 to 8008\n",
      "Seq is: AATCGA\n",
      "4 AATCGA motifs detected in file\n",
      "\n",
      "Complement found from 210 to 216\n",
      "Complement found from 2744 to 2750\n",
      "Complement found from 4469 to 4475\n",
      "Complement found from 4753 to 4759\n",
      "Complement found from 7300 to 7306\n",
      "Complement found from 8220 to 8226\n",
      "Complement found from 10336 to 10342\n",
      "Complement found from 12675 to 12681\n",
      "Complement found from 14781 to 14787\n",
      "Seq is: ATTAAT\n",
      "9 ATTAAT motifs detected in file\n",
      "\n",
      "Complement found from 719 to 725\n",
      "Complement found from 2862 to 2868\n",
      "Complement found from 3471 to 3477\n",
      "Complement found from 6815 to 6821\n",
      "Complement found from 6944 to 6950\n",
      "Complement found from 8134 to 8140\n",
      "Complement found from 9761 to 9767\n",
      "Complement found from 11683 to 11689\n",
      "Complement found from 14353 to 14359\n",
      "Seq is: TTCACC\n",
      "9 TTCACC motifs detected in file\n",
      "\n",
      "Complement found from 1830 to 1836\n",
      "Complement found from 5184 to 5190\n",
      "Complement found from 7928 to 7934\n",
      "Complement found from 8647 to 8653\n",
      "Complement found from 8668 to 8674\n",
      "Seq is: GACTAA\n",
      "5 GACTAA motifs detected in file\n",
      "\n",
      "Complement found from 3404 to 3410\n",
      "Complement found from 4652 to 4658\n",
      "Complement found from 4993 to 4999\n",
      "Complement found from 5620 to 5626\n",
      "Complement found from 14757 to 14763\n",
      "Seq is: ACGCAA\n",
      "5 ACGCAA motifs detected in file\n",
      "\n",
      "Complement found from 2420 to 2426\n",
      "Complement found from 3591 to 3597\n",
      "Complement found from 7852 to 7858\n",
      "Complement found from 13258 to 13264\n",
      "Complement found from 13633 to 13639\n",
      "Seq is: GTCAAC\n",
      "5 GTCAAC motifs detected in file\n",
      "\n",
      "Seq is: TACGTC\n",
      "0 TACGTC motifs detected in file\n",
      "\n",
      "Complement found from 13 to 19\n",
      "Complement found from 429 to 435\n",
      "Complement found from 633 to 639\n",
      "Complement found from 720 to 726\n",
      "Complement found from 952 to 958\n",
      "Complement found from 1489 to 1495\n",
      "Complement found from 2278 to 2284\n",
      "Complement found from 3508 to 3514\n",
      "Complement found from 3736 to 3742\n",
      "Complement found from 4192 to 4198\n",
      "Complement found from 5322 to 5328\n",
      "Complement found from 5891 to 5897\n",
      "Complement found from 6619 to 6625\n",
      "Complement found from 8432 to 8438\n",
      "Complement found from 9567 to 9573\n",
      "Complement found from 11156 to 11162\n",
      "Complement found from 12014 to 12020\n",
      "Complement found from 12964 to 12970\n",
      "Complement found from 13621 to 13627\n",
      "Complement found from 14354 to 14360\n",
      "Complement found from 16067 to 16073\n",
      "Complement found from 16262 to 16268\n",
      "Seq is: TCACCC\n",
      "22 TCACCC motifs detected in file\n",
      "\n",
      "Complement found from 779 to 785\n",
      "Complement found from 1122 to 1128\n",
      "Complement found from 5427 to 5433\n",
      "Complement found from 5595 to 5601\n",
      "Complement found from 7727 to 7733\n",
      "Complement found from 8686 to 8692\n",
      "Complement found from 10420 to 10426\n",
      "Complement found from 11519 to 11525\n",
      "Complement found from 12153 to 12159\n",
      "Complement found from 13424 to 13430\n",
      "Complement found from 13971 to 13977\n",
      "Complement found from 14760 to 14766\n",
      "Complement found from 16178 to 16184\n",
      "Seq is: CAAAAC\n",
      "13 CAAAAC motifs detected in file\n",
      "\n",
      "Seq is: GTCAGG\n",
      "0 GTCAGG motifs detected in file\n",
      "\n",
      "Complement found from 1212 to 1218\n",
      "Complement found from 4341 to 4347\n",
      "Complement found from 7458 to 7464\n",
      "Complement found from 8002 to 8008\n",
      "Seq is: AATCGA\n",
      "4 AATCGA motifs detected in file\n",
      "\n",
      "Complement found from 14193 to 14199\n",
      "Seq is: CAATGT\n",
      "1 CAATGT motifs detected in file\n",
      "\n",
      "Complement found from 1031 to 1037\n",
      "Complement found from 2045 to 2051\n",
      "Complement found from 2735 to 2741\n",
      "Complement found from 3782 to 3788\n",
      "Complement found from 5634 to 5640\n",
      "Complement found from 15968 to 15974\n",
      "Seq is: CTTTAA\n",
      "6 CTTTAA motifs detected in file\n",
      "\n",
      "Complement found from 1182 to 1188\n",
      "Complement found from 3975 to 3981\n",
      "Complement found from 6104 to 6110\n",
      "Complement found from 7536 to 7542\n",
      "Complement found from 12070 to 12076\n",
      "Complement found from 14289 to 14295\n",
      "Complement found from 15127 to 15133\n",
      "Seq is: TTCATA\n",
      "7 TTCATA motifs detected in file\n",
      "\n",
      "Complement found from 3418 to 3424\n",
      "Complement found from 7017 to 7023\n",
      "Complement found from 7994 to 8000\n",
      "Seq is: ACGTTG\n",
      "3 ACGTTG motifs detected in file\n",
      "\n",
      "Complement found from 5409 to 5415\n",
      "Complement found from 8665 to 8671\n",
      "Complement found from 9205 to 9211\n",
      "Complement found from 9283 to 9289\n",
      "Complement found from 13221 to 13227\n",
      "Complement found from 14745 to 14751\n",
      "Complement found from 15462 to 15468\n",
      "Seq is: AATGAC\n",
      "7 AATGAC motifs detected in file\n",
      "\n",
      "Complement found from 3005 to 3011\n",
      "Complement found from 8015 to 8021\n",
      "Complement found from 14263 to 14269\n",
      "Seq is: TCCCGA\n",
      "3 TCCCGA motifs detected in file\n",
      "\n",
      "Complement found from 4179 to 4185\n",
      "Complement found from 10016 to 10022\n",
      "Complement found from 10061 to 10067\n",
      "Complement found from 14839 to 14845\n",
      "Seq is: AACTTC\n",
      "4 AACTTC motifs detected in file\n",
      "\n",
      "Complement found from 2717 to 2723\n",
      "Complement found from 9392 to 9398\n",
      "Complement found from 12052 to 12058\n",
      "Complement found from 12205 to 12211\n",
      "Seq is: CGAGAA\n",
      "4 CGAGAA motifs detected in file\n",
      "\n",
      "Complement found from 1705 to 1711\n",
      "Complement found from 1909 to 1915\n",
      "Complement found from 14914 to 14920\n",
      "Complement found from 15484 to 15490\n",
      "Complement found from 15502 to 15508\n",
      "Seq is: CCAGAC\n",
      "5 CCAGAC motifs detected in file\n",
      "\n",
      "Complement found from 10001 to 10007\n",
      "Complement found from 10484 to 10490\n",
      "Complement found from 14292 to 14298\n",
      "Complement found from 14496 to 14502\n",
      "Complement found from 14592 to 14598\n",
      "Complement found from 16134 to 16140\n",
      "Seq is: ATAAAT\n",
      "6 ATAAAT motifs detected in file\n",
      "\n",
      "Complement found from 1419 to 1425\n",
      "Complement found from 4972 to 4978\n",
      "Seq is: TGGATT\n",
      "2 TGGATT motifs detected in file\n",
      "\n",
      "Complement found from 936 to 942\n",
      "Complement found from 1315 to 1321\n",
      "Complement found from 8329 to 8335\n",
      "Complement found from 11372 to 11378\n",
      "Seq is: TAAAGA\n",
      "4 TAAAGA motifs detected in file\n",
      "\n",
      "Complement found from 2949 to 2955\n",
      "Complement found from 3617 to 3623\n",
      "Complement found from 3970 to 3976\n",
      "Complement found from 5922 to 5928\n",
      "Complement found from 6603 to 6609\n",
      "Complement found from 6634 to 6640\n",
      "Complement found from 7098 to 7104\n",
      "Complement found from 11207 to 11213\n",
      "Complement found from 11735 to 11741\n",
      "Complement found from 14673 to 14679\n",
      "Complement found from 15011 to 15017\n",
      "Complement found from 15338 to 15344\n",
      "Complement found from 15476 to 15482\n",
      "Complement found from 16014 to 16020\n",
      "Seq is: TATTCT\n",
      "14 TATTCT motifs detected in file\n",
      "\n",
      "Complement found from 233 to 239\n",
      "Complement found from 4300 to 4306\n",
      "Complement found from 4756 to 4762\n",
      "Complement found from 5035 to 5041\n",
      "Complement found from 5500 to 5506\n",
      "Complement found from 6094 to 6100\n",
      "Complement found from 6157 to 6163\n",
      "Complement found from 6658 to 6664\n",
      "Complement found from 7303 to 7309\n",
      "Complement found from 8040 to 8046\n",
      "Complement found from 9679 to 9685\n",
      "Complement found from 10078 to 10084\n",
      "Complement found from 10108 to 10114\n",
      "Complement found from 10573 to 10579\n",
      "Complement found from 13610 to 13616\n",
      "Complement found from 14544 to 14550\n",
      "Complement found from 15651 to 15657\n",
      "Seq is: AATAAT\n",
      "17 AATAAT motifs detected in file\n",
      "\n",
      "Complement found from 4075 to 4081\n",
      "Complement found from 6668 to 6674\n",
      "Complement found from 6810 to 6816\n",
      "Complement found from 7150 to 7156\n",
      "Complement found from 10457 to 10463\n",
      "Complement found from 11106 to 11112\n",
      "Complement found from 11985 to 11991\n",
      "Seq is: CATATT\n",
      "7 CATATT motifs detected in file\n",
      "\n",
      "Complement found from 453 to 459\n",
      "Complement found from 3165 to 3171\n",
      "Complement found from 4246 to 4252\n",
      "Complement found from 6150 to 6156\n",
      "Complement found from 6183 to 6189\n",
      "Complement found from 7092 to 7098\n",
      "Complement found from 7623 to 7629\n",
      "Complement found from 7952 to 7958\n",
      "Complement found from 8225 to 8231\n",
      "Complement found from 8602 to 8608\n",
      "Complement found from 10914 to 10920\n",
      "Complement found from 11231 to 11237\n",
      "Complement found from 12382 to 12388\n",
      "Complement found from 12480 to 12486\n",
      "Complement found from 13644 to 13650\n",
      "Complement found from 13751 to 13757\n",
      "Complement found from 14152 to 14158\n",
      "Complement found from 14486 to 14492\n",
      "Complement found from 16543 to 16549\n",
      "Seq is: TTCCCC\n",
      "19 TTCCCC motifs detected in file\n",
      "\n",
      "Complement found from 9 to 15\n",
      "Complement found from 2274 to 2280\n",
      "Complement found from 3097 to 3103\n",
      "Complement found from 6371 to 6377\n",
      "Seq is: TCTATC\n",
      "4 TCTATC motifs detected in file\n",
      "\n",
      "Complement found from 146 to 152\n",
      "Complement found from 501 to 507\n",
      "Complement found from 4393 to 4399\n",
      "Complement found from 7255 to 7261\n",
      "Complement found from 7797 to 7803\n",
      "Complement found from 7830 to 7836\n",
      "Complement found from 10345 to 10351\n",
      "Complement found from 11723 to 11729\n",
      "Complement found from 12389 to 12395\n",
      "Complement found from 12881 to 12887\n",
      "Complement found from 14120 to 14126\n",
      "Complement found from 15636 to 15642\n",
      "Complement found from 15642 to 15648\n",
      "Complement found from 15660 to 15666\n",
      "Complement found from 16403 to 16409\n",
      "Seq is: CATCCT\n",
      "15 CATCCT motifs detected in file\n",
      "\n",
      "Complement found from 3545 to 3551\n",
      "Complement found from 5455 to 5461\n",
      "Complement found from 6835 to 6841\n",
      "Complement found from 7809 to 7815\n",
      "Complement found from 11241 to 11247\n",
      "Complement found from 11430 to 11436\n",
      "Complement found from 13574 to 13580\n",
      "Complement found from 14379 to 14385\n",
      "Complement found from 14454 to 14460\n",
      "Seq is: CATCGC\n",
      "9 CATCGC motifs detected in file\n",
      "\n",
      "Complement found from 1723 to 1729\n",
      "Complement found from 5069 to 5075\n",
      "Complement found from 6700 to 6706\n",
      "Complement found from 7531 to 7537\n",
      "Complement found from 9072 to 9078\n",
      "Complement found from 9764 to 9770\n",
      "Complement found from 13455 to 13461\n",
      "Complement found from 15978 to 15984\n",
      "Seq is: ACCATT\n",
      "8 ACCATT motifs detected in file\n",
      "\n",
      "Complement found from 312 to 318\n",
      "Complement found from 5233 to 5239\n",
      "Complement found from 9571 to 9577\n",
      "Seq is: CCCGCT\n",
      "3 CCCGCT motifs detected in file\n",
      "\n",
      "Complement found from 27 to 33\n",
      "Complement found from 977 to 983\n",
      "Complement found from 4190 to 4196\n",
      "Complement found from 7719 to 7725\n",
      "Complement found from 11257 to 11263\n",
      "Complement found from 11292 to 11298\n",
      "Complement found from 11767 to 11773\n",
      "Complement found from 12012 to 12018\n",
      "Complement found from 13796 to 13802\n",
      "Complement found from 14400 to 14406\n",
      "Complement found from 14909 to 14915\n",
      "Complement found from 16065 to 16071\n",
      "Seq is: ACTCAC\n",
      "12 ACTCAC motifs detected in file\n",
      "\n",
      "Complement found from 1529 to 1535\n",
      "Complement found from 1835 to 1841\n",
      "Complement found from 4832 to 4838\n",
      "Complement found from 5842 to 5848\n",
      "Complement found from 8930 to 8936\n",
      "Complement found from 9253 to 9259\n",
      "Complement found from 10160 to 10166\n",
      "Complement found from 10276 to 10282\n",
      "Complement found from 10977 to 10983\n",
      "Complement found from 11534 to 11540\n",
      "Complement found from 12192 to 12198\n",
      "Complement found from 13024 to 13030\n",
      "Complement found from 14279 to 14285\n",
      "Complement found from 14416 to 14422\n",
      "Complement found from 15524 to 15530\n",
      "Complement found from 15535 to 15541\n",
      "Complement found from 16257 to 16263\n",
      "Seq is: ACCCCT\n",
      "17 ACCCCT motifs detected in file\n",
      "\n",
      "Complement found from 1697 to 1703\n",
      "Complement found from 5461 to 5467\n",
      "Complement found from 5651 to 5657\n",
      "Complement found from 6348 to 6354\n",
      "Complement found from 7866 to 7872\n",
      "Complement found from 8415 to 8421\n",
      "Complement found from 10098 to 10104\n",
      "Complement found from 10163 to 10169\n",
      "Complement found from 11861 to 11867\n",
      "Complement found from 12392 to 12398\n",
      "Complement found from 13211 to 13217\n",
      "Complement found from 13652 to 13658\n",
      "Complement found from 15409 to 15415\n",
      "Seq is: CCTTAC\n",
      "13 CCTTAC motifs detected in file\n",
      "\n",
      "Complement found from 3209 to 3215\n",
      "Complement found from 5303 to 5309\n",
      "Complement found from 5432 to 5438\n",
      "Complement found from 6312 to 6318\n",
      "Complement found from 6845 to 6851\n",
      "Complement found from 7399 to 7405\n",
      "Complement found from 8392 to 8398\n",
      "Complement found from 8619 to 8625\n",
      "Complement found from 9210 to 9216\n",
      "Complement found from 11140 to 11146\n",
      "Complement found from 12017 to 12023\n",
      "Complement found from 13053 to 13059\n",
      "Complement found from 13647 to 13653\n",
      "Complement found from 13677 to 13683\n",
      "Complement found from 14809 to 14815\n",
      "Complement found from 15262 to 15268\n",
      "Complement found from 16289 to 16295\n",
      "Seq is: CCCACC\n",
      "17 CCCACC motifs detected in file\n",
      "\n",
      "Complement found from 779 to 785\n",
      "Complement found from 1122 to 1128\n",
      "Complement found from 5427 to 5433\n",
      "Complement found from 5595 to 5601\n",
      "Complement found from 7727 to 7733\n",
      "Complement found from 8686 to 8692\n",
      "Complement found from 10420 to 10426\n",
      "Complement found from 11519 to 11525\n",
      "Complement found from 12153 to 12159\n",
      "Complement found from 13424 to 13430\n",
      "Complement found from 13971 to 13977\n",
      "Complement found from 14760 to 14766\n",
      "Complement found from 16178 to 16184\n",
      "Seq is: CAAAAC\n",
      "13 CAAAAC motifs detected in file\n",
      "\n",
      "Complement found from 487 to 493\n",
      "Complement found from 3396 to 3402\n",
      "Complement found from 5424 to 5430\n",
      "Complement found from 12854 to 12860\n",
      "Complement found from 14235 to 14241\n",
      "Seq is: ATACAA\n",
      "5 ATACAA motifs detected in file\n",
      "\n",
      "Complement found from 5416 to 5422\n",
      "Complement found from 5772 to 5778\n",
      "Complement found from 9947 to 9953\n",
      "Seq is: GTTTGA\n",
      "3 GTTTGA motifs detected in file\n",
      "\n",
      "Complement found from 1428 to 1434\n",
      "Complement found from 5553 to 5559\n",
      "Complement found from 7293 to 7299\n",
      "Complement found from 14204 to 14210\n",
      "Complement found from 15176 to 15182\n",
      "Complement found from 15770 to 15776\n",
      "Seq is: CAGTAA\n",
      "6 CAGTAA motifs detected in file\n",
      "\n",
      "Complement found from 835 to 841\n",
      "Complement found from 965 to 971\n",
      "Complement found from 1735 to 1741\n",
      "Complement found from 4004 to 4010\n",
      "Complement found from 4570 to 4576\n",
      "Complement found from 4609 to 4615\n",
      "Complement found from 4777 to 4783\n",
      "Complement found from 5404 to 5410\n",
      "Complement found from 8493 to 8499\n",
      "Complement found from 10057 to 10063\n",
      "Complement found from 12326 to 12332\n",
      "Complement found from 13871 to 13877\n",
      "Complement found from 14495 to 14501\n",
      "Complement found from 14775 to 14781\n",
      "Seq is: AATAAA\n",
      "14 AATAAA motifs detected in file\n",
      "\n",
      "Complement found from 770 to 776\n",
      "Complement found from 2747 to 2753\n",
      "Seq is: AATGCA\n",
      "2 AATGCA motifs detected in file\n",
      "\n",
      "Complement found from 2387 to 2393\n",
      "Complement found from 3720 to 3726\n",
      "Complement found from 4023 to 4029\n",
      "Complement found from 4233 to 4239\n",
      "Complement found from 8000 to 8006\n",
      "Complement found from 8562 to 8568\n",
      "Complement found from 10984 to 10990\n",
      "Complement found from 13281 to 13287\n",
      "Complement found from 13776 to 13782\n",
      "Complement found from 13941 to 13947\n",
      "Complement found from 14572 to 14578\n",
      "Complement found from 15418 to 15424\n",
      "Complement found from 15826 to 15832\n",
      "Seq is: ACAATC\n",
      "13 ACAATC motifs detected in file\n",
      "\n",
      "Complement found from 1256 to 1262\n",
      "Complement found from 1840 to 1846\n",
      "Complement found from 3199 to 3205\n",
      "Complement found from 4451 to 4457\n",
      "Complement found from 4893 to 4899\n",
      "Complement found from 5955 to 5961\n",
      "Complement found from 6591 to 6597\n",
      "Complement found from 9366 to 9372\n",
      "Complement found from 15511 to 15517\n",
      "Seq is: TATACC\n",
      "9 TATACC motifs detected in file\n",
      "\n",
      "Complement found from 143 to 149\n",
      "Complement found from 3329 to 3335\n",
      "Complement found from 5293 to 5299\n",
      "Complement found from 7665 to 7671\n",
      "Complement found from 7806 to 7812\n",
      "Complement found from 8427 to 8433\n",
      "Complement found from 9331 to 9337\n",
      "Complement found from 10474 to 10480\n",
      "Complement found from 10543 to 10549\n",
      "Complement found from 11726 to 11732\n",
      "Complement found from 12041 to 12047\n",
      "Complement found from 12063 to 12069\n",
      "Complement found from 15639 to 15645\n",
      "Complement found from 15741 to 15747\n",
      "Seq is: CCTCAT\n",
      "14 CCTCAT motifs detected in file\n",
      "\n",
      "Complement found from 3869 to 3875\n",
      "Complement found from 4516 to 4522\n",
      "Complement found from 5374 to 5380\n",
      "Complement found from 5449 to 5455\n",
      "Complement found from 6877 to 6883\n",
      "Complement found from 9167 to 9173\n",
      "Complement found from 9355 to 9361\n",
      "Complement found from 11130 to 11136\n",
      "Complement found from 11496 to 11502\n",
      "Complement found from 12345 to 12351\n",
      "Complement found from 14643 to 14649\n",
      "Seq is: CACACT\n",
      "11 CACACT motifs detected in file\n",
      "\n",
      "Complement found from 996 to 1002\n",
      "Complement found from 2249 to 2255\n",
      "Complement found from 5132 to 5138\n",
      "Complement found from 7939 to 7945\n",
      "Complement found from 11320 to 11326\n",
      "Complement found from 12319 to 12325\n",
      "Complement found from 15972 to 15978\n",
      "Complement found from 16245 to 16251\n",
      "Seq is: AACTCC\n",
      "8 AACTCC motifs detected in file\n",
      "\n",
      "Complement found from 1103 to 1109\n",
      "Complement found from 1231 to 1237\n",
      "Complement found from 3595 to 3601\n",
      "Complement found from 5366 to 5372\n",
      "Complement found from 5888 to 5894\n",
      "Complement found from 6543 to 6549\n",
      "Complement found from 8682 to 8688\n",
      "Complement found from 9467 to 9473\n",
      "Complement found from 10542 to 10548\n",
      "Complement found from 13843 to 13849\n",
      "Complement found from 14064 to 14070\n",
      "Complement found from 14410 to 14416\n",
      "Seq is: ACCTCA\n",
      "12 ACCTCA motifs detected in file\n",
      "\n",
      "Complement found from 3344 to 3350\n",
      "Complement found from 4678 to 4684\n",
      "Complement found from 9110 to 9116\n",
      "Complement found from 9655 to 9661\n",
      "Complement found from 16002 to 16008\n",
      "Seq is: TCTAAT\n",
      "5 TCTAAT motifs detected in file\n",
      "\n",
      "Complement found from 2330 to 2336\n",
      "Complement found from 4662 to 4668\n",
      "Complement found from 5109 to 5115\n",
      "Complement found from 6187 to 6193\n",
      "Complement found from 13756 to 13762\n",
      "Complement found from 14828 to 14834\n",
      "Seq is: CCGCAT\n",
      "6 CCGCAT motifs detected in file\n",
      "\n",
      "Complement found from 1263 to 1269\n",
      "Complement found from 4503 to 4509\n",
      "Complement found from 4881 to 4887\n",
      "Complement found from 6339 to 6345\n",
      "Complement found from 9969 to 9975\n",
      "Complement found from 10506 to 10512\n",
      "Complement found from 11579 to 11585\n",
      "Seq is: CCATCT\n",
      "7 CCATCT motifs detected in file\n",
      "\n",
      "Complement found from 3182 to 3188\n",
      "Complement found from 4491 to 4497\n",
      "Complement found from 6933 to 6939\n",
      "Complement found from 7262 to 7268\n",
      "Complement found from 9097 to 9103\n",
      "Complement found from 12700 to 12706\n",
      "Complement found from 15293 to 15299\n",
      "Seq is: TCATCT\n",
      "7 TCATCT motifs detected in file\n",
      "\n",
      "Complement found from 2188 to 2194\n",
      "Complement found from 2261 to 2267\n",
      "Complement found from 9535 to 9541\n",
      "Complement found from 10021 to 10027\n",
      "Complement found from 13010 to 13016\n",
      "Seq is: CCAATT\n",
      "5 CCAATT motifs detected in file\n",
      "\n",
      "Complement found from 307 to 313\n",
      "Complement found from 457 to 463\n",
      "Complement found from 958 to 964\n",
      "Complement found from 3115 to 3121\n",
      "Complement found from 3569 to 3575\n",
      "Complement found from 4105 to 4111\n",
      "Complement found from 5443 to 5449\n",
      "Complement found from 6218 to 6224\n",
      "Complement found from 6290 to 6296\n",
      "Complement found from 7815 to 7821\n",
      "Complement found from 7862 to 7868\n",
      "Complement found from 8470 to 8476\n",
      "Complement found from 10551 to 10557\n",
      "Complement found from 13447 to 13453\n",
      "Complement found from 13582 to 13588\n",
      "Complement found from 14261 to 14267\n",
      "Complement found from 14527 to 14533\n",
      "Complement found from 14805 to 14811\n",
      "Complement found from 15141 to 15147\n",
      "Complement found from 15380 to 15386\n",
      "Complement found from 15538 to 15544\n",
      "Complement found from 16186 to 16192\n",
      "Seq is: CCTCCC\n",
      "22 CCTCCC motifs detected in file\n",
      "\n",
      "Complement found from 1701 to 1707\n",
      "Complement found from 2219 to 2225\n",
      "Complement found from 2783 to 2789\n",
      "Complement found from 4733 to 4739\n",
      "Complement found from 5105 to 5111\n",
      "Complement found from 7227 to 7233\n",
      "Complement found from 8380 to 8386\n",
      "Complement found from 8459 to 8465\n",
      "Complement found from 10126 to 10132\n",
      "Complement found from 10791 to 10797\n",
      "Complement found from 10963 to 10969\n",
      "Complement found from 13849 to 13855\n",
      "Seq is: ACTACC\n",
      "12 ACTACC motifs detected in file\n",
      "\n",
      "Seq is: ACCGAT\n",
      "0 ACCGAT motifs detected in file\n",
      "\n",
      "Complement found from 1701 to 1707\n",
      "Complement found from 2219 to 2225\n",
      "Complement found from 2783 to 2789\n",
      "Complement found from 4733 to 4739\n",
      "Complement found from 5105 to 5111\n",
      "Complement found from 7227 to 7233\n",
      "Complement found from 8380 to 8386\n",
      "Complement found from 8459 to 8465\n",
      "Complement found from 10126 to 10132\n",
      "Complement found from 10791 to 10797\n",
      "Complement found from 10963 to 10969\n",
      "Complement found from 13849 to 13855\n",
      "Seq is: ACTACC\n",
      "12 ACTACC motifs detected in file\n",
      "\n",
      "Complement found from 1529 to 1535\n",
      "Complement found from 1835 to 1841\n",
      "Complement found from 4832 to 4838\n",
      "Complement found from 5842 to 5848\n",
      "Complement found from 8930 to 8936\n",
      "Complement found from 9253 to 9259\n",
      "Complement found from 10160 to 10166\n",
      "Complement found from 10276 to 10282\n",
      "Complement found from 10977 to 10983\n",
      "Complement found from 11534 to 11540\n",
      "Complement found from 12192 to 12198\n",
      "Complement found from 13024 to 13030\n",
      "Complement found from 14279 to 14285\n",
      "Complement found from 14416 to 14422\n",
      "Complement found from 15524 to 15530\n",
      "Complement found from 15535 to 15541\n",
      "Complement found from 16257 to 16263\n",
      "Seq is: ACCCCT\n",
      "17 ACCCCT motifs detected in file\n",
      "\n",
      "Complement found from 471 to 477\n",
      "Complement found from 2879 to 2885\n",
      "Complement found from 5102 to 5108\n",
      "Complement found from 5119 to 5125\n",
      "Complement found from 5152 to 5158\n",
      "Complement found from 5377 to 5383\n",
      "Complement found from 6306 to 6312\n",
      "Complement found from 6526 to 6532\n",
      "Complement found from 6681 to 6687\n",
      "Complement found from 10102 to 10108\n",
      "Complement found from 11289 to 11295\n",
      "Complement found from 12348 to 12354\n",
      "Complement found from 12583 to 12589\n",
      "Complement found from 12971 to 12977\n",
      "Complement found from 13418 to 13424\n",
      "Complement found from 14209 to 14215\n",
      "Complement found from 14906 to 14912\n",
      "Seq is: ACTACT\n",
      "17 ACTACT motifs detected in file\n",
      "\n",
      "Complement found from 6171 to 6177\n",
      "Complement found from 15389 to 15395\n",
      "Seq is: CCGATA\n",
      "2 CCGATA motifs detected in file\n",
      "\n",
      "Complement found from 283 to 289\n",
      "Complement found from 1009 to 1015\n",
      "Complement found from 1121 to 1127\n",
      "Complement found from 4867 to 4873\n",
      "Complement found from 5279 to 5285\n",
      "Complement found from 5426 to 5432\n",
      "Complement found from 7444 to 7450\n",
      "Complement found from 7726 to 7732\n",
      "Complement found from 10378 to 10384\n",
      "Complement found from 10419 to 10425\n",
      "Complement found from 11518 to 11524\n",
      "Complement found from 12415 to 12421\n",
      "Complement found from 13217 to 13223\n",
      "Complement found from 15866 to 15872\n",
      "Seq is: ACAAAA\n",
      "14 ACAAAA motifs detected in file\n",
      "\n",
      "Complement found from 1138 to 1144\n",
      "Complement found from 2354 to 2360\n",
      "Complement found from 5682 to 5688\n",
      "Complement found from 5950 to 5956\n",
      "Complement found from 7185 to 7191\n",
      "Complement found from 7716 to 7722\n",
      "Complement found from 8709 to 8715\n",
      "Complement found from 13163 to 13169\n",
      "Complement found from 14397 to 14403\n",
      "Complement found from 15324 to 15330\n",
      "Complement found from 15471 to 15477\n",
      "Complement found from 16462 to 16468\n",
      "Seq is: AACACT\n",
      "12 AACACT motifs detected in file\n",
      "\n",
      "Complement found from 1923 to 1929\n",
      "Complement found from 2193 to 2199\n",
      "Complement found from 4260 to 4266\n",
      "Seq is: TAAGAA\n",
      "3 TAAGAA motifs detected in file\n",
      "\n",
      "Complement found from 3021 to 3027\n",
      "Complement found from 10232 to 10238\n",
      "Complement found from 14015 to 14021\n",
      "Seq is: GCTATT\n",
      "3 GCTATT motifs detected in file\n",
      "\n",
      "Complement found from 8148 to 8154\n",
      "Complement found from 12121 to 12127\n",
      "Seq is: ACCGGG\n",
      "2 ACCGGG motifs detected in file\n",
      "\n",
      "Complement found from 1100 to 1106\n",
      "Complement found from 1218 to 1224\n",
      "Complement found from 1672 to 1678\n",
      "Complement found from 4313 to 4319\n",
      "Complement found from 4611 to 4617\n",
      "Complement found from 4977 to 4983\n",
      "Complement found from 5672 to 5678\n",
      "Complement found from 7371 to 7377\n",
      "Complement found from 8120 to 8126\n",
      "Complement found from 8824 to 8830\n",
      "Complement found from 10746 to 10752\n",
      "Complement found from 13687 to 13693\n",
      "Complement found from 14514 to 14520\n",
      "Complement found from 14583 to 14589\n",
      "Complement found from 14637 to 14643\n",
      "Complement found from 15920 to 15926\n",
      "Seq is: TAAACC\n",
      "16 TAAACC motifs detected in file\n",
      "\n",
      "Complement found from 9161 to 9167\n",
      "Seq is: CGTTTT\n",
      "1 CGTTTT motifs detected in file\n",
      "\n",
      "Complement found from 7194 to 7200\n",
      "Seq is: TCGGCC\n",
      "1 TCGGCC motifs detected in file\n",
      "\n",
      "Complement found from 3250 to 3256\n",
      "Complement found from 3347 to 3353\n",
      "Complement found from 9134 to 9140\n",
      "Complement found from 11606 to 11612\n",
      "Complement found from 11706 to 11712\n",
      "Complement found from 14940 to 14946\n",
      "Seq is: AATCGC\n",
      "6 AATCGC motifs detected in file\n",
      "\n",
      "Complement found from 704 to 710\n",
      "Complement found from 3168 to 3174\n",
      "Seq is: CCCCGT\n",
      "2 CCCCGT motifs detected in file\n",
      "\n",
      "Complement found from 103 to 109\n",
      "Complement found from 3076 to 3082\n",
      "Complement found from 6261 to 6267\n",
      "Complement found from 6570 to 6576\n",
      "Complement found from 15924 to 15930\n",
      "Seq is: CCGGAG\n",
      "5 CCGGAG motifs detected in file\n",
      "\n",
      "Complement found from 7858 to 7864\n",
      "Complement found from 8615 to 8621\n",
      "Seq is: GATCCC\n",
      "2 GATCCC motifs detected in file\n",
      "\n",
      "Complement found from 2324 to 2330\n",
      "Complement found from 4927 to 4933\n",
      "Complement found from 6226 to 6232\n",
      "Complement found from 6345 to 6351\n",
      "Complement found from 6366 to 6372\n",
      "Complement found from 6487 to 6493\n",
      "Complement found from 11069 to 11075\n",
      "Complement found from 11915 to 11921\n",
      "Complement found from 11934 to 11940\n",
      "Complement found from 12090 to 12096\n",
      "Complement found from 14284 to 14290\n",
      "Complement found from 15455 to 15461\n",
      "Complement found from 16442 to 16448\n",
      "Seq is: TCTCCT\n",
      "13 TCTCCT motifs detected in file\n",
      "\n",
      "Complement found from 3209 to 3215\n",
      "Complement found from 5303 to 5309\n",
      "Complement found from 5432 to 5438\n",
      "Complement found from 6312 to 6318\n",
      "Complement found from 6845 to 6851\n",
      "Complement found from 7399 to 7405\n",
      "Complement found from 8392 to 8398\n",
      "Complement found from 8619 to 8625\n",
      "Complement found from 9210 to 9216\n",
      "Complement found from 11140 to 11146\n",
      "Complement found from 12017 to 12023\n",
      "Complement found from 13053 to 13059\n",
      "Complement found from 13647 to 13653\n",
      "Complement found from 13677 to 13683\n",
      "Complement found from 14809 to 14815\n",
      "Complement found from 15262 to 15268\n",
      "Complement found from 16289 to 16295\n",
      "Seq is: CCCACC\n",
      "17 CCCACC motifs detected in file\n",
      "\n",
      "Complement found from 1842 to 1848\n",
      "Complement found from 10237 to 10243\n",
      "Complement found from 13483 to 13489\n",
      "Complement found from 14986 to 14992\n",
      "Complement found from 15282 to 15288\n",
      "Seq is: TACCTT\n",
      "5 TACCTT motifs detected in file\n",
      "\n",
      "Complement found from 3111 to 3117\n",
      "Complement found from 3287 to 3293\n",
      "Complement found from 5197 to 5203\n",
      "Complement found from 8106 to 8112\n",
      "Complement found from 8223 to 8229\n",
      "Complement found from 12380 to 12386\n",
      "Seq is: AATTCC\n",
      "6 AATTCC motifs detected in file\n",
      "\n",
      "Complement found from 1049 to 1055\n",
      "Complement found from 8561 to 8567\n",
      "Complement found from 9104 to 9110\n",
      "Complement found from 10983 to 10989\n",
      "Complement found from 11355 to 11361\n",
      "Complement found from 11999 to 12005\n",
      "Complement found from 13940 to 13946\n",
      "Complement found from 15417 to 15423\n",
      "Complement found from 15579 to 15585\n",
      "Seq is: CACAAT\n",
      "9 CACAAT motifs detected in file\n",
      "\n",
      "Complement found from 2755 to 2761\n",
      "Complement found from 2846 to 2852\n",
      "Complement found from 8584 to 8590\n",
      "Complement found from 11456 to 11462\n",
      "Complement found from 16300 to 16306\n",
      "Seq is: CAGTAC\n",
      "5 CAGTAC motifs detected in file\n",
      "\n",
      "Complement found from 1212 to 1218\n",
      "Complement found from 4341 to 4347\n",
      "Complement found from 7458 to 7464\n",
      "Complement found from 8002 to 8008\n",
      "Seq is: AATCGA\n",
      "4 AATCGA motifs detected in file\n",
      "\n",
      "Complement found from 360 to 366\n",
      "Complement found from 1291 to 1297\n",
      "Complement found from 1617 to 1623\n",
      "Complement found from 3155 to 3161\n",
      "Complement found from 5937 to 5943\n",
      "Complement found from 14237 to 14243\n",
      "Complement found from 14657 to 14663\n",
      "Complement found from 15681 to 15687\n",
      "Seq is: ACAAAG\n",
      "8 ACAAAG motifs detected in file\n",
      "\n",
      "Complement found from 2637 to 2643\n",
      "Complement found from 6882 to 6888\n",
      "Complement found from 9813 to 9819\n",
      "Seq is: TCCACG\n",
      "3 TCCACG motifs detected in file\n",
      "\n",
      "Complement found from 6370 to 6376\n",
      "Complement found from 8608 to 8614\n",
      "Complement found from 9716 to 9722\n",
      "Complement found from 11046 to 11052\n",
      "Seq is: CTCTAT\n",
      "4 CTCTAT motifs detected in file\n",
      "\n",
      "Complement found from 276 to 282\n",
      "Complement found from 3746 to 3752\n",
      "Complement found from 3835 to 3841\n",
      "Complement found from 4749 to 4755\n",
      "Complement found from 4951 to 4957\n",
      "Complement found from 5296 to 5302\n",
      "Complement found from 5308 to 5314\n",
      "Complement found from 6118 to 6124\n",
      "Complement found from 7066 to 7072\n",
      "Complement found from 7794 to 7800\n",
      "Complement found from 10342 to 10348\n",
      "Complement found from 10863 to 10869\n",
      "Complement found from 11778 to 11784\n",
      "Complement found from 12114 to 12120\n",
      "Complement found from 12632 to 12638\n",
      "Complement found from 13370 to 13376\n",
      "Complement found from 13517 to 13523\n",
      "Complement found from 14057 to 14063\n",
      "Complement found from 14342 to 14348\n",
      "Complement found from 14481 to 14487\n",
      "Complement found from 14667 to 14673\n",
      "Complement found from 15789 to 15795\n",
      "Seq is: CATCAT\n",
      "22 CATCAT motifs detected in file\n",
      "\n",
      "Complement found from 508 to 514\n",
      "Complement found from 4986 to 4992\n",
      "Complement found from 9242 to 9248\n",
      "Complement found from 12556 to 12562\n",
      "Seq is: CCCAGC\n",
      "4 CCCAGC motifs detected in file\n",
      "\n",
      "Complement found from 734 to 740\n",
      "Complement found from 5141 to 5147\n",
      "Complement found from 5465 to 5471\n",
      "Complement found from 11908 to 11914\n",
      "Complement found from 14692 to 14698\n",
      "Seq is: ACCACG\n",
      "5 ACCACG motifs detected in file\n",
      "\n",
      "Complement found from 1103 to 1109\n",
      "Complement found from 1231 to 1237\n",
      "Complement found from 3595 to 3601\n",
      "Complement found from 5366 to 5372\n",
      "Complement found from 5888 to 5894\n",
      "Complement found from 6543 to 6549\n",
      "Complement found from 8682 to 8688\n",
      "Complement found from 9467 to 9473\n",
      "Complement found from 10542 to 10548\n",
      "Complement found from 13843 to 13849\n",
      "Complement found from 14064 to 14070\n",
      "Complement found from 14410 to 14416\n",
      "Seq is: ACCTCA\n",
      "12 ACCTCA motifs detected in file\n",
      "\n",
      "Complement found from 5274 to 5280\n",
      "Complement found from 5798 to 5804\n",
      "Complement found from 9682 to 9688\n",
      "Complement found from 14537 to 14543\n",
      "Seq is: AATTCA\n",
      "4 AATTCA motifs detected in file\n",
      "\n",
      "Complement found from 4157 to 4163\n",
      "Complement found from 4519 to 4525\n",
      "Complement found from 4746 to 4752\n",
      "Complement found from 5452 to 5458\n",
      "Complement found from 6981 to 6987\n",
      "Complement found from 8066 to 8072\n",
      "Complement found from 8794 to 8800\n",
      "Complement found from 8972 to 8978\n",
      "Complement found from 9032 to 9038\n",
      "Complement found from 10437 to 10443\n",
      "Complement found from 11238 to 11244\n",
      "Complement found from 11499 to 11505\n",
      "Complement found from 12229 to 12235\n",
      "Complement found from 12424 to 12430\n",
      "Complement found from 12698 to 12704\n",
      "Complement found from 12916 to 12922\n",
      "Complement found from 13112 to 13118\n",
      "Complement found from 13268 to 13274\n",
      "Complement found from 14117 to 14123\n",
      "Complement found from 14792 to 14798\n",
      "Seq is: ACTCAT\n",
      "20 ACTCAT motifs detected in file\n",
      "\n",
      "Complement found from 146 to 152\n",
      "Complement found from 501 to 507\n",
      "Complement found from 4393 to 4399\n",
      "Complement found from 7255 to 7261\n",
      "Complement found from 7797 to 7803\n",
      "Complement found from 7830 to 7836\n",
      "Complement found from 10345 to 10351\n",
      "Complement found from 11723 to 11729\n",
      "Complement found from 12389 to 12395\n",
      "Complement found from 12881 to 12887\n",
      "Complement found from 14120 to 14126\n",
      "Complement found from 15636 to 15642\n",
      "Complement found from 15642 to 15648\n",
      "Complement found from 15660 to 15666\n",
      "Complement found from 16403 to 16409\n",
      "Seq is: CATCCT\n",
      "15 CATCCT motifs detected in file\n",
      "\n",
      "Complement found from 3684 to 3690\n",
      "Complement found from 3948 to 3954\n",
      "Complement found from 5349 to 5355\n",
      "Complement found from 7127 to 7133\n",
      "Complement found from 8998 to 9004\n",
      "Complement found from 11489 to 11495\n",
      "Complement found from 12810 to 12816\n",
      "Seq is: TACGCC\n",
      "7 TACGCC motifs detected in file\n",
      "\n",
      "Complement found from 3181 to 3187\n",
      "Complement found from 5297 to 5303\n",
      "Complement found from 6833 to 6839\n",
      "Complement found from 7261 to 7267\n",
      "Complement found from 7795 to 7801\n",
      "Complement found from 9096 to 9102\n",
      "Complement found from 10340 to 10346\n",
      "Complement found from 10864 to 10870\n",
      "Complement found from 11152 to 11158\n",
      "Complement found from 13371 to 13377\n",
      "Complement found from 13518 to 13524\n",
      "Complement found from 14058 to 14064\n",
      "Complement found from 14977 to 14983\n",
      "Seq is: ATCATC\n",
      "13 ATCATC motifs detected in file\n",
      "\n",
      "Complement found from 2944 to 2950\n",
      "Complement found from 3650 to 3656\n",
      "Complement found from 4672 to 4678\n",
      "Complement found from 7569 to 7575\n",
      "Complement found from 8564 to 8570\n",
      "Complement found from 9454 to 9460\n",
      "Complement found from 11784 to 11790\n",
      "Complement found from 12848 to 12854\n",
      "Complement found from 14368 to 14374\n",
      "Complement found from 15828 to 15834\n",
      "Complement found from 15834 to 15840\n",
      "Seq is: AATCCT\n",
      "11 AATCCT motifs detected in file\n",
      "\n",
      "Complement found from 1804 to 1810\n",
      "Complement found from 3197 to 3203\n",
      "Complement found from 3999 to 4005\n",
      "Complement found from 7558 to 7564\n",
      "Complement found from 8501 to 8507\n",
      "Complement found from 8865 to 8871\n",
      "Complement found from 10490 to 10496\n",
      "Complement found from 10592 to 10598\n",
      "Complement found from 10783 to 10789\n",
      "Complement found from 11077 to 11083\n",
      "Complement found from 11566 to 11572\n",
      "Complement found from 12780 to 12786\n",
      "Complement found from 15509 to 15515\n",
      "Seq is: ATTATA\n",
      "13 ATTATA motifs detected in file\n",
      "\n",
      "Complement found from 156 to 162\n",
      "Complement found from 398 to 404\n",
      "Complement found from 3861 to 3867\n",
      "Complement found from 6755 to 6761\n",
      "Complement found from 12900 to 12906\n",
      "Complement found from 15016 to 15022\n",
      "Seq is: TTTATC\n",
      "6 TTTATC motifs detected in file\n",
      "\n",
      "Complement found from 2084 to 2090\n",
      "Complement found from 2154 to 2160\n",
      "Complement found from 2739 to 2745\n",
      "Complement found from 5078 to 5084\n",
      "Complement found from 5514 to 5520\n",
      "Complement found from 11250 to 11256\n",
      "Complement found from 16005 to 16011\n",
      "Seq is: AATTTA\n",
      "7 AATTTA motifs detected in file\n",
      "\n",
      "Complement found from 3971 to 3977\n",
      "Complement found from 5072 to 5078\n",
      "Complement found from 6635 to 6641\n",
      "Complement found from 10219 to 10225\n",
      "Complement found from 13614 to 13620\n",
      "Complement found from 15012 to 15018\n",
      "Complement found from 15276 to 15282\n",
      "Complement found from 15339 to 15345\n",
      "Seq is: ATTCTT\n",
      "8 ATTCTT motifs detected in file\n",
      "\n",
      "Complement found from 2187 to 2193\n",
      "Complement found from 2269 to 2275\n",
      "Complement found from 4727 to 4733\n",
      "Complement found from 4736 to 4742\n",
      "Complement found from 5660 to 5666\n",
      "Complement found from 7889 to 7895\n",
      "Complement found from 8981 to 8987\n",
      "Complement found from 9213 to 9219\n",
      "Complement found from 9369 to 9375\n",
      "Complement found from 14250 to 14256\n",
      "Complement found from 14365 to 14371\n",
      "Complement found from 14698 to 14704\n",
      "Complement found from 14742 to 14748\n",
      "Seq is: ACCAAT\n",
      "13 ACCAAT motifs detected in file\n",
      "\n",
      "Complement found from 2387 to 2393\n",
      "Complement found from 3720 to 3726\n",
      "Complement found from 4023 to 4029\n",
      "Complement found from 4233 to 4239\n",
      "Complement found from 8000 to 8006\n",
      "Complement found from 8562 to 8568\n",
      "Complement found from 10984 to 10990\n",
      "Complement found from 13281 to 13287\n",
      "Complement found from 13776 to 13782\n",
      "Complement found from 13941 to 13947\n",
      "Complement found from 14572 to 14578\n",
      "Complement found from 15418 to 15424\n",
      "Complement found from 15826 to 15832\n",
      "Seq is: ACAATC\n",
      "13 ACAATC motifs detected in file\n",
      "\n",
      "Complement found from 2395 to 2401\n",
      "Complement found from 2426 to 2432\n",
      "Complement found from 6595 to 6601\n",
      "Complement found from 8344 to 8350\n",
      "Complement found from 8660 to 8666\n",
      "Complement found from 9351 to 9357\n",
      "Complement found from 9557 to 9563\n",
      "Complement found from 10713 to 10719\n",
      "Complement found from 10777 to 10783\n",
      "Complement found from 11330 to 11336\n",
      "Complement found from 12826 to 12832\n",
      "Complement found from 13858 to 13864\n",
      "Complement found from 13903 to 13909\n",
      "Complement found from 14185 to 14191\n",
      "Complement found from 14819 to 14825\n",
      "Complement found from 16277 to 16283\n",
      "Seq is: CCAACA\n",
      "16 CCAACA motifs detected in file\n",
      "\n",
      "Complement found from 3832 to 3838\n",
      "Complement found from 4633 to 4639\n",
      "Complement found from 6424 to 6430\n",
      "Complement found from 7063 to 7069\n",
      "Complement found from 10651 to 10657\n",
      "Seq is: TGCCAT\n",
      "5 TGCCAT motifs detected in file\n",
      "\n",
      "Complement found from 1656 to 1662\n",
      "Complement found from 2905 to 2911\n",
      "Complement found from 7990 to 7996\n",
      "Complement found from 16141 to 16147\n",
      "Complement found from 16394 to 16400\n",
      "Seq is: CTTGAC\n",
      "5 CTTGAC motifs detected in file\n",
      "\n",
      "Complement found from 9451 to 9457\n",
      "Complement found from 10451 to 10457\n",
      "Seq is: GATAAT\n",
      "2 GATAAT motifs detected in file\n",
      "\n",
      "Complement found from 7601 to 7607\n",
      "Complement found from 15800 to 15806\n",
      "Seq is: AAGTAG\n",
      "2 AAGTAG motifs detected in file\n",
      "\n",
      "Complement found from 6708 to 6714\n",
      "Seq is: GATACA\n",
      "1 GATACA motifs detected in file\n",
      "\n",
      "Complement found from 3339 to 3345\n",
      "Complement found from 4227 to 4233\n",
      "Complement found from 5437 to 5443\n",
      "Complement found from 6584 to 6590\n",
      "Complement found from 7956 to 7962\n",
      "Complement found from 8029 to 8035\n",
      "Complement found from 12085 to 12091\n",
      "Complement found from 12433 to 12439\n",
      "Complement found from 13692 to 13698\n",
      "Complement found from 14629 to 14635\n",
      "Complement found from 15383 to 15389\n",
      "Seq is: CCCATT\n",
      "11 CCCATT motifs detected in file\n",
      "\n",
      "Complement found from 996 to 1002\n",
      "Complement found from 2249 to 2255\n",
      "Complement found from 5132 to 5138\n",
      "Complement found from 7939 to 7945\n",
      "Complement found from 11320 to 11326\n",
      "Complement found from 12319 to 12325\n",
      "Complement found from 15972 to 15978\n",
      "Complement found from 16245 to 16251\n",
      "Seq is: AACTCC\n",
      "8 AACTCC motifs detected in file\n",
      "\n",
      "Complement found from 4159 to 4165\n",
      "Complement found from 9333 to 9339\n",
      "Complement found from 12071 to 12077\n",
      "Complement found from 12426 to 12432\n",
      "Complement found from 13539 to 13545\n",
      "Complement found from 14233 to 14239\n",
      "Complement found from 14344 to 14350\n",
      "Seq is: TCATAC\n",
      "7 TCATAC motifs detected in file\n",
      "\n",
      "Complement found from 13915 to 13921\n",
      "Complement found from 15275 to 15281\n",
      "Complement found from 15999 to 16005\n",
      "Seq is: GATTCT\n",
      "3 GATTCT motifs detected in file\n",
      "\n",
      "Complement found from 780 to 786\n",
      "Complement found from 10421 to 10427\n",
      "Seq is: AAAACG\n",
      "2 AAAACG motifs detected in file\n",
      "\n",
      "Complement found from 3941 to 3947\n",
      "Complement found from 13520 to 13526\n",
      "Complement found from 14799 to 14805\n",
      "Seq is: CATCGA\n",
      "3 CATCGA motifs detected in file\n",
      "\n",
      "Complement found from 353 to 359\n",
      "Complement found from 557 to 563\n",
      "Complement found from 1063 to 1069\n",
      "Complement found from 1682 to 1688\n",
      "Complement found from 1731 to 1737\n",
      "Complement found from 2233 to 2239\n",
      "Complement found from 3715 to 3721\n",
      "Complement found from 5252 to 5258\n",
      "Complement found from 7468 to 7474\n",
      "Complement found from 12302 to 12308\n",
      "Complement found from 12547 to 12553\n",
      "Complement found from 12668 to 12674\n",
      "Complement found from 14071 to 14077\n",
      "Complement found from 14532 to 14538\n",
      "Complement found from 15988 to 15994\n",
      "Seq is: CCCAAA\n",
      "15 CCCAAA motifs detected in file\n",
      "\n",
      "Complement found from 1732 to 1738\n",
      "Complement found from 4897 to 4903\n",
      "Complement found from 5253 to 5259\n",
      "Complement found from 8626 to 8632\n",
      "Complement found from 9676 to 9682\n",
      "Complement found from 10465 to 10471\n",
      "Complement found from 10887 to 10893\n",
      "Complement found from 12323 to 12329\n",
      "Complement found from 14042 to 14048\n",
      "Complement found from 14874 to 14880\n",
      "Complement found from 15153 to 15159\n",
      "Seq is: CCAAAT\n",
      "11 CCAAAT motifs detected in file\n",
      "\n",
      "Complement found from 6951 to 6957\n",
      "Seq is: TAGGTG\n",
      "1 TAGGTG motifs detected in file\n",
      "\n",
      "Complement found from 985 to 991\n",
      "Seq is: GAGTTG\n",
      "1 GAGTTG motifs detected in file\n",
      "\n",
      "Complement found from 9816 to 9822\n",
      "Complement found from 11387 to 11393\n",
      "Complement found from 14682 to 14688\n",
      "Seq is: ACGGAC\n",
      "3 ACGGAC motifs detected in file\n",
      "\n",
      "Complement found from 9924 to 9930\n",
      "Complement found from 14438 to 14444\n",
      "Seq is: GATACT\n",
      "2 GATACT motifs detected in file\n",
      "\n",
      "Complement found from 4380 to 4386\n",
      "Complement found from 5479 to 5485\n",
      "Complement found from 5958 to 5964\n",
      "Complement found from 6600 to 6606\n",
      "Complement found from 10901 to 10907\n",
      "Complement found from 12077 to 12083\n",
      "Complement found from 12742 to 12748\n",
      "Complement found from 14147 to 14153\n",
      "Seq is: ACCTAT\n",
      "8 ACCTAT motifs detected in file\n",
      "\n",
      "Complement found from 4100 to 4106\n",
      "Complement found from 4597 to 4603\n",
      "Complement found from 5390 to 5396\n",
      "Complement found from 7168 to 7174\n",
      "Complement found from 7285 to 7291\n",
      "Complement found from 12244 to 12250\n",
      "Complement found from 13160 to 13166\n",
      "Complement found from 13829 to 13835\n",
      "Complement found from 15747 to 15753\n",
      "Seq is: TCTAAC\n",
      "9 TCTAAC motifs detected in file\n",
      "\n",
      "Complement found from 4902 to 4908\n",
      "Complement found from 4936 to 4942\n",
      "Complement found from 6224 to 6230\n",
      "Complement found from 6494 to 6500\n",
      "Complement found from 11789 to 11795\n",
      "Complement found from 15453 to 15459\n",
      "Seq is: TCTCTC\n",
      "6 TCTCTC motifs detected in file\n",
      "\n",
      "Complement found from 2250 to 2256\n",
      "Complement found from 3326 to 3332\n",
      "Complement found from 3827 to 3833\n",
      "Complement found from 5010 to 5016\n",
      "Complement found from 5473 to 5479\n",
      "Complement found from 6232 to 6238\n",
      "Complement found from 7940 to 7946\n",
      "Complement found from 7986 to 7992\n",
      "Complement found from 8412 to 8418\n",
      "Complement found from 8783 to 8789\n",
      "Complement found from 9595 to 9601\n",
      "Complement found from 10971 to 10977\n",
      "Complement found from 11321 to 11327\n",
      "Complement found from 13985 to 13991\n",
      "Complement found from 14132 to 14138\n",
      "Complement found from 14441 to 14447\n",
      "Complement found from 14850 to 14856\n",
      "Complement found from 15723 to 15729\n",
      "Seq is: ACTCCT\n",
      "18 ACTCCT motifs detected in file\n",
      "\n",
      "Seq is: CTTCCG\n",
      "0 CTTCCG motifs detected in file\n",
      "\n",
      "Complement found from 770 to 776\n",
      "Complement found from 2747 to 2753\n",
      "Seq is: AATGCA\n",
      "2 AATGCA motifs detected in file\n",
      "\n",
      "Complement found from 730 to 736\n",
      "Complement found from 2599 to 2605\n",
      "Complement found from 5371 to 5377\n",
      "Complement found from 5811 to 5817\n",
      "Complement found from 6469 to 6475\n",
      "Complement found from 8651 to 8657\n",
      "Complement found from 9216 to 9222\n",
      "Complement found from 9634 to 9640\n",
      "Complement found from 14138 to 14144\n",
      "Complement found from 14877 to 14883\n",
      "Complement found from 15375 to 15381\n",
      "Complement found from 15396 to 15402\n",
      "Complement found from 15710 to 15716\n",
      "Seq is: AATCAC\n",
      "13 AATCAC motifs detected in file\n",
      "\n",
      "Complement found from 1188 to 1194\n",
      "Complement found from 4906 to 4912\n",
      "Complement found from 6220 to 6226\n",
      "Complement found from 7860 to 7866\n",
      "Complement found from 8472 to 8478\n",
      "Complement found from 9083 to 9089\n",
      "Complement found from 10328 to 10334\n",
      "Complement found from 10625 to 10631\n",
      "Complement found from 10868 to 10874\n",
      "Complement found from 11977 to 11983\n",
      "Complement found from 12100 to 12106\n",
      "Complement found from 13449 to 13455\n",
      "Seq is: TCCCTC\n",
      "12 TCCCTC motifs detected in file\n",
      "\n",
      "Complement found from 726 to 732\n",
      "Complement found from 2068 to 2074\n",
      "Complement found from 8118 to 8124\n",
      "Complement found from 12944 to 12950\n",
      "Seq is: TCTAAA\n",
      "4 TCTAAA motifs detected in file\n",
      "\n",
      "Complement found from 2960 to 2966\n",
      "Complement found from 5385 to 5391\n",
      "Complement found from 6667 to 6673\n",
      "Complement found from 6772 to 6778\n",
      "Complement found from 9363 to 9369\n",
      "Complement found from 14519 to 14525\n",
      "Complement found from 15666 to 15672\n",
      "Seq is: CCATAT\n",
      "7 CCATAT motifs detected in file\n",
      "\n",
      "Complement found from 439 to 445\n",
      "Complement found from 861 to 867\n",
      "Complement found from 1831 to 1837\n",
      "Complement found from 5185 to 5191\n",
      "Complement found from 6529 to 6535\n",
      "Complement found from 7731 to 7737\n",
      "Complement found from 7741 to 7747\n",
      "Complement found from 8678 to 8684\n",
      "Complement found from 8769 to 8775\n",
      "Complement found from 9346 to 9352\n",
      "Complement found from 9358 to 9364\n",
      "Complement found from 10301 to 10307\n",
      "Complement found from 10959 to 10965\n",
      "Complement found from 13656 to 13662\n",
      "Complement found from 13740 to 13746\n",
      "Complement found from 14764 to 14770\n",
      "Seq is: ACTAAC\n",
      "16 ACTAAC motifs detected in file\n",
      "\n",
      "Complement found from 1370 to 1376\n",
      "Complement found from 3501 to 3507\n",
      "Complement found from 4499 to 4505\n",
      "Complement found from 5047 to 5053\n",
      "Complement found from 5343 to 5349\n",
      "Complement found from 6285 to 6291\n",
      "Complement found from 8276 to 8282\n",
      "Complement found from 9184 to 9190\n",
      "Complement found from 11039 to 11045\n",
      "Complement found from 13786 to 13792\n",
      "Complement found from 13918 to 13924\n",
      "Seq is: TCTACC\n",
      "11 TCTACC motifs detected in file\n",
      "\n",
      "Complement found from 1223 to 1229\n",
      "Complement found from 3006 to 3012\n",
      "Complement found from 4138 to 4144\n",
      "Complement found from 6170 to 6176\n",
      "Complement found from 7232 to 7238\n",
      "Complement found from 8016 to 8022\n",
      "Complement found from 11159 to 11165\n",
      "Seq is: CCCGAT\n",
      "7 CCCGAT motifs detected in file\n",
      "\n",
      "Complement found from 284 to 290\n",
      "Complement found from 355 to 361\n",
      "Complement found from 2501 to 2507\n",
      "Complement found from 4603 to 4609\n",
      "Complement found from 4868 to 4874\n",
      "Complement found from 5280 to 5286\n",
      "Complement found from 7445 to 7451\n",
      "Complement found from 7513 to 7519\n",
      "Complement found from 9431 to 9437\n",
      "Complement found from 10046 to 10052\n",
      "Complement found from 10379 to 10385\n",
      "Complement found from 10812 to 10818\n",
      "Complement found from 12304 to 12310\n",
      "Complement found from 12416 to 12422\n",
      "Complement found from 13229 to 13235\n",
      "Complement found from 14073 to 14079\n",
      "Seq is: CAAAAA\n",
      "16 CAAAAA motifs detected in file\n",
      "\n",
      "Complement found from 2755 to 2761\n",
      "Complement found from 2846 to 2852\n",
      "Complement found from 8584 to 8590\n",
      "Complement found from 11456 to 11462\n",
      "Complement found from 16300 to 16306\n",
      "Seq is: CAGTAC\n",
      "5 CAGTAC motifs detected in file\n",
      "\n",
      "Complement found from 3467 to 3473\n",
      "Complement found from 6211 to 6217\n",
      "Complement found from 9990 to 9996\n",
      "Complement found from 11460 to 11466\n",
      "Complement found from 11621 to 11627\n",
      "Complement found from 14348 to 14354\n",
      "Seq is: ACTCTT\n",
      "6 ACTCTT motifs detected in file\n",
      "\n",
      "Complement found from 6451 to 6457\n",
      "Seq is: CTTCGT\n",
      "1 CTTCGT motifs detected in file\n",
      "\n",
      "Complement found from 4845 to 4851\n",
      "Complement found from 9291 to 9297\n",
      "Seq is: CCGGCC\n",
      "2 CCGGCC motifs detected in file\n",
      "\n",
      "Complement found from 3272 to 3278\n",
      "Complement found from 5859 to 5865\n",
      "Complement found from 6781 to 6787\n",
      "Complement found from 8356 to 8362\n",
      "Complement found from 16341 to 16347\n",
      "Seq is: TACAGT\n",
      "5 TACAGT motifs detected in file\n",
      "\n",
      "Complement found from 308 to 314\n",
      "Complement found from 959 to 965\n",
      "Complement found from 3570 to 3576\n",
      "Complement found from 4055 to 4061\n",
      "Complement found from 5381 to 5387\n",
      "Complement found from 5444 to 5450\n",
      "Complement found from 5485 to 5491\n",
      "Complement found from 13032 to 13038\n",
      "Complement found from 14528 to 14534\n",
      "Complement found from 14806 to 14812\n",
      "Complement found from 15539 to 15545\n",
      "Complement found from 16187 to 16193\n",
      "Seq is: CTCCCC\n",
      "12 CTCCCC motifs detected in file\n",
      "\n",
      "Complement found from 2913 to 2919\n",
      "Complement found from 6885 to 6891\n",
      "Seq is: ACGGAA\n",
      "2 ACGGAA motifs detected in file\n",
      "\n",
      "Complement found from 3339 to 3345\n",
      "Complement found from 4227 to 4233\n",
      "Complement found from 5437 to 5443\n",
      "Complement found from 6584 to 6590\n",
      "Complement found from 7956 to 7962\n",
      "Complement found from 8029 to 8035\n",
      "Complement found from 12085 to 12091\n",
      "Complement found from 12433 to 12439\n",
      "Complement found from 13692 to 13698\n",
      "Complement found from 14629 to 14635\n",
      "Complement found from 15383 to 15389\n",
      "Seq is: CCCATT\n",
      "11 CCCATT motifs detected in file\n",
      "\n",
      "Complement found from 1550 to 1556\n",
      "Complement found from 6578 to 6584\n",
      "Seq is: GGAGAC\n",
      "2 GGAGAC motifs detected in file\n",
      "\n",
      "Complement found from 982 to 988\n",
      "Complement found from 4354 to 4360\n",
      "Complement found from 4554 to 4560\n",
      "Complement found from 8513 to 8519\n",
      "Complement found from 9639 to 9645\n",
      "Complement found from 11324 to 11330\n",
      "Complement found from 13552 to 13558\n",
      "Seq is: CCTGAG\n",
      "7 CCTGAG motifs detected in file\n",
      "\n",
      "Complement found from 1266 to 1272\n",
      "Complement found from 3469 to 3475\n",
      "Complement found from 3973 to 3979\n",
      "Complement found from 4692 to 4698\n",
      "Complement found from 6102 to 6108\n",
      "Complement found from 7934 to 7940\n",
      "Complement found from 9100 to 9106\n",
      "Complement found from 11623 to 11629\n",
      "Complement found from 12685 to 12691\n",
      "Complement found from 13336 to 13342\n",
      "Seq is: TCTTCA\n",
      "10 TCTTCA motifs detected in file\n",
      "\n",
      "Complement found from 263 to 269\n",
      "Complement found from 9843 to 9849\n",
      "Complement found from 10807 to 10813\n",
      "Complement found from 13486 to 13492\n",
      "Complement found from 13816 to 13822\n",
      "Seq is: CTTTCC\n",
      "5 CTTTCC motifs detected in file\n",
      "\n",
      "Complement found from 1223 to 1229\n",
      "Complement found from 3006 to 3012\n",
      "Complement found from 4138 to 4144\n",
      "Complement found from 6170 to 6176\n",
      "Complement found from 7232 to 7238\n",
      "Complement found from 8016 to 8022\n",
      "Complement found from 11159 to 11165\n",
      "Seq is: CCCGAT\n",
      "7 CCCGAT motifs detected in file\n",
      "\n",
      "Complement found from 10383 to 10389\n",
      "Complement found from 12272 to 12278\n",
      "Seq is: AAGGAT\n",
      "2 AAGGAT motifs detected in file\n",
      "\n",
      "Complement found from 1010 to 1016\n",
      "Complement found from 1958 to 1964\n",
      "Complement found from 4364 to 4370\n",
      "Complement found from 4996 to 5002\n",
      "Complement found from 7132 to 7138\n",
      "Complement found from 8522 to 8528\n",
      "Complement found from 13218 to 13224\n",
      "Complement found from 14534 to 14540\n",
      "Complement found from 15867 to 15873\n",
      "Seq is: CAAAAT\n",
      "9 CAAAAT motifs detected in file\n",
      "\n",
      "Complement found from 3043 to 3049\n",
      "Complement found from 7855 to 7861\n",
      "Seq is: AACGAT\n",
      "2 AACGAT motifs detected in file\n",
      "\n",
      "Complement found from 1213 to 1219\n",
      "Seq is: ATCGAT\n",
      "1 ATCGAT motifs detected in file\n",
      "\n",
      "Complement found from 4730 to 4736\n",
      "Complement found from 4743 to 4749\n",
      "Complement found from 5564 to 5570\n",
      "Complement found from 7738 to 7744\n",
      "Complement found from 8377 to 8383\n",
      "Complement found from 10522 to 10528\n",
      "Complement found from 10576 to 10582\n",
      "Complement found from 10956 to 10962\n",
      "Complement found from 14578 to 14584\n",
      "Complement found from 15870 to 15876\n",
      "Complement found from 16137 to 16143\n",
      "Seq is: AATACT\n",
      "11 AATACT motifs detected in file\n",
      "\n",
      "Complement found from 233 to 239\n",
      "Complement found from 4300 to 4306\n",
      "Complement found from 4756 to 4762\n",
      "Complement found from 5035 to 5041\n",
      "Complement found from 5500 to 5506\n",
      "Complement found from 6094 to 6100\n",
      "Complement found from 6157 to 6163\n",
      "Complement found from 6658 to 6664\n",
      "Complement found from 7303 to 7309\n",
      "Complement found from 8040 to 8046\n",
      "Complement found from 9679 to 9685\n",
      "Complement found from 10078 to 10084\n",
      "Complement found from 10108 to 10114\n",
      "Complement found from 10573 to 10579\n",
      "Complement found from 13610 to 13616\n",
      "Complement found from 14544 to 14550\n",
      "Complement found from 15651 to 15657\n",
      "Seq is: AATAAT\n",
      "17 AATAAT motifs detected in file\n",
      "\n",
      "Complement found from 470 to 476\n",
      "Complement found from 1700 to 1706\n",
      "Complement found from 3439 to 3445\n",
      "Complement found from 4732 to 4738\n",
      "Complement found from 5104 to 5110\n",
      "Complement found from 5118 to 5124\n",
      "Complement found from 5151 to 5157\n",
      "Complement found from 6525 to 6531\n",
      "Complement found from 6680 to 6686\n",
      "Complement found from 6999 to 7005\n",
      "Complement found from 7013 to 7019\n",
      "Complement found from 8158 to 8164\n",
      "Complement found from 8379 to 8385\n",
      "Complement found from 10101 to 10107\n",
      "Complement found from 10790 to 10796\n",
      "Complement found from 11288 to 11294\n",
      "Complement found from 14211 to 14217\n",
      "Complement found from 15412 to 15418\n",
      "Seq is: TACTAC\n",
      "18 TACTAC motifs detected in file\n",
      "\n",
      "Complement found from 278 to 284\n",
      "Complement found from 2441 to 2447\n",
      "Complement found from 4761 to 4767\n",
      "Complement found from 6120 to 6126\n",
      "Complement found from 7537 to 7543\n",
      "Complement found from 7667 to 7673\n",
      "Complement found from 10605 to 10611\n",
      "Complement found from 11702 to 11708\n",
      "Complement found from 11780 to 11786\n",
      "Complement found from 13270 to 13276\n",
      "Complement found from 14290 to 14296\n",
      "Complement found from 16518 to 16524\n",
      "Seq is: TCATAA\n",
      "12 TCATAA motifs detected in file\n",
      "\n",
      "Complement found from 381 to 387\n",
      "Complement found from 529 to 535\n",
      "Complement found from 862 to 868\n",
      "Complement found from 1832 to 1838\n",
      "Complement found from 2578 to 2584\n",
      "Complement found from 4101 to 4107\n",
      "Complement found from 4598 to 4604\n",
      "Complement found from 5237 to 5243\n",
      "Complement found from 5839 to 5845\n",
      "Complement found from 6335 to 6341\n",
      "Complement found from 8679 to 8685\n",
      "Complement found from 8770 to 8776\n",
      "Complement found from 9003 to 9009\n",
      "Complement found from 9347 to 9353\n",
      "Complement found from 9359 to 9365\n",
      "Complement found from 10302 to 10308\n",
      "Complement found from 11852 to 11858\n",
      "Complement found from 12363 to 12369\n",
      "Complement found from 13854 to 13860\n",
      "Complement found from 13998 to 14004\n",
      "Complement found from 14124 to 14130\n",
      "Complement found from 14384 to 14390\n",
      "Complement found from 14765 to 14771\n",
      "Complement found from 15748 to 15754\n",
      "Seq is: CTAACC\n",
      "24 CTAACC motifs detected in file\n",
      "\n",
      "Complement found from 277 to 283\n",
      "Complement found from 4760 to 4766\n",
      "Complement found from 4889 to 4895\n",
      "Complement found from 4952 to 4958\n",
      "Complement found from 5309 to 5315\n",
      "Complement found from 6119 to 6125\n",
      "Complement found from 7067 to 7073\n",
      "Complement found from 7148 to 7154\n",
      "Complement found from 7630 to 7636\n",
      "Complement found from 9228 to 9234\n",
      "Complement found from 10455 to 10461\n",
      "Complement found from 11104 to 11110\n",
      "Complement found from 11779 to 11785\n",
      "Complement found from 12633 to 12639\n",
      "Complement found from 13538 to 13544\n",
      "Complement found from 14232 to 14238\n",
      "Complement found from 14343 to 14349\n",
      "Seq is: ATCATA\n",
      "17 ATCATA motifs detected in file\n",
      "\n",
      "Complement found from 2187 to 2193\n",
      "Complement found from 2269 to 2275\n",
      "Complement found from 4727 to 4733\n",
      "Complement found from 4736 to 4742\n",
      "Complement found from 5660 to 5666\n",
      "Complement found from 7889 to 7895\n",
      "Complement found from 8981 to 8987\n",
      "Complement found from 9213 to 9219\n",
      "Complement found from 9369 to 9375\n",
      "Complement found from 14250 to 14256\n",
      "Complement found from 14365 to 14371\n",
      "Complement found from 14698 to 14704\n",
      "Complement found from 14742 to 14748\n",
      "Seq is: ACCAAT\n",
      "13 ACCAAT motifs detected in file\n",
      "\n",
      "Complement found from 1810 to 1816\n",
      "Complement found from 4086 to 4092\n",
      "Complement found from 5532 to 5538\n",
      "Complement found from 9405 to 9411\n",
      "Complement found from 12508 to 12514\n",
      "Complement found from 14404 to 14410\n",
      "Seq is: ACCAAG\n",
      "6 ACCAAG motifs detected in file\n",
      "\n",
      "Complement found from 423 to 429\n",
      "Complement found from 2371 to 2377\n",
      "Complement found from 2935 to 2941\n",
      "Complement found from 5579 to 5585\n",
      "Complement found from 5692 to 5698\n",
      "Complement found from 6531 to 6537\n",
      "Complement found from 7287 to 7293\n",
      "Complement found from 7841 to 7847\n",
      "Complement found from 9258 to 9264\n",
      "Complement found from 11651 to 11657\n",
      "Complement found from 12277 to 12283\n",
      "Complement found from 13627 to 13633\n",
      "Complement found from 13831 to 13837\n",
      "Complement found from 16297 to 16303\n",
      "Seq is: TAACAG\n",
      "14 TAACAG motifs detected in file\n",
      "\n",
      "Complement found from 9181 to 9187\n",
      "Complement found from 15023 to 15029\n",
      "Seq is: GCCTCT\n",
      "2 GCCTCT motifs detected in file\n",
      "\n",
      "Complement found from 2239 to 2245\n",
      "Complement found from 4891 to 4897\n",
      "Complement found from 6773 to 6779\n",
      "Complement found from 9230 to 9236\n",
      "Complement found from 9364 to 9370\n",
      "Complement found from 14520 to 14526\n",
      "Complement found from 15667 to 15673\n",
      "Seq is: CATATA\n",
      "7 CATATA motifs detected in file\n",
      "\n",
      "Complement found from 3398 to 3404\n",
      "Complement found from 8766 to 8772\n",
      "Complement found from 10132 to 10138\n",
      "Complement found from 10298 to 10304\n",
      "Complement found from 11333 to 11339\n",
      "Seq is: ACAACT\n",
      "5 ACAACT motifs detected in file\n",
      "\n",
      "Complement found from 2324 to 2330\n",
      "Complement found from 4927 to 4933\n",
      "Complement found from 6226 to 6232\n",
      "Complement found from 6345 to 6351\n",
      "Complement found from 6366 to 6372\n",
      "Complement found from 6487 to 6493\n",
      "Complement found from 11069 to 11075\n",
      "Complement found from 11915 to 11921\n",
      "Complement found from 11934 to 11940\n",
      "Complement found from 12090 to 12096\n",
      "Complement found from 14284 to 14290\n",
      "Complement found from 15455 to 15461\n",
      "Complement found from 16442 to 16448\n",
      "Seq is: TCTCCT\n",
      "13 TCTCCT motifs detected in file\n",
      "\n",
      "Complement found from 1213 to 1219\n",
      "Seq is: ATCGAT\n",
      "1 ATCGAT motifs detected in file\n",
      "\n",
      "Complement found from 2032 to 2038\n",
      "Complement found from 2480 to 2486\n",
      "Complement found from 4025 to 4031\n",
      "Complement found from 4942 to 4948\n",
      "Complement found from 4999 to 5005\n",
      "Complement found from 5503 to 5509\n",
      "Complement found from 6097 to 6103\n",
      "Complement found from 7932 to 7938\n",
      "Complement found from 8235 to 8241\n",
      "Complement found from 12722 to 12728\n",
      "Complement found from 13103 to 13109\n",
      "Seq is: AATCTT\n",
      "11 AATCTT motifs detected in file\n",
      "\n",
      "Complement found from 3362 to 3368\n",
      "Complement found from 5353 to 5359\n",
      "Complement found from 5706 to 5712\n",
      "Complement found from 6154 to 6160\n",
      "Complement found from 6466 to 6472\n",
      "Complement found from 7348 to 7354\n",
      "Complement found from 9041 to 9047\n",
      "Complement found from 9280 to 9286\n",
      "Complement found from 10851 to 10857\n",
      "Complement found from 10953 to 10959\n",
      "Complement found from 12377 to 12383\n",
      "Complement found from 12707 to 12713\n",
      "Complement found from 14135 to 14141\n",
      "Complement found from 14772 to 14778\n",
      "Complement found from 15831 to 15837\n",
      "Complement found from 15837 to 15843\n",
      "Complement found from 15855 to 15861\n",
      "Seq is: CCTAAT\n",
      "17 CCTAAT motifs detected in file\n",
      "\n",
      "Complement found from 164 to 170\n",
      "Complement found from 1994 to 2000\n",
      "Complement found from 5346 to 5352\n",
      "Complement found from 7124 to 7130\n",
      "Complement found from 7903 to 7909\n",
      "Complement found from 8465 to 8471\n",
      "Complement found from 8924 to 8930\n",
      "Complement found from 9028 to 9034\n",
      "Complement found from 10749 to 10755\n",
      "Complement found from 11879 to 11885\n",
      "Complement found from 16284 to 16290\n",
      "Seq is: ACCTAC\n",
      "11 ACCTAC motifs detected in file\n",
      "\n",
      "Complement found from 3315 to 3321\n",
      "Complement found from 7485 to 7491\n",
      "Complement found from 9867 to 9873\n",
      "Complement found from 10997 to 11003\n",
      "Complement found from 11329 to 11335\n",
      "Complement found from 12825 to 12831\n",
      "Complement found from 15520 to 15526\n",
      "Seq is: GCCAAC\n",
      "7 GCCAAC motifs detected in file\n",
      "\n",
      "Complement found from 5618 to 5624\n",
      "Complement found from 11179 to 11185\n",
      "Complement found from 11187 to 11193\n",
      "Complement found from 11761 to 11767\n",
      "Seq is: GAACGC\n",
      "4 GAACGC motifs detected in file\n",
      "\n",
      "Complement found from 2250 to 2256\n",
      "Complement found from 3326 to 3332\n",
      "Complement found from 3827 to 3833\n",
      "Complement found from 5010 to 5016\n",
      "Complement found from 5473 to 5479\n",
      "Complement found from 6232 to 6238\n",
      "Complement found from 7940 to 7946\n",
      "Complement found from 7986 to 7992\n",
      "Complement found from 8412 to 8418\n",
      "Complement found from 8783 to 8789\n",
      "Complement found from 9595 to 9601\n",
      "Complement found from 10971 to 10977\n",
      "Complement found from 11321 to 11327\n",
      "Complement found from 13985 to 13991\n",
      "Complement found from 14132 to 14138\n",
      "Complement found from 14441 to 14447\n",
      "Complement found from 14850 to 14856\n",
      "Complement found from 15723 to 15729\n",
      "Seq is: ACTCCT\n",
      "18 ACTCCT motifs detected in file\n",
      "\n",
      "Complement found from 1396 to 1402\n",
      "Complement found from 4116 to 4122\n",
      "Complement found from 8847 to 8853\n",
      "Complement found from 10447 to 10453\n",
      "Complement found from 11398 to 11404\n",
      "Seq is: TTATGA\n",
      "5 TTATGA motifs detected in file\n",
      "\n",
      "Complement found from 1701 to 1707\n",
      "Complement found from 2219 to 2225\n",
      "Complement found from 2783 to 2789\n",
      "Complement found from 4733 to 4739\n",
      "Complement found from 5105 to 5111\n",
      "Complement found from 7227 to 7233\n",
      "Complement found from 8380 to 8386\n",
      "Complement found from 8459 to 8465\n",
      "Complement found from 10126 to 10132\n",
      "Complement found from 10791 to 10797\n",
      "Complement found from 10963 to 10969\n",
      "Complement found from 13849 to 13855\n",
      "Seq is: ACTACC\n",
      "12 ACTACC motifs detected in file\n",
      "\n",
      "Complement found from 1411 to 1417\n",
      "Complement found from 11419 to 11425\n",
      "Seq is: GTCGAA\n",
      "2 GTCGAA motifs detected in file\n",
      "\n",
      "Complement found from 563 to 569\n",
      "Seq is: GACACC\n",
      "1 GACACC motifs detected in file\n",
      "\n",
      "Complement found from 1244 to 1250\n",
      "Complement found from 12792 to 12798\n",
      "Seq is: TTGCTC\n",
      "2 TTGCTC motifs detected in file\n",
      "\n",
      "Complement found from 1732 to 1738\n",
      "Complement found from 4897 to 4903\n",
      "Complement found from 5253 to 5259\n",
      "Complement found from 8626 to 8632\n",
      "Complement found from 9676 to 9682\n",
      "Complement found from 10465 to 10471\n",
      "Complement found from 10887 to 10893\n",
      "Complement found from 12323 to 12329\n",
      "Complement found from 14042 to 14048\n",
      "Complement found from 14874 to 14880\n",
      "Complement found from 15153 to 15159\n",
      "Seq is: CCAAAT\n",
      "11 CCAAAT motifs detected in file\n",
      "\n",
      "Complement found from 285 to 291\n",
      "Complement found from 992 to 998\n",
      "Complement found from 2129 to 2135\n",
      "Complement found from 2150 to 2156\n",
      "Complement found from 2226 to 2232\n",
      "Complement found from 2456 to 2462\n",
      "Complement found from 4175 to 4181\n",
      "Complement found from 4604 to 4610\n",
      "Complement found from 5281 to 5287\n",
      "Complement found from 5746 to 5752\n",
      "Complement found from 6691 to 6697\n",
      "Complement found from 7446 to 7452\n",
      "Complement found from 8496 to 8502\n",
      "Complement found from 10047 to 10053\n",
      "Complement found from 10813 to 10819\n",
      "Complement found from 11031 to 11037\n",
      "Complement found from 12417 to 12423\n",
      "Complement found from 13230 to 13236\n",
      "Complement found from 14503 to 14509\n",
      "Seq is: AAAAAA\n",
      "19 AAAAAA motifs detected in file\n",
      "\n",
      "Complement found from 2187 to 2193\n",
      "Complement found from 2269 to 2275\n",
      "Complement found from 4727 to 4733\n",
      "Complement found from 4736 to 4742\n",
      "Complement found from 5660 to 5666\n",
      "Complement found from 7889 to 7895\n",
      "Complement found from 8981 to 8987\n",
      "Complement found from 9213 to 9219\n",
      "Complement found from 9369 to 9375\n",
      "Complement found from 14250 to 14256\n",
      "Complement found from 14365 to 14371\n",
      "Complement found from 14698 to 14704\n",
      "Complement found from 14742 to 14748\n",
      "Seq is: ACCAAT\n",
      "13 ACCAAT motifs detected in file\n",
      "\n",
      "Complement found from 1656 to 1662\n",
      "Complement found from 2905 to 2911\n",
      "Complement found from 7990 to 7996\n",
      "Complement found from 16141 to 16147\n",
      "Complement found from 16394 to 16400\n",
      "Seq is: CTTGAC\n",
      "5 CTTGAC motifs detected in file\n",
      "\n",
      "Complement found from 4321 to 4327\n",
      "Complement found from 6008 to 6014\n",
      "Complement found from 9695 to 9701\n",
      "Complement found from 10243 to 10249\n",
      "Complement found from 10333 to 10339\n",
      "Complement found from 12196 to 12202\n",
      "Seq is: CTTATT\n",
      "6 CTTATT motifs detected in file\n",
      "\n",
      "Complement found from 9441 to 9447\n",
      "Seq is: TTCGAT\n",
      "1 TTCGAT motifs detected in file\n",
      "\n",
      "Complement found from 1474 to 1480\n",
      "Complement found from 5052 to 5058\n",
      "Complement found from 10735 to 10741\n",
      "Complement found from 16094 to 16100\n",
      "Complement found from 16327 to 16333\n",
      "Seq is: CGTACA\n",
      "5 CGTACA motifs detected in file\n",
      "\n",
      "Complement found from 835 to 841\n",
      "Complement found from 965 to 971\n",
      "Complement found from 1735 to 1741\n",
      "Complement found from 4004 to 4010\n",
      "Complement found from 4570 to 4576\n",
      "Complement found from 4609 to 4615\n",
      "Complement found from 4777 to 4783\n",
      "Complement found from 5404 to 5410\n",
      "Complement found from 8493 to 8499\n",
      "Complement found from 10057 to 10063\n",
      "Complement found from 12326 to 12332\n",
      "Complement found from 13871 to 13877\n",
      "Complement found from 14495 to 14501\n",
      "Complement found from 14775 to 14781\n",
      "Seq is: AATAAA\n",
      "14 AATAAA motifs detected in file\n",
      "\n",
      "Complement found from 6459 to 6465\n",
      "Complement found from 15590 to 15596\n",
      "Seq is: GATCCG\n",
      "2 GATCCG motifs detected in file\n",
      "\n",
      "Complement found from 9978 to 9984\n",
      "Complement found from 11162 to 11168\n",
      "Seq is: GATGAG\n",
      "2 GATGAG motifs detected in file\n",
      "\n",
      "Complement found from 7137 to 7143\n",
      "Complement found from 12286 to 12292\n",
      "Complement found from 12447 to 12453\n",
      "Seq is: TCCATT\n",
      "3 TCCATT motifs detected in file\n",
      "\n",
      "Complement found from 945 to 951\n",
      "Complement found from 9994 to 10000\n",
      "Seq is: TTTTAG\n",
      "2 TTTTAG motifs detected in file\n",
      "\n",
      "Complement found from 4650 to 4656\n",
      "Complement found from 7659 to 7665\n",
      "Complement found from 14991 to 14997\n",
      "Seq is: TCACGC\n",
      "3 TCACGC motifs detected in file\n",
      "\n",
      "Complement found from 3943 to 3949\n",
      "Complement found from 5788 to 5794\n",
      "Complement found from 13607 to 13613\n",
      "Seq is: TCGAAT\n",
      "3 TCGAAT motifs detected in file\n",
      "\n",
      "Seq is: CGCGAC\n",
      "0 CGCGAC motifs detected in file\n",
      "\n",
      "Complement found from 471 to 477\n",
      "Complement found from 2879 to 2885\n",
      "Complement found from 5102 to 5108\n",
      "Complement found from 5119 to 5125\n",
      "Complement found from 5152 to 5158\n",
      "Complement found from 5377 to 5383\n",
      "Complement found from 6306 to 6312\n",
      "Complement found from 6526 to 6532\n",
      "Complement found from 6681 to 6687\n",
      "Complement found from 10102 to 10108\n",
      "Complement found from 11289 to 11295\n",
      "Complement found from 12348 to 12354\n",
      "Complement found from 12583 to 12589\n",
      "Complement found from 12971 to 12977\n",
      "Complement found from 13418 to 13424\n",
      "Complement found from 14209 to 14215\n",
      "Complement found from 14906 to 14912\n",
      "Seq is: ACTACT\n",
      "17 ACTACT motifs detected in file\n",
      "\n",
      "Complement found from 903 to 909\n",
      "Complement found from 6802 to 6808\n",
      "Complement found from 12048 to 12054\n",
      "Complement found from 15270 to 15276\n",
      "Complement found from 16537 to 16543\n",
      "Seq is: CACACG\n",
      "5 CACACG motifs detected in file\n",
      "\n",
      "Complement found from 1319 to 1325\n",
      "Complement found from 7215 to 7221\n",
      "Complement found from 7993 to 7999\n",
      "Seq is: GACGTT\n",
      "3 GACGTT motifs detected in file\n",
      "\n",
      "Complement found from 1370 to 1376\n",
      "Complement found from 3501 to 3507\n",
      "Complement found from 4499 to 4505\n",
      "Complement found from 5047 to 5053\n",
      "Complement found from 5343 to 5349\n",
      "Complement found from 6285 to 6291\n",
      "Complement found from 8276 to 8282\n",
      "Complement found from 9184 to 9190\n",
      "Complement found from 11039 to 11045\n",
      "Complement found from 13786 to 13792\n",
      "Complement found from 13918 to 13924\n",
      "Seq is: TCTACC\n",
      "11 TCTACC motifs detected in file\n",
      "\n",
      "Complement found from 478 to 484\n",
      "Complement found from 3184 to 3190\n",
      "Complement found from 3723 to 3729\n",
      "Complement found from 4883 to 4889\n",
      "Complement found from 7747 to 7753\n",
      "Complement found from 8632 to 8638\n",
      "Complement found from 10508 to 10514\n",
      "Complement found from 14164 to 14170\n",
      "Seq is: ATCTCA\n",
      "8 ATCTCA motifs detected in file\n",
      "\n",
      "Complement found from 3554 to 3560\n",
      "Complement found from 3752 to 3758\n",
      "Complement found from 4494 to 4500\n",
      "Complement found from 5337 to 5343\n",
      "Complement found from 5358 to 5364\n",
      "Complement found from 5725 to 5731\n",
      "Complement found from 9116 to 9122\n",
      "Complement found from 10872 to 10878\n",
      "Complement found from 11286 to 11292\n",
      "Complement found from 11808 to 11814\n",
      "Complement found from 12695 to 12701\n",
      "Complement found from 13501 to 13507\n",
      "Complement found from 15068 to 15074\n",
      "Seq is: TCTACT\n",
      "13 TCTACT motifs detected in file\n",
      "\n",
      "Complement found from 2376 to 2382\n",
      "Complement found from 3714 to 3720\n",
      "Complement found from 4481 to 4487\n",
      "Complement found from 5251 to 5257\n",
      "Complement found from 11305 to 11311\n",
      "Complement found from 13008 to 13014\n",
      "Seq is: GCCCAA\n",
      "6 GCCCAA motifs detected in file\n",
      "\n",
      "Complement found from 3244 to 3250\n",
      "Seq is: CCCGGT\n",
      "1 CCCGGT motifs detected in file\n",
      "\n",
      "Complement found from 1530 to 1536\n",
      "Complement found from 1836 to 1842\n",
      "Complement found from 3428 to 3434\n",
      "Complement found from 3483 to 3489\n",
      "Complement found from 6152 to 6158\n",
      "Complement found from 7094 to 7100\n",
      "Complement found from 7625 to 7631\n",
      "Complement found from 8227 to 8233\n",
      "Complement found from 9254 to 9260\n",
      "Complement found from 9525 to 9531\n",
      "Complement found from 9581 to 9587\n",
      "Complement found from 10277 to 10283\n",
      "Complement found from 10936 to 10942\n",
      "Complement found from 11233 to 11239\n",
      "Complement found from 13128 to 13134\n",
      "Complement found from 13946 to 13952\n",
      "Complement found from 13980 to 13986\n",
      "Complement found from 14489 to 14495\n",
      "Complement found from 14770 to 14776\n",
      "Complement found from 15367 to 15373\n",
      "Seq is: CCCCTA\n",
      "20 CCCCTA motifs detected in file\n",
      "\n",
      "Complement found from 210 to 216\n",
      "Complement found from 2744 to 2750\n",
      "Complement found from 4469 to 4475\n",
      "Complement found from 4753 to 4759\n",
      "Complement found from 7300 to 7306\n",
      "Complement found from 8220 to 8226\n",
      "Complement found from 10336 to 10342\n",
      "Complement found from 12675 to 12681\n",
      "Complement found from 14781 to 14787\n",
      "Seq is: ATTAAT\n",
      "9 ATTAAT motifs detected in file\n",
      "\n",
      "Complement found from 8202 to 8208\n",
      "Complement found from 9221 to 9227\n",
      "Complement found from 12232 to 12238\n",
      "Complement found from 14428 to 14434\n",
      "Seq is: CATGCC\n",
      "4 CATGCC motifs detected in file\n",
      "\n",
      "Complement found from 3164 to 3170\n",
      "Complement found from 4457 to 4463\n",
      "Complement found from 7177 to 7183\n",
      "Complement found from 7622 to 7628\n",
      "Complement found from 7951 to 7957\n",
      "Complement found from 9081 to 9087\n",
      "Complement found from 11230 to 11236\n",
      "Complement found from 12373 to 12379\n",
      "Complement found from 12479 to 12485\n",
      "Complement found from 13643 to 13649\n",
      "Complement found from 14111 to 14117\n",
      "Seq is: CTTCCC\n",
      "11 CTTCCC motifs detected in file\n",
      "\n",
      "Complement found from 6669 to 6675\n",
      "Complement found from 10638 to 10644\n",
      "Complement found from 16119 to 16125\n",
      "Seq is: ATATTG\n",
      "3 ATATTG motifs detected in file\n",
      "\n",
      "Complement found from 987 to 993\n",
      "Complement found from 3420 to 3426\n",
      "Complement found from 7019 to 7025\n",
      "Complement found from 13092 to 13098\n",
      "Complement found from 14720 to 14726\n",
      "Seq is: GTTGTA\n",
      "5 GTTGTA motifs detected in file\n",
      "\n",
      "Complement found from 5545 to 5551\n",
      "Complement found from 8481 to 8487\n",
      "Complement found from 11410 to 11416\n",
      "Complement found from 13341 to 13347\n",
      "Complement found from 14239 to 14245\n",
      "Complement found from 16251 to 16257\n",
      "Complement found from 16315 to 16321\n",
      "Complement found from 16522 to 16528\n",
      "Seq is: AAAGCC\n",
      "8 AAAGCC motifs detected in file\n",
      "\n",
      "Complement found from 467 to 473\n",
      "Complement found from 535 to 541\n",
      "Complement found from 3574 to 3580\n",
      "Complement found from 4222 to 4228\n",
      "Complement found from 4430 to 4436\n",
      "Complement found from 8408 to 8414\n",
      "Complement found from 8701 to 8707\n",
      "Complement found from 8941 to 8947\n",
      "Complement found from 10653 to 10659\n",
      "Complement found from 12715 to 12721\n",
      "Complement found from 13345 to 13351\n",
      "Complement found from 13429 to 13435\n",
      "Complement found from 15205 to 15211\n",
      "Seq is: CCATAC\n",
      "13 CCATAC motifs detected in file\n",
      "\n",
      "Complement found from 16452 to 16458\n",
      "Seq is: CCGGGC\n",
      "1 CCGGGC motifs detected in file\n",
      "\n",
      "Complement found from 5266 to 5272\n",
      "Complement found from 8954 to 8960\n",
      "Seq is: TATCGA\n",
      "2 TATCGA motifs detected in file\n",
      "\n",
      "Complement found from 10001 to 10007\n",
      "Complement found from 10484 to 10490\n",
      "Complement found from 14292 to 14298\n",
      "Complement found from 14496 to 14502\n",
      "Complement found from 14592 to 14598\n",
      "Complement found from 16134 to 16140\n",
      "Seq is: ATAAAT\n",
      "6 ATAAAT motifs detected in file\n",
      "\n",
      "Seq is: CGACTG\n",
      "0 CGACTG motifs detected in file\n",
      "\n",
      "Complement found from 10426 to 10432\n",
      "Complement found from 15556 to 15562\n",
      "Seq is: GAATGA\n",
      "2 GAATGA motifs detected in file\n",
      "\n",
      "Complement found from 2944 to 2950\n",
      "Complement found from 3650 to 3656\n",
      "Complement found from 4672 to 4678\n",
      "Complement found from 7569 to 7575\n",
      "Complement found from 8564 to 8570\n",
      "Complement found from 9454 to 9460\n",
      "Complement found from 11784 to 11790\n",
      "Complement found from 12848 to 12854\n",
      "Complement found from 14368 to 14374\n",
      "Complement found from 15828 to 15834\n",
      "Complement found from 15834 to 15840\n",
      "Seq is: AATCCT\n",
      "11 AATCCT motifs detected in file\n",
      "\n",
      "Complement found from 351 to 357\n",
      "Complement found from 532 to 538\n",
      "Complement found from 555 to 561\n",
      "Complement found from 635 to 641\n",
      "Complement found from 865 to 871\n",
      "Complement found from 1081 to 1087\n",
      "Complement found from 1373 to 1379\n",
      "Complement found from 4389 to 4395\n",
      "Complement found from 5435 to 5441\n",
      "Complement found from 5599 to 5605\n",
      "Complement found from 6582 to 6588\n",
      "Complement found from 7489 to 7495\n",
      "Complement found from 12966 to 12972\n",
      "Complement found from 13056 to 13062\n",
      "Complement found from 13675 to 13681\n",
      "Complement found from 13690 to 13696\n",
      "Complement found from 14338 to 14344\n",
      "Complement found from 14387 to 14393\n",
      "Complement found from 14618 to 14624\n",
      "Complement found from 14627 to 14633\n",
      "Complement found from 14749 to 14755\n",
      "Complement found from 14812 to 14818\n",
      "Seq is: ACCCCA\n",
      "22 ACCCCA motifs detected in file\n",
      "\n",
      "Complement found from 1085 to 1091\n",
      "Complement found from 5952 to 5958\n",
      "Complement found from 6520 to 6526\n",
      "Complement found from 7034 to 7040\n",
      "Complement found from 7144 to 7150\n",
      "Complement found from 8420 to 8426\n",
      "Complement found from 9850 to 9856\n",
      "Complement found from 11021 to 11027\n",
      "Complement found from 11871 to 11877\n",
      "Complement found from 13085 to 13091\n",
      "Complement found from 13165 to 13171\n",
      "Complement found from 13882 to 13888\n",
      "Complement found from 14312 to 14318\n",
      "Complement found from 15473 to 15479\n",
      "Seq is: CACTAT\n",
      "14 CACTAT motifs detected in file\n",
      "\n",
      "Complement found from 893 to 899\n",
      "Complement found from 6846 to 6852\n",
      "Seq is: CCACCG\n",
      "2 CCACCG motifs detected in file\n",
      "\n",
      "Complement found from 141 to 147\n",
      "Complement found from 8788 to 8794\n",
      "Complement found from 14430 to 14436\n",
      "Complement found from 15022 to 15028\n",
      "Seq is: TGCCTC\n",
      "4 TGCCTC motifs detected in file\n",
      "\n",
      "Complement found from 1156 to 1162\n",
      "Complement found from 2350 to 2356\n",
      "Complement found from 2454 to 2460\n",
      "Complement found from 2798 to 2804\n",
      "Complement found from 3263 to 3269\n",
      "Complement found from 8092 to 8098\n",
      "Complement found from 8887 to 8893\n",
      "Complement found from 11464 to 11470\n",
      "Complement found from 13867 to 13873\n",
      "Complement found from 14501 to 14507\n",
      "Seq is: TTAAAA\n",
      "10 TTAAAA motifs detected in file\n",
      "\n",
      "Complement found from 1921 to 1927\n",
      "Complement found from 4258 to 4264\n",
      "Complement found from 10354 to 10360\n",
      "Complement found from 12566 to 12572\n",
      "Seq is: CCTAAG\n",
      "4 CCTAAG motifs detected in file\n",
      "\n",
      "Complement found from 4810 to 4816\n",
      "Complement found from 9589 to 9595\n",
      "Complement found from 15259 to 15265\n",
      "Seq is: AGTCCC\n",
      "3 AGTCCC motifs detected in file\n",
      "\n",
      "Complement found from 506 to 512\n",
      "Complement found from 1307 to 1313\n",
      "Complement found from 1729 to 1735\n",
      "Complement found from 3201 to 3207\n",
      "Complement found from 3307 to 3313\n",
      "Complement found from 3337 to 3343\n",
      "Complement found from 3577 to 3583\n",
      "Complement found from 4225 to 4231\n",
      "Complement found from 4822 to 4828\n",
      "Complement found from 5020 to 5026\n",
      "Complement found from 6114 to 6120\n",
      "Complement found from 13325 to 13331\n",
      "Complement found from 16287 to 16293\n",
      "Seq is: TACCCA\n",
      "13 TACCCA motifs detected in file\n",
      "\n",
      "Complement found from 970 to 976\n",
      "Complement found from 1055 to 1061\n",
      "Complement found from 1669 to 1675\n",
      "Complement found from 1892 to 1898\n",
      "Complement found from 1930 to 1936\n",
      "Complement found from 4410 to 4416\n",
      "Complement found from 5175 to 5181\n",
      "Complement found from 5583 to 5589\n",
      "Complement found from 5643 to 5649\n",
      "Complement found from 5696 to 5702\n",
      "Complement found from 8302 to 8308\n",
      "Complement found from 15993 to 15999\n",
      "Complement found from 16475 to 16481\n",
      "Seq is: AGCTAA\n",
      "13 AGCTAA motifs detected in file\n",
      "\n",
      "Complement found from 9627 to 9633\n",
      "Seq is: GAGTAT\n",
      "1 GAGTAT motifs detected in file\n",
      "\n",
      "Complement found from 9489 to 9495\n",
      "Complement found from 13726 to 13732\n",
      "Complement found from 14435 to 14441\n",
      "Seq is: CAGGAT\n",
      "3 CAGGAT motifs detected in file\n",
      "\n",
      "Complement found from 12464 to 12470\n",
      "Complement found from 15015 to 15021\n",
      "Complement found from 15715 to 15721\n",
      "Seq is: CTTTAT\n",
      "3 CTTTAT motifs detected in file\n",
      "\n",
      "Complement found from 309 to 315\n",
      "Complement found from 3166 to 3172\n",
      "Complement found from 4247 to 4253\n",
      "Complement found from 7953 to 7959\n",
      "Complement found from 8603 to 8609\n",
      "Complement found from 10190 to 10196\n",
      "Complement found from 12082 to 12088\n",
      "Complement found from 12383 to 12389\n",
      "Complement found from 13752 to 13758\n",
      "Complement found from 13761 to 13767\n",
      "Complement found from 13780 to 13786\n",
      "Complement found from 14153 to 14159\n",
      "Complement found from 14487 to 14493\n",
      "Complement found from 14529 to 14535\n",
      "Seq is: TCCCCC\n",
      "14 TCCCCC motifs detected in file\n",
      "\n",
      "Complement found from 3110 to 3116\n",
      "Complement found from 3382 to 3388\n",
      "Complement found from 4366 to 4372\n",
      "Complement found from 10217 to 10223\n",
      "Complement found from 14536 to 14542\n",
      "Seq is: AAATTC\n",
      "5 AAATTC motifs detected in file\n",
      "\n",
      "Seq is: GAGGAT\n",
      "0 GAGGAT motifs detected in file\n",
      "\n",
      "Complement found from 835 to 841\n",
      "Complement found from 965 to 971\n",
      "Complement found from 1735 to 1741\n",
      "Complement found from 4004 to 4010\n",
      "Complement found from 4570 to 4576\n",
      "Complement found from 4609 to 4615\n",
      "Complement found from 4777 to 4783\n",
      "Complement found from 5404 to 5410\n",
      "Complement found from 8493 to 8499\n",
      "Complement found from 10057 to 10063\n",
      "Complement found from 12326 to 12332\n",
      "Complement found from 13871 to 13877\n",
      "Complement found from 14495 to 14501\n",
      "Complement found from 14775 to 14781\n",
      "Seq is: AATAAA\n",
      "14 AATAAA motifs detected in file\n",
      "\n",
      "Seq is: TGAGAT\n",
      "0 TGAGAT motifs detected in file\n",
      "\n",
      "Complement found from 8197 to 8203\n",
      "Seq is: AGTTTC\n",
      "1 AGTTTC motifs detected in file\n",
      "\n",
      "Seq is: ATTGAG\n",
      "0 ATTGAG motifs detected in file\n",
      "\n",
      "Complement found from 1011 to 1017\n",
      "Complement found from 1959 to 1965\n",
      "Complement found from 4607 to 4613\n",
      "Complement found from 5402 to 5408\n",
      "Complement found from 8443 to 8449\n",
      "Complement found from 8491 to 8497\n",
      "Complement found from 13138 to 13144\n",
      "Complement found from 13407 to 13413\n",
      "Complement found from 13669 to 13675\n",
      "Complement found from 13869 to 13875\n",
      "Complement found from 15868 to 15874\n",
      "Seq is: AAAATA\n",
      "11 AAAATA motifs detected in file\n",
      "\n",
      "Seq is: GTCTGT\n",
      "0 GTCTGT motifs detected in file\n",
      "\n",
      "Complement found from 966 to 972\n",
      "Complement found from 1736 to 1742\n",
      "Complement found from 16313 to 16319\n",
      "Complement found from 16520 to 16526\n",
      "Seq is: ATAAAG\n",
      "4 ATAAAG motifs detected in file\n",
      "\n",
      "Complement found from 3083 to 3089\n",
      "Complement found from 4360 to 4366\n",
      "Complement found from 7135 to 7141\n",
      "Complement found from 9149 to 9155\n",
      "Complement found from 10155 to 10161\n",
      "Complement found from 12445 to 12451\n",
      "Complement found from 12954 to 12960\n",
      "Complement found from 13151 to 13157\n",
      "Complement found from 16169 to 16175\n",
      "Seq is: AATCCA\n",
      "9 AATCCA motifs detected in file\n",
      "\n",
      "Complement found from 996 to 1002\n",
      "Complement found from 2249 to 2255\n",
      "Complement found from 5132 to 5138\n",
      "Complement found from 7939 to 7945\n",
      "Complement found from 11320 to 11326\n",
      "Complement found from 12319 to 12325\n",
      "Complement found from 15972 to 15978\n",
      "Complement found from 16245 to 16251\n",
      "Seq is: AACTCC\n",
      "8 AACTCC motifs detected in file\n",
      "\n",
      "Complement found from 1393 to 1399\n",
      "Complement found from 3796 to 3802\n",
      "Complement found from 4319 to 4325\n",
      "Complement found from 5192 to 5198\n",
      "Complement found from 5460 to 5466\n",
      "Complement found from 5650 to 5656\n",
      "Complement found from 6293 to 6299\n",
      "Complement found from 7865 to 7871\n",
      "Complement found from 8844 to 8850\n",
      "Complement found from 8932 to 8938\n",
      "Complement found from 10162 to 10168\n",
      "Complement found from 12194 to 12200\n",
      "Complement found from 13210 to 13216\n",
      "Complement found from 13651 to 13657\n",
      "Complement found from 15408 to 15414\n",
      "Complement found from 15526 to 15532\n",
      "Complement found from 16293 to 16299\n",
      "Complement found from 16546 to 16552\n",
      "Seq is: CCCTTA\n",
      "18 CCCTTA motifs detected in file\n",
      "\n",
      "Complement found from 2817 to 2823\n",
      "Complement found from 2981 to 2987\n",
      "Complement found from 3529 to 3535\n",
      "Complement found from 3899 to 3905\n",
      "Complement found from 7977 to 7983\n",
      "Complement found from 14802 to 14808\n",
      "Seq is: CGACCT\n",
      "6 CGACCT motifs detected in file\n",
      "\n",
      "Complement found from 370 to 376\n",
      "Complement found from 440 to 446\n",
      "Complement found from 2761 to 2767\n",
      "Complement found from 5061 to 5067\n",
      "Complement found from 5177 to 5183\n",
      "Complement found from 5186 to 5192\n",
      "Complement found from 5391 to 5397\n",
      "Complement found from 6530 to 6536\n",
      "Complement found from 7286 to 7292\n",
      "Complement found from 7714 to 7720\n",
      "Complement found from 7742 to 7748\n",
      "Complement found from 9010 to 9016\n",
      "Complement found from 9257 to 9263\n",
      "Complement found from 10939 to 10945\n",
      "Complement found from 12245 to 12251\n",
      "Complement found from 12412 to 12418\n",
      "Complement found from 12736 to 12742\n",
      "Complement found from 13161 to 13167\n",
      "Complement found from 13626 to 13632\n",
      "Complement found from 13657 to 13663\n",
      "Complement found from 13741 to 13747\n",
      "Complement found from 13830 to 13836\n",
      "Complement found from 14569 to 14575\n",
      "Complement found from 15599 to 15605\n",
      "Seq is: CTAACA\n",
      "24 CTAACA motifs detected in file\n",
      "\n",
      "Complement found from 1728 to 1734\n",
      "Complement found from 2409 to 2415\n",
      "Complement found from 2484 to 2490\n",
      "Complement found from 2923 to 2929\n",
      "Complement found from 4821 to 4827\n",
      "Complement found from 5019 to 5025\n",
      "Complement found from 8258 to 8264\n",
      "Complement found from 8402 to 8408\n",
      "Complement found from 9723 to 9729\n",
      "Complement found from 10274 to 10280\n",
      "Complement found from 11863 to 11869\n",
      "Seq is: TTACCC\n",
      "11 TTACCC motifs detected in file\n",
      "\n",
      "Complement found from 1841 to 1847\n",
      "Complement found from 5956 to 5962\n",
      "Complement found from 11377 to 11383\n",
      "Complement found from 13431 to 13437\n",
      "Complement found from 13482 to 13488\n",
      "Seq is: ATACCT\n",
      "5 ATACCT motifs detected in file\n",
      "\n",
      "Complement found from 2624 to 2630\n",
      "Complement found from 7056 to 7062\n",
      "Complement found from 7699 to 7705\n",
      "Complement found from 9959 to 9965\n",
      "Complement found from 16487 to 16493\n",
      "Seq is: CTGTAT\n",
      "5 CTGTAT motifs detected in file\n",
      "\n",
      "Complement found from 1501 to 1507\n",
      "Complement found from 1740 to 1746\n",
      "Complement found from 2302 to 2308\n",
      "Complement found from 9998 to 10004\n",
      "Complement found from 10528 to 10534\n",
      "Complement found from 14463 to 14469\n",
      "Complement found from 15895 to 15901\n",
      "Seq is: AGTATA\n",
      "7 AGTATA motifs detected in file\n",
      "\n",
      "Complement found from 7079 to 7085\n",
      "Complement found from 7277 to 7283\n",
      "Complement found from 8546 to 8552\n",
      "Complement found from 10589 to 10595\n",
      "Complement found from 15304 to 15310\n",
      "Seq is: TTCATT\n",
      "5 TTCATT motifs detected in file\n",
      "\n",
      "Complement found from 737 to 743\n",
      "Complement found from 7856 to 7862\n",
      "Seq is: ACGATC\n",
      "2 ACGATC motifs detected in file\n",
      "\n",
      "Complement found from 25 to 31\n",
      "Complement found from 461 to 467\n",
      "Complement found from 1689 to 1695\n",
      "Complement found from 4188 to 4194\n",
      "Complement found from 5602 to 5608\n",
      "Complement found from 9316 to 9322\n",
      "Complement found from 9511 to 9517\n",
      "Complement found from 9593 to 9599\n",
      "Complement found from 10621 to 10627\n",
      "Complement found from 13076 to 13082\n",
      "Complement found from 13186 to 13192\n",
      "Complement found from 14115 to 14121\n",
      "Complement found from 14790 to 14796\n",
      "Seq is: CCACTC\n",
      "13 CCACTC motifs detected in file\n",
      "\n",
      "Complement found from 3504 to 3510\n",
      "Complement found from 3543 to 3549\n",
      "Complement found from 4502 to 4508\n",
      "Complement found from 5306 to 5312\n",
      "Complement found from 5318 to 5324\n",
      "Complement found from 6338 to 6344\n",
      "Complement found from 7870 to 7876\n",
      "Complement found from 8961 to 8967\n",
      "Complement found from 10505 to 10511\n",
      "Complement found from 14479 to 14485\n",
      "Complement found from 14714 to 14720\n",
      "Complement found from 15787 to 15793\n",
      "Complement found from 16401 to 16407\n",
      "Seq is: ACCATC\n",
      "13 ACCATC motifs detected in file\n",
      "\n",
      "Complement found from 1507 to 1513\n",
      "Complement found from 3935 to 3941\n",
      "Complement found from 4693 to 4699\n",
      "Complement found from 5541 to 5547\n",
      "Complement found from 5719 to 5725\n",
      "Complement found from 7935 to 7941\n",
      "Complement found from 11624 to 11630\n",
      "Complement found from 11800 to 11806\n",
      "Complement found from 12572 to 12578\n",
      "Complement found from 12686 to 12692\n",
      "Complement found from 13252 to 13258\n",
      "Complement found from 13337 to 13343\n",
      "Complement found from 13441 to 13447\n",
      "Seq is: CTTCAA\n",
      "13 CTTCAA motifs detected in file\n",
      "\n",
      "Complement found from 2459 to 2465\n",
      "Complement found from 7342 to 7348\n",
      "Complement found from 12329 to 12335\n",
      "Complement found from 15962 to 15968\n",
      "Seq is: AAAAGT\n",
      "4 AAAAGT motifs detected in file\n",
      "\n",
      "Complement found from 3651 to 3657\n",
      "Complement found from 4688 to 4694\n",
      "Complement found from 10549 to 10555\n",
      "Complement found from 11724 to 11730\n",
      "Complement found from 11785 to 11791\n",
      "Complement found from 12882 to 12888\n",
      "Complement found from 14259 to 14265\n",
      "Complement found from 14869 to 14875\n",
      "Complement found from 15097 to 15103\n",
      "Complement found from 15637 to 15643\n",
      "Complement found from 15661 to 15667\n",
      "Complement found from 16404 to 16410\n",
      "Seq is: ATCCTC\n",
      "12 ATCCTC motifs detected in file\n",
      "\n",
      "Complement found from 5024 to 5030\n",
      "Complement found from 9200 to 9206\n",
      "Complement found from 9400 to 9406\n",
      "Complement found from 10717 to 10723\n",
      "Complement found from 10819 to 10825\n",
      "Complement found from 11196 to 11202\n",
      "Complement found from 11524 to 11530\n",
      "Complement found from 11635 to 11641\n",
      "Complement found from 14141 to 14147\n",
      "Seq is: CACATA\n",
      "9 CACATA motifs detected in file\n",
      "\n",
      "Complement found from 1105 to 1111\n",
      "Complement found from 2209 to 2215\n",
      "Complement found from 3186 to 3192\n",
      "Complement found from 3597 to 3603\n",
      "Complement found from 5123 to 5129\n",
      "Complement found from 6545 to 6551\n",
      "Complement found from 9787 to 9793\n",
      "Complement found from 9838 to 9844\n",
      "Complement found from 10136 to 10142\n",
      "Complement found from 10613 to 10619\n",
      "Complement found from 11506 to 11512\n",
      "Complement found from 11950 to 11956\n",
      "Complement found from 12103 to 12109\n",
      "Complement found from 12261 to 12267\n",
      "Complement found from 13845 to 13851\n",
      "Complement found from 14066 to 14072\n",
      "Complement found from 14412 to 14418\n",
      "Complement found from 14647 to 14653\n",
      "Complement found from 14922 to 14928\n",
      "Complement found from 16222 to 16228\n",
      "Seq is: CTCAAC\n",
      "20 CTCAAC motifs detected in file\n",
      "\n",
      "Complement found from 4129 to 4135\n",
      "Complement found from 4241 to 4247\n",
      "Seq is: CAGCAT\n",
      "2 CAGCAT motifs detected in file\n",
      "\n",
      "Complement found from 3161 to 3167\n",
      "Complement found from 9143 to 9149\n",
      "Complement found from 10066 to 10072\n",
      "Complement found from 11859 to 11865\n",
      "Complement found from 12887 to 12893\n",
      "Complement found from 13331 to 13337\n",
      "Complement found from 14928 to 14934\n",
      "Seq is: CGCCTT\n",
      "7 CGCCTT motifs detected in file\n",
      "\n",
      "Complement found from 1679 to 1685\n",
      "Complement found from 3481 to 3487\n",
      "Complement found from 4792 to 4798\n",
      "Complement found from 4876 to 4882\n",
      "Complement found from 5761 to 5767\n",
      "Complement found from 8025 to 8031\n",
      "Complement found from 9523 to 9529\n",
      "Complement found from 11424 to 11430\n",
      "Complement found from 14241 to 14247\n",
      "Seq is: AGCCCC\n",
      "9 AGCCCC motifs detected in file\n",
      "\n",
      "Seq is: CATACG\n",
      "0 CATACG motifs detected in file\n",
      "\n",
      "Complement found from 750 to 756\n",
      "Complement found from 5172 to 5178\n",
      "Complement found from 9733 to 9739\n",
      "Complement found from 11572 to 11578\n",
      "Complement found from 13590 to 13596\n",
      "Complement found from 16199 to 16205\n",
      "Seq is: ACAAGC\n",
      "6 ACAAGC motifs detected in file\n",
      "\n",
      "Complement found from 8323 to 8329\n",
      "Seq is: TTAAGT\n",
      "1 TTAAGT motifs detected in file\n",
      "\n",
      "Complement found from 3971 to 3977\n",
      "Complement found from 5072 to 5078\n",
      "Complement found from 6635 to 6641\n",
      "Complement found from 10219 to 10225\n",
      "Complement found from 13614 to 13620\n",
      "Complement found from 15012 to 15018\n",
      "Complement found from 15276 to 15282\n",
      "Complement found from 15339 to 15345\n",
      "Seq is: ATTCTT\n",
      "8 ATTCTT motifs detected in file\n",
      "\n",
      "Complement found from 10202 to 10208\n",
      "Complement found from 15595 to 15601\n",
      "Complement found from 16390 to 16396\n",
      "Seq is: GTCCCT\n",
      "3 GTCCCT motifs detected in file\n",
      "\n",
      "Complement found from 2270 to 2276\n",
      "Complement found from 4737 to 4743\n",
      "Complement found from 9214 to 9220\n",
      "Complement found from 14366 to 14372\n",
      "Complement found from 15708 to 15714\n",
      "Complement found from 16167 to 16173\n",
      "Seq is: CCAATC\n",
      "6 CCAATC motifs detected in file\n",
      "\n",
      "Complement found from 6392 to 6398\n",
      "Complement found from 6932 to 6938\n",
      "Complement found from 7154 to 7160\n",
      "Complement found from 9860 to 9866\n",
      "Complement found from 12600 to 12606\n",
      "Complement found from 12756 to 12762\n",
      "Complement found from 12879 to 12885\n",
      "Complement found from 14797 to 14803\n",
      "Complement found from 14934 to 14940\n",
      "Complement found from 15292 to 15298\n",
      "Seq is: TTCATC\n",
      "10 TTCATC motifs detected in file\n",
      "\n",
      "Complement found from 1134 to 1140\n",
      "Complement found from 1376 to 1382\n",
      "Complement found from 11176 to 11182\n",
      "Seq is: CCAGAA\n",
      "3 CCAGAA motifs detected in file\n",
      "\n",
      "Complement found from 5897 to 5903\n",
      "Complement found from 8293 to 8299\n",
      "Complement found from 10795 to 10801\n",
      "Seq is: CCACTG\n",
      "3 CCACTG motifs detected in file\n",
      "\n",
      "Complement found from 397 to 403\n",
      "Complement found from 4585 to 4591\n",
      "Complement found from 5491 to 5497\n",
      "Complement found from 8756 to 8762\n",
      "Complement found from 11110 to 11116\n",
      "Complement found from 11364 to 11370\n",
      "Complement found from 13893 to 13899\n",
      "Seq is: TTTTAT\n",
      "7 TTTTAT motifs detected in file\n",
      "\n",
      "Complement found from 3301 to 3307\n",
      "Complement found from 3804 to 3810\n",
      "Complement found from 4038 to 4044\n",
      "Complement found from 4071 to 4077\n",
      "Complement found from 6195 to 6201\n",
      "Complement found from 6398 to 6404\n",
      "Complement found from 7183 to 7189\n",
      "Complement found from 7723 to 7729\n",
      "Complement found from 8707 to 8713\n",
      "Complement found from 9196 to 9202\n",
      "Complement found from 10038 to 10044\n",
      "Complement found from 10895 to 10901\n",
      "Complement found from 11261 to 11267\n",
      "Complement found from 11995 to 12001\n",
      "Complement found from 12029 to 12035\n",
      "Complement found from 12176 to 12182\n",
      "Complement found from 12248 to 12254\n",
      "Complement found from 12486 to 12492\n",
      "Complement found from 12929 to 12935\n",
      "Complement found from 13744 to 13750\n",
      "Complement found from 13773 to 13779\n",
      "Complement found from 15678 to 15684\n",
      "Complement found from 15823 to 15829\n",
      "Seq is: ACAACA\n",
      "23 ACAACA motifs detected in file\n",
      "\n",
      "Complement found from 339 to 345\n",
      "Complement found from 3183 to 3189\n",
      "Complement found from 4882 to 4888\n",
      "Complement found from 7746 to 7752\n",
      "Complement found from 10507 to 10513\n",
      "Complement found from 14823 to 14829\n",
      "Seq is: CATCTC\n",
      "6 CATCTC motifs detected in file\n",
      "\n",
      "Complement found from 3051 to 3057\n",
      "Complement found from 7344 to 7350\n",
      "Complement found from 9588 to 9594\n",
      "Seq is: AAGTCC\n",
      "3 AAGTCC motifs detected in file\n",
      "\n",
      "Complement found from 3788 to 3794\n",
      "Complement found from 5212 to 5218\n",
      "Complement found from 6222 to 6228\n",
      "Complement found from 9268 to 9274\n",
      "Complement found from 11043 to 11049\n",
      "Complement found from 11787 to 11793\n",
      "Complement found from 13434 to 13440\n",
      "Complement found from 14099 to 14105\n",
      "Complement found from 14282 to 14288\n",
      "Seq is: CCTCTC\n",
      "9 CCTCTC motifs detected in file\n",
      "\n",
      "Complement found from 763 to 769\n",
      "Complement found from 11189 to 11195\n",
      "Seq is: ACGCAG\n",
      "2 ACGCAG motifs detected in file\n",
      "\n",
      "Complement found from 3395 to 3401\n",
      "Complement found from 7428 to 7434\n",
      "Complement found from 12853 to 12859\n",
      "Complement found from 14179 to 14185\n",
      "Seq is: TATACA\n",
      "4 TATACA motifs detected in file\n",
      "\n",
      "Complement found from 8915 to 8921\n",
      "Seq is: ACAAGG\n",
      "1 ACAAGG motifs detected in file\n",
      "\n",
      "Complement found from 4673 to 4679\n",
      "Complement found from 7831 to 7837\n",
      "Complement found from 8744 to 8750\n",
      "Complement found from 12390 to 12396\n",
      "Complement found from 12785 to 12791\n",
      "Seq is: ATCCTT\n",
      "5 ATCCTT motifs detected in file\n",
      "\n",
      "Complement found from 370 to 376\n",
      "Complement found from 440 to 446\n",
      "Complement found from 2761 to 2767\n",
      "Complement found from 5061 to 5067\n",
      "Complement found from 5177 to 5183\n",
      "Complement found from 5186 to 5192\n",
      "Complement found from 5391 to 5397\n",
      "Complement found from 6530 to 6536\n",
      "Complement found from 7286 to 7292\n",
      "Complement found from 7714 to 7720\n",
      "Complement found from 7742 to 7748\n",
      "Complement found from 9010 to 9016\n",
      "Complement found from 9257 to 9263\n",
      "Complement found from 10939 to 10945\n",
      "Complement found from 12245 to 12251\n",
      "Complement found from 12412 to 12418\n",
      "Complement found from 12736 to 12742\n",
      "Complement found from 13161 to 13167\n",
      "Complement found from 13626 to 13632\n",
      "Complement found from 13657 to 13663\n",
      "Complement found from 13741 to 13747\n",
      "Complement found from 13830 to 13836\n",
      "Complement found from 14569 to 14575\n",
      "Complement found from 15599 to 15605\n",
      "Seq is: CTAACA\n",
      "24 CTAACA motifs detected in file\n",
      "\n",
      "Complement found from 732 to 738\n",
      "Complement found from 1234 to 1240\n",
      "Complement found from 2521 to 2527\n",
      "Complement found from 2863 to 2869\n",
      "Complement found from 3472 to 3478\n",
      "Complement found from 3541 to 3547\n",
      "Complement found from 4015 to 4021\n",
      "Complement found from 4084 to 4090\n",
      "Complement found from 8476 to 8482\n",
      "Complement found from 8653 to 8659\n",
      "Complement found from 9646 to 9652\n",
      "Complement found from 9762 to 9768\n",
      "Complement found from 13183 to 13189\n",
      "Complement found from 13453 to 13459\n",
      "Complement found from 14402 to 14408\n",
      "Complement found from 14879 to 14885\n",
      "Complement found from 14911 to 14917\n",
      "Complement found from 15481 to 15487\n",
      "Seq is: TCACCA\n",
      "18 TCACCA motifs detected in file\n",
      "\n",
      "Complement found from 458 to 464\n",
      "Complement found from 464 to 470\n",
      "Complement found from 6310 to 6316\n",
      "Complement found from 6497 to 6503\n",
      "Complement found from 6664 to 6670\n",
      "Complement found from 7816 to 7822\n",
      "Complement found from 11812 to 11818\n",
      "Complement found from 15381 to 15387\n",
      "Seq is: CTCCCA\n",
      "8 CTCCCA motifs detected in file\n",
      "\n",
      "Complement found from 645 to 651\n",
      "Complement found from 1733 to 1739\n",
      "Complement found from 8627 to 8633\n",
      "Complement found from 9677 to 9683\n",
      "Complement found from 11924 to 11930\n",
      "Complement found from 12324 to 12330\n",
      "Complement found from 12689 to 12695\n",
      "Complement found from 12933 to 12939\n",
      "Complement found from 15154 to 15160\n",
      "Seq is: CAAATA\n",
      "9 CAAATA motifs detected in file\n",
      "\n",
      "Complement found from 175 to 181\n",
      "Complement found from 5091 to 5097\n",
      "Complement found from 7298 to 7304\n",
      "Complement found from 8446 to 8452\n",
      "Complement found from 10488 to 10494\n",
      "Complement found from 10786 to 10792\n",
      "Complement found from 15051 to 15057\n",
      "Seq is: ATATTA\n",
      "7 ATATTA motifs detected in file\n",
      "\n",
      "Complement found from 4231 to 4237\n",
      "Complement found from 9699 to 9705\n",
      "Complement found from 13279 to 13285\n",
      "Complement found from 14171 to 14177\n",
      "Complement found from 15182 to 15188\n",
      "Complement found from 16197 to 16203\n",
      "Seq is: TTACAA\n",
      "6 TTACAA motifs detected in file\n",
      "\n",
      "Complement found from 336 to 342\n",
      "Complement found from 443 to 449\n",
      "Complement found from 690 to 696\n",
      "Complement found from 2553 to 2559\n",
      "Complement found from 7410 to 7416\n",
      "Complement found from 9199 to 9205\n",
      "Complement found from 9603 to 9609\n",
      "Complement found from 10716 to 10722\n",
      "Complement found from 10818 to 10824\n",
      "Complement found from 11523 to 11529\n",
      "Complement found from 15033 to 15039\n",
      "Complement found from 16232 to 16238\n",
      "Seq is: ACACAT\n",
      "12 ACACAT motifs detected in file\n",
      "\n",
      "Complement found from 927 to 933\n",
      "Complement found from 9912 to 9918\n",
      "Seq is: AAGCCG\n",
      "2 AAGCCG motifs detected in file\n",
      "\n",
      "Complement found from 190 to 196\n",
      "Complement found from 1504 to 1510\n",
      "Complement found from 5565 to 5571\n",
      "Complement found from 7948 to 7954\n",
      "Complement found from 11199 to 11205\n",
      "Complement found from 15816 to 15822\n",
      "Complement found from 16138 to 16144\n",
      "Seq is: ATACTT\n",
      "7 ATACTT motifs detected in file\n",
      "\n",
      "Complement found from 3798 to 3804\n",
      "Complement found from 4945 to 4951\n",
      "Complement found from 6638 to 6644\n",
      "Complement found from 7642 to 7648\n",
      "Complement found from 7681 to 7687\n",
      "Complement found from 8934 to 8940\n",
      "Complement found from 9093 to 9099\n",
      "Complement found from 11007 to 11013\n",
      "Complement found from 11134 to 11140\n",
      "Seq is: CTTATC\n",
      "9 CTTATC motifs detected in file\n",
      "\n",
      "Complement found from 313 to 319\n",
      "Complement found from 13118 to 13124\n",
      "Seq is: CCGCTT\n",
      "2 CCGCTT motifs detected in file\n",
      "\n",
      "Complement found from 5763 to 5769\n",
      "Seq is: CCCCGG\n",
      "1 CCCCGG motifs detected in file\n",
      "\n",
      "Complement found from 3949 to 3955\n",
      "Seq is: ACGCCG\n",
      "1 ACGCCG motifs detected in file\n",
      "\n",
      "Complement found from 2333 to 2339\n",
      "Complement found from 2442 to 2448\n",
      "Complement found from 6199 to 6205\n",
      "Seq is: CATAAG\n",
      "3 CATAAG motifs detected in file\n",
      "\n",
      "Complement found from 1289 to 1295\n",
      "Complement found from 2385 to 2391\n",
      "Complement found from 3441 to 3447\n",
      "Complement found from 4021 to 4027\n",
      "Complement found from 5928 to 5934\n",
      "Complement found from 6061 to 6067\n",
      "Complement found from 7609 to 7615\n",
      "Complement found from 9731 to 9737\n",
      "Complement found from 10292 to 10298\n",
      "Complement found from 10376 to 10382\n",
      "Complement found from 11062 to 11068\n",
      "Complement found from 14687 to 14693\n",
      "Complement found from 14732 to 14738\n",
      "Seq is: CTACAA\n",
      "13 CTACAA motifs detected in file\n",
      "\n",
      "Complement found from 6652 to 6658\n",
      "Complement found from 14841 to 14847\n",
      "Seq is: CTTCGG\n",
      "2 CTTCGG motifs detected in file\n",
      "\n",
      "Complement found from 5604 to 5610\n",
      "Complement found from 13188 to 13194\n",
      "Seq is: ACTCTG\n",
      "2 ACTCTG motifs detected in file\n",
      "\n",
      "Complement found from 756 to 762\n",
      "Complement found from 4637 to 4643\n",
      "Complement found from 15547 to 15553\n",
      "Seq is: ATCAAG\n",
      "3 ATCAAG motifs detected in file\n",
      "\n",
      "Complement found from 982 to 988\n",
      "Complement found from 4354 to 4360\n",
      "Complement found from 4554 to 4560\n",
      "Complement found from 8513 to 8519\n",
      "Complement found from 9639 to 9645\n",
      "Complement found from 11324 to 11330\n",
      "Complement found from 13552 to 13558\n",
      "Seq is: CCTGAG\n",
      "7 CCTGAG motifs detected in file\n",
      "\n",
      "Complement found from 1967 to 1973\n",
      "Complement found from 16033 to 16039\n",
      "Seq is: GGGAAG\n",
      "2 GGGAAG motifs detected in file\n",
      "\n",
      "Complement found from 706 to 712\n",
      "Seq is: CCGTTC\n",
      "1 CCGTTC motifs detected in file\n",
      "\n",
      "Complement found from 1152 to 1158\n",
      "Complement found from 14302 to 14308\n",
      "Seq is: CAGCTT\n",
      "2 CAGCTT motifs detected in file\n",
      "\n",
      "Complement found from 433 to 439\n",
      "Complement found from 568 to 574\n",
      "Complement found from 801 to 807\n",
      "Complement found from 4878 to 4884\n",
      "Complement found from 5894 to 5900\n",
      "Complement found from 7397 to 7403\n",
      "Complement found from 7466 to 7472\n",
      "Complement found from 7954 to 7960\n",
      "Complement found from 8027 to 8033\n",
      "Complement found from 8405 to 8411\n",
      "Complement found from 8557 to 8563\n",
      "Complement found from 9532 to 9538\n",
      "Complement found from 9554 to 9560\n",
      "Complement found from 11426 to 11432\n",
      "Complement found from 11867 to 11873\n",
      "Complement found from 12083 to 12089\n",
      "Complement found from 12236 to 12242\n",
      "Complement found from 12385 to 12391\n",
      "Complement found from 12431 to 12437\n",
      "Complement found from 14424 to 14430\n",
      "Complement found from 14530 to 14536\n",
      "Complement found from 14587 to 14593\n",
      "Seq is: CCCCCA\n",
      "22 CCCCCA motifs detected in file\n",
      "\n",
      "Complement found from 1717 to 1723\n",
      "Complement found from 1886 to 1892\n",
      "Complement found from 7484 to 7490\n",
      "Complement found from 10633 to 10639\n",
      "Complement found from 10996 to 11002\n",
      "Complement found from 11328 to 11334\n",
      "Complement found from 13968 to 13974\n",
      "Complement found from 15519 to 15525\n",
      "Complement found from 15706 to 15712\n",
      "Seq is: AGCCAA\n",
      "9 AGCCAA motifs detected in file\n",
      "\n",
      "Complement found from 1600 to 1606\n",
      "Complement found from 4814 to 4820\n",
      "Seq is: CCAGAG\n",
      "2 CCAGAG motifs detected in file\n",
      "\n",
      "Complement found from 737 to 743\n",
      "Complement found from 7856 to 7862\n",
      "Seq is: ACGATC\n",
      "2 ACGATC motifs detected in file\n",
      "\n",
      "Complement found from 3815 to 3821\n",
      "Complement found from 4163 to 4169\n",
      "Complement found from 6352 to 6358\n",
      "Complement found from 6598 to 6604\n",
      "Complement found from 8347 to 8353\n",
      "Complement found from 8922 to 8928\n",
      "Complement found from 10540 to 10546\n",
      "Complement found from 12075 to 12081\n",
      "Complement found from 13301 to 13307\n",
      "Seq is: ACACCT\n",
      "9 ACACCT motifs detected in file\n",
      "\n",
      "Complement found from 3612 to 3618\n",
      "Complement found from 5085 to 5091\n",
      "Complement found from 8598 to 8604\n",
      "Complement found from 9458 to 9464\n",
      "Complement found from 9718 to 9724\n",
      "Complement found from 9953 to 9959\n",
      "Complement found from 10876 to 10882\n",
      "Complement found from 10903 to 10909\n",
      "Complement found from 13350 to 13356\n",
      "Seq is: CTATTT\n",
      "9 CTATTT motifs detected in file\n",
      "\n",
      "Complement found from 1501 to 1507\n",
      "Complement found from 1740 to 1746\n",
      "Complement found from 2302 to 2308\n",
      "Complement found from 9998 to 10004\n",
      "Complement found from 10528 to 10534\n",
      "Complement found from 14463 to 14469\n",
      "Complement found from 15895 to 15901\n",
      "Seq is: AGTATA\n",
      "7 AGTATA motifs detected in file\n",
      "\n",
      "Complement found from 1257 to 1263\n",
      "Seq is: ATACCG\n",
      "1 ATACCG motifs detected in file\n",
      "\n",
      "Complement found from 6149 to 6155\n",
      "Complement found from 10913 to 10919\n",
      "Complement found from 16542 to 16548\n",
      "Seq is: GTTCCC\n",
      "3 GTTCCC motifs detected in file\n",
      "\n",
      "Complement found from 8010 to 8016\n",
      "Complement found from 8585 to 8591\n",
      "Complement found from 9744 to 9750\n",
      "Complement found from 11445 to 11451\n",
      "Complement found from 11457 to 11463\n",
      "Seq is: AGTACT\n",
      "5 AGTACT motifs detected in file\n",
      "\n",
      "Complement found from 7771 to 7777\n",
      "Seq is: ACCGTC\n",
      "1 ACCGTC motifs detected in file\n",
      "\n",
      "Complement found from 3330 to 3336\n",
      "Complement found from 7273 to 7279\n",
      "Complement found from 8795 to 8801\n",
      "Complement found from 8973 to 8979\n",
      "Complement found from 10438 to 10444\n",
      "Complement found from 10475 to 10481\n",
      "Complement found from 11500 to 11506\n",
      "Complement found from 11611 to 11617\n",
      "Complement found from 11727 to 11733\n",
      "Complement found from 12042 to 12048\n",
      "Complement found from 13734 to 13740\n",
      "Complement found from 14793 to 14799\n",
      "Complement found from 15742 to 15748\n",
      "Seq is: CTCATT\n",
      "13 CTCATT motifs detected in file\n",
      "\n",
      "Complement found from 3926 to 3932\n",
      "Complement found from 9753 to 9759\n",
      "Complement found from 10702 to 10708\n",
      "Complement found from 12474 to 12480\n",
      "Complement found from 13061 to 13067\n",
      "Seq is: AGTCTC\n",
      "5 AGTCTC motifs detected in file\n",
      "\n",
      "Complement found from 3808 to 3814\n",
      "Complement found from 8051 to 8057\n",
      "Complement found from 8914 to 8920\n",
      "Complement found from 12215 to 12221\n",
      "Complement found from 16428 to 16434\n",
      "Seq is: CACAAG\n",
      "5 CACAAG motifs detected in file\n",
      "\n",
      "Complement found from 335 to 341\n",
      "Complement found from 442 to 448\n",
      "Complement found from 1045 to 1051\n",
      "Complement found from 1614 to 1620\n",
      "Complement found from 2428 to 2434\n",
      "Complement found from 2705 to 2711\n",
      "Complement found from 3806 to 3812\n",
      "Complement found from 8452 to 8458\n",
      "Complement found from 9198 to 9204\n",
      "Complement found from 9353 to 9359\n",
      "Complement found from 9602 to 9608\n",
      "Complement found from 10715 to 10721\n",
      "Complement found from 10817 to 10823\n",
      "Complement found from 10834 to 10840\n",
      "Complement found from 11522 to 11528\n",
      "Complement found from 11997 to 12003\n",
      "Complement found from 12828 to 12834\n",
      "Complement found from 14550 to 14556\n",
      "Seq is: AACACA\n",
      "18 AACACA motifs detected in file\n",
      "\n",
      "Complement found from 1139 to 1145\n",
      "Complement found from 2121 to 2127\n",
      "Complement found from 3870 to 3876\n",
      "Complement found from 5375 to 5381\n",
      "Complement found from 5951 to 5957\n",
      "Complement found from 8419 to 8425\n",
      "Complement found from 8710 to 8716\n",
      "Complement found from 9356 to 9362\n",
      "Complement found from 12346 to 12352\n",
      "Complement found from 13164 to 13170\n",
      "Complement found from 14311 to 14317\n",
      "Complement found from 15472 to 15478\n",
      "Seq is: ACACTA\n",
      "12 ACACTA motifs detected in file\n",
      "\n",
      "Complement found from 7178 to 7184\n",
      "Complement found from 14112 to 14118\n",
      "Seq is: TTCCCA\n",
      "2 TTCCCA motifs detected in file\n",
      "\n",
      "Complement found from 3788 to 3794\n",
      "Complement found from 5212 to 5218\n",
      "Complement found from 6222 to 6228\n",
      "Complement found from 9268 to 9274\n",
      "Complement found from 11043 to 11049\n",
      "Complement found from 11787 to 11793\n",
      "Complement found from 13434 to 13440\n",
      "Complement found from 14099 to 14105\n",
      "Complement found from 14282 to 14288\n",
      "Seq is: CCTCTC\n",
      "9 CCTCTC motifs detected in file\n",
      "\n",
      "Complement found from 879 to 885\n",
      "Complement found from 6388 to 6394\n",
      "Complement found from 9702 to 9708\n",
      "Complement found from 14029 to 14035\n",
      "Seq is: CAATTT\n",
      "4 CAATTT motifs detected in file\n",
      "\n",
      "Complement found from 2820 to 2826\n",
      "Complement found from 5816 to 5822\n",
      "Complement found from 8777 to 8783\n",
      "Complement found from 15432 to 15438\n",
      "Seq is: CCTCGG\n",
      "4 CCTCGG motifs detected in file\n",
      "\n",
      "Complement found from 5033 to 5039\n",
      "Complement found from 16116 to 16122\n",
      "Seq is: TGAATA\n",
      "2 TGAATA motifs detected in file\n",
      "\n",
      "Complement found from 3747 to 3753\n",
      "Complement found from 4750 to 4756\n",
      "Complement found from 7672 to 7678\n",
      "Complement found from 8592 to 8598\n",
      "Complement found from 8751 to 8757\n",
      "Complement found from 12115 to 12121\n",
      "Complement found from 14482 to 14488\n",
      "Complement found from 14668 to 14674\n",
      "Complement found from 15060 to 15066\n",
      "Complement found from 15159 to 15165\n",
      "Complement found from 15790 to 15796\n",
      "Seq is: ATCATT\n",
      "11 ATCATT motifs detected in file\n",
      "\n",
      "Complement found from 3398 to 3404\n",
      "Complement found from 8766 to 8772\n",
      "Complement found from 10132 to 10138\n",
      "Complement found from 10298 to 10304\n",
      "Complement found from 11333 to 11339\n",
      "Seq is: ACAACT\n",
      "5 ACAACT motifs detected in file\n",
      "\n",
      "Complement found from 3181 to 3187\n",
      "Complement found from 5297 to 5303\n",
      "Complement found from 6833 to 6839\n",
      "Complement found from 7261 to 7267\n",
      "Complement found from 7795 to 7801\n",
      "Complement found from 9096 to 9102\n",
      "Complement found from 10340 to 10346\n",
      "Complement found from 10864 to 10870\n",
      "Complement found from 11152 to 11158\n",
      "Complement found from 13371 to 13377\n",
      "Complement found from 13518 to 13524\n",
      "Complement found from 14058 to 14064\n",
      "Complement found from 14977 to 14983\n",
      "Seq is: ATCATC\n",
      "13 ATCATC motifs detected in file\n",
      "\n",
      "Complement found from 194 to 200\n",
      "Complement found from 1699 to 1705\n",
      "Complement found from 3765 to 3771\n",
      "Complement found from 5653 to 5659\n",
      "Complement found from 6679 to 6685\n",
      "Complement found from 10100 to 10106\n",
      "Complement found from 10789 to 10795\n",
      "Complement found from 11731 to 11737\n",
      "Complement found from 13654 to 13660\n",
      "Complement found from 13738 to 13744\n",
      "Complement found from 14633 to 14639\n",
      "Complement found from 15190 to 15196\n",
      "Complement found from 15411 to 15417\n",
      "Complement found from 15628 to 15634\n",
      "Seq is: TTACTA\n",
      "14 TTACTA motifs detected in file\n",
      "\n",
      "Complement found from 1224 to 1230\n",
      "Complement found from 15588 to 15594\n",
      "Seq is: CCGATC\n",
      "2 CCGATC motifs detected in file\n",
      "\n",
      "Complement found from 5897 to 5903\n",
      "Complement found from 8293 to 8299\n",
      "Complement found from 10795 to 10801\n",
      "Seq is: CCACTG\n",
      "3 CCACTG motifs detected in file\n",
      "\n",
      "Complement found from 1500 to 1506\n",
      "Complement found from 1739 to 1745\n",
      "Complement found from 4640 to 4646\n",
      "Complement found from 6858 to 6864\n",
      "Complement found from 16057 to 16063\n",
      "Seq is: AAGTAT\n",
      "5 AAGTAT motifs detected in file\n",
      "\n",
      "Complement found from 4064 to 4070\n",
      "Complement found from 4497 to 4503\n",
      "Complement found from 11037 to 11043\n",
      "Complement found from 11806 to 11812\n",
      "Complement found from 13158 to 13164\n",
      "Seq is: ACTCTA\n",
      "5 ACTCTA motifs detected in file\n",
      "\n",
      "Complement found from 1990 to 1996\n",
      "Complement found from 2764 to 2770\n",
      "Complement found from 3990 to 3996\n",
      "Complement found from 5679 to 5685\n",
      "Complement found from 5930 to 5936\n",
      "Complement found from 8455 to 8461\n",
      "Complement found from 8507 to 8513\n",
      "Complement found from 10294 to 10300\n",
      "Complement found from 11592 to 11598\n",
      "Complement found from 13545 to 13551\n",
      "Complement found from 13861 to 13867\n",
      "Complement found from 14188 to 14194\n",
      "Complement found from 14623 to 14629\n",
      "Complement found from 15184 to 15190\n",
      "Complement found from 15602 to 15608\n",
      "Complement found from 16280 to 16286\n",
      "Seq is: ACAAAC\n",
      "16 ACAAAC motifs detected in file\n",
      "\n",
      "Complement found from 3007 to 3013\n",
      "Complement found from 7233 to 7239\n",
      "Complement found from 11160 to 11166\n",
      "Seq is: CCGATG\n",
      "3 CCGATG motifs detected in file\n",
      "\n",
      "Complement found from 1144 to 1150\n",
      "Complement found from 1913 to 1919\n",
      "Complement found from 6805 to 6811\n",
      "Complement found from 13965 to 13971\n",
      "Seq is: ACGAGC\n",
      "4 ACGAGC motifs detected in file\n",
      "\n",
      "Seq is: GTCACG\n",
      "0 GTCACG motifs detected in file\n",
      "\n",
      "Complement found from 10141 to 10147\n",
      "Complement found from 11475 to 11481\n",
      "Seq is: CGGCTA\n",
      "2 CGGCTA motifs detected in file\n",
      "\n",
      "Seq is: GTCCCG\n",
      "0 GTCCCG motifs detected in file\n",
      "\n",
      "Complement found from 482 to 488\n",
      "Complement found from 755 to 761\n",
      "Complement found from 3670 to 3676\n",
      "Complement found from 4636 to 4642\n",
      "Complement found from 5610 to 5616\n",
      "Complement found from 6385 to 6391\n",
      "Complement found from 7872 to 7878\n",
      "Complement found from 8636 to 8642\n",
      "Complement found from 13226 to 13232\n",
      "Complement found from 13289 to 13295\n",
      "Complement found from 14936 to 14942\n",
      "Complement found from 15546 to 15552\n",
      "Complement found from 16072 to 16078\n",
      "Complement found from 16175 to 16181\n",
      "Complement found from 16235 to 16241\n",
      "Seq is: CATCAA\n",
      "15 CATCAA motifs detected in file\n",
      "\n",
      "Complement found from 1161 to 1167\n",
      "Complement found from 2887 to 2893\n",
      "Complement found from 3646 to 3652\n",
      "Complement found from 3676 to 3682\n",
      "Complement found from 5122 to 5128\n",
      "Complement found from 10135 to 10141\n",
      "Complement found from 11750 to 11756\n",
      "Complement found from 11949 to 11955\n",
      "Complement found from 13078 to 13084\n",
      "Complement found from 13421 to 13427\n",
      "Complement found from 14646 to 14652\n",
      "Complement found from 15873 to 15879\n",
      "Seq is: ACTCAA\n",
      "12 ACTCAA motifs detected in file\n",
      "\n",
      "Complement found from 1018 to 1024\n",
      "Complement found from 1141 to 1147\n",
      "Complement found from 1383 to 1389\n",
      "Complement found from 3401 to 3407\n",
      "Complement found from 3682 to 3688\n",
      "Complement found from 7014 to 7020\n",
      "Complement found from 7919 to 7925\n",
      "Complement found from 8159 to 8165\n",
      "Complement found from 10731 to 10737\n",
      "Complement found from 11756 to 11762\n",
      "Seq is: ACTACG\n",
      "10 ACTACG motifs detected in file\n",
      "\n",
      "Complement found from 1964 to 1970\n",
      "Seq is: AGTGGG\n",
      "1 AGTGGG motifs detected in file\n",
      "\n",
      "Complement found from 3923 to 3929\n",
      "Complement found from 6145 to 6151\n",
      "Complement found from 8738 to 8744\n",
      "Complement found from 8945 to 8951\n",
      "Complement found from 10028 to 10034\n",
      "Complement found from 10525 to 10531\n",
      "Complement found from 10657 to 10663\n",
      "Complement found from 10699 to 10705\n",
      "Complement found from 11958 to 11964\n",
      "Seq is: ACTAGT\n",
      "9 ACTAGT motifs detected in file\n",
      "\n",
      "Complement found from 3321 to 3327\n",
      "Complement found from 3609 to 3615\n",
      "Complement found from 4167 to 4173\n",
      "Complement found from 5474 to 5480\n",
      "Complement found from 6227 to 6233\n",
      "Complement found from 6488 to 6494\n",
      "Complement found from 7941 to 7947\n",
      "Complement found from 9278 to 9284\n",
      "Complement found from 9596 to 9602\n",
      "Complement found from 9728 to 9734\n",
      "Complement found from 10091 to 10097\n",
      "Complement found from 10951 to 10957\n",
      "Complement found from 10972 to 10978\n",
      "Complement found from 11935 to 11941\n",
      "Complement found from 12094 to 12100\n",
      "Complement found from 12984 to 12990\n",
      "Complement found from 13989 to 13995\n",
      "Complement found from 14133 to 14139\n",
      "Complement found from 15334 to 15340\n",
      "Complement found from 15490 to 15496\n",
      "Complement found from 15724 to 15730\n",
      "Seq is: CTCCTA\n",
      "21 CTCCTA motifs detected in file\n",
      "\n",
      "Complement found from 4157 to 4163\n",
      "Complement found from 4519 to 4525\n",
      "Complement found from 4746 to 4752\n",
      "Complement found from 5452 to 5458\n",
      "Complement found from 6981 to 6987\n",
      "Complement found from 8066 to 8072\n",
      "Complement found from 8794 to 8800\n",
      "Complement found from 8972 to 8978\n",
      "Complement found from 9032 to 9038\n",
      "Complement found from 10437 to 10443\n",
      "Complement found from 11238 to 11244\n",
      "Complement found from 11499 to 11505\n",
      "Complement found from 12229 to 12235\n",
      "Complement found from 12424 to 12430\n",
      "Complement found from 12698 to 12704\n",
      "Complement found from 12916 to 12922\n",
      "Complement found from 13112 to 13118\n",
      "Complement found from 13268 to 13274\n",
      "Complement found from 14117 to 14123\n",
      "Complement found from 14792 to 14798\n",
      "Seq is: ACTCAT\n",
      "20 ACTCAT motifs detected in file\n",
      "\n",
      "Complement found from 3904 to 3910\n",
      "Complement found from 10665 to 10671\n",
      "Complement found from 11450 to 11456\n",
      "Seq is: TTGCCG\n",
      "3 TTGCCG motifs detected in file\n",
      "\n",
      "Complement found from 16491 to 16497\n",
      "Seq is: ATCCGA\n",
      "1 ATCCGA motifs detected in file\n",
      "\n",
      "Complement found from 39 to 45\n",
      "Complement found from 3790 to 3796\n",
      "Complement found from 3865 to 3871\n",
      "Complement found from 4219 to 4225\n",
      "Complement found from 4237 to 4243\n",
      "Complement found from 9966 to 9972\n",
      "Complement found from 10209 to 10215\n",
      "Complement found from 10710 to 10716\n",
      "Complement found from 12589 to 12595\n",
      "Complement found from 13019 to 13025\n",
      "Complement found from 13246 to 13252\n",
      "Complement found from 13900 to 13906\n",
      "Complement found from 14047 to 14053\n",
      "Seq is: TCTCCA\n",
      "13 TCTCCA motifs detected in file\n",
      "\n",
      "Complement found from 1224 to 1230\n",
      "Complement found from 15588 to 15594\n",
      "Seq is: CCGATC\n",
      "2 CCGATC motifs detected in file\n",
      "\n",
      "Complement found from 153 to 159\n",
      "Complement found from 448 to 454\n",
      "Complement found from 4322 to 4328\n",
      "Complement found from 10247 to 10253\n",
      "Complement found from 12197 to 12203\n",
      "Complement found from 13895 to 13901\n",
      "Seq is: TTATTT\n",
      "6 TTATTT motifs detected in file\n",
      "\n",
      "Complement found from 3651 to 3657\n",
      "Complement found from 4688 to 4694\n",
      "Complement found from 10549 to 10555\n",
      "Complement found from 11724 to 11730\n",
      "Complement found from 11785 to 11791\n",
      "Complement found from 12882 to 12888\n",
      "Complement found from 14259 to 14265\n",
      "Complement found from 14869 to 14875\n",
      "Complement found from 15097 to 15103\n",
      "Complement found from 15637 to 15643\n",
      "Complement found from 15661 to 15667\n",
      "Complement found from 16404 to 16410\n",
      "Seq is: ATCCTC\n",
      "12 ATCCTC motifs detected in file\n",
      "\n",
      "Complement found from 15057 to 15063\n",
      "Seq is: CGGATC\n",
      "1 CGGATC motifs detected in file\n",
      "\n",
      "Complement found from 4155 to 4161\n",
      "Complement found from 7938 to 7944\n",
      "Complement found from 10133 to 10139\n",
      "Complement found from 12318 to 12324\n",
      "Complement found from 12914 to 12920\n",
      "Complement found from 16244 to 16250\n",
      "Seq is: CAACTC\n",
      "6 CAACTC motifs detected in file\n",
      "\n",
      "Complement found from 5613 to 5619\n",
      "Complement found from 5712 to 5718\n",
      "Complement found from 6139 to 6145\n",
      "Complement found from 12750 to 12756\n",
      "Complement found from 16238 to 16244\n",
      "Seq is: CAACTG\n",
      "5 CAACTG motifs detected in file\n",
      "\n",
      "Complement found from 8077 to 8083\n",
      "Complement found from 16360 to 16366\n",
      "Seq is: GTCCCC\n",
      "2 GTCCCC motifs detected in file\n",
      "\n",
      "Complement found from 490 to 496\n",
      "Complement found from 2422 to 2428\n",
      "Complement found from 3444 to 3450\n",
      "Complement found from 3581 to 3587\n",
      "Complement found from 4484 to 4490\n",
      "Complement found from 5056 to 5062\n",
      "Complement found from 7487 to 7493\n",
      "Complement found from 10943 to 10949\n",
      "Complement found from 11508 to 11514\n",
      "Complement found from 12105 to 12111\n",
      "Complement found from 12544 to 12550\n",
      "Complement found from 12553 to 12559\n",
      "Complement found from 14068 to 14074\n",
      "Complement found from 14271 to 14277\n",
      "Complement found from 14414 to 14420\n",
      "Complement found from 15363 to 15369\n",
      "Complement found from 15522 to 15528\n",
      "Complement found from 16217 to 16223\n",
      "Seq is: CAACCC\n",
      "18 CAACCC motifs detected in file\n",
      "\n",
      "Complement found from 537 to 543\n",
      "Complement found from 1079 to 1085\n",
      "Complement found from 3200 to 3206\n",
      "Complement found from 3306 to 3312\n",
      "Complement found from 3576 to 3582\n",
      "Complement found from 4133 to 4139\n",
      "Complement found from 4224 to 4230\n",
      "Complement found from 4432 to 4438\n",
      "Complement found from 4452 to 4458\n",
      "Complement found from 6113 to 6119\n",
      "Complement found from 12428 to 12434\n",
      "Complement found from 15512 to 15518\n",
      "Seq is: ATACCC\n",
      "12 ATACCC motifs detected in file\n",
      "\n",
      "Complement found from 308 to 314\n",
      "Complement found from 959 to 965\n",
      "Complement found from 3570 to 3576\n",
      "Complement found from 4055 to 4061\n",
      "Complement found from 5381 to 5387\n",
      "Complement found from 5444 to 5450\n",
      "Complement found from 5485 to 5491\n",
      "Complement found from 13032 to 13038\n",
      "Complement found from 14528 to 14534\n",
      "Complement found from 14806 to 14812\n",
      "Complement found from 15539 to 15545\n",
      "Complement found from 16187 to 16193\n",
      "Seq is: CTCCCC\n",
      "12 CTCCCC motifs detected in file\n",
      "\n",
      "Complement found from 915 to 921\n",
      "Complement found from 16055 to 16061\n",
      "Seq is: CCAAGT\n",
      "2 CCAAGT motifs detected in file\n",
      "\n",
      "Complement found from 3181 to 3187\n",
      "Complement found from 5297 to 5303\n",
      "Complement found from 6833 to 6839\n",
      "Complement found from 7261 to 7267\n",
      "Complement found from 7795 to 7801\n",
      "Complement found from 9096 to 9102\n",
      "Complement found from 10340 to 10346\n",
      "Complement found from 10864 to 10870\n",
      "Complement found from 11152 to 11158\n",
      "Complement found from 13371 to 13377\n",
      "Complement found from 13518 to 13524\n",
      "Complement found from 14058 to 14064\n",
      "Complement found from 14977 to 14983\n",
      "Seq is: ATCATC\n",
      "13 ATCATC motifs detected in file\n",
      "\n",
      "Complement found from 7191 to 7197\n",
      "Complement found from 14675 to 14681\n",
      "Complement found from 16355 to 16361\n",
      "Seq is: TTCTCG\n",
      "3 TTCTCG motifs detected in file\n",
      "\n",
      "Complement found from 1702 to 1708\n",
      "Complement found from 2784 to 2790\n",
      "Complement found from 3502 to 3508\n",
      "Complement found from 4185 to 4191\n",
      "Complement found from 4500 to 4506\n",
      "Complement found from 4734 to 4740\n",
      "Complement found from 6644 to 6650\n",
      "Complement found from 6825 to 6831\n",
      "Complement found from 7405 to 7411\n",
      "Complement found from 8460 to 8466\n",
      "Complement found from 10127 to 10133\n",
      "Complement found from 10280 to 10286\n",
      "Complement found from 10792 to 10798\n",
      "Seq is: CTACCA\n",
      "13 CTACCA motifs detected in file\n",
      "\n",
      "Seq is: CTCTCG\n",
      "0 CTCTCG motifs detected in file\n",
      "\n",
      "Complement found from 4589 to 4595\n",
      "Complement found from 5198 to 5204\n",
      "Complement found from 12746 to 12752\n",
      "Seq is: ATTCCA\n",
      "3 ATTCCA motifs detected in file\n",
      "\n",
      "Complement found from 3525 to 3531\n",
      "Complement found from 5762 to 5768\n",
      "Complement found from 7210 to 7216\n",
      "Seq is: GCCCCG\n",
      "3 GCCCCG motifs detected in file\n",
      "\n",
      "Complement found from 1084 to 1090\n",
      "Complement found from 2217 to 2223\n",
      "Complement found from 4018 to 4024\n",
      "Complement found from 7033 to 7039\n",
      "Complement found from 10310 to 10316\n",
      "Complement found from 11020 to 11026\n",
      "Complement found from 11815 to 11821\n",
      "Complement found from 11870 to 11876\n",
      "Complement found from 12969 to 12975\n",
      "Complement found from 13146 to 13152\n",
      "Complement found from 13881 to 13887\n",
      "Complement found from 14390 to 14396\n",
      "Complement found from 15700 to 15706\n",
      "Complement found from 16266 to 16272\n",
      "Seq is: CCACTA\n",
      "14 CCACTA motifs detected in file\n",
      "\n",
      "Complement found from 339 to 345\n",
      "Complement found from 3183 to 3189\n",
      "Complement found from 4882 to 4888\n",
      "Complement found from 7746 to 7752\n",
      "Complement found from 10507 to 10513\n",
      "Complement found from 14823 to 14829\n",
      "Seq is: CATCTC\n",
      "6 CATCTC motifs detected in file\n",
      "\n",
      "Complement found from 1084 to 1090\n",
      "Complement found from 2217 to 2223\n",
      "Complement found from 4018 to 4024\n",
      "Complement found from 7033 to 7039\n",
      "Complement found from 10310 to 10316\n",
      "Complement found from 11020 to 11026\n",
      "Complement found from 11815 to 11821\n",
      "Complement found from 11870 to 11876\n",
      "Complement found from 12969 to 12975\n",
      "Complement found from 13146 to 13152\n",
      "Complement found from 13881 to 13887\n",
      "Complement found from 14390 to 14396\n",
      "Complement found from 15700 to 15706\n",
      "Complement found from 16266 to 16272\n",
      "Seq is: CCACTA\n",
      "14 CCACTA motifs detected in file\n",
      "\n",
      "Complement found from 1263 to 1269\n",
      "Complement found from 4503 to 4509\n",
      "Complement found from 4881 to 4887\n",
      "Complement found from 6339 to 6345\n",
      "Complement found from 9969 to 9975\n",
      "Complement found from 10506 to 10512\n",
      "Complement found from 11579 to 11585\n",
      "Seq is: CCATCT\n",
      "7 CCATCT motifs detected in file\n",
      "\n",
      "Complement found from 520 to 526\n",
      "Complement found from 1479 to 1485\n",
      "Complement found from 7913 to 7919\n",
      "Complement found from 13934 to 13940\n",
      "Complement found from 14563 to 14569\n",
      "Seq is: ACACCG\n",
      "5 ACACCG motifs detected in file\n",
      "\n",
      "Complement found from 353 to 359\n",
      "Complement found from 557 to 563\n",
      "Complement found from 1063 to 1069\n",
      "Complement found from 1682 to 1688\n",
      "Complement found from 1731 to 1737\n",
      "Complement found from 2233 to 2239\n",
      "Complement found from 3715 to 3721\n",
      "Complement found from 5252 to 5258\n",
      "Complement found from 7468 to 7474\n",
      "Complement found from 12302 to 12308\n",
      "Complement found from 12547 to 12553\n",
      "Complement found from 12668 to 12674\n",
      "Complement found from 14071 to 14077\n",
      "Complement found from 14532 to 14538\n",
      "Complement found from 15988 to 15994\n",
      "Seq is: CCCAAA\n",
      "15 CCCAAA motifs detected in file\n",
      "\n",
      "Complement found from 702 to 708\n",
      "Complement found from 2073 to 2079\n",
      "Complement found from 4473 to 4479\n",
      "Complement found from 5300 to 5306\n",
      "Complement found from 6842 to 6848\n",
      "Complement found from 8616 to 8622\n",
      "Complement found from 8841 to 8847\n",
      "Complement found from 8937 to 8943\n",
      "Complement found from 9579 to 9585\n",
      "Complement found from 10189 to 10195\n",
      "Complement found from 11137 to 11143\n",
      "Complement found from 12081 to 12087\n",
      "Complement found from 13760 to 13766\n",
      "Complement found from 13779 to 13785\n",
      "Complement found from 13877 to 13883\n",
      "Complement found from 13944 to 13950\n",
      "Complement found from 15655 to 15661\n",
      "Seq is: ATCCCC\n",
      "17 ATCCCC motifs detected in file\n",
      "\n",
      "Complement found from 3136 to 3142\n",
      "Complement found from 9393 to 9399\n",
      "Complement found from 12053 to 12059\n",
      "Complement found from 12206 to 12212\n",
      "Complement found from 15958 to 15964\n",
      "Seq is: GAGAAA\n",
      "5 GAGAAA motifs detected in file\n",
      "\n",
      "Complement found from 260 to 266\n",
      "Complement found from 5631 to 5637\n",
      "Complement found from 7027 to 7033\n",
      "Complement found from 8129 to 8135\n",
      "Complement found from 8903 to 8909\n",
      "Complement found from 11004 to 11010\n",
      "Complement found from 11394 to 11400\n",
      "Complement found from 13249 to 13255\n",
      "Seq is: CCACTT\n",
      "8 CCACTT motifs detected in file\n",
      "\n",
      "Complement found from 594 to 600\n",
      "Complement found from 777 to 783\n",
      "Complement found from 1162 to 1168\n",
      "Complement found from 3677 to 3683\n",
      "Complement found from 4252 to 4258\n",
      "Complement found from 8684 to 8690\n",
      "Complement found from 11751 to 11757\n",
      "Complement found from 13422 to 13428\n",
      "Complement found from 15874 to 15880\n",
      "Seq is: CTCAAA\n",
      "9 CTCAAA motifs detected in file\n",
      "\n",
      "Complement found from 1257 to 1263\n",
      "Seq is: ATACCG\n",
      "1 ATACCG motifs detected in file\n",
      "\n",
      "Complement found from 11772 to 11778\n",
      "Seq is: CAGTCG\n",
      "1 CAGTCG motifs detected in file\n",
      "\n",
      "Complement found from 3164 to 3170\n",
      "Complement found from 4457 to 4463\n",
      "Complement found from 7177 to 7183\n",
      "Complement found from 7622 to 7628\n",
      "Complement found from 7951 to 7957\n",
      "Complement found from 9081 to 9087\n",
      "Complement found from 11230 to 11236\n",
      "Complement found from 12373 to 12379\n",
      "Complement found from 12479 to 12485\n",
      "Complement found from 13643 to 13649\n",
      "Complement found from 14111 to 14117\n",
      "Seq is: CTTCCC\n",
      "11 CTTCCC motifs detected in file\n",
      "\n",
      "Complement found from 2505 to 2511\n",
      "Complement found from 3939 to 3945\n",
      "Complement found from 7253 to 7259\n",
      "Complement found from 7744 to 7750\n",
      "Complement found from 9894 to 9900\n",
      "Complement found from 12156 to 12162\n",
      "Complement found from 14821 to 14827\n",
      "Complement found from 15085 to 15091\n",
      "Seq is: AACATC\n",
      "8 AACATC motifs detected in file\n",
      "\n",
      "Complement found from 4423 to 4429\n",
      "Complement found from 15037 to 15043\n",
      "Seq is: ATCGGG\n",
      "2 ATCGGG motifs detected in file\n",
      "\n",
      "Complement found from 701 to 707\n",
      "Complement found from 3003 to 3009\n",
      "Complement found from 4350 to 4356\n",
      "Complement found from 5299 to 5305\n",
      "Complement found from 7820 to 7826\n",
      "Complement found from 8840 to 8846\n",
      "Complement found from 10326 to 10332\n",
      "Complement found from 10866 to 10872\n",
      "Complement found from 12602 to 12608\n",
      "Complement found from 13759 to 13765\n",
      "Complement found from 15201 to 15207\n",
      "Seq is: CATCCC\n",
      "11 CATCCC motifs detected in file\n",
      "\n",
      "Seq is: CGGATG\n",
      "0 CGGATG motifs detected in file\n",
      "\n",
      "Complement found from 1876 to 1882\n",
      "Complement found from 5560 to 5566\n",
      "Complement found from 5794 to 5800\n",
      "Complement found from 15107 to 15113\n",
      "Seq is: TTGCAA\n",
      "4 TTGCAA motifs detected in file\n",
      "\n",
      "Complement found from 5763 to 5769\n",
      "Seq is: CCCCGG\n",
      "1 CCCCGG motifs detected in file\n",
      "\n",
      "Complement found from 781 to 787\n",
      "Complement found from 6441 to 6447\n",
      "Complement found from 12947 to 12953\n",
      "Complement found from 13547 to 13553\n",
      "Complement found from 13698 to 13704\n",
      "Seq is: AAACGC\n",
      "5 AAACGC motifs detected in file\n",
      "\n",
      "Complement found from 1117 to 1123\n",
      "Complement found from 1227 to 1233\n",
      "Complement found from 2390 to 2396\n",
      "Complement found from 2964 to 2970\n",
      "Complement found from 3758 to 3764\n",
      "Complement found from 5611 to 5617\n",
      "Complement found from 5710 to 5716\n",
      "Complement found from 8637 to 8643\n",
      "Complement found from 9068 to 9074\n",
      "Complement found from 10082 to 10088\n",
      "Complement found from 10831 to 10837\n",
      "Complement found from 10891 to 10897\n",
      "Complement found from 13290 to 13296\n",
      "Complement found from 14219 to 14225\n",
      "Complement found from 14269 to 14275\n",
      "Complement found from 16073 to 16079\n",
      "Complement found from 16215 to 16221\n",
      "Complement found from 16236 to 16242\n",
      "Seq is: ATCAAC\n",
      "18 ATCAAC motifs detected in file\n",
      "\n",
      "Complement found from 6774 to 6780\n",
      "Complement found from 7575 to 7581\n",
      "Complement found from 12653 to 12659\n",
      "Complement found from 14176 to 14182\n",
      "Complement found from 15668 to 15674\n",
      "Seq is: ATATAT\n",
      "5 ATATAT motifs detected in file\n",
      "\n",
      "Complement found from 15057 to 15063\n",
      "Seq is: CGGATC\n",
      "1 CGGATC motifs detected in file\n",
      "\n",
      "Complement found from 1156 to 1162\n",
      "Complement found from 2350 to 2356\n",
      "Complement found from 2454 to 2460\n",
      "Complement found from 2798 to 2804\n",
      "Complement found from 3263 to 3269\n",
      "Complement found from 8092 to 8098\n",
      "Complement found from 8887 to 8893\n",
      "Complement found from 11464 to 11470\n",
      "Complement found from 13867 to 13873\n",
      "Complement found from 14501 to 14507\n",
      "Seq is: TTAAAA\n",
      "10 TTAAAA motifs detected in file\n",
      "\n",
      "Complement found from 1824 to 1830\n",
      "Complement found from 2710 to 2716\n",
      "Complement found from 11841 to 11847\n",
      "Complement found from 16202 to 16208\n",
      "Seq is: AGCAAG\n",
      "4 AGCAAG motifs detected in file\n",
      "\n",
      "Complement found from 3340 to 3346\n",
      "Complement found from 5070 to 5076\n",
      "Complement found from 5438 to 5444\n",
      "Complement found from 6585 to 6591\n",
      "Complement found from 8030 to 8036\n",
      "Complement found from 11657 to 11663\n",
      "Complement found from 12086 to 12092\n",
      "Complement found from 12838 to 12844\n",
      "Complement found from 15384 to 15390\n",
      "Seq is: CCATTC\n",
      "9 CCATTC motifs detected in file\n",
      "\n",
      "Complement found from 1210 to 1216\n",
      "Complement found from 3081 to 3087\n",
      "Complement found from 3248 to 3254\n",
      "Seq is: GTAATC\n",
      "3 GTAATC motifs detected in file\n",
      "\n",
      "Complement found from 10164 to 10170\n",
      "Complement found from 12186 to 12192\n",
      "Complement found from 13962 to 13968\n",
      "Seq is: CTTACG\n",
      "3 CTTACG motifs detected in file\n",
      "\n",
      "Complement found from 6048 to 6054\n",
      "Seq is: GTAACG\n",
      "1 GTAACG motifs detected in file\n",
      "\n",
      "Complement found from 475 to 481\n",
      "Complement found from 3345 to 3351\n",
      "Complement found from 5354 to 5360\n",
      "Complement found from 5707 to 5713\n",
      "Complement found from 6467 to 6473\n",
      "Complement found from 7930 to 7936\n",
      "Complement found from 8649 to 8655\n",
      "Complement found from 8670 to 8676\n",
      "Complement found from 10768 to 10774\n",
      "Complement found from 11053 to 11059\n",
      "Complement found from 11101 to 11107\n",
      "Complement found from 12720 to 12726\n",
      "Complement found from 12952 to 12958\n",
      "Complement found from 13149 to 13155\n",
      "Complement found from 14136 to 14142\n",
      "Complement found from 14216 to 14222\n",
      "Complement found from 15832 to 15838\n",
      "Seq is: CTAATC\n",
      "17 CTAATC motifs detected in file\n",
      "\n",
      "Complement found from 1728 to 1734\n",
      "Complement found from 2409 to 2415\n",
      "Complement found from 2484 to 2490\n",
      "Complement found from 2923 to 2929\n",
      "Complement found from 4821 to 4827\n",
      "Complement found from 5019 to 5025\n",
      "Complement found from 8258 to 8264\n",
      "Complement found from 8402 to 8408\n",
      "Complement found from 9723 to 9729\n",
      "Complement found from 10274 to 10280\n",
      "Complement found from 11863 to 11869\n",
      "Seq is: TTACCC\n",
      "11 TTACCC motifs detected in file\n",
      "\n",
      "Complement found from 2297 to 2303\n",
      "Seq is: ATGTTA\n",
      "1 ATGTTA motifs detected in file\n",
      "\n",
      "Complement found from 591 to 597\n",
      "Complement found from 1494 to 1500\n",
      "Complement found from 2251 to 2257\n",
      "Complement found from 2325 to 2331\n",
      "Complement found from 3327 to 3333\n",
      "Complement found from 4928 to 4934\n",
      "Complement found from 5011 to 5017\n",
      "Complement found from 5210 to 5216\n",
      "Complement found from 6367 to 6373\n",
      "Complement found from 8775 to 8781\n",
      "Complement found from 9329 to 9335\n",
      "Complement found from 12091 to 12097\n",
      "Complement found from 12981 to 12987\n",
      "Complement found from 13986 to 13992\n",
      "Complement found from 14442 to 14448\n",
      "Complement found from 15739 to 15745\n",
      "Complement found from 16443 to 16449\n",
      "Seq is: CTCCTC\n",
      "17 CTCCTC motifs detected in file\n",
      "\n",
      "Complement found from 3651 to 3657\n",
      "Complement found from 4688 to 4694\n",
      "Complement found from 10549 to 10555\n",
      "Complement found from 11724 to 11730\n",
      "Complement found from 11785 to 11791\n",
      "Complement found from 12882 to 12888\n",
      "Complement found from 14259 to 14265\n",
      "Complement found from 14869 to 14875\n",
      "Complement found from 15097 to 15103\n",
      "Complement found from 15637 to 15643\n",
      "Complement found from 15661 to 15667\n",
      "Complement found from 16404 to 16410\n",
      "Seq is: ATCCTC\n",
      "12 ATCCTC motifs detected in file\n",
      "\n",
      "Complement found from 3884 to 3890\n",
      "Complement found from 4659 to 4665\n",
      "Complement found from 8642 to 8648\n",
      "Complement found from 9667 to 9673\n",
      "Complement found from 12857 to 12863\n",
      "Complement found from 14924 to 14930\n",
      "Complement found from 16078 to 16084\n",
      "Seq is: CAACCG\n",
      "7 CAACCG motifs detected in file\n",
      "\n",
      "Complement found from 1306 to 1312\n",
      "Complement found from 2573 to 2579\n",
      "Complement found from 3336 to 3342\n",
      "Complement found from 13324 to 13330\n",
      "Seq is: GTACCC\n",
      "4 GTACCC motifs detected in file\n",
      "\n",
      "Complement found from 487 to 493\n",
      "Complement found from 3396 to 3402\n",
      "Complement found from 5424 to 5430\n",
      "Complement found from 12854 to 12860\n",
      "Complement found from 14235 to 14241\n",
      "Seq is: ATACAA\n",
      "5 ATACAA motifs detected in file\n",
      "\n",
      "Complement found from 3286 to 3292\n",
      "Complement found from 5797 to 5803\n",
      "Complement found from 8105 to 8111\n",
      "Complement found from 9106 to 9112\n",
      "Complement found from 15581 to 15587\n",
      "Seq is: CAATTC\n",
      "5 CAATTC motifs detected in file\n",
      "\n",
      "Complement found from 2323 to 2329\n",
      "Complement found from 4369 to 4375\n",
      "Complement found from 4926 to 4932\n",
      "Complement found from 5730 to 5736\n",
      "Complement found from 6344 to 6350\n",
      "Complement found from 6486 to 6492\n",
      "Complement found from 10208 to 10214\n",
      "Complement found from 11914 to 11920\n",
      "Complement found from 12089 to 12095\n",
      "Complement found from 12588 to 12594\n",
      "Complement found from 13245 to 13251\n",
      "Complement found from 13899 to 13905\n",
      "Complement found from 15584 to 15590\n",
      "Seq is: TTCTCC\n",
      "13 TTCTCC motifs detected in file\n",
      "\n",
      "Complement found from 848 to 854\n",
      "Complement found from 1520 to 1526\n",
      "Complement found from 1652 to 1658\n",
      "Complement found from 1860 to 1866\n",
      "Complement found from 2087 to 2093\n",
      "Complement found from 5081 to 5087\n",
      "Complement found from 10014 to 10020\n",
      "Complement found from 10025 to 10031\n",
      "Complement found from 15970 to 15976\n",
      "Seq is: TTAACT\n",
      "9 TTAACT motifs detected in file\n",
      "\n",
      "Seq is: TGGAGA\n",
      "0 TGGAGA motifs detected in file\n",
      "\n",
      "Complement found from 2363 to 2369\n",
      "Complement found from 4838 to 4844\n",
      "Complement found from 10798 to 10804\n",
      "Complement found from 11515 to 11521\n",
      "Complement found from 12173 to 12179\n",
      "Complement found from 12530 to 12536\n",
      "Complement found from 13587 to 13593\n",
      "Seq is: CTGACA\n",
      "7 CTGACA motifs detected in file\n",
      "\n",
      "Complement found from 390 to 396\n",
      "Complement found from 1645 to 1651\n",
      "Complement found from 7479 to 7485\n",
      "Complement found from 7510 to 7516\n",
      "Complement found from 14726 to 14732\n",
      "Seq is: TTTCAA\n",
      "5 TTTCAA motifs detected in file\n",
      "\n",
      "Complement found from 5274 to 5280\n",
      "Complement found from 5798 to 5804\n",
      "Complement found from 9682 to 9688\n",
      "Complement found from 14537 to 14543\n",
      "Seq is: AATTCA\n",
      "4 AATTCA motifs detected in file\n",
      "\n",
      "Complement found from 5524 to 5530\n",
      "Complement found from 8376 to 8382\n",
      "Complement found from 15869 to 15875\n",
      "Complement found from 16136 to 16142\n",
      "Seq is: AAATAC\n",
      "4 AAATAC motifs detected in file\n",
      "\n",
      "Complement found from 12951 to 12957\n",
      "Seq is: GCTAAT\n",
      "1 GCTAAT motifs detected in file\n",
      "\n",
      "Complement found from 8249 to 8255\n",
      "Seq is: GGCCCG\n",
      "1 GGCCCG motifs detected in file\n",
      "\n",
      "Complement found from 620 to 626\n",
      "Seq is: AGACGG\n",
      "1 AGACGG motifs detected in file\n",
      "\n",
      "Complement found from 2029 to 2035\n",
      "Complement found from 8216 to 8222\n",
      "Complement found from 12637 to 12643\n",
      "Seq is: TAGAAT\n",
      "3 TAGAAT motifs detected in file\n",
      "\n",
      "Seq is: TGTTTG\n",
      "0 TGTTTG motifs detected in file\n",
      "\n",
      "Complement found from 3130 to 3136\n",
      "Complement found from 4712 to 4718\n",
      "Complement found from 15763 to 15769\n",
      "Complement found from 15796 to 15802\n",
      "Complement found from 15948 to 15954\n",
      "Seq is: GGACAA\n",
      "5 GGACAA motifs detected in file\n",
      "\n",
      "Complement found from 365 to 371\n",
      "Complement found from 2062 to 2068\n",
      "Complement found from 2830 to 2836\n",
      "Complement found from 3562 to 3568\n",
      "Complement found from 3889 to 3895\n",
      "Complement found from 4345 to 4351\n",
      "Complement found from 7361 to 7367\n",
      "Complement found from 7421 to 7427\n",
      "Complement found from 7462 to 7468\n",
      "Seq is: GAACCC\n",
      "9 GAACCC motifs detected in file\n",
      "\n",
      "Complement found from 1308 to 1314\n",
      "Complement found from 1687 to 1693\n",
      "Complement found from 2215 to 2221\n",
      "Complement found from 2768 to 2774\n",
      "Complement found from 3202 to 3208\n",
      "Complement found from 3208 to 3214\n",
      "Complement found from 5021 to 5027\n",
      "Complement found from 5431 to 5437\n",
      "Complement found from 5675 to 5681\n",
      "Complement found from 9209 to 9215\n",
      "Complement found from 10619 to 10625\n",
      "Complement found from 10843 to 10849\n",
      "Complement found from 12016 to 12022\n",
      "Complement found from 12925 to 12931\n",
      "Complement found from 13326 to 13332\n",
      "Complement found from 14356 to 14362\n",
      "Complement found from 14640 to 14646\n",
      "Complement found from 16264 to 16270\n",
      "Complement found from 16288 to 16294\n",
      "Seq is: ACCCAC\n",
      "19 ACCCAC motifs detected in file\n",
      "\n",
      "Complement found from 2162 to 2168\n",
      "Complement found from 3308 to 3314\n",
      "Complement found from 3338 to 3344\n",
      "Complement found from 4226 to 4232\n",
      "Complement found from 4347 to 4353\n",
      "Complement found from 6115 to 6121\n",
      "Complement found from 14517 to 14523\n",
      "Complement found from 16069 to 16075\n",
      "Seq is: ACCCAT\n",
      "8 ACCCAT motifs detected in file\n",
      "\n",
      "Complement found from 10446 to 10452\n",
      "Complement found from 12436 to 12442\n",
      "Complement found from 14966 to 14972\n",
      "Seq is: ATTATG\n",
      "3 ATTATG motifs detected in file\n",
      "\n",
      "Complement found from 5799 to 5805\n",
      "Complement found from 8976 to 8982\n",
      "Complement found from 9683 to 9689\n",
      "Complement found from 10043 to 10049\n",
      "Complement found from 12840 to 12846\n",
      "Seq is: ATTCAA\n",
      "5 ATTCAA motifs detected in file\n",
      "\n",
      "Complement found from 3513 to 3519\n",
      "Complement found from 4065 to 4071\n",
      "Complement found from 4498 to 4504\n",
      "Complement found from 5336 to 5342\n",
      "Complement found from 5926 to 5932\n",
      "Complement found from 8275 to 8281\n",
      "Complement found from 9086 to 9092\n",
      "Complement found from 9183 to 9189\n",
      "Complement found from 10871 to 10877\n",
      "Complement found from 11038 to 11044\n",
      "Complement found from 11807 to 11813\n",
      "Complement found from 11980 to 11986\n",
      "Complement found from 13785 to 13791\n",
      "Complement found from 15067 to 15073\n",
      "Seq is: CTCTAC\n",
      "14 CTCTAC motifs detected in file\n",
      "\n",
      "Complement found from 1777 to 1783\n",
      "Complement found from 6250 to 6256\n",
      "Complement found from 9232 to 9238\n",
      "Complement found from 10409 to 10415\n",
      "Complement found from 11367 to 11373\n",
      "Complement found from 12142 to 12148\n",
      "Complement found from 13088 to 13094\n",
      "Seq is: TATAGT\n",
      "7 TATAGT motifs detected in file\n",
      "\n",
      "Complement found from 8363 to 8369\n",
      "Complement found from 8891 to 8897\n",
      "Complement found from 10467 to 10473\n",
      "Seq is: AAATGC\n",
      "3 AAATGC motifs detected in file\n",
      "\n",
      "Complement found from 2075 to 2081\n",
      "Complement found from 3893 to 3899\n",
      "Complement found from 3959 to 3965\n",
      "Complement found from 4318 to 4324\n",
      "Complement found from 4795 to 4801\n",
      "Complement found from 5487 to 5493\n",
      "Complement found from 8843 to 8849\n",
      "Complement found from 8931 to 8937\n",
      "Complement found from 10161 to 10167\n",
      "Complement found from 11535 to 11541\n",
      "Complement found from 12193 to 12199\n",
      "Complement found from 13763 to 13769\n",
      "Complement found from 15525 to 15531\n",
      "Complement found from 16545 to 16551\n",
      "Seq is: CCCCTT\n",
      "14 CCCCTT motifs detected in file\n",
      "\n",
      "Seq is: CCGCGA\n",
      "0 CCGCGA motifs detected in file\n",
      "\n",
      "Complement found from 1138 to 1144\n",
      "Complement found from 2354 to 2360\n",
      "Complement found from 5682 to 5688\n",
      "Complement found from 5950 to 5956\n",
      "Complement found from 7185 to 7191\n",
      "Complement found from 7716 to 7722\n",
      "Complement found from 8709 to 8715\n",
      "Complement found from 13163 to 13169\n",
      "Complement found from 14397 to 14403\n",
      "Complement found from 15324 to 15330\n",
      "Complement found from 15471 to 15477\n",
      "Complement found from 16462 to 16468\n",
      "Seq is: AACACT\n",
      "12 AACACT motifs detected in file\n",
      "\n",
      "Complement found from 145 to 151\n",
      "Complement found from 5298 to 5304\n",
      "Complement found from 7796 to 7802\n",
      "Complement found from 9861 to 9867\n",
      "Complement found from 10325 to 10331\n",
      "Complement found from 10344 to 10350\n",
      "Complement found from 10865 to 10871\n",
      "Complement found from 11663 to 11669\n",
      "Complement found from 12601 to 12607\n",
      "Complement found from 12880 to 12886\n",
      "Complement found from 13114 to 13120\n",
      "Complement found from 13372 to 13378\n",
      "Complement found from 14119 to 14125\n",
      "Complement found from 14978 to 14984\n",
      "Complement found from 15641 to 15647\n",
      "Seq is: TCATCC\n",
      "15 TCATCC motifs detected in file\n",
      "\n",
      "Complement found from 4787 to 4793\n",
      "Complement found from 6656 to 6662\n",
      "Complement found from 6788 to 6794\n",
      "Complement found from 10520 to 10526\n",
      "Complement found from 10571 to 10577\n",
      "Complement found from 13479 to 13485\n",
      "Seq is: GGAATA\n",
      "6 GGAATA motifs detected in file\n",
      "\n",
      "Complement found from 3134 to 3140\n",
      "Complement found from 8337 to 8343\n",
      "Seq is: AAGAGA\n",
      "2 AAGAGA motifs detected in file\n",
      "\n",
      "Complement found from 11944 to 11950\n",
      "Complement found from 14884 to 14890\n",
      "Seq is: ACAGGA\n",
      "2 ACAGGA motifs detected in file\n",
      "\n",
      "Complement found from 698 to 704\n",
      "Complement found from 752 to 758\n",
      "Complement found from 1813 to 1819\n",
      "Complement found from 14660 to 14666\n",
      "Complement found from 15684 to 15690\n",
      "Seq is: AAGCAT\n",
      "5 AAGCAT motifs detected in file\n",
      "\n",
      "Complement found from 10202 to 10208\n",
      "Complement found from 15595 to 15601\n",
      "Complement found from 16390 to 16396\n",
      "Seq is: GTCCCT\n",
      "3 GTCCCT motifs detected in file\n",
      "\n",
      "Complement found from 2610 to 2616\n",
      "Complement found from 5193 to 5199\n",
      "Complement found from 5329 to 5335\n",
      "Complement found from 8746 to 8752\n",
      "Complement found from 9145 to 9151\n",
      "Complement found from 10068 to 10074\n",
      "Complement found from 11072 to 11078\n",
      "Complement found from 13382 to 13388\n",
      "Complement found from 15458 to 15464\n",
      "Complement found from 15527 to 15533\n",
      "Complement found from 16294 to 16300\n",
      "Complement found from 16547 to 16553\n",
      "Seq is: CCTTAA\n",
      "12 CCTTAA motifs detected in file\n",
      "\n",
      "Seq is: ACTTNC\n",
      "0 ACTTNC motifs detected in file\n",
      "\n",
      "Complement found from 2273 to 2279\n",
      "Complement found from 8819 to 8825\n",
      "Complement found from 9971 to 9977\n",
      "Complement found from 13562 to 13568\n",
      "Seq is: ATCTAT\n",
      "4 ATCTAT motifs detected in file\n",
      "\n",
      "Complement found from 2115 to 2121\n",
      "Complement found from 6133 to 6139\n",
      "Complement found from 9901 to 9907\n",
      "Seq is: CTTTGG\n",
      "3 CTTTGG motifs detected in file\n",
      "\n",
      "Seq is: CTGGAC\n",
      "0 CTGGAC motifs detected in file\n",
      "\n",
      "Complement found from 2294 to 2300\n",
      "Complement found from 3363 to 3369\n",
      "Complement found from 9281 to 9287\n",
      "Seq is: CTAATG\n",
      "3 CTAATG motifs detected in file\n",
      "\n",
      "Complement found from 9023 to 9029\n",
      "Complement found from 9409 to 9415\n",
      "Complement found from 15150 to 15156\n",
      "Seq is: AGGCCA\n",
      "3 AGGCCA motifs detected in file\n",
      "\n",
      "Seq is: GACTTG\n",
      "0 GACTTG motifs detected in file\n",
      "\n",
      "Complement found from 6283 to 6289\n",
      "Complement found from 9653 to 9659\n",
      "Seq is: AGTCTA\n",
      "2 AGTCTA motifs detected in file\n",
      "\n",
      "Seq is: GTGCAT\n",
      "0 GTGCAT motifs detected in file\n",
      "\n",
      "Complement found from 1465 to 1471\n",
      "Complement found from 4059 to 4065\n",
      "Complement found from 5166 to 5172\n",
      "Complement found from 6623 to 6629\n",
      "Complement found from 11184 to 11190\n",
      "Complement found from 11675 to 11681\n",
      "Complement found from 15080 to 15086\n",
      "Complement found from 15752 to 15758\n",
      "Seq is: CCTGAA\n",
      "8 CCTGAA motifs detected in file\n",
      "\n",
      "Complement found from 671 to 677\n",
      "Complement found from 6973 to 6979\n",
      "Complement found from 10858 to 10864\n",
      "Complement found from 13472 to 13478\n",
      "Complement found from 15981 to 15987\n",
      "Seq is: ATTAGC\n",
      "5 ATTAGC motifs detected in file\n",
      "\n",
      "Complement found from 1654 to 1660\n",
      "Complement found from 2903 to 2909\n",
      "Seq is: AACTTG\n",
      "2 AACTTG motifs detected in file\n",
      "\n",
      "Seq is: TTTGCT\n",
      "0 TTTGCT motifs detected in file\n",
      "\n",
      "Complement found from 1579 to 1585\n",
      "Seq is: TGGAAA\n",
      "1 TGGAAA motifs detected in file\n",
      "\n",
      "Complement found from 157 to 163\n",
      "Complement found from 5265 to 5271\n",
      "Complement found from 6069 to 6075\n",
      "Complement found from 6756 to 6762\n",
      "Complement found from 8953 to 8959\n",
      "Seq is: TTATCG\n",
      "5 TTATCG motifs detected in file\n",
      "\n",
      "Complement found from 7213 to 7219\n",
      "Complement found from 9771 to 9777\n",
      "Seq is: CCGACG\n",
      "2 CCGACG motifs detected in file\n",
      "\n",
      "Seq is: TGGTAG\n",
      "0 TGGTAG motifs detected in file\n",
      "\n",
      "Complement found from 1531 to 1537\n",
      "Complement found from 3429 to 3435\n",
      "Complement found from 4093 to 4099\n",
      "Complement found from 5148 to 5154\n",
      "Complement found from 7403 to 7409\n",
      "Complement found from 7823 to 7829\n",
      "Complement found from 9526 to 9532\n",
      "Complement found from 10278 to 10284\n",
      "Complement found from 10290 to 10296\n",
      "Complement found from 10554 to 10560\n",
      "Complement found from 10695 to 10701\n",
      "Complement found from 11060 to 11066\n",
      "Complement found from 11234 to 11240\n",
      "Complement found from 13069 to 13075\n",
      "Complement found from 13681 to 13687\n",
      "Complement found from 13981 to 13987\n",
      "Complement found from 14128 to 14134\n",
      "Seq is: CCCTAC\n",
      "17 CCCTAC motifs detected in file\n",
      "\n",
      "Complement found from 1828 to 1834\n",
      "Complement found from 4330 to 4336\n",
      "Complement found from 5588 to 5594\n",
      "Complement found from 11796 to 11802\n",
      "Complement found from 11946 to 11952\n",
      "Complement found from 13265 to 13271\n",
      "Complement found from 13415 to 13421\n",
      "Complement found from 13823 to 13829\n",
      "Complement found from 14886 to 14892\n",
      "Seq is: AGGACT\n",
      "9 AGGACT motifs detected in file\n",
      "\n",
      "Complement found from 6274 to 6280\n",
      "Seq is: AGGTTG\n",
      "1 AGGTTG motifs detected in file\n",
      "\n",
      "Complement found from 673 to 679\n",
      "Complement found from 3535 to 3541\n",
      "Seq is: TAGCTC\n",
      "2 TAGCTC motifs detected in file\n",
      "\n",
      "Complement found from 4129 to 4135\n",
      "Complement found from 4241 to 4247\n",
      "Seq is: CAGCAT\n",
      "2 CAGCAT motifs detected in file\n",
      "\n",
      "Complement found from 15212 to 15218\n",
      "Seq is: TTGGGA\n",
      "1 TTGGGA motifs detected in file\n",
      "\n",
      "Complement found from 238 to 244\n",
      "Complement found from 281 to 287\n",
      "Complement found from 2762 to 2768\n",
      "Complement found from 3299 to 3305\n",
      "Complement found from 5392 to 5398\n",
      "Complement found from 8505 to 8511\n",
      "Complement found from 10940 to 10946\n",
      "Complement found from 11570 to 11576\n",
      "Complement found from 12027 to 12033\n",
      "Complement found from 12246 to 12252\n",
      "Complement found from 12413 to 12419\n",
      "Complement found from 12737 to 12743\n",
      "Complement found from 13385 to 13391\n",
      "Complement found from 13742 to 13748\n",
      "Complement found from 14570 to 14576\n",
      "Complement found from 15600 to 15606\n",
      "Seq is: TAACAA\n",
      "16 TAACAA motifs detected in file\n",
      "\n",
      "Complement found from 857 to 863\n",
      "Complement found from 1839 to 1845\n",
      "Complement found from 2883 to 2889\n",
      "Complement found from 5954 to 5960\n",
      "Complement found from 6522 to 6528\n",
      "Complement found from 6590 to 6596\n",
      "Complement found from 11048 to 11054\n",
      "Complement found from 11971 to 11977\n",
      "Complement found from 12852 to 12858\n",
      "Complement found from 15814 to 15820\n",
      "Seq is: CTATAC\n",
      "10 CTATAC motifs detected in file\n",
      "\n",
      "Complement found from 4355 to 4361\n",
      "Complement found from 8514 to 8520\n",
      "Complement found from 12764 to 12770\n",
      "Seq is: CTGAGA\n",
      "3 CTGAGA motifs detected in file\n",
      "\n",
      "Complement found from 4114 to 4120\n",
      "Complement found from 4944 to 4950\n",
      "Complement found from 5505 to 5511\n",
      "Complement found from 6637 to 6643\n",
      "Complement found from 8732 to 8738\n",
      "Complement found from 10242 to 10248\n",
      "Complement found from 10332 to 10338\n",
      "Seq is: TCTTAT\n",
      "7 TCTTAT motifs detected in file\n",
      "\n",
      "Complement found from 369 to 375\n",
      "Complement found from 380 to 386\n",
      "Complement found from 2577 to 2583\n",
      "Complement found from 2760 to 2766\n",
      "Complement found from 5060 to 5066\n",
      "Complement found from 5098 to 5104\n",
      "Complement found from 5838 to 5844\n",
      "Complement found from 6334 to 6340\n",
      "Complement found from 7713 to 7719\n",
      "Complement found from 9002 to 9008\n",
      "Complement found from 9256 to 9262\n",
      "Complement found from 10938 to 10944\n",
      "Complement found from 12362 to 12368\n",
      "Complement found from 12411 to 12417\n",
      "Complement found from 13625 to 13631\n",
      "Complement found from 13853 to 13859\n",
      "Complement found from 13997 to 14003\n",
      "Complement found from 14123 to 14129\n",
      "Complement found from 15598 to 15604\n",
      "Seq is: CCTAAC\n",
      "19 CCTAAC motifs detected in file\n",
      "\n",
      "Complement found from 1987 to 1993\n",
      "Seq is: GCGACA\n",
      "1 GCGACA motifs detected in file\n",
      "\n",
      "Complement found from 2616 to 2622\n",
      "Complement found from 2971 to 2977\n",
      "Complement found from 8245 to 8251\n",
      "Complement found from 16384 to 16390\n",
      "Seq is: ATAGGG\n",
      "4 ATAGGG motifs detected in file\n",
      "\n",
      "Complement found from 2231 to 2237\n",
      "Complement found from 15202 to 15208\n",
      "Seq is: ATCCCA\n",
      "2 ATCCCA motifs detected in file\n",
      "\n",
      "Complement found from 1454 to 1460\n",
      "Complement found from 5418 to 5424\n",
      "Complement found from 6277 to 6283\n",
      "Seq is: TTGAAC\n",
      "3 TTGAAC motifs detected in file\n",
      "\n",
      "Complement found from 4827 to 4833\n",
      "Complement found from 8917 to 8923\n",
      "Complement found from 14077 to 14083\n",
      "Seq is: AAGGCA\n",
      "3 AAGGCA motifs detected in file\n",
      "\n",
      "Complement found from 2523 to 2529\n",
      "Complement found from 2668 to 2674\n",
      "Complement found from 2865 to 2871\n",
      "Complement found from 14202 to 14208\n",
      "Complement found from 15768 to 15774\n",
      "Complement found from 15910 to 15916\n",
      "Seq is: ACCAGT\n",
      "6 ACCAGT motifs detected in file\n",
      "\n",
      "Complement found from 172 to 178\n",
      "Complement found from 484 to 490\n",
      "Complement found from 920 to 926\n",
      "Complement found from 4741 to 4747\n",
      "Complement found from 5801 to 5807\n",
      "Complement found from 6408 to 6414\n",
      "Complement found from 7046 to 7052\n",
      "Complement found from 11440 to 11446\n",
      "Complement found from 14446 to 14452\n",
      "Complement found from 14576 to 14582\n",
      "Complement found from 15007 to 15013\n",
      "Complement found from 16417 to 16423\n",
      "Seq is: TCAATA\n",
      "12 TCAATA motifs detected in file\n",
      "\n",
      "Complement found from 1675 to 1681\n",
      "Complement found from 3601 to 3607\n",
      "Complement found from 6354 to 6360\n",
      "Complement found from 8827 to 8833\n",
      "Complement found from 13303 to 13309\n",
      "Complement found from 15221 to 15227\n",
      "Seq is: ACCTAG\n",
      "6 ACCTAG motifs detected in file\n",
      "\n",
      "Complement found from 848 to 854\n",
      "Complement found from 1520 to 1526\n",
      "Complement found from 1652 to 1658\n",
      "Complement found from 1860 to 1866\n",
      "Complement found from 2087 to 2093\n",
      "Complement found from 5081 to 5087\n",
      "Complement found from 10014 to 10020\n",
      "Complement found from 10025 to 10031\n",
      "Complement found from 15970 to 15976\n",
      "Seq is: TTAACT\n",
      "9 TTAACT motifs detected in file\n",
      "\n",
      "Complement found from 1037 to 1043\n",
      "Complement found from 1184 to 1190\n",
      "Complement found from 2961 to 2967\n",
      "Complement found from 5386 to 5392\n",
      "Complement found from 10546 to 10552\n",
      "Complement found from 13535 to 13541\n",
      "Seq is: CATATC\n",
      "6 CATATC motifs detected in file\n",
      "\n",
      "Complement found from 756 to 762\n",
      "Complement found from 4637 to 4643\n",
      "Complement found from 15547 to 15553\n",
      "Seq is: ATCAAG\n",
      "3 ATCAAG motifs detected in file\n",
      "\n",
      "Complement found from 1904 to 1910\n",
      "Complement found from 8957 to 8963\n",
      "Complement found from 9671 to 9677\n",
      "Complement found from 11124 to 11130\n",
      "Complement found from 13523 to 13529\n",
      "Complement found from 15348 to 15354\n",
      "Seq is: CGAAAC\n",
      "6 CGAAAC motifs detected in file\n",
      "\n",
      "Complement found from 2907 to 2913\n",
      "Complement found from 16143 to 16149\n",
      "Complement found from 16396 to 16402\n",
      "Seq is: TGACCA\n",
      "3 TGACCA motifs detected in file\n",
      "\n",
      "Complement found from 1267 to 1273\n",
      "Complement found from 16510 to 16516\n",
      "Seq is: CTTCAG\n",
      "2 CTTCAG motifs detected in file\n",
      "\n",
      "Complement found from 10770 to 10776\n",
      "Complement found from 13235 to 13241\n",
      "Seq is: AATCGT\n",
      "2 AATCGT motifs detected in file\n",
      "\n",
      "Complement found from 2312 to 2318\n",
      "Complement found from 4862 to 4868\n",
      "Complement found from 5180 to 5186\n",
      "Complement found from 7247 to 7253\n",
      "Complement found from 10801 to 10807\n",
      "Seq is: ACATGA\n",
      "5 ACATGA motifs detected in file\n",
      "\n",
      "Complement found from 1145 to 1151\n",
      "Complement found from 2000 to 2006\n",
      "Complement found from 6014 to 6020\n",
      "Complement found from 13966 to 13972\n",
      "Seq is: CGAGCC\n",
      "4 CGAGCC motifs detected in file\n",
      "\n",
      "Complement found from 10712 to 10718\n",
      "Complement found from 12748 to 12754\n",
      "Complement found from 12912 to 12918\n",
      "Complement found from 13902 to 13908\n",
      "Complement found from 14818 to 14824\n",
      "Seq is: TCCAAC\n",
      "5 TCCAAC motifs detected in file\n",
      "\n",
      "Complement found from 2023 to 2029\n",
      "Complement found from 3214 to 3220\n",
      "Complement found from 4087 to 4093\n",
      "Complement found from 5533 to 5539\n",
      "Complement found from 11307 to 11313\n",
      "Complement found from 12509 to 12515\n",
      "Complement found from 14405 to 14411\n",
      "Seq is: CCAAGA\n",
      "7 CCAAGA motifs detected in file\n",
      "\n",
      "Complement found from 15043 to 15049\n",
      "Seq is: CGAGGC\n",
      "1 CGAGGC motifs detected in file\n",
      "\n",
      "Complement found from 4239 to 4245\n",
      "Complement found from 5135 to 5141\n",
      "Complement found from 9515 to 9521\n",
      "Seq is: TCCAGC\n",
      "3 TCCAGC motifs detected in file\n",
      "\n",
      "Seq is: GGGGTT\n",
      "0 GGGGTT motifs detected in file\n",
      "\n",
      "Complement found from 1029 to 1035\n",
      "Complement found from 5243 to 5249\n",
      "Complement found from 6131 to 6137\n",
      "Complement found from 8871 to 8877\n",
      "Complement found from 12255 to 12261\n",
      "Seq is: GGCTTT\n",
      "5 GGCTTT motifs detected in file\n",
      "\n",
      "Complement found from 286 to 292\n",
      "Complement found from 1800 to 1806\n",
      "Complement found from 2151 to 2157\n",
      "Complement found from 2227 to 2233\n",
      "Complement found from 2800 to 2806\n",
      "Complement found from 3380 to 3386\n",
      "Complement found from 4606 to 4612\n",
      "Complement found from 5401 to 5407\n",
      "Complement found from 8232 to 8238\n",
      "Complement found from 8442 to 8448\n",
      "Complement found from 8490 to 8496\n",
      "Complement found from 8497 to 8503\n",
      "Complement found from 8889 to 8895\n",
      "Complement found from 10152 to 10158\n",
      "Complement found from 12305 to 12311\n",
      "Complement found from 13232 to 13238\n",
      "Complement found from 13406 to 13412\n",
      "Seq is: AAAAAT\n",
      "17 AAAAAT motifs detected in file\n",
      "\n",
      "Seq is: TACGTC\n",
      "0 TACGTC motifs detected in file\n",
      "\n",
      "Complement found from 298 to 304\n",
      "Complement found from 348 to 354\n",
      "Complement found from 552 to 558\n",
      "Complement found from 1273 to 1279\n",
      "Complement found from 1684 to 1690\n",
      "Complement found from 1720 to 1726\n",
      "Complement found from 1991 to 1997\n",
      "Complement found from 2765 to 2771\n",
      "Complement found from 2788 to 2794\n",
      "Complement found from 4254 to 4260\n",
      "Complement found from 4982 to 4988\n",
      "Complement found from 5931 to 5937\n",
      "Complement found from 7121 to 7127\n",
      "Complement found from 8125 to 8131\n",
      "Complement found from 8189 to 8195\n",
      "Complement found from 8508 to 8514\n",
      "Complement found from 11668 to 11674\n",
      "Complement found from 14624 to 14630\n",
      "Complement found from 16281 to 16287\n",
      "Seq is: CAAACC\n",
      "19 CAAACC motifs detected in file\n",
      "\n",
      "Complement found from 740 to 746\n",
      "Complement found from 3671 to 3677\n",
      "Complement found from 7873 to 7879\n",
      "Complement found from 8673 to 8679\n",
      "Complement found from 11316 to 11322\n",
      "Complement found from 11922 to 11928\n",
      "Complement found from 13227 to 13233\n",
      "Complement found from 15357 to 15363\n",
      "Complement found from 15421 to 15427\n",
      "Complement found from 16176 to 16182\n",
      "Seq is: ATCAAA\n",
      "10 ATCAAA motifs detected in file\n",
      "\n",
      "Seq is: TCCTGG\n",
      "0 TCCTGG motifs detected in file\n",
      "\n",
      "Complement found from 564 to 570\n",
      "Complement found from 798 to 804\n",
      "Complement found from 1942 to 1948\n",
      "Complement found from 2160 to 2166\n",
      "Complement found from 2213 to 2219\n",
      "Complement found from 2257 to 2263\n",
      "Complement found from 3206 to 3212\n",
      "Complement found from 4009 to 4015\n",
      "Complement found from 4387 to 4393\n",
      "Complement found from 5189 to 5195\n",
      "Complement found from 7110 to 7116\n",
      "Complement found from 8928 to 8934\n",
      "Complement found from 10086 to 10092\n",
      "Complement found from 10617 to 10623\n",
      "Complement found from 11213 to 11219\n",
      "Complement found from 11264 to 11270\n",
      "Complement found from 12059 to 12065\n",
      "Complement found from 14553 to 14559\n",
      "Complement found from 15533 to 15539\n",
      "Seq is: ACACCC\n",
      "19 ACACCC motifs detected in file\n",
      "\n",
      "Complement found from 358 to 364\n",
      "Complement found from 642 to 648\n",
      "Complement found from 3718 to 3724\n",
      "Complement found from 5170 to 5176\n",
      "Complement found from 5284 to 5290\n",
      "Complement found from 6193 to 6199\n",
      "Complement found from 8688 to 8694\n",
      "Complement found from 9664 to 9670\n",
      "Complement found from 10296 to 10302\n",
      "Complement found from 10417 to 10423\n",
      "Complement found from 12550 to 12556\n",
      "Complement found from 13771 to 13777\n",
      "Complement found from 14026 to 14032\n",
      "Complement found from 14190 to 14196\n",
      "Complement found from 14655 to 14661\n",
      "Complement found from 15360 to 15366\n",
      "Complement found from 15676 to 15682\n",
      "Complement found from 15864 to 15870\n",
      "Seq is: AAACAA\n",
      "18 AAACAA motifs detected in file\n",
      "\n",
      "Complement found from 41 to 47\n",
      "Complement found from 7500 to 7506\n",
      "Seq is: TCCATG\n",
      "2 TCCATG motifs detected in file\n",
      "\n",
      "Complement found from 1990 to 1996\n",
      "Complement found from 2764 to 2770\n",
      "Complement found from 3990 to 3996\n",
      "Complement found from 5679 to 5685\n",
      "Complement found from 5930 to 5936\n",
      "Complement found from 8455 to 8461\n",
      "Complement found from 8507 to 8513\n",
      "Complement found from 10294 to 10300\n",
      "Complement found from 11592 to 11598\n",
      "Complement found from 13545 to 13551\n",
      "Complement found from 13861 to 13867\n",
      "Complement found from 14188 to 14194\n",
      "Complement found from 14623 to 14629\n",
      "Complement found from 15184 to 15190\n",
      "Complement found from 15602 to 15608\n",
      "Complement found from 16280 to 16286\n",
      "Seq is: ACAAAC\n",
      "16 ACAAAC motifs detected in file\n",
      "\n",
      "Complement found from 15178 to 15184\n",
      "Seq is: GTAATT\n",
      "1 GTAATT motifs detected in file\n",
      "\n",
      "Complement found from 1518 to 1524\n",
      "Complement found from 2085 to 2091\n",
      "Complement found from 2155 to 2161\n",
      "Complement found from 5079 to 5085\n",
      "Complement found from 16006 to 16012\n",
      "Seq is: ATTTAA\n",
      "5 ATTTAA motifs detected in file\n",
      "\n",
      "Complement found from 10432 to 10438\n",
      "Seq is: TTTCGA\n",
      "1 TTTCGA motifs detected in file\n",
      "\n",
      "Seq is: GGTATC\n",
      "0 GGTATC motifs detected in file\n",
      "\n",
      "Complement found from 1134 to 1140\n",
      "Complement found from 1376 to 1382\n",
      "Complement found from 11176 to 11182\n",
      "Seq is: CCAGAA\n",
      "3 CCAGAA motifs detected in file\n",
      "\n",
      "Complement found from 2825 to 2831\n",
      "Complement found from 2843 to 2849\n",
      "Complement found from 3706 to 3712\n",
      "Complement found from 6264 to 6270\n",
      "Complement found from 12817 to 12823\n",
      "Seq is: GAGCAG\n",
      "5 GAGCAG motifs detected in file\n",
      "\n",
      "Complement found from 6050 to 6056\n",
      "Seq is: AACGAC\n",
      "1 AACGAC motifs detected in file\n",
      "\n",
      "Complement found from 1050 to 1056\n",
      "Complement found from 2968 to 2974\n",
      "Complement found from 4698 to 4704\n",
      "Complement found from 5286 to 5292\n",
      "Complement found from 11356 to 11362\n",
      "Complement found from 12489 to 12495\n",
      "Complement found from 14173 to 14179\n",
      "Seq is: ACAATA\n",
      "7 ACAATA motifs detected in file\n",
      "\n",
      "Complement found from 15039 to 15045\n",
      "Seq is: CGGGCG\n",
      "1 CGGGCG motifs detected in file\n",
      "\n",
      "Complement found from 2718 to 2724\n",
      "Complement found from 5757 to 5763\n",
      "Complement found from 7320 to 7326\n",
      "Complement found from 14600 to 14606\n",
      "Seq is: GAGAAG\n",
      "4 GAGAAG motifs detected in file\n",
      "\n",
      "Complement found from 2684 to 2690\n",
      "Complement found from 7787 to 7793\n",
      "Seq is: TGCCCG\n",
      "2 TGCCCG motifs detected in file\n",
      "\n",
      "Complement found from 710 to 716\n",
      "Complement found from 999 to 1005\n",
      "Complement found from 4591 to 4597\n",
      "Complement found from 11011 to 11017\n",
      "Seq is: TCCAGT\n",
      "4 TCCAGT motifs detected in file\n",
      "\n",
      "Complement found from 198 to 204\n",
      "Complement found from 1737 to 1743\n",
      "Complement found from 3049 to 3055\n",
      "Complement found from 4398 to 4404\n",
      "Complement found from 14318 to 14324\n",
      "Complement found from 16478 to 16484\n",
      "Seq is: TAAAGT\n",
      "6 TAAAGT motifs detected in file\n",
      "\n",
      "Complement found from 2268 to 2274\n",
      "Complement found from 2908 to 2914\n",
      "Complement found from 3881 to 3887\n",
      "Complement found from 4152 to 4158\n",
      "Complement found from 5531 to 5537\n",
      "Complement found from 5659 to 5665\n",
      "Complement found from 7118 to 7124\n",
      "Complement found from 12507 to 12513\n",
      "Complement found from 14697 to 14703\n",
      "Seq is: GACCAA\n",
      "9 GACCAA motifs detected in file\n",
      "\n",
      "Complement found from 6942 to 6948\n",
      "Complement found from 7308 to 7314\n",
      "Complement found from 7509 to 7515\n",
      "Complement found from 9163 to 9169\n",
      "Complement found from 14932 to 14938\n",
      "Seq is: TTTTCA\n",
      "5 TTTTCA motifs detected in file\n",
      "\n",
      "Complement found from 5924 to 5930\n",
      "Complement found from 7282 to 7288\n",
      "Complement found from 15065 to 15071\n",
      "Complement found from 15443 to 15449\n",
      "Complement found from 15452 to 15458\n",
      "Complement found from 16016 to 16022\n",
      "Seq is: TTCTCT\n",
      "6 TTCTCT motifs detected in file\n",
      "\n",
      "Seq is: GTCGAC\n",
      "0 GTCGAC motifs detected in file\n",
      "\n",
      "Complement found from 15212 to 15218\n",
      "Seq is: TTGGGA\n",
      "1 TTGGGA motifs detected in file\n",
      "\n",
      "Complement found from 162 to 168\n",
      "Complement found from 5163 to 5169\n",
      "Complement found from 9038 to 9044\n",
      "Seq is: GCACCT\n",
      "3 GCACCT motifs detected in file\n",
      "\n",
      "Complement found from 3246 to 3252\n",
      "Seq is: CGGTAA\n",
      "1 CGGTAA motifs detected in file\n",
      "\n",
      "Complement found from 9961 to 9967\n",
      "Seq is: GTATGT\n",
      "1 GTATGT motifs detected in file\n",
      "\n",
      "Complement found from 868 to 874\n",
      "Seq is: CCAGGG\n",
      "1 CCAGGG motifs detected in file\n",
      "\n",
      "Complement found from 10001 to 10007\n",
      "Complement found from 10484 to 10490\n",
      "Complement found from 14292 to 14298\n",
      "Complement found from 14496 to 14502\n",
      "Complement found from 14592 to 14598\n",
      "Complement found from 16134 to 16140\n",
      "Seq is: ATAAAT\n",
      "6 ATAAAT motifs detected in file\n",
      "\n",
      "Complement found from 7988 to 7994\n",
      "Complement found from 11540 to 11546\n",
      "Complement found from 14852 to 14858\n",
      "Complement found from 15617 to 15623\n",
      "Complement found from 15888 to 15894\n",
      "Seq is: TCCTTG\n",
      "5 TCCTTG motifs detected in file\n",
      "\n",
      "Complement found from 4800 to 4806\n",
      "Complement found from 5872 to 5878\n",
      "Complement found from 7083 to 7089\n",
      "Complement found from 7142 to 7148\n",
      "Complement found from 9309 to 9315\n",
      "Complement found from 9879 to 9885\n",
      "Complement found from 15287 to 15293\n",
      "Seq is: TTCACT\n",
      "7 TTCACT motifs detected in file\n",
      "\n",
      "Complement found from 3946 to 3952\n",
      "Complement found from 11487 to 11493\n",
      "Complement found from 14754 to 14760\n",
      "Seq is: AATACG\n",
      "3 AATACG motifs detected in file\n",
      "\n",
      "Seq is: ATGGAA\n",
      "0 ATGGAA motifs detected in file\n",
      "\n",
      "Seq is: ACGTGC\n",
      "0 ACGTGC motifs detected in file\n",
      "\n",
      "Complement found from 2943 to 2949\n",
      "Complement found from 3649 to 3655\n",
      "Complement found from 8563 to 8569\n",
      "Complement found from 12847 to 12853\n",
      "Complement found from 13777 to 13783\n",
      "Complement found from 13942 to 13948\n",
      "Complement found from 14367 to 14373\n",
      "Complement found from 15827 to 15833\n",
      "Complement found from 16168 to 16174\n",
      "Seq is: CAATCC\n",
      "9 CAATCC motifs detected in file\n",
      "\n",
      "Complement found from 3311 to 3317\n",
      "Complement found from 7493 to 7499\n",
      "Complement found from 8834 to 8840\n",
      "Complement found from 10989 to 10995\n",
      "Complement found from 12252 to 12258\n",
      "Seq is: CATGGC\n",
      "5 CATGGC motifs detected in file\n",
      "\n",
      "Complement found from 929 to 935\n",
      "Seq is: GCCGGC\n",
      "1 GCCGGC motifs detected in file\n",
      "\n",
      "Complement found from 2050 to 2056\n",
      "Complement found from 5791 to 5797\n",
      "Complement found from 10824 to 10830\n",
      "Seq is: AATTTG\n",
      "3 AATTTG motifs detected in file\n",
      "\n",
      "Complement found from 689 to 695\n",
      "Complement found from 1476 to 1482\n",
      "Complement found from 3987 to 3993\n",
      "Complement found from 4068 to 4074\n",
      "Complement found from 8704 to 8710\n",
      "Complement found from 11353 to 11359\n",
      "Complement found from 13214 to 13220\n",
      "Complement found from 13542 to 13548\n",
      "Complement found from 15032 to 15038\n",
      "Complement found from 15415 to 15421\n",
      "Complement found from 15577 to 15583\n",
      "Seq is: TACACA\n",
      "11 TACACA motifs detected in file\n",
      "\n",
      "Seq is: GTGACC\n",
      "0 GTGACC motifs detected in file\n",
      "\n",
      "Seq is: CGTCCG\n",
      "0 CGTCCG motifs detected in file\n",
      "\n",
      "Complement found from 804 to 810\n",
      "Complement found from 6883 to 6889\n",
      "Complement found from 9814 to 9820\n",
      "Complement found from 11712 to 11718\n",
      "Seq is: CCACGG\n",
      "4 CCACGG motifs detected in file\n",
      "\n",
      "Complement found from 685 to 691\n",
      "Complement found from 2347 to 2353\n",
      "Complement found from 8332 to 8338\n",
      "Complement found from 8884 to 8890\n",
      "Seq is: AGATTA\n",
      "4 AGATTA motifs detected in file\n",
      "\n",
      "Complement found from 2907 to 2913\n",
      "Complement found from 16143 to 16149\n",
      "Complement found from 16396 to 16402\n",
      "Seq is: TGACCA\n",
      "3 TGACCA motifs detected in file\n",
      "\n",
      "Complement found from 1019 to 1025\n",
      "Complement found from 1142 to 1148\n",
      "Complement found from 1384 to 1390\n",
      "Complement found from 4148 to 4154\n",
      "Complement found from 7905 to 7911\n",
      "Complement found from 11587 to 11593\n",
      "Complement found from 11757 to 11763\n",
      "Seq is: CTACGA\n",
      "7 CTACGA motifs detected in file\n",
      "\n",
      "Complement found from 39 to 45\n",
      "Complement found from 3790 to 3796\n",
      "Complement found from 3865 to 3871\n",
      "Complement found from 4219 to 4225\n",
      "Complement found from 4237 to 4243\n",
      "Complement found from 9966 to 9972\n",
      "Complement found from 10209 to 10215\n",
      "Complement found from 10710 to 10716\n",
      "Complement found from 12589 to 12595\n",
      "Complement found from 13019 to 13025\n",
      "Complement found from 13246 to 13252\n",
      "Complement found from 13900 to 13906\n",
      "Complement found from 14047 to 14053\n",
      "Seq is: TCTCCA\n",
      "13 TCTCCA motifs detected in file\n",
      "\n",
      "Complement found from 1289 to 1295\n",
      "Complement found from 2385 to 2391\n",
      "Complement found from 3441 to 3447\n",
      "Complement found from 4021 to 4027\n",
      "Complement found from 5928 to 5934\n",
      "Complement found from 6061 to 6067\n",
      "Complement found from 7609 to 7615\n",
      "Complement found from 9731 to 9737\n",
      "Complement found from 10292 to 10298\n",
      "Complement found from 10376 to 10382\n",
      "Complement found from 11062 to 11068\n",
      "Complement found from 14687 to 14693\n",
      "Complement found from 14732 to 14738\n",
      "Seq is: CTACAA\n",
      "13 CTACAA motifs detected in file\n",
      "\n",
      "Complement found from 1721 to 1727\n",
      "Complement found from 1906 to 1912\n",
      "Complement found from 4978 to 4984\n",
      "Complement found from 5932 to 5938\n",
      "Complement found from 7529 to 7535\n",
      "Complement found from 8121 to 8127\n",
      "Complement found from 8190 to 8196\n",
      "Complement found from 8959 to 8965\n",
      "Complement found from 9673 to 9679\n",
      "Complement found from 11126 to 11132\n",
      "Complement found from 13426 to 13432\n",
      "Complement found from 14712 to 14718\n",
      "Seq is: AAACCA\n",
      "12 AAACCA motifs detected in file\n",
      "\n",
      "Complement found from 4080 to 4086\n",
      "Complement found from 7544 to 7550\n",
      "Seq is: TTTGTC\n",
      "2 TTTGTC motifs detected in file\n",
      "\n",
      "Complement found from 3524 to 3530\n",
      "Complement found from 6444 to 6450\n",
      "Seq is: CGCCCC\n",
      "2 CGCCCC motifs detected in file\n",
      "\n",
      "Complement found from 2950 to 2956\n",
      "Complement found from 3342 to 3348\n",
      "Complement found from 3384 to 3390\n",
      "Complement found from 3618 to 3624\n",
      "Complement found from 3750 to 3756\n",
      "Complement found from 6587 to 6593\n",
      "Complement found from 8595 to 8601\n",
      "Complement found from 9108 to 9114\n",
      "Complement found from 9114 to 9120\n",
      "Complement found from 11208 to 11214\n",
      "Complement found from 11284 to 11290\n",
      "Complement found from 13916 to 13922\n",
      "Complement found from 15745 to 15751\n",
      "Complement found from 16000 to 16006\n",
      "Seq is: ATTCTA\n",
      "14 ATTCTA motifs detected in file\n",
      "\n",
      "Complement found from 2563 to 2569\n",
      "Complement found from 10139 to 10145\n",
      "Seq is: AACGGC\n",
      "2 AACGGC motifs detected in file\n",
      "\n",
      "Complement found from 1328 to 1334\n",
      "Complement found from 11793 to 11799\n",
      "Seq is: TCAAGG\n",
      "2 TCAAGG motifs detected in file\n",
      "\n",
      "Complement found from 611 to 617\n",
      "Complement found from 4440 to 4446\n",
      "Complement found from 5407 to 5413\n",
      "Complement found from 8523 to 8529\n",
      "Complement found from 8890 to 8896\n",
      "Complement found from 13219 to 13225\n",
      "Seq is: AAAATG\n",
      "6 AAAATG motifs detected in file\n",
      "\n",
      "Complement found from 285 to 291\n",
      "Complement found from 992 to 998\n",
      "Complement found from 2129 to 2135\n",
      "Complement found from 2150 to 2156\n",
      "Complement found from 2226 to 2232\n",
      "Complement found from 2456 to 2462\n",
      "Complement found from 4175 to 4181\n",
      "Complement found from 4604 to 4610\n",
      "Complement found from 5281 to 5287\n",
      "Complement found from 5746 to 5752\n",
      "Complement found from 6691 to 6697\n",
      "Complement found from 7446 to 7452\n",
      "Complement found from 8496 to 8502\n",
      "Complement found from 10047 to 10053\n",
      "Complement found from 10813 to 10819\n",
      "Complement found from 11031 to 11037\n",
      "Complement found from 12417 to 12423\n",
      "Complement found from 13230 to 13236\n",
      "Complement found from 14503 to 14509\n",
      "Seq is: AAAAAA\n",
      "19 AAAAAA motifs detected in file\n",
      "\n",
      "Complement found from 2264 to 2270\n",
      "Complement found from 5945 to 5951\n",
      "Complement found from 9045 to 9051\n",
      "Complement found from 15793 to 15799\n",
      "Seq is: ATTGGA\n",
      "4 ATTGGA motifs detected in file\n",
      "\n",
      "Complement found from 745 to 751\n",
      "Complement found from 2445 to 2451\n",
      "Complement found from 2467 to 2473\n",
      "Complement found from 7450 to 7456\n",
      "Complement found from 10569 to 10575\n",
      "Seq is: AAGGAA\n",
      "5 AAGGAA motifs detected in file\n",
      "\n",
      "Complement found from 7220 to 7226\n",
      "Complement found from 9615 to 9621\n",
      "Complement found from 13909 to 13915\n",
      "Seq is: TACTCG\n",
      "3 TACTCG motifs detected in file\n",
      "\n",
      "Complement found from 11386 to 11392\n",
      "Complement found from 15055 to 15061\n",
      "Seq is: TACGGA\n",
      "2 TACGGA motifs detected in file\n",
      "\n",
      "Complement found from 3803 to 3809\n",
      "Complement found from 4070 to 4076\n",
      "Complement found from 6397 to 6403\n",
      "Complement found from 7182 to 7188\n",
      "Complement found from 7722 to 7728\n",
      "Complement found from 8706 to 8712\n",
      "Complement found from 8765 to 8771\n",
      "Complement found from 10131 to 10137\n",
      "Complement found from 10836 to 10842\n",
      "Complement found from 11260 to 11266\n",
      "Complement found from 11994 to 12000\n",
      "Complement found from 12485 to 12491\n",
      "Complement found from 12542 to 12548\n",
      "Complement found from 12928 to 12934\n",
      "Complement found from 13377 to 13383\n",
      "Complement found from 14328 to 14334\n",
      "Complement found from 15822 to 15828\n",
      "Seq is: CACAAC\n",
      "17 CACAAC motifs detected in file\n",
      "\n",
      "Complement found from 436 to 442\n",
      "Complement found from 1626 to 1632\n",
      "Complement found from 4154 to 4160\n",
      "Complement found from 8370 to 8376\n",
      "Complement found from 8436 to 8442\n",
      "Complement found from 8813 to 8819\n",
      "Complement found from 9868 to 9874\n",
      "Complement found from 12749 to 12755\n",
      "Complement found from 12913 to 12919\n",
      "Complement found from 15844 to 15850\n",
      "Seq is: CCAACT\n",
      "10 CCAACT motifs detected in file\n",
      "\n",
      "Complement found from 13812 to 13818\n",
      "Seq is: GTCACT\n",
      "1 GTCACT motifs detected in file\n",
      "\n",
      "Complement found from 3339 to 3345\n",
      "Complement found from 4227 to 4233\n",
      "Complement found from 5437 to 5443\n",
      "Complement found from 6584 to 6590\n",
      "Complement found from 7956 to 7962\n",
      "Complement found from 8029 to 8035\n",
      "Complement found from 12085 to 12091\n",
      "Complement found from 12433 to 12439\n",
      "Complement found from 13692 to 13698\n",
      "Complement found from 14629 to 14635\n",
      "Complement found from 15383 to 15389\n",
      "Seq is: CCCATT\n",
      "11 CCCATT motifs detected in file\n",
      "\n",
      "Complement found from 3764 to 3770\n",
      "Complement found from 3824 to 3830\n",
      "Complement found from 4202 to 4208\n",
      "Complement found from 9015 to 9021\n",
      "Complement found from 9613 to 9619\n",
      "Complement found from 10788 to 10794\n",
      "Complement found from 11730 to 11736\n",
      "Complement found from 13566 to 13572\n",
      "Complement found from 13737 to 13743\n",
      "Complement found from 14632 to 14638\n",
      "Complement found from 15627 to 15633\n",
      "Complement found from 16099 to 16105\n",
      "Seq is: ATTACT\n",
      "12 ATTACT motifs detected in file\n",
      "\n",
      "Complement found from 748 to 754\n",
      "Complement found from 2916 to 2922\n",
      "Complement found from 4036 to 4042\n",
      "Complement found from 13392 to 13398\n",
      "Seq is: GAACAA\n",
      "4 GAACAA motifs detected in file\n",
      "\n",
      "Complement found from 436 to 442\n",
      "Complement found from 1626 to 1632\n",
      "Complement found from 4154 to 4160\n",
      "Complement found from 8370 to 8376\n",
      "Complement found from 8436 to 8442\n",
      "Complement found from 8813 to 8819\n",
      "Complement found from 9868 to 9874\n",
      "Complement found from 12749 to 12755\n",
      "Complement found from 12913 to 12919\n",
      "Complement found from 15844 to 15850\n",
      "Seq is: CCAACT\n",
      "10 CCAACT motifs detected in file\n",
      "\n",
      "Complement found from 2505 to 2511\n",
      "Complement found from 3939 to 3945\n",
      "Complement found from 7253 to 7259\n",
      "Complement found from 7744 to 7750\n",
      "Complement found from 9894 to 9900\n",
      "Complement found from 12156 to 12162\n",
      "Complement found from 14821 to 14827\n",
      "Complement found from 15085 to 15091\n",
      "Seq is: AACATC\n",
      "8 AACATC motifs detected in file\n",
      "\n",
      "Complement found from 1806 to 1812\n",
      "Complement found from 2241 to 2247\n",
      "Complement found from 8503 to 8509\n",
      "Complement found from 11079 to 11085\n",
      "Complement found from 11568 to 11574\n",
      "Complement found from 12353 to 12359\n",
      "Complement found from 14522 to 14528\n",
      "Seq is: TATAAC\n",
      "7 TATAAC motifs detected in file\n",
      "\n",
      "Complement found from 12110 to 12116\n",
      "Complement found from 16493 to 16499\n",
      "Seq is: CCGACA\n",
      "2 CCGACA motifs detected in file\n",
      "\n",
      "Complement found from 20 to 26\n",
      "Complement found from 822 to 828\n",
      "Complement found from 909 to 915\n",
      "Complement found from 1859 to 1865\n",
      "Complement found from 2369 to 2375\n",
      "Complement found from 8314 to 8320\n",
      "Complement found from 9075 to 9081\n",
      "Complement found from 10024 to 10030\n",
      "Complement found from 11875 to 11881\n",
      "Complement found from 12025 to 12031\n",
      "Complement found from 13662 to 13668\n",
      "Complement found from 14785 to 14791\n",
      "Complement found from 15468 to 15474\n",
      "Seq is: ATTAAC\n",
      "13 ATTAAC motifs detected in file\n",
      "\n",
      "Complement found from 918 to 924\n",
      "Complement found from 2868 to 2874\n",
      "Complement found from 13257 to 13263\n",
      "Complement found from 16344 to 16350\n",
      "Seq is: AGTCAA\n",
      "4 AGTCAA motifs detected in file\n",
      "\n",
      "Complement found from 901 to 907\n",
      "Complement found from 6074 to 6080\n",
      "Complement found from 11962 to 11968\n",
      "Seq is: GTCACA\n",
      "3 GTCACA motifs detected in file\n",
      "\n",
      "Complement found from 1802 to 1808\n",
      "Complement found from 7556 to 7562\n",
      "Complement found from 8499 to 8505\n",
      "Complement found from 10444 to 10450\n",
      "Complement found from 14294 to 14300\n",
      "Complement found from 14498 to 14504\n",
      "Complement found from 14779 to 14785\n",
      "Complement found from 14964 to 14970\n",
      "Seq is: AAATTA\n",
      "8 AAATTA motifs detected in file\n",
      "\n",
      "Complement found from 5590 to 5596\n",
      "Seq is: GACTGC\n",
      "1 GACTGC motifs detected in file\n",
      "\n",
      "Complement found from 3917 to 3923\n",
      "Seq is: GTCCGA\n",
      "1 GTCCGA motifs detected in file\n",
      "\n",
      "Complement found from 3947 to 3953\n",
      "Complement found from 11488 to 11494\n",
      "Complement found from 12809 to 12815\n",
      "Complement found from 14755 to 14761\n",
      "Seq is: ATACGC\n",
      "4 ATACGC motifs detected in file\n",
      "\n",
      "Complement found from 591 to 597\n",
      "Complement found from 1494 to 1500\n",
      "Complement found from 2251 to 2257\n",
      "Complement found from 2325 to 2331\n",
      "Complement found from 3327 to 3333\n",
      "Complement found from 4928 to 4934\n",
      "Complement found from 5011 to 5017\n",
      "Complement found from 5210 to 5216\n",
      "Complement found from 6367 to 6373\n",
      "Complement found from 8775 to 8781\n",
      "Complement found from 9329 to 9335\n",
      "Complement found from 12091 to 12097\n",
      "Complement found from 12981 to 12987\n",
      "Complement found from 13986 to 13992\n",
      "Complement found from 14442 to 14448\n",
      "Complement found from 15739 to 15745\n",
      "Complement found from 16443 to 16449\n",
      "Seq is: CTCCTC\n",
      "17 CTCCTC motifs detected in file\n",
      "\n",
      "Complement found from 4231 to 4237\n",
      "Complement found from 9699 to 9705\n",
      "Complement found from 13279 to 13285\n",
      "Complement found from 14171 to 14177\n",
      "Complement found from 15182 to 15188\n",
      "Complement found from 16197 to 16203\n",
      "Seq is: TTACAA\n",
      "6 TTACAA motifs detected in file\n",
      "\n",
      "Complement found from 1304 to 1310\n",
      "Complement found from 16205 to 16211\n",
      "Seq is: AAGTAC\n",
      "2 AAGTAC motifs detected in file\n",
      "\n",
      "Complement found from 1854 to 1860\n",
      "Complement found from 4716 to 4722\n",
      "Complement found from 8525 to 8531\n",
      "Complement found from 13389 to 13395\n",
      "Complement found from 15230 to 15236\n",
      "Seq is: AATGAA\n",
      "5 AATGAA motifs detected in file\n",
      "\n",
      "Complement found from 3856 to 3862\n",
      "Complement found from 4210 to 4216\n",
      "Complement found from 6730 to 6736\n",
      "Complement found from 10448 to 10454\n",
      "Seq is: TATGAT\n",
      "4 TATGAT motifs detected in file\n",
      "\n",
      "Complement found from 1209 to 1215\n",
      "Complement found from 6091 to 6097\n",
      "Seq is: TGTAAT\n",
      "2 TGTAAT motifs detected in file\n",
      "\n",
      "Complement found from 1351 to 1357\n",
      "Complement found from 3215 to 3221\n",
      "Complement found from 3810 to 3816\n",
      "Complement found from 11308 to 11314\n",
      "Complement found from 12217 to 12223\n",
      "Complement found from 12510 to 12516\n",
      "Complement found from 14735 to 14741\n",
      "Seq is: CAAGAA\n",
      "7 CAAGAA motifs detected in file\n",
      "\n",
      "Complement found from 3178 to 3184\n",
      "Complement found from 6733 to 6739\n",
      "Complement found from 12870 to 12876\n",
      "Seq is: GATATC\n",
      "3 GATATC motifs detected in file\n",
      "\n",
      "Complement found from 1084 to 1090\n",
      "Complement found from 2217 to 2223\n",
      "Complement found from 4018 to 4024\n",
      "Complement found from 7033 to 7039\n",
      "Complement found from 10310 to 10316\n",
      "Complement found from 11020 to 11026\n",
      "Complement found from 11815 to 11821\n",
      "Complement found from 11870 to 11876\n",
      "Complement found from 12969 to 12975\n",
      "Complement found from 13146 to 13152\n",
      "Complement found from 13881 to 13887\n",
      "Complement found from 14390 to 14396\n",
      "Complement found from 15700 to 15706\n",
      "Complement found from 16266 to 16272\n",
      "Seq is: CCACTA\n",
      "14 CCACTA motifs detected in file\n",
      "\n",
      "Complement found from 4100 to 4106\n",
      "Complement found from 4597 to 4603\n",
      "Complement found from 5390 to 5396\n",
      "Complement found from 7168 to 7174\n",
      "Complement found from 7285 to 7291\n",
      "Complement found from 12244 to 12250\n",
      "Complement found from 13160 to 13166\n",
      "Complement found from 13829 to 13835\n",
      "Complement found from 15747 to 15753\n",
      "Seq is: TCTAAC\n",
      "9 TCTAAC motifs detected in file\n",
      "\n",
      "Complement found from 5769 to 5775\n",
      "Complement found from 6273 to 6279\n",
      "Complement found from 13495 to 13501\n",
      "Seq is: CAGGTT\n",
      "3 CAGGTT motifs detected in file\n",
      "\n",
      "Complement found from 912 to 918\n",
      "Complement found from 1686 to 1692\n",
      "Complement found from 2423 to 2429\n",
      "Complement found from 2767 to 2773\n",
      "Complement found from 2831 to 2837\n",
      "Complement found from 4346 to 4352\n",
      "Complement found from 4984 to 4990\n",
      "Complement found from 5430 to 5436\n",
      "Complement found from 5674 to 5680\n",
      "Complement found from 6430 to 6436\n",
      "Complement found from 9240 to 9246\n",
      "Complement found from 12545 to 12551\n",
      "Complement found from 12554 to 12560\n",
      "Complement found from 14069 to 14075\n",
      "Complement found from 14516 to 14522\n",
      "Complement found from 14639 to 14645\n",
      "Complement found from 16164 to 16170\n",
      "Seq is: AACCCA\n",
      "17 AACCCA motifs detected in file\n",
      "\n",
      "Complement found from 462 to 468\n",
      "Complement found from 1690 to 1696\n",
      "Complement found from 6879 to 6885\n",
      "Complement found from 9317 to 9323\n",
      "Complement found from 9512 to 9518\n",
      "Complement found from 9594 to 9600\n",
      "Complement found from 10622 to 10628\n",
      "Complement found from 12909 to 12915\n",
      "Complement found from 14849 to 14855\n",
      "Complement found from 15326 to 15332\n",
      "Seq is: CACTCC\n",
      "10 CACTCC motifs detected in file\n",
      "\n",
      "Complement found from 1498 to 1504\n",
      "Complement found from 4638 to 4644\n",
      "Complement found from 13254 to 13260\n",
      "Seq is: TCAAGT\n",
      "3 TCAAGT motifs detected in file\n",
      "\n",
      "Complement found from 173 to 179\n",
      "Complement found from 2379 to 2385\n",
      "Complement found from 4699 to 4705\n",
      "Complement found from 5802 to 5808\n",
      "Complement found from 6409 to 6415\n",
      "Complement found from 6892 to 6898\n",
      "Complement found from 9064 to 9070\n",
      "Complement found from 10636 to 10642\n",
      "Complement found from 12490 to 12496\n",
      "Complement found from 14174 to 14180\n",
      "Complement found from 15008 to 15014\n",
      "Complement found from 16418 to 16424\n",
      "Seq is: CAATAT\n",
      "12 CAATAT motifs detected in file\n",
      "\n",
      "Complement found from 1990 to 1996\n",
      "Complement found from 2764 to 2770\n",
      "Complement found from 3990 to 3996\n",
      "Complement found from 5679 to 5685\n",
      "Complement found from 5930 to 5936\n",
      "Complement found from 8455 to 8461\n",
      "Complement found from 8507 to 8513\n",
      "Complement found from 10294 to 10300\n",
      "Complement found from 11592 to 11598\n",
      "Complement found from 13545 to 13551\n",
      "Complement found from 13861 to 13867\n",
      "Complement found from 14188 to 14194\n",
      "Complement found from 14623 to 14629\n",
      "Complement found from 15184 to 15190\n",
      "Complement found from 15602 to 15608\n",
      "Complement found from 16280 to 16286\n",
      "Seq is: ACAAAC\n",
      "16 ACAAAC motifs detected in file\n",
      "\n",
      "Complement found from 1098 to 1104\n",
      "Complement found from 2174 to 2180\n",
      "Complement found from 2223 to 2229\n",
      "Complement found from 2778 to 2784\n",
      "Complement found from 3485 to 3491\n",
      "Complement found from 4396 to 4402\n",
      "Complement found from 8229 to 8235\n",
      "Complement found from 9598 to 9604\n",
      "Complement found from 10744 to 10750\n",
      "Complement found from 11407 to 11413\n",
      "Complement found from 11601 to 11607\n",
      "Complement found from 13790 to 13796\n",
      "Complement found from 14022 to 14028\n",
      "Complement found from 14491 to 14497\n",
      "Complement found from 16526 to 16532\n",
      "Seq is: CCTAAA\n",
      "15 CCTAAA motifs detected in file\n",
      "\n",
      "Complement found from 2071 to 2077\n",
      "Complement found from 2229 to 2235\n",
      "Complement found from 7134 to 7140\n",
      "Complement found from 7568 to 7574\n",
      "Complement found from 9577 to 9583\n",
      "Complement found from 10154 to 10160\n",
      "Complement found from 12444 to 12450\n",
      "Complement found from 13875 to 13881\n",
      "Complement found from 16348 to 16354\n",
      "Seq is: AAATCC\n",
      "9 AAATCC motifs detected in file\n",
      "\n",
      "Complement found from 12 to 18\n",
      "Complement found from 632 to 638\n",
      "Complement found from 731 to 737\n",
      "Complement found from 951 to 957\n",
      "Complement found from 2277 to 2283\n",
      "Complement found from 2508 to 2514\n",
      "Complement found from 2520 to 2526\n",
      "Complement found from 3507 to 3513\n",
      "Complement found from 3519 to 3525\n",
      "Complement found from 5321 to 5327\n",
      "Complement found from 5812 to 5818\n",
      "Complement found from 7645 to 7651\n",
      "Complement found from 8431 to 8437\n",
      "Complement found from 8652 to 8658\n",
      "Complement found from 9566 to 9572\n",
      "Complement found from 9635 to 9641\n",
      "Complement found from 11155 to 11161\n",
      "Complement found from 13182 to 13188\n",
      "Complement found from 14061 to 14067\n",
      "Complement found from 14878 to 14884\n",
      "Complement found from 15376 to 15382\n",
      "Complement found from 15397 to 15403\n",
      "Seq is: ATCACC\n",
      "22 ATCACC motifs detected in file\n",
      "\n",
      "Complement found from 3803 to 3809\n",
      "Complement found from 4070 to 4076\n",
      "Complement found from 6397 to 6403\n",
      "Complement found from 7182 to 7188\n",
      "Complement found from 7722 to 7728\n",
      "Complement found from 8706 to 8712\n",
      "Complement found from 8765 to 8771\n",
      "Complement found from 10131 to 10137\n",
      "Complement found from 10836 to 10842\n",
      "Complement found from 11260 to 11266\n",
      "Complement found from 11994 to 12000\n",
      "Complement found from 12485 to 12491\n",
      "Complement found from 12542 to 12548\n",
      "Complement found from 12928 to 12934\n",
      "Complement found from 13377 to 13383\n",
      "Complement found from 14328 to 14334\n",
      "Complement found from 15822 to 15828\n",
      "Seq is: CACAAC\n",
      "17 CACAAC motifs detected in file\n",
      "\n",
      "Complement found from 4124 to 4130\n",
      "Complement found from 4343 to 4349\n",
      "Complement found from 7460 to 7466\n",
      "Complement found from 12525 to 12531\n",
      "Seq is: TCGAAC\n",
      "4 TCGAAC motifs detected in file\n",
      "\n",
      "Complement found from 87 to 93\n",
      "Seq is: TTGCGA\n",
      "1 TTGCGA motifs detected in file\n",
      "\n",
      "Complement found from 5271 to 5277\n",
      "Seq is: AAGAAT\n",
      "1 AAGAAT motifs detected in file\n",
      "\n",
      "Complement found from 22 to 28\n",
      "Complement found from 382 to 388\n",
      "Complement found from 1808 to 1814\n",
      "Complement found from 2666 to 2672\n",
      "Complement found from 4599 to 4605\n",
      "Complement found from 4725 to 4731\n",
      "Complement found from 5067 to 5073\n",
      "Complement found from 6336 to 6342\n",
      "Complement found from 8698 to 8704\n",
      "Complement found from 9348 to 9354\n",
      "Complement found from 9360 to 9366\n",
      "Complement found from 10884 to 10890\n",
      "Complement found from 11906 to 11912\n",
      "Complement found from 12149 to 12155\n",
      "Complement found from 12337 to 12343\n",
      "Complement found from 12355 to 12361\n",
      "Complement found from 13855 to 13861\n",
      "Complement found from 14787 to 14793\n",
      "Seq is: TAACCA\n",
      "18 TAACCA motifs detected in file\n",
      "\n",
      "Complement found from 7213 to 7219\n",
      "Complement found from 9771 to 9777\n",
      "Seq is: CCGACG\n",
      "2 CCGACG motifs detected in file\n",
      "\n",
      "Complement found from 2228 to 2234\n",
      "Complement found from 4997 to 5003\n",
      "Complement found from 5809 to 5815\n",
      "Complement found from 7133 to 7139\n",
      "Complement found from 7435 to 7441\n",
      "Complement found from 8233 to 8239\n",
      "Complement found from 8533 to 8539\n",
      "Complement found from 10153 to 10159\n",
      "Complement found from 11604 to 11610\n",
      "Complement found from 12443 to 12449\n",
      "Complement found from 13233 to 13239\n",
      "Complement found from 13874 to 13880\n",
      "Complement found from 15394 to 15400\n",
      "Seq is: AAAATC\n",
      "13 AAAATC motifs detected in file\n",
      "\n",
      "Seq is: CGGATG\n",
      "0 CGGATG motifs detected in file\n",
      "\n",
      "Complement found from 537 to 543\n",
      "Complement found from 1079 to 1085\n",
      "Complement found from 3200 to 3206\n",
      "Complement found from 3306 to 3312\n",
      "Complement found from 3576 to 3582\n",
      "Complement found from 4133 to 4139\n",
      "Complement found from 4224 to 4230\n",
      "Complement found from 4432 to 4438\n",
      "Complement found from 4452 to 4458\n",
      "Complement found from 6113 to 6119\n",
      "Complement found from 12428 to 12434\n",
      "Complement found from 15512 to 15518\n",
      "Seq is: ATACCC\n",
      "12 ATACCC motifs detected in file\n",
      "\n",
      "Complement found from 240 to 246\n",
      "Complement found from 2366 to 2372\n",
      "Complement found from 6401 to 6407\n",
      "Complement found from 9105 to 9111\n",
      "Complement found from 9701 to 9707\n",
      "Complement found from 10780 to 10786\n",
      "Complement found from 14028 to 14034\n",
      "Complement found from 15506 to 15512\n",
      "Complement found from 15580 to 15586\n",
      "Seq is: ACAATT\n",
      "9 ACAATT motifs detected in file\n",
      "\n",
      "Complement found from 991 to 997\n",
      "Complement found from 2149 to 2155\n",
      "Complement found from 2225 to 2231\n",
      "Complement found from 2455 to 2461\n",
      "Complement found from 2799 to 2805\n",
      "Complement found from 5400 to 5406\n",
      "Complement found from 5828 to 5834\n",
      "Complement found from 8093 to 8099\n",
      "Complement found from 8231 to 8237\n",
      "Complement found from 8441 to 8447\n",
      "Complement found from 8489 to 8495\n",
      "Complement found from 8495 to 8501\n",
      "Complement found from 8888 to 8894\n",
      "Complement found from 14502 to 14508\n",
      "Complement found from 16160 to 16166\n",
      "Seq is: TAAAAA\n",
      "15 TAAAAA motifs detected in file\n",
      "\n",
      "Complement found from 4336 to 4342\n",
      "Complement found from 12920 to 12926\n",
      "Seq is: ATGAGA\n",
      "2 ATGAGA motifs detected in file\n",
      "\n",
      "Complement found from 15927 to 15933\n",
      "Seq is: GAGATG\n",
      "1 GAGATG motifs detected in file\n",
      "\n",
      "Complement found from 10019 to 10025\n",
      "Complement found from 10809 to 10815\n",
      "Complement found from 12747 to 12753\n",
      "Complement found from 13767 to 13773\n",
      "Complement found from 15942 to 15948\n",
      "Seq is: TTCCAA\n",
      "5 TTCCAA motifs detected in file\n",
      "\n",
      "Complement found from 743 to 749\n",
      "Complement found from 2465 to 2471\n",
      "Complement found from 5748 to 5754\n",
      "Complement found from 7448 to 7454\n",
      "Complement found from 7515 to 7521\n",
      "Complement found from 9433 to 9439\n",
      "Complement found from 10381 to 10387\n",
      "Complement found from 14075 to 14081\n",
      "Seq is: AAAAGG\n",
      "8 AAAAGG motifs detected in file\n",
      "\n",
      "Complement found from 1 to 7\n",
      "Complement found from 3801 to 3807\n",
      "Complement found from 4384 to 4390\n",
      "Complement found from 4523 to 4529\n",
      "Complement found from 5372 to 5378\n",
      "Complement found from 6395 to 6401\n",
      "Complement found from 6470 to 6476\n",
      "Complement found from 8049 to 8055\n",
      "Complement found from 9217 to 9223\n",
      "Complement found from 13929 to 13935\n",
      "Complement found from 14139 to 14145\n",
      "Complement found from 16229 to 16235\n",
      "Seq is: ATCACA\n",
      "12 ATCACA motifs detected in file\n",
      "\n",
      "Complement found from 3093 to 3099\n",
      "Complement found from 7477 to 7483\n",
      "Complement found from 12876 to 12882\n",
      "Complement found from 13497 to 13503\n",
      "Seq is: GGTTTC\n",
      "4 GGTTTC motifs detected in file\n",
      "\n",
      "Seq is: TCGACA\n",
      "0 TCGACA motifs detected in file\n",
      "\n",
      "Complement found from 1549 to 1555\n",
      "Complement found from 1639 to 1645\n",
      "Complement found from 1881 to 1887\n",
      "Complement found from 6577 to 6583\n",
      "Complement found from 14598 to 14604\n",
      "Seq is: AGGAGA\n",
      "5 AGGAGA motifs detected in file\n",
      "\n",
      "Complement found from 1761 to 1767\n",
      "Complement found from 2790 to 2796\n",
      "Complement found from 7373 to 7379\n",
      "Complement found from 8723 to 8729\n",
      "Complement found from 10304 to 10310\n",
      "Complement found from 13974 to 13980\n",
      "Complement found from 14000 to 14006\n",
      "Complement found from 15078 to 15084\n",
      "Complement found from 15750 to 15756\n",
      "Seq is: AACCTG\n",
      "9 AACCTG motifs detected in file\n",
      "\n",
      "Complement found from 12450 to 12456\n",
      "Seq is: ATTGTC\n",
      "1 ATTGTC motifs detected in file\n",
      "\n",
      "Complement found from 2084 to 2090\n",
      "Complement found from 2154 to 2160\n",
      "Complement found from 2739 to 2745\n",
      "Complement found from 5078 to 5084\n",
      "Complement found from 5514 to 5520\n",
      "Complement found from 11250 to 11256\n",
      "Complement found from 16005 to 16011\n",
      "Seq is: AATTTA\n",
      "7 AATTTA motifs detected in file\n",
      "\n",
      "Complement found from 613 to 619\n",
      "Complement found from 2296 to 2302\n",
      "Complement found from 4442 to 4448\n",
      "Complement found from 14194 to 14200\n",
      "Seq is: AATGTT\n",
      "4 AATGTT motifs detected in file\n",
      "\n",
      "Complement found from 1530 to 1536\n",
      "Complement found from 1836 to 1842\n",
      "Complement found from 3428 to 3434\n",
      "Complement found from 3483 to 3489\n",
      "Complement found from 6152 to 6158\n",
      "Complement found from 7094 to 7100\n",
      "Complement found from 7625 to 7631\n",
      "Complement found from 8227 to 8233\n",
      "Complement found from 9254 to 9260\n",
      "Complement found from 9525 to 9531\n",
      "Complement found from 9581 to 9587\n",
      "Complement found from 10277 to 10283\n",
      "Complement found from 10936 to 10942\n",
      "Complement found from 11233 to 11239\n",
      "Complement found from 13128 to 13134\n",
      "Complement found from 13946 to 13952\n",
      "Complement found from 13980 to 13986\n",
      "Complement found from 14489 to 14495\n",
      "Complement found from 14770 to 14776\n",
      "Complement found from 15367 to 15373\n",
      "Seq is: CCCCTA\n",
      "20 CCCCTA motifs detected in file\n",
      "\n",
      "Complement found from 477 to 483\n",
      "Complement found from 3722 to 3728\n",
      "Complement found from 4235 to 4241\n",
      "Complement found from 4900 to 4906\n",
      "Complement found from 6661 to 6667\n",
      "Complement found from 10708 to 10714\n",
      "Complement found from 11055 to 11061\n",
      "Complement found from 11067 to 11073\n",
      "Complement found from 14045 to 14051\n",
      "Complement found from 14163 to 14169\n",
      "Seq is: AATCTC\n",
      "10 AATCTC motifs detected in file\n",
      "\n",
      "Complement found from 2023 to 2029\n",
      "Complement found from 3214 to 3220\n",
      "Complement found from 4087 to 4093\n",
      "Complement found from 5533 to 5539\n",
      "Complement found from 11307 to 11313\n",
      "Complement found from 12509 to 12515\n",
      "Complement found from 14405 to 14411\n",
      "Seq is: CCAAGA\n",
      "7 CCAAGA motifs detected in file\n",
      "\n",
      "Complement found from 1943 to 1949\n",
      "Complement found from 11157 to 11163\n",
      "Complement found from 14554 to 14560\n",
      "Seq is: CACCCG\n",
      "3 CACCCG motifs detected in file\n",
      "\n",
      "Complement found from 2046 to 2052\n",
      "Complement found from 10414 to 10420\n",
      "Complement found from 12268 to 12274\n",
      "Complement found from 16007 to 16013\n",
      "Seq is: TTTAAA\n",
      "4 TTTAAA motifs detected in file\n",
      "\n",
      "Complement found from 390 to 396\n",
      "Complement found from 1645 to 1651\n",
      "Complement found from 7479 to 7485\n",
      "Complement found from 7510 to 7516\n",
      "Complement found from 14726 to 14732\n",
      "Seq is: TTTCAA\n",
      "5 TTTCAA motifs detected in file\n",
      "\n",
      "Seq is: CTTGAT\n",
      "0 CTTGAT motifs detected in file\n",
      "\n",
      "Complement found from 5998 to 6004\n",
      "Complement found from 8880 to 8886\n",
      "Seq is: TCTAAG\n",
      "2 TCTAAG motifs detected in file\n",
      "\n",
      "Complement found from 923 to 929\n",
      "Complement found from 1750 to 1756\n",
      "Complement found from 2028 to 2034\n",
      "Complement found from 2285 to 2291\n",
      "Complement found from 5509 to 5515\n",
      "Complement found from 7633 to 7639\n",
      "Complement found from 7765 to 7771\n",
      "Complement found from 9659 to 9665\n",
      "Complement found from 10148 to 10154\n",
      "Complement found from 12636 to 12642\n",
      "Complement found from 13044 to 13050\n",
      "Seq is: ATAGAA\n",
      "11 ATAGAA motifs detected in file\n",
      "\n",
      "Complement found from 1202 to 1208\n",
      "Complement found from 2492 to 2498\n",
      "Complement found from 4109 to 4115\n",
      "Seq is: CCTGTT\n",
      "3 CCTGTT motifs detected in file\n",
      "\n",
      "Complement found from 1410 to 1416\n",
      "Seq is: GGTCGA\n",
      "1 GGTCGA motifs detected in file\n",
      "\n",
      "Complement found from 1963 to 1969\n",
      "Complement found from 6252 to 6258\n",
      "Seq is: TAGTGG\n",
      "2 TAGTGG motifs detected in file\n",
      "\n",
      "Complement found from 2840 to 2846\n",
      "Seq is: TCCGAG\n",
      "1 TCCGAG motifs detected in file\n",
      "\n",
      "Complement found from 500 to 506\n",
      "Complement found from 4349 to 4355\n",
      "Complement found from 4392 to 4398\n",
      "Complement found from 5201 to 5207\n",
      "Complement found from 7819 to 7825\n",
      "Complement found from 8839 to 8845\n",
      "Complement found from 12388 to 12394\n",
      "Complement found from 14815 to 14821\n",
      "Complement found from 15200 to 15206\n",
      "Complement found from 15635 to 15641\n",
      "Complement found from 15659 to 15665\n",
      "Complement found from 16402 to 16408\n",
      "Seq is: CCATCC\n",
      "12 CCATCC motifs detected in file\n",
      "\n",
      "Complement found from 811 to 817\n",
      "Complement found from 2752 to 2758\n",
      "Complement found from 8096 to 8102\n",
      "Complement found from 11594 to 11600\n",
      "Seq is: AAACAG\n",
      "4 AAACAG motifs detected in file\n",
      "\n",
      "Complement found from 15925 to 15931\n",
      "Seq is: CGGAGA\n",
      "1 CGGAGA motifs detected in file\n",
      "\n",
      "Complement found from 6706 to 6712\n",
      "Seq is: TGGATA\n",
      "1 TGGATA motifs detected in file\n",
      "\n",
      "Complement found from 617 to 623\n",
      "Complement found from 946 to 952\n",
      "Complement found from 5851 to 5857\n",
      "Seq is: TTTAGA\n",
      "3 TTTAGA motifs detected in file\n",
      "\n",
      "Complement found from 3662 to 3668\n",
      "Seq is: AGGGTG\n",
      "1 AGGGTG motifs detected in file\n",
      "\n",
      "Complement found from 3256 to 3262\n",
      "Complement found from 3462 to 3468\n",
      "Complement found from 4275 to 4281\n",
      "Complement found from 4778 to 4784\n",
      "Complement found from 5405 to 5411\n",
      "Complement found from 6413 to 6419\n",
      "Complement found from 7433 to 7439\n",
      "Complement found from 8488 to 8494\n",
      "Complement found from 8494 to 8500\n",
      "Complement found from 10214 to 10220\n",
      "Complement found from 12034 to 12040\n",
      "Complement found from 12327 to 12333\n",
      "Complement found from 13872 to 13878\n",
      "Complement found from 14776 to 14782\n",
      "Complement found from 15392 to 15398\n",
      "Complement found from 16159 to 16165\n",
      "Seq is: ATAAAA\n",
      "16 ATAAAA motifs detected in file\n",
      "\n",
      "Complement found from 2987 to 2993\n",
      "Complement found from 9382 to 9388\n",
      "Seq is: CGATGT\n",
      "2 CGATGT motifs detected in file\n",
      "\n",
      "Complement found from 6244 to 6250\n",
      "Complement found from 6904 to 6910\n",
      "Complement found from 7684 to 7690\n",
      "Complement found from 9854 to 9860\n",
      "Complement found from 11581 to 11587\n",
      "Complement found from 15019 to 15025\n",
      "Seq is: ATCTGC\n",
      "6 ATCTGC motifs detected in file\n",
      "\n",
      "Complement found from 267 to 273\n",
      "Complement found from 796 to 802\n",
      "Complement found from 3204 to 3210\n",
      "Complement found from 3868 to 3874\n",
      "Complement found from 5448 to 5454\n",
      "Complement found from 6876 to 6882\n",
      "Complement found from 7408 to 7414\n",
      "Complement found from 9415 to 9421\n",
      "Complement found from 11129 to 11135\n",
      "Complement found from 13299 to 13305\n",
      "Complement found from 14561 to 14567\n",
      "Complement found from 14642 to 14648\n",
      "Complement found from 16536 to 16542\n",
      "Seq is: CCACAC\n",
      "13 CCACAC motifs detected in file\n",
      "\n",
      "Complement found from 3136 to 3142\n",
      "Complement found from 9393 to 9399\n",
      "Complement found from 12053 to 12059\n",
      "Complement found from 12206 to 12212\n",
      "Complement found from 15958 to 15964\n",
      "Seq is: GAGAAA\n",
      "5 GAGAAA motifs detected in file\n",
      "\n",
      "Complement found from 14800 to 14806\n",
      "Seq is: ATCGAC\n",
      "1 ATCGAC motifs detected in file\n",
      "\n",
      "Complement found from 5271 to 5277\n",
      "Seq is: AAGAAT\n",
      "1 AAGAAT motifs detected in file\n",
      "\n",
      "Complement found from 3544 to 3550\n",
      "Complement found from 8207 to 8213\n",
      "Complement found from 11429 to 11435\n",
      "Complement found from 14378 to 14384\n",
      "Complement found from 14453 to 14459\n",
      "Complement found from 14715 to 14721\n",
      "Seq is: CCATCG\n",
      "6 CCATCG motifs detected in file\n",
      "\n",
      "Complement found from 2826 to 2832\n",
      "Complement found from 3875 to 3881\n",
      "Complement found from 12818 to 12824\n",
      "Complement found from 13133 to 13139\n",
      "Complement found from 16037 to 16043\n",
      "Seq is: AGCAGA\n",
      "5 AGCAGA motifs detected in file\n",
      "\n",
      "Complement found from 558 to 564\n",
      "Complement found from 1888 to 1894\n",
      "Complement found from 2099 to 2105\n",
      "Complement found from 3475 to 3481\n",
      "Complement found from 7469 to 7475\n",
      "Complement found from 8479 to 8485\n",
      "Complement found from 13507 to 13513\n",
      "Complement found from 14470 to 14476\n",
      "Complement found from 15989 to 15995\n",
      "Complement found from 16249 to 16255\n",
      "Seq is: CCAAAG\n",
      "10 CCAAAG motifs detected in file\n",
      "\n",
      "Complement found from 433 to 439\n",
      "Complement found from 568 to 574\n",
      "Complement found from 801 to 807\n",
      "Complement found from 4878 to 4884\n",
      "Complement found from 5894 to 5900\n",
      "Complement found from 7397 to 7403\n",
      "Complement found from 7466 to 7472\n",
      "Complement found from 7954 to 7960\n",
      "Complement found from 8027 to 8033\n",
      "Complement found from 8405 to 8411\n",
      "Complement found from 8557 to 8563\n",
      "Complement found from 9532 to 9538\n",
      "Complement found from 9554 to 9560\n",
      "Complement found from 11426 to 11432\n",
      "Complement found from 11867 to 11873\n",
      "Complement found from 12083 to 12089\n",
      "Complement found from 12236 to 12242\n",
      "Complement found from 12385 to 12391\n",
      "Complement found from 12431 to 12437\n",
      "Complement found from 14424 to 14430\n",
      "Complement found from 14530 to 14536\n",
      "Complement found from 14587 to 14593\n",
      "Seq is: CCCCCA\n",
      "22 CCCCCA motifs detected in file\n",
      "\n",
      "Complement found from 4340 to 4346\n",
      "Complement found from 7457 to 7463\n",
      "Complement found from 15755 to 15761\n",
      "Seq is: GAATCG\n",
      "3 GAATCG motifs detected in file\n",
      "\n",
      "Complement found from 7769 to 7775\n",
      "Complement found from 13525 to 13531\n",
      "Complement found from 15921 to 15927\n",
      "Seq is: AAACCG\n",
      "3 AAACCG motifs detected in file\n",
      "\n",
      "Complement found from 1195 to 1201\n",
      "Complement found from 1547 to 1553\n",
      "Complement found from 2103 to 2109\n",
      "Seq is: AGAGGA\n",
      "3 AGAGGA motifs detected in file\n",
      "\n",
      "Complement found from 9160 to 9166\n",
      "Seq is: ACGTTT\n",
      "1 ACGTTT motifs detected in file\n",
      "\n",
      "Complement found from 834 to 840\n",
      "Complement found from 964 to 970\n",
      "Complement found from 2899 to 2905\n",
      "Complement found from 4776 to 4782\n",
      "Complement found from 15650 to 15656\n",
      "Seq is: CAATAA\n",
      "5 CAATAA motifs detected in file\n",
      "\n",
      "Complement found from 949 to 955\n",
      "Seq is: AGATCA\n",
      "1 AGATCA motifs detected in file\n",
      "\n",
      "Complement found from 2191 to 2197\n",
      "Complement found from 5639 to 5645\n",
      "Complement found from 8334 to 8340\n",
      "Seq is: ATTAAG\n",
      "3 ATTAAG motifs detected in file\n",
      "\n",
      "Complement found from 6111 to 6117\n",
      "Complement found from 7737 to 7743\n",
      "Complement found from 10575 to 10581\n",
      "Complement found from 10955 to 10961\n",
      "Complement found from 11486 to 11492\n",
      "Complement found from 15839 to 15845\n",
      "Complement found from 15904 to 15910\n",
      "Seq is: TAATAC\n",
      "7 TAATAC motifs detected in file\n",
      "\n",
      "Seq is: GTCTTC\n",
      "0 GTCTTC motifs detected in file\n",
      "\n",
      "Complement found from 1037 to 1043\n",
      "Complement found from 1184 to 1190\n",
      "Complement found from 2961 to 2967\n",
      "Complement found from 5386 to 5392\n",
      "Complement found from 10546 to 10552\n",
      "Complement found from 13535 to 13541\n",
      "Seq is: CATATC\n",
      "6 CATATC motifs detected in file\n",
      "\n",
      "Complement found from 962 to 968\n",
      "Complement found from 2260 to 2266\n",
      "Complement found from 2377 to 2383\n",
      "Complement found from 6432 to 6438\n",
      "Complement found from 9534 to 9540\n",
      "Complement found from 13009 to 13015\n",
      "Complement found from 14751 to 14757\n",
      "Complement found from 16166 to 16172\n",
      "Seq is: CCCAAT\n",
      "8 CCCAAT motifs detected in file\n",
      "\n",
      "Complement found from 6267 to 6273\n",
      "Complement found from 7759 to 7765\n",
      "Complement found from 13099 to 13105\n",
      "Complement found from 13477 to 13483\n",
      "Seq is: CAGGAA\n",
      "4 CAGGAA motifs detected in file\n",
      "\n",
      "Complement found from 6172 to 6178\n",
      "Complement found from 12869 to 12875\n",
      "Seq is: CGATAT\n",
      "2 CGATAT motifs detected in file\n",
      "\n",
      "Complement found from 3946 to 3952\n",
      "Complement found from 11487 to 11493\n",
      "Complement found from 14754 to 14760\n",
      "Seq is: AATACG\n",
      "3 AATACG motifs detected in file\n",
      "\n",
      "Complement found from 545 to 551\n",
      "Complement found from 1809 to 1815\n",
      "Complement found from 2393 to 2399\n",
      "Complement found from 4600 to 4606\n",
      "Complement found from 4726 to 4732\n",
      "Complement found from 4979 to 4985\n",
      "Complement found from 8122 to 8128\n",
      "Complement found from 8342 to 8348\n",
      "Complement found from 8519 to 8525\n",
      "Complement found from 8980 to 8986\n",
      "Complement found from 9349 to 9355\n",
      "Complement found from 9674 to 9680\n",
      "Complement found from 10885 to 10891\n",
      "Complement found from 12150 to 12156\n",
      "Complement found from 13293 to 13299\n",
      "Complement found from 13856 to 13862\n",
      "Seq is: AACCAA\n",
      "16 AACCAA motifs detected in file\n",
      "\n",
      "Complement found from 19 to 25\n",
      "Complement found from 2743 to 2749\n",
      "Complement found from 3023 to 3029\n",
      "Complement found from 7299 to 7305\n",
      "Complement found from 8447 to 8453\n",
      "Complement found from 10335 to 10341\n",
      "Complement found from 11874 to 11880\n",
      "Complement found from 14315 to 14321\n",
      "Complement found from 14511 to 14517\n",
      "Seq is: TATTAA\n",
      "9 TATTAA motifs detected in file\n",
      "\n",
      "Complement found from 1293 to 1299\n",
      "Complement found from 1738 to 1744\n",
      "Complement found from 2460 to 2466\n",
      "Complement found from 4399 to 4405\n",
      "Complement found from 6857 to 6863\n",
      "Complement found from 12330 to 12336\n",
      "Seq is: AAAGTA\n",
      "6 AAAGTA motifs detected in file\n",
      "\n",
      "Complement found from 2448 to 2454\n",
      "Complement found from 3126 to 3132\n",
      "Seq is: GAAAGG\n",
      "2 GAAAGG motifs detected in file\n",
      "\n",
      "Complement found from 5618 to 5624\n",
      "Complement found from 11179 to 11185\n",
      "Complement found from 11187 to 11193\n",
      "Complement found from 11761 to 11767\n",
      "Seq is: GAACGC\n",
      "4 GAACGC motifs detected in file\n",
      "\n",
      "Complement found from 7312 to 7318\n",
      "Complement found from 7654 to 7660\n",
      "Complement found from 12895 to 12901\n",
      "Complement found from 14831 to 14837\n",
      "Seq is: CATGAT\n",
      "4 CATGAT motifs detected in file\n",
      "\n",
      "Complement found from 1772 to 1778\n",
      "Seq is: ATAGAT\n",
      "1 ATAGAT motifs detected in file\n",
      "\n",
      "Seq is: AACGCG\n",
      "0 AACGCG motifs detected in file\n",
      "\n",
      "Complement found from 2021 to 2027\n",
      "Complement found from 2097 to 2103\n",
      "Complement found from 5863 to 5869\n",
      "Complement found from 9428 to 9434\n",
      "Seq is: GTCCAA\n",
      "4 GTCCAA motifs detected in file\n",
      "\n",
      "Complement found from 1111 to 1117\n",
      "Complement found from 5689 to 5695\n",
      "Complement found from 7552 to 7558\n",
      "Complement found from 8326 to 8332\n",
      "Seq is: AGTTAA\n",
      "4 AGTTAA motifs detected in file\n",
      "\n",
      "Complement found from 2026 to 2032\n",
      "Complement found from 16382 to 16388\n",
      "Seq is: AGATAG\n",
      "2 AGATAG motifs detected in file\n",
      "\n",
      "Seq is: CGGATA\n",
      "0 CGGATA motifs detected in file\n",
      "\n",
      "Complement found from 2673 to 2679\n",
      "Complement found from 6897 to 6903\n",
      "Complement found from 8175 to 8181\n",
      "Complement found from 8241 to 8247\n",
      "Complement found from 8361 to 8367\n",
      "Complement found from 16412 to 16418\n",
      "Seq is: TGAAAT\n",
      "6 TGAAAT motifs detected in file\n",
      "\n",
      "Complement found from 299 to 305\n",
      "Complement found from 349 to 355\n",
      "Complement found from 553 to 559\n",
      "Complement found from 1219 to 1225\n",
      "Complement found from 1274 to 1280\n",
      "Complement found from 1527 to 1533\n",
      "Complement found from 1685 to 1691\n",
      "Complement found from 2766 to 2772\n",
      "Complement found from 3489 to 3495\n",
      "Complement found from 4314 to 4320\n",
      "Complement found from 4612 to 4618\n",
      "Complement found from 4983 to 4989\n",
      "Complement found from 5429 to 5435\n",
      "Complement found from 5597 to 5603\n",
      "Complement found from 5673 to 5679\n",
      "Complement found from 6416 to 6422\n",
      "Complement found from 8509 to 8515\n",
      "Complement found from 9239 to 9245\n",
      "Complement found from 11669 to 11675\n",
      "Complement found from 12037 to 12043\n",
      "Complement found from 13688 to 13694\n",
      "Complement found from 14515 to 14521\n",
      "Complement found from 14584 to 14590\n",
      "Complement found from 14616 to 14622\n",
      "Complement found from 14625 to 14631\n",
      "Complement found from 14638 to 14644\n",
      "Complement found from 16163 to 16169\n",
      "Complement found from 16180 to 16186\n",
      "Seq is: AAACCC\n",
      "28 AAACCC motifs detected in file\n",
      "\n",
      "Complement found from 1726 to 1732\n",
      "Complement found from 5856 to 5862\n",
      "Complement found from 6778 to 6784\n",
      "Complement found from 8256 to 8262\n",
      "Complement found from 8798 to 8804\n",
      "Complement found from 10460 to 10466\n",
      "Complement found from 10478 to 10484\n",
      "Complement found from 10501 to 10507\n",
      "Complement found from 11251 to 11257\n",
      "Complement found from 11988 to 11994\n",
      "Complement found from 12199 to 12205\n",
      "Complement found from 16321 to 16327\n",
      "Seq is: ATTTAC\n",
      "12 ATTTAC motifs detected in file\n",
      "\n",
      "Complement found from 298 to 304\n",
      "Complement found from 348 to 354\n",
      "Complement found from 552 to 558\n",
      "Complement found from 1273 to 1279\n",
      "Complement found from 1684 to 1690\n",
      "Complement found from 1720 to 1726\n",
      "Complement found from 1991 to 1997\n",
      "Complement found from 2765 to 2771\n",
      "Complement found from 2788 to 2794\n",
      "Complement found from 4254 to 4260\n",
      "Complement found from 4982 to 4988\n",
      "Complement found from 5931 to 5937\n",
      "Complement found from 7121 to 7127\n",
      "Complement found from 8125 to 8131\n",
      "Complement found from 8189 to 8195\n",
      "Complement found from 8508 to 8514\n",
      "Complement found from 11668 to 11674\n",
      "Complement found from 14624 to 14630\n",
      "Complement found from 16281 to 16287\n",
      "Seq is: CAAACC\n",
      "19 CAAACC motifs detected in file\n",
      "\n",
      "Complement found from 135 to 141\n",
      "Complement found from 4141 to 4147\n",
      "Seq is: GATTCC\n",
      "2 GATTCC motifs detected in file\n",
      "\n",
      "Complement found from 6532 to 6538\n",
      "Complement found from 7842 to 7848\n",
      "Complement found from 8097 to 8103\n",
      "Complement found from 11595 to 11601\n",
      "Complement found from 12178 to 12184\n",
      "Complement found from 14650 to 14656\n",
      "Seq is: AACAGA\n",
      "6 AACAGA motifs detected in file\n",
      "\n",
      "Complement found from 3505 to 3511\n",
      "Complement found from 3745 to 3751\n",
      "Complement found from 3834 to 3840\n",
      "Complement found from 4635 to 4641\n",
      "Complement found from 4950 to 4956\n",
      "Complement found from 5307 to 5313\n",
      "Complement found from 5319 to 5325\n",
      "Complement found from 6117 to 6123\n",
      "Complement found from 6384 to 6390\n",
      "Complement found from 7065 to 7071\n",
      "Complement found from 7793 to 7799\n",
      "Complement found from 7871 to 7877\n",
      "Complement found from 8962 to 8968\n",
      "Complement found from 12631 to 12637\n",
      "Complement found from 13369 to 13375\n",
      "Complement found from 14056 to 14062\n",
      "Complement found from 14341 to 14347\n",
      "Complement found from 14480 to 14486\n",
      "Complement found from 15788 to 15794\n",
      "Complement found from 16071 to 16077\n",
      "Seq is: CCATCA\n",
      "20 CCATCA motifs detected in file\n",
      "\n",
      "Complement found from 265 to 271\n",
      "Complement found from 292 to 298\n",
      "Complement found from 4621 to 4627\n",
      "Complement found from 7031 to 7037\n",
      "Complement found from 9314 to 9320\n",
      "Complement found from 9812 to 9818\n",
      "Complement found from 13122 to 13128\n",
      "Complement found from 15403 to 15409\n",
      "Seq is: TTCCAC\n",
      "8 TTCCAC motifs detected in file\n",
      "\n",
      "Complement found from 978 to 984\n",
      "Complement found from 1233 to 1239\n",
      "Complement found from 3540 to 3546\n",
      "Complement found from 4014 to 4020\n",
      "Complement found from 4191 to 4197\n",
      "Complement found from 5890 to 5896\n",
      "Complement found from 8475 to 8481\n",
      "Complement found from 9645 to 9651\n",
      "Complement found from 12013 to 12019\n",
      "Complement found from 12963 to 12969\n",
      "Complement found from 13452 to 13458\n",
      "Complement found from 13620 to 13626\n",
      "Complement found from 14401 to 14407\n",
      "Complement found from 14910 to 14916\n",
      "Complement found from 15480 to 15486\n",
      "Complement found from 16066 to 16072\n",
      "Complement found from 16261 to 16267\n",
      "Seq is: CTCACC\n",
      "17 CTCACC motifs detected in file\n",
      "\n",
      "Complement found from 298 to 304\n",
      "Complement found from 348 to 354\n",
      "Complement found from 552 to 558\n",
      "Complement found from 1273 to 1279\n",
      "Complement found from 1684 to 1690\n",
      "Complement found from 1720 to 1726\n",
      "Complement found from 1991 to 1997\n",
      "Complement found from 2765 to 2771\n",
      "Complement found from 2788 to 2794\n",
      "Complement found from 4254 to 4260\n",
      "Complement found from 4982 to 4988\n",
      "Complement found from 5931 to 5937\n",
      "Complement found from 7121 to 7127\n",
      "Complement found from 8125 to 8131\n",
      "Complement found from 8189 to 8195\n",
      "Complement found from 8508 to 8514\n",
      "Complement found from 11668 to 11674\n",
      "Complement found from 14624 to 14630\n",
      "Complement found from 16281 to 16287\n",
      "Seq is: CAAACC\n",
      "19 CAAACC motifs detected in file\n",
      "\n",
      "Complement found from 1224 to 1230\n",
      "Complement found from 15588 to 15594\n",
      "Seq is: CCGATC\n",
      "2 CCGATC motifs detected in file\n",
      "\n",
      "Complement found from 2478 to 2484\n",
      "Complement found from 4898 to 4904\n",
      "Complement found from 5623 to 5629\n",
      "Complement found from 7875 to 7881\n",
      "Complement found from 10888 to 10894\n",
      "Complement found from 11065 to 11071\n",
      "Complement found from 13001 to 13007\n",
      "Complement found from 14043 to 14049\n",
      "Complement found from 14875 to 14881\n",
      "Complement found from 15951 to 15957\n",
      "Complement found from 16347 to 16353\n",
      "Seq is: CAAATC\n",
      "11 CAAATC motifs detected in file\n",
      "\n",
      "Complement found from 9752 to 9758\n",
      "Seq is: GAGTCT\n",
      "1 GAGTCT motifs detected in file\n",
      "\n",
      "Complement found from 1132 to 1138\n",
      "Seq is: CGCCAG\n",
      "1 CGCCAG motifs detected in file\n",
      "\n",
      "Complement found from 171 to 177\n",
      "Complement found from 3284 to 3290\n",
      "Complement found from 5720 to 5726\n",
      "Complement found from 5800 to 5806\n",
      "Complement found from 11625 to 11631\n",
      "Complement found from 15227 to 15233\n",
      "Seq is: TTCAAT\n",
      "6 TTCAAT motifs detected in file\n",
      "\n",
      "Complement found from 1647 to 1653\n",
      "Complement found from 2041 to 2047\n",
      "Complement found from 3187 to 3193\n",
      "Complement found from 5124 to 5130\n",
      "Complement found from 5612 to 5618\n",
      "Complement found from 5711 to 5717\n",
      "Complement found from 7937 to 7943\n",
      "Complement found from 9839 to 9845\n",
      "Complement found from 12262 to 12268\n",
      "Complement found from 13259 to 13265\n",
      "Complement found from 13846 to 13852\n",
      "Complement found from 14728 to 14734\n",
      "Complement found from 16223 to 16229\n",
      "Complement found from 16237 to 16243\n",
      "Seq is: TCAACT\n",
      "14 TCAACT motifs detected in file\n",
      "\n",
      "Complement found from 2530 to 2536\n",
      "Seq is: TTAGAG\n",
      "1 TTAGAG motifs detected in file\n",
      "\n",
      "Complement found from 6930 to 6936\n",
      "Seq is: GATTCA\n",
      "1 GATTCA motifs detected in file\n",
      "\n",
      "Complement found from 7081 to 7087\n",
      "Complement found from 7275 to 7281\n",
      "Complement found from 8548 to 8554\n",
      "Complement found from 8975 to 8981\n",
      "Complement found from 10042 to 10048\n",
      "Complement found from 11084 to 11090\n",
      "Complement found from 12044 to 12050\n",
      "Complement found from 12839 to 12845\n",
      "Complement found from 14795 to 14801\n",
      "Seq is: CATTCA\n",
      "9 CATTCA motifs detected in file\n",
      "\n",
      "Complement found from 1308 to 1314\n",
      "Complement found from 1687 to 1693\n",
      "Complement found from 2215 to 2221\n",
      "Complement found from 2768 to 2774\n",
      "Complement found from 3202 to 3208\n",
      "Complement found from 3208 to 3214\n",
      "Complement found from 5021 to 5027\n",
      "Complement found from 5431 to 5437\n",
      "Complement found from 5675 to 5681\n",
      "Complement found from 9209 to 9215\n",
      "Complement found from 10619 to 10625\n",
      "Complement found from 10843 to 10849\n",
      "Complement found from 12016 to 12022\n",
      "Complement found from 12925 to 12931\n",
      "Complement found from 13326 to 13332\n",
      "Complement found from 14356 to 14362\n",
      "Complement found from 14640 to 14646\n",
      "Complement found from 16264 to 16270\n",
      "Complement found from 16288 to 16294\n",
      "Seq is: ACCCAC\n",
      "19 ACCCAC motifs detected in file\n",
      "\n",
      "Complement found from 810 to 816\n",
      "Complement found from 5169 to 5175\n",
      "Complement found from 7251 to 7257\n",
      "Complement found from 14654 to 14660\n",
      "Complement found from 15083 to 15089\n",
      "Seq is: GAAACA\n",
      "5 GAAACA motifs detected in file\n",
      "\n",
      "Complement found from 3286 to 3292\n",
      "Complement found from 5797 to 5803\n",
      "Complement found from 8105 to 8111\n",
      "Complement found from 9106 to 9112\n",
      "Complement found from 15581 to 15587\n",
      "Seq is: CAATTC\n",
      "5 CAATTC motifs detected in file\n",
      "\n",
      "Complement found from 9941 to 9947\n",
      "Seq is: GATGTG\n",
      "1 GATGTG motifs detected in file\n",
      "\n",
      "Complement found from 3880 to 3886\n",
      "Complement found from 5530 to 5536\n",
      "Complement found from 5658 to 5664\n",
      "Complement found from 7117 to 7123\n",
      "Complement found from 12506 to 12512\n",
      "Complement found from 13511 to 13517\n",
      "Seq is: AGACCA\n",
      "6 AGACCA motifs detected in file\n",
      "\n",
      "Complement found from 4957 to 4963\n",
      "Complement found from 6265 to 6271\n",
      "Complement found from 6298 to 6304\n",
      "Complement found from 6358 to 6364\n",
      "Complement found from 12995 to 13001\n",
      "Complement found from 13097 to 13103\n",
      "Complement found from 13475 to 13481\n",
      "Seq is: AGCAGG\n",
      "7 AGCAGG motifs detected in file\n",
      "\n",
      "Complement found from 9822 to 9828\n",
      "Complement found from 14990 to 14996\n",
      "Seq is: TTCACG\n",
      "2 TTCACG motifs detected in file\n",
      "\n",
      "Seq is: TGAAAG\n",
      "0 TGAAAG motifs detected in file\n",
      "\n",
      "Seq is: GTCATG\n",
      "0 GTCATG motifs detected in file\n",
      "\n",
      "Complement found from 245 to 251\n",
      "Complement found from 2629 to 2635\n",
      "Seq is: TGAATG\n",
      "2 TGAATG motifs detected in file\n",
      "\n",
      "Complement found from 5053 to 5059\n",
      "Seq is: GTACAA\n",
      "1 GTACAA motifs detected in file\n",
      "\n",
      "Seq is: TGCTGA\n",
      "0 TGCTGA motifs detected in file\n",
      "\n",
      "Complement found from 12179 to 12185\n",
      "Seq is: ACAGAG\n",
      "1 ACAGAG motifs detected in file\n",
      "\n",
      "Seq is: GAGATA\n",
      "0 GAGATA motifs detected in file\n",
      "\n",
      "Complement found from 154 to 160\n",
      "Complement found from 3613 to 3619\n",
      "Complement found from 5086 to 5092\n",
      "Complement found from 6777 to 6783\n",
      "Complement found from 6861 to 6867\n",
      "Complement found from 8255 to 8261\n",
      "Complement found from 9459 to 9465\n",
      "Complement found from 10459 to 10465\n",
      "Complement found from 10904 to 10910\n",
      "Complement found from 11987 to 11993\n",
      "Complement found from 12198 to 12204\n",
      "Complement found from 13351 to 13357\n",
      "Seq is: TATTTA\n",
      "12 TATTTA motifs detected in file\n",
      "\n",
      "Complement found from 4663 to 4669\n",
      "Complement found from 6241 to 6247\n",
      "Complement found from 7828 to 7834\n",
      "Complement found from 9619 to 9625\n",
      "Complement found from 11776 to 11782\n",
      "Complement found from 12455 to 12461\n",
      "Complement found from 13757 to 13763\n",
      "Seq is: CGCATC\n",
      "7 CGCATC motifs detected in file\n",
      "\n",
      "Complement found from 353 to 359\n",
      "Complement found from 557 to 563\n",
      "Complement found from 1063 to 1069\n",
      "Complement found from 1682 to 1688\n",
      "Complement found from 1731 to 1737\n",
      "Complement found from 2233 to 2239\n",
      "Complement found from 3715 to 3721\n",
      "Complement found from 5252 to 5258\n",
      "Complement found from 7468 to 7474\n",
      "Complement found from 12302 to 12308\n",
      "Complement found from 12547 to 12553\n",
      "Complement found from 12668 to 12674\n",
      "Complement found from 14071 to 14077\n",
      "Complement found from 14532 to 14538\n",
      "Complement found from 15988 to 15994\n",
      "Seq is: CCCAAA\n",
      "15 CCCAAA motifs detected in file\n",
      "\n",
      "Complement found from 483 to 489\n",
      "Complement found from 4740 to 4746\n",
      "Complement found from 6386 to 6392\n",
      "Complement found from 6407 to 6413\n",
      "Complement found from 6736 to 6742\n",
      "Complement found from 7045 to 7051\n",
      "Complement found from 7878 to 7884\n",
      "Complement found from 9631 to 9637\n",
      "Complement found from 14575 to 14581\n",
      "Complement found from 14937 to 14943\n",
      "Complement found from 16416 to 16422\n",
      "Seq is: ATCAAT\n",
      "11 ATCAAT motifs detected in file\n",
      "\n",
      "Complement found from 178 to 184\n",
      "Complement found from 3271 to 3277\n",
      "Complement found from 5858 to 5864\n",
      "Complement found from 6780 to 6786\n",
      "Complement found from 8355 to 8361\n",
      "Complement found from 11942 to 11948\n",
      "Complement found from 16340 to 16346\n",
      "Seq is: TTACAG\n",
      "7 TTACAG motifs detected in file\n",
      "\n",
      "Complement found from 1400 to 1406\n",
      "Complement found from 14837 to 14843\n",
      "Seq is: GAAACT\n",
      "2 GAAACT motifs detected in file\n",
      "\n",
      "Complement found from 1183 to 1189\n",
      "Complement found from 3726 to 3732\n",
      "Complement found from 4890 to 4896\n",
      "Complement found from 7149 to 7155\n",
      "Complement found from 9229 to 9235\n",
      "Complement found from 10456 to 10462\n",
      "Complement found from 10545 to 10551\n",
      "Complement found from 11105 to 11111\n",
      "Seq is: TCATAT\n",
      "8 TCATAT motifs detected in file\n",
      "\n",
      "Complement found from 2248 to 2254\n",
      "Complement found from 2470 to 2476\n",
      "Complement found from 4062 to 4068\n",
      "Complement found from 11890 to 11896\n",
      "Seq is: GAACTC\n",
      "4 GAACTC motifs detected in file\n",
      "\n",
      "Complement found from 458 to 464\n",
      "Complement found from 464 to 470\n",
      "Complement found from 6310 to 6316\n",
      "Complement found from 6497 to 6503\n",
      "Complement found from 6664 to 6670\n",
      "Complement found from 7816 to 7822\n",
      "Complement found from 11812 to 11818\n",
      "Complement found from 15381 to 15387\n",
      "Seq is: CTCCCA\n",
      "8 CTCCCA motifs detected in file\n",
      "\n",
      "Complement found from 2543 to 2549\n",
      "Complement found from 2683 to 2689\n",
      "Complement found from 5228 to 5234\n",
      "Complement found from 7786 to 7792\n",
      "Complement found from 11303 to 11309\n",
      "Complement found from 13977 to 13983\n",
      "Seq is: CTGCCC\n",
      "6 CTGCCC motifs detected in file\n",
      "\n",
      "Complement found from 322 to 328\n",
      "Complement found from 795 to 801\n",
      "Complement found from 1148 to 1154\n",
      "Complement found from 3495 to 3501\n",
      "Complement found from 6875 to 6881\n",
      "Complement found from 8763 to 8769\n",
      "Complement found from 9803 to 9809\n",
      "Complement found from 11092 to 11098\n",
      "Complement found from 11633 to 11639\n",
      "Complement found from 12540 to 12546\n",
      "Complement found from 15172 to 15178\n",
      "Seq is: GCCACA\n",
      "11 GCCACA motifs detected in file\n",
      "\n",
      "Seq is: CATGCG\n",
      "0 CATGCG motifs detected in file\n",
      "\n",
      "Seq is: CGAAGT\n",
      "0 CGAAGT motifs detected in file\n",
      "\n",
      "Seq is: CCCGGG\n",
      "0 CCCGGG motifs detected in file\n",
      "\n",
      "Complement found from 1554 to 1560\n",
      "Complement found from 2398 to 2404\n",
      "Complement found from 2918 to 2924\n",
      "Complement found from 15798 to 15804\n",
      "Seq is: ACAAGT\n",
      "4 ACAAGT motifs detected in file\n",
      "\n",
      "Complement found from 134 to 140\n",
      "Seq is: TGATTC\n",
      "1 TGATTC motifs detected in file\n",
      "\n",
      "Seq is: GTGAGA\n",
      "0 GTGAGA motifs detected in file\n",
      "\n",
      "Complement found from 4337 to 4343\n",
      "Complement found from 4356 to 4362\n",
      "Complement found from 7319 to 7325\n",
      "Complement found from 8515 to 8521\n",
      "Seq is: TGAGAA\n",
      "4 TGAGAA motifs detected in file\n",
      "\n",
      "Complement found from 392 to 398\n",
      "Complement found from 3108 to 3114\n",
      "Complement found from 7874 to 7880\n",
      "Complement found from 11923 to 11929\n",
      "Complement found from 12688 to 12694\n",
      "Complement found from 15875 to 15881\n",
      "Complement found from 16346 to 16352\n",
      "Seq is: TCAAAT\n",
      "7 TCAAAT motifs detected in file\n",
      "\n",
      "Seq is: GACGAT\n",
      "0 GACGAT motifs detected in file\n",
      "\n",
      "Complement found from 1323 to 1329\n",
      "Complement found from 5517 to 5523\n",
      "Complement found from 13014 to 13020\n",
      "Seq is: TTAGGT\n",
      "3 TTAGGT motifs detected in file\n",
      "\n",
      "Complement found from 6887 to 6893\n",
      "Complement found from 9048 to 9054\n",
      "Complement found from 13713 to 13719\n",
      "Complement found from 16034 to 16040\n",
      "Seq is: GGAAGC\n",
      "4 GGAAGC motifs detected in file\n",
      "\n",
      "Complement found from 1966 to 1972\n",
      "Seq is: TGGGAA\n",
      "1 TGGGAA motifs detected in file\n",
      "\n",
      "Complement found from 391 to 397\n",
      "Complement found from 1508 to 1514\n",
      "Complement found from 3107 to 3113\n",
      "Complement found from 5542 to 5548\n",
      "Complement found from 7511 to 7517\n",
      "Complement found from 10044 to 10050\n",
      "Complement found from 11801 to 11807\n",
      "Complement found from 12573 to 12579\n",
      "Complement found from 12687 to 12693\n",
      "Complement found from 13338 to 13344\n",
      "Seq is: TTCAAA\n",
      "10 TTCAAA motifs detected in file\n",
      "\n",
      "Seq is: GTATTC\n",
      "0 GTATTC motifs detected in file\n",
      "\n",
      "Complement found from 6171 to 6177\n",
      "Complement found from 15389 to 15395\n",
      "Seq is: CCGATA\n",
      "2 CCGATA motifs detected in file\n",
      "\n",
      "Complement found from 754 to 760\n",
      "Complement found from 2518 to 2524\n",
      "Complement found from 3669 to 3675\n",
      "Complement found from 5609 to 5615\n",
      "Complement found from 6516 to 6522\n",
      "Complement found from 9564 to 9570\n",
      "Complement found from 9620 to 9626\n",
      "Complement found from 10862 to 10868\n",
      "Complement found from 11777 to 11783\n",
      "Complement found from 13288 to 13294\n",
      "Complement found from 13927 to 13933\n",
      "Seq is: GCATCA\n",
      "11 GCATCA motifs detected in file\n",
      "\n",
      "Complement found from 560 to 566\n",
      "Complement found from 1316 to 1322\n",
      "Complement found from 5939 to 5945\n",
      "Complement found from 13509 to 13515\n",
      "Complement found from 14472 to 14478\n",
      "Complement found from 15424 to 15430\n",
      "Seq is: AAAGAC\n",
      "6 AAAGAC motifs detected in file\n",
      "\n",
      "Complement found from 499 to 505\n",
      "Complement found from 4348 to 4354\n",
      "Complement found from 4391 to 4397\n",
      "Complement found from 4880 to 4886\n",
      "Complement found from 6116 to 6122\n",
      "Complement found from 7818 to 7824\n",
      "Complement found from 8206 to 8212\n",
      "Complement found from 11428 to 11434\n",
      "Complement found from 12387 to 12393\n",
      "Complement found from 14340 to 14346\n",
      "Complement found from 14814 to 14820\n",
      "Complement found from 15658 to 15664\n",
      "Complement found from 16070 to 16076\n",
      "Seq is: CCCATC\n",
      "13 CCCATC motifs detected in file\n",
      "\n",
      "Complement found from 4549 to 4555\n",
      "Complement found from 5884 to 5890\n",
      "Complement found from 9506 to 9512\n",
      "Complement found from 9705 to 9711\n",
      "Complement found from 9721 to 9727\n",
      "Complement found from 10272 to 10278\n",
      "Complement found from 15783 to 15789\n",
      "Seq is: TTTTAC\n",
      "7 TTTTAC motifs detected in file\n",
      "\n",
      "Complement found from 4423 to 4429\n",
      "Complement found from 15037 to 15043\n",
      "Seq is: ATCGGG\n",
      "2 ATCGGG motifs detected in file\n",
      "\n",
      "Complement found from 936 to 942\n",
      "Complement found from 1315 to 1321\n",
      "Complement found from 8329 to 8335\n",
      "Complement found from 11372 to 11378\n",
      "Seq is: TAAAGA\n",
      "4 TAAAGA motifs detected in file\n",
      "\n",
      "Seq is: ACGGTG\n",
      "0 ACGGTG motifs detected in file\n",
      "\n",
      "Complement found from 7909 to 7915\n",
      "Complement found from 9743 to 9749\n",
      "Seq is: GAGTAC\n",
      "2 GAGTAC motifs detected in file\n",
      "\n",
      "Complement found from 3007 to 3013\n",
      "Complement found from 7233 to 7239\n",
      "Complement found from 11160 to 11166\n",
      "Seq is: CCGATG\n",
      "3 CCGATG motifs detected in file\n",
      "\n",
      "Complement found from 5947 to 5953\n",
      "Seq is: TGGAAC\n",
      "1 TGGAAC motifs detected in file\n",
      "\n",
      "Complement found from 1419 to 1425\n",
      "Complement found from 4972 to 4978\n",
      "Seq is: TGGATT\n",
      "2 TGGATT motifs detected in file\n",
      "\n",
      "Complement found from 2827 to 2833\n",
      "Complement found from 13134 to 13140\n",
      "Seq is: GCAGAA\n",
      "2 GCAGAA motifs detected in file\n",
      "\n",
      "Complement found from 72 to 78\n",
      "Complement found from 414 to 420\n",
      "Complement found from 9036 to 9042\n",
      "Complement found from 12342 to 12348\n",
      "Complement found from 13886 to 13892\n",
      "Complement found from 14902 to 14908\n",
      "Seq is: ATGCAC\n",
      "6 ATGCAC motifs detected in file\n",
      "\n",
      "Complement found from 1340 to 1346\n",
      "Complement found from 7501 to 7507\n",
      "Complement found from 9248 to 9254\n",
      "Complement found from 10283 to 10289\n",
      "Complement found from 16113 to 16119\n",
      "Seq is: CCATGA\n",
      "5 CCATGA motifs detected in file\n",
      "\n",
      "Complement found from 76 to 82\n",
      "Seq is: ACGCGA\n",
      "1 ACGCGA motifs detected in file\n",
      "\n",
      "Complement found from 1398 to 1404\n",
      "Complement found from 1797 to 1803\n",
      "Complement found from 2314 to 2320\n",
      "Complement found from 4172 to 4178\n",
      "Complement found from 5806 to 5812\n",
      "Complement found from 6896 to 6902\n",
      "Complement found from 7249 to 7255\n",
      "Complement found from 14707 to 14713\n",
      "Complement found from 14835 to 14841\n",
      "Complement found from 15930 to 15936\n",
      "Seq is: ATGAAA\n",
      "10 ATGAAA motifs detected in file\n",
      "\n",
      "Complement found from 7156 to 7162\n",
      "Complement found from 12758 to 12764\n",
      "Complement found from 15036 to 15042\n",
      "Complement found from 15087 to 15093\n",
      "Seq is: CATCGG\n",
      "4 CATCGG motifs detected in file\n",
      "\n",
      "Complement found from 7601 to 7607\n",
      "Complement found from 15800 to 15806\n",
      "Seq is: AAGTAG\n",
      "2 AAGTAG motifs detected in file\n",
      "\n",
      "Complement found from 2232 to 2238\n",
      "Complement found from 10775 to 10781\n",
      "Seq is: TCCCAA\n",
      "2 TCCCAA motifs detected in file\n",
      "\n",
      "Seq is: ACGACT\n",
      "0 ACGACT motifs detected in file\n",
      "\n",
      "Complement found from 1370 to 1376\n",
      "Complement found from 3501 to 3507\n",
      "Complement found from 4499 to 4505\n",
      "Complement found from 5047 to 5053\n",
      "Complement found from 5343 to 5349\n",
      "Complement found from 6285 to 6291\n",
      "Complement found from 8276 to 8282\n",
      "Complement found from 9184 to 9190\n",
      "Complement found from 11039 to 11045\n",
      "Complement found from 13786 to 13792\n",
      "Complement found from 13918 to 13924\n",
      "Seq is: TCTACC\n",
      "11 TCTACC motifs detected in file\n",
      "\n",
      "Complement found from 3459 to 3465\n",
      "Complement found from 3849 to 3855\n",
      "Complement found from 6425 to 6431\n",
      "Complement found from 10652 to 10658\n",
      "Complement found from 13041 to 13047\n",
      "Complement found from 13344 to 13350\n",
      "Seq is: GCCATA\n",
      "6 GCCATA motifs detected in file\n",
      "\n",
      "Complement found from 7200 to 7206\n",
      "Complement found from 15194 to 15200\n",
      "Complement found from 16490 to 16496\n",
      "Seq is: TATCCG\n",
      "3 TATCCG motifs detected in file\n",
      "\n",
      "Seq is: ACTCGT\n",
      "0 ACTCGT motifs detected in file\n",
      "\n",
      "Complement found from 39 to 45\n",
      "Complement found from 3790 to 3796\n",
      "Complement found from 3865 to 3871\n",
      "Complement found from 4219 to 4225\n",
      "Complement found from 4237 to 4243\n",
      "Complement found from 9966 to 9972\n",
      "Complement found from 10209 to 10215\n",
      "Complement found from 10710 to 10716\n",
      "Complement found from 12589 to 12595\n",
      "Complement found from 13019 to 13025\n",
      "Complement found from 13246 to 13252\n",
      "Complement found from 13900 to 13906\n",
      "Complement found from 14047 to 14053\n",
      "Seq is: TCTCCA\n",
      "13 TCTCCA motifs detected in file\n",
      "\n",
      "Complement found from 25 to 31\n",
      "Complement found from 461 to 467\n",
      "Complement found from 1689 to 1695\n",
      "Complement found from 4188 to 4194\n",
      "Complement found from 5602 to 5608\n",
      "Complement found from 9316 to 9322\n",
      "Complement found from 9511 to 9517\n",
      "Complement found from 9593 to 9599\n",
      "Complement found from 10621 to 10627\n",
      "Complement found from 13076 to 13082\n",
      "Complement found from 13186 to 13192\n",
      "Complement found from 14115 to 14121\n",
      "Complement found from 14790 to 14796\n",
      "Seq is: CCACTC\n",
      "13 CCACTC motifs detected in file\n",
      "\n",
      "Complement found from 437 to 443\n",
      "Complement found from 3399 to 3405\n",
      "Complement found from 8371 to 8377\n",
      "Complement found from 8437 to 8443\n",
      "Complement found from 8767 to 8773\n",
      "Complement found from 8814 to 8820\n",
      "Complement found from 9869 to 9875\n",
      "Complement found from 10299 to 10305\n",
      "Complement found from 13260 to 13266\n",
      "Complement found from 13847 to 13853\n",
      "Complement found from 14729 to 14735\n",
      "Complement found from 15110 to 15116\n",
      "Complement found from 15845 to 15851\n",
      "Complement found from 16224 to 16230\n",
      "Seq is: CAACTA\n",
      "14 CAACTA motifs detected in file\n",
      "\n",
      "Complement found from 1680 to 1686\n",
      "Complement found from 3412 to 3418\n",
      "Complement found from 8367 to 8373\n",
      "Complement found from 12300 to 12306\n",
      "Complement found from 13051 to 13057\n",
      "Seq is: GCCCCA\n",
      "5 GCCCCA motifs detected in file\n",
      "\n",
      "Complement found from 1052 to 1058\n",
      "Complement found from 4681 to 4687\n",
      "Complement found from 4765 to 4771\n",
      "Complement found from 4789 to 4795\n",
      "Complement found from 5038 to 5044\n",
      "Complement found from 5288 to 5294\n",
      "Complement found from 8984 to 8990\n",
      "Complement found from 11358 to 11364\n",
      "Complement found from 11820 to 11826\n",
      "Complement found from 12935 to 12941\n",
      "Complement found from 13140 to 13146\n",
      "Complement found from 14448 to 14454\n",
      "Complement found from 16530 to 16536\n",
      "Seq is: AATAGC\n",
      "13 AATAGC motifs detected in file\n",
      "\n",
      "Complement found from 2295 to 2301\n",
      "Seq is: TAATGT\n",
      "1 TAATGT motifs detected in file\n",
      "\n",
      "Seq is: CTTGTC\n",
      "0 CTTGTC motifs detected in file\n",
      "\n",
      "Seq is: CGAGGA\n",
      "0 CGAGGA motifs detected in file\n",
      "\n",
      "Complement found from 8728 to 8734\n",
      "Seq is: GATCTC\n",
      "1 GATCTC motifs detected in file\n",
      "\n",
      "Complement found from 1252 to 1258\n",
      "Complement found from 1838 to 1844\n",
      "Complement found from 2282 to 2288\n",
      "Complement found from 7572 to 7578\n",
      "Complement found from 8262 to 8268\n",
      "Complement found from 10184 to 10190\n",
      "Complement found from 11970 to 11976\n",
      "Complement found from 12851 to 12857\n",
      "Complement found from 13597 to 13603\n",
      "Complement found from 15048 to 15054\n",
      "Seq is: CCTATA\n",
      "10 CCTATA motifs detected in file\n",
      "\n",
      "Complement found from 6451 to 6457\n",
      "Seq is: CTTCGT\n",
      "1 CTTCGT motifs detected in file\n",
      "\n",
      "Complement found from 408 to 414\n",
      "Complement found from 1174 to 1180\n",
      "Seq is: GGCGGT\n",
      "2 GGCGGT motifs detected in file\n",
      "\n",
      "Seq is: CCAGGA\n",
      "0 CCAGGA motifs detected in file\n",
      "\n",
      "Complement found from 976 to 982\n",
      "Complement found from 1160 to 1166\n",
      "Complement found from 3675 to 3681\n",
      "Complement found from 4156 to 4162\n",
      "Complement found from 6980 to 6986\n",
      "Complement found from 10134 to 10140\n",
      "Complement found from 11749 to 11755\n",
      "Complement found from 12228 to 12234\n",
      "Complement found from 12423 to 12429\n",
      "Complement found from 12660 to 12666\n",
      "Complement found from 12915 to 12921\n",
      "Complement found from 13795 to 13801\n",
      "Seq is: AACTCA\n",
      "12 AACTCA motifs detected in file\n",
      "\n",
      "Complement found from 3110 to 3116\n",
      "Complement found from 3382 to 3388\n",
      "Complement found from 4366 to 4372\n",
      "Complement found from 10217 to 10223\n",
      "Complement found from 14536 to 14542\n",
      "Seq is: AAATTC\n",
      "5 AAATTC motifs detected in file\n",
      "\n",
      "Complement found from 563 to 569\n",
      "Seq is: GACACC\n",
      "1 GACACC motifs detected in file\n",
      "\n",
      "Complement found from 3667 to 3673\n",
      "Complement found from 6807 to 6813\n",
      "Seq is: GAGCAT\n",
      "2 GAGCAT motifs detected in file\n",
      "\n",
      "Complement found from 3808 to 3814\n",
      "Complement found from 8051 to 8057\n",
      "Complement found from 8914 to 8920\n",
      "Complement found from 12215 to 12221\n",
      "Complement found from 16428 to 16434\n",
      "Seq is: CACAAG\n",
      "5 CACAAG motifs detected in file\n",
      "\n",
      "Complement found from 522 to 528\n",
      "Complement found from 1660 to 1666\n",
      "Complement found from 8137 to 8143\n",
      "Complement found from 9006 to 9012\n",
      "Complement found from 12732 to 12738\n",
      "Complement found from 14565 to 14571\n",
      "Complement found from 16080 to 16086\n",
      "Seq is: ACCGCT\n",
      "7 ACCGCT motifs detected in file\n",
      "\n",
      "Complement found from 6853 to 6859\n",
      "Seq is: CGTCAA\n",
      "1 CGTCAA motifs detected in file\n",
      "\n",
      "Complement found from 3300 to 3306\n",
      "Complement found from 4037 to 4043\n",
      "Complement found from 5393 to 5399\n",
      "Complement found from 6194 to 6200\n",
      "Complement found from 8640 to 8646\n",
      "Complement found from 9665 to 9671\n",
      "Complement found from 10297 to 10303\n",
      "Complement found from 10894 to 10900\n",
      "Complement found from 10941 to 10947\n",
      "Complement found from 11332 to 11338\n",
      "Complement found from 12028 to 12034\n",
      "Complement found from 12247 to 12253\n",
      "Complement found from 12551 to 12557\n",
      "Complement found from 12738 to 12744\n",
      "Complement found from 13743 to 13749\n",
      "Complement found from 13772 to 13778\n",
      "Complement found from 15361 to 15367\n",
      "Complement found from 15677 to 15683\n",
      "Complement found from 16076 to 16082\n",
      "Seq is: AACAAC\n",
      "19 AACAAC motifs detected in file\n",
      "\n",
      "Complement found from 2048 to 2054\n",
      "Complement found from 2082 to 2088\n",
      "Complement found from 7555 to 7561\n",
      "Complement found from 10443 to 10449\n",
      "Complement found from 14293 to 14299\n",
      "Complement found from 14497 to 14503\n",
      "Complement found from 14963 to 14969\n",
      "Seq is: TAAATT\n",
      "7 TAAATT motifs detected in file\n",
      "\n",
      "Complement found from 1708 to 1714\n",
      "Complement found from 9195 to 9201\n",
      "Complement found from 10037 to 10043\n",
      "Complement found from 12175 to 12181\n",
      "Complement found from 14475 to 14481\n",
      "Complement found from 15764 to 15770\n",
      "Seq is: GACAAC\n",
      "6 GACAAC motifs detected in file\n",
      "\n",
      "Complement found from 2098 to 2104\n",
      "Complement found from 4362 to 4368\n",
      "Complement found from 8625 to 8631\n",
      "Complement found from 9429 to 9435\n",
      "Complement found from 9890 to 9896\n",
      "Complement found from 10810 to 10816\n",
      "Complement found from 11666 to 11672\n",
      "Complement found from 12322 to 12328\n",
      "Complement found from 13153 to 13159\n",
      "Complement found from 13506 to 13512\n",
      "Complement found from 13768 to 13774\n",
      "Complement found from 14469 to 14475\n",
      "Complement found from 14873 to 14879\n",
      "Complement found from 15673 to 15679\n",
      "Complement found from 16248 to 16254\n",
      "Seq is: TCCAAA\n",
      "15 TCCAAA motifs detected in file\n",
      "\n",
      "Complement found from 3005 to 3011\n",
      "Complement found from 8015 to 8021\n",
      "Complement found from 14263 to 14269\n",
      "Seq is: TCCCGA\n",
      "3 TCCCGA motifs detected in file\n",
      "\n",
      "Complement found from 8033 to 8039\n",
      "Complement found from 16092 to 16098\n",
      "Seq is: TTCGTA\n",
      "2 TTCGTA motifs detected in file\n",
      "\n",
      "Complement found from 13 to 19\n",
      "Complement found from 429 to 435\n",
      "Complement found from 633 to 639\n",
      "Complement found from 720 to 726\n",
      "Complement found from 952 to 958\n",
      "Complement found from 1489 to 1495\n",
      "Complement found from 2278 to 2284\n",
      "Complement found from 3508 to 3514\n",
      "Complement found from 3736 to 3742\n",
      "Complement found from 4192 to 4198\n",
      "Complement found from 5322 to 5328\n",
      "Complement found from 5891 to 5897\n",
      "Complement found from 6619 to 6625\n",
      "Complement found from 8432 to 8438\n",
      "Complement found from 9567 to 9573\n",
      "Complement found from 11156 to 11162\n",
      "Complement found from 12014 to 12020\n",
      "Complement found from 12964 to 12970\n",
      "Complement found from 13621 to 13627\n",
      "Complement found from 14354 to 14360\n",
      "Complement found from 16067 to 16073\n",
      "Complement found from 16262 to 16268\n",
      "Seq is: TCACCC\n",
      "22 TCACCC motifs detected in file\n",
      "\n",
      "Complement found from 7632 to 7638\n",
      "Complement found from 10147 to 10153\n",
      "Complement found from 12635 to 12641\n",
      "Complement found from 13043 to 13049\n",
      "Seq is: CATAGA\n",
      "4 CATAGA motifs detected in file\n",
      "\n",
      "Complement found from 6377 to 6383\n",
      "Seq is: TTAGGG\n",
      "1 TTAGGG motifs detected in file\n",
      "\n",
      "Complement found from 3672 to 3678\n",
      "Complement found from 3678 to 3684\n",
      "Complement found from 4253 to 4259\n",
      "Complement found from 8674 to 8680\n",
      "Complement found from 11317 to 11323\n",
      "Complement found from 11752 to 11758\n",
      "Complement found from 11802 to 11808\n",
      "Complement found from 12574 to 12580\n",
      "Complement found from 15358 to 15364\n",
      "Seq is: TCAAAC\n",
      "9 TCAAAC motifs detected in file\n",
      "\n",
      "Complement found from 1134 to 1140\n",
      "Complement found from 1376 to 1382\n",
      "Complement found from 11176 to 11182\n",
      "Seq is: CCAGAA\n",
      "3 CCAGAA motifs detected in file\n",
      "\n",
      "Complement found from 1214 to 1220\n",
      "Complement found from 9442 to 9448\n",
      "Seq is: TCGATA\n",
      "2 TCGATA motifs detected in file\n",
      "\n",
      "Complement found from 514 to 520\n",
      "Complement found from 1046 to 1052\n",
      "Complement found from 1477 to 1483\n",
      "Complement found from 6801 to 6807\n",
      "Complement found from 9354 to 9360\n",
      "Complement found from 9417 to 9423\n",
      "Complement found from 13932 to 13938\n",
      "Complement found from 14551 to 14557\n",
      "Seq is: ACACAC\n",
      "8 ACACAC motifs detected in file\n",
      "\n",
      "Complement found from 10357 to 10363\n",
      "Complement found from 15964 to 15970\n",
      "Seq is: AAGTCT\n",
      "2 AAGTCT motifs detected in file\n",
      "\n",
      "Complement found from 487 to 493\n",
      "Complement found from 3396 to 3402\n",
      "Complement found from 5424 to 5430\n",
      "Complement found from 12854 to 12860\n",
      "Complement found from 14235 to 14241\n",
      "Seq is: ATACAA\n",
      "5 ATACAA motifs detected in file\n",
      "\n",
      "Complement found from 2805 to 2811\n",
      "Complement found from 6613 to 6619\n",
      "Seq is: TTTCGG\n",
      "2 TTTCGG motifs detected in file\n",
      "\n",
      "Seq is: TGAAAG\n",
      "0 TGAAAG motifs detected in file\n",
      "\n",
      "Complement found from 8963 to 8969\n",
      "Complement found from 9621 to 9627\n",
      "Complement found from 12158 to 12164\n",
      "Complement found from 12797 to 12803\n",
      "Seq is: CATCAG\n",
      "4 CATCAG motifs detected in file\n",
      "\n",
      "Complement found from 3256 to 3262\n",
      "Complement found from 3462 to 3468\n",
      "Complement found from 4275 to 4281\n",
      "Complement found from 4778 to 4784\n",
      "Complement found from 5405 to 5411\n",
      "Complement found from 6413 to 6419\n",
      "Complement found from 7433 to 7439\n",
      "Complement found from 8488 to 8494\n",
      "Complement found from 8494 to 8500\n",
      "Complement found from 10214 to 10220\n",
      "Complement found from 12034 to 12040\n",
      "Complement found from 12327 to 12333\n",
      "Complement found from 13872 to 13878\n",
      "Complement found from 14776 to 14782\n",
      "Complement found from 15392 to 15398\n",
      "Complement found from 16159 to 16165\n",
      "Seq is: ATAAAA\n",
      "16 ATAAAA motifs detected in file\n",
      "\n",
      "Complement found from 572 to 578\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Complement found from 8194 to 8200\n",
      "Complement found from 8858 to 8864\n",
      "Complement found from 11770 to 11776\n",
      "Complement found from 15174 to 15180\n",
      "Seq is: CACAGT\n",
      "5 CACAGT motifs detected in file\n",
      "\n",
      "Complement found from 2679 to 2685\n",
      "Complement found from 9285 to 9291\n",
      "Seq is: TGACCT\n",
      "2 TGACCT motifs detected in file\n",
      "\n",
      "Complement found from 284 to 290\n",
      "Complement found from 355 to 361\n",
      "Complement found from 2501 to 2507\n",
      "Complement found from 4603 to 4609\n",
      "Complement found from 4868 to 4874\n",
      "Complement found from 5280 to 5286\n",
      "Complement found from 7445 to 7451\n",
      "Complement found from 7513 to 7519\n",
      "Complement found from 9431 to 9437\n",
      "Complement found from 10046 to 10052\n",
      "Complement found from 10379 to 10385\n",
      "Complement found from 10812 to 10818\n",
      "Complement found from 12304 to 12310\n",
      "Complement found from 12416 to 12422\n",
      "Complement found from 13229 to 13235\n",
      "Complement found from 14073 to 14079\n",
      "Seq is: CAAAAA\n",
      "16 CAAAAA motifs detected in file\n",
      "\n",
      "Complement found from 2989 to 2995\n",
      "Complement found from 4443 to 4449\n",
      "Seq is: ATGTTG\n",
      "2 ATGTTG motifs detected in file\n",
      "\n",
      "Complement found from 2096 to 2102\n",
      "Complement found from 2957 to 2963\n",
      "Complement found from 5862 to 5868\n",
      "Seq is: AGTCCA\n",
      "3 AGTCCA motifs detected in file\n",
      "\n",
      "Complement found from 594 to 600\n",
      "Complement found from 777 to 783\n",
      "Complement found from 1162 to 1168\n",
      "Complement found from 3677 to 3683\n",
      "Complement found from 4252 to 4258\n",
      "Complement found from 8684 to 8690\n",
      "Complement found from 11751 to 11757\n",
      "Complement found from 13422 to 13428\n",
      "Complement found from 15874 to 15880\n",
      "Seq is: CTCAAA\n",
      "9 CTCAAA motifs detected in file\n",
      "\n",
      "Complement found from 3942 to 3948\n",
      "Complement found from 4342 to 4348\n",
      "Complement found from 5267 to 5273\n",
      "Complement found from 7459 to 7465\n",
      "Complement found from 8955 to 8961\n",
      "Complement found from 13521 to 13527\n",
      "Seq is: ATCGAA\n",
      "6 ATCGAA motifs detected in file\n",
      "\n",
      "Complement found from 1807 to 1813\n",
      "Complement found from 4724 to 4730\n",
      "Complement found from 5066 to 5072\n",
      "Complement found from 6428 to 6434\n",
      "Complement found from 8697 to 8703\n",
      "Complement found from 10607 to 10613\n",
      "Complement found from 10740 to 10746\n",
      "Complement found from 12336 to 12342\n",
      "Complement found from 12354 to 12360\n",
      "Complement found from 13672 to 13678\n",
      "Complement found from 14144 to 14150\n",
      "Complement found from 14523 to 14529\n",
      "Seq is: ATAACC\n",
      "12 ATAACC motifs detected in file\n",
      "\n",
      "Complement found from 307 to 313\n",
      "Complement found from 457 to 463\n",
      "Complement found from 958 to 964\n",
      "Complement found from 3115 to 3121\n",
      "Complement found from 3569 to 3575\n",
      "Complement found from 4105 to 4111\n",
      "Complement found from 5443 to 5449\n",
      "Complement found from 6218 to 6224\n",
      "Complement found from 6290 to 6296\n",
      "Complement found from 7815 to 7821\n",
      "Complement found from 7862 to 7868\n",
      "Complement found from 8470 to 8476\n",
      "Complement found from 10551 to 10557\n",
      "Complement found from 13447 to 13453\n",
      "Complement found from 13582 to 13588\n",
      "Complement found from 14261 to 14267\n",
      "Complement found from 14527 to 14533\n",
      "Complement found from 14805 to 14811\n",
      "Complement found from 15141 to 15147\n",
      "Complement found from 15380 to 15386\n",
      "Complement found from 15538 to 15544\n",
      "Complement found from 16186 to 16192\n",
      "Seq is: CCTCCC\n",
      "22 CCTCCC motifs detected in file\n",
      "\n",
      "Complement found from 1084 to 1090\n",
      "Complement found from 2217 to 2223\n",
      "Complement found from 4018 to 4024\n",
      "Complement found from 7033 to 7039\n",
      "Complement found from 10310 to 10316\n",
      "Complement found from 11020 to 11026\n",
      "Complement found from 11815 to 11821\n",
      "Complement found from 11870 to 11876\n",
      "Complement found from 12969 to 12975\n",
      "Complement found from 13146 to 13152\n",
      "Complement found from 13881 to 13887\n",
      "Complement found from 14390 to 14396\n",
      "Complement found from 15700 to 15706\n",
      "Complement found from 16266 to 16272\n",
      "Seq is: CCACTA\n",
      "14 CCACTA motifs detected in file\n",
      "\n",
      "Complement found from 4545 to 4551\n",
      "Complement found from 6609 to 6615\n",
      "Complement found from 9492 to 9498\n",
      "Seq is: GATTTT\n",
      "3 GATTTT motifs detected in file\n",
      "\n",
      "Seq is: GTGAGA\n",
      "0 GTGAGA motifs detected in file\n",
      "\n",
      "Seq is: AATGCG\n",
      "0 AATGCG motifs detected in file\n",
      "\n",
      "Complement found from 3906 to 3912\n",
      "Complement found from 3981 to 3987\n",
      "Seq is: GCCGAA\n",
      "2 GCCGAA motifs detected in file\n",
      "\n",
      "Complement found from 2933 to 2939\n",
      "Complement found from 8696 to 8702\n",
      "Complement found from 12275 to 12281\n",
      "Seq is: GATAAC\n",
      "3 GATAAC motifs detected in file\n",
      "\n",
      "Complement found from 3561 to 3567\n",
      "Complement found from 4718 to 4724\n",
      "Complement found from 7900 to 7906\n",
      "Complement found from 10394 to 10400\n",
      "Complement found from 11016 to 11022\n",
      "Seq is: TGAACC\n",
      "5 TGAACC motifs detected in file\n",
      "\n",
      "Complement found from 2189 to 2195\n",
      "Complement found from 2367 to 2373\n",
      "Complement found from 5016 to 5022\n",
      "Complement found from 6402 to 6408\n",
      "Complement found from 9536 to 9542\n",
      "Complement found from 10022 to 10028\n",
      "Complement found from 10781 to 10787\n",
      "Complement found from 13011 to 13017\n",
      "Complement found from 14168 to 14174\n",
      "Complement found from 15507 to 15513\n",
      "Seq is: CAATTA\n",
      "10 CAATTA motifs detected in file\n",
      "\n",
      "Complement found from 512 to 518\n",
      "Complement found from 1940 to 1946\n",
      "Complement found from 4515 to 4521\n",
      "Complement found from 6767 to 6773\n",
      "Complement found from 8920 to 8926\n",
      "Complement found from 12344 to 12350\n",
      "Seq is: GCACAC\n",
      "6 GCACAC motifs detected in file\n",
      "\n",
      "Complement found from 1765 to 1771\n",
      "Complement found from 9377 to 9383\n",
      "Complement found from 14856 to 14862\n",
      "Complement found from 15000 to 15006\n",
      "Seq is: TGGCGC\n",
      "4 TGGCGC motifs detected in file\n",
      "\n",
      "Complement found from 7914 to 7920\n",
      "Seq is: CACCGA\n",
      "1 CACCGA motifs detected in file\n",
      "\n",
      "Complement found from 2582 to 2588\n",
      "Complement found from 4373 to 4379\n",
      "Seq is: CCGTGC\n",
      "2 CCGTGC motifs detected in file\n",
      "\n",
      "Complement found from 421 to 427\n",
      "Complement found from 847 to 853\n",
      "Complement found from 1032 to 1038\n",
      "Complement found from 1519 to 1525\n",
      "Complement found from 2086 to 2092\n",
      "Complement found from 2156 to 2162\n",
      "Complement found from 2560 to 2566\n",
      "Complement found from 2664 to 2670\n",
      "Complement found from 3783 to 3789\n",
      "Complement found from 5080 to 5086\n",
      "Complement found from 10882 to 10888\n",
      "Complement found from 12147 to 12153\n",
      "Complement found from 15969 to 15975\n",
      "Seq is: TTTAAC\n",
      "13 TTTAAC motifs detected in file\n",
      "\n",
      "Complement found from 2016 to 2022\n",
      "Seq is: TGGTTG\n",
      "1 TGGTTG motifs detected in file\n",
      "\n",
      "Seq is: GGACCC\n",
      "0 GGACCC motifs detected in file\n",
      "\n",
      "Complement found from 2388 to 2394\n",
      "Complement found from 4738 to 4744\n",
      "Complement found from 4887 to 4893\n",
      "Complement found from 5370 to 5376\n",
      "Complement found from 9215 to 9221\n",
      "Complement found from 9633 to 9639\n",
      "Complement found from 10985 to 10991\n",
      "Complement found from 11627 to 11633\n",
      "Complement found from 14573 to 14579\n",
      "Complement found from 15419 to 15425\n",
      "Complement found from 15709 to 15715\n",
      "Complement found from 16213 to 16219\n",
      "Seq is: CAATCA\n",
      "12 CAATCA motifs detected in file\n",
      "\n",
      "Complement found from 5266 to 5272\n",
      "Complement found from 8954 to 8960\n",
      "Seq is: TATCGA\n",
      "2 TATCGA motifs detected in file\n",
      "\n",
      "Complement found from 483 to 489\n",
      "Complement found from 4740 to 4746\n",
      "Complement found from 6386 to 6392\n",
      "Complement found from 6407 to 6413\n",
      "Complement found from 6736 to 6742\n",
      "Complement found from 7045 to 7051\n",
      "Complement found from 7878 to 7884\n",
      "Complement found from 9631 to 9637\n",
      "Complement found from 14575 to 14581\n",
      "Complement found from 14937 to 14943\n",
      "Complement found from 16416 to 16422\n",
      "Seq is: ATCAAT\n",
      "11 ATCAAT motifs detected in file\n",
      "\n",
      "Complement found from 1757 to 1763\n",
      "Complement found from 8240 to 8246\n",
      "Complement found from 15860 to 15866\n",
      "Seq is: TTGAAA\n",
      "3 TTGAAA motifs detected in file\n",
      "\n",
      "Complement found from 2477 to 2483\n",
      "Complement found from 5622 to 5628\n",
      "Complement found from 13000 to 13006\n",
      "Seq is: GCAAAT\n",
      "3 GCAAAT motifs detected in file\n",
      "\n",
      "Complement found from 3043 to 3049\n",
      "Complement found from 7855 to 7861\n",
      "Seq is: AACGAT\n",
      "2 AACGAT motifs detected in file\n",
      "\n",
      "Complement found from 10019 to 10025\n",
      "Complement found from 10809 to 10815\n",
      "Complement found from 12747 to 12753\n",
      "Complement found from 13767 to 13773\n",
      "Complement found from 15942 to 15948\n",
      "Seq is: TTCCAA\n",
      "5 TTCCAA motifs detected in file\n",
      "\n",
      "Seq is: TTAGTG\n",
      "0 TTAGTG motifs detected in file\n",
      "\n",
      "Complement found from 7005 to 7011\n",
      "Complement found from 9193 to 9199\n",
      "Complement found from 11589 to 11595\n",
      "Seq is: ACGACA\n",
      "3 ACGACA motifs detected in file\n",
      "\n",
      "Seq is: AAGGGC\n",
      "0 AAGGGC motifs detected in file\n",
      "\n",
      "Complement found from 301 to 307\n",
      "Complement found from 431 to 437\n",
      "Complement found from 492 to 498\n",
      "Complement found from 566 to 572\n",
      "Complement found from 800 to 806\n",
      "Complement found from 954 to 960\n",
      "Complement found from 1899 to 1905\n",
      "Complement found from 3564 to 3570\n",
      "Complement found from 3583 to 3589\n",
      "Complement found from 3891 to 3897\n",
      "Complement found from 4135 to 4141\n",
      "Complement found from 4316 to 4322\n",
      "Complement found from 5893 to 5899\n",
      "Complement found from 6418 to 6424\n",
      "Complement found from 7464 to 7470\n",
      "Complement found from 8270 to 8276\n",
      "Complement found from 8279 to 8285\n",
      "Complement found from 8404 to 8410\n",
      "Complement found from 9530 to 9536\n",
      "Complement found from 10934 to 10940\n",
      "Complement found from 10945 to 10951\n",
      "Complement found from 11510 to 11516\n",
      "Complement found from 11671 to 11677\n",
      "Complement found from 11865 to 11871\n",
      "Complement found from 12430 to 12436\n",
      "Complement found from 13126 to 13132\n",
      "Complement found from 14423 to 14429\n",
      "Complement found from 14586 to 14592\n",
      "Complement found from 14768 to 14774\n",
      "Complement found from 15365 to 15371\n",
      "Complement found from 16182 to 16188\n",
      "Complement found from 16373 to 16379\n",
      "Seq is: ACCCCC\n",
      "32 ACCCCC motifs detected in file\n",
      "\n",
      "Complement found from 520 to 526\n",
      "Complement found from 1479 to 1485\n",
      "Complement found from 7913 to 7919\n",
      "Complement found from 13934 to 13940\n",
      "Complement found from 14563 to 14569\n",
      "Seq is: ACACCG\n",
      "5 ACACCG motifs detected in file\n",
      "\n",
      "Complement found from 16491 to 16497\n",
      "Seq is: ATCCGA\n",
      "1 ATCCGA motifs detected in file\n",
      "\n",
      "Complement found from 9485 to 9491\n",
      "Complement found from 13194 to 13200\n",
      "Complement found from 13722 to 13728\n",
      "Seq is: TTCGCA\n",
      "3 TTCGCA motifs detected in file\n",
      "\n",
      "Complement found from 975 to 981\n",
      "Complement found from 995 to 1001\n",
      "Complement found from 1159 to 1165\n",
      "Complement found from 3465 to 3471\n",
      "Complement found from 3674 to 3680\n",
      "Complement found from 5131 to 5137\n",
      "Complement found from 6979 to 6985\n",
      "Complement found from 11035 to 11041\n",
      "Complement found from 11319 to 11325\n",
      "Complement found from 11748 to 11754\n",
      "Complement found from 11804 to 11810\n",
      "Complement found from 12422 to 12428\n",
      "Complement found from 12659 to 12665\n",
      "Complement found from 13156 to 13162\n",
      "Complement found from 13794 to 13800\n",
      "Seq is: AAACTC\n",
      "15 AAACTC motifs detected in file\n",
      "\n",
      "Complement found from 6794 to 6800\n",
      "Complement found from 14959 to 14965\n",
      "Seq is: GACGTA\n",
      "2 GACGTA motifs detected in file\n",
      "\n",
      "Seq is: ACGACG\n",
      "0 ACGACG motifs detected in file\n",
      "\n",
      "Complement found from 11027 to 11033\n",
      "Complement found from 15346 to 15352\n",
      "Seq is: CACGAA\n",
      "2 CACGAA motifs detected in file\n",
      "\n",
      "Complement found from 1019 to 1025\n",
      "Complement found from 1142 to 1148\n",
      "Complement found from 1384 to 1390\n",
      "Complement found from 4148 to 4154\n",
      "Complement found from 7905 to 7911\n",
      "Complement found from 11587 to 11593\n",
      "Complement found from 11757 to 11763\n",
      "Seq is: CTACGA\n",
      "7 CTACGA motifs detected in file\n",
      "\n",
      "Complement found from 8915 to 8921\n",
      "Seq is: ACAAGG\n",
      "1 ACAAGG motifs detected in file\n",
      "\n",
      "Complement found from 974 to 980\n",
      "Complement found from 994 to 1000\n",
      "Complement found from 1123 to 1129\n",
      "Complement found from 1158 to 1164\n",
      "Complement found from 1380 to 1386\n",
      "Complement found from 3258 to 3264\n",
      "Complement found from 3265 to 3271\n",
      "Complement found from 3464 to 3470\n",
      "Complement found from 4177 to 4183\n",
      "Complement found from 4780 to 4786\n",
      "Complement found from 4870 to 4876\n",
      "Complement found from 7728 to 7734\n",
      "Complement found from 10764 to 10770\n",
      "Complement found from 11034 to 11040\n",
      "Complement found from 11466 to 11472\n",
      "Complement found from 12421 to 12427\n",
      "Complement found from 13793 to 13799\n",
      "Complement found from 14506 to 14512\n",
      "Complement found from 14761 to 14767\n",
      "Seq is: AAAACT\n",
      "19 AAAACT motifs detected in file\n",
      "\n",
      "Complement found from 107 to 113\n",
      "Complement found from 1621 to 1627\n",
      "Complement found from 5138 to 5144\n",
      "Complement found from 5701 to 5707\n",
      "Complement found from 8267 to 8273\n",
      "Complement found from 14038 to 14044\n",
      "Complement found from 14362 to 14368\n",
      "Complement found from 15984 to 15990\n",
      "Seq is: AGCACC\n",
      "8 AGCACC motifs detected in file\n",
      "\n",
      "Complement found from 196 to 202\n",
      "Complement found from 1523 to 1529\n",
      "Complement found from 4912 to 4918\n",
      "Complement found from 8373 to 8379\n",
      "Complement found from 8439 to 8445\n",
      "Complement found from 8712 to 8718\n",
      "Complement found from 11277 to 11283\n",
      "Complement found from 13685 to 13691\n",
      "Complement found from 14392 to 14398\n",
      "Complement found from 14581 to 14587\n",
      "Complement found from 14635 to 14641\n",
      "Seq is: ACTAAA\n",
      "11 ACTAAA motifs detected in file\n",
      "\n",
      "Complement found from 4054 to 4060\n",
      "Complement found from 4904 to 4910\n",
      "Complement found from 5214 to 5220\n",
      "Complement found from 5484 to 5490\n",
      "Complement found from 5731 to 5737\n",
      "Complement found from 6496 to 6502\n",
      "Complement found from 6663 to 6669\n",
      "Complement found from 9755 to 9761\n",
      "Complement found from 11057 to 11063\n",
      "Complement found from 12562 to 12568\n",
      "Complement found from 15851 to 15857\n",
      "Seq is: TCTCCC\n",
      "11 TCTCCC motifs detected in file\n",
      "\n",
      "Complement found from 1655 to 1661\n",
      "Complement found from 2904 to 2910\n",
      "Complement found from 16140 to 16146\n",
      "Seq is: ACTTGA\n",
      "3 ACTTGA motifs detected in file\n",
      "\n",
      "Seq is: GTGACC\n",
      "0 GTGACC motifs detected in file\n",
      "\n",
      "Complement found from 2053 to 2059\n",
      "Complement found from 5249 to 5255\n",
      "Complement found from 8554 to 8560\n",
      "Complement found from 10263 to 10269\n",
      "Complement found from 15298 to 15304\n",
      "Complement found from 15620 to 15626\n",
      "Seq is: TTGCCC\n",
      "6 TTGCCC motifs detected in file\n",
      "\n",
      "Complement found from 1019 to 1025\n",
      "Complement found from 1142 to 1148\n",
      "Complement found from 1384 to 1390\n",
      "Complement found from 4148 to 4154\n",
      "Complement found from 7905 to 7911\n",
      "Complement found from 11587 to 11593\n",
      "Complement found from 11757 to 11763\n",
      "Seq is: CTACGA\n",
      "7 CTACGA motifs detected in file\n",
      "\n",
      "Complement found from 7010 to 7016\n",
      "Complement found from 10734 to 10740\n",
      "Seq is: ACGTAC\n",
      "2 ACGTAC motifs detected in file\n",
      "\n",
      "Complement found from 445 to 451\n",
      "Complement found from 3762 to 3768\n",
      "Complement found from 3994 to 4000\n",
      "Complement found from 8083 to 8089\n",
      "Complement found from 9013 to 9019\n",
      "Complement found from 12023 to 12029\n",
      "Complement found from 12673 to 12679\n",
      "Complement found from 13660 to 13666\n",
      "Complement found from 15466 to 15472\n",
      "Complement found from 16097 to 16103\n",
      "Complement found from 16337 to 16343\n",
      "Seq is: ACATTA\n",
      "11 ACATTA motifs detected in file\n",
      "\n",
      "Complement found from 10426 to 10432\n",
      "Complement found from 15556 to 15562\n",
      "Seq is: GAATGA\n",
      "2 GAATGA motifs detected in file\n",
      "\n",
      "Complement found from 7191 to 7197\n",
      "Complement found from 14675 to 14681\n",
      "Complement found from 16355 to 16361\n",
      "Seq is: TTCTCG\n",
      "3 TTCTCG motifs detected in file\n",
      "\n",
      "Complement found from 5093 to 5099\n",
      "Complement found from 5264 to 5270\n",
      "Complement found from 6404 to 6410\n",
      "Complement found from 8952 to 8958\n",
      "Complement found from 12468 to 12474\n",
      "Complement found from 12519 to 12525\n",
      "Complement found from 15094 to 15100\n",
      "Seq is: ATTATC\n",
      "7 ATTATC motifs detected in file\n",
      "\n",
      "Complement found from 9768 to 9774\n",
      "Seq is: TTTCCG\n",
      "1 TTTCCG motifs detected in file\n",
      "\n",
      "Complement found from 7783 to 7789\n",
      "Seq is: ATCCTG\n",
      "1 ATCCTG motifs detected in file\n",
      "\n",
      "Complement found from 652 to 658\n",
      "Seq is: GTTTGG\n",
      "1 GTTTGG motifs detected in file\n",
      "\n",
      "Complement found from 640 to 646\n",
      "Complement found from 836 to 842\n",
      "Complement found from 1217 to 1223\n",
      "Complement found from 4005 to 4011\n",
      "Complement found from 4571 to 4577\n",
      "Complement found from 4610 to 4616\n",
      "Complement found from 6191 to 6197\n",
      "Complement found from 7370 to 7376\n",
      "Complement found from 8823 to 8829\n",
      "Complement found from 10058 to 10064\n",
      "Complement found from 12657 to 12663\n",
      "Complement found from 15898 to 15904\n",
      "Seq is: ATAAAC\n",
      "12 ATAAAC motifs detected in file\n",
      "\n",
      "Complement found from 5524 to 5530\n",
      "Complement found from 8376 to 8382\n",
      "Complement found from 15869 to 15875\n",
      "Complement found from 16136 to 16142\n",
      "Seq is: AAATAC\n",
      "4 AAATAC motifs detected in file\n",
      "\n",
      "Complement found from 460 to 466\n",
      "Complement found from 1083 to 1089\n",
      "Complement found from 1688 to 1694\n",
      "Complement found from 2216 to 2222\n",
      "Complement found from 5601 to 5607\n",
      "Complement found from 5896 to 5902\n",
      "Complement found from 7026 to 7032\n",
      "Complement found from 8292 to 8298\n",
      "Complement found from 8902 to 8908\n",
      "Complement found from 9592 to 9598\n",
      "Complement found from 10620 to 10626\n",
      "Complement found from 11814 to 11820\n",
      "Complement found from 11869 to 11875\n",
      "Complement found from 12968 to 12974\n",
      "Complement found from 13145 to 13151\n",
      "Complement found from 13880 to 13886\n",
      "Complement found from 14114 to 14120\n",
      "Complement found from 14389 to 14395\n",
      "Complement found from 15699 to 15705\n",
      "Complement found from 16265 to 16271\n",
      "Seq is: CCCACT\n",
      "20 CCCACT motifs detected in file\n",
      "\n",
      "Complement found from 4517 to 4523\n",
      "Complement found from 5450 to 5456\n",
      "Complement found from 6878 to 6884\n",
      "Complement found from 7717 to 7723\n",
      "Complement found from 11255 to 11261\n",
      "Complement found from 11497 to 11503\n",
      "Complement found from 12908 to 12914\n",
      "Complement found from 14398 to 14404\n",
      "Complement found from 14644 to 14650\n",
      "Complement found from 15325 to 15331\n",
      "Seq is: ACACTC\n",
      "10 ACACTC motifs detected in file\n",
      "\n",
      "Seq is: GGGCAG\n",
      "0 GGGCAG motifs detected in file\n",
      "\n",
      "Complement found from 6088 to 6094\n",
      "Seq is: ATTTGT\n",
      "1 ATTTGT motifs detected in file\n",
      "\n",
      "Complement found from 2459 to 2465\n",
      "Complement found from 7342 to 7348\n",
      "Complement found from 12329 to 12335\n",
      "Complement found from 15962 to 15968\n",
      "Seq is: AAAAGT\n",
      "4 AAAAGT motifs detected in file\n",
      "\n",
      "Complement found from 5024 to 5030\n",
      "Complement found from 9200 to 9206\n",
      "Complement found from 9400 to 9406\n",
      "Complement found from 10717 to 10723\n",
      "Complement found from 10819 to 10825\n",
      "Complement found from 11196 to 11202\n",
      "Complement found from 11524 to 11530\n",
      "Complement found from 11635 to 11641\n",
      "Complement found from 14141 to 14147\n",
      "Seq is: CACATA\n",
      "9 CACATA motifs detected in file\n",
      "\n",
      "Complement found from 840 to 846\n",
      "Complement found from 1021 to 1027\n",
      "Complement found from 3124 to 3130\n",
      "Complement found from 3377 to 3383\n",
      "Complement found from 8530 to 8536\n",
      "Complement found from 11028 to 11034\n",
      "Complement found from 13666 to 13672\n",
      "Complement found from 15347 to 15353\n",
      "Seq is: ACGAAA\n",
      "8 ACGAAA motifs detected in file\n",
      "\n",
      "Complement found from 591 to 597\n",
      "Complement found from 1494 to 1500\n",
      "Complement found from 2251 to 2257\n",
      "Complement found from 2325 to 2331\n",
      "Complement found from 3327 to 3333\n",
      "Complement found from 4928 to 4934\n",
      "Complement found from 5011 to 5017\n",
      "Complement found from 5210 to 5216\n",
      "Complement found from 6367 to 6373\n",
      "Complement found from 8775 to 8781\n",
      "Complement found from 9329 to 9335\n",
      "Complement found from 12091 to 12097\n",
      "Complement found from 12981 to 12987\n",
      "Complement found from 13986 to 13992\n",
      "Complement found from 14442 to 14448\n",
      "Complement found from 15739 to 15745\n",
      "Complement found from 16443 to 16449\n",
      "Seq is: CTCCTC\n",
      "17 CTCCTC motifs detected in file\n",
      "\n",
      "Complement found from 7413 to 7419\n",
      "Complement found from 8031 to 8037\n",
      "Seq is: CATTCG\n",
      "2 CATTCG motifs detected in file\n",
      "\n",
      "Complement found from 16086 to 16092\n",
      "Seq is: ATGTAT\n",
      "1 ATGTAT motifs detected in file\n",
      "\n",
      "Complement found from 1004 to 1010\n",
      "Complement found from 7997 to 8003\n",
      "Complement found from 10035 to 10041\n",
      "Seq is: TTGACA\n",
      "3 TTGACA motifs detected in file\n",
      "\n",
      "Complement found from 302 to 308\n",
      "Complement found from 432 to 438\n",
      "Complement found from 567 to 573\n",
      "Complement found from 3565 to 3571\n",
      "Complement found from 7396 to 7402\n",
      "Complement found from 7465 to 7471\n",
      "Complement found from 9531 to 9537\n",
      "Complement found from 10946 to 10952\n",
      "Complement found from 11866 to 11872\n",
      "Complement found from 12384 to 12390\n",
      "Complement found from 16374 to 16380\n",
      "Seq is: CCCCCC\n",
      "11 CCCCCC motifs detected in file\n",
      "\n",
      "Complement found from 2059 to 2065\n",
      "Complement found from 4625 to 4631\n",
      "Complement found from 11095 to 11101\n",
      "Complement found from 14651 to 14657\n",
      "Seq is: ACAGAA\n",
      "4 ACAGAA motifs detected in file\n",
      "\n",
      "Complement found from 351 to 357\n",
      "Complement found from 532 to 538\n",
      "Complement found from 555 to 561\n",
      "Complement found from 635 to 641\n",
      "Complement found from 865 to 871\n",
      "Complement found from 1081 to 1087\n",
      "Complement found from 1373 to 1379\n",
      "Complement found from 4389 to 4395\n",
      "Complement found from 5435 to 5441\n",
      "Complement found from 5599 to 5605\n",
      "Complement found from 6582 to 6588\n",
      "Complement found from 7489 to 7495\n",
      "Complement found from 12966 to 12972\n",
      "Complement found from 13056 to 13062\n",
      "Complement found from 13675 to 13681\n",
      "Complement found from 13690 to 13696\n",
      "Complement found from 14338 to 14344\n",
      "Complement found from 14387 to 14393\n",
      "Complement found from 14618 to 14624\n",
      "Complement found from 14627 to 14633\n",
      "Complement found from 14749 to 14755\n",
      "Complement found from 14812 to 14818\n",
      "Seq is: ACCCCA\n",
      "22 ACCCCA motifs detected in file\n",
      "\n",
      "Complement found from 545 to 551\n",
      "Complement found from 1809 to 1815\n",
      "Complement found from 2393 to 2399\n",
      "Complement found from 4600 to 4606\n",
      "Complement found from 4726 to 4732\n",
      "Complement found from 4979 to 4985\n",
      "Complement found from 8122 to 8128\n",
      "Complement found from 8342 to 8348\n",
      "Complement found from 8519 to 8525\n",
      "Complement found from 8980 to 8986\n",
      "Complement found from 9349 to 9355\n",
      "Complement found from 9674 to 9680\n",
      "Complement found from 10885 to 10891\n",
      "Complement found from 12150 to 12156\n",
      "Complement found from 13293 to 13299\n",
      "Complement found from 13856 to 13862\n",
      "Seq is: AACCAA\n",
      "16 AACCAA motifs detected in file\n",
      "\n",
      "Complement found from 1811 to 1817\n",
      "Complement found from 9152 to 9158\n",
      "Complement found from 12957 to 12963\n",
      "Seq is: CCAAGC\n",
      "3 CCAAGC motifs detected in file\n",
      "\n",
      "Complement found from 466 to 472\n",
      "Complement found from 534 to 540\n",
      "Complement found from 637 to 643\n",
      "Complement found from 2163 to 2169\n",
      "Complement found from 3573 to 3579\n",
      "Complement found from 4429 to 4435\n",
      "Complement found from 5384 to 5390\n",
      "Complement found from 6666 to 6672\n",
      "Complement found from 8407 to 8413\n",
      "Complement found from 8485 to 8491\n",
      "Complement found from 8940 to 8946\n",
      "Complement found from 14226 to 14232\n",
      "Complement found from 14518 to 14524\n",
      "Complement found from 14589 to 14595\n",
      "Complement found from 15204 to 15210\n",
      "Complement found from 16457 to 16463\n",
      "Seq is: CCCATA\n",
      "16 CCCATA motifs detected in file\n",
      "\n",
      "Complement found from 352 to 358\n",
      "Complement found from 434 to 440\n",
      "Complement found from 556 to 562\n",
      "Complement found from 961 to 967\n",
      "Complement found from 1681 to 1687\n",
      "Complement found from 3413 to 3419\n",
      "Complement found from 7467 to 7473\n",
      "Complement found from 8368 to 8374\n",
      "Complement found from 9533 to 9539\n",
      "Complement found from 9555 to 9561\n",
      "Complement found from 10916 to 10922\n",
      "Complement found from 12301 to 12307\n",
      "Complement found from 14531 to 14537\n",
      "Complement found from 14750 to 14756\n",
      "Seq is: CCCCAA\n",
      "14 CCCCAA motifs detected in file\n",
      "\n",
      "Seq is: TCGTCG\n",
      "0 TCGTCG motifs detected in file\n",
      "\n",
      "Complement found from 267 to 273\n",
      "Complement found from 796 to 802\n",
      "Complement found from 3204 to 3210\n",
      "Complement found from 3868 to 3874\n",
      "Complement found from 5448 to 5454\n",
      "Complement found from 6876 to 6882\n",
      "Complement found from 7408 to 7414\n",
      "Complement found from 9415 to 9421\n",
      "Complement found from 11129 to 11135\n",
      "Complement found from 13299 to 13305\n",
      "Complement found from 14561 to 14567\n",
      "Complement found from 14642 to 14648\n",
      "Complement found from 16536 to 16542\n",
      "Seq is: CCACAC\n",
      "13 CCACAC motifs detected in file\n",
      "\n",
      "Complement found from 514 to 520\n",
      "Complement found from 1046 to 1052\n",
      "Complement found from 1477 to 1483\n",
      "Complement found from 6801 to 6807\n",
      "Complement found from 9354 to 9360\n",
      "Complement found from 9417 to 9423\n",
      "Complement found from 13932 to 13938\n",
      "Complement found from 14551 to 14557\n",
      "Seq is: ACACAC\n",
      "8 ACACAC motifs detected in file\n",
      "\n",
      "Complement found from 1061 to 1067\n",
      "Complement found from 9208 to 9214\n",
      "Complement found from 12666 to 12672\n",
      "Complement found from 12924 to 12930\n",
      "Complement found from 15499 to 15505\n",
      "Seq is: GACCCA\n",
      "5 GACCCA motifs detected in file\n",
      "\n",
      "Complement found from 503 to 509\n",
      "Complement found from 3054 to 3060\n",
      "Complement found from 3322 to 3328\n",
      "Complement found from 4183 to 4189\n",
      "Complement found from 5115 to 5121\n",
      "Complement found from 5475 to 5481\n",
      "Complement found from 6228 to 6234\n",
      "Complement found from 6480 to 6486\n",
      "Complement found from 6642 to 6648\n",
      "Complement found from 7942 to 7948\n",
      "Complement found from 9729 to 9735\n",
      "Complement found from 10973 to 10979\n",
      "Complement found from 11936 to 11942\n",
      "Complement found from 12904 to 12910\n",
      "Complement found from 14307 to 14313\n",
      "Complement found from 14370 to 14376\n",
      "Complement found from 15029 to 15035\n",
      "Complement found from 16505 to 16511\n",
      "Seq is: TCCTAC\n",
      "18 TCCTAC motifs detected in file\n",
      "\n",
      "Complement found from 496 to 502\n",
      "Complement found from 1482 to 1488\n",
      "Complement found from 3523 to 3529\n",
      "Complement found from 10194 to 10200\n",
      "Seq is: CCGCCC\n",
      "4 CCGCCC motifs detected in file\n",
      "\n",
      "Complement found from 2395 to 2401\n",
      "Complement found from 2426 to 2432\n",
      "Complement found from 6595 to 6601\n",
      "Complement found from 8344 to 8350\n",
      "Complement found from 8660 to 8666\n",
      "Complement found from 9351 to 9357\n",
      "Complement found from 9557 to 9563\n",
      "Complement found from 10713 to 10719\n",
      "Complement found from 10777 to 10783\n",
      "Complement found from 11330 to 11336\n",
      "Complement found from 12826 to 12832\n",
      "Complement found from 13858 to 13864\n",
      "Complement found from 13903 to 13909\n",
      "Complement found from 14185 to 14191\n",
      "Complement found from 14819 to 14825\n",
      "Complement found from 16277 to 16283\n",
      "Seq is: CCAACA\n",
      "16 CCAACA motifs detected in file\n",
      "\n",
      "Complement found from 849 to 855\n",
      "Complement found from 1521 to 1527\n",
      "Complement found from 1861 to 1867\n",
      "Complement found from 5082 to 5088\n",
      "Complement found from 5100 to 5106\n",
      "Complement found from 7733 to 7739\n",
      "Complement found from 10026 to 10032\n",
      "Complement found from 10961 to 10967\n",
      "Complement found from 14207 to 14213\n",
      "Seq is: TAACTA\n",
      "9 TAACTA motifs detected in file\n",
      "\n",
      "Complement found from 725 to 731\n",
      "Complement found from 2067 to 2073\n",
      "Complement found from 5997 to 6003\n",
      "Complement found from 7284 to 7290\n",
      "Complement found from 8879 to 8885\n",
      "Complement found from 13159 to 13165\n",
      "Seq is: CTCTAA\n",
      "6 CTCTAA motifs detected in file\n",
      "\n",
      "Complement found from 481 to 487\n",
      "Complement found from 4521 to 4527\n",
      "Complement found from 4748 to 4754\n",
      "Complement found from 5295 to 5301\n",
      "Complement found from 6393 to 6399\n",
      "Complement found from 6983 to 6989\n",
      "Complement found from 8429 to 8435\n",
      "Complement found from 8635 to 8641\n",
      "Complement found from 10341 to 10347\n",
      "Complement found from 11153 to 11159\n",
      "Complement found from 12796 to 12802\n",
      "Complement found from 14059 to 14065\n",
      "Complement found from 14935 to 14941\n",
      "Seq is: TCATCA\n",
      "13 TCATCA motifs detected in file\n",
      "\n",
      "Complement found from 2410 to 2416\n",
      "Complement found from 2574 to 2580\n",
      "Complement found from 2924 to 2930\n",
      "Complement found from 4453 to 4459\n",
      "Complement found from 6287 to 6293\n",
      "Complement found from 8259 to 8265\n",
      "Complement found from 9724 to 9730\n",
      "Complement found from 13920 to 13926\n",
      "Complement found from 15513 to 15519\n",
      "Complement found from 15778 to 15784\n",
      "Seq is: TACCCT\n",
      "10 TACCCT motifs detected in file\n",
      "\n",
      "Complement found from 14 to 20\n",
      "Complement found from 109 to 115\n",
      "Complement found from 721 to 727\n",
      "Complement found from 1490 to 1496\n",
      "Complement found from 2279 to 2285\n",
      "Complement found from 3509 to 3515\n",
      "Complement found from 3737 to 3743\n",
      "Complement found from 3794 to 3800\n",
      "Complement found from 4010 to 4016\n",
      "Complement found from 4193 to 4199\n",
      "Complement found from 5190 to 5196\n",
      "Complement found from 5206 to 5212\n",
      "Complement found from 5323 to 5329\n",
      "Complement found from 5703 to 5709\n",
      "Complement found from 6314 to 6320\n",
      "Complement found from 6620 to 6626\n",
      "Complement found from 7111 to 7117\n",
      "Complement found from 7401 to 7407\n",
      "Complement found from 9056 to 9062\n",
      "Complement found from 10087 to 10093\n",
      "Complement found from 11214 to 11220\n",
      "Complement found from 11265 to 11271\n",
      "Complement found from 12060 to 12066\n",
      "Complement found from 12359 to 12365\n",
      "Complement found from 12398 to 12404\n",
      "Complement found from 13622 to 13628\n",
      "Complement found from 13649 to 13655\n",
      "Complement found from 13679 to 13685\n",
      "Complement found from 15264 to 15270\n",
      "Complement found from 15406 to 15412\n",
      "Complement found from 16291 to 16297\n",
      "Seq is: CACCCT\n",
      "31 CACCCT motifs detected in file\n",
      "\n",
      "Complement found from 2075 to 2081\n",
      "Complement found from 3893 to 3899\n",
      "Complement found from 3959 to 3965\n",
      "Complement found from 4318 to 4324\n",
      "Complement found from 4795 to 4801\n",
      "Complement found from 5487 to 5493\n",
      "Complement found from 8843 to 8849\n",
      "Complement found from 8931 to 8937\n",
      "Complement found from 10161 to 10167\n",
      "Complement found from 11535 to 11541\n",
      "Complement found from 12193 to 12199\n",
      "Complement found from 13763 to 13769\n",
      "Complement found from 15525 to 15531\n",
      "Complement found from 16545 to 16551\n",
      "Seq is: CCCCTT\n",
      "14 CCCCTT motifs detected in file\n",
      "\n",
      "Complement found from 2406 to 2412\n",
      "Complement found from 2742 to 2748\n",
      "Complement found from 3997 to 4003\n",
      "Complement found from 8950 to 8956\n",
      "Complement found from 9462 to 9468\n",
      "Complement found from 9696 to 9702\n",
      "Complement found from 10113 to 10119\n",
      "Complement found from 10244 to 10250\n",
      "Complement found from 10334 to 10340\n",
      "Complement found from 10856 to 10862\n",
      "Complement found from 12466 to 12472\n",
      "Complement found from 12517 to 12523\n",
      "Seq is: TTATTA\n",
      "12 TTATTA motifs detected in file\n",
      "\n",
      "Complement found from 1049 to 1055\n",
      "Complement found from 8561 to 8567\n",
      "Complement found from 9104 to 9110\n",
      "Complement found from 10983 to 10989\n",
      "Complement found from 11355 to 11361\n",
      "Complement found from 11999 to 12005\n",
      "Complement found from 13940 to 13946\n",
      "Complement found from 15417 to 15423\n",
      "Complement found from 15579 to 15585\n",
      "Seq is: CACAAT\n",
      "9 CACAAT motifs detected in file\n",
      "\n",
      "Complement found from 490 to 496\n",
      "Complement found from 2422 to 2428\n",
      "Complement found from 3444 to 3450\n",
      "Complement found from 3581 to 3587\n",
      "Complement found from 4484 to 4490\n",
      "Complement found from 5056 to 5062\n",
      "Complement found from 7487 to 7493\n",
      "Complement found from 10943 to 10949\n",
      "Complement found from 11508 to 11514\n",
      "Complement found from 12105 to 12111\n",
      "Complement found from 12544 to 12550\n",
      "Complement found from 12553 to 12559\n",
      "Complement found from 14068 to 14074\n",
      "Complement found from 14271 to 14277\n",
      "Complement found from 14414 to 14420\n",
      "Complement found from 15363 to 15369\n",
      "Complement found from 15522 to 15528\n",
      "Complement found from 16217 to 16223\n",
      "Seq is: CAACCC\n",
      "18 CAACCC motifs detected in file\n",
      "\n",
      "Complement found from 460 to 466\n",
      "Complement found from 1083 to 1089\n",
      "Complement found from 1688 to 1694\n",
      "Complement found from 2216 to 2222\n",
      "Complement found from 5601 to 5607\n",
      "Complement found from 5896 to 5902\n",
      "Complement found from 7026 to 7032\n",
      "Complement found from 8292 to 8298\n",
      "Complement found from 8902 to 8908\n",
      "Complement found from 9592 to 9598\n",
      "Complement found from 10620 to 10626\n",
      "Complement found from 11814 to 11820\n",
      "Complement found from 11869 to 11875\n",
      "Complement found from 12968 to 12974\n",
      "Complement found from 13145 to 13151\n",
      "Complement found from 13880 to 13886\n",
      "Complement found from 14114 to 14120\n",
      "Complement found from 14389 to 14395\n",
      "Complement found from 15699 to 15705\n",
      "Complement found from 16265 to 16271\n",
      "Seq is: CCCACT\n",
      "20 CCCACT motifs detected in file\n",
      "\n",
      "Complement found from 2365 to 2371\n",
      "Complement found from 4713 to 4719\n",
      "Complement found from 7999 to 8005\n",
      "Complement found from 15505 to 15511\n",
      "Seq is: GACAAT\n",
      "4 GACAAT motifs detected in file\n",
      "\n",
      "Complement found from 4799 to 4805\n",
      "Complement found from 6814 to 6820\n",
      "Complement found from 6943 to 6949\n",
      "Complement found from 7141 to 7147\n",
      "Complement found from 8133 to 8139\n",
      "Complement found from 9164 to 9170\n",
      "Complement found from 9308 to 9314\n",
      "Complement found from 9878 to 9884\n",
      "Complement found from 14032 to 14038\n",
      "Complement found from 14352 to 14358\n",
      "Complement found from 15286 to 15292\n",
      "Seq is: TTTCAC\n",
      "11 TTTCAC motifs detected in file\n",
      "\n",
      "Complement found from 6717 to 6723\n",
      "Complement found from 8386 to 8392\n",
      "Seq is: GTATGG\n",
      "2 GTATGG motifs detected in file\n",
      "\n",
      "Complement found from 3092 to 3098\n",
      "Complement found from 12875 to 12881\n",
      "Seq is: CGGTTT\n",
      "2 CGGTTT motifs detected in file\n",
      "\n",
      "Complement found from 668 to 674\n",
      "Complement found from 8597 to 8603\n",
      "Complement found from 8609 to 8615\n",
      "Complement found from 9717 to 9723\n",
      "Complement found from 9972 to 9978\n",
      "Complement found from 13563 to 13569\n",
      "Seq is: TCTATT\n",
      "6 TCTATT motifs detected in file\n",
      "\n",
      "Complement found from 332 to 338\n",
      "Complement found from 4312 to 4318\n",
      "Complement found from 4976 to 4982\n",
      "Complement found from 5129 to 5135\n",
      "Complement found from 5671 to 5677\n",
      "Complement found from 8449 to 8455\n",
      "Complement found from 10415 to 10421\n",
      "Complement found from 13696 to 13702\n",
      "Complement found from 14086 to 14092\n",
      "Complement found from 14513 to 14519\n",
      "Complement found from 15529 to 15535\n",
      "Complement found from 16008 to 16014\n",
      "Seq is: TTAAAC\n",
      "12 TTAAAC motifs detected in file\n",
      "\n",
      "Complement found from 617 to 623\n",
      "Complement found from 946 to 952\n",
      "Complement found from 5851 to 5857\n",
      "Seq is: TTTAGA\n",
      "3 TTTAGA motifs detected in file\n",
      "\n",
      "Complement found from 2270 to 2276\n",
      "Complement found from 4737 to 4743\n",
      "Complement found from 9214 to 9220\n",
      "Complement found from 14366 to 14372\n",
      "Complement found from 15708 to 15714\n",
      "Complement found from 16167 to 16173\n",
      "Seq is: CCAATC\n",
      "6 CCAATC motifs detected in file\n",
      "\n",
      "Complement found from 4151 to 4157\n",
      "Complement found from 6052 to 6058\n",
      "Complement found from 14558 to 14564\n",
      "Complement found from 14696 to 14702\n",
      "Seq is: CGACCA\n",
      "4 CGACCA motifs detected in file\n",
      "\n",
      "Complement found from 2943 to 2949\n",
      "Complement found from 3649 to 3655\n",
      "Complement found from 8563 to 8569\n",
      "Complement found from 12847 to 12853\n",
      "Complement found from 13777 to 13783\n",
      "Complement found from 13942 to 13948\n",
      "Complement found from 14367 to 14373\n",
      "Complement found from 15827 to 15833\n",
      "Complement found from 16168 to 16174\n",
      "Seq is: CAATCC\n",
      "9 CAATCC motifs detected in file\n",
      "\n",
      "Complement found from 1351 to 1357\n",
      "Complement found from 3215 to 3221\n",
      "Complement found from 3810 to 3816\n",
      "Complement found from 11308 to 11314\n",
      "Complement found from 12217 to 12223\n",
      "Complement found from 12510 to 12516\n",
      "Complement found from 14735 to 14741\n",
      "Seq is: CAAGAA\n",
      "7 CAAGAA motifs detected in file\n",
      "\n",
      "Complement found from 283 to 289\n",
      "Complement found from 1009 to 1015\n",
      "Complement found from 1121 to 1127\n",
      "Complement found from 4867 to 4873\n",
      "Complement found from 5279 to 5285\n",
      "Complement found from 5426 to 5432\n",
      "Complement found from 7444 to 7450\n",
      "Complement found from 7726 to 7732\n",
      "Complement found from 10378 to 10384\n",
      "Complement found from 10419 to 10425\n",
      "Complement found from 11518 to 11524\n",
      "Complement found from 12415 to 12421\n",
      "Complement found from 13217 to 13223\n",
      "Complement found from 15866 to 15872\n",
      "Seq is: ACAAAA\n",
      "14 ACAAAA motifs detected in file\n",
      "\n",
      "Complement found from 351 to 357\n",
      "Complement found from 532 to 538\n",
      "Complement found from 555 to 561\n",
      "Complement found from 635 to 641\n",
      "Complement found from 865 to 871\n",
      "Complement found from 1081 to 1087\n",
      "Complement found from 1373 to 1379\n",
      "Complement found from 4389 to 4395\n",
      "Complement found from 5435 to 5441\n",
      "Complement found from 5599 to 5605\n",
      "Complement found from 6582 to 6588\n",
      "Complement found from 7489 to 7495\n",
      "Complement found from 12966 to 12972\n",
      "Complement found from 13056 to 13062\n",
      "Complement found from 13675 to 13681\n",
      "Complement found from 13690 to 13696\n",
      "Complement found from 14338 to 14344\n",
      "Complement found from 14387 to 14393\n",
      "Complement found from 14618 to 14624\n",
      "Complement found from 14627 to 14633\n",
      "Complement found from 14749 to 14755\n",
      "Complement found from 14812 to 14818\n",
      "Seq is: ACCCCA\n",
      "22 ACCCCA motifs detected in file\n",
      "\n",
      "Complement found from 2580 to 2586\n",
      "Complement found from 7770 to 7776\n",
      "Complement found from 12858 to 12864\n",
      "Seq is: AACCGT\n",
      "3 AACCGT motifs detected in file\n",
      "\n",
      "Complement found from 3513 to 3519\n",
      "Complement found from 4065 to 4071\n",
      "Complement found from 4498 to 4504\n",
      "Complement found from 5336 to 5342\n",
      "Complement found from 5926 to 5932\n",
      "Complement found from 8275 to 8281\n",
      "Complement found from 9086 to 9092\n",
      "Complement found from 9183 to 9189\n",
      "Complement found from 10871 to 10877\n",
      "Complement found from 11038 to 11044\n",
      "Complement found from 11807 to 11813\n",
      "Complement found from 11980 to 11986\n",
      "Complement found from 13785 to 13791\n",
      "Complement found from 15067 to 15073\n",
      "Seq is: CTCTAC\n",
      "14 CTCTAC motifs detected in file\n",
      "\n",
      "Complement found from 644 to 650\n",
      "Complement found from 8690 to 8696\n",
      "Complement found from 11064 to 11070\n",
      "Complement found from 12932 to 12938\n",
      "Complement found from 15950 to 15956\n",
      "Seq is: ACAAAT\n",
      "5 ACAAAT motifs detected in file\n",
      "\n",
      "Complement found from 11026 to 11032\n",
      "Complement found from 16561 to 16567\n",
      "Seq is: TCACGA\n",
      "2 TCACGA motifs detected in file\n",
      "\n",
      "Complement found from 6847 to 6853\n",
      "Complement found from 11685 to 11691\n",
      "Seq is: CACCGG\n",
      "2 CACCGG motifs detected in file\n",
      "\n",
      "Complement found from 5785 to 5791\n",
      "Complement found from 6450 to 6456\n",
      "Complement found from 6558 to 6564\n",
      "Complement found from 9483 to 9489\n",
      "Complement found from 11120 to 11126\n",
      "Seq is: TCTTCG\n",
      "5 TCTTCG motifs detected in file\n",
      "\n",
      "Complement found from 304 to 310\n",
      "Complement found from 955 to 961\n",
      "Complement found from 3566 to 3572\n",
      "Complement found from 3584 to 3590\n",
      "Complement found from 3892 to 3898\n",
      "Complement found from 4248 to 4254\n",
      "Complement found from 4317 to 4323\n",
      "Complement found from 4794 to 4800\n",
      "Complement found from 6419 to 6425\n",
      "Complement found from 8271 to 8277\n",
      "Complement found from 8280 to 8286\n",
      "Complement found from 8604 to 8610\n",
      "Complement found from 10935 to 10941\n",
      "Complement found from 10947 to 10953\n",
      "Complement found from 11511 to 11517\n",
      "Complement found from 11672 to 11678\n",
      "Complement found from 13127 to 13133\n",
      "Complement found from 13762 to 13768\n",
      "Complement found from 13781 to 13787\n",
      "Complement found from 14488 to 14494\n",
      "Complement found from 14769 to 14775\n",
      "Complement found from 15366 to 15372\n",
      "Complement found from 16183 to 16189\n",
      "Complement found from 16375 to 16381\n",
      "Seq is: CCCCCT\n",
      "24 CCCCCT motifs detected in file\n",
      "\n",
      "Complement found from 302 to 308\n",
      "Complement found from 432 to 438\n",
      "Complement found from 567 to 573\n",
      "Complement found from 3565 to 3571\n",
      "Complement found from 7396 to 7402\n",
      "Complement found from 7465 to 7471\n",
      "Complement found from 9531 to 9537\n",
      "Complement found from 10946 to 10952\n",
      "Complement found from 11866 to 11872\n",
      "Complement found from 12384 to 12390\n",
      "Complement found from 16374 to 16380\n",
      "Seq is: CCCCCC\n",
      "11 CCCCCC motifs detected in file\n",
      "\n",
      "Complement found from 298 to 304\n",
      "Complement found from 348 to 354\n",
      "Complement found from 552 to 558\n",
      "Complement found from 1273 to 1279\n",
      "Complement found from 1684 to 1690\n",
      "Complement found from 1720 to 1726\n",
      "Complement found from 1991 to 1997\n",
      "Complement found from 2765 to 2771\n",
      "Complement found from 2788 to 2794\n",
      "Complement found from 4254 to 4260\n",
      "Complement found from 4982 to 4988\n",
      "Complement found from 5931 to 5937\n",
      "Complement found from 7121 to 7127\n",
      "Complement found from 8125 to 8131\n",
      "Complement found from 8189 to 8195\n",
      "Complement found from 8508 to 8514\n",
      "Complement found from 11668 to 11674\n",
      "Complement found from 14624 to 14630\n",
      "Complement found from 16281 to 16287\n",
      "Seq is: CAAACC\n",
      "19 CAAACC motifs detected in file\n",
      "\n",
      "Complement found from 826 to 832\n",
      "Complement found from 7648 to 7654\n",
      "Complement found from 8318 to 8324\n",
      "Complement found from 10921 to 10927\n",
      "Complement found from 12462 to 12468\n",
      "Complement found from 13484 to 13490\n",
      "Complement found from 15283 to 15289\n",
      "Complement found from 15936 to 15942\n",
      "Seq is: ACCTTT\n",
      "8 ACCTTT motifs detected in file\n",
      "\n",
      "Complement found from 285 to 291\n",
      "Complement found from 992 to 998\n",
      "Complement found from 2129 to 2135\n",
      "Complement found from 2150 to 2156\n",
      "Complement found from 2226 to 2232\n",
      "Complement found from 2456 to 2462\n",
      "Complement found from 4175 to 4181\n",
      "Complement found from 4604 to 4610\n",
      "Complement found from 5281 to 5287\n",
      "Complement found from 5746 to 5752\n",
      "Complement found from 6691 to 6697\n",
      "Complement found from 7446 to 7452\n",
      "Complement found from 8496 to 8502\n",
      "Complement found from 10047 to 10053\n",
      "Complement found from 10813 to 10819\n",
      "Complement found from 11031 to 11037\n",
      "Complement found from 12417 to 12423\n",
      "Complement found from 13230 to 13236\n",
      "Complement found from 14503 to 14509\n",
      "Seq is: AAAAAA\n",
      "19 AAAAAA motifs detected in file\n",
      "\n",
      "Complement found from 485 to 491\n",
      "Complement found from 601 to 607\n",
      "Complement found from 4729 to 4735\n",
      "Complement found from 4742 to 4748\n",
      "Complement found from 5563 to 5569\n",
      "Complement found from 6434 to 6440\n",
      "Complement found from 14577 to 14583\n",
      "Complement found from 14753 to 14759\n",
      "Seq is: CAATAC\n",
      "8 CAATAC motifs detected in file\n",
      "\n",
      "Complement found from 5527 to 5533\n",
      "Seq is: TACAGA\n",
      "1 TACAGA motifs detected in file\n",
      "\n",
      "Complement found from 519 to 525\n",
      "Complement found from 797 to 803\n",
      "Complement found from 1478 to 1484\n",
      "Complement found from 1941 to 1947\n",
      "Complement found from 2256 to 2262\n",
      "Complement found from 3205 to 3211\n",
      "Complement found from 4386 to 4392\n",
      "Complement found from 6768 to 6774\n",
      "Complement found from 8921 to 8927\n",
      "Complement found from 9418 to 9424\n",
      "Complement found from 10539 to 10545\n",
      "Complement found from 13300 to 13306\n",
      "Complement found from 13933 to 13939\n",
      "Complement found from 14552 to 14558\n",
      "Complement found from 14562 to 14568\n",
      "Seq is: CACACC\n",
      "15 CACACC motifs detected in file\n",
      "\n",
      "Complement found from 4799 to 4805\n",
      "Complement found from 6814 to 6820\n",
      "Complement found from 6943 to 6949\n",
      "Complement found from 7141 to 7147\n",
      "Complement found from 8133 to 8139\n",
      "Complement found from 9164 to 9170\n",
      "Complement found from 9308 to 9314\n",
      "Complement found from 9878 to 9884\n",
      "Complement found from 14032 to 14038\n",
      "Complement found from 14352 to 14358\n",
      "Complement found from 15286 to 15292\n",
      "Seq is: TTTCAC\n",
      "11 TTTCAC motifs detected in file\n",
      "\n",
      "Complement found from 7006 to 7012\n",
      "Seq is: CGACAC\n",
      "1 CGACAC motifs detected in file\n",
      "\n",
      "Seq is: GTCTGT\n",
      "0 GTCTGT motifs detected in file\n",
      "\n",
      "Complement found from 2920 to 2926\n",
      "Complement found from 7551 to 7557\n",
      "Complement found from 8325 to 8331\n",
      "Complement found from 12514 to 12520\n",
      "Seq is: AAGTTA\n",
      "4 AAGTTA motifs detected in file\n",
      "\n",
      "Complement found from 1050 to 1056\n",
      "Complement found from 2968 to 2974\n",
      "Complement found from 4698 to 4704\n",
      "Complement found from 5286 to 5292\n",
      "Complement found from 11356 to 11362\n",
      "Complement found from 12489 to 12495\n",
      "Complement found from 14173 to 14179\n",
      "Seq is: ACAATA\n",
      "7 ACAATA motifs detected in file\n",
      "\n",
      "Complement found from 231 to 237\n",
      "Complement found from 1817 to 1823\n",
      "Complement found from 3852 to 3858\n",
      "Complement found from 4002 to 4008\n",
      "Complement found from 4301 to 4307\n",
      "Complement found from 4763 to 4769\n",
      "Complement found from 5036 to 5042\n",
      "Complement found from 8038 to 8044\n",
      "Complement found from 10574 to 10580\n",
      "Complement found from 11485 to 11491\n",
      "Complement found from 12594 to 12600\n",
      "Complement found from 13272 to 13278\n",
      "Complement found from 14545 to 14551\n",
      "Complement found from 15688 to 15694\n",
      "Seq is: ATAATA\n",
      "14 ATAATA motifs detected in file\n",
      "\n",
      "Complement found from 9489 to 9495\n",
      "Complement found from 13726 to 13732\n",
      "Complement found from 14435 to 14441\n",
      "Seq is: CAGGAT\n",
      "3 CAGGAT motifs detected in file\n",
      "\n",
      "Complement found from 3031 to 3037\n",
      "Complement found from 12617 to 12623\n",
      "Seq is: GTTCGT\n",
      "2 GTTCGT motifs detected in file\n",
      "\n",
      "Complement found from 209 to 215\n",
      "Complement found from 1858 to 1864\n",
      "Complement found from 2190 to 2196\n",
      "Complement found from 2368 to 2374\n",
      "Complement found from 4468 to 4474\n",
      "Complement found from 5638 to 5644\n",
      "Complement found from 8219 to 8225\n",
      "Complement found from 10023 to 10029\n",
      "Complement found from 14084 to 14090\n",
      "Complement found from 14499 to 14505\n",
      "Complement found from 14780 to 14786\n",
      "Seq is: AATTAA\n",
      "11 AATTAA motifs detected in file\n",
      "\n",
      "Complement found from 207 to 213\n",
      "Complement found from 2737 to 2743\n",
      "Complement found from 5076 to 5082\n",
      "Complement found from 5195 to 5201\n",
      "Complement found from 5569 to 5575\n",
      "Complement found from 5636 to 5642\n",
      "Complement found from 8221 to 8227\n",
      "Complement found from 10070 to 10076\n",
      "Complement found from 11074 to 11080\n",
      "Complement found from 14782 to 14788\n",
      "Seq is: TTAATT\n",
      "10 TTAATT motifs detected in file\n",
      "\n",
      "Complement found from 6761 to 6767\n",
      "Seq is: GTGTGA\n",
      "1 GTGTGA motifs detected in file\n",
      "\n",
      "Complement found from 4759 to 4765\n",
      "Complement found from 4888 to 4894\n",
      "Complement found from 6832 to 6838\n",
      "Complement found from 7671 to 7677\n",
      "Complement found from 8750 to 8756\n",
      "Complement found from 10339 to 10345\n",
      "Complement found from 10454 to 10460\n",
      "Complement found from 10986 to 10992\n",
      "Complement found from 11103 to 11109\n",
      "Complement found from 14231 to 14237\n",
      "Complement found from 14976 to 14982\n",
      "Seq is: AATCAT\n",
      "11 AATCAT motifs detected in file\n",
      "\n",
      "Complement found from 4159 to 4165\n",
      "Complement found from 9333 to 9339\n",
      "Complement found from 12071 to 12077\n",
      "Complement found from 12426 to 12432\n",
      "Complement found from 13539 to 13545\n",
      "Complement found from 14233 to 14239\n",
      "Complement found from 14344 to 14350\n",
      "Seq is: TCATAC\n",
      "7 TCATAC motifs detected in file\n",
      "\n",
      "Seq is: AAGCGG\n",
      "0 AAGCGG motifs detected in file\n",
      "\n",
      "Complement found from 445 to 451\n",
      "Complement found from 3762 to 3768\n",
      "Complement found from 3994 to 4000\n",
      "Complement found from 8083 to 8089\n",
      "Complement found from 9013 to 9019\n",
      "Complement found from 12023 to 12029\n",
      "Complement found from 12673 to 12679\n",
      "Complement found from 13660 to 13666\n",
      "Complement found from 15466 to 15472\n",
      "Complement found from 16097 to 16103\n",
      "Complement found from 16337 to 16343\n",
      "Seq is: ACATTA\n",
      "11 ACATTA motifs detected in file\n",
      "\n",
      "Complement found from 1653 to 1659\n",
      "Complement found from 1871 to 1877\n",
      "Complement found from 2902 to 2908\n",
      "Complement found from 6675 to 6681\n",
      "Complement found from 7170 to 7176\n",
      "Complement found from 7540 to 7546\n",
      "Complement found from 8305 to 8311\n",
      "Complement found from 10015 to 10021\n",
      "Seq is: TAACTT\n",
      "8 TAACTT motifs detected in file\n",
      "\n",
      "Complement found from 700 to 706\n",
      "Complement found from 4664 to 4670\n",
      "Complement found from 7829 to 7835\n",
      "Complement found from 12456 to 12462\n",
      "Complement found from 13758 to 13764\n",
      "Complement found from 15805 to 15811\n",
      "Seq is: GCATCC\n",
      "6 GCATCC motifs detected in file\n",
      "\n",
      "Complement found from 5468 to 5474\n",
      "Complement found from 7616 to 7622\n",
      "Complement found from 12949 to 12955\n",
      "Seq is: ACGCTA\n",
      "3 ACGCTA motifs detected in file\n",
      "\n",
      "Complement found from 2741 to 2747\n",
      "Complement found from 3615 to 3621\n",
      "Complement found from 4586 to 4592\n",
      "Complement found from 8757 to 8763\n",
      "Complement found from 9461 to 9467\n",
      "Complement found from 12465 to 12471\n",
      "Complement found from 13894 to 13900\n",
      "Complement found from 15716 to 15722\n",
      "Seq is: TTTATT\n",
      "8 TTTATT motifs detected in file\n",
      "\n",
      "Complement found from 147 to 153\n",
      "Complement found from 502 to 508\n",
      "Complement found from 2945 to 2951\n",
      "Complement found from 4394 to 4400\n",
      "Complement found from 5096 to 5102\n",
      "Complement found from 6641 to 6647\n",
      "Complement found from 7256 to 7262\n",
      "Complement found from 7570 to 7576\n",
      "Complement found from 7798 to 7804\n",
      "Complement found from 8565 to 8571\n",
      "Complement found from 9126 to 9132\n",
      "Complement found from 9455 to 9461\n",
      "Complement found from 10346 to 10352\n",
      "Complement found from 12849 to 12855\n",
      "Complement found from 12903 to 12909\n",
      "Complement found from 14121 to 14127\n",
      "Complement found from 14369 to 14375\n",
      "Complement found from 15643 to 15649\n",
      "Complement found from 15829 to 15835\n",
      "Complement found from 15835 to 15841\n",
      "Seq is: ATCCTA\n",
      "20 ATCCTA motifs detected in file\n",
      "\n",
      "Complement found from 4371 to 4377\n",
      "Complement found from 6325 to 6331\n",
      "Complement found from 16407 to 16413\n",
      "Seq is: CTCCGT\n",
      "3 CTCCGT motifs detected in file\n",
      "\n",
      "Complement found from 1713 to 1719\n",
      "Complement found from 3532 to 3538\n",
      "Complement found from 6294 to 6300\n",
      "Complement found from 12889 to 12895\n",
      "Seq is: CCTTAG\n",
      "4 CCTTAG motifs detected in file\n",
      "\n",
      "Complement found from 5573 to 5579\n",
      "Complement found from 9496 to 9502\n",
      "Complement found from 9956 to 9962\n",
      "Seq is: TTTCTG\n",
      "3 TTTCTG motifs detected in file\n",
      "\n",
      "Complement found from 1949 to 1955\n",
      "Seq is: TCTATG\n",
      "1 TCTATG motifs detected in file\n",
      "\n",
      "Complement found from 94 to 100\n",
      "Seq is: ACGCTG\n",
      "1 ACGCTG motifs detected in file\n",
      "\n",
      "Complement found from 1186 to 1192\n",
      "Complement found from 6841 to 6847\n",
      "Complement found from 8936 to 8942\n",
      "Complement found from 10188 to 10194\n",
      "Complement found from 11136 to 11142\n",
      "Complement found from 11549 to 11555\n",
      "Complement found from 12080 to 12086\n",
      "Complement found from 12098 to 12104\n",
      "Complement found from 16421 to 16427\n",
      "Seq is: TATCCC\n",
      "9 TATCCC motifs detected in file\n",
      "\n",
      "Complement found from 2640 to 2646\n",
      "Complement found from 7847 to 7853\n",
      "Seq is: ACGAGG\n",
      "2 ACGAGG motifs detected in file\n",
      "\n",
      "Seq is: CCGAGG\n",
      "0 CCGAGG motifs detected in file\n",
      "\n",
      "Complement found from 118 to 124\n",
      "Complement found from 9486 to 9492\n",
      "Complement found from 13195 to 13201\n",
      "Complement found from 13723 to 13729\n",
      "Seq is: TCGCAG\n",
      "4 TCGCAG motifs detected in file\n",
      "\n",
      "Complement found from 2199 to 2205\n",
      "Seq is: AGCGTT\n",
      "1 AGCGTT motifs detected in file\n",
      "\n",
      "Complement found from 1386 to 1392\n",
      "Seq is: ACGATA\n",
      "1 ACGATA motifs detected in file\n",
      "\n",
      "Complement found from 3697 to 3703\n",
      "Seq is: GCGCAC\n",
      "1 GCGCAC motifs detected in file\n",
      "\n",
      "Complement found from 6717 to 6723\n",
      "Complement found from 8386 to 8392\n",
      "Seq is: GTATGG\n",
      "2 GTATGG motifs detected in file\n",
      "\n",
      "Complement found from 16387 to 16393\n",
      "Seq is: GGGGTC\n",
      "1 GGGGTC motifs detected in file\n",
      "\n",
      "Seq is: TGCTTT\n",
      "0 TGCTTT motifs detected in file\n",
      "\n",
      "Complement found from 6718 to 6724\n",
      "Complement found from 11479 to 11485\n",
      "Seq is: TATGGT\n",
      "2 TATGGT motifs detected in file\n",
      "\n",
      "Seq is: TTACGT\n",
      "0 TTACGT motifs detected in file\n",
      "\n",
      "Complement found from 1239 to 1245\n",
      "Complement found from 2511 to 2517\n",
      "Complement found from 3627 to 3633\n",
      "Complement found from 3787 to 3793\n",
      "Complement found from 3817 to 3823\n",
      "Complement found from 5334 to 5340\n",
      "Complement found from 8349 to 8355\n",
      "Complement found from 11042 to 11048\n",
      "Complement found from 11379 to 11385\n",
      "Complement found from 13433 to 13439\n",
      "Seq is: ACCTCT\n",
      "10 ACCTCT motifs detected in file\n",
      "\n",
      "Complement found from 2641 to 2647\n",
      "Seq is: CGAGGG\n",
      "1 CGAGGG motifs detected in file\n",
      "\n",
      "Complement found from 26 to 32\n",
      "Complement found from 4189 to 4195\n",
      "Complement found from 4518 to 4524\n",
      "Complement found from 5451 to 5457\n",
      "Complement found from 5874 to 5880\n",
      "Complement found from 7718 to 7724\n",
      "Complement found from 8065 to 8071\n",
      "Complement found from 8793 to 8799\n",
      "Complement found from 11256 to 11262\n",
      "Complement found from 11498 to 11504\n",
      "Complement found from 11766 to 11772\n",
      "Complement found from 12011 to 12017\n",
      "Complement found from 13077 to 13083\n",
      "Complement found from 14116 to 14122\n",
      "Complement found from 14399 to 14405\n",
      "Complement found from 14645 to 14651\n",
      "Complement found from 14791 to 14797\n",
      "Seq is: CACTCA\n",
      "17 CACTCA motifs detected in file\n",
      "\n",
      "Complement found from 2188 to 2194\n",
      "Complement found from 2261 to 2267\n",
      "Complement found from 9535 to 9541\n",
      "Complement found from 10021 to 10027\n",
      "Complement found from 13010 to 13016\n",
      "Seq is: CCAATT\n",
      "5 CCAATT motifs detected in file\n",
      "\n",
      "Complement found from 2231 to 2237\n",
      "Complement found from 15202 to 15208\n",
      "Seq is: ATCCCA\n",
      "2 ATCCCA motifs detected in file\n",
      "\n",
      "Complement found from 3098 to 3104\n",
      "Complement found from 5156 to 5162\n",
      "Complement found from 5481 to 5487\n",
      "Complement found from 6372 to 6378\n",
      "Complement found from 6491 to 6497\n",
      "Complement found from 8817 to 8823\n",
      "Complement found from 9852 to 9858\n",
      "Complement found from 13560 to 13566\n",
      "Complement found from 13949 to 13955\n",
      "Complement found from 15848 to 15854\n",
      "Seq is: CTATCT\n",
      "10 CTATCT motifs detected in file\n",
      "\n",
      "Complement found from 2119 to 2125\n",
      "Seq is: GGACAC\n",
      "1 GGACAC motifs detected in file\n",
      "\n",
      "Seq is: GTAGCAC\n",
      "0 GTAGCAC motifs detected in file\n",
      "\n",
      "Seq is: TACAGAA\n",
      "0 TACAGAA motifs detected in file\n",
      "\n",
      "Seq is: TAAATTC\n",
      "0 TAAATTC motifs detected in file\n",
      "\n",
      "Seq is: CCCTTGC\n",
      "0 CCCTTGC motifs detected in file\n",
      "\n",
      "Complement found from 1942 to 1949\n",
      "Complement found from 14553 to 14560\n",
      "Seq is: ACACCCG\n",
      "2 ACACCCG motifs detected in file\n",
      "\n",
      "Seq is: ATAAATC\n",
      "0 ATAAATC motifs detected in file\n",
      "\n",
      "Seq is: CGAAATA\n",
      "0 CGAAATA motifs detected in file\n",
      "\n",
      "Seq is: CTGGGAC\n",
      "0 CTGGGAC motifs detected in file\n",
      "\n",
      "Complement found from 9860 to 9867\n",
      "Complement found from 12600 to 12607\n",
      "Complement found from 12879 to 12886\n",
      "Seq is: TTCATCC\n",
      "3 TTCATCC motifs detected in file\n",
      "\n",
      "Complement found from 12290 to 12297\n",
      "Seq is: TTGGTCT\n",
      "1 TTGGTCT motifs detected in file\n",
      "\n",
      "Complement found from 10847 to 10854\n",
      "Complement found from 15121 to 15128\n",
      "Seq is: ACAGCCT\n",
      "2 ACAGCCT motifs detected in file\n",
      "\n",
      "Seq is: ATGTCAA\n",
      "0 ATGTCAA motifs detected in file\n",
      "\n",
      "Complement found from 2672 to 2679\n",
      "Complement found from 8360 to 8367\n",
      "Complement found from 16411 to 16418\n",
      "Seq is: GTGAAAT\n",
      "3 GTGAAAT motifs detected in file\n",
      "\n",
      "Seq is: CGATGGG\n",
      "0 CGATGGG motifs detected in file\n",
      "\n",
      "Seq is: GGTTCAC\n",
      "0 GGTTCAC motifs detected in file\n",
      "\n",
      "Complement found from 6112 to 6119\n",
      "Seq is: AATACCC\n",
      "1 AATACCC motifs detected in file\n",
      "\n",
      "Seq is: GGGCCTC\n",
      "0 GGGCCTC motifs detected in file\n",
      "\n",
      "Complement found from 9328 to 9335\n",
      "Seq is: GCTCCTC\n",
      "1 GCTCCTC motifs detected in file\n",
      "\n",
      "Seq is: TCATCGT\n",
      "0 TCATCGT motifs detected in file\n",
      "\n",
      "Seq is: GAGAACA\n",
      "0 GAGAACA motifs detected in file\n",
      "\n",
      "Complement found from 3965 to 3972\n",
      "Seq is: CGCCCTA\n",
      "1 CGCCCTA motifs detected in file\n",
      "\n",
      "Complement found from 849 to 856\n",
      "Complement found from 1521 to 1528\n",
      "Complement found from 7733 to 7740\n",
      "Seq is: TAACTAA\n",
      "3 TAACTAA motifs detected in file\n",
      "\n",
      "Seq is: AGTGCCT\n",
      "0 AGTGCCT motifs detected in file\n",
      "\n",
      "Complement found from 4184 to 4191\n",
      "Complement found from 6643 to 6650\n",
      "Complement found from 7404 to 7411\n",
      "Complement found from 10279 to 10286\n",
      "Seq is: CCTACCA\n",
      "4 CCTACCA motifs detected in file\n",
      "\n",
      "Complement found from 4592 to 4599\n",
      "Seq is: CCAGTTC\n",
      "1 CCAGTTC motifs detected in file\n",
      "\n",
      "Seq is: CCTGGGG\n",
      "0 CCTGGGG motifs detected in file\n",
      "\n",
      "Complement found from 1014 to 1021\n",
      "Seq is: ATAGACT\n",
      "1 ATAGACT motifs detected in file\n",
      "\n",
      "Complement found from 432 to 439\n",
      "Complement found from 567 to 574\n",
      "Complement found from 7396 to 7403\n",
      "Complement found from 7465 to 7472\n",
      "Complement found from 9531 to 9538\n",
      "Complement found from 11866 to 11873\n",
      "Complement found from 12384 to 12391\n",
      "Seq is: CCCCCCA\n",
      "7 CCCCCCA motifs detected in file\n",
      "\n",
      "Seq is: GTAGGTA\n",
      "0 GTAGGTA motifs detected in file\n",
      "\n",
      "Complement found from 6420 to 6427\n",
      "Seq is: CCCCTGC\n",
      "1 CCCCTGC motifs detected in file\n",
      "\n",
      "Complement found from 7176 to 7183\n",
      "Complement found from 12478 to 12485\n",
      "Complement found from 14110 to 14117\n",
      "Seq is: TCTTCCC\n",
      "3 TCTTCCC motifs detected in file\n",
      "\n",
      "Seq is: TAAACTG\n",
      "0 TAAACTG motifs detected in file\n",
      "\n",
      "Complement found from 3762 to 3769\n",
      "Complement found from 9013 to 9020\n",
      "Complement found from 16097 to 16104\n",
      "Complement found from 16337 to 16344\n",
      "Seq is: ACATTAC\n",
      "4 ACATTAC motifs detected in file\n",
      "\n",
      "Seq is: ACGATAC\n",
      "0 ACGATAC motifs detected in file\n",
      "\n",
      "Seq is: ATGCCAT\n",
      "0 ATGCCAT motifs detected in file\n",
      "\n",
      "Complement found from 3370 to 3377\n",
      "Complement found from 12201 to 12208\n",
      "Seq is: TTACCGA\n",
      "2 TTACCGA motifs detected in file\n",
      "\n",
      "Seq is: AATACAT\n",
      "0 AATACAT motifs detected in file\n",
      "\n",
      "Complement found from 6708 to 6715\n",
      "Seq is: GATACAT\n",
      "1 GATACAT motifs detected in file\n",
      "\n",
      "Complement found from 2365 to 2372\n",
      "Complement found from 15505 to 15512\n",
      "Seq is: GACAATT\n",
      "2 GACAATT motifs detected in file\n",
      "\n",
      "Complement found from 3209 to 3216\n",
      "Complement found from 5432 to 5439\n",
      "Complement found from 6312 to 6319\n",
      "Complement found from 7399 to 7406\n",
      "Complement found from 13053 to 13060\n",
      "Complement found from 13647 to 13654\n",
      "Complement found from 13677 to 13684\n",
      "Complement found from 14809 to 14816\n",
      "Complement found from 15262 to 15269\n",
      "Complement found from 16289 to 16296\n",
      "Seq is: CCCACCC\n",
      "10 CCCACCC motifs detected in file\n",
      "\n",
      "Complement found from 4361 to 4368\n",
      "Complement found from 9889 to 9896\n",
      "Complement found from 11665 to 11672\n",
      "Complement found from 13152 to 13159\n",
      "Complement found from 14468 to 14475\n",
      "Complement found from 15672 to 15679\n",
      "Seq is: ATCCAAA\n",
      "6 ATCCAAA motifs detected in file\n",
      "\n",
      "Complement found from 9070 to 9077\n",
      "Complement found from 14477 to 14484\n",
      "Seq is: CAACCAT\n",
      "2 CAACCAT motifs detected in file\n",
      "\n",
      "Seq is: AGGATCA\n",
      "0 AGGATCA motifs detected in file\n",
      "\n",
      "Complement found from 460 to 467\n",
      "Complement found from 1688 to 1695\n",
      "Complement found from 5601 to 5608\n",
      "Complement found from 9592 to 9599\n",
      "Complement found from 10620 to 10627\n",
      "Complement found from 14114 to 14121\n",
      "Seq is: CCCACTC\n",
      "6 CCCACTC motifs detected in file\n",
      "\n",
      "Complement found from 6845 to 6852\n",
      "Seq is: CCCACCG\n",
      "1 CCCACCG motifs detected in file\n",
      "\n",
      "Complement found from 1101 to 1108\n",
      "Seq is: AAACCTC\n",
      "1 AAACCTC motifs detected in file\n",
      "\n",
      "Seq is: AACGTCA\n",
      "0 AACGTCA motifs detected in file\n",
      "\n",
      "Complement found from 7000 to 7007\n",
      "Complement found from 15413 to 15420\n",
      "Seq is: ACTACAC\n",
      "2 ACTACAC motifs detected in file\n",
      "\n",
      "Complement found from 3755 to 3762\n",
      "Complement found from 7145 to 7152\n",
      "Complement found from 11022 to 11029\n",
      "Complement found from 11313 to 11320\n",
      "Complement found from 16226 to 16233\n",
      "Seq is: ACTATCA\n",
      "5 ACTATCA motifs detected in file\n",
      "\n",
      "Complement found from 463 to 470\n",
      "Complement found from 6309 to 6316\n",
      "Complement found from 11811 to 11818\n",
      "Seq is: ACTCCCA\n",
      "3 ACTCCCA motifs detected in file\n",
      "\n",
      "Seq is: ACTAACG\n",
      "0 ACTAACG motifs detected in file\n",
      "\n",
      "Complement found from 2312 to 2319\n",
      "Complement found from 7247 to 7254\n",
      "Seq is: ACATGAA\n",
      "2 ACATGAA motifs detected in file\n",
      "\n",
      "Complement found from 185 to 192\n",
      "Seq is: CGAACAT\n",
      "1 CGAACAT motifs detected in file\n",
      "\n",
      "Seq is: TCGTACC\n",
      "0 TCGTACC motifs detected in file\n",
      "\n",
      "Complement found from 307 to 314\n",
      "Complement found from 958 to 965\n",
      "Complement found from 3569 to 3576\n",
      "Complement found from 5443 to 5450\n",
      "Complement found from 14527 to 14534\n",
      "Complement found from 14805 to 14812\n",
      "Complement found from 15538 to 15545\n",
      "Complement found from 16186 to 16193\n",
      "Seq is: CCTCCCC\n",
      "8 CCTCCCC motifs detected in file\n",
      "\n",
      "Complement found from 1122 to 1129\n",
      "Complement found from 7727 to 7734\n",
      "Complement found from 14760 to 14767\n",
      "Seq is: CAAAACT\n",
      "3 CAAAACT motifs detected in file\n",
      "\n",
      "Complement found from 6352 to 6359\n",
      "Complement found from 6598 to 6605\n",
      "Complement found from 8922 to 8929\n",
      "Complement found from 12075 to 12082\n",
      "Complement found from 13301 to 13308\n",
      "Seq is: ACACCTA\n",
      "5 ACACCTA motifs detected in file\n",
      "\n",
      "Complement found from 1062 to 1069\n",
      "Complement found from 1730 to 1737\n",
      "Complement found from 12546 to 12553\n",
      "Complement found from 12667 to 12674\n",
      "Complement found from 14070 to 14077\n",
      "Complement found from 15987 to 15994\n",
      "Seq is: ACCCAAA\n",
      "6 ACCCAAA motifs detected in file\n",
      "\n",
      "Seq is: AATACAT\n",
      "0 AATACAT motifs detected in file\n",
      "\n",
      "Seq is: GATGTCC\n",
      "0 GATGTCC motifs detected in file\n",
      "\n",
      "Seq is: ACCAGTT\n",
      "0 ACCAGTT motifs detected in file\n",
      "\n",
      "Complement found from 10483 to 10490\n",
      "Complement found from 14291 to 14298\n",
      "Complement found from 14591 to 14598\n",
      "Complement found from 16133 to 16140\n",
      "Seq is: CATAAAT\n",
      "4 CATAAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCATGG\n",
      "0 ACCATGG motifs detected in file\n",
      "\n",
      "Seq is: CATGTTA\n",
      "0 CATGTTA motifs detected in file\n",
      "\n",
      "Seq is: TAAGTAC\n",
      "0 TAAGTAC motifs detected in file\n",
      "\n",
      "Complement found from 7914 to 7921\n",
      "Seq is: CACCGAC\n",
      "1 CACCGAC motifs detected in file\n",
      "\n",
      "Complement found from 9826 to 9833\n",
      "Seq is: CGTCATT\n",
      "1 CGTCATT motifs detected in file\n",
      "\n",
      "Complement found from 2850 to 2857\n",
      "Complement found from 4575 to 4582\n",
      "Seq is: ACATGCT\n",
      "2 ACATGCT motifs detected in file\n",
      "\n",
      "Complement found from 578 to 585\n",
      "Complement found from 12437 to 12444\n",
      "Seq is: TTATGTA\n",
      "2 TTATGTA motifs detected in file\n",
      "\n",
      "Seq is: TCGCCAA\n",
      "0 TCGCCAA motifs detected in file\n",
      "\n",
      "Complement found from 3399 to 3406\n",
      "Complement found from 13847 to 13854\n",
      "Complement found from 14729 to 14736\n",
      "Seq is: CAACTAC\n",
      "3 CAACTAC motifs detected in file\n",
      "\n",
      "Complement found from 25 to 32\n",
      "Complement found from 4188 to 4195\n",
      "Complement found from 13076 to 13083\n",
      "Complement found from 14115 to 14122\n",
      "Complement found from 14790 to 14797\n",
      "Seq is: CCACTCA\n",
      "5 CCACTCA motifs detected in file\n",
      "\n",
      "Seq is: GTTATGA\n",
      "0 GTTATGA motifs detected in file\n",
      "\n",
      "Complement found from 3208 to 3215\n",
      "Complement found from 5431 to 5438\n",
      "Complement found from 9209 to 9216\n",
      "Complement found from 12016 to 12023\n",
      "Complement found from 16288 to 16295\n",
      "Seq is: ACCCACC\n",
      "5 ACCCACC motifs detected in file\n",
      "\n",
      "Seq is: ATGGGTT\n",
      "0 ATGGGTT motifs detected in file\n",
      "\n",
      "Seq is: TAGACGA\n",
      "0 TAGACGA motifs detected in file\n",
      "\n",
      "Seq is: AGGGGTA\n",
      "0 AGGGGTA motifs detected in file\n",
      "\n",
      "Complement found from 6937 to 6944\n",
      "Complement found from 7173 to 7180\n",
      "Complement found from 14104 to 14111\n",
      "Seq is: CTTTCTT\n",
      "3 CTTTCTT motifs detected in file\n",
      "\n",
      "Complement found from 2654 to 2661\n",
      "Complement found from 12475 to 12482\n",
      "Seq is: GTCTCTT\n",
      "2 GTCTCTT motifs detected in file\n",
      "\n",
      "Complement found from 7873 to 7880\n",
      "Complement found from 11922 to 11929\n",
      "Seq is: ATCAAAT\n",
      "2 ATCAAAT motifs detected in file\n",
      "\n",
      "Seq is: TTAATCT\n",
      "0 TTAATCT motifs detected in file\n",
      "\n",
      "Complement found from 2029 to 2036\n",
      "Seq is: TAGAATC\n",
      "1 TAGAATC motifs detected in file\n",
      "\n",
      "Seq is: GAAACCC\n",
      "0 GAAACCC motifs detected in file\n",
      "\n",
      "Complement found from 906 to 913\n",
      "Complement found from 3044 to 3051\n",
      "Seq is: ACGATTA\n",
      "2 ACGATTA motifs detected in file\n",
      "\n",
      "Complement found from 1237 to 1244\n",
      "Complement found from 3625 to 3632\n",
      "Complement found from 5364 to 5371\n",
      "Complement found from 8620 to 8627\n",
      "Complement found from 14050 to 14057\n",
      "Complement found from 15330 to 15337\n",
      "Seq is: CCACCTC\n",
      "6 CCACCTC motifs detected in file\n",
      "\n",
      "Complement found from 5571 to 5578\n",
      "Seq is: AATTTCT\n",
      "1 AATTTCT motifs detected in file\n",
      "\n",
      "Complement found from 15960 to 15967\n",
      "Seq is: GAAAAAG\n",
      "1 GAAAAAG motifs detected in file\n",
      "\n",
      "Seq is: AGACTAA\n",
      "0 AGACTAA motifs detected in file\n",
      "\n",
      "Seq is: ACAGGAA\n",
      "0 ACAGGAA motifs detected in file\n",
      "\n",
      "Complement found from 15937 to 15944\n",
      "Seq is: CCTTTTT\n",
      "1 CCTTTTT motifs detected in file\n",
      "\n",
      "Seq is: CCAAAAG\n",
      "0 CCAAAAG motifs detected in file\n",
      "\n",
      "Complement found from 1982 to 1989\n",
      "Complement found from 2531 to 2538\n",
      "Seq is: TAGAGGC\n",
      "2 TAGAGGC motifs detected in file\n",
      "\n",
      "Seq is: CAAATGT\n",
      "0 CAAATGT motifs detected in file\n",
      "\n",
      "Seq is: TCTGACC\n",
      "0 TCTGACC motifs detected in file\n",
      "\n",
      "Complement found from 229 to 236\n",
      "Complement found from 9201 to 9208\n",
      "Complement found from 10820 to 10827\n",
      "Seq is: ACATAAT\n",
      "3 ACATAAT motifs detected in file\n",
      "\n",
      "Complement found from 8627 to 8634\n",
      "Complement found from 11924 to 11931\n",
      "Complement found from 12689 to 12696\n",
      "Complement found from 15154 to 15161\n",
      "Seq is: CAAATAT\n",
      "4 CAAATAT motifs detected in file\n",
      "\n",
      "Complement found from 5902 to 5909\n",
      "Seq is: GATGTTC\n",
      "1 GATGTTC motifs detected in file\n",
      "\n",
      "Seq is: CTGTCCG\n",
      "0 CTGTCCG motifs detected in file\n",
      "\n",
      "Seq is: GGTAAAC\n",
      "0 GGTAAAC motifs detected in file\n",
      "\n",
      "Complement found from 14290 to 14297\n",
      "Complement found from 16518 to 16525\n",
      "Seq is: TCATAAA\n",
      "2 TCATAAA motifs detected in file\n",
      "\n",
      "Seq is: ACAAAAG\n",
      "0 ACAAAAG motifs detected in file\n",
      "\n",
      "Complement found from 4470 to 4477\n",
      "Complement found from 9147 to 9154\n",
      "Seq is: TTAATCC\n",
      "2 TTAATCC motifs detected in file\n",
      "\n",
      "Complement found from 6370 to 6377\n",
      "Seq is: CTCTATC\n",
      "1 CTCTATC motifs detected in file\n",
      "\n",
      "Complement found from 4720 to 4727\n",
      "Complement found from 8699 to 8706\n",
      "Complement found from 9361 to 9368\n",
      "Complement found from 13427 to 13434\n",
      "Seq is: AACCATA\n",
      "4 AACCATA motifs detected in file\n",
      "\n",
      "Complement found from 4394 to 4401\n",
      "Complement found from 5096 to 5103\n",
      "Complement found from 14121 to 14128\n",
      "Complement found from 15829 to 15836\n",
      "Seq is: ATCCTAA\n",
      "4 ATCCTAA motifs detected in file\n",
      "\n",
      "Complement found from 5097 to 5104\n",
      "Complement found from 7712 to 7719\n",
      "Complement found from 14122 to 14129\n",
      "Seq is: TCCTAAC\n",
      "3 TCCTAAC motifs detected in file\n",
      "\n",
      "Complement found from 5682 to 5689\n",
      "Complement found from 7185 to 7192\n",
      "Complement found from 16462 to 16469\n",
      "Seq is: AACACTT\n",
      "3 AACACTT motifs detected in file\n",
      "\n",
      "Complement found from 2961 to 2968\n",
      "Complement found from 13535 to 13542\n",
      "Seq is: CATATCA\n",
      "2 CATATCA motifs detected in file\n",
      "\n",
      "Complement found from 11584 to 11591\n",
      "Seq is: TGCCTAC\n",
      "1 TGCCTAC motifs detected in file\n",
      "\n",
      "Seq is: GATGAAC\n",
      "0 GATGAAC motifs detected in file\n",
      "\n",
      "Complement found from 4818 to 4825\n",
      "Seq is: AGGTTAC\n",
      "1 AGGTTAC motifs detected in file\n",
      "\n",
      "Seq is: TACCATT\n",
      "0 TACCATT motifs detected in file\n",
      "\n",
      "Seq is: TTCCCAT\n",
      "0 TTCCCAT motifs detected in file\n",
      "\n",
      "Complement found from 10425 to 10432\n",
      "Complement found from 15555 to 15562\n",
      "Seq is: CGAATGA\n",
      "2 CGAATGA motifs detected in file\n",
      "\n",
      "Complement found from 9557 to 9564\n",
      "Seq is: CCAACAG\n",
      "1 CCAACAG motifs detected in file\n",
      "\n",
      "Complement found from 15242 to 15249\n",
      "Seq is: GAGGCTA\n",
      "1 GAGGCTA motifs detected in file\n",
      "\n",
      "Complement found from 2096 to 2103\n",
      "Complement found from 5862 to 5869\n",
      "Seq is: AGTCCAA\n",
      "2 AGTCCAA motifs detected in file\n",
      "\n",
      "Complement found from 2657 to 2664\n",
      "Complement found from 9986 to 9993\n",
      "Complement found from 13108 to 13115\n",
      "Seq is: TCTTACT\n",
      "3 TCTTACT motifs detected in file\n",
      "\n",
      "Seq is: CCTCCAG\n",
      "0 CCTCCAG motifs detected in file\n",
      "\n",
      "Seq is: ACGCCGA\n",
      "0 ACGCCGA motifs detected in file\n",
      "\n",
      "Seq is: TCCTCAG\n",
      "0 TCCTCAG motifs detected in file\n",
      "\n",
      "Complement found from 4322 to 4329\n",
      "Complement found from 13895 to 13902\n",
      "Seq is: TTATTTC\n",
      "2 TTATTTC motifs detected in file\n",
      "\n",
      "Complement found from 861 to 868\n",
      "Complement found from 1831 to 1838\n",
      "Complement found from 8678 to 8685\n",
      "Complement found from 8769 to 8776\n",
      "Complement found from 9346 to 9353\n",
      "Complement found from 9358 to 9365\n",
      "Complement found from 10301 to 10308\n",
      "Complement found from 14764 to 14771\n",
      "Seq is: ACTAACC\n",
      "8 ACTAACC motifs detected in file\n",
      "\n",
      "Complement found from 15374 to 15381\n",
      "Seq is: GAATCAC\n",
      "1 GAATCAC motifs detected in file\n",
      "\n",
      "Seq is: CCGCTTT\n",
      "0 CCGCTTT motifs detected in file\n",
      "\n",
      "Complement found from 10574 to 10581\n",
      "Complement found from 11485 to 11492\n",
      "Seq is: ATAATAC\n",
      "2 ATAATAC motifs detected in file\n",
      "\n",
      "Complement found from 5169 to 5176\n",
      "Complement found from 14654 to 14661\n",
      "Seq is: GAAACAA\n",
      "2 GAAACAA motifs detected in file\n",
      "\n",
      "Complement found from 1991 to 1998\n",
      "Complement found from 2788 to 2795\n",
      "Complement found from 4254 to 4261\n",
      "Complement found from 7121 to 7128\n",
      "Complement found from 16281 to 16288\n",
      "Seq is: CAAACCT\n",
      "5 CAAACCT motifs detected in file\n",
      "\n",
      "Complement found from 1255 to 1262\n",
      "Complement found from 4892 to 4899\n",
      "Complement found from 9365 to 9372\n",
      "Seq is: ATATACC\n",
      "3 ATATACC motifs detected in file\n",
      "\n",
      "Complement found from 503 to 510\n",
      "Complement found from 4183 to 4190\n",
      "Complement found from 5475 to 5482\n",
      "Complement found from 6642 to 6649\n",
      "Complement found from 10973 to 10980\n",
      "Complement found from 14370 to 14377\n",
      "Seq is: TCCTACC\n",
      "6 TCCTACC motifs detected in file\n",
      "\n",
      "Complement found from 6154 to 6161\n",
      "Complement found from 7348 to 7355\n",
      "Complement found from 10953 to 10960\n",
      "Complement found from 14772 to 14779\n",
      "Complement found from 15837 to 15844\n",
      "Seq is: CCTAATA\n",
      "5 CCTAATA motifs detected in file\n",
      "\n",
      "Complement found from 7856 to 7863\n",
      "Seq is: ACGATCC\n",
      "1 ACGATCC motifs detected in file\n",
      "\n",
      "Complement found from 3325 to 3332\n",
      "Complement found from 3826 to 3833\n",
      "Complement found from 5009 to 5016\n",
      "Complement found from 5472 to 5479\n",
      "Complement found from 6231 to 6238\n",
      "Complement found from 8411 to 8418\n",
      "Complement found from 13984 to 13991\n",
      "Complement found from 14131 to 14138\n",
      "Complement found from 14440 to 14447\n",
      "Seq is: TACTCCT\n",
      "9 TACTCCT motifs detected in file\n",
      "\n",
      "Complement found from 4380 to 4387\n",
      "Complement found from 5479 to 5486\n",
      "Complement found from 12077 to 12084\n",
      "Seq is: ACCTATC\n",
      "3 ACCTATC motifs detected in file\n",
      "\n",
      "Complement found from 5093 to 5100\n",
      "Complement found from 15094 to 15101\n",
      "Seq is: ATTATCC\n",
      "2 ATTATCC motifs detected in file\n",
      "\n",
      "Seq is: CGTTCCT\n",
      "0 CGTTCCT motifs detected in file\n",
      "\n",
      "Seq is: GCGGAGG\n",
      "0 GCGGAGG motifs detected in file\n",
      "\n",
      "Complement found from 3671 to 3678\n",
      "Complement found from 8673 to 8680\n",
      "Complement found from 11316 to 11323\n",
      "Complement found from 15357 to 15364\n",
      "Seq is: ATCAAAC\n",
      "4 ATCAAAC motifs detected in file\n",
      "\n",
      "Complement found from 16349 to 16356\n",
      "Seq is: AATCCCT\n",
      "1 AATCCCT motifs detected in file\n",
      "\n",
      "Complement found from 790 to 797\n",
      "Complement found from 3634 to 3641\n",
      "Complement found from 9294 to 9301\n",
      "Complement found from 9519 to 9526\n",
      "Complement found from 10689 to 10696\n",
      "Complement found from 11741 to 11748\n",
      "Complement found from 13465 to 13472\n",
      "Seq is: GCCTAGC\n",
      "7 GCCTAGC motifs detected in file\n",
      "\n",
      "Complement found from 11461 to 11468\n",
      "Seq is: CTCTTAA\n",
      "1 CTCTTAA motifs detected in file\n",
      "\n",
      "Complement found from 9604 to 9611\n",
      "Seq is: CACATCC\n",
      "1 CACATCC motifs detected in file\n",
      "\n",
      "Complement found from 7641 to 7648\n",
      "Seq is: GCTTATC\n",
      "1 GCTTATC motifs detected in file\n",
      "\n",
      "Seq is: CTTTATA\n",
      "0 CTTTATA motifs detected in file\n",
      "\n",
      "Complement found from 4919 to 4926\n",
      "Complement found from 9177 to 9184\n",
      "Seq is: GTAAGCC\n",
      "2 GTAAGCC motifs detected in file\n",
      "\n",
      "Complement found from 2876 to 2883\n",
      "Complement found from 3920 to 3927\n",
      "Seq is: CGAACTA\n",
      "2 CGAACTA motifs detected in file\n",
      "\n",
      "Complement found from 4831 to 4838\n",
      "Complement found from 8929 to 8936\n",
      "Complement found from 10159 to 10166\n",
      "Complement found from 13023 to 13030\n",
      "Complement found from 15534 to 15541\n",
      "Complement found from 16256 to 16263\n",
      "Seq is: CACCCCT\n",
      "6 CACCCCT motifs detected in file\n",
      "\n",
      "Complement found from 569 to 576\n",
      "Complement found from 5446 to 5453\n",
      "Complement found from 8079 to 8086\n",
      "Complement found from 8558 to 8565\n",
      "Complement found from 12482 to 12489\n",
      "Complement found from 14619 to 14626\n",
      "Complement found from 15541 to 15548\n",
      "Seq is: CCCCACA\n",
      "7 CCCCACA motifs detected in file\n",
      "\n",
      "Complement found from 8106 to 8113\n",
      "Complement found from 8223 to 8230\n",
      "Complement found from 12380 to 12387\n",
      "Seq is: AATTCCC\n",
      "3 AATTCCC motifs detected in file\n",
      "\n",
      "Complement found from 3884 to 3891\n",
      "Complement found from 8642 to 8649\n",
      "Complement found from 9667 to 9674\n",
      "Seq is: CAACCGA\n",
      "3 CAACCGA motifs detected in file\n",
      "\n",
      "Complement found from 465 to 472\n",
      "Complement found from 6665 to 6672\n",
      "Complement found from 15203 to 15210\n",
      "Seq is: TCCCATA\n",
      "3 TCCCATA motifs detected in file\n",
      "\n",
      "Complement found from 422 to 429\n",
      "Complement found from 2370 to 2377\n",
      "Complement found from 5691 to 5698\n",
      "Complement found from 16296 to 16303\n",
      "Seq is: TTAACAG\n",
      "4 TTAACAG motifs detected in file\n",
      "\n",
      "Complement found from 507 to 514\n",
      "Complement found from 4985 to 4992\n",
      "Complement found from 9241 to 9248\n",
      "Complement found from 12555 to 12562\n",
      "Seq is: ACCCAGC\n",
      "4 ACCCAGC motifs detected in file\n",
      "\n",
      "Complement found from 14257 to 14264\n",
      "Seq is: GGATCCT\n",
      "1 GGATCCT motifs detected in file\n",
      "\n",
      "Complement found from 15484 to 15491\n",
      "Seq is: CCAGACC\n",
      "1 CCAGACC motifs detected in file\n",
      "\n",
      "Complement found from 6211 to 6218\n",
      "Complement found from 11460 to 11467\n",
      "Seq is: ACTCTTA\n",
      "2 ACTCTTA motifs detected in file\n",
      "\n",
      "Complement found from 9103 to 9110\n",
      "Complement found from 10982 to 10989\n",
      "Seq is: TCACAAT\n",
      "2 TCACAAT motifs detected in file\n",
      "\n",
      "Complement found from 6781 to 6788\n",
      "Seq is: TACAGTA\n",
      "1 TACAGTA motifs detected in file\n",
      "\n",
      "Complement found from 3112 to 3119\n",
      "Complement found from 3288 to 3295\n",
      "Complement found from 5440 to 5447\n",
      "Complement found from 8424 to 8431\n",
      "Seq is: ATTCCTC\n",
      "4 ATTCCTC motifs detected in file\n",
      "\n",
      "Complement found from 4027 to 4034\n",
      "Complement found from 12703 to 12710\n",
      "Complement found from 15026 to 15033\n",
      "Complement found from 15446 to 15453\n",
      "Seq is: TCTTCCT\n",
      "4 TCTTCCT motifs detected in file\n",
      "\n",
      "Complement found from 12476 to 12483\n",
      "Complement found from 15444 to 15451\n",
      "Seq is: TCTCTTC\n",
      "2 TCTCTTC motifs detected in file\n",
      "\n",
      "Complement found from 5965 to 5972\n",
      "Seq is: ATTCGGC\n",
      "1 ATTCGGC motifs detected in file\n",
      "\n",
      "Complement found from 16423 to 16430\n",
      "Seq is: TCCCGCA\n",
      "1 TCCCGCA motifs detected in file\n",
      "\n",
      "Seq is: GAAACTA\n",
      "0 GAAACTA motifs detected in file\n",
      "\n",
      "Complement found from 336 to 343\n",
      "Complement found from 9603 to 9610\n",
      "Complement found from 15033 to 15040\n",
      "Complement found from 16232 to 16239\n",
      "Seq is: ACACATC\n",
      "4 ACACATC motifs detected in file\n",
      "\n",
      "Seq is: ATTCCCA\n",
      "0 ATTCCCA motifs detected in file\n",
      "\n",
      "Complement found from 13765 to 13772\n",
      "Complement found from 15401 to 15408\n",
      "Seq is: CCTTCCA\n",
      "2 CCTTCCA motifs detected in file\n",
      "\n",
      "Complement found from 11602 to 11609\n",
      "Seq is: CTAAAAT\n",
      "1 CTAAAAT motifs detected in file\n",
      "\n",
      "Complement found from 10096 to 10103\n",
      "Seq is: AGCCTTA\n",
      "1 AGCCTTA motifs detected in file\n",
      "\n",
      "Complement found from 7364 to 7371\n",
      "Seq is: CCCTCCA\n",
      "1 CCCTCCA motifs detected in file\n",
      "\n",
      "Complement found from 5585 to 5592\n",
      "Seq is: CTAAGGA\n",
      "1 CTAAGGA motifs detected in file\n",
      "\n",
      "Complement found from 7953 to 7960\n",
      "Complement found from 12082 to 12089\n",
      "Complement found from 14529 to 14536\n",
      "Seq is: TCCCCCA\n",
      "3 TCCCCCA motifs detected in file\n",
      "\n",
      "Complement found from 8455 to 8462\n",
      "Complement found from 13861 to 13868\n",
      "Complement found from 15184 to 15191\n",
      "Complement found from 15602 to 15609\n",
      "Seq is: ACAAACT\n",
      "4 ACAAACT motifs detected in file\n",
      "\n",
      "Seq is: AGGGCAA\n",
      "0 AGGGCAA motifs detected in file\n",
      "\n",
      "Seq is: AGCACGT\n",
      "0 AGCACGT motifs detected in file\n",
      "\n",
      "Complement found from 4944 to 4951\n",
      "Complement found from 6637 to 6644\n",
      "Seq is: TCTTATC\n",
      "2 TCTTATC motifs detected in file\n",
      "\n",
      "Complement found from 1692 to 1699\n",
      "Complement found from 3791 to 3798\n",
      "Complement found from 5362 to 5369\n",
      "Complement found from 13020 to 13027\n",
      "Complement found from 14048 to 14055\n",
      "Complement found from 15328 to 15335\n",
      "Complement found from 15974 to 15981\n",
      "Seq is: CTCCACC\n",
      "7 CTCCACC motifs detected in file\n",
      "\n",
      "Complement found from 3802 to 3809\n",
      "Complement found from 6396 to 6403\n",
      "Complement found from 7721 to 7728\n",
      "Complement found from 11259 to 11266\n",
      "Complement found from 15821 to 15828\n",
      "Seq is: TCACAAC\n",
      "5 TCACAAC motifs detected in file\n",
      "\n",
      "Seq is: GATCCCG\n",
      "0 GATCCCG motifs detected in file\n",
      "\n",
      "Complement found from 6066 to 6073\n",
      "Seq is: ACGTTAT\n",
      "1 ACGTTAT motifs detected in file\n",
      "\n",
      "Complement found from 7620 to 7627\n",
      "Complement found from 7949 to 7956\n",
      "Complement found from 11200 to 11207\n",
      "Complement found from 14094 to 14101\n",
      "Seq is: TACTTCC\n",
      "4 TACTTCC motifs detected in file\n",
      "\n",
      "Seq is: CGTTCTA\n",
      "0 CGTTCTA motifs detected in file\n",
      "\n",
      "Complement found from 5871 to 5878\n",
      "Seq is: CTTCACT\n",
      "1 CTTCACT motifs detected in file\n",
      "\n",
      "Seq is: TTCCATT\n",
      "0 TTCCATT motifs detected in file\n",
      "\n",
      "Complement found from 5001 to 5008\n",
      "Complement found from 10629 to 10636\n",
      "Seq is: TCTTAGC\n",
      "2 TCTTAGC motifs detected in file\n",
      "\n",
      "Complement found from 6878 to 6885\n",
      "Complement found from 12908 to 12915\n",
      "Complement found from 15325 to 15332\n",
      "Seq is: ACACTCC\n",
      "3 ACACTCC motifs detected in file\n",
      "\n",
      "Complement found from 6314 to 6321\n",
      "Complement found from 6620 to 6627\n",
      "Seq is: CACCCTG\n",
      "2 CACCCTG motifs detected in file\n",
      "\n",
      "Complement found from 8098 to 8105\n",
      "Seq is: ACAGATG\n",
      "1 ACAGATG motifs detected in file\n",
      "\n",
      "Complement found from 4519 to 4526\n",
      "Complement found from 4746 to 4753\n",
      "Complement found from 5452 to 5459\n",
      "Complement found from 6981 to 6988\n",
      "Complement found from 11238 to 11245\n",
      "Complement found from 12698 to 12705\n",
      "Complement found from 13112 to 13119\n",
      "Complement found from 14117 to 14124\n",
      "Seq is: ACTCATC\n",
      "8 ACTCATC motifs detected in file\n",
      "\n",
      "Complement found from 6572 to 6579\n",
      "Seq is: GGAGGAG\n",
      "1 GGAGGAG motifs detected in file\n",
      "\n",
      "Seq is: TCTAAGT\n",
      "0 TCTAAGT motifs detected in file\n",
      "\n",
      "Complement found from 1654 to 1661\n",
      "Complement found from 2903 to 2910\n",
      "Seq is: AACTTGA\n",
      "2 AACTTGA motifs detected in file\n",
      "\n",
      "Seq is: TCCAGAC\n",
      "0 TCCAGAC motifs detected in file\n",
      "\n",
      "Seq is: AGGGTTA\n",
      "0 AGGGTTA motifs detected in file\n",
      "\n",
      "Complement found from 536 to 543\n",
      "Complement found from 3305 to 3312\n",
      "Complement found from 3575 to 3582\n",
      "Complement found from 4132 to 4139\n",
      "Complement found from 4223 to 4230\n",
      "Complement found from 4431 to 4438\n",
      "Complement found from 12427 to 12434\n",
      "Seq is: CATACCC\n",
      "7 CATACCC motifs detected in file\n",
      "\n",
      "Complement found from 1258 to 1265\n",
      "Seq is: TACCGCC\n",
      "1 TACCGCC motifs detected in file\n",
      "\n",
      "Complement found from 15158 to 15165\n",
      "Seq is: TATCATT\n",
      "1 TATCATT motifs detected in file\n",
      "\n",
      "Complement found from 2235 to 2242\n",
      "Complement found from 3991 to 3998\n",
      "Complement found from 9892 to 9899\n",
      "Complement found from 12670 to 12677\n",
      "Complement found from 13531 to 13538\n",
      "Seq is: CAAACAT\n",
      "5 CAAACAT motifs detected in file\n",
      "\n",
      "Complement found from 9204 to 9211\n",
      "Complement found from 9282 to 9289\n",
      "Complement found from 15461 to 15468\n",
      "Seq is: TAATGAC\n",
      "3 TAATGAC motifs detected in file\n",
      "\n",
      "Complement found from 8148 to 8155\n",
      "Seq is: ACCGGGG\n",
      "1 ACCGGGG motifs detected in file\n",
      "\n",
      "Seq is: AGTCTTA\n",
      "0 AGTCTTA motifs detected in file\n",
      "\n",
      "Complement found from 8821 to 8828\n",
      "Seq is: CTATAAA\n",
      "1 CTATAAA motifs detected in file\n",
      "\n",
      "Complement found from 3076 to 3083\n",
      "Seq is: CCGGAGT\n",
      "1 CCGGAGT motifs detected in file\n",
      "\n",
      "Complement found from 7813 to 7820\n",
      "Complement found from 9275 to 9282\n",
      "Complement found from 10265 to 10272\n",
      "Seq is: GCCCTCC\n",
      "3 GCCCTCC motifs detected in file\n",
      "\n",
      "Seq is: TGTATCG\n",
      "0 TGTATCG motifs detected in file\n",
      "\n",
      "Seq is: GATACTT\n",
      "0 GATACTT motifs detected in file\n",
      "\n",
      "Seq is: CCGACAA\n",
      "0 CCGACAA motifs detected in file\n",
      "\n",
      "Complement found from 12869 to 12876\n",
      "Seq is: CGATATC\n",
      "1 CGATATC motifs detected in file\n",
      "\n",
      "Seq is: AACGTTC\n",
      "0 AACGTTC motifs detected in file\n",
      "\n",
      "Complement found from 15139 to 15146\n",
      "Seq is: GTCCTCC\n",
      "1 GTCCTCC motifs detected in file\n",
      "\n",
      "Complement found from 15052 to 15059\n",
      "Seq is: TATTACG\n",
      "1 TATTACG motifs detected in file\n",
      "\n",
      "Complement found from 1288 to 1295\n",
      "Seq is: GCTACAA\n",
      "1 GCTACAA motifs detected in file\n",
      "\n",
      "Complement found from 2096 to 2103\n",
      "Complement found from 5862 to 5869\n",
      "Seq is: AGTCCAA\n",
      "2 AGTCCAA motifs detected in file\n",
      "\n",
      "Seq is: AGACTCA\n",
      "0 AGACTCA motifs detected in file\n",
      "\n",
      "Complement found from 14101 to 14108\n",
      "Seq is: TCTCTTT\n",
      "1 TCTCTTT motifs detected in file\n",
      "\n",
      "Complement found from 9337 to 9344\n",
      "Complement found from 11469 to 11476\n",
      "Complement found from 12974 to 12981\n",
      "Seq is: ACTAGGC\n",
      "3 ACTAGGC motifs detected in file\n",
      "\n",
      "Complement found from 10783 to 10790\n",
      "Complement found from 12780 to 12787\n",
      "Seq is: ATTATAT\n",
      "2 ATTATAT motifs detected in file\n",
      "\n",
      "Complement found from 103 to 110\n",
      "Complement found from 6261 to 6268\n",
      "Seq is: CCGGAGC\n",
      "2 CCGGAGC motifs detected in file\n",
      "\n",
      "Complement found from 1359 to 1366\n",
      "Complement found from 3435 to 3442\n",
      "Seq is: GGGCTAC\n",
      "2 GGGCTAC motifs detected in file\n",
      "\n",
      "Complement found from 7254 to 7261\n",
      "Complement found from 11722 to 11729\n",
      "Seq is: ACATCCT\n",
      "2 ACATCCT motifs detected in file\n",
      "\n",
      "Complement found from 4370 to 4377\n",
      "Seq is: TCTCCGT\n",
      "1 TCTCCGT motifs detected in file\n",
      "\n",
      "Complement found from 8598 to 8605\n",
      "Complement found from 9953 to 9960\n",
      "Seq is: CTATTTC\n",
      "2 CTATTTC motifs detected in file\n",
      "\n",
      "Complement found from 1805 to 1812\n",
      "Complement found from 8502 to 8509\n",
      "Complement found from 11078 to 11085\n",
      "Complement found from 11567 to 11574\n",
      "Seq is: TTATAAC\n",
      "4 TTATAAC motifs detected in file\n",
      "\n",
      "Seq is: TCCGCGG\n",
      "0 TCCGCGG motifs detected in file\n",
      "\n",
      "Complement found from 9004 to 9011\n",
      "Seq is: TAACCGC\n",
      "1 TAACCGC motifs detected in file\n",
      "\n",
      "Complement found from 7029 to 7036\n",
      "Complement found from 9312 to 9319\n",
      "Complement found from 10017 to 10024\n",
      "Seq is: ACTTCCA\n",
      "3 ACTTCCA motifs detected in file\n",
      "\n",
      "Complement found from 15572 to 15579\n",
      "Seq is: TCGCCTA\n",
      "1 TCGCCTA motifs detected in file\n",
      "\n",
      "Complement found from 10355 to 10362\n",
      "Seq is: CTAAGTC\n",
      "1 CTAAGTC motifs detected in file\n",
      "\n",
      "Complement found from 7519 to 7526\n",
      "Seq is: GGTATTA\n",
      "1 GGTATTA motifs detected in file\n",
      "\n",
      "Complement found from 770 to 777\n",
      "Seq is: AATGCAG\n",
      "1 AATGCAG motifs detected in file\n",
      "\n",
      "Complement found from 9643 to 9650\n",
      "Complement found from 12211 to 12218\n",
      "Seq is: AGCTCAC\n",
      "2 AGCTCAC motifs detected in file\n",
      "\n",
      "Complement found from 7109 to 7116\n",
      "Complement found from 8927 to 8934\n",
      "Complement found from 11212 to 11219\n",
      "Seq is: TACACCC\n",
      "3 TACACCC motifs detected in file\n",
      "\n",
      "Complement found from 8303 to 8310\n",
      "Complement found from 12225 to 12232\n",
      "Seq is: GCTAACT\n",
      "2 GCTAACT motifs detected in file\n",
      "\n",
      "Seq is: ACTTTTC\n",
      "0 ACTTTTC motifs detected in file\n",
      "\n",
      "Complement found from 9866 to 9873\n",
      "Seq is: CGCCAAC\n",
      "1 CGCCAAC motifs detected in file\n",
      "\n",
      "Seq is: TCCGCAG\n",
      "0 TCCGCAG motifs detected in file\n",
      "\n",
      "Complement found from 24 to 31\n",
      "Complement found from 4187 to 4194\n",
      "Complement found from 9510 to 9517\n",
      "Complement found from 13185 to 13192\n",
      "Complement found from 14789 to 14796\n",
      "Seq is: ACCACTC\n",
      "5 ACCACTC motifs detected in file\n",
      "\n",
      "Seq is: ATCACGT\n",
      "0 ATCACGT motifs detected in file\n",
      "\n",
      "Seq is: ACCGATC\n",
      "0 ACCGATC motifs detected in file\n",
      "\n",
      "Complement found from 3798 to 3805\n",
      "Complement found from 7642 to 7649\n",
      "Complement found from 9093 to 9100\n",
      "Seq is: CTTATCA\n",
      "3 CTTATCA motifs detected in file\n",
      "\n",
      "Seq is: GGACACC\n",
      "0 GGACACC motifs detected in file\n",
      "\n",
      "Complement found from 4912 to 4919\n",
      "Complement found from 11277 to 11284\n",
      "Complement found from 13685 to 13692\n",
      "Complement found from 14581 to 14588\n",
      "Complement found from 14635 to 14642\n",
      "Seq is: ACTAAAC\n",
      "5 ACTAAAC motifs detected in file\n",
      "\n",
      "Complement found from 10091 to 10098\n",
      "Complement found from 12984 to 12991\n",
      "Complement found from 13989 to 13996\n",
      "Complement found from 15490 to 15497\n",
      "Complement found from 15724 to 15731\n",
      "Seq is: CTCCTAG\n",
      "5 CTCCTAG motifs detected in file\n",
      "\n",
      "Complement found from 6461 to 6468\n",
      "Complement found from 15592 to 15599\n",
      "Seq is: TCCGTCC\n",
      "2 TCCGTCC motifs detected in file\n",
      "\n",
      "Seq is: GCGGTTC\n",
      "0 GCGGTTC motifs detected in file\n",
      "\n",
      "Complement found from 4931 to 4938\n",
      "Complement found from 8791 to 8798\n",
      "Complement found from 11293 to 11300\n",
      "Complement found from 12009 to 12016\n",
      "Complement found from 14847 to 14854\n",
      "Seq is: CTCACTC\n",
      "5 CTCACTC motifs detected in file\n",
      "\n",
      "Complement found from 3933 to 3940\n",
      "Complement found from 5717 to 5724\n",
      "Complement found from 7076 to 7083\n",
      "Seq is: GGCTTCA\n",
      "3 GGCTTCA motifs detected in file\n",
      "\n",
      "Seq is: AAGTAGT\n",
      "0 AAGTAGT motifs detected in file\n",
      "\n",
      "Complement found from 11490 to 11497\n",
      "Complement found from 14918 to 14925\n",
      "Seq is: ACGCCTC\n",
      "2 ACGCCTC motifs detected in file\n",
      "\n",
      "Complement found from 488 to 495\n",
      "Complement found from 3442 to 3449\n",
      "Complement found from 5054 to 5061\n",
      "Complement found from 12855 to 12862\n",
      "Complement found from 14688 to 14695\n",
      "Seq is: TACAACC\n",
      "5 TACAACC motifs detected in file\n",
      "\n",
      "Complement found from 1080 to 1087\n",
      "Complement found from 1372 to 1379\n",
      "Seq is: TACCCCA\n",
      "2 TACCCCA motifs detected in file\n",
      "\n",
      "Complement found from 305 to 312\n",
      "Complement found from 455 to 462\n",
      "Complement found from 956 to 963\n",
      "Complement found from 3567 to 3574\n",
      "Complement found from 10948 to 10955\n",
      "Complement found from 15536 to 15543\n",
      "Complement found from 16184 to 16191\n",
      "Seq is: CCCCTCC\n",
      "7 CCCCTCC motifs detected in file\n",
      "\n",
      "Complement found from 10597 to 10604\n",
      "Seq is: AGCTACT\n",
      "1 AGCTACT motifs detected in file\n",
      "\n",
      "Complement found from 11291 to 11298\n",
      "Complement found from 14908 to 14915\n",
      "Seq is: TACTCAC\n",
      "2 TACTCAC motifs detected in file\n",
      "\n",
      "Complement found from 2189 to 2196\n",
      "Complement found from 2367 to 2374\n",
      "Complement found from 10022 to 10029\n",
      "Seq is: CAATTAA\n",
      "3 CAATTAA motifs detected in file\n",
      "\n",
      "Complement found from 13867 to 13874\n",
      "Seq is: TTAAAAT\n",
      "1 TTAAAAT motifs detected in file\n",
      "\n",
      "Complement found from 2072 to 2079\n",
      "Complement found from 4472 to 4479\n",
      "Complement found from 9578 to 9585\n",
      "Complement found from 13778 to 13785\n",
      "Complement found from 13876 to 13883\n",
      "Complement found from 13943 to 13950\n",
      "Complement found from 15654 to 15661\n",
      "Seq is: AATCCCC\n",
      "7 AATCCCC motifs detected in file\n",
      "\n",
      "Complement found from 2388 to 2395\n",
      "Complement found from 4738 to 4745\n",
      "Complement found from 14573 to 14580\n",
      "Complement found from 15419 to 15426\n",
      "Complement found from 16213 to 16220\n",
      "Seq is: CAATCAA\n",
      "5 CAATCAA motifs detected in file\n",
      "\n",
      "Seq is: AACGCAT\n",
      "0 AACGCAT motifs detected in file\n",
      "\n",
      "Complement found from 350 to 357\n",
      "Complement found from 531 to 538\n",
      "Complement found from 554 to 561\n",
      "Complement found from 864 to 871\n",
      "Complement found from 5598 to 5605\n",
      "Complement found from 7488 to 7495\n",
      "Complement found from 13674 to 13681\n",
      "Complement found from 13689 to 13696\n",
      "Complement found from 14386 to 14393\n",
      "Complement found from 14617 to 14624\n",
      "Complement found from 14626 to 14633\n",
      "Seq is: AACCCCA\n",
      "11 AACCCCA motifs detected in file\n",
      "\n",
      "Complement found from 11905 to 11912\n",
      "Seq is: GTAACCA\n",
      "1 GTAACCA motifs detected in file\n",
      "\n",
      "Complement found from 3215 to 3222\n",
      "Complement found from 3810 to 3817\n",
      "Complement found from 11308 to 11315\n",
      "Complement found from 12217 to 12224\n",
      "Complement found from 14735 to 14742\n",
      "Seq is: CAAGAAC\n",
      "5 CAAGAAC motifs detected in file\n",
      "\n",
      "Complement found from 5611 to 5618\n",
      "Complement found from 5710 to 5717\n",
      "Complement found from 16236 to 16243\n",
      "Seq is: ATCAACT\n",
      "3 ATCAACT motifs detected in file\n",
      "\n",
      "Seq is: TTATGTT\n",
      "0 TTATGTT motifs detected in file\n",
      "\n",
      "Complement found from 6824 to 6831\n",
      "Seq is: GCTACCA\n",
      "1 GCTACCA motifs detected in file\n",
      "\n",
      "Complement found from 2459 to 2466\n",
      "Complement found from 12329 to 12336\n",
      "Seq is: AAAAGTA\n",
      "2 AAAAGTA motifs detected in file\n",
      "\n",
      "Complement found from 11648 to 11655\n",
      "Complement found from 11903 to 11910\n",
      "Seq is: TAGTAAC\n",
      "2 TAGTAAC motifs detected in file\n",
      "\n",
      "Complement found from 5137 to 5144\n",
      "Complement found from 14037 to 14044\n",
      "Complement found from 14361 to 14368\n",
      "Seq is: CAGCACC\n",
      "3 CAGCACC motifs detected in file\n",
      "\n",
      "Complement found from 2505 to 2512\n",
      "Complement found from 9894 to 9901\n",
      "Complement found from 12156 to 12163\n",
      "Seq is: AACATCA\n",
      "3 AACATCA motifs detected in file\n",
      "\n",
      "Seq is: GGCACGC\n",
      "0 GGCACGC motifs detected in file\n",
      "\n",
      "Complement found from 10242 to 10249\n",
      "Complement found from 10332 to 10339\n",
      "Seq is: TCTTATT\n",
      "2 TCTTATT motifs detected in file\n",
      "\n",
      "Seq is: ACTACAT\n",
      "0 ACTACAT motifs detected in file\n",
      "\n",
      "Complement found from 15347 to 15354\n",
      "Seq is: ACGAAAC\n",
      "1 ACGAAAC motifs detected in file\n",
      "\n",
      "Complement found from 560 to 567\n",
      "Complement found from 5939 to 5946\n",
      "Complement found from 14472 to 14479\n",
      "Seq is: AAAGACA\n",
      "3 AAAGACA motifs detected in file\n",
      "\n",
      "Complement found from 7719 to 7726\n",
      "Complement found from 11257 to 11264\n",
      "Complement found from 11767 to 11774\n",
      "Complement found from 13796 to 13803\n",
      "Seq is: ACTCACA\n",
      "4 ACTCACA motifs detected in file\n",
      "\n",
      "Complement found from 1731 to 1738\n",
      "Complement found from 5252 to 5259\n",
      "Seq is: CCCAAAT\n",
      "2 CCCAAAT motifs detected in file\n",
      "\n",
      "Complement found from 12117 to 12124\n",
      "Seq is: CATTACC\n",
      "1 CATTACC motifs detected in file\n",
      "\n",
      "Complement found from 2234 to 2241\n",
      "Complement found from 3716 to 3723\n",
      "Complement found from 9891 to 9898\n",
      "Complement found from 12548 to 12555\n",
      "Complement found from 12669 to 12676\n",
      "Complement found from 13769 to 13776\n",
      "Complement found from 15674 to 15681\n",
      "Seq is: CCAAACA\n",
      "7 CCAAACA motifs detected in file\n",
      "\n",
      "Complement found from 352 to 359\n",
      "Complement found from 556 to 563\n",
      "Complement found from 1681 to 1688\n",
      "Complement found from 7467 to 7474\n",
      "Complement found from 12301 to 12308\n",
      "Complement found from 14531 to 14538\n",
      "Seq is: CCCCAAA\n",
      "6 CCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: AGAAGAT\n",
      "0 AGAAGAT motifs detected in file\n",
      "\n",
      "Seq is: TCGGAAG\n",
      "0 TCGGAAG motifs detected in file\n",
      "\n",
      "Complement found from 12273 to 12280\n",
      "Seq is: AGGATAA\n",
      "1 AGGATAA motifs detected in file\n",
      "\n",
      "Complement found from 537 to 544\n",
      "Complement found from 1079 to 1086\n",
      "Complement found from 4133 to 4140\n",
      "Complement found from 4432 to 4439\n",
      "Complement found from 12428 to 12435\n",
      "Seq is: ATACCCC\n",
      "5 ATACCCC motifs detected in file\n",
      "\n",
      "Complement found from 5623 to 5630\n",
      "Complement found from 7875 to 7882\n",
      "Complement found from 10888 to 10895\n",
      "Complement found from 13001 to 13008\n",
      "Complement found from 14875 to 14882\n",
      "Complement found from 15951 to 15958\n",
      "Seq is: CAAATCA\n",
      "6 CAAATCA motifs detected in file\n",
      "\n",
      "Complement found from 849 to 856\n",
      "Complement found from 1521 to 1528\n",
      "Complement found from 7733 to 7740\n",
      "Seq is: TAACTAA\n",
      "3 TAACTAA motifs detected in file\n",
      "\n",
      "Complement found from 14939 to 14946\n",
      "Seq is: CAATCGC\n",
      "1 CAATCGC motifs detected in file\n",
      "\n",
      "Complement found from 6768 to 6775\n",
      "Complement found from 9418 to 9425\n",
      "Seq is: CACACCA\n",
      "2 CACACCA motifs detected in file\n",
      "\n",
      "Complement found from 2055 to 2062\n",
      "Complement found from 14944 to 14951\n",
      "Complement found from 16534 to 16541\n",
      "Seq is: GCCCACA\n",
      "3 GCCCACA motifs detected in file\n",
      "\n",
      "Complement found from 15650 to 15657\n",
      "Seq is: CAATAAT\n",
      "1 CAATAAT motifs detected in file\n",
      "\n",
      "Complement found from 2857 to 2864\n",
      "Seq is: AAGACTT\n",
      "1 AAGACTT motifs detected in file\n",
      "\n",
      "Complement found from 1526 to 1533\n",
      "Complement found from 3488 to 3495\n",
      "Complement found from 5428 to 5435\n",
      "Complement found from 5596 to 5603\n",
      "Complement found from 6415 to 6422\n",
      "Complement found from 9238 to 9245\n",
      "Complement found from 12036 to 12043\n",
      "Complement found from 14615 to 14622\n",
      "Complement found from 16162 to 16169\n",
      "Complement found from 16179 to 16186\n",
      "Seq is: AAAACCC\n",
      "10 AAAACCC motifs detected in file\n",
      "\n",
      "Complement found from 8623 to 8630\n",
      "Complement found from 14871 to 14878\n",
      "Seq is: CCTCCAA\n",
      "2 CCTCCAA motifs detected in file\n",
      "\n",
      "Complement found from 3199 to 3206\n",
      "Complement found from 4451 to 4458\n",
      "Complement found from 15511 to 15518\n",
      "Seq is: TATACCC\n",
      "3 TATACCC motifs detected in file\n",
      "\n",
      "Complement found from 1802 to 1809\n",
      "Complement found from 7556 to 7563\n",
      "Complement found from 8499 to 8506\n",
      "Complement found from 10444 to 10451\n",
      "Complement found from 14294 to 14301\n",
      "Complement found from 14964 to 14971\n",
      "Seq is: AAATTAT\n",
      "6 AAATTAT motifs detected in file\n",
      "\n",
      "Complement found from 284 to 291\n",
      "Complement found from 4603 to 4610\n",
      "Complement found from 5280 to 5287\n",
      "Complement found from 7445 to 7452\n",
      "Complement found from 10046 to 10053\n",
      "Complement found from 10812 to 10819\n",
      "Complement found from 12416 to 12423\n",
      "Complement found from 13229 to 13236\n",
      "Seq is: CAAAAAA\n",
      "8 CAAAAAA motifs detected in file\n",
      "\n",
      "Complement found from 10441 to 10448\n",
      "Seq is: ATTAAAT\n",
      "1 ATTAAAT motifs detected in file\n",
      "\n",
      "Complement found from 2071 to 2078\n",
      "Complement found from 2229 to 2236\n",
      "Complement found from 9577 to 9584\n",
      "Complement found from 13875 to 13882\n",
      "Complement found from 16348 to 16355\n",
      "Seq is: AAATCCC\n",
      "5 AAATCCC motifs detected in file\n",
      "\n",
      "Complement found from 1697 to 1704\n",
      "Complement found from 5651 to 5658\n",
      "Complement found from 10098 to 10105\n",
      "Complement found from 13652 to 13659\n",
      "Complement found from 15409 to 15416\n",
      "Seq is: CCTTACT\n",
      "5 CCTTACT motifs detected in file\n",
      "\n",
      "Complement found from 2394 to 2401\n",
      "Complement found from 6594 to 6601\n",
      "Complement found from 8343 to 8350\n",
      "Complement found from 9350 to 9357\n",
      "Complement found from 13857 to 13864\n",
      "Complement found from 14184 to 14191\n",
      "Complement found from 16276 to 16283\n",
      "Seq is: ACCAACA\n",
      "7 ACCAACA motifs detected in file\n",
      "\n",
      "Complement found from 1759 to 1766\n",
      "Complement found from 15076 to 15083\n",
      "Seq is: GAAACCT\n",
      "2 GAAACCT motifs detected in file\n",
      "\n",
      "Complement found from 3855 to 3862\n",
      "Complement found from 4209 to 4216\n",
      "Seq is: ATATGAT\n",
      "2 ATATGAT motifs detected in file\n",
      "\n",
      "Seq is: GTCGCTA\n",
      "0 GTCGCTA motifs detected in file\n",
      "\n",
      "Complement found from 15389 to 15396\n",
      "Seq is: CCGATAA\n",
      "1 CCGATAA motifs detected in file\n",
      "\n",
      "Complement found from 591 to 598\n",
      "Complement found from 1494 to 1501\n",
      "Complement found from 2251 to 2258\n",
      "Complement found from 3327 to 3334\n",
      "Complement found from 4928 to 4935\n",
      "Complement found from 5011 to 5018\n",
      "Complement found from 9329 to 9336\n",
      "Complement found from 14442 to 14449\n",
      "Complement found from 15739 to 15746\n",
      "Seq is: CTCCTCA\n",
      "9 CTCCTCA motifs detected in file\n",
      "\n",
      "Complement found from 4329 to 4336\n",
      "Complement found from 13264 to 13271\n",
      "Complement found from 13822 to 13829\n",
      "Seq is: TAGGACT\n",
      "3 TAGGACT motifs detected in file\n",
      "\n",
      "Seq is: CCGTACC\n",
      "0 CCGTACC motifs detected in file\n",
      "\n",
      "Complement found from 6499 to 6506\n",
      "Complement found from 13058 to 13065\n",
      "Seq is: CCCAGTC\n",
      "2 CCCAGTC motifs detected in file\n",
      "\n",
      "Complement found from 435 to 442\n",
      "Complement found from 1625 to 1632\n",
      "Complement found from 8369 to 8376\n",
      "Complement found from 8435 to 8442\n",
      "Complement found from 8812 to 8819\n",
      "Seq is: CCCAACT\n",
      "5 CCCAACT motifs detected in file\n",
      "\n",
      "Complement found from 1134 to 1141\n",
      "Complement found from 11176 to 11183\n",
      "Seq is: CCAGAAC\n",
      "2 CCAGAAC motifs detected in file\n",
      "\n",
      "Complement found from 26 to 33\n",
      "Complement found from 4189 to 4196\n",
      "Complement found from 7718 to 7725\n",
      "Complement found from 11256 to 11263\n",
      "Complement found from 11766 to 11773\n",
      "Complement found from 12011 to 12018\n",
      "Complement found from 14399 to 14406\n",
      "Seq is: CACTCAC\n",
      "7 CACTCAC motifs detected in file\n",
      "\n",
      "Complement found from 5809 to 5816\n",
      "Complement found from 15394 to 15401\n",
      "Seq is: AAAATCA\n",
      "2 AAAATCA motifs detected in file\n",
      "\n",
      "Complement found from 961 to 968\n",
      "Complement found from 9533 to 9540\n",
      "Complement found from 14750 to 14757\n",
      "Seq is: CCCCAAT\n",
      "3 CCCCAAT motifs detected in file\n",
      "\n",
      "Complement found from 6823 to 6830\n",
      "Complement found from 13577 to 13584\n",
      "Complement found from 14983 to 14990\n",
      "Seq is: CGCTACC\n",
      "3 CGCTACC motifs detected in file\n",
      "\n",
      "Complement found from 5200 to 5207\n",
      "Complement found from 15634 to 15641\n",
      "Seq is: TCCATCC\n",
      "2 TCCATCC motifs detected in file\n",
      "\n",
      "Complement found from 22 to 29\n",
      "Complement found from 11906 to 11913\n",
      "Complement found from 12355 to 12362\n",
      "Complement found from 14787 to 14794\n",
      "Seq is: TAACCAC\n",
      "4 TAACCAC motifs detected in file\n",
      "\n",
      "Complement found from 563 to 570\n",
      "Seq is: GACACCC\n",
      "1 GACACCC motifs detected in file\n",
      "\n",
      "Complement found from 7172 to 7179\n",
      "Complement found from 7188 to 7195\n",
      "Seq is: ACTTTCT\n",
      "2 ACTTTCT motifs detected in file\n",
      "\n",
      "Complement found from 468 to 475\n",
      "Complement found from 8942 to 8949\n",
      "Complement found from 9334 to 9341\n",
      "Complement found from 10654 to 10661\n",
      "Complement found from 11955 to 11962\n",
      "Complement found from 12716 to 12723\n",
      "Complement found from 13346 to 13353\n",
      "Seq is: CATACTA\n",
      "7 CATACTA motifs detected in file\n",
      "\n",
      "Complement found from 5302 to 5309\n",
      "Complement found from 6844 to 6851\n",
      "Complement found from 7398 to 7405\n",
      "Complement found from 8618 to 8625\n",
      "Complement found from 11139 to 11146\n",
      "Complement found from 13052 to 13059\n",
      "Complement found from 13646 to 13653\n",
      "Complement found from 13676 to 13683\n",
      "Complement found from 14808 to 14815\n",
      "Seq is: CCCCACC\n",
      "9 CCCCACC motifs detected in file\n",
      "\n",
      "Complement found from 2394 to 2401\n",
      "Complement found from 6594 to 6601\n",
      "Complement found from 8343 to 8350\n",
      "Complement found from 9350 to 9357\n",
      "Complement found from 13857 to 13864\n",
      "Complement found from 14184 to 14191\n",
      "Complement found from 16276 to 16283\n",
      "Seq is: ACCAACA\n",
      "7 ACCAACA motifs detected in file\n",
      "\n",
      "Complement found from 6701 to 6708\n",
      "Seq is: CCATTTG\n",
      "1 CCATTTG motifs detected in file\n",
      "\n",
      "Complement found from 1802 to 1809\n",
      "Complement found from 7556 to 7563\n",
      "Complement found from 8499 to 8506\n",
      "Complement found from 10444 to 10451\n",
      "Complement found from 14294 to 14301\n",
      "Complement found from 14964 to 14971\n",
      "Seq is: AAATTAT\n",
      "6 AAATTAT motifs detected in file\n",
      "\n",
      "Complement found from 9604 to 9611\n",
      "Seq is: CACATCC\n",
      "1 CACATCC motifs detected in file\n",
      "\n",
      "Complement found from 10433 to 10440\n",
      "Seq is: TTCGACT\n",
      "1 TTCGACT motifs detected in file\n",
      "\n",
      "Complement found from 3023 to 3030\n",
      "Complement found from 8447 to 8454\n",
      "Complement found from 14315 to 14322\n",
      "Complement found from 14511 to 14518\n",
      "Seq is: TATTAAA\n",
      "4 TATTAAA motifs detected in file\n",
      "\n",
      "Complement found from 7620 to 7627\n",
      "Complement found from 7949 to 7956\n",
      "Complement found from 11200 to 11207\n",
      "Complement found from 14094 to 14101\n",
      "Seq is: TACTTCC\n",
      "4 TACTTCC motifs detected in file\n",
      "\n",
      "Complement found from 4054 to 4061\n",
      "Complement found from 5484 to 5491\n",
      "Seq is: TCTCCCC\n",
      "2 TCTCCCC motifs detected in file\n",
      "\n",
      "Complement found from 4810 to 4817\n",
      "Complement found from 9589 to 9596\n",
      "Complement found from 15259 to 15266\n",
      "Seq is: AGTCCCA\n",
      "3 AGTCCCA motifs detected in file\n",
      "\n",
      "Complement found from 851 to 858\n",
      "Complement found from 15702 to 15709\n",
      "Seq is: ACTAAGC\n",
      "2 ACTAAGC motifs detected in file\n",
      "\n",
      "Complement found from 3320 to 3327\n",
      "Complement found from 3608 to 3615\n",
      "Complement found from 4166 to 4173\n",
      "Complement found from 9277 to 9284\n",
      "Complement found from 9727 to 9734\n",
      "Complement found from 10090 to 10097\n",
      "Complement found from 10950 to 10957\n",
      "Complement found from 12093 to 12100\n",
      "Complement found from 12983 to 12990\n",
      "Complement found from 13988 to 13995\n",
      "Complement found from 15333 to 15340\n",
      "Complement found from 15489 to 15496\n",
      "Seq is: CCTCCTA\n",
      "12 CCTCCTA motifs detected in file\n",
      "\n",
      "Complement found from 2932 to 2939\n",
      "Complement found from 12274 to 12281\n",
      "Seq is: GGATAAC\n",
      "2 GGATAAC motifs detected in file\n",
      "\n",
      "Complement found from 7660 to 7667\n",
      "Seq is: CACGCCC\n",
      "1 CACGCCC motifs detected in file\n",
      "\n",
      "Complement found from 1903 to 1910\n",
      "Complement found from 9670 to 9677\n",
      "Seq is: CCGAAAC\n",
      "2 CCGAAAC motifs detected in file\n",
      "\n",
      "Complement found from 859 to 866\n",
      "Complement found from 5495 to 5502\n",
      "Complement found from 7739 to 7746\n",
      "Complement found from 10957 to 10964\n",
      "Complement found from 11050 to 11057\n",
      "Complement found from 12717 to 12724\n",
      "Complement found from 14579 to 14586\n",
      "Seq is: ATACTAA\n",
      "7 ATACTAA motifs detected in file\n",
      "\n",
      "Complement found from 8575 to 8582\n",
      "Seq is: TACCCGC\n",
      "1 TACCCGC motifs detected in file\n",
      "\n",
      "Complement found from 2292 to 2299\n",
      "Complement found from 7734 to 7741\n",
      "Complement found from 9870 to 9877\n",
      "Complement found from 10766 to 10773\n",
      "Complement found from 11099 to 11106\n",
      "Complement found from 15901 to 15908\n",
      "Seq is: AACTAAT\n",
      "6 AACTAAT motifs detected in file\n",
      "\n",
      "Complement found from 5296 to 5303\n",
      "Complement found from 7794 to 7801\n",
      "Complement found from 10342 to 10349\n",
      "Complement found from 10863 to 10870\n",
      "Complement found from 13370 to 13377\n",
      "Complement found from 13517 to 13524\n",
      "Complement found from 14057 to 14064\n",
      "Seq is: CATCATC\n",
      "7 CATCATC motifs detected in file\n",
      "\n",
      "Complement found from 9205 to 9212\n",
      "Complement found from 9283 to 9290\n",
      "Complement found from 14745 to 14752\n",
      "Seq is: AATGACC\n",
      "3 AATGACC motifs detected in file\n",
      "\n",
      "Seq is: AACTTGT\n",
      "0 AACTTGT motifs detected in file\n",
      "\n",
      "Complement found from 2763 to 2770\n",
      "Complement found from 8506 to 8513\n",
      "Complement found from 13860 to 13867\n",
      "Complement found from 14187 to 14194\n",
      "Complement found from 15601 to 15608\n",
      "Complement found from 16279 to 16286\n",
      "Seq is: AACAAAC\n",
      "6 AACAAAC motifs detected in file\n",
      "\n",
      "Complement found from 5933 to 5940\n",
      "Complement found from 8191 to 8198\n",
      "Complement found from 11127 to 11134\n",
      "Complement found from 13297 to 13304\n",
      "Seq is: AACCACA\n",
      "4 AACCACA motifs detected in file\n",
      "\n",
      "Complement found from 12654 to 12661\n",
      "Seq is: TATATAA\n",
      "1 TATATAA motifs detected in file\n",
      "\n",
      "Complement found from 8313 to 8320\n",
      "Complement found from 9074 to 9081\n",
      "Complement found from 12024 to 12031\n",
      "Complement found from 13661 to 13668\n",
      "Complement found from 15467 to 15474\n",
      "Seq is: CATTAAC\n",
      "5 CATTAAC motifs detected in file\n",
      "\n",
      "Seq is: TCTAACG\n",
      "0 TCTAACG motifs detected in file\n",
      "\n",
      "Complement found from 4792 to 4799\n",
      "Complement found from 4876 to 4883\n",
      "Complement found from 8025 to 8032\n",
      "Complement found from 11424 to 11431\n",
      "Complement found from 14241 to 14248\n",
      "Seq is: AGCCCCC\n",
      "5 AGCCCCC motifs detected in file\n",
      "\n",
      "Complement found from 4946 to 4953\n",
      "Complement found from 11008 to 11015\n",
      "Seq is: TTATCCA\n",
      "2 TTATCCA motifs detected in file\n",
      "\n",
      "Complement found from 603 to 610\n",
      "Seq is: ATACACT\n",
      "1 ATACACT motifs detected in file\n",
      "\n",
      "Complement found from 3650 to 3657\n",
      "Complement found from 11784 to 11791\n",
      "Seq is: AATCCTC\n",
      "2 AATCCTC motifs detected in file\n",
      "\n",
      "Complement found from 2231 to 2238\n",
      "Seq is: ATCCCAA\n",
      "1 ATCCCAA motifs detected in file\n",
      "\n",
      "Complement found from 3322 to 3329\n",
      "Complement found from 5115 to 5122\n",
      "Complement found from 6228 to 6235\n",
      "Complement found from 6480 to 6487\n",
      "Complement found from 11936 to 11943\n",
      "Complement found from 16505 to 16512\n",
      "Seq is: TCCTACT\n",
      "6 TCCTACT motifs detected in file\n",
      "\n",
      "Complement found from 3794 to 3801\n",
      "Complement found from 5190 to 5197\n",
      "Complement found from 13649 to 13656\n",
      "Complement found from 15406 to 15413\n",
      "Complement found from 16291 to 16298\n",
      "Seq is: CACCCTT\n",
      "5 CACCCTT motifs detected in file\n",
      "\n",
      "Seq is: CTTCTTT\n",
      "0 CTTCTTT motifs detected in file\n",
      "\n",
      "Complement found from 6225 to 6232\n",
      "Complement found from 11933 to 11940\n",
      "Complement found from 14283 to 14290\n",
      "Complement found from 15454 to 15461\n",
      "Complement found from 16441 to 16448\n",
      "Seq is: CTCTCCT\n",
      "5 CTCTCCT motifs detected in file\n",
      "\n",
      "Complement found from 7278 to 7285\n",
      "Complement found from 15061 to 15068\n",
      "Seq is: TCATTTC\n",
      "2 TCATTTC motifs detected in file\n",
      "\n",
      "Complement found from 14498 to 14505\n",
      "Complement found from 14779 to 14786\n",
      "Seq is: AAATTAA\n",
      "2 AAATTAA motifs detected in file\n",
      "\n",
      "Seq is: TACGGAA\n",
      "0 TACGGAA motifs detected in file\n",
      "\n",
      "Complement found from 1685 to 1692\n",
      "Complement found from 2766 to 2773\n",
      "Complement found from 4983 to 4990\n",
      "Complement found from 5429 to 5436\n",
      "Complement found from 5673 to 5680\n",
      "Complement found from 9239 to 9246\n",
      "Complement found from 14515 to 14522\n",
      "Complement found from 14638 to 14645\n",
      "Complement found from 16163 to 16170\n",
      "Seq is: AAACCCA\n",
      "9 AAACCCA motifs detected in file\n",
      "\n",
      "Complement found from 1691 to 1698\n",
      "Complement found from 5361 to 5368\n",
      "Complement found from 6880 to 6887\n",
      "Complement found from 11391 to 11398\n",
      "Complement found from 13073 to 13080\n",
      "Complement found from 15327 to 15334\n",
      "Complement found from 15973 to 15980\n",
      "Seq is: ACTCCAC\n",
      "7 ACTCCAC motifs detected in file\n",
      "\n",
      "Complement found from 1700 to 1707\n",
      "Complement found from 4732 to 4739\n",
      "Complement found from 5104 to 5111\n",
      "Complement found from 8379 to 8386\n",
      "Complement found from 10790 to 10797\n",
      "Seq is: TACTACC\n",
      "5 TACTACC motifs detected in file\n",
      "\n",
      "Complement found from 1693 to 1700\n",
      "Complement found from 5363 to 5370\n",
      "Complement found from 12459 to 12466\n",
      "Complement found from 14049 to 14056\n",
      "Complement found from 15329 to 15336\n",
      "Seq is: TCCACCT\n",
      "5 TCCACCT motifs detected in file\n",
      "\n",
      "Complement found from 1099 to 1106\n",
      "Complement found from 1671 to 1678\n",
      "Complement found from 8119 to 8126\n",
      "Complement found from 10745 to 10752\n",
      "Complement found from 13686 to 13693\n",
      "Complement found from 14582 to 14589\n",
      "Complement found from 14636 to 14643\n",
      "Seq is: CTAAACC\n",
      "7 CTAAACC motifs detected in file\n",
      "\n",
      "Complement found from 7005 to 7012\n",
      "Seq is: ACGACAC\n",
      "1 ACGACAC motifs detected in file\n",
      "\n",
      "Complement found from 421 to 428\n",
      "Complement found from 1032 to 1039\n",
      "Complement found from 2156 to 2163\n",
      "Seq is: TTTAACA\n",
      "3 TTTAACA motifs detected in file\n",
      "\n",
      "Complement found from 7134 to 7141\n",
      "Complement found from 10154 to 10161\n",
      "Complement found from 12444 to 12451\n",
      "Seq is: AAATCCA\n",
      "3 AAATCCA motifs detected in file\n",
      "\n",
      "Complement found from 5265 to 5272\n",
      "Complement found from 8953 to 8960\n",
      "Seq is: TTATCGA\n",
      "2 TTATCGA motifs detected in file\n",
      "\n",
      "Seq is: AAAGATC\n",
      "0 AAAGATC motifs detected in file\n",
      "\n",
      "Complement found from 2096 to 2103\n",
      "Complement found from 5862 to 5869\n",
      "Seq is: AGTCCAA\n",
      "2 AGTCCAA motifs detected in file\n",
      "\n",
      "Seq is: TCCAGAT\n",
      "0 TCCAGAT motifs detected in file\n",
      "\n",
      "Complement found from 590 to 597\n",
      "Complement found from 1493 to 1500\n",
      "Complement found from 5209 to 5216\n",
      "Complement found from 8774 to 8781\n",
      "Complement found from 12980 to 12987\n",
      "Complement found from 15738 to 15745\n",
      "Seq is: CCTCCTC\n",
      "6 CCTCCTC motifs detected in file\n",
      "\n",
      "Complement found from 702 to 709\n",
      "Seq is: ATCCCCG\n",
      "1 ATCCCCG motifs detected in file\n",
      "\n",
      "Complement found from 4362 to 4369\n",
      "Complement found from 9429 to 9436\n",
      "Complement found from 10810 to 10817\n",
      "Seq is: TCCAAAA\n",
      "3 TCCAAAA motifs detected in file\n",
      "\n",
      "Complement found from 2841 to 2848\n",
      "Complement found from 12815 to 12822\n",
      "Complement found from 14157 to 14164\n",
      "Seq is: CCGAGCA\n",
      "3 CCGAGCA motifs detected in file\n",
      "\n",
      "Complement found from 7175 to 7182\n",
      "Complement found from 14109 to 14116\n",
      "Seq is: TTCTTCC\n",
      "2 TTCTTCC motifs detected in file\n",
      "\n",
      "Seq is: GGATCTA\n",
      "0 GGATCTA motifs detected in file\n",
      "\n",
      "Complement found from 6151 to 6158\n",
      "Complement found from 7093 to 7100\n",
      "Complement found from 7624 to 7631\n",
      "Complement found from 8226 to 8233\n",
      "Complement found from 9580 to 9587\n",
      "Complement found from 11232 to 11239\n",
      "Complement found from 13945 to 13952\n",
      "Seq is: TCCCCTA\n",
      "7 TCCCCTA motifs detected in file\n",
      "\n",
      "Seq is: ACACGCC\n",
      "0 ACACGCC motifs detected in file\n",
      "\n",
      "Complement found from 9354 to 9361\n",
      "Seq is: ACACACT\n",
      "1 ACACACT motifs detected in file\n",
      "\n",
      "Complement found from 7856 to 7863\n",
      "Seq is: ACGATCC\n",
      "1 ACGATCC motifs detected in file\n",
      "\n",
      "Seq is: CATCTTA\n",
      "0 CATCTTA motifs detected in file\n",
      "\n",
      "Complement found from 7271 to 7278\n",
      "Seq is: GGCTCAT\n",
      "1 GGCTCAT motifs detected in file\n",
      "\n",
      "Complement found from 1709 to 1716\n",
      "Complement found from 10898 to 10905\n",
      "Complement found from 12739 to 12746\n",
      "Complement found from 13378 to 13385\n",
      "Seq is: ACAACCT\n",
      "4 ACAACCT motifs detected in file\n",
      "\n",
      "Complement found from 12464 to 12471\n",
      "Complement found from 15715 to 15722\n",
      "Seq is: CTTTATT\n",
      "2 CTTTATT motifs detected in file\n",
      "\n",
      "Seq is: TTACACG\n",
      "0 TTACACG motifs detected in file\n",
      "\n",
      "Complement found from 11 to 18\n",
      "Complement found from 2276 to 2283\n",
      "Complement found from 7644 to 7651\n",
      "Complement found from 13181 to 13188\n",
      "Seq is: TATCACC\n",
      "4 TATCACC motifs detected in file\n",
      "\n",
      "Complement found from 2174 to 2181\n",
      "Complement found from 2223 to 2230\n",
      "Complement found from 3485 to 3492\n",
      "Complement found from 8229 to 8236\n",
      "Complement found from 11601 to 11608\n",
      "Complement found from 13790 to 13797\n",
      "Complement found from 14022 to 14029\n",
      "Seq is: CCTAAAA\n",
      "7 CCTAAAA motifs detected in file\n",
      "\n",
      "Complement found from 10215 to 10222\n",
      "Complement found from 14777 to 14784\n",
      "Seq is: TAAAATT\n",
      "2 TAAAATT motifs detected in file\n",
      "\n",
      "Complement found from 3717 to 3724\n",
      "Complement found from 10295 to 10302\n",
      "Complement found from 12549 to 12556\n",
      "Complement found from 13770 to 13777\n",
      "Complement found from 14189 to 14196\n",
      "Complement found from 15359 to 15366\n",
      "Complement found from 15675 to 15682\n",
      "Seq is: CAAACAA\n",
      "7 CAAACAA motifs detected in file\n",
      "\n",
      "Complement found from 14366 to 14373\n",
      "Complement found from 16167 to 16174\n",
      "Seq is: CCAATCC\n",
      "2 CCAATCC motifs detected in file\n",
      "\n",
      "Complement found from 5611 to 5618\n",
      "Complement found from 5710 to 5717\n",
      "Complement found from 16236 to 16243\n",
      "Seq is: ATCAACT\n",
      "3 ATCAACT motifs detected in file\n",
      "\n",
      "Seq is: CCAGATC\n",
      "0 CCAGATC motifs detected in file\n",
      "\n",
      "Seq is: CCGACAA\n",
      "0 CCGACAA motifs detected in file\n",
      "\n",
      "Complement found from 1266 to 1273\n",
      "Seq is: TCTTCAG\n",
      "1 TCTTCAG motifs detected in file\n",
      "\n",
      "Seq is: GATCCTT\n",
      "0 GATCCTT motifs detected in file\n",
      "\n",
      "Complement found from 2415 to 2422\n",
      "Complement found from 12645 to 12652\n",
      "Seq is: TCACTGT\n",
      "2 TCACTGT motifs detected in file\n",
      "\n",
      "Seq is: CGCTCCC\n",
      "0 CGCTCCC motifs detected in file\n",
      "\n",
      "Seq is: GACACTC\n",
      "0 GACACTC motifs detected in file\n",
      "\n",
      "Complement found from 2228 to 2235\n",
      "Complement found from 7133 to 7140\n",
      "Complement found from 10153 to 10160\n",
      "Complement found from 12443 to 12450\n",
      "Complement found from 13874 to 13881\n",
      "Seq is: AAAATCC\n",
      "5 AAAATCC motifs detected in file\n",
      "\n",
      "Complement found from 5483 to 5490\n",
      "Complement found from 6662 to 6669\n",
      "Complement found from 11056 to 11063\n",
      "Complement found from 15850 to 15857\n",
      "Seq is: ATCTCCC\n",
      "4 ATCTCCC motifs detected in file\n",
      "\n",
      "Complement found from 369 to 376\n",
      "Complement found from 2760 to 2767\n",
      "Complement found from 5060 to 5067\n",
      "Complement found from 7713 to 7720\n",
      "Complement found from 9256 to 9263\n",
      "Complement found from 10938 to 10945\n",
      "Complement found from 12411 to 12418\n",
      "Complement found from 13625 to 13632\n",
      "Complement found from 15598 to 15605\n",
      "Seq is: CCTAACA\n",
      "9 CCTAACA motifs detected in file\n",
      "\n",
      "Complement found from 1990 to 1997\n",
      "Complement found from 2764 to 2771\n",
      "Complement found from 5930 to 5937\n",
      "Complement found from 8507 to 8514\n",
      "Complement found from 14623 to 14630\n",
      "Complement found from 16280 to 16287\n",
      "Seq is: ACAAACC\n",
      "6 ACAAACC motifs detected in file\n",
      "\n",
      "Complement found from 3165 to 3172\n",
      "Complement found from 4246 to 4253\n",
      "Complement found from 7952 to 7959\n",
      "Complement found from 8602 to 8609\n",
      "Complement found from 12382 to 12389\n",
      "Complement found from 13751 to 13758\n",
      "Complement found from 14152 to 14159\n",
      "Complement found from 14486 to 14493\n",
      "Seq is: TTCCCCC\n",
      "8 TTCCCCC motifs detected in file\n",
      "\n",
      "Complement found from 3684 to 3691\n",
      "Complement found from 12810 to 12817\n",
      "Seq is: TACGCCC\n",
      "2 TACGCCC motifs detected in file\n",
      "\n",
      "Complement found from 7833 to 7840\n",
      "Seq is: CCTTTAC\n",
      "1 CCTTTAC motifs detected in file\n",
      "\n",
      "Complement found from 3719 to 3726\n",
      "Complement found from 13775 to 13782\n",
      "Complement found from 14571 to 14578\n",
      "Complement found from 15825 to 15832\n",
      "Seq is: AACAATC\n",
      "4 AACAATC motifs detected in file\n",
      "\n",
      "Complement found from 3824 to 3831\n",
      "Complement found from 9613 to 9620\n",
      "Complement found from 13566 to 13573\n",
      "Seq is: ATTACTC\n",
      "3 ATTACTC motifs detected in file\n",
      "\n",
      "Seq is: TTTAGGA\n",
      "0 TTTAGGA motifs detected in file\n",
      "\n",
      "Seq is: CGCTTAT\n",
      "0 CGCTTAT motifs detected in file\n",
      "\n",
      "Complement found from 3907 to 3914\n",
      "Seq is: CCGAAGG\n",
      "1 CCGAAGG motifs detected in file\n",
      "\n",
      "Complement found from 9771 to 9778\n",
      "Seq is: CCGACGG\n",
      "1 CCGACGG motifs detected in file\n",
      "\n",
      "Seq is: TCCGCAA\n",
      "0 TCCGCAA motifs detected in file\n",
      "\n",
      "Complement found from 2408 to 2415\n",
      "Complement found from 5018 to 5025\n",
      "Complement found from 8401 to 8408\n",
      "Seq is: ATTACCC\n",
      "3 ATTACCC motifs detected in file\n",
      "\n",
      "Complement found from 5623 to 5630\n",
      "Complement found from 7875 to 7882\n",
      "Complement found from 10888 to 10895\n",
      "Complement found from 13001 to 13008\n",
      "Complement found from 14875 to 14882\n",
      "Complement found from 15951 to 15958\n",
      "Seq is: CAAATCA\n",
      "6 CAAATCA motifs detected in file\n",
      "\n",
      "Complement found from 6311 to 6318\n",
      "Complement found from 15261 to 15268\n",
      "Seq is: TCCCACC\n",
      "2 TCCCACC motifs detected in file\n",
      "\n",
      "Complement found from 963 to 970\n",
      "Complement found from 2898 to 2905\n",
      "Seq is: CCAATAA\n",
      "2 CCAATAA motifs detected in file\n",
      "\n",
      "Complement found from 598 to 605\n",
      "Complement found from 6889 to 6896\n",
      "Complement found from 12844 to 12851\n",
      "Seq is: AAGCAAT\n",
      "3 AAGCAAT motifs detected in file\n",
      "\n",
      "Complement found from 2386 to 2393\n",
      "Complement found from 4022 to 4029\n",
      "Complement found from 4232 to 4239\n",
      "Complement found from 13280 to 13287\n",
      "Seq is: TACAATC\n",
      "4 TACAATC motifs detected in file\n",
      "\n",
      "Seq is: ATTCCCA\n",
      "0 ATTCCCA motifs detected in file\n",
      "\n",
      "Complement found from 3893 to 3900\n",
      "Complement found from 3959 to 3966\n",
      "Complement found from 11535 to 11542\n",
      "Complement found from 13763 to 13770\n",
      "Seq is: CCCCTTC\n",
      "4 CCCCTTC motifs detected in file\n",
      "\n",
      "Seq is: GCTCCAA\n",
      "0 GCTCCAA motifs detected in file\n",
      "\n",
      "Seq is: CTGGACA\n",
      "0 CTGGACA motifs detected in file\n",
      "\n",
      "Seq is: ATCCCAC\n",
      "0 ATCCCAC motifs detected in file\n",
      "\n",
      "Complement found from 3293 to 3300\n",
      "Complement found from 6099 to 6106\n",
      "Complement found from 10221 to 10228\n",
      "Complement found from 11117 to 11124\n",
      "Complement found from 13105 to 13112\n",
      "Complement found from 14107 to 14114\n",
      "Seq is: TCTTCTT\n",
      "6 TCTTCTT motifs detected in file\n",
      "\n",
      "Complement found from 4415 to 4422\n",
      "Seq is: AATAAGC\n",
      "1 AATAAGC motifs detected in file\n",
      "\n",
      "Complement found from 16561 to 16568\n",
      "Seq is: TCACGAT\n",
      "1 TCACGAT motifs detected in file\n",
      "\n",
      "Seq is: ATCCGCG\n",
      "0 ATCCGCG motifs detected in file\n",
      "\n",
      "Complement found from 424 to 431\n",
      "Complement found from 6280 to 6287\n",
      "Seq is: AACAGTC\n",
      "2 AACAGTC motifs detected in file\n",
      "\n",
      "Complement found from 7365 to 7372\n",
      "Complement found from 7498 to 7505\n",
      "Complement found from 14053 to 14060\n",
      "Complement found from 14375 to 14382\n",
      "Complement found from 15663 to 15670\n",
      "Seq is: CCTCCAT\n",
      "5 CCTCCAT motifs detected in file\n",
      "\n",
      "Complement found from 5469 to 5476\n",
      "Complement found from 7617 to 7624\n",
      "Seq is: CGCTACT\n",
      "2 CGCTACT motifs detected in file\n",
      "\n",
      "Complement found from 10438 to 10445\n",
      "Complement found from 11727 to 11734\n",
      "Complement found from 13734 to 13741\n",
      "Seq is: CTCATTA\n",
      "3 CTCATTA motifs detected in file\n",
      "\n",
      "Seq is: TCTATCC\n",
      "0 TCTATCC motifs detected in file\n",
      "\n",
      "Seq is: CGAGTCC\n",
      "0 CGAGTCC motifs detected in file\n",
      "\n",
      "Complement found from 2750 to 2757\n",
      "Complement found from 13530 to 13537\n",
      "Seq is: GCAAACA\n",
      "2 GCAAACA motifs detected in file\n",
      "\n",
      "Complement found from 468 to 475\n",
      "Complement found from 8942 to 8949\n",
      "Complement found from 9334 to 9341\n",
      "Complement found from 10654 to 10661\n",
      "Complement found from 11955 to 11962\n",
      "Complement found from 12716 to 12723\n",
      "Complement found from 13346 to 13353\n",
      "Seq is: CATACTA\n",
      "7 CATACTA motifs detected in file\n",
      "\n",
      "Complement found from 5929 to 5936\n",
      "Complement found from 10293 to 10300\n",
      "Complement found from 15183 to 15190\n",
      "Seq is: TACAAAC\n",
      "3 TACAAAC motifs detected in file\n",
      "\n",
      "Complement found from 1887 to 1894\n",
      "Seq is: GCCAAAG\n",
      "1 GCCAAAG motifs detected in file\n",
      "\n",
      "Complement found from 3438 to 3445\n",
      "Complement found from 5103 to 5110\n",
      "Complement found from 5117 to 5124\n",
      "Complement found from 5150 to 5157\n",
      "Complement found from 11287 to 11294\n",
      "Complement found from 14210 to 14217\n",
      "Seq is: CTACTAC\n",
      "6 CTACTAC motifs detected in file\n",
      "\n",
      "Seq is: ACCAGAA\n",
      "0 ACCAGAA motifs detected in file\n",
      "\n",
      "Complement found from 10542 to 10549\n",
      "Seq is: ACCTCAT\n",
      "1 ACCTCAT motifs detected in file\n",
      "\n",
      "Complement found from 2115 to 2122\n",
      "Seq is: CTTTGGA\n",
      "1 CTTTGGA motifs detected in file\n",
      "\n",
      "Complement found from 9594 to 9601\n",
      "Complement found from 14849 to 14856\n",
      "Seq is: CACTCCT\n",
      "2 CACTCCT motifs detected in file\n",
      "\n",
      "Seq is: TTCCATA\n",
      "0 TTCCATA motifs detected in file\n",
      "\n",
      "Complement found from 8717 to 8724\n",
      "Seq is: AGGACGA\n",
      "1 AGGACGA motifs detected in file\n",
      "\n",
      "Seq is: TTACGAT\n",
      "0 TTACGAT motifs detected in file\n",
      "\n",
      "Complement found from 9771 to 9778\n",
      "Seq is: CCGACGG\n",
      "1 CCGACGG motifs detected in file\n",
      "\n",
      "Complement found from 5463 to 5470\n",
      "Complement found from 8910 to 8917\n",
      "Complement found from 9508 to 9515\n",
      "Complement found from 11990 to 11997\n",
      "Complement found from 12394 to 12401\n",
      "Complement found from 14324 to 14331\n",
      "Seq is: TTACCAC\n",
      "6 TTACCAC motifs detected in file\n",
      "\n",
      "Complement found from 7860 to 7867\n",
      "Seq is: TCCCTCC\n",
      "1 TCCCTCC motifs detected in file\n",
      "\n",
      "Seq is: AACTTCA\n",
      "0 AACTTCA motifs detected in file\n",
      "\n",
      "Complement found from 38 to 45\n",
      "Complement found from 3789 to 3796\n",
      "Seq is: CTCTCCA\n",
      "2 CTCTCCA motifs detected in file\n",
      "\n",
      "Complement found from 2498 to 2505\n",
      "Complement found from 2785 to 2792\n",
      "Complement found from 4895 to 4902\n",
      "Complement found from 6437 to 6444\n",
      "Complement found from 10463 to 10470\n",
      "Seq is: TACCAAA\n",
      "5 TACCAAA motifs detected in file\n",
      "\n",
      "Complement found from 4519 to 4526\n",
      "Complement found from 4746 to 4753\n",
      "Complement found from 5452 to 5459\n",
      "Complement found from 6981 to 6988\n",
      "Complement found from 11238 to 11245\n",
      "Complement found from 12698 to 12705\n",
      "Complement found from 13112 to 13119\n",
      "Complement found from 14117 to 14124\n",
      "Seq is: ACTCATC\n",
      "8 ACTCATC motifs detected in file\n",
      "\n",
      "Complement found from 6574 to 6581\n",
      "Complement found from 13412 to 13419\n",
      "Seq is: AGGAGGA\n",
      "2 AGGAGGA motifs detected in file\n",
      "\n",
      "Complement found from 5828 to 5835\n",
      "Seq is: TAAAAAG\n",
      "1 TAAAAAG motifs detected in file\n",
      "\n",
      "Complement found from 5506 to 5513\n",
      "Seq is: CTTATAG\n",
      "1 CTTATAG motifs detected in file\n",
      "\n",
      "Complement found from 2397 to 2404\n",
      "Complement found from 2917 to 2924\n",
      "Seq is: AACAAGT\n",
      "2 AACAAGT motifs detected in file\n",
      "\n",
      "Complement found from 239 to 246\n",
      "Complement found from 6400 to 6407\n",
      "Complement found from 10779 to 10786\n",
      "Complement found from 14027 to 14034\n",
      "Seq is: AACAATT\n",
      "4 AACAATT motifs detected in file\n",
      "\n",
      "Complement found from 2426 to 2433\n",
      "Complement found from 6595 to 6602\n",
      "Complement found from 8344 to 8351\n",
      "Complement found from 9351 to 9358\n",
      "Complement found from 10713 to 10720\n",
      "Complement found from 12826 to 12833\n",
      "Seq is: CCAACAC\n",
      "6 CCAACAC motifs detected in file\n",
      "\n",
      "Complement found from 3438 to 3445\n",
      "Complement found from 5103 to 5110\n",
      "Complement found from 5117 to 5124\n",
      "Complement found from 5150 to 5157\n",
      "Complement found from 11287 to 11294\n",
      "Complement found from 14210 to 14217\n",
      "Seq is: CTACTAC\n",
      "6 CTACTAC motifs detected in file\n",
      "\n",
      "Complement found from 6023 to 6030\n",
      "Seq is: CTGGGCC\n",
      "1 CTGGGCC motifs detected in file\n",
      "\n",
      "Seq is: TCGTGTA\n",
      "0 TCGTGTA motifs detected in file\n",
      "\n",
      "Seq is: TTTATCA\n",
      "0 TTTATCA motifs detected in file\n",
      "\n",
      "Complement found from 3371 to 3378\n",
      "Seq is: TACCGAA\n",
      "1 TACCGAA motifs detected in file\n",
      "\n",
      "Complement found from 8905 to 8912\n",
      "Seq is: ACTTCTT\n",
      "1 ACTTCTT motifs detected in file\n",
      "\n",
      "Complement found from 14246 to 14253\n",
      "Seq is: CCGCACC\n",
      "1 CCGCACC motifs detected in file\n",
      "\n",
      "Complement found from 12240 to 12247\n",
      "Seq is: CATGTCT\n",
      "1 CATGTCT motifs detected in file\n",
      "\n",
      "Seq is: ACACGTC\n",
      "0 ACACGTC motifs detected in file\n",
      "\n",
      "Complement found from 10164 to 10171\n",
      "Complement found from 12186 to 12193\n",
      "Complement found from 13962 to 13969\n",
      "Seq is: CTTACGA\n",
      "3 CTTACGA motifs detected in file\n",
      "\n",
      "Complement found from 266 to 273\n",
      "Complement found from 3867 to 3874\n",
      "Seq is: TCCACAC\n",
      "2 TCCACAC motifs detected in file\n",
      "\n",
      "Complement found from 548 to 555\n",
      "Complement found from 2392 to 2399\n",
      "Complement found from 8979 to 8986\n",
      "Complement found from 13292 to 13299\n",
      "Seq is: CAACCAA\n",
      "4 CAACCAA motifs detected in file\n",
      "\n",
      "Complement found from 7920 to 7927\n",
      "Complement found from 9781 to 9788\n",
      "Seq is: CTACGGC\n",
      "2 CTACGGC motifs detected in file\n",
      "\n",
      "Complement found from 9011 to 9018\n",
      "Complement found from 11081 to 11088\n",
      "Complement found from 13658 to 13665\n",
      "Seq is: TAACATT\n",
      "3 TAACATT motifs detected in file\n",
      "\n",
      "Seq is: GATAATA\n",
      "0 GATAATA motifs detected in file\n",
      "\n",
      "Complement found from 7757 to 7764\n",
      "Complement found from 14433 to 14440\n",
      "Seq is: CTCAGGA\n",
      "2 CTCAGGA motifs detected in file\n",
      "\n",
      "Complement found from 5612 to 5619\n",
      "Complement found from 5711 to 5718\n",
      "Complement found from 16237 to 16244\n",
      "Seq is: TCAACTG\n",
      "3 TCAACTG motifs detected in file\n",
      "\n",
      "Seq is: AACTTCA\n",
      "0 AACTTCA motifs detected in file\n",
      "\n",
      "Complement found from 3291 to 3298\n",
      "Complement found from 4690 to 4697\n",
      "Complement found from 6448 to 6455\n",
      "Complement found from 15024 to 15031\n",
      "Seq is: CCTCTTC\n",
      "4 CCTCTTC motifs detected in file\n",
      "\n",
      "Seq is: CGATGCT\n",
      "0 CGATGCT motifs detected in file\n",
      "\n",
      "Complement found from 2456 to 2463\n",
      "Complement found from 4604 to 4611\n",
      "Complement found from 6691 to 6698\n",
      "Complement found from 11031 to 11038\n",
      "Complement found from 12417 to 12424\n",
      "Complement found from 13230 to 13237\n",
      "Complement found from 14503 to 14510\n",
      "Seq is: AAAAAAA\n",
      "7 AAAAAAA motifs detected in file\n",
      "\n",
      "Seq is: CTACAGT\n",
      "0 CTACAGT motifs detected in file\n",
      "\n",
      "Complement found from 2352 to 2359\n",
      "Complement found from 10815 to 10822\n",
      "Complement found from 11520 to 11527\n",
      "Complement found from 12056 to 12063\n",
      "Complement found from 14395 to 14402\n",
      "Seq is: AAAACAC\n",
      "5 AAAACAC motifs detected in file\n",
      "\n",
      "Complement found from 8107 to 8114\n",
      "Seq is: ATTCCCG\n",
      "1 ATTCCCG motifs detected in file\n",
      "\n",
      "Complement found from 6454 to 6461\n",
      "Complement found from 7773 to 7780\n",
      "Seq is: CGTCTGA\n",
      "2 CGTCTGA motifs detected in file\n",
      "\n",
      "Seq is: CGACGCT\n",
      "0 CGACGCT motifs detected in file\n",
      "\n",
      "Seq is: TGTCTCA\n",
      "0 TGTCTCA motifs detected in file\n",
      "\n",
      "Complement found from 1084 to 1091\n",
      "Complement found from 7033 to 7040\n",
      "Complement found from 11020 to 11027\n",
      "Complement found from 11870 to 11877\n",
      "Complement found from 13881 to 13888\n",
      "Seq is: CCACTAT\n",
      "5 CCACTAT motifs detected in file\n",
      "\n",
      "Seq is: CGCGGAT\n",
      "0 CGCGGAT motifs detected in file\n",
      "\n",
      "Seq is: TCGTATC\n",
      "0 TCGTATC motifs detected in file\n",
      "\n",
      "Complement found from 3720 to 3727\n",
      "Complement found from 4023 to 4030\n",
      "Complement found from 4233 to 4240\n",
      "Seq is: ACAATCT\n",
      "3 ACAATCT motifs detected in file\n",
      "\n",
      "Complement found from 1991 to 1998\n",
      "Complement found from 2788 to 2795\n",
      "Complement found from 4254 to 4261\n",
      "Complement found from 7121 to 7128\n",
      "Complement found from 16281 to 16288\n",
      "Seq is: CAAACCT\n",
      "5 CAAACCT motifs detected in file\n",
      "\n",
      "Complement found from 730 to 737\n",
      "Complement found from 5811 to 5818\n",
      "Complement found from 8651 to 8658\n",
      "Complement found from 9634 to 9641\n",
      "Complement found from 14877 to 14884\n",
      "Complement found from 15375 to 15382\n",
      "Complement found from 15396 to 15403\n",
      "Seq is: AATCACC\n",
      "7 AATCACC motifs detected in file\n",
      "\n",
      "Complement found from 1215 to 1222\n",
      "Complement found from 15390 to 15397\n",
      "Seq is: CGATAAA\n",
      "2 CGATAAA motifs detected in file\n",
      "\n",
      "Seq is: AGACGAT\n",
      "0 AGACGAT motifs detected in file\n",
      "\n",
      "Complement found from 568 to 575\n",
      "Complement found from 801 to 808\n",
      "Complement found from 5894 to 5901\n",
      "Complement found from 7397 to 7404\n",
      "Complement found from 8557 to 8564\n",
      "Complement found from 11867 to 11874\n",
      "Seq is: CCCCCAC\n",
      "6 CCCCCAC motifs detected in file\n",
      "\n",
      "Complement found from 3962 to 3969\n",
      "Complement found from 10063 to 10070\n",
      "Seq is: CTTCGCC\n",
      "2 CTTCGCC motifs detected in file\n",
      "\n",
      "Complement found from 4745 to 4752\n",
      "Complement found from 8971 to 8978\n",
      "Complement found from 9031 to 9038\n",
      "Complement found from 11237 to 11244\n",
      "Complement found from 12697 to 12704\n",
      "Complement found from 13111 to 13118\n",
      "Seq is: TACTCAT\n",
      "6 TACTCAT motifs detected in file\n",
      "\n",
      "Complement found from 3551 to 3558\n",
      "Seq is: TCTTCTA\n",
      "1 TCTTCTA motifs detected in file\n",
      "\n",
      "Complement found from 8717 to 8724\n",
      "Seq is: AGGACGA\n",
      "1 AGGACGA motifs detected in file\n",
      "\n",
      "Seq is: TGTTGAT\n",
      "0 TGTTGAT motifs detected in file\n",
      "\n",
      "Complement found from 11025 to 11032\n",
      "Complement found from 16560 to 16567\n",
      "Seq is: ATCACGA\n",
      "2 ATCACGA motifs detected in file\n",
      "\n",
      "Complement found from 977 to 984\n",
      "Complement found from 4190 to 4197\n",
      "Complement found from 12012 to 12019\n",
      "Complement found from 14400 to 14407\n",
      "Complement found from 14909 to 14916\n",
      "Complement found from 16065 to 16072\n",
      "Seq is: ACTCACC\n",
      "6 ACTCACC motifs detected in file\n",
      "\n",
      "Complement found from 5298 to 5305\n",
      "Complement found from 10325 to 10332\n",
      "Complement found from 10865 to 10872\n",
      "Complement found from 12601 to 12608\n",
      "Seq is: TCATCCC\n",
      "4 TCATCCC motifs detected in file\n",
      "\n",
      "Seq is: GACTCTG\n",
      "0 GACTCTG motifs detected in file\n",
      "\n",
      "Complement found from 1081 to 1088\n",
      "Complement found from 5599 to 5606\n",
      "Complement found from 12966 to 12973\n",
      "Complement found from 13675 to 13682\n",
      "Complement found from 14387 to 14394\n",
      "Complement found from 14618 to 14625\n",
      "Seq is: ACCCCAC\n",
      "6 ACCCCAC motifs detected in file\n",
      "\n",
      "Seq is: CCCGGAA\n",
      "0 CCCGGAA motifs detected in file\n",
      "\n",
      "Seq is: GATACCG\n",
      "0 GATACCG motifs detected in file\n",
      "\n",
      "Complement found from 5380 to 5387\n",
      "Complement found from 13031 to 13038\n",
      "Seq is: ACTCCCC\n",
      "2 ACTCCCC motifs detected in file\n",
      "\n",
      "Seq is: TCAGTCC\n",
      "0 TCAGTCC motifs detected in file\n",
      "\n",
      "Complement found from 1237 to 1244\n",
      "Complement found from 3625 to 3632\n",
      "Complement found from 5364 to 5371\n",
      "Complement found from 8620 to 8627\n",
      "Complement found from 14050 to 14057\n",
      "Complement found from 15330 to 15337\n",
      "Seq is: CCACCTC\n",
      "6 CCACCTC motifs detected in file\n",
      "\n",
      "Complement found from 4972 to 4979\n",
      "Seq is: TGGATTA\n",
      "1 TGGATTA motifs detected in file\n",
      "\n",
      "Complement found from 14555 to 14562\n",
      "Seq is: ACCCGAC\n",
      "1 ACCCGAC motifs detected in file\n",
      "\n",
      "Seq is: TAAACGG\n",
      "0 TAAACGG motifs detected in file\n",
      "\n",
      "Seq is: ACGACGA\n",
      "0 ACGACGA motifs detected in file\n",
      "\n",
      "Seq is: CGATCCT\n",
      "0 CGATCCT motifs detected in file\n",
      "\n",
      "Complement found from 9288 to 9295\n",
      "Seq is: CCTCCGG\n",
      "1 CCTCCGG motifs detected in file\n",
      "\n",
      "Complement found from 10340 to 10347\n",
      "Complement found from 11152 to 11159\n",
      "Complement found from 14058 to 14065\n",
      "Seq is: ATCATCA\n",
      "3 ATCATCA motifs detected in file\n",
      "\n",
      "Complement found from 1082 to 1089\n",
      "Complement found from 5600 to 5607\n",
      "Complement found from 5895 to 5902\n",
      "Complement found from 11868 to 11875\n",
      "Complement found from 12967 to 12974\n",
      "Complement found from 13879 to 13886\n",
      "Complement found from 14388 to 14395\n",
      "Seq is: CCCCACT\n",
      "7 CCCCACT motifs detected in file\n",
      "\n",
      "Complement found from 542 to 549\n",
      "Complement found from 3887 to 3894\n",
      "Seq is: CCGAACC\n",
      "2 CCGAACC motifs detected in file\n",
      "\n",
      "Complement found from 3249 to 3256\n",
      "Complement found from 3346 to 3353\n",
      "Complement found from 11705 to 11712\n",
      "Seq is: TAATCGC\n",
      "3 TAATCGC motifs detected in file\n",
      "\n",
      "Complement found from 2479 to 2486\n",
      "Complement found from 4998 to 5005\n",
      "Complement found from 8234 to 8241\n",
      "Seq is: AAATCTT\n",
      "3 AAATCTT motifs detected in file\n",
      "\n",
      "Complement found from 6170 to 6177\n",
      "Seq is: CCCGATA\n",
      "1 CCCGATA motifs detected in file\n",
      "\n",
      "Complement found from 3300 to 3307\n",
      "Complement found from 4037 to 4044\n",
      "Complement found from 6194 to 6201\n",
      "Complement found from 10894 to 10901\n",
      "Complement found from 12028 to 12035\n",
      "Complement found from 12247 to 12254\n",
      "Complement found from 13743 to 13750\n",
      "Complement found from 13772 to 13779\n",
      "Complement found from 15677 to 15684\n",
      "Seq is: AACAACA\n",
      "9 AACAACA motifs detected in file\n",
      "\n",
      "Complement found from 4813 to 4820\n",
      "Seq is: CCCAGAG\n",
      "1 CCCAGAG motifs detected in file\n",
      "\n",
      "Complement found from 2886 to 2893\n",
      "Complement found from 3645 to 3652\n",
      "Complement found from 5121 to 5128\n",
      "Complement found from 13420 to 13427\n",
      "Complement found from 15872 to 15879\n",
      "Seq is: TACTCAA\n",
      "5 TACTCAA motifs detected in file\n",
      "\n",
      "Complement found from 2253 to 2260\n",
      "Complement found from 10980 to 10987\n",
      "Complement found from 11493 to 11500\n",
      "Complement found from 13490 to 13497\n",
      "Complement found from 15267 to 15274\n",
      "Seq is: CCTCACA\n",
      "5 CCTCACA motifs detected in file\n",
      "\n",
      "Complement found from 148 to 155\n",
      "Complement found from 2946 to 2953\n",
      "Complement found from 3610 to 3617\n",
      "Complement found from 9456 to 9463\n",
      "Complement found from 11204 to 11211\n",
      "Complement found from 15335 to 15342\n",
      "Complement found from 15566 to 15573\n",
      "Seq is: TCCTATT\n",
      "7 TCCTATT motifs detected in file\n",
      "\n",
      "Seq is: TAGTACG\n",
      "0 TAGTACG motifs detected in file\n",
      "\n",
      "Complement found from 4142 to 4149\n",
      "Seq is: ATTCCGC\n",
      "1 ATTCCGC motifs detected in file\n",
      "\n",
      "Complement found from 3322 to 3329\n",
      "Complement found from 5115 to 5122\n",
      "Complement found from 6228 to 6235\n",
      "Complement found from 6480 to 6487\n",
      "Complement found from 11936 to 11943\n",
      "Complement found from 16505 to 16512\n",
      "Seq is: TCCTACT\n",
      "6 TCCTACT motifs detected in file\n",
      "\n",
      "Complement found from 11146 to 11153\n",
      "Seq is: TTGGCTA\n",
      "1 TTGGCTA motifs detected in file\n",
      "\n",
      "Seq is: TAGCGGC\n",
      "0 TAGCGGC motifs detected in file\n",
      "\n",
      "Seq is: TATGCCA\n",
      "0 TATGCCA motifs detected in file\n",
      "\n",
      "Complement found from 1036 to 1043\n",
      "Complement found from 13534 to 13541\n",
      "Seq is: ACATATC\n",
      "2 ACATATC motifs detected in file\n",
      "\n",
      "Seq is: CTAACGA\n",
      "0 CTAACGA motifs detected in file\n",
      "\n",
      "Seq is: ACTTACC\n",
      "0 ACTTACC motifs detected in file\n",
      "\n",
      "Seq is: GACGACA\n",
      "0 GACGACA motifs detected in file\n",
      "\n",
      "Complement found from 12857 to 12864\n",
      "Seq is: CAACCGT\n",
      "1 CAACCGT motifs detected in file\n",
      "\n",
      "Complement found from 1911 to 1918\n",
      "Complement found from 2714 to 2721\n",
      "Complement found from 7845 to 7852\n",
      "Seq is: AGACGAG\n",
      "3 AGACGAG motifs detected in file\n",
      "\n",
      "Complement found from 6186 to 6193\n",
      "Complement found from 13755 to 13762\n",
      "Seq is: CCCGCAT\n",
      "2 CCCGCAT motifs detected in file\n",
      "\n",
      "Seq is: AGTAGTT\n",
      "0 AGTAGTT motifs detected in file\n",
      "\n",
      "Complement found from 12582 to 12589\n",
      "Complement found from 13417 to 13424\n",
      "Seq is: GACTACT\n",
      "2 GACTACT motifs detected in file\n",
      "\n",
      "Seq is: CGTTCTT\n",
      "0 CGTTCTT motifs detected in file\n",
      "\n",
      "Complement found from 1252 to 1259\n",
      "Complement found from 7572 to 7579\n",
      "Complement found from 10184 to 10191\n",
      "Complement found from 15048 to 15055\n",
      "Seq is: CCTATAT\n",
      "4 CCTATAT motifs detected in file\n",
      "\n",
      "Seq is: TAAGGAT\n",
      "0 TAAGGAT motifs detected in file\n",
      "\n",
      "Complement found from 5753 to 5760\n",
      "Seq is: GCGGGAG\n",
      "1 GCGGGAG motifs detected in file\n",
      "\n",
      "Seq is: AGTCGGC\n",
      "0 AGTCGGC motifs detected in file\n",
      "\n",
      "Seq is: TACTTGT\n",
      "0 TACTTGT motifs detected in file\n",
      "\n",
      "Complement found from 1710 to 1717\n",
      "Complement found from 6037 to 6044\n",
      "Complement found from 10919 to 10926\n",
      "Complement found from 13379 to 13386\n",
      "Seq is: CAACCTT\n",
      "4 CAACCTT motifs detected in file\n",
      "\n",
      "Complement found from 14817 to 14824\n",
      "Seq is: ATCCAAC\n",
      "1 ATCCAAC motifs detected in file\n",
      "\n",
      "Complement found from 11706 to 11713\n",
      "Complement found from 14940 to 14947\n",
      "Seq is: AATCGCC\n",
      "2 AATCGCC motifs detected in file\n",
      "\n",
      "Seq is: ATTGATT\n",
      "0 ATTGATT motifs detected in file\n",
      "\n",
      "Complement found from 5707 to 5714\n",
      "Complement found from 6467 to 6474\n",
      "Complement found from 8649 to 8656\n",
      "Complement found from 8670 to 8677\n",
      "Complement found from 11101 to 11108\n",
      "Complement found from 14136 to 14143\n",
      "Complement found from 14216 to 14223\n",
      "Seq is: CTAATCA\n",
      "7 CTAATCA motifs detected in file\n",
      "\n",
      "Seq is: TACCATT\n",
      "0 TACCATT motifs detected in file\n",
      "\n",
      "Complement found from 4672 to 4679\n",
      "Seq is: AATCCTT\n",
      "1 AATCCTT motifs detected in file\n",
      "\n",
      "Complement found from 5120 to 5127\n",
      "Complement found from 8970 to 8977\n",
      "Complement found from 9030 to 9037\n",
      "Complement found from 11236 to 11243\n",
      "Complement found from 11290 to 11297\n",
      "Complement found from 12696 to 12703\n",
      "Complement found from 13419 to 13426\n",
      "Complement found from 14907 to 14914\n",
      "Complement found from 15069 to 15076\n",
      "Complement found from 15246 to 15253\n",
      "Seq is: CTACTCA\n",
      "10 CTACTCA motifs detected in file\n",
      "\n",
      "Complement found from 8820 to 8827\n",
      "Seq is: TCTATAA\n",
      "1 TCTATAA motifs detected in file\n",
      "\n",
      "Complement found from 8905 to 8912\n",
      "Seq is: ACTTCTT\n",
      "1 ACTTCTT motifs detected in file\n",
      "\n",
      "Complement found from 7928 to 7935\n",
      "Complement found from 8647 to 8654\n",
      "Complement found from 8668 to 8675\n",
      "Seq is: GACTAAT\n",
      "3 GACTAAT motifs detected in file\n",
      "\n",
      "Complement found from 5929 to 5936\n",
      "Complement found from 10293 to 10300\n",
      "Complement found from 15183 to 15190\n",
      "Seq is: TACAAAC\n",
      "3 TACAAAC motifs detected in file\n",
      "\n",
      "Seq is: CCAGACT\n",
      "0 CCAGACT motifs detected in file\n",
      "\n",
      "Complement found from 8627 to 8634\n",
      "Complement found from 11924 to 11931\n",
      "Complement found from 12689 to 12696\n",
      "Complement found from 15154 to 15161\n",
      "Seq is: CAAATAT\n",
      "4 CAAATAT motifs detected in file\n",
      "\n",
      "Seq is: ATAGTGT\n",
      "0 ATAGTGT motifs detected in file\n",
      "\n",
      "Seq is: CACTCTT\n",
      "0 CACTCTT motifs detected in file\n",
      "\n",
      "Complement found from 11374 to 11381\n",
      "Seq is: AAGATAC\n",
      "1 AAGATAC motifs detected in file\n",
      "\n",
      "Seq is: TACCTGG\n",
      "0 TACCTGG motifs detected in file\n",
      "\n",
      "Complement found from 15208 to 15215\n",
      "Seq is: TACATTG\n",
      "1 TACATTG motifs detected in file\n",
      "\n",
      "Seq is: CTTGTTA\n",
      "0 CTTGTTA motifs detected in file\n",
      "\n",
      "Seq is: CGATGTC\n",
      "0 CGATGTC motifs detected in file\n",
      "\n",
      "Complement found from 3147 to 3154\n",
      "Complement found from 4094 to 4101\n",
      "Complement found from 6481 to 6488\n",
      "Complement found from 11937 to 11944\n",
      "Complement found from 16506 to 16513\n",
      "Seq is: CCTACTT\n",
      "5 CCTACTT motifs detected in file\n",
      "\n",
      "Complement found from 10574 to 10581\n",
      "Complement found from 11485 to 11492\n",
      "Seq is: ATAATAC\n",
      "2 ATAATAC motifs detected in file\n",
      "\n",
      "Complement found from 3468 to 3475\n",
      "Complement found from 4691 to 4698\n",
      "Complement found from 11622 to 11629\n",
      "Seq is: CTCTTCA\n",
      "3 CTCTTCA motifs detected in file\n",
      "\n",
      "Seq is: TCAGATC\n",
      "0 TCAGATC motifs detected in file\n",
      "\n",
      "Complement found from 4178 to 4185\n",
      "Complement found from 10060 to 10067\n",
      "Complement found from 14838 to 14845\n",
      "Seq is: AAACTTC\n",
      "3 AAACTTC motifs detected in file\n",
      "\n",
      "Seq is: GAATCCC\n",
      "0 GAATCCC motifs detected in file\n",
      "\n",
      "Seq is: TCTCGAC\n",
      "0 TCTCGAC motifs detected in file\n",
      "\n",
      "Complement found from 2395 to 2402\n",
      "Complement found from 8660 to 8667\n",
      "Complement found from 10777 to 10784\n",
      "Complement found from 11330 to 11337\n",
      "Complement found from 13858 to 13865\n",
      "Complement found from 14185 to 14192\n",
      "Complement found from 16277 to 16284\n",
      "Seq is: CCAACAA\n",
      "7 CCAACAA motifs detected in file\n",
      "\n",
      "Complement found from 1624 to 1631\n",
      "Complement found from 2424 to 2431\n",
      "Complement found from 2832 to 2839\n",
      "Complement found from 3578 to 3585\n",
      "Complement found from 8434 to 8441\n",
      "Complement found from 8658 to 8665\n",
      "Complement found from 8811 to 8818\n",
      "Seq is: ACCCAAC\n",
      "7 ACCCAAC motifs detected in file\n",
      "\n",
      "Seq is: ACCGAGT\n",
      "0 ACCGAGT motifs detected in file\n",
      "\n",
      "Complement found from 11770 to 11777\n",
      "Seq is: CACAGTC\n",
      "1 CACAGTC motifs detected in file\n",
      "\n",
      "Seq is: AAGCTCT\n",
      "0 AAGCTCT motifs detected in file\n",
      "\n",
      "Complement found from 9830 to 9837\n",
      "Complement found from 15307 to 15314\n",
      "Seq is: ATTATTG\n",
      "2 ATTATTG motifs detected in file\n",
      "\n",
      "Complement found from 363 to 370\n",
      "Complement found from 6696 to 6703\n",
      "Complement found from 7359 to 7366\n",
      "Complement found from 7419 to 7426\n",
      "Seq is: AAGAACC\n",
      "4 AAGAACC motifs detected in file\n",
      "\n",
      "Seq is: AGATCCG\n",
      "0 AGATCCG motifs detected in file\n",
      "\n",
      "Seq is: TGTACTT\n",
      "0 TGTACTT motifs detected in file\n",
      "\n",
      "Complement found from 237 to 244\n",
      "Complement found from 280 to 287\n",
      "Complement found from 8504 to 8511\n",
      "Complement found from 11569 to 11576\n",
      "Seq is: ATAACAA\n",
      "4 ATAACAA motifs detected in file\n",
      "\n",
      "Complement found from 4831 to 4838\n",
      "Complement found from 8929 to 8936\n",
      "Complement found from 10159 to 10166\n",
      "Complement found from 13023 to 13030\n",
      "Complement found from 15534 to 15541\n",
      "Complement found from 16256 to 16263\n",
      "Seq is: CACCCCT\n",
      "6 CACCCCT motifs detected in file\n",
      "\n",
      "Seq is: TCTCTGA\n",
      "0 TCTCTGA motifs detected in file\n",
      "\n",
      "Complement found from 150 to 157\n",
      "Complement found from 5960 to 5967\n",
      "Seq is: CTATTAT\n",
      "2 CTATTAT motifs detected in file\n",
      "\n",
      "Complement found from 292 to 299\n",
      "Complement found from 13122 to 13129\n",
      "Complement found from 15403 to 15410\n",
      "Seq is: TTCCACC\n",
      "3 TTCCACC motifs detected in file\n",
      "\n",
      "Seq is: TACGCTG\n",
      "0 TACGCTG motifs detected in file\n",
      "\n",
      "Complement found from 14019 to 14026\n",
      "Seq is: TTACCTA\n",
      "1 TTACCTA motifs detected in file\n",
      "\n",
      "Seq is: AAATGTA\n",
      "0 AAATGTA motifs detected in file\n",
      "\n",
      "Complement found from 2484 to 2491\n",
      "Complement found from 8402 to 8409\n",
      "Complement found from 10274 to 10281\n",
      "Complement found from 11863 to 11870\n",
      "Seq is: TTACCCC\n",
      "4 TTACCCC motifs detected in file\n",
      "\n",
      "Complement found from 5007 to 5014\n",
      "Complement found from 8409 to 8416\n",
      "Complement found from 11618 to 11625\n",
      "Complement found from 13907 to 13914\n",
      "Complement found from 14345 to 14352\n",
      "Seq is: CATACTC\n",
      "5 CATACTC motifs detected in file\n",
      "\n",
      "Complement found from 2456 to 2463\n",
      "Complement found from 4604 to 4611\n",
      "Complement found from 6691 to 6698\n",
      "Complement found from 11031 to 11038\n",
      "Complement found from 12417 to 12424\n",
      "Complement found from 13230 to 13237\n",
      "Complement found from 14503 to 14510\n",
      "Seq is: AAAAAAA\n",
      "7 AAAAAAA motifs detected in file\n",
      "\n",
      "Complement found from 8562 to 8569\n",
      "Complement found from 13776 to 13783\n",
      "Complement found from 13941 to 13948\n",
      "Complement found from 15826 to 15833\n",
      "Seq is: ACAATCC\n",
      "4 ACAATCC motifs detected in file\n",
      "\n",
      "Seq is: CAATTGC\n",
      "0 CAATTGC motifs detected in file\n",
      "\n",
      "Complement found from 6311 to 6318\n",
      "Complement found from 15261 to 15268\n",
      "Seq is: TCCCACC\n",
      "2 TCCCACC motifs detected in file\n",
      "\n",
      "Complement found from 3359 to 3366\n",
      "Complement found from 5113 to 5120\n",
      "Complement found from 7962 to 7969\n",
      "Complement found from 14892 to 14899\n",
      "Seq is: ATTCCTA\n",
      "4 ATTCCTA motifs detected in file\n",
      "\n",
      "Complement found from 303 to 310\n",
      "Complement found from 3565 to 3572\n",
      "Complement found from 10946 to 10953\n",
      "Complement found from 16374 to 16381\n",
      "Seq is: CCCCCCT\n",
      "4 CCCCCCT motifs detected in file\n",
      "\n",
      "Complement found from 4471 to 4478\n",
      "Complement found from 15653 to 15660\n",
      "Seq is: TAATCCC\n",
      "2 TAATCCC motifs detected in file\n",
      "\n",
      "Seq is: TTCAGTC\n",
      "0 TTCAGTC motifs detected in file\n",
      "\n",
      "Complement found from 14366 to 14373\n",
      "Complement found from 16167 to 16174\n",
      "Seq is: CCAATCC\n",
      "2 CCAATCC motifs detected in file\n",
      "\n",
      "Complement found from 2186 to 2193\n",
      "Complement found from 7888 to 7895\n",
      "Complement found from 9212 to 9219\n",
      "Complement found from 14249 to 14256\n",
      "Complement found from 14364 to 14371\n",
      "Complement found from 14741 to 14748\n",
      "Seq is: CACCAAT\n",
      "6 CACCAAT motifs detected in file\n",
      "\n",
      "Complement found from 10340 to 10347\n",
      "Complement found from 11152 to 11159\n",
      "Complement found from 14058 to 14065\n",
      "Seq is: ATCATCA\n",
      "3 ATCATCA motifs detected in file\n",
      "\n",
      "Complement found from 7009 to 7016\n",
      "Seq is: CACGTAC\n",
      "1 CACGTAC motifs detected in file\n",
      "\n",
      "Complement found from 15650 to 15657\n",
      "Seq is: CAATAAT\n",
      "1 CAATAAT motifs detected in file\n",
      "\n",
      "Complement found from 13137 to 13144\n",
      "Complement found from 13668 to 13675\n",
      "Seq is: GAAAATA\n",
      "2 GAAAATA motifs detected in file\n",
      "\n",
      "Complement found from 1102 to 1109\n",
      "Complement found from 1230 to 1237\n",
      "Complement found from 3594 to 3601\n",
      "Complement found from 6542 to 6549\n",
      "Complement found from 8681 to 8688\n",
      "Seq is: AACCTCA\n",
      "5 AACCTCA motifs detected in file\n",
      "\n",
      "Seq is: ACGTGGT\n",
      "0 ACGTGGT motifs detected in file\n",
      "\n",
      "Seq is: TTTAAAA\n",
      "0 TTTAAAA motifs detected in file\n",
      "\n",
      "Seq is: ACCCCGG\n",
      "0 ACCCCGG motifs detected in file\n",
      "\n",
      "Seq is: ATTCTGG\n",
      "0 ATTCTGG motifs detected in file\n",
      "\n",
      "Complement found from 14019 to 14026\n",
      "Seq is: TTACCTA\n",
      "1 TTACCTA motifs detected in file\n",
      "\n",
      "Seq is: TCGACAA\n",
      "0 TCGACAA motifs detected in file\n",
      "\n",
      "Complement found from 2125 to 2132\n",
      "Seq is: TAGGAAA\n",
      "1 TAGGAAA motifs detected in file\n",
      "\n",
      "Complement found from 7509 to 7516\n",
      "Seq is: TTTTCAA\n",
      "1 TTTTCAA motifs detected in file\n",
      "\n",
      "Complement found from 14102 to 14109\n",
      "Complement found from 14349 to 14356\n",
      "Seq is: CTCTTTC\n",
      "2 CTCTTTC motifs detected in file\n",
      "\n",
      "Seq is: GGTACAA\n",
      "0 GGTACAA motifs detected in file\n",
      "\n",
      "Seq is: CAATCTG\n",
      "0 CAATCTG motifs detected in file\n",
      "\n",
      "Complement found from 4134 to 4141\n",
      "Complement found from 8278 to 8285\n",
      "Complement found from 8403 to 8410\n",
      "Complement found from 9529 to 9536\n",
      "Complement found from 11864 to 11871\n",
      "Complement found from 12429 to 12436\n",
      "Seq is: TACCCCC\n",
      "6 TACCCCC motifs detected in file\n",
      "\n",
      "Seq is: GTACTCA\n",
      "0 GTACTCA motifs detected in file\n",
      "\n",
      "Complement found from 6071 to 6078\n",
      "Seq is: ATCGTCA\n",
      "1 ATCGTCA motifs detected in file\n",
      "\n",
      "Complement found from 1136 to 1143\n",
      "Complement found from 3812 to 3819\n",
      "Complement found from 14737 to 14744\n",
      "Seq is: AGAACAC\n",
      "3 AGAACAC motifs detected in file\n",
      "\n",
      "Seq is: TCGAAAG\n",
      "0 TCGAAAG motifs detected in file\n",
      "\n",
      "Complement found from 5879 to 5886\n",
      "Complement found from 11655 to 11662\n",
      "Complement found from 12836 to 12843\n",
      "Complement found from 16317 to 16324\n",
      "Seq is: AGCCATT\n",
      "4 AGCCATT motifs detected in file\n",
      "\n",
      "Complement found from 14150 to 14157\n",
      "Seq is: TATTCCC\n",
      "1 TATTCCC motifs detected in file\n",
      "\n",
      "Complement found from 4241 to 4248\n",
      "Seq is: CAGCATT\n",
      "1 CAGCATT motifs detected in file\n",
      "\n",
      "Seq is: CGGAGAC\n",
      "0 CGGAGAC motifs detected in file\n",
      "\n",
      "Complement found from 424 to 431\n",
      "Complement found from 6280 to 6287\n",
      "Seq is: AACAGTC\n",
      "2 AACAGTC motifs detected in file\n",
      "\n",
      "Complement found from 1570 to 1577\n",
      "Seq is: TAAGTGT\n",
      "1 TAAGTGT motifs detected in file\n",
      "\n",
      "Complement found from 1015 to 1022\n",
      "Complement found from 10728 to 10735\n",
      "Complement found from 12580 to 12587\n",
      "Seq is: TAGACTA\n",
      "3 TAGACTA motifs detected in file\n",
      "\n",
      "Complement found from 5427 to 5434\n",
      "Complement found from 5595 to 5602\n",
      "Complement found from 13424 to 13431\n",
      "Complement found from 13971 to 13978\n",
      "Complement found from 16178 to 16185\n",
      "Seq is: CAAAACC\n",
      "5 CAAAACC motifs detected in file\n",
      "\n",
      "Complement found from 10824 to 10831\n",
      "Seq is: AATTTGA\n",
      "1 AATTTGA motifs detected in file\n",
      "\n",
      "Complement found from 10000 to 10007\n",
      "Seq is: TATAAAT\n",
      "1 TATAAAT motifs detected in file\n",
      "\n",
      "Complement found from 11913 to 11920\n",
      "Seq is: GTTCTCC\n",
      "1 GTTCTCC motifs detected in file\n",
      "\n",
      "Seq is: TTTTGGG\n",
      "0 TTTTGGG motifs detected in file\n",
      "\n",
      "Complement found from 4228 to 4235\n",
      "Complement found from 14630 to 14637\n",
      "Seq is: CCATTAC\n",
      "2 CCATTAC motifs detected in file\n",
      "\n",
      "Seq is: TACAGCC\n",
      "0 TACAGCC motifs detected in file\n",
      "\n",
      "Complement found from 4154 to 4161\n",
      "Complement found from 12913 to 12920\n",
      "Seq is: CCAACTC\n",
      "2 CCAACTC motifs detected in file\n",
      "\n",
      "Complement found from 7198 to 7205\n",
      "Complement found from 12078 to 12085\n",
      "Complement found from 12096 to 12103\n",
      "Seq is: CCTATCC\n",
      "3 CCTATCC motifs detected in file\n",
      "\n",
      "Complement found from 3290 to 3297\n",
      "Complement found from 4689 to 4696\n",
      "Complement found from 12130 to 12137\n",
      "Seq is: TCCTCTT\n",
      "3 TCCTCTT motifs detected in file\n",
      "\n",
      "Complement found from 954 to 961\n",
      "Complement found from 3583 to 3590\n",
      "Complement found from 3891 to 3898\n",
      "Complement found from 4316 to 4323\n",
      "Complement found from 6418 to 6425\n",
      "Complement found from 8270 to 8277\n",
      "Complement found from 8279 to 8286\n",
      "Complement found from 10934 to 10941\n",
      "Complement found from 11510 to 11517\n",
      "Complement found from 11671 to 11678\n",
      "Complement found from 13126 to 13133\n",
      "Complement found from 14768 to 14775\n",
      "Complement found from 15365 to 15372\n",
      "Complement found from 16182 to 16189\n",
      "Seq is: ACCCCCT\n",
      "14 ACCCCCT motifs detected in file\n",
      "\n",
      "Complement found from 13374 to 13381\n",
      "Complement found from 16170 to 16177\n",
      "Seq is: ATCCACA\n",
      "2 ATCCACA motifs detected in file\n",
      "\n",
      "Seq is: TACTTGT\n",
      "0 TACTTGT motifs detected in file\n",
      "\n",
      "Complement found from 463 to 470\n",
      "Complement found from 6309 to 6316\n",
      "Complement found from 11811 to 11818\n",
      "Seq is: ACTCCCA\n",
      "3 ACTCCCA motifs detected in file\n",
      "\n",
      "Seq is: CAAAAGA\n",
      "0 CAAAAGA motifs detected in file\n",
      "\n",
      "Complement found from 4515 to 4522\n",
      "Complement found from 12344 to 12351\n",
      "Seq is: GCACACT\n",
      "2 GCACACT motifs detected in file\n",
      "\n",
      "Complement found from 5379 to 5386\n",
      "Complement found from 6308 to 6315\n",
      "Complement found from 8012 to 8019\n",
      "Complement found from 11810 to 11817\n",
      "Complement found from 11974 to 11981\n",
      "Seq is: TACTCCC\n",
      "5 TACTCCC motifs detected in file\n",
      "\n",
      "Complement found from 15868 to 15875\n",
      "Seq is: AAAATAC\n",
      "1 AAAATAC motifs detected in file\n",
      "\n",
      "Complement found from 239 to 246\n",
      "Complement found from 6400 to 6407\n",
      "Complement found from 10779 to 10786\n",
      "Complement found from 14027 to 14034\n",
      "Seq is: AACAATT\n",
      "4 AACAATT motifs detected in file\n",
      "\n",
      "Complement found from 6549 to 6556\n",
      "Complement found from 7242 to 7249\n",
      "Complement found from 9419 to 9426\n",
      "Seq is: ACACCAC\n",
      "3 ACACCAC motifs detected in file\n",
      "\n",
      "Complement found from 25 to 32\n",
      "Complement found from 4188 to 4195\n",
      "Complement found from 13076 to 13083\n",
      "Complement found from 14115 to 14122\n",
      "Complement found from 14790 to 14797\n",
      "Seq is: CCACTCA\n",
      "5 CCACTCA motifs detected in file\n",
      "\n",
      "Seq is: CTCGGGG\n",
      "0 CTCGGGG motifs detected in file\n",
      "\n",
      "Complement found from 441 to 448\n",
      "Complement found from 1613 to 1620\n",
      "Complement found from 2704 to 2711\n",
      "Complement found from 14549 to 14556\n",
      "Seq is: TAACACA\n",
      "4 TAACACA motifs detected in file\n",
      "\n",
      "Complement found from 6769 to 6776\n",
      "Seq is: ACACCAT\n",
      "1 ACACCAT motifs detected in file\n",
      "\n",
      "Seq is: TTATACA\n",
      "0 TTATACA motifs detected in file\n",
      "\n",
      "Complement found from 4904 to 4911\n",
      "Complement found from 5214 to 5221\n",
      "Complement found from 9755 to 9762\n",
      "Complement found from 11057 to 11064\n",
      "Complement found from 12562 to 12569\n",
      "Complement found from 15851 to 15858\n",
      "Seq is: TCTCCCT\n",
      "6 TCTCCCT motifs detected in file\n",
      "\n",
      "Complement found from 1184 to 1191\n",
      "Complement found from 10546 to 10553\n",
      "Seq is: CATATCC\n",
      "2 CATATCC motifs detected in file\n",
      "\n",
      "Seq is: CGACACT\n",
      "0 CGACACT motifs detected in file\n",
      "\n",
      "Seq is: GATCATA\n",
      "0 GATCATA motifs detected in file\n",
      "\n",
      "Complement found from 4155 to 4162\n",
      "Complement found from 10133 to 10140\n",
      "Complement found from 12914 to 12921\n",
      "Seq is: CAACTCA\n",
      "3 CAACTCA motifs detected in file\n",
      "\n",
      "Complement found from 1514 to 1521\n",
      "Seq is: GGACATT\n",
      "1 GGACATT motifs detected in file\n",
      "\n",
      "Complement found from 11723 to 11730\n",
      "Complement found from 12881 to 12888\n",
      "Complement found from 15636 to 15643\n",
      "Complement found from 15660 to 15667\n",
      "Complement found from 16403 to 16410\n",
      "Seq is: CATCCTC\n",
      "5 CATCCTC motifs detected in file\n",
      "\n",
      "Complement found from 6519 to 6526\n",
      "Complement found from 7143 to 7150\n",
      "Complement found from 9849 to 9856\n",
      "Seq is: TCACTAT\n",
      "3 TCACTAT motifs detected in file\n",
      "\n",
      "Complement found from 4781 to 4788\n",
      "Complement found from 4871 to 4878\n",
      "Complement found from 11467 to 11474\n",
      "Complement found from 12576 to 12583\n",
      "Complement found from 15604 to 15611\n",
      "Seq is: AAACTAG\n",
      "5 AAACTAG motifs detected in file\n",
      "\n",
      "Complement found from 3290 to 3297\n",
      "Complement found from 4689 to 4696\n",
      "Complement found from 12130 to 12137\n",
      "Seq is: TCCTCTT\n",
      "3 TCCTCTT motifs detected in file\n",
      "\n",
      "Complement found from 14039 to 14046\n",
      "Complement found from 14248 to 14255\n",
      "Complement found from 14363 to 14370\n",
      "Seq is: GCACCAA\n",
      "3 GCACCAA motifs detected in file\n",
      "\n",
      "Seq is: TGATCGT\n",
      "0 TGATCGT motifs detected in file\n",
      "\n",
      "Seq is: GTCTCTC\n",
      "0 GTCTCTC motifs detected in file\n",
      "\n",
      "Seq is: AAGAGGG\n",
      "0 AAGAGGG motifs detected in file\n",
      "\n",
      "Complement found from 11663 to 11670\n",
      "Complement found from 13372 to 13379\n",
      "Seq is: TCATCCA\n",
      "2 TCATCCA motifs detected in file\n",
      "\n",
      "Complement found from 6404 to 6411\n",
      "Complement found from 12468 to 12475\n",
      "Seq is: ATTATCA\n",
      "2 ATTATCA motifs detected in file\n",
      "\n",
      "Complement found from 432 to 439\n",
      "Complement found from 567 to 574\n",
      "Complement found from 7396 to 7403\n",
      "Complement found from 7465 to 7472\n",
      "Complement found from 9531 to 9538\n",
      "Complement found from 11866 to 11873\n",
      "Complement found from 12384 to 12391\n",
      "Seq is: CCCCCCA\n",
      "7 CCCCCCA motifs detected in file\n",
      "\n",
      "Complement found from 4143 to 4150\n",
      "Seq is: TTCCGCT\n",
      "1 TTCCGCT motifs detected in file\n",
      "\n",
      "Seq is: CCAATCG\n",
      "0 CCAATCG motifs detected in file\n",
      "\n",
      "Seq is: CTCCGAA\n",
      "0 CTCCGAA motifs detected in file\n",
      "\n",
      "Seq is: CGATCTT\n",
      "0 CGATCTT motifs detected in file\n",
      "\n",
      "Seq is: CAGTAGT\n",
      "0 CAGTAGT motifs detected in file\n",
      "\n",
      "Seq is: TTTTCGA\n",
      "0 TTTTCGA motifs detected in file\n",
      "\n",
      "Complement found from 2598 to 2605\n",
      "Complement found from 6468 to 6475\n",
      "Complement found from 8650 to 8657\n",
      "Complement found from 14137 to 14144\n",
      "Seq is: TAATCAC\n",
      "4 TAATCAC motifs detected in file\n",
      "\n",
      "Complement found from 143 to 150\n",
      "Complement found from 5293 to 5300\n",
      "Complement found from 7806 to 7813\n",
      "Complement found from 8427 to 8434\n",
      "Complement found from 15639 to 15646\n",
      "Seq is: CCTCATC\n",
      "5 CCTCATC motifs detected in file\n",
      "\n",
      "Complement found from 3672 to 3679\n",
      "Complement found from 8674 to 8681\n",
      "Complement found from 11317 to 11324\n",
      "Complement found from 11752 to 11759\n",
      "Complement found from 11802 to 11809\n",
      "Complement found from 12574 to 12581\n",
      "Seq is: TCAAACT\n",
      "6 TCAAACT motifs detected in file\n",
      "\n",
      "Complement found from 7758 to 7765\n",
      "Seq is: TCAGGAA\n",
      "1 TCAGGAA motifs detected in file\n",
      "\n",
      "Complement found from 4935 to 4942\n",
      "Complement found from 6223 to 6230\n",
      "Complement found from 11788 to 11795\n",
      "Seq is: CTCTCTC\n",
      "3 CTCTCTC motifs detected in file\n",
      "\n",
      "Complement found from 7736 to 7743\n",
      "Complement found from 10954 to 10961\n",
      "Complement found from 15838 to 15845\n",
      "Complement found from 15903 to 15910\n",
      "Seq is: CTAATAC\n",
      "4 CTAATAC motifs detected in file\n",
      "\n",
      "Seq is: TACGCTG\n",
      "0 TACGCTG motifs detected in file\n",
      "\n",
      "Complement found from 4517 to 4524\n",
      "Complement found from 5450 to 5457\n",
      "Complement found from 7717 to 7724\n",
      "Complement found from 11255 to 11262\n",
      "Complement found from 11497 to 11504\n",
      "Complement found from 14398 to 14405\n",
      "Complement found from 14644 to 14651\n",
      "Seq is: ACACTCA\n",
      "7 ACACTCA motifs detected in file\n",
      "\n",
      "Complement found from 4653 to 4660\n",
      "Seq is: CGCAAGC\n",
      "1 CGCAAGC motifs detected in file\n",
      "\n",
      "Complement found from 3671 to 3678\n",
      "Complement found from 8673 to 8680\n",
      "Complement found from 11316 to 11323\n",
      "Complement found from 15357 to 15364\n",
      "Seq is: ATCAAAC\n",
      "4 ATCAAAC motifs detected in file\n",
      "\n",
      "Seq is: TCAAACG\n",
      "0 TCAAACG motifs detected in file\n",
      "\n",
      "Complement found from 6460 to 6467\n",
      "Complement found from 15591 to 15598\n",
      "Seq is: ATCCGTC\n",
      "2 ATCCGTC motifs detected in file\n",
      "\n",
      "Complement found from 9094 to 9101\n",
      "Seq is: TTATCAT\n",
      "1 TTATCAT motifs detected in file\n",
      "\n",
      "Complement found from 3325 to 3332\n",
      "Complement found from 3826 to 3833\n",
      "Complement found from 5009 to 5016\n",
      "Complement found from 5472 to 5479\n",
      "Complement found from 6231 to 6238\n",
      "Complement found from 8411 to 8418\n",
      "Complement found from 13984 to 13991\n",
      "Complement found from 14131 to 14138\n",
      "Complement found from 14440 to 14447\n",
      "Seq is: TACTCCT\n",
      "9 TACTCCT motifs detected in file\n",
      "\n",
      "Complement found from 7412 to 7419\n",
      "Seq is: ACATTCG\n",
      "1 ACATTCG motifs detected in file\n",
      "\n",
      "Complement found from 4829 to 4836\n",
      "Seq is: GGCACCC\n",
      "1 GGCACCC motifs detected in file\n",
      "\n",
      "Seq is: GCTAATA\n",
      "0 GCTAATA motifs detected in file\n",
      "\n",
      "Complement found from 2656 to 2663\n",
      "Complement found from 6212 to 6219\n",
      "Seq is: CTCTTAC\n",
      "2 CTCTTAC motifs detected in file\n",
      "\n",
      "Seq is: TGACGCG\n",
      "0 TGACGCG motifs detected in file\n",
      "\n",
      "Complement found from 259 to 266\n",
      "Complement found from 5630 to 5637\n",
      "Complement found from 11003 to 11010\n",
      "Seq is: GCCACTT\n",
      "3 GCCACTT motifs detected in file\n",
      "\n",
      "Seq is: CGAAGTC\n",
      "0 CGAAGTC motifs detected in file\n",
      "\n",
      "Complement found from 352 to 359\n",
      "Complement found from 556 to 563\n",
      "Complement found from 1681 to 1688\n",
      "Complement found from 7467 to 7474\n",
      "Complement found from 12301 to 12308\n",
      "Complement found from 14531 to 14538\n",
      "Seq is: CCCCAAA\n",
      "6 CCCCAAA motifs detected in file\n",
      "\n",
      "Complement found from 3323 to 3330\n",
      "Complement found from 6229 to 6236\n",
      "Complement found from 8969 to 8976\n",
      "Complement found from 9029 to 9036\n",
      "Complement found from 10750 to 10757\n",
      "Complement found from 11235 to 11242\n",
      "Complement found from 13070 to 13077\n",
      "Complement found from 13982 to 13989\n",
      "Complement found from 14129 to 14136\n",
      "Seq is: CCTACTC\n",
      "9 CCTACTC motifs detected in file\n",
      "\n",
      "Complement found from 3370 to 3377\n",
      "Complement found from 12201 to 12208\n",
      "Seq is: TTACCGA\n",
      "2 TTACCGA motifs detected in file\n",
      "\n",
      "Complement found from 9371 to 9378\n",
      "Complement found from 14700 to 14707\n",
      "Seq is: CAATGAT\n",
      "2 CAATGAT motifs detected in file\n",
      "\n",
      "Seq is: GCTCCCG\n",
      "0 GCTCCCG motifs detected in file\n",
      "\n",
      "Complement found from 4161 to 4168\n",
      "Complement found from 7240 to 7247\n",
      "Complement found from 12073 to 12080\n",
      "Complement found from 14180 to 14187\n",
      "Complement found from 15906 to 15913\n",
      "Seq is: ATACACC\n",
      "5 ATACACC motifs detected in file\n",
      "\n",
      "Complement found from 1830 to 1837\n",
      "Complement found from 5184 to 5191\n",
      "Seq is: GACTAAC\n",
      "2 GACTAAC motifs detected in file\n",
      "\n",
      "Complement found from 11660 to 11667\n",
      "Complement found from 11699 to 11706\n",
      "Complement found from 13732 to 13739\n",
      "Seq is: TTCTCAT\n",
      "3 TTCTCAT motifs detected in file\n",
      "\n",
      "Complement found from 7216 to 7223\n",
      "Seq is: ACGTTAC\n",
      "1 ACGTTAC motifs detected in file\n",
      "\n",
      "Complement found from 11849 to 11856\n",
      "Complement found from 14381 to 14388\n",
      "Seq is: TCGCTAA\n",
      "2 TCGCTAA motifs detected in file\n",
      "\n",
      "Complement found from 3083 to 3090\n",
      "Seq is: AATCCAG\n",
      "1 AATCCAG motifs detected in file\n",
      "\n",
      "Complement found from 3990 to 3997\n",
      "Complement found from 5679 to 5686\n",
      "Complement found from 10294 to 10301\n",
      "Complement found from 11592 to 11599\n",
      "Complement found from 14188 to 14195\n",
      "Seq is: ACAAACA\n",
      "5 ACAAACA motifs detected in file\n",
      "\n",
      "Complement found from 15805 to 15812\n",
      "Seq is: GCATCCG\n",
      "1 GCATCCG motifs detected in file\n",
      "\n",
      "Complement found from 5343 to 5350\n",
      "Complement found from 9184 to 9191\n",
      "Complement found from 11039 to 11046\n",
      "Complement found from 13786 to 13793\n",
      "Seq is: TCTACCT\n",
      "4 TCTACCT motifs detected in file\n",
      "\n",
      "Seq is: CGAACAA\n",
      "0 CGAACAA motifs detected in file\n",
      "\n",
      "Complement found from 2743 to 2750\n",
      "Complement found from 7299 to 7306\n",
      "Complement found from 10335 to 10342\n",
      "Seq is: TATTAAT\n",
      "3 TATTAAT motifs detected in file\n",
      "\n",
      "Seq is: ACGGAGT\n",
      "0 ACGGAGT motifs detected in file\n",
      "\n",
      "Complement found from 7821 to 7828\n",
      "Complement found from 11550 to 11557\n",
      "Seq is: ATCCCTA\n",
      "2 ATCCCTA motifs detected in file\n",
      "\n",
      "Complement found from 12065 to 12072\n",
      "Seq is: TCATGTT\n",
      "1 TCATGTT motifs detected in file\n",
      "\n",
      "Complement found from 3163 to 3170\n",
      "Complement found from 4456 to 4463\n",
      "Complement found from 9080 to 9087\n",
      "Complement found from 11229 to 11236\n",
      "Seq is: CCTTCCC\n",
      "4 CCTTCCC motifs detected in file\n",
      "\n",
      "Seq is: CATCCGA\n",
      "0 CATCCGA motifs detected in file\n",
      "\n",
      "Complement found from 3987 to 3994\n",
      "Complement found from 4068 to 4075\n",
      "Complement found from 8704 to 8711\n",
      "Complement found from 11353 to 11360\n",
      "Complement found from 13214 to 13221\n",
      "Complement found from 13542 to 13549\n",
      "Complement found from 15415 to 15422\n",
      "Complement found from 15577 to 15584\n",
      "Seq is: TACACAA\n",
      "8 TACACAA motifs detected in file\n",
      "\n",
      "Complement found from 424 to 431\n",
      "Complement found from 6280 to 6287\n",
      "Seq is: AACAGTC\n",
      "2 AACAGTC motifs detected in file\n",
      "\n",
      "Complement found from 434 to 441\n",
      "Complement found from 3413 to 3420\n",
      "Complement found from 8368 to 8375\n",
      "Complement found from 9555 to 9562\n",
      "Complement found from 10916 to 10923\n",
      "Seq is: CCCCAAC\n",
      "5 CCCCAAC motifs detected in file\n",
      "\n",
      "Complement found from 2657 to 2664\n",
      "Complement found from 9986 to 9993\n",
      "Complement found from 13108 to 13115\n",
      "Seq is: TCTTACT\n",
      "3 TCTTACT motifs detected in file\n",
      "\n",
      "Complement found from 4516 to 4523\n",
      "Complement found from 5449 to 5456\n",
      "Complement found from 6877 to 6884\n",
      "Complement found from 11496 to 11503\n",
      "Complement found from 14643 to 14650\n",
      "Seq is: CACACTC\n",
      "5 CACACTC motifs detected in file\n",
      "\n",
      "Complement found from 2331 to 2338\n",
      "Complement found from 3253 to 3260\n",
      "Complement found from 6188 to 6195\n",
      "Seq is: CGCATAA\n",
      "3 CGCATAA motifs detected in file\n",
      "\n",
      "Complement found from 11479 to 11486\n",
      "Seq is: TATGGTA\n",
      "1 TATGGTA motifs detected in file\n",
      "\n",
      "Seq is: TCGGCGG\n",
      "0 TCGGCGG motifs detected in file\n",
      "\n",
      "Complement found from 740 to 747\n",
      "Complement found from 13227 to 13234\n",
      "Complement found from 16176 to 16183\n",
      "Seq is: ATCAAAA\n",
      "3 ATCAAAA motifs detected in file\n",
      "\n",
      "Complement found from 11660 to 11667\n",
      "Complement found from 11699 to 11706\n",
      "Complement found from 13732 to 13739\n",
      "Seq is: TTCTCAT\n",
      "3 TTCTCAT motifs detected in file\n",
      "\n",
      "Seq is: GACGCCG\n",
      "0 GACGCCG motifs detected in file\n",
      "\n",
      "Complement found from 7310 to 7317\n",
      "Complement found from 7652 to 7659\n",
      "Seq is: TTCATGA\n",
      "2 TTCATGA motifs detected in file\n",
      "\n",
      "Seq is: TAACTGG\n",
      "0 TAACTGG motifs detected in file\n",
      "\n",
      "Seq is: GTCGCTA\n",
      "0 GTCGCTA motifs detected in file\n",
      "\n",
      "Complement found from 1900 to 1907\n",
      "Complement found from 4136 to 4143\n",
      "Complement found from 6168 to 6175\n",
      "Complement found from 14154 to 14161\n",
      "Seq is: CCCCCGA\n",
      "4 CCCCCGA motifs detected in file\n",
      "\n",
      "Complement found from 7054 to 7061\n",
      "Seq is: AGCTGTA\n",
      "1 AGCTGTA motifs detected in file\n",
      "\n",
      "Complement found from 1902 to 1909\n",
      "Complement found from 4436 to 4443\n",
      "Seq is: CCCGAAA\n",
      "2 CCCGAAA motifs detected in file\n",
      "\n",
      "Complement found from 4906 to 4913\n",
      "Complement found from 8472 to 8479\n",
      "Complement found from 12100 to 12107\n",
      "Complement found from 13449 to 13456\n",
      "Seq is: TCCCTCA\n",
      "4 TCCCTCA motifs detected in file\n",
      "\n",
      "Seq is: GTATTCA\n",
      "0 GTATTCA motifs detected in file\n",
      "\n",
      "Complement found from 14432 to 14439\n",
      "Seq is: CCTCAGG\n",
      "1 CCTCAGG motifs detected in file\n",
      "\n",
      "Complement found from 7279 to 7286\n",
      "Complement found from 15062 to 15069\n",
      "Seq is: CATTTCT\n",
      "2 CATTTCT motifs detected in file\n",
      "\n",
      "Complement found from 13042 to 13049\n",
      "Seq is: CCATAGA\n",
      "1 CCATAGA motifs detected in file\n",
      "\n",
      "Complement found from 3175 to 3182\n",
      "Complement found from 8693 to 8700\n",
      "Complement found from 14701 to 14708\n",
      "Complement found from 15557 to 15564\n",
      "Seq is: AATGATA\n",
      "4 AATGATA motifs detected in file\n",
      "\n",
      "Seq is: TTTTCGA\n",
      "0 TTTTCGA motifs detected in file\n",
      "\n",
      "Complement found from 441 to 448\n",
      "Complement found from 1613 to 1620\n",
      "Complement found from 2704 to 2711\n",
      "Complement found from 14549 to 14556\n",
      "Seq is: TAACACA\n",
      "4 TAACACA motifs detected in file\n",
      "\n",
      "Seq is: TTCGATC\n",
      "0 TTCGATC motifs detected in file\n",
      "\n",
      "Complement found from 2302 to 2309\n",
      "Complement found from 9998 to 10005\n",
      "Complement found from 15895 to 15902\n",
      "Seq is: AGTATAA\n",
      "3 AGTATAA motifs detected in file\n",
      "\n",
      "Complement found from 3936 to 3943\n",
      "Complement found from 4694 to 4701\n",
      "Seq is: TTCAACA\n",
      "2 TTCAACA motifs detected in file\n",
      "\n",
      "Seq is: ACCGAGT\n",
      "0 ACCGAGT motifs detected in file\n",
      "\n",
      "Complement found from 593 to 600\n",
      "Complement found from 4251 to 4258\n",
      "Complement found from 8683 to 8690\n",
      "Seq is: CCTCAAA\n",
      "3 CCTCAAA motifs detected in file\n",
      "\n",
      "Complement found from 3756 to 3763\n",
      "Complement found from 7043 to 7050\n",
      "Complement found from 11314 to 11321\n",
      "Seq is: CTATCAA\n",
      "3 CTATCAA motifs detected in file\n",
      "\n",
      "Complement found from 7421 to 7428\n",
      "Seq is: GAACCCG\n",
      "1 GAACCCG motifs detected in file\n",
      "\n",
      "Complement found from 4932 to 4939\n",
      "Complement found from 11294 to 11301\n",
      "Complement found from 11930 to 11937\n",
      "Seq is: TCACTCT\n",
      "3 TCACTCT motifs detected in file\n",
      "\n",
      "Complement found from 4518 to 4525\n",
      "Complement found from 5451 to 5458\n",
      "Complement found from 8065 to 8072\n",
      "Complement found from 8793 to 8800\n",
      "Complement found from 11498 to 11505\n",
      "Complement found from 14116 to 14123\n",
      "Complement found from 14791 to 14798\n",
      "Seq is: CACTCAT\n",
      "7 CACTCAT motifs detected in file\n",
      "\n",
      "Seq is: CATCTTA\n",
      "0 CATCTTA motifs detected in file\n",
      "\n",
      "Complement found from 5623 to 5630\n",
      "Complement found from 7875 to 7882\n",
      "Complement found from 10888 to 10895\n",
      "Complement found from 13001 to 13008\n",
      "Complement found from 14875 to 14882\n",
      "Complement found from 15951 to 15958\n",
      "Seq is: CAAATCA\n",
      "6 CAAATCA motifs detected in file\n",
      "\n",
      "Complement found from 13911 to 13918\n",
      "Seq is: CTCGGAT\n",
      "1 CTCGGAT motifs detected in file\n",
      "\n",
      "Complement found from 5676 to 5683\n",
      "Complement found from 7180 to 7187\n",
      "Complement found from 8559 to 8566\n",
      "Complement found from 12483 to 12490\n",
      "Complement found from 12926 to 12933\n",
      "Complement found from 14620 to 14627\n",
      "Seq is: CCCACAA\n",
      "6 CCCACAA motifs detected in file\n",
      "\n",
      "Complement found from 26 to 33\n",
      "Complement found from 4189 to 4196\n",
      "Complement found from 7718 to 7725\n",
      "Complement found from 11256 to 11263\n",
      "Complement found from 11766 to 11773\n",
      "Complement found from 12011 to 12018\n",
      "Complement found from 14399 to 14406\n",
      "Seq is: CACTCAC\n",
      "7 CACTCAC motifs detected in file\n",
      "\n",
      "Seq is: ATTTAAT\n",
      "0 ATTTAAT motifs detected in file\n",
      "\n",
      "Complement found from 5467 to 5474\n",
      "Seq is: CACGCTA\n",
      "1 CACGCTA motifs detected in file\n",
      "\n",
      "Complement found from 144 to 151\n",
      "Complement found from 11662 to 11669\n",
      "Complement found from 13113 to 13120\n",
      "Complement found from 14118 to 14125\n",
      "Complement found from 15640 to 15647\n",
      "Seq is: CTCATCC\n",
      "5 CTCATCC motifs detected in file\n",
      "\n",
      "Complement found from 3163 to 3170\n",
      "Complement found from 4456 to 4463\n",
      "Complement found from 9080 to 9087\n",
      "Complement found from 11229 to 11236\n",
      "Seq is: CCTTCCC\n",
      "4 CCTTCCC motifs detected in file\n",
      "\n",
      "Seq is: TCGGATG\n",
      "0 TCGGATG motifs detected in file\n",
      "\n",
      "Seq is: ATCCCAC\n",
      "0 ATCCCAC motifs detected in file\n",
      "\n",
      "Complement found from 4943 to 4950\n",
      "Complement found from 5504 to 5511\n",
      "Seq is: ATCTTAT\n",
      "2 ATCTTAT motifs detected in file\n",
      "\n",
      "Complement found from 14287 to 14294\n",
      "Complement found from 15125 to 15132\n",
      "Complement found from 15302 to 15309\n",
      "Seq is: CCTTCAT\n",
      "3 CCTTCAT motifs detected in file\n",
      "\n",
      "Seq is: ACACGGA\n",
      "0 ACACGGA motifs detected in file\n",
      "\n",
      "Complement found from 1301 to 1308\n",
      "Complement found from 7598 to 7605\n",
      "Seq is: CGCAAGT\n",
      "2 CGCAAGT motifs detected in file\n",
      "\n",
      "Complement found from 1784 to 1791\n",
      "Seq is: CCGCAAG\n",
      "1 CCGCAAG motifs detected in file\n",
      "\n",
      "Seq is: ACCGACC\n",
      "0 ACCGACC motifs detected in file\n",
      "\n",
      "Seq is: AACGGAG\n",
      "0 AACGGAG motifs detected in file\n",
      "\n",
      "Complement found from 1336 to 1343\n",
      "Complement found from 3712 to 3719\n",
      "Complement found from 7023 to 7030\n",
      "Complement found from 8899 to 8906\n",
      "Complement found from 13142 to 13149\n",
      "Complement found from 16532 to 16539\n",
      "Seq is: TAGCCCA\n",
      "6 TAGCCCA motifs detected in file\n",
      "\n",
      "Complement found from 2880 to 2887\n",
      "Complement found from 3555 to 3562\n",
      "Complement found from 3753 to 3760\n",
      "Complement found from 5153 to 5160\n",
      "Complement found from 10556 to 10563\n",
      "Complement found from 10873 to 10880\n",
      "Complement found from 12349 to 12356\n",
      "Seq is: CTACTAT\n",
      "7 CTACTAT motifs detected in file\n",
      "\n",
      "Seq is: CGGTTCC\n",
      "0 CGGTTCC motifs detected in file\n",
      "\n",
      "Complement found from 1529 to 1536\n",
      "Complement found from 1835 to 1842\n",
      "Complement found from 9253 to 9260\n",
      "Complement found from 10276 to 10283\n",
      "Seq is: ACCCCTA\n",
      "4 ACCCCTA motifs detected in file\n",
      "\n",
      "Complement found from 9165 to 9172\n",
      "Complement found from 12046 to 12053\n",
      "Seq is: TTCACAC\n",
      "2 TTCACAC motifs detected in file\n",
      "\n",
      "Complement found from 1889 to 1896\n",
      "Complement found from 7470 to 7477\n",
      "Complement found from 15990 to 15997\n",
      "Seq is: CAAAGCT\n",
      "3 CAAAGCT motifs detected in file\n",
      "\n",
      "Complement found from 3551 to 3558\n",
      "Seq is: TCTTCTA\n",
      "1 TCTTCTA motifs detected in file\n",
      "\n",
      "Complement found from 9719 to 9726\n",
      "Complement found from 11108 to 11115\n",
      "Seq is: TATTTTA\n",
      "2 TATTTTA motifs detected in file\n",
      "\n",
      "Complement found from 473 to 480\n",
      "Complement found from 3766 to 3773\n",
      "Complement found from 4464 to 4471\n",
      "Complement found from 5496 to 5503\n",
      "Complement found from 10104 to 10111\n",
      "Complement found from 11051 to 11058\n",
      "Complement found from 12718 to 12725\n",
      "Complement found from 14214 to 14221\n",
      "Seq is: TACTAAT\n",
      "8 TACTAAT motifs detected in file\n",
      "\n",
      "Seq is: CAAGACA\n",
      "0 CAAGACA motifs detected in file\n",
      "\n",
      "Seq is: CCGACAC\n",
      "0 CCGACAC motifs detected in file\n",
      "\n",
      "Complement found from 4231 to 4238\n",
      "Complement found from 9699 to 9706\n",
      "Complement found from 13279 to 13286\n",
      "Complement found from 14171 to 14178\n",
      "Seq is: TTACAAT\n",
      "4 TTACAAT motifs detected in file\n",
      "\n",
      "Complement found from 210 to 217\n",
      "Complement found from 8220 to 8227\n",
      "Complement found from 14781 to 14788\n",
      "Seq is: ATTAATT\n",
      "3 ATTAATT motifs detected in file\n",
      "\n",
      "Complement found from 724 to 731\n",
      "Complement found from 2066 to 2073\n",
      "Seq is: CCTCTAA\n",
      "2 CCTCTAA motifs detected in file\n",
      "\n",
      "Complement found from 3002 to 3009\n",
      "Seq is: ACATCCC\n",
      "1 ACATCCC motifs detected in file\n",
      "\n",
      "Complement found from 3344 to 3351\n",
      "Seq is: TCTAATC\n",
      "1 TCTAATC motifs detected in file\n",
      "\n",
      "Complement found from 8630 to 8637\n",
      "Seq is: ATATCTC\n",
      "1 ATATCTC motifs detected in file\n",
      "\n",
      "Complement found from 9968 to 9975\n",
      "Complement found from 11578 to 11585\n",
      "Seq is: TCCATCT\n",
      "2 TCCATCT motifs detected in file\n",
      "\n",
      "Complement found from 284 to 291\n",
      "Complement found from 4603 to 4610\n",
      "Complement found from 5280 to 5287\n",
      "Complement found from 7445 to 7452\n",
      "Complement found from 10046 to 10053\n",
      "Complement found from 10812 to 10819\n",
      "Complement found from 12416 to 12423\n",
      "Complement found from 13229 to 13236\n",
      "Seq is: CAAAAAA\n",
      "8 CAAAAAA motifs detected in file\n",
      "\n",
      "Complement found from 13083 to 13090\n",
      "Seq is: AGCACTA\n",
      "1 AGCACTA motifs detected in file\n",
      "\n",
      "Complement found from 3472 to 3479\n",
      "Complement found from 4084 to 4091\n",
      "Complement found from 8476 to 8483\n",
      "Complement found from 14402 to 14409\n",
      "Seq is: TCACCAA\n",
      "4 TCACCAA motifs detected in file\n",
      "\n",
      "Seq is: GTGACCT\n",
      "0 GTGACCT motifs detected in file\n",
      "\n",
      "Seq is: ATTCACC\n",
      "0 ATTCACC motifs detected in file\n",
      "\n",
      "Complement found from 4658 to 4665\n",
      "Seq is: GCAACCG\n",
      "1 GCAACCG motifs detected in file\n",
      "\n",
      "Seq is: AACGGTA\n",
      "0 AACGGTA motifs detected in file\n",
      "\n",
      "Complement found from 370 to 377\n",
      "Complement found from 440 to 447\n",
      "Complement found from 5186 to 5193\n",
      "Complement found from 7714 to 7721\n",
      "Complement found from 13161 to 13168\n",
      "Seq is: CTAACAC\n",
      "5 CTAACAC motifs detected in file\n",
      "\n",
      "Complement found from 3571 to 3578\n",
      "Complement found from 5382 to 5389\n",
      "Complement found from 8938 to 8945\n",
      "Complement found from 15656 to 15663\n",
      "Complement found from 16188 to 16195\n",
      "Complement found from 16361 to 16368\n",
      "Seq is: TCCCCAT\n",
      "6 TCCCCAT motifs detected in file\n",
      "\n",
      "Complement found from 5550 to 5557\n",
      "Seq is: CCTCAGT\n",
      "1 CCTCAGT motifs detected in file\n",
      "\n",
      "Complement found from 4635 to 4642\n",
      "Complement found from 6384 to 6391\n",
      "Complement found from 7871 to 7878\n",
      "Complement found from 16071 to 16078\n",
      "Seq is: CCATCAA\n",
      "4 CCATCAA motifs detected in file\n",
      "\n",
      "Complement found from 4761 to 4768\n",
      "Complement found from 6120 to 6127\n",
      "Complement found from 7667 to 7674\n",
      "Complement found from 11702 to 11709\n",
      "Complement found from 11780 to 11787\n",
      "Complement found from 13270 to 13277\n",
      "Seq is: TCATAAT\n",
      "6 TCATAAT motifs detected in file\n",
      "\n",
      "Complement found from 307 to 314\n",
      "Complement found from 958 to 965\n",
      "Complement found from 3569 to 3576\n",
      "Complement found from 5443 to 5450\n",
      "Complement found from 14527 to 14534\n",
      "Complement found from 14805 to 14812\n",
      "Complement found from 15538 to 15545\n",
      "Complement found from 16186 to 16193\n",
      "Seq is: CCTCCCC\n",
      "8 CCTCCCC motifs detected in file\n",
      "\n",
      "Complement found from 2395 to 2402\n",
      "Complement found from 8660 to 8667\n",
      "Complement found from 10777 to 10784\n",
      "Complement found from 11330 to 11337\n",
      "Complement found from 13858 to 13865\n",
      "Complement found from 14185 to 14192\n",
      "Complement found from 16277 to 16284\n",
      "Seq is: CCAACAA\n",
      "7 CCAACAA motifs detected in file\n",
      "\n",
      "Complement found from 7953 to 7960\n",
      "Complement found from 12082 to 12089\n",
      "Complement found from 14529 to 14536\n",
      "Seq is: TCCCCCA\n",
      "3 TCCCCCA motifs detected in file\n",
      "\n",
      "Complement found from 3607 to 3614\n",
      "Complement found from 6003 to 6010\n",
      "Complement found from 12979 to 12986\n",
      "Seq is: GCCTCCT\n",
      "3 GCCTCCT motifs detected in file\n",
      "\n",
      "Complement found from 10808 to 10815\n",
      "Complement found from 15941 to 15948\n",
      "Seq is: TTTCCAA\n",
      "2 TTTCCAA motifs detected in file\n",
      "\n",
      "Complement found from 2075 to 2082\n",
      "Seq is: CCCCTTG\n",
      "1 CCCCTTG motifs detected in file\n",
      "\n",
      "Seq is: TCGATTT\n",
      "0 TCGATTT motifs detected in file\n",
      "\n",
      "Complement found from 14817 to 14824\n",
      "Seq is: ATCCAAC\n",
      "1 ATCCAAC motifs detected in file\n",
      "\n",
      "Complement found from 10297 to 10304\n",
      "Complement found from 11332 to 11339\n",
      "Seq is: AACAACT\n",
      "2 AACAACT motifs detected in file\n",
      "\n",
      "Complement found from 4978 to 4985\n",
      "Complement found from 8121 to 8128\n",
      "Complement found from 9673 to 9680\n",
      "Seq is: AAACCAA\n",
      "3 AAACCAA motifs detected in file\n",
      "\n",
      "Complement found from 4547 to 4554\n",
      "Complement found from 10879 to 10886\n",
      "Seq is: TTTTTTA\n",
      "2 TTTTTTA motifs detected in file\n",
      "\n",
      "Complement found from 3182 to 3189\n",
      "Seq is: TCATCTC\n",
      "1 TCATCTC motifs detected in file\n",
      "\n",
      "Complement found from 5116 to 5123\n",
      "Complement found from 5149 to 5156\n",
      "Complement found from 9343 to 9350\n",
      "Complement found from 10555 to 10562\n",
      "Complement found from 10696 to 10703\n",
      "Complement found from 13682 to 13689\n",
      "Seq is: CCTACTA\n",
      "6 CCTACTA motifs detected in file\n",
      "\n",
      "Seq is: CGATTAT\n",
      "0 CGATTAT motifs detected in file\n",
      "\n",
      "Seq is: TAATCCG\n",
      "0 TAATCCG motifs detected in file\n",
      "\n",
      "Complement found from 2160 to 2167\n",
      "Complement found from 2213 to 2220\n",
      "Complement found from 2257 to 2264\n",
      "Complement found from 3206 to 3213\n",
      "Complement found from 10617 to 10624\n",
      "Seq is: ACACCCA\n",
      "5 ACACCCA motifs detected in file\n",
      "\n",
      "Complement found from 2426 to 2433\n",
      "Complement found from 6595 to 6602\n",
      "Complement found from 8344 to 8351\n",
      "Complement found from 9351 to 9358\n",
      "Complement found from 10713 to 10720\n",
      "Complement found from 12826 to 12833\n",
      "Seq is: CCAACAC\n",
      "6 CCAACAC motifs detected in file\n",
      "\n",
      "Complement found from 850 to 857\n",
      "Complement found from 1433 to 1440\n",
      "Seq is: AACTAAG\n",
      "2 AACTAAG motifs detected in file\n",
      "\n",
      "Complement found from 10075 to 10082\n",
      "Seq is: TTTAATA\n",
      "1 TTTAATA motifs detected in file\n",
      "\n",
      "Complement found from 1008 to 1015\n",
      "Complement found from 5278 to 5285\n",
      "Complement found from 13216 to 13223\n",
      "Seq is: CACAAAA\n",
      "3 CACAAAA motifs detected in file\n",
      "\n",
      "Complement found from 825 to 832\n",
      "Complement found from 8317 to 8324\n",
      "Complement found from 10920 to 10927\n",
      "Complement found from 15935 to 15942\n",
      "Seq is: AACCTTT\n",
      "4 AACCTTT motifs detected in file\n",
      "\n",
      "Complement found from 2846 to 2853\n",
      "Complement found from 16300 to 16307\n",
      "Seq is: CAGTACA\n",
      "2 CAGTACA motifs detected in file\n",
      "\n",
      "Complement found from 4083 to 4090\n",
      "Seq is: GTCACCA\n",
      "1 GTCACCA motifs detected in file\n",
      "\n",
      "Complement found from 2403 to 2410\n",
      "Complement found from 9828 to 9835\n",
      "Complement found from 10590 to 10597\n",
      "Complement found from 14669 to 14676\n",
      "Complement found from 15305 to 15312\n",
      "Seq is: TCATTAT\n",
      "5 TCATTAT motifs detected in file\n",
      "\n",
      "Complement found from 2369 to 2376\n",
      "Complement found from 12025 to 12032\n",
      "Complement found from 15468 to 15475\n",
      "Seq is: ATTAACA\n",
      "3 ATTAACA motifs detected in file\n",
      "\n",
      "Seq is: ACCCTGC\n",
      "0 ACCCTGC motifs detected in file\n",
      "\n",
      "Complement found from 5708 to 5715\n",
      "Complement found from 8671 to 8678\n",
      "Complement found from 10080 to 10087\n",
      "Complement found from 14217 to 14224\n",
      "Seq is: TAATCAA\n",
      "4 TAATCAA motifs detected in file\n",
      "\n",
      "Complement found from 13235 to 13242\n",
      "Seq is: AATCGTA\n",
      "1 AATCGTA motifs detected in file\n",
      "\n",
      "Complement found from 10542 to 10549\n",
      "Seq is: ACCTCAT\n",
      "1 ACCTCAT motifs detected in file\n",
      "\n",
      "Complement found from 4897 to 4904\n",
      "Complement found from 10887 to 10894\n",
      "Complement found from 14042 to 14049\n",
      "Complement found from 14874 to 14881\n",
      "Seq is: CCAAATC\n",
      "4 CCAAATC motifs detected in file\n",
      "\n",
      "Complement found from 485 to 492\n",
      "Complement found from 601 to 608\n",
      "Seq is: CAATACA\n",
      "2 CAATACA motifs detected in file\n",
      "\n",
      "Complement found from 5608 to 5615\n",
      "Seq is: TGCATCA\n",
      "1 TGCATCA motifs detected in file\n",
      "\n",
      "Seq is: GATCCGG\n",
      "0 GATCCGG motifs detected in file\n",
      "\n",
      "Complement found from 14179 to 14186\n",
      "Seq is: TATACAC\n",
      "1 TATACAC motifs detected in file\n",
      "\n",
      "Complement found from 3786 to 3793\n",
      "Complement found from 5333 to 5340\n",
      "Seq is: AACCTCT\n",
      "2 AACCTCT motifs detected in file\n",
      "\n",
      "Seq is: AACTCTG\n",
      "0 AACTCTG motifs detected in file\n",
      "\n",
      "Complement found from 5297 to 5304\n",
      "Complement found from 7795 to 7802\n",
      "Complement found from 10343 to 10350\n",
      "Complement found from 10864 to 10871\n",
      "Complement found from 13371 to 13378\n",
      "Complement found from 14977 to 14984\n",
      "Seq is: ATCATCC\n",
      "6 ATCATCC motifs detected in file\n",
      "\n",
      "Complement found from 15589 to 15596\n",
      "Seq is: CGATCCG\n",
      "1 CGATCCG motifs detected in file\n",
      "\n",
      "Seq is: GGTGGCG\n",
      "0 GGTGGCG motifs detected in file\n",
      "\n",
      "Seq is: ACGAAGC\n",
      "0 ACGAAGC motifs detected in file\n",
      "\n",
      "Seq is: GTCCGCC\n",
      "0 GTCCGCC motifs detected in file\n",
      "\n",
      "Seq is: GTTTCTG\n",
      "0 GTTTCTG motifs detected in file\n",
      "\n",
      "Complement found from 13538 to 13545\n",
      "Complement found from 14232 to 14239\n",
      "Complement found from 14343 to 14350\n",
      "Seq is: ATCATAC\n",
      "3 ATCATAC motifs detected in file\n",
      "\n",
      "Complement found from 6067 to 6074\n",
      "Seq is: CGTTATC\n",
      "1 CGTTATC motifs detected in file\n",
      "\n",
      "Seq is: CGTGTTA\n",
      "0 CGTGTTA motifs detected in file\n",
      "\n",
      "Seq is: TAACCGA\n",
      "0 TAACCGA motifs detected in file\n",
      "\n",
      "Complement found from 5730 to 5737\n",
      "Seq is: TTCTCCC\n",
      "1 TTCTCCC motifs detected in file\n",
      "\n",
      "Complement found from 8432 to 8439\n",
      "Complement found from 12014 to 12021\n",
      "Complement found from 14354 to 14361\n",
      "Complement found from 16067 to 16074\n",
      "Complement found from 16262 to 16269\n",
      "Seq is: TCACCCA\n",
      "5 TCACCCA motifs detected in file\n",
      "\n",
      "Complement found from 7938 to 7945\n",
      "Complement found from 12318 to 12325\n",
      "Complement found from 16244 to 16251\n",
      "Seq is: CAACTCC\n",
      "3 CAACTCC motifs detected in file\n",
      "\n",
      "Complement found from 4729 to 4736\n",
      "Complement found from 4742 to 4749\n",
      "Complement found from 5563 to 5570\n",
      "Complement found from 14577 to 14584\n",
      "Seq is: CAATACT\n",
      "4 CAATACT motifs detected in file\n",
      "\n",
      "Complement found from 5453 to 5460\n",
      "Complement found from 7807 to 7814\n",
      "Complement found from 11239 to 11246\n",
      "Complement found from 13572 to 13579\n",
      "Seq is: CTCATCG\n",
      "4 CTCATCG motifs detected in file\n",
      "\n",
      "Complement found from 175 to 182\n",
      "Complement found from 10786 to 10793\n",
      "Complement found from 15051 to 15058\n",
      "Seq is: ATATTAC\n",
      "3 ATATTAC motifs detected in file\n",
      "\n",
      "Complement found from 12451 to 12458\n",
      "Seq is: TTGTCGC\n",
      "1 TTGTCGC motifs detected in file\n",
      "\n",
      "Complement found from 7147 to 7154\n",
      "Complement found from 7629 to 7636\n",
      "Complement found from 9227 to 9234\n",
      "Complement found from 13537 to 13544\n",
      "Seq is: TATCATA\n",
      "4 TATCATA motifs detected in file\n",
      "\n",
      "Complement found from 2443 to 2450\n",
      "Seq is: ATAAGGA\n",
      "1 ATAAGGA motifs detected in file\n",
      "\n",
      "Seq is: AGATCCG\n",
      "0 AGATCCG motifs detected in file\n",
      "\n",
      "Complement found from 3180 to 3187\n",
      "Complement found from 7260 to 7267\n",
      "Complement found from 9095 to 9102\n",
      "Complement found from 11151 to 11158\n",
      "Seq is: TATCATC\n",
      "4 TATCATC motifs detected in file\n",
      "\n",
      "Complement found from 3320 to 3327\n",
      "Complement found from 3608 to 3615\n",
      "Complement found from 4166 to 4173\n",
      "Complement found from 9277 to 9284\n",
      "Complement found from 9727 to 9734\n",
      "Complement found from 10090 to 10097\n",
      "Complement found from 10950 to 10957\n",
      "Complement found from 12093 to 12100\n",
      "Complement found from 12983 to 12990\n",
      "Complement found from 13988 to 13995\n",
      "Complement found from 15333 to 15340\n",
      "Complement found from 15489 to 15496\n",
      "Seq is: CCTCCTA\n",
      "12 CCTCCTA motifs detected in file\n",
      "\n",
      "Complement found from 5360 to 5367\n",
      "Complement found from 10752 to 10759\n",
      "Complement found from 13072 to 13079\n",
      "Complement found from 13503 to 13510\n",
      "Seq is: TACTCCA\n",
      "4 TACTCCA motifs detected in file\n",
      "\n",
      "Seq is: CACTCGC\n",
      "0 CACTCGC motifs detected in file\n",
      "\n",
      "Complement found from 4208 to 4215\n",
      "Seq is: TATATGA\n",
      "1 TATATGA motifs detected in file\n",
      "\n",
      "Complement found from 2441 to 2448\n",
      "Seq is: TCATAAG\n",
      "1 TCATAAG motifs detected in file\n",
      "\n",
      "Seq is: GATCTTC\n",
      "0 GATCTTC motifs detected in file\n",
      "\n",
      "Complement found from 4064 to 4071\n",
      "Complement found from 4497 to 4504\n",
      "Complement found from 11037 to 11044\n",
      "Complement found from 11806 to 11813\n",
      "Seq is: ACTCTAC\n",
      "4 ACTCTAC motifs detected in file\n",
      "\n",
      "Complement found from 1725 to 1732\n",
      "Complement found from 8797 to 8804\n",
      "Complement found from 10477 to 10484\n",
      "Complement found from 10500 to 10507\n",
      "Complement found from 16320 to 16327\n",
      "Seq is: CATTTAC\n",
      "5 CATTTAC motifs detected in file\n",
      "\n",
      "Seq is: GATCATA\n",
      "0 GATCATA motifs detected in file\n",
      "\n",
      "Seq is: TTATAAA\n",
      "0 TTATAAA motifs detected in file\n",
      "\n",
      "Complement found from 1363 to 1370\n",
      "Complement found from 10117 to 10124\n",
      "Seq is: TACATTT\n",
      "2 TACATTT motifs detected in file\n",
      "\n",
      "Complement found from 5380 to 5387\n",
      "Complement found from 13031 to 13038\n",
      "Seq is: ACTCCCC\n",
      "2 ACTCCCC motifs detected in file\n",
      "\n",
      "Complement found from 15919 to 15926\n",
      "Seq is: GTAAACC\n",
      "1 GTAAACC motifs detected in file\n",
      "\n",
      "Complement found from 1539 to 1546\n",
      "Complement found from 1973 to 1980\n",
      "Complement found from 5087 to 5094\n",
      "Seq is: ATTTATA\n",
      "3 ATTTATA motifs detected in file\n",
      "\n",
      "Complement found from 4679 to 4686\n",
      "Complement found from 7349 to 7356\n",
      "Complement found from 9656 to 9663\n",
      "Complement found from 10313 to 10320\n",
      "Complement found from 11818 to 11825\n",
      "Seq is: CTAATAG\n",
      "5 CTAATAG motifs detected in file\n",
      "\n",
      "Complement found from 3023 to 3030\n",
      "Complement found from 8447 to 8454\n",
      "Complement found from 14315 to 14322\n",
      "Complement found from 14511 to 14518\n",
      "Seq is: TATTAAA\n",
      "4 TATTAAA motifs detected in file\n",
      "\n",
      "Complement found from 3644 to 3651\n",
      "Complement found from 13110 to 13117\n",
      "Seq is: TTACTCA\n",
      "2 TTACTCA motifs detected in file\n",
      "\n",
      "Seq is: GCTTTAG\n",
      "0 GCTTTAG motifs detected in file\n",
      "\n",
      "Seq is: TAAGCAA\n",
      "0 TAAGCAA motifs detected in file\n",
      "\n",
      "Complement found from 643 to 650\n",
      "Complement found from 8689 to 8696\n",
      "Complement found from 12931 to 12938\n",
      "Seq is: AACAAAT\n",
      "3 AACAAAT motifs detected in file\n",
      "\n",
      "Seq is: TTGATAT\n",
      "0 TTGATAT motifs detected in file\n",
      "\n",
      "Seq is: ATGGTTA\n",
      "0 ATGGTTA motifs detected in file\n",
      "\n",
      "Seq is: AGCCAAG\n",
      "0 AGCCAAG motifs detected in file\n",
      "\n",
      "Complement found from 2345 to 2352\n",
      "Complement found from 12160 to 12167\n",
      "Seq is: TCAGATT\n",
      "2 TCAGATT motifs detected in file\n",
      "\n",
      "Complement found from 2126 to 2133\n",
      "Seq is: AGGAAAA\n",
      "1 AGGAAAA motifs detected in file\n",
      "\n",
      "Complement found from 12157 to 12164\n",
      "Seq is: ACATCAG\n",
      "1 ACATCAG motifs detected in file\n",
      "\n",
      "Seq is: TGAGTAT\n",
      "0 TGAGTAT motifs detected in file\n",
      "\n",
      "Seq is: CCGGTCT\n",
      "0 CCGGTCT motifs detected in file\n",
      "\n",
      "Seq is: GAATCCC\n",
      "0 GAATCCC motifs detected in file\n",
      "\n",
      "Seq is: GATCCTA\n",
      "0 GATCCTA motifs detected in file\n",
      "\n",
      "Complement found from 472 to 479\n",
      "Complement found from 6527 to 6534\n",
      "Complement found from 9344 to 9351\n",
      "Complement found from 10103 to 10110\n",
      "Complement found from 13683 to 13690\n",
      "Complement found from 14213 to 14220\n",
      "Seq is: CTACTAA\n",
      "6 CTACTAA motifs detected in file\n",
      "\n",
      "Complement found from 3616 to 3623\n",
      "Complement found from 14672 to 14679\n",
      "Seq is: TTATTCT\n",
      "2 TTATTCT motifs detected in file\n",
      "\n",
      "Complement found from 4093 to 4100\n",
      "Complement found from 5148 to 5155\n",
      "Complement found from 10554 to 10561\n",
      "Complement found from 10695 to 10702\n",
      "Complement found from 11234 to 11241\n",
      "Complement found from 13069 to 13076\n",
      "Complement found from 13681 to 13688\n",
      "Complement found from 13981 to 13988\n",
      "Complement found from 14128 to 14135\n",
      "Seq is: CCCTACT\n",
      "9 CCCTACT motifs detected in file\n",
      "\n",
      "Complement found from 16059 to 16066\n",
      "Seq is: GTATTGA\n",
      "1 GTATTGA motifs detected in file\n",
      "\n",
      "Complement found from 2598 to 2605\n",
      "Complement found from 6468 to 6475\n",
      "Complement found from 8650 to 8657\n",
      "Complement found from 14137 to 14144\n",
      "Seq is: TAATCAC\n",
      "4 TAATCAC motifs detected in file\n",
      "\n",
      "Seq is: CGTCCAA\n",
      "0 CGTCCAA motifs detected in file\n",
      "\n",
      "Complement found from 1118 to 1125\n",
      "Complement found from 2965 to 2972\n",
      "Complement found from 4695 to 4702\n",
      "Complement found from 8638 to 8645\n",
      "Complement found from 10892 to 10899\n",
      "Complement found from 16074 to 16081\n",
      "Seq is: TCAACAA\n",
      "6 TCAACAA motifs detected in file\n",
      "\n",
      "Complement found from 3002 to 3009\n",
      "Seq is: ACATCCC\n",
      "1 ACATCCC motifs detected in file\n",
      "\n",
      "Seq is: GAGTACC\n",
      "0 GAGTACC motifs detected in file\n",
      "\n",
      "Complement found from 5300 to 5307\n",
      "Complement found from 6842 to 6849\n",
      "Complement found from 8616 to 8623\n",
      "Complement found from 8937 to 8944\n",
      "Complement found from 11137 to 11144\n",
      "Complement found from 13877 to 13884\n",
      "Complement found from 15655 to 15662\n",
      "Seq is: ATCCCCA\n",
      "7 ATCCCCA motifs detected in file\n",
      "\n",
      "Complement found from 7676 to 7683\n",
      "Complement found from 7709 to 7716\n",
      "Complement found from 10924 to 10931\n",
      "Complement found from 12127 to 12134\n",
      "Seq is: TTTTCCT\n",
      "4 TTTTCCT motifs detected in file\n",
      "\n",
      "Seq is: CCCGTTA\n",
      "0 CCCGTTA motifs detected in file\n",
      "\n",
      "Seq is: AAGATCT\n",
      "0 AAGATCT motifs detected in file\n",
      "\n",
      "Complement found from 575 to 582\n",
      "Complement found from 6628 to 6635\n",
      "Seq is: AGTTTAT\n",
      "2 AGTTTAT motifs detected in file\n",
      "\n",
      "Complement found from 3970 to 3977\n",
      "Complement found from 6634 to 6641\n",
      "Complement found from 15011 to 15018\n",
      "Complement found from 15338 to 15345\n",
      "Seq is: TATTCTT\n",
      "4 TATTCTT motifs detected in file\n",
      "\n",
      "Complement found from 5262 to 5269\n",
      "Complement found from 7957 to 7964\n",
      "Complement found from 12434 to 12441\n",
      "Seq is: CCATTAT\n",
      "3 CCATTAT motifs detected in file\n",
      "\n",
      "Seq is: CGATGAT\n",
      "0 CGATGAT motifs detected in file\n",
      "\n",
      "Complement found from 3293 to 3300\n",
      "Complement found from 6099 to 6106\n",
      "Complement found from 10221 to 10228\n",
      "Complement found from 11117 to 11124\n",
      "Complement found from 13105 to 13112\n",
      "Complement found from 14107 to 14114\n",
      "Seq is: TCTTCTT\n",
      "6 TCTTCTT motifs detected in file\n",
      "\n",
      "Complement found from 15868 to 15875\n",
      "Seq is: AAAATAC\n",
      "1 AAAATAC motifs detected in file\n",
      "\n",
      "Complement found from 14102 to 14109\n",
      "Complement found from 14349 to 14356\n",
      "Seq is: CTCTTTC\n",
      "2 CTCTTTC motifs detected in file\n",
      "\n",
      "Seq is: CCTGCGC\n",
      "0 CCTGCGC motifs detected in file\n",
      "\n",
      "Complement found from 3523 to 3530\n",
      "Seq is: CCGCCCC\n",
      "1 CCGCCCC motifs detected in file\n",
      "\n",
      "Complement found from 10185 to 10192\n",
      "Seq is: CTATATC\n",
      "1 CTATATC motifs detected in file\n",
      "\n",
      "Seq is: CCAGCTT\n",
      "0 CCAGCTT motifs detected in file\n",
      "\n",
      "Seq is: CGGCGTG\n",
      "0 CGGCGTG motifs detected in file\n",
      "\n",
      "Complement found from 4242 to 4249\n",
      "Complement found from 13307 to 13314\n",
      "Seq is: AGCATTC\n",
      "2 AGCATTC motifs detected in file\n",
      "\n",
      "Complement found from 8619 to 8626\n",
      "Complement found from 11140 to 11147\n",
      "Seq is: CCCACCT\n",
      "2 CCCACCT motifs detected in file\n",
      "\n",
      "Complement found from 5829 to 5836\n",
      "Complement found from 6693 to 6700\n",
      "Complement found from 10048 to 10055\n",
      "Seq is: AAAAAGA\n",
      "3 AAAAAGA motifs detected in file\n",
      "\n",
      "Seq is: TACATCG\n",
      "0 TACATCG motifs detected in file\n",
      "\n",
      "Complement found from 12317 to 12324\n",
      "Complement found from 16243 to 16250\n",
      "Seq is: GCAACTC\n",
      "2 GCAACTC motifs detected in file\n",
      "\n",
      "Complement found from 372 to 379\n",
      "Complement found from 6548 to 6555\n",
      "Complement found from 14739 to 14746\n",
      "Seq is: AACACCA\n",
      "3 AACACCA motifs detected in file\n",
      "\n",
      "Seq is: TCAGTTT\n",
      "0 TCAGTTT motifs detected in file\n",
      "\n",
      "Complement found from 16096 to 16103\n",
      "Seq is: TACATTA\n",
      "1 TACATTA motifs detected in file\n",
      "\n",
      "Complement found from 4754 to 4761\n",
      "Complement found from 7301 to 7308\n",
      "Complement found from 10076 to 10083\n",
      "Seq is: TTAATAA\n",
      "3 TTAATAA motifs detected in file\n",
      "\n",
      "Complement found from 4748 to 4755\n",
      "Complement found from 5295 to 5302\n",
      "Complement found from 10341 to 10348\n",
      "Seq is: TCATCAT\n",
      "3 TCATCAT motifs detected in file\n",
      "\n",
      "Complement found from 15587 to 15594\n",
      "Seq is: TCCGATC\n",
      "1 TCCGATC motifs detected in file\n",
      "\n",
      "Complement found from 458 to 465\n",
      "Complement found from 6310 to 6317\n",
      "Complement found from 11812 to 11819\n",
      "Seq is: CTCCCAC\n",
      "3 CTCCCAC motifs detected in file\n",
      "\n",
      "Complement found from 2292 to 2299\n",
      "Complement found from 7734 to 7741\n",
      "Complement found from 9870 to 9877\n",
      "Complement found from 10766 to 10773\n",
      "Complement found from 11099 to 11106\n",
      "Complement found from 15901 to 15908\n",
      "Seq is: AACTAAT\n",
      "6 AACTAAT motifs detected in file\n",
      "\n",
      "Complement found from 395 to 402\n",
      "Complement found from 9703 to 9710\n",
      "Complement found from 10072 to 10079\n",
      "Seq is: AATTTTA\n",
      "3 AATTTTA motifs detected in file\n",
      "\n",
      "Complement found from 4142 to 4149\n",
      "Seq is: ATTCCGC\n",
      "1 ATTCCGC motifs detected in file\n",
      "\n",
      "Complement found from 391 to 398\n",
      "Complement found from 3107 to 3114\n",
      "Complement found from 12687 to 12694\n",
      "Seq is: TTCAAAT\n",
      "3 TTCAAAT motifs detected in file\n",
      "\n",
      "Seq is: AATGAGA\n",
      "0 AATGAGA motifs detected in file\n",
      "\n",
      "Complement found from 993 to 1000\n",
      "Complement found from 4176 to 4183\n",
      "Complement found from 4869 to 4876\n",
      "Complement found from 11033 to 11040\n",
      "Complement found from 12420 to 12427\n",
      "Complement found from 14505 to 14512\n",
      "Seq is: AAAAACT\n",
      "6 AAAAACT motifs detected in file\n",
      "\n",
      "Seq is: TACAACA\n",
      "0 TACAACA motifs detected in file\n",
      "\n",
      "Complement found from 10032 to 10039\n",
      "Seq is: GTTTTGA\n",
      "1 GTTTTGA motifs detected in file\n",
      "\n",
      "Complement found from 5015 to 5022\n",
      "Complement found from 14167 to 14174\n",
      "Seq is: TCAATTA\n",
      "2 TCAATTA motifs detected in file\n",
      "\n",
      "Complement found from 14987 to 14994\n",
      "Seq is: ACCTTCA\n",
      "1 ACCTTCA motifs detected in file\n",
      "\n",
      "Complement found from 8760 to 8767\n",
      "Complement found from 10649 to 10656\n",
      "Seq is: ATTGCCA\n",
      "2 ATTGCCA motifs detected in file\n",
      "\n",
      "Complement found from 4273 to 4280\n",
      "Seq is: TGATAAA\n",
      "1 TGATAAA motifs detected in file\n",
      "\n",
      "Complement found from 3856 to 3863\n",
      "Seq is: TATGATT\n",
      "1 TATGATT motifs detected in file\n",
      "\n",
      "Complement found from 11660 to 11667\n",
      "Complement found from 11699 to 11706\n",
      "Complement found from 13732 to 13739\n",
      "Seq is: TTCTCAT\n",
      "3 TTCTCAT motifs detected in file\n",
      "\n",
      "Seq is: TCTGGGA\n",
      "0 TCTGGGA motifs detected in file\n",
      "\n",
      "Seq is: GTAGTTA\n",
      "0 GTAGTTA motifs detected in file\n",
      "\n",
      "Complement found from 5343 to 5350\n",
      "Complement found from 9184 to 9191\n",
      "Complement found from 11039 to 11046\n",
      "Complement found from 13786 to 13793\n",
      "Seq is: TCTACCT\n",
      "4 TCTACCT motifs detected in file\n",
      "\n",
      "Complement found from 2624 to 2631\n",
      "Complement found from 7699 to 7706\n",
      "Complement found from 9959 to 9966\n",
      "Seq is: CTGTATG\n",
      "3 CTGTATG motifs detected in file\n",
      "\n",
      "Complement found from 15014 to 15021\n",
      "Seq is: TCTTTAT\n",
      "1 TCTTTAT motifs detected in file\n",
      "\n",
      "Complement found from 5414 to 5421\n",
      "Seq is: CAGTTTG\n",
      "1 CAGTTTG motifs detected in file\n",
      "\n",
      "Seq is: GTGTAGA\n",
      "0 GTGTAGA motifs detected in file\n",
      "\n",
      "Complement found from 943 to 950\n",
      "Seq is: TGTTTTA\n",
      "1 TGTTTTA motifs detected in file\n",
      "\n",
      "Seq is: CGGTCAT\n",
      "0 CGGTCAT motifs detected in file\n",
      "\n",
      "Seq is: AGTCCGC\n",
      "0 AGTCCGC motifs detected in file\n",
      "\n",
      "Seq is: CGCCGAA\n",
      "0 CGCCGAA motifs detected in file\n",
      "\n",
      "Complement found from 6651 to 6658\n",
      "Seq is: GCTTCGG\n",
      "1 GCTTCGG motifs detected in file\n",
      "\n",
      "Complement found from 4799 to 4806\n",
      "Complement found from 7141 to 7148\n",
      "Complement found from 9308 to 9315\n",
      "Complement found from 9878 to 9885\n",
      "Complement found from 15286 to 15293\n",
      "Seq is: TTTCACT\n",
      "5 TTTCACT motifs detected in file\n",
      "\n",
      "Complement found from 1990 to 1997\n",
      "Complement found from 2764 to 2771\n",
      "Complement found from 5930 to 5937\n",
      "Complement found from 8507 to 8514\n",
      "Complement found from 14623 to 14630\n",
      "Complement found from 16280 to 16287\n",
      "Seq is: ACAAACC\n",
      "6 ACAAACC motifs detected in file\n",
      "\n",
      "Complement found from 1363 to 1370\n",
      "Complement found from 10117 to 10124\n",
      "Seq is: TACATTT\n",
      "2 TACATTT motifs detected in file\n",
      "\n",
      "Complement found from 5632 to 5639\n",
      "Complement found from 9881 to 9888\n",
      "Complement found from 15713 to 15720\n",
      "Seq is: CACTTTA\n",
      "3 CACTTTA motifs detected in file\n",
      "\n",
      "Complement found from 5708 to 5715\n",
      "Complement found from 8671 to 8678\n",
      "Complement found from 10080 to 10087\n",
      "Complement found from 14217 to 14224\n",
      "Seq is: TAATCAA\n",
      "4 TAATCAA motifs detected in file\n",
      "\n",
      "Seq is: CCGCCTA\n",
      "0 CCGCCTA motifs detected in file\n",
      "\n",
      "Complement found from 6451 to 6458\n",
      "Seq is: CTTCGTC\n",
      "1 CTTCGTC motifs detected in file\n",
      "\n",
      "Complement found from 11928 to 11935\n",
      "Seq is: TATCACT\n",
      "1 TATCACT motifs detected in file\n",
      "\n",
      "Complement found from 4797 to 4804\n",
      "Complement found from 7649 to 7656\n",
      "Complement found from 15284 to 15291\n",
      "Seq is: CCTTTCA\n",
      "3 CCTTTCA motifs detected in file\n",
      "\n",
      "Seq is: ACTCGGT\n",
      "0 ACTCGGT motifs detected in file\n",
      "\n",
      "Complement found from 9612 to 9619\n",
      "Complement found from 10787 to 10794\n",
      "Complement found from 13565 to 13572\n",
      "Complement found from 15626 to 15633\n",
      "Seq is: TATTACT\n",
      "4 TATTACT motifs detected in file\n",
      "\n",
      "Seq is: GCACTTC\n",
      "0 GCACTTC motifs detected in file\n",
      "\n",
      "Complement found from 2534 to 2541\n",
      "Complement found from 4828 to 4835\n",
      "Seq is: AGGCACC\n",
      "2 AGGCACC motifs detected in file\n",
      "\n",
      "Seq is: TTCGGAC\n",
      "0 TTCGGAC motifs detected in file\n",
      "\n",
      "Seq is: ACCGATG\n",
      "0 ACCGATG motifs detected in file\n",
      "\n",
      "Complement found from 4547 to 4554\n",
      "Complement found from 10879 to 10886\n",
      "Seq is: TTTTTTA\n",
      "2 TTTTTTA motifs detected in file\n",
      "\n",
      "Seq is: CAACTCG\n",
      "0 CAACTCG motifs detected in file\n",
      "\n",
      "Complement found from 9777 to 9784\n",
      "Seq is: GCATCTA\n",
      "1 GCATCTA motifs detected in file\n",
      "\n",
      "Seq is: CGGCAGC\n",
      "0 CGGCAGC motifs detected in file\n",
      "\n",
      "Complement found from 15279 to 15286\n",
      "Seq is: CTTTACC\n",
      "1 CTTTACC motifs detected in file\n",
      "\n",
      "Complement found from 7621 to 7628\n",
      "Complement found from 7950 to 7957\n",
      "Complement found from 12372 to 12379\n",
      "Seq is: ACTTCCC\n",
      "3 ACTTCCC motifs detected in file\n",
      "\n",
      "Complement found from 1665 to 1672\n",
      "Complement found from 6723 to 6730\n",
      "Complement found from 6918 to 6925\n",
      "Complement found from 9498 to 9505\n",
      "Seq is: TCTGAGC\n",
      "4 TCTGAGC motifs detected in file\n",
      "\n",
      "Complement found from 7310 to 7317\n",
      "Complement found from 7652 to 7659\n",
      "Seq is: TTCATGA\n",
      "2 TTCATGA motifs detected in file\n",
      "\n",
      "Seq is: GACTCCG\n",
      "0 GACTCCG motifs detected in file\n",
      "\n",
      "Complement found from 7253 to 7260\n",
      "Seq is: AACATCC\n",
      "1 AACATCC motifs detected in file\n",
      "\n",
      "Complement found from 15382 to 15389\n",
      "Seq is: TCCCATT\n",
      "1 TCCCATT motifs detected in file\n",
      "\n",
      "Complement found from 3862 to 3869\n",
      "Complement found from 12520 to 12527\n",
      "Seq is: TTATCTC\n",
      "2 TTATCTC motifs detected in file\n",
      "\n",
      "Complement found from 11436 to 11443\n",
      "Seq is: TGGGTCA\n",
      "1 TGGGTCA motifs detected in file\n",
      "\n",
      "Complement found from 420 to 427\n",
      "Complement found from 2663 to 2670\n",
      "Complement found from 10881 to 10888\n",
      "Seq is: TTTTAAC\n",
      "3 TTTTAAC motifs detected in file\n",
      "\n",
      "Complement found from 5962 to 5969\n",
      "Complement found from 7959 to 7966\n",
      "Complement found from 14296 to 14303\n",
      "Complement found from 14671 to 14678\n",
      "Seq is: ATTATTC\n",
      "4 ATTATTC motifs detected in file\n",
      "\n",
      "Seq is: GTCACGA\n",
      "0 GTCACGA motifs detected in file\n",
      "\n",
      "Complement found from 5567 to 5574\n",
      "Complement found from 11336 to 11343\n",
      "Seq is: ACTTAAT\n",
      "2 ACTTAAT motifs detected in file\n",
      "\n",
      "Complement found from 4978 to 4985\n",
      "Complement found from 8121 to 8128\n",
      "Complement found from 9673 to 9680\n",
      "Seq is: AAACCAA\n",
      "3 AAACCAA motifs detected in file\n",
      "\n",
      "Complement found from 7484 to 7491\n",
      "Complement found from 10996 to 11003\n",
      "Complement found from 11328 to 11335\n",
      "Complement found from 15519 to 15526\n",
      "Seq is: AGCCAAC\n",
      "4 AGCCAAC motifs detected in file\n",
      "\n",
      "Seq is: AAAAGAT\n",
      "0 AAAAGAT motifs detected in file\n",
      "\n",
      "Complement found from 12170 to 12177\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Complement found from 15234 to 15241\n",
      "Seq is: AATCTGA\n",
      "2 AATCTGA motifs detected in file\n",
      "\n",
      "Complement found from 4186 to 4193\n",
      "Complement found from 9509 to 9516\n",
      "Complement found from 10793 to 10800\n",
      "Seq is: TACCACT\n",
      "3 TACCACT motifs detected in file\n",
      "\n",
      "Seq is: CGAGTCC\n",
      "0 CGAGTCC motifs detected in file\n",
      "\n",
      "Complement found from 7731 to 7738\n",
      "Complement found from 10959 to 10966\n",
      "Seq is: ACTAACT\n",
      "2 ACTAACT motifs detected in file\n",
      "\n",
      "Seq is: ATGAGGA\n",
      "0 ATGAGGA motifs detected in file\n",
      "\n",
      "Seq is: CTACGCT\n",
      "0 CTACGCT motifs detected in file\n",
      "\n",
      "Complement found from 12435 to 12442\n",
      "Seq is: CATTATG\n",
      "1 CATTATG motifs detected in file\n",
      "\n",
      "Complement found from 8925 to 8932\n",
      "Complement found from 12905 to 12912\n",
      "Complement found from 14308 to 14315\n",
      "Complement found from 15030 to 15037\n",
      "Complement found from 15575 to 15582\n",
      "Seq is: CCTACAC\n",
      "5 CCTACAC motifs detected in file\n",
      "\n",
      "Complement found from 7568 to 7575\n",
      "Seq is: AAATCCT\n",
      "1 AAATCCT motifs detected in file\n",
      "\n",
      "Complement found from 6314 to 6321\n",
      "Complement found from 6620 to 6627\n",
      "Seq is: CACCCTG\n",
      "2 CACCCTG motifs detected in file\n",
      "\n",
      "Seq is: AAGATCC\n",
      "0 AAGATCC motifs detected in file\n",
      "\n",
      "Complement found from 14491 to 14498\n",
      "Complement found from 16526 to 16533\n",
      "Seq is: CCTAAAT\n",
      "2 CCTAAAT motifs detected in file\n",
      "\n",
      "Seq is: CGCCCCA\n",
      "0 CGCCCCA motifs detected in file\n",
      "\n",
      "Seq is: CTACGGA\n",
      "0 CTACGGA motifs detected in file\n",
      "\n",
      "Complement found from 490 to 497\n",
      "Complement found from 3581 to 3588\n",
      "Complement found from 7487 to 7494\n",
      "Complement found from 10943 to 10950\n",
      "Complement found from 11508 to 11515\n",
      "Complement found from 12105 to 12112\n",
      "Complement found from 14414 to 14421\n",
      "Complement found from 15363 to 15370\n",
      "Complement found from 15522 to 15529\n",
      "Seq is: CAACCCC\n",
      "9 CAACCCC motifs detected in file\n",
      "\n",
      "Complement found from 899 to 906\n",
      "Complement found from 6616 to 6623\n",
      "Seq is: CGGTCAC\n",
      "2 CGGTCAC motifs detected in file\n",
      "\n",
      "Seq is: GGGAGGA\n",
      "0 GGGAGGA motifs detected in file\n",
      "\n",
      "Complement found from 910 to 917\n",
      "Complement found from 12406 to 12413\n",
      "Seq is: TTAACCC\n",
      "2 TTAACCC motifs detected in file\n",
      "\n",
      "Complement found from 499 to 506\n",
      "Complement found from 4348 to 4355\n",
      "Complement found from 4391 to 4398\n",
      "Complement found from 7818 to 7825\n",
      "Complement found from 12387 to 12394\n",
      "Complement found from 14814 to 14821\n",
      "Complement found from 15658 to 15665\n",
      "Seq is: CCCATCC\n",
      "7 CCCATCC motifs detected in file\n",
      "\n",
      "Complement found from 15588 to 15595\n",
      "Seq is: CCGATCC\n",
      "1 CCGATCC motifs detected in file\n",
      "\n",
      "Complement found from 13842 to 13849\n",
      "Complement found from 14409 to 14416\n",
      "Seq is: GACCTCA\n",
      "2 GACCTCA motifs detected in file\n",
      "\n",
      "Complement found from 5881 to 5888\n",
      "Seq is: CCATTTT\n",
      "1 CCATTTT motifs detected in file\n",
      "\n",
      "Seq is: CCGAGTC\n",
      "0 CCGAGTC motifs detected in file\n",
      "\n",
      "Seq is: TTTTTAG\n",
      "0 TTTTTAG motifs detected in file\n",
      "\n",
      "Seq is: GACGCTT\n",
      "0 GACGCTT motifs detected in file\n",
      "\n",
      "Seq is: CTTTTTT\n",
      "0 CTTTTTT motifs detected in file\n",
      "\n",
      "Seq is: TGAAGAC\n",
      "0 TGAAGAC motifs detected in file\n",
      "\n",
      "Seq is: TCCATTA\n",
      "0 TCCATTA motifs detected in file\n",
      "\n",
      "Complement found from 12900 to 12907\n",
      "Seq is: TTTATCC\n",
      "1 TTTATCC motifs detected in file\n",
      "\n",
      "Complement found from 4302 to 4309\n",
      "Seq is: TAATAGG\n",
      "1 TAATAGG motifs detected in file\n",
      "\n",
      "Seq is: GCATAGC\n",
      "0 GCATAGC motifs detected in file\n",
      "\n",
      "Seq is: TTCCGGA\n",
      "0 TTCCGGA motifs detected in file\n",
      "\n",
      "Complement found from 2131 to 2138\n",
      "Complement found from 13972 to 13979\n",
      "Complement found from 15933 to 15940\n",
      "Seq is: AAAACCT\n",
      "3 AAAACCT motifs detected in file\n",
      "\n",
      "Seq is: GTCCACC\n",
      "0 GTCCACC motifs detected in file\n",
      "\n",
      "Complement found from 518 to 525\n",
      "Complement found from 1477 to 1484\n",
      "Complement found from 9417 to 9424\n",
      "Complement found from 13932 to 13939\n",
      "Complement found from 14551 to 14558\n",
      "Seq is: ACACACC\n",
      "5 ACACACC motifs detected in file\n",
      "\n",
      "Seq is: ACCGGAA\n",
      "0 ACCGGAA motifs detected in file\n",
      "\n",
      "Complement found from 8701 to 8708\n",
      "Complement found from 15205 to 15212\n",
      "Seq is: CCATACA\n",
      "2 CCATACA motifs detected in file\n",
      "\n",
      "Seq is: CGAAAGA\n",
      "0 CGAAAGA motifs detected in file\n",
      "\n",
      "Complement found from 13939 to 13946\n",
      "Seq is: GCACAAT\n",
      "1 GCACAAT motifs detected in file\n",
      "\n",
      "Seq is: GTAGCGC\n",
      "0 GTAGCGC motifs detected in file\n",
      "\n",
      "Seq is: GGTAGTA\n",
      "0 GGTAGTA motifs detected in file\n",
      "\n",
      "Complement found from 6771 to 6778\n",
      "Complement found from 9362 to 9369\n",
      "Seq is: ACCATAT\n",
      "2 ACCATAT motifs detected in file\n",
      "\n",
      "Complement found from 2269 to 2276\n",
      "Complement found from 4736 to 4743\n",
      "Complement found from 9213 to 9220\n",
      "Complement found from 14365 to 14372\n",
      "Seq is: ACCAATC\n",
      "4 ACCAATC motifs detected in file\n",
      "\n",
      "Complement found from 4069 to 4076\n",
      "Complement found from 8705 to 8712\n",
      "Complement found from 10835 to 10842\n",
      "Seq is: ACACAAC\n",
      "3 ACACAAC motifs detected in file\n",
      "\n",
      "Complement found from 4887 to 4894\n",
      "Complement found from 10985 to 10992\n",
      "Seq is: CAATCAT\n",
      "2 CAATCAT motifs detected in file\n",
      "\n",
      "Seq is: CCGGATC\n",
      "0 CCGGATC motifs detected in file\n",
      "\n",
      "Complement found from 5008 to 5015\n",
      "Complement found from 8410 to 8417\n",
      "Complement found from 11973 to 11980\n",
      "Complement found from 14439 to 14446\n",
      "Seq is: ATACTCC\n",
      "4 ATACTCC motifs detected in file\n",
      "\n",
      "Complement found from 3349 to 3356\n",
      "Seq is: TCGCAAT\n",
      "1 TCGCAAT motifs detected in file\n",
      "\n",
      "Complement found from 1231 to 1238\n",
      "Complement found from 5888 to 5895\n",
      "Seq is: ACCTCAC\n",
      "2 ACCTCAC motifs detected in file\n",
      "\n",
      "Complement found from 5871 to 5878\n",
      "Seq is: CTTCACT\n",
      "1 CTTCACT motifs detected in file\n",
      "\n",
      "Seq is: TTAGTGT\n",
      "0 TTAGTGT motifs detected in file\n",
      "\n",
      "Seq is: ACCGATC\n",
      "0 ACCGATC motifs detected in file\n",
      "\n",
      "Seq is: CGGCCTC\n",
      "0 CGGCCTC motifs detected in file\n",
      "\n",
      "Complement found from 7293 to 7300\n",
      "Complement found from 15176 to 15183\n",
      "Seq is: CAGTAAT\n",
      "2 CAGTAAT motifs detected in file\n",
      "\n",
      "Complement found from 14261 to 14268\n",
      "Complement found from 15141 to 15148\n",
      "Seq is: CCTCCCG\n",
      "2 CCTCCCG motifs detected in file\n",
      "\n",
      "Complement found from 4052 to 4059\n",
      "Complement found from 4705 to 4712\n",
      "Complement found from 11932 to 11939\n",
      "Complement found from 16440 to 16447\n",
      "Seq is: ACTCTCC\n",
      "4 ACTCTCC motifs detected in file\n",
      "\n",
      "Seq is: CGGGGAC\n",
      "0 CGGGGAC motifs detected in file\n",
      "\n",
      "Complement found from 2072 to 2079\n",
      "Complement found from 4472 to 4479\n",
      "Complement found from 9578 to 9585\n",
      "Complement found from 13778 to 13785\n",
      "Complement found from 13876 to 13883\n",
      "Complement found from 13943 to 13950\n",
      "Complement found from 15654 to 15661\n",
      "Seq is: AATCCCC\n",
      "7 AATCCCC motifs detected in file\n",
      "\n",
      "Complement found from 1305 to 1312\n",
      "Seq is: AGTACCC\n",
      "1 AGTACCC motifs detected in file\n",
      "\n",
      "Complement found from 1061 to 1068\n",
      "Complement found from 12666 to 12673\n",
      "Seq is: GACCCAA\n",
      "2 GACCCAA motifs detected in file\n",
      "\n",
      "Complement found from 3175 to 3182\n",
      "Complement found from 8693 to 8700\n",
      "Complement found from 14701 to 14708\n",
      "Complement found from 15557 to 15564\n",
      "Seq is: AATGATA\n",
      "4 AATGATA motifs detected in file\n",
      "\n",
      "Complement found from 3754 to 3761\n",
      "Complement found from 5154 to 5161\n",
      "Complement found from 10578 to 10585\n",
      "Complement found from 11546 to 11553\n",
      "Complement found from 15191 to 15198\n",
      "Complement found from 15629 to 15636\n",
      "Seq is: TACTATC\n",
      "6 TACTATC motifs detected in file\n",
      "\n",
      "Complement found from 1474 to 1481\n",
      "Seq is: CGTACAC\n",
      "1 CGTACAC motifs detected in file\n",
      "\n",
      "Complement found from 22 to 29\n",
      "Complement found from 11906 to 11913\n",
      "Complement found from 12355 to 12362\n",
      "Complement found from 14787 to 14794\n",
      "Seq is: TAACCAC\n",
      "4 TAACCAC motifs detected in file\n",
      "\n",
      "Complement found from 14203 to 14210\n",
      "Complement found from 15769 to 15776\n",
      "Seq is: CCAGTAA\n",
      "2 CCAGTAA motifs detected in file\n",
      "\n",
      "Complement found from 3987 to 3994\n",
      "Complement found from 4068 to 4075\n",
      "Complement found from 8704 to 8711\n",
      "Complement found from 11353 to 11360\n",
      "Complement found from 13214 to 13221\n",
      "Complement found from 13542 to 13549\n",
      "Complement found from 15415 to 15422\n",
      "Complement found from 15577 to 15584\n",
      "Seq is: TACACAA\n",
      "8 TACACAA motifs detected in file\n",
      "\n",
      "Seq is: CAGCACG\n",
      "0 CAGCACG motifs detected in file\n",
      "\n",
      "Complement found from 9968 to 9975\n",
      "Complement found from 11578 to 11585\n",
      "Seq is: TCCATCT\n",
      "2 TCCATCT motifs detected in file\n",
      "\n",
      "Complement found from 15554 to 15561\n",
      "Seq is: CCGAATG\n",
      "1 CCGAATG motifs detected in file\n",
      "\n",
      "Complement found from 1265 to 1272\n",
      "Complement found from 7933 to 7940\n",
      "Complement found from 9099 to 9106\n",
      "Seq is: ATCTTCA\n",
      "3 ATCTTCA motifs detected in file\n",
      "\n",
      "Complement found from 417 to 424\n",
      "Seq is: CACTTTT\n",
      "1 CACTTTT motifs detected in file\n",
      "\n",
      "Complement found from 15805 to 15812\n",
      "Seq is: GCATCCG\n",
      "1 GCATCCG motifs detected in file\n",
      "\n",
      "Complement found from 4256 to 4263\n",
      "Complement found from 10742 to 10749\n",
      "Seq is: AACCTAA\n",
      "2 AACCTAA motifs detected in file\n",
      "\n",
      "Complement found from 4143 to 4150\n",
      "Seq is: TTCCGCT\n",
      "1 TTCCGCT motifs detected in file\n",
      "\n",
      "Seq is: GTCGCTA\n",
      "0 GTCGCTA motifs detected in file\n",
      "\n",
      "Seq is: AAGATCC\n",
      "0 AAGATCC motifs detected in file\n",
      "\n",
      "Complement found from 12470 to 12477\n",
      "Seq is: TATCAGT\n",
      "1 TATCAGT motifs detected in file\n",
      "\n",
      "Seq is: CATCTTA\n",
      "0 CATCTTA motifs detected in file\n",
      "\n",
      "Complement found from 7579 to 7586\n",
      "Seq is: ATCTTAA\n",
      "1 ATCTTAA motifs detected in file\n",
      "\n",
      "Complement found from 4802 to 4809\n",
      "Complement found from 8904 to 8911\n",
      "Complement found from 9169 to 9176\n",
      "Complement found from 10512 to 10519\n",
      "Seq is: CACTTCT\n",
      "4 CACTTCT motifs detected in file\n",
      "\n",
      "Complement found from 5920 to 5927\n",
      "Complement found from 8421 to 8428\n",
      "Complement found from 11733 to 11740\n",
      "Complement found from 14889 to 14896\n",
      "Complement found from 15474 to 15481\n",
      "Complement found from 16012 to 16019\n",
      "Seq is: ACTATTC\n",
      "6 ACTATTC motifs detected in file\n",
      "\n",
      "Complement found from 338 to 345\n",
      "Complement found from 7745 to 7752\n",
      "Complement found from 14822 to 14829\n",
      "Seq is: ACATCTC\n",
      "3 ACATCTC motifs detected in file\n",
      "\n",
      "Complement found from 13765 to 13772\n",
      "Complement found from 15401 to 15408\n",
      "Seq is: CCTTCCA\n",
      "2 CCTTCCA motifs detected in file\n",
      "\n",
      "Complement found from 12711 to 12718\n",
      "Seq is: ATTACCA\n",
      "1 ATTACCA motifs detected in file\n",
      "\n",
      "Complement found from 12351 to 12358\n",
      "Seq is: ACTATAA\n",
      "1 ACTATAA motifs detected in file\n",
      "\n",
      "Complement found from 7857 to 7864\n",
      "Seq is: CGATCCC\n",
      "1 CGATCCC motifs detected in file\n",
      "\n",
      "Seq is: ACCGCGA\n",
      "0 ACCGCGA motifs detected in file\n",
      "\n",
      "Complement found from 2451 to 2458\n",
      "Complement found from 5519 to 5526\n",
      "Seq is: AGGTTAA\n",
      "2 AGGTTAA motifs detected in file\n",
      "\n",
      "Seq is: TCCACGT\n",
      "0 TCCACGT motifs detected in file\n",
      "\n",
      "Complement found from 4519 to 4526\n",
      "Complement found from 4746 to 4753\n",
      "Complement found from 5452 to 5459\n",
      "Complement found from 6981 to 6988\n",
      "Complement found from 11238 to 11245\n",
      "Complement found from 12698 to 12705\n",
      "Complement found from 13112 to 13119\n",
      "Complement found from 14117 to 14124\n",
      "Seq is: ACTCATC\n",
      "8 ACTCATC motifs detected in file\n",
      "\n",
      "Seq is: CACCGGA\n",
      "0 CACCGGA motifs detected in file\n",
      "\n",
      "Complement found from 9826 to 9833\n",
      "Seq is: CGTCATT\n",
      "1 CGTCATT motifs detected in file\n",
      "\n",
      "Complement found from 8000 to 8007\n",
      "Complement found from 13281 to 13288\n",
      "Seq is: ACAATCG\n",
      "2 ACAATCG motifs detected in file\n",
      "\n",
      "Complement found from 14366 to 14373\n",
      "Complement found from 16167 to 16174\n",
      "Seq is: CCAATCC\n",
      "2 CCAATCC motifs detected in file\n",
      "\n",
      "Seq is: GCATGCC\n",
      "0 GCATGCC motifs detected in file\n",
      "\n",
      "Seq is: GGTCCCG\n",
      "0 GGTCCCG motifs detected in file\n",
      "\n",
      "Complement found from 1807 to 1814\n",
      "Complement found from 4724 to 4731\n",
      "Complement found from 5066 to 5073\n",
      "Complement found from 8697 to 8704\n",
      "Complement found from 12336 to 12343\n",
      "Complement found from 12354 to 12361\n",
      "Seq is: ATAACCA\n",
      "6 ATAACCA motifs detected in file\n",
      "\n",
      "Complement found from 192 to 199\n",
      "Complement found from 6677 to 6684\n",
      "Complement found from 15188 to 15195\n",
      "Seq is: ACTTACT\n",
      "3 ACTTACT motifs detected in file\n",
      "\n",
      "Seq is: CATCCGA\n",
      "0 CATCCGA motifs detected in file\n",
      "\n",
      "Complement found from 2784 to 2791\n",
      "Complement found from 4734 to 4741\n",
      "Seq is: CTACCAA\n",
      "2 CTACCAA motifs detected in file\n",
      "\n",
      "Complement found from 10231 to 10238\n",
      "Complement found from 14014 to 14021\n",
      "Seq is: AGCTATT\n",
      "2 AGCTATT motifs detected in file\n",
      "\n",
      "Complement found from 2892 to 2899\n",
      "Complement found from 8612 to 8619\n",
      "Seq is: ATTGATC\n",
      "2 ATTGATC motifs detected in file\n",
      "\n",
      "Complement found from 537 to 544\n",
      "Complement found from 1079 to 1086\n",
      "Complement found from 4133 to 4140\n",
      "Complement found from 4432 to 4439\n",
      "Complement found from 12428 to 12435\n",
      "Seq is: ATACCCC\n",
      "5 ATACCCC motifs detected in file\n",
      "\n",
      "Complement found from 14150 to 14157\n",
      "Seq is: TATTCCC\n",
      "1 TATTCCC motifs detected in file\n",
      "\n",
      "Complement found from 9604 to 9611\n",
      "Seq is: CACATCC\n",
      "1 CACATCC motifs detected in file\n",
      "\n",
      "Seq is: ACACGGA\n",
      "0 ACACGGA motifs detected in file\n",
      "\n",
      "Complement found from 6769 to 6776\n",
      "Seq is: ACACCAT\n",
      "1 ACACCAT motifs detected in file\n",
      "\n",
      "Complement found from 3469 to 3476\n",
      "Complement found from 9100 to 9107\n",
      "Seq is: TCTTCAC\n",
      "2 TCTTCAC motifs detected in file\n",
      "\n",
      "Complement found from 15588 to 15595\n",
      "Seq is: CCGATCC\n",
      "1 CCGATCC motifs detected in file\n",
      "\n",
      "Complement found from 5398 to 5405\n",
      "Seq is: CGTAAAA\n",
      "1 CGTAAAA motifs detected in file\n",
      "\n",
      "Complement found from 7522 to 7529\n",
      "Seq is: ATTAGAA\n",
      "1 ATTAGAA motifs detected in file\n",
      "\n",
      "Seq is: TCTCGCT\n",
      "0 TCTCGCT motifs detected in file\n",
      "\n",
      "Seq is: TTCGGAT\n",
      "0 TTCGGAT motifs detected in file\n",
      "\n",
      "Seq is: ATTAGTG\n",
      "0 ATTAGTG motifs detected in file\n",
      "\n",
      "Seq is: ACACGGG\n",
      "0 ACACGGG motifs detected in file\n",
      "\n",
      "Seq is: CGAGTAT\n",
      "0 CGAGTAT motifs detected in file\n",
      "\n",
      "Complement found from 6151 to 6158\n",
      "Complement found from 7093 to 7100\n",
      "Complement found from 7624 to 7631\n",
      "Complement found from 8226 to 8233\n",
      "Complement found from 9580 to 9587\n",
      "Complement found from 11232 to 11239\n",
      "Complement found from 13945 to 13952\n",
      "Seq is: TCCCCTA\n",
      "7 TCCCCTA motifs detected in file\n",
      "\n",
      "Seq is: CCGGTAC\n",
      "0 CCGGTAC motifs detected in file\n",
      "\n",
      "Seq is: CGATCCA\n",
      "0 CGATCCA motifs detected in file\n",
      "\n",
      "Complement found from 2380 to 2387\n",
      "Complement found from 8629 to 8636\n",
      "Complement found from 12691 to 12698\n",
      "Seq is: AATATCT\n",
      "3 AATATCT motifs detected in file\n",
      "\n",
      "Complement found from 1227 to 1234\n",
      "Complement found from 2390 to 2397\n",
      "Complement found from 9068 to 9075\n",
      "Complement found from 13290 to 13297\n",
      "Complement found from 14269 to 14276\n",
      "Complement found from 16215 to 16222\n",
      "Seq is: ATCAACC\n",
      "6 ATCAACC motifs detected in file\n",
      "\n",
      "Complement found from 8803 to 8810\n",
      "Complement found from 14183 to 14190\n",
      "Seq is: CACCAAC\n",
      "2 CACCAAC motifs detected in file\n",
      "\n",
      "Complement found from 13889 to 13896\n",
      "Seq is: CACATTT\n",
      "1 CACATTT motifs detected in file\n",
      "\n",
      "Complement found from 11292 to 11299\n",
      "Seq is: ACTCACT\n",
      "1 ACTCACT motifs detected in file\n",
      "\n",
      "Complement found from 6462 to 6469\n",
      "Seq is: CCGTCCT\n",
      "1 CCGTCCT motifs detected in file\n",
      "\n",
      "Seq is: CAGGCTC\n",
      "0 CAGGCTC motifs detected in file\n",
      "\n",
      "Complement found from 10754 to 10761\n",
      "Seq is: CTCCAAT\n",
      "1 CTCCAAT motifs detected in file\n",
      "\n",
      "Complement found from 2211 to 2218\n",
      "Complement found from 6547 to 6554\n",
      "Complement found from 6596 to 6603\n",
      "Complement found from 8345 to 8352\n",
      "Complement found from 10084 to 10091\n",
      "Complement found from 10615 to 10622\n",
      "Complement found from 11262 to 11269\n",
      "Seq is: CAACACC\n",
      "7 CAACACC motifs detected in file\n",
      "\n",
      "Seq is: GTTATTT\n",
      "0 GTTATTT motifs detected in file\n",
      "\n",
      "Complement found from 194 to 201\n",
      "Complement found from 3765 to 3772\n",
      "Complement found from 13654 to 13661\n",
      "Complement found from 13738 to 13745\n",
      "Complement found from 14633 to 14640\n",
      "Seq is: TTACTAA\n",
      "5 TTACTAA motifs detected in file\n",
      "\n",
      "Complement found from 11203 to 11210\n",
      "Complement found from 15565 to 15572\n",
      "Seq is: TTCCTAT\n",
      "2 TTCCTAT motifs detected in file\n",
      "\n",
      "Seq is: GATCATA\n",
      "0 GATCATA motifs detected in file\n",
      "\n",
      "Complement found from 5924 to 5931\n",
      "Complement found from 7282 to 7289\n",
      "Complement found from 15065 to 15072\n",
      "Seq is: TTCTCTA\n",
      "3 TTCTCTA motifs detected in file\n",
      "\n",
      "Complement found from 2021 to 2028\n",
      "Seq is: GTCCAAG\n",
      "1 GTCCAAG motifs detected in file\n",
      "\n",
      "Complement found from 7759 to 7766\n",
      "Seq is: CAGGAAA\n",
      "1 CAGGAAA motifs detected in file\n",
      "\n",
      "Complement found from 3802 to 3809\n",
      "Complement found from 6396 to 6403\n",
      "Complement found from 7721 to 7728\n",
      "Complement found from 11259 to 11266\n",
      "Complement found from 15821 to 15828\n",
      "Seq is: TCACAAC\n",
      "5 TCACAAC motifs detected in file\n",
      "\n",
      "Complement found from 3304 to 3311\n",
      "Complement found from 9401 to 9408\n",
      "Seq is: ACATACC\n",
      "2 ACATACC motifs detected in file\n",
      "\n",
      "Complement found from 7351 to 7358\n",
      "Complement found from 10004 to 10011\n",
      "Complement found from 11442 to 11449\n",
      "Seq is: AATAGTA\n",
      "3 AATAGTA motifs detected in file\n",
      "\n",
      "Complement found from 282 to 289\n",
      "Complement found from 1120 to 1127\n",
      "Complement found from 7725 to 7732\n",
      "Complement found from 10418 to 10425\n",
      "Complement found from 12414 to 12421\n",
      "Complement found from 15865 to 15872\n",
      "Seq is: AACAAAA\n",
      "6 AACAAAA motifs detected in file\n",
      "\n",
      "Complement found from 10754 to 10761\n",
      "Seq is: CTCCAAT\n",
      "1 CTCCAAT motifs detected in file\n",
      "\n",
      "Complement found from 3679 to 3686\n",
      "Complement found from 8456 to 8463\n",
      "Complement found from 8675 to 8682\n",
      "Complement found from 11753 to 11760\n",
      "Complement found from 12575 to 12582\n",
      "Complement found from 15603 to 15610\n",
      "Seq is: CAAACTA\n",
      "6 CAAACTA motifs detected in file\n",
      "\n",
      "Seq is: ATCAGTA\n",
      "0 ATCAGTA motifs detected in file\n",
      "\n",
      "Complement found from 489 to 496\n",
      "Complement found from 3443 to 3450\n",
      "Complement found from 5055 to 5062\n",
      "Complement found from 10942 to 10949\n",
      "Complement found from 12543 to 12550\n",
      "Complement found from 12552 to 12559\n",
      "Complement found from 15362 to 15369\n",
      "Seq is: ACAACCC\n",
      "7 ACAACCC motifs detected in file\n",
      "\n",
      "Complement found from 4017 to 4024\n",
      "Complement found from 11019 to 11026\n",
      "Seq is: ACCACTA\n",
      "2 ACCACTA motifs detected in file\n",
      "\n",
      "Complement found from 5626 to 5633\n",
      "Complement found from 8964 to 8971\n",
      "Complement found from 11629 to 11636\n",
      "Complement found from 13004 to 13011\n",
      "Seq is: ATCAGCC\n",
      "4 ATCAGCC motifs detected in file\n",
      "\n",
      "Complement found from 10297 to 10304\n",
      "Complement found from 11332 to 11339\n",
      "Seq is: AACAACT\n",
      "2 AACAACT motifs detected in file\n",
      "\n",
      "Seq is: ACTCTAT\n",
      "0 ACTCTAT motifs detected in file\n",
      "\n",
      "Complement found from 1101 to 1108\n",
      "Seq is: AAACCTC\n",
      "1 AAACCTC motifs detected in file\n",
      "\n",
      "Complement found from 4831 to 4838\n",
      "Complement found from 8929 to 8936\n",
      "Complement found from 10159 to 10166\n",
      "Complement found from 13023 to 13030\n",
      "Complement found from 15534 to 15541\n",
      "Complement found from 16256 to 16263\n",
      "Seq is: CACCCCT\n",
      "6 CACCCCT motifs detected in file\n",
      "\n",
      "Seq is: AGTTATC\n",
      "0 AGTTATC motifs detected in file\n",
      "\n",
      "Complement found from 4247 to 4254\n",
      "Complement found from 8603 to 8610\n",
      "Complement found from 13761 to 13768\n",
      "Complement found from 13780 to 13787\n",
      "Complement found from 14487 to 14494\n",
      "Seq is: TCCCCCT\n",
      "5 TCCCCCT motifs detected in file\n",
      "\n",
      "Complement found from 399 to 406\n",
      "Seq is: TTATCTT\n",
      "1 TTATCTT motifs detected in file\n",
      "\n",
      "Complement found from 3923 to 3930\n",
      "Complement found from 10657 to 10664\n",
      "Complement found from 10699 to 10706\n",
      "Complement found from 11958 to 11965\n",
      "Seq is: ACTAGTC\n",
      "4 ACTAGTC motifs detected in file\n",
      "\n",
      "Complement found from 4045 to 4052\n",
      "Seq is: ATGACGC\n",
      "1 ATGACGC motifs detected in file\n",
      "\n",
      "Complement found from 497 to 504\n",
      "Complement found from 14224 to 14231\n",
      "Seq is: CGCCCAT\n",
      "2 CGCCCAT motifs detected in file\n",
      "\n",
      "Seq is: CCGGATC\n",
      "0 CCGGATC motifs detected in file\n",
      "\n",
      "Complement found from 370 to 377\n",
      "Complement found from 440 to 447\n",
      "Complement found from 5186 to 5193\n",
      "Complement found from 7714 to 7721\n",
      "Complement found from 13161 to 13168\n",
      "Seq is: CTAACAC\n",
      "5 CTAACAC motifs detected in file\n",
      "\n",
      "Complement found from 704 to 711\n",
      "Seq is: CCCCGTT\n",
      "1 CCCCGTT motifs detected in file\n",
      "\n",
      "Complement found from 192 to 199\n",
      "Complement found from 6677 to 6684\n",
      "Complement found from 15188 to 15195\n",
      "Seq is: ACTTACT\n",
      "3 ACTTACT motifs detected in file\n",
      "\n",
      "Seq is: TCGCTTG\n",
      "0 TCGCTTG motifs detected in file\n",
      "\n",
      "Seq is: TCTAAAA\n",
      "0 TCTAAAA motifs detected in file\n",
      "\n",
      "Complement found from 1302 to 1309\n",
      "Complement found from 7599 to 7606\n",
      "Complement found from 16203 to 16210\n",
      "Seq is: GCAAGTA\n",
      "3 GCAAGTA motifs detected in file\n",
      "\n",
      "Complement found from 4978 to 4985\n",
      "Complement found from 8121 to 8128\n",
      "Complement found from 9673 to 9680\n",
      "Seq is: AAACCAA\n",
      "3 AAACCAA motifs detected in file\n",
      "\n",
      "Complement found from 4809 to 4816\n",
      "Seq is: GAGTCCC\n",
      "1 GAGTCCC motifs detected in file\n",
      "\n",
      "Complement found from 3718 to 3725\n",
      "Complement found from 5284 to 5291\n",
      "Complement found from 14026 to 14033\n",
      "Complement found from 14190 to 14197\n",
      "Seq is: AAACAAT\n",
      "4 AAACAAT motifs detected in file\n",
      "\n",
      "Complement found from 2349 to 2356\n",
      "Complement found from 2797 to 2804\n",
      "Complement found from 8886 to 8893\n",
      "Complement found from 14500 to 14507\n",
      "Seq is: ATTAAAA\n",
      "4 ATTAAAA motifs detected in file\n",
      "\n",
      "Complement found from 4777 to 4784\n",
      "Complement found from 5404 to 5411\n",
      "Complement found from 8493 to 8500\n",
      "Complement found from 12326 to 12333\n",
      "Complement found from 13871 to 13878\n",
      "Complement found from 14775 to 14782\n",
      "Seq is: AATAAAA\n",
      "6 AATAAAA motifs detected in file\n",
      "\n",
      "Seq is: ATACCCG\n",
      "0 ATACCCG motifs detected in file\n",
      "\n",
      "Complement found from 5932 to 5939\n",
      "Complement found from 8126 to 8133\n",
      "Complement found from 8190 to 8197\n",
      "Complement found from 11126 to 11133\n",
      "Seq is: AAACCAC\n",
      "4 AAACCAC motifs detected in file\n",
      "\n",
      "Complement found from 7860 to 7867\n",
      "Seq is: TCCCTCC\n",
      "1 TCCCTCC motifs detected in file\n",
      "\n",
      "Complement found from 5203 to 5210\n",
      "Complement found from 10156 to 10163\n",
      "Complement found from 12458 to 12465\n",
      "Seq is: ATCCACC\n",
      "3 ATCCACC motifs detected in file\n",
      "\n",
      "Complement found from 3671 to 3678\n",
      "Complement found from 8673 to 8680\n",
      "Complement found from 11316 to 11323\n",
      "Complement found from 15357 to 15364\n",
      "Seq is: ATCAAAC\n",
      "4 ATCAAAC motifs detected in file\n",
      "\n",
      "Seq is: ACAAATT\n",
      "0 ACAAATT motifs detected in file\n",
      "\n",
      "Complement found from 8488 to 8495\n",
      "Complement found from 16159 to 16166\n",
      "Seq is: ATAAAAA\n",
      "2 ATAAAAA motifs detected in file\n",
      "\n",
      "Complement found from 2421 to 2428\n",
      "Complement found from 11507 to 11514\n",
      "Complement found from 12104 to 12111\n",
      "Complement found from 14067 to 14074\n",
      "Complement found from 14270 to 14277\n",
      "Complement found from 14413 to 14420\n",
      "Complement found from 16216 to 16223\n",
      "Seq is: TCAACCC\n",
      "7 TCAACCC motifs detected in file\n",
      "\n",
      "Complement found from 5119 to 5126\n",
      "Complement found from 5377 to 5384\n",
      "Complement found from 6306 to 6313\n",
      "Complement found from 6681 to 6688\n",
      "Complement found from 11289 to 11296\n",
      "Complement found from 13418 to 13425\n",
      "Complement found from 14906 to 14913\n",
      "Seq is: ACTACTC\n",
      "7 ACTACTC motifs detected in file\n",
      "\n",
      "Complement found from 3798 to 3805\n",
      "Complement found from 7642 to 7649\n",
      "Complement found from 9093 to 9100\n",
      "Seq is: CTTATCA\n",
      "3 CTTATCA motifs detected in file\n",
      "\n",
      "Complement found from 3358 to 3365\n",
      "Complement found from 5112 to 5119\n",
      "Complement found from 5439 to 5446\n",
      "Complement found from 13309 to 13316\n",
      "Seq is: CATTCCT\n",
      "4 CATTCCT motifs detected in file\n",
      "\n",
      "Complement found from 536 to 543\n",
      "Complement found from 3305 to 3312\n",
      "Complement found from 3575 to 3582\n",
      "Complement found from 4132 to 4139\n",
      "Complement found from 4223 to 4230\n",
      "Complement found from 4431 to 4438\n",
      "Complement found from 12427 to 12434\n",
      "Seq is: CATACCC\n",
      "7 CATACCC motifs detected in file\n",
      "\n",
      "Complement found from 9073 to 9080\n",
      "Complement found from 13693 to 13700\n",
      "Seq is: CCATTAA\n",
      "2 CCATTAA motifs detected in file\n",
      "\n",
      "Complement found from 8461 to 8468\n",
      "Complement found from 12395 to 12402\n",
      "Complement found from 16049 to 16056\n",
      "Seq is: TACCACC\n",
      "3 TACCACC motifs detected in file\n",
      "\n",
      "Complement found from 410 to 417\n",
      "Seq is: CGGTATG\n",
      "1 CGGTATG motifs detected in file\n",
      "\n",
      "Complement found from 3745 to 3752\n",
      "Complement found from 3834 to 3841\n",
      "Complement found from 4950 to 4957\n",
      "Complement found from 5307 to 5314\n",
      "Complement found from 6117 to 6124\n",
      "Complement found from 7065 to 7072\n",
      "Complement found from 7793 to 7800\n",
      "Complement found from 12631 to 12638\n",
      "Complement found from 13369 to 13376\n",
      "Complement found from 14056 to 14063\n",
      "Complement found from 14341 to 14348\n",
      "Complement found from 14480 to 14487\n",
      "Complement found from 15788 to 15795\n",
      "Seq is: CCATCAT\n",
      "13 CCATCAT motifs detected in file\n",
      "\n",
      "Complement found from 1115 to 1122\n",
      "Complement found from 7876 to 7883\n",
      "Complement found from 10889 to 10896\n",
      "Complement found from 16414 to 16421\n",
      "Seq is: AAATCAA\n",
      "4 AAATCAA motifs detected in file\n",
      "\n",
      "Complement found from 3168 to 3175\n",
      "Seq is: CCCCGTA\n",
      "1 CCCCGTA motifs detected in file\n",
      "\n",
      "Seq is: AAGTGAC\n",
      "0 AAGTGAC motifs detected in file\n",
      "\n",
      "Complement found from 7280 to 7287\n",
      "Complement found from 13730 to 13737\n",
      "Complement found from 13897 to 13904\n",
      "Complement found from 15063 to 15070\n",
      "Seq is: ATTTCTC\n",
      "4 ATTTCTC motifs detected in file\n",
      "\n",
      "Complement found from 10836 to 10843\n",
      "Complement found from 12542 to 12549\n",
      "Complement found from 13377 to 13384\n",
      "Complement found from 14328 to 14335\n",
      "Seq is: CACAACC\n",
      "4 CACAACC motifs detected in file\n",
      "\n",
      "Complement found from 3134 to 3141\n",
      "Complement found from 8337 to 8344\n",
      "Seq is: AAGAGAA\n",
      "2 AAGAGAA motifs detected in file\n",
      "\n",
      "Complement found from 7523 to 7530\n",
      "Seq is: TTAGAAA\n",
      "1 TTAGAAA motifs detected in file\n",
      "\n",
      "Seq is: TTGAATT\n",
      "0 TTGAATT motifs detected in file\n",
      "\n",
      "Complement found from 10019 to 10026\n",
      "Seq is: TTCCAAT\n",
      "1 TTCCAAT motifs detected in file\n",
      "\n",
      "Seq is: TACGATT\n",
      "0 TACGATT motifs detected in file\n",
      "\n",
      "Complement found from 8001 to 8008\n",
      "Seq is: CAATCGA\n",
      "1 CAATCGA motifs detected in file\n",
      "\n",
      "Seq is: AATGTCA\n",
      "0 AATGTCA motifs detected in file\n",
      "\n",
      "Seq is: CCCGAGA\n",
      "0 CCCGAGA motifs detected in file\n",
      "\n",
      "Complement found from 4054 to 4061\n",
      "Complement found from 5484 to 5491\n",
      "Seq is: TCTCCCC\n",
      "2 TCTCCCC motifs detected in file\n",
      "\n",
      "Complement found from 14823 to 14830\n",
      "Seq is: CATCTCC\n",
      "1 CATCTCC motifs detected in file\n",
      "\n",
      "Seq is: CCCACGA\n",
      "0 CCCACGA motifs detected in file\n",
      "\n",
      "Seq is: TATCCCA\n",
      "0 TATCCCA motifs detected in file\n",
      "\n",
      "Seq is: TTTATGC\n",
      "0 TTTATGC motifs detected in file\n",
      "\n",
      "Seq is: CCGGGAT\n",
      "0 CCGGGAT motifs detected in file\n",
      "\n",
      "Complement found from 843 to 850\n",
      "Complement found from 14319 to 14326\n",
      "Seq is: AAAGTTT\n",
      "2 AAAGTTT motifs detected in file\n",
      "\n",
      "Complement found from 2224 to 2231\n",
      "Complement found from 8230 to 8237\n",
      "Complement found from 8440 to 8447\n",
      "Seq is: CTAAAAA\n",
      "3 CTAAAAA motifs detected in file\n",
      "\n",
      "Complement found from 2074 to 2081\n",
      "Complement found from 5486 to 5493\n",
      "Complement found from 8842 to 8849\n",
      "Complement found from 16544 to 16551\n",
      "Seq is: TCCCCTT\n",
      "4 TCCCCTT motifs detected in file\n",
      "\n",
      "Complement found from 2190 to 2197\n",
      "Complement found from 5638 to 5645\n",
      "Seq is: AATTAAG\n",
      "2 AATTAAG motifs detected in file\n",
      "\n",
      "Complement found from 7783 to 7790\n",
      "Seq is: ATCCTGC\n",
      "1 ATCCTGC motifs detected in file\n",
      "\n",
      "Seq is: TACCCGT\n",
      "0 TACCCGT motifs detected in file\n",
      "\n",
      "Complement found from 4286 to 4293\n",
      "Seq is: ACTTTGA\n",
      "1 ACTTTGA motifs detected in file\n",
      "\n",
      "Complement found from 295 to 302\n",
      "Complement found from 3473 to 3480\n",
      "Complement found from 8477 to 8484\n",
      "Complement found from 14040 to 14047\n",
      "Seq is: CACCAAA\n",
      "4 CACCAAA motifs detected in file\n",
      "\n",
      "Seq is: CGAGGTG\n",
      "0 CGAGGTG motifs detected in file\n",
      "\n",
      "Seq is: TCTAAAG\n",
      "0 TCTAAAG motifs detected in file\n",
      "\n",
      "Seq is: TCTCGTA\n",
      "0 TCTCGTA motifs detected in file\n",
      "\n",
      "Complement found from 6962 to 6969\n",
      "Complement found from 9927 to 9934\n",
      "Seq is: ACTGGCA\n",
      "2 ACTGGCA motifs detected in file\n",
      "\n",
      "Complement found from 3726 to 3733\n",
      "Seq is: TCATATG\n",
      "1 TCATATG motifs detected in file\n",
      "\n",
      "Complement found from 6380 to 6387\n",
      "Complement found from 15169 to 15176\n",
      "Seq is: GGGGCCA\n",
      "2 GGGGCCA motifs detected in file\n",
      "\n",
      "Complement found from 13316 to 13323\n",
      "Seq is: GCACATC\n",
      "1 GCACATC motifs detected in file\n",
      "\n",
      "Complement found from 259 to 266\n",
      "Complement found from 5630 to 5637\n",
      "Complement found from 11003 to 11010\n",
      "Seq is: GCCACTT\n",
      "3 GCCACTT motifs detected in file\n",
      "\n",
      "Complement found from 3472 to 3479\n",
      "Complement found from 4084 to 4091\n",
      "Complement found from 8476 to 8483\n",
      "Complement found from 14402 to 14409\n",
      "Seq is: TCACCAA\n",
      "4 TCACCAA motifs detected in file\n",
      "\n",
      "Complement found from 4896 to 4903\n",
      "Complement found from 9675 to 9682\n",
      "Complement found from 10464 to 10471\n",
      "Complement found from 10886 to 10893\n",
      "Complement found from 14041 to 14048\n",
      "Seq is: ACCAAAT\n",
      "5 ACCAAAT motifs detected in file\n",
      "\n",
      "Complement found from 9019 to 9026\n",
      "Seq is: CTGCAGG\n",
      "1 CTGCAGG motifs detected in file\n",
      "\n",
      "Complement found from 5192 to 5199\n",
      "Complement found from 15526 to 15533\n",
      "Complement found from 16293 to 16300\n",
      "Complement found from 16546 to 16553\n",
      "Seq is: CCCTTAA\n",
      "4 CCCTTAA motifs detected in file\n",
      "\n",
      "Complement found from 6328 to 6335\n",
      "Complement found from 6796 to 6803\n",
      "Seq is: CGTAGAC\n",
      "2 CGTAGAC motifs detected in file\n",
      "\n",
      "Complement found from 286 to 293\n",
      "Complement found from 1800 to 1807\n",
      "Complement found from 2151 to 2158\n",
      "Complement found from 2800 to 2807\n",
      "Complement found from 3380 to 3387\n",
      "Complement found from 8497 to 8504\n",
      "Complement found from 12305 to 12312\n",
      "Seq is: AAAAATT\n",
      "7 AAAAATT motifs detected in file\n",
      "\n",
      "Seq is: CGGATTA\n",
      "0 CGGATTA motifs detected in file\n",
      "\n",
      "Complement found from 4831 to 4838\n",
      "Complement found from 8929 to 8936\n",
      "Complement found from 10159 to 10166\n",
      "Complement found from 13023 to 13030\n",
      "Complement found from 15534 to 15541\n",
      "Complement found from 16256 to 16263\n",
      "Seq is: CACCCCT\n",
      "6 CACCCCT motifs detected in file\n",
      "\n",
      "Seq is: GTCGAGT\n",
      "0 GTCGAGT motifs detected in file\n",
      "\n",
      "Complement found from 27 to 34\n",
      "Seq is: ACTCACG\n",
      "1 ACTCACG motifs detected in file\n",
      "\n",
      "Complement found from 1846 to 1853\n",
      "Seq is: TTCTGCA\n",
      "1 TTCTGCA motifs detected in file\n",
      "\n",
      "Complement found from 1137 to 1144\n",
      "Complement found from 5949 to 5956\n",
      "Seq is: GAACACT\n",
      "2 GAACACT motifs detected in file\n",
      "\n",
      "Complement found from 12023 to 12030\n",
      "Complement found from 12673 to 12680\n",
      "Complement found from 13660 to 13667\n",
      "Complement found from 15466 to 15473\n",
      "Seq is: ACATTAA\n",
      "4 ACATTAA motifs detected in file\n",
      "\n",
      "Complement found from 3853 to 3860\n",
      "Complement found from 11339 to 11346\n",
      "Seq is: TAATATG\n",
      "2 TAATATG motifs detected in file\n",
      "\n",
      "Complement found from 2483 to 2490\n",
      "Complement found from 11862 to 11869\n",
      "Seq is: CTTACCC\n",
      "2 CTTACCC motifs detected in file\n",
      "\n",
      "Seq is: CCGAAGT\n",
      "0 CCGAAGT motifs detected in file\n",
      "\n",
      "Complement found from 1094 to 1101\n",
      "Complement found from 1390 to 1397\n",
      "Complement found from 8986 to 8993\n",
      "Complement found from 10350 to 10357\n",
      "Complement found from 10692 to 10699\n",
      "Complement found from 11639 to 11646\n",
      "Complement found from 12937 to 12944\n",
      "Seq is: TAGCCCT\n",
      "7 TAGCCCT motifs detected in file\n",
      "\n",
      "Complement found from 14831 to 14838\n",
      "Seq is: CATGATG\n",
      "1 CATGATG motifs detected in file\n",
      "\n",
      "Complement found from 5175 to 5182\n",
      "Complement found from 8302 to 8309\n",
      "Seq is: AGCTAAC\n",
      "2 AGCTAAC motifs detected in file\n",
      "\n",
      "Complement found from 5558 to 5565\n",
      "Seq is: AGTTGCA\n",
      "1 AGTTGCA motifs detected in file\n",
      "\n",
      "Seq is: GTTCCTC\n",
      "0 GTTCCTC motifs detected in file\n",
      "\n",
      "Seq is: AGCGTCC\n",
      "0 AGCGTCC motifs detected in file\n",
      "\n",
      "Seq is: AGCGGAC\n",
      "0 AGCGGAC motifs detected in file\n",
      "\n",
      "Seq is: CAAGATC\n",
      "0 CAAGATC motifs detected in file\n",
      "\n",
      "Complement found from 9695 to 9702\n",
      "Complement found from 10243 to 10250\n",
      "Complement found from 10333 to 10340\n",
      "Seq is: CTTATTA\n",
      "3 CTTATTA motifs detected in file\n",
      "\n",
      "Complement found from 3892 to 3899\n",
      "Complement found from 4317 to 4324\n",
      "Complement found from 4794 to 4801\n",
      "Complement found from 13762 to 13769\n",
      "Seq is: CCCCCTT\n",
      "4 CCCCCTT motifs detected in file\n",
      "\n",
      "Complement found from 14663 to 14670\n",
      "Complement found from 15206 to 15213\n",
      "Seq is: CATACAT\n",
      "2 CATACAT motifs detected in file\n",
      "\n",
      "Complement found from 1104 to 1111\n",
      "Complement found from 3596 to 3603\n",
      "Complement found from 6544 to 6551\n",
      "Complement found from 10612 to 10619\n",
      "Complement found from 12102 to 12109\n",
      "Complement found from 13844 to 13851\n",
      "Complement found from 14065 to 14072\n",
      "Complement found from 14411 to 14418\n",
      "Complement found from 14921 to 14928\n",
      "Complement found from 16221 to 16228\n",
      "Seq is: CCTCAAC\n",
      "10 CCTCAAC motifs detected in file\n",
      "\n",
      "Complement found from 3343 to 3350\n",
      "Complement found from 4677 to 4684\n",
      "Complement found from 9109 to 9116\n",
      "Complement found from 16001 to 16008\n",
      "Seq is: TTCTAAT\n",
      "4 TTCTAAT motifs detected in file\n",
      "\n",
      "Seq is: CAGGCGG\n",
      "0 CAGGCGG motifs detected in file\n",
      "\n",
      "Complement found from 8963 to 8970\n",
      "Seq is: CATCAGC\n",
      "1 CATCAGC motifs detected in file\n",
      "\n",
      "Complement found from 4861 to 4868\n",
      "Complement found from 7246 to 7253\n",
      "Seq is: CACATGA\n",
      "2 CACATGA motifs detected in file\n",
      "\n",
      "Complement found from 4664 to 4671\n",
      "Complement found from 12456 to 12463\n",
      "Seq is: GCATCCA\n",
      "2 GCATCCA motifs detected in file\n",
      "\n",
      "Seq is: AGTCATG\n",
      "0 AGTCATG motifs detected in file\n",
      "\n",
      "Complement found from 11905 to 11912\n",
      "Seq is: GTAACCA\n",
      "1 GTAACCA motifs detected in file\n",
      "\n",
      "Seq is: CCGGTTA\n",
      "0 CCGGTTA motifs detected in file\n",
      "\n",
      "Complement found from 4912 to 4919\n",
      "Complement found from 11277 to 11284\n",
      "Complement found from 13685 to 13692\n",
      "Complement found from 14581 to 14588\n",
      "Complement found from 14635 to 14642\n",
      "Seq is: ACTAAAC\n",
      "5 ACTAAAC motifs detected in file\n",
      "\n",
      "Seq is: TACCATT\n",
      "0 TACCATT motifs detected in file\n",
      "\n",
      "Complement found from 306 to 313\n",
      "Complement found from 456 to 463\n",
      "Complement found from 957 to 964\n",
      "Complement found from 3568 to 3575\n",
      "Complement found from 6289 to 6296\n",
      "Complement found from 7814 to 7821\n",
      "Complement found from 7861 to 7868\n",
      "Complement found from 15537 to 15544\n",
      "Complement found from 16185 to 16192\n",
      "Seq is: CCCTCCC\n",
      "9 CCCTCCC motifs detected in file\n",
      "\n",
      "Complement found from 15116 to 15123\n",
      "Complement found from 15320 to 15327\n",
      "Seq is: TAGCAAC\n",
      "2 TAGCAAC motifs detected in file\n",
      "\n",
      "Complement found from 8184 to 8191\n",
      "Seq is: TGGAGCA\n",
      "1 TGGAGCA motifs detected in file\n",
      "\n",
      "Seq is: GACAATA\n",
      "0 GACAATA motifs detected in file\n",
      "\n",
      "Complement found from 9766 to 9773\n",
      "Complement found from 13748 to 13755\n",
      "Seq is: CATTTCC\n",
      "2 CATTTCC motifs detected in file\n",
      "\n",
      "Complement found from 1534 to 1541\n",
      "Complement found from 7826 to 7833\n",
      "Seq is: TACGCAT\n",
      "2 TACGCAT motifs detected in file\n",
      "\n",
      "Complement found from 7403 to 7410\n",
      "Complement found from 9526 to 9533\n",
      "Complement found from 10278 to 10285\n",
      "Seq is: CCCTACC\n",
      "3 CCCTACC motifs detected in file\n",
      "\n",
      "Complement found from 5732 to 5739\n",
      "Seq is: CTCCCGC\n",
      "1 CTCCCGC motifs detected in file\n",
      "\n",
      "Complement found from 3325 to 3332\n",
      "Complement found from 3826 to 3833\n",
      "Complement found from 5009 to 5016\n",
      "Complement found from 5472 to 5479\n",
      "Complement found from 6231 to 6238\n",
      "Complement found from 8411 to 8418\n",
      "Complement found from 13984 to 13991\n",
      "Complement found from 14131 to 14138\n",
      "Complement found from 14440 to 14447\n",
      "Seq is: TACTCCT\n",
      "9 TACTCCT motifs detected in file\n",
      "\n",
      "Seq is: GATCCTA\n",
      "0 GATCCTA motifs detected in file\n",
      "\n",
      "Complement found from 5106 to 5113\n",
      "Seq is: CTACCGC\n",
      "1 CTACCGC motifs detected in file\n",
      "\n",
      "Complement found from 6462 to 6469\n",
      "Seq is: CCGTCCT\n",
      "1 CCGTCCT motifs detected in file\n",
      "\n",
      "Complement found from 4637 to 4644\n",
      "Seq is: ATCAAGT\n",
      "1 ATCAAGT motifs detected in file\n",
      "\n",
      "Complement found from 344 to 351\n",
      "Seq is: CTGCCAA\n",
      "1 CTGCCAA motifs detected in file\n",
      "\n",
      "Seq is: AGATAAA\n",
      "0 AGATAAA motifs detected in file\n",
      "\n",
      "Seq is: GGACTCG\n",
      "0 GGACTCG motifs detected in file\n",
      "\n",
      "Seq is: CAGACTC\n",
      "0 CAGACTC motifs detected in file\n",
      "\n",
      "Complement found from 2386 to 2393\n",
      "Complement found from 4022 to 4029\n",
      "Complement found from 4232 to 4239\n",
      "Complement found from 13280 to 13287\n",
      "Seq is: TACAATC\n",
      "4 TACAATC motifs detected in file\n",
      "\n",
      "Complement found from 2597 to 2604\n",
      "Complement found from 4757 to 4764\n",
      "Complement found from 6830 to 6837\n",
      "Complement found from 7669 to 7676\n",
      "Complement found from 10079 to 10086\n",
      "Complement found from 10452 to 10459\n",
      "Complement found from 14229 to 14236\n",
      "Seq is: ATAATCA\n",
      "7 ATAATCA motifs detected in file\n",
      "\n",
      "Complement found from 740 to 747\n",
      "Complement found from 13227 to 13234\n",
      "Complement found from 16176 to 16183\n",
      "Seq is: ATCAAAA\n",
      "3 ATCAAAA motifs detected in file\n",
      "\n",
      "Complement found from 7184 to 7191\n",
      "Complement found from 8708 to 8715\n",
      "Complement found from 15323 to 15330\n",
      "Seq is: CAACACT\n",
      "3 CAACACT motifs detected in file\n",
      "\n",
      "Complement found from 8561 to 8568\n",
      "Complement found from 10983 to 10990\n",
      "Complement found from 13940 to 13947\n",
      "Complement found from 15417 to 15424\n",
      "Seq is: CACAATC\n",
      "4 CACAATC motifs detected in file\n",
      "\n",
      "Complement found from 7708 to 7715\n",
      "Complement found from 10923 to 10930\n",
      "Seq is: CTTTTCC\n",
      "2 CTTTTCC motifs detected in file\n",
      "\n",
      "Seq is: CGTATGT\n",
      "0 CGTATGT motifs detected in file\n",
      "\n",
      "Complement found from 3689 to 3696\n",
      "Complement found from 8725 to 8732\n",
      "Complement found from 11918 to 11925\n",
      "Complement found from 14865 to 14872\n",
      "Seq is: CCTGATC\n",
      "4 CCTGATC motifs detected in file\n",
      "\n",
      "Complement found from 6451 to 6458\n",
      "Seq is: CTTCGTC\n",
      "1 CTTCGTC motifs detected in file\n",
      "\n",
      "Complement found from 7961 to 7968\n",
      "Complement found from 8423 to 8430\n",
      "Complement found from 14891 to 14898\n",
      "Seq is: TATTCCT\n",
      "3 TATTCCT motifs detected in file\n",
      "\n",
      "Seq is: TTTACTA\n",
      "0 TTTACTA motifs detected in file\n",
      "\n",
      "Complement found from 5008 to 5015\n",
      "Complement found from 8410 to 8417\n",
      "Complement found from 11973 to 11980\n",
      "Complement found from 14439 to 14446\n",
      "Seq is: ATACTCC\n",
      "4 ATACTCC motifs detected in file\n",
      "\n",
      "Complement found from 11244 to 11251\n",
      "Seq is: CGCACTA\n",
      "1 CGCACTA motifs detected in file\n",
      "\n",
      "Seq is: GTACTTT\n",
      "0 GTACTTT motifs detected in file\n",
      "\n",
      "Complement found from 1084 to 1091\n",
      "Complement found from 7033 to 7040\n",
      "Complement found from 11020 to 11027\n",
      "Complement found from 11870 to 11877\n",
      "Complement found from 13881 to 13888\n",
      "Seq is: CCACTAT\n",
      "5 CCACTAT motifs detected in file\n",
      "\n",
      "Seq is: TCGAGAA\n",
      "0 TCGAGAA motifs detected in file\n",
      "\n",
      "Seq is: GATACTA\n",
      "0 GATACTA motifs detected in file\n",
      "\n",
      "Complement found from 2074 to 2081\n",
      "Complement found from 5486 to 5493\n",
      "Complement found from 8842 to 8849\n",
      "Complement found from 16544 to 16551\n",
      "Seq is: TCCCCTT\n",
      "4 TCCCCTT motifs detected in file\n",
      "\n",
      "Complement found from 11241 to 11248\n",
      "Seq is: CATCGCA\n",
      "1 CATCGCA motifs detected in file\n",
      "\n",
      "Seq is: GAACATC\n",
      "0 GAACATC motifs detected in file\n",
      "\n",
      "Complement found from 16367 to 16374\n",
      "Seq is: TGGATGA\n",
      "1 TGGATGA motifs detected in file\n",
      "\n",
      "Seq is: ACGCGAC\n",
      "0 ACGCGAC motifs detected in file\n",
      "\n",
      "Seq is: GTACACG\n",
      "0 GTACACG motifs detected in file\n",
      "\n",
      "Seq is: GTAATTC\n",
      "0 GTAATTC motifs detected in file\n",
      "\n",
      "Complement found from 7574 to 7581\n",
      "Seq is: TATATAT\n",
      "1 TATATAT motifs detected in file\n",
      "\n",
      "Complement found from 14491 to 14498\n",
      "Complement found from 16526 to 16533\n",
      "Seq is: CCTAAAT\n",
      "2 CCTAAAT motifs detected in file\n",
      "\n",
      "Seq is: CGGATAT\n",
      "0 CGGATAT motifs detected in file\n",
      "\n",
      "Seq is: TAAATTG\n",
      "0 TAAATTG motifs detected in file\n",
      "\n",
      "Seq is: AAACTGT\n",
      "0 AAACTGT motifs detected in file\n",
      "\n",
      "Complement found from 171 to 178\n",
      "Complement found from 5800 to 5807\n",
      "Seq is: TTCAATA\n",
      "2 TTCAATA motifs detected in file\n",
      "\n",
      "Complement found from 15279 to 15286\n",
      "Seq is: CTTTACC\n",
      "1 CTTTACC motifs detected in file\n",
      "\n",
      "Complement found from 5829 to 5836\n",
      "Complement found from 6693 to 6700\n",
      "Complement found from 10048 to 10055\n",
      "Seq is: AAAAAGA\n",
      "3 AAAAAGA motifs detected in file\n",
      "\n",
      "Seq is: TTATGGA\n",
      "0 TTATGGA motifs detected in file\n",
      "\n",
      "Complement found from 3258 to 3265\n",
      "Complement found from 3265 to 3272\n",
      "Complement found from 4177 to 4184\n",
      "Seq is: AAAACTT\n",
      "3 AAAACTT motifs detected in file\n",
      "\n",
      "Seq is: TTTCAGT\n",
      "0 TTTCAGT motifs detected in file\n",
      "\n",
      "Complement found from 2837 to 2844\n",
      "Complement found from 6818 to 6825\n",
      "Complement found from 9287 to 9294\n",
      "Seq is: ACCTCCG\n",
      "3 ACCTCCG motifs detected in file\n",
      "\n",
      "Seq is: GTACCCC\n",
      "0 GTACCCC motifs detected in file\n",
      "\n",
      "Complement found from 3885 to 3892\n",
      "Complement found from 9668 to 9675\n",
      "Complement found from 10396 to 10403\n",
      "Seq is: AACCGAA\n",
      "3 AACCGAA motifs detected in file\n",
      "\n",
      "Seq is: CTTTGGT\n",
      "0 CTTTGGT motifs detected in file\n",
      "\n",
      "Complement found from 1904 to 1911\n",
      "Complement found from 8957 to 8964\n",
      "Complement found from 9671 to 9678\n",
      "Complement found from 11124 to 11131\n",
      "Complement found from 13523 to 13530\n",
      "Seq is: CGAAACC\n",
      "5 CGAAACC motifs detected in file\n",
      "\n",
      "Seq is: CCCCAAG\n",
      "0 CCCCAAG motifs detected in file\n",
      "\n",
      "Complement found from 5585 to 5592\n",
      "Seq is: CTAAGGA\n",
      "1 CTAAGGA motifs detected in file\n",
      "\n",
      "Complement found from 2126 to 2133\n",
      "Seq is: AGGAAAA\n",
      "1 AGGAAAA motifs detected in file\n",
      "\n",
      "Complement found from 8097 to 8104\n",
      "Seq is: AACAGAT\n",
      "1 AACAGAT motifs detected in file\n",
      "\n",
      "Complement found from 11602 to 11609\n",
      "Seq is: CTAAAAT\n",
      "1 CTAAAAT motifs detected in file\n",
      "\n",
      "Complement found from 4041 to 4048\n",
      "Complement found from 10718 to 10725\n",
      "Seq is: ACATATG\n",
      "2 ACATATG motifs detected in file\n",
      "\n",
      "Complement found from 3213 to 3220\n",
      "Complement found from 11306 to 11313\n",
      "Seq is: CCCAAGA\n",
      "2 CCCAAGA motifs detected in file\n",
      "\n",
      "Complement found from 584 to 591\n",
      "Complement found from 11349 to 11356\n",
      "Seq is: AGCTTAC\n",
      "2 AGCTTAC motifs detected in file\n",
      "\n",
      "Complement found from 6769 to 6776\n",
      "Seq is: ACACCAT\n",
      "1 ACACCAT motifs detected in file\n",
      "\n",
      "Complement found from 3209 to 3216\n",
      "Complement found from 5432 to 5439\n",
      "Complement found from 6312 to 6319\n",
      "Complement found from 7399 to 7406\n",
      "Complement found from 13053 to 13060\n",
      "Complement found from 13647 to 13654\n",
      "Complement found from 13677 to 13684\n",
      "Complement found from 14809 to 14816\n",
      "Complement found from 15262 to 15269\n",
      "Complement found from 16289 to 16296\n",
      "Seq is: CCCACCC\n",
      "10 CCCACCC motifs detected in file\n",
      "\n",
      "Seq is: CCCGTAG\n",
      "0 CCCGTAG motifs detected in file\n",
      "\n",
      "Complement found from 10529 to 10536\n",
      "Complement found from 14464 to 14471\n",
      "Seq is: GTATATC\n",
      "2 GTATATC motifs detected in file\n",
      "\n",
      "Seq is: AGTGAGG\n",
      "0 AGTGAGG motifs detected in file\n",
      "\n",
      "Complement found from 8625 to 8632\n",
      "Complement found from 12322 to 12329\n",
      "Complement found from 14873 to 14880\n",
      "Seq is: TCCAAAT\n",
      "3 TCCAAAT motifs detected in file\n",
      "\n",
      "Complement found from 4104 to 4111\n",
      "Complement found from 6217 to 6224\n",
      "Complement found from 8469 to 8476\n",
      "Complement found from 13446 to 13453\n",
      "Complement found from 13581 to 13588\n",
      "Complement found from 14526 to 14533\n",
      "Complement found from 14804 to 14811\n",
      "Complement found from 15379 to 15386\n",
      "Seq is: ACCTCCC\n",
      "8 ACCTCCC motifs detected in file\n",
      "\n",
      "Seq is: AAGAAGA\n",
      "0 AAGAAGA motifs detected in file\n",
      "\n",
      "Complement found from 10450 to 10457\n",
      "Seq is: TGATAAT\n",
      "1 TGATAAT motifs detected in file\n",
      "\n",
      "Complement found from 5166 to 5173\n",
      "Complement found from 15080 to 15087\n",
      "Seq is: CCTGAAA\n",
      "2 CCTGAAA motifs detected in file\n",
      "\n",
      "Seq is: AGCGAAG\n",
      "0 AGCGAAG motifs detected in file\n",
      "\n",
      "Complement found from 3449 to 3456\n",
      "Complement found from 7327 to 7334\n",
      "Seq is: CTTCGCT\n",
      "2 CTTCGCT motifs detected in file\n",
      "\n",
      "Seq is: TCCGAAG\n",
      "0 TCCGAAG motifs detected in file\n",
      "\n",
      "Seq is: AGTTTAG\n",
      "0 AGTTTAG motifs detected in file\n",
      "\n",
      "Complement found from 2660 to 2667\n",
      "Seq is: TACTTTT\n",
      "1 TACTTTT motifs detected in file\n",
      "\n",
      "Complement found from 7303 to 7310\n",
      "Complement found from 8040 to 8047\n",
      "Complement found from 9679 to 9686\n",
      "Complement found from 10108 to 10115\n",
      "Complement found from 13610 to 13617\n",
      "Seq is: AATAATT\n",
      "5 AATAATT motifs detected in file\n",
      "\n",
      "Complement found from 1852 to 1859\n",
      "Complement found from 9203 to 9210\n",
      "Seq is: ATAATGA\n",
      "2 ATAATGA motifs detected in file\n",
      "\n",
      "Seq is: CGACAAT\n",
      "0 CGACAAT motifs detected in file\n",
      "\n",
      "Complement found from 8351 to 8358\n",
      "Complement found from 11381 to 11388\n",
      "Seq is: CTCTTTA\n",
      "2 CTCTTTA motifs detected in file\n",
      "\n",
      "Complement found from 9987 to 9994\n",
      "Complement found from 13109 to 13116\n",
      "Seq is: CTTACTC\n",
      "2 CTTACTC motifs detected in file\n",
      "\n",
      "Seq is: GGATGTC\n",
      "0 GGATGTC motifs detected in file\n",
      "\n",
      "Complement found from 3754 to 3761\n",
      "Complement found from 5154 to 5161\n",
      "Complement found from 10578 to 10585\n",
      "Complement found from 11546 to 11553\n",
      "Complement found from 15191 to 15198\n",
      "Complement found from 15629 to 15636\n",
      "Seq is: TACTATC\n",
      "6 TACTATC motifs detected in file\n",
      "\n",
      "Complement found from 1289 to 1296\n",
      "Complement found from 5928 to 5935\n",
      "Complement found from 10292 to 10299\n",
      "Complement found from 10376 to 10383\n",
      "Complement found from 11062 to 11069\n",
      "Seq is: CTACAAA\n",
      "5 CTACAAA motifs detected in file\n",
      "\n",
      "Complement found from 7911 to 7918\n",
      "Seq is: GTACACC\n",
      "1 GTACACC motifs detected in file\n",
      "\n",
      "Seq is: ACATACG\n",
      "0 ACATACG motifs detected in file\n",
      "\n",
      "Complement found from 1678 to 1685\n",
      "Complement found from 4791 to 4798\n",
      "Complement found from 4875 to 4882\n",
      "Complement found from 9522 to 9529\n",
      "Seq is: TAGCCCC\n",
      "4 TAGCCCC motifs detected in file\n",
      "\n",
      "Seq is: ATCAGGC\n",
      "0 ATCAGGC motifs detected in file\n",
      "\n",
      "Complement found from 8551 to 8558\n",
      "Complement found from 11612 to 11619\n",
      "Seq is: TCATTGC\n",
      "2 TCATTGC motifs detected in file\n",
      "\n",
      "Complement found from 5761 to 5768\n",
      "Seq is: AGCCCCG\n",
      "1 AGCCCCG motifs detected in file\n",
      "\n",
      "Complement found from 3142 to 3149\n",
      "Seq is: TAAGGCC\n",
      "1 TAAGGCC motifs detected in file\n",
      "\n",
      "Complement found from 7200 to 7207\n",
      "Seq is: TATCCGG\n",
      "1 TATCCGG motifs detected in file\n",
      "\n",
      "Complement found from 14102 to 14109\n",
      "Complement found from 14349 to 14356\n",
      "Seq is: CTCTTTC\n",
      "2 CTCTTTC motifs detected in file\n",
      "\n",
      "Complement found from 3804 to 3811\n",
      "Complement found from 7183 to 7190\n",
      "Complement found from 8707 to 8714\n",
      "Complement found from 9196 to 9203\n",
      "Complement found from 11261 to 11268\n",
      "Complement found from 11995 to 12002\n",
      "Seq is: ACAACAC\n",
      "6 ACAACAC motifs detected in file\n",
      "\n",
      "Complement found from 6555 to 6562\n",
      "Complement found from 10239 to 10246\n",
      "Complement found from 12787 to 12794\n",
      "Complement found from 13333 to 13340\n",
      "Complement found from 13958 to 13965\n",
      "Seq is: CCTTCTT\n",
      "5 CCTTCTT motifs detected in file\n",
      "\n",
      "Complement found from 4940 to 4947\n",
      "Complement found from 5721 to 5728\n",
      "Complement found from 10706 to 10713\n",
      "Seq is: TCAATCT\n",
      "3 TCAATCT motifs detected in file\n",
      "\n",
      "Seq is: AAATGCG\n",
      "0 AAATGCG motifs detected in file\n",
      "\n",
      "Complement found from 7618 to 7625\n",
      "Seq is: GCTACTT\n",
      "1 GCTACTT motifs detected in file\n",
      "\n",
      "Complement found from 10577 to 10584\n",
      "Complement found from 13347 to 13354\n",
      "Seq is: ATACTAT\n",
      "2 ATACTAT motifs detected in file\n",
      "\n",
      "Complement found from 5632 to 5639\n",
      "Complement found from 9881 to 9888\n",
      "Complement found from 15713 to 15720\n",
      "Seq is: CACTTTA\n",
      "3 CACTTTA motifs detected in file\n",
      "\n",
      "Complement found from 2174 to 2181\n",
      "Complement found from 2223 to 2230\n",
      "Complement found from 3485 to 3492\n",
      "Complement found from 8229 to 8236\n",
      "Complement found from 11601 to 11608\n",
      "Complement found from 13790 to 13797\n",
      "Complement found from 14022 to 14029\n",
      "Seq is: CCTAAAA\n",
      "7 CCTAAAA motifs detected in file\n",
      "\n",
      "Complement found from 4662 to 4669\n",
      "Complement found from 13756 to 13763\n",
      "Seq is: CCGCATC\n",
      "2 CCGCATC motifs detected in file\n",
      "\n",
      "Complement found from 1720 to 1727\n",
      "Complement found from 5931 to 5938\n",
      "Complement found from 8125 to 8132\n",
      "Complement found from 8189 to 8196\n",
      "Seq is: CAAACCA\n",
      "4 CAAACCA motifs detected in file\n",
      "\n",
      "Seq is: GATCCCA\n",
      "0 GATCCCA motifs detected in file\n",
      "\n",
      "Seq is: CATCGGA\n",
      "0 CATCGGA motifs detected in file\n",
      "\n",
      "Complement found from 8731 to 8738\n",
      "Complement found from 10331 to 10338\n",
      "Seq is: CTCTTAT\n",
      "2 CTCTTAT motifs detected in file\n",
      "\n",
      "Complement found from 4795 to 4802\n",
      "Complement found from 5487 to 5494\n",
      "Seq is: CCCCTTT\n",
      "2 CCCCTTT motifs detected in file\n",
      "\n",
      "Seq is: AGTCACT\n",
      "0 AGTCACT motifs detected in file\n",
      "\n",
      "Complement found from 9746 to 9753\n",
      "Seq is: TACTTCG\n",
      "1 TACTTCG motifs detected in file\n",
      "\n",
      "Seq is: GAGGATA\n",
      "0 GAGGATA motifs detected in file\n",
      "\n",
      "Complement found from 5048 to 5055\n",
      "Complement found from 8381 to 8388\n",
      "Seq is: CTACCGT\n",
      "2 CTACCGT motifs detected in file\n",
      "\n",
      "Complement found from 10320 to 10327\n",
      "Seq is: TTATGTC\n",
      "1 TTATGTC motifs detected in file\n",
      "\n",
      "Seq is: GAGGATA\n",
      "0 GAGGATA motifs detected in file\n",
      "\n",
      "Complement found from 7780 to 7787\n",
      "Complement found from 9123 to 9130\n",
      "Complement found from 11547 to 11554\n",
      "Complement found from 15192 to 15199\n",
      "Complement found from 15630 to 15637\n",
      "Seq is: ACTATCC\n",
      "5 ACTATCC motifs detected in file\n",
      "\n",
      "Seq is: TGTATCA\n",
      "0 TGTATCA motifs detected in file\n",
      "\n",
      "Complement found from 9759 to 9766\n",
      "Complement found from 14988 to 14995\n",
      "Seq is: CCTTCAC\n",
      "2 CCTTCAC motifs detected in file\n",
      "\n",
      "Seq is: CCGATGT\n",
      "0 CCGATGT motifs detected in file\n",
      "\n",
      "Complement found from 5608 to 5615\n",
      "Seq is: TGCATCA\n",
      "1 TGCATCA motifs detected in file\n",
      "\n",
      "Complement found from 253 to 260\n",
      "Seq is: TGCACAG\n",
      "1 TGCACAG motifs detected in file\n",
      "\n",
      "Complement found from 629 to 636\n",
      "Complement found from 13515 to 13522\n",
      "Complement found from 14947 to 14954\n",
      "Complement found from 15544 to 15551\n",
      "Complement found from 16173 to 16180\n",
      "Complement found from 16233 to 16240\n",
      "Seq is: CACATCA\n",
      "6 CACATCA motifs detected in file\n",
      "\n",
      "Seq is: TGCTACA\n",
      "0 TGCTACA motifs detected in file\n",
      "\n",
      "Seq is: GATCACT\n",
      "0 GATCACT motifs detected in file\n",
      "\n",
      "Complement found from 1161 to 1168\n",
      "Complement found from 3676 to 3683\n",
      "Complement found from 11750 to 11757\n",
      "Complement found from 13421 to 13428\n",
      "Complement found from 15873 to 15880\n",
      "Seq is: ACTCAAA\n",
      "5 ACTCAAA motifs detected in file\n",
      "\n",
      "Seq is: CGATTAT\n",
      "0 CGATTAT motifs detected in file\n",
      "\n",
      "Seq is: GTTACGG\n",
      "0 GTTACGG motifs detected in file\n",
      "\n",
      "Seq is: TCAGTCC\n",
      "0 TCAGTCC motifs detected in file\n",
      "\n",
      "Complement found from 1417 to 1424\n",
      "Complement found from 4970 to 4977\n",
      "Seq is: GGTGGAT\n",
      "2 GGTGGAT motifs detected in file\n",
      "\n",
      "Complement found from 259 to 266\n",
      "Complement found from 5630 to 5637\n",
      "Complement found from 11003 to 11010\n",
      "Seq is: GCCACTT\n",
      "3 GCCACTT motifs detected in file\n",
      "\n",
      "Complement found from 16023 to 16030\n",
      "Seq is: TTCTTTC\n",
      "1 TTCTTTC motifs detected in file\n",
      "\n",
      "Seq is: TACTTAG\n",
      "0 TACTTAG motifs detected in file\n",
      "\n",
      "Seq is: GATCCCG\n",
      "0 GATCCCG motifs detected in file\n",
      "\n",
      "Seq is: AGTCTCG\n",
      "0 AGTCTCG motifs detected in file\n",
      "\n",
      "Seq is: TGACGTC\n",
      "0 TGACGTC motifs detected in file\n",
      "\n",
      "Seq is: GTCTAGT\n",
      "0 GTCTAGT motifs detected in file\n",
      "\n",
      "Complement found from 1738 to 1745\n",
      "Complement found from 6857 to 6864\n",
      "Seq is: AAAGTAT\n",
      "2 AAAGTAT motifs detected in file\n",
      "\n",
      "Seq is: AACGAAG\n",
      "0 AACGAAG motifs detected in file\n",
      "\n",
      "Seq is: GCACCTC\n",
      "0 GCACCTC motifs detected in file\n",
      "\n",
      "Complement found from 520 to 527\n",
      "Complement found from 1479 to 1486\n",
      "Complement found from 13934 to 13941\n",
      "Complement found from 14563 to 14570\n",
      "Seq is: ACACCGC\n",
      "4 ACACCGC motifs detected in file\n",
      "\n",
      "Seq is: TCAGTCG\n",
      "0 TCAGTCG motifs detected in file\n",
      "\n",
      "Complement found from 13352 to 13359\n",
      "Seq is: ATTTATG\n",
      "1 ATTTATG motifs detected in file\n",
      "\n",
      "Complement found from 1124 to 1131\n",
      "Seq is: AAACTGC\n",
      "1 AAACTGC motifs detected in file\n",
      "\n",
      "Complement found from 7885 to 7892\n",
      "Complement found from 9024 to 9031\n",
      "Complement found from 9410 to 9417\n",
      "Seq is: GGCCACC\n",
      "3 GGCCACC motifs detected in file\n",
      "\n",
      "Seq is: CCTATCG\n",
      "0 CCTATCG motifs detected in file\n",
      "\n",
      "Complement found from 472 to 479\n",
      "Complement found from 6527 to 6534\n",
      "Complement found from 9344 to 9351\n",
      "Complement found from 10103 to 10110\n",
      "Complement found from 13683 to 13690\n",
      "Complement found from 14213 to 14220\n",
      "Seq is: CTACTAA\n",
      "6 CTACTAA motifs detected in file\n",
      "\n",
      "Complement found from 3503 to 3510\n",
      "Complement found from 4501 to 4508\n",
      "Complement found from 7869 to 7876\n",
      "Complement found from 10504 to 10511\n",
      "Complement found from 15786 to 15793\n",
      "Seq is: TACCATC\n",
      "5 TACCATC motifs detected in file\n",
      "\n",
      "Complement found from 7497 to 7504\n",
      "Seq is: GCCTCCA\n",
      "1 GCCTCCA motifs detected in file\n",
      "\n",
      "Seq is: CTTTATA\n",
      "0 CTTTATA motifs detected in file\n",
      "\n",
      "Seq is: CGAGCAC\n",
      "0 CGAGCAC motifs detected in file\n",
      "\n",
      "Complement found from 8098 to 8105\n",
      "Seq is: ACAGATG\n",
      "1 ACAGATG motifs detected in file\n",
      "\n",
      "Seq is: CAGATAA\n",
      "0 CAGATAA motifs detected in file\n",
      "\n",
      "Complement found from 16368 to 16375\n",
      "Seq is: GGATGAC\n",
      "1 GGATGAC motifs detected in file\n",
      "\n",
      "Complement found from 1539 to 1546\n",
      "Complement found from 1973 to 1980\n",
      "Complement found from 5087 to 5094\n",
      "Seq is: ATTTATA\n",
      "3 ATTTATA motifs detected in file\n",
      "\n",
      "Complement found from 7406 to 7413\n",
      "Complement found from 8911 to 8918\n",
      "Complement found from 10128 to 10135\n",
      "Complement found from 11991 to 11998\n",
      "Complement found from 14325 to 14332\n",
      "Seq is: TACCACA\n",
      "5 TACCACA motifs detected in file\n",
      "\n",
      "Seq is: CGAGTGT\n",
      "0 CGAGTGT motifs detected in file\n",
      "\n",
      "Seq is: GCTATTT\n",
      "0 GCTATTT motifs detected in file\n",
      "\n",
      "Seq is: GGGATCC\n",
      "0 GGGATCC motifs detected in file\n",
      "\n",
      "Complement found from 2806 to 2813\n",
      "Seq is: TTCGGTT\n",
      "1 TTCGGTT motifs detected in file\n",
      "\n",
      "Complement found from 15111 to 15118\n",
      "Seq is: AACTATA\n",
      "1 AACTATA motifs detected in file\n",
      "\n",
      "Seq is: GTATCGA\n",
      "0 GTATCGA motifs detected in file\n",
      "\n",
      "Seq is: GTCTGGT\n",
      "0 GTCTGGT motifs detected in file\n",
      "\n",
      "Seq is: ATGGATA\n",
      "0 ATGGATA motifs detected in file\n",
      "\n",
      "Complement found from 6712 to 6719\n",
      "Seq is: CATAGGT\n",
      "1 CATAGGT motifs detected in file\n",
      "\n",
      "Complement found from 2497 to 2504\n",
      "Complement found from 10462 to 10469\n",
      "Seq is: TTACCAA\n",
      "2 TTACCAA motifs detected in file\n",
      "\n",
      "Complement found from 2128 to 2135\n",
      "Complement found from 4174 to 4181\n",
      "Complement found from 5745 to 5752\n",
      "Complement found from 6690 to 6697\n",
      "Complement found from 11030 to 11037\n",
      "Seq is: GAAAAAA\n",
      "5 GAAAAAA motifs detected in file\n",
      "\n",
      "Complement found from 3605 to 3612\n",
      "Complement found from 12977 to 12984\n",
      "Seq is: AGGCCTC\n",
      "2 AGGCCTC motifs detected in file\n",
      "\n",
      "Complement found from 3747 to 3754\n",
      "Complement found from 8592 to 8599\n",
      "Complement found from 14482 to 14489\n",
      "Complement found from 15159 to 15166\n",
      "Seq is: ATCATTC\n",
      "4 ATCATTC motifs detected in file\n",
      "\n",
      "Complement found from 2296 to 2303\n",
      "Seq is: AATGTTA\n",
      "1 AATGTTA motifs detected in file\n",
      "\n",
      "Complement found from 2410 to 2417\n",
      "Complement found from 6287 to 6294\n",
      "Complement found from 9724 to 9731\n",
      "Seq is: TACCCTC\n",
      "3 TACCCTC motifs detected in file\n",
      "\n",
      "Complement found from 3769 to 3776\n",
      "Seq is: TAATAAG\n",
      "1 TAATAAG motifs detected in file\n",
      "\n",
      "Complement found from 6651 to 6658\n",
      "Seq is: GCTTCGG\n",
      "1 GCTTCGG motifs detected in file\n",
      "\n",
      "Complement found from 16401 to 16408\n",
      "Seq is: ACCATCC\n",
      "1 ACCATCC motifs detected in file\n",
      "\n",
      "Complement found from 3970 to 3977\n",
      "Complement found from 6634 to 6641\n",
      "Complement found from 15011 to 15018\n",
      "Complement found from 15338 to 15345\n",
      "Seq is: TATTCTT\n",
      "4 TATTCTT motifs detected in file\n",
      "\n",
      "Seq is: ATATTTG\n",
      "0 ATATTTG motifs detected in file\n",
      "\n",
      "Complement found from 9588 to 9595\n",
      "Seq is: AAGTCCC\n",
      "1 AAGTCCC motifs detected in file\n",
      "\n",
      "Complement found from 5714 to 5721\n",
      "Seq is: ACTGGCT\n",
      "1 ACTGGCT motifs detected in file\n",
      "\n",
      "Complement found from 9994 to 10001\n",
      "Seq is: TTTTAGT\n",
      "1 TTTTAGT motifs detected in file\n",
      "\n",
      "Complement found from 4945 to 4952\n",
      "Complement found from 6638 to 6645\n",
      "Complement found from 8934 to 8941\n",
      "Complement found from 11007 to 11014\n",
      "Complement found from 11134 to 11141\n",
      "Seq is: CTTATCC\n",
      "5 CTTATCC motifs detected in file\n",
      "\n",
      "Complement found from 10837 to 10844\n",
      "Complement found from 14329 to 14336\n",
      "Complement found from 14476 to 14483\n",
      "Complement found from 14689 to 14696\n",
      "Complement found from 15765 to 15772\n",
      "Seq is: ACAACCA\n",
      "5 ACAACCA motifs detected in file\n",
      "\n",
      "Complement found from 6373 to 6380\n",
      "Complement found from 7578 to 7585\n",
      "Seq is: TATCTTA\n",
      "2 TATCTTA motifs detected in file\n",
      "\n",
      "Complement found from 1374 to 1381\n",
      "Seq is: CCCCAGA\n",
      "1 CCCCAGA motifs detected in file\n",
      "\n",
      "Complement found from 6572 to 6579\n",
      "Seq is: GGAGGAG\n",
      "1 GGAGGAG motifs detected in file\n",
      "\n",
      "Seq is: GCCGCCC\n",
      "0 GCCGCCC motifs detected in file\n",
      "\n",
      "Complement found from 14302 to 14309\n",
      "Seq is: CAGCTTC\n",
      "1 CAGCTTC motifs detected in file\n",
      "\n",
      "Complement found from 292 to 299\n",
      "Complement found from 13122 to 13129\n",
      "Complement found from 15403 to 15410\n",
      "Seq is: TTCCACC\n",
      "3 TTCCACC motifs detected in file\n",
      "\n",
      "Complement found from 10132 to 10139\n",
      "Seq is: ACAACTC\n",
      "1 ACAACTC motifs detected in file\n",
      "\n",
      "Complement found from 10999 to 11006\n",
      "Complement found from 14221 to 14228\n",
      "Seq is: CAACGCC\n",
      "2 CAACGCC motifs detected in file\n",
      "\n",
      "Complement found from 15504 to 15511\n",
      "Seq is: AGACAAT\n",
      "1 AGACAAT motifs detected in file\n",
      "\n",
      "Complement found from 276 to 283\n",
      "Complement found from 4951 to 4958\n",
      "Complement found from 5308 to 5315\n",
      "Complement found from 6118 to 6125\n",
      "Complement found from 7066 to 7073\n",
      "Complement found from 11778 to 11785\n",
      "Complement found from 12632 to 12639\n",
      "Complement found from 14342 to 14349\n",
      "Seq is: CATCATA\n",
      "8 CATCATA motifs detected in file\n",
      "\n",
      "Seq is: TCACTAC\n",
      "0 TCACTAC motifs detected in file\n",
      "\n",
      "Seq is: GGTCGTC\n",
      "0 GGTCGTC motifs detected in file\n",
      "\n",
      "Seq is: GATCCTG\n",
      "0 GATCCTG motifs detected in file\n",
      "\n",
      "Complement found from 722 to 729\n",
      "Complement found from 2064 to 2071\n",
      "Complement found from 3510 to 3517\n",
      "Seq is: ACCCTCT\n",
      "3 ACCCTCT motifs detected in file\n",
      "\n",
      "Complement found from 4686 to 4693\n",
      "Complement found from 7781 to 7788\n",
      "Complement found from 9124 to 9131\n",
      "Seq is: CTATCCT\n",
      "3 CTATCCT motifs detected in file\n",
      "\n",
      "Complement found from 9859 to 9866\n",
      "Complement found from 15291 to 15298\n",
      "Seq is: CTTCATC\n",
      "2 CTTCATC motifs detected in file\n",
      "\n",
      "Seq is: CTGACGA\n",
      "0 CTGACGA motifs detected in file\n",
      "\n",
      "Complement found from 10311 to 10318\n",
      "Complement found from 11246 to 11253\n",
      "Complement found from 11816 to 11823\n",
      "Complement found from 13147 to 13154\n",
      "Seq is: CACTAAT\n",
      "4 CACTAAT motifs detected in file\n",
      "\n",
      "Complement found from 139 to 146\n",
      "Complement found from 8786 to 8793\n",
      "Complement found from 14861 to 14868\n",
      "Seq is: CCTGCCT\n",
      "3 CCTGCCT motifs detected in file\n",
      "\n",
      "Complement found from 13202 to 13209\n",
      "Seq is: AGTCTGC\n",
      "1 AGTCTGC motifs detected in file\n",
      "\n",
      "Complement found from 5730 to 5737\n",
      "Seq is: TTCTCCC\n",
      "1 TTCTCCC motifs detected in file\n",
      "\n",
      "Complement found from 13529 to 13536\n",
      "Seq is: CGCAAAC\n",
      "1 CGCAAAC motifs detected in file\n",
      "\n",
      "Complement found from 4723 to 4730\n",
      "Complement found from 5065 to 5072\n",
      "Complement found from 6427 to 6434\n",
      "Complement found from 10606 to 10613\n",
      "Complement found from 10739 to 10746\n",
      "Complement found from 14143 to 14150\n",
      "Seq is: CATAACC\n",
      "6 CATAACC motifs detected in file\n",
      "\n",
      "Complement found from 6434 to 6441\n",
      "Seq is: CAATACC\n",
      "1 CAATACC motifs detected in file\n",
      "\n",
      "Seq is: GTCCCCC\n",
      "0 GTCCCCC motifs detected in file\n",
      "\n",
      "Complement found from 8443 to 8450\n",
      "Seq is: AAAATAT\n",
      "1 AAAATAT motifs detected in file\n",
      "\n",
      "Complement found from 5083 to 5090\n",
      "Complement found from 14508 to 14515\n",
      "Complement found from 16011 to 16018\n",
      "Seq is: AACTATT\n",
      "3 AACTATT motifs detected in file\n",
      "\n",
      "Complement found from 3300 to 3307\n",
      "Complement found from 4037 to 4044\n",
      "Complement found from 6194 to 6201\n",
      "Complement found from 10894 to 10901\n",
      "Complement found from 12028 to 12035\n",
      "Complement found from 12247 to 12254\n",
      "Complement found from 13743 to 13750\n",
      "Complement found from 13772 to 13779\n",
      "Complement found from 15677 to 15684\n",
      "Seq is: AACAACA\n",
      "9 AACAACA motifs detected in file\n",
      "\n",
      "Seq is: CTACTTT\n",
      "0 CTACTTT motifs detected in file\n",
      "\n",
      "Complement found from 2782 to 2789\n",
      "Complement found from 8458 to 8465\n",
      "Complement found from 10962 to 10969\n",
      "Complement found from 13848 to 13855\n",
      "Seq is: AACTACC\n",
      "4 AACTACC motifs detected in file\n",
      "\n",
      "Seq is: GGGGATT\n",
      "0 GGGGATT motifs detected in file\n",
      "\n",
      "Complement found from 5156 to 5163\n",
      "Complement found from 5481 to 5488\n",
      "Complement found from 6491 to 6498\n",
      "Complement found from 15848 to 15855\n",
      "Seq is: CTATCTC\n",
      "4 CTATCTC motifs detected in file\n",
      "\n",
      "Complement found from 11895 to 11902\n",
      "Seq is: CTCTGTG\n",
      "1 CTCTGTG motifs detected in file\n",
      "\n",
      "Seq is: GACGATC\n",
      "0 GACGATC motifs detected in file\n",
      "\n",
      "Complement found from 7411 to 7418\n",
      "Seq is: CACATTC\n",
      "1 CACATTC motifs detected in file\n",
      "\n",
      "Complement found from 3292 to 3299\n",
      "Complement found from 3550 to 3557\n",
      "Seq is: CTCTTCT\n",
      "2 CTCTTCT motifs detected in file\n",
      "\n",
      "Complement found from 2269 to 2276\n",
      "Complement found from 4736 to 4743\n",
      "Complement found from 9213 to 9220\n",
      "Complement found from 14365 to 14372\n",
      "Seq is: ACCAATC\n",
      "4 ACCAATC motifs detected in file\n",
      "\n",
      "Complement found from 8099 to 8106\n",
      "Complement found from 12820 to 12827\n",
      "Seq is: CAGATGC\n",
      "2 CAGATGC motifs detected in file\n",
      "\n",
      "Complement found from 2839 to 2846\n",
      "Seq is: CTCCGAG\n",
      "1 CTCCGAG motifs detected in file\n",
      "\n",
      "Complement found from 9590 to 9597\n",
      "Complement found from 15260 to 15267\n",
      "Seq is: GTCCCAC\n",
      "2 GTCCCAC motifs detected in file\n",
      "\n",
      "Complement found from 143 to 150\n",
      "Complement found from 5293 to 5300\n",
      "Complement found from 7806 to 7813\n",
      "Complement found from 8427 to 8434\n",
      "Complement found from 15639 to 15646\n",
      "Seq is: CCTCATC\n",
      "5 CCTCATC motifs detected in file\n",
      "\n",
      "Seq is: AAGGGAC\n",
      "0 AAGGGAC motifs detected in file\n",
      "\n",
      "Seq is: GATTCCC\n",
      "0 GATTCCC motifs detected in file\n",
      "\n",
      "Complement found from 7817 to 7824\n",
      "Seq is: TCCCATC\n",
      "1 TCCCATC motifs detected in file\n",
      "\n",
      "Complement found from 13588 to 13595\n",
      "Seq is: TGACAAG\n",
      "1 TGACAAG motifs detected in file\n",
      "\n",
      "Complement found from 2117 to 2124\n",
      "Complement found from 15794 to 15801\n",
      "Seq is: TTGGACA\n",
      "2 TTGGACA motifs detected in file\n",
      "\n",
      "Complement found from 8717 to 8724\n",
      "Seq is: AGGACGA\n",
      "1 AGGACGA motifs detected in file\n",
      "\n",
      "Complement found from 6259 to 6266\n",
      "Seq is: GGCCGGA\n",
      "1 GGCCGGA motifs detected in file\n",
      "\n",
      "Complement found from 2007 to 2014\n",
      "Seq is: GGTGATA\n",
      "1 GGTGATA motifs detected in file\n",
      "\n",
      "Seq is: TCGTCTA\n",
      "0 TCGTCTA motifs detected in file\n",
      "\n",
      "Seq is: CGCTCGT\n",
      "0 CGCTCGT motifs detected in file\n",
      "\n",
      "Complement found from 143 to 150\n",
      "Complement found from 5293 to 5300\n",
      "Complement found from 7806 to 7813\n",
      "Complement found from 8427 to 8434\n",
      "Complement found from 15639 to 15646\n",
      "Seq is: CCTCATC\n",
      "5 CCTCATC motifs detected in file\n",
      "\n",
      "Complement found from 4935 to 4942\n",
      "Complement found from 6223 to 6230\n",
      "Complement found from 11788 to 11795\n",
      "Seq is: CTCTCTC\n",
      "3 CTCTCTC motifs detected in file\n",
      "\n",
      "Complement found from 7365 to 7372\n",
      "Complement found from 7498 to 7505\n",
      "Complement found from 14053 to 14060\n",
      "Complement found from 14375 to 14382\n",
      "Complement found from 15663 to 15670\n",
      "Seq is: CCTCCAT\n",
      "5 CCTCCAT motifs detected in file\n",
      "\n",
      "Seq is: TCTCAGT\n",
      "0 TCTCAGT motifs detected in file\n",
      "\n",
      "Complement found from 5786 to 5793\n",
      "Complement found from 7332 to 7339\n",
      "Complement found from 9907 to 9914\n",
      "Complement found from 11121 to 11128\n",
      "Seq is: CTTCGAA\n",
      "4 CTTCGAA motifs detected in file\n",
      "\n",
      "Seq is: TACAACA\n",
      "0 TACAACA motifs detected in file\n",
      "\n",
      "Complement found from 3946 to 3953\n",
      "Complement found from 11487 to 11494\n",
      "Complement found from 14754 to 14761\n",
      "Seq is: AATACGC\n",
      "3 AATACGC motifs detected in file\n",
      "\n",
      "Seq is: CCCTTTG\n",
      "0 CCCTTTG motifs detected in file\n",
      "\n",
      "Seq is: CGGTATA\n",
      "0 CGGTATA motifs detected in file\n",
      "\n",
      "Complement found from 5230 to 5237\n",
      "Complement found from 6167 to 6174\n",
      "Complement found from 14242 to 14249\n",
      "Seq is: GCCCCCG\n",
      "3 GCCCCCG motifs detected in file\n",
      "\n",
      "Seq is: TGGCTAA\n",
      "0 TGGCTAA motifs detected in file\n",
      "\n",
      "Complement found from 4471 to 4478\n",
      "Complement found from 15653 to 15660\n",
      "Seq is: TAATCCC\n",
      "2 TAATCCC motifs detected in file\n",
      "\n",
      "Seq is: CTTGATC\n",
      "0 CTTGATC motifs detected in file\n",
      "\n",
      "Complement found from 13257 to 13264\n",
      "Seq is: AGTCAAC\n",
      "1 AGTCAAC motifs detected in file\n",
      "\n",
      "Seq is: GGTTTCG\n",
      "0 GGTTTCG motifs detected in file\n",
      "\n",
      "Complement found from 15242 to 15249\n",
      "Seq is: GAGGCTA\n",
      "1 GAGGCTA motifs detected in file\n",
      "\n",
      "Complement found from 469 to 476\n",
      "Complement found from 4731 to 4738\n",
      "Complement found from 6524 to 6531\n",
      "Complement found from 8157 to 8164\n",
      "Complement found from 8378 to 8385\n",
      "Seq is: ATACTAC\n",
      "5 ATACTAC motifs detected in file\n",
      "\n",
      "Complement found from 3850 to 3857\n",
      "Complement found from 4668 to 4675\n",
      "Complement found from 6828 to 6835\n",
      "Complement found from 8396 to 8403\n",
      "Complement found from 12592 to 12599\n",
      "Complement found from 14227 to 14234\n",
      "Seq is: CCATAAT\n",
      "6 CCATAAT motifs detected in file\n",
      "\n",
      "Seq is: GATACTT\n",
      "0 GATACTT motifs detected in file\n",
      "\n",
      "Complement found from 4098 to 4105\n",
      "Complement found from 4676 to 4683\n",
      "Complement found from 12942 to 12949\n",
      "Complement found from 13827 to 13834\n",
      "Seq is: CTTCTAA\n",
      "4 CTTCTAA motifs detected in file\n",
      "\n",
      "Complement found from 2295 to 2302\n",
      "Seq is: TAATGTT\n",
      "1 TAATGTT motifs detected in file\n",
      "\n",
      "Complement found from 10733 to 10740\n",
      "Seq is: TACGTAC\n",
      "1 TACGTAC motifs detected in file\n",
      "\n",
      "Seq is: CCGACAC\n",
      "0 CCGACAC motifs detected in file\n",
      "\n",
      "Seq is: TGCTATT\n",
      "0 TGCTATT motifs detected in file\n",
      "\n",
      "Seq is: GCAACAT\n",
      "0 GCAACAT motifs detected in file\n",
      "\n",
      "Complement found from 7108 to 7115\n",
      "Complement found from 8926 to 8933\n",
      "Complement found from 11211 to 11218\n",
      "Seq is: CTACACC\n",
      "3 CTACACC motifs detected in file\n",
      "\n",
      "Complement found from 767 to 774\n",
      "Seq is: AGCAATG\n",
      "1 AGCAATG motifs detected in file\n",
      "\n",
      "Seq is: GATCTTC\n",
      "0 GATCTTC motifs detected in file\n",
      "\n",
      "Complement found from 2911 to 2918\n",
      "Seq is: CAACGGA\n",
      "1 CAACGGA motifs detected in file\n",
      "\n",
      "Complement found from 5910 to 5917\n",
      "Seq is: CCGACCG\n",
      "1 CCGACCG motifs detected in file\n",
      "\n",
      "Seq is: GGTCGAG\n",
      "0 GGTCGAG motifs detected in file\n",
      "\n",
      "Complement found from 6018 to 6025\n",
      "Seq is: CCGAGCT\n",
      "1 CCGAGCT motifs detected in file\n",
      "\n",
      "Complement found from 7961 to 7968\n",
      "Complement found from 8423 to 8430\n",
      "Complement found from 14891 to 14898\n",
      "Seq is: TATTCCT\n",
      "3 TATTCCT motifs detected in file\n",
      "\n",
      "Complement found from 14265 to 14272\n",
      "Seq is: CCGAATC\n",
      "1 CCGAATC motifs detected in file\n",
      "\n",
      "Seq is: TCGACAC\n",
      "0 TCGACAC motifs detected in file\n",
      "\n",
      "Complement found from 14257 to 14264\n",
      "Seq is: GGATCCT\n",
      "1 GGATCCT motifs detected in file\n",
      "\n",
      "Seq is: GAGGTCG\n",
      "0 GAGGTCG motifs detected in file\n",
      "\n",
      "Seq is: AGTACGC\n",
      "0 AGTACGC motifs detected in file\n",
      "\n",
      "Seq is: GGCTTAT\n",
      "0 GGCTTAT motifs detected in file\n",
      "\n",
      "Complement found from 4947 to 4954\n",
      "Complement found from 12284 to 12291\n",
      "Complement found from 15632 to 15639\n",
      "Seq is: TATCCAT\n",
      "3 TATCCAT motifs detected in file\n",
      "\n",
      "Complement found from 3801 to 3808\n",
      "Complement found from 6395 to 6402\n",
      "Complement found from 8049 to 8056\n",
      "Seq is: ATCACAA\n",
      "3 ATCACAA motifs detected in file\n",
      "\n",
      "Seq is: GGTTACA\n",
      "0 GGTTACA motifs detected in file\n",
      "\n",
      "Seq is: GAAACAC\n",
      "0 GAAACAC motifs detected in file\n",
      "\n",
      "Complement found from 2235 to 2242\n",
      "Complement found from 3991 to 3998\n",
      "Complement found from 9892 to 9899\n",
      "Complement found from 12670 to 12677\n",
      "Complement found from 13531 to 13538\n",
      "Seq is: CAAACAT\n",
      "5 CAAACAT motifs detected in file\n",
      "\n",
      "Complement found from 8731 to 8738\n",
      "Complement found from 10331 to 10338\n",
      "Seq is: CTCTTAT\n",
      "2 CTCTTAT motifs detected in file\n",
      "\n",
      "Seq is: CAGTTGC\n",
      "0 CAGTTGC motifs detected in file\n",
      "\n",
      "Complement found from 3016 to 3023\n",
      "Seq is: CAGCCGC\n",
      "1 CAGCCGC motifs detected in file\n",
      "\n",
      "Complement found from 15891 to 15898\n",
      "Seq is: TTGTAGT\n",
      "1 TTGTAGT motifs detected in file\n",
      "\n",
      "Complement found from 430 to 437\n",
      "Complement found from 565 to 572\n",
      "Complement found from 799 to 806\n",
      "Complement found from 953 to 960\n",
      "Complement found from 5892 to 5899\n",
      "Complement found from 8269 to 8276\n",
      "Complement found from 13125 to 13132\n",
      "Seq is: CACCCCC\n",
      "7 CACCCCC motifs detected in file\n",
      "\n",
      "Complement found from 9318 to 9325\n",
      "Seq is: ACTCCAT\n",
      "1 ACTCCAT motifs detected in file\n",
      "\n",
      "Complement found from 12200 to 12207\n",
      "Complement found from 16322 to 16329\n",
      "Seq is: TTTACCG\n",
      "2 TTTACCG motifs detected in file\n",
      "\n",
      "Complement found from 11292 to 11299\n",
      "Seq is: ACTCACT\n",
      "1 ACTCACT motifs detected in file\n",
      "\n",
      "Complement found from 1559 to 1566\n",
      "Seq is: TCGTAAC\n",
      "1 TCGTAAC motifs detected in file\n",
      "\n",
      "Complement found from 4838 to 4845\n",
      "Complement found from 10798 to 10805\n",
      "Seq is: CTGACAT\n",
      "2 CTGACAT motifs detected in file\n",
      "\n",
      "Complement found from 5852 to 5859\n",
      "Seq is: TTAGATT\n",
      "1 TTAGATT motifs detected in file\n",
      "\n",
      "Seq is: TCTGTCC\n",
      "0 TCTGTCC motifs detected in file\n",
      "\n",
      "Complement found from 14366 to 14373\n",
      "Complement found from 16167 to 16174\n",
      "Seq is: CCAATCC\n",
      "2 CCAATCC motifs detected in file\n",
      "\n",
      "Seq is: GGAGAAA\n",
      "0 GGAGAAA motifs detected in file\n",
      "\n",
      "Seq is: AATGGTC\n",
      "0 AATGGTC motifs detected in file\n",
      "\n",
      "Seq is: GAGGCTC\n",
      "0 GAGGCTC motifs detected in file\n",
      "\n",
      "Complement found from 4911 to 4918\n",
      "Complement found from 8711 to 8718\n",
      "Complement found from 11276 to 11283\n",
      "Complement found from 14391 to 14398\n",
      "Seq is: CACTAAA\n",
      "4 CACTAAA motifs detected in file\n",
      "\n",
      "Complement found from 2302 to 2309\n",
      "Complement found from 9998 to 10005\n",
      "Complement found from 15895 to 15902\n",
      "Seq is: AGTATAA\n",
      "3 AGTATAA motifs detected in file\n",
      "\n",
      "Seq is: CTTAACG\n",
      "0 CTTAACG motifs detected in file\n",
      "\n",
      "Seq is: TTTAAGC\n",
      "0 TTTAAGC motifs detected in file\n",
      "\n",
      "Complement found from 5784 to 5791\n",
      "Complement found from 6557 to 6564\n",
      "Complement found from 9482 to 9489\n",
      "Complement found from 11119 to 11126\n",
      "Seq is: TTCTTCG\n",
      "4 TTCTTCG motifs detected in file\n",
      "\n",
      "Seq is: TCGAAGT\n",
      "0 TCGAAGT motifs detected in file\n",
      "\n",
      "Complement found from 1591 to 1598\n",
      "Seq is: TTGGACG\n",
      "1 TTGGACG motifs detected in file\n",
      "\n",
      "Complement found from 3525 to 3532\n",
      "Complement found from 7210 to 7217\n",
      "Seq is: GCCCCGA\n",
      "2 GCCCCGA motifs detected in file\n",
      "\n",
      "Complement found from 12767 to 12774\n",
      "Seq is: AGAGGGC\n",
      "1 AGAGGGC motifs detected in file\n",
      "\n",
      "Complement found from 2127 to 2134\n",
      "Complement found from 5744 to 5751\n",
      "Complement found from 6689 to 6696\n",
      "Seq is: GGAAAAA\n",
      "3 GGAAAAA motifs detected in file\n",
      "\n",
      "Seq is: AGGGCCG\n",
      "0 AGGGCCG motifs detected in file\n",
      "\n",
      "Seq is: CCGCCCT\n",
      "0 CCGCCCT motifs detected in file\n",
      "\n",
      "Complement found from 9859 to 9866\n",
      "Complement found from 15291 to 15298\n",
      "Seq is: CTTCATC\n",
      "2 CTTCATC motifs detected in file\n",
      "\n",
      "Complement found from 10015 to 10022\n",
      "Seq is: TAACTTC\n",
      "1 TAACTTC motifs detected in file\n",
      "\n",
      "Complement found from 3590 to 3597\n",
      "Complement found from 7851 to 7858\n",
      "Complement found from 13632 to 13639\n",
      "Seq is: GGTCAAC\n",
      "3 GGTCAAC motifs detected in file\n",
      "\n",
      "Complement found from 4471 to 4478\n",
      "Complement found from 15653 to 15660\n",
      "Seq is: TAATCCC\n",
      "2 TAATCCC motifs detected in file\n",
      "\n",
      "Seq is: ACGAATC\n",
      "0 ACGAATC motifs detected in file\n",
      "\n",
      "Complement found from 2365 to 2372\n",
      "Complement found from 15505 to 15512\n",
      "Seq is: GACAATT\n",
      "2 GACAATT motifs detected in file\n",
      "\n",
      "Seq is: GATTCAC\n",
      "0 GATTCAC motifs detected in file\n",
      "\n",
      "Complement found from 4007 to 4014\n",
      "Complement found from 12057 to 12064\n",
      "Complement found from 15531 to 15538\n",
      "Seq is: AAACACC\n",
      "3 AAACACC motifs detected in file\n",
      "\n",
      "Complement found from 3109 to 3116\n",
      "Seq is: CAAATTC\n",
      "1 CAAATTC motifs detected in file\n",
      "\n",
      "Seq is: AGGGTAA\n",
      "0 AGGGTAA motifs detected in file\n",
      "\n",
      "Seq is: CCAGATC\n",
      "0 CCAGATC motifs detected in file\n",
      "\n",
      "Complement found from 8107 to 8114\n",
      "Seq is: ATTCCCG\n",
      "1 ATTCCCG motifs detected in file\n",
      "\n",
      "Complement found from 4341 to 4348\n",
      "Complement found from 7458 to 7465\n",
      "Seq is: AATCGAA\n",
      "2 AATCGAA motifs detected in file\n",
      "\n",
      "Complement found from 2737 to 2744\n",
      "Complement found from 5076 to 5083\n",
      "Complement found from 5569 to 5576\n",
      "Complement found from 10070 to 10077\n",
      "Seq is: TTAATTT\n",
      "4 TTAATTT motifs detected in file\n",
      "\n",
      "Complement found from 7914 to 7921\n",
      "Seq is: CACCGAC\n",
      "1 CACCGAC motifs detected in file\n",
      "\n",
      "Complement found from 12946 to 12953\n",
      "Complement found from 13697 to 13704\n",
      "Seq is: TAAACGC\n",
      "2 TAAACGC motifs detected in file\n",
      "\n",
      "Complement found from 917 to 924\n",
      "Complement found from 13256 to 13263\n",
      "Seq is: AAGTCAA\n",
      "2 AAGTCAA motifs detected in file\n",
      "\n",
      "Seq is: CTACGTC\n",
      "0 CTACGTC motifs detected in file\n",
      "\n",
      "Complement found from 429 to 436\n",
      "Complement found from 633 to 640\n",
      "Complement found from 952 to 959\n",
      "Complement found from 5891 to 5898\n",
      "Complement found from 9567 to 9574\n",
      "Complement found from 12964 to 12971\n",
      "Seq is: TCACCCC\n",
      "6 TCACCCC motifs detected in file\n",
      "\n",
      "Seq is: AAAACGT\n",
      "0 AAAACGT motifs detected in file\n",
      "\n",
      "Complement found from 13099 to 13106\n",
      "Complement found from 13477 to 13484\n",
      "Seq is: CAGGAAT\n",
      "2 CAGGAAT motifs detected in file\n",
      "\n",
      "Seq is: CGACAAT\n",
      "0 CGACAAT motifs detected in file\n",
      "\n",
      "Complement found from 5848 to 5855\n",
      "Complement found from 15966 to 15973\n",
      "Seq is: GTCTTTA\n",
      "2 GTCTTTA motifs detected in file\n",
      "\n",
      "Seq is: ATTCATA\n",
      "0 ATTCATA motifs detected in file\n",
      "\n",
      "Complement found from 7994 to 8001\n",
      "Seq is: ACGTTGA\n",
      "1 ACGTTGA motifs detected in file\n",
      "\n",
      "Complement found from 11400 to 11407\n",
      "Seq is: ATGACTC\n",
      "1 ATGACTC motifs detected in file\n",
      "\n",
      "Complement found from 1903 to 1910\n",
      "Complement found from 9670 to 9677\n",
      "Seq is: CCGAAAC\n",
      "2 CCGAAAC motifs detected in file\n",
      "\n",
      "Seq is: TTCCGAG\n",
      "0 TTCCGAG motifs detected in file\n",
      "\n",
      "Complement found from 383 to 390\n",
      "Complement found from 1598 to 1605\n",
      "Complement found from 1907 to 1914\n",
      "Seq is: AACCAGA\n",
      "3 AACCAGA motifs detected in file\n",
      "\n",
      "Complement found from 10483 to 10490\n",
      "Complement found from 14291 to 14298\n",
      "Complement found from 14591 to 14598\n",
      "Complement found from 16133 to 16140\n",
      "Seq is: CATAAAT\n",
      "4 CATAAAT motifs detected in file\n",
      "\n",
      "Complement found from 1419 to 1426\n",
      "Seq is: TGGATTT\n",
      "1 TGGATTT motifs detected in file\n",
      "\n",
      "Complement found from 11373 to 11380\n",
      "Seq is: AAAGATA\n",
      "1 AAAGATA motifs detected in file\n",
      "\n",
      "Complement found from 3343 to 3350\n",
      "Complement found from 4677 to 4684\n",
      "Complement found from 9109 to 9116\n",
      "Complement found from 16001 to 16008\n",
      "Seq is: TTCTAAT\n",
      "4 TTCTAAT motifs detected in file\n",
      "\n",
      "Complement found from 4759 to 4766\n",
      "Complement found from 4888 to 4895\n",
      "Complement found from 10454 to 10461\n",
      "Complement found from 11103 to 11110\n",
      "Complement found from 14231 to 14238\n",
      "Seq is: AATCATA\n",
      "5 AATCATA motifs detected in file\n",
      "\n",
      "Complement found from 451 to 458\n",
      "Seq is: TTTTCCC\n",
      "1 TTTTCCC motifs detected in file\n",
      "\n",
      "Complement found from 6370 to 6377\n",
      "Seq is: CTCTATC\n",
      "1 CTCTATC motifs detected in file\n",
      "\n",
      "Complement found from 11723 to 11730\n",
      "Complement found from 12881 to 12888\n",
      "Complement found from 15636 to 15643\n",
      "Complement found from 15660 to 15667\n",
      "Complement found from 16403 to 16410\n",
      "Seq is: CATCCTC\n",
      "5 CATCCTC motifs detected in file\n",
      "\n",
      "Complement found from 159 to 166\n",
      "Complement found from 11242 to 11249\n",
      "Seq is: ATCGCAC\n",
      "2 ATCGCAC motifs detected in file\n",
      "\n",
      "Complement found from 4244 to 4251\n",
      "Complement found from 14484 to 14491\n",
      "Seq is: CATTCCC\n",
      "2 CATTCCC motifs detected in file\n",
      "\n",
      "Complement found from 5470 to 5477\n",
      "Complement found from 10598 to 10605\n",
      "Complement found from 15245 to 15252\n",
      "Complement found from 16437 to 16444\n",
      "Seq is: GCTACTC\n",
      "4 GCTACTC motifs detected in file\n",
      "\n",
      "Complement found from 564 to 571\n",
      "Complement found from 798 to 805\n",
      "Complement found from 4387 to 4394\n",
      "Complement found from 8928 to 8935\n",
      "Complement found from 15533 to 15540\n",
      "Seq is: ACACCCC\n",
      "5 ACACCCC motifs detected in file\n",
      "\n",
      "Complement found from 6347 to 6354\n",
      "Complement found from 8414 to 8421\n",
      "Complement found from 12391 to 12398\n",
      "Seq is: TCCTTAC\n",
      "3 TCCTTAC motifs detected in file\n",
      "\n",
      "Complement found from 3209 to 3216\n",
      "Complement found from 5432 to 5439\n",
      "Complement found from 6312 to 6319\n",
      "Complement found from 7399 to 7406\n",
      "Complement found from 13053 to 13060\n",
      "Complement found from 13647 to 13654\n",
      "Complement found from 13677 to 13684\n",
      "Complement found from 14809 to 14816\n",
      "Complement found from 15262 to 15269\n",
      "Complement found from 16289 to 16296\n",
      "Seq is: CCCACCC\n",
      "10 CCCACCC motifs detected in file\n",
      "\n",
      "Complement found from 2317 to 2324\n",
      "Complement found from 2503 to 2510\n",
      "Complement found from 12154 to 12161\n",
      "Seq is: AAAACAT\n",
      "3 AAAACAT motifs detected in file\n",
      "\n",
      "Complement found from 2918 to 2925\n",
      "Seq is: ACAAGTT\n",
      "1 ACAAGTT motifs detected in file\n",
      "\n",
      "Complement found from 5411 to 5418\n",
      "Seq is: TGACAGT\n",
      "1 TGACAGT motifs detected in file\n",
      "\n",
      "Complement found from 4607 to 4614\n",
      "Complement found from 5402 to 5409\n",
      "Complement found from 8491 to 8498\n",
      "Complement found from 13669 to 13676\n",
      "Complement found from 13869 to 13876\n",
      "Seq is: AAAATAA\n",
      "5 AAAATAA motifs detected in file\n",
      "\n",
      "Seq is: AAATGCA\n",
      "0 AAATGCA motifs detected in file\n",
      "\n",
      "Complement found from 3720 to 3727\n",
      "Complement found from 4023 to 4030\n",
      "Complement found from 4233 to 4240\n",
      "Seq is: ACAATCT\n",
      "3 ACAATCT motifs detected in file\n",
      "\n",
      "Complement found from 537 to 544\n",
      "Complement found from 1079 to 1086\n",
      "Complement found from 4133 to 4140\n",
      "Complement found from 4432 to 4439\n",
      "Complement found from 12428 to 12435\n",
      "Seq is: ATACCCC\n",
      "5 ATACCCC motifs detected in file\n",
      "\n",
      "Complement found from 4521 to 4528\n",
      "Complement found from 6393 to 6400\n",
      "Complement found from 6983 to 6990\n",
      "Complement found from 8429 to 8436\n",
      "Complement found from 11153 to 11160\n",
      "Complement found from 14059 to 14066\n",
      "Seq is: TCATCAC\n",
      "6 TCATCAC motifs detected in file\n",
      "\n",
      "Complement found from 7731 to 7738\n",
      "Complement found from 10959 to 10966\n",
      "Seq is: ACTAACT\n",
      "2 ACTAACT motifs detected in file\n",
      "\n",
      "Complement found from 1237 to 1244\n",
      "Complement found from 3625 to 3632\n",
      "Complement found from 5364 to 5371\n",
      "Complement found from 8620 to 8627\n",
      "Complement found from 14050 to 14057\n",
      "Complement found from 15330 to 15337\n",
      "Seq is: CCACCTC\n",
      "6 CCACCTC motifs detected in file\n",
      "\n",
      "Seq is: ATCTAAT\n",
      "0 ATCTAAT motifs detected in file\n",
      "\n",
      "Complement found from 4662 to 4669\n",
      "Complement found from 13756 to 13763\n",
      "Seq is: CCGCATC\n",
      "2 CCGCATC motifs detected in file\n",
      "\n",
      "Complement found from 1264 to 1271\n",
      "Complement found from 6340 to 6347\n",
      "Complement found from 9098 to 9105\n",
      "Complement found from 12701 to 12708\n",
      "Seq is: CATCTTC\n",
      "4 CATCTTC motifs detected in file\n",
      "\n",
      "Complement found from 3864 to 3871\n",
      "Complement found from 4236 to 4243\n",
      "Complement found from 10709 to 10716\n",
      "Complement found from 14046 to 14053\n",
      "Seq is: ATCTCCA\n",
      "4 ATCTCCA motifs detected in file\n",
      "\n",
      "Complement found from 3112 to 3119\n",
      "Complement found from 3288 to 3295\n",
      "Complement found from 5440 to 5447\n",
      "Complement found from 8424 to 8431\n",
      "Seq is: ATTCCTC\n",
      "4 ATTCCTC motifs detected in file\n",
      "\n",
      "Complement found from 2217 to 2224\n",
      "Complement found from 4018 to 4025\n",
      "Complement found from 12969 to 12976\n",
      "Seq is: CCACTAC\n",
      "3 CCACTAC motifs detected in file\n",
      "\n",
      "Seq is: CACCGAT\n",
      "0 CACCGAT motifs detected in file\n",
      "\n",
      "Complement found from 1701 to 1708\n",
      "Complement found from 2783 to 2790\n",
      "Complement found from 4733 to 4740\n",
      "Complement found from 8459 to 8466\n",
      "Complement found from 10126 to 10133\n",
      "Complement found from 10791 to 10798\n",
      "Seq is: ACTACCA\n",
      "6 ACTACCA motifs detected in file\n",
      "\n",
      "Complement found from 1530 to 1537\n",
      "Complement found from 3428 to 3435\n",
      "Complement found from 9525 to 9532\n",
      "Complement found from 10277 to 10284\n",
      "Complement found from 11233 to 11240\n",
      "Complement found from 13980 to 13987\n",
      "Seq is: CCCCTAC\n",
      "6 CCCCTAC motifs detected in file\n",
      "\n",
      "Complement found from 6683 to 6690\n",
      "Seq is: TACTCCG\n",
      "1 TACTCCG motifs detected in file\n",
      "\n",
      "Complement found from 237 to 244\n",
      "Complement found from 280 to 287\n",
      "Complement found from 8504 to 8511\n",
      "Complement found from 11569 to 11576\n",
      "Seq is: ATAACAA\n",
      "4 ATAACAA motifs detected in file\n",
      "\n",
      "Complement found from 2352 to 2359\n",
      "Complement found from 10815 to 10822\n",
      "Complement found from 11520 to 11527\n",
      "Complement found from 12056 to 12063\n",
      "Complement found from 14395 to 14402\n",
      "Seq is: AAAACAC\n",
      "5 AAAACAC motifs detected in file\n",
      "\n",
      "Complement found from 2192 to 2199\n",
      "Seq is: TTAAGAA\n",
      "1 TTAAGAA motifs detected in file\n",
      "\n",
      "Complement found from 3021 to 3028\n",
      "Complement found from 10232 to 10239\n",
      "Complement found from 14015 to 14022\n",
      "Seq is: GCTATTA\n",
      "3 GCTATTA motifs detected in file\n",
      "\n",
      "Seq is: CCGGGTA\n",
      "0 CCGGGTA motifs detected in file\n",
      "\n",
      "Complement found from 4485 to 4492\n",
      "Complement found from 7422 to 7429\n",
      "Seq is: AACCCGT\n",
      "2 AACCCGT motifs detected in file\n",
      "\n",
      "Complement found from 6612 to 6619\n",
      "Seq is: TTTTCGG\n",
      "1 TTTTCGG motifs detected in file\n",
      "\n",
      "Seq is: CCAATCG\n",
      "0 CCAATCG motifs detected in file\n",
      "\n",
      "Complement found from 3167 to 3174\n",
      "Seq is: CCCCCGT\n",
      "1 CCCCCGT motifs detected in file\n",
      "\n",
      "Complement found from 6570 to 6577\n",
      "Seq is: CCGGAGG\n",
      "1 CCGGAGG motifs detected in file\n",
      "\n",
      "Complement found from 1187 to 1194\n",
      "Complement found from 7859 to 7866\n",
      "Complement found from 10327 to 10334\n",
      "Complement found from 10867 to 10874\n",
      "Complement found from 12099 to 12106\n",
      "Seq is: ATCCCTC\n",
      "5 ATCCCTC motifs detected in file\n",
      "\n",
      "Complement found from 3114 to 3121\n",
      "Complement found from 5442 to 5449\n",
      "Complement found from 10550 to 10557\n",
      "Complement found from 14260 to 14267\n",
      "Complement found from 15140 to 15147\n",
      "Seq is: TCCTCCC\n",
      "5 TCCTCCC motifs detected in file\n",
      "\n",
      "Complement found from 1994 to 2001\n",
      "Complement found from 8465 to 8472\n",
      "Complement found from 16284 to 16291\n",
      "Seq is: ACCTACC\n",
      "3 ACCTACC motifs detected in file\n",
      "\n",
      "Complement found from 5195 to 5202\n",
      "Complement found from 8221 to 8228\n",
      "Seq is: TTAATTC\n",
      "2 TTAATTC motifs detected in file\n",
      "\n",
      "Complement found from 8560 to 8567\n",
      "Seq is: CCACAAT\n",
      "1 CCACAAT motifs detected in file\n",
      "\n",
      "Complement found from 2846 to 2853\n",
      "Complement found from 16300 to 16307\n",
      "Seq is: CAGTACA\n",
      "2 CAGTACA motifs detected in file\n",
      "\n",
      "Complement found from 4342 to 4349\n",
      "Complement found from 7459 to 7466\n",
      "Seq is: ATCGAAC\n",
      "2 ATCGAAC motifs detected in file\n",
      "\n",
      "Complement found from 3050 to 3057\n",
      "Complement found from 7343 to 7350\n",
      "Seq is: AAAGTCC\n",
      "2 AAAGTCC motifs detected in file\n",
      "\n",
      "Seq is: ACGCTCT\n",
      "0 ACGCTCT motifs detected in file\n",
      "\n",
      "Complement found from 10340 to 10347\n",
      "Complement found from 11152 to 11159\n",
      "Complement found from 14058 to 14065\n",
      "Seq is: ATCATCA\n",
      "3 ATCATCA motifs detected in file\n",
      "\n",
      "Seq is: TCCCAGC\n",
      "0 TCCCAGC motifs detected in file\n",
      "\n",
      "Complement found from 734 to 741\n",
      "Complement found from 5141 to 5148\n",
      "Complement found from 14692 to 14699\n",
      "Seq is: ACCACGA\n",
      "3 ACCACGA motifs detected in file\n",
      "\n",
      "Complement found from 593 to 600\n",
      "Complement found from 4251 to 4258\n",
      "Complement found from 8683 to 8690\n",
      "Seq is: CCTCAAA\n",
      "3 CCTCAAA motifs detected in file\n",
      "\n",
      "Complement found from 1646 to 1653\n",
      "Complement found from 2040 to 2047\n",
      "Complement found from 7936 to 7943\n",
      "Complement found from 14727 to 14734\n",
      "Seq is: TTCAACT\n",
      "4 TTCAACT motifs detected in file\n",
      "\n",
      "Complement found from 5296 to 5303\n",
      "Complement found from 7794 to 7801\n",
      "Complement found from 10342 to 10349\n",
      "Complement found from 10863 to 10870\n",
      "Complement found from 13370 to 13377\n",
      "Complement found from 13517 to 13524\n",
      "Complement found from 14057 to 14064\n",
      "Seq is: CATCATC\n",
      "7 CATCATC motifs detected in file\n",
      "\n",
      "Seq is: CTTACGC\n",
      "0 CTTACGC motifs detected in file\n",
      "\n",
      "Complement found from 5296 to 5303\n",
      "Complement found from 7794 to 7801\n",
      "Complement found from 10342 to 10349\n",
      "Complement found from 10863 to 10870\n",
      "Complement found from 13370 to 13377\n",
      "Complement found from 13517 to 13524\n",
      "Complement found from 14057 to 14064\n",
      "Seq is: CATCATC\n",
      "7 CATCATC motifs detected in file\n",
      "\n",
      "Complement found from 2944 to 2951\n",
      "Complement found from 7569 to 7576\n",
      "Complement found from 8564 to 8571\n",
      "Complement found from 9454 to 9461\n",
      "Complement found from 12848 to 12855\n",
      "Complement found from 14368 to 14375\n",
      "Complement found from 15828 to 15835\n",
      "Seq is: AATCCTA\n",
      "7 AATCCTA motifs detected in file\n",
      "\n",
      "Complement found from 5089 to 5096\n",
      "Complement found from 6631 to 6638\n",
      "Complement found from 10784 to 10791\n",
      "Seq is: TTATATT\n",
      "3 TTATATT motifs detected in file\n",
      "\n",
      "Complement found from 6406 to 6413\n",
      "Complement found from 6735 to 6742\n",
      "Complement found from 7044 to 7051\n",
      "Complement found from 9630 to 9637\n",
      "Seq is: TATCAAT\n",
      "4 TATCAAT motifs detected in file\n",
      "\n",
      "Complement found from 5195 to 5202\n",
      "Complement found from 8221 to 8228\n",
      "Seq is: TTAATTC\n",
      "2 TTAATTC motifs detected in file\n",
      "\n",
      "Complement found from 2497 to 2504\n",
      "Complement found from 10462 to 10469\n",
      "Seq is: TTACCAA\n",
      "2 TTACCAA motifs detected in file\n",
      "\n",
      "Complement found from 2386 to 2393\n",
      "Complement found from 4022 to 4029\n",
      "Complement found from 4232 to 4239\n",
      "Complement found from 13280 to 13287\n",
      "Seq is: TACAATC\n",
      "4 TACAATC motifs detected in file\n",
      "\n",
      "Complement found from 13903 to 13910\n",
      "Complement found from 14819 to 14826\n",
      "Seq is: CCAACAT\n",
      "2 CCAACAT motifs detected in file\n",
      "\n",
      "Complement found from 1262 to 1269\n",
      "Seq is: GCCATCT\n",
      "1 GCCATCT motifs detected in file\n",
      "\n",
      "Seq is: TGACGAT\n",
      "0 TGACGAT motifs detected in file\n",
      "\n",
      "Complement found from 3770 to 3777\n",
      "Seq is: AATAAGT\n",
      "1 AATAAGT motifs detected in file\n",
      "\n",
      "Complement found from 14436 to 14443\n",
      "Complement found from 16271 to 16278\n",
      "Seq is: AGGATAC\n",
      "2 AGGATAC motifs detected in file\n",
      "\n",
      "Complement found from 3338 to 3345\n",
      "Complement found from 4226 to 4233\n",
      "Seq is: ACCCATT\n",
      "2 ACCCATT motifs detected in file\n",
      "\n",
      "Complement found from 2249 to 2256\n",
      "Complement found from 7939 to 7946\n",
      "Complement found from 11320 to 11327\n",
      "Seq is: AACTCCT\n",
      "3 AACTCCT motifs detected in file\n",
      "\n",
      "Seq is: CATACGA\n",
      "0 CATACGA motifs detected in file\n",
      "\n",
      "Complement found from 12943 to 12950\n",
      "Seq is: TTCTAAA\n",
      "1 TTCTAAA motifs detected in file\n",
      "\n",
      "Complement found from 7827 to 7834\n",
      "Seq is: ACGCATC\n",
      "1 ACGCATC motifs detected in file\n",
      "\n",
      "Complement found from 1061 to 1068\n",
      "Complement found from 12666 to 12673\n",
      "Seq is: GACCCAA\n",
      "2 GACCCAA motifs detected in file\n",
      "\n",
      "Complement found from 4896 to 4903\n",
      "Complement found from 9675 to 9682\n",
      "Complement found from 10464 to 10471\n",
      "Complement found from 10886 to 10893\n",
      "Complement found from 14041 to 14048\n",
      "Seq is: ACCAAAT\n",
      "5 ACCAAAT motifs detected in file\n",
      "\n",
      "Complement found from 6951 to 6958\n",
      "Seq is: TAGGTGG\n",
      "1 TAGGTGG motifs detected in file\n",
      "\n",
      "Complement found from 1002 to 1009\n",
      "Seq is: AGTTGAC\n",
      "1 AGTTGAC motifs detected in file\n",
      "\n",
      "Seq is: GGACGAT\n",
      "0 GGACGAT motifs detected in file\n",
      "\n",
      "Complement found from 2219 to 2226\n",
      "Complement found from 10963 to 10970\n",
      "Complement found from 13849 to 13856\n",
      "Seq is: ACTACCT\n",
      "3 ACTACCT motifs detected in file\n",
      "\n",
      "Complement found from 3342 to 3349\n",
      "Complement found from 9108 to 9115\n",
      "Complement found from 15745 to 15752\n",
      "Complement found from 16000 to 16007\n",
      "Seq is: ATTCTAA\n",
      "4 ATTCTAA motifs detected in file\n",
      "\n",
      "Complement found from 4935 to 4942\n",
      "Complement found from 6223 to 6230\n",
      "Complement found from 11788 to 11795\n",
      "Seq is: CTCTCTC\n",
      "3 CTCTCTC motifs detected in file\n",
      "\n",
      "Complement found from 2250 to 2257\n",
      "Complement found from 3326 to 3333\n",
      "Complement found from 5010 to 5017\n",
      "Complement found from 13985 to 13992\n",
      "Complement found from 14441 to 14448\n",
      "Seq is: ACTCCTC\n",
      "5 ACTCCTC motifs detected in file\n",
      "\n",
      "Seq is: TTCCGAA\n",
      "0 TTCCGAA motifs detected in file\n",
      "\n",
      "Seq is: TGCAAAT\n",
      "0 TGCAAAT motifs detected in file\n",
      "\n",
      "Complement found from 462 to 469\n",
      "Complement found from 10622 to 10629\n",
      "Seq is: CACTCCC\n",
      "2 CACTCCC motifs detected in file\n",
      "\n",
      "Complement found from 7283 to 7290\n",
      "Seq is: TCTCTAA\n",
      "1 TCTCTAA motifs detected in file\n",
      "\n",
      "Complement found from 6771 to 6778\n",
      "Complement found from 9362 to 9369\n",
      "Seq is: ACCATAT\n",
      "2 ACCATAT motifs detected in file\n",
      "\n",
      "Complement found from 7731 to 7738\n",
      "Complement found from 10959 to 10966\n",
      "Seq is: ACTAACT\n",
      "2 ACTAACT motifs detected in file\n",
      "\n",
      "Complement found from 1371 to 1378\n",
      "Complement found from 7228 to 7235\n",
      "Complement found from 8277 to 8284\n",
      "Complement found from 9528 to 9535\n",
      "Complement found from 10975 to 10982\n",
      "Complement found from 11532 to 11539\n",
      "Seq is: CTACCCC\n",
      "6 CTACCCC motifs detected in file\n",
      "\n",
      "Complement found from 738 to 745\n",
      "Complement found from 1225 to 1232\n",
      "Seq is: CGATCAA\n",
      "2 CGATCAA motifs detected in file\n",
      "\n",
      "Complement found from 2752 to 2759\n",
      "Seq is: AAACAGT\n",
      "1 AAACAGT motifs detected in file\n",
      "\n",
      "Seq is: ACACTCT\n",
      "0 ACACTCT motifs detected in file\n",
      "\n",
      "Complement found from 6450 to 6457\n",
      "Seq is: TCTTCGT\n",
      "1 TCTTCGT motifs detected in file\n",
      "\n",
      "Complement found from 4845 to 4852\n",
      "Complement found from 9291 to 9298\n",
      "Seq is: CCGGCCT\n",
      "2 CCGGCCT motifs detected in file\n",
      "\n",
      "Complement found from 6281 to 6288\n",
      "Seq is: ACAGTCT\n",
      "1 ACAGTCT motifs detected in file\n",
      "\n",
      "Complement found from 802 to 809\n",
      "Seq is: CCCCACG\n",
      "1 CCCCACG motifs detected in file\n",
      "\n",
      "Complement found from 2830 to 2837\n",
      "Complement found from 4345 to 4352\n",
      "Seq is: GAACCCA\n",
      "2 GAACCCA motifs detected in file\n",
      "\n",
      "Seq is: TTGGAGA\n",
      "0 TTGGAGA motifs detected in file\n",
      "\n",
      "Complement found from 4353 to 4360\n",
      "Complement found from 8512 to 8519\n",
      "Seq is: CCCTGAG\n",
      "2 CCCTGAG motifs detected in file\n",
      "\n",
      "Complement found from 3469 to 3476\n",
      "Complement found from 9100 to 9107\n",
      "Seq is: TCTTCAC\n",
      "2 TCTTCAC motifs detected in file\n",
      "\n",
      "Complement found from 452 to 459\n",
      "Complement found from 6182 to 6189\n",
      "Complement found from 7091 to 7098\n",
      "Complement found from 8601 to 8608\n",
      "Complement found from 13750 to 13757\n",
      "Seq is: TTTCCCC\n",
      "5 TTTCCCC motifs detected in file\n",
      "\n",
      "Seq is: CGATAAG\n",
      "0 CGATAAG motifs detected in file\n",
      "\n",
      "Complement found from 739 to 746\n",
      "Complement found from 11921 to 11928\n",
      "Complement found from 15356 to 15363\n",
      "Seq is: GATCAAA\n",
      "3 GATCAAA motifs detected in file\n",
      "\n",
      "Seq is: ATAACGA\n",
      "0 ATAACGA motifs detected in file\n",
      "\n",
      "Seq is: TATCGAT\n",
      "0 TATCGAT motifs detected in file\n",
      "\n",
      "Complement found from 4730 to 4737\n",
      "Complement found from 7738 to 7745\n",
      "Complement found from 8377 to 8384\n",
      "Complement found from 10522 to 10529\n",
      "Complement found from 10576 to 10583\n",
      "Complement found from 10956 to 10963\n",
      "Complement found from 14578 to 14585\n",
      "Seq is: AATACTA\n",
      "7 AATACTA motifs detected in file\n",
      "\n",
      "Complement found from 8041 to 8048\n",
      "Complement found from 8398 to 8405\n",
      "Complement found from 10109 to 10116\n",
      "Complement found from 11563 to 11570\n",
      "Complement found from 14082 to 14089\n",
      "Seq is: ATAATTA\n",
      "5 ATAATTA motifs detected in file\n",
      "\n",
      "Complement found from 5120 to 5127\n",
      "Complement found from 8970 to 8977\n",
      "Complement found from 9030 to 9037\n",
      "Complement found from 11236 to 11243\n",
      "Complement found from 11290 to 11297\n",
      "Complement found from 12696 to 12703\n",
      "Complement found from 13419 to 13426\n",
      "Complement found from 14907 to 14914\n",
      "Complement found from 15069 to 15076\n",
      "Complement found from 15246 to 15253\n",
      "Seq is: CTACTCA\n",
      "10 CTACTCA motifs detected in file\n",
      "\n",
      "Complement found from 849 to 856\n",
      "Complement found from 1521 to 1528\n",
      "Complement found from 7733 to 7740\n",
      "Seq is: TAACTAA\n",
      "3 TAACTAA motifs detected in file\n",
      "\n",
      "Complement found from 3745 to 3752\n",
      "Complement found from 3834 to 3841\n",
      "Complement found from 4950 to 4957\n",
      "Complement found from 5307 to 5314\n",
      "Complement found from 6117 to 6124\n",
      "Complement found from 7065 to 7072\n",
      "Complement found from 7793 to 7800\n",
      "Complement found from 12631 to 12638\n",
      "Complement found from 13369 to 13376\n",
      "Complement found from 14056 to 14063\n",
      "Complement found from 14341 to 14348\n",
      "Complement found from 14480 to 14487\n",
      "Complement found from 15788 to 15795\n",
      "Seq is: CCATCAT\n",
      "13 CCATCAT motifs detected in file\n",
      "\n",
      "Complement found from 4726 to 4733\n",
      "Complement found from 8980 to 8987\n",
      "Seq is: AACCAAT\n",
      "2 AACCAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCAAGT\n",
      "0 ACCAAGT motifs detected in file\n",
      "\n",
      "Complement found from 9559 to 9566\n",
      "Seq is: AACAGGC\n",
      "1 AACAGGC motifs detected in file\n",
      "\n",
      "Complement found from 10602 to 10609\n",
      "Complement found from 13570 to 13577\n",
      "Seq is: CTCTCAT\n",
      "2 CTCTCAT motifs detected in file\n",
      "\n",
      "Complement found from 237 to 244\n",
      "Complement found from 280 to 287\n",
      "Complement found from 8504 to 8511\n",
      "Complement found from 11569 to 11576\n",
      "Seq is: ATAACAA\n",
      "4 ATAACAA motifs detected in file\n",
      "\n",
      "Complement found from 4925 to 4932\n",
      "Complement found from 5729 to 5736\n",
      "Complement found from 6343 to 6350\n",
      "Complement found from 6485 to 6492\n",
      "Complement found from 12587 to 12594\n",
      "Complement found from 13244 to 13251\n",
      "Seq is: CTTCTCC\n",
      "6 CTTCTCC motifs detected in file\n",
      "\n",
      "Seq is: TATCGAT\n",
      "0 TATCGAT motifs detected in file\n",
      "\n",
      "Complement found from 4025 to 4032\n",
      "Complement found from 6097 to 6104\n",
      "Complement found from 7932 to 7939\n",
      "Complement found from 13103 to 13110\n",
      "Seq is: AATCTTC\n",
      "4 AATCTTC motifs detected in file\n",
      "\n",
      "Complement found from 7736 to 7743\n",
      "Complement found from 10954 to 10961\n",
      "Complement found from 15838 to 15845\n",
      "Complement found from 15903 to 15910\n",
      "Seq is: CTAATAC\n",
      "4 CTAATAC motifs detected in file\n",
      "\n",
      "Complement found from 3683 to 3690\n",
      "Complement found from 5348 to 5355\n",
      "Complement found from 7126 to 7133\n",
      "Seq is: CTACGCC\n",
      "3 CTACGCC motifs detected in file\n",
      "\n",
      "Seq is: AACGAAC\n",
      "0 AACGAAC motifs detected in file\n",
      "\n",
      "Seq is: GCACTCC\n",
      "0 GCACTCC motifs detected in file\n",
      "\n",
      "Seq is: TTTATGA\n",
      "0 TTTATGA motifs detected in file\n",
      "\n",
      "Complement found from 5105 to 5112\n",
      "Complement found from 8380 to 8387\n",
      "Seq is: ACTACCG\n",
      "2 ACTACCG motifs detected in file\n",
      "\n",
      "Complement found from 5268 to 5275\n",
      "Complement found from 7416 to 7423\n",
      "Seq is: TCGAAGA\n",
      "2 TCGAAGA motifs detected in file\n",
      "\n",
      "Complement found from 11142 to 11149\n",
      "Seq is: CACCTTG\n",
      "1 CACCTTG motifs detected in file\n",
      "\n",
      "Seq is: CTCCCAA\n",
      "0 CTCCCAA motifs detected in file\n",
      "\n",
      "Complement found from 8488 to 8495\n",
      "Complement found from 16159 to 16166\n",
      "Seq is: ATAAAAA\n",
      "2 ATAAAAA motifs detected in file\n",
      "\n",
      "Complement found from 4726 to 4733\n",
      "Complement found from 8980 to 8987\n",
      "Seq is: AACCAAT\n",
      "2 AACCAAT motifs detected in file\n",
      "\n",
      "Complement found from 1656 to 1663\n",
      "Complement found from 2905 to 2912\n",
      "Complement found from 16141 to 16148\n",
      "Complement found from 16394 to 16401\n",
      "Seq is: CTTGACC\n",
      "4 CTTGACC motifs detected in file\n",
      "\n",
      "Complement found from 448 to 455\n",
      "Seq is: TTATTTT\n",
      "1 TTATTTT motifs detected in file\n",
      "\n",
      "Seq is: CGATCGT\n",
      "0 CGATCGT motifs detected in file\n",
      "\n",
      "Complement found from 644 to 651\n",
      "Complement found from 12932 to 12939\n",
      "Seq is: ACAAATA\n",
      "2 ACAAATA motifs detected in file\n",
      "\n",
      "Seq is: AAGATCC\n",
      "0 AAGATCC motifs detected in file\n",
      "\n",
      "Seq is: GGATGAG\n",
      "0 GGATGAG motifs detected in file\n",
      "\n",
      "Complement found from 7137 to 7144\n",
      "Seq is: TCCATTT\n",
      "1 TCCATTT motifs detected in file\n",
      "\n",
      "Seq is: TTTAGTC\n",
      "0 TTTAGTC motifs detected in file\n",
      "\n",
      "Seq is: ACGCTCG\n",
      "0 ACGCTCG motifs detected in file\n",
      "\n",
      "Seq is: AATCGCG\n",
      "0 AATCGCG motifs detected in file\n",
      "\n",
      "Complement found from 1139 to 1146\n",
      "Complement found from 5375 to 5382\n",
      "Complement found from 12346 to 12353\n",
      "Seq is: ACACTAC\n",
      "3 ACACTAC motifs detected in file\n",
      "\n",
      "Complement found from 902 to 909\n",
      "Complement found from 12047 to 12054\n",
      "Complement found from 15269 to 15276\n",
      "Seq is: TCACACG\n",
      "3 TCACACG motifs detected in file\n",
      "\n",
      "Seq is: GACGTTT\n",
      "0 GACGTTT motifs detected in file\n",
      "\n",
      "Complement found from 3502 to 3509\n",
      "Complement found from 4500 to 4507\n",
      "Complement found from 6825 to 6832\n",
      "Complement found from 10280 to 10287\n",
      "Seq is: CTACCAT\n",
      "4 CTACCAT motifs detected in file\n",
      "\n",
      "Complement found from 12699 to 12706\n",
      "Seq is: CTCATCT\n",
      "1 CTCATCT motifs detected in file\n",
      "\n",
      "Complement found from 11302 to 11309\n",
      "Seq is: ACTGCCC\n",
      "1 ACTGCCC motifs detected in file\n",
      "\n",
      "Seq is: AACCCGG\n",
      "0 AACCCGG motifs detected in file\n",
      "\n",
      "Complement found from 6151 to 6158\n",
      "Complement found from 7093 to 7100\n",
      "Complement found from 7624 to 7631\n",
      "Complement found from 8226 to 8233\n",
      "Complement found from 9580 to 9587\n",
      "Complement found from 11232 to 11239\n",
      "Complement found from 13945 to 13952\n",
      "Seq is: TCCCCTA\n",
      "7 TCCCCTA motifs detected in file\n",
      "\n",
      "Complement found from 4469 to 4476\n",
      "Complement found from 10336 to 10343\n",
      "Complement found from 12675 to 12682\n",
      "Seq is: ATTAATC\n",
      "3 ATTAATC motifs detected in file\n",
      "\n",
      "Complement found from 7703 to 7710\n",
      "Complement found from 8893 to 8900\n",
      "Seq is: ATGCCCT\n",
      "2 ATGCCCT motifs detected in file\n",
      "\n",
      "Complement found from 465 to 472\n",
      "Complement found from 6665 to 6672\n",
      "Complement found from 15203 to 15210\n",
      "Seq is: TCCCATA\n",
      "3 TCCCATA motifs detected in file\n",
      "\n",
      "Seq is: TTGGTTG\n",
      "0 TTGGTTG motifs detected in file\n",
      "\n",
      "Complement found from 2176 to 2183\n",
      "Seq is: TAAAAGC\n",
      "1 TAAAAGC motifs detected in file\n",
      "\n",
      "Complement found from 466 to 473\n",
      "Complement found from 534 to 541\n",
      "Complement found from 3573 to 3580\n",
      "Complement found from 4429 to 4436\n",
      "Complement found from 8407 to 8414\n",
      "Complement found from 8940 to 8947\n",
      "Complement found from 15204 to 15211\n",
      "Seq is: CCCATAC\n",
      "7 CCCATAC motifs detected in file\n",
      "\n",
      "Seq is: CCGGGCT\n",
      "0 CCGGGCT motifs detected in file\n",
      "\n",
      "Complement found from 3942 to 3949\n",
      "Seq is: ATCGAAT\n",
      "1 ATCGAAT motifs detected in file\n",
      "\n",
      "Seq is: AAATCGA\n",
      "0 AAATCGA motifs detected in file\n",
      "\n",
      "Seq is: CTGGAAT\n",
      "0 CTGGAAT motifs detected in file\n",
      "\n",
      "Seq is: GAAATCC\n",
      "0 GAAATCC motifs detected in file\n",
      "\n",
      "Complement found from 1080 to 1087\n",
      "Complement found from 1372 to 1379\n",
      "Seq is: TACCCCA\n",
      "2 TACCCCA motifs detected in file\n",
      "\n",
      "Complement found from 7144 to 7151\n",
      "Complement found from 9850 to 9857\n",
      "Complement found from 11021 to 11028\n",
      "Seq is: CACTATC\n",
      "3 CACTATC motifs detected in file\n",
      "\n",
      "Seq is: CACCGTG\n",
      "0 CACCGTG motifs detected in file\n",
      "\n",
      "Complement found from 10330 to 10337\n",
      "Complement found from 10627 to 10634\n",
      "Seq is: CCTCTTA\n",
      "2 CCTCTTA motifs detected in file\n",
      "\n",
      "Complement found from 1673 to 1680\n",
      "Complement found from 1992 to 1999\n",
      "Complement found from 4255 to 4262\n",
      "Complement found from 7122 to 7129\n",
      "Complement found from 8825 to 8832\n",
      "Complement found from 10747 to 10754\n",
      "Complement found from 16282 to 16289\n",
      "Seq is: AAACCTA\n",
      "7 AAACCTA motifs detected in file\n",
      "\n",
      "Complement found from 2955 to 2962\n",
      "Seq is: AGAGTCC\n",
      "1 AGAGTCC motifs detected in file\n",
      "\n",
      "Complement found from 505 to 512\n",
      "Complement found from 16286 to 16293\n",
      "Seq is: CTACCCA\n",
      "2 CTACCCA motifs detected in file\n",
      "\n",
      "Complement found from 1055 to 1062\n",
      "Complement found from 1892 to 1899\n",
      "Complement found from 5583 to 5590\n",
      "Complement found from 5643 to 5650\n",
      "Complement found from 5696 to 5703\n",
      "Complement found from 15993 to 16000\n",
      "Seq is: AGCTAAG\n",
      "6 AGCTAAG motifs detected in file\n",
      "\n",
      "Complement found from 9628 to 9635\n",
      "Seq is: AGTATCA\n",
      "1 AGTATCA motifs detected in file\n",
      "\n",
      "Seq is: GGATCTT\n",
      "0 GGATCTT motifs detected in file\n",
      "\n",
      "Complement found from 14150 to 14157\n",
      "Seq is: TATTCCC\n",
      "1 TATTCCC motifs detected in file\n",
      "\n",
      "Seq is: CCAAATT\n",
      "0 CCAAATT motifs detected in file\n",
      "\n",
      "Seq is: CGAGGAT\n",
      "0 CGAGGAT motifs detected in file\n",
      "\n",
      "Complement found from 14495 to 14502\n",
      "Seq is: AATAAAT\n",
      "1 AATAAAT motifs detected in file\n",
      "\n",
      "Seq is: GAGATAG\n",
      "0 GAGATAG motifs detected in file\n",
      "\n",
      "Seq is: TTTCATT\n",
      "0 TTTCATT motifs detected in file\n",
      "\n",
      "Complement found from 12053 to 12060\n",
      "Complement found from 15958 to 15965\n",
      "Seq is: GAGAAAA\n",
      "2 GAGAAAA motifs detected in file\n",
      "\n",
      "Seq is: TAGTCTG\n",
      "0 TAGTCTG motifs detected in file\n",
      "\n",
      "Seq is: TATAAAG\n",
      "0 TATAAAG motifs detected in file\n",
      "\n",
      "Complement found from 4360 to 4367\n",
      "Complement found from 9149 to 9156\n",
      "Complement found from 12954 to 12961\n",
      "Complement found from 13151 to 13158\n",
      "Seq is: AATCCAA\n",
      "4 AATCCAA motifs detected in file\n",
      "\n",
      "Complement found from 5380 to 5387\n",
      "Complement found from 13031 to 13038\n",
      "Seq is: ACTCCCC\n",
      "2 ACTCCCC motifs detected in file\n",
      "\n",
      "Complement found from 10164 to 10171\n",
      "Complement found from 12186 to 12193\n",
      "Complement found from 13962 to 13969\n",
      "Seq is: CTTACGA\n",
      "3 CTTACGA motifs detected in file\n",
      "\n",
      "Complement found from 724 to 731\n",
      "Complement found from 2066 to 2073\n",
      "Seq is: CCTCTAA\n",
      "2 CCTCTAA motifs detected in file\n",
      "\n",
      "Complement found from 12117 to 12124\n",
      "Seq is: CATTACC\n",
      "1 CATTACC motifs detected in file\n",
      "\n",
      "Complement found from 13430 to 13437\n",
      "Seq is: CATACCT\n",
      "1 CATACCT motifs detected in file\n",
      "\n",
      "Seq is: CTGTATA\n",
      "0 CTGTATA motifs detected in file\n",
      "\n",
      "Seq is: GTATATT\n",
      "0 GTATATT motifs detected in file\n",
      "\n",
      "Seq is: CATTACG\n",
      "0 CATTACG motifs detected in file\n",
      "\n",
      "Seq is: ATCCCAC\n",
      "0 ATCCCAC motifs detected in file\n",
      "\n",
      "Complement found from 3541 to 3548\n",
      "Complement found from 9646 to 9653\n",
      "Complement found from 9762 to 9769\n",
      "Complement found from 13453 to 13460\n",
      "Seq is: TCACCAT\n",
      "4 TCACCAT motifs detected in file\n",
      "\n",
      "Complement found from 5540 to 5547\n",
      "Seq is: CCTTCAA\n",
      "1 CCTTCAA motifs detected in file\n",
      "\n",
      "Complement found from 2459 to 2466\n",
      "Complement found from 12329 to 12336\n",
      "Seq is: AAAAGTA\n",
      "2 AAAAGTA motifs detected in file\n",
      "\n",
      "Complement found from 14870 to 14877\n",
      "Complement found from 15662 to 15669\n",
      "Seq is: TCCTCCA\n",
      "2 TCCTCCA motifs detected in file\n",
      "\n",
      "Complement found from 5007 to 5014\n",
      "Complement found from 8409 to 8416\n",
      "Complement found from 11618 to 11625\n",
      "Complement found from 13907 to 13914\n",
      "Complement found from 14345 to 14352\n",
      "Seq is: CATACTC\n",
      "5 CATACTC motifs detected in file\n",
      "\n",
      "Complement found from 11170 to 11177\n",
      "Seq is: AACCAGC\n",
      "1 AACCAGC motifs detected in file\n",
      "\n",
      "Seq is: ATCGCCT\n",
      "0 ATCGCCT motifs detected in file\n",
      "\n",
      "Complement found from 1678 to 1685\n",
      "Complement found from 4791 to 4798\n",
      "Complement found from 4875 to 4882\n",
      "Complement found from 9522 to 9529\n",
      "Seq is: TAGCCCC\n",
      "4 TAGCCCC motifs detected in file\n",
      "\n",
      "Seq is: CATACGA\n",
      "0 CATACGA motifs detected in file\n",
      "\n",
      "Seq is: CAAGCTT\n",
      "0 CAAGCTT motifs detected in file\n",
      "\n",
      "Complement found from 4640 to 4647\n",
      "Complement found from 6858 to 6865\n",
      "Complement found from 16057 to 16064\n",
      "Seq is: AAGTATT\n",
      "3 AAGTATT motifs detected in file\n",
      "\n",
      "Seq is: CTTGTCC\n",
      "0 CTTGTCC motifs detected in file\n",
      "\n",
      "Complement found from 10754 to 10761\n",
      "Seq is: CTCCAAT\n",
      "1 CTCCAAT motifs detected in file\n",
      "\n",
      "Complement found from 9859 to 9866\n",
      "Complement found from 15291 to 15298\n",
      "Seq is: CTTCATC\n",
      "2 CTTCATC motifs detected in file\n",
      "\n",
      "Complement found from 1134 to 1141\n",
      "Complement found from 11176 to 11183\n",
      "Seq is: CCAGAAC\n",
      "2 CCAGAAC motifs detected in file\n",
      "\n",
      "Seq is: CACTGTT\n",
      "0 CACTGTT motifs detected in file\n",
      "\n",
      "Seq is: TTATACA\n",
      "0 TTATACA motifs detected in file\n",
      "\n",
      "Complement found from 336 to 343\n",
      "Complement found from 9603 to 9610\n",
      "Complement found from 15033 to 15040\n",
      "Complement found from 16232 to 16239\n",
      "Seq is: ACACATC\n",
      "4 ACACATC motifs detected in file\n",
      "\n",
      "Complement found from 13254 to 13261\n",
      "Seq is: TCAAGTC\n",
      "1 TCAAGTC motifs detected in file\n",
      "\n",
      "Complement found from 6221 to 6228\n",
      "Complement found from 9267 to 9274\n",
      "Complement found from 14281 to 14288\n",
      "Seq is: CCCTCTC\n",
      "3 CCCTCTC motifs detected in file\n",
      "\n",
      "Seq is: ACGCAGT\n",
      "0 ACGCAGT motifs detected in file\n",
      "\n",
      "Complement found from 487 to 494\n",
      "Complement found from 3396 to 3403\n",
      "Complement found from 12854 to 12861\n",
      "Seq is: ATACAAC\n",
      "3 ATACAAC motifs detected in file\n",
      "\n",
      "Seq is: AAGGATC\n",
      "0 AAGGATC motifs detected in file\n",
      "\n",
      "Complement found from 4098 to 4105\n",
      "Complement found from 4676 to 4683\n",
      "Complement found from 12942 to 12949\n",
      "Complement found from 13827 to 13834\n",
      "Seq is: CTTCTAA\n",
      "4 CTTCTAA motifs detected in file\n",
      "\n",
      "Complement found from 631 to 638\n",
      "Complement found from 2507 to 2514\n",
      "Complement found from 2519 to 2526\n",
      "Complement found from 3506 to 3513\n",
      "Complement found from 3518 to 3525\n",
      "Complement found from 5320 to 5327\n",
      "Complement found from 8430 to 8437\n",
      "Complement found from 9565 to 9572\n",
      "Complement found from 11154 to 11161\n",
      "Complement found from 14060 to 14067\n",
      "Seq is: CATCACC\n",
      "10 CATCACC motifs detected in file\n",
      "\n",
      "Complement found from 463 to 470\n",
      "Complement found from 6309 to 6316\n",
      "Complement found from 11811 to 11818\n",
      "Seq is: ACTCCCA\n",
      "3 ACTCCCA motifs detected in file\n",
      "\n",
      "Complement found from 1733 to 1740\n",
      "Complement found from 9677 to 9684\n",
      "Complement found from 12324 to 12331\n",
      "Seq is: CAAATAA\n",
      "3 CAAATAA motifs detected in file\n",
      "\n",
      "Complement found from 151 to 158\n",
      "Complement found from 5961 to 5968\n",
      "Complement found from 10245 to 10252\n",
      "Seq is: TATTATT\n",
      "3 TATTATT motifs detected in file\n",
      "\n",
      "Complement found from 3990 to 3997\n",
      "Complement found from 5679 to 5686\n",
      "Complement found from 10294 to 10301\n",
      "Complement found from 11592 to 11599\n",
      "Complement found from 14188 to 14195\n",
      "Seq is: ACAAACA\n",
      "5 ACAAACA motifs detected in file\n",
      "\n",
      "Complement found from 2333 to 2340\n",
      "Complement found from 6199 to 6206\n",
      "Seq is: CATAAGC\n",
      "2 CATAAGC motifs detected in file\n",
      "\n",
      "Seq is: CGATACT\n",
      "0 CGATACT motifs detected in file\n",
      "\n",
      "Complement found from 4944 to 4951\n",
      "Complement found from 6637 to 6644\n",
      "Seq is: TCTTATC\n",
      "2 TCTTATC motifs detected in file\n",
      "\n",
      "Complement found from 313 to 320\n",
      "Complement found from 13118 to 13125\n",
      "Seq is: CCGCTTC\n",
      "2 CCGCTTC motifs detected in file\n",
      "\n",
      "Complement found from 8110 to 8117\n",
      "Seq is: CCCGGAC\n",
      "1 CCCGGAC motifs detected in file\n",
      "\n",
      "Seq is: GCCGCAT\n",
      "0 GCCGCAT motifs detected in file\n",
      "\n",
      "Complement found from 15774 to 15781\n",
      "Seq is: AAGCTAC\n",
      "1 AAGCTAC motifs detected in file\n",
      "\n",
      "Complement found from 10061 to 10068\n",
      "Complement found from 14839 to 14846\n",
      "Seq is: AACTTCG\n",
      "2 AACTTCG motifs detected in file\n",
      "\n",
      "Seq is: GACTCTG\n",
      "0 GACTCTG motifs detected in file\n",
      "\n",
      "Complement found from 756 to 763\n",
      "Complement found from 15547 to 15554\n",
      "Seq is: ATCAAGC\n",
      "2 ATCAAGC motifs detected in file\n",
      "\n",
      "Complement found from 15165 to 15172\n",
      "Seq is: CTGAGGG\n",
      "1 CTGAGGG motifs detected in file\n",
      "\n",
      "Complement found from 926 to 933\n",
      "Complement found from 9911 to 9918\n",
      "Seq is: GAAGCCG\n",
      "2 GAAGCCG motifs detected in file\n",
      "\n",
      "Seq is: TTCCAGC\n",
      "0 TTCCAGC motifs detected in file\n",
      "\n",
      "Complement found from 3165 to 3172\n",
      "Complement found from 4246 to 4253\n",
      "Complement found from 7952 to 7959\n",
      "Complement found from 8602 to 8609\n",
      "Complement found from 12382 to 12389\n",
      "Complement found from 13751 to 13758\n",
      "Complement found from 14152 to 14159\n",
      "Complement found from 14486 to 14493\n",
      "Seq is: TTCCCCC\n",
      "8 TTCCCCC motifs detected in file\n",
      "\n",
      "Complement found from 7483 to 7490\n",
      "Complement found from 10995 to 11002\n",
      "Complement found from 15705 to 15712\n",
      "Seq is: AAGCCAA\n",
      "3 AAGCCAA motifs detected in file\n",
      "\n",
      "Seq is: CCAGAGA\n",
      "0 CCAGAGA motifs detected in file\n",
      "\n",
      "Seq is: CGATCAC\n",
      "0 CGATCAC motifs detected in file\n",
      "\n",
      "Complement found from 2511 to 2518\n",
      "Complement found from 3627 to 3634\n",
      "Complement found from 5334 to 5341\n",
      "Seq is: ACCTCTA\n",
      "3 ACCTCTA motifs detected in file\n",
      "\n",
      "Complement found from 9995 to 10002\n",
      "Seq is: TTTAGTA\n",
      "1 TTTAGTA motifs detected in file\n",
      "\n",
      "Complement found from 6111 to 6118\n",
      "Complement found from 15839 to 15846\n",
      "Seq is: TAATACC\n",
      "2 TAATACC motifs detected in file\n",
      "\n",
      "Seq is: GGTTCCC\n",
      "0 GGTTCCC motifs detected in file\n",
      "\n",
      "Seq is: AGTACTA\n",
      "0 AGTACTA motifs detected in file\n",
      "\n",
      "Complement found from 6462 to 6469\n",
      "Seq is: CCGTCCT\n",
      "1 CCGTCCT motifs detected in file\n",
      "\n",
      "Seq is: CATTAGT\n",
      "0 CATTAGT motifs detected in file\n",
      "\n",
      "Complement found from 3866 to 3873\n",
      "Seq is: CTCCACA\n",
      "1 CTCCACA motifs detected in file\n",
      "\n",
      "Complement found from 1136 to 1143\n",
      "Complement found from 3812 to 3819\n",
      "Complement found from 14737 to 14744\n",
      "Seq is: AGAACAC\n",
      "3 AGAACAC motifs detected in file\n",
      "\n",
      "Complement found from 1138 to 1145\n",
      "Complement found from 5950 to 5957\n",
      "Complement found from 8709 to 8716\n",
      "Complement found from 13163 to 13170\n",
      "Complement found from 15471 to 15478\n",
      "Seq is: AACACTA\n",
      "5 AACACTA motifs detected in file\n",
      "\n",
      "Complement found from 7178 to 7185\n",
      "Complement found from 14112 to 14119\n",
      "Seq is: TTCCCAC\n",
      "2 TTCCCAC motifs detected in file\n",
      "\n",
      "Complement found from 38 to 45\n",
      "Complement found from 3789 to 3796\n",
      "Seq is: CTCTCCA\n",
      "2 CTCTCCA motifs detected in file\n",
      "\n",
      "Complement found from 4644 to 4651\n",
      "Complement found from 15563 to 15570\n",
      "Seq is: ATTTCCT\n",
      "2 ATTTCCT motifs detected in file\n",
      "\n",
      "Seq is: CGGTGAA\n",
      "0 CGGTGAA motifs detected in file\n",
      "\n",
      "Complement found from 4758 to 4765\n",
      "Complement found from 6831 to 6838\n",
      "Complement found from 7670 to 7677\n",
      "Complement found from 8749 to 8756\n",
      "Complement found from 10338 to 10345\n",
      "Complement found from 10453 to 10460\n",
      "Complement found from 11102 to 11109\n",
      "Complement found from 14230 to 14237\n",
      "Seq is: TAATCAT\n",
      "8 TAATCAT motifs detected in file\n",
      "\n",
      "Complement found from 3397 to 3404\n",
      "Seq is: TACAACT\n",
      "1 TACAACT motifs detected in file\n",
      "\n",
      "Complement found from 3181 to 3188\n",
      "Complement found from 7261 to 7268\n",
      "Complement found from 9096 to 9103\n",
      "Seq is: ATCATCT\n",
      "3 ATCATCT motifs detected in file\n",
      "\n",
      "Complement found from 1700 to 1707\n",
      "Complement found from 4732 to 4739\n",
      "Complement found from 5104 to 5111\n",
      "Complement found from 8379 to 8386\n",
      "Complement found from 10790 to 10797\n",
      "Seq is: TACTACC\n",
      "5 TACTACC motifs detected in file\n",
      "\n",
      "Seq is: GATCCCA\n",
      "0 GATCCCA motifs detected in file\n",
      "\n",
      "Complement found from 6624 to 6631\n",
      "Seq is: CTGAAGT\n",
      "1 CTGAAGT motifs detected in file\n",
      "\n",
      "Complement found from 4703 to 4710\n",
      "Complement found from 11619 to 11626\n",
      "Complement found from 14346 to 14353\n",
      "Seq is: ATACTCT\n",
      "3 ATACTCT motifs detected in file\n",
      "\n",
      "Complement found from 2763 to 2770\n",
      "Complement found from 8506 to 8513\n",
      "Complement found from 13860 to 13867\n",
      "Complement found from 14187 to 14194\n",
      "Complement found from 15601 to 15608\n",
      "Complement found from 16279 to 16286\n",
      "Seq is: AACAAAC\n",
      "6 AACAAAC motifs detected in file\n",
      "\n",
      "Complement found from 11160 to 11167\n",
      "Seq is: CCGATGA\n",
      "1 CCGATGA motifs detected in file\n",
      "\n",
      "Seq is: CGAGCGT\n",
      "0 CGAGCGT motifs detected in file\n",
      "\n",
      "Seq is: CACGCGG\n",
      "0 CACGCGG motifs detected in file\n",
      "\n",
      "Complement found from 7693 to 7700\n",
      "Complement found from 7801 to 7808\n",
      "Seq is: CTAGTCC\n",
      "2 CTAGTCC motifs detected in file\n",
      "\n",
      "Complement found from 9619 to 9626\n",
      "Complement found from 11776 to 11783\n",
      "Seq is: CGCATCA\n",
      "2 CGCATCA motifs detected in file\n",
      "\n",
      "Complement found from 1160 to 1167\n",
      "Complement found from 3675 to 3682\n",
      "Complement found from 10134 to 10141\n",
      "Complement found from 11749 to 11756\n",
      "Seq is: AACTCAA\n",
      "4 AACTCAA motifs detected in file\n",
      "\n",
      "Complement found from 1018 to 1025\n",
      "Complement found from 1141 to 1148\n",
      "Complement found from 1383 to 1390\n",
      "Complement found from 11756 to 11763\n",
      "Seq is: ACTACGA\n",
      "4 ACTACGA motifs detected in file\n",
      "\n",
      "Seq is: GTGGGAC\n",
      "0 GTGGGAC motifs detected in file\n",
      "\n",
      "Complement found from 3925 to 3932\n",
      "Complement found from 10701 to 10708\n",
      "Seq is: TAGTCTC\n",
      "2 TAGTCTC motifs detected in file\n",
      "\n",
      "Complement found from 12226 to 12233\n",
      "Seq is: CTAACTC\n",
      "1 CTAACTC motifs detected in file\n",
      "\n",
      "Complement found from 2051 to 2058\n",
      "Complement found from 7060 to 7067\n",
      "Seq is: ATTTGCC\n",
      "2 ATTTGCC motifs detected in file\n",
      "\n",
      "Seq is: GATCCGA\n",
      "0 GATCCGA motifs detected in file\n",
      "\n",
      "Complement found from 3790 to 3797\n",
      "Complement found from 3865 to 3872\n",
      "Complement found from 13019 to 13026\n",
      "Complement found from 13246 to 13253\n",
      "Complement found from 14047 to 14054\n",
      "Seq is: TCTCCAC\n",
      "5 TCTCCAC motifs detected in file\n",
      "\n",
      "Seq is: CGATCTT\n",
      "0 CGATCTT motifs detected in file\n",
      "\n",
      "Complement found from 155 to 162\n",
      "Complement found from 3860 to 3867\n",
      "Complement found from 12899 to 12906\n",
      "Seq is: ATTTATC\n",
      "3 ATTTATC motifs detected in file\n",
      "\n",
      "Complement found from 4708 to 4715\n",
      "Complement found from 6685 to 6692\n",
      "Seq is: CTCCGGA\n",
      "2 CTCCGGA motifs detected in file\n",
      "\n",
      "Complement found from 12748 to 12755\n",
      "Complement found from 12912 to 12919\n",
      "Seq is: TCCAACT\n",
      "2 TCCAACT motifs detected in file\n",
      "\n",
      "Complement found from 12749 to 12756\n",
      "Seq is: CCAACTG\n",
      "1 CCAACTG motifs detected in file\n",
      "\n",
      "Seq is: GTCCCCC\n",
      "0 GTCCCCC motifs detected in file\n",
      "\n",
      "Complement found from 4346 to 4353\n",
      "Complement found from 14516 to 14523\n",
      "Seq is: AACCCAT\n",
      "2 AACCCAT motifs detected in file\n",
      "\n",
      "Complement found from 4832 to 4839\n",
      "Complement found from 10977 to 10984\n",
      "Complement found from 14279 to 14286\n",
      "Complement found from 15535 to 15542\n",
      "Complement found from 16257 to 16264\n",
      "Seq is: ACCCCTC\n",
      "5 ACCCCTC motifs detected in file\n",
      "\n",
      "Complement found from 433 to 440\n",
      "Complement found from 7466 to 7473\n",
      "Complement found from 9532 to 9539\n",
      "Complement found from 9554 to 9561\n",
      "Complement found from 14530 to 14537\n",
      "Seq is: CCCCCAA\n",
      "5 CCCCCAA motifs detected in file\n",
      "\n",
      "Seq is: GTATCAT\n",
      "0 GTATCAT motifs detected in file\n",
      "\n",
      "Complement found from 16354 to 16361\n",
      "Seq is: CTTCTCG\n",
      "1 CTTCTCG motifs detected in file\n",
      "\n",
      "Complement found from 4185 to 4192\n",
      "Complement found from 7405 to 7412\n",
      "Complement found from 8460 to 8467\n",
      "Complement found from 10127 to 10134\n",
      "Complement found from 10792 to 10799\n",
      "Seq is: CTACCAC\n",
      "5 CTACCAC motifs detected in file\n",
      "\n",
      "Seq is: TCTCGAT\n",
      "0 TCTCGAT motifs detected in file\n",
      "\n",
      "Complement found from 9515 to 9522\n",
      "Seq is: TCCAGCC\n",
      "1 TCCAGCC motifs detected in file\n",
      "\n",
      "Complement found from 3493 to 3500\n",
      "Seq is: CCGCCAC\n",
      "1 CCGCCAC motifs detected in file\n",
      "\n",
      "Seq is: TACATCT\n",
      "0 TACATCT motifs detected in file\n",
      "\n",
      "Complement found from 1083 to 1090\n",
      "Complement found from 2216 to 2223\n",
      "Complement found from 11814 to 11821\n",
      "Complement found from 11869 to 11876\n",
      "Complement found from 12968 to 12975\n",
      "Complement found from 13145 to 13152\n",
      "Complement found from 13880 to 13887\n",
      "Complement found from 14389 to 14396\n",
      "Complement found from 15699 to 15706\n",
      "Complement found from 16265 to 16272\n",
      "Seq is: CCCACTA\n",
      "10 CCCACTA motifs detected in file\n",
      "\n",
      "Complement found from 9969 to 9976\n",
      "Seq is: CCATCTA\n",
      "1 CCATCTA motifs detected in file\n",
      "\n",
      "Complement found from 1480 to 1487\n",
      "Complement found from 2537 to 2544\n",
      "Complement found from 3521 to 3528\n",
      "Seq is: CACCGCC\n",
      "3 CACCGCC motifs detected in file\n",
      "\n",
      "Complement found from 4996 to 5003\n",
      "Complement found from 7132 to 7139\n",
      "Seq is: CAAAATC\n",
      "2 CAAAATC motifs detected in file\n",
      "\n",
      "Seq is: CCCGAGA\n",
      "0 CCCGAGA motifs detected in file\n",
      "\n",
      "Complement found from 23 to 30\n",
      "Complement found from 8127 to 8134\n",
      "Complement found from 11018 to 11025\n",
      "Complement found from 14788 to 14795\n",
      "Seq is: AACCACT\n",
      "4 AACCACT motifs detected in file\n",
      "\n",
      "Seq is: TCTCAAA\n",
      "0 TCTCAAA motifs detected in file\n",
      "\n",
      "Complement found from 1257 to 1264\n",
      "Seq is: ATACCGC\n",
      "1 ATACCGC motifs detected in file\n",
      "\n",
      "Seq is: AGTCGCT\n",
      "0 AGTCGCT motifs detected in file\n",
      "\n",
      "Complement found from 10775 to 10782\n",
      "Seq is: TCCCAAC\n",
      "1 TCCCAAC motifs detected in file\n",
      "\n",
      "Complement found from 6833 to 6840\n",
      "Complement found from 13518 to 13525\n",
      "Seq is: ATCATCG\n",
      "2 ATCATCG motifs detected in file\n",
      "\n",
      "Seq is: GGCATCC\n",
      "0 GGCATCC motifs detected in file\n",
      "\n",
      "Seq is: CCGGATG\n",
      "0 CCGGATG motifs detected in file\n",
      "\n",
      "Complement found from 15107 to 15114\n",
      "Seq is: TTGCAAC\n",
      "1 TTGCAAC motifs detected in file\n",
      "\n",
      "Seq is: CCCGGAA\n",
      "0 CCCGGAA motifs detected in file\n",
      "\n",
      "Complement found from 7827 to 7834\n",
      "Seq is: ACGCATC\n",
      "1 ACGCATC motifs detected in file\n",
      "\n",
      "Complement found from 1035 to 1042\n",
      "Complement found from 2237 to 2244\n",
      "Complement found from 4040 to 4047\n",
      "Complement found from 4073 to 4080\n",
      "Complement found from 13533 to 13540\n",
      "Seq is: AACATAT\n",
      "5 AACATAT motifs detected in file\n",
      "\n",
      "Seq is: ATCGGAT\n",
      "0 ATCGGAT motifs detected in file\n",
      "\n",
      "Complement found from 1155 to 1162\n",
      "Complement found from 3262 to 3269\n",
      "Complement found from 8091 to 8098\n",
      "Complement found from 11463 to 11470\n",
      "Complement found from 13866 to 13873\n",
      "Seq is: CTTAAAA\n",
      "5 CTTAAAA motifs detected in file\n",
      "\n",
      "Complement found from 11841 to 11848\n",
      "Seq is: AGCAAGC\n",
      "1 AGCAAGC motifs detected in file\n",
      "\n",
      "Complement found from 8031 to 8038\n",
      "Seq is: CATTCGT\n",
      "1 CATTCGT motifs detected in file\n",
      "\n",
      "Complement found from 4672 to 4679\n",
      "Seq is: AATCCTT\n",
      "1 AATCCTT motifs detected in file\n",
      "\n",
      "Seq is: ACGGTAA\n",
      "0 ACGGTAA motifs detected in file\n",
      "\n",
      "Complement found from 12950 to 12957\n",
      "Seq is: CGCTAAT\n",
      "1 CGCTAAT motifs detected in file\n",
      "\n",
      "Complement found from 2483 to 2490\n",
      "Complement found from 11862 to 11869\n",
      "Seq is: CTTACCC\n",
      "2 CTTACCC motifs detected in file\n",
      "\n",
      "Seq is: ATGTTAC\n",
      "0 ATGTTAC motifs detected in file\n",
      "\n",
      "Complement found from 3328 to 3335\n",
      "Complement found from 7805 to 7812\n",
      "Complement found from 8426 to 8433\n",
      "Complement found from 9330 to 9337\n",
      "Complement found from 11725 to 11732\n",
      "Complement found from 15638 to 15645\n",
      "Complement found from 15740 to 15747\n",
      "Seq is: TCCTCAT\n",
      "7 TCCTCAT motifs detected in file\n",
      "\n",
      "Complement found from 8623 to 8630\n",
      "Complement found from 14871 to 14878\n",
      "Seq is: CCTCCAA\n",
      "2 CCTCCAA motifs detected in file\n",
      "\n",
      "Seq is: CCGGTAC\n",
      "0 CCGGTAC motifs detected in file\n",
      "\n",
      "Complement found from 8701 to 8708\n",
      "Complement found from 15205 to 15212\n",
      "Seq is: CCATACA\n",
      "2 CCATACA motifs detected in file\n",
      "\n",
      "Complement found from 9105 to 9112\n",
      "Complement found from 15580 to 15587\n",
      "Seq is: ACAATTC\n",
      "2 ACAATTC motifs detected in file\n",
      "\n",
      "Complement found from 2323 to 2330\n",
      "Complement found from 4926 to 4933\n",
      "Complement found from 6344 to 6351\n",
      "Complement found from 6486 to 6493\n",
      "Complement found from 11914 to 11921\n",
      "Complement found from 12089 to 12096\n",
      "Seq is: TTCTCCT\n",
      "6 TTCTCCT motifs detected in file\n",
      "\n",
      "Complement found from 1653 to 1660\n",
      "Complement found from 2902 to 2909\n",
      "Seq is: TAACTTG\n",
      "2 TAACTTG motifs detected in file\n",
      "\n",
      "Seq is: GAGACTG\n",
      "0 GAGACTG motifs detected in file\n",
      "\n",
      "Complement found from 13747 to 13754\n",
      "Seq is: ACATTTC\n",
      "1 ACATTTC motifs detected in file\n",
      "\n",
      "Complement found from 3381 to 3388\n",
      "Complement found from 4365 to 4372\n",
      "Complement found from 10216 to 10223\n",
      "Complement found from 14535 to 14542\n",
      "Seq is: AAAATTC\n",
      "4 AAAATTC motifs detected in file\n",
      "\n",
      "Complement found from 15868 to 15875\n",
      "Seq is: AAAATAC\n",
      "1 AAAATAC motifs detected in file\n",
      "\n",
      "Seq is: GCTAATG\n",
      "0 GCTAATG motifs detected in file\n",
      "\n",
      "Complement found from 12813 to 12820\n",
      "Seq is: GCCCGAG\n",
      "1 GCCCGAG motifs detected in file\n",
      "\n",
      "Seq is: ACGGTAG\n",
      "0 ACGGTAG motifs detected in file\n",
      "\n",
      "Seq is: AATTGTT\n",
      "0 AATTGTT motifs detected in file\n",
      "\n",
      "Complement found from 15213 to 15220\n",
      "Seq is: TGGGACA\n",
      "1 TGGGACA motifs detected in file\n",
      "\n",
      "Complement found from 364 to 371\n",
      "Complement found from 2061 to 2068\n",
      "Complement found from 2829 to 2836\n",
      "Complement found from 7360 to 7367\n",
      "Complement found from 7420 to 7427\n",
      "Seq is: AGAACCC\n",
      "5 AGAACCC motifs detected in file\n",
      "\n",
      "Complement found from 2768 to 2775\n",
      "Complement found from 3202 to 3209\n",
      "Complement found from 5021 to 5028\n",
      "Complement found from 5675 to 5682\n",
      "Complement found from 10843 to 10850\n",
      "Complement found from 12925 to 12932\n",
      "Complement found from 14356 to 14363\n",
      "Complement found from 14640 to 14647\n",
      "Seq is: ACCCACA\n",
      "8 ACCCACA motifs detected in file\n",
      "\n",
      "Complement found from 5384 to 5391\n",
      "Complement found from 6666 to 6673\n",
      "Complement found from 14518 to 14525\n",
      "Seq is: CCCATAT\n",
      "3 CCCATAT motifs detected in file\n",
      "\n",
      "Complement found from 3856 to 3863\n",
      "Seq is: TATGATT\n",
      "1 TATGATT motifs detected in file\n",
      "\n",
      "Seq is: CAACTCT\n",
      "0 CAACTCT motifs detected in file\n",
      "\n",
      "Complement found from 13086 to 13093\n",
      "Complement found from 15112 to 15119\n",
      "Seq is: ACTATAG\n",
      "2 ACTATAG motifs detected in file\n",
      "\n",
      "Seq is: TAAATGC\n",
      "0 TAAATGC motifs detected in file\n",
      "\n",
      "Complement found from 3893 to 3900\n",
      "Complement found from 3959 to 3966\n",
      "Complement found from 11535 to 11542\n",
      "Complement found from 13763 to 13770\n",
      "Seq is: CCCCTTC\n",
      "4 CCCCTTC motifs detected in file\n",
      "\n",
      "Seq is: CGCGAAA\n",
      "0 CGCGAAA motifs detected in file\n",
      "\n",
      "Complement found from 13250 to 13257\n",
      "Complement found from 13439 to 13446\n",
      "Complement found from 15289 to 15296\n",
      "Seq is: CACTTCA\n",
      "3 CACTTCA motifs detected in file\n",
      "\n",
      "Complement found from 6686 to 6693\n",
      "Complement found from 7202 to 7209\n",
      "Seq is: TCCGGAA\n",
      "2 TCCGGAA motifs detected in file\n",
      "\n",
      "Complement found from 936 to 943\n",
      "Seq is: TAAAGAG\n",
      "1 TAAAGAG motifs detected in file\n",
      "\n",
      "Seq is: AACAGGA\n",
      "0 AACAGGA motifs detected in file\n",
      "\n",
      "Seq is: AAGCATG\n",
      "0 AAGCATG motifs detected in file\n",
      "\n",
      "Complement found from 7860 to 7867\n",
      "Seq is: TCCCTCC\n",
      "1 TCCCTCC motifs detected in file\n",
      "\n",
      "Complement found from 5129 to 5136\n",
      "Complement found from 14086 to 14093\n",
      "Complement found from 16008 to 16015\n",
      "Seq is: TTAAACT\n",
      "3 TTAAACT motifs detected in file\n",
      "\n",
      "Seq is: TNCATCT\n",
      "0 TNCATCT motifs detected in file\n",
      "\n",
      "Complement found from 4505 to 4512\n",
      "Complement found from 8236 to 8243\n",
      "Seq is: ATCTTTG\n",
      "2 ATCTTTG motifs detected in file\n",
      "\n",
      "Seq is: GCTGGAC\n",
      "0 GCTGGAC motifs detected in file\n",
      "\n",
      "Complement found from 9281 to 9288\n",
      "Seq is: CTAATGA\n",
      "1 CTAATGA motifs detected in file\n",
      "\n",
      "Seq is: GGCCAGA\n",
      "0 GGCCAGA motifs detected in file\n",
      "\n",
      "Seq is: CTTGAGT\n",
      "0 CTTGAGT motifs detected in file\n",
      "\n",
      "Seq is: CTAGTGC\n",
      "0 CTAGTGC motifs detected in file\n",
      "\n",
      "Seq is: ATCCTGA\n",
      "0 ATCCTGA motifs detected in file\n",
      "\n",
      "Seq is: AATTAGC\n",
      "0 AATTAGC motifs detected in file\n",
      "\n",
      "Seq is: AACTTGT\n",
      "0 AACTTGT motifs detected in file\n",
      "\n",
      "Seq is: TTGCTTG\n",
      "0 TTGCTTG motifs detected in file\n",
      "\n",
      "Seq is: GAAATTA\n",
      "0 GAAATTA motifs detected in file\n",
      "\n",
      "Complement found from 5907 to 5914\n",
      "Seq is: TCGCCGA\n",
      "1 TCGCCGA motifs detected in file\n",
      "\n",
      "Seq is: CGTGGTA\n",
      "0 CGTGGTA motifs detected in file\n",
      "\n",
      "Complement found from 10289 to 10296\n",
      "Complement found from 10694 to 10701\n",
      "Complement found from 13068 to 13075\n",
      "Seq is: GCCCTAC\n",
      "3 GCCCTAC motifs detected in file\n",
      "\n",
      "Complement found from 1828 to 1835\n",
      "Complement found from 4330 to 4337\n",
      "Complement found from 13415 to 13422\n",
      "Complement found from 14886 to 14893\n",
      "Seq is: AGGACTA\n",
      "4 AGGACTA motifs detected in file\n",
      "\n",
      "Seq is: GGTTGTA\n",
      "0 GGTTGTA motifs detected in file\n",
      "\n",
      "Seq is: GCTCCAG\n",
      "0 GCTCCAG motifs detected in file\n",
      "\n",
      "Complement found from 47 to 54\n",
      "Complement found from 6702 to 6709\n",
      "Seq is: CATTTGG\n",
      "2 CATTTGG motifs detected in file\n",
      "\n",
      "Complement found from 2933 to 2940\n",
      "Complement found from 12275 to 12282\n",
      "Seq is: GATAACA\n",
      "2 GATAACA motifs detected in file\n",
      "\n",
      "Complement found from 2882 to 2889\n",
      "Complement found from 5953 to 5960\n",
      "Complement found from 6521 to 6528\n",
      "Complement found from 15813 to 15820\n",
      "Seq is: ACTATAC\n",
      "4 ACTATAC motifs detected in file\n",
      "\n",
      "Seq is: CTGAGAT\n",
      "0 CTGAGAT motifs detected in file\n",
      "\n",
      "Complement found from 4945 to 4952\n",
      "Complement found from 6638 to 6645\n",
      "Complement found from 8934 to 8941\n",
      "Complement found from 11007 to 11014\n",
      "Complement found from 11134 to 11141\n",
      "Seq is: CTTATCC\n",
      "5 CTTATCC motifs detected in file\n",
      "\n",
      "Seq is: TAACGCG\n",
      "0 TAACGCG motifs detected in file\n",
      "\n",
      "Complement found from 1050 to 1057\n",
      "Complement found from 2968 to 2975\n",
      "Complement found from 5286 to 5293\n",
      "Complement found from 11356 to 11363\n",
      "Seq is: ACAATAG\n",
      "4 ACAATAG motifs detected in file\n",
      "\n",
      "Seq is: GGATCCC\n",
      "0 GGATCCC motifs detected in file\n",
      "\n",
      "Seq is: ATTGAAC\n",
      "0 ATTGAAC motifs detected in file\n",
      "\n",
      "Seq is: AAGGCAA\n",
      "0 AAGGCAA motifs detected in file\n",
      "\n",
      "Complement found from 4592 to 4599\n",
      "Seq is: CCAGTTC\n",
      "1 CCAGTTC motifs detected in file\n",
      "\n",
      "Complement found from 12335 to 12342\n",
      "Complement found from 13671 to 13678\n",
      "Seq is: AATAACC\n",
      "2 AATAACC motifs detected in file\n",
      "\n",
      "Complement found from 5688 to 5695\n",
      "Seq is: TAGTTAA\n",
      "1 TAGTTAA motifs detected in file\n",
      "\n",
      "Complement found from 3725 to 3732\n",
      "Complement found from 10544 to 10551\n",
      "Seq is: CTCATAT\n",
      "2 CTCATAT motifs detected in file\n",
      "\n",
      "Complement found from 755 to 762\n",
      "Complement found from 4636 to 4643\n",
      "Complement found from 15546 to 15553\n",
      "Seq is: CATCAAG\n",
      "3 CATCAAG motifs detected in file\n",
      "\n",
      "Seq is: CGAAACT\n",
      "0 CGAAACT motifs detected in file\n",
      "\n",
      "Seq is: GACCACT\n",
      "0 GACCACT motifs detected in file\n",
      "\n",
      "Complement found from 14540 to 14547\n",
      "Seq is: TCAGAAT\n",
      "1 TCAGAAT motifs detected in file\n",
      "\n",
      "Complement found from 10735 to 10742\n",
      "Complement found from 16094 to 16101\n",
      "Complement found from 16327 to 16334\n",
      "Seq is: CGTACAT\n",
      "3 CGTACAT motifs detected in file\n",
      "\n",
      "Complement found from 1912 to 1919\n",
      "Seq is: GACGAGC\n",
      "1 GACGAGC motifs detected in file\n",
      "\n",
      "Complement found from 10711 to 10718\n",
      "Complement found from 12911 to 12918\n",
      "Complement found from 13901 to 13908\n",
      "Seq is: CTCCAAC\n",
      "3 CTCCAAC motifs detected in file\n",
      "\n",
      "Complement found from 4087 to 4094\n",
      "Complement found from 14405 to 14412\n",
      "Seq is: CCAAGAC\n",
      "2 CCAAGAC motifs detected in file\n",
      "\n",
      "Seq is: GAGGCTC\n",
      "0 GAGGCTC motifs detected in file\n",
      "\n",
      "Seq is: CAGCGGG\n",
      "0 CAGCGGG motifs detected in file\n",
      "\n",
      "Seq is: GTTGGCT\n",
      "0 GTTGGCT motifs detected in file\n",
      "\n",
      "Complement found from 2454 to 2461\n",
      "Complement found from 2798 to 2805\n",
      "Complement found from 8092 to 8099\n",
      "Complement found from 8887 to 8894\n",
      "Complement found from 14501 to 14508\n",
      "Seq is: TTAAAAA\n",
      "5 TTAAAAA motifs detected in file\n",
      "\n",
      "Seq is: TTACGTC\n",
      "0 TTACGTC motifs detected in file\n",
      "\n",
      "Complement found from 1720 to 1727\n",
      "Complement found from 5931 to 5938\n",
      "Complement found from 8125 to 8132\n",
      "Complement found from 8189 to 8196\n",
      "Seq is: CAAACCA\n",
      "4 CAAACCA motifs detected in file\n",
      "\n",
      "Complement found from 7874 to 7881\n",
      "Complement found from 16346 to 16353\n",
      "Seq is: TCAAATC\n",
      "2 TCAAATC motifs detected in file\n",
      "\n",
      "Seq is: CTGGACA\n",
      "0 CTGGACA motifs detected in file\n",
      "\n",
      "Complement found from 1063 to 1070\n",
      "Complement found from 1682 to 1689\n",
      "Complement found from 2233 to 2240\n",
      "Complement found from 3715 to 3722\n",
      "Complement found from 12547 to 12554\n",
      "Complement found from 12668 to 12675\n",
      "Seq is: CCCAAAC\n",
      "6 CCCAAAC motifs detected in file\n",
      "\n",
      "Complement found from 7135 to 7142\n",
      "Complement found from 12445 to 12452\n",
      "Seq is: AATCCAT\n",
      "2 AATCCAT motifs detected in file\n",
      "\n",
      "Complement found from 1989 to 1996\n",
      "Complement found from 11591 to 11598\n",
      "Seq is: GACAAAC\n",
      "2 GACAAAC motifs detected in file\n",
      "\n",
      "Complement found from 15178 to 15185\n",
      "Seq is: GTAATTA\n",
      "1 GTAATTA motifs detected in file\n",
      "\n",
      "Complement found from 2736 to 2743\n",
      "Complement found from 5635 to 5642\n",
      "Seq is: TTTAATT\n",
      "2 TTTAATT motifs detected in file\n",
      "\n",
      "Seq is: TCGAGGT\n",
      "0 TCGAGGT motifs detected in file\n",
      "\n",
      "Seq is: ATCCCAG\n",
      "0 ATCCCAG motifs detected in file\n",
      "\n",
      "Complement found from 1936 to 1943\n",
      "Seq is: AAGAGCA\n",
      "1 AAGAGCA motifs detected in file\n",
      "\n",
      "Seq is: GAACGAC\n",
      "0 GAACGAC motifs detected in file\n",
      "\n",
      "Seq is: ACAATAC\n",
      "0 ACAATAC motifs detected in file\n",
      "\n",
      "Seq is: GGGCGGA\n",
      "0 GGGCGGA motifs detected in file\n",
      "\n",
      "Seq is: GAAGTGC\n",
      "0 GAAGTGC motifs detected in file\n",
      "\n",
      "Seq is: CCGTCCA\n",
      "0 CCGTCCA motifs detected in file\n",
      "\n",
      "Complement found from 8327 to 8334\n",
      "Seq is: GTTAAAG\n",
      "1 GTTAAAG motifs detected in file\n",
      "\n",
      "Complement found from 2907 to 2914\n",
      "Seq is: TGACCAA\n",
      "1 TGACCAA motifs detected in file\n",
      "\n",
      "Complement found from 7308 to 7315\n",
      "Complement found from 14932 to 14939\n",
      "Seq is: TTTTCAT\n",
      "2 TTTTCAT motifs detected in file\n",
      "\n",
      "Complement found from 11894 to 11901\n",
      "Complement found from 16017 to 16024\n",
      "Seq is: TCTCTGT\n",
      "2 TCTCTGT motifs detected in file\n",
      "\n",
      "Seq is: CGACTTG\n",
      "0 CGACTTG motifs detected in file\n",
      "\n",
      "Complement found from 105 to 112\n",
      "Seq is: GGAGCAC\n",
      "1 GGAGCAC motifs detected in file\n",
      "\n",
      "Seq is: CTCGGTA\n",
      "0 CTCGGTA motifs detected in file\n",
      "\n",
      "Seq is: AGTATGT\n",
      "0 AGTATGT motifs detected in file\n",
      "\n",
      "Seq is: CCAGGGA\n",
      "0 CCAGGGA motifs detected in file\n",
      "\n",
      "Seq is: TAAATTC\n",
      "0 TAAATTC motifs detected in file\n",
      "\n",
      "Complement found from 2604 to 2611\n",
      "Seq is: CTTGTTC\n",
      "1 CTTGTTC motifs detected in file\n",
      "\n",
      "Complement found from 3767 to 3774\n",
      "Complement found from 5497 to 5504\n",
      "Complement found from 7735 to 7742\n",
      "Complement found from 9871 to 9878\n",
      "Complement found from 10105 to 10112\n",
      "Complement found from 10312 to 10319\n",
      "Complement found from 11817 to 11824\n",
      "Complement found from 15902 to 15909\n",
      "Seq is: ACTAATA\n",
      "8 ACTAATA motifs detected in file\n",
      "\n",
      "Seq is: CGATGGA\n",
      "0 CGATGGA motifs detected in file\n",
      "\n",
      "Seq is: AACGTGC\n",
      "0 AACGTGC motifs detected in file\n",
      "\n",
      "Complement found from 13777 to 13784\n",
      "Complement found from 13942 to 13949\n",
      "Seq is: CAATCCC\n",
      "2 CAATCCC motifs detected in file\n",
      "\n",
      "Complement found from 9376 to 9383\n",
      "Complement found from 14999 to 15006\n",
      "Seq is: ATGGCGC\n",
      "2 ATGGCGC motifs detected in file\n",
      "\n",
      "Seq is: CGGCAAT\n",
      "0 CGGCAAT motifs detected in file\n",
      "\n",
      "Seq is: TTGTACA\n",
      "0 TTGTACA motifs detected in file\n",
      "\n",
      "Complement found from 2548 to 2555\n",
      "Seq is: CAGTGAC\n",
      "1 CAGTGAC motifs detected in file\n",
      "\n",
      "Seq is: CCGTCCG\n",
      "0 CCGTCCG motifs detected in file\n",
      "\n",
      "Complement found from 6883 to 6890\n",
      "Complement found from 9814 to 9821\n",
      "Seq is: CCACGGA\n",
      "2 CCACGGA motifs detected in file\n",
      "\n",
      "Seq is: GATTATG\n",
      "0 GATTATG motifs detected in file\n",
      "\n",
      "Complement found from 4017 to 4024\n",
      "Complement found from 11019 to 11026\n",
      "Seq is: ACCACTA\n",
      "2 ACCACTA motifs detected in file\n",
      "\n",
      "Seq is: CGATCTC\n",
      "0 CGATCTC motifs detected in file\n",
      "\n",
      "Complement found from 4019 to 4026\n",
      "Seq is: CACTACA\n",
      "1 CACTACA motifs detected in file\n",
      "\n",
      "Complement found from 7528 to 7535\n",
      "Complement found from 13425 to 13432\n",
      "Complement found from 14711 to 14718\n",
      "Seq is: AAAACCA\n",
      "3 AAAACCA motifs detected in file\n",
      "\n",
      "Seq is: TTTGTCC\n",
      "0 TTTGTCC motifs detected in file\n",
      "\n",
      "Seq is: GCCCCAT\n",
      "0 GCCCCAT motifs detected in file\n",
      "\n",
      "Complement found from 8118 to 8125\n",
      "Complement found from 12944 to 12951\n",
      "Seq is: TCTAAAC\n",
      "2 TCTAAAC motifs detected in file\n",
      "\n",
      "Complement found from 9785 to 9792\n",
      "Complement found from 9836 to 9843\n",
      "Seq is: GGCTCAA\n",
      "2 GGCTCAA motifs detected in file\n",
      "\n",
      "Seq is: GGAAAAT\n",
      "0 GGAAAAT motifs detected in file\n",
      "\n",
      "Complement found from 2128 to 2135\n",
      "Complement found from 4174 to 4181\n",
      "Complement found from 5745 to 5752\n",
      "Complement found from 6690 to 6697\n",
      "Complement found from 11030 to 11037\n",
      "Seq is: GAAAAAA\n",
      "5 GAAAAAA motifs detected in file\n",
      "\n",
      "Complement found from 5945 to 5952\n",
      "Complement found from 9045 to 9052\n",
      "Seq is: ATTGGAA\n",
      "2 ATTGGAA motifs detected in file\n",
      "\n",
      "Complement found from 4786 to 4793\n",
      "Complement found from 6787 to 6794\n",
      "Complement found from 10519 to 10526\n",
      "Complement found from 10570 to 10577\n",
      "Complement found from 13478 to 13485\n",
      "Seq is: AGGAATA\n",
      "5 AGGAATA motifs detected in file\n",
      "\n",
      "Seq is: CTCGTAC\n",
      "0 CTCGTAC motifs detected in file\n",
      "\n",
      "Seq is: GGACACA\n",
      "0 GGACACA motifs detected in file\n",
      "\n",
      "Complement found from 1624 to 1631\n",
      "Complement found from 2424 to 2431\n",
      "Complement found from 2832 to 2839\n",
      "Complement found from 3578 to 3585\n",
      "Complement found from 8434 to 8441\n",
      "Complement found from 8658 to 8665\n",
      "Complement found from 8811 to 8818\n",
      "Seq is: ACCCAAC\n",
      "7 ACCCAAC motifs detected in file\n",
      "\n",
      "Complement found from 13811 to 13818\n",
      "Seq is: TGTCACT\n",
      "1 TGTCACT motifs detected in file\n",
      "\n",
      "Complement found from 4227 to 4234\n",
      "Complement found from 7956 to 7963\n",
      "Complement found from 12433 to 12440\n",
      "Complement found from 13692 to 13699\n",
      "Complement found from 14629 to 14636\n",
      "Seq is: CCCATTA\n",
      "5 CCCATTA motifs detected in file\n",
      "\n",
      "Seq is: TTACTGA\n",
      "0 TTACTGA motifs detected in file\n",
      "\n",
      "Complement found from 10837 to 10844\n",
      "Complement found from 14329 to 14336\n",
      "Complement found from 14476 to 14483\n",
      "Complement found from 14689 to 14696\n",
      "Complement found from 15765 to 15772\n",
      "Seq is: ACAACCA\n",
      "5 ACAACCA motifs detected in file\n",
      "\n",
      "Complement found from 439 to 446\n",
      "Complement found from 5185 to 5192\n",
      "Complement found from 6529 to 6536\n",
      "Complement found from 7741 to 7748\n",
      "Complement found from 13656 to 13663\n",
      "Complement found from 13740 to 13747\n",
      "Seq is: ACTAACA\n",
      "6 ACTAACA motifs detected in file\n",
      "\n",
      "Complement found from 8820 to 8827\n",
      "Seq is: TCTATAA\n",
      "1 TCTATAA motifs detected in file\n",
      "\n",
      "Complement found from 12109 to 12116\n",
      "Seq is: CCCGACA\n",
      "1 CCCGACA motifs detected in file\n",
      "\n",
      "Complement found from 2369 to 2376\n",
      "Complement found from 12025 to 12032\n",
      "Complement found from 15468 to 15475\n",
      "Seq is: ATTAACA\n",
      "3 ATTAACA motifs detected in file\n",
      "\n",
      "Seq is: GTCAAGT\n",
      "0 GTCAAGT motifs detected in file\n",
      "\n",
      "Complement found from 1008 to 1015\n",
      "Complement found from 5278 to 5285\n",
      "Complement found from 13216 to 13223\n",
      "Seq is: CACAAAA\n",
      "3 CACAAAA motifs detected in file\n",
      "\n",
      "Complement found from 10388 to 10395\n",
      "Seq is: TTAGACT\n",
      "1 TTAGACT motifs detected in file\n",
      "\n",
      "Seq is: GCGTCCG\n",
      "0 GCGTCCG motifs detected in file\n",
      "\n",
      "Complement found from 3946 to 3953\n",
      "Complement found from 11487 to 11494\n",
      "Complement found from 14754 to 14761\n",
      "Seq is: AATACGC\n",
      "3 AATACGC motifs detected in file\n",
      "\n",
      "Complement found from 5210 to 5217\n",
      "Complement found from 6367 to 6374\n",
      "Seq is: CTCCTCT\n",
      "2 CTCCTCT motifs detected in file\n",
      "\n",
      "Complement found from 5425 to 5432\n",
      "Complement found from 10377 to 10384\n",
      "Seq is: TACAAAA\n",
      "2 TACAAAA motifs detected in file\n",
      "\n",
      "Seq is: GTACAAT\n",
      "0 GTACAAT motifs detected in file\n",
      "\n",
      "Seq is: GAATATG\n",
      "0 GAATATG motifs detected in file\n",
      "\n",
      "Complement found from 6671 to 6678\n",
      "Seq is: ATTGTAA\n",
      "1 ATTGTAA motifs detected in file\n",
      "\n",
      "Seq is: TCAAGAA\n",
      "0 TCAAGAA motifs detected in file\n",
      "\n",
      "Seq is: GATATCC\n",
      "0 GATATCC motifs detected in file\n",
      "\n",
      "Complement found from 7144 to 7151\n",
      "Complement found from 9850 to 9857\n",
      "Complement found from 11021 to 11028\n",
      "Seq is: CACTATC\n",
      "3 CACTATC motifs detected in file\n",
      "\n",
      "Complement found from 382 to 389\n",
      "Complement found from 2666 to 2673\n",
      "Seq is: TAACCAG\n",
      "2 TAACCAG motifs detected in file\n",
      "\n",
      "Complement found from 12405 to 12412\n",
      "Seq is: GTTAACC\n",
      "1 GTTAACC motifs detected in file\n",
      "\n",
      "Complement found from 4516 to 4523\n",
      "Complement found from 5449 to 5456\n",
      "Complement found from 6877 to 6884\n",
      "Complement found from 11496 to 11503\n",
      "Complement found from 14643 to 14650\n",
      "Seq is: CACACTC\n",
      "5 CACACTC motifs detected in file\n",
      "\n",
      "Complement found from 1497 to 1504\n",
      "Seq is: CTCAAGT\n",
      "1 CTCAAGT motifs detected in file\n",
      "\n",
      "Complement found from 4699 to 4706\n",
      "Complement found from 6409 to 6416\n",
      "Complement found from 14174 to 14181\n",
      "Seq is: CAATATA\n",
      "3 CAATATA motifs detected in file\n",
      "\n",
      "Complement found from 298 to 305\n",
      "Complement found from 348 to 355\n",
      "Complement found from 552 to 559\n",
      "Complement found from 1273 to 1280\n",
      "Complement found from 1684 to 1691\n",
      "Complement found from 2765 to 2772\n",
      "Complement found from 4982 to 4989\n",
      "Complement found from 8508 to 8515\n",
      "Complement found from 11668 to 11675\n",
      "Complement found from 14624 to 14631\n",
      "Seq is: CAAACCC\n",
      "10 CAAACCC motifs detected in file\n",
      "\n",
      "Complement found from 991 to 998\n",
      "Complement found from 2149 to 2156\n",
      "Complement found from 2225 to 2232\n",
      "Complement found from 2455 to 2462\n",
      "Complement found from 8495 to 8502\n",
      "Complement found from 14502 to 14509\n",
      "Seq is: TAAAAAA\n",
      "6 TAAAAAA motifs detected in file\n",
      "\n",
      "Complement found from 4949 to 4956\n",
      "Complement found from 12630 to 12637\n",
      "Complement found from 13368 to 13375\n",
      "Complement found from 14055 to 14062\n",
      "Seq is: TCCATCA\n",
      "4 TCCATCA motifs detected in file\n",
      "\n",
      "Complement found from 5676 to 5683\n",
      "Complement found from 7180 to 7187\n",
      "Complement found from 8559 to 8566\n",
      "Complement found from 12483 to 12490\n",
      "Complement found from 12926 to 12933\n",
      "Complement found from 14620 to 14627\n",
      "Seq is: CCCACAA\n",
      "6 CCCACAA motifs detected in file\n",
      "\n",
      "Complement found from 12524 to 12531\n",
      "Seq is: CTCGAAC\n",
      "1 CTCGAAC motifs detected in file\n",
      "\n",
      "Seq is: TTGCGAA\n",
      "0 TTGCGAA motifs detected in file\n",
      "\n",
      "Complement found from 8217 to 8224\n",
      "Seq is: AGAATTA\n",
      "1 AGAATTA motifs detected in file\n",
      "\n",
      "Seq is: ACCACCG\n",
      "0 ACCACCG motifs detected in file\n",
      "\n",
      "Complement found from 3377 to 3384\n",
      "Complement found from 8530 to 8537\n",
      "Complement found from 11028 to 11035\n",
      "Complement found from 13666 to 13673\n",
      "Seq is: ACGAAAA\n",
      "4 ACGAAAA motifs detected in file\n",
      "\n",
      "Seq is: TCCGGAT\n",
      "0 TCCGGAT motifs detected in file\n",
      "\n",
      "Complement found from 1078 to 1085\n",
      "Seq is: GATACCC\n",
      "1 GATACCC motifs detected in file\n",
      "\n",
      "Complement found from 9701 to 9708\n",
      "Complement found from 14028 to 14035\n",
      "Seq is: ACAATTT\n",
      "2 ACAATTT motifs detected in file\n",
      "\n",
      "Complement found from 285 to 292\n",
      "Complement found from 2150 to 2157\n",
      "Complement found from 2226 to 2233\n",
      "Complement found from 4605 to 4612\n",
      "Complement found from 8496 to 8503\n",
      "Complement found from 13231 to 13238\n",
      "Seq is: AAAAAAT\n",
      "6 AAAAAAT motifs detected in file\n",
      "\n",
      "Complement found from 2142 to 2149\n",
      "Seq is: GAGAGAG\n",
      "1 GAGAGAG motifs detected in file\n",
      "\n",
      "Seq is: ATGTTCC\n",
      "0 ATGTTCC motifs detected in file\n",
      "\n",
      "Complement found from 2457 to 2464\n",
      "Complement found from 5746 to 5753\n",
      "Complement found from 6692 to 6699\n",
      "Complement found from 7446 to 7453\n",
      "Complement found from 10047 to 10054\n",
      "Seq is: AAAAAAG\n",
      "5 AAAAAAG motifs detected in file\n",
      "\n",
      "Complement found from 0 to 7\n",
      "Seq is: GATCACA\n",
      "1 GATCACA motifs detected in file\n",
      "\n",
      "Complement found from 3093 to 3100\n",
      "Complement found from 13497 to 13504\n",
      "Seq is: GGTTTCT\n",
      "2 GGTTTCT motifs detected in file\n",
      "\n",
      "Seq is: CGACAAG\n",
      "0 CGACAAG motifs detected in file\n",
      "\n",
      "Seq is: GAGAAAC\n",
      "0 GAGAAAC motifs detected in file\n",
      "\n",
      "Seq is: CTGATTG\n",
      "0 CTGATTG motifs detected in file\n",
      "\n",
      "Complement found from 878 to 885\n",
      "Complement found from 6387 to 6394\n",
      "Seq is: TCAATTT\n",
      "2 TCAATTT motifs detected in file\n",
      "\n",
      "Complement found from 612 to 619\n",
      "Complement found from 4441 to 4448\n",
      "Seq is: AAATGTT\n",
      "2 AAATGTT motifs detected in file\n",
      "\n",
      "Complement found from 3483 to 3490\n",
      "Complement found from 6152 to 6159\n",
      "Complement found from 8227 to 8234\n",
      "Complement found from 9254 to 9261\n",
      "Complement found from 10936 to 10943\n",
      "Complement found from 14489 to 14496\n",
      "Complement found from 14770 to 14777\n",
      "Seq is: CCCCTAA\n",
      "7 CCCCTAA motifs detected in file\n",
      "\n",
      "Complement found from 5483 to 5490\n",
      "Complement found from 6662 to 6669\n",
      "Complement found from 11056 to 11063\n",
      "Complement found from 15850 to 15857\n",
      "Seq is: ATCTCCC\n",
      "4 ATCTCCC motifs detected in file\n",
      "\n",
      "Complement found from 561 to 568\n",
      "Seq is: AAGACAC\n",
      "1 AAGACAC motifs detected in file\n",
      "\n",
      "Complement found from 3640 to 3647\n",
      "Seq is: CCGTTTA\n",
      "1 CCGTTTA motifs detected in file\n",
      "\n",
      "Complement found from 6389 to 6396\n",
      "Complement found from 14030 to 14037\n",
      "Seq is: AATTTCA\n",
      "2 AATTTCA motifs detected in file\n",
      "\n",
      "Seq is: ACTTGAT\n",
      "0 ACTTGAT motifs detected in file\n",
      "\n",
      "Complement found from 8880 to 8887\n",
      "Seq is: TCTAAGA\n",
      "1 TCTAAGA motifs detected in file\n",
      "\n",
      "Complement found from 7967 to 7974\n",
      "Seq is: TAGAACC\n",
      "1 TAGAACC motifs detected in file\n",
      "\n",
      "Complement found from 4444 to 4451\n",
      "Seq is: TGTTGGT\n",
      "1 TGTTGGT motifs detected in file\n",
      "\n",
      "Seq is: CGATAGT\n",
      "0 CGATAGT motifs detected in file\n",
      "\n",
      "Seq is: GGTCCGA\n",
      "0 GGTCCGA motifs detected in file\n",
      "\n",
      "Complement found from 8838 to 8845\n",
      "Complement found from 15199 to 15206\n",
      "Seq is: GCCATCC\n",
      "2 GCCATCC motifs detected in file\n",
      "\n",
      "Complement found from 811 to 818\n",
      "Seq is: AAACAGC\n",
      "1 AAACAGC motifs detected in file\n",
      "\n",
      "Complement found from 15926 to 15933\n",
      "Seq is: GGAGATG\n",
      "1 GGAGATG motifs detected in file\n",
      "\n",
      "Complement found from 15560 to 15567\n",
      "Seq is: GATATTT\n",
      "1 GATATTT motifs detected in file\n",
      "\n",
      "Seq is: AGAAGGG\n",
      "0 AGAAGGG motifs detected in file\n",
      "\n",
      "Complement found from 4273 to 4280\n",
      "Seq is: TGATAAA\n",
      "1 TGATAAA motifs detected in file\n",
      "\n",
      "Seq is: ACGATGT\n",
      "0 ACGATGT motifs detected in file\n",
      "\n",
      "Complement found from 11581 to 11588\n",
      "Complement found from 15019 to 15026\n",
      "Seq is: ATCTGCC\n",
      "2 ATCTGCC motifs detected in file\n",
      "\n",
      "Complement found from 903 to 910\n",
      "Complement found from 6802 to 6809\n",
      "Complement found from 12048 to 12055\n",
      "Complement found from 15270 to 15277\n",
      "Seq is: CACACGA\n",
      "4 CACACGA motifs detected in file\n",
      "\n",
      "Complement found from 5808 to 5815\n",
      "Complement found from 8532 to 8539\n",
      "Seq is: GAAAATC\n",
      "2 GAAAATC motifs detected in file\n",
      "\n",
      "Complement found from 3131 to 3138\n",
      "Seq is: GACAAGA\n",
      "1 GACAAGA motifs detected in file\n",
      "\n",
      "Complement found from 4948 to 4955\n",
      "Complement found from 15633 to 15640\n",
      "Seq is: ATCCATC\n",
      "2 ATCCATC motifs detected in file\n",
      "\n",
      "Complement found from 2825 to 2832\n",
      "Complement found from 12817 to 12824\n",
      "Seq is: GAGCAGA\n",
      "2 GAGCAGA motifs detected in file\n",
      "\n",
      "Complement found from 1888 to 1895\n",
      "Complement found from 7469 to 7476\n",
      "Complement found from 8479 to 8486\n",
      "Complement found from 15989 to 15996\n",
      "Complement found from 16249 to 16256\n",
      "Seq is: CCAAAGC\n",
      "5 CCAAAGC motifs detected in file\n",
      "\n",
      "Complement found from 1374 to 1381\n",
      "Seq is: CCCCAGA\n",
      "1 CCCCAGA motifs detected in file\n",
      "\n",
      "Complement found from 8955 to 8962\n",
      "Complement found from 13521 to 13528\n",
      "Seq is: ATCGAAA\n",
      "2 ATCGAAA motifs detected in file\n",
      "\n",
      "Seq is: CCGAGAG\n",
      "0 CCGAGAG motifs detected in file\n",
      "\n",
      "Seq is: GAACGTT\n",
      "0 GAACGTT motifs detected in file\n",
      "\n",
      "Seq is: TCAATAA\n",
      "0 TCAATAA motifs detected in file\n",
      "\n",
      "Seq is: AGATCAA\n",
      "0 AGATCAA motifs detected in file\n",
      "\n",
      "Seq is: TTAAGTA\n",
      "0 TTAAGTA motifs detected in file\n",
      "\n",
      "Seq is: ATACGTC\n",
      "0 ATACGTC motifs detected in file\n",
      "\n",
      "Seq is: TTCCATA\n",
      "0 TTCCATA motifs detected in file\n",
      "\n",
      "Complement found from 960 to 967\n",
      "Complement found from 10915 to 10922\n",
      "Seq is: TCCCCAA\n",
      "2 TCCCCAA motifs detected in file\n",
      "\n",
      "Complement found from 7758 to 7765\n",
      "Seq is: TCAGGAA\n",
      "1 TCAGGAA motifs detected in file\n",
      "\n",
      "Seq is: CGATATA\n",
      "0 CGATATA motifs detected in file\n",
      "\n",
      "Seq is: ATACGAA\n",
      "0 ATACGAA motifs detected in file\n",
      "\n",
      "Complement found from 2378 to 2385\n",
      "Complement found from 10635 to 10642\n",
      "Seq is: CCAATAT\n",
      "2 CCAATAT motifs detected in file\n",
      "\n",
      "Complement found from 5828 to 5835\n",
      "Seq is: TAAAAAG\n",
      "1 TAAAAAG motifs detected in file\n",
      "\n",
      "Seq is: TAGAAAG\n",
      "0 TAGAAAG motifs detected in file\n",
      "\n",
      "Seq is: GGAACGC\n",
      "0 GGAACGC motifs detected in file\n",
      "\n",
      "Seq is: CATGATA\n",
      "0 CATGATA motifs detected in file\n",
      "\n",
      "Seq is: TAGATAA\n",
      "0 TAGATAA motifs detected in file\n",
      "\n",
      "Complement found from 897 to 904\n",
      "Seq is: CGCGGTC\n",
      "1 CGCGGTC motifs detected in file\n",
      "\n",
      "Complement found from 7549 to 7556\n",
      "Seq is: CAAAGTT\n",
      "1 CAAAGTT motifs detected in file\n",
      "\n",
      "Complement found from 11373 to 11380\n",
      "Seq is: AAAGATA\n",
      "1 AAAGATA motifs detected in file\n",
      "\n",
      "Seq is: GCGGATA\n",
      "0 GCGGATA motifs detected in file\n",
      "\n",
      "Complement found from 8241 to 8248\n",
      "Seq is: TGAAATA\n",
      "1 TGAAATA motifs detected in file\n",
      "\n",
      "Complement found from 4346 to 4353\n",
      "Complement found from 14516 to 14523\n",
      "Seq is: AACCCAT\n",
      "2 AACCCAT motifs detected in file\n",
      "\n",
      "Complement found from 2497 to 2504\n",
      "Complement found from 10462 to 10469\n",
      "Seq is: TTACCAA\n",
      "2 TTACCAA motifs detected in file\n",
      "\n",
      "Seq is: ACCGATT\n",
      "0 ACCGATT motifs detected in file\n",
      "\n",
      "Complement found from 9557 to 9564\n",
      "Seq is: CCAACAG\n",
      "1 CCAACAG motifs detected in file\n",
      "\n",
      "Complement found from 3504 to 3511\n",
      "Complement found from 5306 to 5313\n",
      "Complement found from 5318 to 5325\n",
      "Complement found from 7870 to 7877\n",
      "Complement found from 8961 to 8968\n",
      "Complement found from 14479 to 14486\n",
      "Complement found from 15787 to 15794\n",
      "Seq is: ACCATCA\n",
      "7 ACCATCA motifs detected in file\n",
      "\n",
      "Complement found from 292 to 299\n",
      "Complement found from 13122 to 13129\n",
      "Complement found from 15403 to 15410\n",
      "Seq is: TTCCACC\n",
      "3 TTCCACC motifs detected in file\n",
      "\n",
      "Complement found from 8432 to 8439\n",
      "Complement found from 12014 to 12021\n",
      "Complement found from 14354 to 14361\n",
      "Complement found from 16067 to 16074\n",
      "Complement found from 16262 to 16269\n",
      "Seq is: TCACCCA\n",
      "5 TCACCCA motifs detected in file\n",
      "\n",
      "Complement found from 1220 to 1227\n",
      "Complement found from 12106 to 12113\n",
      "Seq is: AACCCCG\n",
      "2 AACCCCG motifs detected in file\n",
      "\n",
      "Complement found from 4361 to 4368\n",
      "Complement found from 9889 to 9896\n",
      "Complement found from 11665 to 11672\n",
      "Complement found from 13152 to 13159\n",
      "Complement found from 14468 to 14475\n",
      "Complement found from 15672 to 15679\n",
      "Seq is: ATCCAAA\n",
      "6 ATCCAAA motifs detected in file\n",
      "\n",
      "Complement found from 9750 to 9757\n",
      "Seq is: TCGAGTC\n",
      "1 TCGAGTC motifs detected in file\n",
      "\n",
      "Complement found from 1131 to 1138\n",
      "Seq is: TCGCCAG\n",
      "1 TCGCCAG motifs detected in file\n",
      "\n",
      "Complement found from 3284 to 3291\n",
      "Seq is: TTCAATT\n",
      "1 TTCAATT motifs detected in file\n",
      "\n",
      "Complement found from 2042 to 2049\n",
      "Complement found from 9840 to 9847\n",
      "Complement found from 12263 to 12270\n",
      "Seq is: CAACTTT\n",
      "3 CAACTTT motifs detected in file\n",
      "\n",
      "Seq is: AGAGGAT\n",
      "0 AGAGGAT motifs detected in file\n",
      "\n",
      "Seq is: TCACATT\n",
      "0 TCACATT motifs detected in file\n",
      "\n",
      "Complement found from 2422 to 2429\n",
      "Complement found from 12544 to 12551\n",
      "Complement found from 12553 to 12560\n",
      "Complement found from 14068 to 14075\n",
      "Seq is: CAACCCA\n",
      "4 CAACCCA motifs detected in file\n",
      "\n",
      "Seq is: CGAAACA\n",
      "0 CGAAACA motifs detected in file\n",
      "\n",
      "Seq is: CAATTCG\n",
      "0 CAATTCG motifs detected in file\n",
      "\n",
      "Seq is: ATGTGAG\n",
      "0 ATGTGAG motifs detected in file\n",
      "\n",
      "Complement found from 1810 to 1817\n",
      "Seq is: ACCAAGC\n",
      "1 ACCAAGC motifs detected in file\n",
      "\n",
      "Complement found from 3281 to 3288\n",
      "Seq is: AGGTTCA\n",
      "1 AGGTTCA motifs detected in file\n",
      "\n",
      "Complement found from 16410 to 16417\n",
      "Seq is: CGTGAAA\n",
      "1 CGTGAAA motifs detected in file\n",
      "\n",
      "Seq is: GGTCATG\n",
      "0 GGTCATG motifs detected in file\n",
      "\n",
      "Complement found from 2629 to 2636\n",
      "Seq is: TGAATGG\n",
      "1 TGAATGG motifs detected in file\n",
      "\n",
      "Seq is: TACAATG\n",
      "0 TACAATG motifs detected in file\n",
      "\n",
      "Complement found from 1042 to 1049\n",
      "Seq is: CTGAACA\n",
      "1 CTGAACA motifs detected in file\n",
      "\n",
      "Complement found from 1548 to 1555\n",
      "Complement found from 6576 to 6583\n",
      "Seq is: GAGGAGA\n",
      "2 GAGGAGA motifs detected in file\n",
      "\n",
      "Complement found from 6775 to 6782\n",
      "Seq is: TATATTT\n",
      "1 TATATTT motifs detected in file\n",
      "\n",
      "Complement found from 7827 to 7834\n",
      "Seq is: ACGCATC\n",
      "1 ACGCATC motifs detected in file\n",
      "\n",
      "Complement found from 353 to 360\n",
      "Complement found from 12302 to 12309\n",
      "Complement found from 14071 to 14078\n",
      "Complement found from 14532 to 14539\n",
      "Seq is: CCCAAAA\n",
      "4 CCCAAAA motifs detected in file\n",
      "\n",
      "Complement found from 878 to 885\n",
      "Complement found from 6387 to 6394\n",
      "Seq is: TCAATTT\n",
      "2 TCAATTT motifs detected in file\n",
      "\n",
      "Seq is: ACAGGAA\n",
      "0 ACAGGAA motifs detected in file\n",
      "\n",
      "Complement found from 15290 to 15297\n",
      "Seq is: ACTTCAT\n",
      "1 ACTTCAT motifs detected in file\n",
      "\n",
      "Seq is: ATGAACT\n",
      "0 ATGAACT motifs detected in file\n",
      "\n",
      "Complement found from 457 to 464\n",
      "Complement found from 7815 to 7822\n",
      "Complement found from 15380 to 15387\n",
      "Seq is: CCTCCCA\n",
      "3 CCTCCCA motifs detected in file\n",
      "\n",
      "Complement found from 2683 to 2690\n",
      "Complement found from 7786 to 7793\n",
      "Seq is: CTGCCCG\n",
      "2 CTGCCCG motifs detected in file\n",
      "\n",
      "Complement found from 267 to 274\n",
      "Complement found from 7408 to 7415\n",
      "Complement found from 9415 to 9422\n",
      "Seq is: CCACACA\n",
      "3 CCACACA motifs detected in file\n",
      "\n",
      "Seq is: TGCGCGA\n",
      "0 TGCGCGA motifs detected in file\n",
      "\n",
      "Seq is: AGTCCCG\n",
      "0 AGTCCCG motifs detected in file\n",
      "\n",
      "Complement found from 3130 to 3137\n",
      "Complement found from 15796 to 15803\n",
      "Seq is: GGACAAG\n",
      "2 GGACAAG motifs detected in file\n",
      "\n",
      "Complement found from 133 to 140\n",
      "Seq is: TTGATTC\n",
      "1 TTGATTC motifs detected in file\n",
      "\n",
      "Seq is: GTGAGAT\n",
      "0 GTGAGAT motifs detected in file\n",
      "\n",
      "Complement found from 4338 to 4345\n",
      "Complement found from 4357 to 4364\n",
      "Seq is: GAGAATC\n",
      "2 GAGAATC motifs detected in file\n",
      "\n",
      "Complement found from 5408 to 5415\n",
      "Complement found from 13220 to 13227\n",
      "Seq is: AAATGAC\n",
      "2 AAATGAC motifs detected in file\n",
      "\n",
      "Complement found from 1421 to 1428\n",
      "Seq is: GATTTAG\n",
      "1 GATTTAG motifs detected in file\n",
      "\n",
      "Seq is: GTGGAAG\n",
      "0 GTGGAAG motifs detected in file\n",
      "\n",
      "Seq is: CTGGGAA\n",
      "0 CTGGGAA motifs detected in file\n",
      "\n",
      "Complement found from 1508 to 1515\n",
      "Complement found from 5542 to 5549\n",
      "Complement found from 13338 to 13345\n",
      "Seq is: TTCAAAG\n",
      "3 TTCAAAG motifs detected in file\n",
      "\n",
      "Complement found from 14150 to 14157\n",
      "Seq is: TATTCCC\n",
      "1 TATTCCC motifs detected in file\n",
      "\n",
      "Complement found from 80 to 87\n",
      "Seq is: GATAGCA\n",
      "1 GATAGCA motifs detected in file\n",
      "\n",
      "Complement found from 741 to 748\n",
      "Seq is: TCAAAAG\n",
      "1 TCAAAAG motifs detected in file\n",
      "\n",
      "Complement found from 532 to 539\n",
      "Complement found from 635 to 642\n",
      "Complement found from 4389 to 4396\n",
      "Complement found from 5435 to 5442\n",
      "Complement found from 6582 to 6589\n",
      "Complement found from 7489 to 7496\n",
      "Complement found from 13690 to 13697\n",
      "Complement found from 14338 to 14345\n",
      "Complement found from 14627 to 14634\n",
      "Complement found from 14812 to 14819\n",
      "Seq is: ACCCCAT\n",
      "10 ACCCCAT motifs detected in file\n",
      "\n",
      "Complement found from 9505 to 9512\n",
      "Complement found from 10271 to 10278\n",
      "Complement found from 15782 to 15789\n",
      "Seq is: CTTTTAC\n",
      "3 CTTTTAC motifs detected in file\n",
      "\n",
      "Seq is: ATCGGGT\n",
      "0 ATCGGGT motifs detected in file\n",
      "\n",
      "Complement found from 362 to 369\n",
      "Complement found from 6695 to 6702\n",
      "Seq is: AAAGAAC\n",
      "2 AAAGAAC motifs detected in file\n",
      "\n",
      "Seq is: GGTGGAG\n",
      "0 GGTGGAG motifs detected in file\n",
      "\n",
      "Seq is: TACCCGA\n",
      "0 TACCCGA motifs detected in file\n",
      "\n",
      "Seq is: TGTGGAA\n",
      "0 TGTGGAA motifs detected in file\n",
      "\n",
      "Seq is: CTGGATT\n",
      "0 CTGGATT motifs detected in file\n",
      "\n",
      "Complement found from 13134 to 13141\n",
      "Seq is: GCAGAAA\n",
      "1 GCAGAAA motifs detected in file\n",
      "\n",
      "Seq is: TGCACCC\n",
      "0 TGCACCC motifs detected in file\n",
      "\n",
      "Complement found from 8526 to 8533\n",
      "Seq is: ATGAACG\n",
      "1 ATGAACG motifs detected in file\n",
      "\n",
      "Complement found from 10425 to 10432\n",
      "Complement found from 15555 to 15562\n",
      "Seq is: CGAATGA\n",
      "2 CGAATGA motifs detected in file\n",
      "\n",
      "Complement found from 3939 to 3946\n",
      "Complement found from 15085 to 15092\n",
      "Seq is: AACATCG\n",
      "2 AACATCG motifs detected in file\n",
      "\n",
      "Seq is: GAAGTAG\n",
      "0 GAAGTAG motifs detected in file\n",
      "\n",
      "Complement found from 2232 to 2239\n",
      "Seq is: TCCCAAA\n",
      "1 TCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: CGACTTC\n",
      "0 CGACTTC motifs detected in file\n",
      "\n",
      "Complement found from 1258 to 1265\n",
      "Seq is: TACCGCC\n",
      "1 TACCGCC motifs detected in file\n",
      "\n",
      "Complement found from 7575 to 7582\n",
      "Complement found from 15668 to 15675\n",
      "Seq is: ATATATC\n",
      "2 ATATATC motifs detected in file\n",
      "\n",
      "Seq is: CGACTCG\n",
      "0 CGACTCG motifs detected in file\n",
      "\n",
      "Complement found from 10208 to 10215\n",
      "Complement found from 12588 to 12595\n",
      "Complement found from 13245 to 13252\n",
      "Complement found from 13899 to 13906\n",
      "Seq is: TTCTCCA\n",
      "4 TTCTCCA motifs detected in file\n",
      "\n",
      "Complement found from 461 to 468\n",
      "Complement found from 1689 to 1696\n",
      "Complement found from 9316 to 9323\n",
      "Complement found from 9511 to 9518\n",
      "Complement found from 9593 to 9600\n",
      "Complement found from 10621 to 10628\n",
      "Seq is: CCACTCC\n",
      "6 CCACTCC motifs detected in file\n",
      "\n",
      "Complement found from 4872 to 4879\n",
      "Seq is: AACTAGC\n",
      "1 AACTAGC motifs detected in file\n",
      "\n",
      "Complement found from 1731 to 1738\n",
      "Complement found from 5252 to 5259\n",
      "Seq is: CCCAAAT\n",
      "2 CCCAAAT motifs detected in file\n",
      "\n",
      "Seq is: AGCTAAT\n",
      "0 AGCTAAT motifs detected in file\n",
      "\n",
      "Complement found from 15914 to 15921\n",
      "Seq is: GTCTTGT\n",
      "1 GTCTTGT motifs detected in file\n",
      "\n",
      "Seq is: CCGAGGA\n",
      "0 CCGAGGA motifs detected in file\n",
      "\n",
      "Seq is: GATCTCC\n",
      "0 GATCTCC motifs detected in file\n",
      "\n",
      "Complement found from 857 to 864\n",
      "Complement found from 2883 to 2890\n",
      "Complement found from 6522 to 6529\n",
      "Complement found from 11048 to 11055\n",
      "Complement found from 11971 to 11978\n",
      "Complement found from 15814 to 15821\n",
      "Seq is: CTATACT\n",
      "6 CTATACT motifs detected in file\n",
      "\n",
      "Seq is: TCGTGGC\n",
      "0 TCGTGGC motifs detected in file\n",
      "\n",
      "Seq is: GGTCCAG\n",
      "0 GGTCCAG motifs detected in file\n",
      "\n",
      "Seq is: GAAACTC\n",
      "0 GAAACTC motifs detected in file\n",
      "\n",
      "Complement found from 3381 to 3388\n",
      "Complement found from 4365 to 4372\n",
      "Complement found from 10216 to 10223\n",
      "Complement found from 14535 to 14542\n",
      "Seq is: AAAATTC\n",
      "4 AAAATTC motifs detected in file\n",
      "\n",
      "Seq is: GACACCG\n",
      "0 GACACCG motifs detected in file\n",
      "\n",
      "Complement found from 753 to 760\n",
      "Complement found from 2517 to 2524\n",
      "Complement found from 3668 to 3675\n",
      "Complement found from 10861 to 10868\n",
      "Complement found from 13926 to 13933\n",
      "Seq is: AGCATCA\n",
      "5 AGCATCA motifs detected in file\n",
      "\n",
      "Complement found from 4088 to 4095\n",
      "Complement found from 14406 to 14413\n",
      "Seq is: CAAGACC\n",
      "2 CAAGACC motifs detected in file\n",
      "\n",
      "Seq is: GCTCGTC\n",
      "0 GCTCGTC motifs detected in file\n",
      "\n",
      "Complement found from 357 to 364\n",
      "Complement found from 5283 to 5290\n",
      "Complement found from 8687 to 8694\n",
      "Complement found from 9663 to 9670\n",
      "Complement found from 14025 to 14032\n",
      "Complement found from 15863 to 15870\n",
      "Seq is: AAAACAA\n",
      "6 AAAACAA motifs detected in file\n",
      "\n",
      "Seq is: CTAAATT\n",
      "0 CTAAATT motifs detected in file\n",
      "\n",
      "Seq is: GACAACT\n",
      "0 GACAACT motifs detected in file\n",
      "\n",
      "Complement found from 4897 to 4904\n",
      "Complement found from 10887 to 10894\n",
      "Complement found from 14042 to 14049\n",
      "Complement found from 14874 to 14881\n",
      "Seq is: CCAAATC\n",
      "4 CCAAATC motifs detected in file\n",
      "\n",
      "Complement found from 4139 to 4146\n",
      "Seq is: CCGATTC\n",
      "1 CCGATTC motifs detected in file\n",
      "\n",
      "Seq is: GTATCAC\n",
      "0 GTATCAC motifs detected in file\n",
      "\n",
      "Complement found from 2163 to 2170\n",
      "Seq is: CCCATAG\n",
      "1 CCCATAG motifs detected in file\n",
      "\n",
      "Seq is: ATTAGGG\n",
      "0 ATTAGGG motifs detected in file\n",
      "\n",
      "Complement found from 4253 to 4260\n",
      "Seq is: TCAAACC\n",
      "1 TCAAACC motifs detected in file\n",
      "\n",
      "Seq is: CAGAATC\n",
      "0 CAGAATC motifs detected in file\n",
      "\n",
      "Complement found from 2933 to 2940\n",
      "Complement found from 12275 to 12282\n",
      "Seq is: GATAACA\n",
      "2 GATAACA motifs detected in file\n",
      "\n",
      "Seq is: CACAAGT\n",
      "0 CACAAGT motifs detected in file\n",
      "\n",
      "Complement found from 12852 to 12859\n",
      "Seq is: CTATACA\n",
      "1 CTATACA motifs detected in file\n",
      "\n",
      "Complement found from 2804 to 2811\n",
      "Seq is: ATTTCGG\n",
      "1 ATTTCGG motifs detected in file\n",
      "\n",
      "Seq is: TGAAAGC\n",
      "0 TGAAAGC motifs detected in file\n",
      "\n",
      "Complement found from 12159 to 12166\n",
      "Seq is: ATCAGAT\n",
      "1 ATCAGAT motifs detected in file\n",
      "\n",
      "Complement found from 2352 to 2359\n",
      "Complement found from 10815 to 10822\n",
      "Complement found from 11520 to 11527\n",
      "Complement found from 12056 to 12063\n",
      "Complement found from 14395 to 14402\n",
      "Seq is: AAAACAC\n",
      "5 AAAACAC motifs detected in file\n",
      "\n",
      "Complement found from 1002 to 1009\n",
      "Seq is: AGTTGAC\n",
      "1 AGTTGAC motifs detected in file\n",
      "\n",
      "Complement found from 777 to 784\n",
      "Complement found from 8684 to 8691\n",
      "Complement found from 13422 to 13429\n",
      "Seq is: CTCAAAA\n",
      "3 CTCAAAA motifs detected in file\n",
      "\n",
      "Complement found from 4442 to 4449\n",
      "Seq is: AATGTTG\n",
      "1 AATGTTG motifs detected in file\n",
      "\n",
      "Seq is: AGTCCAC\n",
      "0 AGTCCAC motifs detected in file\n",
      "\n",
      "Seq is: TCAAAAT\n",
      "0 TCAAAAT motifs detected in file\n",
      "\n",
      "Seq is: CGAAATA\n",
      "0 CGAAATA motifs detected in file\n",
      "\n",
      "Complement found from 4832 to 4839\n",
      "Complement found from 10977 to 10984\n",
      "Complement found from 14279 to 14286\n",
      "Complement found from 15535 to 15542\n",
      "Complement found from 16257 to 16264\n",
      "Seq is: ACCCCTC\n",
      "5 ACCCCTC motifs detected in file\n",
      "\n",
      "Complement found from 1082 to 1089\n",
      "Complement found from 5600 to 5607\n",
      "Complement found from 5895 to 5902\n",
      "Complement found from 11868 to 11875\n",
      "Complement found from 12967 to 12974\n",
      "Complement found from 13879 to 13886\n",
      "Complement found from 14388 to 14395\n",
      "Seq is: CCCCACT\n",
      "7 CCCCACT motifs detected in file\n",
      "\n",
      "Seq is: AGATTTT\n",
      "0 AGATTTT motifs detected in file\n",
      "\n",
      "Seq is: GTGAGAA\n",
      "0 GTGAGAA motifs detected in file\n",
      "\n",
      "Seq is: ATGCGGC\n",
      "0 ATGCGGC motifs detected in file\n",
      "\n",
      "Seq is: CGAAGAT\n",
      "0 CGAAGAT motifs detected in file\n",
      "\n",
      "Complement found from 2244 to 2251\n",
      "Complement found from 5614 to 5621\n",
      "Seq is: AACTGAA\n",
      "2 AACTGAA motifs detected in file\n",
      "\n",
      "Complement found from 2260 to 2267\n",
      "Complement found from 9534 to 9541\n",
      "Complement found from 13009 to 13016\n",
      "Seq is: CCCAATT\n",
      "3 CCCAATT motifs detected in file\n",
      "\n",
      "Complement found from 511 to 518\n",
      "Complement found from 1939 to 1946\n",
      "Complement found from 6766 to 6773\n",
      "Seq is: AGCACAC\n",
      "3 AGCACAC motifs detected in file\n",
      "\n",
      "Complement found from 14856 to 14863\n",
      "Complement found from 15000 to 15007\n",
      "Seq is: TGGCGCC\n",
      "2 TGGCGCC motifs detected in file\n",
      "\n",
      "Seq is: ACCGACC\n",
      "0 ACCGACC motifs detected in file\n",
      "\n",
      "Seq is: GTGCTTT\n",
      "0 GTGCTTT motifs detected in file\n",
      "\n",
      "Seq is: AACTGGT\n",
      "0 AACTGGT motifs detected in file\n",
      "\n",
      "Seq is: TGGGACC\n",
      "0 TGGGACC motifs detected in file\n",
      "\n",
      "Complement found from 4737 to 4744\n",
      "Complement found from 9214 to 9221\n",
      "Complement found from 15708 to 15715\n",
      "Seq is: CCAATCA\n",
      "3 CCAATCA motifs detected in file\n",
      "\n",
      "Complement found from 5266 to 5273\n",
      "Complement found from 8954 to 8961\n",
      "Seq is: TATCGAA\n",
      "2 TATCGAA motifs detected in file\n",
      "\n",
      "Complement found from 878 to 885\n",
      "Complement found from 6387 to 6394\n",
      "Seq is: TCAATTT\n",
      "2 TCAATTT motifs detected in file\n",
      "\n",
      "Complement found from 9395 to 9402\n",
      "Seq is: GAAAGCA\n",
      "1 GAAAGCA motifs detected in file\n",
      "\n",
      "Seq is: AATAACG\n",
      "0 AATAACG motifs detected in file\n",
      "\n",
      "Complement found from 290 to 297\n",
      "Seq is: ATTTCCA\n",
      "1 ATTTCCA motifs detected in file\n",
      "\n",
      "Seq is: ATTAGTG\n",
      "0 ATTAGTG motifs detected in file\n",
      "\n",
      "Complement found from 9193 to 9200\n",
      "Complement found from 11589 to 11596\n",
      "Seq is: ACGACAA\n",
      "2 ACGACAA motifs detected in file\n",
      "\n",
      "Complement found from 9544 to 9551\n",
      "Seq is: AGGGCAC\n",
      "1 AGGGCAC motifs detected in file\n",
      "\n",
      "Complement found from 569 to 576\n",
      "Complement found from 5446 to 5453\n",
      "Complement found from 8079 to 8086\n",
      "Complement found from 8558 to 8565\n",
      "Complement found from 12482 to 12489\n",
      "Complement found from 14619 to 14626\n",
      "Complement found from 15541 to 15548\n",
      "Seq is: CCCCACA\n",
      "7 CCCCACA motifs detected in file\n",
      "\n",
      "Complement found from 15588 to 15595\n",
      "Seq is: CCGATCC\n",
      "1 CCGATCC motifs detected in file\n",
      "\n",
      "Seq is: GATTCGC\n",
      "0 GATTCGC motifs detected in file\n",
      "\n",
      "Complement found from 974 to 981\n",
      "Complement found from 994 to 1001\n",
      "Complement found from 1158 to 1165\n",
      "Complement found from 3464 to 3471\n",
      "Complement found from 11034 to 11041\n",
      "Complement found from 12421 to 12428\n",
      "Complement found from 13793 to 13800\n",
      "Seq is: AAAACTC\n",
      "7 AAAACTC motifs detected in file\n",
      "\n",
      "Complement found from 14959 to 14966\n",
      "Seq is: GACGTAA\n",
      "1 GACGTAA motifs detected in file\n",
      "\n",
      "Seq is: CGACGCA\n",
      "0 CGACGCA motifs detected in file\n",
      "\n",
      "Complement found from 2876 to 2883\n",
      "Complement found from 3920 to 3927\n",
      "Seq is: CGAACTA\n",
      "2 CGAACTA motifs detected in file\n",
      "\n",
      "Seq is: CGAACAA\n",
      "0 CGAACAA motifs detected in file\n",
      "\n",
      "Seq is: GGAAAAC\n",
      "0 GGAAAAC motifs detected in file\n",
      "\n",
      "Complement found from 8266 to 8273\n",
      "Complement found from 15983 to 15990\n",
      "Seq is: TAGCACC\n",
      "2 TAGCACC motifs detected in file\n",
      "\n",
      "Complement found from 8373 to 8380\n",
      "Seq is: ACTAAAT\n",
      "1 ACTAAAT motifs detected in file\n",
      "\n",
      "Complement found from 458 to 465\n",
      "Complement found from 6310 to 6317\n",
      "Complement found from 11812 to 11819\n",
      "Seq is: CTCCCAC\n",
      "3 CTCCCAC motifs detected in file\n",
      "\n",
      "Seq is: TTGAGTG\n",
      "0 TTGAGTG motifs detected in file\n",
      "\n",
      "Complement found from 3901 to 3908\n",
      "Seq is: ACCTTGC\n",
      "1 ACCTTGC motifs detected in file\n",
      "\n",
      "Complement found from 1531 to 1538\n",
      "Complement found from 3429 to 3436\n",
      "Complement found from 7823 to 7830\n",
      "Seq is: CCCTACG\n",
      "3 CCCTACG motifs detected in file\n",
      "\n",
      "Seq is: AACGTAC\n",
      "0 AACGTAC motifs detected in file\n",
      "\n",
      "Complement found from 8083 to 8090\n",
      "Seq is: ACATTAG\n",
      "1 ACATTAG motifs detected in file\n",
      "\n",
      "Complement found from 10427 to 10434\n",
      "Seq is: AATGATT\n",
      "1 AATGATT motifs detected in file\n",
      "\n",
      "Seq is: CTCGATT\n",
      "0 CTCGATT motifs detected in file\n",
      "\n",
      "Complement found from 6935 to 6942\n",
      "Seq is: ATCTTTC\n",
      "1 ATCTTTC motifs detected in file\n",
      "\n",
      "Seq is: CGATCCT\n",
      "0 CGATCCT motifs detected in file\n",
      "\n",
      "Complement found from 651 to 658\n",
      "Seq is: GGTTTGG\n",
      "1 GGTTTGG motifs detected in file\n",
      "\n",
      "Complement found from 640 to 647\n",
      "Complement found from 4005 to 4012\n",
      "Complement found from 4571 to 4578\n",
      "Complement found from 6191 to 6198\n",
      "Seq is: ATAAACA\n",
      "4 ATAAACA motifs detected in file\n",
      "\n",
      "Complement found from 6112 to 6119\n",
      "Seq is: AATACCC\n",
      "1 AATACCC motifs detected in file\n",
      "\n",
      "Complement found from 4019 to 4026\n",
      "Seq is: CACTACA\n",
      "1 CACTACA motifs detected in file\n",
      "\n",
      "Seq is: CTCGGGC\n",
      "0 CTCGGGC motifs detected in file\n",
      "\n",
      "Complement found from 16040 to 16047\n",
      "Seq is: AGATTTG\n",
      "1 AGATTTG motifs detected in file\n",
      "\n",
      "Complement found from 12328 to 12335\n",
      "Seq is: TAAAAGT\n",
      "1 TAAAAGT motifs detected in file\n",
      "\n",
      "Complement found from 9200 to 9207\n",
      "Complement found from 10819 to 10826\n",
      "Complement found from 14141 to 14148\n",
      "Seq is: CACATAA\n",
      "3 CACATAA motifs detected in file\n",
      "\n",
      "Seq is: CGAAACT\n",
      "0 CGAAACT motifs detected in file\n",
      "\n",
      "Complement found from 7365 to 7372\n",
      "Complement found from 7498 to 7505\n",
      "Complement found from 14053 to 14060\n",
      "Complement found from 14375 to 14382\n",
      "Complement found from 15663 to 15670\n",
      "Seq is: CCTCCAT\n",
      "5 CCTCCAT motifs detected in file\n",
      "\n",
      "Complement found from 2986 to 2993\n",
      "Seq is: TCGATGT\n",
      "1 TCGATGT motifs detected in file\n",
      "\n",
      "Seq is: ATTTGAC\n",
      "0 ATTTGAC motifs detected in file\n",
      "\n",
      "Complement found from 301 to 308\n",
      "Complement found from 431 to 438\n",
      "Complement found from 566 to 573\n",
      "Complement found from 3564 to 3571\n",
      "Complement found from 7464 to 7471\n",
      "Complement found from 9530 to 9537\n",
      "Complement found from 10945 to 10952\n",
      "Complement found from 11865 to 11872\n",
      "Complement found from 16373 to 16380\n",
      "Seq is: ACCCCCC\n",
      "9 ACCCCCC motifs detected in file\n",
      "\n",
      "Complement found from 14651 to 14658\n",
      "Seq is: ACAGAAA\n",
      "1 ACAGAAA motifs detected in file\n",
      "\n",
      "Complement found from 352 to 359\n",
      "Complement found from 556 to 563\n",
      "Complement found from 1681 to 1688\n",
      "Complement found from 7467 to 7474\n",
      "Complement found from 12301 to 12308\n",
      "Complement found from 14531 to 14538\n",
      "Seq is: CCCCAAA\n",
      "6 CCCCAAA motifs detected in file\n",
      "\n",
      "Complement found from 547 to 554\n",
      "Complement found from 8805 to 8812\n",
      "Complement found from 13295 to 13302\n",
      "Seq is: CCAACCA\n",
      "3 CCAACCA motifs detected in file\n",
      "\n",
      "Complement found from 1679 to 1686\n",
      "Seq is: AGCCCCA\n",
      "1 AGCCCCA motifs detected in file\n",
      "\n",
      "Complement found from 1080 to 1087\n",
      "Complement found from 1372 to 1379\n",
      "Seq is: TACCCCA\n",
      "2 TACCCCA motifs detected in file\n",
      "\n",
      "Seq is: ATCGTCG\n",
      "0 ATCGTCG motifs detected in file\n",
      "\n",
      "Complement found from 267 to 274\n",
      "Complement found from 7408 to 7415\n",
      "Complement found from 9415 to 9422\n",
      "Seq is: CCACACA\n",
      "3 CCACACA motifs detected in file\n",
      "\n",
      "Complement found from 903 to 910\n",
      "Complement found from 6802 to 6809\n",
      "Complement found from 12048 to 12055\n",
      "Complement found from 15270 to 15277\n",
      "Seq is: CACACGA\n",
      "4 CACACGA motifs detected in file\n",
      "\n",
      "Complement found from 499 to 506\n",
      "Complement found from 4348 to 4355\n",
      "Complement found from 4391 to 4398\n",
      "Complement found from 7818 to 7825\n",
      "Complement found from 12387 to 12394\n",
      "Complement found from 14814 to 14821\n",
      "Complement found from 15658 to 15665\n",
      "Seq is: CCCATCC\n",
      "7 CCCATCC motifs detected in file\n",
      "\n",
      "Complement found from 8575 to 8582\n",
      "Seq is: TACCCGC\n",
      "1 TACCCGC motifs detected in file\n",
      "\n",
      "Complement found from 434 to 441\n",
      "Complement found from 3413 to 3420\n",
      "Complement found from 8368 to 8375\n",
      "Complement found from 9555 to 9562\n",
      "Complement found from 10916 to 10923\n",
      "Seq is: CCCCAAC\n",
      "5 CCCCAAC motifs detected in file\n",
      "\n",
      "Seq is: ATAACTA\n",
      "0 ATAACTA motifs detected in file\n",
      "\n",
      "Seq is: CTCTAAT\n",
      "0 CTCTAAT motifs detected in file\n",
      "\n",
      "Complement found from 276 to 283\n",
      "Complement found from 4951 to 4958\n",
      "Complement found from 5308 to 5315\n",
      "Complement found from 6118 to 6125\n",
      "Complement found from 7066 to 7073\n",
      "Complement found from 11778 to 11785\n",
      "Complement found from 12632 to 12639\n",
      "Complement found from 14342 to 14349\n",
      "Seq is: CATCATA\n",
      "8 CATCATA motifs detected in file\n",
      "\n",
      "Complement found from 2412 to 2419\n",
      "Complement found from 4012 to 4019\n",
      "Complement found from 4907 to 4914\n",
      "Complement found from 8473 to 8480\n",
      "Complement found from 10979 to 10986\n",
      "Complement found from 13450 to 13457\n",
      "Complement found from 15266 to 15273\n",
      "Complement found from 16259 to 16266\n",
      "Seq is: CCCTCAC\n",
      "8 CCCTCAC motifs detected in file\n",
      "\n",
      "Complement found from 307 to 314\n",
      "Complement found from 958 to 965\n",
      "Complement found from 3569 to 3576\n",
      "Complement found from 5443 to 5450\n",
      "Complement found from 14527 to 14534\n",
      "Complement found from 14805 to 14812\n",
      "Complement found from 15538 to 15545\n",
      "Complement found from 16186 to 16193\n",
      "Seq is: CCTCCCC\n",
      "8 CCTCCCC motifs detected in file\n",
      "\n",
      "Complement found from 4585 to 4592\n",
      "Complement found from 8756 to 8763\n",
      "Complement found from 13893 to 13900\n",
      "Seq is: TTTTATT\n",
      "3 TTTTATT motifs detected in file\n",
      "\n",
      "Complement found from 1048 to 1055\n",
      "Complement found from 11354 to 11361\n",
      "Complement found from 11998 to 12005\n",
      "Complement found from 15416 to 15423\n",
      "Complement found from 15578 to 15585\n",
      "Seq is: ACACAAT\n",
      "5 ACACAAT motifs detected in file\n",
      "\n",
      "Complement found from 490 to 497\n",
      "Complement found from 3581 to 3588\n",
      "Complement found from 7487 to 7494\n",
      "Complement found from 10943 to 10950\n",
      "Complement found from 11508 to 11515\n",
      "Complement found from 12105 to 12112\n",
      "Complement found from 14414 to 14421\n",
      "Complement found from 15363 to 15370\n",
      "Complement found from 15522 to 15529\n",
      "Seq is: CAACCCC\n",
      "9 CAACCCC motifs detected in file\n",
      "\n",
      "Complement found from 5897 to 5904\n",
      "Complement found from 10795 to 10802\n",
      "Seq is: CCACTGA\n",
      "2 CCACTGA motifs detected in file\n",
      "\n",
      "Complement found from 9702 to 9709\n",
      "Seq is: CAATTTT\n",
      "1 CAATTTT motifs detected in file\n",
      "\n",
      "Seq is: CACGTAT\n",
      "0 CACGTAT motifs detected in file\n",
      "\n",
      "Seq is: GGCGGTT\n",
      "0 GGCGGTT motifs detected in file\n",
      "\n",
      "Complement found from 667 to 674\n",
      "Complement found from 8596 to 8603\n",
      "Seq is: TTCTATT\n",
      "2 TTCTATT motifs detected in file\n",
      "\n",
      "Complement found from 5129 to 5136\n",
      "Complement found from 14086 to 14093\n",
      "Complement found from 16008 to 16015\n",
      "Seq is: TTAAACT\n",
      "3 TTAAACT motifs detected in file\n",
      "\n",
      "Seq is: TTAGACC\n",
      "0 TTAGACC motifs detected in file\n",
      "\n",
      "Seq is: AATCCGA\n",
      "0 AATCCGA motifs detected in file\n",
      "\n",
      "Complement found from 8560 to 8567\n",
      "Seq is: CCACAAT\n",
      "1 CCACAAT motifs detected in file\n",
      "\n",
      "Complement found from 3213 to 3220\n",
      "Complement found from 11306 to 11313\n",
      "Seq is: CCCAAGA\n",
      "2 CCCAAGA motifs detected in file\n",
      "\n",
      "Complement found from 282 to 289\n",
      "Complement found from 1120 to 1127\n",
      "Complement found from 7725 to 7732\n",
      "Complement found from 10418 to 10425\n",
      "Complement found from 12414 to 12421\n",
      "Complement found from 15865 to 15872\n",
      "Seq is: AACAAAA\n",
      "6 AACAAAA motifs detected in file\n",
      "\n",
      "Complement found from 351 to 358\n",
      "Complement found from 555 to 562\n",
      "Complement found from 14749 to 14756\n",
      "Seq is: ACCCCAA\n",
      "3 ACCCCAA motifs detected in file\n",
      "\n",
      "Complement found from 7771 to 7778\n",
      "Seq is: ACCGTCT\n",
      "1 ACCGTCT motifs detected in file\n",
      "\n",
      "Complement found from 4067 to 4074\n",
      "Complement found from 15031 to 15038\n",
      "Complement found from 15414 to 15421\n",
      "Complement found from 15576 to 15583\n",
      "Seq is: CTACACA\n",
      "4 CTACACA motifs detected in file\n",
      "\n",
      "Complement found from 5274 to 5281\n",
      "Seq is: AATTCAC\n",
      "1 AATTCAC motifs detected in file\n",
      "\n",
      "Seq is: GACACCG\n",
      "0 GACACCG motifs detected in file\n",
      "\n",
      "Seq is: GTCTTCG\n",
      "0 GTCTTCG motifs detected in file\n",
      "\n",
      "Complement found from 304 to 311\n",
      "Complement found from 955 to 962\n",
      "Complement found from 3566 to 3573\n",
      "Complement found from 4248 to 4255\n",
      "Complement found from 8271 to 8278\n",
      "Complement found from 8280 to 8287\n",
      "Complement found from 8604 to 8611\n",
      "Complement found from 10947 to 10954\n",
      "Complement found from 13781 to 13788\n",
      "Complement found from 16183 to 16190\n",
      "Complement found from 16375 to 16382\n",
      "Seq is: CCCCCTC\n",
      "11 CCCCCTC motifs detected in file\n",
      "\n",
      "Complement found from 432 to 439\n",
      "Complement found from 567 to 574\n",
      "Complement found from 7396 to 7403\n",
      "Complement found from 7465 to 7472\n",
      "Complement found from 9531 to 9538\n",
      "Complement found from 11866 to 11873\n",
      "Complement found from 12384 to 12391\n",
      "Seq is: CCCCCCA\n",
      "7 CCCCCCA motifs detected in file\n",
      "\n",
      "Complement found from 8807 to 8814\n",
      "Complement found from 10839 to 10846\n",
      "Complement found from 12356 to 12363\n",
      "Complement found from 14331 to 14338\n",
      "Seq is: AACCACC\n",
      "4 AACCACC motifs detected in file\n",
      "\n",
      "Seq is: TTTAAAA\n",
      "0 TTTAAAA motifs detected in file\n",
      "\n",
      "Complement found from 2967 to 2974\n",
      "Complement found from 4697 to 4704\n",
      "Complement found from 5285 to 5292\n",
      "Complement found from 12488 to 12495\n",
      "Seq is: AACAATA\n",
      "4 AACAATA motifs detected in file\n",
      "\n",
      "Seq is: CTACAGA\n",
      "0 CTACAGA motifs detected in file\n",
      "\n",
      "Complement found from 8921 to 8928\n",
      "Complement found from 10539 to 10546\n",
      "Complement found from 13300 to 13307\n",
      "Seq is: CACACCT\n",
      "3 CACACCT motifs detected in file\n",
      "\n",
      "Complement found from 6944 to 6951\n",
      "Complement found from 8134 to 8141\n",
      "Complement found from 11683 to 11690\n",
      "Seq is: TTCACCG\n",
      "3 TTCACCG motifs detected in file\n",
      "\n",
      "Seq is: ACACGTC\n",
      "0 ACACGTC motifs detected in file\n",
      "\n",
      "Seq is: TGTAAGT\n",
      "0 TGTAAGT motifs detected in file\n",
      "\n",
      "Complement found from 238 to 245\n",
      "Complement found from 13385 to 13392\n",
      "Complement found from 14570 to 14577\n",
      "Seq is: TAACAAT\n",
      "3 TAACAAT motifs detected in file\n",
      "\n",
      "Complement found from 233 to 240\n",
      "Complement found from 4300 to 4307\n",
      "Complement found from 5035 to 5042\n",
      "Complement found from 10573 to 10580\n",
      "Complement found from 14544 to 14551\n",
      "Seq is: AATAATA\n",
      "5 AATAATA motifs detected in file\n",
      "\n",
      "Seq is: CAGGATG\n",
      "0 CAGGATG motifs detected in file\n",
      "\n",
      "Seq is: TTCGTAA\n",
      "0 TTCGTAA motifs detected in file\n",
      "\n",
      "Complement found from 207 to 214\n",
      "Complement found from 5636 to 5643\n",
      "Complement found from 11074 to 11081\n",
      "Complement found from 14782 to 14789\n",
      "Seq is: TTAATTA\n",
      "4 TTAATTA motifs detected in file\n",
      "\n",
      "Seq is: ATTGTGT\n",
      "0 ATTGTGT motifs detected in file\n",
      "\n",
      "Complement found from 16413 to 16420\n",
      "Seq is: GAAATCA\n",
      "1 GAAATCA motifs detected in file\n",
      "\n",
      "Complement found from 12070 to 12077\n",
      "Seq is: TTCATAC\n",
      "1 TTCATAC motifs detected in file\n",
      "\n",
      "Seq is: AAGCGGA\n",
      "0 AAGCGGA motifs detected in file\n",
      "\n",
      "Complement found from 10591 to 10598\n",
      "Seq is: CATTATA\n",
      "1 CATTATA motifs detected in file\n",
      "\n",
      "Seq is: ACTTGCA\n",
      "0 ACTTGCA motifs detected in file\n",
      "\n",
      "Seq is: TCCACGC\n",
      "0 TCCACGC motifs detected in file\n",
      "\n",
      "Complement found from 154 to 161\n",
      "Complement found from 3613 to 3620\n",
      "Complement found from 5086 to 5093\n",
      "Complement found from 9459 to 9466\n",
      "Complement found from 13351 to 13358\n",
      "Seq is: TATTTAT\n",
      "5 TATTTAT motifs detected in file\n",
      "\n",
      "Complement found from 5095 to 5102\n",
      "Complement found from 6640 to 6647\n",
      "Complement found from 9125 to 9132\n",
      "Complement found from 12902 to 12909\n",
      "Seq is: TATCCTA\n",
      "4 TATCCTA motifs detected in file\n",
      "\n",
      "Seq is: CTCCGTC\n",
      "0 CTCCGTC motifs detected in file\n",
      "\n",
      "Complement found from 1449 to 1456\n",
      "Complement found from 2035 to 2042\n",
      "Complement found from 5686 to 5693\n",
      "Complement found from 12725 to 12732\n",
      "Seq is: CTTAGTT\n",
      "4 CTTAGTT motifs detected in file\n",
      "\n",
      "Complement found from 126 to 133\n",
      "Seq is: TCTGTCT\n",
      "1 TCTGTCT motifs detected in file\n",
      "\n",
      "Complement found from 4045 to 4052\n",
      "Seq is: ATGACGC\n",
      "1 ATGACGC motifs detected in file\n",
      "\n",
      "Complement found from 16488 to 16495\n",
      "Seq is: TGTATCC\n",
      "1 TGTATCC motifs detected in file\n",
      "\n",
      "Complement found from 2639 to 2646\n",
      "Seq is: CACGAGG\n",
      "1 CACGAGG motifs detected in file\n",
      "\n",
      "Seq is: CCGAGGT\n",
      "0 CCGAGGT motifs detected in file\n",
      "\n",
      "Seq is: CGCAGAG\n",
      "0 CGCAGAG motifs detected in file\n",
      "\n",
      "Seq is: CGTTACG\n",
      "0 CGTTACG motifs detected in file\n",
      "\n",
      "Seq is: ATAGCGC\n",
      "0 ATAGCGC motifs detected in file\n",
      "\n",
      "Seq is: ACGTATG\n",
      "0 ACGTATG motifs detected in file\n",
      "\n",
      "Seq is: GGGGGTC\n",
      "0 GGGGGTC motifs detected in file\n",
      "\n",
      "Seq is: TGCTTTT\n",
      "0 TGCTTTT motifs detected in file\n",
      "\n",
      "Seq is: ATGGTTT\n",
      "0 ATGGTTT motifs detected in file\n",
      "\n",
      "Seq is: ACGTACC\n",
      "0 ACGTACC motifs detected in file\n",
      "\n",
      "Seq is: TCTCGAG\n",
      "0 TCTCGAG motifs detected in file\n",
      "\n",
      "Seq is: GGCACTC\n",
      "0 GGCACTC motifs detected in file\n",
      "\n",
      "Complement found from 2187 to 2194\n",
      "Seq is: ACCAATT\n",
      "1 ACCAATT motifs detected in file\n",
      "\n",
      "Seq is: ATCCCAC\n",
      "0 ATCCCAC motifs detected in file\n",
      "\n",
      "Seq is: TATCTGG\n",
      "0 TATCTGG motifs detected in file\n",
      "\n",
      "Complement found from 2121 to 2128\n",
      "Complement found from 3870 to 3877\n",
      "Seq is: ACACTAG\n",
      "2 ACACTAG motifs detected in file\n",
      "\n",
      "Seq is: GTAGCACT\n",
      "0 GTAGCACT motifs detected in file\n",
      "\n",
      "Seq is: ACAGAATA\n",
      "0 ACAGAATA motifs detected in file\n",
      "\n",
      "Complement found from 8223 to 8231\n",
      "Complement found from 12380 to 12388\n",
      "Seq is: AATTCCCC\n",
      "2 AATTCCCC motifs detected in file\n",
      "\n",
      "Seq is: TTGCACAC\n",
      "0 TTGCACAC motifs detected in file\n",
      "\n",
      "Complement found from 15389 to 15397\n",
      "Seq is: CCGATAAA\n",
      "1 CCGATAAA motifs detected in file\n",
      "\n",
      "Seq is: TCCGAAAT\n",
      "0 TCCGAAAT motifs detected in file\n",
      "\n",
      "Seq is: ACTGGGAC\n",
      "0 ACTGGGAC motifs detected in file\n",
      "\n",
      "Complement found from 12879 to 12887\n",
      "Seq is: TTCATCCT\n",
      "1 TTCATCCT motifs detected in file\n",
      "\n",
      "Seq is: TGGTCTAC\n",
      "0 TGGTCTAC motifs detected in file\n",
      "\n",
      "Seq is: AGCCTATG\n",
      "0 AGCCTATG motifs detected in file\n",
      "\n",
      "Seq is: TCAAGTGA\n",
      "0 TCAAGTGA motifs detected in file\n",
      "\n",
      "Seq is: AATCGATG\n",
      "0 AATCGATG motifs detected in file\n",
      "\n",
      "Seq is: GGGGTTCA\n",
      "0 GGGGTTCA motifs detected in file\n",
      "\n",
      "Seq is: CAATACCC\n",
      "0 CAATACCC motifs detected in file\n",
      "\n",
      "Seq is: GGGCCTCG\n",
      "0 GGGCCTCG motifs detected in file\n",
      "\n",
      "Complement found from 5210 to 5218\n",
      "Seq is: CTCCTCTC\n",
      "1 CTCCTCTC motifs detected in file\n",
      "\n",
      "Seq is: ATCGTGAG\n",
      "0 ATCGTGAG motifs detected in file\n",
      "\n",
      "Seq is: AACACGCC\n",
      "0 AACACGCC motifs detected in file\n",
      "\n",
      "Seq is: CTATAACT\n",
      "0 CTATAACT motifs detected in file\n",
      "\n",
      "Seq is: AAAGTGCC\n",
      "0 AAAGTGCC motifs detected in file\n",
      "\n",
      "Complement found from 4183 to 4191\n",
      "Complement found from 6642 to 6650\n",
      "Seq is: TCCTACCA\n",
      "2 TCCTACCA motifs detected in file\n",
      "\n",
      "Seq is: CCAGTTCC\n",
      "0 CCAGTTCC motifs detected in file\n",
      "\n",
      "Seq is: CTGGGGAT\n",
      "0 CTGGGGAT motifs detected in file\n",
      "\n",
      "Seq is: AGACTCCC\n",
      "0 AGACTCCC motifs detected in file\n",
      "\n",
      "Seq is: CCCAGTAG\n",
      "0 CCCAGTAG motifs detected in file\n",
      "\n",
      "Seq is: GTACCCCT\n",
      "0 GTACCCCT motifs detected in file\n",
      "\n",
      "Seq is: GCTCTTCC\n",
      "0 GCTCTTCC motifs detected in file\n",
      "\n",
      "Seq is: CTAAACTG\n",
      "0 CTAAACTG motifs detected in file\n",
      "\n",
      "Complement found from 16337 to 16345\n",
      "Seq is: ACATTACA\n",
      "1 ACATTACA motifs detected in file\n",
      "\n",
      "Seq is: CGATACAT\n",
      "0 CGATACAT motifs detected in file\n",
      "\n",
      "Complement found from 16318 to 16326\n",
      "Seq is: GCCATTTA\n",
      "1 GCCATTTA motifs detected in file\n",
      "\n",
      "Seq is: CCGAAATA\n",
      "0 CCGAAATA motifs detected in file\n",
      "\n",
      "Seq is: CATGATAC\n",
      "0 CATGATAC motifs detected in file\n",
      "\n",
      "Seq is: ATGACAAT\n",
      "0 ATGACAAT motifs detected in file\n",
      "\n",
      "Complement found from 6311 to 6319\n",
      "Complement found from 15261 to 15269\n",
      "Seq is: TCCCACCC\n",
      "2 TCCCACCC motifs detected in file\n",
      "\n",
      "Complement found from 9889 to 9897\n",
      "Complement found from 11665 to 11673\n",
      "Complement found from 13152 to 13160\n",
      "Complement found from 15672 to 15680\n",
      "Seq is: ATCCAAAC\n",
      "4 ATCCAAAC motifs detected in file\n",
      "\n",
      "Seq is: AACCATAG\n",
      "0 AACCATAG motifs detected in file\n",
      "\n",
      "Complement found from 950 to 958\n",
      "Seq is: GATCACCC\n",
      "1 GATCACCC motifs detected in file\n",
      "\n",
      "Complement found from 5380 to 5388\n",
      "Seq is: ACTCCCCA\n",
      "1 ACTCCCCA motifs detected in file\n",
      "\n",
      "Complement found from 1903 to 1911\n",
      "Complement found from 9670 to 9678\n",
      "Seq is: CCGAAACC\n",
      "2 CCGAAACC motifs detected in file\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Seq is: TCAACGTC\n",
      "0 TCAACGTC motifs detected in file\n",
      "\n",
      "Seq is: AACTACAC\n",
      "0 AACTACAC motifs detected in file\n",
      "\n",
      "Complement found from 3755 to 3763\n",
      "Complement found from 11313 to 11321\n",
      "Seq is: ACTATCAA\n",
      "2 ACTATCAA motifs detected in file\n",
      "\n",
      "Seq is: CTCCCAAC\n",
      "0 CTCCCAAC motifs detected in file\n",
      "\n",
      "Seq is: TAACGACA\n",
      "0 TAACGACA motifs detected in file\n",
      "\n",
      "Complement found from 8527 to 8535\n",
      "Seq is: TGAACGAA\n",
      "1 TGAACGAA motifs detected in file\n",
      "\n",
      "Complement found from 8031 to 8039\n",
      "Seq is: CATTCGTA\n",
      "1 CATTCGTA motifs detected in file\n",
      "\n",
      "Complement found from 305 to 313\n",
      "Complement found from 455 to 463\n",
      "Complement found from 956 to 964\n",
      "Complement found from 3567 to 3575\n",
      "Complement found from 15536 to 15544\n",
      "Complement found from 16184 to 16192\n",
      "Seq is: CCCCTCCC\n",
      "6 CCCCTCCC motifs detected in file\n",
      "\n",
      "Seq is: CCAAAACT\n",
      "0 CCAAAACT motifs detected in file\n",
      "\n",
      "Seq is: ACACCTAA\n",
      "0 ACACCTAA motifs detected in file\n",
      "\n",
      "Complement found from 353 to 361\n",
      "Complement found from 12302 to 12310\n",
      "Complement found from 14071 to 14079\n",
      "Seq is: CCCAAAAA\n",
      "3 CCCAAAAA motifs detected in file\n",
      "\n",
      "Seq is: TACATGAT\n",
      "0 TACATGAT motifs detected in file\n",
      "\n",
      "Seq is: GTCCACCA\n",
      "0 GTCCACCA motifs detected in file\n",
      "\n",
      "Seq is: GTTCATAA\n",
      "0 GTTCATAA motifs detected in file\n",
      "\n",
      "Seq is: ATACCATG\n",
      "0 ATACCATG motifs detected in file\n",
      "\n",
      "Seq is: GCATGTTA\n",
      "0 GCATGTTA motifs detected in file\n",
      "\n",
      "Seq is: TAAGTACC\n",
      "0 TAAGTACC motifs detected in file\n",
      "\n",
      "Seq is: ACCGACCG\n",
      "0 ACCGACCG motifs detected in file\n",
      "\n",
      "Seq is: TCATTACA\n",
      "0 TCATTACA motifs detected in file\n",
      "\n",
      "Seq is: TGCTTTAT\n",
      "0 TGCTTTAT motifs detected in file\n",
      "\n",
      "Seq is: GTATCGCC\n",
      "0 GTATCGCC motifs detected in file\n",
      "\n",
      "Complement found from 10297 to 10305\n",
      "Seq is: AACAACTA\n",
      "1 AACAACTA motifs detected in file\n",
      "\n",
      "Complement found from 14114 to 14122\n",
      "Seq is: CCCACTCA\n",
      "1 CCCACTCA motifs detected in file\n",
      "\n",
      "Seq is: GTTATGAA\n",
      "0 GTTATGAA motifs detected in file\n",
      "\n",
      "Complement found from 5303 to 5311\n",
      "Complement found from 8392 to 8400\n",
      "Seq is: CCCACCAT\n",
      "2 CCCACCAT motifs detected in file\n",
      "\n",
      "Seq is: GGGTTTAG\n",
      "0 GGGTTTAG motifs detected in file\n",
      "\n",
      "Seq is: ACGAAGGG\n",
      "0 ACGAAGGG motifs detected in file\n",
      "\n",
      "Seq is: GTACTTTC\n",
      "0 GTACTTTC motifs detected in file\n",
      "\n",
      "Seq is: TTGTCTCT\n",
      "0 TTGTCTCT motifs detected in file\n",
      "\n",
      "Seq is: TATCAAAT\n",
      "0 TATCAAAT motifs detected in file\n",
      "\n",
      "Seq is: TTAATCTT\n",
      "0 TTAATCTT motifs detected in file\n",
      "\n",
      "Complement found from 4339 to 4347\n",
      "Seq is: AGAATCGA\n",
      "1 AGAATCGA motifs detected in file\n",
      "\n",
      "Seq is: AACCCACG\n",
      "0 AACCCACG motifs detected in file\n",
      "\n",
      "Seq is: ATTACCAC\n",
      "0 ATTACCAC motifs detected in file\n",
      "\n",
      "Seq is: CTCAATTT\n",
      "0 CTCAATTT motifs detected in file\n",
      "\n",
      "Seq is: CTGAAAAA\n",
      "0 CTGAAAAA motifs detected in file\n",
      "\n",
      "Seq is: GAGACTAA\n",
      "0 GAGACTAA motifs detected in file\n",
      "\n",
      "Seq is: ACAGGAAC\n",
      "0 ACAGGAAC motifs detected in file\n",
      "\n",
      "Complement found from 15938 to 15946\n",
      "Seq is: CTTTTTCC\n",
      "1 CTTTTTCC motifs detected in file\n",
      "\n",
      "Seq is: AAAAGTAG\n",
      "0 AAAAGTAG motifs detected in file\n",
      "\n",
      "Complement found from 15150 to 15158\n",
      "Seq is: AGGCCAAA\n",
      "1 AGGCCAAA motifs detected in file\n",
      "\n",
      "Seq is: TGTTCTGA\n",
      "0 TGTTCTGA motifs detected in file\n",
      "\n",
      "Seq is: CCACATAA\n",
      "0 CCACATAA motifs detected in file\n",
      "\n",
      "Complement found from 11923 to 11931\n",
      "Complement found from 12688 to 12696\n",
      "Seq is: TCAAATAT\n",
      "2 TCAAATAT motifs detected in file\n",
      "\n",
      "Seq is: GATGTTCC\n",
      "0 GATGTTCC motifs detected in file\n",
      "\n",
      "Seq is: TGTCCGGG\n",
      "0 TGTCCGGG motifs detected in file\n",
      "\n",
      "Complement found from 12658 to 12666\n",
      "Seq is: TAAACTCA\n",
      "1 TAAACTCA motifs detected in file\n",
      "\n",
      "Complement found from 14024 to 14032\n",
      "Seq is: TAAAACAA\n",
      "1 TAAAACAA motifs detected in file\n",
      "\n",
      "Seq is: AAGTTAAT\n",
      "0 AAGTTAAT motifs detected in file\n",
      "\n",
      "Complement found from 8606 to 8614\n",
      "Seq is: CCCTCTAT\n",
      "1 CCCTCTAT motifs detected in file\n",
      "\n",
      "Seq is: CAACCATA\n",
      "0 CAACCATA motifs detected in file\n",
      "\n",
      "Complement found from 15829 to 15837\n",
      "Seq is: ATCCTAAT\n",
      "1 ATCCTAAT motifs detected in file\n",
      "\n",
      "Complement found from 2760 to 2768\n",
      "Complement found from 10938 to 10946\n",
      "Complement found from 12411 to 12419\n",
      "Complement found from 15598 to 15606\n",
      "Seq is: CCTAACAA\n",
      "4 CCTAACAA motifs detected in file\n",
      "\n",
      "Complement found from 15289 to 15297\n",
      "Seq is: CACTTCAT\n",
      "1 CACTTCAT motifs detected in file\n",
      "\n",
      "Seq is: ATCATGCC\n",
      "0 ATCATGCC motifs detected in file\n",
      "\n",
      "Seq is: TACGATGA\n",
      "0 TACGATGA motifs detected in file\n",
      "\n",
      "Seq is: ACAGGTTA\n",
      "0 ACAGGTTA motifs detected in file\n",
      "\n",
      "Seq is: CTACCATT\n",
      "0 CTACCATT motifs detected in file\n",
      "\n",
      "Seq is: TTCCCATC\n",
      "0 TTCCCATC motifs detected in file\n",
      "\n",
      "Seq is: GAATGACC\n",
      "0 GAATGACC motifs detected in file\n",
      "\n",
      "Seq is: AACAGGAG\n",
      "0 AACAGGAG motifs detected in file\n",
      "\n",
      "Seq is: GCTAAGTC\n",
      "0 GCTAAGTC motifs detected in file\n",
      "\n",
      "Complement found from 4941 to 4949\n",
      "Seq is: CAATCTTA\n",
      "1 CAATCTTA motifs detected in file\n",
      "\n",
      "Seq is: CTCCTCCA\n",
      "0 CTCCTCCA motifs detected in file\n",
      "\n",
      "Seq is: GACGCCGA\n",
      "0 GACGCCGA motifs detected in file\n",
      "\n",
      "Seq is: TCCTCAGT\n",
      "0 TCCTCAGT motifs detected in file\n",
      "\n",
      "Complement found from 6812 to 6820\n",
      "Complement found from 9876 to 9884\n",
      "Seq is: TATTTCAC\n",
      "2 TATTTCAC motifs detected in file\n",
      "\n",
      "Seq is: TAACCGAA\n",
      "0 TAACCGAA motifs detected in file\n",
      "\n",
      "Seq is: TCACCCGC\n",
      "0 TCACCCGC motifs detected in file\n",
      "\n",
      "Seq is: TTTATAAT\n",
      "0 TTTATAAT motifs detected in file\n",
      "\n",
      "Seq is: ACGAAACA\n",
      "0 ACGAAACA motifs detected in file\n",
      "\n",
      "Complement found from 1990 to 1998\n",
      "Complement found from 16280 to 16288\n",
      "Seq is: ACAAACCT\n",
      "2 ACAAACCT motifs detected in file\n",
      "\n",
      "Seq is: ATATACCT\n",
      "0 ATATACCT motifs detected in file\n",
      "\n",
      "Complement found from 9527 to 9535\n",
      "Complement found from 10974 to 10982\n",
      "Complement found from 11531 to 11539\n",
      "Seq is: CCTACCCC\n",
      "3 CCTACCCC motifs detected in file\n",
      "\n",
      "Seq is: TAATAACG\n",
      "0 TAATAACG motifs detected in file\n",
      "\n",
      "Seq is: ATCCTACT\n",
      "0 ATCCTACT motifs detected in file\n",
      "\n",
      "Complement found from 5476 to 5484\n",
      "Seq is: CCTACCTA\n",
      "1 CCTACCTA motifs detected in file\n",
      "\n",
      "Seq is: TCATTATC\n",
      "0 TCATTATC motifs detected in file\n",
      "\n",
      "Seq is: CCGTTCCT\n",
      "0 CCGTTCCT motifs detected in file\n",
      "\n",
      "Seq is: GCGGAGGA\n",
      "0 GCGGAGGA motifs detected in file\n",
      "\n",
      "Complement found from 15358 to 15366\n",
      "Seq is: TCAAACAA\n",
      "1 TCAAACAA motifs detected in file\n",
      "\n",
      "Seq is: TCCCTGCC\n",
      "0 TCCCTGCC motifs detected in file\n",
      "\n",
      "Seq is: TAGCCTCT\n",
      "0 TAGCCTCT motifs detected in file\n",
      "\n",
      "Complement found from 441 to 449\n",
      "Seq is: TAACACAT\n",
      "1 TAACACAT motifs detected in file\n",
      "\n",
      "Seq is: CCGCTTAT\n",
      "0 CCGCTTAT motifs detected in file\n",
      "\n",
      "Seq is: CCTTTATA\n",
      "0 CCTTTATA motifs detected in file\n",
      "\n",
      "Seq is: GTAAGCCC\n",
      "0 GTAAGCCC motifs detected in file\n",
      "\n",
      "Seq is: GAACTACA\n",
      "0 GAACTACA motifs detected in file\n",
      "\n",
      "Complement found from 305 to 313\n",
      "Complement found from 455 to 463\n",
      "Complement found from 956 to 964\n",
      "Complement found from 3567 to 3575\n",
      "Complement found from 15536 to 15544\n",
      "Complement found from 16184 to 16192\n",
      "Seq is: CCCCTCCC\n",
      "6 CCCCTCCC motifs detected in file\n",
      "\n",
      "Seq is: CACAAATT\n",
      "0 CACAAATT motifs detected in file\n",
      "\n",
      "Complement found from 10916 to 10924\n",
      "Seq is: CCCCAACC\n",
      "1 CCCCAACC motifs detected in file\n",
      "\n",
      "Seq is: GATCCCAT\n",
      "0 GATCCCAT motifs detected in file\n",
      "\n",
      "Complement found from 2369 to 2377\n",
      "Seq is: ATTAACAG\n",
      "1 ATTAACAG motifs detected in file\n",
      "\n",
      "Seq is: ACCCAGCG\n",
      "0 ACCCAGCG motifs detected in file\n",
      "\n",
      "Complement found from 14258 to 14266\n",
      "Complement found from 14868 to 14876\n",
      "Seq is: GATCCTCC\n",
      "2 GATCCTCC motifs detected in file\n",
      "\n",
      "Seq is: AGACCACT\n",
      "0 AGACCACT motifs detected in file\n",
      "\n",
      "Complement found from 3798 to 3806\n",
      "Complement found from 7642 to 7650\n",
      "Seq is: CTTATCAC\n",
      "2 CTTATCAC motifs detected in file\n",
      "\n",
      "Seq is: AATTACAG\n",
      "0 AATTACAG motifs detected in file\n",
      "\n",
      "Seq is: TAATTCCT\n",
      "0 TAATTCCT motifs detected in file\n",
      "\n",
      "Complement found from 15025 to 15033\n",
      "Complement found from 15445 to 15453\n",
      "Seq is: CTCTTCCT\n",
      "2 CTCTTCCT motifs detected in file\n",
      "\n",
      "Seq is: TCTCTTCA\n",
      "0 TCTCTTCA motifs detected in file\n",
      "\n",
      "Complement found from 14842 to 14850\n",
      "Seq is: TTCGGCTC\n",
      "1 TTCGGCTC motifs detected in file\n",
      "\n",
      "Seq is: CCGCAGAA\n",
      "0 CCGCAGAA motifs detected in file\n",
      "\n",
      "Complement found from 439 to 447\n",
      "Complement found from 5185 to 5193\n",
      "Seq is: ACTAACAC\n",
      "2 ACTAACAC motifs detected in file\n",
      "\n",
      "Complement found from 14482 to 14490\n",
      "Seq is: ATCATTCC\n",
      "1 ATCATTCC motifs detected in file\n",
      "\n",
      "Complement found from 15399 to 15407\n",
      "Seq is: CACCTTCC\n",
      "1 CACCTTCC motifs detected in file\n",
      "\n",
      "Seq is: ACTAAAAT\n",
      "0 ACTAAAAT motifs detected in file\n",
      "\n",
      "Complement found from 10096 to 10104\n",
      "Seq is: AGCCTTAC\n",
      "1 AGCCTTAC motifs detected in file\n",
      "\n",
      "Seq is: CCTCCACT\n",
      "0 CCTCCACT motifs detected in file\n",
      "\n",
      "Seq is: AAGGATCC\n",
      "0 AAGGATCC motifs detected in file\n",
      "\n",
      "Complement found from 8659 to 8667\n",
      "Complement found from 10776 to 10784\n",
      "Seq is: CCCAACAA\n",
      "2 CCCAACAA motifs detected in file\n",
      "\n",
      "Seq is: ACTAGGGC\n",
      "0 ACTAGGGC motifs detected in file\n",
      "\n",
      "Seq is: AAAGCACG\n",
      "0 AAAGCACG motifs detected in file\n",
      "\n",
      "Complement found from 6636 to 6644\n",
      "Seq is: TTCTTATC\n",
      "1 TTCTTATC motifs detected in file\n",
      "\n",
      "Complement found from 1692 to 1700\n",
      "Complement found from 5362 to 5370\n",
      "Complement found from 14048 to 14056\n",
      "Complement found from 15328 to 15336\n",
      "Seq is: CTCCACCT\n",
      "4 CTCCACCT motifs detected in file\n",
      "\n",
      "Seq is: CACAACGA\n",
      "0 CACAACGA motifs detected in file\n",
      "\n",
      "Seq is: TCCCGACG\n",
      "0 TCCCGACG motifs detected in file\n",
      "\n",
      "Seq is: TTATTACT\n",
      "0 TTATTACT motifs detected in file\n",
      "\n",
      "Seq is: TCCCGTTC\n",
      "0 TCCCGTTC motifs detected in file\n",
      "\n",
      "Complement found from 3149 to 3157\n",
      "Complement found from 15817 to 15825\n",
      "Seq is: TACTTCAC\n",
      "2 TACTTCAC motifs detected in file\n",
      "\n",
      "Seq is: TTTCCATT\n",
      "0 TTTCCATT motifs detected in file\n",
      "\n",
      "Complement found from 5001 to 5009\n",
      "Seq is: TCTTAGCA\n",
      "1 TCTTAGCA motifs detected in file\n",
      "\n",
      "Complement found from 462 to 470\n",
      "Seq is: CACTCCCA\n",
      "1 CACTCCCA motifs detected in file\n",
      "\n",
      "Complement found from 11513 to 11521\n",
      "Complement found from 13585 to 13593\n",
      "Seq is: CCCTGACA\n",
      "2 CCCTGACA motifs detected in file\n",
      "\n",
      "Seq is: GATGACTC\n",
      "0 GATGACTC motifs detected in file\n",
      "\n",
      "Complement found from 6125 to 6133\n",
      "Complement found from 15757 to 15765\n",
      "Seq is: ATCGGAGG\n",
      "2 ATCGGAGG motifs detected in file\n",
      "\n",
      "Seq is: AGTCTAAG\n",
      "0 AGTCTAAG motifs detected in file\n",
      "\n",
      "Complement found from 1653 to 1661\n",
      "Complement found from 2902 to 2910\n",
      "Seq is: TAACTTGA\n",
      "2 TAACTTGA motifs detected in file\n",
      "\n",
      "Seq is: TCCAGACA\n",
      "0 TCCAGACA motifs detected in file\n",
      "\n",
      "Seq is: GGGTTACA\n",
      "0 GGGTTACA motifs detected in file\n",
      "\n",
      "Seq is: TACCCTAC\n",
      "0 TACCCTAC motifs detected in file\n",
      "\n",
      "Seq is: CGCCTATC\n",
      "0 CGCCTATC motifs detected in file\n",
      "\n",
      "Seq is: ATTCAAAC\n",
      "0 ATTCAAAC motifs detected in file\n",
      "\n",
      "Seq is: ATTAATGA\n",
      "0 ATTAATGA motifs detected in file\n",
      "\n",
      "Seq is: CACCGGGG\n",
      "0 CACCGGGG motifs detected in file\n",
      "\n",
      "Seq is: AGTCTTAC\n",
      "0 AGTCTTAC motifs detected in file\n",
      "\n",
      "Complement found from 8822 to 8830\n",
      "Seq is: TATAAACC\n",
      "1 TATAAACC motifs detected in file\n",
      "\n",
      "Seq is: GGAGTGCC\n",
      "0 GGAGTGCC motifs detected in file\n",
      "\n",
      "Seq is: CTCCTGTA\n",
      "0 CTCCTGTA motifs detected in file\n",
      "\n",
      "Seq is: TCGGATAC\n",
      "0 TCGGATAC motifs detected in file\n",
      "\n",
      "Seq is: TTCCGACA\n",
      "0 TTCCGACA motifs detected in file\n",
      "\n",
      "Seq is: ACGATATC\n",
      "0 ACGATATC motifs detected in file\n",
      "\n",
      "Seq is: AACGTTCG\n",
      "0 AACGTTCG motifs detected in file\n",
      "\n",
      "Complement found from 12092 to 12100\n",
      "Complement found from 12982 to 12990\n",
      "Complement found from 13987 to 13995\n",
      "Seq is: TCCTCCTA\n",
      "3 TCCTCCTA motifs detected in file\n",
      "\n",
      "Seq is: TTACGGCT\n",
      "0 TTACGGCT motifs detected in file\n",
      "\n",
      "Seq is: ACAAAGTC\n",
      "0 ACAAAGTC motifs detected in file\n",
      "\n",
      "Seq is: CAAAGACT\n",
      "0 CAAAGACT motifs detected in file\n",
      "\n",
      "Seq is: CATCTCTT\n",
      "0 CATCTCTT motifs detected in file\n",
      "\n",
      "Complement found from 9336 to 9344\n",
      "Complement found from 12973 to 12981\n",
      "Seq is: TACTAGGC\n",
      "2 TACTAGGC motifs detected in file\n",
      "\n",
      "Complement found from 12780 to 12788\n",
      "Seq is: ATTATATC\n",
      "1 ATTATATC motifs detected in file\n",
      "\n",
      "Seq is: CGGAGCGG\n",
      "0 CGGAGCGG motifs detected in file\n",
      "\n",
      "Seq is: GCTACACA\n",
      "0 GCTACACA motifs detected in file\n",
      "\n",
      "Complement found from 15449 to 15457\n",
      "Seq is: TCCTTCTC\n",
      "1 TCCTTCTC motifs detected in file\n",
      "\n",
      "Seq is: CGTCTATT\n",
      "0 CGTCTATT motifs detected in file\n",
      "\n",
      "Seq is: TCTTATAA\n",
      "0 TCTTATAA motifs detected in file\n",
      "\n",
      "Seq is: CTCCGCGG\n",
      "0 CTCCGCGG motifs detected in file\n",
      "\n",
      "Seq is: TAACCGCA\n",
      "0 TAACCGCA motifs detected in file\n",
      "\n",
      "Seq is: CTTCCATC\n",
      "0 CTTCCATC motifs detected in file\n",
      "\n",
      "Complement found from 9342 to 9350\n",
      "Seq is: GCCTACTA\n",
      "1 GCCTACTA motifs detected in file\n",
      "\n",
      "Seq is: AGTCGGTA\n",
      "0 AGTCGGTA motifs detected in file\n",
      "\n",
      "Seq is: TTAAATGC\n",
      "0 TTAAATGC motifs detected in file\n",
      "\n",
      "Seq is: AGAGCTCA\n",
      "0 AGAGCTCA motifs detected in file\n",
      "\n",
      "Complement found from 7108 to 7116\n",
      "Complement found from 8926 to 8934\n",
      "Complement found from 11211 to 11219\n",
      "Seq is: CTACACCC\n",
      "3 CTACACCC motifs detected in file\n",
      "\n",
      "Seq is: GCTAACTA\n",
      "0 GCTAACTA motifs detected in file\n",
      "\n",
      "Seq is: CTTTTCCG\n",
      "0 CTTTTCCG motifs detected in file\n",
      "\n",
      "Seq is: CCAACTCC\n",
      "0 CCAACTCC motifs detected in file\n",
      "\n",
      "Seq is: GCAGACCA\n",
      "0 GCAGACCA motifs detected in file\n",
      "\n",
      "Complement found from 4520 to 4528\n",
      "Complement found from 6982 to 6990\n",
      "Complement found from 8428 to 8436\n",
      "Seq is: CTCATCAC\n",
      "3 CTCATCAC motifs detected in file\n",
      "\n",
      "Seq is: GTACCGAT\n",
      "0 GTACCGAT motifs detected in file\n",
      "\n",
      "Complement found from 3797 to 3805\n",
      "Seq is: CCTTATCA\n",
      "1 CCTTATCA motifs detected in file\n",
      "\n",
      "Seq is: GGACACCA\n",
      "0 GGACACCA motifs detected in file\n",
      "\n",
      "Complement found from 1099 to 1107\n",
      "Complement found from 1671 to 1679\n",
      "Complement found from 10745 to 10753\n",
      "Seq is: CTAAACCT\n",
      "3 CTAAACCT motifs detected in file\n",
      "\n",
      "Complement found from 7692 to 7700\n",
      "Complement found from 7800 to 7808\n",
      "Seq is: CCTAGTCC\n",
      "2 CCTAGTCC motifs detected in file\n",
      "\n",
      "Seq is: GTCCGCGG\n",
      "0 GTCCGCGG motifs detected in file\n",
      "\n",
      "Complement found from 4646 to 4654\n",
      "Complement found from 9845 to 9853\n",
      "Complement found from 13488 to 13496\n",
      "Seq is: TTCCTCAC\n",
      "3 TTCCTCAC motifs detected in file\n",
      "\n",
      "Seq is: TCGGCTTC\n",
      "0 TCGGCTTC motifs detected in file\n",
      "\n",
      "Seq is: AAAGTAGT\n",
      "0 AAAGTAGT motifs detected in file\n",
      "\n",
      "Seq is: ACGCCTCT\n",
      "0 ACGCCTCT motifs detected in file\n",
      "\n",
      "Complement found from 10898 to 10906\n",
      "Complement found from 12739 to 12747\n",
      "Seq is: ACAACCTA\n",
      "2 ACAACCTA motifs detected in file\n",
      "\n",
      "Complement found from 7398 to 7406\n",
      "Complement found from 13052 to 13060\n",
      "Complement found from 13646 to 13654\n",
      "Complement found from 13676 to 13684\n",
      "Complement found from 14808 to 14816\n",
      "Seq is: CCCCACCC\n",
      "5 CCCCACCC motifs detected in file\n",
      "\n",
      "Seq is: CTCCAGCT\n",
      "0 CTCCAGCT motifs detected in file\n",
      "\n",
      "Seq is: ACTTACTC\n",
      "0 ACTTACTC motifs detected in file\n",
      "\n",
      "Complement found from 2187 to 2195\n",
      "Seq is: ACCAATTA\n",
      "1 ACCAATTA motifs detected in file\n",
      "\n",
      "Seq is: ATTAAAAT\n",
      "0 ATTAAAAT motifs detected in file\n",
      "\n",
      "Complement found from 13778 to 13786\n",
      "Seq is: AATCCCCC\n",
      "1 AATCCCCC motifs detected in file\n",
      "\n",
      "Seq is: AATCAAAA\n",
      "0 AATCAAAA motifs detected in file\n",
      "\n",
      "Seq is: CGCATAAC\n",
      "0 CGCATAAC motifs detected in file\n",
      "\n",
      "Seq is: CCCAGTAA\n",
      "0 CCCAGTAA motifs detected in file\n",
      "\n",
      "Seq is: CCACAAGA\n",
      "0 CCACAAGA motifs detected in file\n",
      "\n",
      "Complement found from 16234 to 16242\n",
      "Seq is: ACATCAAC\n",
      "1 ACATCAAC motifs detected in file\n",
      "\n",
      "Seq is: TTTATGTT\n",
      "0 TTTATGTT motifs detected in file\n",
      "\n",
      "Seq is: GCTACCAA\n",
      "0 GCTACCAA motifs detected in file\n",
      "\n",
      "Complement found from 1738 to 1746\n",
      "Seq is: AAAGTATA\n",
      "1 AAAGTATA motifs detected in file\n",
      "\n",
      "Seq is: GTAACCAG\n",
      "0 GTAACCAG motifs detected in file\n",
      "\n",
      "Complement found from 14183 to 14191\n",
      "Seq is: CACCAACA\n",
      "1 CACCAACA motifs detected in file\n",
      "\n",
      "Seq is: TCAGGCAC\n",
      "0 TCAGGCAC motifs detected in file\n",
      "\n",
      "Seq is: GCTCTTAT\n",
      "0 GCTCTTAT motifs detected in file\n",
      "\n",
      "Seq is: TACTACAT\n",
      "0 TACTACAT motifs detected in file\n",
      "\n",
      "Seq is: ACGAAACA\n",
      "0 ACGAAACA motifs detected in file\n",
      "\n",
      "Complement found from 14473 to 14481\n",
      "Seq is: AAGACAAC\n",
      "1 AAGACAAC motifs detected in file\n",
      "\n",
      "Complement found from 2255 to 2263\n",
      "Complement found from 4385 to 4393\n",
      "Seq is: TCACACCC\n",
      "2 TCACACCC motifs detected in file\n",
      "\n",
      "Seq is: AAATCATT\n",
      "0 AAATCATT motifs detected in file\n",
      "\n",
      "Complement found from 351 to 359\n",
      "Complement found from 555 to 563\n",
      "Seq is: ACCCCAAA\n",
      "2 ACCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: CACCCCAA\n",
      "0 CACCCCAA motifs detected in file\n",
      "\n",
      "Seq is: AAGAAGAT\n",
      "0 AAGAAGAT motifs detected in file\n",
      "\n",
      "Seq is: TCGGAAGA\n",
      "0 TCGGAAGA motifs detected in file\n",
      "\n",
      "Seq is: GGATAAAT\n",
      "0 GGATAAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCCCCAA\n",
      "0 ACCCCCAA motifs detected in file\n",
      "\n",
      "Complement found from 277 to 285\n",
      "Seq is: ATCATAAC\n",
      "1 ATCATAAC motifs detected in file\n",
      "\n",
      "Complement found from 14570 to 14578\n",
      "Seq is: TAACAATC\n",
      "1 TAACAATC motifs detected in file\n",
      "\n",
      "Complement found from 795 to 803\n",
      "Seq is: GCCACACC\n",
      "1 GCCACACC motifs detected in file\n",
      "\n",
      "Complement found from 16533 to 16541\n",
      "Seq is: AGCCCACA\n",
      "1 AGCCCACA motifs detected in file\n",
      "\n",
      "Seq is: CAATAATA\n",
      "0 CAATAATA motifs detected in file\n",
      "\n",
      "Seq is: AGACTTAA\n",
      "0 AGACTTAA motifs detected in file\n",
      "\n",
      "Complement found from 3582 to 3590\n",
      "Complement found from 3890 to 3898\n",
      "Complement found from 4315 to 4323\n",
      "Complement found from 6417 to 6425\n",
      "Complement found from 11509 to 11517\n",
      "Complement found from 11670 to 11678\n",
      "Complement found from 14767 to 14775\n",
      "Complement found from 15364 to 15372\n",
      "Complement found from 16181 to 16189\n",
      "Seq is: AACCCCCT\n",
      "9 AACCCCCT motifs detected in file\n",
      "\n",
      "Seq is: CCAATATA\n",
      "0 CCAATATA motifs detected in file\n",
      "\n",
      "Seq is: CCCAAATT\n",
      "0 CCCAAATT motifs detected in file\n",
      "\n",
      "Complement found from 13227 to 13235\n",
      "Seq is: ATCAAAAA\n",
      "1 ATCAAAAA motifs detected in file\n",
      "\n",
      "Seq is: AATTAAAT\n",
      "0 AATTAAAT motifs detected in file\n",
      "\n",
      "Complement found from 2071 to 2079\n",
      "Complement found from 9577 to 9585\n",
      "Complement found from 13875 to 13883\n",
      "Seq is: AAATCCCC\n",
      "3 AAATCCCC motifs detected in file\n",
      "\n",
      "Complement found from 1698 to 1706\n",
      "Complement found from 6678 to 6686\n",
      "Complement found from 10099 to 10107\n",
      "Complement found from 15410 to 15418\n",
      "Seq is: CTTACTAC\n",
      "4 CTTACTAC motifs detected in file\n",
      "\n",
      "Complement found from 14649 to 14657\n",
      "Seq is: CAACAGAA\n",
      "1 CAACAGAA motifs detected in file\n",
      "\n",
      "Seq is: ACCTATAT\n",
      "0 ACCTATAT motifs detected in file\n",
      "\n",
      "Seq is: GATGTCGC\n",
      "0 GATGTCGC motifs detected in file\n",
      "\n",
      "Seq is: TACCGATA\n",
      "0 TACCGATA motifs detected in file\n",
      "\n",
      "Complement found from 2250 to 2258\n",
      "Complement found from 3326 to 3334\n",
      "Complement found from 5010 to 5018\n",
      "Complement found from 14441 to 14449\n",
      "Seq is: ACTCCTCA\n",
      "4 ACTCCTCA motifs detected in file\n",
      "\n",
      "Complement found from 13264 to 13272\n",
      "Seq is: TAGGACTC\n",
      "1 TAGGACTC motifs detected in file\n",
      "\n",
      "Seq is: CGTACCCC\n",
      "0 CGTACCCC motifs detected in file\n",
      "\n",
      "Seq is: CAGTCCCC\n",
      "0 CAGTCCCC motifs detected in file\n",
      "\n",
      "Complement found from 996 to 1004\n",
      "Complement found from 5132 to 5140\n",
      "Seq is: AACTCCAG\n",
      "2 AACTCCAG motifs detected in file\n",
      "\n",
      "Complement found from 23 to 31\n",
      "Complement found from 14788 to 14796\n",
      "Seq is: AACCACTC\n",
      "2 AACCACTC motifs detected in file\n",
      "\n",
      "Seq is: ACAAAATC\n",
      "0 ACAAAATC motifs detected in file\n",
      "\n",
      "Complement found from 14749 to 14757\n",
      "Seq is: ACCCCAAT\n",
      "1 ACCCCAAT motifs detected in file\n",
      "\n",
      "Complement found from 13577 to 13585\n",
      "Complement found from 14983 to 14991\n",
      "Seq is: CGCTACCT\n",
      "2 CGCTACCT motifs detected in file\n",
      "\n",
      "Complement found from 500 to 508\n",
      "Complement found from 4392 to 4400\n",
      "Seq is: CCATCCTA\n",
      "2 CCATCCTA motifs detected in file\n",
      "\n",
      "Complement found from 5141 to 5149\n",
      "Complement found from 14692 to 14700\n",
      "Seq is: ACCACGAC\n",
      "2 ACCACGAC motifs detected in file\n",
      "\n",
      "Seq is: ACCCACTT\n",
      "0 ACCCACTT motifs detected in file\n",
      "\n",
      "Seq is: TCTCATAC\n",
      "0 TCTCATAC motifs detected in file\n",
      "\n",
      "Complement found from 1080 to 1088\n",
      "Seq is: TACCCCAC\n",
      "1 TACCCCAC motifs detected in file\n",
      "\n",
      "Complement found from 14183 to 14191\n",
      "Seq is: CACCAACA\n",
      "1 CACCAACA motifs detected in file\n",
      "\n",
      "Seq is: CCATTTGA\n",
      "0 CCATTTGA motifs detected in file\n",
      "\n",
      "Complement found from 6403 to 6411\n",
      "Seq is: AATTATCA\n",
      "1 AATTATCA motifs detected in file\n",
      "\n",
      "Seq is: CATCCTTC\n",
      "0 CATCCTTC motifs detected in file\n",
      "\n",
      "Seq is: GACTTATT\n",
      "0 GACTTATT motifs detected in file\n",
      "\n",
      "Complement found from 16136 to 16144\n",
      "Seq is: AAATACTT\n",
      "1 AAATACTT motifs detected in file\n",
      "\n",
      "Complement found from 5212 to 5220\n",
      "Seq is: CCTCTCCC\n",
      "1 CCTCTCCC motifs detected in file\n",
      "\n",
      "Complement found from 15258 to 15266\n",
      "Seq is: CAGTCCCA\n",
      "1 CAGTCCCA motifs detected in file\n",
      "\n",
      "Complement found from 15702 to 15710\n",
      "Seq is: ACTAAGCC\n",
      "1 ACTAAGCC motifs detected in file\n",
      "\n",
      "Complement found from 15490 to 15498\n",
      "Seq is: CTCCTAGG\n",
      "1 CTCCTAGG motifs detected in file\n",
      "\n",
      "Complement found from 12354 to 12362\n",
      "Seq is: ATAACCAC\n",
      "1 ATAACCAC motifs detected in file\n",
      "\n",
      "Complement found from 6167 to 6175\n",
      "Seq is: GCCCCCGA\n",
      "1 GCCCCCGA motifs detected in file\n",
      "\n",
      "Complement found from 187 to 195\n",
      "Complement found from 11953 to 11961\n",
      "Complement found from 13905 to 13913\n",
      "Seq is: AACATACT\n",
      "3 AACATACT motifs detected in file\n",
      "\n",
      "Seq is: AATACCCG\n",
      "0 AATACCCG motifs detected in file\n",
      "\n",
      "Complement found from 9869 to 9877\n",
      "Seq is: CAACTAAT\n",
      "1 CAACTAAT motifs detected in file\n",
      "\n",
      "Complement found from 14057 to 14065\n",
      "Seq is: CATCATCA\n",
      "1 CATCATCA motifs detected in file\n",
      "\n",
      "Seq is: ATGACCAA\n",
      "0 ATGACCAA motifs detected in file\n",
      "\n",
      "Seq is: CTTGTAAC\n",
      "0 CTTGTAAC motifs detected in file\n",
      "\n",
      "Complement found from 9664 to 9672\n",
      "Complement found from 12550 to 12558\n",
      "Complement found from 15360 to 15368\n",
      "Seq is: AAACAACC\n",
      "3 AAACAACC motifs detected in file\n",
      "\n",
      "Seq is: ACATATAT\n",
      "0 ACATATAT motifs detected in file\n",
      "\n",
      "Complement found from 12672 to 12680\n",
      "Complement found from 13659 to 13667\n",
      "Seq is: AACATTAA\n",
      "2 AACATTAA motifs detected in file\n",
      "\n",
      "Seq is: CTCTAACG\n",
      "0 CTCTAACG motifs detected in file\n",
      "\n",
      "Complement found from 4792 to 4800\n",
      "Seq is: AGCCCCCT\n",
      "1 AGCCCCCT motifs detected in file\n",
      "\n",
      "Seq is: TATCCAAT\n",
      "0 TATCCAAT motifs detected in file\n",
      "\n",
      "Seq is: ACACTAAT\n",
      "0 ACACTAAT motifs detected in file\n",
      "\n",
      "Complement found from 143 to 151\n",
      "Complement found from 15639 to 15647\n",
      "Seq is: CCTCATCC\n",
      "2 CCTCATCC motifs detected in file\n",
      "\n",
      "Complement found from 2943 to 2951\n",
      "Complement found from 8563 to 8571\n",
      "Complement found from 12847 to 12855\n",
      "Complement found from 14367 to 14375\n",
      "Complement found from 15827 to 15835\n",
      "Seq is: CAATCCTA\n",
      "5 CAATCCTA motifs detected in file\n",
      "\n",
      "Complement found from 4191 to 4199\n",
      "Complement found from 13620 to 13628\n",
      "Seq is: CTCACCCT\n",
      "2 CTCACCCT motifs detected in file\n",
      "\n",
      "Seq is: TCTTCTTT\n",
      "0 TCTTCTTT motifs detected in file\n",
      "\n",
      "Complement found from 14283 to 14291\n",
      "Complement found from 15454 to 15462\n",
      "Seq is: CTCTCCTT\n",
      "2 CTCTCCTT motifs detected in file\n",
      "\n",
      "Seq is: CATTTCAA\n",
      "0 CATTTCAA motifs detected in file\n",
      "\n",
      "Seq is: ATTAATAC\n",
      "0 ATTAATAC motifs detected in file\n",
      "\n",
      "Seq is: GGAAAAAC\n",
      "0 GGAAAAAC motifs detected in file\n",
      "\n",
      "Seq is: CCAACTCC\n",
      "0 CCAACTCC motifs detected in file\n",
      "\n",
      "Complement found from 5102 to 5110\n",
      "Complement found from 14209 to 14217\n",
      "Seq is: ACTACTAC\n",
      "2 ACTACTAC motifs detected in file\n",
      "\n",
      "Complement found from 1692 to 1700\n",
      "Complement found from 5362 to 5370\n",
      "Complement found from 14048 to 14056\n",
      "Complement found from 15328 to 15336\n",
      "Seq is: CTCCACCT\n",
      "4 CTCCACCT motifs detected in file\n",
      "\n",
      "Complement found from 8119 to 8127\n",
      "Seq is: CTAAACCA\n",
      "1 CTAAACCA motifs detected in file\n",
      "\n",
      "Seq is: CGACACTT\n",
      "0 CGACACTT motifs detected in file\n",
      "\n",
      "Complement found from 281 to 289\n",
      "Complement found from 12413 to 12421\n",
      "Seq is: TAACAAAA\n",
      "2 TAACAAAA motifs detected in file\n",
      "\n",
      "Seq is: TCCATTAT\n",
      "0 TCCATTAT motifs detected in file\n",
      "\n",
      "Seq is: CGAAAAGA\n",
      "0 CGAAAAGA motifs detected in file\n",
      "\n",
      "Seq is: TCAGTCCA\n",
      "0 TCAGTCCA motifs detected in file\n",
      "\n",
      "Seq is: ATCCAGAT\n",
      "0 ATCCAGAT motifs detected in file\n",
      "\n",
      "Complement found from 590 to 598\n",
      "Complement found from 1493 to 1501\n",
      "Complement found from 15738 to 15746\n",
      "Seq is: CCTCCTCA\n",
      "3 CCTCCTCA motifs detected in file\n",
      "\n",
      "Seq is: TCCCCGTC\n",
      "0 TCCCCGTC motifs detected in file\n",
      "\n",
      "Seq is: CAAAACCG\n",
      "0 CAAAACCG motifs detected in file\n",
      "\n",
      "Seq is: AGCATTCT\n",
      "0 AGCATTCT motifs detected in file\n",
      "\n",
      "Seq is: TCCGGATC\n",
      "0 TCCGGATC motifs detected in file\n",
      "\n",
      "Seq is: TATCCCCT\n",
      "0 TATCCCCT motifs detected in file\n",
      "\n",
      "Seq is: AACACGCC\n",
      "0 AACACGCC motifs detected in file\n",
      "\n",
      "Complement found from 9354 to 9362\n",
      "Seq is: ACACACTA\n",
      "1 ACACACTA motifs detected in file\n",
      "\n",
      "Seq is: CGATCCCA\n",
      "0 CGATCCCA motifs detected in file\n",
      "\n",
      "Complement found from 12294 to 12302\n",
      "Seq is: TCTTAGGC\n",
      "1 TCTTAGGC motifs detected in file\n",
      "\n",
      "Complement found from 14233 to 14241\n",
      "Seq is: TCATACAA\n",
      "1 TCATACAA motifs detected in file\n",
      "\n",
      "Complement found from 8350 to 8358\n",
      "Complement found from 11380 to 11388\n",
      "Seq is: CCTCTTTA\n",
      "2 CCTCTTTA motifs detected in file\n",
      "\n",
      "Seq is: TTTTACAC\n",
      "0 TTTTACAC motifs detected in file\n",
      "\n",
      "Seq is: GTATCACC\n",
      "0 GTATCACC motifs detected in file\n",
      "\n",
      "Complement found from 11601 to 11609\n",
      "Seq is: CCTAAAAT\n",
      "1 CCTAAAAT motifs detected in file\n",
      "\n",
      "Complement found from 14535 to 14543\n",
      "Seq is: AAAATTCA\n",
      "1 AAAATTCA motifs detected in file\n",
      "\n",
      "Seq is: AACAACCA\n",
      "0 AACAACCA motifs detected in file\n",
      "\n",
      "Complement found from 4948 to 4956\n",
      "Seq is: ATCCATCA\n",
      "1 ATCCATCA motifs detected in file\n",
      "\n",
      "Seq is: ACTCCAGA\n",
      "0 ACTCCAGA motifs detected in file\n",
      "\n",
      "Seq is: TCCCGACA\n",
      "0 TCCCGACA motifs detected in file\n",
      "\n",
      "Complement found from 1265 to 1273\n",
      "Seq is: ATCTTCAG\n",
      "1 ATCTTCAG motifs detected in file\n",
      "\n",
      "Seq is: GATCCTTT\n",
      "0 GATCCTTT motifs detected in file\n",
      "\n",
      "Seq is: CACTGTCG\n",
      "0 CACTGTCG motifs detected in file\n",
      "\n",
      "Seq is: CTCCCGAC\n",
      "0 CTCCCGAC motifs detected in file\n",
      "\n",
      "Complement found from 13421 to 13429\n",
      "Seq is: ACTCAAAA\n",
      "1 ACTCAAAA motifs detected in file\n",
      "\n",
      "Seq is: TCCATCTC\n",
      "0 TCCATCTC motifs detected in file\n",
      "\n",
      "Complement found from 9254 to 9262\n",
      "Complement found from 10936 to 10944\n",
      "Seq is: CCCCTAAC\n",
      "2 CCCCTAAC motifs detected in file\n",
      "\n",
      "Complement found from 2763 to 2771\n",
      "Complement found from 8506 to 8514\n",
      "Complement found from 16279 to 16287\n",
      "Seq is: AACAAACC\n",
      "3 AACAAACC motifs detected in file\n",
      "\n",
      "Complement found from 4246 to 4254\n",
      "Complement found from 8602 to 8610\n",
      "Complement found from 14486 to 14494\n",
      "Seq is: TTCCCCCT\n",
      "3 TTCCCCCT motifs detected in file\n",
      "\n",
      "Seq is: ACGCCCCC\n",
      "0 ACGCCCCC motifs detected in file\n",
      "\n",
      "Seq is: TTTACAAC\n",
      "0 TTTACAAC motifs detected in file\n",
      "\n",
      "Seq is: AATCATTA\n",
      "0 AATCATTA motifs detected in file\n",
      "\n",
      "Seq is: CTCTTTAG\n",
      "0 CTCTTTAG motifs detected in file\n",
      "\n",
      "Seq is: GACGCTTA\n",
      "0 GACGCTTA motifs detected in file\n",
      "\n",
      "Seq is: TCCGAAGG\n",
      "0 TCCGAAGG motifs detected in file\n",
      "\n",
      "Seq is: CCGACGGT\n",
      "0 CCGACGGT motifs detected in file\n",
      "\n",
      "Seq is: CCGCAAAT\n",
      "0 CCGCAAAT motifs detected in file\n",
      "\n",
      "Seq is: TACCCCAA\n",
      "0 TACCCCAA motifs detected in file\n",
      "\n",
      "Complement found from 5297 to 5305\n",
      "Complement found from 10864 to 10872\n",
      "Seq is: ATCATCCC\n",
      "2 ATCATCCC motifs detected in file\n",
      "\n",
      "Complement found from 14749 to 14757\n",
      "Seq is: ACCCCAAT\n",
      "1 ACCCCAAT motifs detected in file\n",
      "\n",
      "Seq is: AAAAGCAA\n",
      "0 AAAAGCAA motifs detected in file\n",
      "\n",
      "Complement found from 4231 to 4239\n",
      "Complement found from 13279 to 13287\n",
      "Seq is: TTACAATC\n",
      "2 TTACAATC motifs detected in file\n",
      "\n",
      "Seq is: ATTCCCAC\n",
      "0 ATTCCCAC motifs detected in file\n",
      "\n",
      "Complement found from 3447 to 3455\n",
      "Complement found from 3960 to 3968\n",
      "Seq is: CCCTTCGC\n",
      "2 CCCTTCGC motifs detected in file\n",
      "\n",
      "Complement found from 12748 to 12756\n",
      "Seq is: TCCAACTG\n",
      "1 TCCAACTG motifs detected in file\n",
      "\n",
      "Seq is: GACAATCC\n",
      "0 GACAATCC motifs detected in file\n",
      "\n",
      "Seq is: CACTCTTC\n",
      "0 CACTCTTC motifs detected in file\n",
      "\n",
      "Seq is: TTAATAAG\n",
      "0 TTAATAAG motifs detected in file\n",
      "\n",
      "Seq is: CTCACGAT\n",
      "0 CTCACGAT motifs detected in file\n",
      "\n",
      "Seq is: ATCCGCGA\n",
      "0 ATCCGCGA motifs detected in file\n",
      "\n",
      "Complement found from 15257 to 15265\n",
      "Seq is: ACAGTCCC\n",
      "1 ACAGTCCC motifs detected in file\n",
      "\n",
      "Complement found from 14377 to 14385\n",
      "Seq is: TCCATCGC\n",
      "1 TCCATCGC motifs detected in file\n",
      "\n",
      "Complement found from 10600 to 10608\n",
      "Complement found from 13568 to 13576\n",
      "Seq is: TACTCTCA\n",
      "2 TACTCTCA motifs detected in file\n",
      "\n",
      "Seq is: TTATCTAT\n",
      "0 TTATCTAT motifs detected in file\n",
      "\n",
      "Seq is: CCCGAGTC\n",
      "0 CCCGAGTC motifs detected in file\n",
      "\n",
      "Complement found from 13529 to 13537\n",
      "Seq is: CGCAAACA\n",
      "1 CGCAAACA motifs detected in file\n",
      "\n",
      "Complement found from 13346 to 13354\n",
      "Seq is: CATACTAT\n",
      "1 CATACTAT motifs detected in file\n",
      "\n",
      "Complement found from 13545 to 13553\n",
      "Seq is: ACAAACGC\n",
      "1 ACAAACGC motifs detected in file\n",
      "\n",
      "Complement found from 1889 to 1897\n",
      "Complement found from 15990 to 15998\n",
      "Seq is: CAAAGCTA\n",
      "2 CAAAGCTA motifs detected in file\n",
      "\n",
      "Seq is: CTACACCA\n",
      "0 CTACACCA motifs detected in file\n",
      "\n",
      "Seq is: GAAACCTC\n",
      "0 GAAACCTC motifs detected in file\n",
      "\n",
      "Seq is: ATCTTTGG\n",
      "0 ATCTTTGG motifs detected in file\n",
      "\n",
      "Seq is: ACACTCCT\n",
      "0 ACACTCCT motifs detected in file\n",
      "\n",
      "Seq is: TTCCATAA\n",
      "0 TTCCATAA motifs detected in file\n",
      "\n",
      "Seq is: GGACGATT\n",
      "0 GGACGATT motifs detected in file\n",
      "\n",
      "Seq is: ACGATCCG\n",
      "0 ACGATCCG motifs detected in file\n",
      "\n",
      "Seq is: ACGGTTAC\n",
      "0 ACGGTTAC motifs detected in file\n",
      "\n",
      "Complement found from 10622 to 10630\n",
      "Seq is: CACTCCCT\n",
      "1 CACTCCCT motifs detected in file\n",
      "\n",
      "Seq is: CCAACTTC\n",
      "0 CCAACTTC motifs detected in file\n",
      "\n",
      "Seq is: ACTCTCCA\n",
      "0 ACTCTCCA motifs detected in file\n",
      "\n",
      "Complement found from 2498 to 2506\n",
      "Seq is: TACCAAAA\n",
      "1 TACCAAAA motifs detected in file\n",
      "\n",
      "Complement found from 12795 to 12803\n",
      "Seq is: CTCATCAG\n",
      "1 CTCATCAG motifs detected in file\n",
      "\n",
      "Seq is: GAGGATAA\n",
      "0 GAGGATAA motifs detected in file\n",
      "\n",
      "Seq is: AAAGCTTA\n",
      "0 AAAGCTTA motifs detected in file\n",
      "\n",
      "Seq is: TAGAACAA\n",
      "0 TAGAACAA motifs detected in file\n",
      "\n",
      "Seq is: GTAACAAT\n",
      "0 GTAACAAT motifs detected in file\n",
      "\n",
      "Complement found from 10712 to 10720\n",
      "Seq is: TCCAACAC\n",
      "1 TCCAACAC motifs detected in file\n",
      "\n",
      "Complement found from 5103 to 5111\n",
      "Seq is: CTACTACC\n",
      "1 CTACTACC motifs detected in file\n",
      "\n",
      "Seq is: TGGGCCTC\n",
      "0 TGGGCCTC motifs detected in file\n",
      "\n",
      "Seq is: GTGTATTT\n",
      "0 GTGTATTT motifs detected in file\n",
      "\n",
      "Seq is: ATCATACC\n",
      "0 ATCATACC motifs detected in file\n",
      "\n",
      "Complement found from 14837 to 14845\n",
      "Seq is: GAAACTTC\n",
      "1 GAAACTTC motifs detected in file\n",
      "\n",
      "Seq is: TTCCGCAC\n",
      "0 TTCCGCAC motifs detected in file\n",
      "\n",
      "Complement found from 12239 to 12247\n",
      "Seq is: CCATGTCT\n",
      "1 CCATGTCT motifs detected in file\n",
      "\n",
      "Seq is: ACACGTCC\n",
      "0 ACACGTCC motifs detected in file\n",
      "\n",
      "Seq is: TTACGATC\n",
      "0 TTACGATC motifs detected in file\n",
      "\n",
      "Seq is: CACACCAA\n",
      "0 CACACCAA motifs detected in file\n",
      "\n",
      "Seq is: CCAACTAC\n",
      "0 CCAACTAC motifs detected in file\n",
      "\n",
      "Seq is: GGCTAACA\n",
      "0 GGCTAACA motifs detected in file\n",
      "\n",
      "Seq is: TTGATAAT\n",
      "0 TTGATAAT motifs detected in file\n",
      "\n",
      "Seq is: ACTCAGGA\n",
      "0 ACTCAGGA motifs detected in file\n",
      "\n",
      "Complement found from 5612 to 5620\n",
      "Seq is: TCAACTGA\n",
      "1 TCAACTGA motifs detected in file\n",
      "\n",
      "Complement found from 2860 to 2868\n",
      "Seq is: ACTTCACC\n",
      "1 ACTTCACC motifs detected in file\n",
      "\n",
      "Seq is: TCTTCCGA\n",
      "0 TCTTCCGA motifs detected in file\n",
      "\n",
      "Complement found from 10760 to 10768\n",
      "Seq is: TGCTAAAA\n",
      "1 TGCTAAAA motifs detected in file\n",
      "\n",
      "Seq is: AAACTACA\n",
      "0 AAACTACA motifs detected in file\n",
      "\n",
      "Seq is: GTAAAACA\n",
      "0 GTAAAACA motifs detected in file\n",
      "\n",
      "Seq is: CATTCCCG\n",
      "0 CATTCCCG motifs detected in file\n",
      "\n",
      "Seq is: CGTCTGAC\n",
      "0 CGTCTGAC motifs detected in file\n",
      "\n",
      "Seq is: GACGCTTG\n",
      "0 GACGCTTG motifs detected in file\n",
      "\n",
      "Complement found from 3539 to 3547\n",
      "Complement found from 15479 to 15487\n",
      "Seq is: TCTCACCA\n",
      "2 TCTCACCA motifs detected in file\n",
      "\n",
      "Seq is: CTATCGCG\n",
      "0 CTATCGCG motifs detected in file\n",
      "\n",
      "Seq is: GATTCGTA\n",
      "0 GATTCGTA motifs detected in file\n",
      "\n",
      "Complement found from 10982 to 10990\n",
      "Seq is: TCACAATC\n",
      "1 TCACAATC motifs detected in file\n",
      "\n",
      "Complement found from 4253 to 4261\n",
      "Seq is: TCAAACCT\n",
      "1 TCAAACCT motifs detected in file\n",
      "\n",
      "Seq is: AATCACCC\n",
      "0 AATCACCC motifs detected in file\n",
      "\n",
      "Complement found from 4274 to 4282\n",
      "Seq is: GATAAAAG\n",
      "1 GATAAAAG motifs detected in file\n",
      "\n",
      "Complement found from 7856 to 7864\n",
      "Seq is: ACGATCCC\n",
      "1 ACGATCCC motifs detected in file\n",
      "\n",
      "Complement found from 6552 to 6560\n",
      "Seq is: CCACCTTC\n",
      "1 CCACCTTC motifs detected in file\n",
      "\n",
      "Complement found from 8968 to 8976\n",
      "Seq is: GCCTACTC\n",
      "1 GCCTACTC motifs detected in file\n",
      "\n",
      "Complement found from 10219 to 10227\n",
      "Complement found from 13614 to 13622\n",
      "Seq is: ATTCTTCT\n",
      "2 ATTCTTCT motifs detected in file\n",
      "\n",
      "Complement found from 8716 to 8724\n",
      "Seq is: AAGGACGA\n",
      "1 AAGGACGA motifs detected in file\n",
      "\n",
      "Seq is: TGTTGATA\n",
      "0 TGTTGATA motifs detected in file\n",
      "\n",
      "Seq is: TCACGAAC\n",
      "0 TCACGAAC motifs detected in file\n",
      "\n",
      "Complement found from 14062 to 14070\n",
      "Seq is: TCACCTCA\n",
      "1 TCACCTCA motifs detected in file\n",
      "\n",
      "Seq is: TCCCGACT\n",
      "0 TCCCGACT motifs detected in file\n",
      "\n",
      "Complement found from 14276 to 14284\n",
      "Complement found from 14420 to 14428\n",
      "Seq is: CTGACCCC\n",
      "2 CTGACCCC motifs detected in file\n",
      "\n",
      "Seq is: ACCCCGGA\n",
      "0 ACCCCGGA motifs detected in file\n",
      "\n",
      "Seq is: AGATACCG\n",
      "0 AGATACCG motifs detected in file\n",
      "\n",
      "Complement found from 13031 to 13039\n",
      "Seq is: ACTCCCCT\n",
      "1 ACTCCCCT motifs detected in file\n",
      "\n",
      "Seq is: CAGTCCCC\n",
      "0 CAGTCCCC motifs detected in file\n",
      "\n",
      "Seq is: ACCTCTGG\n",
      "0 ACCTCTGG motifs detected in file\n",
      "\n",
      "Complement found from 909 to 917\n",
      "Seq is: ATTAACCC\n",
      "1 ATTAACCC motifs detected in file\n",
      "\n",
      "Seq is: GACTAAAC\n",
      "0 GACTAAAC motifs detected in file\n",
      "\n",
      "Seq is: GGACGACG\n",
      "0 GGACGACG motifs detected in file\n",
      "\n",
      "Seq is: ACGATCCT\n",
      "0 ACGATCCT motifs detected in file\n",
      "\n",
      "Seq is: CCTCCGGA\n",
      "0 CCTCCGGA motifs detected in file\n",
      "\n",
      "Complement found from 8429 to 8437\n",
      "Complement found from 11153 to 11161\n",
      "Complement found from 14059 to 14067\n",
      "Seq is: TCATCACC\n",
      "3 TCATCACC motifs detected in file\n",
      "\n",
      "Seq is: CCACTCCG\n",
      "0 CCACTCCG motifs detected in file\n",
      "\n",
      "Seq is: AACCTAAT\n",
      "0 AACCTAAT motifs detected in file\n",
      "\n",
      "Complement found from 5621 to 5629\n",
      "Seq is: CGCAAATC\n",
      "1 CGCAAATC motifs detected in file\n",
      "\n",
      "Seq is: TTCCCGAT\n",
      "0 TTCCCGAT motifs detected in file\n",
      "\n",
      "Complement found from 6193 to 6201\n",
      "Complement found from 13771 to 13779\n",
      "Complement found from 15676 to 15684\n",
      "Seq is: AAACAACA\n",
      "3 AAACAACA motifs detected in file\n",
      "\n",
      "Seq is: CCCAGAGT\n",
      "0 CCCAGAGT motifs detected in file\n",
      "\n",
      "Seq is: ACTCAACC\n",
      "0 ACTCAACC motifs detected in file\n",
      "\n",
      "Seq is: TCACATCC\n",
      "0 TCACATCC motifs detected in file\n",
      "\n",
      "Seq is: TATTTAGT\n",
      "0 TATTTAGT motifs detected in file\n",
      "\n",
      "Seq is: ACGATTCC\n",
      "0 ACGATTCC motifs detected in file\n",
      "\n",
      "Seq is: GCTCCTAC\n",
      "0 GCTCCTAC motifs detected in file\n",
      "\n",
      "Seq is: TTTGGCTA\n",
      "0 TTTGGCTA motifs detected in file\n",
      "\n",
      "Seq is: TAGCGGCT\n",
      "0 TAGCGGCT motifs detected in file\n",
      "\n",
      "Complement found from 12823 to 12831\n",
      "Seq is: ATGCCAAC\n",
      "1 ATGCCAAC motifs detected in file\n",
      "\n",
      "Seq is: ATATCCTA\n",
      "0 ATATCCTA motifs detected in file\n",
      "\n",
      "Seq is: ACGAACTT\n",
      "0 ACGAACTT motifs detected in file\n",
      "\n",
      "Seq is: ACCGACGA\n",
      "0 ACCGACGA motifs detected in file\n",
      "\n",
      "Seq is: CACAACCG\n",
      "0 CACAACCG motifs detected in file\n",
      "\n",
      "Seq is: TAGACGAG\n",
      "0 TAGACGAG motifs detected in file\n",
      "\n",
      "Complement found from 6186 to 6194\n",
      "Seq is: CCCGCATA\n",
      "1 CCCGCATA motifs detected in file\n",
      "\n",
      "Seq is: GTAGTTGA\n",
      "0 GTAGTTGA motifs detected in file\n",
      "\n",
      "Seq is: CTACTCGT\n",
      "0 CTACTCGT motifs detected in file\n",
      "\n",
      "Complement found from 4027 to 4035\n",
      "Complement found from 12703 to 12711\n",
      "Complement found from 15026 to 15034\n",
      "Seq is: TCTTCCTA\n",
      "3 TCTTCCTA motifs detected in file\n",
      "\n",
      "Seq is: TATTAAGG\n",
      "0 TATTAAGG motifs detected in file\n",
      "\n",
      "Seq is: ATGCGGGA\n",
      "0 ATGCGGGA motifs detected in file\n",
      "\n",
      "Seq is: GAGTCGGC\n",
      "0 GAGTCGGC motifs detected in file\n",
      "\n",
      "Seq is: TACTTGTC\n",
      "0 TACTTGTC motifs detected in file\n",
      "\n",
      "Seq is: AACCTTAT\n",
      "0 AACCTTAT motifs detected in file\n",
      "\n",
      "Complement found from 8660 to 8668\n",
      "Complement found from 10777 to 10785\n",
      "Seq is: CCAACAAT\n",
      "2 CCAACAAT motifs detected in file\n",
      "\n",
      "Seq is: CGCCATTG\n",
      "0 CGCCATTG motifs detected in file\n",
      "\n",
      "Complement found from 3342 to 3350\n",
      "Complement found from 9108 to 9116\n",
      "Complement found from 16000 to 16008\n",
      "Seq is: ATTCTAAT\n",
      "3 ATTCTAAT motifs detected in file\n",
      "\n",
      "Seq is: CATACCAT\n",
      "0 CATACCAT motifs detected in file\n",
      "\n",
      "Complement found from 4671 to 4679\n",
      "Seq is: TAATCCTT\n",
      "1 TAATCCTT motifs detected in file\n",
      "\n",
      "Complement found from 8970 to 8978\n",
      "Complement found from 9030 to 9038\n",
      "Complement found from 11236 to 11244\n",
      "Complement found from 12696 to 12704\n",
      "Seq is: CTACTCAT\n",
      "4 CTACTCAT motifs detected in file\n",
      "\n",
      "Complement found from 8821 to 8829\n",
      "Seq is: CTATAAAC\n",
      "1 CTATAAAC motifs detected in file\n",
      "\n",
      "Seq is: TTCTTGAC\n",
      "0 TTCTTGAC motifs detected in file\n",
      "\n",
      "Complement found from 8042 to 8050\n",
      "Complement found from 15179 to 15187\n",
      "Seq is: TAATTACA\n",
      "2 TAATTACA motifs detected in file\n",
      "\n",
      "Seq is: AACCCAGA\n",
      "0 AACCCAGA motifs detected in file\n",
      "\n",
      "Seq is: CTCAAATA\n",
      "0 CTCAAATA motifs detected in file\n",
      "\n",
      "Seq is: TATAGTGT\n",
      "0 TATAGTGT motifs detected in file\n",
      "\n",
      "Seq is: CACTCTTA\n",
      "0 CACTCTTA motifs detected in file\n",
      "\n",
      "Seq is: AGATACTA\n",
      "0 AGATACTA motifs detected in file\n",
      "\n",
      "Seq is: CCTGGTAC\n",
      "0 CCTGGTAC motifs detected in file\n",
      "\n",
      "Seq is: ATTGCTTG\n",
      "0 ATTGCTTG motifs detected in file\n",
      "\n",
      "Seq is: TTACGATG\n",
      "0 TTACGATG motifs detected in file\n",
      "\n",
      "Complement found from 10553 to 10561\n",
      "Seq is: TCCCTACT\n",
      "1 TCCCTACT motifs detected in file\n",
      "\n",
      "Complement found from 11484 to 11492\n",
      "Seq is: TATAATAC\n",
      "1 TATAATAC motifs detected in file\n",
      "\n",
      "Seq is: CTCTTCAT\n",
      "0 CTCTTCAT motifs detected in file\n",
      "\n",
      "Seq is: CAGATCAA\n",
      "0 CAGATCAA motifs detected in file\n",
      "\n",
      "Seq is: ACTTCGAA\n",
      "0 ACTTCGAA motifs detected in file\n",
      "\n",
      "Complement found from 6220 to 6228\n",
      "Seq is: TCCCTCTC\n",
      "1 TCCCTCTC motifs detected in file\n",
      "\n",
      "Seq is: GACCCAAC\n",
      "0 GACCCAAC motifs detected in file\n",
      "\n",
      "Complement found from 16163 to 16171\n",
      "Seq is: AAACCCAA\n",
      "1 AAACCCAA motifs detected in file\n",
      "\n",
      "Seq is: CACCGAGT\n",
      "0 CACCGAGT motifs detected in file\n",
      "\n",
      "Seq is: CACAGTCA\n",
      "0 CACAGTCA motifs detected in file\n",
      "\n",
      "Seq is: AGCTCTAT\n",
      "0 AGCTCTAT motifs detected in file\n",
      "\n",
      "Seq is: TATTGAAG\n",
      "0 TATTGAAG motifs detected in file\n",
      "\n",
      "Complement found from 383 to 391\n",
      "Seq is: AACCAGAT\n",
      "1 AACCAGAT motifs detected in file\n",
      "\n",
      "Seq is: CCGTGTAC\n",
      "0 CCGTGTAC motifs detected in file\n",
      "\n",
      "Complement found from 8502 to 8510\n",
      "Complement found from 11078 to 11086\n",
      "Complement found from 11567 to 11575\n",
      "Seq is: TTATAACA\n",
      "3 TTATAACA motifs detected in file\n",
      "\n",
      "Complement found from 8928 to 8936\n",
      "Complement found from 15533 to 15541\n",
      "Seq is: ACACCCCT\n",
      "2 ACACCCCT motifs detected in file\n",
      "\n",
      "Seq is: TCTCTGAC\n",
      "0 TCTCTGAC motifs detected in file\n",
      "\n",
      "Complement found from 151 to 159\n",
      "Complement found from 10245 to 10253\n",
      "Seq is: TATTATTT\n",
      "2 TATTATTT motifs detected in file\n",
      "\n",
      "Complement found from 8463 to 8471\n",
      "Complement found from 9026 to 9034\n",
      "Seq is: CCACCTAC\n",
      "2 CCACCTAC motifs detected in file\n",
      "\n",
      "Seq is: GCTGTTAC\n",
      "0 GCTGTTAC motifs detected in file\n",
      "\n",
      "Seq is: CTAAAATG\n",
      "0 CTAAAATG motifs detected in file\n",
      "\n",
      "Complement found from 2407 to 2415\n",
      "Seq is: TATTACCC\n",
      "1 TATTACCC motifs detected in file\n",
      "\n",
      "Complement found from 8408 to 8416\n",
      "Seq is: CCATACTC\n",
      "1 CCATACTC motifs detected in file\n",
      "\n",
      "Complement found from 12417 to 12425\n",
      "Seq is: AAAAAAAA\n",
      "1 AAAAAAAA motifs detected in file\n",
      "\n",
      "Seq is: CAATCCCA\n",
      "0 CAATCCCA motifs detected in file\n",
      "\n",
      "Seq is: ATTGCTCC\n",
      "0 ATTGCTCC motifs detected in file\n",
      "\n",
      "Seq is: CACCATTC\n",
      "0 CACCATTC motifs detected in file\n",
      "\n",
      "Complement found from 8277 to 8285\n",
      "Complement found from 9528 to 9536\n",
      "Seq is: CTACCCCC\n",
      "2 CTACCCCC motifs detected in file\n",
      "\n",
      "Complement found from 9146 to 9154\n",
      "Seq is: CTTAATCC\n",
      "1 CTTAATCC motifs detected in file\n",
      "\n",
      "Seq is: CTTCAGTC\n",
      "0 CTTCAGTC motifs detected in file\n",
      "\n",
      "Seq is: CCAATCCC\n",
      "0 CCAATCCC motifs detected in file\n",
      "\n",
      "Seq is: ACCAATAT\n",
      "0 ACCAATAT motifs detected in file\n",
      "\n",
      "Complement found from 13928 to 13936\n",
      "Seq is: CATCACAC\n",
      "1 CATCACAC motifs detected in file\n",
      "\n",
      "Seq is: GTACCAAT\n",
      "0 GTACCAAT motifs detected in file\n",
      "\n",
      "Seq is: AATGAAAA\n",
      "0 AATGAAAA motifs detected in file\n",
      "\n",
      "Complement found from 1100 to 1108\n",
      "Seq is: TAAACCTC\n",
      "1 TAAACCTC motifs detected in file\n",
      "\n",
      "Seq is: AACGTGGT\n",
      "0 AACGTGGT motifs detected in file\n",
      "\n",
      "Seq is: TTTAAAAA\n",
      "0 TTTAAAAA motifs detected in file\n",
      "\n",
      "Seq is: CCCCGGAT\n",
      "0 CCCCGGAT motifs detected in file\n",
      "\n",
      "Seq is: TCTGGTTA\n",
      "0 TCTGGTTA motifs detected in file\n",
      "\n",
      "Seq is: CCTATCGA\n",
      "0 CCTATCGA motifs detected in file\n",
      "\n",
      "Complement found from 7047 to 7055\n",
      "Complement found from 14252 to 14260\n",
      "Seq is: CAATAGGA\n",
      "2 CAATAGGA motifs detected in file\n",
      "\n",
      "Complement found from 7306 to 7314\n",
      "Seq is: AATTTTCA\n",
      "1 AATTTTCA motifs detected in file\n",
      "\n",
      "Complement found from 14348 to 14356\n",
      "Seq is: ACTCTTTC\n",
      "1 ACTCTTTC motifs detected in file\n",
      "\n",
      "Seq is: GGTACAAC\n",
      "0 GGTACAAC motifs detected in file\n",
      "\n",
      "Seq is: AATCTGTA\n",
      "0 AATCTGTA motifs detected in file\n",
      "\n",
      "Complement found from 3167 to 3175\n",
      "Seq is: CCCCCGTA\n",
      "1 CCCCCGTA motifs detected in file\n",
      "\n",
      "Seq is: CTCAATCG\n",
      "0 CTCAATCG motifs detected in file\n",
      "\n",
      "Seq is: TCAAGAAC\n",
      "0 TCAAGAAC motifs detected in file\n",
      "\n",
      "Seq is: ACTCGAAA\n",
      "0 ACTCGAAA motifs detected in file\n",
      "\n",
      "Seq is: GAGCCATT\n",
      "0 GAGCCATT motifs detected in file\n",
      "\n",
      "Complement found from 14150 to 14158\n",
      "Seq is: TATTCCCC\n",
      "1 TATTCCCC motifs detected in file\n",
      "\n",
      "Seq is: AGCATTCG\n",
      "0 AGCATTCG motifs detected in file\n",
      "\n",
      "Seq is: GAGACAAC\n",
      "0 GAGACAAC motifs detected in file\n",
      "\n",
      "Seq is: AGTCTAAG\n",
      "0 AGTCTAAG motifs detected in file\n",
      "\n",
      "Seq is: TGTTAGAC\n",
      "0 TGTTAGAC motifs detected in file\n",
      "\n",
      "Complement found from 5425 to 5433\n",
      "Seq is: TACAAAAC\n",
      "1 TACAAAAC motifs detected in file\n",
      "\n",
      "Seq is: CAATTTGA\n",
      "0 CAATTTGA motifs detected in file\n",
      "\n",
      "Seq is: TATAAATG\n",
      "0 TATAAATG motifs detected in file\n",
      "\n",
      "Complement found from 6344 to 6352\n",
      "Seq is: TTCTCCTT\n",
      "1 TTCTCCTT motifs detected in file\n",
      "\n",
      "Seq is: TTGGGCCA\n",
      "0 TTGGGCCA motifs detected in file\n",
      "\n",
      "Complement found from 15411 to 15419\n",
      "Seq is: TTACTACA\n",
      "1 TTACTACA motifs detected in file\n",
      "\n",
      "Complement found from 3412 to 3420\n",
      "Complement found from 8367 to 8375\n",
      "Seq is: GCCCCAAC\n",
      "2 GCCCCAAC motifs detected in file\n",
      "\n",
      "Seq is: TCCCTATC\n",
      "0 TCCCTATC motifs detected in file\n",
      "\n",
      "Seq is: CTCCTCTT\n",
      "0 CTCCTCTT motifs detected in file\n",
      "\n",
      "Complement found from 10934 to 10942\n",
      "Complement found from 13126 to 13134\n",
      "Complement found from 14768 to 14776\n",
      "Complement found from 15365 to 15373\n",
      "Seq is: ACCCCCTA\n",
      "4 ACCCCCTA motifs detected in file\n",
      "\n",
      "Seq is: TCCACATA\n",
      "0 TCCACATA motifs detected in file\n",
      "\n",
      "Complement found from 11542 to 11550\n",
      "Seq is: CTTGTACT\n",
      "1 CTTGTACT motifs detected in file\n",
      "\n",
      "Complement found from 5676 to 5684\n",
      "Complement found from 14620 to 14628\n",
      "Seq is: CCCACAAA\n",
      "2 CCCACAAA motifs detected in file\n",
      "\n",
      "Complement found from 1937 to 1945\n",
      "Seq is: AGAGCACA\n",
      "1 AGAGCACA motifs detected in file\n",
      "\n",
      "Seq is: CTTACTCC\n",
      "0 CTTACTCC motifs detected in file\n",
      "\n",
      "Complement found from 15867 to 15875\n",
      "Seq is: CAAAATAC\n",
      "1 CAAAATAC motifs detected in file\n",
      "\n",
      "Complement found from 6400 to 6408\n",
      "Complement found from 10779 to 10787\n",
      "Seq is: AACAATTA\n",
      "2 AACAATTA motifs detected in file\n",
      "\n",
      "Complement found from 8654 to 8662\n",
      "Complement found from 14334 to 14342\n",
      "Seq is: CACCACCC\n",
      "2 CACCACCC motifs detected in file\n",
      "\n",
      "Complement found from 11292 to 11300\n",
      "Seq is: ACTCACTC\n",
      "1 ACTCACTC motifs detected in file\n",
      "\n",
      "Seq is: GGGGTAAC\n",
      "0 GGGGTAAC motifs detected in file\n",
      "\n",
      "Complement found from 11261 to 11269\n",
      "Seq is: ACAACACC\n",
      "1 ACAACACC motifs detected in file\n",
      "\n",
      "Seq is: ATTTATAC\n",
      "0 ATTTATAC motifs detected in file\n",
      "\n",
      "Complement found from 11056 to 11064\n",
      "Complement found from 15850 to 15858\n",
      "Seq is: ATCTCCCT\n",
      "2 ATCTCCCT motifs detected in file\n",
      "\n",
      "Complement found from 1184 to 1192\n",
      "Seq is: CATATCCC\n",
      "1 CATATCCC motifs detected in file\n",
      "\n",
      "Complement found from 12532 to 12540\n",
      "Seq is: GACACTGA\n",
      "1 GACACTGA motifs detected in file\n",
      "\n",
      "Complement found from 14233 to 14241\n",
      "Seq is: TCATACAA\n",
      "1 TCATACAA motifs detected in file\n",
      "\n",
      "Seq is: CTCAGGAC\n",
      "0 CTCAGGAC motifs detected in file\n",
      "\n",
      "Complement found from 12599 to 12607\n",
      "Seq is: ATTCATCC\n",
      "1 ATTCATCC motifs detected in file\n",
      "\n",
      "Seq is: TCTCACTA\n",
      "0 TCTCACTA motifs detected in file\n",
      "\n",
      "Seq is: TAAACTAG\n",
      "0 TAAACTAG motifs detected in file\n",
      "\n",
      "Complement found from 12130 to 12138\n",
      "Seq is: TCCTCTTG\n",
      "1 TCCTCTTG motifs detected in file\n",
      "\n",
      "Complement found from 7888 to 7896\n",
      "Complement found from 14741 to 14749\n",
      "Seq is: CACCAATG\n",
      "2 CACCAATG motifs detected in file\n",
      "\n",
      "Seq is: ATCGTGTC\n",
      "0 ATCGTGTC motifs detected in file\n",
      "\n",
      "Seq is: TCTCAAGA\n",
      "0 TCTCAAGA motifs detected in file\n",
      "\n",
      "Seq is: GGGTCATC\n",
      "0 GGGTCATC motifs detected in file\n",
      "\n",
      "Complement found from 6402 to 6410\n",
      "Seq is: CAATTATC\n",
      "1 CAATTATC motifs detected in file\n",
      "\n",
      "Complement found from 431 to 439\n",
      "Complement found from 566 to 574\n",
      "Complement found from 7464 to 7472\n",
      "Complement found from 9530 to 9538\n",
      "Complement found from 11865 to 11873\n",
      "Seq is: ACCCCCCA\n",
      "5 ACCCCCCA motifs detected in file\n",
      "\n",
      "Seq is: TTCCGCTC\n",
      "0 TTCCGCTC motifs detected in file\n",
      "\n",
      "Seq is: CAATCGCT\n",
      "0 CAATCGCT motifs detected in file\n",
      "\n",
      "Complement found from 3373 to 3381\n",
      "Seq is: CCGAACGA\n",
      "1 CCGAACGA motifs detected in file\n",
      "\n",
      "Seq is: TCTTCAGT\n",
      "0 TCTTCAGT motifs detected in file\n",
      "\n",
      "Seq is: AGTTTTTC\n",
      "0 AGTTTTTC motifs detected in file\n",
      "\n",
      "Complement found from 10451 to 10459\n",
      "Seq is: GATAATCA\n",
      "1 GATAATCA motifs detected in file\n",
      "\n",
      "Seq is: CCCTCATC\n",
      "0 CCCTCATC motifs detected in file\n",
      "\n",
      "Seq is: TCAAACTT\n",
      "0 TCAAACTT motifs detected in file\n",
      "\n",
      "Seq is: CAGGAACT\n",
      "0 CAGGAACT motifs detected in file\n",
      "\n",
      "Complement found from 6225 to 6233\n",
      "Complement found from 11933 to 11941\n",
      "Seq is: CTCTCCTA\n",
      "2 CTCTCCTA motifs detected in file\n",
      "\n",
      "Complement found from 8157 to 8165\n",
      "Seq is: ATACTACG\n",
      "1 ATACTACG motifs detected in file\n",
      "\n",
      "Complement found from 12530 to 12538\n",
      "Seq is: CTGACACT\n",
      "1 CTGACACT motifs detected in file\n",
      "\n",
      "Complement found from 4651 to 4659\n",
      "Seq is: CACGCAAG\n",
      "1 CACGCAAG motifs detected in file\n",
      "\n",
      "Complement found from 3670 to 3678\n",
      "Seq is: CATCAAAC\n",
      "1 CATCAAAC motifs detected in file\n",
      "\n",
      "Seq is: TCAAACGA\n",
      "0 TCAAACGA motifs detected in file\n",
      "\n",
      "Seq is: TCCGTCTT\n",
      "0 TCCGTCTT motifs detected in file\n",
      "\n",
      "Complement found from 12115 to 12123\n",
      "Seq is: ATCATTAC\n",
      "1 ATCATTAC motifs detected in file\n",
      "\n",
      "Complement found from 7942 to 7950\n",
      "Seq is: TCCTACAT\n",
      "1 TCCTACAT motifs detected in file\n",
      "\n",
      "Seq is: TCGGGCAC\n",
      "0 TCGGGCAC motifs detected in file\n",
      "\n",
      "Seq is: CCGCTAAT\n",
      "0 CCGCTAAT motifs detected in file\n",
      "\n",
      "Complement found from 6211 to 6219\n",
      "Seq is: ACTCTTAC\n",
      "1 ACTCTTAC motifs detected in file\n",
      "\n",
      "Seq is: TGACGCGG\n",
      "0 TGACGCGG motifs detected in file\n",
      "\n",
      "Seq is: CCACTTCG\n",
      "0 CCACTTCG motifs detected in file\n",
      "\n",
      "Seq is: AAGTCCCC\n",
      "0 AAGTCCCC motifs detected in file\n",
      "\n",
      "Complement found from 1991 to 1999\n",
      "Complement found from 4254 to 4262\n",
      "Complement found from 7121 to 7129\n",
      "Complement found from 16281 to 16289\n",
      "Seq is: CAAACCTA\n",
      "4 CAAACCTA motifs detected in file\n",
      "\n",
      "Complement found from 6212 to 6220\n",
      "Seq is: CTCTTACC\n",
      "1 CTCTTACC motifs detected in file\n",
      "\n",
      "Complement found from 4713 to 4721\n",
      "Seq is: GACAATGA\n",
      "1 GACAATGA motifs detected in file\n",
      "\n",
      "Seq is: TGCTCCCG\n",
      "0 TGCTCCCG motifs detected in file\n",
      "\n",
      "Seq is: ATACACCG\n",
      "0 ATACACCG motifs detected in file\n",
      "\n",
      "Seq is: ACTAACTT\n",
      "0 ACTAACTT motifs detected in file\n",
      "\n",
      "Seq is: CTCATACG\n",
      "0 CTCATACG motifs detected in file\n",
      "\n",
      "Complement found from 9614 to 9622\n",
      "Seq is: TTACTCGC\n",
      "1 TTACTCGC motifs detected in file\n",
      "\n",
      "Complement found from 12442 to 12450\n",
      "Complement found from 13873 to 13881\n",
      "Seq is: TAAAATCC\n",
      "2 TAAAATCC motifs detected in file\n",
      "\n",
      "Seq is: AGACAAAC\n",
      "0 AGACAAAC motifs detected in file\n",
      "\n",
      "Complement found from 15804 to 15812\n",
      "Seq is: AGCATCCG\n",
      "1 AGCATCCG motifs detected in file\n",
      "\n",
      "Complement found from 11039 to 11047\n",
      "Seq is: TCTACCTC\n",
      "1 TCTACCTC motifs detected in file\n",
      "\n",
      "Seq is: GAACAATA\n",
      "0 GAACAATA motifs detected in file\n",
      "\n",
      "Seq is: TTAATACG\n",
      "0 TTAATACG motifs detected in file\n",
      "\n",
      "Seq is: GAGTATCC\n",
      "0 GAGTATCC motifs detected in file\n",
      "\n",
      "Seq is: CTATCATG\n",
      "0 CTATCATG motifs detected in file\n",
      "\n",
      "Seq is: TTCCTTCC\n",
      "0 TTCCTTCC motifs detected in file\n",
      "\n",
      "Seq is: CCATCCGA\n",
      "0 CCATCCGA motifs detected in file\n",
      "\n",
      "Complement found from 3987 to 3995\n",
      "Complement found from 13214 to 13222\n",
      "Complement found from 13542 to 13550\n",
      "Seq is: TACACAAA\n",
      "3 TACACAAA motifs detected in file\n",
      "\n",
      "Complement found from 15257 to 15265\n",
      "Seq is: ACAGTCCC\n",
      "1 ACAGTCCC motifs detected in file\n",
      "\n",
      "Seq is: CCAACTCT\n",
      "0 CCAACTCT motifs detected in file\n",
      "\n",
      "Seq is: TACTCACA\n",
      "0 TACTCACA motifs detected in file\n",
      "\n",
      "Complement found from 2328 to 2336\n",
      "Complement found from 14826 to 14834\n",
      "Seq is: CTCCGCAT\n",
      "2 CTCCGCAT motifs detected in file\n",
      "\n",
      "Seq is: AATATGGT\n",
      "0 AATATGGT motifs detected in file\n",
      "\n",
      "Seq is: ATCGGCGG\n",
      "0 ATCGGCGG motifs detected in file\n",
      "\n",
      "Seq is: ATCAAAAT\n",
      "0 ATCAAAAT motifs detected in file\n",
      "\n",
      "Seq is: TCTCATGA\n",
      "0 TCTCATGA motifs detected in file\n",
      "\n",
      "Seq is: CGCCGTTC\n",
      "0 CGCCGTTC motifs detected in file\n",
      "\n",
      "Complement found from 8694 to 8702\n",
      "Seq is: ATGATAAC\n",
      "1 ATGATAAC motifs detected in file\n",
      "\n",
      "Seq is: TGGGTCGC\n",
      "0 TGGGTCGC motifs detected in file\n",
      "\n",
      "Complement found from 4134 to 4142\n",
      "Seq is: TACCCCCG\n",
      "1 TACCCCCG motifs detected in file\n",
      "\n",
      "Seq is: AAGCTGTA\n",
      "0 AAGCTGTA motifs detected in file\n",
      "\n",
      "Seq is: CCCGAAAT\n",
      "0 CCCGAAAT motifs detected in file\n",
      "\n",
      "Complement found from 5549 to 5557\n",
      "Seq is: CCCTCAGT\n",
      "1 CCCTCAGT motifs detected in file\n",
      "\n",
      "Seq is: ATTCACCT\n",
      "0 ATTCACCT motifs detected in file\n",
      "\n",
      "Seq is: CAGGCATT\n",
      "0 CAGGCATT motifs detected in file\n",
      "\n",
      "Complement found from 4219 to 4227\n",
      "Complement found from 10209 to 10217\n",
      "Complement found from 12589 to 12597\n",
      "Seq is: TCTCCATA\n",
      "3 TCTCCATA motifs detected in file\n",
      "\n",
      "Complement found from 6898 to 6906\n",
      "Seq is: GAAATGAT\n",
      "1 GAAATGAT motifs detected in file\n",
      "\n",
      "Seq is: ATTTTCGA\n",
      "0 ATTTTCGA motifs detected in file\n",
      "\n",
      "Complement found from 441 to 449\n",
      "Seq is: TAACACAT\n",
      "1 TAACACAT motifs detected in file\n",
      "\n",
      "Seq is: TCGATCAG\n",
      "0 TCGATCAG motifs detected in file\n",
      "\n",
      "Seq is: TATAATTC\n",
      "0 TATAATTC motifs detected in file\n",
      "\n",
      "Complement found from 8640 to 8648\n",
      "Complement found from 9665 to 9673\n",
      "Complement found from 16076 to 16084\n",
      "Seq is: AACAACCG\n",
      "3 AACAACCG motifs detected in file\n",
      "\n",
      "Complement found from 7803 to 7811\n",
      "Seq is: AGTCCTCA\n",
      "1 AGTCCTCA motifs detected in file\n",
      "\n",
      "Complement found from 11312 to 11320\n",
      "Complement found from 16225 to 16233\n",
      "Seq is: AACTATCA\n",
      "2 AACTATCA motifs detected in file\n",
      "\n",
      "Complement found from 7420 to 7428\n",
      "Seq is: AGAACCCG\n",
      "1 AGAACCCG motifs detected in file\n",
      "\n",
      "Complement found from 4932 to 4940\n",
      "Complement found from 11294 to 11302\n",
      "Complement found from 11930 to 11938\n",
      "Seq is: TCACTCTC\n",
      "3 TCACTCTC motifs detected in file\n",
      "\n",
      "Complement found from 4519 to 4527\n",
      "Complement found from 4746 to 4754\n",
      "Complement found from 6981 to 6989\n",
      "Seq is: ACTCATCA\n",
      "3 ACTCATCA motifs detected in file\n",
      "\n",
      "Seq is: TCTTACAA\n",
      "0 TCTTACAA motifs detected in file\n",
      "\n",
      "Complement found from 14950 to 14958\n",
      "Seq is: ATCACTCG\n",
      "1 ATCACTCG motifs detected in file\n",
      "\n",
      "Seq is: GATCCCAC\n",
      "0 GATCCCAC motifs detected in file\n",
      "\n",
      "Complement found from 7716 to 7724\n",
      "Complement found from 14397 to 14405\n",
      "Seq is: AACACTCA\n",
      "2 AACACTCA motifs detected in file\n",
      "\n",
      "Seq is: CATTTAAT\n",
      "0 CATTTAAT motifs detected in file\n",
      "\n",
      "Complement found from 5467 to 5475\n",
      "Seq is: CACGCTAC\n",
      "1 CACGCTAC motifs detected in file\n",
      "\n",
      "Complement found from 5298 to 5306\n",
      "Seq is: TCATCCCC\n",
      "1 TCATCCCC motifs detected in file\n",
      "\n",
      "Seq is: TTCCCTCG\n",
      "0 TTCCCTCG motifs detected in file\n",
      "\n",
      "Seq is: GATGATCC\n",
      "0 GATGATCC motifs detected in file\n",
      "\n",
      "Seq is: CACATCTT\n",
      "0 CACATCTT motifs detected in file\n",
      "\n",
      "Seq is: ATCCTTCA\n",
      "0 ATCCTTCA motifs detected in file\n",
      "\n",
      "Seq is: TACACGGA\n",
      "0 TACACGGA motifs detected in file\n",
      "\n",
      "Seq is: CGCAAGTC\n",
      "0 CGCAAGTC motifs detected in file\n",
      "\n",
      "Seq is: CGCAAGAC\n",
      "0 CGCAAGAC motifs detected in file\n",
      "\n",
      "Complement found from 4151 to 4159\n",
      "Seq is: CGACCAAC\n",
      "1 CGACCAAC motifs detected in file\n",
      "\n",
      "Seq is: GGAGTAGC\n",
      "0 GGAGTAGC motifs detected in file\n",
      "\n",
      "Complement found from 12969 to 12977\n",
      "Seq is: CCACTACT\n",
      "1 CCACTACT motifs detected in file\n",
      "\n",
      "Seq is: ATCGGTTC\n",
      "0 ATCGGTTC motifs detected in file\n",
      "\n",
      "Seq is: CACCCCTA\n",
      "0 CACCCCTA motifs detected in file\n",
      "\n",
      "Seq is: TTCACACC\n",
      "0 TTCACACC motifs detected in file\n",
      "\n",
      "Seq is: AAAGCTTC\n",
      "0 AAAGCTTC motifs detected in file\n",
      "\n",
      "Seq is: TTCTATAT\n",
      "0 TTCTATAT motifs detected in file\n",
      "\n",
      "Complement found from 5492 to 5500\n",
      "Seq is: TTTATACT\n",
      "1 TTTATACT motifs detected in file\n",
      "\n",
      "Seq is: AATCAAGA\n",
      "0 AATCAAGA motifs detected in file\n",
      "\n",
      "Seq is: CACCGACA\n",
      "0 CACCGACA motifs detected in file\n",
      "\n",
      "Seq is: CTTACAAT\n",
      "0 CTTACAAT motifs detected in file\n",
      "\n",
      "Complement found from 8220 to 8228\n",
      "Seq is: ATTAATTC\n",
      "1 ATTAATTC motifs detected in file\n",
      "\n",
      "Seq is: CTCTAAAC\n",
      "0 CTCTAAAC motifs detected in file\n",
      "\n",
      "Complement found from 1187 to 1195\n",
      "Complement found from 10327 to 10335\n",
      "Complement found from 10867 to 10875\n",
      "Seq is: ATCCCTCT\n",
      "3 ATCCCTCT motifs detected in file\n",
      "\n",
      "Complement found from 4888 to 4896\n",
      "Complement found from 10454 to 10462\n",
      "Complement found from 11103 to 11111\n",
      "Seq is: AATCATAT\n",
      "3 AATCATAT motifs detected in file\n",
      "\n",
      "Complement found from 38 to 46\n",
      "Seq is: CTCTCCAT\n",
      "1 CTCTCCAT motifs detected in file\n",
      "\n",
      "Seq is: CTCAAAAA\n",
      "0 CTCAAAAA motifs detected in file\n",
      "\n",
      "Complement found from 13082 to 13090\n",
      "Seq is: AAGCACTA\n",
      "1 AAGCACTA motifs detected in file\n",
      "\n",
      "Complement found from 4084 to 4092\n",
      "Complement found from 14402 to 14410\n",
      "Seq is: TCACCAAG\n",
      "2 TCACCAAG motifs detected in file\n",
      "\n",
      "Seq is: TGACCTAT\n",
      "0 TGACCTAT motifs detected in file\n",
      "\n",
      "Seq is: TCACCGCA\n",
      "0 TCACCGCA motifs detected in file\n",
      "\n",
      "Complement found from 3372 to 3380\n",
      "Seq is: ACCGAACG\n",
      "1 ACCGAACG motifs detected in file\n",
      "\n",
      "Seq is: GTACTAAC\n",
      "0 GTACTAAC motifs detected in file\n",
      "\n",
      "Complement found from 5380 to 5388\n",
      "Seq is: ACTCCCCA\n",
      "1 ACTCCCCA motifs detected in file\n",
      "\n",
      "Seq is: TCCTCAGT\n",
      "0 TCCTCAGT motifs detected in file\n",
      "\n",
      "Complement found from 6384 to 6392\n",
      "Seq is: CCATCAAT\n",
      "1 CCATCAAT motifs detected in file\n",
      "\n",
      "Complement found from 4669 to 4677\n",
      "Complement found from 11781 to 11789\n",
      "Seq is: CATAATCC\n",
      "2 CATAATCC motifs detected in file\n",
      "\n",
      "Complement found from 12383 to 12391\n",
      "Seq is: TCCCCCCA\n",
      "1 TCCCCCCA motifs detected in file\n",
      "\n",
      "Complement found from 13776 to 13784\n",
      "Complement found from 13941 to 13949\n",
      "Seq is: ACAATCCC\n",
      "2 ACAATCCC motifs detected in file\n",
      "\n",
      "Seq is: CCAGCCTC\n",
      "0 CCAGCCTC motifs detected in file\n",
      "\n",
      "Seq is: CTTTTCCA\n",
      "0 CTTTTCCA motifs detected in file\n",
      "\n",
      "Seq is: ACCCCTTG\n",
      "0 ACCCCTTG motifs detected in file\n",
      "\n",
      "Seq is: TCGATTTA\n",
      "0 TCGATTTA motifs detected in file\n",
      "\n",
      "Seq is: TCCAACAA\n",
      "0 TCCAACAA motifs detected in file\n",
      "\n",
      "Complement found from 8371 to 8379\n",
      "Complement found from 8437 to 8445\n",
      "Seq is: CAACTAAA\n",
      "2 CAACTAAA motifs detected in file\n",
      "\n",
      "Seq is: CCAATTTT\n",
      "0 CCAATTTT motifs detected in file\n",
      "\n",
      "Complement found from 9094 to 9102\n",
      "Seq is: TTATCATC\n",
      "1 TTATCATC motifs detected in file\n",
      "\n",
      "Complement found from 10553 to 10561\n",
      "Seq is: TCCCTACT\n",
      "1 TCCCTACT motifs detected in file\n",
      "\n",
      "Seq is: ACGATTAT\n",
      "0 ACGATTAT motifs detected in file\n",
      "\n",
      "Seq is: TAATCCGA\n",
      "0 TAATCCGA motifs detected in file\n",
      "\n",
      "Complement found from 3207 to 3215\n",
      "Complement found from 12015 to 12023\n",
      "Seq is: CACCCACC\n",
      "2 CACCCACC motifs detected in file\n",
      "\n",
      "Complement found from 10834 to 10842\n",
      "Seq is: AACACAAC\n",
      "1 AACACAAC motifs detected in file\n",
      "\n",
      "Seq is: TAAGTTTA\n",
      "0 TAAGTTTA motifs detected in file\n",
      "\n",
      "Complement found from 3986 to 3994\n",
      "Complement found from 8703 to 8711\n",
      "Complement found from 13541 to 13549\n",
      "Seq is: ATACACAA\n",
      "3 ATACACAA motifs detected in file\n",
      "\n",
      "Complement found from 2131 to 2139\n",
      "Complement found from 15933 to 15941\n",
      "Seq is: AAAACCTT\n",
      "2 AAAACCTT motifs detected in file\n",
      "\n",
      "Seq is: TCAGTACA\n",
      "0 TCAGTACA motifs detected in file\n",
      "\n",
      "Seq is: GTCACCAT\n",
      "0 GTCACCAT motifs detected in file\n",
      "\n",
      "Seq is: CATTATAT\n",
      "0 CATTATAT motifs detected in file\n",
      "\n",
      "Complement found from 10940 to 10948\n",
      "Complement found from 12737 to 12745\n",
      "Seq is: TAACAACC\n",
      "2 TAACAACC motifs detected in file\n",
      "\n",
      "Seq is: CTGCTAAT\n",
      "0 CTGCTAAT motifs detected in file\n",
      "\n",
      "Seq is: CAAAATCG\n",
      "0 CAAAATCG motifs detected in file\n",
      "\n",
      "Complement found from 8680 to 8688\n",
      "Seq is: TAACCTCA\n",
      "1 TAACCTCA motifs detected in file\n",
      "\n",
      "Complement found from 14873 to 14881\n",
      "Seq is: TCCAAATC\n",
      "1 TCCAAATC motifs detected in file\n",
      "\n",
      "Seq is: CAATACAT\n",
      "0 CAATACAT motifs detected in file\n",
      "\n",
      "Seq is: GCATCAGA\n",
      "0 GCATCAGA motifs detected in file\n",
      "\n",
      "Seq is: TCCGGTAT\n",
      "0 TCCGGTAT motifs detected in file\n",
      "\n",
      "Complement found from 10835 to 10843\n",
      "Seq is: ACACAACC\n",
      "1 ACACAACC motifs detected in file\n",
      "\n",
      "Seq is: TCTAACTC\n",
      "0 TCTAACTC motifs detected in file\n",
      "\n",
      "Seq is: TGATCATC\n",
      "0 TGATCATC motifs detected in file\n",
      "\n",
      "Complement found from 15588 to 15596\n",
      "Seq is: CCGATCCG\n",
      "1 CCGATCCG motifs detected in file\n",
      "\n",
      "Seq is: GGTGGCGA\n",
      "0 GGTGGCGA motifs detected in file\n",
      "\n",
      "Seq is: CGAAGCGT\n",
      "0 CGAAGCGT motifs detected in file\n",
      "\n",
      "Seq is: CCGCCGTT\n",
      "0 CCGCCGTT motifs detected in file\n",
      "\n",
      "Complement found from 3655 to 3663\n",
      "Seq is: TCTGATCA\n",
      "1 TCTGATCA motifs detected in file\n",
      "\n",
      "Complement found from 10009 to 10017\n",
      "Seq is: TACCGTTA\n",
      "1 TACCGTTA motifs detected in file\n",
      "\n",
      "Seq is: TCCGTGTT\n",
      "0 TCCGTGTT motifs detected in file\n",
      "\n",
      "Seq is: ATAACCGA\n",
      "0 ATAACCGA motifs detected in file\n",
      "\n",
      "Seq is: TTCTCCCT\n",
      "0 TTCTCCCT motifs detected in file\n",
      "\n",
      "Complement found from 10842 to 10850\n",
      "Complement found from 14355 to 14363\n",
      "Seq is: CACCCACA\n",
      "2 CACCCACA motifs detected in file\n",
      "\n",
      "Seq is: ACTCCCAA\n",
      "0 ACTCCCAA motifs detected in file\n",
      "\n",
      "Complement found from 10600 to 10608\n",
      "Complement found from 13568 to 13576\n",
      "Seq is: TACTCTCA\n",
      "2 TACTCTCA motifs detected in file\n",
      "\n",
      "Seq is: TCGATATT\n",
      "0 TCGATATT motifs detected in file\n",
      "\n",
      "Seq is: ACTTGTCG\n",
      "0 ACTTGTCG motifs detected in file\n",
      "\n",
      "Complement found from 7146 to 7154\n",
      "Complement found from 7628 to 7636\n",
      "Complement found from 9226 to 9234\n",
      "Seq is: CTATCATA\n",
      "3 CTATCATA motifs detected in file\n",
      "\n",
      "Complement found from 2443 to 2451\n",
      "Seq is: ATAAGGAA\n",
      "1 ATAAGGAA motifs detected in file\n",
      "\n",
      "Seq is: GATCCGTA\n",
      "0 GATCCGTA motifs detected in file\n",
      "\n",
      "Complement found from 10325 to 10333\n",
      "Complement found from 10865 to 10873\n",
      "Complement found from 12601 to 12609\n",
      "Seq is: TCATCCCT\n",
      "3 TCATCCCT motifs detected in file\n",
      "\n",
      "Complement found from 11970 to 11978\n",
      "Seq is: CCTATACT\n",
      "1 CCTATACT motifs detected in file\n",
      "\n",
      "Complement found from 5448 to 5456\n",
      "Complement found from 6876 to 6884\n",
      "Complement found from 14642 to 14650\n",
      "Seq is: CCACACTC\n",
      "3 CCACACTC motifs detected in file\n",
      "\n",
      "Seq is: GCTATATG\n",
      "0 GCTATATG motifs detected in file\n",
      "\n",
      "Seq is: ATCATAAG\n",
      "0 ATCATAAG motifs detected in file\n",
      "\n",
      "Seq is: GATCTTCA\n",
      "0 GATCTTCA motifs detected in file\n",
      "\n",
      "Complement found from 4498 to 4506\n",
      "Seq is: CTCTACCA\n",
      "1 CTCTACCA motifs detected in file\n",
      "\n",
      "Seq is: TTTACGAT\n",
      "0 TTTACGAT motifs detected in file\n",
      "\n",
      "Seq is: CATATTAT\n",
      "0 CATATTAT motifs detected in file\n",
      "\n",
      "Seq is: AAATACAT\n",
      "0 AAATACAT motifs detected in file\n",
      "\n",
      "Seq is: TTACTCCC\n",
      "0 TTACTCCC motifs detected in file\n",
      "\n",
      "Seq is: CGTAAACC\n",
      "0 CGTAAACC motifs detected in file\n",
      "\n",
      "Seq is: ATTTATAC\n",
      "0 ATTTATAC motifs detected in file\n",
      "\n",
      "Complement found from 7350 to 7358\n",
      "Seq is: TAATAGTA\n",
      "1 TAATAGTA motifs detected in file\n",
      "\n",
      "Complement found from 7554 to 7562\n",
      "Complement found from 10442 to 10450\n",
      "Seq is: TTAAATTA\n",
      "2 TTAAATTA motifs detected in file\n",
      "\n",
      "Seq is: CTCAGCTT\n",
      "0 CTCAGCTT motifs detected in file\n",
      "\n",
      "Complement found from 9175 to 9183\n",
      "Seq is: TAGTAAGC\n",
      "1 TAGTAAGC motifs detected in file\n",
      "\n",
      "Complement found from 357 to 365\n",
      "Complement found from 8687 to 8695\n",
      "Complement found from 15863 to 15871\n",
      "Seq is: AAAACAAA\n",
      "3 AAAACAAA motifs detected in file\n",
      "\n",
      "Seq is: TTTGATAT\n",
      "0 TTTGATAT motifs detected in file\n",
      "\n",
      "Seq is: ATGGTTAA\n",
      "0 ATGGTTAA motifs detected in file\n",
      "\n",
      "Seq is: GCCAAGTC\n",
      "0 GCCAAGTC motifs detected in file\n",
      "\n",
      "Seq is: AGATTAGG\n",
      "0 AGATTAGG motifs detected in file\n",
      "\n",
      "Complement found from 2502 to 2510\n",
      "Seq is: AAAAACAT\n",
      "1 AAAAACAT motifs detected in file\n",
      "\n",
      "Complement found from 712 to 720\n",
      "Seq is: CAGTGAGT\n",
      "1 CAGTGAGT motifs detected in file\n",
      "\n",
      "Seq is: ATCCGGTC\n",
      "0 ATCCGGTC motifs detected in file\n",
      "\n",
      "Seq is: TGAATCCC\n",
      "0 TGAATCCC motifs detected in file\n",
      "\n",
      "Seq is: GATCCTAC\n",
      "0 GATCCTAC motifs detected in file\n",
      "\n",
      "Complement found from 4464 to 4472\n",
      "Seq is: TACTAATT\n",
      "1 TACTAATT motifs detected in file\n",
      "\n",
      "Seq is: ATTCTCCC\n",
      "0 ATTCTCCC motifs detected in file\n",
      "\n",
      "Seq is: TACTGTAT\n",
      "0 TACTGTAT motifs detected in file\n",
      "\n",
      "Complement found from 10450 to 10458\n",
      "Seq is: TGATAATC\n",
      "1 TGATAATC motifs detected in file\n",
      "\n",
      "Seq is: ACCGTCCA\n",
      "0 ACCGTCCA motifs detected in file\n",
      "\n",
      "Complement found from 1117 to 1125\n",
      "Complement found from 2964 to 2972\n",
      "Complement found from 8637 to 8645\n",
      "Complement found from 10891 to 10899\n",
      "Complement found from 16073 to 16081\n",
      "Seq is: ATCAACAA\n",
      "5 ATCAACAA motifs detected in file\n",
      "\n",
      "Complement found from 3002 to 3010\n",
      "Seq is: ACATCCCG\n",
      "1 ACATCCCG motifs detected in file\n",
      "\n",
      "Seq is: AGTACCAT\n",
      "0 AGTACCAT motifs detected in file\n",
      "\n",
      "Seq is: CCCCATTT\n",
      "0 CCCCATTT motifs detected in file\n",
      "\n",
      "Complement found from 14260 to 14268\n",
      "Complement found from 15140 to 15148\n",
      "Seq is: TCCTCCCG\n",
      "2 TCCTCCCG motifs detected in file\n",
      "\n",
      "Complement found from 8328 to 8336\n",
      "Seq is: TTAAAGAT\n",
      "1 TTAAAGAT motifs detected in file\n",
      "\n",
      "Seq is: CTAGTTTA\n",
      "0 CTAGTTTA motifs detected in file\n",
      "\n",
      "Seq is: TTATTCTT\n",
      "0 TTATTCTT motifs detected in file\n",
      "\n",
      "Complement found from 5262 to 5270\n",
      "Seq is: CCATTATC\n",
      "1 CCATTATC motifs detected in file\n",
      "\n",
      "Seq is: GATGATTC\n",
      "0 GATGATTC motifs detected in file\n",
      "\n",
      "Seq is: TTCTTAAA\n",
      "0 TTCTTAAA motifs detected in file\n",
      "\n",
      "Complement found from 11377 to 11385\n",
      "Complement found from 13431 to 13439\n",
      "Seq is: ATACCTCT\n",
      "2 ATACCTCT motifs detected in file\n",
      "\n",
      "Seq is: TTCCCTGC\n",
      "0 TTCCCTGC motifs detected in file\n",
      "\n",
      "Seq is: GCCCGCCC\n",
      "0 GCCCGCCC motifs detected in file\n",
      "\n",
      "Complement found from 10184 to 10192\n",
      "Seq is: CCTATATC\n",
      "1 CCTATATC motifs detected in file\n",
      "\n",
      "Seq is: CCAGCTTC\n",
      "0 CCAGCTTC motifs detected in file\n",
      "\n",
      "Seq is: GGCGTGAG\n",
      "0 GGCGTGAG motifs detected in file\n",
      "\n",
      "Complement found from 4244 to 4252\n",
      "Complement found from 14484 to 14492\n",
      "Seq is: CATTCCCC\n",
      "2 CATTCCCC motifs detected in file\n",
      "\n",
      "Complement found from 2222 to 2230\n",
      "Complement found from 11600 to 11608\n",
      "Complement found from 13789 to 13797\n",
      "Complement found from 14021 to 14029\n",
      "Seq is: ACCTAAAA\n",
      "4 ACCTAAAA motifs detected in file\n",
      "\n",
      "Seq is: AGATACAT\n",
      "0 AGATACAT motifs detected in file\n",
      "\n",
      "Seq is: CGGCAACT\n",
      "0 CGGCAACT motifs detected in file\n",
      "\n",
      "Complement found from 6547 to 6555\n",
      "Seq is: CAACACCA\n",
      "1 CAACACCA motifs detected in file\n",
      "\n",
      "Seq is: TCAGTTTT\n",
      "0 TCAGTTTT motifs detected in file\n",
      "\n",
      "Complement found from 445 to 453\n",
      "Complement found from 3994 to 4002\n",
      "Seq is: ACATTATT\n",
      "2 ACATTATT motifs detected in file\n",
      "\n",
      "Complement found from 4756 to 4764\n",
      "Complement found from 10078 to 10086\n",
      "Seq is: AATAATCA\n",
      "2 AATAATCA motifs detected in file\n",
      "\n",
      "Seq is: TCATTCCG\n",
      "0 TCATTCCG motifs detected in file\n",
      "\n",
      "Complement found from 10549 to 10557\n",
      "Complement found from 14259 to 14267\n",
      "Seq is: ATCCTCCC\n",
      "2 ATCCTCCC motifs detected in file\n",
      "\n",
      "Complement found from 8766 to 8774\n",
      "Complement found from 10298 to 10306\n",
      "Seq is: ACAACTAA\n",
      "2 ACAACTAA motifs detected in file\n",
      "\n",
      "Complement found from 10071 to 10079\n",
      "Seq is: TAATTTTA\n",
      "1 TAATTTTA motifs detected in file\n",
      "\n",
      "Complement found from 4142 to 4150\n",
      "Seq is: ATTCCGCT\n",
      "1 ATTCCGCT motifs detected in file\n",
      "\n",
      "Seq is: TCAAATAA\n",
      "0 TCAAATAA motifs detected in file\n",
      "\n",
      "Seq is: TGAGAAAA\n",
      "0 TGAGAAAA motifs detected in file\n",
      "\n",
      "Complement found from 1628 to 1636\n",
      "Seq is: AACTTACA\n",
      "1 AACTTACA motifs detected in file\n",
      "\n",
      "Seq is: ACAGTTTT\n",
      "0 ACAGTTTT motifs detected in file\n",
      "\n",
      "Seq is: GATCAATT\n",
      "0 GATCAATT motifs detected in file\n",
      "\n",
      "Seq is: AACCTTCA\n",
      "0 AACCTTCA motifs detected in file\n",
      "\n",
      "Complement found from 10649 to 10657\n",
      "Seq is: ATTGCCAT\n",
      "1 ATTGCCAT motifs detected in file\n",
      "\n",
      "Seq is: GATAAATA\n",
      "0 GATAAATA motifs detected in file\n",
      "\n",
      "Seq is: TGATTTTC\n",
      "0 TGATTTTC motifs detected in file\n",
      "\n",
      "Complement found from 15160 to 15168\n",
      "Seq is: TCATTCTG\n",
      "1 TCATTCTG motifs detected in file\n",
      "\n",
      "Seq is: GGAGTAGT\n",
      "0 GGAGTAGT motifs detected in file\n",
      "\n",
      "Seq is: TATCTACC\n",
      "0 TATCTACC motifs detected in file\n",
      "\n",
      "Complement found from 9958 to 9966\n",
      "Seq is: TCTGTATG\n",
      "1 TCTGTATG motifs detected in file\n",
      "\n",
      "Complement found from 15014 to 15022\n",
      "Seq is: TCTTTATC\n",
      "1 TCTTTATC motifs detected in file\n",
      "\n",
      "Seq is: AGTTTGGT\n",
      "0 AGTTTGGT motifs detected in file\n",
      "\n",
      "Complement found from 9938 to 9946\n",
      "Seq is: GTAGATGT\n",
      "1 GTAGATGT motifs detected in file\n",
      "\n",
      "Seq is: TTTACGGT\n",
      "0 TTTACGGT motifs detected in file\n",
      "\n",
      "Seq is: CATAGTCC\n",
      "0 CATAGTCC motifs detected in file\n",
      "\n",
      "Seq is: GCCGCCGA\n",
      "0 GCCGCCGA motifs detected in file\n",
      "\n",
      "Seq is: AGCTTCGG\n",
      "0 AGCTTCGG motifs detected in file\n",
      "\n",
      "Complement found from 7141 to 7149\n",
      "Seq is: TTTCACTA\n",
      "1 TTTCACTA motifs detected in file\n",
      "\n",
      "Complement found from 1991 to 1999\n",
      "Complement found from 4254 to 4262\n",
      "Complement found from 7121 to 7129\n",
      "Complement found from 16281 to 16289\n",
      "Seq is: CAAACCTA\n",
      "4 CAAACCTA motifs detected in file\n",
      "\n",
      "Complement found from 7139 to 7147\n",
      "Seq is: CATTTCAC\n",
      "1 CATTTCAC motifs detected in file\n",
      "\n",
      "Seq is: TTTATAAT\n",
      "0 TTTATAAT motifs detected in file\n",
      "\n",
      "Complement found from 14924 to 14932\n",
      "Seq is: CAACCGCC\n",
      "1 CAACCGCC motifs detected in file\n",
      "\n",
      "Seq is: TACTTCGT\n",
      "0 TACTTCGT motifs detected in file\n",
      "\n",
      "Seq is: CTATCACT\n",
      "0 CTATCACT motifs detected in file\n",
      "\n",
      "Seq is: CCTTTCAA\n",
      "0 CCTTTCAA motifs detected in file\n",
      "\n",
      "Seq is: CTCGGTTA\n",
      "0 CTCGGTTA motifs detected in file\n",
      "\n",
      "Complement found from 9016 to 9024\n",
      "Seq is: TTACTGCA\n",
      "1 TTACTGCA motifs detected in file\n",
      "\n",
      "Seq is: CTTCAGGC\n",
      "0 CTTCAGGC motifs detected in file\n",
      "\n",
      "Seq is: ACCTTCGG\n",
      "0 ACCTTCGG motifs detected in file\n",
      "\n",
      "Seq is: ACACCGAT\n",
      "0 ACACCGAT motifs detected in file\n",
      "\n",
      "Seq is: GTTTTTTA\n",
      "0 GTTTTTTA motifs detected in file\n",
      "\n",
      "Seq is: CAACTCGG\n",
      "0 CAACTCGG motifs detected in file\n",
      "\n",
      "Complement found from 9778 to 9786\n",
      "Seq is: CATCTACG\n",
      "1 CATCTACG motifs detected in file\n",
      "\n",
      "Seq is: GCAGCCTT\n",
      "0 GCAGCCTT motifs detected in file\n",
      "\n",
      "Seq is: TACCACTT\n",
      "0 TACCACTT motifs detected in file\n",
      "\n",
      "Complement found from 4834 to 4842\n",
      "Seq is: CCCTCTGA\n",
      "1 CCCTCTGA motifs detected in file\n",
      "\n",
      "Seq is: GCTTCATG\n",
      "0 GCTTCATG motifs detected in file\n",
      "\n",
      "Seq is: AGACTCCG\n",
      "0 AGACTCCG motifs detected in file\n",
      "\n",
      "Complement found from 7253 to 7261\n",
      "Seq is: AACATCCT\n",
      "1 AACATCCT motifs detected in file\n",
      "\n",
      "Seq is: CCCATTTT\n",
      "0 CCCATTTT motifs detected in file\n",
      "\n",
      "Seq is: ATCTCTGG\n",
      "0 ATCTCTGG motifs detected in file\n",
      "\n",
      "Seq is: GTCATTTT\n",
      "0 GTCATTTT motifs detected in file\n",
      "\n",
      "Complement found from 3993 to 4001\n",
      "Seq is: AACATTAT\n",
      "1 AACATTAT motifs detected in file\n",
      "\n",
      "Seq is: TCGTCACG\n",
      "0 TCGTCACG motifs detected in file\n",
      "\n",
      "Complement found from 11335 to 11343\n",
      "Seq is: AACTTAAT\n",
      "1 AACTTAAT motifs detected in file\n",
      "\n",
      "Complement found from 4978 to 4986\n",
      "Complement found from 8121 to 8129\n",
      "Complement found from 9673 to 9681\n",
      "Seq is: AAACCAAA\n",
      "3 AAACCAAA motifs detected in file\n",
      "\n",
      "Complement found from 11329 to 11337\n",
      "Seq is: GCCAACAA\n",
      "1 GCCAACAA motifs detected in file\n",
      "\n",
      "Seq is: AAGATAAT\n",
      "0 AAGATAAT motifs detected in file\n",
      "\n",
      "Seq is: CTGATACC\n",
      "0 CTGATACC motifs detected in file\n",
      "\n",
      "Seq is: ACTCGAGT\n",
      "0 ACTCGAGT motifs detected in file\n",
      "\n",
      "Seq is: CCACTAAC\n",
      "0 CCACTAAC motifs detected in file\n",
      "\n",
      "Seq is: TATGAGGA\n",
      "0 TATGAGGA motifs detected in file\n",
      "\n",
      "Seq is: CTACGCTC\n",
      "0 CTACGCTC motifs detected in file\n",
      "\n",
      "Seq is: ATTATGCC\n",
      "0 ATTATGCC motifs detected in file\n",
      "\n",
      "Complement found from 3987 to 3995\n",
      "Complement found from 13214 to 13222\n",
      "Complement found from 13542 to 13550\n",
      "Seq is: TACACAAA\n",
      "3 TACACAAA motifs detected in file\n",
      "\n",
      "Seq is: TCCTCACC\n",
      "0 TCCTCACC motifs detected in file\n",
      "\n",
      "Seq is: CTGAAGAT\n",
      "0 CTGAAGAT motifs detected in file\n",
      "\n",
      "Complement found from 3483 to 3491\n",
      "Complement found from 8227 to 8235\n",
      "Complement found from 14489 to 14497\n",
      "Seq is: CCCCTAAA\n",
      "3 CCCCTAAA motifs detected in file\n",
      "\n",
      "Seq is: TCGCCCCA\n",
      "0 TCGCCCCA motifs detected in file\n",
      "\n",
      "Seq is: CTACGGAC\n",
      "0 CTACGGAC motifs detected in file\n",
      "\n",
      "Complement found from 491 to 499\n",
      "Seq is: AACCCCCG\n",
      "1 AACCCCCG motifs detected in file\n",
      "\n",
      "Seq is: GTCACGGG\n",
      "0 GTCACGGG motifs detected in file\n",
      "\n",
      "Seq is: AGGATTAA\n",
      "0 AGGATTAA motifs detected in file\n",
      "\n",
      "Complement found from 12384 to 12392\n",
      "Seq is: CCCCCCAT\n",
      "1 CCCCCCAT motifs detected in file\n",
      "\n",
      "Complement found from 1222 to 1230\n",
      "Seq is: CCCCGATC\n",
      "1 CCCCGATC motifs detected in file\n",
      "\n",
      "Seq is: CGACCTCA\n",
      "0 CGACCTCA motifs detected in file\n",
      "\n",
      "Seq is: CCATTTTC\n",
      "0 CCATTTTC motifs detected in file\n",
      "\n",
      "Seq is: CGAGTCTT\n",
      "0 CGAGTCTT motifs detected in file\n",
      "\n",
      "Seq is: TTTAGGAC\n",
      "0 TTTAGGAC motifs detected in file\n",
      "\n",
      "Seq is: GCTTCTTT\n",
      "0 GCTTCTTT motifs detected in file\n",
      "\n",
      "Seq is: TTTTGAAG\n",
      "0 TTTTGAAG motifs detected in file\n",
      "\n",
      "Seq is: ACTCCATT\n",
      "0 ACTCCATT motifs detected in file\n",
      "\n",
      "Complement found from 12899 to 12907\n",
      "Seq is: ATTTATCC\n",
      "1 ATTTATCC motifs detected in file\n",
      "\n",
      "Seq is: TAATAGGG\n",
      "0 TAATAGGG motifs detected in file\n",
      "\n",
      "Seq is: CATAGCTT\n",
      "0 CATAGCTT motifs detected in file\n",
      "\n",
      "Complement found from 6687 to 6695\n",
      "Seq is: CCGGAAAA\n",
      "1 CCGGAAAA motifs detected in file\n",
      "\n",
      "Complement found from 9424 to 9432\n",
      "Seq is: ACCTGTCC\n",
      "1 ACCTGTCC motifs detected in file\n",
      "\n",
      "Complement found from 7407 to 7415\n",
      "Complement found from 9414 to 9422\n",
      "Seq is: ACCACACA\n",
      "2 ACCACACA motifs detected in file\n",
      "\n",
      "Seq is: CCACCGGA\n",
      "0 CCACCGGA motifs detected in file\n",
      "\n",
      "Complement found from 8700 to 8708\n",
      "Seq is: ACCATACA\n",
      "1 ACCATACA motifs detected in file\n",
      "\n",
      "Seq is: CGAAAGAG\n",
      "0 CGAAAGAG motifs detected in file\n",
      "\n",
      "Seq is: CACAATGT\n",
      "0 CACAATGT motifs detected in file\n",
      "\n",
      "Seq is: AGCGCGGT\n",
      "0 AGCGCGGT motifs detected in file\n",
      "\n",
      "Complement found from 11904 to 11912\n",
      "Seq is: AGTAACCA\n",
      "1 AGTAACCA motifs detected in file\n",
      "\n",
      "Complement found from 4893 to 4901\n",
      "Complement found from 6591 to 6599\n",
      "Complement found from 9366 to 9374\n",
      "Seq is: TATACCAA\n",
      "3 TATACCAA motifs detected in file\n",
      "\n",
      "Seq is: TCACACAA\n",
      "0 TCACACAA motifs detected in file\n",
      "\n",
      "Seq is: CCAATCAT\n",
      "0 CCAATCAT motifs detected in file\n",
      "\n",
      "Seq is: CCGGATCA\n",
      "0 CCGGATCA motifs detected in file\n",
      "\n",
      "Complement found from 3325 to 3333\n",
      "Complement found from 5009 to 5017\n",
      "Complement found from 13984 to 13992\n",
      "Complement found from 14440 to 14448\n",
      "Seq is: TACTCCTC\n",
      "4 TACTCCTC motifs detected in file\n",
      "\n",
      "Seq is: GCAATACC\n",
      "0 GCAATACC motifs detected in file\n",
      "\n",
      "Complement found from 15398 to 15406\n",
      "Seq is: TCACCTTC\n",
      "1 TCACCTTC motifs detected in file\n",
      "\n",
      "Seq is: ACTTTAGT\n",
      "0 ACTTTAGT motifs detected in file\n",
      "\n",
      "Seq is: GTACCGAT\n",
      "0 GTACCGAT motifs detected in file\n",
      "\n",
      "Seq is: CCGGCCTC\n",
      "0 CCGGCCTC motifs detected in file\n",
      "\n",
      "Seq is: CAGTAATC\n",
      "0 CAGTAATC motifs detected in file\n",
      "\n",
      "Seq is: CTCCCGAC\n",
      "0 CTCCCGAC motifs detected in file\n",
      "\n",
      "Seq is: TCTCCCGG\n",
      "0 TCTCCCGG motifs detected in file\n",
      "\n",
      "Seq is: GGACAATC\n",
      "0 GGACAATC motifs detected in file\n",
      "\n",
      "Seq is: CCCAGTAC\n",
      "0 CCCAGTAC motifs detected in file\n",
      "\n",
      "Seq is: CCGACCCA\n",
      "0 CCGACCCA motifs detected in file\n",
      "\n",
      "Complement found from 3174 to 3182\n",
      "Complement found from 8692 to 8700\n",
      "Seq is: AAATGATA\n",
      "2 AAATGATA motifs detected in file\n",
      "\n",
      "Complement found from 11546 to 11554\n",
      "Complement found from 15191 to 15199\n",
      "Complement found from 15629 to 15637\n",
      "Seq is: TACTATCC\n",
      "3 TACTATCC motifs detected in file\n",
      "\n",
      "Seq is: GTACACTA\n",
      "0 GTACACTA motifs detected in file\n",
      "\n",
      "Complement found from 8655 to 8663\n",
      "Complement found from 8808 to 8816\n",
      "Complement found from 10840 to 10848\n",
      "Complement found from 16050 to 16058\n",
      "Seq is: ACCACCCA\n",
      "4 ACCACCCA motifs detected in file\n",
      "\n",
      "Seq is: GTAATACA\n",
      "0 GTAATACA motifs detected in file\n",
      "\n",
      "Seq is: CAACAGCA\n",
      "0 CAACAGCA motifs detected in file\n",
      "\n",
      "Seq is: CGTCCATC\n",
      "0 CGTCCATC motifs detected in file\n",
      "\n",
      "Seq is: TCCGAATG\n",
      "0 TCCGAATG motifs detected in file\n",
      "\n",
      "Complement found from 9099 to 9107\n",
      "Seq is: ATCTTCAC\n",
      "1 ATCTTCAC motifs detected in file\n",
      "\n",
      "Seq is: ACTTTTGC\n",
      "0 ACTTTTGC motifs detected in file\n",
      "\n",
      "Seq is: ATCCGAAC\n",
      "0 ATCCGAAC motifs detected in file\n",
      "\n",
      "Complement found from 12378 to 12386\n",
      "Seq is: CTAATTCC\n",
      "1 CTAATTCC motifs detected in file\n",
      "\n",
      "Complement found from 9138 to 9146\n",
      "Seq is: GCTGTCGC\n",
      "1 GCTGTCGC motifs detected in file\n",
      "\n",
      "Seq is: TAAAGATC\n",
      "0 TAAAGATC motifs detected in file\n",
      "\n",
      "Seq is: CTATCAGT\n",
      "0 CTATCAGT motifs detected in file\n",
      "\n",
      "Seq is: CATCTTAA\n",
      "0 CATCTTAA motifs detected in file\n",
      "\n",
      "Complement found from 3296 to 3304\n",
      "Seq is: TCTTAACA\n",
      "1 TCTTAACA motifs detected in file\n",
      "\n",
      "Complement found from 3552 to 3560\n",
      "Seq is: CTTCTACT\n",
      "1 CTTCTACT motifs detected in file\n",
      "\n",
      "Seq is: ATTCACAT\n",
      "0 ATTCACAT motifs detected in file\n",
      "\n",
      "Complement found from 9756 to 9764\n",
      "Complement found from 11226 to 11234\n",
      "Seq is: CTCCCTTC\n",
      "2 CTCCCTTC motifs detected in file\n",
      "\n",
      "Complement found from 5016 to 5024\n",
      "Seq is: CAATTACC\n",
      "1 CAATTACC motifs detected in file\n",
      "\n",
      "Seq is: AACTATAA\n",
      "0 AACTATAA motifs detected in file\n",
      "\n",
      "Seq is: CGATCCCA\n",
      "0 CGATCCCA motifs detected in file\n",
      "\n",
      "Seq is: CCGCGAAG\n",
      "0 CCGCGAAG motifs detected in file\n",
      "\n",
      "Seq is: GTTAATCC\n",
      "0 GTTAATCC motifs detected in file\n",
      "\n",
      "Seq is: ACGTACTC\n",
      "0 ACGTACTC motifs detected in file\n",
      "\n",
      "Seq is: ATCCACCG\n",
      "0 ATCCACCG motifs detected in file\n",
      "\n",
      "Seq is: GACGTCAT\n",
      "0 GACGTCAT motifs detected in file\n",
      "\n",
      "Complement found from 13280 to 13288\n",
      "Seq is: TACAATCG\n",
      "1 TACAATCG motifs detected in file\n",
      "\n",
      "Seq is: CCAATCCG\n",
      "0 CCAATCCG motifs detected in file\n",
      "\n",
      "Seq is: CATGCCGG\n",
      "0 CATGCCGG motifs detected in file\n",
      "\n",
      "Seq is: TCCCGATA\n",
      "0 TCCCGATA motifs detected in file\n",
      "\n",
      "Seq is: ACCAACTT\n",
      "0 ACCAACTT motifs detected in file\n",
      "\n",
      "Complement found from 13112 to 13120\n",
      "Complement found from 14117 to 14125\n",
      "Seq is: ACTCATCC\n",
      "2 ACTCATCC motifs detected in file\n",
      "\n",
      "Complement found from 10125 to 10133\n",
      "Seq is: GACTACCA\n",
      "1 GACTACCA motifs detected in file\n",
      "\n",
      "Complement found from 14013 to 14021\n",
      "Seq is: AAGCTATT\n",
      "1 AAGCTATT motifs detected in file\n",
      "\n",
      "Seq is: ATTGATCA\n",
      "0 ATTGATCA motifs detected in file\n",
      "\n",
      "Complement found from 10275 to 10283\n",
      "Seq is: TACCCCTA\n",
      "1 TACCCCTA motifs detected in file\n",
      "\n",
      "Complement found from 12480 to 12488\n",
      "Complement found from 13644 to 13652\n",
      "Seq is: TTCCCCAC\n",
      "2 TTCCCCAC motifs detected in file\n",
      "\n",
      "Seq is: ATCCACAC\n",
      "0 ATCCACAC motifs detected in file\n",
      "\n",
      "Seq is: GGAACACC\n",
      "0 GGAACACC motifs detected in file\n",
      "\n",
      "Complement found from 3971 to 3979\n",
      "Seq is: ATTCTTCA\n",
      "1 ATTCTTCA motifs detected in file\n",
      "\n",
      "Seq is: CCCGATCC\n",
      "0 CCCGATCC motifs detected in file\n",
      "\n",
      "Complement found from 5398 to 5406\n",
      "Seq is: CGTAAAAA\n",
      "1 CGTAAAAA motifs detected in file\n",
      "\n",
      "Seq is: TTAGAATC\n",
      "0 TTAGAATC motifs detected in file\n",
      "\n",
      "Seq is: TCGCTTTC\n",
      "0 TCGCTTTC motifs detected in file\n",
      "\n",
      "Seq is: GGATATTA\n",
      "0 GGATATTA motifs detected in file\n",
      "\n",
      "Seq is: GTGACACG\n",
      "0 GTGACACG motifs detected in file\n",
      "\n",
      "Seq is: GGCGAGTA\n",
      "0 GGCGAGTA motifs detected in file\n",
      "\n",
      "Complement found from 6150 to 6158\n",
      "Complement found from 7092 to 7100\n",
      "Complement found from 7623 to 7631\n",
      "Complement found from 8225 to 8233\n",
      "Complement found from 11231 to 11239\n",
      "Seq is: TTCCCCTA\n",
      "5 TTCCCCTA motifs detected in file\n",
      "\n",
      "Seq is: CCGGTACC\n",
      "0 CCGGTACC motifs detected in file\n",
      "\n",
      "Seq is: GATCCAAA\n",
      "0 GATCCAAA motifs detected in file\n",
      "\n",
      "Seq is: TATCTATC\n",
      "0 TATCTATC motifs detected in file\n",
      "\n",
      "Complement found from 5430 to 5438\n",
      "Seq is: AACCCACC\n",
      "1 AACCCACC motifs detected in file\n",
      "\n",
      "Seq is: AACCACAT\n",
      "0 AACCACAT motifs detected in file\n",
      "\n",
      "Seq is: TTACTCAC\n",
      "0 TTACTCAC motifs detected in file\n",
      "\n",
      "Complement found from 6461 to 6469\n",
      "Seq is: TCCGTCCT\n",
      "1 TCCGTCCT motifs detected in file\n",
      "\n",
      "Seq is: CAGGCTCC\n",
      "0 CAGGCTCC motifs detected in file\n",
      "\n",
      "Seq is: TCCAATCA\n",
      "0 TCCAATCA motifs detected in file\n",
      "\n",
      "Seq is: ACACCGTT\n",
      "0 ACACCGTT motifs detected in file\n",
      "\n",
      "Seq is: ATTTTTAC\n",
      "0 ATTTTTAC motifs detected in file\n",
      "\n",
      "Seq is: TAATTCCT\n",
      "0 TAATTCCT motifs detected in file\n",
      "\n",
      "Seq is: ATGATCAT\n",
      "0 ATGATCAT motifs detected in file\n",
      "\n",
      "Complement found from 5923 to 5931\n",
      "Seq is: ATTCTCTA\n",
      "1 ATTCTCTA motifs detected in file\n",
      "\n",
      "Seq is: GTCCAAGC\n",
      "0 GTCCAAGC motifs detected in file\n",
      "\n",
      "Seq is: AGGAAATC\n",
      "0 AGGAAATC motifs detected in file\n",
      "\n",
      "Complement found from 3804 to 3812\n",
      "Complement found from 9196 to 9204\n",
      "Complement found from 11995 to 12003\n",
      "Seq is: ACAACACA\n",
      "3 ACAACACA motifs detected in file\n",
      "\n",
      "Seq is: TACCAATA\n",
      "0 TACCAATA motifs detected in file\n",
      "\n",
      "Seq is: GTAAACAA\n",
      "0 GTAAACAA motifs detected in file\n",
      "\n",
      "Complement found from 12319 to 12327\n",
      "Complement found from 16245 to 16253\n",
      "Seq is: AACTCCAA\n",
      "2 AACTCCAA motifs detected in file\n",
      "\n",
      "Complement found from 3678 to 3686\n",
      "Complement found from 8674 to 8682\n",
      "Complement found from 11752 to 11760\n",
      "Complement found from 12574 to 12582\n",
      "Seq is: TCAAACTA\n",
      "4 TCAAACTA motifs detected in file\n",
      "\n",
      "Seq is: ATCAGTAA\n",
      "0 ATCAGTAA motifs detected in file\n",
      "\n",
      "Seq is: CAACCCAC\n",
      "0 CAACCCAC motifs detected in file\n",
      "\n",
      "Complement found from 13147 to 13155\n",
      "Seq is: CACTAATC\n",
      "1 CACTAATC motifs detected in file\n",
      "\n",
      "Complement found from 11328 to 11336\n",
      "Seq is: AGCCAACA\n",
      "1 AGCCAACA motifs detected in file\n",
      "\n",
      "Seq is: ACTACTCT\n",
      "0 ACTACTCT motifs detected in file\n",
      "\n",
      "Complement found from 7370 to 7378\n",
      "Complement found from 8823 to 8831\n",
      "Seq is: ATAAACCT\n",
      "2 ATAAACCT motifs detected in file\n",
      "\n",
      "Complement found from 10158 to 10166\n",
      "Complement found from 13022 to 13030\n",
      "Complement found from 16255 to 16263\n",
      "Seq is: CCACCCCT\n",
      "3 CCACCCCT motifs detected in file\n",
      "\n",
      "Seq is: AGTTATCT\n",
      "0 AGTTATCT motifs detected in file\n",
      "\n",
      "Complement found from 4794 to 4802\n",
      "Seq is: CCCCCTTT\n",
      "1 CCCCCTTT motifs detected in file\n",
      "\n",
      "Seq is: ATCTTACT\n",
      "0 ATCTTACT motifs detected in file\n",
      "\n",
      "Seq is: AGTCATGA\n",
      "0 AGTCATGA motifs detected in file\n",
      "\n",
      "Seq is: CGCCGCCC\n",
      "0 CGCCGCCC motifs detected in file\n",
      "\n",
      "Seq is: ATCCGGAT\n",
      "0 ATCCGGAT motifs detected in file\n",
      "\n",
      "Complement found from 369 to 377\n",
      "Complement found from 7713 to 7721\n",
      "Seq is: CCTAACAC\n",
      "2 CCTAACAC motifs detected in file\n",
      "\n",
      "Seq is: CCCCGTTA\n",
      "0 CCCCGTTA motifs detected in file\n",
      "\n",
      "Complement found from 15438 to 15446\n",
      "Seq is: CTTACTTC\n",
      "1 CTTACTTC motifs detected in file\n",
      "\n",
      "Seq is: GCTTGTCT\n",
      "0 GCTTGTCT motifs detected in file\n",
      "\n",
      "Seq is: AAAAGCAA\n",
      "0 AAAAGCAA motifs detected in file\n",
      "\n",
      "Complement found from 9236 to 9244\n",
      "Seq is: GTAAAACC\n",
      "1 GTAAAACC motifs detected in file\n",
      "\n",
      "Seq is: AAGAGTCC\n",
      "0 AAGAGTCC motifs detected in file\n",
      "\n",
      "Complement found from 3717 to 3725\n",
      "Complement found from 14189 to 14197\n",
      "Seq is: CAAACAAT\n",
      "2 CAAACAAT motifs detected in file\n",
      "\n",
      "Complement found from 2797 to 2805\n",
      "Complement found from 8886 to 8894\n",
      "Complement found from 14500 to 14508\n",
      "Seq is: ATTAAAAA\n",
      "3 ATTAAAAA motifs detected in file\n",
      "\n",
      "Complement found from 8488 to 8496\n",
      "Seq is: ATAAAAAT\n",
      "1 ATAAAAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCCGAAA\n",
      "0 ACCCGAAA motifs detected in file\n",
      "\n",
      "Complement found from 461 to 469\n",
      "Complement found from 10621 to 10629\n",
      "Seq is: CCACTCCC\n",
      "2 CCACTCCC motifs detected in file\n",
      "\n",
      "Complement found from 5200 to 5208\n",
      "Seq is: TCCATCCA\n",
      "1 TCCATCCA motifs detected in file\n",
      "\n",
      "Complement found from 7871 to 7879\n",
      "Seq is: CCATCAAA\n",
      "1 CCATCAAA motifs detected in file\n",
      "\n",
      "Seq is: CACAAATT\n",
      "0 CACAAATT motifs detected in file\n",
      "\n",
      "Complement found from 8488 to 8496\n",
      "Seq is: ATAAAAAT\n",
      "1 ATAAAAAT motifs detected in file\n",
      "\n",
      "Seq is: CAACCCAC\n",
      "0 CAACCCAC motifs detected in file\n",
      "\n",
      "Complement found from 8411 to 8419\n",
      "Seq is: TACTCCTT\n",
      "1 TACTCCTT motifs detected in file\n",
      "\n",
      "Seq is: ATCACATT\n",
      "0 ATCACATT motifs detected in file\n",
      "\n",
      "Complement found from 9331 to 9339\n",
      "Seq is: CCTCATAC\n",
      "1 CCTCATAC motifs detected in file\n",
      "\n",
      "Complement found from 7955 to 7963\n",
      "Complement found from 12432 to 12440\n",
      "Complement found from 13691 to 13699\n",
      "Complement found from 14628 to 14636\n",
      "Seq is: CCCCATTA\n",
      "4 CCCCATTA motifs detected in file\n",
      "\n",
      "Seq is: ATACCACC\n",
      "0 ATACCACC motifs detected in file\n",
      "\n",
      "Complement found from 410 to 418\n",
      "Seq is: CGGTATGC\n",
      "1 CGGTATGC motifs detected in file\n",
      "\n",
      "Complement found from 276 to 284\n",
      "Complement found from 6118 to 6126\n",
      "Complement found from 11778 to 11786\n",
      "Seq is: CATCATAA\n",
      "3 CATCATAA motifs detected in file\n",
      "\n",
      "Complement found from 14269 to 14277\n",
      "Complement found from 16215 to 16223\n",
      "Seq is: ATCAACCC\n",
      "2 ATCAACCC motifs detected in file\n",
      "\n",
      "Seq is: CGTAAAGT\n",
      "0 CGTAAAGT motifs detected in file\n",
      "\n",
      "Seq is: GACATTTC\n",
      "0 GACATTTC motifs detected in file\n",
      "\n",
      "Complement found from 13375 to 13383\n",
      "Seq is: TCCACAAC\n",
      "1 TCCACAAC motifs detected in file\n",
      "\n",
      "Complement found from 3133 to 3141\n",
      "Seq is: CAAGAGAA\n",
      "1 CAAGAGAA motifs detected in file\n",
      "\n",
      "Seq is: TTAGAAAT\n",
      "0 TTAGAAAT motifs detected in file\n",
      "\n",
      "Seq is: TGAATTTT\n",
      "0 TGAATTTT motifs detected in file\n",
      "\n",
      "Seq is: CCAATTAC\n",
      "0 CCAATTAC motifs detected in file\n",
      "\n",
      "Seq is: GATTCAAT\n",
      "0 GATTCAAT motifs detected in file\n",
      "\n",
      "Seq is: CGAAATGT\n",
      "0 CGAAATGT motifs detected in file\n",
      "\n",
      "Seq is: CACCCGAG\n",
      "0 CACCCGAG motifs detected in file\n",
      "\n",
      "Complement found from 5483 to 5491\n",
      "Seq is: ATCTCCCC\n",
      "1 ATCTCCCC motifs detected in file\n",
      "\n",
      "Seq is: CATCTCCC\n",
      "0 CATCTCCC motifs detected in file\n",
      "\n",
      "Seq is: CCACGATA\n",
      "0 CCACGATA motifs detected in file\n",
      "\n",
      "Seq is: TCCCATTT\n",
      "0 TCCCATTT motifs detected in file\n",
      "\n",
      "Seq is: ATGCCCGG\n",
      "0 ATGCCCGG motifs detected in file\n",
      "\n",
      "Seq is: GATAAAGT\n",
      "0 GATAAAGT motifs detected in file\n",
      "\n",
      "Seq is: TTCTAAAA\n",
      "0 TTCTAAAA motifs detected in file\n",
      "\n",
      "Complement found from 2073 to 2081\n",
      "Complement found from 8841 to 8849\n",
      "Seq is: ATCCCCTT\n",
      "2 ATCCCCTT motifs detected in file\n",
      "\n",
      "Complement found from 2190 to 2198\n",
      "Seq is: AATTAAGA\n",
      "1 AATTAAGA motifs detected in file\n",
      "\n",
      "Seq is: TCCTGCTA\n",
      "0 TCCTGCTA motifs detected in file\n",
      "\n",
      "Complement found from 4460 to 4468\n",
      "Seq is: CCCGTACT\n",
      "1 CCCGTACT motifs detected in file\n",
      "\n",
      "Seq is: TTGACACC\n",
      "0 TTGACACC motifs detected in file\n",
      "\n",
      "Seq is: AAACGAGG\n",
      "0 AAACGAGG motifs detected in file\n",
      "\n",
      "Seq is: TGTCTAAA\n",
      "0 TGTCTAAA motifs detected in file\n",
      "\n",
      "Seq is: GTCTCGTA\n",
      "0 GTCTCGTA motifs detected in file\n",
      "\n",
      "Complement found from 6962 to 6970\n",
      "Complement found from 9927 to 9935\n",
      "Seq is: ACTGGCAT\n",
      "2 ACTGGCAT motifs detected in file\n",
      "\n",
      "Seq is: CATATGGG\n",
      "0 CATATGGG motifs detected in file\n",
      "\n",
      "Seq is: GGCCAGCA\n",
      "0 GGCCAGCA motifs detected in file\n",
      "\n",
      "Seq is: CATCGCCA\n",
      "0 CATCGCCA motifs detected in file\n",
      "\n",
      "Complement found from 8132 to 8140\n",
      "Complement found from 14351 to 14359\n",
      "Seq is: CTTTCACC\n",
      "2 CTTTCACC motifs detected in file\n",
      "\n",
      "Complement found from 4978 to 4986\n",
      "Complement found from 8121 to 8129\n",
      "Complement found from 9673 to 9681\n",
      "Seq is: AAACCAAA\n",
      "3 AAACCAAA motifs detected in file\n",
      "\n",
      "Seq is: TCTGCAGG\n",
      "0 TCTGCAGG motifs detected in file\n",
      "\n",
      "Complement found from 16293 to 16301\n",
      "Seq is: CCCTTAAC\n",
      "1 CCCTTAAC motifs detected in file\n",
      "\n",
      "Seq is: GTAGACAA\n",
      "0 GTAGACAA motifs detected in file\n",
      "\n",
      "Seq is: AAATTCGG\n",
      "0 AAATTCGG motifs detected in file\n",
      "\n",
      "Seq is: ATTACACC\n",
      "0 ATTACACC motifs detected in file\n",
      "\n",
      "Seq is: CCTGTCGA\n",
      "0 CCTGTCGA motifs detected in file\n",
      "\n",
      "Seq is: GTACTCAC\n",
      "0 GTACTCAC motifs detected in file\n",
      "\n",
      "Seq is: GTTCTGCA\n",
      "0 GTTCTGCA motifs detected in file\n",
      "\n",
      "Complement found from 1137 to 1145\n",
      "Complement found from 5949 to 5957\n",
      "Seq is: GAACACTA\n",
      "2 GAACACTA motifs detected in file\n",
      "\n",
      "Complement found from 4752 to 4760\n",
      "Seq is: CATTAATA\n",
      "1 CATTAATA motifs detected in file\n",
      "\n",
      "Seq is: ATATGCTT\n",
      "0 ATATGCTT motifs detected in file\n",
      "\n",
      "Complement found from 1899 to 1907\n",
      "Complement found from 4135 to 4143\n",
      "Seq is: ACCCCCGA\n",
      "2 ACCCCCGA motifs detected in file\n",
      "\n",
      "Seq is: AGTTAGCC\n",
      "0 AGTTAGCC motifs detected in file\n",
      "\n",
      "Seq is: CTCATGAT\n",
      "0 CTCATGAT motifs detected in file\n",
      "\n",
      "Seq is: GAGCTAAC\n",
      "0 GAGCTAAC motifs detected in file\n",
      "\n",
      "Seq is: AGTTGCAG\n",
      "0 AGTTGCAG motifs detected in file\n",
      "\n",
      "Seq is: TTCCTCAG\n",
      "0 TTCCTCAG motifs detected in file\n",
      "\n",
      "Seq is: CGTCCAGC\n",
      "0 CGTCCAGC motifs detected in file\n",
      "\n",
      "Seq is: GGACCAAG\n",
      "0 GGACCAAG motifs detected in file\n",
      "\n",
      "Seq is: ATCCTTAT\n",
      "0 ATCCTTAT motifs detected in file\n",
      "\n",
      "Complement found from 8278 to 8286\n",
      "Seq is: TACCCCCT\n",
      "1 TACCCCCT motifs detected in file\n",
      "\n",
      "Seq is: TCATACAT\n",
      "0 TCATACAT motifs detected in file\n",
      "\n",
      "Complement found from 13844 to 13852\n",
      "Complement found from 16221 to 16229\n",
      "Seq is: CCTCAACT\n",
      "2 CCTCAACT motifs detected in file\n",
      "\n",
      "Seq is: TCTAATCA\n",
      "0 TCTAATCA motifs detected in file\n",
      "\n",
      "Seq is: GGCGGCAT\n",
      "0 GGCGGCAT motifs detected in file\n",
      "\n",
      "Complement found from 11090 to 11098\n",
      "Complement found from 11631 to 11639\n",
      "Seq is: CAGCCACA\n",
      "2 CAGCCACA motifs detected in file\n",
      "\n",
      "Complement found from 3666 to 3674\n",
      "Seq is: TGAGCATC\n",
      "1 TGAGCATC motifs detected in file\n",
      "\n",
      "Complement found from 2399 to 2407\n",
      "Seq is: CAAGTCAT\n",
      "1 CAAGTCAT motifs detected in file\n",
      "\n",
      "Seq is: GGTAACCA\n",
      "0 GGTAACCA motifs detected in file\n",
      "\n",
      "Seq is: CCGGTTAA\n",
      "0 CCGGTTAA motifs detected in file\n",
      "\n",
      "Complement found from 2779 to 2787\n",
      "Seq is: CTAAACTA\n",
      "1 CTAAACTA motifs detected in file\n",
      "\n",
      "Seq is: CCATTCCC\n",
      "0 CCATTCCC motifs detected in file\n",
      "\n",
      "Seq is: TCCCTAGC\n",
      "0 TCCCTAGC motifs detected in file\n",
      "\n",
      "Seq is: AACTGGAG\n",
      "0 AACTGGAG motifs detected in file\n",
      "\n",
      "Complement found from 15503 to 15511\n",
      "Seq is: CAGACAAT\n",
      "1 CAGACAAT motifs detected in file\n",
      "\n",
      "Complement found from 13747 to 13755\n",
      "Seq is: ACATTTCC\n",
      "1 ACATTTCC motifs detected in file\n",
      "\n",
      "Complement found from 7826 to 7834\n",
      "Seq is: TACGCATC\n",
      "1 TACGCATC motifs detected in file\n",
      "\n",
      "Seq is: CCTACCCT\n",
      "0 CCTACCCT motifs detected in file\n",
      "\n",
      "Seq is: CCCGCTAC\n",
      "0 CCCGCTAC motifs detected in file\n",
      "\n",
      "Complement found from 11917 to 11925\n",
      "Seq is: TCCTGATC\n",
      "1 TCCTGATC motifs detected in file\n",
      "\n",
      "Complement found from 5103 to 5111\n",
      "Seq is: CTACTACC\n",
      "1 CTACTACC motifs detected in file\n",
      "\n",
      "Seq is: GCCCGTCC\n",
      "0 GCCCGTCC motifs detected in file\n",
      "\n",
      "Seq is: TATCAAGT\n",
      "0 TATCAAGT motifs detected in file\n",
      "\n",
      "Complement found from 344 to 352\n",
      "Seq is: CTGCCAAA\n",
      "1 CTGCCAAA motifs detected in file\n",
      "\n",
      "Seq is: GATAAAGG\n",
      "0 GATAAAGG motifs detected in file\n",
      "\n",
      "Seq is: ACTCGCAG\n",
      "0 ACTCGCAG motifs detected in file\n",
      "\n",
      "Complement found from 4064 to 4072\n",
      "Seq is: ACTCTACA\n",
      "1 ACTCTACA motifs detected in file\n",
      "\n",
      "Complement found from 4760 to 4768\n",
      "Complement found from 6119 to 6127\n",
      "Complement found from 11779 to 11787\n",
      "Seq is: ATCATAAT\n",
      "3 ATCATAAT motifs detected in file\n",
      "\n",
      "Complement found from 15419 to 15427\n",
      "Seq is: CAATCAAA\n",
      "1 CAATCAAA motifs detected in file\n",
      "\n",
      "Complement found from 7183 to 7191\n",
      "Complement found from 8707 to 8715\n",
      "Seq is: ACAACACT\n",
      "2 ACAACACT motifs detected in file\n",
      "\n",
      "Complement found from 8561 to 8569\n",
      "Complement found from 13940 to 13948\n",
      "Seq is: CACAATCC\n",
      "2 CACAATCC motifs detected in file\n",
      "\n",
      "Seq is: TTTTCCCG\n",
      "0 TTTTCCCG motifs detected in file\n",
      "\n",
      "Complement found from 7037 to 7045\n",
      "Complement found from 15136 to 15144\n",
      "Seq is: TATGTCCT\n",
      "2 TATGTCCT motifs detected in file\n",
      "\n",
      "Seq is: GATCCTTC\n",
      "0 GATCCTTC motifs detected in file\n",
      "\n",
      "Seq is: GTCTATTC\n",
      "0 GTCTATTC motifs detected in file\n",
      "\n",
      "Seq is: CTTTTACT\n",
      "0 CTTTTACT motifs detected in file\n",
      "\n",
      "Seq is: AATACTCC\n",
      "0 AATACTCC motifs detected in file\n",
      "\n",
      "Seq is: CGCACTAG\n",
      "0 CGCACTAG motifs detected in file\n",
      "\n",
      "Seq is: TACTTTCC\n",
      "0 TACTTTCC motifs detected in file\n",
      "\n",
      "Seq is: ACTATTCG\n",
      "0 ACTATTCG motifs detected in file\n",
      "\n",
      "Seq is: AGAAGATA\n",
      "0 AGAAGATA motifs detected in file\n",
      "\n",
      "Complement found from 6840 to 6848\n",
      "Complement found from 12079 to 12087\n",
      "Seq is: CTATCCCC\n",
      "2 CTATCCCC motifs detected in file\n",
      "\n",
      "Seq is: TTCATCGC\n",
      "0 TTCATCGC motifs detected in file\n",
      "\n",
      "Seq is: AGAACATC\n",
      "0 AGAACATC motifs detected in file\n",
      "\n",
      "Seq is: TGGATGAA\n",
      "0 TGGATGAA motifs detected in file\n",
      "\n",
      "Seq is: CGCGACGT\n",
      "0 CGCGACGT motifs detected in file\n",
      "\n",
      "Seq is: ACACGGTA\n",
      "0 ACACGGTA motifs detected in file\n",
      "\n",
      "Complement found from 6587 to 6595\n",
      "Seq is: ATTCTATA\n",
      "1 ATTCTATA motifs detected in file\n",
      "\n",
      "Complement found from 5095 to 5103\n",
      "Seq is: TATCCTAA\n",
      "1 TATCCTAA motifs detected in file\n",
      "\n",
      "Seq is: ATCGGATA\n",
      "0 ATCGGATA motifs detected in file\n",
      "\n",
      "Seq is: TTAAATTG\n",
      "0 TTAAATTG motifs detected in file\n",
      "\n",
      "Seq is: AAACTGTT\n",
      "0 AAACTGTT motifs detected in file\n",
      "\n",
      "Complement found from 4741 to 4749\n",
      "Complement found from 14576 to 14584\n",
      "Seq is: TCAATACT\n",
      "2 TCAATACT motifs detected in file\n",
      "\n",
      "Complement found from 2497 to 2505\n",
      "Complement found from 10462 to 10470\n",
      "Seq is: TTACCAAA\n",
      "2 TTACCAAA motifs detected in file\n",
      "\n",
      "Seq is: AAGATTAT\n",
      "0 AAGATTAT motifs detected in file\n",
      "\n",
      "Seq is: GGAAAAAC\n",
      "0 GGAAAAAC motifs detected in file\n",
      "\n",
      "Seq is: TTTTTCAG\n",
      "0 TTTTTCAG motifs detected in file\n",
      "\n",
      "Seq is: TACCTCCG\n",
      "0 TACCTCCG motifs detected in file\n",
      "\n",
      "Seq is: GTACCCCA\n",
      "0 GTACCCCA motifs detected in file\n",
      "\n",
      "Seq is: ACCGAACT\n",
      "0 ACCGAACT motifs detected in file\n",
      "\n",
      "Seq is: TTGGTCGA\n",
      "0 TTGGTCGA motifs detected in file\n",
      "\n",
      "Complement found from 300 to 308\n",
      "Complement found from 3563 to 3571\n",
      "Complement found from 7463 to 7471\n",
      "Complement found from 10944 to 10952\n",
      "Seq is: AACCCCCC\n",
      "4 AACCCCCC motifs detected in file\n",
      "\n",
      "Complement found from 1891 to 1899\n",
      "Complement found from 5642 to 5650\n",
      "Complement found from 15992 to 16000\n",
      "Seq is: AAGCTAAG\n",
      "3 AAGCTAAG motifs detected in file\n",
      "\n",
      "Seq is: GAAGGAAA\n",
      "0 GAAGGAAA motifs detected in file\n",
      "\n",
      "Complement found from 8096 to 8104\n",
      "Seq is: AAACAGAT\n",
      "1 AAACAGAT motifs detected in file\n",
      "\n",
      "Seq is: CTAAAATA\n",
      "0 CTAAAATA motifs detected in file\n",
      "\n",
      "Seq is: CATATGCC\n",
      "0 CATATGCC motifs detected in file\n",
      "\n",
      "Seq is: CAAGAAGC\n",
      "0 CAAGAAGC motifs detected in file\n",
      "\n",
      "Seq is: TTACACAC\n",
      "0 TTACACAC motifs detected in file\n",
      "\n",
      "Seq is: CATCCCAC\n",
      "0 CATCCCAC motifs detected in file\n",
      "\n",
      "Complement found from 3167 to 3175\n",
      "Seq is: CCCCCGTA\n",
      "1 CCCCCGTA motifs detected in file\n",
      "\n",
      "Seq is: GGTATATC\n",
      "0 GGTATATC motifs detected in file\n",
      "\n",
      "Seq is: AGTGAGGT\n",
      "0 AGTGAGGT motifs detected in file\n",
      "\n",
      "Seq is: CCAAATAC\n",
      "0 CCAAATAC motifs detected in file\n",
      "\n",
      "Seq is: CTCCCAAG\n",
      "0 CTCCCAAG motifs detected in file\n",
      "\n",
      "Seq is: AAGATGAT\n",
      "0 AAGATGAT motifs detected in file\n",
      "\n",
      "Seq is: AATCCTGA\n",
      "0 AATCCTGA motifs detected in file\n",
      "\n",
      "Complement found from 2872 to 2880\n",
      "Seq is: AAAGCGAA\n",
      "1 AAAGCGAA motifs detected in file\n",
      "\n",
      "Seq is: GCTTCGCT\n",
      "0 GCTTCGCT motifs detected in file\n",
      "\n",
      "Seq is: TCCGAAGA\n",
      "0 TCCGAAGA motifs detected in file\n",
      "\n",
      "Seq is: GTTTAGTA\n",
      "0 GTTTAGTA motifs detected in file\n",
      "\n",
      "Seq is: CTTTTAAT\n",
      "0 CTTTTAAT motifs detected in file\n",
      "\n",
      "Complement found from 1803 to 1811\n",
      "Complement found from 8500 to 8508\n",
      "Complement found from 11076 to 11084\n",
      "Complement found from 11565 to 11573\n",
      "Seq is: AATTATAA\n",
      "4 AATTATAA motifs detected in file\n",
      "\n",
      "Seq is: TGACGACA\n",
      "0 TGACGACA motifs detected in file\n",
      "\n",
      "Seq is: ATCTCTTT\n",
      "0 ATCTCTTT motifs detected in file\n",
      "\n",
      "Seq is: ACTTACTC\n",
      "0 ACTTACTC motifs detected in file\n",
      "\n",
      "Seq is: GGATGTCT\n",
      "0 GGATGTCT motifs detected in file\n",
      "\n",
      "Complement found from 7780 to 7788\n",
      "Complement found from 9123 to 9131\n",
      "Seq is: ACTATCCT\n",
      "2 ACTATCCT motifs detected in file\n",
      "\n",
      "Complement found from 1291 to 1299\n",
      "Seq is: ACAAAGTA\n",
      "1 ACAAAGTA motifs detected in file\n",
      "\n",
      "Complement found from 7243 to 7251\n",
      "Complement found from 12019 to 12027\n",
      "Seq is: CACCACAT\n",
      "2 CACCACAT motifs detected in file\n",
      "\n",
      "Seq is: ACGTAGCC\n",
      "0 ACGTAGCC motifs detected in file\n",
      "\n",
      "Seq is: CCATCAGG\n",
      "0 CCATCAGG motifs detected in file\n",
      "\n",
      "Complement found from 11611 to 11619\n",
      "Seq is: CTCATTGC\n",
      "1 CTCATTGC motifs detected in file\n",
      "\n",
      "Seq is: AGCCCCGT\n",
      "0 AGCCCCGT motifs detected in file\n",
      "\n",
      "Complement found from 3143 to 3151\n",
      "Seq is: AAGGCCTA\n",
      "1 AAGGCCTA motifs detected in file\n",
      "\n",
      "Seq is: TCCGGCTC\n",
      "0 TCCGGCTC motifs detected in file\n",
      "\n",
      "Seq is: TTTCACAA\n",
      "0 TTTCACAA motifs detected in file\n",
      "\n",
      "Seq is: CACCCTTC\n",
      "0 CACCCTTC motifs detected in file\n",
      "\n",
      "Seq is: TTTCAATC\n",
      "0 TTTCAATC motifs detected in file\n",
      "\n",
      "Seq is: TAAATGCG\n",
      "0 TAAATGCG motifs detected in file\n",
      "\n",
      "Seq is: GCTACTTA\n",
      "0 GCTACTTA motifs detected in file\n",
      "\n",
      "Complement found from 3754 to 3762\n",
      "Seq is: TACTATCA\n",
      "1 TACTATCA motifs detected in file\n",
      "\n",
      "Complement found from 15279 to 15287\n",
      "Seq is: CTTTACCT\n",
      "1 CTTTACCT motifs detected in file\n",
      "\n",
      "Seq is: AAAACCGC\n",
      "0 AAAACCGC motifs detected in file\n",
      "\n",
      "Complement found from 9889 to 9897\n",
      "Complement found from 11665 to 11673\n",
      "Complement found from 13152 to 13160\n",
      "Complement found from 15672 to 15680\n",
      "Seq is: ATCCAAAC\n",
      "4 ATCCAAAC motifs detected in file\n",
      "\n",
      "Seq is: CAGATCCC\n",
      "0 CAGATCCC motifs detected in file\n",
      "\n",
      "Seq is: ACATCGGA\n",
      "0 ACATCGGA motifs detected in file\n",
      "\n",
      "Seq is: CTCTTATC\n",
      "0 CTCTTATC motifs detected in file\n",
      "\n",
      "Seq is: CCCTTTAG\n",
      "0 CCCTTTAG motifs detected in file\n",
      "\n",
      "Seq is: TCACTTAC\n",
      "0 TCACTTAC motifs detected in file\n",
      "\n",
      "Seq is: TTCGGAGG\n",
      "0 TTCGGAGG motifs detected in file\n",
      "\n",
      "Complement found from 4731 to 4739\n",
      "Complement found from 8378 to 8386\n",
      "Seq is: ATACTACC\n",
      "2 ATACTACC motifs detected in file\n",
      "\n",
      "Complement found from 576 to 584\n",
      "Seq is: GTTTATGT\n",
      "1 GTTTATGT motifs detected in file\n",
      "\n",
      "Seq is: CGAGGATA\n",
      "0 CGAGGATA motifs detected in file\n",
      "\n",
      "Complement found from 7780 to 7788\n",
      "Complement found from 9123 to 9131\n",
      "Seq is: ACTATCCT\n",
      "2 ACTATCCT motifs detected in file\n",
      "\n",
      "Seq is: GTATCACC\n",
      "0 GTATCACC motifs detected in file\n",
      "\n",
      "Seq is: TTCACCCG\n",
      "0 TTCACCCG motifs detected in file\n",
      "\n",
      "Seq is: ATGTTGCA\n",
      "0 ATGTTGCA motifs detected in file\n",
      "\n",
      "Complement found from 9034 to 9042\n",
      "Seq is: TCATGCAC\n",
      "1 TCATGCAC motifs detected in file\n",
      "\n",
      "Seq is: AGCACATC\n",
      "0 AGCACATC motifs detected in file\n",
      "\n",
      "Seq is: ATGCTACA\n",
      "0 ATGCTACA motifs detected in file\n",
      "\n",
      "Seq is: GATCACTA\n",
      "0 GATCACTA motifs detected in file\n",
      "\n",
      "Seq is: CTCAAACG\n",
      "0 CTCAAACG motifs detected in file\n",
      "\n",
      "Seq is: ATTATGTT\n",
      "0 ATTATGTT motifs detected in file\n",
      "\n",
      "Seq is: ACGGTCAG\n",
      "0 ACGGTCAG motifs detected in file\n",
      "\n",
      "Seq is: TCCGGTGG\n",
      "0 TCCGGTGG motifs detected in file\n",
      "\n",
      "Seq is: ATGCCACT\n",
      "0 ATGCCACT motifs detected in file\n",
      "\n",
      "Seq is: TTTCTTTC\n",
      "0 TTTCTTTC motifs detected in file\n",
      "\n",
      "Seq is: TACTTAGG\n",
      "0 TACTTAGG motifs detected in file\n",
      "\n",
      "Seq is: ATCCCGAG\n",
      "0 ATCCCGAG motifs detected in file\n",
      "\n",
      "Seq is: TCTCGTGA\n",
      "0 TCTCGTGA motifs detected in file\n",
      "\n",
      "Seq is: CGTCGTCT\n",
      "0 CGTCGTCT motifs detected in file\n",
      "\n",
      "Seq is: AGTAAAGT\n",
      "0 AGTAAAGT motifs detected in file\n",
      "\n",
      "Seq is: ATAACGAA\n",
      "0 ATAACGAA motifs detected in file\n",
      "\n",
      "Seq is: GGCACCTC\n",
      "0 GGCACCTC motifs detected in file\n",
      "\n",
      "Complement found from 520 to 528\n",
      "Complement found from 14563 to 14571\n",
      "Seq is: ACACCGCT\n",
      "2 ACACCGCT motifs detected in file\n",
      "\n",
      "Seq is: CAGTCGAT\n",
      "0 CAGTCGAT motifs detected in file\n",
      "\n",
      "Complement found from 1396 to 1404\n",
      "Seq is: TTATGAAA\n",
      "1 TTATGAAA motifs detected in file\n",
      "\n",
      "Seq is: CTGCGGCC\n",
      "0 CTGCGGCC motifs detected in file\n",
      "\n",
      "Complement found from 1835 to 1843\n",
      "Seq is: ACCCCTAT\n",
      "1 ACCCCTAT motifs detected in file\n",
      "\n",
      "Seq is: CGCTACTA\n",
      "0 CGCTACTA motifs detected in file\n",
      "\n",
      "Seq is: ATACCATC\n",
      "0 ATACCATC motifs detected in file\n",
      "\n",
      "Seq is: GCCTCCAC\n",
      "0 GCCTCCAC motifs detected in file\n",
      "\n",
      "Seq is: TTTATACG\n",
      "0 TTTATACG motifs detected in file\n",
      "\n",
      "Complement found from 511 to 519\n",
      "Seq is: AGCACACA\n",
      "1 AGCACACA motifs detected in file\n",
      "\n",
      "Seq is: GATGCAGA\n",
      "0 GATGCAGA motifs detected in file\n",
      "\n",
      "Seq is: TAAGGATG\n",
      "0 TAAGGATG motifs detected in file\n",
      "\n",
      "Seq is: ACATTTAT\n",
      "0 ACATTTAT motifs detected in file\n",
      "\n",
      "Seq is: ATACCACA\n",
      "0 ATACCACA motifs detected in file\n",
      "\n",
      "Seq is: CGAGTGTG\n",
      "0 CGAGTGTG motifs detected in file\n",
      "\n",
      "Seq is: CTATTTGG\n",
      "0 CTATTTGG motifs detected in file\n",
      "\n",
      "Seq is: GATCCTTC\n",
      "0 GATCCTTC motifs detected in file\n",
      "\n",
      "Seq is: GGTTAACT\n",
      "0 GGTTAACT motifs detected in file\n",
      "\n",
      "Seq is: ATAGTATC\n",
      "0 ATAGTATC motifs detected in file\n",
      "\n",
      "Seq is: GAGTCTGG\n",
      "0 GAGTCTGG motifs detected in file\n",
      "\n",
      "Seq is: TATGGATA\n",
      "0 TATGGATA motifs detected in file\n",
      "\n",
      "Seq is: CATAGGTT\n",
      "0 CATAGGTT motifs detected in file\n",
      "\n",
      "Seq is: TACCAAGA\n",
      "0 TACCAAGA motifs detected in file\n",
      "\n",
      "Complement found from 5746 to 5754\n",
      "Complement found from 7446 to 7454\n",
      "Seq is: AAAAAAGG\n",
      "2 AAAAAAGG motifs detected in file\n",
      "\n",
      "Complement found from 5293 to 5301\n",
      "Complement found from 8427 to 8435\n",
      "Seq is: CCTCATCA\n",
      "2 CCTCATCA motifs detected in file\n",
      "\n",
      "Seq is: TTCAATGT\n",
      "0 TTCAATGT motifs detected in file\n",
      "\n",
      "Complement found from 4451 to 4459\n",
      "Complement found from 15511 to 15519\n",
      "Seq is: TATACCCT\n",
      "2 TATACCCT motifs detected in file\n",
      "\n",
      "Complement found from 3768 to 3776\n",
      "Seq is: CTAATAAG\n",
      "1 CTAATAAG motifs detected in file\n",
      "\n",
      "Complement found from 6651 to 6659\n",
      "Seq is: GCTTCGGA\n",
      "1 GCTTCGGA motifs detected in file\n",
      "\n",
      "Complement found from 500 to 508\n",
      "Complement found from 4392 to 4400\n",
      "Seq is: CCATCCTA\n",
      "2 CCATCCTA motifs detected in file\n",
      "\n",
      "Seq is: TTCTTATA\n",
      "0 TTCTTATA motifs detected in file\n",
      "\n",
      "Seq is: TTTGAAGT\n",
      "0 TTTGAAGT motifs detected in file\n",
      "\n",
      "Seq is: CCCACTGG\n",
      "0 CCCACTGG motifs detected in file\n",
      "\n",
      "Seq is: CTTTTTAG\n",
      "0 CTTTTTAG motifs detected in file\n",
      "\n",
      "Complement found from 4944 to 4952\n",
      "Complement found from 6637 to 6645\n",
      "Seq is: TCTTATCC\n",
      "2 TCTTATCC motifs detected in file\n",
      "\n",
      "Complement found from 14476 to 14484\n",
      "Seq is: ACAACCAT\n",
      "1 ACAACCAT motifs detected in file\n",
      "\n",
      "Complement found from 2481 to 2489\n",
      "Seq is: ATCTTACC\n",
      "1 ATCTTACC motifs detected in file\n",
      "\n",
      "Seq is: CCAGAGGA\n",
      "0 CCAGAGGA motifs detected in file\n",
      "\n",
      "Complement found from 6256 to 6264\n",
      "Seq is: GGAGGCCG\n",
      "1 GGAGGCCG motifs detected in file\n",
      "\n",
      "Seq is: CCCCAGCT\n",
      "0 CCCCAGCT motifs detected in file\n",
      "\n",
      "Seq is: TCTTCCAC\n",
      "0 TCTTCCAC motifs detected in file\n",
      "\n",
      "Complement found from 10131 to 10139\n",
      "Seq is: CACAACTC\n",
      "1 CACAACTC motifs detected in file\n",
      "\n",
      "Complement found from 10999 to 11007\n",
      "Seq is: CAACGCCA\n",
      "1 CAACGCCA motifs detected in file\n",
      "\n",
      "Seq is: GACAATCA\n",
      "0 GACAATCA motifs detected in file\n",
      "\n",
      "Seq is: TCATATCA\n",
      "0 TCATATCA motifs detected in file\n",
      "\n",
      "Complement found from 8160 to 8168\n",
      "Seq is: CTACGGTC\n",
      "1 CTACGGTC motifs detected in file\n",
      "\n",
      "Seq is: GTCGATCC\n",
      "0 GTCGATCC motifs detected in file\n",
      "\n",
      "Seq is: TGACCCTC\n",
      "0 TGACCCTC motifs detected in file\n",
      "\n",
      "Seq is: TCTATCCT\n",
      "0 TCTATCCT motifs detected in file\n",
      "\n",
      "Complement found from 9859 to 9867\n",
      "Seq is: CTTCATCC\n",
      "1 CTTCATCC motifs detected in file\n",
      "\n",
      "Seq is: TGACGACA\n",
      "0 TGACGACA motifs detected in file\n",
      "\n",
      "Complement found from 15832 to 15840\n",
      "Seq is: CTAATCCT\n",
      "1 CTAATCCT motifs detected in file\n",
      "\n",
      "Seq is: GCCTAGTC\n",
      "0 GCCTAGTC motifs detected in file\n",
      "\n",
      "Seq is: TGCTTCTC\n",
      "0 TGCTTCTC motifs detected in file\n",
      "\n",
      "Seq is: CCCGCAAA\n",
      "0 CCCGCAAA motifs detected in file\n",
      "\n",
      "Complement found from 4722 to 4730\n",
      "Complement found from 6426 to 6434\n",
      "Seq is: CCATAACC\n",
      "2 CCATAACC motifs detected in file\n",
      "\n",
      "Seq is: CAATACCG\n",
      "0 CAATACCG motifs detected in file\n",
      "\n",
      "Complement found from 14529 to 14537\n",
      "Seq is: TCCCCCAA\n",
      "1 TCCCCCAA motifs detected in file\n",
      "\n",
      "Seq is: AATATAAC\n",
      "0 AATATAAC motifs detected in file\n",
      "\n",
      "Seq is: TATTAACA\n",
      "0 TATTAACA motifs detected in file\n",
      "\n",
      "Complement found from 5375 to 5383\n",
      "Complement found from 12346 to 12354\n",
      "Seq is: ACACTACT\n",
      "2 ACACTACT motifs detected in file\n",
      "\n",
      "Seq is: TTAACTAC\n",
      "0 TTAACTAC motifs detected in file\n",
      "\n",
      "Seq is: CGGGGATT\n",
      "0 CGGGGATT motifs detected in file\n",
      "\n",
      "Complement found from 5481 to 5489\n",
      "Complement found from 15848 to 15856\n",
      "Seq is: CTATCTCC\n",
      "2 CTATCTCC motifs detected in file\n",
      "\n",
      "Complement found from 8180 to 8188\n",
      "Seq is: TCTGTGGA\n",
      "1 TCTGTGGA motifs detected in file\n",
      "\n",
      "Seq is: CGATCCAC\n",
      "0 CGATCCAC motifs detected in file\n",
      "\n",
      "Complement found from 3288 to 3296\n",
      "Seq is: ATTCCTCT\n",
      "1 ATTCCTCT motifs detected in file\n",
      "\n",
      "Seq is: TCTACCAA\n",
      "0 TCTACCAA motifs detected in file\n",
      "\n",
      "Seq is: TCCAGATG\n",
      "0 TCCAGATG motifs detected in file\n",
      "\n",
      "Complement found from 2838 to 2846\n",
      "Seq is: CCTCCGAG\n",
      "1 CCTCCGAG motifs detected in file\n",
      "\n",
      "Complement found from 15260 to 15268\n",
      "Seq is: GTCCCACC\n",
      "1 GTCCCACC motifs detected in file\n",
      "\n",
      "Complement found from 480 to 488\n",
      "Complement found from 8634 to 8642\n",
      "Seq is: CTCATCAA\n",
      "2 CTCATCAA motifs detected in file\n",
      "\n",
      "Seq is: GGGACGAT\n",
      "0 GGGACGAT motifs detected in file\n",
      "\n",
      "Complement found from 7860 to 7868\n",
      "Seq is: TCCCTCCC\n",
      "1 TCCCTCCC motifs detected in file\n",
      "\n",
      "Complement found from 12171 to 12179\n",
      "Seq is: ATCTGACA\n",
      "1 ATCTGACA motifs detected in file\n",
      "\n",
      "Seq is: AGTTGGAC\n",
      "0 AGTTGGAC motifs detected in file\n",
      "\n",
      "Complement found from 8716 to 8724\n",
      "Seq is: AAGGACGA\n",
      "1 AAGGACGA motifs detected in file\n",
      "\n",
      "Complement found from 6259 to 6267\n",
      "Seq is: GGCCGGAG\n",
      "1 GGCCGGAG motifs detected in file\n",
      "\n",
      "Seq is: GTGATATC\n",
      "0 GTGATATC motifs detected in file\n",
      "\n",
      "Seq is: GTCTACGC\n",
      "0 GTCTACGC motifs detected in file\n",
      "\n",
      "Seq is: TCGTCCTC\n",
      "0 TCGTCCTC motifs detected in file\n",
      "\n",
      "Complement found from 11785 to 11793\n",
      "Seq is: ATCCTCTC\n",
      "1 ATCCTCTC motifs detected in file\n",
      "\n",
      "Seq is: TCCCTCCA\n",
      "0 TCCCTCCA motifs detected in file\n",
      "\n",
      "Seq is: TTCTCAGT\n",
      "0 TTCTCAGT motifs detected in file\n",
      "\n",
      "Complement found from 5786 to 5794\n",
      "Seq is: CTTCGAAT\n",
      "1 CTTCGAAT motifs detected in file\n",
      "\n",
      "Complement found from 7723 to 7731\n",
      "Complement found from 12929 to 12937\n",
      "Complement found from 15678 to 15686\n",
      "Seq is: ACAACAAA\n",
      "3 ACAACAAA motifs detected in file\n",
      "\n",
      "Seq is: TACGCCCC\n",
      "0 TACGCCCC motifs detected in file\n",
      "\n",
      "Seq is: TTTGCGGT\n",
      "0 TTTGCGGT motifs detected in file\n",
      "\n",
      "Complement found from 4790 to 4798\n",
      "Seq is: ATAGCCCC\n",
      "1 ATAGCCCC motifs detected in file\n",
      "\n",
      "Seq is: CGTGGCTA\n",
      "0 CGTGGCTA motifs detected in file\n",
      "\n",
      "Complement found from 15652 to 15660\n",
      "Seq is: ATAATCCC\n",
      "1 ATAATCCC motifs detected in file\n",
      "\n",
      "Seq is: CTTGATCA\n",
      "0 CTTGATCA motifs detected in file\n",
      "\n",
      "Seq is: GTCAACGG\n",
      "0 GTCAACGG motifs detected in file\n",
      "\n",
      "Seq is: TTTCGGAG\n",
      "0 TTTCGGAG motifs detected in file\n",
      "\n",
      "Seq is: GCTAATAC\n",
      "0 GCTAATAC motifs detected in file\n",
      "\n",
      "Seq is: TACCCATA\n",
      "0 TACCCATA motifs detected in file\n",
      "\n",
      "Seq is: ATGATACT\n",
      "0 ATGATACT motifs detected in file\n",
      "\n",
      "Seq is: TCTTCTAA\n",
      "0 TCTTCTAA motifs detected in file\n",
      "\n",
      "Seq is: TAATGTTT\n",
      "0 TAATGTTT motifs detected in file\n",
      "\n",
      "Seq is: ACGTACCC\n",
      "0 ACGTACCC motifs detected in file\n",
      "\n",
      "Seq is: GACACTGC\n",
      "0 GACACTGC motifs detected in file\n",
      "\n",
      "Seq is: TATTGCAA\n",
      "0 TATTGCAA motifs detected in file\n",
      "\n",
      "Complement found from 6058 to 6066\n",
      "Seq is: CATCTACA\n",
      "1 CATCTACA motifs detected in file\n",
      "\n",
      "Seq is: CCAGCAAT\n",
      "0 CCAGCAAT motifs detected in file\n",
      "\n",
      "Seq is: GGATCTTC\n",
      "0 GGATCTTC motifs detected in file\n",
      "\n",
      "Seq is: CAACGGAC\n",
      "0 CAACGGAC motifs detected in file\n",
      "\n",
      "Complement found from 8146 to 8154\n",
      "Seq is: CGACCGGG\n",
      "1 CGACCGGG motifs detected in file\n",
      "\n",
      "Complement found from 6013 to 6021\n",
      "Seq is: TCGAGCCG\n",
      "1 TCGAGCCG motifs detected in file\n",
      "\n",
      "Seq is: AGCTTATT\n",
      "0 AGCTTATT motifs detected in file\n",
      "\n",
      "Seq is: CCTCCGAA\n",
      "0 CCTCCGAA motifs detected in file\n",
      "\n",
      "Seq is: TCTCGACA\n",
      "0 TCTCGACA motifs detected in file\n",
      "\n",
      "Seq is: CGGATCCT\n",
      "0 CGGATCCT motifs detected in file\n",
      "\n",
      "Seq is: GAGGTCGA\n",
      "0 GAGGTCGA motifs detected in file\n",
      "\n",
      "Seq is: GTACGCGG\n",
      "0 GTACGCGG motifs detected in file\n",
      "\n",
      "Complement found from 10243 to 10251\n",
      "Seq is: CTTATTAT\n",
      "1 CTTATTAT motifs detected in file\n",
      "\n",
      "Complement found from 2960 to 2968\n",
      "Seq is: CCATATCA\n",
      "1 CCATATCA motifs detected in file\n",
      "\n",
      "Seq is: CAAGGTTA\n",
      "0 CAAGGTTA motifs detected in file\n",
      "\n",
      "Complement found from 14652 to 14660\n",
      "Seq is: CAGAAACA\n",
      "1 CAGAAACA motifs detected in file\n",
      "\n",
      "Complement found from 2234 to 2242\n",
      "Complement found from 9891 to 9899\n",
      "Complement found from 12669 to 12677\n",
      "Seq is: CCAAACAT\n",
      "3 CCAAACAT motifs detected in file\n",
      "\n",
      "Seq is: CTCTTATC\n",
      "0 CTCTTATC motifs detected in file\n",
      "\n",
      "Seq is: AGTTGCCA\n",
      "0 AGTTGCCA motifs detected in file\n",
      "\n",
      "Seq is: GCCGCTTG\n",
      "0 GCCGCTTG motifs detected in file\n",
      "\n",
      "Seq is: TAGTCACC\n",
      "0 TAGTCACC motifs detected in file\n",
      "\n",
      "Complement found from 460 to 468\n",
      "Complement found from 1688 to 1696\n",
      "Complement found from 9592 to 9600\n",
      "Complement found from 10620 to 10628\n",
      "Seq is: CCCACTCC\n",
      "4 CCCACTCC motifs detected in file\n",
      "\n",
      "Complement found from 5883 to 5891\n",
      "Complement found from 9720 to 9728\n",
      "Seq is: ATTTTACC\n",
      "2 ATTTTACC motifs detected in file\n",
      "\n",
      "Seq is: GACTCACT\n",
      "0 GACTCACT motifs detected in file\n",
      "\n",
      "Seq is: TCGTAACC\n",
      "0 TCGTAACC motifs detected in file\n",
      "\n",
      "Seq is: TGACATTT\n",
      "0 TGACATTT motifs detected in file\n",
      "\n",
      "Seq is: AGATTTCT\n",
      "0 AGATTTCT motifs detected in file\n",
      "\n",
      "Seq is: GTCCCCAA\n",
      "0 GTCCCCAA motifs detected in file\n",
      "\n",
      "Seq is: TCCGGAGA\n",
      "0 TCCGGAGA motifs detected in file\n",
      "\n",
      "Seq is: AAAATGGT\n",
      "0 AAAATGGT motifs detected in file\n",
      "\n",
      "Seq is: CGAGGCTC\n",
      "0 CGAGGCTC motifs detected in file\n",
      "\n",
      "Complement found from 14391 to 14399\n",
      "Seq is: CACTAAAA\n",
      "1 CACTAAAA motifs detected in file\n",
      "\n",
      "Seq is: GTATAACT\n",
      "0 GTATAACT motifs detected in file\n",
      "\n",
      "Seq is: TAACGTTT\n",
      "0 TAACGTTT motifs detected in file\n",
      "\n",
      "Complement found from 6202 to 6210\n",
      "Seq is: AAGCTTCT\n",
      "1 AAGCTTCT motifs detected in file\n",
      "\n",
      "Seq is: TCGTCGAA\n",
      "0 TCGTCGAA motifs detected in file\n",
      "\n",
      "Seq is: GTTTGGAC\n",
      "0 GTTTGGAC motifs detected in file\n",
      "\n",
      "Seq is: GGCCCCGA\n",
      "0 GGCCCCGA motifs detected in file\n",
      "\n",
      "Complement found from 12767 to 12775\n",
      "Seq is: AGAGGGCG\n",
      "1 AGAGGGCG motifs detected in file\n",
      "\n",
      "Complement found from 5745 to 5753\n",
      "Seq is: GAAAAAAG\n",
      "1 GAAAAAAG motifs detected in file\n",
      "\n",
      "Seq is: GGCCGCCG\n",
      "0 GGCCGCCG motifs detected in file\n",
      "\n",
      "Complement found from 6447 to 6455\n",
      "Seq is: CCCTCTTC\n",
      "1 CCCTCTTC motifs detected in file\n",
      "\n",
      "Complement found from 7167 to 7175\n",
      "Seq is: ATCTAACT\n",
      "1 ATCTAACT motifs detected in file\n",
      "\n",
      "Seq is: TCGGTCAA\n",
      "0 TCGGTCAA motifs detected in file\n",
      "\n",
      "Seq is: CTAATCCC\n",
      "0 CTAATCCC motifs detected in file\n",
      "\n",
      "Seq is: ACGAATCG\n",
      "0 ACGAATCG motifs detected in file\n",
      "\n",
      "Complement found from 240 to 248\n",
      "Seq is: ACAATTGA\n",
      "1 ACAATTGA motifs detected in file\n",
      "\n",
      "Complement found from 3152 to 3160\n",
      "Complement found from 5276 to 5284\n",
      "Seq is: TTCACAAA\n",
      "2 TTCACAAA motifs detected in file\n",
      "\n",
      "Complement found from 15986 to 15994\n",
      "Seq is: CACCCAAA\n",
      "1 CACCCAAA motifs detected in file\n",
      "\n",
      "Complement found from 16511 to 16519\n",
      "Seq is: TTCAGGGT\n",
      "1 TTCAGGGT motifs detected in file\n",
      "\n",
      "Complement found from 383 to 391\n",
      "Seq is: AACCAGAT\n",
      "1 AACCAGAT motifs detected in file\n",
      "\n",
      "Seq is: CATTCCCG\n",
      "0 CATTCCCG motifs detected in file\n",
      "\n",
      "Seq is: AATCGAAT\n",
      "0 AATCGAAT motifs detected in file\n",
      "\n",
      "Seq is: TAATTTCA\n",
      "0 TAATTTCA motifs detected in file\n",
      "\n",
      "Complement found from 8645 to 8653\n",
      "Seq is: CCGACTAA\n",
      "1 CCGACTAA motifs detected in file\n",
      "\n",
      "Seq is: ACGCAAGT\n",
      "0 ACGCAAGT motifs detected in file\n",
      "\n",
      "Complement found from 3399 to 3407\n",
      "Seq is: CAACTACG\n",
      "1 CAACTACG motifs detected in file\n",
      "\n",
      "Complement found from 13619 to 13627\n",
      "Seq is: TCTCACCC\n",
      "1 TCTCACCC motifs detected in file\n",
      "\n",
      "Seq is: CAAAACGT\n",
      "0 CAAAACGT motifs detected in file\n",
      "\n",
      "Complement found from 13099 to 13107\n",
      "Seq is: CAGGAATC\n",
      "1 CAGGAATC motifs detected in file\n",
      "\n",
      "Seq is: GACAATGT\n",
      "0 GACAATGT motifs detected in file\n",
      "\n",
      "Complement found from 2735 to 2743\n",
      "Complement found from 5634 to 5642\n",
      "Seq is: CTTTAATT\n",
      "2 CTTTAATT motifs detected in file\n",
      "\n",
      "Seq is: CATAACGT\n",
      "0 CATAACGT motifs detected in file\n",
      "\n",
      "Seq is: TGAATGAC\n",
      "0 TGAATGAC motifs detected in file\n",
      "\n",
      "Seq is: TCCCGAAA\n",
      "0 TCCCGAAA motifs detected in file\n",
      "\n",
      "Seq is: CTTCCGAG\n",
      "0 CTTCCGAG motifs detected in file\n",
      "\n",
      "Complement found from 1907 to 1915\n",
      "Seq is: AACCAGAC\n",
      "1 AACCAGAC motifs detected in file\n",
      "\n",
      "Seq is: ATAAATTG\n",
      "0 ATAAATTG motifs detected in file\n",
      "\n",
      "Seq is: GATTTAAA\n",
      "0 GATTTAAA motifs detected in file\n",
      "\n",
      "Seq is: GATATTCT\n",
      "0 GATATTCT motifs detected in file\n",
      "\n",
      "Complement found from 4756 to 4764\n",
      "Complement found from 10078 to 10086\n",
      "Seq is: AATAATCA\n",
      "2 AATAATCA motifs detected in file\n",
      "\n",
      "Complement found from 449 to 457\n",
      "Seq is: TATTTTCC\n",
      "1 TATTTTCC motifs detected in file\n",
      "\n",
      "Complement found from 6369 to 6377\n",
      "Seq is: CCTCTATC\n",
      "1 CCTCTATC motifs detected in file\n",
      "\n",
      "Complement found from 11723 to 11731\n",
      "Complement found from 15636 to 15644\n",
      "Seq is: CATCCTCA\n",
      "2 CATCCTCA motifs detected in file\n",
      "\n",
      "Seq is: TCGCACCA\n",
      "0 TCGCACCA motifs detected in file\n",
      "\n",
      "Seq is: TTCCCGCT\n",
      "0 TTCCCGCT motifs detected in file\n",
      "\n",
      "Seq is: ACTCACAC\n",
      "0 ACTCACAC motifs detected in file\n",
      "\n",
      "Complement found from 5325 to 5333\n",
      "Complement found from 10266 to 10274\n",
      "Seq is: CCCTCCTT\n",
      "2 CCCTCCTT motifs detected in file\n",
      "\n",
      "Complement found from 13675 to 13683\n",
      "Seq is: ACCCCACC\n",
      "1 ACCCCACC motifs detected in file\n",
      "\n",
      "Complement found from 12153 to 12161\n",
      "Seq is: CAAAACAT\n",
      "1 CAAAACAT motifs detected in file\n",
      "\n",
      "Seq is: ACAAGTTT\n",
      "0 ACAAGTTT motifs detected in file\n",
      "\n",
      "Seq is: GACAGTAA\n",
      "0 GACAGTAA motifs detected in file\n",
      "\n",
      "Complement found from 8493 to 8501\n",
      "Seq is: AATAAAAA\n",
      "1 AATAAAAA motifs detected in file\n",
      "\n",
      "Seq is: TGCAACAA\n",
      "0 TGCAACAA motifs detected in file\n",
      "\n",
      "Complement found from 6589 to 6597\n",
      "Seq is: TCTATACC\n",
      "1 TCTATACC motifs detected in file\n",
      "\n",
      "Complement found from 5293 to 5301\n",
      "Complement found from 8427 to 8435\n",
      "Seq is: CCTCATCA\n",
      "2 CCTCATCA motifs detected in file\n",
      "\n",
      "Seq is: CACTAACT\n",
      "0 CACTAACT motifs detected in file\n",
      "\n",
      "Complement found from 5364 to 5372\n",
      "Seq is: CCACCTCA\n",
      "1 CCACCTCA motifs detected in file\n",
      "\n",
      "Seq is: TCTAATCC\n",
      "0 TCTAATCC motifs detected in file\n",
      "\n",
      "Complement found from 4664 to 4672\n",
      "Seq is: GCATCCAT\n",
      "1 GCATCCAT motifs detected in file\n",
      "\n",
      "Complement found from 15291 to 15299\n",
      "Seq is: CTTCATCT\n",
      "1 CTTCATCT motifs detected in file\n",
      "\n",
      "Seq is: CCAATTCC\n",
      "0 CCAATTCC motifs detected in file\n",
      "\n",
      "Complement found from 11813 to 11821\n",
      "Seq is: TCCCACTA\n",
      "1 TCCCACTA motifs detected in file\n",
      "\n",
      "Seq is: CCACCGAT\n",
      "0 CCACCGAT motifs detected in file\n",
      "\n",
      "Complement found from 8459 to 8467\n",
      "Complement found from 10126 to 10134\n",
      "Complement found from 10791 to 10799\n",
      "Seq is: ACTACCAC\n",
      "3 ACTACCAC motifs detected in file\n",
      "\n",
      "Complement found from 5148 to 5156\n",
      "Complement found from 10554 to 10562\n",
      "Complement found from 10695 to 10703\n",
      "Complement found from 13681 to 13689\n",
      "Seq is: CCCTACTA\n",
      "4 CCCTACTA motifs detected in file\n",
      "\n",
      "Seq is: CTCCGATA\n",
      "0 CTCCGATA motifs detected in file\n",
      "\n",
      "Complement found from 283 to 291\n",
      "Complement found from 5279 to 5287\n",
      "Complement found from 7444 to 7452\n",
      "Complement found from 12415 to 12423\n",
      "Seq is: ACAAAAAA\n",
      "4 ACAAAAAA motifs detected in file\n",
      "\n",
      "Seq is: CACTTAAG\n",
      "0 CACTTAAG motifs detected in file\n",
      "\n",
      "Complement found from 14013 to 14021\n",
      "Seq is: AAGCTATT\n",
      "1 AAGCTATT motifs detected in file\n",
      "\n",
      "Seq is: ACCGGGTA\n",
      "0 ACCGGGTA motifs detected in file\n",
      "\n",
      "Seq is: AACCCGTT\n",
      "0 AACCCGTT motifs detected in file\n",
      "\n",
      "Seq is: TTTCGGCC\n",
      "0 TTTCGGCC motifs detected in file\n",
      "\n",
      "Complement found from 11706 to 11714\n",
      "Complement found from 14940 to 14948\n",
      "Seq is: AATCGCCC\n",
      "2 AATCGCCC motifs detected in file\n",
      "\n",
      "Seq is: CCGTCCGG\n",
      "0 CCGTCCGG motifs detected in file\n",
      "\n",
      "Seq is: AGGATCCC\n",
      "0 AGGATCCC motifs detected in file\n",
      "\n",
      "Complement found from 2324 to 2332\n",
      "Complement found from 12090 to 12098\n",
      "Seq is: TCTCCTCC\n",
      "2 TCTCCTCC motifs detected in file\n",
      "\n",
      "Complement found from 8464 to 8472\n",
      "Seq is: CACCTACC\n",
      "1 CACCTACC motifs detected in file\n",
      "\n",
      "Complement found from 5195 to 5203\n",
      "Complement found from 8221 to 8229\n",
      "Seq is: TTAATTCC\n",
      "2 TTAATTCC motifs detected in file\n",
      "\n",
      "Complement found from 10983 to 10991\n",
      "Complement found from 15417 to 15425\n",
      "Seq is: CACAATCA\n",
      "2 CACAATCA motifs detected in file\n",
      "\n",
      "Seq is: GTACAATC\n",
      "0 GTACAATC motifs detected in file\n",
      "\n",
      "Seq is: GAACAAAG\n",
      "0 GAACAAAG motifs detected in file\n",
      "\n",
      "Seq is: TCCACGCT\n",
      "0 TCCACGCT motifs detected in file\n",
      "\n",
      "Complement found from 7259 to 7267\n",
      "Complement found from 11150 to 11158\n",
      "Seq is: CTATCATC\n",
      "2 CTATCATC motifs detected in file\n",
      "\n",
      "Seq is: ATCCCAGC\n",
      "0 ATCCCAGC motifs detected in file\n",
      "\n",
      "Complement found from 5141 to 5149\n",
      "Complement found from 14692 to 14700\n",
      "Seq is: ACCACGAC\n",
      "2 ACCACGAC motifs detected in file\n",
      "\n",
      "Seq is: CTCAAATT\n",
      "0 CTCAAATT motifs detected in file\n",
      "\n",
      "Complement found from 4155 to 4163\n",
      "Complement found from 12914 to 12922\n",
      "Seq is: CAACTCAT\n",
      "2 CAACTCAT motifs detected in file\n",
      "\n",
      "Complement found from 12389 to 12397\n",
      "Seq is: CATCCTTA\n",
      "1 CATCCTTA motifs detected in file\n",
      "\n",
      "Complement found from 7791 to 7799\n",
      "Seq is: CGCCATCA\n",
      "1 CGCCATCA motifs detected in file\n",
      "\n",
      "Complement found from 3648 to 3656\n",
      "Seq is: TCAATCCT\n",
      "1 TCAATCCT motifs detected in file\n",
      "\n",
      "Complement found from 10783 to 10791\n",
      "Seq is: ATTATATT\n",
      "1 ATTATATT motifs detected in file\n",
      "\n",
      "Complement found from 6735 to 6743\n",
      "Seq is: TATCAATT\n",
      "1 TATCAATT motifs detected in file\n",
      "\n",
      "Complement found from 13612 to 13620\n",
      "Seq is: TAATTCTT\n",
      "1 TAATTCTT motifs detected in file\n",
      "\n",
      "Complement found from 4727 to 4735\n",
      "Seq is: ACCAATAC\n",
      "1 ACCAATAC motifs detected in file\n",
      "\n",
      "Complement found from 2230 to 2238\n",
      "Seq is: AATCCCAA\n",
      "1 AATCCCAA motifs detected in file\n",
      "\n",
      "Seq is: CATGCCAT\n",
      "0 CATGCCAT motifs detected in file\n",
      "\n",
      "Seq is: CTTGACGA\n",
      "0 CTTGACGA motifs detected in file\n",
      "\n",
      "Complement found from 3769 to 3777\n",
      "Seq is: TAATAAGT\n",
      "1 TAATAAGT motifs detected in file\n",
      "\n",
      "Seq is: AGGATACA\n",
      "0 AGGATACA motifs detected in file\n",
      "\n",
      "Complement found from 13692 to 13700\n",
      "Seq is: CCCATTAA\n",
      "1 CCCATTAA motifs detected in file\n",
      "\n",
      "Complement found from 3327 to 3335\n",
      "Complement found from 9329 to 9337\n",
      "Complement found from 15739 to 15747\n",
      "Seq is: CTCCTCAT\n",
      "3 CTCCTCAT motifs detected in file\n",
      "\n",
      "Complement found from 15273 to 15281\n",
      "Seq is: ACGATTCT\n",
      "1 ACGATTCT motifs detected in file\n",
      "\n",
      "Seq is: AAAACGCA\n",
      "0 AAAACGCA motifs detected in file\n",
      "\n",
      "Seq is: TCGACCCA\n",
      "0 TCGACCCA motifs detected in file\n",
      "\n",
      "Complement found from 9674 to 9682\n",
      "Complement found from 10885 to 10893\n",
      "Seq is: AACCAAAT\n",
      "2 AACCAAAT motifs detected in file\n",
      "\n",
      "Seq is: TAGGTGGA\n",
      "0 TAGGTGGA motifs detected in file\n",
      "\n",
      "Seq is: GTTGACGG\n",
      "0 GTTGACGG motifs detected in file\n",
      "\n",
      "Seq is: ACGATACT\n",
      "0 ACGATACT motifs detected in file\n",
      "\n",
      "Complement found from 6600 to 6608\n",
      "Complement found from 12742 to 12750\n",
      "Complement found from 14147 to 14155\n",
      "Seq is: ACCTATTC\n",
      "3 ACCTATTC motifs detected in file\n",
      "\n",
      "Seq is: TAACTCTC\n",
      "0 TAACTCTC motifs detected in file\n",
      "\n",
      "Complement found from 14848 to 14856\n",
      "Seq is: TCACTCCT\n",
      "1 TCACTCCT motifs detected in file\n",
      "\n",
      "Seq is: CTTCCGAA\n",
      "0 CTTCCGAA motifs detected in file\n",
      "\n",
      "Seq is: TGCAAATC\n",
      "0 TGCAAATC motifs detected in file\n",
      "\n",
      "Complement found from 10623 to 10631\n",
      "Complement found from 11975 to 11983\n",
      "Seq is: ACTCCCTC\n",
      "2 ACTCCCTC motifs detected in file\n",
      "\n",
      "Complement found from 8118 to 8126\n",
      "Seq is: TCTAAACC\n",
      "1 TCTAAACC motifs detected in file\n",
      "\n",
      "Seq is: ATATACTA\n",
      "0 ATATACTA motifs detected in file\n",
      "\n",
      "Complement found from 4497 to 4505\n",
      "Complement found from 11037 to 11045\n",
      "Seq is: ACTCTACC\n",
      "2 ACTCTACC motifs detected in file\n",
      "\n",
      "Complement found from 1223 to 1231\n",
      "Seq is: CCCGATCA\n",
      "1 CCCGATCA motifs detected in file\n",
      "\n",
      "Seq is: AAAACAGT\n",
      "0 AAAACAGT motifs detected in file\n",
      "\n",
      "Seq is: ACACTCTT\n",
      "0 ACACTCTT motifs detected in file\n",
      "\n",
      "Seq is: CTTCGTCC\n",
      "0 CTTCGTCC motifs detected in file\n",
      "\n",
      "Seq is: GGCCTACA\n",
      "0 GGCCTACA motifs detected in file\n",
      "\n",
      "Seq is: GTCTCCCC\n",
      "0 GTCTCCCC motifs detected in file\n",
      "\n",
      "Seq is: ACGGAACC\n",
      "0 ACGGAACC motifs detected in file\n",
      "\n",
      "Seq is: CATTGGAG\n",
      "0 CATTGGAG motifs detected in file\n",
      "\n",
      "Complement found from 8511 to 8519\n",
      "Seq is: ACCCTGAG\n",
      "1 ACCCTGAG motifs detected in file\n",
      "\n",
      "Seq is: TCTTCACT\n",
      "0 TCTTCACT motifs detected in file\n",
      "\n",
      "Complement found from 3165 to 3173\n",
      "Complement found from 13751 to 13759\n",
      "Complement found from 14152 to 14160\n",
      "Seq is: TTCCCCCG\n",
      "3 TTCCCCCG motifs detected in file\n",
      "\n",
      "Seq is: ATAAGGAT\n",
      "0 ATAAGGAT motifs detected in file\n",
      "\n",
      "Seq is: CAAAATAA\n",
      "0 CAAAATAA motifs detected in file\n",
      "\n",
      "Complement found from 12869 to 12877\n",
      "Seq is: CGATATCG\n",
      "1 CGATATCG motifs detected in file\n",
      "\n",
      "Complement found from 10574 to 10582\n",
      "Seq is: ATAATACT\n",
      "1 ATAATACT motifs detected in file\n",
      "\n",
      "Complement found from 8040 to 8048\n",
      "Complement found from 10108 to 10116\n",
      "Seq is: AATAATTA\n",
      "2 AATAATTA motifs detected in file\n",
      "\n",
      "Complement found from 8970 to 8978\n",
      "Complement found from 9030 to 9038\n",
      "Complement found from 11236 to 11244\n",
      "Complement found from 12696 to 12704\n",
      "Seq is: CTACTCAT\n",
      "4 CTACTCAT motifs detected in file\n",
      "\n",
      "Complement found from 8677 to 8685\n",
      "Complement found from 8768 to 8776\n",
      "Complement found from 10300 to 10308\n",
      "Complement found from 14763 to 14771\n",
      "Seq is: AACTAACC\n",
      "4 AACTAACC motifs detected in file\n",
      "\n",
      "Complement found from 277 to 285\n",
      "Seq is: ATCATAAC\n",
      "1 ATCATAAC motifs detected in file\n",
      "\n",
      "Complement found from 6434 to 6442\n",
      "Seq is: CAATACCA\n",
      "1 CAATACCA motifs detected in file\n",
      "\n",
      "Complement found from 11649 to 11657\n",
      "Seq is: AGTAACAG\n",
      "1 AGTAACAG motifs detected in file\n",
      "\n",
      "Seq is: GCCTCTCA\n",
      "0 GCCTCTCA motifs detected in file\n",
      "\n",
      "Complement found from 8503 to 8511\n",
      "Complement found from 11568 to 11576\n",
      "Seq is: TATAACAA\n",
      "2 TATAACAA motifs detected in file\n",
      "\n",
      "Complement found from 4925 to 4933\n",
      "Complement found from 6343 to 6351\n",
      "Complement found from 6485 to 6493\n",
      "Seq is: CTTCTCCT\n",
      "3 CTTCTCCT motifs detected in file\n",
      "\n",
      "Complement found from 1213 to 1221\n",
      "Seq is: ATCGATAA\n",
      "1 ATCGATAA motifs detected in file\n",
      "\n",
      "Complement found from 4027 to 4035\n",
      "Complement found from 12703 to 12711\n",
      "Complement found from 15026 to 15034\n",
      "Seq is: TCTTCCTA\n",
      "3 TCTTCCTA motifs detected in file\n",
      "\n",
      "Seq is: ATACCTAC\n",
      "0 ATACCTAC motifs detected in file\n",
      "\n",
      "Seq is: GCCAACGA\n",
      "0 GCCAACGA motifs detected in file\n",
      "\n",
      "Complement found from 4048 to 4056\n",
      "Complement found from 11763 to 11771\n",
      "Seq is: ACGCACTC\n",
      "2 ACGCACTC motifs detected in file\n",
      "\n",
      "Seq is: CTTTATGA\n",
      "0 CTTTATGA motifs detected in file\n",
      "\n",
      "Complement found from 8380 to 8388\n",
      "Seq is: ACTACCGT\n",
      "1 ACTACCGT motifs detected in file\n",
      "\n",
      "Seq is: CGAAGACA\n",
      "0 CGAAGACA motifs detected in file\n",
      "\n",
      "Seq is: CCTTGCTC\n",
      "0 CCTTGCTC motifs detected in file\n",
      "\n",
      "Complement found from 1732 to 1740\n",
      "Complement found from 9676 to 9684\n",
      "Complement found from 12323 to 12331\n",
      "Seq is: CCAAATAA\n",
      "3 CCAAATAA motifs detected in file\n",
      "\n",
      "Complement found from 7527 to 7535\n",
      "Complement found from 14710 to 14718\n",
      "Seq is: AAAAACCA\n",
      "2 AAAAACCA motifs detected in file\n",
      "\n",
      "Seq is: ATCTTGAC\n",
      "0 ATCTTGAC motifs detected in file\n",
      "\n",
      "Seq is: CTTATTTT\n",
      "0 CTTATTTT motifs detected in file\n",
      "\n",
      "Seq is: CGATCGTA\n",
      "0 CGATCGTA motifs detected in file\n",
      "\n",
      "Complement found from 1733 to 1741\n",
      "Complement found from 12324 to 12332\n",
      "Seq is: CAAATAAA\n",
      "2 CAAATAAA motifs detected in file\n",
      "\n",
      "Seq is: GATCCGGA\n",
      "0 GATCCGGA motifs detected in file\n",
      "\n",
      "Seq is: TGAGTCCA\n",
      "0 TGAGTCCA motifs detected in file\n",
      "\n",
      "Seq is: TTTTTTAG\n",
      "0 TTTTTTAG motifs detected in file\n",
      "\n",
      "Seq is: TCACGCTC\n",
      "0 TCACGCTC motifs detected in file\n",
      "\n",
      "Seq is: GAATCGCG\n",
      "0 GAATCGCG motifs detected in file\n",
      "\n",
      "Complement found from 5375 to 5383\n",
      "Complement found from 12346 to 12354\n",
      "Seq is: ACACTACT\n",
      "2 ACACTACT motifs detected in file\n",
      "\n",
      "Seq is: CACACGGA\n",
      "0 CACACGGA motifs detected in file\n",
      "\n",
      "Seq is: CGTTTCTA\n",
      "0 CGTTTCTA motifs detected in file\n",
      "\n",
      "Complement found from 4881 to 4889\n",
      "Complement found from 10506 to 10514\n",
      "Seq is: CCATCTCA\n",
      "2 CCATCTCA motifs detected in file\n",
      "\n",
      "Seq is: TCTACTGC\n",
      "0 TCTACTGC motifs detected in file\n",
      "\n",
      "Complement found from 4483 to 4491\n",
      "Seq is: CCAACCCG\n",
      "1 CCAACCCG motifs detected in file\n",
      "\n",
      "Seq is: GTCCCCTA\n",
      "0 GTCCCCTA motifs detected in file\n",
      "\n",
      "Complement found from 10336 to 10344\n",
      "Complement found from 12675 to 12683\n",
      "Seq is: ATTAATCA\n",
      "2 ATTAATCA motifs detected in file\n",
      "\n",
      "Complement found from 15299 to 15307\n",
      "Seq is: TGCCCTTC\n",
      "1 TGCCCTTC motifs detected in file\n",
      "\n",
      "Complement found from 6667 to 6675\n",
      "Seq is: CCATATTG\n",
      "1 CCATATTG motifs detected in file\n",
      "\n",
      "Complement found from 987 to 995\n",
      "Seq is: GTTGTAAA\n",
      "1 GTTGTAAA motifs detected in file\n",
      "\n",
      "Seq is: AGCCCCAT\n",
      "0 AGCCCCAT motifs detected in file\n",
      "\n",
      "Seq is: ACCCGGGC\n",
      "0 ACCCGGGC motifs detected in file\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Seq is: TATCGAAT\n",
      "0 TATCGAAT motifs detected in file\n",
      "\n",
      "Seq is: AAATCGAC\n",
      "0 AAATCGAC motifs detected in file\n",
      "\n",
      "Seq is: TGGAATGA\n",
      "0 TGGAATGA motifs detected in file\n",
      "\n",
      "Complement found from 14368 to 14376\n",
      "Seq is: AATCCTAC\n",
      "1 AATCCTAC motifs detected in file\n",
      "\n",
      "Complement found from 5447 to 5455\n",
      "Complement found from 14641 to 14649\n",
      "Seq is: CCCACACT\n",
      "2 CCCACACT motifs detected in file\n",
      "\n",
      "Seq is: ATCCACCG\n",
      "0 ATCCACCG motifs detected in file\n",
      "\n",
      "Complement found from 15022 to 15030\n",
      "Seq is: TGCCTCTT\n",
      "1 TGCCTCTT motifs detected in file\n",
      "\n",
      "Seq is: AAAACCTA\n",
      "0 AAAACCTA motifs detected in file\n",
      "\n",
      "Seq is: AGAGTCCC\n",
      "0 AGAGTCCC motifs detected in file\n",
      "\n",
      "Complement found from 4822 to 4830\n",
      "Seq is: TACCCAAG\n",
      "1 TACCCAAG motifs detected in file\n",
      "\n",
      "Complement found from 1435 to 1443\n",
      "Seq is: CTAAGAGT\n",
      "1 CTAAGAGT motifs detected in file\n",
      "\n",
      "Seq is: ATCAGGAT\n",
      "0 ATCAGGAT motifs detected in file\n",
      "\n",
      "Seq is: CTTTATTC\n",
      "0 CTTTATTC motifs detected in file\n",
      "\n",
      "Seq is: CCCCAAAT\n",
      "0 CCCCAAAT motifs detected in file\n",
      "\n",
      "Seq is: TCGAGGAT\n",
      "0 TCGAGGAT motifs detected in file\n",
      "\n",
      "Seq is: AATAAATG\n",
      "0 AATAAATG motifs detected in file\n",
      "\n",
      "Seq is: AGATAGTT\n",
      "0 AGATAGTT motifs detected in file\n",
      "\n",
      "Seq is: TCATTGAG\n",
      "0 TCATTGAG motifs detected in file\n",
      "\n",
      "Complement found from 1959 to 1967\n",
      "Seq is: AAAATAGT\n",
      "1 AAAATAGT motifs detected in file\n",
      "\n",
      "Seq is: CTGTATAA\n",
      "0 CTGTATAA motifs detected in file\n",
      "\n",
      "Complement found from 4358 to 4366\n",
      "Seq is: AGAATCCA\n",
      "1 AGAATCCA motifs detected in file\n",
      "\n",
      "Seq is: AACTCCCC\n",
      "0 AACTCCCC motifs detected in file\n",
      "\n",
      "Complement found from 12186 to 12194\n",
      "Seq is: CTTACGAC\n",
      "1 CTTACGAC motifs detected in file\n",
      "\n",
      "Complement found from 7284 to 7292\n",
      "Complement found from 13159 to 13167\n",
      "Seq is: CTCTAACA\n",
      "2 CTCTAACA motifs detected in file\n",
      "\n",
      "Seq is: TTACCCAT\n",
      "0 TTACCCAT motifs detected in file\n",
      "\n",
      "Seq is: ACCTCTGT\n",
      "0 ACCTCTGT motifs detected in file\n",
      "\n",
      "Seq is: ATAGTATA\n",
      "0 ATAGTATA motifs detected in file\n",
      "\n",
      "Seq is: TTCATTAC\n",
      "0 TTCATTAC motifs detected in file\n",
      "\n",
      "Seq is: GATCCCAC\n",
      "0 GATCCCAC motifs detected in file\n",
      "\n",
      "Complement found from 3541 to 3549\n",
      "Seq is: TCACCATC\n",
      "1 TCACCATC motifs detected in file\n",
      "\n",
      "Seq is: CTTCAAAA\n",
      "0 CTTCAAAA motifs detected in file\n",
      "\n",
      "Seq is: AAGTATCC\n",
      "0 AAGTATCC motifs detected in file\n",
      "\n",
      "Seq is: TCCACATA\n",
      "0 TCCACATA motifs detected in file\n",
      "\n",
      "Seq is: CTCAACCA\n",
      "0 CTCAACCA motifs detected in file\n",
      "\n",
      "Seq is: GCATCGCC\n",
      "0 GCATCGCC motifs detected in file\n",
      "\n",
      "Seq is: TTAGCCCC\n",
      "0 TTAGCCCC motifs detected in file\n",
      "\n",
      "Seq is: CATACGAC\n",
      "0 CATACGAC motifs detected in file\n",
      "\n",
      "Seq is: AAGCTTAA\n",
      "0 AAGCTTAA motifs detected in file\n",
      "\n",
      "Seq is: GTATTCTT\n",
      "0 GTATTCTT motifs detected in file\n",
      "\n",
      "Seq is: GTCCCTCC\n",
      "0 GTCCCTCC motifs detected in file\n",
      "\n",
      "Complement found from 7932 to 7940\n",
      "Seq is: AATCTTCA\n",
      "1 AATCTTCA motifs detected in file\n",
      "\n",
      "Seq is: TCCCAGAA\n",
      "0 TCCCAGAA motifs detected in file\n",
      "\n",
      "Seq is: CCACTGTT\n",
      "0 CCACTGTT motifs detected in file\n",
      "\n",
      "Seq is: TTATACAA\n",
      "0 TTATACAA motifs detected in file\n",
      "\n",
      "Complement found from 337 to 345\n",
      "Seq is: CACATCTC\n",
      "1 CACATCTC motifs detected in file\n",
      "\n",
      "Seq is: AAGTCCCC\n",
      "0 AAGTCCCC motifs detected in file\n",
      "\n",
      "Seq is: TCTCACGC\n",
      "0 TCTCACGC motifs detected in file\n",
      "\n",
      "Seq is: AGTATACA\n",
      "0 AGTATACA motifs detected in file\n",
      "\n",
      "Seq is: ACAAGGAT\n",
      "0 ACAAGGAT motifs detected in file\n",
      "\n",
      "Complement found from 4675 to 4683\n",
      "Complement found from 12941 to 12949\n",
      "Seq is: CCTTCTAA\n",
      "2 CCTTCTAA motifs detected in file\n",
      "\n",
      "Complement found from 2519 to 2527\n",
      "Seq is: CATCACCA\n",
      "1 CATCACCA motifs detected in file\n",
      "\n",
      "Seq is: CTCCCACA\n",
      "0 CTCCCACA motifs detected in file\n",
      "\n",
      "Seq is: AATAATAT\n",
      "0 AATAATAT motifs detected in file\n",
      "\n",
      "Complement found from 9697 to 9705\n",
      "Seq is: TATTACAA\n",
      "1 TATTACAA motifs detected in file\n",
      "\n",
      "Complement found from 9199 to 9207\n",
      "Complement found from 10818 to 10826\n",
      "Seq is: ACACATAA\n",
      "2 ACACATAA motifs detected in file\n",
      "\n",
      "Seq is: GCCGATAC\n",
      "0 GCCGATAC motifs detected in file\n",
      "\n",
      "Complement found from 6636 to 6644\n",
      "Seq is: TTCTTATC\n",
      "1 TTCTTATC motifs detected in file\n",
      "\n",
      "Complement found from 13118 to 13126\n",
      "Seq is: CCGCTTCC\n",
      "1 CCGCTTCC motifs detected in file\n",
      "\n",
      "Seq is: CCGGACGC\n",
      "0 CCGGACGC motifs detected in file\n",
      "\n",
      "Complement found from 2331 to 2339\n",
      "Seq is: CGCATAAG\n",
      "1 CGCATAAG motifs detected in file\n",
      "\n",
      "Seq is: CTACAACT\n",
      "0 CTACAACT motifs detected in file\n",
      "\n",
      "Complement found from 8779 to 8787\n",
      "Seq is: TCGGACTC\n",
      "1 TCGGACTC motifs detected in file\n",
      "\n",
      "Seq is: TGATCAAG\n",
      "0 TGATCAAG motifs detected in file\n",
      "\n",
      "Seq is: CCTGAGGG\n",
      "0 CCTGAGGG motifs detected in file\n",
      "\n",
      "Seq is: GAAGCCGT\n",
      "0 GAAGCCGT motifs detected in file\n",
      "\n",
      "Seq is: TCCAGCTT\n",
      "0 TCCAGCTT motifs detected in file\n",
      "\n",
      "Seq is: CCCCCAAG\n",
      "0 CCCCCAAG motifs detected in file\n",
      "\n",
      "Seq is: CCAACCAG\n",
      "0 CCAACCAG motifs detected in file\n",
      "\n",
      "Seq is: AGACGATC\n",
      "0 AGACGATC motifs detected in file\n",
      "\n",
      "Complement found from 3815 to 3823\n",
      "Complement found from 8347 to 8355\n",
      "Seq is: ACACCTCT\n",
      "2 ACACCTCT motifs detected in file\n",
      "\n",
      "Seq is: ATTTAGTA\n",
      "0 ATTTAGTA motifs detected in file\n",
      "\n",
      "Seq is: TAATACCG\n",
      "0 TAATACCG motifs detected in file\n",
      "\n",
      "Seq is: GTTCCCAG\n",
      "0 GTTCCCAG motifs detected in file\n",
      "\n",
      "Complement found from 5104 to 5112\n",
      "Complement found from 8379 to 8387\n",
      "Seq is: TACTACCG\n",
      "2 TACTACCG motifs detected in file\n",
      "\n",
      "Complement found from 3328 to 3336\n",
      "Complement found from 11725 to 11733\n",
      "Complement found from 15740 to 15748\n",
      "Seq is: TCCTCATT\n",
      "3 TCCTCATT motifs detected in file\n",
      "\n",
      "Seq is: AGTCTCCA\n",
      "0 AGTCTCCA motifs detected in file\n",
      "\n",
      "Complement found from 3215 to 3223\n",
      "Complement found from 3810 to 3818\n",
      "Complement found from 14735 to 14743\n",
      "Seq is: CAAGAACA\n",
      "3 CAAGAACA motifs detected in file\n",
      "\n",
      "Complement found from 8708 to 8716\n",
      "Seq is: CAACACTA\n",
      "1 CAACACTA motifs detected in file\n",
      "\n",
      "Seq is: TTCCCACC\n",
      "0 TTCCCACC motifs detected in file\n",
      "\n",
      "Seq is: TCTCCAAT\n",
      "0 TCTCCAAT motifs detected in file\n",
      "\n",
      "Seq is: TTCCTCGG\n",
      "0 TTCCTCGG motifs detected in file\n",
      "\n",
      "Complement found from 5033 to 5041\n",
      "Seq is: TGAATAAT\n",
      "1 TGAATAAT motifs detected in file\n",
      "\n",
      "Complement found from 4229 to 4237\n",
      "Seq is: CATTACAA\n",
      "1 CATTACAA motifs detected in file\n",
      "\n",
      "Complement found from 7259 to 7267\n",
      "Complement found from 11150 to 11158\n",
      "Seq is: CTATCATC\n",
      "2 CTATCATC motifs detected in file\n",
      "\n",
      "Complement found from 1699 to 1707\n",
      "Complement found from 10789 to 10797\n",
      "Seq is: TTACTACC\n",
      "2 TTACTACC motifs detected in file\n",
      "\n",
      "Seq is: GATCCCAC\n",
      "0 GATCCCAC motifs detected in file\n",
      "\n",
      "Seq is: TGAAGTAT\n",
      "0 TGAAGTAT motifs detected in file\n",
      "\n",
      "Complement found from 13158 to 13166\n",
      "Seq is: ACTCTAAC\n",
      "1 ACTCTAAC motifs detected in file\n",
      "\n",
      "Seq is: AAACCCGA\n",
      "0 AAACCCGA motifs detected in file\n",
      "\n",
      "Seq is: TGACGAGC\n",
      "0 TGACGAGC motifs detected in file\n",
      "\n",
      "Seq is: GTCACGCG\n",
      "0 GTCACGCG motifs detected in file\n",
      "\n",
      "Seq is: GCTAGTCC\n",
      "0 GCTAGTCC motifs detected in file\n",
      "\n",
      "Seq is: CGCATCAA\n",
      "0 CGCATCAA motifs detected in file\n",
      "\n",
      "Complement found from 3676 to 3684\n",
      "Complement found from 11750 to 11758\n",
      "Seq is: ACTCAAAC\n",
      "2 ACTCAAAC motifs detected in file\n",
      "\n",
      "Complement found from 10166 to 10174\n",
      "Seq is: TACGAGTG\n",
      "1 TACGAGTG motifs detected in file\n",
      "\n",
      "Seq is: GGACTAGT\n",
      "0 GGACTAGT motifs detected in file\n",
      "\n",
      "Seq is: CTCCTAAC\n",
      "0 CTCCTAAC motifs detected in file\n",
      "\n",
      "Seq is: TCATTTGC\n",
      "0 TCATTTGC motifs detected in file\n",
      "\n",
      "Seq is: CGATCCGA\n",
      "0 CGATCCGA motifs detected in file\n",
      "\n",
      "Complement found from 3790 to 3798\n",
      "Complement found from 13019 to 13027\n",
      "Complement found from 14047 to 14055\n",
      "Seq is: TCTCCACC\n",
      "3 TCTCCACC motifs detected in file\n",
      "\n",
      "Seq is: GATCTTAT\n",
      "0 GATCTTAT motifs detected in file\n",
      "\n",
      "Complement found from 15095 to 15103\n",
      "Seq is: TTATCCTC\n",
      "1 TTATCCTC motifs detected in file\n",
      "\n",
      "Seq is: CGGATCCA\n",
      "0 CGGATCCA motifs detected in file\n",
      "\n",
      "Complement found from 12910 to 12918\n",
      "Seq is: ACTCCAAC\n",
      "1 ACTCCAAC motifs detected in file\n",
      "\n",
      "Seq is: TGGTCCCC\n",
      "0 TGGTCCCC motifs detected in file\n",
      "\n",
      "Seq is: CAACCCAT\n",
      "0 CAACCCAT motifs detected in file\n",
      "\n",
      "Complement found from 15535 to 15543\n",
      "Seq is: ACCCCTCC\n",
      "1 ACCCCTCC motifs detected in file\n",
      "\n",
      "Seq is: CCCCAAGT\n",
      "0 CCCCAAGT motifs detected in file\n",
      "\n",
      "Complement found from 9096 to 9104\n",
      "Seq is: ATCATCTT\n",
      "1 ATCATCTT motifs detected in file\n",
      "\n",
      "Seq is: CTCGCTAC\n",
      "0 CTCGCTAC motifs detected in file\n",
      "\n",
      "Seq is: CACTCTCG\n",
      "0 CACTCTCG motifs detected in file\n",
      "\n",
      "Seq is: ATTCCAGC\n",
      "0 ATTCCAGC motifs detected in file\n",
      "\n",
      "Complement found from 3492 to 3500\n",
      "Seq is: CCCGCCAC\n",
      "1 CCCGCCAC motifs detected in file\n",
      "\n",
      "Seq is: TACATCTC\n",
      "0 TACATCTC motifs detected in file\n",
      "\n",
      "Complement found from 2217 to 2225\n",
      "Seq is: CCACTACC\n",
      "1 CCACTACC motifs detected in file\n",
      "\n",
      "Seq is: ATCTACAC\n",
      "0 ATCTACAC motifs detected in file\n",
      "\n",
      "Seq is: CGCCCAAA\n",
      "0 CGCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: ATCCCCGA\n",
      "0 ATCCCCGA motifs detected in file\n",
      "\n",
      "Complement found from 11125 to 11133\n",
      "Seq is: GAAACCAC\n",
      "1 GAAACCAC motifs detected in file\n",
      "\n",
      "Seq is: TTCTCAAA\n",
      "0 TTCTCAAA motifs detected in file\n",
      "\n",
      "Seq is: ATACCGCA\n",
      "0 ATACCGCA motifs detected in file\n",
      "\n",
      "Seq is: GTCGCTTC\n",
      "0 GTCGCTTC motifs detected in file\n",
      "\n",
      "Complement found from 14819 to 14827\n",
      "Seq is: CCAACATC\n",
      "1 CCAACATC motifs detected in file\n",
      "\n",
      "Seq is: ATCGGGCA\n",
      "0 ATCGGGCA motifs detected in file\n",
      "\n",
      "Seq is: TCCCCGGA\n",
      "0 TCCCCGGA motifs detected in file\n",
      "\n",
      "Seq is: TGTTGCAA\n",
      "0 TGTTGCAA motifs detected in file\n",
      "\n",
      "Seq is: CCCCGGAA\n",
      "0 CCCCGGAA motifs detected in file\n",
      "\n",
      "Seq is: ACGCATCA\n",
      "0 ACGCATCA motifs detected in file\n",
      "\n",
      "Seq is: ACATATAT\n",
      "0 ACATATAT motifs detected in file\n",
      "\n",
      "Seq is: CGGATCTT\n",
      "0 CGGATCTT motifs detected in file\n",
      "\n",
      "Seq is: AAAAAGCA\n",
      "0 AAAAAGCA motifs detected in file\n",
      "\n",
      "Complement found from 11655 to 11663\n",
      "Complement found from 12836 to 12844\n",
      "Seq is: AGCCATTC\n",
      "2 AGCCATTC motifs detected in file\n",
      "\n",
      "Seq is: GTAATCCT\n",
      "0 GTAATCCT motifs detected in file\n",
      "\n",
      "Seq is: TACGGTAA\n",
      "0 TACGGTAA motifs detected in file\n",
      "\n",
      "Complement found from 12950 to 12958\n",
      "Seq is: CGCTAATC\n",
      "1 CGCTAATC motifs detected in file\n",
      "\n",
      "Seq is: TTACCCAT\n",
      "0 TTACCCAT motifs detected in file\n",
      "\n",
      "Seq is: GTTACTCC\n",
      "0 GTTACTCC motifs detected in file\n",
      "\n",
      "Complement found from 12880 to 12888\n",
      "Seq is: TCATCCTC\n",
      "1 TCATCCTC motifs detected in file\n",
      "\n",
      "Seq is: CAACCGGT\n",
      "0 CAACCGGT motifs detected in file\n",
      "\n",
      "Seq is: ACCCATAC\n",
      "0 ACCCATAC motifs detected in file\n",
      "\n",
      "Seq is: AACAATTC\n",
      "0 AACAATTC motifs detected in file\n",
      "\n",
      "Complement found from 6344 to 6352\n",
      "Seq is: TTCTCCTT\n",
      "1 TTCTCCTT motifs detected in file\n",
      "\n",
      "Seq is: AACTTGGA\n",
      "0 AACTTGGA motifs detected in file\n",
      "\n",
      "Seq is: GACTGACA\n",
      "0 GACTGACA motifs detected in file\n",
      "\n",
      "Complement found from 7510 to 7518\n",
      "Seq is: TTTCAAAA\n",
      "1 TTTCAAAA motifs detected in file\n",
      "\n",
      "Seq is: TTCAAAAT\n",
      "0 TTCAAAAT motifs detected in file\n",
      "\n",
      "Complement found from 12949 to 12957\n",
      "Seq is: ACGCTAAT\n",
      "1 ACGCTAAT motifs detected in file\n",
      "\n",
      "Seq is: GGCCCGAG\n",
      "0 GGCCCGAG motifs detected in file\n",
      "\n",
      "Seq is: ACGGTAGA\n",
      "0 ACGGTAGA motifs detected in file\n",
      "\n",
      "Seq is: ATTGTTTG\n",
      "0 ATTGTTTG motifs detected in file\n",
      "\n",
      "Complement found from 3130 to 3138\n",
      "Seq is: GGACAAGA\n",
      "1 GGACAAGA motifs detected in file\n",
      "\n",
      "Complement found from 3208 to 3216\n",
      "Complement found from 5431 to 5439\n",
      "Complement found from 16288 to 16296\n",
      "Seq is: ACCCACCC\n",
      "3 ACCCACCC motifs detected in file\n",
      "\n",
      "Complement found from 2160 to 2168\n",
      "Seq is: ACACCCAT\n",
      "1 ACACCCAT motifs detected in file\n",
      "\n",
      "Complement found from 10446 to 10454\n",
      "Seq is: ATTATGAT\n",
      "1 ATTATGAT motifs detected in file\n",
      "\n",
      "Seq is: TCAACTCT\n",
      "0 TCAACTCT motifs detected in file\n",
      "\n",
      "Complement found from 13086 to 13094\n",
      "Seq is: ACTATAGT\n",
      "1 ACTATAGT motifs detected in file\n",
      "\n",
      "Complement found from 8363 to 8371\n",
      "Complement found from 8891 to 8899\n",
      "Complement found from 10467 to 10475\n",
      "Seq is: AAATGCCC\n",
      "3 AAATGCCC motifs detected in file\n",
      "\n",
      "Seq is: CCTTCCGC\n",
      "0 CCTTCCGC motifs detected in file\n",
      "\n",
      "Seq is: GAAACACT\n",
      "0 GAAACACT motifs detected in file\n",
      "\n",
      "Seq is: TCATCCGG\n",
      "0 TCATCCGG motifs detected in file\n",
      "\n",
      "Seq is: AATAAAGA\n",
      "0 AATAAAGA motifs detected in file\n",
      "\n",
      "Seq is: GAACAGGA\n",
      "0 GAACAGGA motifs detected in file\n",
      "\n",
      "Seq is: AAGCATGT\n",
      "0 AAGCATGT motifs detected in file\n",
      "\n",
      "Complement found from 5325 to 5333\n",
      "Complement found from 10266 to 10274\n",
      "Seq is: CCCTCCTT\n",
      "2 CCCTCCTT motifs detected in file\n",
      "\n",
      "Seq is: AAACTTNC\n",
      "0 AAACTTNC motifs detected in file\n",
      "\n",
      "Seq is: ATCTATCT\n",
      "0 ATCTATCT motifs detected in file\n",
      "\n",
      "Seq is: TTGGCTGG\n",
      "0 TTGGCTGG motifs detected in file\n",
      "\n",
      "Seq is: ACCTAATG\n",
      "0 ACCTAATG motifs detected in file\n",
      "\n",
      "Seq is: AGGCCAGA\n",
      "0 AGGCCAGA motifs detected in file\n",
      "\n",
      "Seq is: CTTGAGTC\n",
      "0 CTTGAGTC motifs detected in file\n",
      "\n",
      "Seq is: TAGTGCAT\n",
      "0 TAGTGCAT motifs detected in file\n",
      "\n",
      "Seq is: CCTGAAAT\n",
      "0 CCTGAAAT motifs detected in file\n",
      "\n",
      "Seq is: TAGCAACT\n",
      "0 TAGCAACT motifs detected in file\n",
      "\n",
      "Seq is: TGTTTGCT\n",
      "0 TGTTTGCT motifs detected in file\n",
      "\n",
      "Seq is: TGGAAATT\n",
      "0 TGGAAATT motifs detected in file\n",
      "\n",
      "Seq is: ATCGCCGA\n",
      "0 ATCGCCGA motifs detected in file\n",
      "\n",
      "Seq is: CGTGGTAG\n",
      "0 CGTGGTAG motifs detected in file\n",
      "\n",
      "Seq is: CCCTACAG\n",
      "0 CCCTACAG motifs detected in file\n",
      "\n",
      "Seq is: GACTAGGT\n",
      "0 GACTAGGT motifs detected in file\n",
      "\n",
      "Seq is: TGTAGCTC\n",
      "0 TGTAGCTC motifs detected in file\n",
      "\n",
      "Seq is: CAGCATTT\n",
      "0 CAGCATTT motifs detected in file\n",
      "\n",
      "Complement found from 2931 to 2939\n",
      "Seq is: GGGATAAC\n",
      "1 GGGATAAC motifs detected in file\n",
      "\n",
      "Seq is: AACTATAC\n",
      "0 AACTATAC motifs detected in file\n",
      "\n",
      "Seq is: CTGAGATC\n",
      "0 CTGAGATC motifs detected in file\n",
      "\n",
      "Complement found from 5094 to 5102\n",
      "Complement found from 6639 to 6647\n",
      "Complement found from 12901 to 12909\n",
      "Seq is: TTATCCTA\n",
      "3 TTATCCTA motifs detected in file\n",
      "\n",
      "Seq is: ACGCGACA\n",
      "0 ACGCGACA motifs detected in file\n",
      "\n",
      "Seq is: ATAGGGAT\n",
      "0 ATAGGGAT motifs detected in file\n",
      "\n",
      "Seq is: CCCATTGA\n",
      "0 CCCATTGA motifs detected in file\n",
      "\n",
      "Complement found from 8915 to 8923\n",
      "Seq is: ACAAGGCA\n",
      "1 ACAAGGCA motifs detected in file\n",
      "\n",
      "Seq is: ACCAGTTC\n",
      "0 ACCAGTTC motifs detected in file\n",
      "\n",
      "Seq is: AATAACCT\n",
      "0 AATAACCT motifs detected in file\n",
      "\n",
      "Seq is: AGTTAACT\n",
      "0 AGTTAACT motifs detected in file\n",
      "\n",
      "Complement found from 13535 to 13543\n",
      "Seq is: CATATCAT\n",
      "1 CATATCAT motifs detected in file\n",
      "\n",
      "Seq is: CAAGCGAA\n",
      "0 CAAGCGAA motifs detected in file\n",
      "\n",
      "Seq is: ACTGACCA\n",
      "0 ACTGACCA motifs detected in file\n",
      "\n",
      "Seq is: CTTCAGAA\n",
      "0 CTTCAGAA motifs detected in file\n",
      "\n",
      "Complement found from 16093 to 16101\n",
      "Seq is: TCGTACAT\n",
      "1 TCGTACAT motifs detected in file\n",
      "\n",
      "Seq is: GACGAGCC\n",
      "0 GACGAGCC motifs detected in file\n",
      "\n",
      "Seq is: TCCAACCC\n",
      "0 TCCAACCC motifs detected in file\n",
      "\n",
      "Complement found from 2713 to 2721\n",
      "Seq is: AAGACGAG\n",
      "1 AAGACGAG motifs detected in file\n",
      "\n",
      "Seq is: GCTCCAGC\n",
      "0 GCTCCAGC motifs detected in file\n",
      "\n",
      "Seq is: GGGGTTGG\n",
      "0 GGGGTTGG motifs detected in file\n",
      "\n",
      "Seq is: CTTTAAAA\n",
      "0 CTTTAAAA motifs detected in file\n",
      "\n",
      "Seq is: ATTACGTC\n",
      "0 ATTACGTC motifs detected in file\n",
      "\n",
      "Complement found from 1720 to 1728\n",
      "Seq is: CAAACCAT\n",
      "1 CAAACCAT motifs detected in file\n",
      "\n",
      "Seq is: CAAATCCT\n",
      "0 CAAATCCT motifs detected in file\n",
      "\n",
      "Seq is: GGACACCC\n",
      "0 GGACACCC motifs detected in file\n",
      "\n",
      "Complement found from 3718 to 3726\n",
      "Seq is: AAACAATC\n",
      "1 AAACAATC motifs detected in file\n",
      "\n",
      "Complement found from 4863 to 4871\n",
      "Seq is: CATGACAA\n",
      "1 CATGACAA motifs detected in file\n",
      "\n",
      "Seq is: ACGTAATT\n",
      "0 ACGTAATT motifs detected in file\n",
      "\n",
      "Seq is: ATTTAATT\n",
      "0 ATTTAATT motifs detected in file\n",
      "\n",
      "Seq is: TCGAGGTA\n",
      "0 TCGAGGTA motifs detected in file\n",
      "\n",
      "Seq is: TCCCAGAA\n",
      "0 TCCCAGAA motifs detected in file\n",
      "\n",
      "Complement found from 2825 to 2833\n",
      "Seq is: GAGCAGAA\n",
      "1 GAGCAGAA motifs detected in file\n",
      "\n",
      "Seq is: CGACACAA\n",
      "0 CGACACAA motifs detected in file\n",
      "\n",
      "Seq is: TACGGGCG\n",
      "0 TACGGGCG motifs detected in file\n",
      "\n",
      "Seq is: GAGAAGTG\n",
      "0 GAGAAGTG motifs detected in file\n",
      "\n",
      "Seq is: CCCGTCCA\n",
      "0 CCCGTCCA motifs detected in file\n",
      "\n",
      "Seq is: GTTAAAGT\n",
      "0 GTTAAAGT motifs detected in file\n",
      "\n",
      "Seq is: GACCAATT\n",
      "0 GACCAATT motifs detected in file\n",
      "\n",
      "Seq is: TTCATTCT\n",
      "0 TTCATTCT motifs detected in file\n",
      "\n",
      "Seq is: CTGTCGAC\n",
      "0 CTGTCGAC motifs detected in file\n",
      "\n",
      "Seq is: TTGGGAGC\n",
      "0 TTGGGAGC motifs detected in file\n",
      "\n",
      "Seq is: ACCTCGGT\n",
      "0 ACCTCGGT motifs detected in file\n",
      "\n",
      "Seq is: AAGTATGT\n",
      "0 AAGTATGT motifs detected in file\n",
      "\n",
      "Seq is: CCAGGGAT\n",
      "0 CCAGGGAT motifs detected in file\n",
      "\n",
      "Complement found from 3110 to 3118\n",
      "Seq is: AAATTCCT\n",
      "1 AAATTCCT motifs detected in file\n",
      "\n",
      "Seq is: TGTTCACT\n",
      "0 TGTTCACT motifs detected in file\n",
      "\n",
      "Seq is: AATACGAT\n",
      "0 AATACGAT motifs detected in file\n",
      "\n",
      "Seq is: GGAAACGT\n",
      "0 GGAAACGT motifs detected in file\n",
      "\n",
      "Seq is: GCCAATCC\n",
      "0 GCCAATCC motifs detected in file\n",
      "\n",
      "Seq is: CATGGCGC\n",
      "0 CATGGCGC motifs detected in file\n",
      "\n",
      "Seq is: CGGCAATT\n",
      "0 CGGCAATT motifs detected in file\n",
      "\n",
      "Seq is: TGTACACA\n",
      "0 TGTACACA motifs detected in file\n",
      "\n",
      "Seq is: GTGACCCG\n",
      "0 GTGACCCG motifs detected in file\n",
      "\n",
      "Seq is: TCCGCCAC\n",
      "0 TCCGCCAC motifs detected in file\n",
      "\n",
      "Seq is: GGAGATTA\n",
      "0 GGAGATTA motifs detected in file\n",
      "\n",
      "Seq is: TGACCACT\n",
      "0 TGACCACT motifs detected in file\n",
      "\n",
      "Seq is: ACGATCTC\n",
      "0 ACGATCTC motifs detected in file\n",
      "\n",
      "Complement found from 4019 to 4027\n",
      "Seq is: CACTACAA\n",
      "1 CACTACAA motifs detected in file\n",
      "\n",
      "Complement found from 1721 to 1729\n",
      "Complement found from 7529 to 7537\n",
      "Seq is: AAACCATT\n",
      "2 AAACCATT motifs detected in file\n",
      "\n",
      "Seq is: TGTCCGCC\n",
      "0 TGTCCGCC motifs detected in file\n",
      "\n",
      "Complement found from 3340 to 3348\n",
      "Complement found from 6585 to 6593\n",
      "Seq is: CCATTCTA\n",
      "2 CCATTCTA motifs detected in file\n",
      "\n",
      "Seq is: AACGGCTC\n",
      "0 AACGGCTC motifs detected in file\n",
      "\n",
      "Seq is: AAGGAAAA\n",
      "0 AAGGAAAA motifs detected in file\n",
      "\n",
      "Complement found from 4173 to 4181\n",
      "Seq is: TGAAAAAA\n",
      "1 TGAAAAAA motifs detected in file\n",
      "\n",
      "Seq is: ATTGGAAA\n",
      "0 ATTGGAAA motifs detected in file\n",
      "\n",
      "Complement found from 10520 to 10528\n",
      "Seq is: GGAATACT\n",
      "1 GGAATACT motifs detected in file\n",
      "\n",
      "Seq is: CGTACGGA\n",
      "0 CGTACGGA motifs detected in file\n",
      "\n",
      "Complement found from 12542 to 12550\n",
      "Seq is: CACAACCC\n",
      "1 CACAACCC motifs detected in file\n",
      "\n",
      "Seq is: AACTGTCA\n",
      "0 AACTGTCA motifs detected in file\n",
      "\n",
      "Complement found from 15381 to 15389\n",
      "Seq is: CTCCCATT\n",
      "1 CTCCCATT motifs detected in file\n",
      "\n",
      "Seq is: ATTACTGA\n",
      "0 ATTACTGA motifs detected in file\n",
      "\n",
      "Seq is: ACAACCAA\n",
      "0 ACAACCAA motifs detected in file\n",
      "\n",
      "Complement found from 7742 to 7750\n",
      "Seq is: CTAACATC\n",
      "1 CTAACATC motifs detected in file\n",
      "\n",
      "Seq is: TATAACCC\n",
      "0 TATAACCC motifs detected in file\n",
      "\n",
      "Complement found from 2365 to 2373\n",
      "Complement found from 15505 to 15513\n",
      "Seq is: GACAATTA\n",
      "2 GACAATTA motifs detected in file\n",
      "\n",
      "Complement found from 16342 to 16350\n",
      "Seq is: ACAGTCAA\n",
      "1 ACAGTCAA motifs detected in file\n",
      "\n",
      "Seq is: GTCACAAA\n",
      "0 GTCACAAA motifs detected in file\n",
      "\n",
      "Complement found from 10387 to 10395\n",
      "Seq is: ATTAGACT\n",
      "1 ATTAGACT motifs detected in file\n",
      "\n",
      "Seq is: GCGTCCGA\n",
      "0 GCGTCCGA motifs detected in file\n",
      "\n",
      "Complement found from 11488 to 11496\n",
      "Seq is: ATACGCCT\n",
      "1 ATACGCCT motifs detected in file\n",
      "\n",
      "Seq is: CCTCTTAC\n",
      "0 CCTCTTAC motifs detected in file\n",
      "\n",
      "Seq is: AAAAGTAC\n",
      "0 AAAAGTAC motifs detected in file\n",
      "\n",
      "Seq is: AATGAATA\n",
      "0 AATGAATA motifs detected in file\n",
      "\n",
      "Seq is: TGATTGTA\n",
      "0 TGATTGTA motifs detected in file\n",
      "\n",
      "Seq is: ATCAAGAA\n",
      "0 ATCAAGAA motifs detected in file\n",
      "\n",
      "Seq is: GATATCCC\n",
      "0 GATATCCC motifs detected in file\n",
      "\n",
      "Complement found from 8816 to 8824\n",
      "Seq is: ACTATCTA\n",
      "1 ACTATCTA motifs detected in file\n",
      "\n",
      "Seq is: ACCAGGTT\n",
      "0 ACCAGGTT motifs detected in file\n",
      "\n",
      "Complement found from 2767 to 2775\n",
      "Complement found from 5674 to 5682\n",
      "Complement found from 14639 to 14647\n",
      "Seq is: AACCCACA\n",
      "3 AACCCACA motifs detected in file\n",
      "\n",
      "Complement found from 591 to 599\n",
      "Complement found from 1494 to 1502\n",
      "Complement found from 5011 to 5019\n",
      "Complement found from 14442 to 14450\n",
      "Seq is: CTCCTCAA\n",
      "4 CTCCTCAA motifs detected in file\n",
      "\n",
      "Seq is: GTCAATAT\n",
      "0 GTCAATAT motifs detected in file\n",
      "\n",
      "Complement found from 2764 to 2772\n",
      "Complement found from 8507 to 8515\n",
      "Complement found from 14623 to 14631\n",
      "Seq is: ACAAACCC\n",
      "3 ACAAACCC motifs detected in file\n",
      "\n",
      "Complement found from 2149 to 2157\n",
      "Complement found from 2225 to 2233\n",
      "Complement found from 8495 to 8503\n",
      "Seq is: TAAAAAAT\n",
      "3 TAAAAAAT motifs detected in file\n",
      "\n",
      "Complement found from 3505 to 3513\n",
      "Complement found from 5319 to 5327\n",
      "Seq is: CCATCACC\n",
      "2 CCATCACC motifs detected in file\n",
      "\n",
      "Complement found from 10131 to 10139\n",
      "Seq is: CACAACTC\n",
      "1 CACAACTC motifs detected in file\n",
      "\n",
      "Seq is: GAACTTGC\n",
      "0 GAACTTGC motifs detected in file\n",
      "\n",
      "Seq is: GAAAGAAT\n",
      "0 GAAAGAAT motifs detected in file\n",
      "\n",
      "Complement found from 12355 to 12363\n",
      "Seq is: TAACCACC\n",
      "1 TAACCACC motifs detected in file\n",
      "\n",
      "Seq is: GACGAAAA\n",
      "0 GACGAAAA motifs detected in file\n",
      "\n",
      "Seq is: TCCGGATG\n",
      "0 TCCGGATG motifs detected in file\n",
      "\n",
      "Complement found from 3200 to 3208\n",
      "Seq is: ATACCCAC\n",
      "1 ATACCCAC motifs detected in file\n",
      "\n",
      "Complement found from 16005 to 16013\n",
      "Seq is: AATTTAAA\n",
      "1 AATTTAAA motifs detected in file\n",
      "\n",
      "Seq is: AAATGAGA\n",
      "0 AAATGAGA motifs detected in file\n",
      "\n",
      "Seq is: GAGATGTT\n",
      "0 GAGATGTT motifs detected in file\n",
      "\n",
      "Complement found from 4602 to 4610\n",
      "Complement found from 10811 to 10819\n",
      "Seq is: CCAAAAAA\n",
      "2 CCAAAAAA motifs detected in file\n",
      "\n",
      "Seq is: GGATCACA\n",
      "0 GGATCACA motifs detected in file\n",
      "\n",
      "Seq is: GGTTTCTC\n",
      "0 GGTTTCTC motifs detected in file\n",
      "\n",
      "Seq is: GACAAGGA\n",
      "0 GACAAGGA motifs detected in file\n",
      "\n",
      "Complement found from 1759 to 1767\n",
      "Complement found from 15076 to 15084\n",
      "Seq is: GAAACCTG\n",
      "2 GAAACCTG motifs detected in file\n",
      "\n",
      "Seq is: ATTGTCAA\n",
      "0 ATTGTCAA motifs detected in file\n",
      "\n",
      "Seq is: TTTAAATG\n",
      "0 TTTAAATG motifs detected in file\n",
      "\n",
      "Complement found from 6150 to 6158\n",
      "Complement found from 7092 to 7100\n",
      "Complement found from 7623 to 7631\n",
      "Complement found from 8225 to 8233\n",
      "Complement found from 11231 to 11239\n",
      "Seq is: TTCCCCTA\n",
      "5 TTCCCCTA motifs detected in file\n",
      "\n",
      "Complement found from 6661 to 6669\n",
      "Complement found from 11055 to 11063\n",
      "Seq is: AATCTCCC\n",
      "2 AATCTCCC motifs detected in file\n",
      "\n",
      "Complement found from 561 to 569\n",
      "Seq is: AAGACACC\n",
      "1 AAGACACC motifs detected in file\n",
      "\n",
      "Seq is: CGTTTAAA\n",
      "0 CGTTTAAA motifs detected in file\n",
      "\n",
      "Complement found from 1645 to 1653\n",
      "Complement found from 14726 to 14734\n",
      "Seq is: TTTCAACT\n",
      "2 TTTCAACT motifs detected in file\n",
      "\n",
      "Seq is: TGATTCTA\n",
      "0 TGATTCTA motifs detected in file\n",
      "\n",
      "Complement found from 2026 to 2034\n",
      "Seq is: AGATAGAA\n",
      "1 AGATAGAA motifs detected in file\n",
      "\n",
      "Seq is: CCTGTTGG\n",
      "0 CCTGTTGG motifs detected in file\n",
      "\n",
      "Seq is: TCGATAGT\n",
      "0 TCGATAGT motifs detected in file\n",
      "\n",
      "Seq is: GGTCCGAG\n",
      "0 GGTCCGAG motifs detected in file\n",
      "\n",
      "Complement found from 14815 to 14823\n",
      "Seq is: CCATCCAA\n",
      "1 CCATCCAA motifs detected in file\n",
      "\n",
      "Seq is: ACAGCGGA\n",
      "0 ACAGCGGA motifs detected in file\n",
      "\n",
      "Seq is: GATGGATA\n",
      "0 GATGGATA motifs detected in file\n",
      "\n",
      "Seq is: TTTAGAAG\n",
      "0 TTTAGAAG motifs detected in file\n",
      "\n",
      "Seq is: GGTGATAA\n",
      "0 GGTGATAA motifs detected in file\n",
      "\n",
      "Seq is: AACGATGT\n",
      "0 AACGATGT motifs detected in file\n",
      "\n",
      "Seq is: ATCTGCCC\n",
      "0 ATCTGCCC motifs detected in file\n",
      "\n",
      "Complement found from 9389 to 9397\n",
      "Complement found from 12049 to 12057\n",
      "Seq is: ACACGAGA\n",
      "2 ACACGAGA motifs detected in file\n",
      "\n",
      "Seq is: AAATCGAC\n",
      "0 AAATCGAC motifs detected in file\n",
      "\n",
      "Seq is: AAGAATCC\n",
      "0 AAGAATCC motifs detected in file\n",
      "\n",
      "Seq is: ATCGAGCA\n",
      "0 ATCGAGCA motifs detected in file\n",
      "\n",
      "Seq is: GACCAAAG\n",
      "0 GACCAAAG motifs detected in file\n",
      "\n",
      "Seq is: CCCCCAGA\n",
      "0 CCCCCAGA motifs detected in file\n",
      "\n",
      "Complement found from 8955 to 8963\n",
      "Complement found from 13521 to 13529\n",
      "Seq is: ATCGAAAC\n",
      "2 ATCGAAAC motifs detected in file\n",
      "\n",
      "Seq is: CGAGAGGA\n",
      "0 CGAGAGGA motifs detected in file\n",
      "\n",
      "Seq is: ACGTTTCA\n",
      "0 ACGTTTCA motifs detected in file\n",
      "\n",
      "Seq is: ATAAAGAT\n",
      "0 ATAAAGAT motifs detected in file\n",
      "\n",
      "Complement found from 2189 to 2197\n",
      "Seq is: CAATTAAG\n",
      "1 CAATTAAG motifs detected in file\n",
      "\n",
      "Seq is: TAATACGT\n",
      "0 TAATACGT motifs detected in file\n",
      "\n",
      "Seq is: CTTCCATA\n",
      "0 CTTCCATA motifs detected in file\n",
      "\n",
      "Complement found from 960 to 968\n",
      "Seq is: TCCCCAAT\n",
      "1 TCCCCAAT motifs detected in file\n",
      "\n",
      "Seq is: CAGGAACG\n",
      "0 CAGGAACG motifs detected in file\n",
      "\n",
      "Seq is: ATATAATA\n",
      "0 ATATAATA motifs detected in file\n",
      "\n",
      "Complement found from 543 to 551\n",
      "Seq is: CGAACCAA\n",
      "1 CGAACCAA motifs detected in file\n",
      "\n",
      "Seq is: TATTAAAA\n",
      "0 TATTAAAA motifs detected in file\n",
      "\n",
      "Seq is: AGTAGAAA\n",
      "0 AGTAGAAA motifs detected in file\n",
      "\n",
      "Seq is: GGGAACGC\n",
      "0 GGGAACGC motifs detected in file\n",
      "\n",
      "Seq is: CATGATAT\n",
      "0 CATGATAT motifs detected in file\n",
      "\n",
      "Seq is: AGATAACG\n",
      "0 AGATAACG motifs detected in file\n",
      "\n",
      "Seq is: CGGTCCAA\n",
      "0 CGGTCCAA motifs detected in file\n",
      "\n",
      "Complement found from 8326 to 8334\n",
      "Seq is: AGTTAAAG\n",
      "1 AGTTAAAG motifs detected in file\n",
      "\n",
      "Seq is: ATAGCGGA\n",
      "0 ATAGCGGA motifs detected in file\n",
      "\n",
      "Complement found from 6895 to 6903\n",
      "Seq is: TATGAAAT\n",
      "1 TATGAAAT motifs detected in file\n",
      "\n",
      "Complement found from 14515 to 14523\n",
      "Seq is: AAACCCAT\n",
      "1 AAACCCAT motifs detected in file\n",
      "\n",
      "Complement found from 2497 to 2505\n",
      "Complement found from 10462 to 10470\n",
      "Seq is: TTACCAAA\n",
      "2 TTACCAAA motifs detected in file\n",
      "\n",
      "Complement found from 4139 to 4147\n",
      "Seq is: CCGATTCC\n",
      "1 CCGATTCC motifs detected in file\n",
      "\n",
      "Complement found from 6532 to 6540\n",
      "Complement found from 11595 to 11603\n",
      "Seq is: AACAGACC\n",
      "2 AACAGACC motifs detected in file\n",
      "\n",
      "Complement found from 14482 to 14490\n",
      "Seq is: ATCATTCC\n",
      "1 ATCATTCC motifs detected in file\n",
      "\n",
      "Complement found from 1231 to 1239\n",
      "Complement found from 5888 to 5896\n",
      "Seq is: ACCTCACC\n",
      "2 ACCTCACC motifs detected in file\n",
      "\n",
      "Complement found from 298 to 306\n",
      "Complement found from 348 to 356\n",
      "Complement found from 552 to 560\n",
      "Complement found from 11668 to 11676\n",
      "Complement found from 14624 to 14632\n",
      "Seq is: CAAACCCC\n",
      "5 CAAACCCC motifs detected in file\n",
      "\n",
      "Seq is: GATCCAAA\n",
      "0 GATCCAAA motifs detected in file\n",
      "\n",
      "Complement found from 9750 to 9758\n",
      "Seq is: TCGAGTCT\n",
      "1 TCGAGTCT motifs detected in file\n",
      "\n",
      "Seq is: CGCCAGTT\n",
      "0 CGCCAGTT motifs detected in file\n",
      "\n",
      "Complement found from 5797 to 5805\n",
      "Seq is: CAATTCAA\n",
      "1 CAATTCAA motifs detected in file\n",
      "\n",
      "Seq is: CTTTAGAG\n",
      "0 CTTTAGAG motifs detected in file\n",
      "\n",
      "Seq is: GATTCACA\n",
      "0 GATTCACA motifs detected in file\n",
      "\n",
      "Seq is: TTCAACCC\n",
      "0 TTCAACCC motifs detected in file\n",
      "\n",
      "Seq is: ACGAAACA\n",
      "0 ACGAAACA motifs detected in file\n",
      "\n",
      "Seq is: CAATTCGA\n",
      "0 CAATTCGA motifs detected in file\n",
      "\n",
      "Seq is: TGTGAGAC\n",
      "0 TGTGAGAC motifs detected in file\n",
      "\n",
      "Seq is: CAAGCAGG\n",
      "0 CAAGCAGG motifs detected in file\n",
      "\n",
      "Seq is: TTCACGTG\n",
      "0 TTCACGTG motifs detected in file\n",
      "\n",
      "Seq is: AAAGGTCA\n",
      "0 AAAGGTCA motifs detected in file\n",
      "\n",
      "Seq is: TGTGAATG\n",
      "0 TGTGAATG motifs detected in file\n",
      "\n",
      "Seq is: GTACAATG\n",
      "0 GTACAATG motifs detected in file\n",
      "\n",
      "Seq is: CTGAACAG\n",
      "0 CTGAACAG motifs detected in file\n",
      "\n",
      "Seq is: AGGAGATA\n",
      "0 AGGAGATA motifs detected in file\n",
      "\n",
      "Seq is: TATTTACG\n",
      "0 TATTTACG motifs detected in file\n",
      "\n",
      "Complement found from 5299 to 5307\n",
      "Seq is: CATCCCCA\n",
      "1 CATCCCCA motifs detected in file\n",
      "\n",
      "Complement found from 7876 to 7884\n",
      "Complement found from 16414 to 16422\n",
      "Seq is: AAATCAAT\n",
      "2 AAATCAAT motifs detected in file\n",
      "\n",
      "Complement found from 11942 to 11950\n",
      "Seq is: TTACAGGA\n",
      "1 TTACAGGA motifs detected in file\n",
      "\n",
      "Seq is: AACTTCAT\n",
      "0 AACTTCAT motifs detected in file\n",
      "\n",
      "Seq is: ATGAACTC\n",
      "0 ATGAACTC motifs detected in file\n",
      "\n",
      "Complement found from 458 to 466\n",
      "Complement found from 11812 to 11820\n",
      "Seq is: CTCCCACT\n",
      "2 CTCCCACT motifs detected in file\n",
      "\n",
      "Complement found from 7788 to 7796\n",
      "Seq is: GCCCGCCA\n",
      "1 GCCCGCCA motifs detected in file\n",
      "\n",
      "Seq is: CACATGCG\n",
      "0 CACATGCG motifs detected in file\n",
      "\n",
      "Seq is: CGAAGTCC\n",
      "0 CGAAGTCC motifs detected in file\n",
      "\n",
      "Seq is: CGGGACAA\n",
      "0 CGGGACAA motifs detected in file\n",
      "\n",
      "Seq is: GTTGATTC\n",
      "0 GTTGATTC motifs detected in file\n",
      "\n",
      "Seq is: GTGAGATG\n",
      "0 GTGAGATG motifs detected in file\n",
      "\n",
      "Seq is: AGAATCAA\n",
      "0 AGAATCAA motifs detected in file\n",
      "\n",
      "Seq is: ATGACGAT\n",
      "0 ATGACGAT motifs detected in file\n",
      "\n",
      "Seq is: TTAGGTGG\n",
      "0 TTAGGTGG motifs detected in file\n",
      "\n",
      "Seq is: AAGCTGGG\n",
      "0 AAGCTGGG motifs detected in file\n",
      "\n",
      "Seq is: AATTCAAA\n",
      "0 AATTCAAA motifs detected in file\n",
      "\n",
      "Seq is: GTATTCCC\n",
      "0 GTATTCCC motifs detected in file\n",
      "\n",
      "Complement found from 80 to 88\n",
      "Seq is: GATAGCAT\n",
      "1 GATAGCAT motifs detected in file\n",
      "\n",
      "Seq is: CAAAAGAC\n",
      "0 CAAAAGAC motifs detected in file\n",
      "\n",
      "Seq is: CCCATCTT\n",
      "0 CCCATCTT motifs detected in file\n",
      "\n",
      "Seq is: TTACATCG\n",
      "0 TTACATCG motifs detected in file\n",
      "\n",
      "Seq is: GGTAAAGA\n",
      "0 GGTAAAGA motifs detected in file\n",
      "\n",
      "Seq is: ACGGTGGA\n",
      "0 ACGGTGGA motifs detected in file\n",
      "\n",
      "Seq is: GTACCCGA\n",
      "0 GTACCCGA motifs detected in file\n",
      "\n",
      "Seq is: TGTGGAAC\n",
      "0 TGTGGAAC motifs detected in file\n",
      "\n",
      "Seq is: TGGATTGC\n",
      "0 TGGATTGC motifs detected in file\n",
      "\n",
      "Seq is: AGAAATGC\n",
      "0 AGAAATGC motifs detected in file\n",
      "\n",
      "Seq is: ACCCATGA\n",
      "0 ACCCATGA motifs detected in file\n",
      "\n",
      "Seq is: ACGCGAAT\n",
      "0 ACGCGAAT motifs detected in file\n",
      "\n",
      "Complement found from 7251 to 7259\n",
      "Complement found from 15083 to 15091\n",
      "Seq is: GAAACATC\n",
      "2 GAAACATC motifs detected in file\n",
      "\n",
      "Seq is: GGAAGTAG\n",
      "0 GGAAGTAG motifs detected in file\n",
      "\n",
      "Complement found from 2232 to 2240\n",
      "Seq is: TCCCAAAC\n",
      "1 TCCCAAAC motifs detected in file\n",
      "\n",
      "Complement found from 11834 to 11842\n",
      "Complement found from 13825 to 13833\n",
      "Seq is: GACTTCTA\n",
      "2 GACTTCTA motifs detected in file\n",
      "\n",
      "Seq is: CCGCCATA\n",
      "0 CCGCCATA motifs detected in file\n",
      "\n",
      "Complement found from 16490 to 16498\n",
      "Seq is: TATCCGAC\n",
      "1 TATCCGAC motifs detected in file\n",
      "\n",
      "Seq is: TCGTTCTC\n",
      "0 TCGTTCTC motifs detected in file\n",
      "\n",
      "Complement found from 13184 to 13192\n",
      "Seq is: CACCACTC\n",
      "1 CACCACTC motifs detected in file\n",
      "\n",
      "Seq is: CAACTAGC\n",
      "0 CAACTAGC motifs detected in file\n",
      "\n",
      "Complement found from 1731 to 1739\n",
      "Seq is: CCCAAATA\n",
      "1 CCCAAATA motifs detected in file\n",
      "\n",
      "Seq is: GCTAATGT\n",
      "0 GCTAATGT motifs detected in file\n",
      "\n",
      "Seq is: CTTGTCCG\n",
      "0 CTTGTCCG motifs detected in file\n",
      "\n",
      "Seq is: AGGAGATC\n",
      "0 AGGAGATC motifs detected in file\n",
      "\n",
      "Seq is: TCCCTATA\n",
      "0 TCCCTATA motifs detected in file\n",
      "\n",
      "Seq is: CTTCGTGG\n",
      "0 CTTCGTGG motifs detected in file\n",
      "\n",
      "Seq is: CGGTCCAG\n",
      "0 CGGTCCAG motifs detected in file\n",
      "\n",
      "Seq is: GAAACTCA\n",
      "0 GAAACTCA motifs detected in file\n",
      "\n",
      "Seq is: AAATTCGA\n",
      "0 AAATTCGA motifs detected in file\n",
      "\n",
      "Seq is: CACCGAGC\n",
      "0 CACCGAGC motifs detected in file\n",
      "\n",
      "Complement found from 8049 to 8057\n",
      "Seq is: ATCACAAG\n",
      "1 ATCACAAG motifs detected in file\n",
      "\n",
      "Seq is: ACCGCTCG\n",
      "0 ACCGCTCG motifs detected in file\n",
      "\n",
      "Complement found from 8685 to 8693\n",
      "Seq is: TCAAAACA\n",
      "1 TCAAAACA motifs detected in file\n",
      "\n",
      "Seq is: ACTAAATT\n",
      "0 ACTAAATT motifs detected in file\n",
      "\n",
      "Seq is: GACAACTC\n",
      "0 GACAACTC motifs detected in file\n",
      "\n",
      "Complement found from 16347 to 16355\n",
      "Seq is: CAAATCCC\n",
      "1 CAAATCCC motifs detected in file\n",
      "\n",
      "Seq is: GATTCGTA\n",
      "0 GATTCGTA motifs detected in file\n",
      "\n",
      "Complement found from 633 to 641\n",
      "Complement found from 12964 to 12972\n",
      "Seq is: TCACCCCA\n",
      "2 TCACCCCA motifs detected in file\n",
      "\n",
      "Seq is: TAGATTAG\n",
      "0 TAGATTAG motifs detected in file\n",
      "\n",
      "Seq is: GGTCAAAC\n",
      "0 GGTCAAAC motifs detected in file\n",
      "\n",
      "Seq is: CCAGAATC\n",
      "0 CCAGAATC motifs detected in file\n",
      "\n",
      "Seq is: GATAACAC\n",
      "0 GATAACAC motifs detected in file\n",
      "\n",
      "Seq is: ACAAGTCT\n",
      "0 ACAAGTCT motifs detected in file\n",
      "\n",
      "Seq is: ATACAATT\n",
      "0 ATACAATT motifs detected in file\n",
      "\n",
      "Seq is: TCGGTGAA\n",
      "0 TCGGTGAA motifs detected in file\n",
      "\n",
      "Seq is: AGCATCAG\n",
      "0 AGCATCAG motifs detected in file\n",
      "\n",
      "Seq is: ATAAAACA\n",
      "0 ATAAAACA motifs detected in file\n",
      "\n",
      "Complement found from 1001 to 1009\n",
      "Seq is: CAGTTGAC\n",
      "1 CAGTTGAC motifs detected in file\n",
      "\n",
      "Seq is: CTCAAAAA\n",
      "0 CTCAAAAA motifs detected in file\n",
      "\n",
      "Seq is: ATGTTGAG\n",
      "0 ATGTTGAG motifs detected in file\n",
      "\n",
      "Complement found from 13075 to 13083\n",
      "Seq is: TCCACTCA\n",
      "1 TCCACTCA motifs detected in file\n",
      "\n",
      "Seq is: AAATCGAA\n",
      "0 AAATCGAA motifs detected in file\n",
      "\n",
      "Complement found from 13672 to 13680\n",
      "Seq is: ATAACCCC\n",
      "1 ATAACCCC motifs detected in file\n",
      "\n",
      "Seq is: TCCCCCAC\n",
      "0 TCCCCCAC motifs detected in file\n",
      "\n",
      "Seq is: TAGATTTT\n",
      "0 TAGATTTT motifs detected in file\n",
      "\n",
      "Seq is: GTGAGAAA\n",
      "0 GTGAGAAA motifs detected in file\n",
      "\n",
      "Seq is: TGCGGCCG\n",
      "0 TGCGGCCG motifs detected in file\n",
      "\n",
      "Seq is: AAGATAAC\n",
      "0 AAGATAAC motifs detected in file\n",
      "\n",
      "Seq is: TGAACCCA\n",
      "0 TGAACCCA motifs detected in file\n",
      "\n",
      "Complement found from 15981 to 15989\n",
      "Seq is: ATTAGCAC\n",
      "1 ATTAGCAC motifs detected in file\n",
      "\n",
      "Seq is: ACTGGCGC\n",
      "0 ACTGGCGC motifs detected in file\n",
      "\n",
      "Seq is: CACCGACC\n",
      "0 CACCGACC motifs detected in file\n",
      "\n",
      "Seq is: GTGCTTTA\n",
      "0 GTGCTTTA motifs detected in file\n",
      "\n",
      "Seq is: ACTGGTTG\n",
      "0 ACTGGTTG motifs detected in file\n",
      "\n",
      "Seq is: GGACCCCA\n",
      "0 GGACCCCA motifs detected in file\n",
      "\n",
      "Seq is: ATCATATC\n",
      "0 ATCATATC motifs detected in file\n",
      "\n",
      "Seq is: GAATCAAT\n",
      "0 GAATCAAT motifs detected in file\n",
      "\n",
      "Seq is: TTGAAAGC\n",
      "0 TTGAAAGC motifs detected in file\n",
      "\n",
      "Seq is: AAATAACG\n",
      "0 AAATAACG motifs detected in file\n",
      "\n",
      "Seq is: ATTTCCAA\n",
      "0 ATTTCCAA motifs detected in file\n",
      "\n",
      "Seq is: TTAGTGAC\n",
      "0 TTAGTGAC motifs detected in file\n",
      "\n",
      "Seq is: GACAAAGG\n",
      "0 GACAAAGG motifs detected in file\n",
      "\n",
      "Complement found from 8268 to 8276\n",
      "Seq is: GCACCCCC\n",
      "1 GCACCCCC motifs detected in file\n",
      "\n",
      "Seq is: ACACCGAT\n",
      "0 ACACCGAT motifs detected in file\n",
      "\n",
      "Seq is: CCGATTCG\n",
      "0 CCGATTCG motifs detected in file\n",
      "\n",
      "Seq is: CAAAACTC\n",
      "0 CAAAACTC motifs detected in file\n",
      "\n",
      "Seq is: GACGTAAC\n",
      "0 GACGTAAC motifs detected in file\n",
      "\n",
      "Seq is: GACGCACG\n",
      "0 GACGCACG motifs detected in file\n",
      "\n",
      "Complement found from 1382 to 1390\n",
      "Complement found from 11755 to 11763\n",
      "Seq is: AACTACGA\n",
      "2 AACTACGA motifs detected in file\n",
      "\n",
      "Seq is: ACAAGGAA\n",
      "0 ACAAGGAA motifs detected in file\n",
      "\n",
      "Seq is: AACTAGCA\n",
      "0 AACTAGCA motifs detected in file\n",
      "\n",
      "Complement found from 14390 to 14398\n",
      "Seq is: CCACTAAA\n",
      "1 CCACTAAA motifs detected in file\n",
      "\n",
      "Seq is: TCTCCCAC\n",
      "0 TCTCCCAC motifs detected in file\n",
      "\n",
      "Seq is: TTGAGTGA\n",
      "0 TTGAGTGA motifs detected in file\n",
      "\n",
      "Complement found from 15618 to 15626\n",
      "Seq is: CCTTGCCC\n",
      "1 CCTTGCCC motifs detected in file\n",
      "\n",
      "Complement found from 11757 to 11765\n",
      "Seq is: CTACGAAC\n",
      "1 CTACGAAC motifs detected in file\n",
      "\n",
      "Seq is: GTACACAT\n",
      "0 GTACACAT motifs detected in file\n",
      "\n",
      "Seq is: TAGAATGA\n",
      "0 TAGAATGA motifs detected in file\n",
      "\n",
      "Seq is: TTCTCGAT\n",
      "0 TTCTCGAT motifs detected in file\n",
      "\n",
      "Seq is: TATCTTTC\n",
      "0 TATCTTTC motifs detected in file\n",
      "\n",
      "Seq is: CGATCCTG\n",
      "0 CGATCCTG motifs detected in file\n",
      "\n",
      "Seq is: GTTTGGAT\n",
      "0 GTTTGGAT motifs detected in file\n",
      "\n",
      "Complement found from 642 to 650\n",
      "Complement found from 8688 to 8696\n",
      "Seq is: AAACAAAT\n",
      "2 AAACAAAT motifs detected in file\n",
      "\n",
      "Complement found from 1081 to 1089\n",
      "Complement found from 5599 to 5607\n",
      "Complement found from 12966 to 12974\n",
      "Complement found from 14387 to 14395\n",
      "Seq is: ACCCCACT\n",
      "4 ACCCCACT motifs detected in file\n",
      "\n",
      "Seq is: ACACTCGG\n",
      "0 ACACTCGG motifs detected in file\n",
      "\n",
      "Complement found from 16038 to 16046\n",
      "Seq is: GCAGATTT\n",
      "1 GCAGATTT motifs detected in file\n",
      "\n",
      "Seq is: GTAAAAGT\n",
      "0 GTAAAAGT motifs detected in file\n",
      "\n",
      "Complement found from 14141 to 14149\n",
      "Seq is: CACATAAC\n",
      "1 CACATAAC motifs detected in file\n",
      "\n",
      "Seq is: GAAACTCC\n",
      "0 GAAACTCC motifs detected in file\n",
      "\n",
      "Seq is: TCCATTCG\n",
      "0 TCCATTCG motifs detected in file\n",
      "\n",
      "Complement found from 16086 to 16094\n",
      "Seq is: ATGTATTT\n",
      "1 ATGTATTT motifs detected in file\n",
      "\n",
      "Complement found from 563 to 571\n",
      "Seq is: GACACCCC\n",
      "1 GACACCCC motifs detected in file\n",
      "\n",
      "Complement found from 2057 to 2065\n",
      "Complement found from 4623 to 4631\n",
      "Complement found from 11093 to 11101\n",
      "Seq is: CCACAGAA\n",
      "3 CCACAGAA motifs detected in file\n",
      "\n",
      "Complement found from 351 to 359\n",
      "Complement found from 555 to 563\n",
      "Seq is: ACCCCAAA\n",
      "2 ACCCCAAA motifs detected in file\n",
      "\n",
      "Complement found from 547 to 555\n",
      "Seq is: CCAACCAA\n",
      "1 CCAACCAA motifs detected in file\n",
      "\n",
      "Seq is: GCCCCATA\n",
      "0 GCCCCATA motifs detected in file\n",
      "\n",
      "Seq is: CCCCAATC\n",
      "0 CCCCAATC motifs detected in file\n",
      "\n",
      "Seq is: GTCGCCAC\n",
      "0 GTCGCCAC motifs detected in file\n",
      "\n",
      "Complement found from 514 to 522\n",
      "Seq is: ACACACAC\n",
      "1 ACACACAC motifs detected in file\n",
      "\n",
      "Seq is: GACCCATC\n",
      "0 GACCCATC motifs detected in file\n",
      "\n",
      "Complement found from 8574 to 8582\n",
      "Seq is: CTACCCGC\n",
      "1 CTACCCGC motifs detected in file\n",
      "\n",
      "Complement found from 9555 to 9563\n",
      "Seq is: CCCCAACA\n",
      "1 CCCCAACA motifs detected in file\n",
      "\n",
      "Complement found from 5100 to 5108\n",
      "Complement found from 14207 to 14215\n",
      "Seq is: TAACTACT\n",
      "2 TAACTACT motifs detected in file\n",
      "\n",
      "Complement found from 11101 to 11109\n",
      "Seq is: CTAATCAT\n",
      "1 CTAATCAT motifs detected in file\n",
      "\n",
      "Seq is: CATACCCT\n",
      "0 CATACCCT motifs detected in file\n",
      "\n",
      "Complement found from 1490 to 1498\n",
      "Complement found from 5206 to 5214\n",
      "Complement found from 5323 to 5331\n",
      "Complement found from 10087 to 10095\n",
      "Seq is: CACCCTCC\n",
      "4 CACCCTCC motifs detected in file\n",
      "\n",
      "Complement found from 5489 to 5497\n",
      "Seq is: CCTTTTAT\n",
      "1 CCTTTTAT motifs detected in file\n",
      "\n",
      "Complement found from 11353 to 11361\n",
      "Complement found from 15415 to 15423\n",
      "Complement found from 15577 to 15585\n",
      "Seq is: TACACAAT\n",
      "3 TACACAAT motifs detected in file\n",
      "\n",
      "Complement found from 490 to 498\n",
      "Complement found from 3581 to 3589\n",
      "Complement found from 10943 to 10951\n",
      "Complement found from 11508 to 11516\n",
      "Complement found from 15363 to 15371\n",
      "Seq is: CAACCCCC\n",
      "5 CAACCCCC motifs detected in file\n",
      "\n",
      "Complement found from 10796 to 10804\n",
      "Seq is: CACTGACA\n",
      "1 CACTGACA motifs detected in file\n",
      "\n",
      "Seq is: ATTTTCAC\n",
      "0 ATTTTCAC motifs detected in file\n",
      "\n",
      "Seq is: GTATGGCG\n",
      "0 GTATGGCG motifs detected in file\n",
      "\n",
      "Seq is: GTTTTCTA\n",
      "0 GTTTTCTA motifs detected in file\n",
      "\n",
      "Seq is: TTTTAAAC\n",
      "0 TTTTAAAC motifs detected in file\n",
      "\n",
      "Seq is: TTTAGACC\n",
      "0 TTTAGACC motifs detected in file\n",
      "\n",
      "Seq is: AATCCGAC\n",
      "0 AATCCGAC motifs detected in file\n",
      "\n",
      "Complement found from 8561 to 8569\n",
      "Complement found from 13940 to 13948\n",
      "Seq is: CACAATCC\n",
      "2 CACAATCC motifs detected in file\n",
      "\n",
      "Seq is: CAAGAAAC\n",
      "0 CAAGAAAC motifs detected in file\n",
      "\n",
      "Complement found from 16161 to 16169\n",
      "Seq is: AAAAACCC\n",
      "1 AAAAACCC motifs detected in file\n",
      "\n",
      "Seq is: CAAACCGT\n",
      "0 CAAACCGT motifs detected in file\n",
      "\n",
      "Complement found from 4065 to 4073\n",
      "Complement found from 9086 to 9094\n",
      "Seq is: CTCTACAC\n",
      "2 CTCTACAC motifs detected in file\n",
      "\n",
      "Seq is: AAATTCAC\n",
      "0 AAATTCAC motifs detected in file\n",
      "\n",
      "Seq is: GACACCGG\n",
      "0 GACACCGG motifs detected in file\n",
      "\n",
      "Seq is: TCTTCGCC\n",
      "0 TCTTCGCC motifs detected in file\n",
      "\n",
      "Complement found from 306 to 314\n",
      "Complement found from 957 to 965\n",
      "Complement found from 3568 to 3576\n",
      "Complement found from 15537 to 15545\n",
      "Complement found from 16185 to 16193\n",
      "Seq is: CCCTCCCC\n",
      "5 CCCTCCCC motifs detected in file\n",
      "\n",
      "Complement found from 1682 to 1690\n",
      "Seq is: CCCAAACC\n",
      "1 CCCAAACC motifs detected in file\n",
      "\n",
      "Seq is: ACCTTTAA\n",
      "0 ACCTTTAA motifs detected in file\n",
      "\n",
      "Complement found from 5283 to 5291\n",
      "Complement found from 14025 to 14033\n",
      "Seq is: AAAACAAT\n",
      "2 AAAACAAT motifs detected in file\n",
      "\n",
      "Seq is: ACTACAGA\n",
      "0 ACTACAGA motifs detected in file\n",
      "\n",
      "Seq is: CACACCTT\n",
      "0 CACACCTT motifs detected in file\n",
      "\n",
      "Seq is: TCACCGAC\n",
      "0 TCACCGAC motifs detected in file\n",
      "\n",
      "Seq is: ACGTCTGT\n",
      "0 ACGTCTGT motifs detected in file\n",
      "\n",
      "Seq is: AAGTTAAC\n",
      "0 AAGTTAAC motifs detected in file\n",
      "\n",
      "Complement found from 233 to 241\n",
      "Complement found from 14544 to 14552\n",
      "Seq is: AATAATAA\n",
      "2 AATAATAA motifs detected in file\n",
      "\n",
      "Seq is: TACAGGAT\n",
      "0 TACAGGAT motifs detected in file\n",
      "\n",
      "Seq is: GTTCGTAA\n",
      "0 GTTCGTAA motifs detected in file\n",
      "\n",
      "Complement found from 207 to 215\n",
      "Complement found from 5636 to 5644\n",
      "Complement found from 14782 to 14790\n",
      "Seq is: TTAATTAA\n",
      "3 TTAATTAA motifs detected in file\n",
      "\n",
      "Seq is: TTGTGTGA\n",
      "0 TTGTGTGA motifs detected in file\n",
      "\n",
      "Seq is: AATCATTC\n",
      "0 AATCATTC motifs detected in file\n",
      "\n",
      "Seq is: ATACAAGC\n",
      "0 ATACAAGC motifs detected in file\n",
      "\n",
      "Seq is: GGACATTA\n",
      "0 GGACATTA motifs detected in file\n",
      "\n",
      "Seq is: TAACTTGC\n",
      "0 TAACTTGC motifs detected in file\n",
      "\n",
      "Seq is: ATCCACGC\n",
      "0 ATCCACGC motifs detected in file\n",
      "\n",
      "Complement found from 3613 to 3621\n",
      "Complement found from 9459 to 9467\n",
      "Seq is: TATTTATT\n",
      "2 TATTTATT motifs detected in file\n",
      "\n",
      "Seq is: ATCCTACT\n",
      "0 ATCCTACT motifs detected in file\n",
      "\n",
      "Seq is: CCGTCCTT\n",
      "0 CCGTCCTT motifs detected in file\n",
      "\n",
      "Seq is: AGTTTCTG\n",
      "0 AGTTTCTG motifs detected in file\n",
      "\n",
      "Seq is: TCTATGAC\n",
      "0 TCTATGAC motifs detected in file\n",
      "\n",
      "Seq is: GCTGTATC\n",
      "0 GCTGTATC motifs detected in file\n",
      "\n",
      "Complement found from 2638 to 2646\n",
      "Seq is: CCACGAGG\n",
      "1 CCACGAGG motifs detected in file\n",
      "\n",
      "Seq is: CCGAGGTC\n",
      "0 CCGAGGTC motifs detected in file\n",
      "\n",
      "Seq is: GCAGAGCG\n",
      "0 GCAGAGCG motifs detected in file\n",
      "\n",
      "Seq is: TTACGATA\n",
      "0 TTACGATA motifs detected in file\n",
      "\n",
      "Seq is: GCGCACGT\n",
      "0 GCGCACGT motifs detected in file\n",
      "\n",
      "Seq is: ATGGGGGG\n",
      "0 ATGGGGGG motifs detected in file\n",
      "\n",
      "Seq is: TCTGCTTT\n",
      "0 TCTGCTTT motifs detected in file\n",
      "\n",
      "Seq is: TATGGTTT\n",
      "0 TATGGTTT motifs detected in file\n",
      "\n",
      "Seq is: ACGTACCT\n",
      "0 ACGTACCT motifs detected in file\n",
      "\n",
      "Seq is: CTCGAGGG\n",
      "0 CTCGAGGG motifs detected in file\n",
      "\n",
      "Complement found from 4189 to 4197\n",
      "Complement found from 12011 to 12019\n",
      "Complement found from 14399 to 14407\n",
      "Seq is: CACTCACC\n",
      "3 CACTCACC motifs detected in file\n",
      "\n",
      "Seq is: AATTATCC\n",
      "0 AATTATCC motifs detected in file\n",
      "\n",
      "Complement found from 9850 to 9858\n",
      "Seq is: CACTATCT\n",
      "1 CACTATCT motifs detected in file\n",
      "\n",
      "Complement found from 2119 to 2127\n",
      "Seq is: GGACACTA\n",
      "1 GGACACTA motifs detected in file\n",
      "\n",
      "Seq is: GTAGCACTA\n",
      "0 GTAGCACTA motifs detected in file\n",
      "\n",
      "Seq is: CAGAATAAA\n",
      "0 CAGAATAAA motifs detected in file\n",
      "\n",
      "Seq is: TTCCCCTTG\n",
      "0 TTCCCCTTG motifs detected in file\n",
      "\n",
      "Complement found from 14552 to 14561\n",
      "Seq is: CACACCCGA\n",
      "1 CACACCCGA motifs detected in file\n",
      "\n",
      "Seq is: TAAATCCGA\n",
      "0 TAAATCCGA motifs detected in file\n",
      "\n",
      "Seq is: AATACTGGG\n",
      "0 AATACTGGG motifs detected in file\n",
      "\n",
      "Seq is: ACTTCATCC\n",
      "0 ACTTCATCC motifs detected in file\n",
      "\n",
      "Seq is: TTGGTCTAC\n",
      "0 TTGGTCTAC motifs detected in file\n",
      "\n",
      "Seq is: AGCCTATGT\n",
      "0 AGCCTATGT motifs detected in file\n",
      "\n",
      "Seq is: CAAGTGAAA\n",
      "0 CAAGTGAAA motifs detected in file\n",
      "\n",
      "Seq is: TCGATGGGG\n",
      "0 TCGATGGGG motifs detected in file\n",
      "\n",
      "Seq is: GTTCACAAT\n",
      "0 GTTCACAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCCGGGCC\n",
      "0 ACCCGGGCC motifs detected in file\n",
      "\n",
      "Seq is: TCGCTCCTC\n",
      "0 TCGCTCCTC motifs detected in file\n",
      "\n",
      "Seq is: TCATCGTGA\n",
      "0 TCATCGTGA motifs detected in file\n",
      "\n",
      "Seq is: GAACACGCC\n",
      "0 GAACACGCC motifs detected in file\n",
      "\n",
      "Seq is: CTATAACTA\n",
      "0 CTATAACTA motifs detected in file\n",
      "\n",
      "Seq is: AAGTGCCTC\n",
      "0 AAGTGCCTC motifs detected in file\n",
      "\n",
      "Seq is: CTACCACCA\n",
      "0 CTACCACCA motifs detected in file\n",
      "\n",
      "Seq is: GTTCCCTGG\n",
      "0 GTTCCCTGG motifs detected in file\n",
      "\n",
      "Seq is: GGATAGACT\n",
      "0 GGATAGACT motifs detected in file\n",
      "\n",
      "Seq is: CCCCCCAGT\n",
      "0 CCCCCCAGT motifs detected in file\n",
      "\n",
      "Seq is: AGGTACCCC\n",
      "0 AGGTACCCC motifs detected in file\n",
      "\n",
      "Seq is: TGCTCTTCC\n",
      "0 TGCTCTTCC motifs detected in file\n",
      "\n",
      "Seq is: CTAAACTGA\n",
      "0 CTAAACTGA motifs detected in file\n",
      "\n",
      "Seq is: CATTACACG\n",
      "0 CATTACACG motifs detected in file\n",
      "\n",
      "Seq is: ATACATGCC\n",
      "0 ATACATGCC motifs detected in file\n",
      "\n",
      "Complement found from 12199 to 12208\n",
      "Seq is: ATTTACCGA\n",
      "1 ATTTACCGA motifs detected in file\n",
      "\n",
      "Seq is: AATACATGA\n",
      "0 AATACATGA motifs detected in file\n",
      "\n",
      "Seq is: TACATGACA\n",
      "0 TACATGACA motifs detected in file\n",
      "\n",
      "Seq is: ATTCCCACC\n",
      "0 ATTCCCACC motifs detected in file\n",
      "\n",
      "Complement found from 9888 to 9897\n",
      "Complement found from 11664 to 11673\n",
      "Seq is: CATCCAAAC\n",
      "2 CATCCAAAC motifs detected in file\n",
      "\n",
      "Seq is: AACCATAGG\n",
      "0 AACCATAGG motifs detected in file\n",
      "\n",
      "Seq is: ATCACCCAC\n",
      "0 ATCACCCAC motifs detected in file\n",
      "\n",
      "Complement found from 6843 to 6852\n",
      "Seq is: TCCCCACCG\n",
      "1 TCCCCACCG motifs detected in file\n",
      "\n",
      "Complement found from 1101 to 1110\n",
      "Seq is: AAACCTCAA\n",
      "1 AAACCTCAA motifs detected in file\n",
      "\n",
      "Seq is: CGTCAACTA\n",
      "0 CGTCAACTA motifs detected in file\n",
      "\n",
      "Seq is: CACACTATC\n",
      "0 CACACTATC motifs detected in file\n",
      "\n",
      "Seq is: AACTCCCAA\n",
      "0 AACTCCCAA motifs detected in file\n",
      "\n",
      "Seq is: CTAACGACA\n",
      "0 CTAACGACA motifs detected in file\n",
      "\n",
      "Seq is: TGAACGAAC\n",
      "0 TGAACGAAC motifs detected in file\n",
      "\n",
      "Seq is: ATTCGTACC\n",
      "0 ATTCGTACC motifs detected in file\n",
      "\n",
      "Complement found from 14527 to 14536\n",
      "Seq is: CCTCCCCCA\n",
      "1 CCTCCCCCA motifs detected in file\n",
      "\n",
      "Seq is: AAACTACAC\n",
      "0 AAACTACAC motifs detected in file\n",
      "\n",
      "Seq is: CTAACCCAA\n",
      "0 CTAACCCAA motifs detected in file\n",
      "\n",
      "Seq is: AAATACATG\n",
      "0 AAATACATG motifs detected in file\n",
      "\n",
      "Seq is: ATGTCCACC\n",
      "0 ATGTCCACC motifs detected in file\n",
      "\n",
      "Seq is: AGTTCATAA\n",
      "0 AGTTCATAA motifs detected in file\n",
      "\n",
      "Seq is: ATACCATGG\n",
      "0 ATACCATGG motifs detected in file\n",
      "\n",
      "Seq is: CATGTTATA\n",
      "0 CATGTTATA motifs detected in file\n",
      "\n",
      "Seq is: AGTACCACC\n",
      "0 AGTACCACC motifs detected in file\n",
      "\n",
      "Seq is: GACCGTCAT\n",
      "0 GACCGTCAT motifs detected in file\n",
      "\n",
      "Seq is: TACATGCTT\n",
      "0 TACATGCTT motifs detected in file\n",
      "\n",
      "Seq is: TATGTATCG\n",
      "0 TATGTATCG motifs detected in file\n",
      "\n",
      "Complement found from 11330 to 11339\n",
      "Seq is: CCAACAACT\n",
      "1 CCAACAACT motifs detected in file\n",
      "\n",
      "Seq is: ACCCACTCA\n",
      "0 ACCCACTCA motifs detected in file\n",
      "\n",
      "Seq is: GTTATGAAC\n",
      "0 GTTATGAAC motifs detected in file\n",
      "\n",
      "Seq is: CCACCATGG\n",
      "0 CCACCATGG motifs detected in file\n",
      "\n",
      "Complement found from 616 to 625\n",
      "Seq is: GTTTAGACG\n",
      "1 GTTTAGACG motifs detected in file\n",
      "\n",
      "Seq is: AAGGGGTAC\n",
      "0 AAGGGGTAC motifs detected in file\n",
      "\n",
      "Seq is: TTTCTTGTC\n",
      "0 TTTCTTGTC motifs detected in file\n",
      "\n",
      "Seq is: TCTTATCAA\n",
      "0 TCTTATCAA motifs detected in file\n",
      "\n",
      "Seq is: ATTTAATCT\n",
      "0 ATTTAATCT motifs detected in file\n",
      "\n",
      "Seq is: TAGAATCGA\n",
      "0 TAGAATCGA motifs detected in file\n",
      "\n",
      "Seq is: AACCCACGA\n",
      "0 AACCCACGA motifs detected in file\n",
      "\n",
      "Seq is: TTACCACCT\n",
      "0 TTACCACCT motifs detected in file\n",
      "\n",
      "Seq is: CAATTTCTG\n",
      "0 CAATTTCTG motifs detected in file\n",
      "\n",
      "Seq is: AAAAAGAGA\n",
      "0 AAAAAGAGA motifs detected in file\n",
      "\n",
      "Seq is: CTAAACAGG\n",
      "0 CTAAACAGG motifs detected in file\n",
      "\n",
      "Complement found from 15935 to 15944\n",
      "Seq is: AACCTTTTT\n",
      "1 AACCTTTTT motifs detected in file\n",
      "\n",
      "Seq is: CCAAAAGTA\n",
      "0 CCAAAAGTA motifs detected in file\n",
      "\n",
      "Complement found from 15149 to 15158\n",
      "Seq is: GAGGCCAAA\n",
      "1 GAGGCCAAA motifs detected in file\n",
      "\n",
      "Seq is: TGTTCTGAC\n",
      "0 TGTTCTGAC motifs detected in file\n",
      "\n",
      "Seq is: CACATAATC\n",
      "0 CACATAATC motifs detected in file\n",
      "\n",
      "Seq is: AAATATGAT\n",
      "0 AAATATGAT motifs detected in file\n",
      "\n",
      "Seq is: GTTCCTGTC\n",
      "0 GTTCCTGTC motifs detected in file\n",
      "\n",
      "Seq is: CGGGTAAAC\n",
      "0 CGGGTAAAC motifs detected in file\n",
      "\n",
      "Seq is: TCATAAAAC\n",
      "0 TCATAAAAC motifs detected in file\n",
      "\n",
      "Seq is: AAAAGTTAA\n",
      "0 AAAAGTTAA motifs detected in file\n",
      "\n",
      "Seq is: TCCCTCTAT\n",
      "0 TCCCTCTAT motifs detected in file\n",
      "\n",
      "Seq is: CAACCATAA\n",
      "0 CAACCATAA motifs detected in file\n",
      "\n",
      "Complement found from 15830 to 15839\n",
      "Seq is: TCCTAATCC\n",
      "1 TCCTAATCC motifs detected in file\n",
      "\n",
      "Seq is: TAACAACAC\n",
      "0 TAACAACAC motifs detected in file\n",
      "\n",
      "Seq is: TTCATATCA\n",
      "0 TTCATATCA motifs detected in file\n",
      "\n",
      "Complement found from 11584 to 11593\n",
      "Seq is: TGCCTACGA\n",
      "1 TGCCTACGA motifs detected in file\n",
      "\n",
      "Seq is: TGAACAGGT\n",
      "0 TGAACAGGT motifs detected in file\n",
      "\n",
      "Seq is: TACTACCAT\n",
      "0 TACTACCAT motifs detected in file\n",
      "\n",
      "Seq is: TTTCCCATC\n",
      "0 TTTCCCATC motifs detected in file\n",
      "\n",
      "Seq is: GAATGACCA\n",
      "0 GAATGACCA motifs detected in file\n",
      "\n",
      "Seq is: ACAGGAGGC\n",
      "0 ACAGGAGGC motifs detected in file\n",
      "\n",
      "Seq is: TAAGTCCAA\n",
      "0 TAAGTCCAA motifs detected in file\n",
      "\n",
      "Seq is: TCTTACTCC\n",
      "0 TCTTACTCC motifs detected in file\n",
      "\n",
      "Seq is: TCCAGACGC\n",
      "0 TCCAGACGC motifs detected in file\n",
      "\n",
      "Seq is: CGATCCTCA\n",
      "0 CGATCCTCA motifs detected in file\n",
      "\n",
      "Seq is: GTTATTTCA\n",
      "0 GTTATTTCA motifs detected in file\n",
      "\n",
      "Seq is: CTAACCGAA\n",
      "0 CTAACCGAA motifs detected in file\n",
      "\n",
      "Seq is: TCACCCGCT\n",
      "0 TCACCCGCT motifs detected in file\n",
      "\n",
      "Seq is: TTATAATAC\n",
      "0 TTATAATAC motifs detected in file\n",
      "\n",
      "Seq is: GAAACAACA\n",
      "0 GAAACAACA motifs detected in file\n",
      "\n",
      "Seq is: AACCTATAT\n",
      "0 AACCTATAT motifs detected in file\n",
      "\n",
      "Complement found from 3319 to 3328\n",
      "Seq is: ACCTCCTAC\n",
      "1 ACCTCCTAC motifs detected in file\n",
      "\n",
      "Complement found from 6153 to 6162\n",
      "Complement found from 14771 to 14780\n",
      "Seq is: CCCTAATAA\n",
      "2 CCCTAATAA motifs detected in file\n",
      "\n",
      "Seq is: CGATCCTAC\n",
      "0 CGATCCTAC motifs detected in file\n",
      "\n",
      "Complement found from 5475 to 5484\n",
      "Seq is: TCCTACCTA\n",
      "1 TCCTACCTA motifs detected in file\n",
      "\n",
      "Seq is: TCATTATCC\n",
      "0 TCATTATCC motifs detected in file\n",
      "\n",
      "Seq is: CGTTCCTGC\n",
      "0 CGTTCCTGC motifs detected in file\n",
      "\n",
      "Seq is: GGAGGATCA\n",
      "0 GGAGGATCA motifs detected in file\n",
      "\n",
      "Complement found from 13775 to 13784\n",
      "Seq is: AACAATCCC\n",
      "1 AACAATCCC motifs detected in file\n",
      "\n",
      "Seq is: TGCCTAGCC\n",
      "0 TGCCTAGCC motifs detected in file\n",
      "\n",
      "Seq is: TCTTAACAC\n",
      "0 TCTTAACAC motifs detected in file\n",
      "\n",
      "Seq is: ATCCGCTTA\n",
      "0 ATCCGCTTA motifs detected in file\n",
      "\n",
      "Seq is: TCCTTTATA\n",
      "0 TCCTTTATA motifs detected in file\n",
      "\n",
      "Seq is: GTAAGCCCG\n",
      "0 GTAAGCCCG motifs detected in file\n",
      "\n",
      "Seq is: AACTACACC\n",
      "0 AACTACACC motifs detected in file\n",
      "\n",
      "Complement found from 5443 to 5452\n",
      "Complement found from 14805 to 14814\n",
      "Complement found from 15538 to 15547\n",
      "Seq is: CCTCCCCAC\n",
      "3 CCTCCCCAC motifs detected in file\n",
      "\n",
      "Seq is: AAATTCCCC\n",
      "0 AAATTCCCC motifs detected in file\n",
      "\n",
      "Seq is: AACCGATCC\n",
      "0 AACCGATCC motifs detected in file\n",
      "\n",
      "Seq is: CATATTAAC\n",
      "0 CATATTAAC motifs detected in file\n",
      "\n",
      "Seq is: AGACCCAGC\n",
      "0 AGACCCAGC motifs detected in file\n",
      "\n",
      "Complement found from 14257 to 14266\n",
      "Seq is: GGATCCTCC\n",
      "1 GGATCCTCC motifs detected in file\n",
      "\n",
      "Seq is: AGACCACTC\n",
      "0 AGACCACTC motifs detected in file\n",
      "\n",
      "Complement found from 3799 to 3808\n",
      "Seq is: TTATCACAA\n",
      "1 TTATCACAA motifs detected in file\n",
      "\n",
      "Seq is: TTACAGTAA\n",
      "0 TTACAGTAA motifs detected in file\n",
      "\n",
      "Complement found from 14097 to 14106\n",
      "Seq is: TTCCTCTCT\n",
      "1 TTCCTCTCT motifs detected in file\n",
      "\n",
      "Complement found from 15449 to 15458\n",
      "Seq is: TCCTTCTCT\n",
      "1 TCCTTCTCT motifs detected in file\n",
      "\n",
      "Seq is: TCATTCGGC\n",
      "0 TCATTCGGC motifs detected in file\n",
      "\n",
      "Seq is: TCCCGCAGA\n",
      "0 TCCCGCAGA motifs detected in file\n",
      "\n",
      "Complement found from 438 to 447\n",
      "Seq is: AACTAACAC\n",
      "1 AACTAACAC motifs detected in file\n",
      "\n",
      "Complement found from 14482 to 14491\n",
      "Seq is: ATCATTCCC\n",
      "1 ATCATTCCC motifs detected in file\n",
      "\n",
      "Complement found from 15400 to 15409\n",
      "Seq is: ACCTTCCAC\n",
      "1 ACCTTCCAC motifs detected in file\n",
      "\n",
      "Seq is: TAAAATAGC\n",
      "0 TAAAATAGC motifs detected in file\n",
      "\n",
      "Seq is: CTTACCCTC\n",
      "0 CTTACCCTC motifs detected in file\n",
      "\n",
      "Seq is: CACTAAGGA\n",
      "0 CACTAAGGA motifs detected in file\n",
      "\n",
      "Seq is: TCCCCCAAC\n",
      "0 TCCCCCAAC motifs detected in file\n",
      "\n",
      "Seq is: AAACTAGGG\n",
      "0 AAACTAGGG motifs detected in file\n",
      "\n",
      "Seq is: CAAAGCACG\n",
      "0 CAAAGCACG motifs detected in file\n",
      "\n",
      "Complement found from 6636 to 6645\n",
      "Seq is: TTCTTATCC\n",
      "1 TTCTTATCC motifs detected in file\n",
      "\n",
      "Complement found from 5363 to 5372\n",
      "Seq is: TCCACCTCA\n",
      "1 TCCACCTCA motifs detected in file\n",
      "\n",
      "Complement found from 7854 to 7863\n",
      "Seq is: CAACGATCC\n",
      "1 CAACGATCC motifs detected in file\n",
      "\n",
      "Seq is: CGACGTTAT\n",
      "0 CGACGTTAT motifs detected in file\n",
      "\n",
      "Seq is: TACTTCCCG\n",
      "0 TACTTCCCG motifs detected in file\n",
      "\n",
      "Seq is: TTCTACTTC\n",
      "0 TTCTACTTC motifs detected in file\n",
      "\n",
      "Seq is: ACTTTCCAT\n",
      "0 ACTTTCCAT motifs detected in file\n",
      "\n",
      "Seq is: TTCTTAGCA\n",
      "0 TTCTTAGCA motifs detected in file\n",
      "\n",
      "Seq is: CACTCCCAC\n",
      "0 CACTCCCAC motifs detected in file\n",
      "\n",
      "Seq is: CCTGACAGA\n",
      "0 CCTGACAGA motifs detected in file\n",
      "\n",
      "Seq is: TGACTCATC\n",
      "0 TGACTCATC motifs detected in file\n",
      "\n",
      "Seq is: GGAGGAGTC\n",
      "0 GGAGGAGTC motifs detected in file\n",
      "\n",
      "Seq is: TAAGTAACT\n",
      "0 TAAGTAACT motifs detected in file\n",
      "\n",
      "Seq is: TGATCCAGA\n",
      "0 TGATCCAGA motifs detected in file\n",
      "\n",
      "Seq is: CAGGGTTAC\n",
      "0 CAGGGTTAC motifs detected in file\n",
      "\n",
      "Seq is: ATACCCTAC\n",
      "0 ATACCCTAC motifs detected in file\n",
      "\n",
      "Seq is: CGCCTATCA\n",
      "0 CGCCTATCA motifs detected in file\n",
      "\n",
      "Seq is: TTCAAACAT\n",
      "0 TTCAAACAT motifs detected in file\n",
      "\n",
      "Seq is: TAATGACAC\n",
      "0 TAATGACAC motifs detected in file\n",
      "\n",
      "Seq is: CGGGGAGTC\n",
      "0 CGGGGAGTC motifs detected in file\n",
      "\n",
      "Seq is: TTACTATAA\n",
      "0 TTACTATAA motifs detected in file\n",
      "\n",
      "Seq is: ACCGGAGTG\n",
      "0 ACCGGAGTG motifs detected in file\n",
      "\n",
      "Seq is: CCCTCCTGT\n",
      "0 CCCTCCTGT motifs detected in file\n",
      "\n",
      "Seq is: ATCGGATAC\n",
      "0 ATCGGATAC motifs detected in file\n",
      "\n",
      "Seq is: TTCCGACAA\n",
      "0 TTCCGACAA motifs detected in file\n",
      "\n",
      "Seq is: CGATATCAA\n",
      "0 CGATATCAA motifs detected in file\n",
      "\n",
      "Seq is: CGTTCGTCC\n",
      "0 CGTTCGTCC motifs detected in file\n",
      "\n",
      "Seq is: TCCTATTAC\n",
      "0 TCCTATTAC motifs detected in file\n",
      "\n",
      "Complement found from 1287 to 1296\n",
      "Seq is: GGCTACAAA\n",
      "1 GGCTACAAA motifs detected in file\n",
      "\n",
      "Complement found from 2097 to 2106\n",
      "Seq is: GTCCAAAGA\n",
      "1 GTCCAAAGA motifs detected in file\n",
      "\n",
      "Seq is: CTCATCTCT\n",
      "0 CTCATCTCT motifs detected in file\n",
      "\n",
      "Seq is: TTACTAGGC\n",
      "0 TTACTAGGC motifs detected in file\n",
      "\n",
      "Complement found from 12780 to 12789\n",
      "Seq is: ATTATATCC\n",
      "1 ATTATATCC motifs detected in file\n",
      "\n",
      "Seq is: GGAGCGGGC\n",
      "0 GGAGCGGGC motifs detected in file\n",
      "\n",
      "Seq is: TACACATCC\n",
      "0 TACACATCC motifs detected in file\n",
      "\n",
      "Seq is: TTCTCCGTC\n",
      "0 TTCTCCGTC motifs detected in file\n",
      "\n",
      "Seq is: TATTTCTTA\n",
      "0 TATTTCTTA motifs detected in file\n",
      "\n",
      "Seq is: TAACTCCGC\n",
      "0 TAACTCCGC motifs detected in file\n",
      "\n",
      "Seq is: GGTAACCGC\n",
      "0 GGTAACCGC motifs detected in file\n",
      "\n",
      "Seq is: ACTTCCATC\n",
      "0 ACTTCCATC motifs detected in file\n",
      "\n",
      "Complement found from 9342 to 9351\n",
      "Seq is: GCCTACTAA\n",
      "1 GCCTACTAA motifs detected in file\n",
      "\n",
      "Seq is: GTCGGTATT\n",
      "0 GTCGGTATT motifs detected in file\n",
      "\n",
      "Seq is: AAATGCAGA\n",
      "0 AAATGCAGA motifs detected in file\n",
      "\n",
      "Seq is: GCTCACTAC\n",
      "0 GCTCACTAC motifs detected in file\n",
      "\n",
      "Seq is: ACCCGCTAA\n",
      "0 ACCCGCTAA motifs detected in file\n",
      "\n",
      "Seq is: CTACTTTTC\n",
      "0 CTACTTTTC motifs detected in file\n",
      "\n",
      "Seq is: CGCCAACTC\n",
      "0 CGCCAACTC motifs detected in file\n",
      "\n",
      "Seq is: CGCAGACCA\n",
      "0 CGCAGACCA motifs detected in file\n",
      "\n",
      "Seq is: CTCATCACG\n",
      "0 CTCATCACG motifs detected in file\n",
      "\n",
      "Seq is: TACCGATCC\n",
      "0 TACCGATCC motifs detected in file\n",
      "\n",
      "Seq is: TTATCAGGA\n",
      "0 TTATCAGGA motifs detected in file\n",
      "\n",
      "Seq is: CACCACTAA\n",
      "0 CACCACTAA motifs detected in file\n",
      "\n",
      "Complement found from 15488 to 15497\n",
      "Seq is: ACCTCCTAG\n",
      "1 ACCTCCTAG motifs detected in file\n",
      "\n",
      "Seq is: TCCGTCCGC\n",
      "0 TCCGTCCGC motifs detected in file\n",
      "\n",
      "Seq is: GGTTCCTCA\n",
      "0 GGTTCCTCA motifs detected in file\n",
      "\n",
      "Seq is: CTCGGCTTC\n",
      "0 CTCGGCTTC motifs detected in file\n",
      "\n",
      "Seq is: AAAGTAGTA\n",
      "0 AAAGTAGTA motifs detected in file\n",
      "\n",
      "Seq is: CGCCTCTAC\n",
      "0 CGCCTCTAC motifs detected in file\n",
      "\n",
      "Complement found from 16283 to 16292\n",
      "Seq is: AACCTACCC\n",
      "1 AACCTACCC motifs detected in file\n",
      "\n",
      "Complement found from 15534 to 15543\n",
      "Seq is: CACCCCTCC\n",
      "1 CACCCCTCC motifs detected in file\n",
      "\n",
      "Seq is: AGCTACTTA\n",
      "0 AGCTACTTA motifs detected in file\n",
      "\n",
      "Seq is: CTCACCAAT\n",
      "0 CTCACCAAT motifs detected in file\n",
      "\n",
      "Seq is: TAATTAAAA\n",
      "0 TAATTAAAA motifs detected in file\n",
      "\n",
      "Seq is: TAATCCCCC\n",
      "0 TAATCCCCC motifs detected in file\n",
      "\n",
      "Seq is: AATCAAAAC\n",
      "0 AATCAAAAC motifs detected in file\n",
      "\n",
      "Seq is: GCATAACCC\n",
      "0 GCATAACCC motifs detected in file\n",
      "\n",
      "Seq is: CAGTAACCA\n",
      "0 CAGTAACCA motifs detected in file\n",
      "\n",
      "Seq is: CAAGAACAT\n",
      "0 CAAGAACAT motifs detected in file\n",
      "\n",
      "Seq is: CAACTTTAT\n",
      "0 CAACTTTAT motifs detected in file\n",
      "\n",
      "Seq is: GTTGCTACC\n",
      "0 GTTGCTACC motifs detected in file\n",
      "\n",
      "Seq is: AAAAAGTAT\n",
      "0 AAAAAGTAT motifs detected in file\n",
      "\n",
      "Seq is: AGTAACCAG\n",
      "0 AGTAACCAG motifs detected in file\n",
      "\n",
      "Seq is: CACCAACAT\n",
      "0 CACCAACAT motifs detected in file\n",
      "\n",
      "Seq is: CAGGCACGC\n",
      "0 CAGGCACGC motifs detected in file\n",
      "\n",
      "Seq is: TCTTATTAC\n",
      "0 TCTTATTAC motifs detected in file\n",
      "\n",
      "Seq is: TACATACGA\n",
      "0 TACATACGA motifs detected in file\n",
      "\n",
      "Seq is: AACAAAGAC\n",
      "0 AACAAAGAC motifs detected in file\n",
      "\n",
      "Seq is: AACTCACAC\n",
      "0 AACTCACAC motifs detected in file\n",
      "\n",
      "Seq is: CCAAATCAT\n",
      "0 CCAAATCAT motifs detected in file\n",
      "\n",
      "Seq is: TACCCCAAA\n",
      "0 TACCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: CACCCCAAA\n",
      "0 CACCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: AGAAGATTC\n",
      "0 AGAAGATTC motifs detected in file\n",
      "\n",
      "Seq is: GGAAGAGGA\n",
      "0 GGAAGAGGA motifs detected in file\n",
      "\n",
      "Seq is: TAAATACCC\n",
      "0 TAAATACCC motifs detected in file\n",
      "\n",
      "Seq is: CCAAATCAT\n",
      "0 CCAAATCAT motifs detected in file\n",
      "\n",
      "Seq is: AACTAACAA\n",
      "0 AACTAACAA motifs detected in file\n",
      "\n",
      "Complement found from 6873 to 6882\n",
      "Seq is: TCGCCACAC\n",
      "1 TCGCCACAC motifs detected in file\n",
      "\n",
      "Seq is: CAGCCCACA\n",
      "0 CAGCCCACA motifs detected in file\n",
      "\n",
      "Seq is: CAATAATAA\n",
      "0 CAATAATAA motifs detected in file\n",
      "\n",
      "Seq is: GACTTAAAA\n",
      "0 GACTTAAAA motifs detected in file\n",
      "\n",
      "Seq is: CCCCCTCCA\n",
      "0 CCCCCTCCA motifs detected in file\n",
      "\n",
      "Seq is: ATATACCCA\n",
      "0 ATATACCCA motifs detected in file\n",
      "\n",
      "Complement found from 6403 to 6412\n",
      "Seq is: AATTATCAA\n",
      "1 AATTATCAA motifs detected in file\n",
      "\n",
      "Seq is: AAAAATTAA\n",
      "0 AAAAATTAA motifs detected in file\n",
      "\n",
      "Seq is: ATAAATCCC\n",
      "0 ATAAATCCC motifs detected in file\n",
      "\n",
      "Complement found from 1697 to 1706\n",
      "Complement found from 10098 to 10107\n",
      "Complement found from 15409 to 15418\n",
      "Seq is: CCTTACTAC\n",
      "3 CCTTACTAC motifs detected in file\n",
      "\n",
      "Complement found from 14649 to 14658\n",
      "Seq is: CAACAGAAA\n",
      "1 CAACAGAAA motifs detected in file\n",
      "\n",
      "Seq is: CCTATATGA\n",
      "0 CCTATATGA motifs detected in file\n",
      "\n",
      "Seq is: TGTCGCTAC\n",
      "0 TGTCGCTAC motifs detected in file\n",
      "\n",
      "Seq is: CGATAACTC\n",
      "0 CGATAACTC motifs detected in file\n",
      "\n",
      "Seq is: CTCATAGGA\n",
      "0 CTCATAGGA motifs detected in file\n",
      "\n",
      "Seq is: CTCCGTACC\n",
      "0 CTCCGTACC motifs detected in file\n",
      "\n",
      "Seq is: CCCAGTCCC\n",
      "0 CCCAGTCCC motifs detected in file\n",
      "\n",
      "Seq is: CAACTCCAG\n",
      "0 CAACTCCAG motifs detected in file\n",
      "\n",
      "Complement found from 23 to 32\n",
      "Complement found from 14788 to 14797\n",
      "Seq is: AACCACTCA\n",
      "2 AACCACTCA motifs detected in file\n",
      "\n",
      "Seq is: CAAAATCAC\n",
      "0 CAAAATCAC motifs detected in file\n",
      "\n",
      "Seq is: CCCAATCGC\n",
      "0 CCCAATCGC motifs detected in file\n",
      "\n",
      "Complement found from 14373 to 14382\n",
      "Seq is: TACCTCCAT\n",
      "1 TACCTCCAT motifs detected in file\n",
      "\n",
      "Seq is: CCTAACCAC\n",
      "0 CCTAACCAC motifs detected in file\n",
      "\n",
      "Seq is: GACACCCAC\n",
      "0 GACACCCAC motifs detected in file\n",
      "\n",
      "Seq is: TTTCTCATA\n",
      "0 TTTCTCATA motifs detected in file\n",
      "\n",
      "Seq is: CTACCCCAC\n",
      "0 CTACCCCAC motifs detected in file\n",
      "\n",
      "Seq is: CACCAACAC\n",
      "0 CACCAACAC motifs detected in file\n",
      "\n",
      "Seq is: CATTTGAAA\n",
      "0 CATTTGAAA motifs detected in file\n",
      "\n",
      "Seq is: TTATCACAT\n",
      "0 TTATCACAT motifs detected in file\n",
      "\n",
      "Seq is: CCTTCGACT\n",
      "0 CCTTCGACT motifs detected in file\n",
      "\n",
      "Seq is: TATTAAATA\n",
      "0 TATTAAATA motifs detected in file\n",
      "\n",
      "Complement found from 14096 to 14105\n",
      "Seq is: CTTCCTCTC\n",
      "1 CTTCCTCTC motifs detected in file\n",
      "\n",
      "Seq is: CCCAGTCCC\n",
      "0 CCCAGTCCC motifs detected in file\n",
      "\n",
      "Seq is: AACTAAGCC\n",
      "0 AACTAAGCC motifs detected in file\n",
      "\n",
      "Seq is: CTCCTAGGA\n",
      "0 CTCCTAGGA motifs detected in file\n",
      "\n",
      "Seq is: TAACCACGC\n",
      "0 TAACCACGC motifs detected in file\n",
      "\n",
      "Complement found from 1901 to 1910\n",
      "Seq is: CCCCGAAAC\n",
      "1 CCCCGAAAC motifs detected in file\n",
      "\n",
      "Complement found from 5495 to 5504\n",
      "Seq is: ATACTAATA\n",
      "1 ATACTAATA motifs detected in file\n",
      "\n",
      "Seq is: CCCGCAACT\n",
      "0 CCCGCAACT motifs detected in file\n",
      "\n",
      "Complement found from 10339 to 10348\n",
      "Seq is: AATCATCAT\n",
      "1 AATCATCAT motifs detected in file\n",
      "\n",
      "Seq is: CAATGACCA\n",
      "0 CAATGACCA motifs detected in file\n",
      "\n",
      "Seq is: ACTTGTAAC\n",
      "0 ACTTGTAAC motifs detected in file\n",
      "\n",
      "Seq is: AAACAACCA\n",
      "0 AAACAACCA motifs detected in file\n",
      "\n",
      "Seq is: CATATATAA\n",
      "0 CATATATAA motifs detected in file\n",
      "\n",
      "Seq is: CATTAACTC\n",
      "0 CATTAACTC motifs detected in file\n",
      "\n",
      "Seq is: TAACGAGCC\n",
      "0 TAACGAGCC motifs detected in file\n",
      "\n",
      "Complement found from 8932 to 8941\n",
      "Seq is: CCCTTATCC\n",
      "1 CCCTTATCC motifs detected in file\n",
      "\n",
      "Seq is: AATACACTA\n",
      "0 AATACACTA motifs detected in file\n",
      "\n",
      "Complement found from 15637 to 15646\n",
      "Seq is: ATCCTCATC\n",
      "1 ATCCTCATC motifs detected in file\n",
      "\n",
      "Complement found from 14366 to 14375\n",
      "Seq is: CCAATCCTA\n",
      "1 CCAATCCTA motifs detected in file\n",
      "\n",
      "Seq is: CTCACCCTT\n",
      "0 CTCACCCTT motifs detected in file\n",
      "\n",
      "Seq is: CTTCTTTCT\n",
      "0 CTTCTTTCT motifs detected in file\n",
      "\n",
      "Complement found from 14285 to 14294\n",
      "Seq is: CTCCTTCAT\n",
      "1 CTCCTTCAT motifs detected in file\n",
      "\n",
      "Seq is: TTCAAATTA\n",
      "0 TTCAAATTA motifs detected in file\n",
      "\n",
      "Seq is: ATACGGAAA\n",
      "0 ATACGGAAA motifs detected in file\n",
      "\n",
      "Seq is: AACCCAACT\n",
      "0 AACCCAACT motifs detected in file\n",
      "\n",
      "Complement found from 12969 to 12978\n",
      "Seq is: CCACTACTA\n",
      "1 CCACTACTA motifs detected in file\n",
      "\n",
      "Seq is: CCTCCACCT\n",
      "0 CCTCCACCT motifs detected in file\n",
      "\n",
      "Seq is: CTAAACCAC\n",
      "0 CTAAACCAC motifs detected in file\n",
      "\n",
      "Seq is: GACACTTTA\n",
      "0 GACACTTTA motifs detected in file\n",
      "\n",
      "Seq is: ACAAAATCC\n",
      "0 ACAAAATCC motifs detected in file\n",
      "\n",
      "Complement found from 5264 to 5273\n",
      "Complement found from 8952 to 8961\n",
      "Seq is: ATTATCGAA\n",
      "2 ATTATCGAA motifs detected in file\n",
      "\n",
      "Seq is: AAGATCAGT\n",
      "0 AAGATCAGT motifs detected in file\n",
      "\n",
      "Seq is: CCAATCCAG\n",
      "0 CCAATCCAG motifs detected in file\n",
      "\n",
      "Seq is: ATCCTCCTC\n",
      "0 ATCCTCCTC motifs detected in file\n",
      "\n",
      "Seq is: ATCCCCGTC\n",
      "0 ATCCCCGTC motifs detected in file\n",
      "\n",
      "Seq is: CAAAACCGA\n",
      "0 CAAAACCGA motifs detected in file\n",
      "\n",
      "Seq is: GCATTCTTC\n",
      "0 GCATTCTTC motifs detected in file\n",
      "\n",
      "Seq is: CGGATCTAT\n",
      "0 CGGATCTAT motifs detected in file\n",
      "\n",
      "Complement found from 9254 to 9263\n",
      "Complement found from 10936 to 10945\n",
      "Seq is: CCCCTAACA\n",
      "2 CCCCTAACA motifs detected in file\n",
      "\n",
      "Seq is: CGCCACACA\n",
      "0 CGCCACACA motifs detected in file\n",
      "\n",
      "Seq is: CTACGATCC\n",
      "0 CTACGATCC motifs detected in file\n",
      "\n",
      "Seq is: CATCTTAGG\n",
      "0 CATCTTAGG motifs detected in file\n",
      "\n",
      "Seq is: CTCATACAA\n",
      "0 CTCATACAA motifs detected in file\n",
      "\n",
      "Seq is: CCTCTTTAT\n",
      "0 CCTCTTTAT motifs detected in file\n",
      "\n",
      "Seq is: TTTACACGT\n",
      "0 TTTACACGT motifs detected in file\n",
      "\n",
      "Seq is: ATCACCCCT\n",
      "0 ATCACCCCT motifs detected in file\n",
      "\n",
      "Complement found from 5402 to 5411\n",
      "Complement found from 8491 to 8500\n",
      "Complement found from 13869 to 13878\n",
      "Seq is: AAAATAAAA\n",
      "3 AAAATAAAA motifs detected in file\n",
      "\n",
      "Seq is: TTCAAACAA\n",
      "0 TTCAAACAA motifs detected in file\n",
      "\n",
      "Seq is: CCAATCCAT\n",
      "0 CCAATCCAT motifs detected in file\n",
      "\n",
      "Seq is: CAACTCCAG\n",
      "0 CAACTCCAG motifs detected in file\n",
      "\n",
      "Seq is: ATCCCGACA\n",
      "0 ATCCCGACA motifs detected in file\n",
      "\n",
      "Seq is: ATCTTCAGG\n",
      "0 ATCTTCAGG motifs detected in file\n",
      "\n",
      "Seq is: ATCCTTTCA\n",
      "0 ATCCTTTCA motifs detected in file\n",
      "\n",
      "Seq is: CTGTCGCTC\n",
      "0 CTGTCGCTC motifs detected in file\n",
      "\n",
      "Seq is: CCGACACTC\n",
      "0 CCGACACTC motifs detected in file\n",
      "\n",
      "Complement found from 7133 to 7142\n",
      "Complement found from 12443 to 12452\n",
      "Seq is: AAAATCCAT\n",
      "2 AAAATCCAT motifs detected in file\n",
      "\n",
      "Seq is: CTCCCCCTA\n",
      "0 CTCCCCCTA motifs detected in file\n",
      "\n",
      "Seq is: ACAACAAAC\n",
      "0 ACAACAAAC motifs detected in file\n",
      "\n",
      "Seq is: CTTCCCCCT\n",
      "0 CTTCCCCCT motifs detected in file\n",
      "\n",
      "Seq is: ACGCCCCCT\n",
      "0 ACGCCCCCT motifs detected in file\n",
      "\n",
      "Seq is: TTACAACAA\n",
      "0 TTACAACAA motifs detected in file\n",
      "\n",
      "Seq is: TCATTACTC\n",
      "0 TCATTACTC motifs detected in file\n",
      "\n",
      "Seq is: TTTAGGACG\n",
      "0 TTTAGGACG motifs detected in file\n",
      "\n",
      "Seq is: CTTATCCGA\n",
      "0 CTTATCCGA motifs detected in file\n",
      "\n",
      "Seq is: AGGCCGACG\n",
      "0 AGGCCGACG motifs detected in file\n",
      "\n",
      "Seq is: GTCCGCAAA\n",
      "0 GTCCGCAAA motifs detected in file\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Seq is: TTACCCCAA\n",
      "0 TTACCCCAA motifs detected in file\n",
      "\n",
      "Seq is: ATCATCCCA\n",
      "0 ATCATCCCA motifs detected in file\n",
      "\n",
      "Complement found from 961 to 970\n",
      "Seq is: CCCCAATAA\n",
      "1 CCCCAATAA motifs detected in file\n",
      "\n",
      "Seq is: AAGCAATTA\n",
      "0 AAGCAATTA motifs detected in file\n",
      "\n",
      "Seq is: CAATCATTC\n",
      "0 CAATCATTC motifs detected in file\n",
      "\n",
      "Complement found from 10158 to 10167\n",
      "Seq is: CCACCCCTT\n",
      "1 CCACCCCTT motifs detected in file\n",
      "\n",
      "Seq is: CGCTCCAAC\n",
      "0 CGCTCCAAC motifs detected in file\n",
      "\n",
      "Seq is: TGGACAATC\n",
      "0 TGGACAATC motifs detected in file\n",
      "\n",
      "Seq is: CCACTCTTC\n",
      "0 CCACTCTTC motifs detected in file\n",
      "\n",
      "Seq is: TTAATAAGC\n",
      "0 TTAATAAGC motifs detected in file\n",
      "\n",
      "Seq is: TCACGATAT\n",
      "0 TCACGATAT motifs detected in file\n",
      "\n",
      "Seq is: CCGCGAACA\n",
      "0 CCGCGAACA motifs detected in file\n",
      "\n",
      "Seq is: GTCCCTCCA\n",
      "0 GTCCCTCCA motifs detected in file\n",
      "\n",
      "Seq is: TCGCTACTC\n",
      "0 TCGCTACTC motifs detected in file\n",
      "\n",
      "Seq is: TCATTATCT\n",
      "0 TCATTATCT motifs detected in file\n",
      "\n",
      "Seq is: ATCCCGAGT\n",
      "0 ATCCCGAGT motifs detected in file\n",
      "\n",
      "Complement found from 13528 to 13537\n",
      "Seq is: CCGCAAACA\n",
      "1 CCGCAAACA motifs detected in file\n",
      "\n",
      "Seq is: CATACTATA\n",
      "0 CATACTATA motifs detected in file\n",
      "\n",
      "Seq is: CAAACGCCA\n",
      "0 CAAACGCCA motifs detected in file\n",
      "\n",
      "Seq is: AAGCTACTA\n",
      "0 AAGCTACTA motifs detected in file\n",
      "\n",
      "Seq is: CACCAGAAA\n",
      "0 CACCAGAAA motifs detected in file\n",
      "\n",
      "Seq is: CCTCATCTT\n",
      "0 CCTCATCTT motifs detected in file\n",
      "\n",
      "Seq is: TGGACACTC\n",
      "0 TGGACACTC motifs detected in file\n",
      "\n",
      "Seq is: CTTTCCATA\n",
      "0 CTTTCCATA motifs detected in file\n",
      "\n",
      "Seq is: AGGACGATT\n",
      "0 AGGACGATT motifs detected in file\n",
      "\n",
      "Seq is: ACGATCCGA\n",
      "0 ACGATCCGA motifs detected in file\n",
      "\n",
      "Seq is: CGGTTACCA\n",
      "0 CGGTTACCA motifs detected in file\n",
      "\n",
      "Seq is: CTCCCTCCA\n",
      "0 CTCCCTCCA motifs detected in file\n",
      "\n",
      "Seq is: ACTTCACTC\n",
      "0 ACTTCACTC motifs detected in file\n",
      "\n",
      "Seq is: TCCATACCA\n",
      "0 TCCATACCA motifs detected in file\n",
      "\n",
      "Complement found from 6979 to 6988\n",
      "Seq is: AAACTCATC\n",
      "1 AAACTCATC motifs detected in file\n",
      "\n",
      "Seq is: AGGAGGATA\n",
      "0 AGGAGGATA motifs detected in file\n",
      "\n",
      "Seq is: AAAAGCTTA\n",
      "0 AAAAGCTTA motifs detected in file\n",
      "\n",
      "Seq is: TAGAACAAG\n",
      "0 TAGAACAAG motifs detected in file\n",
      "\n",
      "Seq is: TAACAATTC\n",
      "0 TAACAATTC motifs detected in file\n",
      "\n",
      "Complement found from 6596 to 6605\n",
      "Seq is: CAACACCTA\n",
      "1 CAACACCTA motifs detected in file\n",
      "\n",
      "Seq is: CTACCTGGG\n",
      "0 CTACCTGGG motifs detected in file\n",
      "\n",
      "Seq is: CCTCGTGTA\n",
      "0 CCTCGTGTA motifs detected in file\n",
      "\n",
      "Seq is: TTTATCATA\n",
      "0 TTTATCATA motifs detected in file\n",
      "\n",
      "Seq is: CCGAAACTT\n",
      "0 CCGAAACTT motifs detected in file\n",
      "\n",
      "Seq is: CTTCCGCAC\n",
      "0 CTTCCGCAC motifs detected in file\n",
      "\n",
      "Complement found from 12239 to 12248\n",
      "Seq is: CCATGTCTA\n",
      "1 CCATGTCTA motifs detected in file\n",
      "\n",
      "Seq is: CACGTCCTT\n",
      "0 CACGTCCTT motifs detected in file\n",
      "\n",
      "Seq is: ACGATCCAC\n",
      "0 ACGATCCAC motifs detected in file\n",
      "\n",
      "Complement found from 546 to 555\n",
      "Seq is: ACCAACCAA\n",
      "1 ACCAACCAA motifs detected in file\n",
      "\n",
      "Seq is: CTACGGCTA\n",
      "0 CTACGGCTA motifs detected in file\n",
      "\n",
      "Seq is: ACATTGATA\n",
      "0 ACATTGATA motifs detected in file\n",
      "\n",
      "Seq is: ATACTCAGG\n",
      "0 ATACTCAGG motifs detected in file\n",
      "\n",
      "Complement found from 5611 to 5620\n",
      "Seq is: ATCAACTGA\n",
      "1 ATCAACTGA motifs detected in file\n",
      "\n",
      "Seq is: ACTTCACCT\n",
      "0 ACTTCACCT motifs detected in file\n",
      "\n",
      "Seq is: CTTCCGATG\n",
      "0 CTTCCGATG motifs detected in file\n",
      "\n",
      "Seq is: CTAAAAAAA\n",
      "0 CTAAAAAAA motifs detected in file\n",
      "\n",
      "Seq is: CTACAGTAA\n",
      "0 CTACAGTAA motifs detected in file\n",
      "\n",
      "Seq is: AACACATTC\n",
      "0 AACACATTC motifs detected in file\n",
      "\n",
      "Seq is: CCGCGTCTG\n",
      "0 CCGCGTCTG motifs detected in file\n",
      "\n",
      "Seq is: ACGACGCTT\n",
      "0 ACGACGCTT motifs detected in file\n",
      "\n",
      "Seq is: GTCTCACCA\n",
      "0 GTCTCACCA motifs detected in file\n",
      "\n",
      "Seq is: CTATCGCGG\n",
      "0 CTATCGCGG motifs detected in file\n",
      "\n",
      "Seq is: ATTCGTATC\n",
      "0 ATTCGTATC motifs detected in file\n",
      "\n",
      "Complement found from 3720 to 3729\n",
      "Seq is: ACAATCTCA\n",
      "1 ACAATCTCA motifs detected in file\n",
      "\n",
      "Seq is: AACCTAATC\n",
      "0 AACCTAATC motifs detected in file\n",
      "\n",
      "Seq is: ACCCGATAA\n",
      "0 ACCCGATAA motifs detected in file\n",
      "\n",
      "Seq is: AAGACGATC\n",
      "0 AAGACGATC motifs detected in file\n",
      "\n",
      "Complement found from 11139 to 11148\n",
      "Seq is: CCCCACCTT\n",
      "1 CCCCACCTT motifs detected in file\n",
      "\n",
      "Seq is: CGCCTACTC\n",
      "0 CGCCTACTC motifs detected in file\n",
      "\n",
      "Seq is: ATTCTTCTA\n",
      "0 ATTCTTCTA motifs detected in file\n",
      "\n",
      "Seq is: AGGACGATG\n",
      "0 AGGACGATG motifs detected in file\n",
      "\n",
      "Seq is: TTGATATCA\n",
      "0 TTGATATCA motifs detected in file\n",
      "\n",
      "Seq is: CGAACTCAC\n",
      "0 CGAACTCAC motifs detected in file\n",
      "\n",
      "Seq is: CTCATCCCG\n",
      "0 CTCATCCCG motifs detected in file\n",
      "\n",
      "Seq is: ACTCTGACC\n",
      "0 ACTCTGACC motifs detected in file\n",
      "\n",
      "Seq is: CCACCCCGG\n",
      "0 CCACCCCGG motifs detected in file\n",
      "\n",
      "Seq is: AAGATACCG\n",
      "0 AAGATACCG motifs detected in file\n",
      "\n",
      "Complement found from 13031 to 13040\n",
      "Seq is: ACTCCCCTC\n",
      "1 ACTCCCCTC motifs detected in file\n",
      "\n",
      "Seq is: AGTCCCCAC\n",
      "0 AGTCCCCAC motifs detected in file\n",
      "\n",
      "Seq is: CTCTGGATT\n",
      "0 CTCTGGATT motifs detected in file\n",
      "\n",
      "Seq is: AACCCGACT\n",
      "0 AACCCGACT motifs detected in file\n",
      "\n",
      "Seq is: AAACGGACG\n",
      "0 AAACGGACG motifs detected in file\n",
      "\n",
      "Seq is: ACGACGATC\n",
      "0 ACGACGATC motifs detected in file\n",
      "\n",
      "Seq is: CTCCTCCGG\n",
      "0 CTCCTCCGG motifs detected in file\n",
      "\n",
      "Complement found from 11152 to 11161\n",
      "Complement found from 14058 to 14067\n",
      "Seq is: ATCATCACC\n",
      "2 ATCATCACC motifs detected in file\n",
      "\n",
      "Seq is: CCACTCCGA\n",
      "0 CCACTCCGA motifs detected in file\n",
      "\n",
      "Seq is: ACCTAATCG\n",
      "0 ACCTAATCG motifs detected in file\n",
      "\n",
      "Seq is: CAAATCTTC\n",
      "0 CAAATCTTC motifs detected in file\n",
      "\n",
      "Seq is: CCGATAAAC\n",
      "0 CCGATAAAC motifs detected in file\n",
      "\n",
      "Seq is: AACACCCAG\n",
      "0 AACACCCAG motifs detected in file\n",
      "\n",
      "Seq is: AGTACTCAA\n",
      "0 AGTACTCAA motifs detected in file\n",
      "\n",
      "Seq is: CCTCACATC\n",
      "0 CCTCACATC motifs detected in file\n",
      "\n",
      "Seq is: CTATTTAGT\n",
      "0 CTATTTAGT motifs detected in file\n",
      "\n",
      "Seq is: ACGATTCCG\n",
      "0 ACGATTCCG motifs detected in file\n",
      "\n",
      "Complement found from 11935 to 11944\n",
      "Seq is: CTCCTACTT\n",
      "1 CTCCTACTT motifs detected in file\n",
      "\n",
      "Seq is: TGGCTATAG\n",
      "0 TGGCTATAG motifs detected in file\n",
      "\n",
      "Seq is: CGGCTATGC\n",
      "0 CGGCTATGC motifs detected in file\n",
      "\n",
      "Seq is: CAACATATC\n",
      "0 CAACATATC motifs detected in file\n",
      "\n",
      "Seq is: CTAACGAAC\n",
      "0 CTAACGAAC motifs detected in file\n",
      "\n",
      "Seq is: TTACCGACG\n",
      "0 TTACCGACG motifs detected in file\n",
      "\n",
      "Seq is: ACACAACCG\n",
      "0 ACACAACCG motifs detected in file\n",
      "\n",
      "Seq is: TAGACGAGC\n",
      "0 TAGACGAGC motifs detected in file\n",
      "\n",
      "Seq is: CCGCATAGT\n",
      "0 CCGCATAGT motifs detected in file\n",
      "\n",
      "Seq is: AGTTGACTA\n",
      "0 AGTTGACTA motifs detected in file\n",
      "\n",
      "Seq is: CTCGTTCTT\n",
      "0 CTCGTTCTT motifs detected in file\n",
      "\n",
      "Complement found from 15048 to 15057\n",
      "Seq is: CCTATATTA\n",
      "1 CCTATATTA motifs detected in file\n",
      "\n",
      "Seq is: AGGATGCGG\n",
      "0 AGGATGCGG motifs detected in file\n",
      "\n",
      "Seq is: GAGAGTCGG\n",
      "0 GAGAGTCGG motifs detected in file\n",
      "\n",
      "Seq is: CTACTTGTC\n",
      "0 CTACTTGTC motifs detected in file\n",
      "\n",
      "Seq is: AACCTTATC\n",
      "0 AACCTTATC motifs detected in file\n",
      "\n",
      "Seq is: CAACAATCG\n",
      "0 CAACAATCG motifs detected in file\n",
      "\n",
      "Seq is: CCATTGATT\n",
      "0 CCATTGATT motifs detected in file\n",
      "\n",
      "Complement found from 11101 to 11110\n",
      "Seq is: CTAATCATA\n",
      "1 CTAATCATA motifs detected in file\n",
      "\n",
      "Seq is: CCATTAATC\n",
      "0 CCATTAATC motifs detected in file\n",
      "\n",
      "Seq is: CTTCTACTC\n",
      "0 CTTCTACTC motifs detected in file\n",
      "\n",
      "Complement found from 8819 to 8828\n",
      "Seq is: ATCTATAAA\n",
      "1 ATCTATAAA motifs detected in file\n",
      "\n",
      "Seq is: CTTCTTGAC\n",
      "0 CTTCTTGAC motifs detected in file\n",
      "\n",
      "Complement found from 15179 to 15188\n",
      "Seq is: TAATTACAA\n",
      "1 TAATTACAA motifs detected in file\n",
      "\n",
      "Seq is: ACCCAGACT\n",
      "0 ACCCAGACT motifs detected in file\n",
      "\n",
      "Seq is: CAAATATAT\n",
      "0 CAAATATAT motifs detected in file\n",
      "\n",
      "Seq is: AGTGTCACT\n",
      "0 AGTGTCACT motifs detected in file\n",
      "\n",
      "Seq is: CTTAAGATA\n",
      "0 CTTAAGATA motifs detected in file\n",
      "\n",
      "Seq is: CTACCTGGT\n",
      "0 CTACCTGGT motifs detected in file\n",
      "\n",
      "Seq is: ACATTGCTT\n",
      "0 ACATTGCTT motifs detected in file\n",
      "\n",
      "Seq is: GTTACGATG\n",
      "0 GTTACGATG motifs detected in file\n",
      "\n",
      "Seq is: TCCCTACTT\n",
      "0 TCCCTACTT motifs detected in file\n",
      "\n",
      "Seq is: ATAATACCT\n",
      "0 ATAATACCT motifs detected in file\n",
      "\n",
      "Seq is: CTTCATCAG\n",
      "0 CTTCATCAG motifs detected in file\n",
      "\n",
      "Seq is: ATCAAACTT\n",
      "0 ATCAAACTT motifs detected in file\n",
      "\n",
      "Seq is: CGAATCCCT\n",
      "0 CGAATCCCT motifs detected in file\n",
      "\n",
      "Seq is: CTCGACCCA\n",
      "0 CTCGACCCA motifs detected in file\n",
      "\n",
      "Complement found from 2764 to 2773\n",
      "Seq is: ACAAACCCA\n",
      "1 ACAAACCCA motifs detected in file\n",
      "\n",
      "Seq is: ACACCGAGT\n",
      "0 ACACCGAGT motifs detected in file\n",
      "\n",
      "Seq is: CACAGTCAA\n",
      "0 CACAGTCAA motifs detected in file\n",
      "\n",
      "Seq is: GCTCTATTA\n",
      "0 GCTCTATTA motifs detected in file\n",
      "\n",
      "Seq is: TTGAAGAAC\n",
      "0 TTGAAGAAC motifs detected in file\n",
      "\n",
      "Seq is: CAGATCCGT\n",
      "0 CAGATCCGT motifs detected in file\n",
      "\n",
      "Seq is: GTACTTATA\n",
      "0 GTACTTATA motifs detected in file\n",
      "\n",
      "Complement found from 11261 to 11270\n",
      "Seq is: ACAACACCC\n",
      "1 ACAACACCC motifs detected in file\n",
      "\n",
      "Seq is: CTTCTCTGA\n",
      "0 CTTCTCTGA motifs detected in file\n",
      "\n",
      "Complement found from 150 to 159\n",
      "Seq is: CTATTATTT\n",
      "1 CTATTATTT motifs detected in file\n",
      "\n",
      "Seq is: CCACCTACG\n",
      "0 CCACCTACG motifs detected in file\n",
      "\n",
      "Seq is: CTGTTACCT\n",
      "0 CTGTTACCT motifs detected in file\n",
      "\n",
      "Seq is: AAAATGTAT\n",
      "0 AAAATGTAT motifs detected in file\n",
      "\n",
      "Complement found from 8403 to 8412\n",
      "Complement found from 12429 to 12438\n",
      "Seq is: TACCCCCAT\n",
      "2 TACCCCCAT motifs detected in file\n",
      "\n",
      "Seq is: ACTCAAAAA\n",
      "0 ACTCAAAAA motifs detected in file\n",
      "\n",
      "Seq is: AAACAATCC\n",
      "0 AAACAATCC motifs detected in file\n",
      "\n",
      "Seq is: CAATTGCTC\n",
      "0 CAATTGCTC motifs detected in file\n",
      "\n",
      "Seq is: CCACCATTC\n",
      "0 CCACCATTC motifs detected in file\n",
      "\n",
      "Complement found from 9528 to 9537\n",
      "Seq is: CTACCCCCC\n",
      "1 CTACCCCCC motifs detected in file\n",
      "\n",
      "Seq is: TTAATCCCT\n",
      "0 TTAATCCCT motifs detected in file\n",
      "\n",
      "Seq is: TCAGTCCCA\n",
      "0 TCAGTCCCA motifs detected in file\n",
      "\n",
      "Seq is: ATCCCACCA\n",
      "0 ATCCCACCA motifs detected in file\n",
      "\n",
      "Seq is: ATATCATCA\n",
      "0 ATATCATCA motifs detected in file\n",
      "\n",
      "Seq is: CACGTACCA\n",
      "0 CACGTACCA motifs detected in file\n",
      "\n",
      "Seq is: ATAATGAAA\n",
      "0 ATAATGAAA motifs detected in file\n",
      "\n",
      "Seq is: ATAAACCTC\n",
      "0 ATAAACCTC motifs detected in file\n",
      "\n",
      "Seq is: AACGTGGTT\n",
      "0 AACGTGGTT motifs detected in file\n",
      "\n",
      "Seq is: TTAAAAACC\n",
      "0 TTAAAAACC motifs detected in file\n",
      "\n",
      "Seq is: CCGGATTCT\n",
      "0 CCGGATTCT motifs detected in file\n",
      "\n",
      "Seq is: GGTTACCTA\n",
      "0 GGTTACCTA motifs detected in file\n",
      "\n",
      "Seq is: TCGACAATA\n",
      "0 TCGACAATA motifs detected in file\n",
      "\n",
      "Seq is: GGAAATTTT\n",
      "0 GGAAATTTT motifs detected in file\n",
      "\n",
      "Seq is: CAACTCTTT\n",
      "0 CAACTCTTT motifs detected in file\n",
      "\n",
      "Seq is: CGGTACAAC\n",
      "0 CGGTACAAC motifs detected in file\n",
      "\n",
      "Seq is: AATCTGTAC\n",
      "0 AATCTGTAC motifs detected in file\n",
      "\n",
      "Seq is: CCCCGTACT\n",
      "0 CCCCGTACT motifs detected in file\n",
      "\n",
      "Seq is: CAATCGTCA\n",
      "0 CAATCGTCA motifs detected in file\n",
      "\n",
      "Seq is: AGAACACTC\n",
      "0 AGAACACTC motifs detected in file\n",
      "\n",
      "Seq is: GAAAGAGCC\n",
      "0 GAAAGAGCC motifs detected in file\n",
      "\n",
      "Seq is: ATTTATTCC\n",
      "0 ATTTATTCC motifs detected in file\n",
      "\n",
      "Complement found from 4240 to 4249\n",
      "Seq is: CCAGCATTC\n",
      "1 CCAGCATTC motifs detected in file\n",
      "\n",
      "Seq is: GGAGACAAC\n",
      "0 GGAGACAAC motifs detected in file\n",
      "\n",
      "Seq is: AGTCTAAGT\n",
      "0 AGTCTAAGT motifs detected in file\n",
      "\n",
      "Seq is: GTTAGACTA\n",
      "0 GTTAGACTA motifs detected in file\n",
      "\n",
      "Seq is: CAAAACCAA\n",
      "0 CAAAACCAA motifs detected in file\n",
      "\n",
      "Seq is: TTTGATATA\n",
      "0 TTTGATATA motifs detected in file\n",
      "\n",
      "Seq is: AATGTTCTC\n",
      "0 AATGTTCTC motifs detected in file\n",
      "\n",
      "Seq is: CTTTTGGGC\n",
      "0 CTTTTGGGC motifs detected in file\n",
      "\n",
      "Seq is: CATTACTAC\n",
      "0 CATTACTAC motifs detected in file\n",
      "\n",
      "Seq is: AGCCCCAAC\n",
      "0 AGCCCCAAC motifs detected in file\n",
      "\n",
      "Seq is: TCCCTATCC\n",
      "0 TCCCTATCC motifs detected in file\n",
      "\n",
      "Seq is: TCCTCTTAC\n",
      "0 TCCTCTTAC motifs detected in file\n",
      "\n",
      "Seq is: CCCCTATCC\n",
      "0 CCCCTATCC motifs detected in file\n",
      "\n",
      "Seq is: ACATACTTG\n",
      "0 ACATACTTG motifs detected in file\n",
      "\n",
      "Complement found from 6308 to 6317\n",
      "Complement found from 11810 to 11819\n",
      "Seq is: TACTCCCAC\n",
      "2 TACTCCCAC motifs detected in file\n",
      "\n",
      "Complement found from 1934 to 1943\n",
      "Seq is: AAAAGAGCA\n",
      "1 AAAAGAGCA motifs detected in file\n",
      "\n",
      "Seq is: CACTTACTC\n",
      "0 CACTTACTC motifs detected in file\n",
      "\n",
      "Seq is: CCAAAATAC\n",
      "0 CCAAAATAC motifs detected in file\n",
      "\n",
      "Seq is: AACAATTAC\n",
      "0 AACAATTAC motifs detected in file\n",
      "\n",
      "Complement found from 10840 to 10849\n",
      "Seq is: ACCACCCAC\n",
      "1 ACCACCCAC motifs detected in file\n",
      "\n",
      "Seq is: TCACTCGGG\n",
      "0 TCACTCGGG motifs detected in file\n",
      "\n",
      "Seq is: GTAACACAA\n",
      "0 GTAACACAA motifs detected in file\n",
      "\n",
      "Seq is: CACCATTTA\n",
      "0 CACCATTTA motifs detected in file\n",
      "\n",
      "Seq is: TACATCTCC\n",
      "0 TACATCTCC motifs detected in file\n",
      "\n",
      "Complement found from 10544 to 10553\n",
      "Seq is: CTCATATCC\n",
      "1 CTCATATCC motifs detected in file\n",
      "\n",
      "Seq is: CGACACTGA\n",
      "0 CGACACTGA motifs detected in file\n",
      "\n",
      "Seq is: TCATACAAC\n",
      "0 TCATACAAC motifs detected in file\n",
      "\n",
      "Complement found from 2997 to 3006\n",
      "Seq is: TCAGGACAT\n",
      "1 TCAGGACAT motifs detected in file\n",
      "\n",
      "Seq is: TCATCCTCT\n",
      "0 TCATCCTCT motifs detected in file\n",
      "\n",
      "Seq is: CACTATAAA\n",
      "0 CACTATAAA motifs detected in file\n",
      "\n",
      "Complement found from 7801 to 7810\n",
      "Seq is: CTAGTCCTC\n",
      "1 CTAGTCCTC motifs detected in file\n",
      "\n",
      "Seq is: TTGCACCAA\n",
      "0 TTGCACCAA motifs detected in file\n",
      "\n",
      "Seq is: TGATCGTGT\n",
      "0 TGATCGTGT motifs detected in file\n",
      "\n",
      "Seq is: CTCTCAAGA\n",
      "0 CTCTCAAGA motifs detected in file\n",
      "\n",
      "Seq is: GGGTCATCC\n",
      "0 GGGTCATCC motifs detected in file\n",
      "\n",
      "Seq is: AATTATCAC\n",
      "0 AATTATCAC motifs detected in file\n",
      "\n",
      "Complement found from 8027 to 8036\n",
      "Complement found from 12083 to 12092\n",
      "Seq is: CCCCCATTC\n",
      "2 CCCCCATTC motifs detected in file\n",
      "\n",
      "Seq is: CGCTCCAAT\n",
      "0 CGCTCCAAT motifs detected in file\n",
      "\n",
      "Seq is: CGCTCCGAA\n",
      "0 CGCTCCGAA motifs detected in file\n",
      "\n",
      "Seq is: CGATCTTCA\n",
      "0 CGATCTTCA motifs detected in file\n",
      "\n",
      "Seq is: GTAGTTTTT\n",
      "0 GTAGTTTTT motifs detected in file\n",
      "\n",
      "Seq is: CGATAATCA\n",
      "0 CGATAATCA motifs detected in file\n",
      "\n",
      "Seq is: CCCTCATCT\n",
      "0 CCCTCATCT motifs detected in file\n",
      "\n",
      "Seq is: CAAACTTCA\n",
      "0 CAAACTTCA motifs detected in file\n",
      "\n",
      "Seq is: GGAACTCTC\n",
      "0 GGAACTCTC motifs detected in file\n",
      "\n",
      "Complement found from 10952 to 10961\n",
      "Complement found from 15836 to 15845\n",
      "Seq is: TCCTAATAC\n",
      "2 TCCTAATAC motifs detected in file\n",
      "\n",
      "Seq is: TACGCTGAC\n",
      "0 TACGCTGAC motifs detected in file\n",
      "\n",
      "Seq is: ACTCACGCA\n",
      "0 ACTCACGCA motifs detected in file\n",
      "\n",
      "Complement found from 3668 to 3677\n",
      "Seq is: AGCATCAAA\n",
      "1 AGCATCAAA motifs detected in file\n",
      "\n",
      "Seq is: CTCAAACGA\n",
      "0 CTCAAACGA motifs detected in file\n",
      "\n",
      "Seq is: TCCGTCTTA\n",
      "0 TCCGTCTTA motifs detected in file\n",
      "\n",
      "Seq is: TCATTACTC\n",
      "0 TCATTACTC motifs detected in file\n",
      "\n",
      "Seq is: CTACATTCG\n",
      "0 CTACATTCG motifs detected in file\n",
      "\n",
      "Seq is: GGCACCCGC\n",
      "0 GGCACCCGC motifs detected in file\n",
      "\n",
      "Seq is: TAATACTCT\n",
      "0 TAATACTCT motifs detected in file\n",
      "\n",
      "Seq is: TACTGACGC\n",
      "0 TACTGACGC motifs detected in file\n",
      "\n",
      "Seq is: GGCCACTTC\n",
      "0 GGCCACTTC motifs detected in file\n",
      "\n",
      "Seq is: GAAGTCCCC\n",
      "0 GAAGTCCCC motifs detected in file\n",
      "\n",
      "Complement found from 1991 to 2000\n",
      "Complement found from 7121 to 7130\n",
      "Complement found from 16281 to 16290\n",
      "Seq is: CAAACCTAC\n",
      "3 CAAACCTAC motifs detected in file\n",
      "\n",
      "Seq is: TCTTACCGA\n",
      "0 TCTTACCGA motifs detected in file\n",
      "\n",
      "Seq is: CAATGATGC\n",
      "0 CAATGATGC motifs detected in file\n",
      "\n",
      "Seq is: TCCCGATAC\n",
      "0 TCCCGATAC motifs detected in file\n",
      "\n",
      "Complement found from 8644 to 8653\n",
      "Seq is: ACCGACTAA\n",
      "1 ACCGACTAA motifs detected in file\n",
      "\n",
      "Seq is: CTTCTCATA\n",
      "0 CTTCTCATA motifs detected in file\n",
      "\n",
      "Complement found from 7217 to 7226\n",
      "Seq is: CGTTACTCG\n",
      "1 CGTTACTCG motifs detected in file\n",
      "\n",
      "Seq is: CTAAAATCC\n",
      "0 CTAAAATCC motifs detected in file\n",
      "\n",
      "Seq is: AGACAAACA\n",
      "0 AGACAAACA motifs detected in file\n",
      "\n",
      "Seq is: GCATCCGTC\n",
      "0 GCATCCGTC motifs detected in file\n",
      "\n",
      "Seq is: TACCTCGAA\n",
      "0 TACCTCGAA motifs detected in file\n",
      "\n",
      "Seq is: CAATATTAA\n",
      "0 CAATATTAA motifs detected in file\n",
      "\n",
      "Seq is: TACGGAGTA\n",
      "0 TACGGAGTA motifs detected in file\n",
      "\n",
      "Seq is: TCCCTATCA\n",
      "0 TCCCTATCA motifs detected in file\n",
      "\n",
      "Seq is: TGTTCCTTC\n",
      "0 TGTTCCTTC motifs detected in file\n",
      "\n",
      "Seq is: CCCATCCGA\n",
      "0 CCCATCCGA motifs detected in file\n",
      "\n",
      "Complement found from 13214 to 13223\n",
      "Seq is: TACACAAAA\n",
      "1 TACACAAAA motifs detected in file\n",
      "\n",
      "Seq is: CAGTCCCCC\n",
      "0 CAGTCCCCC motifs detected in file\n",
      "\n",
      "Seq is: AACTCTTAC\n",
      "0 AACTCTTAC motifs detected in file\n",
      "\n",
      "Seq is: TCACACTCC\n",
      "0 TCACACTCC motifs detected in file\n",
      "\n",
      "Complement found from 1815 to 1824\n",
      "Complement found from 15686 to 15695\n",
      "Seq is: GCATAATAT\n",
      "2 GCATAATAT motifs detected in file\n",
      "\n",
      "Seq is: GGTATCGGC\n",
      "0 GGTATCGGC motifs detected in file\n",
      "\n",
      "Seq is: GGATCAAAA\n",
      "0 GGATCAAAA motifs detected in file\n",
      "\n",
      "Seq is: TTCTCATGA\n",
      "0 TTCTCATGA motifs detected in file\n",
      "\n",
      "Seq is: CGCCGTTCA\n",
      "0 CGCCGTTCA motifs detected in file\n",
      "\n",
      "Seq is: TGATAACTG\n",
      "0 TGATAACTG motifs detected in file\n",
      "\n",
      "Seq is: GGTCGCTAC\n",
      "0 GGTCGCTAC motifs detected in file\n",
      "\n",
      "Seq is: CCCCGAAGC\n",
      "0 CCCCGAAGC motifs detected in file\n",
      "\n",
      "Seq is: TGTACCCGA\n",
      "0 TGTACCCGA motifs detected in file\n",
      "\n",
      "Seq is: AATCCCTCA\n",
      "0 AATCCCTCA motifs detected in file\n",
      "\n",
      "Seq is: GTATTCACC\n",
      "0 GTATTCACC motifs detected in file\n",
      "\n",
      "Seq is: TCAGGCATT\n",
      "0 TCAGGCATT motifs detected in file\n",
      "\n",
      "Seq is: TCTCCATAG\n",
      "0 TCTCCATAG motifs detected in file\n",
      "\n",
      "Complement found from 3174 to 3183\n",
      "Seq is: AAATGATAT\n",
      "1 AAATGATAT motifs detected in file\n",
      "\n",
      "Seq is: TTTCGATAA\n",
      "0 TTTCGATAA motifs detected in file\n",
      "\n",
      "Complement found from 7411 to 7420\n",
      "Seq is: CACATTCGA\n",
      "1 CACATTCGA motifs detected in file\n",
      "\n",
      "Seq is: TCAGTATAA\n",
      "0 TCAGTATAA motifs detected in file\n",
      "\n",
      "Seq is: TTCAACAAC\n",
      "0 TTCAACAAC motifs detected in file\n",
      "\n",
      "Seq is: CGAGTCCTC\n",
      "0 CGAGTCCTC motifs detected in file\n",
      "\n",
      "Seq is: AAACTATCA\n",
      "0 AAACTATCA motifs detected in file\n",
      "\n",
      "Complement found from 7420 to 7429\n",
      "Seq is: AGAACCCGT\n",
      "1 AGAACCCGT motifs detected in file\n",
      "\n",
      "Complement found from 11295 to 11304\n",
      "Seq is: CACTCTCAC\n",
      "1 CACTCTCAC motifs detected in file\n",
      "\n",
      "Seq is: TCATCATCT\n",
      "0 TCATCATCT motifs detected in file\n",
      "\n",
      "Seq is: TACAAATCA\n",
      "0 TACAAATCA motifs detected in file\n",
      "\n",
      "Seq is: CTCGGATCC\n",
      "0 CTCGGATCC motifs detected in file\n",
      "\n",
      "Complement found from 7182 to 7191\n",
      "Complement found from 8706 to 8715\n",
      "Seq is: CACAACACT\n",
      "2 CACAACACT motifs detected in file\n",
      "\n",
      "Seq is: CACATTTAA\n",
      "0 CACATTTAA motifs detected in file\n",
      "\n",
      "Seq is: TCACGCTAC\n",
      "0 TCACGCTAC motifs detected in file\n",
      "\n",
      "Seq is: TCATCCCCT\n",
      "0 TCATCCCCT motifs detected in file\n",
      "\n",
      "Seq is: TCCCTCGGA\n",
      "0 TCCCTCGGA motifs detected in file\n",
      "\n",
      "Seq is: TGATCCCAC\n",
      "0 TGATCCCAC motifs detected in file\n",
      "\n",
      "Complement found from 4943 to 4952\n",
      "Seq is: ATCTTATCC\n",
      "1 ATCTTATCC motifs detected in file\n",
      "\n",
      "Complement found from 12070 to 12079\n",
      "Seq is: TTCATACAC\n",
      "1 TTCATACAC motifs detected in file\n",
      "\n",
      "Seq is: GGACGCAAG\n",
      "0 GGACGCAAG motifs detected in file\n",
      "\n",
      "Seq is: TCCGCAAGA\n",
      "0 TCCGCAAGA motifs detected in file\n",
      "\n",
      "Seq is: CCGACCAAC\n",
      "0 CCGACCAAC motifs detected in file\n",
      "\n",
      "Seq is: GGAGTAGCC\n",
      "0 GGAGTAGCC motifs detected in file\n",
      "\n",
      "Complement found from 12347 to 12356\n",
      "Seq is: CACTACTAT\n",
      "1 CACTACTAT motifs detected in file\n",
      "\n",
      "Seq is: CGGTTCCAC\n",
      "0 CGGTTCCAC motifs detected in file\n",
      "\n",
      "Seq is: CCCTATTCA\n",
      "0 CCCTATTCA motifs detected in file\n",
      "\n",
      "Complement found from 8477 to 8486\n",
      "Seq is: CACCAAAGC\n",
      "1 CACCAAAGC motifs detected in file\n",
      "\n",
      "Seq is: TTCTTCTAT\n",
      "0 TTCTTCTAT motifs detected in file\n",
      "\n",
      "Seq is: ATTTTATAC\n",
      "0 ATTTTATAC motifs detected in file\n",
      "\n",
      "Seq is: TAATCAAGA\n",
      "0 TAATCAAGA motifs detected in file\n",
      "\n",
      "Seq is: CACCGACAC\n",
      "0 CACCGACAC motifs detected in file\n",
      "\n",
      "Complement found from 14171 to 14180\n",
      "Seq is: TTACAATAT\n",
      "1 TTACAATAT motifs detected in file\n",
      "\n",
      "Seq is: TAATTCCTC\n",
      "0 TAATTCCTC motifs detected in file\n",
      "\n",
      "Seq is: TAAACATCC\n",
      "0 TAAACATCC motifs detected in file\n",
      "\n",
      "Seq is: CTCTAATCA\n",
      "0 CTCTAATCA motifs detected in file\n",
      "\n",
      "Complement found from 6492 to 6501\n",
      "Seq is: TATCTCTCC\n",
      "1 TATCTCTCC motifs detected in file\n",
      "\n",
      "Seq is: ATCTCAAAA\n",
      "0 ATCTCAAAA motifs detected in file\n",
      "\n",
      "Seq is: AAAGCACTA\n",
      "0 AAAGCACTA motifs detected in file\n",
      "\n",
      "Seq is: TCACCAAGT\n",
      "0 TCACCAAGT motifs detected in file\n",
      "\n",
      "Seq is: GACCTATTC\n",
      "0 GACCTATTC motifs detected in file\n",
      "\n",
      "Complement found from 6537 to 6546\n",
      "Seq is: ACCGCAACC\n",
      "1 ACCGCAACC motifs detected in file\n",
      "\n",
      "Seq is: GAACGGTAC\n",
      "0 GAACGGTAC motifs detected in file\n",
      "\n",
      "Seq is: TAACACTCC\n",
      "0 TAACACTCC motifs detected in file\n",
      "\n",
      "Complement found from 15635 to 15644\n",
      "Seq is: CCATCCTCA\n",
      "1 CCATCCTCA motifs detected in file\n",
      "\n",
      "Seq is: GTCCATCAA\n",
      "0 GTCCATCAA motifs detected in file\n",
      "\n",
      "Complement found from 11780 to 11789\n",
      "Seq is: TCATAATCC\n",
      "1 TCATAATCC motifs detected in file\n",
      "\n",
      "Seq is: TCCCCCCAA\n",
      "0 TCCCCCCAA motifs detected in file\n",
      "\n",
      "Complement found from 13777 to 13786\n",
      "Seq is: CAATCCCCC\n",
      "1 CAATCCCCC motifs detected in file\n",
      "\n",
      "Complement found from 6002 to 6011\n",
      "Seq is: AGCCTCCTT\n",
      "1 AGCCTCCTT motifs detected in file\n",
      "\n",
      "Seq is: TTCCAACCC\n",
      "0 TTCCAACCC motifs detected in file\n",
      "\n",
      "Seq is: CTTGTCGAT\n",
      "0 CTTGTCGAT motifs detected in file\n",
      "\n",
      "Seq is: TTATCCAAC\n",
      "0 TTATCCAAC motifs detected in file\n",
      "\n",
      "Complement found from 10297 to 10306\n",
      "Seq is: AACAACTAA\n",
      "1 AACAACTAA motifs detected in file\n",
      "\n",
      "Seq is: ACCAATTTT\n",
      "0 ACCAATTTT motifs detected in file\n",
      "\n",
      "Complement found from 9094 to 9103\n",
      "Seq is: TTATCATCT\n",
      "1 TTATCATCT motifs detected in file\n",
      "\n",
      "Complement found from 5148 to 5157\n",
      "Seq is: CCCTACTAC\n",
      "1 CCCTACTAC motifs detected in file\n",
      "\n",
      "Seq is: GATTATTAA\n",
      "0 GATTATTAA motifs detected in file\n",
      "\n",
      "Seq is: TCCGACACC\n",
      "0 TCCGACACC motifs detected in file\n",
      "\n",
      "Seq is: CACCAACAC\n",
      "0 CACCAACAC motifs detected in file\n",
      "\n",
      "Seq is: AACTAAGTT\n",
      "0 AACTAAGTT motifs detected in file\n",
      "\n",
      "Seq is: TAATACACA\n",
      "0 TAATACACA motifs detected in file\n",
      "\n",
      "Complement found from 2130 to 2139\n",
      "Seq is: AAAAACCTT\n",
      "1 AAAAACCTT motifs detected in file\n",
      "\n",
      "Seq is: TCAGTACAG\n",
      "0 TCAGTACAG motifs detected in file\n",
      "\n",
      "Seq is: TCACCATCA\n",
      "0 TCACCATCA motifs detected in file\n",
      "\n",
      "Seq is: TTATATTAA\n",
      "0 TTATATTAA motifs detected in file\n",
      "\n",
      "Seq is: CAACCCTGC\n",
      "0 CAACCCTGC motifs detected in file\n",
      "\n",
      "Seq is: TAATCAAAA\n",
      "0 TAATCAAAA motifs detected in file\n",
      "\n",
      "Seq is: TCGTAACCT\n",
      "0 TCGTAACCT motifs detected in file\n",
      "\n",
      "Seq is: CATCCAAAT\n",
      "0 CATCCAAAT motifs detected in file\n",
      "\n",
      "Seq is: CCAATACAT\n",
      "0 CCAATACAT motifs detected in file\n",
      "\n",
      "Seq is: GCATCAGAT\n",
      "0 GCATCAGAT motifs detected in file\n",
      "\n",
      "Seq is: CCGGTATAC\n",
      "0 CCGGTATAC motifs detected in file\n",
      "\n",
      "Seq is: ACAACCTCT\n",
      "0 ACAACCTCT motifs detected in file\n",
      "\n",
      "Seq is: AACTCTGAT\n",
      "0 AACTCTGAT motifs detected in file\n",
      "\n",
      "Complement found from 3003 to 3012\n",
      "Seq is: CATCCCGAT\n",
      "1 CATCCCGAT motifs detected in file\n",
      "\n",
      "Seq is: CCGGGTGGC\n",
      "0 CCGGGTGGC motifs detected in file\n",
      "\n",
      "Seq is: GACGAAGCG\n",
      "0 GACGAAGCG motifs detected in file\n",
      "\n",
      "Seq is: TCCGCCGTT\n",
      "0 TCCGCCGTT motifs detected in file\n",
      "\n",
      "Seq is: TCTGATCAT\n",
      "0 TCTGATCAT motifs detected in file\n",
      "\n",
      "Seq is: ACCGTTATC\n",
      "0 ACCGTTATC motifs detected in file\n",
      "\n",
      "Seq is: CGTGTTATA\n",
      "0 CGTGTTATA motifs detected in file\n",
      "\n",
      "Seq is: ACCGATTCT\n",
      "0 ACCGATTCT motifs detected in file\n",
      "\n",
      "Complement found from 16259 to 16268\n",
      "Seq is: CCCTCACCC\n",
      "1 CCCTCACCC motifs detected in file\n",
      "\n",
      "Seq is: ACAACTCCC\n",
      "0 ACAACTCCC motifs detected in file\n",
      "\n",
      "Seq is: AATACTCTC\n",
      "0 AATACTCTC motifs detected in file\n",
      "\n",
      "Seq is: ATCGATATT\n",
      "0 ATCGATATT motifs detected in file\n",
      "\n",
      "Seq is: ACTTGTCGC\n",
      "0 ACTTGTCGC motifs detected in file\n",
      "\n",
      "Seq is: TATCATAAT\n",
      "0 TATCATAAT motifs detected in file\n",
      "\n",
      "Seq is: AAGGAAGAT\n",
      "0 AAGGAAGAT motifs detected in file\n",
      "\n",
      "Seq is: CCGTATCAT\n",
      "0 CCGTATCAT motifs detected in file\n",
      "\n",
      "Seq is: CCCTCCTAT\n",
      "0 CCCTCCTAT motifs detected in file\n",
      "\n",
      "Seq is: ACTCCACAC\n",
      "0 ACTCCACAC motifs detected in file\n",
      "\n",
      "Seq is: TCGCTATAT\n",
      "0 TCGCTATAT motifs detected in file\n",
      "\n",
      "Seq is: GATCATAAG\n",
      "0 GATCATAAG motifs detected in file\n",
      "\n",
      "Seq is: GATCTTCAC\n",
      "0 GATCTTCAC motifs detected in file\n",
      "\n",
      "Seq is: TCTACCATT\n",
      "0 TCTACCATT motifs detected in file\n",
      "\n",
      "Seq is: TACGATCAT\n",
      "0 TACGATCAT motifs detected in file\n",
      "\n",
      "Seq is: ATTATAAAT\n",
      "0 ATTATAAAT motifs detected in file\n",
      "\n",
      "Seq is: ACATTTACT\n",
      "0 ACATTTACT motifs detected in file\n",
      "\n",
      "Complement found from 3168 to 3177\n",
      "Seq is: CCCCGTAAA\n",
      "1 CCCCGTAAA motifs detected in file\n",
      "\n",
      "Seq is: CCATTTATA\n",
      "0 CCATTTATA motifs detected in file\n",
      "\n",
      "Complement found from 7349 to 7358\n",
      "Seq is: CTAATAGTA\n",
      "1 CTAATAGTA motifs detected in file\n",
      "\n",
      "Seq is: TTAAATTAC\n",
      "0 TTAAATTAC motifs detected in file\n",
      "\n",
      "Seq is: TCAGCTTTA\n",
      "0 TCAGCTTTA motifs detected in file\n",
      "\n",
      "Seq is: GTAAGCAAA\n",
      "0 GTAAGCAAA motifs detected in file\n",
      "\n",
      "Seq is: ACAAATTTG\n",
      "0 ACAAATTTG motifs detected in file\n",
      "\n",
      "Seq is: ATATATGGT\n",
      "0 ATATATGGT motifs detected in file\n",
      "\n",
      "Seq is: TAAGCCAAG\n",
      "0 TAAGCCAAG motifs detected in file\n",
      "\n",
      "Seq is: TCAGATTAG\n",
      "0 TCAGATTAG motifs detected in file\n",
      "\n",
      "Seq is: GAAAAACAT\n",
      "0 GAAAAACAT motifs detected in file\n",
      "\n",
      "Seq is: CAGTGAGTA\n",
      "0 CAGTGAGTA motifs detected in file\n",
      "\n",
      "Seq is: TCCGGTCTG\n",
      "0 TCCGGTCTG motifs detected in file\n",
      "\n",
      "Seq is: AATCCCGAT\n",
      "0 AATCCCGAT motifs detected in file\n",
      "\n",
      "Complement found from 5116 to 5125\n",
      "Complement found from 5149 to 5158\n",
      "Seq is: CCTACTACT\n",
      "2 CCTACTACT motifs detected in file\n",
      "\n",
      "Seq is: AATTATTCT\n",
      "0 AATTATTCT motifs detected in file\n",
      "\n",
      "Seq is: CCCTACTGT\n",
      "0 CCCTACTGT motifs detected in file\n",
      "\n",
      "Seq is: ATTGATAAT\n",
      "0 ATTGATAAT motifs detected in file\n",
      "\n",
      "Seq is: CACCGTCCA\n",
      "0 CACCGTCCA motifs detected in file\n",
      "\n",
      "Complement found from 1117 to 1126\n",
      "Seq is: ATCAACAAA\n",
      "1 ATCAACAAA motifs detected in file\n",
      "\n",
      "Seq is: CATCCCGAG\n",
      "0 CATCCCGAG motifs detected in file\n",
      "\n",
      "Seq is: TACCATCCC\n",
      "0 TACCATCCC motifs detected in file\n",
      "\n",
      "Complement found from 7674 to 7683\n",
      "Seq is: CATTTTCCT\n",
      "1 CATTTTCCT motifs detected in file\n",
      "\n",
      "Seq is: CCCGTTAAA\n",
      "0 CCCGTTAAA motifs detected in file\n",
      "\n",
      "Seq is: GATCTAGTT\n",
      "0 GATCTAGTT motifs detected in file\n",
      "\n",
      "Seq is: TATTATTCT\n",
      "0 TATTATTCT motifs detected in file\n",
      "\n",
      "Seq is: TCCATTATC\n",
      "0 TCCATTATC motifs detected in file\n",
      "\n",
      "Seq is: GATGATTCT\n",
      "0 GATGATTCT motifs detected in file\n",
      "\n",
      "Seq is: TCTTAAAAT\n",
      "0 TCTTAAAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCTCTTTC\n",
      "0 ACCTCTTTC motifs detected in file\n",
      "\n",
      "Seq is: CCTGCGCCC\n",
      "0 CCTGCGCCC motifs detected in file\n",
      "\n",
      "Seq is: GCCCCCTAT\n",
      "0 GCCCCCTAT motifs detected in file\n",
      "\n",
      "Seq is: ATCCCAGCT\n",
      "0 ATCCCAGCT motifs detected in file\n",
      "\n",
      "Seq is: TCGGCGTGA\n",
      "0 TCGGCGTGA motifs detected in file\n",
      "\n",
      "Complement found from 4243 to 4252\n",
      "Seq is: GCATTCCCC\n",
      "1 GCATTCCCC motifs detected in file\n",
      "\n",
      "Complement found from 2222 to 2231\n",
      "Seq is: ACCTAAAAA\n",
      "1 ACCTAAAAA motifs detected in file\n",
      "\n",
      "Seq is: GATACATCG\n",
      "0 GATACATCG motifs detected in file\n",
      "\n",
      "Seq is: GCAACTCAA\n",
      "0 GCAACTCAA motifs detected in file\n",
      "\n",
      "Seq is: CACCATCAG\n",
      "0 CACCATCAG motifs detected in file\n",
      "\n",
      "Seq is: TTTTACATT\n",
      "0 TTTTACATT motifs detected in file\n",
      "\n",
      "Complement found from 4753 to 4762\n",
      "Complement found from 7300 to 7309\n",
      "Seq is: ATTAATAAT\n",
      "2 ATTAATAAT motifs detected in file\n",
      "\n",
      "Complement found from 14481 to 14490\n",
      "Seq is: CATCATTCC\n",
      "1 CATCATTCC motifs detected in file\n",
      "\n",
      "Complement found from 14258 to 14267\n",
      "Seq is: GATCCTCCC\n",
      "1 GATCCTCCC motifs detected in file\n",
      "\n",
      "Seq is: ACAACTAAT\n",
      "0 ACAACTAAT motifs detected in file\n",
      "\n",
      "Complement found from 10072 to 10081\n",
      "Seq is: AATTTTAAT\n",
      "1 AATTTTAAT motifs detected in file\n",
      "\n",
      "Seq is: TCCGCTTCA\n",
      "0 TCCGCTTCA motifs detected in file\n",
      "\n",
      "Seq is: AATAATGAG\n",
      "0 AATAATGAG motifs detected in file\n",
      "\n",
      "Complement found from 4175 to 4184\n",
      "Seq is: AAAAAACTT\n",
      "1 AAAAAACTT motifs detected in file\n",
      "\n",
      "Seq is: ACAACAGTT\n",
      "0 ACAACAGTT motifs detected in file\n",
      "\n",
      "Seq is: TTGATCAAT\n",
      "0 TTGATCAAT motifs detected in file\n",
      "\n",
      "Seq is: TAACCTTCA\n",
      "0 TAACCTTCA motifs detected in file\n",
      "\n",
      "Seq is: ATTGCCATG\n",
      "0 ATTGCCATG motifs detected in file\n",
      "\n",
      "Seq is: ATAAATATG\n",
      "0 ATAAATATG motifs detected in file\n",
      "\n",
      "Seq is: ATTTTCTCA\n",
      "0 ATTTTCTCA motifs detected in file\n",
      "\n",
      "Seq is: TTCTGGGAG\n",
      "0 TTCTGGGAG motifs detected in file\n",
      "\n",
      "Seq is: TAGTTATCT\n",
      "0 TAGTTATCT motifs detected in file\n",
      "\n",
      "Seq is: ACCTCTGTA\n",
      "0 ACCTCTGTA motifs detected in file\n",
      "\n",
      "Seq is: TGTCTTTAT\n",
      "0 TGTCTTTAT motifs detected in file\n",
      "\n",
      "Seq is: CAGTTTGGT\n",
      "0 CAGTTTGGT motifs detected in file\n",
      "\n",
      "Seq is: GTAGATGTT\n",
      "0 GTAGATGTT motifs detected in file\n",
      "\n",
      "Seq is: TTACGGTCA\n",
      "0 TTACGGTCA motifs detected in file\n",
      "\n",
      "Seq is: TAGTCCGCC\n",
      "0 TAGTCCGCC motifs detected in file\n",
      "\n",
      "Seq is: GCCGAAGCT\n",
      "0 GCCGAAGCT motifs detected in file\n",
      "\n",
      "Complement found from 12874 to 12883\n",
      "Seq is: TCGGTTTCA\n",
      "1 TCGGTTTCA motifs detected in file\n",
      "\n",
      "Complement found from 5928 to 5937\n",
      "Seq is: CTACAAACC\n",
      "1 CTACAAACC motifs detected in file\n",
      "\n",
      "Seq is: TACATTTCA\n",
      "0 TACATTTCA motifs detected in file\n",
      "\n",
      "Seq is: CTTTATAAT\n",
      "0 CTTTATAAT motifs detected in file\n",
      "\n",
      "Complement found from 14924 to 14933\n",
      "Seq is: CAACCGCCT\n",
      "1 CAACCGCCT motifs detected in file\n",
      "\n",
      "Seq is: ACTTCGTCT\n",
      "0 ACTTCGTCT motifs detected in file\n",
      "\n",
      "Seq is: ATCACTCCT\n",
      "0 ATCACTCCT motifs detected in file\n",
      "\n",
      "Seq is: TTCAACTCG\n",
      "0 TTCAACTCG motifs detected in file\n",
      "\n",
      "Seq is: GTTATTACT\n",
      "0 GTTATTACT motifs detected in file\n",
      "\n",
      "Seq is: GCACTTCAG\n",
      "0 GCACTTCAG motifs detected in file\n",
      "\n",
      "Seq is: GCACCTTCG\n",
      "0 GCACCTTCG motifs detected in file\n",
      "\n",
      "Seq is: GACACCGAT\n",
      "0 GACACCGAT motifs detected in file\n",
      "\n",
      "Seq is: GTTTTTTAC\n",
      "0 GTTTTTTAC motifs detected in file\n",
      "\n",
      "Complement found from 2471 to 2480\n",
      "Seq is: AACTCGGCA\n",
      "1 AACTCGGCA motifs detected in file\n",
      "\n",
      "Seq is: TCTACGGCA\n",
      "0 TCTACGGCA motifs detected in file\n",
      "\n",
      "Seq is: GCCTTTACC\n",
      "0 GCCTTTACC motifs detected in file\n",
      "\n",
      "Seq is: ACTTCCCTC\n",
      "0 ACTTCCCTC motifs detected in file\n",
      "\n",
      "Seq is: TGAGCTTCA\n",
      "0 TGAGCTTCA motifs detected in file\n",
      "\n",
      "Seq is: TGAGACTCC\n",
      "0 TGAGACTCC motifs detected in file\n",
      "\n",
      "Seq is: GAACATCCT\n",
      "0 GAACATCCT motifs detected in file\n",
      "\n",
      "Seq is: CCCATTTTA\n",
      "0 CCCATTTTA motifs detected in file\n",
      "\n",
      "Seq is: TCTCTGGGT\n",
      "0 TCTCTGGGT motifs detected in file\n",
      "\n",
      "Seq is: CATTTTAAC\n",
      "0 CATTTTAAC motifs detected in file\n",
      "\n",
      "Seq is: ATTATTCGT\n",
      "0 ATTATTCGT motifs detected in file\n",
      "\n",
      "Seq is: CACGAACTT\n",
      "0 CACGAACTT motifs detected in file\n",
      "\n",
      "Seq is: AATAAACCA\n",
      "0 AATAAACCA motifs detected in file\n",
      "\n",
      "Seq is: AAGCCAACA\n",
      "0 AAGCCAACA motifs detected in file\n",
      "\n",
      "Seq is: AAAGATAAT\n",
      "0 AAAGATAAT motifs detected in file\n",
      "\n",
      "Seq is: CTGATACCA\n",
      "0 CTGATACCA motifs detected in file\n",
      "\n",
      "Seq is: CTCGAGTCC\n",
      "0 CTCGAGTCC motifs detected in file\n",
      "\n",
      "Seq is: ACTAACTAT\n",
      "0 ACTAACTAT motifs detected in file\n",
      "\n",
      "Complement found from 13414 to 13423\n",
      "Seq is: GAGGACTAC\n",
      "1 GAGGACTAC motifs detected in file\n",
      "\n",
      "Seq is: GCTCATTAT\n",
      "0 GCTCATTAT motifs detected in file\n",
      "\n",
      "Complement found from 15574 to 15583\n",
      "Seq is: GCCTACACA\n",
      "1 GCCTACACA motifs detected in file\n",
      "\n",
      "Seq is: AATCCTCAC\n",
      "0 AATCCTCAC motifs detected in file\n",
      "\n",
      "Seq is: CCTGAAGAT\n",
      "0 CCTGAAGAT motifs detected in file\n",
      "\n",
      "Complement found from 14489 to 14498\n",
      "Seq is: CCCCTAAAT\n",
      "1 CCCCTAAAT motifs detected in file\n",
      "\n",
      "Seq is: CGCCCCACT\n",
      "0 CGCCCCACT motifs detected in file\n",
      "\n",
      "Seq is: ACGGACAAC\n",
      "0 ACGGACAAC motifs detected in file\n",
      "\n",
      "Seq is: CCCCGGTCA\n",
      "0 CCCCGGTCA motifs detected in file\n",
      "\n",
      "Seq is: CGGGAGGAT\n",
      "0 CGGGAGGAT motifs detected in file\n",
      "\n",
      "Seq is: TAACCCCCC\n",
      "0 TAACCCCCC motifs detected in file\n",
      "\n",
      "Seq is: ATCCCCGAT\n",
      "0 ATCCCCGAT motifs detected in file\n",
      "\n",
      "Seq is: CCGACCTCA\n",
      "0 CCGACCTCA motifs detected in file\n",
      "\n",
      "Seq is: CCATTTTCC\n",
      "0 CCATTTTCC motifs detected in file\n",
      "\n",
      "Seq is: GAGTCTTTT\n",
      "0 GAGTCTTTT motifs detected in file\n",
      "\n",
      "Seq is: TAGGACGCT\n",
      "0 TAGGACGCT motifs detected in file\n",
      "\n",
      "Seq is: TCTTTTTTT\n",
      "0 TCTTTTTTT motifs detected in file\n",
      "\n",
      "Seq is: GAAGACTCC\n",
      "0 GAAGACTCC motifs detected in file\n",
      "\n",
      "Complement found from 152 to 161\n",
      "Seq is: ATTATTTAT\n",
      "1 ATTATTTAT motifs detected in file\n",
      "\n",
      "Seq is: CCTAATAGG\n",
      "0 CCTAATAGG motifs detected in file\n",
      "\n",
      "Seq is: GCATAGCTT\n",
      "0 GCATAGCTT motifs detected in file\n",
      "\n",
      "Complement found from 6687 to 6696\n",
      "Seq is: CCGGAAAAA\n",
      "1 CCGGAAAAA motifs detected in file\n",
      "\n",
      "Seq is: CCTGTCCAC\n",
      "0 CCTGTCCAC motifs detected in file\n",
      "\n",
      "Complement found from 9416 to 9425\n",
      "Seq is: CACACACCA\n",
      "1 CACACACCA motifs detected in file\n",
      "\n",
      "Seq is: CCGGAACCA\n",
      "0 CCGGAACCA motifs detected in file\n",
      "\n",
      "Seq is: TACACGAAA\n",
      "0 TACACGAAA motifs detected in file\n",
      "\n",
      "Seq is: GAGCACAAT\n",
      "0 GAGCACAAT motifs detected in file\n",
      "\n",
      "Seq is: GTAGCGCGG\n",
      "0 GTAGCGCGG motifs detected in file\n",
      "\n",
      "Complement found from 11903 to 11912\n",
      "Seq is: TAGTAACCA\n",
      "1 TAGTAACCA motifs detected in file\n",
      "\n",
      "Complement found from 9366 to 9375\n",
      "Seq is: TATACCAAT\n",
      "1 TATACCAAT motifs detected in file\n",
      "\n",
      "Seq is: CACACAACC\n",
      "0 CACACAACC motifs detected in file\n",
      "\n",
      "Complement found from 14976 to 14985\n",
      "Seq is: AATCATCCG\n",
      "1 AATCATCCG motifs detected in file\n",
      "\n",
      "Seq is: GATCATACT\n",
      "0 GATCATACT motifs detected in file\n",
      "\n",
      "Seq is: CCTCGCAAT\n",
      "0 CCTCGCAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCTCACCT\n",
      "0 ACCTCACCT motifs detected in file\n",
      "\n",
      "Seq is: TCACTTTAG\n",
      "0 TCACTTTAG motifs detected in file\n",
      "\n",
      "Seq is: TGTACCGAT\n",
      "0 TGTACCGAT motifs detected in file\n",
      "\n",
      "Seq is: CCGGCCTCC\n",
      "0 CCGGCCTCC motifs detected in file\n",
      "\n",
      "Seq is: AGTAATCCT\n",
      "0 AGTAATCCT motifs detected in file\n",
      "\n",
      "Seq is: CCCGACTCT\n",
      "0 CCCGACTCT motifs detected in file\n",
      "\n",
      "Seq is: CCCGGGGAC\n",
      "0 CCCGGGGAC motifs detected in file\n",
      "\n",
      "Seq is: AATCCCCAG\n",
      "0 AATCCCCAG motifs detected in file\n",
      "\n",
      "Seq is: TACCCGACC\n",
      "0 TACCCGACC motifs detected in file\n",
      "\n",
      "Seq is: CAAAATGAT\n",
      "0 CAAAATGAT motifs detected in file\n",
      "\n",
      "Seq is: ATACTATCC\n",
      "0 ATACTATCC motifs detected in file\n",
      "\n",
      "Seq is: GTACACTAA\n",
      "0 GTACACTAA motifs detected in file\n",
      "\n",
      "Seq is: CCACCCAGT\n",
      "0 CCACCCAGT motifs detected in file\n",
      "\n",
      "Complement found from 3985 to 3994\n",
      "Seq is: AATACACAA\n",
      "1 AATACACAA motifs detected in file\n",
      "\n",
      "Seq is: CAGCACGTC\n",
      "0 CAGCACGTC motifs detected in file\n",
      "\n",
      "Seq is: CATCTCCGA\n",
      "0 CATCTCCGA motifs detected in file\n",
      "\n",
      "Seq is: ATGATCTTC\n",
      "0 ATGATCTTC motifs detected in file\n",
      "\n",
      "Seq is: ACACTTTTG\n",
      "0 ACACTTTTG motifs detected in file\n",
      "\n",
      "Seq is: CATCCGAAC\n",
      "0 CATCCGAAC motifs detected in file\n",
      "\n",
      "Seq is: CTAATTCCG\n",
      "0 CTAATTCCG motifs detected in file\n",
      "\n",
      "Seq is: CTGTCGCTA\n",
      "0 CTGTCGCTA motifs detected in file\n",
      "\n",
      "Seq is: AAGATCCTA\n",
      "0 AAGATCCTA motifs detected in file\n",
      "\n",
      "Seq is: TCAGTCATC\n",
      "0 TCAGTCATC motifs detected in file\n",
      "\n",
      "Seq is: TTAATCTTA\n",
      "0 TTAATCTTA motifs detected in file\n",
      "\n",
      "Complement found from 9168 to 9177\n",
      "Seq is: ACACTTCTA\n",
      "1 ACACTTCTA motifs detected in file\n",
      "\n",
      "Seq is: CTATTCACA\n",
      "0 CTATTCACA motifs detected in file\n",
      "\n",
      "Complement found from 9755 to 9764\n",
      "Seq is: TCTCCCTTC\n",
      "1 TCTCCCTTC motifs detected in file\n",
      "\n",
      "Seq is: CAATTACCA\n",
      "0 CAATTACCA motifs detected in file\n",
      "\n",
      "Seq is: ACTATAACG\n",
      "0 ACTATAACG motifs detected in file\n",
      "\n",
      "Seq is: ATCCCACCG\n",
      "0 ATCCCACCG motifs detected in file\n",
      "\n",
      "Seq is: CGAAGGTTA\n",
      "0 CGAAGGTTA motifs detected in file\n",
      "\n",
      "Seq is: ATCCACGTA\n",
      "0 ATCCACGTA motifs detected in file\n",
      "\n",
      "Seq is: CTCATCCAC\n",
      "0 CTCATCCAC motifs detected in file\n",
      "\n",
      "Seq is: CGGACGTCA\n",
      "0 CGGACGTCA motifs detected in file\n",
      "\n",
      "Complement found from 13279 to 13288\n",
      "Seq is: TTACAATCG\n",
      "1 TTACAATCG motifs detected in file\n",
      "\n",
      "Seq is: CCAATCCGC\n",
      "0 CCAATCCGC motifs detected in file\n",
      "\n",
      "Seq is: ATGCCGGTC\n",
      "0 ATGCCGGTC motifs detected in file\n",
      "\n",
      "Seq is: CCGATAACC\n",
      "0 CCGATAACC motifs detected in file\n",
      "\n",
      "Seq is: AACTTACTC\n",
      "0 AACTTACTC motifs detected in file\n",
      "\n",
      "Seq is: ATCCGACTA\n",
      "0 ATCCGACTA motifs detected in file\n",
      "\n",
      "Complement found from 1888 to 1897\n",
      "Complement found from 15989 to 15998\n",
      "Seq is: CCAAAGCTA\n",
      "2 CCAAAGCTA motifs detected in file\n",
      "\n",
      "Seq is: TTATTGATC\n",
      "0 TTATTGATC motifs detected in file\n",
      "\n",
      "Seq is: ATACCCCTA\n",
      "0 ATACCCCTA motifs detected in file\n",
      "\n",
      "Complement found from 12480 to 12489\n",
      "Seq is: TTCCCCACA\n",
      "1 TTCCCCACA motifs detected in file\n",
      "\n",
      "Seq is: TCCACACGG\n",
      "0 TCCACACGG motifs detected in file\n",
      "\n",
      "Seq is: AACACCATT\n",
      "0 AACACCATT motifs detected in file\n",
      "\n",
      "Seq is: CTTCACCCG\n",
      "0 CTTCACCCG motifs detected in file\n",
      "\n",
      "Seq is: ATCCCGTAA\n",
      "0 ATCCCGTAA motifs detected in file\n",
      "\n",
      "Seq is: AAATTAGAA\n",
      "0 AAATTAGAA motifs detected in file\n",
      "\n",
      "Seq is: TCTCGCTTT\n",
      "0 TCTCGCTTT motifs detected in file\n",
      "\n",
      "Seq is: CGGATATTA\n",
      "0 CGGATATTA motifs detected in file\n",
      "\n",
      "Seq is: GTGACACGG\n",
      "0 GTGACACGG motifs detected in file\n",
      "\n",
      "Seq is: GCGAGTATT\n",
      "0 GCGAGTATT motifs detected in file\n",
      "\n",
      "Seq is: CCCCTACCG\n",
      "0 CCCCTACCG motifs detected in file\n",
      "\n",
      "Seq is: GTACCGATC\n",
      "0 GTACCGATC motifs detected in file\n",
      "\n",
      "Complement found from 8627 to 8636\n",
      "Complement found from 12689 to 12698\n",
      "Seq is: CAAATATCT\n",
      "2 CAAATATCT motifs detected in file\n",
      "\n",
      "Seq is: ATCAACCCA\n",
      "0 ATCAACCCA motifs detected in file\n",
      "\n",
      "Complement found from 13295 to 13304\n",
      "Seq is: CCAACCACA\n",
      "1 CCAACCACA motifs detected in file\n",
      "\n",
      "Seq is: TTTACTCAC\n",
      "0 TTTACTCAC motifs detected in file\n",
      "\n",
      "Seq is: TCCGTCCTC\n",
      "0 TCCGTCCTC motifs detected in file\n",
      "\n",
      "Seq is: AGGCTCCTC\n",
      "0 AGGCTCCTC motifs detected in file\n",
      "\n",
      "Seq is: CAATCAACA\n",
      "0 CAATCAACA motifs detected in file\n",
      "\n",
      "Seq is: CCGTTATTT\n",
      "0 CCGTTATTT motifs detected in file\n",
      "\n",
      "Seq is: TTACTAATT\n",
      "0 TTACTAATT motifs detected in file\n",
      "\n",
      "Seq is: CCTATGATC\n",
      "0 CCTATGATC motifs detected in file\n",
      "\n",
      "Seq is: ATATTCTCT\n",
      "0 ATATTCTCT motifs detected in file\n",
      "\n",
      "Seq is: AGTCCAAGC\n",
      "0 AGTCCAAGC motifs detected in file\n",
      "\n",
      "Seq is: AGGAAATCA\n",
      "0 AGGAAATCA motifs detected in file\n",
      "\n",
      "Complement found from 9197 to 9206\n",
      "Complement found from 10714 to 10723\n",
      "Seq is: CAACACATA\n",
      "2 CAACACATA motifs detected in file\n",
      "\n",
      "Seq is: CCAATAGTA\n",
      "0 CCAATAGTA motifs detected in file\n",
      "\n",
      "Complement found from 1120 to 1129\n",
      "Complement found from 7725 to 7734\n",
      "Seq is: AACAAAACT\n",
      "2 AACAAAACT motifs detected in file\n",
      "\n",
      "Seq is: CCAATCAAA\n",
      "0 CCAATCAAA motifs detected in file\n",
      "\n",
      "Seq is: CTAATCAGT\n",
      "0 CTAATCAGT motifs detected in file\n",
      "\n",
      "Complement found from 12551 to 12560\n",
      "Seq is: AACAACCCA\n",
      "1 AACAACCCA motifs detected in file\n",
      "\n",
      "Complement found from 13146 to 13155\n",
      "Seq is: CCACTAATC\n",
      "1 CCACTAATC motifs detected in file\n",
      "\n",
      "Complement found from 11328 to 11337\n",
      "Seq is: AGCCAACAA\n",
      "1 AGCCAACAA motifs detected in file\n",
      "\n",
      "Seq is: CTACTCTAT\n",
      "0 CTACTCTAT motifs detected in file\n",
      "\n",
      "Seq is: AAACCTCCA\n",
      "0 AAACCTCCA motifs detected in file\n",
      "\n",
      "Seq is: CCCCTAGTT\n",
      "0 CCCCTAGTT motifs detected in file\n",
      "\n",
      "Seq is: ATCTCCCCC\n",
      "0 ATCTCCCCC motifs detected in file\n",
      "\n",
      "Seq is: TTTATCTTA\n",
      "0 TTTATCTTA motifs detected in file\n",
      "\n",
      "Seq is: CTAGTCATG\n",
      "0 CTAGTCATG motifs detected in file\n",
      "\n",
      "Seq is: ACGCCGCCC\n",
      "0 ACGCCGCCC motifs detected in file\n",
      "\n",
      "Seq is: ATCCGGATC\n",
      "0 ATCCGGATC motifs detected in file\n",
      "\n",
      "Complement found from 5186 to 5195\n",
      "Seq is: CTAACACCC\n",
      "1 CTAACACCC motifs detected in file\n",
      "\n",
      "Seq is: CCGTTACTT\n",
      "0 CCGTTACTT motifs detected in file\n",
      "\n",
      "Seq is: ACTTCGCTT\n",
      "0 ACTTCGCTT motifs detected in file\n",
      "\n",
      "Seq is: GTCTAAAAG\n",
      "0 GTCTAAAAG motifs detected in file\n",
      "\n",
      "Seq is: CAAGTAAAA\n",
      "0 CAAGTAAAA motifs detected in file\n",
      "\n",
      "Seq is: CCAAGAGTC\n",
      "0 CCAAGAGTC motifs detected in file\n",
      "\n",
      "Complement found from 3716 to 3725\n",
      "Seq is: CCAAACAAT\n",
      "1 CCAAACAAT motifs detected in file\n",
      "\n",
      "Complement found from 14500 to 14509\n",
      "Seq is: ATTAAAAAA\n",
      "1 ATTAAAAAA motifs detected in file\n",
      "\n",
      "Seq is: TAAAAATAC\n",
      "0 TAAAAATAC motifs detected in file\n",
      "\n",
      "Complement found from 1903 to 1912\n",
      "Complement found from 9670 to 9679\n",
      "Seq is: CCGAAACCA\n",
      "2 CCGAAACCA motifs detected in file\n",
      "\n",
      "Seq is: CTCCCTCCA\n",
      "0 CTCCCTCCA motifs detected in file\n",
      "\n",
      "Seq is: TCCACCATC\n",
      "0 TCCACCATC motifs detected in file\n",
      "\n",
      "Complement found from 8451 to 8460\n",
      "Seq is: AAACACAAA\n",
      "1 AAACACAAA motifs detected in file\n",
      "\n",
      "Seq is: TTATAAAAA\n",
      "0 TTATAAAAA motifs detected in file\n",
      "\n",
      "Seq is: TCAACCCAC\n",
      "0 TCAACCCAC motifs detected in file\n",
      "\n",
      "Complement found from 8411 to 8420\n",
      "Seq is: TACTCCTTA\n",
      "1 TACTCCTTA motifs detected in file\n",
      "\n",
      "Seq is: TCACATTCC\n",
      "0 TCACATTCC motifs detected in file\n",
      "\n",
      "Complement found from 12426 to 12435\n",
      "Seq is: TCATACCCC\n",
      "1 TCATACCCC motifs detected in file\n",
      "\n",
      "Seq is: CATTAATAC\n",
      "0 CATTAATAC motifs detected in file\n",
      "\n",
      "Seq is: CACCCGGTA\n",
      "0 CACCCGGTA motifs detected in file\n",
      "\n",
      "Complement found from 3832 to 3841\n",
      "Complement found from 7063 to 7072\n",
      "Seq is: TGCCATCAT\n",
      "2 TGCCATCAT motifs detected in file\n",
      "\n",
      "Seq is: AAATCAACC\n",
      "0 AAATCAACC motifs detected in file\n",
      "\n",
      "Seq is: CCGTAAAGT\n",
      "0 CCGTAAAGT motifs detected in file\n",
      "\n",
      "Seq is: GACATTTCT\n",
      "0 GACATTTCT motifs detected in file\n",
      "\n",
      "Complement found from 14327 to 14336\n",
      "Seq is: CCACAACCA\n",
      "1 CCACAACCA motifs detected in file\n",
      "\n",
      "Seq is: AGAGAATTA\n",
      "0 AGAGAATTA motifs detected in file\n",
      "\n",
      "Complement found from 1753 to 1762\n",
      "Seq is: GAAATTGAA\n",
      "1 GAAATTGAA motifs detected in file\n",
      "\n",
      "Seq is: TTTTCCAAT\n",
      "0 TTTTCCAAT motifs detected in file\n",
      "\n",
      "Seq is: TACGATTCA\n",
      "0 TACGATTCA motifs detected in file\n",
      "\n",
      "Seq is: ATCGAAATG\n",
      "0 ATCGAAATG motifs detected in file\n",
      "\n",
      "Seq is: TCACCCGAG\n",
      "0 TCACCCGAG motifs detected in file\n",
      "\n",
      "Seq is: ATCTCCCCC\n",
      "0 ATCTCCCCC motifs detected in file\n",
      "\n",
      "Seq is: ATCTCCCCC\n",
      "0 ATCTCCCCC motifs detected in file\n",
      "\n",
      "Seq is: ACGATATCC\n",
      "0 ACGATATCC motifs detected in file\n",
      "\n",
      "Seq is: CATTTATGC\n",
      "0 CATTTATGC motifs detected in file\n",
      "\n",
      "Seq is: CCGGGATAA\n",
      "0 CCGGGATAA motifs detected in file\n",
      "\n",
      "Seq is: AGTTTCTAA\n",
      "0 AGTTTCTAA motifs detected in file\n",
      "\n",
      "Complement found from 2071 to 2080\n",
      "Complement found from 9577 to 9586\n",
      "Seq is: AAATCCCCT\n",
      "2 AAATCCCCT motifs detected in file\n",
      "\n",
      "Seq is: TAATTAAGA\n",
      "0 TAATTAAGA motifs detected in file\n",
      "\n",
      "Seq is: TCCTGCTAC\n",
      "0 TCCTGCTAC motifs detected in file\n",
      "\n",
      "Seq is: CCGTACTTT\n",
      "0 CCGTACTTT motifs detected in file\n",
      "\n",
      "Seq is: GACACCAAA\n",
      "0 GACACCAAA motifs detected in file\n",
      "\n",
      "Seq is: CGAGGTGTC\n",
      "0 CGAGGTGTC motifs detected in file\n",
      "\n",
      "Seq is: TAAAGTCTC\n",
      "0 TAAAGTCTC motifs detected in file\n",
      "\n",
      "Seq is: GTAACTGGC\n",
      "0 GTAACTGGC motifs detected in file\n",
      "\n",
      "Seq is: ATCATATGG\n",
      "0 ATCATATGG motifs detected in file\n",
      "\n",
      "Seq is: GGGCCAGCA\n",
      "0 GGGCCAGCA motifs detected in file\n",
      "\n",
      "Seq is: CATCGCCAC\n",
      "0 CATCGCCAC motifs detected in file\n",
      "\n",
      "Seq is: TTTCACCAA\n",
      "0 TTTCACCAA motifs detected in file\n",
      "\n",
      "Complement found from 4896 to 4905\n",
      "Complement found from 14041 to 14050\n",
      "Seq is: ACCAAATCT\n",
      "2 ACCAAATCT motifs detected in file\n",
      "\n",
      "Seq is: GCAGGCCCT\n",
      "0 GCAGGCCCT motifs detected in file\n",
      "\n",
      "Seq is: TAACGTAGA\n",
      "0 TAACGTAGA motifs detected in file\n",
      "\n",
      "Seq is: CAAAAATTC\n",
      "0 CAAAAATTC motifs detected in file\n",
      "\n",
      "Seq is: GGATTACAC\n",
      "0 GGATTACAC motifs detected in file\n",
      "\n",
      "Seq is: CCCTGTCGA\n",
      "0 CCCTGTCGA motifs detected in file\n",
      "\n",
      "Seq is: GTACTCACG\n",
      "0 GTACTCACG motifs detected in file\n",
      "\n",
      "Seq is: TTCTGCAGA\n",
      "0 TTCTGCAGA motifs detected in file\n",
      "\n",
      "Seq is: ACACTACAT\n",
      "0 ACACTACAT motifs detected in file\n",
      "\n",
      "Seq is: TAATAATAT\n",
      "0 TAATAATAT motifs detected in file\n",
      "\n",
      "Seq is: GCTTACCCC\n",
      "0 GCTTACCCC motifs detected in file\n",
      "\n",
      "Seq is: CGAAGTTAG\n",
      "0 CGAAGTTAG motifs detected in file\n",
      "\n",
      "Seq is: CCCTCATGA\n",
      "0 CCCTCATGA motifs detected in file\n",
      "\n",
      "Seq is: TGAGCTAAC\n",
      "0 TGAGCTAAC motifs detected in file\n",
      "\n",
      "Seq is: AGTTGCAGT\n",
      "0 AGTTGCAGT motifs detected in file\n",
      "\n",
      "Seq is: TCCTCAGCG\n",
      "0 TCCTCAGCG motifs detected in file\n",
      "\n",
      "Seq is: TCCAGCGGA\n",
      "0 TCCAGCGGA motifs detected in file\n",
      "\n",
      "Seq is: CCAAGATCC\n",
      "0 CCAAGATCC motifs detected in file\n",
      "\n",
      "Complement found from 2406 to 2415\n",
      "Seq is: TTATTACCC\n",
      "1 TTATTACCC motifs detected in file\n",
      "\n",
      "Seq is: CCTTCATAC\n",
      "0 CCTTCATAC motifs detected in file\n",
      "\n",
      "Seq is: ATCCTCAAC\n",
      "0 ATCCTCAAC motifs detected in file\n",
      "\n",
      "Seq is: TTCTAATCA\n",
      "0 TTCTAATCA motifs detected in file\n",
      "\n",
      "Seq is: GGCGGCATC\n",
      "0 GGCGGCATC motifs detected in file\n",
      "\n",
      "Seq is: AGCCACATG\n",
      "0 AGCCACATG motifs detected in file\n",
      "\n",
      "Seq is: AGCATCCAA\n",
      "0 AGCATCCAA motifs detected in file\n",
      "\n",
      "Seq is: GTCATGGTA\n",
      "0 GTCATGGTA motifs detected in file\n",
      "\n",
      "Seq is: ACCACCGGT\n",
      "0 ACCACCGGT motifs detected in file\n",
      "\n",
      "Seq is: TAACTAAAC\n",
      "0 TAACTAAAC motifs detected in file\n",
      "\n",
      "Seq is: TACCATTCC\n",
      "0 TACCATTCC motifs detected in file\n",
      "\n",
      "Seq is: CTCCCTAGC\n",
      "0 CTCCCTAGC motifs detected in file\n",
      "\n",
      "Seq is: AACTGGAGC\n",
      "0 AACTGGAGC motifs detected in file\n",
      "\n",
      "Seq is: AGACAATAC\n",
      "0 AGACAATAC motifs detected in file\n",
      "\n",
      "Seq is: ATTTCCTAC\n",
      "0 ATTTCCTAC motifs detected in file\n",
      "\n",
      "Seq is: GCATCCCTA\n",
      "0 GCATCCCTA motifs detected in file\n",
      "\n",
      "Seq is: CCCTCCCGC\n",
      "0 CCCTCCCGC motifs detected in file\n",
      "\n",
      "Seq is: TACTCCTGA\n",
      "0 TACTCCTGA motifs detected in file\n",
      "\n",
      "Complement found from 5115 to 5124\n",
      "Seq is: TCCTACTAC\n",
      "1 TCCTACTAC motifs detected in file\n",
      "\n",
      "Seq is: CGCCCGTCC\n",
      "0 CGCCCGTCC motifs detected in file\n",
      "\n",
      "Seq is: TATCAAGTC\n",
      "0 TATCAAGTC motifs detected in file\n",
      "\n",
      "Seq is: TGCCAAAGA\n",
      "0 TGCCAAAGA motifs detected in file\n",
      "\n",
      "Seq is: TAAAGGACT\n",
      "0 TAAAGGACT motifs detected in file\n",
      "\n",
      "Seq is: CGCAGACTC\n",
      "0 CGCAGACTC motifs detected in file\n",
      "\n",
      "Seq is: TACAATCAT\n",
      "0 TACAATCAT motifs detected in file\n",
      "\n",
      "Seq is: AATCAATCA\n",
      "0 AATCAATCA motifs detected in file\n",
      "\n",
      "Seq is: AAACAACAC\n",
      "0 AAACAACAC motifs detected in file\n",
      "\n",
      "Seq is: TCACAATCC\n",
      "0 TCACAATCC motifs detected in file\n",
      "\n",
      "Seq is: TTTTCCCGT\n",
      "0 TTTTCCCGT motifs detected in file\n",
      "\n",
      "Seq is: ATGTCCTGA\n",
      "0 ATGTCCTGA motifs detected in file\n",
      "\n",
      "Seq is: TCCTTCGTC\n",
      "0 TCCTTCGTC motifs detected in file\n",
      "\n",
      "Seq is: TATTCCTTT\n",
      "0 TATTCCTTT motifs detected in file\n",
      "\n",
      "Seq is: TACTAATAC\n",
      "0 TACTAATAC motifs detected in file\n",
      "\n",
      "Seq is: TCCCGCACT\n",
      "0 TCCCGCACT motifs detected in file\n",
      "\n",
      "Seq is: AGTACTTTC\n",
      "0 AGTACTTTC motifs detected in file\n",
      "\n",
      "Seq is: CACTATTCG\n",
      "0 CACTATTCG motifs detected in file\n",
      "\n",
      "Seq is: AGAAGATAC\n",
      "0 AGAAGATAC motifs detected in file\n",
      "\n",
      "Seq is: TATCCCCTT\n",
      "0 TATCCCCTT motifs detected in file\n",
      "\n",
      "Seq is: CATCGCAGA\n",
      "0 CATCGCAGA motifs detected in file\n",
      "\n",
      "Seq is: ACATCTGGA\n",
      "0 ACATCTGGA motifs detected in file\n",
      "\n",
      "Seq is: TGAACGCGA\n",
      "0 TGAACGCGA motifs detected in file\n",
      "\n",
      "Seq is: CGTACACGG\n",
      "0 CGTACACGG motifs detected in file\n",
      "\n",
      "Seq is: TAATTCTAT\n",
      "0 TAATTCTAT motifs detected in file\n",
      "\n",
      "Seq is: ATATCCTAA\n",
      "0 ATATCCTAA motifs detected in file\n",
      "\n",
      "Seq is: ATCGGATAT\n",
      "0 ATCGGATAT motifs detected in file\n",
      "\n",
      "Seq is: TAAATTGAA\n",
      "0 TAAATTGAA motifs detected in file\n",
      "\n",
      "Seq is: ACTGTTTCA\n",
      "0 ACTGTTTCA motifs detected in file\n",
      "\n",
      "Seq is: ATACTTTAC\n",
      "0 ATACTTTAC motifs detected in file\n",
      "\n",
      "Seq is: CAAAAAGAT\n",
      "0 CAAAAAGAT motifs detected in file\n",
      "\n",
      "Seq is: TATGGAAAA\n",
      "0 TATGGAAAA motifs detected in file\n",
      "\n",
      "Complement found from 7506 to 7515\n",
      "Seq is: ACTTTTTCA\n",
      "1 ACTTTTTCA motifs detected in file\n",
      "\n",
      "Seq is: GTACCTCCG\n",
      "0 GTACCTCCG motifs detected in file\n",
      "\n",
      "Seq is: GTACCCCAA\n",
      "0 GTACCCCAA motifs detected in file\n",
      "\n",
      "Seq is: CCGAACTTT\n",
      "0 CCGAACTTT motifs detected in file\n",
      "\n",
      "Seq is: GGTCGAAAC\n",
      "0 GGTCGAAAC motifs detected in file\n",
      "\n",
      "Seq is: CCCCCAAGC\n",
      "0 CCCCCAAGC motifs detected in file\n",
      "\n",
      "Seq is: TAAGGAAGG\n",
      "0 TAAGGAAGG motifs detected in file\n",
      "\n",
      "Seq is: AAAAAACAG\n",
      "0 AAAAAACAG motifs detected in file\n",
      "\n",
      "Seq is: ATCTAAAAT\n",
      "0 ATCTAAAAT motifs detected in file\n",
      "\n",
      "Seq is: ACATATGCC\n",
      "0 ACATATGCC motifs detected in file\n",
      "\n",
      "Seq is: CAAGAAGCT\n",
      "0 CAAGAAGCT motifs detected in file\n",
      "\n",
      "Seq is: TACACACCA\n",
      "0 TACACACCA motifs detected in file\n",
      "\n",
      "Seq is: TCCCACCCC\n",
      "0 TCCCACCCC motifs detected in file\n",
      "\n",
      "Seq is: CCGTAGGTA\n",
      "0 CCGTAGGTA motifs detected in file\n",
      "\n",
      "Seq is: TATCAGTGA\n",
      "0 TATCAGTGA motifs detected in file\n",
      "\n",
      "Seq is: GGTCCAAAT\n",
      "0 GGTCCAAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCTCCCAA\n",
      "0 ACCTCCCAA motifs detected in file\n",
      "\n",
      "Seq is: GAAGATGAT\n",
      "0 GAAGATGAT motifs detected in file\n",
      "\n",
      "Seq is: AATCCTGAA\n",
      "0 AATCCTGAA motifs detected in file\n",
      "\n",
      "Seq is: AAGCGAAGC\n",
      "0 AAGCGAAGC motifs detected in file\n",
      "\n",
      "Seq is: TTCGCTTCC\n",
      "0 TTCGCTTCC motifs detected in file\n",
      "\n",
      "Seq is: GAAGAGTTT\n",
      "0 GAAGAGTTT motifs detected in file\n",
      "\n",
      "Seq is: AGTACTTTT\n",
      "0 AGTACTTTT motifs detected in file\n",
      "\n",
      "Complement found from 10108 to 10117\n",
      "Seq is: AATAATTAT\n",
      "1 AATAATTAT motifs detected in file\n",
      "\n",
      "Seq is: AATGACGAC\n",
      "0 AATGACGAC motifs detected in file\n",
      "\n",
      "Seq is: AATCTCTTT\n",
      "0 AATCTCTTT motifs detected in file\n",
      "\n",
      "Seq is: ACTTACTCG\n",
      "0 ACTTACTCG motifs detected in file\n",
      "\n",
      "Seq is: GATGTCTAC\n",
      "0 GATGTCTAC motifs detected in file\n",
      "\n",
      "Complement found from 12902 to 12911\n",
      "Seq is: TATCCTACA\n",
      "1 TATCCTACA motifs detected in file\n",
      "\n",
      "Seq is: AAGTACACC\n",
      "0 AAGTACACC motifs detected in file\n",
      "\n",
      "Seq is: ACATACGTA\n",
      "0 ACATACGTA motifs detected in file\n",
      "\n",
      "Seq is: GCCCCATCA\n",
      "0 GCCCCATCA motifs detected in file\n",
      "\n",
      "Seq is: GGCTCATTG\n",
      "0 GGCTCATTG motifs detected in file\n",
      "\n",
      "Seq is: CAGCCCCGT\n",
      "0 CAGCCCCGT motifs detected in file\n",
      "\n",
      "Seq is: AAGGCCTAT\n",
      "0 AAGGCCTAT motifs detected in file\n",
      "\n",
      "Seq is: CCGGCTCTT\n",
      "0 CCGGCTCTT motifs detected in file\n",
      "\n",
      "Complement found from 3802 to 3811\n",
      "Complement found from 11259 to 11268\n",
      "Seq is: TCACAACAC\n",
      "2 TCACAACAC motifs detected in file\n",
      "\n",
      "Seq is: CCTTCTTTC\n",
      "0 CCTTCTTTC motifs detected in file\n",
      "\n",
      "Seq is: AATCTAAAT\n",
      "0 AATCTAAAT motifs detected in file\n",
      "\n",
      "Seq is: GCGGCTACT\n",
      "0 GCGGCTACT motifs detected in file\n",
      "\n",
      "Seq is: TATACTATC\n",
      "0 TATACTATC motifs detected in file\n",
      "\n",
      "Seq is: ACTTTACCT\n",
      "0 ACTTTACCT motifs detected in file\n",
      "\n",
      "Seq is: AAAACCGCA\n",
      "0 AAAACCGCA motifs detected in file\n",
      "\n",
      "Seq is: TCCAAACCA\n",
      "0 TCCAAACCA motifs detected in file\n",
      "\n",
      "Seq is: GATCCCACA\n",
      "0 GATCCCACA motifs detected in file\n",
      "\n",
      "Seq is: TCGGACTCT\n",
      "0 TCGGACTCT motifs detected in file\n",
      "\n",
      "Seq is: TATCCCCTT\n",
      "0 TATCCCCTT motifs detected in file\n",
      "\n",
      "Seq is: TAGTCACTT\n",
      "0 TAGTCACTT motifs detected in file\n",
      "\n",
      "Seq is: ACTTCGGAG\n",
      "0 ACTTCGGAG motifs detected in file\n",
      "\n",
      "Seq is: GATACTACC\n",
      "0 GATACTACC motifs detected in file\n",
      "\n",
      "Seq is: GTTTATGTC\n",
      "0 GTTTATGTC motifs detected in file\n",
      "\n",
      "Seq is: GAGGATAAC\n",
      "0 GAGGATAAC motifs detected in file\n",
      "\n",
      "Seq is: TATCCTGTA\n",
      "0 TATCCTGTA motifs detected in file\n",
      "\n",
      "Seq is: TCACCTTCA\n",
      "0 TCACCTTCA motifs detected in file\n",
      "\n",
      "Seq is: CCCGATGTT\n",
      "0 CCCGATGTT motifs detected in file\n",
      "\n",
      "Seq is: GCATCATGC\n",
      "0 GCATCATGC motifs detected in file\n",
      "\n",
      "Seq is: ACAGCACAT\n",
      "0 ACAGCACAT motifs detected in file\n",
      "\n",
      "Seq is: CATGCTACA\n",
      "0 CATGCTACA motifs detected in file\n",
      "\n",
      "Seq is: GATCACTAC\n",
      "0 GATCACTAC motifs detected in file\n",
      "\n",
      "Seq is: TCAAACGAT\n",
      "0 TCAAACGAT motifs detected in file\n",
      "\n",
      "Seq is: TATGTTACG\n",
      "0 TATGTTACG motifs detected in file\n",
      "\n",
      "Seq is: GTCAGTCCG\n",
      "0 GTCAGTCCG motifs detected in file\n",
      "\n",
      "Seq is: GTGGATGCC\n",
      "0 GTGGATGCC motifs detected in file\n",
      "\n",
      "Seq is: ACTTTTCTT\n",
      "0 ACTTTTCTT motifs detected in file\n",
      "\n",
      "Seq is: TCTACTTAG\n",
      "0 TCTACTTAG motifs detected in file\n",
      "\n",
      "Seq is: GATCCCGAG\n",
      "0 GATCCCGAG motifs detected in file\n",
      "\n",
      "Seq is: TCTCGTGAC\n",
      "0 TCTCGTGAC motifs detected in file\n",
      "\n",
      "Seq is: GTCGTCTAG\n",
      "0 GTCGTCTAG motifs detected in file\n",
      "\n",
      "Complement found from 1737 to 1746\n",
      "Seq is: TAAAGTATA\n",
      "1 TAAAGTATA motifs detected in file\n",
      "\n",
      "Seq is: ACGAAGGCA\n",
      "0 ACGAAGGCA motifs detected in file\n",
      "\n",
      "Complement found from 2253 to 2262\n",
      "Seq is: CCTCACACC\n",
      "1 CCTCACACC motifs detected in file\n",
      "\n",
      "Seq is: GCTCAGTCG\n",
      "0 GCTCAGTCG motifs detected in file\n",
      "\n",
      "Seq is: ATTTATGAA\n",
      "0 ATTTATGAA motifs detected in file\n",
      "\n",
      "Seq is: ACTGCGGCC\n",
      "0 ACTGCGGCC motifs detected in file\n",
      "\n",
      "Seq is: ACCCCTATC\n",
      "0 ACCCCTATC motifs detected in file\n",
      "\n",
      "Seq is: GCTACTAAT\n",
      "0 GCTACTAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCATCGCC\n",
      "0 ACCATCGCC motifs detected in file\n",
      "\n",
      "Seq is: TCCACTTTA\n",
      "0 TCCACTTTA motifs detected in file\n",
      "\n",
      "Seq is: TACGAGCAC\n",
      "0 TACGAGCAC motifs detected in file\n",
      "\n",
      "Complement found from 8098 to 8107\n",
      "Seq is: ACAGATGCA\n",
      "1 ACAGATGCA motifs detected in file\n",
      "\n",
      "Seq is: GATAAGGAT\n",
      "0 GATAAGGAT motifs detected in file\n",
      "\n",
      "Seq is: GACATTTAT\n",
      "0 GACATTTAT motifs detected in file\n",
      "\n",
      "Seq is: ATACCACAC\n",
      "0 ATACCACAC motifs detected in file\n",
      "\n",
      "Seq is: GAGTGTGCT\n",
      "0 GAGTGTGCT motifs detected in file\n",
      "\n",
      "Seq is: ATTTGGGAT\n",
      "0 ATTTGGGAT motifs detected in file\n",
      "\n",
      "Seq is: CCTTCGGTT\n",
      "0 CCTTCGGTT motifs detected in file\n",
      "\n",
      "Seq is: AACTATAGT\n",
      "0 AACTATAGT motifs detected in file\n",
      "\n",
      "Seq is: ATCGAGTCT\n",
      "0 ATCGAGTCT motifs detected in file\n",
      "\n",
      "Seq is: GGTATGGAT\n",
      "0 GGTATGGAT motifs detected in file\n",
      "\n",
      "Seq is: ACATAGGTT\n",
      "0 ACATAGGTT motifs detected in file\n",
      "\n",
      "Seq is: TACCAAGAA\n",
      "0 TACCAAGAA motifs detected in file\n",
      "\n",
      "Complement found from 9432 to 9441\n",
      "Seq is: AAAAAGGCC\n",
      "1 AAAAAGGCC motifs detected in file\n",
      "\n",
      "Seq is: TCATCATTC\n",
      "0 TCATCATTC motifs detected in file\n",
      "\n",
      "Seq is: AATGTTATA\n",
      "0 AATGTTATA motifs detected in file\n",
      "\n",
      "Seq is: CCCTCTAAT\n",
      "0 CCCTCTAAT motifs detected in file\n",
      "\n",
      "Seq is: AAGGCTTCG\n",
      "0 AAGGCTTCG motifs detected in file\n",
      "\n",
      "Seq is: GACCATCCT\n",
      "0 GACCATCCT motifs detected in file\n",
      "\n",
      "Seq is: ATTCTTATA\n",
      "0 ATTCTTATA motifs detected in file\n",
      "\n",
      "Seq is: TTTGAAGTC\n",
      "0 TTTGAAGTC motifs detected in file\n",
      "\n",
      "Seq is: CCACTGGCT\n",
      "0 CCACTGGCT motifs detected in file\n",
      "\n",
      "Seq is: TTTTAGTCT\n",
      "0 TTTTAGTCT motifs detected in file\n",
      "\n",
      "Seq is: TATCCACAA\n",
      "0 TATCCACAA motifs detected in file\n",
      "\n",
      "Seq is: CCATATCTT\n",
      "0 CCATATCTT motifs detected in file\n",
      "\n",
      "Seq is: ACCCCAGAG\n",
      "0 ACCCCAGAG motifs detected in file\n",
      "\n",
      "Seq is: GAGGAGGCC\n",
      "0 GAGGAGGCC motifs detected in file\n",
      "\n",
      "Seq is: GCCCCAGCT\n",
      "0 GCCCCAGCT motifs detected in file\n",
      "\n",
      "Seq is: TCTTCCACC\n",
      "0 TCTTCCACC motifs detected in file\n",
      "\n",
      "Seq is: ACAACTCCA\n",
      "0 ACAACTCCA motifs detected in file\n",
      "\n",
      "Seq is: ACGCCAGAC\n",
      "0 ACGCCAGAC motifs detected in file\n",
      "\n",
      "Complement found from 10339 to 10348\n",
      "Seq is: AATCATCAT\n",
      "1 AATCATCAT motifs detected in file\n",
      "\n",
      "Seq is: ATCACTACG\n",
      "0 ATCACTACG motifs detected in file\n",
      "\n",
      "Seq is: GTCGTCGAT\n",
      "0 GTCGTCGAT motifs detected in file\n",
      "\n",
      "Seq is: CCTGACCCT\n",
      "0 CCTGACCCT motifs detected in file\n",
      "\n",
      "Seq is: CTCTATCCT\n",
      "0 CTCTATCCT motifs detected in file\n",
      "\n",
      "Seq is: CTTCATCCT\n",
      "0 CTTCATCCT motifs detected in file\n",
      "\n",
      "Seq is: GACGACACT\n",
      "0 GACGACACT motifs detected in file\n",
      "\n",
      "Seq is: AATCCTGCC\n",
      "0 AATCCTGCC motifs detected in file\n",
      "\n",
      "Seq is: TAGTCTGCT\n",
      "0 TAGTCTGCT motifs detected in file\n",
      "\n",
      "Seq is: TCTCCCCGC\n",
      "0 TCTCCCCGC motifs detected in file\n",
      "\n",
      "Seq is: AAACCATAA\n",
      "0 AAACCATAA motifs detected in file\n",
      "\n",
      "Complement found from 6432 to 6441\n",
      "Seq is: CCCAATACC\n",
      "1 CCCAATACC motifs detected in file\n",
      "\n",
      "Seq is: GTCCCCCAA\n",
      "0 GTCCCCCAA motifs detected in file\n",
      "\n",
      "Seq is: AATATAACT\n",
      "0 AATATAACT motifs detected in file\n",
      "\n",
      "Complement found from 12025 to 12034\n",
      "Seq is: ATTAACAAC\n",
      "1 ATTAACAAC motifs detected in file\n",
      "\n",
      "Seq is: ACTACTTTA\n",
      "0 ACTACTTTA motifs detected in file\n",
      "\n",
      "Seq is: ACTACCGGG\n",
      "0 ACTACCGGG motifs detected in file\n",
      "\n",
      "Seq is: GATTCTATC\n",
      "0 GATTCTATC motifs detected in file\n",
      "\n",
      "Seq is: TCCTCTGTG\n",
      "0 TCCTCTGTG motifs detected in file\n",
      "\n",
      "Seq is: GACGATCCA\n",
      "0 GACGATCCA motifs detected in file\n",
      "\n",
      "Seq is: CATTCCTCT\n",
      "0 CATTCCTCT motifs detected in file\n",
      "\n",
      "Seq is: TCTACCAAT\n",
      "0 TCTACCAAT motifs detected in file\n",
      "\n",
      "Seq is: CCAGATGCC\n",
      "0 CCAGATGCC motifs detected in file\n",
      "\n",
      "Seq is: TCCGAGGTC\n",
      "0 TCCGAGGTC motifs detected in file\n",
      "\n",
      "Complement found from 15263 to 15272\n",
      "Seq is: CCACCCTCA\n",
      "1 CCACCCTCA motifs detected in file\n",
      "\n",
      "Seq is: TCAAGGGAC\n",
      "0 TCAAGGGAC motifs detected in file\n",
      "\n",
      "Seq is: GATTCCCTC\n",
      "0 GATTCCCTC motifs detected in file\n",
      "\n",
      "Seq is: CCATCTGAC\n",
      "0 CCATCTGAC motifs detected in file\n",
      "\n",
      "Seq is: AAGTTGGAC\n",
      "0 AAGTTGGAC motifs detected in file\n",
      "\n",
      "Seq is: AAGGACGAG\n",
      "0 AAGGACGAG motifs detected in file\n",
      "\n",
      "Seq is: GCCGGAGGT\n",
      "0 GCCGGAGGT motifs detected in file\n",
      "\n",
      "Seq is: GATATCGTC\n",
      "0 GATATCGTC motifs detected in file\n",
      "\n",
      "Seq is: TACGCTCGT\n",
      "0 TACGCTCGT motifs detected in file\n",
      "\n",
      "Complement found from 143 to 152\n",
      "Complement found from 15639 to 15648\n",
      "Seq is: CCTCATCCT\n",
      "2 CCTCATCCT motifs detected in file\n",
      "\n",
      "Complement found from 4903 to 4912\n",
      "Seq is: CTCTCCCTC\n",
      "1 CTCTCCCTC motifs detected in file\n",
      "\n",
      "Seq is: CATTCTCAG\n",
      "0 CATTCTCAG motifs detected in file\n",
      "\n",
      "Complement found from 5785 to 5794\n",
      "Seq is: TCTTCGAAT\n",
      "1 TCTTCGAAT motifs detected in file\n",
      "\n",
      "Complement found from 12929 to 12938\n",
      "Seq is: ACAACAAAT\n",
      "1 ACAACAAAT motifs detected in file\n",
      "\n",
      "Seq is: ACGCCCCTT\n",
      "0 ACGCCCCTT motifs detected in file\n",
      "\n",
      "Seq is: TGCGGTATA\n",
      "0 TGCGGTATA motifs detected in file\n",
      "\n",
      "Seq is: GCCCCCGTG\n",
      "0 GCCCCCGTG motifs detected in file\n",
      "\n",
      "Seq is: GCTAATAAT\n",
      "0 GCTAATAAT motifs detected in file\n",
      "\n",
      "Seq is: CCCCTTGAT\n",
      "0 CCCCTTGAT motifs detected in file\n",
      "\n",
      "Seq is: CAGTCAACG\n",
      "0 CAGTCAACG motifs detected in file\n",
      "\n",
      "Seq is: GTTTCGGAG\n",
      "0 GTTTCGGAG motifs detected in file\n",
      "\n",
      "Seq is: GCTAATACT\n",
      "0 GCTAATACT motifs detected in file\n",
      "\n",
      "Seq is: ACCCATAAT\n",
      "0 ACCCATAAT motifs detected in file\n",
      "\n",
      "Seq is: GATACTTCT\n",
      "0 GATACTTCT motifs detected in file\n",
      "\n",
      "Seq is: TCTAATAAT\n",
      "0 TCTAATAAT motifs detected in file\n",
      "\n",
      "Seq is: GTTTACGTA\n",
      "0 GTTTACGTA motifs detected in file\n",
      "\n",
      "Seq is: CCCGACACT\n",
      "0 CCCGACACT motifs detected in file\n",
      "\n",
      "Seq is: GCTATTGCA\n",
      "0 GCTATTGCA motifs detected in file\n",
      "\n",
      "Complement found from 6057 to 6066\n",
      "Seq is: ACATCTACA\n",
      "1 ACATCTACA motifs detected in file\n",
      "\n",
      "Seq is: CCAGCAATG\n",
      "0 CCAGCAATG motifs detected in file\n",
      "\n",
      "Seq is: GATCTTCCA\n",
      "0 GATCTTCCA motifs detected in file\n",
      "\n",
      "Seq is: ACGGACCGA\n",
      "0 ACGGACCGA motifs detected in file\n",
      "\n",
      "Seq is: CCGGGTCGA\n",
      "0 CCGGGTCGA motifs detected in file\n",
      "\n",
      "Seq is: GCCGAGCTT\n",
      "0 GCCGAGCTT motifs detected in file\n",
      "\n",
      "Seq is: ATTCCTCCG\n",
      "0 ATTCCTCCG motifs detected in file\n",
      "\n",
      "Seq is: AATCTCGAC\n",
      "0 AATCTCGAC motifs detected in file\n",
      "\n",
      "Seq is: ACGGATCCT\n",
      "0 ACGGATCCT motifs detected in file\n",
      "\n",
      "Seq is: GAGGTCGAG\n",
      "0 GAGGTCGAG motifs detected in file\n",
      "\n",
      "Seq is: TACGCGGCT\n",
      "0 TACGCGGCT motifs detected in file\n",
      "\n",
      "Seq is: TATTATCCA\n",
      "0 TATTATCCA motifs detected in file\n",
      "\n",
      "Seq is: TATCACAAG\n",
      "0 TATCACAAG motifs detected in file\n",
      "\n",
      "Seq is: GTTACAGAA\n",
      "0 GTTACAGAA motifs detected in file\n",
      "\n",
      "Seq is: ACACCAAAC\n",
      "0 ACACCAAAC motifs detected in file\n",
      "\n",
      "Complement found from 8729 to 8738\n",
      "Seq is: ATCTCTTAT\n",
      "1 ATCTCTTAT motifs detected in file\n",
      "\n",
      "Seq is: CAGTTGCCA\n",
      "0 CAGTTGCCA motifs detected in file\n",
      "\n",
      "Seq is: GCCGCTTGT\n",
      "0 GCCGCTTGT motifs detected in file\n",
      "\n",
      "Complement found from 427 to 436\n",
      "Seq is: AGTCACCCC\n",
      "1 AGTCACCCC motifs detected in file\n",
      "\n",
      "Seq is: CACTCCATT\n",
      "0 CACTCCATT motifs detected in file\n",
      "\n",
      "Seq is: TTACCGACT\n",
      "0 TTACCGACT motifs detected in file\n",
      "\n",
      "Seq is: CACTTCGTA\n",
      "0 CACTTCGTA motifs detected in file\n",
      "\n",
      "Seq is: ACCTGACAT\n",
      "0 ACCTGACAT motifs detected in file\n",
      "\n",
      "Seq is: TTAGATTTC\n",
      "0 TTAGATTTC motifs detected in file\n",
      "\n",
      "Seq is: TGTCCCCAA\n",
      "0 TGTCCCCAA motifs detected in file\n",
      "\n",
      "Seq is: TCCGGAGAA\n",
      "0 TCCGGAGAA motifs detected in file\n",
      "\n",
      "Seq is: AAATGGTCG\n",
      "0 AAATGGTCG motifs detected in file\n",
      "\n",
      "Seq is: AGGCTCCAC\n",
      "0 AGGCTCCAC motifs detected in file\n",
      "\n",
      "Seq is: TAAAAGTAT\n",
      "0 TAAAAGTAT motifs detected in file\n",
      "\n",
      "Seq is: AACTTAACG\n",
      "0 AACTTAACG motifs detected in file\n",
      "\n",
      "Seq is: TTTAAGCTT\n",
      "0 TTTAAGCTT motifs detected in file\n",
      "\n",
      "Seq is: CTTCGTCGA\n",
      "0 CTTCGTCGA motifs detected in file\n",
      "\n",
      "Seq is: AGTTTGGAC\n",
      "0 AGTTTGGAC motifs detected in file\n",
      "\n",
      "Seq is: GGCCCCGAA\n",
      "0 GGCCCCGAA motifs detected in file\n",
      "\n",
      "Seq is: GAGGGCGGA\n",
      "0 GAGGGCGGA motifs detected in file\n",
      "\n",
      "Seq is: AAAAAGGGC\n",
      "0 AAAAAGGGC motifs detected in file\n",
      "\n",
      "Seq is: CGCCGCCCT\n",
      "0 CGCCGCCCT motifs detected in file\n",
      "\n",
      "Seq is: CTTCATCTA\n",
      "0 CTTCATCTA motifs detected in file\n",
      "\n",
      "Seq is: ACTTCGGTC\n",
      "0 ACTTCGGTC motifs detected in file\n",
      "\n",
      "Seq is: AACTAATCC\n",
      "0 AACTAATCC motifs detected in file\n",
      "\n",
      "Seq is: CACGAATCG\n",
      "0 CACGAATCG motifs detected in file\n",
      "\n",
      "Seq is: ACAATTGAT\n",
      "0 ACAATTGAT motifs detected in file\n",
      "\n",
      "Seq is: TCACAAACA\n",
      "0 TCACAAACA motifs detected in file\n",
      "\n",
      "Seq is: CCCAAATTC\n",
      "0 CCCAAATTC motifs detected in file\n",
      "\n",
      "Seq is: AGGGTAACC\n",
      "0 AGGGTAACC motifs detected in file\n",
      "\n",
      "Seq is: AGATCATTC\n",
      "0 AGATCATTC motifs detected in file\n",
      "\n",
      "Seq is: CCGAATCGA\n",
      "0 CCGAATCGA motifs detected in file\n",
      "\n",
      "Seq is: ATTAATTTC\n",
      "0 ATTAATTTC motifs detected in file\n",
      "\n",
      "Complement found from 8644 to 8653\n",
      "Seq is: ACCGACTAA\n",
      "1 ACCGACTAA motifs detected in file\n",
      "\n",
      "Seq is: ACGCAAGTC\n",
      "0 ACGCAAGTC motifs detected in file\n",
      "\n",
      "Seq is: AACTACGTC\n",
      "0 AACTACGTC motifs detected in file\n",
      "\n",
      "Seq is: TCACCCCAA\n",
      "0 TCACCCCAA motifs detected in file\n",
      "\n",
      "Seq is: AACGTCAGG\n",
      "0 AACGTCAGG motifs detected in file\n",
      "\n",
      "Seq is: AATCGACAA\n",
      "0 AATCGACAA motifs detected in file\n",
      "\n",
      "Seq is: TGTCTTTAA\n",
      "0 TGTCTTTAA motifs detected in file\n",
      "\n",
      "Seq is: TTCATAACG\n",
      "0 TTCATAACG motifs detected in file\n",
      "\n",
      "Seq is: TTGAATGAC\n",
      "0 TTGAATGAC motifs detected in file\n",
      "\n",
      "Seq is: TCCCGAAAC\n",
      "0 TCCCGAAAC motifs detected in file\n",
      "\n",
      "Seq is: TTCCGAGAA\n",
      "0 TTCCGAGAA motifs detected in file\n",
      "\n",
      "Seq is: CCAGACATA\n",
      "0 CCAGACATA motifs detected in file\n",
      "\n",
      "Seq is: AATTGGATT\n",
      "0 AATTGGATT motifs detected in file\n",
      "\n",
      "Seq is: TAAAGATAT\n",
      "0 TAAAGATAT motifs detected in file\n",
      "\n",
      "Seq is: TCTAATAAT\n",
      "0 TCTAATAAT motifs detected in file\n",
      "\n",
      "Seq is: CATATTTTC\n",
      "0 CATATTTTC motifs detected in file\n",
      "\n",
      "Seq is: CCCTCTATC\n",
      "0 CCCTCTATC motifs detected in file\n",
      "\n",
      "Complement found from 11723 to 11732\n",
      "Complement found from 15636 to 15645\n",
      "Seq is: CATCCTCAT\n",
      "2 CATCCTCAT motifs detected in file\n",
      "\n",
      "Seq is: CGCACCATT\n",
      "0 CGCACCATT motifs detected in file\n",
      "\n",
      "Seq is: CCCGCTACT\n",
      "0 CCCGCTACT motifs detected in file\n",
      "\n",
      "Seq is: CACACCCCT\n",
      "0 CACACCCCT motifs detected in file\n",
      "\n",
      "Complement found from 11861 to 11870\n",
      "Seq is: CCTTACCCC\n",
      "1 CCTTACCCC motifs detected in file\n",
      "\n",
      "Seq is: ACCCAAAAC\n",
      "0 ACCCAAAAC motifs detected in file\n",
      "\n",
      "Seq is: ATACAAGTT\n",
      "0 ATACAAGTT motifs detected in file\n",
      "\n",
      "Seq is: TGACAGTAA\n",
      "0 TGACAGTAA motifs detected in file\n",
      "\n",
      "Seq is: AATAAAAAT\n",
      "0 AATAAAAAT motifs detected in file\n",
      "\n",
      "Seq is: GCAACAATC\n",
      "0 GCAACAATC motifs detected in file\n",
      "\n",
      "Seq is: TATACCCCT\n",
      "0 TATACCCCT motifs detected in file\n",
      "\n",
      "Seq is: CATCACACT\n",
      "0 CATCACACT motifs detected in file\n",
      "\n",
      "Complement found from 15972 to 15981\n",
      "Seq is: AACTCCACC\n",
      "1 AACTCCACC motifs detected in file\n",
      "\n",
      "Seq is: TCATCTAAT\n",
      "0 TCATCTAAT motifs detected in file\n",
      "\n",
      "Complement found from 4662 to 4671\n",
      "Seq is: CCGCATCCA\n",
      "1 CCGCATCCA motifs detected in file\n",
      "\n",
      "Seq is: TCTTCATCT\n",
      "0 TCTTCATCT motifs detected in file\n",
      "\n",
      "Seq is: CCAATTCCT\n",
      "0 CCAATTCCT motifs detected in file\n",
      "\n",
      "Complement found from 2216 to 2225\n",
      "Seq is: CCCACTACC\n",
      "1 CCCACTACC motifs detected in file\n",
      "\n",
      "Seq is: ACCGATACT\n",
      "0 ACCGATACT motifs detected in file\n",
      "\n",
      "Seq is: ACCACCCCT\n",
      "0 ACCACCCCT motifs detected in file\n",
      "\n",
      "Complement found from 6681 to 6690\n",
      "Seq is: ACTACTCCG\n",
      "1 ACTACTCCG motifs detected in file\n",
      "\n",
      "Complement found from 280 to 289\n",
      "Seq is: ATAACAAAA\n",
      "1 ATAACAAAA motifs detected in file\n",
      "\n",
      "Seq is: AACACTTAA\n",
      "0 AACACTTAA motifs detected in file\n",
      "\n",
      "Seq is: GAAGCTATT\n",
      "0 GAAGCTATT motifs detected in file\n",
      "\n",
      "Seq is: ACCGGGTAA\n",
      "0 ACCGGGTAA motifs detected in file\n",
      "\n",
      "Seq is: ACCCGTTTT\n",
      "0 ACCCGTTTT motifs detected in file\n",
      "\n",
      "Seq is: TCGGCCAAT\n",
      "0 TCGGCCAAT motifs detected in file\n",
      "\n",
      "Seq is: CGCCCCCGT\n",
      "0 CGCCCCCGT motifs detected in file\n",
      "\n",
      "Seq is: CCGGAGGAT\n",
      "0 CCGGAGGAT motifs detected in file\n",
      "\n",
      "Complement found from 14281 to 14290\n",
      "Seq is: CCCTCTCCT\n",
      "1 CCCTCTCCT motifs detected in file\n",
      "\n",
      "Seq is: CCCACCTAC\n",
      "0 CCCACCTAC motifs detected in file\n",
      "\n",
      "Complement found from 5194 to 5203\n",
      "Seq is: CTTAATTCC\n",
      "1 CTTAATTCC motifs detected in file\n",
      "\n",
      "Seq is: CACAATCAG\n",
      "0 CACAATCAG motifs detected in file\n",
      "\n",
      "Seq is: TACAATCGA\n",
      "0 TACAATCGA motifs detected in file\n",
      "\n",
      "Seq is: ACAAAGTCC\n",
      "0 ACAAAGTCC motifs detected in file\n",
      "\n",
      "Seq is: ACGCTCTAT\n",
      "0 ACGCTCTAT motifs detected in file\n",
      "\n",
      "Complement found from 5296 to 5305\n",
      "Complement found from 10863 to 10872\n",
      "Seq is: CATCATCCC\n",
      "2 CATCATCCC motifs detected in file\n",
      "\n",
      "Complement found from 5138 to 5147\n",
      "Seq is: AGCACCACG\n",
      "1 AGCACCACG motifs detected in file\n",
      "\n",
      "Seq is: ACCTCAAAT\n",
      "0 ACCTCAAAT motifs detected in file\n",
      "\n",
      "Seq is: TCAACTCAT\n",
      "0 TCAACTCAT motifs detected in file\n",
      "\n",
      "Complement found from 12389 to 12398\n",
      "Seq is: CATCCTTAC\n",
      "1 CATCCTTAC motifs detected in file\n",
      "\n",
      "Complement found from 7792 to 7801\n",
      "Seq is: GCCATCATC\n",
      "1 GCCATCATC motifs detected in file\n",
      "\n",
      "Complement found from 2944 to 2953\n",
      "Complement found from 9454 to 9463\n",
      "Seq is: AATCCTATT\n",
      "2 AATCCTATT motifs detected in file\n",
      "\n",
      "Seq is: ATATTTATC\n",
      "0 ATATTTATC motifs detected in file\n",
      "\n",
      "Seq is: AATTTAATT\n",
      "0 AATTTAATT motifs detected in file\n",
      "\n",
      "Seq is: CTTACCAAT\n",
      "0 CTTACCAAT motifs detected in file\n",
      "\n",
      "Seq is: ACAATCCCA\n",
      "0 ACAATCCCA motifs detected in file\n",
      "\n",
      "Seq is: ACATGCCAT\n",
      "0 ACATGCCAT motifs detected in file\n",
      "\n",
      "Seq is: CTTGACGAT\n",
      "0 CTTGACGAT motifs detected in file\n",
      "\n",
      "Seq is: AATAAGTAG\n",
      "0 AATAAGTAG motifs detected in file\n",
      "\n",
      "Seq is: GATACACCC\n",
      "0 GATACACCC motifs detected in file\n",
      "\n",
      "Seq is: ATTAACTCC\n",
      "0 ATTAACTCC motifs detected in file\n",
      "\n",
      "Seq is: TCATACGAT\n",
      "0 TCATACGAT motifs detected in file\n",
      "\n",
      "Seq is: TCTAAAACG\n",
      "0 TCTAAAACG motifs detected in file\n",
      "\n",
      "Seq is: CATCGACCC\n",
      "0 CATCGACCC motifs detected in file\n",
      "\n",
      "Complement found from 9673 to 9682\n",
      "Seq is: AAACCAAAT\n",
      "1 AAACCAAAT motifs detected in file\n",
      "\n",
      "Seq is: TAGGTGGAG\n",
      "0 TAGGTGGAG motifs detected in file\n",
      "\n",
      "Seq is: TTGACGGAC\n",
      "0 TTGACGGAC motifs detected in file\n",
      "\n",
      "Seq is: GATACTACC\n",
      "0 GATACTACC motifs detected in file\n",
      "\n",
      "Seq is: TATTCTAAC\n",
      "0 TATTCTAAC motifs detected in file\n",
      "\n",
      "Seq is: TCTCTCACT\n",
      "0 TCTCTCACT motifs detected in file\n",
      "\n",
      "Seq is: CCTCTTCCG\n",
      "0 CCTCTTCCG motifs detected in file\n",
      "\n",
      "Seq is: AATGCAAAT\n",
      "0 AATGCAAAT motifs detected in file\n",
      "\n",
      "Complement found from 10622 to 10631\n",
      "Seq is: CACTCCCTC\n",
      "1 CACTCCCTC motifs detected in file\n",
      "\n",
      "Complement found from 8118 to 8127\n",
      "Seq is: TCTAAACCA\n",
      "1 TCTAAACCA motifs detected in file\n",
      "\n",
      "Complement found from 858 to 867\n",
      "Seq is: TATACTAAC\n",
      "1 TATACTAAC motifs detected in file\n",
      "\n",
      "Complement found from 8276 to 8285\n",
      "Seq is: TCTACCCCC\n",
      "1 TCTACCCCC motifs detected in file\n",
      "\n",
      "Seq is: GATCAAAAA\n",
      "0 GATCAAAAA motifs detected in file\n",
      "\n",
      "Seq is: CAGTACACT\n",
      "0 CAGTACACT motifs detected in file\n",
      "\n",
      "Seq is: CTTCTTCGT\n",
      "0 CTTCTTCGT motifs detected in file\n",
      "\n",
      "Seq is: CCGGCCTAC\n",
      "0 CCGGCCTAC motifs detected in file\n",
      "\n",
      "Seq is: AGTCTCCCC\n",
      "0 AGTCTCCCC motifs detected in file\n",
      "\n",
      "Seq is: ACGGAACCC\n",
      "0 ACGGAACCC motifs detected in file\n",
      "\n",
      "Seq is: ATTGGAGAC\n",
      "0 ATTGGAGAC motifs detected in file\n",
      "\n",
      "Seq is: CCTGAGTCT\n",
      "0 CCTGAGTCT motifs detected in file\n",
      "\n",
      "Complement found from 13813 to 13822\n",
      "Seq is: TCACTTTCC\n",
      "1 TCACTTTCC motifs detected in file\n",
      "\n",
      "Seq is: CCCGATAAG\n",
      "0 CCCGATAAG motifs detected in file\n",
      "\n",
      "Seq is: GATCAAAAT\n",
      "0 GATCAAAAT motifs detected in file\n",
      "\n",
      "Seq is: AACGATATC\n",
      "0 AACGATATC motifs detected in file\n",
      "\n",
      "Seq is: GATAATACT\n",
      "0 GATAATACT motifs detected in file\n",
      "\n",
      "Complement found from 8040 to 8049\n",
      "Seq is: AATAATTAC\n",
      "1 AATAATTAC motifs detected in file\n",
      "\n",
      "Seq is: TACTCATAA\n",
      "0 TACTCATAA motifs detected in file\n",
      "\n",
      "Complement found from 6335 to 6344\n",
      "Seq is: CTAACCATC\n",
      "1 CTAACCATC motifs detected in file\n",
      "\n",
      "Complement found from 4724 to 4733\n",
      "Seq is: ATAACCAAT\n",
      "1 ATAACCAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCAAGTAA\n",
      "0 ACCAAGTAA motifs detected in file\n",
      "\n",
      "Seq is: CAGGCCTCT\n",
      "0 CAGGCCTCT motifs detected in file\n",
      "\n",
      "Seq is: CATATAACA\n",
      "0 CATATAACA motifs detected in file\n",
      "\n",
      "Complement found from 6484 to 6493\n",
      "Seq is: ACTTCTCCT\n",
      "1 ACTTCTCCT motifs detected in file\n",
      "\n",
      "Seq is: ATCGATAAT\n",
      "0 ATCGATAAT motifs detected in file\n",
      "\n",
      "Complement found from 12704 to 12713\n",
      "Seq is: CTTCCTAAT\n",
      "1 CTTCCTAAT motifs detected in file\n",
      "\n",
      "Complement found from 5346 to 5355\n",
      "Complement found from 7124 to 7133\n",
      "Seq is: ACCTACGCC\n",
      "2 ACCTACGCC motifs detected in file\n",
      "\n",
      "Seq is: AACGAACGC\n",
      "0 AACGAACGC motifs detected in file\n",
      "\n",
      "Seq is: ACTCCTTTA\n",
      "0 ACTCCTTTA motifs detected in file\n",
      "\n",
      "Seq is: TGAACTACC\n",
      "0 TGAACTACC motifs detected in file\n",
      "\n",
      "Seq is: GTCGAAGAC\n",
      "0 GTCGAAGAC motifs detected in file\n",
      "\n",
      "Seq is: ACCTTGCTC\n",
      "0 ACCTTGCTC motifs detected in file\n",
      "\n",
      "Complement found from 1732 to 1741\n",
      "Complement found from 12323 to 12332\n",
      "Seq is: CCAAATAAA\n",
      "2 CCAAATAAA motifs detected in file\n",
      "\n",
      "Seq is: AAAACCAAT\n",
      "0 AAAACCAAT motifs detected in file\n",
      "\n",
      "Seq is: CTTGACCTT\n",
      "0 CTTGACCTT motifs detected in file\n",
      "\n",
      "Seq is: ATTTTCGAT\n",
      "0 ATTTTCGAT motifs detected in file\n",
      "\n",
      "Seq is: CGTACAAAT\n",
      "0 CGTACAAAT motifs detected in file\n",
      "\n",
      "Seq is: AAAGATCCG\n",
      "0 AAAGATCCG motifs detected in file\n",
      "\n",
      "Seq is: GATGAGTCC\n",
      "0 GATGAGTCC motifs detected in file\n",
      "\n",
      "Seq is: ATTTTTTAG\n",
      "0 ATTTTTTAG motifs detected in file\n",
      "\n",
      "Seq is: TCACGCTCG\n",
      "0 TCACGCTCG motifs detected in file\n",
      "\n",
      "Seq is: AATCGCGAC\n",
      "0 AATCGCGAC motifs detected in file\n",
      "\n",
      "Complement found from 11289 to 11298\n",
      "Complement found from 14906 to 14915\n",
      "Seq is: ACTACTCAC\n",
      "2 ACTACTCAC motifs detected in file\n",
      "\n",
      "Seq is: ACGGACGTT\n",
      "0 ACGGACGTT motifs detected in file\n",
      "\n",
      "Complement found from 3501 to 3510\n",
      "Complement found from 4499 to 4508\n",
      "Seq is: TCTACCATC\n",
      "2 TCTACCATC motifs detected in file\n",
      "\n",
      "Complement found from 4491 to 4500\n",
      "Seq is: TCATCTACT\n",
      "1 TCATCTACT motifs detected in file\n",
      "\n",
      "Complement found from 4481 to 4490\n",
      "Seq is: GCCCAACCC\n",
      "1 GCCCAACCC motifs detected in file\n",
      "\n",
      "Seq is: GGTCCCCTA\n",
      "0 GGTCCCCTA motifs detected in file\n",
      "\n",
      "Complement found from 10336 to 10345\n",
      "Seq is: ATTAATCAT\n",
      "1 ATTAATCAT motifs detected in file\n",
      "\n",
      "Seq is: GCCCTTCCC\n",
      "0 GCCCTTCCC motifs detected in file\n",
      "\n",
      "Seq is: ATATTGGTT\n",
      "0 ATATTGGTT motifs detected in file\n",
      "\n",
      "Seq is: GTAAAAGCC\n",
      "0 GTAAAAGCC motifs detected in file\n",
      "\n",
      "Seq is: CCATACCCG\n",
      "0 CCATACCCG motifs detected in file\n",
      "\n",
      "Seq is: GGCTATCGA\n",
      "0 GGCTATCGA motifs detected in file\n",
      "\n",
      "Seq is: ATAAATCGA\n",
      "0 ATAAATCGA motifs detected in file\n",
      "\n",
      "Seq is: CTGGAATGA\n",
      "0 CTGGAATGA motifs detected in file\n",
      "\n",
      "Complement found from 14368 to 14377\n",
      "Seq is: AATCCTACC\n",
      "1 AATCCTACC motifs detected in file\n",
      "\n",
      "Seq is: CCACACTAT\n",
      "0 CCACACTAT motifs detected in file\n",
      "\n",
      "Seq is: CCACCGTGC\n",
      "0 CCACCGTGC motifs detected in file\n",
      "\n",
      "Complement found from 11461 to 11470\n",
      "Seq is: CTCTTAAAA\n",
      "1 CTCTTAAAA motifs detected in file\n",
      "\n",
      "Seq is: CCTAAGAGT\n",
      "0 CCTAAGAGT motifs detected in file\n",
      "\n",
      "Seq is: CCCTACCCA\n",
      "0 CCCTACCCA motifs detected in file\n",
      "\n",
      "Seq is: AGCTAAGAG\n",
      "0 AGCTAAGAG motifs detected in file\n",
      "\n",
      "Seq is: TATCAGGAT\n",
      "0 TATCAGGAT motifs detected in file\n",
      "\n",
      "Seq is: CTTTATTCC\n",
      "0 CTTTATTCC motifs detected in file\n",
      "\n",
      "Seq is: CCCAAATTC\n",
      "0 CCCAAATTC motifs detected in file\n",
      "\n",
      "Seq is: GAGGATAAT\n",
      "0 GAGGATAAT motifs detected in file\n",
      "\n",
      "Seq is: AAATGAGAT\n",
      "0 AAATGAGAT motifs detected in file\n",
      "\n",
      "Seq is: AGTTTCATT\n",
      "0 AGTTTCATT motifs detected in file\n",
      "\n",
      "Seq is: GAGAAAATA\n",
      "0 GAGAAAATA motifs detected in file\n",
      "\n",
      "Seq is: GTCTGTATA\n",
      "0 GTCTGTATA motifs detected in file\n",
      "\n",
      "Seq is: AAGAATCCA\n",
      "0 AAGAATCCA motifs detected in file\n",
      "\n",
      "Seq is: AACTCCCCC\n",
      "0 AACTCCCCC motifs detected in file\n",
      "\n",
      "Complement found from 2978 to 2987\n",
      "Seq is: TTACGACCT\n",
      "1 TTACGACCT motifs detected in file\n",
      "\n",
      "Complement found from 9010 to 9019\n",
      "Complement found from 13657 to 13666\n",
      "Seq is: CTAACATTA\n",
      "2 CTAACATTA motifs detected in file\n",
      "\n",
      "Seq is: CCCATACCT\n",
      "0 CCCATACCT motifs detected in file\n",
      "\n",
      "Seq is: CTGTATAGT\n",
      "0 CTGTATAGT motifs detected in file\n",
      "\n",
      "Seq is: ATATTCATT\n",
      "0 ATATTCATT motifs detected in file\n",
      "\n",
      "Seq is: ACGATCCCA\n",
      "0 ACGATCCCA motifs detected in file\n",
      "\n",
      "Complement found from 3540 to 3549\n",
      "Seq is: CTCACCATC\n",
      "1 CTCACCATC motifs detected in file\n",
      "\n",
      "Seq is: CTTCAAAAA\n",
      "0 CTTCAAAAA motifs detected in file\n",
      "\n",
      "Seq is: AGTATCCTC\n",
      "0 AGTATCCTC motifs detected in file\n",
      "\n",
      "Seq is: CACATACTC\n",
      "0 CACATACTC motifs detected in file\n",
      "\n",
      "Seq is: AACCAGCAT\n",
      "0 AACCAGCAT motifs detected in file\n",
      "\n",
      "Complement found from 12887 to 12896\n",
      "Seq is: CGCCTTAGC\n",
      "1 CGCCTTAGC motifs detected in file\n",
      "\n",
      "Seq is: CCCCATACG\n",
      "0 CCCCATACG motifs detected in file\n",
      "\n",
      "Seq is: ACAAGCTTA\n",
      "0 ACAAGCTTA motifs detected in file\n",
      "\n",
      "Seq is: AGTATTCTT\n",
      "0 AGTATTCTT motifs detected in file\n",
      "\n",
      "Seq is: GTCCCTCCA\n",
      "0 GTCCCTCCA motifs detected in file\n",
      "\n",
      "Seq is: ATCTTCATC\n",
      "0 ATCTTCATC motifs detected in file\n",
      "\n",
      "Seq is: CCAGAACCA\n",
      "0 CCAGAACCA motifs detected in file\n",
      "\n",
      "Seq is: CTGTTTTAT\n",
      "0 CTGTTTTAT motifs detected in file\n",
      "\n",
      "Complement found from 9196 to 9205\n",
      "Seq is: ACAACACAT\n",
      "1 ACAACACAT motifs detected in file\n",
      "\n",
      "Seq is: CTCAAGTCC\n",
      "0 CTCAAGTCC motifs detected in file\n",
      "\n",
      "Seq is: CCTCTCACG\n",
      "0 CCTCTCACG motifs detected in file\n",
      "\n",
      "Seq is: CAGTATACA\n",
      "0 CAGTATACA motifs detected in file\n",
      "\n",
      "Seq is: ACAAGGATC\n",
      "0 ACAAGGATC motifs detected in file\n",
      "\n",
      "Complement found from 13827 to 13836\n",
      "Seq is: CTTCTAACA\n",
      "1 CTTCTAACA motifs detected in file\n",
      "\n",
      "Complement found from 13183 to 13192\n",
      "Seq is: TCACCACTC\n",
      "1 TCACCACTC motifs detected in file\n",
      "\n",
      "Seq is: CCACAAATA\n",
      "0 CCACAAATA motifs detected in file\n",
      "\n",
      "Seq is: ATATTATTA\n",
      "0 ATATTATTA motifs detected in file\n",
      "\n",
      "Seq is: CAAACACAT\n",
      "0 CAAACACAT motifs detected in file\n",
      "\n",
      "Seq is: AAGCCGATA\n",
      "0 AAGCCGATA motifs detected in file\n",
      "\n",
      "Seq is: CTTCTTATC\n",
      "0 CTTCTTATC motifs detected in file\n",
      "\n",
      "Seq is: CCGCTTCCC\n",
      "0 CCGCTTCCC motifs detected in file\n",
      "\n",
      "Seq is: CGGACGCCG\n",
      "0 CGGACGCCG motifs detected in file\n",
      "\n",
      "Seq is: CATAAGCTA\n",
      "0 CATAAGCTA motifs detected in file\n",
      "\n",
      "Seq is: CAACTTCGG\n",
      "0 CAACTTCGG motifs detected in file\n",
      "\n",
      "Seq is: ACTCTGATC\n",
      "0 ACTCTGATC motifs detected in file\n",
      "\n",
      "Seq is: AAGCCTGAG\n",
      "0 AAGCCTGAG motifs detected in file\n",
      "\n",
      "Seq is: GGGAAGCCG\n",
      "0 GGGAAGCCG motifs detected in file\n",
      "\n",
      "Seq is: TTCCAGCTT\n",
      "0 TTCCAGCTT motifs detected in file\n",
      "\n",
      "Seq is: CCCCCAAGC\n",
      "0 CCCCCAAGC motifs detected in file\n",
      "\n",
      "Seq is: CAACCAGAG\n",
      "0 CAACCAGAG motifs detected in file\n",
      "\n",
      "Seq is: ACGATCACA\n",
      "0 ACGATCACA motifs detected in file\n",
      "\n",
      "Seq is: CCTCTATTT\n",
      "0 CCTCTATTT motifs detected in file\n",
      "\n",
      "Seq is: AGTATAATA\n",
      "0 AGTATAATA motifs detected in file\n",
      "\n",
      "Seq is: CCGGTTCCC\n",
      "0 CCGGTTCCC motifs detected in file\n",
      "\n",
      "Seq is: AGTACTACC\n",
      "0 AGTACTACC motifs detected in file\n",
      "\n",
      "Seq is: GTCCTCATT\n",
      "0 GTCCTCATT motifs detected in file\n",
      "\n",
      "Seq is: AGTCTCCAC\n",
      "0 AGTCTCCAC motifs detected in file\n",
      "\n",
      "Seq is: AAGAACACA\n",
      "0 AAGAACACA motifs detected in file\n",
      "\n",
      "Complement found from 8419 to 8428\n",
      "Complement found from 15472 to 15481\n",
      "Seq is: ACACTATTC\n",
      "2 ACACTATTC motifs detected in file\n",
      "\n",
      "Seq is: CCACCTCTC\n",
      "0 CCACCTCTC motifs detected in file\n",
      "\n",
      "Seq is: CAATTTCCT\n",
      "0 CAATTTCCT motifs detected in file\n",
      "\n",
      "Seq is: CGGTGAATA\n",
      "0 CGGTGAATA motifs detected in file\n",
      "\n",
      "Seq is: ATCATTACA\n",
      "0 ATCATTACA motifs detected in file\n",
      "\n",
      "Seq is: ACTATCATC\n",
      "0 ACTATCATC motifs detected in file\n",
      "\n",
      "Seq is: TTACTACCG\n",
      "0 TTACTACCG motifs detected in file\n",
      "\n",
      "Seq is: ATCCCACTG\n",
      "0 ATCCCACTG motifs detected in file\n",
      "\n",
      "Complement found from 1500 to 1509\n",
      "Seq is: AAGTATACT\n",
      "1 AAGTATACT motifs detected in file\n",
      "\n",
      "Complement found from 2761 to 2770\n",
      "Complement found from 15599 to 15608\n",
      "Seq is: CTAACAAAC\n",
      "2 CTAACAAAC motifs detected in file\n",
      "\n",
      "Seq is: CCGATGACG\n",
      "0 CCGATGACG motifs detected in file\n",
      "\n",
      "Seq is: AGCGTCACG\n",
      "0 AGCGTCACG motifs detected in file\n",
      "\n",
      "Seq is: CGGCTAGTC\n",
      "0 CGGCTAGTC motifs detected in file\n",
      "\n",
      "Seq is: CCGCATCAA\n",
      "0 CCGCATCAA motifs detected in file\n",
      "\n",
      "Complement found from 3676 to 3685\n",
      "Complement found from 11750 to 11759\n",
      "Seq is: ACTCAAACT\n",
      "2 ACTCAAACT motifs detected in file\n",
      "\n",
      "Seq is: ACGAGTGGG\n",
      "0 ACGAGTGGG motifs detected in file\n",
      "\n",
      "Complement found from 3923 to 3932\n",
      "Complement found from 10699 to 10708\n",
      "Seq is: ACTAGTCTC\n",
      "2 ACTAGTCTC motifs detected in file\n",
      "\n",
      "Complement found from 12226 to 12235\n",
      "Seq is: CTAACTCAT\n",
      "1 CTAACTCAT motifs detected in file\n",
      "\n",
      "Seq is: TTGCCGATC\n",
      "0 TTGCCGATC motifs detected in file\n",
      "\n",
      "Seq is: CGATCTCCA\n",
      "0 CGATCTCCA motifs detected in file\n",
      "\n",
      "Seq is: CCGATCTTA\n",
      "0 CCGATCTTA motifs detected in file\n",
      "\n",
      "Seq is: TTTATCCTC\n",
      "0 TTTATCCTC motifs detected in file\n",
      "\n",
      "Seq is: CGGATCCAA\n",
      "0 CGGATCCAA motifs detected in file\n",
      "\n",
      "Seq is: CTCCAACTG\n",
      "0 CTCCAACTG motifs detected in file\n",
      "\n",
      "Seq is: GTCCCCCAA\n",
      "0 GTCCCCCAA motifs detected in file\n",
      "\n",
      "Complement found from 534 to 543\n",
      "Complement found from 3573 to 3582\n",
      "Complement found from 4429 to 4438\n",
      "Seq is: CCCATACCC\n",
      "3 CCCATACCC motifs detected in file\n",
      "\n",
      "Seq is: CTCCCCCCA\n",
      "0 CTCCCCCCA motifs detected in file\n",
      "\n",
      "Seq is: AGTATCATC\n",
      "0 AGTATCATC motifs detected in file\n",
      "\n",
      "Seq is: TTCTCGCTA\n",
      "0 TTCTCGCTA motifs detected in file\n",
      "\n",
      "Seq is: CCACTCTCG\n",
      "0 CCACTCTCG motifs detected in file\n",
      "\n",
      "Seq is: ATTCCAGCC\n",
      "0 ATTCCAGCC motifs detected in file\n",
      "\n",
      "Seq is: CCGCCACTA\n",
      "0 CCGCCACTA motifs detected in file\n",
      "\n",
      "Seq is: CATCTCCCA\n",
      "0 CATCTCCCA motifs detected in file\n",
      "\n",
      "Complement found from 4500 to 4509\n",
      "Seq is: CTACCATCT\n",
      "1 CTACCATCT motifs detected in file\n",
      "\n",
      "Complement found from 1479 to 1488\n",
      "Seq is: ACACCGCCC\n",
      "1 ACACCGCCC motifs detected in file\n",
      "\n",
      "Complement found from 13874 to 13883\n",
      "Seq is: AAAATCCCC\n",
      "1 AAAATCCCC motifs detected in file\n",
      "\n",
      "Seq is: GAGAAACCA\n",
      "0 GAGAAACCA motifs detected in file\n",
      "\n",
      "Seq is: CTTCTCAAA\n",
      "0 CTTCTCAAA motifs detected in file\n",
      "\n",
      "Seq is: ATACCGCAG\n",
      "0 ATACCGCAG motifs detected in file\n",
      "\n",
      "Complement found from 13640 to 13649\n",
      "Seq is: TCGCTTCCC\n",
      "1 TCGCTTCCC motifs detected in file\n",
      "\n",
      "Seq is: AACATCATC\n",
      "0 AACATCATC motifs detected in file\n",
      "\n",
      "Seq is: GGGCATCCC\n",
      "0 GGGCATCCC motifs detected in file\n",
      "\n",
      "Seq is: CGGATGTTG\n",
      "0 CGGATGTTG motifs detected in file\n",
      "\n",
      "Seq is: CAACCCCGG\n",
      "0 CAACCCCGG motifs detected in file\n",
      "\n",
      "Seq is: AAACGCATC\n",
      "0 AAACGCATC motifs detected in file\n",
      "\n",
      "Seq is: AACATATAT\n",
      "0 AACATATAT motifs detected in file\n",
      "\n",
      "Seq is: CGGATCTTA\n",
      "0 CGGATCTTA motifs detected in file\n",
      "\n",
      "Seq is: AAAAGCAAG\n",
      "0 AAAAGCAAG motifs detected in file\n",
      "\n",
      "Complement found from 8030 to 8039\n",
      "Seq is: CCATTCGTA\n",
      "1 CCATTCGTA motifs detected in file\n",
      "\n",
      "Seq is: ATCCTTACG\n",
      "0 ATCCTTACG motifs detected in file\n",
      "\n",
      "Seq is: GTAACGCTA\n",
      "0 GTAACGCTA motifs detected in file\n",
      "\n",
      "Complement found from 2481 to 2490\n",
      "Seq is: ATCTTACCC\n",
      "1 ATCTTACCC motifs detected in file\n",
      "\n",
      "Seq is: ATGTTACTC\n",
      "0 ATGTTACTC motifs detected in file\n",
      "\n",
      "Seq is: CTCATCCTC\n",
      "0 CTCATCCTC motifs detected in file\n",
      "\n",
      "Seq is: CAACCGGTA\n",
      "0 CAACCGGTA motifs detected in file\n",
      "\n",
      "Seq is: CCCATACAA\n",
      "0 CCCATACAA motifs detected in file\n",
      "\n",
      "Seq is: CAATTCTTC\n",
      "0 CAATTCTTC motifs detected in file\n",
      "\n",
      "Seq is: TCCTTAACT\n",
      "0 TCCTTAACT motifs detected in file\n",
      "\n",
      "Seq is: TGGAGACTG\n",
      "0 TGGAGACTG motifs detected in file\n",
      "\n",
      "Seq is: ACATTTCAA\n",
      "0 ACATTTCAA motifs detected in file\n",
      "\n",
      "Seq is: AATTCAAAA\n",
      "0 AATTCAAAA motifs detected in file\n",
      "\n",
      "Seq is: TACGCTAAT\n",
      "0 TACGCTAAT motifs detected in file\n",
      "\n",
      "Seq is: GGCCCGAGA\n",
      "0 GGCCCGAGA motifs detected in file\n",
      "\n",
      "Seq is: CGGTAGAAT\n",
      "0 CGGTAGAAT motifs detected in file\n",
      "\n",
      "Seq is: TGTTTGGGA\n",
      "0 TGTTTGGGA motifs detected in file\n",
      "\n",
      "Seq is: CAAGAACCC\n",
      "0 CAAGAACCC motifs detected in file\n",
      "\n",
      "Complement found from 3202 to 3211\n",
      "Seq is: ACCCACACC\n",
      "1 ACCCACACC motifs detected in file\n",
      "\n",
      "Seq is: CATATTATG\n",
      "0 CATATTATG motifs detected in file\n",
      "\n",
      "Seq is: ATTCAACTC\n",
      "0 ATTCAACTC motifs detected in file\n",
      "\n",
      "Seq is: TACTATAGT\n",
      "0 TACTATAGT motifs detected in file\n",
      "\n",
      "Complement found from 8363 to 8372\n",
      "Complement found from 10467 to 10476\n",
      "Seq is: AAATGCCCC\n",
      "2 AAATGCCCC motifs detected in file\n",
      "\n",
      "Seq is: CTTCCGCGA\n",
      "0 CTTCCGCGA motifs detected in file\n",
      "\n",
      "Seq is: AACACTTCA\n",
      "0 AACACTTCA motifs detected in file\n",
      "\n",
      "Seq is: TCCGGAATA\n",
      "0 TCCGGAATA motifs detected in file\n",
      "\n",
      "Seq is: AAGAGAACA\n",
      "0 AAGAGAACA motifs detected in file\n",
      "\n",
      "Seq is: GGAAAGCAT\n",
      "0 GGAAAGCAT motifs detected in file\n",
      "\n",
      "Seq is: GTCCCTCCT\n",
      "0 GTCCCTCCT motifs detected in file\n",
      "\n",
      "Seq is: TAAACTTNC\n",
      "0 TAAACTTNC motifs detected in file\n",
      "\n",
      "Seq is: ATCTATCTT\n",
      "0 ATCTATCTT motifs detected in file\n",
      "\n",
      "Seq is: TGGCTGGAC\n",
      "0 TGGCTGGAC motifs detected in file\n",
      "\n",
      "Seq is: CTAATGAGG\n",
      "0 CTAATGAGG motifs detected in file\n",
      "\n",
      "Seq is: CCAGACTTG\n",
      "0 CCAGACTTG motifs detected in file\n",
      "\n",
      "Seq is: AGTCTAGTG\n",
      "0 AGTCTAGTG motifs detected in file\n",
      "\n",
      "Seq is: CATCCTGAA\n",
      "0 CATCCTGAA motifs detected in file\n",
      "\n",
      "Seq is: ATTAGCAAC\n",
      "0 ATTAGCAAC motifs detected in file\n",
      "\n",
      "Seq is: TTGTTTGCT\n",
      "0 TTGTTTGCT motifs detected in file\n",
      "\n",
      "Seq is: TGGAAATTA\n",
      "0 TGGAAATTA motifs detected in file\n",
      "\n",
      "Seq is: TCGCCGACG\n",
      "0 TCGCCGACG motifs detected in file\n",
      "\n",
      "Seq is: TGGTAGCCC\n",
      "0 TGGTAGCCC motifs detected in file\n",
      "\n",
      "Complement found from 11943 to 11952\n",
      "Seq is: TACAGGACT\n",
      "1 TACAGGACT motifs detected in file\n",
      "\n",
      "Seq is: AGGTTGTAG\n",
      "0 AGGTTGTAG motifs detected in file\n",
      "\n",
      "Complement found from 4238 to 4247\n",
      "Seq is: CTCCAGCAT\n",
      "1 CTCCAGCAT motifs detected in file\n",
      "\n",
      "Seq is: TTGGGATAA\n",
      "0 TTGGGATAA motifs detected in file\n",
      "\n",
      "Seq is: CAACTATAC\n",
      "0 CAACTATAC motifs detected in file\n",
      "\n",
      "Seq is: CTGAGATCT\n",
      "0 CTGAGATCT motifs detected in file\n",
      "\n",
      "Complement found from 5095 to 5104\n",
      "Seq is: TATCCTAAC\n",
      "1 TATCCTAAC motifs detected in file\n",
      "\n",
      "Seq is: GCGACAATA\n",
      "0 GCGACAATA motifs detected in file\n",
      "\n",
      "Seq is: GGGATCCCA\n",
      "0 GGGATCCCA motifs detected in file\n",
      "\n",
      "Seq is: TTGAACAAG\n",
      "0 TTGAACAAG motifs detected in file\n",
      "\n",
      "Seq is: GCAACCAGT\n",
      "0 GCAACCAGT motifs detected in file\n",
      "\n",
      "Seq is: TCAATAACC\n",
      "0 TCAATAACC motifs detected in file\n",
      "\n",
      "Seq is: TAGTTAACT\n",
      "0 TAGTTAACT motifs detected in file\n",
      "\n",
      "Seq is: CATATCATC\n",
      "0 CATATCATC motifs detected in file\n",
      "\n",
      "Seq is: AAGCGAAAC\n",
      "0 AAGCGAAAC motifs detected in file\n",
      "\n",
      "Seq is: TGACCACTT\n",
      "0 TGACCACTT motifs detected in file\n",
      "\n",
      "Seq is: CAGAATCGT\n",
      "0 CAGAATCGT motifs detected in file\n",
      "\n",
      "Seq is: ACATGACGA\n",
      "0 ACATGACGA motifs detected in file\n",
      "\n",
      "Seq is: GCCTCCAAC\n",
      "0 GCCTCCAAC motifs detected in file\n",
      "\n",
      "Seq is: CCAAGACGA\n",
      "0 CCAAGACGA motifs detected in file\n",
      "\n",
      "Seq is: GGCTCCAGC\n",
      "0 GGCTCCAGC motifs detected in file\n",
      "\n",
      "Seq is: GGGGTTGGC\n",
      "0 GGGGTTGGC motifs detected in file\n",
      "\n",
      "Seq is: TTTAAAAAT\n",
      "0 TTTAAAAAT motifs detected in file\n",
      "\n",
      "Seq is: TACGTCCAA\n",
      "0 TACGTCCAA motifs detected in file\n",
      "\n",
      "Complement found from 7870 to 7879\n",
      "Seq is: ACCATCAAA\n",
      "1 ACCATCAAA motifs detected in file\n",
      "\n",
      "Seq is: TCCTGGACA\n",
      "0 TCCTGGACA motifs detected in file\n",
      "\n",
      "Complement found from 3715 to 3724\n",
      "Complement found from 12547 to 12556\n",
      "Seq is: CCCAAACAA\n",
      "2 CCCAAACAA motifs detected in file\n",
      "\n",
      "Seq is: TCCATGACA\n",
      "0 TCCATGACA motifs detected in file\n",
      "\n",
      "Seq is: AACGTAATT\n",
      "0 AACGTAATT motifs detected in file\n",
      "\n",
      "Seq is: ATTTAATTT\n",
      "0 ATTTAATTT motifs detected in file\n",
      "\n",
      "Seq is: CGAGGTATC\n",
      "0 CGAGGTATC motifs detected in file\n",
      "\n",
      "Seq is: CCAGAAGAG\n",
      "0 CCAGAAGAG motifs detected in file\n",
      "\n",
      "Seq is: CAGAACGAC\n",
      "0 CAGAACGAC motifs detected in file\n",
      "\n",
      "Seq is: ACAATACGG\n",
      "0 ACAATACGG motifs detected in file\n",
      "\n",
      "Seq is: GCGGAGAAG\n",
      "0 GCGGAGAAG motifs detected in file\n",
      "\n",
      "Seq is: TGCCCGTCC\n",
      "0 TGCCCGTCC motifs detected in file\n",
      "\n",
      "Seq is: AGTTAAAGT\n",
      "0 AGTTAAAGT motifs detected in file\n",
      "\n",
      "Seq is: GACCAATTT\n",
      "0 GACCAATTT motifs detected in file\n",
      "\n",
      "Seq is: TCATTCTCT\n",
      "0 TCATTCTCT motifs detected in file\n",
      "\n",
      "Seq is: GTCGACTTG\n",
      "0 GTCGACTTG motifs detected in file\n",
      "\n",
      "Seq is: GGAGCACCT\n",
      "0 GGAGCACCT motifs detected in file\n",
      "\n",
      "Seq is: CGGTAAGTA\n",
      "0 CGGTAAGTA motifs detected in file\n",
      "\n",
      "Seq is: TGTCCAGGG\n",
      "0 TGTCCAGGG motifs detected in file\n",
      "\n",
      "Seq is: ATAAATTCC\n",
      "0 ATAAATTCC motifs detected in file\n",
      "\n",
      "Seq is: TTGTTCACT\n",
      "0 TTGTTCACT motifs detected in file\n",
      "\n",
      "Seq is: AATACGATG\n",
      "0 AATACGATG motifs detected in file\n",
      "\n",
      "Seq is: GAAACGTGC\n",
      "0 GAAACGTGC motifs detected in file\n",
      "\n",
      "Seq is: CAATCCCAT\n",
      "0 CAATCCCAT motifs detected in file\n",
      "\n",
      "Seq is: GGCGCCGGC\n",
      "0 GGCGCCGGC motifs detected in file\n",
      "\n",
      "Seq is: AATTTGTAC\n",
      "0 AATTTGTAC motifs detected in file\n",
      "\n",
      "Seq is: ACAGTGACC\n",
      "0 ACAGTGACC motifs detected in file\n",
      "\n",
      "Seq is: CGTCCGCCA\n",
      "0 CGTCCGCCA motifs detected in file\n",
      "\n",
      "Seq is: CGGAGATTA\n",
      "0 CGGAGATTA motifs detected in file\n",
      "\n",
      "Seq is: TGACCACTA\n",
      "0 TGACCACTA motifs detected in file\n",
      "\n",
      "Seq is: CGATCTCCA\n",
      "0 CGATCTCCA motifs detected in file\n",
      "\n",
      "Complement found from 10376 to 10385\n",
      "Seq is: CTACAAAAA\n",
      "1 CTACAAAAA motifs detected in file\n",
      "\n",
      "Seq is: CCATTTGTC\n",
      "0 CCATTTGTC motifs detected in file\n",
      "\n",
      "Seq is: CGCCCCATT\n",
      "0 CGCCCCATT motifs detected in file\n",
      "\n",
      "Seq is: CTAAACGGC\n",
      "0 CTAAACGGC motifs detected in file\n",
      "\n",
      "Seq is: TCAAGGAAA\n",
      "0 TCAAGGAAA motifs detected in file\n",
      "\n",
      "Complement found from 4172 to 4181\n",
      "Seq is: ATGAAAAAA\n",
      "1 ATGAAAAAA motifs detected in file\n",
      "\n",
      "Seq is: ATTGGAAAG\n",
      "0 ATTGGAAAG motifs detected in file\n",
      "\n",
      "Seq is: GAATACTCG\n",
      "0 GAATACTCG motifs detected in file\n",
      "\n",
      "Seq is: TACGGACAC\n",
      "0 TACGGACAC motifs detected in file\n",
      "\n",
      "Seq is: AACCCAACT\n",
      "0 AACCCAACT motifs detected in file\n",
      "\n",
      "Seq is: GTCACTCCC\n",
      "0 GTCACTCCC motifs detected in file\n",
      "\n",
      "Seq is: ATTATTACT\n",
      "0 ATTATTACT motifs detected in file\n",
      "\n",
      "Seq is: GAACAACCA\n",
      "0 GAACAACCA motifs detected in file\n",
      "\n",
      "Complement found from 7741 to 7750\n",
      "Seq is: ACTAACATC\n",
      "1 ACTAACATC motifs detected in file\n",
      "\n",
      "Seq is: TATAACCCG\n",
      "0 TATAACCCG motifs detected in file\n",
      "\n",
      "Complement found from 2366 to 2375\n",
      "Seq is: ACAATTAAC\n",
      "1 ACAATTAAC motifs detected in file\n",
      "\n",
      "Seq is: AGTCAAGTC\n",
      "0 AGTCAAGTC motifs detected in file\n",
      "\n",
      "Seq is: ACAAAATTA\n",
      "0 ACAAAATTA motifs detected in file\n",
      "\n",
      "Seq is: GACTGCGTC\n",
      "0 GACTGCGTC motifs detected in file\n",
      "\n",
      "Complement found from 3944 to 3953\n",
      "Seq is: CGAATACGC\n",
      "1 CGAATACGC motifs detected in file\n",
      "\n",
      "Seq is: CTCCTCTTA\n",
      "0 CTCCTCTTA motifs detected in file\n",
      "\n",
      "Seq is: CAAAAGTAC\n",
      "0 CAAAAGTAC motifs detected in file\n",
      "\n",
      "Seq is: AATGAATAT\n",
      "0 AATGAATAT motifs detected in file\n",
      "\n",
      "Seq is: GATTGTAAT\n",
      "0 GATTGTAAT motifs detected in file\n",
      "\n",
      "Seq is: CAAGAAGAT\n",
      "0 CAAGAAGAT motifs detected in file\n",
      "\n",
      "Seq is: ATCCCACTA\n",
      "0 ATCCCACTA motifs detected in file\n",
      "\n",
      "Seq is: TCTAACCAG\n",
      "0 TCTAACCAG motifs detected in file\n",
      "\n",
      "Seq is: GTTAACCCA\n",
      "0 GTTAACCCA motifs detected in file\n",
      "\n",
      "Seq is: CACTCCTCA\n",
      "0 CACTCCTCA motifs detected in file\n",
      "\n",
      "Seq is: AGTCAATAT\n",
      "0 AGTCAATAT motifs detected in file\n",
      "\n",
      "Complement found from 8507 to 8516\n",
      "Seq is: ACAAACCCT\n",
      "1 ACAAACCCT motifs detected in file\n",
      "\n",
      "Complement found from 2226 to 2235\n",
      "Seq is: AAAAAATCC\n",
      "1 AAAAAATCC motifs detected in file\n",
      "\n",
      "Seq is: ATCACCCAC\n",
      "0 ATCACCCAC motifs detected in file\n",
      "\n",
      "Seq is: AACTCGAAC\n",
      "0 AACTCGAAC motifs detected in file\n",
      "\n",
      "Seq is: TTGCGAAAG\n",
      "0 TTGCGAAAG motifs detected in file\n",
      "\n",
      "Complement found from 14784 to 14793\n",
      "Seq is: AATTAACCA\n",
      "1 AATTAACCA motifs detected in file\n",
      "\n",
      "Seq is: CCGACGAAA\n",
      "0 CCGACGAAA motifs detected in file\n",
      "\n",
      "Seq is: ATCCGGATG\n",
      "0 ATCCGGATG motifs detected in file\n",
      "\n",
      "Complement found from 3200 to 3209\n",
      "Seq is: ATACCCACA\n",
      "1 ATACCCACA motifs detected in file\n",
      "\n",
      "Seq is: ATTTAAAAA\n",
      "0 ATTTAAAAA motifs detected in file\n",
      "\n",
      "Seq is: ATGAGAGAG\n",
      "0 ATGAGAGAG motifs detected in file\n",
      "\n",
      "Seq is: ATGTTCCAA\n",
      "0 ATGTTCCAA motifs detected in file\n",
      "\n",
      "Seq is: AAAAGGATC\n",
      "0 AAAAGGATC motifs detected in file\n",
      "\n",
      "Complement found from 13494 to 13503\n",
      "Seq is: ACAGGTTTC\n",
      "1 ACAGGTTTC motifs detected in file\n",
      "\n",
      "Seq is: TCGACAAGG\n",
      "0 TCGACAAGG motifs detected in file\n",
      "\n",
      "Complement found from 15075 to 15084\n",
      "Seq is: AGAAACCTG\n",
      "1 AGAAACCTG motifs detected in file\n",
      "\n",
      "Seq is: ATTGTCAAT\n",
      "0 ATTGTCAAT motifs detected in file\n",
      "\n",
      "Seq is: TTAAATGTT\n",
      "0 TTAAATGTT motifs detected in file\n",
      "\n",
      "Complement found from 14489 to 14498\n",
      "Seq is: CCCCTAAAT\n",
      "1 CCCCTAAAT motifs detected in file\n",
      "\n",
      "Seq is: CTCCCAAGA\n",
      "0 CTCCCAAGA motifs detected in file\n",
      "\n",
      "Seq is: CACCCGTTT\n",
      "0 CACCCGTTT motifs detected in file\n",
      "\n",
      "Seq is: AAATTTCAA\n",
      "0 AAATTTCAA motifs detected in file\n",
      "\n",
      "Seq is: CTTGATTCT\n",
      "0 CTTGATTCT motifs detected in file\n",
      "\n",
      "Complement found from 2025 to 2034\n",
      "Seq is: AAGATAGAA\n",
      "1 AAGATAGAA motifs detected in file\n",
      "\n",
      "Seq is: CCTGTTGGT\n",
      "0 CCTGTTGGT motifs detected in file\n",
      "\n",
      "Seq is: CGATAGTGG\n",
      "0 CGATAGTGG motifs detected in file\n",
      "\n",
      "Seq is: TCCGAGCCA\n",
      "0 TCCGAGCCA motifs detected in file\n",
      "\n",
      "Seq is: TCCAAACAG\n",
      "0 TCCAAACAG motifs detected in file\n",
      "\n",
      "Seq is: CGGAGATGG\n",
      "0 CGGAGATGG motifs detected in file\n",
      "\n",
      "Seq is: ATATTTAGA\n",
      "0 ATATTTAGA motifs detected in file\n",
      "\n",
      "Seq is: AGGGTGATA\n",
      "0 AGGGTGATA motifs detected in file\n",
      "\n",
      "Seq is: AAACGATGT\n",
      "0 AAACGATGT motifs detected in file\n",
      "\n",
      "Seq is: ATCTGCCCA\n",
      "0 ATCTGCCCA motifs detected in file\n",
      "\n",
      "Complement found from 9390 to 9399\n",
      "Complement found from 12050 to 12059\n",
      "Seq is: CACGAGAAA\n",
      "2 CACGAGAAA motifs detected in file\n",
      "\n",
      "Seq is: ATCGACAAG\n",
      "0 ATCGACAAG motifs detected in file\n",
      "\n",
      "Seq is: AATCCATCG\n",
      "0 AATCCATCG motifs detected in file\n",
      "\n",
      "Seq is: AGCAGACCA\n",
      "0 AGCAGACCA motifs detected in file\n",
      "\n",
      "Complement found from 8024 to 8033\n",
      "Complement found from 11423 to 11432\n",
      "Seq is: AAGCCCCCA\n",
      "2 AAGCCCCCA motifs detected in file\n",
      "\n",
      "Seq is: GAATCGAAA\n",
      "0 GAATCGAAA motifs detected in file\n",
      "\n",
      "Seq is: CCGAGAGGA\n",
      "0 CCGAGAGGA motifs detected in file\n",
      "\n",
      "Seq is: ACGTTTCAA\n",
      "0 ACGTTTCAA motifs detected in file\n",
      "\n",
      "Seq is: TAAAGATCA\n",
      "0 TAAAGATCA motifs detected in file\n",
      "\n",
      "Seq is: ATTAAGTAA\n",
      "0 ATTAAGTAA motifs detected in file\n",
      "\n",
      "Seq is: TACGTCTTC\n",
      "0 TACGTCTTC motifs detected in file\n",
      "\n",
      "Seq is: CATATCCCC\n",
      "0 CATATCCCC motifs detected in file\n",
      "\n",
      "Seq is: AATCAGGAA\n",
      "0 AATCAGGAA motifs detected in file\n",
      "\n",
      "Seq is: CGATATAAT\n",
      "0 CGATATAAT motifs detected in file\n",
      "\n",
      "Seq is: ACGAACCAA\n",
      "0 ACGAACCAA motifs detected in file\n",
      "\n",
      "Seq is: TATTAAAAA\n",
      "0 TATTAAAAA motifs detected in file\n",
      "\n",
      "Seq is: GTAGAAAGG\n",
      "0 GTAGAAAGG motifs detected in file\n",
      "\n",
      "Seq is: GAACGCCAT\n",
      "0 GAACGCCAT motifs detected in file\n",
      "\n",
      "Seq is: GATATAGAT\n",
      "0 GATATAGAT motifs detected in file\n",
      "\n",
      "Seq is: AACGCGGTC\n",
      "0 AACGCGGTC motifs detected in file\n",
      "\n",
      "Complement found from 7549 to 7558\n",
      "Seq is: CAAAGTTAA\n",
      "1 CAAAGTTAA motifs detected in file\n",
      "\n",
      "Seq is: AGATAGCGG\n",
      "0 AGATAGCGG motifs detected in file\n",
      "\n",
      "Complement found from 6894 to 6903\n",
      "Seq is: ATATGAAAT\n",
      "1 ATATGAAAT motifs detected in file\n",
      "\n",
      "Seq is: AAACCCATT\n",
      "0 AAACCCATT motifs detected in file\n",
      "\n",
      "Complement found from 2785 to 2794\n",
      "Seq is: TACCAAACC\n",
      "1 TACCAAACC motifs detected in file\n",
      "\n",
      "Seq is: GATTCCAAC\n",
      "0 GATTCCAAC motifs detected in file\n",
      "\n",
      "Seq is: AGACCATCA\n",
      "0 AGACCATCA motifs detected in file\n",
      "\n",
      "Seq is: TTCCACCTC\n",
      "0 TTCCACCTC motifs detected in file\n",
      "\n",
      "Seq is: ACCCAAACC\n",
      "0 ACCCAAACC motifs detected in file\n",
      "\n",
      "Seq is: CCGATCCAA\n",
      "0 CCGATCCAA motifs detected in file\n",
      "\n",
      "Seq is: ATCGAGTCT\n",
      "0 ATCGAGTCT motifs detected in file\n",
      "\n",
      "Seq is: CGCCAGTTC\n",
      "0 CGCCAGTTC motifs detected in file\n",
      "\n",
      "Seq is: AATTCAACT\n",
      "0 AATTCAACT motifs detected in file\n",
      "\n",
      "Seq is: TTAGAGGAT\n",
      "0 TTAGAGGAT motifs detected in file\n",
      "\n",
      "Seq is: TCACATTCA\n",
      "0 TCACATTCA motifs detected in file\n",
      "\n",
      "Seq is: ACCCACGAA\n",
      "0 ACCCACGAA motifs detected in file\n",
      "\n",
      "Complement found from 15578 to 15587\n",
      "Seq is: ACACAATTC\n",
      "1 ACACAATTC motifs detected in file\n",
      "\n",
      "Seq is: GATGTGAGA\n",
      "0 GATGTGAGA motifs detected in file\n",
      "\n",
      "Seq is: CCAAGCAGG\n",
      "0 CCAAGCAGG motifs detected in file\n",
      "\n",
      "Seq is: TTCACGTGA\n",
      "0 TTCACGTGA motifs detected in file\n",
      "\n",
      "Seq is: AAGGTCATG\n",
      "0 AAGGTCATG motifs detected in file\n",
      "\n",
      "Seq is: TGAATGGTA\n",
      "0 TGAATGGTA motifs detected in file\n",
      "\n",
      "Seq is: CAATGCTGA\n",
      "0 CAATGCTGA motifs detected in file\n",
      "\n",
      "Seq is: ACAGAGGAG\n",
      "0 ACAGAGGAG motifs detected in file\n",
      "\n",
      "Complement found from 6774 to 6783\n",
      "Seq is: ATATATTTA\n",
      "1 ATATATTTA motifs detected in file\n",
      "\n",
      "Complement found from 13757 to 13766\n",
      "Seq is: CGCATCCCC\n",
      "1 CGCATCCCC motifs detected in file\n",
      "\n",
      "Seq is: AAAATCAAT\n",
      "0 AAAATCAAT motifs detected in file\n",
      "\n",
      "Seq is: TTACAGGAA\n",
      "0 TTACAGGAA motifs detected in file\n",
      "\n",
      "Seq is: ACTTCATAT\n",
      "0 ACTTCATAT motifs detected in file\n",
      "\n",
      "Complement found from 2248 to 2257\n",
      "Seq is: GAACTCCTC\n",
      "1 GAACTCCTC motifs detected in file\n",
      "\n",
      "Seq is: CCACTGCCC\n",
      "0 CCACTGCCC motifs detected in file\n",
      "\n",
      "Seq is: GCCACACAT\n",
      "0 GCCACACAT motifs detected in file\n",
      "\n",
      "Seq is: GCGCGAAGT\n",
      "0 GCGCGAAGT motifs detected in file\n",
      "\n",
      "Seq is: CCCGGGACA\n",
      "0 CCCGGGACA motifs detected in file\n",
      "\n",
      "Seq is: AGTTGATTC\n",
      "0 AGTTGATTC motifs detected in file\n",
      "\n",
      "Seq is: GTGAGATGA\n",
      "0 GTGAGATGA motifs detected in file\n",
      "\n",
      "Seq is: GAATCAAAT\n",
      "0 GAATCAAAT motifs detected in file\n",
      "\n",
      "Seq is: GACGATTTA\n",
      "0 GACGATTTA motifs detected in file\n",
      "\n",
      "Seq is: GGTGGAAGC\n",
      "0 GGTGGAAGC motifs detected in file\n",
      "\n",
      "Seq is: TGGGAATTC\n",
      "0 TGGGAATTC motifs detected in file\n",
      "\n",
      "Seq is: AAAGTATTC\n",
      "0 AAAGTATTC motifs detected in file\n",
      "\n",
      "Seq is: CCGATAGCA\n",
      "0 CCGATAGCA motifs detected in file\n",
      "\n",
      "Seq is: TCAAAAGAC\n",
      "0 TCAAAAGAC motifs detected in file\n",
      "\n",
      "Seq is: CCCATCTTT\n",
      "0 CCCATCTTT motifs detected in file\n",
      "\n",
      "Seq is: TACATCGGG\n",
      "0 TACATCGGG motifs detected in file\n",
      "\n",
      "Seq is: TAAAGAACG\n",
      "0 TAAAGAACG motifs detected in file\n",
      "\n",
      "Seq is: GTGGAGTAC\n",
      "0 GTGGAGTAC motifs detected in file\n",
      "\n",
      "Seq is: CCGATGTGG\n",
      "0 CCGATGTGG motifs detected in file\n",
      "\n",
      "Seq is: AACTGGATT\n",
      "0 AACTGGATT motifs detected in file\n",
      "\n",
      "Seq is: GCAGAAATG\n",
      "0 GCAGAAATG motifs detected in file\n",
      "\n",
      "Seq is: CACCCATGA\n",
      "0 CACCCATGA motifs detected in file\n",
      "\n",
      "Seq is: ACGCGAATG\n",
      "0 ACGCGAATG motifs detected in file\n",
      "\n",
      "Complement found from 15084 to 15093\n",
      "Seq is: AAACATCGG\n",
      "1 AAACATCGG motifs detected in file\n",
      "\n",
      "Seq is: AAGTAGTCC\n",
      "0 AAGTAGTCC motifs detected in file\n",
      "\n",
      "Seq is: CAAACGACT\n",
      "0 CAAACGACT motifs detected in file\n",
      "\n",
      "Seq is: TCTACCGCC\n",
      "0 TCTACCGCC motifs detected in file\n",
      "\n",
      "Seq is: ATATATCCG\n",
      "0 ATATATCCG motifs detected in file\n",
      "\n",
      "Seq is: ACTCGTTCT\n",
      "0 ACTCGTTCT motifs detected in file\n",
      "\n",
      "Seq is: CCACCACTC\n",
      "0 CCACCACTC motifs detected in file\n",
      "\n",
      "Seq is: CAACTAGCC\n",
      "0 CAACTAGCC motifs detected in file\n",
      "\n",
      "Seq is: CCAAATAGC\n",
      "0 CCAAATAGC motifs detected in file\n",
      "\n",
      "Seq is: TAATGTCTT\n",
      "0 TAATGTCTT motifs detected in file\n",
      "\n",
      "Seq is: GTCCGAGGA\n",
      "0 GTCCGAGGA motifs detected in file\n",
      "\n",
      "Seq is: GATCTCCCT\n",
      "0 GATCTCCCT motifs detected in file\n",
      "\n",
      "Seq is: ATACTTCGT\n",
      "0 ATACTTCGT motifs detected in file\n",
      "\n",
      "Seq is: GGCGGTCCA\n",
      "0 GGCGGTCCA motifs detected in file\n",
      "\n",
      "Seq is: GGAAACTCA\n",
      "0 GGAAACTCA motifs detected in file\n",
      "\n",
      "Seq is: AAATTCGAC\n",
      "0 AAATTCGAC motifs detected in file\n",
      "\n",
      "Seq is: ACCGAGCAT\n",
      "0 ACCGAGCAT motifs detected in file\n",
      "\n",
      "Seq is: CACAAGACC\n",
      "0 CACAAGACC motifs detected in file\n",
      "\n",
      "Seq is: GCTCGTCAA\n",
      "0 GCTCGTCAA motifs detected in file\n",
      "\n",
      "Complement found from 10297 to 10306\n",
      "Seq is: AACAACTAA\n",
      "1 AACAACTAA motifs detected in file\n",
      "\n",
      "Seq is: ATTGACAAC\n",
      "0 ATTGACAAC motifs detected in file\n",
      "\n",
      "Seq is: TCCAAATCC\n",
      "0 TCCAAATCC motifs detected in file\n",
      "\n",
      "Seq is: CGATTCGTA\n",
      "0 CGATTCGTA motifs detected in file\n",
      "\n",
      "Complement found from 633 to 642\n",
      "Seq is: TCACCCCAT\n",
      "1 TCACCCCAT motifs detected in file\n",
      "\n",
      "Seq is: AGATTAGGG\n",
      "0 AGATTAGGG motifs detected in file\n",
      "\n",
      "Seq is: TCAAACCCA\n",
      "0 TCAAACCCA motifs detected in file\n",
      "\n",
      "Seq is: GAATCGATA\n",
      "0 GAATCGATA motifs detected in file\n",
      "\n",
      "Seq is: ACACACAAG\n",
      "0 ACACACAAG motifs detected in file\n",
      "\n",
      "Seq is: TCTATACAA\n",
      "0 TCTATACAA motifs detected in file\n",
      "\n",
      "Seq is: TTTCGGTGA\n",
      "0 TTTCGGTGA motifs detected in file\n",
      "\n",
      "Seq is: AAGCATCAG\n",
      "0 AAGCATCAG motifs detected in file\n",
      "\n",
      "Seq is: ATAAAACAC\n",
      "0 ATAAAACAC motifs detected in file\n",
      "\n",
      "Seq is: AGTTGACCT\n",
      "0 AGTTGACCT motifs detected in file\n",
      "\n",
      "Seq is: CAAAAAATG\n",
      "0 CAAAAAATG motifs detected in file\n",
      "\n",
      "Seq is: TTGAGTCCA\n",
      "0 TTGAGTCCA motifs detected in file\n",
      "\n",
      "Seq is: CTCAAAATC\n",
      "0 CTCAAAATC motifs detected in file\n",
      "\n",
      "Seq is: GAAATAACC\n",
      "0 GAAATAACC motifs detected in file\n",
      "\n",
      "Complement found from 14527 to 14536\n",
      "Seq is: CCTCCCCCA\n",
      "1 CCTCCCCCA motifs detected in file\n",
      "\n",
      "Seq is: CTAGATTTT\n",
      "0 CTAGATTTT motifs detected in file\n",
      "\n",
      "Seq is: GTGAGAAAT\n",
      "0 GTGAGAAAT motifs detected in file\n",
      "\n",
      "Seq is: GCGGCCGAA\n",
      "0 GCGGCCGAA motifs detected in file\n",
      "\n",
      "Seq is: GATAACTGA\n",
      "0 GATAACTGA motifs detected in file\n",
      "\n",
      "Seq is: ACCCAATTA\n",
      "0 ACCCAATTA motifs detected in file\n",
      "\n",
      "Seq is: GCACACTGG\n",
      "0 GCACACTGG motifs detected in file\n",
      "\n",
      "Seq is: CGCCACCGA\n",
      "0 CGCCACCGA motifs detected in file\n",
      "\n",
      "Seq is: CCGTGCTTT\n",
      "0 CCGTGCTTT motifs detected in file\n",
      "\n",
      "Seq is: AACTGGTTG\n",
      "0 AACTGGTTG motifs detected in file\n",
      "\n",
      "Seq is: GGACCCCAA\n",
      "0 GGACCCCAA motifs detected in file\n",
      "\n",
      "Seq is: TCATATCGA\n",
      "0 TCATATCGA motifs detected in file\n",
      "\n",
      "Seq is: ATCAATTTG\n",
      "0 ATCAATTTG motifs detected in file\n",
      "\n",
      "Seq is: AAAGCAAAT\n",
      "0 AAAGCAAAT motifs detected in file\n",
      "\n",
      "Seq is: AACGATTTC\n",
      "0 AACGATTTC motifs detected in file\n",
      "\n",
      "Seq is: CAATTAGTG\n",
      "0 CAATTAGTG motifs detected in file\n",
      "\n",
      "Seq is: ACGACAAAG\n",
      "0 ACGACAAAG motifs detected in file\n",
      "\n",
      "Seq is: GGCACCCCC\n",
      "0 GGCACCCCC motifs detected in file\n",
      "\n",
      "Seq is: ACACCGATC\n",
      "0 ACACCGATC motifs detected in file\n",
      "\n",
      "Seq is: CGATTCGCA\n",
      "0 CGATTCGCA motifs detected in file\n",
      "\n",
      "Seq is: AAACTCGAC\n",
      "0 AAACTCGAC motifs detected in file\n",
      "\n",
      "Seq is: GTAACGACG\n",
      "0 GTAACGACG motifs detected in file\n",
      "\n",
      "Seq is: CACGAACTA\n",
      "0 CACGAACTA motifs detected in file\n",
      "\n",
      "Seq is: CGAACAAGG\n",
      "0 CGAACAAGG motifs detected in file\n",
      "\n",
      "Complement found from 4870 to 4879\n",
      "Seq is: AAAACTAGC\n",
      "1 AAAACTAGC motifs detected in file\n",
      "\n",
      "Seq is: ACCACTAAA\n",
      "0 ACCACTAAA motifs detected in file\n",
      "\n",
      "Seq is: TCTCCCACT\n",
      "0 TCTCCCACT motifs detected in file\n",
      "\n",
      "Seq is: TGAGTGACC\n",
      "0 TGAGTGACC motifs detected in file\n",
      "\n",
      "Seq is: TTGCCCCTA\n",
      "0 TTGCCCCTA motifs detected in file\n",
      "\n",
      "Seq is: CGAACGTAC\n",
      "0 CGAACGTAC motifs detected in file\n",
      "\n",
      "Seq is: ACATTAGAA\n",
      "0 ACATTAGAA motifs detected in file\n",
      "\n",
      "Seq is: TGATTCTCG\n",
      "0 TGATTCTCG motifs detected in file\n",
      "\n",
      "Seq is: ATTATCTTT\n",
      "0 ATTATCTTT motifs detected in file\n",
      "\n",
      "Seq is: CCGATCCTG\n",
      "0 CCGATCCTG motifs detected in file\n",
      "\n",
      "Seq is: GTTTGGATA\n",
      "0 GTTTGGATA motifs detected in file\n",
      "\n",
      "Seq is: AACAAATAC\n",
      "0 AACAAATAC motifs detected in file\n",
      "\n",
      "Seq is: CCCACTACA\n",
      "0 CCCACTACA motifs detected in file\n",
      "\n",
      "Seq is: CTCGGGCAG\n",
      "0 CTCGGGCAG motifs detected in file\n",
      "\n",
      "Seq is: ATTTGTAAA\n",
      "0 ATTTGTAAA motifs detected in file\n",
      "\n",
      "Seq is: AGTCACATA\n",
      "0 AGTCACATA motifs detected in file\n",
      "\n",
      "Seq is: ACGAAACTC\n",
      "0 ACGAAACTC motifs detected in file\n",
      "\n",
      "Seq is: CTCCATTCG\n",
      "0 CTCCATTCG motifs detected in file\n",
      "\n",
      "Seq is: ATGTATTTG\n",
      "0 ATGTATTTG motifs detected in file\n",
      "\n",
      "Complement found from 564 to 573\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Seq is: ACACCCCCC\n",
      "1 ACACCCCCC motifs detected in file\n",
      "\n",
      "Seq is: ACAGAAACC\n",
      "0 ACAGAAACC motifs detected in file\n",
      "\n",
      "Seq is: CCAAACCAA\n",
      "0 CCAAACCAA motifs detected in file\n",
      "\n",
      "Seq is: CCAAGCCCC\n",
      "0 CCAAGCCCC motifs detected in file\n",
      "\n",
      "Seq is: ATACCCCAA\n",
      "0 ATACCCCAA motifs detected in file\n",
      "\n",
      "Seq is: TCGTCGCCA\n",
      "0 TCGTCGCCA motifs detected in file\n",
      "\n",
      "Complement found from 513 to 522\n",
      "Seq is: CACACACAC\n",
      "1 CACACACAC motifs detected in file\n",
      "\n",
      "Seq is: GACCCATCC\n",
      "0 GACCCATCC motifs detected in file\n",
      "\n",
      "Seq is: TACCCGCCC\n",
      "0 TACCCGCCC motifs detected in file\n",
      "\n",
      "Seq is: CCAACATAA\n",
      "0 CCAACATAA motifs detected in file\n",
      "\n",
      "Seq is: CTACTCTAA\n",
      "0 CTACTCTAA motifs detected in file\n",
      "\n",
      "Seq is: TCATCATAC\n",
      "0 TCATCATAC motifs detected in file\n",
      "\n",
      "Seq is: CCTCACCCT\n",
      "0 CCTCACCCT motifs detected in file\n",
      "\n",
      "Complement found from 5487 to 5496\n",
      "Seq is: CCCCTTTTA\n",
      "1 CCCCTTTTA motifs detected in file\n",
      "\n",
      "Complement found from 11352 to 11361\n",
      "Seq is: TTACACAAT\n",
      "1 TTACACAAT motifs detected in file\n",
      "\n",
      "Complement found from 10943 to 10952\n",
      "Seq is: CAACCCCCC\n",
      "1 CAACCCCCC motifs detected in file\n",
      "\n",
      "Complement found from 2362 to 2371\n",
      "Seq is: ACTGACAAT\n",
      "1 ACTGACAAT motifs detected in file\n",
      "\n",
      "Seq is: TTTCACGTA\n",
      "0 TTTCACGTA motifs detected in file\n",
      "\n",
      "Seq is: TGGCGGTTT\n",
      "0 TGGCGGTTT motifs detected in file\n",
      "\n",
      "Complement found from 9717 to 9726\n",
      "Seq is: TCTATTTTA\n",
      "1 TCTATTTTA motifs detected in file\n",
      "\n",
      "Seq is: AACTTTAGA\n",
      "0 AACTTTAGA motifs detected in file\n",
      "\n",
      "Seq is: CCAATCCGA\n",
      "0 CCAATCCGA motifs detected in file\n",
      "\n",
      "Complement found from 8560 to 8569\n",
      "Seq is: CCACAATCC\n",
      "1 CCACAATCC motifs detected in file\n",
      "\n",
      "Seq is: CAAGAAACA\n",
      "0 CAAGAAACA motifs detected in file\n",
      "\n",
      "Complement found from 5596 to 5605\n",
      "Complement found from 14615 to 14624\n",
      "Seq is: AAAACCCCA\n",
      "2 AAAACCCCA motifs detected in file\n",
      "\n",
      "Seq is: AACCGTCTC\n",
      "0 AACCGTCTC motifs detected in file\n",
      "\n",
      "Seq is: TACACAAAT\n",
      "0 TACACAAAT motifs detected in file\n",
      "\n",
      "Seq is: TCACGACAC\n",
      "0 TCACGACAC motifs detected in file\n",
      "\n",
      "Seq is: CGGTCTTCG\n",
      "0 CGGTCTTCG motifs detected in file\n",
      "\n",
      "Complement found from 304 to 313\n",
      "Complement found from 955 to 964\n",
      "Complement found from 3566 to 3575\n",
      "Complement found from 16183 to 16192\n",
      "Seq is: CCCCCTCCC\n",
      "4 CCCCCTCCC motifs detected in file\n",
      "\n",
      "Complement found from 1681 to 1690\n",
      "Seq is: CCCCAAACC\n",
      "1 CCCCAAACC motifs detected in file\n",
      "\n",
      "Seq is: ACCTTTAAA\n",
      "0 ACCTTTAAA motifs detected in file\n",
      "\n",
      "Seq is: AAACAATAC\n",
      "0 AAACAATAC motifs detected in file\n",
      "\n",
      "Seq is: TACAGACAC\n",
      "0 TACAGACAC motifs detected in file\n",
      "\n",
      "Complement found from 15283 to 15292\n",
      "Seq is: ACCTTTCAC\n",
      "1 ACCTTTCAC motifs detected in file\n",
      "\n",
      "Seq is: CGACACGTC\n",
      "0 CGACACGTC motifs detected in file\n",
      "\n",
      "Seq is: TGTAAGTTA\n",
      "0 TGTAAGTTA motifs detected in file\n",
      "\n",
      "Seq is: ACAATAATA\n",
      "0 ACAATAATA motifs detected in file\n",
      "\n",
      "Seq is: ATACAGGAT\n",
      "0 ATACAGGAT motifs detected in file\n",
      "\n",
      "Seq is: GTTCGTAAT\n",
      "0 GTTCGTAAT motifs detected in file\n",
      "\n",
      "Complement found from 208 to 217\n",
      "Seq is: TAATTAATT\n",
      "1 TAATTAATT motifs detected in file\n",
      "\n",
      "Seq is: GTGTGAAAT\n",
      "0 GTGTGAAAT motifs detected in file\n",
      "\n",
      "Seq is: CATTCATAC\n",
      "0 CATTCATAC motifs detected in file\n",
      "\n",
      "Seq is: AAGCGGACA\n",
      "0 AAGCGGACA motifs detected in file\n",
      "\n",
      "Seq is: TTATAACTT\n",
      "0 TTATAACTT motifs detected in file\n",
      "\n",
      "Seq is: GCATCCACG\n",
      "0 GCATCCACG motifs detected in file\n",
      "\n",
      "Complement found from 3612 to 3621\n",
      "Complement found from 9458 to 9467\n",
      "Seq is: CTATTTATT\n",
      "2 CTATTTATT motifs detected in file\n",
      "\n",
      "Seq is: ATCCTACTC\n",
      "0 ATCCTACTC motifs detected in file\n",
      "\n",
      "Seq is: CGTCCTTAG\n",
      "0 CGTCCTTAG motifs detected in file\n",
      "\n",
      "Seq is: TTTCTGTCT\n",
      "0 TTTCTGTCT motifs detected in file\n",
      "\n",
      "Seq is: ATGACGCTG\n",
      "0 ATGACGCTG motifs detected in file\n",
      "\n",
      "Seq is: TATCCCACG\n",
      "0 TATCCCACG motifs detected in file\n",
      "\n",
      "Seq is: AGGCCGAGG\n",
      "0 AGGCCGAGG motifs detected in file\n",
      "\n",
      "Seq is: TCGCAGAGC\n",
      "0 TCGCAGAGC motifs detected in file\n",
      "\n",
      "Seq is: GTTACGATA\n",
      "0 GTTACGATA motifs detected in file\n",
      "\n",
      "Seq is: GCGCACGTA\n",
      "0 GCGCACGTA motifs detected in file\n",
      "\n",
      "Seq is: TGGGGGGTC\n",
      "0 TGGGGGGTC motifs detected in file\n",
      "\n",
      "Seq is: TGCTTTTAT\n",
      "0 TGCTTTTAT motifs detected in file\n",
      "\n",
      "Seq is: GGTTTACGT\n",
      "0 GGTTTACGT motifs detected in file\n",
      "\n",
      "Seq is: ACCTCTCGA\n",
      "0 ACCTCTCGA motifs detected in file\n",
      "\n",
      "Seq is: GGGCACTCA\n",
      "0 GGGCACTCA motifs detected in file\n",
      "\n",
      "Seq is: CCAATTATC\n",
      "0 CCAATTATC motifs detected in file\n",
      "\n",
      "Seq is: CCACTATCT\n",
      "0 CCACTATCT motifs detected in file\n",
      "\n",
      "Complement found from 2119 to 2128\n",
      "Seq is: GGACACTAG\n",
      "1 GGACACTAG motifs detected in file\n",
      "\n",
      "Seq is: GTAGCACTAC\n",
      "0 GTAGCACTAC motifs detected in file\n",
      "\n",
      "Seq is: AGAATAAATT\n",
      "0 AGAATAAATT motifs detected in file\n",
      "\n",
      "Seq is: CCCCTTGCAC\n",
      "0 CCCCTTGCAC motifs detected in file\n",
      "\n",
      "Seq is: ACCCGATAAA\n",
      "0 ACCCGATAAA motifs detected in file\n",
      "\n",
      "Seq is: TCCGAAATAC\n",
      "0 TCCGAAATAC motifs detected in file\n",
      "\n",
      "Seq is: TGGGACTTCA\n",
      "0 TGGGACTTCA motifs detected in file\n",
      "\n",
      "Seq is: TCCTTGGTCT\n",
      "0 TCCTTGGTCT motifs detected in file\n",
      "\n",
      "Seq is: ACAGCCTATG\n",
      "0 ACAGCCTATG motifs detected in file\n",
      "\n",
      "Seq is: TCAAGTGAAA\n",
      "0 TCAAGTGAAA motifs detected in file\n",
      "\n",
      "Seq is: TCGATGGGGG\n",
      "0 TCGATGGGGG motifs detected in file\n",
      "\n",
      "Seq is: TTCACAATAC\n",
      "0 TTCACAATAC motifs detected in file\n",
      "\n",
      "Seq is: CCGGGCCTCG\n",
      "0 CCGGGCCTCG motifs detected in file\n",
      "\n",
      "Seq is: CTCCTCTCAT\n",
      "0 CTCCTCTCAT motifs detected in file\n",
      "\n",
      "Seq is: CGTGAGAACA\n",
      "0 CGTGAGAACA motifs detected in file\n",
      "\n",
      "Seq is: CGCCCTATAA\n",
      "0 CGCCCTATAA motifs detected in file\n",
      "\n",
      "Seq is: CTAAAGTGCC\n",
      "0 CTAAAGTGCC motifs detected in file\n",
      "\n",
      "Seq is: TCCTACCACC\n",
      "0 TCCTACCACC motifs detected in file\n",
      "\n",
      "Seq is: AGTTCCCTGG\n",
      "0 AGTTCCCTGG motifs detected in file\n",
      "\n",
      "Seq is: GGATAGACTC\n",
      "0 GGATAGACTC motifs detected in file\n",
      "\n",
      "Seq is: CCCCCAGTAG\n",
      "0 CCCCCAGTAG motifs detected in file\n",
      "\n",
      "Seq is: GTACCCCTGC\n",
      "0 GTACCCCTGC motifs detected in file\n",
      "\n",
      "Seq is: TCTTCCCTAA\n",
      "0 TCTTCCCTAA motifs detected in file\n",
      "\n",
      "Seq is: ACTGACATTA\n",
      "0 ACTGACATTA motifs detected in file\n",
      "\n",
      "Seq is: CACGATACAT\n",
      "0 CACGATACAT motifs detected in file\n",
      "\n",
      "Complement found from 16318 to 16328\n",
      "Seq is: GCCATTTACC\n",
      "1 GCCATTTACC motifs detected in file\n",
      "\n",
      "Seq is: GAAATACATG\n",
      "0 GAAATACATG motifs detected in file\n",
      "\n",
      "Seq is: ATACATGACA\n",
      "0 ATACATGACA motifs detected in file\n",
      "\n",
      "Seq is: ATTCCCACCC\n",
      "0 ATTCCCACCC motifs detected in file\n",
      "\n",
      "Complement found from 15672 to 15682\n",
      "Seq is: ATCCAAACAA\n",
      "1 ATCCAAACAA motifs detected in file\n",
      "\n",
      "Seq is: CCATAGGATC\n",
      "0 CCATAGGATC motifs detected in file\n",
      "\n",
      "Complement found from 10619 to 10629\n",
      "Seq is: ACCCACTCCC\n",
      "1 ACCCACTCCC motifs detected in file\n",
      "\n",
      "Seq is: CACCGAAACC\n",
      "0 CACCGAAACC motifs detected in file\n",
      "\n",
      "Seq is: TCAACGTCAA\n",
      "0 TCAACGTCAA motifs detected in file\n",
      "\n",
      "Seq is: CTACACACTA\n",
      "0 CTACACACTA motifs detected in file\n",
      "\n",
      "Seq is: TCAACTCCCA\n",
      "0 TCAACTCCCA motifs detected in file\n",
      "\n",
      "Seq is: ACTAACGACA\n",
      "0 ACTAACGACA motifs detected in file\n",
      "\n",
      "Seq is: TGAACGAACA\n",
      "0 TGAACGAACA motifs detected in file\n",
      "\n",
      "Seq is: TTCGTACCCC\n",
      "0 TTCGTACCCC motifs detected in file\n",
      "\n",
      "Complement found from 14529 to 14539\n",
      "Seq is: TCCCCCAAAA\n",
      "1 TCCCCCAAAA motifs detected in file\n",
      "\n",
      "Seq is: CTACACCTAA\n",
      "0 CTACACCTAA motifs detected in file\n",
      "\n",
      "Seq is: CCCAAAAATA\n",
      "0 CCCAAAAATA motifs detected in file\n",
      "\n",
      "Seq is: CATGATGTCC\n",
      "0 CATGATGTCC motifs detected in file\n",
      "\n",
      "Seq is: ACCAGTTCAT\n",
      "0 ACCAGTTCAT motifs detected in file\n",
      "\n",
      "Seq is: AAATACCATG\n",
      "0 AAATACCATG motifs detected in file\n",
      "\n",
      "Seq is: GCATGTTATA\n",
      "0 GCATGTTATA motifs detected in file\n",
      "\n",
      "Seq is: AGTACCACCG\n",
      "0 AGTACCACCG motifs detected in file\n",
      "\n",
      "Seq is: ACCGTCATTA\n",
      "0 ACCGTCATTA motifs detected in file\n",
      "\n",
      "Seq is: CATGCTTTAT\n",
      "0 CATGCTTTAT motifs detected in file\n",
      "\n",
      "Seq is: GTATCGCCAA\n",
      "0 GTATCGCCAA motifs detected in file\n",
      "\n",
      "Seq is: CAACTACCCA\n",
      "0 CAACTACCCA motifs detected in file\n",
      "\n",
      "Seq is: CTCAGTTATG\n",
      "0 CTCAGTTATG motifs detected in file\n",
      "\n",
      "Seq is: AACCCACCAT\n",
      "0 AACCCACCAT motifs detected in file\n",
      "\n",
      "Seq is: GGGTTTAGAC\n",
      "0 GGGTTTAGAC motifs detected in file\n",
      "\n",
      "Seq is: GAAGGGGTAC\n",
      "0 GAAGGGGTAC motifs detected in file\n",
      "\n",
      "Seq is: TTTCTTGTCT\n",
      "0 TTTCTTGTCT motifs detected in file\n",
      "\n",
      "Seq is: CTTATCAAAT\n",
      "0 CTTATCAAAT motifs detected in file\n",
      "\n",
      "Seq is: TTAATCTTAG\n",
      "0 TTAATCTTAG motifs detected in file\n",
      "\n",
      "Seq is: AATCGAAACC\n",
      "0 AATCGAAACC motifs detected in file\n",
      "\n",
      "Seq is: CACGATTACC\n",
      "0 CACGATTACC motifs detected in file\n",
      "\n",
      "Seq is: ACCTCAATTT\n",
      "0 ACCTCAATTT motifs detected in file\n",
      "\n",
      "Seq is: CTGAAAAAGA\n",
      "0 CTGAAAAAGA motifs detected in file\n",
      "\n",
      "Seq is: GACTAAACAG\n",
      "0 GACTAAACAG motifs detected in file\n",
      "\n",
      "Seq is: GAACCTTTTT\n",
      "0 GAACCTTTTT motifs detected in file\n",
      "\n",
      "Seq is: CCAAAAGTAG\n",
      "0 CCAAAAGTAG motifs detected in file\n",
      "\n",
      "Seq is: AGGCCAAATG\n",
      "0 AGGCCAAATG motifs detected in file\n",
      "\n",
      "Seq is: TTCTGACCAC\n",
      "0 TTCTGACCAC motifs detected in file\n",
      "\n",
      "Seq is: ATAATCAAAT\n",
      "0 ATAATCAAAT motifs detected in file\n",
      "\n",
      "Seq is: ATGATGTTCC\n",
      "0 ATGATGTTCC motifs detected in file\n",
      "\n",
      "Seq is: TGTCCGGGTA\n",
      "0 TGTCCGGGTA motifs detected in file\n",
      "\n",
      "Seq is: AACTCATAAA\n",
      "0 AACTCATAAA motifs detected in file\n",
      "\n",
      "Seq is: ACAAAAGTTA\n",
      "0 ACAAAAGTTA motifs detected in file\n",
      "\n",
      "Seq is: ATCCCTCTAT\n",
      "0 ATCCCTCTAT motifs detected in file\n",
      "\n",
      "Seq is: CAACCATAAT\n",
      "0 CAACCATAAT motifs detected in file\n",
      "\n",
      "Complement found from 15831 to 15841\n",
      "Seq is: CCTAATCCTA\n",
      "1 CCTAATCCTA motifs detected in file\n",
      "\n",
      "Seq is: ACAACACTTC\n",
      "0 ACAACACTTC motifs detected in file\n",
      "\n",
      "Seq is: ATATCATGCC\n",
      "0 ATATCATGCC motifs detected in file\n",
      "\n",
      "Seq is: TACGATGAAC\n",
      "0 TACGATGAAC motifs detected in file\n",
      "\n",
      "Seq is: AGGTTACTAC\n",
      "0 AGGTTACTAC motifs detected in file\n",
      "\n",
      "Seq is: CATTTTCCCA\n",
      "0 CATTTTCCCA motifs detected in file\n",
      "\n",
      "Seq is: TCGAATGACC\n",
      "0 TCGAATGACC motifs detected in file\n",
      "\n",
      "Seq is: AACAGGAGGC\n",
      "0 AACAGGAGGC motifs detected in file\n",
      "\n",
      "Seq is: TAAGTCCAAT\n",
      "0 TAAGTCCAAT motifs detected in file\n",
      "\n",
      "Seq is: CTTACTCCTC\n",
      "0 CTTACTCCTC motifs detected in file\n",
      "\n",
      "Seq is: CAGACGCCGA\n",
      "0 CAGACGCCGA motifs detected in file\n",
      "\n",
      "Seq is: TCCTCAGTTA\n",
      "0 TCCTCAGTTA motifs detected in file\n",
      "\n",
      "Seq is: TTTCACTAAC\n",
      "0 TTTCACTAAC motifs detected in file\n",
      "\n",
      "Seq is: CGAATCACCC\n",
      "0 CGAATCACCC motifs detected in file\n",
      "\n",
      "Seq is: GCTTTATAAT\n",
      "0 GCTTTATAAT motifs detected in file\n",
      "\n",
      "Seq is: ACGAAACAAC\n",
      "0 ACGAAACAAC motifs detected in file\n",
      "\n",
      "Seq is: AAACCTATAT\n",
      "0 AAACCTATAT motifs detected in file\n",
      "\n",
      "Seq is: ACCTCCTACC\n",
      "0 ACCTCCTACC motifs detected in file\n",
      "\n",
      "Seq is: CCTAATAACG\n",
      "0 CCTAATAACG motifs detected in file\n",
      "\n",
      "Seq is: ATCCTACTCC\n",
      "0 ATCCTACTCC motifs detected in file\n",
      "\n",
      "Seq is: TACCTATCAT\n",
      "0 TACCTATCAT motifs detected in file\n",
      "\n",
      "Seq is: TATCCCGTTC\n",
      "0 TATCCCGTTC motifs detected in file\n",
      "\n",
      "Seq is: CTGCGGAGGA\n",
      "0 CTGCGGAGGA motifs detected in file\n",
      "\n",
      "Seq is: TCAAACAATC\n",
      "0 TCAAACAATC motifs detected in file\n",
      "\n",
      "Seq is: CCTGCCTAGC\n",
      "0 CCTGCCTAGC motifs detected in file\n",
      "\n",
      "Seq is: CTCTTAACAC\n",
      "0 CTCTTAACAC motifs detected in file\n",
      "\n",
      "Seq is: ATCCGCTTAT\n",
      "0 ATCCGCTTAT motifs detected in file\n",
      "\n",
      "Seq is: CCTTTATAGT\n",
      "0 CCTTTATAGT motifs detected in file\n",
      "\n",
      "Seq is: AAGCCCGAAC\n",
      "0 AAGCCCGAAC motifs detected in file\n",
      "\n",
      "Seq is: TACACCCCTC\n",
      "0 TACACCCCTC motifs detected in file\n",
      "\n",
      "Seq is: CCCACAAATT\n",
      "0 CCCACAAATT motifs detected in file\n",
      "\n",
      "Seq is: CCCCAACCGA\n",
      "0 CCCCAACCGA motifs detected in file\n",
      "\n",
      "Seq is: TCCCATATTA\n",
      "0 TCCCATATTA motifs detected in file\n",
      "\n",
      "Seq is: ACAGACCCAG\n",
      "0 ACAGACCCAG motifs detected in file\n",
      "\n",
      "Seq is: CGGATCCTCC\n",
      "0 CGGATCCTCC motifs detected in file\n",
      "\n",
      "Seq is: AGACCACTCT\n",
      "0 AGACCACTCT motifs detected in file\n",
      "\n",
      "Seq is: TATCACAATT\n",
      "0 TATCACAATT motifs detected in file\n",
      "\n",
      "Seq is: ACAGTAATTC\n",
      "0 ACAGTAATTC motifs detected in file\n",
      "\n",
      "Seq is: CTCTCTTCCT\n",
      "0 CTCTCTTCCT motifs detected in file\n",
      "\n",
      "Seq is: TCTCTTCATT\n",
      "0 TCTCTTCATT motifs detected in file\n",
      "\n",
      "Seq is: CGGCTCCCGC\n",
      "0 CGGCTCCCGC motifs detected in file\n",
      "\n",
      "Seq is: AGAAACTAAC\n",
      "0 AGAAACTAAC motifs detected in file\n",
      "\n",
      "Seq is: ACATCATTCC\n",
      "0 ACATCATTCC motifs detected in file\n",
      "\n",
      "Complement found from 15399 to 15409\n",
      "Seq is: CACCTTCCAC\n",
      "1 CACCTTCCAC motifs detected in file\n",
      "\n",
      "Seq is: TAAAATAGCC\n",
      "0 TAAAATAGCC motifs detected in file\n",
      "\n",
      "Seq is: TTACCCTCCA\n",
      "0 TTACCCTCCA motifs detected in file\n",
      "\n",
      "Seq is: CTAAGGATCC\n",
      "0 CTAAGGATCC motifs detected in file\n",
      "\n",
      "Seq is: CCCAACAAAC\n",
      "0 CCCAACAAAC motifs detected in file\n",
      "\n",
      "Seq is: TAGGGCAAAG\n",
      "0 TAGGGCAAAG motifs detected in file\n",
      "\n",
      "Seq is: CACGTTCTTA\n",
      "0 CACGTTCTTA motifs detected in file\n",
      "\n",
      "Seq is: TCCTCCACCT\n",
      "0 TCCTCCACCT motifs detected in file\n",
      "\n",
      "Seq is: CACAACGATC\n",
      "0 CACAACGATC motifs detected in file\n",
      "\n",
      "Seq is: CCGACGTTAT\n",
      "0 CCGACGTTAT motifs detected in file\n",
      "\n",
      "Seq is: TACTTCCCGT\n",
      "0 TACTTCCCGT motifs detected in file\n",
      "\n",
      "Seq is: TCTACTTCAC\n",
      "0 TCTACTTCAC motifs detected in file\n",
      "\n",
      "Seq is: TTTCCATTTC\n",
      "0 TTTCCATTTC motifs detected in file\n",
      "\n",
      "Seq is: TTAGCACACT\n",
      "0 TTAGCACACT motifs detected in file\n",
      "\n",
      "Seq is: CCCACCCTGA\n",
      "0 CCCACCCTGA motifs detected in file\n",
      "\n",
      "Seq is: CAGATGACTC\n",
      "0 CAGATGACTC motifs detected in file\n",
      "\n",
      "Seq is: ATCGGAGGAG\n",
      "0 ATCGGAGGAG motifs detected in file\n",
      "\n",
      "Seq is: TCTAAGTAAC\n",
      "0 TCTAAGTAAC motifs detected in file\n",
      "\n",
      "Seq is: TTGATCCAGA\n",
      "0 TTGATCCAGA motifs detected in file\n",
      "\n",
      "Seq is: CAGGGTTACA\n",
      "0 CAGGGTTACA motifs detected in file\n",
      "\n",
      "Seq is: TACCCTACCG\n",
      "0 TACCCTACCG motifs detected in file\n",
      "\n",
      "Seq is: CCTATCATTC\n",
      "0 CCTATCATTC motifs detected in file\n",
      "\n",
      "Complement found from 12671 to 12681\n",
      "Seq is: AAACATTAAT\n",
      "1 AAACATTAAT motifs detected in file\n",
      "\n",
      "Seq is: GACACCGGGG\n",
      "0 GACACCGGGG motifs detected in file\n",
      "\n",
      "Seq is: AGTCTTACTA\n",
      "0 AGTCTTACTA motifs detected in file\n",
      "\n",
      "Complement found from 15920 to 15930\n",
      "Seq is: TAAACCGGAG\n",
      "1 TAAACCGGAG motifs detected in file\n",
      "\n",
      "Seq is: TGCCCTCCTG\n",
      "0 TGCCCTCCTG motifs detected in file\n",
      "\n",
      "Seq is: TATCGGATAC\n",
      "0 TATCGGATAC motifs detected in file\n",
      "\n",
      "Seq is: TTCCGACAAC\n",
      "0 TTCCGACAAC motifs detected in file\n",
      "\n",
      "Seq is: GATATCAACG\n",
      "0 GATATCAACG motifs detected in file\n",
      "\n",
      "Seq is: TTCGTCCTCC\n",
      "0 TTCGTCCTCC motifs detected in file\n",
      "\n",
      "Seq is: TATTACGGCT\n",
      "0 TATTACGGCT motifs detected in file\n",
      "\n",
      "Seq is: ACAAAGTCCA\n",
      "0 ACAAAGTCCA motifs detected in file\n",
      "\n",
      "Seq is: AAGACTCATC\n",
      "0 AAGACTCATC motifs detected in file\n",
      "\n",
      "Seq is: TCTTTACTAG\n",
      "0 TCTTTACTAG motifs detected in file\n",
      "\n",
      "Seq is: GCATTATATC\n",
      "0 GCATTATATC motifs detected in file\n",
      "\n",
      "Seq is: CGGAGCGGGC\n",
      "0 CGGAGCGGGC motifs detected in file\n",
      "\n",
      "Seq is: TACACATCCT\n",
      "0 TACACATCCT motifs detected in file\n",
      "\n",
      "Seq is: TCTCCGTCTA\n",
      "0 TCTCCGTCTA motifs detected in file\n",
      "\n",
      "Seq is: TTTCTTATAA\n",
      "0 TTTCTTATAA motifs detected in file\n",
      "\n",
      "Seq is: CTCCGCGGTA\n",
      "0 CTCCGCGGTA motifs detected in file\n",
      "\n",
      "Seq is: ACCGCACTTC\n",
      "0 ACCGCACTTC motifs detected in file\n",
      "\n",
      "Seq is: CATCGCCTAC\n",
      "0 CATCGCCTAC motifs detected in file\n",
      "\n",
      "Seq is: TAAGTCGGTA\n",
      "0 TAAGTCGGTA motifs detected in file\n",
      "\n",
      "Seq is: TTAAATGCAG\n",
      "0 TTAAATGCAG motifs detected in file\n",
      "\n",
      "Seq is: AGCTCACTAC\n",
      "0 AGCTCACTAC motifs detected in file\n",
      "\n",
      "Seq is: ACCCGCTAAC\n",
      "0 ACCCGCTAAC motifs detected in file\n",
      "\n",
      "Seq is: TACTTTTCCG\n",
      "0 TACTTTTCCG motifs detected in file\n",
      "\n",
      "Seq is: CCAACTCCGC\n",
      "0 CCAACTCCGC motifs detected in file\n",
      "\n",
      "Seq is: AGACCACTCA\n",
      "0 AGACCACTCA motifs detected in file\n",
      "\n",
      "Seq is: TCACGTACCG\n",
      "0 TCACGTACCG motifs detected in file\n",
      "\n",
      "Seq is: ATCCTTATCA\n",
      "0 ATCCTTATCA motifs detected in file\n",
      "\n",
      "Seq is: GGACACCACT\n",
      "0 GGACACCACT motifs detected in file\n",
      "\n",
      "Seq is: AAACCTCCTA\n",
      "0 AAACCTCCTA motifs detected in file\n",
      "\n",
      "Seq is: GTCCGTCCGC\n",
      "0 GTCCGTCCGC motifs detected in file\n",
      "\n",
      "Seq is: GGTTCCTCAC\n",
      "0 GGTTCCTCAC motifs detected in file\n",
      "\n",
      "Seq is: TCGGCTTCAA\n",
      "0 TCGGCTTCAA motifs detected in file\n",
      "\n",
      "Seq is: AGTAGTACGC\n",
      "0 AGTAGTACGC motifs detected in file\n",
      "\n",
      "Seq is: CTCTACAACC\n",
      "0 CTCTACAACC motifs detected in file\n",
      "\n",
      "Seq is: TACCCCACCC\n",
      "0 TACCCCACCC motifs detected in file\n",
      "\n",
      "Seq is: CTCCAGCTAC\n",
      "0 CTCCAGCTAC motifs detected in file\n",
      "\n",
      "Seq is: TTACTCACCA\n",
      "0 TTACTCACCA motifs detected in file\n",
      "\n",
      "Seq is: ATTAATTAAA\n",
      "0 ATTAATTAAA motifs detected in file\n",
      "\n",
      "Seq is: ATAATCCCCC\n",
      "0 ATAATCCCCC motifs detected in file\n",
      "\n",
      "Seq is: AATCAAAACG\n",
      "0 AATCAAAACG motifs detected in file\n",
      "\n",
      "Seq is: CATAACCCCA\n",
      "0 CATAACCCCA motifs detected in file\n",
      "\n",
      "Seq is: GTAACCACAA\n",
      "0 GTAACCACAA motifs detected in file\n",
      "\n",
      "Seq is: GAACATCAAC\n",
      "0 GAACATCAAC motifs detected in file\n",
      "\n",
      "Seq is: TTTATGTTGC\n",
      "0 TTTATGTTGC motifs detected in file\n",
      "\n",
      "Seq is: TACCAAAAAG\n",
      "0 TACCAAAAAG motifs detected in file\n",
      "\n",
      "Seq is: TATAGTAACC\n",
      "0 TATAGTAACC motifs detected in file\n",
      "\n",
      "Seq is: AGCACCAACA\n",
      "0 AGCACCAACA motifs detected in file\n",
      "\n",
      "Seq is: TCAGGCACGC\n",
      "0 TCAGGCACGC motifs detected in file\n",
      "\n",
      "Seq is: TCTTATTACT\n",
      "0 TCTTATTACT motifs detected in file\n",
      "\n",
      "Seq is: ACATACGAAA\n",
      "0 ACATACGAAA motifs detected in file\n",
      "\n",
      "Complement found from 14471 to 14481\n",
      "Seq is: CAAAGACAAC\n",
      "1 CAAAGACAAC motifs detected in file\n",
      "\n",
      "Complement found from 2255 to 2265\n",
      "Seq is: TCACACCCAA\n",
      "1 TCACACCCAA motifs detected in file\n",
      "\n",
      "Seq is: ATCATTACCC\n",
      "0 ATCATTACCC motifs detected in file\n",
      "\n",
      "Seq is: CAAACACCCC\n",
      "0 CAAACACCCC motifs detected in file\n",
      "\n",
      "Seq is: AAAAGAAGAT\n",
      "0 AAAAGAAGAT motifs detected in file\n",
      "\n",
      "Seq is: TCGGAAGAGG\n",
      "0 TCGGAAGAGG motifs detected in file\n",
      "\n",
      "Seq is: ATAAATACCC\n",
      "0 ATAAATACCC motifs detected in file\n",
      "\n",
      "Seq is: CCAAATCATA\n",
      "0 CCAAATCATA motifs detected in file\n",
      "\n",
      "Seq is: ACTAACAATC\n",
      "0 ACTAACAATC motifs detected in file\n",
      "\n",
      "Seq is: GCCACACCAG\n",
      "0 GCCACACCAG motifs detected in file\n",
      "\n",
      "Seq is: CCCACACAAT\n",
      "0 CCCACACAAT motifs detected in file\n",
      "\n",
      "Seq is: AATAAGACTT\n",
      "0 AATAAGACTT motifs detected in file\n",
      "\n",
      "Complement found from 6415 to 6425\n",
      "Complement found from 16179 to 16189\n",
      "Seq is: AAAACCCCCT\n",
      "2 AAAACCCCCT motifs detected in file\n",
      "\n",
      "Seq is: CCAATATACC\n",
      "0 CCAATATACC motifs detected in file\n",
      "\n",
      "Seq is: CAAATTATCA\n",
      "0 CAAATTATCA motifs detected in file\n",
      "\n",
      "Seq is: AAAAAATTAA\n",
      "0 AAAAAATTAA motifs detected in file\n",
      "\n",
      "Seq is: ATAAATCCCC\n",
      "0 ATAAATCCCC motifs detected in file\n",
      "\n",
      "Complement found from 1698 to 1708\n",
      "Seq is: CTTACTACCA\n",
      "1 CTTACTACCA motifs detected in file\n",
      "\n",
      "Seq is: ACAGAAACCT\n",
      "0 ACAGAAACCT motifs detected in file\n",
      "\n",
      "Seq is: ATATGATGTC\n",
      "0 ATATGATGTC motifs detected in file\n",
      "\n",
      "Seq is: GCTACCGATA\n",
      "0 GCTACCGATA motifs detected in file\n",
      "\n",
      "Seq is: ACTCCTCATA\n",
      "0 ACTCCTCATA motifs detected in file\n",
      "\n",
      "Seq is: GGACTCCGTA\n",
      "0 GGACTCCGTA motifs detected in file\n",
      "\n",
      "Seq is: CCCCCAGTCC\n",
      "0 CCCCCAGTCC motifs detected in file\n",
      "\n",
      "Seq is: CCAACTCCAG\n",
      "0 CCAACTCCAG motifs detected in file\n",
      "\n",
      "Complement found from 23 to 33\n",
      "Seq is: AACCACTCAC\n",
      "1 AACCACTCAC motifs detected in file\n",
      "\n",
      "Seq is: AAAATCACCC\n",
      "0 AAAATCACCC motifs detected in file\n",
      "\n",
      "Seq is: CAATCGCTAC\n",
      "0 CAATCGCTAC motifs detected in file\n",
      "\n",
      "Seq is: CTCCATCCTA\n",
      "0 CTCCATCCTA motifs detected in file\n",
      "\n",
      "Seq is: ACCACGACAC\n",
      "0 ACCACGACAC motifs detected in file\n",
      "\n",
      "Seq is: CCACTTTCTC\n",
      "0 CCACTTTCTC motifs detected in file\n",
      "\n",
      "Seq is: ATACTACCCC\n",
      "0 ATACTACCCC motifs detected in file\n",
      "\n",
      "Seq is: ACCACCAACA\n",
      "0 ACCACCAACA motifs detected in file\n",
      "\n",
      "Seq is: CCATTTGAAA\n",
      "0 CCATTTGAAA motifs detected in file\n",
      "\n",
      "Seq is: TTATCACATC\n",
      "0 TTATCACATC motifs detected in file\n",
      "\n",
      "Seq is: CTTCGACTTA\n",
      "0 CTTCGACTTA motifs detected in file\n",
      "\n",
      "Seq is: TTAAATACTT\n",
      "0 TTAAATACTT motifs detected in file\n",
      "\n",
      "Seq is: CCTCTCCCCA\n",
      "0 CCTCTCCCCA motifs detected in file\n",
      "\n",
      "Seq is: GTCCCAACTA\n",
      "0 GTCCCAACTA motifs detected in file\n",
      "\n",
      "Complement found from 9274 to 9284\n",
      "Seq is: AGCCCTCCTA\n",
      "1 AGCCCTCCTA motifs detected in file\n",
      "\n",
      "Seq is: GGATAACCAC\n",
      "0 GGATAACCAC motifs detected in file\n",
      "\n",
      "Seq is: GCCCCCGAAA\n",
      "0 GCCCCCGAAA motifs detected in file\n",
      "\n",
      "Seq is: CATACTAATA\n",
      "0 CATACTAATA motifs detected in file\n",
      "\n",
      "Seq is: CCCGCAACTA\n",
      "0 CCCGCAACTA motifs detected in file\n",
      "\n",
      "Seq is: ATCATCATCA\n",
      "0 ATCATCATCA motifs detected in file\n",
      "\n",
      "Seq is: ATGACCAACT\n",
      "0 ATGACCAACT motifs detected in file\n",
      "\n",
      "Seq is: TGTAACAAAC\n",
      "0 TGTAACAAAC motifs detected in file\n",
      "\n",
      "Seq is: AACCACATAT\n",
      "0 AACCACATAT motifs detected in file\n",
      "\n",
      "Seq is: ATAACATTAA\n",
      "0 ATAACATTAA motifs detected in file\n",
      "\n",
      "Seq is: CTCTAACGAG\n",
      "0 CTCTAACGAG motifs detected in file\n",
      "\n",
      "Seq is: CCCCCTTATC\n",
      "0 CCCCCTTATC motifs detected in file\n",
      "\n",
      "Seq is: CAATACACTA\n",
      "0 CAATACACTA motifs detected in file\n",
      "\n",
      "Complement found from 15637 to 15647\n",
      "Seq is: ATCCTCATCC\n",
      "1 ATCCTCATCC motifs detected in file\n",
      "\n",
      "Seq is: CAATCCTACT\n",
      "0 CAATCCTACT motifs detected in file\n",
      "\n",
      "Seq is: CACCCTTCTT\n",
      "0 CACCCTTCTT motifs detected in file\n",
      "\n",
      "Seq is: CTTTCTCTCC\n",
      "0 CTTTCTCTCC motifs detected in file\n",
      "\n",
      "Seq is: TTCATTTCAA\n",
      "0 TTCATTTCAA motifs detected in file\n",
      "\n",
      "Seq is: ATTAATACGG\n",
      "0 ATTAATACGG motifs detected in file\n",
      "\n",
      "Complement found from 16161 to 16171\n",
      "Seq is: AAAAACCCAA\n",
      "1 AAAAACCCAA motifs detected in file\n",
      "\n",
      "Seq is: CTCCACTACT\n",
      "0 CTCCACTACT motifs detected in file\n",
      "\n",
      "Seq is: ACCTCCACCT\n",
      "0 ACCTCCACCT motifs detected in file\n",
      "\n",
      "Seq is: CTAAACCACG\n",
      "0 CTAAACCACG motifs detected in file\n",
      "\n",
      "Seq is: ACACTTTAAC\n",
      "0 ACACTTTAAC motifs detected in file\n",
      "\n",
      "Complement found from 7133 to 7143\n",
      "Complement found from 12443 to 12453\n",
      "Seq is: AAAATCCATT\n",
      "2 AAAATCCATT motifs detected in file\n",
      "\n",
      "Seq is: ATCGAAAAGA\n",
      "0 ATCGAAAAGA motifs detected in file\n",
      "\n",
      "Seq is: TCAGTCCAAT\n",
      "0 TCAGTCCAAT motifs detected in file\n",
      "\n",
      "Seq is: CCAGATCCTC\n",
      "0 CCAGATCCTC motifs detected in file\n",
      "\n",
      "Seq is: CTCATCCCCG\n",
      "0 CTCATCCCCG motifs detected in file\n",
      "\n",
      "Seq is: TCCAAAACCG\n",
      "0 TCCAAAACCG motifs detected in file\n",
      "\n",
      "Seq is: AGCATTCTTC\n",
      "0 AGCATTCTTC motifs detected in file\n",
      "\n",
      "Seq is: CGGATCTATC\n",
      "0 CGGATCTATC motifs detected in file\n",
      "\n",
      "Seq is: CCCTAACACG\n",
      "0 CCCTAACACG motifs detected in file\n",
      "\n",
      "Seq is: CCACACACTA\n",
      "0 CCACACACTA motifs detected in file\n",
      "\n",
      "Seq is: CGATCCCATC\n",
      "0 CGATCCCATC motifs detected in file\n",
      "\n",
      "Seq is: TTAGGCTCAT\n",
      "0 TTAGGCTCAT motifs detected in file\n",
      "\n",
      "Seq is: ACAACCTCTT\n",
      "0 ACAACCTCTT motifs detected in file\n",
      "\n",
      "Seq is: TATTTTACAC\n",
      "0 TATTTTACAC motifs detected in file\n",
      "\n",
      "Seq is: GTATCACCCC\n",
      "0 GTATCACCCC motifs detected in file\n",
      "\n",
      "Complement found from 13868 to 13878\n",
      "Seq is: TAAAATAAAA\n",
      "1 TAAAATAAAA motifs detected in file\n",
      "\n",
      "Seq is: TTCAAACAAC\n",
      "0 TTCAAACAAC motifs detected in file\n",
      "\n",
      "Seq is: CAATCCATCA\n",
      "0 CAATCCATCA motifs detected in file\n",
      "\n",
      "Seq is: ACTCCAGATC\n",
      "0 ACTCCAGATC motifs detected in file\n",
      "\n",
      "Seq is: CCGACAATCT\n",
      "0 CCGACAATCT motifs detected in file\n",
      "\n",
      "Seq is: TCAGGATCCT\n",
      "0 TCAGGATCCT motifs detected in file\n",
      "\n",
      "Seq is: TTCACTGTCG\n",
      "0 TTCACTGTCG motifs detected in file\n",
      "\n",
      "Seq is: CTCCCGACAC\n",
      "0 CTCCCGACAC motifs detected in file\n",
      "\n",
      "Seq is: TCAAAATCCA\n",
      "0 TCAAAATCCA motifs detected in file\n",
      "\n",
      "Seq is: TCTCCCCCTA\n",
      "0 TCTCCCCCTA motifs detected in file\n",
      "\n",
      "Seq is: ACAACAAACC\n",
      "0 ACAACAAACC motifs detected in file\n",
      "\n",
      "Seq is: TTCCCCCTAC\n",
      "0 TTCCCCCTAC motifs detected in file\n",
      "\n",
      "Seq is: GCCCCCTTTA\n",
      "0 GCCCCCTTTA motifs detected in file\n",
      "\n",
      "Seq is: CAACAATCAT\n",
      "0 CAACAATCAT motifs detected in file\n",
      "\n",
      "Seq is: TACTCTTTAG\n",
      "0 TACTCTTTAG motifs detected in file\n",
      "\n",
      "Seq is: GACGCTTATC\n",
      "0 GACGCTTATC motifs detected in file\n",
      "\n",
      "Seq is: CGAAGGCCGA\n",
      "0 CGAAGGCCGA motifs detected in file\n",
      "\n",
      "Seq is: CGGTCCGCAA\n",
      "0 CGGTCCGCAA motifs detected in file\n",
      "\n",
      "Seq is: ATTACCCCAA\n",
      "0 ATTACCCCAA motifs detected in file\n",
      "\n",
      "Seq is: ATCATCCCAC\n",
      "0 ATCATCCCAC motifs detected in file\n",
      "\n",
      "Seq is: CCCAATAAAA\n",
      "0 CCCAATAAAA motifs detected in file\n",
      "\n",
      "Seq is: GCAATTACAA\n",
      "0 GCAATTACAA motifs detected in file\n",
      "\n",
      "Seq is: TCATTCCCAC\n",
      "0 TCATTCCCAC motifs detected in file\n",
      "\n",
      "Seq is: CCCTTCGCTC\n",
      "0 CCCTTCGCTC motifs detected in file\n",
      "\n",
      "Seq is: CAACTGGACA\n",
      "0 CAACTGGACA motifs detected in file\n",
      "\n",
      "Seq is: ATCCCACTCT\n",
      "0 ATCCCACTCT motifs detected in file\n",
      "\n",
      "Seq is: TCTTAATAAG\n",
      "0 TCTTAATAAG motifs detected in file\n",
      "\n",
      "Seq is: CTCACGATAT\n",
      "0 CTCACGATAT motifs detected in file\n",
      "\n",
      "Seq is: CCGCGAACAG\n",
      "0 CCGCGAACAG motifs detected in file\n",
      "\n",
      "Seq is: TCCCTCCATC\n",
      "0 TCCCTCCATC motifs detected in file\n",
      "\n",
      "Complement found from 10598 to 10608\n",
      "Seq is: GCTACTCTCA\n",
      "1 GCTACTCTCA motifs detected in file\n",
      "\n",
      "Seq is: TTATCTATCC\n",
      "0 TTATCTATCC motifs detected in file\n",
      "\n",
      "Seq is: CGAGTCCGCA\n",
      "0 CGAGTCCGCA motifs detected in file\n",
      "\n",
      "Seq is: AACACATACT\n",
      "0 AACACATACT motifs detected in file\n",
      "\n",
      "Seq is: ATACAAACGC\n",
      "0 ATACAAACGC motifs detected in file\n",
      "\n",
      "Seq is: CAAAGCTACT\n",
      "0 CAAAGCTACT motifs detected in file\n",
      "\n",
      "Seq is: ACACCAGAAA\n",
      "0 ACACCAGAAA motifs detected in file\n",
      "\n",
      "Seq is: CCTCATCTTT\n",
      "0 CCTCATCTTT motifs detected in file\n",
      "\n",
      "Seq is: GGACACTCCT\n",
      "0 GGACACTCCT motifs detected in file\n",
      "\n",
      "Seq is: TTCCATAAGG\n",
      "0 TTCCATAAGG motifs detected in file\n",
      "\n",
      "Seq is: ACGATTACGA\n",
      "0 ACGATTACGA motifs detected in file\n",
      "\n",
      "Seq is: TCCGACGGTT\n",
      "0 TCCGACGGTT motifs detected in file\n",
      "\n",
      "Seq is: ACCACTCCCT\n",
      "0 ACCACTCCCT motifs detected in file\n",
      "\n",
      "Seq is: CCAACTTCAC\n",
      "0 CCAACTTCAC motifs detected in file\n",
      "\n",
      "Complement found from 4219 to 4229\n",
      "Seq is: TCTCCATACC\n",
      "1 TCTCCATACC motifs detected in file\n",
      "\n",
      "Seq is: AAAACTCATC\n",
      "0 AAAACTCATC motifs detected in file\n",
      "\n",
      "Seq is: AGGAGGATAA\n",
      "0 AGGAGGATAA motifs detected in file\n",
      "\n",
      "Seq is: AAAGCTTATA\n",
      "0 AAAGCTTATA motifs detected in file\n",
      "\n",
      "Seq is: GAACAAGTAA\n",
      "0 GAACAAGTAA motifs detected in file\n",
      "\n",
      "Seq is: CAATTCCAAC\n",
      "0 CAATTCCAAC motifs detected in file\n",
      "\n",
      "Seq is: ACCTACTACC\n",
      "0 ACCTACTACC motifs detected in file\n",
      "\n",
      "Seq is: TGGGCCTCGT\n",
      "0 TGGGCCTCGT motifs detected in file\n",
      "\n",
      "Seq is: GTATTTATCA\n",
      "0 GTATTTATCA motifs detected in file\n",
      "\n",
      "Seq is: TACCGAAACT\n",
      "0 TACCGAAACT motifs detected in file\n",
      "\n",
      "Seq is: TCTTCCGCAC\n",
      "0 TCTTCCGCAC motifs detected in file\n",
      "\n",
      "Seq is: CCATGTCTAC\n",
      "0 CCATGTCTAC motifs detected in file\n",
      "\n",
      "Seq is: ACGTCCTTAC\n",
      "0 ACGTCCTTAC motifs detected in file\n",
      "\n",
      "Seq is: GATCCACACC\n",
      "0 GATCCACACC motifs detected in file\n",
      "\n",
      "Seq is: AACCAACTAC\n",
      "0 AACCAACTAC motifs detected in file\n",
      "\n",
      "Seq is: GGCTAACATT\n",
      "0 GGCTAACATT motifs detected in file\n",
      "\n",
      "Seq is: GATAATACTC\n",
      "0 GATAATACTC motifs detected in file\n",
      "\n",
      "Seq is: AGGATCAACT\n",
      "0 AGGATCAACT motifs detected in file\n",
      "\n",
      "Seq is: GAACTTCACC\n",
      "0 GAACTTCACC motifs detected in file\n",
      "\n",
      "Seq is: TCTTCCGATG\n",
      "0 TCTTCCGATG motifs detected in file\n",
      "\n",
      "Seq is: CTAAAAAAAC\n",
      "0 CTAAAAAAAC motifs detected in file\n",
      "\n",
      "Seq is: TACAGTAAAA\n",
      "0 TACAGTAAAA motifs detected in file\n",
      "\n",
      "Seq is: CACATTCCCG\n",
      "0 CACATTCCCG motifs detected in file\n",
      "\n",
      "Seq is: CGTCTGACGA\n",
      "0 CGTCTGACGA motifs detected in file\n",
      "\n",
      "Seq is: CGCTTGTCTC\n",
      "0 CGCTTGTCTC motifs detected in file\n",
      "\n",
      "Seq is: ACCACTATCG\n",
      "0 ACCACTATCG motifs detected in file\n",
      "\n",
      "Seq is: CGGATTCGTA\n",
      "0 CGGATTCGTA motifs detected in file\n",
      "\n",
      "Seq is: TCACAATCTC\n",
      "0 TCACAATCTC motifs detected in file\n",
      "\n",
      "Seq is: AAACCTAATC\n",
      "0 AAACCTAATC motifs detected in file\n",
      "\n",
      "Seq is: ACCCGATAAA\n",
      "0 ACCCGATAAA motifs detected in file\n",
      "\n",
      "Seq is: AGACGATCCC\n",
      "0 AGACGATCCC motifs detected in file\n",
      "\n",
      "Seq is: CCACCTTCGC\n",
      "0 CCACCTTCGC motifs detected in file\n",
      "\n",
      "Complement found from 8970 to 8980\n",
      "Seq is: CTACTCATTC\n",
      "1 CTACTCATTC motifs detected in file\n",
      "\n",
      "Seq is: TTCTAAGGAC\n",
      "0 TTCTAAGGAC motifs detected in file\n",
      "\n",
      "Seq is: GATGTTGATA\n",
      "0 GATGTTGATA motifs detected in file\n",
      "\n",
      "Seq is: TCACGAACTC\n",
      "0 TCACGAACTC motifs detected in file\n",
      "\n",
      "Seq is: ACCTCATCCC\n",
      "0 ACCTCATCCC motifs detected in file\n",
      "\n",
      "Seq is: GACTCTGACC\n",
      "0 GACTCTGACC motifs detected in file\n",
      "\n",
      "Seq is: CCACCCCGGA\n",
      "0 CCACCCCGGA motifs detected in file\n",
      "\n",
      "Seq is: AGATACCGAC\n",
      "0 AGATACCGAC motifs detected in file\n",
      "\n",
      "Seq is: TCCCCTCAGT\n",
      "0 TCCCCTCAGT motifs detected in file\n",
      "\n",
      "Seq is: CCCCACCTCT\n",
      "0 CCCCACCTCT motifs detected in file\n",
      "\n",
      "Seq is: GGATTAACCC\n",
      "0 GGATTAACCC motifs detected in file\n",
      "\n",
      "Seq is: GACTAAACGG\n",
      "0 GACTAAACGG motifs detected in file\n",
      "\n",
      "Seq is: ACGACGACGA\n",
      "0 ACGACGACGA motifs detected in file\n",
      "\n",
      "Seq is: TCCTCCTCCG\n",
      "0 TCCTCCTCCG motifs detected in file\n",
      "\n",
      "Seq is: GATCATCACC\n",
      "0 GATCATCACC motifs detected in file\n",
      "\n",
      "Seq is: CCACTCCGAA\n",
      "0 CCACTCCGAA motifs detected in file\n",
      "\n",
      "Seq is: CCTAATCGCA\n",
      "0 CCTAATCGCA motifs detected in file\n",
      "\n",
      "Seq is: AATCTTCCCG\n",
      "0 AATCTTCCCG motifs detected in file\n",
      "\n",
      "Complement found from 6191 to 6201\n",
      "Seq is: ATAAACAACA\n",
      "1 ATAAACAACA motifs detected in file\n",
      "\n",
      "Seq is: CCCAGAGTAC\n",
      "0 CCCAGAGTAC motifs detected in file\n",
      "\n",
      "Complement found from 1228 to 1238\n",
      "Seq is: TCAACCTCAC\n",
      "1 TCAACCTCAC motifs detected in file\n",
      "\n",
      "Complement found from 9455 to 9465\n",
      "Seq is: ATCCTATTTA\n",
      "1 ATCCTATTTA motifs detected in file\n",
      "\n",
      "Seq is: GTACGATTCC\n",
      "0 GTACGATTCC motifs detected in file\n",
      "\n",
      "Seq is: GCTCCTACTT\n",
      "0 GCTCCTACTT motifs detected in file\n",
      "\n",
      "Seq is: TGGCTATAGC\n",
      "0 TGGCTATAGC motifs detected in file\n",
      "\n",
      "Seq is: GGCTATGCCA\n",
      "0 GGCTATGCCA motifs detected in file\n",
      "\n",
      "Seq is: ACATATCCTA\n",
      "0 ACATATCCTA motifs detected in file\n",
      "\n",
      "Seq is: ACGAACTTAC\n",
      "0 ACGAACTTAC motifs detected in file\n",
      "\n",
      "Seq is: CGACGACACA\n",
      "0 CGACGACACA motifs detected in file\n",
      "\n",
      "Seq is: ACCGTAGACG\n",
      "0 ACCGTAGACG motifs detected in file\n",
      "\n",
      "Seq is: AGCCCGCATA\n",
      "0 AGCCCGCATA motifs detected in file\n",
      "\n",
      "Seq is: GTAGTTGACT\n",
      "0 GTAGTTGACT motifs detected in file\n",
      "\n",
      "Seq is: ACTCGTTCTT\n",
      "0 ACTCGTTCTT motifs detected in file\n",
      "\n",
      "Seq is: CCTATATTAA\n",
      "0 CCTATATTAA motifs detected in file\n",
      "\n",
      "Seq is: GGATGCGGGA\n",
      "0 GGATGCGGGA motifs detected in file\n",
      "\n",
      "Seq is: GAGTCGGCTA\n",
      "0 GAGTCGGCTA motifs detected in file\n",
      "\n",
      "Seq is: CTTGTCAACC\n",
      "0 CTTGTCAACC motifs detected in file\n",
      "\n",
      "Seq is: TTATCCAACA\n",
      "0 TTATCCAACA motifs detected in file\n",
      "\n",
      "Seq is: ATCGCCATTG\n",
      "0 ATCGCCATTG motifs detected in file\n",
      "\n",
      "Seq is: ATTCTAATCA\n",
      "0 ATTCTAATCA motifs detected in file\n",
      "\n",
      "Seq is: TACCATTAAT\n",
      "0 TACCATTAAT motifs detected in file\n",
      "\n",
      "Seq is: CCTTCTACTC\n",
      "0 CCTTCTACTC motifs detected in file\n",
      "\n",
      "Complement found from 8819 to 8829\n",
      "Seq is: ATCTATAAAC\n",
      "1 ATCTATAAAC motifs detected in file\n",
      "\n",
      "Seq is: TTCTTGACTA\n",
      "0 TTCTTGACTA motifs detected in file\n",
      "\n",
      "Seq is: ATTACAAACC\n",
      "0 ATTACAAACC motifs detected in file\n",
      "\n",
      "Seq is: CAGACTCAAA\n",
      "0 CAGACTCAAA motifs detected in file\n",
      "\n",
      "Seq is: TATATAGTGT\n",
      "0 TATATAGTGT motifs detected in file\n",
      "\n",
      "Seq is: CACTCTTAAG\n",
      "0 CACTCTTAAG motifs detected in file\n",
      "\n",
      "Seq is: ATACTACCTG\n",
      "0 ATACTACCTG motifs detected in file\n",
      "\n",
      "Seq is: GTACATTGCT\n",
      "0 GTACATTGCT motifs detected in file\n",
      "\n",
      "Seq is: TGTTACGATG\n",
      "0 TGTTACGATG motifs detected in file\n",
      "\n",
      "Seq is: TCCCTACTTA\n",
      "0 TCCCTACTTA motifs detected in file\n",
      "\n",
      "Seq is: TAATACCTCT\n",
      "0 TAATACCTCT motifs detected in file\n",
      "\n",
      "Seq is: TCATCAGATC\n",
      "0 TCATCAGATC motifs detected in file\n",
      "\n",
      "Seq is: AAACTTCGAA\n",
      "0 AAACTTCGAA motifs detected in file\n",
      "\n",
      "Seq is: TCCCTCTCGA\n",
      "0 TCCCTCTCGA motifs detected in file\n",
      "\n",
      "Seq is: CCCAACAAAC\n",
      "0 CCCAACAAAC motifs detected in file\n",
      "\n",
      "Seq is: CCAACACCGA\n",
      "0 CCAACACCGA motifs detected in file\n",
      "\n",
      "Seq is: GTCACAGTCA\n",
      "0 GTCACAGTCA motifs detected in file\n",
      "\n",
      "Seq is: AGCTCTATTA\n",
      "0 AGCTCTATTA motifs detected in file\n",
      "\n",
      "Seq is: TTGAAGAACC\n",
      "0 TTGAAGAACC motifs detected in file\n",
      "\n",
      "Seq is: AGATCCGTGT\n",
      "0 AGATCCGTGT motifs detected in file\n",
      "\n",
      "Seq is: ACTTATAACA\n",
      "0 ACTTATAACA motifs detected in file\n",
      "\n",
      "Seq is: ACACCCCTTC\n",
      "0 ACACCCCTTC motifs detected in file\n",
      "\n",
      "Seq is: TCTGACTATT\n",
      "0 TCTGACTATT motifs detected in file\n",
      "\n",
      "Seq is: ATTTCCACCT\n",
      "0 ATTTCCACCT motifs detected in file\n",
      "\n",
      "Seq is: ACGCTGTTAC\n",
      "0 ACGCTGTTAC motifs detected in file\n",
      "\n",
      "Seq is: CTAAAATGTA\n",
      "0 CTAAAATGTA motifs detected in file\n",
      "\n",
      "Complement found from 8402 to 8412\n",
      "Seq is: TTACCCCCAT\n",
      "1 TTACCCCCAT motifs detected in file\n",
      "\n",
      "Seq is: ACTCAAAAAA\n",
      "0 ACTCAAAAAA motifs detected in file\n",
      "\n",
      "Seq is: AACAATCCCA\n",
      "0 AACAATCCCA motifs detected in file\n",
      "\n",
      "Seq is: ATTGCTCCCA\n",
      "0 ATTGCTCCCA motifs detected in file\n",
      "\n",
      "Seq is: CCATTCCTAC\n",
      "0 CCATTCCTAC motifs detected in file\n",
      "\n",
      "Seq is: CCCCCTTAAT\n",
      "0 CCCCCTTAAT motifs detected in file\n",
      "\n",
      "Seq is: CCCTTCAGTC\n",
      "0 CCCTTCAGTC motifs detected in file\n",
      "\n",
      "Seq is: CCAATCCCAC\n",
      "0 CCAATCCCAC motifs detected in file\n",
      "\n",
      "Seq is: CAATATCATC\n",
      "0 CAATATCATC motifs detected in file\n",
      "\n",
      "Seq is: ACACGTACCA\n",
      "0 ACACGTACCA motifs detected in file\n",
      "\n",
      "Seq is: ATAATGAAAA\n",
      "0 ATAATGAAAA motifs detected in file\n",
      "\n",
      "Complement found from 1100 to 1110\n",
      "Seq is: TAAACCTCAA\n",
      "1 TAAACCTCAA motifs detected in file\n",
      "\n",
      "Seq is: CGTGGTTTTA\n",
      "0 CGTGGTTTTA motifs detected in file\n",
      "\n",
      "Seq is: AAAACCCCGG\n",
      "0 AAAACCCCGG motifs detected in file\n",
      "\n",
      "Seq is: ATTCTGGTTA\n",
      "0 ATTCTGGTTA motifs detected in file\n",
      "\n",
      "Seq is: CCTATCGACA\n",
      "0 CCTATCGACA motifs detected in file\n",
      "\n",
      "Seq is: ATAGGAAATT\n",
      "0 ATAGGAAATT motifs detected in file\n",
      "\n",
      "Seq is: TTCAACTCTT\n",
      "0 TTCAACTCTT motifs detected in file\n",
      "\n",
      "Seq is: TCGGTACAAC\n",
      "0 TCGGTACAAC motifs detected in file\n",
      "\n",
      "Seq is: AATCTGTACC\n",
      "0 AATCTGTACC motifs detected in file\n",
      "\n",
      "Seq is: CCCGTACTCA\n",
      "0 CCCGTACTCA motifs detected in file\n",
      "\n",
      "Seq is: ATCGTCAAGA\n",
      "0 ATCGTCAAGA motifs detected in file\n",
      "\n",
      "Seq is: ACACTCGAAA\n",
      "0 ACACTCGAAA motifs detected in file\n",
      "\n",
      "Seq is: GAGCCATTTA\n",
      "0 GAGCCATTTA motifs detected in file\n",
      "\n",
      "Seq is: TTCCCCAGCA\n",
      "0 TTCCCCAGCA motifs detected in file\n",
      "\n",
      "Seq is: TTCGGAGACA\n",
      "0 TTCGGAGACA motifs detected in file\n",
      "\n",
      "Seq is: ACAGTCTAAG\n",
      "0 ACAGTCTAAG motifs detected in file\n",
      "\n",
      "Seq is: TGTTAGACTA\n",
      "0 TGTTAGACTA motifs detected in file\n",
      "\n",
      "Seq is: CAAAACCAAT\n",
      "0 CAAAACCAAT motifs detected in file\n",
      "\n",
      "Seq is: TTGATATAAA\n",
      "0 TTGATATAAA motifs detected in file\n",
      "\n",
      "Seq is: TGTTCTCCTT\n",
      "0 TGTTCTCCTT motifs detected in file\n",
      "\n",
      "Seq is: TTGGGCCATT\n",
      "0 TTGGGCCATT motifs detected in file\n",
      "\n",
      "Seq is: ACTACAGCCC\n",
      "0 ACTACAGCCC motifs detected in file\n",
      "\n",
      "Seq is: CAACTCCCTA\n",
      "0 CAACTCCCTA motifs detected in file\n",
      "\n",
      "Seq is: TCCTCCTCTT\n",
      "0 TCCTCCTCTT motifs detected in file\n",
      "\n",
      "Seq is: ACCCCCTATC\n",
      "0 ACCCCCTATC motifs detected in file\n",
      "\n",
      "Seq is: CACATACTTG\n",
      "0 CACATACTTG motifs detected in file\n",
      "\n",
      "Seq is: TACTCCCACA\n",
      "0 TACTCCCACA motifs detected in file\n",
      "\n",
      "Complement found from 1935 to 1945\n",
      "Seq is: AAAGAGCACA\n",
      "1 AAAGAGCACA motifs detected in file\n",
      "\n",
      "Seq is: CTTACTCCCA\n",
      "0 CTTACTCCCA motifs detected in file\n",
      "\n",
      "Seq is: AAATACAACA\n",
      "0 AAATACAACA motifs detected in file\n",
      "\n",
      "Seq is: ATTACACCAC\n",
      "0 ATTACACCAC motifs detected in file\n",
      "\n",
      "Seq is: CCACTCACTC\n",
      "0 CCACTCACTC motifs detected in file\n",
      "\n",
      "Seq is: GGGGTAACAC\n",
      "0 GGGGTAACAC motifs detected in file\n",
      "\n",
      "Seq is: AACACCATTT\n",
      "0 AACACCATTT motifs detected in file\n",
      "\n",
      "Seq is: ATACATCTCC\n",
      "0 ATACATCTCC motifs detected in file\n",
      "\n",
      "Seq is: CTCATATCCC\n",
      "0 CTCATATCCC motifs detected in file\n",
      "\n",
      "Seq is: GACACTGATC\n",
      "0 GACACTGATC motifs detected in file\n",
      "\n",
      "Seq is: ATACAACTCA\n",
      "0 ATACAACTCA motifs detected in file\n",
      "\n",
      "Seq is: GGACATTCAT\n",
      "0 GGACATTCAT motifs detected in file\n",
      "\n",
      "Seq is: CCTCTCACTA\n",
      "0 CCTCTCACTA motifs detected in file\n",
      "\n",
      "Seq is: TAAACTAGTC\n",
      "0 TAAACTAGTC motifs detected in file\n",
      "\n",
      "Seq is: CTCTTGCACC\n",
      "0 CTCTTGCACC motifs detected in file\n",
      "\n",
      "Seq is: AATGATCGTG\n",
      "0 AATGATCGTG motifs detected in file\n",
      "\n",
      "Seq is: TCTCTCAAGA\n",
      "0 TCTCTCAAGA motifs detected in file\n",
      "\n",
      "Seq is: GGGTCATCCA\n",
      "0 GGGTCATCCA motifs detected in file\n",
      "\n",
      "Seq is: ATTATCACCC\n",
      "0 ATTATCACCC motifs detected in file\n",
      "\n",
      "Seq is: CCCATTCCGC\n",
      "0 CCCATTCCGC motifs detected in file\n",
      "\n",
      "Seq is: TCCAATCGCT\n",
      "0 TCCAATCGCT motifs detected in file\n",
      "\n",
      "Seq is: CCGAACGATC\n",
      "0 CCGAACGATC motifs detected in file\n",
      "\n",
      "Seq is: TTCAGTAGTT\n",
      "0 TTCAGTAGTT motifs detected in file\n",
      "\n",
      "Seq is: TTTCGATAAT\n",
      "0 TTTCGATAAT motifs detected in file\n",
      "\n",
      "Seq is: CACCCTCATC\n",
      "0 CACCCTCATC motifs detected in file\n",
      "\n",
      "Seq is: TCAAACTTCA\n",
      "0 TCAAACTTCA motifs detected in file\n",
      "\n",
      "Seq is: GGAACTCTCT\n",
      "0 GGAACTCTCT motifs detected in file\n",
      "\n",
      "Complement found from 10953 to 10963\n",
      "Seq is: CCTAATACTA\n",
      "1 CCTAATACTA motifs detected in file\n",
      "\n",
      "Seq is: CGCTGACACT\n",
      "0 CGCTGACACT motifs detected in file\n",
      "\n",
      "Complement found from 4651 to 4661\n",
      "Seq is: CACGCAAGCA\n",
      "1 CACGCAAGCA motifs detected in file\n",
      "\n",
      "Complement found from 3672 to 3682\n",
      "Seq is: TCAAACTCAA\n",
      "1 TCAAACTCAA motifs detected in file\n",
      "\n",
      "Seq is: ACGATCCGTC\n",
      "0 ACGATCCGTC motifs detected in file\n",
      "\n",
      "Seq is: TTATCATTAC\n",
      "0 TTATCATTAC motifs detected in file\n",
      "\n",
      "Seq is: TCCTACATTC\n",
      "0 TCCTACATTC motifs detected in file\n",
      "\n",
      "Seq is: GGGCACCCGC\n",
      "0 GGGCACCCGC motifs detected in file\n",
      "\n",
      "Seq is: TAATACTCTT\n",
      "0 TAATACTCTT motifs detected in file\n",
      "\n",
      "Seq is: ACTGACGCGG\n",
      "0 ACTGACGCGG motifs detected in file\n",
      "\n",
      "Seq is: CCACTTCGAA\n",
      "0 CCACTTCGAA motifs detected in file\n",
      "\n",
      "Seq is: GTCCCCCAAA\n",
      "0 GTCCCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: CCTACTCTTA\n",
      "0 CCTACTCTTA motifs detected in file\n",
      "\n",
      "Seq is: CCGACAATGA\n",
      "0 CCGACAATGA motifs detected in file\n",
      "\n",
      "Seq is: TGCTCCCGAT\n",
      "0 TGCTCCCGAT motifs detected in file\n",
      "\n",
      "Complement found from 7913 to 7923\n",
      "Seq is: ACACCGACTA\n",
      "1 ACACCGACTA motifs detected in file\n",
      "\n",
      "Seq is: ACTTCTCATA\n",
      "0 ACTTCTCATA motifs detected in file\n",
      "\n",
      "Seq is: CGTTACTCGC\n",
      "0 CGTTACTCGC motifs detected in file\n",
      "\n",
      "Seq is: TAAAATCCAG\n",
      "0 TAAAATCCAG motifs detected in file\n",
      "\n",
      "Seq is: ACAAACAGCA\n",
      "0 ACAAACAGCA motifs detected in file\n",
      "\n",
      "Seq is: TCCGTCTACC\n",
      "0 TCCGTCTACC motifs detected in file\n",
      "\n",
      "Seq is: TCGAACAATA\n",
      "0 TCGAACAATA motifs detected in file\n",
      "\n",
      "Seq is: TTAATACGGA\n",
      "0 TTAATACGGA motifs detected in file\n",
      "\n",
      "Seq is: GTATCCCTAT\n",
      "0 GTATCCCTAT motifs detected in file\n",
      "\n",
      "Seq is: CATGTTCCTT\n",
      "0 CATGTTCCTT motifs detected in file\n",
      "\n",
      "Seq is: CCCCATCCGA\n",
      "0 CCCCATCCGA motifs detected in file\n",
      "\n",
      "Seq is: TACACAAAAC\n",
      "0 TACACAAAAC motifs detected in file\n",
      "\n",
      "Seq is: AGTCCCCCAA\n",
      "0 AGTCCCCCAA motifs detected in file\n",
      "\n",
      "Seq is: CTCTTACTCA\n",
      "0 CTCTTACTCA motifs detected in file\n",
      "\n",
      "Seq is: CACTCCGCAT\n",
      "0 CACTCCGCAT motifs detected in file\n",
      "\n",
      "Seq is: AATATGGTAT\n",
      "0 AATATGGTAT motifs detected in file\n",
      "\n",
      "Seq is: CGGCGGATCA\n",
      "0 CGGCGGATCA motifs detected in file\n",
      "\n",
      "Seq is: AAATTCTCAT\n",
      "0 AAATTCTCAT motifs detected in file\n",
      "\n",
      "Seq is: GACGCCGTTC\n",
      "0 GACGCCGTTC motifs detected in file\n",
      "\n",
      "Seq is: ATGATAACTG\n",
      "0 ATGATAACTG motifs detected in file\n",
      "\n",
      "Seq is: GGTCGCTACC\n",
      "0 GGTCGCTACC motifs detected in file\n",
      "\n",
      "Seq is: CCCGAAGCTG\n",
      "0 CCCGAAGCTG motifs detected in file\n",
      "\n",
      "Seq is: TACCCGAAAT\n",
      "0 TACCCGAAAT motifs detected in file\n",
      "\n",
      "Seq is: CCCTCAGTAT\n",
      "0 CCCTCAGTAT motifs detected in file\n",
      "\n",
      "Seq is: TCACCTCAGG\n",
      "0 TCACCTCAGG motifs detected in file\n",
      "\n",
      "Seq is: CATTTCTCCA\n",
      "0 CATTTCTCCA motifs detected in file\n",
      "\n",
      "Seq is: TAGAAATGAT\n",
      "0 TAGAAATGAT motifs detected in file\n",
      "\n",
      "Seq is: ATTTTCGATA\n",
      "0 ATTTTCGATA motifs detected in file\n",
      "\n",
      "Complement found from 7410 to 7420\n",
      "Seq is: ACACATTCGA\n",
      "1 ACACATTCGA motifs detected in file\n",
      "\n",
      "Seq is: TCAGTATAAT\n",
      "0 TCAGTATAAT motifs detected in file\n",
      "\n",
      "Complement found from 8638 to 8648\n",
      "Complement found from 16074 to 16084\n",
      "Seq is: TCAACAACCG\n",
      "2 TCAACAACCG motifs detected in file\n",
      "\n",
      "Seq is: AGTCCTCAAA\n",
      "0 AGTCCTCAAA motifs detected in file\n",
      "\n",
      "Seq is: CTATCAAGAA\n",
      "0 CTATCAAGAA motifs detected in file\n",
      "\n",
      "Seq is: CCCGTCACTC\n",
      "0 CCCGTCACTC motifs detected in file\n",
      "\n",
      "Seq is: TCACTCATCA\n",
      "0 TCACTCATCA motifs detected in file\n",
      "\n",
      "Seq is: TCTTACAAAT\n",
      "0 TCTTACAAAT motifs detected in file\n",
      "\n",
      "Seq is: CACTCGGATC\n",
      "0 CACTCGGATC motifs detected in file\n",
      "\n",
      "Complement found from 7181 to 7191\n",
      "Seq is: CCACAACACT\n",
      "1 CCACAACACT motifs detected in file\n",
      "\n",
      "Seq is: CACATTTAAT\n",
      "0 CACATTTAAT motifs detected in file\n",
      "\n",
      "Complement found from 5467 to 5477\n",
      "Seq is: CACGCTACTC\n",
      "1 CACGCTACTC motifs detected in file\n",
      "\n",
      "Seq is: ATCCCCTTCC\n",
      "0 ATCCCCTTCC motifs detected in file\n",
      "\n",
      "Seq is: CTCGGATGAT\n",
      "0 CTCGGATGAT motifs detected in file\n",
      "\n",
      "Seq is: CCCACATCTT\n",
      "0 CCCACATCTT motifs detected in file\n",
      "\n",
      "Seq is: ATCCTTCATA\n",
      "0 ATCCTTCATA motifs detected in file\n",
      "\n",
      "Seq is: CACGGACGCA\n",
      "0 CACGGACGCA motifs detected in file\n",
      "\n",
      "Seq is: AGTCCGCAAG\n",
      "0 AGTCCGCAAG motifs detected in file\n",
      "\n",
      "Seq is: ACCGACCAAC\n",
      "0 ACCGACCAAC motifs detected in file\n",
      "\n",
      "Seq is: GGAGTAGCCC\n",
      "0 GGAGTAGCCC motifs detected in file\n",
      "\n",
      "Seq is: ACTACTATCG\n",
      "0 ACTACTATCG motifs detected in file\n",
      "\n",
      "Seq is: GTTCCACCCC\n",
      "0 GTTCCACCCC motifs detected in file\n",
      "\n",
      "Seq is: TATTCACACC\n",
      "0 TATTCACACC motifs detected in file\n",
      "\n",
      "Seq is: AAAGCTTCTT\n",
      "0 AAAGCTTCTT motifs detected in file\n",
      "\n",
      "Seq is: CTATATTTTA\n",
      "0 CTATATTTTA motifs detected in file\n",
      "\n",
      "Complement found from 14214 to 14224\n",
      "Seq is: TACTAATCAA\n",
      "1 TACTAATCAA motifs detected in file\n",
      "\n",
      "Seq is: GACACCGACA\n",
      "0 GACACCGACA motifs detected in file\n",
      "\n",
      "Seq is: CTTACAATAT\n",
      "0 CTTACAATAT motifs detected in file\n",
      "\n",
      "Seq is: TAATTCCTCT\n",
      "0 TAATTCCTCT motifs detected in file\n",
      "\n",
      "Seq is: AAACATCCCT\n",
      "0 AAACATCCCT motifs detected in file\n",
      "\n",
      "Complement found from 11101 to 11111\n",
      "Seq is: CTAATCATAT\n",
      "1 CTAATCATAT motifs detected in file\n",
      "\n",
      "Seq is: CTCTCCATCT\n",
      "0 CTCTCCATCT motifs detected in file\n",
      "\n",
      "Seq is: CAAAAAAAGC\n",
      "0 CAAAAAAAGC motifs detected in file\n",
      "\n",
      "Seq is: ACTATCACCA\n",
      "0 ACTATCACCA motifs detected in file\n",
      "\n",
      "Seq is: AGTGACCTAT\n",
      "0 AGTGACCTAT motifs detected in file\n",
      "\n",
      "Seq is: TCACCGCAAC\n",
      "0 TCACCGCAAC motifs detected in file\n",
      "\n",
      "Seq is: CGAACGGTAC\n",
      "0 CGAACGGTAC motifs detected in file\n",
      "\n",
      "Seq is: TAACACTCCC\n",
      "0 TAACACTCCC motifs detected in file\n",
      "\n",
      "Seq is: CATCCTCAGT\n",
      "0 CATCCTCAGT motifs detected in file\n",
      "\n",
      "Seq is: CCATCAATCA\n",
      "0 CCATCAATCA motifs detected in file\n",
      "\n",
      "Seq is: TAATCCTCCC\n",
      "0 TAATCCTCCC motifs detected in file\n",
      "\n",
      "Seq is: CCCAACAATC\n",
      "0 CCCAACAATC motifs detected in file\n",
      "\n",
      "Seq is: CCCCAGCCTC\n",
      "0 CCCCAGCCTC motifs detected in file\n",
      "\n",
      "Seq is: CTTTTCCAAC\n",
      "0 CTTTTCCAAC motifs detected in file\n",
      "\n",
      "Seq is: CCCTTGTCGA\n",
      "0 CCCTTGTCGA motifs detected in file\n",
      "\n",
      "Seq is: TTTATCCAAC\n",
      "0 TTTATCCAAC motifs detected in file\n",
      "\n",
      "Seq is: AACAACTAAA\n",
      "0 AACAACTAAA motifs detected in file\n",
      "\n",
      "Seq is: CCAATTTTTT\n",
      "0 CCAATTTTTT motifs detected in file\n",
      "\n",
      "Seq is: ATCATCTCCC\n",
      "0 ATCATCTCCC motifs detected in file\n",
      "\n",
      "Seq is: TACTACGATT\n",
      "0 TACTACGATT motifs detected in file\n",
      "\n",
      "Seq is: ATTAATCCGA\n",
      "0 ATTAATCCGA motifs detected in file\n",
      "\n",
      "Seq is: CACCCACCAA\n",
      "0 CACCCACCAA motifs detected in file\n",
      "\n",
      "Seq is: CACAACTAAG\n",
      "0 CACAACTAAG motifs detected in file\n",
      "\n",
      "Seq is: TTTAATACAC\n",
      "0 TTTAATACAC motifs detected in file\n",
      "\n",
      "Complement found from 2129 to 2139\n",
      "Seq is: AAAAAACCTT\n",
      "1 AAAAAACCTT motifs detected in file\n",
      "\n",
      "Seq is: TCAGTACAGT\n",
      "0 TCAGTACAGT motifs detected in file\n",
      "\n",
      "Seq is: CACCATCATT\n",
      "0 CACCATCATT motifs detected in file\n",
      "\n",
      "Seq is: ATATTAACAA\n",
      "0 ATATTAACAA motifs detected in file\n",
      "\n",
      "Seq is: CCCTGCTAAT\n",
      "0 CCCTGCTAAT motifs detected in file\n",
      "\n",
      "Seq is: CAAAATCGTA\n",
      "0 CAAAATCGTA motifs detected in file\n",
      "\n",
      "Seq is: ACCTCATCCA\n",
      "0 ACCTCATCCA motifs detected in file\n",
      "\n",
      "Seq is: AATCCAATAC\n",
      "0 AATCCAATAC motifs detected in file\n",
      "\n",
      "Seq is: ATGCATCAGA\n",
      "0 ATGCATCAGA motifs detected in file\n",
      "\n",
      "Seq is: TCCGGTATAC\n",
      "0 TCCGGTATAC motifs detected in file\n",
      "\n",
      "Seq is: ACAACCTCTA\n",
      "0 ACAACCTCTA motifs detected in file\n",
      "\n",
      "Seq is: ACTCTGATCA\n",
      "0 ACTCTGATCA motifs detected in file\n",
      "\n",
      "Seq is: TCCCGATCCG\n",
      "0 TCCCGATCCG motifs detected in file\n",
      "\n",
      "Seq is: GGTGGCGACG\n",
      "0 GGTGGCGACG motifs detected in file\n",
      "\n",
      "Seq is: AAGCGTCCGC\n",
      "0 AAGCGTCCGC motifs detected in file\n",
      "\n",
      "Seq is: CGTTTCTGAT\n",
      "0 CGTTTCTGAT motifs detected in file\n",
      "\n",
      "Seq is: CATACCGTTA\n",
      "0 CATACCGTTA motifs detected in file\n",
      "\n",
      "Seq is: TCCGTGTTAT\n",
      "0 TCCGTGTTAT motifs detected in file\n",
      "\n",
      "Seq is: AACCGATTCT\n",
      "0 AACCGATTCT motifs detected in file\n",
      "\n",
      "Complement found from 16259 to 16269\n",
      "Seq is: CCCTCACCCA\n",
      "1 CCCTCACCCA motifs detected in file\n",
      "\n",
      "Seq is: CAACTCCCAA\n",
      "0 CAACTCCCAA motifs detected in file\n",
      "\n",
      "Complement found from 13568 to 13578\n",
      "Seq is: TACTCTCATC\n",
      "1 TACTCTCATC motifs detected in file\n",
      "\n",
      "Seq is: GATATTACTT\n",
      "0 GATATTACTT motifs detected in file\n",
      "\n",
      "Seq is: GTCGCTATCA\n",
      "0 GTCGCTATCA motifs detected in file\n",
      "\n",
      "Seq is: TAATAAGGAA\n",
      "0 TAATAAGGAA motifs detected in file\n",
      "\n",
      "Seq is: GATCCGTATC\n",
      "0 GATCCGTATC motifs detected in file\n",
      "\n",
      "Seq is: ATCCCTCCTA\n",
      "0 ATCCCTCCTA motifs detected in file\n",
      "\n",
      "Seq is: TACTCCACAC\n",
      "0 TACTCCACAC motifs detected in file\n",
      "\n",
      "Seq is: TCGCTATATG\n",
      "0 TCGCTATATG motifs detected in file\n",
      "\n",
      "Seq is: ATCATAAGGA\n",
      "0 ATCATAAGGA motifs detected in file\n",
      "\n",
      "Seq is: TCTTCACTCT\n",
      "0 TCTTCACTCT motifs detected in file\n",
      "\n",
      "Seq is: ACCATTTACG\n",
      "0 ACCATTTACG motifs detected in file\n",
      "\n",
      "Seq is: ATCATATTAT\n",
      "0 ATCATATTAT motifs detected in file\n",
      "\n",
      "Seq is: AAATACATTT\n",
      "0 AAATACATTT motifs detected in file\n",
      "\n",
      "Seq is: ACTCCCCGTA\n",
      "0 ACTCCCCGTA motifs detected in file\n",
      "\n",
      "Seq is: AACCATTTAT\n",
      "0 AACCATTTAT motifs detected in file\n",
      "\n",
      "Seq is: ACTAATAGTA\n",
      "0 ACTAATAGTA motifs detected in file\n",
      "\n",
      "Seq is: TTAAATTACT\n",
      "0 TTAAATTACT motifs detected in file\n",
      "\n",
      "Seq is: CAGCTTTAGT\n",
      "0 CAGCTTTAGT motifs detected in file\n",
      "\n",
      "Seq is: AAGCAAAACA\n",
      "0 AAGCAAAACA motifs detected in file\n",
      "\n",
      "Seq is: AATTTGATAT\n",
      "0 AATTTGATAT motifs detected in file\n",
      "\n",
      "Seq is: ATGGTTAAGC\n",
      "0 ATGGTTAAGC motifs detected in file\n",
      "\n",
      "Seq is: CAAGTCAGAT\n",
      "0 CAAGTCAGAT motifs detected in file\n",
      "\n",
      "Seq is: TAGGAAAAAC\n",
      "0 TAGGAAAAAC motifs detected in file\n",
      "\n",
      "Seq is: ATCAGTGAGT\n",
      "0 ATCAGTGAGT motifs detected in file\n",
      "\n",
      "Seq is: ATCCGGTCTG\n",
      "0 ATCCGGTCTG motifs detected in file\n",
      "\n",
      "Seq is: AATCCCGATC\n",
      "0 AATCCCGATC motifs detected in file\n",
      "\n",
      "Complement found from 14210 to 14220\n",
      "Seq is: CTACTACTAA\n",
      "1 CTACTACTAA motifs detected in file\n",
      "\n",
      "Seq is: TTATTCTCCC\n",
      "0 TTATTCTCCC motifs detected in file\n",
      "\n",
      "Seq is: TACTGTATTG\n",
      "0 TACTGTATTG motifs detected in file\n",
      "\n",
      "Seq is: ATAATCACCG\n",
      "0 ATAATCACCG motifs detected in file\n",
      "\n",
      "Seq is: TCCAATCAAC\n",
      "0 TCCAATCAAC motifs detected in file\n",
      "\n",
      "Seq is: AAACATCCCG\n",
      "0 AAACATCCCG motifs detected in file\n",
      "\n",
      "Seq is: AGTACCATCC\n",
      "0 AGTACCATCC motifs detected in file\n",
      "\n",
      "Seq is: CCATTTTCCT\n",
      "0 CCATTTTCCT motifs detected in file\n",
      "\n",
      "Seq is: CCCGTTAAAG\n",
      "0 CCCGTTAAAG motifs detected in file\n",
      "\n",
      "Seq is: ATCTAGTTTA\n",
      "0 ATCTAGTTTA motifs detected in file\n",
      "\n",
      "Seq is: TTATTCTTCC\n",
      "0 TTATTCTTCC motifs detected in file\n",
      "\n",
      "Seq is: ATTATCGATG\n",
      "0 ATTATCGATG motifs detected in file\n",
      "\n",
      "Complement found from 10219 to 10229\n",
      "Seq is: ATTCTTCTTA\n",
      "1 ATTCTTCTTA motifs detected in file\n",
      "\n",
      "Seq is: AAATACCTCT\n",
      "0 AAATACCTCT motifs detected in file\n",
      "\n",
      "Seq is: TTCCCTGCGC\n",
      "0 TTCCCTGCGC motifs detected in file\n",
      "\n",
      "Seq is: CCGCCCCCTA\n",
      "0 CCGCCCCCTA motifs detected in file\n",
      "\n",
      "Seq is: TATCCCAGCT\n",
      "0 TATCCCAGCT motifs detected in file\n",
      "\n",
      "Seq is: TCGGCGTGAG\n",
      "0 TCGGCGTGAG motifs detected in file\n",
      "\n",
      "Seq is: CATTCCCCAC\n",
      "0 CATTCCCCAC motifs detected in file\n",
      "\n",
      "Seq is: CTAAAAAGAT\n",
      "0 CTAAAAAGAT motifs detected in file\n",
      "\n",
      "Seq is: ACATCGGCAA\n",
      "0 ACATCGGCAA motifs detected in file\n",
      "\n",
      "Complement found from 6545 to 6555\n",
      "Seq is: CTCAACACCA\n",
      "1 CTCAACACCA motifs detected in file\n",
      "\n",
      "Seq is: TCAGTTTTAC\n",
      "0 TCAGTTTTAC motifs detected in file\n",
      "\n",
      "Seq is: ATTATTAATA\n",
      "0 ATTATTAATA motifs detected in file\n",
      "\n",
      "Seq is: ATCATCATTC\n",
      "0 ATCATCATTC motifs detected in file\n",
      "\n",
      "Seq is: CGATCCTCCC\n",
      "0 CGATCCTCCC motifs detected in file\n",
      "\n",
      "Seq is: ACAACTAATA\n",
      "0 ACAACTAATA motifs detected in file\n",
      "\n",
      "Seq is: ATTTTAATTC\n",
      "0 ATTTTAATTC motifs detected in file\n",
      "\n",
      "Seq is: CGCTTCAAAT\n",
      "0 CGCTTCAAAT motifs detected in file\n",
      "\n",
      "Seq is: AATGAGAAAA\n",
      "0 AATGAGAAAA motifs detected in file\n",
      "\n",
      "Seq is: AACTTACAAC\n",
      "0 AACTTACAAC motifs detected in file\n",
      "\n",
      "Seq is: AGTTTTGATC\n",
      "0 AGTTTTGATC motifs detected in file\n",
      "\n",
      "Seq is: AATTAACCTT\n",
      "0 AATTAACCTT motifs detected in file\n",
      "\n",
      "Seq is: CAATTGCCAT\n",
      "0 CAATTGCCAT motifs detected in file\n",
      "\n",
      "Seq is: GATAAATATG\n",
      "0 GATAAATATG motifs detected in file\n",
      "\n",
      "Seq is: ATTTTCTCAT\n",
      "0 ATTTTCTCAT motifs detected in file\n",
      "\n",
      "Seq is: TCTGGGAGTA\n",
      "0 TCTGGGAGTA motifs detected in file\n",
      "\n",
      "Seq is: GTTATCTACC\n",
      "0 GTTATCTACC motifs detected in file\n",
      "\n",
      "Complement found from 9958 to 9968\n",
      "Seq is: TCTGTATGTC\n",
      "1 TCTGTATGTC motifs detected in file\n",
      "\n",
      "Seq is: TTTATCAGTT\n",
      "0 TTTATCAGTT motifs detected in file\n",
      "\n",
      "Seq is: TGGTGTAGAT\n",
      "0 TGGTGTAGAT motifs detected in file\n",
      "\n",
      "Seq is: GTTTTACGGT\n",
      "0 GTTTTACGGT motifs detected in file\n",
      "\n",
      "Seq is: CATAGTCCGC\n",
      "0 CATAGTCCGC motifs detected in file\n",
      "\n",
      "Seq is: CGCCGAAGCT\n",
      "0 CGCCGAAGCT motifs detected in file\n",
      "\n",
      "Seq is: TCGGTTTCAC\n",
      "0 TCGGTTTCAC motifs detected in file\n",
      "\n",
      "Seq is: TACAAACCTA\n",
      "0 TACAAACCTA motifs detected in file\n",
      "\n",
      "Seq is: CATTTCACTT\n",
      "0 CATTTCACTT motifs detected in file\n",
      "\n",
      "Seq is: TATAATCAAC\n",
      "0 TATAATCAAC motifs detected in file\n",
      "\n",
      "Seq is: CGCCTACTTC\n",
      "0 CGCCTACTTC motifs detected in file\n",
      "\n",
      "Seq is: GTCTATCACT\n",
      "0 GTCTATCACT motifs detected in file\n",
      "\n",
      "Seq is: CCTTTCAACT\n",
      "0 CCTTTCAACT motifs detected in file\n",
      "\n",
      "Seq is: CGGTTATTAC\n",
      "0 CGGTTATTAC motifs detected in file\n",
      "\n",
      "Seq is: TGCACTTCAG\n",
      "0 TGCACTTCAG motifs detected in file\n",
      "\n",
      "Seq is: GCACCTTCGG\n",
      "0 GCACCTTCGG motifs detected in file\n",
      "\n",
      "Seq is: ACACCGATGT\n",
      "0 ACACCGATGT motifs detected in file\n",
      "\n",
      "Seq is: TTTTTACAAC\n",
      "0 TTTTTACAAC motifs detected in file\n",
      "\n",
      "Seq is: TCGGCATCTA\n",
      "0 TCGGCATCTA motifs detected in file\n",
      "\n",
      "Seq is: CGGCAGCCTT\n",
      "0 CGGCAGCCTT motifs detected in file\n",
      "\n",
      "Seq is: TACCACTTCC\n",
      "0 TACCACTTCC motifs detected in file\n",
      "\n",
      "Seq is: CTCTGAGCTT\n",
      "0 CTCTGAGCTT motifs detected in file\n",
      "\n",
      "Seq is: CATGAGACTC\n",
      "0 CATGAGACTC motifs detected in file\n",
      "\n",
      "Seq is: CGAACATCCT\n",
      "0 CGAACATCCT motifs detected in file\n",
      "\n",
      "Seq is: CCCATTTTAT\n",
      "0 CCCATTTTAT motifs detected in file\n",
      "\n",
      "Seq is: CTCTGGGTCA\n",
      "0 CTCTGGGTCA motifs detected in file\n",
      "\n",
      "Seq is: TTTTAACATT\n",
      "0 TTTTAACATT motifs detected in file\n",
      "\n",
      "Seq is: ATTCGTCACG\n",
      "0 ATTCGTCACG motifs detected in file\n",
      "\n",
      "Seq is: AACTTAATAA\n",
      "0 AACTTAATAA motifs detected in file\n",
      "\n",
      "Seq is: ACCAAAGCCA\n",
      "0 ACCAAAGCCA motifs detected in file\n",
      "\n",
      "Seq is: ACAAAAGATA\n",
      "0 ACAAAAGATA motifs detected in file\n",
      "\n",
      "Seq is: ATCTGATACC\n",
      "0 ATCTGATACC motifs detected in file\n",
      "\n",
      "Seq is: ACTCGAGTCC\n",
      "0 ACTCGAGTCC motifs detected in file\n",
      "\n",
      "Seq is: ACTAACTATG\n",
      "0 ACTAACTATG motifs detected in file\n",
      "\n",
      "Seq is: AGGACTACGC\n",
      "0 AGGACTACGC motifs detected in file\n",
      "\n",
      "Seq is: TCATTATGCC\n",
      "0 TCATTATGCC motifs detected in file\n",
      "\n",
      "Seq is: TACACAAATC\n",
      "0 TACACAAATC motifs detected in file\n",
      "\n",
      "Seq is: CTCACCCTGA\n",
      "0 CTCACCCTGA motifs detected in file\n",
      "\n",
      "Seq is: AGATCCCCTA\n",
      "0 AGATCCCCTA motifs detected in file\n",
      "\n",
      "Seq is: AATCGCCCCA\n",
      "0 AATCGCCCCA motifs detected in file\n",
      "\n",
      "Seq is: CTACGGACAA\n",
      "0 CTACGGACAA motifs detected in file\n",
      "\n",
      "Seq is: CCCCCGGTCA\n",
      "0 CCCCCGGTCA motifs detected in file\n",
      "\n",
      "Seq is: CGGGAGGATT\n",
      "0 CGGGAGGATT motifs detected in file\n",
      "\n",
      "Seq is: AACCCCCCAT\n",
      "0 AACCCCCCAT motifs detected in file\n",
      "\n",
      "Seq is: CCCCGATCCG\n",
      "0 CCCCGATCCG motifs detected in file\n",
      "\n",
      "Seq is: ACCTCACCAT\n",
      "0 ACCTCACCAT motifs detected in file\n",
      "\n",
      "Seq is: TTTCCGAGTC\n",
      "0 TTTCCGAGTC motifs detected in file\n",
      "\n",
      "Seq is: TTTTTAGGAC\n",
      "0 TTTTTAGGAC motifs detected in file\n",
      "\n",
      "Seq is: GCTTCTTTTT\n",
      "0 GCTTCTTTTT motifs detected in file\n",
      "\n",
      "Seq is: TTGAAGACTC\n",
      "0 TTGAAGACTC motifs detected in file\n",
      "\n",
      "Seq is: CATTATTTAT\n",
      "0 CATTATTTAT motifs detected in file\n",
      "\n",
      "Seq is: CCTAATAGGG\n",
      "0 CCTAATAGGG motifs detected in file\n",
      "\n",
      "Seq is: CATAGCTTCC\n",
      "0 CATAGCTTCC motifs detected in file\n",
      "\n",
      "Seq is: GGAAAAACCT\n",
      "0 GGAAAAACCT motifs detected in file\n",
      "\n",
      "Seq is: GTCCACCACA\n",
      "0 GTCCACCACA motifs detected in file\n",
      "\n",
      "Seq is: CACCACCGGA\n",
      "0 CACCACCGGA motifs detected in file\n",
      "\n",
      "Seq is: ACCATACACG\n",
      "0 ACCATACACG motifs detected in file\n",
      "\n",
      "Complement found from 1935 to 1945\n",
      "Seq is: AAAGAGCACA\n",
      "1 AAAGAGCACA motifs detected in file\n",
      "\n",
      "Seq is: ATGTAGCGCG\n",
      "0 ATGTAGCGCG motifs detected in file\n",
      "\n",
      "Seq is: GTAGTAACCA\n",
      "0 GTAGTAACCA motifs detected in file\n",
      "\n",
      "Seq is: TATACCAATC\n",
      "0 TATACCAATC motifs detected in file\n",
      "\n",
      "Seq is: ACACAACCAA\n",
      "0 ACACAACCAA motifs detected in file\n",
      "\n",
      "Seq is: TCATCCGGAT\n",
      "0 TCATCCGGAT motifs detected in file\n",
      "\n",
      "Complement found from 5007 to 5017\n",
      "Seq is: CATACTCCTC\n",
      "1 CATACTCCTC motifs detected in file\n",
      "\n",
      "Seq is: GCAATACCTC\n",
      "0 GCAATACCTC motifs detected in file\n",
      "\n",
      "Seq is: ACCTTCACTT\n",
      "0 ACCTTCACTT motifs detected in file\n",
      "\n",
      "Seq is: TAGTGTACCG\n",
      "0 TAGTGTACCG motifs detected in file\n",
      "\n",
      "Seq is: ATCCGGCCTC\n",
      "0 ATCCGGCCTC motifs detected in file\n",
      "\n",
      "Seq is: CAGTAATCCT\n",
      "0 CAGTAATCCT motifs detected in file\n",
      "\n",
      "Seq is: CCCGACTCTC\n",
      "0 CCCGACTCTC motifs detected in file\n",
      "\n",
      "Seq is: CCGGGGACAA\n",
      "0 CCGGGGACAA motifs detected in file\n",
      "\n",
      "Seq is: TCCCCAGTAC\n",
      "0 TCCCCAGTAC motifs detected in file\n",
      "\n",
      "Seq is: CCGACCCAAA\n",
      "0 CCGACCCAAA motifs detected in file\n",
      "\n",
      "Seq is: ATGATATACT\n",
      "0 ATGATATACT motifs detected in file\n",
      "\n",
      "Seq is: ATCCGTACAC\n",
      "0 ATCCGTACAC motifs detected in file\n",
      "\n",
      "Seq is: TAACCACCCA\n",
      "0 TAACCACCCA motifs detected in file\n",
      "\n",
      "Seq is: GTAATACACA\n",
      "0 GTAATACACA motifs detected in file\n",
      "\n",
      "Seq is: ACAGCACGTC\n",
      "0 ACAGCACGTC motifs detected in file\n",
      "\n",
      "Seq is: CATCTCCGAA\n",
      "0 CATCTCCGAA motifs detected in file\n",
      "\n",
      "Seq is: TGATCTTCAC\n",
      "0 TGATCTTCAC motifs detected in file\n",
      "\n",
      "Seq is: ACTTTTGCAT\n",
      "0 ACTTTTGCAT motifs detected in file\n",
      "\n",
      "Seq is: CCGAACCTAA\n",
      "0 CCGAACCTAA motifs detected in file\n",
      "\n",
      "Seq is: TTCCGCTGTC\n",
      "0 TTCCGCTGTC motifs detected in file\n",
      "\n",
      "Seq is: GCTAAAGATC\n",
      "0 GCTAAAGATC motifs detected in file\n",
      "\n",
      "Seq is: CTATCAGTCA\n",
      "0 CTATCAGTCA motifs detected in file\n",
      "\n",
      "Seq is: TCTTAATCTT\n",
      "0 TCTTAATCTT motifs detected in file\n",
      "\n",
      "Seq is: AACACTTCTA\n",
      "0 AACACTTCTA motifs detected in file\n",
      "\n",
      "Seq is: CTATTCACAT\n",
      "0 CTATTCACAT motifs detected in file\n",
      "\n",
      "Seq is: CTCCCTTCCA\n",
      "0 CTCCCTTCCA motifs detected in file\n",
      "\n",
      "Seq is: ATTACCAACT\n",
      "0 ATTACCAACT motifs detected in file\n",
      "\n",
      "Seq is: ATAACGATCC\n",
      "0 ATAACGATCC motifs detected in file\n",
      "\n",
      "Seq is: CACCGCGAAG\n",
      "0 CACCGCGAAG motifs detected in file\n",
      "\n",
      "Seq is: GTTAATCCAC\n",
      "0 GTTAATCCAC motifs detected in file\n",
      "\n",
      "Seq is: GTACTCATCC\n",
      "0 GTACTCATCC motifs detected in file\n",
      "\n",
      "Seq is: ACCGGACGTC\n",
      "0 ACCGGACGTC motifs detected in file\n",
      "\n",
      "Seq is: ATTACAATCG\n",
      "0 ATTACAATCG motifs detected in file\n",
      "\n",
      "Seq is: CCAATCCGCA\n",
      "0 CCAATCCGCA motifs detected in file\n",
      "\n",
      "Seq is: TGCCGGTCCC\n",
      "0 TGCCGGTCCC motifs detected in file\n",
      "\n",
      "Seq is: GATAACCAAC\n",
      "0 GATAACCAAC motifs detected in file\n",
      "\n",
      "Complement found from 13110 to 13120\n",
      "Seq is: TTACTCATCC\n",
      "1 TTACTCATCC motifs detected in file\n",
      "\n",
      "Seq is: GACTACCAAA\n",
      "0 GACTACCAAA motifs detected in file\n",
      "\n",
      "Seq is: GCTATTATTG\n",
      "0 GCTATTATTG motifs detected in file\n",
      "\n",
      "Seq is: ATCATACCCC\n",
      "0 ATCATACCCC motifs detected in file\n",
      "\n",
      "Seq is: TATTCCCCAC\n",
      "0 TATTCCCCAC motifs detected in file\n",
      "\n",
      "Seq is: ATCCACACGG\n",
      "0 ATCCACACGG motifs detected in file\n",
      "\n",
      "Seq is: AACACCATTC\n",
      "0 AACACCATTC motifs detected in file\n",
      "\n",
      "Seq is: TTCACCCGAT\n",
      "0 TTCACCCGAT motifs detected in file\n",
      "\n",
      "Seq is: CCCGTAAAAA\n",
      "0 CCCGTAAAAA motifs detected in file\n",
      "\n",
      "Seq is: TTAGAATCTC\n",
      "0 TTAGAATCTC motifs detected in file\n",
      "\n",
      "Seq is: GCTTTCGGAT\n",
      "0 GCTTTCGGAT motifs detected in file\n",
      "\n",
      "Seq is: ATTAGTGACA\n",
      "0 ATTAGTGACA motifs detected in file\n",
      "\n",
      "Seq is: CGGGCGAGTA\n",
      "0 CGGGCGAGTA motifs detected in file\n",
      "\n",
      "Seq is: TTCCCCTACC\n",
      "0 TTCCCCTACC motifs detected in file\n",
      "\n",
      "Seq is: GGTACCGATC\n",
      "0 GGTACCGATC motifs detected in file\n",
      "\n",
      "Complement found from 12689 to 12699\n",
      "Seq is: CAAATATCTA\n",
      "1 CAAATATCTA motifs detected in file\n",
      "\n",
      "Seq is: TCAACCCACC\n",
      "0 TCAACCCACC motifs detected in file\n",
      "\n",
      "Seq is: AACCACATTT\n",
      "0 AACCACATTT motifs detected in file\n",
      "\n",
      "Seq is: ACTCACTCCG\n",
      "0 ACTCACTCCG motifs detected in file\n",
      "\n",
      "Seq is: TCCTCAGGCT\n",
      "0 TCCTCAGGCT motifs detected in file\n",
      "\n",
      "Seq is: CCTCCAATCA\n",
      "0 CCTCCAATCA motifs detected in file\n",
      "\n",
      "Seq is: ACACCGTTAT\n",
      "0 ACACCGTTAT motifs detected in file\n",
      "\n",
      "Seq is: TTTTACTAAT\n",
      "0 TTTTACTAAT motifs detected in file\n",
      "\n",
      "Seq is: TCCTATGATC\n",
      "0 TCCTATGATC motifs detected in file\n",
      "\n",
      "Seq is: ATATTCTCTA\n",
      "0 ATATTCTCTA motifs detected in file\n",
      "\n",
      "Seq is: GTCCAAGCAG\n",
      "0 GTCCAAGCAG motifs detected in file\n",
      "\n",
      "Seq is: GAAATCACAA\n",
      "0 GAAATCACAA motifs detected in file\n",
      "\n",
      "Complement found from 9400 to 9410\n",
      "Seq is: CACATACCAA\n",
      "1 CACATACCAA motifs detected in file\n",
      "\n",
      "Seq is: TAGTAAACAA\n",
      "0 TAGTAAACAA motifs detected in file\n",
      "\n",
      "Seq is: AACTCCAATC\n",
      "0 AACTCCAATC motifs detected in file\n",
      "\n",
      "Seq is: AAACTAATCA\n",
      "0 AAACTAATCA motifs detected in file\n",
      "\n",
      "Seq is: GTAACAACCC\n",
      "0 GTAACAACCC motifs detected in file\n",
      "\n",
      "Seq is: ACCACTAATC\n",
      "0 ACCACTAATC motifs detected in file\n",
      "\n",
      "Complement found from 11328 to 11338\n",
      "Seq is: AGCCAACAAC\n",
      "1 AGCCAACAAC motifs detected in file\n",
      "\n",
      "Seq is: TACTCTATAA\n",
      "0 TACTCTATAA motifs detected in file\n",
      "\n",
      "Seq is: ACCTCCACCC\n",
      "0 ACCTCCACCC motifs detected in file\n",
      "\n",
      "Seq is: CTAGTTATCT\n",
      "0 CTAGTTATCT motifs detected in file\n",
      "\n",
      "Seq is: CCCCCTTTAT\n",
      "0 CCCCCTTTAT motifs detected in file\n",
      "\n",
      "Seq is: CTTACTAGTC\n",
      "0 CTTACTAGTC motifs detected in file\n",
      "\n",
      "Seq is: ATGACGCCGC\n",
      "0 ATGACGCCGC motifs detected in file\n",
      "\n",
      "Seq is: CCATCCGGAT\n",
      "0 CCATCCGGAT motifs detected in file\n",
      "\n",
      "Seq is: CCTAACACCC\n",
      "0 CCTAACACCC motifs detected in file\n",
      "\n",
      "Seq is: CCGTTACTTA\n",
      "0 CCGTTACTTA motifs detected in file\n",
      "\n",
      "Seq is: CTTCGCTTGT\n",
      "0 CTTCGCTTGT motifs detected in file\n",
      "\n",
      "Seq is: CTAAAAGCAA\n",
      "0 CTAAAAGCAA motifs detected in file\n",
      "\n",
      "Seq is: GTAAAACCAA\n",
      "0 GTAAAACCAA motifs detected in file\n",
      "\n",
      "Seq is: GAGTCCCAAA\n",
      "0 GAGTCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: CAATATTAAA\n",
      "0 CAATATTAAA motifs detected in file\n",
      "\n",
      "Seq is: AAATAAAAAT\n",
      "0 AAATAAAAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCCGAAACC\n",
      "0 ACCCGAAACC motifs detected in file\n",
      "\n",
      "Seq is: ACTCCCTCCA\n",
      "0 ACTCCCTCCA motifs detected in file\n",
      "\n",
      "Seq is: TCCACCATCA\n",
      "0 TCCACCATCA motifs detected in file\n",
      "\n",
      "Seq is: AACACAAATT\n",
      "0 AACACAAATT motifs detected in file\n",
      "\n",
      "Seq is: ATAAAAATCA\n",
      "0 ATAAAAATCA motifs detected in file\n",
      "\n",
      "Seq is: ACCCACTACT\n",
      "0 ACCCACTACT motifs detected in file\n",
      "\n",
      "Complement found from 3797 to 3807\n",
      "Seq is: CCTTATCACA\n",
      "1 CCTTATCACA motifs detected in file\n",
      "\n",
      "Seq is: TTCCTCATAC\n",
      "0 TTCCTCATAC motifs detected in file\n",
      "\n",
      "Seq is: CCCCATTAAT\n",
      "0 CCCCATTAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCACCCGGT\n",
      "0 ACCACCCGGT motifs detected in file\n",
      "\n",
      "Seq is: ATGCCATCAT\n",
      "0 ATGCCATCAT motifs detected in file\n",
      "\n",
      "Seq is: AAATCAACCC\n",
      "0 AAATCAACCC motifs detected in file\n",
      "\n",
      "Seq is: CGTAAAGTGA\n",
      "0 CGTAAAGTGA motifs detected in file\n",
      "\n",
      "Seq is: CATTTCTCCA\n",
      "0 CATTTCTCCA motifs detected in file\n",
      "\n",
      "Seq is: CAACCAAGAG\n",
      "0 CAACCAAGAG motifs detected in file\n",
      "\n",
      "Seq is: AATTAGAAAT\n",
      "0 AATTAGAAAT motifs detected in file\n",
      "\n",
      "Seq is: TGAATTTTCC\n",
      "0 TGAATTTTCC motifs detected in file\n",
      "\n",
      "Seq is: AATTACGATT\n",
      "0 AATTACGATT motifs detected in file\n",
      "\n",
      "Seq is: CAATCGAAAT\n",
      "0 CAATCGAAAT motifs detected in file\n",
      "\n",
      "Seq is: GTCACCCGAG\n",
      "0 GTCACCCGAG motifs detected in file\n",
      "\n",
      "Seq is: ATCTCCCCCA\n",
      "0 ATCTCCCCCA motifs detected in file\n",
      "\n",
      "Seq is: TCTCCCCCAC\n",
      "0 TCTCCCCCAC motifs detected in file\n",
      "\n",
      "Seq is: GATATCCCAT\n",
      "0 GATATCCCAT motifs detected in file\n",
      "\n",
      "Seq is: TTATGCCCGG\n",
      "0 TTATGCCCGG motifs detected in file\n",
      "\n",
      "Seq is: GATAAAGTTT\n",
      "0 GATAAAGTTT motifs detected in file\n",
      "\n",
      "Seq is: CTAAAAATCC\n",
      "0 CTAAAAATCC motifs detected in file\n",
      "\n",
      "Seq is: CCTTAATTAA\n",
      "0 CCTTAATTAA motifs detected in file\n",
      "\n",
      "Seq is: GATCCTGCTA\n",
      "0 GATCCTGCTA motifs detected in file\n",
      "\n",
      "Seq is: CCCGTACTTT\n",
      "0 CCCGTACTTT motifs detected in file\n",
      "\n",
      "Seq is: GACACCAAAC\n",
      "0 GACACCAAAC motifs detected in file\n",
      "\n",
      "Seq is: GAGGTGTCTA\n",
      "0 GAGGTGTCTA motifs detected in file\n",
      "\n",
      "Seq is: AAGTCTCGTA\n",
      "0 AAGTCTCGTA motifs detected in file\n",
      "\n",
      "Seq is: ACTGGCATCA\n",
      "0 ACTGGCATCA motifs detected in file\n",
      "\n",
      "Seq is: TATGGGGGCC\n",
      "0 TATGGGGGCC motifs detected in file\n",
      "\n",
      "Seq is: AGCACATCGC\n",
      "0 AGCACATCGC motifs detected in file\n",
      "\n",
      "Complement found from 8130 to 8140\n",
      "Seq is: CACTTTCACC\n",
      "1 CACTTTCACC motifs detected in file\n",
      "\n",
      "Seq is: AAACCAAATC\n",
      "0 AAACCAAATC motifs detected in file\n",
      "\n",
      "Seq is: TGCAGGCCCT\n",
      "0 TGCAGGCCCT motifs detected in file\n",
      "\n",
      "Seq is: TAACGTAGAC\n",
      "0 TAACGTAGAC motifs detected in file\n",
      "\n",
      "Seq is: AAAAATTCGG\n",
      "0 AAAAATTCGG motifs detected in file\n",
      "\n",
      "Seq is: ATTACACCCC\n",
      "0 ATTACACCCC motifs detected in file\n",
      "\n",
      "Seq is: TGTCGAGTAC\n",
      "0 TGTCGAGTAC motifs detected in file\n",
      "\n",
      "Seq is: TCACGTTCTG\n",
      "0 TCACGTTCTG motifs detected in file\n",
      "\n",
      "Complement found from 1135 to 1145\n",
      "Seq is: CAGAACACTA\n",
      "1 CAGAACACTA motifs detected in file\n",
      "\n",
      "Complement found from 4752 to 4762\n",
      "Seq is: CATTAATAAT\n",
      "1 CATTAATAAT motifs detected in file\n",
      "\n",
      "Seq is: ATGCTTACCC\n",
      "0 ATGCTTACCC motifs detected in file\n",
      "\n",
      "Seq is: CCGAAGTTAG\n",
      "0 CCGAAGTTAG motifs detected in file\n",
      "\n",
      "Seq is: CCCTCATGAT\n",
      "0 CCCTCATGAT motifs detected in file\n",
      "\n",
      "Seq is: GAGCTAACAG\n",
      "0 GAGCTAACAG motifs detected in file\n",
      "\n",
      "Seq is: TTGCAGTTCC\n",
      "0 TTGCAGTTCC motifs detected in file\n",
      "\n",
      "Seq is: TCAGCGTCCA\n",
      "0 TCAGCGTCCA motifs detected in file\n",
      "\n",
      "Seq is: GCGGACCAAG\n",
      "0 GCGGACCAAG motifs detected in file\n",
      "\n",
      "Seq is: ATCCTTATTA\n",
      "0 ATCCTTATTA motifs detected in file\n",
      "\n",
      "Seq is: CCCCCTTCAT\n",
      "0 CCCCCTTCAT motifs detected in file\n",
      "\n",
      "Seq is: ACATCCTCAA\n",
      "0 ACATCCTCAA motifs detected in file\n",
      "\n",
      "Seq is: CTTCTAATCA\n",
      "0 CTTCTAATCA motifs detected in file\n",
      "\n",
      "Seq is: GGCGGCATCA\n",
      "0 GGCGGCATCA motifs detected in file\n",
      "\n",
      "Seq is: GCCACATGAG\n",
      "0 GCCACATGAG motifs detected in file\n",
      "\n",
      "Seq is: CATCCAAGTC\n",
      "0 CATCCAAGTC motifs detected in file\n",
      "\n",
      "Seq is: ATGGTAACCA\n",
      "0 ATGGTAACCA motifs detected in file\n",
      "\n",
      "Seq is: CCGGTTAACT\n",
      "0 CCGGTTAACT motifs detected in file\n",
      "\n",
      "Seq is: AAACTACCAT\n",
      "0 AAACTACCAT motifs detected in file\n",
      "\n",
      "Seq is: TCCCTCCCTA\n",
      "0 TCCCTCCCTA motifs detected in file\n",
      "\n",
      "Seq is: GCAACTGGAG\n",
      "0 GCAACTGGAG motifs detected in file\n",
      "\n",
      "Seq is: CAGACAATAC\n",
      "0 CAGACAATAC motifs detected in file\n",
      "\n",
      "Seq is: ATTTCCTACG\n",
      "0 ATTTCCTACG motifs detected in file\n",
      "\n",
      "Seq is: CATCCCTACC\n",
      "0 CATCCCTACC motifs detected in file\n",
      "\n",
      "Seq is: CTCCCGCTAC\n",
      "0 CTCCCGCTAC motifs detected in file\n",
      "\n",
      "Seq is: TCCTGATCCT\n",
      "0 TCCTGATCCT motifs detected in file\n",
      "\n",
      "Seq is: ACTACCGCCC\n",
      "0 ACTACCGCCC motifs detected in file\n",
      "\n",
      "Complement found from 7040 to 7050\n",
      "Seq is: GTCCTATCAA\n",
      "1 GTCCTATCAA motifs detected in file\n",
      "\n",
      "Seq is: GTCTGCCAAA\n",
      "0 GTCTGCCAAA motifs detected in file\n",
      "\n",
      "Seq is: GATAAAGGAC\n",
      "0 GATAAAGGAC motifs detected in file\n",
      "\n",
      "Seq is: TCGCAGACTC\n",
      "0 TCGCAGACTC motifs detected in file\n",
      "\n",
      "Seq is: TACAATCATA\n",
      "0 TACAATCATA motifs detected in file\n",
      "\n",
      "Seq is: ATCAATCAAA\n",
      "0 ATCAATCAAA motifs detected in file\n",
      "\n",
      "Seq is: ACAACACTCA\n",
      "0 ACAACACTCA motifs detected in file\n",
      "\n",
      "Seq is: CAATCCTTTT\n",
      "0 CAATCCTTTT motifs detected in file\n",
      "\n",
      "Seq is: CCCGTATGTC\n",
      "0 CCCGTATGTC motifs detected in file\n",
      "\n",
      "Seq is: CTGATCCTTC\n",
      "0 CTGATCCTTC motifs detected in file\n",
      "\n",
      "Seq is: GTCTATTCCT\n",
      "0 GTCTATTCCT motifs detected in file\n",
      "\n",
      "Seq is: TTTACTAATA\n",
      "0 TTTACTAATA motifs detected in file\n",
      "\n",
      "Seq is: CTCCCGCACT\n",
      "0 CTCCCGCACT motifs detected in file\n",
      "\n",
      "Seq is: AGTACTTTCC\n",
      "0 AGTACTTTCC motifs detected in file\n",
      "\n",
      "Seq is: ACTATTCGAG\n",
      "0 ACTATTCGAG motifs detected in file\n",
      "\n",
      "Seq is: AAGATACTAT\n",
      "0 AAGATACTAT motifs detected in file\n",
      "\n",
      "Seq is: CCCCTTCATC\n",
      "0 CCCCTTCATC motifs detected in file\n",
      "\n",
      "Seq is: GCAGAACATC\n",
      "0 GCAGAACATC motifs detected in file\n",
      "\n",
      "Seq is: TGGATGAACG\n",
      "0 TGGATGAACG motifs detected in file\n",
      "\n",
      "Seq is: CGACGTACAC\n",
      "0 CGACGTACAC motifs detected in file\n",
      "\n",
      "Seq is: GGTAATTCTA\n",
      "0 GGTAATTCTA motifs detected in file\n",
      "\n",
      "Seq is: TATATCCTAA\n",
      "0 TATATCCTAA motifs detected in file\n",
      "\n",
      "Seq is: ATCGGATATT\n",
      "0 ATCGGATATT motifs detected in file\n",
      "\n",
      "Complement found from 1754 to 1764\n",
      "Seq is: AAATTGAAAC\n",
      "1 AAATTGAAAC motifs detected in file\n",
      "\n",
      "Seq is: TGTTTCAATA\n",
      "0 TGTTTCAATA motifs detected in file\n",
      "\n",
      "Seq is: CTTTACCAAA\n",
      "0 CTTTACCAAA motifs detected in file\n",
      "\n",
      "Seq is: AAGATTATGG\n",
      "0 AAGATTATGG motifs detected in file\n",
      "\n",
      "Seq is: AAAAACTTTT\n",
      "0 AAAAACTTTT motifs detected in file\n",
      "\n",
      "Seq is: TCAGTACCTC\n",
      "0 TCAGTACCTC motifs detected in file\n",
      "\n",
      "Seq is: CGGTACCCCA\n",
      "0 CGGTACCCCA motifs detected in file\n",
      "\n",
      "Seq is: ACCGAACTTT\n",
      "0 ACCGAACTTT motifs detected in file\n",
      "\n",
      "Seq is: GGTCGAAACC\n",
      "0 GGTCGAAACC motifs detected in file\n",
      "\n",
      "Seq is: CCCCAAGCTA\n",
      "0 CCCCAAGCTA motifs detected in file\n",
      "\n",
      "Seq is: AGGAAGGAAA\n",
      "0 AGGAAGGAAA motifs detected in file\n",
      "\n",
      "Seq is: AAACAGATCT\n",
      "0 AAACAGATCT motifs detected in file\n",
      "\n",
      "Seq is: AAAATACATA\n",
      "0 AAAATACATA motifs detected in file\n",
      "\n",
      "Complement found from 11304 to 11314\n",
      "Seq is: TGCCCAAGAA\n",
      "1 TGCCCAAGAA motifs detected in file\n",
      "\n",
      "Seq is: GCTTACACAC\n",
      "0 GCTTACACAC motifs detected in file\n",
      "\n",
      "Seq is: CATCCCACCC\n",
      "0 CATCCCACCC motifs detected in file\n",
      "\n",
      "Seq is: CCCGTAGGTA\n",
      "0 CCCGTAGGTA motifs detected in file\n",
      "\n",
      "Seq is: TATCAGTGAG\n",
      "0 TATCAGTGAG motifs detected in file\n",
      "\n",
      "Seq is: GTCCAAATAC\n",
      "0 GTCCAAATAC motifs detected in file\n",
      "\n",
      "Seq is: CTCCCAAGAA\n",
      "0 CTCCCAAGAA motifs detected in file\n",
      "\n",
      "Seq is: GATGATAATC\n",
      "0 GATGATAATC motifs detected in file\n",
      "\n",
      "Seq is: CTGAAAAGCG\n",
      "0 CTGAAAAGCG motifs detected in file\n",
      "\n",
      "Seq is: AAGCTTCGCT\n",
      "0 AAGCTTCGCT motifs detected in file\n",
      "\n",
      "Seq is: TCCGAAGAGT\n",
      "0 TCCGAAGAGT motifs detected in file\n",
      "\n",
      "Seq is: TTAGTACTTT\n",
      "0 TTAGTACTTT motifs detected in file\n",
      "\n",
      "Complement found from 10107 to 10117\n",
      "Seq is: TAATAATTAT\n",
      "1 TAATAATTAT motifs detected in file\n",
      "\n",
      "Seq is: AATGACGACA\n",
      "0 AATGACGACA motifs detected in file\n",
      "\n",
      "Seq is: ATCTCTTTAC\n",
      "0 ATCTCTTTAC motifs detected in file\n",
      "\n",
      "Seq is: TTACTCGGAT\n",
      "0 TTACTCGGAT motifs detected in file\n",
      "\n",
      "Seq is: GTCTACTATC\n",
      "0 GTCTACTATC motifs detected in file\n",
      "\n",
      "Complement found from 1289 to 1299\n",
      "Seq is: CTACAAAGTA\n",
      "1 CTACAAAGTA motifs detected in file\n",
      "\n",
      "Seq is: CACCACATAC\n",
      "0 CACCACATAC motifs detected in file\n",
      "\n",
      "Seq is: GTAGCCCCAT\n",
      "0 GTAGCCCCAT motifs detected in file\n",
      "\n",
      "Seq is: CAGGCTCATT\n",
      "0 CAGGCTCATT motifs detected in file\n",
      "\n",
      "Seq is: GCAGCCCCGT\n",
      "0 GCAGCCCCGT motifs detected in file\n",
      "\n",
      "Seq is: AAGGCCTATC\n",
      "0 AAGGCCTATC motifs detected in file\n",
      "\n",
      "Seq is: CGGCTCTTTC\n",
      "0 CGGCTCTTTC motifs detected in file\n",
      "\n",
      "Complement found from 11261 to 11271\n",
      "Seq is: ACAACACCCT\n",
      "1 ACAACACCCT motifs detected in file\n",
      "\n",
      "Seq is: TCTTTCAATC\n",
      "0 TCTTTCAATC motifs detected in file\n",
      "\n",
      "Seq is: TAAATGCGGC\n",
      "0 TAAATGCGGC motifs detected in file\n",
      "\n",
      "Seq is: TACTTATACT\n",
      "0 TACTTATACT motifs detected in file\n",
      "\n",
      "Seq is: ATCACTTTAC\n",
      "0 ATCACTTTAC motifs detected in file\n",
      "\n",
      "Seq is: CTAAAACCGC\n",
      "0 CTAAAACCGC motifs detected in file\n",
      "\n",
      "Seq is: ATCCAAACCA\n",
      "0 ATCCAAACCA motifs detected in file\n",
      "\n",
      "Seq is: GATCCCACAT\n",
      "0 GATCCCACAT motifs detected in file\n",
      "\n",
      "Seq is: CGGACTCTTA\n",
      "0 CGGACTCTTA motifs detected in file\n",
      "\n",
      "Seq is: TCCCCTTTAG\n",
      "0 TCCCCTTTAG motifs detected in file\n",
      "\n",
      "Seq is: TCACTTACTT\n",
      "0 TCACTTACTT motifs detected in file\n",
      "\n",
      "Seq is: CGGAGGATAC\n",
      "0 CGGAGGATAC motifs detected in file\n",
      "\n",
      "Seq is: TACCGTTTAT\n",
      "0 TACCGTTTAT motifs detected in file\n",
      "\n",
      "Seq is: GTCGAGGATA\n",
      "0 GTCGAGGATA motifs detected in file\n",
      "\n",
      "Seq is: ACTATCCTGT\n",
      "0 ACTATCCTGT motifs detected in file\n",
      "\n",
      "Seq is: ATCACCTTCA\n",
      "0 ATCACCTTCA motifs detected in file\n",
      "\n",
      "Seq is: CCCGATGTTG\n",
      "0 CCCGATGTTG motifs detected in file\n",
      "\n",
      "Seq is: CATCATGCAC\n",
      "0 CATCATGCAC motifs detected in file\n",
      "\n",
      "Seq is: AGCACATCAT\n",
      "0 AGCACATCAT motifs detected in file\n",
      "\n",
      "Seq is: GCTACAGATC\n",
      "0 GCTACAGATC motifs detected in file\n",
      "\n",
      "Complement found from 13418 to 13428\n",
      "Seq is: ACTACTCAAA\n",
      "1 ACTACTCAAA motifs detected in file\n",
      "\n",
      "Seq is: CGATTATGTT\n",
      "0 CGATTATGTT motifs detected in file\n",
      "\n",
      "Seq is: ACGGTCAGTC\n",
      "0 ACGGTCAGTC motifs detected in file\n",
      "\n",
      "Seq is: CGGTGGATGC\n",
      "0 CGGTGGATGC motifs detected in file\n",
      "\n",
      "Seq is: CACTTTTCTT\n",
      "0 CACTTTTCTT motifs detected in file\n",
      "\n",
      "Seq is: TCTACTTAGG\n",
      "0 TCTACTTAGG motifs detected in file\n",
      "\n",
      "Seq is: ATCCCGAGTC\n",
      "0 ATCCCGAGTC motifs detected in file\n",
      "\n",
      "Seq is: TCGTGACGTC\n",
      "0 TCGTGACGTC motifs detected in file\n",
      "\n",
      "Seq is: GTCTAGTAAA\n",
      "0 GTCTAGTAAA motifs detected in file\n",
      "\n",
      "Seq is: GTATAACGAA\n",
      "0 GTATAACGAA motifs detected in file\n",
      "\n",
      "Seq is: GGCACCTCAC\n",
      "0 GGCACCTCAC motifs detected in file\n",
      "\n",
      "Seq is: ACCGCTCAGT\n",
      "0 ACCGCTCAGT motifs detected in file\n",
      "\n",
      "Seq is: CGATTTATGA\n",
      "0 CGATTTATGA motifs detected in file\n",
      "\n",
      "Seq is: AACTGCGGCC\n",
      "0 AACTGCGGCC motifs detected in file\n",
      "\n",
      "Seq is: ACCCCTATCG\n",
      "0 ACCCCTATCG motifs detected in file\n",
      "\n",
      "Seq is: CTACTAATAC\n",
      "0 CTACTAATAC motifs detected in file\n",
      "\n",
      "Seq is: CATCGCCTCC\n",
      "0 CATCGCCTCC motifs detected in file\n",
      "\n",
      "Seq is: ACTTTATACG\n",
      "0 ACTTTATACG motifs detected in file\n",
      "\n",
      "Seq is: AGCACACAGA\n",
      "0 AGCACACAGA motifs detected in file\n",
      "\n",
      "Seq is: TGCAGATAAG\n",
      "0 TGCAGATAAG motifs detected in file\n",
      "\n",
      "Seq is: GATGACATTT\n",
      "0 GATGACATTT motifs detected in file\n",
      "\n",
      "Seq is: ATATACCACA\n",
      "0 ATATACCACA motifs detected in file\n",
      "\n",
      "Seq is: CGAGTGTGCT\n",
      "0 CGAGTGTGCT motifs detected in file\n",
      "\n",
      "Seq is: ATTTGGGATC\n",
      "0 ATTTGGGATC motifs detected in file\n",
      "\n",
      "Seq is: CTTCGGTTAA\n",
      "0 CTTCGGTTAA motifs detected in file\n",
      "\n",
      "Seq is: CTATAGTATC\n",
      "0 CTATAGTATC motifs detected in file\n",
      "\n",
      "Seq is: GAGTCTGGTA\n",
      "0 GAGTCTGGTA motifs detected in file\n",
      "\n",
      "Complement found from 6706 to 6716\n",
      "Seq is: TGGATACATA\n",
      "1 TGGATACATA motifs detected in file\n",
      "\n",
      "Seq is: GGTTTACCAA\n",
      "0 GGTTTACCAA motifs detected in file\n",
      "\n",
      "Seq is: GAAAAAAAGG\n",
      "0 GAAAAAAAGG motifs detected in file\n",
      "\n",
      "Seq is: CCTCATCATT\n",
      "0 CCTCATCATT motifs detected in file\n",
      "\n",
      "Seq is: CAATGTTATA\n",
      "0 CAATGTTATA motifs detected in file\n",
      "\n",
      "Seq is: CCCTCTAATA\n",
      "0 CCCTCTAATA motifs detected in file\n",
      "\n",
      "Complement found from 6649 to 6659\n",
      "Seq is: AGGCTTCGGA\n",
      "1 AGGCTTCGGA motifs detected in file\n",
      "\n",
      "Seq is: CCATCCTATT\n",
      "0 CCATCCTATT motifs detected in file\n",
      "\n",
      "Seq is: CTTATATTTG\n",
      "0 CTTATATTTG motifs detected in file\n",
      "\n",
      "Complement found from 9588 to 9598\n",
      "Seq is: AAGTCCCACT\n",
      "1 AAGTCCCACT motifs detected in file\n",
      "\n",
      "Seq is: GGCTTTTTAG\n",
      "0 GGCTTTTTAG motifs detected in file\n",
      "\n",
      "Seq is: TCTTATCCAC\n",
      "0 TCTTATCCAC motifs detected in file\n",
      "\n",
      "Seq is: AACCATATCT\n",
      "0 AACCATATCT motifs detected in file\n",
      "\n",
      "Seq is: TACCCCAGAG\n",
      "0 TACCCCAGAG motifs detected in file\n",
      "\n",
      "Seq is: GAGGAGGCCG\n",
      "0 GAGGAGGCCG motifs detected in file\n",
      "\n",
      "Seq is: CCCCAGCTTC\n",
      "0 CCCCAGCTTC motifs detected in file\n",
      "\n",
      "Seq is: TTCCACCACA\n",
      "0 TTCCACCACA motifs detected in file\n",
      "\n",
      "Seq is: ACTCCAACGC\n",
      "0 ACTCCAACGC motifs detected in file\n",
      "\n",
      "Seq is: CAGACAATCA\n",
      "0 CAGACAATCA motifs detected in file\n",
      "\n",
      "Seq is: TCATATCACT\n",
      "0 TCATATCACT motifs detected in file\n",
      "\n",
      "Seq is: ACGGTCGTCG\n",
      "0 ACGGTCGTCG motifs detected in file\n",
      "\n",
      "Seq is: ATCCTGACCC\n",
      "0 ATCCTGACCC motifs detected in file\n",
      "\n",
      "Seq is: TCTCTATCCT\n",
      "0 TCTCTATCCT motifs detected in file\n",
      "\n",
      "Seq is: CTTCATCCTG\n",
      "0 CTTCATCCTG motifs detected in file\n",
      "\n",
      "Seq is: ACGACACTAA\n",
      "0 ACGACACTAA motifs detected in file\n",
      "\n",
      "Seq is: TCCTGCCTAG\n",
      "0 TCCTGCCTAG motifs detected in file\n",
      "\n",
      "Seq is: TCTGCTTCTC\n",
      "0 TCTGCTTCTC motifs detected in file\n",
      "\n",
      "Seq is: CCCGCAAACC\n",
      "0 CCCGCAAACC motifs detected in file\n",
      "\n",
      "Complement found from 6428 to 6438\n",
      "Seq is: ATAACCCAAT\n",
      "1 ATAACCCAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCGTCCCCC\n",
      "0 ACCGTCCCCC motifs detected in file\n",
      "\n",
      "Seq is: AAAATATAAC\n",
      "0 AAAATATAAC motifs detected in file\n",
      "\n",
      "Seq is: TATTAACAAC\n",
      "0 TATTAACAAC motifs detected in file\n",
      "\n",
      "Seq is: ACTACTTTAA\n",
      "0 ACTACTTTAA motifs detected in file\n",
      "\n",
      "Seq is: CTACCGGGGA\n",
      "0 CTACCGGGGA motifs detected in file\n",
      "\n",
      "Seq is: TTCTATCTCC\n",
      "0 TTCTATCTCC motifs detected in file\n",
      "\n",
      "Seq is: TCTGTGGACG\n",
      "0 TCTGTGGACG motifs detected in file\n",
      "\n",
      "Seq is: ATCCACATTC\n",
      "0 ATCCACATTC motifs detected in file\n",
      "\n",
      "Seq is: CTCTTCTACC\n",
      "0 CTCTTCTACC motifs detected in file\n",
      "\n",
      "Seq is: AATCCAGATG\n",
      "0 AATCCAGATG motifs detected in file\n",
      "\n",
      "Seq is: CCTCCGAGGT\n",
      "0 CCTCCGAGGT motifs detected in file\n",
      "\n",
      "Complement found from 15262 to 15272\n",
      "Seq is: CCCACCCTCA\n",
      "1 CCCACCCTCA motifs detected in file\n",
      "\n",
      "Seq is: TCAAGGGACG\n",
      "0 TCAAGGGACG motifs detected in file\n",
      "\n",
      "Seq is: ATTCCCTCCC\n",
      "0 ATTCCCTCCC motifs detected in file\n",
      "\n",
      "Seq is: ATCTGACAAG\n",
      "0 ATCTGACAAG motifs detected in file\n",
      "\n",
      "Seq is: TTGGACAAGG\n",
      "0 TTGGACAAGG motifs detected in file\n",
      "\n",
      "Seq is: ACGAGGCCGG\n",
      "0 ACGAGGCCGG motifs detected in file\n",
      "\n",
      "Seq is: AGGTGATATC\n",
      "0 AGGTGATATC motifs detected in file\n",
      "\n",
      "Seq is: GTCTACGCTC\n",
      "0 GTCTACGCTC motifs detected in file\n",
      "\n",
      "Seq is: GTCCTCATCC\n",
      "0 GTCCTCATCC motifs detected in file\n",
      "\n",
      "Complement found from 4902 to 4912\n",
      "Seq is: TCTCTCCCTC\n",
      "1 TCTCTCCCTC motifs detected in file\n",
      "\n",
      "Seq is: CATTCTCAGT\n",
      "0 CATTCTCAGT motifs detected in file\n",
      "\n",
      "Seq is: CTTCGAATAC\n",
      "0 CTTCGAATAC motifs detected in file\n",
      "\n",
      "Seq is: AACAAATACG\n",
      "0 AACAAATACG motifs detected in file\n",
      "\n",
      "Seq is: CCCCTTTGCG\n",
      "0 CCCCTTTGCG motifs detected in file\n",
      "\n",
      "Seq is: GTATAGCCCC\n",
      "0 GTATAGCCCC motifs detected in file\n",
      "\n",
      "Seq is: CGTGGCTAAT\n",
      "0 CGTGGCTAAT motifs detected in file\n",
      "\n",
      "Complement found from 2072 to 2082\n",
      "Seq is: AATCCCCTTG\n",
      "1 AATCCCCTTG motifs detected in file\n",
      "\n",
      "Seq is: ATCAGTCAAC\n",
      "0 ATCAGTCAAC motifs detected in file\n",
      "\n",
      "Seq is: GGTTTCGGAG\n",
      "0 GGTTTCGGAG motifs detected in file\n",
      "\n",
      "Seq is: GCTAATACTA\n",
      "0 GCTAATACTA motifs detected in file\n",
      "\n",
      "Seq is: CCCATAATGA\n",
      "0 CCCATAATGA motifs detected in file\n",
      "\n",
      "Seq is: TACTTCTTCT\n",
      "0 TACTTCTTCT motifs detected in file\n",
      "\n",
      "Seq is: AATAATGTTT\n",
      "0 AATAATGTTT motifs detected in file\n",
      "\n",
      "Seq is: ACGTACCCGA\n",
      "0 ACGTACCCGA motifs detected in file\n",
      "\n",
      "Seq is: CACTGCTATT\n",
      "0 CACTGCTATT motifs detected in file\n",
      "\n",
      "Seq is: GCAACATCTA\n",
      "0 GCAACATCTA motifs detected in file\n",
      "\n",
      "Seq is: CACCAGCAAT\n",
      "0 CACCAGCAAT motifs detected in file\n",
      "\n",
      "Seq is: GGATCTTCCA\n",
      "0 GGATCTTCCA motifs detected in file\n",
      "\n",
      "Seq is: ACGGACCGAC\n",
      "0 ACGGACCGAC motifs detected in file\n",
      "\n",
      "Seq is: CGGGTCGAGC\n",
      "0 CGGGTCGAGC motifs detected in file\n",
      "\n",
      "Seq is: CGAGCTTATT\n",
      "0 CGAGCTTATT motifs detected in file\n",
      "\n",
      "Seq is: CCTCCGAATC\n",
      "0 CCTCCGAATC motifs detected in file\n",
      "\n",
      "Seq is: TCGACACGGA\n",
      "0 TCGACACGGA motifs detected in file\n",
      "\n",
      "Seq is: TCCTGAGGTC\n",
      "0 TCCTGAGGTC motifs detected in file\n",
      "\n",
      "Seq is: GAGTACGCGG\n",
      "0 GAGTACGCGG motifs detected in file\n",
      "\n",
      "Seq is: CTTATTATCC\n",
      "0 CTTATTATCC motifs detected in file\n",
      "\n",
      "Seq is: ATATCACAAG\n",
      "0 ATATCACAAG motifs detected in file\n",
      "\n",
      "Seq is: GTTACAGAAA\n",
      "0 GTTACAGAAA motifs detected in file\n",
      "\n",
      "Seq is: CACCAAACAT\n",
      "0 CACCAAACAT motifs detected in file\n",
      "\n",
      "Seq is: CTCTTATCAG\n",
      "0 CTCTTATCAG motifs detected in file\n",
      "\n",
      "Seq is: TTGCCAGCCG\n",
      "0 TTGCCAGCCG motifs detected in file\n",
      "\n",
      "Seq is: CTTGTAGTCA\n",
      "0 CTTGTAGTCA motifs detected in file\n",
      "\n",
      "Seq is: CCCCCACTCC\n",
      "0 CCCCCACTCC motifs detected in file\n",
      "\n",
      "Seq is: ATTTTACCGA\n",
      "0 ATTTTACCGA motifs detected in file\n",
      "\n",
      "Seq is: CTCACTTCGT\n",
      "0 CTCACTTCGT motifs detected in file\n",
      "\n",
      "Seq is: AACCTGACAT\n",
      "0 AACCTGACAT motifs detected in file\n",
      "\n",
      "Seq is: TTAGATTTCT\n",
      "0 TTAGATTTCT motifs detected in file\n",
      "\n",
      "Seq is: GTCCCCAATC\n",
      "0 GTCCCCAATC motifs detected in file\n",
      "\n",
      "Seq is: CGGAGAAAAA\n",
      "0 CGGAGAAAAA motifs detected in file\n",
      "\n",
      "Seq is: TGGTCGAGGC\n",
      "0 TGGTCGAGGC motifs detected in file\n",
      "\n",
      "Seq is: TCCACTAAAA\n",
      "0 TCCACTAAAA motifs detected in file\n",
      "\n",
      "Seq is: GTATAACTTA\n",
      "0 GTATAACTTA motifs detected in file\n",
      "\n",
      "Seq is: ACGTTTAAGC\n",
      "0 ACGTTTAAGC motifs detected in file\n",
      "\n",
      "Seq is: TTCTTCGTCG\n",
      "0 TTCTTCGTCG motifs detected in file\n",
      "\n",
      "Seq is: AAGTTTGGAC\n",
      "0 AAGTTTGGAC motifs detected in file\n",
      "\n",
      "Seq is: GGCCCCGAAG\n",
      "0 GGCCCCGAAG motifs detected in file\n",
      "\n",
      "Seq is: AGGGCGGAAA\n",
      "0 AGGGCGGAAA motifs detected in file\n",
      "\n",
      "Seq is: AAAGGGCCGC\n",
      "0 AAAGGGCCGC motifs detected in file\n",
      "\n",
      "Seq is: CGCCCTCTTC\n",
      "0 CGCCCTCTTC motifs detected in file\n",
      "\n",
      "Seq is: ATCTAACTTC\n",
      "0 ATCTAACTTC motifs detected in file\n",
      "\n",
      "Seq is: GGTCAACTAA\n",
      "0 GGTCAACTAA motifs detected in file\n",
      "\n",
      "Seq is: TCCCACGAAT\n",
      "0 TCCCACGAAT motifs detected in file\n",
      "\n",
      "Seq is: CGACAATTGA\n",
      "0 CGACAATTGA motifs detected in file\n",
      "\n",
      "Seq is: TTCACAAACA\n",
      "0 TTCACAAACA motifs detected in file\n",
      "\n",
      "Seq is: CCCAAATTCA\n",
      "0 CCCAAATTCA motifs detected in file\n",
      "\n",
      "Seq is: GGGTAACCAG\n",
      "0 GGGTAACCAG motifs detected in file\n",
      "\n",
      "Seq is: ATCATTCCCG\n",
      "0 ATCATTCCCG motifs detected in file\n",
      "\n",
      "Seq is: AATCGAATTA\n",
      "0 AATCGAATTA motifs detected in file\n",
      "\n",
      "Seq is: ATTTCACCGA\n",
      "0 ATTTCACCGA motifs detected in file\n",
      "\n",
      "Seq is: CTAAACGCAA\n",
      "0 CTAAACGCAA motifs detected in file\n",
      "\n",
      "Seq is: GTCAACTACG\n",
      "0 GTCAACTACG motifs detected in file\n",
      "\n",
      "Seq is: TCTCACCCCA\n",
      "0 TCTCACCCCA motifs detected in file\n",
      "\n",
      "Seq is: AAACGTCAGG\n",
      "0 AAACGTCAGG motifs detected in file\n",
      "\n",
      "Seq is: AATCGACAAT\n",
      "0 AATCGACAAT motifs detected in file\n",
      "\n",
      "Seq is: GTCTTTAATT\n",
      "0 GTCTTTAATT motifs detected in file\n",
      "\n",
      "Seq is: CATAACGTTG\n",
      "0 CATAACGTTG motifs detected in file\n",
      "\n",
      "Seq is: AATGACTCCC\n",
      "0 AATGACTCCC motifs detected in file\n",
      "\n",
      "Seq is: GAAACTTCCG\n",
      "0 GAAACTTCCG motifs detected in file\n",
      "\n",
      "Seq is: AGAACCAGAC\n",
      "0 AGAACCAGAC motifs detected in file\n",
      "\n",
      "Seq is: ATAAATTGGA\n",
      "0 ATAAATTGGA motifs detected in file\n",
      "\n",
      "Seq is: TTTAAAGATA\n",
      "0 TTTAAAGATA motifs detected in file\n",
      "\n",
      "Seq is: TTCTAATAAT\n",
      "0 TTCTAATAAT motifs detected in file\n",
      "\n",
      "Seq is: CATATTTTCC\n",
      "0 CATATTTTCC motifs detected in file\n",
      "\n",
      "Seq is: CCTCTATCCA\n",
      "0 CCTCTATCCA motifs detected in file\n",
      "\n",
      "Complement found from 7805 to 7815\n",
      "Seq is: TCCTCATCGC\n",
      "1 TCCTCATCGC motifs detected in file\n",
      "\n",
      "Seq is: ACCATTCCCG\n",
      "0 ACCATTCCCG motifs detected in file\n",
      "\n",
      "Seq is: CTACTCACAC\n",
      "0 CTACTCACAC motifs detected in file\n",
      "\n",
      "Seq is: CCCTCCTTAC\n",
      "0 CCCTCCTTAC motifs detected in file\n",
      "\n",
      "Seq is: CCCACCCAAA\n",
      "0 CCCACCCAAA motifs detected in file\n",
      "\n",
      "Seq is: ACATACAAGT\n",
      "0 ACATACAAGT motifs detected in file\n",
      "\n",
      "Seq is: TTGACAGTAA\n",
      "0 TTGACAGTAA motifs detected in file\n",
      "\n",
      "Seq is: AATAAAAATG\n",
      "0 AATAAAAATG motifs detected in file\n",
      "\n",
      "Seq is: CAACAATCTA\n",
      "0 CAACAATCTA motifs detected in file\n",
      "\n",
      "Seq is: TACCCCTCAT\n",
      "0 TACCCCTCAT motifs detected in file\n",
      "\n",
      "Seq is: CACACTAACT\n",
      "0 CACACTAACT motifs detected in file\n",
      "\n",
      "Seq is: CCACCTCATC\n",
      "0 CCACCTCATC motifs detected in file\n",
      "\n",
      "Seq is: TAATCCGCAT\n",
      "0 TAATCCGCAT motifs detected in file\n",
      "\n",
      "Seq is: CCATCTTCAT\n",
      "0 CCATCTTCAT motifs detected in file\n",
      "\n",
      "Seq is: CTCCAATTCC\n",
      "0 CTCCAATTCC motifs detected in file\n",
      "\n",
      "Seq is: TCCCACTACC\n",
      "0 TCCCACTACC motifs detected in file\n",
      "\n",
      "Seq is: ACCGATACTA\n",
      "0 ACCGATACTA motifs detected in file\n",
      "\n",
      "Seq is: CCACCCCTAC\n",
      "0 CCACCCCTAC motifs detected in file\n",
      "\n",
      "Seq is: TACTCCGATA\n",
      "0 TACTCCGATA motifs detected in file\n",
      "\n",
      "Complement found from 5279 to 5289\n",
      "Seq is: ACAAAAAACA\n",
      "1 ACAAAAAACA motifs detected in file\n",
      "\n",
      "Seq is: CTTAAGAAGC\n",
      "0 CTTAAGAAGC motifs detected in file\n",
      "\n",
      "Seq is: TATTACCGGG\n",
      "0 TATTACCGGG motifs detected in file\n",
      "\n",
      "Seq is: TAAACCCGTT\n",
      "0 TAAACCCGTT motifs detected in file\n",
      "\n",
      "Seq is: TTTCGGCCAA\n",
      "0 TTTCGGCCAA motifs detected in file\n",
      "\n",
      "Seq is: TCGCCCCCGT\n",
      "0 TCGCCCCCGT motifs detected in file\n",
      "\n",
      "Seq is: CCGGAGGATC\n",
      "0 CCGGAGGATC motifs detected in file\n",
      "\n",
      "Seq is: CCTCTCCTCC\n",
      "0 CCTCTCCTCC motifs detected in file\n",
      "\n",
      "Seq is: CACCTACCTT\n",
      "0 CACCTACCTT motifs detected in file\n",
      "\n",
      "Seq is: AATTCCCACA\n",
      "0 AATTCCCACA motifs detected in file\n",
      "\n",
      "Seq is: ATCAGTACAA\n",
      "0 ATCAGTACAA motifs detected in file\n",
      "\n",
      "Seq is: TCGAACAAAG\n",
      "0 TCGAACAAAG motifs detected in file\n",
      "\n",
      "Seq is: TCCACGCTCT\n",
      "0 TCCACGCTCT motifs detected in file\n",
      "\n",
      "Complement found from 10340 to 10350\n",
      "Seq is: ATCATCATCC\n",
      "1 ATCATCATCC motifs detected in file\n",
      "\n",
      "Complement found from 5137 to 5147\n",
      "Seq is: CAGCACCACG\n",
      "1 CAGCACCACG motifs detected in file\n",
      "\n",
      "Seq is: ACCTCAAATT\n",
      "0 ACCTCAAATT motifs detected in file\n",
      "\n",
      "Seq is: CAACTCATCA\n",
      "0 CAACTCATCA motifs detected in file\n",
      "\n",
      "Seq is: TCCTTACGCC\n",
      "0 TCCTTACGCC motifs detected in file\n",
      "\n",
      "Seq is: ATCATCAATC\n",
      "0 ATCATCAATC motifs detected in file\n",
      "\n",
      "Seq is: CTATTATATT\n",
      "0 CTATTATATT motifs detected in file\n",
      "\n",
      "Seq is: TATCAATTTA\n",
      "0 TATCAATTTA motifs detected in file\n",
      "\n",
      "Seq is: ATTCTTACCA\n",
      "0 ATTCTTACCA motifs detected in file\n",
      "\n",
      "Seq is: ATACAATCCC\n",
      "0 ATACAATCCC motifs detected in file\n",
      "\n",
      "Seq is: AACATGCCAT\n",
      "0 AACATGCCAT motifs detected in file\n",
      "\n",
      "Seq is: CTTGACGATA\n",
      "0 CTTGACGATA motifs detected in file\n",
      "\n",
      "Seq is: ATAAGTAGGA\n",
      "0 ATAAGTAGGA motifs detected in file\n",
      "\n",
      "Seq is: TACACCCATT\n",
      "0 TACACCCATT motifs detected in file\n",
      "\n",
      "Seq is: AACTCCTCAT\n",
      "0 AACTCCTCAT motifs detected in file\n",
      "\n",
      "Seq is: ACGATTCTAA\n",
      "0 ACGATTCTAA motifs detected in file\n",
      "\n",
      "Seq is: AACGCATCGA\n",
      "0 AACGCATCGA motifs detected in file\n",
      "\n",
      "Seq is: CCCAAACCAA\n",
      "0 CCCAAACCAA motifs detected in file\n",
      "\n",
      "Seq is: ATTAGGTGGA\n",
      "0 ATTAGGTGGA motifs detected in file\n",
      "\n",
      "Seq is: GTTGACGGAC\n",
      "0 GTTGACGGAC motifs detected in file\n",
      "\n",
      "Seq is: GATACTACCT\n",
      "0 GATACTACCT motifs detected in file\n",
      "\n",
      "Seq is: ATTCTAACTC\n",
      "0 ATTCTAACTC motifs detected in file\n",
      "\n",
      "Seq is: TCTCACTCCT\n",
      "0 TCTCACTCCT motifs detected in file\n",
      "\n",
      "Seq is: CTTCCGAATG\n",
      "0 CTTCCGAATG motifs detected in file\n",
      "\n",
      "Seq is: CAAATCACTC\n",
      "0 CAAATCACTC motifs detected in file\n",
      "\n",
      "Seq is: CCTCTCTAAA\n",
      "0 CCTCTCTAAA motifs detected in file\n",
      "\n",
      "Seq is: CCATATACTA\n",
      "0 CCATATACTA motifs detected in file\n",
      "\n",
      "Seq is: ACTCTACCCC\n",
      "0 ACTCTACCCC motifs detected in file\n",
      "\n",
      "Seq is: CGATCAAAAA\n",
      "0 CGATCAAAAA motifs detected in file\n",
      "\n",
      "Seq is: CAGTACACTC\n",
      "0 CAGTACACTC motifs detected in file\n",
      "\n",
      "Seq is: TTCTTCGTCC\n",
      "0 TTCTTCGTCC motifs detected in file\n",
      "\n",
      "Seq is: GGCCTACAGT\n",
      "0 GGCCTACAGT motifs detected in file\n",
      "\n",
      "Seq is: CTCCCCACGG\n",
      "0 CTCCCCACGG motifs detected in file\n",
      "\n",
      "Seq is: AACCCATTGG\n",
      "0 AACCCATTGG motifs detected in file\n",
      "\n",
      "Seq is: AGACCCTGAG\n",
      "0 AGACCCTGAG motifs detected in file\n",
      "\n",
      "Seq is: TCTTCACTTT\n",
      "0 TCTTCACTTT motifs detected in file\n",
      "\n",
      "Seq is: CCCCCGATAA\n",
      "0 CCCCCGATAA motifs detected in file\n",
      "\n",
      "Seq is: GGATCAAAAT\n",
      "0 GGATCAAAAT motifs detected in file\n",
      "\n",
      "Seq is: AACGATATCG\n",
      "0 AACGATATCG motifs detected in file\n",
      "\n",
      "Seq is: ATAATACTAA\n",
      "0 ATAATACTAA motifs detected in file\n",
      "\n",
      "Seq is: TAATTACTAC\n",
      "0 TAATTACTAC motifs detected in file\n",
      "\n",
      "Seq is: TCATAACTAA\n",
      "0 TCATAACTAA motifs detected in file\n",
      "\n",
      "Seq is: CCATCATAAC\n",
      "0 CCATCATAAC motifs detected in file\n",
      "\n",
      "Seq is: CAATACCAAG\n",
      "0 CAATACCAAG motifs detected in file\n",
      "\n",
      "Seq is: TAACAGGCCT\n",
      "0 TAACAGGCCT motifs detected in file\n",
      "\n",
      "Seq is: CTCATATAAC\n",
      "0 CTCATATAAC motifs detected in file\n",
      "\n",
      "Seq is: AACTTCTCCT\n",
      "0 AACTTCTCCT motifs detected in file\n",
      "\n",
      "Seq is: ATCGATAATC\n",
      "0 ATCGATAATC motifs detected in file\n",
      "\n",
      "Seq is: TTCCTAATAC\n",
      "0 TTCCTAATAC motifs detected in file\n",
      "\n",
      "Seq is: CTACGCCAAC\n",
      "0 CTACGCCAAC motifs detected in file\n",
      "\n",
      "Complement found from 11761 to 11771\n",
      "Seq is: GAACGCACTC\n",
      "1 GAACGCACTC motifs detected in file\n",
      "\n",
      "Seq is: CTTTATGAAC\n",
      "0 CTTTATGAAC motifs detected in file\n",
      "\n",
      "Seq is: TACCGTCGAA\n",
      "0 TACCGTCGAA motifs detected in file\n",
      "\n",
      "Seq is: GACACCTTGC\n",
      "0 GACACCTTGC motifs detected in file\n",
      "\n",
      "Seq is: TCCCAAATAA\n",
      "0 TCCCAAATAA motifs detected in file\n",
      "\n",
      "Seq is: AAAAACCAAT\n",
      "0 AAAAACCAAT motifs detected in file\n",
      "\n",
      "Seq is: CTTGACCTTA\n",
      "0 CTTGACCTTA motifs detected in file\n",
      "\n",
      "Seq is: TTTTCGATCG\n",
      "0 TTTTCGATCG motifs detected in file\n",
      "\n",
      "Seq is: TACAAATAAA\n",
      "0 TACAAATAAA motifs detected in file\n",
      "\n",
      "Seq is: GATCCGGATG\n",
      "0 GATCCGGATG motifs detected in file\n",
      "\n",
      "Seq is: AGTCCATTTT\n",
      "0 AGTCCATTTT motifs detected in file\n",
      "\n",
      "Seq is: TTAGTCACGC\n",
      "0 TTAGTCACGC motifs detected in file\n",
      "\n",
      "Seq is: TCGAATCGCG\n",
      "0 TCGAATCGCG motifs detected in file\n",
      "\n",
      "Seq is: ACACTACTCA\n",
      "0 ACACTACTCA motifs detected in file\n",
      "\n",
      "Seq is: CACGGACGTT\n",
      "0 CACGGACGTT motifs detected in file\n",
      "\n",
      "Complement found from 4499 to 4509\n",
      "Seq is: TCTACCATCT\n",
      "1 TCTACCATCT motifs detected in file\n",
      "\n",
      "Seq is: CATCTACTGC\n",
      "0 CATCTACTGC motifs detected in file\n",
      "\n",
      "Seq is: CCAACCCGGT\n",
      "0 CCAACCCGGT motifs detected in file\n",
      "\n",
      "Seq is: CCCCTAATTA\n",
      "0 CCCCTAATTA motifs detected in file\n",
      "\n",
      "Seq is: ATCATGCCCT\n",
      "0 ATCATGCCCT motifs detected in file\n",
      "\n",
      "Complement found from 6665 to 6675\n",
      "Seq is: TCCCATATTG\n",
      "1 TCCCATATTG motifs detected in file\n",
      "\n",
      "Seq is: GTTGTAAAAG\n",
      "0 GTTGTAAAAG motifs detected in file\n",
      "\n",
      "Complement found from 533 to 543\n",
      "Complement found from 3572 to 3582\n",
      "Seq is: CCCCATACCC\n",
      "2 CCCCATACCC motifs detected in file\n",
      "\n",
      "Seq is: GGGCTATCGA\n",
      "0 GGGCTATCGA motifs detected in file\n",
      "\n",
      "Seq is: ATAAATCGAC\n",
      "0 ATAAATCGAC motifs detected in file\n",
      "\n",
      "Seq is: TGGAATGAAA\n",
      "0 TGGAATGAAA motifs detected in file\n",
      "\n",
      "Seq is: TCCTACCCCA\n",
      "0 TCCTACCCCA motifs detected in file\n",
      "\n",
      "Seq is: CACTATCCAC\n",
      "0 CACTATCCAC motifs detected in file\n",
      "\n",
      "Seq is: CGTGCCTCTT\n",
      "0 CGTGCCTCTT motifs detected in file\n",
      "\n",
      "Seq is: AAAACCTAAG\n",
      "0 AAAACCTAAG motifs detected in file\n",
      "\n",
      "Seq is: AGTCCCTACC\n",
      "0 AGTCCCTACC motifs detected in file\n",
      "\n",
      "Seq is: CAAGCTAAGA\n",
      "0 CAAGCTAAGA motifs detected in file\n",
      "\n",
      "Seq is: GTATCAGGAT\n",
      "0 GTATCAGGAT motifs detected in file\n",
      "\n",
      "Seq is: CTTTATTCCC\n",
      "0 CTTTATTCCC motifs detected in file\n",
      "\n",
      "Seq is: CCAAATTCGA\n",
      "0 CCAAATTCGA motifs detected in file\n",
      "\n",
      "Seq is: GGATAATAAA\n",
      "0 GGATAATAAA motifs detected in file\n",
      "\n",
      "Seq is: TGAGATAGTT\n",
      "0 TGAGATAGTT motifs detected in file\n",
      "\n",
      "Seq is: TCATTGAGAA\n",
      "0 TCATTGAGAA motifs detected in file\n",
      "\n",
      "Seq is: AATAGTCTGT\n",
      "0 AATAGTCTGT motifs detected in file\n",
      "\n",
      "Seq is: ATAAAGAATC\n",
      "0 ATAAAGAATC motifs detected in file\n",
      "\n",
      "Seq is: CAAACTCCCC\n",
      "0 CAAACTCCCC motifs detected in file\n",
      "\n",
      "Seq is: CTTACGACCT\n",
      "0 CTTACGACCT motifs detected in file\n",
      "\n",
      "Complement found from 9010 to 9020\n",
      "Seq is: CTAACATTAC\n",
      "1 CTAACATTAC motifs detected in file\n",
      "\n",
      "Complement found from 13429 to 13439\n",
      "Seq is: CCATACCTCT\n",
      "1 CCATACCTCT motifs detected in file\n",
      "\n",
      "Seq is: GTATAGTATA\n",
      "0 GTATAGTATA motifs detected in file\n",
      "\n",
      "Seq is: TTCATTACGA\n",
      "0 TTCATTACGA motifs detected in file\n",
      "\n",
      "Seq is: TCCCACTCAC\n",
      "0 TCCCACTCAC motifs detected in file\n",
      "\n",
      "Seq is: CATCCTTCAA\n",
      "0 CATCCTTCAA motifs detected in file\n",
      "\n",
      "Seq is: AAAAGTATCC\n",
      "0 AAAAGTATCC motifs detected in file\n",
      "\n",
      "Seq is: TCCACATACT\n",
      "0 TCCACATACT motifs detected in file\n",
      "\n",
      "Seq is: CAACCAGCAT\n",
      "0 CAACCAGCAT motifs detected in file\n",
      "\n",
      "Seq is: CGCCTTAGCC\n",
      "0 CGCCTTAGCC motifs detected in file\n",
      "\n",
      "Seq is: CCCATACGAC\n",
      "0 CCCATACGAC motifs detected in file\n",
      "\n",
      "Seq is: AAGCTTAAGT\n",
      "0 AAGCTTAAGT motifs detected in file\n",
      "\n",
      "Seq is: ATTCTTGTCC\n",
      "0 ATTCTTGTCC motifs detected in file\n",
      "\n",
      "Seq is: CTCCAATCTT\n",
      "0 CTCCAATCTT motifs detected in file\n",
      "\n",
      "Seq is: CATCCCAGAA\n",
      "0 CATCCCAGAA motifs detected in file\n",
      "\n",
      "Seq is: CCACTGTTTT\n",
      "0 CCACTGTTTT motifs detected in file\n",
      "\n",
      "Seq is: ATACAACACA\n",
      "0 ATACAACACA motifs detected in file\n",
      "\n",
      "Seq is: TCTCAAGTCC\n",
      "0 TCTCAAGTCC motifs detected in file\n",
      "\n",
      "Seq is: CCTCTCACGC\n",
      "0 CCTCTCACGC motifs detected in file\n",
      "\n",
      "Seq is: AGTATACAAC\n",
      "0 AGTATACAAC motifs detected in file\n",
      "\n",
      "Seq is: AAGGATCCTT\n",
      "0 AAGGATCCTT motifs detected in file\n",
      "\n",
      "Seq is: CTAACATCAC\n",
      "0 CTAACATCAC motifs detected in file\n",
      "\n",
      "Seq is: CACTCCCACA\n",
      "0 CACTCCCACA motifs detected in file\n",
      "\n",
      "Seq is: AATAATATTA\n",
      "0 AATAATATTA motifs detected in file\n",
      "\n",
      "Seq is: TTACAAACAC\n",
      "0 TTACAAACAC motifs detected in file\n",
      "\n",
      "Seq is: ATAAGCCGAT\n",
      "0 ATAAGCCGAT motifs detected in file\n",
      "\n",
      "Seq is: ACTTCTTATC\n",
      "0 ACTTCTTATC motifs detected in file\n",
      "\n",
      "Seq is: CCGCTTCCCC\n",
      "0 CCGCTTCCCC motifs detected in file\n",
      "\n",
      "Seq is: GGACGCCGCA\n",
      "0 GGACGCCGCA motifs detected in file\n",
      "\n",
      "Seq is: TAAGCTACAA\n",
      "0 TAAGCTACAA motifs detected in file\n",
      "\n",
      "Seq is: CTTCGGACTC\n",
      "0 CTTCGGACTC motifs detected in file\n",
      "\n",
      "Seq is: TGATCAAGCC\n",
      "0 TGATCAAGCC motifs detected in file\n",
      "\n",
      "Seq is: TGAGGGGAAG\n",
      "0 TGAGGGGAAG motifs detected in file\n",
      "\n",
      "Seq is: CCGTTCCAGC\n",
      "0 CCGTTCCAGC motifs detected in file\n",
      "\n",
      "Seq is: TTCCCCCAAG\n",
      "0 TTCCCCCAAG motifs detected in file\n",
      "\n",
      "Seq is: CCAACCAGAG\n",
      "0 CCAACCAGAG motifs detected in file\n",
      "\n",
      "Seq is: ACGATCACAC\n",
      "0 ACGATCACAC motifs detected in file\n",
      "\n",
      "Seq is: CTCTATTTAG\n",
      "0 CTCTATTTAG motifs detected in file\n",
      "\n",
      "Seq is: TATAATACCG\n",
      "0 TATAATACCG motifs detected in file\n",
      "\n",
      "Seq is: GTTCCCAGTA\n",
      "0 GTTCCCAGTA motifs detected in file\n",
      "\n",
      "Seq is: CTACCGTCCT\n",
      "0 CTACCGTCCT motifs detected in file\n",
      "\n",
      "Seq is: CATTAGTCTC\n",
      "0 CATTAGTCTC motifs detected in file\n",
      "\n",
      "Complement found from 3808 to 3818\n",
      "Seq is: CACAAGAACA\n",
      "1 CACAAGAACA motifs detected in file\n",
      "\n",
      "Seq is: CAACACTATT\n",
      "0 CAACACTATT motifs detected in file\n",
      "\n",
      "Seq is: CCCACCTCTC\n",
      "0 CCCACCTCTC motifs detected in file\n",
      "\n",
      "Seq is: CAATTTCCTC\n",
      "0 CAATTTCCTC motifs detected in file\n",
      "\n",
      "Seq is: GGTGAATAAT\n",
      "0 GGTGAATAAT motifs detected in file\n",
      "\n",
      "Seq is: CATTACAACT\n",
      "0 CATTACAACT motifs detected in file\n",
      "\n",
      "Seq is: ATCATCTTAC\n",
      "0 ATCATCTTAC motifs detected in file\n",
      "\n",
      "Seq is: TACCGATCCC\n",
      "0 TACCGATCCC motifs detected in file\n",
      "\n",
      "Seq is: ACTGAAGTAT\n",
      "0 ACTGAAGTAT motifs detected in file\n",
      "\n",
      "Seq is: ACTCTAACAA\n",
      "0 ACTCTAACAA motifs detected in file\n",
      "\n",
      "Seq is: ACCCGATGAC\n",
      "0 ACCCGATGAC motifs detected in file\n",
      "\n",
      "Seq is: GAGCGTCACG\n",
      "0 GAGCGTCACG motifs detected in file\n",
      "\n",
      "Seq is: CGGCTAGTCC\n",
      "0 CGGCTAGTCC motifs detected in file\n",
      "\n",
      "Seq is: CGCATCAAAC\n",
      "0 CGCATCAAAC motifs detected in file\n",
      "\n",
      "Complement found from 3678 to 3688\n",
      "Complement found from 11752 to 11762\n",
      "Seq is: TCAAACTACG\n",
      "2 TCAAACTACG motifs detected in file\n",
      "\n",
      "Seq is: AGTGGGACTA\n",
      "0 AGTGGGACTA motifs detected in file\n",
      "\n",
      "Seq is: GTCTCCTAAC\n",
      "0 GTCTCCTAAC motifs detected in file\n",
      "\n",
      "Seq is: TCATTTGCCG\n",
      "0 TCATTTGCCG motifs detected in file\n",
      "\n",
      "Seq is: ATCCGATCTC\n",
      "0 ATCCGATCTC motifs detected in file\n",
      "\n",
      "Seq is: CACCGATCTT\n",
      "0 CACCGATCTT motifs detected in file\n",
      "\n",
      "Seq is: ATTTATCCTC\n",
      "0 ATTTATCCTC motifs detected in file\n",
      "\n",
      "Seq is: CGGATCCAAC\n",
      "0 CGGATCCAAC motifs detected in file\n",
      "\n",
      "Seq is: TCCAACTGGT\n",
      "0 TCCAACTGGT motifs detected in file\n",
      "\n",
      "Seq is: CCCCCAACCC\n",
      "0 CCCCCAACCC motifs detected in file\n",
      "\n",
      "Seq is: ATACCCCTCC\n",
      "0 ATACCCCTCC motifs detected in file\n",
      "\n",
      "Seq is: CCCCAAGTAT\n",
      "0 CCCCAAGTAT motifs detected in file\n",
      "\n",
      "Seq is: CATCTTCTCG\n",
      "0 CATCTTCTCG motifs detected in file\n",
      "\n",
      "Seq is: CTACCACTCT\n",
      "0 CTACCACTCT motifs detected in file\n",
      "\n",
      "Seq is: CGATTCCAGC\n",
      "0 CGATTCCAGC motifs detected in file\n",
      "\n",
      "Seq is: CCCGCCACTA\n",
      "0 CCCGCCACTA motifs detected in file\n",
      "\n",
      "Seq is: CATCTCCCAC\n",
      "0 CATCTCCCAC motifs detected in file\n",
      "\n",
      "Seq is: TACCATCTAC\n",
      "0 TACCATCTAC motifs detected in file\n",
      "\n",
      "Seq is: ACCGCCCAAA\n",
      "0 ACCGCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: ATCCCCGAGA\n",
      "0 ATCCCCGAGA motifs detected in file\n",
      "\n",
      "Seq is: AACCACTTCT\n",
      "0 AACCACTTCT motifs detected in file\n",
      "\n",
      "Seq is: CAAAATACCG\n",
      "0 CAAAATACCG motifs detected in file\n",
      "\n",
      "Seq is: CAGTCGCTTC\n",
      "0 CAGTCGCTTC motifs detected in file\n",
      "\n",
      "Seq is: CCAACATCAT\n",
      "0 CCAACATCAT motifs detected in file\n",
      "\n",
      "Seq is: CGGGCATCCC\n",
      "0 CGGGCATCCC motifs detected in file\n",
      "\n",
      "Seq is: CGGATGTTGC\n",
      "0 CGGATGTTGC motifs detected in file\n",
      "\n",
      "Seq is: AACCCCGGAA\n",
      "0 AACCCCGGAA motifs detected in file\n",
      "\n",
      "Seq is: ACGCATCAAC\n",
      "0 ACGCATCAAC motifs detected in file\n",
      "\n",
      "Seq is: ATATATCGGA\n",
      "0 ATATATCGGA motifs detected in file\n",
      "\n",
      "Seq is: TCTTAAAAAG\n",
      "0 TCTTAAAAAG motifs detected in file\n",
      "\n",
      "Seq is: CAAGCCATTC\n",
      "0 CAAGCCATTC motifs detected in file\n",
      "\n",
      "Seq is: GTAATCCTTA\n",
      "0 GTAATCCTTA motifs detected in file\n",
      "\n",
      "Seq is: CGGTAACGCT\n",
      "0 CGGTAACGCT motifs detected in file\n",
      "\n",
      "Complement found from 2480 to 2490\n",
      "Seq is: AATCTTACCC\n",
      "1 AATCTTACCC motifs detected in file\n",
      "\n",
      "Seq is: ATGTTACTCC\n",
      "0 ATGTTACTCC motifs detected in file\n",
      "\n",
      "Seq is: TCATCCTCCA\n",
      "0 TCATCCTCCA motifs detected in file\n",
      "\n",
      "Seq is: ACCGGTACCC\n",
      "0 ACCGGTACCC motifs detected in file\n",
      "\n",
      "Seq is: ATACAACAAT\n",
      "0 ATACAACAAT motifs detected in file\n",
      "\n",
      "Complement found from 6342 to 6352\n",
      "Seq is: TCTTCTCCTT\n",
      "1 TCTTCTCCTT motifs detected in file\n",
      "\n",
      "Seq is: AACTTGGAGA\n",
      "0 AACTTGGAGA motifs detected in file\n",
      "\n",
      "Seq is: CTGACATTTC\n",
      "0 CTGACATTTC motifs detected in file\n",
      "\n",
      "Seq is: AAAATTCAAA\n",
      "0 AAAATTCAAA motifs detected in file\n",
      "\n",
      "Seq is: ATACGCTAAT\n",
      "0 ATACGCTAAT motifs detected in file\n",
      "\n",
      "Seq is: GGCCCGAGAC\n",
      "0 GGCCCGAGAC motifs detected in file\n",
      "\n",
      "Seq is: GGTAGAATTG\n",
      "0 GGTAGAATTG motifs detected in file\n",
      "\n",
      "Seq is: TTTGGGACAA\n",
      "0 TTTGGGACAA motifs detected in file\n",
      "\n",
      "Seq is: GAACCCACCC\n",
      "0 GAACCCACCC motifs detected in file\n",
      "\n",
      "Seq is: ACACCCATAT\n",
      "0 ACACCCATAT motifs detected in file\n",
      "\n",
      "Seq is: TATGATTCAA\n",
      "0 TATGATTCAA motifs detected in file\n",
      "\n",
      "Seq is: CTCTACTATA\n",
      "0 CTCTACTATA motifs detected in file\n",
      "\n",
      "Seq is: GTAAATGCCC\n",
      "0 GTAAATGCCC motifs detected in file\n",
      "\n",
      "Seq is: CCTTCCGCGA\n",
      "0 CCTTCCGCGA motifs detected in file\n",
      "\n",
      "Seq is: AACACTTCAT\n",
      "0 AACACTTCAT motifs detected in file\n",
      "\n",
      "Seq is: CCGGAATAAA\n",
      "0 CCGGAATAAA motifs detected in file\n",
      "\n",
      "Seq is: GAGAACAGGA\n",
      "0 GAGAACAGGA motifs detected in file\n",
      "\n",
      "Seq is: AAGCATGTCC\n",
      "0 AAGCATGTCC motifs detected in file\n",
      "\n",
      "Seq is: CTCCTTAAAC\n",
      "0 CTCCTTAAAC motifs detected in file\n",
      "\n",
      "Seq is: TTNCATCTAT\n",
      "0 TTNCATCTAT motifs detected in file\n",
      "\n",
      "Seq is: CTTTGGCTGG\n",
      "0 CTTTGGCTGG motifs detected in file\n",
      "\n",
      "Seq is: ACCTAATGAG\n",
      "0 ACCTAATGAG motifs detected in file\n",
      "\n",
      "Seq is: GCCAGACTTG\n",
      "0 GCCAGACTTG motifs detected in file\n",
      "\n",
      "Seq is: AGTCTAGTGC\n",
      "0 AGTCTAGTGC motifs detected in file\n",
      "\n",
      "Seq is: ATCCTGAAAT\n",
      "0 ATCCTGAAAT motifs detected in file\n",
      "\n",
      "Seq is: TAGCAACTTG\n",
      "0 TAGCAACTTG motifs detected in file\n",
      "\n",
      "Seq is: TTTGCTTGGA\n",
      "0 TTTGCTTGGA motifs detected in file\n",
      "\n",
      "Seq is: AATTATCGCC\n",
      "0 AATTATCGCC motifs detected in file\n",
      "\n",
      "Seq is: GACGTGGTAG\n",
      "0 GACGTGGTAG motifs detected in file\n",
      "\n",
      "Seq is: CCCTACAGGA\n",
      "0 CCCTACAGGA motifs detected in file\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Seq is: CTAGGTTGTA\n",
      "0 CTAGGTTGTA motifs detected in file\n",
      "\n",
      "Seq is: GCTCCAGCAT\n",
      "0 GCTCCAGCAT motifs detected in file\n",
      "\n",
      "Seq is: TTGGGATAAC\n",
      "0 TTGGGATAAC motifs detected in file\n",
      "\n",
      "Seq is: AACTATACCT\n",
      "0 AACTATACCT motifs detected in file\n",
      "\n",
      "Seq is: GAGATCTTAT\n",
      "0 GAGATCTTAT motifs detected in file\n",
      "\n",
      "Seq is: CCTAACGCGA\n",
      "0 CCTAACGCGA motifs detected in file\n",
      "\n",
      "Seq is: CAATAGGGAT\n",
      "0 CAATAGGGAT motifs detected in file\n",
      "\n",
      "Seq is: CCCATTGAAC\n",
      "0 CCCATTGAAC motifs detected in file\n",
      "\n",
      "Seq is: AAGGCAACCA\n",
      "0 AAGGCAACCA motifs detected in file\n",
      "\n",
      "Seq is: GTTCAATAAC\n",
      "0 GTTCAATAAC motifs detected in file\n",
      "\n",
      "Seq is: CTAGTTAACT\n",
      "0 CTAGTTAACT motifs detected in file\n",
      "\n",
      "Seq is: CATATCATCA\n",
      "0 CATATCATCA motifs detected in file\n",
      "\n",
      "Seq is: AGCGAAACTG\n",
      "0 AGCGAAACTG motifs detected in file\n",
      "\n",
      "Seq is: ACCACTTCAG\n",
      "0 ACCACTTCAG motifs detected in file\n",
      "\n",
      "Seq is: AATCGTACAT\n",
      "0 AATCGTACAT motifs detected in file\n",
      "\n",
      "Seq is: GACGAGCCTC\n",
      "0 GACGAGCCTC motifs detected in file\n",
      "\n",
      "Seq is: CAACCCAAGA\n",
      "0 CAACCCAAGA motifs detected in file\n",
      "\n",
      "Seq is: CGAGGCTCCA\n",
      "0 CGAGGCTCCA motifs detected in file\n",
      "\n",
      "Seq is: GCGGGGTTGG\n",
      "0 GCGGGGTTGG motifs detected in file\n",
      "\n",
      "Seq is: CTTTAAAAAT\n",
      "0 CTTTAAAAAT motifs detected in file\n",
      "\n",
      "Seq is: TACGTCCAAA\n",
      "0 TACGTCCAAA motifs detected in file\n",
      "\n",
      "Complement found from 7871 to 7881\n",
      "Seq is: CCATCAAATC\n",
      "1 CCATCAAATC motifs detected in file\n",
      "\n",
      "Seq is: CTGGACACCC\n",
      "0 CTGGACACCC motifs detected in file\n",
      "\n",
      "Seq is: AAACAATCCA\n",
      "0 AAACAATCCA motifs detected in file\n",
      "\n",
      "Seq is: TGACAAACGT\n",
      "0 TGACAAACGT motifs detected in file\n",
      "\n",
      "Seq is: AATTATTTAA\n",
      "0 AATTATTTAA motifs detected in file\n",
      "\n",
      "Seq is: TTTCGAGGTA\n",
      "0 TTTCGAGGTA motifs detected in file\n",
      "\n",
      "Seq is: TCCCAGAAGA\n",
      "0 TCCCAGAAGA motifs detected in file\n",
      "\n",
      "Seq is: GCAGAACGAC\n",
      "0 GCAGAACGAC motifs detected in file\n",
      "\n",
      "Seq is: ACAATACGGG\n",
      "0 ACAATACGGG motifs detected in file\n",
      "\n",
      "Seq is: CGGAGAAGTG\n",
      "0 CGGAGAAGTG motifs detected in file\n",
      "\n",
      "Seq is: CCCGTCCAGT\n",
      "0 CCCGTCCAGT motifs detected in file\n",
      "\n",
      "Seq is: TAAAGTGACC\n",
      "0 TAAAGTGACC motifs detected in file\n",
      "\n",
      "Seq is: AATTTTCATT\n",
      "0 AATTTTCATT motifs detected in file\n",
      "\n",
      "Seq is: CTCTGTCGAC\n",
      "0 CTCTGTCGAC motifs detected in file\n",
      "\n",
      "Seq is: TTGGGAGCAC\n",
      "0 TTGGGAGCAC motifs detected in file\n",
      "\n",
      "Seq is: CTCGGTAAGT\n",
      "0 CTCGGTAAGT motifs detected in file\n",
      "\n",
      "Seq is: ATGTCCAGGG\n",
      "0 ATGTCCAGGG motifs detected in file\n",
      "\n",
      "Seq is: ATAAATTCCT\n",
      "0 ATAAATTCCT motifs detected in file\n",
      "\n",
      "Seq is: TGTTCACTAA\n",
      "0 TGTTCACTAA motifs detected in file\n",
      "\n",
      "Seq is: TACGATGGAA\n",
      "0 TACGATGGAA motifs detected in file\n",
      "\n",
      "Seq is: ACGTGCCAAT\n",
      "0 ACGTGCCAAT motifs detected in file\n",
      "\n",
      "Seq is: CCCATGGCGC\n",
      "0 CCCATGGCGC motifs detected in file\n",
      "\n",
      "Seq is: CGGCAATTTG\n",
      "0 CGGCAATTTG motifs detected in file\n",
      "\n",
      "Seq is: TACACAGTGA\n",
      "0 TACACAGTGA motifs detected in file\n",
      "\n",
      "Seq is: CCCGTCCGCC\n",
      "0 CCCGTCCGCC motifs detected in file\n",
      "\n",
      "Seq is: ACGGAGATTA\n",
      "0 ACGGAGATTA motifs detected in file\n",
      "\n",
      "Seq is: TGACCACTAC\n",
      "0 TGACCACTAC motifs detected in file\n",
      "\n",
      "Seq is: GATCTCCACT\n",
      "0 GATCTCCACT motifs detected in file\n",
      "\n",
      "Seq is: ACAAAAACCA\n",
      "0 ACAAAAACCA motifs detected in file\n",
      "\n",
      "Seq is: TTTGTCCGCC\n",
      "0 TTTGTCCGCC motifs detected in file\n",
      "\n",
      "Seq is: CCATTCTAAA\n",
      "0 CCATTCTAAA motifs detected in file\n",
      "\n",
      "Seq is: CGGCTCAAGG\n",
      "0 CGGCTCAAGG motifs detected in file\n",
      "\n",
      "Seq is: AAAATGAAAA\n",
      "0 AAAATGAAAA motifs detected in file\n",
      "\n",
      "Seq is: AAATTGGAAA\n",
      "0 AAATTGGAAA motifs detected in file\n",
      "\n",
      "Seq is: GGAATACTCG\n",
      "0 GGAATACTCG motifs detected in file\n",
      "\n",
      "Seq is: TACGGACACA\n",
      "0 TACGGACACA motifs detected in file\n",
      "\n",
      "Seq is: ACCCAACTGT\n",
      "0 ACCCAACTGT motifs detected in file\n",
      "\n",
      "Seq is: CACTCCCATT\n",
      "0 CACTCCCATT motifs detected in file\n",
      "\n",
      "Seq is: ATTACTGAAC\n",
      "0 ATTACTGAAC motifs detected in file\n",
      "\n",
      "Seq is: AACCAACTAA\n",
      "0 AACCAACTAA motifs detected in file\n",
      "\n",
      "Seq is: CATCTATAAC\n",
      "0 CATCTATAAC motifs detected in file\n",
      "\n",
      "Seq is: CCGACAATTA\n",
      "0 CCGACAATTA motifs detected in file\n",
      "\n",
      "Seq is: ACAGTCAAGT\n",
      "0 ACAGTCAAGT motifs detected in file\n",
      "\n",
      "Seq is: CACAAAATTA\n",
      "0 CACAAAATTA motifs detected in file\n",
      "\n",
      "Seq is: GACTGCGTCC\n",
      "0 GACTGCGTCC motifs detected in file\n",
      "\n",
      "Seq is: GAATACGCCT\n",
      "0 GAATACGCCT motifs detected in file\n",
      "\n",
      "Seq is: CCTCTTACAA\n",
      "0 CCTCTTACAA motifs detected in file\n",
      "\n",
      "Seq is: AAGTACAATG\n",
      "0 AAGTACAATG motifs detected in file\n",
      "\n",
      "Seq is: AATATGATTG\n",
      "0 AATATGATTG motifs detected in file\n",
      "\n",
      "Seq is: TAATCAAGAA\n",
      "0 TAATCAAGAA motifs detected in file\n",
      "\n",
      "Seq is: GATATCCCAC\n",
      "0 GATATCCCAC motifs detected in file\n",
      "\n",
      "Seq is: TATCTAACCA\n",
      "0 TATCTAACCA motifs detected in file\n",
      "\n",
      "Seq is: GGTTAACCCA\n",
      "0 GGTTAACCCA motifs detected in file\n",
      "\n",
      "Seq is: CACTCCTCAA\n",
      "0 CACTCCTCAA motifs detected in file\n",
      "\n",
      "Seq is: GTCAATATAC\n",
      "0 GTCAATATAC motifs detected in file\n",
      "\n",
      "Seq is: AAACCCTAAA\n",
      "0 AAACCCTAAA motifs detected in file\n",
      "\n",
      "Seq is: AAATCCATCA\n",
      "0 AAATCCATCA motifs detected in file\n",
      "\n",
      "Seq is: CCCACAACTC\n",
      "0 CCCACAACTC motifs detected in file\n",
      "\n",
      "Seq is: GAACTTGCGA\n",
      "0 GAACTTGCGA motifs detected in file\n",
      "\n",
      "Seq is: AAGAATTAAC\n",
      "0 AAGAATTAAC motifs detected in file\n",
      "\n",
      "Seq is: CACCGACGAA\n",
      "0 CACCGACGAA motifs detected in file\n",
      "\n",
      "Seq is: AATCCGGATG\n",
      "0 AATCCGGATG motifs detected in file\n",
      "\n",
      "Seq is: ATACCCACAA\n",
      "0 ATACCCACAA motifs detected in file\n",
      "\n",
      "Seq is: TTTAAAAAAT\n",
      "0 TTTAAAAAAT motifs detected in file\n",
      "\n",
      "Seq is: GAGAGAGATG\n",
      "0 GAGAGAGATG motifs detected in file\n",
      "\n",
      "Complement found from 10809 to 10819\n",
      "Seq is: TTCCAAAAAA\n",
      "1 TTCCAAAAAA motifs detected in file\n",
      "\n",
      "Seq is: GGATCACAGG\n",
      "0 GGATCACAGG motifs detected in file\n",
      "\n",
      "Seq is: TTTCTCGACA\n",
      "0 TTTCTCGACA motifs detected in file\n",
      "\n",
      "Seq is: AGGAGAAACC\n",
      "0 AGGAGAAACC motifs detected in file\n",
      "\n",
      "Seq is: TGATTGTCAA\n",
      "0 TGATTGTCAA motifs detected in file\n",
      "\n",
      "Seq is: TTTAAATGTT\n",
      "0 TTTAAATGTT motifs detected in file\n",
      "\n",
      "Seq is: CCCCTAAATC\n",
      "0 CCCCTAAATC motifs detected in file\n",
      "\n",
      "Seq is: TCCCAAGACA\n",
      "0 TCCCAAGACA motifs detected in file\n",
      "\n",
      "Seq is: CCCGTTTAAA\n",
      "0 CCCGTTTAAA motifs detected in file\n",
      "\n",
      "Seq is: TTTCAACTTG\n",
      "0 TTTCAACTTG motifs detected in file\n",
      "\n",
      "Seq is: ATTCTAAGAT\n",
      "0 ATTCTAAGAT motifs detected in file\n",
      "\n",
      "Seq is: AGAACCTGTT\n",
      "0 AGAACCTGTT motifs detected in file\n",
      "\n",
      "Seq is: GGTCGATAGT\n",
      "0 GGTCGATAGT motifs detected in file\n",
      "\n",
      "Seq is: GGTCCGAGCC\n",
      "0 GGTCCGAGCC motifs detected in file\n",
      "\n",
      "Seq is: ATCCAAACAG\n",
      "0 ATCCAAACAG motifs detected in file\n",
      "\n",
      "Seq is: CGGAGATGGA\n",
      "0 CGGAGATGGA motifs detected in file\n",
      "\n",
      "Seq is: TATTTAGAAG\n",
      "0 TATTTAGAAG motifs detected in file\n",
      "\n",
      "Seq is: GGTGATAAAA\n",
      "0 GGTGATAAAA motifs detected in file\n",
      "\n",
      "Seq is: CGATGTATCT\n",
      "0 CGATGTATCT motifs detected in file\n",
      "\n",
      "Seq is: GCCCACACGA\n",
      "0 GCCCACACGA motifs detected in file\n",
      "\n",
      "Seq is: GAAAATCGAC\n",
      "0 GAAAATCGAC motifs detected in file\n",
      "\n",
      "Seq is: AAGAATCCAT\n",
      "0 AAGAATCCAT motifs detected in file\n",
      "\n",
      "Seq is: CGAGCAGACC\n",
      "0 CGAGCAGACC motifs detected in file\n",
      "\n",
      "Seq is: AAAGCCCCCA\n",
      "0 AAAGCCCCCA motifs detected in file\n",
      "\n",
      "Seq is: GAATCGAAAC\n",
      "0 GAATCGAAAC motifs detected in file\n",
      "\n",
      "Seq is: CGAGAGGAAC\n",
      "0 CGAGAGGAAC motifs detected in file\n",
      "\n",
      "Seq is: GTTTCAATAA\n",
      "0 GTTTCAATAA motifs detected in file\n",
      "\n",
      "Seq is: AGATCAATTA\n",
      "0 AGATCAATTA motifs detected in file\n",
      "\n",
      "Seq is: AGTAATACGT\n",
      "0 AGTAATACGT motifs detected in file\n",
      "\n",
      "Seq is: CTTCCATATC\n",
      "0 CTTCCATATC motifs detected in file\n",
      "\n",
      "Seq is: CCCAATCAGG\n",
      "0 CCCAATCAGG motifs detected in file\n",
      "\n",
      "Seq is: AACGATATAA\n",
      "0 AACGATATAA motifs detected in file\n",
      "\n",
      "Seq is: TACGAACCAA\n",
      "0 TACGAACCAA motifs detected in file\n",
      "\n",
      "Seq is: TATTAAAAAG\n",
      "0 TATTAAAAAG motifs detected in file\n",
      "\n",
      "Seq is: TAGAAAGGGA\n",
      "0 TAGAAAGGGA motifs detected in file\n",
      "\n",
      "Seq is: ACGCCATGAT\n",
      "0 ACGCCATGAT motifs detected in file\n",
      "\n",
      "Seq is: ATAGATAACG\n",
      "0 ATAGATAACG motifs detected in file\n",
      "\n",
      "Seq is: CGGTCCAAAG\n",
      "0 CGGTCCAAAG motifs detected in file\n",
      "\n",
      "Seq is: TTAAAGATAG\n",
      "0 TTAAAGATAG motifs detected in file\n",
      "\n",
      "Seq is: CGGATATGAA\n",
      "0 CGGATATGAA motifs detected in file\n",
      "\n",
      "Seq is: ATAAACCCAT\n",
      "0 ATAAACCCAT motifs detected in file\n",
      "\n",
      "Seq is: TTACCAAACC\n",
      "0 TTACCAAACC motifs detected in file\n",
      "\n",
      "Seq is: GATTCCAACA\n",
      "0 GATTCCAACA motifs detected in file\n",
      "\n",
      "Seq is: GACCATCATT\n",
      "0 GACCATCATT motifs detected in file\n",
      "\n",
      "Seq is: CCACCTCACC\n",
      "0 CCACCTCACC motifs detected in file\n",
      "\n",
      "Seq is: CAAACCCCGA\n",
      "0 CAAACCCCGA motifs detected in file\n",
      "\n",
      "Seq is: TCCAAATCGA\n",
      "0 TCCAAATCGA motifs detected in file\n",
      "\n",
      "Seq is: GTCTCGCCAG\n",
      "0 GTCTCGCCAG motifs detected in file\n",
      "\n",
      "Seq is: TTCAATTCAA\n",
      "0 TTCAATTCAA motifs detected in file\n",
      "\n",
      "Seq is: CTTTAGAGGA\n",
      "0 CTTTAGAGGA motifs detected in file\n",
      "\n",
      "Seq is: TTCACATTCA\n",
      "0 TTCACATTCA motifs detected in file\n",
      "\n",
      "Seq is: ACCCACGAAA\n",
      "0 ACCCACGAAA motifs detected in file\n",
      "\n",
      "Seq is: CACAATTCGA\n",
      "0 CACAATTCGA motifs detected in file\n",
      "\n",
      "Seq is: TGTGAGACCA\n",
      "0 TGTGAGACCA motifs detected in file\n",
      "\n",
      "Seq is: AGCAGGTTCA\n",
      "0 AGCAGGTTCA motifs detected in file\n",
      "\n",
      "Seq is: CGTGAAAGGT\n",
      "0 CGTGAAAGGT motifs detected in file\n",
      "\n",
      "Seq is: CATGTGAATG\n",
      "0 CATGTGAATG motifs detected in file\n",
      "\n",
      "Seq is: GTACAATGCT\n",
      "0 GTACAATGCT motifs detected in file\n",
      "\n",
      "Seq is: GAACAGAGGA\n",
      "0 GAACAGAGGA motifs detected in file\n",
      "\n",
      "Seq is: GATATATTTA\n",
      "0 GATATATTTA motifs detected in file\n",
      "\n",
      "Seq is: CGCATCCCCA\n",
      "0 CGCATCCCCA motifs detected in file\n",
      "\n",
      "Seq is: AAATCAATTT\n",
      "0 AAATCAATTT motifs detected in file\n",
      "\n",
      "Seq is: ACAGGAAACT\n",
      "0 ACAGGAAACT motifs detected in file\n",
      "\n",
      "Seq is: TCATATGAAC\n",
      "0 TCATATGAAC motifs detected in file\n",
      "\n",
      "Seq is: TCCTCCCACT\n",
      "0 TCCTCCCACT motifs detected in file\n",
      "\n",
      "Seq is: GCCCGCCACA\n",
      "0 GCCCGCCACA motifs detected in file\n",
      "\n",
      "Seq is: CATGCGCGAA\n",
      "0 CATGCGCGAA motifs detected in file\n",
      "\n",
      "Seq is: GTCCCGGGAC\n",
      "0 GTCCCGGGAC motifs detected in file\n",
      "\n",
      "Seq is: AAGTTGATTC\n",
      "0 AAGTTGATTC motifs detected in file\n",
      "\n",
      "Seq is: GTGAGATGAG\n",
      "0 GTGAGATGAG motifs detected in file\n",
      "\n",
      "Seq is: AATCAAATGA\n",
      "0 AATCAAATGA motifs detected in file\n",
      "\n",
      "Seq is: CGATTTAGGT\n",
      "0 CGATTTAGGT motifs detected in file\n",
      "\n",
      "Seq is: GGAAGCTGGG\n",
      "0 GGAAGCTGGG motifs detected in file\n",
      "\n",
      "Seq is: AATTCAAAGT\n",
      "0 AATTCAAAGT motifs detected in file\n",
      "\n",
      "Seq is: ATTCCCGATA\n",
      "0 ATTCCCGATA motifs detected in file\n",
      "\n",
      "Seq is: GCATCAAAAG\n",
      "0 GCATCAAAAG motifs detected in file\n",
      "\n",
      "Seq is: ACCCCATCTT\n",
      "0 ACCCCATCTT motifs detected in file\n",
      "\n",
      "Seq is: TTACATCGGG\n",
      "0 TTACATCGGG motifs detected in file\n",
      "\n",
      "Seq is: TAAAGAACGG\n",
      "0 TAAAGAACGG motifs detected in file\n",
      "\n",
      "Seq is: TGGAGTACCC\n",
      "0 TGGAGTACCC motifs detected in file\n",
      "\n",
      "Seq is: GATGTGGAAC\n",
      "0 GATGTGGAAC motifs detected in file\n",
      "\n",
      "Seq is: TGGATTGCAG\n",
      "0 TGGATTGCAG motifs detected in file\n",
      "\n",
      "Seq is: AAATGCACCC\n",
      "0 AAATGCACCC motifs detected in file\n",
      "\n",
      "Seq is: ATGAACGCGA\n",
      "0 ATGAACGCGA motifs detected in file\n",
      "\n",
      "Complement found from 7249 to 7259\n",
      "Seq is: ATGAAACATC\n",
      "1 ATGAAACATC motifs detected in file\n",
      "\n",
      "Seq is: GGAAGTAGTC\n",
      "0 GGAAGTAGTC motifs detected in file\n",
      "\n",
      "Seq is: CCAAACGACT\n",
      "0 CCAAACGACT motifs detected in file\n",
      "\n",
      "Seq is: TCTACCGCCA\n",
      "0 TCTACCGCCA motifs detected in file\n",
      "\n",
      "Seq is: TATATCCGAC\n",
      "0 TATATCCGAC motifs detected in file\n",
      "\n",
      "Seq is: TCGTTCTCCA\n",
      "0 TCGTTCTCCA motifs detected in file\n",
      "\n",
      "Seq is: CCACTCCAAC\n",
      "0 CCACTCCAAC motifs detected in file\n",
      "\n",
      "Complement found from 1678 to 1688\n",
      "Seq is: TAGCCCCAAA\n",
      "1 TAGCCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: TAGCTAATGT\n",
      "0 TAGCTAATGT motifs detected in file\n",
      "\n",
      "Seq is: CTTGTCCGAG\n",
      "0 CTTGTCCGAG motifs detected in file\n",
      "\n",
      "Seq is: GAGATCTCCC\n",
      "0 GAGATCTCCC motifs detected in file\n",
      "\n",
      "Seq is: TATACTTCGT\n",
      "0 TATACTTCGT motifs detected in file\n",
      "\n",
      "Seq is: GGCGGTCCAG\n",
      "0 GGCGGTCCAG motifs detected in file\n",
      "\n",
      "Seq is: GAAACTCAAA\n",
      "0 GAAACTCAAA motifs detected in file\n",
      "\n",
      "Seq is: ATTCGACACC\n",
      "0 ATTCGACACC motifs detected in file\n",
      "\n",
      "Seq is: GAGCATCACA\n",
      "0 GAGCATCACA motifs detected in file\n",
      "\n",
      "Seq is: AGACCGCTCG\n",
      "0 AGACCGCTCG motifs detected in file\n",
      "\n",
      "Seq is: TCAAAACAAC\n",
      "0 TCAAAACAAC motifs detected in file\n",
      "\n",
      "Seq is: TAAATTGACA\n",
      "0 TAAATTGACA motifs detected in file\n",
      "\n",
      "Seq is: ACTCCAAATC\n",
      "0 ACTCCAAATC motifs detected in file\n",
      "\n",
      "Seq is: CCGATTCGTA\n",
      "0 CCGATTCGTA motifs detected in file\n",
      "\n",
      "Complement found from 633 to 643\n",
      "Seq is: TCACCCCATA\n",
      "1 TCACCCCATA motifs detected in file\n",
      "\n",
      "Seq is: GATTAGGGTC\n",
      "0 GATTAGGGTC motifs detected in file\n",
      "\n",
      "Seq is: AAACCCAGAA\n",
      "0 AAACCCAGAA motifs detected in file\n",
      "\n",
      "Seq is: TCGATAACAC\n",
      "0 TCGATAACAC motifs detected in file\n",
      "\n",
      "Seq is: ACAAGTCTAT\n",
      "0 ACAAGTCTAT motifs detected in file\n",
      "\n",
      "Seq is: ACAATTTCGG\n",
      "0 ACAATTTCGG motifs detected in file\n",
      "\n",
      "Seq is: TGAAAGCATC\n",
      "0 TGAAAGCATC motifs detected in file\n",
      "\n",
      "Seq is: AGATAAAACA\n",
      "0 AGATAAAACA motifs detected in file\n",
      "\n",
      "Seq is: CAGTTGACCT\n",
      "0 CAGTTGACCT motifs detected in file\n",
      "\n",
      "Seq is: CAAAAAATGT\n",
      "0 CAAAAAATGT motifs detected in file\n",
      "\n",
      "Seq is: TGAGTCCACT\n",
      "0 TGAGTCCACT motifs detected in file\n",
      "\n",
      "Seq is: CAAAATCGAA\n",
      "0 CAAAATCGAA motifs detected in file\n",
      "\n",
      "Seq is: ATAACCCCTC\n",
      "0 ATAACCCCTC motifs detected in file\n",
      "\n",
      "Seq is: CCCCACTAGA\n",
      "0 CCCCACTAGA motifs detected in file\n",
      "\n",
      "Seq is: TTTTGTGAGA\n",
      "0 TTTTGTGAGA motifs detected in file\n",
      "\n",
      "Seq is: AATGCGGCCG\n",
      "0 AATGCGGCCG motifs detected in file\n",
      "\n",
      "Seq is: AAGATAACTG\n",
      "0 AAGATAACTG motifs detected in file\n",
      "\n",
      "Seq is: AACCCAATTA\n",
      "0 AACCCAATTA motifs detected in file\n",
      "\n",
      "Seq is: GCACACTGGC\n",
      "0 GCACACTGGC motifs detected in file\n",
      "\n",
      "Seq is: GCCACCGACC\n",
      "0 GCCACCGACC motifs detected in file\n",
      "\n",
      "Seq is: GTGCTTTAAC\n",
      "0 GTGCTTTAAC motifs detected in file\n",
      "\n",
      "Seq is: TGGTTGGGAC\n",
      "0 TGGTTGGGAC motifs detected in file\n",
      "\n",
      "Seq is: CCCAATCATA\n",
      "0 CCCAATCATA motifs detected in file\n",
      "\n",
      "Seq is: TCGAATCAAT\n",
      "0 TCGAATCAAT motifs detected in file\n",
      "\n",
      "Seq is: TTGAAAGCAA\n",
      "0 TTGAAAGCAA motifs detected in file\n",
      "\n",
      "Seq is: ATAACGATTT\n",
      "0 ATAACGATTT motifs detected in file\n",
      "\n",
      "Seq is: CCAATTAGTG\n",
      "0 CCAATTAGTG motifs detected in file\n",
      "\n",
      "Seq is: ACGACAAAGG\n",
      "0 ACGACAAAGG motifs detected in file\n",
      "\n",
      "Seq is: GCACCCCCAC\n",
      "0 GCACCCCCAC motifs detected in file\n",
      "\n",
      "Seq is: ACCGATCCGA\n",
      "0 ACCGATCCGA motifs detected in file\n",
      "\n",
      "Seq is: TTCGCAAAAC\n",
      "0 TTCGCAAAAC motifs detected in file\n",
      "\n",
      "Seq is: TCGACGTAAC\n",
      "0 TCGACGTAAC motifs detected in file\n",
      "\n",
      "Seq is: GACGCACGAA\n",
      "0 GACGCACGAA motifs detected in file\n",
      "\n",
      "Seq is: CTACGAACAA\n",
      "0 CTACGAACAA motifs detected in file\n",
      "\n",
      "Seq is: GGAAAACTAG\n",
      "0 GGAAAACTAG motifs detected in file\n",
      "\n",
      "Seq is: CACCACTAAA\n",
      "0 CACCACTAAA motifs detected in file\n",
      "\n",
      "Seq is: TCTCCCACTT\n",
      "0 TCTCCCACTT motifs detected in file\n",
      "\n",
      "Seq is: GAGTGACCTT\n",
      "0 GAGTGACCTT motifs detected in file\n",
      "\n",
      "Seq is: GCCCCTACGA\n",
      "0 GCCCCTACGA motifs detected in file\n",
      "\n",
      "Seq is: ACGTACACAT\n",
      "0 ACGTACACAT motifs detected in file\n",
      "\n",
      "Seq is: TAGAATGATT\n",
      "0 TAGAATGATT motifs detected in file\n",
      "\n",
      "Seq is: CTCGATTATC\n",
      "0 CTCGATTATC motifs detected in file\n",
      "\n",
      "Seq is: TTTCCGATCC\n",
      "0 TTTCCGATCC motifs detected in file\n",
      "\n",
      "Seq is: TGGTTTGGAT\n",
      "0 TGGTTTGGAT motifs detected in file\n",
      "\n",
      "Seq is: AAACAAATAC\n",
      "0 AAACAAATAC motifs detected in file\n",
      "\n",
      "Seq is: CCCACTACAC\n",
      "0 CCCACTACAC motifs detected in file\n",
      "\n",
      "Seq is: TCGGGCAGAT\n",
      "0 TCGGGCAGAT motifs detected in file\n",
      "\n",
      "Seq is: TTGTAAAAGT\n",
      "0 TTGTAAAAGT motifs detected in file\n",
      "\n",
      "Seq is: CACATAACGA\n",
      "0 CACATAACGA motifs detected in file\n",
      "\n",
      "Seq is: AACTCCTCCA\n",
      "0 AACTCCTCCA motifs detected in file\n",
      "\n",
      "Seq is: TTCGATGTAT\n",
      "0 TTCGATGTAT motifs detected in file\n",
      "\n",
      "Seq is: TTGACACCCC\n",
      "0 TTGACACCCC motifs detected in file\n",
      "\n",
      "Seq is: CCACAGAAAC\n",
      "0 CCACAGAAAC motifs detected in file\n",
      "\n",
      "Seq is: CCCAAACCAA\n",
      "0 CCCAAACCAA motifs detected in file\n",
      "\n",
      "Seq is: CCAAGCCCCA\n",
      "0 CCAAGCCCCA motifs detected in file\n",
      "\n",
      "Seq is: TACCCCAATC\n",
      "0 TACCCCAATC motifs detected in file\n",
      "\n",
      "Seq is: GTCGCCACAC\n",
      "0 GTCGCCACAC motifs detected in file\n",
      "\n",
      "Seq is: ACACACGACC\n",
      "0 ACACACGACC motifs detected in file\n",
      "\n",
      "Complement found from 501 to 511\n",
      "Seq is: CATCCTACCC\n",
      "1 CATCCTACCC motifs detected in file\n",
      "\n",
      "Complement found from 9553 to 9563\n",
      "Seq is: GCCCCCAACA\n",
      "1 GCCCCCAACA motifs detected in file\n",
      "\n",
      "Seq is: TAACTACTCT\n",
      "0 TAACTACTCT motifs detected in file\n",
      "\n",
      "Seq is: AATCATCATA\n",
      "0 AATCATCATA motifs detected in file\n",
      "\n",
      "Seq is: CCCTCACCCT\n",
      "0 CCCTCACCCT motifs detected in file\n",
      "\n",
      "Complement found from 5487 to 5497\n",
      "Seq is: CCCCTTTTAT\n",
      "1 CCCCTTTTAT motifs detected in file\n",
      "\n",
      "Complement found from 15415 to 15425\n",
      "Seq is: TACACAATCA\n",
      "1 TACACAATCA motifs detected in file\n",
      "\n",
      "Complement found from 11865 to 11875\n",
      "Seq is: ACCCCCCACT\n",
      "1 ACCCCCCACT motifs detected in file\n",
      "\n",
      "Seq is: GACAATTTTC\n",
      "0 GACAATTTTC motifs detected in file\n",
      "\n",
      "Seq is: ACGTATGGCG\n",
      "0 ACGTATGGCG motifs detected in file\n",
      "\n",
      "Seq is: GTTTTCTATT\n",
      "0 GTTTTCTATT motifs detected in file\n",
      "\n",
      "Complement found from 14086 to 14096\n",
      "Seq is: TTAAACTTTA\n",
      "1 TTAAACTTTA motifs detected in file\n",
      "\n",
      "Seq is: GACCAATCCG\n",
      "0 GACCAATCCG motifs detected in file\n",
      "\n",
      "Seq is: ACCACAATCC\n",
      "0 ACCACAATCC motifs detected in file\n",
      "\n",
      "Seq is: CAAGAAACAA\n",
      "0 CAAGAAACAA motifs detected in file\n",
      "\n",
      "Complement found from 349 to 359\n",
      "Complement found from 553 to 563\n",
      "Seq is: AAACCCCAAA\n",
      "2 AAACCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: CCGTCTCTAC\n",
      "0 CCGTCTCTAC motifs detected in file\n",
      "\n",
      "Seq is: ACAAATTCAC\n",
      "0 ACAAATTCAC motifs detected in file\n",
      "\n",
      "Seq is: GACACCGGTC\n",
      "0 GACACCGGTC motifs detected in file\n",
      "\n",
      "Seq is: TTCGCCCCCT\n",
      "0 TTCGCCCCCT motifs detected in file\n",
      "\n",
      "Seq is: CCCCCCCAAA\n",
      "0 CCCCCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: CCACCTTTAA\n",
      "0 CCACCTTTAA motifs detected in file\n",
      "\n",
      "Seq is: AAAACAATAC\n",
      "0 AAAACAATAC motifs detected in file\n",
      "\n",
      "Seq is: TACAGACACA\n",
      "0 TACAGACACA motifs detected in file\n",
      "\n",
      "Seq is: CCTTTCACCG\n",
      "0 CCTTTCACCG motifs detected in file\n",
      "\n",
      "Seq is: ACACGTCTGT\n",
      "0 ACACGTCTGT motifs detected in file\n",
      "\n",
      "Seq is: AAGTTAACAA\n",
      "0 AAGTTAACAA motifs detected in file\n",
      "\n",
      "Seq is: TAATAATACA\n",
      "0 TAATAATACA motifs detected in file\n",
      "\n",
      "Seq is: GGATGTTCGT\n",
      "0 GGATGTTCGT motifs detected in file\n",
      "\n",
      "Complement found from 209 to 219\n",
      "Complement found from 14780 to 14790\n",
      "Seq is: AATTAATTAA\n",
      "2 AATTAATTAA motifs detected in file\n",
      "\n",
      "Seq is: TTGTGTGAAA\n",
      "0 TTGTGTGAAA motifs detected in file\n",
      "\n",
      "Seq is: TCATTCATAC\n",
      "0 TCATTCATAC motifs detected in file\n",
      "\n",
      "Seq is: AAGCGGACAT\n",
      "0 AAGCGGACAT motifs detected in file\n",
      "\n",
      "Seq is: TATAACTTGC\n",
      "0 TATAACTTGC motifs detected in file\n",
      "\n",
      "Seq is: ATCCACGCTA\n",
      "0 ATCCACGCTA motifs detected in file\n",
      "\n",
      "Seq is: TTTATTATCC\n",
      "0 TTTATTATCC motifs detected in file\n",
      "\n",
      "Seq is: TACTCCGTCC\n",
      "0 TACTCCGTCC motifs detected in file\n",
      "\n",
      "Seq is: TTAGTTTCTG\n",
      "0 TTAGTTTCTG motifs detected in file\n",
      "\n",
      "Seq is: TCTATGACGC\n",
      "0 TCTATGACGC motifs detected in file\n",
      "\n",
      "Seq is: TGTATCCCAC\n",
      "0 TGTATCCCAC motifs detected in file\n",
      "\n",
      "Seq is: GAGGCCGAGG\n",
      "0 GAGGCCGAGG motifs detected in file\n",
      "\n",
      "Seq is: TCGCAGAGCG\n",
      "0 TCGCAGAGCG motifs detected in file\n",
      "\n",
      "Seq is: TTACGATAGC\n",
      "0 TTACGATAGC motifs detected in file\n",
      "\n",
      "Seq is: GCACGTATGG\n",
      "0 GCACGTATGG motifs detected in file\n",
      "\n",
      "Seq is: GGGGTCTGCT\n",
      "0 GGGGTCTGCT motifs detected in file\n",
      "\n",
      "Seq is: TTTATGGTTT\n",
      "0 TTTATGGTTT motifs detected in file\n",
      "\n",
      "Seq is: ACGTACCTCT\n",
      "0 ACGTACCTCT motifs detected in file\n",
      "\n",
      "Seq is: CGAGGGCACT\n",
      "0 CGAGGGCACT motifs detected in file\n",
      "\n",
      "Seq is: CACCAATTAT\n",
      "0 CACCAATTAT motifs detected in file\n",
      "\n",
      "Seq is: CCCACTATCT\n",
      "0 CCCACTATCT motifs detected in file\n",
      "\n",
      "Seq is: GTAGCACTACA\n",
      "0 GTAGCACTACA motifs detected in file\n",
      "\n",
      "Seq is: GAATAAATTCC\n",
      "0 GAATAAATTCC motifs detected in file\n",
      "\n",
      "Seq is: CCTTGCACACC\n",
      "0 CCTTGCACACC motifs detected in file\n",
      "\n",
      "Seq is: CGATAAATCCG\n",
      "0 CGATAAATCCG motifs detected in file\n",
      "\n",
      "Seq is: AAATACTGGGA\n",
      "0 AAATACTGGGA motifs detected in file\n",
      "\n",
      "Seq is: CTTCATCCTTG\n",
      "0 CTTCATCCTTG motifs detected in file\n",
      "\n",
      "Seq is: GTCTACAGCCT\n",
      "0 GTCTACAGCCT motifs detected in file\n",
      "\n",
      "Seq is: ATGTCAAGTGA\n",
      "0 ATGTCAAGTGA motifs detected in file\n",
      "\n",
      "Seq is: AATCGATGGGG\n",
      "0 AATCGATGGGG motifs detected in file\n",
      "\n",
      "Seq is: GTTCACAATAC\n",
      "0 GTTCACAATAC motifs detected in file\n",
      "\n",
      "Seq is: CCGGGCCTCGC\n",
      "0 CCGGGCCTCGC motifs detected in file\n",
      "\n",
      "Seq is: TCCTCTCATCG\n",
      "0 TCCTCTCATCG motifs detected in file\n",
      "\n",
      "Seq is: TGAGAACACGC\n",
      "0 TGAGAACACGC motifs detected in file\n",
      "\n",
      "Seq is: CCTATAACTAA\n",
      "0 CCTATAACTAA motifs detected in file\n",
      "\n",
      "Seq is: AGTGCCTCCTA\n",
      "0 AGTGCCTCCTA motifs detected in file\n",
      "\n",
      "Seq is: CCACCAGTTCC\n",
      "0 CCACCAGTTCC motifs detected in file\n",
      "\n",
      "Seq is: CTGGGGATAGA\n",
      "0 CTGGGGATAGA motifs detected in file\n",
      "\n",
      "Seq is: CTCCCCCCAGT\n",
      "0 CTCCCCCCAGT motifs detected in file\n",
      "\n",
      "Seq is: AGGTACCCCTG\n",
      "0 AGGTACCCCTG motifs detected in file\n",
      "\n",
      "Seq is: CTCTTCCCTAA\n",
      "0 CTCTTCCCTAA motifs detected in file\n",
      "\n",
      "Seq is: ACTGACATTAC\n",
      "0 ACTGACATTAC motifs detected in file\n",
      "\n",
      "Seq is: ACGATACATGC\n",
      "0 ACGATACATGC motifs detected in file\n",
      "\n",
      "Seq is: CATTTACCGAA\n",
      "0 CATTTACCGAA motifs detected in file\n",
      "\n",
      "Seq is: ATACATGATAC\n",
      "0 ATACATGATAC motifs detected in file\n",
      "\n",
      "Seq is: ATGACAATTCC\n",
      "0 ATGACAATTCC motifs detected in file\n",
      "\n",
      "Seq is: CACCCATCCAA\n",
      "0 CACCCATCCAA motifs detected in file\n",
      "\n",
      "Seq is: ACAACCATAGG\n",
      "0 ACAACCATAGG motifs detected in file\n",
      "\n",
      "Seq is: ATCACCCACTC\n",
      "0 ATCACCCACTC motifs detected in file\n",
      "\n",
      "Seq is: CCCACCGAAAC\n",
      "0 CCCACCGAAAC motifs detected in file\n",
      "\n",
      "Seq is: CTCAACGTCAA\n",
      "0 CTCAACGTCAA motifs detected in file\n",
      "\n",
      "Seq is: CTACACACTAT\n",
      "0 CTACACACTAT motifs detected in file\n",
      "\n",
      "Seq is: CAACTCCCAAC\n",
      "0 CAACTCCCAAC motifs detected in file\n",
      "\n",
      "Seq is: TAACGACATGA\n",
      "0 TAACGACATGA motifs detected in file\n",
      "\n",
      "Seq is: ACGAACATTCG\n",
      "0 ACGAACATTCG motifs detected in file\n",
      "\n",
      "Seq is: TACCCCTCCCC\n",
      "0 TACCCCTCCCC motifs detected in file\n",
      "\n",
      "Seq is: CAAAACTACAC\n",
      "0 CAAAACTACAC motifs detected in file\n",
      "\n",
      "Seq is: CTAACCCAAAA\n",
      "0 CTAACCCAAAA motifs detected in file\n",
      "\n",
      "Seq is: ATACATGATGT\n",
      "0 ATACATGATGT motifs detected in file\n",
      "\n",
      "Seq is: CCACCAGTTCA\n",
      "0 CCACCAGTTCA motifs detected in file\n",
      "\n",
      "Seq is: TAAATACCATG\n",
      "0 TAAATACCATG motifs detected in file\n",
      "\n",
      "Seq is: GCATGTTATAA\n",
      "0 GCATGTTATAA motifs detected in file\n",
      "\n",
      "Seq is: GTACCACCGAC\n",
      "0 GTACCACCGAC motifs detected in file\n",
      "\n",
      "Seq is: CGTCATTACAT\n",
      "0 CGTCATTACAT motifs detected in file\n",
      "\n",
      "Seq is: GCTTTATGTAT\n",
      "0 GCTTTATGTAT motifs detected in file\n",
      "\n",
      "Seq is: CGCCAACAACT\n",
      "0 CGCCAACAACT motifs detected in file\n",
      "\n",
      "Seq is: ACCCACTCAGT\n",
      "0 ACCCACTCAGT motifs detected in file\n",
      "\n",
      "Seq is: TATGAACCCAC\n",
      "0 TATGAACCCAC motifs detected in file\n",
      "\n",
      "Seq is: CATGGGTTTAG\n",
      "0 CATGGGTTTAG motifs detected in file\n",
      "\n",
      "Seq is: ACGAAGGGGTA\n",
      "0 ACGAAGGGGTA motifs detected in file\n",
      "\n",
      "Seq is: CTTTCTTGTCT\n",
      "0 CTTTCTTGTCT motifs detected in file\n",
      "\n",
      "Seq is: CTTATCAAATT\n",
      "0 CTTATCAAATT motifs detected in file\n",
      "\n",
      "Seq is: TAATCTTAGAA\n",
      "0 TAATCTTAGAA motifs detected in file\n",
      "\n",
      "Seq is: TCGAAACCCAC\n",
      "0 TCGAAACCCAC motifs detected in file\n",
      "\n",
      "Seq is: GATTACCACCT\n",
      "0 GATTACCACCT motifs detected in file\n",
      "\n",
      "Seq is: CAATTTCTGAA\n",
      "0 CAATTTCTGAA motifs detected in file\n",
      "\n",
      "Seq is: AAAGAGACTAA\n",
      "0 AAAGAGACTAA motifs detected in file\n",
      "\n",
      "Seq is: ACAGGAACCTT\n",
      "0 ACAGGAACCTT motifs detected in file\n",
      "\n",
      "Seq is: TTTCCAAAAGT\n",
      "0 TTTCCAAAAGT motifs detected in file\n",
      "\n",
      "Seq is: AGAGGCCAAAT\n",
      "0 AGAGGCCAAAT motifs detected in file\n",
      "\n",
      "Seq is: GTTCTGACCAC\n",
      "0 GTTCTGACCAC motifs detected in file\n",
      "\n",
      "Seq is: ATAATCAAATA\n",
      "0 ATAATCAAATA motifs detected in file\n",
      "\n",
      "Seq is: TGATGTTCCTG\n",
      "0 TGATGTTCCTG motifs detected in file\n",
      "\n",
      "Seq is: TCCGGGTAAAC\n",
      "0 TCCGGGTAAAC motifs detected in file\n",
      "\n",
      "Seq is: TCATAAAACAA\n",
      "0 TCATAAAACAA motifs detected in file\n",
      "\n",
      "Seq is: AAGTTAATCCC\n",
      "0 AAGTTAATCCC motifs detected in file\n",
      "\n",
      "Seq is: TCTATCAACCA\n",
      "0 TCTATCAACCA motifs detected in file\n",
      "\n",
      "Seq is: TAATCCTAATC\n",
      "0 TAATCCTAATC motifs detected in file\n",
      "\n",
      "Seq is: CTAACAACACT\n",
      "0 CTAACAACACT motifs detected in file\n",
      "\n",
      "Seq is: TCATATCATGC\n",
      "0 TCATATCATGC motifs detected in file\n",
      "\n",
      "Seq is: CTACGATGAAC\n",
      "0 CTACGATGAAC motifs detected in file\n",
      "\n",
      "Seq is: AGGTTACTACC\n",
      "0 AGGTTACTACC motifs detected in file\n",
      "\n",
      "Seq is: ATTTTCCCATC\n",
      "0 ATTTTCCCATC motifs detected in file\n",
      "\n",
      "Seq is: GAATGACCAAC\n",
      "0 GAATGACCAAC motifs detected in file\n",
      "\n",
      "Seq is: AGGAGGCTAAG\n",
      "0 AGGAGGCTAAG motifs detected in file\n",
      "\n",
      "Seq is: TCCAATCTTAC\n",
      "0 TCCAATCTTAC motifs detected in file\n",
      "\n",
      "Seq is: TCCTCCAGACG\n",
      "0 TCCTCCAGACG motifs detected in file\n",
      "\n",
      "Seq is: CCGATCCTCAG\n",
      "0 CCGATCCTCAG motifs detected in file\n",
      "\n",
      "Seq is: TTATTTCACTA\n",
      "0 TTATTTCACTA motifs detected in file\n",
      "\n",
      "Seq is: ACCGAATCACC\n",
      "0 ACCGAATCACC motifs detected in file\n",
      "\n",
      "Seq is: CGCTTTATAAT\n",
      "0 CGCTTTATAAT motifs detected in file\n",
      "\n",
      "Seq is: ACGAAACAACA\n",
      "0 ACGAAACAACA motifs detected in file\n",
      "\n",
      "Seq is: AACCTATATAC\n",
      "0 AACCTATATAC motifs detected in file\n",
      "\n",
      "Complement found from 10972 to 10983\n",
      "Seq is: CTCCTACCCCT\n",
      "1 CTCCTACCCCT motifs detected in file\n",
      "\n",
      "Seq is: AATAACGATCC\n",
      "0 AATAACGATCC motifs detected in file\n",
      "\n",
      "Complement found from 5472 to 5483\n",
      "Seq is: TACTCCTACCT\n",
      "1 TACTCCTACCT motifs detected in file\n",
      "\n",
      "Seq is: ATCATTATCCC\n",
      "0 ATCATTATCCC motifs detected in file\n",
      "\n",
      "Seq is: GTTCCTGCGGA\n",
      "0 GTTCCTGCGGA motifs detected in file\n",
      "\n",
      "Complement found from 15355 to 15366\n",
      "Seq is: GGATCAAACAA\n",
      "1 GGATCAAACAA motifs detected in file\n",
      "\n",
      "Seq is: TCCCTGCCTAG\n",
      "0 TCCCTGCCTAG motifs detected in file\n",
      "\n",
      "Seq is: CCTCTTAACAC\n",
      "0 CCTCTTAACAC motifs detected in file\n",
      "\n",
      "Seq is: ATCCGCTTATC\n",
      "0 ATCCGCTTATC motifs detected in file\n",
      "\n",
      "Seq is: CTTTATAGTAA\n",
      "0 CTTTATAGTAA motifs detected in file\n",
      "\n",
      "Seq is: GCCCGAACTAC\n",
      "0 GCCCGAACTAC motifs detected in file\n",
      "\n",
      "Complement found from 15535 to 15546\n",
      "Seq is: ACCCCTCCCCA\n",
      "1 ACCCCTCCCCA motifs detected in file\n",
      "\n",
      "Seq is: CAAATTCCCCA\n",
      "0 CAAATTCCCCA motifs detected in file\n",
      "\n",
      "Seq is: ACCGATCCCAT\n",
      "0 ACCGATCCCAT motifs detected in file\n",
      "\n",
      "Seq is: ATTAACAGACC\n",
      "0 ATTAACAGACC motifs detected in file\n",
      "\n",
      "Seq is: CAGCGGATCCT\n",
      "0 CAGCGGATCCT motifs detected in file\n",
      "\n",
      "Seq is: CCAGACCACTC\n",
      "0 CCAGACCACTC motifs detected in file\n",
      "\n",
      "Seq is: TTATCACAATT\n",
      "0 TTATCACAATT motifs detected in file\n",
      "\n",
      "Seq is: ACAGTAATTCC\n",
      "0 ACAGTAATTCC motifs detected in file\n",
      "\n",
      "Complement found from 15444 to 15455\n",
      "Seq is: TCTCTTCCTTC\n",
      "1 TCTCTTCCTTC motifs detected in file\n",
      "\n",
      "Seq is: TCTTCATTCGG\n",
      "0 TCTTCATTCGG motifs detected in file\n",
      "\n",
      "Seq is: CTCCCGCAGAA\n",
      "0 CTCCCGCAGAA motifs detected in file\n",
      "\n",
      "Seq is: ACTAACACATC\n",
      "0 ACTAACACATC motifs detected in file\n",
      "\n",
      "Seq is: ATTCCCACCTT\n",
      "0 ATTCCCACCTT motifs detected in file\n",
      "\n",
      "Seq is: CCACTAAAATA\n",
      "0 CCACTAAAATA motifs detected in file\n",
      "\n",
      "Seq is: GCCTTACCCTC\n",
      "0 GCCTTACCCTC motifs detected in file\n",
      "\n",
      "Seq is: CACTAAGGATC\n",
      "0 CACTAAGGATC motifs detected in file\n",
      "\n",
      "Seq is: CCCCAACAAAC\n",
      "0 CCCCAACAAAC motifs detected in file\n",
      "\n",
      "Seq is: TAGGGCAAAGC\n",
      "0 TAGGGCAAAGC motifs detected in file\n",
      "\n",
      "Seq is: ACGTTCTTATC\n",
      "0 ACGTTCTTATC motifs detected in file\n",
      "\n",
      "Seq is: CTCCACCTCAC\n",
      "0 CTCCACCTCAC motifs detected in file\n",
      "\n",
      "Seq is: AACGATCCCGA\n",
      "0 AACGATCCCGA motifs detected in file\n",
      "\n",
      "Seq is: CGTTATTACTT\n",
      "0 CGTTATTACTT motifs detected in file\n",
      "\n",
      "Seq is: CCCGTTCTACT\n",
      "0 CCCGTTCTACT motifs detected in file\n",
      "\n",
      "Seq is: TCACTTTCCAT\n",
      "0 TCACTTTCCAT motifs detected in file\n",
      "\n",
      "Seq is: TTCTTAGCACA\n",
      "0 TTCTTAGCACA motifs detected in file\n",
      "\n",
      "Complement found from 6310 to 6321\n",
      "Seq is: CTCCCACCCTG\n",
      "1 CTCCCACCCTG motifs detected in file\n",
      "\n",
      "Seq is: ACAGATGACTC\n",
      "0 ACAGATGACTC motifs detected in file\n",
      "\n",
      "Seq is: ATCGGAGGAGT\n",
      "0 ATCGGAGGAGT motifs detected in file\n",
      "\n",
      "Seq is: CTAAGTAACTT\n",
      "0 CTAAGTAACTT motifs detected in file\n",
      "\n",
      "Seq is: GATCCAGACAG\n",
      "0 GATCCAGACAG motifs detected in file\n",
      "\n",
      "Seq is: GGTTACATACC\n",
      "0 GGTTACATACC motifs detected in file\n",
      "\n",
      "Seq is: CTACCGCCTAT\n",
      "0 CTACCGCCTAT motifs detected in file\n",
      "\n",
      "Seq is: CATTCAAACAT\n",
      "0 CATTCAAACAT motifs detected in file\n",
      "\n",
      "Seq is: TAATGACACCG\n",
      "0 TAATGACACCG motifs detected in file\n",
      "\n",
      "Seq is: GGGAGTCTTAC\n",
      "0 GGGAGTCTTAC motifs detected in file\n",
      "\n",
      "Seq is: TATAAACCGGA\n",
      "0 TATAAACCGGA motifs detected in file\n",
      "\n",
      "Seq is: GTGCCCTCCTG\n",
      "0 GTGCCCTCCTG motifs detected in file\n",
      "\n",
      "Seq is: TATCGGATACT\n",
      "0 TATCGGATACT motifs detected in file\n",
      "\n",
      "Seq is: TCCGACAACGA\n",
      "0 TCCGACAACGA motifs detected in file\n",
      "\n",
      "Seq is: TATCAACGTTC\n",
      "0 TATCAACGTTC motifs detected in file\n",
      "\n",
      "Seq is: GTCCTCCTATT\n",
      "0 GTCCTCCTATT motifs detected in file\n",
      "\n",
      "Seq is: ACGGCTACAAA\n",
      "0 ACGGCTACAAA motifs detected in file\n",
      "\n",
      "Seq is: GTCCAAAGACT\n",
      "0 GTCCAAAGACT motifs detected in file\n",
      "\n",
      "Seq is: CATCTCTTTAC\n",
      "0 CATCTCTTTAC motifs detected in file\n",
      "\n",
      "Seq is: TAGGCATTATA\n",
      "0 TAGGCATTATA motifs detected in file\n",
      "\n",
      "Seq is: TCCGGAGCGGG\n",
      "0 TCCGGAGCGGG motifs detected in file\n",
      "\n",
      "Seq is: CTACACATCCT\n",
      "0 CTACACATCCT motifs detected in file\n",
      "\n",
      "Seq is: TCTCCGTCTAT\n",
      "0 TCTCCGTCTAT motifs detected in file\n",
      "\n",
      "Seq is: TTCTTATAACT\n",
      "0 TTCTTATAACT motifs detected in file\n",
      "\n",
      "Seq is: CCGCGGTAACC\n",
      "0 CCGCGGTAACC motifs detected in file\n",
      "\n",
      "Seq is: GCACTTCCATC\n",
      "0 GCACTTCCATC motifs detected in file\n",
      "\n",
      "Seq is: GCCTACTAAGT\n",
      "0 GCCTACTAAGT motifs detected in file\n",
      "\n",
      "Seq is: CGGTATTAAAT\n",
      "0 CGGTATTAAAT motifs detected in file\n",
      "\n",
      "Seq is: GCAGAGCTCAC\n",
      "0 GCAGAGCTCAC motifs detected in file\n",
      "\n",
      "Seq is: TACACCCGCTA\n",
      "0 TACACCCGCTA motifs detected in file\n",
      "\n",
      "Seq is: ACTACTTTTCC\n",
      "0 ACTACTTTTCC motifs detected in file\n",
      "\n",
      "Seq is: GCCAACTCCGC\n",
      "0 GCCAACTCCGC motifs detected in file\n",
      "\n",
      "Seq is: AGACCACTCAT\n",
      "0 AGACCACTCAT motifs detected in file\n",
      "\n",
      "Seq is: CACGTACCGAT\n",
      "0 CACGTACCGAT motifs detected in file\n",
      "\n",
      "Seq is: CCTTATCAGGA\n",
      "0 CCTTATCAGGA motifs detected in file\n",
      "\n",
      "Seq is: CACCACTAAAC\n",
      "0 CACCACTAAAC motifs detected in file\n",
      "\n",
      "Seq is: CTCCTAGTCCG\n",
      "0 CTCCTAGTCCG motifs detected in file\n",
      "\n",
      "Seq is: TCCGCGGTTCC\n",
      "0 TCCGCGGTTCC motifs detected in file\n",
      "\n",
      "Seq is: TCACTCGGCTT\n",
      "0 TCACTCGGCTT motifs detected in file\n",
      "\n",
      "Seq is: CAAAGTAGTAC\n",
      "0 CAAAGTAGTAC motifs detected in file\n",
      "\n",
      "Seq is: GCCTCTACAAC\n",
      "0 GCCTCTACAAC motifs detected in file\n",
      "\n",
      "Seq is: CTACCCCACCC\n",
      "0 CTACCCCACCC motifs detected in file\n",
      "\n",
      "Seq is: CTCCAGCTACT\n",
      "0 CTCCAGCTACT motifs detected in file\n",
      "\n",
      "Seq is: TACTCACCAAT\n",
      "0 TACTCACCAAT motifs detected in file\n",
      "\n",
      "Seq is: TAATTAAAATA\n",
      "0 TAATTAAAATA motifs detected in file\n",
      "\n",
      "Seq is: ATCCCCCAATC\n",
      "0 ATCCCCCAATC motifs detected in file\n",
      "\n",
      "Seq is: AAAACGCATAA\n",
      "0 AAAACGCATAA motifs detected in file\n",
      "\n",
      "Seq is: CCCCAGTAACC\n",
      "0 CCCCAGTAACC motifs detected in file\n",
      "\n",
      "Seq is: ACAAGAACATC\n",
      "0 ACAAGAACATC motifs detected in file\n",
      "\n",
      "Seq is: AACTTTATGTT\n",
      "0 AACTTTATGTT motifs detected in file\n",
      "\n",
      "Seq is: GCTACCAAAAA\n",
      "0 GCTACCAAAAA motifs detected in file\n",
      "\n",
      "Seq is: GTATAGTAACC\n",
      "0 GTATAGTAACC motifs detected in file\n",
      "\n",
      "Seq is: AGCACCAACAT\n",
      "0 AGCACCAACAT motifs detected in file\n",
      "\n",
      "Seq is: CAGGCACGCTC\n",
      "0 CAGGCACGCTC motifs detected in file\n",
      "\n",
      "Seq is: TTATTACTACA\n",
      "0 TTATTACTACA motifs detected in file\n",
      "\n",
      "Seq is: TACGAAACAAA\n",
      "0 TACGAAACAAA motifs detected in file\n",
      "\n",
      "Seq is: GACAACTCACA\n",
      "0 GACAACTCACA motifs detected in file\n",
      "\n",
      "Seq is: CCCAAATCATT\n",
      "0 CCCAAATCATT motifs detected in file\n",
      "\n",
      "Seq is: ACCCCAAACAC\n",
      "0 ACCCCAAACAC motifs detected in file\n",
      "\n",
      "Seq is: CCCAAAAGAAG\n",
      "0 CCCAAAAGAAG motifs detected in file\n",
      "\n",
      "Seq is: ATTCGGAAGAG\n",
      "0 ATTCGGAAGAG motifs detected in file\n",
      "\n",
      "Seq is: GATAAATACCC\n",
      "0 GATAAATACCC motifs detected in file\n",
      "\n",
      "Seq is: CCAAATCATAA\n",
      "0 CCAAATCATAA motifs detected in file\n",
      "\n",
      "Seq is: CTAACAATCGC\n",
      "0 CTAACAATCGC motifs detected in file\n",
      "\n",
      "Seq is: CACACCAGCCC\n",
      "0 CACACCAGCCC motifs detected in file\n",
      "\n",
      "Seq is: ACACAATAATA\n",
      "0 ACACAATAATA motifs detected in file\n",
      "\n",
      "Seq is: AGACTTAAAAC\n",
      "0 AGACTTAAAAC motifs detected in file\n",
      "\n",
      "Seq is: CCCCTCCAATA\n",
      "0 CCCCTCCAATA motifs detected in file\n",
      "\n",
      "Seq is: TACCCAAATTA\n",
      "0 TACCCAAATTA motifs detected in file\n",
      "\n",
      "Seq is: TCAAAAAAATT\n",
      "0 TCAAAAAAATT motifs detected in file\n",
      "\n",
      "Seq is: AAATAAATCCC\n",
      "0 AAATAAATCCC motifs detected in file\n",
      "\n",
      "Complement found from 1697 to 1708\n",
      "Seq is: CCTTACTACCA\n",
      "1 CCTTACTACCA motifs detected in file\n",
      "\n",
      "Seq is: ACAGAAACCTA\n",
      "0 ACAGAAACCTA motifs detected in file\n",
      "\n",
      "Seq is: TATGATGTCGC\n",
      "0 TATGATGTCGC motifs detected in file\n",
      "\n",
      "Seq is: TACCGATAACT\n",
      "0 TACCGATAACT motifs detected in file\n",
      "\n",
      "Seq is: CCTCATAGGAC\n",
      "0 CCTCATAGGAC motifs detected in file\n",
      "\n",
      "Seq is: TCCGTACCCCC\n",
      "0 TCCGTACCCCC motifs detected in file\n",
      "\n",
      "Seq is: AGTCCCCAACT\n",
      "0 AGTCCCCAACT motifs detected in file\n",
      "\n",
      "Seq is: CCAGAACCACT\n",
      "0 CCAGAACCACT motifs detected in file\n",
      "\n",
      "Seq is: CACAAAATCAC\n",
      "0 CACAAAATCAC motifs detected in file\n",
      "\n",
      "Seq is: CCCAATCGCTA\n",
      "0 CCCAATCGCTA motifs detected in file\n",
      "\n",
      "Seq is: CCTCCATCCTA\n",
      "0 CCTCCATCCTA motifs detected in file\n",
      "\n",
      "Seq is: ACCACGACACC\n",
      "0 ACCACGACACC motifs detected in file\n",
      "\n",
      "Seq is: CACTTTCTCAT\n",
      "0 CACTTTCTCAT motifs detected in file\n",
      "\n",
      "Seq is: ACTACCCCACC\n",
      "0 ACTACCCCACC motifs detected in file\n",
      "\n",
      "Seq is: ACCAACACCAT\n",
      "0 ACCAACACCAT motifs detected in file\n",
      "\n",
      "Seq is: TTGAAATTATC\n",
      "0 TTGAAATTATC motifs detected in file\n",
      "\n",
      "Seq is: ACATCCTTCGA\n",
      "0 ACATCCTTCGA motifs detected in file\n",
      "\n",
      "Seq is: CTTATTAAATA\n",
      "0 CTTATTAAATA motifs detected in file\n",
      "\n",
      "Seq is: CTTCCTCTCCC\n",
      "0 CTTCCTCTCCC motifs detected in file\n",
      "\n",
      "Seq is: CAGTCCCAACT\n",
      "0 CAGTCCCAACT motifs detected in file\n",
      "\n",
      "Seq is: AAGCCCTCCTA\n",
      "0 AAGCCCTCCTA motifs detected in file\n",
      "\n",
      "Seq is: GGATAACCACG\n",
      "0 GGATAACCACG motifs detected in file\n",
      "\n",
      "Seq is: CCCCCGAAACA\n",
      "0 CCCCCGAAACA motifs detected in file\n",
      "\n",
      "Seq is: TACTAATACCC\n",
      "0 TACTAATACCC motifs detected in file\n",
      "\n",
      "Seq is: GCAACTAATCA\n",
      "0 GCAACTAATCA motifs detected in file\n",
      "\n",
      "Seq is: TCATCAATGAC\n",
      "0 TCATCAATGAC motifs detected in file\n",
      "\n",
      "Seq is: CAACTTGTAAC\n",
      "0 CAACTTGTAAC motifs detected in file\n",
      "\n",
      "Seq is: AAACAACCACA\n",
      "0 AAACAACCACA motifs detected in file\n",
      "\n",
      "Seq is: TATATAACATT\n",
      "0 TATATAACATT motifs detected in file\n",
      "\n",
      "Seq is: AACTCTAACGA\n",
      "0 AACTCTAACGA motifs detected in file\n",
      "\n",
      "Seq is: GCCCCCTTATC\n",
      "0 GCCCCCTTATC motifs detected in file\n",
      "\n",
      "Seq is: CAATACACTAA\n",
      "0 CAATACACTAA motifs detected in file\n",
      "\n",
      "Seq is: TCCTCATCCCA\n",
      "0 TCCTCATCCCA motifs detected in file\n",
      "\n",
      "Seq is: ATCCTACTCAC\n",
      "0 ATCCTACTCAC motifs detected in file\n",
      "\n",
      "Seq is: CCTTCTTCTTT\n",
      "0 CCTTCTTCTTT motifs detected in file\n",
      "\n",
      "Complement found from 14283 to 14294\n",
      "Seq is: CTCTCCTTCAT\n",
      "1 CTCTCCTTCAT motifs detected in file\n",
      "\n",
      "Seq is: TTCAAATTAAT\n",
      "0 TTCAAATTAAT motifs detected in file\n",
      "\n",
      "Seq is: ACGGAAAAACC\n",
      "0 ACGGAAAAACC motifs detected in file\n",
      "\n",
      "Seq is: CAACTCCACTA\n",
      "0 CAACTCCACTA motifs detected in file\n",
      "\n",
      "Seq is: CTACCTCCACC\n",
      "0 CTACCTCCACC motifs detected in file\n",
      "\n",
      "Seq is: TCTAAACCACG\n",
      "0 TCTAAACCACG motifs detected in file\n",
      "\n",
      "Seq is: ACACTTTAACA\n",
      "0 ACACTTTAACA motifs detected in file\n",
      "\n",
      "Seq is: AAATCCATTAT\n",
      "0 AAATCCATTAT motifs detected in file\n",
      "\n",
      "Seq is: CGAAAAGATCA\n",
      "0 CGAAAAGATCA motifs detected in file\n",
      "\n",
      "Seq is: GTCCAATCCAG\n",
      "0 GTCCAATCCAG motifs detected in file\n",
      "\n",
      "Seq is: ATCCTCCTCAT\n",
      "0 ATCCTCCTCAT motifs detected in file\n",
      "\n",
      "Seq is: CCCCGTCCAAA\n",
      "0 CCCCGTCCAAA motifs detected in file\n",
      "\n",
      "Seq is: ACCGAGCATTC\n",
      "0 ACCGAGCATTC motifs detected in file\n",
      "\n",
      "Seq is: TTCCGGATCTA\n",
      "0 TTCCGGATCTA motifs detected in file\n",
      "\n",
      "Seq is: TCCCCTAACAC\n",
      "0 TCCCCTAACAC motifs detected in file\n",
      "\n",
      "Seq is: GCCACACACTA\n",
      "0 GCCACACACTA motifs detected in file\n",
      "\n",
      "Seq is: CGATCCCATCT\n",
      "0 CGATCCCATCT motifs detected in file\n",
      "\n",
      "Seq is: TAGGCTCATAC\n",
      "0 TAGGCTCATAC motifs detected in file\n",
      "\n",
      "Seq is: AACCTCTTTAT\n",
      "0 AACCTCTTTAT motifs detected in file\n",
      "\n",
      "Seq is: TTTACACGTAT\n",
      "0 TTTACACGTAT motifs detected in file\n",
      "\n",
      "Seq is: CACCCCTAAAA\n",
      "0 CACCCCTAAAA motifs detected in file\n",
      "\n",
      "Seq is: TAAAATTCAAA\n",
      "0 TAAAATTCAAA motifs detected in file\n",
      "\n",
      "Seq is: CAACCAATCCA\n",
      "0 CAACCAATCCA motifs detected in file\n",
      "\n",
      "Seq is: TCAACTCCAGA\n",
      "0 TCAACTCCAGA motifs detected in file\n",
      "\n",
      "Seq is: TCCCGACAATC\n",
      "0 TCCCGACAATC motifs detected in file\n",
      "\n",
      "Seq is: TTCAGGATCCT\n",
      "0 TTCAGGATCCT motifs detected in file\n",
      "\n",
      "Seq is: TTCACTGTCGC\n",
      "0 TTCACTGTCGC motifs detected in file\n",
      "\n",
      "Seq is: TCCCGACACTC\n",
      "0 TCCCGACACTC motifs detected in file\n",
      "\n",
      "Seq is: AAAATCCATCT\n",
      "0 AAAATCCATCT motifs detected in file\n",
      "\n",
      "Complement found from 10935 to 10946\n",
      "Seq is: CCCCCTAACAA\n",
      "1 CCCCCTAACAA motifs detected in file\n",
      "\n",
      "Seq is: CAAACCTTCCC\n",
      "0 CAAACCTTCCC motifs detected in file\n",
      "\n",
      "Seq is: CCTACGCCCCC\n",
      "0 CCTACGCCCCC motifs detected in file\n",
      "\n",
      "Seq is: TTTACAACAAT\n",
      "0 TTTACAACAAT motifs detected in file\n",
      "\n",
      "Seq is: CATTACTCTTT\n",
      "0 CATTACTCTTT motifs detected in file\n",
      "\n",
      "Seq is: AGGACGCTTAT\n",
      "0 AGGACGCTTAT motifs detected in file\n",
      "\n",
      "Seq is: CCGAAGGCCGA\n",
      "0 CCGAAGGCCGA motifs detected in file\n",
      "\n",
      "Seq is: CGGTCCGCAAA\n",
      "0 CGGTCCGCAAA motifs detected in file\n",
      "\n",
      "Seq is: TTACCCCAAAT\n",
      "0 TTACCCCAAAT motifs detected in file\n",
      "\n",
      "Seq is: CATCCCACCCC\n",
      "0 CATCCCACCCC motifs detected in file\n",
      "\n",
      "Seq is: AATAAAAGCAA\n",
      "0 AATAAAAGCAA motifs detected in file\n",
      "\n",
      "Seq is: TTACAATCATT\n",
      "0 TTACAATCATT motifs detected in file\n",
      "\n",
      "Seq is: CCCACCCCTTC\n",
      "0 CCCACCCCTTC motifs detected in file\n",
      "\n",
      "Seq is: GCTCCAACTGG\n",
      "0 GCTCCAACTGG motifs detected in file\n",
      "\n",
      "Seq is: ACAATCCCACT\n",
      "0 ACAATCCCACT motifs detected in file\n",
      "\n",
      "Seq is: CTTCTTAATAA\n",
      "0 CTTCTTAATAA motifs detected in file\n",
      "\n",
      "Seq is: GCTCACGATAT\n",
      "0 GCTCACGATAT motifs detected in file\n",
      "\n",
      "Seq is: CCGCGAACAGT\n",
      "0 CCGCGAACAGT motifs detected in file\n",
      "\n",
      "Seq is: CCCTCCATCGC\n",
      "0 CCCTCCATCGC motifs detected in file\n",
      "\n",
      "Seq is: TACTCTCATTA\n",
      "0 TACTCTCATTA motifs detected in file\n",
      "\n",
      "Seq is: TCTATCCCGAG\n",
      "0 TCTATCCCGAG motifs detected in file\n",
      "\n",
      "Seq is: TCCGCAAACAC\n",
      "0 TCCGCAAACAC motifs detected in file\n",
      "\n",
      "Seq is: ATACTATACAA\n",
      "0 ATACTATACAA motifs detected in file\n",
      "\n",
      "Seq is: ACGCCAAAGCT\n",
      "0 ACGCCAAAGCT motifs detected in file\n",
      "\n",
      "Seq is: ACTACACCAGA\n",
      "0 ACTACACCAGA motifs detected in file\n",
      "\n",
      "Seq is: AACCTCATCTT\n",
      "0 AACCTCATCTT motifs detected in file\n",
      "\n",
      "Seq is: TGGACACTCCT\n",
      "0 TGGACACTCCT motifs detected in file\n",
      "\n",
      "Seq is: TTCCATAAGGA\n",
      "0 TTCCATAAGGA motifs detected in file\n",
      "\n",
      "Seq is: CGATTACGATC\n",
      "0 CGATTACGATC motifs detected in file\n",
      "\n",
      "Seq is: CGACGGTTACC\n",
      "0 CGACGGTTACC motifs detected in file\n",
      "\n",
      "Seq is: ACTCCCTCCAA\n",
      "0 ACTCCCTCCAA motifs detected in file\n",
      "\n",
      "Seq is: CTTCACTCTCC\n",
      "0 CTTCACTCTCC motifs detected in file\n",
      "\n",
      "Seq is: ATACCAAAACT\n",
      "0 ATACCAAAACT motifs detected in file\n",
      "\n",
      "Seq is: CATCAGGAGGA\n",
      "0 CATCAGGAGGA motifs detected in file\n",
      "\n",
      "Seq is: TAAAAAGCTTA\n",
      "0 TAAAAAGCTTA motifs detected in file\n",
      "\n",
      "Seq is: TAGAACAAGTA\n",
      "0 TAGAACAAGTA motifs detected in file\n",
      "\n",
      "Seq is: ACAATTCCAAC\n",
      "0 ACAATTCCAAC motifs detected in file\n",
      "\n",
      "Seq is: ACCTACTACCT\n",
      "0 ACCTACTACCT motifs detected in file\n",
      "\n",
      "Seq is: GGGCCTCGTGT\n",
      "0 GGGCCTCGTGT motifs detected in file\n",
      "\n",
      "Seq is: ATTTATCATAC\n",
      "0 ATTTATCATAC motifs detected in file\n",
      "\n",
      "Seq is: CGAAACTTCTT\n",
      "0 CGAAACTTCTT motifs detected in file\n",
      "\n",
      "Seq is: CCGCACCCATG\n",
      "0 CCGCACCCATG motifs detected in file\n",
      "\n",
      "Seq is: TCTACACGTCC\n",
      "0 TCTACACGTCC motifs detected in file\n",
      "\n",
      "Seq is: TTACGATCCAC\n",
      "0 TTACGATCCAC motifs detected in file\n",
      "\n",
      "Seq is: ACCAACCAACT\n",
      "0 ACCAACCAACT motifs detected in file\n",
      "\n",
      "Seq is: ACGGCTAACAT\n",
      "0 ACGGCTAACAT motifs detected in file\n",
      "\n",
      "Seq is: TGATAATACTC\n",
      "0 TGATAATACTC motifs detected in file\n",
      "\n",
      "Seq is: AGGATCAACTG\n",
      "0 AGGATCAACTG motifs detected in file\n",
      "\n",
      "Seq is: AACTTCACCTC\n",
      "0 AACTTCACCTC motifs detected in file\n",
      "\n",
      "Seq is: TTCCGATGCTA\n",
      "0 TTCCGATGCTA motifs detected in file\n",
      "\n",
      "Seq is: AAAAAACTACA\n",
      "0 AAAAAACTACA motifs detected in file\n",
      "\n",
      "Seq is: GTAAAACACAT\n",
      "0 GTAAAACACAT motifs detected in file\n",
      "\n",
      "Seq is: TCCCGCGTCTG\n",
      "0 TCCCGCGTCTG motifs detected in file\n",
      "\n",
      "Seq is: ACGACGCTTGT\n",
      "0 ACGACGCTTGT motifs detected in file\n",
      "\n",
      "Seq is: CTCACCACTAT\n",
      "0 CTCACCACTAT motifs detected in file\n",
      "\n",
      "Seq is: CGCGGATTCGT\n",
      "0 CGCGGATTCGT motifs detected in file\n",
      "\n",
      "Seq is: ATCACAATCTC\n",
      "0 ATCACAATCTC motifs detected in file\n",
      "\n",
      "Seq is: AAACCTAATCA\n",
      "0 AAACCTAATCA motifs detected in file\n",
      "\n",
      "Seq is: CCCGATAAAAG\n",
      "0 CCCGATAAAAG motifs detected in file\n",
      "\n",
      "Seq is: ACGATCCCCCA\n",
      "0 ACGATCCCCCA motifs detected in file\n",
      "\n",
      "Seq is: CCTTCGCCTAC\n",
      "0 CCTTCGCCTAC motifs detected in file\n",
      "\n",
      "Seq is: TCATTCTTCTA\n",
      "0 TCATTCTTCTA motifs detected in file\n",
      "\n",
      "Seq is: AGGACGATGTT\n",
      "0 AGGACGATGTT motifs detected in file\n",
      "\n",
      "Seq is: GATATCACGAA\n",
      "0 GATATCACGAA motifs detected in file\n",
      "\n",
      "Seq is: CTCACCTCATC\n",
      "0 CTCACCTCATC motifs detected in file\n",
      "\n",
      "Seq is: CCGACTCTGAC\n",
      "0 CCGACTCTGAC motifs detected in file\n",
      "\n",
      "Seq is: CCCACCCCGGA\n",
      "0 CCCACCCCGGA motifs detected in file\n",
      "\n",
      "Seq is: AGATACCGACT\n",
      "0 AGATACCGACT motifs detected in file\n",
      "\n",
      "Seq is: CCCCTCAGTCC\n",
      "0 CCCCTCAGTCC motifs detected in file\n",
      "\n",
      "Seq is: CCACCTCTGGA\n",
      "0 CCACCTCTGGA motifs detected in file\n",
      "\n",
      "Seq is: TTAACCCGACT\n",
      "0 TTAACCCGACT motifs detected in file\n",
      "\n",
      "Seq is: AAACGGACGAC\n",
      "0 AAACGGACGAC motifs detected in file\n",
      "\n",
      "Seq is: GACGATCCTCC\n",
      "0 GACGATCCTCC motifs detected in file\n",
      "\n",
      "Seq is: TCCGGATCATC\n",
      "0 TCCGGATCATC motifs detected in file\n",
      "\n",
      "Seq is: ACCCCACTCCG\n",
      "0 ACCCCACTCCG motifs detected in file\n",
      "\n",
      "Seq is: AACCTAATCGC\n",
      "0 AACCTAATCGC motifs detected in file\n",
      "\n",
      "Seq is: AAATCTTCCCG\n",
      "0 AAATCTTCCCG motifs detected in file\n",
      "\n",
      "Seq is: ATAAACAACAC\n",
      "0 ATAAACAACAC motifs detected in file\n",
      "\n",
      "Seq is: CCAGAGTACTC\n",
      "0 CCAGAGTACTC motifs detected in file\n",
      "\n",
      "Seq is: AACCTCACATC\n",
      "0 AACCTCACATC motifs detected in file\n",
      "\n",
      "Seq is: CTATTTAGTAC\n",
      "0 CTATTTAGTAC motifs detected in file\n",
      "\n",
      "Seq is: GATTCCGCTCC\n",
      "0 GATTCCGCTCC motifs detected in file\n",
      "\n",
      "Seq is: TACTTTGGCTA\n",
      "0 TACTTTGGCTA motifs detected in file\n",
      "\n",
      "Seq is: TAGCGGCTATG\n",
      "0 TAGCGGCTATG motifs detected in file\n",
      "\n",
      "Seq is: CCAACATATCC\n",
      "0 CCAACATATCC motifs detected in file\n",
      "\n",
      "Seq is: TAACGAACTTA\n",
      "0 TAACGAACTTA motifs detected in file\n",
      "\n",
      "Seq is: CCGACGACACA\n",
      "0 CCGACGACACA motifs detected in file\n",
      "\n",
      "Seq is: ACCGTAGACGA\n",
      "0 ACCGTAGACGA motifs detected in file\n",
      "\n",
      "Seq is: GCCCGCATAGT\n",
      "0 GCCCGCATAGT motifs detected in file\n",
      "\n",
      "Seq is: AGTTGACTACT\n",
      "0 AGTTGACTACT motifs detected in file\n",
      "\n",
      "Seq is: CGTTCTTCCTA\n",
      "0 CGTTCTTCCTA motifs detected in file\n",
      "\n",
      "Seq is: TATTAAGGATG\n",
      "0 TATTAAGGATG motifs detected in file\n",
      "\n",
      "Seq is: CGGGAGAGTCG\n",
      "0 CGGGAGAGTCG motifs detected in file\n",
      "\n",
      "Seq is: GCTACTTGTCA\n",
      "0 GCTACTTGTCA motifs detected in file\n",
      "\n",
      "Seq is: ACCTTATCCAA\n",
      "0 ACCTTATCCAA motifs detected in file\n",
      "\n",
      "Seq is: CAATCGCCATT\n",
      "0 CAATCGCCATT motifs detected in file\n",
      "\n",
      "Seq is: GATTCTAATCA\n",
      "0 GATTCTAATCA motifs detected in file\n",
      "\n",
      "Seq is: TACCATTAATC\n",
      "0 TACCATTAATC motifs detected in file\n",
      "\n",
      "Seq is: CTTCTACTCAT\n",
      "0 CTTCTACTCAT motifs detected in file\n",
      "\n",
      "Seq is: CTATAAACTTC\n",
      "0 CTATAAACTTC motifs detected in file\n",
      "\n",
      "Seq is: TTGACTAATTA\n",
      "0 TTGACTAATTA motifs detected in file\n",
      "\n",
      "Seq is: CAAACCCAGAC\n",
      "0 CAAACCCAGAC motifs detected in file\n",
      "\n",
      "Seq is: TCAAATATATA\n",
      "0 TCAAATATATA motifs detected in file\n",
      "\n",
      "Seq is: GTGTCACTCTT\n",
      "0 GTGTCACTCTT motifs detected in file\n",
      "\n",
      "Seq is: AAGATACTACC\n",
      "0 AAGATACTACC motifs detected in file\n",
      "\n",
      "Seq is: TGGTACATTGC\n",
      "0 TGGTACATTGC motifs detected in file\n",
      "\n",
      "Seq is: TTGTTACGATG\n",
      "0 TTGTTACGATG motifs detected in file\n",
      "\n",
      "Seq is: TCCCTACTTAT\n",
      "0 TCCCTACTTAT motifs detected in file\n",
      "\n",
      "Seq is: AATACCTCTTC\n",
      "0 AATACCTCTTC motifs detected in file\n",
      "\n",
      "Seq is: ATCAGATCAAA\n",
      "0 ATCAGATCAAA motifs detected in file\n",
      "\n",
      "Seq is: CTTCGAATCCC\n",
      "0 CTTCGAATCCC motifs detected in file\n",
      "\n",
      "Seq is: TCTCGACCCAA\n",
      "0 TCTCGACCCAA motifs detected in file\n",
      "\n",
      "Seq is: CAAACCCAACA\n",
      "0 CAAACCCAACA motifs detected in file\n",
      "\n",
      "Seq is: CCGAGTCACAG\n",
      "0 CCGAGTCACAG motifs detected in file\n",
      "\n",
      "Seq is: TCAAGCTCTAT\n",
      "0 TCAAGCTCTAT motifs detected in file\n",
      "\n",
      "Seq is: TATTGAAGAAC\n",
      "0 TATTGAAGAAC motifs detected in file\n",
      "\n",
      "Seq is: CAGATCCGTGT\n",
      "0 CAGATCCGTGT motifs detected in file\n",
      "\n",
      "Seq is: ACTTATAACAA\n",
      "0 ACTTATAACAA motifs detected in file\n",
      "\n",
      "Seq is: CACCCCTTCTC\n",
      "0 CACCCCTTCTC motifs detected in file\n",
      "\n",
      "Seq is: TGACTATTATT\n",
      "0 TGACTATTATT motifs detected in file\n",
      "\n",
      "Seq is: TCCACCTACGC\n",
      "0 TCCACCTACGC motifs detected in file\n",
      "\n",
      "Seq is: TGTTACCTAAA\n",
      "0 TGTTACCTAAA motifs detected in file\n",
      "\n",
      "Seq is: ATGTATTACCC\n",
      "0 ATGTATTACCC motifs detected in file\n",
      "\n",
      "Seq is: CCATACTCAAA\n",
      "0 CCATACTCAAA motifs detected in file\n",
      "\n",
      "Seq is: AAAAACAATCC\n",
      "0 AAAAACAATCC motifs detected in file\n",
      "\n",
      "Seq is: CAATTGCTCCC\n",
      "0 CAATTGCTCCC motifs detected in file\n",
      "\n",
      "Seq is: ACCATTCCTAC\n",
      "0 ACCATTCCTAC motifs detected in file\n",
      "\n",
      "Seq is: CCCCCTTAATC\n",
      "0 CCCCCTTAATC motifs detected in file\n",
      "\n",
      "Seq is: CCTTCAGTCCC\n",
      "0 CCTTCAGTCCC motifs detected in file\n",
      "\n",
      "Seq is: AATCCCACCAA\n",
      "0 AATCCCACCAA motifs detected in file\n",
      "\n",
      "Seq is: TATCATCACAC\n",
      "0 TATCATCACAC motifs detected in file\n",
      "\n",
      "Seq is: GTACCAATAAT\n",
      "0 GTACCAATAAT motifs detected in file\n",
      "\n",
      "Seq is: GAAAATAAACC\n",
      "0 GAAAATAAACC motifs detected in file\n",
      "\n",
      "Seq is: TCAACGTGGTT\n",
      "0 TCAACGTGGTT motifs detected in file\n",
      "\n",
      "Seq is: TTAAAAACCCC\n",
      "0 TTAAAAACCCC motifs detected in file\n",
      "\n",
      "Seq is: GGATTCTGGTT\n",
      "0 GGATTCTGGTT motifs detected in file\n",
      "\n",
      "Seq is: ACCTATCGACA\n",
      "0 ACCTATCGACA motifs detected in file\n",
      "\n",
      "Seq is: ATAGGAAATTT\n",
      "0 ATAGGAAATTT motifs detected in file\n",
      "\n",
      "Seq is: TCAACTCTTTC\n",
      "0 TCAACTCTTTC motifs detected in file\n",
      "\n",
      "Seq is: GGTACAACAAT\n",
      "0 GGTACAACAAT motifs detected in file\n",
      "\n",
      "Seq is: CTGTACCCCCG\n",
      "0 CTGTACCCCCG motifs detected in file\n",
      "\n",
      "Seq is: TACTCAATCGT\n",
      "0 TACTCAATCGT motifs detected in file\n",
      "\n",
      "Seq is: CAAGAACACTC\n",
      "0 CAAGAACACTC motifs detected in file\n",
      "\n",
      "Seq is: GAAAGAGCCAT\n",
      "0 GAAAGAGCCAT motifs detected in file\n",
      "\n",
      "Seq is: TTATTCCCCAG\n",
      "0 TTATTCCCCAG motifs detected in file\n",
      "\n",
      "Seq is: CATTCGGAGAC\n",
      "0 CATTCGGAGAC motifs detected in file\n",
      "\n",
      "Seq is: AACAGTCTAAG\n",
      "0 AACAGTCTAAG motifs detected in file\n",
      "\n",
      "Seq is: TGTTAGACTAC\n",
      "0 TGTTAGACTAC motifs detected in file\n",
      "\n",
      "Seq is: AAAACCAATTT\n",
      "0 AAAACCAATTT motifs detected in file\n",
      "\n",
      "Seq is: GATATAAATGT\n",
      "0 GATATAAATGT motifs detected in file\n",
      "\n",
      "Seq is: TCTCCTTTTGG\n",
      "0 TCTCCTTTTGG motifs detected in file\n",
      "\n",
      "Seq is: GCCATTACTAC\n",
      "0 GCCATTACTAC motifs detected in file\n",
      "\n",
      "Seq is: AGCCCCAACTC\n",
      "0 AGCCCCAACTC motifs detected in file\n",
      "\n",
      "Seq is: CCTATCCTCCT\n",
      "0 CCTATCCTCCT motifs detected in file\n",
      "\n",
      "Seq is: CTTACCCCCTA\n",
      "0 CTTACCCCCTA motifs detected in file\n",
      "\n",
      "Seq is: TCCACATACTT\n",
      "0 TCCACATACTT motifs detected in file\n",
      "\n",
      "Seq is: GTACTCCCACA\n",
      "0 GTACTCCCACA motifs detected in file\n",
      "\n",
      "Complement found from 1935 to 1946\n",
      "Seq is: AAAGAGCACAC\n",
      "1 AAAGAGCACAC motifs detected in file\n",
      "\n",
      "Seq is: TTACTCCCAAA\n",
      "0 TTACTCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: ATACAACAATT\n",
      "0 ATACAACAATT motifs detected in file\n",
      "\n",
      "Seq is: ACACCACCCAC\n",
      "0 ACACCACCCAC motifs detected in file\n",
      "\n",
      "Seq is: TCACTCGGGGT\n",
      "0 TCACTCGGGGT motifs detected in file\n",
      "\n",
      "Seq is: AACACAACACC\n",
      "0 AACACAACACC motifs detected in file\n",
      "\n",
      "Seq is: ATTTATACATC\n",
      "0 ATTTATACATC motifs detected in file\n",
      "\n",
      "Seq is: TCCCTCATATC\n",
      "0 TCCCTCATATC motifs detected in file\n",
      "\n",
      "Seq is: CCGACACTGAT\n",
      "0 CCGACACTGAT motifs detected in file\n",
      "\n",
      "Seq is: CATACAACTCA\n",
      "0 CATACAACTCA motifs detected in file\n",
      "\n",
      "Seq is: GGACATTCATC\n",
      "0 GGACATTCATC motifs detected in file\n",
      "\n",
      "Seq is: CTCTCACTATA\n",
      "0 CTCTCACTATA motifs detected in file\n",
      "\n",
      "Seq is: AACTAGTCCTC\n",
      "0 AACTAGTCCTC motifs detected in file\n",
      "\n",
      "Seq is: TTGCACCAATG\n",
      "0 TTGCACCAATG motifs detected in file\n",
      "\n",
      "Seq is: ATCGTGTCTCT\n",
      "0 ATCGTGTCTCT motifs detected in file\n",
      "\n",
      "Seq is: CAAGAGGGTCA\n",
      "0 CAAGAGGGTCA motifs detected in file\n",
      "\n",
      "Seq is: TCCAATTATCA\n",
      "0 TCCAATTATCA motifs detected in file\n",
      "\n",
      "Seq is: CCCCCCATTCC\n",
      "0 CCCCCCATTCC motifs detected in file\n",
      "\n",
      "Seq is: GCTCCAATCGC\n",
      "0 GCTCCAATCGC motifs detected in file\n",
      "\n",
      "Seq is: TCCGAACGATC\n",
      "0 TCCGAACGATC motifs detected in file\n",
      "\n",
      "Seq is: TTCAGTAGTTT\n",
      "0 TTCAGTAGTTT motifs detected in file\n",
      "\n",
      "Seq is: TTCGATAATCA\n",
      "0 TTCGATAATCA motifs detected in file\n",
      "\n",
      "Seq is: CCCTCATCTCA\n",
      "0 CCCTCATCTCA motifs detected in file\n",
      "\n",
      "Seq is: AACTTCAGGAA\n",
      "0 AACTTCAGGAA motifs detected in file\n",
      "\n",
      "Seq is: CTCTCTCCTAA\n",
      "0 CTCTCTCCTAA motifs detected in file\n",
      "\n",
      "Seq is: TACTACGCTGA\n",
      "0 TACTACGCTGA motifs detected in file\n",
      "\n",
      "Seq is: CACTCACGCAA\n",
      "0 CACTCACGCAA motifs detected in file\n",
      "\n",
      "Complement found from 3669 to 3680\n",
      "Seq is: GCATCAAACTC\n",
      "1 GCATCAAACTC motifs detected in file\n",
      "\n",
      "Seq is: AAACGATCCGT\n",
      "0 AAACGATCCGT motifs detected in file\n",
      "\n",
      "Seq is: CTTATCATTAC\n",
      "0 CTTATCATTAC motifs detected in file\n",
      "\n",
      "Seq is: TCCTACATTCG\n",
      "0 TCCTACATTCG motifs detected in file\n",
      "\n",
      "Seq is: GGCACCCGCTA\n",
      "0 GGCACCCGCTA motifs detected in file\n",
      "\n",
      "Seq is: ATACTCTTACT\n",
      "0 ATACTCTTACT motifs detected in file\n",
      "\n",
      "Seq is: GACGCGGCCAC\n",
      "0 GACGCGGCCAC motifs detected in file\n",
      "\n",
      "Seq is: TTCGAAGTCCC\n",
      "0 TTCGAAGTCCC motifs detected in file\n",
      "\n",
      "Seq is: CCAAACCTACT\n",
      "0 CCAAACCTACT motifs detected in file\n",
      "\n",
      "Seq is: CTTACCGACAA\n",
      "0 CTTACCGACAA motifs detected in file\n",
      "\n",
      "Seq is: TGATGCTCCCG\n",
      "0 TGATGCTCCCG motifs detected in file\n",
      "\n",
      "Seq is: ATACACCGACT\n",
      "0 ATACACCGACT motifs detected in file\n",
      "\n",
      "Seq is: AACTTCTCATA\n",
      "0 AACTTCTCATA motifs detected in file\n",
      "\n",
      "Seq is: CGTTACTCGCT\n",
      "0 CGTTACTCGCT motifs detected in file\n",
      "\n",
      "Seq is: AAAATCCAGAC\n",
      "0 AAAATCCAGAC motifs detected in file\n",
      "\n",
      "Seq is: AAACAGCATCC\n",
      "0 AAACAGCATCC motifs detected in file\n",
      "\n",
      "Seq is: GTCTACCTCGA\n",
      "0 GTCTACCTCGA motifs detected in file\n",
      "\n",
      "Seq is: ACAATATTAAT\n",
      "0 ACAATATTAAT motifs detected in file\n",
      "\n",
      "Seq is: ACGGAGTATCC\n",
      "0 ACGGAGTATCC motifs detected in file\n",
      "\n",
      "Seq is: CTATCATGTTC\n",
      "0 CTATCATGTTC motifs detected in file\n",
      "\n",
      "Seq is: CTTCCCCATCC\n",
      "0 CTTCCCCATCC motifs detected in file\n",
      "\n",
      "Seq is: GATACACAAAA\n",
      "0 GATACACAAAA motifs detected in file\n",
      "\n",
      "Seq is: CAGTCCCCCAA\n",
      "0 CAGTCCCCCAA motifs detected in file\n",
      "\n",
      "Seq is: CTCTTACTCAC\n",
      "0 CTCTTACTCAC motifs detected in file\n",
      "\n",
      "Seq is: ACTCCGCATAA\n",
      "0 ACTCCGCATAA motifs detected in file\n",
      "\n",
      "Seq is: TATGGTATCGG\n",
      "0 TATGGTATCGG motifs detected in file\n",
      "\n",
      "Seq is: CGGATCAAAAT\n",
      "0 CGGATCAAAAT motifs detected in file\n",
      "\n",
      "Seq is: TCTCATGACGC\n",
      "0 TCTCATGACGC motifs detected in file\n",
      "\n",
      "Seq is: CGTTCATGATA\n",
      "0 CGTTCATGATA motifs detected in file\n",
      "\n",
      "Seq is: ACTGGGTCGCT\n",
      "0 ACTGGGTCGCT motifs detected in file\n",
      "\n",
      "Seq is: ACCCCCGAAGC\n",
      "0 ACCCCCGAAGC motifs detected in file\n",
      "\n",
      "Seq is: TGTACCCGAAA\n",
      "0 TGTACCCGAAA motifs detected in file\n",
      "\n",
      "Seq is: TCCCTCAGTAT\n",
      "0 TCCCTCAGTAT motifs detected in file\n",
      "\n",
      "Seq is: TCACCTCAGGC\n",
      "0 TCACCTCAGGC motifs detected in file\n",
      "\n",
      "Seq is: ATTTCTCCATA\n",
      "0 ATTTCTCCATA motifs detected in file\n",
      "\n",
      "Seq is: GAAATGATATT\n",
      "0 GAAATGATATT motifs detected in file\n",
      "\n",
      "Seq is: TTCGATAACAC\n",
      "0 TTCGATAACAC motifs detected in file\n",
      "\n",
      "Seq is: ATTCGATCAGT\n",
      "0 ATTCGATCAGT motifs detected in file\n",
      "\n",
      "Seq is: ATAATTCAACA\n",
      "0 ATAATTCAACA motifs detected in file\n",
      "\n",
      "Seq is: ACCGAGTCCTC\n",
      "0 ACCGAGTCCTC motifs detected in file\n",
      "\n",
      "Seq is: AAACTATCAAG\n",
      "0 AAACTATCAAG motifs detected in file\n",
      "\n",
      "Seq is: AACCCGTCACT\n",
      "0 AACCCGTCACT motifs detected in file\n",
      "\n",
      "Seq is: CTCACTCATCA\n",
      "0 CTCACTCATCA motifs detected in file\n",
      "\n",
      "Seq is: TCTTACAAATC\n",
      "0 TCTTACAAATC motifs detected in file\n",
      "\n",
      "Seq is: ACTCGGATCCC\n",
      "0 ACTCGGATCCC motifs detected in file\n",
      "\n",
      "Seq is: ACAACACTCAC\n",
      "0 ACAACACTCAC motifs detected in file\n",
      "\n",
      "Seq is: ATTTAATCACG\n",
      "0 ATTTAATCACG motifs detected in file\n",
      "\n",
      "Seq is: CTACTCATCCC\n",
      "0 CTACTCATCCC motifs detected in file\n",
      "\n",
      "Seq is: CTTCCCTCGGA\n",
      "0 CTTCCCTCGGA motifs detected in file\n",
      "\n",
      "Seq is: TGATCCCACAT\n",
      "0 TGATCCCACAT motifs detected in file\n",
      "\n",
      "Seq is: CTTATCCTTCA\n",
      "0 CTTATCCTTCA motifs detected in file\n",
      "\n",
      "Seq is: TACACGGACGC\n",
      "0 TACACGGACGC motifs detected in file\n",
      "\n",
      "Seq is: AAGTCCGCAAG\n",
      "0 AAGTCCGCAAG motifs detected in file\n",
      "\n",
      "Seq is: ACCGACCAACG\n",
      "0 ACCGACCAACG motifs detected in file\n",
      "\n",
      "Seq is: GAGTAGCCCAC\n",
      "0 GAGTAGCCCAC motifs detected in file\n",
      "\n",
      "Seq is: TACTATCGGTT\n",
      "0 TACTATCGGTT motifs detected in file\n",
      "\n",
      "Seq is: CCACCCCTATT\n",
      "0 CCACCCCTATT motifs detected in file\n",
      "\n",
      "Seq is: CACACCAAAGC\n",
      "0 CACACCAAAGC motifs detected in file\n",
      "\n",
      "Seq is: TTCTTCTATAT\n",
      "0 TTCTTCTATAT motifs detected in file\n",
      "\n",
      "Complement found from 5492 to 5503\n",
      "Seq is: TTTATACTAAT\n",
      "1 TTTATACTAAT motifs detected in file\n",
      "\n",
      "Seq is: CAAGACACCGA\n",
      "0 CAAGACACCGA motifs detected in file\n",
      "\n",
      "Seq is: CACTTACAATA\n",
      "0 CACTTACAATA motifs detected in file\n",
      "\n",
      "Seq is: TTAATTCCTCT\n",
      "0 TTAATTCCTCT motifs detected in file\n",
      "\n",
      "Seq is: AAACATCCCTC\n",
      "0 AAACATCCCTC motifs detected in file\n",
      "\n",
      "Seq is: TAATCATATCT\n",
      "0 TAATCATATCT motifs detected in file\n",
      "\n",
      "Seq is: CTCCATCTCAA\n",
      "0 CTCCATCTCAA motifs detected in file\n",
      "\n",
      "Seq is: AAAAAGCACTA\n",
      "0 AAAAAGCACTA motifs detected in file\n",
      "\n",
      "Seq is: TCACCAAGTGA\n",
      "0 TCACCAAGTGA motifs detected in file\n",
      "\n",
      "Seq is: CCTATTCACCG\n",
      "0 CCTATTCACCG motifs detected in file\n",
      "\n",
      "Seq is: CAACCGAACGG\n",
      "0 CAACCGAACGG motifs detected in file\n",
      "\n",
      "Seq is: TACTAACACTC\n",
      "0 TACTAACACTC motifs detected in file\n",
      "\n",
      "Seq is: CCCATCCTCAG\n",
      "0 CCCATCCTCAG motifs detected in file\n",
      "\n",
      "Seq is: TCCATCAATCA\n",
      "0 TCCATCAATCA motifs detected in file\n",
      "\n",
      "Seq is: TAATCCTCCCC\n",
      "0 TAATCCTCCCC motifs detected in file\n",
      "\n",
      "Seq is: CCAACAATCCC\n",
      "0 CCAACAATCCC motifs detected in file\n",
      "\n",
      "Seq is: CCAGCCTCCTT\n",
      "0 CCAGCCTCCTT motifs detected in file\n",
      "\n",
      "Seq is: TTCCAACCCCT\n",
      "0 TTCCAACCCCT motifs detected in file\n",
      "\n",
      "Seq is: TGTCGATTTAT\n",
      "0 TGTCGATTTAT motifs detected in file\n",
      "\n",
      "Seq is: CCAACAACAAC\n",
      "0 CCAACAACAAC motifs detected in file\n",
      "\n",
      "Seq is: TAAACCAATTT\n",
      "0 TAAACCAATTT motifs detected in file\n",
      "\n",
      "Seq is: TTTATCATCTC\n",
      "0 TTTATCATCTC motifs detected in file\n",
      "\n",
      "Seq is: CCTACTACGAT\n",
      "0 CCTACTACGAT motifs detected in file\n",
      "\n",
      "Seq is: TATTAATCCGA\n",
      "0 TATTAATCCGA motifs detected in file\n",
      "\n",
      "Seq is: CACCCACCAAC\n",
      "0 CACCCACCAAC motifs detected in file\n",
      "\n",
      "Seq is: ACAACTAAGTT\n",
      "0 ACAACTAAGTT motifs detected in file\n",
      "\n",
      "Seq is: TAATACACAAA\n",
      "0 TAATACACAAA motifs detected in file\n",
      "\n",
      "Seq is: AAACCTTTCAG\n",
      "0 AAACCTTTCAG motifs detected in file\n",
      "\n",
      "Seq is: TACAGTCACCA\n",
      "0 TACAGTCACCA motifs detected in file\n",
      "\n",
      "Seq is: TCATTATATTA\n",
      "0 TCATTATATTA motifs detected in file\n",
      "\n",
      "Seq is: ACAACCCTGCT\n",
      "0 ACAACCCTGCT motifs detected in file\n",
      "\n",
      "Seq is: AATCAAAATCG\n",
      "0 AATCAAAATCG motifs detected in file\n",
      "\n",
      "Seq is: TAACCTCATCC\n",
      "0 TAACCTCATCC motifs detected in file\n",
      "\n",
      "Seq is: AAATCCAATAC\n",
      "0 AAATCCAATAC motifs detected in file\n",
      "\n",
      "Seq is: ATGCATCAGAT\n",
      "0 ATGCATCAGAT motifs detected in file\n",
      "\n",
      "Seq is: CCGGTATACAC\n",
      "0 CCGGTATACAC motifs detected in file\n",
      "\n",
      "Seq is: AACCTCTAACT\n",
      "0 AACCTCTAACT motifs detected in file\n",
      "\n",
      "Seq is: CTGATCATCCC\n",
      "0 CTGATCATCCC motifs detected in file\n",
      "\n",
      "Seq is: GATCCGGGTGG\n",
      "0 GATCCGGGTGG motifs detected in file\n",
      "\n",
      "Seq is: CGACGAAGCGT\n",
      "0 CGACGAAGCGT motifs detected in file\n",
      "\n",
      "Seq is: CCGCCGTTTCT\n",
      "0 CCGCCGTTTCT motifs detected in file\n",
      "\n",
      "Seq is: GATCATACCGT\n",
      "0 GATCATACCGT motifs detected in file\n",
      "\n",
      "Seq is: TATCCGTGTTA\n",
      "0 TATCCGTGTTA motifs detected in file\n",
      "\n",
      "Seq is: TAACCGATTCT\n",
      "0 TAACCGATTCT motifs detected in file\n",
      "\n",
      "Complement found from 16259 to 16270\n",
      "Seq is: CCCTCACCCAC\n",
      "1 CCCTCACCCAC motifs detected in file\n",
      "\n",
      "Seq is: AACTCCCAATA\n",
      "0 AACTCCCAATA motifs detected in file\n",
      "\n",
      "Seq is: CTCTCATCGAT\n",
      "0 CTCTCATCGAT motifs detected in file\n",
      "\n",
      "Seq is: ATTACTTGTCG\n",
      "0 ATTACTTGTCG motifs detected in file\n",
      "\n",
      "Seq is: CTATCATAATA\n",
      "0 CTATCATAATA motifs detected in file\n",
      "\n",
      "Seq is: AGGAAGATCCG\n",
      "0 AGGAAGATCCG motifs detected in file\n",
      "\n",
      "Seq is: TATCATCCCTC\n",
      "0 TATCATCCCTC motifs detected in file\n",
      "\n",
      "Seq is: CTATACTCCAC\n",
      "0 CTATACTCCAC motifs detected in file\n",
      "\n",
      "Seq is: ACTCGCTATAT\n",
      "0 ACTCGCTATAT motifs detected in file\n",
      "\n",
      "Seq is: GATCATAAGGA\n",
      "0 GATCATAAGGA motifs detected in file\n",
      "\n",
      "Seq is: TCTTCACTCTA\n",
      "0 TCTTCACTCTA motifs detected in file\n",
      "\n",
      "Seq is: CCATTTACGAT\n",
      "0 CCATTTACGAT motifs detected in file\n",
      "\n",
      "Seq is: CATATTATAAA\n",
      "0 CATATTATAAA motifs detected in file\n",
      "\n",
      "Seq is: TACATTTACTC\n",
      "0 TACATTTACTC motifs detected in file\n",
      "\n",
      "Seq is: CCCGTAAACCA\n",
      "0 CCCGTAAACCA motifs detected in file\n",
      "\n",
      "Complement found from 5492 to 5503\n",
      "Seq is: TTTATACTAAT\n",
      "1 TTTATACTAAT motifs detected in file\n",
      "\n",
      "Seq is: AGTATTAAATT\n",
      "0 AGTATTAAATT motifs detected in file\n",
      "\n",
      "Seq is: ACTCAGCTTTA\n",
      "0 ACTCAGCTTTA motifs detected in file\n",
      "\n",
      "Seq is: GTAAGCAAAAC\n",
      "0 GTAAGCAAAAC motifs detected in file\n",
      "\n",
      "Seq is: AAATTTGATAT\n",
      "0 AAATTTGATAT motifs detected in file\n",
      "\n",
      "Seq is: ATGGTTAAGCC\n",
      "0 ATGGTTAAGCC motifs detected in file\n",
      "\n",
      "Seq is: AAGTCAGATTA\n",
      "0 AAGTCAGATTA motifs detected in file\n",
      "\n",
      "Seq is: GGAAAAACATC\n",
      "0 GGAAAAACATC motifs detected in file\n",
      "\n",
      "Seq is: AGTGAGTATCC\n",
      "0 AGTGAGTATCC motifs detected in file\n",
      "\n",
      "Seq is: GGTCTGAATCC\n",
      "0 GGTCTGAATCC motifs detected in file\n",
      "\n",
      "Seq is: CGATCCTACTA\n",
      "0 CGATCCTACTA motifs detected in file\n",
      "\n",
      "Seq is: CTAATTATTCT\n",
      "0 CTAATTATTCT motifs detected in file\n",
      "\n",
      "Seq is: CCCTACTGTAT\n",
      "0 CCCTACTGTAT motifs detected in file\n",
      "\n",
      "Seq is: TGATAATCACC\n",
      "0 TGATAATCACC motifs detected in file\n",
      "\n",
      "Seq is: GTCCAATCAAC\n",
      "0 GTCCAATCAAC motifs detected in file\n",
      "\n",
      "Seq is: AAACATCCCGA\n",
      "0 AAACATCCCGA motifs detected in file\n",
      "\n",
      "Seq is: GTACCATCCCC\n",
      "0 GTACCATCCCC motifs detected in file\n",
      "\n",
      "Seq is: ATTTTCCTCCC\n",
      "0 ATTTTCCTCCC motifs detected in file\n",
      "\n",
      "Seq is: GTTAAAGATCT\n",
      "0 GTTAAAGATCT motifs detected in file\n",
      "\n",
      "Seq is: AGTTTATTATT\n",
      "0 AGTTTATTATT motifs detected in file\n",
      "\n",
      "Seq is: CTTCCATTATC\n",
      "0 CTTCCATTATC motifs detected in file\n",
      "\n",
      "Seq is: GATGATTCTTC\n",
      "0 GATGATTCTTC motifs detected in file\n",
      "\n",
      "Seq is: TTAAAATACCT\n",
      "0 TTAAAATACCT motifs detected in file\n",
      "\n",
      "Seq is: CTTTCCCTGCG\n",
      "0 CTTTCCCTGCG motifs detected in file\n",
      "\n",
      "Seq is: CCCGCCCCCTA\n",
      "0 CCCGCCCCCTA motifs detected in file\n",
      "\n",
      "Seq is: TATCCCAGCTT\n",
      "0 TATCCCAGCTT motifs detected in file\n",
      "\n",
      "Seq is: CGGCGTGAGCA\n",
      "0 CGGCGTGAGCA motifs detected in file\n",
      "\n",
      "Seq is: TTCCCCACCTA\n",
      "0 TTCCCCACCTA motifs detected in file\n",
      "\n",
      "Seq is: AAAAGATACAT\n",
      "0 AAAAGATACAT motifs detected in file\n",
      "\n",
      "Seq is: CGGCAACTCAA\n",
      "0 CGGCAACTCAA motifs detected in file\n",
      "\n",
      "Seq is: CACCATCAGTT\n",
      "0 CACCATCAGTT motifs detected in file\n",
      "\n",
      "Seq is: TTACATTATTA\n",
      "0 TTACATTATTA motifs detected in file\n",
      "\n",
      "Seq is: ATAATCATCAT\n",
      "0 ATAATCATCAT motifs detected in file\n",
      "\n",
      "Seq is: TCCGATCCTCC\n",
      "0 TCCGATCCTCC motifs detected in file\n",
      "\n",
      "Seq is: CACAACTAATA\n",
      "0 CACAACTAATA motifs detected in file\n",
      "\n",
      "Seq is: ATTTTAATTCC\n",
      "0 ATTTTAATTCC motifs detected in file\n",
      "\n",
      "Seq is: GCTTCAAATAA\n",
      "0 GCTTCAAATAA motifs detected in file\n",
      "\n",
      "Seq is: TGAGAAAAAAC\n",
      "0 TGAGAAAAAAC motifs detected in file\n",
      "\n",
      "Seq is: TTACAACAGTT\n",
      "0 TTACAACAGTT motifs detected in file\n",
      "\n",
      "Seq is: TTGATCAATTA\n",
      "0 TTGATCAATTA motifs detected in file\n",
      "\n",
      "Seq is: ACCTTCAATTG\n",
      "0 ACCTTCAATTG motifs detected in file\n",
      "\n",
      "Seq is: CCATGATAAAT\n",
      "0 CCATGATAAAT motifs detected in file\n",
      "\n",
      "Seq is: ATGATTTTCTC\n",
      "0 ATGATTTTCTC motifs detected in file\n",
      "\n",
      "Seq is: ATTCTGGGAGT\n",
      "0 ATTCTGGGAGT motifs detected in file\n",
      "\n",
      "Seq is: AGTTATCTACC\n",
      "0 AGTTATCTACC motifs detected in file\n",
      "\n",
      "Complement found from 9958 to 9969\n",
      "Seq is: TCTGTATGTCT\n",
      "1 TCTGTATGTCT motifs detected in file\n",
      "\n",
      "Seq is: TTATCAGTTTG\n",
      "0 TTATCAGTTTG motifs detected in file\n",
      "\n",
      "Seq is: GTGTAGATGTT\n",
      "0 GTGTAGATGTT motifs detected in file\n",
      "\n",
      "Seq is: TTACGGTCATA\n",
      "0 TTACGGTCATA motifs detected in file\n",
      "\n",
      "Seq is: GTCCGCCGCCG\n",
      "0 GTCCGCCGCCG motifs detected in file\n",
      "\n",
      "Seq is: AAGCTTCGGTT\n",
      "0 AAGCTTCGGTT motifs detected in file\n",
      "\n",
      "Seq is: TCACTACAAAC\n",
      "0 TCACTACAAAC motifs detected in file\n",
      "\n",
      "Seq is: CTACATTTCAC\n",
      "0 CTACATTTCAC motifs detected in file\n",
      "\n",
      "Seq is: TTTATAATCAA\n",
      "0 TTTATAATCAA motifs detected in file\n",
      "\n",
      "Seq is: CCGCCTACTTC\n",
      "0 CCGCCTACTTC motifs detected in file\n",
      "\n",
      "Seq is: GTCTATCACTC\n",
      "0 GTCTATCACTC motifs detected in file\n",
      "\n",
      "Seq is: CTTTCAACTCG\n",
      "0 CTTTCAACTCG motifs detected in file\n",
      "\n",
      "Seq is: GTTATTACTGC\n",
      "0 GTTATTACTGC motifs detected in file\n",
      "\n",
      "Seq is: ACTTCAGGCAC\n",
      "0 ACTTCAGGCAC motifs detected in file\n",
      "\n",
      "Seq is: CTTCGGACACC\n",
      "0 CTTCGGACACC motifs detected in file\n",
      "\n",
      "Seq is: GATGTTTTTTA\n",
      "0 GATGTTTTTTA motifs detected in file\n",
      "\n",
      "Seq is: CAACTCGGCAT\n",
      "0 CAACTCGGCAT motifs detected in file\n",
      "\n",
      "Seq is: CTACGGCAGCC\n",
      "0 CTACGGCAGCC motifs detected in file\n",
      "\n",
      "Seq is: TTTACCACTTC\n",
      "0 TTTACCACTTC motifs detected in file\n",
      "\n",
      "Seq is: CCTCTGAGCTT\n",
      "0 CCTCTGAGCTT motifs detected in file\n",
      "\n",
      "Seq is: CATGAGACTCC\n",
      "0 CATGAGACTCC motifs detected in file\n",
      "\n",
      "Seq is: GAACATCCTCC\n",
      "0 GAACATCCTCC motifs detected in file\n",
      "\n",
      "Seq is: CATTTTATCTC\n",
      "0 CATTTTATCTC motifs detected in file\n",
      "\n",
      "Seq is: TGGGTCATTTT\n",
      "0 TGGGTCATTTT motifs detected in file\n",
      "\n",
      "Seq is: AACATTATTCG\n",
      "0 AACATTATTCG motifs detected in file\n",
      "\n",
      "Seq is: TCACGAACTTA\n",
      "0 TCACGAACTTA motifs detected in file\n",
      "\n",
      "Seq is: ATAAACCAAAG\n",
      "0 ATAAACCAAAG motifs detected in file\n",
      "\n",
      "Seq is: CCAACAAAAGA\n",
      "0 CCAACAAAAGA motifs detected in file\n",
      "\n",
      "Seq is: TAATCTGATAC\n",
      "0 TAATCTGATAC motifs detected in file\n",
      "\n",
      "Seq is: CACTCGAGTCC\n",
      "0 CACTCGAGTCC motifs detected in file\n",
      "\n",
      "Seq is: ACTAACTATGA\n",
      "0 ACTAACTATGA motifs detected in file\n",
      "\n",
      "Seq is: GGACTACGCTC\n",
      "0 GGACTACGCTC motifs detected in file\n",
      "\n",
      "Seq is: ATTATGCCTAC\n",
      "0 ATTATGCCTAC motifs detected in file\n",
      "\n",
      "Seq is: ACAAATCCTCA\n",
      "0 ACAAATCCTCA motifs detected in file\n",
      "\n",
      "Seq is: CCCTGAAGATC\n",
      "0 CCCTGAAGATC motifs detected in file\n",
      "\n",
      "Seq is: CCCTAAATCGC\n",
      "0 CCCTAAATCGC motifs detected in file\n",
      "\n",
      "Seq is: CCCACTACGGA\n",
      "0 CCCACTACGGA motifs detected in file\n",
      "\n",
      "Seq is: CAACCCCCGGT\n",
      "0 CAACCCCCGGT motifs detected in file\n",
      "\n",
      "Seq is: CACGGGAGGAT\n",
      "0 CACGGGAGGAT motifs detected in file\n",
      "\n",
      "Seq is: TAACCCCCCAT\n",
      "0 TAACCCCCCAT motifs detected in file\n",
      "\n",
      "Seq is: CCCCGATCCGA\n",
      "0 CCCCGATCCGA motifs detected in file\n",
      "\n",
      "Seq is: CCTCACCATTT\n",
      "0 CCTCACCATTT motifs detected in file\n",
      "\n",
      "Seq is: TCCGAGTCTTT\n",
      "0 TCCGAGTCTTT motifs detected in file\n",
      "\n",
      "Seq is: TTAGGACGCTT\n",
      "0 TTAGGACGCTT motifs detected in file\n",
      "\n",
      "Seq is: CTTTTTTTGAA\n",
      "0 CTTTTTTTGAA motifs detected in file\n",
      "\n",
      "Seq is: GACTCCATTAT\n",
      "0 GACTCCATTAT motifs detected in file\n",
      "\n",
      "Seq is: TTATCCTAATA\n",
      "0 TTATCCTAATA motifs detected in file\n",
      "\n",
      "Seq is: GGGCATAGCTT\n",
      "0 GGGCATAGCTT motifs detected in file\n",
      "\n",
      "Seq is: CCGGAAAAACC\n",
      "0 CCGGAAAAACC motifs detected in file\n",
      "\n",
      "Seq is: TGTCCACCACA\n",
      "0 TGTCCACCACA motifs detected in file\n",
      "\n",
      "Seq is: CACCACCGGAA\n",
      "0 CACCACCGGAA motifs detected in file\n",
      "\n",
      "Seq is: CCATACACGAA\n",
      "0 CCATACACGAA motifs detected in file\n",
      "\n",
      "Seq is: AGAGCACAATG\n",
      "0 AGAGCACAATG motifs detected in file\n",
      "\n",
      "Seq is: TAGCGCGGTAG\n",
      "0 TAGCGCGGTAG motifs detected in file\n",
      "\n",
      "Complement found from 9360 to 9371\n",
      "Seq is: TAACCATATAC\n",
      "1 TAACCATATAC motifs detected in file\n",
      "\n",
      "Seq is: CAATCACACAA\n",
      "0 CAATCACACAA motifs detected in file\n",
      "\n",
      "Seq is: CCAATCATCCG\n",
      "0 CCAATCATCCG motifs detected in file\n",
      "\n",
      "Seq is: GATCATACTCC\n",
      "0 GATCATACTCC motifs detected in file\n",
      "\n",
      "Seq is: TCGCAATACCT\n",
      "0 TCGCAATACCT motifs detected in file\n",
      "\n",
      "Seq is: CACCTTCACTT\n",
      "0 CACCTTCACTT motifs detected in file\n",
      "\n",
      "Seq is: TAGTGTACCGA\n",
      "0 TAGTGTACCGA motifs detected in file\n",
      "\n",
      "Seq is: TCCGGCCTCCA\n",
      "0 TCCGGCCTCCA motifs detected in file\n",
      "\n",
      "Seq is: GTAATCCTCCC\n",
      "0 GTAATCCTCCC motifs detected in file\n",
      "\n",
      "Seq is: GACTCTCCCGG\n",
      "0 GACTCTCCCGG motifs detected in file\n",
      "\n",
      "Seq is: GGACAATCCCC\n",
      "0 GGACAATCCCC motifs detected in file\n",
      "\n",
      "Seq is: AGTACCCGACC\n",
      "0 AGTACCCGACC motifs detected in file\n",
      "\n",
      "Seq is: CAAAATGATAT\n",
      "0 CAAAATGATAT motifs detected in file\n",
      "\n",
      "Seq is: ACTATCCGTAC\n",
      "0 ACTATCCGTAC motifs detected in file\n",
      "\n",
      "Seq is: ACTAACCACCC\n",
      "0 ACTAACCACCC motifs detected in file\n",
      "\n",
      "Seq is: AGTAATACACA\n",
      "0 AGTAATACACA motifs detected in file\n",
      "\n",
      "Seq is: ACAGCACGTCC\n",
      "0 ACAGCACGTCC motifs detected in file\n",
      "\n",
      "Seq is: ATCTCCGAATG\n",
      "0 ATCTCCGAATG motifs detected in file\n",
      "\n",
      "Seq is: ATCTTCACACT\n",
      "0 ATCTTCACACT motifs detected in file\n",
      "\n",
      "Seq is: TTTGCATCCGA\n",
      "0 TTTGCATCCGA motifs detected in file\n",
      "\n",
      "Seq is: ACCTAATTCCG\n",
      "0 ACCTAATTCCG motifs detected in file\n",
      "\n",
      "Seq is: CTGTCGCTAAA\n",
      "0 CTGTCGCTAAA motifs detected in file\n",
      "\n",
      "Seq is: GATCCTATCAG\n",
      "0 GATCCTATCAG motifs detected in file\n",
      "\n",
      "Seq is: TCATCTTAATC\n",
      "0 TCATCTTAATC motifs detected in file\n",
      "\n",
      "Seq is: TTAACACTTCT\n",
      "0 TTAACACTTCT motifs detected in file\n",
      "\n",
      "Seq is: ACTATTCACAT\n",
      "0 ACTATTCACAT motifs detected in file\n",
      "\n",
      "Seq is: CTCCCTTCCAA\n",
      "0 CTCCCTTCCAA motifs detected in file\n",
      "\n",
      "Seq is: TTACCAACTAT\n",
      "0 TTACCAACTAT motifs detected in file\n",
      "\n",
      "Seq is: AACGATCCCAC\n",
      "0 AACGATCCCAC motifs detected in file\n",
      "\n",
      "Seq is: CGCGAAGGTTA\n",
      "0 CGCGAAGGTTA motifs detected in file\n",
      "\n",
      "Seq is: ATCCACGTACT\n",
      "0 ATCCACGTACT motifs detected in file\n",
      "\n",
      "Seq is: CATCCACCGGA\n",
      "0 CATCCACCGGA motifs detected in file\n",
      "\n",
      "Seq is: CGTCATTACAA\n",
      "0 CGTCATTACAA motifs detected in file\n",
      "\n",
      "Seq is: TCGCCAATCCG\n",
      "0 TCGCCAATCCG motifs detected in file\n",
      "\n",
      "Seq is: CATGCCGGTCC\n",
      "0 CATGCCGGTCC motifs detected in file\n",
      "\n",
      "Seq is: CGATAACCAAC\n",
      "0 CGATAACCAAC motifs detected in file\n",
      "\n",
      "Complement found from 13110 to 13121\n",
      "Seq is: TTACTCATCCG\n",
      "1 TTACTCATCCG motifs detected in file\n",
      "\n",
      "Seq is: ACTACCAAAGC\n",
      "0 ACTACCAAAGC motifs detected in file\n",
      "\n",
      "Seq is: TATTATTGATC\n",
      "0 TATTATTGATC motifs detected in file\n",
      "\n",
      "Seq is: ATACCCCTATT\n",
      "0 ATACCCCTATT motifs detected in file\n",
      "\n",
      "Seq is: CCCCACATCCA\n",
      "0 CCCCACATCCA motifs detected in file\n",
      "\n",
      "Seq is: CACGGAACACC\n",
      "0 CACGGAACACC motifs detected in file\n",
      "\n",
      "Seq is: ATTCTTCACCC\n",
      "0 ATTCTTCACCC motifs detected in file\n",
      "\n",
      "Seq is: GATCCCGTAAA\n",
      "0 GATCCCGTAAA motifs detected in file\n",
      "\n",
      "Seq is: AATTAGAATCT\n",
      "0 AATTAGAATCT motifs detected in file\n",
      "\n",
      "Seq is: CGCTTTCGGAT\n",
      "0 CGCTTTCGGAT motifs detected in file\n",
      "\n",
      "Seq is: ATTAGTGACAC\n",
      "0 ATTAGTGACAC motifs detected in file\n",
      "\n",
      "Seq is: GGGCGAGTATT\n",
      "0 GGGCGAGTATT motifs detected in file\n",
      "\n",
      "Seq is: CCCCTACCGGT\n",
      "0 CCCCTACCGGT motifs detected in file\n",
      "\n",
      "Seq is: ACCGATCCAAA\n",
      "0 ACCGATCCAAA motifs detected in file\n",
      "\n",
      "Seq is: TATCTATCAAC\n",
      "0 TATCTATCAAC motifs detected in file\n",
      "\n",
      "Seq is: CCACCAACCAC\n",
      "0 CCACCAACCAC motifs detected in file\n",
      "\n",
      "Seq is: ATTTACTCACT\n",
      "0 ATTTACTCACT motifs detected in file\n",
      "\n",
      "Seq is: CCGTCCTCAGG\n",
      "0 CCGTCCTCAGG motifs detected in file\n",
      "\n",
      "Seq is: CTCCTCCAATC\n",
      "0 CTCCTCCAATC motifs detected in file\n",
      "\n",
      "Seq is: AACACCGTTAT\n",
      "0 AACACCGTTAT motifs detected in file\n",
      "\n",
      "Seq is: TTTTACTAATT\n",
      "0 TTTTACTAATT motifs detected in file\n",
      "\n",
      "Seq is: CCTATGATCAT\n",
      "0 CCTATGATCAT motifs detected in file\n",
      "\n",
      "Seq is: ATTCTCTAGTC\n",
      "0 ATTCTCTAGTC motifs detected in file\n",
      "\n",
      "Seq is: CAAGCAGGAAA\n",
      "0 CAAGCAGGAAA motifs detected in file\n",
      "\n",
      "Seq is: TCACAACACAT\n",
      "0 TCACAACACAT motifs detected in file\n",
      "\n",
      "Seq is: ACCAATAGTAA\n",
      "0 ACCAATAGTAA motifs detected in file\n",
      "\n",
      "Seq is: ACAAAACTCCA\n",
      "0 ACAAAACTCCA motifs detected in file\n",
      "\n",
      "Seq is: ATCAAACTAAT\n",
      "0 ATCAAACTAAT motifs detected in file\n",
      "\n",
      "Seq is: CAGTAACAACC\n",
      "0 CAGTAACAACC motifs detected in file\n",
      "\n",
      "Seq is: CACCACTAATC\n",
      "0 CACCACTAATC motifs detected in file\n",
      "\n",
      "Complement found from 11328 to 11339\n",
      "Seq is: AGCCAACAACT\n",
      "1 AGCCAACAACT motifs detected in file\n",
      "\n",
      "Seq is: ACTCTATAAAC\n",
      "0 ACTCTATAAAC motifs detected in file\n",
      "\n",
      "Seq is: CTCCACCCCTA\n",
      "0 CTCCACCCCTA motifs detected in file\n",
      "\n",
      "Seq is: GTTATCTCCCC\n",
      "0 GTTATCTCCCC motifs detected in file\n",
      "\n",
      "Seq is: CTTTATCTTAC\n",
      "0 CTTTATCTTAC motifs detected in file\n",
      "\n",
      "Seq is: TAGTCATGACG\n",
      "0 TAGTCATGACG motifs detected in file\n",
      "\n",
      "Seq is: CCGCCCATCCG\n",
      "0 CCGCCCATCCG motifs detected in file\n",
      "\n",
      "Seq is: GATCCTAACAC\n",
      "0 GATCCTAACAC motifs detected in file\n",
      "\n",
      "Seq is: CCCCGTTACTT\n",
      "0 CCCCGTTACTT motifs detected in file\n",
      "\n",
      "Seq is: ACTTCGCTTGT\n",
      "0 ACTTCGCTTGT motifs detected in file\n",
      "\n",
      "Seq is: CTAAAAGCAAG\n",
      "0 CTAAAAGCAAG motifs detected in file\n",
      "\n",
      "Seq is: TAAAACCAAGA\n",
      "0 TAAAACCAAGA motifs detected in file\n",
      "\n",
      "Seq is: GTCCCAAACAA\n",
      "0 GTCCCAAACAA motifs detected in file\n",
      "\n",
      "Seq is: TATTAAAAAAT\n",
      "0 TATTAAAAAAT motifs detected in file\n",
      "\n",
      "Seq is: AAAAATACCCG\n",
      "0 AAAAATACCCG motifs detected in file\n",
      "\n",
      "Seq is: AAACCACTCCC\n",
      "0 AAACCACTCCC motifs detected in file\n",
      "\n",
      "Seq is: TCCATCCACCA\n",
      "0 TCCATCCACCA motifs detected in file\n",
      "\n",
      "Seq is: TCAAACACAAA\n",
      "0 TCAAACACAAA motifs detected in file\n",
      "\n",
      "Seq is: TTATAAAAATC\n",
      "0 TTATAAAAATC motifs detected in file\n",
      "\n",
      "Seq is: AACCCACTACT\n",
      "0 AACCCACTACT motifs detected in file\n",
      "\n",
      "Seq is: CCTTATCACAT\n",
      "0 CCTTATCACAT motifs detected in file\n",
      "\n",
      "Seq is: TCCTCATACCC\n",
      "0 TCCTCATACCC motifs detected in file\n",
      "\n",
      "Seq is: CCATTAATACC\n",
      "0 CCATTAATACC motifs detected in file\n",
      "\n",
      "Seq is: ACCCGGTATGC\n",
      "0 ACCCGGTATGC motifs detected in file\n",
      "\n",
      "Seq is: CATCATAAATC\n",
      "0 CATCATAAATC motifs detected in file\n",
      "\n",
      "Seq is: AACCCCGTAAA\n",
      "0 AACCCCGTAAA motifs detected in file\n",
      "\n",
      "Seq is: GTGACATTTCT\n",
      "0 GTGACATTTCT motifs detected in file\n",
      "\n",
      "Seq is: CCACAACCAAG\n",
      "0 CCACAACCAAG motifs detected in file\n",
      "\n",
      "Seq is: AGAATTAGAAA\n",
      "0 AGAATTAGAAA motifs detected in file\n",
      "\n",
      "Seq is: TTGAATTTTCC\n",
      "0 TTGAATTTTCC motifs detected in file\n",
      "\n",
      "Seq is: AATTACGATTC\n",
      "0 AATTACGATTC motifs detected in file\n",
      "\n",
      "Seq is: AATCGAAATGT\n",
      "0 AATCGAAATGT motifs detected in file\n",
      "\n",
      "Seq is: CACCCGAGATC\n",
      "0 CACCCGAGATC motifs detected in file\n",
      "\n",
      "Seq is: TCCCCCATCTC\n",
      "0 TCCCCCATCTC motifs detected in file\n",
      "\n",
      "Seq is: CCCCACGATAT\n",
      "0 CCCCACGATAT motifs detected in file\n",
      "\n",
      "Seq is: CCCATTTATGC\n",
      "0 CCCATTTATGC motifs detected in file\n",
      "\n",
      "Seq is: CCGGGATAAAG\n",
      "0 CCGGGATAAAG motifs detected in file\n",
      "\n",
      "Seq is: TTTCTAAAAAT\n",
      "0 TTTCTAAAAAT motifs detected in file\n",
      "\n",
      "Seq is: CCCCTTAATTA\n",
      "0 CCCCTTAATTA motifs detected in file\n",
      "\n",
      "Seq is: AGATCCTGCTA\n",
      "0 AGATCCTGCTA motifs detected in file\n",
      "\n",
      "Seq is: CCCGTACTTTG\n",
      "0 CCCGTACTTTG motifs detected in file\n",
      "\n",
      "Seq is: ACACCAAACGA\n",
      "0 ACACCAAACGA motifs detected in file\n",
      "\n",
      "Seq is: GGTGTCTAAAG\n",
      "0 GGTGTCTAAAG motifs detected in file\n",
      "\n",
      "Seq is: TCTCGTAACTG\n",
      "0 TCTCGTAACTG motifs detected in file\n",
      "\n",
      "Seq is: GCATCATATGG\n",
      "0 GCATCATATGG motifs detected in file\n",
      "\n",
      "Seq is: GGGCCAGCACA\n",
      "0 GGGCCAGCACA motifs detected in file\n",
      "\n",
      "Seq is: TCGCCACTTTC\n",
      "0 TCGCCACTTTC motifs detected in file\n",
      "\n",
      "Seq is: ACCAAACCAAA\n",
      "0 ACCAAACCAAA motifs detected in file\n",
      "\n",
      "Seq is: TCTGCAGGCCC\n",
      "0 TCTGCAGGCCC motifs detected in file\n",
      "\n",
      "Seq is: TTAACGTAGAC\n",
      "0 TTAACGTAGAC motifs detected in file\n",
      "\n",
      "Seq is: AAAAATTCGGA\n",
      "0 AAAAATTCGGA motifs detected in file\n",
      "\n",
      "Seq is: TTACACCCCTG\n",
      "0 TTACACCCCTG motifs detected in file\n",
      "\n",
      "Seq is: TCGAGTACTCA\n",
      "0 TCGAGTACTCA motifs detected in file\n",
      "\n",
      "Seq is: CGTTCTGCAGA\n",
      "0 CGTTCTGCAGA motifs detected in file\n",
      "\n",
      "Seq is: ACACTACATTA\n",
      "0 ACACTACATTA motifs detected in file\n",
      "\n",
      "Seq is: ATAATATGCTT\n",
      "0 ATAATATGCTT motifs detected in file\n",
      "\n",
      "Seq is: ACCCCCGAAGT\n",
      "0 ACCCCCGAAGT motifs detected in file\n",
      "\n",
      "Seq is: TAGCCCTCATG\n",
      "0 TAGCCCTCATG motifs detected in file\n",
      "\n",
      "Seq is: ATGAGCTAACA\n",
      "0 ATGAGCTAACA motifs detected in file\n",
      "\n",
      "Seq is: GTTGCAGTTCC\n",
      "0 GTTGCAGTTCC motifs detected in file\n",
      "\n",
      "Seq is: TCAGCGTCCAG\n",
      "0 TCAGCGTCCAG motifs detected in file\n",
      "\n",
      "Seq is: CGGACCAAGAT\n",
      "0 CGGACCAAGAT motifs detected in file\n",
      "\n",
      "Seq is: CCTTATTACCC\n",
      "0 CCTTATTACCC motifs detected in file\n",
      "\n",
      "Seq is: CCTTCATACAT\n",
      "0 CCTTCATACAT motifs detected in file\n",
      "\n",
      "Seq is: CCTCAACTTCT\n",
      "0 CCTCAACTTCT motifs detected in file\n",
      "\n",
      "Seq is: AATCAGGCGGC\n",
      "0 AATCAGGCGGC motifs detected in file\n",
      "\n",
      "Complement found from 11629 to 11640\n",
      "Seq is: ATCAGCCACAT\n",
      "1 ATCAGCCACAT motifs detected in file\n",
      "\n",
      "Seq is: GAGCATCCAAG\n",
      "0 GAGCATCCAAG motifs detected in file\n",
      "\n",
      "Seq is: TCATGGTAACC\n",
      "0 TCATGGTAACC motifs detected in file\n",
      "\n",
      "Seq is: ACCGGTTAACT\n",
      "0 ACCGGTTAACT motifs detected in file\n",
      "\n",
      "Seq is: AAACTACCATT\n",
      "0 AAACTACCATT motifs detected in file\n",
      "\n",
      "Seq is: CCCTCCCTAGC\n",
      "0 CCCTCCCTAGC motifs detected in file\n",
      "\n",
      "Seq is: AACTGGAGCAG\n",
      "0 AACTGGAGCAG motifs detected in file\n",
      "\n",
      "Seq is: ACAATACATTT\n",
      "0 ACAATACATTT motifs detected in file\n",
      "\n",
      "Complement found from 7824 to 7835\n",
      "Seq is: CCTACGCATCC\n",
      "1 CCTACGCATCC motifs detected in file\n",
      "\n",
      "Seq is: CTACCCTCCCG\n",
      "0 CTACCCTCCCG motifs detected in file\n",
      "\n",
      "Seq is: CTACTCCTGAT\n",
      "0 CTACTCCTGAT motifs detected in file\n",
      "\n",
      "Seq is: CCTACTACCGC\n",
      "0 CCTACTACCGC motifs detected in file\n",
      "\n",
      "Seq is: CCGTCCTATCA\n",
      "0 CCGTCCTATCA motifs detected in file\n",
      "\n",
      "Seq is: AGTCTGCCAAA\n",
      "0 AGTCTGCCAAA motifs detected in file\n",
      "\n",
      "Seq is: GATAAAGGACT\n",
      "0 GATAAAGGACT motifs detected in file\n",
      "\n",
      "Seq is: CGCAGACTCTA\n",
      "0 CGCAGACTCTA motifs detected in file\n",
      "\n",
      "Seq is: CAATCATAATC\n",
      "0 CAATCATAATC motifs detected in file\n",
      "\n",
      "Seq is: AATCAAAACAA\n",
      "0 AATCAAAACAA motifs detected in file\n",
      "\n",
      "Seq is: CACTCACAATC\n",
      "0 CACTCACAATC motifs detected in file\n",
      "\n",
      "Seq is: CTTTTCCCGTA\n",
      "0 CTTTTCCCGTA motifs detected in file\n",
      "\n",
      "Seq is: TGTCCTGATCC\n",
      "0 TGTCCTGATCC motifs detected in file\n",
      "\n",
      "Seq is: TTCGTCTATTC\n",
      "0 TTCGTCTATTC motifs detected in file\n",
      "\n",
      "Seq is: CTTTTACTAAT\n",
      "0 CTTTTACTAAT motifs detected in file\n",
      "\n",
      "Seq is: ACTCCCGCACT\n",
      "0 ACTCCCGCACT motifs detected in file\n",
      "\n",
      "Seq is: AGTACTTTCCA\n",
      "0 AGTACTTTCCA motifs detected in file\n",
      "\n",
      "Seq is: CTATTCGAGAA\n",
      "0 CTATTCGAGAA motifs detected in file\n",
      "\n",
      "Seq is: GATACTATCCC\n",
      "0 GATACTATCCC motifs detected in file\n",
      "\n",
      "Seq is: CTTCATCGCAG\n",
      "0 CTTCATCGCAG motifs detected in file\n",
      "\n",
      "Seq is: AACATCTGGAT\n",
      "0 AACATCTGGAT motifs detected in file\n",
      "\n",
      "Seq is: GAACGCGACGT\n",
      "0 GAACGCGACGT motifs detected in file\n",
      "\n",
      "Seq is: ACACGGTAATT\n",
      "0 ACACGGTAATT motifs detected in file\n",
      "\n",
      "Seq is: CTATATATCCT\n",
      "0 CTATATATCCT motifs detected in file\n",
      "\n",
      "Seq is: AAATCGGATAT\n",
      "0 AAATCGGATAT motifs detected in file\n",
      "\n",
      "Seq is: TAAATTGAAAC\n",
      "0 TAAATTGAAAC motifs detected in file\n",
      "\n",
      "Seq is: TGTTTCAATAC\n",
      "0 TGTTTCAATAC motifs detected in file\n",
      "\n",
      "Complement found from 2496 to 2507\n",
      "Seq is: TTTACCAAAAA\n",
      "1 TTTACCAAAAA motifs detected in file\n",
      "\n",
      "Seq is: GATTATGGAAA\n",
      "0 GATTATGGAAA motifs detected in file\n",
      "\n",
      "Seq is: AACTTTTTCAG\n",
      "0 AACTTTTTCAG motifs detected in file\n",
      "\n",
      "Seq is: TACCTCCGGTA\n",
      "0 TACCTCCGGTA motifs detected in file\n",
      "\n",
      "Seq is: CCCCAACCGAA\n",
      "0 CCCCAACCGAA motifs detected in file\n",
      "\n",
      "Seq is: CTTTGGTCGAA\n",
      "0 CTTTGGTCGAA motifs detected in file\n",
      "\n",
      "Seq is: ACCCCCCAAGC\n",
      "0 ACCCCCCAAGC motifs detected in file\n",
      "\n",
      "Seq is: TAAGGAAGGAA\n",
      "0 TAAGGAAGGAA motifs detected in file\n",
      "\n",
      "Seq is: AAAACAGATCT\n",
      "0 AAAACAGATCT motifs detected in file\n",
      "\n",
      "Seq is: AAAATACATAT\n",
      "0 AAAATACATAT motifs detected in file\n",
      "\n",
      "Seq is: GCCCAAGAAGC\n",
      "0 GCCCAAGAAGC motifs detected in file\n",
      "\n",
      "Seq is: TTACACACCAT\n",
      "0 TTACACACCAT motifs detected in file\n",
      "\n",
      "Seq is: CCCACCCCCCG\n",
      "0 CCCACCCCCCG motifs detected in file\n",
      "\n",
      "Seq is: TAGGTATATCA\n",
      "0 TAGGTATATCA motifs detected in file\n",
      "\n",
      "Seq is: GTGAGGTCCAA\n",
      "0 GTGAGGTCCAA motifs detected in file\n",
      "\n",
      "Seq is: ATACCTCCCAA\n",
      "0 ATACCTCCCAA motifs detected in file\n",
      "\n",
      "Seq is: GAAGATGATAA\n",
      "0 GAAGATGATAA motifs detected in file\n",
      "\n",
      "Seq is: TCCTGAAAAGC\n",
      "0 TCCTGAAAAGC motifs detected in file\n",
      "\n",
      "Seq is: GAAGCTTCGCT\n",
      "0 GAAGCTTCGCT motifs detected in file\n",
      "\n",
      "Seq is: TCCGAAGAGTT\n",
      "0 TCCGAAGAGTT motifs detected in file\n",
      "\n",
      "Seq is: TAGTACTTTTA\n",
      "0 TAGTACTTTTA motifs detected in file\n",
      "\n",
      "Seq is: ATAATTATAAT\n",
      "0 ATAATTATAAT motifs detected in file\n",
      "\n",
      "Seq is: GACGACAATCT\n",
      "0 GACGACAATCT motifs detected in file\n",
      "\n",
      "Seq is: CTTTACTTACT\n",
      "0 CTTTACTTACT motifs detected in file\n",
      "\n",
      "Seq is: CGGATGTCTAC\n",
      "0 CGGATGTCTAC motifs detected in file\n",
      "\n",
      "Seq is: TATCCTACAAA\n",
      "0 TATCCTACAAA motifs detected in file\n",
      "\n",
      "Seq is: GTACACCACAT\n",
      "0 GTACACCACAT motifs detected in file\n",
      "\n",
      "Seq is: ACGTAGCCCCA\n",
      "0 ACGTAGCCCCA motifs detected in file\n",
      "\n",
      "Seq is: TCAGGCTCATT\n",
      "0 TCAGGCTCATT motifs detected in file\n",
      "\n",
      "Seq is: GCAGCCCCGTA\n",
      "0 GCAGCCCCGTA motifs detected in file\n",
      "\n",
      "Seq is: AGGCCTATCCG\n",
      "0 AGGCCTATCCG motifs detected in file\n",
      "\n",
      "Seq is: GCTCTTTCACA\n",
      "0 GCTCTTTCACA motifs detected in file\n",
      "\n",
      "Seq is: ACACCCTTCTT\n",
      "0 ACACCCTTCTT motifs detected in file\n",
      "\n",
      "Seq is: TCAATCTAAAT\n",
      "0 TCAATCTAAAT motifs detected in file\n",
      "\n",
      "Seq is: GCGGCTACTTA\n",
      "0 GCGGCTACTTA motifs detected in file\n",
      "\n",
      "Seq is: TACTATCACTT\n",
      "0 TACTATCACTT motifs detected in file\n",
      "\n",
      "Seq is: TACCTAAAACC\n",
      "0 TACCTAAAACC motifs detected in file\n",
      "\n",
      "Seq is: GCATCCAAACC\n",
      "0 GCATCCAAACC motifs detected in file\n",
      "\n",
      "Seq is: AGATCCCACAT\n",
      "0 AGATCCCACAT motifs detected in file\n",
      "\n",
      "Seq is: CGGACTCTTAT\n",
      "0 CGGACTCTTAT motifs detected in file\n",
      "\n",
      "Seq is: CCCCTTTAGTC\n",
      "0 CCCCTTTAGTC motifs detected in file\n",
      "\n",
      "Seq is: ACTTACTTCGG\n",
      "0 ACTTACTTCGG motifs detected in file\n",
      "\n",
      "Seq is: AGGATACTACC\n",
      "0 AGGATACTACC motifs detected in file\n",
      "\n",
      "Seq is: GTTTATGTCGA\n",
      "0 GTTTATGTCGA motifs detected in file\n",
      "\n",
      "Seq is: GGATAACTATC\n",
      "0 GGATAACTATC motifs detected in file\n",
      "\n",
      "Seq is: CTGTATCACCT\n",
      "0 CTGTATCACCT motifs detected in file\n",
      "\n",
      "Seq is: TCACCCGATGT\n",
      "0 TCACCCGATGT motifs detected in file\n",
      "\n",
      "Seq is: TGCATCATGCA\n",
      "0 TGCATCATGCA motifs detected in file\n",
      "\n",
      "Seq is: CAGCACATCAT\n",
      "0 CAGCACATCAT motifs detected in file\n",
      "\n",
      "Seq is: GCTACAGATCA\n",
      "0 GCTACAGATCA motifs detected in file\n",
      "\n",
      "Seq is: CTACTCAAACG\n",
      "0 CTACTCAAACG motifs detected in file\n",
      "\n",
      "Seq is: ATTATGTTACG\n",
      "0 ATTATGTTACG motifs detected in file\n",
      "\n",
      "Seq is: GTCAGTCCGGT\n",
      "0 GTCAGTCCGGT motifs detected in file\n",
      "\n",
      "Seq is: GGATGCCACTT\n",
      "0 GGATGCCACTT motifs detected in file\n",
      "\n",
      "Seq is: TTCTTTCTACT\n",
      "0 TTCTTTCTACT motifs detected in file\n",
      "\n",
      "Seq is: TAGGATCCCGA\n",
      "0 TAGGATCCCGA motifs detected in file\n",
      "\n",
      "Seq is: GTCTCGTGACG\n",
      "0 GTCTCGTGACG motifs detected in file\n",
      "\n",
      "Seq is: TCGTCTAGTAA\n",
      "0 TCGTCTAGTAA motifs detected in file\n",
      "\n",
      "Seq is: AGTATAACGAA\n",
      "0 AGTATAACGAA motifs detected in file\n",
      "\n",
      "Seq is: GGCACCTCACA\n",
      "0 GGCACCTCACA motifs detected in file\n",
      "\n",
      "Seq is: CCGCTCAGTCG\n",
      "0 CCGCTCAGTCG motifs detected in file\n",
      "\n",
      "Seq is: ATTTATGAAAC\n",
      "0 ATTTATGAAAC motifs detected in file\n",
      "\n",
      "Seq is: TGCGGCCACCC\n",
      "0 TGCGGCCACCC motifs detected in file\n",
      "\n",
      "Seq is: CTATCGCTACT\n",
      "0 CTATCGCTACT motifs detected in file\n",
      "\n",
      "Seq is: AATACCATCGC\n",
      "0 AATACCATCGC motifs detected in file\n",
      "\n",
      "Seq is: CTCCACTTTAT\n",
      "0 CTCCACTTTAT motifs detected in file\n",
      "\n",
      "Seq is: ACGAGCACACA\n",
      "0 ACGAGCACACA motifs detected in file\n",
      "\n",
      "Seq is: GATGCAGATAA\n",
      "0 GATGCAGATAA motifs detected in file\n",
      "\n",
      "Seq is: GGATGACATTT\n",
      "0 GGATGACATTT motifs detected in file\n",
      "\n",
      "Seq is: ATATACCACAC\n",
      "0 ATATACCACAC motifs detected in file\n",
      "\n",
      "Seq is: GAGTGTGCTAT\n",
      "0 GAGTGTGCTAT motifs detected in file\n",
      "\n",
      "Seq is: TTGGGATCCTT\n",
      "0 TTGGGATCCTT motifs detected in file\n",
      "\n",
      "Seq is: CGGTTAACTAT\n",
      "0 CGGTTAACTAT motifs detected in file\n",
      "\n",
      "Seq is: AGTATCGAGTC\n",
      "0 AGTATCGAGTC motifs detected in file\n",
      "\n",
      "Seq is: TGGTATGGATA\n",
      "0 TGGTATGGATA motifs detected in file\n",
      "\n",
      "Seq is: CATAGGTTTAC\n",
      "0 CATAGGTTTAC motifs detected in file\n",
      "\n",
      "Seq is: CAAGAAAAAAA\n",
      "0 CAAGAAAAAAA motifs detected in file\n",
      "\n",
      "Seq is: GGCCTCATCAT\n",
      "0 GGCCTCATCAT motifs detected in file\n",
      "\n",
      "Seq is: TCAATGTTATA\n",
      "0 TCAATGTTATA motifs detected in file\n",
      "\n",
      "Seq is: CCCTCTAATAA\n",
      "0 CCCTCTAATAA motifs detected in file\n",
      "\n",
      "Seq is: GGCTTCGGACC\n",
      "0 GGCTTCGGACC motifs detected in file\n",
      "\n",
      "Seq is: ATCCTATTCTT\n",
      "0 ATCCTATTCTT motifs detected in file\n",
      "\n",
      "Seq is: ATATTTGAAGT\n",
      "0 ATATTTGAAGT motifs detected in file\n",
      "\n",
      "Seq is: CCCACTGGCTT\n",
      "0 CCCACTGGCTT motifs detected in file\n",
      "\n",
      "Seq is: TTTAGTCTTAT\n",
      "0 TTTAGTCTTAT motifs detected in file\n",
      "\n",
      "Seq is: CCACAACCATA\n",
      "0 CCACAACCATA motifs detected in file\n",
      "\n",
      "Seq is: TCTTACCCCAG\n",
      "0 TCTTACCCCAG motifs detected in file\n",
      "\n",
      "Seq is: AGGAGGAGGCC\n",
      "0 AGGAGGAGGCC motifs detected in file\n",
      "\n",
      "Seq is: GCCCCAGCTTC\n",
      "0 GCCCCAGCTTC motifs detected in file\n",
      "\n",
      "Seq is: TTCCACCACAA\n",
      "0 TTCCACCACAA motifs detected in file\n",
      "\n",
      "Seq is: CTCCAACGCCA\n",
      "0 CTCCAACGCCA motifs detected in file\n",
      "\n",
      "Seq is: GACAATCATCA\n",
      "0 GACAATCATCA motifs detected in file\n",
      "\n",
      "Seq is: TATCACTACGG\n",
      "0 TATCACTACGG motifs detected in file\n",
      "\n",
      "Seq is: TCGTCGATCCT\n",
      "0 TCGTCGATCCT motifs detected in file\n",
      "\n",
      "Seq is: GACCCTCTCTA\n",
      "0 GACCCTCTCTA motifs detected in file\n",
      "\n",
      "Seq is: TCCTCTTCATC\n",
      "0 TCCTCTTCATC motifs detected in file\n",
      "\n",
      "Seq is: CTGACGACACT\n",
      "0 CTGACGACACT motifs detected in file\n",
      "\n",
      "Seq is: AATCCTGCCTA\n",
      "0 AATCCTGCCTA motifs detected in file\n",
      "\n",
      "Seq is: GTCTGCTTCTC\n",
      "0 GTCTGCTTCTC motifs detected in file\n",
      "\n",
      "Seq is: CCCGCAAACCA\n",
      "0 CCCGCAAACCA motifs detected in file\n",
      "\n",
      "Complement found from 6429 to 6440\n",
      "Seq is: TAACCCAATAC\n",
      "1 TAACCCAATAC motifs detected in file\n",
      "\n",
      "Seq is: CGTCCCCCAAA\n",
      "0 CGTCCCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: ATATAACTATT\n",
      "0 ATATAACTATT motifs detected in file\n",
      "\n",
      "Seq is: AACAACACTAC\n",
      "0 AACAACACTAC motifs detected in file\n",
      "\n",
      "Seq is: TTTAACTACCG\n",
      "0 TTTAACTACCG motifs detected in file\n",
      "\n",
      "Seq is: GGGATTCTATC\n",
      "0 GGGATTCTATC motifs detected in file\n",
      "\n",
      "Seq is: TCCTCTGTGGA\n",
      "0 TCCTCTGTGGA motifs detected in file\n",
      "\n",
      "Seq is: CGATCCACATT\n",
      "0 CGATCCACATT motifs detected in file\n",
      "\n",
      "Seq is: CCTCTTCTACC\n",
      "0 CCTCTTCTACC motifs detected in file\n",
      "\n",
      "Seq is: AATCCAGATGC\n",
      "0 AATCCAGATGC motifs detected in file\n",
      "\n",
      "Seq is: CTCCGAGGTCC\n",
      "0 CTCCGAGGTCC motifs detected in file\n",
      "\n",
      "Seq is: CACCCTCATCA\n",
      "0 CACCCTCATCA motifs detected in file\n",
      "\n",
      "Seq is: AGGGACGATTC\n",
      "0 AGGGACGATTC motifs detected in file\n",
      "\n",
      "Seq is: CCTCCCATCTG\n",
      "0 CCTCCCATCTG motifs detected in file\n",
      "\n",
      "Seq is: ACAAGTTGGAC\n",
      "0 ACAAGTTGGAC motifs detected in file\n",
      "\n",
      "Seq is: AAGGACGAGGC\n",
      "0 AAGGACGAGGC motifs detected in file\n",
      "\n",
      "Seq is: CGGAGGTGATA\n",
      "0 CGGAGGTGATA motifs detected in file\n",
      "\n",
      "Seq is: TCGTCTACGCT\n",
      "0 TCGTCTACGCT motifs detected in file\n",
      "\n",
      "Seq is: CGTCCTCATCC\n",
      "0 CGTCCTCATCC motifs detected in file\n",
      "\n",
      "Seq is: TCTCTCCCTCC\n",
      "0 TCTCTCCCTCC motifs detected in file\n",
      "\n",
      "Seq is: ATTCTCAGTCT\n",
      "0 ATTCTCAGTCT motifs detected in file\n",
      "\n",
      "Seq is: TCGAATACAAC\n",
      "0 TCGAATACAAC motifs detected in file\n",
      "\n",
      "Seq is: AAATACGCCCC\n",
      "0 AAATACGCCCC motifs detected in file\n",
      "\n",
      "Seq is: TTTGCGGTATA\n",
      "0 TTTGCGGTATA motifs detected in file\n",
      "\n",
      "Seq is: GCCCCCGTGGC\n",
      "0 GCCCCCGTGGC motifs detected in file\n",
      "\n",
      "Seq is: TAATAATCCCC\n",
      "0 TAATAATCCCC motifs detected in file\n",
      "\n",
      "Seq is: TTGATCAGTCA\n",
      "0 TTGATCAGTCA motifs detected in file\n",
      "\n",
      "Seq is: ACGGTTTCGGA\n",
      "0 ACGGTTTCGGA motifs detected in file\n",
      "\n",
      "Seq is: GGCTAATACTA\n",
      "0 GGCTAATACTA motifs detected in file\n",
      "\n",
      "Seq is: CCCATAATGAT\n",
      "0 CCCATAATGAT motifs detected in file\n",
      "\n",
      "Seq is: ACTTCTTCTAA\n",
      "0 ACTTCTTCTAA motifs detected in file\n",
      "\n",
      "Seq is: TAATGTTTACG\n",
      "0 TAATGTTTACG motifs detected in file\n",
      "\n",
      "Seq is: TACCCGACACT\n",
      "0 TACCCGACACT motifs detected in file\n",
      "\n",
      "Seq is: GCTATTGCAAC\n",
      "0 GCTATTGCAAC motifs detected in file\n",
      "\n",
      "Seq is: ATCTACACCAG\n",
      "0 ATCTACACCAG motifs detected in file\n",
      "\n",
      "Seq is: CAATGGATCTT\n",
      "0 CAATGGATCTT motifs detected in file\n",
      "\n",
      "Seq is: CCAACGGACCG\n",
      "0 CCAACGGACCG motifs detected in file\n",
      "\n",
      "Seq is: ACCGGGTCGAG\n",
      "0 ACCGGGTCGAG motifs detected in file\n",
      "\n",
      "Seq is: CCGAGCTTATT\n",
      "0 CCGAGCTTATT motifs detected in file\n",
      "\n",
      "Seq is: CCTCCGAATCT\n",
      "0 CCTCCGAATCT motifs detected in file\n",
      "\n",
      "Seq is: CGACACGGATC\n",
      "0 CGACACGGATC motifs detected in file\n",
      "\n",
      "Seq is: CTGAGGTCGAG\n",
      "0 CTGAGGTCGAG motifs detected in file\n",
      "\n",
      "Seq is: TACGCGGCTTA\n",
      "0 TACGCGGCTTA motifs detected in file\n",
      "\n",
      "Seq is: TTATCCATATC\n",
      "0 TTATCCATATC motifs detected in file\n",
      "\n",
      "Seq is: ACAAGGTTACA\n",
      "0 ACAAGGTTACA motifs detected in file\n",
      "\n",
      "Seq is: GAAACACCAAA\n",
      "0 GAAACACCAAA motifs detected in file\n",
      "\n",
      "Seq is: CATCTCTTATC\n",
      "0 CATCTCTTATC motifs detected in file\n",
      "\n",
      "Seq is: AGTTGCCAGCC\n",
      "0 AGTTGCCAGCC motifs detected in file\n",
      "\n",
      "Seq is: GCTTGTAGTCA\n",
      "0 GCTTGTAGTCA motifs detected in file\n",
      "\n",
      "Seq is: CCCCCACTCCA\n",
      "0 CCCCCACTCCA motifs detected in file\n",
      "\n",
      "Seq is: TTTTACCGACT\n",
      "0 TTTTACCGACT motifs detected in file\n",
      "\n",
      "Seq is: CACTTCGTAAC\n",
      "0 CACTTCGTAAC motifs detected in file\n",
      "\n",
      "Seq is: CTGACATTTAG\n",
      "0 CTGACATTTAG motifs detected in file\n",
      "\n",
      "Seq is: ATTTCTGTCCC\n",
      "0 ATTTCTGTCCC motifs detected in file\n",
      "\n",
      "Seq is: CAATCCGGAGA\n",
      "0 CAATCCGGAGA motifs detected in file\n",
      "\n",
      "Seq is: AAAATGGTCGA\n",
      "0 AAAATGGTCGA motifs detected in file\n",
      "\n",
      "Seq is: GGCTCCACTAA\n",
      "0 GGCTCCACTAA motifs detected in file\n",
      "\n",
      "Seq is: AAGTATAACTT\n",
      "0 AAGTATAACTT motifs detected in file\n",
      "\n",
      "Seq is: AACGTTTAAGC\n",
      "0 AACGTTTAAGC motifs detected in file\n",
      "\n",
      "Seq is: TTCTTCGTCGA\n",
      "0 TTCTTCGTCGA motifs detected in file\n",
      "\n",
      "Seq is: AGTTTGGACGG\n",
      "0 AGTTTGGACGG motifs detected in file\n",
      "\n",
      "Seq is: CCCCGAAGAGG\n",
      "0 CCCCGAAGAGG motifs detected in file\n",
      "\n",
      "Seq is: GCGGAAAAAAG\n",
      "0 GCGGAAAAAAG motifs detected in file\n",
      "\n",
      "Seq is: GGCCGCCGCCC\n",
      "0 GGCCGCCGCCC motifs detected in file\n",
      "\n",
      "Seq is: TCTTCATCTAA\n",
      "0 TCTTCATCTAA motifs detected in file\n",
      "\n",
      "Seq is: CTTCGGTCAAC\n",
      "0 CTTCGGTCAAC motifs detected in file\n",
      "\n",
      "Seq is: TAATCCCACGA\n",
      "0 TAATCCCACGA motifs detected in file\n",
      "\n",
      "Seq is: ATCGACAATTG\n",
      "0 ATCGACAATTG motifs detected in file\n",
      "\n",
      "Seq is: ATTCACAAACA\n",
      "0 ATTCACAAACA motifs detected in file\n",
      "\n",
      "Seq is: CCCAAATTCAG\n",
      "0 CCCAAATTCAG motifs detected in file\n",
      "\n",
      "Seq is: GGTAACCAGAT\n",
      "0 GGTAACCAGAT motifs detected in file\n",
      "\n",
      "Seq is: CATTCCCGAAT\n",
      "0 CATTCCCGAAT motifs detected in file\n",
      "\n",
      "Seq is: CGAATTAATTT\n",
      "0 CGAATTAATTT motifs detected in file\n",
      "\n",
      "Seq is: CACCGACTAAA\n",
      "0 CACCGACTAAA motifs detected in file\n",
      "\n",
      "Seq is: CGCAAGTCAAC\n",
      "0 CGCAAGTCAAC motifs detected in file\n",
      "\n",
      "Seq is: TACGTCTCACC\n",
      "0 TACGTCTCACC motifs detected in file\n",
      "\n",
      "Seq is: CCAAAACGTCA\n",
      "0 CCAAAACGTCA motifs detected in file\n",
      "\n",
      "Seq is: GGAATCGACAA\n",
      "0 GGAATCGACAA motifs detected in file\n",
      "\n",
      "Seq is: TGTCTTTAATT\n",
      "0 TGTCTTTAATT motifs detected in file\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Seq is: CATAACGTTGA\n",
      "0 CATAACGTTGA motifs detected in file\n",
      "\n",
      "Seq is: ATGACTCCCGA\n",
      "0 ATGACTCCCGA motifs detected in file\n",
      "\n",
      "Seq is: AACTTCCGAGA\n",
      "0 AACTTCCGAGA motifs detected in file\n",
      "\n",
      "Seq is: ACCAGACATAA\n",
      "0 ACCAGACATAA motifs detected in file\n",
      "\n",
      "Seq is: ATTGGATTTAA\n",
      "0 ATTGGATTTAA motifs detected in file\n",
      "\n",
      "Seq is: AGATATTCTAA\n",
      "0 AGATATTCTAA motifs detected in file\n",
      "\n",
      "Complement found from 10453 to 10464\n",
      "Complement found from 11102 to 11113\n",
      "Seq is: TAATCATATTT\n",
      "2 TAATCATATTT motifs detected in file\n",
      "\n",
      "Seq is: TCCCCTCTATC\n",
      "0 TCCCCTCTATC motifs detected in file\n",
      "\n",
      "Seq is: CATCCTCATCG\n",
      "0 CATCCTCATCG motifs detected in file\n",
      "\n",
      "Seq is: CACCATTCCCG\n",
      "0 CACCATTCCCG motifs detected in file\n",
      "\n",
      "Seq is: CTACTCACACC\n",
      "0 CTACTCACACC motifs detected in file\n",
      "\n",
      "Seq is: CCTCCTTACCC\n",
      "0 CCTCCTTACCC motifs detected in file\n",
      "\n",
      "Seq is: CACCCAAAACA\n",
      "0 CACCCAAAACA motifs detected in file\n",
      "\n",
      "Seq is: TACAAGTTTGA\n",
      "0 TACAAGTTTGA motifs detected in file\n",
      "\n",
      "Seq is: CAGTAAAATAA\n",
      "0 CAGTAAAATAA motifs detected in file\n",
      "\n",
      "Seq is: AAATGCAACAA\n",
      "0 AAATGCAACAA motifs detected in file\n",
      "\n",
      "Seq is: TCTATACCCCT\n",
      "0 TCTATACCCCT motifs detected in file\n",
      "\n",
      "Seq is: CATCACACTAA\n",
      "0 CATCACACTAA motifs detected in file\n",
      "\n",
      "Seq is: CTCCACCTCAT\n",
      "0 CTCCACCTCAT motifs detected in file\n",
      "\n",
      "Seq is: CTAATCCGCAT\n",
      "0 CTAATCCGCAT motifs detected in file\n",
      "\n",
      "Seq is: CCATCTTCATC\n",
      "0 CCATCTTCATC motifs detected in file\n",
      "\n",
      "Seq is: TCCAATTCCTC\n",
      "0 TCCAATTCCTC motifs detected in file\n",
      "\n",
      "Seq is: CCACTACCACC\n",
      "0 CCACTACCACC motifs detected in file\n",
      "\n",
      "Seq is: GATACTACCAC\n",
      "0 GATACTACCAC motifs detected in file\n",
      "\n",
      "Seq is: CCCTACTACTC\n",
      "0 CCCTACTACTC motifs detected in file\n",
      "\n",
      "Seq is: CGATAACAAAA\n",
      "0 CGATAACAAAA motifs detected in file\n",
      "\n",
      "Seq is: AACACTTAAGA\n",
      "0 AACACTTAAGA motifs detected in file\n",
      "\n",
      "Seq is: AGCTATTACCG\n",
      "0 AGCTATTACCG motifs detected in file\n",
      "\n",
      "Seq is: GGTAAACCCGT\n",
      "0 GGTAAACCCGT motifs detected in file\n",
      "\n",
      "Seq is: TTTTCGGCCAA\n",
      "0 TTTTCGGCCAA motifs detected in file\n",
      "\n",
      "Seq is: TCGCCCCCGTC\n",
      "0 TCGCCCCCGTC motifs detected in file\n",
      "\n",
      "Seq is: CGGAGGATCCC\n",
      "0 CGGAGGATCCC motifs detected in file\n",
      "\n",
      "Seq is: TCTCCTCCCAC\n",
      "0 TCTCCTCCCAC motifs detected in file\n",
      "\n",
      "Seq is: CTACCTTAATT\n",
      "0 CTACCTTAATT motifs detected in file\n",
      "\n",
      "Seq is: CCCACAATCAG\n",
      "0 CCCACAATCAG motifs detected in file\n",
      "\n",
      "Seq is: TACAATCGAAC\n",
      "0 TACAATCGAAC motifs detected in file\n",
      "\n",
      "Seq is: AAAGTCCACGC\n",
      "0 AAAGTCCACGC motifs detected in file\n",
      "\n",
      "Seq is: TCTATCATCAT\n",
      "0 TCTATCATCAT motifs detected in file\n",
      "\n",
      "Seq is: CCCAGCACCAC\n",
      "0 CCCAGCACCAC motifs detected in file\n",
      "\n",
      "Seq is: GACCTCAAATT\n",
      "0 GACCTCAAATT motifs detected in file\n",
      "\n",
      "Seq is: CAACTCATCAT\n",
      "0 CAACTCATCAT motifs detected in file\n",
      "\n",
      "Seq is: CCTTACGCCAT\n",
      "0 CCTTACGCCAT motifs detected in file\n",
      "\n",
      "Seq is: CATCAATCCTA\n",
      "0 CATCAATCCTA motifs detected in file\n",
      "\n",
      "Seq is: TTATATTTATC\n",
      "0 TTATATTTATC motifs detected in file\n",
      "\n",
      "Seq is: AATTTAATTCT\n",
      "0 AATTTAATTCT motifs detected in file\n",
      "\n",
      "Seq is: TACCAATACAA\n",
      "0 TACCAATACAA motifs detected in file\n",
      "\n",
      "Seq is: TCCCAACATGC\n",
      "0 TCCCAACATGC motifs detected in file\n",
      "\n",
      "Seq is: CATCTTGACGA\n",
      "0 CATCTTGACGA motifs detected in file\n",
      "\n",
      "Seq is: TAATAAGTAGG\n",
      "0 TAATAAGTAGG motifs detected in file\n",
      "\n",
      "Seq is: ATACACCCATT\n",
      "0 ATACACCCATT motifs detected in file\n",
      "\n",
      "Seq is: AACTCCTCATA\n",
      "0 AACTCCTCATA motifs detected in file\n",
      "\n",
      "Seq is: CGATTCTAAAA\n",
      "0 CGATTCTAAAA motifs detected in file\n",
      "\n",
      "Seq is: CGCATCGACCC\n",
      "0 CGCATCGACCC motifs detected in file\n",
      "\n",
      "Seq is: AAACCAAATTA\n",
      "0 AAACCAAATTA motifs detected in file\n",
      "\n",
      "Seq is: GGTGGAGTTGA\n",
      "0 GGTGGAGTTGA motifs detected in file\n",
      "\n",
      "Seq is: CGGACGATACT\n",
      "0 CGGACGATACT motifs detected in file\n",
      "\n",
      "Seq is: ACCTATTCTAA\n",
      "0 ACCTATTCTAA motifs detected in file\n",
      "\n",
      "Seq is: CTCTCTCACTC\n",
      "0 CTCTCTCACTC motifs detected in file\n",
      "\n",
      "Seq is: CTCTTCCGAAT\n",
      "0 CTCTTCCGAAT motifs detected in file\n",
      "\n",
      "Seq is: GCAAATCACTC\n",
      "0 GCAAATCACTC motifs detected in file\n",
      "\n",
      "Seq is: CCTCTCTAAAC\n",
      "0 CCTCTCTAAAC motifs detected in file\n",
      "\n",
      "Seq is: CATATACTAAC\n",
      "0 CATATACTAAC motifs detected in file\n",
      "\n",
      "Seq is: TCTACCCCCGA\n",
      "0 TCTACCCCCGA motifs detected in file\n",
      "\n",
      "Seq is: TCAAAAACAGT\n",
      "0 TCAAAAACAGT motifs detected in file\n",
      "\n",
      "Seq is: ACACTCTTCTT\n",
      "0 ACACTCTTCTT motifs detected in file\n",
      "\n",
      "Seq is: CGTCCGGCCTA\n",
      "0 CGTCCGGCCTA motifs detected in file\n",
      "\n",
      "Seq is: CAGTCTCCCCA\n",
      "0 CAGTCTCCCCA motifs detected in file\n",
      "\n",
      "Seq is: CGGAACCCATT\n",
      "0 CGGAACCCATT motifs detected in file\n",
      "\n",
      "Seq is: GGAGACCCTGA\n",
      "0 GGAGACCCTGA motifs detected in file\n",
      "\n",
      "Seq is: GTCTTCACTTT\n",
      "0 GTCTTCACTTT motifs detected in file\n",
      "\n",
      "Seq is: CCCCCGATAAG\n",
      "0 CCCCCGATAAG motifs detected in file\n",
      "\n",
      "Seq is: GATCAAAATAA\n",
      "0 GATCAAAATAA motifs detected in file\n",
      "\n",
      "Seq is: CGATATCGATA\n",
      "0 CGATATCGATA motifs detected in file\n",
      "\n",
      "Complement found from 5495 to 5506\n",
      "Seq is: ATACTAATAAT\n",
      "1 ATACTAATAAT motifs detected in file\n",
      "\n",
      "Seq is: TACTACTCATA\n",
      "0 TACTACTCATA motifs detected in file\n",
      "\n",
      "Seq is: ACTAACCATCA\n",
      "0 ACTAACCATCA motifs detected in file\n",
      "\n",
      "Seq is: TAACCAATACC\n",
      "0 TAACCAATACC motifs detected in file\n",
      "\n",
      "Seq is: AAGTAACAGGC\n",
      "0 AAGTAACAGGC motifs detected in file\n",
      "\n",
      "Seq is: CTCTCATATAA\n",
      "0 CTCTCATATAA motifs detected in file\n",
      "\n",
      "Seq is: CAACTTCTCCT\n",
      "0 CAACTTCTCCT motifs detected in file\n",
      "\n",
      "Seq is: ATCGATAATCT\n",
      "0 ATCGATAATCT motifs detected in file\n",
      "\n",
      "Seq is: TCCTAATACCT\n",
      "0 TCCTAATACCT motifs detected in file\n",
      "\n",
      "Seq is: ACGCCAACGAA\n",
      "0 ACGCCAACGAA motifs detected in file\n",
      "\n",
      "Seq is: CGCACTCCTTT\n",
      "0 CGCACTCCTTT motifs detected in file\n",
      "\n",
      "Seq is: ATGAACTACCG\n",
      "0 ATGAACTACCG motifs detected in file\n",
      "\n",
      "Seq is: TCGAAGACACC\n",
      "0 TCGAAGACACC motifs detected in file\n",
      "\n",
      "Seq is: TTGCTCCCAAA\n",
      "0 TTGCTCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: TAAAAAAACCA\n",
      "0 TAAAAAAACCA motifs detected in file\n",
      "\n",
      "Seq is: ATCTTGACCTT\n",
      "0 ATCTTGACCTT motifs detected in file\n",
      "\n",
      "Seq is: ATTTTCGATCG\n",
      "0 ATTTTCGATCG motifs detected in file\n",
      "\n",
      "Seq is: TACAAATAAAG\n",
      "0 TACAAATAAAG motifs detected in file\n",
      "\n",
      "Seq is: ATCCGGATGAG\n",
      "0 ATCCGGATGAG motifs detected in file\n",
      "\n",
      "Seq is: TCCATTTTTTA\n",
      "0 TCCATTTTTTA motifs detected in file\n",
      "\n",
      "Seq is: GTCACGCTCGA\n",
      "0 GTCACGCTCGA motifs detected in file\n",
      "\n",
      "Seq is: ATCGCGACACT\n",
      "0 ATCGCGACACT motifs detected in file\n",
      "\n",
      "Seq is: ACTCACACGGA\n",
      "0 ACTCACACGGA motifs detected in file\n",
      "\n",
      "Seq is: CGTTTCTACCA\n",
      "0 CGTTTCTACCA motifs detected in file\n",
      "\n",
      "Seq is: TCTCATCTACT\n",
      "0 TCTCATCTACT motifs detected in file\n",
      "\n",
      "Seq is: GCCCAACCCGG\n",
      "0 GCCCAACCCGG motifs detected in file\n",
      "\n",
      "Seq is: TCCCCTAATTA\n",
      "0 TCCCCTAATTA motifs detected in file\n",
      "\n",
      "Seq is: ATCATGCCCTT\n",
      "0 ATCATGCCCTT motifs detected in file\n",
      "\n",
      "Seq is: CCCATATTGGT\n",
      "0 CCCATATTGGT motifs detected in file\n",
      "\n",
      "Seq is: TGTAAAAGCCC\n",
      "0 TGTAAAAGCCC motifs detected in file\n",
      "\n",
      "Seq is: CATACCCGGGC\n",
      "0 CATACCCGGGC motifs detected in file\n",
      "\n",
      "Seq is: TATCGAATAAA\n",
      "0 TATCGAATAAA motifs detected in file\n",
      "\n",
      "Seq is: TCGACTGGAAT\n",
      "0 TCGACTGGAAT motifs detected in file\n",
      "\n",
      "Seq is: GAAATCCTACC\n",
      "0 GAAATCCTACC motifs detected in file\n",
      "\n",
      "Seq is: CCACACTATCC\n",
      "0 CCACACTATCC motifs detected in file\n",
      "\n",
      "Seq is: ACCGTGCCTCT\n",
      "0 ACCGTGCCTCT motifs detected in file\n",
      "\n",
      "Seq is: TAAAACCTAAG\n",
      "0 TAAAACCTAAG motifs detected in file\n",
      "\n",
      "Seq is: AGTCCCTACCC\n",
      "0 AGTCCCTACCC motifs detected in file\n",
      "\n",
      "Seq is: AAGCTAAGAGT\n",
      "0 AAGCTAAGAGT motifs detected in file\n",
      "\n",
      "Seq is: ATCAGGATCTT\n",
      "0 ATCAGGATCTT motifs detected in file\n",
      "\n",
      "Seq is: TATTCCCCCAA\n",
      "0 TATTCCCCCAA motifs detected in file\n",
      "\n",
      "Seq is: ATTCGAGGATA\n",
      "0 ATTCGAGGATA motifs detected in file\n",
      "\n",
      "Seq is: ATAAATGAGAT\n",
      "0 ATAAATGAGAT motifs detected in file\n",
      "\n",
      "Seq is: AGTTTCATTGA\n",
      "0 AGTTTCATTGA motifs detected in file\n",
      "\n",
      "Seq is: GAAAATAGTCT\n",
      "0 GAAAATAGTCT motifs detected in file\n",
      "\n",
      "Seq is: GTATAAAGAAT\n",
      "0 GTATAAAGAAT motifs detected in file\n",
      "\n",
      "Seq is: CCAAACTCCCC\n",
      "0 CCAAACTCCCC motifs detected in file\n",
      "\n",
      "Seq is: CTTACGACCTC\n",
      "0 CTTACGACCTC motifs detected in file\n",
      "\n",
      "Seq is: TAACATTACCC\n",
      "0 TAACATTACCC motifs detected in file\n",
      "\n",
      "Seq is: ATACCTCTGTA\n",
      "0 ATACCTCTGTA motifs detected in file\n",
      "\n",
      "Seq is: TAGTATATTCA\n",
      "0 TAGTATATTCA motifs detected in file\n",
      "\n",
      "Seq is: TTACGATCCCA\n",
      "0 TTACGATCCCA motifs detected in file\n",
      "\n",
      "Seq is: CTCACCATCCT\n",
      "0 CTCACCATCCT motifs detected in file\n",
      "\n",
      "Seq is: TCAAAAAAGTA\n",
      "0 TCAAAAAAGTA motifs detected in file\n",
      "\n",
      "Seq is: TCCTCCACATA\n",
      "0 TCCTCCACATA motifs detected in file\n",
      "\n",
      "Seq is: CTCAACCAGCA\n",
      "0 CTCAACCAGCA motifs detected in file\n",
      "\n",
      "Seq is: TCGCCTTAGCC\n",
      "0 TCGCCTTAGCC motifs detected in file\n",
      "\n",
      "Seq is: CCCATACGACA\n",
      "0 CCCATACGACA motifs detected in file\n",
      "\n",
      "Seq is: AGCTTAAGTAT\n",
      "0 AGCTTAAGTAT motifs detected in file\n",
      "\n",
      "Seq is: TCTTGTCCCTC\n",
      "0 TCTTGTCCCTC motifs detected in file\n",
      "\n",
      "Seq is: CAATCTTCATC\n",
      "0 CAATCTTCATC motifs detected in file\n",
      "\n",
      "Seq is: CCAGAACCACT\n",
      "0 CCAGAACCACT motifs detected in file\n",
      "\n",
      "Seq is: GTTTTATACAA\n",
      "0 GTTTTATACAA motifs detected in file\n",
      "\n",
      "Seq is: CACATCTCAAG\n",
      "0 CACATCTCAAG motifs detected in file\n",
      "\n",
      "Seq is: TCCCCTCTCAC\n",
      "0 TCCCCTCTCAC motifs detected in file\n",
      "\n",
      "Seq is: GCAGTATACAA\n",
      "0 GCAGTATACAA motifs detected in file\n",
      "\n",
      "Seq is: CAAGGATCCTT\n",
      "0 CAAGGATCCTT motifs detected in file\n",
      "\n",
      "Seq is: CTAACATCACC\n",
      "0 CTAACATCACC motifs detected in file\n",
      "\n",
      "Seq is: ACTCCCACAAA\n",
      "0 ACTCCCACAAA motifs detected in file\n",
      "\n",
      "Seq is: TAATATTATTA\n",
      "0 TAATATTATTA motifs detected in file\n",
      "\n",
      "Seq is: CAAACACATAA\n",
      "0 CAAACACATAA motifs detected in file\n",
      "\n",
      "Seq is: GCCGATACTTC\n",
      "0 GCCGATACTTC motifs detected in file\n",
      "\n",
      "Seq is: TTATCCCGCTT\n",
      "0 TTATCCCGCTT motifs detected in file\n",
      "\n",
      "Seq is: CCCCGGACGCC\n",
      "0 CCCCGGACGCC motifs detected in file\n",
      "\n",
      "Seq is: GCATAAGCTAC\n",
      "0 GCATAAGCTAC motifs detected in file\n",
      "\n",
      "Seq is: AACTTCGGACT\n",
      "0 AACTTCGGACT motifs detected in file\n",
      "\n",
      "Seq is: CTGATCAAGCC\n",
      "0 CTGATCAAGCC motifs detected in file\n",
      "\n",
      "Seq is: TGAGGGGAAGC\n",
      "0 TGAGGGGAAGC motifs detected in file\n",
      "\n",
      "Seq is: CGTTCCAGCTT\n",
      "0 CGTTCCAGCTT motifs detected in file\n",
      "\n",
      "Seq is: CCCCCAAGCCA\n",
      "0 CCCCCAAGCCA motifs detected in file\n",
      "\n",
      "Seq is: ACCAGAGACGA\n",
      "0 ACCAGAGACGA motifs detected in file\n",
      "\n",
      "Seq is: TCACACCTCTA\n",
      "0 TCACACCTCTA motifs detected in file\n",
      "\n",
      "Seq is: TTTAGTATAAT\n",
      "0 TTTAGTATAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCGGTTCCCA\n",
      "0 ACCGGTTCCCA motifs detected in file\n",
      "\n",
      "Seq is: GTACTACCGTC\n",
      "0 GTACTACCGTC motifs detected in file\n",
      "\n",
      "Seq is: CTCATTAGTCT\n",
      "0 CTCATTAGTCT motifs detected in file\n",
      "\n",
      "Seq is: CCACAAGAACA\n",
      "0 CCACAAGAACA motifs detected in file\n",
      "\n",
      "Seq is: CAACACTATTC\n",
      "0 CAACACTATTC motifs detected in file\n",
      "\n",
      "Seq is: CCACCTCTCCA\n",
      "0 CCACCTCTCCA motifs detected in file\n",
      "\n",
      "Seq is: ATTTCCTCGGT\n",
      "0 ATTTCCTCGGT motifs detected in file\n",
      "\n",
      "Seq is: GAATAATCATT\n",
      "0 GAATAATCATT motifs detected in file\n",
      "\n",
      "Seq is: ACAACTATCAT\n",
      "0 ACAACTATCAT motifs detected in file\n",
      "\n",
      "Seq is: CTTACTACCGA\n",
      "0 CTTACTACCGA motifs detected in file\n",
      "\n",
      "Seq is: TCCCACTGAAG\n",
      "0 TCCCACTGAAG motifs detected in file\n",
      "\n",
      "Seq is: TATACTCTAAC\n",
      "0 TATACTCTAAC motifs detected in file\n",
      "\n",
      "Seq is: AAACCCGATGA\n",
      "0 AAACCCGATGA motifs detected in file\n",
      "\n",
      "Seq is: CGAGCGTCACG\n",
      "0 CGAGCGTCACG motifs detected in file\n",
      "\n",
      "Seq is: CGGCTAGTCCC\n",
      "0 CGGCTAGTCCC motifs detected in file\n",
      "\n",
      "Complement found from 3669 to 3680\n",
      "Seq is: GCATCAAACTC\n",
      "1 GCATCAAACTC motifs detected in file\n",
      "\n",
      "Seq is: AAACTACGAGT\n",
      "0 AAACTACGAGT motifs detected in file\n",
      "\n",
      "Seq is: GGGACTAGTCT\n",
      "0 GGGACTAGTCT motifs detected in file\n",
      "\n",
      "Seq is: CCTAACTCATT\n",
      "0 CCTAACTCATT motifs detected in file\n",
      "\n",
      "Seq is: TGCCGATCCGA\n",
      "0 TGCCGATCCGA motifs detected in file\n",
      "\n",
      "Seq is: TCTCCACCGAT\n",
      "0 TCTCCACCGAT motifs detected in file\n",
      "\n",
      "Seq is: CTTATTTATCC\n",
      "0 CTTATTTATCC motifs detected in file\n",
      "\n",
      "Seq is: TCCGGATCCAA\n",
      "0 TCCGGATCCAA motifs detected in file\n",
      "\n",
      "Seq is: CTCCAACTGGT\n",
      "0 CTCCAACTGGT motifs detected in file\n",
      "\n",
      "Seq is: CCCCCAACCCA\n",
      "0 CCCCCAACCCA motifs detected in file\n",
      "\n",
      "Seq is: TACCCCTCCCC\n",
      "0 TACCCCTCCCC motifs detected in file\n",
      "\n",
      "Seq is: CCAAGTATCAT\n",
      "0 CCAAGTATCAT motifs detected in file\n",
      "\n",
      "Seq is: CTTCTCGCTAC\n",
      "0 CTTCTCGCTAC motifs detected in file\n",
      "\n",
      "Seq is: CACTCTCGATT\n",
      "0 CACTCTCGATT motifs detected in file\n",
      "\n",
      "Seq is: CCAGCCCCGCC\n",
      "0 CCAGCCCCGCC motifs detected in file\n",
      "\n",
      "Seq is: ACTACATCTCC\n",
      "0 ACTACATCTCC motifs detected in file\n",
      "\n",
      "Seq is: CACTACCATCT\n",
      "0 CACTACCATCT motifs detected in file\n",
      "\n",
      "Seq is: ACACCGCCCAA\n",
      "0 ACACCGCCCAA motifs detected in file\n",
      "\n",
      "Seq is: AATCCCCGAGA\n",
      "0 AATCCCCGAGA motifs detected in file\n",
      "\n",
      "Seq is: AACCACTTCTC\n",
      "0 AACCACTTCTC motifs detected in file\n",
      "\n",
      "Seq is: AAAATACCGCA\n",
      "0 AAAATACCGCA motifs detected in file\n",
      "\n",
      "Seq is: GTCGCTTCCCA\n",
      "0 GTCGCTTCCCA motifs detected in file\n",
      "\n",
      "Seq is: ACATCATCGGG\n",
      "0 ACATCATCGGG motifs detected in file\n",
      "\n",
      "Seq is: CATCCCCGGAT\n",
      "0 CATCCCCGGAT motifs detected in file\n",
      "\n",
      "Seq is: GTTGCAACCCC\n",
      "0 GTTGCAACCCC motifs detected in file\n",
      "\n",
      "Seq is: GGAAACGCATC\n",
      "0 GGAAACGCATC motifs detected in file\n",
      "\n",
      "Seq is: AACATATATCG\n",
      "0 AACATATATCG motifs detected in file\n",
      "\n",
      "Seq is: GATCTTAAAAA\n",
      "0 GATCTTAAAAA motifs detected in file\n",
      "\n",
      "Seq is: GCAAGCCATTC\n",
      "0 GCAAGCCATTC motifs detected in file\n",
      "\n",
      "Seq is: GTAATCCTTAC\n",
      "0 GTAATCCTTAC motifs detected in file\n",
      "\n",
      "Seq is: GGTAACGCTAA\n",
      "0 GGTAACGCTAA motifs detected in file\n",
      "\n",
      "Seq is: TCTTACCCATG\n",
      "0 TCTTACCCATG motifs detected in file\n",
      "\n",
      "Seq is: TTACTCCTCAT\n",
      "0 TTACTCCTCAT motifs detected in file\n",
      "\n",
      "Seq is: CCTCCAACCGG\n",
      "0 CCTCCAACCGG motifs detected in file\n",
      "\n",
      "Seq is: TACCCATACAA\n",
      "0 TACCCATACAA motifs detected in file\n",
      "\n",
      "Seq is: CAATTCTTCTC\n",
      "0 CAATTCTTCTC motifs detected in file\n",
      "\n",
      "Seq is: CTTAACTTGGA\n",
      "0 CTTAACTTGGA motifs detected in file\n",
      "\n",
      "Seq is: GACTGACATTT\n",
      "0 GACTGACATTT motifs detected in file\n",
      "\n",
      "Seq is: CAAAATTCAAA\n",
      "0 CAAAATTCAAA motifs detected in file\n",
      "\n",
      "Seq is: ATACGCTAATG\n",
      "0 ATACGCTAATG motifs detected in file\n",
      "\n",
      "Seq is: GCCCGAGACGG\n",
      "0 GCCCGAGACGG motifs detected in file\n",
      "\n",
      "Seq is: TAGAATTGTTT\n",
      "0 TAGAATTGTTT motifs detected in file\n",
      "\n",
      "Seq is: GGGACAAGAAC\n",
      "0 GGGACAAGAAC motifs detected in file\n",
      "\n",
      "Seq is: CCACCCACACC\n",
      "0 CCACCCACACC motifs detected in file\n",
      "\n",
      "Seq is: CATATTATGAT\n",
      "0 CATATTATGAT motifs detected in file\n",
      "\n",
      "Seq is: TCAACTCTACT\n",
      "0 TCAACTCTACT motifs detected in file\n",
      "\n",
      "Seq is: ATAGTAAATGC\n",
      "0 ATAGTAAATGC motifs detected in file\n",
      "\n",
      "Seq is: CCCCTTCCGCG\n",
      "0 CCCCTTCCGCG motifs detected in file\n",
      "\n",
      "Seq is: AAACACTTCAT\n",
      "0 AAACACTTCAT motifs detected in file\n",
      "\n",
      "Seq is: CCGGAATAAAG\n",
      "0 CCGGAATAAAG motifs detected in file\n",
      "\n",
      "Seq is: AGAACAGGAAA\n",
      "0 AGAACAGGAAA motifs detected in file\n",
      "\n",
      "Seq is: GCATGTCCCTC\n",
      "0 GCATGTCCCTC motifs detected in file\n",
      "\n",
      "Seq is: CTTAAACTTNC\n",
      "0 CTTAAACTTNC motifs detected in file\n",
      "\n",
      "Seq is: ATCTATCTTTG\n",
      "0 ATCTATCTTTG motifs detected in file\n",
      "\n",
      "Seq is: GCTGGACCTAA\n",
      "0 GCTGGACCTAA motifs detected in file\n",
      "\n",
      "Seq is: TGAGGCCAGAC\n",
      "0 TGAGGCCAGAC motifs detected in file\n",
      "\n",
      "Seq is: TTGAGTCTAGT\n",
      "0 TTGAGTCTAGT motifs detected in file\n",
      "\n",
      "Seq is: GCATCCTGAAA\n",
      "0 GCATCCTGAAA motifs detected in file\n",
      "\n",
      "Seq is: TTAGCAACTTG\n",
      "0 TTAGCAACTTG motifs detected in file\n",
      "\n",
      "Seq is: TTTGCTTGGAA\n",
      "0 TTTGCTTGGAA motifs detected in file\n",
      "\n",
      "Seq is: ATTATCGCCGA\n",
      "0 ATTATCGCCGA motifs detected in file\n",
      "\n",
      "Seq is: CGTGGTAGCCC\n",
      "0 CGTGGTAGCCC motifs detected in file\n",
      "\n",
      "Seq is: TACAGGACTAG\n",
      "0 TACAGGACTAG motifs detected in file\n",
      "\n",
      "Seq is: GTTGTAGCTCC\n",
      "0 GTTGTAGCTCC motifs detected in file\n",
      "\n",
      "Seq is: AGCATTTGGGA\n",
      "0 AGCATTTGGGA motifs detected in file\n",
      "\n",
      "Seq is: TAACAACTATA\n",
      "0 TAACAACTATA motifs detected in file\n",
      "\n",
      "Seq is: CCTGAGATCTT\n",
      "0 CCTGAGATCTT motifs detected in file\n",
      "\n",
      "Seq is: ATCCTAACGCG\n",
      "0 ATCCTAACGCG motifs detected in file\n",
      "\n",
      "Seq is: ACAATAGGGAT\n",
      "0 ACAATAGGGAT motifs detected in file\n",
      "\n",
      "Seq is: CCCATTGAACA\n",
      "0 CCCATTGAACA motifs detected in file\n",
      "\n",
      "Seq is: AGGCAACCAGT\n",
      "0 AGGCAACCAGT motifs detected in file\n",
      "\n",
      "Seq is: TCAATAACCTA\n",
      "0 TCAATAACCTA motifs detected in file\n",
      "\n",
      "Seq is: GTTAACTCATA\n",
      "0 GTTAACTCATA motifs detected in file\n",
      "\n",
      "Seq is: TCATCAAGCGA\n",
      "0 TCATCAAGCGA motifs detected in file\n",
      "\n",
      "Seq is: AACTGACCACT\n",
      "0 AACTGACCACT motifs detected in file\n",
      "\n",
      "Seq is: TCAGAATCGTA\n",
      "0 TCAGAATCGTA motifs detected in file\n",
      "\n",
      "Seq is: CATGACGAGCC\n",
      "0 CATGACGAGCC motifs detected in file\n",
      "\n",
      "Seq is: TCCAACCCAAG\n",
      "0 TCCAACCCAAG motifs detected in file\n",
      "\n",
      "Seq is: ACGAGGCTCCA\n",
      "0 ACGAGGCTCCA motifs detected in file\n",
      "\n",
      "Seq is: GCGGGGTTGGC\n",
      "0 GCGGGGTTGGC motifs detected in file\n",
      "\n",
      "Seq is: TTTAAAAATTA\n",
      "0 TTTAAAAATTA motifs detected in file\n",
      "\n",
      "Seq is: CGTCCAAACCA\n",
      "0 CGTCCAAACCA motifs detected in file\n",
      "\n",
      "Seq is: TCAAATCCTGG\n",
      "0 TCAAATCCTGG motifs detected in file\n",
      "\n",
      "Seq is: ACACCCAAACA\n",
      "0 ACACCCAAACA motifs detected in file\n",
      "\n",
      "Seq is: ATCCATGACAA\n",
      "0 ATCCATGACAA motifs detected in file\n",
      "\n",
      "Seq is: ACGTAATTATT\n",
      "0 ACGTAATTATT motifs detected in file\n",
      "\n",
      "Seq is: TAATTTCGAGG\n",
      "0 TAATTTCGAGG motifs detected in file\n",
      "\n",
      "Seq is: TATCCCAGAAG\n",
      "0 TATCCCAGAAG motifs detected in file\n",
      "\n",
      "Seq is: AGCAGAACGAC\n",
      "0 AGCAGAACGAC motifs detected in file\n",
      "\n",
      "Seq is: ACAATACGGGC\n",
      "0 ACAATACGGGC motifs detected in file\n",
      "\n",
      "Seq is: GGAGAAGTGCC\n",
      "0 GGAGAAGTGCC motifs detected in file\n",
      "\n",
      "Seq is: CGTCCAGTTAA\n",
      "0 CGTCCAGTTAA motifs detected in file\n",
      "\n",
      "Seq is: AGTGACCAATT\n",
      "0 AGTGACCAATT motifs detected in file\n",
      "\n",
      "Seq is: TTCATTCTCTG\n",
      "0 TTCATTCTCTG motifs detected in file\n",
      "\n",
      "Seq is: TCGACTTGGGA\n",
      "0 TCGACTTGGGA motifs detected in file\n",
      "\n",
      "Seq is: GCACCTCGGTA\n",
      "0 GCACCTCGGTA motifs detected in file\n",
      "\n",
      "Seq is: AGTATGTCCAG\n",
      "0 AGTATGTCCAG motifs detected in file\n",
      "\n",
      "Seq is: GGATAAATTCC\n",
      "0 GGATAAATTCC motifs detected in file\n",
      "\n",
      "Seq is: TTGTTCACTAA\n",
      "0 TTGTTCACTAA motifs detected in file\n",
      "\n",
      "Seq is: TACGATGGAAA\n",
      "0 TACGATGGAAA motifs detected in file\n",
      "\n",
      "Seq is: CGTGCCAATCC\n",
      "0 CGTGCCAATCC motifs detected in file\n",
      "\n",
      "Seq is: CATGGCGCCGG\n",
      "0 CATGGCGCCGG motifs detected in file\n",
      "\n",
      "Seq is: CAATTTGTACA\n",
      "0 CAATTTGTACA motifs detected in file\n",
      "\n",
      "Seq is: CAGTGACCCGT\n",
      "0 CAGTGACCCGT motifs detected in file\n",
      "\n",
      "Seq is: CCGCCACGGAG\n",
      "0 CCGCCACGGAG motifs detected in file\n",
      "\n",
      "Seq is: ATTATGACCAC\n",
      "0 ATTATGACCAC motifs detected in file\n",
      "\n",
      "Seq is: TACGATCTCCA\n",
      "0 TACGATCTCCA motifs detected in file\n",
      "\n",
      "Seq is: CTACAAAAACC\n",
      "0 CTACAAAAACC motifs detected in file\n",
      "\n",
      "Seq is: ATTTGTCCGCC\n",
      "0 ATTTGTCCGCC motifs detected in file\n",
      "\n",
      "Seq is: CCATTCTAAAC\n",
      "0 CCATTCTAAAC motifs detected in file\n",
      "\n",
      "Seq is: GGCTCAAGGAA\n",
      "0 GGCTCAAGGAA motifs detected in file\n",
      "\n",
      "Seq is: AATGAAAAAAA\n",
      "0 AATGAAAAAAA motifs detected in file\n",
      "\n",
      "Seq is: TTGGAAAGGAA\n",
      "0 TTGGAAAGGAA motifs detected in file\n",
      "\n",
      "Seq is: TACTCGTACGG\n",
      "0 TACTCGTACGG motifs detected in file\n",
      "\n",
      "Seq is: ACACAACCCAA\n",
      "0 ACACAACCCAA motifs detected in file\n",
      "\n",
      "Seq is: CTGTCACTCCC\n",
      "0 CTGTCACTCCC motifs detected in file\n",
      "\n",
      "Seq is: ATTATTACTGA\n",
      "0 ATTATTACTGA motifs detected in file\n",
      "\n",
      "Seq is: ACAACCAACTA\n",
      "0 ACAACCAACTA motifs detected in file\n",
      "\n",
      "Seq is: ACATCTATAAC\n",
      "0 ACATCTATAAC motifs detected in file\n",
      "\n",
      "Seq is: CCGACAATTAA\n",
      "0 CCGACAATTAA motifs detected in file\n",
      "\n",
      "Seq is: CAGTCAAGTCA\n",
      "0 CAGTCAAGTCA motifs detected in file\n",
      "\n",
      "Seq is: CAAAATTAGAC\n",
      "0 CAAAATTAGAC motifs detected in file\n",
      "\n",
      "Seq is: TGCGTCCGAAT\n",
      "0 TGCGTCCGAAT motifs detected in file\n",
      "\n",
      "Seq is: ACGCCTCCTCT\n",
      "0 ACGCCTCCTCT motifs detected in file\n",
      "\n",
      "Seq is: TACAAAAGTAC\n",
      "0 TACAAAAGTAC motifs detected in file\n",
      "\n",
      "Seq is: AATGAATATGA\n",
      "0 AATGAATATGA motifs detected in file\n",
      "\n",
      "Seq is: TTGTAATCAAG\n",
      "0 TTGTAATCAAG motifs detected in file\n",
      "\n",
      "Seq is: AAGATATCCCA\n",
      "0 AAGATATCCCA motifs detected in file\n",
      "\n",
      "Seq is: CTATCTAACCA\n",
      "0 CTATCTAACCA motifs detected in file\n",
      "\n",
      "Seq is: GGTTAACCCAC\n",
      "0 GGTTAACCCAC motifs detected in file\n",
      "\n",
      "Seq is: ACTCCTCAAGT\n",
      "0 ACTCCTCAAGT motifs detected in file\n",
      "\n",
      "Seq is: CAATATACAAA\n",
      "0 CAATATACAAA motifs detected in file\n",
      "\n",
      "Seq is: CCCTAAAAAAT\n",
      "0 CCCTAAAAAAT motifs detected in file\n",
      "\n",
      "Seq is: CCATCACCCAC\n",
      "0 CCATCACCCAC motifs detected in file\n",
      "\n",
      "Seq is: AACTCGAACTT\n",
      "0 AACTCGAACTT motifs detected in file\n",
      "\n",
      "Seq is: GCGAAAGAATT\n",
      "0 GCGAAAGAATT motifs detected in file\n",
      "\n",
      "Seq is: AACCACCGACG\n",
      "0 AACCACCGACG motifs detected in file\n",
      "\n",
      "Seq is: AAAATCCGGAT\n",
      "0 AAAATCCGGAT motifs detected in file\n",
      "\n",
      "Seq is: GATACCCACAA\n",
      "0 GATACCCACAA motifs detected in file\n",
      "\n",
      "Seq is: TTTAAAAAATG\n",
      "0 TTTAAAAAATG motifs detected in file\n",
      "\n",
      "Seq is: AGAGAGATGTT\n",
      "0 AGAGAGATGTT motifs detected in file\n",
      "\n",
      "Seq is: CCAAAAAAGGA\n",
      "0 CCAAAAAAGGA motifs detected in file\n",
      "\n",
      "Complement found from 13492 to 13503\n",
      "Seq is: TCACAGGTTTC\n",
      "1 TCACAGGTTTC motifs detected in file\n",
      "\n",
      "Seq is: TCGACAAGGAG\n",
      "0 TCGACAAGGAG motifs detected in file\n",
      "\n",
      "Seq is: AAACCTGATTG\n",
      "0 AAACCTGATTG motifs detected in file\n",
      "\n",
      "Seq is: TCAATTTAAAT\n",
      "0 TCAATTTAAAT motifs detected in file\n",
      "\n",
      "Seq is: GTTCCCCTAAA\n",
      "0 GTTCCCCTAAA motifs detected in file\n",
      "\n",
      "Seq is: TCTCCCAAGAC\n",
      "0 TCTCCCAAGAC motifs detected in file\n",
      "\n",
      "Seq is: ACCCGTTTAAA\n",
      "0 ACCCGTTTAAA motifs detected in file\n",
      "\n",
      "Seq is: TTTCAACTTGA\n",
      "0 TTTCAACTTGA motifs detected in file\n",
      "\n",
      "Seq is: TTCTAAGATAG\n",
      "0 TTCTAAGATAG motifs detected in file\n",
      "\n",
      "Seq is: AACCTGTTGGT\n",
      "0 AACCTGTTGGT motifs detected in file\n",
      "\n",
      "Seq is: CGATAGTGGTC\n",
      "0 CGATAGTGGTC motifs detected in file\n",
      "\n",
      "Seq is: CGAGCCATCCA\n",
      "0 CGAGCCATCCA motifs detected in file\n",
      "\n",
      "Seq is: AACAGCGGAGA\n",
      "0 AACAGCGGAGA motifs detected in file\n",
      "\n",
      "Seq is: TGGATATTTAG\n",
      "0 TGGATATTTAG motifs detected in file\n",
      "\n",
      "Seq is: AAGGGTGATAA\n",
      "0 AAGGGTGATAA motifs detected in file\n",
      "\n",
      "Seq is: AACGATGTATC\n",
      "0 AACGATGTATC motifs detected in file\n",
      "\n",
      "Seq is: TGCCCACACGA\n",
      "0 TGCCCACACGA motifs detected in file\n",
      "\n",
      "Seq is: GAAAATCGACA\n",
      "0 GAAAATCGACA motifs detected in file\n",
      "\n",
      "Seq is: AGAATCCATCG\n",
      "0 AGAATCCATCG motifs detected in file\n",
      "\n",
      "Seq is: AGCAGACCAAA\n",
      "0 AGCAGACCAAA motifs detected in file\n",
      "\n",
      "Seq is: GCCCCCAGAAT\n",
      "0 GCCCCCAGAAT motifs detected in file\n",
      "\n",
      "Seq is: CGAAACCGAGA\n",
      "0 CGAAACCGAGA motifs detected in file\n",
      "\n",
      "Seq is: GGAACGTTTCA\n",
      "0 GGAACGTTTCA motifs detected in file\n",
      "\n",
      "Seq is: ATAAAGATCAA\n",
      "0 ATAAAGATCAA motifs detected in file\n",
      "\n",
      "Seq is: TTAAGTAATAC\n",
      "0 TTAAGTAATAC motifs detected in file\n",
      "\n",
      "Seq is: GTCTTCCATAT\n",
      "0 GTCTTCCATAT motifs detected in file\n",
      "\n",
      "Seq is: CCCCAATCAGG\n",
      "0 CCCCAATCAGG motifs detected in file\n",
      "\n",
      "Seq is: AACGATATAAT\n",
      "0 AACGATATAAT motifs detected in file\n",
      "\n",
      "Seq is: ACGAACCAATA\n",
      "0 ACGAACCAATA motifs detected in file\n",
      "\n",
      "Seq is: TTAAAAAGTAG\n",
      "0 TTAAAAAGTAG motifs detected in file\n",
      "\n",
      "Seq is: AAAGGGAACGC\n",
      "0 AAAGGGAACGC motifs detected in file\n",
      "\n",
      "Seq is: CATGATATAGA\n",
      "0 CATGATATAGA motifs detected in file\n",
      "\n",
      "Seq is: TAACGCGGTCC\n",
      "0 TAACGCGGTCC motifs detected in file\n",
      "\n",
      "Seq is: AAAGTTAAAGA\n",
      "0 AAAGTTAAAGA motifs detected in file\n",
      "\n",
      "Seq is: TAGCGGATATG\n",
      "0 TAGCGGATATG motifs detected in file\n",
      "\n",
      "Seq is: AAATAAACCCA\n",
      "0 AAATAAACCCA motifs detected in file\n",
      "\n",
      "Seq is: TTTACCAAACC\n",
      "0 TTTACCAAACC motifs detected in file\n",
      "\n",
      "Seq is: GATTCCAACAG\n",
      "0 GATTCCAACAG motifs detected in file\n",
      "\n",
      "Complement found from 14479 to 14490\n",
      "Seq is: ACCATCATTCC\n",
      "1 ACCATCATTCC motifs detected in file\n",
      "\n",
      "Seq is: ACCTCACCCAA\n",
      "0 ACCTCACCCAA motifs detected in file\n",
      "\n",
      "Seq is: ACCCCGATCCA\n",
      "0 ACCCCGATCCA motifs detected in file\n",
      "\n",
      "Seq is: AATCGAGTCTC\n",
      "0 AATCGAGTCTC motifs detected in file\n",
      "\n",
      "Seq is: GCCAGTTCAAT\n",
      "0 GCCAGTTCAAT motifs detected in file\n",
      "\n",
      "Seq is: TCAACTTTAGA\n",
      "0 TCAACTTTAGA motifs detected in file\n",
      "\n",
      "Seq is: GGATTCACATT\n",
      "0 GGATTCACATT motifs detected in file\n",
      "\n",
      "Seq is: CAACCCACGAA\n",
      "0 CAACCCACGAA motifs detected in file\n",
      "\n",
      "Seq is: ACACAATTCGA\n",
      "0 ACACAATTCGA motifs detected in file\n",
      "\n",
      "Seq is: TGTGAGACCAA\n",
      "0 TGTGAGACCAA motifs detected in file\n",
      "\n",
      "Seq is: GCAGGTTCACG\n",
      "0 GCAGGTTCACG motifs detected in file\n",
      "\n",
      "Seq is: TGAAAGGTCAT\n",
      "0 TGAAAGGTCAT motifs detected in file\n",
      "\n",
      "Seq is: GTGAATGGTAC\n",
      "0 GTGAATGGTAC motifs detected in file\n",
      "\n",
      "Seq is: AATGCTGAACA\n",
      "0 AATGCTGAACA motifs detected in file\n",
      "\n",
      "Seq is: GAGGAGATATA\n",
      "0 GAGGAGATATA motifs detected in file\n",
      "\n",
      "Seq is: TTTACGCATCC\n",
      "0 TTTACGCATCC motifs detected in file\n",
      "\n",
      "Seq is: CCAAAATCAAT\n",
      "0 CCAAAATCAAT motifs detected in file\n",
      "\n",
      "Seq is: TTACAGGAAAC\n",
      "0 TTACAGGAAAC motifs detected in file\n",
      "\n",
      "Seq is: TTCATATGAAC\n",
      "0 TTCATATGAAC motifs detected in file\n",
      "\n",
      "Seq is: TCCTCCCACTG\n",
      "0 TCCTCCCACTG motifs detected in file\n",
      "\n",
      "Seq is: CCCGCCACACA\n",
      "0 CCCGCCACACA motifs detected in file\n",
      "\n",
      "Seq is: TGCGCGAAGTC\n",
      "0 TGCGCGAAGTC motifs detected in file\n",
      "\n",
      "Seq is: CCGGGACAAGT\n",
      "0 CCGGGACAAGT motifs detected in file\n",
      "\n",
      "Seq is: TGATTCGTGAG\n",
      "0 TGATTCGTGAG motifs detected in file\n",
      "\n",
      "Seq is: ATGAGAATCAA\n",
      "0 ATGAGAATCAA motifs detected in file\n",
      "\n",
      "Seq is: ATGACGATTTA\n",
      "0 ATGACGATTTA motifs detected in file\n",
      "\n",
      "Seq is: GGTGGAAGCTG\n",
      "0 GGTGGAAGCTG motifs detected in file\n",
      "\n",
      "Seq is: GGAATTCAAAG\n",
      "0 GGAATTCAAAG motifs detected in file\n",
      "\n",
      "Seq is: TATTCCCGATA\n",
      "0 TATTCCCGATA motifs detected in file\n",
      "\n",
      "Seq is: GCATCAAAAGA\n",
      "0 GCATCAAAAGA motifs detected in file\n",
      "\n",
      "Seq is: CCCCATCTTTT\n",
      "0 CCCCATCTTTT motifs detected in file\n",
      "\n",
      "Seq is: ACATCGGGTAA\n",
      "0 ACATCGGGTAA motifs detected in file\n",
      "\n",
      "Seq is: AGAACGGTGGA\n",
      "0 AGAACGGTGGA motifs detected in file\n",
      "\n",
      "Seq is: GTACCCGATGT\n",
      "0 GTACCCGATGT motifs detected in file\n",
      "\n",
      "Seq is: GGAACTGGATT\n",
      "0 GGAACTGGATT motifs detected in file\n",
      "\n",
      "Seq is: GCAGAAATGCA\n",
      "0 GCAGAAATGCA motifs detected in file\n",
      "\n",
      "Seq is: CCCATGAACGC\n",
      "0 CCCATGAACGC motifs detected in file\n",
      "\n",
      "Seq is: GAATGAAACAT\n",
      "0 GAATGAAACAT motifs detected in file\n",
      "\n",
      "Seq is: CGGAAGTAGTC\n",
      "0 CGGAAGTAGTC motifs detected in file\n",
      "\n",
      "Seq is: CCAAACGACTT\n",
      "0 CCAAACGACTT motifs detected in file\n",
      "\n",
      "Seq is: CTACCGCCATA\n",
      "0 CTACCGCCATA motifs detected in file\n",
      "\n",
      "Seq is: TATCCGACTCG\n",
      "0 TATCCGACTCG motifs detected in file\n",
      "\n",
      "Seq is: TTCTCCACCAC\n",
      "0 TTCTCCACCAC motifs detected in file\n",
      "\n",
      "Seq is: TCCAACTAGCC\n",
      "0 TCCAACTAGCC motifs detected in file\n",
      "\n",
      "Seq is: CCAAATAGCTA\n",
      "0 CCAAATAGCTA motifs detected in file\n",
      "\n",
      "Seq is: ATGTCTTGTCC\n",
      "0 ATGTCTTGTCC motifs detected in file\n",
      "\n",
      "Seq is: GAGGAGATCTC\n",
      "0 GAGGAGATCTC motifs detected in file\n",
      "\n",
      "Seq is: CCTATACTTCG\n",
      "0 CCTATACTTCG motifs detected in file\n",
      "\n",
      "Seq is: TGGCGGTCCAG\n",
      "0 TGGCGGTCCAG motifs detected in file\n",
      "\n",
      "Seq is: GAAACTCAAAA\n",
      "0 GAAACTCAAAA motifs detected in file\n",
      "\n",
      "Seq is: TTCGACACCGA\n",
      "0 TTCGACACCGA motifs detected in file\n",
      "\n",
      "Seq is: GCATCACAAGA\n",
      "0 GCATCACAAGA motifs detected in file\n",
      "\n",
      "Seq is: CCGCTCGTCAA\n",
      "0 CCGCTCGTCAA motifs detected in file\n",
      "\n",
      "Seq is: AACAACTAAAT\n",
      "0 AACAACTAAAT motifs detected in file\n",
      "\n",
      "Seq is: TGACAACTCCA\n",
      "0 TGACAACTCCA motifs detected in file\n",
      "\n",
      "Seq is: AATCCCGATTC\n",
      "0 AATCCCGATTC motifs detected in file\n",
      "\n",
      "Seq is: GTATCACCCCA\n",
      "0 GTATCACCCCA motifs detected in file\n",
      "\n",
      "Seq is: TAGATTAGGGT\n",
      "0 TAGATTAGGGT motifs detected in file\n",
      "\n",
      "Seq is: CAAACCCAGAA\n",
      "0 CAAACCCAGAA motifs detected in file\n",
      "\n",
      "Seq is: TCGATAACACA\n",
      "0 TCGATAACACA motifs detected in file\n",
      "\n",
      "Seq is: CAAGTCTATAC\n",
      "0 CAAGTCTATAC motifs detected in file\n",
      "\n",
      "Seq is: AATTTCGGTGA\n",
      "0 AATTTCGGTGA motifs detected in file\n",
      "\n",
      "Seq is: AAGCATCAGAT\n",
      "0 AAGCATCAGAT motifs detected in file\n",
      "\n",
      "Seq is: AAAACACAGTT\n",
      "0 AAAACACAGTT motifs detected in file\n",
      "\n",
      "Seq is: GACCTCAAAAA\n",
      "0 GACCTCAAAAA motifs detected in file\n",
      "\n",
      "Seq is: ATGTTGAGTCC\n",
      "0 ATGTTGAGTCC motifs detected in file\n",
      "\n",
      "Seq is: ACTCAAAATCG\n",
      "0 ACTCAAAATCG motifs detected in file\n",
      "\n",
      "Seq is: AAATAACCCCT\n",
      "0 AAATAACCCCT motifs detected in file\n",
      "\n",
      "Seq is: CCCCCACTAGA\n",
      "0 CCCCCACTAGA motifs detected in file\n",
      "\n",
      "Seq is: TTTTGTGAGAA\n",
      "0 TTTTGTGAGAA motifs detected in file\n",
      "\n",
      "Seq is: ATGCGGCCGAA\n",
      "0 ATGCGGCCGAA motifs detected in file\n",
      "\n",
      "Seq is: GATAACTGAAC\n",
      "0 GATAACTGAAC motifs detected in file\n",
      "\n",
      "Seq is: CCAATTAGCAC\n",
      "0 CCAATTAGCAC motifs detected in file\n",
      "\n",
      "Seq is: ACTGGCGCCAC\n",
      "0 ACTGGCGCCAC motifs detected in file\n",
      "\n",
      "Seq is: CGACCGTGCTT\n",
      "0 CGACCGTGCTT motifs detected in file\n",
      "\n",
      "Seq is: TAACTGGTTGG\n",
      "0 TAACTGGTTGG motifs detected in file\n",
      "\n",
      "Seq is: GACCCCAATCA\n",
      "0 GACCCCAATCA motifs detected in file\n",
      "\n",
      "Seq is: TATCGAATCAA\n",
      "0 TATCGAATCAA motifs detected in file\n",
      "\n",
      "Seq is: TTTGAAAGCAA\n",
      "0 TTTGAAAGCAA motifs detected in file\n",
      "\n",
      "Seq is: ATAACGATTTC\n",
      "0 ATAACGATTTC motifs detected in file\n",
      "\n",
      "Seq is: CAATTAGTGAC\n",
      "0 CAATTAGTGAC motifs detected in file\n",
      "\n",
      "Seq is: GACAAAGGGCA\n",
      "0 GACAAAGGGCA motifs detected in file\n",
      "\n",
      "Seq is: CCCCCACACCG\n",
      "0 CCCCCACACCG motifs detected in file\n",
      "\n",
      "Seq is: ATCCGATTCGC\n",
      "0 ATCCGATTCGC motifs detected in file\n",
      "\n",
      "Seq is: AAAACTCGACG\n",
      "0 AAAACTCGACG motifs detected in file\n",
      "\n",
      "Seq is: TAACGACGCAC\n",
      "0 TAACGACGCAC motifs detected in file\n",
      "\n",
      "Seq is: GAACTACGAAC\n",
      "0 GAACTACGAAC motifs detected in file\n",
      "\n",
      "Seq is: AAGGAAAACTA\n",
      "0 AAGGAAAACTA motifs detected in file\n",
      "\n",
      "Seq is: GCACCACTAAA\n",
      "0 GCACCACTAAA motifs detected in file\n",
      "\n",
      "Seq is: TCTCCCACTTG\n",
      "0 TCTCCCACTTG motifs detected in file\n",
      "\n",
      "Seq is: AGTGACCTTGC\n",
      "0 AGTGACCTTGC motifs detected in file\n",
      "\n",
      "Seq is: CCCTACGAACG\n",
      "0 CCCTACGAACG motifs detected in file\n",
      "\n",
      "Seq is: TACACATTAGA\n",
      "0 TACACATTAGA motifs detected in file\n",
      "\n",
      "Seq is: ATGATTCTCGA\n",
      "0 ATGATTCTCGA motifs detected in file\n",
      "\n",
      "Seq is: TTATCTTTCCG\n",
      "0 TTATCTTTCCG motifs detected in file\n",
      "\n",
      "Seq is: ATCCTGGTTTG\n",
      "0 ATCCTGGTTTG motifs detected in file\n",
      "\n",
      "Seq is: GATAAACAAAT\n",
      "0 GATAAACAAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCCCACTACA\n",
      "0 ACCCCACTACA motifs detected in file\n",
      "\n",
      "Seq is: CTCGGGCAGAT\n",
      "0 CTCGGGCAGAT motifs detected in file\n",
      "\n",
      "Seq is: TTGTAAAAGTC\n",
      "0 TTGTAAAAGTC motifs detected in file\n",
      "\n",
      "Seq is: ACATAACGAAA\n",
      "0 ACATAACGAAA motifs detected in file\n",
      "\n",
      "Seq is: CTCCTCCATTC\n",
      "0 CTCCTCCATTC motifs detected in file\n",
      "\n",
      "Seq is: GATGTATTTGA\n",
      "0 GATGTATTTGA motifs detected in file\n",
      "\n",
      "Complement found from 565 to 576\n",
      "Seq is: CACCCCCCACA\n",
      "1 CACCCCCCACA motifs detected in file\n",
      "\n",
      "Seq is: GAAACCCCAAA\n",
      "0 GAAACCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: CCAACCAAGCC\n",
      "0 CCAACCAAGCC motifs detected in file\n",
      "\n",
      "Seq is: CCATACCCCAA\n",
      "0 CCATACCCCAA motifs detected in file\n",
      "\n",
      "Seq is: TCGTCGCCACA\n",
      "0 TCGTCGCCACA motifs detected in file\n",
      "\n",
      "Seq is: CACACACGACC\n",
      "0 CACACACGACC motifs detected in file\n",
      "\n",
      "Seq is: CATCCTACCCG\n",
      "0 CATCCTACCCG motifs detected in file\n",
      "\n",
      "Seq is: CCCCCAACATA\n",
      "0 CCCCCAACATA motifs detected in file\n",
      "\n",
      "Seq is: ACTACTCTAAT\n",
      "0 ACTACTCTAAT motifs detected in file\n",
      "\n",
      "Seq is: CATCATACCCT\n",
      "0 CATCATACCCT motifs detected in file\n",
      "\n",
      "Seq is: CACCCTCCCCT\n",
      "0 CACCCTCCCCT motifs detected in file\n",
      "\n",
      "Seq is: TTTATTACACA\n",
      "0 TTTATTACACA motifs detected in file\n",
      "\n",
      "Seq is: ATCAACCCCCC\n",
      "0 ATCAACCCCCC motifs detected in file\n",
      "\n",
      "Seq is: ACTGACAATTT\n",
      "0 ACTGACAATTT motifs detected in file\n",
      "\n",
      "Seq is: TCACGTATGGC\n",
      "0 TCACGTATGGC motifs detected in file\n",
      "\n",
      "Seq is: GGTTTTCTATT\n",
      "0 GGTTTTCTATT motifs detected in file\n",
      "\n",
      "Seq is: TTAAACTTTAG\n",
      "0 TTAAACTTTAG motifs detected in file\n",
      "\n",
      "Seq is: ACCAATCCGAC\n",
      "0 ACCAATCCGAC motifs detected in file\n",
      "\n",
      "Seq is: CACAATCCCAA\n",
      "0 CACAATCCCAA motifs detected in file\n",
      "\n",
      "Seq is: GAAACAAAAAC\n",
      "0 GAAACAAAAAC motifs detected in file\n",
      "\n",
      "Seq is: CCCAAACCGTC\n",
      "0 CCCAAACCGTC motifs detected in file\n",
      "\n",
      "Seq is: TCTACACAAAT\n",
      "0 TCTACACAAAT motifs detected in file\n",
      "\n",
      "Seq is: TCACGACACCG\n",
      "0 TCACGACACCG motifs detected in file\n",
      "\n",
      "Seq is: GTCTTCGCCCC\n",
      "0 GTCTTCGCCCC motifs detected in file\n",
      "\n",
      "Seq is: CTCCCCCCCAA\n",
      "0 CTCCCCCCCAA motifs detected in file\n",
      "\n",
      "Seq is: ACCACCTTTAA\n",
      "0 ACCACCTTTAA motifs detected in file\n",
      "\n",
      "Seq is: AAAACAATACT\n",
      "0 AAAACAATACT motifs detected in file\n",
      "\n",
      "Seq is: ACAGACACACC\n",
      "0 ACAGACACACC motifs detected in file\n",
      "\n",
      "Seq is: TTTCACCGACA\n",
      "0 TTTCACCGACA motifs detected in file\n",
      "\n",
      "Seq is: CGTCTGTAAGT\n",
      "0 CGTCTGTAAGT motifs detected in file\n",
      "\n",
      "Seq is: TAACAATAATA\n",
      "0 TAACAATAATA motifs detected in file\n",
      "\n",
      "Seq is: ATACAGGATGT\n",
      "0 ATACAGGATGT motifs detected in file\n",
      "\n",
      "Seq is: TCGTAATTAAT\n",
      "0 TCGTAATTAAT motifs detected in file\n",
      "\n",
      "Seq is: TAATTGTGTGA\n",
      "0 TAATTGTGTGA motifs detected in file\n",
      "\n",
      "Seq is: AATCATTCATA\n",
      "0 AATCATTCATA motifs detected in file\n",
      "\n",
      "Seq is: CAAGCGGACAT\n",
      "0 CAAGCGGACAT motifs detected in file\n",
      "\n",
      "Seq is: TATAACTTGCA\n",
      "0 TATAACTTGCA motifs detected in file\n",
      "\n",
      "Seq is: TCCACGCTATT\n",
      "0 TCCACGCTATT motifs detected in file\n",
      "\n",
      "Seq is: TATTATCCTAC\n",
      "0 TATTATCCTAC motifs detected in file\n",
      "\n",
      "Seq is: TCCGTCCTTAG\n",
      "0 TCCGTCCTTAG motifs detected in file\n",
      "\n",
      "Seq is: TTTCTGTCTAT\n",
      "0 TTTCTGTCTAT motifs detected in file\n",
      "\n",
      "Seq is: GACGCTGTATC\n",
      "0 GACGCTGTATC motifs detected in file\n",
      "\n",
      "Seq is: CCACGAGGCCG\n",
      "0 CCACGAGGCCG motifs detected in file\n",
      "\n",
      "Seq is: AGGTCGCAGAG\n",
      "0 AGGTCGCAGAG motifs detected in file\n",
      "\n",
      "Seq is: CGTTACGATAG\n",
      "0 CGTTACGATAG motifs detected in file\n",
      "\n",
      "Seq is: CGCACGTATGG\n",
      "0 CGCACGTATGG motifs detected in file\n",
      "\n",
      "Seq is: GGGGTCTGCTT\n",
      "0 GGGGTCTGCTT motifs detected in file\n",
      "\n",
      "Seq is: TTATGGTTTAC\n",
      "0 TTATGGTTTAC motifs detected in file\n",
      "\n",
      "Seq is: GTACCTCTCGA\n",
      "0 GTACCTCTCGA motifs detected in file\n",
      "\n",
      "Seq is: GGGCACTCACC\n",
      "0 GGGCACTCACC motifs detected in file\n",
      "\n",
      "Seq is: AATTATCCCAC\n",
      "0 AATTATCCCAC motifs detected in file\n",
      "\n",
      "Seq is: TATCTGGACAC\n",
      "0 TATCTGGACAC motifs detected in file\n",
      "\n",
      "Seq is: GTAGCACTACAG\n",
      "0 GTAGCACTACAG motifs detected in file\n",
      "\n",
      "Seq is: AATAAATTCCCC\n",
      "0 AATAAATTCCCC motifs detected in file\n",
      "\n",
      "Seq is: TTGCACACCCGA\n",
      "0 TTGCACACCCGA motifs detected in file\n",
      "\n",
      "Seq is: TAAATCCGAAAT\n",
      "0 TAAATCCGAAAT motifs detected in file\n",
      "\n",
      "Seq is: ACTGGGACTTCA\n",
      "0 ACTGGGACTTCA motifs detected in file\n",
      "\n",
      "Seq is: TCCTTGGTCTAC\n",
      "0 TCCTTGGTCTAC motifs detected in file\n",
      "\n",
      "Seq is: AGCCTATGTCAA\n",
      "0 AGCCTATGTCAA motifs detected in file\n",
      "\n",
      "Seq is: GTGAAATCGATG\n",
      "0 GTGAAATCGATG motifs detected in file\n",
      "\n",
      "Seq is: GGGGTTCACAAT\n",
      "0 GGGGTTCACAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCCGGGCCTCG\n",
      "0 ACCCGGGCCTCG motifs detected in file\n",
      "\n",
      "Seq is: CTCCTCTCATCG\n",
      "0 CTCCTCTCATCG motifs detected in file\n",
      "\n",
      "Seq is: TGAGAACACGCC\n",
      "0 TGAGAACACGCC motifs detected in file\n",
      "\n",
      "Seq is: CTATAACTAAAG\n",
      "0 CTATAACTAAAG motifs detected in file\n",
      "\n",
      "Seq is: TGCCTCCTACCA\n",
      "0 TGCCTCCTACCA motifs detected in file\n",
      "\n",
      "Seq is: CCAGTTCCCTGG\n",
      "0 CCAGTTCCCTGG motifs detected in file\n",
      "\n",
      "Seq is: GGATAGACTCCC\n",
      "0 GGATAGACTCCC motifs detected in file\n",
      "\n",
      "Seq is: CCCAGTAGGTAC\n",
      "0 CCCAGTAGGTAC motifs detected in file\n",
      "\n",
      "Seq is: CCCTGCTCTTCC\n",
      "0 CCCTGCTCTTCC motifs detected in file\n",
      "\n",
      "Seq is: CTAAACTGACAT\n",
      "0 CTAAACTGACAT motifs detected in file\n",
      "\n",
      "Seq is: TACACGATACAT\n",
      "0 TACACGATACAT motifs detected in file\n",
      "\n",
      "Seq is: GCCATTTACCGA\n",
      "0 GCCATTTACCGA motifs detected in file\n",
      "\n",
      "Seq is: AATACATGATAC\n",
      "0 AATACATGATAC motifs detected in file\n",
      "\n",
      "Seq is: ATGACAATTCCC\n",
      "0 ATGACAATTCCC motifs detected in file\n",
      "\n",
      "Seq is: ACCCATCCAAAC\n",
      "0 ACCCATCCAAAC motifs detected in file\n",
      "\n",
      "Seq is: AACCATAGGATC\n",
      "0 AACCATAGGATC motifs detected in file\n",
      "\n",
      "Seq is: ACCCACTCCCCA\n",
      "0 ACCCACTCCCCA motifs detected in file\n",
      "\n",
      "Seq is: CCGAAACCTCAA\n",
      "0 CCGAAACCTCAA motifs detected in file\n",
      "\n",
      "Seq is: CGTCAACTACAC\n",
      "0 CGTCAACTACAC motifs detected in file\n",
      "\n",
      "Seq is: ACTATCAACTCC\n",
      "0 ACTATCAACTCC motifs detected in file\n",
      "\n",
      "Seq is: CAACTAACGACA\n",
      "0 CAACTAACGACA motifs detected in file\n",
      "\n",
      "Seq is: TGAACGAACATT\n",
      "0 TGAACGAACATT motifs detected in file\n",
      "\n",
      "Seq is: CGTACCCCTCCC\n",
      "0 CGTACCCCTCCC motifs detected in file\n",
      "\n",
      "Seq is: CCAAAACTACAC\n",
      "0 CCAAAACTACAC motifs detected in file\n",
      "\n",
      "Seq is: CTAACCCAAAAA\n",
      "0 CTAACCCAAAAA motifs detected in file\n",
      "\n",
      "Seq is: TACATGATGTCC\n",
      "0 TACATGATGTCC motifs detected in file\n",
      "\n",
      "Seq is: ACCAGTTCATAA\n",
      "0 ACCAGTTCATAA motifs detected in file\n",
      "\n",
      "Seq is: ATACCATGGCAT\n",
      "0 ATACCATGGCAT motifs detected in file\n",
      "\n",
      "Seq is: GTTATAAGTACC\n",
      "0 GTTATAAGTACC motifs detected in file\n",
      "\n",
      "Seq is: ACCGACCGTCAT\n",
      "0 ACCGACCGTCAT motifs detected in file\n",
      "\n",
      "Seq is: TACATGCTTTAT\n",
      "0 TACATGCTTTAT motifs detected in file\n",
      "\n",
      "Seq is: GTATCGCCAACA\n",
      "0 GTATCGCCAACA motifs detected in file\n",
      "\n",
      "Seq is: ACTACCCACTCA\n",
      "0 ACTACCCACTCA motifs detected in file\n",
      "\n",
      "Seq is: GTTATGAACCCA\n",
      "0 GTTATGAACCCA motifs detected in file\n",
      "\n",
      "Seq is: CCATGGGTTTAG\n",
      "0 CCATGGGTTTAG motifs detected in file\n",
      "\n",
      "Seq is: ACGAAGGGGTAC\n",
      "0 ACGAAGGGGTAC motifs detected in file\n",
      "\n",
      "Seq is: TTTCTTGTCTCT\n",
      "0 TTTCTTGTCTCT motifs detected in file\n",
      "\n",
      "Seq is: TATCAAATTTAA\n",
      "0 TATCAAATTTAA motifs detected in file\n",
      "\n",
      "Seq is: TCTTAGAATCGA\n",
      "0 TCTTAGAATCGA motifs detected in file\n",
      "\n",
      "Seq is: AACCCACGATTA\n",
      "0 AACCCACGATTA motifs detected in file\n",
      "\n",
      "Seq is: CCACCTCAATTT\n",
      "0 CCACCTCAATTT motifs detected in file\n",
      "\n",
      "Seq is: CTGAAAAAGAGA\n",
      "0 CTGAAAAAGAGA motifs detected in file\n",
      "\n",
      "Seq is: CTAAACAGGAAC\n",
      "0 CTAAACAGGAAC motifs detected in file\n",
      "\n",
      "Seq is: CTTTTTCCAAAA\n",
      "0 CTTTTTCCAAAA motifs detected in file\n",
      "\n",
      "Seq is: GTAGAGGCCAAA\n",
      "0 GTAGAGGCCAAA motifs detected in file\n",
      "\n",
      "Seq is: TGTTCTGACCAC\n",
      "0 TGTTCTGACCAC motifs detected in file\n",
      "\n",
      "Seq is: ATAATCAAATAT\n",
      "0 ATAATCAAATAT motifs detected in file\n",
      "\n",
      "Seq is: GATGTTCCTGTC\n",
      "0 GATGTTCCTGTC motifs detected in file\n",
      "\n",
      "Seq is: CGGGTAAACTCA\n",
      "0 CGGGTAAACTCA motifs detected in file\n",
      "\n",
      "Seq is: TAAAACAAAAGT\n",
      "0 TAAAACAAAAGT motifs detected in file\n",
      "\n",
      "Seq is: TAATCCCTCTAT\n",
      "0 TAATCCCTCTAT motifs detected in file\n",
      "\n",
      "Seq is: CAACCATAATCC\n",
      "0 CAACCATAATCC motifs detected in file\n",
      "\n",
      "Seq is: TAATCCTAACAA\n",
      "0 TAATCCTAACAA motifs detected in file\n",
      "\n",
      "Seq is: CACTTCATATCA\n",
      "0 CACTTCATATCA motifs detected in file\n",
      "\n",
      "Seq is: TGCCTACGATGA\n",
      "0 TGCCTACGATGA motifs detected in file\n",
      "\n",
      "Seq is: ACAGGTTACTAC\n",
      "0 ACAGGTTACTAC motifs detected in file\n",
      "\n",
      "Seq is: CATTTTCCCATC\n",
      "0 CATTTTCCCATC motifs detected in file\n",
      "\n",
      "Seq is: GAATGACCAACA\n",
      "0 GAATGACCAACA motifs detected in file\n",
      "\n",
      "Seq is: GGAGGCTAAGTC\n",
      "0 GGAGGCTAAGTC motifs detected in file\n",
      "\n",
      "Seq is: CAATCTTACTCC\n",
      "0 CAATCTTACTCC motifs detected in file\n",
      "\n",
      "Seq is: TCCAGACGCCGA\n",
      "0 TCCAGACGCCGA motifs detected in file\n",
      "\n",
      "Seq is: TCCTCAGTTATT\n",
      "0 TCCTCAGTTATT motifs detected in file\n",
      "\n",
      "Seq is: TCACTAACCGAA\n",
      "0 TCACTAACCGAA motifs detected in file\n",
      "\n",
      "Seq is: TCACCCGCTTTA\n",
      "0 TCACCCGCTTTA motifs detected in file\n",
      "\n",
      "Seq is: TAATACGAAACA\n",
      "0 TAATACGAAACA motifs detected in file\n",
      "\n",
      "Seq is: ACAAACCTATAT\n",
      "0 ACAAACCTATAT motifs detected in file\n",
      "\n",
      "Seq is: ACCTCCTACCCC\n",
      "0 ACCTCCTACCCC motifs detected in file\n",
      "\n",
      "Seq is: TAATAACGATCC\n",
      "0 TAATAACGATCC motifs detected in file\n",
      "\n",
      "Complement found from 5472 to 5484\n",
      "Seq is: TACTCCTACCTA\n",
      "1 TACTCCTACCTA motifs detected in file\n",
      "\n",
      "Seq is: TCATTATCCCGT\n",
      "0 TCATTATCCCGT motifs detected in file\n",
      "\n",
      "Seq is: TCCTGCGGAGGA\n",
      "0 TCCTGCGGAGGA motifs detected in file\n",
      "\n",
      "Seq is: TCAAACAATCCC\n",
      "0 TCAAACAATCCC motifs detected in file\n",
      "\n",
      "Seq is: TGCCTAGCCTCT\n",
      "0 TGCCTAGCCTCT motifs detected in file\n",
      "\n",
      "Seq is: TAACACATCCGC\n",
      "0 TAACACATCCGC motifs detected in file\n",
      "\n",
      "Seq is: TTATCCTTTATA\n",
      "0 TTATCCTTTATA motifs detected in file\n",
      "\n",
      "Seq is: GTAAGCCCGAAC\n",
      "0 GTAAGCCCGAAC motifs detected in file\n",
      "\n",
      "Seq is: TACACCCCTCCC\n",
      "0 TACACCCCTCCC motifs detected in file\n",
      "\n",
      "Seq is: CACAAATTCCCC\n",
      "0 CACAAATTCCCC motifs detected in file\n",
      "\n",
      "Seq is: AACCGATCCCAT\n",
      "0 AACCGATCCCAT motifs detected in file\n",
      "\n",
      "Seq is: ATTAACAGACCC\n",
      "0 ATTAACAGACCC motifs detected in file\n",
      "\n",
      "Seq is: AGCGGATCCTCC\n",
      "0 AGCGGATCCTCC motifs detected in file\n",
      "\n",
      "Seq is: AGACCACTCTTA\n",
      "0 AGACCACTCTTA motifs detected in file\n",
      "\n",
      "Seq is: TCACAATTACAG\n",
      "0 TCACAATTACAG motifs detected in file\n",
      "\n",
      "Seq is: TAATTCCTCTCT\n",
      "0 TAATTCCTCTCT motifs detected in file\n",
      "\n",
      "Seq is: TCCTTCTCTTCA\n",
      "0 TCCTTCTCTTCA motifs detected in file\n",
      "\n",
      "Seq is: TTCGGCTCCCGC\n",
      "0 TTCGGCTCCCGC motifs detected in file\n",
      "\n",
      "Seq is: AGAAACTAACAC\n",
      "0 AGAAACTAACAC motifs detected in file\n",
      "\n",
      "Seq is: ATCATTCCCACC\n",
      "0 ATCATTCCCACC motifs detected in file\n",
      "\n",
      "Seq is: TTCCACTAAAAT\n",
      "0 TTCCACTAAAAT motifs detected in file\n",
      "\n",
      "Seq is: AGCCTTACCCTC\n",
      "0 AGCCTTACCCTC motifs detected in file\n",
      "\n",
      "Seq is: CACTAAGGATCC\n",
      "0 CACTAAGGATCC motifs detected in file\n",
      "\n",
      "Seq is: CCCAACAAACTA\n",
      "0 CCCAACAAACTA motifs detected in file\n",
      "\n",
      "Seq is: GGGCAAAGCACG\n",
      "0 GGGCAAAGCACG motifs detected in file\n",
      "\n",
      "Seq is: TTCTTATCCTCC\n",
      "0 TTCTTATCCTCC motifs detected in file\n",
      "\n",
      "Seq is: ACCTCACAACGA\n",
      "0 ACCTCACAACGA motifs detected in file\n",
      "\n",
      "Seq is: TCCCGACGTTAT\n",
      "0 TCCCGACGTTAT motifs detected in file\n",
      "\n",
      "Seq is: TACTTCCCGTTC\n",
      "0 TACTTCCCGTTC motifs detected in file\n",
      "\n",
      "Seq is: TACTTCACTTTC\n",
      "0 TACTTCACTTTC motifs detected in file\n",
      "\n",
      "Seq is: CATTTCTTAGCA\n",
      "0 CATTTCTTAGCA motifs detected in file\n",
      "\n",
      "Seq is: CACTCCCACCCT\n",
      "0 CACTCCCACCCT motifs detected in file\n",
      "\n",
      "Seq is: GACAGATGACTC\n",
      "0 GACAGATGACTC motifs detected in file\n",
      "\n",
      "Seq is: ATCGGAGGAGTC\n",
      "0 ATCGGAGGAGTC motifs detected in file\n",
      "\n",
      "Seq is: TAAGTAACTTGA\n",
      "0 TAAGTAACTTGA motifs detected in file\n",
      "\n",
      "Seq is: TCCAGACAGGGT\n",
      "0 TCCAGACAGGGT motifs detected in file\n",
      "\n",
      "Seq is: TACATACCCTAC\n",
      "0 TACATACCCTAC motifs detected in file\n",
      "\n",
      "Seq is: CGCCTATCATTC\n",
      "0 CGCCTATCATTC motifs detected in file\n",
      "\n",
      "Seq is: AAACATTAATGA\n",
      "0 AAACATTAATGA motifs detected in file\n",
      "\n",
      "Seq is: CACCGGGGAGTC\n",
      "0 CACCGGGGAGTC motifs detected in file\n",
      "\n",
      "Seq is: TTACTATAAACC\n",
      "0 TTACTATAAACC motifs detected in file\n",
      "\n",
      "Seq is: GGAGTGCCCTCC\n",
      "0 GGAGTGCCCTCC motifs detected in file\n",
      "\n",
      "Seq is: TGTATCGGATAC\n",
      "0 TGTATCGGATAC motifs detected in file\n",
      "\n",
      "Seq is: TTCCGACAACGA\n",
      "0 TTCCGACAACGA motifs detected in file\n",
      "\n",
      "Seq is: TATCAACGTTCG\n",
      "0 TATCAACGTTCG motifs detected in file\n",
      "\n",
      "Seq is: TCCTCCTATTAC\n",
      "0 TCCTCCTATTAC motifs detected in file\n",
      "\n",
      "Seq is: GGCTACAAAGTC\n",
      "0 GGCTACAAAGTC motifs detected in file\n",
      "\n",
      "Seq is: CAAAGACTCATC\n",
      "0 CAAAGACTCATC motifs detected in file\n",
      "\n",
      "Seq is: TCTTTACTAGGC\n",
      "0 TCTTTACTAGGC motifs detected in file\n",
      "\n",
      "Seq is: ATTATATCCGGA\n",
      "0 ATTATATCCGGA motifs detected in file\n",
      "\n",
      "Seq is: GCGGGCTACACA\n",
      "0 GCGGGCTACACA motifs detected in file\n",
      "\n",
      "Seq is: TCCTTCTCCGTC\n",
      "0 TCCTTCTCCGTC motifs detected in file\n",
      "\n",
      "Seq is: TATTTCTTATAA\n",
      "0 TATTTCTTATAA motifs detected in file\n",
      "\n",
      "Seq is: CTCCGCGGTAAC\n",
      "0 CTCCGCGGTAAC motifs detected in file\n",
      "\n",
      "Seq is: CGCACTTCCATC\n",
      "0 CGCACTTCCATC motifs detected in file\n",
      "\n",
      "Seq is: GCCTACTAAGTC\n",
      "0 GCCTACTAAGTC motifs detected in file\n",
      "\n",
      "Seq is: GGTATTAAATGC\n",
      "0 GGTATTAAATGC motifs detected in file\n",
      "\n",
      "Seq is: AGAGCTCACTAC\n",
      "0 AGAGCTCACTAC motifs detected in file\n",
      "\n",
      "Seq is: ACCCGCTAACTA\n",
      "0 ACCCGCTAACTA motifs detected in file\n",
      "\n",
      "Seq is: CTTTTCCGCCAA\n",
      "0 CTTTTCCGCCAA motifs detected in file\n",
      "\n",
      "Seq is: CTCCGCAGACCA\n",
      "0 CTCCGCAGACCA motifs detected in file\n",
      "\n",
      "Seq is: CTCATCACGTAC\n",
      "0 CTCATCACGTAC motifs detected in file\n",
      "\n",
      "Seq is: CGATCCTTATCA\n",
      "0 CGATCCTTATCA motifs detected in file\n",
      "\n",
      "Seq is: GGACACCACTAA\n",
      "0 GGACACCACTAA motifs detected in file\n",
      "\n",
      "Seq is: ACCTCCTAGTCC\n",
      "0 ACCTCCTAGTCC motifs detected in file\n",
      "\n",
      "Seq is: GTCCGCGGTTCC\n",
      "0 GTCCGCGGTTCC motifs detected in file\n",
      "\n",
      "Seq is: TCACTCGGCTTC\n",
      "0 TCACTCGGCTTC motifs detected in file\n",
      "\n",
      "Seq is: AAAGTAGTACGC\n",
      "0 AAAGTAGTACGC motifs detected in file\n",
      "\n",
      "Seq is: CTCTACAACCTA\n",
      "0 CTCTACAACCTA motifs detected in file\n",
      "\n",
      "Seq is: CCCCACCCCTCC\n",
      "0 CCCCACCCCTCC motifs detected in file\n",
      "\n",
      "Seq is: AGCTACTTACTC\n",
      "0 AGCTACTTACTC motifs detected in file\n",
      "\n",
      "Seq is: ACCAATTAATTA\n",
      "0 ACCAATTAATTA motifs detected in file\n",
      "\n",
      "Seq is: AAATAATCCCCC\n",
      "0 AAATAATCCCCC motifs detected in file\n",
      "\n",
      "Seq is: AATCAAAACGCA\n",
      "0 AATCAAAACGCA motifs detected in file\n",
      "\n",
      "Seq is: TAACCCCAGTAA\n",
      "0 TAACCCCAGTAA motifs detected in file\n",
      "\n",
      "Seq is: CCACAAGAACAT\n",
      "0 CCACAAGAACAT motifs detected in file\n",
      "\n",
      "Seq is: CAACTTTATGTT\n",
      "0 CAACTTTATGTT motifs detected in file\n",
      "\n",
      "Seq is: GCTACCAAAAAG\n",
      "0 GCTACCAAAAAG motifs detected in file\n",
      "\n",
      "Seq is: TATAGTAACCAG\n",
      "0 TATAGTAACCAG motifs detected in file\n",
      "\n",
      "Seq is: CACCAACATCAG\n",
      "0 CACCAACATCAG motifs detected in file\n",
      "\n",
      "Seq is: GCACGCTCTTAT\n",
      "0 GCACGCTCTTAT motifs detected in file\n",
      "\n",
      "Seq is: TACTACATACGA\n",
      "0 TACTACATACGA motifs detected in file\n",
      "\n",
      "Seq is: AACAAAGACAAC\n",
      "0 AACAAAGACAAC motifs detected in file\n",
      "\n",
      "Seq is: TCACACCCAAAT\n",
      "0 TCACACCCAAAT motifs detected in file\n",
      "\n",
      "Seq is: CATTACCCCAAA\n",
      "0 CATTACCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: CACCCCAAAAGA\n",
      "0 CACCCCAAAAGA motifs detected in file\n",
      "\n",
      "Seq is: AGATTCGGAAGA\n",
      "0 AGATTCGGAAGA motifs detected in file\n",
      "\n",
      "Seq is: GGATAAATACCC\n",
      "0 GGATAAATACCC motifs detected in file\n",
      "\n",
      "Seq is: CCAAATCATAAC\n",
      "0 CCAAATCATAAC motifs detected in file\n",
      "\n",
      "Seq is: TAACAATCGCCA\n",
      "0 TAACAATCGCCA motifs detected in file\n",
      "\n",
      "Seq is: CACCAGCCCACA\n",
      "0 CACCAGCCCACA motifs detected in file\n",
      "\n",
      "Seq is: CAATAATAAGAC\n",
      "0 CAATAATAAGAC motifs detected in file\n",
      "\n",
      "Seq is: TTAAAACCCCCT\n",
      "0 TTAAAACCCCCT motifs detected in file\n",
      "\n",
      "Seq is: CCAATATACCCA\n",
      "0 CCAATATACCCA motifs detected in file\n",
      "\n",
      "Seq is: AATTATCAAAAA\n",
      "0 AATTATCAAAAA motifs detected in file\n",
      "\n",
      "Seq is: AATTAAATAAAT\n",
      "0 AATTAAATAAAT motifs detected in file\n",
      "\n",
      "Seq is: CCCCCTTACTAC\n",
      "0 CCCCCTTACTAC motifs detected in file\n",
      "\n",
      "Seq is: CAACAGAAACCT\n",
      "0 CAACAGAAACCT motifs detected in file\n",
      "\n",
      "Seq is: ATATGATGTCGC\n",
      "0 ATATGATGTCGC motifs detected in file\n",
      "\n",
      "Seq is: TACCGATAACTC\n",
      "0 TACCGATAACTC motifs detected in file\n",
      "\n",
      "Seq is: CTCATAGGACTC\n",
      "0 CTCATAGGACTC motifs detected in file\n",
      "\n",
      "Seq is: CGTACCCCCAGT\n",
      "0 CGTACCCCCAGT motifs detected in file\n",
      "\n",
      "Seq is: CCCCAACTCCAG\n",
      "0 CCCCAACTCCAG motifs detected in file\n",
      "\n",
      "Seq is: AACCACTCACAA\n",
      "0 AACCACTCACAA motifs detected in file\n",
      "\n",
      "Seq is: AATCACCCCAAT\n",
      "0 AATCACCCCAAT motifs detected in file\n",
      "\n",
      "Seq is: CGCTACCTCCAT\n",
      "0 CGCTACCTCCAT motifs detected in file\n",
      "\n",
      "Seq is: CCTAACCACGAC\n",
      "0 CCTAACCACGAC motifs detected in file\n",
      "\n",
      "Seq is: ACCCACTTTCTC\n",
      "0 ACCCACTTTCTC motifs detected in file\n",
      "\n",
      "Seq is: ATACTACCCCAC\n",
      "0 ATACTACCCCAC motifs detected in file\n",
      "\n",
      "Seq is: CACCAACACCAT\n",
      "0 CACCAACACCAT motifs detected in file\n",
      "\n",
      "Seq is: TTGAAATTATCA\n",
      "0 TTGAAATTATCA motifs detected in file\n",
      "\n",
      "Seq is: CATCCTTCGACT\n",
      "0 CATCCTTCGACT motifs detected in file\n",
      "\n",
      "Seq is: TATTAAATACTT\n",
      "0 TATTAAATACTT motifs detected in file\n",
      "\n",
      "Seq is: CCTCTCCCCAGT\n",
      "0 CCTCTCCCCAGT motifs detected in file\n",
      "\n",
      "Seq is: CCCAACTAAGCC\n",
      "0 CCCAACTAAGCC motifs detected in file\n",
      "\n",
      "Seq is: CTCCTAGGATAA\n",
      "0 CTCCTAGGATAA motifs detected in file\n",
      "\n",
      "Seq is: CCACGCCCCCGA\n",
      "0 CCACGCCCCCGA motifs detected in file\n",
      "\n",
      "Seq is: AACATACTAATA\n",
      "0 AACATACTAATA motifs detected in file\n",
      "\n",
      "Seq is: CCCGCAACTAAT\n",
      "0 CCCGCAACTAAT motifs detected in file\n",
      "\n",
      "Seq is: CATCATCAATGA\n",
      "0 CATCATCAATGA motifs detected in file\n",
      "\n",
      "Seq is: CCAACTTGTAAC\n",
      "0 CCAACTTGTAAC motifs detected in file\n",
      "\n",
      "Seq is: AAACAACCACAT\n",
      "0 AAACAACCACAT motifs detected in file\n",
      "\n",
      "Seq is: ATATAACATTAA\n",
      "0 ATATAACATTAA motifs detected in file\n",
      "\n",
      "Seq is: CTCTAACGAGCC\n",
      "0 CTCTAACGAGCC motifs detected in file\n",
      "\n",
      "Seq is: CCCTTATCCAAT\n",
      "0 CCCTTATCCAAT motifs detected in file\n",
      "\n",
      "Seq is: ACACTAATCCTC\n",
      "0 ACACTAATCCTC motifs detected in file\n",
      "\n",
      "Seq is: ATCCCAATCCTA\n",
      "0 ATCCCAATCCTA motifs detected in file\n",
      "\n",
      "Seq is: CTCACCCTTCTT\n",
      "0 CTCACCCTTCTT motifs detected in file\n",
      "\n",
      "Seq is: CTTTCTCTCCTT\n",
      "0 CTTTCTCTCCTT motifs detected in file\n",
      "\n",
      "Seq is: CATTTCAAATTA\n",
      "0 CATTTCAAATTA motifs detected in file\n",
      "\n",
      "Seq is: ATACGGAAAAAC\n",
      "0 ATACGGAAAAAC motifs detected in file\n",
      "\n",
      "Seq is: CCAACTCCACTA\n",
      "0 CCAACTCCACTA motifs detected in file\n",
      "\n",
      "Seq is: CTACCTCCACCT\n",
      "0 CTACCTCCACCT motifs detected in file\n",
      "\n",
      "Seq is: CTAAACCACGAC\n",
      "0 CTAAACCACGAC motifs detected in file\n",
      "\n",
      "Seq is: ACTTTAACAAAA\n",
      "0 ACTTTAACAAAA motifs detected in file\n",
      "\n",
      "Seq is: TCCATTATCGAA\n",
      "0 TCCATTATCGAA motifs detected in file\n",
      "\n",
      "Seq is: AAGATCAGTCCA\n",
      "0 AAGATCAGTCCA motifs detected in file\n",
      "\n",
      "Seq is: ATCCAGATCCTC\n",
      "0 ATCCAGATCCTC motifs detected in file\n",
      "\n",
      "Seq is: CTCATCCCCGTC\n",
      "0 CTCATCCCCGTC motifs detected in file\n",
      "\n",
      "Seq is: CAAAACCGAGCA\n",
      "0 CAAAACCGAGCA motifs detected in file\n",
      "\n",
      "Seq is: TTCTTCCGGATC\n",
      "0 TTCTTCCGGATC motifs detected in file\n",
      "\n",
      "Seq is: TATCCCCTAACA\n",
      "0 TATCCCCTAACA motifs detected in file\n",
      "\n",
      "Seq is: CGCCACACACTA\n",
      "0 CGCCACACACTA motifs detected in file\n",
      "\n",
      "Seq is: CGATCCCATCTT\n",
      "0 CGATCCCATCTT motifs detected in file\n",
      "\n",
      "Seq is: AGGCTCATACAA\n",
      "0 AGGCTCATACAA motifs detected in file\n",
      "\n",
      "Seq is: CCTCTTTATTTT\n",
      "0 CCTCTTTATTTT motifs detected in file\n",
      "\n",
      "Seq is: ACACGTATCACC\n",
      "0 ACACGTATCACC motifs detected in file\n",
      "\n",
      "Seq is: CCTAAAATAAAA\n",
      "0 CCTAAAATAAAA motifs detected in file\n",
      "\n",
      "Seq is: TTCAAACAACCA\n",
      "0 TTCAAACAACCA motifs detected in file\n",
      "\n",
      "Seq is: ATCCATCAACTC\n",
      "0 ATCCATCAACTC motifs detected in file\n",
      "\n",
      "Seq is: CAGATCCCGACA\n",
      "0 CAGATCCCGACA motifs detected in file\n",
      "\n",
      "Seq is: ATCTTCAGGATC\n",
      "0 ATCTTCAGGATC motifs detected in file\n",
      "\n",
      "Seq is: CTTTCACTGTCG\n",
      "0 CTTTCACTGTCG motifs detected in file\n",
      "\n",
      "Seq is: CTCCCGACACTC\n",
      "0 CTCCCGACACTC motifs detected in file\n",
      "\n",
      "Seq is: AAAATCCATCTC\n",
      "0 AAAATCCATCTC motifs detected in file\n",
      "\n",
      "Seq is: CCCCTAACAACA\n",
      "0 CCCCTAACAACA motifs detected in file\n",
      "\n",
      "Seq is: AACCTTCCCCCT\n",
      "0 AACCTTCCCCCT motifs detected in file\n",
      "\n",
      "Seq is: ACGCCCCCTTTA\n",
      "0 ACGCCCCCTTTA motifs detected in file\n",
      "\n",
      "Seq is: CAACAATCATTA\n",
      "0 CAACAATCATTA motifs detected in file\n",
      "\n",
      "Seq is: CTCTTTAGGACG\n",
      "0 CTCTTTAGGACG motifs detected in file\n",
      "\n",
      "Seq is: CTTATCCGAAGG\n",
      "0 CTTATCCGAAGG motifs detected in file\n",
      "\n",
      "Seq is: CCGACGGTCCGC\n",
      "0 CCGACGGTCCGC motifs detected in file\n",
      "\n",
      "Seq is: AAATTACCCCAA\n",
      "0 AAATTACCCCAA motifs detected in file\n",
      "\n",
      "Seq is: ATCATCCCACCC\n",
      "0 ATCATCCCACCC motifs detected in file\n",
      "\n",
      "Seq is: CAATAAAAGCAA\n",
      "0 CAATAAAAGCAA motifs detected in file\n",
      "\n",
      "Seq is: TTACAATCATTC\n",
      "0 TTACAATCATTC motifs detected in file\n",
      "\n",
      "Seq is: CCACCCCTTCGC\n",
      "0 CCACCCCTTCGC motifs detected in file\n",
      "\n",
      "Seq is: TCCAACTGGACA\n",
      "0 TCCAACTGGACA motifs detected in file\n",
      "\n",
      "Seq is: ATCCCACTCTTC\n",
      "0 ATCCCACTCTTC motifs detected in file\n",
      "\n",
      "Seq is: TTAATAAGCTCA\n",
      "0 TTAATAAGCTCA motifs detected in file\n",
      "\n",
      "Seq is: CGATATCCGCGA\n",
      "0 CGATATCCGCGA motifs detected in file\n",
      "\n",
      "Seq is: ACAGTCCCTCCA\n",
      "0 ACAGTCCCTCCA motifs detected in file\n",
      "\n",
      "Seq is: TCGCTACTCTCA\n",
      "0 TCGCTACTCTCA motifs detected in file\n",
      "\n",
      "Seq is: TTATCTATCCCG\n",
      "0 TTATCTATCCCG motifs detected in file\n",
      "\n",
      "Seq is: AGTCCGCAAACA\n",
      "0 AGTCCGCAAACA motifs detected in file\n",
      "\n",
      "Seq is: CATACTATACAA\n",
      "0 CATACTATACAA motifs detected in file\n",
      "\n",
      "Seq is: ACGCCAAAGCTA\n",
      "0 ACGCCAAAGCTA motifs detected in file\n",
      "\n",
      "Seq is: CTACACCAGAAA\n",
      "0 CTACACCAGAAA motifs detected in file\n",
      "\n",
      "Seq is: CCTCATCTTTGG\n",
      "0 CCTCATCTTTGG motifs detected in file\n",
      "\n",
      "Seq is: ACACTCCTTTCC\n",
      "0 ACACTCCTTTCC motifs detected in file\n",
      "\n",
      "Seq is: ATAAGGACGATT\n",
      "0 ATAAGGACGATT motifs detected in file\n",
      "\n",
      "Seq is: ACGATCCGACGG\n",
      "0 ACGATCCGACGG motifs detected in file\n",
      "\n",
      "Seq is: TTACCACTCCCT\n",
      "0 TTACCACTCCCT motifs detected in file\n",
      "\n",
      "Seq is: CCAACTTCACTC\n",
      "0 CCAACTTCACTC motifs detected in file\n",
      "\n",
      "Seq is: TCCATACCAAAA\n",
      "0 TCCATACCAAAA motifs detected in file\n",
      "\n",
      "Seq is: CTCATCAGGAGG\n",
      "0 CTCATCAGGAGG motifs detected in file\n",
      "\n",
      "Seq is: ATAAAAAGCTTA\n",
      "0 ATAAAAAGCTTA motifs detected in file\n",
      "\n",
      "Seq is: TAGAACAAGTAA\n",
      "0 TAGAACAAGTAA motifs detected in file\n",
      "\n",
      "Seq is: CAATTCCAACAC\n",
      "0 CAATTCCAACAC motifs detected in file\n",
      "\n",
      "Seq is: CTACTACCTGGG\n",
      "0 CTACTACCTGGG motifs detected in file\n",
      "\n",
      "Seq is: CCTCGTGTATTT\n",
      "0 CCTCGTGTATTT motifs detected in file\n",
      "\n",
      "Seq is: ATCATACCGAAA\n",
      "0 ATCATACCGAAA motifs detected in file\n",
      "\n",
      "Seq is: CTTCTTCCGCAC\n",
      "0 CTTCTTCCGCAC motifs detected in file\n",
      "\n",
      "Seq is: CCATGTCTACAC\n",
      "0 CCATGTCTACAC motifs detected in file\n",
      "\n",
      "Seq is: GTCCTTACGATC\n",
      "0 GTCCTTACGATC motifs detected in file\n",
      "\n",
      "Seq is: CACACCAACCAA\n",
      "0 CACACCAACCAA motifs detected in file\n",
      "\n",
      "Seq is: CTACGGCTAACA\n",
      "0 CTACGGCTAACA motifs detected in file\n",
      "\n",
      "Seq is: TTGATAATACTC\n",
      "0 TTGATAATACTC motifs detected in file\n",
      "\n",
      "Seq is: AGGATCAACTGA\n",
      "0 AGGATCAACTGA motifs detected in file\n",
      "\n",
      "Seq is: ACTTCACCTCTT\n",
      "0 ACTTCACCTCTT motifs detected in file\n",
      "\n",
      "Seq is: CCGATGCTAAAA\n",
      "0 CCGATGCTAAAA motifs detected in file\n",
      "\n",
      "Seq is: AAACTACAGTAA\n",
      "0 AAACTACAGTAA motifs detected in file\n",
      "\n",
      "Seq is: AACACATTCCCG\n",
      "0 AACACATTCCCG motifs detected in file\n",
      "\n",
      "Seq is: CGTCTGACGACG\n",
      "0 CGTCTGACGACG motifs detected in file\n",
      "\n",
      "Seq is: CTTGTCTCACCA\n",
      "0 CTTGTCTCACCA motifs detected in file\n",
      "\n",
      "Seq is: CTATCGCGGATT\n",
      "0 CTATCGCGGATT motifs detected in file\n",
      "\n",
      "Seq is: CGTATCACAATC\n",
      "0 CGTATCACAATC motifs detected in file\n",
      "\n",
      "Seq is: TCAAACCTAATC\n",
      "0 TCAAACCTAATC motifs detected in file\n",
      "\n",
      "Seq is: ACCCGATAAAAG\n",
      "0 ACCCGATAAAAG motifs detected in file\n",
      "\n",
      "Seq is: ACGATCCCCCAC\n",
      "0 ACGATCCCCCAC motifs detected in file\n",
      "\n",
      "Seq is: CTTCGCCTACTC\n",
      "0 CTTCGCCTACTC motifs detected in file\n",
      "\n",
      "Seq is: ATTCTTCTAAGG\n",
      "0 ATTCTTCTAAGG motifs detected in file\n",
      "\n",
      "Seq is: ACGATGTTGATA\n",
      "0 ACGATGTTGATA motifs detected in file\n",
      "\n",
      "Seq is: TCACGAACTCAC\n",
      "0 TCACGAACTCAC motifs detected in file\n",
      "\n",
      "Seq is: CTCATCCCGACT\n",
      "0 CTCATCCCGACT motifs detected in file\n",
      "\n",
      "Seq is: CTGACCCCACCC\n",
      "0 CTGACCCCACCC motifs detected in file\n",
      "\n",
      "Seq is: CGGAAGATACCG\n",
      "0 CGGAAGATACCG motifs detected in file\n",
      "\n",
      "Seq is: ACTCCCCTCAGT\n",
      "0 ACTCCCCTCAGT motifs detected in file\n",
      "\n",
      "Seq is: CCCCACCTCTGG\n",
      "0 CCCCACCTCTGG motifs detected in file\n",
      "\n",
      "Seq is: ATTAACCCGACT\n",
      "0 ATTAACCCGACT motifs detected in file\n",
      "\n",
      "Seq is: AAACGGACGACG\n",
      "0 AAACGGACGACG motifs detected in file\n",
      "\n",
      "Seq is: ACGATCCTCCTC\n",
      "0 ACGATCCTCCTC motifs detected in file\n",
      "\n",
      "Seq is: CGGATCATCACC\n",
      "0 CGGATCATCACC motifs detected in file\n",
      "\n",
      "Seq is: CCACTCCGAACC\n",
      "0 CCACTCCGAACC motifs detected in file\n",
      "\n",
      "Seq is: TAATCGCAAATC\n",
      "0 TAATCGCAAATC motifs detected in file\n",
      "\n",
      "Seq is: TTCCCGATAAAC\n",
      "0 TTCCCGATAAAC motifs detected in file\n",
      "\n",
      "Seq is: AACACCCAGAGT\n",
      "0 AACACCCAGAGT motifs detected in file\n",
      "\n",
      "Seq is: ACTCAACCTCAC\n",
      "0 ACTCAACCTCAC motifs detected in file\n",
      "\n",
      "Seq is: ATCCTATTTAGT\n",
      "0 ATCCTATTTAGT motifs detected in file\n",
      "\n",
      "Seq is: ACGATTCCGCTC\n",
      "0 ACGATTCCGCTC motifs detected in file\n",
      "\n",
      "Seq is: CTACTTTGGCTA\n",
      "0 CTACTTTGGCTA motifs detected in file\n",
      "\n",
      "Seq is: TAGCGGCTATGC\n",
      "0 TAGCGGCTATGC motifs detected in file\n",
      "\n",
      "Seq is: CAACATATCCTA\n",
      "0 CAACATATCCTA motifs detected in file\n",
      "\n",
      "Seq is: ACGAACTTACCG\n",
      "0 ACGAACTTACCG motifs detected in file\n",
      "\n",
      "Seq is: ACGACACAACCG\n",
      "0 ACGACACAACCG motifs detected in file\n",
      "\n",
      "Seq is: TAGACGAGCCCG\n",
      "0 TAGACGAGCCCG motifs detected in file\n",
      "\n",
      "Seq is: CATAGTAGTTGA\n",
      "0 CATAGTAGTTGA motifs detected in file\n",
      "\n",
      "Seq is: CTACTCGTTCTT\n",
      "0 CTACTCGTTCTT motifs detected in file\n",
      "\n",
      "Seq is: CCTATATTAAGG\n",
      "0 CCTATATTAAGG motifs detected in file\n",
      "\n",
      "Seq is: ATGCGGGAGAGT\n",
      "0 ATGCGGGAGAGT motifs detected in file\n",
      "\n",
      "Seq is: CGGCTACTTGTC\n",
      "0 CGGCTACTTGTC motifs detected in file\n",
      "\n",
      "Seq is: AACCTTATCCAA\n",
      "0 AACCTTATCCAA motifs detected in file\n",
      "\n",
      "Seq is: CAATCGCCATTG\n",
      "0 CAATCGCCATTG motifs detected in file\n",
      "\n",
      "Seq is: ATTCTAATCATA\n",
      "0 ATTCTAATCATA motifs detected in file\n",
      "\n",
      "Seq is: CCATTAATCCTT\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0 CCATTAATCCTT motifs detected in file\n",
      "\n",
      "Seq is: CTACTCATCTAT\n",
      "0 CTACTCATCTAT motifs detected in file\n",
      "\n",
      "Seq is: AAACTTCTTGAC\n",
      "0 AAACTTCTTGAC motifs detected in file\n",
      "\n",
      "Seq is: TAATTACAAACC\n",
      "0 TAATTACAAACC motifs detected in file\n",
      "\n",
      "Seq is: CAGACTCAAATA\n",
      "0 CAGACTCAAATA motifs detected in file\n",
      "\n",
      "Seq is: TATAGTGTCACT\n",
      "0 TATAGTGTCACT motifs detected in file\n",
      "\n",
      "Seq is: CTTAAGATACTA\n",
      "0 CTTAAGATACTA motifs detected in file\n",
      "\n",
      "Seq is: CCTGGTACATTG\n",
      "0 CCTGGTACATTG motifs detected in file\n",
      "\n",
      "Seq is: CTTGTTACGATG\n",
      "0 CTTGTTACGATG motifs detected in file\n",
      "\n",
      "Seq is: TCCCTACTTATA\n",
      "0 TCCCTACTTATA motifs detected in file\n",
      "\n",
      "Seq is: ATACCTCTTCAT\n",
      "0 ATACCTCTTCAT motifs detected in file\n",
      "\n",
      "Seq is: CAGATCAAACTT\n",
      "0 CAGATCAAACTT motifs detected in file\n",
      "\n",
      "Seq is: CGAATCCCTCTC\n",
      "0 CGAATCCCTCTC motifs detected in file\n",
      "\n",
      "Seq is: GACCCAACAAAC\n",
      "0 GACCCAACAAAC motifs detected in file\n",
      "\n",
      "Seq is: CCAACACCGAGT\n",
      "0 CCAACACCGAGT motifs detected in file\n",
      "\n",
      "Seq is: CACAGTCAAGCT\n",
      "0 CACAGTCAAGCT motifs detected in file\n",
      "\n",
      "Seq is: CTATTATTGAAG\n",
      "0 CTATTATTGAAG motifs detected in file\n",
      "\n",
      "Seq is: AACCAGATCCGT\n",
      "0 AACCAGATCCGT motifs detected in file\n",
      "\n",
      "Seq is: GTACTTATAACA\n",
      "0 GTACTTATAACA motifs detected in file\n",
      "\n",
      "Seq is: ACACCCCTTCTC\n",
      "0 ACACCCCTTCTC motifs detected in file\n",
      "\n",
      "Seq is: TGACTATTATTT\n",
      "0 TGACTATTATTT motifs detected in file\n",
      "\n",
      "Seq is: CCACCTACGCTG\n",
      "0 CCACCTACGCTG motifs detected in file\n",
      "\n",
      "Seq is: TTACCTAAAATG\n",
      "0 TTACCTAAAATG motifs detected in file\n",
      "\n",
      "Seq is: TATTACCCCCAT\n",
      "0 TATTACCCCCAT motifs detected in file\n",
      "\n",
      "Seq is: ACTCAAAAAAAA\n",
      "0 ACTCAAAAAAAA motifs detected in file\n",
      "\n",
      "Seq is: CAATCCCAATTG\n",
      "0 CAATCCCAATTG motifs detected in file\n",
      "\n",
      "Seq is: CTCCCACCATTC\n",
      "0 CTCCCACCATTC motifs detected in file\n",
      "\n",
      "Seq is: CTACCCCCCTTA\n",
      "0 CTACCCCCCTTA motifs detected in file\n",
      "\n",
      "Seq is: ATCCCTTCAGTC\n",
      "0 ATCCCTTCAGTC motifs detected in file\n",
      "\n",
      "Seq is: CCAATCCCACCA\n",
      "0 CCAATCCCACCA motifs detected in file\n",
      "\n",
      "Seq is: ATATCATCACAC\n",
      "0 ATATCATCACAC motifs detected in file\n",
      "\n",
      "Seq is: GTACCAATAATG\n",
      "0 GTACCAATAATG motifs detected in file\n",
      "\n",
      "Seq is: AAAATAAACCTC\n",
      "0 AAAATAAACCTC motifs detected in file\n",
      "\n",
      "Seq is: AACGTGGTTTTA\n",
      "0 AACGTGGTTTTA motifs detected in file\n",
      "\n",
      "Seq is: AAAACCCCGGAT\n",
      "0 AAAACCCCGGAT motifs detected in file\n",
      "\n",
      "Seq is: TCTGGTTACCTA\n",
      "0 TCTGGTTACCTA motifs detected in file\n",
      "\n",
      "Seq is: TCGACAATAGGA\n",
      "0 TCGACAATAGGA motifs detected in file\n",
      "\n",
      "Seq is: AATTTTCAACTC\n",
      "0 AATTTTCAACTC motifs detected in file\n",
      "\n",
      "Seq is: TTTCGGTACAAC\n",
      "0 TTTCGGTACAAC motifs detected in file\n",
      "\n",
      "Seq is: AATCTGTACCCC\n",
      "0 AATCTGTACCCC motifs detected in file\n",
      "\n",
      "Seq is: CGTACTCAATCG\n",
      "0 CGTACTCAATCG motifs detected in file\n",
      "\n",
      "Seq is: TCAAGAACACTC\n",
      "0 TCAAGAACACTC motifs detected in file\n",
      "\n",
      "Seq is: GAAAGAGCCATT\n",
      "0 GAAAGAGCCATT motifs detected in file\n",
      "\n",
      "Seq is: TATTCCCCAGCA\n",
      "0 TATTCCCCAGCA motifs detected in file\n",
      "\n",
      "Seq is: TTCGGAGACAAC\n",
      "0 TTCGGAGACAAC motifs detected in file\n",
      "\n",
      "Seq is: AGTCTAAGTGTT\n",
      "0 AGTCTAAGTGTT motifs detected in file\n",
      "\n",
      "Seq is: AGACTACAAAAC\n",
      "0 AGACTACAAAAC motifs detected in file\n",
      "\n",
      "Seq is: CAATTTGATATA\n",
      "0 CAATTTGATATA motifs detected in file\n",
      "\n",
      "Seq is: AATGTTCTCCTT\n",
      "0 AATGTTCTCCTT motifs detected in file\n",
      "\n",
      "Seq is: TTGGGCCATTAC\n",
      "0 TTGGGCCATTAC motifs detected in file\n",
      "\n",
      "Seq is: TACAGCCCCAAC\n",
      "0 TACAGCCCCAAC motifs detected in file\n",
      "\n",
      "Seq is: TCCCTATCCTCC\n",
      "0 TCCCTATCCTCC motifs detected in file\n",
      "\n",
      "Seq is: TCTTACCCCCTA\n",
      "0 TCTTACCCCCTA motifs detected in file\n",
      "\n",
      "Seq is: TCCACATACTTG\n",
      "0 TCCACATACTTG motifs detected in file\n",
      "\n",
      "Seq is: TACTCCCACAAA\n",
      "0 TACTCCCACAAA motifs detected in file\n",
      "\n",
      "Seq is: AGAGCACACTTA\n",
      "0 AGAGCACACTTA motifs detected in file\n",
      "\n",
      "Seq is: CTCCCAAAATAC\n",
      "0 CTCCCAAAATAC motifs detected in file\n",
      "\n",
      "Seq is: AACAATTACACC\n",
      "0 AACAATTACACC motifs detected in file\n",
      "\n",
      "Seq is: ACCCACTCACTC\n",
      "0 ACCCACTCACTC motifs detected in file\n",
      "\n",
      "Seq is: GGGGTAACACAA\n",
      "0 GGGGTAACACAA motifs detected in file\n",
      "\n",
      "Seq is: CACCATTTATAC\n",
      "0 CACCATTTATAC motifs detected in file\n",
      "\n",
      "Seq is: ATCTCCCTCATA\n",
      "0 ATCTCCCTCATA motifs detected in file\n",
      "\n",
      "Seq is: TCCCGACACTGA\n",
      "0 TCCCGACACTGA motifs detected in file\n",
      "\n",
      "Seq is: TCATACAACTCA\n",
      "0 TCATACAACTCA motifs detected in file\n",
      "\n",
      "Seq is: GGACATTCATCC\n",
      "0 GGACATTCATCC motifs detected in file\n",
      "\n",
      "Seq is: TCTCACTATAAA\n",
      "0 TCTCACTATAAA motifs detected in file\n",
      "\n",
      "Seq is: CTAGTCCTCTTG\n",
      "0 CTAGTCCTCTTG motifs detected in file\n",
      "\n",
      "Seq is: CACCAATGATCG\n",
      "0 CACCAATGATCG motifs detected in file\n",
      "\n",
      "Seq is: TGTCTCTCAAGA\n",
      "0 TGTCTCTCAAGA motifs detected in file\n",
      "\n",
      "Seq is: GGGTCATCCAAT\n",
      "0 GGGTCATCCAAT motifs detected in file\n",
      "\n",
      "Seq is: TATCACCCCCCA\n",
      "0 TATCACCCCCCA motifs detected in file\n",
      "\n",
      "Seq is: TTCCGCTCCAAT\n",
      "0 TTCCGCTCCAAT motifs detected in file\n",
      "\n",
      "Seq is: CGCTCCGAACGA\n",
      "0 CGCTCCGAACGA motifs detected in file\n",
      "\n",
      "Seq is: TCTTCAGTAGTT\n",
      "0 TCTTCAGTAGTT motifs detected in file\n",
      "\n",
      "Seq is: TTTCGATAATCA\n",
      "0 TTTCGATAATCA motifs detected in file\n",
      "\n",
      "Seq is: CCCTCATCTCAA\n",
      "0 CCCTCATCTCAA motifs detected in file\n",
      "\n",
      "Seq is: ACTTCAGGAACT\n",
      "0 ACTTCAGGAACT motifs detected in file\n",
      "\n",
      "Seq is: CTCTCCTAATAC\n",
      "0 CTCTCCTAATAC motifs detected in file\n",
      "\n",
      "Seq is: TACGCTGACACT\n",
      "0 TACGCTGACACT motifs detected in file\n",
      "\n",
      "Seq is: CACGCAAGCATC\n",
      "0 CACGCAAGCATC motifs detected in file\n",
      "\n",
      "Seq is: AAACTCAAACGA\n",
      "0 AAACTCAAACGA motifs detected in file\n",
      "\n",
      "Seq is: TCCGTCTTATCA\n",
      "0 TCCGTCTTATCA motifs detected in file\n",
      "\n",
      "Seq is: TTACTCCTACAT\n",
      "0 TTACTCCTACAT motifs detected in file\n",
      "\n",
      "Seq is: TCGGGCACCCGC\n",
      "0 TCGGGCACCCGC motifs detected in file\n",
      "\n",
      "Seq is: TAATACTCTTAC\n",
      "0 TAATACTCTTAC motifs detected in file\n",
      "\n",
      "Seq is: TGACGCGGCCAC\n",
      "0 TGACGCGGCCAC motifs detected in file\n",
      "\n",
      "Seq is: TTCGAAGTCCCC\n",
      "0 TTCGAAGTCCCC motifs detected in file\n",
      "\n",
      "Seq is: CAAACCTACTCT\n",
      "0 CAAACCTACTCT motifs detected in file\n",
      "\n",
      "Seq is: TACCGACAATGA\n",
      "0 TACCGACAATGA motifs detected in file\n",
      "\n",
      "Seq is: TGCTCCCGATAC\n",
      "0 TGCTCCCGATAC motifs detected in file\n",
      "\n",
      "Seq is: ACCGACTAACTT\n",
      "0 ACCGACTAACTT motifs detected in file\n",
      "\n",
      "Seq is: CTCATACGTTAC\n",
      "0 CTCATACGTTAC motifs detected in file\n",
      "\n",
      "Seq is: TCGCTAAAATCC\n",
      "0 TCGCTAAAATCC motifs detected in file\n",
      "\n",
      "Seq is: AGACAAACAGCA\n",
      "0 AGACAAACAGCA motifs detected in file\n",
      "\n",
      "Seq is: TCCGTCTACCTC\n",
      "0 TCCGTCTACCTC motifs detected in file\n",
      "\n",
      "Seq is: GAACAATATTAA\n",
      "0 GAACAATATTAA motifs detected in file\n",
      "\n",
      "Seq is: TACGGAGTATCC\n",
      "0 TACGGAGTATCC motifs detected in file\n",
      "\n",
      "Seq is: CTATCATGTTCC\n",
      "0 CTATCATGTTCC motifs detected in file\n",
      "\n",
      "Seq is: TTCCCCATCCGA\n",
      "0 TTCCCCATCCGA motifs detected in file\n",
      "\n",
      "Seq is: TACACAAAACAG\n",
      "0 TACACAAAACAG motifs detected in file\n",
      "\n",
      "Seq is: TCCCCCAACTCT\n",
      "0 TCCCCCAACTCT motifs detected in file\n",
      "\n",
      "Seq is: TACTCACACTCC\n",
      "0 TACTCACACTCC motifs detected in file\n",
      "\n",
      "Seq is: GCATAATATGGT\n",
      "0 GCATAATATGGT motifs detected in file\n",
      "\n",
      "Seq is: ATCGGCGGATCA\n",
      "0 ATCGGCGGATCA motifs detected in file\n",
      "\n",
      "Seq is: AAATTCTCATGA\n",
      "0 AAATTCTCATGA motifs detected in file\n",
      "\n",
      "Seq is: CGCCGTTCATGA\n",
      "0 CGCCGTTCATGA motifs detected in file\n",
      "\n",
      "Seq is: TAACTGGGTCGC\n",
      "0 TAACTGGGTCGC motifs detected in file\n",
      "\n",
      "Seq is: TACCCCCGAAGC\n",
      "0 TACCCCCGAAGC motifs detected in file\n",
      "\n",
      "Seq is: TGTACCCGAAAT\n",
      "0 TGTACCCGAAAT motifs detected in file\n",
      "\n",
      "Seq is: CCCTCAGTATTC\n",
      "0 CCCTCAGTATTC motifs detected in file\n",
      "\n",
      "Seq is: ACCTCAGGCATT\n",
      "0 ACCTCAGGCATT motifs detected in file\n",
      "\n",
      "Seq is: TCTCCATAGAAA\n",
      "0 TCTCCATAGAAA motifs detected in file\n",
      "\n",
      "Seq is: TGATATTTTCGA\n",
      "0 TGATATTTTCGA motifs detected in file\n",
      "\n",
      "Seq is: TAACACATTCGA\n",
      "0 TAACACATTCGA motifs detected in file\n",
      "\n",
      "Seq is: TCAGTATAATTC\n",
      "0 TCAGTATAATTC motifs detected in file\n",
      "\n",
      "Seq is: AACAACCGAGTC\n",
      "0 AACAACCGAGTC motifs detected in file\n",
      "\n",
      "Seq is: CTCAAACTATCA\n",
      "0 CTCAAACTATCA motifs detected in file\n",
      "\n",
      "Seq is: AGAACCCGTCAC\n",
      "0 AGAACCCGTCAC motifs detected in file\n",
      "\n",
      "Seq is: TCTCACTCATCA\n",
      "0 TCTCACTCATCA motifs detected in file\n",
      "\n",
      "Seq is: TCTTACAAATCA\n",
      "0 TCTTACAAATCA motifs detected in file\n",
      "\n",
      "Seq is: CTCGGATCCCAC\n",
      "0 CTCGGATCCCAC motifs detected in file\n",
      "\n",
      "Seq is: AACACTCACATT\n",
      "0 AACACTCACATT motifs detected in file\n",
      "\n",
      "Seq is: TAATCACGCTAC\n",
      "0 TAATCACGCTAC motifs detected in file\n",
      "\n",
      "Seq is: TCATCCCCTTCC\n",
      "0 TCATCCCCTTCC motifs detected in file\n",
      "\n",
      "Seq is: CTCGGATGATCC\n",
      "0 CTCGGATGATCC motifs detected in file\n",
      "\n",
      "Seq is: CACATCTTATCC\n",
      "0 CACATCTTATCC motifs detected in file\n",
      "\n",
      "Seq is: TTCATACACGGA\n",
      "0 TTCATACACGGA motifs detected in file\n",
      "\n",
      "Seq is: CGCAAGTCCGCA\n",
      "0 CGCAAGTCCGCA motifs detected in file\n",
      "\n",
      "Seq is: AGACCGACCAAC\n",
      "0 AGACCGACCAAC motifs detected in file\n",
      "\n",
      "Seq is: GGAGTAGCCCAC\n",
      "0 GGAGTAGCCCAC motifs detected in file\n",
      "\n",
      "Seq is: TACTATCGGTTC\n",
      "0 TACTATCGGTTC motifs detected in file\n",
      "\n",
      "Seq is: CACCCCTATTCA\n",
      "0 CACCCCTATTCA motifs detected in file\n",
      "\n",
      "Seq is: CACCAAAGCTTC\n",
      "0 CACCAAAGCTTC motifs detected in file\n",
      "\n",
      "Seq is: TTCTATATTTTA\n",
      "0 TTCTATATTTTA motifs detected in file\n",
      "\n",
      "Seq is: TACTAATCAAGA\n",
      "0 TACTAATCAAGA motifs detected in file\n",
      "\n",
      "Seq is: CACCGACACTTA\n",
      "0 CACCGACACTTA motifs detected in file\n",
      "\n",
      "Seq is: CAATATTAATTC\n",
      "0 CAATATTAATTC motifs detected in file\n",
      "\n",
      "Seq is: CTCTAAACATCC\n",
      "0 CTCTAAACATCC motifs detected in file\n",
      "\n",
      "Seq is: CTCTAATCATAT\n",
      "0 CTCTAATCATAT motifs detected in file\n",
      "\n",
      "Seq is: CTCTCCATCTCA\n",
      "0 CTCTCCATCTCA motifs detected in file\n",
      "\n",
      "Seq is: AAAAAAGCACTA\n",
      "0 AAAAAAGCACTA motifs detected in file\n",
      "\n",
      "Seq is: TCACCAAGTGAC\n",
      "0 TCACCAAGTGAC motifs detected in file\n",
      "\n",
      "Seq is: CTATTCACCGCA\n",
      "0 CTATTCACCGCA motifs detected in file\n",
      "\n",
      "Seq is: ACCGAACGGTAC\n",
      "0 ACCGAACGGTAC motifs detected in file\n",
      "\n",
      "Seq is: TAACACTCCCCA\n",
      "0 TAACACTCCCCA motifs detected in file\n",
      "\n",
      "Seq is: TCCTCAGTCCAT\n",
      "0 TCCTCAGTCCAT motifs detected in file\n",
      "\n",
      "Seq is: CAATCATAATCC\n",
      "0 CAATCATAATCC motifs detected in file\n",
      "\n",
      "Seq is: TCCCCCCAACAA\n",
      "0 TCCCCCCAACAA motifs detected in file\n",
      "\n",
      "Seq is: TCCCCCAGCCTC\n",
      "0 TCCCCCAGCCTC motifs detected in file\n",
      "\n",
      "Seq is: CTTTTCCAACCC\n",
      "0 CTTTTCCAACCC motifs detected in file\n",
      "\n",
      "Seq is: CTTGTCGATTTA\n",
      "0 CTTGTCGATTTA motifs detected in file\n",
      "\n",
      "Seq is: TCCAACAACAAC\n",
      "0 TCCAACAACAAC motifs detected in file\n",
      "\n",
      "Seq is: TAAACCAATTTT\n",
      "0 TAAACCAATTTT motifs detected in file\n",
      "\n",
      "Seq is: TTATCATCTCCC\n",
      "0 TTATCATCTCCC motifs detected in file\n",
      "\n",
      "Seq is: TACTACGATTAT\n",
      "0 TACTACGATTAT motifs detected in file\n",
      "\n",
      "Seq is: TAATCCGACACC\n",
      "0 TAATCCGACACC motifs detected in file\n",
      "\n",
      "Seq is: CACCAACACAAC\n",
      "0 CACCAACACAAC motifs detected in file\n",
      "\n",
      "Seq is: TAAGTTTAATAC\n",
      "0 TAAGTTTAATAC motifs detected in file\n",
      "\n",
      "Seq is: ACAAAAAACCTT\n",
      "0 ACAAAAAACCTT motifs detected in file\n",
      "\n",
      "Seq is: TCAGTACAGTCA\n",
      "0 TCAGTACAGTCA motifs detected in file\n",
      "\n",
      "Seq is: CCATCATTATAT\n",
      "0 CCATCATTATAT motifs detected in file\n",
      "\n",
      "Seq is: TAACAACCCTGC\n",
      "0 TAACAACCCTGC motifs detected in file\n",
      "\n",
      "Seq is: TAATCAAAATCG\n",
      "0 TAATCAAAATCG motifs detected in file\n",
      "\n",
      "Seq is: TAACCTCATCCA\n",
      "0 TAACCTCATCCA motifs detected in file\n",
      "\n",
      "Seq is: AATCCAATACAT\n",
      "0 AATCCAATACAT motifs detected in file\n",
      "\n",
      "Seq is: GCATCAGATCCG\n",
      "0 GCATCAGATCCG motifs detected in file\n",
      "\n",
      "Seq is: GTATACACAACC\n",
      "0 GTATACACAACC motifs detected in file\n",
      "\n",
      "Seq is: TCTAACTCTGAT\n",
      "0 TCTAACTCTGAT motifs detected in file\n",
      "\n",
      "Seq is: CATCCCGATCCG\n",
      "0 CATCCCGATCCG motifs detected in file\n",
      "\n",
      "Seq is: GGTGGCGACGAA\n",
      "0 GGTGGCGACGAA motifs detected in file\n",
      "\n",
      "Seq is: GCGTCCGCCGTT\n",
      "0 GCGTCCGCCGTT motifs detected in file\n",
      "\n",
      "Seq is: TCTGATCATACC\n",
      "0 TCTGATCATACC motifs detected in file\n",
      "\n",
      "Seq is: GTTATCCGTGTT\n",
      "0 GTTATCCGTGTT motifs detected in file\n",
      "\n",
      "Seq is: ATAACCGATTCT\n",
      "0 ATAACCGATTCT motifs detected in file\n",
      "\n",
      "Seq is: CCCTCACCCACA\n",
      "0 CCCTCACCCACA motifs detected in file\n",
      "\n",
      "Seq is: ACTCCCAATACT\n",
      "0 ACTCCCAATACT motifs detected in file\n",
      "\n",
      "Seq is: CTCATCGATATT\n",
      "0 CTCATCGATATT motifs detected in file\n",
      "\n",
      "Seq is: ACTTGTCGCTAT\n",
      "0 ACTTGTCGCTAT motifs detected in file\n",
      "\n",
      "Seq is: CATAATAAGGAA\n",
      "0 CATAATAAGGAA motifs detected in file\n",
      "\n",
      "Seq is: GATCCGTATCAT\n",
      "0 GATCCGTATCAT motifs detected in file\n",
      "\n",
      "Seq is: CCCTCCTATACT\n",
      "0 CCCTCCTATACT motifs detected in file\n",
      "\n",
      "Seq is: CCACACTCGCTA\n",
      "0 CCACACTCGCTA motifs detected in file\n",
      "\n",
      "Seq is: TATGATCATAAG\n",
      "0 TATGATCATAAG motifs detected in file\n",
      "\n",
      "Seq is: GATCTTCACTCT\n",
      "0 GATCTTCACTCT motifs detected in file\n",
      "\n",
      "Seq is: ACCATTTACGAT\n",
      "0 ACCATTTACGAT motifs detected in file\n",
      "\n",
      "Seq is: CATATTATAAAT\n",
      "0 CATATTATAAAT motifs detected in file\n",
      "\n",
      "Seq is: ACATTTACTCCC\n",
      "0 ACATTTACTCCC motifs detected in file\n",
      "\n",
      "Seq is: CGTAAACCATTT\n",
      "0 CGTAAACCATTT motifs detected in file\n",
      "\n",
      "Seq is: ATACTAATAGTA\n",
      "0 ATACTAATAGTA motifs detected in file\n",
      "\n",
      "Seq is: TTAAATTACTCA\n",
      "0 TTAAATTACTCA motifs detected in file\n",
      "\n",
      "Seq is: GCTTTAGTAAGC\n",
      "0 GCTTTAGTAAGC motifs detected in file\n",
      "\n",
      "Seq is: AAAACAAATTTG\n",
      "0 AAAACAAATTTG motifs detected in file\n",
      "\n",
      "Seq is: ATATATGGTTAA\n",
      "0 ATATATGGTTAA motifs detected in file\n",
      "\n",
      "Seq is: GCCAAGTCAGAT\n",
      "0 GCCAAGTCAGAT motifs detected in file\n",
      "\n",
      "Seq is: TAGGAAAAACAT\n",
      "0 TAGGAAAAACAT motifs detected in file\n",
      "\n",
      "Seq is: CAGTGAGTATCC\n",
      "0 CAGTGAGTATCC motifs detected in file\n",
      "\n",
      "Seq is: GGTCTGAATCCC\n",
      "0 GGTCTGAATCCC motifs detected in file\n",
      "\n",
      "Seq is: GATCCTACTACT\n",
      "0 GATCCTACTACT motifs detected in file\n",
      "\n",
      "Seq is: AATTATTCTCCC\n",
      "0 AATTATTCTCCC motifs detected in file\n",
      "\n",
      "Seq is: TACTGTATTGAT\n",
      "0 TACTGTATTGAT motifs detected in file\n",
      "\n",
      "Seq is: AATCACCGTCCA\n",
      "0 AATCACCGTCCA motifs detected in file\n",
      "\n",
      "Seq is: ATCAACAAACAT\n",
      "0 ATCAACAAACAT motifs detected in file\n",
      "\n",
      "Seq is: CCCGAGTACCAT\n",
      "0 CCCGAGTACCAT motifs detected in file\n",
      "\n",
      "Seq is: CCCCATTTTCCT\n",
      "0 CCCCATTTTCCT motifs detected in file\n",
      "\n",
      "Seq is: CCCGTTAAAGAT\n",
      "0 CCCGTTAAAGAT motifs detected in file\n",
      "\n",
      "Seq is: CTAGTTTATTAT\n",
      "0 CTAGTTTATTAT motifs detected in file\n",
      "\n",
      "Seq is: TCTTCCATTATC\n",
      "0 TCTTCCATTATC motifs detected in file\n",
      "\n",
      "Seq is: GATGATTCTTCT\n",
      "0 GATGATTCTTCT motifs detected in file\n",
      "\n",
      "Seq is: TAAAATACCTCT\n",
      "0 TAAAATACCTCT motifs detected in file\n",
      "\n",
      "Seq is: TTCCCTGCGCCC\n",
      "0 TTCCCTGCGCCC motifs detected in file\n",
      "\n",
      "Seq is: GCCCCCTATATC\n",
      "0 GCCCCCTATATC motifs detected in file\n",
      "\n",
      "Seq is: CCAGCTTCGGCG\n",
      "0 CCAGCTTCGGCG motifs detected in file\n",
      "\n",
      "Seq is: TGAGCATTCCCC\n",
      "0 TGAGCATTCCCC motifs detected in file\n",
      "\n",
      "Seq is: ACCTAAAAAGAT\n",
      "0 ACCTAAAAAGAT motifs detected in file\n",
      "\n",
      "Seq is: ACATCGGCAACT\n",
      "0 ACATCGGCAACT motifs detected in file\n",
      "\n",
      "Seq is: CAACACCATCAG\n",
      "0 CAACACCATCAG motifs detected in file\n",
      "\n",
      "Seq is: TTTTACATTATT\n",
      "0 TTTTACATTATT motifs detected in file\n",
      "\n",
      "Seq is: AATAATCATCAT\n",
      "0 AATAATCATCAT motifs detected in file\n",
      "\n",
      "Seq is: TCCGATCCTCCC\n",
      "0 TCCGATCCTCCC motifs detected in file\n",
      "\n",
      "Seq is: ACAACTAATAAT\n",
      "0 ACAACTAATAAT motifs detected in file\n",
      "\n",
      "Seq is: TTTAATTCCGCT\n",
      "0 TTTAATTCCGCT motifs detected in file\n",
      "\n",
      "Seq is: TCAAATAATGAG\n",
      "0 TCAAATAATGAG motifs detected in file\n",
      "\n",
      "Seq is: AAAAAACTTACA\n",
      "0 AAAAAACTTACA motifs detected in file\n",
      "\n",
      "Seq is: ACAGTTTTGATC\n",
      "0 ACAGTTTTGATC motifs detected in file\n",
      "\n",
      "Seq is: AATTAACCTTCA\n",
      "0 AATTAACCTTCA motifs detected in file\n",
      "\n",
      "Seq is: ATTGCCATGATA\n",
      "0 ATTGCCATGATA motifs detected in file\n",
      "\n",
      "Seq is: AATATGATTTTC\n",
      "0 AATATGATTTTC motifs detected in file\n",
      "\n",
      "Seq is: TCATTCTGGGAG\n",
      "0 TCATTCTGGGAG motifs detected in file\n",
      "\n",
      "Seq is: TAGTTATCTACC\n",
      "0 TAGTTATCTACC motifs detected in file\n",
      "\n",
      "Seq is: TCTGTATGTCTT\n",
      "0 TCTGTATGTCTT motifs detected in file\n",
      "\n",
      "Seq is: TATCAGTTTGGT\n",
      "0 TATCAGTTTGGT motifs detected in file\n",
      "\n",
      "Seq is: GTAGATGTTTTA\n",
      "0 GTAGATGTTTTA motifs detected in file\n",
      "\n",
      "Seq is: CGGTCATAGTCC\n",
      "0 CGGTCATAGTCC motifs detected in file\n",
      "\n",
      "Seq is: GCCGCCGAAGCT\n",
      "0 GCCGCCGAAGCT motifs detected in file\n",
      "\n",
      "Seq is: TCGGTTTCACTA\n",
      "0 TCGGTTTCACTA motifs detected in file\n",
      "\n",
      "Seq is: CAAACCTACATT\n",
      "0 CAAACCTACATT motifs detected in file\n",
      "\n",
      "Seq is: TCACTTTATAAT\n",
      "0 TCACTTTATAAT motifs detected in file\n",
      "\n",
      "Seq is: CAACCGCCTACT\n",
      "0 CAACCGCCTACT motifs detected in file\n",
      "\n",
      "Seq is: TCGTCTATCACT\n",
      "0 TCGTCTATCACT motifs detected in file\n",
      "\n",
      "Seq is: CCTTTCAACTCG\n",
      "0 CCTTTCAACTCG motifs detected in file\n",
      "\n",
      "Seq is: GTTATTACTGCA\n",
      "0 GTTATTACTGCA motifs detected in file\n",
      "\n",
      "Seq is: CTTCAGGCACCT\n",
      "0 CTTCAGGCACCT motifs detected in file\n",
      "\n",
      "Seq is: TCGGACACCGAT\n",
      "0 TCGGACACCGAT motifs detected in file\n",
      "\n",
      "Seq is: GTTTTTTACAAC\n",
      "0 GTTTTTTACAAC motifs detected in file\n",
      "\n",
      "Seq is: TCGGCATCTACG\n",
      "0 TCGGCATCTACG motifs detected in file\n",
      "\n",
      "Seq is: GCAGCCTTTACC\n",
      "0 GCAGCCTTTACC motifs detected in file\n",
      "\n",
      "Seq is: ACTTCCCTCTGA\n",
      "0 ACTTCCCTCTGA motifs detected in file\n",
      "\n",
      "Seq is: GCTTCATGAGAC\n",
      "0 GCTTCATGAGAC motifs detected in file\n",
      "\n",
      "Seq is: TCCGAACATCCT\n",
      "0 TCCGAACATCCT motifs detected in file\n",
      "\n",
      "Seq is: CCCATTTTATCT\n",
      "0 CCCATTTTATCT motifs detected in file\n",
      "\n",
      "Seq is: CTGGGTCATTTT\n",
      "0 CTGGGTCATTTT motifs detected in file\n",
      "\n",
      "Seq is: AACATTATTCGT\n",
      "0 AACATTATTCGT motifs detected in file\n",
      "\n",
      "Seq is: CACGAACTTAAT\n",
      "0 CACGAACTTAAT motifs detected in file\n",
      "\n",
      "Seq is: AAACCAAAGCCA\n",
      "0 AAACCAAAGCCA motifs detected in file\n",
      "\n",
      "Seq is: ACAAAAGATAAT\n",
      "0 ACAAAAGATAAT motifs detected in file\n",
      "\n",
      "Seq is: CTGATACCACTC\n",
      "0 CTGATACCACTC motifs detected in file\n",
      "\n",
      "Seq is: GAGTCCACTAAC\n",
      "0 GAGTCCACTAAC motifs detected in file\n",
      "\n",
      "Seq is: TATGAGGACTAC\n",
      "0 TATGAGGACTAC motifs detected in file\n",
      "\n",
      "Seq is: GCTCATTATGCC\n",
      "0 GCTCATTATGCC motifs detected in file\n",
      "\n",
      "Seq is: TACACAAATCCT\n",
      "0 TACACAAATCCT motifs detected in file\n",
      "\n",
      "Seq is: CACCCTGAAGAT\n",
      "0 CACCCTGAAGAT motifs detected in file\n",
      "\n",
      "Seq is: CCCCTAAATCGC\n",
      "0 CCCCTAAATCGC motifs detected in file\n",
      "\n",
      "Seq is: CCCACTACGGAC\n",
      "0 CCCACTACGGAC motifs detected in file\n",
      "\n",
      "Seq is: AACCCCCGGTCA\n",
      "0 AACCCCCGGTCA motifs detected in file\n",
      "\n",
      "Seq is: CGGGAGGATTAA\n",
      "0 CGGGAGGATTAA motifs detected in file\n",
      "\n",
      "Seq is: CCCCCCATCCCC\n",
      "0 CCCCCCATCCCC motifs detected in file\n",
      "\n",
      "Seq is: GATCCGACCTCA\n",
      "0 GATCCGACCTCA motifs detected in file\n",
      "\n",
      "Seq is: CCATTTTCCGAG\n",
      "0 CCATTTTCCGAG motifs detected in file\n",
      "\n",
      "Seq is: TCTTTTTAGGAC\n",
      "0 TCTTTTTAGGAC motifs detected in file\n",
      "\n",
      "Seq is: GCTTCTTTTTTT\n",
      "0 GCTTCTTTTTTT motifs detected in file\n",
      "\n",
      "Seq is: GAAGACTCCATT\n",
      "0 GAAGACTCCATT motifs detected in file\n",
      "\n",
      "Seq is: ATTTATCCTAAT\n",
      "0 ATTTATCCTAAT motifs detected in file\n",
      "\n",
      "Seq is: AGGGCATAGCTT\n",
      "0 AGGGCATAGCTT motifs detected in file\n",
      "\n",
      "Seq is: CCGGAAAAACCT\n",
      "0 CCGGAAAAACCT motifs detected in file\n",
      "\n",
      "Seq is: GTCCACCACACA\n",
      "0 GTCCACCACACA motifs detected in file\n",
      "\n",
      "Seq is: CCACCGGAACCA\n",
      "0 CCACCGGAACCA motifs detected in file\n",
      "\n",
      "Seq is: TACACGAAAGAG\n",
      "0 TACACGAAAGAG motifs detected in file\n",
      "\n",
      "Seq is: CACAATGTAGCG\n",
      "0 CACAATGTAGCG motifs detected in file\n",
      "\n",
      "Seq is: CGGTAGTAACCA\n",
      "0 CGGTAGTAACCA motifs detected in file\n",
      "\n",
      "Seq is: TATACCAATCAC\n",
      "0 TATACCAATCAC motifs detected in file\n",
      "\n",
      "Seq is: ACAACCAATCAT\n",
      "0 ACAACCAATCAT motifs detected in file\n",
      "\n",
      "Seq is: CCGGATCATACT\n",
      "0 CCGGATCATACT motifs detected in file\n",
      "\n",
      "Seq is: CCTCGCAATACC\n",
      "0 CCTCGCAATACC motifs detected in file\n",
      "\n",
      "Seq is: TCACCTTCACTT\n",
      "0 TCACCTTCACTT motifs detected in file\n",
      "\n",
      "Seq is: TAGTGTACCGAT\n",
      "0 TAGTGTACCGAT motifs detected in file\n",
      "\n",
      "Seq is: CCGGCCTCCAGT\n",
      "0 CCGGCCTCCAGT motifs detected in file\n",
      "\n",
      "Seq is: AATCCTCCCGAC\n",
      "0 AATCCTCCCGAC motifs detected in file\n",
      "\n",
      "Seq is: TCTCCCGGGGAC\n",
      "0 TCTCCCGGGGAC motifs detected in file\n",
      "\n",
      "Seq is: AATCCCCAGTAC\n",
      "0 AATCCCCAGTAC motifs detected in file\n",
      "\n",
      "Seq is: CCGACCCAAAAT\n",
      "0 CCGACCCAAAAT motifs detected in file\n",
      "\n",
      "Seq is: GATATACTATCC\n",
      "0 GATATACTATCC motifs detected in file\n",
      "\n",
      "Seq is: GTACACTAACCA\n",
      "0 GTACACTAACCA motifs detected in file\n",
      "\n",
      "Seq is: CCCAGTAATACA\n",
      "0 CCCAGTAATACA motifs detected in file\n",
      "\n",
      "Seq is: CAACAGCACGTC\n",
      "0 CAACAGCACGTC motifs detected in file\n",
      "\n",
      "Seq is: CATCTCCGAATG\n",
      "0 CATCTCCGAATG motifs detected in file\n",
      "\n",
      "Seq is: ATCTTCACACTT\n",
      "0 ATCTTCACACTT motifs detected in file\n",
      "\n",
      "Seq is: TTGCATCCGAAC\n",
      "0 TTGCATCCGAAC motifs detected in file\n",
      "\n",
      "Seq is: CTAATTCCGCTG\n",
      "0 CTAATTCCGCTG motifs detected in file\n",
      "\n",
      "Seq is: TCGCTAAAGATC\n",
      "0 TCGCTAAAGATC motifs detected in file\n",
      "\n",
      "Seq is: CTATCAGTCATC\n",
      "0 CTATCAGTCATC motifs detected in file\n",
      "\n",
      "Seq is: TTAATCTTAACA\n",
      "0 TTAATCTTAACA motifs detected in file\n",
      "\n",
      "Seq is: CTTCTACTATTC\n",
      "0 CTTCTACTATTC motifs detected in file\n",
      "\n",
      "Seq is: ACATCTCCCTTC\n",
      "0 ACATCTCCCTTC motifs detected in file\n",
      "\n",
      "Seq is: CAATTACCAACT\n",
      "0 CAATTACCAACT motifs detected in file\n",
      "\n",
      "Seq is: ATAACGATCCCA\n",
      "0 ATAACGATCCCA motifs detected in file\n",
      "\n",
      "Seq is: CCGCGAAGGTTA\n",
      "0 CCGCGAAGGTTA motifs detected in file\n",
      "\n",
      "Seq is: ATCCACGTACTC\n",
      "0 ATCCACGTACTC motifs detected in file\n",
      "\n",
      "Seq is: ATCCACCGGACG\n",
      "0 ATCCACCGGACG motifs detected in file\n",
      "\n",
      "Seq is: TCATTACAATCG\n",
      "0 TCATTACAATCG motifs detected in file\n",
      "\n",
      "Seq is: CCAATCCGCATG\n",
      "0 CCAATCCGCATG motifs detected in file\n",
      "\n",
      "Seq is: CCGGTCCCGATA\n",
      "0 CCGGTCCCGATA motifs detected in file\n",
      "\n",
      "Seq is: ACCAACTTACTC\n",
      "0 ACCAACTTACTC motifs detected in file\n",
      "\n",
      "Seq is: ATCCGACTACCA\n",
      "0 ATCCGACTACCA motifs detected in file\n",
      "\n",
      "Seq is: AAGCTATTATTG\n",
      "0 AAGCTATTATTG motifs detected in file\n",
      "\n",
      "Seq is: ATCATACCCCTA\n",
      "0 ATCATACCCCTA motifs detected in file\n",
      "\n",
      "Seq is: TTCCCCACATCC\n",
      "0 TTCCCCACATCC motifs detected in file\n",
      "\n",
      "Seq is: ACACGGAACACC\n",
      "0 ACACGGAACACC motifs detected in file\n",
      "\n",
      "Seq is: ATTCTTCACCCG\n",
      "0 ATTCTTCACCCG motifs detected in file\n",
      "\n",
      "Seq is: ATCCCGTAAAAA\n",
      "0 ATCCCGTAAAAA motifs detected in file\n",
      "\n",
      "Seq is: TTAGAATCTCGC\n",
      "0 TTAGAATCTCGC motifs detected in file\n",
      "\n",
      "Seq is: TTTCGGATATTA\n",
      "0 TTTCGGATATTA motifs detected in file\n",
      "\n",
      "Seq is: GTGACACGGGCG\n",
      "0 GTGACACGGGCG motifs detected in file\n",
      "\n",
      "Seq is: AGTATTCCCCTA\n",
      "0 AGTATTCCCCTA motifs detected in file\n",
      "\n",
      "Seq is: CCGGTACCGATC\n",
      "0 CCGGTACCGATC motifs detected in file\n",
      "\n",
      "Seq is: CAAATATCTATC\n",
      "0 CAAATATCTATC motifs detected in file\n",
      "\n",
      "Seq is: AACCCACCAACC\n",
      "0 AACCCACCAACC motifs detected in file\n",
      "\n",
      "Seq is: ACATTTACTCAC\n",
      "0 ACATTTACTCAC motifs detected in file\n",
      "\n",
      "Seq is: TCCGTCCTCAGG\n",
      "0 TCCGTCCTCAGG motifs detected in file\n",
      "\n",
      "Seq is: CTCCTCCAATCA\n",
      "0 CTCCTCCAATCA motifs detected in file\n",
      "\n",
      "Seq is: ACACCGTTATTT\n",
      "0 ACACCGTTATTT motifs detected in file\n",
      "\n",
      "Seq is: TTACTAATTCCT\n",
      "0 TTACTAATTCCT motifs detected in file\n",
      "\n",
      "Seq is: ATGATCATATTC\n",
      "0 ATGATCATATTC motifs detected in file\n",
      "\n",
      "Seq is: TCTAGTCCAAGC\n",
      "0 TCTAGTCCAAGC motifs detected in file\n",
      "\n",
      "Seq is: AGGAAATCACAA\n",
      "0 AGGAAATCACAA motifs detected in file\n",
      "\n",
      "Seq is: CACATACCAATA\n",
      "0 CACATACCAATA motifs detected in file\n",
      "\n",
      "Seq is: GTAAACAAAACT\n",
      "0 GTAAACAAAACT motifs detected in file\n",
      "\n",
      "Seq is: CCAATCAAACTA\n",
      "0 CCAATCAAACTA motifs detected in file\n",
      "\n",
      "Seq is: ATCAGTAACAAC\n",
      "0 ATCAGTAACAAC motifs detected in file\n",
      "\n",
      "Seq is: CCACCACTAATC\n",
      "0 CCACCACTAATC motifs detected in file\n",
      "\n",
      "Seq is: AGCCAACAACTA\n",
      "0 AGCCAACAACTA motifs detected in file\n",
      "\n",
      "Seq is: CTCTATAAACCT\n",
      "0 CTCTATAAACCT motifs detected in file\n",
      "\n",
      "Seq is: CCACCCCTAGTT\n",
      "0 CCACCCCTAGTT motifs detected in file\n",
      "\n",
      "Seq is: ATCTCCCCCTTT\n",
      "0 ATCTCCCCCTTT motifs detected in file\n",
      "\n",
      "Seq is: ATCTTACTAGTC\n",
      "0 ATCTTACTAGTC motifs detected in file\n",
      "\n",
      "Seq is: ATGACGCCGCCC\n",
      "0 ATGACGCCGCCC motifs detected in file\n",
      "\n",
      "Seq is: ATCCGGATCCTA\n",
      "0 ATCCGGATCCTA motifs detected in file\n",
      "\n",
      "Seq is: ACACCCCCGTTA\n",
      "0 ACACCCCCGTTA motifs detected in file\n",
      "\n",
      "Seq is: CTTACTTCGCTT\n",
      "0 CTTACTTCGCTT motifs detected in file\n",
      "\n",
      "Seq is: GTCTAAAAGCAA\n",
      "0 GTCTAAAAGCAA motifs detected in file\n",
      "\n",
      "Seq is: GTAAAACCAAGA\n",
      "0 GTAAAACCAAGA motifs detected in file\n",
      "\n",
      "Seq is: GTCCCAAACAAT\n",
      "0 GTCCCAAACAAT motifs detected in file\n",
      "\n",
      "Seq is: ATTAAAAAATAA\n",
      "0 ATTAAAAAATAA motifs detected in file\n",
      "\n",
      "Seq is: AAATACCCGAAA\n",
      "0 AAATACCCGAAA motifs detected in file\n",
      "\n",
      "Seq is: CCACTCCCTCCA\n",
      "0 CCACTCCCTCCA motifs detected in file\n",
      "\n",
      "Seq is: TCCACCATCAAA\n",
      "0 TCCACCATCAAA motifs detected in file\n",
      "\n",
      "Seq is: CACAAATTATAA\n",
      "0 CACAAATTATAA motifs detected in file\n",
      "\n",
      "Seq is: AAATCAACCCAC\n",
      "0 AAATCAACCCAC motifs detected in file\n",
      "\n",
      "Seq is: TACTCCTTATCA\n",
      "0 TACTCCTTATCA motifs detected in file\n",
      "\n",
      "Seq is: CATTCCTCATAC\n",
      "0 CATTCCTCATAC motifs detected in file\n",
      "\n",
      "Seq is: CCCCATTAATAC\n",
      "0 CCCCATTAATAC motifs detected in file\n",
      "\n",
      "Seq is: CACCCGGTATGC\n",
      "0 CACCCGGTATGC motifs detected in file\n",
      "\n",
      "Seq is: CATCATAAATCA\n",
      "0 CATCATAAATCA motifs detected in file\n",
      "\n",
      "Seq is: ACCCCGTAAAGT\n",
      "0 ACCCCGTAAAGT motifs detected in file\n",
      "\n",
      "Seq is: GACATTTCTCCA\n",
      "0 GACATTTCTCCA motifs detected in file\n",
      "\n",
      "Seq is: CAACCAAGAGAA\n",
      "0 CAACCAAGAGAA motifs detected in file\n",
      "\n",
      "Seq is: TTAGAAATTGAA\n",
      "0 TTAGAAATTGAA motifs detected in file\n",
      "\n",
      "Seq is: TTTTCCAATTAC\n",
      "0 TTTTCCAATTAC motifs detected in file\n",
      "\n",
      "Seq is: GATTCAATCGAA\n",
      "0 GATTCAATCGAA motifs detected in file\n",
      "\n",
      "Seq is: ATGTCACCCGAG\n",
      "0 ATGTCACCCGAG motifs detected in file\n",
      "\n",
      "Seq is: ATCTCCCCCATC\n",
      "0 ATCTCCCCCATC motifs detected in file\n",
      "\n",
      "Seq is: TCCCCCACGATA\n",
      "0 TCCCCCACGATA motifs detected in file\n",
      "\n",
      "Seq is: TCCCATTTATGC\n",
      "0 TCCCATTTATGC motifs detected in file\n",
      "\n",
      "Seq is: CCGGGATAAAGT\n",
      "0 CCGGGATAAAGT motifs detected in file\n",
      "\n",
      "Seq is: TTCTAAAAATCC\n",
      "0 TTCTAAAAATCC motifs detected in file\n",
      "\n",
      "Seq is: CCTTAATTAAGA\n",
      "0 CCTTAATTAAGA motifs detected in file\n",
      "\n",
      "Seq is: TCCTGCTACCCG\n",
      "0 TCCTGCTACCCG motifs detected in file\n",
      "\n",
      "Seq is: TACTTTGACACC\n",
      "0 TACTTTGACACC motifs detected in file\n",
      "\n",
      "Seq is: AAACGAGGTGTC\n",
      "0 AAACGAGGTGTC motifs detected in file\n",
      "\n",
      "Seq is: TAAAGTCTCGTA\n",
      "0 TAAAGTCTCGTA motifs detected in file\n",
      "\n",
      "Seq is: ACTGGCATCATA\n",
      "0 ACTGGCATCATA motifs detected in file\n",
      "\n",
      "Seq is: TGGGGGCCAGCA\n",
      "0 TGGGGGCCAGCA motifs detected in file\n",
      "\n",
      "Seq is: CATCGCCACTTT\n",
      "0 CATCGCCACTTT motifs detected in file\n",
      "\n",
      "Seq is: CACCAAACCAAA\n",
      "0 CACCAAACCAAA motifs detected in file\n",
      "\n",
      "Seq is: TCTGCAGGCCCT\n",
      "0 TCTGCAGGCCCT motifs detected in file\n",
      "\n",
      "Seq is: TAACGTAGACAA\n",
      "0 TAACGTAGACAA motifs detected in file\n",
      "\n",
      "Seq is: AAATTCGGATTA\n",
      "0 AAATTCGGATTA motifs detected in file\n",
      "\n",
      "Seq is: CACCCCTGTCGA\n",
      "0 CACCCCTGTCGA motifs detected in file\n",
      "\n",
      "Seq is: GTACTCACGTTC\n",
      "0 GTACTCACGTTC motifs detected in file\n",
      "\n",
      "Seq is: TGCAGAACACTA\n",
      "0 TGCAGAACACTA motifs detected in file\n",
      "\n",
      "Seq is: CATTAATAATAT\n",
      "0 CATTAATAATAT motifs detected in file\n",
      "\n",
      "Seq is: GCTTACCCCCGA\n",
      "0 GCTTACCCCCGA motifs detected in file\n",
      "\n",
      "Seq is: AGTTAGCCCTCA\n",
      "0 AGTTAGCCCTCA motifs detected in file\n",
      "\n",
      "Seq is: TGATGAGCTAAC\n",
      "0 TGATGAGCTAAC motifs detected in file\n",
      "\n",
      "Seq is: AGTTGCAGTTCC\n",
      "0 AGTTGCAGTTCC motifs detected in file\n",
      "\n",
      "Seq is: TCAGCGTCCAGC\n",
      "0 TCAGCGTCCAGC motifs detected in file\n",
      "\n",
      "Seq is: GGACCAAGATCC\n",
      "0 GGACCAAGATCC motifs detected in file\n",
      "\n",
      "Seq is: TTATTACCCCCT\n",
      "0 TTATTACCCCCT motifs detected in file\n",
      "\n",
      "Seq is: TCATACATCCTC\n",
      "0 TCATACATCCTC motifs detected in file\n",
      "\n",
      "Seq is: AACTTCTAATCA\n",
      "0 AACTTCTAATCA motifs detected in file\n",
      "\n",
      "Seq is: GGCGGCATCAGC\n",
      "0 GGCGGCATCAGC motifs detected in file\n",
      "\n",
      "Seq is: CACATGAGCATC\n",
      "0 CACATGAGCATC motifs detected in file\n",
      "\n",
      "Seq is: CAAGTCATGGTA\n",
      "0 CAAGTCATGGTA motifs detected in file\n",
      "\n",
      "Seq is: ACCACCGGTTAA\n",
      "0 ACCACCGGTTAA motifs detected in file\n",
      "\n",
      "Seq is: CTAAACTACCAT\n",
      "0 CTAAACTACCAT motifs detected in file\n",
      "\n",
      "Seq is: TCCCTCCCTAGC\n",
      "0 TCCCTCCCTAGC motifs detected in file\n",
      "\n",
      "Seq is: AACTGGAGCAGA\n",
      "0 AACTGGAGCAGA motifs detected in file\n",
      "\n",
      "Seq is: CAATACATTTCC\n",
      "0 CAATACATTTCC motifs detected in file\n",
      "\n",
      "Seq is: TACGCATCCCTA\n",
      "0 TACGCATCCCTA motifs detected in file\n",
      "\n",
      "Seq is: CCCTCCCGCTAC\n",
      "0 CCCTCCCGCTAC motifs detected in file\n",
      "\n",
      "Seq is: TCCTGATCCTAC\n",
      "0 TCCTGATCCTAC motifs detected in file\n",
      "\n",
      "Seq is: TACCGCCCGTCC\n",
      "0 TACCGCCCGTCC motifs detected in file\n",
      "\n",
      "Seq is: TATCAAGTCTGC\n",
      "0 TATCAAGTCTGC motifs detected in file\n",
      "\n",
      "Seq is: CAAAGATAAAGG\n",
      "0 CAAAGATAAAGG motifs detected in file\n",
      "\n",
      "Seq is: ACTCGCAGACTC\n",
      "0 ACTCGCAGACTC motifs detected in file\n",
      "\n",
      "Seq is: TACAATCATAAT\n",
      "0 TACAATCATAAT motifs detected in file\n",
      "\n",
      "Seq is: CAATCAAAACAA\n",
      "0 CAATCAAAACAA motifs detected in file\n",
      "\n",
      "Seq is: CACTCACAATCC\n",
      "0 CACTCACAATCC motifs detected in file\n",
      "\n",
      "Seq is: TTTTCCCGTATG\n",
      "0 TTTTCCCGTATG motifs detected in file\n",
      "\n",
      "Seq is: TCCTGATCCTTC\n",
      "0 TCCTGATCCTTC motifs detected in file\n",
      "\n",
      "Seq is: GTCTATTCCTTT\n",
      "0 GTCTATTCCTTT motifs detected in file\n",
      "\n",
      "Seq is: TACTAATACTCC\n",
      "0 TACTAATACTCC motifs detected in file\n",
      "\n",
      "Seq is: CGCACTAGTACT\n",
      "0 CGCACTAGTACT motifs detected in file\n",
      "\n",
      "Seq is: TTCCACTATTCG\n",
      "0 TTCCACTATTCG motifs detected in file\n",
      "\n",
      "Seq is: AGAAGATACTAT\n",
      "0 AGAAGATACTAT motifs detected in file\n",
      "\n",
      "Seq is: CCCCTTCATCGC\n",
      "0 CCCCTTCATCGC motifs detected in file\n",
      "\n",
      "Seq is: AGAACATCTGGA\n",
      "0 AGAACATCTGGA motifs detected in file\n",
      "\n",
      "Seq is: TGAACGCGACGT\n",
      "0 TGAACGCGACGT motifs detected in file\n",
      "\n",
      "Seq is: ACACGGTAATTC\n",
      "0 ACACGGTAATTC motifs detected in file\n",
      "\n",
      "Seq is: TATATATCCTAA\n",
      "0 TATATATCCTAA motifs detected in file\n",
      "\n",
      "Seq is: ATCGGATATTAA\n",
      "0 ATCGGATATTAA motifs detected in file\n",
      "\n",
      "Seq is: ATTGAAACTGTT\n",
      "0 ATTGAAACTGTT motifs detected in file\n",
      "\n",
      "Seq is: TCAATACTTTAC\n",
      "0 TCAATACTTTAC motifs detected in file\n",
      "\n",
      "Seq is: CAAAAAGATTAT\n",
      "0 CAAAAAGATTAT motifs detected in file\n",
      "\n",
      "Seq is: GGAAAAACTTTT\n",
      "0 GGAAAAACTTTT motifs detected in file\n",
      "\n",
      "Seq is: TCAGTACCTCCG\n",
      "0 TCAGTACCTCCG motifs detected in file\n",
      "\n",
      "Seq is: GTACCCCAACCG\n",
      "0 GTACCCCAACCG motifs detected in file\n",
      "\n",
      "Seq is: AACTTTGGTCGA\n",
      "0 AACTTTGGTCGA motifs detected in file\n",
      "\n",
      "Seq is: AACCCCCCAAGC\n",
      "0 AACCCCCCAAGC motifs detected in file\n",
      "\n",
      "Seq is: TAAGGAAGGAAA\n",
      "0 TAAGGAAGGAAA motifs detected in file\n",
      "\n",
      "Seq is: AAACAGATCTAA\n",
      "0 AAACAGATCTAA motifs detected in file\n",
      "\n",
      "Seq is: AATACATATGCC\n",
      "0 AATACATATGCC motifs detected in file\n",
      "\n",
      "Seq is: CAAGAAGCTTAC\n",
      "0 CAAGAAGCTTAC motifs detected in file\n",
      "\n",
      "Seq is: ACACCATCCCAC\n",
      "0 ACACCATCCCAC motifs detected in file\n",
      "\n",
      "Seq is: CCCCCGTAGGTA\n",
      "0 CCCCCGTAGGTA motifs detected in file\n",
      "\n",
      "Seq is: TATCAGTGAGGT\n",
      "0 TATCAGTGAGGT motifs detected in file\n",
      "\n",
      "Seq is: CCAAATACCTCC\n",
      "0 CCAAATACCTCC motifs detected in file\n",
      "\n",
      "Seq is: CAAGAAGATGAT\n",
      "0 CAAGAAGATGAT motifs detected in file\n",
      "\n",
      "Seq is: AATCCTGAAAAG\n",
      "0 AATCCTGAAAAG motifs detected in file\n",
      "\n",
      "Seq is: CGAAGCTTCGCT\n",
      "0 CGAAGCTTCGCT motifs detected in file\n",
      "\n",
      "Seq is: TCCGAAGAGTTT\n",
      "0 TCCGAAGAGTTT motifs detected in file\n",
      "\n",
      "Seq is: AGTACTTTTAAT\n",
      "0 AGTACTTTTAAT motifs detected in file\n",
      "\n",
      "Seq is: AATTATAATGAC\n",
      "0 AATTATAATGAC motifs detected in file\n",
      "\n",
      "Seq is: GACAATCTCTTT\n",
      "0 GACAATCTCTTT motifs detected in file\n",
      "\n",
      "Seq is: ACTTACTCGGAT\n",
      "0 ACTTACTCGGAT motifs detected in file\n",
      "\n",
      "Seq is: GTCTACTATCCT\n",
      "0 GTCTACTATCCT motifs detected in file\n",
      "\n",
      "Seq is: ACAAAGTACACC\n",
      "0 ACAAAGTACACC motifs detected in file\n",
      "\n",
      "Seq is: ACATACGTAGCC\n",
      "0 ACATACGTAGCC motifs detected in file\n",
      "\n",
      "Seq is: CCATCAGGCTCA\n",
      "0 CCATCAGGCTCA motifs detected in file\n",
      "\n",
      "Seq is: TTGCAGCCCCGT\n",
      "0 TTGCAGCCCCGT motifs detected in file\n",
      "\n",
      "Seq is: AAGGCCTATCCG\n",
      "0 AAGGCCTATCCG motifs detected in file\n",
      "\n",
      "Seq is: GCTCTTTCACAA\n",
      "0 GCTCTTTCACAA motifs detected in file\n",
      "\n",
      "Seq is: CACCCTTCTTTC\n",
      "0 CACCCTTCTTTC motifs detected in file\n",
      "\n",
      "Seq is: AATCTAAATGCG\n",
      "0 AATCTAAATGCG motifs detected in file\n",
      "\n",
      "Seq is: GCTACTTATACT\n",
      "0 GCTACTTATACT motifs detected in file\n",
      "\n",
      "Seq is: ATCACTTTACCT\n",
      "0 ATCACTTTACCT motifs detected in file\n",
      "\n",
      "Seq is: AAAACCGCATCC\n",
      "0 AAAACCGCATCC motifs detected in file\n",
      "\n",
      "Seq is: AAACCAGATCCC\n",
      "0 AAACCAGATCCC motifs detected in file\n",
      "\n",
      "Seq is: ACATCGGACTCT\n",
      "0 ACATCGGACTCT motifs detected in file\n",
      "\n",
      "Seq is: TATCCCCTTTAG\n",
      "0 TATCCCCTTTAG motifs detected in file\n",
      "\n",
      "Seq is: TCACTTACTTCG\n",
      "0 TCACTTACTTCG motifs detected in file\n",
      "\n",
      "Seq is: GAGGATACTACC\n",
      "0 GAGGATACTACC motifs detected in file\n",
      "\n",
      "Seq is: GTTTATGTCGAG\n",
      "0 GTTTATGTCGAG motifs detected in file\n",
      "\n",
      "Seq is: GATAACTATCCT\n",
      "0 GATAACTATCCT motifs detected in file\n",
      "\n",
      "Seq is: GTATCACCTTCA\n",
      "0 GTATCACCTTCA motifs detected in file\n",
      "\n",
      "Seq is: CCCGATGTTGCA\n",
      "0 CCCGATGTTGCA motifs detected in file\n",
      "\n",
      "Seq is: TCATGCACAGCA\n",
      "0 TCATGCACAGCA motifs detected in file\n",
      "\n",
      "Seq is: CATCATGCTACA\n",
      "0 CATCATGCTACA motifs detected in file\n",
      "\n",
      "Seq is: GATCACTACTCA\n",
      "0 GATCACTACTCA motifs detected in file\n",
      "\n",
      "Seq is: AACGATTATGTT\n",
      "0 AACGATTATGTT motifs detected in file\n",
      "\n",
      "Seq is: ACGGTCAGTCCG\n",
      "0 ACGGTCAGTCCG motifs detected in file\n",
      "\n",
      "Seq is: GTGGATGCCACT\n",
      "0 GTGGATGCCACT motifs detected in file\n",
      "\n",
      "Seq is: TTTCTTTCTACT\n",
      "0 TTTCTTTCTACT motifs detected in file\n",
      "\n",
      "Seq is: TAGGATCCCGAG\n",
      "0 TAGGATCCCGAG motifs detected in file\n",
      "\n",
      "Seq is: TCTCGTGACGTC\n",
      "0 TCTCGTGACGTC motifs detected in file\n",
      "\n",
      "Seq is: GTCTAGTAAAGT\n",
      "0 GTCTAGTAAAGT motifs detected in file\n",
      "\n",
      "Seq is: ATAACGAAGGCA\n",
      "0 ATAACGAAGGCA motifs detected in file\n",
      "\n",
      "Seq is: CCTCACACCGCT\n",
      "0 CCTCACACCGCT motifs detected in file\n",
      "\n",
      "Seq is: CAGTCGATTTAT\n",
      "0 CAGTCGATTTAT motifs detected in file\n",
      "\n",
      "Seq is: GAAACTGCGGCC\n",
      "0 GAAACTGCGGCC motifs detected in file\n",
      "\n",
      "Seq is: ACCCCTATCGCT\n",
      "0 ACCCCTATCGCT motifs detected in file\n",
      "\n",
      "Seq is: ACTAATACCATC\n",
      "0 ACTAATACCATC motifs detected in file\n",
      "\n",
      "Seq is: GCCTCCACTTTA\n",
      "0 GCCTCCACTTTA motifs detected in file\n",
      "\n",
      "Seq is: TACGAGCACACA\n",
      "0 TACGAGCACACA motifs detected in file\n",
      "\n",
      "Seq is: GATGCAGATAAG\n",
      "0 GATGCAGATAAG motifs detected in file\n",
      "\n",
      "Seq is: GATGACATTTAT\n",
      "0 GATGACATTTAT motifs detected in file\n",
      "\n",
      "Seq is: ATACCACACGAG\n",
      "0 ATACCACACGAG motifs detected in file\n",
      "\n",
      "Seq is: TGTGCTATTTGG\n",
      "0 TGTGCTATTTGG motifs detected in file\n",
      "\n",
      "Seq is: GATCCTTCGGTT\n",
      "0 GATCCTTCGGTT motifs detected in file\n",
      "\n",
      "Seq is: AACTATAGTATC\n",
      "0 AACTATAGTATC motifs detected in file\n",
      "\n",
      "Seq is: GAGTCTGGTATG\n",
      "0 GAGTCTGGTATG motifs detected in file\n",
      "\n",
      "Seq is: GATACATAGGTT\n",
      "0 GATACATAGGTT motifs detected in file\n",
      "\n",
      "Seq is: TACCAAGAAAAA\n",
      "0 TACCAAGAAAAA motifs detected in file\n",
      "\n",
      "Seq is: AAGGCCTCATCA\n",
      "0 AAGGCCTCATCA motifs detected in file\n",
      "\n",
      "Seq is: TTCAATGTTATA\n",
      "0 TTCAATGTTATA motifs detected in file\n",
      "\n",
      "Seq is: CCCTCTAATAAG\n",
      "0 CCCTCTAATAAG motifs detected in file\n",
      "\n",
      "Seq is: GCTTCGGACCAT\n",
      "0 GCTTCGGACCAT motifs detected in file\n",
      "\n",
      "Seq is: CCTATTCTTATA\n",
      "0 CCTATTCTTATA motifs detected in file\n",
      "\n",
      "Seq is: TTTGAAGTCCCA\n",
      "0 TTTGAAGTCCCA motifs detected in file\n",
      "\n",
      "Seq is: CTGGCTTTTTAG\n",
      "0 CTGGCTTTTTAG motifs detected in file\n",
      "\n",
      "Seq is: TCTTATCCACAA\n",
      "0 TCTTATCCACAA motifs detected in file\n",
      "\n",
      "Seq is: CCATATCTTACC\n",
      "0 CCATATCTTACC motifs detected in file\n",
      "\n",
      "Seq is: CCAGAGGAGGAG\n",
      "0 CCAGAGGAGGAG motifs detected in file\n",
      "\n",
      "Seq is: GCCGCCCCAGCT\n",
      "0 GCCGCCCCAGCT motifs detected in file\n",
      "\n",
      "Seq is: TCTTCCACCACA\n",
      "0 TCTTCCACCACA motifs detected in file\n",
      "\n",
      "Seq is: ACTCCAACGCCA\n",
      "0 ACTCCAACGCCA motifs detected in file\n",
      "\n",
      "Seq is: GACAATCATCAT\n",
      "0 GACAATCATCAT motifs detected in file\n",
      "\n",
      "Seq is: ATCACTACGGTC\n",
      "0 ATCACTACGGTC motifs detected in file\n",
      "\n",
      "Seq is: GTCGATCCTGAC\n",
      "0 GTCGATCCTGAC motifs detected in file\n",
      "\n",
      "Seq is: CCTCTCTATCCT\n",
      "0 CCTCTCTATCCT motifs detected in file\n",
      "\n",
      "Seq is: CTTCATCCTGAC\n",
      "0 CTTCATCCTGAC motifs detected in file\n",
      "\n",
      "Seq is: GACACTAATCCT\n",
      "0 GACACTAATCCT motifs detected in file\n",
      "\n",
      "Seq is: GCCTAGTCTGCT\n",
      "0 GCCTAGTCTGCT motifs detected in file\n",
      "\n",
      "Seq is: TCTCCCCGCAAA\n",
      "0 TCTCCCCGCAAA motifs detected in file\n",
      "\n",
      "Complement found from 6426 to 6438\n",
      "Seq is: CCATAACCCAAT\n",
      "1 CCATAACCCAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCGTCCCCCAA\n",
      "0 ACCGTCCCCCAA motifs detected in file\n",
      "\n",
      "Seq is: AATATAACTATT\n",
      "0 AATATAACTATT motifs detected in file\n",
      "\n",
      "Seq is: AACAACACTACT\n",
      "0 AACAACACTACT motifs detected in file\n",
      "\n",
      "Seq is: TTAACTACCGGG\n",
      "0 TTAACTACCGGG motifs detected in file\n",
      "\n",
      "Seq is: GATTCTATCTCC\n",
      "0 GATTCTATCTCC motifs detected in file\n",
      "\n",
      "Seq is: TCTGTGGACGAT\n",
      "0 TCTGTGGACGAT motifs detected in file\n",
      "\n",
      "Seq is: CCACATTCCTCT\n",
      "0 CCACATTCCTCT motifs detected in file\n",
      "\n",
      "Seq is: TCTACCAATCCA\n",
      "0 TCTACCAATCCA motifs detected in file\n",
      "\n",
      "Seq is: GATGCCTCCGAG\n",
      "0 GATGCCTCCGAG motifs detected in file\n",
      "\n",
      "Complement found from 15260 to 15272\n",
      "Seq is: GTCCCACCCTCA\n",
      "1 GTCCCACCCTCA motifs detected in file\n",
      "\n",
      "Seq is: TCAAGGGACGAT\n",
      "0 TCAAGGGACGAT motifs detected in file\n",
      "\n",
      "Seq is: TCCCTCCCATCT\n",
      "0 TCCCTCCCATCT motifs detected in file\n",
      "\n",
      "Seq is: GACAAGTTGGAC\n",
      "0 GACAAGTTGGAC motifs detected in file\n",
      "\n",
      "Seq is: AAGGACGAGGCC\n",
      "0 AAGGACGAGGCC motifs detected in file\n",
      "\n",
      "Seq is: GGAGGTGATATC\n",
      "0 GGAGGTGATATC motifs detected in file\n",
      "\n",
      "Seq is: GTCTACGCTCGT\n",
      "0 GTCTACGCTCGT motifs detected in file\n",
      "\n",
      "Seq is: CCTCATCCTCTC\n",
      "0 CCTCATCCTCTC motifs detected in file\n",
      "\n",
      "Seq is: TCCCTCCATTCT\n",
      "0 TCCCTCCATTCT motifs detected in file\n",
      "\n",
      "Seq is: CAGTCTTCGAAT\n",
      "0 CAGTCTTCGAAT motifs detected in file\n",
      "\n",
      "Seq is: ACAACAAATACG\n",
      "0 ACAACAAATACG motifs detected in file\n",
      "\n",
      "Seq is: CCCCTTTGCGGT\n",
      "0 CCCCTTTGCGGT motifs detected in file\n",
      "\n",
      "Seq is: ATAGCCCCCGTG\n",
      "0 ATAGCCCCCGTG motifs detected in file\n",
      "\n",
      "Seq is: GCTAATAATCCC\n",
      "0 GCTAATAATCCC motifs detected in file\n",
      "\n",
      "Seq is: CTTGATCAGTCA\n",
      "0 CTTGATCAGTCA motifs detected in file\n",
      "\n",
      "Seq is: ACGGTTTCGGAG\n",
      "0 ACGGTTTCGGAG motifs detected in file\n",
      "\n",
      "Seq is: GCTAATACTACC\n",
      "0 GCTAATACTACC motifs detected in file\n",
      "\n",
      "Seq is: CATAATGATACT\n",
      "0 CATAATGATACT motifs detected in file\n",
      "\n",
      "Seq is: TCTTCTAATAAT\n",
      "0 TCTTCTAATAAT motifs detected in file\n",
      "\n",
      "Seq is: GTTTACGTACCC\n",
      "0 GTTTACGTACCC motifs detected in file\n",
      "\n",
      "Seq is: GACACTGCTATT\n",
      "0 GACACTGCTATT motifs detected in file\n",
      "\n",
      "Seq is: GCAACATCTACA\n",
      "0 GCAACATCTACA motifs detected in file\n",
      "\n",
      "Seq is: CCAGCAATGGAT\n",
      "0 CCAGCAATGGAT motifs detected in file\n",
      "\n",
      "Seq is: CTTCCAACGGAC\n",
      "0 CTTCCAACGGAC motifs detected in file\n",
      "\n",
      "Seq is: CGACCGGGTCGA\n",
      "0 CGACCGGGTCGA motifs detected in file\n",
      "\n",
      "Seq is: GCCGAGCTTATT\n",
      "0 GCCGAGCTTATT motifs detected in file\n",
      "\n",
      "Seq is: CCTCCGAATCTC\n",
      "0 CCTCCGAATCTC motifs detected in file\n",
      "\n",
      "Seq is: GACACGGATCCT\n",
      "0 GACACGGATCCT motifs detected in file\n",
      "\n",
      "Seq is: GAGGTCGAGTAC\n",
      "0 GAGGTCGAGTAC motifs detected in file\n",
      "\n",
      "Seq is: GCGGCTTATTAT\n",
      "0 GCGGCTTATTAT motifs detected in file\n",
      "\n",
      "Seq is: CCATATCACAAG\n",
      "0 CCATATCACAAG motifs detected in file\n",
      "\n",
      "Seq is: GTTACAGAAACA\n",
      "0 GTTACAGAAACA motifs detected in file\n",
      "\n",
      "Seq is: CCAAACATCTCT\n",
      "0 CCAAACATCTCT motifs detected in file\n",
      "\n",
      "Seq is: TATCAGTTGCCA\n",
      "0 TATCAGTTGCCA motifs detected in file\n",
      "\n",
      "Seq is: GCCGCTTGTAGT\n",
      "0 GCCGCTTGTAGT motifs detected in file\n",
      "\n",
      "Seq is: CACCCCCACTCC\n",
      "0 CACCCCCACTCC motifs detected in file\n",
      "\n",
      "Seq is: ATTTTACCGACT\n",
      "0 ATTTTACCGACT motifs detected in file\n",
      "\n",
      "Seq is: CACTTCGTAACC\n",
      "0 CACTTCGTAACC motifs detected in file\n",
      "\n",
      "Seq is: TGACATTTAGAT\n",
      "0 TGACATTTAGAT motifs detected in file\n",
      "\n",
      "Seq is: TTCTGTCCCCAA\n",
      "0 TTCTGTCCCCAA motifs detected in file\n",
      "\n",
      "Seq is: TCCGGAGAAAAA\n",
      "0 TCCGGAGAAAAA motifs detected in file\n",
      "\n",
      "Seq is: TGGTCGAGGCTC\n",
      "0 TGGTCGAGGCTC motifs detected in file\n",
      "\n",
      "Seq is: CACTAAAAGTAT\n",
      "0 CACTAAAAGTAT motifs detected in file\n",
      "\n",
      "Seq is: AACTTAACGTTT\n",
      "0 AACTTAACGTTT motifs detected in file\n",
      "\n",
      "Seq is: AAGCTTCTTCGT\n",
      "0 AAGCTTCTTCGT motifs detected in file\n",
      "\n",
      "Seq is: CGAAGTTTGGAC\n",
      "0 CGAAGTTTGGAC motifs detected in file\n",
      "\n",
      "Seq is: GGCCCCGAAGAG\n",
      "0 GGCCCCGAAGAG motifs detected in file\n",
      "\n",
      "Seq is: GGCGGAAAAAAG\n",
      "0 GGCGGAAAAAAG motifs detected in file\n",
      "\n",
      "Seq is: GGCCGCCGCCCT\n",
      "0 GGCCGCCGCCCT motifs detected in file\n",
      "\n",
      "Seq is: CTTCATCTAACT\n",
      "0 CTTCATCTAACT motifs detected in file\n",
      "\n",
      "Seq is: TCGGTCAACTAA\n",
      "0 TCGGTCAACTAA motifs detected in file\n",
      "\n",
      "Seq is: TCCCACGAATCG\n",
      "0 TCCCACGAATCG motifs detected in file\n",
      "\n",
      "Seq is: ACAATTGATTCA\n",
      "0 ACAATTGATTCA motifs detected in file\n",
      "\n",
      "Seq is: CAAACACCCAAA\n",
      "0 CAAACACCCAAA motifs detected in file\n",
      "\n",
      "Seq is: TTCAGGGTAACC\n",
      "0 TTCAGGGTAACC motifs detected in file\n",
      "\n",
      "Seq is: AGATCATTCCCG\n",
      "0 AGATCATTCCCG motifs detected in file\n",
      "\n",
      "Seq is: AATCGAATTAAT\n",
      "0 AATCGAATTAAT motifs detected in file\n",
      "\n",
      "Seq is: TTCACCGACTAA\n",
      "0 TTCACCGACTAA motifs detected in file\n",
      "\n",
      "Seq is: ACGCAAGTCAAC\n",
      "0 ACGCAAGTCAAC motifs detected in file\n",
      "\n",
      "Seq is: TACGTCTCACCC\n",
      "0 TACGTCTCACCC motifs detected in file\n",
      "\n",
      "Seq is: CAAAACGTCAGG\n",
      "0 CAAAACGTCAGG motifs detected in file\n",
      "\n",
      "Seq is: AATCGACAATGT\n",
      "0 AATCGACAATGT motifs detected in file\n",
      "\n",
      "Seq is: CTTTAATTCATA\n",
      "0 CTTTAATTCATA motifs detected in file\n",
      "\n",
      "Seq is: ACGTTGAATGAC\n",
      "0 ACGTTGAATGAC motifs detected in file\n",
      "\n",
      "Seq is: TCCCGAAACTTC\n",
      "0 TCCCGAAACTTC motifs detected in file\n",
      "\n",
      "Seq is: CGAGAACCAGAC\n",
      "0 CGAGAACCAGAC motifs detected in file\n",
      "\n",
      "Seq is: ATAAATTGGATT\n",
      "0 ATAAATTGGATT motifs detected in file\n",
      "\n",
      "Seq is: TAAAGATATTCT\n",
      "0 TAAAGATATTCT motifs detected in file\n",
      "\n",
      "Seq is: AATAATCATATT\n",
      "0 AATAATCATATT motifs detected in file\n",
      "\n",
      "Seq is: TTCCCCTCTATC\n",
      "0 TTCCCCTCTATC motifs detected in file\n",
      "\n",
      "Seq is: CATCCTCATCGC\n",
      "0 CATCCTCATCGC motifs detected in file\n",
      "\n",
      "Seq is: ACCATTCCCGCT\n",
      "0 ACCATTCCCGCT motifs detected in file\n",
      "\n",
      "Seq is: ACTCACACCCCT\n",
      "0 ACTCACACCCCT motifs detected in file\n",
      "\n",
      "Seq is: CCTTACCCCACC\n",
      "0 CCTTACCCCACC motifs detected in file\n",
      "\n",
      "Seq is: CAAAACATACAA\n",
      "0 CAAAACATACAA motifs detected in file\n",
      "\n",
      "Seq is: GTTTGACAGTAA\n",
      "0 GTTTGACAGTAA motifs detected in file\n",
      "\n",
      "Seq is: AATAAAAATGCA\n",
      "0 AATAAAAATGCA motifs detected in file\n",
      "\n",
      "Seq is: ACAATCTATACC\n",
      "0 ACAATCTATACC motifs detected in file\n",
      "\n",
      "Seq is: CCTCATCACACT\n",
      "0 CCTCATCACACT motifs detected in file\n",
      "\n",
      "Seq is: AACTCCACCTCA\n",
      "0 AACTCCACCTCA motifs detected in file\n",
      "\n",
      "Seq is: TCTAATCCGCAT\n",
      "0 TCTAATCCGCAT motifs detected in file\n",
      "\n",
      "Seq is: CCATCTTCATCT\n",
      "0 CCATCTTCATCT motifs detected in file\n",
      "\n",
      "Seq is: CCAATTCCTCCC\n",
      "0 CCAATTCCTCCC motifs detected in file\n",
      "\n",
      "Seq is: ACTACCACCGAT\n",
      "0 ACTACCACCGAT motifs detected in file\n",
      "\n",
      "Seq is: ACTACCACCCCT\n",
      "0 ACTACCACCCCT motifs detected in file\n",
      "\n",
      "Seq is: ACTACTCCGATA\n",
      "0 ACTACTCCGATA motifs detected in file\n",
      "\n",
      "Seq is: ACAAAAAACACT\n",
      "0 ACAAAAAACACT motifs detected in file\n",
      "\n",
      "Seq is: TAAGAAGCTATT\n",
      "0 TAAGAAGCTATT motifs detected in file\n",
      "\n",
      "Seq is: ACCGGGTAAACC\n",
      "0 ACCGGGTAAACC motifs detected in file\n",
      "\n",
      "Seq is: CGTTTTTCGGCC\n",
      "0 CGTTTTTCGGCC motifs detected in file\n",
      "\n",
      "Seq is: AATCGCCCCCGT\n",
      "0 AATCGCCCCCGT motifs detected in file\n",
      "\n",
      "Seq is: CCGGAGGATCCC\n",
      "0 CCGGAGGATCCC motifs detected in file\n",
      "\n",
      "Seq is: TCTCCTCCCACC\n",
      "0 TCTCCTCCCACC motifs detected in file\n",
      "\n",
      "Seq is: TACCTTAATTCC\n",
      "0 TACCTTAATTCC motifs detected in file\n",
      "\n",
      "Seq is: CACAATCAGTAC\n",
      "0 CACAATCAGTAC motifs detected in file\n",
      "\n",
      "Seq is: AATCGAACAAAG\n",
      "0 AATCGAACAAAG motifs detected in file\n",
      "\n",
      "Seq is: TCCACGCTCTAT\n",
      "0 TCCACGCTCTAT motifs detected in file\n",
      "\n",
      "Seq is: CATCATCCCAGC\n",
      "0 CATCATCCCAGC motifs detected in file\n",
      "\n",
      "Seq is: ACCACGACCTCA\n",
      "0 ACCACGACCTCA motifs detected in file\n",
      "\n",
      "Seq is: AATTCAACTCAT\n",
      "0 AATTCAACTCAT motifs detected in file\n",
      "\n",
      "Seq is: CATCCTTACGCC\n",
      "0 CATCCTTACGCC motifs detected in file\n",
      "\n",
      "Seq is: ATCATCAATCCT\n",
      "0 ATCATCAATCCT motifs detected in file\n",
      "\n",
      "Seq is: ATTATATTTATC\n",
      "0 ATTATATTTATC motifs detected in file\n",
      "\n",
      "Seq is: AATTTAATTCTT\n",
      "0 AATTTAATTCTT motifs detected in file\n",
      "\n",
      "Seq is: ACCAATACAATC\n",
      "0 ACCAATACAATC motifs detected in file\n",
      "\n",
      "Seq is: CCAACATGCCAT\n",
      "0 CCAACATGCCAT motifs detected in file\n",
      "\n",
      "Seq is: CTTGACGATAAT\n",
      "0 CTTGACGATAAT motifs detected in file\n",
      "\n",
      "Seq is: AAGTAGGATACA\n",
      "0 AAGTAGGATACA motifs detected in file\n",
      "\n",
      "Seq is: CCCATTAACTCC\n",
      "0 CCCATTAACTCC motifs detected in file\n",
      "\n",
      "Seq is: TCATACGATTCT\n",
      "0 TCATACGATTCT motifs detected in file\n",
      "\n",
      "Seq is: AAAACGCATCGA\n",
      "0 AAAACGCATCGA motifs detected in file\n",
      "\n",
      "Seq is: CCCAAACCAAAT\n",
      "0 CCCAAACCAAAT motifs detected in file\n",
      "\n",
      "Seq is: TAGGTGGAGTTG\n",
      "0 TAGGTGGAGTTG motifs detected in file\n",
      "\n",
      "Seq is: ACGGACGATACT\n",
      "0 ACGGACGATACT motifs detected in file\n",
      "\n",
      "Seq is: ACCTATTCTAAC\n",
      "0 ACCTATTCTAAC motifs detected in file\n",
      "\n",
      "Seq is: TCTCTCACTCCT\n",
      "0 TCTCTCACTCCT motifs detected in file\n",
      "\n",
      "Seq is: CTTCCGAATGCA\n",
      "0 CTTCCGAATGCA motifs detected in file\n",
      "\n",
      "Seq is: AATCACTCCCTC\n",
      "0 AATCACTCCCTC motifs detected in file\n",
      "\n",
      "Seq is: TCTAAACCATAT\n",
      "0 TCTAAACCATAT motifs detected in file\n",
      "\n",
      "Seq is: ACTAACTCTACC\n",
      "0 ACTAACTCTACC motifs detected in file\n",
      "\n",
      "Seq is: CCCGATCAAAAA\n",
      "0 CCCGATCAAAAA motifs detected in file\n",
      "\n",
      "Seq is: CAGTACACTCTT\n",
      "0 CAGTACACTCTT motifs detected in file\n",
      "\n",
      "Seq is: CTTCGTCCGGCC\n",
      "0 CTTCGTCCGGCC motifs detected in file\n",
      "\n",
      "Seq is: TACAGTCTCCCC\n",
      "0 TACAGTCTCCCC motifs detected in file\n",
      "\n",
      "Seq is: ACGGAACCCATT\n",
      "0 ACGGAACCCATT motifs detected in file\n",
      "\n",
      "Seq is: GGAGACCCTGAG\n",
      "0 GGAGACCCTGAG motifs detected in file\n",
      "\n",
      "Seq is: TCTTCACTTTCC\n",
      "0 TCTTCACTTTCC motifs detected in file\n",
      "\n",
      "Seq is: CCCGATAAGGAT\n",
      "0 CCCGATAAGGAT motifs detected in file\n",
      "\n",
      "Seq is: CAAAATAACGAT\n",
      "0 CAAAATAACGAT motifs detected in file\n",
      "\n",
      "Seq is: ATCGATAATACT\n",
      "0 ATCGATAATACT motifs detected in file\n",
      "\n",
      "Seq is: AATAATTACTAC\n",
      "0 AATAATTACTAC motifs detected in file\n",
      "\n",
      "Seq is: TCATAACTAACC\n",
      "0 TCATAACTAACC motifs detected in file\n",
      "\n",
      "Seq is: ATCATAACCAAT\n",
      "0 ATCATAACCAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCAAGTAACAG\n",
      "0 ACCAAGTAACAG motifs detected in file\n",
      "\n",
      "Seq is: GCCTCTCATATA\n",
      "0 GCCTCTCATATA motifs detected in file\n",
      "\n",
      "Seq is: ACAACTTCTCCT\n",
      "0 ACAACTTCTCCT motifs detected in file\n",
      "\n",
      "Seq is: ATCGATAATCTT\n",
      "0 ATCGATAATCTT motifs detected in file\n",
      "\n",
      "Seq is: CCTAATACCTAC\n",
      "0 CCTAATACCTAC motifs detected in file\n",
      "\n",
      "Seq is: GCCAACGAACGC\n",
      "0 GCCAACGAACGC motifs detected in file\n",
      "\n",
      "Seq is: ACTCCTTTATGA\n",
      "0 ACTCCTTTATGA motifs detected in file\n",
      "\n",
      "Seq is: ACTACCGTCGAA\n",
      "0 ACTACCGTCGAA motifs detected in file\n",
      "\n",
      "Seq is: GACACCTTGCTC\n",
      "0 GACACCTTGCTC motifs detected in file\n",
      "\n",
      "Seq is: CCAAATAAAAAA\n",
      "0 CCAAATAAAAAA motifs detected in file\n",
      "\n",
      "Seq is: ACCAATCTTGAC\n",
      "0 ACCAATCTTGAC motifs detected in file\n",
      "\n",
      "Seq is: CTTATTTTCGAT\n",
      "0 CTTATTTTCGAT motifs detected in file\n",
      "\n",
      "Seq is: CGTACAAATAAA\n",
      "0 CGTACAAATAAA motifs detected in file\n",
      "\n",
      "Seq is: GATCCGGATGAG\n",
      "0 GATCCGGATGAG motifs detected in file\n",
      "\n",
      "Seq is: TCCATTTTTTAG\n",
      "0 TCCATTTTTTAG motifs detected in file\n",
      "\n",
      "Seq is: TCACGCTCGAAT\n",
      "0 TCACGCTCGAAT motifs detected in file\n",
      "\n",
      "Seq is: CGCGACACTACT\n",
      "0 CGCGACACTACT motifs detected in file\n",
      "\n",
      "Seq is: CACACGGACGTT\n",
      "0 CACACGGACGTT motifs detected in file\n",
      "\n",
      "Seq is: TCTACCATCTCA\n",
      "0 TCTACCATCTCA motifs detected in file\n",
      "\n",
      "Seq is: TCTACTGCCCAA\n",
      "0 TCTACTGCCCAA motifs detected in file\n",
      "\n",
      "Seq is: CCCGGTCCCCTA\n",
      "0 CCCGGTCCCCTA motifs detected in file\n",
      "\n",
      "Seq is: ATTAATCATGCC\n",
      "0 ATTAATCATGCC motifs detected in file\n",
      "\n",
      "Seq is: CTTCCCATATTG\n",
      "0 CTTCCCATATTG motifs detected in file\n",
      "\n",
      "Seq is: GTTGTAAAAGCC\n",
      "0 GTTGTAAAAGCC motifs detected in file\n",
      "\n",
      "Seq is: CCATACCCGGGC\n",
      "0 CCATACCCGGGC motifs detected in file\n",
      "\n",
      "Seq is: TATCGAATAAAT\n",
      "0 TATCGAATAAAT motifs detected in file\n",
      "\n",
      "Seq is: CGACTGGAATGA\n",
      "0 CGACTGGAATGA motifs detected in file\n",
      "\n",
      "Seq is: AATCCTACCCCA\n",
      "0 AATCCTACCCCA motifs detected in file\n",
      "\n",
      "Seq is: CACTATCCACCG\n",
      "0 CACTATCCACCG motifs detected in file\n",
      "\n",
      "Seq is: TGCCTCTTAAAA\n",
      "0 TGCCTCTTAAAA motifs detected in file\n",
      "\n",
      "Seq is: CCTAAGAGTCCC\n",
      "0 CCTAAGAGTCCC motifs detected in file\n",
      "\n",
      "Seq is: TACCCAAGCTAA\n",
      "0 TACCCAAGCTAA motifs detected in file\n",
      "\n",
      "Seq is: GAGTATCAGGAT\n",
      "0 GAGTATCAGGAT motifs detected in file\n",
      "\n",
      "Seq is: CTTTATTCCCCC\n",
      "0 CTTTATTCCCCC motifs detected in file\n",
      "\n",
      "Seq is: AAATTCGAGGAT\n",
      "0 AAATTCGAGGAT motifs detected in file\n",
      "\n",
      "Seq is: AATAAATGAGAT\n",
      "0 AATAAATGAGAT motifs detected in file\n",
      "\n",
      "Seq is: AGTTTCATTGAG\n",
      "0 AGTTTCATTGAG motifs detected in file\n",
      "\n",
      "Seq is: AAAATAGTCTGT\n",
      "0 AAAATAGTCTGT motifs detected in file\n",
      "\n",
      "Seq is: ATAAAGAATCCA\n",
      "0 ATAAAGAATCCA motifs detected in file\n",
      "\n",
      "Seq is: AACTCCCCCTTA\n",
      "0 AACTCCCCCTTA motifs detected in file\n",
      "\n",
      "Seq is: CGACCTCTAACA\n",
      "0 CGACCTCTAACA motifs detected in file\n",
      "\n",
      "Seq is: TTACCCATACCT\n",
      "0 TTACCCATACCT motifs detected in file\n",
      "\n",
      "Seq is: CTGTATAGTATA\n",
      "0 CTGTATAGTATA motifs detected in file\n",
      "\n",
      "Seq is: TTCATTACGATC\n",
      "0 TTCATTACGATC motifs detected in file\n",
      "\n",
      "Seq is: CCACTCACCATC\n",
      "0 CCACTCACCATC motifs detected in file\n",
      "\n",
      "Seq is: CTTCAAAAAAGT\n",
      "0 CTTCAAAAAAGT motifs detected in file\n",
      "\n",
      "Seq is: ATCCTCCACATA\n",
      "0 ATCCTCCACATA motifs detected in file\n",
      "\n",
      "Seq is: CTCAACCAGCAT\n",
      "0 CTCAACCAGCAT motifs detected in file\n",
      "\n",
      "Seq is: CGCCTTAGCCCC\n",
      "0 CGCCTTAGCCCC motifs detected in file\n",
      "\n",
      "Seq is: CATACGACAAGC\n",
      "0 CATACGACAAGC motifs detected in file\n",
      "\n",
      "Seq is: TTAAGTATTCTT\n",
      "0 TTAAGTATTCTT motifs detected in file\n",
      "\n",
      "Seq is: GTCCCTCCAATC\n",
      "0 GTCCCTCCAATC motifs detected in file\n",
      "\n",
      "Seq is: TTCATCCCAGAA\n",
      "0 TTCATCCCAGAA motifs detected in file\n",
      "\n",
      "Seq is: CCACTGTTTTAT\n",
      "0 CCACTGTTTTAT motifs detected in file\n",
      "\n",
      "Seq is: ACAACACATCTC\n",
      "0 ACAACACATCTC motifs detected in file\n",
      "\n",
      "Seq is: AAGTCCCCTCTC\n",
      "0 AAGTCCCCTCTC motifs detected in file\n",
      "\n",
      "Seq is: ACGCAGTATACA\n",
      "0 ACGCAGTATACA motifs detected in file\n",
      "\n",
      "Seq is: ACAAGGATCCTT\n",
      "0 ACAAGGATCCTT motifs detected in file\n",
      "\n",
      "Seq is: CTAACATCACCA\n",
      "0 CTAACATCACCA motifs detected in file\n",
      "\n",
      "Seq is: CTCCCACAAATA\n",
      "0 CTCCCACAAATA motifs detected in file\n",
      "\n",
      "Seq is: ATATTATTACAA\n",
      "0 ATATTATTACAA motifs detected in file\n",
      "\n",
      "Seq is: ACACATAAGCCG\n",
      "0 ACACATAAGCCG motifs detected in file\n",
      "\n",
      "Seq is: ATACTTCTTATC\n",
      "0 ATACTTCTTATC motifs detected in file\n",
      "\n",
      "Seq is: CCGCTTCCCCGG\n",
      "0 CCGCTTCCCCGG motifs detected in file\n",
      "\n",
      "Seq is: ACGCCGCATAAG\n",
      "0 ACGCCGCATAAG motifs detected in file\n",
      "\n",
      "Seq is: CTACAACTTCGG\n",
      "0 CTACAACTTCGG motifs detected in file\n",
      "\n",
      "Seq is: ACTCTGATCAAG\n",
      "0 ACTCTGATCAAG motifs detected in file\n",
      "\n",
      "Seq is: CCTGAGGGGAAG\n",
      "0 CCTGAGGGGAAG motifs detected in file\n",
      "\n",
      "Seq is: CCGTTCCAGCTT\n",
      "0 CCGTTCCAGCTT motifs detected in file\n",
      "\n",
      "Seq is: CCCCCAAGCCAA\n",
      "0 CCCCCAAGCCAA motifs detected in file\n",
      "\n",
      "Seq is: CCAGAGACGATC\n",
      "0 CCAGAGACGATC motifs detected in file\n",
      "\n",
      "Seq is: ACACCTCTATTT\n",
      "0 ACACCTCTATTT motifs detected in file\n",
      "\n",
      "Seq is: AGTATAATACCG\n",
      "0 AGTATAATACCG motifs detected in file\n",
      "\n",
      "Seq is: GTTCCCAGTACT\n",
      "0 GTTCCCAGTACT motifs detected in file\n",
      "\n",
      "Seq is: ACCGTCCTCATT\n",
      "0 ACCGTCCTCATT motifs detected in file\n",
      "\n",
      "Seq is: AGTCTCCACAAG\n",
      "0 AGTCTCCACAAG motifs detected in file\n",
      "\n",
      "Seq is: AACACAACACTA\n",
      "0 AACACAACACTA motifs detected in file\n",
      "\n",
      "Seq is: TTCCCACCTCTC\n",
      "0 TTCCCACCTCTC motifs detected in file\n",
      "\n",
      "Seq is: CAATTTCCTCGG\n",
      "0 CAATTTCCTCGG motifs detected in file\n",
      "\n",
      "Seq is: TGAATAATCATT\n",
      "0 TGAATAATCATT motifs detected in file\n",
      "\n",
      "Seq is: ACAACTATCATC\n",
      "0 ACAACTATCATC motifs detected in file\n",
      "\n",
      "Seq is: TTACTACCGATC\n",
      "0 TTACTACCGATC motifs detected in file\n",
      "\n",
      "Seq is: CCACTGAAGTAT\n",
      "0 CCACTGAAGTAT motifs detected in file\n",
      "\n",
      "Seq is: ACTCTAACAAAC\n",
      "0 ACTCTAACAAAC motifs detected in file\n",
      "\n",
      "Seq is: CCGATGACGAGC\n",
      "0 CCGATGACGAGC motifs detected in file\n",
      "\n",
      "Seq is: GTCACGCGGCTA\n",
      "0 GTCACGCGGCTA motifs detected in file\n",
      "\n",
      "Seq is: GTCCCGCATCAA\n",
      "0 GTCCCGCATCAA motifs detected in file\n",
      "\n",
      "Complement found from 3676 to 3688\n",
      "Complement found from 11750 to 11762\n",
      "Seq is: ACTCAAACTACG\n",
      "2 ACTCAAACTACG motifs detected in file\n",
      "\n",
      "Seq is: AGTGGGACTAGT\n",
      "0 AGTGGGACTAGT motifs detected in file\n",
      "\n",
      "Seq is: CTCCTAACTCAT\n",
      "0 CTCCTAACTCAT motifs detected in file\n",
      "\n",
      "Seq is: TTGCCGATCCGA\n",
      "0 TTGCCGATCCGA motifs detected in file\n",
      "\n",
      "Seq is: TCTCCACCGATC\n",
      "0 TCTCCACCGATC motifs detected in file\n",
      "\n",
      "Seq is: TTATTTATCCTC\n",
      "0 TTATTTATCCTC motifs detected in file\n",
      "\n",
      "Seq is: CGGATCCAACTC\n",
      "0 CGGATCCAACTC motifs detected in file\n",
      "\n",
      "Seq is: CAACTGGTCCCC\n",
      "0 CAACTGGTCCCC motifs detected in file\n",
      "\n",
      "Seq is: CAACCCATACCC\n",
      "0 CAACCCATACCC motifs detected in file\n",
      "\n",
      "Seq is: CTCCCCCCAAGT\n",
      "0 CTCCCCCCAAGT motifs detected in file\n",
      "\n",
      "Seq is: ATCATCTTCTCG\n",
      "0 ATCATCTTCTCG motifs detected in file\n",
      "\n",
      "Seq is: CTACCACTCTCG\n",
      "0 CTACCACTCTCG motifs detected in file\n",
      "\n",
      "Seq is: ATTCCAGCCCCG\n",
      "0 ATTCCAGCCCCG motifs detected in file\n",
      "\n",
      "Seq is: CCACTACATCTC\n",
      "0 CCACTACATCTC motifs detected in file\n",
      "\n",
      "Seq is: CCACTACCATCT\n",
      "0 CCACTACCATCT motifs detected in file\n",
      "\n",
      "Seq is: ACACCGCCCAAA\n",
      "0 ACACCGCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: ATCCCCGAGAAA\n",
      "0 ATCCCCGAGAAA motifs detected in file\n",
      "\n",
      "Seq is: CCACTTCTCAAA\n",
      "0 CCACTTCTCAAA motifs detected in file\n",
      "\n",
      "Seq is: ATACCGCAGTCG\n",
      "0 ATACCGCAGTCG motifs detected in file\n",
      "\n",
      "Seq is: CTTCCCAACATC\n",
      "0 CTTCCCAACATC motifs detected in file\n",
      "\n",
      "Seq is: ATCGGGCATCCC\n",
      "0 ATCGGGCATCCC motifs detected in file\n",
      "\n",
      "Seq is: CGGATGTTGCAA\n",
      "0 CGGATGTTGCAA motifs detected in file\n",
      "\n",
      "Seq is: CCCCGGAAACGC\n",
      "0 CCCCGGAAACGC motifs detected in file\n",
      "\n",
      "Seq is: ATCAACATATAT\n",
      "0 ATCAACATATAT motifs detected in file\n",
      "\n",
      "Seq is: CGGATCTTAAAA\n",
      "0 CGGATCTTAAAA motifs detected in file\n",
      "\n",
      "Seq is: AGCAAGCCATTC\n",
      "0 AGCAAGCCATTC motifs detected in file\n",
      "\n",
      "Seq is: GTAATCCTTACG\n",
      "0 GTAATCCTTACG motifs detected in file\n",
      "\n",
      "Seq is: GTAACGCTAATC\n",
      "0 GTAACGCTAATC motifs detected in file\n",
      "\n",
      "Seq is: TTACCCATGTTA\n",
      "0 TTACCCATGTTA motifs detected in file\n",
      "\n",
      "Seq is: CTCCTCATCCTC\n",
      "0 CTCCTCATCCTC motifs detected in file\n",
      "\n",
      "Seq is: CAACCGGTACCC\n",
      "0 CAACCGGTACCC motifs detected in file\n",
      "\n",
      "Seq is: ATACAACAATTC\n",
      "0 ATACAACAATTC motifs detected in file\n",
      "\n",
      "Seq is: TTCTCCTTAACT\n",
      "0 TTCTCCTTAACT motifs detected in file\n",
      "\n",
      "Seq is: TGGAGACTGACA\n",
      "0 TGGAGACTGACA motifs detected in file\n",
      "\n",
      "Seq is: TTTCAAAATTCA\n",
      "0 TTTCAAAATTCA motifs detected in file\n",
      "\n",
      "Seq is: AAATACGCTAAT\n",
      "0 AAATACGCTAAT motifs detected in file\n",
      "\n",
      "Seq is: GGCCCGAGACGG\n",
      "0 GGCCCGAGACGG motifs detected in file\n",
      "\n",
      "Seq is: TAGAATTGTTTG\n",
      "0 TAGAATTGTTTG motifs detected in file\n",
      "\n",
      "Seq is: GGACAAGAACCC\n",
      "0 GGACAAGAACCC motifs detected in file\n",
      "\n",
      "Seq is: ACCCACACCCAT\n",
      "0 ACCCACACCCAT motifs detected in file\n",
      "\n",
      "Seq is: ATTATGATTCAA\n",
      "0 ATTATGATTCAA motifs detected in file\n",
      "\n",
      "Seq is: CTCTACTATAGT\n",
      "0 CTCTACTATAGT motifs detected in file\n",
      "\n",
      "Seq is: AAATGCCCCCTT\n",
      "0 AAATGCCCCCTT motifs detected in file\n",
      "\n",
      "Seq is: CCGCGAAACACT\n",
      "0 CCGCGAAACACT motifs detected in file\n",
      "\n",
      "Seq is: TCATCCGGAATA\n",
      "0 TCATCCGGAATA motifs detected in file\n",
      "\n",
      "Seq is: AAGAGAACAGGA\n",
      "0 AAGAGAACAGGA motifs detected in file\n",
      "\n",
      "Seq is: AAGCATGTCCCT\n",
      "0 AAGCATGTCCCT motifs detected in file\n",
      "\n",
      "Seq is: CCTTAAACTTNC\n",
      "0 CCTTAAACTTNC motifs detected in file\n",
      "\n",
      "Seq is: ATCTATCTTTGG\n",
      "0 ATCTATCTTTGG motifs detected in file\n",
      "\n",
      "Seq is: CTGGACCTAATG\n",
      "0 CTGGACCTAATG motifs detected in file\n",
      "\n",
      "Seq is: AGGCCAGACTTG\n",
      "0 AGGCCAGACTTG motifs detected in file\n",
      "\n",
      "Seq is: AGTCTAGTGCAT\n",
      "0 AGTCTAGTGCAT motifs detected in file\n",
      "\n",
      "Seq is: CCTGAAATTAGC\n",
      "0 CCTGAAATTAGC motifs detected in file\n",
      "\n",
      "Seq is: AACTTGTTTGCT\n",
      "0 AACTTGTTTGCT motifs detected in file\n",
      "\n",
      "Seq is: TGGAAATTATCG\n",
      "0 TGGAAATTATCG motifs detected in file\n",
      "\n",
      "Seq is: CCGACGTGGTAG\n",
      "0 CCGACGTGGTAG motifs detected in file\n",
      "\n",
      "Seq is: CCCTACAGGACT\n",
      "0 CCCTACAGGACT motifs detected in file\n",
      "\n",
      "Seq is: AGGTTGTAGCTC\n",
      "0 AGGTTGTAGCTC motifs detected in file\n",
      "\n",
      "Seq is: CAGCATTTGGGA\n",
      "0 CAGCATTTGGGA motifs detected in file\n",
      "\n",
      "Seq is: TAACAACTATAC\n",
      "0 TAACAACTATAC motifs detected in file\n",
      "\n",
      "Seq is: CTGAGATCTTAT\n",
      "0 CTGAGATCTTAT motifs detected in file\n",
      "\n",
      "Seq is: CCTAACGCGACA\n",
      "0 CCTAACGCGACA motifs detected in file\n",
      "\n",
      "Seq is: ATAGGGATCCCA\n",
      "0 ATAGGGATCCCA motifs detected in file\n",
      "\n",
      "Seq is: TTGAACAAGGCA\n",
      "0 TTGAACAAGGCA motifs detected in file\n",
      "\n",
      "Seq is: ACCAGTTCAATA\n",
      "0 ACCAGTTCAATA motifs detected in file\n",
      "\n",
      "Seq is: ACCTAGTTAACT\n",
      "0 ACCTAGTTAACT motifs detected in file\n",
      "\n",
      "Seq is: CATATCATCAAG\n",
      "0 CATATCATCAAG motifs detected in file\n",
      "\n",
      "Seq is: CGAAACTGACCA\n",
      "0 CGAAACTGACCA motifs detected in file\n",
      "\n",
      "Seq is: CTTCAGAATCGT\n",
      "0 CTTCAGAATCGT motifs detected in file\n",
      "\n",
      "Seq is: ACATGACGAGCC\n",
      "0 ACATGACGAGCC motifs detected in file\n",
      "\n",
      "Seq is: TCCAACCCAAGA\n",
      "0 TCCAACCCAAGA motifs detected in file\n",
      "\n",
      "Seq is: CGAGGCTCCAGC\n",
      "0 CGAGGCTCCAGC motifs detected in file\n",
      "\n",
      "Seq is: GGGGTTGGCTTT\n",
      "0 GGGGTTGGCTTT motifs detected in file\n",
      "\n",
      "Seq is: AAAAATTACGTC\n",
      "0 AAAAATTACGTC motifs detected in file\n",
      "\n",
      "Seq is: CAAACCATCAAA\n",
      "0 CAAACCATCAAA motifs detected in file\n",
      "\n",
      "Seq is: TCCTGGACACCC\n",
      "0 TCCTGGACACCC motifs detected in file\n",
      "\n",
      "Seq is: AAACAATCCATG\n",
      "0 AAACAATCCATG motifs detected in file\n",
      "\n",
      "Seq is: ACAAACGTAATT\n",
      "0 ACAAACGTAATT motifs detected in file\n",
      "\n",
      "Seq is: ATTTAATTTCGA\n",
      "0 ATTTAATTTCGA motifs detected in file\n",
      "\n",
      "Seq is: GGTATCCCAGAA\n",
      "0 GGTATCCCAGAA motifs detected in file\n",
      "\n",
      "Seq is: GAGCAGAACGAC\n",
      "0 GAGCAGAACGAC motifs detected in file\n",
      "\n",
      "Seq is: ACAATACGGGCG\n",
      "0 ACAATACGGGCG motifs detected in file\n",
      "\n",
      "Seq is: GAGAAGTGCCCG\n",
      "0 GAGAAGTGCCCG motifs detected in file\n",
      "\n",
      "Seq is: TCCAGTTAAAGT\n",
      "0 TCCAGTTAAAGT motifs detected in file\n",
      "\n",
      "Seq is: GACCAATTTTCA\n",
      "0 GACCAATTTTCA motifs detected in file\n",
      "\n",
      "Seq is: TTCTCTGTCGAC\n",
      "0 TTCTCTGTCGAC motifs detected in file\n",
      "\n",
      "Seq is: TTGGGAGCACCT\n",
      "0 TTGGGAGCACCT motifs detected in file\n",
      "\n",
      "Seq is: CGGTAAGTATGT\n",
      "0 CGGTAAGTATGT motifs detected in file\n",
      "\n",
      "Seq is: CCAGGGATAAAT\n",
      "0 CCAGGGATAAAT motifs detected in file\n",
      "\n",
      "Seq is: TCCTTGTTCACT\n",
      "0 TCCTTGTTCACT motifs detected in file\n",
      "\n",
      "Seq is: AATACGATGGAA\n",
      "0 AATACGATGGAA motifs detected in file\n",
      "\n",
      "Seq is: ACGTGCCAATCC\n",
      "0 ACGTGCCAATCC motifs detected in file\n",
      "\n",
      "Seq is: CATGGCGCCGGC\n",
      "0 CATGGCGCCGGC motifs detected in file\n",
      "\n",
      "Seq is: AATTTGTACACA\n",
      "0 AATTTGTACACA motifs detected in file\n",
      "\n",
      "Seq is: GTGACCCGTCCG\n",
      "0 GTGACCCGTCCG motifs detected in file\n",
      "\n",
      "Seq is: CCACGGAGATTA\n",
      "0 CCACGGAGATTA motifs detected in file\n",
      "\n",
      "Seq is: TGACCACTACGA\n",
      "0 TGACCACTACGA motifs detected in file\n",
      "\n",
      "Seq is: TCTCCACTACAA\n",
      "0 TCTCCACTACAA motifs detected in file\n",
      "\n",
      "Seq is: AAACCATTTGTC\n",
      "0 AAACCATTTGTC motifs detected in file\n",
      "\n",
      "Seq is: CGCCCCATTCTA\n",
      "0 CGCCCCATTCTA motifs detected in file\n",
      "\n",
      "Seq is: AACGGCTCAAGG\n",
      "0 AACGGCTCAAGG motifs detected in file\n",
      "\n",
      "Seq is: AAAATGAAAAAA\n",
      "0 AAAATGAAAAAA motifs detected in file\n",
      "\n",
      "Seq is: ATTGGAAAGGAA\n",
      "0 ATTGGAAAGGAA motifs detected in file\n",
      "\n",
      "Seq is: TACTCGTACGGA\n",
      "0 TACTCGTACGGA motifs detected in file\n",
      "\n",
      "Seq is: CACAACCCAACT\n",
      "0 CACAACCCAACT motifs detected in file\n",
      "\n",
      "Seq is: GTCACTCCCATT\n",
      "0 GTCACTCCCATT motifs detected in file\n",
      "\n",
      "Seq is: ATTACTGAACAA\n",
      "0 ATTACTGAACAA motifs detected in file\n",
      "\n",
      "Seq is: CCAACTAACATC\n",
      "0 CCAACTAACATC motifs detected in file\n",
      "\n",
      "Seq is: TATAACCCGACA\n",
      "0 TATAACCCGACA motifs detected in file\n",
      "\n",
      "Seq is: ATTAACAGTCAA\n",
      "0 ATTAACAGTCAA motifs detected in file\n",
      "\n",
      "Seq is: GTCACAAAATTA\n",
      "0 GTCACAAAATTA motifs detected in file\n",
      "\n",
      "Seq is: GACTGCGTCCGA\n",
      "0 GACTGCGTCCGA motifs detected in file\n",
      "\n",
      "Seq is: ATACGCCTCCTC\n",
      "0 ATACGCCTCCTC motifs detected in file\n",
      "\n",
      "Seq is: TTACAAAAGTAC\n",
      "0 TTACAAAAGTAC motifs detected in file\n",
      "\n",
      "Seq is: AATGAATATGAT\n",
      "0 AATGAATATGAT motifs detected in file\n",
      "\n",
      "Seq is: TGTAATCAAGAA\n",
      "0 TGTAATCAAGAA motifs detected in file\n",
      "\n",
      "Seq is: GATATCCCACTA\n",
      "0 GATATCCCACTA motifs detected in file\n",
      "\n",
      "Seq is: TCTAACCAGGTT\n",
      "0 TCTAACCAGGTT motifs detected in file\n",
      "\n",
      "Seq is: AACCCACACTCC\n",
      "0 AACCCACACTCC motifs detected in file\n",
      "\n",
      "Seq is: TCAAGTCAATAT\n",
      "0 TCAAGTCAATAT motifs detected in file\n",
      "\n",
      "Seq is: ACAAACCCTAAA\n",
      "0 ACAAACCCTAAA motifs detected in file\n",
      "\n",
      "Seq is: AAATCCATCACC\n",
      "0 AAATCCATCACC motifs detected in file\n",
      "\n",
      "Seq is: CACAACTCGAAC\n",
      "0 CACAACTCGAAC motifs detected in file\n",
      "\n",
      "Seq is: TTGCGAAAGAAT\n",
      "0 TTGCGAAAGAAT motifs detected in file\n",
      "\n",
      "Seq is: TAACCACCGACG\n",
      "0 TAACCACCGACG motifs detected in file\n",
      "\n",
      "Seq is: AAAATCCGGATG\n",
      "0 AAAATCCGGATG motifs detected in file\n",
      "\n",
      "Seq is: ATACCCACAATT\n",
      "0 ATACCCACAATT motifs detected in file\n",
      "\n",
      "Seq is: TAAAAAATGAGA\n",
      "0 TAAAAAATGAGA motifs detected in file\n",
      "\n",
      "Seq is: GAGATGTTCCAA\n",
      "0 GAGATGTTCCAA motifs detected in file\n",
      "\n",
      "Seq is: AAAAGGATCACA\n",
      "0 AAAAGGATCACA motifs detected in file\n",
      "\n",
      "Seq is: GGTTTCTCGACA\n",
      "0 GGTTTCTCGACA motifs detected in file\n",
      "\n",
      "Seq is: AGGAGAAACCTG\n",
      "0 AGGAGAAACCTG motifs detected in file\n",
      "\n",
      "Seq is: ATTGTCAATTTA\n",
      "0 ATTGTCAATTTA motifs detected in file\n",
      "\n",
      "Seq is: AATGTTCCCCTA\n",
      "0 AATGTTCCCCTA motifs detected in file\n",
      "\n",
      "Seq is: AATCTCCCAAGA\n",
      "0 AATCTCCCAAGA motifs detected in file\n",
      "\n",
      "Seq is: CACCCGTTTAAA\n",
      "0 CACCCGTTTAAA motifs detected in file\n",
      "\n",
      "Seq is: TTTCAACTTGAT\n",
      "0 TTTCAACTTGAT motifs detected in file\n",
      "\n",
      "Seq is: TCTAAGATAGAA\n",
      "0 TCTAAGATAGAA motifs detected in file\n",
      "\n",
      "Seq is: CCTGTTGGTCGA\n",
      "0 CCTGTTGGTCGA motifs detected in file\n",
      "\n",
      "Seq is: TAGTGGTCCGAG\n",
      "0 TAGTGGTCCGAG motifs detected in file\n",
      "\n",
      "Seq is: CCATCCAAACAG\n",
      "0 CCATCCAAACAG motifs detected in file\n",
      "\n",
      "Seq is: CGGAGATGGATA\n",
      "0 CGGAGATGGATA motifs detected in file\n",
      "\n",
      "Seq is: TTTAGAAGGGTG\n",
      "0 TTTAGAAGGGTG motifs detected in file\n",
      "\n",
      "Seq is: ATAAAACGATGT\n",
      "0 ATAAAACGATGT motifs detected in file\n",
      "\n",
      "Seq is: ATCTGCCCACAC\n",
      "0 ATCTGCCCACAC motifs detected in file\n",
      "\n",
      "Seq is: GAGAAAATCGAC\n",
      "0 GAGAAAATCGAC motifs detected in file\n",
      "\n",
      "Seq is: AAGAATCCATCG\n",
      "0 AAGAATCCATCG motifs detected in file\n",
      "\n",
      "Seq is: AGCAGACCAAAG\n",
      "0 AGCAGACCAAAG motifs detected in file\n",
      "\n",
      "Seq is: CCCCCAGAATCG\n",
      "0 CCCCCAGAATCG motifs detected in file\n",
      "\n",
      "Seq is: AAACCGAGAGGA\n",
      "0 AAACCGAGAGGA motifs detected in file\n",
      "\n",
      "Seq is: ACGTTTCAATAA\n",
      "0 ACGTTTCAATAA motifs detected in file\n",
      "\n",
      "Seq is: AGATCAATTAAG\n",
      "0 AGATCAATTAAG motifs detected in file\n",
      "\n",
      "Seq is: TAATACGTCTTC\n",
      "0 TAATACGTCTTC motifs detected in file\n",
      "\n",
      "Seq is: CATATCCCCAAT\n",
      "0 CATATCCCCAAT motifs detected in file\n",
      "\n",
      "Seq is: CAGGAACGATAT\n",
      "0 CAGGAACGATAT motifs detected in file\n",
      "\n",
      "Seq is: AATACGAACCAA\n",
      "0 AATACGAACCAA motifs detected in file\n",
      "\n",
      "Seq is: TATTAAAAAGTA\n",
      "0 TATTAAAAAGTA motifs detected in file\n",
      "\n",
      "Seq is: GAAAGGGAACGC\n",
      "0 GAAAGGGAACGC motifs detected in file\n",
      "\n",
      "Seq is: CATGATATAGAT\n",
      "0 CATGATATAGAT motifs detected in file\n",
      "\n",
      "Seq is: AACGCGGTCCAA\n",
      "0 AACGCGGTCCAA motifs detected in file\n",
      "\n",
      "Seq is: AGTTAAAGATAG\n",
      "0 AGTTAAAGATAG motifs detected in file\n",
      "\n",
      "Seq is: CGGATATGAAAT\n",
      "0 CGGATATGAAAT motifs detected in file\n",
      "\n",
      "Seq is: AAACCCATTTAC\n",
      "0 AAACCCATTTAC motifs detected in file\n",
      "\n",
      "Seq is: CAAACCGATTCC\n",
      "0 CAAACCGATTCC motifs detected in file\n",
      "\n",
      "Seq is: AACAGACCATCA\n",
      "0 AACAGACCATCA motifs detected in file\n",
      "\n",
      "Seq is: TTCCACCTCACC\n",
      "0 TTCCACCTCACC motifs detected in file\n",
      "\n",
      "Seq is: CAAACCCCGATC\n",
      "0 CAAACCCCGATC motifs detected in file\n",
      "\n",
      "Seq is: CAAATCGAGTCT\n",
      "0 CAAATCGAGTCT motifs detected in file\n",
      "\n",
      "Seq is: CGCCAGTTCAAT\n",
      "0 CGCCAGTTCAAT motifs detected in file\n",
      "\n",
      "Seq is: TCAACTTTAGAG\n",
      "0 TCAACTTTAGAG motifs detected in file\n",
      "\n",
      "Seq is: GATTCACATTCA\n",
      "0 GATTCACATTCA motifs detected in file\n",
      "\n",
      "Seq is: ACCCACGAAACA\n",
      "0 ACCCACGAAACA motifs detected in file\n",
      "\n",
      "Seq is: CAATTCGATGTG\n",
      "0 CAATTCGATGTG motifs detected in file\n",
      "\n",
      "Seq is: AGACCAAGCAGG\n",
      "0 AGACCAAGCAGG motifs detected in file\n",
      "\n",
      "Seq is: TTCACGTGAAAG\n",
      "0 TTCACGTGAAAG motifs detected in file\n",
      "\n",
      "Seq is: GTCATGTGAATG\n",
      "0 GTCATGTGAATG motifs detected in file\n",
      "\n",
      "Seq is: GTACAATGCTGA\n",
      "0 GTACAATGCTGA motifs detected in file\n",
      "\n",
      "Seq is: ACAGAGGAGATA\n",
      "0 ACAGAGGAGATA motifs detected in file\n",
      "\n",
      "Seq is: TATTTACGCATC\n",
      "0 TATTTACGCATC motifs detected in file\n",
      "\n",
      "Seq is: CCCAAAATCAAT\n",
      "0 CCCAAAATCAAT motifs detected in file\n",
      "\n",
      "Seq is: TTACAGGAAACT\n",
      "0 TTACAGGAAACT motifs detected in file\n",
      "\n",
      "Seq is: TCATATGAACTC\n",
      "0 TCATATGAACTC motifs detected in file\n",
      "\n",
      "Seq is: CTCCCACTGCCC\n",
      "0 CTCCCACTGCCC motifs detected in file\n",
      "\n",
      "Seq is: GCCACACATGCG\n",
      "0 GCCACACATGCG motifs detected in file\n",
      "\n",
      "Seq is: CGAAGTCCCGGG\n",
      "0 CGAAGTCCCGGG motifs detected in file\n",
      "\n",
      "Seq is: ACAAGTTGATTC\n",
      "0 ACAAGTTGATTC motifs detected in file\n",
      "\n",
      "Seq is: GTGAGATGAGAA\n",
      "0 GTGAGATGAGAA motifs detected in file\n",
      "\n",
      "Seq is: TCAAATGACGAT\n",
      "0 TCAAATGACGAT motifs detected in file\n",
      "\n",
      "Seq is: TTAGGTGGAAGC\n",
      "0 TTAGGTGGAAGC motifs detected in file\n",
      "\n",
      "Seq is: TGGGAATTCAAA\n",
      "0 TGGGAATTCAAA motifs detected in file\n",
      "\n",
      "Seq is: GTATTCCCGATA\n",
      "0 GTATTCCCGATA motifs detected in file\n",
      "\n",
      "Seq is: GCATCAAAAGAC\n",
      "0 GCATCAAAAGAC motifs detected in file\n",
      "\n",
      "Seq is: CCCATCTTTTAC\n",
      "0 CCCATCTTTTAC motifs detected in file\n",
      "\n",
      "Seq is: ATCGGGTAAAGA\n",
      "0 ATCGGGTAAAGA motifs detected in file\n",
      "\n",
      "Seq is: ACGGTGGAGTAC\n",
      "0 ACGGTGGAGTAC motifs detected in file\n",
      "\n",
      "Seq is: CCGATGTGGAAC\n",
      "0 CCGATGTGGAAC motifs detected in file\n",
      "\n",
      "Seq is: TGGATTGCAGAA\n",
      "0 TGGATTGCAGAA motifs detected in file\n",
      "\n",
      "Seq is: ATGCACCCATGA\n",
      "0 ATGCACCCATGA motifs detected in file\n",
      "\n",
      "Seq is: ACGCGAATGAAA\n",
      "0 ACGCGAATGAAA motifs detected in file\n",
      "\n",
      "Seq is: CATCGGAAGTAG\n",
      "0 CATCGGAAGTAG motifs detected in file\n",
      "\n",
      "Seq is: TCCCAAACGACT\n",
      "0 TCCCAAACGACT motifs detected in file\n",
      "\n",
      "Seq is: TCTACCGCCATA\n",
      "0 TCTACCGCCATA motifs detected in file\n",
      "\n",
      "Seq is: TATCCGACTCGT\n",
      "0 TATCCGACTCGT motifs detected in file\n",
      "\n",
      "Seq is: TCTCCACCACTC\n",
      "0 TCTCCACCACTC motifs detected in file\n",
      "\n",
      "Seq is: CAACTAGCCCCA\n",
      "0 CAACTAGCCCCA motifs detected in file\n",
      "\n",
      "Seq is: AATAGCTAATGT\n",
      "0 AATAGCTAATGT motifs detected in file\n",
      "\n",
      "Seq is: CTTGTCCGAGGA\n",
      "0 CTTGTCCGAGGA motifs detected in file\n",
      "\n",
      "Seq is: GATCTCCCTATA\n",
      "0 GATCTCCCTATA motifs detected in file\n",
      "\n",
      "Seq is: CTTCGTGGCGGT\n",
      "0 CTTCGTGGCGGT motifs detected in file\n",
      "\n",
      "Seq is: CCAGGAAACTCA\n",
      "0 CCAGGAAACTCA motifs detected in file\n",
      "\n",
      "Seq is: AAATTCGACACC\n",
      "0 AAATTCGACACC motifs detected in file\n",
      "\n",
      "Seq is: GAGCATCACAAG\n",
      "0 GAGCATCACAAG motifs detected in file\n",
      "\n",
      "Seq is: ACCGCTCGTCAA\n",
      "0 ACCGCTCGTCAA motifs detected in file\n",
      "\n",
      "Seq is: AACAACTAAATT\n",
      "0 AACAACTAAATT motifs detected in file\n",
      "\n",
      "Seq is: GACAACTCCAAA\n",
      "0 GACAACTCCAAA motifs detected in file\n",
      "\n",
      "Seq is: TCCCGATTCGTA\n",
      "0 TCCCGATTCGTA motifs detected in file\n",
      "\n",
      "Seq is: TCACCCCATAGA\n",
      "0 TCACCCCATAGA motifs detected in file\n",
      "\n",
      "Seq is: TTAGGGTCAAAC\n",
      "0 TTAGGGTCAAAC motifs detected in file\n",
      "\n",
      "Seq is: CCAGAATCGATA\n",
      "0 CCAGAATCGATA motifs detected in file\n",
      "\n",
      "Seq is: ACACACAAGTCT\n",
      "0 ACACACAAGTCT motifs detected in file\n",
      "\n",
      "Seq is: ATACAATTTCGG\n",
      "0 ATACAATTTCGG motifs detected in file\n",
      "\n",
      "Seq is: TGAAAGCATCAG\n",
      "0 TGAAAGCATCAG motifs detected in file\n",
      "\n",
      "Seq is: ATAAAACACAGT\n",
      "0 ATAAAACACAGT motifs detected in file\n",
      "\n",
      "Seq is: TGACCTCAAAAA\n",
      "0 TGACCTCAAAAA motifs detected in file\n",
      "\n",
      "Seq is: ATGTTGAGTCCA\n",
      "0 ATGTTGAGTCCA motifs detected in file\n",
      "\n",
      "Seq is: CTCAAAATCGAA\n",
      "0 CTCAAAATCGAA motifs detected in file\n",
      "\n",
      "Seq is: ATAACCCCTCCC\n",
      "0 ATAACCCCTCCC motifs detected in file\n",
      "\n",
      "Seq is: CCACTAGATTTT\n",
      "0 CCACTAGATTTT motifs detected in file\n",
      "\n",
      "Seq is: GTGAGAAATGCG\n",
      "0 GTGAGAAATGCG motifs detected in file\n",
      "\n",
      "Seq is: GCCGAAGATAAC\n",
      "0 GCCGAAGATAAC motifs detected in file\n",
      "\n",
      "Seq is: TGAACCCAATTA\n",
      "0 TGAACCCAATTA motifs detected in file\n",
      "\n",
      "Seq is: GCACACTGGCGC\n",
      "0 GCACACTGGCGC motifs detected in file\n",
      "\n",
      "Seq is: CACCGACCGTGC\n",
      "0 CACCGACCGTGC motifs detected in file\n",
      "\n",
      "Seq is: TTTAACTGGTTG\n",
      "0 TTTAACTGGTTG motifs detected in file\n",
      "\n",
      "Seq is: GGACCCCAATCA\n",
      "0 GGACCCCAATCA motifs detected in file\n",
      "\n",
      "Seq is: TATCGAATCAAT\n",
      "0 TATCGAATCAAT motifs detected in file\n",
      "\n",
      "Seq is: TTGAAAGCAAAT\n",
      "0 TTGAAAGCAAAT motifs detected in file\n",
      "\n",
      "Seq is: AACGATTTCCAA\n",
      "0 AACGATTTCCAA motifs detected in file\n",
      "\n",
      "Seq is: TTAGTGACGACA\n",
      "0 TTAGTGACGACA motifs detected in file\n",
      "\n",
      "Seq is: AAGGGCACCCCC\n",
      "0 AAGGGCACCCCC motifs detected in file\n",
      "\n",
      "Seq is: ACACCGATCCGA\n",
      "0 ACACCGATCCGA motifs detected in file\n",
      "\n",
      "Seq is: TTCGCAAAACTC\n",
      "0 TTCGCAAAACTC motifs detected in file\n",
      "\n",
      "Seq is: GACGTAACGACG\n",
      "0 GACGTAACGACG motifs detected in file\n",
      "\n",
      "Seq is: CACGAACTACGA\n",
      "0 CACGAACTACGA motifs detected in file\n",
      "\n",
      "Seq is: ACAAGGAAAACT\n",
      "0 ACAAGGAAAACT motifs detected in file\n",
      "\n",
      "Seq is: AGCACCACTAAA\n",
      "0 AGCACCACTAAA motifs detected in file\n",
      "\n",
      "Seq is: TCTCCCACTTGA\n",
      "0 TCTCCCACTTGA motifs detected in file\n",
      "\n",
      "Seq is: GTGACCTTGCCC\n",
      "0 GTGACCTTGCCC motifs detected in file\n",
      "\n",
      "Seq is: CTACGAACGTAC\n",
      "0 CTACGAACGTAC motifs detected in file\n",
      "\n",
      "Seq is: ACATTAGAATGA\n",
      "0 ACATTAGAATGA motifs detected in file\n",
      "\n",
      "Seq is: TTCTCGATTATC\n",
      "0 TTCTCGATTATC motifs detected in file\n",
      "\n",
      "Seq is: TTTCCGATCCTG\n",
      "0 TTTCCGATCCTG motifs detected in file\n",
      "\n",
      "Seq is: GTTTGGATAAAC\n",
      "0 GTTTGGATAAAC motifs detected in file\n",
      "\n",
      "Seq is: AAATACCCCACT\n",
      "0 AAATACCCCACT motifs detected in file\n",
      "\n",
      "Seq is: ACACTCGGGCAG\n",
      "0 ACACTCGGGCAG motifs detected in file\n",
      "\n",
      "Seq is: ATTTGTAAAAGT\n",
      "0 ATTTGTAAAAGT motifs detected in file\n",
      "\n",
      "Seq is: CACATAACGAAA\n",
      "0 CACATAACGAAA motifs detected in file\n",
      "\n",
      "Seq is: CTCCTCCATTCG\n",
      "0 CTCCTCCATTCG motifs detected in file\n",
      "\n",
      "Seq is: ATGTATTTGACA\n",
      "0 ATGTATTTGACA motifs detected in file\n",
      "\n",
      "Seq is: CCCCCCACAGAA\n",
      "0 CCCCCCACAGAA motifs detected in file\n",
      "\n",
      "Seq is: ACCCCAAACCAA\n",
      "0 ACCCCAAACCAA motifs detected in file\n",
      "\n",
      "Seq is: CCAAGCCCCATA\n",
      "0 CCAAGCCCCATA motifs detected in file\n",
      "\n",
      "Seq is: CCCCAATCGTCG\n",
      "0 CCCCAATCGTCG motifs detected in file\n",
      "\n",
      "Seq is: CCACACACACAC\n",
      "0 CCACACACACAC motifs detected in file\n",
      "\n",
      "Seq is: GACCCATCCTAC\n",
      "0 GACCCATCCTAC motifs detected in file\n",
      "\n",
      "Seq is: CCGCCCCCAACA\n",
      "0 CCGCCCCCAACA motifs detected in file\n",
      "\n",
      "Seq is: TAACTACTCTAA\n",
      "0 TAACTACTCTAA motifs detected in file\n",
      "\n",
      "Seq is: TCATCATACCCT\n",
      "0 TCATCATACCCT motifs detected in file\n",
      "\n",
      "Seq is: CACCCTCCCCTT\n",
      "0 CACCCTCCCCTT motifs detected in file\n",
      "\n",
      "Seq is: TTATTACACAAT\n",
      "0 TTATTACACAAT motifs detected in file\n",
      "\n",
      "Seq is: CAACCCCCCACT\n",
      "0 CAACCCCCCACT motifs detected in file\n",
      "\n",
      "Seq is: GACAATTTTCAC\n",
      "0 GACAATTTTCAC motifs detected in file\n",
      "\n",
      "Seq is: GTATGGCGGTTT\n",
      "0 GTATGGCGGTTT motifs detected in file\n",
      "\n",
      "Seq is: TCTATTTTAAAC\n",
      "0 TCTATTTTAAAC motifs detected in file\n",
      "\n",
      "Seq is: TTTAGACCAATC\n",
      "0 TTTAGACCAATC motifs detected in file\n",
      "\n",
      "Seq is: CGACCACAATCC\n",
      "0 CGACCACAATCC motifs detected in file\n",
      "\n",
      "Seq is: CAAGAAACAAAA\n",
      "0 CAAGAAACAAAA motifs detected in file\n",
      "\n",
      "Seq is: ACCCCAAACCGT\n",
      "0 ACCCCAAACCGT motifs detected in file\n",
      "\n",
      "Seq is: CTCTACACAAAT\n",
      "0 CTCTACACAAAT motifs detected in file\n",
      "\n",
      "Seq is: TCACGACACCGG\n",
      "0 TCACGACACCGG motifs detected in file\n",
      "\n",
      "Seq is: TCTTCGCCCCCT\n",
      "0 TCTTCGCCCCCT motifs detected in file\n",
      "\n",
      "Seq is: CCCCCCCAAACC\n",
      "0 CCCCCCCAAACC motifs detected in file\n",
      "\n",
      "Seq is: ACCTTTAAAAAA\n",
      "0 ACCTTTAAAAAA motifs detected in file\n",
      "\n",
      "Seq is: CAATACTACAGA\n",
      "0 CAATACTACAGA motifs detected in file\n",
      "\n",
      "Seq is: CACACCTTTCAC\n",
      "0 CACACCTTTCAC motifs detected in file\n",
      "\n",
      "Seq is: CGACACGTCTGT\n",
      "0 CGACACGTCTGT motifs detected in file\n",
      "\n",
      "Seq is: AAGTTAACAATA\n",
      "0 AAGTTAACAATA motifs detected in file\n",
      "\n",
      "Seq is: ATAATACAGGAT\n",
      "0 ATAATACAGGAT motifs detected in file\n",
      "\n",
      "Seq is: GTTCGTAATTAA\n",
      "0 GTTCGTAATTAA motifs detected in file\n",
      "\n",
      "Seq is: TTAATTGTGTGA\n",
      "0 TTAATTGTGTGA motifs detected in file\n",
      "\n",
      "Seq is: AATCATTCATAC\n",
      "0 AATCATTCATAC motifs detected in file\n",
      "\n",
      "Seq is: AAGCGGACATTA\n",
      "0 AAGCGGACATTA motifs detected in file\n",
      "\n",
      "Seq is: TAACTTGCATCC\n",
      "0 TAACTTGCATCC motifs detected in file\n",
      "\n",
      "Seq is: ACGCTATTTATT\n",
      "0 ACGCTATTTATT motifs detected in file\n",
      "\n",
      "Seq is: ATCCTACTCCGT\n",
      "0 ATCCTACTCCGT motifs detected in file\n",
      "\n",
      "Seq is: CCTTAGTTTCTG\n",
      "0 CCTTAGTTTCTG motifs detected in file\n",
      "\n",
      "Seq is: TCTATGACGCTG\n",
      "0 TCTATGACGCTG motifs detected in file\n",
      "\n",
      "Seq is: TATCCCACGAGG\n",
      "0 TATCCCACGAGG motifs detected in file\n",
      "\n",
      "Seq is: CCGAGGTCGCAG\n",
      "0 CCGAGGTCGCAG motifs detected in file\n",
      "\n",
      "Seq is: AGCGTTACGATA\n",
      "0 AGCGTTACGATA motifs detected in file\n",
      "\n",
      "Seq is: GCGCACGTATGG\n",
      "0 GCGCACGTATGG motifs detected in file\n",
      "\n",
      "Seq is: GGGGTCTGCTTT\n",
      "0 GGGGTCTGCTTT motifs detected in file\n",
      "\n",
      "Seq is: TATGGTTTACGT\n",
      "0 TATGGTTTACGT motifs detected in file\n",
      "\n",
      "Seq is: ACCTCTCGAGGG\n",
      "0 ACCTCTCGAGGG motifs detected in file\n",
      "\n",
      "Seq is: CACTCACCAATT\n",
      "0 CACTCACCAATT motifs detected in file\n",
      "\n",
      "Seq is: ATCCCACTATCT\n",
      "0 ATCCCACTATCT motifs detected in file\n",
      "\n",
      "Seq is: GTAGCACTACAGA\n",
      "0 GTAGCACTACAGA motifs detected in file\n",
      "\n",
      "Seq is: ATAAATTCCCCTT\n",
      "0 ATAAATTCCCCTT motifs detected in file\n",
      "\n",
      "Seq is: GCACACCCGATAA\n",
      "0 GCACACCCGATAA motifs detected in file\n",
      "\n",
      "Seq is: ATCCGAAATACTG\n",
      "0 ATCCGAAATACTG motifs detected in file\n",
      "\n",
      "Seq is: GGACTTCATCCTT\n",
      "0 GGACTTCATCCTT motifs detected in file\n",
      "\n",
      "Seq is: GGTCTACAGCCTA\n",
      "0 GGTCTACAGCCTA motifs detected in file\n",
      "\n",
      "Seq is: TGTCAAGTGAAAT\n",
      "0 TGTCAAGTGAAAT motifs detected in file\n",
      "\n",
      "Seq is: CGATGGGGGTTCA\n",
      "0 CGATGGGGGTTCA motifs detected in file\n",
      "\n",
      "Seq is: CAATACCCGGGCC\n",
      "0 CAATACCCGGGCC motifs detected in file\n",
      "\n",
      "Seq is: TCGCTCCTCTCAT\n",
      "0 TCGCTCCTCTCAT motifs detected in file\n",
      "\n",
      "Seq is: CGTGAGAACACGC\n",
      "0 CGTGAGAACACGC motifs detected in file\n",
      "\n",
      "Seq is: CCTATAACTAAAG\n",
      "0 CCTATAACTAAAG motifs detected in file\n",
      "\n",
      "Seq is: TGCCTCCTACCAC\n",
      "0 TGCCTCCTACCAC motifs detected in file\n",
      "\n",
      "Seq is: CAGTTCCCTGGGG\n",
      "0 CAGTTCCCTGGGG motifs detected in file\n",
      "\n",
      "Seq is: ATAGACTCCCCCC\n",
      "0 ATAGACTCCCCCC motifs detected in file\n",
      "\n",
      "Seq is: AGTAGGTACCCCT\n",
      "0 AGTAGGTACCCCT motifs detected in file\n",
      "\n",
      "Seq is: GCTCTTCCCTAAA\n",
      "0 GCTCTTCCCTAAA motifs detected in file\n",
      "\n",
      "Seq is: CTGACATTACACG\n",
      "0 CTGACATTACACG motifs detected in file\n",
      "\n",
      "Seq is: ATACATGCCATTT\n",
      "0 ATACATGCCATTT motifs detected in file\n",
      "\n",
      "Seq is: ACCGAAATACATG\n",
      "0 ACCGAAATACATG motifs detected in file\n",
      "\n",
      "Seq is: ATACATGACAATT\n",
      "0 ATACATGACAATT motifs detected in file\n",
      "\n",
      "Seq is: CCCACCCATCCAA\n",
      "0 CCCACCCATCCAA motifs detected in file\n",
      "\n",
      "Seq is: ACAACCATAGGAT\n",
      "0 ACAACCATAGGAT motifs detected in file\n",
      "\n",
      "Seq is: CACCCACTCCCCA\n",
      "0 CACCCACTCCCCA motifs detected in file\n",
      "\n",
      "Seq is: CCGAAACCTCAAC\n",
      "0 CCGAAACCTCAAC motifs detected in file\n",
      "\n",
      "Seq is: GTCAACTACACAC\n",
      "0 GTCAACTACACAC motifs detected in file\n",
      "\n",
      "Seq is: TATCAACTCCCAA\n",
      "0 TATCAACTCCCAA motifs detected in file\n",
      "\n",
      "Seq is: CTAACGACATGAA\n",
      "0 CTAACGACATGAA motifs detected in file\n",
      "\n",
      "Seq is: CGAACATTCGTAC\n",
      "0 CGAACATTCGTAC motifs detected in file\n",
      "\n",
      "Seq is: CCCTCCCCCAAAA\n",
      "0 CCCTCCCCCAAAA motifs detected in file\n",
      "\n",
      "Seq is: CTACACCTAACCC\n",
      "0 CTACACCTAACCC motifs detected in file\n",
      "\n",
      "Seq is: AAAAATACATGAT\n",
      "0 AAAAATACATGAT motifs detected in file\n",
      "\n",
      "Seq is: GTCCACCAGTTCA\n",
      "0 GTCCACCAGTTCA motifs detected in file\n",
      "\n",
      "Seq is: TAAATACCATGGC\n",
      "0 TAAATACCATGGC motifs detected in file\n",
      "\n",
      "Seq is: ATGTTATAAGTAC\n",
      "0 ATGTTATAAGTAC motifs detected in file\n",
      "\n",
      "Seq is: CACCGACCGTCAT\n",
      "0 CACCGACCGTCAT motifs detected in file\n",
      "\n",
      "Seq is: TACATGCTTTATG\n",
      "0 TACATGCTTTATG motifs detected in file\n",
      "\n",
      "Seq is: TATCGCCAACAAC\n",
      "0 TATCGCCAACAAC motifs detected in file\n",
      "\n",
      "Seq is: TACCCACTCAGTT\n",
      "0 TACCCACTCAGTT motifs detected in file\n",
      "\n",
      "Seq is: ATGAACCCACCAT\n",
      "0 ATGAACCCACCAT motifs detected in file\n",
      "\n",
      "Seq is: GGGTTTAGACGAA\n",
      "0 GGGTTTAGACGAA motifs detected in file\n",
      "\n",
      "Seq is: GGGGTACTTTCTT\n",
      "0 GGGGTACTTTCTT motifs detected in file\n",
      "\n",
      "Seq is: GTCTCTTATCAAA\n",
      "0 GTCTCTTATCAAA motifs detected in file\n",
      "\n",
      "Seq is: TTTAATCTTAGAA\n",
      "0 TTTAATCTTAGAA motifs detected in file\n",
      "\n",
      "Seq is: TCGAAACCCACGA\n",
      "0 TCGAAACCCACGA motifs detected in file\n",
      "\n",
      "Seq is: TTACCACCTCAAT\n",
      "0 TTACCACCTCAAT motifs detected in file\n",
      "\n",
      "Seq is: TTCTGAAAAAGAG\n",
      "0 TTCTGAAAAAGAG motifs detected in file\n",
      "\n",
      "Seq is: ACTAAACAGGAAC\n",
      "0 ACTAAACAGGAAC motifs detected in file\n",
      "\n",
      "Seq is: CTTTTTCCAAAAG\n",
      "0 CTTTTTCCAAAAG motifs detected in file\n",
      "\n",
      "Seq is: TAGAGGCCAAATG\n",
      "0 TAGAGGCCAAATG motifs detected in file\n",
      "\n",
      "Seq is: TTCTGACCACATA\n",
      "0 TTCTGACCACATA motifs detected in file\n",
      "\n",
      "Seq is: ATCAAATATGATG\n",
      "0 ATCAAATATGATG motifs detected in file\n",
      "\n",
      "Seq is: TTCCTGTCCGGGT\n",
      "0 TTCCTGTCCGGGT motifs detected in file\n",
      "\n",
      "Seq is: AAACTCATAAAAC\n",
      "0 AAACTCATAAAAC motifs detected in file\n",
      "\n",
      "Seq is: AAAAGTTAATCCC\n",
      "0 AAAAGTTAATCCC motifs detected in file\n",
      "\n",
      "Seq is: TCTATCAACCATA\n",
      "0 TCTATCAACCATA motifs detected in file\n",
      "\n",
      "Complement found from 15829 to 15842\n",
      "Seq is: ATCCTAATCCTAA\n",
      "1 ATCCTAATCCTAA motifs detected in file\n",
      "\n",
      "Seq is: CAACACTTCATAT\n",
      "0 CAACACTTCATAT motifs detected in file\n",
      "\n",
      "Seq is: CATGCCTACGATG\n",
      "0 CATGCCTACGATG motifs detected in file\n",
      "\n",
      "Seq is: AACAGGTTACTAC\n",
      "0 AACAGGTTACTAC motifs detected in file\n",
      "\n",
      "Seq is: CATTTTCCCATCG\n",
      "0 CATTTTCCCATCG motifs detected in file\n",
      "\n",
      "Seq is: AATGACCAACAGG\n",
      "0 AATGACCAACAGG motifs detected in file\n",
      "\n",
      "Seq is: AGGCTAAGTCCAA\n",
      "0 AGGCTAAGTCCAA motifs detected in file\n",
      "\n",
      "Seq is: TCTTACTCCTCCA\n",
      "0 TCTTACTCCTCCA motifs detected in file\n",
      "\n",
      "Seq is: GACGCCGATCCTC\n",
      "0 GACGCCGATCCTC motifs detected in file\n",
      "\n",
      "Seq is: AGTTATTTCACTA\n",
      "0 AGTTATTTCACTA motifs detected in file\n",
      "\n",
      "Seq is: ACCGAATCACCCG\n",
      "0 ACCGAATCACCCG motifs detected in file\n",
      "\n",
      "Seq is: CTTTATAATACGA\n",
      "0 CTTTATAATACGA motifs detected in file\n",
      "\n",
      "Seq is: AACAACAAACCTA\n",
      "0 AACAACAAACCTA motifs detected in file\n",
      "\n",
      "Seq is: TATACCTCCTACC\n",
      "0 TATACCTCCTACC motifs detected in file\n",
      "\n",
      "Seq is: CCTAATAACGATC\n",
      "0 CCTAATAACGATC motifs detected in file\n",
      "\n",
      "Complement found from 5471 to 5484\n",
      "Seq is: CTACTCCTACCTA\n",
      "1 CTACTCCTACCTA motifs detected in file\n",
      "\n",
      "Seq is: TCATTATCCCGTT\n",
      "0 TCATTATCCCGTT motifs detected in file\n",
      "\n",
      "Seq is: CCTGCGGAGGATC\n",
      "0 CCTGCGGAGGATC motifs detected in file\n",
      "\n",
      "Seq is: AAACAATCCCTGC\n",
      "0 AAACAATCCCTGC motifs detected in file\n",
      "\n",
      "Seq is: CTAGCCTCTTAAC\n",
      "0 CTAGCCTCTTAAC motifs detected in file\n",
      "\n",
      "Seq is: ACATCCGCTTATC\n",
      "0 ACATCCGCTTATC motifs detected in file\n",
      "\n",
      "Seq is: CTTTATAGTAAGC\n",
      "0 CTTTATAGTAAGC motifs detected in file\n",
      "\n",
      "Seq is: CCGAACTACACCC\n",
      "0 CCGAACTACACCC motifs detected in file\n",
      "\n",
      "Seq is: CTCCCCACAAATT\n",
      "0 CTCCCCACAAATT motifs detected in file\n",
      "\n",
      "Seq is: CCCCAACCGATCC\n",
      "0 CCCCAACCGATCC motifs detected in file\n",
      "\n",
      "Seq is: CATATTAACAGAC\n",
      "0 CATATTAACAGAC motifs detected in file\n",
      "\n",
      "Seq is: CCAGCGGATCCTC\n",
      "0 CCAGCGGATCCTC motifs detected in file\n",
      "\n",
      "Seq is: CAGACCACTCTTA\n",
      "0 CAGACCACTCTTA motifs detected in file\n",
      "\n",
      "Seq is: TCACAATTACAGT\n",
      "0 TCACAATTACAGT motifs detected in file\n",
      "\n",
      "Seq is: AATTCCTCTCTTC\n",
      "0 AATTCCTCTCTTC motifs detected in file\n",
      "\n",
      "Seq is: CTTCTCTTCATTC\n",
      "0 CTTCTCTTCATTC motifs detected in file\n",
      "\n",
      "Seq is: GGCTCCCGCAGAA\n",
      "0 GGCTCCCGCAGAA motifs detected in file\n",
      "\n",
      "Seq is: ACTAACACATCAT\n",
      "0 ACTAACACATCAT motifs detected in file\n",
      "\n",
      "Seq is: TCCCACCTTCCAC\n",
      "0 TCCCACCTTCCAC motifs detected in file\n",
      "\n",
      "Seq is: TAAAATAGCCTTA\n",
      "0 TAAAATAGCCTTA motifs detected in file\n",
      "\n",
      "Seq is: CCCTCCACTAAGG\n",
      "0 CCCTCCACTAAGG motifs detected in file\n",
      "\n",
      "Seq is: ATCCCCCAACAAA\n",
      "0 ATCCCCCAACAAA motifs detected in file\n",
      "\n",
      "Seq is: CTAGGGCAAAGCA\n",
      "0 CTAGGGCAAAGCA motifs detected in file\n",
      "\n",
      "Seq is: CGTTCTTATCCTC\n",
      "0 CGTTCTTATCCTC motifs detected in file\n",
      "\n",
      "Seq is: CACCTCACAACGA\n",
      "0 CACCTCACAACGA motifs detected in file\n",
      "\n",
      "Seq is: TCCCGACGTTATT\n",
      "0 TCCCGACGTTATT motifs detected in file\n",
      "\n",
      "Seq is: ACTTCCCGTTCTA\n",
      "0 ACTTCCCGTTCTA motifs detected in file\n",
      "\n",
      "Seq is: CTTCACTTTCCAT\n",
      "0 CTTCACTTTCCAT motifs detected in file\n",
      "\n",
      "Seq is: TTCTTAGCACACT\n",
      "0 TTCTTAGCACACT motifs detected in file\n",
      "\n",
      "Seq is: CCCACCCTGACAG\n",
      "0 CCCACCCTGACAG motifs detected in file\n",
      "\n",
      "Seq is: ATGACTCATCGGA\n",
      "0 ATGACTCATCGGA motifs detected in file\n",
      "\n",
      "Seq is: GGAGTCTAAGTAA\n",
      "0 GGAGTCTAAGTAA motifs detected in file\n",
      "\n",
      "Seq is: CTTGATCCAGACA\n",
      "0 CTTGATCCAGACA motifs detected in file\n",
      "\n",
      "Seq is: GGGTTACATACCC\n",
      "0 GGGTTACATACCC motifs detected in file\n",
      "\n",
      "Seq is: TACCGCCTATCAT\n",
      "0 TACCGCCTATCAT motifs detected in file\n",
      "\n",
      "Seq is: TCAAACATTAATG\n",
      "0 TCAAACATTAATG motifs detected in file\n",
      "\n",
      "Seq is: ACACCGGGGAGTC\n",
      "0 ACACCGGGGAGTC motifs detected in file\n",
      "\n",
      "Seq is: TTACTATAAACCG\n",
      "0 TTACTATAAACCG motifs detected in file\n",
      "\n",
      "Seq is: GAGTGCCCTCCTG\n",
      "0 GAGTGCCCTCCTG motifs detected in file\n",
      "\n",
      "Seq is: TATCGGATACTTC\n",
      "0 TATCGGATACTTC motifs detected in file\n",
      "\n",
      "Seq is: CGACAACGATATC\n",
      "0 CGACAACGATATC motifs detected in file\n",
      "\n",
      "Seq is: AACGTTCGTCCTC\n",
      "0 AACGTTCGTCCTC motifs detected in file\n",
      "\n",
      "Seq is: CTATTACGGCTAC\n",
      "0 CTATTACGGCTAC motifs detected in file\n",
      "\n",
      "Seq is: AAAGTCCAAAGAC\n",
      "0 AAAGTCCAAAGAC motifs detected in file\n",
      "\n",
      "Seq is: TCATCTCTTTACT\n",
      "0 TCATCTCTTTACT motifs detected in file\n",
      "\n",
      "Seq is: AGGCATTATATCC\n",
      "0 AGGCATTATATCC motifs detected in file\n",
      "\n",
      "Seq is: GGAGCGGGCTACA\n",
      "0 GGAGCGGGCTACA motifs detected in file\n",
      "\n",
      "Seq is: CATCCTTCTCCGT\n",
      "0 CATCCTTCTCCGT motifs detected in file\n",
      "\n",
      "Seq is: CTATTTCTTATAA\n",
      "0 CTATTTCTTATAA motifs detected in file\n",
      "\n",
      "Seq is: CTCCGCGGTAACC\n",
      "0 CTCCGCGGTAACC motifs detected in file\n",
      "\n",
      "Seq is: GCACTTCCATCGC\n",
      "0 GCACTTCCATCGC motifs detected in file\n",
      "\n",
      "Seq is: CTACTAAGTCGGT\n",
      "0 CTACTAAGTCGGT motifs detected in file\n",
      "\n",
      "Seq is: ATTAAATGCAGAG\n",
      "0 ATTAAATGCAGAG motifs detected in file\n",
      "\n",
      "Seq is: CTCACTACACCCG\n",
      "0 CTCACTACACCCG motifs detected in file\n",
      "\n",
      "Seq is: CTAACTACTTTTC\n",
      "0 CTAACTACTTTTC motifs detected in file\n",
      "\n",
      "Seq is: CGCCAACTCCGCA\n",
      "0 CGCCAACTCCGCA motifs detected in file\n",
      "\n",
      "Seq is: GACCACTCATCAC\n",
      "0 GACCACTCATCAC motifs detected in file\n",
      "\n",
      "Seq is: GTACCGATCCTTA\n",
      "0 GTACCGATCCTTA motifs detected in file\n",
      "\n",
      "Seq is: TCAGGACACCACT\n",
      "0 TCAGGACACCACT motifs detected in file\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Seq is: AAACCTCCTAGTC\n",
      "0 AAACCTCCTAGTC motifs detected in file\n",
      "\n",
      "Seq is: CGTCCGCGGTTCC\n",
      "0 CGTCCGCGGTTCC motifs detected in file\n",
      "\n",
      "Seq is: TCACTCGGCTTCA\n",
      "0 TCACTCGGCTTCA motifs detected in file\n",
      "\n",
      "Seq is: AAGTAGTACGCCT\n",
      "0 AAGTAGTACGCCT motifs detected in file\n",
      "\n",
      "Seq is: CTACAACCTACCC\n",
      "0 CTACAACCTACCC motifs detected in file\n",
      "\n",
      "Seq is: CACCCCTCCAGCT\n",
      "0 CACCCCTCCAGCT motifs detected in file\n",
      "\n",
      "Seq is: ACTTACTCACCAA\n",
      "0 ACTTACTCACCAA motifs detected in file\n",
      "\n",
      "Seq is: TTAATTAAAATAA\n",
      "0 TTAATTAAAATAA motifs detected in file\n",
      "\n",
      "Seq is: TCCCCCAATCAAA\n",
      "0 TCCCCCAATCAAA motifs detected in file\n",
      "\n",
      "Seq is: ACGCATAACCCCA\n",
      "0 ACGCATAACCCCA motifs detected in file\n",
      "\n",
      "Seq is: GTAACCACAAGAA\n",
      "0 GTAACCACAAGAA motifs detected in file\n",
      "\n",
      "Seq is: CATCAACTTTATG\n",
      "0 CATCAACTTTATG motifs detected in file\n",
      "\n",
      "Seq is: TTGCTACCAAAAA\n",
      "0 TTGCTACCAAAAA motifs detected in file\n",
      "\n",
      "Seq is: GTATAGTAACCAG\n",
      "0 GTATAGTAACCAG motifs detected in file\n",
      "\n",
      "Seq is: CACCAACATCAGG\n",
      "0 CACCAACATCAGG motifs detected in file\n",
      "\n",
      "Seq is: CACGCTCTTATTA\n",
      "0 CACGCTCTTATTA motifs detected in file\n",
      "\n",
      "Seq is: CTACATACGAAAC\n",
      "0 CTACATACGAAAC motifs detected in file\n",
      "\n",
      "Seq is: AAAGACAACTCAC\n",
      "0 AAAGACAACTCAC motifs detected in file\n",
      "\n",
      "Seq is: ACCCAAATCATTA\n",
      "0 ACCCAAATCATTA motifs detected in file\n",
      "\n",
      "Seq is: CCCCAAACACCCC\n",
      "0 CCCCAAACACCCC motifs detected in file\n",
      "\n",
      "Seq is: AAAAGAAGATTCG\n",
      "0 AAAAGAAGATTCG motifs detected in file\n",
      "\n",
      "Seq is: GAAGAGGATAAAT\n",
      "0 GAAGAGGATAAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCCCCAAATCAT\n",
      "0 ACCCCCAAATCAT motifs detected in file\n",
      "\n",
      "Seq is: AACTAACAATCGC\n",
      "0 AACTAACAATCGC motifs detected in file\n",
      "\n",
      "Seq is: CACACCAGCCCAC\n",
      "0 CACACCAGCCCAC motifs detected in file\n",
      "\n",
      "Seq is: ACAATAATAAGAC\n",
      "0 ACAATAATAAGAC motifs detected in file\n",
      "\n",
      "Seq is: TTAAAACCCCCTC\n",
      "0 TTAAAACCCCCTC motifs detected in file\n",
      "\n",
      "Seq is: CAATATACCCAAA\n",
      "0 CAATATACCCAAA motifs detected in file\n",
      "\n",
      "Seq is: TTATCAAAAAAAT\n",
      "0 TTATCAAAAAAAT motifs detected in file\n",
      "\n",
      "Seq is: TAAATAAATCCCC\n",
      "0 TAAATAAATCCCC motifs detected in file\n",
      "\n",
      "Seq is: CTTACTACCAACA\n",
      "0 CTTACTACCAACA motifs detected in file\n",
      "\n",
      "Seq is: GAAACCTATATGA\n",
      "0 GAAACCTATATGA motifs detected in file\n",
      "\n",
      "Seq is: TGTCGCTACCGAT\n",
      "0 TGTCGCTACCGAT motifs detected in file\n",
      "\n",
      "Seq is: AACTCCTCATAGG\n",
      "0 AACTCCTCATAGG motifs detected in file\n",
      "\n",
      "Seq is: ACTCCGTACCCCC\n",
      "0 ACTCCGTACCCCC motifs detected in file\n",
      "\n",
      "Seq is: AGTCCCCAACTCC\n",
      "0 AGTCCCCAACTCC motifs detected in file\n",
      "\n",
      "Seq is: AGAACCACTCACA\n",
      "0 AGAACCACTCACA motifs detected in file\n",
      "\n",
      "Seq is: AAATCACCCCAAT\n",
      "0 AAATCACCCCAAT motifs detected in file\n",
      "\n",
      "Seq is: CGCTACCTCCATC\n",
      "0 CGCTACCTCCATC motifs detected in file\n",
      "\n",
      "Seq is: CTAACCACGACAC\n",
      "0 CTAACCACGACAC motifs detected in file\n",
      "\n",
      "Seq is: CCACTTTCTCATA\n",
      "0 CCACTTTCTCATA motifs detected in file\n",
      "\n",
      "Seq is: CTACCCCACCACC\n",
      "0 CTACCCCACCACC motifs detected in file\n",
      "\n",
      "Seq is: AACACCATTTGAA\n",
      "0 AACACCATTTGAA motifs detected in file\n",
      "\n",
      "Seq is: ATTATCACATCCT\n",
      "0 ATTATCACATCCT motifs detected in file\n",
      "\n",
      "Seq is: TCGACTTATTAAA\n",
      "0 TCGACTTATTAAA motifs detected in file\n",
      "\n",
      "Seq is: TACTTCCTCTCCC\n",
      "0 TACTTCCTCTCCC motifs detected in file\n",
      "\n",
      "Seq is: CAGTCCCAACTAA\n",
      "0 CAGTCCCAACTAA motifs detected in file\n",
      "\n",
      "Seq is: GCCCTCCTAGGAT\n",
      "0 GCCCTCCTAGGAT motifs detected in file\n",
      "\n",
      "Seq is: AACCACGCCCCCG\n",
      "0 AACCACGCCCCCG motifs detected in file\n",
      "\n",
      "Seq is: AAACATACTAATA\n",
      "0 AAACATACTAATA motifs detected in file\n",
      "\n",
      "Seq is: CCCGCAACTAATC\n",
      "0 CCCGCAACTAATC motifs detected in file\n",
      "\n",
      "Seq is: ATCATCAATGACC\n",
      "0 ATCATCAATGACC motifs detected in file\n",
      "\n",
      "Seq is: AACTTGTAACAAA\n",
      "0 AACTTGTAACAAA motifs detected in file\n",
      "\n",
      "Seq is: CAACCACATATAT\n",
      "0 CAACCACATATAT motifs detected in file\n",
      "\n",
      "Seq is: AACATTAACTCTA\n",
      "0 AACATTAACTCTA motifs detected in file\n",
      "\n",
      "Seq is: ACGAGCCCCCTTA\n",
      "0 ACGAGCCCCCTTA motifs detected in file\n",
      "\n",
      "Seq is: TCCAATACACTAA\n",
      "0 TCCAATACACTAA motifs detected in file\n",
      "\n",
      "Seq is: TCCTCATCCCAAT\n",
      "0 TCCTCATCCCAAT motifs detected in file\n",
      "\n",
      "Seq is: CCTACTCACCCTT\n",
      "0 CCTACTCACCCTT motifs detected in file\n",
      "\n",
      "Seq is: CTTCTTTCTCTCC\n",
      "0 CTTCTTTCTCTCC motifs detected in file\n",
      "\n",
      "Seq is: TTCATTTCAAATT\n",
      "0 TTCATTTCAAATT motifs detected in file\n",
      "\n",
      "Seq is: AATACGGAAAAAC\n",
      "0 AATACGGAAAAAC motifs detected in file\n",
      "\n",
      "Seq is: CCAACTCCACTAC\n",
      "0 CCAACTCCACTAC motifs detected in file\n",
      "\n",
      "Seq is: TACCTCCACCTCT\n",
      "0 TACCTCCACCTCT motifs detected in file\n",
      "\n",
      "Seq is: AAACCACGACACT\n",
      "0 AAACCACGACACT motifs detected in file\n",
      "\n",
      "Seq is: TTAACAAAATCCA\n",
      "0 TTAACAAAATCCA motifs detected in file\n",
      "\n",
      "Seq is: TTATCGAAAAGAT\n",
      "0 TTATCGAAAAGAT motifs detected in file\n",
      "\n",
      "Seq is: CAGTCCAATCCAG\n",
      "0 CAGTCCAATCCAG motifs detected in file\n",
      "\n",
      "Seq is: ATCCTCCTCATCC\n",
      "0 ATCCTCCTCATCC motifs detected in file\n",
      "\n",
      "Seq is: CCGTCCAAAACCG\n",
      "0 CCGTCCAAAACCG motifs detected in file\n",
      "\n",
      "Seq is: AGCATTCTTCCGG\n",
      "0 AGCATTCTTCCGG motifs detected in file\n",
      "\n",
      "Seq is: ATCTATCCCCTAA\n",
      "0 ATCTATCCCCTAA motifs detected in file\n",
      "\n",
      "Seq is: CACGCCACACACT\n",
      "0 CACGCCACACACT motifs detected in file\n",
      "\n",
      "Seq is: ACGATCCCATCTT\n",
      "0 ACGATCCCATCTT motifs detected in file\n",
      "\n",
      "Seq is: AGGCTCATACAAC\n",
      "0 AGGCTCATACAAC motifs detected in file\n",
      "\n",
      "Seq is: CTCTTTATTTTAC\n",
      "0 CTCTTTATTTTAC motifs detected in file\n",
      "\n",
      "Seq is: ACGTATCACCCCT\n",
      "0 ACGTATCACCCCT motifs detected in file\n",
      "\n",
      "Seq is: AAAATAAAATTCA\n",
      "0 AAAATAAAATTCA motifs detected in file\n",
      "\n",
      "Seq is: AACAACCAATCCA\n",
      "0 AACAACCAATCCA motifs detected in file\n",
      "\n",
      "Seq is: TCAACTCCAGATC\n",
      "0 TCAACTCCAGATC motifs detected in file\n",
      "\n",
      "Seq is: CCGACAATCTTCA\n",
      "0 CCGACAATCTTCA motifs detected in file\n",
      "\n",
      "Seq is: GGATCCTTTCACT\n",
      "0 GGATCCTTTCACT motifs detected in file\n",
      "\n",
      "Seq is: GTCGCTCCCGACA\n",
      "0 GTCGCTCCCGACA motifs detected in file\n",
      "\n",
      "Seq is: CTCAAAATCCATC\n",
      "0 CTCAAAATCCATC motifs detected in file\n",
      "\n",
      "Seq is: TCCCCCTAACAAC\n",
      "0 TCCCCCTAACAAC motifs detected in file\n",
      "\n",
      "Seq is: AAACCTTCCCCCT\n",
      "0 AAACCTTCCCCCT motifs detected in file\n",
      "\n",
      "Seq is: ACGCCCCCTTTAC\n",
      "0 ACGCCCCCTTTAC motifs detected in file\n",
      "\n",
      "Seq is: AACAATCATTACT\n",
      "0 AACAATCATTACT motifs detected in file\n",
      "\n",
      "Seq is: CTTTAGGACGCTT\n",
      "0 CTTTAGGACGCTT motifs detected in file\n",
      "\n",
      "Seq is: ATCCGAAGGCCGA\n",
      "0 ATCCGAAGGCCGA motifs detected in file\n",
      "\n",
      "Seq is: CGGTCCGCAAATT\n",
      "0 CGGTCCGCAAATT motifs detected in file\n",
      "\n",
      "Seq is: ACCCCAAATCATC\n",
      "0 ACCCCAAATCATC motifs detected in file\n",
      "\n",
      "Seq is: CCACCCCAATAAA\n",
      "0 CCACCCCAATAAA motifs detected in file\n",
      "\n",
      "Seq is: AGCAATTACAATC\n",
      "0 AGCAATTACAATC motifs detected in file\n",
      "\n",
      "Seq is: ATTCCCACCCCTT\n",
      "0 ATTCCCACCCCTT motifs detected in file\n",
      "\n",
      "Seq is: CGCTCCAACTGGA\n",
      "0 CGCTCCAACTGGA motifs detected in file\n",
      "\n",
      "Seq is: CAATCCCACTCTT\n",
      "0 CAATCCCACTCTT motifs detected in file\n",
      "\n",
      "Seq is: CTTAATAAGCTCA\n",
      "0 CTTAATAAGCTCA motifs detected in file\n",
      "\n",
      "Seq is: CGATATCCGCGAA\n",
      "0 CGATATCCGCGAA motifs detected in file\n",
      "\n",
      "Seq is: CAGTCCCTCCATC\n",
      "0 CAGTCCCTCCATC motifs detected in file\n",
      "\n",
      "Seq is: GCTACTCTCATTA\n",
      "0 GCTACTCTCATTA motifs detected in file\n",
      "\n",
      "Seq is: TCTATCCCGAGTC\n",
      "0 TCTATCCCGAGTC motifs detected in file\n",
      "\n",
      "Seq is: CGCAAACACATAC\n",
      "0 CGCAAACACATAC motifs detected in file\n",
      "\n",
      "Seq is: TATACAAACGCCA\n",
      "0 TATACAAACGCCA motifs detected in file\n",
      "\n",
      "Seq is: AAGCTACTACACC\n",
      "0 AAGCTACTACACC motifs detected in file\n",
      "\n",
      "Seq is: AGAAACCTCATCT\n",
      "0 AGAAACCTCATCT motifs detected in file\n",
      "\n",
      "Seq is: TTGGACACTCCTT\n",
      "0 TTGGACACTCCTT motifs detected in file\n",
      "\n",
      "Seq is: TCCATAAGGACGA\n",
      "0 TCCATAAGGACGA motifs detected in file\n",
      "\n",
      "Seq is: TTACGATCCGACG\n",
      "0 TTACGATCCGACG motifs detected in file\n",
      "\n",
      "Seq is: GTTACCACTCCCT\n",
      "0 GTTACCACTCCCT motifs detected in file\n",
      "\n",
      "Seq is: CCAACTTCACTCT\n",
      "0 CCAACTTCACTCT motifs detected in file\n",
      "\n",
      "Seq is: CCATACCAAAACT\n",
      "0 CCATACCAAAACT motifs detected in file\n",
      "\n",
      "Seq is: CATCAGGAGGATA\n",
      "0 CATCAGGAGGATA motifs detected in file\n",
      "\n",
      "Seq is: AAAAGCTTATAGA\n",
      "0 AAAAGCTTATAGA motifs detected in file\n",
      "\n",
      "Seq is: ACAAGTAACAATT\n",
      "0 ACAAGTAACAATT motifs detected in file\n",
      "\n",
      "Seq is: CCAACACCTACTA\n",
      "0 CCAACACCTACTA motifs detected in file\n",
      "\n",
      "Seq is: CCTGGGCCTCGTG\n",
      "0 CCTGGGCCTCGTG motifs detected in file\n",
      "\n",
      "Seq is: TATTTATCATACC\n",
      "0 TATTTATCATACC motifs detected in file\n",
      "\n",
      "Seq is: GAAACTTCTTCCG\n",
      "0 GAAACTTCTTCCG motifs detected in file\n",
      "\n",
      "Seq is: CACCCATGTCTAC\n",
      "0 CACCCATGTCTAC motifs detected in file\n",
      "\n",
      "Seq is: ACGTCCTTACGAT\n",
      "0 ACGTCCTTACGAT motifs detected in file\n",
      "\n",
      "Seq is: CCACACCAACCAA\n",
      "0 CCACACCAACCAA motifs detected in file\n",
      "\n",
      "Seq is: CTACGGCTAACAT\n",
      "0 CTACGGCTAACAT motifs detected in file\n",
      "\n",
      "Seq is: TGATAATACTCAG\n",
      "0 TGATAATACTCAG motifs detected in file\n",
      "\n",
      "Seq is: GATCAACTGAACT\n",
      "0 GATCAACTGAACT motifs detected in file\n",
      "\n",
      "Seq is: TCACCTCTTCCGA\n",
      "0 TCACCTCTTCCGA motifs detected in file\n",
      "\n",
      "Seq is: TGCTAAAAAAACT\n",
      "0 TGCTAAAAAAACT motifs detected in file\n",
      "\n",
      "Seq is: ACAGTAAAACACA\n",
      "0 ACAGTAAAACACA motifs detected in file\n",
      "\n",
      "Seq is: TTCCCGCGTCTGA\n",
      "0 TTCCCGCGTCTGA motifs detected in file\n",
      "\n",
      "Seq is: CGACGCTTGTCTC\n",
      "0 CGACGCTTGTCTC motifs detected in file\n",
      "\n",
      "Seq is: ACCACTATCGCGG\n",
      "0 ACCACTATCGCGG motifs detected in file\n",
      "\n",
      "Seq is: ATTCGTATCACAA\n",
      "0 ATTCGTATCACAA motifs detected in file\n",
      "\n",
      "Seq is: TCTCAAACCTAAT\n",
      "0 TCTCAAACCTAAT motifs detected in file\n",
      "\n",
      "Seq is: CACCCGATAAAAG\n",
      "0 CACCCGATAAAAG motifs detected in file\n",
      "\n",
      "Seq is: ACGATCCCCCACC\n",
      "0 ACGATCCCCCACC motifs detected in file\n",
      "\n",
      "Seq is: TTCGCCTACTCAT\n",
      "0 TTCGCCTACTCAT motifs detected in file\n",
      "\n",
      "Seq is: TCTTCTAAGGACG\n",
      "0 TCTTCTAAGGACG motifs detected in file\n",
      "\n",
      "Seq is: ATGTTGATATCAC\n",
      "0 ATGTTGATATCAC motifs detected in file\n",
      "\n",
      "Seq is: GAACTCACCTCAT\n",
      "0 GAACTCACCTCAT motifs detected in file\n",
      "\n",
      "Seq is: CCCGACTCTGACC\n",
      "0 CCCGACTCTGACC motifs detected in file\n",
      "\n",
      "Seq is: CCACCCCGGAAGA\n",
      "0 CCACCCCGGAAGA motifs detected in file\n",
      "\n",
      "Seq is: TACCGACTCCCCT\n",
      "0 TACCGACTCCCCT motifs detected in file\n",
      "\n",
      "Seq is: CAGTCCCCACCTC\n",
      "0 CAGTCCCCACCTC motifs detected in file\n",
      "\n",
      "Seq is: TGGATTAACCCGA\n",
      "0 TGGATTAACCCGA motifs detected in file\n",
      "\n",
      "Seq is: CTAAACGGACGAC\n",
      "0 CTAAACGGACGAC motifs detected in file\n",
      "\n",
      "Seq is: GACGATCCTCCTC\n",
      "0 GACGATCCTCCTC motifs detected in file\n",
      "\n",
      "Seq is: CGGATCATCACCC\n",
      "0 CGGATCATCACCC motifs detected in file\n",
      "\n",
      "Seq is: CACTCCGAACCTA\n",
      "0 CACTCCGAACCTA motifs detected in file\n",
      "\n",
      "Seq is: ATCGCAAATCTTC\n",
      "0 ATCGCAAATCTTC motifs detected in file\n",
      "\n",
      "Seq is: CCGATAAACAACA\n",
      "0 CCGATAAACAACA motifs detected in file\n",
      "\n",
      "Seq is: CCCAGAGTACTCA\n",
      "0 CCCAGAGTACTCA motifs detected in file\n",
      "\n",
      "Seq is: ACCTCACATCCTA\n",
      "0 ACCTCACATCCTA motifs detected in file\n",
      "\n",
      "Seq is: TTTAGTACGATTC\n",
      "0 TTTAGTACGATTC motifs detected in file\n",
      "\n",
      "Seq is: CGCTCCTACTTTG\n",
      "0 CGCTCCTACTTTG motifs detected in file\n",
      "\n",
      "Seq is: GCTATAGCGGCTA\n",
      "0 GCTATAGCGGCTA motifs detected in file\n",
      "\n",
      "Seq is: TGCCAACATATCC\n",
      "0 TGCCAACATATCC motifs detected in file\n",
      "\n",
      "Seq is: TAACGAACTTACC\n",
      "0 TAACGAACTTACC motifs detected in file\n",
      "\n",
      "Seq is: GACGACACAACCG\n",
      "0 GACGACACAACCG motifs detected in file\n",
      "\n",
      "Seq is: TAGACGAGCCCGC\n",
      "0 TAGACGAGCCCGC motifs detected in file\n",
      "\n",
      "Seq is: ATAGTAGTTGACT\n",
      "0 ATAGTAGTTGACT motifs detected in file\n",
      "\n",
      "Seq is: ACTCGTTCTTCCT\n",
      "0 ACTCGTTCTTCCT motifs detected in file\n",
      "\n",
      "Seq is: ATATTAAGGATGC\n",
      "0 ATATTAAGGATGC motifs detected in file\n",
      "\n",
      "Seq is: GGGAGAGTCGGCT\n",
      "0 GGGAGAGTCGGCT motifs detected in file\n",
      "\n",
      "Seq is: ACTTGTCAACCTT\n",
      "0 ACTTGTCAACCTT motifs detected in file\n",
      "\n",
      "Seq is: ATCCAACAATCGC\n",
      "0 ATCCAACAATCGC motifs detected in file\n",
      "\n",
      "Seq is: CATTGATTCTAAT\n",
      "0 CATTGATTCTAAT motifs detected in file\n",
      "\n",
      "Seq is: CATACCATTAATC\n",
      "0 CATACCATTAATC motifs detected in file\n",
      "\n",
      "Seq is: CTTCTACTCATCT\n",
      "0 CTTCTACTCATCT motifs detected in file\n",
      "\n",
      "Seq is: ATAAACTTCTTGA\n",
      "0 ATAAACTTCTTGA motifs detected in file\n",
      "\n",
      "Seq is: CTAATTACAAACC\n",
      "0 CTAATTACAAACC motifs detected in file\n",
      "\n",
      "Seq is: CAGACTCAAATAT\n",
      "0 CAGACTCAAATAT motifs detected in file\n",
      "\n",
      "Seq is: ATAGTGTCACTCT\n",
      "0 ATAGTGTCACTCT motifs detected in file\n",
      "\n",
      "Seq is: TAAGATACTACCT\n",
      "0 TAAGATACTACCT motifs detected in file\n",
      "\n",
      "Seq is: GGTACATTGCTTG\n",
      "0 GGTACATTGCTTG motifs detected in file\n",
      "\n",
      "Seq is: TTACGATGTCCCT\n",
      "0 TTACGATGTCCCT motifs detected in file\n",
      "\n",
      "Seq is: ACTTATAATACCT\n",
      "0 ACTTATAATACCT motifs detected in file\n",
      "\n",
      "Seq is: CTTCATCAGATCA\n",
      "0 CTTCATCAGATCA motifs detected in file\n",
      "\n",
      "Seq is: AACTTCGAATCCC\n",
      "0 AACTTCGAATCCC motifs detected in file\n",
      "\n",
      "Seq is: TCTCGACCCAACA\n",
      "0 TCTCGACCCAACA motifs detected in file\n",
      "\n",
      "Seq is: AACCCAACACCGA\n",
      "0 AACCCAACACCGA motifs detected in file\n",
      "\n",
      "Seq is: GTCACAGTCAAGC\n",
      "0 GTCACAGTCAAGC motifs detected in file\n",
      "\n",
      "Seq is: TCTATTATTGAAG\n",
      "0 TCTATTATTGAAG motifs detected in file\n",
      "\n",
      "Seq is: AACCAGATCCGTG\n",
      "0 AACCAGATCCGTG motifs detected in file\n",
      "\n",
      "Seq is: TACTTATAACAAC\n",
      "0 TACTTATAACAAC motifs detected in file\n",
      "\n",
      "Seq is: ACCCCTTCTCTGA\n",
      "0 ACCCCTTCTCTGA motifs detected in file\n",
      "\n",
      "Seq is: CTATTATTTCCAC\n",
      "0 CTATTATTTCCAC motifs detected in file\n",
      "\n",
      "Seq is: CTACGCTGTTACC\n",
      "0 CTACGCTGTTACC motifs detected in file\n",
      "\n",
      "Seq is: TAAAATGTATTAC\n",
      "0 TAAAATGTATTAC motifs detected in file\n",
      "\n",
      "Seq is: CCCCATACTCAAA\n",
      "0 CCCCATACTCAAA motifs detected in file\n",
      "\n",
      "Seq is: AAAAACAATCCCA\n",
      "0 AAAAACAATCCCA motifs detected in file\n",
      "\n",
      "Seq is: ATTGCTCCCACCA\n",
      "0 ATTGCTCCCACCA motifs detected in file\n",
      "\n",
      "Seq is: TTCCTACCCCCCT\n",
      "0 TTCCTACCCCCCT motifs detected in file\n",
      "\n",
      "Seq is: TAATCCCTTCAGT\n",
      "0 TAATCCCTTCAGT motifs detected in file\n",
      "\n",
      "Seq is: CCCAATCCCACCA\n",
      "0 CCCAATCCCACCA motifs detected in file\n",
      "\n",
      "Seq is: ATATCATCACACG\n",
      "0 ATATCATCACACG motifs detected in file\n",
      "\n",
      "Seq is: TACCAATAATGAA\n",
      "0 TACCAATAATGAA motifs detected in file\n",
      "\n",
      "Seq is: AATAAACCTCAAC\n",
      "0 AATAAACCTCAAC motifs detected in file\n",
      "\n",
      "Seq is: GTGGTTTTAAAAA\n",
      "0 GTGGTTTTAAAAA motifs detected in file\n",
      "\n",
      "Seq is: CCCCGGATTCTGG\n",
      "0 CCCCGGATTCTGG motifs detected in file\n",
      "\n",
      "Seq is: TTACCTATCGACA\n",
      "0 TTACCTATCGACA motifs detected in file\n",
      "\n",
      "Seq is: ATAGGAAATTTTC\n",
      "0 ATAGGAAATTTTC motifs detected in file\n",
      "\n",
      "Seq is: AACTCTTTCGGTA\n",
      "0 AACTCTTTCGGTA motifs detected in file\n",
      "\n",
      "Seq is: CAACAATCTGTAC\n",
      "0 CAACAATCTGTAC motifs detected in file\n",
      "\n",
      "Seq is: CCCCGTACTCAAT\n",
      "0 CCCCGTACTCAAT motifs detected in file\n",
      "\n",
      "Seq is: CGTCAAGAACACT\n",
      "0 CGTCAAGAACACT motifs detected in file\n",
      "\n",
      "Seq is: CGAAAGAGCCATT\n",
      "0 CGAAAGAGCCATT motifs detected in file\n",
      "\n",
      "Seq is: TATTCCCCAGCAT\n",
      "0 TATTCCCCAGCAT motifs detected in file\n",
      "\n",
      "Seq is: TCGGAGACAACAG\n",
      "0 TCGGAGACAACAG motifs detected in file\n",
      "\n",
      "Seq is: TCTAAGTGTTAGA\n",
      "0 TCTAAGTGTTAGA motifs detected in file\n",
      "\n",
      "Seq is: CTACAAAACCAAT\n",
      "0 CTACAAAACCAAT motifs detected in file\n",
      "\n",
      "Seq is: TTGATATAAATGT\n",
      "0 TTGATATAAATGT motifs detected in file\n",
      "\n",
      "Seq is: TCTCCTTTTGGGC\n",
      "0 TCTCCTTTTGGGC motifs detected in file\n",
      "\n",
      "Seq is: CATTACTACAGCC\n",
      "0 CATTACTACAGCC motifs detected in file\n",
      "\n",
      "Seq is: CCAACTCCCTATC\n",
      "0 CCAACTCCCTATC motifs detected in file\n",
      "\n",
      "Seq is: CTCCTCTTACCCC\n",
      "0 CTCCTCTTACCCC motifs detected in file\n",
      "\n",
      "Seq is: CTATCCACATACT\n",
      "0 CTATCCACATACT motifs detected in file\n",
      "\n",
      "Seq is: TGTACTCCCACAA\n",
      "0 TGTACTCCCACAA motifs detected in file\n",
      "\n",
      "Seq is: AAGAGCACACTTA\n",
      "0 AAGAGCACACTTA motifs detected in file\n",
      "\n",
      "Seq is: CTCCCAAAATACA\n",
      "0 CTCCCAAAATACA motifs detected in file\n",
      "\n",
      "Seq is: ACAATTACACCAC\n",
      "0 ACAATTACACCAC motifs detected in file\n",
      "\n",
      "Seq is: CCACTCACTCGGG\n",
      "0 CCACTCACTCGGG motifs detected in file\n",
      "\n",
      "Seq is: GTAACACAACACC\n",
      "0 GTAACACAACACC motifs detected in file\n",
      "\n",
      "Seq is: ATTTATACATCTC\n",
      "0 ATTTATACATCTC motifs detected in file\n",
      "\n",
      "Seq is: CCTCATATCCCGA\n",
      "0 CCTCATATCCCGA motifs detected in file\n",
      "\n",
      "Seq is: CACTGATCATACA\n",
      "0 CACTGATCATACA motifs detected in file\n",
      "\n",
      "Seq is: ACTCAGGACATTC\n",
      "0 ACTCAGGACATTC motifs detected in file\n",
      "\n",
      "Seq is: ATCCTCTCACTAT\n",
      "0 ATCCTCTCACTAT motifs detected in file\n",
      "\n",
      "Seq is: AAACTAGTCCTCT\n",
      "0 AAACTAGTCCTCT motifs detected in file\n",
      "\n",
      "Seq is: TGCACCAATGATC\n",
      "0 TGCACCAATGATC motifs detected in file\n",
      "\n",
      "Seq is: GTGTCTCTCAAGA\n",
      "0 GTGTCTCTCAAGA motifs detected in file\n",
      "\n",
      "Seq is: GGGTCATCCAATT\n",
      "0 GGGTCATCCAATT motifs detected in file\n",
      "\n",
      "Seq is: ATCACCCCCCATT\n",
      "0 ATCACCCCCCATT motifs detected in file\n",
      "\n",
      "Seq is: CCGCTCCAATCGC\n",
      "0 CCGCTCCAATCGC motifs detected in file\n",
      "\n",
      "Seq is: TCCGAACGATCTT\n",
      "0 TCCGAACGATCTT motifs detected in file\n",
      "\n",
      "Seq is: CAGTAGTTTTTCG\n",
      "0 CAGTAGTTTTTCG motifs detected in file\n",
      "\n",
      "Seq is: ATAATCACCCTCA\n",
      "0 ATAATCACCCTCA motifs detected in file\n",
      "\n",
      "Seq is: TCTCAAACTTCAG\n",
      "0 TCTCAAACTTCAG motifs detected in file\n",
      "\n",
      "Seq is: GAACTCTCTCCTA\n",
      "0 GAACTCTCTCCTA motifs detected in file\n",
      "\n",
      "Seq is: ATACTACGCTGAC\n",
      "0 ATACTACGCTGAC motifs detected in file\n",
      "\n",
      "Seq is: ACTCACGCAAGCA\n",
      "0 ACTCACGCAAGCA motifs detected in file\n",
      "\n",
      "Seq is: TCAAACTCAAACG\n",
      "0 TCAAACTCAAACG motifs detected in file\n",
      "\n",
      "Seq is: ATCCGTCTTATCA\n",
      "0 ATCCGTCTTATCA motifs detected in file\n",
      "\n",
      "Seq is: TTACTCCTACATT\n",
      "0 TTACTCCTACATT motifs detected in file\n",
      "\n",
      "Seq is: CGGGCACCCGCTA\n",
      "0 CGGGCACCCGCTA motifs detected in file\n",
      "\n",
      "Seq is: ATACTCTTACTGA\n",
      "0 ATACTCTTACTGA motifs detected in file\n",
      "\n",
      "Seq is: CGCGGCCACTTCG\n",
      "0 CGCGGCCACTTCG motifs detected in file\n",
      "\n",
      "Seq is: AAGTCCCCCAAAC\n",
      "0 AAGTCCCCCAAAC motifs detected in file\n",
      "\n",
      "Seq is: CTACTCTTACCGA\n",
      "0 CTACTCTTACCGA motifs detected in file\n",
      "\n",
      "Seq is: CAATGATGCTCCC\n",
      "0 CAATGATGCTCCC motifs detected in file\n",
      "\n",
      "Seq is: GATACACCGACTA\n",
      "0 GATACACCGACTA motifs detected in file\n",
      "\n",
      "Seq is: ACTTCTCATACGT\n",
      "0 ACTTCTCATACGT motifs detected in file\n",
      "\n",
      "Seq is: TACTCGCTAAAAT\n",
      "0 TACTCGCTAAAAT motifs detected in file\n",
      "\n",
      "Seq is: CCAGACAAACAGC\n",
      "0 CCAGACAAACAGC motifs detected in file\n",
      "\n",
      "Seq is: ATCCGTCTACCTC\n",
      "0 ATCCGTCTACCTC motifs detected in file\n",
      "\n",
      "Seq is: GAACAATATTAAT\n",
      "0 GAACAATATTAAT motifs detected in file\n",
      "\n",
      "Seq is: ACGGAGTATCCCT\n",
      "0 ACGGAGTATCCCT motifs detected in file\n",
      "\n",
      "Seq is: ATCATGTTCCTTC\n",
      "0 ATCATGTTCCTTC motifs detected in file\n",
      "\n",
      "Seq is: CCCATCCGATACA\n",
      "0 CCCATCCGATACA motifs detected in file\n",
      "\n",
      "Seq is: CAAAACAGTCCCC\n",
      "0 CAAAACAGTCCCC motifs detected in file\n",
      "\n",
      "Seq is: CAACTCTTACTCA\n",
      "0 CAACTCTTACTCA motifs detected in file\n",
      "\n",
      "Seq is: CACTCCGCATAAT\n",
      "0 CACTCCGCATAAT motifs detected in file\n",
      "\n",
      "Seq is: ATGGTATCGGCGG\n",
      "0 ATGGTATCGGCGG motifs detected in file\n",
      "\n",
      "Seq is: ATCAAAATTCTCA\n",
      "0 ATCAAAATTCTCA motifs detected in file\n",
      "\n",
      "Seq is: TGACGCCGTTCAT\n",
      "0 TGACGCCGTTCAT motifs detected in file\n",
      "\n",
      "Seq is: GATAACTGGGTCG\n",
      "0 GATAACTGGGTCG motifs detected in file\n",
      "\n",
      "Seq is: CTACCCCCGAAGC\n",
      "0 CTACCCCCGAAGC motifs detected in file\n",
      "\n",
      "Seq is: TGTACCCGAAATC\n",
      "0 TGTACCCGAAATC motifs detected in file\n",
      "\n",
      "Seq is: CCTCAGTATTCAC\n",
      "0 CCTCAGTATTCAC motifs detected in file\n",
      "\n",
      "Seq is: CTCAGGCATTTCT\n",
      "0 CTCAGGCATTTCT motifs detected in file\n",
      "\n",
      "Seq is: CCATAGAAATGAT\n",
      "0 CCATAGAAATGAT motifs detected in file\n",
      "\n",
      "Seq is: ATTTTCGATAACA\n",
      "0 ATTTTCGATAACA motifs detected in file\n",
      "\n",
      "Seq is: CATTCGATCAGTA\n",
      "0 CATTCGATCAGTA motifs detected in file\n",
      "\n",
      "Seq is: TAATTCAACAACC\n",
      "0 TAATTCAACAACC motifs detected in file\n",
      "\n",
      "Seq is: GAGTCCTCAAACT\n",
      "0 GAGTCCTCAAACT motifs detected in file\n",
      "\n",
      "Seq is: ATCAAGAACCCGT\n",
      "0 ATCAAGAACCCGT motifs detected in file\n",
      "\n",
      "Seq is: CACTCTCACTCAT\n",
      "0 CACTCTCACTCAT motifs detected in file\n",
      "\n",
      "Seq is: CATCTTACAAATC\n",
      "0 CATCTTACAAATC motifs detected in file\n",
      "\n",
      "Seq is: ACTCGGATCCCAC\n",
      "0 ACTCGGATCCCAC motifs detected in file\n",
      "\n",
      "Seq is: AACACTCACATTT\n",
      "0 AACACTCACATTT motifs detected in file\n",
      "\n",
      "Seq is: AATCACGCTACTC\n",
      "0 AATCACGCTACTC motifs detected in file\n",
      "\n",
      "Seq is: ATCCCCTTCCCTC\n",
      "0 ATCCCCTTCCCTC motifs detected in file\n",
      "\n",
      "Seq is: GGATGATCCCACA\n",
      "0 GGATGATCCCACA motifs detected in file\n",
      "\n",
      "Seq is: TCTTATCCTTCAT\n",
      "0 TCTTATCCTTCAT motifs detected in file\n",
      "\n",
      "Seq is: ACACGGACGCAAG\n",
      "0 ACACGGACGCAAG motifs detected in file\n",
      "\n",
      "Seq is: TCCGCAAGACCGA\n",
      "0 TCCGCAAGACCGA motifs detected in file\n",
      "\n",
      "Seq is: CCAACGGAGTAGC\n",
      "0 CCAACGGAGTAGC motifs detected in file\n",
      "\n",
      "Seq is: CCACTACTATCGG\n",
      "0 CCACTACTATCGG motifs detected in file\n",
      "\n",
      "Seq is: TTCCACCCCTATT\n",
      "0 TTCCACCCCTATT motifs detected in file\n",
      "\n",
      "Seq is: CACACCAAAGCTT\n",
      "0 CACACCAAAGCTT motifs detected in file\n",
      "\n",
      "Seq is: CTTCTATATTTTA\n",
      "0 CTTCTATATTTTA motifs detected in file\n",
      "\n",
      "Seq is: TACTAATCAAGAC\n",
      "0 TACTAATCAAGAC motifs detected in file\n",
      "\n",
      "Seq is: ACCGACACTTACA\n",
      "0 ACCGACACTTACA motifs detected in file\n",
      "\n",
      "Seq is: ATATTAATTCCTC\n",
      "0 ATATTAATTCCTC motifs detected in file\n",
      "\n",
      "Seq is: TAAACATCCCTCT\n",
      "0 TAAACATCCCTCT motifs detected in file\n",
      "\n",
      "Seq is: AATCATATCTCTC\n",
      "0 AATCATATCTCTC motifs detected in file\n",
      "\n",
      "Seq is: CATCTCAAAAAAA\n",
      "0 CATCTCAAAAAAA motifs detected in file\n",
      "\n",
      "Seq is: GCACTATCACCAA\n",
      "0 GCACTATCACCAA motifs detected in file\n",
      "\n",
      "Seq is: GTGACCTATTCAC\n",
      "0 GTGACCTATTCAC motifs detected in file\n",
      "\n",
      "Seq is: CGCAACCGAACGG\n",
      "0 CGCAACCGAACGG motifs detected in file\n",
      "\n",
      "Seq is: TACTAACACTCCC\n",
      "0 TACTAACACTCCC motifs detected in file\n",
      "\n",
      "Seq is: CATCCTCAGTCCA\n",
      "0 CATCCTCAGTCCA motifs detected in file\n",
      "\n",
      "Seq is: TCAATCATAATCC\n",
      "0 TCAATCATAATCC motifs detected in file\n",
      "\n",
      "Seq is: TCCCCCCAACAAT\n",
      "0 TCCCCCCAACAAT motifs detected in file\n",
      "\n",
      "Seq is: CCCCCAGCCTCCT\n",
      "0 CCCCCAGCCTCCT motifs detected in file\n",
      "\n",
      "Seq is: TTTCCAACCCCTT\n",
      "0 TTTCCAACCCCTT motifs detected in file\n",
      "\n",
      "Seq is: GTCGATTTATCCA\n",
      "0 GTCGATTTATCCA motifs detected in file\n",
      "\n",
      "Seq is: ACAACAACTAAAC\n",
      "0 ACAACAACTAAAC motifs detected in file\n",
      "\n",
      "Seq is: CAATTTTTTATCA\n",
      "0 CAATTTTTTATCA motifs detected in file\n",
      "\n",
      "Seq is: TCTCCCTACTACG\n",
      "0 TCTCCCTACTACG motifs detected in file\n",
      "\n",
      "Seq is: ATTATTAATCCGA\n",
      "0 ATTATTAATCCGA motifs detected in file\n",
      "\n",
      "Seq is: CACCCACCAACAC\n",
      "0 CACCCACCAACAC motifs detected in file\n",
      "\n",
      "Seq is: AACTAAGTTTAAT\n",
      "0 AACTAAGTTTAAT motifs detected in file\n",
      "\n",
      "Seq is: ACACAAAAAACCT\n",
      "0 ACACAAAAAACCT motifs detected in file\n",
      "\n",
      "Seq is: TTCAGTACAGTCA\n",
      "0 TTCAGTACAGTCA motifs detected in file\n",
      "\n",
      "Seq is: CCATCATTATATT\n",
      "0 CCATCATTATATT motifs detected in file\n",
      "\n",
      "Seq is: AACAACCCTGCTA\n",
      "0 AACAACCCTGCTA motifs detected in file\n",
      "\n",
      "Seq is: ATCAAAATCGTAA\n",
      "0 ATCAAAATCGTAA motifs detected in file\n",
      "\n",
      "Seq is: CCTCATCCAAATC\n",
      "0 CCTCATCCAAATC motifs detected in file\n",
      "\n",
      "Seq is: CAATACATGCATC\n",
      "0 CAATACATGCATC motifs detected in file\n",
      "\n",
      "Seq is: AGATCCGGTATAC\n",
      "0 AGATCCGGTATAC motifs detected in file\n",
      "\n",
      "Seq is: ACAACCTCTAACT\n",
      "0 ACAACCTCTAACT motifs detected in file\n",
      "\n",
      "Seq is: CTGATCATCCCGA\n",
      "0 CTGATCATCCCGA motifs detected in file\n",
      "\n",
      "Seq is: TCCGGGTGGCGAC\n",
      "0 TCCGGGTGGCGAC motifs detected in file\n",
      "\n",
      "Seq is: GAAGCGTCCGCCG\n",
      "0 GAAGCGTCCGCCG motifs detected in file\n",
      "\n",
      "Seq is: TTTCTGATCATAC\n",
      "0 TTTCTGATCATAC motifs detected in file\n",
      "\n",
      "Seq is: CGTTATCCGTGTT\n",
      "0 CGTTATCCGTGTT motifs detected in file\n",
      "\n",
      "Seq is: ATAACCGATTCTC\n",
      "0 ATAACCGATTCTC motifs detected in file\n",
      "\n",
      "Seq is: CCTCACCCACAAC\n",
      "0 CCTCACCCACAAC motifs detected in file\n",
      "\n",
      "Seq is: TCCCAATACTCTC\n",
      "0 TCCCAATACTCTC motifs detected in file\n",
      "\n",
      "Seq is: ATCGATATTACTT\n",
      "0 ATCGATATTACTT motifs detected in file\n",
      "\n",
      "Seq is: GTCGCTATCATAA\n",
      "0 GTCGCTATCATAA motifs detected in file\n",
      "\n",
      "Seq is: TAAGGAAGATCCG\n",
      "0 TAAGGAAGATCCG motifs detected in file\n",
      "\n",
      "Seq is: TATCATCCCTCCT\n",
      "0 TATCATCCCTCCT motifs detected in file\n",
      "\n",
      "Seq is: ATACTCCACACTC\n",
      "0 ATACTCCACACTC motifs detected in file\n",
      "\n",
      "Seq is: GCTATATGATCAT\n",
      "0 GCTATATGATCAT motifs detected in file\n",
      "\n",
      "Seq is: AAGGATCTTCACT\n",
      "0 AAGGATCTTCACT motifs detected in file\n",
      "\n",
      "Seq is: CTACCATTTACGA\n",
      "0 CTACCATTTACGA motifs detected in file\n",
      "\n",
      "Seq is: TCATATTATAAAT\n",
      "0 TCATATTATAAAT motifs detected in file\n",
      "\n",
      "Seq is: ACATTTACTCCCC\n",
      "0 ACATTTACTCCCC motifs detected in file\n",
      "\n",
      "Seq is: GTAAACCATTTAT\n",
      "0 GTAAACCATTTAT motifs detected in file\n",
      "\n",
      "Seq is: ACTAATAGTATTA\n",
      "0 ACTAATAGTATTA motifs detected in file\n",
      "\n",
      "Seq is: AATTACTCAGCTT\n",
      "0 AATTACTCAGCTT motifs detected in file\n",
      "\n",
      "Seq is: TAGTAAGCAAAAC\n",
      "0 TAGTAAGCAAAAC motifs detected in file\n",
      "\n",
      "Seq is: AAATTTGATATAT\n",
      "0 AAATTTGATATAT motifs detected in file\n",
      "\n",
      "Seq is: GGTTAAGCCAAGT\n",
      "0 GGTTAAGCCAAGT motifs detected in file\n",
      "\n",
      "Seq is: CAGATTAGGAAAA\n",
      "0 CAGATTAGGAAAA motifs detected in file\n",
      "\n",
      "Seq is: ACATCAGTGAGTA\n",
      "0 ACATCAGTGAGTA motifs detected in file\n",
      "\n",
      "Seq is: TCCGGTCTGAATC\n",
      "0 TCCGGTCTGAATC motifs detected in file\n",
      "\n",
      "Seq is: CCGATCCTACTAC\n",
      "0 CCGATCCTACTAC motifs detected in file\n",
      "\n",
      "Seq is: TAATTATTCTCCC\n",
      "0 TAATTATTCTCCC motifs detected in file\n",
      "\n",
      "Seq is: TACTGTATTGATA\n",
      "0 TACTGTATTGATA motifs detected in file\n",
      "\n",
      "Seq is: ATCACCGTCCAAT\n",
      "0 ATCACCGTCCAAT motifs detected in file\n",
      "\n",
      "Seq is: CAACAAACATCCC\n",
      "0 CAACAAACATCCC motifs detected in file\n",
      "\n",
      "Seq is: GAGTACCATCCCC\n",
      "0 GAGTACCATCCCC motifs detected in file\n",
      "\n",
      "Seq is: ATTTTCCTCCCGT\n",
      "0 ATTTTCCTCCCGT motifs detected in file\n",
      "\n",
      "Seq is: TAAAGATCTAGTT\n",
      "0 TAAAGATCTAGTT motifs detected in file\n",
      "\n",
      "Seq is: TATTATTCTTCCA\n",
      "0 TATTATTCTTCCA motifs detected in file\n",
      "\n",
      "Seq is: TTATCGATGATTC\n",
      "0 TTATCGATGATTC motifs detected in file\n",
      "\n",
      "Seq is: TTCTTAAAATACC\n",
      "0 TTCTTAAAATACC motifs detected in file\n",
      "\n",
      "Seq is: TCTTTCCCTGCGC\n",
      "0 TCTTTCCCTGCGC motifs detected in file\n",
      "\n",
      "Seq is: CCGCCCCCTATAT\n",
      "0 CCGCCCCCTATAT motifs detected in file\n",
      "\n",
      "Seq is: CCCAGCTTCGGCG\n",
      "0 CCCAGCTTCGGCG motifs detected in file\n",
      "\n",
      "Seq is: TGAGCATTCCCCA\n",
      "0 TGAGCATTCCCCA motifs detected in file\n",
      "\n",
      "Seq is: CCTAAAAAGATAC\n",
      "0 CCTAAAAAGATAC motifs detected in file\n",
      "\n",
      "Seq is: ATCGGCAACTCAA\n",
      "0 ATCGGCAACTCAA motifs detected in file\n",
      "\n",
      "Seq is: CACCATCAGTTTT\n",
      "0 CACCATCAGTTTT motifs detected in file\n",
      "\n",
      "Seq is: ACATTATTAATAA\n",
      "0 ACATTATTAATAA motifs detected in file\n",
      "\n",
      "Seq is: TCATCATTCCGAT\n",
      "0 TCATCATTCCGAT motifs detected in file\n",
      "\n",
      "Seq is: CCTCCCACAACTA\n",
      "0 CCTCCCACAACTA motifs detected in file\n",
      "\n",
      "Seq is: ATAATTTTAATTC\n",
      "0 ATAATTTTAATTC motifs detected in file\n",
      "\n",
      "Seq is: CGCTTCAAATAAT\n",
      "0 CGCTTCAAATAAT motifs detected in file\n",
      "\n",
      "Seq is: GAGAAAAAACTTA\n",
      "0 GAGAAAAAACTTA motifs detected in file\n",
      "\n",
      "Seq is: CAACAGTTTTGAT\n",
      "0 CAACAGTTTTGAT motifs detected in file\n",
      "\n",
      "Seq is: CAATTAACCTTCA\n",
      "0 CAATTAACCTTCA motifs detected in file\n",
      "\n",
      "Seq is: ATTGCCATGATAA\n",
      "0 ATTGCCATGATAA motifs detected in file\n",
      "\n",
      "Seq is: ATATGATTTTCTC\n",
      "0 ATATGATTTTCTC motifs detected in file\n",
      "\n",
      "Seq is: ATTCTGGGAGTAG\n",
      "0 ATTCTGGGAGTAG motifs detected in file\n",
      "\n",
      "Seq is: TTATCTACCTCTG\n",
      "0 TTATCTACCTCTG motifs detected in file\n",
      "\n",
      "Seq is: TATGTCTTTATCA\n",
      "0 TATGTCTTTATCA motifs detected in file\n",
      "\n",
      "Seq is: GTTTGGTGTAGAT\n",
      "0 GTTTGGTGTAGAT motifs detected in file\n",
      "\n",
      "Seq is: GTTTTACGGTCAT\n",
      "0 GTTTTACGGTCAT motifs detected in file\n",
      "\n",
      "Seq is: AGTCCGCCGCCGA\n",
      "0 AGTCCGCCGCCGA motifs detected in file\n",
      "\n",
      "Seq is: AGCTTCGGTTTCA\n",
      "0 AGCTTCGGTTTCA motifs detected in file\n",
      "\n",
      "Seq is: CTACAAACCTACA\n",
      "0 CTACAAACCTACA motifs detected in file\n",
      "\n",
      "Seq is: TTTCACTTTATAA\n",
      "0 TTTCACTTTATAA motifs detected in file\n",
      "\n",
      "Seq is: TCAACCGCCTACT\n",
      "0 TCAACCGCCTACT motifs detected in file\n",
      "\n",
      "Seq is: TCGTCTATCACTC\n",
      "0 TCGTCTATCACTC motifs detected in file\n",
      "\n",
      "Seq is: CTTTCAACTCGGT\n",
      "0 CTTTCAACTCGGT motifs detected in file\n",
      "\n",
      "Seq is: TATTACTGCACTT\n",
      "0 TATTACTGCACTT motifs detected in file\n",
      "\n",
      "Seq is: CAGGCACCTTCGG\n",
      "0 CAGGCACCTTCGG motifs detected in file\n",
      "\n",
      "Seq is: ACACCGATGTTTT\n",
      "0 ACACCGATGTTTT motifs detected in file\n",
      "\n",
      "Seq is: TTACAACTCGGCA\n",
      "0 TTACAACTCGGCA motifs detected in file\n",
      "\n",
      "Seq is: TCTACGGCAGCCT\n",
      "0 TCTACGGCAGCCT motifs detected in file\n",
      "\n",
      "Seq is: TTACCACTTCCCT\n",
      "0 TTACCACTTCCCT motifs detected in file\n",
      "\n",
      "Seq is: CTGAGCTTCATGA\n",
      "0 CTGAGCTTCATGA motifs detected in file\n",
      "\n",
      "Seq is: GACTCCGAACATC\n",
      "0 GACTCCGAACATC motifs detected in file\n",
      "\n",
      "Seq is: CTCCCATTTTATC\n",
      "0 CTCCCATTTTATC motifs detected in file\n",
      "\n",
      "Seq is: TCTGGGTCATTTT\n",
      "0 TCTGGGTCATTTT motifs detected in file\n",
      "\n",
      "Seq is: AACATTATTCGTC\n",
      "0 AACATTATTCGTC motifs detected in file\n",
      "\n",
      "Seq is: ACGAACTTAATAA\n",
      "0 ACGAACTTAATAA motifs detected in file\n",
      "\n",
      "Seq is: ACCAAAGCCAACA\n",
      "0 ACCAAAGCCAACA motifs detected in file\n",
      "\n",
      "Seq is: AAAGATAATCTGA\n",
      "0 AAAGATAATCTGA motifs detected in file\n",
      "\n",
      "Seq is: TACCACTCGAGTC\n",
      "0 TACCACTCGAGTC motifs detected in file\n",
      "\n",
      "Seq is: CACTAACTATGAG\n",
      "0 CACTAACTATGAG motifs detected in file\n",
      "\n",
      "Seq is: GACTACGCTCATT\n",
      "0 GACTACGCTCATT motifs detected in file\n",
      "\n",
      "Seq is: ATGCCTACACAAA\n",
      "0 ATGCCTACACAAA motifs detected in file\n",
      "\n",
      "Seq is: TCCTCACCCTGAA\n",
      "0 TCCTCACCCTGAA motifs detected in file\n",
      "\n",
      "Seq is: GATCCCCTAAATC\n",
      "0 GATCCCCTAAATC motifs detected in file\n",
      "\n",
      "Seq is: GCCCCACTACGGA\n",
      "0 GCCCCACTACGGA motifs detected in file\n",
      "\n",
      "Seq is: CAACCCCCGGTCA\n",
      "0 CAACCCCCGGTCA motifs detected in file\n",
      "\n",
      "Seq is: CGGGAGGATTAAC\n",
      "0 CGGGAGGATTAAC motifs detected in file\n",
      "\n",
      "Seq is: CCCCCATCCCCGA\n",
      "0 CCCCCATCCCCGA motifs detected in file\n",
      "\n",
      "Seq is: TCCGACCTCACCA\n",
      "0 TCCGACCTCACCA motifs detected in file\n",
      "\n",
      "Seq is: TTTTCCGAGTCTT\n",
      "0 TTTTCCGAGTCTT motifs detected in file\n",
      "\n",
      "Seq is: TTTAGGACGCTTC\n",
      "0 TTTAGGACGCTTC motifs detected in file\n",
      "\n",
      "Seq is: TTTTTTTGAAGAC\n",
      "0 TTTTTTTGAAGAC motifs detected in file\n",
      "\n",
      "Seq is: TCCATTATTTATC\n",
      "0 TCCATTATTTATC motifs detected in file\n",
      "\n",
      "Seq is: CTAATAGGGCATA\n",
      "0 CTAATAGGGCATA motifs detected in file\n",
      "\n",
      "Seq is: GCTTCCGGAAAAA\n",
      "0 GCTTCCGGAAAAA motifs detected in file\n",
      "\n",
      "Seq is: CCTGTCCACCACA\n",
      "0 CCTGTCCACCACA motifs detected in file\n",
      "\n",
      "Seq is: CACCACCGGAACC\n",
      "0 CACCACCGGAACC motifs detected in file\n",
      "\n",
      "Seq is: ATACACGAAAGAG\n",
      "0 ATACACGAAAGAG motifs detected in file\n",
      "\n",
      "Seq is: CACAATGTAGCGC\n",
      "0 CACAATGTAGCGC motifs detected in file\n",
      "\n",
      "Seq is: GGTAGTAACCATA\n",
      "0 GGTAGTAACCATA motifs detected in file\n",
      "\n",
      "Seq is: TACCAATCACACA\n",
      "0 TACCAATCACACA motifs detected in file\n",
      "\n",
      "Seq is: ACCAATCATCCGG\n",
      "0 ACCAATCATCCGG motifs detected in file\n",
      "\n",
      "Seq is: ATCATACTCCTCG\n",
      "0 ATCATACTCCTCG motifs detected in file\n",
      "\n",
      "Seq is: CAATACCTCACCT\n",
      "0 CAATACCTCACCT motifs detected in file\n",
      "\n",
      "Seq is: TCACTTTAGTGTA\n",
      "0 TCACTTTAGTGTA motifs detected in file\n",
      "\n",
      "Seq is: CCGATCCGGCCTC\n",
      "0 CCGATCCGGCCTC motifs detected in file\n",
      "\n",
      "Seq is: CAGTAATCCTCCC\n",
      "0 CAGTAATCCTCCC motifs detected in file\n",
      "\n",
      "Seq is: GACTCTCCCGGGG\n",
      "0 GACTCTCCCGGGG motifs detected in file\n",
      "\n",
      "Seq is: ACAATCCCCAGTA\n",
      "0 ACAATCCCCAGTA motifs detected in file\n",
      "\n",
      "Seq is: CCCGACCCAAAAT\n",
      "0 CCCGACCCAAAAT motifs detected in file\n",
      "\n",
      "Seq is: GATATACTATCCG\n",
      "0 GATATACTATCCG motifs detected in file\n",
      "\n",
      "Seq is: TACACTAACCACC\n",
      "0 TACACTAACCACC motifs detected in file\n",
      "\n",
      "Seq is: CAGTAATACACAA\n",
      "0 CAGTAATACACAA motifs detected in file\n",
      "\n",
      "Seq is: CAGCACGTCCATC\n",
      "0 CAGCACGTCCATC motifs detected in file\n",
      "\n",
      "Seq is: TCCGAATGATCTT\n",
      "0 TCCGAATGATCTT motifs detected in file\n",
      "\n",
      "Seq is: CACACTTTTGCAT\n",
      "0 CACACTTTTGCAT motifs detected in file\n",
      "\n",
      "Seq is: CCGAACCTAATTC\n",
      "0 CCGAACCTAATTC motifs detected in file\n",
      "\n",
      "Seq is: CGCTGTCGCTAAA\n",
      "0 CGCTGTCGCTAAA motifs detected in file\n",
      "\n",
      "Seq is: GATCCTATCAGTC\n",
      "0 GATCCTATCAGTC motifs detected in file\n",
      "\n",
      "Seq is: ATCTTAATCTTAA\n",
      "0 ATCTTAATCTTAA motifs detected in file\n",
      "\n",
      "Seq is: CACTTCTACTATT\n",
      "0 CACTTCTACTATT motifs detected in file\n",
      "\n",
      "Seq is: CACATCTCCCTTC\n",
      "0 CACATCTCCCTTC motifs detected in file\n",
      "\n",
      "Seq is: CAATTACCAACTA\n",
      "0 CAATTACCAACTA motifs detected in file\n",
      "\n",
      "Seq is: TAACGATCCCACC\n",
      "0 TAACGATCCCACC motifs detected in file\n",
      "\n",
      "Seq is: GCGAAGGTTAATC\n",
      "0 GCGAAGGTTAATC motifs detected in file\n",
      "\n",
      "Seq is: CACGTACTCATCC\n",
      "0 CACGTACTCATCC motifs detected in file\n",
      "\n",
      "Seq is: ACCGGACGTCATT\n",
      "0 ACCGGACGTCATT motifs detected in file\n",
      "\n",
      "Seq is: ACAATCGCCAATC\n",
      "0 ACAATCGCCAATC motifs detected in file\n",
      "\n",
      "Seq is: CGCATGCCGGTCC\n",
      "0 CGCATGCCGGTCC motifs detected in file\n",
      "\n",
      "Seq is: CGATAACCAACTT\n",
      "0 CGATAACCAACTT motifs detected in file\n",
      "\n",
      "Seq is: ACTCATCCGACTA\n",
      "0 ACTCATCCGACTA motifs detected in file\n",
      "\n",
      "Seq is: CCAAAGCTATTAT\n",
      "0 CCAAAGCTATTAT motifs detected in file\n",
      "\n",
      "Seq is: TGATCATACCCCT\n",
      "0 TGATCATACCCCT motifs detected in file\n",
      "\n",
      "Seq is: ATTCCCCACATCC\n",
      "0 ATTCCCCACATCC motifs detected in file\n",
      "\n",
      "Seq is: ACACGGAACACCA\n",
      "0 ACACGGAACACCA motifs detected in file\n",
      "\n",
      "Seq is: TTCTTCACCCGAT\n",
      "0 TTCTTCACCCGAT motifs detected in file\n",
      "\n",
      "Seq is: CCCGTAAAAATTA\n",
      "0 CCCGTAAAAATTA motifs detected in file\n",
      "\n",
      "Seq is: GAATCTCGCTTTC\n",
      "0 GAATCTCGCTTTC motifs detected in file\n",
      "\n",
      "Seq is: GGATATTAGTGAC\n",
      "0 GGATATTAGTGAC motifs detected in file\n",
      "\n",
      "Seq is: ACGGGCGAGTATT\n",
      "0 ACGGGCGAGTATT motifs detected in file\n",
      "\n",
      "Seq is: CCCCTACCGGTAC\n",
      "0 CCCCTACCGGTAC motifs detected in file\n",
      "\n",
      "Seq is: CGATCCAAATATC\n",
      "0 CGATCCAAATATC motifs detected in file\n",
      "\n",
      "Seq is: TATCAACCCACCA\n",
      "0 TATCAACCCACCA motifs detected in file\n",
      "\n",
      "Seq is: ACCACATTTACTC\n",
      "0 ACCACATTTACTC motifs detected in file\n",
      "\n",
      "Seq is: ACTCCGTCCTCAG\n",
      "0 ACTCCGTCCTCAG motifs detected in file\n",
      "\n",
      "Seq is: GCTCCTCCAATCA\n",
      "0 GCTCCTCCAATCA motifs detected in file\n",
      "\n",
      "Seq is: ACACCGTTATTTT\n",
      "0 ACACCGTTATTTT motifs detected in file\n",
      "\n",
      "Seq is: TACTAATTCCTAT\n",
      "0 TACTAATTCCTAT motifs detected in file\n",
      "\n",
      "Seq is: GATCATATTCTCT\n",
      "0 GATCATATTCTCT motifs detected in file\n",
      "\n",
      "Seq is: AGTCCAAGCAGGA\n",
      "0 AGTCCAAGCAGGA motifs detected in file\n",
      "\n",
      "Seq is: AATCACAACACAT\n",
      "0 AATCACAACACAT motifs detected in file\n",
      "\n",
      "Seq is: ACCAATAGTAAAC\n",
      "0 ACCAATAGTAAAC motifs detected in file\n",
      "\n",
      "Seq is: AAAACTCCAATCA\n",
      "0 AAAACTCCAATCA motifs detected in file\n",
      "\n",
      "Seq is: AACTAATCAGTAA\n",
      "0 AACTAATCAGTAA motifs detected in file\n",
      "\n",
      "Seq is: CAACCCACCACTA\n",
      "0 CAACCCACCACTA motifs detected in file\n",
      "\n",
      "Seq is: ATCAGCCAACAAC\n",
      "0 ATCAGCCAACAAC motifs detected in file\n",
      "\n",
      "Seq is: TACTCTATAAACC\n",
      "0 TACTCTATAAACC motifs detected in file\n",
      "\n",
      "Seq is: TCCACCCCTAGTT\n",
      "0 TCCACCCCTAGTT motifs detected in file\n",
      "\n",
      "Seq is: ATCTCCCCCTTTA\n",
      "0 ATCTCCCCCTTTA motifs detected in file\n",
      "\n",
      "Seq is: TCTTACTAGTCAT\n",
      "0 TCTTACTAGTCAT motifs detected in file\n",
      "\n",
      "Seq is: GACGCCGCCCATC\n",
      "0 GACGCCGCCCATC motifs detected in file\n",
      "\n",
      "Seq is: CGGATCCTAACAC\n",
      "0 CGGATCCTAACAC motifs detected in file\n",
      "\n",
      "Seq is: CCCCGTTACTTAC\n",
      "0 CCCCGTTACTTAC motifs detected in file\n",
      "\n",
      "Seq is: TTCGCTTGTCTAA\n",
      "0 TTCGCTTGTCTAA motifs detected in file\n",
      "\n",
      "Seq is: AAGCAAGTAAAAC\n",
      "0 AAGCAAGTAAAAC motifs detected in file\n",
      "\n",
      "Seq is: CAAGAGTCCCAAA\n",
      "0 CAAGAGTCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: CAATATTAAAAAA\n",
      "0 CAATATTAAAAAA motifs detected in file\n",
      "\n",
      "Seq is: TAAAAATACCCGA\n",
      "0 TAAAAATACCCGA motifs detected in file\n",
      "\n",
      "Seq is: AACCACTCCCTCC\n",
      "0 AACCACTCCCTCC motifs detected in file\n",
      "\n",
      "Seq is: ATCCACCATCAAA\n",
      "0 ATCCACCATCAAA motifs detected in file\n",
      "\n",
      "Seq is: CACAAATTATAAA\n",
      "0 CACAAATTATAAA motifs detected in file\n",
      "\n",
      "Seq is: AATCAACCCACTA\n",
      "0 AATCAACCCACTA motifs detected in file\n",
      "\n",
      "Seq is: CTCCTTATCACAT\n",
      "0 CTCCTTATCACAT motifs detected in file\n",
      "\n",
      "Seq is: TCCTCATACCCCC\n",
      "0 TCCTCATACCCCC motifs detected in file\n",
      "\n",
      "Seq is: ATTAATACCACCC\n",
      "0 ATTAATACCACCC motifs detected in file\n",
      "\n",
      "Seq is: GGTATGCCATCAT\n",
      "0 GGTATGCCATCAT motifs detected in file\n",
      "\n",
      "Seq is: AAATCAACCCCGT\n",
      "0 AAATCAACCCCGT motifs detected in file\n",
      "\n",
      "Seq is: AAAGTGACATTTC\n",
      "0 AAAGTGACATTTC motifs detected in file\n",
      "\n",
      "Seq is: TCCACAACCAAGA\n",
      "0 TCCACAACCAAGA motifs detected in file\n",
      "\n",
      "Seq is: GAATTAGAAATTG\n",
      "0 GAATTAGAAATTG motifs detected in file\n",
      "\n",
      "Seq is: AATTTTCCAATTA\n",
      "0 AATTTTCCAATTA motifs detected in file\n",
      "\n",
      "Seq is: CGATTCAATCGAA\n",
      "0 CGATTCAATCGAA motifs detected in file\n",
      "\n",
      "Seq is: ATGTCACCCGAGA\n",
      "0 ATGTCACCCGAGA motifs detected in file\n",
      "\n",
      "Seq is: TCTCCCCCATCTC\n",
      "0 TCTCCCCCATCTC motifs detected in file\n",
      "\n",
      "Seq is: CCCCACGATATCC\n",
      "0 CCCCACGATATCC motifs detected in file\n",
      "\n",
      "Seq is: CATTTATGCCCGG\n",
      "0 CATTTATGCCCGG motifs detected in file\n",
      "\n",
      "Seq is: GATAAAGTTTCTA\n",
      "0 GATAAAGTTTCTA motifs detected in file\n",
      "\n",
      "Seq is: AAAATCCCCTTAA\n",
      "0 AAAATCCCCTTAA motifs detected in file\n",
      "\n",
      "Seq is: TTAAGATCCTGCT\n",
      "0 TTAAGATCCTGCT motifs detected in file\n",
      "\n",
      "Seq is: ACCCGTACTTTGA\n",
      "0 ACCCGTACTTTGA motifs detected in file\n",
      "\n",
      "Seq is: CACCAAACGAGGT\n",
      "0 CACCAAACGAGGT motifs detected in file\n",
      "\n",
      "Seq is: GTCTAAAGTCTCG\n",
      "0 GTCTAAAGTCTCG motifs detected in file\n",
      "\n",
      "Seq is: TAACTGGCATCAT\n",
      "0 TAACTGGCATCAT motifs detected in file\n",
      "\n",
      "Seq is: ATGGGGGCCAGCA\n",
      "0 ATGGGGGCCAGCA motifs detected in file\n",
      "\n",
      "Seq is: CATCGCCACTTTC\n",
      "0 CATCGCCACTTTC motifs detected in file\n",
      "\n",
      "Seq is: ACCAAACCAAATC\n",
      "0 ACCAAACCAAATC motifs detected in file\n",
      "\n",
      "Seq is: TGCAGGCCCTTAA\n",
      "0 TGCAGGCCCTTAA motifs detected in file\n",
      "\n",
      "Seq is: CGTAGACAAAAAT\n",
      "0 CGTAGACAAAAAT motifs detected in file\n",
      "\n",
      "Seq is: TCGGATTACACCC\n",
      "0 TCGGATTACACCC motifs detected in file\n",
      "\n",
      "Seq is: CTGTCGAGTACTC\n",
      "0 CTGTCGAGTACTC motifs detected in file\n",
      "\n",
      "Seq is: ACGTTCTGCAGAA\n",
      "0 ACGTTCTGCAGAA motifs detected in file\n",
      "\n",
      "Seq is: CACTACATTAATA\n",
      "0 CACTACATTAATA motifs detected in file\n",
      "\n",
      "Seq is: ATATGCTTACCCC\n",
      "0 ATATGCTTACCCC motifs detected in file\n",
      "\n",
      "Seq is: CGAAGTTAGCCCT\n",
      "0 CGAAGTTAGCCCT motifs detected in file\n",
      "\n",
      "Seq is: CATGATGAGCTAA\n",
      "0 CATGATGAGCTAA motifs detected in file\n",
      "\n",
      "Seq is: CAGTTGCAGTTCC\n",
      "0 CAGTTGCAGTTCC motifs detected in file\n",
      "\n",
      "Seq is: TCAGCGTCCAGCG\n",
      "0 TCAGCGTCCAGCG motifs detected in file\n",
      "\n",
      "Seq is: GACCAAGATCCTT\n",
      "0 GACCAAGATCCTT motifs detected in file\n",
      "\n",
      "Seq is: ATTACCCCCTTCA\n",
      "0 ATTACCCCCTTCA motifs detected in file\n",
      "\n",
      "Seq is: TACATCCTCAACT\n",
      "0 TACATCCTCAACT motifs detected in file\n",
      "\n",
      "Seq is: TCTAATCAGGCGG\n",
      "0 TCTAATCAGGCGG motifs detected in file\n",
      "\n",
      "Seq is: CATCAGCCACATG\n",
      "0 CATCAGCCACATG motifs detected in file\n",
      "\n",
      "Seq is: AGCATCCAAGTCA\n",
      "0 AGCATCCAAGTCA motifs detected in file\n",
      "\n",
      "Seq is: TGGTAACCACCGG\n",
      "0 TGGTAACCACCGG motifs detected in file\n",
      "\n",
      "Seq is: TTAACTAAACTAC\n",
      "0 TTAACTAAACTAC motifs detected in file\n",
      "\n",
      "Seq is: CATTCCCTCCCTA\n",
      "0 CATTCCCTCCCTA motifs detected in file\n",
      "\n",
      "Seq is: GCAACTGGAGCAG\n",
      "0 GCAACTGGAGCAG motifs detected in file\n",
      "\n",
      "Seq is: ACAATACATTTCC\n",
      "0 ACAATACATTTCC motifs detected in file\n",
      "\n",
      "Seq is: TACGCATCCCTAC\n",
      "0 TACGCATCCCTAC motifs detected in file\n",
      "\n",
      "Seq is: CCTCCCGCTACTC\n",
      "0 CCTCCCGCTACTC motifs detected in file\n",
      "\n",
      "Seq is: CTGATCCTACTAC\n",
      "0 CTGATCCTACTAC motifs detected in file\n",
      "\n",
      "Seq is: CGCCCGTCCTATC\n",
      "0 CGCCCGTCCTATC motifs detected in file\n",
      "\n",
      "Seq is: AAGTCTGCCAAAG\n",
      "0 AAGTCTGCCAAAG motifs detected in file\n",
      "\n",
      "Seq is: ATAAAGGACTCGC\n",
      "0 ATAAAGGACTCGC motifs detected in file\n",
      "\n",
      "Seq is: AGACTCTACAATC\n",
      "0 AGACTCTACAATC motifs detected in file\n",
      "\n",
      "Seq is: ATAATCAATCAAA\n",
      "0 ATAATCAATCAAA motifs detected in file\n",
      "\n",
      "Seq is: ACAACACTCACAA\n",
      "0 ACAACACTCACAA motifs detected in file\n",
      "\n",
      "Seq is: TCCTTTTCCCGTA\n",
      "0 TCCTTTTCCCGTA motifs detected in file\n",
      "\n",
      "Seq is: TGTCCTGATCCTT\n",
      "0 TGTCCTGATCCTT motifs detected in file\n",
      "\n",
      "Seq is: CGTCTATTCCTTT\n",
      "0 CGTCTATTCCTTT motifs detected in file\n",
      "\n",
      "Seq is: TACTAATACTCCC\n",
      "0 TACTAATACTCCC motifs detected in file\n",
      "\n",
      "Seq is: GCACTAGTACTTT\n",
      "0 GCACTAGTACTTT motifs detected in file\n",
      "\n",
      "Seq is: CCACTATTCGAGA\n",
      "0 CCACTATTCGAGA motifs detected in file\n",
      "\n",
      "Seq is: AGATACTATCCCC\n",
      "0 AGATACTATCCCC motifs detected in file\n",
      "\n",
      "Seq is: TTCATCGCAGAAC\n",
      "0 TTCATCGCAGAAC motifs detected in file\n",
      "\n",
      "Seq is: ATCTGGATGAACG\n",
      "0 ATCTGGATGAACG motifs detected in file\n",
      "\n",
      "Seq is: CGACGTACACGGT\n",
      "0 CGACGTACACGGT motifs detected in file\n",
      "\n",
      "Seq is: AATTCTATATATC\n",
      "0 AATTCTATATATC motifs detected in file\n",
      "\n",
      "Seq is: CTAAATCGGATAT\n",
      "0 CTAAATCGGATAT motifs detected in file\n",
      "\n",
      "Seq is: TAAATTGAAACTG\n",
      "0 TAAATTGAAACTG motifs detected in file\n",
      "\n",
      "Seq is: TTTCAATACTTTA\n",
      "0 TTTCAATACTTTA motifs detected in file\n",
      "\n",
      "Seq is: CCAAAAAGATTAT\n",
      "0 CCAAAAAGATTAT motifs detected in file\n",
      "\n",
      "Seq is: GGAAAAACTTTTT\n",
      "0 GGAAAAACTTTTT motifs detected in file\n",
      "\n",
      "Seq is: CAGTACCTCCGGT\n",
      "0 CAGTACCTCCGGT motifs detected in file\n",
      "\n",
      "Seq is: ACCCCAACCGAAC\n",
      "0 ACCCCAACCGAAC motifs detected in file\n",
      "\n",
      "Seq is: TTTGGTCGAAACC\n",
      "0 TTTGGTCGAAACC motifs detected in file\n",
      "\n",
      "Seq is: CCCCAAGCTAAGG\n",
      "0 CCCCAAGCTAAGG motifs detected in file\n",
      "\n",
      "Seq is: AAGGAAAAAACAG\n",
      "0 AAGGAAAAAACAG motifs detected in file\n",
      "\n",
      "Seq is: ATCTAAAATACAT\n",
      "0 ATCTAAAATACAT motifs detected in file\n",
      "\n",
      "Seq is: ATGCCCAAGAAGC\n",
      "0 ATGCCCAAGAAGC motifs detected in file\n",
      "\n",
      "Seq is: TTACACACCATCC\n",
      "0 TTACACACCATCC motifs detected in file\n",
      "\n",
      "Seq is: CACCCCCCGTAGG\n",
      "0 CACCCCCCGTAGG motifs detected in file\n",
      "\n",
      "Seq is: TATATCAGTGAGG\n",
      "0 TATATCAGTGAGG motifs detected in file\n",
      "\n",
      "Seq is: TCCAAATACCTCC\n",
      "0 TCCAAATACCTCC motifs detected in file\n",
      "\n",
      "Seq is: CAAGAAGATGATA\n",
      "0 CAAGAAGATGATA motifs detected in file\n",
      "\n",
      "Seq is: ATCCTGAAAAGCG\n",
      "0 ATCCTGAAAAGCG motifs detected in file\n",
      "\n",
      "Seq is: AAGCTTCGCTTCC\n",
      "0 AAGCTTCGCTTCC motifs detected in file\n",
      "\n",
      "Seq is: GAAGAGTTTAGTA\n",
      "0 GAAGAGTTTAGTA motifs detected in file\n",
      "\n",
      "Seq is: CTTTTAATAATTA\n",
      "0 CTTTTAATAATTA motifs detected in file\n",
      "\n",
      "Seq is: TAATGACGACAAT\n",
      "0 TAATGACGACAAT motifs detected in file\n",
      "\n",
      "Seq is: CTCTTTACTTACT\n",
      "0 CTCTTTACTTACT motifs detected in file\n",
      "\n",
      "Seq is: CGGATGTCTACTA\n",
      "0 CGGATGTCTACTA motifs detected in file\n",
      "\n",
      "Seq is: TCCTACAAAGTAC\n",
      "0 TCCTACAAAGTAC motifs detected in file\n",
      "\n",
      "Seq is: ACCACATACGTAG\n",
      "0 ACCACATACGTAG motifs detected in file\n",
      "\n",
      "Seq is: CCCCATCAGGCTC\n",
      "0 CCCCATCAGGCTC motifs detected in file\n",
      "\n",
      "Seq is: ATTGCAGCCCCGT\n",
      "0 ATTGCAGCCCCGT motifs detected in file\n",
      "\n",
      "Seq is: AAGGCCTATCCGG\n",
      "0 AAGGCCTATCCGG motifs detected in file\n",
      "\n",
      "Seq is: CTCTTTCACAACA\n",
      "0 CTCTTTCACAACA motifs detected in file\n",
      "\n",
      "Seq is: CCCTTCTTTCAAT\n",
      "0 CCCTTCTTTCAAT motifs detected in file\n",
      "\n",
      "Seq is: CTAAATGCGGCTA\n",
      "0 CTAAATGCGGCTA motifs detected in file\n",
      "\n",
      "Seq is: CTTATACTATCAC\n",
      "0 CTTATACTATCAC motifs detected in file\n",
      "\n",
      "Seq is: TTTACCTAAAACC\n",
      "0 TTTACCTAAAACC motifs detected in file\n",
      "\n",
      "Seq is: GCATCCAAACCAG\n",
      "0 GCATCCAAACCAG motifs detected in file\n",
      "\n",
      "Seq is: ATCCCACATCGGA\n",
      "0 ATCCCACATCGGA motifs detected in file\n",
      "\n",
      "Seq is: CTCTTATCCCCTT\n",
      "0 CTCTTATCCCCTT motifs detected in file\n",
      "\n",
      "Seq is: TAGTCACTTACTT\n",
      "0 TAGTCACTTACTT motifs detected in file\n",
      "\n",
      "Seq is: CGGAGGATACTAC\n",
      "0 CGGAGGATACTAC motifs detected in file\n",
      "\n",
      "Seq is: CGTTTATGTCGAG\n",
      "0 CGTTTATGTCGAG motifs detected in file\n",
      "\n",
      "Seq is: GATAACTATCCTG\n",
      "0 GATAACTATCCTG motifs detected in file\n",
      "\n",
      "Seq is: TATCACCTTCACC\n",
      "0 TATCACCTTCACC motifs detected in file\n",
      "\n",
      "Seq is: CGATGTTGCATCA\n",
      "0 CGATGTTGCATCA motifs detected in file\n",
      "\n",
      "Seq is: TGCACAGCACATC\n",
      "0 TGCACAGCACATC motifs detected in file\n",
      "\n",
      "Seq is: ATGCTACAGATCA\n",
      "0 ATGCTACAGATCA motifs detected in file\n",
      "\n",
      "Seq is: CTACTCAAACGAT\n",
      "0 CTACTCAAACGAT motifs detected in file\n",
      "\n",
      "Seq is: TATGTTACGGTCA\n",
      "0 TATGTTACGGTCA motifs detected in file\n",
      "\n",
      "Seq is: GTCCGGTGGATGC\n",
      "0 GTCCGGTGGATGC motifs detected in file\n",
      "\n",
      "Seq is: CACTTTTCTTTCT\n",
      "0 CACTTTTCTTTCT motifs detected in file\n",
      "\n",
      "Seq is: ACTTAGGATCCCG\n",
      "0 ACTTAGGATCCCG motifs detected in file\n",
      "\n",
      "Seq is: AGTCTCGTGACGT\n",
      "0 AGTCTCGTGACGT motifs detected in file\n",
      "\n",
      "Seq is: CGTCTAGTAAAGT\n",
      "0 CGTCTAGTAAAGT motifs detected in file\n",
      "\n",
      "Seq is: ATAACGAAGGCAC\n",
      "0 ATAACGAAGGCAC motifs detected in file\n",
      "\n",
      "Seq is: CTCACACCGCTCA\n",
      "0 CTCACACCGCTCA motifs detected in file\n",
      "\n",
      "Seq is: GTCGATTTATGAA\n",
      "0 GTCGATTTATGAA motifs detected in file\n",
      "\n",
      "Seq is: ACTGCGGCCACCC\n",
      "0 ACTGCGGCCACCC motifs detected in file\n",
      "\n",
      "Seq is: CTATCGCTACTAA\n",
      "0 CTATCGCTACTAA motifs detected in file\n",
      "\n",
      "Seq is: TACCATCGCCTCC\n",
      "0 TACCATCGCCTCC motifs detected in file\n",
      "\n",
      "Seq is: ACTTTATACGAGC\n",
      "0 ACTTTATACGAGC motifs detected in file\n",
      "\n",
      "Seq is: ACACAGATGCAGA\n",
      "0 ACACAGATGCAGA motifs detected in file\n",
      "\n",
      "Seq is: TAAGGATGACATT\n",
      "0 TAAGGATGACATT motifs detected in file\n",
      "\n",
      "Seq is: TATATACCACACG\n",
      "0 TATATACCACACG motifs detected in file\n",
      "\n",
      "Seq is: AGTGTGCTATTTG\n",
      "0 AGTGTGCTATTTG motifs detected in file\n",
      "\n",
      "Seq is: GGATCCTTCGGTT\n",
      "0 GGATCCTTCGGTT motifs detected in file\n",
      "\n",
      "Seq is: AACTATAGTATCG\n",
      "0 AACTATAGTATCG motifs detected in file\n",
      "\n",
      "Seq is: AGTCTGGTATGGA\n",
      "0 AGTCTGGTATGGA motifs detected in file\n",
      "\n",
      "Seq is: TACATAGGTTTAC\n",
      "0 TACATAGGTTTAC motifs detected in file\n",
      "\n",
      "Seq is: CAAGAAAAAAAGG\n",
      "0 CAAGAAAAAAAGG motifs detected in file\n",
      "\n",
      "Seq is: CCTCATCATTCAA\n",
      "0 CCTCATCATTCAA motifs detected in file\n",
      "\n",
      "Seq is: TGTTATACCCTCT\n",
      "0 TGTTATACCCTCT motifs detected in file\n",
      "\n",
      "Seq is: AATAAGGCTTCGG\n",
      "0 AATAAGGCTTCGG motifs detected in file\n",
      "\n",
      "Seq is: ACCATCCTATTCT\n",
      "0 ACCATCCTATTCT motifs detected in file\n",
      "\n",
      "Seq is: TATATTTGAAGTC\n",
      "0 TATATTTGAAGTC motifs detected in file\n",
      "\n",
      "Seq is: CCACTGGCTTTTT\n",
      "0 CCACTGGCTTTTT motifs detected in file\n",
      "\n",
      "Seq is: AGTCTTATCCACA\n",
      "0 AGTCTTATCCACA motifs detected in file\n",
      "\n",
      "Seq is: ACCATATCTTACC\n",
      "0 ACCATATCTTACC motifs detected in file\n",
      "\n",
      "Seq is: CCAGAGGAGGAGG\n",
      "0 CCAGAGGAGGAGG motifs detected in file\n",
      "\n",
      "Seq is: CCGCCCCAGCTTC\n",
      "0 CCGCCCCAGCTTC motifs detected in file\n",
      "\n",
      "Seq is: TTCCACCACAACT\n",
      "0 TTCCACCACAACT motifs detected in file\n",
      "\n",
      "Seq is: CCAACGCCAGACA\n",
      "0 CCAACGCCAGACA motifs detected in file\n",
      "\n",
      "Seq is: ATCATCATATCAC\n",
      "0 ATCATCATATCAC motifs detected in file\n",
      "\n",
      "Seq is: TACGGTCGTCGAT\n",
      "0 TACGGTCGTCGAT motifs detected in file\n",
      "\n",
      "Seq is: CCTGACCCTCTCT\n",
      "0 CCTGACCCTCTCT motifs detected in file\n",
      "\n",
      "Seq is: ATCCTCTTCATCC\n",
      "0 ATCCTCTTCATCC motifs detected in file\n",
      "\n",
      "Seq is: TGACGACACTAAT\n",
      "0 TGACGACACTAAT motifs detected in file\n",
      "\n",
      "Seq is: CCTGCCTAGTCTG\n",
      "0 CCTGCCTAGTCTG motifs detected in file\n",
      "\n",
      "Seq is: CTTCTCCCCGCAA\n",
      "0 CTTCTCCCCGCAA motifs detected in file\n",
      "\n",
      "Seq is: ACCATAACCCAAT\n",
      "0 ACCATAACCCAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCGTCCCCCAAA\n",
      "0 ACCGTCCCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: ATATAACTATTAA\n",
      "0 ATATAACTATTAA motifs detected in file\n",
      "\n",
      "Seq is: CAACACTACTTTA\n",
      "0 CAACACTACTTTA motifs detected in file\n",
      "\n",
      "Seq is: ACTACCGGGGATT\n",
      "0 ACTACCGGGGATT motifs detected in file\n",
      "\n",
      "Seq is: CTATCTCCTCTGT\n",
      "0 CTATCTCCTCTGT motifs detected in file\n",
      "\n",
      "Seq is: GGACGATCCACAT\n",
      "0 GGACGATCCACAT motifs detected in file\n",
      "\n",
      "Seq is: TCCTCTTCTACCA\n",
      "0 TCCTCTTCTACCA motifs detected in file\n",
      "\n",
      "Seq is: ATCCAGATGCCTC\n",
      "0 ATCCAGATGCCTC motifs detected in file\n",
      "\n",
      "Seq is: CGAGGTCCCACCC\n",
      "0 CGAGGTCCCACCC motifs detected in file\n",
      "\n",
      "Seq is: TCATCAAGGGACG\n",
      "0 TCATCAAGGGACG motifs detected in file\n",
      "\n",
      "Seq is: ATTCCCTCCCATC\n",
      "0 ATTCCCTCCCATC motifs detected in file\n",
      "\n",
      "Seq is: TGACAAGTTGGAC\n",
      "0 TGACAAGTTGGAC motifs detected in file\n",
      "\n",
      "Seq is: AAGGACGAGGCCG\n",
      "0 AAGGACGAGGCCG motifs detected in file\n",
      "\n",
      "Seq is: GAGGTGATATCGT\n",
      "0 GAGGTGATATCGT motifs detected in file\n",
      "\n",
      "Seq is: CTACGCTCGTCCT\n",
      "0 CTACGCTCGTCCT motifs detected in file\n",
      "\n",
      "Seq is: CATCCTCTCTCCC\n",
      "0 CATCCTCTCTCCC motifs detected in file\n",
      "\n",
      "Seq is: TCCATTCTCAGTC\n",
      "0 TCCATTCTCAGTC motifs detected in file\n",
      "\n",
      "Seq is: TTCGAATACAACA\n",
      "0 TTCGAATACAACA motifs detected in file\n",
      "\n",
      "Seq is: AATACGCCCCTTT\n",
      "0 AATACGCCCCTTT motifs detected in file\n",
      "\n",
      "Seq is: GCGGTATAGCCCC\n",
      "0 GCGGTATAGCCCC motifs detected in file\n",
      "\n",
      "Seq is: CGTGGCTAATAAT\n",
      "0 CGTGGCTAATAAT motifs detected in file\n",
      "\n",
      "Seq is: CCCCTTGATCAGT\n",
      "0 CCCCTTGATCAGT motifs detected in file\n",
      "\n",
      "Seq is: CAACGGTTTCGGA\n",
      "0 CAACGGTTTCGGA motifs detected in file\n",
      "\n",
      "Seq is: GGCTAATACTACC\n",
      "0 GGCTAATACTACC motifs detected in file\n",
      "\n",
      "Seq is: CATAATGATACTT\n",
      "0 CATAATGATACTT motifs detected in file\n",
      "\n",
      "Seq is: CTTCTAATAATGT\n",
      "0 CTTCTAATAATGT motifs detected in file\n",
      "\n",
      "Seq is: TTACGTACCCGAC\n",
      "0 TTACGTACCCGAC motifs detected in file\n",
      "\n",
      "Seq is: ACTGCTATTGCAA\n",
      "0 ACTGCTATTGCAA motifs detected in file\n",
      "\n",
      "Seq is: CATCTACACCAGC\n",
      "0 CATCTACACCAGC motifs detected in file\n",
      "\n",
      "Seq is: AATGGATCTTCCA\n",
      "0 AATGGATCTTCCA motifs detected in file\n",
      "\n",
      "Seq is: ACGGACCGACCGG\n",
      "0 ACGGACCGACCGG motifs detected in file\n",
      "\n",
      "Seq is: GTCGAGCCGAGCT\n",
      "0 GTCGAGCCGAGCT motifs detected in file\n",
      "\n",
      "Seq is: TATTCCTCCGAAT\n",
      "0 TATTCCTCCGAAT motifs detected in file\n",
      "\n",
      "Seq is: CTCGACACGGATC\n",
      "0 CTCGACACGGATC motifs detected in file\n",
      "\n",
      "Seq is: CTGAGGTCGAGTA\n",
      "0 CTGAGGTCGAGTA motifs detected in file\n",
      "\n",
      "Seq is: CGCGGCTTATTAT\n",
      "0 CGCGGCTTATTAT motifs detected in file\n",
      "\n",
      "Seq is: CCATATCACAAGG\n",
      "0 CCATATCACAAGG motifs detected in file\n",
      "\n",
      "Seq is: TTACAGAAACACC\n",
      "0 TTACAGAAACACC motifs detected in file\n",
      "\n",
      "Seq is: AAACATCTCTTAT\n",
      "0 AAACATCTCTTAT motifs detected in file\n",
      "\n",
      "Seq is: CAGTTGCCAGCCG\n",
      "0 CAGTTGCCAGCCG motifs detected in file\n",
      "\n",
      "Seq is: CTTGTAGTCACCC\n",
      "0 CTTGTAGTCACCC motifs detected in file\n",
      "\n",
      "Seq is: CCACTCCATTTTA\n",
      "0 CCACTCCATTTTA motifs detected in file\n",
      "\n",
      "Seq is: CCGACTCACTTCG\n",
      "0 CCGACTCACTTCG motifs detected in file\n",
      "\n",
      "Seq is: TAACCTGACATTT\n",
      "0 TAACCTGACATTT motifs detected in file\n",
      "\n",
      "Seq is: AGATTTCTGTCCC\n",
      "0 AGATTTCTGTCCC motifs detected in file\n",
      "\n",
      "Seq is: CAATCCGGAGAAA\n",
      "0 CAATCCGGAGAAA motifs detected in file\n",
      "\n",
      "Seq is: AATGGTCGAGGCT\n",
      "0 AATGGTCGAGGCT motifs detected in file\n",
      "\n",
      "Seq is: CCACTAAAAGTAT\n",
      "0 CCACTAAAAGTAT motifs detected in file\n",
      "\n",
      "Seq is: AACTTAACGTTTA\n",
      "0 AACTTAACGTTTA motifs detected in file\n",
      "\n",
      "Seq is: AGCTTCTTCGTCG\n",
      "0 AGCTTCTTCGTCG motifs detected in file\n",
      "\n",
      "Seq is: AAGTTTGGACGGC\n",
      "0 AAGTTTGGACGGC motifs detected in file\n",
      "\n",
      "Seq is: CCCGAAGAGGGCG\n",
      "0 CCCGAAGAGGGCG motifs detected in file\n",
      "\n",
      "Seq is: GAAAAAAGGGCCG\n",
      "0 GAAAAAAGGGCCG motifs detected in file\n",
      "\n",
      "Seq is: CCGCCCTCTTCAT\n",
      "0 CCGCCCTCTTCAT motifs detected in file\n",
      "\n",
      "Seq is: CTAACTTCGGTCA\n",
      "0 CTAACTTCGGTCA motifs detected in file\n",
      "\n",
      "Seq is: ACTAATCCCACGA\n",
      "0 ACTAATCCCACGA motifs detected in file\n",
      "\n",
      "Seq is: ATCGACAATTGAT\n",
      "0 ATCGACAATTGAT motifs detected in file\n",
      "\n",
      "Seq is: TCACAAACACCCA\n",
      "0 TCACAAACACCCA motifs detected in file\n",
      "\n",
      "Seq is: AATTCAGGGTAAC\n",
      "0 AATTCAGGGTAAC motifs detected in file\n",
      "\n",
      "Seq is: CAGATCATTCCCG\n",
      "0 CAGATCATTCCCG motifs detected in file\n",
      "\n",
      "Seq is: AATCGAATTAATT\n",
      "0 AATCGAATTAATT motifs detected in file\n",
      "\n",
      "Seq is: TCACCGACTAAAC\n",
      "0 TCACCGACTAAAC motifs detected in file\n",
      "\n",
      "Seq is: GCAAGTCAACTAC\n",
      "0 GCAAGTCAACTAC motifs detected in file\n",
      "\n",
      "Seq is: GTCTCACCCCAAA\n",
      "0 GTCTCACCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: ACGTCAGGAATCG\n",
      "0 ACGTCAGGAATCG motifs detected in file\n",
      "\n",
      "Seq is: ACAATGTCTTTAA\n",
      "0 ACAATGTCTTTAA motifs detected in file\n",
      "\n",
      "Seq is: TTCATAACGTTGA\n",
      "0 TTCATAACGTTGA motifs detected in file\n",
      "\n",
      "Seq is: ATGACTCCCGAAA\n",
      "0 ATGACTCCCGAAA motifs detected in file\n",
      "\n",
      "Seq is: CTTCCGAGAACCA\n",
      "0 CTTCCGAGAACCA motifs detected in file\n",
      "\n",
      "Seq is: GACATAAATTGGA\n",
      "0 GACATAAATTGGA motifs detected in file\n",
      "\n",
      "Seq is: TTTAAAGATATTC\n",
      "0 TTTAAAGATATTC motifs detected in file\n",
      "\n",
      "Seq is: TAATAATCATATT\n",
      "0 TAATAATCATATT motifs detected in file\n",
      "\n",
      "Seq is: TTCCCCTCTATCC\n",
      "0 TTCCCCTCTATCC motifs detected in file\n",
      "\n",
      "Seq is: ATCCTCATCGCAC\n",
      "0 ATCCTCATCGCAC motifs detected in file\n",
      "\n",
      "Seq is: CATTCCCGCTACT\n",
      "0 CATTCCCGCTACT motifs detected in file\n",
      "\n",
      "Seq is: CACACCCCTCCTT\n",
      "0 CACACCCCTCCTT motifs detected in file\n",
      "\n",
      "Seq is: ACCCCACCCAAAA\n",
      "0 ACCCCACCCAAAA motifs detected in file\n",
      "\n",
      "Seq is: CATACAAGTTTGA\n",
      "0 CATACAAGTTTGA motifs detected in file\n",
      "\n",
      "Seq is: CAGTAAAATAAAA\n",
      "0 CAGTAAAATAAAA motifs detected in file\n",
      "\n",
      "Seq is: ATGCAACAATCTA\n",
      "0 ATGCAACAATCTA motifs detected in file\n",
      "\n",
      "Seq is: TACCCCTCATCAC\n",
      "0 TACCCCTCATCAC motifs detected in file\n",
      "\n",
      "Seq is: ACTAACTCCACCT\n",
      "0 ACTAACTCCACCT motifs detected in file\n",
      "\n",
      "Seq is: CATCTAATCCGCA\n",
      "0 CATCTAATCCGCA motifs detected in file\n",
      "\n",
      "Seq is: TCCATCTTCATCT\n",
      "0 TCCATCTTCATCT motifs detected in file\n",
      "\n",
      "Seq is: CCAATTCCTCCCA\n",
      "0 CCAATTCCTCCCA motifs detected in file\n",
      "\n",
      "Seq is: CTACCACCGATAC\n",
      "0 CTACCACCGATAC motifs detected in file\n",
      "\n",
      "Seq is: TACCACCCCTACT\n",
      "0 TACCACCCCTACT motifs detected in file\n",
      "\n",
      "Seq is: ACTCCGATAACAA\n",
      "0 ACTCCGATAACAA motifs detected in file\n",
      "\n",
      "Seq is: AAAACACTTAAGA\n",
      "0 AAAACACTTAAGA motifs detected in file\n",
      "\n",
      "Seq is: AGCTATTACCGGG\n",
      "0 AGCTATTACCGGG motifs detected in file\n",
      "\n",
      "Seq is: TAAACCCGTTTTT\n",
      "0 TAAACCCGTTTTT motifs detected in file\n",
      "\n",
      "Seq is: CGGCCAATCGCCC\n",
      "0 CGGCCAATCGCCC motifs detected in file\n",
      "\n",
      "Seq is: CCGTCCGGAGGAT\n",
      "0 CCGTCCGGAGGAT motifs detected in file\n",
      "\n",
      "Seq is: CCCTCTCCTCCCA\n",
      "0 CCCTCTCCTCCCA motifs detected in file\n",
      "\n",
      "Seq is: CCTACCTTAATTC\n",
      "0 CCTACCTTAATTC motifs detected in file\n",
      "\n",
      "Seq is: CCACAATCAGTAC\n",
      "0 CCACAATCAGTAC motifs detected in file\n",
      "\n",
      "Seq is: AATCGAACAAAGT\n",
      "0 AATCGAACAAAGT motifs detected in file\n",
      "\n",
      "Seq is: CCACGCTCTATCA\n",
      "0 CCACGCTCTATCA motifs detected in file\n",
      "\n",
      "Seq is: TCATCCCAGCACC\n",
      "0 TCATCCCAGCACC motifs detected in file\n",
      "\n",
      "Seq is: ACGACCTCAAATT\n",
      "0 ACGACCTCAAATT motifs detected in file\n",
      "\n",
      "Seq is: CAACTCATCATCC\n",
      "0 CAACTCATCATCC motifs detected in file\n",
      "\n",
      "Seq is: TTACGCCATCATC\n",
      "0 TTACGCCATCATC motifs detected in file\n",
      "\n",
      "Seq is: AATCCTATTATAT\n",
      "0 AATCCTATTATAT motifs detected in file\n",
      "\n",
      "Seq is: TTATCAATTTAAT\n",
      "0 TTATCAATTTAAT motifs detected in file\n",
      "\n",
      "Seq is: TCTTACCAATACA\n",
      "0 TCTTACCAATACA motifs detected in file\n",
      "\n",
      "Seq is: ATCCCAACATGCC\n",
      "0 ATCCCAACATGCC motifs detected in file\n",
      "\n",
      "Seq is: ATCTTGACGATAA\n",
      "0 ATCTTGACGATAA motifs detected in file\n",
      "\n",
      "Seq is: TAAGTAGGATACA\n",
      "0 TAAGTAGGATACA motifs detected in file\n",
      "\n",
      "Seq is: CCCATTAACTCCT\n",
      "0 CCCATTAACTCCT motifs detected in file\n",
      "\n",
      "Seq is: CATACGATTCTAA\n",
      "0 CATACGATTCTAA motifs detected in file\n",
      "\n",
      "Seq is: AACGCATCGACCC\n",
      "0 AACGCATCGACCC motifs detected in file\n",
      "\n",
      "Seq is: AAACCAAATTAGG\n",
      "0 AAACCAAATTAGG motifs detected in file\n",
      "\n",
      "Seq is: TGGAGTTGACGGA\n",
      "0 TGGAGTTGACGGA motifs detected in file\n",
      "\n",
      "Seq is: CGATACTACCTAT\n",
      "0 CGATACTACCTAT motifs detected in file\n",
      "\n",
      "Seq is: TCTAACTCTCTCA\n",
      "0 TCTAACTCTCTCA motifs detected in file\n",
      "\n",
      "Seq is: CTCCTCTTCCGAA\n",
      "0 CTCCTCTTCCGAA motifs detected in file\n",
      "\n",
      "Seq is: TGCAAATCACTCC\n",
      "0 TGCAAATCACTCC motifs detected in file\n",
      "\n",
      "Seq is: CTCTCTAAACCAT\n",
      "0 CTCTCTAAACCAT motifs detected in file\n",
      "\n",
      "Seq is: ATACTAACTCTAC\n",
      "0 ATACTAACTCTAC motifs detected in file\n",
      "\n",
      "Seq is: CCCCGATCAAAAA\n",
      "0 CCCCGATCAAAAA motifs detected in file\n",
      "\n",
      "Seq is: CAGTACACTCTTC\n",
      "0 CAGTACACTCTTC motifs detected in file\n",
      "\n",
      "Seq is: TTCGTCCGGCCTA\n",
      "0 TTCGTCCGGCCTA motifs detected in file\n",
      "\n",
      "Seq is: CAGTCTCCCCACG\n",
      "0 CAGTCTCCCCACG motifs detected in file\n",
      "\n",
      "Seq is: GAACCCATTGGAG\n",
      "0 GAACCCATTGGAG motifs detected in file\n",
      "\n",
      "Seq is: ACCCTGAGTCTTC\n",
      "0 ACCCTGAGTCTTC motifs detected in file\n",
      "\n",
      "Seq is: ACTTTCCCCCGAT\n",
      "0 ACTTTCCCCCGAT motifs detected in file\n",
      "\n",
      "Seq is: AAGGATCAAAATA\n",
      "0 AAGGATCAAAATA motifs detected in file\n",
      "\n",
      "Seq is: ACGATATCGATAA\n",
      "0 ACGATATCGATAA motifs detected in file\n",
      "\n",
      "Seq is: TACTAATAATTAC\n",
      "0 TACTAATAATTAC motifs detected in file\n",
      "\n",
      "Seq is: TACTCATAACTAA\n",
      "0 TACTCATAACTAA motifs detected in file\n",
      "\n",
      "Seq is: CCATCATAACCAA\n",
      "0 CCATCATAACCAA motifs detected in file\n",
      "\n",
      "Seq is: TACCAAGTAACAG\n",
      "0 TACCAAGTAACAG motifs detected in file\n",
      "\n",
      "Seq is: GCCTCTCATATAA\n",
      "0 GCCTCTCATATAA motifs detected in file\n",
      "\n",
      "Seq is: CAACTTCTCCTAT\n",
      "0 CAACTTCTCCTAT motifs detected in file\n",
      "\n",
      "Seq is: CGATAATCTTCCT\n",
      "0 CGATAATCTTCCT motifs detected in file\n",
      "\n",
      "Seq is: AATACCTACGCCA\n",
      "0 AATACCTACGCCA motifs detected in file\n",
      "\n",
      "Seq is: ACGAACGCACTCC\n",
      "0 ACGAACGCACTCC motifs detected in file\n",
      "\n",
      "Seq is: TTTATGAACTACC\n",
      "0 TTTATGAACTACC motifs detected in file\n",
      "\n",
      "Seq is: GTCGAAGACACCT\n",
      "0 GTCGAAGACACCT motifs detected in file\n",
      "\n",
      "Seq is: TGCTCCCAAATAA\n",
      "0 TGCTCCCAAATAA motifs detected in file\n",
      "\n",
      "Seq is: AAAAACCAATCTT\n",
      "0 AAAAACCAATCTT motifs detected in file\n",
      "\n",
      "Seq is: GACCTTATTTTCG\n",
      "0 GACCTTATTTTCG motifs detected in file\n",
      "\n",
      "Seq is: ATCGTACAAATAA\n",
      "0 ATCGTACAAATAA motifs detected in file\n",
      "\n",
      "Seq is: AGATCCGGATGAG\n",
      "0 AGATCCGGATGAG motifs detected in file\n",
      "\n",
      "Seq is: TCCATTTTTTAGT\n",
      "0 TCCATTTTTTAGT motifs detected in file\n",
      "\n",
      "Seq is: CACGCTCGAATCG\n",
      "0 CACGCTCGAATCG motifs detected in file\n",
      "\n",
      "Seq is: CGACACTACTCAC\n",
      "0 CGACACTACTCAC motifs detected in file\n",
      "\n",
      "Seq is: ACGGACGTTTCTA\n",
      "0 ACGGACGTTTCTA motifs detected in file\n",
      "\n",
      "Seq is: CCATCTCATCTAC\n",
      "0 CCATCTCATCTAC motifs detected in file\n",
      "\n",
      "Seq is: TGCCCAACCCGGT\n",
      "0 TGCCCAACCCGGT motifs detected in file\n",
      "\n",
      "Seq is: CCCCTAATTAATC\n",
      "0 CCCCTAATTAATC motifs detected in file\n",
      "\n",
      "Seq is: ATGCCCTTCCCAT\n",
      "0 ATGCCCTTCCCAT motifs detected in file\n",
      "\n",
      "Seq is: ATTGGTTGTAAAA\n",
      "0 ATTGGTTGTAAAA motifs detected in file\n",
      "\n",
      "Seq is: GCCCCATACCCGG\n",
      "0 GCCCCATACCCGG motifs detected in file\n",
      "\n",
      "Seq is: GCTATCGAATAAA\n",
      "0 GCTATCGAATAAA motifs detected in file\n",
      "\n",
      "Seq is: TCGACTGGAATGA\n",
      "0 TCGACTGGAATGA motifs detected in file\n",
      "\n",
      "Seq is: AATCCTACCCCAC\n",
      "0 AATCCTACCCCAC motifs detected in file\n",
      "\n",
      "Seq is: ACTATCCACCGTG\n",
      "0 ACTATCCACCGTG motifs detected in file\n",
      "\n",
      "Seq is: CCTCTTAAAACCT\n",
      "0 CCTCTTAAAACCT motifs detected in file\n",
      "\n",
      "Seq is: AAGAGTCCCTACC\n",
      "0 AAGAGTCCCTACC motifs detected in file\n",
      "\n",
      "Seq is: CAAGCTAAGAGTA\n",
      "0 CAAGCTAAGAGTA motifs detected in file\n",
      "\n",
      "Seq is: TCAGGATCTTTAT\n",
      "0 TCAGGATCTTTAT motifs detected in file\n",
      "\n",
      "Seq is: TCCCCCAAATTCG\n",
      "0 TCCCCCAAATTCG motifs detected in file\n",
      "\n",
      "Seq is: AGGATAATAAATG\n",
      "0 AGGATAATAAATG motifs detected in file\n",
      "\n",
      "Seq is: AGATAGTTTCATT\n",
      "0 AGATAGTTTCATT motifs detected in file\n",
      "\n",
      "Seq is: GAGAAAATAGTCT\n",
      "0 GAGAAAATAGTCT motifs detected in file\n",
      "\n",
      "Seq is: GTATAAAGAATCC\n",
      "0 GTATAAAGAATCC motifs detected in file\n",
      "\n",
      "Seq is: AAACTCCCCCTTA\n",
      "0 AAACTCCCCCTTA motifs detected in file\n",
      "\n",
      "Seq is: CGACCTCTAACAT\n",
      "0 CGACCTCTAACAT motifs detected in file\n",
      "\n",
      "Seq is: TACCCATACCTCT\n",
      "0 TACCCATACCTCT motifs detected in file\n",
      "\n",
      "Seq is: GTATAGTATATTC\n",
      "0 GTATAGTATATTC motifs detected in file\n",
      "\n",
      "Seq is: ATTACGATCCCAC\n",
      "0 ATTACGATCCCAC motifs detected in file\n",
      "\n",
      "Seq is: TCACCATCCTTCA\n",
      "0 TCACCATCCTTCA motifs detected in file\n",
      "\n",
      "Seq is: AAAAAGTATCCTC\n",
      "0 AAAAAGTATCCTC motifs detected in file\n",
      "\n",
      "Seq is: CACATACTCAACC\n",
      "0 CACATACTCAACC motifs detected in file\n",
      "\n",
      "Seq is: AGCATCGCCTTAG\n",
      "0 AGCATCGCCTTAG motifs detected in file\n",
      "\n",
      "Seq is: CCCCCATACGACA\n",
      "0 CCCCCATACGACA motifs detected in file\n",
      "\n",
      "Seq is: AGCTTAAGTATTC\n",
      "0 AGCTTAAGTATTC motifs detected in file\n",
      "\n",
      "Seq is: TTGTCCCTCCAAT\n",
      "0 TTGTCCCTCCAAT motifs detected in file\n",
      "\n",
      "Seq is: CTTCATCCCAGAA\n",
      "0 CTTCATCCCAGAA motifs detected in file\n",
      "\n",
      "Seq is: CCACTGTTTTATA\n",
      "0 CCACTGTTTTATA motifs detected in file\n",
      "\n",
      "Seq is: CAACACATCTCAA\n",
      "0 CAACACATCTCAA motifs detected in file\n",
      "\n",
      "Seq is: GTCCCCTCTCACG\n",
      "0 GTCCCCTCTCACG motifs detected in file\n",
      "\n",
      "Seq is: CAGTATACAACAA\n",
      "0 CAGTATACAACAA motifs detected in file\n",
      "\n",
      "Seq is: GGATCCTTCTAAC\n",
      "0 GGATCCTTCTAAC motifs detected in file\n",
      "\n",
      "Seq is: ATCACCACTCCCA\n",
      "0 ATCACCACTCCCA motifs detected in file\n",
      "\n",
      "Seq is: CAAATAATATTAT\n",
      "0 CAAATAATATTAT motifs detected in file\n",
      "\n",
      "Seq is: TACAAACACATAA\n",
      "0 TACAAACACATAA motifs detected in file\n",
      "\n",
      "Seq is: GCCGATACTTCTT\n",
      "0 GCCGATACTTCTT motifs detected in file\n",
      "\n",
      "Seq is: ATCCCGCTTCCCC\n",
      "0 ATCCCGCTTCCCC motifs detected in file\n",
      "\n",
      "Seq is: GGACGCCGCATAA\n",
      "0 GGACGCCGCATAA motifs detected in file\n",
      "\n",
      "Seq is: GCTACAACTTCGG\n",
      "0 GCTACAACTTCGG motifs detected in file\n",
      "\n",
      "Seq is: ACTCTGATCAAGC\n",
      "0 ACTCTGATCAAGC motifs detected in file\n",
      "\n",
      "Seq is: CTGAGGGGAAGCC\n",
      "0 CTGAGGGGAAGCC motifs detected in file\n",
      "\n",
      "Seq is: GTTCCAGCTTCCC\n",
      "0 GTTCCAGCTTCCC motifs detected in file\n",
      "\n",
      "Seq is: CCAAGCCAACCAG\n",
      "0 CCAAGCCAACCAG motifs detected in file\n",
      "\n",
      "Seq is: AGACGATCACACC\n",
      "0 AGACGATCACACC motifs detected in file\n",
      "\n",
      "Seq is: TCTATTTAGTATA\n",
      "0 TCTATTTAGTATA motifs detected in file\n",
      "\n",
      "Seq is: ATACCGGTTCCCA\n",
      "0 ATACCGGTTCCCA motifs detected in file\n",
      "\n",
      "Seq is: GTACTACCGTCCT\n",
      "0 GTACTACCGTCCT motifs detected in file\n",
      "\n",
      "Seq is: CATTAGTCTCCAC\n",
      "0 CATTAGTCTCCAC motifs detected in file\n",
      "\n",
      "Seq is: AAGAACACAACAC\n",
      "0 AAGAACACAACAC motifs detected in file\n",
      "\n",
      "Seq is: TATTCCCACCTCT\n",
      "0 TATTCCCACCTCT motifs detected in file\n",
      "\n",
      "Seq is: CCAATTTCCTCGG\n",
      "0 CCAATTTCCTCGG motifs detected in file\n",
      "\n",
      "Seq is: TGAATAATCATTA\n",
      "0 TGAATAATCATTA motifs detected in file\n",
      "\n",
      "Seq is: CAACTATCATCTT\n",
      "0 CAACTATCATCTT motifs detected in file\n",
      "\n",
      "Seq is: ACTACCGATCCCA\n",
      "0 ACTACCGATCCCA motifs detected in file\n",
      "\n",
      "Seq is: CTGAAGTATACTC\n",
      "0 CTGAAGTATACTC motifs detected in file\n",
      "\n",
      "Seq is: TAACAAACCCGAT\n",
      "0 TAACAAACCCGAT motifs detected in file\n",
      "\n",
      "Seq is: GACGAGCGTCACG\n",
      "0 GACGAGCGTCACG motifs detected in file\n",
      "\n",
      "Seq is: CGGCTAGTCCCGC\n",
      "0 CGGCTAGTCCCGC motifs detected in file\n",
      "\n",
      "Complement found from 3671 to 3684\n",
      "Seq is: ATCAAACTCAAAC\n",
      "1 ATCAAACTCAAAC motifs detected in file\n",
      "\n",
      "Seq is: TACGAGTGGGACT\n",
      "0 TACGAGTGGGACT motifs detected in file\n",
      "\n",
      "Seq is: AGTCTCCTAACTC\n",
      "0 AGTCTCCTAACTC motifs detected in file\n",
      "\n",
      "Seq is: ATTTGCCGATCCG\n",
      "0 ATTTGCCGATCCG motifs detected in file\n",
      "\n",
      "Seq is: ATCTCCACCGATC\n",
      "0 ATCTCCACCGATC motifs detected in file\n",
      "\n",
      "Seq is: TTATTTATCCTCC\n",
      "0 TTATTTATCCTCC motifs detected in file\n",
      "\n",
      "Seq is: GGATCCAACTCCA\n",
      "0 GGATCCAACTCCA motifs detected in file\n",
      "\n",
      "Seq is: ACTGGTCCCCCAA\n",
      "0 ACTGGTCCCCCAA motifs detected in file\n",
      "\n",
      "Seq is: CCCATACCCCTCC\n",
      "0 CCCATACCCCTCC motifs detected in file\n",
      "\n",
      "Seq is: CCCCAAGTATCAT\n",
      "0 CCCCAAGTATCAT motifs detected in file\n",
      "\n",
      "Seq is: CTTCTCGCTACCA\n",
      "0 CTTCTCGCTACCA motifs detected in file\n",
      "\n",
      "Seq is: CTCTCGATTCCAG\n",
      "0 CTCTCGATTCCAG motifs detected in file\n",
      "\n",
      "Seq is: CCCCGCCACTACA\n",
      "0 CCCCGCCACTACA motifs detected in file\n",
      "\n",
      "Seq is: TCTCCCACTACCA\n",
      "0 TCTCCCACTACCA motifs detected in file\n",
      "\n",
      "Seq is: TCTACACCGCCCA\n",
      "0 TCTACACCGCCCA motifs detected in file\n",
      "\n",
      "Seq is: AAATCCCCGAGAA\n",
      "0 AAATCCCCGAGAA motifs detected in file\n",
      "\n",
      "Seq is: ACCACTTCTCAAA\n",
      "0 ACCACTTCTCAAA motifs detected in file\n",
      "\n",
      "Seq is: ATACCGCAGTCGC\n",
      "0 ATACCGCAGTCGC motifs detected in file\n",
      "\n",
      "Seq is: TTCCCAACATCAT\n",
      "0 TTCCCAACATCAT motifs detected in file\n",
      "\n",
      "Seq is: CGGGCATCCCCGG\n",
      "0 CGGGCATCCCCGG motifs detected in file\n",
      "\n",
      "Seq is: ATGTTGCAACCCC\n",
      "0 ATGTTGCAACCCC motifs detected in file\n",
      "\n",
      "Seq is: GGAAACGCATCAA\n",
      "0 GGAAACGCATCAA motifs detected in file\n",
      "\n",
      "Seq is: CATATATCGGATC\n",
      "0 CATATATCGGATC motifs detected in file\n",
      "\n",
      "Seq is: TTAAAAAGCAAGC\n",
      "0 TTAAAAAGCAAGC motifs detected in file\n",
      "\n",
      "Seq is: CATTCGTAATCCT\n",
      "0 CATTCGTAATCCT motifs detected in file\n",
      "\n",
      "Seq is: TACGGTAACGCTA\n",
      "0 TACGGTAACGCTA motifs detected in file\n",
      "\n",
      "Seq is: ATCTTACCCATGT\n",
      "0 ATCTTACCCATGT motifs detected in file\n",
      "\n",
      "Seq is: TACTCCTCATCCT\n",
      "0 TACTCCTCATCCT motifs detected in file\n",
      "\n",
      "Seq is: CCAACCGGTACCC\n",
      "0 CCAACCGGTACCC motifs detected in file\n",
      "\n",
      "Seq is: ATACAACAATTCT\n",
      "0 ATACAACAATTCT motifs detected in file\n",
      "\n",
      "Seq is: TCTCCTTAACTTG\n",
      "0 TCTCCTTAACTTG motifs detected in file\n",
      "\n",
      "Seq is: GAGACTGACATTT\n",
      "0 GAGACTGACATTT motifs detected in file\n",
      "\n",
      "Seq is: CAAAATTCAAAAT\n",
      "0 CAAAATTCAAAAT motifs detected in file\n",
      "\n",
      "Seq is: ACGCTAATGGCCC\n",
      "0 ACGCTAATGGCCC motifs detected in file\n",
      "\n",
      "Seq is: GAGACGGTAGAAT\n",
      "0 GAGACGGTAGAAT motifs detected in file\n",
      "\n",
      "Seq is: TGTTTGGGACAAG\n",
      "0 TGTTTGGGACAAG motifs detected in file\n",
      "\n",
      "Seq is: AACCCACCCACAC\n",
      "0 AACCCACCCACAC motifs detected in file\n",
      "\n",
      "Seq is: CCATATTATGATT\n",
      "0 CCATATTATGATT motifs detected in file\n",
      "\n",
      "Seq is: CAACTCTACTATA\n",
      "0 CAACTCTACTATA motifs detected in file\n",
      "\n",
      "Seq is: GTAAATGCCCCCT\n",
      "0 GTAAATGCCCCCT motifs detected in file\n",
      "\n",
      "Seq is: TCCGCGAAACACT\n",
      "0 TCCGCGAAACACT motifs detected in file\n",
      "\n",
      "Seq is: TCATCCGGAATAA\n",
      "0 TCATCCGGAATAA motifs detected in file\n",
      "\n",
      "Seq is: AGAGAACAGGAAA\n",
      "0 AGAGAACAGGAAA motifs detected in file\n",
      "\n",
      "Seq is: GCATGTCCCTCCT\n",
      "0 GCATGTCCCTCCT motifs detected in file\n",
      "\n",
      "Seq is: TAAACTTNCATCT\n",
      "0 TAAACTTNCATCT motifs detected in file\n",
      "\n",
      "Seq is: ATCTTTGGCTGGA\n",
      "0 ATCTTTGGCTGGA motifs detected in file\n",
      "\n",
      "Seq is: CCTAATGAGGCCA\n",
      "0 CCTAATGAGGCCA motifs detected in file\n",
      "\n",
      "Seq is: GACTTGAGTCTAG\n",
      "0 GACTTGAGTCTAG motifs detected in file\n",
      "\n",
      "Seq is: TGCATCCTGAAAT\n",
      "0 TGCATCCTGAAAT motifs detected in file\n",
      "\n",
      "Seq is: TAGCAACTTGTTT\n",
      "0 TAGCAACTTGTTT motifs detected in file\n",
      "\n",
      "Seq is: GCTTGGAAATTAT\n",
      "0 GCTTGGAAATTAT motifs detected in file\n",
      "\n",
      "Seq is: CGCCGACGTGGTA\n",
      "0 CGCCGACGTGGTA motifs detected in file\n",
      "\n",
      "Seq is: GCCCTACAGGACT\n",
      "0 GCCCTACAGGACT motifs detected in file\n",
      "\n",
      "Seq is: AGGTTGTAGCTCC\n",
      "0 AGGTTGTAGCTCC motifs detected in file\n",
      "\n",
      "Seq is: AGCATTTGGGATA\n",
      "0 AGCATTTGGGATA motifs detected in file\n",
      "\n",
      "Seq is: ACAACTATACCTG\n",
      "0 ACAACTATACCTG motifs detected in file\n",
      "\n",
      "Seq is: AGATCTTATCCTA\n",
      "0 AGATCTTATCCTA motifs detected in file\n",
      "\n",
      "Seq is: ACGCGACAATAGG\n",
      "0 ACGCGACAATAGG motifs detected in file\n",
      "\n",
      "Seq is: GATCCCATTGAAC\n",
      "0 GATCCCATTGAAC motifs detected in file\n",
      "\n",
      "Seq is: AAGGCAACCAGTT\n",
      "0 AAGGCAACCAGTT motifs detected in file\n",
      "\n",
      "Seq is: CAATAACCTAGTT\n",
      "0 CAATAACCTAGTT motifs detected in file\n",
      "\n",
      "Seq is: AACTCATATCATC\n",
      "0 AACTCATATCATC motifs detected in file\n",
      "\n",
      "Seq is: AAGCGAAACTGAC\n",
      "0 AAGCGAAACTGAC motifs detected in file\n",
      "\n",
      "Seq is: CACTTCAGAATCG\n",
      "0 CACTTCAGAATCG motifs detected in file\n",
      "\n",
      "Seq is: TACATGACGAGCC\n",
      "0 TACATGACGAGCC motifs detected in file\n",
      "\n",
      "Seq is: TCCAACCCAAGAC\n",
      "0 TCCAACCCAAGAC motifs detected in file\n",
      "\n",
      "Seq is: GAGGCTCCAGCGG\n",
      "0 GAGGCTCCAGCGG motifs detected in file\n",
      "\n",
      "Seq is: GGTTGGCTTTAAA\n",
      "0 GGTTGGCTTTAAA motifs detected in file\n",
      "\n",
      "Seq is: AATTACGTCCAAA\n",
      "0 AATTACGTCCAAA motifs detected in file\n",
      "\n",
      "Seq is: CCATCAAATCCTG\n",
      "0 CCATCAAATCCTG motifs detected in file\n",
      "\n",
      "Seq is: GACACCCAAACAA\n",
      "0 GACACCCAAACAA motifs detected in file\n",
      "\n",
      "Seq is: TCCATGACAAACG\n",
      "0 TCCATGACAAACG motifs detected in file\n",
      "\n",
      "Seq is: TAATTATTTAATT\n",
      "0 TAATTATTTAATT motifs detected in file\n",
      "\n",
      "Seq is: TCGAGGTATCCCA\n",
      "0 TCGAGGTATCCCA motifs detected in file\n",
      "\n",
      "Seq is: GAAGAGCAGAACG\n",
      "0 GAAGAGCAGAACG motifs detected in file\n",
      "\n",
      "Seq is: ACACAATACGGGC\n",
      "0 ACACAATACGGGC motifs detected in file\n",
      "\n",
      "Seq is: GGAGAAGTGCCCG\n",
      "0 GGAGAAGTGCCCG motifs detected in file\n",
      "\n",
      "Seq is: TCCAGTTAAAGTG\n",
      "0 TCCAGTTAAAGTG motifs detected in file\n",
      "\n",
      "Seq is: ACCAATTTTCATT\n",
      "0 ACCAATTTTCATT motifs detected in file\n",
      "\n",
      "Seq is: CTCTGTCGACTTG\n",
      "0 CTCTGTCGACTTG motifs detected in file\n",
      "\n",
      "Seq is: GGAGCACCTCGGT\n",
      "0 GGAGCACCTCGGT motifs detected in file\n",
      "\n",
      "Seq is: AAGTATGTCCAGG\n",
      "0 AAGTATGTCCAGG motifs detected in file\n",
      "\n",
      "Seq is: GATAAATTCCTTG\n",
      "0 GATAAATTCCTTG motifs detected in file\n",
      "\n",
      "Seq is: TTCACTAATACGA\n",
      "0 TTCACTAATACGA motifs detected in file\n",
      "\n",
      "Seq is: TGGAAACGTGCCA\n",
      "0 TGGAAACGTGCCA motifs detected in file\n",
      "\n",
      "Seq is: ATCCCATGGCGCC\n",
      "0 ATCCCATGGCGCC motifs detected in file\n",
      "\n",
      "Seq is: GGCAATTTGTACA\n",
      "0 GGCAATTTGTACA motifs detected in file\n",
      "\n",
      "Seq is: CAGTGACCCGTCC\n",
      "0 CAGTGACCCGTCC motifs detected in file\n",
      "\n",
      "Seq is: GCCACGGAGATTA\n",
      "0 GCCACGGAGATTA motifs detected in file\n",
      "\n",
      "Seq is: TGACCACTACGAT\n",
      "0 TGACCACTACGAT motifs detected in file\n",
      "\n",
      "Seq is: CTCCACTACAAAA\n",
      "0 CTCCACTACAAAA motifs detected in file\n",
      "\n",
      "Seq is: ACCATTTGTCCGC\n",
      "0 ACCATTTGTCCGC motifs detected in file\n",
      "\n",
      "Seq is: CCCATTCTAAACG\n",
      "0 CCCATTCTAAACG motifs detected in file\n",
      "\n",
      "Seq is: GCTCAAGGAAAAT\n",
      "0 GCTCAAGGAAAAT motifs detected in file\n",
      "\n",
      "Seq is: GAAAAAAATTGGA\n",
      "0 GAAAAAAATTGGA motifs detected in file\n",
      "\n",
      "Seq is: AAGGAATACTCGT\n",
      "0 AAGGAATACTCGT motifs detected in file\n",
      "\n",
      "Seq is: ACGGACACAACCC\n",
      "0 ACGGACACAACCC motifs detected in file\n",
      "\n",
      "Seq is: AACTGTCACTCCC\n",
      "0 AACTGTCACTCCC motifs detected in file\n",
      "\n",
      "Seq is: ATTATTACTGAAC\n",
      "0 ATTATTACTGAAC motifs detected in file\n",
      "\n",
      "Seq is: AACCAACTAACAT\n",
      "0 AACCAACTAACAT motifs detected in file\n",
      "\n",
      "Seq is: CTATAACCCGACA\n",
      "0 CTATAACCCGACA motifs detected in file\n",
      "\n",
      "Seq is: ATTAACAGTCAAG\n",
      "0 ATTAACAGTCAAG motifs detected in file\n",
      "\n",
      "Seq is: TCACAAAATTAGA\n",
      "0 TCACAAAATTAGA motifs detected in file\n",
      "\n",
      "Seq is: CTGCGTCCGAATA\n",
      "0 CTGCGTCCGAATA motifs detected in file\n",
      "\n",
      "Seq is: CGCCTCCTCTTAC\n",
      "0 CGCCTCCTCTTAC motifs detected in file\n",
      "\n",
      "Seq is: AAAAGTACAATGA\n",
      "0 AAAAGTACAATGA motifs detected in file\n",
      "\n",
      "Seq is: ATATGATTGTAAT\n",
      "0 ATATGATTGTAAT motifs detected in file\n",
      "\n",
      "Seq is: CAAGAAGATATCC\n",
      "0 CAAGAAGATATCC motifs detected in file\n",
      "\n",
      "Seq is: CACTATCTAACCA\n",
      "0 CACTATCTAACCA motifs detected in file\n",
      "\n",
      "Seq is: GGTTAACCCACAC\n",
      "0 GGTTAACCCACAC motifs detected in file\n",
      "\n",
      "Seq is: TCCTCAAGTCAAT\n",
      "0 TCCTCAAGTCAAT motifs detected in file\n",
      "\n",
      "Seq is: ATACAAACCCTAA\n",
      "0 ATACAAACCCTAA motifs detected in file\n",
      "\n",
      "Seq is: AAAATCCATCACC\n",
      "0 AAAATCCATCACC motifs detected in file\n",
      "\n",
      "Seq is: CACAACTCGAACT\n",
      "0 CACAACTCGAACT motifs detected in file\n",
      "\n",
      "Seq is: TGCGAAAGAATTA\n",
      "0 TGCGAAAGAATTA motifs detected in file\n",
      "\n",
      "Seq is: ACCACCGACGAAA\n",
      "0 ACCACCGACGAAA motifs detected in file\n",
      "\n",
      "Seq is: ATCCGGATGATAC\n",
      "0 ATCCGGATGATAC motifs detected in file\n",
      "\n",
      "Seq is: CCACAATTTAAAA\n",
      "0 CCACAATTTAAAA motifs detected in file\n",
      "\n",
      "Seq is: AATGAGAGAGATG\n",
      "0 AATGAGAGAGATG motifs detected in file\n",
      "\n",
      "Seq is: TTCCAAAAAAGGA\n",
      "0 TTCCAAAAAAGGA motifs detected in file\n",
      "\n",
      "Seq is: TCACAGGTTTCTC\n",
      "0 TCACAGGTTTCTC motifs detected in file\n",
      "\n",
      "Seq is: GACAAGGAGAAAC\n",
      "0 GACAAGGAGAAAC motifs detected in file\n",
      "\n",
      "Seq is: CTGATTGTCAATT\n",
      "0 CTGATTGTCAATT motifs detected in file\n",
      "\n",
      "Seq is: TAAATGTTCCCCT\n",
      "0 TAAATGTTCCCCT motifs detected in file\n",
      "\n",
      "Seq is: AAATCTCCCAAGA\n",
      "0 AAATCTCCCAAGA motifs detected in file\n",
      "\n",
      "Seq is: CACCCGTTTAAAT\n",
      "0 CACCCGTTTAAAT motifs detected in file\n",
      "\n",
      "Seq is: TTCAACTTGATTC\n",
      "0 TTCAACTTGATTC motifs detected in file\n",
      "\n",
      "Seq is: TAAGATAGAACCT\n",
      "0 TAAGATAGAACCT motifs detected in file\n",
      "\n",
      "Seq is: GTTGGTCGATAGT\n",
      "0 GTTGGTCGATAGT motifs detected in file\n",
      "\n",
      "Seq is: GGTCCGAGCCATC\n",
      "0 GGTCCGAGCCATC motifs detected in file\n",
      "\n",
      "Seq is: CAAACAGCGGAGA\n",
      "0 CAAACAGCGGAGA motifs detected in file\n",
      "\n",
      "Seq is: TGGATATTTAGAA\n",
      "0 TGGATATTTAGAA motifs detected in file\n",
      "\n",
      "Seq is: GGGTGATAAAACG\n",
      "0 GGGTGATAAAACG motifs detected in file\n",
      "\n",
      "Seq is: ATGTATCTGCCCA\n",
      "0 ATGTATCTGCCCA motifs detected in file\n",
      "\n",
      "Seq is: CACGAGAAAATCG\n",
      "0 CACGAGAAAATCG motifs detected in file\n",
      "\n",
      "Seq is: ACAAGAATCCATC\n",
      "0 ACAAGAATCCATC motifs detected in file\n",
      "\n",
      "Seq is: GAGCAGACCAAAG\n",
      "0 GAGCAGACCAAAG motifs detected in file\n",
      "\n",
      "Seq is: CCCCCAGAATCGA\n",
      "0 CCCCCAGAATCGA motifs detected in file\n",
      "\n",
      "Seq is: AACCGAGAGGAAC\n",
      "0 AACCGAGAGGAAC motifs detected in file\n",
      "\n",
      "Seq is: GTTTCAATAAAGA\n",
      "0 GTTTCAATAAAGA motifs detected in file\n",
      "\n",
      "Seq is: TCAATTAAGTAAT\n",
      "0 TCAATTAAGTAAT motifs detected in file\n",
      "\n",
      "Seq is: ACGTCTTCCATAT\n",
      "0 ACGTCTTCCATAT motifs detected in file\n",
      "\n",
      "Seq is: CCCCAATCAGGAA\n",
      "0 CCCCAATCAGGAA motifs detected in file\n",
      "\n",
      "Seq is: CGATATAATACGA\n",
      "0 CGATATAATACGA motifs detected in file\n",
      "\n",
      "Seq is: ACCAATATTAAAA\n",
      "0 ACCAATATTAAAA motifs detected in file\n",
      "\n",
      "Seq is: AGTAGAAAGGGAA\n",
      "0 AGTAGAAAGGGAA motifs detected in file\n",
      "\n",
      "Seq is: CGCCATGATATAG\n",
      "0 CGCCATGATATAG motifs detected in file\n",
      "\n",
      "Seq is: ATAACGCGGTCCA\n",
      "0 ATAACGCGGTCCA motifs detected in file\n",
      "\n",
      "Seq is: AAGTTAAAGATAG\n",
      "0 AAGTTAAAGATAG motifs detected in file\n",
      "\n",
      "Seq is: CGGATATGAAATA\n",
      "0 CGGATATGAAATA motifs detected in file\n",
      "\n",
      "Seq is: AACCCATTTACCA\n",
      "0 AACCCATTTACCA motifs detected in file\n",
      "\n",
      "Seq is: AACCGATTCCAAC\n",
      "0 AACCGATTCCAAC motifs detected in file\n",
      "\n",
      "Seq is: AGACCATCATTCC\n",
      "0 AGACCATCATTCC motifs detected in file\n",
      "\n",
      "Seq is: ACCTCACCCAAAC\n",
      "0 ACCTCACCCAAAC motifs detected in file\n",
      "\n",
      "Seq is: CCCGATCCAAATC\n",
      "0 CCCGATCCAAATC motifs detected in file\n",
      "\n",
      "Seq is: GAGTCTCGCCAGT\n",
      "0 GAGTCTCGCCAGT motifs detected in file\n",
      "\n",
      "Seq is: TCAATTCAACTTT\n",
      "0 TCAATTCAACTTT motifs detected in file\n",
      "\n",
      "Seq is: AGAGGATTCACAT\n",
      "0 AGAGGATTCACAT motifs detected in file\n",
      "\n",
      "Seq is: TCAACCCACGAAA\n",
      "0 TCAACCCACGAAA motifs detected in file\n",
      "\n",
      "Seq is: CACAATTCGATGT\n",
      "0 CACAATTCGATGT motifs detected in file\n",
      "\n",
      "Seq is: GAGACCAAGCAGG\n",
      "0 GAGACCAAGCAGG motifs detected in file\n",
      "\n",
      "Seq is: TTCACGTGAAAGG\n",
      "0 TTCACGTGAAAGG motifs detected in file\n",
      "\n",
      "Seq is: TCATGTGAATGGT\n",
      "0 TCATGTGAATGGT motifs detected in file\n",
      "\n",
      "Seq is: ACAATGCTGAACA\n",
      "0 ACAATGCTGAACA motifs detected in file\n",
      "\n",
      "Seq is: GAGGAGATATATT\n",
      "0 GAGGAGATATATT motifs detected in file\n",
      "\n",
      "Seq is: TACGCATCCCCAA\n",
      "0 TACGCATCCCCAA motifs detected in file\n",
      "\n",
      "Seq is: AATCAATTTACAG\n",
      "0 AATCAATTTACAG motifs detected in file\n",
      "\n",
      "Seq is: GAAACTTCATATG\n",
      "0 GAAACTTCATATG motifs detected in file\n",
      "\n",
      "Seq is: AACTCCTCCCACT\n",
      "0 AACTCCTCCCACT motifs detected in file\n",
      "\n",
      "Seq is: GCCCGCCACACAT\n",
      "0 GCCCGCCACACAT motifs detected in file\n",
      "\n",
      "Seq is: GCGCGAAGTCCCG\n",
      "0 GCGCGAAGTCCCG motifs detected in file\n",
      "\n",
      "Seq is: GGACAAGTTGATT\n",
      "0 GGACAAGTTGATT motifs detected in file\n",
      "\n",
      "Seq is: CGTGAGATGAGAA\n",
      "0 CGTGAGATGAGAA motifs detected in file\n",
      "\n",
      "Seq is: TCAAATGACGATT\n",
      "0 TCAAATGACGATT motifs detected in file\n",
      "\n",
      "Seq is: TAGGTGGAAGCTG\n",
      "0 TAGGTGGAAGCTG motifs detected in file\n",
      "\n",
      "Seq is: GGAATTCAAAGTA\n",
      "0 GGAATTCAAAGTA motifs detected in file\n",
      "\n",
      "Seq is: TTCCCGATAGCAT\n",
      "0 TTCCCGATAGCAT motifs detected in file\n",
      "\n",
      "Seq is: CAAAAGACCCCAT\n",
      "0 CAAAAGACCCCAT motifs detected in file\n",
      "\n",
      "Seq is: CTTTTACATCGGG\n",
      "0 CTTTTACATCGGG motifs detected in file\n",
      "\n",
      "Seq is: TAAAGAACGGTGG\n",
      "0 TAAAGAACGGTGG motifs detected in file\n",
      "\n",
      "Seq is: AGTACCCGATGTG\n",
      "0 AGTACCCGATGTG motifs detected in file\n",
      "\n",
      "Seq is: GAACTGGATTGCA\n",
      "0 GAACTGGATTGCA motifs detected in file\n",
      "\n",
      "Seq is: GAAATGCACCCAT\n",
      "0 GAAATGCACCCAT motifs detected in file\n",
      "\n",
      "Seq is: GAACGCGAATGAA\n",
      "0 GAACGCGAATGAA motifs detected in file\n",
      "\n",
      "Seq is: ACATCGGAAGTAG\n",
      "0 ACATCGGAAGTAG motifs detected in file\n",
      "\n",
      "Seq is: TCCCAAACGACTT\n",
      "0 TCCCAAACGACTT motifs detected in file\n",
      "\n",
      "Seq is: CTACCGCCATATA\n",
      "0 CTACCGCCATATA motifs detected in file\n",
      "\n",
      "Seq is: TCCGACTCGTTCT\n",
      "0 TCCGACTCGTTCT motifs detected in file\n",
      "\n",
      "Seq is: CCACCACTCCAAC\n",
      "0 CCACCACTCCAAC motifs detected in file\n",
      "\n",
      "Seq is: TAGCCCCAAATAG\n",
      "0 TAGCCCCAAATAG motifs detected in file\n",
      "\n",
      "Seq is: CTAATGTCTTGTC\n",
      "0 CTAATGTCTTGTC motifs detected in file\n",
      "\n",
      "Seq is: CGAGGAGATCTCC\n",
      "0 CGAGGAGATCTCC motifs detected in file\n",
      "\n",
      "Seq is: CTATACTTCGTGG\n",
      "0 CTATACTTCGTGG motifs detected in file\n",
      "\n",
      "Seq is: CGGTCCAGGAAAC\n",
      "0 CGGTCCAGGAAAC motifs detected in file\n",
      "\n",
      "Seq is: TCAAAATTCGACA\n",
      "0 TCAAAATTCGACA motifs detected in file\n",
      "\n",
      "Seq is: CCGAGCATCACAA\n",
      "0 CCGAGCATCACAA motifs detected in file\n",
      "\n",
      "Seq is: GACCGCTCGTCAA\n",
      "0 GACCGCTCGTCAA motifs detected in file\n",
      "\n",
      "Seq is: AACAACTAAATTG\n",
      "0 AACAACTAAATTG motifs detected in file\n",
      "\n",
      "Seq is: ACAACTCCAAATC\n",
      "0 ACAACTCCAAATC motifs detected in file\n",
      "\n",
      "Seq is: CCGATTCGTATCA\n",
      "0 CCGATTCGTATCA motifs detected in file\n",
      "\n",
      "Seq is: CCCCATAGATTAG\n",
      "0 CCCCATAGATTAG motifs detected in file\n",
      "\n",
      "Seq is: GGTCAAACCCAGA\n",
      "0 GGTCAAACCCAGA motifs detected in file\n",
      "\n",
      "Seq is: ATCGATAACACAC\n",
      "0 ATCGATAACACAC motifs detected in file\n",
      "\n",
      "Seq is: AAGTCTATACAAT\n",
      "0 AAGTCTATACAAT motifs detected in file\n",
      "\n",
      "Seq is: TTCGGTGAAAGCA\n",
      "0 TTCGGTGAAAGCA motifs detected in file\n",
      "\n",
      "Seq is: TCAGATAAAACAC\n",
      "0 TCAGATAAAACAC motifs detected in file\n",
      "\n",
      "Seq is: AGTTGACCTCAAA\n",
      "0 AGTTGACCTCAAA motifs detected in file\n",
      "\n",
      "Seq is: AAATGTTGAGTCC\n",
      "0 AAATGTTGAGTCC motifs detected in file\n",
      "\n",
      "Seq is: ACTCAAAATCGAA\n",
      "0 ACTCAAAATCGAA motifs detected in file\n",
      "\n",
      "Seq is: ATAACCCCTCCCC\n",
      "0 ATAACCCCTCCCC motifs detected in file\n",
      "\n",
      "Seq is: CACTAGATTTTGT\n",
      "0 CACTAGATTTTGT motifs detected in file\n",
      "\n",
      "Seq is: GAGAAATGCGGCC\n",
      "0 GAGAAATGCGGCC motifs detected in file\n",
      "\n",
      "Seq is: GAAGATAACTGAA\n",
      "0 GAAGATAACTGAA motifs detected in file\n",
      "\n",
      "Seq is: CCCAATTAGCACA\n",
      "0 CCCAATTAGCACA motifs detected in file\n",
      "\n",
      "Seq is: CTGGCGCCACCGA\n",
      "0 CTGGCGCCACCGA motifs detected in file\n",
      "\n",
      "Seq is: CCGTGCTTTAACT\n",
      "0 CCGTGCTTTAACT motifs detected in file\n",
      "\n",
      "Seq is: GGTTGGGACCCCA\n",
      "0 GGTTGGGACCCCA motifs detected in file\n",
      "\n",
      "Seq is: ATCATATCGAATC\n",
      "0 ATCATATCGAATC motifs detected in file\n",
      "\n",
      "Seq is: AATTTGAAAGCAA\n",
      "0 AATTTGAAAGCAA motifs detected in file\n",
      "\n",
      "Seq is: ATAACGATTTCCA\n",
      "0 ATAACGATTTCCA motifs detected in file\n",
      "\n",
      "Seq is: ATTAGTGACGACA\n",
      "0 ATTAGTGACGACA motifs detected in file\n",
      "\n",
      "Seq is: AAGGGCACCCCCA\n",
      "0 AAGGGCACCCCCA motifs detected in file\n",
      "\n",
      "Seq is: CACCGATCCGATT\n",
      "0 CACCGATCCGATT motifs detected in file\n",
      "\n",
      "Seq is: CGCAAAACTCGAC\n",
      "0 CGCAAAACTCGAC motifs detected in file\n",
      "\n",
      "Seq is: GTAACGACGCACG\n",
      "0 GTAACGACGCACG motifs detected in file\n",
      "\n",
      "Seq is: AACTACGAACAAG\n",
      "0 AACTACGAACAAG motifs detected in file\n",
      "\n",
      "Seq is: GAAAACTAGCACC\n",
      "0 GAAAACTAGCACC motifs detected in file\n",
      "\n",
      "Seq is: ACTAAATCTCCCA\n",
      "0 ACTAAATCTCCCA motifs detected in file\n",
      "\n",
      "Seq is: CTTGAGTGACCTT\n",
      "0 CTTGAGTGACCTT motifs detected in file\n",
      "\n",
      "Seq is: GCCCCTACGAACG\n",
      "0 GCCCCTACGAACG motifs detected in file\n",
      "\n",
      "Seq is: TACACATTAGAAT\n",
      "0 TACACATTAGAAT motifs detected in file\n",
      "\n",
      "Seq is: GATTCTCGATTAT\n",
      "0 GATTCTCGATTAT motifs detected in file\n",
      "\n",
      "Seq is: CTTTCCGATCCTG\n",
      "0 CTTTCCGATCCTG motifs detected in file\n",
      "\n",
      "Seq is: GTTTGGATAAACA\n",
      "0 GTTTGGATAAACA motifs detected in file\n",
      "\n",
      "Seq is: AATACCCCACTAC\n",
      "0 AATACCCCACTAC motifs detected in file\n",
      "\n",
      "Seq is: ACTCGGGCAGATT\n",
      "0 ACTCGGGCAGATT motifs detected in file\n",
      "\n",
      "Seq is: TGTAAAAGTCACA\n",
      "0 TGTAAAAGTCACA motifs detected in file\n",
      "\n",
      "Seq is: TAACGAAACTCCT\n",
      "0 TAACGAAACTCCT motifs detected in file\n",
      "\n",
      "Seq is: CCATTCGATGTAT\n",
      "0 CCATTCGATGTAT motifs detected in file\n",
      "\n",
      "Seq is: TTGACACCCCCCA\n",
      "0 TTGACACCCCCCA motifs detected in file\n",
      "\n",
      "Seq is: CAGAAACCCCAAA\n",
      "0 CAGAAACCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: CCAACCAAGCCCC\n",
      "0 CCAACCAAGCCCC motifs detected in file\n",
      "\n",
      "Seq is: ATACCCCAATCGT\n",
      "0 ATACCCCAATCGT motifs detected in file\n",
      "\n",
      "Seq is: CGCCACACACACA\n",
      "0 CGCCACACACACA motifs detected in file\n",
      "\n",
      "Seq is: CGACCCATCCTAC\n",
      "0 CGACCCATCCTAC motifs detected in file\n",
      "\n",
      "Seq is: CCGCCCCCAACAT\n",
      "0 CCGCCCCCAACAT motifs detected in file\n",
      "\n",
      "Seq is: AACTACTCTAATC\n",
      "0 AACTACTCTAATC motifs detected in file\n",
      "\n",
      "Seq is: ATCATACCCTCAC\n",
      "0 ATCATACCCTCAC motifs detected in file\n",
      "\n",
      "Seq is: CCTCCCCTTTTAT\n",
      "0 CCTCCCCTTTTAT motifs detected in file\n",
      "\n",
      "Seq is: TACACAATCAACC\n",
      "0 TACACAATCAACC motifs detected in file\n",
      "\n",
      "Seq is: CCCCACTGACAAT\n",
      "0 CCCCACTGACAAT motifs detected in file\n",
      "\n",
      "Seq is: TTTCACGTATGGC\n",
      "0 TTTCACGTATGGC motifs detected in file\n",
      "\n",
      "Seq is: GGTTTTCTATTTT\n",
      "0 GGTTTTCTATTTT motifs detected in file\n",
      "\n",
      "Seq is: AAACTTTAGACCA\n",
      "0 AAACTTTAGACCA motifs detected in file\n",
      "\n",
      "Seq is: ATCCGACCACAAT\n",
      "0 ATCCGACCACAAT motifs detected in file\n",
      "\n",
      "Seq is: CCCAAGAAACAAA\n",
      "0 CCCAAGAAACAAA motifs detected in file\n",
      "\n",
      "Seq is: AACCCCAAACCGT\n",
      "0 AACCCCAAACCGT motifs detected in file\n",
      "\n",
      "Seq is: CTCTACACAAATT\n",
      "0 CTCTACACAAATT motifs detected in file\n",
      "\n",
      "Seq is: CACGACACCGGTC\n",
      "0 CACGACACCGGTC motifs detected in file\n",
      "\n",
      "Seq is: TTCGCCCCCTCCC\n",
      "0 TTCGCCCCCTCCC motifs detected in file\n",
      "\n",
      "Seq is: CCCCAAACCACCT\n",
      "0 CCCCAAACCACCT motifs detected in file\n",
      "\n",
      "Seq is: TTAAAAAACAATA\n",
      "0 TTAAAAAACAATA motifs detected in file\n",
      "\n",
      "Seq is: CTACAGACACACC\n",
      "0 CTACAGACACACC motifs detected in file\n",
      "\n",
      "Seq is: TTTCACCGACACG\n",
      "0 TTTCACCGACACG motifs detected in file\n",
      "\n",
      "Seq is: TCTGTAAGTTAAC\n",
      "0 TCTGTAAGTTAAC motifs detected in file\n",
      "\n",
      "Seq is: AATAATAATACAG\n",
      "0 AATAATAATACAG motifs detected in file\n",
      "\n",
      "Seq is: GATGTTCGTAATT\n",
      "0 GATGTTCGTAATT motifs detected in file\n",
      "\n",
      "Seq is: AATTAATTGTGTG\n",
      "0 AATTAATTGTGTG motifs detected in file\n",
      "\n",
      "Seq is: AAATCATTCATAC\n",
      "0 AAATCATTCATAC motifs detected in file\n",
      "\n",
      "Seq is: AAGCGGACATTAT\n",
      "0 AAGCGGACATTAT motifs detected in file\n",
      "\n",
      "Seq is: AACTTGCATCCAC\n",
      "0 AACTTGCATCCAC motifs detected in file\n",
      "\n",
      "Seq is: GCTATTTATTATC\n",
      "0 GCTATTTATTATC motifs detected in file\n",
      "\n",
      "Seq is: CTACTCCGTCCTT\n",
      "0 CTACTCCGTCCTT motifs detected in file\n",
      "\n",
      "Seq is: AGTTTCTGTCTAT\n",
      "0 AGTTTCTGTCTAT motifs detected in file\n",
      "\n",
      "Seq is: GACGCTGTATCCC\n",
      "0 GACGCTGTATCCC motifs detected in file\n",
      "\n",
      "Seq is: ACGAGGCCGAGGT\n",
      "0 ACGAGGCCGAGGT motifs detected in file\n",
      "\n",
      "Seq is: CGCAGAGCGTTAC\n",
      "0 CGCAGAGCGTTAC motifs detected in file\n",
      "\n",
      "Seq is: GATAGCGCACGTA\n",
      "0 GATAGCGCACGTA motifs detected in file\n",
      "\n",
      "Seq is: TGGGGGGTCTGCT\n",
      "0 TGGGGGGTCTGCT motifs detected in file\n",
      "\n",
      "Seq is: TTTATGGTTTACG\n",
      "0 TTTATGGTTTACG motifs detected in file\n",
      "\n",
      "Seq is: TACCTCTCGAGGG\n",
      "0 TACCTCTCGAGGG motifs detected in file\n",
      "\n",
      "Seq is: CACTCACCAATTA\n",
      "0 CACTCACCAATTA motifs detected in file\n",
      "\n",
      "Seq is: TCCCACTATCTGG\n",
      "0 TCCCACTATCTGG motifs detected in file\n",
      "\n",
      "Seq is: GTAGCACTACAGAA\n",
      "0 GTAGCACTACAGAA motifs detected in file\n",
      "\n",
      "Seq is: TAAATTCCCCTTGC\n",
      "0 TAAATTCCCCTTGC motifs detected in file\n",
      "\n",
      "Seq is: ACACCCGATAAATC\n",
      "0 ACACCCGATAAATC motifs detected in file\n",
      "\n",
      "Seq is: CGAAATACTGGGAC\n",
      "0 CGAAATACTGGGAC motifs detected in file\n",
      "\n",
      "Seq is: TTCATCCTTGGTCT\n",
      "0 TTCATCCTTGGTCT motifs detected in file\n",
      "\n",
      "Seq is: ACAGCCTATGTCAA\n",
      "0 ACAGCCTATGTCAA motifs detected in file\n",
      "\n",
      "Seq is: GTGAAATCGATGGG\n",
      "0 GTGAAATCGATGGG motifs detected in file\n",
      "\n",
      "Seq is: GGTTCACAATACCC\n",
      "0 GGTTCACAATACCC motifs detected in file\n",
      "\n",
      "Seq is: GGGCCTCGCTCCTC\n",
      "0 GGGCCTCGCTCCTC motifs detected in file\n",
      "\n",
      "Seq is: TCATCGTGAGAACA\n",
      "0 TCATCGTGAGAACA motifs detected in file\n",
      "\n",
      "Seq is: CGCCCTATAACTAA\n",
      "0 CGCCCTATAACTAA motifs detected in file\n",
      "\n",
      "Seq is: AGTGCCTCCTACCA\n",
      "0 AGTGCCTCCTACCA motifs detected in file\n",
      "\n",
      "Seq is: CCAGTTCCCTGGGG\n",
      "0 CCAGTTCCCTGGGG motifs detected in file\n",
      "\n",
      "Seq is: ATAGACTCCCCCCA\n",
      "0 ATAGACTCCCCCCA motifs detected in file\n",
      "\n",
      "Seq is: GTAGGTACCCCTGC\n",
      "0 GTAGGTACCCCTGC motifs detected in file\n",
      "\n",
      "Seq is: TCTTCCCTAAACTG\n",
      "0 TCTTCCCTAAACTG motifs detected in file\n",
      "\n",
      "Seq is: ACATTACACGATAC\n",
      "0 ACATTACACGATAC motifs detected in file\n",
      "\n",
      "Seq is: ATGCCATTTACCGA\n",
      "0 ATGCCATTTACCGA motifs detected in file\n",
      "\n",
      "Seq is: AATACATGATACAT\n",
      "0 AATACATGATACAT motifs detected in file\n",
      "\n",
      "Seq is: GACAATTCCCACCC\n",
      "0 GACAATTCCCACCC motifs detected in file\n",
      "\n",
      "Seq is: ATCCAAACAACCAT\n",
      "0 ATCCAAACAACCAT motifs detected in file\n",
      "\n",
      "Seq is: AGGATCACCCACTC\n",
      "0 AGGATCACCCACTC motifs detected in file\n",
      "\n",
      "Seq is: CCCACCGAAACCTC\n",
      "0 CCCACCGAAACCTC motifs detected in file\n",
      "\n",
      "Seq is: AACGTCAACTACAC\n",
      "0 AACGTCAACTACAC motifs detected in file\n",
      "\n",
      "Seq is: ACTATCAACTCCCA\n",
      "0 ACTATCAACTCCCA motifs detected in file\n",
      "\n",
      "Seq is: ACTAACGACATGAA\n",
      "0 ACTAACGACATGAA motifs detected in file\n",
      "\n",
      "Seq is: CGAACATTCGTACC\n",
      "0 CGAACATTCGTACC motifs detected in file\n",
      "\n",
      "Seq is: CCTCCCCCAAAACT\n",
      "0 CCTCCCCCAAAACT motifs detected in file\n",
      "\n",
      "Seq is: ACACCTAACCCAAA\n",
      "0 ACACCTAACCCAAA motifs detected in file\n",
      "\n",
      "Seq is: AATACATGATGTCC\n",
      "0 AATACATGATGTCC motifs detected in file\n",
      "\n",
      "Seq is: ACCAGTTCATAAAT\n",
      "0 ACCAGTTCATAAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCATGGCATGTTA\n",
      "0 ACCATGGCATGTTA motifs detected in file\n",
      "\n",
      "Seq is: TAAGTACCACCGAC\n",
      "0 TAAGTACCACCGAC motifs detected in file\n",
      "\n",
      "Seq is: CGTCATTACATGCT\n",
      "0 CGTCATTACATGCT motifs detected in file\n",
      "\n",
      "Seq is: TTATGTATCGCCAA\n",
      "0 TTATGTATCGCCAA motifs detected in file\n",
      "\n",
      "Seq is: CAACTACCCACTCA\n",
      "0 CAACTACCCACTCA motifs detected in file\n",
      "\n",
      "Seq is: GTTATGAACCCACC\n",
      "0 GTTATGAACCCACC motifs detected in file\n",
      "\n",
      "Seq is: ATGGGTTTAGACGA\n",
      "0 ATGGGTTTAGACGA motifs detected in file\n",
      "\n",
      "Seq is: AGGGGTACTTTCTT\n",
      "0 AGGGGTACTTTCTT motifs detected in file\n",
      "\n",
      "Seq is: GTCTCTTATCAAAT\n",
      "0 GTCTCTTATCAAAT motifs detected in file\n",
      "\n",
      "Seq is: TTAATCTTAGAATC\n",
      "0 TTAATCTTAGAATC motifs detected in file\n",
      "\n",
      "Seq is: GAAACCCACGATTA\n",
      "0 GAAACCCACGATTA motifs detected in file\n",
      "\n",
      "Seq is: CCACCTCAATTTCT\n",
      "0 CCACCTCAATTTCT motifs detected in file\n",
      "\n",
      "Seq is: GAAAAAGAGACTAA\n",
      "0 GAAAAAGAGACTAA motifs detected in file\n",
      "\n",
      "Seq is: ACAGGAACCTTTTT\n",
      "0 ACAGGAACCTTTTT motifs detected in file\n",
      "\n",
      "Seq is: CCAAAAGTAGAGGC\n",
      "0 CCAAAAGTAGAGGC motifs detected in file\n",
      "\n",
      "Seq is: CAAATGTTCTGACC\n",
      "0 CAAATGTTCTGACC motifs detected in file\n",
      "\n",
      "Seq is: ACATAATCAAATAT\n",
      "0 ACATAATCAAATAT motifs detected in file\n",
      "\n",
      "Seq is: GATGTTCCTGTCCG\n",
      "0 GATGTTCCTGTCCG motifs detected in file\n",
      "\n",
      "Seq is: GGTAAACTCATAAA\n",
      "0 GGTAAACTCATAAA motifs detected in file\n",
      "\n",
      "Seq is: ACAAAAGTTAATCC\n",
      "0 ACAAAAGTTAATCC motifs detected in file\n",
      "\n",
      "Seq is: CTCTATCAACCATA\n",
      "0 CTCTATCAACCATA motifs detected in file\n",
      "\n",
      "Seq is: ATCCTAATCCTAAC\n",
      "0 ATCCTAATCCTAAC motifs detected in file\n",
      "\n",
      "Seq is: AACACTTCATATCA\n",
      "0 AACACTTCATATCA motifs detected in file\n",
      "\n",
      "Seq is: TGCCTACGATGAAC\n",
      "0 TGCCTACGATGAAC motifs detected in file\n",
      "\n",
      "Seq is: AGGTTACTACCATT\n",
      "0 AGGTTACTACCATT motifs detected in file\n",
      "\n",
      "Seq is: TTCCCATCGAATGA\n",
      "0 TTCCCATCGAATGA motifs detected in file\n",
      "\n",
      "Seq is: CCAACAGGAGGCTA\n",
      "0 CCAACAGGAGGCTA motifs detected in file\n",
      "\n",
      "Seq is: AGTCCAATCTTACT\n",
      "0 AGTCCAATCTTACT motifs detected in file\n",
      "\n",
      "Seq is: CCTCCAGACGCCGA\n",
      "0 CCTCCAGACGCCGA motifs detected in file\n",
      "\n",
      "Seq is: TCCTCAGTTATTTC\n",
      "0 TCCTCAGTTATTTC motifs detected in file\n",
      "\n",
      "Seq is: ACTAACCGAATCAC\n",
      "0 ACTAACCGAATCAC motifs detected in file\n",
      "\n",
      "Seq is: CCGCTTTATAATAC\n",
      "0 CCGCTTTATAATAC motifs detected in file\n",
      "\n",
      "Seq is: GAAACAACAAACCT\n",
      "0 GAAACAACAAACCT motifs detected in file\n",
      "\n",
      "Seq is: ATATACCTCCTACC\n",
      "0 ATATACCTCCTACC motifs detected in file\n",
      "\n",
      "Seq is: CCTAATAACGATCC\n",
      "0 CCTAATAACGATCC motifs detected in file\n",
      "\n",
      "Complement found from 5472 to 5486\n",
      "Seq is: TACTCCTACCTATC\n",
      "1 TACTCCTACCTATC motifs detected in file\n",
      "\n",
      "Seq is: ATTATCCCGTTCCT\n",
      "0 ATTATCCCGTTCCT motifs detected in file\n",
      "\n",
      "Seq is: GCGGAGGATCAAAC\n",
      "0 GCGGAGGATCAAAC motifs detected in file\n",
      "\n",
      "Seq is: AATCCCTGCCTAGC\n",
      "0 AATCCCTGCCTAGC motifs detected in file\n",
      "\n",
      "Seq is: CTCTTAACACATCC\n",
      "0 CTCTTAACACATCC motifs detected in file\n",
      "\n",
      "Seq is: GCTTATCCTTTATA\n",
      "0 GCTTATCCTTTATA motifs detected in file\n",
      "\n",
      "Seq is: GTAAGCCCGAACTA\n",
      "0 GTAAGCCCGAACTA motifs detected in file\n",
      "\n",
      "Complement found from 15534 to 15548\n",
      "Seq is: CACCCCTCCCCACA\n",
      "1 CACCCCTCCCCACA motifs detected in file\n",
      "\n",
      "Seq is: AATTCCCCAACCGA\n",
      "0 AATTCCCCAACCGA motifs detected in file\n",
      "\n",
      "Seq is: TCCCATATTAACAG\n",
      "0 TCCCATATTAACAG motifs detected in file\n",
      "\n",
      "Seq is: ACCCAGCGGATCCT\n",
      "0 ACCCAGCGGATCCT motifs detected in file\n",
      "\n",
      "Seq is: CCAGACCACTCTTA\n",
      "0 CCAGACCACTCTTA motifs detected in file\n",
      "\n",
      "Seq is: TCACAATTACAGTA\n",
      "0 TCACAATTACAGTA motifs detected in file\n",
      "\n",
      "Seq is: ATTCCTCTCTTCCT\n",
      "0 ATTCCTCTCTTCCT motifs detected in file\n",
      "\n",
      "Seq is: TCTCTTCATTCGGC\n",
      "0 TCTCTTCATTCGGC motifs detected in file\n",
      "\n",
      "Seq is: TCCCGCAGAAACTA\n",
      "0 TCCCGCAGAAACTA motifs detected in file\n",
      "\n",
      "Seq is: ACACATCATTCCCA\n",
      "0 ACACATCATTCCCA motifs detected in file\n",
      "\n",
      "Seq is: CCTTCCACTAAAAT\n",
      "0 CCTTCCACTAAAAT motifs detected in file\n",
      "\n",
      "Seq is: AGCCTTACCCTCCA\n",
      "0 AGCCTTACCCTCCA motifs detected in file\n",
      "\n",
      "Seq is: CTAAGGATCCCCCA\n",
      "0 CTAAGGATCCCCCA motifs detected in file\n",
      "\n",
      "Seq is: ACAAACTAGGGCAA\n",
      "0 ACAAACTAGGGCAA motifs detected in file\n",
      "\n",
      "Seq is: AGCACGTTCTTATC\n",
      "0 AGCACGTTCTTATC motifs detected in file\n",
      "\n",
      "Seq is: CTCCACCTCACAAC\n",
      "0 CTCCACCTCACAAC motifs detected in file\n",
      "\n",
      "Seq is: GATCCCGACGTTAT\n",
      "0 GATCCCGACGTTAT motifs detected in file\n",
      "\n",
      "Seq is: TACTTCCCGTTCTA\n",
      "0 TACTTCCCGTTCTA motifs detected in file\n",
      "\n",
      "Seq is: CTTCACTTTCCATT\n",
      "0 CTTCACTTTCCATT motifs detected in file\n",
      "\n",
      "Seq is: TCTTAGCACACTCC\n",
      "0 TCTTAGCACACTCC motifs detected in file\n",
      "\n",
      "Seq is: CACCCTGACAGATG\n",
      "0 CACCCTGACAGATG motifs detected in file\n",
      "\n",
      "Seq is: ACTCATCGGAGGAG\n",
      "0 ACTCATCGGAGGAG motifs detected in file\n",
      "\n",
      "Seq is: TCTAAGTAACTTGA\n",
      "0 TCTAAGTAACTTGA motifs detected in file\n",
      "\n",
      "Seq is: TCCAGACAGGGTTA\n",
      "0 TCCAGACAGGGTTA motifs detected in file\n",
      "\n",
      "Seq is: CATACCCTACCGCC\n",
      "0 CATACCCTACCGCC motifs detected in file\n",
      "\n",
      "Seq is: TATCATTCAAACAT\n",
      "0 TATCATTCAAACAT motifs detected in file\n",
      "\n",
      "Seq is: TAATGACACCGGGG\n",
      "0 TAATGACACCGGGG motifs detected in file\n",
      "\n",
      "Seq is: AGTCTTACTATAAA\n",
      "0 AGTCTTACTATAAA motifs detected in file\n",
      "\n",
      "Seq is: CCGGAGTGCCCTCC\n",
      "0 CCGGAGTGCCCTCC motifs detected in file\n",
      "\n",
      "Seq is: TGTATCGGATACTT\n",
      "0 TGTATCGGATACTT motifs detected in file\n",
      "\n",
      "Seq is: CCGACAACGATATC\n",
      "0 CCGACAACGATATC motifs detected in file\n",
      "\n",
      "Seq is: AACGTTCGTCCTCC\n",
      "0 AACGTTCGTCCTCC motifs detected in file\n",
      "\n",
      "Seq is: TATTACGGCTACAA\n",
      "0 TATTACGGCTACAA motifs detected in file\n",
      "\n",
      "Seq is: AGTCCAAAGACTCA\n",
      "0 AGTCCAAAGACTCA motifs detected in file\n",
      "\n",
      "Seq is: TCTCTTTACTAGGC\n",
      "0 TCTCTTTACTAGGC motifs detected in file\n",
      "\n",
      "Seq is: ATTATATCCGGAGC\n",
      "0 ATTATATCCGGAGC motifs detected in file\n",
      "\n",
      "Seq is: GGGCTACACATCCT\n",
      "0 GGGCTACACATCCT motifs detected in file\n",
      "\n",
      "Seq is: TCTCCGTCTATTTC\n",
      "0 TCTCCGTCTATTTC motifs detected in file\n",
      "\n",
      "Seq is: TTATAACTCCGCGG\n",
      "0 TTATAACTCCGCGG motifs detected in file\n",
      "\n",
      "Seq is: TAACCGCACTTCCA\n",
      "0 TAACCGCACTTCCA motifs detected in file\n",
      "\n",
      "Seq is: TCGCCTACTAAGTC\n",
      "0 TCGCCTACTAAGTC motifs detected in file\n",
      "\n",
      "Seq is: GGTATTAAATGCAG\n",
      "0 GGTATTAAATGCAG motifs detected in file\n",
      "\n",
      "Seq is: AGCTCACTACACCC\n",
      "0 AGCTCACTACACCC motifs detected in file\n",
      "\n",
      "Seq is: GCTAACTACTTTTC\n",
      "0 GCTAACTACTTTTC motifs detected in file\n",
      "\n",
      "Seq is: CGCCAACTCCGCAG\n",
      "0 CGCCAACTCCGCAG motifs detected in file\n",
      "\n",
      "Seq is: ACCACTCATCACGT\n",
      "0 ACCACTCATCACGT motifs detected in file\n",
      "\n",
      "Seq is: ACCGATCCTTATCA\n",
      "0 ACCGATCCTTATCA motifs detected in file\n",
      "\n",
      "Seq is: GGACACCACTAAAC\n",
      "0 GGACACCACTAAAC motifs detected in file\n",
      "\n",
      "Seq is: CTCCTAGTCCGTCC\n",
      "0 CTCCTAGTCCGTCC motifs detected in file\n",
      "\n",
      "Seq is: GCGGTTCCTCACTC\n",
      "0 GCGGTTCCTCACTC motifs detected in file\n",
      "\n",
      "Seq is: GGCTTCAAAGTAGT\n",
      "0 GGCTTCAAAGTAGT motifs detected in file\n",
      "\n",
      "Seq is: ACGCCTCTACAACC\n",
      "0 ACGCCTCTACAACC motifs detected in file\n",
      "\n",
      "Seq is: TACCCCACCCCTCC\n",
      "0 TACCCCACCCCTCC motifs detected in file\n",
      "\n",
      "Seq is: AGCTACTTACTCAC\n",
      "0 AGCTACTTACTCAC motifs detected in file\n",
      "\n",
      "Seq is: CAATTAATTAAAAT\n",
      "0 CAATTAATTAAAAT motifs detected in file\n",
      "\n",
      "Seq is: AATCCCCCAATCAA\n",
      "0 AATCCCCCAATCAA motifs detected in file\n",
      "\n",
      "Seq is: AACGCATAACCCCA\n",
      "0 AACGCATAACCCCA motifs detected in file\n",
      "\n",
      "Seq is: GTAACCACAAGAAC\n",
      "0 GTAACCACAAGAAC motifs detected in file\n",
      "\n",
      "Seq is: ATCAACTTTATGTT\n",
      "0 ATCAACTTTATGTT motifs detected in file\n",
      "\n",
      "Seq is: GCTACCAAAAAGTA\n",
      "0 GCTACCAAAAAGTA motifs detected in file\n",
      "\n",
      "Seq is: TAGTAACCAGCACC\n",
      "0 TAGTAACCAGCACC motifs detected in file\n",
      "\n",
      "Seq is: AACATCAGGCACGC\n",
      "0 AACATCAGGCACGC motifs detected in file\n",
      "\n",
      "Seq is: TCTTATTACTACAT\n",
      "0 TCTTATTACTACAT motifs detected in file\n",
      "\n",
      "Seq is: ACGAAACAAAGACA\n",
      "0 ACGAAACAAAGACA motifs detected in file\n",
      "\n",
      "Seq is: ACTCACACCCAAAT\n",
      "0 ACTCACACCCAAAT motifs detected in file\n",
      "\n",
      "Seq is: CATTACCCCAAACA\n",
      "0 CATTACCCCAAACA motifs detected in file\n",
      "\n",
      "Seq is: CCCCAAAAGAAGAT\n",
      "0 CCCCAAAAGAAGAT motifs detected in file\n",
      "\n",
      "Seq is: TCGGAAGAGGATAA\n",
      "0 TCGGAAGAGGATAA motifs detected in file\n",
      "\n",
      "Seq is: ATACCCCCAAATCA\n",
      "0 ATACCCCCAAATCA motifs detected in file\n",
      "\n",
      "Seq is: TAACTAACAATCGC\n",
      "0 TAACTAACAATCGC motifs detected in file\n",
      "\n",
      "Seq is: CACACCAGCCCACA\n",
      "0 CACACCAGCCCACA motifs detected in file\n",
      "\n",
      "Seq is: CAATAATAAGACTT\n",
      "0 CAATAATAAGACTT motifs detected in file\n",
      "\n",
      "Seq is: AAAACCCCCTCCAA\n",
      "0 AAAACCCCCTCCAA motifs detected in file\n",
      "\n",
      "Seq is: TATACCCAAATTAT\n",
      "0 TATACCCAAATTAT motifs detected in file\n",
      "\n",
      "Seq is: CAAAAAAATTAAAT\n",
      "0 CAAAAAAATTAAAT motifs detected in file\n",
      "\n",
      "Seq is: AAATCCCCCTTACT\n",
      "0 AAATCCCCCTTACT motifs detected in file\n",
      "\n",
      "Seq is: ACCAACAGAAACCT\n",
      "0 ACCAACAGAAACCT motifs detected in file\n",
      "\n",
      "Seq is: ATATGATGTCGCTA\n",
      "0 ATATGATGTCGCTA motifs detected in file\n",
      "\n",
      "Seq is: CCGATAACTCCTCA\n",
      "0 CCGATAACTCCTCA motifs detected in file\n",
      "\n",
      "Seq is: TAGGACTCCGTACC\n",
      "0 TAGGACTCCGTACC motifs detected in file\n",
      "\n",
      "Seq is: CCCAGTCCCCAACT\n",
      "0 CCCAGTCCCCAACT motifs detected in file\n",
      "\n",
      "Seq is: CCAGAACCACTCAC\n",
      "0 CCAGAACCACTCAC motifs detected in file\n",
      "\n",
      "Seq is: AAAATCACCCCAAT\n",
      "0 AAAATCACCCCAAT motifs detected in file\n",
      "\n",
      "Seq is: CGCTACCTCCATCC\n",
      "0 CGCTACCTCCATCC motifs detected in file\n",
      "\n",
      "Seq is: TAACCACGACACCC\n",
      "0 TAACCACGACACCC motifs detected in file\n",
      "\n",
      "Seq is: ACTTTCTCATACTA\n",
      "0 ACTTTCTCATACTA motifs detected in file\n",
      "\n",
      "Seq is: CCCCACCACCAACA\n",
      "0 CCCCACCACCAACA motifs detected in file\n",
      "\n",
      "Seq is: CCATTTGAAATTAT\n",
      "0 CCATTTGAAATTAT motifs detected in file\n",
      "\n",
      "Seq is: CACATCCTTCGACT\n",
      "0 CACATCCTTCGACT motifs detected in file\n",
      "\n",
      "Seq is: TATTAAATACTTCC\n",
      "0 TATTAAATACTTCC motifs detected in file\n",
      "\n",
      "Seq is: TCTCCCCAGTCCCA\n",
      "0 TCTCCCCAGTCCCA motifs detected in file\n",
      "\n",
      "Seq is: ACTAAGCCCTCCTA\n",
      "0 ACTAAGCCCTCCTA motifs detected in file\n",
      "\n",
      "Seq is: GGATAACCACGCCC\n",
      "0 GGATAACCACGCCC motifs detected in file\n",
      "\n",
      "Seq is: CCGAAACATACTAA\n",
      "0 CCGAAACATACTAA motifs detected in file\n",
      "\n",
      "Seq is: TACCCGCAACTAAT\n",
      "0 TACCCGCAACTAAT motifs detected in file\n",
      "\n",
      "Seq is: CATCATCAATGACC\n",
      "0 CATCATCAATGACC motifs detected in file\n",
      "\n",
      "Seq is: AACTTGTAACAAAC\n",
      "0 AACTTGTAACAAAC motifs detected in file\n",
      "\n",
      "Seq is: AACCACATATATAA\n",
      "0 AACCACATATATAA motifs detected in file\n",
      "\n",
      "Seq is: CATTAACTCTAACG\n",
      "0 CATTAACTCTAACG motifs detected in file\n",
      "\n",
      "Seq is: AGCCCCCTTATCCA\n",
      "0 AGCCCCCTTATCCA motifs detected in file\n",
      "\n",
      "Seq is: ATACACTAATCCTC\n",
      "0 ATACACTAATCCTC motifs detected in file\n",
      "\n",
      "Seq is: ATCCCAATCCTACT\n",
      "0 ATCCCAATCCTACT motifs detected in file\n",
      "\n",
      "Seq is: CACCCTTCTTCTTT\n",
      "0 CACCCTTCTTCTTT motifs detected in file\n",
      "\n",
      "Seq is: CTCTCCTTCATTTC\n",
      "0 CTCTCCTTCATTTC motifs detected in file\n",
      "\n",
      "Seq is: AAATTAATACGGAA\n",
      "0 AAATTAATACGGAA motifs detected in file\n",
      "\n",
      "Seq is: AAACCCAACTCCAC\n",
      "0 AAACCCAACTCCAC motifs detected in file\n",
      "\n",
      "Seq is: TACTACCTCCACCT\n",
      "0 TACTACCTCCACCT motifs detected in file\n",
      "\n",
      "Seq is: CTAAACCACGACAC\n",
      "0 CTAAACCACGACAC motifs detected in file\n",
      "\n",
      "Seq is: TTTAACAAAATCCA\n",
      "0 TTTAACAAAATCCA motifs detected in file\n",
      "\n",
      "Seq is: TTATCGAAAAGATC\n",
      "0 TTATCGAAAAGATC motifs detected in file\n",
      "\n",
      "Seq is: AGTCCAATCCAGAT\n",
      "0 AGTCCAATCCAGAT motifs detected in file\n",
      "\n",
      "Seq is: CCTCCTCATCCCCG\n",
      "0 CCTCCTCATCCCCG motifs detected in file\n",
      "\n",
      "Seq is: TCCAAAACCGAGCA\n",
      "0 TCCAAAACCGAGCA motifs detected in file\n",
      "\n",
      "Seq is: TTCTTCCGGATCTA\n",
      "0 TTCTTCCGGATCTA motifs detected in file\n",
      "\n",
      "Seq is: TCCCCTAACACGCC\n",
      "0 TCCCCTAACACGCC motifs detected in file\n",
      "\n",
      "Seq is: ACACACTACGATCC\n",
      "0 ACACACTACGATCC motifs detected in file\n",
      "\n",
      "Seq is: CATCTTAGGCTCAT\n",
      "0 CATCTTAGGCTCAT motifs detected in file\n",
      "\n",
      "Seq is: ACAACCTCTTTATT\n",
      "0 ACAACCTCTTTATT motifs detected in file\n",
      "\n",
      "Seq is: TTACACGTATCACC\n",
      "0 TTACACGTATCACC motifs detected in file\n",
      "\n",
      "Seq is: CCTAAAATAAAATT\n",
      "0 CCTAAAATAAAATT motifs detected in file\n",
      "\n",
      "Seq is: CAAACAACCAATCC\n",
      "0 CAAACAACCAATCC motifs detected in file\n",
      "\n",
      "Seq is: ATCAACTCCAGATC\n",
      "0 ATCAACTCCAGATC motifs detected in file\n",
      "\n",
      "Seq is: CCGACAATCTTCAG\n",
      "0 CCGACAATCTTCAG motifs detected in file\n",
      "\n",
      "Seq is: GATCCTTTCACTGT\n",
      "0 GATCCTTTCACTGT motifs detected in file\n",
      "\n",
      "Seq is: CGCTCCCGACACTC\n",
      "0 CGCTCCCGACACTC motifs detected in file\n",
      "\n",
      "Seq is: AAAATCCATCTCCC\n",
      "0 AAAATCCATCTCCC motifs detected in file\n",
      "\n",
      "Seq is: CCTAACAACAAACC\n",
      "0 CCTAACAACAAACC motifs detected in file\n",
      "\n",
      "Seq is: TTCCCCCTACGCCC\n",
      "0 TTCCCCCTACGCCC motifs detected in file\n",
      "\n",
      "Seq is: CCTTTACAACAATC\n",
      "0 CCTTTACAACAATC motifs detected in file\n",
      "\n",
      "Seq is: ATTACTCTTTAGGA\n",
      "0 ATTACTCTTTAGGA motifs detected in file\n",
      "\n",
      "Seq is: CGCTTATCCGAAGG\n",
      "0 CGCTTATCCGAAGG motifs detected in file\n",
      "\n",
      "Seq is: CCGACGGTCCGCAA\n",
      "0 CCGACGGTCCGCAA motifs detected in file\n",
      "\n",
      "Seq is: ATTACCCCAAATCA\n",
      "0 ATTACCCCAAATCA motifs detected in file\n",
      "\n",
      "Seq is: TCCCACCCCAATAA\n",
      "0 TCCCACCCCAATAA motifs detected in file\n",
      "\n",
      "Seq is: AAGCAATTACAATC\n",
      "0 AAGCAATTACAATC motifs detected in file\n",
      "\n",
      "Seq is: ATTCCCACCCCTTC\n",
      "0 ATTCCCACCCCTTC motifs detected in file\n",
      "\n",
      "Seq is: GCTCCAACTGGACA\n",
      "0 GCTCCAACTGGACA motifs detected in file\n",
      "\n",
      "Seq is: ATCCCACTCTTCTT\n",
      "0 ATCCCACTCTTCTT motifs detected in file\n",
      "\n",
      "Seq is: AATAAGCTCACGAT\n",
      "0 AATAAGCTCACGAT motifs detected in file\n",
      "\n",
      "Seq is: ATCCGCGAACAGTC\n",
      "0 ATCCGCGAACAGTC motifs detected in file\n",
      "\n",
      "Seq is: CCTCCATCGCTACT\n",
      "0 CCTCCATCGCTACT motifs detected in file\n",
      "\n",
      "Seq is: CTCATTATCTATCC\n",
      "0 CTCATTATCTATCC motifs detected in file\n",
      "\n",
      "Seq is: CGAGTCCGCAAACA\n",
      "0 CGAGTCCGCAAACA motifs detected in file\n",
      "\n",
      "Seq is: CATACTATACAAAC\n",
      "0 CATACTATACAAAC motifs detected in file\n",
      "\n",
      "Seq is: GCCAAAGCTACTAC\n",
      "0 GCCAAAGCTACTAC motifs detected in file\n",
      "\n",
      "Seq is: ACCAGAAACCTCAT\n",
      "0 ACCAGAAACCTCAT motifs detected in file\n",
      "\n",
      "Seq is: CTTTGGACACTCCT\n",
      "0 CTTTGGACACTCCT motifs detected in file\n",
      "\n",
      "Seq is: TTCCATAAGGACGA\n",
      "0 TTCCATAAGGACGA motifs detected in file\n",
      "\n",
      "Seq is: TTACGATCCGACGG\n",
      "0 TTACGATCCGACGG motifs detected in file\n",
      "\n",
      "Seq is: TTACCACTCCCTCC\n",
      "0 TTACCACTCCCTCC motifs detected in file\n",
      "\n",
      "Seq is: AACTTCACTCTCCA\n",
      "0 AACTTCACTCTCCA motifs detected in file\n",
      "\n",
      "Seq is: TACCAAAACTCATC\n",
      "0 TACCAAAACTCATC motifs detected in file\n",
      "\n",
      "Seq is: AGGAGGATAAAAAG\n",
      "0 AGGAGGATAAAAAG motifs detected in file\n",
      "\n",
      "Seq is: CTTATAGAACAAGT\n",
      "0 CTTATAGAACAAGT motifs detected in file\n",
      "\n",
      "Seq is: AACAATTCCAACAC\n",
      "0 AACAATTCCAACAC motifs detected in file\n",
      "\n",
      "Seq is: CTACTACCTGGGCC\n",
      "0 CTACTACCTGGGCC motifs detected in file\n",
      "\n",
      "Seq is: TCGTGTATTTATCA\n",
      "0 TCGTGTATTTATCA motifs detected in file\n",
      "\n",
      "Seq is: TACCGAAACTTCTT\n",
      "0 TACCGAAACTTCTT motifs detected in file\n",
      "\n",
      "Seq is: CCGCACCCATGTCT\n",
      "0 CCGCACCCATGTCT motifs detected in file\n",
      "\n",
      "Seq is: ACACGTCCTTACGA\n",
      "0 ACACGTCCTTACGA motifs detected in file\n",
      "\n",
      "Seq is: TCCACACCAACCAA\n",
      "0 TCCACACCAACCAA motifs detected in file\n",
      "\n",
      "Seq is: CTACGGCTAACATT\n",
      "0 CTACGGCTAACATT motifs detected in file\n",
      "\n",
      "Seq is: GATAATACTCAGGA\n",
      "0 GATAATACTCAGGA motifs detected in file\n",
      "\n",
      "Seq is: TCAACTGAACTTCA\n",
      "0 TCAACTGAACTTCA motifs detected in file\n",
      "\n",
      "Seq is: CCTCTTCCGATGCT\n",
      "0 CCTCTTCCGATGCT motifs detected in file\n",
      "\n",
      "Seq is: AAAAAAACTACAGT\n",
      "0 AAAAAAACTACAGT motifs detected in file\n",
      "\n",
      "Seq is: AAAACACATTCCCG\n",
      "0 AAAACACATTCCCG motifs detected in file\n",
      "\n",
      "Seq is: CGTCTGACGACGCT\n",
      "0 CGTCTGACGACGCT motifs detected in file\n",
      "\n",
      "Seq is: TGTCTCACCACTAT\n",
      "0 TGTCTCACCACTAT motifs detected in file\n",
      "\n",
      "Seq is: CGCGGATTCGTATC\n",
      "0 CGCGGATTCGTATC motifs detected in file\n",
      "\n",
      "Seq is: ACAATCTCAAACCT\n",
      "0 ACAATCTCAAACCT motifs detected in file\n",
      "\n",
      "Seq is: AATCACCCGATAAA\n",
      "0 AATCACCCGATAAA motifs detected in file\n",
      "\n",
      "Seq is: AGACGATCCCCCAC\n",
      "0 AGACGATCCCCCAC motifs detected in file\n",
      "\n",
      "Seq is: CTTCGCCTACTCAT\n",
      "0 CTTCGCCTACTCAT motifs detected in file\n",
      "\n",
      "Seq is: TCTTCTAAGGACGA\n",
      "0 TCTTCTAAGGACGA motifs detected in file\n",
      "\n",
      "Seq is: TGTTGATATCACGA\n",
      "0 TGTTGATATCACGA motifs detected in file\n",
      "\n",
      "Seq is: ACTCACCTCATCCC\n",
      "0 ACTCACCTCATCCC motifs detected in file\n",
      "\n",
      "Seq is: GACTCTGACCCCAC\n",
      "0 GACTCTGACCCCAC motifs detected in file\n",
      "\n",
      "Seq is: CCCGGAAGATACCG\n",
      "0 CCCGGAAGATACCG motifs detected in file\n",
      "\n",
      "Seq is: ACTCCCCTCAGTCC\n",
      "0 ACTCCCCTCAGTCC motifs detected in file\n",
      "\n",
      "Seq is: CCACCTCTGGATTA\n",
      "0 CCACCTCTGGATTA motifs detected in file\n",
      "\n",
      "Seq is: ACCCGACTAAACGG\n",
      "0 ACCCGACTAAACGG motifs detected in file\n",
      "\n",
      "Seq is: ACGACGACGATCCT\n",
      "0 ACGACGACGATCCT motifs detected in file\n",
      "\n",
      "Seq is: CCTCCGGATCATCA\n",
      "0 CCTCCGGATCATCA motifs detected in file\n",
      "\n",
      "Seq is: CCCCACTCCGAACC\n",
      "0 CCCCACTCCGAACC motifs detected in file\n",
      "\n",
      "Seq is: TAATCGCAAATCTT\n",
      "0 TAATCGCAAATCTT motifs detected in file\n",
      "\n",
      "Seq is: CCCGATAAACAACA\n",
      "0 CCCGATAAACAACA motifs detected in file\n",
      "\n",
      "Seq is: CCCAGAGTACTCAA\n",
      "0 CCCAGAGTACTCAA motifs detected in file\n",
      "\n",
      "Seq is: CCTCACATCCTATT\n",
      "0 CCTCACATCCTATT motifs detected in file\n",
      "\n",
      "Seq is: TAGTACGATTCCGC\n",
      "0 TAGTACGATTCCGC motifs detected in file\n",
      "\n",
      "Seq is: TCCTACTTTGGCTA\n",
      "0 TCCTACTTTGGCTA motifs detected in file\n",
      "\n",
      "Seq is: TAGCGGCTATGCCA\n",
      "0 TAGCGGCTATGCCA motifs detected in file\n",
      "\n",
      "Seq is: ACATATCCTAACGA\n",
      "0 ACATATCCTAACGA motifs detected in file\n",
      "\n",
      "Seq is: ACTTACCGACGACA\n",
      "0 ACTTACCGACGACA motifs detected in file\n",
      "\n",
      "Seq is: CAACCGTAGACGAG\n",
      "0 CAACCGTAGACGAG motifs detected in file\n",
      "\n",
      "Seq is: CCCGCATAGTAGTT\n",
      "0 CCCGCATAGTAGTT motifs detected in file\n",
      "\n",
      "Seq is: GACTACTCGTTCTT\n",
      "0 GACTACTCGTTCTT motifs detected in file\n",
      "\n",
      "Seq is: CCTATATTAAGGAT\n",
      "0 CCTATATTAAGGAT motifs detected in file\n",
      "\n",
      "Seq is: GCGGGAGAGTCGGC\n",
      "0 GCGGGAGAGTCGGC motifs detected in file\n",
      "\n",
      "Seq is: TACTTGTCAACCTT\n",
      "0 TACTTGTCAACCTT motifs detected in file\n",
      "\n",
      "Seq is: ATCCAACAATCGCC\n",
      "0 ATCCAACAATCGCC motifs detected in file\n",
      "\n",
      "Seq is: ATTGATTCTAATCA\n",
      "0 ATTGATTCTAATCA motifs detected in file\n",
      "\n",
      "Seq is: TACCATTAATCCTT\n",
      "0 TACCATTAATCCTT motifs detected in file\n",
      "\n",
      "Seq is: CTACTCATCTATAA\n",
      "0 CTACTCATCTATAA motifs detected in file\n",
      "\n",
      "Seq is: ACTTCTTGACTAAT\n",
      "0 ACTTCTTGACTAAT motifs detected in file\n",
      "\n",
      "Seq is: TACAAACCCAGACT\n",
      "0 TACAAACCCAGACT motifs detected in file\n",
      "\n",
      "Seq is: CAAATATATAGTGT\n",
      "0 CAAATATATAGTGT motifs detected in file\n",
      "\n",
      "Seq is: CACTCTTAAGATAC\n",
      "0 CACTCTTAAGATAC motifs detected in file\n",
      "\n",
      "Seq is: TACCTGGTACATTG\n",
      "0 TACCTGGTACATTG motifs detected in file\n",
      "\n",
      "Seq is: CTTGTTACGATGTC\n",
      "0 CTTGTTACGATGTC motifs detected in file\n",
      "\n",
      "Seq is: CCTACTTATAATAC\n",
      "0 CCTACTTATAATAC motifs detected in file\n",
      "\n",
      "Seq is: CTCTTCATCAGATC\n",
      "0 CTCTTCATCAGATC motifs detected in file\n",
      "\n",
      "Seq is: AAACTTCGAATCCC\n",
      "0 AAACTTCGAATCCC motifs detected in file\n",
      "\n",
      "Seq is: TCTCGACCCAACAA\n",
      "0 TCTCGACCCAACAA motifs detected in file\n",
      "\n",
      "Seq is: ACCCAACACCGAGT\n",
      "0 ACCCAACACCGAGT motifs detected in file\n",
      "\n",
      "Seq is: CACAGTCAAGCTCT\n",
      "0 CACAGTCAAGCTCT motifs detected in file\n",
      "\n",
      "Seq is: ATTATTGAAGAACC\n",
      "0 ATTATTGAAGAACC motifs detected in file\n",
      "\n",
      "Seq is: AGATCCGTGTACTT\n",
      "0 AGATCCGTGTACTT motifs detected in file\n",
      "\n",
      "Seq is: ATAACAACACCCCT\n",
      "0 ATAACAACACCCCT motifs detected in file\n",
      "\n",
      "Seq is: TCTCTGACTATTAT\n",
      "0 TCTCTGACTATTAT motifs detected in file\n",
      "\n",
      "Seq is: TTCCACCTACGCTG\n",
      "0 TTCCACCTACGCTG motifs detected in file\n",
      "\n",
      "Seq is: TTACCTAAAATGTA\n",
      "0 TTACCTAAAATGTA motifs detected in file\n",
      "\n",
      "Complement found from 8402 to 8416\n",
      "Seq is: TTACCCCCATACTC\n",
      "1 TTACCCCCATACTC motifs detected in file\n",
      "\n",
      "Seq is: AAAAAAAACAATCC\n",
      "0 AAAAAAAACAATCC motifs detected in file\n",
      "\n",
      "Seq is: CAATTGCTCCCACC\n",
      "0 CAATTGCTCCCACC motifs detected in file\n",
      "\n",
      "Seq is: ATTCCTACCCCCCT\n",
      "0 ATTCCTACCCCCCT motifs detected in file\n",
      "\n",
      "Seq is: TAATCCCTTCAGTC\n",
      "0 TAATCCCTTCAGTC motifs detected in file\n",
      "\n",
      "Seq is: CCAATCCCACCAAT\n",
      "0 CCAATCCCACCAAT motifs detected in file\n",
      "\n",
      "Seq is: ATCATCACACGTAC\n",
      "0 ATCATCACACGTAC motifs detected in file\n",
      "\n",
      "Seq is: CAATAATGAAAATA\n",
      "0 CAATAATGAAAATA motifs detected in file\n",
      "\n",
      "Seq is: AACCTCAACGTGGT\n",
      "0 AACCTCAACGTGGT motifs detected in file\n",
      "\n",
      "Seq is: TTTAAAAACCCCGG\n",
      "0 TTTAAAAACCCCGG motifs detected in file\n",
      "\n",
      "Seq is: ATTCTGGTTACCTA\n",
      "0 ATTCTGGTTACCTA motifs detected in file\n",
      "\n",
      "Seq is: TCGACAATAGGAAA\n",
      "0 TCGACAATAGGAAA motifs detected in file\n",
      "\n",
      "Seq is: TTTTCAACTCTTTC\n",
      "0 TTTTCAACTCTTTC motifs detected in file\n",
      "\n",
      "Seq is: GGTACAACAATCTG\n",
      "0 GGTACAACAATCTG motifs detected in file\n",
      "\n",
      "Seq is: TACCCCCGTACTCA\n",
      "0 TACCCCCGTACTCA motifs detected in file\n",
      "\n",
      "Seq is: ATCGTCAAGAACAC\n",
      "0 ATCGTCAAGAACAC motifs detected in file\n",
      "\n",
      "Seq is: TCGAAAGAGCCATT\n",
      "0 TCGAAAGAGCCATT motifs detected in file\n",
      "\n",
      "Seq is: TATTCCCCAGCATT\n",
      "0 TATTCCCCAGCATT motifs detected in file\n",
      "\n",
      "Seq is: CGGAGACAACAGTC\n",
      "0 CGGAGACAACAGTC motifs detected in file\n",
      "\n",
      "Seq is: TAAGTGTTAGACTA\n",
      "0 TAAGTGTTAGACTA motifs detected in file\n",
      "\n",
      "Seq is: CAAAACCAATTTGA\n",
      "0 CAAAACCAATTTGA motifs detected in file\n",
      "\n",
      "Seq is: TATAAATGTTCTCC\n",
      "0 TATAAATGTTCTCC motifs detected in file\n",
      "\n",
      "Seq is: TTTTGGGCCATTAC\n",
      "0 TTTTGGGCCATTAC motifs detected in file\n",
      "\n",
      "Seq is: TACAGCCCCAACTC\n",
      "0 TACAGCCCCAACTC motifs detected in file\n",
      "\n",
      "Seq is: CCTATCCTCCTCTT\n",
      "0 CCTATCCTCCTCTT motifs detected in file\n",
      "\n",
      "Seq is: ACCCCCTATCCACA\n",
      "0 ACCCCCTATCCACA motifs detected in file\n",
      "\n",
      "Seq is: TACTTGTACTCCCA\n",
      "0 TACTTGTACTCCCA motifs detected in file\n",
      "\n",
      "Seq is: CAAAAGAGCACACT\n",
      "0 CAAAAGAGCACACT motifs detected in file\n",
      "\n",
      "Seq is: TACTCCCAAAATAC\n",
      "0 TACTCCCAAAATAC motifs detected in file\n",
      "\n",
      "Seq is: AACAATTACACCAC\n",
      "0 AACAATTACACCAC motifs detected in file\n",
      "\n",
      "Seq is: CCACTCACTCGGGG\n",
      "0 CCACTCACTCGGGG motifs detected in file\n",
      "\n",
      "Seq is: TAACACAACACCAT\n",
      "0 TAACACAACACCAT motifs detected in file\n",
      "\n",
      "Seq is: TTATACATCTCCCT\n",
      "0 TTATACATCTCCCT motifs detected in file\n",
      "\n",
      "Seq is: CATATCCCGACACT\n",
      "0 CATATCCCGACACT motifs detected in file\n",
      "\n",
      "Seq is: GATCATACAACTCA\n",
      "0 GATCATACAACTCA motifs detected in file\n",
      "\n",
      "Seq is: GGACATTCATCCTC\n",
      "0 GGACATTCATCCTC motifs detected in file\n",
      "\n",
      "Seq is: TCACTATAAACTAG\n",
      "0 TCACTATAAACTAG motifs detected in file\n",
      "\n",
      "Seq is: TCCTCTTGCACCAA\n",
      "0 TCCTCTTGCACCAA motifs detected in file\n",
      "\n",
      "Seq is: TGATCGTGTCTCTC\n",
      "0 TGATCGTGTCTCTC motifs detected in file\n",
      "\n",
      "Seq is: AAGAGGGTCATCCA\n",
      "0 AAGAGGGTCATCCA motifs detected in file\n",
      "\n",
      "Seq is: ATTATCACCCCCCA\n",
      "0 ATTATCACCCCCCA motifs detected in file\n",
      "\n",
      "Seq is: TTCCGCTCCAATCG\n",
      "0 TTCCGCTCCAATCG motifs detected in file\n",
      "\n",
      "Seq is: CTCCGAACGATCTT\n",
      "0 CTCCGAACGATCTT motifs detected in file\n",
      "\n",
      "Seq is: CAGTAGTTTTTCGA\n",
      "0 CAGTAGTTTTTCGA motifs detected in file\n",
      "\n",
      "Seq is: TAATCACCCTCATC\n",
      "0 TAATCACCCTCATC motifs detected in file\n",
      "\n",
      "Seq is: TCAAACTTCAGGAA\n",
      "0 TCAAACTTCAGGAA motifs detected in file\n",
      "\n",
      "Seq is: CTCTCTCCTAATAC\n",
      "0 CTCTCTCCTAATAC motifs detected in file\n",
      "\n",
      "Seq is: TACGCTGACACTCA\n",
      "0 TACGCTGACACTCA motifs detected in file\n",
      "\n",
      "Seq is: CGCAAGCATCAAAC\n",
      "0 CGCAAGCATCAAAC motifs detected in file\n",
      "\n",
      "Seq is: TCAAACGATCCGTC\n",
      "0 TCAAACGATCCGTC motifs detected in file\n",
      "\n",
      "Seq is: TTATCATTACTCCT\n",
      "0 TTATCATTACTCCT motifs detected in file\n",
      "\n",
      "Seq is: ACATTCGGGCACCC\n",
      "0 ACATTCGGGCACCC motifs detected in file\n",
      "\n",
      "Seq is: GCTAATACTCTTAC\n",
      "0 GCTAATACTCTTAC motifs detected in file\n",
      "\n",
      "Seq is: TGACGCGGCCACTT\n",
      "0 TGACGCGGCCACTT motifs detected in file\n",
      "\n",
      "Seq is: CGAAGTCCCCCAAA\n",
      "0 CGAAGTCCCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: CCTACTCTTACCGA\n",
      "0 CCTACTCTTACCGA motifs detected in file\n",
      "\n",
      "Seq is: CAATGATGCTCCCG\n",
      "0 CAATGATGCTCCCG motifs detected in file\n",
      "\n",
      "Seq is: ATACACCGACTAAC\n",
      "0 ATACACCGACTAAC motifs detected in file\n",
      "\n",
      "Seq is: TTCTCATACGTTAC\n",
      "0 TTCTCATACGTTAC motifs detected in file\n",
      "\n",
      "Seq is: TCGCTAAAATCCAG\n",
      "0 TCGCTAAAATCCAG motifs detected in file\n",
      "\n",
      "Seq is: ACAAACAGCATCCG\n",
      "0 ACAAACAGCATCCG motifs detected in file\n",
      "\n",
      "Seq is: TCTACCTCGAACAA\n",
      "0 TCTACCTCGAACAA motifs detected in file\n",
      "\n",
      "Seq is: TATTAATACGGAGT\n",
      "0 TATTAATACGGAGT motifs detected in file\n",
      "\n",
      "Seq is: ATCCCTATCATGTT\n",
      "0 ATCCCTATCATGTT motifs detected in file\n",
      "\n",
      "Seq is: CCTTCCCCATCCGA\n",
      "0 CCTTCCCCATCCGA motifs detected in file\n",
      "\n",
      "Seq is: TACACAAAACAGTC\n",
      "0 TACACAAAACAGTC motifs detected in file\n",
      "\n",
      "Seq is: CCCCAACTCTTACT\n",
      "0 CCCCAACTCTTACT motifs detected in file\n",
      "\n",
      "Seq is: CACACTCCGCATAA\n",
      "0 CACACTCCGCATAA motifs detected in file\n",
      "\n",
      "Seq is: TATGGTATCGGCGG\n",
      "0 TATGGTATCGGCGG motifs detected in file\n",
      "\n",
      "Seq is: ATCAAAATTCTCAT\n",
      "0 ATCAAAATTCTCAT motifs detected in file\n",
      "\n",
      "Seq is: GACGCCGTTCATGA\n",
      "0 GACGCCGTTCATGA motifs detected in file\n",
      "\n",
      "Seq is: TAACTGGGTCGCTA\n",
      "0 TAACTGGGTCGCTA motifs detected in file\n",
      "\n",
      "Seq is: CCCCCGAAGCTGTA\n",
      "0 CCCCCGAAGCTGTA motifs detected in file\n",
      "\n",
      "Seq is: CCCGAAATCCCTCA\n",
      "0 CCCGAAATCCCTCA motifs detected in file\n",
      "\n",
      "Seq is: GTATTCACCTCAGG\n",
      "0 GTATTCACCTCAGG motifs detected in file\n",
      "\n",
      "Seq is: CATTTCTCCATAGA\n",
      "0 CATTTCTCCATAGA motifs detected in file\n",
      "\n",
      "Seq is: AATGATATTTTCGA\n",
      "0 AATGATATTTTCGA motifs detected in file\n",
      "\n",
      "Seq is: TAACACATTCGATC\n",
      "0 TAACACATTCGATC motifs detected in file\n",
      "\n",
      "Seq is: AGTATAATTCAACA\n",
      "0 AGTATAATTCAACA motifs detected in file\n",
      "\n",
      "Seq is: ACCGAGTCCTCAAA\n",
      "0 ACCGAGTCCTCAAA motifs detected in file\n",
      "\n",
      "Seq is: CTATCAAGAACCCG\n",
      "0 CTATCAAGAACCCG motifs detected in file\n",
      "\n",
      "Seq is: TCACTCTCACTCAT\n",
      "0 TCACTCTCACTCAT motifs detected in file\n",
      "\n",
      "Seq is: CATCTTACAAATCA\n",
      "0 CATCTTACAAATCA motifs detected in file\n",
      "\n",
      "Seq is: CTCGGATCCCACAA\n",
      "0 CTCGGATCCCACAA motifs detected in file\n",
      "\n",
      "Seq is: CACTCACATTTAAT\n",
      "0 CACTCACATTTAAT motifs detected in file\n",
      "\n",
      "Seq is: CACGCTACTCATCC\n",
      "0 CACGCTACTCATCC motifs detected in file\n",
      "\n",
      "Seq is: CCTTCCCTCGGATG\n",
      "0 CCTTCCCTCGGATG motifs detected in file\n",
      "\n",
      "Seq is: ATCCCACATCTTAT\n",
      "0 ATCCCACATCTTAT motifs detected in file\n",
      "\n",
      "Seq is: CCTTCATACACGGA\n",
      "0 CCTTCATACACGGA motifs detected in file\n",
      "\n",
      "Seq is: CGCAAGTCCGCAAG\n",
      "0 CGCAAGTCCGCAAG motifs detected in file\n",
      "\n",
      "Seq is: ACCGACCAACGGAG\n",
      "0 ACCGACCAACGGAG motifs detected in file\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Seq is: TAGCCCACTACTAT\n",
      "0 TAGCCCACTACTAT motifs detected in file\n",
      "\n",
      "Seq is: CGGTTCCACCCCTA\n",
      "0 CGGTTCCACCCCTA motifs detected in file\n",
      "\n",
      "Seq is: TTCACACCAAAGCT\n",
      "0 TTCACACCAAAGCT motifs detected in file\n",
      "\n",
      "Seq is: TCTTCTATATTTTA\n",
      "0 TCTTCTATATTTTA motifs detected in file\n",
      "\n",
      "Seq is: TACTAATCAAGACA\n",
      "0 TACTAATCAAGACA motifs detected in file\n",
      "\n",
      "Seq is: CCGACACTTACAAT\n",
      "0 CCGACACTTACAAT motifs detected in file\n",
      "\n",
      "Seq is: ATTAATTCCTCTAA\n",
      "0 ATTAATTCCTCTAA motifs detected in file\n",
      "\n",
      "Seq is: ACATCCCTCTAATC\n",
      "0 ACATCCCTCTAATC motifs detected in file\n",
      "\n",
      "Seq is: ATATCTCTCCATCT\n",
      "0 ATATCTCTCCATCT motifs detected in file\n",
      "\n",
      "Seq is: CAAAAAAAGCACTA\n",
      "0 CAAAAAAAGCACTA motifs detected in file\n",
      "\n",
      "Seq is: TCACCAAGTGACCT\n",
      "0 TCACCAAGTGACCT motifs detected in file\n",
      "\n",
      "Seq is: ATTCACCGCAACCG\n",
      "0 ATTCACCGCAACCG motifs detected in file\n",
      "\n",
      "Seq is: AACGGTACTAACAC\n",
      "0 AACGGTACTAACAC motifs detected in file\n",
      "\n",
      "Seq is: TCCCCATCCTCAGT\n",
      "0 TCCCCATCCTCAGT motifs detected in file\n",
      "\n",
      "Seq is: CCATCAATCATAAT\n",
      "0 CCATCAATCATAAT motifs detected in file\n",
      "\n",
      "Seq is: CCTCCCCCCAACAA\n",
      "0 CCTCCCCCCAACAA motifs detected in file\n",
      "\n",
      "Seq is: TCCCCCAGCCTCCT\n",
      "0 TCCCCCAGCCTCCT motifs detected in file\n",
      "\n",
      "Seq is: TTTCCAACCCCTTG\n",
      "0 TTTCCAACCCCTTG motifs detected in file\n",
      "\n",
      "Seq is: TCGATTTATCCAAC\n",
      "0 TCGATTTATCCAAC motifs detected in file\n",
      "\n",
      "Seq is: AACAACTAAACCAA\n",
      "0 AACAACTAAACCAA motifs detected in file\n",
      "\n",
      "Seq is: TTTTTTATCATCTC\n",
      "0 TTTTTTATCATCTC motifs detected in file\n",
      "\n",
      "Seq is: CCTACTACGATTAT\n",
      "0 CCTACTACGATTAT motifs detected in file\n",
      "\n",
      "Seq is: TAATCCGACACCCA\n",
      "0 TAATCCGACACCCA motifs detected in file\n",
      "\n",
      "Seq is: CCAACACAACTAAG\n",
      "0 CCAACACAACTAAG motifs detected in file\n",
      "\n",
      "Seq is: TTTAATACACAAAA\n",
      "0 TTTAATACACAAAA motifs detected in file\n",
      "\n",
      "Seq is: AACCTTTCAGTACA\n",
      "0 AACCTTTCAGTACA motifs detected in file\n",
      "\n",
      "Seq is: GTCACCATCATTAT\n",
      "0 GTCACCATCATTAT motifs detected in file\n",
      "\n",
      "Seq is: ATTAACAACCCTGC\n",
      "0 ATTAACAACCCTGC motifs detected in file\n",
      "\n",
      "Seq is: TAATCAAAATCGTA\n",
      "0 TAATCAAAATCGTA motifs detected in file\n",
      "\n",
      "Seq is: ACCTCATCCAAATC\n",
      "0 ACCTCATCCAAATC motifs detected in file\n",
      "\n",
      "Seq is: CAATACATGCATCA\n",
      "0 CAATACATGCATCA motifs detected in file\n",
      "\n",
      "Seq is: GATCCGGTATACAC\n",
      "0 GATCCGGTATACAC motifs detected in file\n",
      "\n",
      "Seq is: AACCTCTAACTCTG\n",
      "0 AACCTCTAACTCTG motifs detected in file\n",
      "\n",
      "Seq is: ATCATCCCGATCCG\n",
      "0 ATCATCCCGATCCG motifs detected in file\n",
      "\n",
      "Seq is: GGTGGCGACGAAGC\n",
      "0 GGTGGCGACGAAGC motifs detected in file\n",
      "\n",
      "Seq is: GTCCGCCGTTTCTG\n",
      "0 GTCCGCCGTTTCTG motifs detected in file\n",
      "\n",
      "Seq is: ATCATACCGTTATC\n",
      "0 ATCATACCGTTATC motifs detected in file\n",
      "\n",
      "Seq is: CGTGTTATAACCGA\n",
      "0 CGTGTTATAACCGA motifs detected in file\n",
      "\n",
      "Seq is: TTCTCCCTCACCCA\n",
      "0 TTCTCCCTCACCCA motifs detected in file\n",
      "\n",
      "Seq is: CAACTCCCAATACT\n",
      "0 CAACTCCCAATACT motifs detected in file\n",
      "\n",
      "Seq is: CTCATCGATATTAC\n",
      "0 CTCATCGATATTAC motifs detected in file\n",
      "\n",
      "Seq is: TTGTCGCTATCATA\n",
      "0 TTGTCGCTATCATA motifs detected in file\n",
      "\n",
      "Seq is: ATAAGGAAGATCCG\n",
      "0 ATAAGGAAGATCCG motifs detected in file\n",
      "\n",
      "Seq is: TATCATCCCTCCTA\n",
      "0 TATCATCCCTCCTA motifs detected in file\n",
      "\n",
      "Seq is: TACTCCACACTCGC\n",
      "0 TACTCCACACTCGC motifs detected in file\n",
      "\n",
      "Seq is: TATATGATCATAAG\n",
      "0 TATATGATCATAAG motifs detected in file\n",
      "\n",
      "Seq is: GATCTTCACTCTAC\n",
      "0 GATCTTCACTCTAC motifs detected in file\n",
      "\n",
      "Seq is: CATTTACGATCATA\n",
      "0 CATTTACGATCATA motifs detected in file\n",
      "\n",
      "Seq is: TTATAAATACATTT\n",
      "0 TTATAAATACATTT motifs detected in file\n",
      "\n",
      "Seq is: ACTCCCCGTAAACC\n",
      "0 ACTCCCCGTAAACC motifs detected in file\n",
      "\n",
      "Seq is: ATTTATACTAATAG\n",
      "0 ATTTATACTAATAG motifs detected in file\n",
      "\n",
      "Seq is: TATTAAATTACTCA\n",
      "0 TATTAAATTACTCA motifs detected in file\n",
      "\n",
      "Seq is: GCTTTAGTAAGCAA\n",
      "0 GCTTTAGTAAGCAA motifs detected in file\n",
      "\n",
      "Seq is: AACAAATTTGATAT\n",
      "0 AACAAATTTGATAT motifs detected in file\n",
      "\n",
      "Seq is: ATGGTTAAGCCAAG\n",
      "0 ATGGTTAAGCCAAG motifs detected in file\n",
      "\n",
      "Seq is: TCAGATTAGGAAAA\n",
      "0 TCAGATTAGGAAAA motifs detected in file\n",
      "\n",
      "Seq is: ACATCAGTGAGTAT\n",
      "0 ACATCAGTGAGTAT motifs detected in file\n",
      "\n",
      "Seq is: CCGGTCTGAATCCC\n",
      "0 CCGGTCTGAATCCC motifs detected in file\n",
      "\n",
      "Seq is: GATCCTACTACTAA\n",
      "0 GATCCTACTACTAA motifs detected in file\n",
      "\n",
      "Seq is: TTATTCTCCCTACT\n",
      "0 TTATTCTCCCTACT motifs detected in file\n",
      "\n",
      "Seq is: GTATTGATAATCAC\n",
      "0 GTATTGATAATCAC motifs detected in file\n",
      "\n",
      "Seq is: CGTCCAATCAACAA\n",
      "0 CGTCCAATCAACAA motifs detected in file\n",
      "\n",
      "Seq is: ACATCCCGAGTACC\n",
      "0 ACATCCCGAGTACC motifs detected in file\n",
      "\n",
      "Seq is: ATCCCCATTTTCCT\n",
      "0 ATCCCCATTTTCCT motifs detected in file\n",
      "\n",
      "Seq is: CCCGTTAAAGATCT\n",
      "0 CCCGTTAAAGATCT motifs detected in file\n",
      "\n",
      "Seq is: AGTTTATTATTCTT\n",
      "0 AGTTTATTATTCTT motifs detected in file\n",
      "\n",
      "Seq is: CCATTATCGATGAT\n",
      "0 CCATTATCGATGAT motifs detected in file\n",
      "\n",
      "Seq is: TCTTCTTAAAATAC\n",
      "0 TCTTCTTAAAATAC motifs detected in file\n",
      "\n",
      "Seq is: CTCTTTCCCTGCGC\n",
      "0 CTCTTTCCCTGCGC motifs detected in file\n",
      "\n",
      "Seq is: CCGCCCCCTATATC\n",
      "0 CCGCCCCCTATATC motifs detected in file\n",
      "\n",
      "Seq is: CCAGCTTCGGCGTG\n",
      "0 CCAGCTTCGGCGTG motifs detected in file\n",
      "\n",
      "Seq is: AGCATTCCCCACCT\n",
      "0 AGCATTCCCCACCT motifs detected in file\n",
      "\n",
      "Seq is: AAAAAGATACATCG\n",
      "0 AAAAAGATACATCG motifs detected in file\n",
      "\n",
      "Seq is: GCAACTCAACACCA\n",
      "0 GCAACTCAACACCA motifs detected in file\n",
      "\n",
      "Seq is: TCAGTTTTACATTA\n",
      "0 TCAGTTTTACATTA motifs detected in file\n",
      "\n",
      "Seq is: TTAATAATCATCAT\n",
      "0 TTAATAATCATCAT motifs detected in file\n",
      "\n",
      "Seq is: TCCGATCCTCCCAC\n",
      "0 TCCGATCCTCCCAC motifs detected in file\n",
      "\n",
      "Seq is: AACTAATAATTTTA\n",
      "0 AACTAATAATTTTA motifs detected in file\n",
      "\n",
      "Seq is: ATTCCGCTTCAAAT\n",
      "0 ATTCCGCTTCAAAT motifs detected in file\n",
      "\n",
      "Seq is: AATGAGAAAAAACT\n",
      "0 AATGAGAAAAAACT motifs detected in file\n",
      "\n",
      "Seq is: TACAACAGTTTTGA\n",
      "0 TACAACAGTTTTGA motifs detected in file\n",
      "\n",
      "Seq is: TCAATTAACCTTCA\n",
      "0 TCAATTAACCTTCA motifs detected in file\n",
      "\n",
      "Seq is: ATTGCCATGATAAA\n",
      "0 ATTGCCATGATAAA motifs detected in file\n",
      "\n",
      "Seq is: TATGATTTTCTCAT\n",
      "0 TATGATTTTCTCAT motifs detected in file\n",
      "\n",
      "Seq is: TCTGGGAGTAGTTA\n",
      "0 TCTGGGAGTAGTTA motifs detected in file\n",
      "\n",
      "Seq is: TCTACCTCTGTATG\n",
      "0 TCTACCTCTGTATG motifs detected in file\n",
      "\n",
      "Seq is: TCTTTATCAGTTTG\n",
      "0 TCTTTATCAGTTTG motifs detected in file\n",
      "\n",
      "Seq is: GTGTAGATGTTTTA\n",
      "0 GTGTAGATGTTTTA motifs detected in file\n",
      "\n",
      "Seq is: CGGTCATAGTCCGC\n",
      "0 CGGTCATAGTCCGC motifs detected in file\n",
      "\n",
      "Seq is: CGCCGAAGCTTCGG\n",
      "0 CGCCGAAGCTTCGG motifs detected in file\n",
      "\n",
      "Seq is: TTTCACTACAAACC\n",
      "0 TTTCACTACAAACC motifs detected in file\n",
      "\n",
      "Seq is: TACATTTCACTTTA\n",
      "0 TACATTTCACTTTA motifs detected in file\n",
      "\n",
      "Seq is: TAATCAACCGCCTA\n",
      "0 TAATCAACCGCCTA motifs detected in file\n",
      "\n",
      "Seq is: CTTCGTCTATCACT\n",
      "0 CTTCGTCTATCACT motifs detected in file\n",
      "\n",
      "Seq is: CCTTTCAACTCGGT\n",
      "0 CCTTTCAACTCGGT motifs detected in file\n",
      "\n",
      "Seq is: TATTACTGCACTTC\n",
      "0 TATTACTGCACTTC motifs detected in file\n",
      "\n",
      "Seq is: AGGCACCTTCGGAC\n",
      "0 AGGCACCTTCGGAC motifs detected in file\n",
      "\n",
      "Seq is: ACCGATGTTTTTTA\n",
      "0 ACCGATGTTTTTTA motifs detected in file\n",
      "\n",
      "Seq is: CAACTCGGCATCTA\n",
      "0 CAACTCGGCATCTA motifs detected in file\n",
      "\n",
      "Seq is: CGGCAGCCTTTACC\n",
      "0 CGGCAGCCTTTACC motifs detected in file\n",
      "\n",
      "Seq is: ACTTCCCTCTGAGC\n",
      "0 ACTTCCCTCTGAGC motifs detected in file\n",
      "\n",
      "Seq is: TTCATGAGACTCCG\n",
      "0 TTCATGAGACTCCG motifs detected in file\n",
      "\n",
      "Seq is: AACATCCTCCCATT\n",
      "0 AACATCCTCCCATT motifs detected in file\n",
      "\n",
      "Seq is: TTATCTCTGGGTCA\n",
      "0 TTATCTCTGGGTCA motifs detected in file\n",
      "\n",
      "Seq is: TTTTAACATTATTC\n",
      "0 TTTTAACATTATTC motifs detected in file\n",
      "\n",
      "Seq is: GTCACGAACTTAAT\n",
      "0 GTCACGAACTTAAT motifs detected in file\n",
      "\n",
      "Seq is: AAACCAAAGCCAAC\n",
      "0 AAACCAAAGCCAAC motifs detected in file\n",
      "\n",
      "Seq is: AAAAGATAATCTGA\n",
      "0 AAAAGATAATCTGA motifs detected in file\n",
      "\n",
      "Seq is: TACCACTCGAGTCC\n",
      "0 TACCACTCGAGTCC motifs detected in file\n",
      "\n",
      "Seq is: ACTAACTATGAGGA\n",
      "0 ACTAACTATGAGGA motifs detected in file\n",
      "\n",
      "Seq is: CTACGCTCATTATG\n",
      "0 CTACGCTCATTATG motifs detected in file\n",
      "\n",
      "Seq is: CCTACACAAATCCT\n",
      "0 CCTACACAAATCCT motifs detected in file\n",
      "\n",
      "Seq is: CACCCTGAAGATCC\n",
      "0 CACCCTGAAGATCC motifs detected in file\n",
      "\n",
      "Seq is: CCTAAATCGCCCCA\n",
      "0 CCTAAATCGCCCCA motifs detected in file\n",
      "\n",
      "Seq is: CTACGGACAACCCC\n",
      "0 CTACGGACAACCCC motifs detected in file\n",
      "\n",
      "Seq is: CGGTCACGGGAGGA\n",
      "0 CGGTCACGGGAGGA motifs detected in file\n",
      "\n",
      "Seq is: TTAACCCCCCATCC\n",
      "0 TTAACCCCCCATCC motifs detected in file\n",
      "\n",
      "Seq is: CCGATCCGACCTCA\n",
      "0 CCGATCCGACCTCA motifs detected in file\n",
      "\n",
      "Seq is: CCATTTTCCGAGTC\n",
      "0 CCATTTTCCGAGTC motifs detected in file\n",
      "\n",
      "Seq is: TTTTTAGGACGCTT\n",
      "0 TTTTTAGGACGCTT motifs detected in file\n",
      "\n",
      "Seq is: CTTTTTTTGAAGAC\n",
      "0 CTTTTTTTGAAGAC motifs detected in file\n",
      "\n",
      "Seq is: TCCATTATTTATCC\n",
      "0 TCCATTATTTATCC motifs detected in file\n",
      "\n",
      "Seq is: TAATAGGGCATAGC\n",
      "0 TAATAGGGCATAGC motifs detected in file\n",
      "\n",
      "Seq is: TTCCGGAAAAACCT\n",
      "0 TTCCGGAAAAACCT motifs detected in file\n",
      "\n",
      "Seq is: GTCCACCACACACC\n",
      "0 GTCCACCACACACC motifs detected in file\n",
      "\n",
      "Seq is: ACCGGAACCATACA\n",
      "0 ACCGGAACCATACA motifs detected in file\n",
      "\n",
      "Seq is: CGAAAGAGCACAAT\n",
      "0 CGAAAGAGCACAAT motifs detected in file\n",
      "\n",
      "Seq is: GTAGCGCGGTAGTA\n",
      "0 GTAGCGCGGTAGTA motifs detected in file\n",
      "\n",
      "Seq is: ACCATATACCAATC\n",
      "0 ACCATATACCAATC motifs detected in file\n",
      "\n",
      "Seq is: ACACAACCAATCAT\n",
      "0 ACACAACCAATCAT motifs detected in file\n",
      "\n",
      "Seq is: CCGGATCATACTCC\n",
      "0 CCGGATCATACTCC motifs detected in file\n",
      "\n",
      "Seq is: TCGCAATACCTCAC\n",
      "0 TCGCAATACCTCAC motifs detected in file\n",
      "\n",
      "Seq is: CTTCACTTTAGTGT\n",
      "0 CTTCACTTTAGTGT motifs detected in file\n",
      "\n",
      "Seq is: ACCGATCCGGCCTC\n",
      "0 ACCGATCCGGCCTC motifs detected in file\n",
      "\n",
      "Seq is: CAGTAATCCTCCCG\n",
      "0 CAGTAATCCTCCCG motifs detected in file\n",
      "\n",
      "Seq is: ACTCTCCCGGGGAC\n",
      "0 ACTCTCCCGGGGAC motifs detected in file\n",
      "\n",
      "Seq is: AATCCCCAGTACCC\n",
      "0 AATCCCCAGTACCC motifs detected in file\n",
      "\n",
      "Seq is: GACCCAAAATGATA\n",
      "0 GACCCAAAATGATA motifs detected in file\n",
      "\n",
      "Seq is: TACTATCCGTACAC\n",
      "0 TACTATCCGTACAC motifs detected in file\n",
      "\n",
      "Seq is: TAACCACCCAGTAA\n",
      "0 TAACCACCCAGTAA motifs detected in file\n",
      "\n",
      "Seq is: TACACAACAGCACG\n",
      "0 TACACAACAGCACG motifs detected in file\n",
      "\n",
      "Seq is: TCCATCTCCGAATG\n",
      "0 TCCATCTCCGAATG motifs detected in file\n",
      "\n",
      "Seq is: ATCTTCACACTTTT\n",
      "0 ATCTTCACACTTTT motifs detected in file\n",
      "\n",
      "Seq is: GCATCCGAACCTAA\n",
      "0 GCATCCGAACCTAA motifs detected in file\n",
      "\n",
      "Seq is: TTCCGCTGTCGCTA\n",
      "0 TTCCGCTGTCGCTA motifs detected in file\n",
      "\n",
      "Seq is: AAGATCCTATCAGT\n",
      "0 AAGATCCTATCAGT motifs detected in file\n",
      "\n",
      "Seq is: CATCTTAATCTTAA\n",
      "0 CATCTTAATCTTAA motifs detected in file\n",
      "\n",
      "Seq is: CACTTCTACTATTC\n",
      "0 CACTTCTACTATTC motifs detected in file\n",
      "\n",
      "Seq is: ACATCTCCCTTCCA\n",
      "0 ACATCTCCCTTCCA motifs detected in file\n",
      "\n",
      "Seq is: ATTACCAACTATAA\n",
      "0 ATTACCAACTATAA motifs detected in file\n",
      "\n",
      "Seq is: CGATCCCACCGCGA\n",
      "0 CGATCCCACCGCGA motifs detected in file\n",
      "\n",
      "Seq is: AGGTTAATCCACGT\n",
      "0 AGGTTAATCCACGT motifs detected in file\n",
      "\n",
      "Seq is: ACTCATCCACCGGA\n",
      "0 ACTCATCCACCGGA motifs detected in file\n",
      "\n",
      "Seq is: CGTCATTACAATCG\n",
      "0 CGTCATTACAATCG motifs detected in file\n",
      "\n",
      "Seq is: CCAATCCGCATGCC\n",
      "0 CCAATCCGCATGCC motifs detected in file\n",
      "\n",
      "Seq is: GGTCCCGATAACCA\n",
      "0 GGTCCCGATAACCA motifs detected in file\n",
      "\n",
      "Seq is: ACTTACTCATCCGA\n",
      "0 ACTTACTCATCCGA motifs detected in file\n",
      "\n",
      "Seq is: CTACCAAAGCTATT\n",
      "0 CTACCAAAGCTATT motifs detected in file\n",
      "\n",
      "Seq is: ATTGATCATACCCC\n",
      "0 ATTGATCATACCCC motifs detected in file\n",
      "\n",
      "Seq is: TATTCCCCACATCC\n",
      "0 TATTCCCCACATCC motifs detected in file\n",
      "\n",
      "Seq is: ACACGGAACACCAT\n",
      "0 ACACGGAACACCAT motifs detected in file\n",
      "\n",
      "Seq is: TCTTCACCCGATCC\n",
      "0 TCTTCACCCGATCC motifs detected in file\n",
      "\n",
      "Seq is: CGTAAAAATTAGAA\n",
      "0 CGTAAAAATTAGAA motifs detected in file\n",
      "\n",
      "Seq is: TCTCGCTTTCGGAT\n",
      "0 TCTCGCTTTCGGAT motifs detected in file\n",
      "\n",
      "Seq is: ATTAGTGACACGGG\n",
      "0 ATTAGTGACACGGG motifs detected in file\n",
      "\n",
      "Seq is: CGAGTATTCCCCTA\n",
      "0 CGAGTATTCCCCTA motifs detected in file\n",
      "\n",
      "Seq is: CCGGTACCGATCCA\n",
      "0 CCGGTACCGATCCA motifs detected in file\n",
      "\n",
      "Seq is: AATATCTATCAACC\n",
      "0 AATATCTATCAACC motifs detected in file\n",
      "\n",
      "Seq is: CACCAACCACATTT\n",
      "0 CACCAACCACATTT motifs detected in file\n",
      "\n",
      "Seq is: ACTCACTCCGTCCT\n",
      "0 ACTCACTCCGTCCT motifs detected in file\n",
      "\n",
      "Seq is: CAGGCTCCTCCAAT\n",
      "0 CAGGCTCCTCCAAT motifs detected in file\n",
      "\n",
      "Seq is: CAACACCGTTATTT\n",
      "0 CAACACCGTTATTT motifs detected in file\n",
      "\n",
      "Seq is: TTACTAATTCCTAT\n",
      "0 TTACTAATTCCTAT motifs detected in file\n",
      "\n",
      "Seq is: GATCATATTCTCTA\n",
      "0 GATCATATTCTCTA motifs detected in file\n",
      "\n",
      "Seq is: GTCCAAGCAGGAAA\n",
      "0 GTCCAAGCAGGAAA motifs detected in file\n",
      "\n",
      "Seq is: TCACAACACATACC\n",
      "0 TCACAACACATACC motifs detected in file\n",
      "\n",
      "Seq is: AATAGTAAACAAAA\n",
      "0 AATAGTAAACAAAA motifs detected in file\n",
      "\n",
      "Seq is: CTCCAATCAAACTA\n",
      "0 CTCCAATCAAACTA motifs detected in file\n",
      "\n",
      "Seq is: ATCAGTAACAACCC\n",
      "0 ATCAGTAACAACCC motifs detected in file\n",
      "\n",
      "Seq is: ACCACTAATCAGCC\n",
      "0 ACCACTAATCAGCC motifs detected in file\n",
      "\n",
      "Seq is: AACAACTACTCTAT\n",
      "0 AACAACTACTCTAT motifs detected in file\n",
      "\n",
      "Seq is: AAACCTCCACCCCT\n",
      "0 AAACCTCCACCCCT motifs detected in file\n",
      "\n",
      "Seq is: AGTTATCTCCCCCT\n",
      "0 AGTTATCTCCCCCT motifs detected in file\n",
      "\n",
      "Seq is: TTATCTTACTAGTC\n",
      "0 TTATCTTACTAGTC motifs detected in file\n",
      "\n",
      "Seq is: ATGACGCCGCCCAT\n",
      "0 ATGACGCCGCCCAT motifs detected in file\n",
      "\n",
      "Seq is: CCGGATCCTAACAC\n",
      "0 CCGGATCCTAACAC motifs detected in file\n",
      "\n",
      "Seq is: CCCCGTTACTTACT\n",
      "0 CCCCGTTACTTACT motifs detected in file\n",
      "\n",
      "Seq is: TCGCTTGTCTAAAA\n",
      "0 TCGCTTGTCTAAAA motifs detected in file\n",
      "\n",
      "Seq is: GCAAGTAAAACCAA\n",
      "0 GCAAGTAAAACCAA motifs detected in file\n",
      "\n",
      "Seq is: GAGTCCCAAACAAT\n",
      "0 GAGTCCCAAACAAT motifs detected in file\n",
      "\n",
      "Seq is: ATTAAAAAATAAAA\n",
      "0 ATTAAAAAATAAAA motifs detected in file\n",
      "\n",
      "Seq is: ATACCCGAAACCAC\n",
      "0 ATACCCGAAACCAC motifs detected in file\n",
      "\n",
      "Seq is: TCCCTCCATCCACC\n",
      "0 TCCCTCCATCCACC motifs detected in file\n",
      "\n",
      "Seq is: ATCAAACACAAATT\n",
      "0 ATCAAACACAAATT motifs detected in file\n",
      "\n",
      "Seq is: ATAAAAATCAACCC\n",
      "0 ATAAAAATCAACCC motifs detected in file\n",
      "\n",
      "Seq is: ACTACTCCTTATCA\n",
      "0 ACTACTCCTTATCA motifs detected in file\n",
      "\n",
      "Seq is: CATTCCTCATACCC\n",
      "0 CATTCCTCATACCC motifs detected in file\n",
      "\n",
      "Seq is: CCATTAATACCACC\n",
      "0 CCATTAATACCACC motifs detected in file\n",
      "\n",
      "Seq is: CGGTATGCCATCAT\n",
      "0 CGGTATGCCATCAT motifs detected in file\n",
      "\n",
      "Seq is: AAATCAACCCCGTA\n",
      "0 AAATCAACCCCGTA motifs detected in file\n",
      "\n",
      "Seq is: AAGTGACATTTCTC\n",
      "0 AAGTGACATTTCTC motifs detected in file\n",
      "\n",
      "Seq is: CACAACCAAGAGAA\n",
      "0 CACAACCAAGAGAA motifs detected in file\n",
      "\n",
      "Seq is: TTAGAAATTGAATT\n",
      "0 TTAGAAATTGAATT motifs detected in file\n",
      "\n",
      "Seq is: TTCCAATTACGATT\n",
      "0 TTCCAATTACGATT motifs detected in file\n",
      "\n",
      "Seq is: CAATCGAAATGTCA\n",
      "0 CAATCGAAATGTCA motifs detected in file\n",
      "\n",
      "Seq is: CCCGAGATCTCCCC\n",
      "0 CCCGAGATCTCCCC motifs detected in file\n",
      "\n",
      "Seq is: CATCTCCCCCACGA\n",
      "0 CATCTCCCCCACGA motifs detected in file\n",
      "\n",
      "Seq is: TATCCCATTTATGC\n",
      "0 TATCCCATTTATGC motifs detected in file\n",
      "\n",
      "Seq is: CCGGGATAAAGTTT\n",
      "0 CCGGGATAAAGTTT motifs detected in file\n",
      "\n",
      "Seq is: CTAAAAATCCCCTT\n",
      "0 CTAAAAATCCCCTT motifs detected in file\n",
      "\n",
      "Seq is: AATTAAGATCCTGC\n",
      "0 AATTAAGATCCTGC motifs detected in file\n",
      "\n",
      "Seq is: TACCCGTACTTTGA\n",
      "0 TACCCGTACTTTGA motifs detected in file\n",
      "\n",
      "Seq is: CACCAAACGAGGTG\n",
      "0 CACCAAACGAGGTG motifs detected in file\n",
      "\n",
      "Seq is: TCTAAAGTCTCGTA\n",
      "0 TCTAAAGTCTCGTA motifs detected in file\n",
      "\n",
      "Seq is: ACTGGCATCATATG\n",
      "0 ACTGGCATCATATG motifs detected in file\n",
      "\n",
      "Seq is: GGGGCCAGCACATC\n",
      "0 GGGGCCAGCACATC motifs detected in file\n",
      "\n",
      "Seq is: GCCACTTTCACCAA\n",
      "0 GCCACTTTCACCAA motifs detected in file\n",
      "\n",
      "Seq is: ACCAAATCTGCAGG\n",
      "0 ACCAAATCTGCAGG motifs detected in file\n",
      "\n",
      "Seq is: CCCTTAACGTAGAC\n",
      "0 CCCTTAACGTAGAC motifs detected in file\n",
      "\n",
      "Seq is: AAAAATTCGGATTA\n",
      "0 AAAAATTCGGATTA motifs detected in file\n",
      "\n",
      "Seq is: CACCCCTGTCGAGT\n",
      "0 CACCCCTGTCGAGT motifs detected in file\n",
      "\n",
      "Seq is: ACTCACGTTCTGCA\n",
      "0 ACTCACGTTCTGCA motifs detected in file\n",
      "\n",
      "Seq is: GAACACTACATTAA\n",
      "0 GAACACTACATTAA motifs detected in file\n",
      "\n",
      "Seq is: TAATATGCTTACCC\n",
      "0 TAATATGCTTACCC motifs detected in file\n",
      "\n",
      "Seq is: CCGAAGTTAGCCCT\n",
      "0 CCGAAGTTAGCCCT motifs detected in file\n",
      "\n",
      "Seq is: CATGATGAGCTAAC\n",
      "0 CATGATGAGCTAAC motifs detected in file\n",
      "\n",
      "Seq is: AGTTGCAGTTCCTC\n",
      "0 AGTTGCAGTTCCTC motifs detected in file\n",
      "\n",
      "Seq is: AGCGTCCAGCGGAC\n",
      "0 AGCGTCCAGCGGAC motifs detected in file\n",
      "\n",
      "Seq is: CAAGATCCTTATTA\n",
      "0 CAAGATCCTTATTA motifs detected in file\n",
      "\n",
      "Seq is: CCCCCTTCATACAT\n",
      "0 CCCCCTTCATACAT motifs detected in file\n",
      "\n",
      "Seq is: CCTCAACTTCTAAT\n",
      "0 CCTCAACTTCTAAT motifs detected in file\n",
      "\n",
      "Seq is: CAGGCGGCATCAGC\n",
      "0 CAGGCGGCATCAGC motifs detected in file\n",
      "\n",
      "Seq is: CACATGAGCATCCA\n",
      "0 CACATGAGCATCCA motifs detected in file\n",
      "\n",
      "Seq is: AGTCATGGTAACCA\n",
      "0 AGTCATGGTAACCA motifs detected in file\n",
      "\n",
      "Seq is: CCGGTTAACTAAAC\n",
      "0 CCGGTTAACTAAAC motifs detected in file\n",
      "\n",
      "Seq is: TACCATTCCCTCCC\n",
      "0 TACCATTCCCTCCC motifs detected in file\n",
      "\n",
      "Seq is: TAGCAACTGGAGCA\n",
      "0 TAGCAACTGGAGCA motifs detected in file\n",
      "\n",
      "Seq is: GACAATACATTTCC\n",
      "0 GACAATACATTTCC motifs detected in file\n",
      "\n",
      "Seq is: TACGCATCCCTACC\n",
      "0 TACGCATCCCTACC motifs detected in file\n",
      "\n",
      "Seq is: CTCCCGCTACTCCT\n",
      "0 CTCCCGCTACTCCT motifs detected in file\n",
      "\n",
      "Seq is: GATCCTACTACCGC\n",
      "0 GATCCTACTACCGC motifs detected in file\n",
      "\n",
      "Seq is: CCGTCCTATCAAGT\n",
      "0 CCGTCCTATCAAGT motifs detected in file\n",
      "\n",
      "Seq is: CTGCCAAAGATAAA\n",
      "0 CTGCCAAAGATAAA motifs detected in file\n",
      "\n",
      "Seq is: GGACTCGCAGACTC\n",
      "0 GGACTCGCAGACTC motifs detected in file\n",
      "\n",
      "Seq is: TACAATCATAATCA\n",
      "0 TACAATCATAATCA motifs detected in file\n",
      "\n",
      "Seq is: ATCAAAACAACACT\n",
      "0 ATCAAAACAACACT motifs detected in file\n",
      "\n",
      "Seq is: CACAATCCTTTTCC\n",
      "0 CACAATCCTTTTCC motifs detected in file\n",
      "\n",
      "Seq is: CGTATGTCCTGATC\n",
      "0 CGTATGTCCTGATC motifs detected in file\n",
      "\n",
      "Seq is: CTTCGTCTATTCCT\n",
      "0 CTTCGTCTATTCCT motifs detected in file\n",
      "\n",
      "Seq is: TTTACTAATACTCC\n",
      "0 TTTACTAATACTCC motifs detected in file\n",
      "\n",
      "Seq is: CGCACTAGTACTTT\n",
      "0 CGCACTAGTACTTT motifs detected in file\n",
      "\n",
      "Seq is: CCACTATTCGAGAA\n",
      "0 CCACTATTCGAGAA motifs detected in file\n",
      "\n",
      "Seq is: GATACTATCCCCTT\n",
      "0 GATACTATCCCCTT motifs detected in file\n",
      "\n",
      "Seq is: CATCGCAGAACATC\n",
      "0 CATCGCAGAACATC motifs detected in file\n",
      "\n",
      "Seq is: TGGATGAACGCGAC\n",
      "0 TGGATGAACGCGAC motifs detected in file\n",
      "\n",
      "Seq is: GTACACGGTAATTC\n",
      "0 GTACACGGTAATTC motifs detected in file\n",
      "\n",
      "Seq is: TATATATCCTAAAT\n",
      "0 TATATATCCTAAAT motifs detected in file\n",
      "\n",
      "Seq is: CGGATATTAAATTG\n",
      "0 CGGATATTAAATTG motifs detected in file\n",
      "\n",
      "Seq is: AAACTGTTTCAATA\n",
      "0 AAACTGTTTCAATA motifs detected in file\n",
      "\n",
      "Seq is: CTTTACCAAAAAGA\n",
      "0 CTTTACCAAAAAGA motifs detected in file\n",
      "\n",
      "Seq is: TTATGGAAAAACTT\n",
      "0 TTATGGAAAAACTT motifs detected in file\n",
      "\n",
      "Seq is: TTTCAGTACCTCCG\n",
      "0 TTTCAGTACCTCCG motifs detected in file\n",
      "\n",
      "Seq is: GTACCCCAACCGAA\n",
      "0 GTACCCCAACCGAA motifs detected in file\n",
      "\n",
      "Seq is: CTTTGGTCGAAACC\n",
      "0 CTTTGGTCGAAACC motifs detected in file\n",
      "\n",
      "Seq is: CCCCAAGCTAAGGA\n",
      "0 CCCCAAGCTAAGGA motifs detected in file\n",
      "\n",
      "Seq is: AGGAAAAAACAGAT\n",
      "0 AGGAAAAAACAGAT motifs detected in file\n",
      "\n",
      "Seq is: CTAAAATACATATG\n",
      "0 CTAAAATACATATG motifs detected in file\n",
      "\n",
      "Seq is: CCCAAGAAGCTTAC\n",
      "0 CCCAAGAAGCTTAC motifs detected in file\n",
      "\n",
      "Seq is: ACACCATCCCACCC\n",
      "0 ACACCATCCCACCC motifs detected in file\n",
      "\n",
      "Seq is: CCCGTAGGTATATC\n",
      "0 CCCGTAGGTATATC motifs detected in file\n",
      "\n",
      "Seq is: AGTGAGGTCCAAAT\n",
      "0 AGTGAGGTCCAAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCTCCCAAGAAGA\n",
      "0 ACCTCCCAAGAAGA motifs detected in file\n",
      "\n",
      "Seq is: TGATAATCCTGAAA\n",
      "0 TGATAATCCTGAAA motifs detected in file\n",
      "\n",
      "Seq is: AGCGAAGCTTCGCT\n",
      "0 AGCGAAGCTTCGCT motifs detected in file\n",
      "\n",
      "Seq is: TCCGAAGAGTTTAG\n",
      "0 TCCGAAGAGTTTAG motifs detected in file\n",
      "\n",
      "Seq is: TACTTTTAATAATT\n",
      "0 TACTTTTAATAATT motifs detected in file\n",
      "\n",
      "Seq is: ATAATGACGACAAT\n",
      "0 ATAATGACGACAAT motifs detected in file\n",
      "\n",
      "Seq is: CTCTTTACTTACTC\n",
      "0 CTCTTTACTTACTC motifs detected in file\n",
      "\n",
      "Seq is: GGATGTCTACTATC\n",
      "0 GGATGTCTACTATC motifs detected in file\n",
      "\n",
      "Seq is: CTACAAAGTACACC\n",
      "0 CTACAAAGTACACC motifs detected in file\n",
      "\n",
      "Seq is: ACATACGTAGCCCC\n",
      "0 ACATACGTAGCCCC motifs detected in file\n",
      "\n",
      "Seq is: ATCAGGCTCATTGC\n",
      "0 ATCAGGCTCATTGC motifs detected in file\n",
      "\n",
      "Seq is: AGCCCCGTAAGGCC\n",
      "0 AGCCCCGTAAGGCC motifs detected in file\n",
      "\n",
      "Seq is: TATCCGGCTCTTTC\n",
      "0 TATCCGGCTCTTTC motifs detected in file\n",
      "\n",
      "Seq is: ACAACACCCTTCTT\n",
      "0 ACAACACCCTTCTT motifs detected in file\n",
      "\n",
      "Seq is: TCAATCTAAATGCG\n",
      "0 TCAATCTAAATGCG motifs detected in file\n",
      "\n",
      "Seq is: GCTACTTATACTAT\n",
      "0 GCTACTTATACTAT motifs detected in file\n",
      "\n",
      "Seq is: CACTTTACCTAAAA\n",
      "0 CACTTTACCTAAAA motifs detected in file\n",
      "\n",
      "Seq is: CCGCATCCAAACCA\n",
      "0 CCGCATCCAAACCA motifs detected in file\n",
      "\n",
      "Seq is: GATCCCACATCGGA\n",
      "0 GATCCCACATCGGA motifs detected in file\n",
      "\n",
      "Seq is: CTCTTATCCCCTTT\n",
      "0 CTCTTATCCCCTTT motifs detected in file\n",
      "\n",
      "Seq is: AGTCACTTACTTCG\n",
      "0 AGTCACTTACTTCG motifs detected in file\n",
      "\n",
      "Seq is: GAGGATACTACCGT\n",
      "0 GAGGATACTACCGT motifs detected in file\n",
      "\n",
      "Seq is: TTATGTCGAGGATA\n",
      "0 TTATGTCGAGGATA motifs detected in file\n",
      "\n",
      "Seq is: ACTATCCTGTATCA\n",
      "0 ACTATCCTGTATCA motifs detected in file\n",
      "\n",
      "Seq is: CCTTCACCCGATGT\n",
      "0 CCTTCACCCGATGT motifs detected in file\n",
      "\n",
      "Seq is: TGCATCATGCACAG\n",
      "0 TGCATCATGCACAG motifs detected in file\n",
      "\n",
      "Seq is: CACATCATGCTACA\n",
      "0 CACATCATGCTACA motifs detected in file\n",
      "\n",
      "Seq is: GATCACTACTCAAA\n",
      "0 GATCACTACTCAAA motifs detected in file\n",
      "\n",
      "Seq is: CGATTATGTTACGG\n",
      "0 CGATTATGTTACGG motifs detected in file\n",
      "\n",
      "Seq is: TCAGTCCGGTGGAT\n",
      "0 TCAGTCCGGTGGAT motifs detected in file\n",
      "\n",
      "Seq is: GCCACTTTTCTTTC\n",
      "0 GCCACTTTTCTTTC motifs detected in file\n",
      "\n",
      "Seq is: TACTTAGGATCCCG\n",
      "0 TACTTAGGATCCCG motifs detected in file\n",
      "\n",
      "Seq is: AGTCTCGTGACGTC\n",
      "0 AGTCTCGTGACGTC motifs detected in file\n",
      "\n",
      "Seq is: GTCTAGTAAAGTAT\n",
      "0 GTCTAGTAAAGTAT motifs detected in file\n",
      "\n",
      "Seq is: AACGAAGGCACCTC\n",
      "0 AACGAAGGCACCTC motifs detected in file\n",
      "\n",
      "Seq is: ACACCGCTCAGTCG\n",
      "0 ACACCGCTCAGTCG motifs detected in file\n",
      "\n",
      "Seq is: ATTTATGAAACTGC\n",
      "0 ATTTATGAAACTGC motifs detected in file\n",
      "\n",
      "Seq is: GGCCACCCCTATCG\n",
      "0 GGCCACCCCTATCG motifs detected in file\n",
      "\n",
      "Seq is: CTACTAATACCATC\n",
      "0 CTACTAATACCATC motifs detected in file\n",
      "\n",
      "Seq is: GCCTCCACTTTATA\n",
      "0 GCCTCCACTTTATA motifs detected in file\n",
      "\n",
      "Seq is: CGAGCACACAGATG\n",
      "0 CGAGCACACAGATG motifs detected in file\n",
      "\n",
      "Seq is: CAGATAAGGATGAC\n",
      "0 CAGATAAGGATGAC motifs detected in file\n",
      "\n",
      "Seq is: ATTTATATACCACA\n",
      "0 ATTTATATACCACA motifs detected in file\n",
      "\n",
      "Seq is: CGAGTGTGCTATTT\n",
      "0 CGAGTGTGCTATTT motifs detected in file\n",
      "\n",
      "Seq is: GGGATCCTTCGGTT\n",
      "0 GGGATCCTTCGGTT motifs detected in file\n",
      "\n",
      "Seq is: AACTATAGTATCGA\n",
      "0 AACTATAGTATCGA motifs detected in file\n",
      "\n",
      "Seq is: GTCTGGTATGGATA\n",
      "0 GTCTGGTATGGATA motifs detected in file\n",
      "\n",
      "Seq is: CATAGGTTTACCAA\n",
      "0 CATAGGTTTACCAA motifs detected in file\n",
      "\n",
      "Seq is: GAAAAAAAGGCCTC\n",
      "0 GAAAAAAAGGCCTC motifs detected in file\n",
      "\n",
      "Seq is: ATCATTCAATGTTA\n",
      "0 ATCATTCAATGTTA motifs detected in file\n",
      "\n",
      "Seq is: TACCCTCTAATAAG\n",
      "0 TACCCTCTAATAAG motifs detected in file\n",
      "\n",
      "Seq is: GCTTCGGACCATCC\n",
      "0 GCTTCGGACCATCC motifs detected in file\n",
      "\n",
      "Seq is: TATTCTTATATTTG\n",
      "0 TATTCTTATATTTG motifs detected in file\n",
      "\n",
      "Seq is: AAGTCCCACTGGCT\n",
      "0 AAGTCCCACTGGCT motifs detected in file\n",
      "\n",
      "Seq is: TTTTAGTCTTATCC\n",
      "0 TTTTAGTCTTATCC motifs detected in file\n",
      "\n",
      "Seq is: ACAACCATATCTTA\n",
      "0 ACAACCATATCTTA motifs detected in file\n",
      "\n",
      "Seq is: CCCCAGAGGAGGAG\n",
      "0 CCCCAGAGGAGGAG motifs detected in file\n",
      "\n",
      "Seq is: GCCGCCCCAGCTTC\n",
      "0 GCCGCCCCAGCTTC motifs detected in file\n",
      "\n",
      "Seq is: TTCCACCACAACTC\n",
      "0 TTCCACCACAACTC motifs detected in file\n",
      "\n",
      "Seq is: CAACGCCAGACAAT\n",
      "0 CAACGCCAGACAAT motifs detected in file\n",
      "\n",
      "Seq is: CATCATATCACTAC\n",
      "0 CATCATATCACTAC motifs detected in file\n",
      "\n",
      "Seq is: GGTCGTCGATCCTG\n",
      "0 GGTCGTCGATCCTG motifs detected in file\n",
      "\n",
      "Seq is: ACCCTCTCTATCCT\n",
      "0 ACCCTCTCTATCCT motifs detected in file\n",
      "\n",
      "Seq is: CTTCATCCTGACGA\n",
      "0 CTTCATCCTGACGA motifs detected in file\n",
      "\n",
      "Seq is: CACTAATCCTGCCT\n",
      "0 CACTAATCCTGCCT motifs detected in file\n",
      "\n",
      "Seq is: AGTCTGCTTCTCCC\n",
      "0 AGTCTGCTTCTCCC motifs detected in file\n",
      "\n",
      "Seq is: CGCAAACCATAACC\n",
      "0 CGCAAACCATAACC motifs detected in file\n",
      "\n",
      "Seq is: CAATACCGTCCCCC\n",
      "0 CAATACCGTCCCCC motifs detected in file\n",
      "\n",
      "Seq is: AAAATATAACTATT\n",
      "0 AAAATATAACTATT motifs detected in file\n",
      "\n",
      "Seq is: AACAACACTACTTT\n",
      "0 AACAACACTACTTT motifs detected in file\n",
      "\n",
      "Seq is: AACTACCGGGGATT\n",
      "0 AACTACCGGGGATT motifs detected in file\n",
      "\n",
      "Seq is: CTATCTCCTCTGTG\n",
      "0 CTATCTCCTCTGTG motifs detected in file\n",
      "\n",
      "Seq is: GACGATCCACATTC\n",
      "0 GACGATCCACATTC motifs detected in file\n",
      "\n",
      "Seq is: CTCTTCTACCAATC\n",
      "0 CTCTTCTACCAATC motifs detected in file\n",
      "\n",
      "Seq is: CAGATGCCTCCGAG\n",
      "0 CAGATGCCTCCGAG motifs detected in file\n",
      "\n",
      "Seq is: GTCCCACCCTCATC\n",
      "0 GTCCCACCCTCATC motifs detected in file\n",
      "\n",
      "Seq is: AAGGGACGATTCCC\n",
      "0 AAGGGACGATTCCC motifs detected in file\n",
      "\n",
      "Seq is: TCCCATCTGACAAG\n",
      "0 TCCCATCTGACAAG motifs detected in file\n",
      "\n",
      "Seq is: TTGGACAAGGACGA\n",
      "0 TTGGACAAGGACGA motifs detected in file\n",
      "\n",
      "Seq is: GGCCGGAGGTGATA\n",
      "0 GGCCGGAGGTGATA motifs detected in file\n",
      "\n",
      "Seq is: TCGTCTACGCTCGT\n",
      "0 TCGTCTACGCTCGT motifs detected in file\n",
      "\n",
      "Seq is: CCTCATCCTCTCTC\n",
      "0 CCTCATCCTCTCTC motifs detected in file\n",
      "\n",
      "Seq is: CCTCCATTCTCAGT\n",
      "0 CCTCCATTCTCAGT motifs detected in file\n",
      "\n",
      "Seq is: CTTCGAATACAACA\n",
      "0 CTTCGAATACAACA motifs detected in file\n",
      "\n",
      "Seq is: AATACGCCCCTTTG\n",
      "0 AATACGCCCCTTTG motifs detected in file\n",
      "\n",
      "Seq is: CGGTATAGCCCCCG\n",
      "0 CGGTATAGCCCCCG motifs detected in file\n",
      "\n",
      "Seq is: TGGCTAATAATCCC\n",
      "0 TGGCTAATAATCCC motifs detected in file\n",
      "\n",
      "Seq is: CTTGATCAGTCAAC\n",
      "0 CTTGATCAGTCAAC motifs detected in file\n",
      "\n",
      "Seq is: GGTTTCGGAGGCTA\n",
      "0 GGTTTCGGAGGCTA motifs detected in file\n",
      "\n",
      "Seq is: ATACTACCCATAAT\n",
      "0 ATACTACCCATAAT motifs detected in file\n",
      "\n",
      "Seq is: GATACTTCTTCTAA\n",
      "0 GATACTTCTTCTAA motifs detected in file\n",
      "\n",
      "Seq is: TAATGTTTACGTAC\n",
      "0 TAATGTTTACGTAC motifs detected in file\n",
      "\n",
      "Seq is: CCGACACTGCTATT\n",
      "0 CCGACACTGCTATT motifs detected in file\n",
      "\n",
      "Seq is: GCAACATCTACACC\n",
      "0 GCAACATCTACACC motifs detected in file\n",
      "\n",
      "Seq is: AGCAATGGATCTTC\n",
      "0 AGCAATGGATCTTC motifs detected in file\n",
      "\n",
      "Seq is: CAACGGACCGACCG\n",
      "0 CAACGGACCGACCG motifs detected in file\n",
      "\n",
      "Seq is: GGTCGAGCCGAGCT\n",
      "0 GGTCGAGCCGAGCT motifs detected in file\n",
      "\n",
      "Seq is: TATTCCTCCGAATC\n",
      "0 TATTCCTCCGAATC motifs detected in file\n",
      "\n",
      "Seq is: TCGACACGGATCCT\n",
      "0 TCGACACGGATCCT motifs detected in file\n",
      "\n",
      "Seq is: GAGGTCGAGTACGC\n",
      "0 GAGGTCGAGTACGC motifs detected in file\n",
      "\n",
      "Seq is: GGCTTATTATCCAT\n",
      "0 GGCTTATTATCCAT motifs detected in file\n",
      "\n",
      "Seq is: ATCACAAGGTTACA\n",
      "0 ATCACAAGGTTACA motifs detected in file\n",
      "\n",
      "Seq is: GAAACACCAAACAT\n",
      "0 GAAACACCAAACAT motifs detected in file\n",
      "\n",
      "Seq is: CTCTTATCAGTTGC\n",
      "0 CTCTTATCAGTTGC motifs detected in file\n",
      "\n",
      "Seq is: CAGCCGCTTGTAGT\n",
      "0 CAGCCGCTTGTAGT motifs detected in file\n",
      "\n",
      "Seq is: CACCCCCACTCCAT\n",
      "0 CACCCCCACTCCAT motifs detected in file\n",
      "\n",
      "Seq is: TTTACCGACTCACT\n",
      "0 TTTACCGACTCACT motifs detected in file\n",
      "\n",
      "Seq is: TCGTAACCTGACAT\n",
      "0 TCGTAACCTGACAT motifs detected in file\n",
      "\n",
      "Seq is: TTAGATTTCTGTCC\n",
      "0 TTAGATTTCTGTCC motifs detected in file\n",
      "\n",
      "Seq is: CCAATCCGGAGAAA\n",
      "0 CCAATCCGGAGAAA motifs detected in file\n",
      "\n",
      "Seq is: AATGGTCGAGGCTC\n",
      "0 AATGGTCGAGGCTC motifs detected in file\n",
      "\n",
      "Seq is: CACTAAAAGTATAA\n",
      "0 CACTAAAAGTATAA motifs detected in file\n",
      "\n",
      "Seq is: CTTAACGTTTAAGC\n",
      "0 CTTAACGTTTAAGC motifs detected in file\n",
      "\n",
      "Seq is: TTCTTCGTCGAAGT\n",
      "0 TTCTTCGTCGAAGT motifs detected in file\n",
      "\n",
      "Seq is: TTGGACGGCCCCGA\n",
      "0 TTGGACGGCCCCGA motifs detected in file\n",
      "\n",
      "Seq is: AGAGGGCGGAAAAA\n",
      "0 AGAGGGCGGAAAAA motifs detected in file\n",
      "\n",
      "Seq is: AGGGCCGCCGCCCT\n",
      "0 AGGGCCGCCGCCCT motifs detected in file\n",
      "\n",
      "Seq is: CTTCATCTAACTTC\n",
      "0 CTTCATCTAACTTC motifs detected in file\n",
      "\n",
      "Seq is: GGTCAACTAATCCC\n",
      "0 GGTCAACTAATCCC motifs detected in file\n",
      "\n",
      "Seq is: ACGAATCGACAATT\n",
      "0 ACGAATCGACAATT motifs detected in file\n",
      "\n",
      "Seq is: GATTCACAAACACC\n",
      "0 GATTCACAAACACC motifs detected in file\n",
      "\n",
      "Seq is: CAAATTCAGGGTAA\n",
      "0 CAAATTCAGGGTAA motifs detected in file\n",
      "\n",
      "Seq is: CCAGATCATTCCCG\n",
      "0 CCAGATCATTCCCG motifs detected in file\n",
      "\n",
      "Seq is: AATCGAATTAATTT\n",
      "0 AATCGAATTAATTT motifs detected in file\n",
      "\n",
      "Seq is: CACCGACTAAACGC\n",
      "0 CACCGACTAAACGC motifs detected in file\n",
      "\n",
      "Seq is: AAGTCAACTACGTC\n",
      "0 AAGTCAACTACGTC motifs detected in file\n",
      "\n",
      "Seq is: TCACCCCAAAACGT\n",
      "0 TCACCCCAAAACGT motifs detected in file\n",
      "\n",
      "Seq is: CAGGAATCGACAAT\n",
      "0 CAGGAATCGACAAT motifs detected in file\n",
      "\n",
      "Seq is: GTCTTTAATTCATA\n",
      "0 GTCTTTAATTCATA motifs detected in file\n",
      "\n",
      "Seq is: ACGTTGAATGACTC\n",
      "0 ACGTTGAATGACTC motifs detected in file\n",
      "\n",
      "Seq is: CCGAAACTTCCGAG\n",
      "0 CCGAAACTTCCGAG motifs detected in file\n",
      "\n",
      "Seq is: AACCAGACATAAAT\n",
      "0 AACCAGACATAAAT motifs detected in file\n",
      "\n",
      "Seq is: TGGATTTAAAGATA\n",
      "0 TGGATTTAAAGATA motifs detected in file\n",
      "\n",
      "Seq is: TTCTAATAATCATA\n",
      "0 TTCTAATAATCATA motifs detected in file\n",
      "\n",
      "Seq is: TTTTCCCCTCTATC\n",
      "0 TTTTCCCCTCTATC motifs detected in file\n",
      "\n",
      "Seq is: CATCCTCATCGCAC\n",
      "0 CATCCTCATCGCAC motifs detected in file\n",
      "\n",
      "Seq is: CATTCCCGCTACTC\n",
      "0 CATTCCCGCTACTC motifs detected in file\n",
      "\n",
      "Seq is: ACACCCCTCCTTAC\n",
      "0 ACACCCCTCCTTAC motifs detected in file\n",
      "\n",
      "Seq is: CCCACCCAAAACAT\n",
      "0 CCCACCCAAAACAT motifs detected in file\n",
      "\n",
      "Seq is: ACAAGTTTGACAGT\n",
      "0 ACAAGTTTGACAGT motifs detected in file\n",
      "\n",
      "Seq is: AAAATAAAAATGCA\n",
      "0 AAAATAAAAATGCA motifs detected in file\n",
      "\n",
      "Seq is: ACAATCTATACCCC\n",
      "0 ACAATCTATACCCC motifs detected in file\n",
      "\n",
      "Seq is: TCATCACACTAACT\n",
      "0 TCATCACACTAACT motifs detected in file\n",
      "\n",
      "Seq is: CCACCTCATCTAAT\n",
      "0 CCACCTCATCTAAT motifs detected in file\n",
      "\n",
      "Seq is: CCGCATCCATCTTC\n",
      "0 CCGCATCCATCTTC motifs detected in file\n",
      "\n",
      "Seq is: ATCTCCAATTCCTC\n",
      "0 ATCTCCAATTCCTC motifs detected in file\n",
      "\n",
      "Seq is: CCACTACCACCGAT\n",
      "0 CCACTACCACCGAT motifs detected in file\n",
      "\n",
      "Seq is: ACTACCACCCCTAC\n",
      "0 ACTACCACCCCTAC motifs detected in file\n",
      "\n",
      "Seq is: TACTCCGATAACAA\n",
      "0 TACTCCGATAACAA motifs detected in file\n",
      "\n",
      "Seq is: AAAACACTTAAGAA\n",
      "0 AAAACACTTAAGAA motifs detected in file\n",
      "\n",
      "Seq is: GCTATTACCGGGTA\n",
      "0 GCTATTACCGGGTA motifs detected in file\n",
      "\n",
      "Seq is: AACCCGTTTTTCGG\n",
      "0 AACCCGTTTTTCGG motifs detected in file\n",
      "\n",
      "Seq is: CCAATCGCCCCCGT\n",
      "0 CCAATCGCCCCCGT motifs detected in file\n",
      "\n",
      "Seq is: CCGGAGGATCCCTC\n",
      "0 CCGGAGGATCCCTC motifs detected in file\n",
      "\n",
      "Seq is: TCCTCCCACCTACC\n",
      "0 TCCTCCCACCTACC motifs detected in file\n",
      "\n",
      "Seq is: TTAATTCCCACAAT\n",
      "0 TTAATTCCCACAAT motifs detected in file\n",
      "\n",
      "Seq is: CAGTACAATCGAAC\n",
      "0 CAGTACAATCGAAC motifs detected in file\n",
      "\n",
      "Seq is: AAAGTCCACGCTCT\n",
      "0 AAAGTCCACGCTCT motifs detected in file\n",
      "\n",
      "Seq is: ATCATCATCCCAGC\n",
      "0 ATCATCATCCCAGC motifs detected in file\n",
      "\n",
      "Seq is: ACCACGACCTCAAA\n",
      "0 ACCACGACCTCAAA motifs detected in file\n",
      "\n",
      "Seq is: TTCAACTCATCATC\n",
      "0 TTCAACTCATCATC motifs detected in file\n",
      "\n",
      "Seq is: CTTACGCCATCATC\n",
      "0 CTTACGCCATCATC motifs detected in file\n",
      "\n",
      "Seq is: AATCCTATTATATT\n",
      "0 AATCCTATTATATT motifs detected in file\n",
      "\n",
      "Seq is: TATCAATTTAATTC\n",
      "0 TATCAATTTAATTC motifs detected in file\n",
      "\n",
      "Seq is: TTACCAATACAATC\n",
      "0 TTACCAATACAATC motifs detected in file\n",
      "\n",
      "Seq is: CCAACATGCCATCT\n",
      "0 CCAACATGCCATCT motifs detected in file\n",
      "\n",
      "Seq is: TGACGATAATAAGT\n",
      "0 TGACGATAATAAGT motifs detected in file\n",
      "\n",
      "Seq is: AGGATACACCCATT\n",
      "0 AGGATACACCCATT motifs detected in file\n",
      "\n",
      "Seq is: AACTCCTCATACGA\n",
      "0 AACTCCTCATACGA motifs detected in file\n",
      "\n",
      "Seq is: TTCTAAAACGCATC\n",
      "0 TTCTAAAACGCATC motifs detected in file\n",
      "\n",
      "Seq is: GACCCAAACCAAAT\n",
      "0 GACCCAAACCAAAT motifs detected in file\n",
      "\n",
      "Seq is: TAGGTGGAGTTGAC\n",
      "0 TAGGTGGAGTTGAC motifs detected in file\n",
      "\n",
      "Seq is: GGACGATACTACCT\n",
      "0 GGACGATACTACCT motifs detected in file\n",
      "\n",
      "Seq is: ATTCTAACTCTCTC\n",
      "0 ATTCTAACTCTCTC motifs detected in file\n",
      "\n",
      "Seq is: ACTCCTCTTCCGAA\n",
      "0 ACTCCTCTTCCGAA motifs detected in file\n",
      "\n",
      "Seq is: TGCAAATCACTCCC\n",
      "0 TGCAAATCACTCCC motifs detected in file\n",
      "\n",
      "Seq is: TCTCTAAACCATAT\n",
      "0 TCTCTAAACCATAT motifs detected in file\n",
      "\n",
      "Seq is: ACTAACTCTACCCC\n",
      "0 ACTAACTCTACCCC motifs detected in file\n",
      "\n",
      "Seq is: CGATCAAAAACAGT\n",
      "0 CGATCAAAAACAGT motifs detected in file\n",
      "\n",
      "Seq is: ACACTCTTCTTCGT\n",
      "0 ACACTCTTCTTCGT motifs detected in file\n",
      "\n",
      "Seq is: CCGGCCTACAGTCT\n",
      "0 CCGGCCTACAGTCT motifs detected in file\n",
      "\n",
      "Seq is: CCCCACGGAACCCA\n",
      "0 CCCCACGGAACCCA motifs detected in file\n",
      "\n",
      "Seq is: TTGGAGACCCTGAG\n",
      "0 TTGGAGACCCTGAG motifs detected in file\n",
      "\n",
      "Seq is: TCTTCACTTTCCCC\n",
      "0 TCTTCACTTTCCCC motifs detected in file\n",
      "\n",
      "Seq is: CGATAAGGATCAAA\n",
      "0 CGATAAGGATCAAA motifs detected in file\n",
      "\n",
      "Seq is: ATAACGATATCGAT\n",
      "0 ATAACGATATCGAT motifs detected in file\n",
      "\n",
      "Seq is: AATACTAATAATTA\n",
      "0 AATACTAATAATTA motifs detected in file\n",
      "\n",
      "Seq is: CTACTCATAACTAA\n",
      "0 CTACTCATAACTAA motifs detected in file\n",
      "\n",
      "Seq is: CCATCATAACCAAT\n",
      "0 CCATCATAACCAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCAAGTAACAGGC\n",
      "0 ACCAAGTAACAGGC motifs detected in file\n",
      "\n",
      "Seq is: CTCTCATATAACAA\n",
      "0 CTCTCATATAACAA motifs detected in file\n",
      "\n",
      "Seq is: CTTCTCCTATCGAT\n",
      "0 CTTCTCCTATCGAT motifs detected in file\n",
      "\n",
      "Seq is: AATCTTCCTAATAC\n",
      "0 AATCTTCCTAATAC motifs detected in file\n",
      "\n",
      "Seq is: CTACGCCAACGAAC\n",
      "0 CTACGCCAACGAAC motifs detected in file\n",
      "\n",
      "Seq is: GCACTCCTTTATGA\n",
      "0 GCACTCCTTTATGA motifs detected in file\n",
      "\n",
      "Seq is: ACTACCGTCGAAGA\n",
      "0 ACTACCGTCGAAGA motifs detected in file\n",
      "\n",
      "Seq is: CACCTTGCTCCCAA\n",
      "0 CACCTTGCTCCCAA motifs detected in file\n",
      "\n",
      "Seq is: ATAAAAAAACCAAT\n",
      "0 ATAAAAAAACCAAT motifs detected in file\n",
      "\n",
      "Seq is: CTTGACCTTATTTT\n",
      "0 CTTGACCTTATTTT motifs detected in file\n",
      "\n",
      "Seq is: CGATCGTACAAATA\n",
      "0 CGATCGTACAAATA motifs detected in file\n",
      "\n",
      "Seq is: AAGATCCGGATGAG\n",
      "0 AAGATCCGGATGAG motifs detected in file\n",
      "\n",
      "Seq is: TCCATTTTTTAGTC\n",
      "0 TCCATTTTTTAGTC motifs detected in file\n",
      "\n",
      "Seq is: ACGCTCGAATCGCG\n",
      "0 ACGCTCGAATCGCG motifs detected in file\n",
      "\n",
      "Seq is: ACACTACTCACACG\n",
      "0 ACACTACTCACACG motifs detected in file\n",
      "\n",
      "Seq is: GACGTTTCTACCAT\n",
      "0 GACGTTTCTACCAT motifs detected in file\n",
      "\n",
      "Seq is: CTCATCTACTGCCC\n",
      "0 CTCATCTACTGCCC motifs detected in file\n",
      "\n",
      "Seq is: AACCCGGTCCCCTA\n",
      "0 AACCCGGTCCCCTA motifs detected in file\n",
      "\n",
      "Seq is: ATTAATCATGCCCT\n",
      "0 ATTAATCATGCCCT motifs detected in file\n",
      "\n",
      "Seq is: TCCCATATTGGTTG\n",
      "0 TCCCATATTGGTTG motifs detected in file\n",
      "\n",
      "Seq is: TAAAAGCCCCATAC\n",
      "0 TAAAAGCCCCATAC motifs detected in file\n",
      "\n",
      "Seq is: CCGGGCTATCGAAT\n",
      "0 CCGGGCTATCGAAT motifs detected in file\n",
      "\n",
      "Seq is: AAATCGACTGGAAT\n",
      "0 AAATCGACTGGAAT motifs detected in file\n",
      "\n",
      "Seq is: GAAATCCTACCCCA\n",
      "0 GAAATCCTACCCCA motifs detected in file\n",
      "\n",
      "Seq is: CACTATCCACCGTG\n",
      "0 CACTATCCACCGTG motifs detected in file\n",
      "\n",
      "Seq is: CCTCTTAAAACCTA\n",
      "0 CCTCTTAAAACCTA motifs detected in file\n",
      "\n",
      "Seq is: AGAGTCCCTACCCA\n",
      "0 AGAGTCCCTACCCA motifs detected in file\n",
      "\n",
      "Seq is: AGCTAAGAGTATCA\n",
      "0 AGCTAAGAGTATCA motifs detected in file\n",
      "\n",
      "Seq is: GGATCTTTATTCCC\n",
      "0 GGATCTTTATTCCC motifs detected in file\n",
      "\n",
      "Seq is: CCAAATTCGAGGAT\n",
      "0 CCAAATTCGAGGAT motifs detected in file\n",
      "\n",
      "Seq is: AATAAATGAGATAG\n",
      "0 AATAAATGAGATAG motifs detected in file\n",
      "\n",
      "Seq is: TTTCATTGAGAAAA\n",
      "0 TTTCATTGAGAAAA motifs detected in file\n",
      "\n",
      "Seq is: TAGTCTGTATAAAG\n",
      "0 TAGTCTGTATAAAG motifs detected in file\n",
      "\n",
      "Seq is: AATCCAAACTCCCC\n",
      "0 AATCCAAACTCCCC motifs detected in file\n",
      "\n",
      "Seq is: CTTACGACCTCTAA\n",
      "0 CTTACGACCTCTAA motifs detected in file\n",
      "\n",
      "Seq is: CATTACCCATACCT\n",
      "0 CATTACCCATACCT motifs detected in file\n",
      "\n",
      "Seq is: CTGTATAGTATATT\n",
      "0 CTGTATAGTATATT motifs detected in file\n",
      "\n",
      "Seq is: CATTACGATCCCAC\n",
      "0 CATTACGATCCCAC motifs detected in file\n",
      "\n",
      "Seq is: TCACCATCCTTCAA\n",
      "0 TCACCATCCTTCAA motifs detected in file\n",
      "\n",
      "Seq is: AAAAGTATCCTCCA\n",
      "0 AAAAGTATCCTCCA motifs detected in file\n",
      "\n",
      "Seq is: CATACTCAACCAGC\n",
      "0 CATACTCAACCAGC motifs detected in file\n",
      "\n",
      "Seq is: ATCGCCTTAGCCCC\n",
      "0 ATCGCCTTAGCCCC motifs detected in file\n",
      "\n",
      "Seq is: CATACGACAAGCTT\n",
      "0 CATACGACAAGCTT motifs detected in file\n",
      "\n",
      "Seq is: AAGTATTCTTGTCC\n",
      "0 AAGTATTCTTGTCC motifs detected in file\n",
      "\n",
      "Seq is: CTCCAATCTTCATC\n",
      "0 CTCCAATCTTCATC motifs detected in file\n",
      "\n",
      "Seq is: CCAGAACCACTGTT\n",
      "0 CCAGAACCACTGTT motifs detected in file\n",
      "\n",
      "Seq is: TTATACAACACATC\n",
      "0 TTATACAACACATC motifs detected in file\n",
      "\n",
      "Seq is: TCAAGTCCCCTCTC\n",
      "0 TCAAGTCCCCTCTC motifs detected in file\n",
      "\n",
      "Seq is: ACGCAGTATACAAC\n",
      "0 ACGCAGTATACAAC motifs detected in file\n",
      "\n",
      "Seq is: AAGGATCCTTCTAA\n",
      "0 AAGGATCCTTCTAA motifs detected in file\n",
      "\n",
      "Seq is: CATCACCACTCCCA\n",
      "0 CATCACCACTCCCA motifs detected in file\n",
      "\n",
      "Seq is: CAAATAATATTATT\n",
      "0 CAAATAATATTATT motifs detected in file\n",
      "\n",
      "Seq is: ACAAACACATAAGC\n",
      "0 ACAAACACATAAGC motifs detected in file\n",
      "\n",
      "Seq is: CGATACTTCTTATC\n",
      "0 CGATACTTCTTATC motifs detected in file\n",
      "\n",
      "Seq is: CCGCTTCCCCGGAC\n",
      "0 CCGCTTCCCCGGAC motifs detected in file\n",
      "\n",
      "Seq is: GCCGCATAAGCTAC\n",
      "0 GCCGCATAAGCTAC motifs detected in file\n",
      "\n",
      "Seq is: AACTTCGGACTCTG\n",
      "0 AACTTCGGACTCTG motifs detected in file\n",
      "\n",
      "Seq is: ATCAAGCCTGAGGG\n",
      "0 ATCAAGCCTGAGGG motifs detected in file\n",
      "\n",
      "Seq is: GAAGCCGTTCCAGC\n",
      "0 GAAGCCGTTCCAGC motifs detected in file\n",
      "\n",
      "Seq is: TTCCCCCAAGCCAA\n",
      "0 TTCCCCCAAGCCAA motifs detected in file\n",
      "\n",
      "Seq is: CCAGAGACGATCAC\n",
      "0 CCAGAGACGATCAC motifs detected in file\n",
      "\n",
      "Seq is: ACCTCTATTTAGTA\n",
      "0 ACCTCTATTTAGTA motifs detected in file\n",
      "\n",
      "Seq is: TAATACCGGTTCCC\n",
      "0 TAATACCGGTTCCC motifs detected in file\n",
      "\n",
      "Seq is: AGTACTACCGTCCT\n",
      "0 AGTACTACCGTCCT motifs detected in file\n",
      "\n",
      "Seq is: CATTAGTCTCCACA\n",
      "0 CATTAGTCTCCACA motifs detected in file\n",
      "\n",
      "Seq is: AGAACACAACACTA\n",
      "0 AGAACACAACACTA motifs detected in file\n",
      "\n",
      "Seq is: TTCCCACCTCTCCA\n",
      "0 TTCCCACCTCTCCA motifs detected in file\n",
      "\n",
      "Seq is: ATTTCCTCGGTGAA\n",
      "0 ATTTCCTCGGTGAA motifs detected in file\n",
      "\n",
      "Seq is: TAATCATTACAACT\n",
      "0 TAATCATTACAACT motifs detected in file\n",
      "\n",
      "Seq is: ATCATCTTACTACC\n",
      "0 ATCATCTTACTACC motifs detected in file\n",
      "\n",
      "Seq is: GATCCCACTGAAGT\n",
      "0 GATCCCACTGAAGT motifs detected in file\n",
      "\n",
      "Seq is: ATACTCTAACAAAC\n",
      "0 ATACTCTAACAAAC motifs detected in file\n",
      "\n",
      "Seq is: CCGATGACGAGCGT\n",
      "0 CCGATGACGAGCGT motifs detected in file\n",
      "\n",
      "Seq is: CACGCGGCTAGTCC\n",
      "0 CACGCGGCTAGTCC motifs detected in file\n",
      "\n",
      "Seq is: CGCATCAAACTCAA\n",
      "0 CGCATCAAACTCAA motifs detected in file\n",
      "\n",
      "Seq is: ACTACGAGTGGGAC\n",
      "0 ACTACGAGTGGGAC motifs detected in file\n",
      "\n",
      "Seq is: TAGTCTCCTAACTC\n",
      "0 TAGTCTCCTAACTC motifs detected in file\n",
      "\n",
      "Seq is: ATTTGCCGATCCGA\n",
      "0 ATTTGCCGATCCGA motifs detected in file\n",
      "\n",
      "Seq is: TCTCCACCGATCTT\n",
      "0 TCTCCACCGATCTT motifs detected in file\n",
      "\n",
      "Seq is: ATTTATCCTCCGGA\n",
      "0 ATTTATCCTCCGGA motifs detected in file\n",
      "\n",
      "Seq is: TCCAACTCCAACTG\n",
      "0 TCCAACTCCAACTG motifs detected in file\n",
      "\n",
      "Seq is: GTCCCCCAACCCAT\n",
      "0 GTCCCCCAACCCAT motifs detected in file\n",
      "\n",
      "Seq is: ACCCCTCCCCCCAA\n",
      "0 ACCCCTCCCCCCAA motifs detected in file\n",
      "\n",
      "Seq is: GTATCATCTTCTCG\n",
      "0 GTATCATCTTCTCG motifs detected in file\n",
      "\n",
      "Seq is: CTACCACTCTCGAT\n",
      "0 CTACCACTCTCGAT motifs detected in file\n",
      "\n",
      "Seq is: TCCAGCCCCGCCAC\n",
      "0 TCCAGCCCCGCCAC motifs detected in file\n",
      "\n",
      "Seq is: TACATCTCCCACTA\n",
      "0 TACATCTCCCACTA motifs detected in file\n",
      "\n",
      "Seq is: CCATCTACACCGCC\n",
      "0 CCATCTACACCGCC motifs detected in file\n",
      "\n",
      "Seq is: CAAAATCCCCGAGA\n",
      "0 CAAAATCCCCGAGA motifs detected in file\n",
      "\n",
      "Seq is: AACCACTTCTCAAA\n",
      "0 AACCACTTCTCAAA motifs detected in file\n",
      "\n",
      "Seq is: ATACCGCAGTCGCT\n",
      "0 ATACCGCAGTCGCT motifs detected in file\n",
      "\n",
      "Seq is: TCCCAACATCATCG\n",
      "0 TCCCAACATCATCG motifs detected in file\n",
      "\n",
      "Seq is: GGCATCCCCGGATG\n",
      "0 GGCATCCCCGGATG motifs detected in file\n",
      "\n",
      "Seq is: TTGCAACCCCGGAA\n",
      "0 TTGCAACCCCGGAA motifs detected in file\n",
      "\n",
      "Seq is: ACGCATCAACATAT\n",
      "0 ACGCATCAACATAT motifs detected in file\n",
      "\n",
      "Seq is: ATCGGATCTTAAAA\n",
      "0 ATCGGATCTTAAAA motifs detected in file\n",
      "\n",
      "Seq is: AGCAAGCCATTCGT\n",
      "0 AGCAAGCCATTCGT motifs detected in file\n",
      "\n",
      "Seq is: AATCCTTACGGTAA\n",
      "0 AATCCTTACGGTAA motifs detected in file\n",
      "\n",
      "Seq is: CGCTAATCTTACCC\n",
      "0 CGCTAATCTTACCC motifs detected in file\n",
      "\n",
      "Seq is: ATGTTACTCCTCAT\n",
      "0 ATGTTACTCCTCAT motifs detected in file\n",
      "\n",
      "Seq is: CCTCCAACCGGTAC\n",
      "0 CCTCCAACCGGTAC motifs detected in file\n",
      "\n",
      "Seq is: CCATACAACAATTC\n",
      "0 CCATACAACAATTC motifs detected in file\n",
      "\n",
      "Seq is: TTCTCCTTAACTTG\n",
      "0 TTCTCCTTAACTTG motifs detected in file\n",
      "\n",
      "Seq is: GAGACTGACATTTC\n",
      "0 GAGACTGACATTTC motifs detected in file\n",
      "\n",
      "Seq is: AAAATTCAAAATAC\n",
      "0 AAAATTCAAAATAC motifs detected in file\n",
      "\n",
      "Seq is: GCTAATGGCCCGAG\n",
      "0 GCTAATGGCCCGAG motifs detected in file\n",
      "\n",
      "Seq is: ACGGTAGAATTGTT\n",
      "0 ACGGTAGAATTGTT motifs detected in file\n",
      "\n",
      "Seq is: TGGGACAAGAACCC\n",
      "0 TGGGACAAGAACCC motifs detected in file\n",
      "\n",
      "Seq is: ACCCACACCCATAT\n",
      "0 ACCCACACCCATAT motifs detected in file\n",
      "\n",
      "Seq is: TATGATTCAACTCT\n",
      "0 TATGATTCAACTCT motifs detected in file\n",
      "\n",
      "Seq is: ACTATAGTAAATGC\n",
      "0 ACTATAGTAAATGC motifs detected in file\n",
      "\n",
      "Seq is: CCCCTTCCGCGAAA\n",
      "0 CCCCTTCCGCGAAA motifs detected in file\n",
      "\n",
      "Seq is: CACTTCATCCGGAA\n",
      "0 CACTTCATCCGGAA motifs detected in file\n",
      "\n",
      "Seq is: TAAAGAGAACAGGA\n",
      "0 TAAAGAGAACAGGA motifs detected in file\n",
      "\n",
      "Seq is: AAGCATGTCCCTCC\n",
      "0 AAGCATGTCCCTCC motifs detected in file\n",
      "\n",
      "Seq is: TTAAACTTNCATCT\n",
      "0 TTAAACTTNCATCT motifs detected in file\n",
      "\n",
      "Seq is: ATCTTTGGCTGGAC\n",
      "0 ATCTTTGGCTGGAC motifs detected in file\n",
      "\n",
      "Seq is: CTAATGAGGCCAGA\n",
      "0 CTAATGAGGCCAGA motifs detected in file\n",
      "\n",
      "Seq is: CTTGAGTCTAGTGC\n",
      "0 CTTGAGTCTAGTGC motifs detected in file\n",
      "\n",
      "Seq is: ATCCTGAAATTAGC\n",
      "0 ATCCTGAAATTAGC motifs detected in file\n",
      "\n",
      "Seq is: AACTTGTTTGCTTG\n",
      "0 AACTTGTTTGCTTG motifs detected in file\n",
      "\n",
      "Seq is: GAAATTATCGCCGA\n",
      "0 GAAATTATCGCCGA motifs detected in file\n",
      "\n",
      "Seq is: CGTGGTAGCCCTAC\n",
      "0 CGTGGTAGCCCTAC motifs detected in file\n",
      "\n",
      "Seq is: AGGACTAGGTTGTA\n",
      "0 AGGACTAGGTTGTA motifs detected in file\n",
      "\n",
      "Seq is: GCTCCAGCATTTGG\n",
      "0 GCTCCAGCATTTGG motifs detected in file\n",
      "\n",
      "Seq is: GATAACAACTATAC\n",
      "0 GATAACAACTATAC motifs detected in file\n",
      "\n",
      "Seq is: CTGAGATCTTATCC\n",
      "0 CTGAGATCTTATCC motifs detected in file\n",
      "\n",
      "Seq is: TAACGCGACAATAG\n",
      "0 TAACGCGACAATAG motifs detected in file\n",
      "\n",
      "Seq is: GGATCCCATTGAAC\n",
      "0 GGATCCCATTGAAC motifs detected in file\n",
      "\n",
      "Seq is: AAGGCAACCAGTTC\n",
      "0 AAGGCAACCAGTTC motifs detected in file\n",
      "\n",
      "Seq is: AATAACCTAGTTAA\n",
      "0 AATAACCTAGTTAA motifs detected in file\n",
      "\n",
      "Seq is: CTCATATCATCAAG\n",
      "0 CTCATATCATCAAG motifs detected in file\n",
      "\n",
      "Seq is: CGAAACTGACCACT\n",
      "0 CGAAACTGACCACT motifs detected in file\n",
      "\n",
      "Seq is: TCAGAATCGTACAT\n",
      "0 TCAGAATCGTACAT motifs detected in file\n",
      "\n",
      "Seq is: GACGAGCCTCCAAC\n",
      "0 GACGAGCCTCCAAC motifs detected in file\n",
      "\n",
      "Seq is: CCAAGACGAGGCTC\n",
      "0 CCAAGACGAGGCTC motifs detected in file\n",
      "\n",
      "Seq is: CAGCGGGGTTGGCT\n",
      "0 CAGCGGGGTTGGCT motifs detected in file\n",
      "\n",
      "Seq is: TTAAAAATTACGTC\n",
      "0 TTAAAAATTACGTC motifs detected in file\n",
      "\n",
      "Seq is: CAAACCATCAAATC\n",
      "0 CAAACCATCAAATC motifs detected in file\n",
      "\n",
      "Seq is: CTGGACACCCAAAC\n",
      "0 CTGGACACCCAAAC motifs detected in file\n",
      "\n",
      "Seq is: AATCCATGACAAAC\n",
      "0 AATCCATGACAAAC motifs detected in file\n",
      "\n",
      "Seq is: GTAATTATTTAATT\n",
      "0 GTAATTATTTAATT motifs detected in file\n",
      "\n",
      "Seq is: TCGAGGTATCCCAG\n",
      "0 TCGAGGTATCCCAG motifs detected in file\n",
      "\n",
      "Seq is: AAGAGCAGAACGAC\n",
      "0 AAGAGCAGAACGAC motifs detected in file\n",
      "\n",
      "Seq is: ACAATACGGGCGGA\n",
      "0 ACAATACGGGCGGA motifs detected in file\n",
      "\n",
      "Seq is: GAAGTGCCCGTCCA\n",
      "0 GAAGTGCCCGTCCA motifs detected in file\n",
      "\n",
      "Seq is: GTTAAAGTGACCAA\n",
      "0 GTTAAAGTGACCAA motifs detected in file\n",
      "\n",
      "Seq is: TTTTCATTCTCTGT\n",
      "0 TTTTCATTCTCTGT motifs detected in file\n",
      "\n",
      "Seq is: CGACTTGGGAGCAC\n",
      "0 CGACTTGGGAGCAC motifs detected in file\n",
      "\n",
      "Seq is: CTCGGTAAGTATGT\n",
      "0 CTCGGTAAGTATGT motifs detected in file\n",
      "\n",
      "Seq is: CCAGGGATAAATTC\n",
      "0 CCAGGGATAAATTC motifs detected in file\n",
      "\n",
      "Seq is: CTTGTTCACTAATA\n",
      "0 CTTGTTCACTAATA motifs detected in file\n",
      "\n",
      "Seq is: CGATGGAAACGTGC\n",
      "0 CGATGGAAACGTGC motifs detected in file\n",
      "\n",
      "Seq is: CAATCCCATGGCGC\n",
      "0 CAATCCCATGGCGC motifs detected in file\n",
      "\n",
      "Seq is: CGGCAATTTGTACA\n",
      "0 CGGCAATTTGTACA motifs detected in file\n",
      "\n",
      "Seq is: CAGTGACCCGTCCG\n",
      "0 CAGTGACCCGTCCG motifs detected in file\n",
      "\n",
      "Seq is: CCACGGAGATTATG\n",
      "0 CCACGGAGATTATG motifs detected in file\n",
      "\n",
      "Seq is: ACCACTACGATCTC\n",
      "0 ACCACTACGATCTC motifs detected in file\n",
      "\n",
      "Seq is: CACTACAAAAACCA\n",
      "0 CACTACAAAAACCA motifs detected in file\n",
      "\n",
      "Seq is: TTTGTCCGCCCCAT\n",
      "0 TTTGTCCGCCCCAT motifs detected in file\n",
      "\n",
      "Seq is: TCTAAACGGCTCAA\n",
      "0 TCTAAACGGCTCAA motifs detected in file\n",
      "\n",
      "Seq is: GGAAAATGAAAAAA\n",
      "0 GGAAAATGAAAAAA motifs detected in file\n",
      "\n",
      "Seq is: ATTGGAAAGGAATA\n",
      "0 ATTGGAAAGGAATA motifs detected in file\n",
      "\n",
      "Seq is: CTCGTACGGACACA\n",
      "0 CTCGTACGGACACA motifs detected in file\n",
      "\n",
      "Seq is: ACCCAACTGTCACT\n",
      "0 ACCCAACTGTCACT motifs detected in file\n",
      "\n",
      "Seq is: CCCATTATTACTGA\n",
      "0 CCCATTATTACTGA motifs detected in file\n",
      "\n",
      "Seq is: ACAACCAACTAACA\n",
      "0 ACAACCAACTAACA motifs detected in file\n",
      "\n",
      "Seq is: TCTATAACCCGACA\n",
      "0 TCTATAACCCGACA motifs detected in file\n",
      "\n",
      "Seq is: ATTAACAGTCAAGT\n",
      "0 ATTAACAGTCAAGT motifs detected in file\n",
      "\n",
      "Seq is: CACAAAATTAGACT\n",
      "0 CACAAAATTAGACT motifs detected in file\n",
      "\n",
      "Seq is: GCGTCCGAATACGC\n",
      "0 GCGTCCGAATACGC motifs detected in file\n",
      "\n",
      "Seq is: CTCCTCTTACAAAA\n",
      "0 CTCCTCTTACAAAA motifs detected in file\n",
      "\n",
      "Seq is: GTACAATGAATATG\n",
      "0 GTACAATGAATATG motifs detected in file\n",
      "\n",
      "Seq is: ATTGTAATCAAGAA\n",
      "0 ATTGTAATCAAGAA motifs detected in file\n",
      "\n",
      "Seq is: GATATCCCACTATC\n",
      "0 GATATCCCACTATC motifs detected in file\n",
      "\n",
      "Seq is: TAACCAGGTTAACC\n",
      "0 TAACCAGGTTAACC motifs detected in file\n",
      "\n",
      "Seq is: CACACTCCTCAAGT\n",
      "0 CACACTCCTCAAGT motifs detected in file\n",
      "\n",
      "Seq is: CAATATACAAACCC\n",
      "0 CAATATACAAACCC motifs detected in file\n",
      "\n",
      "Seq is: TAAAAAATCCATCA\n",
      "0 TAAAAAATCCATCA motifs detected in file\n",
      "\n",
      "Seq is: CCCACAACTCGAAC\n",
      "0 CCCACAACTCGAAC motifs detected in file\n",
      "\n",
      "Seq is: TTGCGAAAGAATTA\n",
      "0 TTGCGAAAGAATTA motifs detected in file\n",
      "\n",
      "Seq is: ACCACCGACGAAAA\n",
      "0 ACCACCGACGAAAA motifs detected in file\n",
      "\n",
      "Seq is: TCCGGATGATACCC\n",
      "0 TCCGGATGATACCC motifs detected in file\n",
      "\n",
      "Seq is: ACAATTTAAAAAAT\n",
      "0 ACAATTTAAAAAAT motifs detected in file\n",
      "\n",
      "Seq is: GAGAGAGATGTTCC\n",
      "0 GAGAGAGATGTTCC motifs detected in file\n",
      "\n",
      "Seq is: AAAAAAGGATCACA\n",
      "0 AAAAAAGGATCACA motifs detected in file\n",
      "\n",
      "Seq is: GGTTTCTCGACAAG\n",
      "0 GGTTTCTCGACAAG motifs detected in file\n",
      "\n",
      "Seq is: GAGAAACCTGATTG\n",
      "0 GAGAAACCTGATTG motifs detected in file\n",
      "\n",
      "Seq is: TCAATTTAAATGTT\n",
      "0 TCAATTTAAATGTT motifs detected in file\n",
      "\n",
      "Seq is: CCCCTAAATCTCCC\n",
      "0 CCCCTAAATCTCCC motifs detected in file\n",
      "\n",
      "Seq is: AAGACACCCGTTTA\n",
      "0 AAGACACCCGTTTA motifs detected in file\n",
      "\n",
      "Seq is: AATTTCAACTTGAT\n",
      "0 AATTTCAACTTGAT motifs detected in file\n",
      "\n",
      "Seq is: TCTAAGATAGAACC\n",
      "0 TCTAAGATAGAACC motifs detected in file\n",
      "\n",
      "Seq is: TGTTGGTCGATAGT\n",
      "0 TGTTGGTCGATAGT motifs detected in file\n",
      "\n",
      "Seq is: GGTCCGAGCCATCC\n",
      "0 GGTCCGAGCCATCC motifs detected in file\n",
      "\n",
      "Seq is: AAACAGCGGAGATG\n",
      "0 AAACAGCGGAGATG motifs detected in file\n",
      "\n",
      "Seq is: GATATTTAGAAGGG\n",
      "0 GATATTTAGAAGGG motifs detected in file\n",
      "\n",
      "Seq is: TGATAAAACGATGT\n",
      "0 TGATAAAACGATGT motifs detected in file\n",
      "\n",
      "Seq is: ATCTGCCCACACGA\n",
      "0 ATCTGCCCACACGA motifs detected in file\n",
      "\n",
      "Seq is: GAAAATCGACAAGA\n",
      "0 GAAAATCGACAAGA motifs detected in file\n",
      "\n",
      "Seq is: ATCCATCGAGCAGA\n",
      "0 ATCCATCGAGCAGA motifs detected in file\n",
      "\n",
      "Seq is: CCAAAGCCCCCAGA\n",
      "0 CCAAAGCCCCCAGA motifs detected in file\n",
      "\n",
      "Seq is: ATCGAAACCGAGAG\n",
      "0 ATCGAAACCGAGAG motifs detected in file\n",
      "\n",
      "Seq is: GAACGTTTCAATAA\n",
      "0 GAACGTTTCAATAA motifs detected in file\n",
      "\n",
      "Seq is: AGATCAATTAAGTA\n",
      "0 AGATCAATTAAGTA motifs detected in file\n",
      "\n",
      "Seq is: ATACGTCTTCCATA\n",
      "0 ATACGTCTTCCATA motifs detected in file\n",
      "\n",
      "Seq is: TCCCCAATCAGGAA\n",
      "0 TCCCCAATCAGGAA motifs detected in file\n",
      "\n",
      "Seq is: CGATATAATACGAA\n",
      "0 CGATATAATACGAA motifs detected in file\n",
      "\n",
      "Seq is: CCAATATTAAAAAG\n",
      "0 CCAATATTAAAAAG motifs detected in file\n",
      "\n",
      "Seq is: TAGAAAGGGAACGC\n",
      "0 TAGAAAGGGAACGC motifs detected in file\n",
      "\n",
      "Seq is: CATGATATAGATAA\n",
      "0 CATGATATAGATAA motifs detected in file\n",
      "\n",
      "Seq is: CGCGGTCCAAAGTT\n",
      "0 CGCGGTCCAAAGTT motifs detected in file\n",
      "\n",
      "Seq is: AAAGATAGCGGATA\n",
      "0 AAAGATAGCGGATA motifs detected in file\n",
      "\n",
      "Seq is: TGAAATAAACCCAT\n",
      "0 TGAAATAAACCCAT motifs detected in file\n",
      "\n",
      "Seq is: TTACCAAACCGATT\n",
      "0 TTACCAAACCGATT motifs detected in file\n",
      "\n",
      "Seq is: CCAACAGACCATCA\n",
      "0 CCAACAGACCATCA motifs detected in file\n",
      "\n",
      "Seq is: TTCCACCTCACCCA\n",
      "0 TTCCACCTCACCCA motifs detected in file\n",
      "\n",
      "Seq is: AACCCCGATCCAAA\n",
      "0 AACCCCGATCCAAA motifs detected in file\n",
      "\n",
      "Seq is: TCGAGTCTCGCCAG\n",
      "0 TCGAGTCTCGCCAG motifs detected in file\n",
      "\n",
      "Seq is: TTCAATTCAACTTT\n",
      "0 TTCAATTCAACTTT motifs detected in file\n",
      "\n",
      "Seq is: AGAGGATTCACATT\n",
      "0 AGAGGATTCACATT motifs detected in file\n",
      "\n",
      "Seq is: CAACCCACGAAACA\n",
      "0 CAACCCACGAAACA motifs detected in file\n",
      "\n",
      "Seq is: CAATTCGATGTGAG\n",
      "0 CAATTCGATGTGAG motifs detected in file\n",
      "\n",
      "Seq is: ACCAAGCAGGTTCA\n",
      "0 ACCAAGCAGGTTCA motifs detected in file\n",
      "\n",
      "Seq is: CGTGAAAGGTCATG\n",
      "0 CGTGAAAGGTCATG motifs detected in file\n",
      "\n",
      "Seq is: TGAATGGTACAATG\n",
      "0 TGAATGGTACAATG motifs detected in file\n",
      "\n",
      "Seq is: CTGAACAGAGGAGA\n",
      "0 CTGAACAGAGGAGA motifs detected in file\n",
      "\n",
      "Seq is: TATATTTACGCATC\n",
      "0 TATATTTACGCATC motifs detected in file\n",
      "\n",
      "Seq is: CCCAAAATCAATTT\n",
      "0 CCCAAAATCAATTT motifs detected in file\n",
      "\n",
      "Seq is: ACAGGAAACTTCAT\n",
      "0 ACAGGAAACTTCAT motifs detected in file\n",
      "\n",
      "Seq is: ATGAACTCCTCCCA\n",
      "0 ATGAACTCCTCCCA motifs detected in file\n",
      "\n",
      "Seq is: CTGCCCGCCACACA\n",
      "0 CTGCCCGCCACACA motifs detected in file\n",
      "\n",
      "Seq is: TGCGCGAAGTCCCG\n",
      "0 TGCGCGAAGTCCCG motifs detected in file\n",
      "\n",
      "Seq is: GGACAAGTTGATTC\n",
      "0 GGACAAGTTGATTC motifs detected in file\n",
      "\n",
      "Seq is: GTGAGATGAGAATC\n",
      "0 GTGAGATGAGAATC motifs detected in file\n",
      "\n",
      "Seq is: AAATGACGATTTAG\n",
      "0 AAATGACGATTTAG motifs detected in file\n",
      "\n",
      "Seq is: GTGGAAGCTGGGAA\n",
      "0 GTGGAAGCTGGGAA motifs detected in file\n",
      "\n",
      "Seq is: TTCAAAGTATTCCC\n",
      "0 TTCAAAGTATTCCC motifs detected in file\n",
      "\n",
      "Seq is: GATAGCATCAAAAG\n",
      "0 GATAGCATCAAAAG motifs detected in file\n",
      "\n",
      "Seq is: ACCCCATCTTTTAC\n",
      "0 ACCCCATCTTTTAC motifs detected in file\n",
      "\n",
      "Seq is: ATCGGGTAAAGAAC\n",
      "0 ATCGGGTAAAGAAC motifs detected in file\n",
      "\n",
      "Seq is: GGTGGAGTACCCGA\n",
      "0 GGTGGAGTACCCGA motifs detected in file\n",
      "\n",
      "Seq is: TGTGGAACTGGATT\n",
      "0 TGTGGAACTGGATT motifs detected in file\n",
      "\n",
      "Seq is: GCAGAAATGCACCC\n",
      "0 GCAGAAATGCACCC motifs detected in file\n",
      "\n",
      "Seq is: ATGAACGCGAATGA\n",
      "0 ATGAACGCGAATGA motifs detected in file\n",
      "\n",
      "Seq is: AACATCGGAAGTAG\n",
      "0 AACATCGGAAGTAG motifs detected in file\n",
      "\n",
      "Seq is: TCCCAAACGACTTC\n",
      "0 TCCCAAACGACTTC motifs detected in file\n",
      "\n",
      "Seq is: TACCGCCATATATC\n",
      "0 TACCGCCATATATC motifs detected in file\n",
      "\n",
      "Seq is: CGACTCGTTCTCCA\n",
      "0 CGACTCGTTCTCCA motifs detected in file\n",
      "\n",
      "Seq is: CCACTCCAACTAGC\n",
      "0 CCACTCCAACTAGC motifs detected in file\n",
      "\n",
      "Seq is: CCCAAATAGCTAAT\n",
      "0 CCCAAATAGCTAAT motifs detected in file\n",
      "\n",
      "Seq is: GTCTTGTCCGAGGA\n",
      "0 GTCTTGTCCGAGGA motifs detected in file\n",
      "\n",
      "Seq is: GATCTCCCTATACT\n",
      "0 GATCTCCCTATACT motifs detected in file\n",
      "\n",
      "Seq is: TCGTGGCGGTCCAG\n",
      "0 TCGTGGCGGTCCAG motifs detected in file\n",
      "\n",
      "Seq is: GAAACTCAAAATTC\n",
      "0 GAAACTCAAAATTC motifs detected in file\n",
      "\n",
      "Seq is: GACACCGAGCATCA\n",
      "0 GACACCGAGCATCA motifs detected in file\n",
      "\n",
      "Seq is: CAAGACCGCTCGTC\n",
      "0 CAAGACCGCTCGTC motifs detected in file\n",
      "\n",
      "Seq is: AAAACAACTAAATT\n",
      "0 AAAACAACTAAATT motifs detected in file\n",
      "\n",
      "Seq is: GACAACTCCAAATC\n",
      "0 GACAACTCCAAATC motifs detected in file\n",
      "\n",
      "Seq is: CCGATTCGTATCAC\n",
      "0 CCGATTCGTATCAC motifs detected in file\n",
      "\n",
      "Seq is: CCCATAGATTAGGG\n",
      "0 CCCATAGATTAGGG motifs detected in file\n",
      "\n",
      "Seq is: TCAAACCCAGAATC\n",
      "0 TCAAACCCAGAATC motifs detected in file\n",
      "\n",
      "Seq is: GATAACACACAAGT\n",
      "0 GATAACACACAAGT motifs detected in file\n",
      "\n",
      "Seq is: CTATACAATTTCGG\n",
      "0 CTATACAATTTCGG motifs detected in file\n",
      "\n",
      "Seq is: TGAAAGCATCAGAT\n",
      "0 TGAAAGCATCAGAT motifs detected in file\n",
      "\n",
      "Seq is: AAAACACAGTTGAC\n",
      "0 AAAACACAGTTGAC motifs detected in file\n",
      "\n",
      "Seq is: CTCAAAAAATGTTG\n",
      "0 CTCAAAAAATGTTG motifs detected in file\n",
      "\n",
      "Seq is: AGTCCACTCAAAAT\n",
      "0 AGTCCACTCAAAAT motifs detected in file\n",
      "\n",
      "Seq is: CGAAATAACCCCTC\n",
      "0 CGAAATAACCCCTC motifs detected in file\n",
      "\n",
      "Seq is: CCCCACTAGATTTT\n",
      "0 CCCCACTAGATTTT motifs detected in file\n",
      "\n",
      "Seq is: GTGAGAAATGCGGC\n",
      "0 GTGAGAAATGCGGC motifs detected in file\n",
      "\n",
      "Seq is: CGAAGATAACTGAA\n",
      "0 CGAAGATAACTGAA motifs detected in file\n",
      "\n",
      "Seq is: CCCAATTAGCACAC\n",
      "0 CCCAATTAGCACAC motifs detected in file\n",
      "\n",
      "Seq is: TGGCGCCACCGACC\n",
      "0 TGGCGCCACCGACC motifs detected in file\n",
      "\n",
      "Seq is: GTGCTTTAACTGGT\n",
      "0 GTGCTTTAACTGGT motifs detected in file\n",
      "\n",
      "Seq is: TGGGACCCCAATCA\n",
      "0 TGGGACCCCAATCA motifs detected in file\n",
      "\n",
      "Seq is: TATCGAATCAATTT\n",
      "0 TATCGAATCAATTT motifs detected in file\n",
      "\n",
      "Seq is: GAAAGCAAATAACG\n",
      "0 GAAAGCAAATAACG motifs detected in file\n",
      "\n",
      "Seq is: ATTTCCAATTAGTG\n",
      "0 ATTTCCAATTAGTG motifs detected in file\n",
      "\n",
      "Seq is: ACGACAAAGGGCAC\n",
      "0 ACGACAAAGGGCAC motifs detected in file\n",
      "\n",
      "Seq is: CCCCACACCGATCC\n",
      "0 CCCCACACCGATCC motifs detected in file\n",
      "\n",
      "Seq is: GATTCGCAAAACTC\n",
      "0 GATTCGCAAAACTC motifs detected in file\n",
      "\n",
      "Seq is: GACGTAACGACGCA\n",
      "0 GACGTAACGACGCA motifs detected in file\n",
      "\n",
      "Seq is: CGAACTACGAACAA\n",
      "0 CGAACTACGAACAA motifs detected in file\n",
      "\n",
      "Seq is: GGAAAACTAGCACC\n",
      "0 GGAAAACTAGCACC motifs detected in file\n",
      "\n",
      "Seq is: ACTAAATCTCCCAC\n",
      "0 ACTAAATCTCCCAC motifs detected in file\n",
      "\n",
      "Seq is: TTGAGTGACCTTGC\n",
      "0 TTGAGTGACCTTGC motifs detected in file\n",
      "\n",
      "Seq is: CCCTACGAACGTAC\n",
      "0 CCCTACGAACGTAC motifs detected in file\n",
      "\n",
      "Seq is: ACATTAGAATGATT\n",
      "0 ACATTAGAATGATT motifs detected in file\n",
      "\n",
      "Seq is: CTCGATTATCTTTC\n",
      "0 CTCGATTATCTTTC motifs detected in file\n",
      "\n",
      "Seq is: CGATCCTGGTTTGG\n",
      "0 CGATCCTGGTTTGG motifs detected in file\n",
      "\n",
      "Seq is: ATAAACAAATACCC\n",
      "0 ATAAACAAATACCC motifs detected in file\n",
      "\n",
      "Seq is: CACTACACTCGGGC\n",
      "0 CACTACACTCGGGC motifs detected in file\n",
      "\n",
      "Seq is: AGATTTGTAAAAGT\n",
      "0 AGATTTGTAAAAGT motifs detected in file\n",
      "\n",
      "Seq is: CACATAACGAAACT\n",
      "0 CACATAACGAAACT motifs detected in file\n",
      "\n",
      "Seq is: CCTCCATTCGATGT\n",
      "0 CCTCCATTCGATGT motifs detected in file\n",
      "\n",
      "Seq is: ATTTGACACCCCCC\n",
      "0 ATTTGACACCCCCC motifs detected in file\n",
      "\n",
      "Seq is: ACAGAAACCCCAAA\n",
      "0 ACAGAAACCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: CCAACCAAGCCCCA\n",
      "0 CCAACCAAGCCCCA motifs detected in file\n",
      "\n",
      "Seq is: TACCCCAATCGTCG\n",
      "0 TACCCCAATCGTCG motifs detected in file\n",
      "\n",
      "Seq is: CCACACACACACGA\n",
      "0 CCACACACACACGA motifs detected in file\n",
      "\n",
      "Seq is: CCCATCCTACCCGC\n",
      "0 CCCATCCTACCCGC motifs detected in file\n",
      "\n",
      "Seq is: CCCCAACATAACTA\n",
      "0 CCCCAACATAACTA motifs detected in file\n",
      "\n",
      "Seq is: CTCTAATCATCATA\n",
      "0 CTCTAATCATCATA motifs detected in file\n",
      "\n",
      "Seq is: CCCTCACCCTCCCC\n",
      "0 CCCTCACCCTCCCC motifs detected in file\n",
      "\n",
      "Seq is: TTTTATTACACAAT\n",
      "0 TTTTATTACACAAT motifs detected in file\n",
      "\n",
      "Seq is: CAACCCCCCACTGA\n",
      "0 CAACCCCCCACTGA motifs detected in file\n",
      "\n",
      "Seq is: CAATTTTCACGTAT\n",
      "0 CAATTTTCACGTAT motifs detected in file\n",
      "\n",
      "Seq is: GGCGGTTTTCTATT\n",
      "0 GGCGGTTTTCTATT motifs detected in file\n",
      "\n",
      "Seq is: TTAAACTTTAGACC\n",
      "0 TTAAACTTTAGACC motifs detected in file\n",
      "\n",
      "Seq is: AATCCGACCACAAT\n",
      "0 AATCCGACCACAAT motifs detected in file\n",
      "\n",
      "Seq is: CCCAAGAAACAAAA\n",
      "0 CCCAAGAAACAAAA motifs detected in file\n",
      "\n",
      "Seq is: ACCCCAAACCGTCT\n",
      "0 ACCCCAAACCGTCT motifs detected in file\n",
      "\n",
      "Seq is: CTACACAAATTCAC\n",
      "0 CTACACAAATTCAC motifs detected in file\n",
      "\n",
      "Seq is: GACACCGGTCTTCG\n",
      "0 GACACCGGTCTTCG motifs detected in file\n",
      "\n",
      "Seq is: CCCCCTCCCCCCCA\n",
      "0 CCCCCTCCCCCCCA motifs detected in file\n",
      "\n",
      "Seq is: AACCACCTTTAAAA\n",
      "0 AACCACCTTTAAAA motifs detected in file\n",
      "\n",
      "Seq is: AACAATACTACAGA\n",
      "0 AACAATACTACAGA motifs detected in file\n",
      "\n",
      "Seq is: CACACCTTTCACCG\n",
      "0 CACACCTTTCACCG motifs detected in file\n",
      "\n",
      "Seq is: ACACGTCTGTAAGT\n",
      "0 ACACGTCTGTAAGT motifs detected in file\n",
      "\n",
      "Seq is: TAACAATAATAATA\n",
      "0 TAACAATAATAATA motifs detected in file\n",
      "\n",
      "Seq is: CAGGATGTTCGTAA\n",
      "0 CAGGATGTTCGTAA motifs detected in file\n",
      "\n",
      "Seq is: TTAATTAATTGTGT\n",
      "0 TTAATTAATTGTGT motifs detected in file\n",
      "\n",
      "Seq is: GAAATCATTCATAC\n",
      "0 GAAATCATTCATAC motifs detected in file\n",
      "\n",
      "Seq is: AAGCGGACATTATA\n",
      "0 AAGCGGACATTATA motifs detected in file\n",
      "\n",
      "Seq is: ACTTGCATCCACGC\n",
      "0 ACTTGCATCCACGC motifs detected in file\n",
      "\n",
      "Seq is: TATTTATTATCCTA\n",
      "0 TATTTATTATCCTA motifs detected in file\n",
      "\n",
      "Seq is: CTCCGTCCTTAGTT\n",
      "0 CTCCGTCCTTAGTT motifs detected in file\n",
      "\n",
      "Seq is: TCTGTCTATGACGC\n",
      "0 TCTGTCTATGACGC motifs detected in file\n",
      "\n",
      "Seq is: TGTATCCCACGAGG\n",
      "0 TGTATCCCACGAGG motifs detected in file\n",
      "\n",
      "Seq is: CCGAGGTCGCAGAG\n",
      "0 CCGAGGTCGCAGAG motifs detected in file\n",
      "\n",
      "Seq is: CGTTACGATAGCGC\n",
      "0 CGTTACGATAGCGC motifs detected in file\n",
      "\n",
      "Seq is: ACGTATGGGGGGTC\n",
      "0 ACGTATGGGGGGTC motifs detected in file\n",
      "\n",
      "Seq is: TGCTTTTATGGTTT\n",
      "0 TGCTTTTATGGTTT motifs detected in file\n",
      "\n",
      "Seq is: ACGTACCTCTCGAG\n",
      "0 ACGTACCTCTCGAG motifs detected in file\n",
      "\n",
      "Seq is: GGCACTCACCAATT\n",
      "0 GGCACTCACCAATT motifs detected in file\n",
      "\n",
      "Seq is: ATCCCACTATCTGG\n",
      "0 ATCCCACTATCTGG motifs detected in file\n",
      "\n",
      "Seq is: GTAGCACTACAGAAT\n",
      "0 GTAGCACTACAGAAT motifs detected in file\n",
      "\n",
      "Seq is: AAATTCCCCTTGCAC\n",
      "0 AAATTCCCCTTGCAC motifs detected in file\n",
      "\n",
      "Seq is: ACCCGATAAATCCGA\n",
      "0 ACCCGATAAATCCGA motifs detected in file\n",
      "\n",
      "Seq is: AATACTGGGACTTCA\n",
      "0 AATACTGGGACTTCA motifs detected in file\n",
      "\n",
      "Seq is: TCCTTGGTCTACAGC\n",
      "0 TCCTTGGTCTACAGC motifs detected in file\n",
      "\n",
      "Seq is: CTATGTCAAGTGAAA\n",
      "0 CTATGTCAAGTGAAA motifs detected in file\n",
      "\n",
      "Seq is: TCGATGGGGGTTCAC\n",
      "0 TCGATGGGGGTTCAC motifs detected in file\n",
      "\n",
      "Seq is: AATACCCGGGCCTCG\n",
      "0 AATACCCGGGCCTCG motifs detected in file\n",
      "\n",
      "Seq is: CTCCTCTCATCGTGA\n",
      "0 CTCCTCTCATCGTGA motifs detected in file\n",
      "\n",
      "Seq is: GAACACGCCCTATAA\n",
      "0 GAACACGCCCTATAA motifs detected in file\n",
      "\n",
      "Seq is: CTAAAGTGCCTCCTA\n",
      "0 CTAAAGTGCCTCCTA motifs detected in file\n",
      "\n",
      "Seq is: CCACCAGTTCCCTGG\n",
      "0 CCACCAGTTCCCTGG motifs detected in file\n",
      "\n",
      "Seq is: GGATAGACTCCCCCC\n",
      "0 GGATAGACTCCCCCC motifs detected in file\n",
      "\n",
      "Seq is: AGTAGGTACCCCTGC\n",
      "0 AGTAGGTACCCCTGC motifs detected in file\n",
      "\n",
      "Seq is: TCTTCCCTAAACTGA\n",
      "0 TCTTCCCTAAACTGA motifs detected in file\n",
      "\n",
      "Seq is: CATTACACGATACAT\n",
      "0 CATTACACGATACAT motifs detected in file\n",
      "\n",
      "Seq is: GCCATTTACCGAAAT\n",
      "0 GCCATTTACCGAAAT motifs detected in file\n",
      "\n",
      "Seq is: ACATGATACATGACA\n",
      "0 ACATGATACATGACA motifs detected in file\n",
      "\n",
      "Seq is: ATTCCCACCCATCCA\n",
      "0 ATTCCCACCCATCCA motifs detected in file\n",
      "\n",
      "Seq is: AACAACCATAGGATC\n",
      "0 AACAACCATAGGATC motifs detected in file\n",
      "\n",
      "Seq is: ACCCACTCCCCACCG\n",
      "0 ACCCACTCCCCACCG motifs detected in file\n",
      "\n",
      "Seq is: AAACCTCAACGTCAA\n",
      "0 AAACCTCAACGTCAA motifs detected in file\n",
      "\n",
      "Seq is: CTACACACTATCAAC\n",
      "0 CTACACACTATCAAC motifs detected in file\n",
      "\n",
      "Seq is: TCCCAACTAACGACA\n",
      "0 TCCCAACTAACGACA motifs detected in file\n",
      "\n",
      "Seq is: TGAACGAACATTCGT\n",
      "0 TGAACGAACATTCGT motifs detected in file\n",
      "\n",
      "Seq is: ACCCCTCCCCCAAAA\n",
      "0 ACCCCTCCCCCAAAA motifs detected in file\n",
      "\n",
      "Seq is: CTACACCTAACCCAA\n",
      "0 CTACACCTAACCCAA motifs detected in file\n",
      "\n",
      "Seq is: AAATACATGATGTCC\n",
      "0 AAATACATGATGTCC motifs detected in file\n",
      "\n",
      "Seq is: ACCAGTTCATAAATA\n",
      "0 ACCAGTTCATAAATA motifs detected in file\n",
      "\n",
      "Seq is: CCATGGCATGTTATA\n",
      "0 CCATGGCATGTTATA motifs detected in file\n",
      "\n",
      "Seq is: AGTACCACCGACCGT\n",
      "0 AGTACCACCGACCGT motifs detected in file\n",
      "\n",
      "Seq is: CATTACATGCTTTAT\n",
      "0 CATTACATGCTTTAT motifs detected in file\n",
      "\n",
      "Seq is: GTATCGCCAACAACT\n",
      "0 GTATCGCCAACAACT motifs detected in file\n",
      "\n",
      "Seq is: ACCCACTCAGTTATG\n",
      "0 ACCCACTCAGTTATG motifs detected in file\n",
      "\n",
      "Seq is: AACCCACCATGGGTT\n",
      "0 AACCCACCATGGGTT motifs detected in file\n",
      "\n",
      "Seq is: TAGACGAAGGGGTAC\n",
      "0 TAGACGAAGGGGTAC motifs detected in file\n",
      "\n",
      "Seq is: TTTCTTGTCTCTTAT\n",
      "0 TTTCTTGTCTCTTAT motifs detected in file\n",
      "\n",
      "Seq is: CAAATTTAATCTTAG\n",
      "0 CAAATTTAATCTTAG motifs detected in file\n",
      "\n",
      "Seq is: AATCGAAACCCACGA\n",
      "0 AATCGAAACCCACGA motifs detected in file\n",
      "\n",
      "Seq is: TTACCACCTCAATTT\n",
      "0 TTACCACCTCAATTT motifs detected in file\n",
      "\n",
      "Seq is: CTGAAAAAGAGACTA\n",
      "0 CTGAAAAAGAGACTA motifs detected in file\n",
      "\n",
      "Seq is: AACAGGAACCTTTTT\n",
      "0 AACAGGAACCTTTTT motifs detected in file\n",
      "\n",
      "Seq is: CCAAAAGTAGAGGCC\n",
      "0 CCAAAAGTAGAGGCC motifs detected in file\n",
      "\n",
      "Seq is: AAATGTTCTGACCAC\n",
      "0 AAATGTTCTGACCAC motifs detected in file\n",
      "\n",
      "Seq is: ATAATCAAATATGAT\n",
      "0 ATAATCAAATATGAT motifs detected in file\n",
      "\n",
      "Seq is: GTTCCTGTCCGGGTA\n",
      "0 GTTCCTGTCCGGGTA motifs detected in file\n",
      "\n",
      "Seq is: AACTCATAAAACAAA\n",
      "0 AACTCATAAAACAAA motifs detected in file\n",
      "\n",
      "Seq is: AGTTAATCCCTCTAT\n",
      "0 AGTTAATCCCTCTAT motifs detected in file\n",
      "\n",
      "Seq is: CAACCATAATCCTAA\n",
      "0 CAACCATAATCCTAA motifs detected in file\n",
      "\n",
      "Seq is: TCCTAACAACACTTC\n",
      "0 TCCTAACAACACTTC motifs detected in file\n",
      "\n",
      "Seq is: ATATCATGCCTACGA\n",
      "0 ATATCATGCCTACGA motifs detected in file\n",
      "\n",
      "Seq is: TGAACAGGTTACTAC\n",
      "0 TGAACAGGTTACTAC motifs detected in file\n",
      "\n",
      "Seq is: CATTTTCCCATCGAA\n",
      "0 CATTTTCCCATCGAA motifs detected in file\n",
      "\n",
      "Seq is: TGACCAACAGGAGGC\n",
      "0 TGACCAACAGGAGGC motifs detected in file\n",
      "\n",
      "Seq is: TAAGTCCAATCTTAC\n",
      "0 TAAGTCCAATCTTAC motifs detected in file\n",
      "\n",
      "Seq is: TCCTCCAGACGCCGA\n",
      "0 TCCTCCAGACGCCGA motifs detected in file\n",
      "\n",
      "Seq is: TCCTCAGTTATTTCA\n",
      "0 TCCTCAGTTATTTCA motifs detected in file\n",
      "\n",
      "Seq is: CTAACCGAATCACCC\n",
      "0 CTAACCGAATCACCC motifs detected in file\n",
      "\n",
      "Seq is: GCTTTATAATACGAA\n",
      "0 GCTTTATAATACGAA motifs detected in file\n",
      "\n",
      "Seq is: ACAACAAACCTATAT\n",
      "0 ACAACAAACCTATAT motifs detected in file\n",
      "\n",
      "Seq is: ACCTCCTACCCCTAA\n",
      "0 ACCTCCTACCCCTAA motifs detected in file\n",
      "\n",
      "Seq is: TAACGATCCTACTCC\n",
      "0 TAACGATCCTACTCC motifs detected in file\n",
      "\n",
      "Seq is: TACCTATCATTATCC\n",
      "0 TACCTATCATTATCC motifs detected in file\n",
      "\n",
      "Seq is: CGTTCCTGCGGAGGA\n",
      "0 CGTTCCTGCGGAGGA motifs detected in file\n",
      "\n",
      "Seq is: TCAAACAATCCCTGC\n",
      "0 TCAAACAATCCCTGC motifs detected in file\n",
      "\n",
      "Seq is: CTAGCCTCTTAACAC\n",
      "0 CTAGCCTCTTAACAC motifs detected in file\n",
      "\n",
      "Seq is: ATCCGCTTATCCTTT\n",
      "0 ATCCGCTTATCCTTT motifs detected in file\n",
      "\n",
      "Seq is: ATAGTAAGCCCGAAC\n",
      "0 ATAGTAAGCCCGAAC motifs detected in file\n",
      "\n",
      "Seq is: TACACCCCTCCCCAC\n",
      "0 TACACCCCTCCCCAC motifs detected in file\n",
      "\n",
      "Seq is: AAATTCCCCAACCGA\n",
      "0 AAATTCCCCAACCGA motifs detected in file\n",
      "\n",
      "Seq is: TCCCATATTAACAGA\n",
      "0 TCCCATATTAACAGA motifs detected in file\n",
      "\n",
      "Seq is: CCCAGCGGATCCTCC\n",
      "0 CCCAGCGGATCCTCC motifs detected in file\n",
      "\n",
      "Seq is: AGACCACTCTTATCA\n",
      "0 AGACCACTCTTATCA motifs detected in file\n",
      "\n",
      "Seq is: CAATTACAGTAATTC\n",
      "0 CAATTACAGTAATTC motifs detected in file\n",
      "\n",
      "Seq is: CTCTCTTCCTTCTCT\n",
      "0 CTCTCTTCCTTCTCT motifs detected in file\n",
      "\n",
      "Seq is: TCATTCGGCTCCCGC\n",
      "0 TCATTCGGCTCCCGC motifs detected in file\n",
      "\n",
      "Seq is: AGAAACTAACACATC\n",
      "0 AGAAACTAACACATC motifs detected in file\n",
      "\n",
      "Seq is: ATTCCCACCTTCCAC\n",
      "0 ATTCCCACCTTCCAC motifs detected in file\n",
      "\n",
      "Seq is: TAAAATAGCCTTACC\n",
      "0 TAAAATAGCCTTACC motifs detected in file\n",
      "\n",
      "Seq is: CTCCACTAAGGATCC\n",
      "0 CTCCACTAAGGATCC motifs detected in file\n",
      "\n",
      "Seq is: CCCAACAAACTAGGG\n",
      "0 CCCAACAAACTAGGG motifs detected in file\n",
      "\n",
      "Seq is: CAAAGCACGTTCTTA\n",
      "0 CAAAGCACGTTCTTA motifs detected in file\n",
      "\n",
      "Seq is: TCCTCCACCTCACAA\n",
      "0 TCCTCCACCTCACAA motifs detected in file\n",
      "\n",
      "Seq is: CGATCCCGACGTTAT\n",
      "0 CGATCCCGACGTTAT motifs detected in file\n",
      "\n",
      "Seq is: TACTTCCCGTTCTAC\n",
      "0 TACTTCCCGTTCTAC motifs detected in file\n",
      "\n",
      "Seq is: TTCACTTTCCATTTC\n",
      "0 TTCACTTTCCATTTC motifs detected in file\n",
      "\n",
      "Seq is: TTAGCACACTCCCAC\n",
      "0 TTAGCACACTCCCAC motifs detected in file\n",
      "\n",
      "Seq is: CCTGACAGATGACTC\n",
      "0 CCTGACAGATGACTC motifs detected in file\n",
      "\n",
      "Seq is: ATCGGAGGAGTCTAA\n",
      "0 ATCGGAGGAGTCTAA motifs detected in file\n",
      "\n",
      "Seq is: GTAACTTGATCCAGA\n",
      "0 GTAACTTGATCCAGA motifs detected in file\n",
      "\n",
      "Seq is: CAGGGTTACATACCC\n",
      "0 CAGGGTTACATACCC motifs detected in file\n",
      "\n",
      "Seq is: TACCGCCTATCATTC\n",
      "0 TACCGCCTATCATTC motifs detected in file\n",
      "\n",
      "Seq is: AAACATTAATGACAC\n",
      "0 AAACATTAATGACAC motifs detected in file\n",
      "\n",
      "Seq is: CGGGGAGTCTTACTA\n",
      "0 CGGGGAGTCTTACTA motifs detected in file\n",
      "\n",
      "Seq is: TAAACCGGAGTGCCC\n",
      "0 TAAACCGGAGTGCCC motifs detected in file\n",
      "\n",
      "Seq is: TCCTGTATCGGATAC\n",
      "0 TCCTGTATCGGATAC motifs detected in file\n",
      "\n",
      "Seq is: TTCCGACAACGATAT\n",
      "0 TTCCGACAACGATAT motifs detected in file\n",
      "\n",
      "Seq is: CAACGTTCGTCCTCC\n",
      "0 CAACGTTCGTCCTCC motifs detected in file\n",
      "\n",
      "Seq is: TATTACGGCTACAAA\n",
      "0 TATTACGGCTACAAA motifs detected in file\n",
      "\n",
      "Seq is: GTCCAAAGACTCATC\n",
      "0 GTCCAAAGACTCATC motifs detected in file\n",
      "\n",
      "Seq is: TCTTTACTAGGCATT\n",
      "0 TCTTTACTAGGCATT motifs detected in file\n",
      "\n",
      "Seq is: ATATCCGGAGCGGGC\n",
      "0 ATATCCGGAGCGGGC motifs detected in file\n",
      "\n",
      "Seq is: TACACATCCTTCTCC\n",
      "0 TACACATCCTTCTCC motifs detected in file\n",
      "\n",
      "Seq is: GTCTATTTCTTATAA\n",
      "0 GTCTATTTCTTATAA motifs detected in file\n",
      "\n",
      "Seq is: CTCCGCGGTAACCGC\n",
      "0 CTCCGCGGTAACCGC motifs detected in file\n",
      "\n",
      "Seq is: ACTTCCATCGCCTAC\n",
      "0 ACTTCCATCGCCTAC motifs detected in file\n",
      "\n",
      "Seq is: TAAGTCGGTATTAAA\n",
      "0 TAAGTCGGTATTAAA motifs detected in file\n",
      "\n",
      "Seq is: TGCAGAGCTCACTAC\n",
      "0 TGCAGAGCTCACTAC motifs detected in file\n",
      "\n",
      "Seq is: ACCCGCTAACTACTT\n",
      "0 ACCCGCTAACTACTT motifs detected in file\n",
      "\n",
      "Seq is: TTCCGCCAACTCCGC\n",
      "0 TTCCGCCAACTCCGC motifs detected in file\n",
      "\n",
      "Seq is: AGACCACTCATCACG\n",
      "0 AGACCACTCATCACG motifs detected in file\n",
      "\n",
      "Seq is: TACCGATCCTTATCA\n",
      "0 TACCGATCCTTATCA motifs detected in file\n",
      "\n",
      "Seq is: GGACACCACTAAACC\n",
      "0 GGACACCACTAAACC motifs detected in file\n",
      "\n",
      "Seq is: TCCTAGTCCGTCCGC\n",
      "0 TCCTAGTCCGTCCGC motifs detected in file\n",
      "\n",
      "Seq is: GGTTCCTCACTCGGC\n",
      "0 GGTTCCTCACTCGGC motifs detected in file\n",
      "\n",
      "Seq is: TTCAAAGTAGTACGC\n",
      "0 TTCAAAGTAGTACGC motifs detected in file\n",
      "\n",
      "Seq is: CTCTACAACCTACCC\n",
      "0 CTCTACAACCTACCC motifs detected in file\n",
      "\n",
      "Seq is: CACCCCTCCAGCTAC\n",
      "0 CACCCCTCCAGCTAC motifs detected in file\n",
      "\n",
      "Seq is: TTACTCACCAATTAA\n",
      "0 TTACTCACCAATTAA motifs detected in file\n",
      "\n",
      "Seq is: TTAAAATAATCCCCC\n",
      "0 TTAAAATAATCCCCC motifs detected in file\n",
      "\n",
      "Seq is: AATCAAAACGCATAA\n",
      "0 AATCAAAACGCATAA motifs detected in file\n",
      "\n",
      "Seq is: CCCCAGTAACCACAA\n",
      "0 CCCCAGTAACCACAA motifs detected in file\n",
      "\n",
      "Seq is: GAACATCAACTTTAT\n",
      "0 GAACATCAACTTTAT motifs detected in file\n",
      "\n",
      "Seq is: GTTGCTACCAAAAAG\n",
      "0 GTTGCTACCAAAAAG motifs detected in file\n",
      "\n",
      "Seq is: TATAGTAACCAGCAC\n",
      "0 TATAGTAACCAGCAC motifs detected in file\n",
      "\n",
      "Seq is: CAACATCAGGCACGC\n",
      "0 CAACATCAGGCACGC motifs detected in file\n",
      "\n",
      "Seq is: TCTTATTACTACATA\n",
      "0 TCTTATTACTACATA motifs detected in file\n",
      "\n",
      "Seq is: CGAAACAAAGACAAC\n",
      "0 CGAAACAAAGACAAC motifs detected in file\n",
      "\n",
      "Seq is: TCACACCCAAATCAT\n",
      "0 TCACACCCAAATCAT motifs detected in file\n",
      "\n",
      "Seq is: TACCCCAAACACCCC\n",
      "0 TACCCCAAACACCCC motifs detected in file\n",
      "\n",
      "Seq is: AAAAGAAGATTCGGA\n",
      "0 AAAAGAAGATTCGGA motifs detected in file\n",
      "\n",
      "Seq is: AGAGGATAAATACCC\n",
      "0 AGAGGATAAATACCC motifs detected in file\n",
      "\n",
      "Seq is: CCAAATCATAACTAA\n",
      "0 CCAAATCATAACTAA motifs detected in file\n",
      "\n",
      "Seq is: CAATCGCCACACCAG\n",
      "0 CAATCGCCACACCAG motifs detected in file\n",
      "\n",
      "Seq is: CCCACACAATAATAA\n",
      "0 CCCACACAATAATAA motifs detected in file\n",
      "\n",
      "Seq is: GACTTAAAACCCCCT\n",
      "0 GACTTAAAACCCCCT motifs detected in file\n",
      "\n",
      "Seq is: CCAATATACCCAAAT\n",
      "0 CCAATATACCCAAAT motifs detected in file\n",
      "\n",
      "Seq is: TATCAAAAAAATTAA\n",
      "0 TATCAAAAAAATTAA motifs detected in file\n",
      "\n",
      "Seq is: ATAAATCCCCCTTAC\n",
      "0 ATAAATCCCCCTTAC motifs detected in file\n",
      "\n",
      "Seq is: TACCAACAGAAACCT\n",
      "0 TACCAACAGAAACCT motifs detected in file\n",
      "\n",
      "Seq is: ATATGATGTCGCTAC\n",
      "0 ATATGATGTCGCTAC motifs detected in file\n",
      "\n",
      "Seq is: CGATAACTCCTCATA\n",
      "0 CGATAACTCCTCATA motifs detected in file\n",
      "\n",
      "Seq is: GGACTCCGTACCCCC\n",
      "0 GGACTCCGTACCCCC motifs detected in file\n",
      "\n",
      "Seq is: AGTCCCCAACTCCAG\n",
      "0 AGTCCCCAACTCCAG motifs detected in file\n",
      "\n",
      "Seq is: AACCACTCACAAAAT\n",
      "0 AACCACTCACAAAAT motifs detected in file\n",
      "\n",
      "Seq is: CACCCCAATCGCTAC\n",
      "0 CACCCCAATCGCTAC motifs detected in file\n",
      "\n",
      "Seq is: CTCCATCCTAACCAC\n",
      "0 CTCCATCCTAACCAC motifs detected in file\n",
      "\n",
      "Seq is: GACACCCACTTTCTC\n",
      "0 GACACCCACTTTCTC motifs detected in file\n",
      "\n",
      "Seq is: ATACTACCCCACCAC\n",
      "0 ATACTACCCCACCAC motifs detected in file\n",
      "\n",
      "Seq is: CAACACCATTTGAAA\n",
      "0 CAACACCATTTGAAA motifs detected in file\n",
      "\n",
      "Seq is: TTATCACATCCTTCG\n",
      "0 TTATCACATCCTTCG motifs detected in file\n",
      "\n",
      "Seq is: ACTTATTAAATACTT\n",
      "0 ACTTATTAAATACTT motifs detected in file\n",
      "\n",
      "Seq is: CCTCTCCCCAGTCCC\n",
      "0 CCTCTCCCCAGTCCC motifs detected in file\n",
      "\n",
      "Seq is: AACTAAGCCCTCCTA\n",
      "0 AACTAAGCCCTCCTA motifs detected in file\n",
      "\n",
      "Seq is: GGATAACCACGCCCC\n",
      "0 GGATAACCACGCCCC motifs detected in file\n",
      "\n",
      "Seq is: CGAAACATACTAATA\n",
      "0 CGAAACATACTAATA motifs detected in file\n",
      "\n",
      "Seq is: CCCGCAACTAATCAT\n",
      "0 CCCGCAACTAATCAT motifs detected in file\n",
      "\n",
      "Seq is: CATCAATGACCAACT\n",
      "0 CATCAATGACCAACT motifs detected in file\n",
      "\n",
      "Seq is: TGTAACAAACAACCA\n",
      "0 TGTAACAAACAACCA motifs detected in file\n",
      "\n",
      "Seq is: CATATATAACATTAA\n",
      "0 CATATATAACATTAA motifs detected in file\n",
      "\n",
      "Seq is: CTCTAACGAGCCCCC\n",
      "0 CTCTAACGAGCCCCC motifs detected in file\n",
      "\n",
      "Seq is: TTATCCAATACACTA\n",
      "0 TTATCCAATACACTA motifs detected in file\n",
      "\n",
      "Seq is: ATCCTCATCCCAATC\n",
      "0 ATCCTCATCCCAATC motifs detected in file\n",
      "\n",
      "Seq is: CTACTCACCCTTCTT\n",
      "0 CTACTCACCCTTCTT motifs detected in file\n",
      "\n",
      "Seq is: CTTTCTCTCCTTCAT\n",
      "0 CTTTCTCTCCTTCAT motifs detected in file\n",
      "\n",
      "Seq is: TTCAAATTAATACGG\n",
      "0 TTCAAATTAATACGG motifs detected in file\n",
      "\n",
      "Seq is: AAAAACCCAACTCCA\n",
      "0 AAAAACCCAACTCCA motifs detected in file\n",
      "\n",
      "Seq is: CTACTACCTCCACCT\n",
      "0 CTACTACCTCCACCT motifs detected in file\n",
      "\n",
      "Seq is: CTAAACCACGACACT\n",
      "0 CTAAACCACGACACT motifs detected in file\n",
      "\n",
      "Seq is: TTAACAAAATCCATT\n",
      "0 TTAACAAAATCCATT motifs detected in file\n",
      "\n",
      "Seq is: ATCGAAAAGATCAGT\n",
      "0 ATCGAAAAGATCAGT motifs detected in file\n",
      "\n",
      "Seq is: CCAATCCAGATCCTC\n",
      "0 CCAATCCAGATCCTC motifs detected in file\n",
      "\n",
      "Seq is: CTCATCCCCGTCCAA\n",
      "0 CTCATCCCCGTCCAA motifs detected in file\n",
      "\n",
      "Seq is: AACCGAGCATTCTTC\n",
      "0 AACCGAGCATTCTTC motifs detected in file\n",
      "\n",
      "Seq is: CGGATCTATCCCCTA\n",
      "0 CGGATCTATCCCCTA motifs detected in file\n",
      "\n",
      "Seq is: ACACGCCACACACTA\n",
      "0 ACACGCCACACACTA motifs detected in file\n",
      "\n",
      "Seq is: CGATCCCATCTTAGG\n",
      "0 CGATCCCATCTTAGG motifs detected in file\n",
      "\n",
      "Seq is: CTCATACAACCTCTT\n",
      "0 CTCATACAACCTCTT motifs detected in file\n",
      "\n",
      "Seq is: TATTTTACACGTATC\n",
      "0 TATTTTACACGTATC motifs detected in file\n",
      "\n",
      "Seq is: ACCCCTAAAATAAAA\n",
      "0 ACCCCTAAAATAAAA motifs detected in file\n",
      "\n",
      "Seq is: TTCAAACAACCAATC\n",
      "0 TTCAAACAACCAATC motifs detected in file\n",
      "\n",
      "Seq is: CATCAACTCCAGATC\n",
      "0 CATCAACTCCAGATC motifs detected in file\n",
      "\n",
      "Seq is: CCGACAATCTTCAGG\n",
      "0 CCGACAATCTTCAGG motifs detected in file\n",
      "\n",
      "Seq is: ATCCTTTCACTGTCG\n",
      "0 ATCCTTTCACTGTCG motifs detected in file\n",
      "\n",
      "Seq is: CTCCCGACACTCAAA\n",
      "0 CTCCCGACACTCAAA motifs detected in file\n",
      "\n",
      "Seq is: ATCCATCTCCCCCTA\n",
      "0 ATCCATCTCCCCCTA motifs detected in file\n",
      "\n",
      "Seq is: ACAACAAACCTTCCC\n",
      "0 ACAACAAACCTTCCC motifs detected in file\n",
      "\n",
      "Seq is: CCTACGCCCCCTTTA\n",
      "0 CCTACGCCCCCTTTA motifs detected in file\n",
      "\n",
      "Seq is: CAACAATCATTACTC\n",
      "0 CAACAATCATTACTC motifs detected in file\n",
      "\n",
      "Seq is: TTTAGGACGCTTATC\n",
      "0 TTTAGGACGCTTATC motifs detected in file\n",
      "\n",
      "Seq is: CGAAGGCCGACGGTC\n",
      "0 CGAAGGCCGACGGTC motifs detected in file\n",
      "\n",
      "Seq is: CGCAAATTACCCCAA\n",
      "0 CGCAAATTACCCCAA motifs detected in file\n",
      "\n",
      "Seq is: ATCATCCCACCCCAA\n",
      "0 ATCATCCCACCCCAA motifs detected in file\n",
      "\n",
      "Seq is: TAAAAGCAATTACAA\n",
      "0 TAAAAGCAATTACAA motifs detected in file\n",
      "\n",
      "Seq is: TCATTCCCACCCCTT\n",
      "0 TCATTCCCACCCCTT motifs detected in file\n",
      "\n",
      "Seq is: CGCTCCAACTGGACA\n",
      "0 CGCTCCAACTGGACA motifs detected in file\n",
      "\n",
      "Seq is: ATCCCACTCTTCTTA\n",
      "0 ATCCCACTCTTCTTA motifs detected in file\n",
      "\n",
      "Seq is: ATAAGCTCACGATAT\n",
      "0 ATAAGCTCACGATAT motifs detected in file\n",
      "\n",
      "Seq is: CCGCGAACAGTCCCT\n",
      "0 CCGCGAACAGTCCCT motifs detected in file\n",
      "\n",
      "Seq is: CCATCGCTACTCTCA\n",
      "0 CCATCGCTACTCTCA motifs detected in file\n",
      "\n",
      "Seq is: TTATCTATCCCGAGT\n",
      "0 TTATCTATCCCGAGT motifs detected in file\n",
      "\n",
      "Seq is: CCGCAAACACATACT\n",
      "0 CCGCAAACACATACT motifs detected in file\n",
      "\n",
      "Seq is: ATACAAACGCCAAAG\n",
      "0 ATACAAACGCCAAAG motifs detected in file\n",
      "\n",
      "Seq is: CTACTACACCAGAAA\n",
      "0 CTACTACACCAGAAA motifs detected in file\n",
      "\n",
      "Seq is: CCTCATCTTTGGACA\n",
      "0 CCTCATCTTTGGACA motifs detected in file\n",
      "\n",
      "Seq is: CTCCTTTCCATAAGG\n",
      "0 CTCCTTTCCATAAGG motifs detected in file\n",
      "\n",
      "Seq is: ACGATTACGATCCGA\n",
      "0 ACGATTACGATCCGA motifs detected in file\n",
      "\n",
      "Seq is: CGGTTACCACTCCCT\n",
      "0 CGGTTACCACTCCCT motifs detected in file\n",
      "\n",
      "Seq is: CCAACTTCACTCTCC\n",
      "0 CCAACTTCACTCTCC motifs detected in file\n",
      "\n",
      "Seq is: ATACCAAAACTCATC\n",
      "0 ATACCAAAACTCATC motifs detected in file\n",
      "\n",
      "Seq is: AGGAGGATAAAAAGC\n",
      "0 AGGAGGATAAAAAGC motifs detected in file\n",
      "\n",
      "Seq is: TTATAGAACAAGTAA\n",
      "0 TTATAGAACAAGTAA motifs detected in file\n",
      "\n",
      "Seq is: CAATTCCAACACCTA\n",
      "0 CAATTCCAACACCTA motifs detected in file\n",
      "\n",
      "Seq is: CTACCTGGGCCTCGT\n",
      "0 CTACCTGGGCCTCGT motifs detected in file\n",
      "\n",
      "Seq is: GTATTTATCATACCG\n",
      "0 GTATTTATCATACCG motifs detected in file\n",
      "\n",
      "Seq is: AAACTTCTTCCGCAC\n",
      "0 AAACTTCTTCCGCAC motifs detected in file\n",
      "\n",
      "Seq is: CCATGTCTACACGTC\n",
      "0 CCATGTCTACACGTC motifs detected in file\n",
      "\n",
      "Seq is: CTTACGATCCACACC\n",
      "0 CTTACGATCCACACC motifs detected in file\n",
      "\n",
      "Seq is: AACCAACTACGGCTA\n",
      "0 AACCAACTACGGCTA motifs detected in file\n",
      "\n",
      "Seq is: ACATTGATAATACTC\n",
      "0 ACATTGATAATACTC motifs detected in file\n",
      "\n",
      "Seq is: AGGATCAACTGAACT\n",
      "0 AGGATCAACTGAACT motifs detected in file\n",
      "\n",
      "Seq is: TCACCTCTTCCGATG\n",
      "0 TCACCTCTTCCGATG motifs detected in file\n",
      "\n",
      "Seq is: CTAAAAAAACTACAG\n",
      "0 CTAAAAAAACTACAG motifs detected in file\n",
      "\n",
      "Seq is: TAAAACACATTCCCG\n",
      "0 TAAAACACATTCCCG motifs detected in file\n",
      "\n",
      "Seq is: CGTCTGACGACGCTT\n",
      "0 CGTCTGACGACGCTT motifs detected in file\n",
      "\n",
      "Seq is: GTCTCACCACTATCG\n",
      "0 GTCTCACCACTATCG motifs detected in file\n",
      "\n",
      "Seq is: CGGATTCGTATCACA\n",
      "0 CGGATTCGTATCACA motifs detected in file\n",
      "\n",
      "Seq is: ATCTCAAACCTAATC\n",
      "0 ATCTCAAACCTAATC motifs detected in file\n",
      "\n",
      "Seq is: ACCCGATAAAAGACG\n",
      "0 ACCCGATAAAAGACG motifs detected in file\n",
      "\n",
      "Seq is: ATCCCCCACCTTCGC\n",
      "0 ATCCCCCACCTTCGC motifs detected in file\n",
      "\n",
      "Seq is: CTACTCATTCTTCTA\n",
      "0 CTACTCATTCTTCTA motifs detected in file\n",
      "\n",
      "Seq is: AGGACGATGTTGATA\n",
      "0 AGGACGATGTTGATA motifs detected in file\n",
      "\n",
      "Seq is: TCACGAACTCACCTC\n",
      "0 TCACGAACTCACCTC motifs detected in file\n",
      "\n",
      "Seq is: ATCCCGACTCTGACC\n",
      "0 ATCCCGACTCTGACC motifs detected in file\n",
      "\n",
      "Seq is: CCACCCCGGAAGATA\n",
      "0 CCACCCCGGAAGATA motifs detected in file\n",
      "\n",
      "Seq is: CCGACTCCCCTCAGT\n",
      "0 CCGACTCCCCTCAGT motifs detected in file\n",
      "\n",
      "Seq is: CCCCACCTCTGGATT\n",
      "0 CCCCACCTCTGGATT motifs detected in file\n",
      "\n",
      "Seq is: AACCCGACTAAACGG\n",
      "0 AACCCGACTAAACGG motifs detected in file\n",
      "\n",
      "Seq is: ACGACGACGATCCTC\n",
      "0 ACGACGACGATCCTC motifs detected in file\n",
      "\n",
      "Seq is: CTCCGGATCATCACC\n",
      "0 CTCCGGATCATCACC motifs detected in file\n",
      "\n",
      "Seq is: CCACTCCGAACCTAA\n",
      "0 CCACTCCGAACCTAA motifs detected in file\n",
      "\n",
      "Seq is: TCGCAAATCTTCCCG\n",
      "0 TCGCAAATCTTCCCG motifs detected in file\n",
      "\n",
      "Seq is: ATAAACAACACCCAG\n",
      "0 ATAAACAACACCCAG motifs detected in file\n",
      "\n",
      "Seq is: AGTACTCAACCTCAC\n",
      "0 AGTACTCAACCTCAC motifs detected in file\n",
      "\n",
      "Seq is: ATCCTATTTAGTACG\n",
      "0 ATCCTATTTAGTACG motifs detected in file\n",
      "\n",
      "Seq is: ATTCCGCTCCTACTT\n",
      "0 ATTCCGCTCCTACTT motifs detected in file\n",
      "\n",
      "Seq is: TGGCTATAGCGGCTA\n",
      "0 TGGCTATAGCGGCTA motifs detected in file\n",
      "\n",
      "Seq is: TGCCAACATATCCTA\n",
      "0 TGCCAACATATCCTA motifs detected in file\n",
      "\n",
      "Seq is: ACGAACTTACCGACG\n",
      "0 ACGAACTTACCGACG motifs detected in file\n",
      "\n",
      "Seq is: ACACAACCGTAGACG\n",
      "0 ACACAACCGTAGACG motifs detected in file\n",
      "\n",
      "Seq is: AGCCCGCATAGTAGT\n",
      "0 AGCCCGCATAGTAGT motifs detected in file\n",
      "\n",
      "Seq is: TGACTACTCGTTCTT\n",
      "0 TGACTACTCGTTCTT motifs detected in file\n",
      "\n",
      "Seq is: CCTATATTAAGGATG\n",
      "0 CCTATATTAAGGATG motifs detected in file\n",
      "\n",
      "Seq is: CGGGAGAGTCGGCTA\n",
      "0 CGGGAGAGTCGGCTA motifs detected in file\n",
      "\n",
      "Seq is: CTTGTCAACCTTATC\n",
      "0 CTTGTCAACCTTATC motifs detected in file\n",
      "\n",
      "Seq is: CAACAATCGCCATTG\n",
      "0 CAACAATCGCCATTG motifs detected in file\n",
      "\n",
      "Seq is: ATTCTAATCATACCA\n",
      "0 ATTCTAATCATACCA motifs detected in file\n",
      "\n",
      "Seq is: TTAATCCTTCTACTC\n",
      "0 TTAATCCTTCTACTC motifs detected in file\n",
      "\n",
      "Seq is: ATCTATAAACTTCTT\n",
      "0 ATCTATAAACTTCTT motifs detected in file\n",
      "\n",
      "Seq is: GACTAATTACAAACC\n",
      "0 GACTAATTACAAACC motifs detected in file\n",
      "\n",
      "Seq is: CAGACTCAAATATAT\n",
      "0 CAGACTCAAATATAT motifs detected in file\n",
      "\n",
      "Seq is: AGTGTCACTCTTAAG\n",
      "0 AGTGTCACTCTTAAG motifs detected in file\n",
      "\n",
      "Seq is: ATACTACCTGGTACA\n",
      "0 ATACTACCTGGTACA motifs detected in file\n",
      "\n",
      "Seq is: TTGCTTGTTACGATG\n",
      "0 TTGCTTGTTACGATG motifs detected in file\n",
      "\n",
      "Seq is: TCCCTACTTATAATA\n",
      "0 TCCCTACTTATAATA motifs detected in file\n",
      "\n",
      "Seq is: CCTCTTCATCAGATC\n",
      "0 CCTCTTCATCAGATC motifs detected in file\n",
      "\n",
      "Seq is: AAACTTCGAATCCCT\n",
      "0 AAACTTCGAATCCCT motifs detected in file\n",
      "\n",
      "Seq is: CTCGACCCAACAAAC\n",
      "0 CTCGACCCAACAAAC motifs detected in file\n",
      "\n",
      "Seq is: CCAACACCGAGTCAC\n",
      "0 CCAACACCGAGTCAC motifs detected in file\n",
      "\n",
      "Seq is: AGTCAAGCTCTATTA\n",
      "0 AGTCAAGCTCTATTA motifs detected in file\n",
      "\n",
      "Seq is: TTGAAGAACCAGATC\n",
      "0 TTGAAGAACCAGATC motifs detected in file\n",
      "\n",
      "Seq is: CGTGTACTTATAACA\n",
      "0 CGTGTACTTATAACA motifs detected in file\n",
      "\n",
      "Seq is: ACACCCCTTCTCTGA\n",
      "0 ACACCCCTTCTCTGA motifs detected in file\n",
      "\n",
      "Seq is: CTATTATTTCCACCT\n",
      "0 CTATTATTTCCACCT motifs detected in file\n",
      "\n",
      "Seq is: ACGCTGTTACCTAAA\n",
      "0 ACGCTGTTACCTAAA motifs detected in file\n",
      "\n",
      "Seq is: ATGTATTACCCCCAT\n",
      "0 ATGTATTACCCCCAT motifs detected in file\n",
      "\n",
      "Seq is: ACTCAAAAAAAACAA\n",
      "0 ACTCAAAAAAAACAA motifs detected in file\n",
      "\n",
      "Seq is: TCCCAATTGCTCCCA\n",
      "0 TCCCAATTGCTCCCA motifs detected in file\n",
      "\n",
      "Seq is: CCATTCCTACCCCCC\n",
      "0 CCATTCCTACCCCCC motifs detected in file\n",
      "\n",
      "Seq is: TTAATCCCTTCAGTC\n",
      "0 TTAATCCCTTCAGTC motifs detected in file\n",
      "\n",
      "Seq is: CCAATCCCACCAATA\n",
      "0 CCAATCCCACCAATA motifs detected in file\n",
      "\n",
      "Seq is: TCATCACACGTACCA\n",
      "0 TCATCACACGTACCA motifs detected in file\n",
      "\n",
      "Seq is: ATAATGAAAATAAAC\n",
      "0 ATAATGAAAATAAAC motifs detected in file\n",
      "\n",
      "Seq is: CTCAACGTGGTTTTA\n",
      "0 CTCAACGTGGTTTTA motifs detected in file\n",
      "\n",
      "Seq is: AAAACCCCGGATTCT\n",
      "0 AAAACCCCGGATTCT motifs detected in file\n",
      "\n",
      "Seq is: GGTTACCTATCGACA\n",
      "0 GGTTACCTATCGACA motifs detected in file\n",
      "\n",
      "Seq is: ATAGGAAATTTTCAA\n",
      "0 ATAGGAAATTTTCAA motifs detected in file\n",
      "\n",
      "Seq is: CTCTTTCGGTACAAC\n",
      "0 CTCTTTCGGTACAAC motifs detected in file\n",
      "\n",
      "Seq is: AATCTGTACCCCCGT\n",
      "0 AATCTGTACCCCCGT motifs detected in file\n",
      "\n",
      "Seq is: ACTCAATCGTCAAGA\n",
      "0 ACTCAATCGTCAAGA motifs detected in file\n",
      "\n",
      "Seq is: ACACTCGAAAGAGCC\n",
      "0 ACACTCGAAAGAGCC motifs detected in file\n",
      "\n",
      "Seq is: ATTTATTCCCCAGCA\n",
      "0 ATTTATTCCCCAGCA motifs detected in file\n",
      "\n",
      "Seq is: TTCGGAGACAACAGT\n",
      "0 TTCGGAGACAACAGT motifs detected in file\n",
      "\n",
      "Seq is: CTAAGTGTTAGACTA\n",
      "0 CTAAGTGTTAGACTA motifs detected in file\n",
      "\n",
      "Seq is: CAAAACCAATTTGAT\n",
      "0 CAAAACCAATTTGAT motifs detected in file\n",
      "\n",
      "Seq is: ATAAATGTTCTCCTT\n",
      "0 ATAAATGTTCTCCTT motifs detected in file\n",
      "\n",
      "Seq is: TTGGGCCATTACTAC\n",
      "0 TTGGGCCATTACTAC motifs detected in file\n",
      "\n",
      "Seq is: AGCCCCAACTCCCTA\n",
      "0 AGCCCCAACTCCCTA motifs detected in file\n",
      "\n",
      "Seq is: TCCTCCTCTTACCCC\n",
      "0 TCCTCCTCTTACCCC motifs detected in file\n",
      "\n",
      "Seq is: CTATCCACATACTTG\n",
      "0 CTATCCACATACTTG motifs detected in file\n",
      "\n",
      "Seq is: TACTCCCACAAAAGA\n",
      "0 TACTCCCACAAAAGA motifs detected in file\n",
      "\n",
      "Seq is: GCACACTTACTCCCA\n",
      "0 GCACACTTACTCCCA motifs detected in file\n",
      "\n",
      "Seq is: AAATACAACAATTAC\n",
      "0 AAATACAACAATTAC motifs detected in file\n",
      "\n",
      "Seq is: ACCACCCACTCACTC\n",
      "0 ACCACCCACTCACTC motifs detected in file\n",
      "\n",
      "Seq is: GGGGTAACACAACAC\n",
      "0 GGGGTAACACAACAC motifs detected in file\n",
      "\n",
      "Seq is: CATTTATACATCTCC\n",
      "0 CATTTATACATCTCC motifs detected in file\n",
      "\n",
      "Seq is: CTCATATCCCGACAC\n",
      "0 CTCATATCCCGACAC motifs detected in file\n",
      "\n",
      "Seq is: TGATCATACAACTCA\n",
      "0 TGATCATACAACTCA motifs detected in file\n",
      "\n",
      "Seq is: GGACATTCATCCTCT\n",
      "0 GGACATTCATCCTCT motifs detected in file\n",
      "\n",
      "Seq is: CACTATAAACTAGTC\n",
      "0 CACTATAAACTAGTC motifs detected in file\n",
      "\n",
      "Seq is: CTCTTGCACCAATGA\n",
      "0 CTCTTGCACCAATGA motifs detected in file\n",
      "\n",
      "Seq is: TCGTGTCTCTCAAGA\n",
      "0 TCGTGTCTCTCAAGA motifs detected in file\n",
      "\n",
      "Seq is: GGGTCATCCAATTAT\n",
      "0 GGGTCATCCAATTAT motifs detected in file\n",
      "\n",
      "Seq is: CACCCCCCATTCCGC\n",
      "0 CACCCCCCATTCCGC motifs detected in file\n",
      "\n",
      "Seq is: TCCAATCGCTCCGAA\n",
      "0 TCCAATCGCTCCGAA motifs detected in file\n",
      "\n",
      "Seq is: CGATCTTCAGTAGTT\n",
      "0 CGATCTTCAGTAGTT motifs detected in file\n",
      "\n",
      "Seq is: TTTCGATAATCACCC\n",
      "0 TTTCGATAATCACCC motifs detected in file\n",
      "\n",
      "Seq is: TCATCTCAAACTTCA\n",
      "0 TCATCTCAAACTTCA motifs detected in file\n",
      "\n",
      "Seq is: GGAACTCTCTCCTAA\n",
      "0 GGAACTCTCTCCTAA motifs detected in file\n",
      "\n",
      "Seq is: TACTACGCTGACACT\n",
      "0 TACTACGCTGACACT motifs detected in file\n",
      "\n",
      "Seq is: CACGCAAGCATCAAA\n",
      "0 CACGCAAGCATCAAA motifs detected in file\n",
      "\n",
      "Seq is: CTCAAACGATCCGTC\n",
      "0 CTCAAACGATCCGTC motifs detected in file\n",
      "\n",
      "Seq is: TTATCATTACTCCTA\n",
      "0 TTATCATTACTCCTA motifs detected in file\n",
      "\n",
      "Seq is: CATTCGGGCACCCGC\n",
      "0 CATTCGGGCACCCGC motifs detected in file\n",
      "\n",
      "Seq is: TAATACTCTTACTGA\n",
      "0 TAATACTCTTACTGA motifs detected in file\n",
      "\n",
      "Seq is: CGCGGCCACTTCGAA\n",
      "0 CGCGGCCACTTCGAA motifs detected in file\n",
      "\n",
      "Seq is: GTCCCCCAAACCTAC\n",
      "0 GTCCCCCAAACCTAC motifs detected in file\n",
      "\n",
      "Seq is: TCTTACCGACAATGA\n",
      "0 TCTTACCGACAATGA motifs detected in file\n",
      "\n",
      "Seq is: TGCTCCCGATACACC\n",
      "0 TGCTCCCGATACACC motifs detected in file\n",
      "\n",
      "Seq is: GACTAACTTCTCATA\n",
      "0 GACTAACTTCTCATA motifs detected in file\n",
      "\n",
      "Seq is: CGTTACTCGCTAAAA\n",
      "0 CGTTACTCGCTAAAA motifs detected in file\n",
      "\n",
      "Seq is: TCCAGACAAACAGCA\n",
      "0 TCCAGACAAACAGCA motifs detected in file\n",
      "\n",
      "Seq is: TCCGTCTACCTCGAA\n",
      "0 TCCGTCTACCTCGAA motifs detected in file\n",
      "\n",
      "Seq is: CAATATTAATACGGA\n",
      "0 CAATATTAATACGGA motifs detected in file\n",
      "\n",
      "Seq is: GTATCCCTATCATGT\n",
      "0 GTATCCCTATCATGT motifs detected in file\n",
      "\n",
      "Seq is: TCCTTCCCCATCCGA\n",
      "0 TCCTTCCCCATCCGA motifs detected in file\n",
      "\n",
      "Seq is: TACACAAAACAGTCC\n",
      "0 TACACAAAACAGTCC motifs detected in file\n",
      "\n",
      "Seq is: CCCAACTCTTACTCA\n",
      "0 CCCAACTCTTACTCA motifs detected in file\n",
      "\n",
      "Seq is: CACTCCGCATAATAT\n",
      "0 CACTCCGCATAATAT motifs detected in file\n",
      "\n",
      "Seq is: GGTATCGGCGGATCA\n",
      "0 GGTATCGGCGGATCA motifs detected in file\n",
      "\n",
      "Seq is: AAATTCTCATGACGC\n",
      "0 AAATTCTCATGACGC motifs detected in file\n",
      "\n",
      "Seq is: CGTTCATGATAACTG\n",
      "0 CGTTCATGATAACTG motifs detected in file\n",
      "\n",
      "Seq is: GGTCGCTACCCCCGA\n",
      "0 GGTCGCTACCCCCGA motifs detected in file\n",
      "\n",
      "Seq is: AGCTGTACCCGAAAT\n",
      "0 AGCTGTACCCGAAAT motifs detected in file\n",
      "\n",
      "Seq is: CCCTCAGTATTCACC\n",
      "0 CCCTCAGTATTCACC motifs detected in file\n",
      "\n",
      "Seq is: TCAGGCATTTCTCCA\n",
      "0 TCAGGCATTTCTCCA motifs detected in file\n",
      "\n",
      "Seq is: TAGAAATGATATTTT\n",
      "0 TAGAAATGATATTTT motifs detected in file\n",
      "\n",
      "Seq is: CGATAACACATTCGA\n",
      "0 CGATAACACATTCGA motifs detected in file\n",
      "\n",
      "Seq is: TCAGTATAATTCAAC\n",
      "0 TCAGTATAATTCAAC motifs detected in file\n",
      "\n",
      "Seq is: AACCGAGTCCTCAAA\n",
      "0 AACCGAGTCCTCAAA motifs detected in file\n",
      "\n",
      "Seq is: CTATCAAGAACCCGT\n",
      "0 CTATCAAGAACCCGT motifs detected in file\n",
      "\n",
      "Seq is: CACTCTCACTCATCA\n",
      "0 CACTCTCACTCATCA motifs detected in file\n",
      "\n",
      "Seq is: TCTTACAAATCACTC\n",
      "0 TCTTACAAATCACTC motifs detected in file\n",
      "\n",
      "Seq is: GGATCCCACAACACT\n",
      "0 GGATCCCACAACACT motifs detected in file\n",
      "\n",
      "Seq is: CACATTTAATCACGC\n",
      "0 CACATTTAATCACGC motifs detected in file\n",
      "\n",
      "Seq is: TACTCATCCCCTTCC\n",
      "0 TACTCATCCCCTTCC motifs detected in file\n",
      "\n",
      "Seq is: CTCGGATGATCCCAC\n",
      "0 CTCGGATGATCCCAC motifs detected in file\n",
      "\n",
      "Seq is: ATCTTATCCTTCATA\n",
      "0 ATCTTATCCTTCATA motifs detected in file\n",
      "\n",
      "Seq is: CACGGACGCAAGTCC\n",
      "0 CACGGACGCAAGTCC motifs detected in file\n",
      "\n",
      "Seq is: GCAAGACCGACCAAC\n",
      "0 GCAAGACCGACCAAC motifs detected in file\n",
      "\n",
      "Seq is: GGAGTAGCCCACTAC\n",
      "0 GGAGTAGCCCACTAC motifs detected in file\n",
      "\n",
      "Seq is: TATCGGTTCCACCCC\n",
      "0 TATCGGTTCCACCCC motifs detected in file\n",
      "\n",
      "Seq is: TATTCACACCAAAGC\n",
      "0 TATTCACACCAAAGC motifs detected in file\n",
      "\n",
      "Seq is: TTCTTCTATATTTTA\n",
      "0 TTCTTCTATATTTTA motifs detected in file\n",
      "\n",
      "Seq is: TACTAATCAAGACAC\n",
      "0 TACTAATCAAGACAC motifs detected in file\n",
      "\n",
      "Seq is: CGACACTTACAATAT\n",
      "0 CGACACTTACAATAT motifs detected in file\n",
      "\n",
      "Seq is: TAATTCCTCTAAACA\n",
      "0 TAATTCCTCTAAACA motifs detected in file\n",
      "\n",
      "Seq is: TCCCTCTAATCATAT\n",
      "0 TCCCTCTAATCATAT motifs detected in file\n",
      "\n",
      "Seq is: CTCTCCATCTCAAAA\n",
      "0 CTCTCCATCTCAAAA motifs detected in file\n",
      "\n",
      "Seq is: AAAGCACTATCACCA\n",
      "0 AAAGCACTATCACCA motifs detected in file\n",
      "\n",
      "Seq is: AGTGACCTATTCACC\n",
      "0 AGTGACCTATTCACC motifs detected in file\n",
      "\n",
      "Seq is: GCAACCGAACGGTAC\n",
      "0 GCAACCGAACGGTAC motifs detected in file\n",
      "\n",
      "Seq is: TAACACTCCCCATCC\n",
      "0 TAACACTCCCCATCC motifs detected in file\n",
      "\n",
      "Seq is: TCAGTCCATCAATCA\n",
      "0 TCAGTCCATCAATCA motifs detected in file\n",
      "\n",
      "Seq is: TAATCCTCCCCCCAA\n",
      "0 TAATCCTCCCCCCAA motifs detected in file\n",
      "\n",
      "Seq is: CAATCCCCCAGCCTC\n",
      "0 CAATCCCCCAGCCTC motifs detected in file\n",
      "\n",
      "Seq is: CTTTTCCAACCCCTT\n",
      "0 CTTTTCCAACCCCTT motifs detected in file\n",
      "\n",
      "Seq is: GTCGATTTATCCAAC\n",
      "0 GTCGATTTATCCAAC motifs detected in file\n",
      "\n",
      "Seq is: AACAACTAAACCAAT\n",
      "0 AACAACTAAACCAAT motifs detected in file\n",
      "\n",
      "Seq is: TTTTTATCATCTCCC\n",
      "0 TTTTTATCATCTCCC motifs detected in file\n",
      "\n",
      "Seq is: TACTACGATTATTAA\n",
      "0 TACTACGATTATTAA motifs detected in file\n",
      "\n",
      "Seq is: TCCGACACCCACCAA\n",
      "0 TCCGACACCCACCAA motifs detected in file\n",
      "\n",
      "Seq is: CACAACTAAGTTTAA\n",
      "0 CACAACTAAGTTTAA motifs detected in file\n",
      "\n",
      "Seq is: TACACAAAAAACCTT\n",
      "0 TACACAAAAAACCTT motifs detected in file\n",
      "\n",
      "Seq is: TCAGTACAGTCACCA\n",
      "0 TCAGTACAGTCACCA motifs detected in file\n",
      "\n",
      "Seq is: TCATTATATTAACAA\n",
      "0 TCATTATATTAACAA motifs detected in file\n",
      "\n",
      "Seq is: CCCTGCTAATCAAAA\n",
      "0 CCCTGCTAATCAAAA motifs detected in file\n",
      "\n",
      "Seq is: TCGTAACCTCATCCA\n",
      "0 TCGTAACCTCATCCA motifs detected in file\n",
      "\n",
      "Seq is: AATCCAATACATGCA\n",
      "0 AATCCAATACATGCA motifs detected in file\n",
      "\n",
      "Seq is: TCAGATCCGGTATAC\n",
      "0 TCAGATCCGGTATAC motifs detected in file\n",
      "\n",
      "Seq is: ACAACCTCTAACTCT\n",
      "0 ACAACCTCTAACTCT motifs detected in file\n",
      "\n",
      "Seq is: GATCATCCCGATCCG\n",
      "0 GATCATCCCGATCCG motifs detected in file\n",
      "\n",
      "Seq is: GGTGGCGACGAAGCG\n",
      "0 GGTGGCGACGAAGCG motifs detected in file\n",
      "\n",
      "Seq is: TCCGCCGTTTCTGAT\n",
      "0 TCCGCCGTTTCTGAT motifs detected in file\n",
      "\n",
      "Seq is: CATACCGTTATCCGT\n",
      "0 CATACCGTTATCCGT motifs detected in file\n",
      "\n",
      "Seq is: GTTATAACCGATTCT\n",
      "0 GTTATAACCGATTCT motifs detected in file\n",
      "\n",
      "Seq is: CCCTCACCCACAACT\n",
      "0 CCCTCACCCACAACT motifs detected in file\n",
      "\n",
      "Seq is: CCCAATACTCTCATC\n",
      "0 CCCAATACTCTCATC motifs detected in file\n",
      "\n",
      "Seq is: GATATTACTTGTCGC\n",
      "0 GATATTACTTGTCGC motifs detected in file\n",
      "\n",
      "Seq is: TATCATAATAAGGAA\n",
      "0 TATCATAATAAGGAA motifs detected in file\n",
      "\n",
      "Seq is: GATCCGTATCATCCC\n",
      "0 GATCCGTATCATCCC motifs detected in file\n",
      "\n",
      "Seq is: TCCTATACTCCACAC\n",
      "0 TCCTATACTCCACAC motifs detected in file\n",
      "\n",
      "Seq is: TCGCTATATGATCAT\n",
      "0 TCGCTATATGATCAT motifs detected in file\n",
      "\n",
      "Seq is: AAGGATCTTCACTCT\n",
      "0 AAGGATCTTCACTCT motifs detected in file\n",
      "\n",
      "Seq is: ACCATTTACGATCAT\n",
      "0 ACCATTTACGATCAT motifs detected in file\n",
      "\n",
      "Seq is: ATTATAAATACATTT\n",
      "0 ATTATAAATACATTT motifs detected in file\n",
      "\n",
      "Seq is: ACTCCCCGTAAACCA\n",
      "0 ACTCCCCGTAAACCA motifs detected in file\n",
      "\n",
      "Seq is: TTTATACTAATAGTA\n",
      "0 TTTATACTAATAGTA motifs detected in file\n",
      "\n",
      "Seq is: TTAAATTACTCAGCT\n",
      "0 TTAAATTACTCAGCT motifs detected in file\n",
      "\n",
      "Seq is: TTAGTAAGCAAAACA\n",
      "0 TTAGTAAGCAAAACA motifs detected in file\n",
      "\n",
      "Seq is: AATTTGATATATGGT\n",
      "0 AATTTGATATATGGT motifs detected in file\n",
      "\n",
      "Seq is: TAAGCCAAGTCAGAT\n",
      "0 TAAGCCAAGTCAGAT motifs detected in file\n",
      "\n",
      "Seq is: TAGGAAAAACATCAG\n",
      "0 TAGGAAAAACATCAG motifs detected in file\n",
      "\n",
      "Seq is: TGAGTATCCGGTCTG\n",
      "0 TGAGTATCCGGTCTG motifs detected in file\n",
      "\n",
      "Seq is: AATCCCGATCCTACT\n",
      "0 AATCCCGATCCTACT motifs detected in file\n",
      "\n",
      "Seq is: ACTAATTATTCTCCC\n",
      "0 ACTAATTATTCTCCC motifs detected in file\n",
      "\n",
      "Seq is: TACTGTATTGATAAT\n",
      "0 TACTGTATTGATAAT motifs detected in file\n",
      "\n",
      "Seq is: CACCGTCCAATCAAC\n",
      "0 CACCGTCCAATCAAC motifs detected in file\n",
      "\n",
      "Seq is: AAACATCCCGAGTAC\n",
      "0 AAACATCCCGAGTAC motifs detected in file\n",
      "\n",
      "Seq is: CATCCCCATTTTCCT\n",
      "0 CATCCCCATTTTCCT motifs detected in file\n",
      "\n",
      "Seq is: CCCGTTAAAGATCTA\n",
      "0 CCCGTTAAAGATCTA motifs detected in file\n",
      "\n",
      "Seq is: GTTTATTATTCTTCC\n",
      "0 GTTTATTATTCTTCC motifs detected in file\n",
      "\n",
      "Seq is: ATTATCGATGATTCT\n",
      "0 ATTATCGATGATTCT motifs detected in file\n",
      "\n",
      "Seq is: TCTTAAAATACCTCT\n",
      "0 TCTTAAAATACCTCT motifs detected in file\n",
      "\n",
      "Seq is: TTCCCTGCGCCCGCC\n",
      "0 TTCCCTGCGCCCGCC motifs detected in file\n",
      "\n",
      "Seq is: CCCTATATCCCAGCT\n",
      "0 CCCTATATCCCAGCT motifs detected in file\n",
      "\n",
      "Seq is: TCGGCGTGAGCATTC\n",
      "0 TCGGCGTGAGCATTC motifs detected in file\n",
      "\n",
      "Seq is: CCCACCTAAAAAGAT\n",
      "0 CCCACCTAAAAAGAT motifs detected in file\n",
      "\n",
      "Seq is: ACATCGGCAACTCAA\n",
      "0 ACATCGGCAACTCAA motifs detected in file\n",
      "\n",
      "Seq is: CACCATCAGTTTTAC\n",
      "0 CACCATCAGTTTTAC motifs detected in file\n",
      "\n",
      "Seq is: ATTATTAATAATCAT\n",
      "0 ATTATTAATAATCAT motifs detected in file\n",
      "\n",
      "Seq is: CATTCCGATCCTCCC\n",
      "0 CATTCCGATCCTCCC motifs detected in file\n",
      "\n",
      "Seq is: ACAACTAATAATTTT\n",
      "0 ACAACTAATAATTTT motifs detected in file\n",
      "\n",
      "Seq is: AATTCCGCTTCAAAT\n",
      "0 AATTCCGCTTCAAAT motifs detected in file\n",
      "\n",
      "Seq is: AATGAGAAAAAACTT\n",
      "0 AATGAGAAAAAACTT motifs detected in file\n",
      "\n",
      "Seq is: ACAACAGTTTTGATC\n",
      "0 ACAACAGTTTTGATC motifs detected in file\n",
      "\n",
      "Seq is: AATTAACCTTCAATT\n",
      "0 AATTAACCTTCAATT motifs detected in file\n",
      "\n",
      "Seq is: GCCATGATAAATATG\n",
      "0 GCCATGATAAATATG motifs detected in file\n",
      "\n",
      "Seq is: ATTTTCTCATTCTGG\n",
      "0 ATTTTCTCATTCTGG motifs detected in file\n",
      "\n",
      "Seq is: GAGTAGTTATCTACC\n",
      "0 GAGTAGTTATCTACC motifs detected in file\n",
      "\n",
      "Seq is: TCTGTATGTCTTTAT\n",
      "0 TCTGTATGTCTTTAT motifs detected in file\n",
      "\n",
      "Seq is: CAGTTTGGTGTAGAT\n",
      "0 CAGTTTGGTGTAGAT motifs detected in file\n",
      "\n",
      "Seq is: GTTTTACGGTCATAG\n",
      "0 GTTTTACGGTCATAG motifs detected in file\n",
      "\n",
      "Seq is: TCCGCCGCCGAAGCT\n",
      "0 TCCGCCGCCGAAGCT motifs detected in file\n",
      "\n",
      "Seq is: TCGGTTTCACTACAA\n",
      "0 TCGGTTTCACTACAA motifs detected in file\n",
      "\n",
      "Seq is: ACCTACATTTCACTT\n",
      "0 ACCTACATTTCACTT motifs detected in file\n",
      "\n",
      "Seq is: TATAATCAACCGCCT\n",
      "0 TATAATCAACCGCCT motifs detected in file\n",
      "\n",
      "Seq is: ACTTCGTCTATCACT\n",
      "0 ACTTCGTCTATCACT motifs detected in file\n",
      "\n",
      "Seq is: CCTTTCAACTCGGTT\n",
      "0 CCTTTCAACTCGGTT motifs detected in file\n",
      "\n",
      "Seq is: ATTACTGCACTTCAG\n",
      "0 ATTACTGCACTTCAG motifs detected in file\n",
      "\n",
      "Seq is: GCACCTTCGGACACC\n",
      "0 GCACCTTCGGACACC motifs detected in file\n",
      "\n",
      "Seq is: GATGTTTTTTACAAC\n",
      "0 GATGTTTTTTACAAC motifs detected in file\n",
      "\n",
      "Seq is: TCGGCATCTACGGCA\n",
      "0 TCGGCATCTACGGCA motifs detected in file\n",
      "\n",
      "Seq is: GCCTTTACCACTTCC\n",
      "0 GCCTTTACCACTTCC motifs detected in file\n",
      "\n",
      "Seq is: CTCTGAGCTTCATGA\n",
      "0 CTCTGAGCTTCATGA motifs detected in file\n",
      "\n",
      "Seq is: GACTCCGAACATCCT\n",
      "0 GACTCCGAACATCCT motifs detected in file\n",
      "\n",
      "Seq is: CCCATTTTATCTCTG\n",
      "0 CCCATTTTATCTCTG motifs detected in file\n",
      "\n",
      "Seq is: GGTCATTTTAACATT\n",
      "0 GGTCATTTTAACATT motifs detected in file\n",
      "\n",
      "Seq is: ATTCGTCACGAACTT\n",
      "0 ATTCGTCACGAACTT motifs detected in file\n",
      "\n",
      "Seq is: AATAAACCAAAGCCA\n",
      "0 AATAAACCAAAGCCA motifs detected in file\n",
      "\n",
      "Seq is: ACAAAAGATAATCTG\n",
      "0 ACAAAAGATAATCTG motifs detected in file\n",
      "\n",
      "Seq is: ATACCACTCGAGTCC\n",
      "0 ATACCACTCGAGTCC motifs detected in file\n",
      "\n",
      "Seq is: ACTAACTATGAGGAC\n",
      "0 ACTAACTATGAGGAC motifs detected in file\n",
      "\n",
      "Seq is: TACGCTCATTATGCC\n",
      "0 TACGCTCATTATGCC motifs detected in file\n",
      "\n",
      "Seq is: TACACAAATCCTCAC\n",
      "0 TACACAAATCCTCAC motifs detected in file\n",
      "\n",
      "Seq is: CCTGAAGATCCCCTA\n",
      "0 CCTGAAGATCCCCTA motifs detected in file\n",
      "\n",
      "Seq is: AATCGCCCCACTACG\n",
      "0 AATCGCCCCACTACG motifs detected in file\n",
      "\n",
      "Seq is: GACAACCCCCGGTCA\n",
      "0 GACAACCCCCGGTCA motifs detected in file\n",
      "\n",
      "Seq is: CGGGAGGATTAACCC\n",
      "0 CGGGAGGATTAACCC motifs detected in file\n",
      "\n",
      "Seq is: CCCATCCCCGATCCG\n",
      "0 CCCATCCCCGATCCG motifs detected in file\n",
      "\n",
      "Seq is: ACCTCACCATTTTCC\n",
      "0 ACCTCACCATTTTCC motifs detected in file\n",
      "\n",
      "Seq is: GAGTCTTTTTAGGAC\n",
      "0 GAGTCTTTTTAGGAC motifs detected in file\n",
      "\n",
      "Seq is: GCTTCTTTTTTTGAA\n",
      "0 GCTTCTTTTTTTGAA motifs detected in file\n",
      "\n",
      "Seq is: GACTCCATTATTTAT\n",
      "0 GACTCCATTATTTAT motifs detected in file\n",
      "\n",
      "Seq is: CCTAATAGGGCATAG\n",
      "0 CCTAATAGGGCATAG motifs detected in file\n",
      "\n",
      "Seq is: CTTCCGGAAAAACCT\n",
      "0 CTTCCGGAAAAACCT motifs detected in file\n",
      "\n",
      "Seq is: GTCCACCACACACCA\n",
      "0 GTCCACCACACACCA motifs detected in file\n",
      "\n",
      "Seq is: CCGGAACCATACACG\n",
      "0 CCGGAACCATACACG motifs detected in file\n",
      "\n",
      "Seq is: AAAGAGCACAATGTA\n",
      "0 AAAGAGCACAATGTA motifs detected in file\n",
      "\n",
      "Seq is: GCGCGGTAGTAACCA\n",
      "0 GCGCGGTAGTAACCA motifs detected in file\n",
      "\n",
      "Seq is: TATACCAATCACACA\n",
      "0 TATACCAATCACACA motifs detected in file\n",
      "\n",
      "Seq is: ACCAATCATCCGGAT\n",
      "0 ACCAATCATCCGGAT motifs detected in file\n",
      "\n",
      "Seq is: CATACTCCTCGCAAT\n",
      "0 CATACTCCTCGCAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCTCACCTTCACTT\n",
      "0 ACCTCACCTTCACTT motifs detected in file\n",
      "\n",
      "Seq is: TAGTGTACCGATCCG\n",
      "0 TAGTGTACCGATCCG motifs detected in file\n",
      "\n",
      "Seq is: GCCTCCAGTAATCCT\n",
      "0 GCCTCCAGTAATCCT motifs detected in file\n",
      "\n",
      "Seq is: CCCGACTCTCCCGGG\n",
      "0 CCCGACTCTCCCGGG motifs detected in file\n",
      "\n",
      "Seq is: GACAATCCCCAGTAC\n",
      "0 GACAATCCCCAGTAC motifs detected in file\n",
      "\n",
      "Seq is: CCGACCCAAAATGAT\n",
      "0 CCGACCCAAAATGAT motifs detected in file\n",
      "\n",
      "Seq is: ATACTATCCGTACAC\n",
      "0 ATACTATCCGTACAC motifs detected in file\n",
      "\n",
      "Seq is: TAACCACCCAGTAAT\n",
      "0 TAACCACCCAGTAAT motifs detected in file\n",
      "\n",
      "Seq is: ACACAACAGCACGTC\n",
      "0 ACACAACAGCACGTC motifs detected in file\n",
      "\n",
      "Seq is: CATCTCCGAATGATC\n",
      "0 CATCTCCGAATGATC motifs detected in file\n",
      "\n",
      "Seq is: TTCACACTTTTGCAT\n",
      "0 TTCACACTTTTGCAT motifs detected in file\n",
      "\n",
      "Seq is: CCGAACCTAATTCCG\n",
      "0 CCGAACCTAATTCCG motifs detected in file\n",
      "\n",
      "Seq is: CTGTCGCTAAAGATC\n",
      "0 CTGTCGCTAAAGATC motifs detected in file\n",
      "\n",
      "Seq is: CTATCAGTCATCTTA\n",
      "0 CTATCAGTCATCTTA motifs detected in file\n",
      "\n",
      "Seq is: ATCTTAACACTTCTA\n",
      "0 ATCTTAACACTTCTA motifs detected in file\n",
      "\n",
      "Seq is: CTATTCACATCTCCC\n",
      "0 CTATTCACATCTCCC motifs detected in file\n",
      "\n",
      "Seq is: TTCCAATTACCAACT\n",
      "0 TTCCAATTACCAACT motifs detected in file\n",
      "\n",
      "Seq is: ATAACGATCCCACCG\n",
      "0 ATAACGATCCCACCG motifs detected in file\n",
      "\n",
      "Seq is: CGAAGGTTAATCCAC\n",
      "0 CGAAGGTTAATCCAC motifs detected in file\n",
      "\n",
      "Seq is: GTACTCATCCACCGG\n",
      "0 GTACTCATCCACCGG motifs detected in file\n",
      "\n",
      "Seq is: ACGTCATTACAATCG\n",
      "0 ACGTCATTACAATCG motifs detected in file\n",
      "\n",
      "Seq is: CCAATCCGCATGCCG\n",
      "0 CCAATCCGCATGCCG motifs detected in file\n",
      "\n",
      "Seq is: GTCCCGATAACCAAC\n",
      "0 GTCCCGATAACCAAC motifs detected in file\n",
      "\n",
      "Seq is: TTACTCATCCGACTA\n",
      "0 TTACTCATCCGACTA motifs detected in file\n",
      "\n",
      "Seq is: CCAAAGCTATTATTG\n",
      "0 CCAAAGCTATTATTG motifs detected in file\n",
      "\n",
      "Seq is: ATCATACCCCTATTC\n",
      "0 ATCATACCCCTATTC motifs detected in file\n",
      "\n",
      "Seq is: CCCACATCCACACGG\n",
      "0 CCCACATCCACACGG motifs detected in file\n",
      "\n",
      "Seq is: AACACCATTCTTCAC\n",
      "0 AACACCATTCTTCAC motifs detected in file\n",
      "\n",
      "Seq is: CCGATCCCGTAAAAA\n",
      "0 CCGATCCCGTAAAAA motifs detected in file\n",
      "\n",
      "Seq is: TTAGAATCTCGCTTT\n",
      "0 TTAGAATCTCGCTTT motifs detected in file\n",
      "\n",
      "Seq is: CGGATATTAGTGACA\n",
      "0 CGGATATTAGTGACA motifs detected in file\n",
      "\n",
      "Seq is: CGGGCGAGTATTCCC\n",
      "0 CGGGCGAGTATTCCC motifs detected in file\n",
      "\n",
      "Seq is: CTACCGGTACCGATC\n",
      "0 CTACCGGTACCGATC motifs detected in file\n",
      "\n",
      "Seq is: CAAATATCTATCAAC\n",
      "0 CAAATATCTATCAAC motifs detected in file\n",
      "\n",
      "Seq is: CCACCAACCACATTT\n",
      "0 CCACCAACCACATTT motifs detected in file\n",
      "\n",
      "Seq is: ACTCACTCCGTCCTC\n",
      "0 ACTCACTCCGTCCTC motifs detected in file\n",
      "\n",
      "Seq is: AGGCTCCTCCAATCA\n",
      "0 AGGCTCCTCCAATCA motifs detected in file\n",
      "\n",
      "Seq is: ACACCGTTATTTTTA\n",
      "0 ACACCGTTATTTTTA motifs detected in file\n",
      "\n",
      "Seq is: CTAATTCCTATGATC\n",
      "0 CTAATTCCTATGATC motifs detected in file\n",
      "\n",
      "Seq is: ATATTCTCTAGTCCA\n",
      "0 ATATTCTCTAGTCCA motifs detected in file\n",
      "\n",
      "Seq is: AGCAGGAAATCACAA\n",
      "0 AGCAGGAAATCACAA motifs detected in file\n",
      "\n",
      "Seq is: CACATACCAATAGTA\n",
      "0 CACATACCAATAGTA motifs detected in file\n",
      "\n",
      "Seq is: AACAAAACTCCAATC\n",
      "0 AACAAAACTCCAATC motifs detected in file\n",
      "\n",
      "Seq is: AAACTAATCAGTAAC\n",
      "0 AAACTAATCAGTAAC motifs detected in file\n",
      "\n",
      "Seq is: AACCCACCACTAATC\n",
      "0 AACCCACCACTAATC motifs detected in file\n",
      "\n",
      "Seq is: AGCCAACAACTACTC\n",
      "0 AGCCAACAACTACTC motifs detected in file\n",
      "\n",
      "Seq is: TATAAACCTCCACCC\n",
      "0 TATAAACCTCCACCC motifs detected in file\n",
      "\n",
      "Seq is: CTAGTTATCTCCCCC\n",
      "0 CTAGTTATCTCCCCC motifs detected in file\n",
      "\n",
      "Seq is: TTTATCTTACTAGTC\n",
      "0 TTTATCTTACTAGTC motifs detected in file\n",
      "\n",
      "Seq is: ATGACGCCGCCCATC\n",
      "0 ATGACGCCGCCCATC motifs detected in file\n",
      "\n",
      "Seq is: CGGATCCTAACACCC\n",
      "0 CGGATCCTAACACCC motifs detected in file\n",
      "\n",
      "Seq is: CCGTTACTTACTTCG\n",
      "0 CCGTTACTTACTTCG motifs detected in file\n",
      "\n",
      "Seq is: CTTGTCTAAAAGCAA\n",
      "0 CTTGTCTAAAAGCAA motifs detected in file\n",
      "\n",
      "Seq is: GTAAAACCAAGAGTC\n",
      "0 GTAAAACCAAGAGTC motifs detected in file\n",
      "\n",
      "Seq is: CCAAACAATATTAAA\n",
      "0 CCAAACAATATTAAA motifs detected in file\n",
      "\n",
      "Seq is: AAATAAAAATACCCG\n",
      "0 AAATAAAAATACCCG motifs detected in file\n",
      "\n",
      "Seq is: AAACCACTCCCTCCA\n",
      "0 AAACCACTCCCTCCA motifs detected in file\n",
      "\n",
      "Seq is: TCCACCATCAAACAC\n",
      "0 TCCACCATCAAACAC motifs detected in file\n",
      "\n",
      "Seq is: AAATTATAAAAATCA\n",
      "0 AAATTATAAAAATCA motifs detected in file\n",
      "\n",
      "Seq is: ACCCACTACTCCTTA\n",
      "0 ACCCACTACTCCTTA motifs detected in file\n",
      "\n",
      "Seq is: TCACATTCCTCATAC\n",
      "0 TCACATTCCTCATAC motifs detected in file\n",
      "\n",
      "Seq is: CCCCATTAATACCAC\n",
      "0 CCCCATTAATACCAC motifs detected in file\n",
      "\n",
      "Seq is: CCGGTATGCCATCAT\n",
      "0 CCGGTATGCCATCAT motifs detected in file\n",
      "\n",
      "Seq is: AAATCAACCCCGTAA\n",
      "0 AAATCAACCCCGTAA motifs detected in file\n",
      "\n",
      "Seq is: AGTGACATTTCTCCA\n",
      "0 AGTGACATTTCTCCA motifs detected in file\n",
      "\n",
      "Seq is: CAACCAAGAGAATTA\n",
      "0 CAACCAAGAGAATTA motifs detected in file\n",
      "\n",
      "Seq is: GAAATTGAATTTTCC\n",
      "0 GAAATTGAATTTTCC motifs detected in file\n",
      "\n",
      "Seq is: AATTACGATTCAATC\n",
      "0 AATTACGATTCAATC motifs detected in file\n",
      "\n",
      "Seq is: GAAATGTCACCCGAG\n",
      "0 GAAATGTCACCCGAG motifs detected in file\n",
      "\n",
      "Seq is: ATCTCCCCCATCTCC\n",
      "0 ATCTCCCCCATCTCC motifs detected in file\n",
      "\n",
      "Seq is: CCCACGATATCCCAT\n",
      "0 CCCACGATATCCCAT motifs detected in file\n",
      "\n",
      "Seq is: TTATGCCCGGGATAA\n",
      "0 TTATGCCCGGGATAA motifs detected in file\n",
      "\n",
      "Seq is: AGTTTCTAAAAATCC\n",
      "0 AGTTTCTAAAAATCC motifs detected in file\n",
      "\n",
      "Seq is: CCTTAATTAAGATCC\n",
      "0 CCTTAATTAAGATCC motifs detected in file\n",
      "\n",
      "Seq is: TGCTACCCGTACTTT\n",
      "0 TGCTACCCGTACTTT motifs detected in file\n",
      "\n",
      "Seq is: GACACCAAACGAGGT\n",
      "0 GACACCAAACGAGGT motifs detected in file\n",
      "\n",
      "Seq is: GTCTAAAGTCTCGTA\n",
      "0 GTCTAAAGTCTCGTA motifs detected in file\n",
      "\n",
      "Seq is: ACTGGCATCATATGG\n",
      "0 ACTGGCATCATATGG motifs detected in file\n",
      "\n",
      "Seq is: GGGCCAGCACATCGC\n",
      "0 GGGCCAGCACATCGC motifs detected in file\n",
      "\n",
      "Seq is: CACTTTCACCAAACC\n",
      "0 CACTTTCACCAAACC motifs detected in file\n",
      "\n",
      "Seq is: AAATCTGCAGGCCCT\n",
      "0 AAATCTGCAGGCCCT motifs detected in file\n",
      "\n",
      "Seq is: TAACGTAGACAAAAA\n",
      "0 TAACGTAGACAAAAA motifs detected in file\n",
      "\n",
      "Seq is: TTCGGATTACACCCC\n",
      "0 TTCGGATTACACCCC motifs detected in file\n",
      "\n",
      "Seq is: TGTCGAGTACTCACG\n",
      "0 TGTCGAGTACTCACG motifs detected in file\n",
      "\n",
      "Seq is: TTCTGCAGAACACTA\n",
      "0 TTCTGCAGAACACTA motifs detected in file\n",
      "\n",
      "Seq is: CATTAATAATATGCT\n",
      "0 CATTAATAATATGCT motifs detected in file\n",
      "\n",
      "Seq is: TACCCCCGAAGTTAG\n",
      "0 TACCCCCGAAGTTAG motifs detected in file\n",
      "\n",
      "Seq is: CCCTCATGATGAGCT\n",
      "0 CCCTCATGATGAGCT motifs detected in file\n",
      "\n",
      "Seq is: AACAGTTGCAGTTCC\n",
      "0 AACAGTTGCAGTTCC motifs detected in file\n",
      "\n",
      "Seq is: TCAGCGTCCAGCGGA\n",
      "0 TCAGCGTCCAGCGGA motifs detected in file\n",
      "\n",
      "Seq is: CCAAGATCCTTATTA\n",
      "0 CCAAGATCCTTATTA motifs detected in file\n",
      "\n",
      "Seq is: CCCCCTTCATACATC\n",
      "0 CCCCCTTCATACATC motifs detected in file\n",
      "\n",
      "Seq is: CTCAACTTCTAATCA\n",
      "0 CTCAACTTCTAATCA motifs detected in file\n",
      "\n",
      "Seq is: GGCGGCATCAGCCAC\n",
      "0 GGCGGCATCAGCCAC motifs detected in file\n",
      "\n",
      "Seq is: ATGAGCATCCAAGTC\n",
      "0 ATGAGCATCCAAGTC motifs detected in file\n",
      "\n",
      "Seq is: ATGGTAACCACCGGT\n",
      "0 ATGGTAACCACCGGT motifs detected in file\n",
      "\n",
      "Seq is: TAACTAAACTACCAT\n",
      "0 TAACTAAACTACCAT motifs detected in file\n",
      "\n",
      "Seq is: TCCCTCCCTAGCAAC\n",
      "0 TCCCTCCCTAGCAAC motifs detected in file\n",
      "\n",
      "Seq is: TGGAGCAGACAATAC\n",
      "0 TGGAGCAGACAATAC motifs detected in file\n",
      "\n",
      "Seq is: ATTTCCTACGCATCC\n",
      "0 ATTTCCTACGCATCC motifs detected in file\n",
      "\n",
      "Seq is: CTACCCTCCCGCTAC\n",
      "0 CTACCCTCCCGCTAC motifs detected in file\n",
      "\n",
      "Seq is: TCCTGATCCTACTAC\n",
      "0 TCCTGATCCTACTAC motifs detected in file\n",
      "\n",
      "Seq is: CGCCCGTCCTATCAA\n",
      "0 CGCCCGTCCTATCAA motifs detected in file\n",
      "\n",
      "Seq is: GTCTGCCAAAGATAA\n",
      "0 GTCTGCCAAAGATAA motifs detected in file\n",
      "\n",
      "Seq is: AGGACTCGCAGACTC\n",
      "0 AGGACTCGCAGACTC motifs detected in file\n",
      "\n",
      "Seq is: TACAATCATAATCAA\n",
      "0 TACAATCATAATCAA motifs detected in file\n",
      "\n",
      "Seq is: TCAAAACAACACTCA\n",
      "0 TCAAAACAACACTCA motifs detected in file\n",
      "\n",
      "Seq is: CAATCCTTTTCCCGT\n",
      "0 CAATCCTTTTCCCGT motifs detected in file\n",
      "\n",
      "Seq is: ATGTCCTGATCCTTC\n",
      "0 ATGTCCTGATCCTTC motifs detected in file\n",
      "\n",
      "Seq is: GTCTATTCCTTTTAC\n",
      "0 GTCTATTCCTTTTAC motifs detected in file\n",
      "\n",
      "Seq is: TAATACTCCCGCACT\n",
      "0 TAATACTCCCGCACT motifs detected in file\n",
      "\n",
      "Seq is: AGTACTTTCCACTAT\n",
      "0 AGTACTTTCCACTAT motifs detected in file\n",
      "\n",
      "Seq is: TCGAGAAGATACTAT\n",
      "0 TCGAGAAGATACTAT motifs detected in file\n",
      "\n",
      "Seq is: CCCCTTCATCGCAGA\n",
      "0 CCCCTTCATCGCAGA motifs detected in file\n",
      "\n",
      "Seq is: ACATCTGGATGAACG\n",
      "0 ACATCTGGATGAACG motifs detected in file\n",
      "\n",
      "Seq is: CGACGTACACGGTAA\n",
      "0 CGACGTACACGGTAA motifs detected in file\n",
      "\n",
      "Seq is: TTCTATATATCCTAA\n",
      "0 TTCTATATATCCTAA motifs detected in file\n",
      "\n",
      "Seq is: ATCGGATATTAAATT\n",
      "0 ATCGGATATTAAATT motifs detected in file\n",
      "\n",
      "Seq is: GAAACTGTTTCAATA\n",
      "0 GAAACTGTTTCAATA motifs detected in file\n",
      "\n",
      "Seq is: CTTTACCAAAAAGAT\n",
      "0 CTTTACCAAAAAGAT motifs detected in file\n",
      "\n",
      "Seq is: TATGGAAAAACTTTT\n",
      "0 TATGGAAAAACTTTT motifs detected in file\n",
      "\n",
      "Seq is: TCAGTACCTCCGGTA\n",
      "0 TCAGTACCTCCGGTA motifs detected in file\n",
      "\n",
      "Seq is: CCCCAACCGAACTTT\n",
      "0 CCCCAACCGAACTTT motifs detected in file\n",
      "\n",
      "Seq is: GGTCGAAACCCCCCA\n",
      "0 GGTCGAAACCCCCCA motifs detected in file\n",
      "\n",
      "Seq is: AGCTAAGGAAGGAAA\n",
      "0 AGCTAAGGAAGGAAA motifs detected in file\n",
      "\n",
      "Seq is: AAACAGATCTAAAAT\n",
      "0 AAACAGATCTAAAAT motifs detected in file\n",
      "\n",
      "Seq is: ACATATGCCCAAGAA\n",
      "0 ACATATGCCCAAGAA motifs detected in file\n",
      "\n",
      "Seq is: GCTTACACACCATCC\n",
      "0 GCTTACACACCATCC motifs detected in file\n",
      "\n",
      "Seq is: CACCCCCCGTAGGTA\n",
      "0 CACCCCCCGTAGGTA motifs detected in file\n",
      "\n",
      "Seq is: TATCAGTGAGGTCCA\n",
      "0 TATCAGTGAGGTCCA motifs detected in file\n",
      "\n",
      "Seq is: AATACCTCCCAAGAA\n",
      "0 AATACCTCCCAAGAA motifs detected in file\n",
      "\n",
      "Seq is: GATGATAATCCTGAA\n",
      "0 GATGATAATCCTGAA motifs detected in file\n",
      "\n",
      "Seq is: AAGCGAAGCTTCGCT\n",
      "0 AAGCGAAGCTTCGCT motifs detected in file\n",
      "\n",
      "Seq is: TCCGAAGAGTTTAGT\n",
      "0 TCCGAAGAGTTTAGT motifs detected in file\n",
      "\n",
      "Seq is: ACTTTTAATAATTAT\n",
      "0 ACTTTTAATAATTAT motifs detected in file\n",
      "\n",
      "Seq is: AATGACGACAATCTC\n",
      "0 AATGACGACAATCTC motifs detected in file\n",
      "\n",
      "Seq is: TTTACTTACTCGGAT\n",
      "0 TTTACTTACTCGGAT motifs detected in file\n",
      "\n",
      "Seq is: GTCTACTATCCTACA\n",
      "0 GTCTACTATCCTACA motifs detected in file\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Seq is: AAGTACACCACATAC\n",
      "0 AAGTACACCACATAC motifs detected in file\n",
      "\n",
      "Seq is: GTAGCCCCATCAGGC\n",
      "0 GTAGCCCCATCAGGC motifs detected in file\n",
      "\n",
      "Seq is: TCATTGCAGCCCCGT\n",
      "0 TCATTGCAGCCCCGT motifs detected in file\n",
      "\n",
      "Seq is: AAGGCCTATCCGGCT\n",
      "0 AAGGCCTATCCGGCT motifs detected in file\n",
      "\n",
      "Seq is: CTTTCACAACACCCT\n",
      "0 CTTTCACAACACCCT motifs detected in file\n",
      "\n",
      "Seq is: TCTTTCAATCTAAAT\n",
      "0 TCTTTCAATCTAAAT motifs detected in file\n",
      "\n",
      "Seq is: GCGGCTACTTATACT\n",
      "0 GCGGCTACTTATACT motifs detected in file\n",
      "\n",
      "Seq is: ATCACTTTACCTAAA\n",
      "0 ATCACTTTACCTAAA motifs detected in file\n",
      "\n",
      "Seq is: ACCGCATCCAAACCA\n",
      "0 ACCGCATCCAAACCA motifs detected in file\n",
      "\n",
      "Seq is: GATCCCACATCGGAC\n",
      "0 GATCCCACATCGGAC motifs detected in file\n",
      "\n",
      "Seq is: TCTTATCCCCTTTAG\n",
      "0 TCTTATCCCCTTTAG motifs detected in file\n",
      "\n",
      "Seq is: TCACTTACTTCGGAG\n",
      "0 TCACTTACTTCGGAG motifs detected in file\n",
      "\n",
      "Seq is: GATACTACCGTTTAT\n",
      "0 GATACTACCGTTTAT motifs detected in file\n",
      "\n",
      "Seq is: GTCGAGGATAACTAT\n",
      "0 GTCGAGGATAACTAT motifs detected in file\n",
      "\n",
      "Seq is: CCTGTATCACCTTCA\n",
      "0 CCTGTATCACCTTCA motifs detected in file\n",
      "\n",
      "Seq is: CCCGATGTTGCATCA\n",
      "0 CCCGATGTTGCATCA motifs detected in file\n",
      "\n",
      "Seq is: TGCACAGCACATCAT\n",
      "0 TGCACAGCACATCAT motifs detected in file\n",
      "\n",
      "Seq is: GCTACAGATCACTAC\n",
      "0 GCTACAGATCACTAC motifs detected in file\n",
      "\n",
      "Seq is: TCAAACGATTATGTT\n",
      "0 TCAAACGATTATGTT motifs detected in file\n",
      "\n",
      "Seq is: ACGGTCAGTCCGGTG\n",
      "0 ACGGTCAGTCCGGTG motifs detected in file\n",
      "\n",
      "Seq is: GATGCCACTTTTCTT\n",
      "0 GATGCCACTTTTCTT motifs detected in file\n",
      "\n",
      "Seq is: TCTACTTAGGATCCC\n",
      "0 TCTACTTAGGATCCC motifs detected in file\n",
      "\n",
      "Seq is: GAGTCTCGTGACGTC\n",
      "0 GAGTCTCGTGACGTC motifs detected in file\n",
      "\n",
      "Seq is: GTCTAGTAAAGTATA\n",
      "0 GTCTAGTAAAGTATA motifs detected in file\n",
      "\n",
      "Seq is: ACGAAGGCACCTCAC\n",
      "0 ACGAAGGCACCTCAC motifs detected in file\n",
      "\n",
      "Seq is: ACCGCTCAGTCGATT\n",
      "0 ACCGCTCAGTCGATT motifs detected in file\n",
      "\n",
      "Seq is: TATGAAACTGCGGCC\n",
      "0 TATGAAACTGCGGCC motifs detected in file\n",
      "\n",
      "Seq is: ACCCCTATCGCTACT\n",
      "0 ACCCCTATCGCTACT motifs detected in file\n",
      "\n",
      "Seq is: AATACCATCGCCTCC\n",
      "0 AATACCATCGCCTCC motifs detected in file\n",
      "\n",
      "Seq is: ACTTTATACGAGCAC\n",
      "0 ACTTTATACGAGCAC motifs detected in file\n",
      "\n",
      "Seq is: ACAGATGCAGATAAG\n",
      "0 ACAGATGCAGATAAG motifs detected in file\n",
      "\n",
      "Seq is: GATGACATTTATATA\n",
      "0 GATGACATTTATATA motifs detected in file\n",
      "\n",
      "Seq is: CCACACGAGTGTGCT\n",
      "0 CCACACGAGTGTGCT motifs detected in file\n",
      "\n",
      "Seq is: ATTTGGGATCCTTCG\n",
      "0 ATTTGGGATCCTTCG motifs detected in file\n",
      "\n",
      "Seq is: GTTAACTATAGTATC\n",
      "0 GTTAACTATAGTATC motifs detected in file\n",
      "\n",
      "Seq is: GAGTCTGGTATGGAT\n",
      "0 GAGTCTGGTATGGAT motifs detected in file\n",
      "\n",
      "Seq is: ACATAGGTTTACCAA\n",
      "0 ACATAGGTTTACCAA motifs detected in file\n",
      "\n",
      "Seq is: GAAAAAAAGGCCTCA\n",
      "0 GAAAAAAAGGCCTCA motifs detected in file\n",
      "\n",
      "Seq is: TCATTCAATGTTATA\n",
      "0 TCATTCAATGTTATA motifs detected in file\n",
      "\n",
      "Seq is: CCCTCTAATAAGGCT\n",
      "0 CCCTCTAATAAGGCT motifs detected in file\n",
      "\n",
      "Seq is: TCGGACCATCCTATT\n",
      "0 TCGGACCATCCTATT motifs detected in file\n",
      "\n",
      "Seq is: CTTATATTTGAAGTC\n",
      "0 CTTATATTTGAAGTC motifs detected in file\n",
      "\n",
      "Seq is: CCACTGGCTTTTTAG\n",
      "0 CCACTGGCTTTTTAG motifs detected in file\n",
      "\n",
      "Seq is: TCTTATCCACAACCA\n",
      "0 TCTTATCCACAACCA motifs detected in file\n",
      "\n",
      "Seq is: TATCTTACCCCAGAG\n",
      "0 TATCTTACCCCAGAG motifs detected in file\n",
      "\n",
      "Seq is: GAGGAGGCCGCCCCA\n",
      "0 GAGGAGGCCGCCCCA motifs detected in file\n",
      "\n",
      "Seq is: GCTTCTTCCACCACA\n",
      "0 GCTTCTTCCACCACA motifs detected in file\n",
      "\n",
      "Seq is: ACTCCAACGCCAGAC\n",
      "0 ACTCCAACGCCAGAC motifs detected in file\n",
      "\n",
      "Seq is: AATCATCATATCACT\n",
      "0 AATCATCATATCACT motifs detected in file\n",
      "\n",
      "Seq is: ACGGTCGTCGATCCT\n",
      "0 ACGGTCGTCGATCCT motifs detected in file\n",
      "\n",
      "Seq is: GACCCTCTCTATCCT\n",
      "0 GACCCTCTCTATCCT motifs detected in file\n",
      "\n",
      "Seq is: CTTCATCCTGACGAC\n",
      "0 CTTCATCCTGACGAC motifs detected in file\n",
      "\n",
      "Seq is: ACTAATCCTGCCTAG\n",
      "0 ACTAATCCTGCCTAG motifs detected in file\n",
      "\n",
      "Seq is: TCTGCTTCTCCCCGC\n",
      "0 TCTGCTTCTCCCCGC motifs detected in file\n",
      "\n",
      "Seq is: AAACCATAACCCAAT\n",
      "0 AAACCATAACCCAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCGTCCCCCAAAAT\n",
      "0 ACCGTCCCCCAAAAT motifs detected in file\n",
      "\n",
      "Seq is: ATAACTATTAACAAC\n",
      "0 ATAACTATTAACAAC motifs detected in file\n",
      "\n",
      "Seq is: ACTACTTTAACTACC\n",
      "0 ACTACTTTAACTACC motifs detected in file\n",
      "\n",
      "Seq is: GGGGATTCTATCTCC\n",
      "0 GGGGATTCTATCTCC motifs detected in file\n",
      "\n",
      "Seq is: TCTGTGGACGATCCA\n",
      "0 TCTGTGGACGATCCA motifs detected in file\n",
      "\n",
      "Seq is: CATTCCTCTTCTACC\n",
      "0 CATTCCTCTTCTACC motifs detected in file\n",
      "\n",
      "Seq is: AATCCAGATGCCTCC\n",
      "0 AATCCAGATGCCTCC motifs detected in file\n",
      "\n",
      "Seq is: GAGGTCCCACCCTCA\n",
      "0 GAGGTCCCACCCTCA motifs detected in file\n",
      "\n",
      "Seq is: TCAAGGGACGATTCC\n",
      "0 TCAAGGGACGATTCC motifs detected in file\n",
      "\n",
      "Seq is: CTCCCATCTGACAAG\n",
      "0 CTCCCATCTGACAAG motifs detected in file\n",
      "\n",
      "Seq is: TTGGACAAGGACGAG\n",
      "0 TTGGACAAGGACGAG motifs detected in file\n",
      "\n",
      "Seq is: GCCGGAGGTGATATC\n",
      "0 GCCGGAGGTGATATC motifs detected in file\n",
      "\n",
      "Seq is: GTCTACGCTCGTCCT\n",
      "0 GTCTACGCTCGTCCT motifs detected in file\n",
      "\n",
      "Seq is: CATCCTCTCTCCCTC\n",
      "0 CATCCTCTCTCCCTC motifs detected in file\n",
      "\n",
      "Seq is: CATTCTCAGTCTTCG\n",
      "0 CATTCTCAGTCTTCG motifs detected in file\n",
      "\n",
      "Seq is: AATACAACAAATACG\n",
      "0 AATACAACAAATACG motifs detected in file\n",
      "\n",
      "Seq is: CCCCTTTGCGGTATA\n",
      "0 CCCCTTTGCGGTATA motifs detected in file\n",
      "\n",
      "Seq is: GCCCCCGTGGCTAAT\n",
      "0 GCCCCCGTGGCTAAT motifs detected in file\n",
      "\n",
      "Seq is: AATCCCCTTGATCAG\n",
      "0 AATCCCCTTGATCAG motifs detected in file\n",
      "\n",
      "Seq is: TCAACGGTTTCGGAG\n",
      "0 TCAACGGTTTCGGAG motifs detected in file\n",
      "\n",
      "Seq is: GCTAATACTACCCAT\n",
      "0 GCTAATACTACCCAT motifs detected in file\n",
      "\n",
      "Seq is: AATGATACTTCTTCT\n",
      "0 AATGATACTTCTTCT motifs detected in file\n",
      "\n",
      "Seq is: AATAATGTTTACGTA\n",
      "0 AATAATGTTTACGTA motifs detected in file\n",
      "\n",
      "Seq is: CCCGACACTGCTATT\n",
      "0 CCCGACACTGCTATT motifs detected in file\n",
      "\n",
      "Seq is: GCAACATCTACACCA\n",
      "0 GCAACATCTACACCA motifs detected in file\n",
      "\n",
      "Seq is: GCAATGGATCTTCCA\n",
      "0 GCAATGGATCTTCCA motifs detected in file\n",
      "\n",
      "Seq is: ACGGACCGACCGGGT\n",
      "0 ACGGACCGACCGGGT motifs detected in file\n",
      "\n",
      "Seq is: CGAGCCGAGCTTATT\n",
      "0 CGAGCCGAGCTTATT motifs detected in file\n",
      "\n",
      "Seq is: CCTCCGAATCTCGAC\n",
      "0 CCTCCGAATCTCGAC motifs detected in file\n",
      "\n",
      "Seq is: ACGGATCCTGAGGTC\n",
      "0 ACGGATCCTGAGGTC motifs detected in file\n",
      "\n",
      "Seq is: GAGTACGCGGCTTAT\n",
      "0 GAGTACGCGGCTTAT motifs detected in file\n",
      "\n",
      "Seq is: TATCCATATCACAAG\n",
      "0 TATCCATATCACAAG motifs detected in file\n",
      "\n",
      "Seq is: GTTACAGAAACACCA\n",
      "0 GTTACAGAAACACCA motifs detected in file\n",
      "\n",
      "Seq is: AACATCTCTTATCAG\n",
      "0 AACATCTCTTATCAG motifs detected in file\n",
      "\n",
      "Seq is: TTGCCAGCCGCTTGT\n",
      "0 TTGCCAGCCGCTTGT motifs detected in file\n",
      "\n",
      "Seq is: AGTCACCCCCACTCC\n",
      "0 AGTCACCCCCACTCC motifs detected in file\n",
      "\n",
      "Seq is: ATTTTACCGACTCAC\n",
      "0 ATTTTACCGACTCAC motifs detected in file\n",
      "\n",
      "Seq is: TTCGTAACCTGACAT\n",
      "0 TTCGTAACCTGACAT motifs detected in file\n",
      "\n",
      "Seq is: TTAGATTTCTGTCCC\n",
      "0 TTAGATTTCTGTCCC motifs detected in file\n",
      "\n",
      "Seq is: CAATCCGGAGAAAAA\n",
      "0 CAATCCGGAGAAAAA motifs detected in file\n",
      "\n",
      "Seq is: TGGTCGAGGCTCCAC\n",
      "0 TGGTCGAGGCTCCAC motifs detected in file\n",
      "\n",
      "Seq is: TAAAAGTATAACTTA\n",
      "0 TAAAAGTATAACTTA motifs detected in file\n",
      "\n",
      "Seq is: ACGTTTAAGCTTCTT\n",
      "0 ACGTTTAAGCTTCTT motifs detected in file\n",
      "\n",
      "Seq is: CGTCGAAGTTTGGAC\n",
      "0 CGTCGAAGTTTGGAC motifs detected in file\n",
      "\n",
      "Seq is: GGCCCCGAAGAGGGC\n",
      "0 GGCCCCGAAGAGGGC motifs detected in file\n",
      "\n",
      "Seq is: GGAAAAAAGGGCCGC\n",
      "0 GGAAAAAAGGGCCGC motifs detected in file\n",
      "\n",
      "Seq is: CGCCCTCTTCATCTA\n",
      "0 CGCCCTCTTCATCTA motifs detected in file\n",
      "\n",
      "Seq is: ACTTCGGTCAACTAA\n",
      "0 ACTTCGGTCAACTAA motifs detected in file\n",
      "\n",
      "Seq is: TCCCACGAATCGACA\n",
      "0 TCCCACGAATCGACA motifs detected in file\n",
      "\n",
      "Seq is: ATTGATTCACAAACA\n",
      "0 ATTGATTCACAAACA motifs detected in file\n",
      "\n",
      "Seq is: CCCAAATTCAGGGTA\n",
      "0 CCCAAATTCAGGGTA motifs detected in file\n",
      "\n",
      "Seq is: ACCAGATCATTCCCG\n",
      "0 ACCAGATCATTCCCG motifs detected in file\n",
      "\n",
      "Seq is: AATCGAATTAATTTC\n",
      "0 AATCGAATTAATTTC motifs detected in file\n",
      "\n",
      "Seq is: ACCGACTAAACGCAA\n",
      "0 ACCGACTAAACGCAA motifs detected in file\n",
      "\n",
      "Seq is: GTCAACTACGTCTCA\n",
      "0 GTCAACTACGTCTCA motifs detected in file\n",
      "\n",
      "Seq is: CCCCAAAACGTCAGG\n",
      "0 CCCCAAAACGTCAGG motifs detected in file\n",
      "\n",
      "Seq is: AATCGACAATGTCTT\n",
      "0 AATCGACAATGTCTT motifs detected in file\n",
      "\n",
      "Seq is: TAATTCATAACGTTG\n",
      "0 TAATTCATAACGTTG motifs detected in file\n",
      "\n",
      "Seq is: AATGACTCCCGAAAC\n",
      "0 AATGACTCCCGAAAC motifs detected in file\n",
      "\n",
      "Seq is: TTCCGAGAACCAGAC\n",
      "0 TTCCGAGAACCAGAC motifs detected in file\n",
      "\n",
      "Seq is: ATAAATTGGATTTAA\n",
      "0 ATAAATTGGATTTAA motifs detected in file\n",
      "\n",
      "Seq is: AGATATTCTAATAAT\n",
      "0 AGATATTCTAATAAT motifs detected in file\n",
      "\n",
      "Seq is: CATATTTTCCCCTCT\n",
      "0 CATATTTTCCCCTCT motifs detected in file\n",
      "\n",
      "Seq is: ATCCATCCTCATCGC\n",
      "0 ATCCATCCTCATCGC motifs detected in file\n",
      "\n",
      "Seq is: ACCATTCCCGCTACT\n",
      "0 ACCATTCCCGCTACT motifs detected in file\n",
      "\n",
      "Seq is: CACACCCCTCCTTAC\n",
      "0 CACACCCCTCCTTAC motifs detected in file\n",
      "\n",
      "Seq is: CCCACCCAAAACATA\n",
      "0 CCCACCCAAAACATA motifs detected in file\n",
      "\n",
      "Seq is: CAAGTTTGACAGTAA\n",
      "0 CAAGTTTGACAGTAA motifs detected in file\n",
      "\n",
      "Seq is: AATAAAAATGCAACA\n",
      "0 AATAAAAATGCAACA motifs detected in file\n",
      "\n",
      "Seq is: ATCTATACCCCTCAT\n",
      "0 ATCTATACCCCTCAT motifs detected in file\n",
      "\n",
      "Seq is: CACACTAACTCCACC\n",
      "0 CACACTAACTCCACC motifs detected in file\n",
      "\n",
      "Seq is: TCATCTAATCCGCAT\n",
      "0 TCATCTAATCCGCAT motifs detected in file\n",
      "\n",
      "Seq is: CCATCTTCATCTCCA\n",
      "0 CCATCTTCATCTCCA motifs detected in file\n",
      "\n",
      "Seq is: ATTCCTCCCACTACC\n",
      "0 ATTCCTCCCACTACC motifs detected in file\n",
      "\n",
      "Seq is: ACCGATACTACCACC\n",
      "0 ACCGATACTACCACC motifs detected in file\n",
      "\n",
      "Seq is: CCTACTACTCCGATA\n",
      "0 CCTACTACTCCGATA motifs detected in file\n",
      "\n",
      "Seq is: ACAAAAAACACTTAA\n",
      "0 ACAAAAAACACTTAA motifs detected in file\n",
      "\n",
      "Seq is: GAAGCTATTACCGGG\n",
      "0 GAAGCTATTACCGGG motifs detected in file\n",
      "\n",
      "Seq is: TAAACCCGTTTTTCG\n",
      "0 TAAACCCGTTTTTCG motifs detected in file\n",
      "\n",
      "Seq is: GCCAATCGCCCCCGT\n",
      "0 GCCAATCGCCCCCGT motifs detected in file\n",
      "\n",
      "Seq is: CCGGAGGATCCCTCT\n",
      "0 CCGGAGGATCCCTCT motifs detected in file\n",
      "\n",
      "Seq is: CCTCCCACCTACCTT\n",
      "0 CCTCCCACCTACCTT motifs detected in file\n",
      "\n",
      "Seq is: AATTCCCACAATCAG\n",
      "0 AATTCCCACAATCAG motifs detected in file\n",
      "\n",
      "Seq is: TACAATCGAACAAAG\n",
      "0 TACAATCGAACAAAG motifs detected in file\n",
      "\n",
      "Seq is: TCCACGCTCTATCAT\n",
      "0 TCCACGCTCTATCAT motifs detected in file\n",
      "\n",
      "Seq is: CATCCCAGCACCACG\n",
      "0 CATCCCAGCACCACG motifs detected in file\n",
      "\n",
      "Seq is: ACCTCAAATTCAACT\n",
      "0 ACCTCAAATTCAACT motifs detected in file\n",
      "\n",
      "Seq is: CATCATCCTTACGCC\n",
      "0 CATCATCCTTACGCC motifs detected in file\n",
      "\n",
      "Seq is: ATCATCAATCCTATT\n",
      "0 ATCATCAATCCTATT motifs detected in file\n",
      "\n",
      "Seq is: ATATTTATCAATTTA\n",
      "0 ATATTTATCAATTTA motifs detected in file\n",
      "\n",
      "Seq is: ATTCTTACCAATACA\n",
      "0 ATTCTTACCAATACA motifs detected in file\n",
      "\n",
      "Seq is: ATCCCAACATGCCAT\n",
      "0 ATCCCAACATGCCAT motifs detected in file\n",
      "\n",
      "Seq is: CTTGACGATAATAAG\n",
      "0 CTTGACGATAATAAG motifs detected in file\n",
      "\n",
      "Seq is: TAGGATACACCCATT\n",
      "0 TAGGATACACCCATT motifs detected in file\n",
      "\n",
      "Seq is: AACTCCTCATACGAT\n",
      "0 AACTCCTCATACGAT motifs detected in file\n",
      "\n",
      "Seq is: TCTAAAACGCATCGA\n",
      "0 TCTAAAACGCATCGA motifs detected in file\n",
      "\n",
      "Seq is: CCCAAACCAAATTAG\n",
      "0 CCCAAACCAAATTAG motifs detected in file\n",
      "\n",
      "Seq is: GTGGAGTTGACGGAC\n",
      "0 GTGGAGTTGACGGAC motifs detected in file\n",
      "\n",
      "Seq is: GATACTACCTATTCT\n",
      "0 GATACTACCTATTCT motifs detected in file\n",
      "\n",
      "Seq is: AACTCTCTCACTCCT\n",
      "0 AACTCTCTCACTCCT motifs detected in file\n",
      "\n",
      "Seq is: CTTCCGAATGCAAAT\n",
      "0 CTTCCGAATGCAAAT motifs detected in file\n",
      "\n",
      "Seq is: CACTCCCTCTCTAAA\n",
      "0 CACTCCCTCTCTAAA motifs detected in file\n",
      "\n",
      "Seq is: CCATATACTAACTCT\n",
      "0 CCATATACTAACTCT motifs detected in file\n",
      "\n",
      "Seq is: ACCCCCGATCAAAAA\n",
      "0 ACCCCCGATCAAAAA motifs detected in file\n",
      "\n",
      "Seq is: CAGTACACTCTTCTT\n",
      "0 CAGTACACTCTTCTT motifs detected in file\n",
      "\n",
      "Seq is: CGTCCGGCCTACAGT\n",
      "0 CGTCCGGCCTACAGT motifs detected in file\n",
      "\n",
      "Seq is: CTCCCCACGGAACCC\n",
      "0 CTCCCCACGGAACCC motifs detected in file\n",
      "\n",
      "Seq is: ATTGGAGACCCTGAG\n",
      "0 ATTGGAGACCCTGAG motifs detected in file\n",
      "\n",
      "Seq is: TCTTCACTTTCCCCC\n",
      "0 TCTTCACTTTCCCCC motifs detected in file\n",
      "\n",
      "Seq is: GATAAGGATCAAAAT\n",
      "0 GATAAGGATCAAAAT motifs detected in file\n",
      "\n",
      "Seq is: AACGATATCGATAAT\n",
      "0 AACGATATCGATAAT motifs detected in file\n",
      "\n",
      "Seq is: ACTAATAATTACTAC\n",
      "0 ACTAATAATTACTAC motifs detected in file\n",
      "\n",
      "Seq is: TCATAACTAACCATC\n",
      "0 TCATAACTAACCATC motifs detected in file\n",
      "\n",
      "Seq is: ATAACCAATACCAAG\n",
      "0 ATAACCAATACCAAG motifs detected in file\n",
      "\n",
      "Seq is: TAACAGGCCTCTCAT\n",
      "0 TAACAGGCCTCTCAT motifs detected in file\n",
      "\n",
      "Seq is: ATAACAACTTCTCCT\n",
      "0 ATAACAACTTCTCCT motifs detected in file\n",
      "\n",
      "Seq is: ATCGATAATCTTCCT\n",
      "0 ATCGATAATCTTCCT motifs detected in file\n",
      "\n",
      "Seq is: AATACCTACGCCAAC\n",
      "0 AATACCTACGCCAAC motifs detected in file\n",
      "\n",
      "Seq is: GAACGCACTCCTTTA\n",
      "0 GAACGCACTCCTTTA motifs detected in file\n",
      "\n",
      "Seq is: TGAACTACCGTCGAA\n",
      "0 TGAACTACCGTCGAA motifs detected in file\n",
      "\n",
      "Seq is: GACACCTTGCTCCCA\n",
      "0 GACACCTTGCTCCCA motifs detected in file\n",
      "\n",
      "Seq is: AATAAAAAAACCAAT\n",
      "0 AATAAAAAAACCAAT motifs detected in file\n",
      "\n",
      "Seq is: CTTGACCTTATTTTC\n",
      "0 CTTGACCTTATTTTC motifs detected in file\n",
      "\n",
      "Seq is: GATCGTACAAATAAA\n",
      "0 GATCGTACAAATAAA motifs detected in file\n",
      "\n",
      "Seq is: GATCCGGATGAGTCC\n",
      "0 GATCCGGATGAGTCC motifs detected in file\n",
      "\n",
      "Seq is: ATTTTTTAGTCACGC\n",
      "0 ATTTTTTAGTCACGC motifs detected in file\n",
      "\n",
      "Seq is: TCGAATCGCGACACT\n",
      "0 TCGAATCGCGACACT motifs detected in file\n",
      "\n",
      "Seq is: ACTCACACGGACGTT\n",
      "0 ACTCACACGGACGTT motifs detected in file\n",
      "\n",
      "Seq is: TCTACCATCTCATCT\n",
      "0 TCTACCATCTCATCT motifs detected in file\n",
      "\n",
      "Seq is: ACTGCCCAACCCGGT\n",
      "0 ACTGCCCAACCCGGT motifs detected in file\n",
      "\n",
      "Seq is: CCCCTAATTAATCAT\n",
      "0 CCCCTAATTAATCAT motifs detected in file\n",
      "\n",
      "Seq is: GCCCTTCCCATATTG\n",
      "0 GCCCTTCCCATATTG motifs detected in file\n",
      "\n",
      "Seq is: GTTGTAAAAGCCCCA\n",
      "0 GTTGTAAAAGCCCCA motifs detected in file\n",
      "\n",
      "Seq is: TACCCGGGCTATCGA\n",
      "0 TACCCGGGCTATCGA motifs detected in file\n",
      "\n",
      "Seq is: ATAAATCGACTGGAA\n",
      "0 ATAAATCGACTGGAA motifs detected in file\n",
      "\n",
      "Seq is: TGAAATCCTACCCCA\n",
      "0 TGAAATCCTACCCCA motifs detected in file\n",
      "\n",
      "Seq is: CACTATCCACCGTGC\n",
      "0 CACTATCCACCGTGC motifs detected in file\n",
      "\n",
      "Seq is: CTCTTAAAACCTAAG\n",
      "0 CTCTTAAAACCTAAG motifs detected in file\n",
      "\n",
      "Seq is: AGTCCCTACCCAAGC\n",
      "0 AGTCCCTACCCAAGC motifs detected in file\n",
      "\n",
      "Seq is: TAAGAGTATCAGGAT\n",
      "0 TAAGAGTATCAGGAT motifs detected in file\n",
      "\n",
      "Seq is: CTTTATTCCCCCAAA\n",
      "0 CTTTATTCCCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: TTCGAGGATAATAAA\n",
      "0 TTCGAGGATAATAAA motifs detected in file\n",
      "\n",
      "Seq is: TGAGATAGTTTCATT\n",
      "0 TGAGATAGTTTCATT motifs detected in file\n",
      "\n",
      "Seq is: GAGAAAATAGTCTGT\n",
      "0 GAGAAAATAGTCTGT motifs detected in file\n",
      "\n",
      "Seq is: ATAAAGAATCCAAAC\n",
      "0 ATAAAGAATCCAAAC motifs detected in file\n",
      "\n",
      "Seq is: TCCCCCTTACGACCT\n",
      "0 TCCCCCTTACGACCT motifs detected in file\n",
      "\n",
      "Seq is: CTAACATTACCCATA\n",
      "0 CTAACATTACCCATA motifs detected in file\n",
      "\n",
      "Seq is: CCTCTGTATAGTATA\n",
      "0 CCTCTGTATAGTATA motifs detected in file\n",
      "\n",
      "Seq is: TTCATTACGATCCCA\n",
      "0 TTCATTACGATCCCA motifs detected in file\n",
      "\n",
      "Seq is: CTCACCATCCTTCAA\n",
      "0 CTCACCATCCTTCAA motifs detected in file\n",
      "\n",
      "Seq is: AAAAGTATCCTCCAC\n",
      "0 AAAAGTATCCTCCAC motifs detected in file\n",
      "\n",
      "Seq is: ATACTCAACCAGCAT\n",
      "0 ATACTCAACCAGCAT motifs detected in file\n",
      "\n",
      "Seq is: CGCCTTAGCCCCCAT\n",
      "0 CGCCTTAGCCCCCAT motifs detected in file\n",
      "\n",
      "Seq is: ACGACAAGCTTAAGT\n",
      "0 ACGACAAGCTTAAGT motifs detected in file\n",
      "\n",
      "Seq is: ATTCTTGTCCCTCCA\n",
      "0 ATTCTTGTCCCTCCA motifs detected in file\n",
      "\n",
      "Seq is: ATCTTCATCCCAGAA\n",
      "0 ATCTTCATCCCAGAA motifs detected in file\n",
      "\n",
      "Seq is: CCACTGTTTTATACA\n",
      "0 CCACTGTTTTATACA motifs detected in file\n",
      "\n",
      "Seq is: ACACATCTCAAGTCC\n",
      "0 ACACATCTCAAGTCC motifs detected in file\n",
      "\n",
      "Seq is: CCTCTCACGCAGTAT\n",
      "0 CCTCTCACGCAGTAT motifs detected in file\n",
      "\n",
      "Seq is: ACAACAAGGATCCTT\n",
      "0 ACAACAAGGATCCTT motifs detected in file\n",
      "\n",
      "Seq is: CTAACATCACCACTC\n",
      "0 CTAACATCACCACTC motifs detected in file\n",
      "\n",
      "Seq is: CCACAAATAATATTA\n",
      "0 CCACAAATAATATTA motifs detected in file\n",
      "\n",
      "Seq is: TTACAAACACATAAG\n",
      "0 TTACAAACACATAAG motifs detected in file\n",
      "\n",
      "Seq is: CCGATACTTCTTATC\n",
      "0 CCGATACTTCTTATC motifs detected in file\n",
      "\n",
      "Seq is: CCGCTTCCCCGGACG\n",
      "0 CCGCTTCCCCGGACG motifs detected in file\n",
      "\n",
      "Seq is: CCGCATAAGCTACAA\n",
      "0 CCGCATAAGCTACAA motifs detected in file\n",
      "\n",
      "Seq is: CTTCGGACTCTGATC\n",
      "0 CTTCGGACTCTGATC motifs detected in file\n",
      "\n",
      "Seq is: AAGCCTGAGGGGAAG\n",
      "0 AAGCCTGAGGGGAAG motifs detected in file\n",
      "\n",
      "Seq is: CCGTTCCAGCTTCCC\n",
      "0 CCGTTCCAGCTTCCC motifs detected in file\n",
      "\n",
      "Seq is: CCAAGCCAACCAGAG\n",
      "0 CCAAGCCAACCAGAG motifs detected in file\n",
      "\n",
      "Seq is: ACGATCACACCTCTA\n",
      "0 ACGATCACACCTCTA motifs detected in file\n",
      "\n",
      "Seq is: TTTAGTATAATACCG\n",
      "0 TTTAGTATAATACCG motifs detected in file\n",
      "\n",
      "Seq is: GTTCCCAGTACTACC\n",
      "0 GTTCCCAGTACTACC motifs detected in file\n",
      "\n",
      "Seq is: GTCCTCATTAGTCTC\n",
      "0 GTCCTCATTAGTCTC motifs detected in file\n",
      "\n",
      "Seq is: CACAAGAACACAACA\n",
      "0 CACAAGAACACAACA motifs detected in file\n",
      "\n",
      "Seq is: CTATTCCCACCTCTC\n",
      "0 CTATTCCCACCTCTC motifs detected in file\n",
      "\n",
      "Seq is: CAATTTCCTCGGTGA\n",
      "0 CAATTTCCTCGGTGA motifs detected in file\n",
      "\n",
      "Seq is: ATAATCATTACAACT\n",
      "0 ATAATCATTACAACT motifs detected in file\n",
      "\n",
      "Seq is: ATCATCTTACTACCG\n",
      "0 ATCATCTTACTACCG motifs detected in file\n",
      "\n",
      "Seq is: ATCCCACTGAAGTAT\n",
      "0 ATCCCACTGAAGTAT motifs detected in file\n",
      "\n",
      "Seq is: ACTCTAACAAACCCG\n",
      "0 ACTCTAACAAACCCG motifs detected in file\n",
      "\n",
      "Seq is: ATGACGAGCGTCACG\n",
      "0 ATGACGAGCGTCACG motifs detected in file\n",
      "\n",
      "Seq is: CGGCTAGTCCCGCAT\n",
      "0 CGGCTAGTCCCGCAT motifs detected in file\n",
      "\n",
      "Complement found from 3673 to 3688\n",
      "Complement found from 11747 to 11762\n",
      "Seq is: CAAACTCAAACTACG\n",
      "2 CAAACTCAAACTACG motifs detected in file\n",
      "\n",
      "Seq is: AGTGGGACTAGTCTC\n",
      "0 AGTGGGACTAGTCTC motifs detected in file\n",
      "\n",
      "Seq is: CTAACTCATTTGCCG\n",
      "0 CTAACTCATTTGCCG motifs detected in file\n",
      "\n",
      "Seq is: ATCCGATCTCCACCG\n",
      "0 ATCCGATCTCCACCG motifs detected in file\n",
      "\n",
      "Seq is: ATCTTATTTATCCTC\n",
      "0 ATCTTATTTATCCTC motifs detected in file\n",
      "\n",
      "Seq is: CGGATCCAACTCCAA\n",
      "0 CGGATCCAACTCCAA motifs detected in file\n",
      "\n",
      "Seq is: CTGGTCCCCCAACCC\n",
      "0 CTGGTCCCCCAACCC motifs detected in file\n",
      "\n",
      "Seq is: ATACCCCTCCCCCCA\n",
      "0 ATACCCCTCCCCCCA motifs detected in file\n",
      "\n",
      "Seq is: AGTATCATCTTCTCG\n",
      "0 AGTATCATCTTCTCG motifs detected in file\n",
      "\n",
      "Seq is: CTACCACTCTCGATT\n",
      "0 CTACCACTCTCGATT motifs detected in file\n",
      "\n",
      "Seq is: CCAGCCCCGCCACTA\n",
      "0 CCAGCCCCGCCACTA motifs detected in file\n",
      "\n",
      "Seq is: CATCTCCCACTACCA\n",
      "0 CATCTCCCACTACCA motifs detected in file\n",
      "\n",
      "Seq is: TCTACACCGCCCAAA\n",
      "0 TCTACACCGCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: ATCCCCGAGAAACCA\n",
      "0 ATCCCCGAGAAACCA motifs detected in file\n",
      "\n",
      "Seq is: CTTCTCAAAATACCG\n",
      "0 CTTCTCAAAATACCG motifs detected in file\n",
      "\n",
      "Seq is: CAGTCGCTTCCCAAC\n",
      "0 CAGTCGCTTCCCAAC motifs detected in file\n",
      "\n",
      "Seq is: ATCATCGGGCATCCC\n",
      "0 ATCATCGGGCATCCC motifs detected in file\n",
      "\n",
      "Seq is: CGGATGTTGCAACCC\n",
      "0 CGGATGTTGCAACCC motifs detected in file\n",
      "\n",
      "Seq is: CGGAAACGCATCAAC\n",
      "0 CGGAAACGCATCAAC motifs detected in file\n",
      "\n",
      "Seq is: ATATATCGGATCTTA\n",
      "0 ATATATCGGATCTTA motifs detected in file\n",
      "\n",
      "Seq is: AAAAGCAAGCCATTC\n",
      "0 AAAAGCAAGCCATTC motifs detected in file\n",
      "\n",
      "Seq is: GTAATCCTTACGGTA\n",
      "0 GTAATCCTTACGGTA motifs detected in file\n",
      "\n",
      "Seq is: ACGCTAATCTTACCC\n",
      "0 ACGCTAATCTTACCC motifs detected in file\n",
      "\n",
      "Seq is: ATGTTACTCCTCATC\n",
      "0 ATGTTACTCCTCATC motifs detected in file\n",
      "\n",
      "Seq is: CTCCAACCGGTACCC\n",
      "0 CTCCAACCGGTACCC motifs detected in file\n",
      "\n",
      "Seq is: ATACAACAATTCTTC\n",
      "0 ATACAACAATTCTTC motifs detected in file\n",
      "\n",
      "Seq is: TCCTTAACTTGGAGA\n",
      "0 TCCTTAACTTGGAGA motifs detected in file\n",
      "\n",
      "Seq is: CTGACATTTCAAAAT\n",
      "0 CTGACATTTCAAAAT motifs detected in file\n",
      "\n",
      "Seq is: TCAAAATACGCTAAT\n",
      "0 TCAAAATACGCTAAT motifs detected in file\n",
      "\n",
      "Seq is: GGCCCGAGACGGTAG\n",
      "0 GGCCCGAGACGGTAG motifs detected in file\n",
      "\n",
      "Seq is: AATTGTTTGGGACAA\n",
      "0 AATTGTTTGGGACAA motifs detected in file\n",
      "\n",
      "Seq is: GAACCCACCCACACC\n",
      "0 GAACCCACCCACACC motifs detected in file\n",
      "\n",
      "Seq is: CATATTATGATTCAA\n",
      "0 CATATTATGATTCAA motifs detected in file\n",
      "\n",
      "Seq is: CTCTACTATAGTAAA\n",
      "0 CTCTACTATAGTAAA motifs detected in file\n",
      "\n",
      "Seq is: TGCCCCCTTCCGCGA\n",
      "0 TGCCCCCTTCCGCGA motifs detected in file\n",
      "\n",
      "Seq is: AACACTTCATCCGGA\n",
      "0 AACACTTCATCCGGA motifs detected in file\n",
      "\n",
      "Seq is: ATAAAGAGAACAGGA\n",
      "0 ATAAAGAGAACAGGA motifs detected in file\n",
      "\n",
      "Seq is: AAGCATGTCCCTCCT\n",
      "0 AAGCATGTCCCTCCT motifs detected in file\n",
      "\n",
      "Seq is: TAAACTTNCATCTAT\n",
      "0 TAAACTTNCATCTAT motifs detected in file\n",
      "\n",
      "Seq is: CTTTGGCTGGACCTA\n",
      "0 CTTTGGCTGGACCTA motifs detected in file\n",
      "\n",
      "Seq is: ATGAGGCCAGACTTG\n",
      "0 ATGAGGCCAGACTTG motifs detected in file\n",
      "\n",
      "Seq is: AGTCTAGTGCATCCT\n",
      "0 AGTCTAGTGCATCCT motifs detected in file\n",
      "\n",
      "Seq is: GAAATTAGCAACTTG\n",
      "0 GAAATTAGCAACTTG motifs detected in file\n",
      "\n",
      "Seq is: TTTGCTTGGAAATTA\n",
      "0 TTTGCTTGGAAATTA motifs detected in file\n",
      "\n",
      "Seq is: TCGCCGACGTGGTAG\n",
      "0 TCGCCGACGTGGTAG motifs detected in file\n",
      "\n",
      "Seq is: CCCTACAGGACTAGG\n",
      "0 CCCTACAGGACTAGG motifs detected in file\n",
      "\n",
      "Seq is: TTGTAGCTCCAGCAT\n",
      "0 TTGTAGCTCCAGCAT motifs detected in file\n",
      "\n",
      "Seq is: TTGGGATAACAACTA\n",
      "0 TTGGGATAACAACTA motifs detected in file\n",
      "\n",
      "Seq is: TACCTGAGATCTTAT\n",
      "0 TACCTGAGATCTTAT motifs detected in file\n",
      "\n",
      "Seq is: CCTAACGCGACAATA\n",
      "0 CCTAACGCGACAATA motifs detected in file\n",
      "\n",
      "Seq is: GGGATCCCATTGAAC\n",
      "0 GGGATCCCATTGAAC motifs detected in file\n",
      "\n",
      "Seq is: AAGGCAACCAGTTCA\n",
      "0 AAGGCAACCAGTTCA motifs detected in file\n",
      "\n",
      "Seq is: ATAACCTAGTTAACT\n",
      "0 ATAACCTAGTTAACT motifs detected in file\n",
      "\n",
      "Seq is: CATATCATCAAGCGA\n",
      "0 CATATCATCAAGCGA motifs detected in file\n",
      "\n",
      "Seq is: AACTGACCACTTCAG\n",
      "0 AACTGACCACTTCAG motifs detected in file\n",
      "\n",
      "Seq is: AATCGTACATGACGA\n",
      "0 AATCGTACATGACGA motifs detected in file\n",
      "\n",
      "Seq is: GCCTCCAACCCAAGA\n",
      "0 GCCTCCAACCCAAGA motifs detected in file\n",
      "\n",
      "Seq is: CGAGGCTCCAGCGGG\n",
      "0 CGAGGCTCCAGCGGG motifs detected in file\n",
      "\n",
      "Seq is: GTTGGCTTTAAAAAT\n",
      "0 GTTGGCTTTAAAAAT motifs detected in file\n",
      "\n",
      "Seq is: TACGTCCAAACCATC\n",
      "0 TACGTCCAAACCATC motifs detected in file\n",
      "\n",
      "Seq is: AAATCCTGGACACCC\n",
      "0 AAATCCTGGACACCC motifs detected in file\n",
      "\n",
      "Seq is: AAACAATCCATGACA\n",
      "0 AAACAATCCATGACA motifs detected in file\n",
      "\n",
      "Seq is: AACGTAATTATTTAA\n",
      "0 AACGTAATTATTTAA motifs detected in file\n",
      "\n",
      "Seq is: TTTCGAGGTATCCCA\n",
      "0 TTTCGAGGTATCCCA motifs detected in file\n",
      "\n",
      "Seq is: GAAGAGCAGAACGAC\n",
      "0 GAAGAGCAGAACGAC motifs detected in file\n",
      "\n",
      "Seq is: ACAATACGGGCGGAG\n",
      "0 ACAATACGGGCGGAG motifs detected in file\n",
      "\n",
      "Seq is: AAGTGCCCGTCCAGT\n",
      "0 AAGTGCCCGTCCAGT motifs detected in file\n",
      "\n",
      "Seq is: TAAAGTGACCAATTT\n",
      "0 TAAAGTGACCAATTT motifs detected in file\n",
      "\n",
      "Seq is: TCATTCTCTGTCGAC\n",
      "0 TCATTCTCTGTCGAC motifs detected in file\n",
      "\n",
      "Seq is: TTGGGAGCACCTCGG\n",
      "0 TTGGGAGCACCTCGG motifs detected in file\n",
      "\n",
      "Seq is: TAAGTATGTCCAGGG\n",
      "0 TAAGTATGTCCAGGG motifs detected in file\n",
      "\n",
      "Seq is: ATAAATTCCTTGTTC\n",
      "0 ATAAATTCCTTGTTC motifs detected in file\n",
      "\n",
      "Seq is: ACTAATACGATGGAA\n",
      "0 ACTAATACGATGGAA motifs detected in file\n",
      "\n",
      "Seq is: ACGTGCCAATCCCAT\n",
      "0 ACGTGCCAATCCCAT motifs detected in file\n",
      "\n",
      "Seq is: GGCGCCGGCAATTTG\n",
      "0 GGCGCCGGCAATTTG motifs detected in file\n",
      "\n",
      "Seq is: TACACAGTGACCCGT\n",
      "0 TACACAGTGACCCGT motifs detected in file\n",
      "\n",
      "Seq is: CCGCCACGGAGATTA\n",
      "0 CCGCCACGGAGATTA motifs detected in file\n",
      "\n",
      "Seq is: TGACCACTACGATCT\n",
      "0 TGACCACTACGATCT motifs detected in file\n",
      "\n",
      "Seq is: CCACTACAAAAACCA\n",
      "0 CCACTACAAAAACCA motifs detected in file\n",
      "\n",
      "Seq is: TTTGTCCGCCCCATT\n",
      "0 TTTGTCCGCCCCATT motifs detected in file\n",
      "\n",
      "Seq is: CTAAACGGCTCAAGG\n",
      "0 CTAAACGGCTCAAGG motifs detected in file\n",
      "\n",
      "Seq is: AAAATGAAAAAAATT\n",
      "0 AAAATGAAAAAAATT motifs detected in file\n",
      "\n",
      "Seq is: GGAAAGGAATACTCG\n",
      "0 GGAAAGGAATACTCG motifs detected in file\n",
      "\n",
      "Seq is: TACGGACACAACCCA\n",
      "0 TACGGACACAACCCA motifs detected in file\n",
      "\n",
      "Seq is: ACTGTCACTCCCATT\n",
      "0 ACTGTCACTCCCATT motifs detected in file\n",
      "\n",
      "Seq is: ATTACTGAACAACCA\n",
      "0 ATTACTGAACAACCA motifs detected in file\n",
      "\n",
      "Seq is: ACTAACATCTATAAC\n",
      "0 ACTAACATCTATAAC motifs detected in file\n",
      "\n",
      "Seq is: CCGACAATTAACAGT\n",
      "0 CCGACAATTAACAGT motifs detected in file\n",
      "\n",
      "Seq is: CAAGTCACAAAATTA\n",
      "0 CAAGTCACAAAATTA motifs detected in file\n",
      "\n",
      "Seq is: GACTGCGTCCGAATA\n",
      "0 GACTGCGTCCGAATA motifs detected in file\n",
      "\n",
      "Seq is: CGCCTCCTCTTACAA\n",
      "0 CGCCTCCTCTTACAA motifs detected in file\n",
      "\n",
      "Seq is: AAGTACAATGAATAT\n",
      "0 AAGTACAATGAATAT motifs detected in file\n",
      "\n",
      "Seq is: GATTGTAATCAAGAA\n",
      "0 GATTGTAATCAAGAA motifs detected in file\n",
      "\n",
      "Seq is: GATATCCCACTATCT\n",
      "0 GATATCCCACTATCT motifs detected in file\n",
      "\n",
      "Seq is: AACCAGGTTAACCCA\n",
      "0 AACCAGGTTAACCCA motifs detected in file\n",
      "\n",
      "Seq is: CACTCCTCAAGTCAA\n",
      "0 CACTCCTCAAGTCAA motifs detected in file\n",
      "\n",
      "Seq is: TATACAAACCCTAAA\n",
      "0 TATACAAACCCTAAA motifs detected in file\n",
      "\n",
      "Seq is: AAATCCATCACCCAC\n",
      "0 AAATCCATCACCCAC motifs detected in file\n",
      "\n",
      "Seq is: AACTCGAACTTGCGA\n",
      "0 AACTCGAACTTGCGA motifs detected in file\n",
      "\n",
      "Seq is: AAGAATTAACCACCG\n",
      "0 AAGAATTAACCACCG motifs detected in file\n",
      "\n",
      "Seq is: ACGAAAATCCGGATG\n",
      "0 ACGAAAATCCGGATG motifs detected in file\n",
      "\n",
      "Seq is: ATACCCACAATTTAA\n",
      "0 ATACCCACAATTTAA motifs detected in file\n",
      "\n",
      "Seq is: AAAATGAGAGAGATG\n",
      "0 AAAATGAGAGAGATG motifs detected in file\n",
      "\n",
      "Seq is: TTCCAAAAAAGGATC\n",
      "0 TTCCAAAAAAGGATC motifs detected in file\n",
      "\n",
      "Seq is: ACAGGTTTCTCGACA\n",
      "0 ACAGGTTTCTCGACA motifs detected in file\n",
      "\n",
      "Seq is: AGGAGAAACCTGATT\n",
      "0 AGGAGAAACCTGATT motifs detected in file\n",
      "\n",
      "Seq is: GTCAATTTAAATGTT\n",
      "0 GTCAATTTAAATGTT motifs detected in file\n",
      "\n",
      "Seq is: CCCCTAAATCTCCCA\n",
      "0 CCCCTAAATCTCCCA motifs detected in file\n",
      "\n",
      "Seq is: AGACACCCGTTTAAA\n",
      "0 AGACACCCGTTTAAA motifs detected in file\n",
      "\n",
      "Seq is: TTTCAACTTGATTCT\n",
      "0 TTTCAACTTGATTCT motifs detected in file\n",
      "\n",
      "Seq is: AAGATAGAACCTGTT\n",
      "0 AAGATAGAACCTGTT motifs detected in file\n",
      "\n",
      "Seq is: GGTCGATAGTGGTCC\n",
      "0 GGTCGATAGTGGTCC motifs detected in file\n",
      "\n",
      "Seq is: GAGCCATCCAAACAG\n",
      "0 GAGCCATCCAAACAG motifs detected in file\n",
      "\n",
      "Seq is: CGGAGATGGATATTT\n",
      "0 CGGAGATGGATATTT motifs detected in file\n",
      "\n",
      "Seq is: AGAAGGGTGATAAAA\n",
      "0 AGAAGGGTGATAAAA motifs detected in file\n",
      "\n",
      "Seq is: CGATGTATCTGCCCA\n",
      "0 CGATGTATCTGCCCA motifs detected in file\n",
      "\n",
      "Seq is: CACGAGAAAATCGAC\n",
      "0 CACGAGAAAATCGAC motifs detected in file\n",
      "\n",
      "Seq is: AAGAATCCATCGAGC\n",
      "0 AAGAATCCATCGAGC motifs detected in file\n",
      "\n",
      "Seq is: AGACCAAAGCCCCCA\n",
      "0 AGACCAAAGCCCCCA motifs detected in file\n",
      "\n",
      "Seq is: GAATCGAAACCGAGA\n",
      "0 GAATCGAAACCGAGA motifs detected in file\n",
      "\n",
      "Seq is: GGAACGTTTCAATAA\n",
      "0 GGAACGTTTCAATAA motifs detected in file\n",
      "\n",
      "Seq is: AGATCAATTAAGTAA\n",
      "0 AGATCAATTAAGTAA motifs detected in file\n",
      "\n",
      "Seq is: TACGTCTTCCATATC\n",
      "0 TACGTCTTCCATATC motifs detected in file\n",
      "\n",
      "Seq is: CCCAATCAGGAACGA\n",
      "0 CCCAATCAGGAACGA motifs detected in file\n",
      "\n",
      "Seq is: TATAATACGAACCAA\n",
      "0 TATAATACGAACCAA motifs detected in file\n",
      "\n",
      "Seq is: TATTAAAAAGTAGAA\n",
      "0 TATTAAAAAGTAGAA motifs detected in file\n",
      "\n",
      "Seq is: AGGGAACGCCATGAT\n",
      "0 AGGGAACGCCATGAT motifs detected in file\n",
      "\n",
      "Seq is: ATAGATAACGCGGTC\n",
      "0 ATAGATAACGCGGTC motifs detected in file\n",
      "\n",
      "Seq is: CAAAGTTAAAGATAG\n",
      "0 CAAAGTTAAAGATAG motifs detected in file\n",
      "\n",
      "Seq is: CGGATATGAAATAAA\n",
      "0 CGGATATGAAATAAA motifs detected in file\n",
      "\n",
      "Seq is: CCCATTTACCAAACC\n",
      "0 CCCATTTACCAAACC motifs detected in file\n",
      "\n",
      "Seq is: GATTCCAACAGACCA\n",
      "0 GATTCCAACAGACCA motifs detected in file\n",
      "\n",
      "Seq is: TCATTCCACCTCACC\n",
      "0 TCATTCCACCTCACC motifs detected in file\n",
      "\n",
      "Seq is: CAAACCCCGATCCAA\n",
      "0 CAAACCCCGATCCAA motifs detected in file\n",
      "\n",
      "Seq is: ATCGAGTCTCGCCAG\n",
      "0 ATCGAGTCTCGCCAG motifs detected in file\n",
      "\n",
      "Seq is: TTCAATTCAACTTTA\n",
      "0 TTCAATTCAACTTTA motifs detected in file\n",
      "\n",
      "Seq is: GAGGATTCACATTCA\n",
      "0 GAGGATTCACATTCA motifs detected in file\n",
      "\n",
      "Seq is: ACCCACGAAACACAA\n",
      "0 ACCCACGAAACACAA motifs detected in file\n",
      "\n",
      "Seq is: TTCGATGTGAGACCA\n",
      "0 TTCGATGTGAGACCA motifs detected in file\n",
      "\n",
      "Seq is: AGCAGGTTCACGTGA\n",
      "0 AGCAGGTTCACGTGA motifs detected in file\n",
      "\n",
      "Seq is: AAGGTCATGTGAATG\n",
      "0 AAGGTCATGTGAATG motifs detected in file\n",
      "\n",
      "Seq is: GTACAATGCTGAACA\n",
      "0 GTACAATGCTGAACA motifs detected in file\n",
      "\n",
      "Seq is: GAGGAGATATATTTA\n",
      "0 GAGGAGATATATTTA motifs detected in file\n",
      "\n",
      "Seq is: CGCATCCCCAAAATC\n",
      "0 CGCATCCCCAAAATC motifs detected in file\n",
      "\n",
      "Seq is: AATTTACAGGAAACT\n",
      "0 AATTTACAGGAAACT motifs detected in file\n",
      "\n",
      "Seq is: TCATATGAACTCCTC\n",
      "0 TCATATGAACTCCTC motifs detected in file\n",
      "\n",
      "Seq is: CCACTGCCCGCCACA\n",
      "0 CCACTGCCCGCCACA motifs detected in file\n",
      "\n",
      "Seq is: CATGCGCGAAGTCCC\n",
      "0 CATGCGCGAAGTCCC motifs detected in file\n",
      "\n",
      "Seq is: GGGACAAGTTGATTC\n",
      "0 GGGACAAGTTGATTC motifs detected in file\n",
      "\n",
      "Seq is: GTGAGATGAGAATCA\n",
      "0 GTGAGATGAGAATCA motifs detected in file\n",
      "\n",
      "Seq is: AATGACGATTTAGGT\n",
      "0 AATGACGATTTAGGT motifs detected in file\n",
      "\n",
      "Seq is: GGAAGCTGGGAATTC\n",
      "0 GGAAGCTGGGAATTC motifs detected in file\n",
      "\n",
      "Seq is: AAAGTATTCCCGATA\n",
      "0 AAAGTATTCCCGATA motifs detected in file\n",
      "\n",
      "Seq is: GCATCAAAAGACCCC\n",
      "0 GCATCAAAAGACCCC motifs detected in file\n",
      "\n",
      "Seq is: ATCTTTTACATCGGG\n",
      "0 ATCTTTTACATCGGG motifs detected in file\n",
      "\n",
      "Seq is: TAAAGAACGGTGGAG\n",
      "0 TAAAGAACGGTGGAG motifs detected in file\n",
      "\n",
      "Seq is: TACCCGATGTGGAAC\n",
      "0 TACCCGATGTGGAAC motifs detected in file\n",
      "\n",
      "Seq is: TGGATTGCAGAAATG\n",
      "0 TGGATTGCAGAAATG motifs detected in file\n",
      "\n",
      "Seq is: CACCCATGAACGCGA\n",
      "0 CACCCATGAACGCGA motifs detected in file\n",
      "\n",
      "Seq is: ATGAAACATCGGAAG\n",
      "0 ATGAAACATCGGAAG motifs detected in file\n",
      "\n",
      "Seq is: TAGTCCCAAACGACT\n",
      "0 TAGTCCCAAACGACT motifs detected in file\n",
      "\n",
      "Seq is: TCTACCGCCATATAT\n",
      "0 TCTACCGCCATATAT motifs detected in file\n",
      "\n",
      "Seq is: CCGACTCGTTCTCCA\n",
      "0 CCGACTCGTTCTCCA motifs detected in file\n",
      "\n",
      "Seq is: CCACTCCAACTAGCC\n",
      "0 CCACTCCAACTAGCC motifs detected in file\n",
      "\n",
      "Seq is: CCAAATAGCTAATGT\n",
      "0 CCAAATAGCTAATGT motifs detected in file\n",
      "\n",
      "Seq is: CTTGTCCGAGGAGAT\n",
      "0 CTTGTCCGAGGAGAT motifs detected in file\n",
      "\n",
      "Seq is: CTCCCTATACTTCGT\n",
      "0 CTCCCTATACTTCGT motifs detected in file\n",
      "\n",
      "Seq is: GGCGGTCCAGGAAAC\n",
      "0 GGCGGTCCAGGAAAC motifs detected in file\n",
      "\n",
      "Seq is: TCAAAATTCGACACC\n",
      "0 TCAAAATTCGACACC motifs detected in file\n",
      "\n",
      "Seq is: GAGCATCACAAGACC\n",
      "0 GAGCATCACAAGACC motifs detected in file\n",
      "\n",
      "Seq is: GCTCGTCAAAACAAC\n",
      "0 GCTCGTCAAAACAAC motifs detected in file\n",
      "\n",
      "Seq is: TAAATTGACAACTCC\n",
      "0 TAAATTGACAACTCC motifs detected in file\n",
      "\n",
      "Seq is: AAATCCCGATTCGTA\n",
      "0 AAATCCCGATTCGTA motifs detected in file\n",
      "\n",
      "Seq is: TCACCCCATAGATTA\n",
      "0 TCACCCCATAGATTA motifs detected in file\n",
      "\n",
      "Seq is: GGGTCAAACCCAGAA\n",
      "0 GGGTCAAACCCAGAA motifs detected in file\n",
      "\n",
      "Seq is: TCGATAACACACAAG\n",
      "0 TCGATAACACACAAG motifs detected in file\n",
      "\n",
      "Seq is: TCTATACAATTTCGG\n",
      "0 TCTATACAATTTCGG motifs detected in file\n",
      "\n",
      "Seq is: TGAAAGCATCAGATA\n",
      "0 TGAAAGCATCAGATA motifs detected in file\n",
      "\n",
      "Seq is: AAACACAGTTGACCT\n",
      "0 AAACACAGTTGACCT motifs detected in file\n",
      "\n",
      "Seq is: CAAAAAATGTTGAGT\n",
      "0 CAAAAAATGTTGAGT motifs detected in file\n",
      "\n",
      "Seq is: CCACTCAAAATCGAA\n",
      "0 CCACTCAAAATCGAA motifs detected in file\n",
      "\n",
      "Seq is: ATAACCCCTCCCCCA\n",
      "0 ATAACCCCTCCCCCA motifs detected in file\n",
      "\n",
      "Seq is: CTAGATTTTGTGAGA\n",
      "0 CTAGATTTTGTGAGA motifs detected in file\n",
      "\n",
      "Seq is: AATGCGGCCGAAGAT\n",
      "0 AATGCGGCCGAAGAT motifs detected in file\n",
      "\n",
      "Seq is: AACTGAACCCAATTA\n",
      "0 AACTGAACCCAATTA motifs detected in file\n",
      "\n",
      "Seq is: GCACACTGGCGCCAC\n",
      "0 GCACACTGGCGCCAC motifs detected in file\n",
      "\n",
      "Seq is: CGACCGTGCTTTAAC\n",
      "0 CGACCGTGCTTTAAC motifs detected in file\n",
      "\n",
      "Seq is: TGGTTGGGACCCCAA\n",
      "0 TGGTTGGGACCCCAA motifs detected in file\n",
      "\n",
      "Seq is: TCATATCGAATCAAT\n",
      "0 TCATATCGAATCAAT motifs detected in file\n",
      "\n",
      "Seq is: TTGAAAGCAAATAAC\n",
      "0 TTGAAAGCAAATAAC motifs detected in file\n",
      "\n",
      "Seq is: GATTTCCAATTAGTG\n",
      "0 GATTTCCAATTAGTG motifs detected in file\n",
      "\n",
      "Seq is: ACGACAAAGGGCACC\n",
      "0 ACGACAAAGGGCACC motifs detected in file\n",
      "\n",
      "Seq is: CCCACACCGATCCGA\n",
      "0 CCCACACCGATCCGA motifs detected in file\n",
      "\n",
      "Seq is: TTCGCAAAACTCGAC\n",
      "0 TTCGCAAAACTCGAC motifs detected in file\n",
      "\n",
      "Seq is: GTAACGACGCACGAA\n",
      "0 GTAACGACGCACGAA motifs detected in file\n",
      "\n",
      "Seq is: CTACGAACAAGGAAA\n",
      "0 CTACGAACAAGGAAA motifs detected in file\n",
      "\n",
      "Seq is: ACTAGCACCACTAAA\n",
      "0 ACTAGCACCACTAAA motifs detected in file\n",
      "\n",
      "Seq is: TCTCCCACTTGAGTG\n",
      "0 TCTCCCACTTGAGTG motifs detected in file\n",
      "\n",
      "Seq is: ACCTTGCCCCTACGA\n",
      "0 ACCTTGCCCCTACGA motifs detected in file\n",
      "\n",
      "Seq is: ACGTACACATTAGAA\n",
      "0 ACGTACACATTAGAA motifs detected in file\n",
      "\n",
      "Seq is: TGATTCTCGATTATC\n",
      "0 TGATTCTCGATTATC motifs detected in file\n",
      "\n",
      "Seq is: TTTCCGATCCTGGTT\n",
      "0 TTTCCGATCCTGGTT motifs detected in file\n",
      "\n",
      "Seq is: TGGATAAACAAATAC\n",
      "0 TGGATAAACAAATAC motifs detected in file\n",
      "\n",
      "Seq is: CCCACTACACTCGGG\n",
      "0 CCCACTACACTCGGG motifs detected in file\n",
      "\n",
      "Seq is: CAGATTTGTAAAAGT\n",
      "0 CAGATTTGTAAAAGT motifs detected in file\n",
      "\n",
      "Seq is: CACATAACGAAACTC\n",
      "0 CACATAACGAAACTC motifs detected in file\n",
      "\n",
      "Seq is: CTCCATTCGATGTAT\n",
      "0 CTCCATTCGATGTAT motifs detected in file\n",
      "\n",
      "Seq is: TTGACACCCCCCACA\n",
      "0 TTGACACCCCCCACA motifs detected in file\n",
      "\n",
      "Seq is: GAAACCCCAAACCAA\n",
      "0 GAAACCCCAAACCAA motifs detected in file\n",
      "\n",
      "Seq is: CCAAGCCCCATACCC\n",
      "0 CCAAGCCCCATACCC motifs detected in file\n",
      "\n",
      "Seq is: CAATCGTCGCCACAC\n",
      "0 CAATCGTCGCCACAC motifs detected in file\n",
      "\n",
      "Seq is: ACACACGACCCATCC\n",
      "0 ACACACGACCCATCC motifs detected in file\n",
      "\n",
      "Seq is: TACCCGCCCCCAACA\n",
      "0 TACCCGCCCCCAACA motifs detected in file\n",
      "\n",
      "Seq is: TAACTACTCTAATCA\n",
      "0 TAACTACTCTAATCA motifs detected in file\n",
      "\n",
      "Seq is: TCATACCCTCACCCT\n",
      "0 TCATACCCTCACCCT motifs detected in file\n",
      "\n",
      "Seq is: CCCCTTTTATTACAC\n",
      "0 CCCCTTTTATTACAC motifs detected in file\n",
      "\n",
      "Seq is: AATCAACCCCCCACT\n",
      "0 AATCAACCCCCCACT motifs detected in file\n",
      "\n",
      "Seq is: GACAATTTTCACGTA\n",
      "0 GACAATTTTCACGTA motifs detected in file\n",
      "\n",
      "Seq is: TGGCGGTTTTCTATT\n",
      "0 TGGCGGTTTTCTATT motifs detected in file\n",
      "\n",
      "Seq is: TTAAACTTTAGACCA\n",
      "0 TTAAACTTTAGACCA motifs detected in file\n",
      "\n",
      "Seq is: ATCCGACCACAATCC\n",
      "0 ATCCGACCACAATCC motifs detected in file\n",
      "\n",
      "Seq is: CAAGAAACAAAAACC\n",
      "0 CAAGAAACAAAAACC motifs detected in file\n",
      "\n",
      "Seq is: CCAAACCGTCTCTAC\n",
      "0 CCAAACCGTCTCTAC motifs detected in file\n",
      "\n",
      "Seq is: ACAAATTCACGACAC\n",
      "0 ACAAATTCACGACAC motifs detected in file\n",
      "\n",
      "Seq is: CGGTCTTCGCCCCCT\n",
      "0 CGGTCTTCGCCCCCT motifs detected in file\n",
      "\n",
      "Seq is: CCCCCCCAAACCACC\n",
      "0 CCCCCCCAAACCACC motifs detected in file\n",
      "\n",
      "Seq is: TTTAAAAAACAATAC\n",
      "0 TTTAAAAAACAATAC motifs detected in file\n",
      "\n",
      "Seq is: TACAGACACACCTTT\n",
      "0 TACAGACACACCTTT motifs detected in file\n",
      "\n",
      "Seq is: CACCGACACGTCTGT\n",
      "0 CACCGACACGTCTGT motifs detected in file\n",
      "\n",
      "Seq is: AAGTTAACAATAATA\n",
      "0 AAGTTAACAATAATA motifs detected in file\n",
      "\n",
      "Seq is: ATACAGGATGTTCGT\n",
      "0 ATACAGGATGTTCGT motifs detected in file\n",
      "\n",
      "Seq is: AATTAATTAATTGTG\n",
      "0 AATTAATTAATTGTG motifs detected in file\n",
      "\n",
      "Seq is: TGAAATCATTCATAC\n",
      "0 TGAAATCATTCATAC motifs detected in file\n",
      "\n",
      "Seq is: AAGCGGACATTATAA\n",
      "0 AAGCGGACATTATAA motifs detected in file\n",
      "\n",
      "Seq is: CTTGCATCCACGCTA\n",
      "0 CTTGCATCCACGCTA motifs detected in file\n",
      "\n",
      "Seq is: TTTATTATCCTACTC\n",
      "0 TTTATTATCCTACTC motifs detected in file\n",
      "\n",
      "Seq is: CGTCCTTAGTTTCTG\n",
      "0 CGTCCTTAGTTTCTG motifs detected in file\n",
      "\n",
      "Seq is: TCTATGACGCTGTAT\n",
      "0 TCTATGACGCTGTAT motifs detected in file\n",
      "\n",
      "Seq is: CCCACGAGGCCGAGG\n",
      "0 CCCACGAGGCCGAGG motifs detected in file\n",
      "\n",
      "Seq is: TCGCAGAGCGTTACG\n",
      "0 TCGCAGAGCGTTACG motifs detected in file\n",
      "\n",
      "Seq is: ATAGCGCACGTATGG\n",
      "0 ATAGCGCACGTATGG motifs detected in file\n",
      "\n",
      "Seq is: GGGGTCTGCTTTTAT\n",
      "0 GGGGTCTGCTTTTAT motifs detected in file\n",
      "\n",
      "Seq is: GGTTTACGTACCTCT\n",
      "0 GGTTTACGTACCTCT motifs detected in file\n",
      "\n",
      "Seq is: CGAGGGCACTCACCA\n",
      "0 CGAGGGCACTCACCA motifs detected in file\n",
      "\n",
      "Seq is: ATTATCCCACTATCT\n",
      "0 ATTATCCCACTATCT motifs detected in file\n",
      "\n",
      "Seq is: GTAGCACTACAGAATA\n",
      "0 GTAGCACTACAGAATA motifs detected in file\n",
      "\n",
      "Seq is: AATTCCCCTTGCACAC\n",
      "0 AATTCCCCTTGCACAC motifs detected in file\n",
      "\n",
      "Seq is: CCGATAAATCCGAAAT\n",
      "0 CCGATAAATCCGAAAT motifs detected in file\n",
      "\n",
      "Seq is: ACTGGGACTTCATCCT\n",
      "0 ACTGGGACTTCATCCT motifs detected in file\n",
      "\n",
      "Seq is: TGGTCTACAGCCTATG\n",
      "0 TGGTCTACAGCCTATG motifs detected in file\n",
      "\n",
      "Seq is: TCAAGTGAAATCGATG\n",
      "0 TCAAGTGAAATCGATG motifs detected in file\n",
      "\n",
      "Seq is: GGGGTTCACAATACCC\n",
      "0 GGGGTTCACAATACCC motifs detected in file\n",
      "\n",
      "Seq is: GGGCCTCGCTCCTCTC\n",
      "0 GGGCCTCGCTCCTCTC motifs detected in file\n",
      "\n",
      "Seq is: ATCGTGAGAACACGCC\n",
      "0 ATCGTGAGAACACGCC motifs detected in file\n",
      "\n",
      "Seq is: CTATAACTAAAGTGCC\n",
      "0 CTATAACTAAAGTGCC motifs detected in file\n",
      "\n",
      "Seq is: TCCTACCACCAGTTCC\n",
      "0 TCCTACCACCAGTTCC motifs detected in file\n",
      "\n",
      "Seq is: CTGGGGATAGACTCCC\n",
      "0 CTGGGGATAGACTCCC motifs detected in file\n",
      "\n",
      "Seq is: CCCAGTAGGTACCCCT\n",
      "0 CCCAGTAGGTACCCCT motifs detected in file\n",
      "\n",
      "Seq is: GCTCTTCCCTAAACTG\n",
      "0 GCTCTTCCCTAAACTG motifs detected in file\n",
      "\n",
      "Seq is: ACATTACACGATACAT\n",
      "0 ACATTACACGATACAT motifs detected in file\n",
      "\n",
      "Seq is: GCCATTTACCGAAATA\n",
      "0 GCCATTTACCGAAATA motifs detected in file\n",
      "\n",
      "Seq is: CATGATACATGACAAT\n",
      "0 CATGATACATGACAAT motifs detected in file\n",
      "\n",
      "Seq is: TCCCACCCATCCAAAC\n",
      "0 TCCCACCCATCCAAAC motifs detected in file\n",
      "\n",
      "Seq is: AACCATAGGATCACCC\n",
      "0 AACCATAGGATCACCC motifs detected in file\n",
      "\n",
      "Seq is: ACTCCCCACCGAAACC\n",
      "0 ACTCCCCACCGAAACC motifs detected in file\n",
      "\n",
      "Seq is: TCAACGTCAACTACAC\n",
      "0 TCAACGTCAACTACAC motifs detected in file\n",
      "\n",
      "Seq is: ACTATCAACTCCCAAC\n",
      "0 ACTATCAACTCCCAAC motifs detected in file\n",
      "\n",
      "Seq is: TAACGACATGAACGAA\n",
      "0 TAACGACATGAACGAA motifs detected in file\n",
      "\n",
      "Seq is: CATTCGTACCCCTCCC\n",
      "0 CATTCGTACCCCTCCC motifs detected in file\n",
      "\n",
      "Seq is: CCAAAACTACACCTAA\n",
      "0 CCAAAACTACACCTAA motifs detected in file\n",
      "\n",
      "Seq is: CCCAAAAATACATGAT\n",
      "0 CCCAAAAATACATGAT motifs detected in file\n",
      "\n",
      "Seq is: GTCCACCAGTTCATAA\n",
      "0 GTCCACCAGTTCATAA motifs detected in file\n",
      "\n",
      "Seq is: ATACCATGGCATGTTA\n",
      "0 ATACCATGGCATGTTA motifs detected in file\n",
      "\n",
      "Seq is: TAAGTACCACCGACCG\n",
      "0 TAAGTACCACCGACCG motifs detected in file\n",
      "\n",
      "Seq is: TCATTACATGCTTTAT\n",
      "0 TCATTACATGCTTTAT motifs detected in file\n",
      "\n",
      "Seq is: GTATCGCCAACAACTA\n",
      "0 GTATCGCCAACAACTA motifs detected in file\n",
      "\n",
      "Seq is: CCCACTCAGTTATGAA\n",
      "0 CCCACTCAGTTATGAA motifs detected in file\n",
      "\n",
      "Seq is: CCCACCATGGGTTTAG\n",
      "0 CCCACCATGGGTTTAG motifs detected in file\n",
      "\n",
      "Seq is: ACGAAGGGGTACTTTC\n",
      "0 ACGAAGGGGTACTTTC motifs detected in file\n",
      "\n",
      "Seq is: TTGTCTCTTATCAAAT\n",
      "0 TTGTCTCTTATCAAAT motifs detected in file\n",
      "\n",
      "Seq is: TTAATCTTAGAATCGA\n",
      "0 TTAATCTTAGAATCGA motifs detected in file\n",
      "\n",
      "Seq is: AACCCACGATTACCAC\n",
      "0 AACCCACGATTACCAC motifs detected in file\n",
      "\n",
      "Seq is: CTCAATTTCTGAAAAA\n",
      "0 CTCAATTTCTGAAAAA motifs detected in file\n",
      "\n",
      "Seq is: GAGACTAAACAGGAAC\n",
      "0 GAGACTAAACAGGAAC motifs detected in file\n",
      "\n",
      "Seq is: CTTTTTCCAAAAGTAG\n",
      "0 CTTTTTCCAAAAGTAG motifs detected in file\n",
      "\n",
      "Seq is: AGGCCAAATGTTCTGA\n",
      "0 AGGCCAAATGTTCTGA motifs detected in file\n",
      "\n",
      "Seq is: CCACATAATCAAATAT\n",
      "0 CCACATAATCAAATAT motifs detected in file\n",
      "\n",
      "Seq is: GATGTTCCTGTCCGGG\n",
      "0 GATGTTCCTGTCCGGG motifs detected in file\n",
      "\n",
      "Seq is: TAAACTCATAAAACAA\n",
      "0 TAAACTCATAAAACAA motifs detected in file\n",
      "\n",
      "Seq is: AAGTTAATCCCTCTAT\n",
      "0 AAGTTAATCCCTCTAT motifs detected in file\n",
      "\n",
      "Seq is: CAACCATAATCCTAAT\n",
      "0 CAACCATAATCCTAAT motifs detected in file\n",
      "\n",
      "Seq is: CCTAACAACACTTCAT\n",
      "0 CCTAACAACACTTCAT motifs detected in file\n",
      "\n",
      "Seq is: ATCATGCCTACGATGA\n",
      "0 ATCATGCCTACGATGA motifs detected in file\n",
      "\n",
      "Seq is: ACAGGTTACTACCATT\n",
      "0 ACAGGTTACTACCATT motifs detected in file\n",
      "\n",
      "Seq is: TTCCCATCGAATGACC\n",
      "0 TTCCCATCGAATGACC motifs detected in file\n",
      "\n",
      "Seq is: AACAGGAGGCTAAGTC\n",
      "0 AACAGGAGGCTAAGTC motifs detected in file\n",
      "\n",
      "Seq is: CAATCTTACTCCTCCA\n",
      "0 CAATCTTACTCCTCCA motifs detected in file\n",
      "\n",
      "Seq is: GACGCCGATCCTCAGT\n",
      "0 GACGCCGATCCTCAGT motifs detected in file\n",
      "\n",
      "Seq is: TATTTCACTAACCGAA\n",
      "0 TATTTCACTAACCGAA motifs detected in file\n",
      "\n",
      "Seq is: TCACCCGCTTTATAAT\n",
      "0 TCACCCGCTTTATAAT motifs detected in file\n",
      "\n",
      "Seq is: ACGAAACAACAAACCT\n",
      "0 ACGAAACAACAAACCT motifs detected in file\n",
      "\n",
      "Seq is: ATATACCTCCTACCCC\n",
      "0 ATATACCTCCTACCCC motifs detected in file\n",
      "\n",
      "Seq is: TAATAACGATCCTACT\n",
      "0 TAATAACGATCCTACT motifs detected in file\n",
      "\n",
      "Seq is: CCTACCTATCATTATC\n",
      "0 CCTACCTATCATTATC motifs detected in file\n",
      "\n",
      "Seq is: CCGTTCCTGCGGAGGA\n",
      "0 CCGTTCCTGCGGAGGA motifs detected in file\n",
      "\n",
      "Seq is: TCAAACAATCCCTGCC\n",
      "0 TCAAACAATCCCTGCC motifs detected in file\n",
      "\n",
      "Seq is: TAGCCTCTTAACACAT\n",
      "0 TAGCCTCTTAACACAT motifs detected in file\n",
      "\n",
      "Seq is: CCGCTTATCCTTTATA\n",
      "0 CCGCTTATCCTTTATA motifs detected in file\n",
      "\n",
      "Seq is: GTAAGCCCGAACTACA\n",
      "0 GTAAGCCCGAACTACA motifs detected in file\n",
      "\n",
      "Seq is: CCCCTCCCCACAAATT\n",
      "0 CCCCTCCCCACAAATT motifs detected in file\n",
      "\n",
      "Seq is: CCCCAACCGATCCCAT\n",
      "0 CCCCAACCGATCCCAT motifs detected in file\n",
      "\n",
      "Seq is: ATTAACAGACCCAGCG\n",
      "0 ATTAACAGACCCAGCG motifs detected in file\n",
      "\n",
      "Seq is: GATCCTCCAGACCACT\n",
      "0 GATCCTCCAGACCACT motifs detected in file\n",
      "\n",
      "Seq is: CTTATCACAATTACAG\n",
      "0 CTTATCACAATTACAG motifs detected in file\n",
      "\n",
      "Seq is: TAATTCCTCTCTTCCT\n",
      "0 TAATTCCTCTCTTCCT motifs detected in file\n",
      "\n",
      "Seq is: TCTCTTCATTCGGCTC\n",
      "0 TCTCTTCATTCGGCTC motifs detected in file\n",
      "\n",
      "Seq is: CCGCAGAAACTAACAC\n",
      "0 CCGCAGAAACTAACAC motifs detected in file\n",
      "\n",
      "Seq is: ATCATTCCCACCTTCC\n",
      "0 ATCATTCCCACCTTCC motifs detected in file\n",
      "\n",
      "Seq is: ACTAAAATAGCCTTAC\n",
      "0 ACTAAAATAGCCTTAC motifs detected in file\n",
      "\n",
      "Seq is: CCTCCACTAAGGATCC\n",
      "0 CCTCCACTAAGGATCC motifs detected in file\n",
      "\n",
      "Seq is: CCCAACAAACTAGGGC\n",
      "0 CCCAACAAACTAGGGC motifs detected in file\n",
      "\n",
      "Seq is: AAAGCACGTTCTTATC\n",
      "0 AAAGCACGTTCTTATC motifs detected in file\n",
      "\n",
      "Seq is: CTCCACCTCACAACGA\n",
      "0 CTCCACCTCACAACGA motifs detected in file\n",
      "\n",
      "Seq is: TCCCGACGTTATTACT\n",
      "0 TCCCGACGTTATTACT motifs detected in file\n",
      "\n",
      "Seq is: TCCCGTTCTACTTCAC\n",
      "0 TCCCGTTCTACTTCAC motifs detected in file\n",
      "\n",
      "Seq is: TTTCCATTTCTTAGCA\n",
      "0 TTTCCATTTCTTAGCA motifs detected in file\n",
      "\n",
      "Seq is: CACTCCCACCCTGACA\n",
      "0 CACTCCCACCCTGACA motifs detected in file\n",
      "\n",
      "Seq is: GATGACTCATCGGAGG\n",
      "0 GATGACTCATCGGAGG motifs detected in file\n",
      "\n",
      "Seq is: AGTCTAAGTAACTTGA\n",
      "0 AGTCTAAGTAACTTGA motifs detected in file\n",
      "\n",
      "Seq is: TCCAGACAGGGTTACA\n",
      "0 TCCAGACAGGGTTACA motifs detected in file\n",
      "\n",
      "Seq is: TACCCTACCGCCTATC\n",
      "0 TACCCTACCGCCTATC motifs detected in file\n",
      "\n",
      "Seq is: ATTCAAACATTAATGA\n",
      "0 ATTCAAACATTAATGA motifs detected in file\n",
      "\n",
      "Seq is: CACCGGGGAGTCTTAC\n",
      "0 CACCGGGGAGTCTTAC motifs detected in file\n",
      "\n",
      "Seq is: TATAAACCGGAGTGCC\n",
      "0 TATAAACCGGAGTGCC motifs detected in file\n",
      "\n",
      "Seq is: CTCCTGTATCGGATAC\n",
      "0 CTCCTGTATCGGATAC motifs detected in file\n",
      "\n",
      "Seq is: TTCCGACAACGATATC\n",
      "0 TTCCGACAACGATATC motifs detected in file\n",
      "\n",
      "Seq is: AACGTTCGTCCTCCTA\n",
      "0 AACGTTCGTCCTCCTA motifs detected in file\n",
      "\n",
      "Seq is: TTACGGCTACAAAGTC\n",
      "0 TTACGGCTACAAAGTC motifs detected in file\n",
      "\n",
      "Seq is: CAAAGACTCATCTCTT\n",
      "0 CAAAGACTCATCTCTT motifs detected in file\n",
      "\n",
      "Seq is: TACTAGGCATTATATC\n",
      "0 TACTAGGCATTATATC motifs detected in file\n",
      "\n",
      "Seq is: CGGAGCGGGCTACACA\n",
      "0 CGGAGCGGGCTACACA motifs detected in file\n",
      "\n",
      "Seq is: TCCTTCTCCGTCTATT\n",
      "0 TCCTTCTCCGTCTATT motifs detected in file\n",
      "\n",
      "Seq is: TCTTATAACTCCGCGG\n",
      "0 TCTTATAACTCCGCGG motifs detected in file\n",
      "\n",
      "Seq is: TAACCGCACTTCCATC\n",
      "0 TAACCGCACTTCCATC motifs detected in file\n",
      "\n",
      "Seq is: GCCTACTAAGTCGGTA\n",
      "0 GCCTACTAAGTCGGTA motifs detected in file\n",
      "\n",
      "Seq is: TTAAATGCAGAGCTCA\n",
      "0 TTAAATGCAGAGCTCA motifs detected in file\n",
      "\n",
      "Seq is: CTACACCCGCTAACTA\n",
      "0 CTACACCCGCTAACTA motifs detected in file\n",
      "\n",
      "Seq is: CTTTTCCGCCAACTCC\n",
      "0 CTTTTCCGCCAACTCC motifs detected in file\n",
      "\n",
      "Seq is: GCAGACCACTCATCAC\n",
      "0 GCAGACCACTCATCAC motifs detected in file\n",
      "\n",
      "Seq is: GTACCGATCCTTATCA\n",
      "0 GTACCGATCCTTATCA motifs detected in file\n",
      "\n",
      "Seq is: GGACACCACTAAACCT\n",
      "0 GGACACCACTAAACCT motifs detected in file\n",
      "\n",
      "Seq is: CCTAGTCCGTCCGCGG\n",
      "0 CCTAGTCCGTCCGCGG motifs detected in file\n",
      "\n",
      "Seq is: TTCCTCACTCGGCTTC\n",
      "0 TTCCTCACTCGGCTTC motifs detected in file\n",
      "\n",
      "Seq is: AAAGTAGTACGCCTCT\n",
      "0 AAAGTAGTACGCCTCT motifs detected in file\n",
      "\n",
      "Seq is: ACAACCTACCCCACCC\n",
      "0 ACAACCTACCCCACCC motifs detected in file\n",
      "\n",
      "Seq is: CTCCAGCTACTTACTC\n",
      "0 CTCCAGCTACTTACTC motifs detected in file\n",
      "\n",
      "Seq is: ACCAATTAATTAAAAT\n",
      "0 ACCAATTAATTAAAAT motifs detected in file\n",
      "\n",
      "Seq is: AATCCCCCAATCAAAA\n",
      "0 AATCCCCCAATCAAAA motifs detected in file\n",
      "\n",
      "Seq is: CGCATAACCCCAGTAA\n",
      "0 CGCATAACCCCAGTAA motifs detected in file\n",
      "\n",
      "Seq is: CCACAAGAACATCAAC\n",
      "0 CCACAAGAACATCAAC motifs detected in file\n",
      "\n",
      "Seq is: TTTATGTTGCTACCAA\n",
      "0 TTTATGTTGCTACCAA motifs detected in file\n",
      "\n",
      "Seq is: AAAGTATAGTAACCAG\n",
      "0 AAAGTATAGTAACCAG motifs detected in file\n",
      "\n",
      "Seq is: CACCAACATCAGGCAC\n",
      "0 CACCAACATCAGGCAC motifs detected in file\n",
      "\n",
      "Seq is: GCTCTTATTACTACAT\n",
      "0 GCTCTTATTACTACAT motifs detected in file\n",
      "\n",
      "Seq is: ACGAAACAAAGACAAC\n",
      "0 ACGAAACAAAGACAAC motifs detected in file\n",
      "\n",
      "Seq is: TCACACCCAAATCATT\n",
      "0 TCACACCCAAATCATT motifs detected in file\n",
      "\n",
      "Seq is: ACCCCAAACACCCCAA\n",
      "0 ACCCCAAACACCCCAA motifs detected in file\n",
      "\n",
      "Seq is: AAGAAGATTCGGAAGA\n",
      "0 AAGAAGATTCGGAAGA motifs detected in file\n",
      "\n",
      "Seq is: GGATAAATACCCCCAA\n",
      "0 GGATAAATACCCCCAA motifs detected in file\n",
      "\n",
      "Seq is: ATCATAACTAACAATC\n",
      "0 ATCATAACTAACAATC motifs detected in file\n",
      "\n",
      "Seq is: GCCACACCAGCCCACA\n",
      "0 GCCACACCAGCCCACA motifs detected in file\n",
      "\n",
      "Seq is: CAATAATAAGACTTAA\n",
      "0 CAATAATAAGACTTAA motifs detected in file\n",
      "\n",
      "Seq is: AACCCCCTCCAATATA\n",
      "0 AACCCCCTCCAATATA motifs detected in file\n",
      "\n",
      "Seq is: CCCAAATTATCAAAAA\n",
      "0 CCCAAATTATCAAAAA motifs detected in file\n",
      "\n",
      "Seq is: AATTAAATAAATCCCC\n",
      "0 AATTAAATAAATCCCC motifs detected in file\n",
      "\n",
      "Seq is: CTTACTACCAACAGAA\n",
      "0 CTTACTACCAACAGAA motifs detected in file\n",
      "\n",
      "Seq is: ACCTATATGATGTCGC\n",
      "0 ACCTATATGATGTCGC motifs detected in file\n",
      "\n",
      "Seq is: TACCGATAACTCCTCA\n",
      "0 TACCGATAACTCCTCA motifs detected in file\n",
      "\n",
      "Seq is: TAGGACTCCGTACCCC\n",
      "0 TAGGACTCCGTACCCC motifs detected in file\n",
      "\n",
      "Seq is: CAGTCCCCAACTCCAG\n",
      "0 CAGTCCCCAACTCCAG motifs detected in file\n",
      "\n",
      "Seq is: AACCACTCACAAAATC\n",
      "0 AACCACTCACAAAATC motifs detected in file\n",
      "\n",
      "Seq is: ACCCCAATCGCTACCT\n",
      "0 ACCCCAATCGCTACCT motifs detected in file\n",
      "\n",
      "Seq is: CCATCCTAACCACGAC\n",
      "0 CCATCCTAACCACGAC motifs detected in file\n",
      "\n",
      "Seq is: ACCCACTTTCTCATAC\n",
      "0 ACCCACTTTCTCATAC motifs detected in file\n",
      "\n",
      "Seq is: TACCCCACCACCAACA\n",
      "0 TACCCCACCACCAACA motifs detected in file\n",
      "\n",
      "Seq is: CCATTTGAAATTATCA\n",
      "0 CCATTTGAAATTATCA motifs detected in file\n",
      "\n",
      "Seq is: CATCCTTCGACTTATT\n",
      "0 CATCCTTCGACTTATT motifs detected in file\n",
      "\n",
      "Seq is: AAATACTTCCTCTCCC\n",
      "0 AAATACTTCCTCTCCC motifs detected in file\n",
      "\n",
      "Seq is: CAGTCCCAACTAAGCC\n",
      "0 CAGTCCCAACTAAGCC motifs detected in file\n",
      "\n",
      "Seq is: CTCCTAGGATAACCAC\n",
      "0 CTCCTAGGATAACCAC motifs detected in file\n",
      "\n",
      "Seq is: GCCCCCGAAACATACT\n",
      "0 GCCCCCGAAACATACT motifs detected in file\n",
      "\n",
      "Seq is: AATACCCGCAACTAAT\n",
      "0 AATACCCGCAACTAAT motifs detected in file\n",
      "\n",
      "Seq is: CATCATCAATGACCAA\n",
      "0 CATCATCAATGACCAA motifs detected in file\n",
      "\n",
      "Seq is: CTTGTAACAAACAACC\n",
      "0 CTTGTAACAAACAACC motifs detected in file\n",
      "\n",
      "Seq is: ACATATATAACATTAA\n",
      "0 ACATATATAACATTAA motifs detected in file\n",
      "\n",
      "Seq is: CTCTAACGAGCCCCCT\n",
      "0 CTCTAACGAGCCCCCT motifs detected in file\n",
      "\n",
      "Seq is: TATCCAATACACTAAT\n",
      "0 TATCCAATACACTAAT motifs detected in file\n",
      "\n",
      "Seq is: CCTCATCCCAATCCTA\n",
      "0 CCTCATCCCAATCCTA motifs detected in file\n",
      "\n",
      "Seq is: CTCACCCTTCTTCTTT\n",
      "0 CTCACCCTTCTTCTTT motifs detected in file\n",
      "\n",
      "Seq is: CTCTCCTTCATTTCAA\n",
      "0 CTCTCCTTCATTTCAA motifs detected in file\n",
      "\n",
      "Seq is: ATTAATACGGAAAAAC\n",
      "0 ATTAATACGGAAAAAC motifs detected in file\n",
      "\n",
      "Seq is: CCAACTCCACTACTAC\n",
      "0 CCAACTCCACTACTAC motifs detected in file\n",
      "\n",
      "Seq is: CTCCACCTCTAAACCA\n",
      "0 CTCCACCTCTAAACCA motifs detected in file\n",
      "\n",
      "Seq is: CGACACTTTAACAAAA\n",
      "0 CGACACTTTAACAAAA motifs detected in file\n",
      "\n",
      "Seq is: TCCATTATCGAAAAGA\n",
      "0 TCCATTATCGAAAAGA motifs detected in file\n",
      "\n",
      "Seq is: TCAGTCCAATCCAGAT\n",
      "0 TCAGTCCAATCCAGAT motifs detected in file\n",
      "\n",
      "Seq is: CCTCCTCATCCCCGTC\n",
      "0 CCTCCTCATCCCCGTC motifs detected in file\n",
      "\n",
      "Seq is: CAAAACCGAGCATTCT\n",
      "0 CAAAACCGAGCATTCT motifs detected in file\n",
      "\n",
      "Seq is: TCCGGATCTATCCCCT\n",
      "0 TCCGGATCTATCCCCT motifs detected in file\n",
      "\n",
      "Seq is: AACACGCCACACACTA\n",
      "0 AACACGCCACACACTA motifs detected in file\n",
      "\n",
      "Seq is: CGATCCCATCTTAGGC\n",
      "0 CGATCCCATCTTAGGC motifs detected in file\n",
      "\n",
      "Seq is: TCATACAACCTCTTTA\n",
      "0 TCATACAACCTCTTTA motifs detected in file\n",
      "\n",
      "Seq is: TTTTACACGTATCACC\n",
      "0 TTTTACACGTATCACC motifs detected in file\n",
      "\n",
      "Seq is: CCTAAAATAAAATTCA\n",
      "0 CCTAAAATAAAATTCA motifs detected in file\n",
      "\n",
      "Seq is: AACAACCAATCCATCA\n",
      "0 AACAACCAATCCATCA motifs detected in file\n",
      "\n",
      "Seq is: ACTCCAGATCCCGACA\n",
      "0 ACTCCAGATCCCGACA motifs detected in file\n",
      "\n",
      "Seq is: ATCTTCAGGATCCTTT\n",
      "0 ATCTTCAGGATCCTTT motifs detected in file\n",
      "\n",
      "Seq is: CACTGTCGCTCCCGAC\n",
      "0 CACTGTCGCTCCCGAC motifs detected in file\n",
      "\n",
      "Seq is: ACTCAAAATCCATCTC\n",
      "0 ACTCAAAATCCATCTC motifs detected in file\n",
      "\n",
      "Seq is: CCCCTAACAACAAACC\n",
      "0 CCCCTAACAACAAACC motifs detected in file\n",
      "\n",
      "Seq is: TTCCCCCTACGCCCCC\n",
      "0 TTCCCCCTACGCCCCC motifs detected in file\n",
      "\n",
      "Seq is: TTTACAACAATCATTA\n",
      "0 TTTACAACAATCATTA motifs detected in file\n",
      "\n",
      "Seq is: CTCTTTAGGACGCTTA\n",
      "0 CTCTTTAGGACGCTTA motifs detected in file\n",
      "\n",
      "Seq is: TCCGAAGGCCGACGGT\n",
      "0 TCCGAAGGCCGACGGT motifs detected in file\n",
      "\n",
      "Seq is: CCGCAAATTACCCCAA\n",
      "0 CCGCAAATTACCCCAA motifs detected in file\n",
      "\n",
      "Seq is: ATCATCCCACCCCAAT\n",
      "0 ATCATCCCACCCCAAT motifs detected in file\n",
      "\n",
      "Seq is: AAAAGCAATTACAATC\n",
      "0 AAAAGCAATTACAATC motifs detected in file\n",
      "\n",
      "Seq is: ATTCCCACCCCTTCGC\n",
      "0 ATTCCCACCCCTTCGC motifs detected in file\n",
      "\n",
      "Seq is: TCCAACTGGACAATCC\n",
      "0 TCCAACTGGACAATCC motifs detected in file\n",
      "\n",
      "Seq is: CACTCTTCTTAATAAG\n",
      "0 CACTCTTCTTAATAAG motifs detected in file\n",
      "\n",
      "Seq is: CTCACGATATCCGCGA\n",
      "0 CTCACGATATCCGCGA motifs detected in file\n",
      "\n",
      "Seq is: ACAGTCCCTCCATCGC\n",
      "0 ACAGTCCCTCCATCGC motifs detected in file\n",
      "\n",
      "Seq is: TACTCTCATTATCTAT\n",
      "0 TACTCTCATTATCTAT motifs detected in file\n",
      "\n",
      "Seq is: CCCGAGTCCGCAAACA\n",
      "0 CCCGAGTCCGCAAACA motifs detected in file\n",
      "\n",
      "Seq is: CATACTATACAAACGC\n",
      "0 CATACTATACAAACGC motifs detected in file\n",
      "\n",
      "Seq is: CAAAGCTACTACACCA\n",
      "0 CAAAGCTACTACACCA motifs detected in file\n",
      "\n",
      "Seq is: GAAACCTCATCTTTGG\n",
      "0 GAAACCTCATCTTTGG motifs detected in file\n",
      "\n",
      "Seq is: ACACTCCTTTCCATAA\n",
      "0 ACACTCCTTTCCATAA motifs detected in file\n",
      "\n",
      "Seq is: GGACGATTACGATCCG\n",
      "0 GGACGATTACGATCCG motifs detected in file\n",
      "\n",
      "Seq is: ACGGTTACCACTCCCT\n",
      "0 ACGGTTACCACTCCCT motifs detected in file\n",
      "\n",
      "Seq is: CCAACTTCACTCTCCA\n",
      "0 CCAACTTCACTCTCCA motifs detected in file\n",
      "\n",
      "Seq is: TACCAAAACTCATCAG\n",
      "0 TACCAAAACTCATCAG motifs detected in file\n",
      "\n",
      "Seq is: GAGGATAAAAAGCTTA\n",
      "0 GAGGATAAAAAGCTTA motifs detected in file\n",
      "\n",
      "Seq is: TAGAACAAGTAACAAT\n",
      "0 TAGAACAAGTAACAAT motifs detected in file\n",
      "\n",
      "Seq is: TCCAACACCTACTACC\n",
      "0 TCCAACACCTACTACC motifs detected in file\n",
      "\n",
      "Seq is: TGGGCCTCGTGTATTT\n",
      "0 TGGGCCTCGTGTATTT motifs detected in file\n",
      "\n",
      "Seq is: ATCATACCGAAACTTC\n",
      "0 ATCATACCGAAACTTC motifs detected in file\n",
      "\n",
      "Seq is: TTCCGCACCCATGTCT\n",
      "0 TTCCGCACCCATGTCT motifs detected in file\n",
      "\n",
      "Seq is: ACACGTCCTTACGATC\n",
      "0 ACACGTCCTTACGATC motifs detected in file\n",
      "\n",
      "Seq is: CACACCAACCAACTAC\n",
      "0 CACACCAACCAACTAC motifs detected in file\n",
      "\n",
      "Seq is: GGCTAACATTGATAAT\n",
      "0 GGCTAACATTGATAAT motifs detected in file\n",
      "\n",
      "Seq is: ACTCAGGATCAACTGA\n",
      "0 ACTCAGGATCAACTGA motifs detected in file\n",
      "\n",
      "Seq is: ACTTCACCTCTTCCGA\n",
      "0 ACTTCACCTCTTCCGA motifs detected in file\n",
      "\n",
      "Seq is: TGCTAAAAAAACTACA\n",
      "0 TGCTAAAAAAACTACA motifs detected in file\n",
      "\n",
      "Seq is: GTAAAACACATTCCCG\n",
      "0 GTAAAACACATTCCCG motifs detected in file\n",
      "\n",
      "Seq is: CGTCTGACGACGCTTG\n",
      "0 CGTCTGACGACGCTTG motifs detected in file\n",
      "\n",
      "Seq is: TCTCACCACTATCGCG\n",
      "0 TCTCACCACTATCGCG motifs detected in file\n",
      "\n",
      "Seq is: GATTCGTATCACAATC\n",
      "0 GATTCGTATCACAATC motifs detected in file\n",
      "\n",
      "Seq is: TCAAACCTAATCACCC\n",
      "0 TCAAACCTAATCACCC motifs detected in file\n",
      "\n",
      "Seq is: GATAAAAGACGATCCC\n",
      "0 GATAAAAGACGATCCC motifs detected in file\n",
      "\n",
      "Seq is: CCACCTTCGCCTACTC\n",
      "0 CCACCTTCGCCTACTC motifs detected in file\n",
      "\n",
      "Seq is: ATTCTTCTAAGGACGA\n",
      "0 ATTCTTCTAAGGACGA motifs detected in file\n",
      "\n",
      "Seq is: TGTTGATATCACGAAC\n",
      "0 TGTTGATATCACGAAC motifs detected in file\n",
      "\n",
      "Seq is: TCACCTCATCCCGACT\n",
      "0 TCACCTCATCCCGACT motifs detected in file\n",
      "\n",
      "Seq is: CTGACCCCACCCCGGA\n",
      "0 CTGACCCCACCCCGGA motifs detected in file\n",
      "\n",
      "Seq is: AGATACCGACTCCCCT\n",
      "0 AGATACCGACTCCCCT motifs detected in file\n",
      "\n",
      "Seq is: CAGTCCCCACCTCTGG\n",
      "0 CAGTCCCCACCTCTGG motifs detected in file\n",
      "\n",
      "Seq is: ATTAACCCGACTAAAC\n",
      "0 ATTAACCCGACTAAAC motifs detected in file\n",
      "\n",
      "Seq is: GGACGACGACGATCCT\n",
      "0 GGACGACGACGATCCT motifs detected in file\n",
      "\n",
      "Seq is: CCTCCGGATCATCACC\n",
      "0 CCTCCGGATCATCACC motifs detected in file\n",
      "\n",
      "Seq is: CCACTCCGAACCTAAT\n",
      "0 CCACTCCGAACCTAAT motifs detected in file\n",
      "\n",
      "Seq is: CGCAAATCTTCCCGAT\n",
      "0 CGCAAATCTTCCCGAT motifs detected in file\n",
      "\n",
      "Seq is: AAACAACACCCAGAGT\n",
      "0 AAACAACACCCAGAGT motifs detected in file\n",
      "\n",
      "Seq is: ACTCAACCTCACATCC\n",
      "0 ACTCAACCTCACATCC motifs detected in file\n",
      "\n",
      "Seq is: TATTTAGTACGATTCC\n",
      "0 TATTTAGTACGATTCC motifs detected in file\n",
      "\n",
      "Seq is: GCTCCTACTTTGGCTA\n",
      "0 GCTCCTACTTTGGCTA motifs detected in file\n",
      "\n",
      "Seq is: TAGCGGCTATGCCAAC\n",
      "0 TAGCGGCTATGCCAAC motifs detected in file\n",
      "\n",
      "Seq is: ATATCCTAACGAACTT\n",
      "0 ATATCCTAACGAACTT motifs detected in file\n",
      "\n",
      "Seq is: ACCGACGACACAACCG\n",
      "0 ACCGACGACACAACCG motifs detected in file\n",
      "\n",
      "Seq is: TAGACGAGCCCGCATA\n",
      "0 TAGACGAGCCCGCATA motifs detected in file\n",
      "\n",
      "Seq is: GTAGTTGACTACTCGT\n",
      "0 GTAGTTGACTACTCGT motifs detected in file\n",
      "\n",
      "Seq is: TCTTCCTATATTAAGG\n",
      "0 TCTTCCTATATTAAGG motifs detected in file\n",
      "\n",
      "Seq is: ATGCGGGAGAGTCGGC\n",
      "0 ATGCGGGAGAGTCGGC motifs detected in file\n",
      "\n",
      "Seq is: TACTTGTCAACCTTAT\n",
      "0 TACTTGTCAACCTTAT motifs detected in file\n",
      "\n",
      "Seq is: CCAACAATCGCCATTG\n",
      "0 CCAACAATCGCCATTG motifs detected in file\n",
      "\n",
      "Seq is: ATTCTAATCATACCAT\n",
      "0 ATTCTAATCATACCAT motifs detected in file\n",
      "\n",
      "Seq is: TAATCCTTCTACTCAT\n",
      "0 TAATCCTTCTACTCAT motifs detected in file\n",
      "\n",
      "Seq is: CTATAAACTTCTTGAC\n",
      "0 CTATAAACTTCTTGAC motifs detected in file\n",
      "\n",
      "Seq is: TAATTACAAACCCAGA\n",
      "0 TAATTACAAACCCAGA motifs detected in file\n",
      "\n",
      "Seq is: CTCAAATATATAGTGT\n",
      "0 CTCAAATATATAGTGT motifs detected in file\n",
      "\n",
      "Seq is: CACTCTTAAGATACTA\n",
      "0 CACTCTTAAGATACTA motifs detected in file\n",
      "\n",
      "Seq is: CCTGGTACATTGCTTG\n",
      "0 CCTGGTACATTGCTTG motifs detected in file\n",
      "\n",
      "Seq is: TTACGATGTCCCTACT\n",
      "0 TTACGATGTCCCTACT motifs detected in file\n",
      "\n",
      "Seq is: TATAATACCTCTTCAT\n",
      "0 TATAATACCTCTTCAT motifs detected in file\n",
      "\n",
      "Seq is: CAGATCAAACTTCGAA\n",
      "0 CAGATCAAACTTCGAA motifs detected in file\n",
      "\n",
      "Seq is: TCCCTCTCGACCCAAC\n",
      "0 TCCCTCTCGACCCAAC motifs detected in file\n",
      "\n",
      "Seq is: AAACCCAACACCGAGT\n",
      "0 AAACCCAACACCGAGT motifs detected in file\n",
      "\n",
      "Seq is: CACAGTCAAGCTCTAT\n",
      "0 CACAGTCAAGCTCTAT motifs detected in file\n",
      "\n",
      "Seq is: TATTGAAGAACCAGAT\n",
      "0 TATTGAAGAACCAGAT motifs detected in file\n",
      "\n",
      "Seq is: CCGTGTACTTATAACA\n",
      "0 CCGTGTACTTATAACA motifs detected in file\n",
      "\n",
      "Seq is: ACACCCCTTCTCTGAC\n",
      "0 ACACCCCTTCTCTGAC motifs detected in file\n",
      "\n",
      "Seq is: TATTATTTCCACCTAC\n",
      "0 TATTATTTCCACCTAC motifs detected in file\n",
      "\n",
      "Seq is: GCTGTTACCTAAAATG\n",
      "0 GCTGTTACCTAAAATG motifs detected in file\n",
      "\n",
      "Seq is: TATTACCCCCATACTC\n",
      "0 TATTACCCCCATACTC motifs detected in file\n",
      "\n",
      "Seq is: AAAAAAAACAATCCCA\n",
      "0 AAAAAAAACAATCCCA motifs detected in file\n",
      "\n",
      "Seq is: ATTGCTCCCACCATTC\n",
      "0 ATTGCTCCCACCATTC motifs detected in file\n",
      "\n",
      "Seq is: CTACCCCCCTTAATCC\n",
      "0 CTACCCCCCTTAATCC motifs detected in file\n",
      "\n",
      "Seq is: CTTCAGTCCCAATCCC\n",
      "0 CTTCAGTCCCAATCCC motifs detected in file\n",
      "\n",
      "Seq is: ACCAATATCATCACAC\n",
      "0 ACCAATATCATCACAC motifs detected in file\n",
      "\n",
      "Seq is: GTACCAATAATGAAAA\n",
      "0 GTACCAATAATGAAAA motifs detected in file\n",
      "\n",
      "Seq is: TAAACCTCAACGTGGT\n",
      "0 TAAACCTCAACGTGGT motifs detected in file\n",
      "\n",
      "Seq is: TTTAAAAACCCCGGAT\n",
      "0 TTTAAAAACCCCGGAT motifs detected in file\n",
      "\n",
      "Seq is: TCTGGTTACCTATCGA\n",
      "0 TCTGGTTACCTATCGA motifs detected in file\n",
      "\n",
      "Seq is: CAATAGGAAATTTTCA\n",
      "0 CAATAGGAAATTTTCA motifs detected in file\n",
      "\n",
      "Seq is: ACTCTTTCGGTACAAC\n",
      "0 ACTCTTTCGGTACAAC motifs detected in file\n",
      "\n",
      "Seq is: AATCTGTACCCCCGTA\n",
      "0 AATCTGTACCCCCGTA motifs detected in file\n",
      "\n",
      "Seq is: CTCAATCGTCAAGAAC\n",
      "0 CTCAATCGTCAAGAAC motifs detected in file\n",
      "\n",
      "Seq is: ACTCGAAAGAGCCATT\n",
      "0 ACTCGAAAGAGCCATT motifs detected in file\n",
      "\n",
      "Seq is: TATTCCCCAGCATTCG\n",
      "0 TATTCCCCAGCATTCG motifs detected in file\n",
      "\n",
      "Seq is: GAGACAACAGTCTAAG\n",
      "0 GAGACAACAGTCTAAG motifs detected in file\n",
      "\n",
      "Seq is: TGTTAGACTACAAAAC\n",
      "0 TGTTAGACTACAAAAC motifs detected in file\n",
      "\n",
      "Seq is: CAATTTGATATAAATG\n",
      "0 CAATTTGATATAAATG motifs detected in file\n",
      "\n",
      "Seq is: TTCTCCTTTTGGGCCA\n",
      "0 TTCTCCTTTTGGGCCA motifs detected in file\n",
      "\n",
      "Seq is: TTACTACAGCCCCAAC\n",
      "0 TTACTACAGCCCCAAC motifs detected in file\n",
      "\n",
      "Seq is: TCCCTATCCTCCTCTT\n",
      "0 TCCCTATCCTCCTCTT motifs detected in file\n",
      "\n",
      "Seq is: ACCCCCTATCCACATA\n",
      "0 ACCCCCTATCCACATA motifs detected in file\n",
      "\n",
      "Seq is: CTTGTACTCCCACAAA\n",
      "0 CTTGTACTCCCACAAA motifs detected in file\n",
      "\n",
      "Seq is: AGAGCACACTTACTCC\n",
      "0 AGAGCACACTTACTCC motifs detected in file\n",
      "\n",
      "Seq is: CAAAATACAACAATTA\n",
      "0 CAAAATACAACAATTA motifs detected in file\n",
      "\n",
      "Seq is: CACCACCCACTCACTC\n",
      "0 CACCACCCACTCACTC motifs detected in file\n",
      "\n",
      "Seq is: GGGGTAACACAACACC\n",
      "0 GGGGTAACACAACACC motifs detected in file\n",
      "\n",
      "Seq is: ATTTATACATCTCCCT\n",
      "0 ATTTATACATCTCCCT motifs detected in file\n",
      "\n",
      "Seq is: CATATCCCGACACTGA\n",
      "0 CATATCCCGACACTGA motifs detected in file\n",
      "\n",
      "Seq is: TCATACAACTCAGGAC\n",
      "0 TCATACAACTCAGGAC motifs detected in file\n",
      "\n",
      "Seq is: ATTCATCCTCTCACTA\n",
      "0 ATTCATCCTCTCACTA motifs detected in file\n",
      "\n",
      "Seq is: TAAACTAGTCCTCTTG\n",
      "0 TAAACTAGTCCTCTTG motifs detected in file\n",
      "\n",
      "Seq is: CACCAATGATCGTGTC\n",
      "0 CACCAATGATCGTGTC motifs detected in file\n",
      "\n",
      "Seq is: TCTCAAGAGGGTCATC\n",
      "0 TCTCAAGAGGGTCATC motifs detected in file\n",
      "\n",
      "Seq is: CAATTATCACCCCCCA\n",
      "0 CAATTATCACCCCCCA motifs detected in file\n",
      "\n",
      "Seq is: TTCCGCTCCAATCGCT\n",
      "0 TTCCGCTCCAATCGCT motifs detected in file\n",
      "\n",
      "Seq is: CCGAACGATCTTCAGT\n",
      "0 CCGAACGATCTTCAGT motifs detected in file\n",
      "\n",
      "Seq is: AGTTTTTCGATAATCA\n",
      "0 AGTTTTTCGATAATCA motifs detected in file\n",
      "\n",
      "Seq is: CCCTCATCTCAAACTT\n",
      "0 CCCTCATCTCAAACTT motifs detected in file\n",
      "\n",
      "Seq is: CAGGAACTCTCTCCTA\n",
      "0 CAGGAACTCTCTCCTA motifs detected in file\n",
      "\n",
      "Seq is: ATACTACGCTGACACT\n",
      "0 ATACTACGCTGACACT motifs detected in file\n",
      "\n",
      "Seq is: CACGCAAGCATCAAAC\n",
      "0 CACGCAAGCATCAAAC motifs detected in file\n",
      "\n",
      "Seq is: TCAAACGATCCGTCTT\n",
      "0 TCAAACGATCCGTCTT motifs detected in file\n",
      "\n",
      "Seq is: ATCATTACTCCTACAT\n",
      "0 ATCATTACTCCTACAT motifs detected in file\n",
      "\n",
      "Seq is: TCGGGCACCCGCTAAT\n",
      "0 TCGGGCACCCGCTAAT motifs detected in file\n",
      "\n",
      "Seq is: ACTCTTACTGACGCGG\n",
      "0 ACTCTTACTGACGCGG motifs detected in file\n",
      "\n",
      "Seq is: CCACTTCGAAGTCCCC\n",
      "0 CCACTTCGAAGTCCCC motifs detected in file\n",
      "\n",
      "Seq is: CAAACCTACTCTTACC\n",
      "0 CAAACCTACTCTTACC motifs detected in file\n",
      "\n",
      "Seq is: GACAATGATGCTCCCG\n",
      "0 GACAATGATGCTCCCG motifs detected in file\n",
      "\n",
      "Seq is: ATACACCGACTAACTT\n",
      "0 ATACACCGACTAACTT motifs detected in file\n",
      "\n",
      "Seq is: CTCATACGTTACTCGC\n",
      "0 CTCATACGTTACTCGC motifs detected in file\n",
      "\n",
      "Seq is: TAAAATCCAGACAAAC\n",
      "0 TAAAATCCAGACAAAC motifs detected in file\n",
      "\n",
      "Seq is: AGCATCCGTCTACCTC\n",
      "0 AGCATCCGTCTACCTC motifs detected in file\n",
      "\n",
      "Seq is: GAACAATATTAATACG\n",
      "0 GAACAATATTAATACG motifs detected in file\n",
      "\n",
      "Seq is: GAGTATCCCTATCATG\n",
      "0 GAGTATCCCTATCATG motifs detected in file\n",
      "\n",
      "Seq is: TTCCTTCCCCATCCGA\n",
      "0 TTCCTTCCCCATCCGA motifs detected in file\n",
      "\n",
      "Seq is: TACACAAAACAGTCCC\n",
      "0 TACACAAAACAGTCCC motifs detected in file\n",
      "\n",
      "Seq is: CCAACTCTTACTCACA\n",
      "0 CCAACTCTTACTCACA motifs detected in file\n",
      "\n",
      "Seq is: CTCCGCATAATATGGT\n",
      "0 CTCCGCATAATATGGT motifs detected in file\n",
      "\n",
      "Seq is: ATCGGCGGATCAAAAT\n",
      "0 ATCGGCGGATCAAAAT motifs detected in file\n",
      "\n",
      "Seq is: TCTCATGACGCCGTTC\n",
      "0 TCTCATGACGCCGTTC motifs detected in file\n",
      "\n",
      "Seq is: ATGATAACTGGGTCGC\n",
      "0 ATGATAACTGGGTCGC motifs detected in file\n",
      "\n",
      "Seq is: TACCCCCGAAGCTGTA\n",
      "0 TACCCCCGAAGCTGTA motifs detected in file\n",
      "\n",
      "Seq is: CCCGAAATCCCTCAGT\n",
      "0 CCCGAAATCCCTCAGT motifs detected in file\n",
      "\n",
      "Seq is: ATTCACCTCAGGCATT\n",
      "0 ATTCACCTCAGGCATT motifs detected in file\n",
      "\n",
      "Seq is: TCTCCATAGAAATGAT\n",
      "0 TCTCCATAGAAATGAT motifs detected in file\n",
      "\n",
      "Seq is: ATTTTCGATAACACAT\n",
      "0 ATTTTCGATAACACAT motifs detected in file\n",
      "\n",
      "Seq is: TCGATCAGTATAATTC\n",
      "0 TCGATCAGTATAATTC motifs detected in file\n",
      "\n",
      "Seq is: AACAACCGAGTCCTCA\n",
      "0 AACAACCGAGTCCTCA motifs detected in file\n",
      "\n",
      "Seq is: AACTATCAAGAACCCG\n",
      "0 AACTATCAAGAACCCG motifs detected in file\n",
      "\n",
      "Seq is: TCACTCTCACTCATCA\n",
      "0 TCACTCTCACTCATCA motifs detected in file\n",
      "\n",
      "Seq is: TCTTACAAATCACTCG\n",
      "0 TCTTACAAATCACTCG motifs detected in file\n",
      "\n",
      "Seq is: GATCCCACAACACTCA\n",
      "0 GATCCCACAACACTCA motifs detected in file\n",
      "\n",
      "Seq is: CATTTAATCACGCTAC\n",
      "0 CATTTAATCACGCTAC motifs detected in file\n",
      "\n",
      "Seq is: TCATCCCCTTCCCTCG\n",
      "0 TCATCCCCTTCCCTCG motifs detected in file\n",
      "\n",
      "Seq is: GATGATCCCACATCTT\n",
      "0 GATGATCCCACATCTT motifs detected in file\n",
      "\n",
      "Seq is: ATCCTTCATACACGGA\n",
      "0 ATCCTTCATACACGGA motifs detected in file\n",
      "\n",
      "Seq is: CGCAAGTCCGCAAGAC\n",
      "0 CGCAAGTCCGCAAGAC motifs detected in file\n",
      "\n",
      "Seq is: CGACCAACGGAGTAGC\n",
      "0 CGACCAACGGAGTAGC motifs detected in file\n",
      "\n",
      "Seq is: CCACTACTATCGGTTC\n",
      "0 CCACTACTATCGGTTC motifs detected in file\n",
      "\n",
      "Seq is: CACCCCTATTCACACC\n",
      "0 CACCCCTATTCACACC motifs detected in file\n",
      "\n",
      "Seq is: AAAGCTTCTTCTATAT\n",
      "0 AAAGCTTCTTCTATAT motifs detected in file\n",
      "\n",
      "Seq is: TTTATACTAATCAAGA\n",
      "0 TTTATACTAATCAAGA motifs detected in file\n",
      "\n",
      "Seq is: CACCGACACTTACAAT\n",
      "0 CACCGACACTTACAAT motifs detected in file\n",
      "\n",
      "Seq is: ATTAATTCCTCTAAAC\n",
      "0 ATTAATTCCTCTAAAC motifs detected in file\n",
      "\n",
      "Seq is: ATCCCTCTAATCATAT\n",
      "0 ATCCCTCTAATCATAT motifs detected in file\n",
      "\n",
      "Seq is: CTCTCCATCTCAAAAA\n",
      "0 CTCTCCATCTCAAAAA motifs detected in file\n",
      "\n",
      "Seq is: AAGCACTATCACCAAG\n",
      "0 AAGCACTATCACCAAG motifs detected in file\n",
      "\n",
      "Seq is: TGACCTATTCACCGCA\n",
      "0 TGACCTATTCACCGCA motifs detected in file\n",
      "\n",
      "Seq is: ACCGAACGGTACTAAC\n",
      "0 ACCGAACGGTACTAAC motifs detected in file\n",
      "\n",
      "Seq is: ACTCCCCATCCTCAGT\n",
      "0 ACTCCCCATCCTCAGT motifs detected in file\n",
      "\n",
      "Seq is: CCATCAATCATAATCC\n",
      "0 CCATCAATCATAATCC motifs detected in file\n",
      "\n",
      "Seq is: TCCCCCCAACAATCCC\n",
      "0 TCCCCCCAACAATCCC motifs detected in file\n",
      "\n",
      "Seq is: CCAGCCTCCTTTTCCA\n",
      "0 CCAGCCTCCTTTTCCA motifs detected in file\n",
      "\n",
      "Seq is: ACCCCTTGTCGATTTA\n",
      "0 ACCCCTTGTCGATTTA motifs detected in file\n",
      "\n",
      "Seq is: TCCAACAACAACTAAA\n",
      "0 TCCAACAACAACTAAA motifs detected in file\n",
      "\n",
      "Seq is: CCAATTTTTTATCATC\n",
      "0 CCAATTTTTTATCATC motifs detected in file\n",
      "\n",
      "Seq is: TCCCTACTACGATTAT\n",
      "0 TCCCTACTACGATTAT motifs detected in file\n",
      "\n",
      "Seq is: TAATCCGACACCCACC\n",
      "0 TAATCCGACACCCACC motifs detected in file\n",
      "\n",
      "Seq is: AACACAACTAAGTTTA\n",
      "0 AACACAACTAAGTTTA motifs detected in file\n",
      "\n",
      "Seq is: ATACACAAAAAACCTT\n",
      "0 ATACACAAAAAACCTT motifs detected in file\n",
      "\n",
      "Seq is: TCAGTACAGTCACCAT\n",
      "0 TCAGTACAGTCACCAT motifs detected in file\n",
      "\n",
      "Seq is: CATTATATTAACAACC\n",
      "0 CATTATATTAACAACC motifs detected in file\n",
      "\n",
      "Seq is: CTGCTAATCAAAATCG\n",
      "0 CTGCTAATCAAAATCG motifs detected in file\n",
      "\n",
      "Seq is: TAACCTCATCCAAATC\n",
      "0 TAACCTCATCCAAATC motifs detected in file\n",
      "\n",
      "Seq is: CAATACATGCATCAGA\n",
      "0 CAATACATGCATCAGA motifs detected in file\n",
      "\n",
      "Seq is: TCCGGTATACACAACC\n",
      "0 TCCGGTATACACAACC motifs detected in file\n",
      "\n",
      "Seq is: TCTAACTCTGATCATC\n",
      "0 TCTAACTCTGATCATC motifs detected in file\n",
      "\n",
      "Seq is: CCGATCCGGGTGGCGA\n",
      "0 CCGATCCGGGTGGCGA motifs detected in file\n",
      "\n",
      "Seq is: CGAAGCGTCCGCCGTT\n",
      "0 CGAAGCGTCCGCCGTT motifs detected in file\n",
      "\n",
      "Seq is: TCTGATCATACCGTTA\n",
      "0 TCTGATCATACCGTTA motifs detected in file\n",
      "\n",
      "Seq is: TCCGTGTTATAACCGA\n",
      "0 TCCGTGTTATAACCGA motifs detected in file\n",
      "\n",
      "Seq is: TTCTCCCTCACCCACA\n",
      "0 TTCTCCCTCACCCACA motifs detected in file\n",
      "\n",
      "Seq is: ACTCCCAATACTCTCA\n",
      "0 ACTCCCAATACTCTCA motifs detected in file\n",
      "\n",
      "Seq is: TCGATATTACTTGTCG\n",
      "0 TCGATATTACTTGTCG motifs detected in file\n",
      "\n",
      "Seq is: CTATCATAATAAGGAA\n",
      "0 CTATCATAATAAGGAA motifs detected in file\n",
      "\n",
      "Seq is: GATCCGTATCATCCCT\n",
      "0 GATCCGTATCATCCCT motifs detected in file\n",
      "\n",
      "Seq is: CCTATACTCCACACTC\n",
      "0 CCTATACTCCACACTC motifs detected in file\n",
      "\n",
      "Seq is: GCTATATGATCATAAG\n",
      "0 GCTATATGATCATAAG motifs detected in file\n",
      "\n",
      "Seq is: GATCTTCACTCTACCA\n",
      "0 GATCTTCACTCTACCA motifs detected in file\n",
      "\n",
      "Seq is: TTTACGATCATATTAT\n",
      "0 TTTACGATCATATTAT motifs detected in file\n",
      "\n",
      "Seq is: AAATACATTTACTCCC\n",
      "0 AAATACATTTACTCCC motifs detected in file\n",
      "\n",
      "Seq is: CGTAAACCATTTATAC\n",
      "0 CGTAAACCATTTATAC motifs detected in file\n",
      "\n",
      "Seq is: TAATAGTATTAAATTA\n",
      "0 TAATAGTATTAAATTA motifs detected in file\n",
      "\n",
      "Seq is: CTCAGCTTTAGTAAGC\n",
      "0 CTCAGCTTTAGTAAGC motifs detected in file\n",
      "\n",
      "Seq is: AAAACAAATTTGATAT\n",
      "0 AAAACAAATTTGATAT motifs detected in file\n",
      "\n",
      "Seq is: ATGGTTAAGCCAAGTC\n",
      "0 ATGGTTAAGCCAAGTC motifs detected in file\n",
      "\n",
      "Seq is: AGATTAGGAAAAACAT\n",
      "0 AGATTAGGAAAAACAT motifs detected in file\n",
      "\n",
      "Seq is: CAGTGAGTATCCGGTC\n",
      "0 CAGTGAGTATCCGGTC motifs detected in file\n",
      "\n",
      "Seq is: TGAATCCCGATCCTAC\n",
      "0 TGAATCCCGATCCTAC motifs detected in file\n",
      "\n",
      "Seq is: TACTAATTATTCTCCC\n",
      "0 TACTAATTATTCTCCC motifs detected in file\n",
      "\n",
      "Seq is: TACTGTATTGATAATC\n",
      "0 TACTGTATTGATAATC motifs detected in file\n",
      "\n",
      "Seq is: ACCGTCCAATCAACAA\n",
      "0 ACCGTCCAATCAACAA motifs detected in file\n",
      "\n",
      "Seq is: ACATCCCGAGTACCAT\n",
      "0 ACATCCCGAGTACCAT motifs detected in file\n",
      "\n",
      "Seq is: CCCCATTTTCCTCCCG\n",
      "0 CCCCATTTTCCTCCCG motifs detected in file\n",
      "\n",
      "Seq is: TTAAAGATCTAGTTTA\n",
      "0 TTAAAGATCTAGTTTA motifs detected in file\n",
      "\n",
      "Seq is: TTATTCTTCCATTATC\n",
      "0 TTATTCTTCCATTATC motifs detected in file\n",
      "\n",
      "Seq is: GATGATTCTTCTTAAA\n",
      "0 GATGATTCTTCTTAAA motifs detected in file\n",
      "\n",
      "Seq is: ATACCTCTTTCCCTGC\n",
      "0 ATACCTCTTTCCCTGC motifs detected in file\n",
      "\n",
      "Seq is: GCCCGCCCCCTATATC\n",
      "0 GCCCGCCCCCTATATC motifs detected in file\n",
      "\n",
      "Seq is: CCAGCTTCGGCGTGAG\n",
      "0 CCAGCTTCGGCGTGAG motifs detected in file\n",
      "\n",
      "Seq is: CATTCCCCACCTAAAA\n",
      "0 CATTCCCCACCTAAAA motifs detected in file\n",
      "\n",
      "Seq is: AGATACATCGGCAACT\n",
      "0 AGATACATCGGCAACT motifs detected in file\n",
      "\n",
      "Seq is: CAACACCATCAGTTTT\n",
      "0 CAACACCATCAGTTTT motifs detected in file\n",
      "\n",
      "Seq is: ACATTATTAATAATCA\n",
      "0 ACATTATTAATAATCA motifs detected in file\n",
      "\n",
      "Seq is: TCATTCCGATCCTCCC\n",
      "0 TCATTCCGATCCTCCC motifs detected in file\n",
      "\n",
      "Seq is: ACAACTAATAATTTTA\n",
      "0 ACAACTAATAATTTTA motifs detected in file\n",
      "\n",
      "Seq is: ATTCCGCTTCAAATAA\n",
      "0 ATTCCGCTTCAAATAA motifs detected in file\n",
      "\n",
      "Seq is: TGAGAAAAAACTTACA\n",
      "0 TGAGAAAAAACTTACA motifs detected in file\n",
      "\n",
      "Seq is: ACAGTTTTGATCAATT\n",
      "0 ACAGTTTTGATCAATT motifs detected in file\n",
      "\n",
      "Seq is: AACCTTCAATTGCCAT\n",
      "0 AACCTTCAATTGCCAT motifs detected in file\n",
      "\n",
      "Seq is: GATAAATATGATTTTC\n",
      "0 GATAAATATGATTTTC motifs detected in file\n",
      "\n",
      "Seq is: TCATTCTGGGAGTAGT\n",
      "0 TCATTCTGGGAGTAGT motifs detected in file\n",
      "\n",
      "Seq is: TATCTACCTCTGTATG\n",
      "0 TATCTACCTCTGTATG motifs detected in file\n",
      "\n",
      "Seq is: TCTTTATCAGTTTGGT\n",
      "0 TCTTTATCAGTTTGGT motifs detected in file\n",
      "\n",
      "Seq is: GTAGATGTTTTACGGT\n",
      "0 GTAGATGTTTTACGGT motifs detected in file\n",
      "\n",
      "Seq is: CATAGTCCGCCGCCGA\n",
      "0 CATAGTCCGCCGCCGA motifs detected in file\n",
      "\n",
      "Seq is: AGCTTCGGTTTCACTA\n",
      "0 AGCTTCGGTTTCACTA motifs detected in file\n",
      "\n",
      "Seq is: CAAACCTACATTTCAC\n",
      "0 CAAACCTACATTTCAC motifs detected in file\n",
      "\n",
      "Seq is: TTTATAATCAACCGCC\n",
      "0 TTTATAATCAACCGCC motifs detected in file\n",
      "\n",
      "Seq is: TACTTCGTCTATCACT\n",
      "0 TACTTCGTCTATCACT motifs detected in file\n",
      "\n",
      "Seq is: CCTTTCAACTCGGTTA\n",
      "0 CCTTTCAACTCGGTTA motifs detected in file\n",
      "\n",
      "Seq is: TTACTGCACTTCAGGC\n",
      "0 TTACTGCACTTCAGGC motifs detected in file\n",
      "\n",
      "Seq is: ACCTTCGGACACCGAT\n",
      "0 ACCTTCGGACACCGAT motifs detected in file\n",
      "\n",
      "Seq is: GTTTTTTACAACTCGG\n",
      "0 GTTTTTTACAACTCGG motifs detected in file\n",
      "\n",
      "Seq is: CATCTACGGCAGCCTT\n",
      "0 CATCTACGGCAGCCTT motifs detected in file\n",
      "\n",
      "Seq is: TACCACTTCCCTCTGA\n",
      "0 TACCACTTCCCTCTGA motifs detected in file\n",
      "\n",
      "Seq is: GCTTCATGAGACTCCG\n",
      "0 GCTTCATGAGACTCCG motifs detected in file\n",
      "\n",
      "Seq is: AACATCCTCCCATTTT\n",
      "0 AACATCCTCCCATTTT motifs detected in file\n",
      "\n",
      "Seq is: ATCTCTGGGTCATTTT\n",
      "0 ATCTCTGGGTCATTTT motifs detected in file\n",
      "\n",
      "Seq is: AACATTATTCGTCACG\n",
      "0 AACATTATTCGTCACG motifs detected in file\n",
      "\n",
      "Seq is: AACTTAATAAACCAAA\n",
      "0 AACTTAATAAACCAAA motifs detected in file\n",
      "\n",
      "Seq is: GCCAACAAAAGATAAT\n",
      "0 GCCAACAAAAGATAAT motifs detected in file\n",
      "\n",
      "Seq is: CTGATACCACTCGAGT\n",
      "0 CTGATACCACTCGAGT motifs detected in file\n",
      "\n",
      "Seq is: CCACTAACTATGAGGA\n",
      "0 CCACTAACTATGAGGA motifs detected in file\n",
      "\n",
      "Seq is: CTACGCTCATTATGCC\n",
      "0 CTACGCTCATTATGCC motifs detected in file\n",
      "\n",
      "Seq is: TACACAAATCCTCACC\n",
      "0 TACACAAATCCTCACC motifs detected in file\n",
      "\n",
      "Seq is: CTGAAGATCCCCTAAA\n",
      "0 CTGAAGATCCCCTAAA motifs detected in file\n",
      "\n",
      "Seq is: TCGCCCCACTACGGAC\n",
      "0 TCGCCCCACTACGGAC motifs detected in file\n",
      "\n",
      "Seq is: AACCCCCGGTCACGGG\n",
      "0 AACCCCCGGTCACGGG motifs detected in file\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Seq is: AGGATTAACCCCCCAT\n",
      "0 AGGATTAACCCCCCAT motifs detected in file\n",
      "\n",
      "Seq is: CCCCGATCCGACCTCA\n",
      "0 CCCCGATCCGACCTCA motifs detected in file\n",
      "\n",
      "Seq is: CCATTTTCCGAGTCTT\n",
      "0 CCATTTTCCGAGTCTT motifs detected in file\n",
      "\n",
      "Seq is: TTTAGGACGCTTCTTT\n",
      "0 TTTAGGACGCTTCTTT motifs detected in file\n",
      "\n",
      "Seq is: TTTTGAAGACTCCATT\n",
      "0 TTTTGAAGACTCCATT motifs detected in file\n",
      "\n",
      "Seq is: ATTTATCCTAATAGGG\n",
      "0 ATTTATCCTAATAGGG motifs detected in file\n",
      "\n",
      "Seq is: CATAGCTTCCGGAAAA\n",
      "0 CATAGCTTCCGGAAAA motifs detected in file\n",
      "\n",
      "Seq is: ACCTGTCCACCACACA\n",
      "0 ACCTGTCCACCACACA motifs detected in file\n",
      "\n",
      "Seq is: CCACCGGAACCATACA\n",
      "0 CCACCGGAACCATACA motifs detected in file\n",
      "\n",
      "Seq is: CGAAAGAGCACAATGT\n",
      "0 CGAAAGAGCACAATGT motifs detected in file\n",
      "\n",
      "Seq is: AGCGCGGTAGTAACCA\n",
      "0 AGCGCGGTAGTAACCA motifs detected in file\n",
      "\n",
      "Seq is: TATACCAATCACACAA\n",
      "0 TATACCAATCACACAA motifs detected in file\n",
      "\n",
      "Seq is: CCAATCATCCGGATCA\n",
      "0 CCAATCATCCGGATCA motifs detected in file\n",
      "\n",
      "Seq is: TACTCCTCGCAATACC\n",
      "0 TACTCCTCGCAATACC motifs detected in file\n",
      "\n",
      "Seq is: TCACCTTCACTTTAGT\n",
      "0 TCACCTTCACTTTAGT motifs detected in file\n",
      "\n",
      "Seq is: GTACCGATCCGGCCTC\n",
      "0 GTACCGATCCGGCCTC motifs detected in file\n",
      "\n",
      "Seq is: CAGTAATCCTCCCGAC\n",
      "0 CAGTAATCCTCCCGAC motifs detected in file\n",
      "\n",
      "Seq is: TCTCCCGGGGACAATC\n",
      "0 TCTCCCGGGGACAATC motifs detected in file\n",
      "\n",
      "Seq is: CCCAGTACCCGACCCA\n",
      "0 CCCAGTACCCGACCCA motifs detected in file\n",
      "\n",
      "Seq is: AAATGATATACTATCC\n",
      "0 AAATGATATACTATCC motifs detected in file\n",
      "\n",
      "Seq is: GTACACTAACCACCCA\n",
      "0 GTACACTAACCACCCA motifs detected in file\n",
      "\n",
      "Seq is: GTAATACACAACAGCA\n",
      "0 GTAATACACAACAGCA motifs detected in file\n",
      "\n",
      "Seq is: CGTCCATCTCCGAATG\n",
      "0 CGTCCATCTCCGAATG motifs detected in file\n",
      "\n",
      "Seq is: ATCTTCACACTTTTGC\n",
      "0 ATCTTCACACTTTTGC motifs detected in file\n",
      "\n",
      "Seq is: ATCCGAACCTAATTCC\n",
      "0 ATCCGAACCTAATTCC motifs detected in file\n",
      "\n",
      "Seq is: GCTGTCGCTAAAGATC\n",
      "0 GCTGTCGCTAAAGATC motifs detected in file\n",
      "\n",
      "Seq is: CTATCAGTCATCTTAA\n",
      "0 CTATCAGTCATCTTAA motifs detected in file\n",
      "\n",
      "Seq is: TCTTAACACTTCTACT\n",
      "0 TCTTAACACTTCTACT motifs detected in file\n",
      "\n",
      "Seq is: ATTCACATCTCCCTTC\n",
      "0 ATTCACATCTCCCTTC motifs detected in file\n",
      "\n",
      "Seq is: CAATTACCAACTATAA\n",
      "0 CAATTACCAACTATAA motifs detected in file\n",
      "\n",
      "Seq is: CGATCCCACCGCGAAG\n",
      "0 CGATCCCACCGCGAAG motifs detected in file\n",
      "\n",
      "Seq is: GTTAATCCACGTACTC\n",
      "0 GTTAATCCACGTACTC motifs detected in file\n",
      "\n",
      "Seq is: ATCCACCGGACGTCAT\n",
      "0 ATCCACCGGACGTCAT motifs detected in file\n",
      "\n",
      "Seq is: TACAATCGCCAATCCG\n",
      "0 TACAATCGCCAATCCG motifs detected in file\n",
      "\n",
      "Seq is: CATGCCGGTCCCGATA\n",
      "0 CATGCCGGTCCCGATA motifs detected in file\n",
      "\n",
      "Seq is: ACCAACTTACTCATCC\n",
      "0 ACCAACTTACTCATCC motifs detected in file\n",
      "\n",
      "Seq is: GACTACCAAAGCTATT\n",
      "0 GACTACCAAAGCTATT motifs detected in file\n",
      "\n",
      "Seq is: ATTGATCATACCCCTA\n",
      "0 ATTGATCATACCCCTA motifs detected in file\n",
      "\n",
      "Seq is: TTCCCCACATCCACAC\n",
      "0 TTCCCCACATCCACAC motifs detected in file\n",
      "\n",
      "Seq is: GGAACACCATTCTTCA\n",
      "0 GGAACACCATTCTTCA motifs detected in file\n",
      "\n",
      "Seq is: CCCGATCCCGTAAAAA\n",
      "0 CCCGATCCCGTAAAAA motifs detected in file\n",
      "\n",
      "Seq is: TTAGAATCTCGCTTTC\n",
      "0 TTAGAATCTCGCTTTC motifs detected in file\n",
      "\n",
      "Seq is: GGATATTAGTGACACG\n",
      "0 GGATATTAGTGACACG motifs detected in file\n",
      "\n",
      "Seq is: GGCGAGTATTCCCCTA\n",
      "0 GGCGAGTATTCCCCTA motifs detected in file\n",
      "\n",
      "Seq is: CCGGTACCGATCCAAA\n",
      "0 CCGGTACCGATCCAAA motifs detected in file\n",
      "\n",
      "Seq is: TATCTATCAACCCACC\n",
      "0 TATCTATCAACCCACC motifs detected in file\n",
      "\n",
      "Seq is: AACCACATTTACTCAC\n",
      "0 AACCACATTTACTCAC motifs detected in file\n",
      "\n",
      "Seq is: TCCGTCCTCAGGCTCC\n",
      "0 TCCGTCCTCAGGCTCC motifs detected in file\n",
      "\n",
      "Seq is: TCCAATCAACACCGTT\n",
      "0 TCCAATCAACACCGTT motifs detected in file\n",
      "\n",
      "Seq is: ATTTTTACTAATTCCT\n",
      "0 ATTTTTACTAATTCCT motifs detected in file\n",
      "\n",
      "Seq is: ATGATCATATTCTCTA\n",
      "0 ATGATCATATTCTCTA motifs detected in file\n",
      "\n",
      "Seq is: GTCCAAGCAGGAAATC\n",
      "0 GTCCAAGCAGGAAATC motifs detected in file\n",
      "\n",
      "Seq is: ACAACACATACCAATA\n",
      "0 ACAACACATACCAATA motifs detected in file\n",
      "\n",
      "Seq is: GTAAACAAAACTCCAA\n",
      "0 GTAAACAAAACTCCAA motifs detected in file\n",
      "\n",
      "Seq is: TCAAACTAATCAGTAA\n",
      "0 TCAAACTAATCAGTAA motifs detected in file\n",
      "\n",
      "Seq is: CAACCCACCACTAATC\n",
      "0 CAACCCACCACTAATC motifs detected in file\n",
      "\n",
      "Seq is: AGCCAACAACTACTCT\n",
      "0 AGCCAACAACTACTCT motifs detected in file\n",
      "\n",
      "Seq is: ATAAACCTCCACCCCT\n",
      "0 ATAAACCTCCACCCCT motifs detected in file\n",
      "\n",
      "Seq is: AGTTATCTCCCCCTTT\n",
      "0 AGTTATCTCCCCCTTT motifs detected in file\n",
      "\n",
      "Seq is: ATCTTACTAGTCATGA\n",
      "0 ATCTTACTAGTCATGA motifs detected in file\n",
      "\n",
      "Seq is: CGCCGCCCATCCGGAT\n",
      "0 CGCCGCCCATCCGGAT motifs detected in file\n",
      "\n",
      "Seq is: CCTAACACCCCCGTTA\n",
      "0 CCTAACACCCCCGTTA motifs detected in file\n",
      "\n",
      "Seq is: CTTACTTCGCTTGTCT\n",
      "0 CTTACTTCGCTTGTCT motifs detected in file\n",
      "\n",
      "Seq is: AAAAGCAAGTAAAACC\n",
      "0 AAAAGCAAGTAAAACC motifs detected in file\n",
      "\n",
      "Seq is: AAGAGTCCCAAACAAT\n",
      "0 AAGAGTCCCAAACAAT motifs detected in file\n",
      "\n",
      "Seq is: ATTAAAAAATAAAAAT\n",
      "0 ATTAAAAAATAAAAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCCGAAACCACTCCC\n",
      "0 ACCCGAAACCACTCCC motifs detected in file\n",
      "\n",
      "Seq is: TCCATCCACCATCAAA\n",
      "0 TCCATCCACCATCAAA motifs detected in file\n",
      "\n",
      "Seq is: CACAAATTATAAAAAT\n",
      "0 CACAAATTATAAAAAT motifs detected in file\n",
      "\n",
      "Seq is: CAACCCACTACTCCTT\n",
      "0 CAACCCACTACTCCTT motifs detected in file\n",
      "\n",
      "Seq is: ATCACATTCCTCATAC\n",
      "0 ATCACATTCCTCATAC motifs detected in file\n",
      "\n",
      "Seq is: CCCCATTAATACCACC\n",
      "0 CCCCATTAATACCACC motifs detected in file\n",
      "\n",
      "Seq is: CGGTATGCCATCATAA\n",
      "0 CGGTATGCCATCATAA motifs detected in file\n",
      "\n",
      "Seq is: ATCAACCCCGTAAAGT\n",
      "0 ATCAACCCCGTAAAGT motifs detected in file\n",
      "\n",
      "Seq is: GACATTTCTCCACAAC\n",
      "0 GACATTTCTCCACAAC motifs detected in file\n",
      "\n",
      "Seq is: CAAGAGAATTAGAAAT\n",
      "0 CAAGAGAATTAGAAAT motifs detected in file\n",
      "\n",
      "Seq is: TGAATTTTCCAATTAC\n",
      "0 TGAATTTTCCAATTAC motifs detected in file\n",
      "\n",
      "Seq is: GATTCAATCGAAATGT\n",
      "0 GATTCAATCGAAATGT motifs detected in file\n",
      "\n",
      "Seq is: CACCCGAGATCTCCCC\n",
      "0 CACCCGAGATCTCCCC motifs detected in file\n",
      "\n",
      "Seq is: CATCTCCCCCACGATA\n",
      "0 CATCTCCCCCACGATA motifs detected in file\n",
      "\n",
      "Seq is: TCCCATTTATGCCCGG\n",
      "0 TCCCATTTATGCCCGG motifs detected in file\n",
      "\n",
      "Seq is: GATAAAGTTTCTAAAA\n",
      "0 GATAAAGTTTCTAAAA motifs detected in file\n",
      "\n",
      "Seq is: ATCCCCTTAATTAAGA\n",
      "0 ATCCCCTTAATTAAGA motifs detected in file\n",
      "\n",
      "Seq is: TCCTGCTACCCGTACT\n",
      "0 TCCTGCTACCCGTACT motifs detected in file\n",
      "\n",
      "Seq is: TTGACACCAAACGAGG\n",
      "0 TTGACACCAAACGAGG motifs detected in file\n",
      "\n",
      "Seq is: TGTCTAAAGTCTCGTA\n",
      "0 TGTCTAAAGTCTCGTA motifs detected in file\n",
      "\n",
      "Seq is: ACTGGCATCATATGGG\n",
      "0 ACTGGCATCATATGGG motifs detected in file\n",
      "\n",
      "Seq is: GGCCAGCACATCGCCA\n",
      "0 GGCCAGCACATCGCCA motifs detected in file\n",
      "\n",
      "Seq is: CTTTCACCAAACCAAA\n",
      "0 CTTTCACCAAACCAAA motifs detected in file\n",
      "\n",
      "Seq is: TCTGCAGGCCCTTAAC\n",
      "0 TCTGCAGGCCCTTAAC motifs detected in file\n",
      "\n",
      "Seq is: GTAGACAAAAATTCGG\n",
      "0 GTAGACAAAAATTCGG motifs detected in file\n",
      "\n",
      "Seq is: ATTACACCCCTGTCGA\n",
      "0 ATTACACCCCTGTCGA motifs detected in file\n",
      "\n",
      "Seq is: GTACTCACGTTCTGCA\n",
      "0 GTACTCACGTTCTGCA motifs detected in file\n",
      "\n",
      "Seq is: GAACACTACATTAATA\n",
      "0 GAACACTACATTAATA motifs detected in file\n",
      "\n",
      "Seq is: ATATGCTTACCCCCGA\n",
      "0 ATATGCTTACCCCCGA motifs detected in file\n",
      "\n",
      "Seq is: AGTTAGCCCTCATGAT\n",
      "0 AGTTAGCCCTCATGAT motifs detected in file\n",
      "\n",
      "Seq is: GAGCTAACAGTTGCAG\n",
      "0 GAGCTAACAGTTGCAG motifs detected in file\n",
      "\n",
      "Seq is: TTCCTCAGCGTCCAGC\n",
      "0 TTCCTCAGCGTCCAGC motifs detected in file\n",
      "\n",
      "Seq is: GGACCAAGATCCTTAT\n",
      "0 GGACCAAGATCCTTAT motifs detected in file\n",
      "\n",
      "Seq is: TACCCCCTTCATACAT\n",
      "0 TACCCCCTTCATACAT motifs detected in file\n",
      "\n",
      "Seq is: CCTCAACTTCTAATCA\n",
      "0 CCTCAACTTCTAATCA motifs detected in file\n",
      "\n",
      "Seq is: GGCGGCATCAGCCACA\n",
      "0 GGCGGCATCAGCCACA motifs detected in file\n",
      "\n",
      "Seq is: TGAGCATCCAAGTCAT\n",
      "0 TGAGCATCCAAGTCAT motifs detected in file\n",
      "\n",
      "Seq is: GGTAACCACCGGTTAA\n",
      "0 GGTAACCACCGGTTAA motifs detected in file\n",
      "\n",
      "Seq is: CTAAACTACCATTCCC\n",
      "0 CTAAACTACCATTCCC motifs detected in file\n",
      "\n",
      "Seq is: TCCCTAGCAACTGGAG\n",
      "0 TCCCTAGCAACTGGAG motifs detected in file\n",
      "\n",
      "Seq is: CAGACAATACATTTCC\n",
      "0 CAGACAATACATTTCC motifs detected in file\n",
      "\n",
      "Seq is: TACGCATCCCTACCCT\n",
      "0 TACGCATCCCTACCCT motifs detected in file\n",
      "\n",
      "Seq is: CCCGCTACTCCTGATC\n",
      "0 CCCGCTACTCCTGATC motifs detected in file\n",
      "\n",
      "Seq is: CTACTACCGCCCGTCC\n",
      "0 CTACTACCGCCCGTCC motifs detected in file\n",
      "\n",
      "Seq is: TATCAAGTCTGCCAAA\n",
      "0 TATCAAGTCTGCCAAA motifs detected in file\n",
      "\n",
      "Seq is: GATAAAGGACTCGCAG\n",
      "0 GATAAAGGACTCGCAG motifs detected in file\n",
      "\n",
      "Seq is: ACTCTACAATCATAAT\n",
      "0 ACTCTACAATCATAAT motifs detected in file\n",
      "\n",
      "Seq is: CAATCAAAACAACACT\n",
      "0 CAATCAAAACAACACT motifs detected in file\n",
      "\n",
      "Seq is: CACAATCCTTTTCCCG\n",
      "0 CACAATCCTTTTCCCG motifs detected in file\n",
      "\n",
      "Seq is: TATGTCCTGATCCTTC\n",
      "0 TATGTCCTGATCCTTC motifs detected in file\n",
      "\n",
      "Seq is: GTCTATTCCTTTTACT\n",
      "0 GTCTATTCCTTTTACT motifs detected in file\n",
      "\n",
      "Seq is: AATACTCCCGCACTAG\n",
      "0 AATACTCCCGCACTAG motifs detected in file\n",
      "\n",
      "Seq is: TACTTTCCACTATTCG\n",
      "0 TACTTTCCACTATTCG motifs detected in file\n",
      "\n",
      "Seq is: AGAAGATACTATCCCC\n",
      "0 AGAAGATACTATCCCC motifs detected in file\n",
      "\n",
      "Seq is: TTCATCGCAGAACATC\n",
      "0 TTCATCGCAGAACATC motifs detected in file\n",
      "\n",
      "Seq is: TGGATGAACGCGACGT\n",
      "0 TGGATGAACGCGACGT motifs detected in file\n",
      "\n",
      "Seq is: ACACGGTAATTCTATA\n",
      "0 ACACGGTAATTCTATA motifs detected in file\n",
      "\n",
      "Seq is: TATCCTAAATCGGATA\n",
      "0 TATCCTAAATCGGATA motifs detected in file\n",
      "\n",
      "Seq is: TTAAATTGAAACTGTT\n",
      "0 TTAAATTGAAACTGTT motifs detected in file\n",
      "\n",
      "Seq is: TCAATACTTTACCAAA\n",
      "0 TCAATACTTTACCAAA motifs detected in file\n",
      "\n",
      "Seq is: AAGATTATGGAAAAAC\n",
      "0 AAGATTATGGAAAAAC motifs detected in file\n",
      "\n",
      "Seq is: TTTTTCAGTACCTCCG\n",
      "0 TTTTTCAGTACCTCCG motifs detected in file\n",
      "\n",
      "Seq is: GTACCCCAACCGAACT\n",
      "0 GTACCCCAACCGAACT motifs detected in file\n",
      "\n",
      "Seq is: TTGGTCGAAACCCCCC\n",
      "0 TTGGTCGAAACCCCCC motifs detected in file\n",
      "\n",
      "Seq is: AAGCTAAGGAAGGAAA\n",
      "0 AAGCTAAGGAAGGAAA motifs detected in file\n",
      "\n",
      "Seq is: AAACAGATCTAAAATA\n",
      "0 AAACAGATCTAAAATA motifs detected in file\n",
      "\n",
      "Seq is: CATATGCCCAAGAAGC\n",
      "0 CATATGCCCAAGAAGC motifs detected in file\n",
      "\n",
      "Seq is: TTACACACCATCCCAC\n",
      "0 TTACACACCATCCCAC motifs detected in file\n",
      "\n",
      "Seq is: CCCCCGTAGGTATATC\n",
      "0 CCCCCGTAGGTATATC motifs detected in file\n",
      "\n",
      "Seq is: AGTGAGGTCCAAATAC\n",
      "0 AGTGAGGTCCAAATAC motifs detected in file\n",
      "\n",
      "Seq is: CTCCCAAGAAGATGAT\n",
      "0 CTCCCAAGAAGATGAT motifs detected in file\n",
      "\n",
      "Seq is: AATCCTGAAAAGCGAA\n",
      "0 AATCCTGAAAAGCGAA motifs detected in file\n",
      "\n",
      "Seq is: GCTTCGCTTCCGAAGA\n",
      "0 GCTTCGCTTCCGAAGA motifs detected in file\n",
      "\n",
      "Seq is: GTTTAGTACTTTTAAT\n",
      "0 GTTTAGTACTTTTAAT motifs detected in file\n",
      "\n",
      "Seq is: AATTATAATGACGACA\n",
      "0 AATTATAATGACGACA motifs detected in file\n",
      "\n",
      "Seq is: ATCTCTTTACTTACTC\n",
      "0 ATCTCTTTACTTACTC motifs detected in file\n",
      "\n",
      "Seq is: GGATGTCTACTATCCT\n",
      "0 GGATGTCTACTATCCT motifs detected in file\n",
      "\n",
      "Seq is: ACAAAGTACACCACAT\n",
      "0 ACAAAGTACACCACAT motifs detected in file\n",
      "\n",
      "Seq is: ACGTAGCCCCATCAGG\n",
      "0 ACGTAGCCCCATCAGG motifs detected in file\n",
      "\n",
      "Seq is: CTCATTGCAGCCCCGT\n",
      "0 CTCATTGCAGCCCCGT motifs detected in file\n",
      "\n",
      "Seq is: AAGGCCTATCCGGCTC\n",
      "0 AAGGCCTATCCGGCTC motifs detected in file\n",
      "\n",
      "Seq is: TTTCACAACACCCTTC\n",
      "0 TTTCACAACACCCTTC motifs detected in file\n",
      "\n",
      "Seq is: TTTCAATCTAAATGCG\n",
      "0 TTTCAATCTAAATGCG motifs detected in file\n",
      "\n",
      "Seq is: GCTACTTATACTATCA\n",
      "0 GCTACTTATACTATCA motifs detected in file\n",
      "\n",
      "Seq is: CTTTACCTAAAACCGC\n",
      "0 CTTTACCTAAAACCGC motifs detected in file\n",
      "\n",
      "Seq is: ATCCAAACCAGATCCC\n",
      "0 ATCCAAACCAGATCCC motifs detected in file\n",
      "\n",
      "Seq is: ACATCGGACTCTTATC\n",
      "0 ACATCGGACTCTTATC motifs detected in file\n",
      "\n",
      "Seq is: CCCTTTAGTCACTTAC\n",
      "0 CCCTTTAGTCACTTAC motifs detected in file\n",
      "\n",
      "Seq is: TTCGGAGGATACTACC\n",
      "0 TTCGGAGGATACTACC motifs detected in file\n",
      "\n",
      "Seq is: GTTTATGTCGAGGATA\n",
      "0 GTTTATGTCGAGGATA motifs detected in file\n",
      "\n",
      "Seq is: ACTATCCTGTATCACC\n",
      "0 ACTATCCTGTATCACC motifs detected in file\n",
      "\n",
      "Seq is: TTCACCCGATGTTGCA\n",
      "0 TTCACCCGATGTTGCA motifs detected in file\n",
      "\n",
      "Seq is: TCATGCACAGCACATC\n",
      "0 TCATGCACAGCACATC motifs detected in file\n",
      "\n",
      "Seq is: ATGCTACAGATCACTA\n",
      "0 ATGCTACAGATCACTA motifs detected in file\n",
      "\n",
      "Seq is: CTCAAACGATTATGTT\n",
      "0 CTCAAACGATTATGTT motifs detected in file\n",
      "\n",
      "Seq is: ACGGTCAGTCCGGTGG\n",
      "0 ACGGTCAGTCCGGTGG motifs detected in file\n",
      "\n",
      "Seq is: ATGCCACTTTTCTTTC\n",
      "0 ATGCCACTTTTCTTTC motifs detected in file\n",
      "\n",
      "Seq is: TACTTAGGATCCCGAG\n",
      "0 TACTTAGGATCCCGAG motifs detected in file\n",
      "\n",
      "Seq is: TCTCGTGACGTCGTCT\n",
      "0 TCTCGTGACGTCGTCT motifs detected in file\n",
      "\n",
      "Seq is: AGTAAAGTATAACGAA\n",
      "0 AGTAAAGTATAACGAA motifs detected in file\n",
      "\n",
      "Seq is: GGCACCTCACACCGCT\n",
      "0 GGCACCTCACACCGCT motifs detected in file\n",
      "\n",
      "Seq is: CAGTCGATTTATGAAA\n",
      "0 CAGTCGATTTATGAAA motifs detected in file\n",
      "\n",
      "Seq is: CTGCGGCCACCCCTAT\n",
      "0 CTGCGGCCACCCCTAT motifs detected in file\n",
      "\n",
      "Seq is: CGCTACTAATACCATC\n",
      "0 CGCTACTAATACCATC motifs detected in file\n",
      "\n",
      "Seq is: GCCTCCACTTTATACG\n",
      "0 GCCTCCACTTTATACG motifs detected in file\n",
      "\n",
      "Seq is: AGCACACAGATGCAGA\n",
      "0 AGCACACAGATGCAGA motifs detected in file\n",
      "\n",
      "Seq is: TAAGGATGACATTTAT\n",
      "0 TAAGGATGACATTTAT motifs detected in file\n",
      "\n",
      "Seq is: ATACCACACGAGTGTG\n",
      "0 ATACCACACGAGTGTG motifs detected in file\n",
      "\n",
      "Seq is: CTATTTGGGATCCTTC\n",
      "0 CTATTTGGGATCCTTC motifs detected in file\n",
      "\n",
      "Seq is: GGTTAACTATAGTATC\n",
      "0 GGTTAACTATAGTATC motifs detected in file\n",
      "\n",
      "Seq is: GAGTCTGGTATGGATA\n",
      "0 GAGTCTGGTATGGATA motifs detected in file\n",
      "\n",
      "Seq is: CATAGGTTTACCAAGA\n",
      "0 CATAGGTTTACCAAGA motifs detected in file\n",
      "\n",
      "Seq is: AAAAAAGGCCTCATCA\n",
      "0 AAAAAAGGCCTCATCA motifs detected in file\n",
      "\n",
      "Seq is: TTCAATGTTATACCCT\n",
      "0 TTCAATGTTATACCCT motifs detected in file\n",
      "\n",
      "Seq is: CTAATAAGGCTTCGGA\n",
      "0 CTAATAAGGCTTCGGA motifs detected in file\n",
      "\n",
      "Seq is: CCATCCTATTCTTATA\n",
      "0 CCATCCTATTCTTATA motifs detected in file\n",
      "\n",
      "Seq is: TTTGAAGTCCCACTGG\n",
      "0 TTTGAAGTCCCACTGG motifs detected in file\n",
      "\n",
      "Seq is: CTTTTTAGTCTTATCC\n",
      "0 CTTTTTAGTCTTATCC motifs detected in file\n",
      "\n",
      "Seq is: ACAACCATATCTTACC\n",
      "0 ACAACCATATCTTACC motifs detected in file\n",
      "\n",
      "Seq is: CCAGAGGAGGAGGCCG\n",
      "0 CCAGAGGAGGAGGCCG motifs detected in file\n",
      "\n",
      "Seq is: CCCCAGCTTCTTCCAC\n",
      "0 CCCCAGCTTCTTCCAC motifs detected in file\n",
      "\n",
      "Seq is: CACAACTCCAACGCCA\n",
      "0 CACAACTCCAACGCCA motifs detected in file\n",
      "\n",
      "Seq is: GACAATCATCATATCA\n",
      "0 GACAATCATCATATCA motifs detected in file\n",
      "\n",
      "Seq is: CTACGGTCGTCGATCC\n",
      "0 CTACGGTCGTCGATCC motifs detected in file\n",
      "\n",
      "Seq is: TGACCCTCTCTATCCT\n",
      "0 TGACCCTCTCTATCCT motifs detected in file\n",
      "\n",
      "Seq is: CTTCATCCTGACGACA\n",
      "0 CTTCATCCTGACGACA motifs detected in file\n",
      "\n",
      "Seq is: CTAATCCTGCCTAGTC\n",
      "0 CTAATCCTGCCTAGTC motifs detected in file\n",
      "\n",
      "Seq is: TGCTTCTCCCCGCAAA\n",
      "0 TGCTTCTCCCCGCAAA motifs detected in file\n",
      "\n",
      "Seq is: CCATAACCCAATACCG\n",
      "0 CCATAACCCAATACCG motifs detected in file\n",
      "\n",
      "Seq is: TCCCCCAAAATATAAC\n",
      "0 TCCCCCAAAATATAAC motifs detected in file\n",
      "\n",
      "Seq is: TATTAACAACACTACT\n",
      "0 TATTAACAACACTACT motifs detected in file\n",
      "\n",
      "Seq is: TTAACTACCGGGGATT\n",
      "0 TTAACTACCGGGGATT motifs detected in file\n",
      "\n",
      "Seq is: CTATCTCCTCTGTGGA\n",
      "0 CTATCTCCTCTGTGGA motifs detected in file\n",
      "\n",
      "Seq is: CGATCCACATTCCTCT\n",
      "0 CGATCCACATTCCTCT motifs detected in file\n",
      "\n",
      "Seq is: TCTACCAATCCAGATG\n",
      "0 TCTACCAATCCAGATG motifs detected in file\n",
      "\n",
      "Seq is: CCTCCGAGGTCCCACC\n",
      "0 CCTCCGAGGTCCCACC motifs detected in file\n",
      "\n",
      "Seq is: CTCATCAAGGGACGAT\n",
      "0 CTCATCAAGGGACGAT motifs detected in file\n",
      "\n",
      "Seq is: TCCCTCCCATCTGACA\n",
      "0 TCCCTCCCATCTGACA motifs detected in file\n",
      "\n",
      "Seq is: AGTTGGACAAGGACGA\n",
      "0 AGTTGGACAAGGACGA motifs detected in file\n",
      "\n",
      "Seq is: GGCCGGAGGTGATATC\n",
      "0 GGCCGGAGGTGATATC motifs detected in file\n",
      "\n",
      "Seq is: GTCTACGCTCGTCCTC\n",
      "0 GTCTACGCTCGTCCTC motifs detected in file\n",
      "\n",
      "Seq is: ATCCTCTCTCCCTCCA\n",
      "0 ATCCTCTCTCCCTCCA motifs detected in file\n",
      "\n",
      "Seq is: TTCTCAGTCTTCGAAT\n",
      "0 TTCTCAGTCTTCGAAT motifs detected in file\n",
      "\n",
      "Seq is: ACAACAAATACGCCCC\n",
      "0 ACAACAAATACGCCCC motifs detected in file\n",
      "\n",
      "Seq is: TTTGCGGTATAGCCCC\n",
      "0 TTTGCGGTATAGCCCC motifs detected in file\n",
      "\n",
      "Seq is: CGTGGCTAATAATCCC\n",
      "0 CGTGGCTAATAATCCC motifs detected in file\n",
      "\n",
      "Seq is: CTTGATCAGTCAACGG\n",
      "0 CTTGATCAGTCAACGG motifs detected in file\n",
      "\n",
      "Seq is: TTTCGGAGGCTAATAC\n",
      "0 TTTCGGAGGCTAATAC motifs detected in file\n",
      "\n",
      "Seq is: TACCCATAATGATACT\n",
      "0 TACCCATAATGATACT motifs detected in file\n",
      "\n",
      "Seq is: TCTTCTAATAATGTTT\n",
      "0 TCTTCTAATAATGTTT motifs detected in file\n",
      "\n",
      "Seq is: ACGTACCCGACACTGC\n",
      "0 ACGTACCCGACACTGC motifs detected in file\n",
      "\n",
      "Seq is: TATTGCAACATCTACA\n",
      "0 TATTGCAACATCTACA motifs detected in file\n",
      "\n",
      "Seq is: CCAGCAATGGATCTTC\n",
      "0 CCAGCAATGGATCTTC motifs detected in file\n",
      "\n",
      "Seq is: CAACGGACCGACCGGG\n",
      "0 CAACGGACCGACCGGG motifs detected in file\n",
      "\n",
      "Seq is: TCGAGCCGAGCTTATT\n",
      "0 TCGAGCCGAGCTTATT motifs detected in file\n",
      "\n",
      "Seq is: CCTCCGAATCTCGACA\n",
      "0 CCTCCGAATCTCGACA motifs detected in file\n",
      "\n",
      "Seq is: CGGATCCTGAGGTCGA\n",
      "0 CGGATCCTGAGGTCGA motifs detected in file\n",
      "\n",
      "Seq is: GTACGCGGCTTATTAT\n",
      "0 GTACGCGGCTTATTAT motifs detected in file\n",
      "\n",
      "Seq is: CCATATCACAAGGTTA\n",
      "0 CCATATCACAAGGTTA motifs detected in file\n",
      "\n",
      "Seq is: CAGAAACACCAAACAT\n",
      "0 CAGAAACACCAAACAT motifs detected in file\n",
      "\n",
      "Seq is: CTCTTATCAGTTGCCA\n",
      "0 CTCTTATCAGTTGCCA motifs detected in file\n",
      "\n",
      "Seq is: GCCGCTTGTAGTCACC\n",
      "0 GCCGCTTGTAGTCACC motifs detected in file\n",
      "\n",
      "Seq is: CCCACTCCATTTTACC\n",
      "0 CCCACTCCATTTTACC motifs detected in file\n",
      "\n",
      "Seq is: GACTCACTTCGTAACC\n",
      "0 GACTCACTTCGTAACC motifs detected in file\n",
      "\n",
      "Seq is: TGACATTTAGATTTCT\n",
      "0 TGACATTTAGATTTCT motifs detected in file\n",
      "\n",
      "Seq is: GTCCCCAATCCGGAGA\n",
      "0 GTCCCCAATCCGGAGA motifs detected in file\n",
      "\n",
      "Seq is: AAAATGGTCGAGGCTC\n",
      "0 AAAATGGTCGAGGCTC motifs detected in file\n",
      "\n",
      "Seq is: CACTAAAAGTATAACT\n",
      "0 CACTAAAAGTATAACT motifs detected in file\n",
      "\n",
      "Seq is: TAACGTTTAAGCTTCT\n",
      "0 TAACGTTTAAGCTTCT motifs detected in file\n",
      "\n",
      "Seq is: TCGTCGAAGTTTGGAC\n",
      "0 TCGTCGAAGTTTGGAC motifs detected in file\n",
      "\n",
      "Seq is: GGCCCCGAAGAGGGCG\n",
      "0 GGCCCCGAAGAGGGCG motifs detected in file\n",
      "\n",
      "Seq is: GAAAAAAGGGCCGCCG\n",
      "0 GAAAAAAGGGCCGCCG motifs detected in file\n",
      "\n",
      "Seq is: CCCTCTTCATCTAACT\n",
      "0 CCCTCTTCATCTAACT motifs detected in file\n",
      "\n",
      "Seq is: TCGGTCAACTAATCCC\n",
      "0 TCGGTCAACTAATCCC motifs detected in file\n",
      "\n",
      "Seq is: ACGAATCGACAATTGA\n",
      "0 ACGAATCGACAATTGA motifs detected in file\n",
      "\n",
      "Seq is: TTCACAAACACCCAAA\n",
      "0 TTCACAAACACCCAAA motifs detected in file\n",
      "\n",
      "Seq is: TTCAGGGTAACCAGAT\n",
      "0 TTCAGGGTAACCAGAT motifs detected in file\n",
      "\n",
      "Seq is: CATTCCCGAATCGAAT\n",
      "0 CATTCCCGAATCGAAT motifs detected in file\n",
      "\n",
      "Seq is: TAATTTCACCGACTAA\n",
      "0 TAATTTCACCGACTAA motifs detected in file\n",
      "\n",
      "Seq is: ACGCAAGTCAACTACG\n",
      "0 ACGCAAGTCAACTACG motifs detected in file\n",
      "\n",
      "Seq is: TCTCACCCCAAAACGT\n",
      "0 TCTCACCCCAAAACGT motifs detected in file\n",
      "\n",
      "Seq is: CAGGAATCGACAATGT\n",
      "0 CAGGAATCGACAATGT motifs detected in file\n",
      "\n",
      "Seq is: CTTTAATTCATAACGT\n",
      "0 CTTTAATTCATAACGT motifs detected in file\n",
      "\n",
      "Seq is: TGAATGACTCCCGAAA\n",
      "0 TGAATGACTCCCGAAA motifs detected in file\n",
      "\n",
      "Seq is: CTTCCGAGAACCAGAC\n",
      "0 CTTCCGAGAACCAGAC motifs detected in file\n",
      "\n",
      "Seq is: ATAAATTGGATTTAAA\n",
      "0 ATAAATTGGATTTAAA motifs detected in file\n",
      "\n",
      "Seq is: GATATTCTAATAATCA\n",
      "0 GATATTCTAATAATCA motifs detected in file\n",
      "\n",
      "Seq is: TATTTTCCCCTCTATC\n",
      "0 TATTTTCCCCTCTATC motifs detected in file\n",
      "\n",
      "Seq is: CATCCTCATCGCACCA\n",
      "0 CATCCTCATCGCACCA motifs detected in file\n",
      "\n",
      "Seq is: TTCCCGCTACTCACAC\n",
      "0 TTCCCGCTACTCACAC motifs detected in file\n",
      "\n",
      "Seq is: CCCTCCTTACCCCACC\n",
      "0 CCCTCCTTACCCCACC motifs detected in file\n",
      "\n",
      "Seq is: CAAAACATACAAGTTT\n",
      "0 CAAAACATACAAGTTT motifs detected in file\n",
      "\n",
      "Seq is: GACAGTAAAATAAAAA\n",
      "0 GACAGTAAAATAAAAA motifs detected in file\n",
      "\n",
      "Seq is: TGCAACAATCTATACC\n",
      "0 TGCAACAATCTATACC motifs detected in file\n",
      "\n",
      "Seq is: CCTCATCACACTAACT\n",
      "0 CCTCATCACACTAACT motifs detected in file\n",
      "\n",
      "Seq is: CCACCTCATCTAATCC\n",
      "0 CCACCTCATCTAATCC motifs detected in file\n",
      "\n",
      "Seq is: GCATCCATCTTCATCT\n",
      "0 GCATCCATCTTCATCT motifs detected in file\n",
      "\n",
      "Seq is: CCAATTCCTCCCACTA\n",
      "0 CCAATTCCTCCCACTA motifs detected in file\n",
      "\n",
      "Seq is: CCACCGATACTACCAC\n",
      "0 CCACCGATACTACCAC motifs detected in file\n",
      "\n",
      "Seq is: CCCTACTACTCCGATA\n",
      "0 CCCTACTACTCCGATA motifs detected in file\n",
      "\n",
      "Seq is: ACAAAAAACACTTAAG\n",
      "0 ACAAAAAACACTTAAG motifs detected in file\n",
      "\n",
      "Seq is: AAGCTATTACCGGGTA\n",
      "0 AAGCTATTACCGGGTA motifs detected in file\n",
      "\n",
      "Seq is: AACCCGTTTTTCGGCC\n",
      "0 AACCCGTTTTTCGGCC motifs detected in file\n",
      "\n",
      "Seq is: AATCGCCCCCGTCCGG\n",
      "0 AATCGCCCCCGTCCGG motifs detected in file\n",
      "\n",
      "Seq is: AGGATCCCTCTCCTCC\n",
      "0 AGGATCCCTCTCCTCC motifs detected in file\n",
      "\n",
      "Seq is: CACCTACCTTAATTCC\n",
      "0 CACCTACCTTAATTCC motifs detected in file\n",
      "\n",
      "Seq is: CACAATCAGTACAATC\n",
      "0 CACAATCAGTACAATC motifs detected in file\n",
      "\n",
      "Seq is: GAACAAAGTCCACGCT\n",
      "0 GAACAAAGTCCACGCT motifs detected in file\n",
      "\n",
      "Seq is: CTATCATCATCCCAGC\n",
      "0 CTATCATCATCCCAGC motifs detected in file\n",
      "\n",
      "Seq is: ACCACGACCTCAAATT\n",
      "0 ACCACGACCTCAAATT motifs detected in file\n",
      "\n",
      "Seq is: CAACTCATCATCCTTA\n",
      "0 CAACTCATCATCCTTA motifs detected in file\n",
      "\n",
      "Seq is: CGCCATCATCAATCCT\n",
      "0 CGCCATCATCAATCCT motifs detected in file\n",
      "\n",
      "Seq is: ATTATATTTATCAATT\n",
      "0 ATTATATTTATCAATT motifs detected in file\n",
      "\n",
      "Seq is: TAATTCTTACCAATAC\n",
      "0 TAATTCTTACCAATAC motifs detected in file\n",
      "\n",
      "Seq is: AATCCCAACATGCCAT\n",
      "0 AATCCCAACATGCCAT motifs detected in file\n",
      "\n",
      "Seq is: CTTGACGATAATAAGT\n",
      "0 CTTGACGATAATAAGT motifs detected in file\n",
      "\n",
      "Seq is: AGGATACACCCATTAA\n",
      "0 AGGATACACCCATTAA motifs detected in file\n",
      "\n",
      "Seq is: CTCCTCATACGATTCT\n",
      "0 CTCCTCATACGATTCT motifs detected in file\n",
      "\n",
      "Seq is: AAAACGCATCGACCCA\n",
      "0 AAAACGCATCGACCCA motifs detected in file\n",
      "\n",
      "Seq is: AACCAAATTAGGTGGA\n",
      "0 AACCAAATTAGGTGGA motifs detected in file\n",
      "\n",
      "Seq is: GTTGACGGACGATACT\n",
      "0 GTTGACGGACGATACT motifs detected in file\n",
      "\n",
      "Seq is: ACCTATTCTAACTCTC\n",
      "0 ACCTATTCTAACTCTC motifs detected in file\n",
      "\n",
      "Seq is: TCACTCCTCTTCCGAA\n",
      "0 TCACTCCTCTTCCGAA motifs detected in file\n",
      "\n",
      "Seq is: TGCAAATCACTCCCTC\n",
      "0 TGCAAATCACTCCCTC motifs detected in file\n",
      "\n",
      "Seq is: TCTAAACCATATACTA\n",
      "0 TCTAAACCATATACTA motifs detected in file\n",
      "\n",
      "Seq is: ACTCTACCCCCGATCA\n",
      "0 ACTCTACCCCCGATCA motifs detected in file\n",
      "\n",
      "Seq is: AAAACAGTACACTCTT\n",
      "0 AAAACAGTACACTCTT motifs detected in file\n",
      "\n",
      "Seq is: CTTCGTCCGGCCTACA\n",
      "0 CTTCGTCCGGCCTACA motifs detected in file\n",
      "\n",
      "Seq is: GTCTCCCCACGGAACC\n",
      "0 GTCTCCCCACGGAACC motifs detected in file\n",
      "\n",
      "Seq is: CATTGGAGACCCTGAG\n",
      "0 CATTGGAGACCCTGAG motifs detected in file\n",
      "\n",
      "Seq is: TCTTCACTTTCCCCCG\n",
      "0 TCTTCACTTTCCCCCG motifs detected in file\n",
      "\n",
      "Seq is: ATAAGGATCAAAATAA\n",
      "0 ATAAGGATCAAAATAA motifs detected in file\n",
      "\n",
      "Seq is: CGATATCGATAATACT\n",
      "0 CGATATCGATAATACT motifs detected in file\n",
      "\n",
      "Seq is: AATAATTACTACTCAT\n",
      "0 AATAATTACTACTCAT motifs detected in file\n",
      "\n",
      "Seq is: AACTAACCATCATAAC\n",
      "0 AACTAACCATCATAAC motifs detected in file\n",
      "\n",
      "Seq is: CAATACCAAGTAACAG\n",
      "0 CAATACCAAGTAACAG motifs detected in file\n",
      "\n",
      "Seq is: GCCTCTCATATAACAA\n",
      "0 GCCTCTCATATAACAA motifs detected in file\n",
      "\n",
      "Seq is: CTTCTCCTATCGATAA\n",
      "0 CTTCTCCTATCGATAA motifs detected in file\n",
      "\n",
      "Seq is: TCTTCCTAATACCTAC\n",
      "0 TCTTCCTAATACCTAC motifs detected in file\n",
      "\n",
      "Seq is: GCCAACGAACGCACTC\n",
      "0 GCCAACGAACGCACTC motifs detected in file\n",
      "\n",
      "Seq is: CTTTATGAACTACCGT\n",
      "0 CTTTATGAACTACCGT motifs detected in file\n",
      "\n",
      "Seq is: CGAAGACACCTTGCTC\n",
      "0 CGAAGACACCTTGCTC motifs detected in file\n",
      "\n",
      "Seq is: CCAAATAAAAAAACCA\n",
      "0 CCAAATAAAAAAACCA motifs detected in file\n",
      "\n",
      "Seq is: ATCTTGACCTTATTTT\n",
      "0 ATCTTGACCTTATTTT motifs detected in file\n",
      "\n",
      "Seq is: CGATCGTACAAATAAA\n",
      "0 CGATCGTACAAATAAA motifs detected in file\n",
      "\n",
      "Seq is: GATCCGGATGAGTCCA\n",
      "0 GATCCGGATGAGTCCA motifs detected in file\n",
      "\n",
      "Seq is: TTTTTTAGTCACGCTC\n",
      "0 TTTTTTAGTCACGCTC motifs detected in file\n",
      "\n",
      "Seq is: GAATCGCGACACTACT\n",
      "0 GAATCGCGACACTACT motifs detected in file\n",
      "\n",
      "Seq is: CACACGGACGTTTCTA\n",
      "0 CACACGGACGTTTCTA motifs detected in file\n",
      "\n",
      "Seq is: CCATCTCATCTACTGC\n",
      "0 CCATCTCATCTACTGC motifs detected in file\n",
      "\n",
      "Seq is: CCAACCCGGTCCCCTA\n",
      "0 CCAACCCGGTCCCCTA motifs detected in file\n",
      "\n",
      "Seq is: ATTAATCATGCCCTTC\n",
      "0 ATTAATCATGCCCTTC motifs detected in file\n",
      "\n",
      "Seq is: CCATATTGGTTGTAAA\n",
      "0 CCATATTGGTTGTAAA motifs detected in file\n",
      "\n",
      "Seq is: AGCCCCATACCCGGGC\n",
      "0 AGCCCCATACCCGGGC motifs detected in file\n",
      "\n",
      "Seq is: TATCGAATAAATCGAC\n",
      "0 TATCGAATAAATCGAC motifs detected in file\n",
      "\n",
      "Seq is: TGGAATGAAATCCTAC\n",
      "0 TGGAATGAAATCCTAC motifs detected in file\n",
      "\n",
      "Seq is: CCCACACTATCCACCG\n",
      "0 CCCACACTATCCACCG motifs detected in file\n",
      "\n",
      "Seq is: TGCCTCTTAAAACCTA\n",
      "0 TGCCTCTTAAAACCTA motifs detected in file\n",
      "\n",
      "Seq is: AGAGTCCCTACCCAAG\n",
      "0 AGAGTCCCTACCCAAG motifs detected in file\n",
      "\n",
      "Seq is: CTAAGAGTATCAGGAT\n",
      "0 CTAAGAGTATCAGGAT motifs detected in file\n",
      "\n",
      "Seq is: CTTTATTCCCCCAAAT\n",
      "0 CTTTATTCCCCCAAAT motifs detected in file\n",
      "\n",
      "Seq is: TCGAGGATAATAAATG\n",
      "0 TCGAGGATAATAAATG motifs detected in file\n",
      "\n",
      "Seq is: AGATAGTTTCATTGAG\n",
      "0 AGATAGTTTCATTGAG motifs detected in file\n",
      "\n",
      "Seq is: AAAATAGTCTGTATAA\n",
      "0 AAAATAGTCTGTATAA motifs detected in file\n",
      "\n",
      "Seq is: AGAATCCAAACTCCCC\n",
      "0 AGAATCCAAACTCCCC motifs detected in file\n",
      "\n",
      "Seq is: CTTACGACCTCTAACA\n",
      "0 CTTACGACCTCTAACA motifs detected in file\n",
      "\n",
      "Seq is: TTACCCATACCTCTGT\n",
      "0 TTACCCATACCTCTGT motifs detected in file\n",
      "\n",
      "Seq is: ATAGTATATTCATTAC\n",
      "0 ATAGTATATTCATTAC motifs detected in file\n",
      "\n",
      "Seq is: GATCCCACTCACCATC\n",
      "0 GATCCCACTCACCATC motifs detected in file\n",
      "\n",
      "Seq is: CTTCAAAAAAGTATCC\n",
      "0 CTTCAAAAAAGTATCC motifs detected in file\n",
      "\n",
      "Seq is: TCCACATACTCAACCA\n",
      "0 TCCACATACTCAACCA motifs detected in file\n",
      "\n",
      "Seq is: GCATCGCCTTAGCCCC\n",
      "0 GCATCGCCTTAGCCCC motifs detected in file\n",
      "\n",
      "Seq is: CATACGACAAGCTTAA\n",
      "0 CATACGACAAGCTTAA motifs detected in file\n",
      "\n",
      "Seq is: GTATTCTTGTCCCTCC\n",
      "0 GTATTCTTGTCCCTCC motifs detected in file\n",
      "\n",
      "Seq is: AATCTTCATCCCAGAA\n",
      "0 AATCTTCATCCCAGAA motifs detected in file\n",
      "\n",
      "Seq is: CCACTGTTTTATACAA\n",
      "0 CCACTGTTTTATACAA motifs detected in file\n",
      "\n",
      "Seq is: CACATCTCAAGTCCCC\n",
      "0 CACATCTCAAGTCCCC motifs detected in file\n",
      "\n",
      "Seq is: TCTCACGCAGTATACA\n",
      "0 TCTCACGCAGTATACA motifs detected in file\n",
      "\n",
      "Seq is: ACAAGGATCCTTCTAA\n",
      "0 ACAAGGATCCTTCTAA motifs detected in file\n",
      "\n",
      "Seq is: CATCACCACTCCCACA\n",
      "0 CATCACCACTCCCACA motifs detected in file\n",
      "\n",
      "Seq is: AATAATATTATTACAA\n",
      "0 AATAATATTATTACAA motifs detected in file\n",
      "\n",
      "Seq is: ACACATAAGCCGATAC\n",
      "0 ACACATAAGCCGATAC motifs detected in file\n",
      "\n",
      "Seq is: TTCTTATCCCGCTTCC\n",
      "0 TTCTTATCCCGCTTCC motifs detected in file\n",
      "\n",
      "Seq is: CCGGACGCCGCATAAG\n",
      "0 CCGGACGCCGCATAAG motifs detected in file\n",
      "\n",
      "Seq is: CTACAACTTCGGACTC\n",
      "0 CTACAACTTCGGACTC motifs detected in file\n",
      "\n",
      "Seq is: TGATCAAGCCTGAGGG\n",
      "0 TGATCAAGCCTGAGGG motifs detected in file\n",
      "\n",
      "Seq is: GAAGCCGTTCCAGCTT\n",
      "0 GAAGCCGTTCCAGCTT motifs detected in file\n",
      "\n",
      "Seq is: CCCCCAAGCCAACCAG\n",
      "0 CCCCCAAGCCAACCAG motifs detected in file\n",
      "\n",
      "Seq is: AGACGATCACACCTCT\n",
      "0 AGACGATCACACCTCT motifs detected in file\n",
      "\n",
      "Seq is: ATTTAGTATAATACCG\n",
      "0 ATTTAGTATAATACCG motifs detected in file\n",
      "\n",
      "Seq is: GTTCCCAGTACTACCG\n",
      "0 GTTCCCAGTACTACCG motifs detected in file\n",
      "\n",
      "Seq is: TCCTCATTAGTCTCCA\n",
      "0 TCCTCATTAGTCTCCA motifs detected in file\n",
      "\n",
      "Seq is: CAAGAACACAACACTA\n",
      "0 CAAGAACACAACACTA motifs detected in file\n",
      "\n",
      "Seq is: TTCCCACCTCTCCAAT\n",
      "0 TTCCCACCTCTCCAAT motifs detected in file\n",
      "\n",
      "Seq is: TTCCTCGGTGAATAAT\n",
      "0 TTCCTCGGTGAATAAT motifs detected in file\n",
      "\n",
      "Seq is: CATTACAACTATCATC\n",
      "0 CATTACAACTATCATC motifs detected in file\n",
      "\n",
      "Seq is: TTACTACCGATCCCAC\n",
      "0 TTACTACCGATCCCAC motifs detected in file\n",
      "\n",
      "Seq is: TGAAGTATACTCTAAC\n",
      "0 TGAAGTATACTCTAAC motifs detected in file\n",
      "\n",
      "Seq is: AAACCCGATGACGAGC\n",
      "0 AAACCCGATGACGAGC motifs detected in file\n",
      "\n",
      "Seq is: GTCACGCGGCTAGTCC\n",
      "0 GTCACGCGGCTAGTCC motifs detected in file\n",
      "\n",
      "Seq is: CGCATCAAACTCAAAC\n",
      "0 CGCATCAAACTCAAAC motifs detected in file\n",
      "\n",
      "Seq is: TACGAGTGGGACTAGT\n",
      "0 TACGAGTGGGACTAGT motifs detected in file\n",
      "\n",
      "Seq is: CTCCTAACTCATTTGC\n",
      "0 CTCCTAACTCATTTGC motifs detected in file\n",
      "\n",
      "Seq is: CGATCCGATCTCCACC\n",
      "0 CGATCCGATCTCCACC motifs detected in file\n",
      "\n",
      "Seq is: GATCTTATTTATCCTC\n",
      "0 GATCTTATTTATCCTC motifs detected in file\n",
      "\n",
      "Seq is: CGGATCCAACTCCAAC\n",
      "0 CGGATCCAACTCCAAC motifs detected in file\n",
      "\n",
      "Seq is: TGGTCCCCCAACCCAT\n",
      "0 TGGTCCCCCAACCCAT motifs detected in file\n",
      "\n",
      "Seq is: ACCCCTCCCCCCAAGT\n",
      "0 ACCCCTCCCCCCAAGT motifs detected in file\n",
      "\n",
      "Seq is: ATCATCTTCTCGCTAC\n",
      "0 ATCATCTTCTCGCTAC motifs detected in file\n",
      "\n",
      "Seq is: CACTCTCGATTCCAGC\n",
      "0 CACTCTCGATTCCAGC motifs detected in file\n",
      "\n",
      "Seq is: CCCGCCACTACATCTC\n",
      "0 CCCGCCACTACATCTC motifs detected in file\n",
      "\n",
      "Seq is: CCACTACCATCTACAC\n",
      "0 CCACTACCATCTACAC motifs detected in file\n",
      "\n",
      "Seq is: CGCCCAAAATCCCCGA\n",
      "0 CGCCCAAAATCCCCGA motifs detected in file\n",
      "\n",
      "Seq is: GAAACCACTTCTCAAA\n",
      "0 GAAACCACTTCTCAAA motifs detected in file\n",
      "\n",
      "Seq is: ATACCGCAGTCGCTTC\n",
      "0 ATACCGCAGTCGCTTC motifs detected in file\n",
      "\n",
      "Seq is: CCAACATCATCGGGCA\n",
      "0 CCAACATCATCGGGCA motifs detected in file\n",
      "\n",
      "Seq is: TCCCCGGATGTTGCAA\n",
      "0 TCCCCGGATGTTGCAA motifs detected in file\n",
      "\n",
      "Seq is: CCCCGGAAACGCATCA\n",
      "0 CCCCGGAAACGCATCA motifs detected in file\n",
      "\n",
      "Seq is: ACATATATCGGATCTT\n",
      "0 ACATATATCGGATCTT motifs detected in file\n",
      "\n",
      "Seq is: AAAAAGCAAGCCATTC\n",
      "0 AAAAAGCAAGCCATTC motifs detected in file\n",
      "\n",
      "Seq is: GTAATCCTTACGGTAA\n",
      "0 GTAATCCTTACGGTAA motifs detected in file\n",
      "\n",
      "Seq is: CGCTAATCTTACCCAT\n",
      "0 CGCTAATCTTACCCAT motifs detected in file\n",
      "\n",
      "Seq is: GTTACTCCTCATCCTC\n",
      "0 GTTACTCCTCATCCTC motifs detected in file\n",
      "\n",
      "Seq is: CAACCGGTACCCATAC\n",
      "0 CAACCGGTACCCATAC motifs detected in file\n",
      "\n",
      "Seq is: AACAATTCTTCTCCTT\n",
      "0 AACAATTCTTCTCCTT motifs detected in file\n",
      "\n",
      "Seq is: AACTTGGAGACTGACA\n",
      "0 AACTTGGAGACTGACA motifs detected in file\n",
      "\n",
      "Seq is: TTTCAAAATTCAAAAT\n",
      "0 TTTCAAAATTCAAAAT motifs detected in file\n",
      "\n",
      "Seq is: ACGCTAATGGCCCGAG\n",
      "0 ACGCTAATGGCCCGAG motifs detected in file\n",
      "\n",
      "Seq is: ACGGTAGAATTGTTTG\n",
      "0 ACGGTAGAATTGTTTG motifs detected in file\n",
      "\n",
      "Seq is: GGACAAGAACCCACCC\n",
      "0 GGACAAGAACCCACCC motifs detected in file\n",
      "\n",
      "Seq is: ACACCCATATTATGAT\n",
      "0 ACACCCATATTATGAT motifs detected in file\n",
      "\n",
      "Seq is: TCAACTCTACTATAGT\n",
      "0 TCAACTCTACTATAGT motifs detected in file\n",
      "\n",
      "Seq is: AAATGCCCCCTTCCGC\n",
      "0 AAATGCCCCCTTCCGC motifs detected in file\n",
      "\n",
      "Seq is: GAAACACTTCATCCGG\n",
      "0 GAAACACTTCATCCGG motifs detected in file\n",
      "\n",
      "Seq is: AATAAAGAGAACAGGA\n",
      "0 AATAAAGAGAACAGGA motifs detected in file\n",
      "\n",
      "Seq is: AAGCATGTCCCTCCTT\n",
      "0 AAGCATGTCCCTCCTT motifs detected in file\n",
      "\n",
      "Seq is: AAACTTNCATCTATCT\n",
      "0 AAACTTNCATCTATCT motifs detected in file\n",
      "\n",
      "Seq is: TTGGCTGGACCTAATG\n",
      "0 TTGGCTGGACCTAATG motifs detected in file\n",
      "\n",
      "Seq is: AGGCCAGACTTGAGTC\n",
      "0 AGGCCAGACTTGAGTC motifs detected in file\n",
      "\n",
      "Seq is: TAGTGCATCCTGAAAT\n",
      "0 TAGTGCATCCTGAAAT motifs detected in file\n",
      "\n",
      "Seq is: TAGCAACTTGTTTGCT\n",
      "0 TAGCAACTTGTTTGCT motifs detected in file\n",
      "\n",
      "Seq is: TGGAAATTATCGCCGA\n",
      "0 TGGAAATTATCGCCGA motifs detected in file\n",
      "\n",
      "Seq is: CGTGGTAGCCCTACAG\n",
      "0 CGTGGTAGCCCTACAG motifs detected in file\n",
      "\n",
      "Seq is: GACTAGGTTGTAGCTC\n",
      "0 GACTAGGTTGTAGCTC motifs detected in file\n",
      "\n",
      "Seq is: CAGCATTTGGGATAAC\n",
      "0 CAGCATTTGGGATAAC motifs detected in file\n",
      "\n",
      "Seq is: AACTATACCTGAGATC\n",
      "0 AACTATACCTGAGATC motifs detected in file\n",
      "\n",
      "Seq is: TTATCCTAACGCGACA\n",
      "0 TTATCCTAACGCGACA motifs detected in file\n",
      "\n",
      "Seq is: ATAGGGATCCCATTGA\n",
      "0 ATAGGGATCCCATTGA motifs detected in file\n",
      "\n",
      "Seq is: ACAAGGCAACCAGTTC\n",
      "0 ACAAGGCAACCAGTTC motifs detected in file\n",
      "\n",
      "Seq is: AATAACCTAGTTAACT\n",
      "0 AATAACCTAGTTAACT motifs detected in file\n",
      "\n",
      "Seq is: CATATCATCAAGCGAA\n",
      "0 CATATCATCAAGCGAA motifs detected in file\n",
      "\n",
      "Seq is: ACTGACCACTTCAGAA\n",
      "0 ACTGACCACTTCAGAA motifs detected in file\n",
      "\n",
      "Seq is: TCGTACATGACGAGCC\n",
      "0 TCGTACATGACGAGCC motifs detected in file\n",
      "\n",
      "Seq is: TCCAACCCAAGACGAG\n",
      "0 TCCAACCCAAGACGAG motifs detected in file\n",
      "\n",
      "Seq is: GCTCCAGCGGGGTTGG\n",
      "0 GCTCCAGCGGGGTTGG motifs detected in file\n",
      "\n",
      "Seq is: CTTTAAAAATTACGTC\n",
      "0 CTTTAAAAATTACGTC motifs detected in file\n",
      "\n",
      "Seq is: CAAACCATCAAATCCT\n",
      "0 CAAACCATCAAATCCT motifs detected in file\n",
      "\n",
      "Seq is: GGACACCCAAACAATC\n",
      "0 GGACACCCAAACAATC motifs detected in file\n",
      "\n",
      "Seq is: CATGACAAACGTAATT\n",
      "0 CATGACAAACGTAATT motifs detected in file\n",
      "\n",
      "Seq is: ATTTAATTTCGAGGTA\n",
      "0 ATTTAATTTCGAGGTA motifs detected in file\n",
      "\n",
      "Seq is: TCCCAGAAGAGCAGAA\n",
      "0 TCCCAGAAGAGCAGAA motifs detected in file\n",
      "\n",
      "Seq is: CGACACAATACGGGCG\n",
      "0 CGACACAATACGGGCG motifs detected in file\n",
      "\n",
      "Seq is: GAGAAGTGCCCGTCCA\n",
      "0 GAGAAGTGCCCGTCCA motifs detected in file\n",
      "\n",
      "Seq is: GTTAAAGTGACCAATT\n",
      "0 GTTAAAGTGACCAATT motifs detected in file\n",
      "\n",
      "Seq is: TTCATTCTCTGTCGAC\n",
      "0 TTCATTCTCTGTCGAC motifs detected in file\n",
      "\n",
      "Seq is: TTGGGAGCACCTCGGT\n",
      "0 TTGGGAGCACCTCGGT motifs detected in file\n",
      "\n",
      "Seq is: AAGTATGTCCAGGGAT\n",
      "0 AAGTATGTCCAGGGAT motifs detected in file\n",
      "\n",
      "Seq is: AAATTCCTTGTTCACT\n",
      "0 AAATTCCTTGTTCACT motifs detected in file\n",
      "\n",
      "Seq is: AATACGATGGAAACGT\n",
      "0 AATACGATGGAAACGT motifs detected in file\n",
      "\n",
      "Seq is: GCCAATCCCATGGCGC\n",
      "0 GCCAATCCCATGGCGC motifs detected in file\n",
      "\n",
      "Seq is: CGGCAATTTGTACACA\n",
      "0 CGGCAATTTGTACACA motifs detected in file\n",
      "\n",
      "Seq is: GTGACCCGTCCGCCAC\n",
      "0 GTGACCCGTCCGCCAC motifs detected in file\n",
      "\n",
      "Seq is: GGAGATTATGACCACT\n",
      "0 GGAGATTATGACCACT motifs detected in file\n",
      "\n",
      "Seq is: ACGATCTCCACTACAA\n",
      "0 ACGATCTCCACTACAA motifs detected in file\n",
      "\n",
      "Seq is: AAACCATTTGTCCGCC\n",
      "0 AAACCATTTGTCCGCC motifs detected in file\n",
      "\n",
      "Seq is: CCATTCTAAACGGCTC\n",
      "0 CCATTCTAAACGGCTC motifs detected in file\n",
      "\n",
      "Seq is: AAGGAAAATGAAAAAA\n",
      "0 AAGGAAAATGAAAAAA motifs detected in file\n",
      "\n",
      "Seq is: ATTGGAAAGGAATACT\n",
      "0 ATTGGAAAGGAATACT motifs detected in file\n",
      "\n",
      "Seq is: CGTACGGACACAACCC\n",
      "0 CGTACGGACACAACCC motifs detected in file\n",
      "\n",
      "Seq is: AACTGTCACTCCCATT\n",
      "0 AACTGTCACTCCCATT motifs detected in file\n",
      "\n",
      "Seq is: ATTACTGAACAACCAA\n",
      "0 ATTACTGAACAACCAA motifs detected in file\n",
      "\n",
      "Seq is: CTAACATCTATAACCC\n",
      "0 CTAACATCTATAACCC motifs detected in file\n",
      "\n",
      "Seq is: GACAATTAACAGTCAA\n",
      "0 GACAATTAACAGTCAA motifs detected in file\n",
      "\n",
      "Seq is: GTCACAAAATTAGACT\n",
      "0 GTCACAAAATTAGACT motifs detected in file\n",
      "\n",
      "Seq is: GCGTCCGAATACGCCT\n",
      "0 GCGTCCGAATACGCCT motifs detected in file\n",
      "\n",
      "Seq is: CCTCTTACAAAAGTAC\n",
      "0 CCTCTTACAAAAGTAC motifs detected in file\n",
      "\n",
      "Seq is: AATGAATATGATTGTA\n",
      "0 AATGAATATGATTGTA motifs detected in file\n",
      "\n",
      "Seq is: ATCAAGAAGATATCCC\n",
      "0 ATCAAGAAGATATCCC motifs detected in file\n",
      "\n",
      "Seq is: ACTATCTAACCAGGTT\n",
      "0 ACTATCTAACCAGGTT motifs detected in file\n",
      "\n",
      "Seq is: AACCCACACTCCTCAA\n",
      "0 AACCCACACTCCTCAA motifs detected in file\n",
      "\n",
      "Seq is: GTCAATATACAAACCC\n",
      "0 GTCAATATACAAACCC motifs detected in file\n",
      "\n",
      "Seq is: TAAAAAATCCATCACC\n",
      "0 TAAAAAATCCATCACC motifs detected in file\n",
      "\n",
      "Seq is: CACAACTCGAACTTGC\n",
      "0 CACAACTCGAACTTGC motifs detected in file\n",
      "\n",
      "Seq is: GAAAGAATTAACCACC\n",
      "0 GAAAGAATTAACCACC motifs detected in file\n",
      "\n",
      "Seq is: GACGAAAATCCGGATG\n",
      "0 GACGAAAATCCGGATG motifs detected in file\n",
      "\n",
      "Seq is: ATACCCACAATTTAAA\n",
      "0 ATACCCACAATTTAAA motifs detected in file\n",
      "\n",
      "Seq is: AAATGAGAGAGATGTT\n",
      "0 AAATGAGAGAGATGTT motifs detected in file\n",
      "\n",
      "Seq is: CCAAAAAAGGATCACA\n",
      "0 CCAAAAAAGGATCACA motifs detected in file\n",
      "\n",
      "Seq is: GGTTTCTCGACAAGGA\n",
      "0 GGTTTCTCGACAAGGA motifs detected in file\n",
      "\n",
      "Seq is: GAAACCTGATTGTCAA\n",
      "0 GAAACCTGATTGTCAA motifs detected in file\n",
      "\n",
      "Seq is: TTTAAATGTTCCCCTA\n",
      "0 TTTAAATGTTCCCCTA motifs detected in file\n",
      "\n",
      "Seq is: AATCTCCCAAGACACC\n",
      "0 AATCTCCCAAGACACC motifs detected in file\n",
      "\n",
      "Seq is: CGTTTAAATTTCAACT\n",
      "0 CGTTTAAATTTCAACT motifs detected in file\n",
      "\n",
      "Seq is: TGATTCTAAGATAGAA\n",
      "0 TGATTCTAAGATAGAA motifs detected in file\n",
      "\n",
      "Seq is: CCTGTTGGTCGATAGT\n",
      "0 CCTGTTGGTCGATAGT motifs detected in file\n",
      "\n",
      "Seq is: GGTCCGAGCCATCCAA\n",
      "0 GGTCCGAGCCATCCAA motifs detected in file\n",
      "\n",
      "Seq is: ACAGCGGAGATGGATA\n",
      "0 ACAGCGGAGATGGATA motifs detected in file\n",
      "\n",
      "Seq is: TTTAGAAGGGTGATAA\n",
      "0 TTTAGAAGGGTGATAA motifs detected in file\n",
      "\n",
      "Seq is: AACGATGTATCTGCCC\n",
      "0 AACGATGTATCTGCCC motifs detected in file\n",
      "\n",
      "Seq is: ACACGAGAAAATCGAC\n",
      "0 ACACGAGAAAATCGAC motifs detected in file\n",
      "\n",
      "Seq is: AAGAATCCATCGAGCA\n",
      "0 AAGAATCCATCGAGCA motifs detected in file\n",
      "\n",
      "Seq is: GACCAAAGCCCCCAGA\n",
      "0 GACCAAAGCCCCCAGA motifs detected in file\n",
      "\n",
      "Seq is: ATCGAAACCGAGAGGA\n",
      "0 ATCGAAACCGAGAGGA motifs detected in file\n",
      "\n",
      "Seq is: ACGTTTCAATAAAGAT\n",
      "0 ACGTTTCAATAAAGAT motifs detected in file\n",
      "\n",
      "Seq is: CAATTAAGTAATACGT\n",
      "0 CAATTAAGTAATACGT motifs detected in file\n",
      "\n",
      "Seq is: CTTCCATATCCCCAAT\n",
      "0 CTTCCATATCCCCAAT motifs detected in file\n",
      "\n",
      "Seq is: CAGGAACGATATAATA\n",
      "0 CAGGAACGATATAATA motifs detected in file\n",
      "\n",
      "Seq is: CGAACCAATATTAAAA\n",
      "0 CGAACCAATATTAAAA motifs detected in file\n",
      "\n",
      "Seq is: AGTAGAAAGGGAACGC\n",
      "0 AGTAGAAAGGGAACGC motifs detected in file\n",
      "\n",
      "Seq is: CATGATATAGATAACG\n",
      "0 CATGATATAGATAACG motifs detected in file\n",
      "\n",
      "Seq is: CGGTCCAAAGTTAAAG\n",
      "0 CGGTCCAAAGTTAAAG motifs detected in file\n",
      "\n",
      "Seq is: ATAGCGGATATGAAAT\n",
      "0 ATAGCGGATATGAAAT motifs detected in file\n",
      "\n",
      "Seq is: AAACCCATTTACCAAA\n",
      "0 AAACCCATTTACCAAA motifs detected in file\n",
      "\n",
      "Seq is: CCGATTCCAACAGACC\n",
      "0 CCGATTCCAACAGACC motifs detected in file\n",
      "\n",
      "Seq is: ATCATTCCACCTCACC\n",
      "0 ATCATTCCACCTCACC motifs detected in file\n",
      "\n",
      "Seq is: CAAACCCCGATCCAAA\n",
      "0 CAAACCCCGATCCAAA motifs detected in file\n",
      "\n",
      "Seq is: TCGAGTCTCGCCAGTT\n",
      "0 TCGAGTCTCGCCAGTT motifs detected in file\n",
      "\n",
      "Seq is: CAATTCAACTTTAGAG\n",
      "0 CAATTCAACTTTAGAG motifs detected in file\n",
      "\n",
      "Seq is: GATTCACATTCAACCC\n",
      "0 GATTCACATTCAACCC motifs detected in file\n",
      "\n",
      "Seq is: ACGAAACACAATTCGA\n",
      "0 ACGAAACACAATTCGA motifs detected in file\n",
      "\n",
      "Seq is: TGTGAGACCAAGCAGG\n",
      "0 TGTGAGACCAAGCAGG motifs detected in file\n",
      "\n",
      "Seq is: TTCACGTGAAAGGTCA\n",
      "0 TTCACGTGAAAGGTCA motifs detected in file\n",
      "\n",
      "Seq is: TGTGAATGGTACAATG\n",
      "0 TGTGAATGGTACAATG motifs detected in file\n",
      "\n",
      "Seq is: CTGAACAGAGGAGATA\n",
      "0 CTGAACAGAGGAGATA motifs detected in file\n",
      "\n",
      "Seq is: TATTTACGCATCCCCA\n",
      "0 TATTTACGCATCCCCA motifs detected in file\n",
      "\n",
      "Seq is: AAATCAATTTACAGGA\n",
      "0 AAATCAATTTACAGGA motifs detected in file\n",
      "\n",
      "Seq is: AACTTCATATGAACTC\n",
      "0 AACTTCATATGAACTC motifs detected in file\n",
      "\n",
      "Seq is: CTCCCACTGCCCGCCA\n",
      "0 CTCCCACTGCCCGCCA motifs detected in file\n",
      "\n",
      "Seq is: CACATGCGCGAAGTCC\n",
      "0 CACATGCGCGAAGTCC motifs detected in file\n",
      "\n",
      "Seq is: CGGGACAAGTTGATTC\n",
      "0 CGGGACAAGTTGATTC motifs detected in file\n",
      "\n",
      "Seq is: GTGAGATGAGAATCAA\n",
      "0 GTGAGATGAGAATCAA motifs detected in file\n",
      "\n",
      "Seq is: ATGACGATTTAGGTGG\n",
      "0 ATGACGATTTAGGTGG motifs detected in file\n",
      "\n",
      "Seq is: AAGCTGGGAATTCAAA\n",
      "0 AAGCTGGGAATTCAAA motifs detected in file\n",
      "\n",
      "Seq is: GTATTCCCGATAGCAT\n",
      "0 GTATTCCCGATAGCAT motifs detected in file\n",
      "\n",
      "Seq is: CAAAAGACCCCATCTT\n",
      "0 CAAAAGACCCCATCTT motifs detected in file\n",
      "\n",
      "Seq is: TTACATCGGGTAAAGA\n",
      "0 TTACATCGGGTAAAGA motifs detected in file\n",
      "\n",
      "Seq is: ACGGTGGAGTACCCGA\n",
      "0 ACGGTGGAGTACCCGA motifs detected in file\n",
      "\n",
      "Seq is: TGTGGAACTGGATTGC\n",
      "0 TGTGGAACTGGATTGC motifs detected in file\n",
      "\n",
      "Seq is: AGAAATGCACCCATGA\n",
      "0 AGAAATGCACCCATGA motifs detected in file\n",
      "\n",
      "Seq is: ACGCGAATGAAACATC\n",
      "0 ACGCGAATGAAACATC motifs detected in file\n",
      "\n",
      "Seq is: GGAAGTAGTCCCAAAC\n",
      "0 GGAAGTAGTCCCAAAC motifs detected in file\n",
      "\n",
      "Seq is: GACTTCTACCGCCATA\n",
      "0 GACTTCTACCGCCATA motifs detected in file\n",
      "\n",
      "Seq is: TATCCGACTCGTTCTC\n",
      "0 TATCCGACTCGTTCTC motifs detected in file\n",
      "\n",
      "Seq is: CACCACTCCAACTAGC\n",
      "0 CACCACTCCAACTAGC motifs detected in file\n",
      "\n",
      "Seq is: CCCAAATAGCTAATGT\n",
      "0 CCCAAATAGCTAATGT motifs detected in file\n",
      "\n",
      "Seq is: CTTGTCCGAGGAGATC\n",
      "0 CTTGTCCGAGGAGATC motifs detected in file\n",
      "\n",
      "Seq is: TCCCTATACTTCGTGG\n",
      "0 TCCCTATACTTCGTGG motifs detected in file\n",
      "\n",
      "Seq is: CGGTCCAGGAAACTCA\n",
      "0 CGGTCCAGGAAACTCA motifs detected in file\n",
      "\n",
      "Seq is: AAATTCGACACCGAGC\n",
      "0 AAATTCGACACCGAGC motifs detected in file\n",
      "\n",
      "Seq is: ATCACAAGACCGCTCG\n",
      "0 ATCACAAGACCGCTCG motifs detected in file\n",
      "\n",
      "Seq is: TCAAAACAACTAAATT\n",
      "0 TCAAAACAACTAAATT motifs detected in file\n",
      "\n",
      "Seq is: GACAACTCCAAATCCC\n",
      "0 GACAACTCCAAATCCC motifs detected in file\n",
      "\n",
      "Seq is: GATTCGTATCACCCCA\n",
      "0 GATTCGTATCACCCCA motifs detected in file\n",
      "\n",
      "Seq is: TAGATTAGGGTCAAAC\n",
      "0 TAGATTAGGGTCAAAC motifs detected in file\n",
      "\n",
      "Seq is: CCAGAATCGATAACAC\n",
      "0 CCAGAATCGATAACAC motifs detected in file\n",
      "\n",
      "Seq is: ACAAGTCTATACAATT\n",
      "0 ACAAGTCTATACAATT motifs detected in file\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Seq is: TCGGTGAAAGCATCAG\n",
      "0 TCGGTGAAAGCATCAG motifs detected in file\n",
      "\n",
      "Seq is: ATAAAACACAGTTGAC\n",
      "0 ATAAAACACAGTTGAC motifs detected in file\n",
      "\n",
      "Seq is: CTCAAAAAATGTTGAG\n",
      "0 CTCAAAAAATGTTGAG motifs detected in file\n",
      "\n",
      "Seq is: TCCACTCAAAATCGAA\n",
      "0 TCCACTCAAAATCGAA motifs detected in file\n",
      "\n",
      "Seq is: ATAACCCCTCCCCCAC\n",
      "0 ATAACCCCTCCCCCAC motifs detected in file\n",
      "\n",
      "Seq is: TAGATTTTGTGAGAAA\n",
      "0 TAGATTTTGTGAGAAA motifs detected in file\n",
      "\n",
      "Seq is: TGCGGCCGAAGATAAC\n",
      "0 TGCGGCCGAAGATAAC motifs detected in file\n",
      "\n",
      "Seq is: TGAACCCAATTAGCAC\n",
      "0 TGAACCCAATTAGCAC motifs detected in file\n",
      "\n",
      "Seq is: ACTGGCGCCACCGACC\n",
      "0 ACTGGCGCCACCGACC motifs detected in file\n",
      "\n",
      "Seq is: GTGCTTTAACTGGTTG\n",
      "0 GTGCTTTAACTGGTTG motifs detected in file\n",
      "\n",
      "Seq is: GGACCCCAATCATATC\n",
      "0 GGACCCCAATCATATC motifs detected in file\n",
      "\n",
      "Seq is: GAATCAATTTGAAAGC\n",
      "0 GAATCAATTTGAAAGC motifs detected in file\n",
      "\n",
      "Seq is: AAATAACGATTTCCAA\n",
      "0 AAATAACGATTTCCAA motifs detected in file\n",
      "\n",
      "Seq is: TTAGTGACGACAAAGG\n",
      "0 TTAGTGACGACAAAGG motifs detected in file\n",
      "\n",
      "Seq is: GCACCCCCACACCGAT\n",
      "0 GCACCCCCACACCGAT motifs detected in file\n",
      "\n",
      "Seq is: CCGATTCGCAAAACTC\n",
      "0 CCGATTCGCAAAACTC motifs detected in file\n",
      "\n",
      "Seq is: GACGTAACGACGCACG\n",
      "0 GACGTAACGACGCACG motifs detected in file\n",
      "\n",
      "Seq is: AACTACGAACAAGGAA\n",
      "0 AACTACGAACAAGGAA motifs detected in file\n",
      "\n",
      "Seq is: AACTAGCACCACTAAA\n",
      "0 AACTAGCACCACTAAA motifs detected in file\n",
      "\n",
      "Seq is: TCTCCCACTTGAGTGA\n",
      "0 TCTCCCACTTGAGTGA motifs detected in file\n",
      "\n",
      "Seq is: CCTTGCCCCTACGAAC\n",
      "0 CCTTGCCCCTACGAAC motifs detected in file\n",
      "\n",
      "Seq is: GTACACATTAGAATGA\n",
      "0 GTACACATTAGAATGA motifs detected in file\n",
      "\n",
      "Seq is: TTCTCGATTATCTTTC\n",
      "0 TTCTCGATTATCTTTC motifs detected in file\n",
      "\n",
      "Seq is: CGATCCTGGTTTGGAT\n",
      "0 CGATCCTGGTTTGGAT motifs detected in file\n",
      "\n",
      "Seq is: AAACAAATACCCCACT\n",
      "0 AAACAAATACCCCACT motifs detected in file\n",
      "\n",
      "Seq is: ACACTCGGGCAGATTT\n",
      "0 ACACTCGGGCAGATTT motifs detected in file\n",
      "\n",
      "Seq is: GTAAAAGTCACATAAC\n",
      "0 GTAAAAGTCACATAAC motifs detected in file\n",
      "\n",
      "Seq is: GAAACTCCTCCATTCG\n",
      "0 GAAACTCCTCCATTCG motifs detected in file\n",
      "\n",
      "Seq is: ATGTATTTGACACCCC\n",
      "0 ATGTATTTGACACCCC motifs detected in file\n",
      "\n",
      "Seq is: CCACAGAAACCCCAAA\n",
      "0 CCACAGAAACCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: CCAACCAAGCCCCATA\n",
      "0 CCAACCAAGCCCCATA motifs detected in file\n",
      "\n",
      "Seq is: CCCCAATCGTCGCCAC\n",
      "0 CCCCAATCGTCGCCAC motifs detected in file\n",
      "\n",
      "Seq is: ACACACACGACCCATC\n",
      "0 ACACACACGACCCATC motifs detected in file\n",
      "\n",
      "Seq is: CTACCCGCCCCCAACA\n",
      "0 CTACCCGCCCCCAACA motifs detected in file\n",
      "\n",
      "Seq is: TAACTACTCTAATCAT\n",
      "0 TAACTACTCTAATCAT motifs detected in file\n",
      "\n",
      "Seq is: CATACCCTCACCCTCC\n",
      "0 CATACCCTCACCCTCC motifs detected in file\n",
      "\n",
      "Seq is: CCTTTTATTACACAAT\n",
      "0 CCTTTTATTACACAAT motifs detected in file\n",
      "\n",
      "Seq is: CAACCCCCCACTGACA\n",
      "0 CAACCCCCCACTGACA motifs detected in file\n",
      "\n",
      "Seq is: ATTTTCACGTATGGCG\n",
      "0 ATTTTCACGTATGGCG motifs detected in file\n",
      "\n",
      "Seq is: GTTTTCTATTTTAAAC\n",
      "0 GTTTTCTATTTTAAAC motifs detected in file\n",
      "\n",
      "Seq is: TTTAGACCAATCCGAC\n",
      "0 TTTAGACCAATCCGAC motifs detected in file\n",
      "\n",
      "Seq is: CACAATCCCAAGAAAC\n",
      "0 CACAATCCCAAGAAAC motifs detected in file\n",
      "\n",
      "Seq is: AAAAACCCCAAACCGT\n",
      "0 AAAAACCCCAAACCGT motifs detected in file\n",
      "\n",
      "Seq is: CTCTACACAAATTCAC\n",
      "0 CTCTACACAAATTCAC motifs detected in file\n",
      "\n",
      "Seq is: GACACCGGTCTTCGCC\n",
      "0 GACACCGGTCTTCGCC motifs detected in file\n",
      "\n",
      "Seq is: CCCTCCCCCCCAAACC\n",
      "0 CCCTCCCCCCCAAACC motifs detected in file\n",
      "\n",
      "Seq is: ACCTTTAAAAAACAAT\n",
      "0 ACCTTTAAAAAACAAT motifs detected in file\n",
      "\n",
      "Seq is: ACTACAGACACACCTT\n",
      "0 ACTACAGACACACCTT motifs detected in file\n",
      "\n",
      "Seq is: TCACCGACACGTCTGT\n",
      "0 TCACCGACACGTCTGT motifs detected in file\n",
      "\n",
      "Seq is: AAGTTAACAATAATAA\n",
      "0 AAGTTAACAATAATAA motifs detected in file\n",
      "\n",
      "Seq is: TACAGGATGTTCGTAA\n",
      "0 TACAGGATGTTCGTAA motifs detected in file\n",
      "\n",
      "Seq is: TTAATTAATTGTGTGA\n",
      "0 TTAATTAATTGTGTGA motifs detected in file\n",
      "\n",
      "Seq is: AATCATTCATACAAGC\n",
      "0 AATCATTCATACAAGC motifs detected in file\n",
      "\n",
      "Seq is: GGACATTATAACTTGC\n",
      "0 GGACATTATAACTTGC motifs detected in file\n",
      "\n",
      "Seq is: ATCCACGCTATTTATT\n",
      "0 ATCCACGCTATTTATT motifs detected in file\n",
      "\n",
      "Seq is: ATCCTACTCCGTCCTT\n",
      "0 ATCCTACTCCGTCCTT motifs detected in file\n",
      "\n",
      "Seq is: AGTTTCTGTCTATGAC\n",
      "0 AGTTTCTGTCTATGAC motifs detected in file\n",
      "\n",
      "Seq is: GCTGTATCCCACGAGG\n",
      "0 GCTGTATCCCACGAGG motifs detected in file\n",
      "\n",
      "Seq is: CCGAGGTCGCAGAGCG\n",
      "0 CCGAGGTCGCAGAGCG motifs detected in file\n",
      "\n",
      "Seq is: TTACGATAGCGCACGT\n",
      "0 TTACGATAGCGCACGT motifs detected in file\n",
      "\n",
      "Seq is: ATGGGGGGTCTGCTTT\n",
      "0 ATGGGGGGTCTGCTTT motifs detected in file\n",
      "\n",
      "Seq is: TATGGTTTACGTACCT\n",
      "0 TATGGTTTACGTACCT motifs detected in file\n",
      "\n",
      "Seq is: CTCGAGGGCACTCACC\n",
      "0 CTCGAGGGCACTCACC motifs detected in file\n",
      "\n",
      "Seq is: AATTATCCCACTATCT\n",
      "0 AATTATCCCACTATCT motifs detected in file\n",
      "\n",
      "Seq is: GTAGCACTACAGAATAA\n",
      "0 GTAGCACTACAGAATAA motifs detected in file\n",
      "\n",
      "Seq is: ATTCCCCTTGCACACCC\n",
      "0 ATTCCCCTTGCACACCC motifs detected in file\n",
      "\n",
      "Seq is: GATAAATCCGAAATACT\n",
      "0 GATAAATCCGAAATACT motifs detected in file\n",
      "\n",
      "Seq is: GGGACTTCATCCTTGGT\n",
      "0 GGGACTTCATCCTTGGT motifs detected in file\n",
      "\n",
      "Seq is: CTACAGCCTATGTCAAG\n",
      "0 CTACAGCCTATGTCAAG motifs detected in file\n",
      "\n",
      "Seq is: TGAAATCGATGGGGGTT\n",
      "0 TGAAATCGATGGGGGTT motifs detected in file\n",
      "\n",
      "Seq is: CACAATACCCGGGCCTC\n",
      "0 CACAATACCCGGGCCTC motifs detected in file\n",
      "\n",
      "Seq is: GCTCCTCTCATCGTGAG\n",
      "0 GCTCCTCTCATCGTGAG motifs detected in file\n",
      "\n",
      "Seq is: AACACGCCCTATAACTA\n",
      "0 AACACGCCCTATAACTA motifs detected in file\n",
      "\n",
      "Seq is: AAGTGCCTCCTACCACC\n",
      "0 AAGTGCCTCCTACCACC motifs detected in file\n",
      "\n",
      "Seq is: AGTTCCCTGGGGATAGA\n",
      "0 AGTTCCCTGGGGATAGA motifs detected in file\n",
      "\n",
      "Seq is: CTCCCCCCAGTAGGTAC\n",
      "0 CTCCCCCCAGTAGGTAC motifs detected in file\n",
      "\n",
      "Seq is: CCCTGCTCTTCCCTAAA\n",
      "0 CCCTGCTCTTCCCTAAA motifs detected in file\n",
      "\n",
      "Seq is: CTGACATTACACGATAC\n",
      "0 CTGACATTACACGATAC motifs detected in file\n",
      "\n",
      "Seq is: ATGCCATTTACCGAAAT\n",
      "0 ATGCCATTTACCGAAAT motifs detected in file\n",
      "\n",
      "Seq is: ACATGATACATGACAAT\n",
      "0 ACATGATACATGACAAT motifs detected in file\n",
      "\n",
      "Seq is: TCCCACCCATCCAAACA\n",
      "0 TCCCACCCATCCAAACA motifs detected in file\n",
      "\n",
      "Seq is: ACCATAGGATCACCCAC\n",
      "0 ACCATAGGATCACCCAC motifs detected in file\n",
      "\n",
      "Seq is: TCCCCACCGAAACCTCA\n",
      "0 TCCCCACCGAAACCTCA motifs detected in file\n",
      "\n",
      "Seq is: ACGTCAACTACACACTA\n",
      "0 ACGTCAACTACACACTA motifs detected in file\n",
      "\n",
      "Seq is: TCAACTCCCAACTAACG\n",
      "0 TCAACTCCCAACTAACG motifs detected in file\n",
      "\n",
      "Seq is: ACATGAACGAACATTCG\n",
      "0 ACATGAACGAACATTCG motifs detected in file\n",
      "\n",
      "Seq is: TACCCCTCCCCCAAAAC\n",
      "0 TACCCCTCCCCCAAAAC motifs detected in file\n",
      "\n",
      "Seq is: TACACCTAACCCAAAAA\n",
      "0 TACACCTAACCCAAAAA motifs detected in file\n",
      "\n",
      "Seq is: TACATGATGTCCACCAG\n",
      "0 TACATGATGTCCACCAG motifs detected in file\n",
      "\n",
      "Seq is: TTCATAAATACCATGGC\n",
      "0 TTCATAAATACCATGGC motifs detected in file\n",
      "\n",
      "Seq is: ATGTTATAAGTACCACC\n",
      "0 ATGTTATAAGTACCACC motifs detected in file\n",
      "\n",
      "Seq is: GACCGTCATTACATGCT\n",
      "0 GACCGTCATTACATGCT motifs detected in file\n",
      "\n",
      "Seq is: TTATGTATCGCCAACAA\n",
      "0 TTATGTATCGCCAACAA motifs detected in file\n",
      "\n",
      "Seq is: CTACCCACTCAGTTATG\n",
      "0 CTACCCACTCAGTTATG motifs detected in file\n",
      "\n",
      "Seq is: AACCCACCATGGGTTTA\n",
      "0 AACCCACCATGGGTTTA motifs detected in file\n",
      "\n",
      "Seq is: GACGAAGGGGTACTTTC\n",
      "0 GACGAAGGGGTACTTTC motifs detected in file\n",
      "\n",
      "Seq is: TTGTCTCTTATCAAATT\n",
      "0 TTGTCTCTTATCAAATT motifs detected in file\n",
      "\n",
      "Seq is: TAATCTTAGAATCGAAA\n",
      "0 TAATCTTAGAATCGAAA motifs detected in file\n",
      "\n",
      "Seq is: CCCACGATTACCACCTC\n",
      "0 CCCACGATTACCACCTC motifs detected in file\n",
      "\n",
      "Seq is: AATTTCTGAAAAAGAGA\n",
      "0 AATTTCTGAAAAAGAGA motifs detected in file\n",
      "\n",
      "Seq is: CTAAACAGGAACCTTTT\n",
      "0 CTAAACAGGAACCTTTT motifs detected in file\n",
      "\n",
      "Seq is: TCCAAAAGTAGAGGCCA\n",
      "0 TCCAAAAGTAGAGGCCA motifs detected in file\n",
      "\n",
      "Seq is: AATGTTCTGACCACATA\n",
      "0 AATGTTCTGACCACATA motifs detected in file\n",
      "\n",
      "Seq is: ATCAAATATGATGTTCC\n",
      "0 ATCAAATATGATGTTCC motifs detected in file\n",
      "\n",
      "Seq is: TGTCCGGGTAAACTCAT\n",
      "0 TGTCCGGGTAAACTCAT motifs detected in file\n",
      "\n",
      "Seq is: AAAACAAAAGTTAATCC\n",
      "0 AAAACAAAAGTTAATCC motifs detected in file\n",
      "\n",
      "Seq is: CTCTATCAACCATAATC\n",
      "0 CTCTATCAACCATAATC motifs detected in file\n",
      "\n",
      "Seq is: CTAATCCTAACAACACT\n",
      "0 CTAATCCTAACAACACT motifs detected in file\n",
      "\n",
      "Seq is: TCATATCATGCCTACGA\n",
      "0 TCATATCATGCCTACGA motifs detected in file\n",
      "\n",
      "Seq is: TGAACAGGTTACTACCA\n",
      "0 TGAACAGGTTACTACCA motifs detected in file\n",
      "\n",
      "Seq is: TTTTCCCATCGAATGAC\n",
      "0 TTTTCCCATCGAATGAC motifs detected in file\n",
      "\n",
      "Seq is: CAACAGGAGGCTAAGTC\n",
      "0 CAACAGGAGGCTAAGTC motifs detected in file\n",
      "\n",
      "Seq is: CAATCTTACTCCTCCAG\n",
      "0 CAATCTTACTCCTCCAG motifs detected in file\n",
      "\n",
      "Seq is: ACGCCGATCCTCAGTTA\n",
      "0 ACGCCGATCCTCAGTTA motifs detected in file\n",
      "\n",
      "Seq is: TTTCACTAACCGAATCA\n",
      "0 TTTCACTAACCGAATCA motifs detected in file\n",
      "\n",
      "Seq is: CCCGCTTTATAATACGA\n",
      "0 CCCGCTTTATAATACGA motifs detected in file\n",
      "\n",
      "Seq is: AACAACAAACCTATATA\n",
      "0 AACAACAAACCTATATA motifs detected in file\n",
      "\n",
      "Seq is: CCTCCTACCCCTAATAA\n",
      "0 CCTCCTACCCCTAATAA motifs detected in file\n",
      "\n",
      "Seq is: CGATCCTACTCCTACCT\n",
      "0 CGATCCTACTCCTACCT motifs detected in file\n",
      "\n",
      "Seq is: ATCATTATCCCGTTCCT\n",
      "0 ATCATTATCCCGTTCCT motifs detected in file\n",
      "\n",
      "Seq is: GCGGAGGATCAAACAAT\n",
      "0 GCGGAGGATCAAACAAT motifs detected in file\n",
      "\n",
      "Seq is: CCCTGCCTAGCCTCTTA\n",
      "0 CCCTGCCTAGCCTCTTA motifs detected in file\n",
      "\n",
      "Seq is: ACACATCCGCTTATCCT\n",
      "0 ACACATCCGCTTATCCT motifs detected in file\n",
      "\n",
      "Seq is: TTATAGTAAGCCCGAAC\n",
      "0 TTATAGTAAGCCCGAAC motifs detected in file\n",
      "\n",
      "Seq is: TACACCCCTCCCCACAA\n",
      "0 TACACCCCTCCCCACAA motifs detected in file\n",
      "\n",
      "Seq is: ATTCCCCAACCGATCCC\n",
      "0 ATTCCCCAACCGATCCC motifs detected in file\n",
      "\n",
      "Seq is: ATATTAACAGACCCAGC\n",
      "0 ATATTAACAGACCCAGC motifs detected in file\n",
      "\n",
      "Seq is: GGATCCTCCAGACCACT\n",
      "0 GGATCCTCCAGACCACT motifs detected in file\n",
      "\n",
      "Seq is: CTTATCACAATTACAGT\n",
      "0 CTTATCACAATTACAGT motifs detected in file\n",
      "\n",
      "Seq is: AATTCCTCTCTTCCTTC\n",
      "0 AATTCCTCTCTTCCTTC motifs detected in file\n",
      "\n",
      "Seq is: TCTTCATTCGGCTCCCG\n",
      "0 TCTTCATTCGGCTCCCG motifs detected in file\n",
      "\n",
      "Seq is: CAGAAACTAACACATCA\n",
      "0 CAGAAACTAACACATCA motifs detected in file\n",
      "\n",
      "Seq is: TTCCCACCTTCCACTAA\n",
      "0 TTCCCACCTTCCACTAA motifs detected in file\n",
      "\n",
      "Seq is: AATAGCCTTACCCTCCA\n",
      "0 AATAGCCTTACCCTCCA motifs detected in file\n",
      "\n",
      "Seq is: CTAAGGATCCCCCAACA\n",
      "0 CTAAGGATCCCCCAACA motifs detected in file\n",
      "\n",
      "Seq is: AACTAGGGCAAAGCACG\n",
      "0 AACTAGGGCAAAGCACG motifs detected in file\n",
      "\n",
      "Seq is: TTCTTATCCTCCACCTC\n",
      "0 TTCTTATCCTCCACCTC motifs detected in file\n",
      "\n",
      "Seq is: ACAACGATCCCGACGTT\n",
      "0 ACAACGATCCCGACGTT motifs detected in file\n",
      "\n",
      "Seq is: ATTACTTCCCGTTCTAC\n",
      "0 ATTACTTCCCGTTCTAC motifs detected in file\n",
      "\n",
      "Seq is: TTCACTTTCCATTTCTT\n",
      "0 TTCACTTTCCATTTCTT motifs detected in file\n",
      "\n",
      "Seq is: AGCACACTCCCACCCTG\n",
      "0 AGCACACTCCCACCCTG motifs detected in file\n",
      "\n",
      "Seq is: ACAGATGACTCATCGGA\n",
      "0 ACAGATGACTCATCGGA motifs detected in file\n",
      "\n",
      "Seq is: GGAGTCTAAGTAACTTG\n",
      "0 GGAGTCTAAGTAACTTG motifs detected in file\n",
      "\n",
      "Seq is: ATCCAGACAGGGTTACA\n",
      "0 ATCCAGACAGGGTTACA motifs detected in file\n",
      "\n",
      "Seq is: TACCCTACCGCCTATCA\n",
      "0 TACCCTACCGCCTATCA motifs detected in file\n",
      "\n",
      "Seq is: TTCAAACATTAATGACA\n",
      "0 TTCAAACATTAATGACA motifs detected in file\n",
      "\n",
      "Seq is: CCGGGGAGTCTTACTAT\n",
      "0 CCGGGGAGTCTTACTAT motifs detected in file\n",
      "\n",
      "Seq is: AAACCGGAGTGCCCTCC\n",
      "0 AAACCGGAGTGCCCTCC motifs detected in file\n",
      "\n",
      "Seq is: TGTATCGGATACTTCCG\n",
      "0 TGTATCGGATACTTCCG motifs detected in file\n",
      "\n",
      "Seq is: ACAACGATATCAACGTT\n",
      "0 ACAACGATATCAACGTT motifs detected in file\n",
      "\n",
      "Seq is: CGTCCTCCTATTACGGC\n",
      "0 CGTCCTCCTATTACGGC motifs detected in file\n",
      "\n",
      "Seq is: TACAAAGTCCAAAGACT\n",
      "0 TACAAAGTCCAAAGACT motifs detected in file\n",
      "\n",
      "Seq is: CATCTCTTTACTAGGCA\n",
      "0 CATCTCTTTACTAGGCA motifs detected in file\n",
      "\n",
      "Seq is: TTATATCCGGAGCGGGC\n",
      "0 TTATATCCGGAGCGGGC motifs detected in file\n",
      "\n",
      "Seq is: TACACATCCTTCTCCGT\n",
      "0 TACACATCCTTCTCCGT motifs detected in file\n",
      "\n",
      "Seq is: CTATTTCTTATAACTCC\n",
      "0 CTATTTCTTATAACTCC motifs detected in file\n",
      "\n",
      "Seq is: GCGGTAACCGCACTTCC\n",
      "0 GCGGTAACCGCACTTCC motifs detected in file\n",
      "\n",
      "Seq is: ATCGCCTACTAAGTCGG\n",
      "0 ATCGCCTACTAAGTCGG motifs detected in file\n",
      "\n",
      "Seq is: TATTAAATGCAGAGCTC\n",
      "0 TATTAAATGCAGAGCTC motifs detected in file\n",
      "\n",
      "Seq is: ACTACACCCGCTAACTA\n",
      "0 ACTACACCCGCTAACTA motifs detected in file\n",
      "\n",
      "Seq is: CTTTTCCGCCAACTCCG\n",
      "0 CTTTTCCGCCAACTCCG motifs detected in file\n",
      "\n",
      "Seq is: CAGACCACTCATCACGT\n",
      "0 CAGACCACTCATCACGT motifs detected in file\n",
      "\n",
      "Seq is: ACCGATCCTTATCAGGA\n",
      "0 ACCGATCCTTATCAGGA motifs detected in file\n",
      "\n",
      "Seq is: CACCACTAAACCTCCTA\n",
      "0 CACCACTAAACCTCCTA motifs detected in file\n",
      "\n",
      "Seq is: GTCCGTCCGCGGTTCCT\n",
      "0 GTCCGTCCGCGGTTCCT motifs detected in file\n",
      "\n",
      "Seq is: CACTCGGCTTCAAAGTA\n",
      "0 CACTCGGCTTCAAAGTA motifs detected in file\n",
      "\n",
      "Seq is: GTACGCCTCTACAACCT\n",
      "0 GTACGCCTCTACAACCT motifs detected in file\n",
      "\n",
      "Seq is: ACCCCACCCCTCCAGCT\n",
      "0 ACCCCACCCCTCCAGCT motifs detected in file\n",
      "\n",
      "Seq is: ACTTACTCACCAATTAA\n",
      "0 ACTTACTCACCAATTAA motifs detected in file\n",
      "\n",
      "Seq is: TTAAAATAATCCCCCAA\n",
      "0 TTAAAATAATCCCCCAA motifs detected in file\n",
      "\n",
      "Seq is: TCAAAACGCATAACCCC\n",
      "0 TCAAAACGCATAACCCC motifs detected in file\n",
      "\n",
      "Seq is: AGTAACCACAAGAACAT\n",
      "0 AGTAACCACAAGAACAT motifs detected in file\n",
      "\n",
      "Seq is: CAACTTTATGTTGCTAC\n",
      "0 CAACTTTATGTTGCTAC motifs detected in file\n",
      "\n",
      "Seq is: CAAAAAGTATAGTAACC\n",
      "0 CAAAAAGTATAGTAACC motifs detected in file\n",
      "\n",
      "Seq is: AGCACCAACATCAGGCA\n",
      "0 AGCACCAACATCAGGCA motifs detected in file\n",
      "\n",
      "Seq is: CGCTCTTATTACTACAT\n",
      "0 CGCTCTTATTACTACAT motifs detected in file\n",
      "\n",
      "Seq is: ACGAAACAAAGACAACT\n",
      "0 ACGAAACAAAGACAACT motifs detected in file\n",
      "\n",
      "Seq is: CACACCCAAATCATTAC\n",
      "0 CACACCCAAATCATTAC motifs detected in file\n",
      "\n",
      "Seq is: CCCAAACACCCCAAAAG\n",
      "0 CCCAAACACCCCAAAAG motifs detected in file\n",
      "\n",
      "Seq is: AAGATTCGGAAGAGGAT\n",
      "0 AAGATTCGGAAGAGGAT motifs detected in file\n",
      "\n",
      "Seq is: AAATACCCCCAAATCAT\n",
      "0 AAATACCCCCAAATCAT motifs detected in file\n",
      "\n",
      "Seq is: AACTAACAATCGCCACA\n",
      "0 AACTAACAATCGCCACA motifs detected in file\n",
      "\n",
      "Seq is: CCAGCCCACACAATAAT\n",
      "0 CCAGCCCACACAATAAT motifs detected in file\n",
      "\n",
      "Seq is: AAGACTTAAAACCCCCT\n",
      "0 AAGACTTAAAACCCCCT motifs detected in file\n",
      "\n",
      "Seq is: CCAATATACCCAAATTA\n",
      "0 CCAATATACCCAAATTA motifs detected in file\n",
      "\n",
      "Seq is: TCAAAAAAATTAAATAA\n",
      "0 TCAAAAAAATTAAATAA motifs detected in file\n",
      "\n",
      "Seq is: ATCCCCCTTACTACCAA\n",
      "0 ATCCCCCTTACTACCAA motifs detected in file\n",
      "\n",
      "Seq is: CAGAAACCTATATGATG\n",
      "0 CAGAAACCTATATGATG motifs detected in file\n",
      "\n",
      "Seq is: TCGCTACCGATAACTCC\n",
      "0 TCGCTACCGATAACTCC motifs detected in file\n",
      "\n",
      "Seq is: TCATAGGACTCCGTACC\n",
      "0 TCATAGGACTCCGTACC motifs detected in file\n",
      "\n",
      "Seq is: CCCAGTCCCCAACTCCA\n",
      "0 CCCAGTCCCCAACTCCA motifs detected in file\n",
      "\n",
      "Seq is: GAACCACTCACAAAATC\n",
      "0 GAACCACTCACAAAATC motifs detected in file\n",
      "\n",
      "Seq is: ACCCCAATCGCTACCTC\n",
      "0 ACCCCAATCGCTACCTC motifs detected in file\n",
      "\n",
      "Seq is: CATCCTAACCACGACAC\n",
      "0 CATCCTAACCACGACAC motifs detected in file\n",
      "\n",
      "Seq is: CCACTTTCTCATACTAC\n",
      "0 CCACTTTCTCATACTAC motifs detected in file\n",
      "\n",
      "Seq is: CCCACCACCAACACCAT\n",
      "0 CCCACCACCAACACCAT motifs detected in file\n",
      "\n",
      "Seq is: TTGAAATTATCACATCC\n",
      "0 TTGAAATTATCACATCC motifs detected in file\n",
      "\n",
      "Seq is: TTCGACTTATTAAATAC\n",
      "0 TTCGACTTATTAAATAC motifs detected in file\n",
      "\n",
      "Seq is: TTCCTCTCCCCAGTCCC\n",
      "0 TTCCTCTCCCCAGTCCC motifs detected in file\n",
      "\n",
      "Seq is: AACTAAGCCCTCCTAGG\n",
      "0 AACTAAGCCCTCCTAGG motifs detected in file\n",
      "\n",
      "Seq is: ATAACCACGCCCCCGAA\n",
      "0 ATAACCACGCCCCCGAA motifs detected in file\n",
      "\n",
      "Seq is: ACATACTAATACCCGCA\n",
      "0 ACATACTAATACCCGCA motifs detected in file\n",
      "\n",
      "Seq is: ACTAATCATCATCAATG\n",
      "0 ACTAATCATCATCAATG motifs detected in file\n",
      "\n",
      "Seq is: ACCAACTTGTAACAAAC\n",
      "0 ACCAACTTGTAACAAAC motifs detected in file\n",
      "\n",
      "Seq is: AACCACATATATAACAT\n",
      "0 AACCACATATATAACAT motifs detected in file\n",
      "\n",
      "Seq is: TAACTCTAACGAGCCCC\n",
      "0 TAACTCTAACGAGCCCC motifs detected in file\n",
      "\n",
      "Seq is: CTTATCCAATACACTAA\n",
      "0 CTTATCCAATACACTAA motifs detected in file\n",
      "\n",
      "Seq is: TCCTCATCCCAATCCTA\n",
      "0 TCCTCATCCCAATCCTA motifs detected in file\n",
      "\n",
      "Seq is: CTCACCCTTCTTCTTTC\n",
      "0 CTCACCCTTCTTCTTTC motifs detected in file\n",
      "\n",
      "Seq is: TCTCCTTCATTTCAAAT\n",
      "0 TCTCCTTCATTTCAAAT motifs detected in file\n",
      "\n",
      "Seq is: TAATACGGAAAAACCCA\n",
      "0 TAATACGGAAAAACCCA motifs detected in file\n",
      "\n",
      "Seq is: ACTCCACTACTACCTCC\n",
      "0 ACTCCACTACTACCTCC motifs detected in file\n",
      "\n",
      "Seq is: ACCTCTAAACCACGACA\n",
      "0 ACCTCTAAACCACGACA motifs detected in file\n",
      "\n",
      "Seq is: CTTTAACAAAATCCATT\n",
      "0 CTTTAACAAAATCCATT motifs detected in file\n",
      "\n",
      "Seq is: ATCGAAAAGATCAGTCC\n",
      "0 ATCGAAAAGATCAGTCC motifs detected in file\n",
      "\n",
      "Seq is: AATCCAGATCCTCCTCA\n",
      "0 AATCCAGATCCTCCTCA motifs detected in file\n",
      "\n",
      "Seq is: TCCCCGTCCAAAACCGA\n",
      "0 TCCCCGTCCAAAACCGA motifs detected in file\n",
      "\n",
      "Seq is: GCATTCTTCCGGATCTA\n",
      "0 GCATTCTTCCGGATCTA motifs detected in file\n",
      "\n",
      "Seq is: TCCCCTAACACGCCACA\n",
      "0 TCCCCTAACACGCCACA motifs detected in file\n",
      "\n",
      "Seq is: CACTACGATCCCATCTT\n",
      "0 CACTACGATCCCATCTT motifs detected in file\n",
      "\n",
      "Seq is: AGGCTCATACAACCTCT\n",
      "0 AGGCTCATACAACCTCT motifs detected in file\n",
      "\n",
      "Seq is: TTATTTTACACGTATCA\n",
      "0 TTATTTTACACGTATCA motifs detected in file\n",
      "\n",
      "Seq is: CCCCTAAAATAAAATTC\n",
      "0 CCCCTAAAATAAAATTC motifs detected in file\n",
      "\n",
      "Seq is: AAACAACCAATCCATCA\n",
      "0 AAACAACCAATCCATCA motifs detected in file\n",
      "\n",
      "Seq is: ACTCCAGATCCCGACAA\n",
      "0 ACTCCAGATCCCGACAA motifs detected in file\n",
      "\n",
      "Seq is: TCTTCAGGATCCTTTCA\n",
      "0 TCTTCAGGATCCTTTCA motifs detected in file\n",
      "\n",
      "Seq is: CTGTCGCTCCCGACACT\n",
      "0 CTGTCGCTCCCGACACT motifs detected in file\n",
      "\n",
      "Seq is: CAAAATCCATCTCCCCC\n",
      "0 CAAAATCCATCTCCCCC motifs detected in file\n",
      "\n",
      "Seq is: TAACAACAAACCTTCCC\n",
      "0 TAACAACAAACCTTCCC motifs detected in file\n",
      "\n",
      "Seq is: CCTACGCCCCCTTTACA\n",
      "0 CCTACGCCCCCTTTACA motifs detected in file\n",
      "\n",
      "Seq is: ACAATCATTACTCTTTA\n",
      "0 ACAATCATTACTCTTTA motifs detected in file\n",
      "\n",
      "Seq is: GGACGCTTATCCGAAGG\n",
      "0 GGACGCTTATCCGAAGG motifs detected in file\n",
      "\n",
      "Seq is: CCGACGGTCCGCAAATT\n",
      "0 CCGACGGTCCGCAAATT motifs detected in file\n",
      "\n",
      "Seq is: ACCCCAAATCATCCCAC\n",
      "0 ACCCCAAATCATCCCAC motifs detected in file\n",
      "\n",
      "Seq is: CCCAATAAAAGCAATTA\n",
      "0 CCCAATAAAAGCAATTA motifs detected in file\n",
      "\n",
      "Seq is: CAATCATTCCCACCCCT\n",
      "0 CAATCATTCCCACCCCT motifs detected in file\n",
      "\n",
      "Seq is: TCGCTCCAACTGGACAA\n",
      "0 TCGCTCCAACTGGACAA motifs detected in file\n",
      "\n",
      "Seq is: TCCCACTCTTCTTAATA\n",
      "0 TCCCACTCTTCTTAATA motifs detected in file\n",
      "\n",
      "Seq is: AGCTCACGATATCCGCG\n",
      "0 AGCTCACGATATCCGCG motifs detected in file\n",
      "\n",
      "Seq is: AACAGTCCCTCCATCGC\n",
      "0 AACAGTCCCTCCATCGC motifs detected in file\n",
      "\n",
      "Seq is: TACTCTCATTATCTATC\n",
      "0 TACTCTCATTATCTATC motifs detected in file\n",
      "\n",
      "Seq is: CCGAGTCCGCAAACACA\n",
      "0 CCGAGTCCGCAAACACA motifs detected in file\n",
      "\n",
      "Seq is: TACTATACAAACGCCAA\n",
      "0 TACTATACAAACGCCAA motifs detected in file\n",
      "\n",
      "Seq is: AGCTACTACACCAGAAA\n",
      "0 AGCTACTACACCAGAAA motifs detected in file\n",
      "\n",
      "Seq is: CCTCATCTTTGGACACT\n",
      "0 CCTCATCTTTGGACACT motifs detected in file\n",
      "\n",
      "Seq is: CCTTTCCATAAGGACGA\n",
      "0 CCTTTCCATAAGGACGA motifs detected in file\n",
      "\n",
      "Seq is: TTACGATCCGACGGTTA\n",
      "0 TTACGATCCGACGGTTA motifs detected in file\n",
      "\n",
      "Seq is: CCACTCCCTCCAACTTC\n",
      "0 CCACTCCCTCCAACTTC motifs detected in file\n",
      "\n",
      "Seq is: ACTCTCCATACCAAAAC\n",
      "0 ACTCTCCATACCAAAAC motifs detected in file\n",
      "\n",
      "Seq is: TCATCAGGAGGATAAAA\n",
      "0 TCATCAGGAGGATAAAA motifs detected in file\n",
      "\n",
      "Seq is: AGCTTATAGAACAAGTA\n",
      "0 AGCTTATAGAACAAGTA motifs detected in file\n",
      "\n",
      "Seq is: ACAATTCCAACACCTAC\n",
      "0 ACAATTCCAACACCTAC motifs detected in file\n",
      "\n",
      "Seq is: TACCTGGGCCTCGTGTA\n",
      "0 TACCTGGGCCTCGTGTA motifs detected in file\n",
      "\n",
      "Seq is: TTTATCATACCGAAACT\n",
      "0 TTTATCATACCGAAACT motifs detected in file\n",
      "\n",
      "Seq is: TCTTCCGCACCCATGTC\n",
      "0 TCTTCCGCACCCATGTC motifs detected in file\n",
      "\n",
      "Seq is: TACACGTCCTTACGATC\n",
      "0 TACACGTCCTTACGATC motifs detected in file\n",
      "\n",
      "Seq is: CACACCAACCAACTACG\n",
      "0 CACACCAACCAACTACG motifs detected in file\n",
      "\n",
      "Seq is: GCTAACATTGATAATAC\n",
      "0 GCTAACATTGATAATAC motifs detected in file\n",
      "\n",
      "Seq is: TCAGGATCAACTGAACT\n",
      "0 TCAGGATCAACTGAACT motifs detected in file\n",
      "\n",
      "Seq is: TCACCTCTTCCGATGCT\n",
      "0 TCACCTCTTCCGATGCT motifs detected in file\n",
      "\n",
      "Seq is: AAAAAAACTACAGTAAA\n",
      "0 AAAAAAACTACAGTAAA motifs detected in file\n",
      "\n",
      "Seq is: ACACATTCCCGCGTCTG\n",
      "0 ACACATTCCCGCGTCTG motifs detected in file\n",
      "\n",
      "Seq is: ACGACGCTTGTCTCACC\n",
      "0 ACGACGCTTGTCTCACC motifs detected in file\n",
      "\n",
      "Seq is: ACTATCGCGGATTCGTA\n",
      "0 ACTATCGCGGATTCGTA motifs detected in file\n",
      "\n",
      "Seq is: TCACAATCTCAAACCTA\n",
      "0 TCACAATCTCAAACCTA motifs detected in file\n",
      "\n",
      "Seq is: ATCACCCGATAAAAGAC\n",
      "0 ATCACCCGATAAAAGAC motifs detected in file\n",
      "\n",
      "Seq is: GATCCCCCACCTTCGCC\n",
      "0 GATCCCCCACCTTCGCC motifs detected in file\n",
      "\n",
      "Seq is: TACTCATTCTTCTAAGG\n",
      "0 TACTCATTCTTCTAAGG motifs detected in file\n",
      "\n",
      "Seq is: ACGATGTTGATATCACG\n",
      "0 ACGATGTTGATATCACG motifs detected in file\n",
      "\n",
      "Seq is: AACTCACCTCATCCCGA\n",
      "0 AACTCACCTCATCCCGA motifs detected in file\n",
      "\n",
      "Seq is: CTCTGACCCCACCCCGG\n",
      "0 CTCTGACCCCACCCCGG motifs detected in file\n",
      "\n",
      "Seq is: AAGATACCGACTCCCCT\n",
      "0 AAGATACCGACTCCCCT motifs detected in file\n",
      "\n",
      "Seq is: CAGTCCCCACCTCTGGA\n",
      "0 CAGTCCCCACCTCTGGA motifs detected in file\n",
      "\n",
      "Seq is: TTAACCCGACTAAACGG\n",
      "0 TTAACCCGACTAAACGG motifs detected in file\n",
      "\n",
      "Seq is: ACGACGACGATCCTCCT\n",
      "0 ACGACGACGATCCTCCT motifs detected in file\n",
      "\n",
      "Seq is: CCGGATCATCACCCCAC\n",
      "0 CCGGATCATCACCCCAC motifs detected in file\n",
      "\n",
      "Seq is: TCCGAACCTAATCGCAA\n",
      "0 TCCGAACCTAATCGCAA motifs detected in file\n",
      "\n",
      "Seq is: ATCTTCCCGATAAACAA\n",
      "0 ATCTTCCCGATAAACAA motifs detected in file\n",
      "\n",
      "Seq is: CACCCAGAGTACTCAAC\n",
      "0 CACCCAGAGTACTCAAC motifs detected in file\n",
      "\n",
      "Seq is: CTCACATCCTATTTAGT\n",
      "0 CTCACATCCTATTTAGT motifs detected in file\n",
      "\n",
      "Seq is: ACGATTCCGCTCCTACT\n",
      "0 ACGATTCCGCTCCTACT motifs detected in file\n",
      "\n",
      "Seq is: TTGGCTATAGCGGCTAT\n",
      "0 TTGGCTATAGCGGCTAT motifs detected in file\n",
      "\n",
      "Seq is: GCCAACATATCCTAACG\n",
      "0 GCCAACATATCCTAACG motifs detected in file\n",
      "\n",
      "Seq is: AACTTACCGACGACACA\n",
      "0 AACTTACCGACGACACA motifs detected in file\n",
      "\n",
      "Seq is: ACCGTAGACGAGCCCGC\n",
      "0 ACCGTAGACGAGCCCGC motifs detected in file\n",
      "\n",
      "Seq is: ATAGTAGTTGACTACTC\n",
      "0 ATAGTAGTTGACTACTC motifs detected in file\n",
      "\n",
      "Seq is: GTTCTTCCTATATTAAG\n",
      "0 GTTCTTCCTATATTAAG motifs detected in file\n",
      "\n",
      "Seq is: GATGCGGGAGAGTCGGC\n",
      "0 GATGCGGGAGAGTCGGC motifs detected in file\n",
      "\n",
      "Seq is: TACTTGTCAACCTTATC\n",
      "0 TACTTGTCAACCTTATC motifs detected in file\n",
      "\n",
      "Seq is: CAACAATCGCCATTGAT\n",
      "0 CAACAATCGCCATTGAT motifs detected in file\n",
      "\n",
      "Seq is: TCTAATCATACCATTAA\n",
      "0 TCTAATCATACCATTAA motifs detected in file\n",
      "\n",
      "Seq is: TCCTTCTACTCATCTAT\n",
      "0 TCCTTCTACTCATCTAT motifs detected in file\n",
      "\n",
      "Seq is: AAACTTCTTGACTAATT\n",
      "0 AAACTTCTTGACTAATT motifs detected in file\n",
      "\n",
      "Seq is: ACAAACCCAGACTCAAA\n",
      "0 ACAAACCCAGACTCAAA motifs detected in file\n",
      "\n",
      "Seq is: TATATAGTGTCACTCTT\n",
      "0 TATATAGTGTCACTCTT motifs detected in file\n",
      "\n",
      "Seq is: AAGATACTACCTGGTAC\n",
      "0 AAGATACTACCTGGTAC motifs detected in file\n",
      "\n",
      "Seq is: ATTGCTTGTTACGATGT\n",
      "0 ATTGCTTGTTACGATGT motifs detected in file\n",
      "\n",
      "Seq is: CCCTACTTATAATACCT\n",
      "0 CCCTACTTATAATACCT motifs detected in file\n",
      "\n",
      "Seq is: CTTCATCAGATCAAACT\n",
      "0 CTTCATCAGATCAAACT motifs detected in file\n",
      "\n",
      "Seq is: TCGAATCCCTCTCGACC\n",
      "0 TCGAATCCCTCTCGACC motifs detected in file\n",
      "\n",
      "Seq is: CAACAAACCCAACACCG\n",
      "0 CAACAAACCCAACACCG motifs detected in file\n",
      "\n",
      "Seq is: AGTCACAGTCAAGCTCT\n",
      "0 AGTCACAGTCAAGCTCT motifs detected in file\n",
      "\n",
      "Seq is: ATTATTGAAGAACCAGA\n",
      "0 ATTATTGAAGAACCAGA motifs detected in file\n",
      "\n",
      "Seq is: TCCGTGTACTTATAACA\n",
      "0 TCCGTGTACTTATAACA motifs detected in file\n",
      "\n",
      "Seq is: ACACCCCTTCTCTGACT\n",
      "0 ACACCCCTTCTCTGACT motifs detected in file\n",
      "\n",
      "Seq is: ATTATTTCCACCTACGC\n",
      "0 ATTATTTCCACCTACGC motifs detected in file\n",
      "\n",
      "Seq is: TGTTACCTAAAATGTAT\n",
      "0 TGTTACCTAAAATGTAT motifs detected in file\n",
      "\n",
      "Seq is: TACCCCCATACTCAAAA\n",
      "0 TACCCCCATACTCAAAA motifs detected in file\n",
      "\n",
      "Seq is: AAAACAATCCCAATTGC\n",
      "0 AAAACAATCCCAATTGC motifs detected in file\n",
      "\n",
      "Seq is: TCCCACCATTCCTACCC\n",
      "0 TCCCACCATTCCTACCC motifs detected in file\n",
      "\n",
      "Seq is: CCCTTAATCCCTTCAGT\n",
      "0 CCCTTAATCCCTTCAGT motifs detected in file\n",
      "\n",
      "Seq is: CCCAATCCCACCAATAT\n",
      "0 CCCAATCCCACCAATAT motifs detected in file\n",
      "\n",
      "Seq is: CATCACACGTACCAATA\n",
      "0 CATCACACGTACCAATA motifs detected in file\n",
      "\n",
      "Seq is: ATGAAAATAAACCTCAA\n",
      "0 ATGAAAATAAACCTCAA motifs detected in file\n",
      "\n",
      "Seq is: CGTGGTTTTAAAAACCC\n",
      "0 CGTGGTTTTAAAAACCC motifs detected in file\n",
      "\n",
      "Seq is: CGGATTCTGGTTACCTA\n",
      "0 CGGATTCTGGTTACCTA motifs detected in file\n",
      "\n",
      "Seq is: TCGACAATAGGAAATTT\n",
      "0 TCGACAATAGGAAATTT motifs detected in file\n",
      "\n",
      "Seq is: TCAACTCTTTCGGTACA\n",
      "0 TCAACTCTTTCGGTACA motifs detected in file\n",
      "\n",
      "Seq is: ACAATCTGTACCCCCGT\n",
      "0 ACAATCTGTACCCCCGT motifs detected in file\n",
      "\n",
      "Seq is: ACTCAATCGTCAAGAAC\n",
      "0 ACTCAATCGTCAAGAAC motifs detected in file\n",
      "\n",
      "Seq is: ACTCGAAAGAGCCATTT\n",
      "0 ACTCGAAAGAGCCATTT motifs detected in file\n",
      "\n",
      "Seq is: ATTCCCCAGCATTCGGA\n",
      "0 ATTCCCCAGCATTCGGA motifs detected in file\n",
      "\n",
      "Seq is: GACAACAGTCTAAGTGT\n",
      "0 GACAACAGTCTAAGTGT motifs detected in file\n",
      "\n",
      "Seq is: TAGACTACAAAACCAAT\n",
      "0 TAGACTACAAAACCAAT motifs detected in file\n",
      "\n",
      "Seq is: TTGATATAAATGTTCTC\n",
      "0 TTGATATAAATGTTCTC motifs detected in file\n",
      "\n",
      "Seq is: CTTTTGGGCCATTACTA\n",
      "0 CTTTTGGGCCATTACTA motifs detected in file\n",
      "\n",
      "Seq is: CAGCCCCAACTCCCTAT\n",
      "0 CAGCCCCAACTCCCTAT motifs detected in file\n",
      "\n",
      "Seq is: CCTCCTCTTACCCCCTA\n",
      "0 CCTCCTCTTACCCCCTA motifs detected in file\n",
      "\n",
      "Seq is: TCCACATACTTGTACTC\n",
      "0 TCCACATACTTGTACTC motifs detected in file\n",
      "\n",
      "Seq is: CCACAAAAGAGCACACT\n",
      "0 CCACAAAAGAGCACACT motifs detected in file\n",
      "\n",
      "Seq is: TACTCCCAAAATACAAC\n",
      "0 TACTCCCAAAATACAAC motifs detected in file\n",
      "\n",
      "Seq is: AATTACACCACCCACTC\n",
      "0 AATTACACCACCCACTC motifs detected in file\n",
      "\n",
      "Seq is: ACTCGGGGTAACACAAC\n",
      "0 ACTCGGGGTAACACAAC motifs detected in file\n",
      "\n",
      "Seq is: ACCATTTATACATCTCC\n",
      "0 ACCATTTATACATCTCC motifs detected in file\n",
      "\n",
      "Seq is: CTCATATCCCGACACTG\n",
      "0 CTCATATCCCGACACTG motifs detected in file\n",
      "\n",
      "Seq is: ATCATACAACTCAGGAC\n",
      "0 ATCATACAACTCAGGAC motifs detected in file\n",
      "\n",
      "Seq is: ATTCATCCTCTCACTAT\n",
      "0 ATTCATCCTCTCACTAT motifs detected in file\n",
      "\n",
      "Seq is: AAACTAGTCCTCTTGCA\n",
      "0 AAACTAGTCCTCTTGCA motifs detected in file\n",
      "\n",
      "Seq is: CCAATGATCGTGTCTCT\n",
      "0 CCAATGATCGTGTCTCT motifs detected in file\n",
      "\n",
      "Seq is: CAAGAGGGTCATCCAAT\n",
      "0 CAAGAGGGTCATCCAAT motifs detected in file\n",
      "\n",
      "Seq is: TATCACCCCCCATTCCG\n",
      "0 TATCACCCCCCATTCCG motifs detected in file\n",
      "\n",
      "Seq is: CTCCAATCGCTCCGAAC\n",
      "0 CTCCAATCGCTCCGAAC motifs detected in file\n",
      "\n",
      "Seq is: GATCTTCAGTAGTTTTT\n",
      "0 GATCTTCAGTAGTTTTT motifs detected in file\n",
      "\n",
      "Seq is: CGATAATCACCCTCATC\n",
      "0 CGATAATCACCCTCATC motifs detected in file\n",
      "\n",
      "Seq is: TCAAACTTCAGGAACTC\n",
      "0 TCAAACTTCAGGAACTC motifs detected in file\n",
      "\n",
      "Seq is: TCTCCTAATACTACGCT\n",
      "0 TCTCCTAATACTACGCT motifs detected in file\n",
      "\n",
      "Seq is: GACACTCACGCAAGCAT\n",
      "0 GACACTCACGCAAGCAT motifs detected in file\n",
      "\n",
      "Seq is: CAAACTCAAACGATCCG\n",
      "0 CAAACTCAAACGATCCG motifs detected in file\n",
      "\n",
      "Seq is: TCTTATCATTACTCCTA\n",
      "0 TCTTATCATTACTCCTA motifs detected in file\n",
      "\n",
      "Seq is: CATTCGGGCACCCGCTA\n",
      "0 CATTCGGGCACCCGCTA motifs detected in file\n",
      "\n",
      "Seq is: ATACTCTTACTGACGCG\n",
      "0 ATACTCTTACTGACGCG motifs detected in file\n",
      "\n",
      "Seq is: GCCACTTCGAAGTCCCC\n",
      "0 GCCACTTCGAAGTCCCC motifs detected in file\n",
      "\n",
      "Seq is: CAAACCTACTCTTACCG\n",
      "0 CAAACCTACTCTTACCG motifs detected in file\n",
      "\n",
      "Seq is: ACAATGATGCTCCCGAT\n",
      "0 ACAATGATGCTCCCGAT motifs detected in file\n",
      "\n",
      "Seq is: ACACCGACTAACTTCTC\n",
      "0 ACACCGACTAACTTCTC motifs detected in file\n",
      "\n",
      "Seq is: ATACGTTACTCGCTAAA\n",
      "0 ATACGTTACTCGCTAAA motifs detected in file\n",
      "\n",
      "Seq is: ATCCAGACAAACAGCAT\n",
      "0 ATCCAGACAAACAGCAT motifs detected in file\n",
      "\n",
      "Seq is: CCGTCTACCTCGAACAA\n",
      "0 CCGTCTACCTCGAACAA motifs detected in file\n",
      "\n",
      "Seq is: TATTAATACGGAGTATC\n",
      "0 TATTAATACGGAGTATC motifs detected in file\n",
      "\n",
      "Seq is: CCTATCATGTTCCTTCC\n",
      "0 CCTATCATGTTCCTTCC motifs detected in file\n",
      "\n",
      "Seq is: CCATCCGATACACAAAA\n",
      "0 CCATCCGATACACAAAA motifs detected in file\n",
      "\n",
      "Seq is: CAGTCCCCCAACTCTTA\n",
      "0 CAGTCCCCCAACTCTTA motifs detected in file\n",
      "\n",
      "Seq is: CTCACACTCCGCATAAT\n",
      "0 CTCACACTCCGCATAAT motifs detected in file\n",
      "\n",
      "Seq is: ATGGTATCGGCGGATCA\n",
      "0 ATGGTATCGGCGGATCA motifs detected in file\n",
      "\n",
      "Seq is: AAATTCTCATGACGCCG\n",
      "0 AAATTCTCATGACGCCG motifs detected in file\n",
      "\n",
      "Seq is: TTCATGATAACTGGGTC\n",
      "0 TTCATGATAACTGGGTC motifs detected in file\n",
      "\n",
      "Seq is: GCTACCCCCGAAGCTGT\n",
      "0 GCTACCCCCGAAGCTGT motifs detected in file\n",
      "\n",
      "Seq is: ACCCGAAATCCCTCAGT\n",
      "0 ACCCGAAATCCCTCAGT motifs detected in file\n",
      "\n",
      "Seq is: ATTCACCTCAGGCATTT\n",
      "0 ATTCACCTCAGGCATTT motifs detected in file\n",
      "\n",
      "Seq is: CTCCATAGAAATGATAT\n",
      "0 CTCCATAGAAATGATAT motifs detected in file\n",
      "\n",
      "Seq is: TTTCGATAACACATTCG\n",
      "0 TTTCGATAACACATTCG motifs detected in file\n",
      "\n",
      "Seq is: ATCAGTATAATTCAACA\n",
      "0 ATCAGTATAATTCAACA motifs detected in file\n",
      "\n",
      "Seq is: ACCGAGTCCTCAAACTA\n",
      "0 ACCGAGTCCTCAAACTA motifs detected in file\n",
      "\n",
      "Seq is: TCAAGAACCCGTCACTC\n",
      "0 TCAAGAACCCGTCACTC motifs detected in file\n",
      "\n",
      "Seq is: TCACTCATCATCTTACA\n",
      "0 TCACTCATCATCTTACA motifs detected in file\n",
      "\n",
      "Seq is: AATCACTCGGATCCCAC\n",
      "0 AATCACTCGGATCCCAC motifs detected in file\n",
      "\n",
      "Seq is: AACACTCACATTTAATC\n",
      "0 AACACTCACATTTAATC motifs detected in file\n",
      "\n",
      "Seq is: ACGCTACTCATCCCCTT\n",
      "0 ACGCTACTCATCCCCTT motifs detected in file\n",
      "\n",
      "Seq is: CCCTCGGATGATCCCAC\n",
      "0 CCCTCGGATGATCCCAC motifs detected in file\n",
      "\n",
      "Seq is: ATCTTATCCTTCATACA\n",
      "0 ATCTTATCCTTCATACA motifs detected in file\n",
      "\n",
      "Seq is: CGGACGCAAGTCCGCAA\n",
      "0 CGGACGCAAGTCCGCAA motifs detected in file\n",
      "\n",
      "Seq is: GACCGACCAACGGAGTA\n",
      "0 GACCGACCAACGGAGTA motifs detected in file\n",
      "\n",
      "Seq is: GCCCACTACTATCGGTT\n",
      "0 GCCCACTACTATCGGTT motifs detected in file\n",
      "\n",
      "Seq is: CCACCCCTATTCACACC\n",
      "0 CCACCCCTATTCACACC motifs detected in file\n",
      "\n",
      "Seq is: AAAGCTTCTTCTATATT\n",
      "0 AAAGCTTCTTCTATATT motifs detected in file\n",
      "\n",
      "Seq is: TTATACTAATCAAGACA\n",
      "0 TTATACTAATCAAGACA motifs detected in file\n",
      "\n",
      "Seq is: CCGACACTTACAATATT\n",
      "0 CCGACACTTACAATATT motifs detected in file\n",
      "\n",
      "Seq is: AATTCCTCTAAACATCC\n",
      "0 AATTCCTCTAAACATCC motifs detected in file\n",
      "\n",
      "Seq is: CTCTAATCATATCTCTC\n",
      "0 CTCTAATCATATCTCTC motifs detected in file\n",
      "\n",
      "Seq is: CATCTCAAAAAAAGCAC\n",
      "0 CATCTCAAAAAAAGCAC motifs detected in file\n",
      "\n",
      "Seq is: TATCACCAAGTGACCTA\n",
      "0 TATCACCAAGTGACCTA motifs detected in file\n",
      "\n",
      "Seq is: TTCACCGCAACCGAACG\n",
      "0 TTCACCGCAACCGAACG motifs detected in file\n",
      "\n",
      "Seq is: GTACTAACACTCCCCAT\n",
      "0 GTACTAACACTCCCCAT motifs detected in file\n",
      "\n",
      "Seq is: CCTCAGTCCATCAATCA\n",
      "0 CCTCAGTCCATCAATCA motifs detected in file\n",
      "\n",
      "Seq is: TAATCCTCCCCCCAACA\n",
      "0 TAATCCTCCCCCCAACA motifs detected in file\n",
      "\n",
      "Seq is: ATCCCCCAGCCTCCTTT\n",
      "0 ATCCCCCAGCCTCCTTT motifs detected in file\n",
      "\n",
      "Seq is: TCCAACCCCTTGTCGAT\n",
      "0 TCCAACCCCTTGTCGAT motifs detected in file\n",
      "\n",
      "Seq is: TTATCCAACAACAACTA\n",
      "0 TTATCCAACAACAACTA motifs detected in file\n",
      "\n",
      "Seq is: AACCAATTTTTTATCAT\n",
      "0 AACCAATTTTTTATCAT motifs detected in file\n",
      "\n",
      "Seq is: CTCCCTACTACGATTAT\n",
      "0 CTCCCTACTACGATTAT motifs detected in file\n",
      "\n",
      "Seq is: TAATCCGACACCCACCA\n",
      "0 TAATCCGACACCCACCA motifs detected in file\n",
      "\n",
      "Seq is: ACACAACTAAGTTTAAT\n",
      "0 ACACAACTAAGTTTAAT motifs detected in file\n",
      "\n",
      "Seq is: ACACAAAAAACCTTTCA\n",
      "0 ACACAAAAAACCTTTCA motifs detected in file\n",
      "\n",
      "Seq is: GTACAGTCACCATCATT\n",
      "0 GTACAGTCACCATCATT motifs detected in file\n",
      "\n",
      "Seq is: ATATTAACAACCCTGCT\n",
      "0 ATATTAACAACCCTGCT motifs detected in file\n",
      "\n",
      "Seq is: AATCAAAATCGTAACCT\n",
      "0 AATCAAAATCGTAACCT motifs detected in file\n",
      "\n",
      "Seq is: CATCCAAATCCAATACA\n",
      "0 CATCCAAATCCAATACA motifs detected in file\n",
      "\n",
      "Seq is: TGCATCAGATCCGGTAT\n",
      "0 TGCATCAGATCCGGTAT motifs detected in file\n",
      "\n",
      "Seq is: ACACAACCTCTAACTCT\n",
      "0 ACACAACCTCTAACTCT motifs detected in file\n",
      "\n",
      "Seq is: GATCATCCCGATCCGGG\n",
      "0 GATCATCCCGATCCGGG motifs detected in file\n",
      "\n",
      "Seq is: TGGCGACGAAGCGTCCG\n",
      "0 TGGCGACGAAGCGTCCG motifs detected in file\n",
      "\n",
      "Seq is: CCGTTTCTGATCATACC\n",
      "0 CCGTTTCTGATCATACC motifs detected in file\n",
      "\n",
      "Seq is: GTTATCCGTGTTATAAC\n",
      "0 GTTATCCGTGTTATAAC motifs detected in file\n",
      "\n",
      "Seq is: CGATTCTCCCTCACCCA\n",
      "0 CGATTCTCCCTCACCCA motifs detected in file\n",
      "\n",
      "Seq is: CAACTCCCAATACTCTC\n",
      "0 CAACTCCCAATACTCTC motifs detected in file\n",
      "\n",
      "Seq is: ATCGATATTACTTGTCG\n",
      "0 ATCGATATTACTTGTCG motifs detected in file\n",
      "\n",
      "Seq is: CTATCATAATAAGGAAG\n",
      "0 CTATCATAATAAGGAAG motifs detected in file\n",
      "\n",
      "Seq is: ATCCGTATCATCCCTCC\n",
      "0 ATCCGTATCATCCCTCC motifs detected in file\n",
      "\n",
      "Seq is: TATACTCCACACTCGCT\n",
      "0 TATACTCCACACTCGCT motifs detected in file\n",
      "\n",
      "Seq is: ATATGATCATAAGGATC\n",
      "0 ATATGATCATAAGGATC motifs detected in file\n",
      "\n",
      "Seq is: TTCACTCTACCATTTAC\n",
      "0 TTCACTCTACCATTTAC motifs detected in file\n",
      "\n",
      "Seq is: GATCATATTATAAATAC\n",
      "0 GATCATATTATAAATAC motifs detected in file\n",
      "\n",
      "Seq is: ATTTACTCCCCGTAAAC\n",
      "0 ATTTACTCCCCGTAAAC motifs detected in file\n",
      "\n",
      "Seq is: CATTTATACTAATAGTA\n",
      "0 CATTTATACTAATAGTA motifs detected in file\n",
      "\n",
      "Seq is: TTAAATTACTCAGCTTT\n",
      "0 TTAAATTACTCAGCTTT motifs detected in file\n",
      "\n",
      "Seq is: AGTAAGCAAAACAAATT\n",
      "0 AGTAAGCAAAACAAATT motifs detected in file\n",
      "\n",
      "Seq is: TGATATATGGTTAAGCC\n",
      "0 TGATATATGGTTAAGCC motifs detected in file\n",
      "\n",
      "Seq is: AAGTCAGATTAGGAAAA\n",
      "0 AAGTCAGATTAGGAAAA motifs detected in file\n",
      "\n",
      "Seq is: ACATCAGTGAGTATCCG\n",
      "0 ACATCAGTGAGTATCCG motifs detected in file\n",
      "\n",
      "Seq is: GTCTGAATCCCGATCCT\n",
      "0 GTCTGAATCCCGATCCT motifs detected in file\n",
      "\n",
      "Seq is: ACTACTAATTATTCTCC\n",
      "0 ACTACTAATTATTCTCC motifs detected in file\n",
      "\n",
      "Seq is: CTACTGTATTGATAATC\n",
      "0 CTACTGTATTGATAATC motifs detected in file\n",
      "\n",
      "Seq is: ACCGTCCAATCAACAAA\n",
      "0 ACCGTCCAATCAACAAA motifs detected in file\n",
      "\n",
      "Seq is: CATCCCGAGTACCATCC\n",
      "0 CATCCCGAGTACCATCC motifs detected in file\n",
      "\n",
      "Seq is: CCATTTTCCTCCCGTTA\n",
      "0 CCATTTTCCTCCCGTTA motifs detected in file\n",
      "\n",
      "Seq is: AAGATCTAGTTTATTAT\n",
      "0 AAGATCTAGTTTATTAT motifs detected in file\n",
      "\n",
      "Seq is: TCTTCCATTATCGATGA\n",
      "0 TCTTCCATTATCGATGA motifs detected in file\n",
      "\n",
      "Seq is: TTCTTCTTAAAATACCT\n",
      "0 TTCTTCTTAAAATACCT motifs detected in file\n",
      "\n",
      "Seq is: CTTTCCCTGCGCCCGCC\n",
      "0 CTTTCCCTGCGCCCGCC motifs detected in file\n",
      "\n",
      "Seq is: CCCTATATCCCAGCTTC\n",
      "0 CCCTATATCCCAGCTTC motifs detected in file\n",
      "\n",
      "Seq is: GGCGTGAGCATTCCCCA\n",
      "0 GGCGTGAGCATTCCCCA motifs detected in file\n",
      "\n",
      "Seq is: CCTAAAAAGATACATCG\n",
      "0 CCTAAAAAGATACATCG motifs detected in file\n",
      "\n",
      "Seq is: GCAACTCAACACCATCA\n",
      "0 GCAACTCAACACCATCA motifs detected in file\n",
      "\n",
      "Seq is: GTTTTACATTATTAATA\n",
      "0 GTTTTACATTATTAATA motifs detected in file\n",
      "\n",
      "Seq is: ATCATCATTCCGATCCT\n",
      "0 ATCATCATTCCGATCCT motifs detected in file\n",
      "\n",
      "Seq is: CCCACAACTAATAATTT\n",
      "0 CCCACAACTAATAATTT motifs detected in file\n",
      "\n",
      "Seq is: TAATTCCGCTTCAAATA\n",
      "0 TAATTCCGCTTCAAATA motifs detected in file\n",
      "\n",
      "Seq is: ATGAGAAAAAACTTACA\n",
      "0 ATGAGAAAAAACTTACA motifs detected in file\n",
      "\n",
      "Seq is: ACAGTTTTGATCAATTA\n",
      "0 ACAGTTTTGATCAATTA motifs detected in file\n",
      "\n",
      "Seq is: ACCTTCAATTGCCATGA\n",
      "0 ACCTTCAATTGCCATGA motifs detected in file\n",
      "\n",
      "Seq is: TAAATATGATTTTCTCA\n",
      "0 TAAATATGATTTTCTCA motifs detected in file\n",
      "\n",
      "Seq is: TTCTGGGAGTAGTTATC\n",
      "0 TTCTGGGAGTAGTTATC motifs detected in file\n",
      "\n",
      "Seq is: TACCTCTGTATGTCTTT\n",
      "0 TACCTCTGTATGTCTTT motifs detected in file\n",
      "\n",
      "Seq is: ATCAGTTTGGTGTAGAT\n",
      "0 ATCAGTTTGGTGTAGAT motifs detected in file\n",
      "\n",
      "Seq is: GTTTTACGGTCATAGTC\n",
      "0 GTTTTACGGTCATAGTC motifs detected in file\n",
      "\n",
      "Seq is: CGCCGCCGAAGCTTCGG\n",
      "0 CGCCGCCGAAGCTTCGG motifs detected in file\n",
      "\n",
      "Seq is: TTTCACTACAAACCTAC\n",
      "0 TTTCACTACAAACCTAC motifs detected in file\n",
      "\n",
      "Seq is: ATTTCACTTTATAATCA\n",
      "0 ATTTCACTTTATAATCA motifs detected in file\n",
      "\n",
      "Seq is: ACCGCCTACTTCGTCTA\n",
      "0 ACCGCCTACTTCGTCTA motifs detected in file\n",
      "\n",
      "Seq is: TCACTCCTTTCAACTCG\n",
      "0 TCACTCCTTTCAACTCG motifs detected in file\n",
      "\n",
      "Seq is: GTTATTACTGCACTTCA\n",
      "0 GTTATTACTGCACTTCA motifs detected in file\n",
      "\n",
      "Seq is: GGCACCTTCGGACACCG\n",
      "0 GGCACCTTCGGACACCG motifs detected in file\n",
      "\n",
      "Seq is: ATGTTTTTTACAACTCG\n",
      "0 ATGTTTTTTACAACTCG motifs detected in file\n",
      "\n",
      "Seq is: GCATCTACGGCAGCCTT\n",
      "0 GCATCTACGGCAGCCTT motifs detected in file\n",
      "\n",
      "Seq is: TACCACTTCCCTCTGAG\n",
      "0 TACCACTTCCCTCTGAG motifs detected in file\n",
      "\n",
      "Seq is: CTTCATGAGACTCCGAA\n",
      "0 CTTCATGAGACTCCGAA motifs detected in file\n",
      "\n",
      "Seq is: CATCCTCCCATTTTATC\n",
      "0 CATCCTCCCATTTTATC motifs detected in file\n",
      "\n",
      "Seq is: TCTGGGTCATTTTAACA\n",
      "0 TCTGGGTCATTTTAACA motifs detected in file\n",
      "\n",
      "Seq is: TTATTCGTCACGAACTT\n",
      "0 TTATTCGTCACGAACTT motifs detected in file\n",
      "\n",
      "Seq is: AATAAACCAAAGCCAAC\n",
      "0 AATAAACCAAAGCCAAC motifs detected in file\n",
      "\n",
      "Seq is: AAAAGATAATCTGATAC\n",
      "0 AAAAGATAATCTGATAC motifs detected in file\n",
      "\n",
      "Seq is: CACTCGAGTCCACTAAC\n",
      "0 CACTCGAGTCCACTAAC motifs detected in file\n",
      "\n",
      "Seq is: TATGAGGACTACGCTCA\n",
      "0 TATGAGGACTACGCTCA motifs detected in file\n",
      "\n",
      "Seq is: TTATGCCTACACAAATC\n",
      "0 TTATGCCTACACAAATC motifs detected in file\n",
      "\n",
      "Seq is: CTCACCCTGAAGATCCC\n",
      "0 CTCACCCTGAAGATCCC motifs detected in file\n",
      "\n",
      "Seq is: CTAAATCGCCCCACTAC\n",
      "0 CTAAATCGCCCCACTAC motifs detected in file\n",
      "\n",
      "Seq is: GGACAACCCCCGGTCAC\n",
      "0 GGACAACCCCCGGTCAC motifs detected in file\n",
      "\n",
      "Seq is: GGGAGGATTAACCCCCC\n",
      "0 GGGAGGATTAACCCCCC motifs detected in file\n",
      "\n",
      "Seq is: ATCCCCGATCCGACCTC\n",
      "0 ATCCCCGATCCGACCTC motifs detected in file\n",
      "\n",
      "Seq is: ACCATTTTCCGAGTCTT\n",
      "0 ACCATTTTCCGAGTCTT motifs detected in file\n",
      "\n",
      "Seq is: TTTAGGACGCTTCTTTT\n",
      "0 TTTAGGACGCTTCTTTT motifs detected in file\n",
      "\n",
      "Seq is: TTTGAAGACTCCATTAT\n",
      "0 TTTGAAGACTCCATTAT motifs detected in file\n",
      "\n",
      "Seq is: TTATCCTAATAGGGCAT\n",
      "0 TTATCCTAATAGGGCAT motifs detected in file\n",
      "\n",
      "Seq is: AGCTTCCGGAAAAACCT\n",
      "0 AGCTTCCGGAAAAACCT motifs detected in file\n",
      "\n",
      "Seq is: GTCCACCACACACCACC\n",
      "0 GTCCACCACACACCACC motifs detected in file\n",
      "\n",
      "Seq is: GGAACCATACACGAAAG\n",
      "0 GGAACCATACACGAAAG motifs detected in file\n",
      "\n",
      "Seq is: AGCACAATGTAGCGCGG\n",
      "0 AGCACAATGTAGCGCGG motifs detected in file\n",
      "\n",
      "Seq is: TAGTAACCATATACCAA\n",
      "0 TAGTAACCATATACCAA motifs detected in file\n",
      "\n",
      "Seq is: TCACACAACCAATCATC\n",
      "0 TCACACAACCAATCATC motifs detected in file\n",
      "\n",
      "Seq is: CGGATCATACTCCTCGC\n",
      "0 CGGATCATACTCCTCGC motifs detected in file\n",
      "\n",
      "Seq is: AATACCTCACCTTCACT\n",
      "0 AATACCTCACCTTCACT motifs detected in file\n",
      "\n",
      "Seq is: TTAGTGTACCGATCCGG\n",
      "0 TTAGTGTACCGATCCGG motifs detected in file\n",
      "\n",
      "Seq is: CCTCCAGTAATCCTCCC\n",
      "0 CCTCCAGTAATCCTCCC motifs detected in file\n",
      "\n",
      "Seq is: GACTCTCCCGGGGACAA\n",
      "0 GACTCTCCCGGGGACAA motifs detected in file\n",
      "\n",
      "Seq is: TCCCCAGTACCCGACCC\n",
      "0 TCCCCAGTACCCGACCC motifs detected in file\n",
      "\n",
      "Seq is: AAAATGATATACTATCC\n",
      "0 AAAATGATATACTATCC motifs detected in file\n",
      "\n",
      "Seq is: GTACACTAACCACCCAG\n",
      "0 GTACACTAACCACCCAG motifs detected in file\n",
      "\n",
      "Seq is: TAATACACAACAGCACG\n",
      "0 TAATACACAACAGCACG motifs detected in file\n",
      "\n",
      "Seq is: TCCATCTCCGAATGATC\n",
      "0 TCCATCTCCGAATGATC motifs detected in file\n",
      "\n",
      "Seq is: TTCACACTTTTGCATCC\n",
      "0 TTCACACTTTTGCATCC motifs detected in file\n",
      "\n",
      "Seq is: GAACCTAATTCCGCTGT\n",
      "0 GAACCTAATTCCGCTGT motifs detected in file\n",
      "\n",
      "Seq is: CGCTAAAGATCCTATCA\n",
      "0 CGCTAAAGATCCTATCA motifs detected in file\n",
      "\n",
      "Seq is: GTCATCTTAATCTTAAC\n",
      "0 GTCATCTTAATCTTAAC motifs detected in file\n",
      "\n",
      "Seq is: ACTTCTACTATTCACAT\n",
      "0 ACTTCTACTATTCACAT motifs detected in file\n",
      "\n",
      "Seq is: CTCCCTTCCAATTACCA\n",
      "0 CTCCCTTCCAATTACCA motifs detected in file\n",
      "\n",
      "Seq is: ACTATAACGATCCCACC\n",
      "0 ACTATAACGATCCCACC motifs detected in file\n",
      "\n",
      "Seq is: GCGAAGGTTAATCCACG\n",
      "0 GCGAAGGTTAATCCACG motifs detected in file\n",
      "\n",
      "Seq is: TACTCATCCACCGGACG\n",
      "0 TACTCATCCACCGGACG motifs detected in file\n",
      "\n",
      "Seq is: TCATTACAATCGCCAAT\n",
      "0 TCATTACAATCGCCAAT motifs detected in file\n",
      "\n",
      "Seq is: CCGCATGCCGGTCCCGA\n",
      "0 CCGCATGCCGGTCCCGA motifs detected in file\n",
      "\n",
      "Seq is: TAACCAACTTACTCATC\n",
      "0 TAACCAACTTACTCATC motifs detected in file\n",
      "\n",
      "Seq is: CGACTACCAAAGCTATT\n",
      "0 CGACTACCAAAGCTATT motifs detected in file\n",
      "\n",
      "Seq is: ATTGATCATACCCCTAT\n",
      "0 ATTGATCATACCCCTAT motifs detected in file\n",
      "\n",
      "Seq is: TCCCCACATCCACACGG\n",
      "0 TCCCCACATCCACACGG motifs detected in file\n",
      "\n",
      "Seq is: AACACCATTCTTCACCC\n",
      "0 AACACCATTCTTCACCC motifs detected in file\n",
      "\n",
      "Seq is: GATCCCGTAAAAATTAG\n",
      "0 GATCCCGTAAAAATTAG motifs detected in file\n",
      "\n",
      "Seq is: AATCTCGCTTTCGGATA\n",
      "0 AATCTCGCTTTCGGATA motifs detected in file\n",
      "\n",
      "Seq is: TTAGTGACACGGGCGAG\n",
      "0 TTAGTGACACGGGCGAG motifs detected in file\n",
      "\n",
      "Seq is: TATTCCCCTACCGGTAC\n",
      "0 TATTCCCCTACCGGTAC motifs detected in file\n",
      "\n",
      "Seq is: CGATCCAAATATCTATC\n",
      "0 CGATCCAAATATCTATC motifs detected in file\n",
      "\n",
      "Seq is: AACCCACCAACCACATT\n",
      "0 AACCCACCAACCACATT motifs detected in file\n",
      "\n",
      "Seq is: TACTCACTCCGTCCTCA\n",
      "0 TACTCACTCCGTCCTCA motifs detected in file\n",
      "\n",
      "Seq is: GGCTCCTCCAATCAACA\n",
      "0 GGCTCCTCCAATCAACA motifs detected in file\n",
      "\n",
      "Seq is: CCGTTATTTTTACTAAT\n",
      "0 CCGTTATTTTTACTAAT motifs detected in file\n",
      "\n",
      "Seq is: TCCTATGATCATATTCT\n",
      "0 TCCTATGATCATATTCT motifs detected in file\n",
      "\n",
      "Seq is: CTAGTCCAAGCAGGAAA\n",
      "0 CTAGTCCAAGCAGGAAA motifs detected in file\n",
      "\n",
      "Seq is: TCACAACACATACCAAT\n",
      "0 TCACAACACATACCAAT motifs detected in file\n",
      "\n",
      "Seq is: AGTAAACAAAACTCCAA\n",
      "0 AGTAAACAAAACTCCAA motifs detected in file\n",
      "\n",
      "Seq is: TCAAACTAATCAGTAAC\n",
      "0 TCAAACTAATCAGTAAC motifs detected in file\n",
      "\n",
      "Seq is: AACCCACCACTAATCAG\n",
      "0 AACCCACCACTAATCAG motifs detected in file\n",
      "\n",
      "Seq is: CCAACAACTACTCTATA\n",
      "0 CCAACAACTACTCTATA motifs detected in file\n",
      "\n",
      "Seq is: AACCTCCACCCCTAGTT\n",
      "0 AACCTCCACCCCTAGTT motifs detected in file\n",
      "\n",
      "Seq is: ATCTCCCCCTTTATCTT\n",
      "0 ATCTCCCCCTTTATCTT motifs detected in file\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Seq is: ACTAGTCATGACGCCGC\n",
      "0 ACTAGTCATGACGCCGC motifs detected in file\n",
      "\n",
      "Seq is: CCATCCGGATCCTAACA\n",
      "0 CCATCCGGATCCTAACA motifs detected in file\n",
      "\n",
      "Seq is: CCCCCGTTACTTACTTC\n",
      "0 CCCCCGTTACTTACTTC motifs detected in file\n",
      "\n",
      "Seq is: GCTTGTCTAAAAGCAAG\n",
      "0 GCTTGTCTAAAAGCAAG motifs detected in file\n",
      "\n",
      "Seq is: TAAAACCAAGAGTCCCA\n",
      "0 TAAAACCAAGAGTCCCA motifs detected in file\n",
      "\n",
      "Seq is: AACAATATTAAAAAATA\n",
      "0 AACAATATTAAAAAATA motifs detected in file\n",
      "\n",
      "Seq is: AAAATACCCGAAACCAC\n",
      "0 AAAATACCCGAAACCAC motifs detected in file\n",
      "\n",
      "Seq is: TCCCTCCATCCACCATC\n",
      "0 TCCCTCCATCCACCATC motifs detected in file\n",
      "\n",
      "Seq is: AAACACAAATTATAAAA\n",
      "0 AAACACAAATTATAAAA motifs detected in file\n",
      "\n",
      "Seq is: ATCAACCCACTACTCCT\n",
      "0 ATCAACCCACTACTCCT motifs detected in file\n",
      "\n",
      "Seq is: TATCACATTCCTCATAC\n",
      "0 TATCACATTCCTCATAC motifs detected in file\n",
      "\n",
      "Seq is: CCCCATTAATACCACCC\n",
      "0 CCCCATTAATACCACCC motifs detected in file\n",
      "\n",
      "Seq is: GGTATGCCATCATAAAT\n",
      "0 GGTATGCCATCATAAAT motifs detected in file\n",
      "\n",
      "Seq is: CAACCCCGTAAAGTGAC\n",
      "0 CAACCCCGTAAAGTGAC motifs detected in file\n",
      "\n",
      "Seq is: ATTTCTCCACAACCAAG\n",
      "0 ATTTCTCCACAACCAAG motifs detected in file\n",
      "\n",
      "Seq is: AGAATTAGAAATTGAAT\n",
      "0 AGAATTAGAAATTGAAT motifs detected in file\n",
      "\n",
      "Seq is: TTTCCAATTACGATTCA\n",
      "0 TTTCCAATTACGATTCA motifs detected in file\n",
      "\n",
      "Seq is: ATCGAAATGTCACCCGA\n",
      "0 ATCGAAATGTCACCCGA motifs detected in file\n",
      "\n",
      "Seq is: GATCTCCCCCATCTCCC\n",
      "0 GATCTCCCCCATCTCCC motifs detected in file\n",
      "\n",
      "Seq is: CCACGATATCCCATTTA\n",
      "0 CCACGATATCCCATTTA motifs detected in file\n",
      "\n",
      "Seq is: TGCCCGGGATAAAGTTT\n",
      "0 TGCCCGGGATAAAGTTT motifs detected in file\n",
      "\n",
      "Seq is: CTAAAAATCCCCTTAAT\n",
      "0 CTAAAAATCCCCTTAAT motifs detected in file\n",
      "\n",
      "Seq is: TAAGATCCTGCTACCCG\n",
      "0 TAAGATCCTGCTACCCG motifs detected in file\n",
      "\n",
      "Seq is: TACTTTGACACCAAACG\n",
      "0 TACTTTGACACCAAACG motifs detected in file\n",
      "\n",
      "Seq is: AGGTGTCTAAAGTCTCG\n",
      "0 AGGTGTCTAAAGTCTCG motifs detected in file\n",
      "\n",
      "Seq is: TAACTGGCATCATATGG\n",
      "0 TAACTGGCATCATATGG motifs detected in file\n",
      "\n",
      "Seq is: GGGCCAGCACATCGCCA\n",
      "0 GGGCCAGCACATCGCCA motifs detected in file\n",
      "\n",
      "Seq is: CTTTCACCAAACCAAAT\n",
      "0 CTTTCACCAAACCAAAT motifs detected in file\n",
      "\n",
      "Seq is: CTGCAGGCCCTTAACGT\n",
      "0 CTGCAGGCCCTTAACGT motifs detected in file\n",
      "\n",
      "Seq is: AGACAAAAATTCGGATT\n",
      "0 AGACAAAAATTCGGATT motifs detected in file\n",
      "\n",
      "Seq is: ACACCCCTGTCGAGTAC\n",
      "0 ACACCCCTGTCGAGTAC motifs detected in file\n",
      "\n",
      "Seq is: TCACGTTCTGCAGAACA\n",
      "0 TCACGTTCTGCAGAACA motifs detected in file\n",
      "\n",
      "Seq is: CTACATTAATAATATGC\n",
      "0 CTACATTAATAATATGC motifs detected in file\n",
      "\n",
      "Seq is: TTACCCCCGAAGTTAGC\n",
      "0 TTACCCCCGAAGTTAGC motifs detected in file\n",
      "\n",
      "Seq is: CCTCATGATGAGCTAAC\n",
      "0 CCTCATGATGAGCTAAC motifs detected in file\n",
      "\n",
      "Seq is: AGTTGCAGTTCCTCAGC\n",
      "0 AGTTGCAGTTCCTCAGC motifs detected in file\n",
      "\n",
      "Seq is: GTCCAGCGGACCAAGAT\n",
      "0 GTCCAGCGGACCAAGAT motifs detected in file\n",
      "\n",
      "Seq is: CCTTATTACCCCCTTCA\n",
      "0 CCTTATTACCCCCTTCA motifs detected in file\n",
      "\n",
      "Seq is: TACATCCTCAACTTCTA\n",
      "0 TACATCCTCAACTTCTA motifs detected in file\n",
      "\n",
      "Seq is: ATCAGGCGGCATCAGCC\n",
      "0 ATCAGGCGGCATCAGCC motifs detected in file\n",
      "\n",
      "Seq is: ACATGAGCATCCAAGTC\n",
      "0 ACATGAGCATCCAAGTC motifs detected in file\n",
      "\n",
      "Seq is: ATGGTAACCACCGGTTA\n",
      "0 ATGGTAACCACCGGTTA motifs detected in file\n",
      "\n",
      "Seq is: ACTAAACTACCATTCCC\n",
      "0 ACTAAACTACCATTCCC motifs detected in file\n",
      "\n",
      "Seq is: TCCCTAGCAACTGGAGC\n",
      "0 TCCCTAGCAACTGGAGC motifs detected in file\n",
      "\n",
      "Seq is: AGACAATACATTTCCTA\n",
      "0 AGACAATACATTTCCTA motifs detected in file\n",
      "\n",
      "Seq is: CGCATCCCTACCCTCCC\n",
      "0 CGCATCCCTACCCTCCC motifs detected in file\n",
      "\n",
      "Seq is: GCTACTCCTGATCCTAC\n",
      "0 GCTACTCCTGATCCTAC motifs detected in file\n",
      "\n",
      "Seq is: TACCGCCCGTCCTATCA\n",
      "0 TACCGCCCGTCCTATCA motifs detected in file\n",
      "\n",
      "Seq is: AGTCTGCCAAAGATAAA\n",
      "0 AGTCTGCCAAAGATAAA motifs detected in file\n",
      "\n",
      "Seq is: GGACTCGCAGACTCTAC\n",
      "0 GGACTCGCAGACTCTAC motifs detected in file\n",
      "\n",
      "Seq is: AATCATAATCAATCAAA\n",
      "0 AATCATAATCAATCAAA motifs detected in file\n",
      "\n",
      "Seq is: ACAACACTCACAATCCT\n",
      "0 ACAACACTCACAATCCT motifs detected in file\n",
      "\n",
      "Seq is: TTTCCCGTATGTCCTGA\n",
      "0 TTTCCCGTATGTCCTGA motifs detected in file\n",
      "\n",
      "Seq is: TCCTTCGTCTATTCCTT\n",
      "0 TCCTTCGTCTATTCCTT motifs detected in file\n",
      "\n",
      "Seq is: TTACTAATACTCCCGCA\n",
      "0 TTACTAATACTCCCGCA motifs detected in file\n",
      "\n",
      "Seq is: CTAGTACTTTCCACTAT\n",
      "0 CTAGTACTTTCCACTAT motifs detected in file\n",
      "\n",
      "Seq is: TCGAGAAGATACTATCC\n",
      "0 TCGAGAAGATACTATCC motifs detected in file\n",
      "\n",
      "Seq is: CCTTCATCGCAGAACAT\n",
      "0 CCTTCATCGCAGAACAT motifs detected in file\n",
      "\n",
      "Seq is: CTGGATGAACGCGACGT\n",
      "0 CTGGATGAACGCGACGT motifs detected in file\n",
      "\n",
      "Seq is: ACACGGTAATTCTATAT\n",
      "0 ACACGGTAATTCTATAT motifs detected in file\n",
      "\n",
      "Seq is: ATCCTAAATCGGATATT\n",
      "0 ATCCTAAATCGGATATT motifs detected in file\n",
      "\n",
      "Seq is: AAATTGAAACTGTTTCA\n",
      "0 AAATTGAAACTGTTTCA motifs detected in file\n",
      "\n",
      "Seq is: ATACTTTACCAAAAAGA\n",
      "0 ATACTTTACCAAAAAGA motifs detected in file\n",
      "\n",
      "Seq is: TTATGGAAAAACTTTTT\n",
      "0 TTATGGAAAAACTTTTT motifs detected in file\n",
      "\n",
      "Seq is: CAGTACCTCCGGTACCC\n",
      "0 CAGTACCTCCGGTACCC motifs detected in file\n",
      "\n",
      "Seq is: CAACCGAACTTTGGTCG\n",
      "0 CAACCGAACTTTGGTCG motifs detected in file\n",
      "\n",
      "Seq is: AAACCCCCCAAGCTAAG\n",
      "0 AAACCCCCCAAGCTAAG motifs detected in file\n",
      "\n",
      "Seq is: GAAGGAAAAAACAGATC\n",
      "0 GAAGGAAAAAACAGATC motifs detected in file\n",
      "\n",
      "Seq is: TAAAATACATATGCCCA\n",
      "0 TAAAATACATATGCCCA motifs detected in file\n",
      "\n",
      "Seq is: AGAAGCTTACACACCAT\n",
      "0 AGAAGCTTACACACCAT motifs detected in file\n",
      "\n",
      "Seq is: CCCACCCCCCGTAGGTA\n",
      "0 CCCACCCCCCGTAGGTA motifs detected in file\n",
      "\n",
      "Seq is: TATCAGTGAGGTCCAAA\n",
      "0 TATCAGTGAGGTCCAAA motifs detected in file\n",
      "\n",
      "Seq is: TACCTCCCAAGAAGATG\n",
      "0 TACCTCCCAAGAAGATG motifs detected in file\n",
      "\n",
      "Seq is: ATAATCCTGAAAAGCGA\n",
      "0 ATAATCCTGAAAAGCGA motifs detected in file\n",
      "\n",
      "Seq is: AGCTTCGCTTCCGAAGA\n",
      "0 AGCTTCGCTTCCGAAGA motifs detected in file\n",
      "\n",
      "Seq is: GTTTAGTACTTTTAATA\n",
      "0 GTTTAGTACTTTTAATA motifs detected in file\n",
      "\n",
      "Seq is: ATTATAATGACGACAAT\n",
      "0 ATTATAATGACGACAAT motifs detected in file\n",
      "\n",
      "Seq is: CTCTTTACTTACTCGGA\n",
      "0 CTCTTTACTTACTCGGA motifs detected in file\n",
      "\n",
      "Seq is: TGTCTACTATCCTACAA\n",
      "0 TGTCTACTATCCTACAA motifs detected in file\n",
      "\n",
      "Seq is: AGTACACCACATACGTA\n",
      "0 AGTACACCACATACGTA motifs detected in file\n",
      "\n",
      "Seq is: GCCCCATCAGGCTCATT\n",
      "0 GCCCCATCAGGCTCATT motifs detected in file\n",
      "\n",
      "Seq is: GCAGCCCCGTAAGGCCT\n",
      "0 GCAGCCCCGTAAGGCCT motifs detected in file\n",
      "\n",
      "Seq is: ATCCGGCTCTTTCACAA\n",
      "0 ATCCGGCTCTTTCACAA motifs detected in file\n",
      "\n",
      "Seq is: CACCCTTCTTTCAATCT\n",
      "0 CACCCTTCTTTCAATCT motifs detected in file\n",
      "\n",
      "Seq is: AAATGCGGCTACTTATA\n",
      "0 AAATGCGGCTACTTATA motifs detected in file\n",
      "\n",
      "Seq is: CTATCACTTTACCTAAA\n",
      "0 CTATCACTTTACCTAAA motifs detected in file\n",
      "\n",
      "Seq is: ACCGCATCCAAACCAGA\n",
      "0 ACCGCATCCAAACCAGA motifs detected in file\n",
      "\n",
      "Seq is: TCCCACATCGGACTCTT\n",
      "0 TCCCACATCGGACTCTT motifs detected in file\n",
      "\n",
      "Seq is: ATCCCCTTTAGTCACTT\n",
      "0 ATCCCCTTTAGTCACTT motifs detected in file\n",
      "\n",
      "Seq is: ACTTCGGAGGATACTAC\n",
      "0 ACTTCGGAGGATACTAC motifs detected in file\n",
      "\n",
      "Seq is: CGTTTATGTCGAGGATA\n",
      "0 CGTTTATGTCGAGGATA motifs detected in file\n",
      "\n",
      "Seq is: ACTATCCTGTATCACCT\n",
      "0 ACTATCCTGTATCACCT motifs detected in file\n",
      "\n",
      "Seq is: TCACCCGATGTTGCATC\n",
      "0 TCACCCGATGTTGCATC motifs detected in file\n",
      "\n",
      "Seq is: ATGCACAGCACATCATG\n",
      "0 ATGCACAGCACATCATG motifs detected in file\n",
      "\n",
      "Seq is: CTACAGATCACTACTCA\n",
      "0 CTACAGATCACTACTCA motifs detected in file\n",
      "\n",
      "Seq is: AACGATTATGTTACGGT\n",
      "0 AACGATTATGTTACGGT motifs detected in file\n",
      "\n",
      "Seq is: CAGTCCGGTGGATGCCA\n",
      "0 CAGTCCGGTGGATGCCA motifs detected in file\n",
      "\n",
      "Seq is: CTTTTCTTTCTACTTAG\n",
      "0 CTTTTCTTTCTACTTAG motifs detected in file\n",
      "\n",
      "Seq is: GATCCCGAGTCTCGTGA\n",
      "0 GATCCCGAGTCTCGTGA motifs detected in file\n",
      "\n",
      "Seq is: CGTCGTCTAGTAAAGTA\n",
      "0 CGTCGTCTAGTAAAGTA motifs detected in file\n",
      "\n",
      "Seq is: TAACGAAGGCACCTCAC\n",
      "0 TAACGAAGGCACCTCAC motifs detected in file\n",
      "\n",
      "Seq is: ACCGCTCAGTCGATTTA\n",
      "0 ACCGCTCAGTCGATTTA motifs detected in file\n",
      "\n",
      "Seq is: TGAAACTGCGGCCACCC\n",
      "0 TGAAACTGCGGCCACCC motifs detected in file\n",
      "\n",
      "Seq is: CTATCGCTACTAATACC\n",
      "0 CTATCGCTACTAATACC motifs detected in file\n",
      "\n",
      "Seq is: ATCGCCTCCACTTTATA\n",
      "0 ATCGCCTCCACTTTATA motifs detected in file\n",
      "\n",
      "Seq is: CGAGCACACAGATGCAG\n",
      "0 CGAGCACACAGATGCAG motifs detected in file\n",
      "\n",
      "Seq is: ATAAGGATGACATTTAT\n",
      "0 ATAAGGATGACATTTAT motifs detected in file\n",
      "\n",
      "Seq is: ATACCACACGAGTGTGC\n",
      "0 ATACCACACGAGTGTGC motifs detected in file\n",
      "\n",
      "Seq is: TATTTGGGATCCTTCGG\n",
      "0 TATTTGGGATCCTTCGG motifs detected in file\n",
      "\n",
      "Seq is: TTAACTATAGTATCGAG\n",
      "0 TTAACTATAGTATCGAG motifs detected in file\n",
      "\n",
      "Seq is: TCTGGTATGGATACATA\n",
      "0 TCTGGTATGGATACATA motifs detected in file\n",
      "\n",
      "Seq is: GGTTTACCAAGAAAAAA\n",
      "0 GGTTTACCAAGAAAAAA motifs detected in file\n",
      "\n",
      "Seq is: AGGCCTCATCATTCAAT\n",
      "0 AGGCCTCATCATTCAAT motifs detected in file\n",
      "\n",
      "Seq is: GTTATACCCTCTAATAA\n",
      "0 GTTATACCCTCTAATAA motifs detected in file\n",
      "\n",
      "Seq is: GGCTTCGGACCATCCTA\n",
      "0 GGCTTCGGACCATCCTA motifs detected in file\n",
      "\n",
      "Seq is: TTCTTATATTTGAAGTC\n",
      "0 TTCTTATATTTGAAGTC motifs detected in file\n",
      "\n",
      "Seq is: CCACTGGCTTTTTAGTC\n",
      "0 CCACTGGCTTTTTAGTC motifs detected in file\n",
      "\n",
      "Seq is: TTATCCACAACCATATC\n",
      "0 TTATCCACAACCATATC motifs detected in file\n",
      "\n",
      "Seq is: TTACCCCAGAGGAGGAG\n",
      "0 TTACCCCAGAGGAGGAG motifs detected in file\n",
      "\n",
      "Seq is: GCCGCCCCAGCTTCTTC\n",
      "0 GCCGCCCCAGCTTCTTC motifs detected in file\n",
      "\n",
      "Seq is: CACCACAACTCCAACGC\n",
      "0 CACCACAACTCCAACGC motifs detected in file\n",
      "\n",
      "Seq is: CAGACAATCATCATATC\n",
      "0 CAGACAATCATCATATC motifs detected in file\n",
      "\n",
      "Seq is: ACTACGGTCGTCGATCC\n",
      "0 ACTACGGTCGTCGATCC motifs detected in file\n",
      "\n",
      "Seq is: TGACCCTCTCTATCCTC\n",
      "0 TGACCCTCTCTATCCTC motifs detected in file\n",
      "\n",
      "Seq is: TTCATCCTGACGACACT\n",
      "0 TTCATCCTGACGACACT motifs detected in file\n",
      "\n",
      "Seq is: AATCCTGCCTAGTCTGC\n",
      "0 AATCCTGCCTAGTCTGC motifs detected in file\n",
      "\n",
      "Seq is: TTCTCCCCGCAAACCAT\n",
      "0 TTCTCCCCGCAAACCAT motifs detected in file\n",
      "\n",
      "Seq is: AACCCAATACCGTCCCC\n",
      "0 AACCCAATACCGTCCCC motifs detected in file\n",
      "\n",
      "Seq is: CAAAATATAACTATTAA\n",
      "0 CAAAATATAACTATTAA motifs detected in file\n",
      "\n",
      "Seq is: CAACACTACTTTAACTA\n",
      "0 CAACACTACTTTAACTA motifs detected in file\n",
      "\n",
      "Seq is: CCGGGGATTCTATCTCC\n",
      "0 CCGGGGATTCTATCTCC motifs detected in file\n",
      "\n",
      "Seq is: TCTGTGGACGATCCACA\n",
      "0 TCTGTGGACGATCCACA motifs detected in file\n",
      "\n",
      "Seq is: TTCCTCTTCTACCAATC\n",
      "0 TTCCTCTTCTACCAATC motifs detected in file\n",
      "\n",
      "Seq is: CAGATGCCTCCGAGGTC\n",
      "0 CAGATGCCTCCGAGGTC motifs detected in file\n",
      "\n",
      "Seq is: CCACCCTCATCAAGGGA\n",
      "0 CCACCCTCATCAAGGGA motifs detected in file\n",
      "\n",
      "Seq is: CGATTCCCTCCCATCTG\n",
      "0 CGATTCCCTCCCATCTG motifs detected in file\n",
      "\n",
      "Seq is: ACAAGTTGGACAAGGAC\n",
      "0 ACAAGTTGGACAAGGAC motifs detected in file\n",
      "\n",
      "Seq is: GAGGCCGGAGGTGATAT\n",
      "0 GAGGCCGGAGGTGATAT motifs detected in file\n",
      "\n",
      "Seq is: CGTCTACGCTCGTCCTC\n",
      "0 CGTCTACGCTCGTCCTC motifs detected in file\n",
      "\n",
      "Seq is: ATCCTCTCTCCCTCCAT\n",
      "0 ATCCTCTCTCCCTCCAT motifs detected in file\n",
      "\n",
      "Seq is: TCTCAGTCTTCGAATAC\n",
      "0 TCTCAGTCTTCGAATAC motifs detected in file\n",
      "\n",
      "Seq is: AACAAATACGCCCCTTT\n",
      "0 AACAAATACGCCCCTTT motifs detected in file\n",
      "\n",
      "Seq is: GCGGTATAGCCCCCGTG\n",
      "0 GCGGTATAGCCCCCGTG motifs detected in file\n",
      "\n",
      "Seq is: GCTAATAATCCCCTTGA\n",
      "0 GCTAATAATCCCCTTGA motifs detected in file\n",
      "\n",
      "Seq is: TCAGTCAACGGTTTCGG\n",
      "0 TCAGTCAACGGTTTCGG motifs detected in file\n",
      "\n",
      "Seq is: AGGCTAATACTACCCAT\n",
      "0 AGGCTAATACTACCCAT motifs detected in file\n",
      "\n",
      "Seq is: AATGATACTTCTTCTAA\n",
      "0 AATGATACTTCTTCTAA motifs detected in file\n",
      "\n",
      "Seq is: TAATGTTTACGTACCCG\n",
      "0 TAATGTTTACGTACCCG motifs detected in file\n",
      "\n",
      "Seq is: ACACTGCTATTGCAACA\n",
      "0 ACACTGCTATTGCAACA motifs detected in file\n",
      "\n",
      "Seq is: TCTACACCAGCAATGGA\n",
      "0 TCTACACCAGCAATGGA motifs detected in file\n",
      "\n",
      "Seq is: TCTTCCAACGGACCGAC\n",
      "0 TCTTCCAACGGACCGAC motifs detected in file\n",
      "\n",
      "Seq is: CGGGTCGAGCCGAGCTT\n",
      "0 CGGGTCGAGCCGAGCTT motifs detected in file\n",
      "\n",
      "Seq is: ATTCCTCCGAATCTCGA\n",
      "0 ATTCCTCCGAATCTCGA motifs detected in file\n",
      "\n",
      "Seq is: CACGGATCCTGAGGTCG\n",
      "0 CACGGATCCTGAGGTCG motifs detected in file\n",
      "\n",
      "Seq is: AGTACGCGGCTTATTAT\n",
      "0 AGTACGCGGCTTATTAT motifs detected in file\n",
      "\n",
      "Seq is: CCATATCACAAGGTTAC\n",
      "0 CCATATCACAAGGTTAC motifs detected in file\n",
      "\n",
      "Seq is: AGAAACACCAAACATCT\n",
      "0 AGAAACACCAAACATCT motifs detected in file\n",
      "\n",
      "Seq is: CTTATCAGTTGCCAGCC\n",
      "0 CTTATCAGTTGCCAGCC motifs detected in file\n",
      "\n",
      "Seq is: GCTTGTAGTCACCCCCA\n",
      "0 GCTTGTAGTCACCCCCA motifs detected in file\n",
      "\n",
      "Seq is: CTCCATTTTACCGACTC\n",
      "0 CTCCATTTTACCGACTC motifs detected in file\n",
      "\n",
      "Seq is: ACTTCGTAACCTGACAT\n",
      "0 ACTTCGTAACCTGACAT motifs detected in file\n",
      "\n",
      "Seq is: TTAGATTTCTGTCCCCA\n",
      "0 TTAGATTTCTGTCCCCA motifs detected in file\n",
      "\n",
      "Seq is: ATCCGGAGAAAAATGGT\n",
      "0 ATCCGGAGAAAAATGGT motifs detected in file\n",
      "\n",
      "Seq is: CGAGGCTCCACTAAAAG\n",
      "0 CGAGGCTCCACTAAAAG motifs detected in file\n",
      "\n",
      "Seq is: TATAACTTAACGTTTAA\n",
      "0 TATAACTTAACGTTTAA motifs detected in file\n",
      "\n",
      "Seq is: GCTTCTTCGTCGAAGTT\n",
      "0 GCTTCTTCGTCGAAGTT motifs detected in file\n",
      "\n",
      "Seq is: TGGACGGCCCCGAAGAG\n",
      "0 TGGACGGCCCCGAAGAG motifs detected in file\n",
      "\n",
      "Seq is: GGCGGAAAAAAGGGCCG\n",
      "0 GGCGGAAAAAAGGGCCG motifs detected in file\n",
      "\n",
      "Seq is: CCGCCCTCTTCATCTAA\n",
      "0 CCGCCCTCTTCATCTAA motifs detected in file\n",
      "\n",
      "Seq is: CTTCGGTCAACTAATCC\n",
      "0 CTTCGGTCAACTAATCC motifs detected in file\n",
      "\n",
      "Seq is: CACGAATCGACAATTGA\n",
      "0 CACGAATCGACAATTGA motifs detected in file\n",
      "\n",
      "Seq is: TTCACAAACACCCAAAT\n",
      "0 TTCACAAACACCCAAAT motifs detected in file\n",
      "\n",
      "Seq is: TCAGGGTAACCAGATCA\n",
      "0 TCAGGGTAACCAGATCA motifs detected in file\n",
      "\n",
      "Seq is: TTCCCGAATCGAATTAA\n",
      "0 TTCCCGAATCGAATTAA motifs detected in file\n",
      "\n",
      "Seq is: TTTCACCGACTAAACGC\n",
      "0 TTTCACCGACTAAACGC motifs detected in file\n",
      "\n",
      "Seq is: AAGTCAACTACGTCTCA\n",
      "0 AAGTCAACTACGTCTCA motifs detected in file\n",
      "\n",
      "Seq is: CCCCAAAACGTCAGGAA\n",
      "0 CCCCAAAACGTCAGGAA motifs detected in file\n",
      "\n",
      "Seq is: TCGACAATGTCTTTAAT\n",
      "0 TCGACAATGTCTTTAAT motifs detected in file\n",
      "\n",
      "Seq is: TCATAACGTTGAATGAC\n",
      "0 TCATAACGTTGAATGAC motifs detected in file\n",
      "\n",
      "Seq is: TCCCGAAACTTCCGAGA\n",
      "0 TCCCGAAACTTCCGAGA motifs detected in file\n",
      "\n",
      "Seq is: ACCAGACATAAATTGGA\n",
      "0 ACCAGACATAAATTGGA motifs detected in file\n",
      "\n",
      "Seq is: TTTAAAGATATTCTAAT\n",
      "0 TTTAAAGATATTCTAAT motifs detected in file\n",
      "\n",
      "Seq is: AATCATATTTTCCCCTC\n",
      "0 AATCATATTTTCCCCTC motifs detected in file\n",
      "\n",
      "Seq is: TATCCATCCTCATCGCA\n",
      "0 TATCCATCCTCATCGCA motifs detected in file\n",
      "\n",
      "Seq is: CCATTCCCGCTACTCAC\n",
      "0 CCATTCCCGCTACTCAC motifs detected in file\n",
      "\n",
      "Seq is: ACCCCTCCTTACCCCAC\n",
      "0 ACCCCTCCTTACCCCAC motifs detected in file\n",
      "\n",
      "Seq is: CCAAAACATACAAGTTT\n",
      "0 CCAAAACATACAAGTTT motifs detected in file\n",
      "\n",
      "Seq is: GACAGTAAAATAAAAAT\n",
      "0 GACAGTAAAATAAAAAT motifs detected in file\n",
      "\n",
      "Seq is: GCAACAATCTATACCCC\n",
      "0 GCAACAATCTATACCCC motifs detected in file\n",
      "\n",
      "Seq is: TCATCACACTAACTCCA\n",
      "0 TCATCACACTAACTCCA motifs detected in file\n",
      "\n",
      "Seq is: CCTCATCTAATCCGCAT\n",
      "0 CCTCATCTAATCCGCAT motifs detected in file\n",
      "\n",
      "Seq is: CCATCTTCATCTCCAAT\n",
      "0 CCATCTTCATCTCCAAT motifs detected in file\n",
      "\n",
      "Seq is: TCCTCCCACTACCACCG\n",
      "0 TCCTCCCACTACCACCG motifs detected in file\n",
      "\n",
      "Seq is: ATACTACCACCCCTACT\n",
      "0 ATACTACCACCCCTACT motifs detected in file\n",
      "\n",
      "Seq is: ACTCCGATAACAAAAAA\n",
      "0 ACTCCGATAACAAAAAA motifs detected in file\n",
      "\n",
      "Seq is: CACTTAAGAAGCTATTA\n",
      "0 CACTTAAGAAGCTATTA motifs detected in file\n",
      "\n",
      "Seq is: CCGGGTAAACCCGTTTT\n",
      "0 CCGGGTAAACCCGTTTT motifs detected in file\n",
      "\n",
      "Seq is: TCGGCCAATCGCCCCCG\n",
      "0 TCGGCCAATCGCCCCCG motifs detected in file\n",
      "\n",
      "Seq is: TCCGGAGGATCCCTCTC\n",
      "0 TCCGGAGGATCCCTCTC motifs detected in file\n",
      "\n",
      "Seq is: CTCCCACCTACCTTAAT\n",
      "0 CTCCCACCTACCTTAAT motifs detected in file\n",
      "\n",
      "Seq is: TCCCACAATCAGTACAA\n",
      "0 TCCCACAATCAGTACAA motifs detected in file\n",
      "\n",
      "Seq is: TCGAACAAAGTCCACGC\n",
      "0 TCGAACAAAGTCCACGC motifs detected in file\n",
      "\n",
      "Seq is: TCTATCATCATCCCAGC\n",
      "0 TCTATCATCATCCCAGC motifs detected in file\n",
      "\n",
      "Seq is: ACCACGACCTCAAATTC\n",
      "0 ACCACGACCTCAAATTC motifs detected in file\n",
      "\n",
      "Seq is: AACTCATCATCCTTACG\n",
      "0 AACTCATCATCCTTACG motifs detected in file\n",
      "\n",
      "Seq is: CCATCATCAATCCTATT\n",
      "0 CCATCATCAATCCTATT motifs detected in file\n",
      "\n",
      "Seq is: ATATTTATCAATTTAAT\n",
      "0 ATATTTATCAATTTAAT motifs detected in file\n",
      "\n",
      "Seq is: TCTTACCAATACAATCC\n",
      "0 TCTTACCAATACAATCC motifs detected in file\n",
      "\n",
      "Seq is: CAACATGCCATCTTGAC\n",
      "0 CAACATGCCATCTTGAC motifs detected in file\n",
      "\n",
      "Seq is: GATAATAAGTAGGATAC\n",
      "0 GATAATAAGTAGGATAC motifs detected in file\n",
      "\n",
      "Seq is: ACCCATTAACTCCTCAT\n",
      "0 ACCCATTAACTCCTCAT motifs detected in file\n",
      "\n",
      "Seq is: ACGATTCTAAAACGCAT\n",
      "0 ACGATTCTAAAACGCAT motifs detected in file\n",
      "\n",
      "Seq is: CGACCCAAACCAAATTA\n",
      "0 CGACCCAAACCAAATTA motifs detected in file\n",
      "\n",
      "Seq is: GGTGGAGTTGACGGACG\n",
      "0 GGTGGAGTTGACGGACG motifs detected in file\n",
      "\n",
      "Seq is: ATACTACCTATTCTAAC\n",
      "0 ATACTACCTATTCTAAC motifs detected in file\n",
      "\n",
      "Seq is: TCTCTCACTCCTCTTCC\n",
      "0 TCTCTCACTCCTCTTCC motifs detected in file\n",
      "\n",
      "Seq is: GAATGCAAATCACTCCC\n",
      "0 GAATGCAAATCACTCCC motifs detected in file\n",
      "\n",
      "Seq is: TCTCTAAACCATATACT\n",
      "0 TCTCTAAACCATATACT motifs detected in file\n",
      "\n",
      "Seq is: AACTCTACCCCCGATCA\n",
      "0 AACTCTACCCCCGATCA motifs detected in file\n",
      "\n",
      "Seq is: AAAACAGTACACTCTTC\n",
      "0 AAAACAGTACACTCTTC motifs detected in file\n",
      "\n",
      "Seq is: TTCGTCCGGCCTACAGT\n",
      "0 TTCGTCCGGCCTACAGT motifs detected in file\n",
      "\n",
      "Seq is: CTCCCCACGGAACCCAT\n",
      "0 CTCCCCACGGAACCCAT motifs detected in file\n",
      "\n",
      "Seq is: TGGAGACCCTGAGTCTT\n",
      "0 TGGAGACCCTGAGTCTT motifs detected in file\n",
      "\n",
      "Seq is: CACTTTCCCCCGATAAG\n",
      "0 CACTTTCCCCCGATAAG motifs detected in file\n",
      "\n",
      "Seq is: GATCAAAATAACGATAT\n",
      "0 GATCAAAATAACGATAT motifs detected in file\n",
      "\n",
      "Seq is: CGATAATACTAATAATT\n",
      "0 CGATAATACTAATAATT motifs detected in file\n",
      "\n",
      "Seq is: ACTACTCATAACTAACC\n",
      "0 ACTACTCATAACTAACC motifs detected in file\n",
      "\n",
      "Seq is: ATCATAACCAATACCAA\n",
      "0 ATCATAACCAATACCAA motifs detected in file\n",
      "\n",
      "Seq is: GTAACAGGCCTCTCATA\n",
      "0 GTAACAGGCCTCTCATA motifs detected in file\n",
      "\n",
      "Seq is: TAACAACTTCTCCTATC\n",
      "0 TAACAACTTCTCCTATC motifs detected in file\n",
      "\n",
      "Seq is: GATAATCTTCCTAATAC\n",
      "0 GATAATCTTCCTAATAC motifs detected in file\n",
      "\n",
      "Seq is: CTACGCCAACGAACGCA\n",
      "0 CTACGCCAACGAACGCA motifs detected in file\n",
      "\n",
      "Seq is: CTCCTTTATGAACTACC\n",
      "0 CTCCTTTATGAACTACC motifs detected in file\n",
      "\n",
      "Seq is: GTCGAAGACACCTTGCT\n",
      "0 GTCGAAGACACCTTGCT motifs detected in file\n",
      "\n",
      "Seq is: CCCAAATAAAAAAACCA\n",
      "0 CCCAAATAAAAAAACCA motifs detected in file\n",
      "\n",
      "Seq is: ATCTTGACCTTATTTTC\n",
      "0 ATCTTGACCTTATTTTC motifs detected in file\n",
      "\n",
      "Seq is: GATCGTACAAATAAAGA\n",
      "0 GATCGTACAAATAAAGA motifs detected in file\n",
      "\n",
      "Seq is: TCCGGATGAGTCCATTT\n",
      "0 TCCGGATGAGTCCATTT motifs detected in file\n",
      "\n",
      "Seq is: TTTAGTCACGCTCGAAT\n",
      "0 TTTAGTCACGCTCGAAT motifs detected in file\n",
      "\n",
      "Seq is: CGCGACACTACTCACAC\n",
      "0 CGCGACACTACTCACAC motifs detected in file\n",
      "\n",
      "Seq is: GGACGTTTCTACCATCT\n",
      "0 GGACGTTTCTACCATCT motifs detected in file\n",
      "\n",
      "Seq is: CATCTACTGCCCAACCC\n",
      "0 CATCTACTGCCCAACCC motifs detected in file\n",
      "\n",
      "Seq is: GGTCCCCTAATTAATCA\n",
      "0 GGTCCCCTAATTAATCA motifs detected in file\n",
      "\n",
      "Seq is: TGCCCTTCCCATATTGG\n",
      "0 TGCCCTTCCCATATTGG motifs detected in file\n",
      "\n",
      "Seq is: TTGTAAAAGCCCCATAC\n",
      "0 TTGTAAAAGCCCCATAC motifs detected in file\n",
      "\n",
      "Seq is: CCGGGCTATCGAATAAA\n",
      "0 CCGGGCTATCGAATAAA motifs detected in file\n",
      "\n",
      "Seq is: TCGACTGGAATGAAATC\n",
      "0 TCGACTGGAATGAAATC motifs detected in file\n",
      "\n",
      "Seq is: CTACCCCACACTATCCA\n",
      "0 CTACCCCACACTATCCA motifs detected in file\n",
      "\n",
      "Seq is: CCGTGCCTCTTAAAACC\n",
      "0 CCGTGCCTCTTAAAACC motifs detected in file\n",
      "\n",
      "Seq is: TAAGAGTCCCTACCCAA\n",
      "0 TAAGAGTCCCTACCCAA motifs detected in file\n",
      "\n",
      "Seq is: GCTAAGAGTATCAGGAT\n",
      "0 GCTAAGAGTATCAGGAT motifs detected in file\n",
      "\n",
      "Seq is: CTTTATTCCCCCAAATT\n",
      "0 CTTTATTCCCCCAAATT motifs detected in file\n",
      "\n",
      "Seq is: CGAGGATAATAAATGAG\n",
      "0 CGAGGATAATAAATGAG motifs detected in file\n",
      "\n",
      "Seq is: ATAGTTTCATTGAGAAA\n",
      "0 ATAGTTTCATTGAGAAA motifs detected in file\n",
      "\n",
      "Seq is: ATAGTCTGTATAAAGAA\n",
      "0 ATAGTCTGTATAAAGAA motifs detected in file\n",
      "\n",
      "Seq is: TCCAAACTCCCCCTTAC\n",
      "0 TCCAAACTCCCCCTTAC motifs detected in file\n",
      "\n",
      "Seq is: GACCTCTAACATTACCC\n",
      "0 GACCTCTAACATTACCC motifs detected in file\n",
      "\n",
      "Seq is: ATACCTCTGTATAGTAT\n",
      "0 ATACCTCTGTATAGTAT motifs detected in file\n",
      "\n",
      "Seq is: ATTCATTACGATCCCAC\n",
      "0 ATTCATTACGATCCCAC motifs detected in file\n",
      "\n",
      "Seq is: TCACCATCCTTCAAAAA\n",
      "0 TCACCATCCTTCAAAAA motifs detected in file\n",
      "\n",
      "Seq is: AGTATCCTCCACATACT\n",
      "0 AGTATCCTCCACATACT motifs detected in file\n",
      "\n",
      "Seq is: CAACCAGCATCGCCTTA\n",
      "0 CAACCAGCATCGCCTTA motifs detected in file\n",
      "\n",
      "Seq is: GCCCCCATACGACAAGC\n",
      "0 GCCCCCATACGACAAGC motifs detected in file\n",
      "\n",
      "Seq is: TTAAGTATTCTTGTCCC\n",
      "0 TTAAGTATTCTTGTCCC motifs detected in file\n",
      "\n",
      "Seq is: TCCAATCTTCATCCCAG\n",
      "0 TCCAATCTTCATCCCAG motifs detected in file\n",
      "\n",
      "Seq is: AACCACTGTTTTATACA\n",
      "0 AACCACTGTTTTATACA motifs detected in file\n",
      "\n",
      "Seq is: ACACATCTCAAGTCCCC\n",
      "0 ACACATCTCAAGTCCCC motifs detected in file\n",
      "\n",
      "Seq is: TCTCACGCAGTATACAA\n",
      "0 TCTCACGCAGTATACAA motifs detected in file\n",
      "\n",
      "Seq is: CAAGGATCCTTCTAACA\n",
      "0 CAAGGATCCTTCTAACA motifs detected in file\n",
      "\n",
      "Seq is: TCACCACTCCCACAAAT\n",
      "0 TCACCACTCCCACAAAT motifs detected in file\n",
      "\n",
      "Seq is: AATATTATTACAAACAC\n",
      "0 AATATTATTACAAACAC motifs detected in file\n",
      "\n",
      "Seq is: ATAAGCCGATACTTCTT\n",
      "0 ATAAGCCGATACTTCTT motifs detected in file\n",
      "\n",
      "Seq is: ATCCCGCTTCCCCGGAC\n",
      "0 ATCCCGCTTCCCCGGAC motifs detected in file\n",
      "\n",
      "Seq is: GCCGCATAAGCTACAAC\n",
      "0 GCCGCATAAGCTACAAC motifs detected in file\n",
      "\n",
      "Seq is: TTCGGACTCTGATCAAG\n",
      "0 TTCGGACTCTGATCAAG motifs detected in file\n",
      "\n",
      "Seq is: CCTGAGGGGAAGCCGTT\n",
      "0 CCTGAGGGGAAGCCGTT motifs detected in file\n",
      "\n",
      "Seq is: CCAGCTTCCCCCAAGCC\n",
      "0 CCAGCTTCCCCCAAGCC motifs detected in file\n",
      "\n",
      "Seq is: AACCAGAGACGATCACA\n",
      "0 AACCAGAGACGATCACA motifs detected in file\n",
      "\n",
      "Seq is: CCTCTATTTAGTATAAT\n",
      "0 CCTCTATTTAGTATAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCGGTTCCCAGTACTA\n",
      "0 ACCGGTTCCCAGTACTA motifs detected in file\n",
      "\n",
      "Seq is: CCGTCCTCATTAGTCTC\n",
      "0 CCGTCCTCATTAGTCTC motifs detected in file\n",
      "\n",
      "Seq is: CACAAGAACACAACACT\n",
      "0 CACAAGAACACAACACT motifs detected in file\n",
      "\n",
      "Seq is: ATTCCCACCTCTCCAAT\n",
      "0 ATTCCCACCTCTCCAAT motifs detected in file\n",
      "\n",
      "Seq is: TTCCTCGGTGAATAATC\n",
      "0 TTCCTCGGTGAATAATC motifs detected in file\n",
      "\n",
      "Seq is: ATTACAACTATCATCTT\n",
      "0 ATTACAACTATCATCTT motifs detected in file\n",
      "\n",
      "Seq is: ACTACCGATCCCACTGA\n",
      "0 ACTACCGATCCCACTGA motifs detected in file\n",
      "\n",
      "Seq is: AGTATACTCTAACAAAC\n",
      "0 AGTATACTCTAACAAAC motifs detected in file\n",
      "\n",
      "Seq is: CCGATGACGAGCGTCAC\n",
      "0 CCGATGACGAGCGTCAC motifs detected in file\n",
      "\n",
      "Seq is: GCGGCTAGTCCCGCATC\n",
      "0 GCGGCTAGTCCCGCATC motifs detected in file\n",
      "\n",
      "Seq is: AAACTCAAACTACGAGT\n",
      "0 AAACTCAAACTACGAGT motifs detected in file\n",
      "\n",
      "Seq is: GGGACTAGTCTCCTAAC\n",
      "0 GGGACTAGTCTCCTAAC motifs detected in file\n",
      "\n",
      "Seq is: TCATTTGCCGATCCGAT\n",
      "0 TCATTTGCCGATCCGAT motifs detected in file\n",
      "\n",
      "Seq is: CTCCACCGATCTTATTT\n",
      "0 CTCCACCGATCTTATTT motifs detected in file\n",
      "\n",
      "Seq is: ATCCTCCGGATCCAACT\n",
      "0 ATCCTCCGGATCCAACT motifs detected in file\n",
      "\n",
      "Seq is: CCAACTGGTCCCCCAAC\n",
      "0 CCAACTGGTCCCCCAAC motifs detected in file\n",
      "\n",
      "Seq is: CCATACCCCTCCCCCCA\n",
      "0 CCATACCCCTCCCCCCA motifs detected in file\n",
      "\n",
      "Seq is: AGTATCATCTTCTCGCT\n",
      "0 AGTATCATCTTCTCGCT motifs detected in file\n",
      "\n",
      "Seq is: ACCACTCTCGATTCCAG\n",
      "0 ACCACTCTCGATTCCAG motifs detected in file\n",
      "\n",
      "Seq is: CCCCGCCACTACATCTC\n",
      "0 CCCCGCCACTACATCTC motifs detected in file\n",
      "\n",
      "Seq is: CCACTACCATCTACACC\n",
      "0 CCACTACCATCTACACC motifs detected in file\n",
      "\n",
      "Seq is: GCCCAAAATCCCCGAGA\n",
      "0 GCCCAAAATCCCCGAGA motifs detected in file\n",
      "\n",
      "Seq is: AACCACTTCTCAAAATA\n",
      "0 AACCACTTCTCAAAATA motifs detected in file\n",
      "\n",
      "Seq is: CCGCAGTCGCTTCCCAA\n",
      "0 CCGCAGTCGCTTCCCAA motifs detected in file\n",
      "\n",
      "Seq is: CATCATCGGGCATCCCC\n",
      "0 CATCATCGGGCATCCCC motifs detected in file\n",
      "\n",
      "Seq is: GGATGTTGCAACCCCGG\n",
      "0 GGATGTTGCAACCCCGG motifs detected in file\n",
      "\n",
      "Seq is: AAACGCATCAACATATA\n",
      "0 AAACGCATCAACATATA motifs detected in file\n",
      "\n",
      "Seq is: TCGGATCTTAAAAAGCA\n",
      "0 TCGGATCTTAAAAAGCA motifs detected in file\n",
      "\n",
      "Seq is: AGCCATTCGTAATCCTT\n",
      "0 AGCCATTCGTAATCCTT motifs detected in file\n",
      "\n",
      "Seq is: ACGGTAACGCTAATCTT\n",
      "0 ACGGTAACGCTAATCTT motifs detected in file\n",
      "\n",
      "Seq is: ACCCATGTTACTCCTCA\n",
      "0 ACCCATGTTACTCCTCA motifs detected in file\n",
      "\n",
      "Seq is: TCCTCCAACCGGTACCC\n",
      "0 TCCTCCAACCGGTACCC motifs detected in file\n",
      "\n",
      "Seq is: ATACAACAATTCTTCTC\n",
      "0 ATACAACAATTCTTCTC motifs detected in file\n",
      "\n",
      "Seq is: CTTAACTTGGAGACTGA\n",
      "0 CTTAACTTGGAGACTGA motifs detected in file\n",
      "\n",
      "Seq is: CATTTCAAAATTCAAAA\n",
      "0 CATTTCAAAATTCAAAA motifs detected in file\n",
      "\n",
      "Seq is: TACGCTAATGGCCCGAG\n",
      "0 TACGCTAATGGCCCGAG motifs detected in file\n",
      "\n",
      "Seq is: ACGGTAGAATTGTTTGG\n",
      "0 ACGGTAGAATTGTTTGG motifs detected in file\n",
      "\n",
      "Seq is: GACAAGAACCCACCCAC\n",
      "0 GACAAGAACCCACCCAC motifs detected in file\n",
      "\n",
      "Seq is: ACCCATATTATGATTCA\n",
      "0 ACCCATATTATGATTCA motifs detected in file\n",
      "\n",
      "Seq is: ACTCTACTATAGTAAAT\n",
      "0 ACTCTACTATAGTAAAT motifs detected in file\n",
      "\n",
      "Seq is: GCCCCCTTCCGCGAAAC\n",
      "0 GCCCCCTTCCGCGAAAC motifs detected in file\n",
      "\n",
      "Seq is: ACTTCATCCGGAATAAA\n",
      "0 ACTTCATCCGGAATAAA motifs detected in file\n",
      "\n",
      "Seq is: GAGAACAGGAAAGCATG\n",
      "0 GAGAACAGGAAAGCATG motifs detected in file\n",
      "\n",
      "Seq is: TCCCTCCTTAAACTTNC\n",
      "0 TCCCTCCTTAAACTTNC motifs detected in file\n",
      "\n",
      "Seq is: ATCTATCTTTGGCTGGA\n",
      "0 ATCTATCTTTGGCTGGA motifs detected in file\n",
      "\n",
      "Seq is: CCTAATGAGGCCAGACT\n",
      "0 CCTAATGAGGCCAGACT motifs detected in file\n",
      "\n",
      "Seq is: TGAGTCTAGTGCATCCT\n",
      "0 TGAGTCTAGTGCATCCT motifs detected in file\n",
      "\n",
      "Seq is: GAAATTAGCAACTTGTT\n",
      "0 GAAATTAGCAACTTGTT motifs detected in file\n",
      "\n",
      "Seq is: TGCTTGGAAATTATCGC\n",
      "0 TGCTTGGAAATTATCGC motifs detected in file\n",
      "\n",
      "Seq is: CGACGTGGTAGCCCTAC\n",
      "0 CGACGTGGTAGCCCTAC motifs detected in file\n",
      "\n",
      "Seq is: AGGACTAGGTTGTAGCT\n",
      "0 AGGACTAGGTTGTAGCT motifs detected in file\n",
      "\n",
      "Seq is: CCAGCATTTGGGATAAC\n",
      "0 CCAGCATTTGGGATAAC motifs detected in file\n",
      "\n",
      "Seq is: AACTATACCTGAGATCT\n",
      "0 AACTATACCTGAGATCT motifs detected in file\n",
      "\n",
      "Seq is: TATCCTAACGCGACAAT\n",
      "0 TATCCTAACGCGACAAT motifs detected in file\n",
      "\n",
      "Seq is: AGGGATCCCATTGAACA\n",
      "0 AGGGATCCCATTGAACA motifs detected in file\n",
      "\n",
      "Seq is: AGGCAACCAGTTCAATA\n",
      "0 AGGCAACCAGTTCAATA motifs detected in file\n",
      "\n",
      "Seq is: ACCTAGTTAACTCATAT\n",
      "0 ACCTAGTTAACTCATAT motifs detected in file\n",
      "\n",
      "Seq is: CATCAAGCGAAACTGAC\n",
      "0 CATCAAGCGAAACTGAC motifs detected in file\n",
      "\n",
      "Seq is: CACTTCAGAATCGTACA\n",
      "0 CACTTCAGAATCGTACA motifs detected in file\n",
      "\n",
      "Seq is: TGACGAGCCTCCAACCC\n",
      "0 TGACGAGCCTCCAACCC motifs detected in file\n",
      "\n",
      "Seq is: AAGACGAGGCTCCAGCG\n",
      "0 AAGACGAGGCTCCAGCG motifs detected in file\n",
      "\n",
      "Seq is: GGGTTGGCTTTAAAAAT\n",
      "0 GGGTTGGCTTTAAAAAT motifs detected in file\n",
      "\n",
      "Seq is: TACGTCCAAACCATCAA\n",
      "0 TACGTCCAAACCATCAA motifs detected in file\n",
      "\n",
      "Seq is: ATCCTGGACACCCAAAC\n",
      "0 ATCCTGGACACCCAAAC motifs detected in file\n",
      "\n",
      "Seq is: AATCCATGACAAACGTA\n",
      "0 AATCCATGACAAACGTA motifs detected in file\n",
      "\n",
      "Seq is: ATTATTTAATTTCGAGG\n",
      "0 ATTATTTAATTTCGAGG motifs detected in file\n",
      "\n",
      "Seq is: TATCCCAGAAGAGCAGA\n",
      "0 TATCCCAGAAGAGCAGA motifs detected in file\n",
      "\n",
      "Seq is: ACGACACAATACGGGCG\n",
      "0 ACGACACAATACGGGCG motifs detected in file\n",
      "\n",
      "Seq is: GAGAAGTGCCCGTCCAG\n",
      "0 GAGAAGTGCCCGTCCAG motifs detected in file\n",
      "\n",
      "Seq is: TTAAAGTGACCAATTTT\n",
      "0 TTAAAGTGACCAATTTT motifs detected in file\n",
      "\n",
      "Seq is: CATTCTCTGTCGACTTG\n",
      "0 CATTCTCTGTCGACTTG motifs detected in file\n",
      "\n",
      "Seq is: GGAGCACCTCGGTAAGT\n",
      "0 GGAGCACCTCGGTAAGT motifs detected in file\n",
      "\n",
      "Seq is: ATGTCCAGGGATAAATT\n",
      "0 ATGTCCAGGGATAAATT motifs detected in file\n",
      "\n",
      "Seq is: CCTTGTTCACTAATACG\n",
      "0 CCTTGTTCACTAATACG motifs detected in file\n",
      "\n",
      "Seq is: ATGGAAACGTGCCAATC\n",
      "0 ATGGAAACGTGCCAATC motifs detected in file\n",
      "\n",
      "Seq is: CCATGGCGCCGGCAATT\n",
      "0 CCATGGCGCCGGCAATT motifs detected in file\n",
      "\n",
      "Seq is: TGTACACAGTGACCCGT\n",
      "0 TGTACACAGTGACCCGT motifs detected in file\n",
      "\n",
      "Seq is: CCGCCACGGAGATTATG\n",
      "0 CCGCCACGGAGATTATG motifs detected in file\n",
      "\n",
      "Seq is: ACCACTACGATCTCCAC\n",
      "0 ACCACTACGATCTCCAC motifs detected in file\n",
      "\n",
      "Seq is: TACAAAAACCATTTGTC\n",
      "0 TACAAAAACCATTTGTC motifs detected in file\n",
      "\n",
      "Seq is: CGCCCCATTCTAAACGG\n",
      "0 CGCCCCATTCTAAACGG motifs detected in file\n",
      "\n",
      "Seq is: CTCAAGGAAAATGAAAA\n",
      "0 CTCAAGGAAAATGAAAA motifs detected in file\n",
      "\n",
      "Seq is: AAATTGGAAAGGAATAC\n",
      "0 AAATTGGAAAGGAATAC motifs detected in file\n",
      "\n",
      "Seq is: TCGTACGGACACAACCC\n",
      "0 TCGTACGGACACAACCC motifs detected in file\n",
      "\n",
      "Seq is: AACTGTCACTCCCATTA\n",
      "0 AACTGTCACTCCCATTA motifs detected in file\n",
      "\n",
      "Seq is: TTACTGAACAACCAACT\n",
      "0 TTACTGAACAACCAACT motifs detected in file\n",
      "\n",
      "Seq is: AACATCTATAACCCGAC\n",
      "0 AACATCTATAACCCGAC motifs detected in file\n",
      "\n",
      "Seq is: AATTAACAGTCAAGTCA\n",
      "0 AATTAACAGTCAAGTCA motifs detected in file\n",
      "\n",
      "Seq is: CAAAATTAGACTGCGTC\n",
      "0 CAAAATTAGACTGCGTC motifs detected in file\n",
      "\n",
      "Seq is: CGAATACGCCTCCTCTT\n",
      "0 CGAATACGCCTCCTCTT motifs detected in file\n",
      "\n",
      "Seq is: ACAAAAGTACAATGAAT\n",
      "0 ACAAAAGTACAATGAAT motifs detected in file\n",
      "\n",
      "Seq is: ATGATTGTAATCAAGAA\n",
      "0 ATGATTGTAATCAAGAA motifs detected in file\n",
      "\n",
      "Seq is: GATATCCCACTATCTAA\n",
      "0 GATATCCCACTATCTAA motifs detected in file\n",
      "\n",
      "Seq is: CCAGGTTAACCCACACT\n",
      "0 CCAGGTTAACCCACACT motifs detected in file\n",
      "\n",
      "Seq is: CCTCAAGTCAATATACA\n",
      "0 CCTCAAGTCAATATACA motifs detected in file\n",
      "\n",
      "Seq is: AACCCTAAAAAATCCAT\n",
      "0 AACCCTAAAAAATCCAT motifs detected in file\n",
      "\n",
      "Seq is: CACCCACAACTCGAACT\n",
      "0 CACCCACAACTCGAACT motifs detected in file\n",
      "\n",
      "Seq is: TGCGAAAGAATTAACCA\n",
      "0 TGCGAAAGAATTAACCA motifs detected in file\n",
      "\n",
      "Seq is: CCGACGAAAATCCGGAT\n",
      "0 CCGACGAAAATCCGGAT motifs detected in file\n",
      "\n",
      "Seq is: GATACCCACAATTTAAA\n",
      "0 GATACCCACAATTTAAA motifs detected in file\n",
      "\n",
      "Seq is: AAATGAGAGAGATGTTC\n",
      "0 AAATGAGAGAGATGTTC motifs detected in file\n",
      "\n",
      "Seq is: CAAAAAAGGATCACAGG\n",
      "0 CAAAAAAGGATCACAGG motifs detected in file\n",
      "\n",
      "Seq is: TTTCTCGACAAGGAGAA\n",
      "0 TTTCTCGACAAGGAGAA motifs detected in file\n",
      "\n",
      "Seq is: ACCTGATTGTCAATTTA\n",
      "0 ACCTGATTGTCAATTTA motifs detected in file\n",
      "\n",
      "Seq is: AATGTTCCCCTAAATCT\n",
      "0 AATGTTCCCCTAAATCT motifs detected in file\n",
      "\n",
      "Seq is: CCCAAGACACCCGTTTA\n",
      "0 CCCAAGACACCCGTTTA motifs detected in file\n",
      "\n",
      "Seq is: AATTTCAACTTGATTCT\n",
      "0 AATTTCAACTTGATTCT motifs detected in file\n",
      "\n",
      "Seq is: AAGATAGAACCTGTTGG\n",
      "0 AAGATAGAACCTGTTGG motifs detected in file\n",
      "\n",
      "Seq is: TCGATAGTGGTCCGAGC\n",
      "0 TCGATAGTGGTCCGAGC motifs detected in file\n",
      "\n",
      "Seq is: CATCCAAACAGCGGAGA\n",
      "0 CATCCAAACAGCGGAGA motifs detected in file\n",
      "\n",
      "Seq is: TGGATATTTAGAAGGGT\n",
      "0 TGGATATTTAGAAGGGT motifs detected in file\n",
      "\n",
      "Seq is: GATAAAACGATGTATCT\n",
      "0 GATAAAACGATGTATCT motifs detected in file\n",
      "\n",
      "Seq is: GCCCACACGAGAAAATC\n",
      "0 GCCCACACGAGAAAATC motifs detected in file\n",
      "\n",
      "Seq is: GACAAGAATCCATCGAG\n",
      "0 GACAAGAATCCATCGAG motifs detected in file\n",
      "\n",
      "Seq is: CAGACCAAAGCCCCCAG\n",
      "0 CAGACCAAAGCCCCCAG motifs detected in file\n",
      "\n",
      "Seq is: AATCGAAACCGAGAGGA\n",
      "0 AATCGAAACCGAGAGGA motifs detected in file\n",
      "\n",
      "Seq is: ACGTTTCAATAAAGATC\n",
      "0 ACGTTTCAATAAAGATC motifs detected in file\n",
      "\n",
      "Seq is: AATTAAGTAATACGTCT\n",
      "0 AATTAAGTAATACGTCT motifs detected in file\n",
      "\n",
      "Seq is: TCCATATCCCCAATCAG\n",
      "0 TCCATATCCCCAATCAG motifs detected in file\n",
      "\n",
      "Seq is: GAACGATATAATACGAA\n",
      "0 GAACGATATAATACGAA motifs detected in file\n",
      "\n",
      "Seq is: CCAATATTAAAAAGTAG\n",
      "0 CCAATATTAAAAAGTAG motifs detected in file\n",
      "\n",
      "Seq is: AAAGGGAACGCCATGAT\n",
      "0 AAAGGGAACGCCATGAT motifs detected in file\n",
      "\n",
      "Seq is: ATAGATAACGCGGTCCA\n",
      "0 ATAGATAACGCGGTCCA motifs detected in file\n",
      "\n",
      "Seq is: AAGTTAAAGATAGCGGA\n",
      "0 AAGTTAAAGATAGCGGA motifs detected in file\n",
      "\n",
      "Seq is: TATGAAATAAACCCATT\n",
      "0 TATGAAATAAACCCATT motifs detected in file\n",
      "\n",
      "Seq is: TACCAAACCGATTCCAA\n",
      "0 TACCAAACCGATTCCAA motifs detected in file\n",
      "\n",
      "Seq is: CAGACCATCATTCCACC\n",
      "0 CAGACCATCATTCCACC motifs detected in file\n",
      "\n",
      "Seq is: TCACCCAAACCCCGATC\n",
      "0 TCACCCAAACCCCGATC motifs detected in file\n",
      "\n",
      "Seq is: CAAATCGAGTCTCGCCA\n",
      "0 CAAATCGAGTCTCGCCA motifs detected in file\n",
      "\n",
      "Seq is: GTTCAATTCAACTTTAG\n",
      "0 GTTCAATTCAACTTTAG motifs detected in file\n",
      "\n",
      "Seq is: AGGATTCACATTCAACC\n",
      "0 AGGATTCACATTCAACC motifs detected in file\n",
      "\n",
      "Seq is: CACGAAACACAATTCGA\n",
      "0 CACGAAACACAATTCGA motifs detected in file\n",
      "\n",
      "Seq is: TGTGAGACCAAGCAGGT\n",
      "0 TGTGAGACCAAGCAGGT motifs detected in file\n",
      "\n",
      "Seq is: TCACGTGAAAGGTCATG\n",
      "0 TCACGTGAAAGGTCATG motifs detected in file\n",
      "\n",
      "Seq is: TGAATGGTACAATGCTG\n",
      "0 TGAATGGTACAATGCTG motifs detected in file\n",
      "\n",
      "Seq is: AACAGAGGAGATATATT\n",
      "0 AACAGAGGAGATATATT motifs detected in file\n",
      "\n",
      "Seq is: TACGCATCCCCAAAATC\n",
      "0 TACGCATCCCCAAAATC motifs detected in file\n",
      "\n",
      "Seq is: AATTTACAGGAAACTTC\n",
      "0 AATTTACAGGAAACTTC motifs detected in file\n",
      "\n",
      "Seq is: ATATGAACTCCTCCCAC\n",
      "0 ATATGAACTCCTCCCAC motifs detected in file\n",
      "\n",
      "Seq is: TGCCCGCCACACATGCG\n",
      "0 TGCCCGCCACACATGCG motifs detected in file\n",
      "\n",
      "Seq is: CGAAGTCCCGGGACAAG\n",
      "0 CGAAGTCCCGGGACAAG motifs detected in file\n",
      "\n",
      "Seq is: TTGATTCGTGAGATGAG\n",
      "0 TTGATTCGTGAGATGAG motifs detected in file\n",
      "\n",
      "Seq is: AATCAAATGACGATTTA\n",
      "0 AATCAAATGACGATTTA motifs detected in file\n",
      "\n",
      "Seq is: GGTGGAAGCTGGGAATT\n",
      "0 GGTGGAAGCTGGGAATT motifs detected in file\n",
      "\n",
      "Seq is: CAAAGTATTCCCGATAG\n",
      "0 CAAAGTATTCCCGATAG motifs detected in file\n",
      "\n",
      "Seq is: CATCAAAAGACCCCATC\n",
      "0 CATCAAAAGACCCCATC motifs detected in file\n",
      "\n",
      "Seq is: TTTTACATCGGGTAAAG\n",
      "0 TTTTACATCGGGTAAAG motifs detected in file\n",
      "\n",
      "Seq is: AACGGTGGAGTACCCGA\n",
      "0 AACGGTGGAGTACCCGA motifs detected in file\n",
      "\n",
      "Seq is: TGTGGAACTGGATTGCA\n",
      "0 TGTGGAACTGGATTGCA motifs detected in file\n",
      "\n",
      "Seq is: GAAATGCACCCATGAAC\n",
      "0 GAAATGCACCCATGAAC motifs detected in file\n",
      "\n",
      "Seq is: GCGAATGAAACATCGGA\n",
      "0 GCGAATGAAACATCGGA motifs detected in file\n",
      "\n",
      "Seq is: AGTAGTCCCAAACGACT\n",
      "0 AGTAGTCCCAAACGACT motifs detected in file\n",
      "\n",
      "Seq is: TCTACCGCCATATATCC\n",
      "0 TCTACCGCCATATATCC motifs detected in file\n",
      "\n",
      "Seq is: GACTCGTTCTCCACCAC\n",
      "0 GACTCGTTCTCCACCAC motifs detected in file\n",
      "\n",
      "Seq is: TCCAACTAGCCCCAAAT\n",
      "0 TCCAACTAGCCCCAAAT motifs detected in file\n",
      "\n",
      "Seq is: AGCTAATGTCTTGTCCG\n",
      "0 AGCTAATGTCTTGTCCG motifs detected in file\n",
      "\n",
      "Seq is: AGGAGATCTCCCTATAC\n",
      "0 AGGAGATCTCCCTATAC motifs detected in file\n",
      "\n",
      "Seq is: TTCGTGGCGGTCCAGGA\n",
      "0 TTCGTGGCGGTCCAGGA motifs detected in file\n",
      "\n",
      "Seq is: AACTCAAAATTCGACAC\n",
      "0 AACTCAAAATTCGACAC motifs detected in file\n",
      "\n",
      "Seq is: CGAGCATCACAAGACCG\n",
      "0 CGAGCATCACAAGACCG motifs detected in file\n",
      "\n",
      "Seq is: CTCGTCAAAACAACTAA\n",
      "0 CTCGTCAAAACAACTAA motifs detected in file\n",
      "\n",
      "Seq is: ATTGACAACTCCAAATC\n",
      "0 ATTGACAACTCCAAATC motifs detected in file\n",
      "\n",
      "Seq is: CCGATTCGTATCACCCC\n",
      "0 CCGATTCGTATCACCCC motifs detected in file\n",
      "\n",
      "Seq is: ATAGATTAGGGTCAAAC\n",
      "0 ATAGATTAGGGTCAAAC motifs detected in file\n",
      "\n",
      "Seq is: CCAGAATCGATAACACA\n",
      "0 CCAGAATCGATAACACA motifs detected in file\n",
      "\n",
      "Seq is: CAAGTCTATACAATTTC\n",
      "0 CAAGTCTATACAATTTC motifs detected in file\n",
      "\n",
      "Seq is: GGTGAAAGCATCAGATA\n",
      "0 GGTGAAAGCATCAGATA motifs detected in file\n",
      "\n",
      "Seq is: AAACACAGTTGACCTCA\n",
      "0 AAACACAGTTGACCTCA motifs detected in file\n",
      "\n",
      "Seq is: AAAAATGTTGAGTCCAC\n",
      "0 AAAAATGTTGAGTCCAC motifs detected in file\n",
      "\n",
      "Seq is: TCAAAATCGAAATAACC\n",
      "0 TCAAAATCGAAATAACC motifs detected in file\n",
      "\n",
      "Seq is: CCTCCCCCACTAGATTT\n",
      "0 CCTCCCCCACTAGATTT motifs detected in file\n",
      "\n",
      "Seq is: TGTGAGAAATGCGGCCG\n",
      "0 TGTGAGAAATGCGGCCG motifs detected in file\n",
      "\n",
      "Seq is: AAGATAACTGAACCCAA\n",
      "0 AAGATAACTGAACCCAA motifs detected in file\n",
      "\n",
      "Seq is: TTAGCACACTGGCGCCA\n",
      "0 TTAGCACACTGGCGCCA motifs detected in file\n",
      "\n",
      "Seq is: CCGACCGTGCTTTAACT\n",
      "0 CCGACCGTGCTTTAACT motifs detected in file\n",
      "\n",
      "Seq is: GGTTGGGACCCCAATCA\n",
      "0 GGTTGGGACCCCAATCA motifs detected in file\n",
      "\n",
      "Seq is: TATCGAATCAATTTGAA\n",
      "0 TATCGAATCAATTTGAA motifs detected in file\n",
      "\n",
      "Seq is: AGCAAATAACGATTTCC\n",
      "0 AGCAAATAACGATTTCC motifs detected in file\n",
      "\n",
      "Seq is: AATTAGTGACGACAAAG\n",
      "0 AATTAGTGACGACAAAG motifs detected in file\n",
      "\n",
      "Seq is: GGCACCCCCACACCGAT\n",
      "0 GGCACCCCCACACCGAT motifs detected in file\n",
      "\n",
      "Seq is: CCGATTCGCAAAACTCG\n",
      "0 CCGATTCGCAAAACTCG motifs detected in file\n",
      "\n",
      "Seq is: ACGTAACGACGCACGAA\n",
      "0 ACGTAACGACGCACGAA motifs detected in file\n",
      "\n",
      "Seq is: CTACGAACAAGGAAAAC\n",
      "0 CTACGAACAAGGAAAAC motifs detected in file\n",
      "\n",
      "Seq is: TAGCACCACTAAATCTC\n",
      "0 TAGCACCACTAAATCTC motifs detected in file\n",
      "\n",
      "Seq is: CCACTTGAGTGACCTTG\n",
      "0 CCACTTGAGTGACCTTG motifs detected in file\n",
      "\n",
      "Seq is: CCCCTACGAACGTACAC\n",
      "0 CCCCTACGAACGTACAC motifs detected in file\n",
      "\n",
      "Seq is: ATTAGAATGATTCTCGA\n",
      "0 ATTAGAATGATTCTCGA motifs detected in file\n",
      "\n",
      "Seq is: TTATCTTTCCGATCCTG\n",
      "0 TTATCTTTCCGATCCTG motifs detected in file\n",
      "\n",
      "Seq is: GTTTGGATAAACAAATA\n",
      "0 GTTTGGATAAACAAATA motifs detected in file\n",
      "\n",
      "Seq is: CCCCACTACACTCGGGC\n",
      "0 CCCCACTACACTCGGGC motifs detected in file\n",
      "\n",
      "Seq is: AGATTTGTAAAAGTCAC\n",
      "0 AGATTTGTAAAAGTCAC motifs detected in file\n",
      "\n",
      "Seq is: ATAACGAAACTCCTCCA\n",
      "0 ATAACGAAACTCCTCCA motifs detected in file\n",
      "\n",
      "Seq is: TTCGATGTATTTGACAC\n",
      "0 TTCGATGTATTTGACAC motifs detected in file\n",
      "\n",
      "Seq is: CCCCCACAGAAACCCCA\n",
      "0 CCCCCACAGAAACCCCA motifs detected in file\n",
      "\n",
      "Seq is: AACCAACCAAGCCCCAT\n",
      "0 AACCAACCAAGCCCCAT motifs detected in file\n",
      "\n",
      "Seq is: ACCCCAATCGTCGCCAC\n",
      "0 ACCCCAATCGTCGCCAC motifs detected in file\n",
      "\n",
      "Seq is: ACACACACGACCCATCC\n",
      "0 ACACACACGACCCATCC motifs detected in file\n",
      "\n",
      "Seq is: TACCCGCCCCCAACATA\n",
      "0 TACCCGCCCCCAACATA motifs detected in file\n",
      "\n",
      "Seq is: ACTACTCTAATCATCAT\n",
      "0 ACTACTCTAATCATCAT motifs detected in file\n",
      "\n",
      "Seq is: ACCCTCACCCTCCCCTT\n",
      "0 ACCCTCACCCTCCCCTT motifs detected in file\n",
      "\n",
      "Seq is: TTATTACACAATCAACC\n",
      "0 TTATTACACAATCAACC motifs detected in file\n",
      "\n",
      "Seq is: CCCCACTGACAATTTTC\n",
      "0 CCCCACTGACAATTTTC motifs detected in file\n",
      "\n",
      "Seq is: ACGTATGGCGGTTTTCT\n",
      "0 ACGTATGGCGGTTTTCT motifs detected in file\n",
      "\n",
      "Seq is: ATTTTAAACTTTAGACC\n",
      "0 ATTTTAAACTTTAGACC motifs detected in file\n",
      "\n",
      "Seq is: AATCCGACCACAATCCC\n",
      "0 AATCCGACCACAATCCC motifs detected in file\n",
      "\n",
      "Seq is: AAGAAACAAAAACCCCA\n",
      "0 AAGAAACAAAAACCCCA motifs detected in file\n",
      "\n",
      "Seq is: AACCGTCTCTACACAAA\n",
      "0 AACCGTCTCTACACAAA motifs detected in file\n",
      "\n",
      "Seq is: TTCACGACACCGGTCTT\n",
      "0 TTCACGACACCGGTCTT motifs detected in file\n",
      "\n",
      "Seq is: CGCCCCCTCCCCCCCAA\n",
      "0 CGCCCCCTCCCCCCCAA motifs detected in file\n",
      "\n",
      "Seq is: ACCACCTTTAAAAAACA\n",
      "0 ACCACCTTTAAAAAACA motifs detected in file\n",
      "\n",
      "Seq is: ATACTACAGACACACCT\n",
      "0 ATACTACAGACACACCT motifs detected in file\n",
      "\n",
      "Seq is: TTCACCGACACGTCTGT\n",
      "0 TTCACCGACACGTCTGT motifs detected in file\n",
      "\n",
      "Seq is: AAGTTAACAATAATAAT\n",
      "0 AAGTTAACAATAATAAT motifs detected in file\n",
      "\n",
      "Seq is: ACAGGATGTTCGTAATT\n",
      "0 ACAGGATGTTCGTAATT motifs detected in file\n",
      "\n",
      "Seq is: AATTAATTGTGTGAAAT\n",
      "0 AATTAATTGTGTGAAAT motifs detected in file\n",
      "\n",
      "Seq is: CATTCATACAAGCGGAC\n",
      "0 CATTCATACAAGCGGAC motifs detected in file\n",
      "\n",
      "Seq is: ATTATAACTTGCATCCA\n",
      "0 ATTATAACTTGCATCCA motifs detected in file\n",
      "\n",
      "Seq is: CGCTATTTATTATCCTA\n",
      "0 CGCTATTTATTATCCTA motifs detected in file\n",
      "\n",
      "Seq is: CTCCGTCCTTAGTTTCT\n",
      "0 CTCCGTCCTTAGTTTCT motifs detected in file\n",
      "\n",
      "Seq is: GTCTATGACGCTGTATC\n",
      "0 GTCTATGACGCTGTATC motifs detected in file\n",
      "\n",
      "Seq is: CCACGAGGCCGAGGTCG\n",
      "0 CCACGAGGCCGAGGTCG motifs detected in file\n",
      "\n",
      "Seq is: CAGAGCGTTACGATAGC\n",
      "0 CAGAGCGTTACGATAGC motifs detected in file\n",
      "\n",
      "Seq is: GCACGTATGGGGGGTCT\n",
      "0 GCACGTATGGGGGGTCT motifs detected in file\n",
      "\n",
      "Seq is: GCTTTTATGGTTTACGT\n",
      "0 GCTTTTATGGTTTACGT motifs detected in file\n",
      "\n",
      "Seq is: ACCTCTCGAGGGCACTC\n",
      "0 ACCTCTCGAGGGCACTC motifs detected in file\n",
      "\n",
      "Seq is: ACCAATTATCCCACTAT\n",
      "0 ACCAATTATCCCACTAT motifs detected in file\n",
      "\n",
      "Seq is: GTAGCACTACAGAATAAA\n",
      "0 GTAGCACTACAGAATAAA motifs detected in file\n",
      "\n",
      "Seq is: TTCCCCTTGCACACCCGA\n",
      "0 TTCCCCTTGCACACCCGA motifs detected in file\n",
      "\n",
      "Seq is: TAAATCCGAAATACTGGG\n",
      "0 TAAATCCGAAATACTGGG motifs detected in file\n",
      "\n",
      "Seq is: ACTTCATCCTTGGTCTAC\n",
      "0 ACTTCATCCTTGGTCTAC motifs detected in file\n",
      "\n",
      "Seq is: AGCCTATGTCAAGTGAAA\n",
      "0 AGCCTATGTCAAGTGAAA motifs detected in file\n",
      "\n",
      "Seq is: TCGATGGGGGTTCACAAT\n",
      "0 TCGATGGGGGTTCACAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCCGGGCCTCGCTCCTC\n",
      "0 ACCCGGGCCTCGCTCCTC motifs detected in file\n",
      "\n",
      "Seq is: TCATCGTGAGAACACGCC\n",
      "0 TCATCGTGAGAACACGCC motifs detected in file\n",
      "\n",
      "Seq is: CTATAACTAAAGTGCCTC\n",
      "0 CTATAACTAAAGTGCCTC motifs detected in file\n",
      "\n",
      "Seq is: CTACCACCAGTTCCCTGG\n",
      "0 CTACCACCAGTTCCCTGG motifs detected in file\n",
      "\n",
      "Seq is: GGATAGACTCCCCCCAGT\n",
      "0 GGATAGACTCCCCCCAGT motifs detected in file\n",
      "\n",
      "Seq is: AGGTACCCCTGCTCTTCC\n",
      "0 AGGTACCCCTGCTCTTCC motifs detected in file\n",
      "\n",
      "Seq is: CTAAACTGACATTACACG\n",
      "0 CTAAACTGACATTACACG motifs detected in file\n",
      "\n",
      "Seq is: ATACATGCCATTTACCGA\n",
      "0 ATACATGCCATTTACCGA motifs detected in file\n",
      "\n",
      "Seq is: AATACATGATACATGACA\n",
      "0 AATACATGATACATGACA motifs detected in file\n",
      "\n",
      "Seq is: ATTCCCACCCATCCAAAC\n",
      "0 ATTCCCACCCATCCAAAC motifs detected in file\n",
      "\n",
      "Seq is: AACCATAGGATCACCCAC\n",
      "0 AACCATAGGATCACCCAC motifs detected in file\n",
      "\n",
      "Seq is: TCCCCACCGAAACCTCAA\n",
      "0 TCCCCACCGAAACCTCAA motifs detected in file\n",
      "\n",
      "Seq is: CGTCAACTACACACTATC\n",
      "0 CGTCAACTACACACTATC motifs detected in file\n",
      "\n",
      "Seq is: AACTCCCAACTAACGACA\n",
      "0 AACTCCCAACTAACGACA motifs detected in file\n",
      "\n",
      "Seq is: TGAACGAACATTCGTACC\n",
      "0 TGAACGAACATTCGTACC motifs detected in file\n",
      "\n",
      "Seq is: CCTCCCCCAAAACTACAC\n",
      "0 CCTCCCCCAAAACTACAC motifs detected in file\n",
      "\n",
      "Seq is: CTAACCCAAAAATACATG\n",
      "0 CTAACCCAAAAATACATG motifs detected in file\n",
      "\n",
      "Seq is: ATGTCCACCAGTTCATAA\n",
      "0 ATGTCCACCAGTTCATAA motifs detected in file\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Seq is: ATACCATGGCATGTTATA\n",
      "0 ATACCATGGCATGTTATA motifs detected in file\n",
      "\n",
      "Seq is: AGTACCACCGACCGTCAT\n",
      "0 AGTACCACCGACCGTCAT motifs detected in file\n",
      "\n",
      "Seq is: TACATGCTTTATGTATCG\n",
      "0 TACATGCTTTATGTATCG motifs detected in file\n",
      "\n",
      "Seq is: CCAACAACTACCCACTCA\n",
      "0 CCAACAACTACCCACTCA motifs detected in file\n",
      "\n",
      "Seq is: GTTATGAACCCACCATGG\n",
      "0 GTTATGAACCCACCATGG motifs detected in file\n",
      "\n",
      "Seq is: GTTTAGACGAAGGGGTAC\n",
      "0 GTTTAGACGAAGGGGTAC motifs detected in file\n",
      "\n",
      "Seq is: TTTCTTGTCTCTTATCAA\n",
      "0 TTTCTTGTCTCTTATCAA motifs detected in file\n",
      "\n",
      "Seq is: ATTTAATCTTAGAATCGA\n",
      "0 ATTTAATCTTAGAATCGA motifs detected in file\n",
      "\n",
      "Seq is: AACCCACGATTACCACCT\n",
      "0 AACCCACGATTACCACCT motifs detected in file\n",
      "\n",
      "Seq is: CAATTTCTGAAAAAGAGA\n",
      "0 CAATTTCTGAAAAAGAGA motifs detected in file\n",
      "\n",
      "Seq is: CTAAACAGGAACCTTTTT\n",
      "0 CTAAACAGGAACCTTTTT motifs detected in file\n",
      "\n",
      "Seq is: CCAAAAGTAGAGGCCAAA\n",
      "0 CCAAAAGTAGAGGCCAAA motifs detected in file\n",
      "\n",
      "Seq is: TGTTCTGACCACATAATC\n",
      "0 TGTTCTGACCACATAATC motifs detected in file\n",
      "\n",
      "Seq is: AAATATGATGTTCCTGTC\n",
      "0 AAATATGATGTTCCTGTC motifs detected in file\n",
      "\n",
      "Seq is: CGGGTAAACTCATAAAAC\n",
      "0 CGGGTAAACTCATAAAAC motifs detected in file\n",
      "\n",
      "Seq is: AAAAGTTAATCCCTCTAT\n",
      "0 AAAAGTTAATCCCTCTAT motifs detected in file\n",
      "\n",
      "Seq is: CAACCATAATCCTAATCC\n",
      "0 CAACCATAATCCTAATCC motifs detected in file\n",
      "\n",
      "Seq is: TAACAACACTTCATATCA\n",
      "0 TAACAACACTTCATATCA motifs detected in file\n",
      "\n",
      "Seq is: TGCCTACGATGAACAGGT\n",
      "0 TGCCTACGATGAACAGGT motifs detected in file\n",
      "\n",
      "Seq is: TACTACCATTTTCCCATC\n",
      "0 TACTACCATTTTCCCATC motifs detected in file\n",
      "\n",
      "Seq is: GAATGACCAACAGGAGGC\n",
      "0 GAATGACCAACAGGAGGC motifs detected in file\n",
      "\n",
      "Seq is: TAAGTCCAATCTTACTCC\n",
      "0 TAAGTCCAATCTTACTCC motifs detected in file\n",
      "\n",
      "Seq is: TCCAGACGCCGATCCTCA\n",
      "0 TCCAGACGCCGATCCTCA motifs detected in file\n",
      "\n",
      "Seq is: GTTATTTCACTAACCGAA\n",
      "0 GTTATTTCACTAACCGAA motifs detected in file\n",
      "\n",
      "Seq is: TCACCCGCTTTATAATAC\n",
      "0 TCACCCGCTTTATAATAC motifs detected in file\n",
      "\n",
      "Seq is: GAAACAACAAACCTATAT\n",
      "0 GAAACAACAAACCTATAT motifs detected in file\n",
      "\n",
      "Seq is: ACCTCCTACCCCTAATAA\n",
      "0 ACCTCCTACCCCTAATAA motifs detected in file\n",
      "\n",
      "Seq is: CGATCCTACTCCTACCTA\n",
      "0 CGATCCTACTCCTACCTA motifs detected in file\n",
      "\n",
      "Seq is: TCATTATCCCGTTCCTGC\n",
      "0 TCATTATCCCGTTCCTGC motifs detected in file\n",
      "\n",
      "Seq is: GGAGGATCAAACAATCCC\n",
      "0 GGAGGATCAAACAATCCC motifs detected in file\n",
      "\n",
      "Seq is: TGCCTAGCCTCTTAACAC\n",
      "0 TGCCTAGCCTCTTAACAC motifs detected in file\n",
      "\n",
      "Seq is: ATCCGCTTATCCTTTATA\n",
      "0 ATCCGCTTATCCTTTATA motifs detected in file\n",
      "\n",
      "Seq is: GTAAGCCCGAACTACACC\n",
      "0 GTAAGCCCGAACTACACC motifs detected in file\n",
      "\n",
      "Seq is: CCTCCCCACAAATTCCCC\n",
      "0 CCTCCCCACAAATTCCCC motifs detected in file\n",
      "\n",
      "Seq is: AACCGATCCCATATTAAC\n",
      "0 AACCGATCCCATATTAAC motifs detected in file\n",
      "\n",
      "Seq is: AGACCCAGCGGATCCTCC\n",
      "0 AGACCCAGCGGATCCTCC motifs detected in file\n",
      "\n",
      "Seq is: AGACCACTCTTATCACAA\n",
      "0 AGACCACTCTTATCACAA motifs detected in file\n",
      "\n",
      "Seq is: TTACAGTAATTCCTCTCT\n",
      "0 TTACAGTAATTCCTCTCT motifs detected in file\n",
      "\n",
      "Seq is: TCCTTCTCTTCATTCGGC\n",
      "0 TCCTTCTCTTCATTCGGC motifs detected in file\n",
      "\n",
      "Seq is: TCCCGCAGAAACTAACAC\n",
      "0 TCCCGCAGAAACTAACAC motifs detected in file\n",
      "\n",
      "Seq is: ATCATTCCCACCTTCCAC\n",
      "0 ATCATTCCCACCTTCCAC motifs detected in file\n",
      "\n",
      "Seq is: TAAAATAGCCTTACCCTC\n",
      "0 TAAAATAGCCTTACCCTC motifs detected in file\n",
      "\n",
      "Seq is: CACTAAGGATCCCCCAAC\n",
      "0 CACTAAGGATCCCCCAAC motifs detected in file\n",
      "\n",
      "Seq is: AAACTAGGGCAAAGCACG\n",
      "0 AAACTAGGGCAAAGCACG motifs detected in file\n",
      "\n",
      "Seq is: TTCTTATCCTCCACCTCA\n",
      "0 TTCTTATCCTCCACCTCA motifs detected in file\n",
      "\n",
      "Seq is: CAACGATCCCGACGTTAT\n",
      "0 CAACGATCCCGACGTTAT motifs detected in file\n",
      "\n",
      "Seq is: TACTTCCCGTTCTACTTC\n",
      "0 TACTTCCCGTTCTACTTC motifs detected in file\n",
      "\n",
      "Seq is: ACTTTCCATTTCTTAGCA\n",
      "0 ACTTTCCATTTCTTAGCA motifs detected in file\n",
      "\n",
      "Seq is: CACTCCCACCCTGACAGA\n",
      "0 CACTCCCACCCTGACAGA motifs detected in file\n",
      "\n",
      "Seq is: TGACTCATCGGAGGAGTC\n",
      "0 TGACTCATCGGAGGAGTC motifs detected in file\n",
      "\n",
      "Seq is: TAAGTAACTTGATCCAGA\n",
      "0 TAAGTAACTTGATCCAGA motifs detected in file\n",
      "\n",
      "Seq is: CAGGGTTACATACCCTAC\n",
      "0 CAGGGTTACATACCCTAC motifs detected in file\n",
      "\n",
      "Seq is: CGCCTATCATTCAAACAT\n",
      "0 CGCCTATCATTCAAACAT motifs detected in file\n",
      "\n",
      "Seq is: TAATGACACCGGGGAGTC\n",
      "0 TAATGACACCGGGGAGTC motifs detected in file\n",
      "\n",
      "Seq is: TTACTATAAACCGGAGTG\n",
      "0 TTACTATAAACCGGAGTG motifs detected in file\n",
      "\n",
      "Seq is: CCCTCCTGTATCGGATAC\n",
      "0 CCCTCCTGTATCGGATAC motifs detected in file\n",
      "\n",
      "Seq is: TTCCGACAACGATATCAA\n",
      "0 TTCCGACAACGATATCAA motifs detected in file\n",
      "\n",
      "Seq is: CGTTCGTCCTCCTATTAC\n",
      "0 CGTTCGTCCTCCTATTAC motifs detected in file\n",
      "\n",
      "Seq is: GGCTACAAAGTCCAAAGA\n",
      "0 GGCTACAAAGTCCAAAGA motifs detected in file\n",
      "\n",
      "Seq is: CTCATCTCTTTACTAGGC\n",
      "0 CTCATCTCTTTACTAGGC motifs detected in file\n",
      "\n",
      "Seq is: ATTATATCCGGAGCGGGC\n",
      "0 ATTATATCCGGAGCGGGC motifs detected in file\n",
      "\n",
      "Seq is: TACACATCCTTCTCCGTC\n",
      "0 TACACATCCTTCTCCGTC motifs detected in file\n",
      "\n",
      "Seq is: TATTTCTTATAACTCCGC\n",
      "0 TATTTCTTATAACTCCGC motifs detected in file\n",
      "\n",
      "Seq is: GGTAACCGCACTTCCATC\n",
      "0 GGTAACCGCACTTCCATC motifs detected in file\n",
      "\n",
      "Seq is: GCCTACTAAGTCGGTATT\n",
      "0 GCCTACTAAGTCGGTATT motifs detected in file\n",
      "\n",
      "Seq is: AAATGCAGAGCTCACTAC\n",
      "0 AAATGCAGAGCTCACTAC motifs detected in file\n",
      "\n",
      "Seq is: ACCCGCTAACTACTTTTC\n",
      "0 ACCCGCTAACTACTTTTC motifs detected in file\n",
      "\n",
      "Seq is: CGCCAACTCCGCAGACCA\n",
      "0 CGCCAACTCCGCAGACCA motifs detected in file\n",
      "\n",
      "Seq is: CTCATCACGTACCGATCC\n",
      "0 CTCATCACGTACCGATCC motifs detected in file\n",
      "\n",
      "Seq is: TTATCAGGACACCACTAA\n",
      "0 TTATCAGGACACCACTAA motifs detected in file\n",
      "\n",
      "Seq is: ACCTCCTAGTCCGTCCGC\n",
      "0 ACCTCCTAGTCCGTCCGC motifs detected in file\n",
      "\n",
      "Seq is: GGTTCCTCACTCGGCTTC\n",
      "0 GGTTCCTCACTCGGCTTC motifs detected in file\n",
      "\n",
      "Seq is: AAAGTAGTACGCCTCTAC\n",
      "0 AAAGTAGTACGCCTCTAC motifs detected in file\n",
      "\n",
      "Seq is: AACCTACCCCACCCCTCC\n",
      "0 AACCTACCCCACCCCTCC motifs detected in file\n",
      "\n",
      "Seq is: AGCTACTTACTCACCAAT\n",
      "0 AGCTACTTACTCACCAAT motifs detected in file\n",
      "\n",
      "Seq is: TAATTAAAATAATCCCCC\n",
      "0 TAATTAAAATAATCCCCC motifs detected in file\n",
      "\n",
      "Seq is: AATCAAAACGCATAACCC\n",
      "0 AATCAAAACGCATAACCC motifs detected in file\n",
      "\n",
      "Seq is: CAGTAACCACAAGAACAT\n",
      "0 CAGTAACCACAAGAACAT motifs detected in file\n",
      "\n",
      "Seq is: CAACTTTATGTTGCTACC\n",
      "0 CAACTTTATGTTGCTACC motifs detected in file\n",
      "\n",
      "Seq is: AAAAAGTATAGTAACCAG\n",
      "0 AAAAAGTATAGTAACCAG motifs detected in file\n",
      "\n",
      "Seq is: CACCAACATCAGGCACGC\n",
      "0 CACCAACATCAGGCACGC motifs detected in file\n",
      "\n",
      "Seq is: TCTTATTACTACATACGA\n",
      "0 TCTTATTACTACATACGA motifs detected in file\n",
      "\n",
      "Seq is: AACAAAGACAACTCACAC\n",
      "0 AACAAAGACAACTCACAC motifs detected in file\n",
      "\n",
      "Seq is: CCAAATCATTACCCCAAA\n",
      "0 CCAAATCATTACCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: CACCCCAAAAGAAGATTC\n",
      "0 CACCCCAAAAGAAGATTC motifs detected in file\n",
      "\n",
      "Seq is: GGAAGAGGATAAATACCC\n",
      "0 GGAAGAGGATAAATACCC motifs detected in file\n",
      "\n",
      "Seq is: CCAAATCATAACTAACAA\n",
      "0 CCAAATCATAACTAACAA motifs detected in file\n",
      "\n",
      "Seq is: TCGCCACACCAGCCCACA\n",
      "0 TCGCCACACCAGCCCACA motifs detected in file\n",
      "\n",
      "Seq is: CAATAATAAGACTTAAAA\n",
      "0 CAATAATAAGACTTAAAA motifs detected in file\n",
      "\n",
      "Seq is: CCCCCTCCAATATACCCA\n",
      "0 CCCCCTCCAATATACCCA motifs detected in file\n",
      "\n",
      "Seq is: AATTATCAAAAAAATTAA\n",
      "0 AATTATCAAAAAAATTAA motifs detected in file\n",
      "\n",
      "Seq is: ATAAATCCCCCTTACTAC\n",
      "0 ATAAATCCCCCTTACTAC motifs detected in file\n",
      "\n",
      "Seq is: CAACAGAAACCTATATGA\n",
      "0 CAACAGAAACCTATATGA motifs detected in file\n",
      "\n",
      "Seq is: TGTCGCTACCGATAACTC\n",
      "0 TGTCGCTACCGATAACTC motifs detected in file\n",
      "\n",
      "Seq is: CTCATAGGACTCCGTACC\n",
      "0 CTCATAGGACTCCGTACC motifs detected in file\n",
      "\n",
      "Seq is: CCCAGTCCCCAACTCCAG\n",
      "0 CCCAGTCCCCAACTCCAG motifs detected in file\n",
      "\n",
      "Seq is: AACCACTCACAAAATCAC\n",
      "0 AACCACTCACAAAATCAC motifs detected in file\n",
      "\n",
      "Seq is: CCCAATCGCTACCTCCAT\n",
      "0 CCCAATCGCTACCTCCAT motifs detected in file\n",
      "\n",
      "Seq is: CCTAACCACGACACCCAC\n",
      "0 CCTAACCACGACACCCAC motifs detected in file\n",
      "\n",
      "Seq is: TTTCTCATACTACCCCAC\n",
      "0 TTTCTCATACTACCCCAC motifs detected in file\n",
      "\n",
      "Seq is: CACCAACACCATTTGAAA\n",
      "0 CACCAACACCATTTGAAA motifs detected in file\n",
      "\n",
      "Seq is: TTATCACATCCTTCGACT\n",
      "0 TTATCACATCCTTCGACT motifs detected in file\n",
      "\n",
      "Seq is: TATTAAATACTTCCTCTC\n",
      "0 TATTAAATACTTCCTCTC motifs detected in file\n",
      "\n",
      "Seq is: CCCAGTCCCAACTAAGCC\n",
      "0 CCCAGTCCCAACTAAGCC motifs detected in file\n",
      "\n",
      "Seq is: CTCCTAGGATAACCACGC\n",
      "0 CTCCTAGGATAACCACGC motifs detected in file\n",
      "\n",
      "Seq is: CCCCGAAACATACTAATA\n",
      "0 CCCCGAAACATACTAATA motifs detected in file\n",
      "\n",
      "Seq is: CCCGCAACTAATCATCAT\n",
      "0 CCCGCAACTAATCATCAT motifs detected in file\n",
      "\n",
      "Seq is: CAATGACCAACTTGTAAC\n",
      "0 CAATGACCAACTTGTAAC motifs detected in file\n",
      "\n",
      "Seq is: AAACAACCACATATATAA\n",
      "0 AAACAACCACATATATAA motifs detected in file\n",
      "\n",
      "Seq is: CATTAACTCTAACGAGCC\n",
      "0 CATTAACTCTAACGAGCC motifs detected in file\n",
      "\n",
      "Seq is: CCCTTATCCAATACACTA\n",
      "0 CCCTTATCCAATACACTA motifs detected in file\n",
      "\n",
      "Seq is: ATCCTCATCCCAATCCTA\n",
      "0 ATCCTCATCCCAATCCTA motifs detected in file\n",
      "\n",
      "Seq is: CTCACCCTTCTTCTTTCT\n",
      "0 CTCACCCTTCTTCTTTCT motifs detected in file\n",
      "\n",
      "Seq is: CTCCTTCATTTCAAATTA\n",
      "0 CTCCTTCATTTCAAATTA motifs detected in file\n",
      "\n",
      "Seq is: ATACGGAAAAACCCAACT\n",
      "0 ATACGGAAAAACCCAACT motifs detected in file\n",
      "\n",
      "Seq is: CCACTACTACCTCCACCT\n",
      "0 CCACTACTACCTCCACCT motifs detected in file\n",
      "\n",
      "Seq is: CTAAACCACGACACTTTA\n",
      "0 CTAAACCACGACACTTTA motifs detected in file\n",
      "\n",
      "Seq is: ACAAAATCCATTATCGAA\n",
      "0 ACAAAATCCATTATCGAA motifs detected in file\n",
      "\n",
      "Seq is: AAGATCAGTCCAATCCAG\n",
      "0 AAGATCAGTCCAATCCAG motifs detected in file\n",
      "\n",
      "Seq is: ATCCTCCTCATCCCCGTC\n",
      "0 ATCCTCCTCATCCCCGTC motifs detected in file\n",
      "\n",
      "Seq is: CAAAACCGAGCATTCTTC\n",
      "0 CAAAACCGAGCATTCTTC motifs detected in file\n",
      "\n",
      "Seq is: CGGATCTATCCCCTAACA\n",
      "0 CGGATCTATCCCCTAACA motifs detected in file\n",
      "\n",
      "Seq is: CGCCACACACTACGATCC\n",
      "0 CGCCACACACTACGATCC motifs detected in file\n",
      "\n",
      "Seq is: CATCTTAGGCTCATACAA\n",
      "0 CATCTTAGGCTCATACAA motifs detected in file\n",
      "\n",
      "Seq is: CCTCTTTATTTTACACGT\n",
      "0 CCTCTTTATTTTACACGT motifs detected in file\n",
      "\n",
      "Seq is: ATCACCCCTAAAATAAAA\n",
      "0 ATCACCCCTAAAATAAAA motifs detected in file\n",
      "\n",
      "Seq is: TTCAAACAACCAATCCAT\n",
      "0 TTCAAACAACCAATCCAT motifs detected in file\n",
      "\n",
      "Seq is: CAACTCCAGATCCCGACA\n",
      "0 CAACTCCAGATCCCGACA motifs detected in file\n",
      "\n",
      "Seq is: ATCTTCAGGATCCTTTCA\n",
      "0 ATCTTCAGGATCCTTTCA motifs detected in file\n",
      "\n",
      "Seq is: CTGTCGCTCCCGACACTC\n",
      "0 CTGTCGCTCCCGACACTC motifs detected in file\n",
      "\n",
      "Seq is: AAAATCCATCTCCCCCTA\n",
      "0 AAAATCCATCTCCCCCTA motifs detected in file\n",
      "\n",
      "Seq is: ACAACAAACCTTCCCCCT\n",
      "0 ACAACAAACCTTCCCCCT motifs detected in file\n",
      "\n",
      "Seq is: ACGCCCCCTTTACAACAA\n",
      "0 ACGCCCCCTTTACAACAA motifs detected in file\n",
      "\n",
      "Seq is: TCATTACTCTTTAGGACG\n",
      "0 TCATTACTCTTTAGGACG motifs detected in file\n",
      "\n",
      "Seq is: CTTATCCGAAGGCCGACG\n",
      "0 CTTATCCGAAGGCCGACG motifs detected in file\n",
      "\n",
      "Seq is: GTCCGCAAATTACCCCAA\n",
      "0 GTCCGCAAATTACCCCAA motifs detected in file\n",
      "\n",
      "Seq is: ATCATCCCACCCCAATAA\n",
      "0 ATCATCCCACCCCAATAA motifs detected in file\n",
      "\n",
      "Seq is: AAGCAATTACAATCATTC\n",
      "0 AAGCAATTACAATCATTC motifs detected in file\n",
      "\n",
      "Seq is: CCACCCCTTCGCTCCAAC\n",
      "0 CCACCCCTTCGCTCCAAC motifs detected in file\n",
      "\n",
      "Seq is: TGGACAATCCCACTCTTC\n",
      "0 TGGACAATCCCACTCTTC motifs detected in file\n",
      "\n",
      "Seq is: TTAATAAGCTCACGATAT\n",
      "0 TTAATAAGCTCACGATAT motifs detected in file\n",
      "\n",
      "Seq is: CCGCGAACAGTCCCTCCA\n",
      "0 CCGCGAACAGTCCCTCCA motifs detected in file\n",
      "\n",
      "Seq is: TCGCTACTCTCATTATCT\n",
      "0 TCGCTACTCTCATTATCT motifs detected in file\n",
      "\n",
      "Seq is: ATCCCGAGTCCGCAAACA\n",
      "0 ATCCCGAGTCCGCAAACA motifs detected in file\n",
      "\n",
      "Seq is: CATACTATACAAACGCCA\n",
      "0 CATACTATACAAACGCCA motifs detected in file\n",
      "\n",
      "Seq is: AAGCTACTACACCAGAAA\n",
      "0 AAGCTACTACACCAGAAA motifs detected in file\n",
      "\n",
      "Seq is: CCTCATCTTTGGACACTC\n",
      "0 CCTCATCTTTGGACACTC motifs detected in file\n",
      "\n",
      "Seq is: CTTTCCATAAGGACGATT\n",
      "0 CTTTCCATAAGGACGATT motifs detected in file\n",
      "\n",
      "Seq is: ACGATCCGACGGTTACCA\n",
      "0 ACGATCCGACGGTTACCA motifs detected in file\n",
      "\n",
      "Seq is: CTCCCTCCAACTTCACTC\n",
      "0 CTCCCTCCAACTTCACTC motifs detected in file\n",
      "\n",
      "Seq is: TCCATACCAAAACTCATC\n",
      "0 TCCATACCAAAACTCATC motifs detected in file\n",
      "\n",
      "Seq is: AGGAGGATAAAAAGCTTA\n",
      "0 AGGAGGATAAAAAGCTTA motifs detected in file\n",
      "\n",
      "Seq is: TAGAACAAGTAACAATTC\n",
      "0 TAGAACAAGTAACAATTC motifs detected in file\n",
      "\n",
      "Seq is: CAACACCTACTACCTGGG\n",
      "0 CAACACCTACTACCTGGG motifs detected in file\n",
      "\n",
      "Seq is: CCTCGTGTATTTATCATA\n",
      "0 CCTCGTGTATTTATCATA motifs detected in file\n",
      "\n",
      "Seq is: CCGAAACTTCTTCCGCAC\n",
      "0 CCGAAACTTCTTCCGCAC motifs detected in file\n",
      "\n",
      "Seq is: CCATGTCTACACGTCCTT\n",
      "0 CCATGTCTACACGTCCTT motifs detected in file\n",
      "\n",
      "Seq is: ACGATCCACACCAACCAA\n",
      "0 ACGATCCACACCAACCAA motifs detected in file\n",
      "\n",
      "Seq is: CTACGGCTAACATTGATA\n",
      "0 CTACGGCTAACATTGATA motifs detected in file\n",
      "\n",
      "Seq is: ATACTCAGGATCAACTGA\n",
      "0 ATACTCAGGATCAACTGA motifs detected in file\n",
      "\n",
      "Seq is: ACTTCACCTCTTCCGATG\n",
      "0 ACTTCACCTCTTCCGATG motifs detected in file\n",
      "\n",
      "Seq is: CTAAAAAAACTACAGTAA\n",
      "0 CTAAAAAAACTACAGTAA motifs detected in file\n",
      "\n",
      "Seq is: AACACATTCCCGCGTCTG\n",
      "0 AACACATTCCCGCGTCTG motifs detected in file\n",
      "\n",
      "Seq is: ACGACGCTTGTCTCACCA\n",
      "0 ACGACGCTTGTCTCACCA motifs detected in file\n",
      "\n",
      "Seq is: CTATCGCGGATTCGTATC\n",
      "0 CTATCGCGGATTCGTATC motifs detected in file\n",
      "\n",
      "Seq is: ACAATCTCAAACCTAATC\n",
      "0 ACAATCTCAAACCTAATC motifs detected in file\n",
      "\n",
      "Seq is: ACCCGATAAAAGACGATC\n",
      "0 ACCCGATAAAAGACGATC motifs detected in file\n",
      "\n",
      "Seq is: CCCCACCTTCGCCTACTC\n",
      "0 CCCCACCTTCGCCTACTC motifs detected in file\n",
      "\n",
      "Seq is: ATTCTTCTAAGGACGATG\n",
      "0 ATTCTTCTAAGGACGATG motifs detected in file\n",
      "\n",
      "Seq is: TTGATATCACGAACTCAC\n",
      "0 TTGATATCACGAACTCAC motifs detected in file\n",
      "\n",
      "Seq is: CTCATCCCGACTCTGACC\n",
      "0 CTCATCCCGACTCTGACC motifs detected in file\n",
      "\n",
      "Seq is: CCACCCCGGAAGATACCG\n",
      "0 CCACCCCGGAAGATACCG motifs detected in file\n",
      "\n",
      "Seq is: ACTCCCCTCAGTCCCCAC\n",
      "0 ACTCCCCTCAGTCCCCAC motifs detected in file\n",
      "\n",
      "Seq is: CTCTGGATTAACCCGACT\n",
      "0 CTCTGGATTAACCCGACT motifs detected in file\n",
      "\n",
      "Seq is: AAACGGACGACGACGATC\n",
      "0 AAACGGACGACGACGATC motifs detected in file\n",
      "\n",
      "Seq is: CTCCTCCGGATCATCACC\n",
      "0 CTCCTCCGGATCATCACC motifs detected in file\n",
      "\n",
      "Seq is: CCACTCCGAACCTAATCG\n",
      "0 CCACTCCGAACCTAATCG motifs detected in file\n",
      "\n",
      "Seq is: CAAATCTTCCCGATAAAC\n",
      "0 CAAATCTTCCCGATAAAC motifs detected in file\n",
      "\n",
      "Seq is: AACACCCAGAGTACTCAA\n",
      "0 AACACCCAGAGTACTCAA motifs detected in file\n",
      "\n",
      "Seq is: CCTCACATCCTATTTAGT\n",
      "0 CCTCACATCCTATTTAGT motifs detected in file\n",
      "\n",
      "Seq is: ACGATTCCGCTCCTACTT\n",
      "0 ACGATTCCGCTCCTACTT motifs detected in file\n",
      "\n",
      "Seq is: TGGCTATAGCGGCTATGC\n",
      "0 TGGCTATAGCGGCTATGC motifs detected in file\n",
      "\n",
      "Seq is: CAACATATCCTAACGAAC\n",
      "0 CAACATATCCTAACGAAC motifs detected in file\n",
      "\n",
      "Seq is: TTACCGACGACACAACCG\n",
      "0 TTACCGACGACACAACCG motifs detected in file\n",
      "\n",
      "Seq is: TAGACGAGCCCGCATAGT\n",
      "0 TAGACGAGCCCGCATAGT motifs detected in file\n",
      "\n",
      "Seq is: AGTTGACTACTCGTTCTT\n",
      "0 AGTTGACTACTCGTTCTT motifs detected in file\n",
      "\n",
      "Seq is: CCTATATTAAGGATGCGG\n",
      "0 CCTATATTAAGGATGCGG motifs detected in file\n",
      "\n",
      "Seq is: GAGAGTCGGCTACTTGTC\n",
      "0 GAGAGTCGGCTACTTGTC motifs detected in file\n",
      "\n",
      "Seq is: AACCTTATCCAACAATCG\n",
      "0 AACCTTATCCAACAATCG motifs detected in file\n",
      "\n",
      "Seq is: CCATTGATTCTAATCATA\n",
      "0 CCATTGATTCTAATCATA motifs detected in file\n",
      "\n",
      "Seq is: CCATTAATCCTTCTACTC\n",
      "0 CCATTAATCCTTCTACTC motifs detected in file\n",
      "\n",
      "Seq is: ATCTATAAACTTCTTGAC\n",
      "0 ATCTATAAACTTCTTGAC motifs detected in file\n",
      "\n",
      "Seq is: TAATTACAAACCCAGACT\n",
      "0 TAATTACAAACCCAGACT motifs detected in file\n",
      "\n",
      "Seq is: CAAATATATAGTGTCACT\n",
      "0 CAAATATATAGTGTCACT motifs detected in file\n",
      "\n",
      "Seq is: CTTAAGATACTACCTGGT\n",
      "0 CTTAAGATACTACCTGGT motifs detected in file\n",
      "\n",
      "Seq is: ACATTGCTTGTTACGATG\n",
      "0 ACATTGCTTGTTACGATG motifs detected in file\n",
      "\n",
      "Seq is: TCCCTACTTATAATACCT\n",
      "0 TCCCTACTTATAATACCT motifs detected in file\n",
      "\n",
      "Seq is: CTTCATCAGATCAAACTT\n",
      "0 CTTCATCAGATCAAACTT motifs detected in file\n",
      "\n",
      "Seq is: CGAATCCCTCTCGACCCA\n",
      "0 CGAATCCCTCTCGACCCA motifs detected in file\n",
      "\n",
      "Seq is: ACAAACCCAACACCGAGT\n",
      "0 ACAAACCCAACACCGAGT motifs detected in file\n",
      "\n",
      "Seq is: CACAGTCAAGCTCTATTA\n",
      "0 CACAGTCAAGCTCTATTA motifs detected in file\n",
      "\n",
      "Seq is: TTGAAGAACCAGATCCGT\n",
      "0 TTGAAGAACCAGATCCGT motifs detected in file\n",
      "\n",
      "Seq is: GTACTTATAACAACACCC\n",
      "0 GTACTTATAACAACACCC motifs detected in file\n",
      "\n",
      "Seq is: CTTCTCTGACTATTATTT\n",
      "0 CTTCTCTGACTATTATTT motifs detected in file\n",
      "\n",
      "Seq is: CCACCTACGCTGTTACCT\n",
      "0 CCACCTACGCTGTTACCT motifs detected in file\n",
      "\n",
      "Seq is: AAAATGTATTACCCCCAT\n",
      "0 AAAATGTATTACCCCCAT motifs detected in file\n",
      "\n",
      "Seq is: ACTCAAAAAAAACAATCC\n",
      "0 ACTCAAAAAAAACAATCC motifs detected in file\n",
      "\n",
      "Seq is: CAATTGCTCCCACCATTC\n",
      "0 CAATTGCTCCCACCATTC motifs detected in file\n",
      "\n",
      "Seq is: CTACCCCCCTTAATCCCT\n",
      "0 CTACCCCCCTTAATCCCT motifs detected in file\n",
      "\n",
      "Seq is: TCAGTCCCAATCCCACCA\n",
      "0 TCAGTCCCAATCCCACCA motifs detected in file\n",
      "\n",
      "Seq is: ATATCATCACACGTACCA\n",
      "0 ATATCATCACACGTACCA motifs detected in file\n",
      "\n",
      "Seq is: ATAATGAAAATAAACCTC\n",
      "0 ATAATGAAAATAAACCTC motifs detected in file\n",
      "\n",
      "Seq is: AACGTGGTTTTAAAAACC\n",
      "0 AACGTGGTTTTAAAAACC motifs detected in file\n",
      "\n",
      "Seq is: CCGGATTCTGGTTACCTA\n",
      "0 CCGGATTCTGGTTACCTA motifs detected in file\n",
      "\n",
      "Seq is: TCGACAATAGGAAATTTT\n",
      "0 TCGACAATAGGAAATTTT motifs detected in file\n",
      "\n",
      "Seq is: CAACTCTTTCGGTACAAC\n",
      "0 CAACTCTTTCGGTACAAC motifs detected in file\n",
      "\n",
      "Seq is: AATCTGTACCCCCGTACT\n",
      "0 AATCTGTACCCCCGTACT motifs detected in file\n",
      "\n",
      "Seq is: CAATCGTCAAGAACACTC\n",
      "0 CAATCGTCAAGAACACTC motifs detected in file\n",
      "\n",
      "Seq is: GAAAGAGCCATTTATTCC\n",
      "0 GAAAGAGCCATTTATTCC motifs detected in file\n",
      "\n",
      "Seq is: CCAGCATTCGGAGACAAC\n",
      "0 CCAGCATTCGGAGACAAC motifs detected in file\n",
      "\n",
      "Seq is: AGTCTAAGTGTTAGACTA\n",
      "0 AGTCTAAGTGTTAGACTA motifs detected in file\n",
      "\n",
      "Seq is: CAAAACCAATTTGATATA\n",
      "0 CAAAACCAATTTGATATA motifs detected in file\n",
      "\n",
      "Seq is: AATGTTCTCCTTTTGGGC\n",
      "0 AATGTTCTCCTTTTGGGC motifs detected in file\n",
      "\n",
      "Seq is: CATTACTACAGCCCCAAC\n",
      "0 CATTACTACAGCCCCAAC motifs detected in file\n",
      "\n",
      "Seq is: TCCCTATCCTCCTCTTAC\n",
      "0 TCCCTATCCTCCTCTTAC motifs detected in file\n",
      "\n",
      "Seq is: CCCCTATCCACATACTTG\n",
      "0 CCCCTATCCACATACTTG motifs detected in file\n",
      "\n",
      "Seq is: TACTCCCACAAAAGAGCA\n",
      "0 TACTCCCACAAAAGAGCA motifs detected in file\n",
      "\n",
      "Seq is: CACTTACTCCCAAAATAC\n",
      "0 CACTTACTCCCAAAATAC motifs detected in file\n",
      "\n",
      "Seq is: AACAATTACACCACCCAC\n",
      "0 AACAATTACACCACCCAC motifs detected in file\n",
      "\n",
      "Seq is: TCACTCGGGGTAACACAA\n",
      "0 TCACTCGGGGTAACACAA motifs detected in file\n",
      "\n",
      "Seq is: CACCATTTATACATCTCC\n",
      "0 CACCATTTATACATCTCC motifs detected in file\n",
      "\n",
      "Seq is: CTCATATCCCGACACTGA\n",
      "0 CTCATATCCCGACACTGA motifs detected in file\n",
      "\n",
      "Seq is: TCATACAACTCAGGACAT\n",
      "0 TCATACAACTCAGGACAT motifs detected in file\n",
      "\n",
      "Seq is: TCATCCTCTCACTATAAA\n",
      "0 TCATCCTCTCACTATAAA motifs detected in file\n",
      "\n",
      "Seq is: CTAGTCCTCTTGCACCAA\n",
      "0 CTAGTCCTCTTGCACCAA motifs detected in file\n",
      "\n",
      "Seq is: TGATCGTGTCTCTCAAGA\n",
      "0 TGATCGTGTCTCTCAAGA motifs detected in file\n",
      "\n",
      "Seq is: GGGTCATCCAATTATCAC\n",
      "0 GGGTCATCCAATTATCAC motifs detected in file\n",
      "\n",
      "Seq is: CCCCCATTCCGCTCCAAT\n",
      "0 CCCCCATTCCGCTCCAAT motifs detected in file\n",
      "\n",
      "Seq is: CGCTCCGAACGATCTTCA\n",
      "0 CGCTCCGAACGATCTTCA motifs detected in file\n",
      "\n",
      "Seq is: GTAGTTTTTCGATAATCA\n",
      "0 GTAGTTTTTCGATAATCA motifs detected in file\n",
      "\n",
      "Seq is: CCCTCATCTCAAACTTCA\n",
      "0 CCCTCATCTCAAACTTCA motifs detected in file\n",
      "\n",
      "Seq is: GGAACTCTCTCCTAATAC\n",
      "0 GGAACTCTCTCCTAATAC motifs detected in file\n",
      "\n",
      "Seq is: TACGCTGACACTCACGCA\n",
      "0 TACGCTGACACTCACGCA motifs detected in file\n",
      "\n",
      "Seq is: AGCATCAAACTCAAACGA\n",
      "0 AGCATCAAACTCAAACGA motifs detected in file\n",
      "\n",
      "Seq is: TCCGTCTTATCATTACTC\n",
      "0 TCCGTCTTATCATTACTC motifs detected in file\n",
      "\n",
      "Seq is: CTACATTCGGGCACCCGC\n",
      "0 CTACATTCGGGCACCCGC motifs detected in file\n",
      "\n",
      "Seq is: TAATACTCTTACTGACGC\n",
      "0 TAATACTCTTACTGACGC motifs detected in file\n",
      "\n",
      "Seq is: GGCCACTTCGAAGTCCCC\n",
      "0 GGCCACTTCGAAGTCCCC motifs detected in file\n",
      "\n",
      "Seq is: CAAACCTACTCTTACCGA\n",
      "0 CAAACCTACTCTTACCGA motifs detected in file\n",
      "\n",
      "Seq is: CAATGATGCTCCCGATAC\n",
      "0 CAATGATGCTCCCGATAC motifs detected in file\n",
      "\n",
      "Seq is: ACCGACTAACTTCTCATA\n",
      "0 ACCGACTAACTTCTCATA motifs detected in file\n",
      "\n",
      "Seq is: CGTTACTCGCTAAAATCC\n",
      "0 CGTTACTCGCTAAAATCC motifs detected in file\n",
      "\n",
      "Seq is: AGACAAACAGCATCCGTC\n",
      "0 AGACAAACAGCATCCGTC motifs detected in file\n",
      "\n",
      "Seq is: TACCTCGAACAATATTAA\n",
      "0 TACCTCGAACAATATTAA motifs detected in file\n",
      "\n",
      "Seq is: TACGGAGTATCCCTATCA\n",
      "0 TACGGAGTATCCCTATCA motifs detected in file\n",
      "\n",
      "Seq is: TGTTCCTTCCCCATCCGA\n",
      "0 TGTTCCTTCCCCATCCGA motifs detected in file\n",
      "\n",
      "Seq is: TACACAAAACAGTCCCCC\n",
      "0 TACACAAAACAGTCCCCC motifs detected in file\n",
      "\n",
      "Seq is: AACTCTTACTCACACTCC\n",
      "0 AACTCTTACTCACACTCC motifs detected in file\n",
      "\n",
      "Seq is: GCATAATATGGTATCGGC\n",
      "0 GCATAATATGGTATCGGC motifs detected in file\n",
      "\n",
      "Seq is: GGATCAAAATTCTCATGA\n",
      "0 GGATCAAAATTCTCATGA motifs detected in file\n",
      "\n",
      "Seq is: CGCCGTTCATGATAACTG\n",
      "0 CGCCGTTCATGATAACTG motifs detected in file\n",
      "\n",
      "Seq is: GGTCGCTACCCCCGAAGC\n",
      "0 GGTCGCTACCCCCGAAGC motifs detected in file\n",
      "\n",
      "Seq is: TGTACCCGAAATCCCTCA\n",
      "0 TGTACCCGAAATCCCTCA motifs detected in file\n",
      "\n",
      "Seq is: GTATTCACCTCAGGCATT\n",
      "0 GTATTCACCTCAGGCATT motifs detected in file\n",
      "\n",
      "Seq is: TCTCCATAGAAATGATAT\n",
      "0 TCTCCATAGAAATGATAT motifs detected in file\n",
      "\n",
      "Seq is: TTTCGATAACACATTCGA\n",
      "0 TTTCGATAACACATTCGA motifs detected in file\n",
      "\n",
      "Seq is: TCAGTATAATTCAACAAC\n",
      "0 TCAGTATAATTCAACAAC motifs detected in file\n",
      "\n",
      "Seq is: CGAGTCCTCAAACTATCA\n",
      "0 CGAGTCCTCAAACTATCA motifs detected in file\n",
      "\n",
      "Seq is: AGAACCCGTCACTCTCAC\n",
      "0 AGAACCCGTCACTCTCAC motifs detected in file\n",
      "\n",
      "Seq is: TCATCATCTTACAAATCA\n",
      "0 TCATCATCTTACAAATCA motifs detected in file\n",
      "\n",
      "Seq is: CTCGGATCCCACAACACT\n",
      "0 CTCGGATCCCACAACACT motifs detected in file\n",
      "\n",
      "Seq is: CACATTTAATCACGCTAC\n",
      "0 CACATTTAATCACGCTAC motifs detected in file\n",
      "\n",
      "Seq is: TCATCCCCTTCCCTCGGA\n",
      "0 TCATCCCCTTCCCTCGGA motifs detected in file\n",
      "\n",
      "Seq is: TGATCCCACATCTTATCC\n",
      "0 TGATCCCACATCTTATCC motifs detected in file\n",
      "\n",
      "Seq is: TTCATACACGGACGCAAG\n",
      "0 TTCATACACGGACGCAAG motifs detected in file\n",
      "\n",
      "Seq is: TCCGCAAGACCGACCAAC\n",
      "0 TCCGCAAGACCGACCAAC motifs detected in file\n",
      "\n",
      "Seq is: GGAGTAGCCCACTACTAT\n",
      "0 GGAGTAGCCCACTACTAT motifs detected in file\n",
      "\n",
      "Seq is: CGGTTCCACCCCTATTCA\n",
      "0 CGGTTCCACCCCTATTCA motifs detected in file\n",
      "\n",
      "Seq is: CACCAAAGCTTCTTCTAT\n",
      "0 CACCAAAGCTTCTTCTAT motifs detected in file\n",
      "\n",
      "Seq is: ATTTTATACTAATCAAGA\n",
      "0 ATTTTATACTAATCAAGA motifs detected in file\n",
      "\n",
      "Seq is: CACCGACACTTACAATAT\n",
      "0 CACCGACACTTACAATAT motifs detected in file\n",
      "\n",
      "Seq is: TAATTCCTCTAAACATCC\n",
      "0 TAATTCCTCTAAACATCC motifs detected in file\n",
      "\n",
      "Seq is: CTCTAATCATATCTCTCC\n",
      "0 CTCTAATCATATCTCTCC motifs detected in file\n",
      "\n",
      "Seq is: ATCTCAAAAAAAGCACTA\n",
      "0 ATCTCAAAAAAAGCACTA motifs detected in file\n",
      "\n",
      "Seq is: TCACCAAGTGACCTATTC\n",
      "0 TCACCAAGTGACCTATTC motifs detected in file\n",
      "\n",
      "Seq is: ACCGCAACCGAACGGTAC\n",
      "0 ACCGCAACCGAACGGTAC motifs detected in file\n",
      "\n",
      "Seq is: TAACACTCCCCATCCTCA\n",
      "0 TAACACTCCCCATCCTCA motifs detected in file\n",
      "\n",
      "Seq is: GTCCATCAATCATAATCC\n",
      "0 GTCCATCAATCATAATCC motifs detected in file\n",
      "\n",
      "Seq is: TCCCCCCAACAATCCCCC\n",
      "0 TCCCCCCAACAATCCCCC motifs detected in file\n",
      "\n",
      "Seq is: AGCCTCCTTTTCCAACCC\n",
      "0 AGCCTCCTTTTCCAACCC motifs detected in file\n",
      "\n",
      "Seq is: CTTGTCGATTTATCCAAC\n",
      "0 CTTGTCGATTTATCCAAC motifs detected in file\n",
      "\n",
      "Seq is: AACAACTAAACCAATTTT\n",
      "0 AACAACTAAACCAATTTT motifs detected in file\n",
      "\n",
      "Seq is: TTATCATCTCCCTACTAC\n",
      "0 TTATCATCTCCCTACTAC motifs detected in file\n",
      "\n",
      "Seq is: GATTATTAATCCGACACC\n",
      "0 GATTATTAATCCGACACC motifs detected in file\n",
      "\n",
      "Seq is: CACCAACACAACTAAGTT\n",
      "0 CACCAACACAACTAAGTT motifs detected in file\n",
      "\n",
      "Seq is: TAATACACAAAAAACCTT\n",
      "0 TAATACACAAAAAACCTT motifs detected in file\n",
      "\n",
      "Seq is: TCAGTACAGTCACCATCA\n",
      "0 TCAGTACAGTCACCATCA motifs detected in file\n",
      "\n",
      "Seq is: TTATATTAACAACCCTGC\n",
      "0 TTATATTAACAACCCTGC motifs detected in file\n",
      "\n",
      "Seq is: TAATCAAAATCGTAACCT\n",
      "0 TAATCAAAATCGTAACCT motifs detected in file\n",
      "\n",
      "Seq is: CATCCAAATCCAATACAT\n",
      "0 CATCCAAATCCAATACAT motifs detected in file\n",
      "\n",
      "Seq is: GCATCAGATCCGGTATAC\n",
      "0 GCATCAGATCCGGTATAC motifs detected in file\n",
      "\n",
      "Seq is: ACAACCTCTAACTCTGAT\n",
      "0 ACAACCTCTAACTCTGAT motifs detected in file\n",
      "\n",
      "Seq is: CATCCCGATCCGGGTGGC\n",
      "0 CATCCCGATCCGGGTGGC motifs detected in file\n",
      "\n",
      "Seq is: GACGAAGCGTCCGCCGTT\n",
      "0 GACGAAGCGTCCGCCGTT motifs detected in file\n",
      "\n",
      "Seq is: TCTGATCATACCGTTATC\n",
      "0 TCTGATCATACCGTTATC motifs detected in file\n",
      "\n",
      "Seq is: CGTGTTATAACCGATTCT\n",
      "0 CGTGTTATAACCGATTCT motifs detected in file\n",
      "\n",
      "Seq is: CCCTCACCCACAACTCCC\n",
      "0 CCCTCACCCACAACTCCC motifs detected in file\n",
      "\n",
      "Seq is: AATACTCTCATCGATATT\n",
      "0 AATACTCTCATCGATATT motifs detected in file\n",
      "\n",
      "Seq is: ACTTGTCGCTATCATAAT\n",
      "0 ACTTGTCGCTATCATAAT motifs detected in file\n",
      "\n",
      "Seq is: AAGGAAGATCCGTATCAT\n",
      "0 AAGGAAGATCCGTATCAT motifs detected in file\n",
      "\n",
      "Seq is: CCCTCCTATACTCCACAC\n",
      "0 CCCTCCTATACTCCACAC motifs detected in file\n",
      "\n",
      "Seq is: TCGCTATATGATCATAAG\n",
      "0 TCGCTATATGATCATAAG motifs detected in file\n",
      "\n",
      "Seq is: GATCTTCACTCTACCATT\n",
      "0 GATCTTCACTCTACCATT motifs detected in file\n",
      "\n",
      "Seq is: TACGATCATATTATAAAT\n",
      "0 TACGATCATATTATAAAT motifs detected in file\n",
      "\n",
      "Seq is: ACATTTACTCCCCGTAAA\n",
      "0 ACATTTACTCCCCGTAAA motifs detected in file\n",
      "\n",
      "Seq is: CCATTTATACTAATAGTA\n",
      "0 CCATTTATACTAATAGTA motifs detected in file\n",
      "\n",
      "Seq is: TTAAATTACTCAGCTTTA\n",
      "0 TTAAATTACTCAGCTTTA motifs detected in file\n",
      "\n",
      "Seq is: GTAAGCAAAACAAATTTG\n",
      "0 GTAAGCAAAACAAATTTG motifs detected in file\n",
      "\n",
      "Seq is: ATATATGGTTAAGCCAAG\n",
      "0 ATATATGGTTAAGCCAAG motifs detected in file\n",
      "\n",
      "Seq is: TCAGATTAGGAAAAACAT\n",
      "0 TCAGATTAGGAAAAACAT motifs detected in file\n",
      "\n",
      "Seq is: CAGTGAGTATCCGGTCTG\n",
      "0 CAGTGAGTATCCGGTCTG motifs detected in file\n",
      "\n",
      "Seq is: AATCCCGATCCTACTACT\n",
      "0 AATCCCGATCCTACTACT motifs detected in file\n",
      "\n",
      "Seq is: AATTATTCTCCCTACTGT\n",
      "0 AATTATTCTCCCTACTGT motifs detected in file\n",
      "\n",
      "Seq is: ATTGATAATCACCGTCCA\n",
      "0 ATTGATAATCACCGTCCA motifs detected in file\n",
      "\n",
      "Seq is: ATCAACAAACATCCCGAG\n",
      "0 ATCAACAAACATCCCGAG motifs detected in file\n",
      "\n",
      "Seq is: TACCATCCCCATTTTCCT\n",
      "0 TACCATCCCCATTTTCCT motifs detected in file\n",
      "\n",
      "Seq is: CCCGTTAAAGATCTAGTT\n",
      "0 CCCGTTAAAGATCTAGTT motifs detected in file\n",
      "\n",
      "Seq is: TATTATTCTTCCATTATC\n",
      "0 TATTATTCTTCCATTATC motifs detected in file\n",
      "\n",
      "Seq is: GATGATTCTTCTTAAAAT\n",
      "0 GATGATTCTTCTTAAAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCTCTTTCCCTGCGCCC\n",
      "0 ACCTCTTTCCCTGCGCCC motifs detected in file\n",
      "\n",
      "Seq is: GCCCCCTATATCCCAGCT\n",
      "0 GCCCCCTATATCCCAGCT motifs detected in file\n",
      "\n",
      "Seq is: TCGGCGTGAGCATTCCCC\n",
      "0 TCGGCGTGAGCATTCCCC motifs detected in file\n",
      "\n",
      "Seq is: ACCTAAAAAGATACATCG\n",
      "0 ACCTAAAAAGATACATCG motifs detected in file\n",
      "\n",
      "Seq is: GCAACTCAACACCATCAG\n",
      "0 GCAACTCAACACCATCAG motifs detected in file\n",
      "\n",
      "Seq is: TTTTACATTATTAATAAT\n",
      "0 TTTTACATTATTAATAAT motifs detected in file\n",
      "\n",
      "Seq is: CATCATTCCGATCCTCCC\n",
      "0 CATCATTCCGATCCTCCC motifs detected in file\n",
      "\n",
      "Seq is: ACAACTAATAATTTTAAT\n",
      "0 ACAACTAATAATTTTAAT motifs detected in file\n",
      "\n",
      "Seq is: TCCGCTTCAAATAATGAG\n",
      "0 TCCGCTTCAAATAATGAG motifs detected in file\n",
      "\n",
      "Seq is: AAAAAACTTACAACAGTT\n",
      "0 AAAAAACTTACAACAGTT motifs detected in file\n",
      "\n",
      "Seq is: TTGATCAATTAACCTTCA\n",
      "0 TTGATCAATTAACCTTCA motifs detected in file\n",
      "\n",
      "Seq is: ATTGCCATGATAAATATG\n",
      "0 ATTGCCATGATAAATATG motifs detected in file\n",
      "\n",
      "Seq is: ATTTTCTCATTCTGGGAG\n",
      "0 ATTTTCTCATTCTGGGAG motifs detected in file\n",
      "\n",
      "Seq is: TAGTTATCTACCTCTGTA\n",
      "0 TAGTTATCTACCTCTGTA motifs detected in file\n",
      "\n",
      "Seq is: TGTCTTTATCAGTTTGGT\n",
      "0 TGTCTTTATCAGTTTGGT motifs detected in file\n",
      "\n",
      "Seq is: GTAGATGTTTTACGGTCA\n",
      "0 GTAGATGTTTTACGGTCA motifs detected in file\n",
      "\n",
      "Seq is: TAGTCCGCCGCCGAAGCT\n",
      "0 TAGTCCGCCGCCGAAGCT motifs detected in file\n",
      "\n",
      "Seq is: TCGGTTTCACTACAAACC\n",
      "0 TCGGTTTCACTACAAACC motifs detected in file\n",
      "\n",
      "Seq is: TACATTTCACTTTATAAT\n",
      "0 TACATTTCACTTTATAAT motifs detected in file\n",
      "\n",
      "Seq is: CAACCGCCTACTTCGTCT\n",
      "0 CAACCGCCTACTTCGTCT motifs detected in file\n",
      "\n",
      "Seq is: ATCACTCCTTTCAACTCG\n",
      "0 ATCACTCCTTTCAACTCG motifs detected in file\n",
      "\n",
      "Seq is: GTTATTACTGCACTTCAG\n",
      "0 GTTATTACTGCACTTCAG motifs detected in file\n",
      "\n",
      "Seq is: GCACCTTCGGACACCGAT\n",
      "0 GCACCTTCGGACACCGAT motifs detected in file\n",
      "\n",
      "Seq is: GTTTTTTACAACTCGGCA\n",
      "0 GTTTTTTACAACTCGGCA motifs detected in file\n",
      "\n",
      "Seq is: TCTACGGCAGCCTTTACC\n",
      "0 TCTACGGCAGCCTTTACC motifs detected in file\n",
      "\n",
      "Seq is: ACTTCCCTCTGAGCTTCA\n",
      "0 ACTTCCCTCTGAGCTTCA motifs detected in file\n",
      "\n",
      "Seq is: TGAGACTCCGAACATCCT\n",
      "0 TGAGACTCCGAACATCCT motifs detected in file\n",
      "\n",
      "Seq is: CCCATTTTATCTCTGGGT\n",
      "0 CCCATTTTATCTCTGGGT motifs detected in file\n",
      "\n",
      "Seq is: CATTTTAACATTATTCGT\n",
      "0 CATTTTAACATTATTCGT motifs detected in file\n",
      "\n",
      "Seq is: CACGAACTTAATAAACCA\n",
      "0 CACGAACTTAATAAACCA motifs detected in file\n",
      "\n",
      "Seq is: AAGCCAACAAAAGATAAT\n",
      "0 AAGCCAACAAAAGATAAT motifs detected in file\n",
      "\n",
      "Seq is: CTGATACCACTCGAGTCC\n",
      "0 CTGATACCACTCGAGTCC motifs detected in file\n",
      "\n",
      "Seq is: ACTAACTATGAGGACTAC\n",
      "0 ACTAACTATGAGGACTAC motifs detected in file\n",
      "\n",
      "Seq is: GCTCATTATGCCTACACA\n",
      "0 GCTCATTATGCCTACACA motifs detected in file\n",
      "\n",
      "Seq is: AATCCTCACCCTGAAGAT\n",
      "0 AATCCTCACCCTGAAGAT motifs detected in file\n",
      "\n",
      "Seq is: CCCCTAAATCGCCCCACT\n",
      "0 CCCCTAAATCGCCCCACT motifs detected in file\n",
      "\n",
      "Seq is: ACGGACAACCCCCGGTCA\n",
      "0 ACGGACAACCCCCGGTCA motifs detected in file\n",
      "\n",
      "Seq is: CGGGAGGATTAACCCCCC\n",
      "0 CGGGAGGATTAACCCCCC motifs detected in file\n",
      "\n",
      "Seq is: ATCCCCGATCCGACCTCA\n",
      "0 ATCCCCGATCCGACCTCA motifs detected in file\n",
      "\n",
      "Seq is: CCATTTTCCGAGTCTTTT\n",
      "0 CCATTTTCCGAGTCTTTT motifs detected in file\n",
      "\n",
      "Seq is: TAGGACGCTTCTTTTTTT\n",
      "0 TAGGACGCTTCTTTTTTT motifs detected in file\n",
      "\n",
      "Seq is: GAAGACTCCATTATTTAT\n",
      "0 GAAGACTCCATTATTTAT motifs detected in file\n",
      "\n",
      "Seq is: CCTAATAGGGCATAGCTT\n",
      "0 CCTAATAGGGCATAGCTT motifs detected in file\n",
      "\n",
      "Seq is: CCGGAAAAACCTGTCCAC\n",
      "0 CCGGAAAAACCTGTCCAC motifs detected in file\n",
      "\n",
      "Seq is: CACACACCACCGGAACCA\n",
      "0 CACACACCACCGGAACCA motifs detected in file\n",
      "\n",
      "Seq is: TACACGAAAGAGCACAAT\n",
      "0 TACACGAAAGAGCACAAT motifs detected in file\n",
      "\n",
      "Seq is: GTAGCGCGGTAGTAACCA\n",
      "0 GTAGCGCGGTAGTAACCA motifs detected in file\n",
      "\n",
      "Seq is: TATACCAATCACACAACC\n",
      "0 TATACCAATCACACAACC motifs detected in file\n",
      "\n",
      "Seq is: AATCATCCGGATCATACT\n",
      "0 AATCATCCGGATCATACT motifs detected in file\n",
      "\n",
      "Seq is: CCTCGCAATACCTCACCT\n",
      "0 CCTCGCAATACCTCACCT motifs detected in file\n",
      "\n",
      "Seq is: TCACTTTAGTGTACCGAT\n",
      "0 TCACTTTAGTGTACCGAT motifs detected in file\n",
      "\n",
      "Seq is: CCGGCCTCCAGTAATCCT\n",
      "0 CCGGCCTCCAGTAATCCT motifs detected in file\n",
      "\n",
      "Seq is: CCCGACTCTCCCGGGGAC\n",
      "0 CCCGACTCTCCCGGGGAC motifs detected in file\n",
      "\n",
      "Seq is: AATCCCCAGTACCCGACC\n",
      "0 AATCCCCAGTACCCGACC motifs detected in file\n",
      "\n",
      "Seq is: CAAAATGATATACTATCC\n",
      "0 CAAAATGATATACTATCC motifs detected in file\n",
      "\n",
      "Seq is: GTACACTAACCACCCAGT\n",
      "0 GTACACTAACCACCCAGT motifs detected in file\n",
      "\n",
      "Seq is: AATACACAACAGCACGTC\n",
      "0 AATACACAACAGCACGTC motifs detected in file\n",
      "\n",
      "Seq is: CATCTCCGAATGATCTTC\n",
      "0 CATCTCCGAATGATCTTC motifs detected in file\n",
      "\n",
      "Seq is: ACACTTTTGCATCCGAAC\n",
      "0 ACACTTTTGCATCCGAAC motifs detected in file\n",
      "\n",
      "Seq is: CTAATTCCGCTGTCGCTA\n",
      "0 CTAATTCCGCTGTCGCTA motifs detected in file\n",
      "\n",
      "Seq is: AAGATCCTATCAGTCATC\n",
      "0 AAGATCCTATCAGTCATC motifs detected in file\n",
      "\n",
      "Seq is: TTAATCTTAACACTTCTA\n",
      "0 TTAATCTTAACACTTCTA motifs detected in file\n",
      "\n",
      "Seq is: CTATTCACATCTCCCTTC\n",
      "0 CTATTCACATCTCCCTTC motifs detected in file\n",
      "\n",
      "Seq is: CAATTACCAACTATAACG\n",
      "0 CAATTACCAACTATAACG motifs detected in file\n",
      "\n",
      "Seq is: ATCCCACCGCGAAGGTTA\n",
      "0 ATCCCACCGCGAAGGTTA motifs detected in file\n",
      "\n",
      "Seq is: ATCCACGTACTCATCCAC\n",
      "0 ATCCACGTACTCATCCAC motifs detected in file\n",
      "\n",
      "Seq is: CGGACGTCATTACAATCG\n",
      "0 CGGACGTCATTACAATCG motifs detected in file\n",
      "\n",
      "Seq is: CCAATCCGCATGCCGGTC\n",
      "0 CCAATCCGCATGCCGGTC motifs detected in file\n",
      "\n",
      "Seq is: CCGATAACCAACTTACTC\n",
      "0 CCGATAACCAACTTACTC motifs detected in file\n",
      "\n",
      "Seq is: ATCCGACTACCAAAGCTA\n",
      "0 ATCCGACTACCAAAGCTA motifs detected in file\n",
      "\n",
      "Seq is: TTATTGATCATACCCCTA\n",
      "0 TTATTGATCATACCCCTA motifs detected in file\n",
      "\n",
      "Seq is: TTCCCCACATCCACACGG\n",
      "0 TTCCCCACATCCACACGG motifs detected in file\n",
      "\n",
      "Seq is: AACACCATTCTTCACCCG\n",
      "0 AACACCATTCTTCACCCG motifs detected in file\n",
      "\n",
      "Seq is: ATCCCGTAAAAATTAGAA\n",
      "0 ATCCCGTAAAAATTAGAA motifs detected in file\n",
      "\n",
      "Seq is: TCTCGCTTTCGGATATTA\n",
      "0 TCTCGCTTTCGGATATTA motifs detected in file\n",
      "\n",
      "Seq is: GTGACACGGGCGAGTATT\n",
      "0 GTGACACGGGCGAGTATT motifs detected in file\n",
      "\n",
      "Seq is: CCCCTACCGGTACCGATC\n",
      "0 CCCCTACCGGTACCGATC motifs detected in file\n",
      "\n",
      "Seq is: CAAATATCTATCAACCCA\n",
      "0 CAAATATCTATCAACCCA motifs detected in file\n",
      "\n",
      "Seq is: CCAACCACATTTACTCAC\n",
      "0 CCAACCACATTTACTCAC motifs detected in file\n",
      "\n",
      "Seq is: TCCGTCCTCAGGCTCCTC\n",
      "0 TCCGTCCTCAGGCTCCTC motifs detected in file\n",
      "\n",
      "Seq is: CAATCAACACCGTTATTT\n",
      "0 CAATCAACACCGTTATTT motifs detected in file\n",
      "\n",
      "Seq is: TTACTAATTCCTATGATC\n",
      "0 TTACTAATTCCTATGATC motifs detected in file\n",
      "\n",
      "Seq is: ATATTCTCTAGTCCAAGC\n",
      "0 ATATTCTCTAGTCCAAGC motifs detected in file\n",
      "\n",
      "Seq is: AGGAAATCACAACACATA\n",
      "0 AGGAAATCACAACACATA motifs detected in file\n",
      "\n",
      "Seq is: CCAATAGTAAACAAAACT\n",
      "0 CCAATAGTAAACAAAACT motifs detected in file\n",
      "\n",
      "Seq is: CCAATCAAACTAATCAGT\n",
      "0 CCAATCAAACTAATCAGT motifs detected in file\n",
      "\n",
      "Seq is: AACAACCCACCACTAATC\n",
      "0 AACAACCCACCACTAATC motifs detected in file\n",
      "\n",
      "Seq is: AGCCAACAACTACTCTAT\n",
      "0 AGCCAACAACTACTCTAT motifs detected in file\n",
      "\n",
      "Seq is: AAACCTCCACCCCTAGTT\n",
      "0 AAACCTCCACCCCTAGTT motifs detected in file\n",
      "\n",
      "Seq is: ATCTCCCCCTTTATCTTA\n",
      "0 ATCTCCCCCTTTATCTTA motifs detected in file\n",
      "\n",
      "Seq is: CTAGTCATGACGCCGCCC\n",
      "0 CTAGTCATGACGCCGCCC motifs detected in file\n",
      "\n",
      "Seq is: ATCCGGATCCTAACACCC\n",
      "0 ATCCGGATCCTAACACCC motifs detected in file\n",
      "\n",
      "Seq is: CCGTTACTTACTTCGCTT\n",
      "0 CCGTTACTTACTTCGCTT motifs detected in file\n",
      "\n",
      "Seq is: GTCTAAAAGCAAGTAAAA\n",
      "0 GTCTAAAAGCAAGTAAAA motifs detected in file\n",
      "\n",
      "Seq is: CCAAGAGTCCCAAACAAT\n",
      "0 CCAAGAGTCCCAAACAAT motifs detected in file\n",
      "\n",
      "Seq is: ATTAAAAAATAAAAATAC\n",
      "0 ATTAAAAAATAAAAATAC motifs detected in file\n",
      "\n",
      "Seq is: CCGAAACCACTCCCTCCA\n",
      "0 CCGAAACCACTCCCTCCA motifs detected in file\n",
      "\n",
      "Seq is: TCCACCATCAAACACAAA\n",
      "0 TCCACCATCAAACACAAA motifs detected in file\n",
      "\n",
      "Seq is: TTATAAAAATCAACCCAC\n",
      "0 TTATAAAAATCAACCCAC motifs detected in file\n",
      "\n",
      "Seq is: TACTCCTTATCACATTCC\n",
      "0 TACTCCTTATCACATTCC motifs detected in file\n",
      "\n",
      "Seq is: TCATACCCCCATTAATAC\n",
      "0 TCATACCCCCATTAATAC motifs detected in file\n",
      "\n",
      "Seq is: CACCCGGTATGCCATCAT\n",
      "0 CACCCGGTATGCCATCAT motifs detected in file\n",
      "\n",
      "Seq is: AAATCAACCCCGTAAAGT\n",
      "0 AAATCAACCCCGTAAAGT motifs detected in file\n",
      "\n",
      "Seq is: GACATTTCTCCACAACCA\n",
      "0 GACATTTCTCCACAACCA motifs detected in file\n",
      "\n",
      "Seq is: AGAGAATTAGAAATTGAA\n",
      "0 AGAGAATTAGAAATTGAA motifs detected in file\n",
      "\n",
      "Seq is: TTTTCCAATTACGATTCA\n",
      "0 TTTTCCAATTACGATTCA motifs detected in file\n",
      "\n",
      "Seq is: ATCGAAATGTCACCCGAG\n",
      "0 ATCGAAATGTCACCCGAG motifs detected in file\n",
      "\n",
      "Seq is: ATCTCCCCCATCTCCCCC\n",
      "0 ATCTCCCCCATCTCCCCC motifs detected in file\n",
      "\n",
      "Seq is: ACGATATCCCATTTATGC\n",
      "0 ACGATATCCCATTTATGC motifs detected in file\n",
      "\n",
      "Seq is: CCGGGATAAAGTTTCTAA\n",
      "0 CCGGGATAAAGTTTCTAA motifs detected in file\n",
      "\n",
      "Seq is: AAATCCCCTTAATTAAGA\n",
      "0 AAATCCCCTTAATTAAGA motifs detected in file\n",
      "\n",
      "Seq is: TCCTGCTACCCGTACTTT\n",
      "0 TCCTGCTACCCGTACTTT motifs detected in file\n",
      "\n",
      "Seq is: GACACCAAACGAGGTGTC\n",
      "0 GACACCAAACGAGGTGTC motifs detected in file\n",
      "\n",
      "Seq is: TAAAGTCTCGTAACTGGC\n",
      "0 TAAAGTCTCGTAACTGGC motifs detected in file\n",
      "\n",
      "Seq is: ATCATATGGGGGCCAGCA\n",
      "0 ATCATATGGGGGCCAGCA motifs detected in file\n",
      "\n",
      "Seq is: CATCGCCACTTTCACCAA\n",
      "0 CATCGCCACTTTCACCAA motifs detected in file\n",
      "\n",
      "Seq is: ACCAAATCTGCAGGCCCT\n",
      "0 ACCAAATCTGCAGGCCCT motifs detected in file\n",
      "\n",
      "Seq is: TAACGTAGACAAAAATTC\n",
      "0 TAACGTAGACAAAAATTC motifs detected in file\n",
      "\n",
      "Seq is: GGATTACACCCCTGTCGA\n",
      "0 GGATTACACCCCTGTCGA motifs detected in file\n",
      "\n",
      "Seq is: GTACTCACGTTCTGCAGA\n",
      "0 GTACTCACGTTCTGCAGA motifs detected in file\n",
      "\n",
      "Seq is: ACACTACATTAATAATAT\n",
      "0 ACACTACATTAATAATAT motifs detected in file\n",
      "\n",
      "Seq is: GCTTACCCCCGAAGTTAG\n",
      "0 GCTTACCCCCGAAGTTAG motifs detected in file\n",
      "\n",
      "Seq is: CCCTCATGATGAGCTAAC\n",
      "0 CCCTCATGATGAGCTAAC motifs detected in file\n",
      "\n",
      "Seq is: AGTTGCAGTTCCTCAGCG\n",
      "0 AGTTGCAGTTCCTCAGCG motifs detected in file\n",
      "\n",
      "Seq is: TCCAGCGGACCAAGATCC\n",
      "0 TCCAGCGGACCAAGATCC motifs detected in file\n",
      "\n",
      "Seq is: TTATTACCCCCTTCATAC\n",
      "0 TTATTACCCCCTTCATAC motifs detected in file\n",
      "\n",
      "Seq is: ATCCTCAACTTCTAATCA\n",
      "0 ATCCTCAACTTCTAATCA motifs detected in file\n",
      "\n",
      "Seq is: GGCGGCATCAGCCACATG\n",
      "0 GGCGGCATCAGCCACATG motifs detected in file\n",
      "\n",
      "Seq is: AGCATCCAAGTCATGGTA\n",
      "0 AGCATCCAAGTCATGGTA motifs detected in file\n",
      "\n",
      "Seq is: ACCACCGGTTAACTAAAC\n",
      "0 ACCACCGGTTAACTAAAC motifs detected in file\n",
      "\n",
      "Seq is: TACCATTCCCTCCCTAGC\n",
      "0 TACCATTCCCTCCCTAGC motifs detected in file\n",
      "\n",
      "Seq is: AACTGGAGCAGACAATAC\n",
      "0 AACTGGAGCAGACAATAC motifs detected in file\n",
      "\n",
      "Seq is: ATTTCCTACGCATCCCTA\n",
      "0 ATTTCCTACGCATCCCTA motifs detected in file\n",
      "\n",
      "Seq is: CCCTCCCGCTACTCCTGA\n",
      "0 CCCTCCCGCTACTCCTGA motifs detected in file\n",
      "\n",
      "Seq is: TCCTACTACCGCCCGTCC\n",
      "0 TCCTACTACCGCCCGTCC motifs detected in file\n",
      "\n",
      "Seq is: TATCAAGTCTGCCAAAGA\n",
      "0 TATCAAGTCTGCCAAAGA motifs detected in file\n",
      "\n",
      "Seq is: TAAAGGACTCGCAGACTC\n",
      "0 TAAAGGACTCGCAGACTC motifs detected in file\n",
      "\n",
      "Seq is: TACAATCATAATCAATCA\n",
      "0 TACAATCATAATCAATCA motifs detected in file\n",
      "\n",
      "Seq is: AAACAACACTCACAATCC\n",
      "0 AAACAACACTCACAATCC motifs detected in file\n",
      "\n",
      "Seq is: TTTTCCCGTATGTCCTGA\n",
      "0 TTTTCCCGTATGTCCTGA motifs detected in file\n",
      "\n",
      "Seq is: TCCTTCGTCTATTCCTTT\n",
      "0 TCCTTCGTCTATTCCTTT motifs detected in file\n",
      "\n",
      "Seq is: TACTAATACTCCCGCACT\n",
      "0 TACTAATACTCCCGCACT motifs detected in file\n",
      "\n",
      "Seq is: AGTACTTTCCACTATTCG\n",
      "0 AGTACTTTCCACTATTCG motifs detected in file\n",
      "\n",
      "Seq is: AGAAGATACTATCCCCTT\n",
      "0 AGAAGATACTATCCCCTT motifs detected in file\n",
      "\n",
      "Seq is: CATCGCAGAACATCTGGA\n",
      "0 CATCGCAGAACATCTGGA motifs detected in file\n",
      "\n",
      "Seq is: TGAACGCGACGTACACGG\n",
      "0 TGAACGCGACGTACACGG motifs detected in file\n",
      "\n",
      "Seq is: TAATTCTATATATCCTAA\n",
      "0 TAATTCTATATATCCTAA motifs detected in file\n",
      "\n",
      "Seq is: ATCGGATATTAAATTGAA\n",
      "0 ATCGGATATTAAATTGAA motifs detected in file\n",
      "\n",
      "Seq is: ACTGTTTCAATACTTTAC\n",
      "0 ACTGTTTCAATACTTTAC motifs detected in file\n",
      "\n",
      "Seq is: CAAAAAGATTATGGAAAA\n",
      "0 CAAAAAGATTATGGAAAA motifs detected in file\n",
      "\n",
      "Seq is: ACTTTTTCAGTACCTCCG\n",
      "0 ACTTTTTCAGTACCTCCG motifs detected in file\n",
      "\n",
      "Seq is: GTACCCCAACCGAACTTT\n",
      "0 GTACCCCAACCGAACTTT motifs detected in file\n",
      "\n",
      "Seq is: GGTCGAAACCCCCCAAGC\n",
      "0 GGTCGAAACCCCCCAAGC motifs detected in file\n",
      "\n",
      "Seq is: TAAGGAAGGAAAAAACAG\n",
      "0 TAAGGAAGGAAAAAACAG motifs detected in file\n",
      "\n",
      "Seq is: ATCTAAAATACATATGCC\n",
      "0 ATCTAAAATACATATGCC motifs detected in file\n",
      "\n",
      "Seq is: CAAGAAGCTTACACACCA\n",
      "0 CAAGAAGCTTACACACCA motifs detected in file\n",
      "\n",
      "Seq is: TCCCACCCCCCGTAGGTA\n",
      "0 TCCCACCCCCCGTAGGTA motifs detected in file\n",
      "\n",
      "Seq is: TATCAGTGAGGTCCAAAT\n",
      "0 TATCAGTGAGGTCCAAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCTCCCAAGAAGATGAT\n",
      "0 ACCTCCCAAGAAGATGAT motifs detected in file\n",
      "\n",
      "Seq is: AATCCTGAAAAGCGAAGC\n",
      "0 AATCCTGAAAAGCGAAGC motifs detected in file\n",
      "\n",
      "Seq is: TTCGCTTCCGAAGAGTTT\n",
      "0 TTCGCTTCCGAAGAGTTT motifs detected in file\n",
      "\n",
      "Seq is: AGTACTTTTAATAATTAT\n",
      "0 AGTACTTTTAATAATTAT motifs detected in file\n",
      "\n",
      "Seq is: AATGACGACAATCTCTTT\n",
      "0 AATGACGACAATCTCTTT motifs detected in file\n",
      "\n",
      "Seq is: ACTTACTCGGATGTCTAC\n",
      "0 ACTTACTCGGATGTCTAC motifs detected in file\n",
      "\n",
      "Seq is: TATCCTACAAAGTACACC\n",
      "0 TATCCTACAAAGTACACC motifs detected in file\n",
      "\n",
      "Seq is: ACATACGTAGCCCCATCA\n",
      "0 ACATACGTAGCCCCATCA motifs detected in file\n",
      "\n",
      "Seq is: GGCTCATTGCAGCCCCGT\n",
      "0 GGCTCATTGCAGCCCCGT motifs detected in file\n",
      "\n",
      "Seq is: AAGGCCTATCCGGCTCTT\n",
      "0 AAGGCCTATCCGGCTCTT motifs detected in file\n",
      "\n",
      "Seq is: TCACAACACCCTTCTTTC\n",
      "0 TCACAACACCCTTCTTTC motifs detected in file\n",
      "\n",
      "Seq is: AATCTAAATGCGGCTACT\n",
      "0 AATCTAAATGCGGCTACT motifs detected in file\n",
      "\n",
      "Seq is: TATACTATCACTTTACCT\n",
      "0 TATACTATCACTTTACCT motifs detected in file\n",
      "\n",
      "Seq is: AAAACCGCATCCAAACCA\n",
      "0 AAAACCGCATCCAAACCA motifs detected in file\n",
      "\n",
      "Seq is: GATCCCACATCGGACTCT\n",
      "0 GATCCCACATCGGACTCT motifs detected in file\n",
      "\n",
      "Seq is: TATCCCCTTTAGTCACTT\n",
      "0 TATCCCCTTTAGTCACTT motifs detected in file\n",
      "\n",
      "Seq is: ACTTCGGAGGATACTACC\n",
      "0 ACTTCGGAGGATACTACC motifs detected in file\n",
      "\n",
      "Seq is: GTTTATGTCGAGGATAAC\n",
      "0 GTTTATGTCGAGGATAAC motifs detected in file\n",
      "\n",
      "Seq is: TATCCTGTATCACCTTCA\n",
      "0 TATCCTGTATCACCTTCA motifs detected in file\n",
      "\n",
      "Seq is: CCCGATGTTGCATCATGC\n",
      "0 CCCGATGTTGCATCATGC motifs detected in file\n",
      "\n",
      "Seq is: ACAGCACATCATGCTACA\n",
      "0 ACAGCACATCATGCTACA motifs detected in file\n",
      "\n",
      "Seq is: GATCACTACTCAAACGAT\n",
      "0 GATCACTACTCAAACGAT motifs detected in file\n",
      "\n",
      "Seq is: TATGTTACGGTCAGTCCG\n",
      "0 TATGTTACGGTCAGTCCG motifs detected in file\n",
      "\n",
      "Seq is: GTGGATGCCACTTTTCTT\n",
      "0 GTGGATGCCACTTTTCTT motifs detected in file\n",
      "\n",
      "Seq is: TCTACTTAGGATCCCGAG\n",
      "0 TCTACTTAGGATCCCGAG motifs detected in file\n",
      "\n",
      "Seq is: TCTCGTGACGTCGTCTAG\n",
      "0 TCTCGTGACGTCGTCTAG motifs detected in file\n",
      "\n",
      "Seq is: TAAAGTATAACGAAGGCA\n",
      "0 TAAAGTATAACGAAGGCA motifs detected in file\n",
      "\n",
      "Seq is: CCTCACACCGCTCAGTCG\n",
      "0 CCTCACACCGCTCAGTCG motifs detected in file\n",
      "\n",
      "Seq is: ATTTATGAAACTGCGGCC\n",
      "0 ATTTATGAAACTGCGGCC motifs detected in file\n",
      "\n",
      "Seq is: ACCCCTATCGCTACTAAT\n",
      "0 ACCCCTATCGCTACTAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCATCGCCTCCACTTTA\n",
      "0 ACCATCGCCTCCACTTTA motifs detected in file\n",
      "\n",
      "Seq is: TACGAGCACACAGATGCA\n",
      "0 TACGAGCACACAGATGCA motifs detected in file\n",
      "\n",
      "Seq is: GATAAGGATGACATTTAT\n",
      "0 GATAAGGATGACATTTAT motifs detected in file\n",
      "\n",
      "Seq is: ATACCACACGAGTGTGCT\n",
      "0 ATACCACACGAGTGTGCT motifs detected in file\n",
      "\n",
      "Seq is: ATTTGGGATCCTTCGGTT\n",
      "0 ATTTGGGATCCTTCGGTT motifs detected in file\n",
      "\n",
      "Seq is: AACTATAGTATCGAGTCT\n",
      "0 AACTATAGTATCGAGTCT motifs detected in file\n",
      "\n",
      "Seq is: GGTATGGATACATAGGTT\n",
      "0 GGTATGGATACATAGGTT motifs detected in file\n",
      "\n",
      "Seq is: TACCAAGAAAAAAAGGCC\n",
      "0 TACCAAGAAAAAAAGGCC motifs detected in file\n",
      "\n",
      "Seq is: TCATCATTCAATGTTATA\n",
      "0 TCATCATTCAATGTTATA motifs detected in file\n",
      "\n",
      "Seq is: CCCTCTAATAAGGCTTCG\n",
      "0 CCCTCTAATAAGGCTTCG motifs detected in file\n",
      "\n",
      "Seq is: GACCATCCTATTCTTATA\n",
      "0 GACCATCCTATTCTTATA motifs detected in file\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Seq is: TTTGAAGTCCCACTGGCT\n",
      "0 TTTGAAGTCCCACTGGCT motifs detected in file\n",
      "\n",
      "Seq is: TTTTAGTCTTATCCACAA\n",
      "0 TTTTAGTCTTATCCACAA motifs detected in file\n",
      "\n",
      "Seq is: CCATATCTTACCCCAGAG\n",
      "0 CCATATCTTACCCCAGAG motifs detected in file\n",
      "\n",
      "Seq is: GAGGAGGCCGCCCCAGCT\n",
      "0 GAGGAGGCCGCCCCAGCT motifs detected in file\n",
      "\n",
      "Seq is: TCTTCCACCACAACTCCA\n",
      "0 TCTTCCACCACAACTCCA motifs detected in file\n",
      "\n",
      "Seq is: ACGCCAGACAATCATCAT\n",
      "0 ACGCCAGACAATCATCAT motifs detected in file\n",
      "\n",
      "Seq is: ATCACTACGGTCGTCGAT\n",
      "0 ATCACTACGGTCGTCGAT motifs detected in file\n",
      "\n",
      "Seq is: CCTGACCCTCTCTATCCT\n",
      "0 CCTGACCCTCTCTATCCT motifs detected in file\n",
      "\n",
      "Seq is: CTTCATCCTGACGACACT\n",
      "0 CTTCATCCTGACGACACT motifs detected in file\n",
      "\n",
      "Seq is: AATCCTGCCTAGTCTGCT\n",
      "0 AATCCTGCCTAGTCTGCT motifs detected in file\n",
      "\n",
      "Seq is: TCTCCCCGCAAACCATAA\n",
      "0 TCTCCCCGCAAACCATAA motifs detected in file\n",
      "\n",
      "Seq is: CCCAATACCGTCCCCCAA\n",
      "0 CCCAATACCGTCCCCCAA motifs detected in file\n",
      "\n",
      "Seq is: AATATAACTATTAACAAC\n",
      "0 AATATAACTATTAACAAC motifs detected in file\n",
      "\n",
      "Seq is: ACTACTTTAACTACCGGG\n",
      "0 ACTACTTTAACTACCGGG motifs detected in file\n",
      "\n",
      "Seq is: GATTCTATCTCCTCTGTG\n",
      "0 GATTCTATCTCCTCTGTG motifs detected in file\n",
      "\n",
      "Seq is: GACGATCCACATTCCTCT\n",
      "0 GACGATCCACATTCCTCT motifs detected in file\n",
      "\n",
      "Seq is: TCTACCAATCCAGATGCC\n",
      "0 TCTACCAATCCAGATGCC motifs detected in file\n",
      "\n",
      "Seq is: TCCGAGGTCCCACCCTCA\n",
      "0 TCCGAGGTCCCACCCTCA motifs detected in file\n",
      "\n",
      "Seq is: TCAAGGGACGATTCCCTC\n",
      "0 TCAAGGGACGATTCCCTC motifs detected in file\n",
      "\n",
      "Seq is: CCATCTGACAAGTTGGAC\n",
      "0 CCATCTGACAAGTTGGAC motifs detected in file\n",
      "\n",
      "Seq is: AAGGACGAGGCCGGAGGT\n",
      "0 AAGGACGAGGCCGGAGGT motifs detected in file\n",
      "\n",
      "Seq is: GATATCGTCTACGCTCGT\n",
      "0 GATATCGTCTACGCTCGT motifs detected in file\n",
      "\n",
      "Seq is: CCTCATCCTCTCTCCCTC\n",
      "0 CCTCATCCTCTCTCCCTC motifs detected in file\n",
      "\n",
      "Seq is: CATTCTCAGTCTTCGAAT\n",
      "0 CATTCTCAGTCTTCGAAT motifs detected in file\n",
      "\n",
      "Seq is: ACAACAAATACGCCCCTT\n",
      "0 ACAACAAATACGCCCCTT motifs detected in file\n",
      "\n",
      "Seq is: TGCGGTATAGCCCCCGTG\n",
      "0 TGCGGTATAGCCCCCGTG motifs detected in file\n",
      "\n",
      "Seq is: GCTAATAATCCCCTTGAT\n",
      "0 GCTAATAATCCCCTTGAT motifs detected in file\n",
      "\n",
      "Seq is: CAGTCAACGGTTTCGGAG\n",
      "0 CAGTCAACGGTTTCGGAG motifs detected in file\n",
      "\n",
      "Seq is: GCTAATACTACCCATAAT\n",
      "0 GCTAATACTACCCATAAT motifs detected in file\n",
      "\n",
      "Seq is: GATACTTCTTCTAATAAT\n",
      "0 GATACTTCTTCTAATAAT motifs detected in file\n",
      "\n",
      "Seq is: GTTTACGTACCCGACACT\n",
      "0 GTTTACGTACCCGACACT motifs detected in file\n",
      "\n",
      "Seq is: GCTATTGCAACATCTACA\n",
      "0 GCTATTGCAACATCTACA motifs detected in file\n",
      "\n",
      "Seq is: CCAGCAATGGATCTTCCA\n",
      "0 CCAGCAATGGATCTTCCA motifs detected in file\n",
      "\n",
      "Seq is: ACGGACCGACCGGGTCGA\n",
      "0 ACGGACCGACCGGGTCGA motifs detected in file\n",
      "\n",
      "Seq is: GCCGAGCTTATTCCTCCG\n",
      "0 GCCGAGCTTATTCCTCCG motifs detected in file\n",
      "\n",
      "Seq is: AATCTCGACACGGATCCT\n",
      "0 AATCTCGACACGGATCCT motifs detected in file\n",
      "\n",
      "Seq is: GAGGTCGAGTACGCGGCT\n",
      "0 GAGGTCGAGTACGCGGCT motifs detected in file\n",
      "\n",
      "Seq is: TATTATCCATATCACAAG\n",
      "0 TATTATCCATATCACAAG motifs detected in file\n",
      "\n",
      "Seq is: GTTACAGAAACACCAAAC\n",
      "0 GTTACAGAAACACCAAAC motifs detected in file\n",
      "\n",
      "Seq is: ATCTCTTATCAGTTGCCA\n",
      "0 ATCTCTTATCAGTTGCCA motifs detected in file\n",
      "\n",
      "Seq is: GCCGCTTGTAGTCACCCC\n",
      "0 GCCGCTTGTAGTCACCCC motifs detected in file\n",
      "\n",
      "Seq is: CACTCCATTTTACCGACT\n",
      "0 CACTCCATTTTACCGACT motifs detected in file\n",
      "\n",
      "Seq is: CACTTCGTAACCTGACAT\n",
      "0 CACTTCGTAACCTGACAT motifs detected in file\n",
      "\n",
      "Seq is: TTAGATTTCTGTCCCCAA\n",
      "0 TTAGATTTCTGTCCCCAA motifs detected in file\n",
      "\n",
      "Seq is: TCCGGAGAAAAATGGTCG\n",
      "0 TCCGGAGAAAAATGGTCG motifs detected in file\n",
      "\n",
      "Seq is: AGGCTCCACTAAAAGTAT\n",
      "0 AGGCTCCACTAAAAGTAT motifs detected in file\n",
      "\n",
      "Seq is: AACTTAACGTTTAAGCTT\n",
      "0 AACTTAACGTTTAAGCTT motifs detected in file\n",
      "\n",
      "Seq is: CTTCGTCGAAGTTTGGAC\n",
      "0 CTTCGTCGAAGTTTGGAC motifs detected in file\n",
      "\n",
      "Seq is: GGCCCCGAAGAGGGCGGA\n",
      "0 GGCCCCGAAGAGGGCGGA motifs detected in file\n",
      "\n",
      "Seq is: AAAAAGGGCCGCCGCCCT\n",
      "0 AAAAAGGGCCGCCGCCCT motifs detected in file\n",
      "\n",
      "Seq is: CTTCATCTAACTTCGGTC\n",
      "0 CTTCATCTAACTTCGGTC motifs detected in file\n",
      "\n",
      "Seq is: AACTAATCCCACGAATCG\n",
      "0 AACTAATCCCACGAATCG motifs detected in file\n",
      "\n",
      "Seq is: ACAATTGATTCACAAACA\n",
      "0 ACAATTGATTCACAAACA motifs detected in file\n",
      "\n",
      "Seq is: CCCAAATTCAGGGTAACC\n",
      "0 CCCAAATTCAGGGTAACC motifs detected in file\n",
      "\n",
      "Seq is: AGATCATTCCCGAATCGA\n",
      "0 AGATCATTCCCGAATCGA motifs detected in file\n",
      "\n",
      "Seq is: ATTAATTTCACCGACTAA\n",
      "0 ATTAATTTCACCGACTAA motifs detected in file\n",
      "\n",
      "Seq is: ACGCAAGTCAACTACGTC\n",
      "0 ACGCAAGTCAACTACGTC motifs detected in file\n",
      "\n",
      "Seq is: TCACCCCAAAACGTCAGG\n",
      "0 TCACCCCAAAACGTCAGG motifs detected in file\n",
      "\n",
      "Seq is: AATCGACAATGTCTTTAA\n",
      "0 AATCGACAATGTCTTTAA motifs detected in file\n",
      "\n",
      "Seq is: TTCATAACGTTGAATGAC\n",
      "0 TTCATAACGTTGAATGAC motifs detected in file\n",
      "\n",
      "Seq is: TCCCGAAACTTCCGAGAA\n",
      "0 TCCCGAAACTTCCGAGAA motifs detected in file\n",
      "\n",
      "Seq is: CCAGACATAAATTGGATT\n",
      "0 CCAGACATAAATTGGATT motifs detected in file\n",
      "\n",
      "Seq is: TAAAGATATTCTAATAAT\n",
      "0 TAAAGATATTCTAATAAT motifs detected in file\n",
      "\n",
      "Seq is: CATATTTTCCCCTCTATC\n",
      "0 CATATTTTCCCCTCTATC motifs detected in file\n",
      "\n",
      "Seq is: CATCCTCATCGCACCATT\n",
      "0 CATCCTCATCGCACCATT motifs detected in file\n",
      "\n",
      "Seq is: CCCGCTACTCACACCCCT\n",
      "0 CCCGCTACTCACACCCCT motifs detected in file\n",
      "\n",
      "Seq is: CCTTACCCCACCCAAAAC\n",
      "0 CCTTACCCCACCCAAAAC motifs detected in file\n",
      "\n",
      "Seq is: ATACAAGTTTGACAGTAA\n",
      "0 ATACAAGTTTGACAGTAA motifs detected in file\n",
      "\n",
      "Seq is: AATAAAAATGCAACAATC\n",
      "0 AATAAAAATGCAACAATC motifs detected in file\n",
      "\n",
      "Seq is: TATACCCCTCATCACACT\n",
      "0 TATACCCCTCATCACACT motifs detected in file\n",
      "\n",
      "Seq is: AACTCCACCTCATCTAAT\n",
      "0 AACTCCACCTCATCTAAT motifs detected in file\n",
      "\n",
      "Seq is: CCGCATCCATCTTCATCT\n",
      "0 CCGCATCCATCTTCATCT motifs detected in file\n",
      "\n",
      "Seq is: CCAATTCCTCCCACTACC\n",
      "0 CCAATTCCTCCCACTACC motifs detected in file\n",
      "\n",
      "Seq is: ACCGATACTACCACCCCT\n",
      "0 ACCGATACTACCACCCCT motifs detected in file\n",
      "\n",
      "Seq is: ACTACTCCGATAACAAAA\n",
      "0 ACTACTCCGATAACAAAA motifs detected in file\n",
      "\n",
      "Seq is: AACACTTAAGAAGCTATT\n",
      "0 AACACTTAAGAAGCTATT motifs detected in file\n",
      "\n",
      "Seq is: ACCGGGTAAACCCGTTTT\n",
      "0 ACCGGGTAAACCCGTTTT motifs detected in file\n",
      "\n",
      "Seq is: TCGGCCAATCGCCCCCGT\n",
      "0 TCGGCCAATCGCCCCCGT motifs detected in file\n",
      "\n",
      "Seq is: CCGGAGGATCCCTCTCCT\n",
      "0 CCGGAGGATCCCTCTCCT motifs detected in file\n",
      "\n",
      "Seq is: CCCACCTACCTTAATTCC\n",
      "0 CCCACCTACCTTAATTCC motifs detected in file\n",
      "\n",
      "Seq is: CACAATCAGTACAATCGA\n",
      "0 CACAATCAGTACAATCGA motifs detected in file\n",
      "\n",
      "Seq is: ACAAAGTCCACGCTCTAT\n",
      "0 ACAAAGTCCACGCTCTAT motifs detected in file\n",
      "\n",
      "Seq is: CATCATCCCAGCACCACG\n",
      "0 CATCATCCCAGCACCACG motifs detected in file\n",
      "\n",
      "Seq is: ACCTCAAATTCAACTCAT\n",
      "0 ACCTCAAATTCAACTCAT motifs detected in file\n",
      "\n",
      "Seq is: CATCCTTACGCCATCATC\n",
      "0 CATCCTTACGCCATCATC motifs detected in file\n",
      "\n",
      "Seq is: AATCCTATTATATTTATC\n",
      "0 AATCCTATTATATTTATC motifs detected in file\n",
      "\n",
      "Seq is: AATTTAATTCTTACCAAT\n",
      "0 AATTTAATTCTTACCAAT motifs detected in file\n",
      "\n",
      "Seq is: ACAATCCCAACATGCCAT\n",
      "0 ACAATCCCAACATGCCAT motifs detected in file\n",
      "\n",
      "Seq is: CTTGACGATAATAAGTAG\n",
      "0 CTTGACGATAATAAGTAG motifs detected in file\n",
      "\n",
      "Seq is: GATACACCCATTAACTCC\n",
      "0 GATACACCCATTAACTCC motifs detected in file\n",
      "\n",
      "Seq is: TCATACGATTCTAAAACG\n",
      "0 TCATACGATTCTAAAACG motifs detected in file\n",
      "\n",
      "Seq is: CATCGACCCAAACCAAAT\n",
      "0 CATCGACCCAAACCAAAT motifs detected in file\n",
      "\n",
      "Seq is: TAGGTGGAGTTGACGGAC\n",
      "0 TAGGTGGAGTTGACGGAC motifs detected in file\n",
      "\n",
      "Seq is: GATACTACCTATTCTAAC\n",
      "0 GATACTACCTATTCTAAC motifs detected in file\n",
      "\n",
      "Seq is: TCTCTCACTCCTCTTCCG\n",
      "0 TCTCTCACTCCTCTTCCG motifs detected in file\n",
      "\n",
      "Seq is: AATGCAAATCACTCCCTC\n",
      "0 AATGCAAATCACTCCCTC motifs detected in file\n",
      "\n",
      "Seq is: TCTAAACCATATACTAAC\n",
      "0 TCTAAACCATATACTAAC motifs detected in file\n",
      "\n",
      "Seq is: TCTACCCCCGATCAAAAA\n",
      "0 TCTACCCCCGATCAAAAA motifs detected in file\n",
      "\n",
      "Seq is: CAGTACACTCTTCTTCGT\n",
      "0 CAGTACACTCTTCTTCGT motifs detected in file\n",
      "\n",
      "Seq is: CCGGCCTACAGTCTCCCC\n",
      "0 CCGGCCTACAGTCTCCCC motifs detected in file\n",
      "\n",
      "Seq is: ACGGAACCCATTGGAGAC\n",
      "0 ACGGAACCCATTGGAGAC motifs detected in file\n",
      "\n",
      "Seq is: CCTGAGTCTTCACTTTCC\n",
      "0 CCTGAGTCTTCACTTTCC motifs detected in file\n",
      "\n",
      "Seq is: CCCGATAAGGATCAAAAT\n",
      "0 CCCGATAAGGATCAAAAT motifs detected in file\n",
      "\n",
      "Seq is: AACGATATCGATAATACT\n",
      "0 AACGATATCGATAATACT motifs detected in file\n",
      "\n",
      "Seq is: AATAATTACTACTCATAA\n",
      "0 AATAATTACTACTCATAA motifs detected in file\n",
      "\n",
      "Seq is: CTAACCATCATAACCAAT\n",
      "0 CTAACCATCATAACCAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCAAGTAACAGGCCTCT\n",
      "0 ACCAAGTAACAGGCCTCT motifs detected in file\n",
      "\n",
      "Seq is: CATATAACAACTTCTCCT\n",
      "0 CATATAACAACTTCTCCT motifs detected in file\n",
      "\n",
      "Seq is: ATCGATAATCTTCCTAAT\n",
      "0 ATCGATAATCTTCCTAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCTACGCCAACGAACGC\n",
      "0 ACCTACGCCAACGAACGC motifs detected in file\n",
      "\n",
      "Seq is: ACTCCTTTATGAACTACC\n",
      "0 ACTCCTTTATGAACTACC motifs detected in file\n",
      "\n",
      "Seq is: GTCGAAGACACCTTGCTC\n",
      "0 GTCGAAGACACCTTGCTC motifs detected in file\n",
      "\n",
      "Seq is: CCAAATAAAAAAACCAAT\n",
      "0 CCAAATAAAAAAACCAAT motifs detected in file\n",
      "\n",
      "Seq is: CTTGACCTTATTTTCGAT\n",
      "0 CTTGACCTTATTTTCGAT motifs detected in file\n",
      "\n",
      "Seq is: CGTACAAATAAAGATCCG\n",
      "0 CGTACAAATAAAGATCCG motifs detected in file\n",
      "\n",
      "Seq is: GATGAGTCCATTTTTTAG\n",
      "0 GATGAGTCCATTTTTTAG motifs detected in file\n",
      "\n",
      "Seq is: TCACGCTCGAATCGCGAC\n",
      "0 TCACGCTCGAATCGCGAC motifs detected in file\n",
      "\n",
      "Seq is: ACTACTCACACGGACGTT\n",
      "0 ACTACTCACACGGACGTT motifs detected in file\n",
      "\n",
      "Seq is: TCTACCATCTCATCTACT\n",
      "0 TCTACCATCTCATCTACT motifs detected in file\n",
      "\n",
      "Seq is: GCCCAACCCGGTCCCCTA\n",
      "0 GCCCAACCCGGTCCCCTA motifs detected in file\n",
      "\n",
      "Seq is: ATTAATCATGCCCTTCCC\n",
      "0 ATTAATCATGCCCTTCCC motifs detected in file\n",
      "\n",
      "Seq is: ATATTGGTTGTAAAAGCC\n",
      "0 ATATTGGTTGTAAAAGCC motifs detected in file\n",
      "\n",
      "Seq is: CCATACCCGGGCTATCGA\n",
      "0 CCATACCCGGGCTATCGA motifs detected in file\n",
      "\n",
      "Seq is: ATAAATCGACTGGAATGA\n",
      "0 ATAAATCGACTGGAATGA motifs detected in file\n",
      "\n",
      "Seq is: AATCCTACCCCACACTAT\n",
      "0 AATCCTACCCCACACTAT motifs detected in file\n",
      "\n",
      "Seq is: CCACCGTGCCTCTTAAAA\n",
      "0 CCACCGTGCCTCTTAAAA motifs detected in file\n",
      "\n",
      "Seq is: CCTAAGAGTCCCTACCCA\n",
      "0 CCTAAGAGTCCCTACCCA motifs detected in file\n",
      "\n",
      "Seq is: AGCTAAGAGTATCAGGAT\n",
      "0 AGCTAAGAGTATCAGGAT motifs detected in file\n",
      "\n",
      "Seq is: CTTTATTCCCCCAAATTC\n",
      "0 CTTTATTCCCCCAAATTC motifs detected in file\n",
      "\n",
      "Seq is: GAGGATAATAAATGAGAT\n",
      "0 GAGGATAATAAATGAGAT motifs detected in file\n",
      "\n",
      "Seq is: AGTTTCATTGAGAAAATA\n",
      "0 AGTTTCATTGAGAAAATA motifs detected in file\n",
      "\n",
      "Seq is: GTCTGTATAAAGAATCCA\n",
      "0 GTCTGTATAAAGAATCCA motifs detected in file\n",
      "\n",
      "Seq is: AACTCCCCCTTACGACCT\n",
      "0 AACTCCCCCTTACGACCT motifs detected in file\n",
      "\n",
      "Seq is: CTAACATTACCCATACCT\n",
      "0 CTAACATTACCCATACCT motifs detected in file\n",
      "\n",
      "Seq is: CTGTATAGTATATTCATT\n",
      "0 CTGTATAGTATATTCATT motifs detected in file\n",
      "\n",
      "Seq is: ACGATCCCACTCACCATC\n",
      "0 ACGATCCCACTCACCATC motifs detected in file\n",
      "\n",
      "Seq is: CTTCAAAAAAGTATCCTC\n",
      "0 CTTCAAAAAAGTATCCTC motifs detected in file\n",
      "\n",
      "Seq is: CACATACTCAACCAGCAT\n",
      "0 CACATACTCAACCAGCAT motifs detected in file\n",
      "\n",
      "Seq is: CGCCTTAGCCCCCATACG\n",
      "0 CGCCTTAGCCCCCATACG motifs detected in file\n",
      "\n",
      "Seq is: ACAAGCTTAAGTATTCTT\n",
      "0 ACAAGCTTAAGTATTCTT motifs detected in file\n",
      "\n",
      "Seq is: GTCCCTCCAATCTTCATC\n",
      "0 GTCCCTCCAATCTTCATC motifs detected in file\n",
      "\n",
      "Seq is: CCAGAACCACTGTTTTAT\n",
      "0 CCAGAACCACTGTTTTAT motifs detected in file\n",
      "\n",
      "Seq is: ACAACACATCTCAAGTCC\n",
      "0 ACAACACATCTCAAGTCC motifs detected in file\n",
      "\n",
      "Seq is: CCTCTCACGCAGTATACA\n",
      "0 CCTCTCACGCAGTATACA motifs detected in file\n",
      "\n",
      "Seq is: ACAAGGATCCTTCTAACA\n",
      "0 ACAAGGATCCTTCTAACA motifs detected in file\n",
      "\n",
      "Seq is: TCACCACTCCCACAAATA\n",
      "0 TCACCACTCCCACAAATA motifs detected in file\n",
      "\n",
      "Seq is: ATATTATTACAAACACAT\n",
      "0 ATATTATTACAAACACAT motifs detected in file\n",
      "\n",
      "Seq is: AAGCCGATACTTCTTATC\n",
      "0 AAGCCGATACTTCTTATC motifs detected in file\n",
      "\n",
      "Seq is: CCGCTTCCCCGGACGCCG\n",
      "0 CCGCTTCCCCGGACGCCG motifs detected in file\n",
      "\n",
      "Seq is: CATAAGCTACAACTTCGG\n",
      "0 CATAAGCTACAACTTCGG motifs detected in file\n",
      "\n",
      "Seq is: ACTCTGATCAAGCCTGAG\n",
      "0 ACTCTGATCAAGCCTGAG motifs detected in file\n",
      "\n",
      "Seq is: GGGAAGCCGTTCCAGCTT\n",
      "0 GGGAAGCCGTTCCAGCTT motifs detected in file\n",
      "\n",
      "Seq is: CCCCCAAGCCAACCAGAG\n",
      "0 CCCCCAAGCCAACCAGAG motifs detected in file\n",
      "\n",
      "Seq is: ACGATCACACCTCTATTT\n",
      "0 ACGATCACACCTCTATTT motifs detected in file\n",
      "\n",
      "Seq is: AGTATAATACCGGTTCCC\n",
      "0 AGTATAATACCGGTTCCC motifs detected in file\n",
      "\n",
      "Seq is: AGTACTACCGTCCTCATT\n",
      "0 AGTACTACCGTCCTCATT motifs detected in file\n",
      "\n",
      "Seq is: AGTCTCCACAAGAACACA\n",
      "0 AGTCTCCACAAGAACACA motifs detected in file\n",
      "\n",
      "Seq is: ACACTATTCCCACCTCTC\n",
      "0 ACACTATTCCCACCTCTC motifs detected in file\n",
      "\n",
      "Seq is: CAATTTCCTCGGTGAATA\n",
      "0 CAATTTCCTCGGTGAATA motifs detected in file\n",
      "\n",
      "Seq is: ATCATTACAACTATCATC\n",
      "0 ATCATTACAACTATCATC motifs detected in file\n",
      "\n",
      "Seq is: TTACTACCGATCCCACTG\n",
      "0 TTACTACCGATCCCACTG motifs detected in file\n",
      "\n",
      "Seq is: AAGTATACTCTAACAAAC\n",
      "0 AAGTATACTCTAACAAAC motifs detected in file\n",
      "\n",
      "Seq is: CCGATGACGAGCGTCACG\n",
      "0 CCGATGACGAGCGTCACG motifs detected in file\n",
      "\n",
      "Seq is: CGGCTAGTCCCGCATCAA\n",
      "0 CGGCTAGTCCCGCATCAA motifs detected in file\n",
      "\n",
      "Seq is: ACTCAAACTACGAGTGGG\n",
      "0 ACTCAAACTACGAGTGGG motifs detected in file\n",
      "\n",
      "Seq is: ACTAGTCTCCTAACTCAT\n",
      "0 ACTAGTCTCCTAACTCAT motifs detected in file\n",
      "\n",
      "Seq is: TTGCCGATCCGATCTCCA\n",
      "0 TTGCCGATCCGATCTCCA motifs detected in file\n",
      "\n",
      "Seq is: CCGATCTTATTTATCCTC\n",
      "0 CCGATCTTATTTATCCTC motifs detected in file\n",
      "\n",
      "Seq is: CGGATCCAACTCCAACTG\n",
      "0 CGGATCCAACTCCAACTG motifs detected in file\n",
      "\n",
      "Seq is: GTCCCCCAACCCATACCC\n",
      "0 GTCCCCCAACCCATACCC motifs detected in file\n",
      "\n",
      "Seq is: CTCCCCCCAAGTATCATC\n",
      "0 CTCCCCCCAAGTATCATC motifs detected in file\n",
      "\n",
      "Seq is: TTCTCGCTACCACTCTCG\n",
      "0 TTCTCGCTACCACTCTCG motifs detected in file\n",
      "\n",
      "Seq is: ATTCCAGCCCCGCCACTA\n",
      "0 ATTCCAGCCCCGCCACTA motifs detected in file\n",
      "\n",
      "Seq is: CATCTCCCACTACCATCT\n",
      "0 CATCTCCCACTACCATCT motifs detected in file\n",
      "\n",
      "Seq is: ACACCGCCCAAAATCCCC\n",
      "0 ACACCGCCCAAAATCCCC motifs detected in file\n",
      "\n",
      "Seq is: GAGAAACCACTTCTCAAA\n",
      "0 GAGAAACCACTTCTCAAA motifs detected in file\n",
      "\n",
      "Seq is: ATACCGCAGTCGCTTCCC\n",
      "0 ATACCGCAGTCGCTTCCC motifs detected in file\n",
      "\n",
      "Seq is: AACATCATCGGGCATCCC\n",
      "0 AACATCATCGGGCATCCC motifs detected in file\n",
      "\n",
      "Seq is: CGGATGTTGCAACCCCGG\n",
      "0 CGGATGTTGCAACCCCGG motifs detected in file\n",
      "\n",
      "Seq is: AAACGCATCAACATATAT\n",
      "0 AAACGCATCAACATATAT motifs detected in file\n",
      "\n",
      "Seq is: CGGATCTTAAAAAGCAAG\n",
      "0 CGGATCTTAAAAAGCAAG motifs detected in file\n",
      "\n",
      "Seq is: CCATTCGTAATCCTTACG\n",
      "0 CCATTCGTAATCCTTACG motifs detected in file\n",
      "\n",
      "Seq is: GTAACGCTAATCTTACCC\n",
      "0 GTAACGCTAATCTTACCC motifs detected in file\n",
      "\n",
      "Seq is: ATGTTACTCCTCATCCTC\n",
      "0 ATGTTACTCCTCATCCTC motifs detected in file\n",
      "\n",
      "Seq is: CAACCGGTACCCATACAA\n",
      "0 CAACCGGTACCCATACAA motifs detected in file\n",
      "\n",
      "Seq is: CAATTCTTCTCCTTAACT\n",
      "0 CAATTCTTCTCCTTAACT motifs detected in file\n",
      "\n",
      "Seq is: TGGAGACTGACATTTCAA\n",
      "0 TGGAGACTGACATTTCAA motifs detected in file\n",
      "\n",
      "Seq is: AATTCAAAATACGCTAAT\n",
      "0 AATTCAAAATACGCTAAT motifs detected in file\n",
      "\n",
      "Seq is: GGCCCGAGACGGTAGAAT\n",
      "0 GGCCCGAGACGGTAGAAT motifs detected in file\n",
      "\n",
      "Seq is: TGTTTGGGACAAGAACCC\n",
      "0 TGTTTGGGACAAGAACCC motifs detected in file\n",
      "\n",
      "Seq is: ACCCACACCCATATTATG\n",
      "0 ACCCACACCCATATTATG motifs detected in file\n",
      "\n",
      "Seq is: ATTCAACTCTACTATAGT\n",
      "0 ATTCAACTCTACTATAGT motifs detected in file\n",
      "\n",
      "Seq is: AAATGCCCCCTTCCGCGA\n",
      "0 AAATGCCCCCTTCCGCGA motifs detected in file\n",
      "\n",
      "Seq is: AACACTTCATCCGGAATA\n",
      "0 AACACTTCATCCGGAATA motifs detected in file\n",
      "\n",
      "Seq is: AAGAGAACAGGAAAGCAT\n",
      "0 AAGAGAACAGGAAAGCAT motifs detected in file\n",
      "\n",
      "Seq is: GTCCCTCCTTAAACTTNC\n",
      "0 GTCCCTCCTTAAACTTNC motifs detected in file\n",
      "\n",
      "Seq is: ATCTATCTTTGGCTGGAC\n",
      "0 ATCTATCTTTGGCTGGAC motifs detected in file\n",
      "\n",
      "Seq is: CTAATGAGGCCAGACTTG\n",
      "0 CTAATGAGGCCAGACTTG motifs detected in file\n",
      "\n",
      "Seq is: AGTCTAGTGCATCCTGAA\n",
      "0 AGTCTAGTGCATCCTGAA motifs detected in file\n",
      "\n",
      "Seq is: ATTAGCAACTTGTTTGCT\n",
      "0 ATTAGCAACTTGTTTGCT motifs detected in file\n",
      "\n",
      "Seq is: TGGAAATTATCGCCGACG\n",
      "0 TGGAAATTATCGCCGACG motifs detected in file\n",
      "\n",
      "Seq is: TGGTAGCCCTACAGGACT\n",
      "0 TGGTAGCCCTACAGGACT motifs detected in file\n",
      "\n",
      "Seq is: AGGTTGTAGCTCCAGCAT\n",
      "0 AGGTTGTAGCTCCAGCAT motifs detected in file\n",
      "\n",
      "Seq is: TTGGGATAACAACTATAC\n",
      "0 TTGGGATAACAACTATAC motifs detected in file\n",
      "\n",
      "Seq is: CTGAGATCTTATCCTAAC\n",
      "0 CTGAGATCTTATCCTAAC motifs detected in file\n",
      "\n",
      "Seq is: GCGACAATAGGGATCCCA\n",
      "0 GCGACAATAGGGATCCCA motifs detected in file\n",
      "\n",
      "Seq is: TTGAACAAGGCAACCAGT\n",
      "0 TTGAACAAGGCAACCAGT motifs detected in file\n",
      "\n",
      "Seq is: TCAATAACCTAGTTAACT\n",
      "0 TCAATAACCTAGTTAACT motifs detected in file\n",
      "\n",
      "Seq is: CATATCATCAAGCGAAAC\n",
      "0 CATATCATCAAGCGAAAC motifs detected in file\n",
      "\n",
      "Seq is: TGACCACTTCAGAATCGT\n",
      "0 TGACCACTTCAGAATCGT motifs detected in file\n",
      "\n",
      "Seq is: ACATGACGAGCCTCCAAC\n",
      "0 ACATGACGAGCCTCCAAC motifs detected in file\n",
      "\n",
      "Seq is: CCAAGACGAGGCTCCAGC\n",
      "0 CCAAGACGAGGCTCCAGC motifs detected in file\n",
      "\n",
      "Seq is: GGGGTTGGCTTTAAAAAT\n",
      "0 GGGGTTGGCTTTAAAAAT motifs detected in file\n",
      "\n",
      "Seq is: TACGTCCAAACCATCAAA\n",
      "0 TACGTCCAAACCATCAAA motifs detected in file\n",
      "\n",
      "Seq is: TCCTGGACACCCAAACAA\n",
      "0 TCCTGGACACCCAAACAA motifs detected in file\n",
      "\n",
      "Seq is: TCCATGACAAACGTAATT\n",
      "0 TCCATGACAAACGTAATT motifs detected in file\n",
      "\n",
      "Seq is: ATTTAATTTCGAGGTATC\n",
      "0 ATTTAATTTCGAGGTATC motifs detected in file\n",
      "\n",
      "Seq is: CCAGAAGAGCAGAACGAC\n",
      "0 CCAGAAGAGCAGAACGAC motifs detected in file\n",
      "\n",
      "Seq is: ACAATACGGGCGGAGAAG\n",
      "0 ACAATACGGGCGGAGAAG motifs detected in file\n",
      "\n",
      "Seq is: TGCCCGTCCAGTTAAAGT\n",
      "0 TGCCCGTCCAGTTAAAGT motifs detected in file\n",
      "\n",
      "Seq is: GACCAATTTTCATTCTCT\n",
      "0 GACCAATTTTCATTCTCT motifs detected in file\n",
      "\n",
      "Seq is: GTCGACTTGGGAGCACCT\n",
      "0 GTCGACTTGGGAGCACCT motifs detected in file\n",
      "\n",
      "Seq is: CGGTAAGTATGTCCAGGG\n",
      "0 CGGTAAGTATGTCCAGGG motifs detected in file\n",
      "\n",
      "Seq is: ATAAATTCCTTGTTCACT\n",
      "0 ATAAATTCCTTGTTCACT motifs detected in file\n",
      "\n",
      "Seq is: AATACGATGGAAACGTGC\n",
      "0 AATACGATGGAAACGTGC motifs detected in file\n",
      "\n",
      "Seq is: CAATCCCATGGCGCCGGC\n",
      "0 CAATCCCATGGCGCCGGC motifs detected in file\n",
      "\n",
      "Seq is: AATTTGTACACAGTGACC\n",
      "0 AATTTGTACACAGTGACC motifs detected in file\n",
      "\n",
      "Seq is: CGTCCGCCACGGAGATTA\n",
      "0 CGTCCGCCACGGAGATTA motifs detected in file\n",
      "\n",
      "Seq is: TGACCACTACGATCTCCA\n",
      "0 TGACCACTACGATCTCCA motifs detected in file\n",
      "\n",
      "Seq is: CTACAAAAACCATTTGTC\n",
      "0 CTACAAAAACCATTTGTC motifs detected in file\n",
      "\n",
      "Seq is: CGCCCCATTCTAAACGGC\n",
      "0 CGCCCCATTCTAAACGGC motifs detected in file\n",
      "\n",
      "Seq is: TCAAGGAAAATGAAAAAA\n",
      "0 TCAAGGAAAATGAAAAAA motifs detected in file\n",
      "\n",
      "Seq is: ATTGGAAAGGAATACTCG\n",
      "0 ATTGGAAAGGAATACTCG motifs detected in file\n",
      "\n",
      "Seq is: TACGGACACAACCCAACT\n",
      "0 TACGGACACAACCCAACT motifs detected in file\n",
      "\n",
      "Seq is: GTCACTCCCATTATTACT\n",
      "0 GTCACTCCCATTATTACT motifs detected in file\n",
      "\n",
      "Seq is: GAACAACCAACTAACATC\n",
      "0 GAACAACCAACTAACATC motifs detected in file\n",
      "\n",
      "Seq is: TATAACCCGACAATTAAC\n",
      "0 TATAACCCGACAATTAAC motifs detected in file\n",
      "\n",
      "Seq is: AGTCAAGTCACAAAATTA\n",
      "0 AGTCAAGTCACAAAATTA motifs detected in file\n",
      "\n",
      "Seq is: GACTGCGTCCGAATACGC\n",
      "0 GACTGCGTCCGAATACGC motifs detected in file\n",
      "\n",
      "Seq is: CTCCTCTTACAAAAGTAC\n",
      "0 CTCCTCTTACAAAAGTAC motifs detected in file\n",
      "\n",
      "Seq is: AATGAATATGATTGTAAT\n",
      "0 AATGAATATGATTGTAAT motifs detected in file\n",
      "\n",
      "Seq is: CAAGAAGATATCCCACTA\n",
      "0 CAAGAAGATATCCCACTA motifs detected in file\n",
      "\n",
      "Seq is: TCTAACCAGGTTAACCCA\n",
      "0 TCTAACCAGGTTAACCCA motifs detected in file\n",
      "\n",
      "Seq is: CACTCCTCAAGTCAATAT\n",
      "0 CACTCCTCAAGTCAATAT motifs detected in file\n",
      "\n",
      "Seq is: ACAAACCCTAAAAAATCC\n",
      "0 ACAAACCCTAAAAAATCC motifs detected in file\n",
      "\n",
      "Seq is: ATCACCCACAACTCGAAC\n",
      "0 ATCACCCACAACTCGAAC motifs detected in file\n",
      "\n",
      "Seq is: TTGCGAAAGAATTAACCA\n",
      "0 TTGCGAAAGAATTAACCA motifs detected in file\n",
      "\n",
      "Seq is: CCGACGAAAATCCGGATG\n",
      "0 CCGACGAAAATCCGGATG motifs detected in file\n",
      "\n",
      "Seq is: ATACCCACAATTTAAAAA\n",
      "0 ATACCCACAATTTAAAAA motifs detected in file\n",
      "\n",
      "Seq is: ATGAGAGAGATGTTCCAA\n",
      "0 ATGAGAGAGATGTTCCAA motifs detected in file\n",
      "\n",
      "Seq is: AAAAGGATCACAGGTTTC\n",
      "0 AAAAGGATCACAGGTTTC motifs detected in file\n",
      "\n",
      "Seq is: TCGACAAGGAGAAACCTG\n",
      "0 TCGACAAGGAGAAACCTG motifs detected in file\n",
      "\n",
      "Seq is: ATTGTCAATTTAAATGTT\n",
      "0 ATTGTCAATTTAAATGTT motifs detected in file\n",
      "\n",
      "Seq is: CCCCTAAATCTCCCAAGA\n",
      "0 CCCCTAAATCTCCCAAGA motifs detected in file\n",
      "\n",
      "Seq is: CACCCGTTTAAATTTCAA\n",
      "0 CACCCGTTTAAATTTCAA motifs detected in file\n",
      "\n",
      "Seq is: CTTGATTCTAAGATAGAA\n",
      "0 CTTGATTCTAAGATAGAA motifs detected in file\n",
      "\n",
      "Seq is: CCTGTTGGTCGATAGTGG\n",
      "0 CCTGTTGGTCGATAGTGG motifs detected in file\n",
      "\n",
      "Seq is: TCCGAGCCATCCAAACAG\n",
      "0 TCCGAGCCATCCAAACAG motifs detected in file\n",
      "\n",
      "Seq is: CGGAGATGGATATTTAGA\n",
      "0 CGGAGATGGATATTTAGA motifs detected in file\n",
      "\n",
      "Seq is: AGGGTGATAAAACGATGT\n",
      "0 AGGGTGATAAAACGATGT motifs detected in file\n",
      "\n",
      "Seq is: ATCTGCCCACACGAGAAA\n",
      "0 ATCTGCCCACACGAGAAA motifs detected in file\n",
      "\n",
      "Seq is: ATCGACAAGAATCCATCG\n",
      "0 ATCGACAAGAATCCATCG motifs detected in file\n",
      "\n",
      "Seq is: AGCAGACCAAAGCCCCCA\n",
      "0 AGCAGACCAAAGCCCCCA motifs detected in file\n",
      "\n",
      "Seq is: GAATCGAAACCGAGAGGA\n",
      "0 GAATCGAAACCGAGAGGA motifs detected in file\n",
      "\n",
      "Seq is: ACGTTTCAATAAAGATCA\n",
      "0 ACGTTTCAATAAAGATCA motifs detected in file\n",
      "\n",
      "Seq is: ATTAAGTAATACGTCTTC\n",
      "0 ATTAAGTAATACGTCTTC motifs detected in file\n",
      "\n",
      "Seq is: CATATCCCCAATCAGGAA\n",
      "0 CATATCCCCAATCAGGAA motifs detected in file\n",
      "\n",
      "Seq is: CGATATAATACGAACCAA\n",
      "0 CGATATAATACGAACCAA motifs detected in file\n",
      "\n",
      "Seq is: TATTAAAAAGTAGAAAGG\n",
      "0 TATTAAAAAGTAGAAAGG motifs detected in file\n",
      "\n",
      "Seq is: GAACGCCATGATATAGAT\n",
      "0 GAACGCCATGATATAGAT motifs detected in file\n",
      "\n",
      "Seq is: AACGCGGTCCAAAGTTAA\n",
      "0 AACGCGGTCCAAAGTTAA motifs detected in file\n",
      "\n",
      "Seq is: AGATAGCGGATATGAAAT\n",
      "0 AGATAGCGGATATGAAAT motifs detected in file\n",
      "\n",
      "Seq is: AAACCCATTTACCAAACC\n",
      "0 AAACCCATTTACCAAACC motifs detected in file\n",
      "\n",
      "Seq is: GATTCCAACAGACCATCA\n",
      "0 GATTCCAACAGACCATCA motifs detected in file\n",
      "\n",
      "Seq is: TTCCACCTCACCCAAACC\n",
      "0 TTCCACCTCACCCAAACC motifs detected in file\n",
      "\n",
      "Seq is: CCGATCCAAATCGAGTCT\n",
      "0 CCGATCCAAATCGAGTCT motifs detected in file\n",
      "\n",
      "Seq is: CGCCAGTTCAATTCAACT\n",
      "0 CGCCAGTTCAATTCAACT motifs detected in file\n",
      "\n",
      "Seq is: TTAGAGGATTCACATTCA\n",
      "0 TTAGAGGATTCACATTCA motifs detected in file\n",
      "\n",
      "Seq is: ACCCACGAAACACAATTC\n",
      "0 ACCCACGAAACACAATTC motifs detected in file\n",
      "\n",
      "Seq is: GATGTGAGACCAAGCAGG\n",
      "0 GATGTGAGACCAAGCAGG motifs detected in file\n",
      "\n",
      "Seq is: TTCACGTGAAAGGTCATG\n",
      "0 TTCACGTGAAAGGTCATG motifs detected in file\n",
      "\n",
      "Seq is: TGAATGGTACAATGCTGA\n",
      "0 TGAATGGTACAATGCTGA motifs detected in file\n",
      "\n",
      "Seq is: ACAGAGGAGATATATTTA\n",
      "0 ACAGAGGAGATATATTTA motifs detected in file\n",
      "\n",
      "Seq is: CGCATCCCCAAAATCAAT\n",
      "0 CGCATCCCCAAAATCAAT motifs detected in file\n",
      "\n",
      "Seq is: TTACAGGAAACTTCATAT\n",
      "0 TTACAGGAAACTTCATAT motifs detected in file\n",
      "\n",
      "Seq is: GAACTCCTCCCACTGCCC\n",
      "0 GAACTCCTCCCACTGCCC motifs detected in file\n",
      "\n",
      "Seq is: GCCACACATGCGCGAAGT\n",
      "0 GCCACACATGCGCGAAGT motifs detected in file\n",
      "\n",
      "Seq is: CCCGGGACAAGTTGATTC\n",
      "0 CCCGGGACAAGTTGATTC motifs detected in file\n",
      "\n",
      "Seq is: GTGAGATGAGAATCAAAT\n",
      "0 GTGAGATGAGAATCAAAT motifs detected in file\n",
      "\n",
      "Seq is: GACGATTTAGGTGGAAGC\n",
      "0 GACGATTTAGGTGGAAGC motifs detected in file\n",
      "\n",
      "Seq is: TGGGAATTCAAAGTATTC\n",
      "0 TGGGAATTCAAAGTATTC motifs detected in file\n",
      "\n",
      "Seq is: CCGATAGCATCAAAAGAC\n",
      "0 CCGATAGCATCAAAAGAC motifs detected in file\n",
      "\n",
      "Seq is: CCCATCTTTTACATCGGG\n",
      "0 CCCATCTTTTACATCGGG motifs detected in file\n",
      "\n",
      "Seq is: TAAAGAACGGTGGAGTAC\n",
      "0 TAAAGAACGGTGGAGTAC motifs detected in file\n",
      "\n",
      "Seq is: CCGATGTGGAACTGGATT\n",
      "0 CCGATGTGGAACTGGATT motifs detected in file\n",
      "\n",
      "Seq is: GCAGAAATGCACCCATGA\n",
      "0 GCAGAAATGCACCCATGA motifs detected in file\n",
      "\n",
      "Seq is: ACGCGAATGAAACATCGG\n",
      "0 ACGCGAATGAAACATCGG motifs detected in file\n",
      "\n",
      "Seq is: AAGTAGTCCCAAACGACT\n",
      "0 AAGTAGTCCCAAACGACT motifs detected in file\n",
      "\n",
      "Seq is: TCTACCGCCATATATCCG\n",
      "0 TCTACCGCCATATATCCG motifs detected in file\n",
      "\n",
      "Seq is: ACTCGTTCTCCACCACTC\n",
      "0 ACTCGTTCTCCACCACTC motifs detected in file\n",
      "\n",
      "Seq is: CAACTAGCCCCAAATAGC\n",
      "0 CAACTAGCCCCAAATAGC motifs detected in file\n",
      "\n",
      "Seq is: TAATGTCTTGTCCGAGGA\n",
      "0 TAATGTCTTGTCCGAGGA motifs detected in file\n",
      "\n",
      "Seq is: GATCTCCCTATACTTCGT\n",
      "0 GATCTCCCTATACTTCGT motifs detected in file\n",
      "\n",
      "Seq is: GGCGGTCCAGGAAACTCA\n",
      "0 GGCGGTCCAGGAAACTCA motifs detected in file\n",
      "\n",
      "Seq is: AAATTCGACACCGAGCAT\n",
      "0 AAATTCGACACCGAGCAT motifs detected in file\n",
      "\n",
      "Seq is: CACAAGACCGCTCGTCAA\n",
      "0 CACAAGACCGCTCGTCAA motifs detected in file\n",
      "\n",
      "Seq is: AACAACTAAATTGACAAC\n",
      "0 AACAACTAAATTGACAAC motifs detected in file\n",
      "\n",
      "Seq is: TCCAAATCCCGATTCGTA\n",
      "0 TCCAAATCCCGATTCGTA motifs detected in file\n",
      "\n",
      "Seq is: TCACCCCATAGATTAGGG\n",
      "0 TCACCCCATAGATTAGGG motifs detected in file\n",
      "\n",
      "Seq is: TCAAACCCAGAATCGATA\n",
      "0 TCAAACCCAGAATCGATA motifs detected in file\n",
      "\n",
      "Seq is: ACACACAAGTCTATACAA\n",
      "0 ACACACAAGTCTATACAA motifs detected in file\n",
      "\n",
      "Seq is: TTTCGGTGAAAGCATCAG\n",
      "0 TTTCGGTGAAAGCATCAG motifs detected in file\n",
      "\n",
      "Seq is: ATAAAACACAGTTGACCT\n",
      "0 ATAAAACACAGTTGACCT motifs detected in file\n",
      "\n",
      "Seq is: CAAAAAATGTTGAGTCCA\n",
      "0 CAAAAAATGTTGAGTCCA motifs detected in file\n",
      "\n",
      "Seq is: CTCAAAATCGAAATAACC\n",
      "0 CTCAAAATCGAAATAACC motifs detected in file\n",
      "\n",
      "Seq is: CCTCCCCCACTAGATTTT\n",
      "0 CCTCCCCCACTAGATTTT motifs detected in file\n",
      "\n",
      "Seq is: GTGAGAAATGCGGCCGAA\n",
      "0 GTGAGAAATGCGGCCGAA motifs detected in file\n",
      "\n",
      "Seq is: GATAACTGAACCCAATTA\n",
      "0 GATAACTGAACCCAATTA motifs detected in file\n",
      "\n",
      "Seq is: GCACACTGGCGCCACCGA\n",
      "0 GCACACTGGCGCCACCGA motifs detected in file\n",
      "\n",
      "Seq is: CCGTGCTTTAACTGGTTG\n",
      "0 CCGTGCTTTAACTGGTTG motifs detected in file\n",
      "\n",
      "Seq is: GGACCCCAATCATATCGA\n",
      "0 GGACCCCAATCATATCGA motifs detected in file\n",
      "\n",
      "Seq is: ATCAATTTGAAAGCAAAT\n",
      "0 ATCAATTTGAAAGCAAAT motifs detected in file\n",
      "\n",
      "Seq is: AACGATTTCCAATTAGTG\n",
      "0 AACGATTTCCAATTAGTG motifs detected in file\n",
      "\n",
      "Seq is: ACGACAAAGGGCACCCCC\n",
      "0 ACGACAAAGGGCACCCCC motifs detected in file\n",
      "\n",
      "Seq is: ACACCGATCCGATTCGCA\n",
      "0 ACACCGATCCGATTCGCA motifs detected in file\n",
      "\n",
      "Seq is: AAACTCGACGTAACGACG\n",
      "0 AAACTCGACGTAACGACG motifs detected in file\n",
      "\n",
      "Seq is: CACGAACTACGAACAAGG\n",
      "0 CACGAACTACGAACAAGG motifs detected in file\n",
      "\n",
      "Seq is: AAAACTAGCACCACTAAA\n",
      "0 AAAACTAGCACCACTAAA motifs detected in file\n",
      "\n",
      "Seq is: TCTCCCACTTGAGTGACC\n",
      "0 TCTCCCACTTGAGTGACC motifs detected in file\n",
      "\n",
      "Seq is: TTGCCCCTACGAACGTAC\n",
      "0 TTGCCCCTACGAACGTAC motifs detected in file\n",
      "\n",
      "Seq is: ACATTAGAATGATTCTCG\n",
      "0 ACATTAGAATGATTCTCG motifs detected in file\n",
      "\n",
      "Seq is: ATTATCTTTCCGATCCTG\n",
      "0 ATTATCTTTCCGATCCTG motifs detected in file\n",
      "\n",
      "Seq is: GTTTGGATAAACAAATAC\n",
      "0 GTTTGGATAAACAAATAC motifs detected in file\n",
      "\n",
      "Seq is: CCCACTACACTCGGGCAG\n",
      "0 CCCACTACACTCGGGCAG motifs detected in file\n",
      "\n",
      "Seq is: ATTTGTAAAAGTCACATA\n",
      "0 ATTTGTAAAAGTCACATA motifs detected in file\n",
      "\n",
      "Seq is: ACGAAACTCCTCCATTCG\n",
      "0 ACGAAACTCCTCCATTCG motifs detected in file\n",
      "\n",
      "Seq is: ATGTATTTGACACCCCCC\n",
      "0 ATGTATTTGACACCCCCC motifs detected in file\n",
      "\n",
      "Seq is: ACAGAAACCCCAAACCAA\n",
      "0 ACAGAAACCCCAAACCAA motifs detected in file\n",
      "\n",
      "Seq is: CCAAGCCCCATACCCCAA\n",
      "0 CCAAGCCCCATACCCCAA motifs detected in file\n",
      "\n",
      "Seq is: TCGTCGCCACACACACAC\n",
      "0 TCGTCGCCACACACACAC motifs detected in file\n",
      "\n",
      "Seq is: GACCCATCCTACCCGCCC\n",
      "0 GACCCATCCTACCCGCCC motifs detected in file\n",
      "\n",
      "Seq is: CCAACATAACTACTCTAA\n",
      "0 CCAACATAACTACTCTAA motifs detected in file\n",
      "\n",
      "Seq is: TCATCATACCCTCACCCT\n",
      "0 TCATCATACCCTCACCCT motifs detected in file\n",
      "\n",
      "Seq is: CCCCTTTTATTACACAAT\n",
      "0 CCCCTTTTATTACACAAT motifs detected in file\n",
      "\n",
      "Seq is: CAACCCCCCACTGACAAT\n",
      "0 CAACCCCCCACTGACAAT motifs detected in file\n",
      "\n",
      "Seq is: TTTCACGTATGGCGGTTT\n",
      "0 TTTCACGTATGGCGGTTT motifs detected in file\n",
      "\n",
      "Seq is: TCTATTTTAAACTTTAGA\n",
      "0 TCTATTTTAAACTTTAGA motifs detected in file\n",
      "\n",
      "Seq is: CCAATCCGACCACAATCC\n",
      "0 CCAATCCGACCACAATCC motifs detected in file\n",
      "\n",
      "Seq is: CAAGAAACAAAAACCCCA\n",
      "0 CAAGAAACAAAAACCCCA motifs detected in file\n",
      "\n",
      "Seq is: AACCGTCTCTACACAAAT\n",
      "0 AACCGTCTCTACACAAAT motifs detected in file\n",
      "\n",
      "Seq is: TCACGACACCGGTCTTCG\n",
      "0 TCACGACACCGGTCTTCG motifs detected in file\n",
      "\n",
      "Seq is: CCCCCTCCCCCCCAAACC\n",
      "0 CCCCCTCCCCCCCAAACC motifs detected in file\n",
      "\n",
      "Seq is: ACCTTTAAAAAACAATAC\n",
      "0 ACCTTTAAAAAACAATAC motifs detected in file\n",
      "\n",
      "Seq is: TACAGACACACCTTTCAC\n",
      "0 TACAGACACACCTTTCAC motifs detected in file\n",
      "\n",
      "Seq is: CGACACGTCTGTAAGTTA\n",
      "0 CGACACGTCTGTAAGTTA motifs detected in file\n",
      "\n",
      "Seq is: ACAATAATAATACAGGAT\n",
      "0 ACAATAATAATACAGGAT motifs detected in file\n",
      "\n",
      "Seq is: GTTCGTAATTAATTAATT\n",
      "0 GTTCGTAATTAATTAATT motifs detected in file\n",
      "\n",
      "Seq is: GTGTGAAATCATTCATAC\n",
      "0 GTGTGAAATCATTCATAC motifs detected in file\n",
      "\n",
      "Seq is: AAGCGGACATTATAACTT\n",
      "0 AAGCGGACATTATAACTT motifs detected in file\n",
      "\n",
      "Seq is: GCATCCACGCTATTTATT\n",
      "0 GCATCCACGCTATTTATT motifs detected in file\n",
      "\n",
      "Seq is: ATCCTACTCCGTCCTTAG\n",
      "0 ATCCTACTCCGTCCTTAG motifs detected in file\n",
      "\n",
      "Seq is: TTTCTGTCTATGACGCTG\n",
      "0 TTTCTGTCTATGACGCTG motifs detected in file\n",
      "\n",
      "Seq is: TATCCCACGAGGCCGAGG\n",
      "0 TATCCCACGAGGCCGAGG motifs detected in file\n",
      "\n",
      "Seq is: TCGCAGAGCGTTACGATA\n",
      "0 TCGCAGAGCGTTACGATA motifs detected in file\n",
      "\n",
      "Seq is: GCGCACGTATGGGGGGTC\n",
      "0 GCGCACGTATGGGGGGTC motifs detected in file\n",
      "\n",
      "Seq is: TGCTTTTATGGTTTACGT\n",
      "0 TGCTTTTATGGTTTACGT motifs detected in file\n",
      "\n",
      "Seq is: ACCTCTCGAGGGCACTCA\n",
      "0 ACCTCTCGAGGGCACTCA motifs detected in file\n",
      "\n",
      "Seq is: CCAATTATCCCACTATCT\n",
      "0 CCAATTATCCCACTATCT motifs detected in file\n",
      "\n",
      "Seq is: GTAGCACTACAGAATAAAT\n",
      "0 GTAGCACTACAGAATAAAT motifs detected in file\n",
      "\n",
      "Seq is: TCCCCTTGCACACCCGATA\n",
      "0 TCCCCTTGCACACCCGATA motifs detected in file\n",
      "\n",
      "Seq is: AATCCGAAATACTGGGACT\n",
      "0 AATCCGAAATACTGGGACT motifs detected in file\n",
      "\n",
      "Seq is: TCATCCTTGGTCTACAGCC\n",
      "0 TCATCCTTGGTCTACAGCC motifs detected in file\n",
      "\n",
      "Seq is: TATGTCAAGTGAAATCGAT\n",
      "0 TATGTCAAGTGAAATCGAT motifs detected in file\n",
      "\n",
      "Seq is: GGGGGTTCACAATACCCGG\n",
      "0 GGGGGTTCACAATACCCGG motifs detected in file\n",
      "\n",
      "Seq is: GCCTCGCTCCTCTCATCGT\n",
      "0 GCCTCGCTCCTCTCATCGT motifs detected in file\n",
      "\n",
      "Seq is: GAGAACACGCCCTATAACT\n",
      "0 GAGAACACGCCCTATAACT motifs detected in file\n",
      "\n",
      "Seq is: AAAGTGCCTCCTACCACCA\n",
      "0 AAAGTGCCTCCTACCACCA motifs detected in file\n",
      "\n",
      "Seq is: GTTCCCTGGGGATAGACTC\n",
      "0 GTTCCCTGGGGATAGACTC motifs detected in file\n",
      "\n",
      "Seq is: CCCCCAGTAGGTACCCCTG\n",
      "0 CCCCCAGTAGGTACCCCTG motifs detected in file\n",
      "\n",
      "Seq is: CTCTTCCCTAAACTGACAT\n",
      "0 CTCTTCCCTAAACTGACAT motifs detected in file\n",
      "\n",
      "Seq is: TACACGATACATGCCATTT\n",
      "0 TACACGATACATGCCATTT motifs detected in file\n",
      "\n",
      "Seq is: ACCGAAATACATGATACAT\n",
      "0 ACCGAAATACATGATACAT motifs detected in file\n",
      "\n",
      "Seq is: GACAATTCCCACCCATCCA\n",
      "0 GACAATTCCCACCCATCCA motifs detected in file\n",
      "\n",
      "Seq is: AACAACCATAGGATCACCC\n",
      "0 AACAACCATAGGATCACCC motifs detected in file\n",
      "\n",
      "Seq is: ACTCCCCACCGAAACCTCA\n",
      "0 ACTCCCCACCGAAACCTCA motifs detected in file\n",
      "\n",
      "Seq is: ACGTCAACTACACACTATC\n",
      "0 ACGTCAACTACACACTATC motifs detected in file\n",
      "\n",
      "Seq is: AACTCCCAACTAACGACAT\n",
      "0 AACTCCCAACTAACGACAT motifs detected in file\n",
      "\n",
      "Seq is: GAACGAACATTCGTACCCC\n",
      "0 GAACGAACATTCGTACCCC motifs detected in file\n",
      "\n",
      "Seq is: TCCCCCAAAACTACACCTA\n",
      "0 TCCCCCAAAACTACACCTA motifs detected in file\n",
      "\n",
      "Seq is: ACCCAAAAATACATGATGT\n",
      "0 ACCCAAAAATACATGATGT motifs detected in file\n",
      "\n",
      "Seq is: CCACCAGTTCATAAATACC\n",
      "0 CCACCAGTTCATAAATACC motifs detected in file\n",
      "\n",
      "Seq is: ATGGCATGTTATAAGTACC\n",
      "0 ATGGCATGTTATAAGTACC motifs detected in file\n",
      "\n",
      "Seq is: ACCGACCGTCATTACATGC\n",
      "0 ACCGACCGTCATTACATGC motifs detected in file\n",
      "\n",
      "Seq is: TTTATGTATCGCCAACAAC\n",
      "0 TTTATGTATCGCCAACAAC motifs detected in file\n",
      "\n",
      "Seq is: TACCCACTCAGTTATGAAC\n",
      "0 TACCCACTCAGTTATGAAC motifs detected in file\n",
      "\n",
      "Seq is: CCACCATGGGTTTAGACGA\n",
      "0 CCACCATGGGTTTAGACGA motifs detected in file\n",
      "\n",
      "Seq is: AGGGGTACTTTCTTGTCTC\n",
      "0 AGGGGTACTTTCTTGTCTC motifs detected in file\n",
      "\n",
      "Seq is: TTATCAAATTTAATCTTAG\n",
      "0 TTATCAAATTTAATCTTAG motifs detected in file\n",
      "\n",
      "Seq is: AATCGAAACCCACGATTAC\n",
      "0 AATCGAAACCCACGATTAC motifs detected in file\n",
      "\n",
      "Seq is: CACCTCAATTTCTGAAAAA\n",
      "0 CACCTCAATTTCTGAAAAA motifs detected in file\n",
      "\n",
      "Seq is: GAGACTAAACAGGAACCTT\n",
      "0 GAGACTAAACAGGAACCTT motifs detected in file\n",
      "\n",
      "Seq is: TTTCCAAAAGTAGAGGCCA\n",
      "0 TTTCCAAAAGTAGAGGCCA motifs detected in file\n",
      "\n",
      "Seq is: AATGTTCTGACCACATAAT\n",
      "0 AATGTTCTGACCACATAAT motifs detected in file\n",
      "\n",
      "Seq is: CAAATATGATGTTCCTGTC\n",
      "0 CAAATATGATGTTCCTGTC motifs detected in file\n",
      "\n",
      "Seq is: CGGGTAAACTCATAAAACA\n",
      "0 CGGGTAAACTCATAAAACA motifs detected in file\n",
      "\n",
      "Seq is: AAAGTTAATCCCTCTATCA\n",
      "0 AAAGTTAATCCCTCTATCA motifs detected in file\n",
      "\n",
      "Seq is: ACCATAATCCTAATCCTAA\n",
      "0 ACCATAATCCTAATCCTAA motifs detected in file\n",
      "\n",
      "Seq is: CAACACTTCATATCATGCC\n",
      "0 CAACACTTCATATCATGCC motifs detected in file\n",
      "\n",
      "Seq is: TACGATGAACAGGTTACTA\n",
      "0 TACGATGAACAGGTTACTA motifs detected in file\n",
      "\n",
      "Seq is: CCATTTTCCCATCGAATGA\n",
      "0 CCATTTTCCCATCGAATGA motifs detected in file\n",
      "\n",
      "Seq is: CCAACAGGAGGCTAAGTCC\n",
      "0 CCAACAGGAGGCTAAGTCC motifs detected in file\n",
      "\n",
      "Seq is: AATCTTACTCCTCCAGACG\n",
      "0 AATCTTACTCCTCCAGACG motifs detected in file\n",
      "\n",
      "Seq is: CCGATCCTCAGTTATTTCA\n",
      "0 CCGATCCTCAGTTATTTCA motifs detected in file\n",
      "\n",
      "Seq is: CTAACCGAATCACCCGCTT\n",
      "0 CTAACCGAATCACCCGCTT motifs detected in file\n",
      "\n",
      "Seq is: TATAATACGAAACAACAAA\n",
      "0 TATAATACGAAACAACAAA motifs detected in file\n",
      "\n",
      "Seq is: CCTATATACCTCCTACCCC\n",
      "0 CCTATATACCTCCTACCCC motifs detected in file\n",
      "\n",
      "Seq is: TAATAACGATCCTACTCCT\n",
      "0 TAATAACGATCCTACTCCT motifs detected in file\n",
      "\n",
      "Seq is: ACCTATCATTATCCCGTTC\n",
      "0 ACCTATCATTATCCCGTTC motifs detected in file\n",
      "\n",
      "Seq is: CTGCGGAGGATCAAACAAT\n",
      "0 CTGCGGAGGATCAAACAAT motifs detected in file\n",
      "\n",
      "Seq is: CCCTGCCTAGCCTCTTAAC\n",
      "0 CCCTGCCTAGCCTCTTAAC motifs detected in file\n",
      "\n",
      "Seq is: ACATCCGCTTATCCTTTAT\n",
      "0 ACATCCGCTTATCCTTTAT motifs detected in file\n",
      "\n",
      "Seq is: AGTAAGCCCGAACTACACC\n",
      "0 AGTAAGCCCGAACTACACC motifs detected in file\n",
      "\n",
      "Seq is: CCTCCCCACAAATTCCCCA\n",
      "0 CCTCCCCACAAATTCCCCA motifs detected in file\n",
      "\n",
      "Seq is: ACCGATCCCATATTAACAG\n",
      "0 ACCGATCCCATATTAACAG motifs detected in file\n",
      "\n",
      "Seq is: ACCCAGCGGATCCTCCAGA\n",
      "0 ACCCAGCGGATCCTCCAGA motifs detected in file\n",
      "\n",
      "Seq is: CCACTCTTATCACAATTAC\n",
      "0 CCACTCTTATCACAATTAC motifs detected in file\n",
      "\n",
      "Seq is: AGTAATTCCTCTCTTCCTT\n",
      "0 AGTAATTCCTCTCTTCCTT motifs detected in file\n",
      "\n",
      "Seq is: CTCTTCATTCGGCTCCCGC\n",
      "0 CTCTTCATTCGGCTCCCGC motifs detected in file\n",
      "\n",
      "Seq is: AGAAACTAACACATCATTC\n",
      "0 AGAAACTAACACATCATTC motifs detected in file\n",
      "\n",
      "Seq is: CCACCTTCCACTAAAATAG\n",
      "0 CCACCTTCCACTAAAATAG motifs detected in file\n",
      "\n",
      "Seq is: CCTTACCCTCCACTAAGGA\n",
      "0 CCTTACCCTCCACTAAGGA motifs detected in file\n",
      "\n",
      "Seq is: TCCCCCAACAAACTAGGGC\n",
      "0 TCCCCCAACAAACTAGGGC motifs detected in file\n",
      "\n",
      "Seq is: AAAGCACGTTCTTATCCTC\n",
      "0 AAAGCACGTTCTTATCCTC motifs detected in file\n",
      "\n",
      "Seq is: CACCTCACAACGATCCCGA\n",
      "0 CACCTCACAACGATCCCGA motifs detected in file\n",
      "\n",
      "Seq is: CGTTATTACTTCCCGTTCT\n",
      "0 CGTTATTACTTCCCGTTCT motifs detected in file\n",
      "\n",
      "Seq is: ACTTCACTTTCCATTTCTT\n",
      "0 ACTTCACTTTCCATTTCTT motifs detected in file\n",
      "\n",
      "Seq is: AGCACACTCCCACCCTGAC\n",
      "0 AGCACACTCCCACCCTGAC motifs detected in file\n",
      "\n",
      "Seq is: AGATGACTCATCGGAGGAG\n",
      "0 AGATGACTCATCGGAGGAG motifs detected in file\n",
      "\n",
      "Seq is: TCTAAGTAACTTGATCCAG\n",
      "0 TCTAAGTAACTTGATCCAG motifs detected in file\n",
      "\n",
      "Seq is: ACAGGGTTACATACCCTAC\n",
      "0 ACAGGGTTACATACCCTAC motifs detected in file\n",
      "\n",
      "Seq is: CGCCTATCATTCAAACATT\n",
      "0 CGCCTATCATTCAAACATT motifs detected in file\n",
      "\n",
      "Seq is: AATGACACCGGGGAGTCTT\n",
      "0 AATGACACCGGGGAGTCTT motifs detected in file\n",
      "\n",
      "Seq is: ACTATAAACCGGAGTGCCC\n",
      "0 ACTATAAACCGGAGTGCCC motifs detected in file\n",
      "\n",
      "Seq is: TCCTGTATCGGATACTTCC\n",
      "0 TCCTGTATCGGATACTTCC motifs detected in file\n",
      "\n",
      "Seq is: GACAACGATATCAACGTTC\n",
      "0 GACAACGATATCAACGTTC motifs detected in file\n",
      "\n",
      "Seq is: GTCCTCCTATTACGGCTAC\n",
      "0 GTCCTCCTATTACGGCTAC motifs detected in file\n",
      "\n",
      "Seq is: AAAGTCCAAAGACTCATCT\n",
      "0 AAAGTCCAAAGACTCATCT motifs detected in file\n",
      "\n",
      "Seq is: CTTTACTAGGCATTATATC\n",
      "0 CTTTACTAGGCATTATATC motifs detected in file\n",
      "\n",
      "Seq is: CGGAGCGGGCTACACATCC\n",
      "0 CGGAGCGGGCTACACATCC motifs detected in file\n",
      "\n",
      "Seq is: TTCTCCGTCTATTTCTTAT\n",
      "0 TTCTCCGTCTATTTCTTAT motifs detected in file\n",
      "\n",
      "Seq is: AACTCCGCGGTAACCGCAC\n",
      "0 AACTCCGCGGTAACCGCAC motifs detected in file\n",
      "\n",
      "Seq is: TTCCATCGCCTACTAAGTC\n",
      "0 TTCCATCGCCTACTAAGTC motifs detected in file\n",
      "\n",
      "Seq is: GGTATTAAATGCAGAGCTC\n",
      "0 GGTATTAAATGCAGAGCTC motifs detected in file\n",
      "\n",
      "Seq is: ACTACACCCGCTAACTACT\n",
      "0 ACTACACCCGCTAACTACT motifs detected in file\n",
      "\n",
      "Seq is: TTTCCGCCAACTCCGCAGA\n",
      "0 TTTCCGCCAACTCCGCAGA motifs detected in file\n",
      "\n",
      "Seq is: CCACTCATCACGTACCGAT\n",
      "0 CCACTCATCACGTACCGAT motifs detected in file\n",
      "\n",
      "Seq is: CCTTATCAGGACACCACTA\n",
      "0 CCTTATCAGGACACCACTA motifs detected in file\n",
      "\n",
      "Seq is: AACCTCCTAGTCCGTCCGC\n",
      "0 AACCTCCTAGTCCGTCCGC motifs detected in file\n",
      "\n",
      "Seq is: GGTTCCTCACTCGGCTTCA\n",
      "0 GGTTCCTCACTCGGCTTCA motifs detected in file\n",
      "\n",
      "Seq is: AAGTAGTACGCCTCTACAA\n",
      "0 AAGTAGTACGCCTCTACAA motifs detected in file\n",
      "\n",
      "Seq is: CCTACCCCACCCCTCCAGC\n",
      "0 CCTACCCCACCCCTCCAGC motifs detected in file\n",
      "\n",
      "Seq is: TACTTACTCACCAATTAAT\n",
      "0 TACTTACTCACCAATTAAT motifs detected in file\n",
      "\n",
      "Seq is: TAAAATAATCCCCCAATCA\n",
      "0 TAAAATAATCCCCCAATCA motifs detected in file\n",
      "\n",
      "Seq is: AAACGCATAACCCCAGTAA\n",
      "0 AAACGCATAACCCCAGTAA motifs detected in file\n",
      "\n",
      "Seq is: CCACAAGAACATCAACTTT\n",
      "0 CCACAAGAACATCAACTTT motifs detected in file\n",
      "\n",
      "Seq is: ATGTTGCTACCAAAAAGTA\n",
      "0 ATGTTGCTACCAAAAAGTA motifs detected in file\n",
      "\n",
      "Seq is: TAGTAACCAGCACCAACAT\n",
      "0 TAGTAACCAGCACCAACAT motifs detected in file\n",
      "\n",
      "Seq is: CAGGCACGCTCTTATTACT\n",
      "0 CAGGCACGCTCTTATTACT motifs detected in file\n",
      "\n",
      "Seq is: ACATACGAAACAAAGACAA\n",
      "0 ACATACGAAACAAAGACAA motifs detected in file\n",
      "\n",
      "Seq is: CTCACACCCAAATCATTAC\n",
      "0 CTCACACCCAAATCATTAC motifs detected in file\n",
      "\n",
      "Seq is: CCCAAACACCCCAAAAGAA\n",
      "0 CCCAAACACCCCAAAAGAA motifs detected in file\n",
      "\n",
      "Seq is: GATTCGGAAGAGGATAAAT\n",
      "0 GATTCGGAAGAGGATAAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCCCCAAATCATAACTAA\n",
      "0 ACCCCCAAATCATAACTAA motifs detected in file\n",
      "\n",
      "Seq is: CAATCGCCACACCAGCCCA\n",
      "0 CAATCGCCACACCAGCCCA motifs detected in file\n",
      "\n",
      "Seq is: CACAATAATAAGACTTAAA\n",
      "0 CACAATAATAAGACTTAAA motifs detected in file\n",
      "\n",
      "Seq is: ACCCCCTCCAATATACCCA\n",
      "0 ACCCCCTCCAATATACCCA motifs detected in file\n",
      "\n",
      "Seq is: AATTATCAAAAAAATTAAA\n",
      "0 AATTATCAAAAAAATTAAA motifs detected in file\n",
      "\n",
      "Seq is: TAAATCCCCCTTACTACCA\n",
      "0 TAAATCCCCCTTACTACCA motifs detected in file\n",
      "\n",
      "Seq is: ACAGAAACCTATATGATGT\n",
      "0 ACAGAAACCTATATGATGT motifs detected in file\n",
      "\n",
      "Seq is: CGCTACCGATAACTCCTCA\n",
      "0 CGCTACCGATAACTCCTCA motifs detected in file\n",
      "\n",
      "Seq is: TAGGACTCCGTACCCCCAG\n",
      "0 TAGGACTCCGTACCCCCAG motifs detected in file\n",
      "\n",
      "Seq is: TCCCCAACTCCAGAACCAC\n",
      "0 TCCCCAACTCCAGAACCAC motifs detected in file\n",
      "\n",
      "Seq is: TCACAAAATCACCCCAATC\n",
      "0 TCACAAAATCACCCCAATC motifs detected in file\n",
      "\n",
      "Seq is: GCTACCTCCATCCTAACCA\n",
      "0 GCTACCTCCATCCTAACCA motifs detected in file\n",
      "\n",
      "Seq is: CGACACCCACTTTCTCATA\n",
      "0 CGACACCCACTTTCTCATA motifs detected in file\n",
      "\n",
      "Seq is: CTACCCCACCACCAACACC\n",
      "0 CTACCCCACCACCAACACC motifs detected in file\n",
      "\n",
      "Seq is: ATTTGAAATTATCACATCC\n",
      "0 ATTTGAAATTATCACATCC motifs detected in file\n",
      "\n",
      "Seq is: TTCGACTTATTAAATACTT\n",
      "0 TTCGACTTATTAAATACTT motifs detected in file\n",
      "\n",
      "Seq is: CCTCTCCCCAGTCCCAACT\n",
      "0 CCTCTCCCCAGTCCCAACT motifs detected in file\n",
      "\n",
      "Seq is: AAGCCCTCCTAGGATAACC\n",
      "0 AAGCCCTCCTAGGATAACC motifs detected in file\n",
      "\n",
      "Seq is: ACGCCCCCGAAACATACTA\n",
      "0 ACGCCCCCGAAACATACTA motifs detected in file\n",
      "\n",
      "Seq is: ATACCCGCAACTAATCATC\n",
      "0 ATACCCGCAACTAATCATC motifs detected in file\n",
      "\n",
      "Seq is: ATCAATGACCAACTTGTAA\n",
      "0 ATCAATGACCAACTTGTAA motifs detected in file\n",
      "\n",
      "Seq is: CAAACAACCACATATATAA\n",
      "0 CAAACAACCACATATATAA motifs detected in file\n",
      "\n",
      "Seq is: CATTAACTCTAACGAGCCC\n",
      "0 CATTAACTCTAACGAGCCC motifs detected in file\n",
      "\n",
      "Seq is: CCTTATCCAATACACTAAT\n",
      "0 CCTTATCCAATACACTAAT motifs detected in file\n",
      "\n",
      "Seq is: CCTCATCCCAATCCTACTC\n",
      "0 CCTCATCCCAATCCTACTC motifs detected in file\n",
      "\n",
      "Seq is: ACCCTTCTTCTTTCTCTCC\n",
      "0 ACCCTTCTTCTTTCTCTCC motifs detected in file\n",
      "\n",
      "Seq is: TTCATTTCAAATTAATACG\n",
      "0 TTCATTTCAAATTAATACG motifs detected in file\n",
      "\n",
      "Seq is: GAAAAACCCAACTCCACTA\n",
      "0 GAAAAACCCAACTCCACTA motifs detected in file\n",
      "\n",
      "Seq is: CTACCTCCACCTCTAAACC\n",
      "0 CTACCTCCACCTCTAAACC motifs detected in file\n",
      "\n",
      "Seq is: ACGACACTTTAACAAAATC\n",
      "0 ACGACACTTTAACAAAATC motifs detected in file\n",
      "\n",
      "Seq is: CATTATCGAAAAGATCAGT\n",
      "0 CATTATCGAAAAGATCAGT motifs detected in file\n",
      "\n",
      "Seq is: CCAATCCAGATCCTCCTCA\n",
      "0 CCAATCCAGATCCTCCTCA motifs detected in file\n",
      "\n",
      "Seq is: TCCCCGTCCAAAACCGAGC\n",
      "0 TCCCCGTCCAAAACCGAGC motifs detected in file\n",
      "\n",
      "Seq is: ATTCTTCCGGATCTATCCC\n",
      "0 ATTCTTCCGGATCTATCCC motifs detected in file\n",
      "\n",
      "Seq is: CTAACACGCCACACACTAC\n",
      "0 CTAACACGCCACACACTAC motifs detected in file\n",
      "\n",
      "Seq is: GATCCCATCTTAGGCTCAT\n",
      "0 GATCCCATCTTAGGCTCAT motifs detected in file\n",
      "\n",
      "Seq is: ACAACCTCTTTATTTTACA\n",
      "0 ACAACCTCTTTATTTTACA motifs detected in file\n",
      "\n",
      "Seq is: CGTATCACCCCTAAAATAA\n",
      "0 CGTATCACCCCTAAAATAA motifs detected in file\n",
      "\n",
      "Seq is: AATTCAAACAACCAATCCA\n",
      "0 AATTCAAACAACCAATCCA motifs detected in file\n",
      "\n",
      "Seq is: TCAACTCCAGATCCCGACA\n",
      "0 TCAACTCCAGATCCCGACA motifs detected in file\n",
      "\n",
      "Seq is: ATCTTCAGGATCCTTTCAC\n",
      "0 ATCTTCAGGATCCTTTCAC motifs detected in file\n",
      "\n",
      "Seq is: TGTCGCTCCCGACACTCAA\n",
      "0 TGTCGCTCCCGACACTCAA motifs detected in file\n",
      "\n",
      "Seq is: AATCCATCTCCCCCTAACA\n",
      "0 AATCCATCTCCCCCTAACA motifs detected in file\n",
      "\n",
      "Seq is: ACAAACCTTCCCCCTACGC\n",
      "0 ACAAACCTTCCCCCTACGC motifs detected in file\n",
      "\n",
      "Seq is: CCCCTTTACAACAATCATT\n",
      "0 CCCCTTTACAACAATCATT motifs detected in file\n",
      "\n",
      "Seq is: ACTCTTTAGGACGCTTATC\n",
      "0 ACTCTTTAGGACGCTTATC motifs detected in file\n",
      "\n",
      "Seq is: CGAAGGCCGACGGTCCGCA\n",
      "0 CGAAGGCCGACGGTCCGCA motifs detected in file\n",
      "\n",
      "Seq is: AATTACCCCAAATCATCCC\n",
      "0 AATTACCCCAAATCATCCC motifs detected in file\n",
      "\n",
      "Seq is: ACCCCAATAAAAGCAATTA\n",
      "0 ACCCCAATAAAAGCAATTA motifs detected in file\n",
      "\n",
      "Seq is: CAATCATTCCCACCCCTTC\n",
      "0 CAATCATTCCCACCCCTTC motifs detected in file\n",
      "\n",
      "Seq is: GCTCCAACTGGACAATCCC\n",
      "0 GCTCCAACTGGACAATCCC motifs detected in file\n",
      "\n",
      "Seq is: ACTCTTCTTAATAAGCTCA\n",
      "0 ACTCTTCTTAATAAGCTCA motifs detected in file\n",
      "\n",
      "Seq is: CGATATCCGCGAACAGTCC\n",
      "0 CGATATCCGCGAACAGTCC motifs detected in file\n",
      "\n",
      "Seq is: CTCCATCGCTACTCTCATT\n",
      "0 CTCCATCGCTACTCTCATT motifs detected in file\n",
      "\n",
      "Seq is: ATCTATCCCGAGTCCGCAA\n",
      "0 ATCTATCCCGAGTCCGCAA motifs detected in file\n",
      "\n",
      "Seq is: ACACATACTATACAAACGC\n",
      "0 ACACATACTATACAAACGC motifs detected in file\n",
      "\n",
      "Seq is: CAAAGCTACTACACCAGAA\n",
      "0 CAAAGCTACTACACCAGAA motifs detected in file\n",
      "\n",
      "Seq is: ACCTCATCTTTGGACACTC\n",
      "0 ACCTCATCTTTGGACACTC motifs detected in file\n",
      "\n",
      "Seq is: CTTTCCATAAGGACGATTA\n",
      "0 CTTTCCATAAGGACGATTA motifs detected in file\n",
      "\n",
      "Seq is: CGATCCGACGGTTACCACT\n",
      "0 CGATCCGACGGTTACCACT motifs detected in file\n",
      "\n",
      "Seq is: CCCTCCAACTTCACTCTCC\n",
      "0 CCCTCCAACTTCACTCTCC motifs detected in file\n",
      "\n",
      "Seq is: ATACCAAAACTCATCAGGA\n",
      "0 ATACCAAAACTCATCAGGA motifs detected in file\n",
      "\n",
      "Seq is: GGATAAAAAGCTTATAGAA\n",
      "0 GGATAAAAAGCTTATAGAA motifs detected in file\n",
      "\n",
      "Seq is: CAAGTAACAATTCCAACAC\n",
      "0 CAAGTAACAATTCCAACAC motifs detected in file\n",
      "\n",
      "Seq is: CTACTACCTGGGCCTCGTG\n",
      "0 CTACTACCTGGGCCTCGTG motifs detected in file\n",
      "\n",
      "Seq is: TATTTATCATACCGAAACT\n",
      "0 TATTTATCATACCGAAACT motifs detected in file\n",
      "\n",
      "Seq is: TCTTCCGCACCCATGTCTA\n",
      "0 TCTTCCGCACCCATGTCTA motifs detected in file\n",
      "\n",
      "Seq is: CACGTCCTTACGATCCACA\n",
      "0 CACGTCCTTACGATCCACA motifs detected in file\n",
      "\n",
      "Seq is: CCAACCAACTACGGCTAAC\n",
      "0 CCAACCAACTACGGCTAAC motifs detected in file\n",
      "\n",
      "Seq is: ATTGATAATACTCAGGATC\n",
      "0 ATTGATAATACTCAGGATC motifs detected in file\n",
      "\n",
      "Seq is: AACTGAACTTCACCTCTTC\n",
      "0 AACTGAACTTCACCTCTTC motifs detected in file\n",
      "\n",
      "Seq is: CGATGCTAAAAAAACTACA\n",
      "0 CGATGCTAAAAAAACTACA motifs detected in file\n",
      "\n",
      "Seq is: GTAAAACACATTCCCGCGT\n",
      "0 GTAAAACACATTCCCGCGT motifs detected in file\n",
      "\n",
      "Seq is: CTGACGACGCTTGTCTCAC\n",
      "0 CTGACGACGCTTGTCTCAC motifs detected in file\n",
      "\n",
      "Seq is: CACTATCGCGGATTCGTAT\n",
      "0 CACTATCGCGGATTCGTAT motifs detected in file\n",
      "\n",
      "Seq is: CACAATCTCAAACCTAATC\n",
      "0 CACAATCTCAAACCTAATC motifs detected in file\n",
      "\n",
      "Seq is: ACCCGATAAAAGACGATCC\n",
      "0 ACCCGATAAAAGACGATCC motifs detected in file\n",
      "\n",
      "Seq is: CCCACCTTCGCCTACTCAT\n",
      "0 CCCACCTTCGCCTACTCAT motifs detected in file\n",
      "\n",
      "Seq is: TCTTCTAAGGACGATGTTG\n",
      "0 TCTTCTAAGGACGATGTTG motifs detected in file\n",
      "\n",
      "Seq is: ATATCACGAACTCACCTCA\n",
      "0 ATATCACGAACTCACCTCA motifs detected in file\n",
      "\n",
      "Seq is: TCCCGACTCTGACCCCACC\n",
      "0 TCCCGACTCTGACCCCACC motifs detected in file\n",
      "\n",
      "Seq is: CCGGAAGATACCGACTCCC\n",
      "0 CCGGAAGATACCGACTCCC motifs detected in file\n",
      "\n",
      "Seq is: CTCAGTCCCCACCTCTGGA\n",
      "0 CTCAGTCCCCACCTCTGGA motifs detected in file\n",
      "\n",
      "Seq is: TTAACCCGACTAAACGGAC\n",
      "0 TTAACCCGACTAAACGGAC motifs detected in file\n",
      "\n",
      "Seq is: GACGACGATCCTCCTCCGG\n",
      "0 GACGACGATCCTCCTCCGG motifs detected in file\n",
      "\n",
      "Seq is: ATCATCACCCCACTCCGAA\n",
      "0 ATCATCACCCCACTCCGAA motifs detected in file\n",
      "\n",
      "Seq is: CCTAATCGCAAATCTTCCC\n",
      "0 CCTAATCGCAAATCTTCCC motifs detected in file\n",
      "\n",
      "Seq is: GATAAACAACACCCAGAGT\n",
      "0 GATAAACAACACCCAGAGT motifs detected in file\n",
      "\n",
      "Seq is: ACTCAACCTCACATCCTAT\n",
      "0 ACTCAACCTCACATCCTAT motifs detected in file\n",
      "\n",
      "Seq is: TTAGTACGATTCCGCTCCT\n",
      "0 TTAGTACGATTCCGCTCCT motifs detected in file\n",
      "\n",
      "Seq is: ACTTTGGCTATAGCGGCTA\n",
      "0 ACTTTGGCTATAGCGGCTA motifs detected in file\n",
      "\n",
      "Seq is: TGCCAACATATCCTAACGA\n",
      "0 TGCCAACATATCCTAACGA motifs detected in file\n",
      "\n",
      "Seq is: ACTTACCGACGACACAACC\n",
      "0 ACTTACCGACGACACAACC motifs detected in file\n",
      "\n",
      "Seq is: GTAGACGAGCCCGCATAGT\n",
      "0 GTAGACGAGCCCGCATAGT motifs detected in file\n",
      "\n",
      "Seq is: AGTTGACTACTCGTTCTTC\n",
      "0 AGTTGACTACTCGTTCTTC motifs detected in file\n",
      "\n",
      "Seq is: CTATATTAAGGATGCGGGA\n",
      "0 CTATATTAAGGATGCGGGA motifs detected in file\n",
      "\n",
      "Seq is: GAGTCGGCTACTTGTCAAC\n",
      "0 GAGTCGGCTACTTGTCAAC motifs detected in file\n",
      "\n",
      "Seq is: CTTATCCAACAATCGCCAT\n",
      "0 CTTATCCAACAATCGCCAT motifs detected in file\n",
      "\n",
      "Seq is: TGATTCTAATCATACCATT\n",
      "0 TGATTCTAATCATACCATT motifs detected in file\n",
      "\n",
      "Seq is: AATCCTTCTACTCATCTAT\n",
      "0 AATCCTTCTACTCATCTAT motifs detected in file\n",
      "\n",
      "Seq is: AAACTTCTTGACTAATTAC\n",
      "0 AAACTTCTTGACTAATTAC motifs detected in file\n",
      "\n",
      "Seq is: AAACCCAGACTCAAATATA\n",
      "0 AAACCCAGACTCAAATATA motifs detected in file\n",
      "\n",
      "Seq is: TAGTGTCACTCTTAAGATA\n",
      "0 TAGTGTCACTCTTAAGATA motifs detected in file\n",
      "\n",
      "Seq is: CTACCTGGTACATTGCTTG\n",
      "0 CTACCTGGTACATTGCTTG motifs detected in file\n",
      "\n",
      "Seq is: TTACGATGTCCCTACTTAT\n",
      "0 TTACGATGTCCCTACTTAT motifs detected in file\n",
      "\n",
      "Seq is: AATACCTCTTCATCAGATC\n",
      "0 AATACCTCTTCATCAGATC motifs detected in file\n",
      "\n",
      "Seq is: AAACTTCGAATCCCTCTCG\n",
      "0 AAACTTCGAATCCCTCTCG motifs detected in file\n",
      "\n",
      "Seq is: ACCCAACAAACCCAACACC\n",
      "0 ACCCAACAAACCCAACACC motifs detected in file\n",
      "\n",
      "Seq is: GAGTCACAGTCAAGCTCTA\n",
      "0 GAGTCACAGTCAAGCTCTA motifs detected in file\n",
      "\n",
      "Seq is: TTATTGAAGAACCAGATCC\n",
      "0 TTATTGAAGAACCAGATCC motifs detected in file\n",
      "\n",
      "Seq is: GTGTACTTATAACAACACC\n",
      "0 GTGTACTTATAACAACACC motifs detected in file\n",
      "\n",
      "Seq is: CCTTCTCTGACTATTATTT\n",
      "0 CCTTCTCTGACTATTATTT motifs detected in file\n",
      "\n",
      "Seq is: CCACCTACGCTGTTACCTA\n",
      "0 CCACCTACGCTGTTACCTA motifs detected in file\n",
      "\n",
      "Seq is: AAATGTATTACCCCCATAC\n",
      "0 AAATGTATTACCCCCATAC motifs detected in file\n",
      "\n",
      "Seq is: TCAAAAAAAACAATCCCAA\n",
      "0 TCAAAAAAAACAATCCCAA motifs detected in file\n",
      "\n",
      "Seq is: TTGCTCCCACCATTCCTAC\n",
      "0 TTGCTCCCACCATTCCTAC motifs detected in file\n",
      "\n",
      "Seq is: CCCCCTTAATCCCTTCAGT\n",
      "0 CCCCCTTAATCCCTTCAGT motifs detected in file\n",
      "\n",
      "Seq is: CCCAATCCCACCAATATCA\n",
      "0 CCCAATCCCACCAATATCA motifs detected in file\n",
      "\n",
      "Seq is: TCACACGTACCAATAATGA\n",
      "0 TCACACGTACCAATAATGA motifs detected in file\n",
      "\n",
      "Seq is: AAATAAACCTCAACGTGGT\n",
      "0 AAATAAACCTCAACGTGGT motifs detected in file\n",
      "\n",
      "Seq is: TTTAAAAACCCCGGATTCT\n",
      "0 TTTAAAAACCCCGGATTCT motifs detected in file\n",
      "\n",
      "Seq is: GGTTACCTATCGACAATAG\n",
      "0 GGTTACCTATCGACAATAG motifs detected in file\n",
      "\n",
      "Seq is: GAAATTTTCAACTCTTTCG\n",
      "0 GAAATTTTCAACTCTTTCG motifs detected in file\n",
      "\n",
      "Seq is: GTACAACAATCTGTACCCC\n",
      "0 GTACAACAATCTGTACCCC motifs detected in file\n",
      "\n",
      "Seq is: CGTACTCAATCGTCAAGAA\n",
      "0 CGTACTCAATCGTCAAGAA motifs detected in file\n",
      "\n",
      "Seq is: CACTCGAAAGAGCCATTTA\n",
      "0 CACTCGAAAGAGCCATTTA motifs detected in file\n",
      "\n",
      "Seq is: TTCCCCAGCATTCGGAGAC\n",
      "0 TTCCCCAGCATTCGGAGAC motifs detected in file\n",
      "\n",
      "Seq is: AACAGTCTAAGTGTTAGAC\n",
      "0 AACAGTCTAAGTGTTAGAC motifs detected in file\n",
      "\n",
      "Seq is: TACAAAACCAATTTGATAT\n",
      "0 TACAAAACCAATTTGATAT motifs detected in file\n",
      "\n",
      "Seq is: AAATGTTCTCCTTTTGGGC\n",
      "0 AAATGTTCTCCTTTTGGGC motifs detected in file\n",
      "\n",
      "Seq is: CATTACTACAGCCCCAACT\n",
      "0 CATTACTACAGCCCCAACT motifs detected in file\n",
      "\n",
      "Seq is: CCCTATCCTCCTCTTACCC\n",
      "0 CCCTATCCTCCTCTTACCC motifs detected in file\n",
      "\n",
      "Seq is: CCTATCCACATACTTGTAC\n",
      "0 CCTATCCACATACTTGTAC motifs detected in file\n",
      "\n",
      "Seq is: TCCCACAAAAGAGCACACT\n",
      "0 TCCCACAAAAGAGCACACT motifs detected in file\n",
      "\n",
      "Seq is: TACTCCCAAAATACAACAA\n",
      "0 TACTCCCAAAATACAACAA motifs detected in file\n",
      "\n",
      "Seq is: TTACACCACCCACTCACTC\n",
      "0 TTACACCACCCACTCACTC motifs detected in file\n",
      "\n",
      "Seq is: GGGGTAACACAACACCATT\n",
      "0 GGGGTAACACAACACCATT motifs detected in file\n",
      "\n",
      "Seq is: TATACATCTCCCTCATATC\n",
      "0 TATACATCTCCCTCATATC motifs detected in file\n",
      "\n",
      "Seq is: CCGACACTGATCATACAAC\n",
      "0 CCGACACTGATCATACAAC motifs detected in file\n",
      "\n",
      "Seq is: TCAGGACATTCATCCTCTC\n",
      "0 TCAGGACATTCATCCTCTC motifs detected in file\n",
      "\n",
      "Seq is: ACTATAAACTAGTCCTCTT\n",
      "0 ACTATAAACTAGTCCTCTT motifs detected in file\n",
      "\n",
      "Seq is: GCACCAATGATCGTGTCTC\n",
      "0 GCACCAATGATCGTGTCTC motifs detected in file\n",
      "\n",
      "Seq is: TCAAGAGGGTCATCCAATT\n",
      "0 TCAAGAGGGTCATCCAATT motifs detected in file\n",
      "\n",
      "Seq is: ATCACCCCCCATTCCGCTC\n",
      "0 ATCACCCCCCATTCCGCTC motifs detected in file\n",
      "\n",
      "Seq is: CAATCGCTCCGAACGATCT\n",
      "0 CAATCGCTCCGAACGATCT motifs detected in file\n",
      "\n",
      "Seq is: TCAGTAGTTTTTCGATAAT\n",
      "0 TCAGTAGTTTTTCGATAAT motifs detected in file\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Seq is: CACCCTCATCTCAAACTTC\n",
      "0 CACCCTCATCTCAAACTTC motifs detected in file\n",
      "\n",
      "Seq is: AGGAACTCTCTCCTAATAC\n",
      "0 AGGAACTCTCTCCTAATAC motifs detected in file\n",
      "\n",
      "Seq is: TACGCTGACACTCACGCAA\n",
      "0 TACGCTGACACTCACGCAA motifs detected in file\n",
      "\n",
      "Seq is: GCATCAAACTCAAACGATC\n",
      "0 GCATCAAACTCAAACGATC motifs detected in file\n",
      "\n",
      "Seq is: CGTCTTATCATTACTCCTA\n",
      "0 CGTCTTATCATTACTCCTA motifs detected in file\n",
      "\n",
      "Seq is: CATTCGGGCACCCGCTAAT\n",
      "0 CATTCGGGCACCCGCTAAT motifs detected in file\n",
      "\n",
      "Seq is: ACTCTTACTGACGCGGCCA\n",
      "0 ACTCTTACTGACGCGGCCA motifs detected in file\n",
      "\n",
      "Seq is: CTTCGAAGTCCCCCAAACC\n",
      "0 CTTCGAAGTCCCCCAAACC motifs detected in file\n",
      "\n",
      "Seq is: TACTCTTACCGACAATGAT\n",
      "0 TACTCTTACCGACAATGAT motifs detected in file\n",
      "\n",
      "Seq is: GCTCCCGATACACCGACTA\n",
      "0 GCTCCCGATACACCGACTA motifs detected in file\n",
      "\n",
      "Seq is: ACTTCTCATACGTTACTCG\n",
      "0 ACTTCTCATACGTTACTCG motifs detected in file\n",
      "\n",
      "Seq is: CTAAAATCCAGACAAACAG\n",
      "0 CTAAAATCCAGACAAACAG motifs detected in file\n",
      "\n",
      "Seq is: CATCCGTCTACCTCGAACA\n",
      "0 CATCCGTCTACCTCGAACA motifs detected in file\n",
      "\n",
      "Seq is: ATATTAATACGGAGTATCC\n",
      "0 ATATTAATACGGAGTATCC motifs detected in file\n",
      "\n",
      "Seq is: CTATCATGTTCCTTCCCCA\n",
      "0 CTATCATGTTCCTTCCCCA motifs detected in file\n",
      "\n",
      "Seq is: TCCGATACACAAAACAGTC\n",
      "0 TCCGATACACAAAACAGTC motifs detected in file\n",
      "\n",
      "Seq is: CCCCAACTCTTACTCACAC\n",
      "0 CCCCAACTCTTACTCACAC motifs detected in file\n",
      "\n",
      "Seq is: TCCGCATAATATGGTATCG\n",
      "0 TCCGCATAATATGGTATCG motifs detected in file\n",
      "\n",
      "Seq is: GCGGATCAAAATTCTCATG\n",
      "0 GCGGATCAAAATTCTCATG motifs detected in file\n",
      "\n",
      "Seq is: ACGCCGTTCATGATAACTG\n",
      "0 ACGCCGTTCATGATAACTG motifs detected in file\n",
      "\n",
      "Seq is: GGTCGCTACCCCCGAAGCT\n",
      "0 GGTCGCTACCCCCGAAGCT motifs detected in file\n",
      "\n",
      "Seq is: GTACCCGAAATCCCTCAGT\n",
      "0 GTACCCGAAATCCCTCAGT motifs detected in file\n",
      "\n",
      "Seq is: ATTCACCTCAGGCATTTCT\n",
      "0 ATTCACCTCAGGCATTTCT motifs detected in file\n",
      "\n",
      "Seq is: CCATAGAAATGATATTTTC\n",
      "0 CCATAGAAATGATATTTTC motifs detected in file\n",
      "\n",
      "Seq is: GATAACACATTCGATCAGT\n",
      "0 GATAACACATTCGATCAGT motifs detected in file\n",
      "\n",
      "Seq is: ATAATTCAACAACCGAGTC\n",
      "0 ATAATTCAACAACCGAGTC motifs detected in file\n",
      "\n",
      "Seq is: CTCAAACTATCAAGAACCC\n",
      "0 CTCAAACTATCAAGAACCC motifs detected in file\n",
      "\n",
      "Seq is: GTCACTCTCACTCATCATC\n",
      "0 GTCACTCTCACTCATCATC motifs detected in file\n",
      "\n",
      "Seq is: TTACAAATCACTCGGATCC\n",
      "0 TTACAAATCACTCGGATCC motifs detected in file\n",
      "\n",
      "Seq is: CACAACACTCACATTTAAT\n",
      "0 CACAACACTCACATTTAAT motifs detected in file\n",
      "\n",
      "Seq is: CACGCTACTCATCCCCTTC\n",
      "0 CACGCTACTCATCCCCTTC motifs detected in file\n",
      "\n",
      "Seq is: CCTCGGATGATCCCACATC\n",
      "0 CCTCGGATGATCCCACATC motifs detected in file\n",
      "\n",
      "Seq is: TTATCCTTCATACACGGAC\n",
      "0 TTATCCTTCATACACGGAC motifs detected in file\n",
      "\n",
      "Seq is: GCAAGTCCGCAAGACCGAC\n",
      "0 GCAAGTCCGCAAGACCGAC motifs detected in file\n",
      "\n",
      "Seq is: CAACGGAGTAGCCCACTAC\n",
      "0 CAACGGAGTAGCCCACTAC motifs detected in file\n",
      "\n",
      "Seq is: TATCGGTTCCACCCCTATT\n",
      "0 TATCGGTTCCACCCCTATT motifs detected in file\n",
      "\n",
      "Seq is: CACACCAAAGCTTCTTCTA\n",
      "0 CACACCAAAGCTTCTTCTA motifs detected in file\n",
      "\n",
      "Seq is: TATTTTATACTAATCAAGA\n",
      "0 TATTTTATACTAATCAAGA motifs detected in file\n",
      "\n",
      "Seq is: CACCGACACTTACAATATT\n",
      "0 CACCGACACTTACAATATT motifs detected in file\n",
      "\n",
      "Seq is: AATTCCTCTAAACATCCCT\n",
      "0 AATTCCTCTAAACATCCCT motifs detected in file\n",
      "\n",
      "Seq is: CTAATCATATCTCTCCATC\n",
      "0 CTAATCATATCTCTCCATC motifs detected in file\n",
      "\n",
      "Seq is: TCAAAAAAAGCACTATCAC\n",
      "0 TCAAAAAAAGCACTATCAC motifs detected in file\n",
      "\n",
      "Seq is: CAAGTGACCTATTCACCGC\n",
      "0 CAAGTGACCTATTCACCGC motifs detected in file\n",
      "\n",
      "Seq is: AACCGAACGGTACTAACAC\n",
      "0 AACCGAACGGTACTAACAC motifs detected in file\n",
      "\n",
      "Seq is: TCCCCATCCTCAGTCCATC\n",
      "0 TCCCCATCCTCAGTCCATC motifs detected in file\n",
      "\n",
      "Seq is: AATCATAATCCTCCCCCCA\n",
      "0 AATCATAATCCTCCCCCCA motifs detected in file\n",
      "\n",
      "Seq is: ACAATCCCCCAGCCTCCTT\n",
      "0 ACAATCCCCCAGCCTCCTT motifs detected in file\n",
      "\n",
      "Seq is: TTCCAACCCCTTGTCGATT\n",
      "0 TTCCAACCCCTTGTCGATT motifs detected in file\n",
      "\n",
      "Seq is: TATCCAACAACAACTAAAC\n",
      "0 TATCCAACAACAACTAAAC motifs detected in file\n",
      "\n",
      "Seq is: CAATTTTTTATCATCTCCC\n",
      "0 CAATTTTTTATCATCTCCC motifs detected in file\n",
      "\n",
      "Seq is: TACTACGATTATTAATCCG\n",
      "0 TACTACGATTATTAATCCG motifs detected in file\n",
      "\n",
      "Seq is: ACACCCACCAACACAACTA\n",
      "0 ACACCCACCAACACAACTA motifs detected in file\n",
      "\n",
      "Seq is: AGTTTAATACACAAAAAAC\n",
      "0 AGTTTAATACACAAAAAAC motifs detected in file\n",
      "\n",
      "Seq is: CTTTCAGTACAGTCACCAT\n",
      "0 CTTTCAGTACAGTCACCAT motifs detected in file\n",
      "\n",
      "Seq is: CATTATATTAACAACCCTG\n",
      "0 CATTATATTAACAACCCTG motifs detected in file\n",
      "\n",
      "Seq is: CTAATCAAAATCGTAACCT\n",
      "0 CTAATCAAAATCGTAACCT motifs detected in file\n",
      "\n",
      "Seq is: CATCCAAATCCAATACATG\n",
      "0 CATCCAAATCCAATACATG motifs detected in file\n",
      "\n",
      "Seq is: CATCAGATCCGGTATACAC\n",
      "0 CATCAGATCCGGTATACAC motifs detected in file\n",
      "\n",
      "Seq is: AACCTCTAACTCTGATCAT\n",
      "0 AACCTCTAACTCTGATCAT motifs detected in file\n",
      "\n",
      "Seq is: CCCGATCCGGGTGGCGACG\n",
      "0 CCCGATCCGGGTGGCGACG motifs detected in file\n",
      "\n",
      "Seq is: AAGCGTCCGCCGTTTCTGA\n",
      "0 AAGCGTCCGCCGTTTCTGA motifs detected in file\n",
      "\n",
      "Seq is: TCATACCGTTATCCGTGTT\n",
      "0 TCATACCGTTATCCGTGTT motifs detected in file\n",
      "\n",
      "Seq is: ATAACCGATTCTCCCTCAC\n",
      "0 ATAACCGATTCTCCCTCAC motifs detected in file\n",
      "\n",
      "Seq is: CCACAACTCCCAATACTCT\n",
      "0 CCACAACTCCCAATACTCT motifs detected in file\n",
      "\n",
      "Seq is: CATCGATATTACTTGTCGC\n",
      "0 CATCGATATTACTTGTCGC motifs detected in file\n",
      "\n",
      "Seq is: TATCATAATAAGGAAGATC\n",
      "0 TATCATAATAAGGAAGATC motifs detected in file\n",
      "\n",
      "Seq is: CGTATCATCCCTCCTATAC\n",
      "0 CGTATCATCCCTCCTATAC motifs detected in file\n",
      "\n",
      "Seq is: TCCACACTCGCTATATGAT\n",
      "0 TCCACACTCGCTATATGAT motifs detected in file\n",
      "\n",
      "Seq is: CATAAGGATCTTCACTCTA\n",
      "0 CATAAGGATCTTCACTCTA motifs detected in file\n",
      "\n",
      "Seq is: CCATTTACGATCATATTAT\n",
      "0 CCATTTACGATCATATTAT motifs detected in file\n",
      "\n",
      "Seq is: AAATACATTTACTCCCCGT\n",
      "0 AAATACATTTACTCCCCGT motifs detected in file\n",
      "\n",
      "Seq is: AAACCATTTATACTAATAG\n",
      "0 AAACCATTTATACTAATAG motifs detected in file\n",
      "\n",
      "Seq is: TATTAAATTACTCAGCTTT\n",
      "0 TATTAAATTACTCAGCTTT motifs detected in file\n",
      "\n",
      "Seq is: AGTAAGCAAAACAAATTTG\n",
      "0 AGTAAGCAAAACAAATTTG motifs detected in file\n",
      "\n",
      "Seq is: ATATATGGTTAAGCCAAGT\n",
      "0 ATATATGGTTAAGCCAAGT motifs detected in file\n",
      "\n",
      "Seq is: CAGATTAGGAAAAACATCA\n",
      "0 CAGATTAGGAAAAACATCA motifs detected in file\n",
      "\n",
      "Seq is: GTGAGTATCCGGTCTGAAT\n",
      "0 GTGAGTATCCGGTCTGAAT motifs detected in file\n",
      "\n",
      "Seq is: CCCGATCCTACTACTAATT\n",
      "0 CCCGATCCTACTACTAATT motifs detected in file\n",
      "\n",
      "Seq is: ATTCTCCCTACTGTATTGA\n",
      "0 ATTCTCCCTACTGTATTGA motifs detected in file\n",
      "\n",
      "Seq is: TAATCACCGTCCAATCAAC\n",
      "0 TAATCACCGTCCAATCAAC motifs detected in file\n",
      "\n",
      "Seq is: AAACATCCCGAGTACCATC\n",
      "0 AAACATCCCGAGTACCATC motifs detected in file\n",
      "\n",
      "Seq is: CCCATTTTCCTCCCGTTAA\n",
      "0 CCCATTTTCCTCCCGTTAA motifs detected in file\n",
      "\n",
      "Seq is: AGATCTAGTTTATTATTCT\n",
      "0 AGATCTAGTTTATTATTCT motifs detected in file\n",
      "\n",
      "Seq is: TCCATTATCGATGATTCTT\n",
      "0 TCCATTATCGATGATTCTT motifs detected in file\n",
      "\n",
      "Seq is: CTTAAAATACCTCTTTCCC\n",
      "0 CTTAAAATACCTCTTTCCC motifs detected in file\n",
      "\n",
      "Seq is: TGCGCCCGCCCCCTATATC\n",
      "0 TGCGCCCGCCCCCTATATC motifs detected in file\n",
      "\n",
      "Seq is: CCAGCTTCGGCGTGAGCAT\n",
      "0 CCAGCTTCGGCGTGAGCAT motifs detected in file\n",
      "\n",
      "Seq is: TCCCCACCTAAAAAGATAC\n",
      "0 TCCCCACCTAAAAAGATAC motifs detected in file\n",
      "\n",
      "Seq is: ATCGGCAACTCAACACCAT\n",
      "0 ATCGGCAACTCAACACCAT motifs detected in file\n",
      "\n",
      "Seq is: CAGTTTTACATTATTAATA\n",
      "0 CAGTTTTACATTATTAATA motifs detected in file\n",
      "\n",
      "Seq is: ATCATCATTCCGATCCTCC\n",
      "0 ATCATCATTCCGATCCTCC motifs detected in file\n",
      "\n",
      "Seq is: CACAACTAATAATTTTAAT\n",
      "0 CACAACTAATAATTTTAAT motifs detected in file\n",
      "\n",
      "Seq is: TCCGCTTCAAATAATGAGA\n",
      "0 TCCGCTTCAAATAATGAGA motifs detected in file\n",
      "\n",
      "Seq is: AAAAACTTACAACAGTTTT\n",
      "0 AAAAACTTACAACAGTTTT motifs detected in file\n",
      "\n",
      "Seq is: GATCAATTAACCTTCAATT\n",
      "0 GATCAATTAACCTTCAATT motifs detected in file\n",
      "\n",
      "Seq is: GCCATGATAAATATGATTT\n",
      "0 GCCATGATAAATATGATTT motifs detected in file\n",
      "\n",
      "Seq is: TCTCATTCTGGGAGTAGTT\n",
      "0 TCTCATTCTGGGAGTAGTT motifs detected in file\n",
      "\n",
      "Seq is: ATCTACCTCTGTATGTCTT\n",
      "0 ATCTACCTCTGTATGTCTT motifs detected in file\n",
      "\n",
      "Seq is: TATCAGTTTGGTGTAGATG\n",
      "0 TATCAGTTTGGTGTAGATG motifs detected in file\n",
      "\n",
      "Seq is: TTTTACGGTCATAGTCCGC\n",
      "0 TTTTACGGTCATAGTCCGC motifs detected in file\n",
      "\n",
      "Seq is: CGCCGAAGCTTCGGTTTCA\n",
      "0 CGCCGAAGCTTCGGTTTCA motifs detected in file\n",
      "\n",
      "Seq is: CTACAAACCTACATTTCAC\n",
      "0 CTACAAACCTACATTTCAC motifs detected in file\n",
      "\n",
      "Seq is: TTTATAATCAACCGCCTAC\n",
      "0 TTTATAATCAACCGCCTAC motifs detected in file\n",
      "\n",
      "Seq is: TTCGTCTATCACTCCTTTC\n",
      "0 TTCGTCTATCACTCCTTTC motifs detected in file\n",
      "\n",
      "Seq is: AACTCGGTTATTACTGCAC\n",
      "0 AACTCGGTTATTACTGCAC motifs detected in file\n",
      "\n",
      "Seq is: TTCAGGCACCTTCGGACAC\n",
      "0 TTCAGGCACCTTCGGACAC motifs detected in file\n",
      "\n",
      "Seq is: CGATGTTTTTTACAACTCG\n",
      "0 CGATGTTTTTTACAACTCG motifs detected in file\n",
      "\n",
      "Seq is: GCATCTACGGCAGCCTTTA\n",
      "0 GCATCTACGGCAGCCTTTA motifs detected in file\n",
      "\n",
      "Seq is: CCACTTCCCTCTGAGCTTC\n",
      "0 CCACTTCCCTCTGAGCTTC motifs detected in file\n",
      "\n",
      "Seq is: ATGAGACTCCGAACATCCT\n",
      "0 ATGAGACTCCGAACATCCT motifs detected in file\n",
      "\n",
      "Seq is: CCCATTTTATCTCTGGGTC\n",
      "0 CCCATTTTATCTCTGGGTC motifs detected in file\n",
      "\n",
      "Seq is: ATTTTAACATTATTCGTCA\n",
      "0 ATTTTAACATTATTCGTCA motifs detected in file\n",
      "\n",
      "Seq is: CGAACTTAATAAACCAAAG\n",
      "0 CGAACTTAATAAACCAAAG motifs detected in file\n",
      "\n",
      "Seq is: CCAACAAAAGATAATCTGA\n",
      "0 CCAACAAAAGATAATCTGA motifs detected in file\n",
      "\n",
      "Seq is: TACCACTCGAGTCCACTAA\n",
      "0 TACCACTCGAGTCCACTAA motifs detected in file\n",
      "\n",
      "Seq is: CTATGAGGACTACGCTCAT\n",
      "0 CTATGAGGACTACGCTCAT motifs detected in file\n",
      "\n",
      "Seq is: TATGCCTACACAAATCCTC\n",
      "0 TATGCCTACACAAATCCTC motifs detected in file\n",
      "\n",
      "Seq is: ACCCTGAAGATCCCCTAAA\n",
      "0 ACCCTGAAGATCCCCTAAA motifs detected in file\n",
      "\n",
      "Seq is: TCGCCCCACTACGGACAAC\n",
      "0 TCGCCCCACTACGGACAAC motifs detected in file\n",
      "\n",
      "Seq is: CCCCGGTCACGGGAGGATT\n",
      "0 CCCCGGTCACGGGAGGATT motifs detected in file\n",
      "\n",
      "Seq is: AACCCCCCATCCCCGATCC\n",
      "0 AACCCCCCATCCCCGATCC motifs detected in file\n",
      "\n",
      "Seq is: GACCTCACCATTTTCCGAG\n",
      "0 GACCTCACCATTTTCCGAG motifs detected in file\n",
      "\n",
      "Seq is: TCTTTTTAGGACGCTTCTT\n",
      "0 TCTTTTTAGGACGCTTCTT motifs detected in file\n",
      "\n",
      "Seq is: TTTTTGAAGACTCCATTAT\n",
      "0 TTTTTGAAGACTCCATTAT motifs detected in file\n",
      "\n",
      "Seq is: TTATCCTAATAGGGCATAG\n",
      "0 TTATCCTAATAGGGCATAG motifs detected in file\n",
      "\n",
      "Seq is: CTTCCGGAAAAACCTGTCC\n",
      "0 CTTCCGGAAAAACCTGTCC motifs detected in file\n",
      "\n",
      "Seq is: ACCACACACCACCGGAACC\n",
      "0 ACCACACACCACCGGAACC motifs detected in file\n",
      "\n",
      "Seq is: ATACACGAAAGAGCACAAT\n",
      "0 ATACACGAAAGAGCACAAT motifs detected in file\n",
      "\n",
      "Seq is: GTAGCGCGGTAGTAACCAT\n",
      "0 GTAGCGCGGTAGTAACCAT motifs detected in file\n",
      "\n",
      "Seq is: ATACCAATCACACAACCAA\n",
      "0 ATACCAATCACACAACCAA motifs detected in file\n",
      "\n",
      "Seq is: TCATCCGGATCATACTCCT\n",
      "0 TCATCCGGATCATACTCCT motifs detected in file\n",
      "\n",
      "Seq is: CGCAATACCTCACCTTCAC\n",
      "0 CGCAATACCTCACCTTCAC motifs detected in file\n",
      "\n",
      "Seq is: TTTAGTGTACCGATCCGGC\n",
      "0 TTTAGTGTACCGATCCGGC motifs detected in file\n",
      "\n",
      "Seq is: CTCCAGTAATCCTCCCGAC\n",
      "0 CTCCAGTAATCCTCCCGAC motifs detected in file\n",
      "\n",
      "Seq is: TCTCCCGGGGACAATCCCC\n",
      "0 TCTCCCGGGGACAATCCCC motifs detected in file\n",
      "\n",
      "Seq is: AGTACCCGACCCAAAATGA\n",
      "0 AGTACCCGACCCAAAATGA motifs detected in file\n",
      "\n",
      "Seq is: TATACTATCCGTACACTAA\n",
      "0 TATACTATCCGTACACTAA motifs detected in file\n",
      "\n",
      "Seq is: CCACCCAGTAATACACAAC\n",
      "0 CCACCCAGTAATACACAAC motifs detected in file\n",
      "\n",
      "Seq is: AGCACGTCCATCTCCGAAT\n",
      "0 AGCACGTCCATCTCCGAAT motifs detected in file\n",
      "\n",
      "Seq is: GATCTTCACACTTTTGCAT\n",
      "0 GATCTTCACACTTTTGCAT motifs detected in file\n",
      "\n",
      "Seq is: CCGAACCTAATTCCGCTGT\n",
      "0 CCGAACCTAATTCCGCTGT motifs detected in file\n",
      "\n",
      "Seq is: CGCTAAAGATCCTATCAGT\n",
      "0 CGCTAAAGATCCTATCAGT motifs detected in file\n",
      "\n",
      "Seq is: CATCTTAATCTTAACACTT\n",
      "0 CATCTTAATCTTAACACTT motifs detected in file\n",
      "\n",
      "Seq is: CTACTATTCACATCTCCCT\n",
      "0 CTACTATTCACATCTCCCT motifs detected in file\n",
      "\n",
      "Seq is: TCCAATTACCAACTATAAC\n",
      "0 TCCAATTACCAACTATAAC motifs detected in file\n",
      "\n",
      "Seq is: GATCCCACCGCGAAGGTTA\n",
      "0 GATCCCACCGCGAAGGTTA motifs detected in file\n",
      "\n",
      "Seq is: ATCCACGTACTCATCCACC\n",
      "0 ATCCACGTACTCATCCACC motifs detected in file\n",
      "\n",
      "Seq is: GGACGTCATTACAATCGCC\n",
      "0 GGACGTCATTACAATCGCC motifs detected in file\n",
      "\n",
      "Seq is: AATCCGCATGCCGGTCCCG\n",
      "0 AATCCGCATGCCGGTCCCG motifs detected in file\n",
      "\n",
      "Seq is: ATAACCAACTTACTCATCC\n",
      "0 ATAACCAACTTACTCATCC motifs detected in file\n",
      "\n",
      "Seq is: GACTACCAAAGCTATTATT\n",
      "0 GACTACCAAAGCTATTATT motifs detected in file\n",
      "\n",
      "Seq is: GATCATACCCCTATTCCCC\n",
      "0 GATCATACCCCTATTCCCC motifs detected in file\n",
      "\n",
      "Seq is: ACATCCACACGGAACACCA\n",
      "0 ACATCCACACGGAACACCA motifs detected in file\n",
      "\n",
      "Seq is: TTCTTCACCCGATCCCGTA\n",
      "0 TTCTTCACCCGATCCCGTA motifs detected in file\n",
      "\n",
      "Seq is: AAAATTAGAATCTCGCTTT\n",
      "0 AAAATTAGAATCTCGCTTT motifs detected in file\n",
      "\n",
      "Seq is: CGGATATTAGTGACACGGG\n",
      "0 CGGATATTAGTGACACGGG motifs detected in file\n",
      "\n",
      "Seq is: CGAGTATTCCCCTACCGGT\n",
      "0 CGAGTATTCCCCTACCGGT motifs detected in file\n",
      "\n",
      "Seq is: ACCGATCCAAATATCTATC\n",
      "0 ACCGATCCAAATATCTATC motifs detected in file\n",
      "\n",
      "Seq is: AACCCACCAACCACATTTA\n",
      "0 AACCCACCAACCACATTTA motifs detected in file\n",
      "\n",
      "Seq is: CTCACTCCGTCCTCAGGCT\n",
      "0 CTCACTCCGTCCTCAGGCT motifs detected in file\n",
      "\n",
      "Seq is: CCTCCAATCAACACCGTTA\n",
      "0 CCTCCAATCAACACCGTTA motifs detected in file\n",
      "\n",
      "Seq is: TTTTTACTAATTCCTATGA\n",
      "0 TTTTTACTAATTCCTATGA motifs detected in file\n",
      "\n",
      "Seq is: TCATATTCTCTAGTCCAAG\n",
      "0 TCATATTCTCTAGTCCAAG motifs detected in file\n",
      "\n",
      "Seq is: CAGGAAATCACAACACATA\n",
      "0 CAGGAAATCACAACACATA motifs detected in file\n",
      "\n",
      "Seq is: CCAATAGTAAACAAAACTC\n",
      "0 CCAATAGTAAACAAAACTC motifs detected in file\n",
      "\n",
      "Seq is: CAATCAAACTAATCAGTAA\n",
      "0 CAATCAAACTAATCAGTAA motifs detected in file\n",
      "\n",
      "Seq is: CAACCCACCACTAATCAGC\n",
      "0 CAACCCACCACTAATCAGC motifs detected in file\n",
      "\n",
      "Seq is: CAACAACTACTCTATAAAC\n",
      "0 CAACAACTACTCTATAAAC motifs detected in file\n",
      "\n",
      "Seq is: CTCCACCCCTAGTTATCTC\n",
      "0 CTCCACCCCTAGTTATCTC motifs detected in file\n",
      "\n",
      "Seq is: CCCCTTTATCTTACTAGTC\n",
      "0 CCCCTTTATCTTACTAGTC motifs detected in file\n",
      "\n",
      "Seq is: ATGACGCCGCCCATCCGGA\n",
      "0 ATGACGCCGCCCATCCGGA motifs detected in file\n",
      "\n",
      "Seq is: TCCTAACACCCCCGTTACT\n",
      "0 TCCTAACACCCCCGTTACT motifs detected in file\n",
      "\n",
      "Seq is: TACTTCGCTTGTCTAAAAG\n",
      "0 TACTTCGCTTGTCTAAAAG motifs detected in file\n",
      "\n",
      "Seq is: CAAGTAAAACCAAGAGTCC\n",
      "0 CAAGTAAAACCAAGAGTCC motifs detected in file\n",
      "\n",
      "Seq is: CAAACAATATTAAAAAATA\n",
      "0 CAAACAATATTAAAAAATA motifs detected in file\n",
      "\n",
      "Seq is: AAAATACCCGAAACCACTC\n",
      "0 AAAATACCCGAAACCACTC motifs detected in file\n",
      "\n",
      "Seq is: CCTCCATCCACCATCAAAC\n",
      "0 CCTCCATCCACCATCAAAC motifs detected in file\n",
      "\n",
      "Seq is: ACAAATTATAAAAATCAAC\n",
      "0 ACAAATTATAAAAATCAAC motifs detected in file\n",
      "\n",
      "Seq is: CCACTACTCCTTATCACAT\n",
      "0 CCACTACTCCTTATCACAT motifs detected in file\n",
      "\n",
      "Seq is: TCCTCATACCCCCATTAAT\n",
      "0 TCCTCATACCCCCATTAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCACCCGGTATGCCATCA\n",
      "0 ACCACCCGGTATGCCATCA motifs detected in file\n",
      "\n",
      "Seq is: TAAATCAACCCCGTAAAGT\n",
      "0 TAAATCAACCCCGTAAAGT motifs detected in file\n",
      "\n",
      "Seq is: GACATTTCTCCACAACCAA\n",
      "0 GACATTTCTCCACAACCAA motifs detected in file\n",
      "\n",
      "Seq is: GAGAATTAGAAATTGAATT\n",
      "0 GAGAATTAGAAATTGAATT motifs detected in file\n",
      "\n",
      "Seq is: TTCCAATTACGATTCAATC\n",
      "0 TTCCAATTACGATTCAATC motifs detected in file\n",
      "\n",
      "Seq is: GAAATGTCACCCGAGATCT\n",
      "0 GAAATGTCACCCGAGATCT motifs detected in file\n",
      "\n",
      "Seq is: CCCCCATCTCCCCCACGAT\n",
      "0 CCCCCATCTCCCCCACGAT motifs detected in file\n",
      "\n",
      "Seq is: ATCCCATTTATGCCCGGGA\n",
      "0 ATCCCATTTATGCCCGGGA motifs detected in file\n",
      "\n",
      "Seq is: TAAAGTTTCTAAAAATCCC\n",
      "0 TAAAGTTTCTAAAAATCCC motifs detected in file\n",
      "\n",
      "Seq is: CTTAATTAAGATCCTGCTA\n",
      "0 CTTAATTAAGATCCTGCTA motifs detected in file\n",
      "\n",
      "Seq is: CCCGTACTTTGACACCAAA\n",
      "0 CCCGTACTTTGACACCAAA motifs detected in file\n",
      "\n",
      "Seq is: CGAGGTGTCTAAAGTCTCG\n",
      "0 CGAGGTGTCTAAAGTCTCG motifs detected in file\n",
      "\n",
      "Seq is: TAACTGGCATCATATGGGG\n",
      "0 TAACTGGCATCATATGGGG motifs detected in file\n",
      "\n",
      "Seq is: GCCAGCACATCGCCACTTT\n",
      "0 GCCAGCACATCGCCACTTT motifs detected in file\n",
      "\n",
      "Seq is: CACCAAACCAAATCTGCAG\n",
      "0 CACCAAACCAAATCTGCAG motifs detected in file\n",
      "\n",
      "Seq is: GCCCTTAACGTAGACAAAA\n",
      "0 GCCCTTAACGTAGACAAAA motifs detected in file\n",
      "\n",
      "Seq is: ATTCGGATTACACCCCTGT\n",
      "0 ATTCGGATTACACCCCTGT motifs detected in file\n",
      "\n",
      "Seq is: CGAGTACTCACGTTCTGCA\n",
      "0 CGAGTACTCACGTTCTGCA motifs detected in file\n",
      "\n",
      "Seq is: GAACACTACATTAATAATA\n",
      "0 GAACACTACATTAATAATA motifs detected in file\n",
      "\n",
      "Seq is: TGCTTACCCCCGAAGTTAG\n",
      "0 TGCTTACCCCCGAAGTTAG motifs detected in file\n",
      "\n",
      "Seq is: CCCTCATGATGAGCTAACA\n",
      "0 CCCTCATGATGAGCTAACA motifs detected in file\n",
      "\n",
      "Seq is: GTTGCAGTTCCTCAGCGTC\n",
      "0 GTTGCAGTTCCTCAGCGTC motifs detected in file\n",
      "\n",
      "Seq is: CAGCGGACCAAGATCCTTA\n",
      "0 CAGCGGACCAAGATCCTTA motifs detected in file\n",
      "\n",
      "Seq is: TTACCCCCTTCATACATCC\n",
      "0 TTACCCCCTTCATACATCC motifs detected in file\n",
      "\n",
      "Seq is: TCAACTTCTAATCAGGCGG\n",
      "0 TCAACTTCTAATCAGGCGG motifs detected in file\n",
      "\n",
      "Seq is: CATCAGCCACATGAGCATC\n",
      "0 CATCAGCCACATGAGCATC motifs detected in file\n",
      "\n",
      "Seq is: CAAGTCATGGTAACCACCG\n",
      "0 CAAGTCATGGTAACCACCG motifs detected in file\n",
      "\n",
      "Seq is: GTTAACTAAACTACCATTC\n",
      "0 GTTAACTAAACTACCATTC motifs detected in file\n",
      "\n",
      "Seq is: CCTCCCTAGCAACTGGAGC\n",
      "0 CCTCCCTAGCAACTGGAGC motifs detected in file\n",
      "\n",
      "Seq is: AGACAATACATTTCCTACG\n",
      "0 AGACAATACATTTCCTACG motifs detected in file\n",
      "\n",
      "Seq is: CATCCCTACCCTCCCGCTA\n",
      "0 CATCCCTACCCTCCCGCTA motifs detected in file\n",
      "\n",
      "Seq is: CTCCTGATCCTACTACCGC\n",
      "0 CTCCTGATCCTACTACCGC motifs detected in file\n",
      "\n",
      "Seq is: CCGTCCTATCAAGTCTGCC\n",
      "0 CCGTCCTATCAAGTCTGCC motifs detected in file\n",
      "\n",
      "Seq is: AAAGATAAAGGACTCGCAG\n",
      "0 AAAGATAAAGGACTCGCAG motifs detected in file\n",
      "\n",
      "Seq is: ACTCTACAATCATAATCAA\n",
      "0 ACTCTACAATCATAATCAA motifs detected in file\n",
      "\n",
      "Seq is: TCAAAACAACACTCACAAT\n",
      "0 TCAAAACAACACTCACAAT motifs detected in file\n",
      "\n",
      "Seq is: CCTTTTCCCGTATGTCCTG\n",
      "0 CCTTTTCCCGTATGTCCTG motifs detected in file\n",
      "\n",
      "Seq is: ATCCTTCGTCTATTCCTTT\n",
      "0 ATCCTTCGTCTATTCCTTT motifs detected in file\n",
      "\n",
      "Seq is: TACTAATACTCCCGCACTA\n",
      "0 TACTAATACTCCCGCACTA motifs detected in file\n",
      "\n",
      "Seq is: GTACTTTCCACTATTCGAG\n",
      "0 GTACTTTCCACTATTCGAG motifs detected in file\n",
      "\n",
      "Seq is: AAGATACTATCCCCTTCAT\n",
      "0 AAGATACTATCCCCTTCAT motifs detected in file\n",
      "\n",
      "Seq is: CGCAGAACATCTGGATGAA\n",
      "0 CGCAGAACATCTGGATGAA motifs detected in file\n",
      "\n",
      "Seq is: CGCGACGTACACGGTAATT\n",
      "0 CGCGACGTACACGGTAATT motifs detected in file\n",
      "\n",
      "Seq is: CTATATATCCTAAATCGGA\n",
      "0 CTATATATCCTAAATCGGA motifs detected in file\n",
      "\n",
      "Seq is: TATTAAATTGAAACTGTTT\n",
      "0 TATTAAATTGAAACTGTTT motifs detected in file\n",
      "\n",
      "Seq is: CAATACTTTACCAAAAAGA\n",
      "0 CAATACTTTACCAAAAAGA motifs detected in file\n",
      "\n",
      "Seq is: TTATGGAAAAACTTTTTCA\n",
      "0 TTATGGAAAAACTTTTTCA motifs detected in file\n",
      "\n",
      "Seq is: GTACCTCCGGTACCCCAAC\n",
      "0 GTACCTCCGGTACCCCAAC motifs detected in file\n",
      "\n",
      "Seq is: CGAACTTTGGTCGAAACCC\n",
      "0 CGAACTTTGGTCGAAACCC motifs detected in file\n",
      "\n",
      "Seq is: CCCAAGCTAAGGAAGGAAA\n",
      "0 CCCAAGCTAAGGAAGGAAA motifs detected in file\n",
      "\n",
      "Seq is: AAACAGATCTAAAATACAT\n",
      "0 AAACAGATCTAAAATACAT motifs detected in file\n",
      "\n",
      "Seq is: ATGCCCAAGAAGCTTACAC\n",
      "0 ATGCCCAAGAAGCTTACAC motifs detected in file\n",
      "\n",
      "Seq is: ACCATCCCACCCCCCGTAG\n",
      "0 ACCATCCCACCCCCCGTAG motifs detected in file\n",
      "\n",
      "Seq is: GTATATCAGTGAGGTCCAA\n",
      "0 GTATATCAGTGAGGTCCAA motifs detected in file\n",
      "\n",
      "Seq is: ATACCTCCCAAGAAGATGA\n",
      "0 ATACCTCCCAAGAAGATGA motifs detected in file\n",
      "\n",
      "Seq is: TAATCCTGAAAAGCGAAGC\n",
      "0 TAATCCTGAAAAGCGAAGC motifs detected in file\n",
      "\n",
      "Seq is: TTCGCTTCCGAAGAGTTTA\n",
      "0 TTCGCTTCCGAAGAGTTTA motifs detected in file\n",
      "\n",
      "Seq is: GTACTTTTAATAATTATAA\n",
      "0 GTACTTTTAATAATTATAA motifs detected in file\n",
      "\n",
      "Seq is: TGACGACAATCTCTTTACT\n",
      "0 TGACGACAATCTCTTTACT motifs detected in file\n",
      "\n",
      "Seq is: TACTCGGATGTCTACTATC\n",
      "0 TACTCGGATGTCTACTATC motifs detected in file\n",
      "\n",
      "Seq is: CTACAAAGTACACCACATA\n",
      "0 CTACAAAGTACACCACATA motifs detected in file\n",
      "\n",
      "Seq is: CGTAGCCCCATCAGGCTCA\n",
      "0 CGTAGCCCCATCAGGCTCA motifs detected in file\n",
      "\n",
      "Seq is: TTGCAGCCCCGTAAGGCCT\n",
      "0 TTGCAGCCCCGTAAGGCCT motifs detected in file\n",
      "\n",
      "Seq is: ATCCGGCTCTTTCACAACA\n",
      "0 ATCCGGCTCTTTCACAACA motifs detected in file\n",
      "\n",
      "Seq is: CCCTTCTTTCAATCTAAAT\n",
      "0 CCCTTCTTTCAATCTAAAT motifs detected in file\n",
      "\n",
      "Seq is: GCGGCTACTTATACTATCA\n",
      "0 GCGGCTACTTATACTATCA motifs detected in file\n",
      "\n",
      "Seq is: CTTTACCTAAAACCGCATC\n",
      "0 CTTTACCTAAAACCGCATC motifs detected in file\n",
      "\n",
      "Seq is: CAAACCAGATCCCACATCG\n",
      "0 CAAACCAGATCCCACATCG motifs detected in file\n",
      "\n",
      "Seq is: GACTCTTATCCCCTTTAGT\n",
      "0 GACTCTTATCCCCTTTAGT motifs detected in file\n",
      "\n",
      "Seq is: CACTTACTTCGGAGGATAC\n",
      "0 CACTTACTTCGGAGGATAC motifs detected in file\n",
      "\n",
      "Seq is: TACCGTTTATGTCGAGGAT\n",
      "0 TACCGTTTATGTCGAGGAT motifs detected in file\n",
      "\n",
      "Seq is: AACTATCCTGTATCACCTT\n",
      "0 AACTATCCTGTATCACCTT motifs detected in file\n",
      "\n",
      "Seq is: CACCCGATGTTGCATCATG\n",
      "0 CACCCGATGTTGCATCATG motifs detected in file\n",
      "\n",
      "Seq is: CACAGCACATCATGCTACA\n",
      "0 CACAGCACATCATGCTACA motifs detected in file\n",
      "\n",
      "Seq is: GATCACTACTCAAACGATT\n",
      "0 GATCACTACTCAAACGATT motifs detected in file\n",
      "\n",
      "Seq is: ATGTTACGGTCAGTCCGGT\n",
      "0 ATGTTACGGTCAGTCCGGT motifs detected in file\n",
      "\n",
      "Seq is: GGATGCCACTTTTCTTTCT\n",
      "0 GGATGCCACTTTTCTTTCT motifs detected in file\n",
      "\n",
      "Seq is: ACTTAGGATCCCGAGTCTC\n",
      "0 ACTTAGGATCCCGAGTCTC motifs detected in file\n",
      "\n",
      "Seq is: GTGACGTCGTCTAGTAAAG\n",
      "0 GTGACGTCGTCTAGTAAAG motifs detected in file\n",
      "\n",
      "Seq is: TATAACGAAGGCACCTCAC\n",
      "0 TATAACGAAGGCACCTCAC motifs detected in file\n",
      "\n",
      "Seq is: ACCGCTCAGTCGATTTATG\n",
      "0 ACCGCTCAGTCGATTTATG motifs detected in file\n",
      "\n",
      "Seq is: AAACTGCGGCCACCCCTAT\n",
      "0 AAACTGCGGCCACCCCTAT motifs detected in file\n",
      "\n",
      "Seq is: CGCTACTAATACCATCGCC\n",
      "0 CGCTACTAATACCATCGCC motifs detected in file\n",
      "\n",
      "Seq is: TCCACTTTATACGAGCACA\n",
      "0 TCCACTTTATACGAGCACA motifs detected in file\n",
      "\n",
      "Seq is: CAGATGCAGATAAGGATGA\n",
      "0 CAGATGCAGATAAGGATGA motifs detected in file\n",
      "\n",
      "Seq is: CATTTATATACCACACGAG\n",
      "0 CATTTATATACCACACGAG motifs detected in file\n",
      "\n",
      "Seq is: TGTGCTATTTGGGATCCTT\n",
      "0 TGTGCTATTTGGGATCCTT motifs detected in file\n",
      "\n",
      "Seq is: CGGTTAACTATAGTATCGA\n",
      "0 CGGTTAACTATAGTATCGA motifs detected in file\n",
      "\n",
      "Seq is: GTCTGGTATGGATACATAG\n",
      "0 GTCTGGTATGGATACATAG motifs detected in file\n",
      "\n",
      "Seq is: GTTTACCAAGAAAAAAAGG\n",
      "0 GTTTACCAAGAAAAAAAGG motifs detected in file\n",
      "\n",
      "Seq is: CCTCATCATTCAATGTTAT\n",
      "0 CCTCATCATTCAATGTTAT motifs detected in file\n",
      "\n",
      "Seq is: ACCCTCTAATAAGGCTTCG\n",
      "0 ACCCTCTAATAAGGCTTCG motifs detected in file\n",
      "\n",
      "Seq is: GACCATCCTATTCTTATAT\n",
      "0 GACCATCCTATTCTTATAT motifs detected in file\n",
      "\n",
      "Seq is: TTGAAGTCCCACTGGCTTT\n",
      "0 TTGAAGTCCCACTGGCTTT motifs detected in file\n",
      "\n",
      "Seq is: TTAGTCTTATCCACAACCA\n",
      "0 TTAGTCTTATCCACAACCA motifs detected in file\n",
      "\n",
      "Seq is: TATCTTACCCCAGAGGAGG\n",
      "0 TATCTTACCCCAGAGGAGG motifs detected in file\n",
      "\n",
      "Seq is: AGGCCGCCCCAGCTTCTTC\n",
      "0 AGGCCGCCCCAGCTTCTTC motifs detected in file\n",
      "\n",
      "Seq is: CACCACAACTCCAACGCCA\n",
      "0 CACCACAACTCCAACGCCA motifs detected in file\n",
      "\n",
      "Seq is: GACAATCATCATATCACTA\n",
      "0 GACAATCATCATATCACTA motifs detected in file\n",
      "\n",
      "Seq is: CGGTCGTCGATCCTGACCC\n",
      "0 CGGTCGTCGATCCTGACCC motifs detected in file\n",
      "\n",
      "Seq is: TCTCTATCCTCTTCATCCT\n",
      "0 TCTCTATCCTCTTCATCCT motifs detected in file\n",
      "\n",
      "Seq is: GACGACACTAATCCTGCCT\n",
      "0 GACGACACTAATCCTGCCT motifs detected in file\n",
      "\n",
      "Seq is: AGTCTGCTTCTCCCCGCAA\n",
      "0 AGTCTGCTTCTCCCCGCAA motifs detected in file\n",
      "\n",
      "Seq is: ACCATAACCCAATACCGTC\n",
      "0 ACCATAACCCAATACCGTC motifs detected in file\n",
      "\n",
      "Seq is: CCCCAAAATATAACTATTA\n",
      "0 CCCCAAAATATAACTATTA motifs detected in file\n",
      "\n",
      "Seq is: ACAACACTACTTTAACTAC\n",
      "0 ACAACACTACTTTAACTAC motifs detected in file\n",
      "\n",
      "Seq is: CGGGGATTCTATCTCCTCT\n",
      "0 CGGGGATTCTATCTCCTCT motifs detected in file\n",
      "\n",
      "Seq is: GTGGACGATCCACATTCCT\n",
      "0 GTGGACGATCCACATTCCT motifs detected in file\n",
      "\n",
      "Seq is: CTTCTACCAATCCAGATGC\n",
      "0 CTTCTACCAATCCAGATGC motifs detected in file\n",
      "\n",
      "Seq is: CTCCGAGGTCCCACCCTCA\n",
      "0 CTCCGAGGTCCCACCCTCA motifs detected in file\n",
      "\n",
      "Seq is: TCAAGGGACGATTCCCTCC\n",
      "0 TCAAGGGACGATTCCCTCC motifs detected in file\n",
      "\n",
      "Seq is: CATCTGACAAGTTGGACAA\n",
      "0 CATCTGACAAGTTGGACAA motifs detected in file\n",
      "\n",
      "Seq is: GGACGAGGCCGGAGGTGAT\n",
      "0 GGACGAGGCCGGAGGTGAT motifs detected in file\n",
      "\n",
      "Seq is: ATCGTCTACGCTCGTCCTC\n",
      "0 ATCGTCTACGCTCGTCCTC motifs detected in file\n",
      "\n",
      "Seq is: ATCCTCTCTCCCTCCATTC\n",
      "0 ATCCTCTCTCCCTCCATTC motifs detected in file\n",
      "\n",
      "Seq is: TCAGTCTTCGAATACAACA\n",
      "0 TCAGTCTTCGAATACAACA motifs detected in file\n",
      "\n",
      "Seq is: AATACGCCCCTTTGCGGTA\n",
      "0 AATACGCCCCTTTGCGGTA motifs detected in file\n",
      "\n",
      "Seq is: TAGCCCCCGTGGCTAATAA\n",
      "0 TAGCCCCCGTGGCTAATAA motifs detected in file\n",
      "\n",
      "Seq is: TCCCCTTGATCAGTCAACG\n",
      "0 TCCCCTTGATCAGTCAACG motifs detected in file\n",
      "\n",
      "Seq is: GTTTCGGAGGCTAATACTA\n",
      "0 GTTTCGGAGGCTAATACTA motifs detected in file\n",
      "\n",
      "Seq is: CCCATAATGATACTTCTTC\n",
      "0 CCCATAATGATACTTCTTC motifs detected in file\n",
      "\n",
      "Seq is: TAATAATGTTTACGTACCC\n",
      "0 TAATAATGTTTACGTACCC motifs detected in file\n",
      "\n",
      "Seq is: GACACTGCTATTGCAACAT\n",
      "0 GACACTGCTATTGCAACAT motifs detected in file\n",
      "\n",
      "Seq is: CTACACCAGCAATGGATCT\n",
      "0 CTACACCAGCAATGGATCT motifs detected in file\n",
      "\n",
      "Seq is: TCCAACGGACCGACCGGGT\n",
      "0 TCCAACGGACCGACCGGGT motifs detected in file\n",
      "\n",
      "Seq is: CGAGCCGAGCTTATTCCTC\n",
      "0 CGAGCCGAGCTTATTCCTC motifs detected in file\n",
      "\n",
      "Seq is: CGAATCTCGACACGGATCC\n",
      "0 CGAATCTCGACACGGATCC motifs detected in file\n",
      "\n",
      "Seq is: TGAGGTCGAGTACGCGGCT\n",
      "0 TGAGGTCGAGTACGCGGCT motifs detected in file\n",
      "\n",
      "Seq is: TATTATCCATATCACAAGG\n",
      "0 TATTATCCATATCACAAGG motifs detected in file\n",
      "\n",
      "Seq is: TTACAGAAACACCAAACAT\n",
      "0 TTACAGAAACACCAAACAT motifs detected in file\n",
      "\n",
      "Seq is: CTCTTATCAGTTGCCAGCC\n",
      "0 CTCTTATCAGTTGCCAGCC motifs detected in file\n",
      "\n",
      "Seq is: GCTTGTAGTCACCCCCACT\n",
      "0 GCTTGTAGTCACCCCCACT motifs detected in file\n",
      "\n",
      "Seq is: CCATTTTACCGACTCACTT\n",
      "0 CCATTTTACCGACTCACTT motifs detected in file\n",
      "\n",
      "Seq is: CGTAACCTGACATTTAGAT\n",
      "0 CGTAACCTGACATTTAGAT motifs detected in file\n",
      "\n",
      "Seq is: TTCTGTCCCCAATCCGGAG\n",
      "0 TTCTGTCCCCAATCCGGAG motifs detected in file\n",
      "\n",
      "Seq is: AAAAATGGTCGAGGCTCCA\n",
      "0 AAAAATGGTCGAGGCTCCA motifs detected in file\n",
      "\n",
      "Seq is: CTAAAAGTATAACTTAACG\n",
      "0 CTAAAAGTATAACTTAACG motifs detected in file\n",
      "\n",
      "Seq is: TTTAAGCTTCTTCGTCGAA\n",
      "0 TTTAAGCTTCTTCGTCGAA motifs detected in file\n",
      "\n",
      "Seq is: GTTTGGACGGCCCCGAAGA\n",
      "0 GTTTGGACGGCCCCGAAGA motifs detected in file\n",
      "\n",
      "Seq is: GGGCGGAAAAAAGGGCCGC\n",
      "0 GGGCGGAAAAAAGGGCCGC motifs detected in file\n",
      "\n",
      "Seq is: CGCCCTCTTCATCTAACTT\n",
      "0 CGCCCTCTTCATCTAACTT motifs detected in file\n",
      "\n",
      "Seq is: CGGTCAACTAATCCCACGA\n",
      "0 CGGTCAACTAATCCCACGA motifs detected in file\n",
      "\n",
      "Seq is: ATCGACAATTGATTCACAA\n",
      "0 ATCGACAATTGATTCACAA motifs detected in file\n",
      "\n",
      "Seq is: ACACCCAAATTCAGGGTAA\n",
      "0 ACACCCAAATTCAGGGTAA motifs detected in file\n",
      "\n",
      "Seq is: CCAGATCATTCCCGAATCG\n",
      "0 CCAGATCATTCCCGAATCG motifs detected in file\n",
      "\n",
      "Seq is: AATTAATTTCACCGACTAA\n",
      "0 AATTAATTTCACCGACTAA motifs detected in file\n",
      "\n",
      "Seq is: ACGCAAGTCAACTACGTCT\n",
      "0 ACGCAAGTCAACTACGTCT motifs detected in file\n",
      "\n",
      "Seq is: CACCCCAAAACGTCAGGAA\n",
      "0 CACCCCAAAACGTCAGGAA motifs detected in file\n",
      "\n",
      "Seq is: TCGACAATGTCTTTAATTC\n",
      "0 TCGACAATGTCTTTAATTC motifs detected in file\n",
      "\n",
      "Seq is: ATAACGTTGAATGACTCCC\n",
      "0 ATAACGTTGAATGACTCCC motifs detected in file\n",
      "\n",
      "Seq is: GAAACTTCCGAGAACCAGA\n",
      "0 GAAACTTCCGAGAACCAGA motifs detected in file\n",
      "\n",
      "Seq is: CATAAATTGGATTTAAAGA\n",
      "0 CATAAATTGGATTTAAAGA motifs detected in file\n",
      "\n",
      "Seq is: TATTCTAATAATCATATTT\n",
      "0 TATTCTAATAATCATATTT motifs detected in file\n",
      "\n",
      "Seq is: TCCCCTCTATCCATCCTCA\n",
      "0 TCCCCTCTATCCATCCTCA motifs detected in file\n",
      "\n",
      "Seq is: TCGCACCATTCCCGCTACT\n",
      "0 TCGCACCATTCCCGCTACT motifs detected in file\n",
      "\n",
      "Seq is: CACACCCCTCCTTACCCCA\n",
      "0 CACACCCCTCCTTACCCCA motifs detected in file\n",
      "\n",
      "Seq is: CCCAAAACATACAAGTTTG\n",
      "0 CCCAAAACATACAAGTTTG motifs detected in file\n",
      "\n",
      "Seq is: ACAGTAAAATAAAAATGCA\n",
      "0 ACAGTAAAATAAAAATGCA motifs detected in file\n",
      "\n",
      "Seq is: ACAATCTATACCCCTCATC\n",
      "0 ACAATCTATACCCCTCATC motifs detected in file\n",
      "\n",
      "Seq is: ACACTAACTCCACCTCATC\n",
      "0 ACACTAACTCCACCTCATC motifs detected in file\n",
      "\n",
      "Seq is: TAATCCGCATCCATCTTCA\n",
      "0 TAATCCGCATCCATCTTCA motifs detected in file\n",
      "\n",
      "Seq is: TCTCCAATTCCTCCCACTA\n",
      "0 TCTCCAATTCCTCCCACTA motifs detected in file\n",
      "\n",
      "Seq is: CCACCGATACTACCACCCC\n",
      "0 CCACCGATACTACCACCCC motifs detected in file\n",
      "\n",
      "Seq is: TACTACTCCGATAACAAAA\n",
      "0 TACTACTCCGATAACAAAA motifs detected in file\n",
      "\n",
      "Seq is: AACACTTAAGAAGCTATTA\n",
      "0 AACACTTAAGAAGCTATTA motifs detected in file\n",
      "\n",
      "Seq is: CCGGGTAAACCCGTTTTTC\n",
      "0 CCGGGTAAACCCGTTTTTC motifs detected in file\n",
      "\n",
      "Seq is: GGCCAATCGCCCCCGTCCG\n",
      "0 GGCCAATCGCCCCCGTCCG motifs detected in file\n",
      "\n",
      "Seq is: GAGGATCCCTCTCCTCCCA\n",
      "0 GAGGATCCCTCTCCTCCCA motifs detected in file\n",
      "\n",
      "Seq is: CCTACCTTAATTCCCACAA\n",
      "0 CCTACCTTAATTCCCACAA motifs detected in file\n",
      "\n",
      "Seq is: TCAGTACAATCGAACAAAG\n",
      "0 TCAGTACAATCGAACAAAG motifs detected in file\n",
      "\n",
      "Seq is: TCCACGCTCTATCATCATC\n",
      "0 TCCACGCTCTATCATCATC motifs detected in file\n",
      "\n",
      "Seq is: CCAGCACCACGACCTCAAA\n",
      "0 CCAGCACCACGACCTCAAA motifs detected in file\n",
      "\n",
      "Seq is: TTCAACTCATCATCCTTAC\n",
      "0 TTCAACTCATCATCCTTAC motifs detected in file\n",
      "\n",
      "Seq is: GCCATCATCAATCCTATTA\n",
      "0 GCCATCATCAATCCTATTA motifs detected in file\n",
      "\n",
      "Seq is: TATTTATCAATTTAATTCT\n",
      "0 TATTTATCAATTTAATTCT motifs detected in file\n",
      "\n",
      "Seq is: TACCAATACAATCCCAACA\n",
      "0 TACCAATACAATCCCAACA motifs detected in file\n",
      "\n",
      "Seq is: TGCCATCTTGACGATAATA\n",
      "0 TGCCATCTTGACGATAATA motifs detected in file\n",
      "\n",
      "Seq is: AGTAGGATACACCCATTAA\n",
      "0 AGTAGGATACACCCATTAA motifs detected in file\n",
      "\n",
      "Seq is: CTCCTCATACGATTCTAAA\n",
      "0 CTCCTCATACGATTCTAAA motifs detected in file\n",
      "\n",
      "Seq is: ACGCATCGACCCAAACCAA\n",
      "0 ACGCATCGACCCAAACCAA motifs detected in file\n",
      "\n",
      "Seq is: ATTAGGTGGAGTTGACGGA\n",
      "0 ATTAGGTGGAGTTGACGGA motifs detected in file\n",
      "\n",
      "Seq is: CGATACTACCTATTCTAAC\n",
      "0 CGATACTACCTATTCTAAC motifs detected in file\n",
      "\n",
      "Seq is: TCTCTCACTCCTCTTCCGA\n",
      "0 TCTCTCACTCCTCTTCCGA motifs detected in file\n",
      "\n",
      "Seq is: ATGCAAATCACTCCCTCTC\n",
      "0 ATGCAAATCACTCCCTCTC motifs detected in file\n",
      "\n",
      "Seq is: TAAACCATATACTAACTCT\n",
      "0 TAAACCATATACTAACTCT motifs detected in file\n",
      "\n",
      "Seq is: ACCCCCGATCAAAAACAGT\n",
      "0 ACCCCCGATCAAAAACAGT motifs detected in file\n",
      "\n",
      "Seq is: ACACTCTTCTTCGTCCGGC\n",
      "0 ACACTCTTCTTCGTCCGGC motifs detected in file\n",
      "\n",
      "Seq is: CTACAGTCTCCCCACGGAA\n",
      "0 CTACAGTCTCCCCACGGAA motifs detected in file\n",
      "\n",
      "Seq is: CCCATTGGAGACCCTGAGT\n",
      "0 CCCATTGGAGACCCTGAGT motifs detected in file\n",
      "\n",
      "Seq is: CTTCACTTTCCCCCGATAA\n",
      "0 CTTCACTTTCCCCCGATAA motifs detected in file\n",
      "\n",
      "Seq is: GGATCAAAATAACGATATC\n",
      "0 GGATCAAAATAACGATATC motifs detected in file\n",
      "\n",
      "Seq is: GATAATACTAATAATTACT\n",
      "0 GATAATACTAATAATTACT motifs detected in file\n",
      "\n",
      "Seq is: ACTCATAACTAACCATCAT\n",
      "0 ACTCATAACTAACCATCAT motifs detected in file\n",
      "\n",
      "Seq is: AACCAATACCAAGTAACAG\n",
      "0 AACCAATACCAAGTAACAG motifs detected in file\n",
      "\n",
      "Seq is: GCCTCTCATATAACAACTT\n",
      "0 GCCTCTCATATAACAACTT motifs detected in file\n",
      "\n",
      "Seq is: CTCCTATCGATAATCTTCC\n",
      "0 CTCCTATCGATAATCTTCC motifs detected in file\n",
      "\n",
      "Seq is: TAATACCTACGCCAACGAA\n",
      "0 TAATACCTACGCCAACGAA motifs detected in file\n",
      "\n",
      "Seq is: CGCACTCCTTTATGAACTA\n",
      "0 CGCACTCCTTTATGAACTA motifs detected in file\n",
      "\n",
      "Seq is: CCGTCGAAGACACCTTGCT\n",
      "0 CCGTCGAAGACACCTTGCT motifs detected in file\n",
      "\n",
      "Seq is: CCCAAATAAAAAAACCAAT\n",
      "0 CCCAAATAAAAAAACCAAT motifs detected in file\n",
      "\n",
      "Seq is: CTTGACCTTATTTTCGATC\n",
      "0 CTTGACCTTATTTTCGATC motifs detected in file\n",
      "\n",
      "Seq is: GTACAAATAAAGATCCGGA\n",
      "0 GTACAAATAAAGATCCGGA motifs detected in file\n",
      "\n",
      "Seq is: TGAGTCCATTTTTTAGTCA\n",
      "0 TGAGTCCATTTTTTAGTCA motifs detected in file\n",
      "\n",
      "Seq is: CGCTCGAATCGCGACACTA\n",
      "0 CGCTCGAATCGCGACACTA motifs detected in file\n",
      "\n",
      "Seq is: CTCACACGGACGTTTCTAC\n",
      "0 CTCACACGGACGTTTCTAC motifs detected in file\n",
      "\n",
      "Seq is: CATCTCATCTACTGCCCAA\n",
      "0 CATCTCATCTACTGCCCAA motifs detected in file\n",
      "\n",
      "Seq is: CCCGGTCCCCTAATTAATC\n",
      "0 CCCGGTCCCCTAATTAATC motifs detected in file\n",
      "\n",
      "Seq is: ATGCCCTTCCCATATTGGT\n",
      "0 ATGCCCTTCCCATATTGGT motifs detected in file\n",
      "\n",
      "Seq is: TGTAAAAGCCCCATACCCG\n",
      "0 TGTAAAAGCCCCATACCCG motifs detected in file\n",
      "\n",
      "Seq is: GGCTATCGAATAAATCGAC\n",
      "0 GGCTATCGAATAAATCGAC motifs detected in file\n",
      "\n",
      "Seq is: TGGAATGAAATCCTACCCC\n",
      "0 TGGAATGAAATCCTACCCC motifs detected in file\n",
      "\n",
      "Seq is: ACACTATCCACCGTGCCTC\n",
      "0 ACACTATCCACCGTGCCTC motifs detected in file\n",
      "\n",
      "Seq is: TTAAAACCTAAGAGTCCCT\n",
      "0 TTAAAACCTAAGAGTCCCT motifs detected in file\n",
      "\n",
      "Seq is: ACCCAAGCTAAGAGTATCA\n",
      "0 ACCCAAGCTAAGAGTATCA motifs detected in file\n",
      "\n",
      "Seq is: GGATCTTTATTCCCCCAAA\n",
      "0 GGATCTTTATTCCCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: TTCGAGGATAATAAATGAG\n",
      "0 TTCGAGGATAATAAATGAG motifs detected in file\n",
      "\n",
      "Seq is: ATAGTTTCATTGAGAAAAT\n",
      "0 ATAGTTTCATTGAGAAAAT motifs detected in file\n",
      "\n",
      "Seq is: AGTCTGTATAAAGAATCCA\n",
      "0 AGTCTGTATAAAGAATCCA motifs detected in file\n",
      "\n",
      "Seq is: AACTCCCCCTTACGACCTC\n",
      "0 AACTCCCCCTTACGACCTC motifs detected in file\n",
      "\n",
      "Seq is: TAACATTACCCATACCTCT\n",
      "0 TAACATTACCCATACCTCT motifs detected in file\n",
      "\n",
      "Seq is: GTATAGTATATTCATTACG\n",
      "0 GTATAGTATATTCATTACG motifs detected in file\n",
      "\n",
      "Seq is: ATCCCACTCACCATCCTTC\n",
      "0 ATCCCACTCACCATCCTTC motifs detected in file\n",
      "\n",
      "Seq is: AAAAAAGTATCCTCCACAT\n",
      "0 AAAAAAGTATCCTCCACAT motifs detected in file\n",
      "\n",
      "Seq is: ACTCAACCAGCATCGCCTT\n",
      "0 ACTCAACCAGCATCGCCTT motifs detected in file\n",
      "\n",
      "Seq is: AGCCCCCATACGACAAGCT\n",
      "0 AGCCCCCATACGACAAGCT motifs detected in file\n",
      "\n",
      "Seq is: TAAGTATTCTTGTCCCTCC\n",
      "0 TAAGTATTCTTGTCCCTCC motifs detected in file\n",
      "\n",
      "Seq is: AATCTTCATCCCAGAACCA\n",
      "0 AATCTTCATCCCAGAACCA motifs detected in file\n",
      "\n",
      "Seq is: CTGTTTTATACAACACATC\n",
      "0 CTGTTTTATACAACACATC motifs detected in file\n",
      "\n",
      "Seq is: TCAAGTCCCCTCTCACGCA\n",
      "0 TCAAGTCCCCTCTCACGCA motifs detected in file\n",
      "\n",
      "Seq is: GTATACAACAAGGATCCTT\n",
      "0 GTATACAACAAGGATCCTT motifs detected in file\n",
      "\n",
      "Seq is: CTAACATCACCACTCCCAC\n",
      "0 CTAACATCACCACTCCCAC motifs detected in file\n",
      "\n",
      "Seq is: AAATAATATTATTACAAAC\n",
      "0 AAATAATATTATTACAAAC motifs detected in file\n",
      "\n",
      "Seq is: ACATAAGCCGATACTTCTT\n",
      "0 ACATAAGCCGATACTTCTT motifs detected in file\n",
      "\n",
      "Seq is: ATCCCGCTTCCCCGGACGC\n",
      "0 ATCCCGCTTCCCCGGACGC motifs detected in file\n",
      "\n",
      "Seq is: CGCATAAGCTACAACTTCG\n",
      "0 CGCATAAGCTACAACTTCG motifs detected in file\n",
      "\n",
      "Seq is: GACTCTGATCAAGCCTGAG\n",
      "0 GACTCTGATCAAGCCTGAG motifs detected in file\n",
      "\n",
      "Seq is: GGGAAGCCGTTCCAGCTTC\n",
      "0 GGGAAGCCGTTCCAGCTTC motifs detected in file\n",
      "\n",
      "Seq is: CCCCAAGCCAACCAGAGAC\n",
      "0 CCCCAAGCCAACCAGAGAC motifs detected in file\n",
      "\n",
      "Seq is: GATCACACCTCTATTTAGT\n",
      "0 GATCACACCTCTATTTAGT motifs detected in file\n",
      "\n",
      "Seq is: ATAATACCGGTTCCCAGTA\n",
      "0 ATAATACCGGTTCCCAGTA motifs detected in file\n",
      "\n",
      "Seq is: CTACCGTCCTCATTAGTCT\n",
      "0 CTACCGTCCTCATTAGTCT motifs detected in file\n",
      "\n",
      "Seq is: CCACAAGAACACAACACTA\n",
      "0 CCACAAGAACACAACACTA motifs detected in file\n",
      "\n",
      "Seq is: TTCCCACCTCTCCAATTTC\n",
      "0 TTCCCACCTCTCCAATTTC motifs detected in file\n",
      "\n",
      "Seq is: CTCGGTGAATAATCATTAC\n",
      "0 CTCGGTGAATAATCATTAC motifs detected in file\n",
      "\n",
      "Seq is: AACTATCATCTTACTACCG\n",
      "0 AACTATCATCTTACTACCG motifs detected in file\n",
      "\n",
      "Seq is: ATCCCACTGAAGTATACTC\n",
      "0 ATCCCACTGAAGTATACTC motifs detected in file\n",
      "\n",
      "Seq is: TAACAAACCCGATGACGAG\n",
      "0 TAACAAACCCGATGACGAG motifs detected in file\n",
      "\n",
      "Seq is: CGTCACGCGGCTAGTCCCG\n",
      "0 CGTCACGCGGCTAGTCCCG motifs detected in file\n",
      "\n",
      "Seq is: CATCAAACTCAAACTACGA\n",
      "0 CATCAAACTCAAACTACGA motifs detected in file\n",
      "\n",
      "Seq is: GTGGGACTAGTCTCCTAAC\n",
      "0 GTGGGACTAGTCTCCTAAC motifs detected in file\n",
      "\n",
      "Seq is: TCATTTGCCGATCCGATCT\n",
      "0 TCATTTGCCGATCCGATCT motifs detected in file\n",
      "\n",
      "Seq is: CCACCGATCTTATTTATCC\n",
      "0 CCACCGATCTTATTTATCC motifs detected in file\n",
      "\n",
      "Seq is: TCCGGATCCAACTCCAACT\n",
      "0 TCCGGATCCAACTCCAACT motifs detected in file\n",
      "\n",
      "Seq is: GGTCCCCCAACCCATACCC\n",
      "0 GGTCCCCCAACCCATACCC motifs detected in file\n",
      "\n",
      "Seq is: CTCCCCCCAAGTATCATCT\n",
      "0 CTCCCCCCAAGTATCATCT motifs detected in file\n",
      "\n",
      "Seq is: TCTCGCTACCACTCTCGAT\n",
      "0 TCTCGCTACCACTCTCGAT motifs detected in file\n",
      "\n",
      "Seq is: TCCAGCCCCGCCACTACAT\n",
      "0 TCCAGCCCCGCCACTACAT motifs detected in file\n",
      "\n",
      "Seq is: CTCCCACTACCATCTACAC\n",
      "0 CTCCCACTACCATCTACAC motifs detected in file\n",
      "\n",
      "Seq is: CGCCCAAAATCCCCGAGAA\n",
      "0 CGCCCAAAATCCCCGAGAA motifs detected in file\n",
      "\n",
      "Seq is: ACCACTTCTCAAAATACCG\n",
      "0 ACCACTTCTCAAAATACCG motifs detected in file\n",
      "\n",
      "Seq is: CAGTCGCTTCCCAACATCA\n",
      "0 CAGTCGCTTCCCAACATCA motifs detected in file\n",
      "\n",
      "Seq is: TCGGGCATCCCCGGATGTT\n",
      "0 TCGGGCATCCCCGGATGTT motifs detected in file\n",
      "\n",
      "Seq is: GCAACCCCGGAAACGCATC\n",
      "0 GCAACCCCGGAAACGCATC motifs detected in file\n",
      "\n",
      "Seq is: AACATATATCGGATCTTAA\n",
      "0 AACATATATCGGATCTTAA motifs detected in file\n",
      "\n",
      "Seq is: AAAGCAAGCCATTCGTAAT\n",
      "0 AAAGCAAGCCATTCGTAAT motifs detected in file\n",
      "\n",
      "Seq is: CCTTACGGTAACGCTAATC\n",
      "0 CCTTACGGTAACGCTAATC motifs detected in file\n",
      "\n",
      "Seq is: TTACCCATGTTACTCCTCA\n",
      "0 TTACCCATGTTACTCCTCA motifs detected in file\n",
      "\n",
      "Seq is: TCCTCCAACCGGTACCCAT\n",
      "0 TCCTCCAACCGGTACCCAT motifs detected in file\n",
      "\n",
      "Seq is: ACAACAATTCTTCTCCTTA\n",
      "0 ACAACAATTCTTCTCCTTA motifs detected in file\n",
      "\n",
      "Seq is: ACTTGGAGACTGACATTTC\n",
      "0 ACTTGGAGACTGACATTTC motifs detected in file\n",
      "\n",
      "Seq is: AAAATTCAAAATACGCTAA\n",
      "0 AAAATTCAAAATACGCTAA motifs detected in file\n",
      "\n",
      "Seq is: TGGCCCGAGACGGTAGAAT\n",
      "0 TGGCCCGAGACGGTAGAAT motifs detected in file\n",
      "\n",
      "Seq is: TGTTTGGGACAAGAACCCA\n",
      "0 TGTTTGGGACAAGAACCCA motifs detected in file\n",
      "\n",
      "Seq is: CCCACACCCATATTATGAT\n",
      "0 CCCACACCCATATTATGAT motifs detected in file\n",
      "\n",
      "Seq is: TCAACTCTACTATAGTAAA\n",
      "0 TCAACTCTACTATAGTAAA motifs detected in file\n",
      "\n",
      "Seq is: TGCCCCCTTCCGCGAAACA\n",
      "0 TGCCCCCTTCCGCGAAACA motifs detected in file\n",
      "\n",
      "Seq is: CTTCATCCGGAATAAAGAG\n",
      "0 CTTCATCCGGAATAAAGAG motifs detected in file\n",
      "\n",
      "Seq is: AACAGGAAAGCATGTCCCT\n",
      "0 AACAGGAAAGCATGTCCCT motifs detected in file\n",
      "\n",
      "Seq is: CCTTAAACTTNCATCTATC\n",
      "0 CCTTAAACTTNCATCTATC motifs detected in file\n",
      "\n",
      "Seq is: TTTGGCTGGACCTAATGAG\n",
      "0 TTTGGCTGGACCTAATGAG motifs detected in file\n",
      "\n",
      "Seq is: GCCAGACTTGAGTCTAGTG\n",
      "0 GCCAGACTTGAGTCTAGTG motifs detected in file\n",
      "\n",
      "Seq is: CATCCTGAAATTAGCAACT\n",
      "0 CATCCTGAAATTAGCAACT motifs detected in file\n",
      "\n",
      "Seq is: TGTTTGCTTGGAAATTATC\n",
      "0 TGTTTGCTTGGAAATTATC motifs detected in file\n",
      "\n",
      "Seq is: GCCGACGTGGTAGCCCTAC\n",
      "0 GCCGACGTGGTAGCCCTAC motifs detected in file\n",
      "\n",
      "Seq is: AGGACTAGGTTGTAGCTCC\n",
      "0 AGGACTAGGTTGTAGCTCC motifs detected in file\n",
      "\n",
      "Seq is: AGCATTTGGGATAACAACT\n",
      "0 AGCATTTGGGATAACAACT motifs detected in file\n",
      "\n",
      "Seq is: ATACCTGAGATCTTATCCT\n",
      "0 ATACCTGAGATCTTATCCT motifs detected in file\n",
      "\n",
      "Seq is: AACGCGACAATAGGGATCC\n",
      "0 AACGCGACAATAGGGATCC motifs detected in file\n",
      "\n",
      "Seq is: CATTGAACAAGGCAACCAG\n",
      "0 CATTGAACAAGGCAACCAG motifs detected in file\n",
      "\n",
      "Seq is: TTCAATAACCTAGTTAACT\n",
      "0 TTCAATAACCTAGTTAACT motifs detected in file\n",
      "\n",
      "Seq is: CATATCATCAAGCGAAACT\n",
      "0 CATATCATCAAGCGAAACT motifs detected in file\n",
      "\n",
      "Seq is: GACCACTTCAGAATCGTAC\n",
      "0 GACCACTTCAGAATCGTAC motifs detected in file\n",
      "\n",
      "Seq is: ATGACGAGCCTCCAACCCA\n",
      "0 ATGACGAGCCTCCAACCCA motifs detected in file\n",
      "\n",
      "Seq is: AGACGAGGCTCCAGCGGGG\n",
      "0 AGACGAGGCTCCAGCGGGG motifs detected in file\n",
      "\n",
      "Seq is: TTGGCTTTAAAAATTACGT\n",
      "0 TTGGCTTTAAAAATTACGT motifs detected in file\n",
      "\n",
      "Seq is: CCAAACCATCAAATCCTGG\n",
      "0 CCAAACCATCAAATCCTGG motifs detected in file\n",
      "\n",
      "Seq is: ACACCCAAACAATCCATGA\n",
      "0 ACACCCAAACAATCCATGA motifs detected in file\n",
      "\n",
      "Seq is: CAAACGTAATTATTTAATT\n",
      "0 CAAACGTAATTATTTAATT motifs detected in file\n",
      "\n",
      "Seq is: TCGAGGTATCCCAGAAGAG\n",
      "0 TCGAGGTATCCCAGAAGAG motifs detected in file\n",
      "\n",
      "Seq is: CAGAACGACACAATACGGG\n",
      "0 CAGAACGACACAATACGGG motifs detected in file\n",
      "\n",
      "Seq is: CGGAGAAGTGCCCGTCCAG\n",
      "0 CGGAGAAGTGCCCGTCCAG motifs detected in file\n",
      "\n",
      "Seq is: TTAAAGTGACCAATTTTCA\n",
      "0 TTAAAGTGACCAATTTTCA motifs detected in file\n",
      "\n",
      "Seq is: TTCTCTGTCGACTTGGGAG\n",
      "0 TTCTCTGTCGACTTGGGAG motifs detected in file\n",
      "\n",
      "Seq is: CACCTCGGTAAGTATGTCC\n",
      "0 CACCTCGGTAAGTATGTCC motifs detected in file\n",
      "\n",
      "Seq is: AGGGATAAATTCCTTGTTC\n",
      "0 AGGGATAAATTCCTTGTTC motifs detected in file\n",
      "\n",
      "Seq is: ACTAATACGATGGAAACGT\n",
      "0 ACTAATACGATGGAAACGT motifs detected in file\n",
      "\n",
      "Seq is: GCCAATCCCATGGCGCCGG\n",
      "0 GCCAATCCCATGGCGCCGG motifs detected in file\n",
      "\n",
      "Seq is: CAATTTGTACACAGTGACC\n",
      "0 CAATTTGTACACAGTGACC motifs detected in file\n",
      "\n",
      "Seq is: CGTCCGCCACGGAGATTAT\n",
      "0 CGTCCGCCACGGAGATTAT motifs detected in file\n",
      "\n",
      "Seq is: GACCACTACGATCTCCACT\n",
      "0 GACCACTACGATCTCCACT motifs detected in file\n",
      "\n",
      "Seq is: ACAAAAACCATTTGTCCGC\n",
      "0 ACAAAAACCATTTGTCCGC motifs detected in file\n",
      "\n",
      "Seq is: CCCATTCTAAACGGCTCAA\n",
      "0 CCCATTCTAAACGGCTCAA motifs detected in file\n",
      "\n",
      "Seq is: GGAAAATGAAAAAAATTGG\n",
      "0 GGAAAATGAAAAAAATTGG motifs detected in file\n",
      "\n",
      "Seq is: AAAGGAATACTCGTACGGA\n",
      "0 AAAGGAATACTCGTACGGA motifs detected in file\n",
      "\n",
      "Seq is: CACAACCCAACTGTCACTC\n",
      "0 CACAACCCAACTGTCACTC motifs detected in file\n",
      "\n",
      "Seq is: CCATTATTACTGAACAACC\n",
      "0 CCATTATTACTGAACAACC motifs detected in file\n",
      "\n",
      "Seq is: AACTAACATCTATAACCCG\n",
      "0 AACTAACATCTATAACCCG motifs detected in file\n",
      "\n",
      "Seq is: ACAATTAACAGTCAAGTCA\n",
      "0 ACAATTAACAGTCAAGTCA motifs detected in file\n",
      "\n",
      "Seq is: CAAAATTAGACTGCGTCCG\n",
      "0 CAAAATTAGACTGCGTCCG motifs detected in file\n",
      "\n",
      "Seq is: AATACGCCTCCTCTTACAA\n",
      "0 AATACGCCTCCTCTTACAA motifs detected in file\n",
      "\n",
      "Seq is: AAGTACAATGAATATGATT\n",
      "0 AAGTACAATGAATATGATT motifs detected in file\n",
      "\n",
      "Seq is: GTAATCAAGAAGATATCCC\n",
      "0 GTAATCAAGAAGATATCCC motifs detected in file\n",
      "\n",
      "Seq is: ACTATCTAACCAGGTTAAC\n",
      "0 ACTATCTAACCAGGTTAAC motifs detected in file\n",
      "\n",
      "Seq is: CCACACTCCTCAAGTCAAT\n",
      "0 CCACACTCCTCAAGTCAAT motifs detected in file\n",
      "\n",
      "Seq is: ATACAAACCCTAAAAAATC\n",
      "0 ATACAAACCCTAAAAAATC motifs detected in file\n",
      "\n",
      "Seq is: CATCACCCACAACTCGAAC\n",
      "0 CATCACCCACAACTCGAAC motifs detected in file\n",
      "\n",
      "Seq is: TTGCGAAAGAATTAACCAC\n",
      "0 TTGCGAAAGAATTAACCAC motifs detected in file\n",
      "\n",
      "Seq is: CGACGAAAATCCGGATGAT\n",
      "0 CGACGAAAATCCGGATGAT motifs detected in file\n",
      "\n",
      "Seq is: ACCCACAATTTAAAAAATG\n",
      "0 ACCCACAATTTAAAAAATG motifs detected in file\n",
      "\n",
      "Seq is: AGAGAGATGTTCCAAAAAA\n",
      "0 AGAGAGATGTTCCAAAAAA motifs detected in file\n",
      "\n",
      "Seq is: GGATCACAGGTTTCTCGAC\n",
      "0 GGATCACAGGTTTCTCGAC motifs detected in file\n",
      "\n",
      "Seq is: AAGGAGAAACCTGATTGTC\n",
      "0 AAGGAGAAACCTGATTGTC motifs detected in file\n",
      "\n",
      "Seq is: AATTTAAATGTTCCCCTAA\n",
      "0 AATTTAAATGTTCCCCTAA motifs detected in file\n",
      "\n",
      "Seq is: ATCTCCCAAGACACCCGTT\n",
      "0 ATCTCCCAAGACACCCGTT motifs detected in file\n",
      "\n",
      "Seq is: TAAATTTCAACTTGATTCT\n",
      "0 TAAATTTCAACTTGATTCT motifs detected in file\n",
      "\n",
      "Seq is: AAGATAGAACCTGTTGGTC\n",
      "0 AAGATAGAACCTGTTGGTC motifs detected in file\n",
      "\n",
      "Seq is: GATAGTGGTCCGAGCCATC\n",
      "0 GATAGTGGTCCGAGCCATC motifs detected in file\n",
      "\n",
      "Seq is: CAAACAGCGGAGATGGATA\n",
      "0 CAAACAGCGGAGATGGATA motifs detected in file\n",
      "\n",
      "Seq is: TTTAGAAGGGTGATAAAAC\n",
      "0 TTTAGAAGGGTGATAAAAC motifs detected in file\n",
      "\n",
      "Seq is: GATGTATCTGCCCACACGA\n",
      "0 GATGTATCTGCCCACACGA motifs detected in file\n",
      "\n",
      "Seq is: GAAAATCGACAAGAATCCA\n",
      "0 GAAAATCGACAAGAATCCA motifs detected in file\n",
      "\n",
      "Seq is: TCGAGCAGACCAAAGCCCC\n",
      "0 TCGAGCAGACCAAAGCCCC motifs detected in file\n",
      "\n",
      "Seq is: CAGAATCGAAACCGAGAGG\n",
      "0 CAGAATCGAAACCGAGAGG motifs detected in file\n",
      "\n",
      "Seq is: AACGTTTCAATAAAGATCA\n",
      "0 AACGTTTCAATAAAGATCA motifs detected in file\n",
      "\n",
      "Seq is: ATTAAGTAATACGTCTTCC\n",
      "0 ATTAAGTAATACGTCTTCC motifs detected in file\n",
      "\n",
      "Seq is: ATATCCCCAATCAGGAACG\n",
      "0 ATATCCCCAATCAGGAACG motifs detected in file\n",
      "\n",
      "Seq is: ATATAATACGAACCAATAT\n",
      "0 ATATAATACGAACCAATAT motifs detected in file\n",
      "\n",
      "Seq is: TAAAAAGTAGAAAGGGAAC\n",
      "0 TAAAAAGTAGAAAGGGAAC motifs detected in file\n",
      "\n",
      "Seq is: GCCATGATATAGATAACGC\n",
      "0 GCCATGATATAGATAACGC motifs detected in file\n",
      "\n",
      "Seq is: GGTCCAAAGTTAAAGATAG\n",
      "0 GGTCCAAAGTTAAAGATAG motifs detected in file\n",
      "\n",
      "Seq is: CGGATATGAAATAAACCCA\n",
      "0 CGGATATGAAATAAACCCA motifs detected in file\n",
      "\n",
      "Seq is: TTTACCAAACCGATTCCAA\n",
      "0 TTTACCAAACCGATTCCAA motifs detected in file\n",
      "\n",
      "Seq is: CAGACCATCATTCCACCTC\n",
      "0 CAGACCATCATTCCACCTC motifs detected in file\n",
      "\n",
      "Seq is: ACCCAAACCCCGATCCAAA\n",
      "0 ACCCAAACCCCGATCCAAA motifs detected in file\n",
      "\n",
      "Seq is: TCGAGTCTCGCCAGTTCAA\n",
      "0 TCGAGTCTCGCCAGTTCAA motifs detected in file\n",
      "\n",
      "Seq is: TTCAACTTTAGAGGATTCA\n",
      "0 TTCAACTTTAGAGGATTCA motifs detected in file\n",
      "\n",
      "Seq is: CATTCAACCCACGAAACAC\n",
      "0 CATTCAACCCACGAAACAC motifs detected in file\n",
      "\n",
      "Seq is: AATTCGATGTGAGACCAAG\n",
      "0 AATTCGATGTGAGACCAAG motifs detected in file\n",
      "\n",
      "Seq is: CAGGTTCACGTGAAAGGTC\n",
      "0 CAGGTTCACGTGAAAGGTC motifs detected in file\n",
      "\n",
      "Seq is: ATGTGAATGGTACAATGCT\n",
      "0 ATGTGAATGGTACAATGCT motifs detected in file\n",
      "\n",
      "Seq is: GAACAGAGGAGATATATTT\n",
      "0 GAACAGAGGAGATATATTT motifs detected in file\n",
      "\n",
      "Seq is: ACGCATCCCCAAAATCAAT\n",
      "0 ACGCATCCCCAAAATCAAT motifs detected in file\n",
      "\n",
      "Seq is: TTACAGGAAACTTCATATG\n",
      "0 TTACAGGAAACTTCATATG motifs detected in file\n",
      "\n",
      "Seq is: AACTCCTCCCACTGCCCGC\n",
      "0 AACTCCTCCCACTGCCCGC motifs detected in file\n",
      "\n",
      "Seq is: CACACATGCGCGAAGTCCC\n",
      "0 CACACATGCGCGAAGTCCC motifs detected in file\n",
      "\n",
      "Seq is: GGGACAAGTTGATTCGTGA\n",
      "0 GGGACAAGTTGATTCGTGA motifs detected in file\n",
      "\n",
      "Seq is: GATGAGAATCAAATGACGA\n",
      "0 GATGAGAATCAAATGACGA motifs detected in file\n",
      "\n",
      "Seq is: TTTAGGTGGAAGCTGGGAA\n",
      "0 TTTAGGTGGAAGCTGGGAA motifs detected in file\n",
      "\n",
      "Seq is: TTCAAAGTATTCCCGATAG\n",
      "0 TTCAAAGTATTCCCGATAG motifs detected in file\n",
      "\n",
      "Seq is: CATCAAAAGACCCCATCTT\n",
      "0 CATCAAAAGACCCCATCTT motifs detected in file\n",
      "\n",
      "Seq is: TTACATCGGGTAAAGAACG\n",
      "0 TTACATCGGGTAAAGAACG motifs detected in file\n",
      "\n",
      "Seq is: GTGGAGTACCCGATGTGGA\n",
      "0 GTGGAGTACCCGATGTGGA motifs detected in file\n",
      "\n",
      "Seq is: ACTGGATTGCAGAAATGCA\n",
      "0 ACTGGATTGCAGAAATGCA motifs detected in file\n",
      "\n",
      "Seq is: CCCATGAACGCGAATGAAA\n",
      "0 CCCATGAACGCGAATGAAA motifs detected in file\n",
      "\n",
      "Seq is: CATCGGAAGTAGTCCCAAA\n",
      "0 CATCGGAAGTAGTCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: CGACTTCTACCGCCATATA\n",
      "0 CGACTTCTACCGCCATATA motifs detected in file\n",
      "\n",
      "Seq is: TCCGACTCGTTCTCCACCA\n",
      "0 TCCGACTCGTTCTCCACCA motifs detected in file\n",
      "\n",
      "Seq is: CTCCAACTAGCCCCAAATA\n",
      "0 CTCCAACTAGCCCCAAATA motifs detected in file\n",
      "\n",
      "Seq is: GCTAATGTCTTGTCCGAGG\n",
      "0 GCTAATGTCTTGTCCGAGG motifs detected in file\n",
      "\n",
      "Seq is: AGATCTCCCTATACTTCGT\n",
      "0 AGATCTCCCTATACTTCGT motifs detected in file\n",
      "\n",
      "Seq is: GGCGGTCCAGGAAACTCAA\n",
      "0 GGCGGTCCAGGAAACTCAA motifs detected in file\n",
      "\n",
      "Seq is: AATTCGACACCGAGCATCA\n",
      "0 AATTCGACACCGAGCATCA motifs detected in file\n",
      "\n",
      "Seq is: CAAGACCGCTCGTCAAAAC\n",
      "0 CAAGACCGCTCGTCAAAAC motifs detected in file\n",
      "\n",
      "Seq is: AACTAAATTGACAACTCCA\n",
      "0 AACTAAATTGACAACTCCA motifs detected in file\n",
      "\n",
      "Seq is: AATCCCGATTCGTATCACC\n",
      "0 AATCCCGATTCGTATCACC motifs detected in file\n",
      "\n",
      "Seq is: CCATAGATTAGGGTCAAAC\n",
      "0 CCATAGATTAGGGTCAAAC motifs detected in file\n",
      "\n",
      "Seq is: CCAGAATCGATAACACACA\n",
      "0 CCAGAATCGATAACACACA motifs detected in file\n",
      "\n",
      "Seq is: AGTCTATACAATTTCGGTG\n",
      "0 AGTCTATACAATTTCGGTG motifs detected in file\n",
      "\n",
      "Seq is: AAAGCATCAGATAAAACAC\n",
      "0 AAAGCATCAGATAAAACAC motifs detected in file\n",
      "\n",
      "Seq is: AGTTGACCTCAAAAAATGT\n",
      "0 AGTTGACCTCAAAAAATGT motifs detected in file\n",
      "\n",
      "Seq is: TGAGTCCACTCAAAATCGA\n",
      "0 TGAGTCCACTCAAAATCGA motifs detected in file\n",
      "\n",
      "Seq is: AATAACCCCTCCCCCACTA\n",
      "0 AATAACCCCTCCCCCACTA motifs detected in file\n",
      "\n",
      "Seq is: GATTTTGTGAGAAATGCGG\n",
      "0 GATTTTGTGAGAAATGCGG motifs detected in file\n",
      "\n",
      "Seq is: CCGAAGATAACTGAACCCA\n",
      "0 CCGAAGATAACTGAACCCA motifs detected in file\n",
      "\n",
      "Seq is: ATTAGCACACTGGCGCCAC\n",
      "0 ATTAGCACACTGGCGCCAC motifs detected in file\n",
      "\n",
      "Seq is: CGACCGTGCTTTAACTGGT\n",
      "0 CGACCGTGCTTTAACTGGT motifs detected in file\n",
      "\n",
      "Seq is: TGGGACCCCAATCATATCG\n",
      "0 TGGGACCCCAATCATATCG motifs detected in file\n",
      "\n",
      "Seq is: AATCAATTTGAAAGCAAAT\n",
      "0 AATCAATTTGAAAGCAAAT motifs detected in file\n",
      "\n",
      "Seq is: AACGATTTCCAATTAGTGA\n",
      "0 AACGATTTCCAATTAGTGA motifs detected in file\n",
      "\n",
      "Seq is: CGACAAAGGGCACCCCCAC\n",
      "0 CGACAAAGGGCACCCCCAC motifs detected in file\n",
      "\n",
      "Seq is: ACCGATCCGATTCGCAAAA\n",
      "0 ACCGATCCGATTCGCAAAA motifs detected in file\n",
      "\n",
      "Seq is: CTCGACGTAACGACGCACG\n",
      "0 CTCGACGTAACGACGCACG motifs detected in file\n",
      "\n",
      "Seq is: AACTACGAACAAGGAAAAC\n",
      "0 AACTACGAACAAGGAAAAC motifs detected in file\n",
      "\n",
      "Seq is: TAGCACCACTAAATCTCCC\n",
      "0 TAGCACCACTAAATCTCCC motifs detected in file\n",
      "\n",
      "Seq is: ACTTGAGTGACCTTGCCCC\n",
      "0 ACTTGAGTGACCTTGCCCC motifs detected in file\n",
      "\n",
      "Seq is: TACGAACGTACACATTAGA\n",
      "0 TACGAACGTACACATTAGA motifs detected in file\n",
      "\n",
      "Seq is: ATGATTCTCGATTATCTTT\n",
      "0 ATGATTCTCGATTATCTTT motifs detected in file\n",
      "\n",
      "Seq is: CCGATCCTGGTTTGGATAA\n",
      "0 CCGATCCTGGTTTGGATAA motifs detected in file\n",
      "\n",
      "Seq is: ACAAATACCCCACTACACT\n",
      "0 ACAAATACCCCACTACACT motifs detected in file\n",
      "\n",
      "Seq is: CGGGCAGATTTGTAAAAGT\n",
      "0 CGGGCAGATTTGTAAAAGT motifs detected in file\n",
      "\n",
      "Seq is: CACATAACGAAACTCCTCC\n",
      "0 CACATAACGAAACTCCTCC motifs detected in file\n",
      "\n",
      "Seq is: ATTCGATGTATTTGACACC\n",
      "0 ATTCGATGTATTTGACACC motifs detected in file\n",
      "\n",
      "Seq is: CCCCACAGAAACCCCAAAC\n",
      "0 CCCCACAGAAACCCCAAAC motifs detected in file\n",
      "\n",
      "Seq is: CAACCAAGCCCCATACCCC\n",
      "0 CAACCAAGCCCCATACCCC motifs detected in file\n",
      "\n",
      "Seq is: AATCGTCGCCACACACACA\n",
      "0 AATCGTCGCCACACACACA motifs detected in file\n",
      "\n",
      "Seq is: CGACCCATCCTACCCGCCC\n",
      "0 CGACCCATCCTACCCGCCC motifs detected in file\n",
      "\n",
      "Seq is: CCAACATAACTACTCTAAT\n",
      "0 CCAACATAACTACTCTAAT motifs detected in file\n",
      "\n",
      "Seq is: CATCATACCCTCACCCTCC\n",
      "0 CATCATACCCTCACCCTCC motifs detected in file\n",
      "\n",
      "Seq is: CCTTTTATTACACAATCAA\n",
      "0 CCTTTTATTACACAATCAA motifs detected in file\n",
      "\n",
      "Seq is: CCCCCCACTGACAATTTTC\n",
      "0 CCCCCCACTGACAATTTTC motifs detected in file\n",
      "\n",
      "Seq is: ACGTATGGCGGTTTTCTAT\n",
      "0 ACGTATGGCGGTTTTCTAT motifs detected in file\n",
      "\n",
      "Seq is: TTTAAACTTTAGACCAATC\n",
      "0 TTTAAACTTTAGACCAATC motifs detected in file\n",
      "\n",
      "Seq is: CGACCACAATCCCAAGAAA\n",
      "0 CGACCACAATCCCAAGAAA motifs detected in file\n",
      "\n",
      "Seq is: CAAAAACCCCAAACCGTCT\n",
      "0 CAAAAACCCCAAACCGTCT motifs detected in file\n",
      "\n",
      "Seq is: CTACACAAATTCACGACAC\n",
      "0 CTACACAAATTCACGACAC motifs detected in file\n",
      "\n",
      "Seq is: CGGTCTTCGCCCCCTCCCC\n",
      "0 CGGTCTTCGCCCCCTCCCC motifs detected in file\n",
      "\n",
      "Seq is: CCCAAACCACCTTTAAAAA\n",
      "0 CCCAAACCACCTTTAAAAA motifs detected in file\n",
      "\n",
      "Seq is: ACAATACTACAGACACACC\n",
      "0 ACAATACTACAGACACACC motifs detected in file\n",
      "\n",
      "Seq is: TTTCACCGACACGTCTGTA\n",
      "0 TTTCACCGACACGTCTGTA motifs detected in file\n",
      "\n",
      "Seq is: AGTTAACAATAATAATACA\n",
      "0 AGTTAACAATAATAATACA motifs detected in file\n",
      "\n",
      "Seq is: GGATGTTCGTAATTAATTA\n",
      "0 GGATGTTCGTAATTAATTA motifs detected in file\n",
      "\n",
      "Seq is: ATTGTGTGAAATCATTCAT\n",
      "0 ATTGTGTGAAATCATTCAT motifs detected in file\n",
      "\n",
      "Seq is: ACAAGCGGACATTATAACT\n",
      "0 ACAAGCGGACATTATAACT motifs detected in file\n",
      "\n",
      "Seq is: TGCATCCACGCTATTTATT\n",
      "0 TGCATCCACGCTATTTATT motifs detected in file\n",
      "\n",
      "Seq is: ATCCTACTCCGTCCTTAGT\n",
      "0 ATCCTACTCCGTCCTTAGT motifs detected in file\n",
      "\n",
      "Seq is: TTCTGTCTATGACGCTGTA\n",
      "0 TTCTGTCTATGACGCTGTA motifs detected in file\n",
      "\n",
      "Seq is: TCCCACGAGGCCGAGGTCG\n",
      "0 TCCCACGAGGCCGAGGTCG motifs detected in file\n",
      "\n",
      "Seq is: CAGAGCGTTACGATAGCGC\n",
      "0 CAGAGCGTTACGATAGCGC motifs detected in file\n",
      "\n",
      "Seq is: ACGTATGGGGGGTCTGCTT\n",
      "0 ACGTATGGGGGGTCTGCTT motifs detected in file\n",
      "\n",
      "Seq is: TTATGGTTTACGTACCTCT\n",
      "0 TTATGGTTTACGTACCTCT motifs detected in file\n",
      "\n",
      "Seq is: CGAGGGCACTCACCAATTA\n",
      "0 CGAGGGCACTCACCAATTA motifs detected in file\n",
      "\n",
      "Seq is: TCCCACTATCTGGACACTA\n",
      "0 TCCCACTATCTGGACACTA motifs detected in file\n",
      "\n",
      "Seq is: GTAGCACTACAGAATAAATT\n",
      "0 GTAGCACTACAGAATAAATT motifs detected in file\n",
      "\n",
      "Seq is: CCCCTTGCACACCCGATAAA\n",
      "0 CCCCTTGCACACCCGATAAA motifs detected in file\n",
      "\n",
      "Seq is: TCCGAAATACTGGGACTTCA\n",
      "0 TCCGAAATACTGGGACTTCA motifs detected in file\n",
      "\n",
      "Seq is: TCCTTGGTCTACAGCCTATG\n",
      "0 TCCTTGGTCTACAGCCTATG motifs detected in file\n",
      "\n",
      "Seq is: TCAAGTGAAATCGATGGGGG\n",
      "0 TCAAGTGAAATCGATGGGGG motifs detected in file\n",
      "\n",
      "Seq is: TTCACAATACCCGGGCCTCG\n",
      "0 TTCACAATACCCGGGCCTCG motifs detected in file\n",
      "\n",
      "Seq is: CTCCTCTCATCGTGAGAACA\n",
      "0 CTCCTCTCATCGTGAGAACA motifs detected in file\n",
      "\n",
      "Seq is: CGCCCTATAACTAAAGTGCC\n",
      "0 CGCCCTATAACTAAAGTGCC motifs detected in file\n",
      "\n",
      "Seq is: TCCTACCACCAGTTCCCTGG\n",
      "0 TCCTACCACCAGTTCCCTGG motifs detected in file\n",
      "\n",
      "Seq is: GGATAGACTCCCCCCAGTAG\n",
      "0 GGATAGACTCCCCCCAGTAG motifs detected in file\n",
      "\n",
      "Seq is: GTACCCCTGCTCTTCCCTAA\n",
      "0 GTACCCCTGCTCTTCCCTAA motifs detected in file\n",
      "\n",
      "Seq is: ACTGACATTACACGATACAT\n",
      "0 ACTGACATTACACGATACAT motifs detected in file\n",
      "\n",
      "Seq is: GCCATTTACCGAAATACATG\n",
      "0 GCCATTTACCGAAATACATG motifs detected in file\n",
      "\n",
      "Seq is: ATACATGACAATTCCCACCC\n",
      "0 ATACATGACAATTCCCACCC motifs detected in file\n",
      "\n",
      "Seq is: ATCCAAACAACCATAGGATC\n",
      "0 ATCCAAACAACCATAGGATC motifs detected in file\n",
      "\n",
      "Seq is: ACCCACTCCCCACCGAAACC\n",
      "0 ACCCACTCCCCACCGAAACC motifs detected in file\n",
      "\n",
      "Seq is: TCAACGTCAACTACACACTA\n",
      "0 TCAACGTCAACTACACACTA motifs detected in file\n",
      "\n",
      "Seq is: TCAACTCCCAACTAACGACA\n",
      "0 TCAACTCCCAACTAACGACA motifs detected in file\n",
      "\n",
      "Seq is: TGAACGAACATTCGTACCCC\n",
      "0 TGAACGAACATTCGTACCCC motifs detected in file\n",
      "\n",
      "Seq is: TCCCCCAAAACTACACCTAA\n",
      "0 TCCCCCAAAACTACACCTAA motifs detected in file\n",
      "\n",
      "Seq is: CCCAAAAATACATGATGTCC\n",
      "0 CCCAAAAATACATGATGTCC motifs detected in file\n",
      "\n",
      "Seq is: ACCAGTTCATAAATACCATG\n",
      "0 ACCAGTTCATAAATACCATG motifs detected in file\n",
      "\n",
      "Seq is: GCATGTTATAAGTACCACCG\n",
      "0 GCATGTTATAAGTACCACCG motifs detected in file\n",
      "\n",
      "Seq is: ACCGTCATTACATGCTTTAT\n",
      "0 ACCGTCATTACATGCTTTAT motifs detected in file\n",
      "\n",
      "Seq is: GTATCGCCAACAACTACCCA\n",
      "0 GTATCGCCAACAACTACCCA motifs detected in file\n",
      "\n",
      "Seq is: CTCAGTTATGAACCCACCAT\n",
      "0 CTCAGTTATGAACCCACCAT motifs detected in file\n",
      "\n",
      "Seq is: GGGTTTAGACGAAGGGGTAC\n",
      "0 GGGTTTAGACGAAGGGGTAC motifs detected in file\n",
      "\n",
      "Seq is: TTTCTTGTCTCTTATCAAAT\n",
      "0 TTTCTTGTCTCTTATCAAAT motifs detected in file\n",
      "\n",
      "Seq is: TTAATCTTAGAATCGAAACC\n",
      "0 TTAATCTTAGAATCGAAACC motifs detected in file\n",
      "\n",
      "Seq is: CACGATTACCACCTCAATTT\n",
      "0 CACGATTACCACCTCAATTT motifs detected in file\n",
      "\n",
      "Seq is: CTGAAAAAGAGACTAAACAG\n",
      "0 CTGAAAAAGAGACTAAACAG motifs detected in file\n",
      "\n",
      "Seq is: GAACCTTTTTCCAAAAGTAG\n",
      "0 GAACCTTTTTCCAAAAGTAG motifs detected in file\n",
      "\n",
      "Seq is: AGGCCAAATGTTCTGACCAC\n",
      "0 AGGCCAAATGTTCTGACCAC motifs detected in file\n",
      "\n",
      "Seq is: ATAATCAAATATGATGTTCC\n",
      "0 ATAATCAAATATGATGTTCC motifs detected in file\n",
      "\n",
      "Seq is: TGTCCGGGTAAACTCATAAA\n",
      "0 TGTCCGGGTAAACTCATAAA motifs detected in file\n",
      "\n",
      "Seq is: ACAAAAGTTAATCCCTCTAT\n",
      "0 ACAAAAGTTAATCCCTCTAT motifs detected in file\n",
      "\n",
      "Seq is: CAACCATAATCCTAATCCTA\n",
      "0 CAACCATAATCCTAATCCTA motifs detected in file\n",
      "\n",
      "Seq is: ACAACACTTCATATCATGCC\n",
      "0 ACAACACTTCATATCATGCC motifs detected in file\n",
      "\n",
      "Seq is: TACGATGAACAGGTTACTAC\n",
      "0 TACGATGAACAGGTTACTAC motifs detected in file\n",
      "\n",
      "Seq is: CATTTTCCCATCGAATGACC\n",
      "0 CATTTTCCCATCGAATGACC motifs detected in file\n",
      "\n",
      "Seq is: AACAGGAGGCTAAGTCCAAT\n",
      "0 AACAGGAGGCTAAGTCCAAT motifs detected in file\n",
      "\n",
      "Seq is: CTTACTCCTCCAGACGCCGA\n",
      "0 CTTACTCCTCCAGACGCCGA motifs detected in file\n",
      "\n",
      "Seq is: TCCTCAGTTATTTCACTAAC\n",
      "0 TCCTCAGTTATTTCACTAAC motifs detected in file\n",
      "\n",
      "Seq is: CGAATCACCCGCTTTATAAT\n",
      "0 CGAATCACCCGCTTTATAAT motifs detected in file\n",
      "\n",
      "Seq is: ACGAAACAACAAACCTATAT\n",
      "0 ACGAAACAACAAACCTATAT motifs detected in file\n",
      "\n",
      "Seq is: ACCTCCTACCCCTAATAACG\n",
      "0 ACCTCCTACCCCTAATAACG motifs detected in file\n",
      "\n",
      "Seq is: ATCCTACTCCTACCTATCAT\n",
      "0 ATCCTACTCCTACCTATCAT motifs detected in file\n",
      "\n",
      "Seq is: TATCCCGTTCCTGCGGAGGA\n",
      "0 TATCCCGTTCCTGCGGAGGA motifs detected in file\n",
      "\n",
      "Seq is: TCAAACAATCCCTGCCTAGC\n",
      "0 TCAAACAATCCCTGCCTAGC motifs detected in file\n",
      "\n",
      "Seq is: CTCTTAACACATCCGCTTAT\n",
      "0 CTCTTAACACATCCGCTTAT motifs detected in file\n",
      "\n",
      "Seq is: CCTTTATAGTAAGCCCGAAC\n",
      "0 CCTTTATAGTAAGCCCGAAC motifs detected in file\n",
      "\n",
      "Seq is: TACACCCCTCCCCACAAATT\n",
      "0 TACACCCCTCCCCACAAATT motifs detected in file\n",
      "\n",
      "Seq is: CCCCAACCGATCCCATATTA\n",
      "0 CCCCAACCGATCCCATATTA motifs detected in file\n",
      "\n",
      "Seq is: ACAGACCCAGCGGATCCTCC\n",
      "0 ACAGACCCAGCGGATCCTCC motifs detected in file\n",
      "\n",
      "Seq is: AGACCACTCTTATCACAATT\n",
      "0 AGACCACTCTTATCACAATT motifs detected in file\n",
      "\n",
      "Seq is: ACAGTAATTCCTCTCTTCCT\n",
      "0 ACAGTAATTCCTCTCTTCCT motifs detected in file\n",
      "\n",
      "Seq is: TCTCTTCATTCGGCTCCCGC\n",
      "0 TCTCTTCATTCGGCTCCCGC motifs detected in file\n",
      "\n",
      "Seq is: AGAAACTAACACATCATTCC\n",
      "0 AGAAACTAACACATCATTCC motifs detected in file\n",
      "\n",
      "Seq is: CACCTTCCACTAAAATAGCC\n",
      "0 CACCTTCCACTAAAATAGCC motifs detected in file\n",
      "\n",
      "Seq is: TTACCCTCCACTAAGGATCC\n",
      "0 TTACCCTCCACTAAGGATCC motifs detected in file\n",
      "\n",
      "Seq is: CCCAACAAACTAGGGCAAAG\n",
      "0 CCCAACAAACTAGGGCAAAG motifs detected in file\n",
      "\n",
      "Seq is: CACGTTCTTATCCTCCACCT\n",
      "0 CACGTTCTTATCCTCCACCT motifs detected in file\n",
      "\n",
      "Seq is: CACAACGATCCCGACGTTAT\n",
      "0 CACAACGATCCCGACGTTAT motifs detected in file\n",
      "\n",
      "Seq is: TACTTCCCGTTCTACTTCAC\n",
      "0 TACTTCCCGTTCTACTTCAC motifs detected in file\n",
      "\n",
      "Seq is: TTTCCATTTCTTAGCACACT\n",
      "0 TTTCCATTTCTTAGCACACT motifs detected in file\n",
      "\n",
      "Seq is: CCCACCCTGACAGATGACTC\n",
      "0 CCCACCCTGACAGATGACTC motifs detected in file\n",
      "\n",
      "Seq is: ATCGGAGGAGTCTAAGTAAC\n",
      "0 ATCGGAGGAGTCTAAGTAAC motifs detected in file\n",
      "\n",
      "Seq is: TTGATCCAGACAGGGTTACA\n",
      "0 TTGATCCAGACAGGGTTACA motifs detected in file\n",
      "\n",
      "Seq is: TACCCTACCGCCTATCATTC\n",
      "0 TACCCTACCGCCTATCATTC motifs detected in file\n",
      "\n",
      "Seq is: AAACATTAATGACACCGGGG\n",
      "0 AAACATTAATGACACCGGGG motifs detected in file\n",
      "\n",
      "Seq is: AGTCTTACTATAAACCGGAG\n",
      "0 AGTCTTACTATAAACCGGAG motifs detected in file\n",
      "\n",
      "Seq is: TGCCCTCCTGTATCGGATAC\n",
      "0 TGCCCTCCTGTATCGGATAC motifs detected in file\n",
      "\n",
      "Seq is: TTCCGACAACGATATCAACG\n",
      "0 TTCCGACAACGATATCAACG motifs detected in file\n",
      "\n",
      "Seq is: TTCGTCCTCCTATTACGGCT\n",
      "0 TTCGTCCTCCTATTACGGCT motifs detected in file\n",
      "\n",
      "Seq is: ACAAAGTCCAAAGACTCATC\n",
      "0 ACAAAGTCCAAAGACTCATC motifs detected in file\n",
      "\n",
      "Seq is: TCTTTACTAGGCATTATATC\n",
      "0 TCTTTACTAGGCATTATATC motifs detected in file\n",
      "\n",
      "Seq is: CGGAGCGGGCTACACATCCT\n",
      "0 CGGAGCGGGCTACACATCCT motifs detected in file\n",
      "\n",
      "Seq is: TCTCCGTCTATTTCTTATAA\n",
      "0 TCTCCGTCTATTTCTTATAA motifs detected in file\n",
      "\n",
      "Seq is: CTCCGCGGTAACCGCACTTC\n",
      "0 CTCCGCGGTAACCGCACTTC motifs detected in file\n",
      "\n",
      "Seq is: CATCGCCTACTAAGTCGGTA\n",
      "0 CATCGCCTACTAAGTCGGTA motifs detected in file\n",
      "\n",
      "Seq is: TTAAATGCAGAGCTCACTAC\n",
      "0 TTAAATGCAGAGCTCACTAC motifs detected in file\n",
      "\n",
      "Seq is: ACCCGCTAACTACTTTTCCG\n",
      "0 ACCCGCTAACTACTTTTCCG motifs detected in file\n",
      "\n",
      "Seq is: CCAACTCCGCAGACCACTCA\n",
      "0 CCAACTCCGCAGACCACTCA motifs detected in file\n",
      "\n",
      "Seq is: TCACGTACCGATCCTTATCA\n",
      "0 TCACGTACCGATCCTTATCA motifs detected in file\n",
      "\n",
      "Seq is: GGACACCACTAAACCTCCTA\n",
      "0 GGACACCACTAAACCTCCTA motifs detected in file\n",
      "\n",
      "Seq is: GTCCGTCCGCGGTTCCTCAC\n",
      "0 GTCCGTCCGCGGTTCCTCAC motifs detected in file\n",
      "\n",
      "Seq is: TCGGCTTCAAAGTAGTACGC\n",
      "0 TCGGCTTCAAAGTAGTACGC motifs detected in file\n",
      "\n",
      "Seq is: CTCTACAACCTACCCCACCC\n",
      "0 CTCTACAACCTACCCCACCC motifs detected in file\n",
      "\n",
      "Seq is: CTCCAGCTACTTACTCACCA\n",
      "0 CTCCAGCTACTTACTCACCA motifs detected in file\n",
      "\n",
      "Seq is: ATTAATTAAAATAATCCCCC\n",
      "0 ATTAATTAAAATAATCCCCC motifs detected in file\n",
      "\n",
      "Seq is: AATCAAAACGCATAACCCCA\n",
      "0 AATCAAAACGCATAACCCCA motifs detected in file\n",
      "\n",
      "Seq is: GTAACCACAAGAACATCAAC\n",
      "0 GTAACCACAAGAACATCAAC motifs detected in file\n",
      "\n",
      "Seq is: TTTATGTTGCTACCAAAAAG\n",
      "0 TTTATGTTGCTACCAAAAAG motifs detected in file\n",
      "\n",
      "Seq is: TATAGTAACCAGCACCAACA\n",
      "0 TATAGTAACCAGCACCAACA motifs detected in file\n",
      "\n",
      "Seq is: TCAGGCACGCTCTTATTACT\n",
      "0 TCAGGCACGCTCTTATTACT motifs detected in file\n",
      "\n",
      "Seq is: ACATACGAAACAAAGACAAC\n",
      "0 ACATACGAAACAAAGACAAC motifs detected in file\n",
      "\n",
      "Seq is: TCACACCCAAATCATTACCC\n",
      "0 TCACACCCAAATCATTACCC motifs detected in file\n",
      "\n",
      "Seq is: CAAACACCCCAAAAGAAGAT\n",
      "0 CAAACACCCCAAAAGAAGAT motifs detected in file\n",
      "\n",
      "Seq is: TCGGAAGAGGATAAATACCC\n",
      "0 TCGGAAGAGGATAAATACCC motifs detected in file\n",
      "\n",
      "Seq is: CCAAATCATAACTAACAATC\n",
      "0 CCAAATCATAACTAACAATC motifs detected in file\n",
      "\n",
      "Seq is: GCCACACCAGCCCACACAAT\n",
      "0 GCCACACCAGCCCACACAAT motifs detected in file\n",
      "\n",
      "Seq is: AATAAGACTTAAAACCCCCT\n",
      "0 AATAAGACTTAAAACCCCCT motifs detected in file\n",
      "\n",
      "Seq is: CCAATATACCCAAATTATCA\n",
      "0 CCAATATACCCAAATTATCA motifs detected in file\n",
      "\n",
      "Seq is: AAAAAATTAAATAAATCCCC\n",
      "0 AAAAAATTAAATAAATCCCC motifs detected in file\n",
      "\n",
      "Seq is: CTTACTACCAACAGAAACCT\n",
      "0 CTTACTACCAACAGAAACCT motifs detected in file\n",
      "\n",
      "Seq is: ATATGATGTCGCTACCGATA\n",
      "0 ATATGATGTCGCTACCGATA motifs detected in file\n",
      "\n",
      "Seq is: ACTCCTCATAGGACTCCGTA\n",
      "0 ACTCCTCATAGGACTCCGTA motifs detected in file\n",
      "\n",
      "Seq is: CCCCCAGTCCCCAACTCCAG\n",
      "0 CCCCCAGTCCCCAACTCCAG motifs detected in file\n",
      "\n",
      "Seq is: AACCACTCACAAAATCACCC\n",
      "0 AACCACTCACAAAATCACCC motifs detected in file\n",
      "\n",
      "Seq is: CAATCGCTACCTCCATCCTA\n",
      "0 CAATCGCTACCTCCATCCTA motifs detected in file\n",
      "\n",
      "Seq is: ACCACGACACCCACTTTCTC\n",
      "0 ACCACGACACCCACTTTCTC motifs detected in file\n",
      "\n",
      "Seq is: ATACTACCCCACCACCAACA\n",
      "0 ATACTACCCCACCACCAACA motifs detected in file\n",
      "\n",
      "Seq is: CCATTTGAAATTATCACATC\n",
      "0 CCATTTGAAATTATCACATC motifs detected in file\n",
      "\n",
      "Seq is: CTTCGACTTATTAAATACTT\n",
      "0 CTTCGACTTATTAAATACTT motifs detected in file\n",
      "\n",
      "Seq is: CCTCTCCCCAGTCCCAACTA\n",
      "0 CCTCTCCCCAGTCCCAACTA motifs detected in file\n",
      "\n",
      "Seq is: AGCCCTCCTAGGATAACCAC\n",
      "0 AGCCCTCCTAGGATAACCAC motifs detected in file\n",
      "\n",
      "Seq is: GCCCCCGAAACATACTAATA\n",
      "0 GCCCCCGAAACATACTAATA motifs detected in file\n",
      "\n",
      "Seq is: CCCGCAACTAATCATCATCA\n",
      "0 CCCGCAACTAATCATCATCA motifs detected in file\n",
      "\n",
      "Seq is: ATGACCAACTTGTAACAAAC\n",
      "0 ATGACCAACTTGTAACAAAC motifs detected in file\n",
      "\n",
      "Seq is: AACCACATATATAACATTAA\n",
      "0 AACCACATATATAACATTAA motifs detected in file\n",
      "\n",
      "Seq is: CTCTAACGAGCCCCCTTATC\n",
      "0 CTCTAACGAGCCCCCTTATC motifs detected in file\n",
      "\n",
      "Seq is: CAATACACTAATCCTCATCC\n",
      "0 CAATACACTAATCCTCATCC motifs detected in file\n",
      "\n",
      "Seq is: CAATCCTACTCACCCTTCTT\n",
      "0 CAATCCTACTCACCCTTCTT motifs detected in file\n",
      "\n",
      "Seq is: CTTTCTCTCCTTCATTTCAA\n",
      "0 CTTTCTCTCCTTCATTTCAA motifs detected in file\n",
      "\n",
      "Seq is: ATTAATACGGAAAAACCCAA\n",
      "0 ATTAATACGGAAAAACCCAA motifs detected in file\n",
      "\n",
      "Seq is: CTCCACTACTACCTCCACCT\n",
      "0 CTCCACTACTACCTCCACCT motifs detected in file\n",
      "\n",
      "Seq is: CTAAACCACGACACTTTAAC\n",
      "0 CTAAACCACGACACTTTAAC motifs detected in file\n",
      "\n",
      "Seq is: AAAATCCATTATCGAAAAGA\n",
      "0 AAAATCCATTATCGAAAAGA motifs detected in file\n",
      "\n",
      "Seq is: TCAGTCCAATCCAGATCCTC\n",
      "0 TCAGTCCAATCCAGATCCTC motifs detected in file\n",
      "\n",
      "Seq is: CTCATCCCCGTCCAAAACCG\n",
      "0 CTCATCCCCGTCCAAAACCG motifs detected in file\n",
      "\n",
      "Seq is: AGCATTCTTCCGGATCTATC\n",
      "0 AGCATTCTTCCGGATCTATC motifs detected in file\n",
      "\n",
      "Seq is: CCCTAACACGCCACACACTA\n",
      "0 CCCTAACACGCCACACACTA motifs detected in file\n",
      "\n",
      "Seq is: CGATCCCATCTTAGGCTCAT\n",
      "0 CGATCCCATCTTAGGCTCAT motifs detected in file\n",
      "\n",
      "Seq is: ACAACCTCTTTATTTTACAC\n",
      "0 ACAACCTCTTTATTTTACAC motifs detected in file\n",
      "\n",
      "Seq is: GTATCACCCCTAAAATAAAA\n",
      "0 GTATCACCCCTAAAATAAAA motifs detected in file\n",
      "\n",
      "Seq is: TTCAAACAACCAATCCATCA\n",
      "0 TTCAAACAACCAATCCATCA motifs detected in file\n",
      "\n",
      "Seq is: ACTCCAGATCCCGACAATCT\n",
      "0 ACTCCAGATCCCGACAATCT motifs detected in file\n",
      "\n",
      "Seq is: TCAGGATCCTTTCACTGTCG\n",
      "0 TCAGGATCCTTTCACTGTCG motifs detected in file\n",
      "\n",
      "Seq is: CTCCCGACACTCAAAATCCA\n",
      "0 CTCCCGACACTCAAAATCCA motifs detected in file\n",
      "\n",
      "Seq is: TCTCCCCCTAACAACAAACC\n",
      "0 TCTCCCCCTAACAACAAACC motifs detected in file\n",
      "\n",
      "Seq is: TTCCCCCTACGCCCCCTTTA\n",
      "0 TTCCCCCTACGCCCCCTTTA motifs detected in file\n",
      "\n",
      "Seq is: CAACAATCATTACTCTTTAG\n",
      "0 CAACAATCATTACTCTTTAG motifs detected in file\n",
      "\n",
      "Seq is: GACGCTTATCCGAAGGCCGA\n",
      "0 GACGCTTATCCGAAGGCCGA motifs detected in file\n",
      "\n",
      "Seq is: CGGTCCGCAAATTACCCCAA\n",
      "0 CGGTCCGCAAATTACCCCAA motifs detected in file\n",
      "\n",
      "Seq is: ATCATCCCACCCCAATAAAA\n",
      "0 ATCATCCCACCCCAATAAAA motifs detected in file\n",
      "\n",
      "Seq is: GCAATTACAATCATTCCCAC\n",
      "0 GCAATTACAATCATTCCCAC motifs detected in file\n",
      "\n",
      "Seq is: CCCTTCGCTCCAACTGGACA\n",
      "0 CCCTTCGCTCCAACTGGACA motifs detected in file\n",
      "\n",
      "Seq is: ATCCCACTCTTCTTAATAAG\n",
      "0 ATCCCACTCTTCTTAATAAG motifs detected in file\n",
      "\n",
      "Seq is: CTCACGATATCCGCGAACAG\n",
      "0 CTCACGATATCCGCGAACAG motifs detected in file\n",
      "\n",
      "Seq is: TCCCTCCATCGCTACTCTCA\n",
      "0 TCCCTCCATCGCTACTCTCA motifs detected in file\n",
      "\n",
      "Seq is: TTATCTATCCCGAGTCCGCA\n",
      "0 TTATCTATCCCGAGTCCGCA motifs detected in file\n",
      "\n",
      "Seq is: AACACATACTATACAAACGC\n",
      "0 AACACATACTATACAAACGC motifs detected in file\n",
      "\n",
      "Seq is: CAAAGCTACTACACCAGAAA\n",
      "0 CAAAGCTACTACACCAGAAA motifs detected in file\n",
      "\n",
      "Seq is: CCTCATCTTTGGACACTCCT\n",
      "0 CCTCATCTTTGGACACTCCT motifs detected in file\n",
      "\n",
      "Seq is: TTCCATAAGGACGATTACGA\n",
      "0 TTCCATAAGGACGATTACGA motifs detected in file\n",
      "\n",
      "Seq is: TCCGACGGTTACCACTCCCT\n",
      "0 TCCGACGGTTACCACTCCCT motifs detected in file\n",
      "\n",
      "Seq is: CCAACTTCACTCTCCATACC\n",
      "0 CCAACTTCACTCTCCATACC motifs detected in file\n",
      "\n",
      "Seq is: AAAACTCATCAGGAGGATAA\n",
      "0 AAAACTCATCAGGAGGATAA motifs detected in file\n",
      "\n",
      "Seq is: AAAGCTTATAGAACAAGTAA\n",
      "0 AAAGCTTATAGAACAAGTAA motifs detected in file\n",
      "\n",
      "Seq is: CAATTCCAACACCTACTACC\n",
      "0 CAATTCCAACACCTACTACC motifs detected in file\n",
      "\n",
      "Seq is: TGGGCCTCGTGTATTTATCA\n",
      "0 TGGGCCTCGTGTATTTATCA motifs detected in file\n",
      "\n",
      "Seq is: TACCGAAACTTCTTCCGCAC\n",
      "0 TACCGAAACTTCTTCCGCAC motifs detected in file\n",
      "\n",
      "Seq is: CCATGTCTACACGTCCTTAC\n",
      "0 CCATGTCTACACGTCCTTAC motifs detected in file\n",
      "\n",
      "Seq is: GATCCACACCAACCAACTAC\n",
      "0 GATCCACACCAACCAACTAC motifs detected in file\n",
      "\n",
      "Seq is: GGCTAACATTGATAATACTC\n",
      "0 GGCTAACATTGATAATACTC motifs detected in file\n",
      "\n",
      "Seq is: AGGATCAACTGAACTTCACC\n",
      "0 AGGATCAACTGAACTTCACC motifs detected in file\n",
      "\n",
      "Seq is: TCTTCCGATGCTAAAAAAAC\n",
      "0 TCTTCCGATGCTAAAAAAAC motifs detected in file\n",
      "\n",
      "Seq is: TACAGTAAAACACATTCCCG\n",
      "0 TACAGTAAAACACATTCCCG motifs detected in file\n",
      "\n",
      "Seq is: CGTCTGACGACGCTTGTCTC\n",
      "0 CGTCTGACGACGCTTGTCTC motifs detected in file\n",
      "\n",
      "Seq is: ACCACTATCGCGGATTCGTA\n",
      "0 ACCACTATCGCGGATTCGTA motifs detected in file\n",
      "\n",
      "Seq is: TCACAATCTCAAACCTAATC\n",
      "0 TCACAATCTCAAACCTAATC motifs detected in file\n",
      "\n",
      "Seq is: ACCCGATAAAAGACGATCCC\n",
      "0 ACCCGATAAAAGACGATCCC motifs detected in file\n",
      "\n",
      "Seq is: CCACCTTCGCCTACTCATTC\n",
      "0 CCACCTTCGCCTACTCATTC motifs detected in file\n",
      "\n",
      "Seq is: TTCTAAGGACGATGTTGATA\n",
      "0 TTCTAAGGACGATGTTGATA motifs detected in file\n",
      "\n",
      "Seq is: TCACGAACTCACCTCATCCC\n",
      "0 TCACGAACTCACCTCATCCC motifs detected in file\n",
      "\n",
      "Seq is: GACTCTGACCCCACCCCGGA\n",
      "0 GACTCTGACCCCACCCCGGA motifs detected in file\n",
      "\n",
      "Seq is: AGATACCGACTCCCCTCAGT\n",
      "0 AGATACCGACTCCCCTCAGT motifs detected in file\n",
      "\n",
      "Seq is: CCCCACCTCTGGATTAACCC\n",
      "0 CCCCACCTCTGGATTAACCC motifs detected in file\n",
      "\n",
      "Seq is: GACTAAACGGACGACGACGA\n",
      "0 GACTAAACGGACGACGACGA motifs detected in file\n",
      "\n",
      "Seq is: TCCTCCTCCGGATCATCACC\n",
      "0 TCCTCCTCCGGATCATCACC motifs detected in file\n",
      "\n",
      "Seq is: CCACTCCGAACCTAATCGCA\n",
      "0 CCACTCCGAACCTAATCGCA motifs detected in file\n",
      "\n",
      "Seq is: AATCTTCCCGATAAACAACA\n",
      "0 AATCTTCCCGATAAACAACA motifs detected in file\n",
      "\n",
      "Seq is: CCCAGAGTACTCAACCTCAC\n",
      "0 CCCAGAGTACTCAACCTCAC motifs detected in file\n",
      "\n",
      "Seq is: ATCCTATTTAGTACGATTCC\n",
      "0 ATCCTATTTAGTACGATTCC motifs detected in file\n",
      "\n",
      "Seq is: GCTCCTACTTTGGCTATAGC\n",
      "0 GCTCCTACTTTGGCTATAGC motifs detected in file\n",
      "\n",
      "Seq is: GGCTATGCCAACATATCCTA\n",
      "0 GGCTATGCCAACATATCCTA motifs detected in file\n",
      "\n",
      "Seq is: ACGAACTTACCGACGACACA\n",
      "0 ACGAACTTACCGACGACACA motifs detected in file\n",
      "\n",
      "Seq is: ACCGTAGACGAGCCCGCATA\n",
      "0 ACCGTAGACGAGCCCGCATA motifs detected in file\n",
      "\n",
      "Seq is: GTAGTTGACTACTCGTTCTT\n",
      "0 GTAGTTGACTACTCGTTCTT motifs detected in file\n",
      "\n",
      "Seq is: CCTATATTAAGGATGCGGGA\n",
      "0 CCTATATTAAGGATGCGGGA motifs detected in file\n",
      "\n",
      "Seq is: GAGTCGGCTACTTGTCAACC\n",
      "0 GAGTCGGCTACTTGTCAACC motifs detected in file\n",
      "\n",
      "Seq is: TTATCCAACAATCGCCATTG\n",
      "0 TTATCCAACAATCGCCATTG motifs detected in file\n",
      "\n",
      "Seq is: ATTCTAATCATACCATTAAT\n",
      "0 ATTCTAATCATACCATTAAT motifs detected in file\n",
      "\n",
      "Seq is: CCTTCTACTCATCTATAAAC\n",
      "0 CCTTCTACTCATCTATAAAC motifs detected in file\n",
      "\n",
      "Seq is: TTCTTGACTAATTACAAACC\n",
      "0 TTCTTGACTAATTACAAACC motifs detected in file\n",
      "\n",
      "Seq is: CAGACTCAAATATATAGTGT\n",
      "0 CAGACTCAAATATATAGTGT motifs detected in file\n",
      "\n",
      "Seq is: CACTCTTAAGATACTACCTG\n",
      "0 CACTCTTAAGATACTACCTG motifs detected in file\n",
      "\n",
      "Seq is: GTACATTGCTTGTTACGATG\n",
      "0 GTACATTGCTTGTTACGATG motifs detected in file\n",
      "\n",
      "Seq is: TCCCTACTTATAATACCTCT\n",
      "0 TCCCTACTTATAATACCTCT motifs detected in file\n",
      "\n",
      "Seq is: TCATCAGATCAAACTTCGAA\n",
      "0 TCATCAGATCAAACTTCGAA motifs detected in file\n",
      "\n",
      "Seq is: TCCCTCTCGACCCAACAAAC\n",
      "0 TCCCTCTCGACCCAACAAAC motifs detected in file\n",
      "\n",
      "Seq is: CCAACACCGAGTCACAGTCA\n",
      "0 CCAACACCGAGTCACAGTCA motifs detected in file\n",
      "\n",
      "Seq is: AGCTCTATTATTGAAGAACC\n",
      "0 AGCTCTATTATTGAAGAACC motifs detected in file\n",
      "\n",
      "Seq is: AGATCCGTGTACTTATAACA\n",
      "0 AGATCCGTGTACTTATAACA motifs detected in file\n",
      "\n",
      "Seq is: ACACCCCTTCTCTGACTATT\n",
      "0 ACACCCCTTCTCTGACTATT motifs detected in file\n",
      "\n",
      "Seq is: ATTTCCACCTACGCTGTTAC\n",
      "0 ATTTCCACCTACGCTGTTAC motifs detected in file\n",
      "\n",
      "Seq is: CTAAAATGTATTACCCCCAT\n",
      "0 CTAAAATGTATTACCCCCAT motifs detected in file\n",
      "\n",
      "Seq is: ACTCAAAAAAAACAATCCCA\n",
      "0 ACTCAAAAAAAACAATCCCA motifs detected in file\n",
      "\n",
      "Seq is: ATTGCTCCCACCATTCCTAC\n",
      "0 ATTGCTCCCACCATTCCTAC motifs detected in file\n",
      "\n",
      "Seq is: CCCCCTTAATCCCTTCAGTC\n",
      "0 CCCCCTTAATCCCTTCAGTC motifs detected in file\n",
      "\n",
      "Seq is: CCAATCCCACCAATATCATC\n",
      "0 CCAATCCCACCAATATCATC motifs detected in file\n",
      "\n",
      "Seq is: ACACGTACCAATAATGAAAA\n",
      "0 ACACGTACCAATAATGAAAA motifs detected in file\n",
      "\n",
      "Seq is: TAAACCTCAACGTGGTTTTA\n",
      "0 TAAACCTCAACGTGGTTTTA motifs detected in file\n",
      "\n",
      "Seq is: AAAACCCCGGATTCTGGTTA\n",
      "0 AAAACCCCGGATTCTGGTTA motifs detected in file\n",
      "\n",
      "Seq is: CCTATCGACAATAGGAAATT\n",
      "0 CCTATCGACAATAGGAAATT motifs detected in file\n",
      "\n",
      "Seq is: TTCAACTCTTTCGGTACAAC\n",
      "0 TTCAACTCTTTCGGTACAAC motifs detected in file\n",
      "\n",
      "Seq is: AATCTGTACCCCCGTACTCA\n",
      "0 AATCTGTACCCCCGTACTCA motifs detected in file\n",
      "\n",
      "Seq is: ATCGTCAAGAACACTCGAAA\n",
      "0 ATCGTCAAGAACACTCGAAA motifs detected in file\n",
      "\n",
      "Seq is: GAGCCATTTATTCCCCAGCA\n",
      "0 GAGCCATTTATTCCCCAGCA motifs detected in file\n",
      "\n",
      "Seq is: TTCGGAGACAACAGTCTAAG\n",
      "0 TTCGGAGACAACAGTCTAAG motifs detected in file\n",
      "\n",
      "Seq is: TGTTAGACTACAAAACCAAT\n",
      "0 TGTTAGACTACAAAACCAAT motifs detected in file\n",
      "\n",
      "Seq is: TTGATATAAATGTTCTCCTT\n",
      "0 TTGATATAAATGTTCTCCTT motifs detected in file\n",
      "\n",
      "Seq is: TTGGGCCATTACTACAGCCC\n",
      "0 TTGGGCCATTACTACAGCCC motifs detected in file\n",
      "\n",
      "Seq is: CAACTCCCTATCCTCCTCTT\n",
      "0 CAACTCCCTATCCTCCTCTT motifs detected in file\n",
      "\n",
      "Seq is: ACCCCCTATCCACATACTTG\n",
      "0 ACCCCCTATCCACATACTTG motifs detected in file\n",
      "\n",
      "Seq is: TACTCCCACAAAAGAGCACA\n",
      "0 TACTCCCACAAAAGAGCACA motifs detected in file\n",
      "\n",
      "Seq is: CTTACTCCCAAAATACAACA\n",
      "0 CTTACTCCCAAAATACAACA motifs detected in file\n",
      "\n",
      "Seq is: ATTACACCACCCACTCACTC\n",
      "0 ATTACACCACCCACTCACTC motifs detected in file\n",
      "\n",
      "Seq is: GGGGTAACACAACACCATTT\n",
      "0 GGGGTAACACAACACCATTT motifs detected in file\n",
      "\n",
      "Seq is: ATACATCTCCCTCATATCCC\n",
      "0 ATACATCTCCCTCATATCCC motifs detected in file\n",
      "\n",
      "Seq is: GACACTGATCATACAACTCA\n",
      "0 GACACTGATCATACAACTCA motifs detected in file\n",
      "\n",
      "Seq is: GGACATTCATCCTCTCACTA\n",
      "0 GGACATTCATCCTCTCACTA motifs detected in file\n",
      "\n",
      "Seq is: TAAACTAGTCCTCTTGCACC\n",
      "0 TAAACTAGTCCTCTTGCACC motifs detected in file\n",
      "\n",
      "Seq is: AATGATCGTGTCTCTCAAGA\n",
      "0 AATGATCGTGTCTCTCAAGA motifs detected in file\n",
      "\n",
      "Seq is: GGGTCATCCAATTATCACCC\n",
      "0 GGGTCATCCAATTATCACCC motifs detected in file\n",
      "\n",
      "Seq is: CCCATTCCGCTCCAATCGCT\n",
      "0 CCCATTCCGCTCCAATCGCT motifs detected in file\n",
      "\n",
      "Seq is: CCGAACGATCTTCAGTAGTT\n",
      "0 CCGAACGATCTTCAGTAGTT motifs detected in file\n",
      "\n",
      "Seq is: TTTCGATAATCACCCTCATC\n",
      "0 TTTCGATAATCACCCTCATC motifs detected in file\n",
      "\n",
      "Seq is: TCAAACTTCAGGAACTCTCT\n",
      "0 TCAAACTTCAGGAACTCTCT motifs detected in file\n",
      "\n",
      "Seq is: CCTAATACTACGCTGACACT\n",
      "0 CCTAATACTACGCTGACACT motifs detected in file\n",
      "\n",
      "Seq is: CACGCAAGCATCAAACTCAA\n",
      "0 CACGCAAGCATCAAACTCAA motifs detected in file\n",
      "\n",
      "Seq is: ACGATCCGTCTTATCATTAC\n",
      "0 ACGATCCGTCTTATCATTAC motifs detected in file\n",
      "\n",
      "Seq is: TCCTACATTCGGGCACCCGC\n",
      "0 TCCTACATTCGGGCACCCGC motifs detected in file\n",
      "\n",
      "Seq is: TAATACTCTTACTGACGCGG\n",
      "0 TAATACTCTTACTGACGCGG motifs detected in file\n",
      "\n",
      "Seq is: CCACTTCGAAGTCCCCCAAA\n",
      "0 CCACTTCGAAGTCCCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: CCTACTCTTACCGACAATGA\n",
      "0 CCTACTCTTACCGACAATGA motifs detected in file\n",
      "\n",
      "Seq is: TGCTCCCGATACACCGACTA\n",
      "0 TGCTCCCGATACACCGACTA motifs detected in file\n",
      "\n",
      "Seq is: ACTTCTCATACGTTACTCGC\n",
      "0 ACTTCTCATACGTTACTCGC motifs detected in file\n",
      "\n",
      "Seq is: TAAAATCCAGACAAACAGCA\n",
      "0 TAAAATCCAGACAAACAGCA motifs detected in file\n",
      "\n",
      "Seq is: TCCGTCTACCTCGAACAATA\n",
      "0 TCCGTCTACCTCGAACAATA motifs detected in file\n",
      "\n",
      "Seq is: TTAATACGGAGTATCCCTAT\n",
      "0 TTAATACGGAGTATCCCTAT motifs detected in file\n",
      "\n",
      "Seq is: CATGTTCCTTCCCCATCCGA\n",
      "0 CATGTTCCTTCCCCATCCGA motifs detected in file\n",
      "\n",
      "Seq is: TACACAAAACAGTCCCCCAA\n",
      "0 TACACAAAACAGTCCCCCAA motifs detected in file\n",
      "\n",
      "Seq is: CTCTTACTCACACTCCGCAT\n",
      "0 CTCTTACTCACACTCCGCAT motifs detected in file\n",
      "\n",
      "Seq is: AATATGGTATCGGCGGATCA\n",
      "0 AATATGGTATCGGCGGATCA motifs detected in file\n",
      "\n",
      "Seq is: AAATTCTCATGACGCCGTTC\n",
      "0 AAATTCTCATGACGCCGTTC motifs detected in file\n",
      "\n",
      "Seq is: ATGATAACTGGGTCGCTACC\n",
      "0 ATGATAACTGGGTCGCTACC motifs detected in file\n",
      "\n",
      "Seq is: CCCGAAGCTGTACCCGAAAT\n",
      "0 CCCGAAGCTGTACCCGAAAT motifs detected in file\n",
      "\n",
      "Seq is: CCCTCAGTATTCACCTCAGG\n",
      "0 CCCTCAGTATTCACCTCAGG motifs detected in file\n",
      "\n",
      "Seq is: CATTTCTCCATAGAAATGAT\n",
      "0 CATTTCTCCATAGAAATGAT motifs detected in file\n",
      "\n",
      "Seq is: ATTTTCGATAACACATTCGA\n",
      "0 ATTTTCGATAACACATTCGA motifs detected in file\n",
      "\n",
      "Seq is: TCAGTATAATTCAACAACCG\n",
      "0 TCAGTATAATTCAACAACCG motifs detected in file\n",
      "\n",
      "Seq is: AGTCCTCAAACTATCAAGAA\n",
      "0 AGTCCTCAAACTATCAAGAA motifs detected in file\n",
      "\n",
      "Seq is: CCCGTCACTCTCACTCATCA\n",
      "0 CCCGTCACTCTCACTCATCA motifs detected in file\n",
      "\n",
      "Seq is: TCTTACAAATCACTCGGATC\n",
      "0 TCTTACAAATCACTCGGATC motifs detected in file\n",
      "\n",
      "Seq is: CCACAACACTCACATTTAAT\n",
      "0 CCACAACACTCACATTTAAT motifs detected in file\n",
      "\n",
      "Seq is: CACGCTACTCATCCCCTTCC\n",
      "0 CACGCTACTCATCCCCTTCC motifs detected in file\n",
      "\n",
      "Seq is: CTCGGATGATCCCACATCTT\n",
      "0 CTCGGATGATCCCACATCTT motifs detected in file\n",
      "\n",
      "Seq is: ATCCTTCATACACGGACGCA\n",
      "0 ATCCTTCATACACGGACGCA motifs detected in file\n",
      "\n",
      "Seq is: AGTCCGCAAGACCGACCAAC\n",
      "0 AGTCCGCAAGACCGACCAAC motifs detected in file\n",
      "\n",
      "Seq is: GGAGTAGCCCACTACTATCG\n",
      "0 GGAGTAGCCCACTACTATCG motifs detected in file\n",
      "\n",
      "Seq is: GTTCCACCCCTATTCACACC\n",
      "0 GTTCCACCCCTATTCACACC motifs detected in file\n",
      "\n",
      "Seq is: AAAGCTTCTTCTATATTTTA\n",
      "0 AAAGCTTCTTCTATATTTTA motifs detected in file\n",
      "\n",
      "Seq is: TACTAATCAAGACACCGACA\n",
      "0 TACTAATCAAGACACCGACA motifs detected in file\n",
      "\n",
      "Seq is: CTTACAATATTAATTCCTCT\n",
      "0 CTTACAATATTAATTCCTCT motifs detected in file\n",
      "\n",
      "Seq is: AAACATCCCTCTAATCATAT\n",
      "0 AAACATCCCTCTAATCATAT motifs detected in file\n",
      "\n",
      "Seq is: CTCTCCATCTCAAAAAAAGC\n",
      "0 CTCTCCATCTCAAAAAAAGC motifs detected in file\n",
      "\n",
      "Seq is: ACTATCACCAAGTGACCTAT\n",
      "0 ACTATCACCAAGTGACCTAT motifs detected in file\n",
      "\n",
      "Seq is: TCACCGCAACCGAACGGTAC\n",
      "0 TCACCGCAACCGAACGGTAC motifs detected in file\n",
      "\n",
      "Seq is: TAACACTCCCCATCCTCAGT\n",
      "0 TAACACTCCCCATCCTCAGT motifs detected in file\n",
      "\n",
      "Seq is: CCATCAATCATAATCCTCCC\n",
      "0 CCATCAATCATAATCCTCCC motifs detected in file\n",
      "\n",
      "Seq is: CCCAACAATCCCCCAGCCTC\n",
      "0 CCCAACAATCCCCCAGCCTC motifs detected in file\n",
      "\n",
      "Seq is: CTTTTCCAACCCCTTGTCGA\n",
      "0 CTTTTCCAACCCCTTGTCGA motifs detected in file\n",
      "\n",
      "Seq is: TTTATCCAACAACAACTAAA\n",
      "0 TTTATCCAACAACAACTAAA motifs detected in file\n",
      "\n",
      "Seq is: CCAATTTTTTATCATCTCCC\n",
      "0 CCAATTTTTTATCATCTCCC motifs detected in file\n",
      "\n",
      "Seq is: TACTACGATTATTAATCCGA\n",
      "0 TACTACGATTATTAATCCGA motifs detected in file\n",
      "\n",
      "Seq is: CACCCACCAACACAACTAAG\n",
      "0 CACCCACCAACACAACTAAG motifs detected in file\n",
      "\n",
      "Seq is: TTTAATACACAAAAAACCTT\n",
      "0 TTTAATACACAAAAAACCTT motifs detected in file\n",
      "\n",
      "Seq is: TCAGTACAGTCACCATCATT\n",
      "0 TCAGTACAGTCACCATCATT motifs detected in file\n",
      "\n",
      "Seq is: ATATTAACAACCCTGCTAAT\n",
      "0 ATATTAACAACCCTGCTAAT motifs detected in file\n",
      "\n",
      "Seq is: CAAAATCGTAACCTCATCCA\n",
      "0 CAAAATCGTAACCTCATCCA motifs detected in file\n",
      "\n",
      "Seq is: AATCCAATACATGCATCAGA\n",
      "0 AATCCAATACATGCATCAGA motifs detected in file\n",
      "\n",
      "Seq is: TCCGGTATACACAACCTCTA\n",
      "0 TCCGGTATACACAACCTCTA motifs detected in file\n",
      "\n",
      "Seq is: ACTCTGATCATCCCGATCCG\n",
      "0 ACTCTGATCATCCCGATCCG motifs detected in file\n",
      "\n",
      "Seq is: GGTGGCGACGAAGCGTCCGC\n",
      "0 GGTGGCGACGAAGCGTCCGC motifs detected in file\n",
      "\n",
      "Seq is: CGTTTCTGATCATACCGTTA\n",
      "0 CGTTTCTGATCATACCGTTA motifs detected in file\n",
      "\n",
      "Seq is: TCCGTGTTATAACCGATTCT\n",
      "0 TCCGTGTTATAACCGATTCT motifs detected in file\n",
      "\n",
      "Seq is: CCCTCACCCACAACTCCCAA\n",
      "0 CCCTCACCCACAACTCCCAA motifs detected in file\n",
      "\n",
      "Seq is: TACTCTCATCGATATTACTT\n",
      "0 TACTCTCATCGATATTACTT motifs detected in file\n",
      "\n",
      "Seq is: GTCGCTATCATAATAAGGAA\n",
      "0 GTCGCTATCATAATAAGGAA motifs detected in file\n",
      "\n",
      "Seq is: GATCCGTATCATCCCTCCTA\n",
      "0 GATCCGTATCATCCCTCCTA motifs detected in file\n",
      "\n",
      "Seq is: TACTCCACACTCGCTATATG\n",
      "0 TACTCCACACTCGCTATATG motifs detected in file\n",
      "\n",
      "Seq is: ATCATAAGGATCTTCACTCT\n",
      "0 ATCATAAGGATCTTCACTCT motifs detected in file\n",
      "\n",
      "Seq is: ACCATTTACGATCATATTAT\n",
      "0 ACCATTTACGATCATATTAT motifs detected in file\n",
      "\n",
      "Seq is: AAATACATTTACTCCCCGTA\n",
      "0 AAATACATTTACTCCCCGTA motifs detected in file\n",
      "\n",
      "Seq is: AACCATTTATACTAATAGTA\n",
      "0 AACCATTTATACTAATAGTA motifs detected in file\n",
      "\n",
      "Seq is: TTAAATTACTCAGCTTTAGT\n",
      "0 TTAAATTACTCAGCTTTAGT motifs detected in file\n",
      "\n",
      "Seq is: AAGCAAAACAAATTTGATAT\n",
      "0 AAGCAAAACAAATTTGATAT motifs detected in file\n",
      "\n",
      "Seq is: ATGGTTAAGCCAAGTCAGAT\n",
      "0 ATGGTTAAGCCAAGTCAGAT motifs detected in file\n",
      "\n",
      "Seq is: TAGGAAAAACATCAGTGAGT\n",
      "0 TAGGAAAAACATCAGTGAGT motifs detected in file\n",
      "\n",
      "Seq is: ATCCGGTCTGAATCCCGATC\n",
      "0 ATCCGGTCTGAATCCCGATC motifs detected in file\n",
      "\n",
      "Seq is: CTACTACTAATTATTCTCCC\n",
      "0 CTACTACTAATTATTCTCCC motifs detected in file\n",
      "\n",
      "Seq is: TACTGTATTGATAATCACCG\n",
      "0 TACTGTATTGATAATCACCG motifs detected in file\n",
      "\n",
      "Seq is: TCCAATCAACAAACATCCCG\n",
      "0 TCCAATCAACAAACATCCCG motifs detected in file\n",
      "\n",
      "Seq is: AGTACCATCCCCATTTTCCT\n",
      "0 AGTACCATCCCCATTTTCCT motifs detected in file\n",
      "\n",
      "Seq is: CCCGTTAAAGATCTAGTTTA\n",
      "0 CCCGTTAAAGATCTAGTTTA motifs detected in file\n",
      "\n",
      "Seq is: TTATTCTTCCATTATCGATG\n",
      "0 TTATTCTTCCATTATCGATG motifs detected in file\n",
      "\n",
      "Seq is: ATTCTTCTTAAAATACCTCT\n",
      "0 ATTCTTCTTAAAATACCTCT motifs detected in file\n",
      "\n",
      "Seq is: TTCCCTGCGCCCGCCCCCTA\n",
      "0 TTCCCTGCGCCCGCCCCCTA motifs detected in file\n",
      "\n",
      "Seq is: TATCCCAGCTTCGGCGTGAG\n",
      "0 TATCCCAGCTTCGGCGTGAG motifs detected in file\n",
      "\n",
      "Seq is: CATTCCCCACCTAAAAAGAT\n",
      "0 CATTCCCCACCTAAAAAGAT motifs detected in file\n",
      "\n",
      "Seq is: ACATCGGCAACTCAACACCA\n",
      "0 ACATCGGCAACTCAACACCA motifs detected in file\n",
      "\n",
      "Seq is: TCAGTTTTACATTATTAATA\n",
      "0 TCAGTTTTACATTATTAATA motifs detected in file\n",
      "\n",
      "Seq is: ATCATCATTCCGATCCTCCC\n",
      "0 ATCATCATTCCGATCCTCCC motifs detected in file\n",
      "\n",
      "Seq is: ACAACTAATAATTTTAATTC\n",
      "0 ACAACTAATAATTTTAATTC motifs detected in file\n",
      "\n",
      "Seq is: CGCTTCAAATAATGAGAAAA\n",
      "0 CGCTTCAAATAATGAGAAAA motifs detected in file\n",
      "\n",
      "Seq is: AACTTACAACAGTTTTGATC\n",
      "0 AACTTACAACAGTTTTGATC motifs detected in file\n",
      "\n",
      "Seq is: AATTAACCTTCAATTGCCAT\n",
      "0 AATTAACCTTCAATTGCCAT motifs detected in file\n",
      "\n",
      "Seq is: GATAAATATGATTTTCTCAT\n",
      "0 GATAAATATGATTTTCTCAT motifs detected in file\n",
      "\n",
      "Seq is: TCTGGGAGTAGTTATCTACC\n",
      "0 TCTGGGAGTAGTTATCTACC motifs detected in file\n",
      "\n",
      "Seq is: TCTGTATGTCTTTATCAGTT\n",
      "0 TCTGTATGTCTTTATCAGTT motifs detected in file\n",
      "\n",
      "Seq is: TGGTGTAGATGTTTTACGGT\n",
      "0 TGGTGTAGATGTTTTACGGT motifs detected in file\n",
      "\n",
      "Seq is: CATAGTCCGCCGCCGAAGCT\n",
      "0 CATAGTCCGCCGCCGAAGCT motifs detected in file\n",
      "\n",
      "Seq is: TCGGTTTCACTACAAACCTA\n",
      "0 TCGGTTTCACTACAAACCTA motifs detected in file\n",
      "\n",
      "Seq is: CATTTCACTTTATAATCAAC\n",
      "0 CATTTCACTTTATAATCAAC motifs detected in file\n",
      "\n",
      "Seq is: CGCCTACTTCGTCTATCACT\n",
      "0 CGCCTACTTCGTCTATCACT motifs detected in file\n",
      "\n",
      "Seq is: CCTTTCAACTCGGTTATTAC\n",
      "0 CCTTTCAACTCGGTTATTAC motifs detected in file\n",
      "\n",
      "Seq is: TGCACTTCAGGCACCTTCGG\n",
      "0 TGCACTTCAGGCACCTTCGG motifs detected in file\n",
      "\n",
      "Seq is: ACACCGATGTTTTTTACAAC\n",
      "0 ACACCGATGTTTTTTACAAC motifs detected in file\n",
      "\n",
      "Seq is: TCGGCATCTACGGCAGCCTT\n",
      "0 TCGGCATCTACGGCAGCCTT motifs detected in file\n",
      "\n",
      "Seq is: TACCACTTCCCTCTGAGCTT\n",
      "0 TACCACTTCCCTCTGAGCTT motifs detected in file\n",
      "\n",
      "Seq is: CATGAGACTCCGAACATCCT\n",
      "0 CATGAGACTCCGAACATCCT motifs detected in file\n",
      "\n",
      "Seq is: CCCATTTTATCTCTGGGTCA\n",
      "0 CCCATTTTATCTCTGGGTCA motifs detected in file\n",
      "\n",
      "Seq is: TTTTAACATTATTCGTCACG\n",
      "0 TTTTAACATTATTCGTCACG motifs detected in file\n",
      "\n",
      "Seq is: AACTTAATAAACCAAAGCCA\n",
      "0 AACTTAATAAACCAAAGCCA motifs detected in file\n",
      "\n",
      "Seq is: ACAAAAGATAATCTGATACC\n",
      "0 ACAAAAGATAATCTGATACC motifs detected in file\n",
      "\n",
      "Seq is: ACTCGAGTCCACTAACTATG\n",
      "0 ACTCGAGTCCACTAACTATG motifs detected in file\n",
      "\n",
      "Seq is: AGGACTACGCTCATTATGCC\n",
      "0 AGGACTACGCTCATTATGCC motifs detected in file\n",
      "\n",
      "Seq is: TACACAAATCCTCACCCTGA\n",
      "0 TACACAAATCCTCACCCTGA motifs detected in file\n",
      "\n",
      "Seq is: AGATCCCCTAAATCGCCCCA\n",
      "0 AGATCCCCTAAATCGCCCCA motifs detected in file\n",
      "\n",
      "Seq is: CTACGGACAACCCCCGGTCA\n",
      "0 CTACGGACAACCCCCGGTCA motifs detected in file\n",
      "\n",
      "Seq is: CGGGAGGATTAACCCCCCAT\n",
      "0 CGGGAGGATTAACCCCCCAT motifs detected in file\n",
      "\n",
      "Seq is: CCCCGATCCGACCTCACCAT\n",
      "0 CCCCGATCCGACCTCACCAT motifs detected in file\n",
      "\n",
      "Seq is: TTTCCGAGTCTTTTTAGGAC\n",
      "0 TTTCCGAGTCTTTTTAGGAC motifs detected in file\n",
      "\n",
      "Seq is: GCTTCTTTTTTTGAAGACTC\n",
      "0 GCTTCTTTTTTTGAAGACTC motifs detected in file\n",
      "\n",
      "Seq is: CATTATTTATCCTAATAGGG\n",
      "0 CATTATTTATCCTAATAGGG motifs detected in file\n",
      "\n",
      "Seq is: CATAGCTTCCGGAAAAACCT\n",
      "0 CATAGCTTCCGGAAAAACCT motifs detected in file\n",
      "\n",
      "Seq is: GTCCACCACACACCACCGGA\n",
      "0 GTCCACCACACACCACCGGA motifs detected in file\n",
      "\n",
      "Seq is: ACCATACACGAAAGAGCACA\n",
      "0 ACCATACACGAAAGAGCACA motifs detected in file\n",
      "\n",
      "Seq is: ATGTAGCGCGGTAGTAACCA\n",
      "0 ATGTAGCGCGGTAGTAACCA motifs detected in file\n",
      "\n",
      "Seq is: TATACCAATCACACAACCAA\n",
      "0 TATACCAATCACACAACCAA motifs detected in file\n",
      "\n",
      "Seq is: TCATCCGGATCATACTCCTC\n",
      "0 TCATCCGGATCATACTCCTC motifs detected in file\n",
      "\n",
      "Seq is: GCAATACCTCACCTTCACTT\n",
      "0 GCAATACCTCACCTTCACTT motifs detected in file\n",
      "\n",
      "Seq is: TAGTGTACCGATCCGGCCTC\n",
      "0 TAGTGTACCGATCCGGCCTC motifs detected in file\n",
      "\n",
      "Seq is: CAGTAATCCTCCCGACTCTC\n",
      "0 CAGTAATCCTCCCGACTCTC motifs detected in file\n",
      "\n",
      "Seq is: CCGGGGACAATCCCCAGTAC\n",
      "0 CCGGGGACAATCCCCAGTAC motifs detected in file\n",
      "\n",
      "Seq is: CCGACCCAAAATGATATACT\n",
      "0 CCGACCCAAAATGATATACT motifs detected in file\n",
      "\n",
      "Seq is: ATCCGTACACTAACCACCCA\n",
      "0 ATCCGTACACTAACCACCCA motifs detected in file\n",
      "\n",
      "Seq is: GTAATACACAACAGCACGTC\n",
      "0 GTAATACACAACAGCACGTC motifs detected in file\n",
      "\n",
      "Seq is: CATCTCCGAATGATCTTCAC\n",
      "0 CATCTCCGAATGATCTTCAC motifs detected in file\n",
      "\n",
      "Seq is: ACTTTTGCATCCGAACCTAA\n",
      "0 ACTTTTGCATCCGAACCTAA motifs detected in file\n",
      "\n",
      "Seq is: TTCCGCTGTCGCTAAAGATC\n",
      "0 TTCCGCTGTCGCTAAAGATC motifs detected in file\n",
      "\n",
      "Seq is: CTATCAGTCATCTTAATCTT\n",
      "0 CTATCAGTCATCTTAATCTT motifs detected in file\n",
      "\n",
      "Seq is: AACACTTCTACTATTCACAT\n",
      "0 AACACTTCTACTATTCACAT motifs detected in file\n",
      "\n",
      "Seq is: CTCCCTTCCAATTACCAACT\n",
      "0 CTCCCTTCCAATTACCAACT motifs detected in file\n",
      "\n",
      "Seq is: ATAACGATCCCACCGCGAAG\n",
      "0 ATAACGATCCCACCGCGAAG motifs detected in file\n",
      "\n",
      "Seq is: GTTAATCCACGTACTCATCC\n",
      "0 GTTAATCCACGTACTCATCC motifs detected in file\n",
      "\n",
      "Seq is: ACCGGACGTCATTACAATCG\n",
      "0 ACCGGACGTCATTACAATCG motifs detected in file\n",
      "\n",
      "Seq is: CCAATCCGCATGCCGGTCCC\n",
      "0 CCAATCCGCATGCCGGTCCC motifs detected in file\n",
      "\n",
      "Seq is: GATAACCAACTTACTCATCC\n",
      "0 GATAACCAACTTACTCATCC motifs detected in file\n",
      "\n",
      "Seq is: GACTACCAAAGCTATTATTG\n",
      "0 GACTACCAAAGCTATTATTG motifs detected in file\n",
      "\n",
      "Seq is: ATCATACCCCTATTCCCCAC\n",
      "0 ATCATACCCCTATTCCCCAC motifs detected in file\n",
      "\n",
      "Seq is: ATCCACACGGAACACCATTC\n",
      "0 ATCCACACGGAACACCATTC motifs detected in file\n",
      "\n",
      "Seq is: TTCACCCGATCCCGTAAAAA\n",
      "0 TTCACCCGATCCCGTAAAAA motifs detected in file\n",
      "\n",
      "Seq is: TTAGAATCTCGCTTTCGGAT\n",
      "0 TTAGAATCTCGCTTTCGGAT motifs detected in file\n",
      "\n",
      "Seq is: ATTAGTGACACGGGCGAGTA\n",
      "0 ATTAGTGACACGGGCGAGTA motifs detected in file\n",
      "\n",
      "Seq is: TTCCCCTACCGGTACCGATC\n",
      "0 TTCCCCTACCGGTACCGATC motifs detected in file\n",
      "\n",
      "Seq is: CAAATATCTATCAACCCACC\n",
      "0 CAAATATCTATCAACCCACC motifs detected in file\n",
      "\n",
      "Seq is: AACCACATTTACTCACTCCG\n",
      "0 AACCACATTTACTCACTCCG motifs detected in file\n",
      "\n",
      "Seq is: TCCTCAGGCTCCTCCAATCA\n",
      "0 TCCTCAGGCTCCTCCAATCA motifs detected in file\n",
      "\n",
      "Seq is: ACACCGTTATTTTTACTAAT\n",
      "0 ACACCGTTATTTTTACTAAT motifs detected in file\n",
      "\n",
      "Seq is: TCCTATGATCATATTCTCTA\n",
      "0 TCCTATGATCATATTCTCTA motifs detected in file\n",
      "\n",
      "Seq is: GTCCAAGCAGGAAATCACAA\n",
      "0 GTCCAAGCAGGAAATCACAA motifs detected in file\n",
      "\n",
      "Seq is: CACATACCAATAGTAAACAA\n",
      "0 CACATACCAATAGTAAACAA motifs detected in file\n",
      "\n",
      "Seq is: AACTCCAATCAAACTAATCA\n",
      "0 AACTCCAATCAAACTAATCA motifs detected in file\n",
      "\n",
      "Seq is: GTAACAACCCACCACTAATC\n",
      "0 GTAACAACCCACCACTAATC motifs detected in file\n",
      "\n",
      "Seq is: AGCCAACAACTACTCTATAA\n",
      "0 AGCCAACAACTACTCTATAA motifs detected in file\n",
      "\n",
      "Seq is: ACCTCCACCCCTAGTTATCT\n",
      "0 ACCTCCACCCCTAGTTATCT motifs detected in file\n",
      "\n",
      "Seq is: CCCCCTTTATCTTACTAGTC\n",
      "0 CCCCCTTTATCTTACTAGTC motifs detected in file\n",
      "\n",
      "Seq is: ATGACGCCGCCCATCCGGAT\n",
      "0 ATGACGCCGCCCATCCGGAT motifs detected in file\n",
      "\n",
      "Seq is: CCTAACACCCCCGTTACTTA\n",
      "0 CCTAACACCCCCGTTACTTA motifs detected in file\n",
      "\n",
      "Seq is: CTTCGCTTGTCTAAAAGCAA\n",
      "0 CTTCGCTTGTCTAAAAGCAA motifs detected in file\n",
      "\n",
      "Seq is: GTAAAACCAAGAGTCCCAAA\n",
      "0 GTAAAACCAAGAGTCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: CAATATTAAAAAATAAAAAT\n",
      "0 CAATATTAAAAAATAAAAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCCGAAACCACTCCCTCCA\n",
      "0 ACCCGAAACCACTCCCTCCA motifs detected in file\n",
      "\n",
      "Seq is: TCCACCATCAAACACAAATT\n",
      "0 TCCACCATCAAACACAAATT motifs detected in file\n",
      "\n",
      "Seq is: ATAAAAATCAACCCACTACT\n",
      "0 ATAAAAATCAACCCACTACT motifs detected in file\n",
      "\n",
      "Seq is: CCTTATCACATTCCTCATAC\n",
      "0 CCTTATCACATTCCTCATAC motifs detected in file\n",
      "\n",
      "Seq is: CCCCATTAATACCACCCGGT\n",
      "0 CCCCATTAATACCACCCGGT motifs detected in file\n",
      "\n",
      "Seq is: ATGCCATCATAAATCAACCC\n",
      "0 ATGCCATCATAAATCAACCC motifs detected in file\n",
      "\n",
      "Seq is: CGTAAAGTGACATTTCTCCA\n",
      "0 CGTAAAGTGACATTTCTCCA motifs detected in file\n",
      "\n",
      "Seq is: CAACCAAGAGAATTAGAAAT\n",
      "0 CAACCAAGAGAATTAGAAAT motifs detected in file\n",
      "\n",
      "Seq is: TGAATTTTCCAATTACGATT\n",
      "0 TGAATTTTCCAATTACGATT motifs detected in file\n",
      "\n",
      "Seq is: CAATCGAAATGTCACCCGAG\n",
      "0 CAATCGAAATGTCACCCGAG motifs detected in file\n",
      "\n",
      "Seq is: ATCTCCCCCATCTCCCCCAC\n",
      "0 ATCTCCCCCATCTCCCCCAC motifs detected in file\n",
      "\n",
      "Seq is: GATATCCCATTTATGCCCGG\n",
      "0 GATATCCCATTTATGCCCGG motifs detected in file\n",
      "\n",
      "Seq is: GATAAAGTTTCTAAAAATCC\n",
      "0 GATAAAGTTTCTAAAAATCC motifs detected in file\n",
      "\n",
      "Seq is: CCTTAATTAAGATCCTGCTA\n",
      "0 CCTTAATTAAGATCCTGCTA motifs detected in file\n",
      "\n",
      "Seq is: CCCGTACTTTGACACCAAAC\n",
      "0 CCCGTACTTTGACACCAAAC motifs detected in file\n",
      "\n",
      "Seq is: GAGGTGTCTAAAGTCTCGTA\n",
      "0 GAGGTGTCTAAAGTCTCGTA motifs detected in file\n",
      "\n",
      "Seq is: ACTGGCATCATATGGGGGCC\n",
      "0 ACTGGCATCATATGGGGGCC motifs detected in file\n",
      "\n",
      "Seq is: AGCACATCGCCACTTTCACC\n",
      "0 AGCACATCGCCACTTTCACC motifs detected in file\n",
      "\n",
      "Seq is: AAACCAAATCTGCAGGCCCT\n",
      "0 AAACCAAATCTGCAGGCCCT motifs detected in file\n",
      "\n",
      "Seq is: TAACGTAGACAAAAATTCGG\n",
      "0 TAACGTAGACAAAAATTCGG motifs detected in file\n",
      "\n",
      "Seq is: ATTACACCCCTGTCGAGTAC\n",
      "0 ATTACACCCCTGTCGAGTAC motifs detected in file\n",
      "\n",
      "Seq is: TCACGTTCTGCAGAACACTA\n",
      "0 TCACGTTCTGCAGAACACTA motifs detected in file\n",
      "\n",
      "Seq is: CATTAATAATATGCTTACCC\n",
      "0 CATTAATAATATGCTTACCC motifs detected in file\n",
      "\n",
      "Seq is: CCGAAGTTAGCCCTCATGAT\n",
      "0 CCGAAGTTAGCCCTCATGAT motifs detected in file\n",
      "\n",
      "Seq is: GAGCTAACAGTTGCAGTTCC\n",
      "0 GAGCTAACAGTTGCAGTTCC motifs detected in file\n",
      "\n",
      "Seq is: TCAGCGTCCAGCGGACCAAG\n",
      "0 TCAGCGTCCAGCGGACCAAG motifs detected in file\n",
      "\n",
      "Seq is: ATCCTTATTACCCCCTTCAT\n",
      "0 ATCCTTATTACCCCCTTCAT motifs detected in file\n",
      "\n",
      "Seq is: ACATCCTCAACTTCTAATCA\n",
      "0 ACATCCTCAACTTCTAATCA motifs detected in file\n",
      "\n",
      "Seq is: GGCGGCATCAGCCACATGAG\n",
      "0 GGCGGCATCAGCCACATGAG motifs detected in file\n",
      "\n",
      "Seq is: CATCCAAGTCATGGTAACCA\n",
      "0 CATCCAAGTCATGGTAACCA motifs detected in file\n",
      "\n",
      "Seq is: CCGGTTAACTAAACTACCAT\n",
      "0 CCGGTTAACTAAACTACCAT motifs detected in file\n",
      "\n",
      "Seq is: TCCCTCCCTAGCAACTGGAG\n",
      "0 TCCCTCCCTAGCAACTGGAG motifs detected in file\n",
      "\n",
      "Seq is: CAGACAATACATTTCCTACG\n",
      "0 CAGACAATACATTTCCTACG motifs detected in file\n",
      "\n",
      "Seq is: CATCCCTACCCTCCCGCTAC\n",
      "0 CATCCCTACCCTCCCGCTAC motifs detected in file\n",
      "\n",
      "Seq is: TCCTGATCCTACTACCGCCC\n",
      "0 TCCTGATCCTACTACCGCCC motifs detected in file\n",
      "\n",
      "Seq is: GTCCTATCAAGTCTGCCAAA\n",
      "0 GTCCTATCAAGTCTGCCAAA motifs detected in file\n",
      "\n",
      "Seq is: GATAAAGGACTCGCAGACTC\n",
      "0 GATAAAGGACTCGCAGACTC motifs detected in file\n",
      "\n",
      "Seq is: TACAATCATAATCAATCAAA\n",
      "0 TACAATCATAATCAATCAAA motifs detected in file\n",
      "\n",
      "Seq is: ACAACACTCACAATCCTTTT\n",
      "0 ACAACACTCACAATCCTTTT motifs detected in file\n",
      "\n",
      "Seq is: CCCGTATGTCCTGATCCTTC\n",
      "0 CCCGTATGTCCTGATCCTTC motifs detected in file\n",
      "\n",
      "Seq is: GTCTATTCCTTTTACTAATA\n",
      "0 GTCTATTCCTTTTACTAATA motifs detected in file\n",
      "\n",
      "Seq is: CTCCCGCACTAGTACTTTCC\n",
      "0 CTCCCGCACTAGTACTTTCC motifs detected in file\n",
      "\n",
      "Seq is: ACTATTCGAGAAGATACTAT\n",
      "0 ACTATTCGAGAAGATACTAT motifs detected in file\n",
      "\n",
      "Seq is: CCCCTTCATCGCAGAACATC\n",
      "0 CCCCTTCATCGCAGAACATC motifs detected in file\n",
      "\n",
      "Seq is: TGGATGAACGCGACGTACAC\n",
      "0 TGGATGAACGCGACGTACAC motifs detected in file\n",
      "\n",
      "Seq is: GGTAATTCTATATATCCTAA\n",
      "0 GGTAATTCTATATATCCTAA motifs detected in file\n",
      "\n",
      "Seq is: ATCGGATATTAAATTGAAAC\n",
      "0 ATCGGATATTAAATTGAAAC motifs detected in file\n",
      "\n",
      "Seq is: TGTTTCAATACTTTACCAAA\n",
      "0 TGTTTCAATACTTTACCAAA motifs detected in file\n",
      "\n",
      "Seq is: AAGATTATGGAAAAACTTTT\n",
      "0 AAGATTATGGAAAAACTTTT motifs detected in file\n",
      "\n",
      "Seq is: TCAGTACCTCCGGTACCCCA\n",
      "0 TCAGTACCTCCGGTACCCCA motifs detected in file\n",
      "\n",
      "Seq is: ACCGAACTTTGGTCGAAACC\n",
      "0 ACCGAACTTTGGTCGAAACC motifs detected in file\n",
      "\n",
      "Seq is: CCCCAAGCTAAGGAAGGAAA\n",
      "0 CCCCAAGCTAAGGAAGGAAA motifs detected in file\n",
      "\n",
      "Seq is: AAACAGATCTAAAATACATA\n",
      "0 AAACAGATCTAAAATACATA motifs detected in file\n",
      "\n",
      "Seq is: TGCCCAAGAAGCTTACACAC\n",
      "0 TGCCCAAGAAGCTTACACAC motifs detected in file\n",
      "\n",
      "Seq is: CATCCCACCCCCCGTAGGTA\n",
      "0 CATCCCACCCCCCGTAGGTA motifs detected in file\n",
      "\n",
      "Seq is: TATCAGTGAGGTCCAAATAC\n",
      "0 TATCAGTGAGGTCCAAATAC motifs detected in file\n",
      "\n",
      "Seq is: CTCCCAAGAAGATGATAATC\n",
      "0 CTCCCAAGAAGATGATAATC motifs detected in file\n",
      "\n",
      "Seq is: CTGAAAAGCGAAGCTTCGCT\n",
      "0 CTGAAAAGCGAAGCTTCGCT motifs detected in file\n",
      "\n",
      "Seq is: TCCGAAGAGTTTAGTACTTT\n",
      "0 TCCGAAGAGTTTAGTACTTT motifs detected in file\n",
      "\n",
      "Seq is: TAATAATTATAATGACGACA\n",
      "0 TAATAATTATAATGACGACA motifs detected in file\n",
      "\n",
      "Seq is: ATCTCTTTACTTACTCGGAT\n",
      "0 ATCTCTTTACTTACTCGGAT motifs detected in file\n",
      "\n",
      "Seq is: GTCTACTATCCTACAAAGTA\n",
      "0 GTCTACTATCCTACAAAGTA motifs detected in file\n",
      "\n",
      "Seq is: CACCACATACGTAGCCCCAT\n",
      "0 CACCACATACGTAGCCCCAT motifs detected in file\n",
      "\n",
      "Seq is: CAGGCTCATTGCAGCCCCGT\n",
      "0 CAGGCTCATTGCAGCCCCGT motifs detected in file\n",
      "\n",
      "Seq is: AAGGCCTATCCGGCTCTTTC\n",
      "0 AAGGCCTATCCGGCTCTTTC motifs detected in file\n",
      "\n",
      "Seq is: ACAACACCCTTCTTTCAATC\n",
      "0 ACAACACCCTTCTTTCAATC motifs detected in file\n",
      "\n",
      "Seq is: TAAATGCGGCTACTTATACT\n",
      "0 TAAATGCGGCTACTTATACT motifs detected in file\n",
      "\n",
      "Seq is: ATCACTTTACCTAAAACCGC\n",
      "0 ATCACTTTACCTAAAACCGC motifs detected in file\n",
      "\n",
      "Seq is: ATCCAAACCAGATCCCACAT\n",
      "0 ATCCAAACCAGATCCCACAT motifs detected in file\n",
      "\n",
      "Seq is: CGGACTCTTATCCCCTTTAG\n",
      "0 CGGACTCTTATCCCCTTTAG motifs detected in file\n",
      "\n",
      "Seq is: TCACTTACTTCGGAGGATAC\n",
      "0 TCACTTACTTCGGAGGATAC motifs detected in file\n",
      "\n",
      "Seq is: TACCGTTTATGTCGAGGATA\n",
      "0 TACCGTTTATGTCGAGGATA motifs detected in file\n",
      "\n",
      "Seq is: ACTATCCTGTATCACCTTCA\n",
      "0 ACTATCCTGTATCACCTTCA motifs detected in file\n",
      "\n",
      "Seq is: CCCGATGTTGCATCATGCAC\n",
      "0 CCCGATGTTGCATCATGCAC motifs detected in file\n",
      "\n",
      "Seq is: AGCACATCATGCTACAGATC\n",
      "0 AGCACATCATGCTACAGATC motifs detected in file\n",
      "\n",
      "Seq is: ACTACTCAAACGATTATGTT\n",
      "0 ACTACTCAAACGATTATGTT motifs detected in file\n",
      "\n",
      "Seq is: ACGGTCAGTCCGGTGGATGC\n",
      "0 ACGGTCAGTCCGGTGGATGC motifs detected in file\n",
      "\n",
      "Seq is: CACTTTTCTTTCTACTTAGG\n",
      "0 CACTTTTCTTTCTACTTAGG motifs detected in file\n",
      "\n",
      "Seq is: ATCCCGAGTCTCGTGACGTC\n",
      "0 ATCCCGAGTCTCGTGACGTC motifs detected in file\n",
      "\n",
      "Seq is: GTCTAGTAAAGTATAACGAA\n",
      "0 GTCTAGTAAAGTATAACGAA motifs detected in file\n",
      "\n",
      "Seq is: GGCACCTCACACCGCTCAGT\n",
      "0 GGCACCTCACACCGCTCAGT motifs detected in file\n",
      "\n",
      "Seq is: CGATTTATGAAACTGCGGCC\n",
      "0 CGATTTATGAAACTGCGGCC motifs detected in file\n",
      "\n",
      "Seq is: ACCCCTATCGCTACTAATAC\n",
      "0 ACCCCTATCGCTACTAATAC motifs detected in file\n",
      "\n",
      "Seq is: CATCGCCTCCACTTTATACG\n",
      "0 CATCGCCTCCACTTTATACG motifs detected in file\n",
      "\n",
      "Seq is: AGCACACAGATGCAGATAAG\n",
      "0 AGCACACAGATGCAGATAAG motifs detected in file\n",
      "\n",
      "Seq is: GATGACATTTATATACCACA\n",
      "0 GATGACATTTATATACCACA motifs detected in file\n",
      "\n",
      "Seq is: CGAGTGTGCTATTTGGGATC\n",
      "0 CGAGTGTGCTATTTGGGATC motifs detected in file\n",
      "\n",
      "Seq is: CTTCGGTTAACTATAGTATC\n",
      "0 CTTCGGTTAACTATAGTATC motifs detected in file\n",
      "\n",
      "Seq is: GAGTCTGGTATGGATACATA\n",
      "0 GAGTCTGGTATGGATACATA motifs detected in file\n",
      "\n",
      "Seq is: GGTTTACCAAGAAAAAAAGG\n",
      "0 GGTTTACCAAGAAAAAAAGG motifs detected in file\n",
      "\n",
      "Seq is: CCTCATCATTCAATGTTATA\n",
      "0 CCTCATCATTCAATGTTATA motifs detected in file\n",
      "\n",
      "Seq is: CCCTCTAATAAGGCTTCGGA\n",
      "0 CCCTCTAATAAGGCTTCGGA motifs detected in file\n",
      "\n",
      "Seq is: CCATCCTATTCTTATATTTG\n",
      "0 CCATCCTATTCTTATATTTG motifs detected in file\n",
      "\n",
      "Seq is: AAGTCCCACTGGCTTTTTAG\n",
      "0 AAGTCCCACTGGCTTTTTAG motifs detected in file\n",
      "\n",
      "Seq is: TCTTATCCACAACCATATCT\n",
      "0 TCTTATCCACAACCATATCT motifs detected in file\n",
      "\n",
      "Seq is: TACCCCAGAGGAGGAGGCCG\n",
      "0 TACCCCAGAGGAGGAGGCCG motifs detected in file\n",
      "\n",
      "Seq is: CCCCAGCTTCTTCCACCACA\n",
      "0 CCCCAGCTTCTTCCACCACA motifs detected in file\n",
      "\n",
      "Seq is: ACTCCAACGCCAGACAATCA\n",
      "0 ACTCCAACGCCAGACAATCA motifs detected in file\n",
      "\n",
      "Seq is: TCATATCACTACGGTCGTCG\n",
      "0 TCATATCACTACGGTCGTCG motifs detected in file\n",
      "\n",
      "Seq is: ATCCTGACCCTCTCTATCCT\n",
      "0 ATCCTGACCCTCTCTATCCT motifs detected in file\n",
      "\n",
      "Seq is: CTTCATCCTGACGACACTAA\n",
      "0 CTTCATCCTGACGACACTAA motifs detected in file\n",
      "\n",
      "Seq is: TCCTGCCTAGTCTGCTTCTC\n",
      "0 TCCTGCCTAGTCTGCTTCTC motifs detected in file\n",
      "\n",
      "Seq is: CCCGCAAACCATAACCCAAT\n",
      "0 CCCGCAAACCATAACCCAAT motifs detected in file\n",
      "\n",
      "Seq is: ACCGTCCCCCAAAATATAAC\n",
      "0 ACCGTCCCCCAAAATATAAC motifs detected in file\n",
      "\n",
      "Seq is: TATTAACAACACTACTTTAA\n",
      "0 TATTAACAACACTACTTTAA motifs detected in file\n",
      "\n",
      "Seq is: CTACCGGGGATTCTATCTCC\n",
      "0 CTACCGGGGATTCTATCTCC motifs detected in file\n",
      "\n",
      "Seq is: TCTGTGGACGATCCACATTC\n",
      "0 TCTGTGGACGATCCACATTC motifs detected in file\n",
      "\n",
      "Seq is: CTCTTCTACCAATCCAGATG\n",
      "0 CTCTTCTACCAATCCAGATG motifs detected in file\n",
      "\n",
      "Seq is: CCTCCGAGGTCCCACCCTCA\n",
      "0 CCTCCGAGGTCCCACCCTCA motifs detected in file\n",
      "\n",
      "Seq is: TCAAGGGACGATTCCCTCCC\n",
      "0 TCAAGGGACGATTCCCTCCC motifs detected in file\n",
      "\n",
      "Seq is: ATCTGACAAGTTGGACAAGG\n",
      "0 ATCTGACAAGTTGGACAAGG motifs detected in file\n",
      "\n",
      "Seq is: ACGAGGCCGGAGGTGATATC\n",
      "0 ACGAGGCCGGAGGTGATATC motifs detected in file\n",
      "\n",
      "Seq is: GTCTACGCTCGTCCTCATCC\n",
      "0 GTCTACGCTCGTCCTCATCC motifs detected in file\n",
      "\n",
      "Seq is: TCTCTCCCTCCATTCTCAGT\n",
      "0 TCTCTCCCTCCATTCTCAGT motifs detected in file\n",
      "\n",
      "Seq is: CTTCGAATACAACAAATACG\n",
      "0 CTTCGAATACAACAAATACG motifs detected in file\n",
      "\n",
      "Seq is: CCCCTTTGCGGTATAGCCCC\n",
      "0 CCCCTTTGCGGTATAGCCCC motifs detected in file\n",
      "\n",
      "Seq is: CGTGGCTAATAATCCCCTTG\n",
      "0 CGTGGCTAATAATCCCCTTG motifs detected in file\n",
      "\n",
      "Seq is: ATCAGTCAACGGTTTCGGAG\n",
      "0 ATCAGTCAACGGTTTCGGAG motifs detected in file\n",
      "\n",
      "Seq is: GCTAATACTACCCATAATGA\n",
      "0 GCTAATACTACCCATAATGA motifs detected in file\n",
      "\n",
      "Seq is: TACTTCTTCTAATAATGTTT\n",
      "0 TACTTCTTCTAATAATGTTT motifs detected in file\n",
      "\n",
      "Seq is: ACGTACCCGACACTGCTATT\n",
      "0 ACGTACCCGACACTGCTATT motifs detected in file\n",
      "\n",
      "Seq is: GCAACATCTACACCAGCAAT\n",
      "0 GCAACATCTACACCAGCAAT motifs detected in file\n",
      "\n",
      "Seq is: GGATCTTCCAACGGACCGAC\n",
      "0 GGATCTTCCAACGGACCGAC motifs detected in file\n",
      "\n",
      "Seq is: CGGGTCGAGCCGAGCTTATT\n",
      "0 CGGGTCGAGCCGAGCTTATT motifs detected in file\n",
      "\n",
      "Seq is: CCTCCGAATCTCGACACGGA\n",
      "0 CCTCCGAATCTCGACACGGA motifs detected in file\n",
      "\n",
      "Seq is: TCCTGAGGTCGAGTACGCGG\n",
      "0 TCCTGAGGTCGAGTACGCGG motifs detected in file\n",
      "\n",
      "Seq is: CTTATTATCCATATCACAAG\n",
      "0 CTTATTATCCATATCACAAG motifs detected in file\n",
      "\n",
      "Seq is: GTTACAGAAACACCAAACAT\n",
      "0 GTTACAGAAACACCAAACAT motifs detected in file\n",
      "\n",
      "Seq is: CTCTTATCAGTTGCCAGCCG\n",
      "0 CTCTTATCAGTTGCCAGCCG motifs detected in file\n",
      "\n",
      "Seq is: CTTGTAGTCACCCCCACTCC\n",
      "0 CTTGTAGTCACCCCCACTCC motifs detected in file\n",
      "\n",
      "Seq is: ATTTTACCGACTCACTTCGT\n",
      "0 ATTTTACCGACTCACTTCGT motifs detected in file\n",
      "\n",
      "Seq is: AACCTGACATTTAGATTTCT\n",
      "0 AACCTGACATTTAGATTTCT motifs detected in file\n",
      "\n",
      "Seq is: GTCCCCAATCCGGAGAAAAA\n",
      "0 GTCCCCAATCCGGAGAAAAA motifs detected in file\n",
      "\n",
      "Seq is: TGGTCGAGGCTCCACTAAAA\n",
      "0 TGGTCGAGGCTCCACTAAAA motifs detected in file\n",
      "\n",
      "Seq is: GTATAACTTAACGTTTAAGC\n",
      "0 GTATAACTTAACGTTTAAGC motifs detected in file\n",
      "\n",
      "Seq is: TTCTTCGTCGAAGTTTGGAC\n",
      "0 TTCTTCGTCGAAGTTTGGAC motifs detected in file\n",
      "\n",
      "Seq is: GGCCCCGAAGAGGGCGGAAA\n",
      "0 GGCCCCGAAGAGGGCGGAAA motifs detected in file\n",
      "\n",
      "Seq is: AAAGGGCCGCCGCCCTCTTC\n",
      "0 AAAGGGCCGCCGCCCTCTTC motifs detected in file\n",
      "\n",
      "Seq is: ATCTAACTTCGGTCAACTAA\n",
      "0 ATCTAACTTCGGTCAACTAA motifs detected in file\n",
      "\n",
      "Seq is: TCCCACGAATCGACAATTGA\n",
      "0 TCCCACGAATCGACAATTGA motifs detected in file\n",
      "\n",
      "Seq is: TTCACAAACACCCAAATTCA\n",
      "0 TTCACAAACACCCAAATTCA motifs detected in file\n",
      "\n",
      "Seq is: GGGTAACCAGATCATTCCCG\n",
      "0 GGGTAACCAGATCATTCCCG motifs detected in file\n",
      "\n",
      "Seq is: AATCGAATTAATTTCACCGA\n",
      "0 AATCGAATTAATTTCACCGA motifs detected in file\n",
      "\n",
      "Seq is: CTAAACGCAAGTCAACTACG\n",
      "0 CTAAACGCAAGTCAACTACG motifs detected in file\n",
      "\n",
      "Seq is: TCTCACCCCAAAACGTCAGG\n",
      "0 TCTCACCCCAAAACGTCAGG motifs detected in file\n",
      "\n",
      "Seq is: AATCGACAATGTCTTTAATT\n",
      "0 AATCGACAATGTCTTTAATT motifs detected in file\n",
      "\n",
      "Seq is: CATAACGTTGAATGACTCCC\n",
      "0 CATAACGTTGAATGACTCCC motifs detected in file\n",
      "\n",
      "Seq is: GAAACTTCCGAGAACCAGAC\n",
      "0 GAAACTTCCGAGAACCAGAC motifs detected in file\n",
      "\n",
      "Seq is: ATAAATTGGATTTAAAGATA\n",
      "0 ATAAATTGGATTTAAAGATA motifs detected in file\n",
      "\n",
      "Seq is: TTCTAATAATCATATTTTCC\n",
      "0 TTCTAATAATCATATTTTCC motifs detected in file\n",
      "\n",
      "Seq is: CCTCTATCCATCCTCATCGC\n",
      "0 CCTCTATCCATCCTCATCGC motifs detected in file\n",
      "\n",
      "Seq is: ACCATTCCCGCTACTCACAC\n",
      "0 ACCATTCCCGCTACTCACAC motifs detected in file\n",
      "\n",
      "Seq is: CCCTCCTTACCCCACCCAAA\n",
      "0 CCCTCCTTACCCCACCCAAA motifs detected in file\n",
      "\n",
      "Seq is: ACATACAAGTTTGACAGTAA\n",
      "0 ACATACAAGTTTGACAGTAA motifs detected in file\n",
      "\n",
      "Seq is: AATAAAAATGCAACAATCTA\n",
      "0 AATAAAAATGCAACAATCTA motifs detected in file\n",
      "\n",
      "Seq is: TACCCCTCATCACACTAACT\n",
      "0 TACCCCTCATCACACTAACT motifs detected in file\n",
      "\n",
      "Seq is: CCACCTCATCTAATCCGCAT\n",
      "0 CCACCTCATCTAATCCGCAT motifs detected in file\n",
      "\n",
      "Seq is: CCATCTTCATCTCCAATTCC\n",
      "0 CCATCTTCATCTCCAATTCC motifs detected in file\n",
      "\n",
      "Seq is: TCCCACTACCACCGATACTA\n",
      "0 TCCCACTACCACCGATACTA motifs detected in file\n",
      "\n",
      "Seq is: CCACCCCTACTACTCCGATA\n",
      "0 CCACCCCTACTACTCCGATA motifs detected in file\n",
      "\n",
      "Seq is: ACAAAAAACACTTAAGAAGC\n",
      "0 ACAAAAAACACTTAAGAAGC motifs detected in file\n",
      "\n",
      "Seq is: TATTACCGGGTAAACCCGTT\n",
      "0 TATTACCGGGTAAACCCGTT motifs detected in file\n",
      "\n",
      "Seq is: TTTCGGCCAATCGCCCCCGT\n",
      "0 TTTCGGCCAATCGCCCCCGT motifs detected in file\n",
      "\n",
      "Seq is: CCGGAGGATCCCTCTCCTCC\n",
      "0 CCGGAGGATCCCTCTCCTCC motifs detected in file\n",
      "\n",
      "Seq is: CACCTACCTTAATTCCCACA\n",
      "0 CACCTACCTTAATTCCCACA motifs detected in file\n",
      "\n",
      "Seq is: ATCAGTACAATCGAACAAAG\n",
      "0 ATCAGTACAATCGAACAAAG motifs detected in file\n",
      "\n",
      "Seq is: TCCACGCTCTATCATCATCC\n",
      "0 TCCACGCTCTATCATCATCC motifs detected in file\n",
      "\n",
      "Seq is: CAGCACCACGACCTCAAATT\n",
      "0 CAGCACCACGACCTCAAATT motifs detected in file\n",
      "\n",
      "Seq is: CAACTCATCATCCTTACGCC\n",
      "0 CAACTCATCATCCTTACGCC motifs detected in file\n",
      "\n",
      "Seq is: ATCATCAATCCTATTATATT\n",
      "0 ATCATCAATCCTATTATATT motifs detected in file\n",
      "\n",
      "Seq is: TATCAATTTAATTCTTACCA\n",
      "0 TATCAATTTAATTCTTACCA motifs detected in file\n",
      "\n",
      "Seq is: ATACAATCCCAACATGCCAT\n",
      "0 ATACAATCCCAACATGCCAT motifs detected in file\n",
      "\n",
      "Seq is: CTTGACGATAATAAGTAGGA\n",
      "0 CTTGACGATAATAAGTAGGA motifs detected in file\n",
      "\n",
      "Seq is: TACACCCATTAACTCCTCAT\n",
      "0 TACACCCATTAACTCCTCAT motifs detected in file\n",
      "\n",
      "Seq is: ACGATTCTAAAACGCATCGA\n",
      "0 ACGATTCTAAAACGCATCGA motifs detected in file\n",
      "\n",
      "Seq is: CCCAAACCAAATTAGGTGGA\n",
      "0 CCCAAACCAAATTAGGTGGA motifs detected in file\n",
      "\n",
      "Seq is: GTTGACGGACGATACTACCT\n",
      "0 GTTGACGGACGATACTACCT motifs detected in file\n",
      "\n",
      "Seq is: ATTCTAACTCTCTCACTCCT\n",
      "0 ATTCTAACTCTCTCACTCCT motifs detected in file\n",
      "\n",
      "Seq is: CTTCCGAATGCAAATCACTC\n",
      "0 CTTCCGAATGCAAATCACTC motifs detected in file\n",
      "\n",
      "Seq is: CCTCTCTAAACCATATACTA\n",
      "0 CCTCTCTAAACCATATACTA motifs detected in file\n",
      "\n",
      "Seq is: ACTCTACCCCCGATCAAAAA\n",
      "0 ACTCTACCCCCGATCAAAAA motifs detected in file\n",
      "\n",
      "Seq is: CAGTACACTCTTCTTCGTCC\n",
      "0 CAGTACACTCTTCTTCGTCC motifs detected in file\n",
      "\n",
      "Seq is: GGCCTACAGTCTCCCCACGG\n",
      "0 GGCCTACAGTCTCCCCACGG motifs detected in file\n",
      "\n",
      "Seq is: AACCCATTGGAGACCCTGAG\n",
      "0 AACCCATTGGAGACCCTGAG motifs detected in file\n",
      "\n",
      "Seq is: TCTTCACTTTCCCCCGATAA\n",
      "0 TCTTCACTTTCCCCCGATAA motifs detected in file\n",
      "\n",
      "Seq is: GGATCAAAATAACGATATCG\n",
      "0 GGATCAAAATAACGATATCG motifs detected in file\n",
      "\n",
      "Seq is: ATAATACTAATAATTACTAC\n",
      "0 ATAATACTAATAATTACTAC motifs detected in file\n",
      "\n",
      "Seq is: TCATAACTAACCATCATAAC\n",
      "0 TCATAACTAACCATCATAAC motifs detected in file\n",
      "\n",
      "Seq is: CAATACCAAGTAACAGGCCT\n",
      "0 CAATACCAAGTAACAGGCCT motifs detected in file\n",
      "\n",
      "Seq is: CTCATATAACAACTTCTCCT\n",
      "0 CTCATATAACAACTTCTCCT motifs detected in file\n",
      "\n",
      "Seq is: ATCGATAATCTTCCTAATAC\n",
      "0 ATCGATAATCTTCCTAATAC motifs detected in file\n",
      "\n",
      "Seq is: CTACGCCAACGAACGCACTC\n",
      "0 CTACGCCAACGAACGCACTC motifs detected in file\n",
      "\n",
      "Seq is: CTTTATGAACTACCGTCGAA\n",
      "0 CTTTATGAACTACCGTCGAA motifs detected in file\n",
      "\n",
      "Seq is: GACACCTTGCTCCCAAATAA\n",
      "0 GACACCTTGCTCCCAAATAA motifs detected in file\n",
      "\n",
      "Seq is: AAAAACCAATCTTGACCTTA\n",
      "0 AAAAACCAATCTTGACCTTA motifs detected in file\n",
      "\n",
      "Seq is: TTTTCGATCGTACAAATAAA\n",
      "0 TTTTCGATCGTACAAATAAA motifs detected in file\n",
      "\n",
      "Seq is: GATCCGGATGAGTCCATTTT\n",
      "0 GATCCGGATGAGTCCATTTT motifs detected in file\n",
      "\n",
      "Seq is: TTAGTCACGCTCGAATCGCG\n",
      "0 TTAGTCACGCTCGAATCGCG motifs detected in file\n",
      "\n",
      "Seq is: ACACTACTCACACGGACGTT\n",
      "0 ACACTACTCACACGGACGTT motifs detected in file\n",
      "\n",
      "Seq is: TCTACCATCTCATCTACTGC\n",
      "0 TCTACCATCTCATCTACTGC motifs detected in file\n",
      "\n",
      "Seq is: CCAACCCGGTCCCCTAATTA\n",
      "0 CCAACCCGGTCCCCTAATTA motifs detected in file\n",
      "\n",
      "Seq is: ATCATGCCCTTCCCATATTG\n",
      "0 ATCATGCCCTTCCCATATTG motifs detected in file\n",
      "\n",
      "Seq is: GTTGTAAAAGCCCCATACCC\n",
      "0 GTTGTAAAAGCCCCATACCC motifs detected in file\n",
      "\n",
      "Seq is: GGGCTATCGAATAAATCGAC\n",
      "0 GGGCTATCGAATAAATCGAC motifs detected in file\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Seq is: TGGAATGAAATCCTACCCCA\n",
      "0 TGGAATGAAATCCTACCCCA motifs detected in file\n",
      "\n",
      "Seq is: CACTATCCACCGTGCCTCTT\n",
      "0 CACTATCCACCGTGCCTCTT motifs detected in file\n",
      "\n",
      "Seq is: AAAACCTAAGAGTCCCTACC\n",
      "0 AAAACCTAAGAGTCCCTACC motifs detected in file\n",
      "\n",
      "Seq is: CAAGCTAAGAGTATCAGGAT\n",
      "0 CAAGCTAAGAGTATCAGGAT motifs detected in file\n",
      "\n",
      "Seq is: CTTTATTCCCCCAAATTCGA\n",
      "0 CTTTATTCCCCCAAATTCGA motifs detected in file\n",
      "\n",
      "Seq is: GGATAATAAATGAGATAGTT\n",
      "0 GGATAATAAATGAGATAGTT motifs detected in file\n",
      "\n",
      "Seq is: TCATTGAGAAAATAGTCTGT\n",
      "0 TCATTGAGAAAATAGTCTGT motifs detected in file\n",
      "\n",
      "Seq is: ATAAAGAATCCAAACTCCCC\n",
      "0 ATAAAGAATCCAAACTCCCC motifs detected in file\n",
      "\n",
      "Seq is: CTTACGACCTCTAACATTAC\n",
      "0 CTTACGACCTCTAACATTAC motifs detected in file\n",
      "\n",
      "Seq is: CCATACCTCTGTATAGTATA\n",
      "0 CCATACCTCTGTATAGTATA motifs detected in file\n",
      "\n",
      "Seq is: TTCATTACGATCCCACTCAC\n",
      "0 TTCATTACGATCCCACTCAC motifs detected in file\n",
      "\n",
      "Seq is: CATCCTTCAAAAAAGTATCC\n",
      "0 CATCCTTCAAAAAAGTATCC motifs detected in file\n",
      "\n",
      "Seq is: TCCACATACTCAACCAGCAT\n",
      "0 TCCACATACTCAACCAGCAT motifs detected in file\n",
      "\n",
      "Seq is: CGCCTTAGCCCCCATACGAC\n",
      "0 CGCCTTAGCCCCCATACGAC motifs detected in file\n",
      "\n",
      "Seq is: AAGCTTAAGTATTCTTGTCC\n",
      "0 AAGCTTAAGTATTCTTGTCC motifs detected in file\n",
      "\n",
      "Seq is: CTCCAATCTTCATCCCAGAA\n",
      "0 CTCCAATCTTCATCCCAGAA motifs detected in file\n",
      "\n",
      "Seq is: CCACTGTTTTATACAACACA\n",
      "0 CCACTGTTTTATACAACACA motifs detected in file\n",
      "\n",
      "Seq is: TCTCAAGTCCCCTCTCACGC\n",
      "0 TCTCAAGTCCCCTCTCACGC motifs detected in file\n",
      "\n",
      "Seq is: AGTATACAACAAGGATCCTT\n",
      "0 AGTATACAACAAGGATCCTT motifs detected in file\n",
      "\n",
      "Seq is: CTAACATCACCACTCCCACA\n",
      "0 CTAACATCACCACTCCCACA motifs detected in file\n",
      "\n",
      "Seq is: AATAATATTATTACAAACAC\n",
      "0 AATAATATTATTACAAACAC motifs detected in file\n",
      "\n",
      "Seq is: ATAAGCCGATACTTCTTATC\n",
      "0 ATAAGCCGATACTTCTTATC motifs detected in file\n",
      "\n",
      "Seq is: CCGCTTCCCCGGACGCCGCA\n",
      "0 CCGCTTCCCCGGACGCCGCA motifs detected in file\n",
      "\n",
      "Seq is: TAAGCTACAACTTCGGACTC\n",
      "0 TAAGCTACAACTTCGGACTC motifs detected in file\n",
      "\n",
      "Seq is: TGATCAAGCCTGAGGGGAAG\n",
      "0 TGATCAAGCCTGAGGGGAAG motifs detected in file\n",
      "\n",
      "Seq is: CCGTTCCAGCTTCCCCCAAG\n",
      "0 CCGTTCCAGCTTCCCCCAAG motifs detected in file\n",
      "\n",
      "Seq is: CCAACCAGAGACGATCACAC\n",
      "0 CCAACCAGAGACGATCACAC motifs detected in file\n",
      "\n",
      "Seq is: CTCTATTTAGTATAATACCG\n",
      "0 CTCTATTTAGTATAATACCG motifs detected in file\n",
      "\n",
      "Seq is: GTTCCCAGTACTACCGTCCT\n",
      "0 GTTCCCAGTACTACCGTCCT motifs detected in file\n",
      "\n",
      "Seq is: CATTAGTCTCCACAAGAACA\n",
      "0 CATTAGTCTCCACAAGAACA motifs detected in file\n",
      "\n",
      "Seq is: CAACACTATTCCCACCTCTC\n",
      "0 CAACACTATTCCCACCTCTC motifs detected in file\n",
      "\n",
      "Seq is: CAATTTCCTCGGTGAATAAT\n",
      "0 CAATTTCCTCGGTGAATAAT motifs detected in file\n",
      "\n",
      "Seq is: CATTACAACTATCATCTTAC\n",
      "0 CATTACAACTATCATCTTAC motifs detected in file\n",
      "\n",
      "Seq is: TACCGATCCCACTGAAGTAT\n",
      "0 TACCGATCCCACTGAAGTAT motifs detected in file\n",
      "\n",
      "Seq is: ACTCTAACAAACCCGATGAC\n",
      "0 ACTCTAACAAACCCGATGAC motifs detected in file\n",
      "\n",
      "Seq is: GAGCGTCACGCGGCTAGTCC\n",
      "0 GAGCGTCACGCGGCTAGTCC motifs detected in file\n",
      "\n",
      "Seq is: CGCATCAAACTCAAACTACG\n",
      "0 CGCATCAAACTCAAACTACG motifs detected in file\n",
      "\n",
      "Seq is: AGTGGGACTAGTCTCCTAAC\n",
      "0 AGTGGGACTAGTCTCCTAAC motifs detected in file\n",
      "\n",
      "Seq is: TCATTTGCCGATCCGATCTC\n",
      "0 TCATTTGCCGATCCGATCTC motifs detected in file\n",
      "\n",
      "Seq is: CACCGATCTTATTTATCCTC\n",
      "0 CACCGATCTTATTTATCCTC motifs detected in file\n",
      "\n",
      "Seq is: CGGATCCAACTCCAACTGGT\n",
      "0 CGGATCCAACTCCAACTGGT motifs detected in file\n",
      "\n",
      "Seq is: CCCCCAACCCATACCCCTCC\n",
      "0 CCCCCAACCCATACCCCTCC motifs detected in file\n",
      "\n",
      "Seq is: CCCCAAGTATCATCTTCTCG\n",
      "0 CCCCAAGTATCATCTTCTCG motifs detected in file\n",
      "\n",
      "Seq is: CTACCACTCTCGATTCCAGC\n",
      "0 CTACCACTCTCGATTCCAGC motifs detected in file\n",
      "\n",
      "Seq is: CCCGCCACTACATCTCCCAC\n",
      "0 CCCGCCACTACATCTCCCAC motifs detected in file\n",
      "\n",
      "Seq is: TACCATCTACACCGCCCAAA\n",
      "0 TACCATCTACACCGCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: ATCCCCGAGAAACCACTTCT\n",
      "0 ATCCCCGAGAAACCACTTCT motifs detected in file\n",
      "\n",
      "Seq is: CAAAATACCGCAGTCGCTTC\n",
      "0 CAAAATACCGCAGTCGCTTC motifs detected in file\n",
      "\n",
      "Seq is: CCAACATCATCGGGCATCCC\n",
      "0 CCAACATCATCGGGCATCCC motifs detected in file\n",
      "\n",
      "Seq is: CGGATGTTGCAACCCCGGAA\n",
      "0 CGGATGTTGCAACCCCGGAA motifs detected in file\n",
      "\n",
      "Seq is: ACGCATCAACATATATCGGA\n",
      "0 ACGCATCAACATATATCGGA motifs detected in file\n",
      "\n",
      "Seq is: TCTTAAAAAGCAAGCCATTC\n",
      "0 TCTTAAAAAGCAAGCCATTC motifs detected in file\n",
      "\n",
      "Seq is: GTAATCCTTACGGTAACGCT\n",
      "0 GTAATCCTTACGGTAACGCT motifs detected in file\n",
      "\n",
      "Seq is: AATCTTACCCATGTTACTCC\n",
      "0 AATCTTACCCATGTTACTCC motifs detected in file\n",
      "\n",
      "Seq is: TCATCCTCCAACCGGTACCC\n",
      "0 TCATCCTCCAACCGGTACCC motifs detected in file\n",
      "\n",
      "Seq is: ATACAACAATTCTTCTCCTT\n",
      "0 ATACAACAATTCTTCTCCTT motifs detected in file\n",
      "\n",
      "Seq is: AACTTGGAGACTGACATTTC\n",
      "0 AACTTGGAGACTGACATTTC motifs detected in file\n",
      "\n",
      "Seq is: AAAATTCAAAATACGCTAAT\n",
      "0 AAAATTCAAAATACGCTAAT motifs detected in file\n",
      "\n",
      "Seq is: GGCCCGAGACGGTAGAATTG\n",
      "0 GGCCCGAGACGGTAGAATTG motifs detected in file\n",
      "\n",
      "Seq is: TTTGGGACAAGAACCCACCC\n",
      "0 TTTGGGACAAGAACCCACCC motifs detected in file\n",
      "\n",
      "Seq is: ACACCCATATTATGATTCAA\n",
      "0 ACACCCATATTATGATTCAA motifs detected in file\n",
      "\n",
      "Seq is: CTCTACTATAGTAAATGCCC\n",
      "0 CTCTACTATAGTAAATGCCC motifs detected in file\n",
      "\n",
      "Seq is: CCTTCCGCGAAACACTTCAT\n",
      "0 CCTTCCGCGAAACACTTCAT motifs detected in file\n",
      "\n",
      "Seq is: CCGGAATAAAGAGAACAGGA\n",
      "0 CCGGAATAAAGAGAACAGGA motifs detected in file\n",
      "\n",
      "Seq is: AAGCATGTCCCTCCTTAAAC\n",
      "0 AAGCATGTCCCTCCTTAAAC motifs detected in file\n",
      "\n",
      "Seq is: TTNCATCTATCTTTGGCTGG\n",
      "0 TTNCATCTATCTTTGGCTGG motifs detected in file\n",
      "\n",
      "Seq is: ACCTAATGAGGCCAGACTTG\n",
      "0 ACCTAATGAGGCCAGACTTG motifs detected in file\n",
      "\n",
      "Seq is: AGTCTAGTGCATCCTGAAAT\n",
      "0 AGTCTAGTGCATCCTGAAAT motifs detected in file\n",
      "\n",
      "Seq is: TAGCAACTTGTTTGCTTGGA\n",
      "0 TAGCAACTTGTTTGCTTGGA motifs detected in file\n",
      "\n",
      "Seq is: AATTATCGCCGACGTGGTAG\n",
      "0 AATTATCGCCGACGTGGTAG motifs detected in file\n",
      "\n",
      "Seq is: CCCTACAGGACTAGGTTGTA\n",
      "0 CCCTACAGGACTAGGTTGTA motifs detected in file\n",
      "\n",
      "Seq is: GCTCCAGCATTTGGGATAAC\n",
      "0 GCTCCAGCATTTGGGATAAC motifs detected in file\n",
      "\n",
      "Seq is: AACTATACCTGAGATCTTAT\n",
      "0 AACTATACCTGAGATCTTAT motifs detected in file\n",
      "\n",
      "Seq is: CCTAACGCGACAATAGGGAT\n",
      "0 CCTAACGCGACAATAGGGAT motifs detected in file\n",
      "\n",
      "Seq is: CCCATTGAACAAGGCAACCA\n",
      "0 CCCATTGAACAAGGCAACCA motifs detected in file\n",
      "\n",
      "Seq is: GTTCAATAACCTAGTTAACT\n",
      "0 GTTCAATAACCTAGTTAACT motifs detected in file\n",
      "\n",
      "Seq is: CATATCATCAAGCGAAACTG\n",
      "0 CATATCATCAAGCGAAACTG motifs detected in file\n",
      "\n",
      "Seq is: ACCACTTCAGAATCGTACAT\n",
      "0 ACCACTTCAGAATCGTACAT motifs detected in file\n",
      "\n",
      "Seq is: GACGAGCCTCCAACCCAAGA\n",
      "0 GACGAGCCTCCAACCCAAGA motifs detected in file\n",
      "\n",
      "Seq is: CGAGGCTCCAGCGGGGTTGG\n",
      "0 CGAGGCTCCAGCGGGGTTGG motifs detected in file\n",
      "\n",
      "Seq is: CTTTAAAAATTACGTCCAAA\n",
      "0 CTTTAAAAATTACGTCCAAA motifs detected in file\n",
      "\n",
      "Seq is: CCATCAAATCCTGGACACCC\n",
      "0 CCATCAAATCCTGGACACCC motifs detected in file\n",
      "\n",
      "Seq is: AAACAATCCATGACAAACGT\n",
      "0 AAACAATCCATGACAAACGT motifs detected in file\n",
      "\n",
      "Seq is: AATTATTTAATTTCGAGGTA\n",
      "0 AATTATTTAATTTCGAGGTA motifs detected in file\n",
      "\n",
      "Seq is: TCCCAGAAGAGCAGAACGAC\n",
      "0 TCCCAGAAGAGCAGAACGAC motifs detected in file\n",
      "\n",
      "Seq is: ACAATACGGGCGGAGAAGTG\n",
      "0 ACAATACGGGCGGAGAAGTG motifs detected in file\n",
      "\n",
      "Seq is: CCCGTCCAGTTAAAGTGACC\n",
      "0 CCCGTCCAGTTAAAGTGACC motifs detected in file\n",
      "\n",
      "Seq is: AATTTTCATTCTCTGTCGAC\n",
      "0 AATTTTCATTCTCTGTCGAC motifs detected in file\n",
      "\n",
      "Seq is: TTGGGAGCACCTCGGTAAGT\n",
      "0 TTGGGAGCACCTCGGTAAGT motifs detected in file\n",
      "\n",
      "Seq is: ATGTCCAGGGATAAATTCCT\n",
      "0 ATGTCCAGGGATAAATTCCT motifs detected in file\n",
      "\n",
      "Seq is: TGTTCACTAATACGATGGAA\n",
      "0 TGTTCACTAATACGATGGAA motifs detected in file\n",
      "\n",
      "Seq is: ACGTGCCAATCCCATGGCGC\n",
      "0 ACGTGCCAATCCCATGGCGC motifs detected in file\n",
      "\n",
      "Seq is: CGGCAATTTGTACACAGTGA\n",
      "0 CGGCAATTTGTACACAGTGA motifs detected in file\n",
      "\n",
      "Seq is: CCCGTCCGCCACGGAGATTA\n",
      "0 CCCGTCCGCCACGGAGATTA motifs detected in file\n",
      "\n",
      "Seq is: TGACCACTACGATCTCCACT\n",
      "0 TGACCACTACGATCTCCACT motifs detected in file\n",
      "\n",
      "Seq is: ACAAAAACCATTTGTCCGCC\n",
      "0 ACAAAAACCATTTGTCCGCC motifs detected in file\n",
      "\n",
      "Seq is: CCATTCTAAACGGCTCAAGG\n",
      "0 CCATTCTAAACGGCTCAAGG motifs detected in file\n",
      "\n",
      "Seq is: AAAATGAAAAAAATTGGAAA\n",
      "0 AAAATGAAAAAAATTGGAAA motifs detected in file\n",
      "\n",
      "Seq is: GGAATACTCGTACGGACACA\n",
      "0 GGAATACTCGTACGGACACA motifs detected in file\n",
      "\n",
      "Seq is: ACCCAACTGTCACTCCCATT\n",
      "0 ACCCAACTGTCACTCCCATT motifs detected in file\n",
      "\n",
      "Seq is: ATTACTGAACAACCAACTAA\n",
      "0 ATTACTGAACAACCAACTAA motifs detected in file\n",
      "\n",
      "Seq is: CATCTATAACCCGACAATTA\n",
      "0 CATCTATAACCCGACAATTA motifs detected in file\n",
      "\n",
      "Seq is: ACAGTCAAGTCACAAAATTA\n",
      "0 ACAGTCAAGTCACAAAATTA motifs detected in file\n",
      "\n",
      "Seq is: GACTGCGTCCGAATACGCCT\n",
      "0 GACTGCGTCCGAATACGCCT motifs detected in file\n",
      "\n",
      "Seq is: CCTCTTACAAAAGTACAATG\n",
      "0 CCTCTTACAAAAGTACAATG motifs detected in file\n",
      "\n",
      "Seq is: AATATGATTGTAATCAAGAA\n",
      "0 AATATGATTGTAATCAAGAA motifs detected in file\n",
      "\n",
      "Seq is: GATATCCCACTATCTAACCA\n",
      "0 GATATCCCACTATCTAACCA motifs detected in file\n",
      "\n",
      "Seq is: GGTTAACCCACACTCCTCAA\n",
      "0 GGTTAACCCACACTCCTCAA motifs detected in file\n",
      "\n",
      "Seq is: GTCAATATACAAACCCTAAA\n",
      "0 GTCAATATACAAACCCTAAA motifs detected in file\n",
      "\n",
      "Seq is: AAATCCATCACCCACAACTC\n",
      "0 AAATCCATCACCCACAACTC motifs detected in file\n",
      "\n",
      "Seq is: GAACTTGCGAAAGAATTAAC\n",
      "0 GAACTTGCGAAAGAATTAAC motifs detected in file\n",
      "\n",
      "Seq is: CACCGACGAAAATCCGGATG\n",
      "0 CACCGACGAAAATCCGGATG motifs detected in file\n",
      "\n",
      "Seq is: ATACCCACAATTTAAAAAAT\n",
      "0 ATACCCACAATTTAAAAAAT motifs detected in file\n",
      "\n",
      "Seq is: GAGAGAGATGTTCCAAAAAA\n",
      "0 GAGAGAGATGTTCCAAAAAA motifs detected in file\n",
      "\n",
      "Seq is: GGATCACAGGTTTCTCGACA\n",
      "0 GGATCACAGGTTTCTCGACA motifs detected in file\n",
      "\n",
      "Seq is: AGGAGAAACCTGATTGTCAA\n",
      "0 AGGAGAAACCTGATTGTCAA motifs detected in file\n",
      "\n",
      "Seq is: TTTAAATGTTCCCCTAAATC\n",
      "0 TTTAAATGTTCCCCTAAATC motifs detected in file\n",
      "\n",
      "Seq is: TCCCAAGACACCCGTTTAAA\n",
      "0 TCCCAAGACACCCGTTTAAA motifs detected in file\n",
      "\n",
      "Seq is: TTTCAACTTGATTCTAAGAT\n",
      "0 TTTCAACTTGATTCTAAGAT motifs detected in file\n",
      "\n",
      "Seq is: AGAACCTGTTGGTCGATAGT\n",
      "0 AGAACCTGTTGGTCGATAGT motifs detected in file\n",
      "\n",
      "Seq is: GGTCCGAGCCATCCAAACAG\n",
      "0 GGTCCGAGCCATCCAAACAG motifs detected in file\n",
      "\n",
      "Seq is: CGGAGATGGATATTTAGAAG\n",
      "0 CGGAGATGGATATTTAGAAG motifs detected in file\n",
      "\n",
      "Seq is: GGTGATAAAACGATGTATCT\n",
      "0 GGTGATAAAACGATGTATCT motifs detected in file\n",
      "\n",
      "Seq is: GCCCACACGAGAAAATCGAC\n",
      "0 GCCCACACGAGAAAATCGAC motifs detected in file\n",
      "\n",
      "Seq is: AAGAATCCATCGAGCAGACC\n",
      "0 AAGAATCCATCGAGCAGACC motifs detected in file\n",
      "\n",
      "Seq is: AAAGCCCCCAGAATCGAAAC\n",
      "0 AAAGCCCCCAGAATCGAAAC motifs detected in file\n",
      "\n",
      "Seq is: CGAGAGGAACGTTTCAATAA\n",
      "0 CGAGAGGAACGTTTCAATAA motifs detected in file\n",
      "\n",
      "Seq is: AGATCAATTAAGTAATACGT\n",
      "0 AGATCAATTAAGTAATACGT motifs detected in file\n",
      "\n",
      "Seq is: CTTCCATATCCCCAATCAGG\n",
      "0 CTTCCATATCCCCAATCAGG motifs detected in file\n",
      "\n",
      "Seq is: AACGATATAATACGAACCAA\n",
      "0 AACGATATAATACGAACCAA motifs detected in file\n",
      "\n",
      "Seq is: TATTAAAAAGTAGAAAGGGA\n",
      "0 TATTAAAAAGTAGAAAGGGA motifs detected in file\n",
      "\n",
      "Seq is: ACGCCATGATATAGATAACG\n",
      "0 ACGCCATGATATAGATAACG motifs detected in file\n",
      "\n",
      "Seq is: CGGTCCAAAGTTAAAGATAG\n",
      "0 CGGTCCAAAGTTAAAGATAG motifs detected in file\n",
      "\n",
      "Seq is: CGGATATGAAATAAACCCAT\n",
      "0 CGGATATGAAATAAACCCAT motifs detected in file\n",
      "\n",
      "Seq is: TTACCAAACCGATTCCAACA\n",
      "0 TTACCAAACCGATTCCAACA motifs detected in file\n",
      "\n",
      "Seq is: GACCATCATTCCACCTCACC\n",
      "0 GACCATCATTCCACCTCACC motifs detected in file\n",
      "\n",
      "Seq is: CAAACCCCGATCCAAATCGA\n",
      "0 CAAACCCCGATCCAAATCGA motifs detected in file\n",
      "\n",
      "Seq is: GTCTCGCCAGTTCAATTCAA\n",
      "0 GTCTCGCCAGTTCAATTCAA motifs detected in file\n",
      "\n",
      "Seq is: CTTTAGAGGATTCACATTCA\n",
      "0 CTTTAGAGGATTCACATTCA motifs detected in file\n",
      "\n",
      "Seq is: ACCCACGAAACACAATTCGA\n",
      "0 ACCCACGAAACACAATTCGA motifs detected in file\n",
      "\n",
      "Seq is: TGTGAGACCAAGCAGGTTCA\n",
      "0 TGTGAGACCAAGCAGGTTCA motifs detected in file\n",
      "\n",
      "Seq is: CGTGAAAGGTCATGTGAATG\n",
      "0 CGTGAAAGGTCATGTGAATG motifs detected in file\n",
      "\n",
      "Seq is: GTACAATGCTGAACAGAGGA\n",
      "0 GTACAATGCTGAACAGAGGA motifs detected in file\n",
      "\n",
      "Seq is: GATATATTTACGCATCCCCA\n",
      "0 GATATATTTACGCATCCCCA motifs detected in file\n",
      "\n",
      "Seq is: AAATCAATTTACAGGAAACT\n",
      "0 AAATCAATTTACAGGAAACT motifs detected in file\n",
      "\n",
      "Seq is: TCATATGAACTCCTCCCACT\n",
      "0 TCATATGAACTCCTCCCACT motifs detected in file\n",
      "\n",
      "Seq is: GCCCGCCACACATGCGCGAA\n",
      "0 GCCCGCCACACATGCGCGAA motifs detected in file\n",
      "\n",
      "Seq is: GTCCCGGGACAAGTTGATTC\n",
      "0 GTCCCGGGACAAGTTGATTC motifs detected in file\n",
      "\n",
      "Seq is: GTGAGATGAGAATCAAATGA\n",
      "0 GTGAGATGAGAATCAAATGA motifs detected in file\n",
      "\n",
      "Seq is: CGATTTAGGTGGAAGCTGGG\n",
      "0 CGATTTAGGTGGAAGCTGGG motifs detected in file\n",
      "\n",
      "Seq is: AATTCAAAGTATTCCCGATA\n",
      "0 AATTCAAAGTATTCCCGATA motifs detected in file\n",
      "\n",
      "Seq is: GCATCAAAAGACCCCATCTT\n",
      "0 GCATCAAAAGACCCCATCTT motifs detected in file\n",
      "\n",
      "Seq is: TTACATCGGGTAAAGAACGG\n",
      "0 TTACATCGGGTAAAGAACGG motifs detected in file\n",
      "\n",
      "Seq is: TGGAGTACCCGATGTGGAAC\n",
      "0 TGGAGTACCCGATGTGGAAC motifs detected in file\n",
      "\n",
      "Seq is: TGGATTGCAGAAATGCACCC\n",
      "0 TGGATTGCAGAAATGCACCC motifs detected in file\n",
      "\n",
      "Seq is: ATGAACGCGAATGAAACATC\n",
      "0 ATGAACGCGAATGAAACATC motifs detected in file\n",
      "\n",
      "Seq is: GGAAGTAGTCCCAAACGACT\n",
      "0 GGAAGTAGTCCCAAACGACT motifs detected in file\n",
      "\n",
      "Seq is: TCTACCGCCATATATCCGAC\n",
      "0 TCTACCGCCATATATCCGAC motifs detected in file\n",
      "\n",
      "Seq is: TCGTTCTCCACCACTCCAAC\n",
      "0 TCGTTCTCCACCACTCCAAC motifs detected in file\n",
      "\n",
      "Seq is: TAGCCCCAAATAGCTAATGT\n",
      "0 TAGCCCCAAATAGCTAATGT motifs detected in file\n",
      "\n",
      "Seq is: CTTGTCCGAGGAGATCTCCC\n",
      "0 CTTGTCCGAGGAGATCTCCC motifs detected in file\n",
      "\n",
      "Seq is: TATACTTCGTGGCGGTCCAG\n",
      "0 TATACTTCGTGGCGGTCCAG motifs detected in file\n",
      "\n",
      "Seq is: GAAACTCAAAATTCGACACC\n",
      "0 GAAACTCAAAATTCGACACC motifs detected in file\n",
      "\n",
      "Seq is: GAGCATCACAAGACCGCTCG\n",
      "0 GAGCATCACAAGACCGCTCG motifs detected in file\n",
      "\n",
      "Seq is: TCAAAACAACTAAATTGACA\n",
      "0 TCAAAACAACTAAATTGACA motifs detected in file\n",
      "\n",
      "Seq is: ACTCCAAATCCCGATTCGTA\n",
      "0 ACTCCAAATCCCGATTCGTA motifs detected in file\n",
      "\n",
      "Seq is: TCACCCCATAGATTAGGGTC\n",
      "0 TCACCCCATAGATTAGGGTC motifs detected in file\n",
      "\n",
      "Seq is: AAACCCAGAATCGATAACAC\n",
      "0 AAACCCAGAATCGATAACAC motifs detected in file\n",
      "\n",
      "Seq is: ACAAGTCTATACAATTTCGG\n",
      "0 ACAAGTCTATACAATTTCGG motifs detected in file\n",
      "\n",
      "Seq is: TGAAAGCATCAGATAAAACA\n",
      "0 TGAAAGCATCAGATAAAACA motifs detected in file\n",
      "\n",
      "Seq is: CAGTTGACCTCAAAAAATGT\n",
      "0 CAGTTGACCTCAAAAAATGT motifs detected in file\n",
      "\n",
      "Seq is: TGAGTCCACTCAAAATCGAA\n",
      "0 TGAGTCCACTCAAAATCGAA motifs detected in file\n",
      "\n",
      "Seq is: ATAACCCCTCCCCCACTAGA\n",
      "0 ATAACCCCTCCCCCACTAGA motifs detected in file\n",
      "\n",
      "Seq is: TTTTGTGAGAAATGCGGCCG\n",
      "0 TTTTGTGAGAAATGCGGCCG motifs detected in file\n",
      "\n",
      "Seq is: AAGATAACTGAACCCAATTA\n",
      "0 AAGATAACTGAACCCAATTA motifs detected in file\n",
      "\n",
      "Seq is: GCACACTGGCGCCACCGACC\n",
      "0 GCACACTGGCGCCACCGACC motifs detected in file\n",
      "\n",
      "Seq is: GTGCTTTAACTGGTTGGGAC\n",
      "0 GTGCTTTAACTGGTTGGGAC motifs detected in file\n",
      "\n",
      "Seq is: CCCAATCATATCGAATCAAT\n",
      "0 CCCAATCATATCGAATCAAT motifs detected in file\n",
      "\n",
      "Seq is: TTGAAAGCAAATAACGATTT\n",
      "0 TTGAAAGCAAATAACGATTT motifs detected in file\n",
      "\n",
      "Seq is: CCAATTAGTGACGACAAAGG\n",
      "0 CCAATTAGTGACGACAAAGG motifs detected in file\n",
      "\n",
      "Seq is: GCACCCCCACACCGATCCGA\n",
      "0 GCACCCCCACACCGATCCGA motifs detected in file\n",
      "\n",
      "Seq is: TTCGCAAAACTCGACGTAAC\n",
      "0 TTCGCAAAACTCGACGTAAC motifs detected in file\n",
      "\n",
      "Seq is: GACGCACGAACTACGAACAA\n",
      "0 GACGCACGAACTACGAACAA motifs detected in file\n",
      "\n",
      "Seq is: GGAAAACTAGCACCACTAAA\n",
      "0 GGAAAACTAGCACCACTAAA motifs detected in file\n",
      "\n",
      "Seq is: TCTCCCACTTGAGTGACCTT\n",
      "0 TCTCCCACTTGAGTGACCTT motifs detected in file\n",
      "\n",
      "Seq is: GCCCCTACGAACGTACACAT\n",
      "0 GCCCCTACGAACGTACACAT motifs detected in file\n",
      "\n",
      "Seq is: TAGAATGATTCTCGATTATC\n",
      "0 TAGAATGATTCTCGATTATC motifs detected in file\n",
      "\n",
      "Seq is: TTTCCGATCCTGGTTTGGAT\n",
      "0 TTTCCGATCCTGGTTTGGAT motifs detected in file\n",
      "\n",
      "Seq is: AAACAAATACCCCACTACAC\n",
      "0 AAACAAATACCCCACTACAC motifs detected in file\n",
      "\n",
      "Seq is: TCGGGCAGATTTGTAAAAGT\n",
      "0 TCGGGCAGATTTGTAAAAGT motifs detected in file\n",
      "\n",
      "Seq is: CACATAACGAAACTCCTCCA\n",
      "0 CACATAACGAAACTCCTCCA motifs detected in file\n",
      "\n",
      "Seq is: TTCGATGTATTTGACACCCC\n",
      "0 TTCGATGTATTTGACACCCC motifs detected in file\n",
      "\n",
      "Seq is: CCACAGAAACCCCAAACCAA\n",
      "0 CCACAGAAACCCCAAACCAA motifs detected in file\n",
      "\n",
      "Seq is: CCAAGCCCCATACCCCAATC\n",
      "0 CCAAGCCCCATACCCCAATC motifs detected in file\n",
      "\n",
      "Seq is: GTCGCCACACACACACGACC\n",
      "0 GTCGCCACACACACACGACC motifs detected in file\n",
      "\n",
      "Seq is: CATCCTACCCGCCCCCAACA\n",
      "0 CATCCTACCCGCCCCCAACA motifs detected in file\n",
      "\n",
      "Seq is: TAACTACTCTAATCATCATA\n",
      "0 TAACTACTCTAATCATCATA motifs detected in file\n",
      "\n",
      "Seq is: CCCTCACCCTCCCCTTTTAT\n",
      "0 CCCTCACCCTCCCCTTTTAT motifs detected in file\n",
      "\n",
      "Seq is: TACACAATCAACCCCCCACT\n",
      "0 TACACAATCAACCCCCCACT motifs detected in file\n",
      "\n",
      "Seq is: GACAATTTTCACGTATGGCG\n",
      "0 GACAATTTTCACGTATGGCG motifs detected in file\n",
      "\n",
      "Seq is: GTTTTCTATTTTAAACTTTA\n",
      "0 GTTTTCTATTTTAAACTTTA motifs detected in file\n",
      "\n",
      "Seq is: GACCAATCCGACCACAATCC\n",
      "0 GACCAATCCGACCACAATCC motifs detected in file\n",
      "\n",
      "Seq is: CAAGAAACAAAAACCCCAAA\n",
      "0 CAAGAAACAAAAACCCCAAA motifs detected in file\n",
      "\n",
      "Seq is: CCGTCTCTACACAAATTCAC\n",
      "0 CCGTCTCTACACAAATTCAC motifs detected in file\n",
      "\n",
      "Seq is: GACACCGGTCTTCGCCCCCT\n",
      "0 GACACCGGTCTTCGCCCCCT motifs detected in file\n",
      "\n",
      "Seq is: CCCCCCCAAACCACCTTTAA\n",
      "0 CCCCCCCAAACCACCTTTAA motifs detected in file\n",
      "\n",
      "Seq is: AAAACAATACTACAGACACA\n",
      "0 AAAACAATACTACAGACACA motifs detected in file\n",
      "\n",
      "Seq is: CCTTTCACCGACACGTCTGT\n",
      "0 CCTTTCACCGACACGTCTGT motifs detected in file\n",
      "\n",
      "Seq is: AAGTTAACAATAATAATACA\n",
      "0 AAGTTAACAATAATAATACA motifs detected in file\n",
      "\n",
      "Seq is: GGATGTTCGTAATTAATTAA\n",
      "0 GGATGTTCGTAATTAATTAA motifs detected in file\n",
      "\n",
      "Seq is: TTGTGTGAAATCATTCATAC\n",
      "0 TTGTGTGAAATCATTCATAC motifs detected in file\n",
      "\n",
      "Seq is: AAGCGGACATTATAACTTGC\n",
      "0 AAGCGGACATTATAACTTGC motifs detected in file\n",
      "\n",
      "Seq is: ATCCACGCTATTTATTATCC\n",
      "0 ATCCACGCTATTTATTATCC motifs detected in file\n",
      "\n",
      "Seq is: TACTCCGTCCTTAGTTTCTG\n",
      "0 TACTCCGTCCTTAGTTTCTG motifs detected in file\n",
      "\n",
      "Seq is: TCTATGACGCTGTATCCCAC\n",
      "0 TCTATGACGCTGTATCCCAC motifs detected in file\n",
      "\n",
      "Seq is: GAGGCCGAGGTCGCAGAGCG\n",
      "0 GAGGCCGAGGTCGCAGAGCG motifs detected in file\n",
      "\n",
      "Seq is: TTACGATAGCGCACGTATGG\n",
      "0 TTACGATAGCGCACGTATGG motifs detected in file\n",
      "\n",
      "Seq is: GGGGTCTGCTTTTATGGTTT\n",
      "0 GGGGTCTGCTTTTATGGTTT motifs detected in file\n",
      "\n",
      "Seq is: ACGTACCTCTCGAGGGCACT\n",
      "0 ACGTACCTCTCGAGGGCACT motifs detected in file\n",
      "\n",
      "Seq is: CACCAATTATCCCACTATCT\n",
      "0 CACCAATTATCCCACTATCT motifs detected in file\n",
      "\n"
     ]
    }
   ],
   "source": [
    "def complements(s):\n",
    "    \"\"\"\n",
    "    :params s: String\n",
    "    \"\"\"\n",
    "    complement = s.replace('A','t') + s.replace('T','a') + s.replace('G', 'c') + s.replace('C', 'g')\n",
    "\n",
    "\n",
    "\n",
    "from tabulate import tabulate\n",
    "\n",
    "import re\n",
    "\n",
    "chromosomal = []\n",
    "with open('/home/josh/Documents/MA5114_ProgrammingforBio/chrM.fa') as pain_file:\n",
    "    for line in pain_file:\n",
    "        line = line.strip()\n",
    "        line = line.upper()\n",
    "        if line != \"\" and line[0].isalpha():\n",
    "            chromosomal.append(line)#cleans up and spits out the .fa file as a single string\n",
    "chromosomal = \"\".join(chromosomal)\n",
    "\n",
    "\n",
    "\n",
    "complement = chromosomal[::-1]\n",
    "            \n",
    "windowlength = list(range(6,21,1))\n",
    "\n",
    "\n",
    "tabledata = []\n",
    "chunklist = []\n",
    "\n",
    "for length in windowlength:\n",
    "    n = length\n",
    "    chunks = [complement[i:i+n] for i in range(0, len(complement), n)]\n",
    "    for chunk in chunks: #breaks down the complement into chunks based on the size of the window(6-20)\n",
    "        if len(chunk) == length:\n",
    "            searchseq = chunk\n",
    "            motif_regex = re.compile(f'{searchseq}') #The sequence we're searching for, i.e. the earlier chunk\n",
    "            runs = re.finditer(motif_regex,chromosomal) #finditer used because we want to scan entire code\n",
    "            match_count=0 #used to keep track of the number of hits\n",
    "\n",
    "            for match in runs:\n",
    "                match_count += 1\n",
    "                run_start = match.start()\n",
    "                run_end = match.end()\n",
    "                match_seq = chromosomal[run_start:run_end] #isolates the part of the string corresponding to the ctcf motif for later printing\n",
    "                print(\"Complement found from \" + str(run_start) + \" to \" +str(run_end))\n",
    "                chunklist.append(match)\n",
    "            print(\"Seq is: \"+searchseq)\n",
    "            print(f\"{match_count} {searchseq} motifs detected in file\") \n",
    "            print(\"\")\n",
    "\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
